<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-03 09:25:50 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>27</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>64</td>
          <td>135</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, , , Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>104</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, , A. Henssen, Howard Y. Chang, , P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>309</td>
          <td>135</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>266</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent driver of cancer, whose random segregation promotes aggressive tumors. Acentric ecDNAs attach to chromosomes during mitosis for segregation. However, the molecular mechanism governing ecDNA-chromosome mitotic interactions remains poorly understood. This study shows that ecDNAs attach to histone 3 lysine 27 acetylation (H3K27ac)-marked chromatin during mitosis. H3K27ac depletion resulted in ecDNA detachment from mitotic chromosomes. Diverse bromodomain proteins, which are known readers of H3K27ac, stabilize ecDNAs’ mitotic interaction, exhibiting context-dependent and mutually complementary roles. Furthermore, disruptions of the Mediator complex and RNA polymerase II transcription activity both dissociate ecDNAs from mitotic chromosomes, suggesting that the transcription machinery mediates ecDNA segregation. Mis-segregated ecDNAs were expelled into the cytosol and degraded, leading to diminished oncogene expression and a reversal of therapy resistance. Our research provides new insights into the interplay between RNA transcription and acentric ecDNA inheritance in cancer, offering a novel avenue for disrupting ecDNA-driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b99e11fc3712d503e0a3c800fc35c4668a35ad3" target='_blank'>
              Transcription Machinery Anchors ecDNAs to Mitotic Chromosomes for Segregation
              </a>
            </td>
          <td>
            Yipeng Xie, Jun Yi Stanley Lim, Wenyue Liu, C. Gilbreath, Yoon Jung Kim, Sihan Wu
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e5d3752411db0d4a1f1f0e0c0f053217196ee" target='_blank'>
              Unveiling the mysteries of extrachromosomal circular DNA: from generation to clinical relevance in human cancers and health
              </a>
            </td>
          <td>
            Zilong Wang, , Wenli Zhu, Xiaoning Hong, Zhen Xu, Shuang Mao, , Peng Han, Chunxiao He, Changze Song, Xi Xiang
          </td>
          <td>2024-12-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="LINE-1 (L1) retrotransposition is widespread in many cancers, especially those with a high burden of chromosomal rearrangements. However, whether and to what degree L1 activity directly impacts genome integrity is unclear. Here, we apply whole-genome sequencing to experimental models of L1 expression to comprehensively define the spectrum of genomic changes caused by L1. We provide definitive evidence that L1 expression frequently and directly causes both local and long-range chromosomal rearrangements, small and large segmental copy-number alterations, and subclonal copy-number heterogeneity due to ongoing chromosomal instability. Mechanistically, all these alterations arise from DNA double-strand breaks (DSBs) generated by L1-encoded ORF2p. The processing of ORF2p-generated DSB ends prior to their ligation can produce diverse rearrangements of the target sequences. Ligation between DSB ends generated at distal loci can generate either stable chromosomes or unstable dicentric, acentric, or ring chromosomes that undergo subsequent evolution through breakage-fusion bridge cycles or DNA fragmentation. Together, these findings suggest L1 is a potent mutagenic force capable of driving genome evolution beyond simple insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b6d0c6b40e4e17794e34761d2e6492abd74c31" target='_blank'>
              Chromosomal rearrangements and instability caused by the LINE-1 retrotransposon
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Xi Zeng, Jennifer A. Karlow, Phillip Schofield, Serafina Turner, Jupiter Kalinowski, Danielle Denisko, E. A. Lee, Kathleen H. Burns, Cheng-Zhong Zhang
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, , Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, , Irene Roberts, Ronald W. Stam, , James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Lampbrush chromosomes are giant meiotic bivalents in growing oocyte nuclei that have served as a classic model system for studying chromatin organization and RNA synthesis for over a century. Despite their importance, the molecular mechanisms underlying lampbrush chromosomes formation and their distinctive chromomere-loop architecture have remained poorly understood. Here, we provide the first comprehensive genomic, cytological, and biophysical analysis of lampbrush chromosome organization by integrating single-cell Hi-C, RNA-seq, NOMe-seq, FISH mapping, and chromatin simulations. Single-nucleus Hi-C analysis revealed CTCF-independent contact domains with stable boundaries defined by transcription units in a convergent orientation. Сontact domains identified through Hi-C analysis correspond to insulated chromomeres in lampbrush chromosomes. Small transcriptionally inactive contact domains surrounded by transcription units in the diverged orientation form ‘chromatin knots’, which are often detached from the chromosome axis. Transcription loops frequently manifest as a ‘cross’ pattern with reduced contacts within chromatin domains. Integrative analysis of the whole-genome data uncovers the mechanisms underlying lampbrush chromosome structure, revealing how hypertranscription modulates chromatin stiffness and repositions SMC complexes to establish the distinctive chromomere-loop organisation. Biophysical modeling through polymer simulation reproduces key features of lampbrush chromosomes, including transcription loop formation, chromomere compaction, and insulation patterns. These findings offer a unifying framework for understanding the remarkable chromatin architecture of lampbrush chromosomes and their transcription-dependent organization. Highlights - First integration of single-cell Hi-C, RNA-seq, NOMe-seq and microscopy methods uncovers molecular mechanisms underlying lampbrush chromosome architecture. - Hi-C reveals contact patterns corresponding to lampbrush chromomeres and transcription loops, validated through BAC-based FISH mapping. - Lampbrush chromosomes are segmented into contact domains formed via a CTCF-independent mechanism, with boundaries coinciding with convergently oriented gene pairs. - Hypertranscription shapes lampbrush chromosome through multiple mechanisms, increasing stiffness and decreasing compaction of transcribed units, generating outward pressure, pushing transcription loops away from the chromosome axis, and repositioning SMC complexes to form transcription-dependent domains with stable boundaries. - Polymer simulations including these mechanisms replicate key features of LBC organization, supporting the proposed model of lampbrush chromosomes formation. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e642c22825b99c48d9f70a1c1f9236c18b4d5e" target='_blank'>
              The 3D genomics of lampbrush chromosomes highlights the role of active transcription in chromatin organization
              </a>
            </td>
          <td>
            T. Lagunov, M. Gridina, A. Nurislamov, T. Kulikova, A. Maslova, V. Konstantinov, A. Popov, A. Krasikova, V. Fishman
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f0a75813c454a5e8e9bd04b719f1b32b01034b" target='_blank'>
              ZIC1 is a context-dependent medulloblastoma driver in the rhombic lip
              </a>
            </td>
          <td>
            John J. Y. Lee, Ran Tao, Z. You, P. Haldipur, Anders W Erickson, H. Farooq, Liam D Hendriske, Namal Abeysundara, Cory Richman, Evan Y Wang, Neha Das Gupta, Jennifer L. Hadley, Melissa Batts, Christopher W. Mount, Xiaochong Wu, Alexandra Rasnitsyn, Swneke Bailey, F. Cavalli, Sorana A. Morrissy, Livia Garzia, K. Michealraj, , Vernon Fong, Jonelle Palotta, Raúl A. Suárez, Bryn Livingston, Miao Liu, B. Luu, Craig Daniels, J. Loukides, A. Bendel, P. French, J. Kros, Andrey Korshunov, Marcel Kool, F. Chico Ponce de León, M. Perezpeña-Diazconti, Boleslaw Lach, Sheila K Singh, S. Leary, Byung-Kyu Cho, Seung-Ki Kim, Kyu-Chang Wang, Ji-Yeoun Lee, Teiji Tominaga, William A. Weiss, Joanna J Phillips, Shizhong Dai, Gelareh Zadeh, Ali G Saad, L. Bognár, Á. Klekner, Ian F. Pollack, Ronald L Hamilton, Young-Shin Ra, W. Grajkowska, M. Perek-Polnik, Reid C. Thompson, Anna M. Kenney, Michael K. Cooper, S C Mack, Nada Jabado, Mathieu Lupien, Marco Gallo, Vijay Ramaswamy, M. Suvà, Hiromishi Suzuki, K. Millen, L. F. Huang, P. Northcott, Michael D. Taylor
          </td>
          <td>2025-01-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/865e3224ea8530a73a268350cf8b48819bfe3161" target='_blank'>
              Inter-chromosomal transcription hubs shape the 3D genome architecture of African trypanosomes
              </a>
            </td>
          <td>
            Claudia Rabuffo, Markus R Schmidt, Prateek Yadav, Pin Tong, Roberta Carloni, Anna Barcons-Simon, Raúl O. Cosentino, Stefan Krebs, Keith R. Matthews, Robin C. Allshire, Nicolai Siegel
          </td>
          <td>2024-12-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Characterization of tumor epigenetic aberrations is integral to understanding the mechanisms of tumorigenesis and provide diagnostic, prognostic, and predictive information of high clinical relevance. Among the different tumor-associated epigenetic signatures, 5 methyl-cytosine (5mC) and 5-hydroxymethylcytosine (5hmC) are the two most well-characterized DNA methylation alterations linked to cancer pathogenesis. 5hmC has a tissue-specific distribution and its abundance is subjected to changes in tumor DNA, making it a promising biomarker. Detecting tumor-related DNA methylation alterations in tissues is highly invasive, while the analysis of the cell-free DNA (cfDNA) is poised to supplement, if not replace, surgical biopsies. Despite many studies attempted to identify new epigenetic targets for liquid biopsy assays, little is known about the regulatory roles of 5hmC, its impacts on the molecular phenotypes in tumors. Most importantly, whether the oncogenic-associated 5hmC signatures found in tumor tissues can be recapitulated in patient cfDNA. In this study, we performed the unbiased and simultaneous detection of 5mC and 5hmC whole-genome DNA modifications at base-resolution from two distinct cancer cohorts, from patients with bladder cancer or B-Cell lymphoma, their corresponding normal tissues, and cfDNAs from plasma. We analyzed tissue-specific methylation patters and searched for signatures in gene coding and regulatory regions linked to cancerous states. We then looked for methylation signatures in patients cfDNA to determine if they were consistent with the tumor-specific patterns. We determined the functional significance of 5hmC in tissue specific transcription and uncovered hundreds of tumor-associated 5hmC signatures. These tumor-associated 5hmC changes, particularly in genes and enhancers, were functionally significant in tumorigenesis pathways and correlated with tumor specific gene expression. To investigate if cfDNA is a faithful surrogate for tumor-associated 5hmC, we devised a targeted capture strategy to examine the alterations of 5hmC in cfDNA from patients with bladder cancer and lymphoma with sufficient sensitivity and specificity and confirmed that they recapitulated the patterns we observed in tumor tissues. Our results provide analytic validation of 5hmC as a cancer-specific biomarker. The methods described here for systematic characterization of 5hmC at functional elements open new avenues to discover epigenetic markers for non-invasive diagnosis, monitoring, and stratifying cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a233a1b3a2e680f52000d36e68abfc546f2d4b33" target='_blank'>
              Global DNA methylomes reveal oncogenic-associated 5-hydroxylmethylated cytosine (5hmC) signatures in the cell-free DNA of cancer patients
              </a>
            </td>
          <td>
            Gabriel E Rech, Alyssa C Lau, Rachel L Goldfeder, Rahul Maurya, Alexey V Danilov, 
          </td>
          <td>2025-01-15</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a6c622797b8d89bba676a8b9a3206b6fe5c1df9" target='_blank'>
              A constitutive heterochromatic region shapes genome organization and impacts gene expression in Neurospora crassa
              </a>
            </td>
          <td>
            Andrew T. Reckard, Abhishek Pandeya, Jacob M. Voris, Carlos G. Gonzalez Cruz, Oluwatosin Oluwadare, Andrew D. Klocko
          </td>
          <td>2024-12-20</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="RNase H2 is a heterotrimeric endoribonuclease that resolves RNA:DNA hybrids and genome-embedded ribonucleotides, which are implicated in DNA replication stress and cancer development. Protein and/or mRNA levels of individual RNase H2 subunits are elevated in some cancers, but little is known about the mechanisms or consequences of RNase H2 upregulation. We report that RNase H2 subunits are upregulated at the protein level in response to replication stress induced by oncogenes and chemotherapy drugs in human cancer and non-cancer cell lines. We show that inducible overexpression of the RNASEH2B subunit increases levels of the active RNase H2 heterotrimer. While causing only subtle changes to gene expression, RNASEH2B overexpression is unexpectedly associated with increased RNA:DNA hybrid levels. RNASEH2B overexpression prevents further increases in RNA:DNA hybrid levels by camptothecin or hydroxyurea and reduces replication fork stalling in presence of these drugs. Surprisingly, RNase H2 levels do not strongly impact survival of chemotherapy treatments but appear to have more subtle effects on genome instability and innate immune signalling. In contrast, increased RNase H2 levels in presence of oncogenic HRAS limit not only RAS-induced replication fork stalling but also cell death. Our findings shed new light on the functions of RNase H2 and suggest that upregulation of RNase H2 may be an important aspect of replication stress responses in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54af3146eb7acac3bc5186f9ac5d0287ea7b019a" target='_blank'>
              Human RNase H2 upregulation counteracts oncogene- and chemotherapy-induced replication stress
              </a>
            </td>
          <td>
            Rosanna J. Wilkins, Abirami Kannan, Siobhan A. Plass, Claire Wilson, Richard D. W. Kelly, Claire H. M. Tang, P. Kotsantis, Martin A. M. Reijns, Aditi Kanhere, Eva Petermann
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd6b6473526b044dda123e9fa9d72ac0fc045ca" target='_blank'>
              A dual role of Cohesin in DNA DSB repair
              </a>
            </td>
          <td>
            Michael Fedkenheuer, Yafang Shang, Seolkyoung Jung, Kevin Fedkenheuer, Solji Park, Davide Mazza, Robin Sebastian, Hiroyuki Nagashima, D. Zong, Hua Tan, S. Jaiswal, Haiqing Fu, Anthony Cruz, Supriya V Vartak, Jan Wisniewski, Vittorio Sartorelli, John J O'Shea, L. Elnitski, A. Nussenzweig, M. Aladjem, Fei Meng, Rafael Casellas
          </td>
          <td>2025-01-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Two decades after the initial report on increased micronuclei in human chronic liver disease (CLD) and hepatocellular carcinoma (HCC), their role in HCC development is still poorly understood. Here we show that micronuclei in hepatocytes trigger hepatic immune response and promote HCC development via an increased level of extrachromosomal circular DNA (eccDNA). Livers from a CLD model (Mcl1Δhep mice) show increased micronuclei in parallel to eccDNA. Circular sequencing confirms higher eccDNA levels in micronuclei compared to primary nuclei. We developed nuclei-segregated DNA fiber (NuSeF) assay to show that micronuclei are more susceptible to replication stress, showing increased replication fork slowing. By comparing different murine liver disease models, high eccDNA is correlated with increased tumor incidence. EccDNA is a strong immunostimulant and promotes a crosstalk between hepatocytes and immune cells through the cGAS-STING pathway. Deletion of Sting1 in Mcl1Δhep mice reduces immune cell chemotaxis, as well as tumor incidence. Our findings suggest that eccDNA from micronuclei mediates inflammation-driven liver carcinogenesis in CLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/563897a8a27ca1930e83755c188ad3a2e992e96a" target='_blank'>
              Extrachromosomal circular DNA promotes inflammation and hepatocellular carcinoma development
              </a>
            </td>
          <td>
            Lap Kwan Chan, Juanjuan Shan, Elias Rodriguez-Fos, Marc E. Healy, Peter Leary, Rossella Parrotta, Nina Desboeufs, Gabriel Semere, Nadine Wittstruck, A. Henssen, Achim Weber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Histones scaffold genomic DNA and regulate access to the transcriptional machinery. However, naturally occurring histone variants can alter histone-DNA interactions, DNA and histone modifications, and the chromatin interactome. Hence, alterations in histone variant deposition can disrupt chromatin, and are increasingly recognized as a way to trigger various disease, including cancer. While significant attention has been placed on the biochemical and functional roles of H2A, H3, and H4 histone variants, the variants of H2B remain largely understudied. Here, we show that H2B variants are dysregulated in breast cancer and that certain variants are associated with specific breast cancer subtypes. HIST1H2BO overexpression (in particular) is more common in Asian, African American/Black, and young female populations and is associated with a worse prognosis. In vitro studies show that H2B1O compacts nucleosome structure. Incorporating H2B1O into chromatin activates pro-inflammatory and oncogenic pathways, induces the epithelial-to-mesenchymal transition (EMT), and generates resistance to first-line chemotherapeutic agents. Thus, H2B1O acts much like an onco-histone, with H2B variant expression being a prognostic biomarker for breast cancer and a potential new target for drug therapies to enhance treatment efficacy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afd8a4ca936d0e08dadf10708fe50ded7fd3756c" target='_blank'>
              Incorporating histone H2B variants into chromatin modifies chromatin accessibility to induce epithelial-to-mesenchymal transition in breast cancer
              </a>
            </td>
          <td>
            Hejer Dhahri, Kin H. Lau, Wesley N. Saintilnord, Elisson Lopes, Hannah Damico, Flavio R. Palma, Daniёl P. Melters, Darrell P Chandler, Yamini Dalal, Jonathan D. Licht, Marcello G. Bonini, Yvonne N Fondufe-Mittendorf
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Loss of DNA methylation is a hallmark of cancer that is proposed to promote carcinogenesis through gene expression alterations, retrotransposon activation and induction of genomic instability. Cancer-associated hypomethylation does not occur across the whole genome but leads to the formation of partially methylated domains (PMDs). However, the mechanisms underpinning PMD formation remain unclear. PMDs replicate late in S-phase leading to the proposal that they become hypomethylated due to incomplete re-methylation by the maintenance methyltransferase DNMT1 during cell division. Here we investigate the role of DNMT1 in the formation of PMDs in cancer by conducting whole genome bisulfite sequencing (WGBS), repli-seq and ChIP-seq on DNMT1 knockout HCT116 colorectal cancer cells (DNMT1 KO cells). We find that DNMT1 loss leads to preferential hypomethylation in late replicating, heterochromatic PMDs marked by the constitutive heterochromatic mark H3K9me3 or the facultative heterochromatic mark H3K27me3. However, we also observe that a subset of H3K9me3-marked PMDs gain methylation in DNMT1 KO cells. We find that, in DNMT1 KO cells, these hypermethylated PMDs remain late replicating but gain DNMT3A localisation. This is accompanied by loss of heterochromatic H3K9me3 and specific gain of euchromatic H3K36me2. Our observations suggest that hypermethylated PMDs lose their heterochromatic state, enabling their methylation by DNMT3A and the establishment of a hypermethylated, non-PMD state, despite their late replication timing. More generally, our findings suggest that the de novo DNMTs play a key role in establishing domain level DNA methylation patterns in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d7b5eef5f42a16c3caf9791eabcbd6893beae10" target='_blank'>
              DNMT1 loss leads to hypermethylation of a subset of late replicating domains by DNMT3A
              </a>
            </td>
          <td>
            Ioannis Kafetzopoulos, Francesca Taglini, H. Davidson-Smith, D. Sproul
          </td>
          <td>2024-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="An aggressive subtype of acute myeloid leukemia (AML) is caused by enhancer hijacking resulting in MECOM overexpression. Several chromosomal rearrangements can lead to this: the most common (inv(3)/t(3;3)) results in a hijacked GATA2 enhancer, and there are several atypical MECOM rearrangements involving enhancers from other hematopoietic genes. The set of enhancers which can be hijacked by MECOM can also be hijacked by BCL11B. Enhancer deregulation is also a driver of oncogenesis in a range of other malignancies. The mechanisms of enhancer deregulation observed in other cancer types, including TAD boundary disruptions and the creation of de novo (super-) enhancers, may explain overexpression of MECOM or other oncogenes in AML without enhancer hijacking upon translocation. Gaining mechanistic insight in both enhancer deregulation and super-enhancer activity is critical to pave the way for new treatments for AML and other cancers that are the result of enhancer deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88804fb71dad9d4f7cceb8ac265d5c25968818" target='_blank'>
              Mechanisms of enhancer-driven oncogene activation.
              </a>
            </td>
          <td>
            Joyce Vriend, Ruud Delwel, Dorien Pastoors
          </td>
          <td>2025-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="All great apes differ karyotypically from humans due to the fusion of chromosomes 2a and 2b, resulting in human chromosome 2. Yet, the structure, function, and evolutionary history of the genomic regions associated with this fusion remain poorly understood. Here, we analyze finished telomere-to-telomere chromosomes in great apes and macaques to show that the fusion was associated with multiple pericentric inversions, segmental duplications (SDs), and the rapid turnover of subterminal repetitive DNA. We characterized the fusion site at single-base-pair resolution and identified three distinct SDs that originated more than 5 million years ago. These three distinct SDs were differentially distributed among African great apes as a result of incomplete lineage sorting (ILS) and lineage-specific duplication. Most conspicuously, one of these SDs shares homology to a hypomethylated SD spacer sequence present in hundreds of copies in the subterminal heterochromatin of chimpanzees and bonobos. The fusion in human was accompanied by a systematic degradation of the three divergent α-satellite arrays representing the ancestral centromere creating five distinct structural haplotypes in humans. CRISPR/Cas9-mediated depletion of the fusion site in human cell lines significantly alters the expression of 108 genes, indicating a potential regulatory consequence to this human-specific karyotypic change.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b16610644425427f06758c18d45678c36f10190" target='_blank'>
              Incomplete lineage sorting of segmental duplications defines the human chromosome 2 fusion site early during African great ape speciation
              </a>
            </td>
          <td>
            Xinrui Jiang, Lu Zhang, Zikun Yang, Xiangyu Yang, Kaiyue Ma, DongAhn Yoo, Yong Lu, Shilong Zhang, Jieyi Chen, Yanhong Nie, Xinyan Bian, Junmin Han, , Juan Zhang, Guojie Zhang, Qiang Sun, E. Eichler, Yafei Mao
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved using specialized helicases. Previous in vitro studies showed that DEAH-box Helicase 36 (DHX36/G4R1/RHAU), has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout (KO) of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 mRNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6a2067e514e41d31ed3af27971fbc66a915c36" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, 
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="A significant number of castration-resistant prostate cancer (CRPC) evolve into a neuroendocrine (NE) subtype termed NEPC, leading to resistance to androgen receptor (AR) pathway inhibitors and poor clinical outcomes. Through Hi-C analyses of a panel of patient-derived xenograft tumors, here we report drastically different 3D chromatin architectures between NEPC and CRPC samples. Such chromatin re-organization was faithfully recapitulated in vitro on isogenic cells undergoing NE transformation (NET). Mechanistically, neural transcription factor (TF) NKX2-1 is selectively and highly expressed in NEPC tumors and is indispensable for NET across various models. NKX2-1 preferentially binds to gene promoters, but it interacts with chromatin-pioneering factors such as FOXA2 at enhancer elements through chromatin looping, further strengthening FOXA2 binding at NE enhancers. Conversely, FOXA2 mediates regional DNA demethylation, attributing to NE enhancer priming and inducing NKX2-1 expression, forming a feed-forward loop. Single-cell multiome analyses of isogenic cells over time-course NET cells identify individual cells amid luminal-to-NE transformation, exhibiting intermediate epigenetic and transcriptome states. Lastly, NKX2-1/FOXA2 interacts with, and recruits CBP/p300 proteins to activate NE enhancers, and pharmacological inhibitors of CBP/p300 effectively blunted NE gene expression and abolished NEPC tumor growth. Thus, our study reports a hierarchical network of TFs governed by NKX2-1 in regulating the 2D and 3D chromatin re-organization during NET and uncovers a promising therapeutic approach to eradicate NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f60ea05e69028e7c615b1e7620f9deafcd0cdd" target='_blank'>
              Epigenetic remodeling and 3D chromatin reorganization governed by NKX2-1 drive neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Xiaodong Lu, Viriya Keo, Irina Cheng, Wanqing Xie, Galina A Gritsina, , Q. Jin, Peng Jin, Feng Yue, Martin G. Sanda, Victor Corces, Nicolas Altemose, Jonathan C. Zhao, Jindan Yu
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Ovarian cancer is a deadly gynecological disease with frequent recurrence. Current treatments for patients include platinum-based therapy regimens with PARP inhibitors specific for HR-deficient high-grade serous ovarian cancers (HGSOCs). Despite initial effectiveness, patients inevitably develop disease progression as tumor cells acquire resistance. Toward the development of new therapeutic avenues, we describe a gene amplification involving both HSF1 and MYC, wherein these two genes are co-amplified in over 30% of HGSCO patients. We further found that HSF1 and MYC transcriptional activity was highly correlated in human HGSOC tumors and cell lines, suggesting they may cooperate in the disease. CUT&RUN for HSF1 and MYC revealed overlapping HSF1 and MYC binding throughout the genome. Moreover, binding peaks of both transcription factors in HGSOC cells were nearly identical, and a protein-protein interaction between HSF1 and MYC was detected, supporting molecular cooperation. Supporting a functional cooperation of these two transcription factors, growth of HGSOC cells with the co-amplification was dependent on both HSF1 and MYC. To identify a therapeutic target that could take advantage of this unique HSF1 and MYC dependency, polo-like kinase 1 (PLK1) was correlated with HSF1 and MYC in HGSOC specimens. Targeting PLK1 with volasertib revealed a greater than 200-fold increased potency in HSF1-MYC co-amplified HGSOC cells compared to those with wild-type HSF1 and MYC copy number. Although the success of volasertib and other PLK1 inhibitors in clinical trials has been modest, the current study suggests that targeting PLK1 in a precision medicine approach using HSF1-MYC co-amplification as a biomarker in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37b1725a2f0ac79cd573a1bb5f9e94eafaf83ac" target='_blank'>
              MYC and HSF1 Cooperate to Drive Sensitivity to Polo-like Kinase 1 Inhibitor Volasertib in High Grade Serous Ovarian Cancer.
              </a>
            </td>
          <td>
            Imade Williams, Matthew O'Malley, Haddie DeHart, Bobby Walker, , Pranav Jothirajah, Haimanti Ray, Lisa M Landrum, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2025-01-20</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6960e9d1d69ddbed72b6eb04c8ec6b5d4992a9b7" target='_blank'>
              Structural polymorphism and diversity of human segmental duplications.
              </a>
            </td>
          <td>
            Hyeonsoo Jeong, Philip C. Dishuck, DongAhn Yoo, William T. Harvey, Katherine M. Munson, , Jennifer R. Kordosky, Gage H. Garcia, F. Yilmaz, P. Hallast, Charles Lee, T. Pastinen, E. Eichler
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="We lack tools to edit DNA sequences at scales necessary to study 99% of the human genome that is noncoding. To address this gap, we applied CRISPR prime editing to insert recombination handles into repetitive sequences, up to 1697 per cell line, which enables generating large-scale deletions, inversions, translocations, and circular DNA. Recombinase induction produced more than 100 stochastic megabase-sized rearrangements in each cell. We tracked these rearrangements over time to measure selection pressures, finding a preference for shorter variants that avoided essential genes. We characterized 29 clones with multiple rearrangements, finding an impact of deletions on expression of genes in the variant but not on nearby genes. This genome-scrambling strategy enables large deletions, sequence relocations, and the insertion of regulatory elements to explore genome dispensability and organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a90f87137d367e42fa5bf7ac26c12b1be111991" target='_blank'>
              Randomizing the human genome by engineering recombination between repeat elements.
              </a>
            </td>
          <td>
            Jonas Koeppel, Raphael Ferreira, T. Vanderstichele, Lisa M. Riedmayr, Elin Madli Peets, Gareth Girling, Juliane Weller, Pierre Murat, Fabio Giuseppe Liberante, Tom Ellis, George M. Church, Leopold Parts
          </td>
          <td>2025-01-31</td>
          <td>Science</td>
          <td>2</td>
          <td>8</td>
        </tr>

        <tr id="Kinase-related gene fusion and point mutations play pivotal roles as drivers in cancer, necessitating optimized, targeted therapy against these alterations. The efficacy of molecularly targeted therapeutics varies depending on the specific alteration, with great success reported for such therapeutics in the treatment of cancer with kinase fusion proteins. However, the involvement of actionable alterations in solid tumors, especially regarding kinase fusions, remains unclear. Therefore, in this study, we aimed to compare the number of actionable alterations in patients with tyrosine or serine/threonine kinase domain fusions, mutations, and copy number alterations (CNAs). We analyzed 613 patients with 40 solid cancer types who visited our division between June 2020 and April 2024. Furthermore, to detect alterations involving multiple-fusion calling, we performed comprehensive genomic sequencing using FoundationOne® companion diagnostic (F1CDx) and FoundationOne® Liquid companion diagnostic (F1LCDx). Patient characteristics and genomic profiles were analyzed to assess the frequency and distribution of actionable alterations across different cancer types. Notably, 44 of the 613 patients had fusions involving kinases, transcriptional regulators, or tumor suppressors. F1CDx and F1LCDx detected 13 cases with kinase-domain fusions. We identified 117 patients with kinase-domain mutations and 58 with kinase-domain CNAs. The number of actionable alterations in patients with kinase-domain fusion, mutation, or CNA (median [interquartile range; IQR]) was 2 (1–3), 5 (3–7), and 6 (4–8), respectively. Patients with kinase fusion had significantly fewer actionable alterations than those with kinase-domain mutations and CNAs. However, those with fusion involving tumor suppressors tended to have more actionable alterations (median [IQR]; 4 [2–9]). Cancers with kinase fusions exhibited fewer actionable alterations than those with kinase mutations and CNAs. These findings underscore the importance of detecting kinase alterations and indicate the pivotal role of kinase fusions as strong drivers of cancer development, highlighting their potential as prime targets for molecular therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5249feb71c9f79a3c5d96e6debe6951cd92ec6" target='_blank'>
              Comparison of actionable alterations in cancers with kinase fusion, mutation, and copy number alteration
              </a>
            </td>
          <td>
            S. Suzuki, T. Akahane, A. Tanimoto, M. Higashi, I. Kitazono, M. Kirishima, M. Nishigaki, T. Ikeda, S. Kanemitsu, J. Nakazawa, E. Akahane, H. Nishihara, Kimiharu Uozumi, M. Yoshimitsu, K. Ishitsuka, S.-i. Ueno
          </td>
          <td>2025-01-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Retrotransposons, multi-copy sequences that propagate via copy-and-paste mechanisms involving an RNA intermediate, occupy large portions of all eukaryotic genomes. A great majority of their manifold copies remain silenced in somatic cells, nevertheless, some are transcribed, often in a tissue specific manner, and a small fraction retains its ability to mobilize. Retrotransposon expression or mobility are increasingly recognized to contribute to normal development and tissue homeostasis, as well as to aging and disease. While it is well characterized that retrotransposon sequences may provide cis regulatory elements for neighboring genes, how their own expression and mobility are achieved in different somatic contexts is not well understood. Here, using long-read DNA sequencing, we characterize somatic retrotransposition in the Drosophila intestine. We show that retroelement mobility does not change significantly upon aging and is limited to very few active sub-families of retrotransposons. Importantly, we identify a polymorphic donor locus of an endogenous LTR retroviral element rover, active in the intestinal tissue. We reveal that gut activity of the rover donor copy depends on its genomic environment. Without affecting local gene expression, the copy co-opts its upstream enhancer sequence, rich in transcription factor binding sites, for somatic expression. Further we show that escargot, a snail-type transcription factor critical for gut progenitor cell function, can drive transcriptional activity of the active rover copy. These data provide new insights into how locus-specific features allow active retrotransposons to produce functional transcripts and mobilize in a somatic lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1e2218d35b486ab25b581d00df030a25e3f5364" target='_blank'>
              An endogenous retroviral element co-opts an upstream regulatory sequence to achieve somatic expression and mobility
              </a>
            </td>
          <td>
            Natalia Rubanova, Darshika Singh, Louis Barolle, Fabienne Chalvet, Sophie Netter, Mickael Poidevin, Nicolas Servant, Allison J. Bardin, Katarzyna Siudeja
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="DNA G-quadruplexes (G4s) are secondary structures with significant roles in regulating genome function and stability. Dysregulation of the dynamic formation of G4s is linked to genomic instability and disease, but the underlying mechanisms are not fully understood. In this study, we conducted a screen of chromatin-modifying enzymes and identified nine potential inhibitors of G4 formation, including seven that were not previously characterized. Among these, we highlight the role of BAZ2 chromatin remodelers as key suppressors of G4 DNA and G4-related genome instability. Depletion of BAZ2 subunits led to increased G4 formation, especially at transcriptional regulatory elements. BAZ2B was found to associate with G4 loci, suggesting that it plays a direct role in suppressing G4s. While BAZ2-deficient cells exhibited modest genomic instability, treatment with the G4-stabilizing ligand BRACO19 exacerbated double-strand breaks (DSBs), highlighting its utility as a tool to study G4-dependent genome instability. DSB profiling using INDUCE-seq uncovered distinct breakage patterns around G4s, further underscoring the impact of G4s on genome integrity. Notably, we found that within G4s, G repeats were more susceptible to DSBs than loops. These results establish BAZ2 chromatin remodeling complexes as direct regulators of G4 dynamics and provide new insights into G4-dependent genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a708bc98c35ab68fd43151d91f694aa98b85bda" target='_blank'>
              The Role of BAZ2-dependent Chromatin Remodeling in Suppressing G4 DNA Structures and Associated Genomic Instability
              </a>
            </td>
          <td>
            Adrianna L. Vandeuren, Kierney O’Dare, Rosemary H. C. Wilson, Patrick van Eijk, Lindsay R. Julio, Shannon G. MacLeod, Ella Chee, , Emma M. Kriz, George A. Lantz, Shellaina J. V. Gordon, Simon J. Elsässer, Simon H. Reed, Tovah A. Day
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c8ca5fbab3938ff9f07a03caf56814d5bc9a81f" target='_blank'>
              Engineered extrachromosomal oncogene amplifications promote tumorigenesis
              </a>
            </td>
          <td>
            D. Pradella, Minsi Zhang, Rui Gao, Melissa A Yao, Katarzyna M. Gluchowska, Ylenia Cendon-Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H M Nguyen, Marta Lisi, Mateusz M Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, , Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Y. Pritykin, C. Sigel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2024-12-18</td>
          <td>Nature</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="Renowned as a highly invasive and lethal tumor derived from neural stem cells in the central nervous system, glioblastoma (GBM) exhibits substantial histopathological variation and genomic complexity, which drive its rapid progression and therapeutic resistance. Alterations in mitochondrial DNA (mtDNA) copy number (CN) serve a crucial role in GBM development and progression, affecting various aspects of tumor biology, including energy production, oxidative stress regulation and cellular adaptability. Fluctuations in mtDNA levels, whether elevated or diminished, can impair mitochondrial function, potentially disrupting oxidative phosphorylation and amplifying reactive oxygen species generation, thereby fueling tumor growth and influencing treatment responses. Understanding the mechanisms of mtDNA‑CN variations, and their interplay with genetic and environmental elements in the tumor microenvironment, is essential for advancing diagnostic and therapeutic strategies. Targeting mtDNA alterations could strengthen treatment efficacy, mitigate resistance and ultimately enhance the prognosis of patients with this aggressive brain tumor. The present review summarizes the existing literature on mtDNA alterations, specifically emphasizing variations in mtDNA‑CN and their association with GBM by surveying articles published between 1996 and 2024, sourced from databases such as Scopus, PubMed and Google Scholar. In addition, the review provides a brief overview of mitochondrial genome architecture, knowledge regarding the regulation of mtDNA integrity and CN, and how mitochondria significantly impact GBM tumorigenesis. This review further presents information on therapeutic approaches for restoring mtDNA‑CN that contribute to optimized mitochondrial function and improved health outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/608231a30fdc2050cf19ab35c688a4cbcbde93d3" target='_blank'>
              Mitochondrial DNA copy number alterations: Key players in the complexity of glioblastoma (Review).
              </a>
            </td>
          <td>
            Abdul Aziz Mohamed Yusoff, Siti Zulaikha Nashwa Mohd Khair, Siti Muslihah Abd Radzak
          </td>
          <td>2025-01-24</td>
          <td>Molecular medicine reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7b4b6b0cd06f7bc05e01f01febfa31643b31b8" target='_blank'>
              Prevalent integration of genomic repetitive and regulatory elements and donor sequences at CRISPR-Cas9-induced breaks
              </a>
            </td>
          <td>
            C. Bi, , Yingzi Zhang, Mengge Wang, Yeteng Tian, Mo Li
          </td>
          <td>2025-01-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The ribosomal genes (rDNA genes) encode 47S rRNA which accounts for up to 80% of all cellular RNA. At any given time, no more than 50% of rDNA genes are actively transcribed, and the other half is silent by forming heterochromatin structures through DNA methylation. In cancer cells, upregulation of ribosome biogenesis has been recognized as a hallmark feature, thus, the reduced methylation of rDNA promoter has been thought to support conformational changes of chromatin accessibility and the subsequent increase in rDNA transcription. However, an increase in the heterochromatin state through rDNA hypermethylation can be a protective mechanism teetering on the brink of a threshold where cancer cells rarely successfully proliferate. Hence, clarifying hypo- or hypermethylation of rDNA will unravel its additional cellular functions, including organization of genome architecture and regulation of gene expression, in response to growth signaling, cellular stressors, and carcinogenesis. Using the bisulfite-based quantitative real-time methylation-specific PCR (qMSP) method after ensuring unbiased amplification and complete bisulfite conversion of the minuscule DNA amount of 1 ng, we established that the rDNA promoter was significantly hypermethylated in 107 breast, 65 lung, and 135 colon tumour tissue samples (46.81%, 51.02% and 96.60%, respectively) as compared with their corresponding adjacent normal samples (26.84%, 38.26% and 77.52%, respectively; p < 0.0001). An excessive DNA input of 1 μg resulted in double-stranded rDNA remaining unconverted even after bisulfite conversion, hence the dramatic drop in the single-stranded DNA that strictly required for bisulfite conversion, and leading to an underestimation of rDNA promoter methylation, in other words, a faulty hypomethylation status of the rDNA promoter. Our results are in line with the hypothesis that an increase in rDNA methylation is a natural pathway protecting rDNA repeats that are extremely sensitive to DNA damage in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e41a5d7f178f6e61d7cd45d8e11b2f922f3fa334" target='_blank'>
              Hypermethylation at 45S rDNA promoter in cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Trang Hien Do, Tung The Pham, Phương Thi Thu Luu, Oanh Minh Pham, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Tuan Van Mai, Son Van Ho, Than Thi Nguyen, Lan Thi Thuong Vo
          </td>
          <td>2025-01-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Squamous cancers are the most frequently diagnosed solid tumours world-wide. The closely related transcriptional co-regulators YAP and TAZ(WWTR1) have emerged as important drivers of tumour initiation and progression in various types of squamous cell carcinoma. How YAP and TAZ execute their oncogenic functions in squamous cancers is still not fully understood. Here, we report that in addition to controlling transcriptional programmes that determine the balance between proliferation and terminal differentiation, YAP also engages with RIF1, a key regulator of DNA replication timing and protector of stalled replication forks under replication stress. YAP modulates gene expression of RIF1 via TEAD transcription factors and also stabilizes RIF1 protein on the chromatin. YAP-RIF1 complex formation increases in response to exogenous replication stress, and treatment with hydroxyurea to induce replication fork stalling acts synergistically with YAP depletion to impair cell proliferation. Our results thus demonstrate that YAP’s oncogenic functions in squamous cell carcinoma involve both transcriptional and non-transcriptional mechanisms, and that RIF1 is key for squamous cell carcinoma cells to survive with their high levels of endogenous replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3985e13774d5b1194193c3f3fcbde897bc3a759f" target='_blank'>
              YAP engages RIF1 to dampen replication stress in squamous cell carcinoma
              </a>
            </td>
          <td>
            Jodie Bojko, Benjamin Flynn, Bertram Aschenbrenner, Alexander Howard, Emily Lay, Emma Bailey, Natalia Krajic, Sandra Catalan, Kelli Gallacher, Elodie Sins, Jun Wang, Ute Jungwirth, B. Lichtenberger, Gernot Walko
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Mitotic fidelity depends on proper chromosome alignment at the spindle equator, a process known as chromosome congression, driven by well-established protein networks. Whereas RNA-binding proteins and noncoding RNAs have been implicated in cell division, their interplay during this process remains unknown. Here, we discover that RSRC2, an arginine/serine-rich RNA-binding protein, plays an essential role in cell division by interacting with the long non-coding RNA C1QTNF1-AS1. The loss of either RSRC2 or C1QTNF1-AS1 results in defects in chromosome congression and mitotic progression. We show that RSRC2 interacts with distinct sets of proteins involved in splicing and centrosome biogenesis, contributing to the fidelity of cell division through two different mechanisms: one linked to the splicing of mitotic regulators and the other by localising to mitotic centrosomes for which the interaction with the C1QTNF1-AS1 RNA is required. Our study uncovers RSRC2 as a new regulator of cell division and illustrates how RNA-protein complexes promote error-free mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb0d898e39c9e3e90e4923dfd61d6149838f2b92" target='_blank'>
              The RNA-binding protein RSRC2 promotes mitotic fidelity by interacting with the lncRNA C1QTNF1-AS1
              </a>
            </td>
          <td>
            Kaliya Georgieva, Alice O Coomer, Parnia Babaei, Giulia Guiducci, Martin Dodel, Eleni Maniati, Anisha Thind, S. Krishnamurthy, Anna Nawrocka, Sam Wallis, Alena Shkumatava, Jun Wang, F. Mardakheh, Lovorka Stojic
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background
The causes of breast cancer are poorly understood. A potential risk factor is Epstein-Barr virus (EBV), a lifelong infection nearly everyone acquires. EBV-transformed human mammary cells accelerate breast cancer when transplanted into immunosuppressed mice, but the virus can disappear as malignant cells reproduce. If this model applies to human breast cancers, then they should have genome damage characteristic of EBV infection.


Objective
This study tests the hypothesis that EBV infection predisposes one to breast cancer by causing permanent genome damage that compromises cancer safeguards.


Methods
Publicly available genome data from approximately 2100 breast cancers and 25 ovarian cancers were compared to cancers with proven associations to EBV, including 70 nasopharyngeal cancers, 90 Burkitt lymphomas, 88 diffuse large B-cell lymphomas, and 34 gastric cancers. Calculation algorithms to make these comparisons were developed.


Results
Chromosome breakpoints in breast and ovarian cancer clustered around breakpoints in EBV-associated cancers. Breakpoint distributions in breast and EBV-associated cancers on some chromosomes were not confidently distinguished (P>.05), but differed from controls unrelated to EBV infection. Viral breakpoint clusters occurred in high-risk, sporadic, and other breast cancer subgroups. Breakpoint clusters disrupted gene functions essential for cancer protection, which remain compromised even if EBV infection disappears. As CRISPR (clustered regularly interspaced short palindromic repeats)-like reminders of past infection during evolution, EBV genome fragments were found regularly interspaced between Piwi-interacting RNA (piRNA) genes on chromosome 6. Both breast and EBV-associated cancers had inactivated genes that guard piRNA defenses and the major histocompatibility complex (MHC) locus. Breast and EBV-associated cancer breakpoints and other variations converged around the highly polymorphic MHC. Not everyone develops cancer because MHC differences produce differing responses to EBV infection. Chromosome shattering and mutation hot spots in breast cancers preferentially occurred at incorporated viral sequences. On chromosome 17, breast cancer breakpoints that clustered around those in EBV-mediated cancers were linked to estrogen effects. Other breast cancer breaks affected sites where EBV inhibits JAK-STAT and SWI-SNF signaling pathways. A characteristic EBV-cancer gene deletion that shifts metabolism to favor tumors was also found in breast cancers. These changes push breast cancer into metastasis and then favor survival of metastatic cells.


Conclusions
EBV infection predisposes one to breast cancer and metastasis, even if the virus disappears. Identifying this pathogenic viral damage may improve screening, treatment, and prevention. Immunizing children against EBV may protect against breast, ovarian, other cancers, and potentially even chronic unexplained diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31f55b81eb64a7ed0044ee061bd1cada36b4674e" target='_blank'>
              Identifying Safeguards Disabled by Epstein-Barr Virus Infections in Genomes From Patients With Breast Cancer: Chromosomal Bioinformatics Analysis.
              </a>
            </td>
          <td>
            B. Friedenson
          </td>
          <td>2023-07-10</td>
          <td>JMIRx med</td>
          <td>2</td>
          <td>0</td>
        </tr>

        <tr id="Structural variation is a major contributor to human diversity, adaptation, and disease. Simple structural variant (SV) types include deletions, insertions, duplications, inversions, and translocations, and SVs account for most of the variable bases between genomes. Complex structural variants (CSVs) that consist of one or more simple events in cis appear more frequently in diseases and cancers where DNA repair, apoptosis, and cell cycle checkpoints are compromised, although CSVs can also appear in germline genome sequences of healthy individuals. CSVs are often characterized by short tracts of homology or no homology, and while CSVs are more prevalent in complex regions that contain large repeats, smaller stretches of homology can also enable their formation across more unique loci. Long-read assemblies have increased the size of detectable SVs and expanded variant detection into more complex regions of the genome, and while they reconstruct CSVs, methods for identifying CSVs from assemblies is limited. Here, we have developed a new assembly-based approach to trace through complex loci rather than relying upon reference representations of alignments. We can now access CSVs in large complex segmental duplications, reveal structures that were previously unknown, and identify SV breakpoints with greater accuracy. We find 72 large CSVs per genome and 128 unique complex structures and CSVs in highly repetitive regions can now be detected including several distinct complex events in repetitive NBPF genes that was not previously callable with short-read or long-read CSV methods. This approach is implemented within a key assembly-based variant calling tool, PAV, and represents a substantial improvement identifying complex variants now ascertainable from contiguous genome assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbfc7a889c92ffa27e1ac1ac1e0ce4aa98dae5ef" target='_blank'>
              Large complex structural rearrangements in human genomes harbor cryptic structures
              </a>
            </td>
          <td>
            P. Audano, Carolyn A. Paisie, Christine R. Beck
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In pancreatic ductal adenocarcinoma cancer (PDAC) drug resistance is a severe clinical problem and patients relapse within a few months after receiving the standard-of-care chemotherapy. One contributing factor to treatment resistance is the desmoplastic nature of PDAC; the tumours are surrounded by thick layers of stroma composing up to 90% of the tumour mass. This stroma, which is mostly comprised of extracellular matrix (ECM) proteins, is secreted by cancer-associated fibroblasts (CAFs) residing in the tumour microenvironment. However, the mechanistic basis by which the tumour stroma directly contributes to chemoresistance remains unclear. Here, we show that CAF-secreted ECM proteins induce chemoresistance by blunting chemotherapy-induced DNA damage. Mechanistically, we identify N-myc downstream regulated gene 1 (NDRG1) as a key protein required for stroma-induced chemoresistance that responds to signals from the ECM and adhesion receptors. We further show that NDRG1 is a novel DNA repair protein that physically interacts with replication forks, maintains DNA replication and functions to resolve stalled forks caused by chemotherapy. More specifically, NDRG1 reduces R-loops, RNA-DNA hybrids that are known to cause genomic instability. R-loops occur during replication-transcription conflicts in S-phase and after chemotherapy treatments, thus posing a major threat to normal replication fork homeostasis. We identify NDRG1 as highly expressed in PDAC tumours, and its high expression correlates with chemoresistance and poor disease-specific survival. Importantly, knock-out of NDRG1 or inhibition of its phosphorylation restores chemotherapy-induced DNA damage and resensitizes tumour cells to treatment. In conclusion, our data reveal an unexpected role for CAF-secreted ECM proteins in enhancing DNA repair via NDRG1, a novel DNA repair protein, directly linking tumour stroma to replication fork homeostasis and R-loop biology, with important therapeutic implications for restoring DNA damage response pathways in pancreatic cancer. Summary paragraph Drug resistance is a severe clinical problem in stroma-rich tumours, such as pancreatic ductal adenocarcinoma (PDAC), and patients often relapse within a few months on chemotherapy1–9. The stroma, comprised of extracellular matrix (ECM) proteins, is secreted by cancer-associated fibroblasts (CAFs) residing in the tumour microenvironment10–13. Prior work show that ECM proteins provide survival benefits to cancer cells14,15. However, the precise role of CAF-secreted ECM in resistance to DNA damaging chemotherapies remains poorly understood. Here, we link ECM proteins to chemoresistance by enhanced DNA damage repair (DDR). Mechanistically, we identify N-myc downstream-regulated gene 1 (NDRG1) as a key effector downstream of ECM and the integrin-Src-SGK1-signalling axis that mediates enhanced DDR. We show that NDRG1 loss, mutation of conserved His194, or inhibition of NDRG1 phosphorylation by SGK1 lead to replication fork stalling, increased R-loops, and higher transcription-replication conflicts, resulting in genomic instability and sensitivity to chemotherapies. Our analysis of PDAC patient cohorts16 found that high NDRG1 expression correlates with chemoresistance and poor patient survival. In conclusion, we uncover an unexpected role for CAF-secreted ECM proteins in promoting therapeutic resistance by enhancing DDR and establish NDRG1 as a novel DNA repair protein directly linking tumour stroma to DDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fa48f066c27823d8d4f1f47c6def3fbe11eb73e" target='_blank'>
              A novel DNA repair protein, N-Myc downstream regulated gene 1 (NDRG1), links stromal tumour microenvironment to chemoresistance
              </a>
            </td>
          <td>
            Nina Kozlova, Kayla A Cruz, Hanna M Doh, Antoine A Ruzette, Nicholas A. Willis, , Raul S Gonzalez, Monika Vyas, , Stephan B. Dreyer, Rosie Upstill-Goddard, , Steve Wenglowsky, Josh Close, Alica K Beutel, Zeljka Jutric, Michael U J Oliphant, Byanjana Thapa, , Venla Mustonen, Pradeep Mangalath, Christopher J Halbrook, Joseph E Grossman, Rosa F. Hwang, J. Clohessy, S. Ruskamo, Petri Kursula, Boryana Petrova, Naama Kanarek, Philip A. Cole, David K. Chang, S. Norrelykke, Ralph Scully, 
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="INTRODUCTION
The SEWA cell line, which is derived from a virus induced murine osteosarcoma (OS) ascites was established in the 1980s from a serially transplanted male derived tumor that first published in 1961. It has been applied in about 50 studies but was never genetically characterized in detail; this study fills that gap.


METHODS
The SEWA cell line was analyzed for its chromosomal constitution using molecular cytogenetic approaches. Array comparative genomic hybridization was performed to characterize copy number alterations.


RESULTS
SEWA has a near diploid karyotype without Y chromosome material. The complex karyotype includes neocentrics and simple and complex rearrangements. Amplification of MYC oncogene was detected in two homogeneously staining regions on two different derivative chromosomes.


CONCLUSION
An in silico translation of the obtained results to the human genome indicated that SEWA is suitable as a model for advanced human OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ec328ac3891c2b11391d74ef5192bbc587c6a2" target='_blank'>
              Cytogenomic characterization of murine osteosarcoma cell line SEWA.
              </a>
            </td>
          <td>
            Thomas Liehr, Martina Rinčic
          </td>
          <td>2024-12-11</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chromosomal inversions play a crucial role in evolution by influencing phenotypes through the linkage of co-adapted alleles. While inversions have been found across a large number of taxa, mapping and characterizing inversion breakpoint regions remains challenging, often due to the presence of complex tandem repeats and transposable elements (TEs). Here, we identify and quantify TEs in the breakpoints of the four large-scale inversions previously reported in Atlantic cod, leveraging on three high-quality long-read-based reference genome assemblies for the Norwegian Coastal cod, the Northeast Arctic cod and Celtic cod ecotypes. We detected a significant enrichment of TE orders and superfamilies with terminal inverted repeats (TIRs) within the inversion breakpoints of chromosomes 1, 7 and 12. Notably, we discovered a tandem accumulation of miniature inverted-repeat transposable elements (MITEs) belonging to a family of hAT transposons, exclusively residing in the breakpoints of the inverted haplotype on chromosomes 1 and 7 found in the Northeast Arctic cod. The accumulation of tandemly arranged TEs in breakpoint regions suggests that they have driven the appearance of inversions through ectopic recombination, further supporting the potential of TEs in facilitating chromosomal reorganizations with large evolutionary implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c69b85f1eed1a39a185be57ad812eafc541ce3b" target='_blank'>
              Chromosomal inversions mediated by tandem insertions of transposable elements
              </a>
            </td>
          <td>
            Robin Aasegg Araya, William B. Reinar, O. K. Tørresen, C. Goubert, Tara J Daughton, S. Hoff, H. T. Baalsrud, Marine Servane Ono Brieuc, Anna Komisarczuk, S. Jentoft, José Cerca, K.S. Jakobsen
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Endogenous retroviral (ERV) RNA is highly expressed in cancer, although the molecular causes and consequences remain unknown. We found that ZC3H18 (Z18), a component of multiple nuclear RNA surveillance complexes, has recurrent truncating mutations in cancer. We show that Z18trunc mutations are oncogenic and that Z18 plays an evolutionarily conserved role in nuclear RNA surveillance of ERV RNA. In zebrafish, Z18trunc expedited melanoma onset and promoted a specific accumulation of ERV RNA. Z18 mutant human cell lines from the Cancer Cell Line Encyclopedia also expressed higher levels of ERV RNA. In engineered human melanoma cells, Z18trunc enhanced ERV RNA accumulation more than loss of one Z18 copy, indicating dominant negative activity. Z18trunc directly bound and stabilized ERV RNA. Notably, expression of ERV RNA was sufficient to expedite oncogenesis in a zebrafish model, which is the first evidence of which we are aware that ERV transcripts can play a functional role in cancer. Our work illuminates a mechanism for elevated ERV transcripts in cancer and supports that aberrant RNA accumulation is broadly oncogenic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85414cac93dcb00a3b2cbff32569ab0497071b3c" target='_blank'>
              Recurrent oncogenic ZC3H18 mutations stabilize endogenous retroviral RNA
              </a>
            </td>
          <td>
            Tanzina Tanu, Anna M. Cox, Jennifer A. Karlow, Priyanka Sharma, Xueyang He, Constance Wu, Swathy Babu, Jared Brown, Kevin M. Brown, Stephen J. Chanock, David Liu, Tongwu Zhang, Kathleen H. Burns, P. Boutz, Megan L. Insco
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is initiated during early mammalian embryogenesis by a long non-coding RNA (lncRNA) XIST, which coats one of the two X-chromosomes and facilitates epigenetic transcriptional silencing. A second, evolutionarily recent primate-specific lncRNA XACT was proposed to antagonize XIST’s ability to induce XCI. XACT expression is restricted to pluripotent states and early embryonic stages and coats the active X-chromosome in both females and males. Here, we report a novel XACT transcript expressed in normal and cancerous somatic cells from both the inactive (Xi) and active (Xa) X chromosomes. It coexists with XIST on the Xi without affecting XIST expression or (re)activating X-linked genes inactivated by XCI. During hematopoietic stem cell (HSC) differentiation, XACT is primarily expressed in myeloid progenitors in both sexes. XACT expression is activated in HSCs and peaks in megakaryocyte-erythrocyte progenitor cells (MEPs) before rapidly declining as the MEPs differentiate into megakaryocytes or erythrocytes. By combining CRISPR-based XACT perturbation with epigenomic and transcriptional studies, we revealed the key role of XACT in the self-renewal and differentiation of erythroid progenitors into erythrocytes, by recruiting cis-regulatory proteins and regulating transcription through ETS and AP-1 transcription factors. Furthermore, XACT is expressed in a subset of acute myeloid leukemia (AML) patients, with high levels found in erythroid-megakaryocytic blast cells; suggesting XACT’s potential as a marker for AML subtypes. Thus, we identified a novel XACT transcript with a unique expression profile and important roles in normal and cancer cells, revealing XACT lncRNA functions beyond its previously proposed role in XCI during embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7754eb09126d1d587b2765e70f76183499d8bcbf" target='_blank'>
              A Novel XACT lncRNA Transcript with Functions Transcending X-Chromosome Inactivation
              </a>
            </td>
          <td>
            Peifen Zhang, Paula Haro Angles, Hong Zhang, Tingyue Li, Roger Mulet-Lazaro, W. V. van Ijcken, S. Philipsen, Ruud Delwel, J. Gribnau, D. Huylebroeck, Frank Grosveld, C. Rougeulle, Eskeatnaf Mulugeta
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="The genome-wide chromosome conformation capture method, Hi-C, has greatly advanced our understanding of genome organization. However, its quantitative properties, including sensitivity, bias, and linearity, remain challenging to assess. Measuring these properties in vivo is difficult due to the heterogenous and dynamic nature of chromosomal interactions. Here, using Chemically Induced Chromosomal Interaction (CICI) method, we create stable intra- and inter-chromosomal interactions in G1-phase budding yeast across a broad range of contact frequencies. Hi-C analysis of these engineered cell populations demonstrates that static intra-chromosomal loops do not generate Topologically Associated Domains (TADs) and only promote 3D proximity within ∼50kb flanking regions. At moderate sequencing depth, Hi-C is sensitive enough to detect interactions occurring in 5-10% of cells. It also shows no inherent bias toward intra-versus inter-chromosomal interactions. Furthermore, we observe a linear relationship between Hi-C signal intensity and contact frequency. These findings illuminate the intrinsic properties of the Hi-C assay and provide a robust framework for its calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5303f7ba4784e24be34cf968c0e7ddd32d5ebcd" target='_blank'>
              Hi-C Calibration by Chemically Induced Chromosomal Interactions
              </a>
            </td>
          <td>
            Yi Li, Fan Zou, Lu Bai
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8feadfbb33b4fde036009823f20cf7ee9457518a" target='_blank'>
              BET inhibition revealed varying MYC dependency mechanisms independent of gene alterations in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            Loris Delrieu, Sieme Hamaidia, Emilie Montaut, Andrea Garcìa-Sandoval, Camille Teste, Patricia Betton-Fraisse, Thierry Bonnefoix, S. Carras, Rémy Gressin, Christine Lefebvre, Jérôme Govin, A. Emadali
          </td>
          <td>2024-12-19</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Rearrangements associated with gene amplification are frequently initiated by the formation of inverted repeats. Among these rearrangements are breakage-fusion-bridge cycles and tandem inverted duplications (TIDs), both of which can be mediated through palindromic sequences. We have investigated spontaneous rearrangements within the genome of the filamentous fungus Aspergillus flavus by high-coverage third-generation sequencing of vegetatively cultured strains. Analysis of sequence data identified a class of rare and transient rearrangements having structures typical of TIDs. These putative TIDs (pTIDs) form both unprocessed and processed species and have a variably sized deletion at the junction between direct and inverted sequences. Most intact junctions contain the predicted palindromic sequences, supporting the TID model. However, palindrome-free junctions are also infrequently observed. pTIDs are highly enriched in heterochromatic centromeric and subtelomeric A+T rich regions, suggesting that this process is a source of genetic instability in these domains. Consistent with this finding, A+T rich regions contain elevated levels of direct, inverted, and perfect palindromic repeats. We propose a replication-based model in which palindromes and short inverted repeats, under condition of replication stress, are responsible for pTID structures. TEASER The dynamic formation of duplication/inversions clustered in fungal chromosomes reveals the genetic instability of centromeric and subtelomeric regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728781bd2be224742eb5106c8451f24b72cd7b56" target='_blank'>
              Clustering of Tandem Inverse Duplications in Centromeric and Subtelomeric Chromosomal Regions of Aspergillus flavus
              </a>
            </td>
          <td>
            Jeffrey W. Cary, Amna Malik, Alexie Smith, Erik K Flemington, Arthur J Lustig
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The SNF2 family chromatin remodeler HELLS has emerged as an important regulator of cell proliferation, genome stability, and several cancer pathways. Significant upregulation of HELLS has been reported in 33 human cancer types. While HELLS has been implicated in DNA damage response, its function in DNA repair is poorly understood. Here we report a new regulatory link between HELLS and single-strand break (SSB) repair in cellular responses to DNA alkylation damage. We found that loss of HELLS impairs SSB repair, and selectively sensitizes cells to DNA alkylating agents and PARP inhibitors (PARPi). Furthermore, we found that HELLS is co-expressed with PARP1 in cancer cells, and its loss is synthetic lethal with homologous recombination deficiency (HRD). This work unveils new functions of HELLS in modulating SSB repair and responses to clinically relevant DNA alkylation damage, thus offering new insights into the potential therapeutic value of targeting HELLS in cancer. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd4d063ed0793e8f2d3e58d28ea55c516edbff0" target='_blank'>
              Chromatin regulator HELLS mediates SSB repair and responses to DNA alkylation damage
              </a>
            </td>
          <td>
            Joyous T Joseph, Christine M Wright, Estanislao Peixoto, Asad Khan, Yong Li, Jason S Romero Neidigk, Olivia Decker, Azait Imtiaz, Brianna A Jones, Yangfeng Zhang, Etsuko Shiabata, S. Gradilone, Zachary A. Lewis, Rafael Contreras-Galindo, Arko Sen, A. Dutta, Wioletta Czaja
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Atypical teratoid rhabdoid tumor (ATRT) is the most common malignant brain tumor in infants, and more than 60% of children with ATRT die from their tumor. ATRT is associated with mutational inactivation/deletion of SMARCB1, a member of the SWI/SNF chromatin remodeling complex, suggesting that epigenetic events play a critical role in tumor development and progression. Moreover, disruption of SWI/SNF allows unopposed activity of epigenetic repressors, which contribute to tumorigenicity. We therefore explored the role of the CoREST repressor complex in ATRT. Methods We evaluated the effects of the bifunctional LSD1/HDAC1/2 small molecule CoREST inhibitor, corin, on ATRT tumor cell growth, apoptosis, differentiation, gene expression and chromatin accessibility. Results We found that corin inhibited the growth of ATRT cells regardless of their epigenetic subgroup, and was associated with increased tumor cell apoptosis and differentiation. ATAC-seq showed increases in chromatin accessibility in corin-treated ATRT cells, with changes seen at genes associated with neuronal differentiation and synaptic function. RNA-seq confirmed increased expression of neuronal differentiation genes and decreased DNA replication/cell cycle-associated genes in ATRT cells treated with corin. Corin suppressed orthotopic ATRT tumor growth, leading to significant extension of lifespan. In addition, increased histone acetylation (H3K9ac, H3K27ac) and methylation (H3K4Me1) was seen in corin-treated ATRT orthotopic xenografts, consistent with on-target pharmacodynamics. Conclusion The CoREST inhibitor, corin, suppresses tumor growth, induces differentiation, and promotes apoptosis in ATRT, leading to significantly increased survival of mice bearing ATRT orthotopic xenografts. Our results suggest a potential application of CoREST complex inhibitors in patients with ATRT. Key Points CoREST complex inhibition by corin leads to decreased cell growth and increased apoptosis in ATRT Corin promotes chromatin accessibility and neuronal differentiation in ATRT Corin inhibits tumor growth and extends lifespan in ATRT animal models Importance of the Study Loss of function of SMARCB1 is a hallmark of ATRT which leads to dysfunction of the mammalian SWI/SNF complex and an inability to counteract epigenetic repressor complexes. The CoREST complex functions as a chromatin remodeling complex that represses neuronal differentiation genes during development. Inhibition of the CoREST complex by corin in ATRT leads to decreased tumor cell growth, induction of apoptosis and increased survival of mice bearing ATRT orthotopic xenografts. These changes are associated with increased chromatin accessibility and expression of genes associated with neuronal differentiation. Corin therefore reverses the primary block of differentiation that maintains a stem cell state in ATRT, which contributes to tumorigenesis. These studies significantly improve our understanding of how to therapeutically address the underlying epigenetic drivers of ATRT and support further development of corin for ATRT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024c492e649d7b4c64d548dac81278e98253c902" target='_blank'>
              The CoREST complex inhibitor, corin, leads to decreased tumor growth, increased cellular differentiation and extended lifespan in atypical teratoid rhabdoid tumor xenograft models
              </a>
            </td>
          <td>
            Anupa Geethadevi, Nikhil Vaidya, Robert J. Fisher, Tyler Findlay, Yiming Deng, Khoa Pham, Vikas Kumar, Samuel Beck, Calixto-Hope G Lucas, C. Eberhart, Jinchong Xu, Philip A. Cole, Jeffrey A. Rubens, Marianne Collard, Eric H. Raabe, R. Alani
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, , Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Lomentospora prolificans is a fungal pathogen responsible for serious, often fatal, illness in patients with compromised immune systems. Treatment is rarely successful because L. prolificans is inherently resistant to all major classes of antifungal drugs. In this study we publish three chromosome-level de novo genome assemblies, including the first complete-level assembly of L. prolificans, along with genome annotations. The L. prolificans genome is packaged in 11 nuclear chromosomes and one mitochondrial chromosome, has 36.7-37.1 Mb, and encodes for a putative 7357-7640 genes. The length and composition of contigs in one stain varied from those of the other two strains, supporting the hypothesis that a chromosomal translocation took place. These assemblies were confirmed with pulsed-field gel electrophoresis. The availability of more complete genomes will hopefully help the search for new antifungal drugs and provides insights into the evolutionary history of this pathogenic fungus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f38c6f4c1713d9e763d43ccdd74bd10162966e74" target='_blank'>
              Three annotated chromosome-level de novo genome assemblies of Lomentospora prolificans provide evidence for a chromosomal translocation event
              </a>
            </td>
          <td>
            Nina Grossman, Yunfan Fan, Aleksey V. Zimin, Maggie P. Wear, Anne Jedlicka, Amanda Dziedzic, Livia Liporagi-Lopes, W. Timp, A. Casadevall
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, , G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The study investigates the impact of targeted chromosome engineering on telomere dynamics, chromatin structure, gene expression, and phenotypic stability in Arabidopsis thaliana. Using precise CRISPR/Cas-based engineering, reciprocal translocations of chromosome arms were introduced between non-homologous chromosomes. The subsequent homozygous generations of plants were assessed for phenotype, transcriptomic changes and chromatin modifications near translocation breakpoints, and telomere length maintenance. Phenotypically, translocated lines were indistinguishable from wild-type plants, as confirmed through morphological assessments and principal component analysis. Gene expression profiling detected minimal differential expression, with affected genes dispersed across the genome, indicating negligible transcriptional impact. Similarly, ChIPseq analysis showed no substantial alterations in the enrichment of key histone marks (H3K27me3, H3K4me1, H3K56ac) near junction sites or across the genome. Finally, bulk and arm-specific telomere lengths remained stable across multiple generations, except for minor variations in one translocation line. These findings highlight the remarkable genomic and phenotypic robustness of A. thaliana despite large-scale chromosomal rearrangements. The study offers insights into the cis-acting mechanisms underlying chromosome arm-specific telomere length setting and establishes the feasibility of chromosome engineering for studies of plant genome evolution and crop improvement strategies. Significance statement This study demonstrates the robustness of Arabidopsis thaliana in maintaining telomere stability, chromatin integrity, and wild-type phenotype despite large-scale chromosomal translocations. It underscores the potential of chromosome engineering in advancing genome evolution research and crop improvement, contributes to a deeper understanding of genome dynamics, and opens new avenues for precision breeding in plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c4a5f7dc0b1cbe9e13ab94b7dc2055482d7e96c" target='_blank'>
              Chromosome engineering points to the cis-acting mechanism of chromosome arm-specific telomere length setting and robustness of plant phenotype, chromatin structure and gene expression
              </a>
            </td>
          <td>
            Ondřej Helia, Barbora Matúšová, Kateřina Havlová, Anna Hýsková, Martin Lyčka, Natalja Beying, , Jíří Fajkus, M. Fojtová
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The impact of cancer driving mutations in regulating immunosurveillance throughout tumor development remains poorly understood. To better understand the contribution of tumor genotype to immunosurveillance, we generated and validated lentiviral vectors that create an epi-allelic series of increasingly immunogenic neoantigens. This vector system is compatible with autochthonous Cre-regulated cancer models, CRISPR/Cas9-mediated somatic genome editing, and tumor barcoding. Here, we show that in the context of KRAS-driven lung cancer and strong neoantigen expression, tumor suppressor genotype dictates the degree of immune cell recruitment, positive selection of tumors with neoantigen silencing, and tumor outgrowth. By quantifying the impact of 11 commonly inactivated tumor suppressor genes on tumor growth across neoantigenic contexts, we show that the growth promoting effects of tumor suppressor gene inactivation correlate with increasing sensitivity to immunosurveillance. Importantly, specific genotypes dramatically increase or decrease sensitivity to immunosurveillance independently of their growth promoting effects. We propose a model of immunoediting in which tumor suppressor gene inactivation works in tandem with neoantigen expression to shape tumor immunosurveillance and immunoediting such that the same neoantigens uniquely modulate tumor immunoediting depending on the genetic context. One Sentence Summary Here we uncover an under-appreciated role for tumor suppressor gene inactivation in shaping immunoediting upon neoantigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a2622766d9e8241bb1972eff5c441baddd14069" target='_blank'>
              Tumor suppressor genotype influences the extent and mode of immunosurveillance in lung cancer
              </a>
            </td>
          <td>
            Keren M. Adler, Haiqing Xu, Amy C. Gladstein, Valerie M. Irizarry-Negron, Maggie R. Robertson, Katherine R. Doerig, D. Petrov, M. Winslow, D. Feldser
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d46fb9d0f82d02098610fd0641e0285651c8da7" target='_blank'>
              Elucidating the expression and role of cGAS in pan-cancer using integrated bioinformatics and experimental approaches
              </a>
            </td>
          <td>
            Zhen Lian, Xue Liu, Xue Li
          </td>
          <td>2025-01-02</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Structural maintenance of chromosome-1A (SMC1A) is overexpressed in various malignancies including triple-negative breast cancer (TNBC). As a core component of the cohesin complex, SMC1A was initially recognized for its involvement in chromosomal cohesion and DNA-repair pathways. However, recent studies have unveiled its pivotal role in epithelial–mesenchymal transition (EMT), metastasis, and chemo- and radio-resistance in cancer cells. In hepatocellular carcinoma, aberrant phosphorylation of SMC1A has been associated with enhanced cell proliferation and migration. Despite these insights, the precise role of SMC1A phosphorylation in breast cancer remains largely unexplored. This study represents the first investigation to test the phosphorylation status and subcellular localization of SMC1A (p-SMC1A) in breast cancer and normal breast tissues. Immunohistochemical (IHC) staining was conducted using previously validated phospho-SMC1A antibodies on a histological section and tissue microarray (TMA) comprising samples from primary, invasive, and metastatic breast cancer and normal breast tissues. Our results revealed that p-SMC1A staining intensity was lower in normal breast tissues compared to invasive or metastatic breast cancer tissues (p < 0.001). Approximately 40% of breast cancer tissue exhibited cytoplasmic/membranous localization of p-SMC1A, whereas nuclear expression was observed in normal breast tissues. Moreover, elevated phosphorylation levels were significantly associated with higher tumor grade and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ac66fe9226183dafb37845efa13c23cb0dc9fe" target='_blank'>
              Association of Structural Maintenance of Chromosome-1A Phosphorylation with Progression of Breast Cancer
              </a>
            </td>
          <td>
            Sushma Yadav, C. Kowolik, Daniel Schmolze, Yuan Yuan, Min Lin, Arthur D. Riggs, David A. Horne
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are a rich source of genetic variation, but are difficult to genotype. While specialized repeat variant callers exist, they typically assume a euploid human genome. This means recent findings regarding phenotypic effects of STR variants in human health and disease cannot be readily extended to polyploid organisms or cancer, which is characterised by copy number alterations (CNAs). Here we present ConSTRain, a novel STR variant caller that explicitly accounts for the copy number of loci in its genotyping approach. We benchmark ConSTRain using a euploid human 100X whole genome sequencing sample where it calls STR allele lengths for over 1.7 × 106 loci in under 20 minutes with an accuracy of 98.28%. Subsequently, we show that ConSTRain resolves complex STR genotypes in an artificial trisomy 21 sample and a polyploid Dwarf Cavendish banana harbouring a large duplication. Finally, we analyse a microsatellite instable colorectal cancer tumoroid, where ConSTRain tackles CNAs and whole-genome duplications. ConSTRain is the first STR variant caller that allows for the investigation of repeats affected by CNAs, aneuploidies, and polyploid genomes. This unlocks the investigation of STRs across a wide range of contexts and organisms where they previously could not be easily studied.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42abd302650eb39af080a99a7d76a61bf128e039" target='_blank'>
              Genotyping Short Tandem Repeats Across Copy Number Alterations, Aneuploidies, and Polyploid Organisms
              </a>
            </td>
          <td>
            M. Verbiest, Elena Grassi, Andrea Bertotti, Maria Anisimova
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9043774f4d71a3f413b248306070648bf9c170e4" target='_blank'>
              Multi-omics approaches reveal that diffuse midline gliomas present altered DNA replication and are susceptible to replication stress therapy
              </a>
            </td>
          <td>
            Anastasia E. Hains, Kashish Chetal, Tsunetoshi Nakatani, Joana G. Marques, Andreas Ettinger, C. B. Junior, Adriana Gonzalez-Sandoval, Renjitha Pillai, Mariella G. Filbin, M. Torres-Padilla, Ruslan I. Sadreyev, Capucine Van Rechem
          </td>
          <td>2024-12-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Polyploidization is a driving force of wheat evolution and speciation, yet its impact on epigenetic regulation and gene expression remains unclear. Here, we constructed a high-resolution epigenetic landscape across leaves, spikes, and roots of hexaploidy wheat and its tetraploid and diploid relatives. Inter-species stable-expression genes exhibited conserved amino acid sequences under strong purifying selection, while dynamic-expression genes were linked to species-specific adaptation. During hexaploidization, dominant D-subgenome homoeolog expression was suppressed via reduced activating epigenetic signals, converging expression with the A and B subgenomes. Proximal chromatin regions near genes were more stable, whereas distal regions, particularly enhancer-like elements mediated by H3K27ac and H3K4me3, exhibit higher dynamism. Sequence variations in these enhancers lead to differential gene regulation, influencing traits such as spike development. For instance, the two haplotypes of dCRE region of TaDEP-B1 resulted in significant differences in its expression and spikelet numbers. We also observed a coevolution of transcription factors and their binding sites, particularly within the expanded ERF family, which regulates spike morphology. This study highlights the interplay between sequence variation and epigenetic modifications in shaping transcriptional regulation during wheat speciation, offering valuable insights for genetic improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3eb492b176e5f2e28bdd73ac3853852b58a5d2c" target='_blank'>
              Reshaping epigenomic landscapes in facilitating the speciation of bread wheat
              </a>
            </td>
          <td>
            Zhaoheng Zhang, Xuelei Lin, Jingjing Yue, Yong-Chang Xu, Lingfeng Miao, Wenqiang Tang, Weilong Guo, Jun Xiao
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Spatial organization of the genome is fundamental for ensuring accurate gene expression. This process depends on the communication between gene promoters and distal cis-regulatory elements (CREs), which together make up 8% of the human genome and are supported by the chromatin structure. It is estimated that over 90% of disease-associated variants are located in the non-coding region of the genome and may affect CRE. For the cystic fibrosis transmembrane conductance regulator (CFTR) gene, a complete understanding of tissue-specific CFTR expression and regulation is missing, in particular in the pancreas. Mechanistic insights into tissue-specific expression may provide clarity on the clinical heterogeneity observed in Cystic Fibrosis and CFTR-related disorders. Methods To understand the role of 3D chromatin architecture in establishing tissue-specific expression of the CFTR gene, we mapped chromatin interactions via circular chromosome conformation capture (4C) and epigenomic regulation through H3K27ac and DNase Hypersensitive site I (DHS) in Capan-1 pancreatic cells. Candidate regulatory regions are validated by luciferase reporter assay and CRISPR-knock out. Results We identified active regulatory regions not only around the CFTR gene but also outside the topologically associating domain (TAD). By performing functional assays, we validated our targets and revealed a cooperative effect of the −44 kb, −35 kb, +15.6 kb and 37.7 kb regions, which share common predicted transcription factor (TF) motifs. Comparative 3D genomic analysis and functional assays using the Caco-2 intestinal cell line revealed the presence of tissue-specific CREs. Conclusion By studying the chromatin architecture of the CFTR locus in Capan-1 cells, we demonstrated the involvement of multiple CREs upstream and downstream of the CFTR gene. We also extend our analysis to compare intestinal and pancreatic cells and provide information on the tissue-specificity of CRE. These findings highlight the importance of expanding the search for causative variants beyond the gene coding sequence but also by considering the tissue-specific 3D genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a9279897acf411b562291c89812e3d397609427" target='_blank'>
              New insights into the cis-regulation of the CFTR gene in pancreatic cells
              </a>
            </td>
          <td>
            Clara Blotas, A. Le Nabec, M. Collobert, Mattijs Bulcaen, Marianne S. Carlon, Claude Férec, S. Moisan
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/070c22030bfb86f8cf3bc0c2063075e7b4258116" target='_blank'>
              Identification of the clinical and genetic characteristics of gliomas with gene fusions by integrated genomic and transcriptomic analysis
              </a>
            </td>
          <td>
            Guo-zhong Yi, Hua-yang Zhang, Tian-shi Que, Shan-qiang Qu, Zhi-yong Li, Song-tao Qi, , 
          </td>
          <td>2025-01-23</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="REV7, also known as MAD2B, MAD2L2, and FANCV, is a HORMA-domain family protein crucial to multiple genome stability pathways. REV7’s canonical role is as a member of polymerase ζ, a specialized translesion synthesis polymerase essential for DNA damage tolerance. REV7 also ensures accurate cell cycle progression and prevents premature mitotic progression by sequestering an anaphase-promoting complex/cyclosome activator. Additionally, REV7 supports genome integrity by directing double-strand break repair pathway choice as part of the recently characterized mammalian shieldin complex. Given that genome instability is a hallmark of cancer, it is unsurprising that REV7, with its numerous genome maintenance roles, is implicated in multiple malignancies, including ovarian cancer, glioma, breast cancer, malignant melanoma, and small-cell lung cancer. Moreover, high REV7 expression is associated with poor prognoses and treatment resistance in these and other cancers. Promisingly, early studies indicate that REV7 suppression enhances sensitivity to chemotherapeutics, including cisplatin. This review aims to provide a comprehensive overview of REV7’s myriad roles in genome maintenance and other functions as well as offer an updated summary of its connections to cancer and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ab376b5318f868977c071b0b61bc16785632df" target='_blank'>
              REV7: a small but mighty regulator of genome maintenance and cancer development
              </a>
            </td>
          <td>
            Lara Maggs, Mitch McVey
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, 
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3abca3c8d34b3839d67fd9bca980673db8e3a1ff" target='_blank'>
              Evaluating H2BC9 as a potential diagnostic and prognostic biomarker in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Lanhua Wu, Liang Li, Mingjing Zhu, Zi-jian Zhou, Xuejin Su, Yueming Jiang, Min Kang, 
          </td>
          <td>2025-01-27</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5181ddbe51c7dee5ac944fbf5ce920c81fec81b8" target='_blank'>
              Enhancer regulatory networks globally connect non-coding breast cancer loci to cancer genes
              </a>
            </td>
          <td>
            Yihan Wang, Daniel A Armendariz, Lei Wang, Huan Zhao, Shiqi Xie, 
          </td>
          <td>2025-01-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Numerical abnormalities in chromosomal states, referred to as aneuploidy, is commonly observed in many cancer cells. Although numerous internal and external factors induce aneuploidy, the primary cause of aneuploidy in humans remains unclear. DNA damage is identified as a potential cause of aneuploidy by inducing chromosome segregation errors. However, a direct relationship between DNA damage and aneuploidy remains poorly understood. A major reason for this is the extremely low frequency of aneuploidy in cultured cells, making quantitative analyses challenging. In this study, we investigated the relationship between DNA damage and aneuploidy in cell lines containing minichromosomes. These chromosomes are more prone to loss than normal chromosomes, with the rate of loss substantially increased following exposure to various DNA-damaging agents. To determine whether damaged chromosomes were subjected to direct loss or whether chromosome loss occurred as an indirect consequence of a prolonged G2 phase or other factors, we used the CRISPR-Cas9 system to introduce a single DNA double-strand break (DSB) on a minichromosome. The rate of minichromosome loss increased by approximately seven-fold compared with that of the control. Furthermore, the loss rate was significantly elevated in the absence of KU70, a key factor in non-homologous end joining, and upon inhibition of ataxia telangiectasia mutated (ATM), a DNA damage checkpoint protein. Finally, two closely spaced nicks, believed to generate a 5’-overhang, were also shown to induce minichromosome loss. These findings indicated that a single DSB or two closely spaced nicks can cause aneuploidy if improperly repaired in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f1468476733a77f4e32e0f0402d6f84ad1e95" target='_blank'>
              Quantitative analysis of the frequency of chromosome loss following DSB induction
              </a>
            </td>
          <td>
            Seiya Matsuno, Ryo Ishida, Ryotaro Kawasumi, Kouji Hirota, Takuya Abe
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Significance Mutations in the metabolic enzyme isocitrate dehydrogenase (IDH1) are frequent and considered early events in a subset of adult gliomas. The mutant IDH1 enzyme changes functionally, favoring the production of the oncometabolite 2-hydroxyglutarate, and alters metabolic and epigenetic pathways. However, a comprehensive understanding of the epigenetic landscape in IDH1mut cells and how it contributes to tumorigenesis is lacking. This study presents a high-dimensional single-cell analysis of chromatin alterations driven by mutant-IDH1. Our analysis reveals extensive remodeling of chromatin patterns, highlighting a rapid decrease in histone acetylation marks and increased epigenetic heterogeneity. These changes in histone acetylation, affecting enhancer patterns, are strongly associated with the metabolic abnormalities of IDH1mut cells, underscoring the interplay between chromatin and metabolism dysregulation in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1395bf17f297913f9bbe2a4e1db052179fa9f3f" target='_blank'>
              Oncogenic IDH1mut drives robust loss of histone acetylation and increases chromatin heterogeneity
              </a>
            </td>
          <td>
            N. Furth, Niv Cohen, Avishay Spitzer, T. Salame, B. Dassa, Tevie Mehlman, Alexander Brandis, Arieh Moussaieff, D. Friedmann-Morvinski, Maria G Castro, Jerome Fortin, M. Suvà, I. Tirosh, A. Erez, Guy Ron, E. Shema
          </td>
          <td>2024-12-30</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="In most eukaryotic cells, euchromatin is localized in the nuclear interior, whereas heterochromatin is enriched at the nuclear envelope (NE). This conventional chromatin organization is established by heterochromatin tethering to the NE, however its importance for cellular homeostasis is largely unexplored. Peripheral heterochromatin localization relies on redundant NE-tethering systems. One tether is constituted by the lamin B receptor (LBR) in mammals, but the enigmatic nature of the other tethers has hampered functional analyses. Here we demonstrate that the downregulation of abundant, ubiquitous NE proteins can induce the global detachment of heterochromatin from the NE. Among these factors, we identify LBR and LAP2 as major players in bulk heterochromatin attachment to the NE in pluripotent and differentiated mammalian cells. Their loss leads to repositioning of heterochromatin to the nuclear interior, changes in chromatin accessibility, deregulation of gene expression including activation of antiviral innate immunity, and defects in cell fate determination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fde3d7b0c4771c759bd6fcaac94c205c62bfd4c" target='_blank'>
              LBR and LAP2 mediate heterochromatin tethering to the nuclear periphery to preserve genome homeostasis
              </a>
            </td>
          <td>
            Renard Lewis, Virginia Sinigiani, K. Koos, Cristiana Bersaglieri, , Raffaella Santoro, C. Ciaudo, Peter Horvath, Puneet Sharma, Ulrike Kutay
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Both DNA replication and RNA transcription utilize genomic DNA as their template, necessitating spatial and temporal separation of these processes. Conflicts between the replication and transcription machinery, termed transcription-replication conflicts (TRCs), pose a considerable risk to genome stability, a critical factor in cancer development. While several factors regulating these collisions have been identified, pinpointing primary causes remains difficult due to limited tools for direct visualization and clear interpretation. In this study, we directly visualize TRCs using a proximity ligation assay (PLA), leveraging antibodies specific to PCNA and phosphorylated CTD of RNA polymerase II. This approach allows precise measurement of TRCs between replication and transcription processes mediated by RNA polymerase II. The method is further enhanced through DNA primers conjugated covalently to these antibodies, coupled with PCR amplification using fluorescent probes, providing a highly sensitive and specific means of detecting endogenous TRCs. Fluorescence microscopy enables the visualization of these conflicts, offering a powerful tool to study genome instability mechanisms associated with cancer. This technique addresses the gap in direct TRC visualization, allowing for a more comprehensive analysis and understanding of the underlying processes driving genome instability in cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f574fa952d860bc38fdda116785a24f5d5ebcd3f" target='_blank'>
              Proximity Ligation Assay to Study Oncogene-Derived Transcription-Replication Conflicts.
              </a>
            </td>
          <td>
            Linling Ke, Qian Xie, Xiaoman Wang, Yuanhang Gong, Min Li
          </td>
          <td>2025-01-10</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Upon infection, human papillomavirus (HPV) manipulates host cell gene expression to create an environment that is supportive of a productive and persistent infection. The virus-induced changes to the host cell's transcriptome are thought to contribute to carcinogenesis. Here, we show by RNA-sequencing that oncogenic HPV18 episome replication in primary human foreskin keratinocytes (HFKs) drives host transcriptional changes that are consistent between multiple HFK donors. We have previously shown that HPV18 recruits the host protein CTCF to viral episomes to control the differentiation-dependent viral transcriptional programme. Since CTCF is an important regulator of host cell transcription via coordination of epigenetic boundaries and long-range chromosomal interactions, we hypothesised that HPV18 may also manipulate CTCF to contribute to host transcription reprogramming. Analysis of CTCF binding in the host cell genome by ChIP-Seq revealed that while the total number of CTCF binding sites is not altered by the virus, there are a sub-set of CTCF binding sites that are either enriched or depleted of CTCF. Many of these altered sites are clustered within regulatory elements of differentially expressed genes, including the tumour suppressor gene cell adhesion molecule 1 (CADM1), which supresses epithelial cell growth and invasion. We show that HPV18 establishment results in reduced CTCF binding at the CADM1 promoter and upstream enhancer. Loss of CTCF binding is coincident with epigenetic repression of CADM1, in the absence of CpG hypermethylation, while adjacent genes including the transcriptional regulator ZBTB16 are activated. These data indicate that the CADM1 locus is subject to topological rearrangement following HPV18 establishment. We tested this hypothesis using 4C-Seq (circular chromosome confirmation capture-sequencing) and show that HPV18 establishment causes a loss of long-range chromosomal interactions between the CADM1 transcriptional start site and the upstream transcriptional enhancer. These data show that HPV18 manipulates host cell promoter-enhancer interactions to drive transcriptional reprogramming that may contribute to HPV-induced disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/701c7f934a3a0661690be118144129e2077bd9be" target='_blank'>
              Repression of CADM1 transcription by HPV type 18 is mediated by three-dimensional rearrangement of promoter-enhancer interactions.
              </a>
            </td>
          <td>
            Karen Campos-León, J. Ferguson, Thomas Günther, C. D. Wood, S. Wingett, Selin Pekel, C. S. Varghese, Leanne S Jones, Joanne D. Stockton, C. Várnai, M. West, Andrew Beggs, A. Grundhoff, B. Noyvert, Sally Roberts, Joanna L. Parish
          </td>
          <td>2025-01-27</td>
          <td>PLoS pathogens</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="TAL1 is overexpressed in 40-60% of T-cell acute lymphoblastic leukemia (T-ALL) cases and forms an oncogenic core regulatory circuit (CRC) with other transcription factors such as LMO1, LMO2 and GATA3. In 5% of T-ALL cases an insertion of a consensus GT dinucleotide (MuTE) is observed upstream of the TAL1 gene, driving TAL1 overexpression. Using an in vitro reconstitution DNA pull-down assay combined with quantitative mass spectrometry, we identified proteins that preferentially bound to the MuTE sequence and demonstrated that among the candidates the RNA methyltransferase TARBP1 and the zinc finger proteins ZBTB2, ZBTB25 and ZNF639 form a complex that we term TARZN. Interestingly, the TARZN complex also bound to de novo super enhancer sites upstream of the LMO1 and LMO2 genes in T-ALL cells, indicating a putative common mechanism between these different non-coding driver mutations. Furthermore, knock-down of all TARZN members resulted in lower TAL1 protein expression in MuTE-positive but not in MuTE-negative T-ALL cells. Given TARZN’s methyltransferase activity and the lack of concomitant TAL1 mRNA level changes, we investigated reduced TAL1 translation and identified reduced neo-synthesised TAL1 protein levels upon TARBP1 knockdown. Overall, these data suggest that the TARZN complex promotes oncogenic expression in T-ALL via co-transcriptional RNA methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c3cc9fd0c83ba3cc4d7a530bb55531a564291e6" target='_blank'>
              The TARZN complex binds de novo enhancer mutations and promotes oncogenic expression in T-ALL
              </a>
            </td>
          <td>
            Nurkaiyisah Zaal Anuar, C. Goh, Boon Haow Chua, Shi Hao Tan, Joana R Costa, Marc R. Mansour, Takaomi Sanda, Dennis Kappei
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, , Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, , Gabrielle Goldman Levy, , Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="BRAF inhibitor-resistant melanomas (BRAFiR) acquire (epi)genetic and functional alterations that enable them to evade alternative treatments. Identifying these alterations is critical to advancing treatment strategies. Here, we explored the effect of Chk1 inhibition (Chk1i) on BRAFiR cells, revealing higher sensitivity compared to treatment-naïve cells both in vitro and in vivo. Using FUCCI-labeling and time-lapse microscopy, we show that S phase progression is required for Chk1i-induced cytotoxicity in BRAFiR cells, but not in treatment-naïve cells. Replication stress markers, including reduced BrdU incorporation and increased phospho-RPA and γH2AX, were exclusive to BRAFiR cells exposed to Chk1i. Untreated BRAFiR cells exhibited upregulated DNA replication genes, reduced progressing forks and increased origin firing, suggesting intrinsic replication changes. MAPK pathway reactivation in treatment-naïve cells mimicked BRAFiR traits, increasing sensitivity to Chk1i. These findings indicate that Chk1i exploits elevated replication stress specifically in BRAFiR, highlighting its therapeutic potential in overcoming MAPK inhibitor resistance in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b5d8e2bacae8698be1cc41dc94d4d67c11c52a" target='_blank'>
              Resistance to BRAF inhibitors drives melanoma sensitivity to Chk1 inhibition
              </a>
            </td>
          <td>
            D. G. Carvalho, J. Kenski, Daniel A. Moreira, Matheus A. Rajão, O. Krijgsman, Carolina Furtado, M. Boroni, João P. B. Viola, D. Peeper, P. Possik
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="RAD18 is a conserved eukaryotic E3 ubiquitin ligase that promotes genome stability through multiple pathways. One of these is gap-filling DNA synthesis at active replication forks and in post-replicative DNA. RAD18 also regulates homologous recombination (HR) repair of DNA breaks; however, the current literature describing the contribution of RAD18 to HR in mammalian systems has not reached a consensus. To investigate this, we examined three independent RAD18-null human cell lines. Our analyses found that loss of RAD18 in HCT116, but neither hTERT RPE-1 nor DLD1 cell lines, resulted in elevated sister chromatid exchange, gene conversion, and gene targeting, i.e., HCT116 mutants were hyper-recombinogenic (hyper-rec). Interestingly, these phenotypes were linked to RAD18’s role in PCNA K164 ubiquitination, as HCT116 PCNAK164R/+ mutants were also hyper-rec, consistent with previous studies in rad18−/− and pcnaK164R avian DT40 cells. Importantly, the knockdown of UBC9 to prevent PCNA K164 SUMOylation did not affect hyper-recombination, strengthening the link between increased recombination and RAD18-catalyzed PCNA K164 ubiquitination, but not K164 SUMOylation. We propose that the hierarchy of post-replicative repair and HR, intrinsic to each cell type, dictates whether RAD18 is required for suppression of hyper-recombination and that this function is linked to PCNA K164 ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ade4d531e6f454e305c916fc9c53a0c286d620e" target='_blank'>
              Cell Type Specific Suppression of Hyper-Recombination by Human RAD18 Is Linked to Proliferating Cell Nuclear Antigen K164 Ubiquitination
              </a>
            </td>
          <td>
            Colette B. Rogers, W. Leung, Ryan M. Baxley, Rachel E. Kram, Liangjun Wang, Joseph P. Buytendorp, Khoi Le, D. Largaespada, Eric A. Hendrickson, A. Bielinsky
          </td>
          <td>2025-01-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Identifying the genes capable of driving tumorigenesis in different tissues is one of the central goals of cancer genomics. Computational methods that exploit different signals of positive selection in the pattern of mutations observed in genes across tumors have been developed to this end. One such signal of positive selection is clustering of mutations in areas of the three-dimensional (3D) structure of the protein above the expectation under neutrality. Methods that exploit this signal have been hindered by the paucity in proteins with experimentally solved 3D structures covering their entire sequence. Here, we present Oncodrive3D, a computational method that by exploiting AlphaFold 2 structural models extends the identification of proteins with significant mutational 3D clusters to the entire human proteome. Oncodrive3D shows sensitivity and specificity on par with state-of-the-art cancer driver gene identification methods exploiting mutational clustering, and clearly outperforms them in computational efficiency. We demonstrate, through several examples, how significant mutational 3D clusters identified by Oncodrive3D in different known or potential cancer driver genes can reveal details about the mechanism of tumorigenesis in different cancer types and in clonal hematopoiesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f19ed942c2bb363999085861c8459ffabedab4e8" target='_blank'>
              Oncodrive3D: Fast and accurate detection of structural clusters of somatic mutations under positive selection
              </a>
            </td>
          <td>
            Stefano Pellegrini, Ferran Muiños, Núria López-Bigas, A. González-Pérez
          </td>
          <td>2025-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="SETD6 is a non-histone lysine methylatransferase, previously shown to participate in several housekeeping signaling pathways such as the NFkB pathway, Wnt signaling pathway, mitosis and more. In the current study we show evidence that SETD6 methylation is involved in the regulation of cytokinesis - the final process that divides cell contents into two daughter cells. SETD6 depleted HeLa cells presented high levels of chromatin bridges and actin patches, which are commonly observed following chromosomal segregation errors. In a proteomic screen we identified Aurora-B as a novel SETD6 substrate. Aurora-B kinase is an essential regulator of cytokinesis, known to actively delay cytokinesis as a response to the presence of chromatin in the midzone. We found that SETD6 binds and methylates Aurora-B on two adjacent lysine residues. Upon replication stress, Aurora-B methylation by SETD6 increases but is abolished when the two lysine methylation targets are substituted. In addition, replication stress led to a high tendency of SETD6 depleted cells to multinucleate, a major chromosomal-instability (CIN) phenotype. We detected a significant reduction in the Aurora-B kinase activity during cytokinesis in SETD6 knockout cells upon replication stress, which could be the mechanism underlying the accumulation of CIN phenotypes in these cells. CIN is a hallmark of cancer and is associated with tumor cell malignancy. Our findings suggest that Aurora-B methylation by SETD6 carries meaningful implications on tumorigenic cellular pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30079fd8588b1378d0602ce8bccaf091eb97d109" target='_blank'>
              AuroraB-kinase methylation by SETD6 regulates cytokinesis and protects cells from chromosomal instability
              </a>
            </td>
          <td>
            , , Dikla Nachmias, K. Biggar, Daniel Sevilla, Natalie Elia, D. Levy
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Tumor protein 53 (TP53) is a tumor repressor gene that is highly responsible for activating cell cycle arrest in the event of DNA damage, stopping the replication of damaged cells, which may cause cancer. However, when mutated, TP53 is an oncogene, with mutated variants of it being seen in over half of all cancers diagnosed including the aforementioned four. With groundbreaking CRISPR-Cas9 gene editing technology coming to the forefront of biotechnological research, the applications to cancer research and gene therapy have been increasing. This review explores CRISPR-Cas9 and its uses for treatment and medication investigation in various cancers, including lung, prostate, breast, and bone cancer through the lens of the TP53 gene. Due to TP53s commonality in cancer, CRISPR-Cas9 gene editing technology can be used to its maximum potential in this context as a treatment or investigative tool, with it yielding positive results in reverting tumor growth, as well as great versatility when it comes to screening related genes that inhibit or encourage tumor growth which can be targeted by drugs, or even the screening of the drugs effectiveness themselves. Thus, CRISPR-Cas9s applications in TP53-related cancer treatment and chemotherapeutic fields prove this technology to be successful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da087d15d0c931988cddd598ffd040ef96f3cb9b" target='_blank'>
              Using CRISPR/Cas9 Technology to Target TP53 Related Cancers
              </a>
            </td>
          <td>
            Man Ho Damien Hsieh
          </td>
          <td>2025-01-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b4cf48abf7265148385bfea0bb2728f6fd6101" target='_blank'>
              SMYD3 drives cell cycle and epithelial-mesenchymal transition pathways through dual gene transcriptional repression and activation in HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Madhavi Murali, Abbas Saeed, Sohyoung Kim, K. Burkitt, Hui Cheng, Arfa Moshiri, Jawad Akhtar, D. Tsai, Marie Luff, Baktiar Karim, V. Saloura
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Molecular targeted drugs are frequently used in precision cancer therapies. However, the prolonged administration of these agents can result in drug resistance. Cancer cells exhibiting defects in homologous recombination are particularly susceptible to genotoxic stress, such as that induced by PARP inhibitors (PARPi), resulting in synthetic lethality. Here, we show that sustained treatment of such cancer cells with PARPi selectively upregulates YAP1-2α, a minor isoform of YAP1. Elevated YAP1-2α heterodimerizes with TAZ to induce liquid-liquid phase separation (LLPS) in the nucleus. This LLPS-driven process generates nuclear condensates that activate a super-enhancer comprising YAP1-2α, TAZ, TEAD, and BRD4. The super-enhancer reconfigures transcriptional networks to enhance cancer stem-like malignant properties, thereby potentiating cell resilience and augmenting cell resistance to adverse conditions, including precision cancer therapies. Our study reveals the presence of a super-enhancer that promotes malignant tumor progression. Targeting the YAP1-2α/TAZ-induced super-enhancer is a promising strategy for overcoming resistance to anticancer modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0a7f11bb9d0a8943e8f9ed992501df776b3e17" target='_blank'>
              A super-enhancer actuated by heterodimerization of a specific YAP1 isoform, YAP1-2α, with TAZ endows cancer stemness and drug resistance
              </a>
            </td>
          <td>
            Takuya Ooki, Takeru Hayashi, Yui Yamamoto, Masanori Hatakeyama
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Structural variation (SV), defined as balanced and unbalanced chromosomal rearrangements >1 kb, is a major contributor to germline and neoplastic disease. Large variants have historically been evaluated by chromosome analysis and now are commonly recognized by chromosomal microarray analysis (CMA). The increasing application of genome sequencing (GS) in the clinic and the relatively high incidence of chromosomal abnormalities in sick newborns and children highlights the need for accurate SV interpretation and reporting. In this review, we describe SV patterns of common cytogenetic abnormalities for laboratorians who review GS data.


CONTENT
GS has the potential to detect diverse chromosomal abnormalities and sequence breakpoint junctions to clarify variant structure. No single GS analysis pipeline can detect all SV, and visualization of sequence data is crucial to recognize specific patterns. Here we describe genomic signatures of translocations, inverted duplications adjacent to terminal deletions, recombinant chromosomes, marker chromosomes, ring chromosomes, isodicentric and isochromosomes, and mosaic aneuploidy. Distinguishing these more complex abnormalities from simple deletions and duplications is critical for phenotypic interpretation and recurrence risk recommendations.


SUMMARY
Unlike single-nucleotide variant calling, identification of chromosome rearrangements by GS requires further processing and multiple callers. SV databases have caveats and limitations depending on the platform (CMA vs sequencing) and resolution (exome vs genome). In the rapidly evolving era of clinical genomics, where a single test can identify both sequence and structural variants, optimal patient care stems from the integration of molecular and cytogenetic expertise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a2a826f1810ca4839038af6a42ce3daef01b34" target='_blank'>
              Structural Variation Interpretation in the Genome Sequencing Era: Lessons from Cytogenetics.
              </a>
            </td>
          <td>
            Lucilla Pizzo, M. K. Rudd
          </td>
          <td>2025-01-01</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genome functions are regulated by the means of chromatin and 3D chromosome organization. Dynamic condensation and relaxation of chromosomes during cell cycle is largely controlled by the Condensin complexes. We mapped mutants in Condensin II subunits SMC2A, CAP-D3 and CAP-H2 as hypersensitive to DNA-protein crosslink (DPC) inducer zebularine. This suggested that Condensin II complex is required for resistance to genotoxic stress in Arabidopsis and prompted us to explore the underlying phenotypes. We show that the Condensin II role in resistance to zebularine is independent of the DNA damage response signaling by SOG1 and the homology-directed repair pathway. Furthermore, we found that Arabidopsis Condensin II mutants have incompletely condensed mitotic chromosomes and show abnormal chromatin connections during anaphase. Based on the structural and temporal analyses, we propose that the connections represent catenated chromatids. The chromosome connections were more frequent when the Condensin II mutants were treated by zebularine or TOP2 crosslinker and inhibitor ICRF-187. Altogether, we demonstrate that the proper large-scale chromatin organization by Condensin II is important for resistance to DNA damage response-inducing agents in Arabidopsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6084d605a281f1ce54dec4a816d64fd93e825187" target='_blank'>
              Condensin II mediates efficient chromatid resolution and resistance to genotoxic stress in Arabidopsis
              </a>
            </td>
          <td>
            Jovanka Vladejić, Klára Procházková, E. Tomaštíková, Zynke de Cock, Jana Zwyrtková, Kateřina Kaduchová, Markéta Pavlíková, A. Pečinka
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, 
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, , Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, , Katelyn A. Queen, Allison R Racela, , Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetic therapies are emerging for pediatric cancers. Due to the relatively low mutational burden in pediatric tumors, epigenetic dysregulation and differentiation blockade is a hallmark of oncogenesis in some childhood cancers. By targeting epigenetic regulators that maintain tumor cells in a primitive developmental state, epigenetic therapies may induce differentiation. The most well-studied and clinically advanced epigenetic-targeted therapies include azacitidine and decitabine, which inhibit DNA methylation through competitive inhibition of the enzymatic activity of the DNA methyltransferase family enzymes. These DNA hypomethylating agents are Food and Drug Administration (FDA) approved for hematologic malignancies. The discovery that DNA hypermethylation occurs in patients with isocitrate dehydrogenase (IDH) mutations has led to the development and FDA approval of IDH inhibitors for hematologic and solid tumors. Epigenetic dysregulation in pediatric tumors is also driven by changes in the "histone code" that either promote oncogene expression or repress tumor suppressors. Cancers whose chromatin landscape is characterized by such aberrant histone posttranslational modifications may be amenable to targeted therapies that inhibit the chromatin-modifying enzymes that read, write, and erase these histone modifications. Small molecules that inhibit the enzymatic activity of histone deacetylases, acetyltransferases, and methyltransferases have been approved for the treatment of some adult cancers, and these agents are currently under investigation in various pediatric tumors. Chromatin regulatory complexes can be hijacked by oncogenic fusion proteins that are produced by chromosomal translocations, which are common drivers in pediatric cancer. Small molecules that disrupt oncogenic fusion protein activity and their associated chromatin complexes have demonstrated remarkable promise, and this approach has become the standard treatment for a subset of leukemias driven by the PML-RARA oncogenic fusion protein. A deeper understanding of the mechanisms that drive epigenetic dysregulation in pediatric cancer may hold the key to future success in this field, as the landscape of druggable epigenetic targets is also expanding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a807fa68d0895be1a6925ed8f18dcca64f6453e" target='_blank'>
              Epigenetic Therapies.
              </a>
            </td>
          <td>
            Wallace Bourgeois, Scott A Armstrong, Emily B. Heikamp
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MERVL (murine endogenous retrovirus with leucine tRNA primer) is expressed during zygotic genome activation (ZGA) in mammalian embryos. Here, we show that the Polycomb group ring finger 5 (Pcgf5), a key component of Polycomb repressive complex 1 (PRC1), forms a chimeric transcript with MT2C_Mm, one of the long terminal repeat sequences of MERVL. Knockdown of Pcgf5 reduced developmental rates and decreased H3K27me3 and H2AK119ub1 modification during embryogenesis. In addition, not only genes expressed during ZGA but also imprinting genes were upregulated in Pcgf5 knockdown embryos. Moreover, Pcgf5 was involved in the addition of the H3K27me3 modification to the maternal Xist region. This is the first report of a MERVL-regulated transcript regulating Xist expression in mouse preimplantation embryos. Our results suggest that analysis of chimeric transcripts with MERVL will provide insight into the relationship between ZGA and noncanonical imprinting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2677f268796f2526b7e3f4f5f39fc1d3e544cfd" target='_blank'>
              Endogenous retrovirus-driven Pcgf5 plays critical roles in zygotic genome activation and noncanonical imprinting
              </a>
            </td>
          <td>
            Satoshi Mashiko, Takuto Yamamoto, N. Minami, Shunta Ikeda, Shinnosuke Honda
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Deregulated MYCN is a driver of aggressive pediatric and adult neuroendocrine tumors, but critical oncogenic processes downstream of MYCN remain poorly defined. In neuroblastoma, MYCN interacts with and activates the Aurora-A kinase. Here we show that Aurora-A is a CDK-activating kinase for CDK12 by phosphorylating T893 in the T-loop, thereby enhancing its kinase activity. Aurora-A-dependent activation of CDK12 controls phosphorylation of T4 of RNA polymerase and recruits transcription termination complexes, thereby preventing transcription-replication conflicts. Enhanced crosslinking and immunoprecipitation sequencing reveals that Aurora-A associates with splice sites on nascent RNA. RNA-bound Aurora-A is catalytically inactive. MYCN competes with RNA for binding to Aurora-A and displaces Aurora-A from RNA in cells, promoting its CDK12 kinase activity. Combining Aurora-A and CDK12 inhibition potently suppresses the growth of MYCN-amplified neuroblastoma cells and patient-derived xenografts. Our data demonstrate that an Aurora-A/CDK12-dependent transcription termination pathway is a critical and targetable dependency of MYCN-driven tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5883a603efd4d96de04c7785382de3b59b13dcb8" target='_blank'>
              The MYCN/Aurora-A complex is a cyclin activating kinase for CDK12
              </a>
            </td>
          <td>
            , Lorenz Eing, Leonie Uhl, Maximilian Schmitz, Iliyana Kaneva, Dominic P Byrne, Daniel Fleischhauer, S. G. Burgess, , Nick Gebauer, Vanessa Nancy-Portebois, I. H. Kaltheuner, Stefanie Anh Ha, Christina Schülein-Völk, Daniel Solvie, Mark W. Richards, Raphael Vidal, Matthias Brand, Dimitrios Papadopoulos, Richard Bayliss, P. Eyers, C. Eyers, Matthias Geyer, Martin Eilers, G. Büchel
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f345edfe89fa16dab1e87b151db2db75ec0ac8dc" target='_blank'>
              Insulin signaling regulates R2 retrotransposon expression to orchestrate transgenerational rDNA copy number maintenance
              </a>
            </td>
          <td>
            Jonathan O. Nelson, Alyssa Slicko, Amelie A Raz, Yukiko M. Yamashita
          </td>
          <td>2025-01-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Selection of a suitable chromatin environment during retroviral integration is a tightly regulated and multilayered process that involves interplay between viral and host factors. However, whether intrinsic chromatin dynamics during mitosis modulate retroviral genome invasion is currently poorly described. Direct interaction between the spumaretrovirus prototype foamy virus (PFV) Gag protein and cellular chromatin has been described as a major determinant for integration site selection. A previous Gag chromatin-binding site (CBS)–nucleosome co-crystal structure revealed an interaction with the histone H2A-H2B acidic patch via a highly conserved arginine anchor residue. Yet, the molecular mechanisms regulating Gag-chromatin capture during PFV infection remain obscure. Here, we investigated the kinetics of Gag-chromatin interactions during mitosis and proviral integration of PFV-infected synchronized cells. Using Gag CBS variant viruses, we showed that alteration of Gag affinity for nucleosome binding induced untimely chromatin tethering during mitosis, decreased infectivity and redistributed viral integration sites to markers associated with late replication timing of host chromosomes. Mutant Gag proteins were moreover defective in their ability to displace the histone H4 tail from the nucleosome acidic patch of highly condensed mitotic chromatin. These data indicate that the mitotic chromatin landscape during Gag–nucleosome interactions hosts PFV integration site selection determinants and that spumaretroviruses evolved high-affinity chromatin binding to overcome early mitosis chromatin condensation for optimal viral DNA tethering, integration and infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c034dba5335395c865d3f3e018bafaae4ee93b" target='_blank'>
              Timed chromatin invasion during mitosis governs prototype foamy virus integration site selection and infectivity
              </a>
            </td>
          <td>
            F. Lagadec, P. K. Singh, C. Calmels, D. Lapaillerie, D. Lindemann, V. Parissi, P. Cherepanov, A. N. Engelman, P. Lesbats
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA, our findings rather suggest that protein-DNA co-condensates ‘sense’ the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d84d86b264f5c1ad8de5c81e2ae317173c400dd" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Thejas Satheesh, Rohit Kumar Singh, Prakshi Gaur, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, , Mahipal Ganji
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Random mutations followed by natural selection is one of the key mechanisms during natural evolution that results in the generation of new functions. This process is typically slow in nature because of low mutational frequency. In contrast, the B cells of mammalian immune systems have evolved somatic hypermutation (SHM) mechanisms that introduce mutations at the immunoglobulin genomic loci at a significantly higher frequency than the rest of the genome. SHM allows B cells to rapidly evolve new antibody sequences without compromising their fitness as a consequence of genome-wide mutations. In this work, we developed a continuous directed evolution platform in human B cell lines (CODE-HB) that recruits and repurposes the SHM mechanisms to rapidly evolve reporter proteins like the Green Fluorescent Protein. This approach uses a stable, non-immunoglobulin locus within the genomes of human B cell lines. To comprehensively characterize the mutational profile and breadth of this strategy, we performed single-molecule sequencing experiments. We illustrate the utility of the platform by rapidly evolving antibody fragments in a continuous manner by B cell surface display targeting avian subtypes of influenza hemagglutinin (e.g., H5) from emerging strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a837142332a2e1fd6fe0933cdb79622e5382b338" target='_blank'>
              Virus-free continuous directed evolution in human cells using somatic hypermutation
              </a>
            </td>
          <td>
            Stanley Bram, Jordan Quenneville, , Sarah Leach, Jenna J. Guthmiller, Angad P Mehta
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The DNA double-strand break (DSB) always occurs within genome. The pathways that repair the DSB are essential in keeping cell viability. This review will focus on the recent understanding of TMEJ, especially two key protein factors involved in the TMEJ repair pathway2, DNA polymerase theta (Polθ) and DNA polymerase delta (Polδ), and discuss the potential mechanism of choosing repair pathways when there is a DNA Double Strand Break. We dissect TMEJ's unique mechanism of action, including recognition of DNA ends, microhomology search, end pairing, and DNA synthesis, by focusing on the enzyme Polθ as the central player to understand the repair pathway choice. We also discuss how BRCA-mutated cancer uses the TMEJ repair pathway to facilitate breast cancer cells' growth3. This review aims to provide a comprehensive overview of TMEJ's cellular functions, regulatory mechanisms, and pivotal role in BRCA-mutated breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ba6a1c7fb204a897897b1b79890ca6faef600e" target='_blank'>
              Polymerase Theta-mediated DNA End-Joining Repair in the Maintenance of Genome Stability
              </a>
            </td>
          <td>
            Xiangyu Yan
          </td>
          <td>2024-12-24</td>
          <td>Transactions on Materials, Biotechnology and Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genome maintenance is essential for the integrity of the genetic blueprint, of which only a small fraction is transcribed in higher eukaryotes. DNA lesions occurring in the transcribed genome trigger transcription pausing and transcription-coupled DNA repair. There are two major transcription-coupled DNA repair pathways. The transcription-coupled nucleotide excision repair (TC-NER) pathway has been well studied for decades, while the transcription-coupled homologous recombination repair (TC-HR) pathway has recently gained attention. Importantly, recent studies have uncovered crucial roles of RNA transcripts in TC-HR, opening exciting directions for future research. Transcription also plays pivotal roles in regulating the stability of highly specialized genomic structures such as telomeres, centromeres, and fragile sites. Despite their positive function in genome maintenance, transcription and RNA transcripts can also be the sources of genomic instability, especially when colliding with DNA replication and forming unscheduled pathological RNA:DNA hybrids (R-loops), respectively. Pathological R-loops can result from transcriptional stress, which may be induced by transcription dysregulation. Future investigation into the interplay between transcription and DNA repair will reveal novel molecular bases for genome maintenance and transcriptional stress-associated genomic instability, providing therapeutic targets for human disease intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd6afacedaa734f271cf97ef11d6135943f79a8" target='_blank'>
              Transcription as a double-edged sword in genome maintenance.
              </a>
            </td>
          <td>
            Ouyang Jian
          </td>
          <td>2024-12-20</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Short tandem repeats (STRs) have been reported to influence gene expression across various human tissues. While STR variations are enriched in colorectal (CRC), stomach (STAD) and endometrial (UCEC) cancers, particularly in microsatellite instable (MSI) tumors, their functional effects and regulatory mechanisms on gene expression remain poorly understood across these cancer types. Results Here, we leverage whole-exome sequencing and gene expression data to identify STRs for which repeat lengths are associated with the expression of nearby genes (eSTRs) in CRC, STAD and UCEC tumors. Our analyses reveal that tumor STR profiles effectively capture both MSI phenotype and population structure. While most eSTRs are cancer-specific, shared eSTRs across multiple cancers exhibit consistent effects on gene expression. Notably, coding-region eSTRs identified in all three cancer types show positive correlations with nearby gene expression. We further validate the functional effects of eSTRs by demonstrating associations between somatic eSTR mutations and gene expression changes during the transition from normal to tumor tissues, suggesting their potential roles in tumorigenesis. Combined with DNA methylation data, we perform the first quantitative analysis of the interplay between STR variations and DNA methylation in tumors. We identify eSTRs where repeat lengths are associated with methylation levels of nearby CpG sites (meSTRs) and show that over 70% of eSTRs are significantly linked to local DNA methylation. Importantly, the effects of meSTRs on DNA methylation remain consistent across cancer types. Conclusions Overall, our findings enhance the understanding of how functional STR variations influence gene expression and DNA methylation. Our study highlights shared regulatory mechanisms of STRs across multiple cancers, offering a foundation for future research into their broader implications in tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a625ce20d0d6f7f915320a2751a41cca4a94608" target='_blank'>
              Multicancer analyses of short tandem repeat variations reveal shared gene regulatory mechanisms
              </a>
            </td>
          <td>
            Feifei Xia, M. Verbiest, Oxana Lundström, Tugce Bilgin Sonay, M. Baudis, Maria Anisimova
          </td>
          <td>2025-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            , Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) is an aggressive and lethal neoplasm, with its early neoplastic transformation mechanisms remaining poorly understood. Here, we characterize over 25 genetically-defined organoid models derived from murine and human oral/esophageal tissues harboring key driver mutations. Double knockout of TP53 and CDKN2A induced morphological dysplasia, hyperproliferation, loss of squamous differentiation, and tumorigenicity, which were further exacerbated by additional driver mutations (e.g., PIK3CA, NOTCH1, KMT2C). Single-cell analysis revealed an expansion of quiescent basal cells and proliferative squamous cells, alongside a loss of differentiated squamous cells during malignant transformation. A distinct senescence program, regulated by ANXA1, was markedly diminished during early neoplastic evolution. Mechanistically, the ANXA1-SMAD3-p27KIP1 pathway was identified as a critical regulator of this senescence program, acting to suppress neoplastic features in organoid models. Lastly, our high-throughput, single-organoid-resolution drug screens unexpectedly revealed PIK3CA-driven organoids exhibited sensitivity to Mitomycin C and Onalespib. This study provides novel mechanistic insights into early neoplastic evolution and underscores the value of genetically-defined organoid models for investigating cancer biology and identifying targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7127b7b7361c88e16950c954ef2bf66d966c5299" target='_blank'>
              Genetically Defined Organoid Models Reveal Mechanisms Driving Squamous Cell Neoplastic Evolution and Identify Potential Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hua Zhao, Young Min Park, Yueyuan Zheng, Qiong Mao, Casey Collet, Boyan Hu, Tianming Zhou, Luda Lin, Stephanie Wong, Yuhao Pan, Anette Vistoro Monreal, Uttam Sinha, Parish Sedghizadeh, A. Soragni, De-Chen Lin
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, , 
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The nucleolus is a major subnuclear compartment where ribosomal DNA (rDNA) is transcribed and ribosomes are assembled. In addition, recent studies have shown that the nucleolus is a dynamic organizer of chromatin architecture that modulates developmental gene expression. rDNA gene units are assembled into arrays located in the p-arms of five human acrocentric chromosomes. Distal junctions (DJs) are ∼400 kb sequences adjacent to rDNA arrays that are thought to anchor them at the nucleolus, although the underlying regulatory elements remain unclear. Here we show that DJs display a dynamic chromosome conformation profile in human embryonic stem cells (hESCs). We identified a primate-specific, full-length insertion of the retrotransposon long interspersed nuclear element 1 (LINE1) in a conserved position across all human DJs. This DJ-LINE1 locus interacts with specific regions of the DJ and is upregulated in naïve hESCs. CRISPR-based deletion and interference approaches revealed that DJ-LINE1 contributes to nucleolar positioning of the DJs. Moreover, we found that the expression of DJ-LINE1 is required for maintenance of the structure and transcriptional output of the nucleolus in hESCs. Silencing of DJ-LINE1 leads to loss of self-renewal, disruption of the landscape of chromatin accessibility, and derepression of earlier developmental programs in naïve hESCs. This work uncovers specific LINE1 elements with a fundamental role in nucleolar organization in hESCs and provides new insights into how the nucleolus functions as a key genome-organizing hub.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6f03971b0054eedbed60fad051bf9b167c6012" target='_blank'>
              LINE1 elements at distal junctions of rDNA repeats regulate nucleolar organization in human embryonic stem cells.
              </a>
            </td>
          <td>
            Lamisa Ataei, Juan Zhang, Simon Monis, , Kirti Mittal, Joshua Yang, Mayu Shimomura, B. McStay, Michael D. Wilson, M. Ramalho-Santos
          </td>
          <td>2024-12-20</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background Soluble N-ethylmaleimide-sensitive factor attachment protein receptor (SNARE)-mediated membrane fusion is crucial for autophagy, making YKT6, a key modulator of cell membrane fusion, a potential target for cancer therapy. However, its oncogenic role across different cancers remains unclear. This study was to investigate the prognostic value and potential immunological functions of YKT6, including cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC). Methods Multiple bioinformatics databases, including The Cancer Genome Atlas (TCGA), Cancer Cell Line Encyclopedia (CCLE), and Genotype-Tissue Expression (GTEx) databases, were used to investigate the correlation of the YKT6 expression pattern with the pathological stage and survival rate across cancers. Furthermore, ImmuCellAI, the UCSC Xena platform, and the ESTIMATE algorithm were subsequently utilized to explore the potential relationship between YKT6 expression, the tumor microenvironment, and tumor immune infiltration. Profiling of YKT6 gene mutation and amplification, methylation, and copy number alteration (CNA) was performed on the basis of the TCGA database. Moreover, q-PCR, TMA staining, and siRNA assays were used to validate the cancer-promoting role of YKT6 in CESCs. Results Our results reveal that YKT6 is a potential prognostic and cancer immunity biomarker. Elevated YKT6 expression is correlated with poor overall survival (OS) and disease-free survival (DFS). Distinct gene mutation, methylation, and CNA patterns for YKT6 were found in certain types of cancers. The correlation of YKT6 expression with tumor-infiltrating immune cells was verified by analyzing the StromalScore, ESTIMATEScore, ImmuneScore, and tumor purity. In vitro analysis confirmed that YKT6 was highly expressed in advanced-grade CESCs and that the knockdown of YKT6 inhibited the proliferation of cervical cancer cells. Conclusion The SNARE protein YKT6 serves as a biomarker and candidate oncogene with actionable mutations. Moreover, YKT6 has the potential to be a prognostic indicator in CESCs. Targeting YKT6 could enhance autophagy regulation and improve therapeutic strategies for personalized cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a52302356b487729845e3b92ca230e9c0d31b8ca" target='_blank'>
              Pan-Cancer Analysis Identifies YKT6 as a Prognostic and Immunotherapy Biomarker, with an Emphasis on Cervical Cancer
              </a>
            </td>
          <td>
            Jiamin Liu, Qiang Zhang, Ling He, Huangyu Hu, Yixuan Wang, Ping Xie
          </td>
          <td>2025-01-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinoma (HNSCC) imposes a significant health burden, necessitating innovative therapeutic strategies to enhance treatment efficacy. Current treatments such as surgery, radiation, and chemotherapy have limited effectiveness and yield severe side effects, emphasizing the need for targeted therapies. We have focused on DNA polymerase beta (Polβ) and its roles in replication stress, cellular responses to DNA damaging therapies, and DNA damage response modifiers. Our investigations reveal a regulatory role for base excision repair (BER) proteins, including Polβ, in the cellular response to inhibitors of poly(ADP-ribose) glycohydrolase (PARG), an enzyme involved in poly(ADP-ribose) (PAR) degradation. The inhibition of PARG, in HNSCC cells, elicits replication stress. Further, this activates the PAR-induced S-phase/ATR checkpoint, leading to a block to replication, cell cycle arrest, and the onset of apoptosis. However, Polβ overexpression mitigates this response, reducing replication-stress induced PAR foci formation, suggesting a modulation of replication checkpoint activation. We found that PARG inhibitor treatment is ineffective on HNSCC cells that overexpress Polβ, implying that the PARG inhibitor-induced PAR and apoptotic response is dependent on the level of Polβ. Further, our in vitro experiments demonstrate that combining PARG and ATR/CHK1 inhibitors overcomes Polβ-mediated treatment resistance in HNSCC cells, producing enhanced effects as compared to the individual treatment conditions. Our findings suggest a possible treatment paradigm for HNSCC, employing ATR or CHK1 inhibitors in combination with PARG inhibitors. This strategy offers a promising path for more effective HNSCC treatments, potentially overcoming Polβ-related resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/546af29aab21a1579fd4129cca1f86d70f7aaf27" target='_blank'>
              DNA polymerase beta expression in head & neck cancer modulates the poly(ADP-ribose)-mediated replication checkpoint
              </a>
            </td>
          <td>
            Md Maruf Khan, Wynand P. Roos, Anusha Angajala, Denise Y. Gibbs, Jeffrey C. Liu, Camille Ragin, Robert W. Sobol
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5907495884a56c80db9ffbc9876a701f5ced793e" target='_blank'>
              Role of BRCA1 in glioblastoma etiology.
              </a>
            </td>
          <td>
            Emirhan Harbi, Michael Aschner
          </td>
          <td>2024-12-10</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel F Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, , C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Purpose: Pancreatic cancer (PC) is an aggressive lethal tumor with an unmet need for novel therapeutic approaches. KRAS activating mutations occur in 90-95% of PC and contribute to tumor progression and resistance to therapy, including radiation. The mechanisms by which oncogenic KRAS promotes radiation resistance are critical to understand in order to identify novel therapies. Methods: We first analyzed the expression levels of DNA damage response and repair genes using Affymetrix RNA expression microarray in isogenic HCT116 and SW48 cells with KRAS wide-type and KRASG13D activating mutations to identify novel targets by which KRAS mutations may confer radiation resistance. We analyzed the expression of STN1 in pancreas normal and cancer tissues and assessed the correlation with PC clinical outcomes using TCGA dataset. Human tumor xenografts were generated to explore the role of STN1 on tumor growth in vivo. Radiation response was assessed through clonogenicity and gH2AX foci assays. Homologous recombination (HR) and non-homologous end joining (NHEJ) repair reporter assays, chromatin spreading assay, cell cycle analysis, mitotic catastrophe, Annexin-V assays were performed to investigate the mechanisms of radiation-induced cell death. Mass spectrometry analysis was performed to identify STN1 interacting proteins important in DNA damage response and further validated by immunoprecipitation and immunoblotting. Results: We find that KRAS activation increases STN1 expression to enhance DNA double strand break repair capacity in PC. STN1 is a component of the CST complex normally important for telomere duplication and maintenance. We find that STN1 is significantly upregulated in PC, especially in aggressive subtypes of PC, associates with KRAS oncogenic mutations, and correlates with poor patient clinical outcomes. Genetic silencing or pharmacologic inhibition of KRAS signaling decreases STN1 expression in PC cells, suggesting KRAS signaling positively regulates STN1 expression. Interestingly, STN1 depletion reduces tumor growth in a heterotopic model of KRAS mutant PC. Mechanistically, depletion of STN1 potentiates DNA damage, replication stress, and sensitizes PC cells to ionizing radiation independent of CTC1 and TEN1. In support of this finding, STN1 silencing reduces both HR and NHEJ repair of DSBs. Furthermore, knockdown of STN1 impairs cell cycle arrest at the G2/M phase in response to ionizing radiation, which is accompanied with increased mitotic catastrophe, radiation-induced apoptosis. Proteomic analysis reveals that STN1 physically interacts with many proteins important for DNA repair, replication and cell cycle progression, including ATM, DICER1, CEP164, and CEP250. Conclusion: Our findings have revealed a novel, potentially CST complex-independent role of STN1 in DSB repair after radiation. STN1 may function at one of the apical nodes in the DNA damage response pathway by interacting with ATM. Our findings suggest STN1 may be a promising target for improving genotoxic therapies in KRAS mutant cancers, including PC.
 Citation Format: Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, Sergio Corrales-Guerrero, Sindhu Nair, Joanna M. Karasinska, James T. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams.STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P011">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce1e95ed7e839e47ffa944834be9b3c39c5d903" target='_blank'>
              Abstract P011: STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer
              </a>
            </td>
          <td>
            Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, , Sindhu Nair, J. Karasinska, J. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="UCK2 (Uridine-Cytidine Kinase 2) is a promising prognostic marker for malignant tumors, but its association with immune infiltration and cancer stemness in pan-cancer remains to be fully understood. we find that gene UCK2 is closed related to RNA stemness scores (RNAss) and DNA stemness scores (DNAss), which is measured the tumor stemness. We also discover an association between UCK2 expression and immune cells by CIBERSORT algorithm, ESTIMATE algorithm and ssGSEA algorithm, especially, related to T cell, monocytes, mast cells, and macrophages. This study aims to shed light on the role and possible mechanism of UCK2 in pan-cancer.We used the R programming language for pan-cancer bulk sequencing data analysis, which were obtained from the University of California, Santa Cruz (UCSC) datasets. UCSC database is a very useful for explore data from TCGA and other cancer genomics datasets, The data we explored at the UCK2 transcriptome level came from TCGA data in the UCSC database. We explored differential UCK2 expression between tumor and normal samples. Immunohistochemistry (IHC) was utilized to validate the expression of UCK2 in different types cancers using tumor tissue chips. The correlations of UCK2 with prognosis, genetic instability, DNA repair, cancer stem cell characteristics, and immune cell infiltration were investigated. Furthermore, single-cell datasets, acquired from the Gene Expression Omnibus (GEO) database, were used to validate the relationship between UCK2 and immune cells. GEO is a famous public genomics database supporting freely disseminates microarray data. Finally, we analyzed the correlation between UCK2 and drug sensitivity.UCK2 expression was observed to be high in most cancers and was remarkably related to the prognosis of pan-cancers. We found that the increased UCK2 expression was associated with higher genetic instability. Additionally, positive relationships were observed between UCK2 expression and mismatch repair genes, homologous recombination repair genes, and cancer stemness across different cancer types. There were significant correlations between UCK2 and T cells, monocytes, mast cells, and macrophages. Moreover, as expected, the immune checkpoint human leucocyte antigen (HLA) was found to be negatively related to UCK2. Similarly, UCK2 was also observed to have a negative association with major histocompatibility complex (MHC) genes. We noted that UCK2 had significant correlations with the sensitivity to various anti-cancer drug.We have observed that UCK2 plays pivotal roles in prognosis and tumor immunity, and it is associated with DNA repair and cancer stemness. The UCK2 gene exhibits a strong correlation with the immune checkpoints HLA. This study highlights its potential impact on drug sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3d615de6acd1c023835e11b2fb04b8e2aecd39" target='_blank'>
              Uridine-cytidine kinase 2 is correlated with immune, DNA damage repair and promotion of cancer stemness in pan-cancer
              </a>
            </td>
          <td>
            Jinlong Tian, Yanlei Li, Yu Tong, Yuan Zhang, Tingxiao Zhao, Yao Kang, Qing Bi
          </td>
          <td>2025-01-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite the evolutionary importance of supergenes, their properties in polyploids remain unexplored. Polyploid genomes are expected to undergo chromosomal rearrangements and gene losses over time, potentially affecting supergene architecture. The iconic distyly supergene (S-locus), controlling a floral heteromorphism with two self-incompatible morphs, has been well-documented in diploids, but remains unknown in polyploids. Primula, the classic model for distyly since Darwin, is ancestrally diploid and distylous, yet polyploid, homostylous species with a single, self-compatible floral morph evolved repeatedly. The intraspecific loss of distyly is associated with small loss-of-function mutations in the S-locus CYPT gene controlling style length and female self-incompatibility. Over longer timescales, relaxed selection on CYPT should generate greater accumulation of larger mutations, including exon and gene loss. By analyzing the first assembled genome of an allotetraploid, homostylous species (Primula grandis) in a comparative framework, we discovered two, nearly identical S-locus alleles in the same subgenome, suggesting it originated via inter-specific hybridization between a homostylous and a distylous progenitor. Conformant to predictions from theory, the macroevolutionary loss of distyly coincided with considerable degeneration of CYPT, while other S-locus genes remained largely unaffected, suggesting the shift to homostyly preceded and facilitated polyploid establishment. At the whole-genome level, we found minimal subgenome dominance — as expected, given the inferred recent origin of P. grandis — and highly reduced genetic diversity, congruently with its narrow distribution and self-compatibility. This study provides the first comparison of a supergene across ploidy levels and reproductive systems, contributing new knowledge on the previously unknown fate of supergenes in polyploids. SIGNIFICANCE This study advances knowledge on genome evolution by elucidating how supergenes (clusters of tightly linked genes) evolve across species with different sets of chromosomes and reproductive systems. By analyzing the newly assembled genome of the polyploid, self-compatible Primula grandis in a broad framework, we provide the first comparison of the distyly supergene between diploid outcrossers and polyploid self-fertilizers. We discovered one pair of identical supergene alleles in the same subgenome, rather than one pair per subgenome, revealing the species originated via a cross between a self-compatible and a self-incompatible progenitor. Conformant to theory, the gene controlling female self-incompatibility and style length (CYPT) was considerably degenerated, because of relaxed selection over time, with the rest of the supergene largely unaffected.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd99c78bc08b60ac655953b43bdc71e27d5d4f87" target='_blank'>
              Fate of a supergene in the shift from diploidy to polyploidy
              </a>
            </td>
          <td>
            E. Mora-Carrera, N. Yousefi, G. Potente, R. Stubbs, B. Keller, Étienne Léveillé-Bourret, Grob Stefan, F. Celep, G. Tedoradze, E. Conti
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Structural variations (SVs) play important roles in genetic diversity, evolution, and carcinogenesis and are, as such, important for human health. However, it remains unclear how spatial proximity of double-strand breaks (DSBs) affects the formation of SVs. To investigate if spatial proximity between two DSBs affects DNA repair, we used data from 3C experiments (Hi-C, ChIA-PET, and ChIP-seq) to identify highly interacting loci on six different chromosomes. The target regions correlate with the borders of mega-base sized Topologically Associated Domains (TADs), and we used CRISPR-Cas9 nuclease and pairs of single guide RNAs (sgRNAs) against these targets to generate DSBs in both K562 cells and H9 human embryonic stem cells (hESC). Droplet Digital PCR (ddPCR) was used to quantify the resulting recombination events, and high-throughput sequencing was used to analyze the chimeric junctions created between the two DSBs. We observe a significantly higher formation frequency of deletions and inversions with DSBs in proximity as compared to deletions and inversions with DSBs not in proximity in K562 cells. Additionally, our results suggest that DSB proximity may affect the ligation of chimeric deletion junctions. Taken together, spatial proximity between DSBs is a significant predictor of large-scale deletion and inversion frequency induced by CRISPR-Cas9 in K562 cells. This finding has implications for understanding SVs in the human genome and for the future application of CRISPR-Cas9 in gene editing and the modelling of rare SVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9534f559b29005a3cc7a49a03b73e3b8f75c23b4" target='_blank'>
              Characterization of the role of spatial proximity of DNA double-strand breaks in the formation of CRISPR-Cas9-induced large structural variations.
              </a>
            </td>
          <td>
            Mikkel Dahl-Jessen, T. Terkelsen, R. Bak, Uffe Birk Jensen
          </td>
          <td>2025-01-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Mutations in somatic cells are inflicted by both extrinsic and intrinsic sources and contribute over time to cancer. Tobacco smoke contains chemical carcinogens that have been causatively implicated with cancers of the lung and head & neck1,2. APOBEC family DNA cytosine deaminases have emerged as endogenous sources of mutation in cancer, with hallmark mutational signatures (SBS2/SBS13) that often co-occur in tumors of tobacco smokers with an equally diagnostic mutational signature (SBS4)3,4. Here we challenge the dogma that mutational processes are thought to occur independently and with additive impact by showing that 4-nitroquinoline 1-oxide (NQO), a model carcinogen for tobacco exposure, sensitizes cells to APOBEC3B (A3B) mutagenesis and leads to synergistic increases in both SBS2 mutation loads and oral carcinomas in vivo. NQO-exposed/A3B-expressing animals exhibit twice as many head & neck lesions as carcinogen-exposed wildtype animals. This increase in carcinogenesis is accompanied by a synergistic increase in mutations from APOBEC signature SBS2, but not from NQO signature SBS4. Interestingly, a large proportion of A3B-catalyzed SBS2 mutations occurs as strand-coordinated pairs within 32 nucleotides of each other in transcribed regions, suggesting a mechanism in which removal of NQO-DNA adducts by nucleotide excision repair exposes short single-stranded DNA tracts to enzymatic deamination. These highly enriched pairs of APOBEC signature mutations are termed didyma (Greek for twins) and are mechanistically distinct from other types of clustered mutation (omikli and kataegis). Computational analyses of lung and head & neck tumor genomes show that both APOBEC mutagenesis and didyma are elevated in cancers from smokers compared to non-smokers. APOBEC signature mutations and didyma are also elevated in normal lung tissues in smokers prior to cancer initiation. Collectively, these results indicate that DNA adducting mutagens in tobacco smoke can amplify DNA damage and mutagenesis by endogenous APOBEC enzymes and, more broadly, suggest that mutational mechanisms can interact synergistically in both cancer initiation and promotion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de637cc3bf2890bf07d4603dfc1f19004aa5b7c2" target='_blank'>
              Tobacco smoke carcinogens exacerbate APOBEC mutagenesis and carcinogenesis
              </a>
            </td>
          <td>
            Cameron Durfee, E. N. Bergstrom, M. Díaz-Gay, Yufan Zhou, N. A. Temiz, Mahmoud A. Ibrahim, Shuvro P. Nandi, , Xingyu Liu, Christopher D. Steele, Joshua Proehl, Rachel I Vogel, P. Argyris, Ludmil B. Alexandrov, R. Harris
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The replicative polymerase delta is inefficient copying repetitive DNA sequences. Error-prone translesion polymerases have been shown to switch with high-fidelity replicative polymerases to help navigate repetitive DNA. We and others have demonstrated the importance of one such translesion polymerase, polymerase Eta (pol eta), in facilitating replication at genomic regions called common fragile sites (CFS), which are difficult-to-replicate genomic regions that are hypersensitive to replication stress. However, the mechanistic basis for pol eta’s role in facilitating DNA replication at CFS and(or) at other genomic regions is currently unclear. Importantly, the functional importance of three non-catalytic domains of pol eta, the Ubiquitin-binding Zinc finger (UBZ), PCNA interacting protein (PIP) domain, and the F1 domain which mediates its switch with replicative DNA polymerases in mediating replication stress, especially at CFS loci is not clear. Here, we report that the PIP and UBZ domains of Pol Eta are both critical for its role in mediating cellular replication stress, especially at CFS. The absence of either domain induced elevated replication stress, replication stalling and DNA damage accumulation genome wide. This effect was even more pronounced at CFS loci leading to the accumulation of under replication DNA in G2/M. Importantly, while the inactivation of the UBZ domain resulted in a robust FANCD2 monoubiquitylation (a prominent marker of FANCD2 activation), FANCD2 recruitment genome wide was significantly impacted, especially at CFSs such as FRA16D. These S-phase phenotypes result in ssDNA gap formation and the persistence of under-replicated genomic regions upon transition to G2/M. While post-replicative gap filing/ repair by Mitotic DNA synthesis is activated in the mutants, it only effectively resolves UFBs in the F1* cells. The PIP*, UBZ* and pol eta−/− cells unfortunately manifest excessive toxic cytosolic DNA that instigates a strong innate immune response. These results collectively show that translesion polymerase Eta functions in a common pathway with FANCD2 to prevent replication perturbation and instability at CFS loci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5573d7888e9977cc24f9b4600bf351154f863de7" target='_blank'>
              Polymerase Eta Recruits FANCD2 to Common Fragile Sites to Maintain Genome Stability
              </a>
            </td>
          <td>
            Mrunmai Niljikar, Angelica Barreto-Galvez, Saloni Patel, J. Gagliardi, Vasudha Kumar, Archana Pradeep, Aastha Juwarwala, Jeannine Gerhardt, , Cristina Montagna, Advaitha Madireddy
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="A central goal of cancer genomics is to identify, in each patient, all the cancer-driving mutations. Among them, point mutations are referred to as cancer-driving nucleotides (CDNs), which recur in cancers. The companion study shows that the probability of i recurrent hits in n patients would decrease exponentially with i; hence, any mutation with i ≥ 3 hits in The Cancer Genome Atlas (TCGA) database is a high-probability CDN. This study characterizes the 50–150 CDNs identifiable for each cancer type of TCGA (while anticipating 10 times more undiscovered ones) as follows: (i) CDNs tend to code for amino acids of divergent chemical properties. (ii) At the genic level, far more CDNs (more than fivefold) fall on noncanonical than canonical cancer-driving genes (CDGs). Most undiscovered CDNs are expected to be on unknown CDGs. (iii) CDNs tend to be more widely shared among cancer types than canonical CDGs, mainly because of the higher resolution at the nucleotide than the whole-gene level. (iv) Most important, among the 50–100 coding region mutations carried by a cancer patient, 5–8 CDNs are expected but only 0–2 CDNs have been identified at present. This low level of identification has hampered functional test and gene-targeted therapy. We show that, by expanding the sample size to 105, most CDNs can be identified. Full CDN identification will then facilitate the design of patient-specific targeting against multiple CDN-harboring genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb003947f43738ae589b3a6538f7d27fb518829" target='_blank'>
              Characterization of cancer-driving nucleotides (CDNs) across genes, cancer types, and patients
              </a>
            </td>
          <td>
            Lingjie Zhang, Tong Deng, Zhongqi Liufu, Xiangnyu Chen, Shijie Wu, Xueyu Liu, Changhao Shi, Bingjie Chen, Zheng Hu, Qichun Cai, Chenli Liu, Mengfeng Li, Miles E. Tracy, Xuemei Lu, Chung-I Wu, Hai-Jun Wen
          </td>
          <td>2024-12-17</td>
          <td>eLife</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5730f3b1a7e75be7288a46c1b4d8d857387d1346" target='_blank'>
              Positive Selection Shapes Breast Cancer Tumor Suppressor Genes: Unveiling Insights into BRCA1, BRCA2, and MDC1 Stability.
              </a>
            </td>
          <td>
            Youssef M. Fadel, Marwan Khaled, Mohamed Emam, , Sief El-Din Sobih, Habiba Abd-Elaty, Wafaa M Elrashedy, Gehad Mostafa, Salma Alm Eldeen, Mohaned Bador, Agostinho Antunes, Mohamed El Hadidi
          </td>
          <td>2024-12-16</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aede480e99c41ff823fe07aee10fb1b86fa8b" target='_blank'>
              DNA methylation-based analysis reveals accelerated epigenetic aging in giant cell-enriched adult-type glioblastoma
              </a>
            </td>
          <td>
            P. Cakmak, Philipp Jurmeister, I. Divé, P. Zeiner, Joachim P. Steinbach, Tim R Fenton, K. H. Plate, M. Czabanka, Patrick Harter, 
          </td>
          <td>2024-12-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, , E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Here, we have discussed the molecular mechanisms of p53-responsive microRNAs dysregulation in response to genotoxic stress in diffuse large B-cell lymphoma (DLBCL) patients. The role of micro ribonucleic acids (microRNAs) in p53-signaling cellular stress has been studied. MicroRNAs are the small non-coding RNAs, which regulate genes expression at post-transcriptional level. Many of them play a crucial role in carcinogenesis and may act as oncogenes or suppressor of tumor growth. The understanding of the effect of p53-responsive microRNA dysregulation on oncogenesis achieved in recent decades opens wide opportunities for the diagnosis, prediction and of microRNA-based cancer therapy. Development of new bioinformatics tools and databases for microRNA supports DLBCL research. We overview the studies on the role of miRNAs in regulating gene expression associated with tumorigenesis processes, with particular emphasis on their role as tumor growth-suppressing factors. The starting point is a brief description of the classical microRNA biogenesis pathway and the role of p53 in regulating the expression of these molecules. We analyze various molecular mechanisms leading to this dysregulation, including mutations in the TP53 gene, DNA methylation, changes in host-genes expression or microRNA gene copy number, mutations in microRNA and microRNA biogenesis genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740d5b189850125f69ad77979cf0d447a7b59eef" target='_blank'>
              Deregulation mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            E. Voropaeva, Yuriy L. Orlov, Anastasia B. Loginova, O. Seregina, Vladimir N. Maksimov, T. I. Pospelova
          </td>
          <td>2025-01-07</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) patients with microsatellite-stable (MSS) tumors are mostly treated with chemotherapy. Clinical benefits of targeted therapies depend on mutational states and tumor location. Many tumors carry mutations in KRAS proto-oncogene, GTPase (KRAS) or B-Raf proto-oncogene, serine/threonine kinase (BRAF), rendering them more resistant to therapies. We performed whole-exome sequencing and RNA-Sequencing of 28 tumors of the Athens Comprehensive Cancer Center CRC cohort, and molecularly characterized CRC patients based on their microsatellite instability (MSI) status, single-nucleotide variations (SNVs)/copy number alterations (CNAs), and pathway/transcription factor activities at the individual patient level. Variants were classified using a computational score for integrative cancer variant annotation and prioritization. Complementing this with public multi-omics datasets, we identified activation of transforming growth factor beta (TGFβ) signaling to be more strongly activated in MSS patients, whereas Janus kinase (JAK)-signal transducer and activator of transcription (STAT) and mitogen-activated protein kinase (MAPK) molecular cascades were activated specifically in MSI tumors. We unraveled mechanisms consistently perturbed in the transcriptional and mutational circuits and identified Runt-related transcription factors (RUNX transcription factors) as putative biomarkers in CRC, given their role in the regulation of pathways involved in tumor progression and immune evasion. Assessing the immunogenicity of CRC tumors in the context of RAS/RAF mutations and MSI/MSS status revealed a critical impact that KRAS mutations have on immunogenicity, particularly in the MSS patient subgroup, with implications for diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74b1bcf5fa54757058aca72542a0424e2f6a831" target='_blank'>
              Molecular and functional profiling unravels targetable vulnerabilities in colorectal cancer.
              </a>
            </td>
          <td>
            E. Vlachavas, Konstantinos Voutetakis, Vivian Kosmidou, Spyridon Tsikalakis, Spyridon Roditis, Konstantinos Pateas, Ryangguk Kim, Kymberleigh Pagel, Stephan Wolf, , Antonia Dimitrakopoulou-Strauss, Georgios N Zografos, Alexander Pintzas, J. Betge, O. Papadodima, Stefan Wiemann
          </td>
          <td>2025-01-28</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Dysregulated epigenetic programs that restrict differentiation, reactivate fetal genes, and confer phenotypic plasticity are critical to colorectal cancer (CRC) development. By screening a small molecule library targeting epigenetic regulators using our dual reporter system, we found that inhibiting histone deacetylase (HDAC) 1/2 promotes CRC differentiation and anti-tumor activity. Comprehensive biochemical, chemical, and genetic experiments revealed that on-target blockade of the HDAC1/2 catalytic domain mediated the differentiated phenotype. Unbiased profiling of histone posttranslational modifications induced by HDAC1/2 inhibition nominated acetylation of specific histone lysine residues as potential regulators of differentiation. Genome-wide assessment of implicated marks indicated that H3K27ac gains at HDAC1/2-bound regions associated with open chromatin and upregulation of differentiation genes upon HDAC1/2 inhibition. Disrupting H3K27ac by degrading acetyltransferase EP300 rescued HDAC1/2 inhibitor-mediated differentiation of a patient-derived CRC model using single cell RNA-sequencing. Genetic screens revealed that DAPK3 contributes to CRC differentiation induced by HDAC1/2 inhibition. These results highlight the importance of specific chemically targetable histone modifications in governing cancer cell states and epigenetic reprogramming as a therapeutic strategy in CRC. BRIEF SUMMARY HDAC1/2 inhibition promotes colorectal cancer differentiation via gains in H3K27ac, which can be reversed by blocking its acetyltransferase EP300.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207eb16a60f11154e81c963033e896c8385daaa7" target='_blank'>
              Chemical perturbations impacting histone acetylation govern colorectal cancer differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Akhouri Kishore Raghawan, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, P. Sahgal, Daulet Aitymbayev, Jennifer S. Thalappillil, Malvina Papanastasiou, William Hawkins, Steven A. Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Chromosomes and their constituent compartments are core elements of the cell division machinery. In mammalian oocytes, defects in the structure or composition of chromosomes are a leading cause of aberrant or failed meiosis, and by extension, can cause miscarriage and infertility. The underlying mechanisms are poorly understood, but are critical for development of novel diagnostics and treatments. The chromosome periphery, the least understood chromosome compartment, has recently emerged as an essential component of mitotic chromosomes and somatic cell division. However, in female meiosis it remains completely unexplored. This study provides the first comprehensive survey of a meiotic chromosome periphery compartment in both human and mouse oocytes. Using a combination of time-lapse microscopy, super-resolution imaging and 3DCLEM we show that removing the chromosome periphery, via Ki67 depletion, has substantial negative impact on chromosome structure, spatial organization and positional awareness, with many oocytes stalling and arresting in meiosis I. Importantly, we also reveal key differences between the mitotic and oocyte meiotic chromosome periphery compartments, most remarkably in the retention of Ki67 in the oocyte through anaphase I, where unwanted chromosomes are stripped of Ki67 before being ejected from the oocyte. This work presents the discovery of an exciting new pathway operating during female meiosis and a provides a platform for future work exploring the meiotic chromosome periphery for therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d4e852fd562f1b6718584414157f534cabd865" target='_blank'>
              The Chromosome Periphery is an Essential Compartment of Oocyte Chromosomes
              </a>
            </td>
          <td>
            Eva L Simpson, B. Wetherall, A. Byrska, Liam P. Cheeseman, Tania Mendonca, , Alison J. Beckett, H. Maiato, Alexandra Sarginson, Ian A. Prior, Geraldine M Hartshorne, A. McAinsh, S. Madgwick, Daniel G. Booth
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Pseudouridine (Ψ) is a post-transcriptional modifier of RNA, often referred to as the ‘fifth nucleotide’ owing to its regulatory role in various biological functions as well as because of its significant involvement in the pathogenesis of human cancer. In recent years, research has revealed various Ψ modifications in different RNA types, including messenger RNA, transfer RNA, ribosomal RNA, small nuclear RNA, and long noncoding RNA. Pseudouridylation can significantly alter RNA structure and thermodynamic stability, as the Ψ-adenine (A) base pair is more stable than the typical uridine (U)-A base pair is due to its structural similarity to adenine. Studies have linked Ψ expression to the development and progression of several digestive system cancers, such as liver cancer and colorectal cancer, and nondigestive system cancers, such as breast cancer, non-small cell lung cancer, prostate cancer, glioblastoma, ovarian cancer, oral squamous cell carcinoma, and pituitary cancer. The present review briefly outlines the chemical structure, synthesis, and regulatory mechanisms of Ψ. This review summarizes the effects of pseudouridylation on various substrates of RNA and briefly discusses methods for detecting Ψ. Last, it focuses on how RNA pseudouridylation influences different cancers, emphasizing the search for novel approaches to cancer diagnosis, treatment, and prognosis through Ψ modification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9466c719e8cf6ff18a146061c808211210df53e3" target='_blank'>
              Advancements in pseudouridine modifying enzyme and cancer
              </a>
            </td>
          <td>
            Kaijie Liu, Shujun Zhang, Yafeng Liu, Xinjun Hu, Xinyu Gu
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c108f9b6c451482458b3d22e6cbf9ba4232a44ff" target='_blank'>
              Repeated ionizing radiation exposure induces TRIP13 expression, conferring radioresistance in lung cancer cells
              </a>
            </td>
          <td>
            Wenqing Liu, Qijing Lei, A. V. van Pelt, G. Hamer
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b3f437af02a37eaf52895dfc3343570b14c40b" target='_blank'>
              The role of Box A of HMGB1 in producing γH2AX associated DNA breaks in lung cancer
              </a>
            </td>
          <td>
            Sirapat Settayanon, P. Chanvorachote, Apiwat Mutirangura
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Disturbances in DNA damage repair may lead to cancer. SIRT1, an NAD+-dependent deacetylase, plays a crucial role in maintaining cellular homeostasis through the regulation of processes such as histone posttranslational modifications, DNA repair, and cellular metabolism. However, a comprehensive exploration of SIRT1’s involvement in pan-cancer remains lacking. Our study aimed to analyze the role of SIRT1 in pan-cancer to gain a more comprehensive understanding of its role in multiple malignancies. Methods We systematically examined the role of SIRT1 in pan-cancer by analyzing data from The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) databases. Various tools, including R, Cytoscape, HPA, Archs4, TISIDB, cBioPortal, STRING, GSCALite, and CancerSEA, were used to integrate and analyze SIRT1 gene expression, prognosis, protein interactions, signaling pathways, immune infiltration, and other relevant information. Furthermore, we validated the differential expression of SIRT1 in normal human kidney cells and kidney cancer cell lines via experimental verification. Results SIRT1 expression was significantly reduced in various cancers and was different across molecular and immune subtypes. SIRT1 is intricately linked to numerous cancer pathways. In most cancer types, increased SIRT1 expression is positively associated with eosinophils, helper T cells, central memory T cells, effector memory T cells, γδ T cells, and Th2 cells. SIRT1 expression is significantly correlated with immune regulatory factors across various cancer types. Quantitative reverse transcription polymerase chain reaction (qRT–PCR) and Western blot (WB) analyses confirmed that SIRT1 is differentially expressed in kidney renal clear cell carcinoma (KIRC). Conclusions Using an integrative approach involving bioinformatics analysis and experimental validation, we clarified the potential roles and mechanisms of SIRT1 in pan-cancer, providing a theoretical basis for the development of SIRT1-targeted therapies in clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d206cdebba52a7d638c5e85ad74b73761518e836" target='_blank'>
              Comprehensive analysis of the prognostic, immunological, and diagnostic roles of SIRT1 in pan-cancer and its validation in KIRC
              </a>
            </td>
          <td>
            Qi Liu, Songxian Sun, Chunxiang Zhou, Houxi Xu
          </td>
          <td>2025-01-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="With the practical implementation of the CRISPR/Cas technology for targeted genome editing, it has become possible to carry out genetic engineering manipulations with eukaryotic genomes with high efficiency. One of the key stages of this technology is the targeted induction of site-specific DNA cleavages (breaks). The cell repairs these breaks via one of two pathways: nonhomologous end joining or homologous recombination. The choice of DNA repair pathway is determined by the architecture of the sites at the DNA break area formed as a result of terminal resection and depends on the phases of the cell cycle. Nonhomologous end joining is the main pathway for repair of double-stranded DNA breaks in mammalian cells. It involves a nonspecific ligation reaction, the accuracy of which depends on the structure of the ends of the break, and can result in various insertions or deletions in the target region of the genome. Integration of the desired sequence into the genome occurs along the path of homologous recombination, the implementation of which requires a matrix with homology regions on both sides of the double-strand break. The introduction of a genetic construct into a given location in the genome is an important, but currently complex and labor-intensive task. At the same time, for fundamental studies of gene function and the creation of animal models of human diseases, the choice of the repair pathway can be of fundamental importance. This review is an attempt to combine and structure all known information on approaches to increasing the efficiency of DNA repair involving homologous recombination. The article lists the most effective strategies to shift the balance towards homologous repair, such as the use of inhibitors of the non-homologous end joining mechanism, regulation of key factors of homologous recombination, control of the cell cycle, chromatin status, construction of templates for homologous recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23816bdc012e17808d654e03c85cd07d45917c31" target='_blank'>
              How to shift the equilibrium of dna break repair in favor of homology recombination
              </a>
            </td>
          <td>
            O. A. Averina, S. A. Kuznetsova, O. Permyakov, P. V. Sergiev
          </td>
          <td>2024-12-30</td>
          <td>Molekulârnaâ biologiâ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df367ac3cfcfbd1960fe9e19b1423ebb8f57c2e" target='_blank'>
              Pan-cancer analysis and single-cell analysis identifies the CENPN as a biomarker for survival prognosis and immunotherapy
              </a>
            </td>
          <td>
            Nie Zhang, Zhuoying He, Xuejin Qin, Ke Han, Zhengchun Zhu, Fei Zhong
          </td>
          <td>2025-01-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transposable elements (TEs) are significant drivers of genome evolution, yet their recent dynamics and impacts within and among species, as well as the roles of host genes and non-coding RNAs in the transposition process, remain elusive. With advancements in large-scale pan-genome sequencing and the development of open data sharing, large-scale comparative genomics studies have become feasible. Here, we performed complete de novo TE annotations and identified active TEs in 310 plant genome assemblies across 119 species and seven crop populations. Using 811 high-quality genomes, we detected 13 844 553 TE-induced structural variants (TE-SVs), providing unprecedented resolution in delineating recent TE activities. Our integrative analysis revealed a mutual evolutionary relationship between TEs and host genomes. On one hand, host genes and ncRNAs are involved in the transposition process, as evidenced by their colocalization and coactivation with TEs, and may play a role in chromatin regulation. On the other hand, TEs drive genetic innovation by promoting the duplication of host genes and inserting into regulatory regions. Moreover, genes influenced by active TEs are linked to plant growth, nutrient absorption, storage metabolism and environmental adaptation, aiding in crop domestication and adaptation. This TE dynamics atlas not only reveals evolutionary and functional features linked to transposition activity but also highlights the role of TEs in crop domestication and adaptation, paving the way for future exploration of TE-mediated genome evolution and crop improvement strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e3e6dd3ee1f3baea529da4c3c4ab4bf7659f210" target='_blank'>
              Deciphering recent transposition patterns in plants through comparison of 811 genome assemblies.
              </a>
            </td>
          <td>
            Yan Huang, Sunil Kumar Sahu, Xin Liu
          </td>
          <td>2025-01-10</td>
          <td>Plant biotechnology journal</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be789fb687d3f78efa3b5343344eaabbca303d35" target='_blank'>
              Chromatin licensing and DNA replication factor 1 as a potential prognostic and diagnostic biomarker for gastric cancer
              </a>
            </td>
          <td>
            Sijia Wang, Min Cui, Na Zhou, Meichen Tong, Shuhua Wu, Hong Li, Zhang Cao
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="ABSTRACT Gene editing is emerging as a powerful tool for introducing novel functionalities in mushrooms. While CRISPR/Cas9-induced double-strand breaks (DSBs) typically rely on non-homologous end joining (NHEJ) for gene disruption, precise insertion of heterologous DNA in mushrooms is less explored. Here, we evaluated the efficacy of inserting donor DNAs (8–1008 bp) with or without homologous arms at Cas9-gRNA RNP-induced DSBs. Co-transformation of donor DNAs with RNP targeting the pyrG gene in Ganoderma lucidum yielded 184 transformants without homologous arms and 781 with 300-bp homologous arms (HR_donor DNAs). Restriction analysis and sequencing identified 122 hR_donor DNA transformants with complete donor DNA sequences, achieving 15.6% HDR efficiency (122/781), contrasting with 8 instances via NHEJ from the 184 transformants. These findings highlight the viability of HDR for precise genomic editing in mushrooms, enabling targeted modifications to enhance functionalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b87a3000ba3fba65c04d201ba2965fcf2923390d" target='_blank'>
              Targeted insertion of heterogenous DNA using Cas9-gRNA ribonucleoprotein-mediated gene editing in Ganoderma lucidum
              </a>
            </td>
          <td>
            Hye-Hyun Eom, Yeon-Jae Choi, R. Nandre, Minseek Kim, , Sinil Kim, T. Nakazawa, Yoichi Honda, H. Ro
          </td>
          <td>2025-01-29</td>
          <td>Bioengineered</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The cohesin complex is a critical regulator of gene expression. STAG2 is the most frequently mutated cohesin subunit across several cancer types and is a key tumor suppressor in lung cancer. Here, we coupled somatic CRISPR-Cas9 genome editing and tumor barcoding with an autochthonous oncogenic KRAS-driven lung cancer model and showed that STAG2 is uniquely tumor-suppressive among all core and auxiliary cohesin components. The heterodimeric complex components PAXIP1 and PAGR1 have highly correlated effects with STAG2 in human lung cancer cell lines, are tumor suppressors in vivo, and are epistatic to STAG2 in oncogenic KRAS-driven lung tumorigenesis in vivo. STAG2 inactivation elicits changes in gene expression, chromatin accessibility, and 3D genome conformation that impact the cancer cell state. Gene expression and chromatin accessibility similarities between STAG2- and PAXIP1-deficient neoplastic cells further relate STAG2-cohesin to PAXIP1/PAGR1. These findings reveal a STAG2-PAXIP1/PAGR1 tumor-suppressive axis and uncover novel PAXIP1-dependent and PAXIP1-independent STAG2-cohesin-mediated mechanisms of lung tumor suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77c9e3bd051e20d8c03518cc31ea52a29e0f1f0" target='_blank'>
              A STAG2-PAXIP1/PAGR1 axis suppresses lung tumorigenesis.
              </a>
            </td>
          <td>
            Emily L Ashkin, Yuning J Tang, Haiqing Xu, King L. Hung, J. Belk, Hongchen Cai, Steven S. Lopez, D. N. Dolcen, Jess D. Hebert, Rui Li, Paloma A. Ruiz, Tula Keal, Laura Andrejka, Howard Y. Chang, D. Petrov, Jesse R. Dixon, Zhichao Xu, M. Winslow
          </td>
          <td>2024-12-09</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Since the discovery of the Musashi (MSI) protein, its ability to affect the mitosis of Drosophila progenitor cells has garnered significant interest among scientists. In the following 20 years, it has lived up to expectations. A substantial body of evidence has demonstrated that it is closely related to the development, metastasis, migration, and drug resistance of malignant tumors. In recent years, research on the MSI protein has advanced, and many novel viewpoints and drug resistance attempts have been derived; for example, tumor protein p53 mutations and MSI-binding proteins lead to resistance to protein arginine N-methyltransferase 5-targeted therapy in lymphoma patients. Moreover, the high expression of MSI2 in pancreatic cancer might suppress its development and progression. As a significant member of the MSI family, MSI2 is closely associated with multiple malignant tumors, including hematological disorders, common abdominal tumors, and other tumor types (e.g., glioblastoma, breast cancer). MSI2 is highly expressed in the majority of tumors and is related to a poor disease prognosis. However, its specific expression levels and regulatory mechanisms may differ based on the tumor type. This review summarizes the research progress related to MSI2 in recent years, including its occurrence, migration mechanism, and drug resistance, as well as the prospect of developing tumor immunosuppressants and biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dd0dc9ec8f8cc8ab893bc32f47ba299917d963" target='_blank'>
              Update on the Progress of Musashi-2 in Malignant Tumors.
              </a>
            </td>
          <td>
            Yiting Niu, Tao Zhou, Yanjun Li
          </td>
          <td>2025-01-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene conversion is a process in which genetic material from a donor sequence is unidirectionally copied to an acceptor sequence during the homologous recombination repair of a DNA double-strand break. Although gene conversion has been widely studied in the context of meiosis, hereditary diseases, and cancer development, gene conversion between parental homologs in the zygotes remains controversial. Here, we developed a method to detect interparental gene conversions by focusing on Mendelian errors and identified gene conversion events in one out of every 21.8 births. Some of these events were observed in genetic regions, potentially affecting offspring phenotypes. Interparental gene conversion leads to the offspring inheriting two identical alleles from one parent, resulting in a loss of heterozygosity. Our findings suggest that naturally occurring interparental gene conversions may provide a novel mechanism for the development of certain genetic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ddbca341c42c36a2e20339125b0cf1e5d09505" target='_blank'>
              Interparental Gene Conversion in General Population: A Novel Mechanism For Loss of Heterozygosity
              </a>
            </td>
          <td>
            Jumpei Toratani, Masahito Tachibana, Junichi Sugawara, , Takeki Sato, Yuri Takahashi, Hiroaki Hiraga, Emi Yokoyama, Zen Watanabe, Masatoshi Saito, Nobuo Yaegashi, Gen Tamiya, J. Takayama
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The natural ends of chromosomes resemble double-strand breaks (DSBs), which would activate the DNA damage response (DDR) pathway without the protection provided by a specialized protein complex called shelterin. Over the past decades, extensive research has uncovered the mechanism of action and the high degree of specialization provided by the shelterin complex to prevent aberrant activation of DNA repair machinery at chromosome ends in somatic cells. However, recent findings have revealed striking differences in the mechanisms of end protection in stem cells compared to somatic cells. In this review, we discuss what is known about the differences between stem cells and somatic cells regarding chromosome end protection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64f88771a224944498f40c132aea82d1a6ba4a" target='_blank'>
              Telomere Protection in Stem Cells.
              </a>
            </td>
          <td>
            Marta Markiewicz-Potoczny, E. L. Denchi
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The altered expression of genes encoding histone acetyltransferases (HATs) and histone deacetylases (HDACs) has been implicated in the tumorigenesis and progression of various solid tumors. However, systematic characterization of the transcriptomic landscape and clinical relevance of HATs and HDACs in pan-cancer contexts remains lacking.Transcriptome and clinical data of 9,483 patients across 31 tumor types from The Cancer Genome Atlas were collected for systematic pan-cancer analysis. Additional glioma-specific datasets (Chinese Glioma Genome Atlas, GlioVis, GSE43378, and GSE182109) were also collected to validate the transcriptional characteristics of HATs and HDACs in gliomas. Consensus clustering analysis was applied to identify distinct expression patterns of HATs and HDACs.Based on the transcriptomic data of 25 genes encoding 9 HATs and 16 HDACs, we identified five major subtypes across 31 cancer types (AC-I to AC-V). Notably, the AC-V subtype comprised over 95% of glioma patients, suggesting glioma patients exhibited distinct expression patterns of histone acetylation-modifying enzymes compared to patients with other solid tumors. Therefore, we re-conducted the consensus clustering analysis specifically within the context of gliomas and identified five subtypes, denoted “AC-GI” to “AC-GV”, which were characterized by differences in HATs/HDACs expression patterns, biological and immune status, genetic alterations, and clinical outcomes. The AC-GII patients exhibited the best prognosis and were sensitive to temozolomide, while AC-GV patients had the poorest prognosis and the lowest sensitivity to temozolomide among all subtypes. Moreover, based on the Connectivity Map database analysis and experimental verification, we identified several pan-HDAC inhibitors that could serve as sensitizers for temozolomide therapy in AC-GV patients, such as panobinostat and scriptaid. Considering the distinctive clinical characteristics of patients with AC-GII and AC-GV, we constructed the “ACG score” model capable of effectively recognizing patients with these subtypes and predicting patient prognosis.Herein, we established novel biologically and clinically relevant molecular classifications for pan-solid tumors and gliomas based on transcriptional expression profiles of HATs and HDACs. Moreover, the ACG score model, calculated by the transcriptional expression of 29 genes, was not only an independent prognostic factor for glioma patients, but can also provide valuable references for promoting more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9506518bda05def7ccbdb74a1bb450d215aa14aa" target='_blank'>
              Pan-cancer analysis of the transcriptional expression of histone acetylation enzymes in solid tumors defines a new classification scheme for gliomas
              </a>
            </td>
          <td>
            Junhao Zhang, Lingbo Li, Aiwei Tang, Chucheng Wang, Yupeng Wang, Yongqi Hu, Guangting He, Wangjun Liao, Rui Zhou
          </td>
          <td>2025-01-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, 
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Background SET domain‐containing protein 4 (SETD4) is a histone methyltransferase that has been shown to modulate cell proliferation, differentiation, and inflammatory responses by regulating histone H4 trimethylation (H4K20me3). Previous reports have demonstrated its function in the quiescence of cancer stem cells as well as drug resistance in several cancers. A limited number of systematic studies have examined SETD4's role in the tumor microenvironment, pathogenesis, prognosis, and therapeutic response. Methods Utilizing The Cancer Genome Atlas database, and other publicly accessible platforms, we comprehensively analyzed SETD4 gene expression, methylation patterns, and prognostic significance. Furthermore, we investigated its association with cancer‐related pathways, the immune microenvironment, immunotherapy markers, and drug resistance signatures of chemotherapy. Additionally, qRT‐PCR was performed to validate SETD4 expression in clinical specimens. Results The expression of SETD4 was abnormal across a variety of cancer types and the expression of SETD4 in colorectal cancer tissues was verified in clinical specimens. The upregulation of SETD4 may be a prognostic risk factor predicting poor overall survival and progression‐free survival. The analysis revealed that the mRNA level of SETD4 was modulated by promoter methylation, and patients with lower methylation levels showed shorter survival times. Pathway analysis showed that SETD4 influenced several key cell cycle pathways, including the G2M checkpoint, and mitotic spindle pathways. In addition, SETD4 negatively affects immune cell infiltration in most cancers, including B cells, CD8 T cells, and macrophages. The correlation between SETD4 and cancer stemness as well as homologous recombination deficiency varied across tumor types, suggesting that SETD4 may play a multifaceted role in tumor resistance. Notably, we identified several potential agents targeting SETD4. Conclusions This study demonstrates that SETD4 is an immune‐oncogenic molecule in multiple cancers, with the potential to be a diagnosis, prognosis, and targeted therapy marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f76f7155098d30f047c5ee80fc1fcf8c03b367a" target='_blank'>
              Identification of SETD4 as an Onco‐Immunological Biomarker Encompassing the Tumor Microenvironment, Prognoses, and Therapeutic Responses in Various Human Cancers
              </a>
            </td>
          <td>
            Yuyun Zhong, Ruiqi Wang, Zijie Huang, Zhaoting Hu, Bin Peng, Bin Chen, 
          </td>
          <td>2025-01-01</td>
          <td>Immunity, Inflammation and Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The Chromosome-Centric Human Proteome Project (C-HPP) is an international initiative. It aims to create a protein list expressed in human cells by each chromosomal and mitochondrial DNA to enhance our understanding of disease mechanisms, akin to the gene list generated by the Human Genome Project. Transmembrane protein 160 (TMEM160) is a member of the transmembrane proteins (TMEM) family. TMEM proteins have been implicated in cancer-related processes, including cell proliferation, migration, epithelial-mesenchymal transition, metastasis, and resistance to chemotherapy and radiotherapy. This study aimed to investigate the role of TMEM160 in non-small cell lung cancer and cervical cancer using cell lines, clinical samples, and xenograft studies. Our findings demonstrated that TMEM160 knockdown decreased the proliferation of lung and cervical cancer cell lines. We observed that TMEM160 is localized in the nucleus and cytoplasm and dynamic localization during mitosis of cancer cells and discovered a novel interaction between TMEM160 and nuclear proteins such as NUP50. Furthermore, the TMEM160 interactome was enriched in processes associated with apical junctions, xenobiotic metabolism, glycolysis, epithelial-mesenchymal transition, reactive oxygen species, UV response DNA, the P53 pathway, and the mitotic spindle. This study provides an initial understanding of the function of TMEM160 in lung and cervical cancer progression and clarifies the need to continue investigating the participation of TMEM160 in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e13c463e1e4f1ddbb09b31ff1b7171d0bdfe61c" target='_blank'>
              TMEM160 Promotes Tumor Growth in Lung Adenocarcinoma and Cervical Adenocarcinoma Cell Lines
              </a>
            </td>
          <td>
            Gloria Angelina Herrera-Quiterio, H. A. Valencia-González, K. G. de la Cruz-López, D. L. Fernández-Coto, Jeovanis Gil, Gyorgy Marko-Varga, Josué Morales-Gálvez, Nilda C. Sánchez, Rubén Rodríguez-Bautista, A. Avilés-Salas, Oscar Arrieta, Alejandro García-Carrancá, S. Encarnación-Guevara
          </td>
          <td>2025-01-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, , Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Prostate cancer is one of the most prevalent malignant tumors in men, particularly in regions with a high Human Development Index. While the long-term survival rate for localized prostate cancer is relatively high, the mortality rate remains significantly elevated once the disease progresses to advanced stages, even with various intensive treatment modalities. The primary obstacle to curing advanced prostate cancer is the absence of comprehensive treatment strategies that effectively target the highly heterogeneous tumors at both genetic and molecular levels. Prostate cancer development is a complex, multigenic, and multistep process that involves numerous gene mutations, alteration in gene expression, and changes in signaling pathways. Key genetic and pathway alterations include the amplification and/or mutation of the androgen receptor, the loss of Rb, PTEN, and p53, the activation of the WNT signaling pathway, and the amplification of the MYC oncogene. This review summarizes the mechanisms by which these genes influence the progression of prostate cancer and highlights the interactions between multiple genes and their relationship with prostate cancer. Additionally, we reviewed the current state of treatments targeting these genes and signaling pathways, providing a comprehensive overview of therapeutic approaches in the context of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fab9adcd074d173268fdd423c799e2e23c962b" target='_blank'>
              Interactions between key genes and pathways in prostate cancer progression and therapy resistance
              </a>
            </td>
          <td>
            Fan Wu, Heng Zhang, Miaomiao Hao
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colon cancer is the second most common cause of cancer death worldwide. Despite advances in the development of new molecular strategies for stratifying patients with colon cancer, many of these patients do not respond adequately to the standard of care. While previous studies have focused on the development of prognostic gene expression signatures, the exploration of predictive signatures to inform treatment decisions remains incomplete. In this study, we leveraged public gene expression datasets to design and experimentally validate a 37-gene expression signature for prognosis in colon cancer patients. We obtained a C-index of 0.732 (0.610-0.853) in four independent studies. Specifically, we discovered that the signature is associated with the mitotic phase of the cell cycle. Furthermore, the signature identified a population of colon cancer patients sensitive to tubulin inhibitor drugs. In particular, we validated in vitro and in vivo the efficacy of paclitaxel, a commonly used tubulin inhibitor in breast cancer treatment, in patient-derived preclinical models. These results highlight the importance of incorporating gene expression signatures to identify new therapeutic options for colon cancer treatment. Furthermore, the identification of alternative treatment options with potentially improved efficacy holds promise for the development of new clinical trials, and reshapes the biomarker-based treatment strategy for second line and refractory colon cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec4c73cdc7ad79204cf105e9ddd7970b91e50e10" target='_blank'>
              Transcriptomic signature reveals sensitivity to tubulin inhibitors in colon cancer
              </a>
            </td>
          <td>
            J. Liñares-Blanco, I. Chicote, Javier Ros, Anna M Alcántara, Jordi Martínez-Quintanilla, E. Élez, Héctor G Palmer, Carlos Fernandez-Lozano, Jose A Seoane
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc8fc819ffccd10115ccb040d51408d1fd6a5f7" target='_blank'>
              Nanopore sequencing reveals that DNA replication compartmentalisation dictates genome stability and instability in Trypanosoma brucei
              </a>
            </td>
          <td>
            M. Krasiļņikova, Catarina A Marques, E. Briggs, C. Lapsley, Graham Hamilton, Dario Beraldi, K. Crouch, R. McCulloch
          </td>
          <td>2025-01-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mutations in the NPM1 gene represent the most common (>30% of patients) genetic alteration in Acute Myeloid Leukaemia (AML) and results in the mis-localisation of the mutated NPM1 protein from a predominantly nucleolar localisation to a predominantly cytoplasmic distribution. Numerous studies of NPM1 mutated AML have focussed on the aberrant cytoplasmic localisation of the mutated protein but efforts to reverse this mis-localisation therapeutically have so far resulted in limited clinical benefit. More recently, attention has shifted towards the nucleus with studies showing that mutant NPM1 binds to specific chromatin regions, where it directly regulates oncogenic gene expression. Here, we use high resolution imaging to demonstrate that Nucleophosmin (NPM1) is critical for maintaining normal nucleoli architecture and specifically the integrity of the nucleoli rim. We report for the first time that NPM1 mutated cell lines and primary samples have aberrant nucleoli architecture and demonstrate that the abnormal nucleoli phenotype is reversible. We also report the novel finding that NPM1 mutated protein forms distinct aggregates in NPM1 mutated cells and characterise these for the first time. This work reveals how nucleolar organisation contributes to the molecular mechanisms underpinning NPM1 driven AML and reveals unexpected novel vulnerabilities to be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b14a9c2136580651e7ba63ac9928b96ae556480" target='_blank'>
              Nucleophosmin mutations lead to abnormal, but reversible, nucleoli architecture and aggregate formation – Implications for NPM1-targeting therapies in AML
              </a>
            </td>
          <td>
            Martin Grundy, Kellie Lucken, , Eva L Simpson, Ahmed Bayyoomi, Alison J. Beckett, Ian A. Prior, Daniel G. Booth, C. Seedhouse
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Simple Summary This review presents an overview of recent findings on various epigenetic and post-transcriptional modifications of the epidermal growth factor receptor (EGFR), which activates the oncogenes RAS and MYC. Alterations in EGFR, RAS, and MYC play a crucial role in dysregulated processes that contribute to oncogenesis. Focused, targeted research aimed at understanding the distinct effects of these oncogenes and their carcinogenic modifications has facilitated the successful treatment of various malignancies using inhibitors of EGFR tyrosine kinase activity and other therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6048dcc607aa9e108befb3b09588362de65007" target='_blank'>
              Dynamic Multilevel Regulation of EGFR, KRAS, and MYC Oncogenes: Driving Cancer Cell Proliferation Through (Epi)Genetic and Post-Transcriptional/Translational Pathways
              </a>
            </td>
          <td>
            M. Šereš, Katarina Spacayova, Z. Sulová, Jana Špaldová, Albert Breier, L. Pavlikova
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a small, yet pivotal subpopulation of tumor cells that play significant roles in tumor initiation, progression and therapeutic resistance. Circular RNAs (circRNAs) are a distinct class of RNAs characterized by their closed-loop structures, lacking 5′ to 3′ends. There is growing evidence that circRNAs are integral to the development and regulation of CSCs. Aberrant expression of circRNAs in CSCs can contribute to oncogenic properties and drug resistance. Specifically, oncogenic circRNAs modulate CSC behavior via key signaling pathways, thereby promoting CSC self-renewal and maintenance, as well as tumor progression. This review summarizes the latest research on the functional roles and regulatory mechanisms of circRNAs in CSC behavior and discusses potential applications and challenges of targeting circRNAs in CSCs. Understanding the intricate interactions between circRNAs and CSCs may lead to novel therapeutic strategies that effectively combat treatment resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01b87d0f590a46f3144a9c75dc4396105c17dca" target='_blank'>
              Circular RNAs in cancer stem cells: Insights into their roles and mechanisms (Review)
              </a>
            </td>
          <td>
            Lunyu Yang, Yuling Yi, Zhu Mei, Dongmei Huang, Sitian Tang, Liyi Hu, Ling Liu
          </td>
          <td>2025-01-23</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In mammals, X-linked dosage compensation involves two processes: X-chromosome inactivation (XCI) to balance X chromosome dosage between males and females, and hyperactivation of the remaining X chromosome (Xa-hyperactivation) to achieve X-autosome balance in both sexes. Studies of both processes have largely focused on coding genes and have not accounted for transposable elements (TEs) which comprise 50% of the X-chromosome, despite TEs being suspected to have numerous epigenetic functions. This oversight is due in part to the technical challenge of capturing repeat RNAs, bioinformatically aligning them, and determining allelic origin. To overcome these challenges, here we develop a new bioinformatic pipeline tailored to repetitive elements with capability for allelic discrimination. We then apply the pipeline to our recent So-Smart-Seq analysis of single embryos to comprehensively interrogate whether X-linked TEs are subject to either XCI or Xa-hyperactivation. With regards to XCI, we observe significant differences in TE silencing in parentally driven “imprinted” XCI versus zygotically driven “random” XCI. Chromosomal positioning and genetic background impact TE silencing. We also find that SINEs may influence 3D organization during XCI. In contrast, TEs do not undergo Xa-hyperactivation. Thus, while coding genes are subject to both forms of dosage compensation, TEs participate only in Xi silencing. Evolutionary and functional implications are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b9ae3587e3b90516f521d0a832022aafe22d64b" target='_blank'>
              Dosage compensation of transposable elements in mammals
              </a>
            </td>
          <td>
            Chunyao Wei, Barry A. Kesner, , Peera Wasserzug-Pash, Priyojit Das, Jeannie T Lee
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The autonomous and active Long-Interspersed Element-1 (LINE-1, L1) and the non-autonomous Alu retrotransposon elements, contributing to 30% of the human genome, are the most abundant repeated sequences. With more than 90% of their sequences being methylated in normal cells, these elements undeniably contribute to the global DNA methylation level and constitute a major part of circulating-cell-free DNA (cfDNA). So far, the hypomethylation status of LINE-1 and Alu in cellular and extracellular DNA has long been considered a prevailing hallmark of ageing-related diseases and cancer. This study demonstrated that errors in LINE-1 and Alu methylation level measurements were caused by an excessive input quantity of genomic DNA used for bisulfite conversion. Using the minuscule DNA amount of 0.5 ng, much less than what has been used and recommended so far (500 ng-2 μg) or 1 μL of cfDNA extracted from 1 mL of blood, we revealed hypermethylation of LINE-1 and Alu in 407 tumour samples of primary breast, colon and lung cancers when compared with the corresponding pair-matched adjacent normal tissue samples (P < 0.05–0.001), and in cfDNA from 296 samples of lung cancers as compared with 477 samples from healthy controls (P < 0.0001). More importantly, LINE-1 hypermethylation in cfDNA is associated with healthy ageing. Our results have not only contributed to the standardized bisulfite-based protocols for DNA methylation assays, particularly in applications on repeated sequences but also provided another perspective for other repetitive sequences whose epigenetic properties may have crucial impacts on genome architecture and human health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af9de458a2650aad970d8197a4d94f0fb4283e9" target='_blank'>
              An appropriate DNA input for bisulfite conversion reveals LINE-1 and Alu hypermethylation in tissues and circulating cell-free DNA from cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Tung The Pham, Than Thi Nguyen, Trang Hien Do, Phương Thi Thu Luu, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Son Van Ho, , Tong Van Hoang, Lan Thi Thuong Vo
          </td>
          <td>2024-12-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/067cac427326fdea9459d93348affd2bc0e1346c" target='_blank'>
              Nucleus-translocated GCLM promotes chemoresistance in colorectal cancer through a moonlighting function
              </a>
            </td>
          <td>
            Jin-Fei Lin, Ze-xian Liu, Dong-Liang Chen, Ren-Ze Huang, Fen Cao, Kai Yu, Ting Li, Hai-Yu Mo, Hui Sheng, Zhi-Bing Liang, Kun Liao, Yi Han, Shan-Shan Li, Zhao-Lei Zeng, Song Gao, Huai-qiang Ju, Rui-Hua Xu
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERV) represent nearly 8% of the human genome. Of these, HERV-K subtype HML-2 is a transposable element that plays a critical role in embryonic development and in the pathogenesis of several diseases. Quantification and characterization of these multiple HML-2 insertions in the human chromosome has been challenging due to their size, sequence homology with each other, and their repetitive nature. We examined a cohort of 222 individuals for HML-2 proviruses 6q14.1 and 7p22.1a, two loci that are capable of producing full-length viral proteins and have been previously implicated in several cancers, autoimmune disorders and neurodegenerative diseases, using long-read DNA sequencing. While the reference genome for both regions suggests these two loci are structurally dissimilar, we found that for both loci about 5% of individuals have a unique tandem repeat-like sequence (three long terminal repeat sequences sandwiching two internal, potentially protein coding sequences), while most individuals have a standard proviral structure (one internal region sandwiched by two long terminal repeats). Moreover, both proviruses can make full-length, or nearly full-length, HERV-K proteins in multiple transcription orientations. The amino acid sequences from different loci in the same transcriptional orientation share sequence homology with each other. These results demonstrate a clear, previously unreported, relationship between HML-2 loci 6q14.1 and 7p22.1a and highlight the utility of long-read sequencing to study repetitive elements. Future studies need to determine if these polymorphisms determine genetic susceptibility to diseases that are associated with them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd67bb0a75d66d24c8cfbfff72aa27068e59aecc" target='_blank'>
              Characterization of novel human endogenous retrovirus structures on chromosomes 6 and 7
              </a>
            </td>
          <td>
            Nicholas Pasternack, Ole Paulsen, Avindra Nath
          </td>
          <td>2025-01-27</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3245a4598675a5f03862517e366728182b100458" target='_blank'>
              Molecular mechanism targeting condensin for chromosome condensation.
              </a>
            </td>
          <td>
            Menglu Wang, Daniel Robertson, Juan Zou, Christos Spanos, J. Rappsilber, A. Marston
          </td>
          <td>2024-12-17</td>
          <td>The EMBO journal</td>
          <td>1</td>
          <td>85</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, , Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mutations in the exonuclease domains of the replicative nuclear DNA polymerases POLD1 and POLE are associated with increased cancer incidence, elevated tumor mutation burden (TMB), and enhanced response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond, highlighting the need for a better understanding of how TMB affects tumor biology and subsequently immunotherapy response. To address this, we generated mice with germline and conditional mutations in the exonuclease domains of Pold1 and Pole. Engineered mice with Pold1 and Pole mutator alleles presented with spontaneous cancers, primarily lymphomas, lung cancer, and intestinal tumors, while Pold1 mutant mice also developed tail skin carcinomas. These cancers had highly variable tissue-type dependent increased TMB with mutational signatures associated with POLD1 and POLE mutations found in human cancers. The Pold1 mutant tail tumors displayed increased TMB, however, only a subset of established tumors responded to ICB. Similarly, introducing the mutator alleles into mice with lung cancer driven by mutant Kras and Trp53 deletion did not improve survival, whereas passaging these tumor cells in vitro without immune editing and subsequently implanting them into immune-competent mice caused tumor rejection in vivo. These results demonstrated the efficiency by which cells with antigenic mutations are eliminated in vivo. Finally, ICB treatment of mutator mice earlier, before observable tumors had developed delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the potential of ICB in high-risk individuals for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274ef0221dce61440b76214e33f127197aac5688" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer Development and Extends Survival in DNA Polymerase Mutator Syndromes.
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, , Zhixian Hu, Somer Abdelfattah, Jennele Baul, , Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae5f6bb3e804b17358c04ea87f466f1ee8583c6" target='_blank'>
              Shedding light on DNA methylation and its clinical implications: the impact of long-read-based nanopore technology
              </a>
            </td>
          <td>
            Alexandra Chera, Mircea Stancu-Cretu, N. Zabet, Octavian Bucur
          </td>
          <td>2024-12-30</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8eb632e8dcc4d371aa3266d6271acffef83ed2" target='_blank'>
              The integrated stress response drives MET oncogene overexpression in cancers.
              </a>
            </td>
          <td>
            M. Cerqua, Marco Foiani, C. Boccaccio, Paolo M Comoglio, 
          </td>
          <td>2025-01-07</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mutations and genome instability are hallmarks of cancer. The advanced Next Generation Sequencing(NGS) technology now enables rapid and cost-effective analysis for tumoral and normal genomic translational research. In this research, mRNA sequencing was applied to get the whole transcriptome data of mouse B16F10 melanoma and CT26 colon carcinoma cells. Up regulation genes and down regulation genes were identified in tumor cells with differentially expressed genes(DEGs) analysis. The DEGs were analyzed with GO, KEGG and Reactome pathway enrichment study. Results demonstrated the DEGs and enriched pathways were mainly associated with cell proliferation and biosynthetic processing. Further, gene fusion and alternative splicing analysis reveled fusion genes and skipped exons were extensive exist within the chromosomes of tumor cells. Together, the integrated DEGs and bioinformatics analysis in this article may shed light on further research of cancer treating targets and molecular mechanisms of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6e94aa8f9282678d155caf1a32035cd06536087" target='_blank'>
              Identification and analysis of DEGs of mouse tumor cells by integrated bioinformatics methods
              </a>
            </td>
          <td>
            Xizhong Jing, Baiyong Li, Yifei Wang
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="The human genome is like a museum of ancient retroviral infections. It contains a large number of endogenous retroviruses (ERVs) that bear witness to past integration events. About 5,000 of them are so-called long terminal repeat 12 (LTR12) elements. Compared with 20,000 human genes, this is a remarkable number. Although LTR12 elements can act as promoters or enhancers of cellular genes, the function of most of these retroviral elements has remained unclear. In our mini-review, we show that different LTR12 elements share many similarities, including common transcription factor binding sites. Furthermore, we summarize novel insights into the epigenetic mechanisms governing their silencing and activation. Specific examples of genes and pathways that are regulated by LTR12 loci are used to illustrate the regulatory network built by these repetitive elements. A particular focus is on their role in the regulation of antiviral immune responses, tumor cell proliferation, and senescence. Finally, we describe how a targeted activation of this fascinating ERV family could be used for diagnostic or therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4041a47b44612c0a5daa01efba716fa89ab732e8" target='_blank'>
              Viral influencers: deciphering the role of endogenous retroviral LTR12 repeats in cellular gene expression.
              </a>
            </td>
          <td>
            Veronika Krchlikova, Yueshuang Lu, D. Sauter
          </td>
          <td>2025-01-31</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="It is thought that cells surviving ionizing radiation exposure repair DNA double-strand breaks (DSBs) and restore their genomes. However, the recent biochemical and genetic characterization of DSB repair pathways reveals that only homologous recombination (HR) can function in an error-free manner and that the non-homologous end joining (NHEJ) pathways canonical NHEJ (c-NHEJ), alternative end joining (alt-EJ), and single-strand annealing (SSA) are error-prone, and potentially leave behind genomic scars and altered genomes. The strong cell cycle restriction of HR to S/G2 phases and the unparalleled efficiency of c-NHEJ throughout the cell cycle, raise the intriguing question as to how far a surviving cell "reaches" after repairing the genome back to its pre-irradiation state. Indeed, there is evidence that the genomes of cells surviving radiation treatment harbor extensive genomic alterations. To directly investigate this possibility, we adopted next-generation sequencing (NGS) technologies and tested a normal human fibroblast cell line, 82-6 hTert, after exposure up to 6 Gy. Cells were irradiated and surviving colonies expanded and the cells froze. Sequencing analysis using the Illumina sequencing platform and comparison with the unirradiated genome detected frequent genomic alterations in the six investigated radiation survivor clones, including translocations and large deletions. Translocations detected by this analysis and predicted to generate visible cytogenetic alterations were frequently (three out of five) confirmed using mFISH cytogenetic analysis. PCR analysis of selected deletions also confirmed seven of the ten examined. We conclude that cells surviving radiation exposure tolerate and pass to their progeny a wide spectrum of genomic alterations. This recognition needs to be integrated into the interpretation of biological results at all endpoints, as well as in the formulation of mathematical models of radiation action. NGS analysis of irradiated genomes promises to enhance molecular cytogenetics by increasing the spectrum of detectable genomic alterations and advance our understanding of key molecular radiobiological effects and the logic underpinning DSB repair. However, further developments in the technology will be required to harness its full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b3247759666806f5cf74d149fcc038397e2c814" target='_blank'>
              NGS Detects Extensive Genomic Alterations in Survivors of Irradiated Normal Human Fibroblast Cells.
              </a>
            </td>
          <td>
            A. Soni, Daniela Beisser, E. Mladenov, Matthias Höller, Inken Wohlers, Vladimir Nikolov, S. Magin, Tamara Mussfeldt, Ludger Klein-Hitpass, Michael N. Cornforth, B. Loucas, Sven Rahmann, G. Iliakis
          </td>
          <td>2024-12-27</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b4d59be0a1ccb5ccd42ae33408325f684d6c1d" target='_blank'>
              RRM1 promotes homologous recombination and radio/chemo-sensitivity via enhancing USP11 and E2F1-mediated RAD51AP1 transcription
              </a>
            </td>
          <td>
            Shuai Yang, Ruru Wang, Lingling Liu, Feng Xu, Xipeng Zhao, Zhicheng Yao, Jie Zhang, Xu Cheng, An Xu, Lijun Wu, Guoping Zhao
          </td>
          <td>2024-12-18</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e33b574995e584f7a063a425a60100a1abcb82e" target='_blank'>
              Identification of a novel TOP2B::AFF2 fusion gene in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tao Liu, Yang Wang, Xi-Zhou An, Jiaqi Liu, , Yan Xiang, Yong-Jie Zhang, Lan Huang, Jia-Cheng Li, Yu-Zhuo-Pu Li, Jie Yu
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="
 DNA double strand breaks (DSBs) result in activation of several key DNA damage response (DDR) kinases including ATM, ATR, and DNA-PK. These protein kinases not only promote DNA damage-induced checkpoint control, but also facilitate DSB repair in humans. Thus, these DDR kinases have become promising drug targets for cancer therapy. However, the benefits of targeting DDR kinases remain to be realized, in part due to the lack of predictive biomarkers. By undertaking CRISPR screens with inhibitors targeting key DDR kinases, we obtained a global and unbiased view of genetic interactions with DDR inhibition. Additionally, we compared the synergistic effects of combining different DDR inhibitors and found that ATM and PARP inhibitors showed remarkable synergy, which depends on the dominant negative function of ATM inhibition. Moreover, to provide comprehensive and unbiased perspective of DDR signaling pathways, we performed 30 fluorescence-activated cell sorting–based genome-wide CRISPR screens with antibodies recognizing distinct endogenous DNA damage–signaling proteins to identify new regulators involved in DNA damage response (DDR). We discovered that proteasome-mediated processing is an early and prerequisite event for cells to trigger camptothecin- and etoposide-induced DDR signaling. Furthermore, we identified PRMT1 and PRMT5 as new modulators that regulate ATM protein level. Additionally, we discovered that GNB1L is a master regulator of DDR signaling via its role as a co-chaperone for PIKK proteins. Collectively, these screens offer a rich resource for further investigation of DDR, which may provide insight into strategies of targeting these DDR pathways to improve therapeutic outcomes. More recently, we have adopted dTAG technology for the investigation of many essential DDR and replication proteins, which provide mechanistic insights into the roles of these essential proteins in genome maintenance. Additionally, we have expanded FACS-based screens for the studies of other cancer-associated pathways and explored key biological processes critical for cancer progression in vivo. These studies will facilitate the development of better therapies for cancer patients.
 Citation Format: Junjie Chen. Targeting DNA damage responsive pathways in cancer therapy. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b542469fcebc2361e69bec6d43fd73238a87bdc" target='_blank'>
              Abstract A009: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Formalin preservation of museum specimens has long been considered a barrier to molecular research due to extensive crosslinking and chemical modification. However, recent optimisation of hot alkaline lysis and proteinase K digestion DNA extraction methods have enabled a growing number of studies to overcome these challenges and conduct genome-wide re-sequencing and targeted locus-specific sequencing. The newest, and perhaps most unexpected utility of formalin preservation in archival samples is its ability to preserve in situ DNA-protein interactions at a molecular level. Retrieving this signal provides information about the relative compaction or accessibility of the genome to the transcriptional machinery required for gene expression. Thus, exposure to formalin essentially corresponds to taking a snapshot of organism-wide gene expression at the time of death. While DNA methylation and RNA-Seq analyses of dried tissues have provided glimpses into historical gene regulation, these techniques were previously limited to skeletal or desiccated remains, offering only partial insights. By examining fluid-preserved specimens, molecular tools can now be applied to a broader range of tissues, enabling more detailed tissue-specific gene regulation profiling across vertebrates. In this review, we chronicle the historical use of formaldehyde in collections and discuss how targeted chromatin profiling with assays like MNase-seq and FAIRE-seq are surmounting fixation challenges and unlocking invaluable insights into historical genomes and gene expression profiles. The deeper integration of molecular genetics with museum collections bridges the gap between past and present and provides a vital tool that could help us predict and mitigate some of the impacts of future environmental change, novel pathogens, or invasive species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d71cdced5ded68073de0b09e74b9af4ad05548ef" target='_blank'>
              Reframing Formalin: A Molecular Opportunity Enabling Historical Epigenomics and Retrospective Gene Expression Studies.
              </a>
            </td>
          <td>
            C. E. Holleley, Erin E. Hahn
          </td>
          <td>2025-01-02</td>
          <td>Molecular ecology resources</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Large genetic variants can be generated via homologous recombination (HR), such as polymerase theta-mediated end joining (TMEJ) or single-strand annealing (SSA). Given that these HR-based mechanisms leave specific genomic signatures, we developed GDBr, a genomic signature interpretation tool for DNA double-strand break repair mechanisms using high-quality genome assemblies. We applied GDBr to a draft human pangenome reference. We found that 78.1% of non-repetitive insertions and deletions and 11.0% of non-repetitive complex substitutions contained specific signatures. Of these, we interpreted that 98.7% and 1.3% of the insertions and deletions were generated via TMEJ and SSA, respectively, and all complex substitutions via TMEJ. Since population-level pangenome datasets are being dramatically accumulated, GDBr can provide mechanistic insights into how variants are formed. GDBr is available on GitHub at https://github.com/Chemical118/GDBr.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24886490d8b6e2e3d68ad1604f388da5f8ce50c" target='_blank'>
              GDBr: genomic signature interpretation tool for DNA double-strand break repair mechanisms
              </a>
            </td>
          <td>
            Hyunwoo Ryu, H. Han, Chuna Kim, Jun Kim
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Polycomb repressive complex 2 (PRC2) is known to regulate gene expression and chromatin structure as it methylates H3K27, resulting in gene silencing. Studies have shown that PRC2 has dual functions in oncogenesis that allow it to function as both an oncogene and a tumor suppressor. Because of this, nuanced strategies are necessary to promote or inhibit PRC2 activity therapeutically. Given the therapeutic vulnerabilities and associated risks in oncological applications, a structured literature review on PRC2 was conducted to showcase similar cofactor competitor inhibitors of PRC2. Key inhibitors such as Tazemetostat, GSK126, Valemetostat, and UNC1999 have shown promise for clinical use within various studies. Tazemetostat and GSK126 are both highly selective for wild-type and lymphoma-associated EZH2 mutants. Valemetostat and UNC1999 have shown promise as orally bioavailable and SAM-competitive inhibitors of both EZH1 and EZH2, giving them greater efficacy against potential drug resistance. The development of other PRC2 inhibitors, particularly inhibitors targeting the EED or SUZ12 subunit, is also being explored with the development of drugs like EED 226. This review aims to bridge gaps in the current literature and provide a unified perspective on promising PRC2 inhibitors as therapeutic agents in the treatment of lymphomas and solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d39c17f73d3909cec9400437d3632effbc2233b4" target='_blank'>
              Pharmacological Advancements of PRC2 in Cancer Therapy: A Narrative Review
              </a>
            </td>
          <td>
            Michael S. Wang, Jonathan H Sussman, Jessica A. Xu, Reema Patel, Omar Elghawy, Prashanth Rawla
          </td>
          <td>2024-12-01</td>
          <td>Life</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/352956df3f50dd3a8802a00f3ce2fb6c110258ba" target='_blank'>
              SMC translocation is unaffected by an excess of nucleoid associated proteins in vivo
              </a>
            </td>
          <td>
            Zhongqing Ren, Lindsey E. Way, Xindan Wang
          </td>
          <td>2025-01-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Most of the risk factors associated with chronic and complex diseases, such as cancer, stem from exogenous and endogenous exposures experienced throughout an individual’s life, collectively known as the exposome. These exposures can modify DNA, which can subsequently lead to the somatic mutations found in all normal and tumor tissues. Understanding the precise origins of specific somatic mutations has been challenging due to multitude of DNA adducts (i.e. the DNA adductome) and their diverse positions within the genome. Thus far, this limitation has prevented researchers from precisely linking exposures to DNA adducts and DNA adducts to subsequent mutational outcomes. Indeed, many common mutations observed in human cancers appear to originate from error-prone endogenous processes. Consequently, it remains unclear whether these mutations result from exposure-induced DNA adducts, or arise indirectly from endogenous processes or are a combination of both. In this review, we summarize approaches that aim to bridge our understanding of the mechanism by which exposure leads to DNA damage and then to mutation and highlight some of the remaining challenges and shortcomings to fully supporting this paradigm. We emphasize the need to integrate cellular DNA adductomics, long read-based mapping, single-molecule duplex sequencing of native DNA molecules and advanced computational analysis. This proposed holistic approach aims to unveil the causal connections between key DNA modifications and the mutational landscape, whether they originate from external exposures, internal processes or a combination of both, thereby addressing key questions in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdf82ec8445e0cef06c9a8eb260342f4ef9a96b" target='_blank'>
              Investigating the origins of the mutational signatures in cancer
              </a>
            </td>
          <td>
            Gunnar Boysen, Ludmil B. Alexandrov, R. Rahbari, I. Nookaew, Dave Ussery, M. Chao, Chiung-Wen Hu, Marcus S. Cooke
          </td>
          <td>2025-01-07</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Telomere maintenance is crucial for preventing the linear eukaryotic chromosome ends from being mistaken for DNA double-strand breaks, thereby avoiding chromosome fusions and the loss of genetic material. Unlike most eukaryotes that use telomerase for telomere maintenance, Drosophila relies on retrotransposable elements-specifically HeT-A, TAHRE, and TART (collectively referred to as HTT)-which are regulated and precisely targeted to chromosome ends. Drosophila telomere protection is mediated by a set of fast-evolving proteins, termed terminin, which bind to chromosome termini without sequence specificity, balancing DNA damage response factors to avoid erroneous repair mechanisms. This unique telomere capping mechanism highlights an alternative evolutionary strategy to compensate for telomerase loss. The modulation of recombination and transcription at Drosophila telomeres offers insights into the diverse mechanisms of telomere maintenance. Recent studies at the population level have begun to reveal the architecture of telomere arrays, the diversity among the HTT subfamilies, and their relative frequencies, aiming to understand whether and how these elements have evolved to reach an equilibrium with the host and to resolve genetic conflicts. Further studies may shed light on the complex relationships between telomere transcription, recombination, and maintenance, underscoring the adaptive plasticity of telomeric complexes across eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc71fe25065e2464b3633178f88c0066a196f35c" target='_blank'>
              Maintaining Telomeres without Telomerase in Drosophila: Novel Mechanisms and Rapid Evolution to Save a Genus.
              </a>
            </td>
          <td>
            S. Cacchione, G. Cenci, Anne-Marie Dion-Côté, D. Barbash, G. Raffa
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87208750323d073805f3062a30f76b4849f02905" target='_blank'>
              Unveiling cell-type-specific microRNA networks through alternative polyadenylation in glioblastoma
              </a>
            </td>
          <td>
            Mert Cihan, , Maximilian Sprang, Miguel Andrade
          </td>
          <td>2025-01-21</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="ABSTRACT Epstein‐Barr virus (EBV) is a ubiquitous human ɣ‐herpesvirus implicated in various malignancies, including Burkitt's lymphoma and gastric carcinomas. In most EBV‐associated cancers, the viral genome is maintained as an extrachromosomal episome by the EBV nuclear antigen‐1 (EBNA1). EBNA1 is considered to be a highly stable protein that interacts with the ubiquitin‐specific protease 7 (USP7). Here, we show that pharmacological inhibitors and small interfering RNA (siRNA) targeting USP7 reduce EBNA1 protein levels in a proteosome‐dependent manner. Proteomic analysis revealed that USP7 inhibitor GNE6776 altered the EBNA1 protein interactome, including disrupting USP7 association with EBNA1. GNE6776 also inhibited EBNA1 binding to EBV oriP DNA and reduced viral episome copy number. Transcriptomic studies revealed that USP7 inhibition affected chromosome segregation and mitotic cell division pathways in EBV+ cells. Finally, we show that GNE6776 selectively inhibited EBV+ gastric and lymphoid cell proliferation in cell culture and slowed EBV+ tumor growth in mouse xenograft models. These findings suggest that USP7 inhibitors perturb EBNA1 stability and function and may be exploited to treat EBV latent infection and tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5972601207babc010d4b0b866abd5a2b3ddcc3" target='_blank'>
              USP7 Inhibitors Destabilize EBNA1 and Suppress Epstein‐Barr Virus Tumorigenesis
              </a>
            </td>
          <td>
            Christopher Chen, Kush Addepalli, S. Soldan, L. J. Castro-Muñoz, Sarah Preston-Alp, Rishi J Patel, Coltin J Albitz, Hsin-Yao Tang, I. Tempera, P. Lieberman
          </td>
          <td>2025-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a significant global health challenge, underscoring the need for innovative therapeutic strategies. Oncogenic miRNAs (oncomiRs) play a significant biological role in the initiation and progression of colorectal cancer. Inspired by the cooperative mechanisms of plant nanovirus, which employ multiple circular single-stranded DNA (CssDNA) genomes, it is hypothesized that the development and delivery of CssDNA to target oncomiRs would achieve therapeutic benefits in CRC. In this study, a multi-omics approach is utilized to identify key tumor suppressor genes (TSGs) and their related oncomiRs implicated in CRC, followed by the development of CssDNA, each of which is loaded with multiple miRNA binding sites targeting one oncomiR. When transfected into the cells, these CssDNA can effectively target and sequester the corresponding oncomiRs to restore the expression of TSGs, leading to a marked reduction in CRC development both in vitro and in vivo. The findings highlight the therapeutic potential of nanovirus-inspired CssDNA in modulating the miRNA-mediated regulatory network in CRC. This study lays the groundwork for the development of non-coding DNA-based therapies with broad implications for the treatment of colorectal cancer and potentially other malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a389d3e4c09149bfce4cb4ae8bf6fdc4a619a01" target='_blank'>
              Circular Single-Stranded DNA-Based Artificial Nanoviruses Mitigate Colorectal Cancer Development.
              </a>
            </td>
          <td>
            Jinghao Wang, Pengfei Zhang, Yonglian Huang, Guang Hu, Kexuan Zou, Songtao Zhou, Dandan Shao, Jianming Wang, Jie Song
          </td>
          <td>2025-01-21</td>
          <td>Small methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In many eukaryotes, meiotic recombination occurs preferentially at discrete sites, called recombination hotspots. In various lineages, recombination hotspots are located in regions with promoter-like features and are evolutionarily stable. Conversely, in some mammals, hotspots are driven by PRDM9 that targets recombination away from promoters. Paradoxically, PRDM9 induces the self-destruction of its targets and this triggers an ultra-fast evolution of mammalian hotspots. PRDM9 is ancestral to all animals, suggesting a critical importance for the meiotic program, but has been lost in many lineages with surprisingly little effect on meiosis success. However, it is unclear whether the function of PRDM9 described in mammals is shared by other species. To investigate this, we analyzed the recombination landscape of several salmonids, the genome of which harbors one full-length PRDM9 and several truncated paralogs. We identified recombination initiation sites in Oncorhynchus mykiss by mapping meiotic DNA double-strand breaks (DSBs). We found that DSBs clustered at hotspots positioned away from promoters, enriched for the H3K4me3 and H3K36me3 and the location of which depended on the genotype of full-length Prdm9. We observed a high level of polymorphism in the zinc finger domain of full-length Prdm9, indicating diversification driven by positive selection. Moreover, population-scaled recombination maps in O. mykiss, Oncorhynchus kisutch and Salmo salar revealed a rapid turnover of recombination hotspots caused by PRDM9 target motif erosion. Our results imply that PRDM9 function is conserved across vertebrates and that the peculiar evolutionary runaway caused by PRDM9 has been active for several hundred million years.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f09c603402d3b52a99c356601f8730c3715c9b0" target='_blank'>
              PRDM9 drives the location and rapid evolution of recombination hotspots in salmonid fish
              </a>
            </td>
          <td>
            Marie-Laure Raynaud, Paola Sanna, Julien Joseph, J. Clément, Yukiko Imai, J. Lareyre, Audrey Laurent, Nicolas Galtier, F. Baudat, Laurent Duret, P. Gagnaire, B. de Massy
          </td>
          <td>2025-01-01</td>
          <td>PLOS Biology</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="From RNA interference to chromatin silencing, diverse genome defense pathways silence selfish genetic elements to safeguard genome integrity. Despite their diversity, different defense pathways share a modular organization, where numerous specificity factors identify diverse targets and common effectors silence them. In the PIWI-interacting RNA (piRNA) pathway, target RNAs are first identified by complementary base pairing with piRNAs and then silenced by PIWI-clade nucleases. Such a binary architecture allows the defense systems to be readily adaptable, where new targets can be captured via innovation of specificity factors. Thus, our current understanding of genome defense against lineage-specific selfish genes has been largely limited to specificity factor innovations, while it remains poorly understood whether other types of innovations are required. Here, we describe a new type of innovation, which escalates the genome defense capacity to control a recently expanded selfish gene in Drosophila melanogaster. Through a targeted RNAi screen for repressors of Stellate-a recently evolved meiotic driver-we identified a defense factor, Trailblazer. Trailblazer is a transcription factor that promotes the expression of two PIWI-clade nucleases, Aub and AGO3, to match Stellate in abundance. Recent innovation in the DNA-binding domain of Trailblazer enabled it to elevate Aub and AGO3 expression, thereby escalating the silencing capacity of piRNA pathway to tame expanded Stellate and safeguard fertility. As copy-number expansion is a recurrent feature of diverse selfish genes across the tree of life, we envision that augmenting the defense capacity to quantitatively match selfish genes is a repeatedly employed defense strategy in evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b459a2d133898ca4732a04488f0f54cfda8ecf00" target='_blank'>
              Escalation of genome defense capacity enables control of an expanding meiotic driver.
              </a>
            </td>
          <td>
            Peiwei Chen, , Eunice H. Park, Yicheng Luo, Y. C. G. Lee, A. Aravin
          </td>
          <td>2025-01-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba737d66daf461c8c86cfa467ed3f95f6246b1d2" target='_blank'>
              Targeting BRCA1-deficient PARP inhibitor-resistant cells with nickases reveals nick resection as a cancer vulnerability.
              </a>
            </td>
          <td>
            Jenna M Whalen, Jillian Earley, C. Wisniewski, A. Mercurio, Sharon B Cantor
          </td>
          <td>2025-01-21</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125e72b0b5ffa707da90fa52c6a8e151fa428862" target='_blank'>
              TRACERx analysis identifies a role for FAT1 in regulating chromosomal instability and whole-genome doubling via Hippo signalling
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, Alastair Magness, K. Thol, Thomas A. Webber, Ming Jiang, R. Saunders, Yun-Hsin Liu, D. Biswas, Esther O. Ige, Birgit Aerne, E. Grönroos, Subramanian Venkatesan, Georgia Stavrou, Takahiro Karasaki, M. Al Bakir, M. Renshaw, Hang Xu, Deborah Schneider-Luftman, Natasha Sharma, Laura Tovini, M. Jamal-Hanjani, S. McClelland, K. Litchfield, Nicolai J. Birkbak, Michael Howell, N. Tapon, K. Fugger, N. Mcgranahan, J. Bartek, N. Kanu, C. Swanton
          </td>
          <td>2024-12-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1a74726e395311f7dc9708215816307d41962f" target='_blank'>
              Asymmetric distribution of G-quadruplex forming sequences in genomes of retroviruses
              </a>
            </td>
          <td>
            Filip Kledus, Michaela Dobrovolná, J. Mergny, Václav Brázda
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is one of the deadliest human cancers, with the overall 5-year survival rate stagnating in recent decades due to the lack of innovative treatment approaches. Apart from the recently Food and Drug Administration-approved epidermal growth factor receptor inhibitor and immune checkpoint inhibitor, alternative therapeutic strategies that target epigenetic abnormalities, an emerging cancer hallmark, remain to be fully explored. A pathological epigenetic landscape, characterized by widespread reprogramming of chromatin modifications such as DNA methylation and histone modifications, which drives transcription deregulation and genome reorganization, has been extensively documented in numerous cancers, including HNSCC. Growing evidence indicates that these frequent epigenomic alterations play pivotal roles in regulating malignant transformation, promoting metastasis and invasion, and reshaping the tumor microenvironment. Furthermore, these epigenetic changes also present unique vulnerabilities that open new avenues for identifying novel prognostic biomarkers and developing targeted antitumor therapies. In this review, we summarize recent discoveries of epigenetic dysregulations in HNSCC, with a focus on deregulated chromatin modifications, which include aberrant DNA methylation, oncohistone H3 lysine 36 to methionine (H3K36M) mutation, as well as recurrent mutations or altered expression of chromatin-modifying enzymes such as NSD1, EZH2, and KMT2C/D. Importantly, we discuss the various molecular mechanisms underlying the contributions of these epigenetic alterations to HNSCC development, particularly their involvement in deregulated cell proliferation and cell death, metabolic reprogramming, tumor immune evasion, and phenotypic plasticity. Finally, we conclude by highlighting the translational and clinical implications of targeting the epigenetic machinery, which offers promising prospects for overcoming resistance to conventional radiotherapy/chemotherapy and enhancing the response to immunotherapy in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f6940a95157f857134f2791aea649c5fe14cbd" target='_blank'>
              Targeting Epigenetic Dysregulations in Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            , C. Lu
          </td>
          <td>2024-12-19</td>
          <td>Journal of dental research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Simple Summary The homeobox transcription factor NKX3.1 is essential for the development of the prostate and has long been thought of as a tumor suppressor. We hypothesize that much like the androgen receptor (AR), NKX3.1 plays a pro-differentiative, tumor suppressive role in early-stage disease and potentially acts as an oncogene in late-stage, AR-driven prostate cancer. Our results indicate that NKX3.1 is a critical co-factor for AR in prostate cancer cells, and knockdown leads to decreased cell survival, while overexpression is enough to promote growth in androgen depleted and AR-inhibited conditions. Finally, we suggest that NKX3.1 may promote oncogenic activity in late-stage disease through upregulation of genes driven by AR at selective androgen response elements. Altogether, these data indicate that NKX3.1 is necessary and sufficient for prostate cancer growth, and like AR, functions as an oncogene in a context-dependent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d23adb1a041976f4f7a7bd36179e78ccc6d3c88" target='_blank'>
              The Homeobox Transcription Factor NKX3.1 Displays an Oncogenic Role in Castration-Resistant Prostate Cancer Cells
              </a>
            </td>
          <td>
            Audris Budreika, John T. Phoenix, R. Kostlan, Carleen D. Deegan, Marina G Ferrari, K. Young, Sean W. Fanning, S. Kregel
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aea14196d457e30b1e0c0858af5e94a5516dad2" target='_blank'>
              RBM47 promotes cell proliferation and immune evasion by upregulating PDIA6: a novel mechanism of pancreatic cancer progression
              </a>
            </td>
          <td>
            Yihui Ma, Enjie Liu, , Chenfei Li, Pei Huang, Meiying Cui, Zhengyang Wang, Jing Zhou, Kuisheng Chen
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5cabef889301b036a508a4127fd265270ae7d40" target='_blank'>
              DNA hypermethylation of tumor suppressor genes among oral squamous cell carcinoma patients: a prominent diagnostic biomarker.
              </a>
            </td>
          <td>
            Nistha Agarwal, A. Jha
          </td>
          <td>2024-12-07</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5202cd39039a85ffc734b688a9ee572ce5b91b15" target='_blank'>
              Inhibition of GSK3β is synthetic lethal with FHIT loss in lung cancer by blocking homologous recombination repair.
              </a>
            </td>
          <td>
            Shishi Tao, Yue Pu, E. Yang, Guowen Ren, Changxiang Shi, Li-Jie Chen, , J. Shim
          </td>
          <td>2025-01-06</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="DNA methylation is an important epigenetic mechanism that helps define and maintain cellular functions. It is influenced by many factors, including environmental exposures, genotype, cell type, sex, and aging. Since age is the primary risk factor for developing neurodegenerative diseases, it is important to determine if aging-related DNA methylation is retained when cells are reprogrammed to an induced Pluripotent Stem Cell (iPSC) state. Here, we selected peripheral blood mononuclear cells (PBMCs; n = 99) from a cohort of diverse and healthy individuals enrolled in the Genetic and Epigenetic Signatures of Translational Aging Laboratory Testing (GESTALT) study to convert to iPSCs. After reprogramming we evaluated the resulting iPSCs for DNA methylation signatures to determine if they reflect the confounding factors of age and environmental factors. We used genome-wide DNA methylation arrays in both cell types to show that the epigenetic clock is largely reset to an early methylation age after conversion of PBMCs to iPSCs. We further examined the epigenetic age of each cell type using an Epigenome-wide Association Study (EWAS). Finally, we identified a set of methylation Quantitative Trait Loci (methQTL) in each cell type. Our results show that age-related DNA methylation is largely reset in iPSCs, and each cell type has a unique set of methylation sites that are genetically influenced. Graphical Abstract Highlights Generation of a population-level set of iPSC lines from healthy individuals across the lifespan Aging-related features were reset based on epigenetic markers of cytosine methylation and telomere length By comparing methQTLs in iPSC vs. their donor PBMCs, we find that detection of methQTLs reflect biological functions of different cell types">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/399ecd8f3690914bdcf9c8d820f3abd8488f2bee" target='_blank'>
              Characterization of DNA methylation in PBMCs and donor-matched iPSCs shows methylation is reset during stem cell reprogramming
              </a>
            </td>
          <td>
            X. Reed, Cory A. Weller, S. Sáez-Atienzar, A. Beilina, Sultana Solaiman, Makayla K Portley, Mary Kaileh, Roshni Roy, Jinhui Ding, A. Moore, D. T. Whitaker, Bryan J. Traynor, J. R. Gibbs, Sonja W. Scholz, Mark R. Cookson
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Breast cancer is known as one of the most predominant cancers that affect both females and
males worldwide. The most crucial risk factor in breast cancer is the mutations in the RAD51C
gene that have been considered in most hereditary breast cancers. RAD51C, the RAD51
paralogs, is also a deoxyribonucleic acid (DNA) repair protein related to breast and ovarian
cancers. DNA double-strand breaks (DSBs) account for the significant detrimental form of
DNA damage. RAD51C mutants also have been recognized in breast/ovarian cancer patients.
However, the role of the RAD51C protein in hereditary breast cancer and its three-dimensional
(3D) structures remains unclear. Thus, this study was conducted to identify the 3D structure of
RAD51C protein from its amino acid sequences. The homology modeling for the 3D structure
of the RAD51C protein was carried out by using three automated webservers: I-TASSER,
SWISS-MODEL, and Phyre2. PyMOL was applied to visualize the 3D structure of RAD51C
protein. Next, the MolProbity, ProSA, and SAVES v6.0 programs have been employed to
check the stereo-chemical quality of RAD51C protein. The RAD51C-IT models were found to
be the best models for the RAD1C protein after being evaluated and validated, and the models
were constructed using full-length RAD51C protein sequences. Thus, these protein models can
be utilized as a virtual screening tool in discovering potential inhibitors of RAD51C protein.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be054c5f435a476d5df616bc8bacb7f34a1b8877" target='_blank'>
              Homology Modeling of Human DNA Repair Protein RAD51 Homolog 3 (RAD51C) in Breast Cancer
              </a>
            </td>
          <td>
            Ruzianisra Mohamed
          </td>
          <td>2024-12-30</td>
          <td>International Journal of Pharmaceuticals, Nutraceuticals and Cosmetic Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell cycle checkpoints are stringent quality control mechanisms that regulate cell cycle progression and division. Cancer cells often develop a dependency on the G2/M cell cycle checkpoint to facilitate DNA repair and resolve intrinsic or therapy-induced DNA damage. This dependency leads to therapy resistance, continuous cell division, and disease progression. Targeting G2/M checkpoints has been heavily pursued over the past two decades and has progressed into clinical studies. Recent genome-scale functional genomic studies have revealed that Myt1 kinase, an essential but previously overlooked molecule for the G2/M checkpoint, is a promising target for multiple types of cancers. In this work, we summarize the latest discoveries in molecular targeting of Myt1 kinase and discuss the challenges and limitations in expanding its clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17deb173d71c056d6283f804746aaf46a913ddbb" target='_blank'>
              Myt1 kinase: An Emerging Cell Cycle Regulator for Cancer Therapeutics.
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Javeria Zaheer, Diego Luis Ribeiro, Chunzhang Yang
          </td>
          <td>2025-01-16</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Studying the functional consequences of structural variants (SVs) in mammalian genomes is challenging because (i) SVs arise much less commonly than single-nucleotide variants or small indels and (ii) methods to generate, map, and characterize SVs in model systems are underdeveloped. To address these challenges, we developed Genome-Shuffle-seq, a method that enables the multiplex generation and mapping of thousands of SVs (deletions, inversions, translocations, and extrachromosomal circles) throughout mammalian genomes. We also demonstrate the co-capture of SV identity with single-cell transcriptomes, facilitating the measurement of SV impact on gene expression. We anticipate that Genome-Shuffle-seq will be broadly useful for the systematic exploration of the functional consequences of SVs on gene expression, the chromatin landscape, and three-dimensional nuclear architecture, while also initiating a path toward a minimal mammalian genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd57d0a97983513ebb6d083a8255399d3304d8d" target='_blank'>
              Multiplex generation and single-cell analysis of structural variants in mammalian genomes.
              </a>
            </td>
          <td>
            Sudarshan Pinglay, Jean-Benoît Lalanne, R. Daza, Sanjay Kottapalli, Faaiz Quaisar, Jonas Koeppel, Riddhiman K. Garge, Xiaoyi Li, David S Lee, Jay Shendure
          </td>
          <td>2025-01-31</td>
          <td>Science</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b0e375dc93cf14802320945eec674fcdd4837e" target='_blank'>
              Identification of MORF4L1 as an endogenous substrate of CRBN and its potential role as a therapeutic target in cancer
              </a>
            </td>
          <td>
            Luyao Zhang, Shunfang Liu, Jingliang He, Zhongke Hu, , Hongyi Huang, Qi Gao, Dan Wang, Lu Chen, Xingyu Zhang, Ruotong Liu, Jiayun Wang, Yizhuo Song, , Xiuming Li, Yulu Chen, Xun Zou, Shaojie Ma, Xiujun Wang, Guofeng Xu, Wei Liu, Bin Liu
          </td>
          <td>2025-01-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Ovarian cancer is the seventh most common lethal tumor among women in the world. FOXM1 is a transcription factor implicated in the initiation and progression of ovarian cancer by regulating key oncogenic genes. The role of regulatory regions in regulating the expression of FOXM1 in ovarian cancer is not completely clarified. Treatment with bromodomain and extraterminal (BET) inhibitors JQ-1 and I-BET were explored in ovarian cancer cell lines (OVCAR3, A2780, or SKOV3) to evaluate FOXM1 expression and biological behavior by qPCR, CCK8 assay, colony formation assay, wound-healing, and transwell assays. The regulatory regions (enhancer sequence spanning promoter or exon 1) of FOXM1 were deleted using CRISPR-Cas9 in the OVCAR3 cell line. FOXM1 expression and tumor biological behavior were further assessed in FOXM1 regulatory regions deleted OVCAR3 cell line. The mouse xenograft model was assessed at the indicated time points following subcutaneous injection of enhancer-deleted cells. Treatment with the JQ-1 and I-BET reduced the expression of FOXM1, decreasing cell proliferation, migration, and invasion in a panel of ovarian cancer cell lines including OVCAR3, A2780, and SKOV3 cells. By mining the published ChIP-sequencing data (H3K27Ac) from 12 ovarian cancer cell lines, we identified a potential enhancer and promoter region. Deletion of the spanning enhancer and promoter region of FOXM1 reduced mRNA and protein expression. Similarly, cell proliferation, migration, invasion, and tumorigenesis in both cells and mouse xenograft models were significantly attenuated. Our study demonstrates that JQ-1 and I-BET can regulate the expression of the FOXM1 gene-relating network. These data also indicate that disruption of the span enhancer and promoter region activity of FOXM1 has a vital role in the anti-ovarian cancer effect, hiding a potential opportunity for the evaluation of this non-coding DNA deletion disrupts the FOXM1 transcriptional network in ovarian cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55ebbbb00ccf04834db80407e28cfcaa78405226" target='_blank'>
              Disruption of the FOXM1 Regulatory Region Inhibits Tumor Progression in Ovarian Cancer by CRISPR-Cas9.
              </a>
            </td>
          <td>
            Yujie Chen, Yingzhuo Xue, Qiuwen Jiang, Yunfeng Jin, Weiguan Chen, Minhui Hua
          </td>
          <td>2025-01-01</td>
          <td>Drug development research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tumor suppressor gene TP53 is ubiquitously mutated in ovarian cancer precursor lesions that undergo pervasive accumulation of DNA damage. Prior literature provides evidence that the expression of mutant p53 protein in epithelial cells is associated with increased survival in response to DNA-damaging treatments. Hence, identification and understanding of the mechanisms that, in response to DNA damage accumulation, support survival of ovarian cancer precursor cells carrying p53 mutations might provide important information about the evolution of the disease. Here we used a combination of OC precursor cell models, biochemistry, microscopy, and flow cytometry to provide evidence that the taurine transporter, the SLC6A6 molecule, contributes to cell protection from DNA-damaging (cisplatin) treatment. We found that expression of mutant p53R175H in OC precursor cells, the fallopian tube non-ciliated epithelial (FNE) cells, induced resistance to the DNA-damaging agent cisplatin. Most importantly, shRNA-mediated targeting of SLC6A6 transcript re-sensitized FNE cells expressing mutant p53R175H to cisplatin treatment. Our studies are consistent with the model that the loss of SLC6A6 alters mechanisms involved in the regulation of cell survival in response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbf2e1956ceee35cd1cab8cf25a742bffadc692d" target='_blank'>
              Mutant p53R175H-Associated Protection of Ovarian Cancer Precursor Cells from Cisplatin Requires Expression of SLC6A6 Taurine Transporter
              </a>
            </td>
          <td>
            Teagan Polotaye, Daniel Centeno, Marcin Iwanicki
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite recent advances, improvements to long-term survival in metastatic carcinomas, such as pancreatic or ovarian cancer, remain limited. Current therapies suppress growth-promoting biochemical signals, ablate cells expressing tumor-associated antigens, or promote adaptive immunity to tumor neoantigens. However, these approaches are limited by toxicity to normal cells using the same signaling pathways or expressing the same antigens, or by the low frequency of neoantigens in most carcinomas. Here, we report a fundamentally different strategy for designing safer and more effective anti-cancer therapies through the sensing of cancer-driving biochemical signals and their rewiring to virotherapeutic activation. Specifically, we rationally engineer a RNA vector to self-replicate and cause cytotoxicity in cancer cells exhibiting hyperactive HER2 (ErbB2), but not in normal cells with normal HER2 signaling. Compared to a widely tested virotherapeutic from the same vector family, our hyperactive ErbB2-restricted RNA vector (HERV) exhibits lower toxicity and greater activity against metastatic HER2-positive ovarian cancer in mice, extending survival independently of tumor antigenicity. Most importantly, HERV synergizes with standard-of-care chemotherapy against ovarian cancer metastases in vivo, with 43% of combination-treated subjects surviving for months beyond subjects treated with either therapy alone. Taken together, these results introduce rewiring of cancer-driving signaling pathways to virotherapeutic activation as a strategy for more specific and effective cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519eb18e674bcbdbaf122e51b6002c650afbd522" target='_blank'>
              Rewiring oncogenic signaling to RNA vector replication for the treatment of metastatic cancer
              </a>
            </td>
          <td>
            Xinzhi Zou, Cynthia Zhao, Kevin T. Beier, Chil-Yong Kang, Michael Z. Lin
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d241948b318cfaca392b0e51828be765d7828" target='_blank'>
              Stable minichromosome and functional neocentromere derived from rye 7R chromosome arm
              </a>
            </td>
          <td>
            Zong-xiang Tang, Qian Liu, Zijin Pan, Chang Liu, Jieran Dong, Fang Han, S. Fu
          </td>
          <td>2024-12-18</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Achieving targeted hypermutation of specific genomic sequences without affecting other regions remains a key challenge in continuous evolution. To address this, we evolved a T7 RNA polymerase (RNAP) mutant that synthesizes single-stranded DNA (ssDNA) instead of RNA in vivo, while still exclusively recognizing the T7 promoter. By increasing the error rate of the T7 RNAP mutant, it generates mutated ssDNA that recombines with homologous sequences in the genome, leading to targeted genomic hypermutation. This approach, termed T7 RNAP mutant-assisted continuous evolution (T7ACE), functions effectively in both typical prokaryotic and eukaryotic microorganisms (Escherichia coli and Saccharomyces cerevisiae), achieving targeted hypermutations at rates 2800- and 1200-fold higher than the genomic mutation rates, respectively. Using T7ACE, we successfully evolved an eight-fold increase in tigecycline resistance within 7 days and doubled the efficiency of a xylose utilization pathway in 10 days, demonstrating the efficiency and broad applicability of this single-component tool for continuous evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bff78f49477a4ea5070aa46ecc36f6c1f66021" target='_blank'>
              An evolved, orthogonal ssDNA generator for targeted hypermutation of multiple genomic loci
              </a>
            </td>
          <td>
            Weiran Chu, Rongzhen Tian, Yaxin Guo, , Fabian B H Rehm, Long Liu, Jianghua Li, Guocheng Du, Jian Chen, Yanfeng Liu
          </td>
          <td>2025-01-24</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c26d06bb50d508e1ca0550e77b81ac840e53150" target='_blank'>
              PARP4 deficiency enhances sensitivity to ATM inhibitor by impairing DNA damage repair in melanoma
              </a>
            </td>
          <td>
            Yuehua Li, Yu Liu, Jingjing Ma, Yuqi Yang, Q. Yue, Guannan Zhu, Weinan Guo, Tianwen Gao, Qiong Shi, Chunying Li
          </td>
          <td>2025-01-30</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Protein phosphatases are critical for regulating cell signaling, cell cycle, and cell fate decisions, and their dysregulation leads to an array of human diseases like cancer. The dual specificity phosphatases (DUSPs) have emerged as important factors driving tumorigenesis and cancer therapy resistance. DUSP12 is a poorly characterized atypical DUSP widely conserved throughout evolution. Although no direct substrate has been firmly established, DUSP12 that has been implicated in protecting cells from stress, regulating ribosomal biogenesis, and modulating cellular DNA content. In this study, we used affinity- and proximity-based biochemical purification approaches coupled to mass spectrometry to identify the zinc finger protein ZPR9 as a novel DUSP12 interactor, which was validated by in-cell and in-vitro IP assays. Interestingly, ZPR9 binds to the unique zinc-binding domain of DUSP12, which previous reports indicated was important for many of DUSP12’s functions within the cell. Prior studies had implicated ZPR9 as a modulator of apoptosis, but it remained unclear if and how ZPR9 participated in the cell cycle and, more so, how it promoted cell death. Using mass spectrometry analyses, we found that overexpression of DUSP12 promoted de-phosphorylation of ZPR9 at Ser143. Overexpression of ZPR9, but not Ser143 phosphomimetic and phosphorylation-deficient mutants, led to an increase in pre-metaphase mitotic defects while knockdown of DUSP12 also showed mitotic defects in metaphase. Furthermore, knockdown of DUSP12 promoted, while knockdown of ZPR9 suppressed, stress-induced apoptosis. Our results support a model where DUSP12 protects cells from stress-induced apoptosis by promoting de-phosphorylation of ZPR9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7479ca7476284506ba203b0cbc2b3084cb60b315" target='_blank'>
              DUSP12 promotes cell cycle progression and protects cells from cell death by regulating ZPR9
              </a>
            </td>
          <td>
            Mai Abdusamad, Xiao Guo, Ivan Ramirez, Erick F. Velasquez, Whitaker Cohn, Ankur A. Gholkar, Julian P Whitelegge, Jorge Z. Torres
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Identifying the roles of genes in cancer is critical in discovering potential genetic therapies for cancer care. Translocon-associated protein delta (TRAPδ), also known as signal sequence receptor 4 (SSR4), is a constituent unit in the TRAP/SSR complex that resides in the endoplasmic reticulum and plays a key role in transporting newly synthesized proteins into the endoplasmic reticulumn. However, its biological role in disease development remains unknown to date. This is the first study to identify the role of TRAPδ/SSR4 in colorectal cancer cells in vitro. Upon successful transient knockdown of TRAPδ/SSR4, we observed significant reduction of cell viability in all colorectal cancer cell lines tested. Both HCT 116 and SW480 cell lines were significantly arrested at S and G1 phases, while DLD-1 cells were significantly apoptotic. Moreover, TRAPδ/SSR4 stable knockdown HCT 116 and SW480 cells showed significantly lower viability, anchorage-independent growth, and increased S and G1 phase arrests. Overall, we conclude TRAPδ/SSR4 is a potential oncogene in human colorectal cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c566aa31384f1776417a4c1880816c68b25b7d5" target='_blank'>
              Identification of Translocon-associated Protein Delta as An Oncogene in Human Colorectal Cancer Cells
              </a>
            </td>
          <td>
            Darshika Amarakoon, Wu-Joo Lee, Jing Peng, Seong-Ho Lee
          </td>
          <td>2024-12-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states.
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, , G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P. Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Productive infections of oncogenic human papillomaviruses (HPVs) are closely linked to the differentiation of host epithelial cells, a process that the virus manipulates to create conditions favorable to produce virion progeny. This viral interference involves altering the expression of numerous host genes. Among these, proprotein convertases (PCs) have emerged as potential oncogenes due to their central role in cellular functions. Using RT-qPCR, aberrant PC gene expression was detected across the progression from early HPV infection stages to cancer. These findings demonstrated a progressive disruption of normal PC expression profiles, with FURIN consistently downregulated and other PCs upregulated, transitioning from the episomal stage to neoplastic and carcinoma phenotypes. This pattern of dysregulation was distinct from the broader trends observed in a variety of cancer types through bioinformatic analysis of publicly available transcriptomic datasets, where FURIN expression was predominantly upregulated compared to other PCs. Further bioinformatic investigations revealed a correlation between PC gene expression and cancer phenotype diversity, suggesting a potential link between the loss of normal PC gene expression patterns and the progression of HPV infections toward malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f01d2dc341bbc570cc91f7701d59ca7b10ccc365" target='_blank'>
              Dysregulation of FURIN and Other Proprotein Convertase Genes in the Progression from HPV Infection to Cancer
              </a>
            </td>
          <td>
            Gonzalo Izaguirre, Natalia Zirou, Craig Meyers
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is the most common soft tissue sarcoma in children, but the outcomes of high-grade RMS patients remain poor, underscoring the critical need for novel therapeutic strategies. Although metabolic pathways in RMS are incompletely characterized, emerging evidence suggests that metabolic adaptations in RMS resemble those in other malignancies. Here, we identify elevated cholesterol biosynthesis driven by the PROX1 transcription factor as a defining feature of RMS. Our findings demonstrate that the cholesterol biosynthesis pathway is essential for RMS cell growth, proliferation, and survival. Blocking this pathway through genetic or pharmacological inhibition of the key cholesterol biosynthesis enzymes significantly impairs RMS cell proliferation, halts cell cycle progression, and triggers apoptosis through activation of endoplasmic reticulum stress pathways. We furthermore validate the critical role of cholesterol biosynthesis in RMS progression in tumor xenograft models, demonstrating that silencing of the DHCR7 gene significantly suppresses tumor growth. Transcriptomic analysis revealed widespread downregulation of cell cycle-related genes following DHCR7 silencing, further supporting the role of cholesterol metabolism in cell cycle regulation. These results highlight the vulnerability of RMS cells to cholesterol biosynthesis inhibition and suggest that targeting this metabolic pathway as a promising therapeutic approach for improving RMS outcomes. Our findings provide a rationale for the development of novel therapies targeted to cholesterol biosynthesis in this aggressive cancer. Significance This study reveals that targeting cholesterol biosynthesis in rhabdomyosarcoma induces ER-stress, apoptosis and cell cycle arrest, highlighting a potential therapeutic strategy for treating this aggressive pediatric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1168b7a6afff81a3b6df61f48c435d90b22e7cf3" target='_blank'>
              Targeting de novo cholesterol synthesis in rhabdomyosarcoma induces cell cycle arrest and triggers apoptosis through ER stress-mediated pathways
              </a>
            </td>
          <td>
            N. Y. Gizaw, Kalle Kolari, Kari Alitalo, R. Kivelä
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, , Noemi Andor, Robert A Gatenby, Elsa R. Flores, Christopher J. Whelan, , Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Animal genomes are packaged into chromatin, a highly dynamic macromolecular structure of DNA and histone proteins organised into nucleosomes. This accommodates packaging of lengthy genomic sequences within the physical confines of the nucleus while also enabling precise regulation of access to genetic information. However, histones existed before chromatin and have lesser-known functions beyond genome regulation. Most notably, histones are potent antimicrobial agents, and the release of chromatin to the extracellular space is a defence mechanism nearly as ancient and widespread as chromatin itself. Histone sequences have changed very little throughout evolution, suggesting the possibility that some of their 'non-canonical' functions are at play in parallel or in concert with their genome regulatory functions. In this Review, we take an evolutionary perspective of histone, nuclear chromatin and extracellular chromatin biology and describe the known extranuclear and extracellular functions of histones. We detail molecular mechanisms of chromatin release and extracellular chromatin sensing, and we discuss their roles in physiology and disease. Finally, we present evidence and give a perspective on the potential of extracellular histones to act as bioactive, cell modulatory factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2006f62e95e3010581233345151a9c7075d4f96c" target='_blank'>
              Fantastic proteins and where to find them - histones, in the nucleus and beyond.
              </a>
            </td>
          <td>
            , Noah P Shriever, M. A. Christophorou
          </td>
          <td>2024-12-15</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Among the world's biggest causes of death is cancer. DNA damage is prevented by P53, a transcriptional factor that is encoded by the TP53 gene. A TP53 mutation was found in more than 50% of human cancers. In cancer cell lines with mutant or wild-type p53, CP-31398 has been used to stabilize the active state of p53 and increase p53 activity. Nonetheless, several issues persist, including deficiencies that directly impact the ensuing gene development and the absence of p53 selectivity. In vitro, CP-31398 demonstrates its anti-cancer properties by successfully stabilizing the DNA core domain of the tumor suppressor protein p53. According to the present study, CP-31398 saved the mutant p53 protein's transcriptional activation function by restoring the ability of mice and human tissue culture cells to bind DNA. Future research is thus necessary to fully assess CP-31398's potential. CP-31398 does not, it has been shown in some research, stabilize p53. The DNA-p53 core domain complex has also been seen to be altered and harmed by CP-31398, which is contained in DNA. Moreover, CP-31398 decreased both the His 273 mutant and wild-type p53's sequence-specific DNA binding. CP-3138's therapeutic potential has been shown in a number of in vitro and in vivo cancer models, such as hepatocellular carcinoma (HCC), colorectal cancer, pancreatic cancer, breast cancer, bladder cancer, neck squamous cell carcinoma, etc. In order to facilitate the applications of CP-31398, more research is still needed to comprehend the mechanism of CP-31398 in malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627c38440ba1e99cf398af06c70c6994116a801" target='_blank'>
              Applications of CP-31398 Targeting P53 on Cancers
              </a>
            </td>
          <td>
            Xintong Cai
          </td>
          <td>2025-01-03</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8663834f0e1d3ecd98b4319fc6ee165b1d3d20b" target='_blank'>
              Analysis of mRNA Pentatricopeptide Repeat Domain 1 as a prospective oncogene in clear cell renal cell carcinoma that accelerates tumor cells proliferation and invasion via the Akt/GSK3β/β-catenin pathway
              </a>
            </td>
          <td>
            Z. Zhou, Yulong Li, Y. Chai, Yong Zhang, 
          </td>
          <td>2025-01-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Enhancer of zeste homolog 2 (EZH2) is a methyltransferase involved in cell cycle regulation, cell differentiation, and cell death and plays a role in modulating the immune response. Although it mainly functions by catalyzing the tri-methylation of H3 histone on K27 (H3K27), to inhibit the transcription of target genes, EZH2 can directly methylate several transcription factors or form complexes with them, regulating their functions. EZH2 expression/activity is often dysregulated in cancer, contributing to carcinogenesis and immune escape, thereby representing an important target in anti-cancer therapy. This review summarizes some of the mechanisms through which EZH2 regulates the expression and function of tumor suppressor genes and oncogenic molecules such as STAT3, mutant p53, and c-Myc and how it modulates the anti-cancer immune response. The influence of posttranslational modifications on EZH2 activity and stability and the possible strategies leading to its inhibition are also reviewed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0220899c23d1ac05dd7a74006eb2a6d3a973aaf" target='_blank'>
              Targeting EZH2 in Cancer: Mechanisms, Pathways, and Therapeutic Potential
              </a>
            </td>
          <td>
            M. S. Gilardini Montani, Rossella Benedetti, M. Cirone
          </td>
          <td>2024-12-01</td>
          <td>Molecules</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Biomolecular condensates (BCs), RNA–protein complexes, have emerged as potential therapeutic targets for various cellular pathologies, including cancer. The mechanisms underlying liquid–liquid phase separation rely on the properties of proteins with intrinsically disordered regions and prion-like domains, which drive phase separation in conjunction with their RNA partners. These ribonucleoprotein complexes are distinct in their localization, compartmentalization, epigenetic regulation, and dynamics. BCs are categorized as either nuclear—such as promyelocytic leukemia nuclear bodies, speckles, paraspeckles, and Cajal bodies—or cytosolic, including stress granules (SGs), P bodies, and U bodies. Regulatory RNAs, assembled with protein partners through phase separation, exhibit oncogenic properties and perform key biological functions, including gene transcription, euchromatin/heterochromatin formation, mRNA splicing, mRNA translation, protein compartmentalization, and degradation. Epitranscriptome-modifying enzymes regulate the stability and oncogenic potential of noncoding RNAs (ncRNAs). These RNAs, along with their associated epitranscriptomic and protein modifications, play critical roles in the functioning and dissolution of condensates. Recent advancements in cancer therapy have focused on developing drugs targeting the epitranscriptomic machinery, including the writers, readers, and erasers of RNA modifications. Therapeutic strategies aim to target oncogenic RNAs, tumor-promoting proteins, and RNA–protein interaction domains. Several cancer therapeutic compounds have been developed using the PROTAC and RIBOTAC approaches. Moreover, small molecules targeting protein–RNA interactions and antisense oligonucleotides have been developed. Promising avenues in cancer therapeutics involve the inhibition of ncRNAs and their associated protein complexes, modulation of BCs, regulation of SGs and paraspeckles, and development of small molecule compounds with potential applications across various cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562198ac466da5ff5183f52f8424cad1457ee4aa" target='_blank'>
              Targeting the interplay between biomolecular condensates and regulatory RNAs in cancer
              </a>
            </td>
          <td>
            P. Poltronieri, Sudipta Joardar
          </td>
          <td>2024-12-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56da4ec3ca3ec43d5746b51ede7497a19adee094" target='_blank'>
              Regulatory non-coding somatic mutations as drivers of neuroblastoma.
              </a>
            </td>
          <td>
            Annalaura Montella, Matilde Tirelli, V. A. Lasorsa, Vincenzo Aievola, Vincenza Cerbone, Rosa Manganiello, Achille Iolascon, Mario Capasso
          </td>
          <td>2025-01-23</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is the most prevalent type of malignant epithelial neoplasm that affects the oral cavity. It has been a significant health concern in many countries for a long time since it was usually treated with surgery, radiation, and/or chemotherapy. Drug resistance is the primary issue in patient populations and scientific research, which promotes OSCC tumour cell invasion and migration. Thus, identifying highly specific therapeutic targets could be the potential approach for more successful treatment of OSCC. It is still challenging to understand the genetic causes of oral carcinogenesis due to its highly varied clinic-pathological parameters. It is important to remember that signaling channels and complexes that affect chromatin accessibility control gene expression, which in turn affects cell development and differentiation. Histones undergo post-translational alteration to give this platform. Understanding the processes of gene regulation through histone methylation and its modifications could enhance the early detection, prognostic prediction, and therapy of OSCC. To be properly used as a therapeutic target, histone methylation in OSCC requires more investigation. This review details the dysregulated histone methylation and the modifying enzymes linked to the development and aetiology of OSCC. Furthermore, the part that lysine methylation plays in cell migration, chemo-resistance, and OSCC invasion is also investigated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ac8fe27c84ad3ea1843eef61f7ad80802b8dbb" target='_blank'>
              Molecular insight into histone methylation as a novel target for oral squamous cell carcinoma: future hope in personalised medicine
              </a>
            </td>
          <td>
            P. Rajendran, Monisha Prasad, Enas M. Ali, Ramya Sekar, Abdullah M AlZahrani, M. Karobari, M. A. M. Genena, B. Abdallah
          </td>
          <td>2025-01-27</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="BACKGROUND
α thalassemia/mental retardation syndrome X-linked (ATRX) serves as a part of the sucrose nonfermenting 2 (SNF2) chromatin-remodeling complex. In interphase, ATRX localizes to pericentromeric heterochromatin, contributing to DNA double-strand break repair, DNA replication, and telomere maintenance. During mitosis, most ATRX proteins are removed from chromosomal arms, leaving a pool near the centromere region in mammalian cells, which is critical for accurate chromosome congression and sister chromatid cohesion protection. However, the function and localization mechanisms of ATRX at mitotic centromeres remain largely unresolved.


METHODS
The clustered regularly interspaced short palindromic repeats with CRISPR-associated protein 9 (CRISPR-Cas9) system and overexpression approaches were employed alongside immunofluorescence to investigate the mechanism of ATRX localization at the centromere. To study the binding mechanism between ATRX and heterochromatin protein 1 (HP1), both full-length and truncated mutants of hemagglutinin (HA)-ATRX were generated for co-immunoprecipitation and glutathione S-transferase (GST)-pull assays. Wild-type ATRX and HP1 binding-deficient mutants were created to investigate the role of ATRX binding to HP1 during mitosis, with the Z-Leu-Leu-Leu-al (MG132) maintenance assay, cohesion function assay, and kinetochore distance measurement.


RESULTS AND CONCLUSIONS
Our research demonstrated that HP1α, HP1β, and HP1γ facilitate the positioning of ATRX within the mitotic centromere area through their interaction with the first two [P/L]-X-V-X-[M/L/V] (PxVxL)motifs at the N-terminus of ATRX. ATRX deficiency causes aberrant mitosis and decreased centromeric cohesion. Furthermore, reducing Wapl activity can bypass the need for ATRX to protect centromeric cohesion. These results provide insights into the mechanism of ATRX's centromeric localization and its critical function in preserving centromeric cohesion by reducing Wapl activity in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa2508ee19573d906286f1694bdd429742f1bd10" target='_blank'>
              HP1 Promotes the Centromeric Localization of ATRX and Protects Cohesion by Interfering Wapl Activity in Mitosis.
              </a>
            </td>
          <td>
            Erchen Zhang, Lei Peng, Kejia Yuan, Zexian Ding, Qi Yi
          </td>
          <td>2025-01-21</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogenic gamma herpesviruses, including Epstein–Barr Virus (EBV) and Kaposi’s Sarcoma-associated Herpesvirus (KSHV), are opportunistic cancer-causing viruses and induces oncogenesis through complex mechanisms, which involves manipulation of cellular physiology as well as epigenetic and epitranscriptomic reprogramming. In this review, we describe the intricate processes by which these viruses interact with the epigenetic machinery, leading to alterations in DNA methylation, histone modifications, and the involvement of non-coding RNAs. The key viral proteins such as EBNA1 and LMP1 encoded by EBV; LANA and vGPCR encoded by KSHV; play pivotal roles in these modifications by interacting with host factors, and dysregulating signaling pathways. The resultant reprogramming can lead to activation of oncogenes, silencing of tumor suppressor genes, and evasion of the immune response, which ultimately contributes to the oncogenic potential of these viruses. Furthermore, in this review, we explore current therapeutic strategies targeting these epigenetic alterations and discuss future directions for research and treatment. Through this comprehensive examination of the epigenetic and epitranscriptomic reprogramming mechanisms employed by oncogenic gamma herpesviruses, we aim to provide valuable insights into potential avenues for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d106bf10134bf9b8889820e2db155164b9012c0" target='_blank'>
              Epigenetic and epitranscriptomic regulation during oncogenic γ-herpesvirus infection
              </a>
            </td>
          <td>
            Rajnish Kumar Singh, Ramakrishna Vangala, Atharva S. Torne, Dipayan Bose, 
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic cancer (PC), known as the "king of cancers," is characterized by an exceptionally low five-year survival rate, posing a formidable challenge to global public health. N6-methyladenosine (m6A) methylation is prevalent across various stages of eukaryotic RNA expression, including splicing, maturation, stability, translation, and localization, and represents a pivotal mechanism of epigenetic regulation. m6A methylation influences tumor initiation and progression by modulating post-transcriptional processes, playing a critical role in sustaining cancer cell stemness, promoting cell proliferation, and mediating drug resistance. Extensive research underscores the substantial contribution of m6A modifications to PC development. However, the multiplicity of m6A regulators and their intricate mechanisms of action complicate the landscape. This review aims to deepen the understanding of m6A's role in PC by delineating its involvement in four key areas of tumorigenesis: the hypoxic tumor microenvironment, metabolic reprogramming, immune microenvironment, and resistance mechanisms. Additionally, the review addresses the emerging frontier of m6A interactions with non-coding RNAs (ncRNAs), offering insights into the potential therapeutic and prognostic applications of m6A in the treatment and prognosis prediction of PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323b08ca5467f445ec0f42c55b9835ad0ac4ba47" target='_blank'>
              The Mechanism and Latest Progress of m6A Methylation in the Progression of Pancreatic Cancer
              </a>
            </td>
          <td>
            , Peng Ma, Ying He, , , Ting Fang, Wei Wang, Kaihuan Yu
          </td>
          <td>2025-01-13</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: RAD51C, a critical member of the RAD51 paralog family, is essential for homologous recombination (HR)-mediated DNA repair, a pathway crucial for maintaining genomic stability. Mutations in RAD51C have been linked to cancer susceptibility, particularly in breast and ovarian cancers, where impaired DNA repair mechanisms contribute to genomic instability and tumor progression. Despite its clinical significance, the functional impact of specific RAD51C variants remains poorly understood, necessitating a comprehensive investigation into their biological implications. 
Methods: This study classified RAD51C gene variants into damaging and tolerant categories using computational prediction tools, including SIFT, PolyPhen, CADD, MetaLR, and Mutation Assessor. Variants were prioritized based on consensus scores and classified as high-confidence damaging variants. Correlation and agreement among tools were analyzed to refine predictions. Principal Component Analysis (PCA) and clustering methods were employed to group variants based on prediction patterns. Protein-protein interaction (PPI) networks and pathway enrichment analyses were conducted to contextualize damaging variants within broader biological systems, with a focus on their roles in HR, DNA repair, and cellular processes. 
Results: A total of 2526 variants were analyzed, with damaging variants showing consistent patterns across tools. Consensus scores highlighted 302 high-confidence damaging variants, which were associated with disrupted biological processes, including double-strand break repair via homologous recombination, telomere maintenance, and regulation of cell cycle checkpoints. PPI analysis revealed an interconnected network with 11 nodes and 54 edges, with a clustering coefficient of 0.982, indicating tightly coordinated interactions among DNA repair proteins. Pathway enrichment analyses identified significant associations with homologous recombination (FDR = 2.55E-17) and the Fanconi anemia pathway (FDR = 2.96E-06). 
Conclusion: This study provides a comprehensive framework for assessing the functional impacts of RAD51C variants by integrating computational predictions with biological analyses. The findings underscore the importance of RAD51C in HR and DNA repair pathways, offering insights into its role in genomic stability and cancer progression. These results can inform the prioritization of variants for experimental validation and guide therapeutic strategies targeting DNA repair deficiencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16240c1097223735006d241b04503aff291d133b" target='_blank'>
              Statistical Analysis of Gene Variants for Homologous Recombination Pathways of DNA Repair leading to Cancer Susceptibility
              </a>
            </td>
          <td>
            Usha Adiga, B. Jyoti, P. Reddemma, Alfred J. Augustine, S. Vasishta
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Statistics in Medical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT: Background: Exposure to mutagens causes DNA damage, which, if not repaired properly, can lead to diseases like cancer. Ovarian cancer is a major concern for women globally, including in India, as it is often diagnosed at an advanced stage, making treatment more challenging. Recent research implicates DNA repair proteins like DNA polymerase β (Pol β) in cancer development, emphasising the need to understand these pathways for targeted therapy. This study uses bibliometric analysis to explore ovarian cancer research and DNA repair pathways, providing insights for future research and treatment. Materials and Methods: Data from 37,539 articles related to cancer, ovarian cancer, DNA polymerase β, DNA repair pathways, and inhibitors were analysed from the Dimensions database. Publication distribution, national cooperation, leading authors, and research trends were examined. Results: Variations in publication distribution were observed across journals, with notable contributions from countries like Germany, Canada, and the Netherlands. Prolific authors and institutions were identified, shedding light on the global academic landscape. Co-occurrence analysis revealed thematic clusters, including pathophysiology, cancer risk associations, therapeutic targets, and genomic research. Conclusion: This bibliometric analysis offers valuable insights into ovarian cancer research and DNA repair pathways. It highlights the importance of targeting DNA repair mechanisms in cancer therapy and suggests opportunities for collaboration and personalised medicine. Identifying key trends and future directions aids in advancing our understanding and treatment of ovarian cancer, aiming to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9fa584429563f18cf84fd2ccbda0da4e26ecf1" target='_blank'>
              Advancements in Ovarian Cancer Research: Targeting DNA Repair Mechanisms and the Role of DNA Polymerase β Inhibitors
              </a>
            </td>
          <td>
            A. Patra, Abhishek Samanta, Anindita Chakraborty, Nandan Bhattacharyya
          </td>
          <td>2024-12-20</td>
          <td>Biosciences Biotechnology Research Asia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human gammaherpesvirus 4 or Epstein-Barr virus (EBV) is an oncogenic virus linked to malignancies like gastric adenocarcinoma. Notably, EBV infection induces genetic and epigenetic modifications that play a crucial role in oncogenesis and tumor progression, underscoring the importance of analyzing viral gene expression in the context of gastric cancer (GC) to elucidate its unique characteristics. This study aimed to perform a molecular characterization of EBV gene expression using next-generation sequencing (NGS). The analysis included human gene expression patterns in EBV-positive and EBV-negative samples and the expression of viral genes in EBV-positive samples. The study received approval from the Ethics and Research Committee of João de Barros Barreto University Hospital under reference number 47580121.9.0000.5634. It utilized 76 tumor tissue samples from patients with gastric cancer who had undergone surgical resection, and both fresh and paraffin-embedded samples were gathered for total RNA sequencing (RNA-seq) and in situ hybridization (ISH). The RNA-seq was conducted in a pair-end manner on the NextSeq® platform (Illumina®, US). The NextSeq® 500 MID Output V2 kit - 150 cycles (Illumina®) were utilized following the manufacturer’s instructions. ISH targeting RNA-1 of EBER1 (Y5200, DAKO, Carpinteria) was performed using the automated Dako system. Molecular characterization was conducted using the Kraken2 software. Subsequently, to elucidate the mechanisms through which EBV may influence gastric cancer, we analyzed the patterns of human gene expression in EBV-positive and EBV-negative samples. Of the 76 samples, 8 were classified as EBV-positive according to the applied methodology. Our analysis identified approximately 834 differentially expressed genes, 92 of which exhibited an AUC > 0.85. These genes are implicated in tumor progression, cellular metabolism, and both innate and adaptive immune responses. Additionally, viral genes expressed in the positive samples were evaluated, and we found manifestations of both lytic phase and latent phase genes. Finally, our study presents an efficient strategy for molecular classification of EBV-positive gastric cancer based on NGS and shows the effects of EBV on human gene expression. Author summary In our study, we explored how EBV influences the development of stomach cancer. EBV is a virus known to be linked to various cancers, including gastric cancer, and it can alter the behavior of both human and viral genes within infected cells. To investigate this, we analyzed tissue samples from 76 patients with stomach cancer, focusing on differences between samples with and without EBV. Using advanced sequencing technology, we identified over 800 genes that behave differently in EBV-positive cancers. These genes are involved in critical processes like how cells grow, how the immune system responds, and how energy is produced within cells. We also examined which EBV genes were active in the cancer samples and found evidence of both dormant and active phases of the virus. Our work demonstrated how EBV may contribute to stomach cancer and suggests new ways to classify and understand this disease. By uncovering these details, we hope to pave the way for more targeted treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e59a9ec7d04a548887f690aaa41c832c787b656" target='_blank'>
              Unmasking Epstein-Barr Role in Gastric Carcinogenesis: A Gene Expression Approach to Virus-Positive Tumors
              </a>
            </td>
          <td>
            Valéria Cristiane Santos da Silva, Diego Pereira, Daniel de Souza Avelar, , Ronald Matheus da Silva Mourão, Rubem Ferreira da Silva, Davi Josué Marcon, Leandro Magalhães, A. Vidal, Tatiane Neotti, Ana K M Anaissi, W. Barra, Samir Mansour Casseb, Ândrea Kely Ribeiro dos Santos, S. Demachki, G. Ishak, P. P. de Assumpção, F. Moreira, R. Burbano
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background/Objectives: The highly polymorphic Major Histocompatibility Complex (MHC) genomic region, located on the short arm of chromosome 6, is implicated genetically in Parkinson’s disease (PD), a progressive neurodegenerative disorder with motor and non-motor symptoms. Previously, we reported significant associations between SINE-VNTR-Alu (SVA) expression quantitative trait loci (eQTLs) and Human Leucocyte Antigen (HLA) class I genotypes in PD. In this study, we aimed to evaluate SVA associations and their regulatory effects on transposable element (TE) transcription in the MHC class I region. Methods: Transcriptome data from the peripheral blood cells of 1530 individuals in the Parkinson’s Progression Markers Initiative (PPMI) cohort were reanalyzed for TE and gene expression using publicly available bioinformatics tools, including Salmon and Matrix-eQTL. Results: Four structurally polymorphic SVAs regulated the transcription of 18 distinct clusters of 235 TE loci, comprising LINEs (33%), SINEs (19%), LTRs (35%), and ancient transposon DNA elements (12%) located near HLA genes. The transcribed TEs were predominantly short, with an average length of 445 nucleotides. The regulatory effects of these SVAs varied significantly in terms of TE types, numbers, and transcriptional activation or repression. The SVA-regulated TE RNAs in blood cells appear to function as enhancer-like elements, differentially influencing the expression of HLA class I genes, non-HLA genes, and noncoding RNAs. Conclusions: These findings highlight the roles of SVAs and their associated TEs in the complex regulatory networks governing coding and noncoding gene expression in the MHC class I region, with potential implications for immune function and disease susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7af8d419a6f31034c910704dd0806ace1362d2" target='_blank'>
              The Transcription of Transposable Elements Differentially Regulated by SVAs in the Major Histocompatibility Complex Class I Region of a Parkinson’s Progression Markers Initiative Cohort
              </a>
            </td>
          <td>
            Jerzy K. Kulski, A. Pfaff, S. Kõks
          </td>
          <td>2025-01-06</td>
          <td>Journal of Molecular Pathology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N. Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA topology is a direct consequence of the double helical nature of DNA and is defined by how the two complementary DNA strands are intertwined. Virtually every reaction involving DNA is influenced by DNA topology or has topological effects. It is therefore of fundamental importance to understand how this phenomenon is controlled in living cells. DNA topoisomerases are the key actors dedicated to the regulation of DNA topology in cells from all domains of life. While significant progress has been made in the last two decades in understanding how these enzymes operate in vivo in Bacteria and Eukaryotes, studies in Archaea have been lagging behind. This review article aims to summarize what is currently known about DNA topology regulation by DNA topoisomerases in main archaeal model organisms. These model archaea exhibit markedly different lifestyles, genome organization and topoisomerase content, thus highlighting the diversity and the complexity of DNA topology regulation mechanisms and their evolution in this domain of life. The recent development of functional genomic assays supported by next-generation sequencing now allows to delve deeper into this timely and exciting, yet still understudied topic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4860c875d7d4484e2675f291d10098c1fde66" target='_blank'>
              Regulation of DNA Topology in Archaea: State of the Art and Perspectives.
              </a>
            </td>
          <td>
            Paul Villain, 
          </td>
          <td>2024-12-22</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Variant histone H3.3 is thought to be critical for survival of many cells, since it is deposited in expressed genes, a feature different from core histones. For example, H3.3 deletion leads to embryonic lethality in mice. However, requirement of H3.3 in later stage of development has remained unclear. The aim of this work was to elucidate the role of H3.3 for development of myeloid lineage, important for innate immunity. We conditionally knocked out (cKO) the H3.3 genes in myeloid progenitor cells differentiating into bone marrow derived macrophages (BMDMs). Progenitor cells lacking H3.3 were defective in replication, suffered from extensive DNA damage, and underwent apoptosis. Surviving H3.3cKO cells expressed many interferon stimulated genes (ISGs) throughout differentiation. Further, H3.3cKO BMDMs possessed chromatin accessible sites, and histone posttranslational modifications consistent with the gene expression profiles, Accordingly, H3.3cKO BMDMs retained general nucleosomal structure genome wide. In summary, H3.3 is required for proliferation of myeloid progenitor cells, but is in large part dispensable for differentiation of BMDMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee58f7554d51c36d858c992137d0b639e970461a" target='_blank'>
              Histone H3.3 ensures cell proliferation and genomic stability during myeloid cell development
              </a>
            </td>
          <td>
            Sakshi Chauhan, A. Dey, Todd S Macfarlan, Keiko Ozato
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive and clinically challenging subtype of breast cancer, lacking the expression of estrogen receptor (ER), progesterone receptor (PR), and HER2/neu. The absence of these receptors limits therapeutic options necessitating the exploration of novel treatment strategies. Epigenetic modifications, which include DNA methylation, histone modifications, and microRNA (miRNA) regulation, play a pivotal role in TNBC pathogenesis and represent promising therapeutic targets. This review delves into the therapeutic potential of epigenetic interventions in TNBC, with a focus on DNA methylation, histone modifications, and miRNA therapeutics. We examine the role of DNA methylation in gene silencing within TNBC and the development of DNA methylation inhibitors designed to reactivate silenced tumor suppressor genes. Histone modifications, through histone deacetylation and acetylation in particular, are critical in regulating gene expression. We explore the efficacy of histone deacetylase inhibitors (HDACi), which have shown promise in reversing aberrant histone deacetylation patterns, thereby restoring normal gene function, and suppressing tumor growth. Furthermore, the review highlights the dual role of miRNAs in TNBC as both oncogenes and tumor suppressors and discusses the therapeutic potential of miRNA mimics and inhibitors in modulating these regulatory molecules to inhibit cancer progression. By integrating these epigenetic therapies, we propose a multifaceted approach to target the underlying epigenetic mechanisms that drive TNBC progression. The synergistic use of DNA methylation inhibitors, HDACi, and the miRNA-based therapies offers a promising avenue for personalized treatment strategies, aiming to enhance the clinical outcome for patients with TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f782a5505b5b2b26e898e78e4e01384ddeb62d3" target='_blank'>
              Unlocking the epigenetic code: new insights into triple-negative breast cancer
              </a>
            </td>
          <td>
            G. Mahendran, Ann Dharshika, Ricardo Romero-Moreno, Nadeeshika Wickramarachchige Dona, S. H. G. S. Sarasija, Sumeth Perera, Gayathri N. Silva, Fang Yang, Guochun Zhang, Wickramarachchige Dona, Perera Sarasija, Silva. This
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This paper provides a critical analysis of the molecular mechanisms presently used to explain transcriptional strand asymmetries of single base substitution (SBS) signatures observed in cancer genomes curated at the Catalogue of Somatic Mutations in Cancer (COSMIC) database (Wellcome Trust Sanger Institute). The analysis is based on a deaminase-driven reverse transcriptase (DRT) mutagenesis model of cancer oncogenesis involving both the cytosine (AID/APOBEC) and adenosine (ADAR) mutagenic deaminases. In this analysis we apply what is known, or can reasonably be inferred, of the immunoglobulin somatic hypermutation (Ig SHM) mechanism to the analysis of the transcriptional stand asymmetries of the COSMIC SBS signatures that are observed in cancer genomes. The underlying assumption is that somatic mutations arising in cancer genomes are driven by dysregulated off-target Ig SHM-like mutagenic processes at non-Ig loci. It is reasoned that most SBS signatures whether of “unknown etiology” or assigned-molecular causation, can be readily understood in terms of the DRT-paradigm. These include the major age-related “clock-like” SBS5 signature observed in all cancer genomes sequenced and many other common subset signatures including SBS1, SBS3, SBS2/13, SBS6, SBS12, SBS16, SBS17a/17b, SBS19, SBS21, as well as signatures clearly arising from exogenous causation. We conclude that the DRT-model provides a plausible molecular framework that augments our current understanding of immunogenetic mechanisms driving oncogenesis. It accommodates both what is known about AID/APOBEC and ADAR somatic mutation strand asymmetries and provides a fully integrated understanding into the molecular origins of common COSMIC SBS signatures. The DRT-paradigm thus provides scientists and clinicians with additional molecular insights into the causal links between deaminase-associated genomic signatures and oncogenic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a875e39ec433bc73f24db2e1195da36a00350df" target='_blank'>
              Deaminase-Driven Reverse Transcription Mutagenesis in Oncogenesis: Critical Analysis of Transcriptional Strand Asymmetries of Single Base Substitution Signatures
              </a>
            </td>
          <td>
            Edward J. Steele, R. Lindley
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f02778fd06b36e0f2376644c23b8af48578fee" target='_blank'>
              Purine-rich element binding protein alpha: a DNA/RNA binding protein with multiple roles in cancers
              </a>
            </td>
          <td>
            Shiyi Yu, Chengyang Jiang, Yawen Yang, Fei Cheng, Fangchen Liu, Chang Liu, Xue Gong
          </td>
          <td>2025-01-22</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dc1a99977067c119e44f09575781707edcc62" target='_blank'>
              Interplay of aurora kinase a functional residues and Epstein-barr Nuclear Antigen 1 in Epstein-barr virus associated Gastric cancer using AGS cells
              </a>
            </td>
          <td>
            N. Varshney, Siddharth Singh, M. Kandpal, Vaishali Saini, Erle S. Roberston, Hem Chandra Jha
          </td>
          <td>2025-01-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT This study explored the genomic alterations in Yarrowia lipolytica, a key yeast in industrial biotechnology, under both spontaneous and mutagen-induced conditions. Our findings reveal that spontaneous mutations occur at a rate of approximately 4 × 10−10 events per base pair per cell division, primarily manifesting as single-nucleotide variations (SNVs) and small insertions and deletions (InDels). Notably, C-to-T/G-to-A transitions and C-to-A/G-to-T transversions dominate the spontaneous SNVs, while 1 bp deletions, likely resulting from template slippage, are the most frequent InDels. Furthermore, chromosomal aneuploidy and rearrangements occur, albeit at a lower frequency. Exposure to ultraviolet (UV) light, methylmethane sulfonate (MMS), and Zeocin significantly enhances the rates of SNVs and alters their mutational spectra in distinct patterns. Notably, Zeocin-induced SNVs are predominantly T-to-A and T-to-G substitutions, often occurring within the 5′-TGT*−3′ motif (* denotes the mutated base). Additionally, Zeocin exhibits a higher potency in stimulating InDels compared to UV and MMS. Translesion DNA synthesis is implicated as the primary mechanism behind most Zeocin-induced SNVs and some InDels, whereas non-homologous end joining serves as the main pathway for Zeocin-mediated InDels. Intriguingly, the study identifies the gene YALI1_E21053g, encoding a protein kinase, as negatively associated with Zeocin resistance. Overall, our results not only deepened our knowledge about the genome evolution in Y. lipolytica but also provided reference to develop innovative strategies to harness its genetic potential. IMPORTANCE Yarrowia lipolytica exhibits high environmental stress tolerance and lipid metabolism capabilities, making it a microorganism with significant industrial application potential. In this study, we investigated the genomic variation and evolutionary patterns of this yeast under both spontaneous and induced mutation conditions. Our results reveal distinctive mutation spectra induced by different mutagenic conditions and elucidate the underlying genetic mechanisms. We further highlight the roles of non-homologous end joining and translesion synthesis pathways in Zeocin-induced mutations, demonstrating that such treatments can rapidly confer drug resistance to the cells. Overall, our research enhances the understanding of how yeast genomes evolve under various conditions and provides guidance for developing more effective mutagenesis and breeding techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b4727fb2831092d46e2f4802e9e2fadf0e55a7" target='_blank'>
              Patterns of spontaneous and induced genomic alterations in Yarrowia lipolytica
              </a>
            </td>
          <td>
            Yuan-Ru Xiong, Yuan-Chun Fang, Min He, Keke Li, Lei Qi, Yang Sui, Ke Zhang, Xuechang Wu, Liang Meng, Ou Li, Dao-Qiong Zheng
          </td>
          <td>2024-12-23</td>
          <td>Applied and Environmental Microbiology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef7d2fbf8a9f9c82b98c7984b3782b85218b8319" target='_blank'>
              Crypt density and recruited enhancers underlie intestinal tumour initiation.
              </a>
            </td>
          <td>
            Liam T. Gaynor, Harshabad Singh, Guodong Tie, Krithika Badarinath, Shariq Madha, Andrew Mancini, Swarnabh Bhattacharya, Mikio Hoshino, Frederic J. de Sauvage, Kazutaka Murata, Unmesh Jadhav, Ramesh A Shivdasani
          </td>
          <td>2025-01-08</td>
          <td>Nature</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The strong size increase of the human neocortex is supported both by the amplification and the basal translocation of a neural stem cell population, the basal radial glial cells (or bRG cells). Using live imaging of second trimester human fetal tissue and cortical organoids, we identify two independent translocation modes for bRG cell colonization of the human neocortex. On top of an actomyosin-dependent movement called mitotic somal translocation (MST), we identify a microtubule-dependent motion occurring during interphase, that we call interphasic somal translocation (IST). We show that IST is driven by the LINC complex, through the nuclear envelope recruitment of the dynein motor and of its activator LIS1. Consequently, IST severely altered in LIS1 patient-derived cortical organoids. We also demonstrate that MST occurs during prometaphase and is a mitotic spindle translocation event. MST is controlled by the mitotic cell rounding molecular pathway, via Moesin and Vimentin, driving translocation. We report that 85% of bRG cell translocation is due to IST, for a total movement of 0,67 mm per month of human fetal gestation. Our work identifies how bRG cells colonize the human fetal cortex, and further shows that IST and MST are conserved in bRG-related migrating glioblastoma cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5162f39b5b35fd089916583d60a7b8a2e59a995" target='_blank'>
              Two independent translocation modes drive neural stem cell dissemination into the human fetal cortex
              </a>
            </td>
          <td>
            Ryszard Wimmer, Pauline Lestienne, Clarisse Brunet, L. Coquand, Amandine Di Cicco, Christophe Chehade, A. Artioli, Matthieu Cortes, Anne-Sophie Macé, Xiuyu Wang, Jean-Baptiste Manneville, Bettina Bessière, Ananya Roy, K. Forsberg-Nilsson, Julia Ladewig, Fabien Guimiot, Alexandre D. Baffet
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="DNA methylation is a crucial epigenetic modification that orchestrates chromatin remodelers that suppress transcription, and aberrations in DNA methylation result in a variety of conditions such as cancers and developmental disorders. While it is understood that methylation occurs at CpG-rich DNA regions, it is less understood how distinct methylation profiles are established within various cell types. In this work, we develop a molecular-transport model that depicts the genomic exploration of DNA methyltransferase within a multiscale DNA environment, incorporating biologically relevant factors like methylation rate and CpG density to predict how patterns are established. Our model predicts DNA methylation-state correlation distributions arising from the transport and kinetic properties that are crucial for the establishment of unique methylation profiles. We model the methylation correlation distributions of nine cancerous human cell types to determine how these properties affect the epigenetic profile. Our theory is capable of recapitulating experimental methylation patterns, suggesting the importance of DNA methyltransferase transport in epigenetic regulation. Through this work, we propose a mechanistic description for the establishment of methylation profiles, capturing the key behavioral characteristics of methyltransferase that lead to aberrant methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e458fbb01165e95120272a65d7850681688311a0" target='_blank'>
              Modeling DNA methyltransferase function to predict epigenetic correlation patterns in healthy and cancer cells.
              </a>
            </td>
          <td>
            Ariana Y Tse, A. Spakowitz
          </td>
          <td>2025-01-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Carcinomatosis is one of the leading threats to human public fitness. HNF1B is a critical transcription element in vertebrate proliferation and oncogenesis, which has been shown to play roles in reactive oxygen species (ROS) metabolism. Our previous results have identified HNF1B as a tumor suppressor that could inhibit the malignant progression of prostate cancer. Yet there is no pan-carcinomatosis analysis of HNF1B, which could help us better understand common and unique underlying mechanisms in mankind knubs to enhance novel and competent treatment. Here, in our research, we evaluated the utterance pattern and explored the function of HNF1B in 33 knub categories using the data from the Cancer Genome Atlas Program (TCGA), Gene Expression Omnibus (GEO), and CLNICAL PROTEOMICTUMOR ANALYSIS CONSORTIUM (CPTAC) dataset. We found different HNF1B roles in various cancer types. HNF1B was upregulated in CHOL, STAD, KIRP, and THCA, and was downregulated in GBM, KICH, COAD, KIRC, LUSC, SARC, PAAD, and TGCT. Prognostic analyses indicated that higher HNF1B displayed better illness outcomes in BLCA, READ, and PRAD, while poorer outcomes in LUSC and THYM. HNF1B mutation was most frequent in endometrial cancer but was not associated with disease prognosis. It was discovered that HNF1B utterance relevant to endothelial cell penetration status in BLCA, ESCA, LUAD, LUSC, and TGCT, and carcinomatosis-associated fibroblast infiltration was observed in ESCA, KIRC, LIHC, and TGCT. Moreover, functional enrichment analysis disclosed that metabolism-related functions were implicated in the function of HNF1B. Taken together, our pan- carcinomatosis analysis showed the complicated roles of HNF1B in a variety of carcinomatoses, being able to improve the extensive comprehension of HNF1B's role in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ca455ccbeec3df2b50b09edddce0a592028bda" target='_blank'>
              The role and mechanism of hepatocyte nuclear factor 1β in the occurrence and development of different human tumors: A pan-cancer analysis.
              </a>
            </td>
          <td>
            Liang Li, Haikun Li, , Chunchun Zhao, Fei Wang, Jian Sun, Jianqing Wang
          </td>
          <td>2025-01-31</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e564005ed56626c695c387291b72229a4171b45" target='_blank'>
              Synchronized long-read genome, methylome, epigenome and transcriptome profiling resolve a Mendelian condition.
              </a>
            </td>
          <td>
            Mitchell R. Vollger, Jonas Korlach, K. Eldred, Elliott G. Swanson, Jason G. Underwood, Stephanie C. Bohaczuk, Yizi Mao, Y. H. Cheng, Jane Ranchalis, Elizabeth E. Blue, Ulrike Schwarze, Katherine M. Munson, , Aaron M. Wenger, Aimee Allworth, S. Chanprasert, Brittney L. Duerden, Ian A. Glass, M. Horike‐Pyne, Michelle Kim, K. Leppig, Ian J. McLaughlin, Jessica Ogawa, E. A. Rosenthal, Sam Sheppeard, Stephanie M Sherman, Samuel Strohbehn, Amy L Yuen, Andrew W Stacey, Thomas A Reh, Peter H. Byers, M J Bamshad, F. Hisama, Gail P. Jarvik, Yasemin Sancak, Katrina M. Dipple, A. Stergachis
          </td>
          <td>2025-01-29</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9029d65e0d9d3415e021015d3185275320222a73" target='_blank'>
              G-quadruplex stabilization provokes DNA breaks in human PKD1, revealing a second hit mechanism for ADPKD
              </a>
            </td>
          <td>
            Agata M Parsons, Seth Byrne, Jesse Kooistra, John Dewey, Aaron L Zebolsky, Gloria Alvarado, Gerrit J Bouma, Gregory B Vanden Heuvel, Erik D Larson
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite its relatively recent emergence, single-cell sequencing has cemented its place in scientific research. It has grown exponentially in less than two decades since its start, with broad impact in the biological sciences. The blood represents an attractive system for early development and application of single-cell technologies. As a result, single-cell analyses in blood and leukemia have led the way in describing how cellular heterogeneity affects cancer progression. In this review, we discuss the technological and conceptual advances brought by single-cell genomics, ranging from genetic evolution and differentiation states that mediate drug resistance to the complex interactions required for immunotherapy responses. These high-resolution insights are starting to enter clinical assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fa592467bc471ca2cb2ea46b2951c1d4aa2628" target='_blank'>
              Insights into Leukemia from Single-Cell Genomics
              </a>
            </td>
          <td>
            Nurefsan Sariipek, Peter van Galen
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, M. Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 239


 Background:
 Inactivating genomic alteration (GA) of the TP53 tumor suppressor gene leads to inactivation of p53 protein and is currently the most frequently associated GA across all cancers. Colorectal cancer is a common and lethal subtype of cancer, in the top 5 for incidence, prevalence and cancer-related deaths worldwide. TP53 is the second most frequent GA identified in clinically advanced CRC. It has been postulated that GA in TP53 not only affects the molecular biology of malignant cells but also impacts the tumor microenvironment in CRC. This GA is currently untargetable by approved drugs however, recently a novel approach designed to revert to wild type functioning of TP53 inactivated by Y220C base substitution has gained significant clinical interest.
 Methods:
 67,301 cases of clinically advanced CRC underwent hybrid capture based comprehensive genomic profiling to assess all classes of GA. Cases were sequenced to a mean coverage depth of 650X with microsatellite instability (MSI) status and tumor mutation burden (TMB) determined from the sequencing data and PD-L1 expression measured by immunohistochemistry using the DAKO tumor proportional staining system (low positive set at 1-49% staining and high positive set at ≥50% staining).
 Results:
 422 (0.6%) of the CRC featured the
 TP53
 Y220C GA (Y220C+). When compared with the CRC that were
 TP53
 Y220C negative (Y220C-), the Y220C+ cases were of similar age, gender and number of GA per tumor. The Y220C- cases has significantly higher frequencies of MSI status (6.1% vs 2.2%; p=.0007) and TMB levels greater >10 mutations/Mb (8.7% vs 4.7%; p=.004). The Y220C+ featured lower frequencies of GA in
 PIK3CA
 (12.9% vs 18.6%; p=.002),
 BRAF
 (8.8% vs 10.1%; NS) and
 PTEN
 (7.4% vs 8.3%, NS) and a higher frequency of
 ERBB2
 amplification (5.0% vs 2.7%; p=.004). GA in
 APC
 ,
 KRAS
 ,
 NRAS
 ,
 BRCA2
 ,
 EGFR
 and
 MET
 were similar in both Y220C+ and Y220C- CRC. PD-L1 low expression was similarly infrequent (12.1% to 16.7% range) in both groups.
 Conclusions:
 The potentially targetable
 TP53
 Y220C short variant mutation is uncommon in clinically advanced CRC but is associated with unique genomic and biomarker characteristics. Further research might elucidate subpopulations or CRC subtypes with a greater frequency of this mutation who might benefit from a novel targeted therapy.






 CRC
 TP53
 Y220C+(422 Cases)


 CRC
 TP53
 Y220C-(66,879 Cases)

 P Value




 Gender
 41.2% F/ 58.8% M
 44.4% F/55.6% M
 NS


 Median Age (range)
 62 (23-89+)
 61 (8-89+)
 NS


 GA/tumor
 6.1
 6.5
 NS



 TP53
 non-Y220C GA

 16.1%
 75.7%
 <.0001



 PIK3CA

 12.9%
 18.6%
 .002



 ERBB2

 6.2% (5.0% amp)
 5.1% (2.7% amp)
 .004 (amp)


 MSI High
 2.2%
 6.1%
 .0007


 TMB > 10 mut/Mb
 4.7%
 8.7%
 .004


 PD-L1 Low (1-49%TPS)/ PD-L1 High (>50% TPS)
 16.7% (132 cases)/2.3%
 12.1% (20,759 cases)/1.4%
 NS




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Genetically targeted therapies have proven to be a successful approach to treating cancer because they inhibit the causal (driver) biology of tumors that is absent in healthy cells. While targeted therapies exist for most oncogenes, only a small fraction of loss-of-function (LoF) tumor suppressor genes (TSGs) have been drugged to date, despite TSG mutations causing 2/3 of all cancer. Here we report a novel 1st-in-class genetically targeted opportunity using BET inhibitors to treat EP300 LoF cancers. Among the hundreds of pharmacogenetic screens that we have conducted, this interaction is similar to PARP-BRCA1/HRD in significance, and among the strongest we have discovered. It is cancer-type agnostic and holds up with all potent BET inhibitors in various pre-clinical models, including PDXs. Mechanistically, we demonstrate that apoptosis induced by DNA catastrophe arising from pre-mature entry into S phase plays a significant role. BET inhibition blocks timely assembly of DNA replication factors, which leads to G1 arrest in healthy cells; cancer cells that lack p300, a key regulator of CDK2, and CDK7, proceed through G1-S before replication deficiencies can be resolved. EP300 loss of function is a causal driver in more than 25,000 new cancer cases per year in the US and there currently are no genetically targeted options for these patients. This presentation will focus on the discovery, characterization, and clinical re-entry plan for our BET inhibitor LFB-190, to treat EP300 LoF solid tumors.
 Citation Format: Tomas Babak, Peter Truesdell, Greg Vontz, Peter Heinecke, Doris Coto Villa, Meiou Dai. EP300 loss of function is a pan-cancer sensitizer to BET inhibition. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr PR002">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ac682cf7a63fde34524b1f5679ce5db377d602" target='_blank'>
              Abstract PR002: EP300 loss of function is a pan-cancer sensitizer to BET inhibition
              </a>
            </td>
          <td>
            Tomas Babak, Peter Truesdell, Greg Vontz, Peter Heinecke, Doris Coto Villa, Meiou Dai
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVE
Our study presents a novel analysis of the oncogenes and tumor suppressor proteins directly modulated by E6/E7 of high-risk HPV types 16 and 18, in colorectal cancer (CRC).


METHODS
HCT 116 (KRAS mutant) & HT-29 (TP53 mutant) cell models of CRC were transduced with E6/E7 of HPV16 and HPV18, individually and in combination. Further, we utilized a liquid chromatography mass spectrometry (LC-MS/MS) approach to analyze and compare the proteomes of both CRC cell models.


RESULTS
We generated six stably transduced cell lines. Our data revealed a significantly higher, HPV-induced modulation of oncogenes and tumor suppressor proteins in the TP53 mutant model, as compared to the KRAS mutant model (p ≤ 0.01). Less than 1% of the genes were commonly modulated by HPV, between both models. We also report that HT-29 cells, expressing E6/E7 of both HPV types, significantly reduced the suppression of oncogenes as compared to cells expressing E6/E7 of either HPV types individually (p-value ≤0.00001).


CONCLUSION
Our data imply that HPV coinfections leads to the sustenance of a pro-oncogenic environment in CRC. HPV modulates different oncogenes/tumor suppressor proteins in CRC of varying mutational backgrounds, thus highlighting the importance of personalized therapies for such diseases with mutational heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8413199da82e1f8d1d547e5a890f2a85923ff990" target='_blank'>
              Oncoproteins E6/E7 of the human papillomavirus types 16 & 18 synergize in modulating oncogenes and tumor suppressor proteins in colorectal cancer.
              </a>
            </td>
          <td>
            Queenie Fernandes, Lubna Therachiyil, Shahd M. Younis, , A. Al Moustafa
          </td>
          <td>2025-01-16</td>
          <td>Expert review of proteomics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The relatively slow pace of cortical development in humans has long been a topic of investigation. Studies seeking to understand the underlying mechanisms have mostly focused on neurogenetic comparisons with extant species. Here we ask if developmental tempo differences may have also existed between us and our extinct relatives for whom genomes are available. To do so, we nominate a sapiens-specific derived allele, virtually fixed in contemporary populations, which resides in an enhancer region active during early cortical development. The single nucleotide variant is predicted to significantly affect CHD2 expression, a chromatin remodeler known to play an important role in neural development and for which haploinsufficiency is associated with epilepsy and autism. We leverage patient induced pluripotent stem lines (iPSC) and engineered iPSCs in which we reintroduced the ancestral allele and generated heterozygous loss-of-function mutations. We reveal that CHD2 deficiency impairs lysosomal acidification and autophagosome flux. In contrast, ancestralized lines, which we find express higher levels of CHD2, exhibit enhanced lysosomal function and consequently accelerated autophagosome flux, consistent with our observations in chimpanzee and bonobo lines. This set of findings demonstrates that CHD2 dosage critically regulates the autolysosomal pathway. Through deep phenotyping of cortical organoid and neuron cultures, we show that the CHD2-modulated autolysosomal pathway impacts the timing of developmental programs, acquisition of neuronal functional properties and circuit maturation. Finally, we validate an estrogen-dependent rewiring of CHD2 regulation in the evolution of our lineage, providing a mechanistic understanding of how a single nucleotide variant in a regulatory region contributed to the modern pace of neuronal development and maturation. Together, our findings establish CHD2 as a regulator in setting neurodevelopmental tempo via the autolysosomal pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/904b38fb25bb091c9a26a9d357ebf5fee41222be" target='_blank'>
              CHD2 Dosage Ties Autolysosomal Pathway to Cortical Maturation in Disease and Evolution
              </a>
            </td>
          <td>
            Oliviero Leonardi, Elly I. Lewerissa, Davide Aprile, T. S. Barakat, Lorenza Culotta, Ruizhi Deng, Linda Haazen, K. Linda, Marian Majoie, A. Mordelt, Aurelio Ortale, A. Oudakker, Filippo Prazzoli, S. Puvogel, Nicky Scheefhals, Helenius J Schelhaas, C. Schoenmaker, E. V. van Hugte, H. van Bokhoven, Judith Verhoeven, Alessandro Vitriolo, Cedric Boeckx, G. Testa, N. N. Kasri
          </td>
          <td>2025-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c103a4b54230f99fc3c742a3396a3568699c7f8" target='_blank'>
              H3K56 acetylation regulates chromatin maturation following DNA replication
              </a>
            </td>
          <td>
            Shoufu Duan, I. Nodelman, Hui Zhou, T. Tsukiyama, G. Bowman, Zhiguo Zhang
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a rare aggressive haematological malignancy characterised by the clonal expansion of immature T-cell precursors. It accounts for 15% of paediatric and 25% of adult ALL. T-ALL is associated with the overexpression of major transcription factors (TLX1/3, TAL1, HOXA) that drive specific transcriptional programmes and constitute the molecular classifying subgroups of T-ALL. Although the dysregulation of transcription factor oncogenes is frequently associated with chromosomal translocations in T-ALL, epigenetic dysregulation resulting in changes to post-translational modifications of histones has also been reported. This includes non-coding intergenic mutations that form oncogenic neo-enhancers. This review will focus on the known epigenetically activating intergenic mutations reported in T-ALL, and will discuss the wider implications of neo-enhancer mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12f059393054a743b5ecf3a79dba4386f2b175eb" target='_blank'>
              Neo-enhancers in T-cell acute lymphoblastic Leukaemia (T-ALL) and beyond.
              </a>
            </td>
          <td>
            Charlotte Smith, V. Asnafi, A. Touzart
          </td>
          <td>2025-01-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="


 RNaseH2 plays a critical role in removing mis-bounded RNA from the DNA double strand helix1. Absence of RNaseH2 results in spontaneous double-stranded DNA breaks and cytosolic release of dsDNA2. We have previously shown that epithelial deficiency of RNaseH2b (H2bΔIEC) leads to a p53-dependent stem cell suppression and additional deficiency of p53 results in spontaneous intestinal carcinogenesis3. Due to the central role of cGAS/STING in sensing dsDNA, we aim to investigate the role of epithelial STING in a mouse model of chronic epithelial DNA damage and tumor prevention in this study.



 DNA damage was assessed using comet assay in ModeK cell line with cGAS and STING knock out. Protein kinase activity were assessed by Pamgene Kinase Assay. Intestinal organoids were generated from H2bΔIEC and H2bΔIEC/Tmem173-/-mice. Colony forming assay and EdU staining were performed in all genotypes to assess the impact of STING on epithelial stem cell function and proliferation were assessed by CellTiter Glo assay.
 RNA sequencing was performed in intestinal organoids. Spontaneous carcinogenesis was assessed in 1-year aged mice.



 Loss of RNaseH2b results in activation of cGAS/STING pathway and subsequent upregulation of IFN-I signature. Deficiency of cGAS/STING lead to impaired IFN-I independent DNA damage repair which were accessed by WB and comet assay. During the DNA damage response, STING but not cGAS mediates the P53 pathway and the P21-CDK axis. Organoid EdU stanning displayed restored proliferation in H2bΔIEC/Tmem173-/- compared to H2bΔIEC. RNA sequencing from H2bΔIEC/Tmem173-/-organoid further confirmed decreased G2/M DNA damage checkpoint and P53 signature compared to H2bΔIEC. In vivo, we show that H2bΔIEC/Tmem173-/- mice have downregulated DNA damage repair and developed spontaneous small bowel adenocarcinoma after 53 weeks. Tumor derived from H2bΔIEC/Tmem173-/- mice show sensitivity to CDK inhibitor consistent with in vitro validation.



 We provide evidence that non-canonical role of epithelial STING mediates DNA damage response and guides P21-CDK1 axis in preventing tumor genesis. Our research provided evidence understand the role of STING in DNA damage response and intestine carcinogenesis.



 1.Reijns, Martin A.M., et al., Enzymatic Removal of Ribonucleotides from DNA Is Essential for Mammalian Genome Integrity and Development. Cell, 2012. 149(5): p. 1008-1022.
 2.Bartsch, K., et al., Absence of RNase H2 triggers generation of immunogenic micronuclei removed by autophagy. Hum Mol Genet, 2017. 26(20): p. 3960-3972.
 3.Aden, K., et al., Epithelial RNase H2 Maintains Genome Integrity and Prevents Intestinal Tumorigenesis in Mice. Gastroenterology, 2019. 156(1): p. 145-159 e19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0539a26d1a5ba9dc2728ef562232f66e1b1c39" target='_blank'>
              P0212 Non-canonical STING signaling in intestinal epithelial cells guides DNA damage-driven stem cell function and protects from intestinal carcinogenesis
              </a>
            </td>
          <td>
            H. Xiang, C. Woitaske-Proske, J. Bernardes, G. R. Martini, F. Wottawa, J. Bornhaeuser, N. Kakavand, Y. Guang, Q. Wu, J. Kugler, L. Welz, B. Konukiewitz, C. Peifer, S. Schreiber, M. Tschurtschenthaler, P. Rosenstiel, K. Aden
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer deaths in the United States. New targeted therapies against the once-deemed undruggable oncogenic KRAS are changing current therapeutic paradigms. However, resistance to targeted KRAS inhibitors almost inevitably occurs; resistance can be driven by tumor cell–intrinsic changes or by changes in the microenvironment. Here, we utilized a genetically engineered mouse model of KRASG12D-driven lung cancer that allows for inducible and reversible expression of the oncogene: activation of oncogenic KRASG12D induces tumor growth; conversely, inactivation of KRASG12D causes tumor regression. We showed that in addition to regulating cancer cell growth and survival, oncogenic KRAS regulated the transcriptional status of cancer-associated fibroblasts and macrophages in this model. Utilizing ex vivo approaches, we showed that secreted factors from cancer cells induced the expression of multiple cytokines in lung fibroblasts, and in turn drove expression of immunosuppressive factors, such as arginase 1, in macrophages. In summary, fibroblasts emerged as a key source of immune regulatory signals, and a potential therapeutic target for improving the efficacy of KRAS inhibitors in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdae42d5132ec7ce05cdb9d5dd6bd5ef386a7a36" target='_blank'>
              KRASG12D drives immunosuppression in lung adenocarcinoma through paracrine signaling
              </a>
            </td>
          <td>
            Emily L. Lasse-Opsahl, Ivana Barravecchia, Elyse McLintock, Jen-Mei Lee, Sarah F Ferris, Carlos E Espinoza, Rachael Hinshaw, Sophia Cavanaugh, Marzia Robotti, Lily Rober, Kristee L Brown, Kristena Y. Abdelmalak, C. Galbán, Timothy L. Frankel, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2025-01-09</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1007f695f333241637dbe5b7a88f2f915a6e8136" target='_blank'>
              Resolution of oncogene-induced senescence markers in HPV-infected cervical cancer tissue
              </a>
            </td>
          <td>
            Ashraf I. Khasawneh, Sofian Al Shboul, Nisreen Himsawi, Amani Al Rousan, Nisreen Abu Shahin, Mohammed El-Sadoni, Ahmad Alhesa, Ala' Abu Ghalioun, Suzan Khawaldeh, Bayan Shawish, Salem Abu Mahfouz, Mais Al-Shayeb, Shatha Abo Dawoud, Raghad Tlilan, Mohammad Nuseir, M. Alotaibi, Ola Abu Al Karsaneh, F. Asali, Marcos Yébenes Mayordomo, Raghda Barham, Rame H. Khasawneh, Tareq Saleh
          </td>
          <td>2025-01-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Introduction Breast cancer is one of the most common cancers in women and poses a serious threat to women's health. Circular RNAs (circRNAs) have been found to be specifically expressed in cancers and regulate the growth and death of tumor cells. The role of circRNAs in breast cancer remain unknown. In this study, we explored the impacts of circRNAs on the progression of breast cancer cells. Methods Using bioinformatics analysis, we screened out one up-regulated circRNA in breast cancer, and its function and regulatory mechanisms were confirmed by quantitative real-time PCR, cell counting kit-8 experiment, migration assay, dual luciferase reporter assay, Kyoto Encyclopedia of Genes and Genomes enrichment analysis, cell immunofluorescence, clone formation assay, scratch wound healing experiment, RNA immunoprecipitation and subcutaneous tumor-bearing experiments. Results Circ_0022382 was highly expressed in breast cancer cell lines MDA-MB-231, MCF-7 as well as breast cancer tissues, and promoted the proliferative and migratory capacity of breast cancer cells. In terms of regulatory mechanisms, circ_0022382 activated PI3K/AKT/mTOR signaling pathway and SLC7A11 by sponging let-7a-5p, while knockdown of circ_0022382 contributed to the occurrence of disulfidptosis. In addition, EIF4A3 promoted the expression of circ_0022382 in MDA-MB-231 and MCF-7. Consistently, knockdown of circ_0022382 inhibited the growth of breast cancer cells in vivo. Discussion Circ_0022382 and its related molecules may be effective targets for diagnosis or targeted therapy of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2884bbbc76cab2394ad82e1d267d901c415df5c0" target='_blank'>
              EIF4A3-induced circ_0022382 promotes breast cancer cell progression through the let-7a-5p/PI3K/AKT/mTOR signaling pathway and SLC7A11 axis
              </a>
            </td>
          <td>
            Wei Liu, Jun Zhang, Jiawen Zhang, Yu Ye, Jianqin Zhu, Q. Yu, Tao Li, Xiaochun Sun, Huabiao Chen
          </td>
          <td>2025-01-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background: Epigenetic dysregulation is a common feature of cancer. Promoter demethylation of tumor-promoting genes and global DNA hypomethylation may trigger tumor progression. Epigenetic changes are unstable; thus, research has focused on detecting remedies that target epigenetic regulators. Previous studies have suggested that concordantly targeting hypomethylation and hypermethylation is beneficial for suppressing both the oncogenic and pro-metastatic functions of cancer cells. Therefore, we aimed to investigate the effect of a combination of S-adenosylmethionine (SAM) and the demethylating agent decitabine on prostate cancer cells. Materials and Methods: Prostate cancer cells (PC-3) were treated with SAM, decitabine, or a combination of both. Proliferation, migration, invasion, and methylation assays were also performed. A transcriptome study was conducted to detect different gene clusters between the treatment groups, followed by analyses using the Kyoto Encyclopedia of Genes and Genomes pathway and ingenuity pathway analysis. Finally, to gain information on differential gene expression, promoter methylation studies were performed. Results: Groups treated with decitabine, SAM, or their combination showed reduced proliferative capacity. The decitabine-treated group showed a marginal increase in cell migration and invasion, whereas the SAM-treated and combination treatment groups showed reduced invasion and migration potential. Methylation assays demonstrated the restoration of decitabine-induced demethylation in prostate cancer samples, whereas the transcriptome study revealed the upregulation of different gene clusters between the treatment groups. Methylation studies confirmed that SAM could restore the decitabine-induced demethylation of proto-oncogenes, but it did not induce the re-methylation of tumor-suppressor genes. Conclusions: Combination treatment with SAM and decitabine had an additive effect and did not nullify each other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ece44a8b2382c13d866bb1b93e63772b3f622f" target='_blank'>
              The Simultaneous Treatment of PC-3 Cells with the DNA-Demethylating Agent Decitabine and S-Adenosylmethionine Leads to Synergistic Anticancer Effects
              </a>
            </td>
          <td>
            Thomas Schmidt, Carsten Sticht
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, , Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1708204ce5d02d098867433430233edb7d10ec3" target='_blank'>
              Profiling the epigenome using long-read sequencing.
              </a>
            </td>
          <td>
            Tianyuan Liu, Ana Conesa
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumorigenesis and progression are complex process with multifarious molecules and related mechanisms, while most of them are still elusive. Most research into the mechanisms of cancer today are focusing on encoding RNAs. However, for the past few years as the emergence and development of epigenetics, a kind of new molecule: non-coding RNAs (ncRNAs), RNAs without protein-coding function, have gained prominence in recent scientific discussions. Since the first ncRNA was discovered in 1990s, extensive research has shown that ncRNAs significantly influence processes such as cell growth, differentiation, metabolic regulation, and programmed cell death at both transcriptional and post-transcriptional stages. By means of acting as the tumor suppression and oncogenesis, research has identified several ncRNAs with abnormal expression patterns in cancer cells, marking them as primary oncogenic factors or valuable targets in cancer treatment. This review focus on three kinds of ncRNA, MicroRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs), the molecule structure, generative mechanism, characteristic function and regulatory mechanism in tumor progression. In addition, we will discuss the application value and future expiration of the therapeutic that are focusing on ncRNA, that will open up a new idea and methods for the treatment of cancer for future investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/026f448482a66bffaadbd5d2efa692a3ff825932" target='_blank'>
              Non-coding RNAs in Tumor Biology: Exploring miRNAs, lncRNAs, and circRNAs Roles and Therapeutic Potentials
              </a>
            </td>
          <td>
            Tianhao Zhang
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Motivation Allele-specific methylation (ASM) refers to differential DNA methylation patterns between two alleles at a given locus. This phenomenon is often driven by genetic variants, such as single nucleotide variants (SNVs), which influence methylation in cis by affecting transcription factor or methylation regulator binding, leading to allele-specific differences. Understanding ASM is critical for elucidating gene regulation, as it impacts gene expression and contributes to normal biological variation as well as disease processes, including cancer [1] and autoimmune disorders [2],[3]. Another key driver of ASM is genomic imprinting, an epigenetic mechanism in which gene expression is regulated in a parent-of-origin-specific manner. Imprinted regions, marked during gametogenesis and maintained through cell divisions, are essential for growth, development, and metabolism. Dysregulation of imprinting is associated with developmental and metabolic disorders, such as Prader-Willi and Angelman syndromes, and certain cancers. Detecting ASM across the genome remains challenging due to its tissue- and cell-specific nature and the technical difficulty of phasing reads to assign methylation patterns to specific alleles. Current ASM detection pipelines (e.g., [4])) often require phasing via genetic variants, a computationally intensive process that is limited in regions with low heterozygosity. Results To address these limitations, we developed asms (Allele-Specific Methylation Scanner), a tool designed to detect ASM directly from methylation data without the need for prior phasing. asms offers a faster and complementary approach to uncovering the regulatory effects of ASM, particularly in genomic regions where genetic variants or imprinting play a critical role. For demonstration purposes we leverage the fact that reads generated by Oxford Nanopore (ONT) technology measure sequence and methylation status at once, but the same software can be used with other sequencing technologies. asms can check thousands of loci in a short time. The initial list of loci to examine can be given by the user, or generated by asms through a genomic scan. The asms cluster subcommand separates the reads based on methylation. If phasing results are available, asms can use them to verify whether distinct alleles correspond to distinct base modifications patterns. We benchmark our software using publicly available Ashkenazi trio data [5]. Implementation and availability asms is implemented in rust and python. The software is available at https://github.com/ecmra/asms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7dfc00aae1ea08fcc56a90d449bd2018337af0" target='_blank'>
              ASMS: finding allele specific methylation in human genomes without phasing
              </a>
            </td>
          <td>
            E. Raineri, Miguel Ángel Esteve Marco, Anna Esteve Codina
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Circular RNA (CircRNA)s, a newly discovered type of noncoding RNAs, have been found to play a role in controlling the development and aggressiveness of tumors. Abnormal control of circRNA has been observed in various types of human cancers, including bladder cancer, hepatocellular carcinoma (HCC), breast cancer, and gastric cancer (GC). CircRNAs possess binding sites for microRNAs (miRNAs) and function as miRNA sponges in posttranscriptional regulation. This mechanism has been documented to influence the course of cancer. Significantly, among these putative circRNAs, circular RNA protein tyrosine kinase 2 (circPTK2) exhibited increased expression and displayed a substantial association with adverse clinical characteristics and a negative prognosis. The production of these transcripts occurs via a back-splicing mechanism. The enclosed conformation of circRNAs shields them from destruction and enhances their potential as biomarkers. Gaining insight into the molecular mechanisms involved in these processes would aid in the development of treatment approaches and the discovery of new tumor markers. This article provides a comprehensive assessment of the latest research on the biosynthesis and features of circRNAs. It examines the role of circPTK2 in the diagnosis, treatment, and prognosis evaluation of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdd1e91ef7563824a8ebebdc925d90d66cc38471" target='_blank'>
              CircPTK2 as a Valuable Biomarker and Treatment Target in Cancer.
              </a>
            </td>
          <td>
            Chengqiu Yan, Yu Du, Lihong Cui, Han Bao, Hui Li
          </td>
          <td>2025-01-30</td>
          <td>Journal of biochemical and molecular toxicology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7be5c20b4cd0ad9d6c8146c5936042d6f39bb61" target='_blank'>
              CRYβB2 alters cell adhesion to promote invasion in a triple-negative breast cancer cell line
              </a>
            </td>
          <td>
            Amr A. Waly, London Harper, Jodie M. Fleming, Lindsey M. Costantini
          </td>
          <td>2025-01-21</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Purpose Cell division cycle protein 45 (CDC45) plays a crucial role in DNA replication. This study investigates its role in breast cancer (BC) and its impact on tumor progression. Methods We utilized the GEO database to screen differentially expressed genes (DEGs) and conducted enrichment analysis on these genes. We established a Nomogram model based on CDC45 and other clinical indicators. Additionally, we performed protein-protein interaction (PPI) network construction, drug sensitivity analysis, and immune correlation analysis of CDC45. The function of CDC45 was further verified through cell and animal experiments. Results CDC45 is highly expressed in most tumors, including BC. The expression level of CDC45 was significantly associated with age, sex, race, cancer stage, and molecular subtypes (all p < 0.05). CDC45 was incorporated into a Nomogram model, which showed moderate accuracy in predicting patient prognosis. We also analyzed the co-expression genes of CDC45, including TOPBP1, GINS2, MCM5, GINS1, GINS4, POLE2, MCM2, MCM6, MCM4, and MCM7. Furthermore, CDC45 expression was closely linked to immune infiltration levels, immune checkpoint inhibitors, and the therapeutic response to small molecule drugs. Finally, both in vitro and in vivo experiments confirmed the cancer-promoting effect of CDC45 in BC. Conclusion The expression level of CDC45 is linked to the prognosis, immune infiltration, and drug sensitivity of BC. In vitro and in vivo experiments have confirmed that CDC45 acts as a cancer-promoting protein in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d49e145866fa460069ce8c352c7be711015beb40" target='_blank'>
              Functional Analysis and Experimental Validation of the Prognostic and Immune Effects of the Oncogenic Protein CDC45 in Breast Cancer
              </a>
            </td>
          <td>
            Jia-Ning Zhang, Lin-Wei Li, Man-Qing Cao, Xin Liu, Zi-Lu Yi, Sha-Sha Liu, Hong Liu
          </td>
          <td>2025-01-01</td>
          <td>Breast Cancer : Targets and Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02fb4ad81cdcfd96f458e312f0500d09b4e4efb2" target='_blank'>
              Computational analysis of MYC gene variants: structural and functional impact of non-synonymous SNPs.
              </a>
            </td>
          <td>
            Plabita Bhuyan, , Sangju Basumatary, Aido Lego, Juman Sarma, 
          </td>
          <td>2024-12-14</td>
          <td>Journal of applied genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract HAP1 is a near-haploid human cell line commonly used for mutagenesis and genome editing studies due to its hemizygous nature. We noticed an unusual hypersensitivity of HAP1 to camptothecin, an antineoplastic drug that stabilizes topoisomerase I cleavage complexes (TOP1ccs). We have attributed this hypersensitivity to a deficiency of TDP1, a key phosphodiesterase involved in resolving abortive TOP1ccs. Through whole-exome sequencing and subsequent restoration of TDP1 protein via CRISPR-Cas9 endogenous genome editing, we demonstrate that TDP1 deficiency and camptothecin hypersensitivity in HAP1 cells are a result of a splice-site mutation (TDP1 c.660–1G > A) that causes exon skipping and TDP1 loss of function. The lack of TDP1 in HAP1 cells should be considered when studying topoisomerase-associated DNA lesions and when generalizing mechanisms of DNA damage repair using HAP1 cells. Finally, we also report the generation of HAP1 STAR clones with restored TDP1 expression and function, which may be useful in further studies to probe cellular phenotypes relating to TOP1cc repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7891032e07f96c869d97db1867e4b8a4b2832cd9" target='_blank'>
              TDP1 splice-site mutation causes HAP1 cell hypersensitivity to topoisomerase I inhibition
              </a>
            </td>
          <td>
            Chen Gang Goh, Aldo S. Bader, Tuan-Anh Tran, R. Belotserkovskaya, Giuseppina D'Alessandro, S. P. Jackson
          </td>
          <td>2024-12-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="MicroRNA‑145‑5p (miRNA‑145‑5p) is a short non‑coding RNA located at chromosome 5q33.1, which has gained significant attention in several aspects of cellular regulation and biological functions. In malignant tumours, miRNA‑145‑5p may function as either a tumour suppressor or an oncogene, affecting tumour progression by targeting downstream genes or modulating their expression through upstream regulators. However, the full extent of miRNA‑145‑5p's role in cancer has remained to be determined. This review provides an overview of the role of miRNA‑145‑5p in cancer, investigates its potential as a biomarker for diagnosis, prognosis and treatment response, and evaluates its influence on cancer chemotherapy and radiotherapy. Finally, current strategies for systemic delivery of miRNA‑145‑5p in cancer therapies are summarized.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da59e82a81e22bdead540b3d2f6b358d51bc0d46" target='_blank'>
              Role of miRNA‑145‑5p in cancer (Review).
              </a>
            </td>
          <td>
            Zeshan Chen, Yijue Qin
          </td>
          <td>2025-01-30</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Viral mimicry refers to an active antiviral response triggered by the activation of endogenous retroviruses (ERVs), usually manifested by the formation of double-stranded RNA (dsRNA) and activation of the cellular interferon response, which activates the immune system and produces anti-tumor effects. Epigenetic studies have shown that epigenetic modifications (e.g. DNA methylation, histone modifications, etc.) play a crucial role in tumorigenesis, progression, and treatment resistance. Particularly, alterations in DNA methylation may be closely associated with the suppression of ERVs expression, and treatment by demethylation may restore ERVs activity and thus strengthen the tumor immune response. Therefore, we propose that viral mimicry can induce immune responses in the tumor microenvironment by activating the expression of ERVs, and that epigenetic alterations may play a key regulatory role in this process. In this paper, we review the intersection of viral mimicry, epigenetics and tumor immunotherapy, and explore the possible interactions and synergistic effects among the three, aiming to provide a new theoretical basis and potential strategies for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916b8820f91cde5f4b984b3eba25d2fd40a973b1" target='_blank'>
              Exploring viral mimicry combined with epigenetics and tumor immunity: new perspectives in cancer therapy
              </a>
            </td>
          <td>
            Ruirui Wang, Xin Dong, Xiongjian Zhang, Jinzhuang Liao, Wei Cui, Wei Li
          </td>
          <td>2025-01-06</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18bddf54a0b7a939c01f99932e638ad6022b7e70" target='_blank'>
              CHAF1B promotes the progression of lung squamous-cell carcinoma by inhibiting SETD7 expression.
              </a>
            </td>
          <td>
            Zhuo Zheng, Yongfang Lin, Hua Guo, Zheng Liu, Xiaoliang Jie, Guizhen Wang, Guangbiao Zhou
          </td>
          <td>2025-01-25</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Occurrence of degenerative interactions is thought to serve as a mechanism underlying hybrid unfitness in most animal systems. However, the molecular mechanisms underpinning the genetic interaction and how they contribute to overall hybrid incompatibilities are limited to only a handful of examples. A vertebrate model organism, Xiphophorus, is used to study hybrid dysfunction, and it has been shown from this model that diseases, such as melanoma, can occur in certain interspecies hybrids. Melanoma development is due to hybrid inheritance of an oncogene, xmrk, and loss of a co-evolved tumor modifier. It was recently found that adgre5, a G protein-coupled receptor involved in cell adhesion, is a tumor regulator gene in naturally hybridizing Xiphophorus species Xiphophorus birchmanni (X. birchmanni) and Xiphophorus malinche (X. malinche). We hypothesized that 1 of the 2 parental alleles of adgre5 is involved in regulation of cell growth, migration, and melanomagenesis. Accordingly, we assessed the function of adgre5 alleles from each parental species of the melanoma-bearing hybrids using in vitro cell growth and migration assays. In addition, we expressed each adgre5 allele with the xmrk oncogene in transgenic medaka. We found that cells transfected with the X. birchmanni adgre5 exhibited decreased growth and migration compared to those with the X. malinche allele. Moreover, X. birchmanni allele of adgre5 completely inhibited melanoma development in xmrk-transgenic medaka, while X. malinche adgre5 expression did not exhibit melanoma suppressive activity in medaka. These findings provide evidence that adgre5 is a natural melanoma suppressor and provide new insight in melanoma etiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35473f18fdca14702fd87c87ffcf454b6421341" target='_blank'>
              Functional test of a naturally occurred tumor modifier gene provides insights to melanoma development.
              </a>
            </td>
          <td>
            Mateo Garcia-Olazabal, M. Adolfi, B. Wilde, , Rupesh Paudel, Yuan Lu, Svenja Meierjohann, Gil Rosenthal, Manfred Schartl
          </td>
          <td>2025-01-17</td>
          <td>G3</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Increased telomerase activity has been considered as a conspicuous sign of human cancers. The catalytic cores of telomerase involve a reverse transcriptase and the human telomerase RNA (hTR). However, current detection of telomerase is largely limited to its activity at the tissue and single-cell levels. To reveal the precise distribution of subcellular hTR and telomerase activity, here a modular engineered DNA nanodevice (DNA-ND) is designed capable of imaging hTR and telomerase activity in cytoplasm and nucleus, enabling colocalization analysis. DNA-ND is a modular DNA complex comprising hTR and telomerase activity detection modules, which respectively sense intercellular hTR and telomerase activity via target-sensitive allosteric transition of DNA switches, actuating orthogonal activation of fluorescence signals to achieve in situ co-imaging of hTR and telomerase activity. By integrating DNA-ND with specific localized signals, the DNA-ND based precise profiling of colocalization of hTR and telomerase activity in different cell lines as well as their dynamic changes during pharmacological interventions is demonstrated. Notably, the results suggest that the locations of hTR and telomerase activity are not exactly overlapped, indicating the influence of intracellular environment on the binding of hTR to telomerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5a34a17a7f6af1948fe0851efc3da900886ee2f" target='_blank'>
              A Modular Engineered DNA Nanodevice for Precise Profiling of Telomerase RNA Location and Activity.
              </a>
            </td>
          <td>
            Shiyi Zhang, Jian Lv, Ze-Rui Zhou, Peter X Geng, Dawei Li, Ruocan Qian, 
          </td>
          <td>2024-12-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d173ca4b0414b7ce03e7c7ceb9b88bb092a887d" target='_blank'>
              Chemical inducers of proximity: precision tools for apoptosis in transcriptional regulation
              </a>
            </td>
          <td>
            Kaituo Huang
          </td>
          <td>2024-12-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            , K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract The KRAS gene is nearly ubiquitously subjected to activating mutation in pancreatic adenocarcinomas (PDAC), occurring at a frequency of over 90% in tumors. Mutant KRAS drives sustained signaling through the MAPK pathway to affect frequently disrupted cancer phenotypes including transcription, proliferation and cell survival. Recent research has shown that PDAC tumor growth and survival required a guanine nucleotide exchange factor for RAS homolog family member A (RhoA) called GEF-H1. The GEF-H1 protein, encoded by the ARHGEF2 gene, is a microtubule-associated GEF for RhoA that promotes invasion-migration of PDAC cells via activation of RhoA. Unexpectedly, independent of its RhoGEF activity, GEF-H1 was found to potentiate MAPK signaling by scaffolding protein phosphatase 2A (PP2A) to the kinase suppressor of Ras 1 (KSR-1). In a feedback-dependent manner, enhanced MAPK activity drives expression of ARHGEF2 via regulation of transcription factors ETS and SP, and the RAS responsive element-binding protein 1 (RREB1). RREB1 a negative regulator of ARHGEF2 expression, is downregulated in PDAC cells, which permits sustained expression of GEF-H1 for PDAC tumor survival and subsequent MAPK pathway activation. Given that MAPK targeted therapies show limited clinical efficacy, highlights the need for novel targets. This review describes the unexpected complexity of GEF-H1 function leading to positive feedback that potentiates RAS-MAPK signaling and suggests inhibition of GEF-H1 as a therapeutic strategy for RAS-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f610b92c6965aec299b0a5a0325df9a496fbf4a" target='_blank'>
              Dynamic Coupling of MAPK Signaling to the Guanine Nucleotide Exchange Factor GEF-H1
              </a>
            </td>
          <td>
            Kévin Leguay, Oliver Kent
          </td>
          <td>2025-01-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/774b26878c155e7c22bc9f18e20f8b922e64a4e4" target='_blank'>
              Dissecting the Kaiso binding profile in clear renal cancer cells
              </a>
            </td>
          <td>
            Alexey Starshin, Pavel Abramov, Yaroslava Lobanova, Fedor Sharko, G. Filonova, Dmitry Kaluzhny, D. Kaplun, Igor Deyev, Alexander Mazur, Egor Prokchortchouk, S. Zhenilo
          </td>
          <td>2024-12-19</td>
          <td>Epigenetics & Chromatin</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6666f30aff9bcd1de0a39ac7dc32869d01658a2" target='_blank'>
              Integrated analysis of single-cell and bulk-RNA sequencing for the cellular senescence in prognosis of lung adenocarcinoma
              </a>
            </td>
          <td>
            Fengqiang Yu, Liangyu Zhang, Xun Zhang, Jianshen Zeng, Fancai Lai
          </td>
          <td>2025-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Amidst the prevalent and notable characteristic of a hypoxic microenvironment present in the majority of solid tumors, a burgeoning number of studies have revealed the significance of noncoding RNAs (ncRNAs) in hypoxic tumor regions. The transcriptome of cancers is highly heterogeneous, with noncoding transcripts playing crucial roles. Long noncoding RNAs (lncRNAs) and circular RNAs (circRNAs) are two distinctive classes of ncRNA that are garnering increasing attention. Biologically, they possess intriguing properties and possess significant regulatory functions. Clinically, they present as promising biomarkers and therapeutic targets. Additionally, recent research has evaluated the clinical applications of these ncRNAs in RNA-based treatments and noninvasive liquid biopsies. This review provides a comprehensive summary of recent studies on lncRNAs and circRNAs within the hypoxic tumor microenvironment. Furthermore, the clinical significance of lncRNAs and circRNAs in cancer diagnosis and treatment is emphasized, which could pave the way for the development of effective targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/334eb766e645372ee9515024ced95b053cc01f29" target='_blank'>
              Landscape of Noncoding RNA in the Hypoxic Tumor Microenvironment
              </a>
            </td>
          <td>
            Lianfeng Gong, Chuanxin Zou, Haixia Zhang, Fei Yang, Gui Qi, Zhaowu Ma
          </td>
          <td>2025-01-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f33f540ba07c7d118037674f121aaa35d22c9a7" target='_blank'>
              Prolonged persistence of mutagenic DNA lesions in somatic cells.
              </a>
            </td>
          <td>
            M. Spencer Chapman, E. Mitchell, Kenichi Yoshida, N. Williams, Margarete A. Fabre, A. Ranzoni, Philip S. Robinson, Lori D. Kregar, Matthias Wilk, Steffen Boettcher, Krishnaa Mahbubani, Kourosh Saeb Parsy, K. Gowers, Sam M Janes, Stanley W K Ng, Matt Hoare, Anthony R Green, G. Vassiliou, A. Cvejic, M. Manz, E. Laurenti, I. Martincorena, Michael R Stratton, J. Nangalia, Tim H. H. Coorens, P. J. Campbell
          </td>
          <td>2025-01-15</td>
          <td>Nature</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="Telomeres, the protective caps at the ends of human chromosomes, shorten with each round of cell division, representing a counter in the form of a biological clock. Beyond 50 to 60 divisions, the protective function of the telomeres will become unsustainable, and cells will usually experience senescence and death. Loss of cell function is manifested in the form of aging and the onset of diseases, including cancer. Cancer cells have found a way around this by overexpressing an enzyme, called telomerase, which counteracts the telomere shortening, thus allowing the neoplastic cells to keep replicating. This narrative review outlines why telomeres undergo shortening and how cancer cells exploit and take advantage of that phenomenon. The fact that malignant cells derail the biological telomere clock could be targeted for therapeutic benefit. The review also highlights the diverse telomere-based strategies explored in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0525525651060dbbe10ba3d71b4ebb15d3a5fe92" target='_blank'>
              Telomere Length and Cancer
              </a>
            </td>
          <td>
            Ismail Ibrahim Al-Janabi
          </td>
          <td>2025-01-09</td>
          <td>Al-Rafidain Journal of Medical Sciences ( ISSN 2789-3219 )</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="To achieve replicative immortality, cancer cells must activate telomere maintenance mechanisms. In 10 to 15% of cancers, this is enabled by recombination-based alternative lengthening of telomeres pathways (ALT). ALT cells display several hallmarks including heterogeneous telomere length, extrachromosomal telomeric repeats, and ALT-associated PML bodies. ALT cells also have high telomeric replication stress (RS) enhanced by fork-stalling structures (R-loops and G4s) and altered chromatin states. In ALT cells, telomeric RS promotes telomere elongation but above a certain threshold becomes detrimental to cell survival. Manipulating RS at telomeres has thus been proposed as a therapeutic strategy against ALT cancers. Through analysis of genome-wide CRISPR fitness screens, we identified ALT-specific vulnerabilities and describe here our characterization of the roles of SUB1, a ssDNA-binding protein, in telomere stability. SUB1 depletion increases RS at ALT telomeres, profoundly impairing ALT cell growth without impacting telomerase-positive cells. During RS, SUB1 is recruited to stalled forks and ALT telomeres via its ssDNA-binding domain. This recruitment is potentiated by RPA depletion, suggesting that these factors may compete for ssDNA. The viability of ALT cells and their resilience toward RS also requires ssDNA binding by SUB1. SUB1 depletion accelerates cell death induced by FANCM depletion, triggering unsustainable levels of telomeric damage in ALT cells. Finally, combining SUB1 depletion with RS-inducing drugs rapidly induces replication catastrophe in ALT cells. Altogether, our work identifies SUB1 as an ALT susceptibility with roles in the mitigation of RS at ALT telomeres and suggests advanced therapeutic strategies for a host of still poorly managed cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6497c277f2efbf01c25fef9e39f419c5577c8b02" target='_blank'>
              The single-stranded DNA-binding factor SUB1/PC4 alleviates replication stress at telomeres and is a vulnerability of ALT cancer cells.
              </a>
            </td>
          <td>
            Jean-Christophe Dubois, Erin Bonnell, Amélie Filion, Julie Frion, Samuel Zimmer, Muhammad Riaz Khan, Gabriela M. Teplitz, Lisa Casimir, Élie Méthot, , Mouhamed Idrissou, Pierre-Étienne Jacques, , Alexandre Maréchal
          </td>
          <td>2025-01-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Pituitary neuroendocrine tumors (PitNETs) are unusual among neoplasms in that sporadic tumors are not typically associated with genetic mutations. Instead, epigenetics, a non-mutational process by which gene expression is modified via a variety of mechanisms, may be a driving factor in PitNET growth and behavior. DNA methylation is one of the most well understood forms of epigenetic modification. Research on DNA methylation profiles of PitNETs has identified a large number of genes silenced or upregulated by DNA methylation, particularly when methylated at CpG islands of gene promoter regions. Global patterns of DNA methylation may provide valuable insight into origins of pituitary adenoma subtypes, assist with PitNet diagnostics and have been found to correspond to the current WHO classification of PitNETs based on transcription factor lineage. Analysis of differentially methylated regions of individual genes may have prognostic value as well as guide research toward nonsurgical therapeutic strategies. Pituitary epigenetics and DNA methylation analysis are rapidly growing areas of interest with the potential to shape the future of pituitary tumor diagnostics and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91afb11897061f0dd790a1feb3aca845da9bfbb" target='_blank'>
              Current understanding of the role of DNA methylation in pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Racheal Peterson, David J Cote, G. Zada
          </td>
          <td>2025-01-02</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Ovarian cancer (OC) is the sixth most common malignancy in women and the poor 5-year survival emphasises the need for novel therapies. NK cells play an important role in the control of malignant disease but the nature of tumour-infiltrating and peripheral NK cells in OC remains unclear. Methods Using flow cytometric analysis, we studied the phenotype and function of NK cells in blood, primary tumour and metastatic tissue in 80 women with OC. The cell type contexture of metastatic OC tissue was explored utilising scRNAseq analysis, with a focus on portraying an immunogenic tumour microenvironment and determining the characteristics of a dysfunctional NK cell population. Results The proportion of peripheral NK cells was markedly elevated with a highly activated profile and increased cytotoxicity. In contrast, NK cell numbers in primary tumour and metastasis were substantially reduced, with downregulation of activatory receptors together with elevated PD-1 expression. scRNA-Seq identified 5 NK cell subpopulations along with increased exhausted and immature NK cells within tumour tissue compared to normal tissue. These features were attenuated following chemotherapy where higher levels of activated and cytotoxic NK cells associated with improved disease-free survival. Correlation of NK cell phenotype with clinical outcomes revealed high levels of DNAM-1 expression on tissue-localised and peripheral NK cells to be associated with reduced survival. Expression of PVR, the DNAM-1 ligand, was significantly increased on tumours and DNAM-1 mediated NK cell lysis of primary tumour tissue was observed in vitro. Discussion These findings reveal profound modulation of the tumour tissue and systemic profile of NK cells which likely contributes to the high rates of local progression and metastasis seen with OC. Immunotherapeutic approaches that overcome local immune suppression and enhance DNAM-1-targeted lysis of OC offer the potential to improve disease control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec3263f862b7dea4d3ddaf31f9b8ccd7a6ea974" target='_blank'>
              The emergence of DNAM-1 as the facilitator of NK cell-mediated killing in ovarian cancer
              </a>
            </td>
          <td>
            R. Pounds, W. Croft, Hayden Pearce, Tasnia Hossain, Kavita Singh, J. Bálega, D. Jeevan, S. Sundar, S. Kehoe, J. Yap, Paul Moss, J. Zuo
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024', '2025'],
    y: [2, 3, 6],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>