<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-21 09:36:55 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>36</td>
          <td>104</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>73</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>145</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>341</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>299</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a dynamic and versatile genomic element with key roles in physiological regulation and disease pathology. This review synthesizes current knowledge on eccDNA, covering its classification, biogenesis, detection methods, biological functions, and clinical implications. Once considered rare anomalies, eccDNAs are now recognized as major drivers of oncogene amplification, genomic plasticity, and therapeutic resistance, particularly in cancer. EccDNA subtypes such as microDNA, double minutes, and ecDNA are defined by their structural, genomic, and pathological features. EccDNAs originate through diverse mechanisms including DNA repair, chromothripsis, breakage fusion bridge cycles, and apoptosis, occurring in both normal and stressed cells. Advances in long-read and single-cell sequencing, CRISPR-based synthesis, and computational tools have improved detection and functional analysis. Functionally, eccDNAs contribute to transcriptional amplification, activate immune responses through cGAS-STING signaling, and facilitate intercellular communication. They are found across a range of tissues and disease states—including cancer, cardiovascular, neurological, autoimmune, and metabolic disorders—and serve as both biomarkers and regulatory elements. We introduce the concept of the stress selection theory, which proposes eccDNA as an adaptive reservoir that enhances cellular fitness in response to environmental and therapeutic pressures. Despite growing insights, challenges remain in understanding tissue-specific roles, achieving clinical translation, and standardizing methodologies. Emerging tools in multi-omics, spatial biology, and artificial intelligence are expected to drive future breakthroughs in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31cb2df8f26047d209cf38790b651f7df3921fd" target='_blank'>
              Extrachromosomal circular DNA drives dynamic genome plasticity: emerging roles in disease progression and clinical potential
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-05-25</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="The role of extrachromosomal DNA (ecDNA) in lung cancer, particularly in subjects who never smoked (LCINS), remains unclear. Examination of 1,216 whole-genome-sequenced lung cancers identified ecDNA in 18.9% of patients. Enrichment of MDM2 and other oncogenes’ amplification via ecDNA possibly drives a LCINS subset. Tumors harboring ecDNA or other focal amplifications showed similarly poor survival. A strong association with whole-genome doubling suggests most ecDNA reflects genomic instability in treatment-naïve lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20bf29ecb0d2ae5c2ac96f25fe4450672918b4ba" target='_blank'>
              Examining the Role of Extrachromosomal DNA in 1,216 Lung Cancers
              </a>
            </td>
          <td>
            Azhar Khandekar, P. Hoang, J. Luebeck, M. Díaz-Gay, Wei Zhao, John P. McElderry, Caleb Hartman, Mona Miraftab, Olivia W. Lee, Kara M. Barnao, E. Bergstrom, Yang Yang, Martin A. Nowak, Nathaniel Rothman, Robert J Homer, S. Yang, Qing Lan, David C Wedge, Lixing Yang, Stephen J Chanock, Tongwu Zhang, Ludmil B. Alexandrov, Marion Landi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chromothripsis, the chaotic shattering and repair of chromosomes, is common in cancer. Whether chromothripsis generates actionable therapeutic targets remains an open question. In a cohort of 64 patients in blast phase of a myeloproliferative neoplasm (BP-MPN), we describe recurrent amplification of a region of chromosome 21q (‘chr. 21amp’) in 25%, driven by chromothripsis in a third of these cases. We report that chr. 21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. DYRK1A, a serine threonine kinase, is the only gene in the 2.7-megabase minimally amplified region that showed both increased expression and chromatin accessibility compared with non-chr. 21amp BP-MPN controls. DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development and is essential for BP-MPN cell proliferation in vitro and in vivo, and represents a druggable axis. Collectively, these findings define chr. 21amp as a prognostic biomarker in BP-MPN, and link chromothripsis to a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4397c5f80ce07fe2ffd98add28275e15180327b" target='_blank'>
              Chromothripsis-associated chromosome 21 amplification orchestrates transformation to blast-phase MPN through targetable overexpression of DYRK1A
              </a>
            </td>
          <td>
            Charlotte K Brierley, B. Yip, G. Orlando, J. Wen, S. Wen, H. Goyal, M. Levine, G. M. Jakobsdottir, Avraam Tapinos, A. Cornish, A. Rodriguez-Romera, A. Rodriguez-Meira, M. Bashton, A. Hamblin, SA Clark, Joseph C Hamley, Olivia Fox, M. Giurgiu, Jennifer O'Sullivan, L. Murphy, A. Adamo, Aude-Anais Olijnik, A. Cotton, Emily Hendrix, S. Narina, S. Pruett-Miller, A. Enshaei, Claire N Harrison, Mark Drummond, S. Knapper, A. Tefferi, I. Antony-Debré, James O. J. Davies, A. Henssen, S. Thongjuea, David C Wedge, S. Constantinescu, E. Papaemmanuil, B. Psaila, John D. Crispino, Adam J Mead
          </td>
          <td>2025-06-09</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6865d140b6544d7dfc0dca54fb01a54fdf212f1a" target='_blank'>
              Mitotic BLM functions are required to maintain genomic stability
              </a>
            </td>
          <td>
            Tamara Eleanore Hamann, A. Wieland, Andrea Tirincsi, Kruno Vukušić, Farbod Mohseni, René Wardenaar, Marialucrezia Losito, Ipek Ilgin Gönenc, B. Wollnik, F. Foijer, I. Tolic, Zuzana Storchová, M. Räschle
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Testicular germ cell tumors (TGCTs) are the most common malignancy in young men, exhibit a unique developmental origin and exceptional chemosensitivity. However, the molecular distinctions between TGCT subtypes remain poorly understood. Here we present a comprehensive genomic analysis of 252 treatment-naive primary TGCTs, integrating deep whole-genome sequencing with matched transcriptomic and epigenomic data. We identify new driver genes and uncover defining features of TGCTs, including pervasive chromosome X amplification with subtype-specific X chromosome inactivation, and a germ cell-like transcriptional program. Although previously reported, whole genome doubling (WGD) in TGCTs is further characterized here as ubiquitous, developmentally early, and associated with age at onset. Seminomas are enriched for early driver mutations, secondary WGD events, sustained XIST expression and replication stress-associated indel mutational signatures, while non-seminomas show greater structural complexity, subclonal diversity, relatively earlier-onset WGD, extended tumor latency, and telomere elongation. Moreover, we identify a mutational signature, SBS87, that is exceptionally rare across cancers with exception of thiopurine-treated leukemia, but strikingly prevalent in TGCT, especially non-seminomas. SBS87 is linked to extended tumor latency and telomere elongation, implicating possible environmental or endogenous processes that mimic thiopurine-induced DNA damage in TGCT pathogenesis. Collectively, our findings define TGCTs as molecularly distinct tumors shaped by early genomic instability and highlight SBS87 as a novel mutational footprint with potential etiologic and clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="ABSTRACT Aims & methods DNA methylation and genomic instability are critical drivers of cancer initiation and malignant progression. However, the roles of methylation aberrations and genomic instability in malignant progression have not been thoroughly investigated in intrahepatic cholangiocarcinoma (ICC). To address this, we identified differentially methylated regions (DMRs) and somatic copy number alterations (SCNAs) from 341 ICC samples across various stages. Results Our findings revealed that stages IAIB, II, IIIA, and IIIB exhibited comparable methylation changes, whereas stage IV ICC showed a pronounced accumulation of stage-specific methylation alterations. Leveraging these findings, we developed a classification model that effectively distinguished stage IV ICC from earlier stages with high accuracy using 15 DMRs. Furthermore, stage IV ICC exhibited slightly higher genomic instability, including an elevated aneuploidy score and a greater proportion of focal amplifications. We also observed a positive correlation between SCNA burden and DNA methylation entropy in the promoter, gene body, and CpG island regions, with the gene body of MDM2 serving as a notable example. Conclusions These findings highlight the potential of DNA methylation as a biomarker for metastasis diagnosis and the interplay between local genomic instability and aberrant methylation, emphasizing their synergistic roles in driving the evolutionary trajectory of ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d995c84a0cfeeb8c871fd5bd4ce5a040e83eca26" target='_blank'>
              Methylation aberrations and genomic instability synergistically drive the evolution of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Guanghao Li, Youhuang Bai, Feng Tao, Tingting Hu, Ting Wang, Yong Zeng, Deqiang Sun
          </td>
          <td>2025-06-17</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842e0482ffc6f8bbb0e4561726b21f49e5e47e0b" target='_blank'>
              An Enhancement of Extrachromosomal Circular DNA Enrichment and Amplification to Address the Extreme Low Overlap Between Replicates
              </a>
            </td>
          <td>
            Charles M. Burnham, Alongkorn Kurilung, Visanu Wanchai, Birgritte Regenberg, J. Delgado-Calle, Alexei G. Basnakian, I. Nookaew
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) is among the most frequently diagnosed cancer types, and affected patients frequently experience poor prognostic outcomes and high mortality rates. Many genomic studies of ESCC have been performed in recent years, yet the mutational mechanisms driving ESCC and their clinical implications remain incompletely understood. In this study, paired tumor and normal tissue samples from 22 patients with ESCC were used for whole genome sequencing-based analyses of genome-wide mutational events. These comprehensive analyses enabled the detection and characterization of various mutation subtypes in ESCC including somatic single-nucleotide variants, small insertions and deletions, copy number variations, structural variations, and circular extrachromosomal DNA. Of identified genes harboring non-silent mutations, TP53, NOTCH1, CSMD3, EP300, and FAM135B were the most frequently mutated genes in this study and they were annotated in the COSMIC Cancer Gene Census. With the exception of aging-related signatures, an APOBEC-associated mutational signature was the dominant mutational feature detected in ESCC samples, suggesting that APOBEC-mediated cytidine deamination is likely a major driver of mutations in this cancer type. Notably, our study also detected circular extrachromosomal DNA (ecDNA) events in these ESCC patient samples. The oncogenes COX6C, PVT1, and MMP12 as well as the oncogenic long non-coding RNA AZIN1-AS1 which were detected in ecDNA regions in these analyses may be associated with worse disease-free survival in ESCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55745d3238cc107d72d837356b7050823d1fc40" target='_blank'>
              Whole genome sequencing revealed esophageal squamous cell carcinoma related biomarkers
              </a>
            </td>
          <td>
            Mingjun Li, Lei Li, Xizi Wang, Yanwei Zhao, Peina Du, Wei Wang, , Yadong Wang, Yanxing Sheng, Mingliang Gu, Xiaodong Jia
          </td>
          <td>2025-06-26</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cfe224e356c6e68ceccf6ec7ce94377f472868" target='_blank'>
              KLF5 controls subtype-independent highly interactive enhancers in pancreatic cancer to regulate cell survival
              </a>
            </td>
          <td>
            Thomas L. Ekstrom, Zhangshuai Dai, Julia Thiel, Meghana Manjunath, Nadine Schacherer, Bishakha Joyeeta Saha, Yara Souto, A. Abdelrahman, Frank Essmann, Sven Beyes, M. Truty, Meng Dong, Steven A. Johnsen
          </td>
          <td>2025-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 3090


 Background:
 Recurrent cytogenetic abnormalities represent candidate therapeutic targets for children with neuroblastoma (NB).
 MYCN
 oncogene amplification is associated with significantly worse survival rates for children with NB and remains one of the primary predictors of patient prognosis.
 MYCN
 amplifications in NB can be found both within the linear genome and on circular extrachromosomal DNA (ecDNA), and therapeutic targeting of the mechanisms underlying
 MYCN
 amplification represents a novel and promising strategy in NB. However, the molecular features and clinical and biological significance of these amplifications in NB tumors are not sufficiently understood.
 Methods:
 Whole genome and RNA sequencing data were analyzed for NB cell lines and NCI TARGET NB samples using AmpliconSuite software for ecDNA identification and characterization. GISTIC was used for identification of recurrently amplified regions. Gene expression levels were determined using StringTie, and gene clustering heatmaps were generated using FeatureCounts software. For differential gene expression analyses, samples were divided into ecDNA
 +
 and ecDNA
 -
 , and genes contained on ecDNA were compared to the same regions on linear DNA across samples using DESeq2. Associations between ecDNA quantity, content, and patient survival were performed using multivariate Cox regression survival analysis. Associations of gene expression with patient survival were performed using the R2 Platform. The efficacy of targeting ecDNA-associated gene products was assessed using live cell imaging and cell viability assays.
 Results:
 WGS analysis confirmed 7/20 NB patient tumors from the TARGET database to be ecDNA amplified with 1-5 independent ecDNA elements and
 MYCN
 gene expression correlated with the ecDNA copy number. ecDNAs in
 MYCN
 -amplified neuroblastoma cell lines contained distinct gene combinations and possessed unique structures. MYCN overexpression in NB cells has been shown to be associated with replication stress (RS), and tumor cells containing ecDNA are hyper-reliant on the DNA damage response (DDR) kinase CHK1 to manage heightened replication stress. Expression of the
 CHK1
 gene was associated with neuroblastoma patient outcomes and neuroblastoma was most significantly associated with
 CHK1
 RNA dependency. We further validated CHK1i as a promising therapeutic strategy in
 MYCN
 amplified NB, as CHK1 inhibition with the novel inhibitor BBI-2779 was most effective against ecDNA+,
 MYCN
 -amplified neuroblastoma cell lines.
 Conclusions:
 Our results emphasize the critical role of ecDNA in NB. We identify a synthetic lethality axis shaped by ecDNA
 MYCN
 amplification and CHK1 dependence. We further demonstrate the feasibility of targeting this vulnerability through CHK1 inhibition, thus offering new avenues for treatment in
 MYCN
 amplified tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e222ec29cad608a8fd4761ccb837028a1803c8a" target='_blank'>
              Effect of extrachromosomal DNA (ecDNA) on
 MYCN
 amplified neuroblastoma and patient outcomes.
              </a>
            </td>
          <td>
            Mihika Sonalkar, Carla Sampaio, Ting Yang, Taelor Getz, Catherine Shaw, Vicky Pham, Joshua Lange, Sudhir Chowdhry, Christian A Hassig, L. Alexandrov, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10436da9e9bcdf67b6e4cfcf17b6e58df4672a34" target='_blank'>
              Spatio-temporal chromosomal arrangements by late-replicating heterochromatin
              </a>
            </td>
          <td>
            Satyadev Polisetty, Shuvadip Dutta, Rakesh Netha Vadnala, D. Notani, R. Padinhateeri, K. Sanyal
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is recognized for its aggressiveness, yet the mechanism underlying metastasis remains unclear. Here, we report that CREPT/RPRD1B, which exhibits somatic gene copy-number amplifications and elevated expression, correlates with poor patient survival and drives TNBC metastasis. We demonstrate that CREPT alters three-dimensional genome structures in topologically-associating domain (TAD) status and chromatin loops via occupying promoters and enhancers. Specifically, CREPT mediates 1082 co-operational chromatin loops configured by enhancer-promoter and promoter-termination loops, which are validated by HiChIP analyses and visualized by Tn5-FISH experiments. These loops orchestrate RNAPII loading and recycling to enhance the metastatic gene expression. Disruption of these co-operational loops using CRISPR-dCas9 suppresses TNBC metastasis in vivo. Furthermore, depletion of CREPT using an AAV-based shRNA blocks TNBC metastasis in both preventative and therapeutic mouse models. We propose that targeting CREPT to disrupt the co-operational chromatin loop structures represents a promising therapeutic strategy for metastatic TNBC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02361-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99f51f69c1d7213a6e3e1569c1bdb0463d7e42a" target='_blank'>
              CREPT is required for the metastasis of triple-negative breast cancer through a co-operational-chromatin loop-based gene regulation
              </a>
            </td>
          <td>
            Jianghua Li, Lu Xu, Jiayu Wang, Xuning Wang, Yuting Lin, Alex Yutian Zou, Fangli Ren, Yinyin Wang, Jun Li, Zhijie Chang
          </td>
          <td>2025-06-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76dc4a293cd6dc10ea69cf397e4b517d730cdc8e" target='_blank'>
              Synergistic role of Alu and core duplicon sequences in driving genomic instability at the disease-associated 16p12.3–p13.11 region
              </a>
            </td>
          <td>
            Stefania Fornezza, Giuliana Giannuzzi
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Loss-of-function mutations in genome maintenance genes fuel tumorigenesis through increased genomic instability. A subset of these tumor suppressors are challenging to identify due to context dependency, including functional interactions with other genes and pathways. Here, we searched for potential causal genes that impact tumor development and/or progression in breast cancer through functional-genetic screening of candidate genes. MYH4, encoding a class II myosin, emerged as a top hit impacting genomic stability. We show that MYH4 suppresses DNA replication stress by promoting replication licensing and replication fork progression. Moreover, we observed a strong synergistic relationship among class II myosins in suppressing replication-associated DNA damage. Genomic analysis of Pan-Cancer Analysis of Whole Genomes project breast cancer samples revealed frequent concomitant loss of TP53 with MYH4 and class II myosins on chromosome 17p. Notably, Myh4 disruption accelerated mouse mammary tumorigenesis in a Trp53-deficient background. In conclusion, our results suggest an unanticipated function of MYH4 in p53-mediated tumor suppression that can explain their combined loss in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64806f05c07b8d24d35ea0d2ce3f2fd4c89aa895" target='_blank'>
              The class II myosin MYH4 safeguards genome integrity and suppresses tumor progression
              </a>
            </td>
          <td>
            J. Thatte, Ana Moisés da Silva, Judit Börcsök, Thorkell Gudjónsson, Jan Benada, Xin Li, Muthiah Bose, H. van der Gulden, Ji-Ying Song, Renée Menezes, Elena Martín-Doncel, Luis Toledo, Valdemaras Petrosius, Cord Brakebusch, J. Jonkers, F. C. Nielsen, Maria Rossing, C. S. Sørensen
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eeecebde718f44128a5af1add184f4e02b1d49" target='_blank'>
              NuMA promotes constitutive heterochromatin compaction by stabilizing linker histone H1 on chromatin
              </a>
            </td>
          <td>
            Yao Wang, Wenxue Zhao, Jiahao Niu, Cuifang Liu, Xiaotian Wang, Weihong Yuan, Shanshan Ai, Wolfgang Baumeister, Guohong Li, Aibin He, Peng Xu, Cheng Li, Yujie Sun
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aebba9c32cce818d2e97fcfa124c22eff76b4f3" target='_blank'>
              LSH-mediated resolution of R-loops mitigates transcription-replication conflicts to preserve genomic stability in prostate cancer cells
              </a>
            </td>
          <td>
            Kai Ni, Yu Zhang, Kun Zheng, Jianwen Huang, Nuorong Xiong, Rong Liang, Yingxiao Chen, Honglin Jin, Youlong Hai, Wang Li, Jiayi Ma, Ranxing Yang, Guanggai Xia, Yueqing Bai, Lujie Song, Xiaoyong Hu, Zijian Tang, Qiang Fu
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c19d5f1ca11d81a47f9f7924f6f192beb7cb85" target='_blank'>
              Cancer type-specific association of p53 deficiency with aneuploidy and chromosome losses
              </a>
            </td>
          <td>
            Joana F. Marques, Marco António Dias Louro, Teresa Davoli, G. Kops
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Reactivation of transposable elements (TEs) in somatic tissues, particularly of LINE-1, is associated with disease by causing gene mutations and DNA damage. Previous work has shown that the PIWI pathway is crucial for TE suppression in the germline. However, the status and function of this pathway has not been well characterized in differentiated somatic cells and there is lack of consensus on the role of the pathway in somatic tumorigenesis. To shed light on this conundrum, we examined the PIWI pathway in colon cancer through a combination of bioinformatic analyses and cell-based assays. Shifted Weighted Annotation Network (SWAN) analysis revealed that the pathway experiences significant allelic losses in colon cancer and that PIWIL2, the main catalytic component of the pathway responsible for TE silencing, experiences the highest percent deletions. PIWIL2 is downregulated in colon tumors of advanced stage, nodal metastasis, and in certain subtypes, correlating with poor survival, while it is even downregulated in ulcerative colitis, an inflammatory bowel disease that predisposes to colon cancer. Knockout studies in colon epithelial Caco2 cells show that PIWIL2 depletion leads to increased anchorage-independent growth, increased LINE-1 levels and activity, and in DNA damage, altogether highlighting a tumor-suppressing role of PIWIL2 in the colon. SUMMARY STATEMENT This study investigates the PIWI-piRNA pathway in colon cancer using a nuanced bioinformatic and cell-based approach, linking the downregulation of PIWIL2 to disease progression, transposable element activation and DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6553455e0dd4bcf44406169f133238204316d7c1" target='_blank'>
              PIWIL2 downregulation in colon cancer promotes transposon activity and pro-tumorigenic phenotypes
              </a>
            </td>
          <td>
            Alyssa Risner, J. Nair-Menon, Abhinav Cheedipudi, Joe R Delaney, Vamsi K Gangaraju, A. Kourtidis
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Focal gene amplification serves as an oncogenic driver during tumorigenesis and is a hallmark of many forms of cancer. Oncogene amplifications promote genomic instability, which is integral to cancer cell survival and evolution. However, focal gene amplification potentially affords an opportunity for therapeutic exploitation. As a proof-of-concept, we leverage CRISPR-Cas9 nickase to selectively promote cancer cell death in MYCN-amplified neuroblastoma in a gene amplification-dependent manner. Our analysis demonstrates that CRISPR-Cas9 nickase can generate a lethal number of highly toxic, replication-dependent double-strand breaks in cells harboring amplified loci. Furthermore, we demonstrate that Cas9 nickase—mediated toxicity can be modulated in combination with small molecule inhibitors targeting key regulators of the DNA-damage response or cell death pathways. Importantly, our findings in MYCN-amplified neuroblastoma translate to other cancer types with distinct oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426fa411ebfe0306ef5fb3727f439d058ec49944" target='_blank'>
              Selective targeting of genome amplifications and repeat elements by CRISPR-Cas9 nickases to promote cancer cell death
              </a>
            </td>
          <td>
            Matthew B Hanlon, Jason M Shohet, S. Wolfe
          </td>
          <td>2025-06-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer, closely associated with tumor evolution, metastasis, immune evasion, and resistance to treatment. CIN is driven by persistent chromosome missegregation, resulting in abnormal chromosomal copy numbers and promoting tumor progression through mitotic errors, faulty chromosome segregation, and the formation of micronuclei or chromosome bridges. Previous studies have demonstrated that p62 localizes to micronuclei, where it interacts with mitochondria, enhancing ROS-mediated cysteine oxidation and promoting p62 homo-oligomerization. This disrupts repair by recruiting CHMP7 and other ESCRT-III components, impairing their function in restoring the micronuclear envelope. Notably, we observed a time-dependent increase in nuclear p62 levels following radiation therapy, which correlated with enhanced chromatin fragmentation and chromothripsis. While p62 inhibits homologous recombination repair and promotes non-homologous end joining, we hypothesize that radiation-induced nuclear p62 accumulation impairs nuclear envelope resealing by inhibiting ESCRT-III components, thereby contributing to chromosomal fragmentation and chromothripsis. These chromosomal alterations may play a significant role in tumor evolution and treatment resistance. Our findings suggest that irradiation-induced p62 accumulation is crucial for chromothripsis and may affect nuclear repair processes, impacting chromosomal stability. Future research will focus on elucidating how p62 contributes to these abnormalities and their implications for tumor progression and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a93ad5e20684542464f1d1902962e0915d7957" target='_blank'>
              Irradiation-induced increase in nuclear p62 levels contributes to chromosomal fragmentation and chromothripsis.
              </a>
            </td>
          <td>
            Xiaojuan Yang, Ying Zhou, Buzhe Zhang, Xue Tao, Weili Qi, Hong Wu
          </td>
          <td>2025-06-05</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ribonucleoside monophosphates (rNMPs) are abundant in DNA, but their distribution and function in human nuclear genomes remain unknown. Here, we mapped nearly one million rNMPs per genome across diverse human cell types, defining a nuclear “ribome” with non-random distribution patterns. rNMPs are enriched in C/G-rich sequences, epigenetically marked regions, and telomeres. Conserved ribonucleotide-enriched zones (REZs) overlap with CpG islands and R-loops. rNMP concentration near transcription start sites (TSSs) correlates positively with gene expression. Wild-type cells display a broader gene-expression range than ribonuclease H2A (RNH2A) knockouts, in which loss of rNMP cleavage causes pronounced retention of embedded rG and strand-biased rC near TSSs, both increasing with gene expression. These findings establish DNA-embedded rNMPs as a novel epigenetic mark that modulates human gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a34d4cbfcfa7f6007c5ecd2489744cc0bcfeecb" target='_blank'>
              Human ribomes reveal DNA-embedded ribonucleotides as a new type of epigenetic mark
              </a>
            </td>
          <td>
            Deepali L. Kundnani, Taehwan Yang, Tejasvi Channagiri, Penghao Xu, Yeunsoo Lee, Mo Sun, Francisco Martinez-Figueroa, Supreet Randhawa, Ashlesha Gogate, Youngkyu Jeon, Stefania Marsili, Gary Newnam, Yilin Lu, Vivian S. Park, Sijia Tao, Justin A Ling, Raymond F. Schinazi, Z. Pursell, Abdulmelik Mohammed, P. Opresko, Bret D. Freudenthal, Baek Kim, Soojin V Yi, Nataša Jonoska, F. Storici
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea18029e3c5c41c740db47ee4da2a815d1304135" target='_blank'>
              5-hydroxymethylcytosine deposition mediates Polycomb Repressive Complex 2 function in MYCN-amplified neuroblastoma
              </a>
            </td>
          <td>
            Mohansrinivas Chennakesavalu, Gabriel L. Lopez, Varsha Gupta, Kelley Moore, Yuqing Xue, Josie Majowka, Sahil Veeravalli, Ryan Borchert, Monica Pomaville, A. Chlenski, Chuan He, Andrea Piunti, M. Applebaum
          </td>
          <td>2025-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1231b220c32fa68e474d1bf7533e08b4effde6bb" target='_blank'>
              Cell enlargement causes mitotic errors and aneuploidy in cells that evade senescence after CDK4/6 inhibition
              </a>
            </td>
          <td>
            Aanchal U. Pareri, Marialucrezia Losito, F. Foijer, AT Saurin
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Plasmids have fundamentally transformed how we resolve regulatory grammar across the tree of life. However, although chromatin plays an integral role in regulating the function of regulatory elements along the nuclear genome, our understanding of how, or whether, similar chromatin structures form on plasmids transfected into mammalian cells remains limited. We demonstrate that plasmid single-molecule chromatin fiber sequencing (plasmid Fiber-seq) can accurately map chromatin architectures along individual, full-length transfected plasmid molecules at near single-nucleotide resolution. Application of plasmid Fiber-seq to diverse plasmids and cell lines demonstrates that plasmids are chromatinized in a sequence-dependent organized manner and adopt a heterogeneous and incomplete chromatin architecture relative to nuclear-encoded chromatin fibers. We show that the focal occupancy of nucleosomes and transcription factors along transfected plasmids is central to their transcriptional activity within mammalian cells, and demonstrate that plasmids indeed are capable of recapitulating nuclear genome-encoded chromatin architectures, although not always. Finally, we demonstrate that combining plasmid Fiber-seq with high-throughput reporter assays can establish the molecular mechanisms underlying pathogenic non-coding variants, including disentangling the effects of transcriptional activators and repressors with near-single-nucleotide resolution. Overall, our findings reveal the principles by which plasmid-based assays can be used for accurate fine-scale mapping of chromatin-dependent regulatory grammar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75d71ed390846f18de84c61671f46cfb09ecd9af" target='_blank'>
              Principles and functional consequences of plasmid chromatinization in mammalian cells
              </a>
            </td>
          <td>
            Ben Mallory, Thomas W. Tullius, Carina G. Biar, Jonas A. Gustafson, Stephanie C. Bohaczuk, Danilo Dubocanin, L. Starita, A. Stergachis
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The evolution of techniques used to identify structural variants (SVs) and copy number variants (CNVs) in genomes have seen significant development in the last decade. With the growing use of more technologies including chromosomal microarray, genome sequencing and genome mapping in clinical cytogenetics laboratories, reporting the frequency of SVs and CNVs has increased the complexity of genomic results. In conventional testing (e.g. karyotype or FISH) individual cells are analyzed and abnormalities are reported at the single cell level directly as a proportion of the analyzed cells. Whereas for bulk genome assays structural and sequence changes are often reported as variant allele frequencies and fractional copy number states. The International System of Cytogenomic Nomenclature (ISCN) recommends converting these values into a “proportion of the sample”, which requires different calculations and underlying assumptions based on the data type. This review illustrates how the different methods of interpreting and reporting data are performed and identifies challenges in the conversion of these values to a proportion of the sample. We stress the need for careful interpretation of data with consideration for factors that may alter how proportions are reported including overlapping SVs and CNVs or regions with acquired homozygosity. We also demonstrate, using validation data of SVs and CNVs tested by multiple techniques how results are largely consistent across methodologies, but can show dramatic differences in rare circumstances. This review focuses on illustrating many of the challenges with aligning reporting using different techniques and their underlying assumptions. As hematologic disease classifications start to incorporate numeric limits (e.g. VAF defining thresholds), it is important for laboratory geneticists, pathologists and clinicians to appreciate the differences in methodologies, potential pitfalls and the nuances when comparing bulk genome analyses to the more conventional single cell techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/883b8047db9fe9c01ef1b4982d92e6a1387b4be0" target='_blank'>
              What the VAF? A guide to the interpretation of variant allele fraction, percent mosaicism, and copy number in cancer
              </a>
            </td>
          <td>
            Adam C Smith, Hubert Tsui, Sila Usta, Jose-Mario Capo-Chichi
          </td>
          <td>2025-07-08</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Objective Repetitive DNA comprises the majority of nuclear genomes in eukaryotes and is critical for genome stability, gene regulation and evolutionary innovation. Yet, most genomic surveys focus on lowcopy regions, leaving repeats underexplored. Here, we assess how repetitive elements shape genomic diversity and rapid radiation in Dyckia (Bromeliaceae) by characterizing the abundance, composition and variability of major repeat families using lowcoverage whole-genome sequencing and the RepeatExplorer2 pipeline. Results description A substantial proportion of Dyckia genome consists of repetitive DNA, reaching ~ 71% in D. densiflora, D. elata, and D. consimilis. Notable interspecific and intraspecific variation was observed, with D. consimilis ranging from 50.3 to 69.1%. Satellite DNA, though present in all species, varied in abundance (0.1–4.7%), indicating heterochromatin flexibility. Ty3/Gypsy and Ty1/Copia LTR retrotransposons dominate the repeatome, with lineage-specific expansions of Tekay and Ogre elements. Despite general conservation, population-level differences in repeat composition suggest a role in genome restructuring and phenotypic plasticity. These patterns point to repeat dynamics as a key driver of genome evolution, taxonomic complexity, and ecological adaptability in Dyckia. Supplementary Information The online version contains supplementary material available at 10.1186/s13104-025-07359-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3a9ee02793b5181e16f379a04487e1d2a8906d6" target='_blank'>
              Genomic characterization of repetitive DNA and transposable elements in Dyckia (Pitcairnioideae) species
              </a>
            </td>
          <td>
            João Victor da Silva Rabelo-Araujo, A. F. Francisconi, Zirlane Portugal da Costa, Caroline Bertocco Garcia, Otavio Batista de Castro Ribeiro, Ana Cristina Silva Amoroso Anastacio, Maria Imaculada Zucchi
          </td>
          <td>2025-07-09</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e18058


 Background:
 HNCs are a diverse group of malignancies characterized by genetic, epigenetic, and proteomic alterations that drive tumor progression, treatment resistance, and poor outcomes.
 Methods:
 Molecular profiling of 273 HNC samples was conducted using a semiconductor-based next-generation sequencing (NGS) platform at Datar Cancer Genetics. A subset of 106 cases underwent targeted transcriptomic analysis of 20,802 genes. Additionally, PD-L1 (22C3) IHC, microsatellite instability (MSI), and tumor mutational burden (TMB) were analyzed in selected samples.
 Results:
 Transcriptomic analysis highlighted significant alterations in critical cellular pathways, including cell cycle control, apoptosis, DNA repair mechanisms, and transcription regulation. Among the most dysregulated genes were
 DLGAP5
 ,
 CASP14
 ,
 BCL2
 , and
 JUN
 . Copy number profiling in a subset (N=74) revealed significant chromosomal instability, with 34% of cases showing losses on specific chromosome arms and 9% exhibiting gains, notably in regions such as
 4p, 19p
 and
 1p.
 Mutations were predominantly observed in tumor suppressor genes, with a lower frequency in oncogenes. Frequently altered genes include
 TP53
 (63%),
 CDKN2A
 (19%),
 PIK3CA
 (12%),
 TERT
 (15%),
 HRAS
 (6%) and
 EGFR
 (1%).
 HRAS
 mutations offer potential for targeted therapy with tipifarnib, a farnesyltransferase inhibitor.
 EGFR
 alterations, though rare, have proven challenging to effectively target with
 EGFR
 -specific therapies such as cetuximab or tyrosine kinase inhibitors, underscoring the critical need for innovative strategies to exploit this target. Amplifications were observed in
 MYC
 (9%),
 CCND1
 (8%),
 FGF19
 (6%),
 FGF3
 (4%) and
 FGF4
 (4%). Targetable amplifications were seen in
 EGFR
 (6%),
 ERBB2
 (2%) and
 MET
 (1%), offering potential opportunities for personalized therapies.
 MET
 amplifications are also linked to aggressive tumor behavior. Fusion was a relatively less common event, observed in 9.5% cases, including one case each of targetable
 BRAF-MRPS33
 and
 FGFR1-PLAG1
 fusion. High TMB (≥10 muts/mb) was seen in 21% cases (Median 12, range 0-28). None of the tumors showed MSI-high status (n=63). Higher PD-L1 positivity was observed in TMB high samples (76% vs 59%), with PD-L1 22-C3 CPS ≥1 in 66% (50/ 76) and PD-L1 28-8 TPS ≥1 in 46% (36/ 78).
 Conclusions:
 This study underscores the genetic heterogeneity of HNCs, pointing to deregulated pathways involved in cell cycle control, apoptosis, and DNA repair as promising targets for future therapeutic development and personalized treatment strategies. Also this study points towards a subset of patients who may derive a greater benefit with immunotherapy, based on TMB/PDL1.


 ESCAT TIER distribution of the molecular alterations detected in head and neck cohort (N=273).




 TIERs
 Total Cases
 % occurrence




 IC
 6
 2%


 IIA
 21
 8%


 IIIA
 85
 31%


 IIIA
 1
 0%


 IIIB
 57
 21%


 IVA
 180
 66%


 IVB
 71
 26%


 X
 142
 52%




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a204052c6edd84c3f9c00c6d7bada097fe56677e" target='_blank'>
              Molecular profiling of head and neck cancers (HNCs): Genetic heterogeneity, immune landscape, and therapeutic targets.
              </a>
            </td>
          <td>
            R. K. Choudhary, S. Limaye, Padman Vamadevan, Vivek Agarwala, Vikas Goswami, H. P, Vijay Anand Reddy, Niyati Krunal Shah, Darshit Kalpeshkumar Shah, Shivam Shingla, Dhanashri Ahire, Vinayak Rao, R. Gosavi, Priyanka Desale, R. Datar, D. Patil
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/037c750c2146efe1a36aa3818222e02772636491" target='_blank'>
              Mitotic phosphorylation of ADAR1 regulates its centromeric localization and is required for faithful mitotic progression
              </a>
            </td>
          <td>
            Yuxi Yang, Mai Kubota, Kokone Hasegawa, Chao Zeng, Kazuko Nishikura, Michiaki Hamada, Masayuki Sakurai
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8533a84bf23dfa93f9b981af3fd2fcfd03e25b" target='_blank'>
              AURKA inhibition amplifies DNA replication stress to foster WEE1 kinase dependency and synergistic antitumor effects with WEE1 inhibition in cancers
              </a>
            </td>
          <td>
            Jong Woo Lee, Jackie Shi, Julian Barrantes, Pratima Chaurasia, Sundong Kim, S. Cruz-Gomez, Cindy Yang, Jianlei Gu, Hongyu Zhao, Dejian Zhao, Benjamin Schiff, Ansley Roche, Elizabeth B. Perry, Jeffrey P. Townsend, Gary M. Kupfer, Katerina Politi, E. Golemis, Barbara Burtness
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d5c649254c4e2aba35b14c09f15fffc91fc90e" target='_blank'>
              RB dependent transcriptional regulation at mitotic centromeres preserves genome stability
              </a>
            </td>
          <td>
            Elizabeth A Crowley, Amity L Manning
          </td>
          <td>2025-06-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Genome instability (GIN) is a cell pathology linked to cancer promotion and tumor evolution. Transcription is an essential cellular process but also a potential source of DNA damage and GIN. Transcription-replication conflicts (TRCs) are a predominant source of GIN, and defective TRC resolution may seriously compromise genome integrity. Importantly, chromatin dynamics helps orchestrate the response to TRCs to preserve genome integrity. Multiple epigenetic deficiencies have been shown to cause transcription-induced replication stress, resulting in DNA breaks and mutations. Consistently, chromatin alterations are frequent in cancer and correlate with increased mutation burden at TRC sites in tumors. Here, we review our current knowledge of TRC processing, the consequences of its dysfunction, and its relevance in cancer. We focus on the interplay between the DNA damage response (DDR) and chromatin dynamics and discuss the clinical potential of targeting TRCs as anticancer strategies and drugging the associated epigenetic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/633badfaea4225294a91ff97893c1fea15ce4095" target='_blank'>
              Transcription-Replication Conflicts: Unlocking New Frontiers in Cancer.
              </a>
            </td>
          <td>
            Aleix Bayona-Feliú, A. Aguilera
          </td>
          <td>2025-06-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc53004c4a4eae46c08e03815bc43c29345a6bec" target='_blank'>
              The landscape and consequences of transcription stress
              </a>
            </td>
          <td>
            Osama Hidmi, Sara Oster Flayshman, Diala Shatleh, Jonathan Monin, Rami I. Aqeilan
          </td>
          <td>2025-05-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Osimertinib, a third-generation EGFR tyrosine kinase inhibitor (EGFR-TKI), has dramatically transformed the treatment landscape for patients with EGFR-mutant NSCLC. However, the long-term success of this therapy is often compromised by the onset of acquired resistance, with non-genetic mechanisms increasingly recognized as pivotal contributors. Here, we exploit a multi-omics approach to profile genome-wide chromatin accessibility and transcriptional landscapes between drug sensitive and resistant EGFR-mutant cells. Our findings reveal a robust concordance between epigenetic regulome and transcriptomic changes that characterize the osimertinib resistant state. Through CRISPR-based functional genomics screen targeting epigenetic regulators and transcription factors, we uncover a critical regulatory network featuring key members of the NuRD and PRC2 complexes that mediate resistance. Most critically, our screen identifies FOSL1 and JUN, two subunits of the AP-1 transcription factor within this network, as the most significant hits. Mechanistically, we demonstrate that cis-regulatory elements exhibiting altered chromatin accessibility in the resistant state are enriched for cognate AP-1 binding motifs, enabling AP-1 to orchestrate a gene expression program that underpins the druggable MEK/ERK signaling axis, potentially enhancing cell viability and fitness of resistant cells. Importantly, genetic depletion or pharmacological inhibition of AP-1 reinstates cellular and molecular sensitivity, and reverts resistance-associated phenotypes, such as epithelial-to-mesenchymal transition, upon anti-EGFR rechallenge by repressing AKT and ERK signaling. These findings provide novel insights into the epigenetic and transcriptional control of osimertinib resistance in EGFR-mutant NSCLC, highlighting AP-1 as a targetable vulnerability of resistance-related hallmarks and offering a promising avenue for developing resistance reversal strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b98b05c64d117a31d26404e164fc0fbed4525adc" target='_blank'>
              Integrative multi-omics identifies AP-1 transcription factor as a targetable mediator of acquired osimertinib resistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            B. Dayanc, Sude Eris, Nazife Ege Gulfirat, Gulden Ozden-Yilmaz, E. Cakiroglu, Ozlem Silan Coskun Deniz, G. Karakülah, Serap Erkek-Ozhan, Serif Senturk
          </td>
          <td>2025-05-25</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779c13815a53f4713be7d07b72908d9198ac23e6" target='_blank'>
              Nucleosome-mediated conformational switches in micro-eccDNAs
              </a>
            </td>
          <td>
            Sarah Harris, Victor Velasco Berrelleza, William Sandel, T. Sutthibutpong, Victoria N. Parikh, Euan A. Ashley, Andrew Fire, S. Levene, Thomas C Bishop, M. Shoura
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosome instability leading to aneuploidy and accumulation of copy number gains or losses is a hallmark of cancer. Copy number alteration (CNA) signatures are increasingly used for cancer risk stratification, but size thresholds for defining CNAs across cancers are variable and the biological and clinical implications of CNA size heterogeneity and co-occurrence are incompletely understood. Here we analyze CNA and clinical data from 691 meningiomas and 10,383 tumors from The Cancer Genome Atlas to develop cancer- and chromosome-specific size-dependent CNA and CNA co-occurrence models to predict tumor control and overall survival. Our results shed light on technical considerations for biomarker development and reveal prognostic CNAs with optimized size thresholds and co-occurrence patterns that refine risk stratification across a diversity of cancer types. These data suggest that consideration of CNA size, focality, number, and co-occurrence can be used to identify biomarkers of aggressive tumor behavior that may be useful for individualized risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa10dcbb2ebaf779a0622a82815ec7ac7b25cca" target='_blank'>
              Pan-cancer copy number analysis identifies optimized size thresholds and co-occurrence models for individualized risk stratification
              </a>
            </td>
          <td>
            M. P. Nguyen, William C Chen, K. Mirchia, A. Choudhury, Naomi Zakimi, Vijay Nitturi, Tiemo J. Klisch, Stephen T. Magill, Calixto-Hope G. Lucas, Akash J. Patel, David R Raleigh
          </td>
          <td>2025-07-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background/Objectives: Aneuploidy is near-ubiquitous in cancer and can decrease chemotherapy efficacy while also sensitizing cells to other drugs. Methods: To systematically identify treatment strategies that target aneuploid cancers, data were integrated from The Cancer Genome Atlas (TCGA; 10,967 samples, 16,948 aneuploidy events) and the Broad Institute’s Profiling Relative Inhibition Simultaneously in Mixtures (PRISM) screen of 578 cancer cell lines and 4518 compounds. Results: Our analyses uncovered 37,720 significant positive and negative associations linking specific aneuploidies and treatments with patient prognosis or cell viability. Within TCGA data, 22 treatments correlated with improved 5-year survival for specific aneuploid cancers, whereas 46 were linked to worse outcomes. A complementary analysis of PRISM identified 17,946 compound–aneuploidy associations and 16,189 mechanism of action (MOA)–aneuploidy associations. Pathway-altering compounds that selectively reduce viability in cells with aneuploidy profiles were discovered, including an unexpectedly prominent number of glucocorticoid receptor agonists. Conclusions: This integrated dataset provides a resource for designing therapeutic decision hypotheses, identifying drug-repurposing opportunities, and informing future studies aimed at targeting aneuploidy-induced vulnerabilities in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811f8e93329a18a966c2c968f4a1e0431dca5e53" target='_blank'>
              Half the Chromosome It Used to Be: Identifying Cancer Treatments Targeting Aneuploid Losses
              </a>
            </td>
          <td>
            Andrew O. Disharoon, Joe R. Delaney
          </td>
          <td>2025-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13b5cab876cec5cb4938bafcc68be31f6630d5cb" target='_blank'>
              Disruption of TAD hierarchy promotes LTR co-option in cancer.
              </a>
            </td>
          <td>
            Elissa W P Wong, Merve Sahin, Rui Yang, UkJin Lee, Dan Li, Y. Zhan, Rohan Misra, Fanny Tomas, Nawaf Alomran, A. Polyzos, Cindy J. Lee, Tuan Trieu, A. Martínez-Fundichely, Thomas Wiesner, Andrew Rosowicz, Shuyuan Cheng, Christina Liu, Morgan Lallo, A. Shoushtari, T. Merghoub, Pierre-Jacques Hamard, R. Koche, Ekta Khurana, E. Apostolou, Deyou Zheng, Yu Chen, Christina S. Leslie, Ping Chi
          </td>
          <td>2025-06-30</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="The tumorigenesis of small intestinal neuroendocrine tumors (siNETs) is not understood and comprehensive genomic and transcriptomic data sets are limited. Therefore, we performed whole genome and transcriptome analysis of 39 well differentiated siNET samples. Our genomic data revealed a lack of recurrent driver mutations and demonstrated that multifocal siNETs from individual patients can arise genetically independently. We detected germline mutations in Fanconi anemia DNA repair pathway (FANC) genes, involved in homologous recombination (HR) DNA repair, in 9% of patients and found mutational signatures of defective HR DNA repair in late-stage tumor evolution. Furthermore, transcriptomic analysis revealed low expression of the transcriptional repressor REST. Summarizing, we identify a novel common transcriptomic signature of siNETs and demonstrate that genomic alterations alone do not explain initial tumor formation, while impaired DNA repair likely contributes to tumor evolution and represents a potential pharmaceutical target in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44f7cb964c906b47270fb2a184506e3a4a4d940" target='_blank'>
              Small intestinal neuroendocrine tumors lack early genomic drivers, acquire DNA repair defects and harbor hallmarks of low REST expression
              </a>
            </td>
          <td>
            Felix Bolduan, Niklas Müller-Bötticher, Olivia Debnath, I. Eichhorn, Yvonne Giesecke, Alexandra Wetzel, S. Sahay, Tomasz Zemojtel, Marten Jaeger, Ute Ungethuem, Christoph Roderburg, C. A. Kunze, Annika Lehmann, David Horst, Frank Tacke, Roland Eils, Bertram Wiedenmann, Michael Sigal, N. Ishaque
          </td>
          <td>2025-05-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Imprinted genes are epigenetically regulated in normal tissues to follow monoallelic expression according to the parent of origin of each allele. Some of these patterns are dysregulated in cancer.


RESULTS
We developed a novel computational multi-omic pipeline to evaluate monoallelic and biallelic expression patterns based on matched RNA-seq expression data, whole-exome sequencing information, and copy number data. We analyzed allelic expression of the entire genes, individual isoforms, and each exon of 59,283 autosomal protein-coding and ncRNA genes, with a focus on 94 genes previously reported to be imprinted. We analyzed 108 cell lines from 9 different tumor histologies using molecular data from the DepMap Portal for the Cancer Cell Line Encyclopedia. Allelic expression patterns of imprinted genes and isoforms in tumor cells were variable. We also identified additional genes and isoforms with predominantly monoallelic expression due to a variety of potential mechanisms. We provide a novel public dataset of transcriptome-wide allelic expression patterns in cell lines from diverse tumor categories, which can serve as a resource for future cancer studies. We examined associations of in vitro cell line response to antitumor agents and repurposed drugs with allelic patterns and overall levels of isoform expression of imprinted genes and of additional genes with predominantly monoallelic expression. Drug response was associated with isoform expression patterns of multiple imprinted genes including CPA4, DGCR6, DNMT1, GNAS, GRB10, H19, NAA60, OSBPL5, PHACTR2, and ZFAT, predominantly monoallelically expressed MAP2K5 and BCLAF1, and additional predominantly monoallelically expressed genes. Multiple associations may be related to mechanisms of drug activity, including associations between the response to the DNA damaging agents and allelic expression of ZFAT, CDC27, and BCLAF1 isoforms, and the response to inhibitors of multiple signaling pathways with expression patterns of GNAS isoforms.


CONCLUSIONS
Tumor cells have a range of monoallelic and biallelic expression patterns in both imprinted and non-imprinted genes and are likely affected by the complex interplay among changes in allelic expression, sequence variants, copy number changes, and expression changes of biologically important genes. Multiple isoform-specific patterns of allelic expression were associated with drug response, indicating complex mechanisms of cancer chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e53122e0648c0b26f74137cf51cf8b0a5877a45c" target='_blank'>
              Allelic expression patterns of imprinted and non-imprinted genes in cancer cell lines from multiple histologies
              </a>
            </td>
          <td>
            Julia Krushkal, Travis L. Jensen, George Wright, Yingdong Zhao
          </td>
          <td>2025-05-25</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c3d47e157de191e71468217bfe01f9c8e3d31f0" target='_blank'>
              ILF2 blockade suppresses helicase-mediated R-loop resolution to induce lethality in homologous recombination-deficient cancers
              </a>
            </td>
          <td>
            Yi Chieh Lim, Soak-Kuan Lai, Ru Han Yang, Yong Kang Chua, Rémi Patouret, Vaishnervi Manoj, Colin Chun Xuan Soh, Benjamin Lebeau, Ning Cham, Wei Meng, Sze Keong Tey, Szu Ling Yeap, Songjing Zhang, Jessie Yong Xing Woon, Zahin Ali, Marek Mutwil, Pritpal Singh, M. Fullwood, Linda LD Zhong, Xue Ming Dong, Chiba Shunsuke, Cheng-Gee Koh, Li Hoi Yeung
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Background: Genomic structural variation (SV) is a recognized property of cancer cells, contributing to genomic instability and oncogenesis. SV breakpoints and rearrangement patterns are often tumor-specific and can reflect underlying tumor biology. The landscape and implications of SV in breast cancer has been incompletely characterized. Furthermore, the utility of using SVs for prospective circulating tumor DNA (ctDNA) detection and monitoring in early breast cancer (EBC) has not been evaluated. Methods: SV was evaluated through whole genome sequencing (WGS) analysis in two independent breast cancer datasets: (1) The 100,000 genomes project [n=3044 patients; Genomics England, GEL], made possible through access to data and findings in the National Genomic Research Library via the Genomics England Research Environment and (2) a cohort of patients with EBC treated with neoadjuvant chemotherapy [ctDNA evaluation in early breast cancer (TRACER; n=210 patients) Princess Margaret Cancer Centre, Canada]. SV burden and type was evaluated in the GEL dataset using MANTA and in TRACER using an in-house pipeline (SV size cut-off of >50 bp). Survival correlates within the GEL dataset were evaluated using Cox proportional hazard models with cases censored after 6 years. The performance of an SV-based ctDNA assay was evaluated in BT-474 (cell line, contrived samples) and in FFPE tumors (TRACER, clinical samples) using shallow depth (∼15X) WGS followed by SV detection (up to 16) via proprietary multiplex digital PCR. Longitudinal ctDNA detection was performed in the TRACER cohort. Results: SV was common across all breast cancer subtypes (GEL data; median SV burden: 108, range: 4-1448; median aggregate copy number of the top 16 SV: 57, range: 16-160). A higher proportion of inversions were seen in HER2-positive tumors, while deletions were common in TNBC. In a multivariate model (including clinical stage, subtype, and tumor mutational burden), there was a significant association between higher SV count and worse overall survival (OS) in ER+/HER2- breast cancer (HR: 2.31, p=0.0207). Patients with ER+/HER2- breast cancer who experienced a clinical recurrence had higher SV copy number in the top 16 variants than those who did not (p<0.0001). In silico analyses demonstrated that personalized SV-based ctDNA panels (fingerprints) could be successfully designed for 97.1% of GEL cases. To assess the characteristics of an SV-based ctDNA assay, a limit of detection (LoD) study was performed with BT-474 using contrived cfDNA (70 ng). The LoD95 was estimated at 0.00052% tumor fraction (5 PPM) with variants detected as low as 1 in 10 million (0.00001% or 0.1 PPM in 31% of cases). A specificity of 100% was seen in 134 healthy donors using 24 different fingerprints (assessment of 1600 SVs). In an initial cohort of 55 patients (TRACER; ER+/HER2-:19, HER2+:23, TNBC:13), shallow WGS and fingerprint design were successful in all patients. The median number of SVs was 336 (range: 73-1345) with the SV type distributed in a similar fashion (more inversions in HER2+ tumors, deletions in TNBC). A trend towards higher SV copy number based on tumor-only WGS was also seen in those with ER+/HER2- disease who experienced subsequent recurrence (TRACER; SV in no recurrence vs. recurrence: 54.3 vs. 134.1, p=0.099). Shallow WGS and SV-based ctDNA assay design is underway for additional patients in the TRACER cohort. Conclusion: SVs are prevalent in breast cancer and associated with prognosis. Personalized SV-based panels permitted ultrasensitive ctDNA detection with high sensitivity and specificity in contrived samples, supporting assay feasibility for early breast cancer. Further analysis of the prognostic impact of SV-burden and type as well as on-treatment and adjuvant ctDNA detection in patients with EBC (TRACER) using SV tracking will be presented at the meeting.
 Citation Format: Mitchell Elliott, Karen Howarth, Sasha Main, Jesús Fuentes Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, Michelle B. Nadler, Elizabeth Shah, Celeste Yu, Scott Bratman, Taylor Bird, June Roh, Elza C. de Bruin, Christopher Rushton, Sofia Birkeälv, Miguel Alcaide, Lucia Oton, Sergii Gladchuk, Yilun Chen, Anthony George, Girish Putcha, Samuel Woodhouse, Philippe L. Bedard, Lillian L. Siu, Hal K. Berman, David W. Cescon. Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS9-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8337c1a084ec3751caa33b2040b1336acad4d2d" target='_blank'>
              Abstract PS9-05: Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer
              </a>
            </td>
          <td>
            M. Elliott, Karen Howarth, S. Main, J. F. Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, M.B. Nadler, Elizabeth Shah, Celeste Yu, Scott V. Bratman, Taylor Bird, June Roh, Elza C de Bruin, Christopher Rushton, Sofia Birkeälv, M. Alcaide, L. Oton, Sergii Gladchuk, Yilun Chen, A. George, Girish Putcha, Samuel Woodhouse, P. Bedard, Lillian L. Siu, H. Berman, D. Cescon
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="During cancer development, mutations promote gene expression changes that cause transformation. Leukemia is frequently associated with aberrant HOXA expression driven by translocations in nucleoporin genes or KMT2A, and mutations in NPM1. How disparate mutations converge on this regulatory pathway is not understood. Here we demonstrate that mutant NPM1 (NPM1c) forms nuclear condensates in multiple human cell lines, mouse models, and primary patient samples. We show NPM1c phase separation is necessary and sufficient to coordinate the recruitment of NUP98 and KMT2A to condensates. Through extensive mutagenesis and pharmacological destabilization of phase separation, we find that NPM1c condensates are necessary for regulating gene expression, promoting in vivo expansion, and maintaining the undifferentiated leukemic state. Finally, we show that nucleoporin and KMT2A fusion proteins form condensates that are biophysically indistinguishable from NPM1c condensates. Together, these data define a new condensate underlying leukemias that we term coordinating bodies (C-bodies), and propose C-bodies as a therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0742ebc47874d0a681b5ec4cb75157b2b57840b5" target='_blank'>
              Nuclear Phase Separation Drives NPM1-mutant Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Gandhar K. Datar, Elmira Khabusheva, Archish Anand, Joshua Beale, Marwa Sadek, Chun-Wei Chen, Evdokiia Potolitsyna, Nayara Alcantara-Contessoto, Guangyuan Liu, Josephine De La Fuente, Christina Dollinger, A. Guzman, Alejandra G Martell, Katharina Wohlan, A. Maiti, N. Short, S. S. Yi, V. Andresen, Bjørn T. Gjertsen, B. Falini, R. E. Rau, L. Brunetti, Nidhi Sahni, M. Goodell, Joshua A. Riback
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="To systematically investigate the molecular and pathological mechanisms of enhancer RNA (eRNA)–mediated transcriptional regulation in glioma recurrence and progression, transcriptomic, regulatome, and genomic data were integrated to analyze eRNA behavior in lower-grade gliomas (stages II/III) and glioblastomas (stage IV). Most eRNAs exhibited dynamic expression during glioma progression, regulated by master transcription factors (TFs) and affected by genomic mutations. The constructed perturbed TF-mediated eRNA-promoter regulatory landscape revealed that rewiring eRNA-promoter networks contributed to glioma malignancy. Drug response–related eRNAs associated with poor prognosis were identified, highlighting their clinical potential. Overall, integrative analysis highlights the critical role of eRNA-mediated regulatory rewiring in glioma progression, providing valuable insights into transcriptional mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b9a681aad2033d6a1c3522f91909b48b06e614" target='_blank'>
              Enhancer RNA–mediated transcriptional regulatory programs reveal the malignant progression of glioma
              </a>
            </td>
          <td>
            Yingying Ma, Shuyuan Wang, Yijie Jia, Shijie Wang, Wendian Liu, Jiaxin Yu, Jun Wang, Jingkai Geng, Jiayi Zhao, Zheng Zhao, Juan Xu, Hui Liu
          </td>
          <td>2025-06-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48b921317e55f0a7a7e7ce1f3c412ed182dc30b" target='_blank'>
              DNA2 and FANCM function in two distinctive pathways in disrupting TERRA R-loops and suppressing replication stress at ALT telomeres
              </a>
            </td>
          <td>
            Ashwin Ragupathi, Heba Z. Abid, Yun Chen, Rongwei Zhao, Settapong T. Kosiyatrakul, Derin I. Yetil, Julia Neiswander, Rachel Feltman, Shannon Thomas, Benjamin Yusupov, Manrose Singh, Li Zheng, Binghui Shen, Huaiying Zhang, H. C. Chu, Carl L. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24cfd9851764715ead654d1c47914a25b505dff" target='_blank'>
              Die Vermessung des Genoms
              </a>
            </td>
          <td>
            Clemens Steinek, H. Leonhardt
          </td>
          <td>2025-06-01</td>
          <td>BIOspektrum</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The spatial organization of the genome within the nucleus is a fundamental regulator of gene expression, genome stability, and cell identity. This review addresses the central question of how nuclear genome architecture contributes to disease mechanisms and diagnostics, and how technological advances enable its clinical exploration. We first outline the principles of nuclear genome architecture, including chromosome territories, replication timing, and 3D domains, and their role in gene regulation and disease. We then explore the mechanisms and consequences of chromosomal rearrangements, and how replication dynamics intersect with epigenetic regulation and genome stability. Diagnostic tools are presented in chronological progression, from conventional cytogenetics to high-resolution genomic and single-cell techniques. A dedicated section focuses on cancer cytogenomics and its clinical implications. We further highlight emerging technologies for 3D genome and epigenome profiling and their integration into diagnostic workflows. Finally, we discuss current challenges, such as standardization and cost, and the transformative potential of multi-omics and artificial intelligence for future precision diagnostics. Overall, we provide a comprehensive overview of how cytogenetics and cytogenomics contribute to the understanding and clinical diagnosis of genetic and neoplastic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0253c2480a5ecc52bcd5c1eda31a7ec8e922ab1" target='_blank'>
              Cytogenetics and Cytogenomics in Clinical Diagnostics: Genome Architecture, Structural Variants, and Translational Applications
              </a>
            </td>
          <td>
            Concetta Federico, Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background CRISPR interference (CRISPRi), the repurposing of the RNA-guided endonuclease dCas9 as a programmable transcriptional repressor, allows highly specific repression (knockdown) of gene expression. CRISPRi platforms can often have incomplete knockdown, performance variability across cell lines and gene targets, and inconsistencies dependent on the guide RNA sequence employed. Results Here, we explore the combination of novel repressor domains with strong Krüppel-associated box (KRAB) repressors, screening > 100 bipartite and tripartite fusion proteins for their ability to reduce gene expression as CRISPRi effectors. We show that these novel repressor fusions have reduced dependence on guide RNA sequences, better slow cell growth when used to knock down expression of essential genes, and function in either fusion or scaffold modalities. Furthermore, we isolate and characterize a particularly effective CRISPRi platform, dCas9-ZIM3(KRAB)-MeCP2(t), which shows improved gene repression of endogenous targets both at the transcript and protein level across several cell lines and when deployed in genome-wide screens. Conclusions We posit that these novel repressor fusions can enhance the reproducibility and utility of CRISPRi in mammalian cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03640-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/541b174ba857f7a16f7cb6e09012bb05165641ba" target='_blank'>
              Engineering novel CRISPRi repressors for highly efficient mammalian gene regulation
              </a>
            </td>
          <td>
            Andrew J Kristof, Krithika Karunakaran, Christopher Allen, Paula Mizote, Sophie Briggs, Zixin Jian, Patrick Nash, J. Blazeck
          </td>
          <td>2025-06-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The capacity of a plasmid to express genes is constrained by its length and copy number. However, the interplay between these parameters and their constraints on plasmid evolution have remained elusive due to the absence of comprehensive quantitative analyses. Here, we present ‘Pseudoalignment and Probabilistic Iterative Read Assignment’ (pseuPIRA), a computational method that overcomes previous computational bottlenecks, enabling rapid and accurate determination of plasmid copy numbers at large scale. We apply pseuPIRA to all microbial genomes in the NCBI RefSeq database with linked short-read sequencing data (4644 bacterial and archaeal genomes including 12,006 plasmids). The analysis reveals three scaling laws of plasmids: first, an inverse power-law correlation between plasmid copy number and plasmid length; second, a positive linear correlation between protein-coding genes and plasmid length; and third, a positive correlation between metabolic genes per plasmid and plasmid length, particularly for large plasmids. These scaling laws imply fundamental constraints on plasmid evolution and functional organization, indicating that as plasmids increase in length, they converge toward chromosomal characteristics in copy number and functional content.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d0a945dacbeb8801a3cb57a4c7f7eb8870d06a" target='_blank'>
              Scaling laws of bacterial and archaeal plasmids
              </a>
            </td>
          <td>
            Rohan Maddamsetti, Irida Shyti, Maggie L. Wilson, Hye-in Son, Yasa Baig, Zhengqing Zhou, Jia Lu, Lingchong You
          </td>
          <td>2025-07-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4028f5dcc0e86189184980c13bb56279010a29ab" target='_blank'>
              Clonal lineage tracing and parallel multiomics profiling reveal transcriptional diversification induced by ARID1A deficiency
              </a>
            </td>
          <td>
            Kenichi Miyata, Liying Yang, Yuxiao Yang, Kohei Kumegawa, R. Maruyama
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadf2932e90bcc5681634229bcd158ed1d080d8b" target='_blank'>
              Single-cell Characterization of DNA Hydroxymethylation of the Mouse Brain During Aging
              </a>
            </td>
          <td>
            Yali Bai, Tianjiao Yuan, Liuhao Ren, Yuhang Huan, Fa Yang, Yutong Li, Aicui Zhang, Yueyang Liu, Tian Tian, Ningxia Kang, Danni Chen, Xuchao Xie, Qiang Wang, Wei Chen, Yinghui Zheng, Xiaoliang Sunney Xie, Yunlong Cao
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, M. T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Chromosome arm-level aneuploidies (CAAs) are a common consequence of genomic instability during the evolution of solid tumor and often associated with metastasis and therapy resistance. In breast cancer, chromosome arms 1q and 8q are the frequent amplified regions for CAAs. This study aims to investigate whether critical genes amplified in genomic regions with CAAs contribute to the pathophysiology in triple-negative breast cancer (TNBC). We established multiple long-term culturable TNBC patient-derived cells (PDCs) using spheroid culture technique, which is favorable for the enrichment of cell population with cancer stemness. We performed a genome-wide chromatin immunoprecipitation study for histone H3K27ac and super-enhancer analysis in these TNBC PDCs and cell lines. Super-enhancer analysis revealed that 8q is a substantial chromosomal region with a high density of super-enhancers. Among genes in the vicinity of 8q super-enhancers, we identified that MYB proto-oncogene like 1 (MYBL1) is a transcription factor with high abundance in TNBC PDCs as well as in basal-like BT549 cells. In TCGA breast cancer database, 8q gain is a common genomic feature in MYBL1 gene-amplified breast cancer tissues and MYBL1 expression is substantially correlated with 8q-related genes and proliferation-related genes like MKI67 and BUB1. In TNBC PDCs and cell lines, we showed that MYBL1-specific siRNAs significantly repressed TNBC cell proliferation and migration. RNA sequencing analysis in TNBC cells revealed that MYBL1 silencing substantially downregulated genes involved in the transcription machinery such as nuclear envelope, chromatin dynamics, and DNA replication. In immunohistochemical analysis of clinical TNBC tissues from a Japanese cohort, we showed that a positive MYBL1 immunoreactivity (IR) was significantly associated with shorter disease-free survival (DFS). Among candidate MYBL1 targets, we demonstrated that NCAPH, a subunit of condensin I complex, was a prognostic factor for patients with TNBC based on the immunohistochemistry of clinical TNBC tissues. Notably, patients with double IR positivity for MYBL1 and NCAPH showed shorter DFS than those with MYBL1 or NCAPH positivity alone. MYBL1 has been characterized as an essential transcriptional regulator in male meiosis and in female mammary gland development. Our findings indicate that MYBL1 plays an essential role in the pathophysiology for advanced breast cancers such as TNBC, and MYBL1 and its downstream genes can be potential diagnostic and therapeutic targets for the disease.
 Citation Format: Akihiro Fujimoto, Kazuhiro Ikeda, Keiichi Kinowaki, Hidetaka Kawabata, Akihiko Osaki, Satoshi Inoue, Kuniko Horie. Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bc0fd91427900f720610a3a8305e7e955f0eb39" target='_blank'>
              Abstract P2-02-08: Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer
              </a>
            </td>
          <td>
            Akihiro Fujimoto, K. Ikeda, Keiichi Kinowaki, H. Kawabata, Akihiko Osaki, Satoshi Inoue, K. Horie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d7251ffb7f87367d6e05c4f72a28b57671f1431" target='_blank'>
              Senescence-Associated Chromatin Rewiring Promotes Inflammation and Transposable Element Activation
              </a>
            </td>
          <td>
            A. Dalgarno, Shane A. Evans, M. Kelsey, Thomas A. Nunez, Azucena Rocha, Kelly Clark, J. Sedivy, Nicola Neretti
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4f9af4dcbefbfc2af9850cbacc2d2628f2e0e1e" target='_blank'>
              TEITbase: a database for transposable element (TE)-initiated transcripts in human cancers
              </a>
            </td>
          <td>
            Yun Zhang, Jianqi She, Xueyan Hu, Yueqi Jin, Changyu Tao, Minghao Du, Ence Yang
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Mutation, deletion, or silencing of genes encoding cellular metabolism factors occurs frequently in human malignancies. Neomorphic mutations in isocitrate dehydrogenases 1 and 2 (IDH1/2) promoting the production of R-2-hydroxyglutarate (R-2HG) instead of α-ketoglutarate (αKG) are recurrent in human brain cancers and constitute an early event in low-grade gliomagenesis. Due to its structural similarity with αKG, R-2HG acts as an inhibitor of αKG-dependent enzymes. These include the JUMONJI family of lysine demethylases, among which KDM4A is particularly sensitive to R-2HG-mediated inhibition. However, the precise molecular mechanism through which inhibition of αKG-dependent enzymes by R-2HG promotes gliomagenesis remains poorly understood. Here, we show that treatment with R-2HG induces cellular senescence in a p53-dependent manner. Furthermore, expression of mutated IDH1R132H or exposure to R-2HG, which leads to KDM4A inhibition, causes telomeric dysfunction. We demonstrate that KDM4A localizes to telomeric repeats and regulates abundance of H3K9(me3) at telomeres. We show that R-2HG caused reduced replication fork progression, and that depletion of SMARCAL1, a helicase involved in replication fork reversal, rescues telomeric defects caused by R-2HG or KDM4A depletion. These results establish a model whereby IDH1/2 mutations cause R-2HG-mediated inhibition of KDM4A, leading to telomeric DNA replication defects, telomere dysfunction, and associated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728014013df3b457cbd616bc46c4a758e940b3f8" target='_blank'>
              R-2-hydroxyglutarate-mediated inhibition of KDM4A compromises telomere integrity
              </a>
            </td>
          <td>
            Florence Couteau, Laurence M Gagné, Karine Boulay, Philippe Rousseau, Mélissa Carbonneau, Mary McQuaid, Jyoti Sharma, C. Sawchyn, Erlinda Fernandez, Dagmar Glatz, Rana Rizk, Marie-Eve Lalonde, Yosra Mehrjoo, Tsz Wai Chu, G. Moquin-Beaudry, Christian Beauséjour, Mikhail Sergeev, Santiago Costantino, D. Avizonis, I. Topisirovic, Nada Jabado, Hugo Wurtele, Chantal Autexier, Frédérick A. Mallette
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Understanding how the genome is organized into multi-level chromatin structures within cells and how these chromatin structures regulate gene transcription influencing animal development and human diseases has long been a major goal in genetics and cell biology. Recent evidence suggests that chromatin structure formation and remodeling is regulated not only by chromatin loop extrusion but also by phase-separated condensates. Here, we discuss recent findings on the mechanisms of chromatin organization mediated by phase separation, with a focus on the roles of phase-separated condensates in chromatin structural dysregulation in human diseases. Indeed, these mechanistic revelations herald promising therapeutic strategies targeting phase-separated condensates—leveraging their intrinsic biophysical susceptibilities to restore chromatin structure dysregulated by aberrant phase separation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9474125e5ffd125b4ecbd37e346a326591b679a6" target='_blank'>
              Phase Separation in Chromatin Organization and Human Diseases
              </a>
            </td>
          <td>
            Ziwei Zhai, Fei Meng, J. Kuang, Duanqing Pei
          </td>
          <td>2025-05-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b1a766c9476f757f76140a52e67514cf195c7f" target='_blank'>
              Longitudinal and multisite sampling reveals mutational and copy number evolution in tumors during metastatic dissemination.
              </a>
            </td>
          <td>
            K. Zhao, J. Vos, S. Lam, Lillian A Boe, Daniel Muldoon, Catherine Y Han, Cristina Valero, Mark Lee, C. Fitzgerald, A. Lee, Manu Prasad, Swati Jain, Xinzhu Deng, Timothy A Chan, Michael F. Berger, C. Bandlamudi, Xi Kathy Zhou, Luc G. T. Morris
          </td>
          <td>2025-06-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Aneuploidy is defined as the loss or gain of a whole chromosome or its region. Even at early stages of development, it usually leads to fatal consequences, including developmental defects/abnormalities and death. For a long time, it was believed that the disruption of gene balance results in pronounced effects at both the cellular and organismal levels, adversely affecting organism formation. It has been shown that the gene imbalance resulting from aneuploidy leads to proteotoxic and metabolic stress within the cell, reduced cell proliferation, genomic instability, oxidative stress, etc. However, some organisms have exhibited tolerance to aneuploidies, which may even confer adaptive advantages, such as antibiotic resistance in pathogenic fungal strains. A significant factor likely lies in the complexity of the tissue and organ organization of specific species. Polyploid organisms are generally more tolerant of aneuploidy, particularly those that have recently undergone whole-genome duplication. This review places special emphasis on the examination of sex chromosome aneuploidies in humans. In addition to primary effects, or cis effects (changes in the quantity of the transcripts of genes located on the aneuploid chromosome), aneuploidy can induce secondary or trans effects (changes in the expression levels of genes located on other chromosomes). The results of recent studies have prompted a reevaluation of the impact of aneuploidy on the structural-functional organization of the genome, transcriptome, and proteome of both the cell and the entire organism. Despite the fact that, in the cases of aneuploidy, the expression levels for most genes correlate with their altered copy numbers in the cell, there have been instances of dosage compensation, where the transcript levels of genes located on the aneuploid chromosome remained unchanged. The review presents findings from recent studies focused on compensatory mechanisms of dosage compensation that modify gene product quantities at post-transcriptional and post-translational levels, alleviating the negative effects of aneuploidy on cellular homeostasis. It also discusses the influence of extrachromosomal elements on the spatial organization of the genome and the changes in gene expression patterns resulting from their presence. Additionally, the review specifically examines cases of segmental aneuploidy and changes in copy number variants (CNVs) in the genome. Not only the implications of their composition are considered, but also their localization within the chromosome and in various compartments of the interphase nucleus. Addressing these questions could significantly contribute to enhancing cytogenomic diagnostics and establishing a necessary database for accurate interpretation of identified cases of segmental aneuploidy and CNVs in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271a442ac782b355db581186dfcccbd9f3d50dd8" target='_blank'>
              From cytogenetics to proteogenomics: new horizons in the study of aneuploidies
              </a>
            </td>
          <td>
            K. Zadesenets, N. B. Rubtsov
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0065672d86f3f94440ecf420ec518fca8aa5a1ec" target='_blank'>
              Integrated Spatial Analysis of Ovarian Precancerous Lesions
              </a>
            </td>
          <td>
            Tu-Yung Chang, Yen-Wei Chien, Szu-Hua Chen, Annelise Sokolow, Yeh Wang, Brant G. Wang, Tricia A Numan, M. Herman Chui, Rebecca Stone, T. Pisanic, Nicholas Papadopoulos, Tian-Li Wang, Christopher Douville, Leslie Cope, I.-M. Shih
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Enhancers, as distal cis-regulatory elements in the genome, have a pivotal influence on orchestrating precise gene expression. Enhancer RNAs (eRNAs), transcribed from active enhancer regions, are increasingly recognized as key regulators of transcription. N6-methyladenosine (m6A), the most plentiful internal modification in eukaryotic mRNAs, has garnered significant research interest in recent years. With advancements in high-throughput sequencing technologies, it has been established that m6A modifications are also present on eRNAs. An accumulative body of evidence demonstrates that aberrant enhancers, eRNAs, and m6A modifications are intimately connected with carcinoma onset, progression, invasion, metastasis, treatment response, drug resistance, and prognosis. However, the underlying molecular mechanisms governing m6A modification of eRNAs in cancer remain elusive. Here, we review and synthesize current understanding of the regulatory roles of enhancers, eRNAs, and m6A modifications in cancer. Furthermore, we investigate the possible roles of eRNAs m6A modification in tumorigenesis based on existing literature, offering novel perspectives and directions for future research on epigenetic regulatory mechanisms in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6966deaca0f1174a84788609926a02bf29bca084" target='_blank'>
              Biological roles of enhancer RNA m6A modification and its implications in cancer
              </a>
            </td>
          <td>
            Yangyang Han, Jingqi Sun, Minghui Yao, L. Miao, Mengjia Li
          </td>
          <td>2025-05-30</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41db77b7599b92a63937c7f3610506611aa97e8f" target='_blank'>
              A survey of human cancer-germline genes : linking X chromosome localization, DNA methylation and sex-biased expression in early embryos
              </a>
            </td>
          <td>
            A. Loriot, Julie Devis, Laurent Gatto, C. De Smet
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c29c85da72e04b5001274eb1b2f06a493d988d5" target='_blank'>
              Reprogramming of Cellular Plasticity via ETS and MYC Core-regulatory Circuits During Response to MAPK Inhibition in BRAF-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            H. Lee, Zhao Zheng, Alexey Sorokin, Chi Wut Wong, Stefania Napolitano, S. Chowdhury, Preeti Kanikarla, A. Singh, Veena Kochat, Christopher Bristow, Sanjana Srinivasan, Mike Peoples, Emre Arslan, J. Alshenaifi, O. Villarreal, V. Morris, J. P. Shen, F. Meric-Bernstam, Abhinav K. Jain, Natalie W Fowlkes, Amanda Anderson, D. Menter, A. Saw, Kunal Rai, S. Kopetz
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In mitosis the duplicated genome is aligned and accurately segregated between daughter nuclei. CTCF is a chromatin looping protein in interphase with an unknown role in mitosis. We previously published data showing that CTCF constitutive knockdown causes mitotic failure, but the mechanism remains unknown. To determine the role of CTCF in mitosis, we used a CRISPR CTCF auxin inducible degron cell line for rapid degradation. CTCF degradation for 3 days resulted in increased failure of mitosis and decreased circularity in post-mitotic nuclei. Upon CTCF degradation CENP-E is still recruited to the kinetochore and there is a low incidence of polar chromosomes which occur upon CENP-E inhibition. Instead, immunofluorescence imaging of mitotic spindles reveals that CTCF degradation causes increased intercentromere distances and a wider and more disorganized metaphase plate, a disruption of key functions of the pericentromere. These results are similar to partial loss of cohesin, an established component of the pericentromere. Thus, we reveal that CTCF is a key maintenance factor of pericentromere function, successful mitosis, and post-mitotic nuclear shape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/529b194fb855f3438e03aaee838ab3df0a5f9eb4" target='_blank'>
              CTCF maintains pericentromere function and mitotic fidelity
              </a>
            </td>
          <td>
            Erin Walsh, A. Stephens
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Inter-chromosomal interactions play a crucial role in 3D genome organization, yet the organizational principles and functional significances remain elusive. In general, lncRNA loci and transcripts are frequently associated with transcriptional programs modulated by long-range chromatin interactions. Here, we identified a novel lncRNA named Gm26793, which is abundantly distributed in the primitive streak and mesodermal cells of embryonic day 7.5 mouse gastrula. Through genetic ablation of Gm26793, we observed a preferential responsiveness to primitive endoderm lineage during stem cell differentiation, as well as enhanced occurrence of transient and degenerative state cells in early mouse embryos when the cell fate segregates between epiblast and primitive endoderm. Mechanistically, we revealed that the genomic locus of Gm26793, rather than the lncRNA transcript or adjacent gene, governs the cell fate preference towards primitive endoderm. Concretely, Gm26793 locus (Chromosome 7) forms an inter-chromosomal molecular lock with Cubn (Chromosome 2) via CTCF, restraining the expression of Cubn and maintaining a natural epigenetic landscape, thus ensuring the proper lineage specification in vitro and in vivo. Overall, our study provides a clear paradigm that inter-chromosomal interaction collaborates with architectural factors to stabilize nuclear conformation and guarantee faithful gene expression during stem cell differentiation and mammalian embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5737d88dd5a339188134898f4ba7c9851ba9807" target='_blank'>
              Genomic locus of lncRNA-Gm26793 forms an inter-chromosomal interaction with Cubn to ensure proper stem cell differentiation in vitro and in vivo
              </a>
            </td>
          <td>
            Zhiwen Liu, Xin Wan, Jiehui Chen, Yongjian Ma, Yonggao Fu, Yingying Chen, Mingzhu Wen, Yun Yang, Yun Qian, Yong Zhang, Dahai Zhu, Jinsong Li, Naihe Jing, Xianfa Yang
          </td>
          <td>2025-06-03</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b369f79b329c117da716bed78cecbcb615823b8f" target='_blank'>
              JAK2 signalling to HNRNPA1 represses retrotransposon activity in haematopoietic stem cells
              </a>
            </td>
          <td>
            Jeremy W. Deuel, Andrejs Makarovs, Sheidasadat Goosheh, George Winder, Hisham Mohammed, Jason S. Carroll, Michael Spencer Chapman, C. Kapadia, Peter Campbell, J. Nangalia, Elisa Laurenti, Constanze Schneider, T. Oellerich, Anthony R Green, S. Loughran
          </td>
          <td>2025-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea313319983bdba2155f6bb08bd3d5f317d33921" target='_blank'>
              Impact of Sperm Fractionation on Chromosome Positioning, Chromatin Integrity, DNA Methylation and Hydroxymethylation Level
              </a>
            </td>
          <td>
            Zuzanna Graczyk, Jagoda Kostyk, Julia Pospieszna, Zuzanna Myslicka, M. Kamieniczna, M. Fraczek, Marta Olszewska, Maciej Kurpisz
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The human genome contains regulatory DNA elements, enhancers, that can activate gene transcription over long chromosomal distances. Here, we show that enhancer distance can be critical for gene silencing. We demonstrate that linear recruitment of the normally distal strong HBB enhancer to developmentally silenced embryonic HBE or fetal HBG promoters, through deletion or inversion of intervening DNA sequences, results in their strongly reactivated expression in adult erythroid cells and ex vivo differentiated hematopoietic stem and progenitor cells. A similar observation is made in the HBA locus, where deletion-to-recruit of the distal enhancer strongly reactivates embryonic HBZ expression. Overall, our work assigns function to seemingly non-regulatory genomic segments: by providing linear separation they may support genes to autonomously control their transcriptional response to distal enhancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d408a95c592c57473a14d7c2700a606f3cb349c" target='_blank'>
              Reactivation of developmentally silenced globin genes through forced linear recruitment of remote enhancers.
              </a>
            </td>
          <td>
            Anna-Karina Felder, Sjoerd J D Tjalsma, H. Verhagen, Rezin Majied, M. Verstegen, Thijs C. J. Verheul, Jeffrey van Haren, Rebecca Mohnani, Richard Gremmen, Peter H. L. Krijger, S. Philipsen, Emile van den Akker, W. de Laat
          </td>
          <td>2025-06-02</td>
          <td>Blood</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Aurantiochytrium limacinum holds great promise for producing sustainable single-cell oil as an alternative to fish oil. However, research into its complex biological and biochemical characteristics and efforts toward strain improvement have been hampered by insufficient genetic tools. Until now, genetic transformations of A. limacinum have relied solely on chromosome integration, which is inefficient and prone to insertional mutagenesis and other issues related to genetically modified organisms (GMOs). This paper describes the first centromeric plasmid for A. limacinum. Amplification of this shuttle vector by E. coli enables direct delivery into A. limacinum via electroporation, where it undergoes stable replication and segregation into daughter cells. The key to the stable plasmid maintenance lies in a 500 bp segment derived from chromosome 24 of Phaeodactylum tricornutum. While this segment does not significantly enhance the efficiency of vector transformation, it enables the replication and maintenance of the shuttle vector in the host cell as closed circular DNA. The plasmid from three transformants demonstrates a high segregation efficiency of 96.8 ± 0.3% (n = 3), even in the absence of antibiotic selection. This novel centromeric plasmid considerably enhances the flexibility of genetic manipulations and gene expression in A. limacinum, opening new avenues for its study and industrial application. Key points • First centromeric plasmid developed for genetic transformation in A. limacinum. • The novel plasmid enhances flexibility in genetic manipulation and gene expression. • The plasmid achieves 96.8 ± 0.3% (n = 3) segregation efficiency without antibiotic selection. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13527-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1a22ec96ed65caa34d28f985f7648b9334af03" target='_blank'>
              Novel centromeric plasmid for stable extrachromosomal gene expression in Aurantiochytrium limacinum
              </a>
            </td>
          <td>
            P. P. Renta, Cian-Huei Syu, Ta-Yu Huang, Yi-Ting Chang, Yu-Feng Liang, Ssu-Ting Chen, Po-Wei Weng, Ming-Chen Hsu, Keng-Hung Lin, Tsu-tung Liu, Anna C-C Jang, Che-Chia Tsao, Han-Jia Lin, Hung-Yun Lin, Yi-Min Chen
          </td>
          <td>2025-07-04</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Gliomas are highly aggressive, life-threatening tumors with poor prognosis, and remain a leading cause of mortality among brain cancers. Although the role of mitochondrial proteins in cancer has garnered increasing attention, their specific functions in the nervous system, particularly in gliomas, remain poorly understood. Methods We integrated single-cell RNA sequencing with cellular assays and flow cytometry to investigate the molecular characteristics and cellular interactions within glioblastoma subpopulations during tumor progression. Results Single-cell RNA sequencing revealed several differentially expressed genes (DEGs) within glioblastoma subpopulations. Trajectory analysis identified CHCHD2P9 as a pivotal marker for the terminal subpopulation. Moreover, elevated expression of CHCHD2P9 was found to correlate with poorer clinical outcomes. Subsequent cellular experiments further explored the underlying mechanisms driving these observations. Conclusions CHCHD2P9 is significantly overexpressed in glioma patients, and its differential expression plays a crucial role in regulating glioma cell proliferation and migration. A CHCHD2P9-based risk model holds promise as both a prognostic biomarker and a potential therapeutic target, providing novel insights into the pathogenesis of gliomas and opening avenues for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d625edcb2052943e2c1f19b8599a0238510719f1" target='_blank'>
              Molecular insights into glioblastoma progression: role of CHCHD2P9 in tumor heterogeneity and prognosis
              </a>
            </td>
          <td>
            Yanyu Ding, Lizhi Xiao, Xiaoling Zhou, Jiaxin Zhao, Jianli Ke, Hua-Bao Cai, Mengyu Zhao, Cun-Zhi Wang, Tian-Hang Yu, Zhijie Zhao, Yucai Wang, Jiyuan Ke
          </td>
          <td>2025-06-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The identification of recurrent oncogenic drivers has enabled targeted therapeutic strategies for subsets of non-small cell lung carcinoma (NSCLC) and other malignancies. Oncogenic fusions involving the RAS-activating guanine exchange factor (GEF) RASGRF1 are reported in multiple tumors, but their prevalence and genetic heterogeneity remain undefined. Here, we query RNA-seq data from a real-world database of diverse human malignancies and identify 40 tumors with rearrangements involving RASGRF1 or the related RASGRF2 predicted to generate chimeric proteins. Half of these fusions occur in NSCLC, pancreatic cancer, and melanoma and are enriched in tumors without other established driver alterations. A subset of RASGRF fusions contains transmembrane partners, and membrane localization enhances RAS activation and transforming activity. Loss of N-terminal PH1 and DH domains in RASGRF fusions also promotes transformation. Although some fusions lack the PH1 but not the DH domain, our functional assays indicate that loss of the PH1 domain alone is insufficient to drive cellular transformation. Our findings provide insights about the tissue distribution, structural diversity, and oncogenic mechanisms of RASGRF fusions. As cell models driven by these fusions are sensitive to MAPK pathway inhibition, oncogenic RASGRF fusions may represent a therapeutic target in rare molecular subsets of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bb9a75bf2438a8363d43341f77587497989fb9b" target='_blank'>
              Genetic and oncogenic features of RASGRF fusions
              </a>
            </td>
          <td>
            Sreya Das, Daniel S Lenchner, Ellen Jaeger, Lisa Hunihan, Dana F DeSantis, Stamatina Fragkogianni, Karyn Ronski, Frederick H Wilson
          </td>
          <td>2025-07-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129265cf9a8d9036012e909d0315f3224ece47d6" target='_blank'>
              The epigenetic regulator TRIM24 controls melanoma cell dedifferentiation and resistance to treatment in melanoma
              </a>
            </td>
          <td>
            Simon Durand, Félix Boivin, Roxane M. Pommier, L. Barbollat-Boutrand, Maxime Grimont, Félix Pham, Marion Dufeu, Eric Cumunel, Raphaël Schneider, Anthony Ferrari, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eded65f17b590b88c69cd9abd535f6f282a95b44" target='_blank'>
              The mutational dynamics of the Arabidopsis centromeres
              </a>
            </td>
          <td>
            Xiao Dong, Wenbiao Jiao, J. A. Campoy, Fernando A. Rabanal, Jurriaan Ton, Lisa M Smith, Detlef Weigel, Korbinian Schneeberger
          </td>
          <td>2025-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef02803fbaf5cb1abbd79351dc6a4ca1c68ba79" target='_blank'>
              Constitutional copy number amplifications: rare or under-evaluated? Revisiting a 25-year-old cold case.
              </a>
            </td>
          <td>
            Eliana Salvo, Romano Tenconi, R. Giorda, Sara Bertuzzo, Luca Cesana, Roberta Murru, S. Giglio, Mana M Mehrjouy, Niels Tommerup, Orsetta Zuffardi, M. Bonaglia
          </td>
          <td>2025-06-04</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background Endogenized giant viruses are emerging as major contributors to the genome evolution of microbial eukaryotes, with both degraded and fully functional latent viruses being found integrated in diverse lineages. The mechanisms that determine the fate of viral integrants are poorly understood, however. Acanthamoeba is a unicellular eukaryote known for undergoing lateral gene transfer (LGT) with viruses. Here we have leveraged chromosome-scale assemblies of two strains of Acanthamoeba, Neff and C3, to investigate the genomic mechanisms that mediate the fate of viral integrations in eukaryotic genomes. Results Viral integrations in the C3 and Neff genomes are largely non-overlapping and disproportionately found in sub-telomeric regions. Multiple partial copies of these insertions are found throughout the Neff genome, but they are not expressed, do not obviously encode functions associated with their own mobility, and are colonized by host mobile elements. Viral regions are hypermethylated and highly condensed, suggesting that the expression of recently acquired viral DNA is suppressed in heterochromatic regions. Conclusions We propose a model for the trajectory of viral sequences in Acanthamoeba: (i) integration of DNA from giant viruses, (ii) epigenetic suppression of the viral DNAs, allowing them to persist in the genome, and (iii) deterioration of viral genomes by point mutation, mobile element colonization, and intra- and inter-chromosomal recombination. Viral integrations in Acanthamoeba spp. are transient and may not have long-lasting effects on the fitness of the amoeba. Our work highlights the importance of host genome dynamics and epigenetic silencing for understanding the evolution of endogenized viral elements. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02280-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f54d3754da37f780fd27a13a0f10078e856d51" target='_blank'>
              Epigenetic silencing and genome dynamics determine the fate of giant virus endogenizations in Acanthamoeba
              </a>
            </td>
          <td>
            Cédric Blais, Morgan J. Colp, Luke A. Sarre, Alex de Mendoza, John M. Archibald
          </td>
          <td>2025-07-01</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis M. Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background DNA methylation plays a crucial role in cancer development and progression and has been linked to genetically and clinically distinct tumor classes, including IDH-mutated and IDH-wildtype adult-type diffuse gliomas. Here, we identify a CpG-island methylator phenotype (CIMP) that characterizes the receptor tyrosine kinase 2 (RTK2) subtype of IDH-wildtype glioblastoma. Results This RTK2-CIMP affects genomic locations and cell functions distinct from those of IDH mutation-associated IDH-CIMP and suppresses the expression of its target genes. The RTK2-CIMP-region chromatin is characterized by a combination of repressive and activating marks, including polycomb-associated H3K27me3 and enhancer-associated H3K4me1, consistent with DNA methylation-mediated silencing of genes with bivalent-state promoters in neural progenitor cells. Functionally, RTK2-CIMP affects neuronal lineage genes and is significantly associated with astrocyte-like glioblastoma, suggesting that RTK2-CIMP is an epigenetic signature of the astrocyte-like cell state. Furthermore, we demonstrate that RTK2-CIMP can be induced by genetic manipulation in glioblastoma cells. Conclusions Our results suggest that RTK2-CIMP is a key contributor to cell-state plasticity in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03670-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94bee397b61a1f7011617f82ad34a983c326fca3" target='_blank'>
              A new IDH-independent hypermethylation phenotype is associated with astrocyte-like cell state in glioblastoma
              </a>
            </td>
          <td>
            Ana Luisa Costa, Daria Doncevic, Yonghe Wu, Lin Yang, Ka-Hou Man, Anna-Sophie Spreng, Hannah Winter, M. Fletcher, B. Radlwimmer, Carl Herrmann
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Abstract Despite being a consolidated practice in modern cancer genomics, mutational signature analysis poses various challenges. First, determining the number of signatures is a complex task and depends on heuristics. Second, several signatures lack a clear etiology, raising concerns about whether they result from computational artifacts or actual mutagenic processes. Last, approaches for signature assignment are highly influenced by the catalogue of signatures used for the analysis. To overcome these limitations, we introduce RESOLVE (Robust EStimation Of mutationaL signatures Via rEgularization), a framework designed for the efficient extraction, assignment, and confidence estimation of mutational signatures. RESOLVE enables the stratification of cancer genomes according to the active mutational processes, providing insights into those that are consistently shared among various cancer types. We applied RESOLVE to 20 000 samples from adult and pediatric cancers, providing a comprehensive characterization of subtypes associated with specific mutational processes and alterations in driver genes. Our analysis demonstrates that RESOLVE accurately fits observed mutations with a smaller set of signatures compared to existing catalogues, suggesting the existence of a limited number of dominant mutational processes in cancer genomes. Clustering analysis revealed distinct patient groups characterized by specific signatures, with significant associations between certain signatures and patient prognosis. Additionally, we identified strong associations between signatures and driver gene mutations, offering insights into cancer subtype mechanisms and evolution. Our findings highlight the efficiency of RESOLVE and its potential impact on personalized cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f33c3f703237609301199e20410a7b8d03b3cefb" target='_blank'>
              Comprehensive analysis of mutational processes across 20 000 adult and pediatric tumors
              </a>
            </td>
          <td>
            M. Villa, Federica Malighetti, Luca De Sano, Alberto Maria Villa, N. Cordani, A. Aroldi, M. Antoniotti, G. Caravagna, Alex Graudenzi, R. Piazza, L. Mologni, Daniele Ramazzotti
          </td>
          <td>2025-07-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 11561


 Background:
 Liposarcoma (LPS) is characterized by unstable genomes and high occurrence of gene fusions. Hotspots of both recurrent and non-recurrent gene fusions can provide information about structural alterations in certain LPS subtypes. For instance, myxoid (M) LPS expresses the oncogenic
 FUS-DDIT3
 protein and is known to be fusion-driven. Moreover, disrupted locus 12q13-15 is an important feature of well-dedifferentiated (WD) and dedifferentiated (DD) LPS, and is the site of copy number alterations (CNAs) and gene fusions. Here, we used RNA sequencing to uncover hotspots of intra- and transchromosomal gene fusions in LPS patient samples, identifying potentially clinically relevant events in certain chromosomal regions.
 Methods:
 The BostonGene internal LPS cohort (n=150) was analyzed by bulk whole-transcriptome sequencing, using STAR-fusion for sequence calling. Quality control was performed using FastQC, FastQ Screen, RSeQC, and MultiQC. Tumor purity was assessed via pathological and bioinformatics examination with a threshold of 20%. One sample Poisson rate test was used to evaluate statistical significance of gene fusion hotspots.
 Results:
 We identified 4,080 gene fusions among four LPS subtypes (DDLPS, WDLPS, MLPS, and PLPS - pleomorphic LPS). Of those, 2,302 (56.4%) were intrachromosomal and 1,778 (43.6%) were transchromosomal. Over half of these fusions (1,263/2,302 intrachromosomal fusions, 54.9%; 1,047/1,778 transchromosomal fusions, 58,9%) were detected on chromosome 12. Most identified fusions occurred in the 12q13-15 region (q-value < 0.001), with q15 being especially prevalent in transchromosomal fusions (q-value < 0.001). The most prevalent recurrent fusion across our LPS cohort was
 FUS-DDIT3
 (N=21, MLPS samples). Another notable recurrent fusion was
 TRIO-TERT
 (N=4, DDLPS). We also identified extended gene fusion hotspots in regions containing important oncogenes such as
 MDM2
 and
 FRS2
 in LPS subtypes (DDLPS, WDLPS, PLPS) that are not considered fusion-driven (Table). These findings suggest an oncogenic role of such fusions in these LPS subtypes, along with known CNAs like
 MDM2
 amplification.
 Conclusions:
 Our comprehensive transcriptomic analysis of gene fusions in LPS samples uncovered both new and established hotspots of chromosomal rearrangements. Identification of such hotspots improves our understanding of LPS oncogenesis and thus can enhance the diagnostic accuracy and discovery of new biomarkers.


 Hotspots of gene fusions in liposarcoma.




 Hotspot cytoband
 Diagnosis
 q-value
 Important genes




 1q23.3
 DDLPS
 < 0.001

 ATF6



 1q24.3
 DDLPS, WDLPS
 < 0.001

 DNM3



 12q13.3
 MLPS
 < 0.001

 DDIT3



 12q14.1
 DDLPS, WDLPS
 < 0.001

 CDK4



 12q14.3
 DDLPS, WDLPS
 < 0.001

 HMGA2, YEATS4



 12q15
 DDLPS, WDLPS, PLPS
 < 0.001

 MDM2, FRS2, CPM



 16p11.2
 MLPS
 < 0.001

 FUS





">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3d585e6b1b252046eb1085a1fdfa70c3a17980" target='_blank'>
              Detecting hotspots of intra- and transchromosomal fusions in liposarcomas by RNA sequencing.
              </a>
            </td>
          <td>
            Dmitrii Grachev, Danil Ivanov, Oleg Baranov, Vladimir Kushnarev, Melissa Clemons, Sheila T. Yong, N. Kotlov, Konstantin Chernyshov, A. Bagaev, N. Fowler, A. Conley, G.M. Cote, Sant P Chawla
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 10615


 Background:
 ATM
 is a tumor suppressor gene involved in DNA repair and telomere maintenance.
 ATM
 biallelic pathogenic or likely pathogenic germline variants (gPV) are associated with

 ataxia-telangiectasia syndrome. Monoallelic ATM gPV are associated with increased cancer risk; however, their contribution to carcinogenesis has not been elucidated. We sought to characterize the genomic landscape of gPV
 ATM
 -associated cancers in a pan-cancer cohort.
 Methods:
 ATM
 alterations (germline and somatic) were identified in patients (Pts) with a solid tumor diagnosis sequenced with MSK-IMPACT, an FDA-approved, tumor-normal paired targeted NGS. Genetic-imputed ancestry clinicopathologic characteristics and FACETS estimated allele-specific copy number profiles were evaluated. Samples with purity <0.2 or with undetermined copy number profiles were excluded. HRD score was evaluated using FACETS profiles. Both somatic and germline sequencing data were analyzed within protocol NCT01775072.
 Results:
 Among 40,136 Pts with cancer who underwent germline testing, 1.1% (n=442) harbored an
 ATM
 gPV, inclusive of 2 Pts with biallelic
 ATM
 gPVs. The most frequent mutations were: R2547_S2549del (n=17), K2756* (n=14) and V1268* (n=11). Among these, R2547_S2549del and K2756* were identified exclusively in Pts with European ancestry, while E343Ifs*2 (n= 9/10) and c.1065+1G>T (n= 6/7) were most prevalent in Pts with Ashkenazi Jewish (AJ) ancestry. Concomitant gPVs in other genes were observed in 10% (n=47), with the most frequent ones being
 APC
 (I1307K),
 MUTYH
 ,
 BRCA2, CHEK2
 , and
 BRCA1
 . 63,270 out of 86,039 tumor samples had usable FACETS profiles.
 ATM
 somatic allele-specific information was available for 67% (297/442) of Pts. 62% (197/317) of tumor samples from Pts with monoallelic
 ATM
 gPVs exhibited somatic biallelic (Bi)
 ATM
 inactivation, while 38% (120/317) retained monoallelic (Mono) ATM status. The underlying mechanism of Bi-ATM inactivation was loss of heterozygosity in 75% (149/197) and additional somatic
 ATM
 mutations in 25% (48/197). Samples from Pts with tumors known to be associated with ATM gPVs (Breast, Pancreatic and Prostate cancers) had significant enrichment in Bi-ATM inactivation compared to tumors without strong association with ATM gPVs (88% Vs 58%; p<0.01). Although samples with Bi-
 ATM
 had higher overall HRD-scores compared to Mono-
 ATM
 (median = 34 Vs 20; p<0.01), no significant enrichment in HRD-High phenotype was seen in Bi-
 ATM
 (23 Vs 18; p=0.49).
 Conclusions:
 Evaluation of a pan-cancer Pt population with
 ATM
 gPVs demonstrated a high prevalence of biallelic somatic inactivation. Most Pts with
 ATM
 gPVs who had malignancies implicated in
 ATM
 -associated cancer risk, had biallelic somatic inactivation in their tumors suggestive of their contribution to tumorigenesis. While biallelic ATM inactivation is associated with higher genomic instability, no enrichment in HRD phenotype was observed.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171b6148e94d64a55d3944e70772bbb94d153ffa" target='_blank'>
              Clinicopathologic and allele-specific analysis of germline
 ATM
 alterations in a pan-cancer cohort.
              </a>
            </td>
          <td>
            Matteo Repetto, Daniel Muldoon, P. Selenica, Kanika Arora, Ying L. Liu, M. Carlo, Lauren Banaszak, Alicia J Latham, C. Bandlamudi, E. O'Reilly, M. E. Robson, B. Rousseau, D. Mandelker, A. Drilon, Britta Weigelt, K. Offit, Michael F. Berger, Y. Murciano-Goroff, Z. Stadler, Mohammad Abbass
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Background Precursor lesions like polyps and adenomas in the colon commonly precede colorectal cancer (CRC), advancing through the “normal-polyp-adenoma-carcinoma” sequence towards malignancy. Yet, the cellular heterogeneity and molecular mechanisms involved in CRC development remain inadequately characterized. Methods To understand the molecular mechanisms driving the onset and progression of CRC, we conducted a comprehensive analysis of ten clinical colorectal samples representing sequential pathological stages using single-cell RNA sequencing (scRNA-seq). Validation was performed through immunofluorescence and immunohistochemistry analyses in a separate human colorectal tissue cohort. Additional verification was carried out using bioinformatics analyses of public TCGA and GEO datasets. Results Our comprehensive analyses not only reveal the cellular diversity and transcriptomic differences throughout disease progression but also highlight the importance of leveraging ligand–receptor gene expression to distinguish various cell subtypes. Subsequent examination and validation with a larger sample cohort uncover the specific involvement of ligand–receptor genes, transcription factors, immunoglobulin genes, and heat shock genes in regulating immune responses and microenvironment changes during colorectal tumorigenesis. Conclusions Our extensive transcriptome dataset provides valuable insights and acts as a fundamental resource to deepen our understanding of the complex molecular landscape in CRC. This dataset facilitates improved diagnostic accuracy and the creation of more personalized therapeutic approaches. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06785-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e15cc889bc4d3cf181a1a8c15dbc72bc14a7b3" target='_blank'>
              A dynamic molecular landscape in colorectal cancer progression at single-cell resolution
              </a>
            </td>
          <td>
            Jianwen Sheng, Nianshuang Li, Shuai Li, Yuman Ye, Yuanhang Liao, Yupeng Zhang, Huan Chen, Mengxian Tu, Guping Zhong, Chaoliang Xiong, Pan Zheng, Yuting Lei, Zekun He, Xingxing He, Yao Zhang, Lei Wang, Xiaolin Gao, Yin Zhu, Jianping Liu, Huizhen Fan
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract MYC proteins are potent oncoproteins that drive tumorigenesis in a wide range of cancers, making it critical to understand their oncogenic functions and underlying mechanisms. Although MYC overexpression induces transcriptional and replication‐associated stress, recent studies have paradoxically identified MYC as a key resilience factor that protects cancer cells from these stressors. In this review, we explore the dual role of MYC in both driving and mitigating cellular stress to achieve its oncogenic function. We also examine how MYC‐induced transcriptional and replicative stress generates potentially immunogenic nucleic acid species while simultaneously helping cancer cells evade host immune recognition. We propose a model in which MYC plays a critical role in managing the stress it induces, thereby maintaining a balance that promotes tumor growth. Based on this model, we discuss potential therapeutic strategies targeting MYC‐dependent stress responses, offering new avenues for cancer treatment and highlighting the complexity of MYC‐driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ce286c658baf7f61e9c844f34e8e87797cdc79" target='_blank'>
              MYC: The Guardian of Its Own Chaos
              </a>
            </td>
          <td>
            A. Gaballa, Bastian Krenz, Leonie Uhl
          </td>
          <td>2025-06-09</td>
          <td>Bioessays</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Oncogenic viruses typically manipulate host cellular mechanisms to drive tumorigenesis. They exploit pioneering transcription factors to modify gene expression, enabling uncontrolled proliferation. These viruses alter chromatin accessibility and induce chromatin remodelling, disrupting DNA repair and promoting viral genome integration. Additionally, epigenetic reprogramming through mechanisms like DNA methylation and histone modifications silences tumor suppressor genes and activates oncogenes. Understanding these mechanisms is critical for identifying more improved therapeutic targets, improving diagnostics, and predicting disease progression. Advances in this field can guide the development of innovative treatments and early detection tools. This comprehensive review synthesizes existing knowledge on the contributions of oncogenic viruses such as hepatitis B virus (HBV), hepatitis C virus (HCV), human papillomavirus (HPV), and human T-cell leukaemia virus type 1 (HTLV-1), Epstein–Barr virus (EBV), human herpesvirus 8 (HHV-8), and Merkel cell polyomavirus (MCV) to cancer development, highlighting their therapeutic relevance and driving forward research in viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36b6df879c24dbe78b1949e0a2207cf7cabf08ee" target='_blank'>
              The role of pioneering transcription factors, chromatin accessibility and epigenetic reprogramming in oncogenic viruses
              </a>
            </td>
          <td>
            M. Kgatle, S. Mbambara, Leon Khoza, Olalekan Fadebi, T. Mashamba-Thompson, M. Sathekge
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37e8efeea3f80c12980365d094cd561d8a7d326" target='_blank'>
              LINE-1 ORF1p is a shared and immunogenic antigen in cancer
              </a>
            </td>
          <td>
            Wilson McKerrow, Megan Snyder, Chong Chu, Deena Kelly, Heike Keilhack, Liyang Diao
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2944900d47d857ea99c57a79dff86ef949dfd7c" target='_blank'>
              Genomic alterations in the YAP/TAZ pathway are associated with stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, W. Li, T. A. Gonzalez, A. Tehim, H. Xu, S. ElNaggar, M. Myers, R. Bareja, P. Dorsaint, K. Gorski, M. Asad, M. A. Assaad, B. D. Robinson, M. Sigouros, E. Barnett, J. Manohar, S. Tagawa, D. Nanus, A. Molina, J. Nauseef, C. N. Sternberg, J. Mosquera, H. Scher, A. Sboner, B. Raphael, Y. Chen, E. Khurana
          </td>
          <td>2025-07-02</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Aging and cancer, though distinct biological processes, share overlapping molecular pathways, particularly in epigenetic regulation. Among these, DNA methylation is central to mediating gene expression, maintaining cellular identity, and regulating genome stability. This review explores how age-associated changes in DNA methylation, characterized by both global hypomethylation and focal hypermethylation, contribute to the emergence of cancer. We discuss mechanisms of DNA methylation drift, the development of epigenetic clocks, and the role of entropy and epigenetic mosaicism, in aging and tumorigenesis. Emphasis is placed on how stochastic methylation errors accumulate in aging cells and lead to epiallelic shifts and gene silencing, predisposing tissues to malignant transformation, even despite recently increased cancer incidences at younger ages. We also highlight the translational potential of DNA methylation-based biomarkers, and therapeutic targets, in age-related diseases. By framing cancer as a disease of accelerated epigenetic aging, this review offers a unifying perspective and calls for age-aware approaches to both basic research and clinical oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80ee6ccb4c2cbc13d84b76cf48a4e779ceb4234" target='_blank'>
              DNA Methylation, Aging, and Cancer
              </a>
            </td>
          <td>
            Himani Vaidya, Jaroslav Jelinek, Jean-Pierre J. Issa
          </td>
          <td>2025-06-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95bb823c01319c257e9b78eae2e9bd2984b27b68" target='_blank'>
              CRISPR activation screens map the genomic landscape of cancer glycome remodeling
              </a>
            </td>
          <td>
            John Daly, Lidia Piatnitca, Mohammed Al-Seragi, Vignesh K Krishnamoorthy, S. Wisnovsky
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a leading cause of cancer-related mortality among men worldwide. Despite progress in the understanding of tumor biology, the prognosis for advanced prostate cancer remains poor, necessitating the identification of novel diagnostic, prognostic, and therapeutic biomarkers. Methuosis, a recently identified form of programmed cell death (PCD), is characterized by cytoplasmic vacuole accumulation and subsequent cell rupture, distinct from classical apoptosis and necrosis. The key regulatory gene in Methuosis, ARF6 (ADP-ribosylation factor 6), has emerged as a potential marker for cancer diagnosis and treatment. However, its role in prostate cancer and other malignancies remains insufficiently understood. Methods In this study, we performed a comprehensive pan-cancer multi-omics analysis to investigate the role of ARF6 in Methuosis across multiple cancer types, with a specific focus on PCa as the primary context. Using data from public databases, including RNA sequencing, gene expression profiling, and clinical outcomes, we assessed the association between ARF6 expression and patient prognosis in PCa within this broader pan-cancer framework. Additionally, we employed functional enrichment analyses and survival analysis to explore the potential of ARF6 as a diagnostic and prognostic marker for prostate cancer. Immunotherapy-related gene expression signatures were also evaluated to determine the therapeutic relevance of ARF6. Results ARF6 was significantly overexpressed in PCa tissues compared to normal tissues and was associated with poor prognosis (p < 0.05), particularly in advanced and metastatic stages. Receiver operating characteristic (ROC) analysis revealed a diagnostic AUC of 0.792 for ARF6. Functional analyses indicated that ARF6 regulates pathways critical to cell migration, invasion, and drug resistance. Moreover, ARF6 expression showed a strong negative correlation with immune checkpoint markers, such as PD-L1 (r = − 0.74), suggesting its potential as an immunotherapy target. These findings underscore ARF6’s pivotal role in Methuosis and its promise as a biomarker in PCa. Conclusion ARF6 is a key regulator of Methuosis in prostate cancer, contributing to tumor progression, metastasis, and resistance to treatment. Our findings support the potential of ARF6 as a diagnostic, prognostic, and immunotherapeutic target in prostate cancer. Further experimental validation is needed to confirm these observations and to explore the therapeutic implications of targeting ARF6 in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51c475c0dde283a17762878be80bdc39f46da628" target='_blank'>
              Methuosis key gene ARF6 as a diagnostic, prognostic and immunotherapeutic marker for prostate cancer: based on a comprehensive pan-cancer multi-omics analysis
              </a>
            </td>
          <td>
            Yongjin Yang, Liangliang Qing, Chengyu You, Qingchao Li, Wenbo Xu, Zhilong Dong
          </td>
          <td>2025-05-23</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 How can we improve the safety and efficiency of germline gene correction using CRISPR/Cas9 in early mouse embryos?



 We observed a higher frequency of desirable gene editing events when using crRNA XT for CRISPR/Cas9-mediated gene correction during fertilization of mouse oocytes.



 CRISPR/Cas9 induces double-stranded DNA breaks, which can be repaired via homology-directed repair (HDR) using a repair template to achieve precise gene editing. Undesired events may occur, including non-homologous end joining, leading to error-prone repair causing insertions or deletions (indels), unrepaired DNA breaks resulting in large deletions or partial chromosomal loss, or HDR using the untargeted parental chromosome (gene conversion), causing loss of heterozygosity.
 HDR frequencies in early embryos are typically much lower compared to gene editing outcomes in somatic cell lines. In this study, we aimed to improve HDR efficiency by using chemically modified crRNA optimized for high nuclease environments.



 To achieve gene correction with reduced mosaicism, we co-injected CRISPR/Cas9 along with mutant C57BL/6J mouse sperm (carrying a homozygous Plcz1 deletion, causing failed fertilization after ICSI) into wild-type B6D2F1 and DBA/2 mouse oocytes for subsequent developmental and detailed genetic analysis, respectively.
 We analysed 59 embryos injected with crRNA XT and 60 embryos injected with classical crRNA. The repair template used contained synonymous variants, which, if detected, indicate successful gene correction through HDR with template use.



 We injected CRISPR/Cas9 ribonucleoprotein complexes, designed for correcting the Plcz1 deletion, simultaneously with mutant mouse sperm into wild-type mouse oocytes during piezo-assisted ICSI, followed by artificial activation. We compared the use of crRNA XT, containing additional modifications for improved functional stability, to classical crRNA. After four days of in vitro culture, DNA was extracted and used for targeted sequencing, whole genome amplification, and copy number variation (CNV) analysis to evaluate the events following gene editing.



 After gene correction using classical crRNA, targeted sequencing revealed that the deletion was undetectable in 8 out of 60 embryos (13.3%). However, none of these embryos contained the synonymous variants which would indicate HDR with the provided repair template. In contrast, using crRNA XT, the deletion was no longer detected in 22 out of 59 embryos (37.3%). The synonymous variants were found in 50% of the embryos with undetectable deletions, suggesting a remarkable increase in HDR rates with the repair template. These results indicate that crRNA XT, with its enhanced stability, leads to a remarkable improvement in the frequency of desired gene editing events.
 To explain further the events in which the deletion is no longer detected without incorporation of the repair template’s synonymous variants, we performed whole-genome amplification and copy number variation (CNV) analysis using shallow whole-genome sequencing. In the embryos edited using classical crRNA, we observed partial chromosomal loss of chromosome 6 (carrying the Plcz1 gene) in 1 out of 7 analysed embryos and aneuploidy with partial chromosomal loss of chromosome 6 in 1 out of 7 analysed embryos. CNV analysis of embryos edited with crRNA XT is still ongoing to complete our comparison.



 The resolution of the shallow whole-genome sequencing is limited to 5 Mbp. To detect larger deletions smaller than this threshold, we will perform long-read sequencing.



 Targeted sequencing of embryos undergoing early gene correction for the Plcz1 deletion revealed that the use of crRNA XT, which offers greater stability, significantly enhanced the frequency of HDR with the repair template compared to classical crRNA. These findings contribute to a deeper understanding of germline gene editing events.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797fea7f3b3a355f785c9bfcca4bad7300fab085" target='_blank'>
              P-804 Improving CRISPR/Cas9 gene correction in early mouse embryos using RNase-resistant crRNA XT
              </a>
            </td>
          <td>
            S. Jooris, A. Rybouchkin, F. Jalali, J. Parrington, F. Vanden Meerschaut, T. Hochepied, K. Claes, D. Stoop, B. Heindryckx, B. Menten
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Eukaryotic centromeres highlight the remarkable plasticity of eukaryotic chromosomes through their conserved functionality and sequence divergence. Holocentric chromosomes, where centromere activity is distributed along the entire chromosome length, offer a unique model for investigating the molecular mechanisms underlying adaptive evolution between centromeres and chromosomes. In this review, we summarise and speculate on the multiple changes and prerequisites potentially involved in the evolution of holocentromeres. The interplay between environmental factors, chromosomal rearrangements, and centromere plasticity drives the transition from regional to holocentric characteristics. The centromeric histone H3 (CenH3) protein mediates neocentromere formation by recognising non-centromeric chromosomal regions with appropriate AT content, thereby facilitating chromosome restructuring in the transition from regional to holocentric chromosomes. Dynamic changes in repetitive sequences provide functional sites for centromere assembly, chromosomal recombination and repair and centromere spreading and maturation. Epigenetic modifications maintain functional coordination among multiple centromeric units by modulating chromatin states, CenH3 localisation, and kinetochore assembly. This review provides a comprehensive framework for understanding the evolutionary mechanisms of holocentromeres derived from monocentromere and offers insights into the design of artificial centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34691ddef62fc1465f67e23ac70d8368228f19a" target='_blank'>
              Eukaryotic Centromere Remodeling: Plasticity, Dynamics, and Holocentromere Formation.
              </a>
            </td>
          <td>
            Dan Yang, Zhaoxin Xiao, Ke Li, Jiayi Hou, Fengfeng Zhang, Jianjun Qiao, Ning Li, Mingzhang Wen
          </td>
          <td>2025-05-27</td>
          <td>Plant, cell & environment</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Homologous recombination is a high-fidelity DNA repair mechanism essential for maintaining genome stability. Impairment of this pathway, often due to BRCA1 or BRCA2 inactivation, leads to homologous recombination deficiency (HRD), forcing cells to rely on error-prone mechanisms for repairing DNA double-strand breaks, such as nonhomologous or microhomology-mediated end joining. HRD is a clinically important biomarker, particularly in breast and ovarian cancers, as it predicts responsiveness to platinum-based chemotherapies and PARP inhibitors. However, current tests in the clinical setting, mostly based on targeted panel sequencing, lack specificity and lead to a substantial number of false positives. In contrast, whole-genome sequencing, despite its high accuracy, remains largely confined to research because of high costs and logistical constraints. In this issue of Cancer Research, Abbasi and colleagues present HRProfiler, a machine learning-based tool that accurately detects HRD using whole-exome sequencing (WES) data, which is increasingly accessible in clinical oncology. Notably, it demonstrates improved sensitivity in the WES setting compared with existing tools, such as HRDetect and SigMA. As WES continues to gain traction, HRProfiler offers a promising step toward democratizing HRD detection and enabling more precise, genomics-guided treatment strategies. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI. See related article by Abbasi et al., p. 2504.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab81bfb225ffd3e5881d30b8dbaccdbb884d86b5" target='_blank'>
              Leveraging Whole-Exome Sequencing and Mutational Signatures to Detect Homologous Recombination Deficiency in Cancer.
              </a>
            </td>
          <td>
            Joonoh Lim, Young Seok Ju
          </td>
          <td>2025-07-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Ultraviolet (UV) light causes cyclobutane pyrimidine dimers (CPDs) and other DNA lesions that must be efficiently repaired to prevent cell death and mutagenesis. While mammals utilize the nucleotide excision repair (NER) pathway to repair CPDs, many species primarily utilize photolyase enzymes to repair UV damage. Our understanding of how different genomic and chromatin features impact photolyase repair across a eukaryotic genome is limited. Here, we map repair of CPDs by photolyase across the yeast genome at single-nucleotide resolution. Our data indicate that yeast photolyase repairs CPDs more rapidly than NER, but photolyase activity is inhibited at certain classes of transcription factor binding sites and in nucleosomes. Repair in nucleosomes is particularly inhibited when CPDs are located along the 3′ side of the nucleosomal DNA or at minor-in rotational settings. Our data indicate that photolyase efficiently repairs the non-transcribed strand of yeast genes, but repair of the transcribed strand (TS) is inhibited. Genome-wide analysis of UV-induced mutations in NER-deficient photoreactivated yeast reveals a striking enrichment of mutations along the TS of yeast genes. These data indicate that inhibition of photolyase repair along the TS, likely due to occlusion of CPDs by RNA polymerase II stalling, promotes UV mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7e81bd67cc35d46ad0fa3937b62cb144d357b6" target='_blank'>
              Genome-wide maps of UV damage repair and mutation suppression by CPD photolyase
              </a>
            </td>
          <td>
            Kaitlynne A. Bohm, Marian F. Laughery, P. A. Mieczkowski, Steven A. Roberts, John J Wyrick Data Curation
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The spatial resolution of omics dynamics is fundamental to understanding tissue biology. Spatial profiling of DNA methylation, which is a canonical epigenetic mark extensively implicated in transcriptional regulation, remains an unmet demand. Here, we introduce a method for whole genome spatial co-profiling of DNA methylation and transcriptome of the same tissue section at near single-cell resolution. Applying this technology to mouse embryogenesis and postnatal brain resulted in rich DNA-RNA bimodal tissue maps. These maps revealed the spatial context of known methylation biology and its interplay with gene expression. The two modalities’ concordance and distinction in spatial patterns highlighted a synergistic molecular definition of cell identity in spatial programming of mammalian development and brain function. By integrating spatial maps of mouse embryos at two different developmental stages, we reconstructed the dynamics of both epigenome and transcriptome underlying mammalian embryogenesis, revealing details in sequence, cell type, and region-specific methylation-mediated transcriptional regulation. This method extends the scope of spatial omics to DNA cytosine methylation for a more comprehensive understanding of tissue biology over development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e580ba04766eb95572371b96391de4351133faff" target='_blank'>
              Spatial joint profiling of DNA methylome and transcriptome in mammalian tissues
              </a>
            </td>
          <td>
            Chin Nien Lee, Hongxiang Fu, Angelysia Cardilla, Wanding Zhou, Yanxiang Deng
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef9a4a9c46c279f0d7bcc3c37222b01d2f23045" target='_blank'>
              Minimal activation of the p53 DNA damage response by a modular cytosine base editor enables effective multiplexed gene knockout in induced pluripotent stem cells
              </a>
            </td>
          <td>
            R. Blassberg, Olga Mielczarek, J. Stombaugh, H. Okkenhaug, Alice Abreu Torres, Paul Russell, John Lambourne, Immacolata Porreca
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc1079a033fb614c3439063ef9f1c82d7614ef4" target='_blank'>
              Phage-Encoded TelN Inhibits Mre11-Rad50 to Protect Hairpin Telomeres
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The epigenetic deregulation of CpG islands (CGIs) plays a crucial role in cancer initiation and progression. CGIs comprise 1-2% of the human genome and are rich in differentially methylated regions (DMRs) that can serve as cancer biomarkers in clinical samples and liquid biopsies. Focusing epigenetic sequencing on CpG-rich sequences, including CGIs and avoiding non-informative regions, offers an efficient and sensitive approach for cancer identification and tracking, especially within samples containing excess of unaltered, normal DNA. To this end, we have developed Adaptor-anchored Methylation amplification via Proximity Primers (aMAPP), a versatile PCR-based enrichment method. aMAPP employs specially designed primers to selectively enrich either methylated or unmethylated CpGs, depending on the upstream methylation conversion method employed. aMAPP achieves high coverage of genome-wide CGIs and detects hundreds of DMRs in tumor samples compared to adjacent normal tissue using ultra-low depth sequencing (∼300,000 reads). It enables tracing of aberrant methylation down to allelic frequency 0.01% in dilutions of tumor DNA and in cell-free DNA samples, can be applied using picogram amounts of DNA, and can be adapted to enrich either small panels of cancer-specific DMRs, or the majority (>90%) of genomic CGIs and CpGs. aMAPP offers a simple, cost-effective, and highly sensitive approach for capturing the epigenetic footprint of genome-wide CpGs and identifying aberrantly methylated or un-methylated genomic regions. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030972eec1be1d377d8b7bfd84130f3e834f7bbf" target='_blank'>
              Genome-wide extraction of differentially methylated DNA regions using adapter-anchored proximity primers
              </a>
            </td>
          <td>
            Farzaneh Darbeheshti, Hayet Radia Zeggar, Hamzeh Salmani, Yibin Liu, Ruolin Liu, V. Adalsteinsson, G. M. Makrigiorgos
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Epitranscriptomics has emerged as a rapidly evolving field that focused on studying post-transcriptional RNA modifications and their role in spatiotemporal regulation of gene expression. N6-methyladenosine (m6A) and 5-methylcytosine (m5C) represent the most extensively studied modifications on mRNAs. These reversible modifications, mediated by ‘writer,’ ‘eraser,’ and ‘reader’ proteins, dynamically fine-tune mRNA stability, splicing, and translation. Growing evidence links their dysregulation to pathological states, including cancer progression and metastasis, where their aberrant deposition on oncogenes or tumor suppressors alters cellular signaling and therapeutic responses. In the current study, we present a detailed analysis of the m5C epitranscriptomic landscape across distinct breast cancer molecular subtypes. Using CRISPR/Cas9, we confirm NSUN2 as a key m5C writer in human mRNAs. NSUN2 loss was validated by targeted sequencing and Western blotting. Furthermore, we demonstrate the regulatory effects of NSUN2 on its canonical mRNA targets, revealing its critical role in maintaining proper gene expression networks. Our findings strongly suggest that additional m5C writers contribute to m5C methylation machinery. Additionally, we assessed the functional impact of NSUN2 depletion on mRNAs harboring m5C sites using mRNA stability assays. Furthermore, our analysis revealed distinct m5C methylation patterns among breast cancer subtypes, highlighting unique m5C signatures associated with the disease. Notably, we identified specific hypomethylated and hypermethylated m5C sites in each breast cancer cell line, representing a universal m5C methylation signature for breast cancer. Our study constitutes the first comprehensive m5C epitranscriptomic atlas in human breast cancer and paves the way for future research aimed at developing targeted therapeutic interventions that leverage the m5C methylation landscape. Supplementary Information The online version contains supplementary material available at 10.1007/s10142-025-01648-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c7ab96a995706a16195a85c4f63f01b36d8d6b" target='_blank'>
              Exploring the m5C epitranscriptome of mRNAs in breast cancer cells through genome engineering and long-read sequencing approaches
              </a>
            </td>
          <td>
            Konstantina Athanasopoulou, P. Adamopoulos, P. Tsiakanikas, Andreas Scorilas
          </td>
          <td>2025-06-25</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61956d7589478355803ed0351b120e6a0e356110" target='_blank'>
              SLFN11 Enhances Cisplatin Sensitivity in Pediatric Cancer via Activation of Stress-Response and Suppression of Survival Pathways
              </a>
            </td>
          <td>
            A. Jabeen, Dina Awartani, S. Sherif, Eiman I. Ahmed, R. Alanany, Ayman Saleh, W. Hendrickx, C. Raynaud
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Retroviral and retrotransposon invasion pose a constant threat to genome integrity and have driven the evolution of host defense pathways able to counter these attacks. The human silencing hub (HUSH complex) is an epigenetic transcriptional repressor complex that recognizes and silences newly integrated retroelements through the establishment of ectopic heterochromatin and chromatin compaction. HUSH provides a genome-wide immunosurveillance system whose challenging task is to detect and silence any newly integrated retroelements, and it thus plays a key role in host defense. To distinguish self from nonself genomic DNA, HUSH recognizes long single-exon (intronless) DNA, the essential hallmark of reverse transcription. Retroelements, being RNA derived, lack classical, noncoding cellular introns, so a long, intronless sequence of DNA is the abnormal molecular pattern that allows HUSH to distinguish invading retroelements from intron-containing host genes. As a newly identified component of the innate immune system, HUSH protects the genome from the reverse flow of genetic information from RNA to DNA, revealing an unanticipated mechanism of postintegration genome immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b1d5426e615e5a445a45b06d1bcad8bd22f0c5" target='_blank'>
              Silencing by the HUSH Epigenetic Transcriptional Repressor Complex.
              </a>
            </td>
          <td>
            Paul J. Lehner
          </td>
          <td>2025-06-01</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Double‐strand breaks (DSBs) are universally acknowledged as the most detrimental type of DNA damage, and their effective repair primarily depends on the non‐homologous end joining (NHEJ) pathway. Such DSBs, which require NHEJ for resolution, can arise from intrinsic and extrinsic DNA‐damaging factors or emerge naturally during essential biological processes like V(D)J recombination and antibody class switch recombination.Failure to properly repair DSBs may lead to genomic instability, disruption of cellular functions, and immunodeficiency, thereby promoting the development of hematologic malignancies. Conversely, overexpression of NHEJ‐related genes can enhance resistance to DNA‐damaging therapies in these cancers. Analyzing mutations in key classical NHEJ (cNHEJ) components and understanding their mechanisms could provide valuable biomarkers for predicting therapeutic outcomes and guiding treatment decisions. Consequently, defects in cNHEJ may offer insights into the development of novel drugs targeting DNA repair pathways.We focus on genetic changes and alterations in gene regulation, while also providing an overview of cNHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d192621ee63d84a5e302bfd454c006794e79d5" target='_blank'>
              Current advances in the role of classical non‐homologous end joining in hematologic malignancies
              </a>
            </td>
          <td>
            Pengcheng Liu, Zi-zhen Xu
          </td>
          <td>2025-05-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA topoisomerases are essential for maintaining DNA topology, gene expression and the accurate transmission of genetic information. Mitochondria possess circular DNA (mtDNA), which, unlike nuclear chromosomes, lacks protective histones and exists in nucleoprotein complexes called nucleoids, which are vital for mtDNA stability. Although the mitochondrial genome encodes essential genes involved in ATP production via oxidative phosphorylation, it does not encode crucial mtDNA maintenance genes and depends entirely on nuclear-encoded proteins for mtDNA maintenance. These include nuclear-encoded topoisomerases (i.e. Top1mt, Top2α, Top2β and Top3α), which alleviate topological stress during mtDNA transcription and replication, and mitochondrial transcription factor A (TFAM), are crucial for ensuring proper nucleoid structure and mtDNA packaging. Furthermore, tyrosyl-DNA phosphodiesterase 1 and 2 (TDP1 and TDP2) participate in the repair of mtDNA damage associated with trapped topoisomerase-mtDNA complexes, which can compromise mtDNA integrity and contribute to neurodegeneration, cancer and premature aging. Drugs that stabilize these protein-DNA adducts (PDAs) to induce mtDNA damage and mitochondrial dysfunction are promising new strategies for cancer therapy. This Review explores the essential roles of mitochondrial topoisomerases, overviews mechanisms involved in mtDNA repair and discusses how mitochondrial fission and mitophagy are employed as a survival strategy for clearing damaged mtDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc13f830961da01cf0f5401d6373dc48a97645fa" target='_blank'>
              Mitochondrial topoisomerases, nucleoid architecture and mtDNA repair in human disease.
              </a>
            </td>
          <td>
            Sangheeta Bhattacharjee, B. Das
          </td>
          <td>2025-07-01</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94b93bbdecb6b043f4125dc44dd985694b019b6" target='_blank'>
              ZNF865 (BLST) Regulates Human Cell Senescence and DNA Damage
              </a>
            </td>
          <td>
            Christian Lewis, H. Levis, Jonah Holbrook, Jacob T. Polaski, Timothy D. Jacobsen, S. Gullbrand, B. Diekman, J. Iatridis, Jason Gertz, Brandon Lawrence, Robert D Bowles
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Lymphomas are highly heterogeneous malignancies characterized by diverse genetic, epigenetic, and phenotypic profiles, which complicate diagnosis, prognosis, and treatment. Traditional bulk sequencing methods mask the complexity of intratumoral variation, often overlooking rare subclones that may drive disease progression and therapeutic resistance. Single-cell genomics has emerged as a transformative approach to decipher tumor heterogeneity at unprecedented resolution. This technology enables the dissection of individual cellular populations within lymphomas, offering insights into clonal evolution, transcriptional diversity, and microenvironmental interactions. By applying single-cell RNA sequencing (scRNA-seq), chromatin accessibility assays (scATAC-seq), and single-cell DNA sequencing (scDNA-seq), researchers can unravel lineage relationships, identify resistant subpopulations, and track dynamic changes in response to therapy. In lymphomas such as diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL), and mantle cell lymphoma (MCL), single-cell approaches have revealed distinct malignant and non-malignant cell states that correlate with treatment outcomes. Moreover, integrating single-cell data with spatial transcriptomics and immune profiling enhances the understanding of the tumor microenvironment, including immune evasion mechanisms. These insights can inform personalized treatment strategies, identify novel therapeutic targets, and enable early detection of relapse. Despite technical challenges such as data complexity, sample viability, and cost, the application of single-cell genomics in lymphoma research is rapidly advancing. Future directions include multi-omics integration, real-time patient monitoring, and clinical translation of predictive biomarkers. This review underscores the pivotal role of single-cell genomics in resolving tumor heterogeneity and predicting therapeutic resistance, positioning it as a cornerstone for next-generation precision oncology in lymphomas,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/438856abbab164e715372bfe3f2a6c07df9594ec" target='_blank'>
              Role of mutational signatures and clonality assessments in tailoring targeted therapies for lymphoma
              </a>
            </td>
          <td>
            Andoh Sheikh Atta-ullah, Uche Philip, Sampson Janice Candy
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: DNA replication and sister chromatid cohesion1 (DSCC1) is a component of the alternative replication factor C complex (RFC), which plays an important role in sister chromatid cohesion and regulates the cell cycle. DSCC1 is reported to have increased expression in tumor progression, but the underlying mechanisms of DSCC1 in tumor immunity remain obscure. Methods: Data were collected from The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) project to obtain DSCC1 expression and clinicopathologically relevant data. Differential expression of DSCC1 and its association with prognosis, tumor microenvironment, immune infiltration, immune regulation, and genomic stability in various cancers were analyzed using R software. Results: The results of pan-cancer analysis showed that DSCC1 expression was elevated in 22 tumors. In 15 tumors, high expression of DSCC1 was observed to correlate with dismal overall survival. Furthermore, a correlation between DSCC1 expression and immune checkpoints was determined. In addition, DSCC1 expression was correlated with tumor-infiltrating immune cells, especially in Thymoma (THYM). Finally, gene set enrichment analysis (GSEA) demonstrated that DSCC1 is critically involved in tumor proliferation, immunity, and metabolism. Conclusions: High DSCC1 expression is found in many common tumors and is associated with poor prognosis. The gene is an essential factor in sister chromatid cohesion and may contribute to tumor progression by affecting the tumor immune microenvironment and genomic stability. DSCC1 has the potential to be a prognostic marker, and therapies targeting it may benefit patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175f7489907d723129eee83d91ea9ec6a461cb27" target='_blank'>
              Gene DSCC1 is a Potential Biomarker in Pan-cancer
              </a>
            </td>
          <td>
            Jialing Jiang, Xiaolong Yan, Dan Huang, Jieyu Chen, Xiaodan Hao
          </td>
          <td>2025-05-29</td>
          <td>Journal of Contemporary Medical Practice</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Transposable elements (TE), also called transposons, are repetitive DNA sequences making up half of the human genome. Initially, TEs were described as “junk DNA” because they lack specific function. However, they have been recognized for their ability to replicate and integrate into different genomic locations; this “jumping gene” activity results in genomic instability, variation of the chromosomal architecture, and transcriptional dysregulation, all of which represent major hallmarks of cancer. In this respect, the involvement of TE in tumorigenesis was extensively studied and their role as diagnostic and therapeutic tools in cancer is now well-established. Transposons’ products, including TE-derived cancer antigens, transcripts, and associated epigenetic modifications, mainly hypomethylation, were found to be promising biomarkers in several types of cancer ensuring early disease detection and prognosis. In addition, TE are currently used to design innovative approaches, with transposon-based systems, namely, Sleeping Beauty and PiggyBac, enabling precise genomic modifications and novel strategies for cancer therapy. Therefore, the aim of this review is to provide an overview on the dual application of TE as diagnostic and therapeutic tools in cancer, paving the way to improved clinical outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4636ade80b58aaf2a75f12fb6871d3d8ab2244c5" target='_blank'>
              Exploring transposable elements: new horizons in cancer diagnostics and therapeutics
              </a>
            </td>
          <td>
            Ahlam Chaaban, Reem Sleem, Jana Santina, Mohamad Rima, José-Noel Ibrahim
          </td>
          <td>2025-07-12</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Among the various genotypes of the human papillomavirus (HPV), the HPV16 genotype is the most oncogenic by far, and some (sub)lineages may be associated with an increased risk of cancer. Thus, characterizing the full genome is important to understand the link between the HPV16 genome variability and its transforming role. We set up a multiplex PCR approach combined with Oxford Nanopore Technologies for the HPV16 full-genome characterization. The primers were designed with PrimalScheme, and the optimization of the monoplex/multiplex PCR was performed on SiHa cells. After library preparation, the sequencing was performed on a GridION sequencing device to determine the sensitivity, the specificity, and the performance of the method. Fourteen primer pairs were selected to span the full genome. With the monoplex PCR, 12 primer pairs showed good amplification, while a double concentration of two primers was required to improve the amplification. We then performed a multiplex PCR approach by generating two pools comprising non-overlapping primer pairs. The multiplex PCR showed good sensitivity, allowing the amplification of HPV16 with a Ct value below 27 and providing a coverage greater than 99.9% and a sequencing depth greater than 100×. In addition, the specificity of the method was validated by the absence of amplification of other high-risk HPV genotypes compared with HPV16. We set up an amplicon-based approach to characterize the full genome and diversity of the HPV16 (sub)lineages. This approach shows good sensibility and specificity with limited cost, opening new perspectives in the field of whole-genome HPV sequencing. IMPORTANCE HPV16 is by far the most oncogenic genotype, so characterizing the genetic variability of its genome is important to better understand the link with its transforming role. We developed an amplicon-based approach combined with Oxford Nanopore Technologies next-generation sequencing to overlap the HPV16 genome, which is easy to implement in the laboratory and inexpensive in the field. HPV16 is by far the most oncogenic genotype, so characterizing the genetic variability of its genome is important to better understand the link with its transforming role. We developed an amplicon-based approach combined with Oxford Nanopore Technologies next-generation sequencing to overlap the HPV16 genome, which is easy to implement in the laboratory and inexpensive in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/836ca2c5fafa3f7dae222db6854bdcbbe566eaa9" target='_blank'>
              An amplicon-based approach for full-genome characterization of HPV16
              </a>
            </td>
          <td>
            I. Malet, Inès Draa, V. Leducq, Fanny Vuong, Pascale Bonnafous, A. Marcelin, V. Calvez, A. Jary
          </td>
          <td>2025-06-09</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="
 Background: While recent advances have improved outcomes for Triple Negative Breast Cancer (TNBC), it continues to have a poor prognosis. Long non-coding RNAs are among a recent class of epigenetic regulators that function in the nucleus to support the stability of cells and maintain the fidelity of chromatin interactions. Our laboratory discovered the long noncoding RNA MANCR (LINC00704) as being upregulated in human breast cancer. Furthermore, our work demonstrated that MANCR is enriched in TCGA breast cancer patient samples that are not estrogen or progesterone receptor positive. We also demonstrated that the 10-year survival in this TCGA analysis is substantially worse in patients with high MANCR expression. More recently, we have found MANCR to be aberrantly expressed in TNBC cells, and these cells are highly dependent on MANCR to retain their tumorigenic characteristics. Methods: Functional in vitro studies in MDA-MB-231 TNBC cells used short antisense nucleic acids (GapmerRs) to knockdown MANCR. For in vivo studies, TNBC cells were injected into the mammary fat pad of mice, allowing tumor formation, and subsequent treatment with 2 nmol/g of a negative control or MANCR targeting GapmeR. We also identified genome interaction sites at single nucleotide resolution by chromatin isolation by RNA purification sequencing (ChIRP-seq) to determine the mechanism of MANCR activity in TNBC cells. Results: We now demonstrate that MANCR knockdown promotes DNA damage and decreases cell proliferation, migration, anchorage-independent colony formation, transwell invasion, and cellular survival. Additionally, in vivo targeting of MANCR drastically inhibited tumor growth over time and the end-point tumor mass. The MANCR GapmeR treatment also inhibited the ability of TNBC cells to circulate and disseminate to distant organs in vivo. After performing ChIRP-seq in the MDA-MB-231 cells, we identified 1206 genome-wide binding sites that exhibit MANCR interactions, of which 48% are intergenic and 52% are in genic regions. Furthermore, many MANCR ChIRP peaks were found to overlap with fragile sites in the genome, indicating MANCR provides stability to these sites. Conclusions: These data suggest that targeting MANCR has therapeutic potential for patients with “MANCR-high” TNBC tumors by disrupting genome stability. Indeed, our in vivo studies demonstrate that “MANCR-high” TNBC tumors require MANCR to rapidly grow and promote disease progression. Significantly, many of the MANCR-chromatin interactions identified were found in intergenic regions and overlap with fragile sites within the genome. Collectively, these data strongly indicate that MANCR stabilizes the TNBC genome, and disrupting genome stability by targeting MANCR has therapeutic potential.
 Citation Format: Janine S. A. Warren, Bodhisattwa Banerjee, Jonathan A. R. Gordon, Prachi N. Ghule, Janet L. Stein, Gary S. Stein, Jane B. Lian, Peter A. Kaufman. Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb40dcaca0c24edab0b1fb3286bc3d79cdad166" target='_blank'>
              Abstract P1-07-17: Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Janine Warren, Bodhisattwa Banerjee, J. Gordon, Prachi N Ghule, Jane L. Stein, G. S. Stein, J. Lian, Peter A. Kaufman
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Ewing sarcomas (ESs) are aggressive paediatric tumours of bone and soft tissues afflicting children and adolescents. Despite current therapies having improved the 5-year survival rate to 70% in patients with localized disease, 25% of patients relapse and most have metastasis at diagnosis. Resistance to chemotherapy, together with the high propensity to metastasize, remain the main causes of treatment failure. Thus, identifying novel targets for alternative therapeutic approaches is urgently needed. Methods Biochemical and functional analyses were carried out to elucidate the mechanism of regulation of YAP1 expression by pncRNA_YAP1-1 in ES cells. Results Here, we identified a novel promoter-associated noncoding RNA, pncRNA_YAP1-1, transcribed from the YAP1 promoter in ES cells. We found that pncRNA_YAP1-1 level exerts antitumour effects on ES by destabilizing YAP1 protein. The molecular mechanism relies on the interaction of pncRNA_YAP1-1 with the RNA binding protein FUS, which stabilizes the transcript. Furthermore, pncRNA_YAP1-1 binding to TEAD impairs its interaction with YAP1, thus determining YAP1 translocation into the cytoplasm, its phosphorylation and degradation. Conclusions Overall, our findings reveal a novel layer of regulation of YAP1 protein expression by pncRNA_YAP1-1 in Ewing sarcoma. Considering the role of YAP1 in therapy response and cell propensity to metastasize, our results indicate pncRNA_YAP1-1 as an actionable target that could be exploited to enhance chemotherapy efficacy in Ewing sarcoma. Significance PncRNA_YAP1-1 counteracts the YAP1 oncogenic transcriptional program in Ewing sarcoma cells by interfering with YAP1-TEAD interaction and impairing YAP1 protein stability. These findings uncover a novel treatment option for Ewing sarcoma. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00736-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/945aa0f8515a96b425b539864e7b5d32769ff4ff" target='_blank'>
              YAP1 promoter-associated noncoding RNA affects Ewing sarcoma cell tumorigenicity by regulating YAP1 expression
              </a>
            </td>
          <td>
            L. Chellini, Arianna Del Verme, Veronica Riccioni, M. Paronetto
          </td>
          <td>2025-05-25</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ead10baece9545ae1e3801dde43ce3d766e518" target='_blank'>
              METTL5 Blockade Enhances Anti-Tumor Immune Response via Inducing Neoantigen Generation
              </a>
            </td>
          <td>
            Yangyi Zhang, Xiaoyan Shi, Yuci Wang, Ruiqi Wang, Yanlan Cao, Hao Chen
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human papillomavirus associated head and neck cancer (HPV + HNC) is rising globally, emphasizing the need for improved therapeutic and screening strategies. To test novel therapies and study HPV-related disease progression, it is vital to develop relevant preclinical models. However, many fail to address critical concerns, including generating a representative immune microenvironment and adequately modeling HPV-driven malignant transformation. Recent multi-omics studies reveal the significance of HPV integration location in HPV-related carcinogenesis and highlight the necessity of targeted treatment methods. Thus, we have developed a murine model of HPV16 + HNC modifying the published CRISPR-based Somatic Oncogene kNock-In for Cancer Modeling (CRISPR-SONIC) system for precise integration of HPV oncogenes. We showed that CRISPR-SONIC knock-in of KrasG12D, HPV16 E6 and E7, and a luciferase reporter at the murine β-actin 3’-UTR locus could induce spontaneous buccal tumors with sarcomatous morphology under transient or selective immunosuppression. Both preventative and therapeutic pNGVL4a-CRT/E7(detox) DNA vaccination could induce HPV16 E7-specific immune response and reduce tumor growth. Furthermore, CRISPR-SONIC knock-in of HPV16 E6 and E7 with co-delivery of HNC-relevant oncogenes AKT and c-Myc produced tumors in NSG mice capturing the characteristic carcinomic morphology of HPV + HNC. Overall, our model offers a robust platform for evaluating new therapies and exploring HPV-related carcinogenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01427-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c396727d3082a56902b37a32cd0299a9e91f92a0" target='_blank'>
              Generation of a spontaneous murine HPV + oral cancer model with site-specific oncogene insertion using CRISPR-SONIC
              </a>
            </td>
          <td>
            Julia Tao, Jason Murray, Hsin-Fang Tu, Darrell Fan, Y. Tsai, Ming-Hung Hu, Annie A Wu, Deyin Xing, Chien-Fu Hung, T-C Wu
          </td>
          <td>2025-06-18</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) contributes to 3% of all childhood cancers with roughly 400-500 cases diagnosed each year in the United States. The World Health Organization classifies rhabdomyosarcoma into four histological subtypes which include alveolar, embryonal, spindle-cell and pleomorphic. The primary genetic drivers in a subset of alveolar and spindle-cell histological subtypes are gene fusions. This review explores the fusion oncogenes identified in RMS such as PAX-and NCOA2-based fusions, along with discussing studies defining fusion oncogene biology and tumorigenic mechanisms. Focus areas include data around transformation events and progression along with dysregulated biological processes. Furthermore, we summarize model systems, ranging from cell to animal models, that have been implemented to study fusion oncogenes identified in RMS. With the constant identification of novel fusion oncogenes, this review also emphasizes the need for genetically characterizing RMS tumors and rapidly developing new model systems. These models are critical to study fusion oncogene activity and to delineate key regulatory players and potential therapeutic targets that suppress tumorigenesis. The identification of RMS fusion oncogenes and integration with animal and cell culture models will help identify conserved molecular targets, optimize therapeutic approaches, and ultimately improve clinical outcomes for children with RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b23d84102b5431b0bc6bd415779fd94bf54425c" target='_blank'>
              Fusion oncogenes in rhabdomyosarcoma: model systems, mechanisms of tumorigenesis, and therapeutic implications
              </a>
            </td>
          <td>
            Chinmay S. Sankhe, Lisa Hall, Genevieve C. Kendall
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genomic instability underlies various diseases, including cancer. This instability arises from defects in critical cellular processes, particularly those involved in DNA damage repair. Therefore, a detailed understanding of these repair mechanisms is essential for developing strategies to prevent or diagnose such diseases. The MRN complex, composed of MRE11, NBS1, and RAD50, is among the earliest elements involved in detecting DNA damage. Upon detecting DNA breaks, this complex triggers a cascade of signaling events that regulate both cell cycle arrest and DNA repair. These signaling pathways are tightly controlled by various post-translational modifications, notably ubiquitination. Although several ubiquitin ligases have been implicated in different stages of the DNA damage response, our knowledge remains limited. In this study, we reveal that βTrCP, a substrate-recognizing subunit of the SCF (SKP1/CUL1/F-box protein) ubiquitin ligase, interacts in vivo with the proteins of the MRN complex. These interactions occur in normally proliferating cells and are dependent on the GSK3 kinase. Moreover, we show that βTrCP enhances the recruitment of the MRN complex to chromatin through MRE11, thereby promoting the efficient DNA damage repair. Hence, alterations in βTrCP function affecting MRN dynamics could have severe consequences for the cell homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47da9e36f36ee547f6e7e42f2ff12a54a26f4687" target='_blank'>
              βTrCP facilitates MRN complex localization on chromatin to enhance DNA repair
              </a>
            </td>
          <td>
            Alejandro Belmonte-Fernández, Joaquín Herrero-Ruíz, Carmen Sáez, M. Japón, M. Mora-Santos, F. Romero
          </td>
          <td>2025-07-11</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The DNA damage response comprises a set of imperfect pathways that maintain cell survival following exposure to DNA damaging agents. Cancers frequently exhibit DNA repair pathway alterations that contribute to their intrinsic genome instability. This, in part, facilitates a therapeutic window for many chemotherapeutic agents whose mechanisms of action often converge at the generation of a double-strand DNA break. The development of therapy resistance occurs through countless molecular mechanisms that promote tolerance to DNA damage, often by preventing break formation or increasing repair capacity. This review broadly discusses the DNA damaging mechanisms of action for different classes of chemotherapeutics, how avoidance and repair of double-strand breaks can promote resistance, and strategic directions for counteracting therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77a69284f3f1f8221d2572829a8005590b029476" target='_blank'>
              DNA repair and the contribution to chemotherapy resistance
              </a>
            </td>
          <td>
            K. Nešić, Phoebe Parker, Elizabeth M. Swisher, John J Krais
          </td>
          <td>2025-05-26</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/774b7db029018d408461488af66408e59c0c7b28" target='_blank'>
              Physical principles of phase-separation action on chromatin looping associated to pathogenic gene activation
              </a>
            </td>
          <td>
            Sougata Guha, Andrea Fontana, Alex Abraham, Simona Bianco, Andrea Esposito, Mattia Conte, Sumanta Kundu, Ciro Di Carluccio, Francesca Vercellone, Florinda Di Pierno, Mario Nicodemi, A. Chiariello
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Recent advancements in genomic technologies have become critical tools for deciphering the genetic complexities of cancer tissues, enabling precision medicine strategies aimed at improving patient clinical outcomes. Here we performed a comprehensive analysis of clinically annotated whole genome and whole transcriptome sequences from 1,364 breast cancer cases. Our investigation provides the most detailed genomic landscapes of breast cancer to date, which allowed us to comprehensively correlate genomic changes with clinical characteristics. Our findings include, but not limited to, (1) whole-genome-based homologous recombination deficiency profiles effectively predict responses to adjuvant chemotherapy and first-line CDK4/6 inhibitor treatment, (2) focal ERBB2 amplifications, often arising via extrachromosomal DNA mechanisms, are strongly associated with a positive response to neoadjuvant chemotherapy. Additionally, our study identified (3) recurrent copy number amplifications linked to patient survival, and (4) whole-genome-based intratumoral heterogeneity profiles that can predict patient survival as well as response to anti-HER2 therapies. Our work underscores the power of highly annotated whole-genome and transcriptome sequencing of a population-scale cancer cohorts for uncovering clinically relevant genomic mutations and novel molecular targets, thereby advancing precision oncology strategies for breast cancer.
 Citation Format: Ryul Kim,JongHan Yu, Joonoh Lim, Brian Baek-Lok Oh, Seok Jin Nam, Seok Won Kim, Jeong Eon Lee, Byung Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, Chang Ik Yoon, Young Joo Lee, Dooreh Kim, Kabsoo Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, Erin Connolly-Strong, Sangmoon Lee, Bo Rahm Lee, Yuna Lee, Ki Jong Yi, Young Oh Kwon, In Hwan Chun, Junggil Park, Jihye Kim, Chahyun Choi, Jong Yeon Shin, Hyunjung Lee, Minji Kim, Hansol Park, Ilecheon Jeong, Boram Yi, Won-Chul Lee, Jeong Seok Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park. Whole-genome landscapes of 1,364 breast cancers with clinical outcomes [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-01-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea155495e72da0b7ab53fc77458b87559af5c5e0" target='_blank'>
              Abstract P1-01-06: Whole-genome landscapes of 1,364 breast cancers with clinical outcomes
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, S. Kim, Jeong-Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, Jung-Woon Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyunjun Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeongmin Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Y. Park
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Zwilch Kinetochore Protein(ZWILCH) has been reported to prevent cells from prematurely exiting mitosis. However, the underlying mechanisms or involvement of ZWILCH in the tumor immune microenvironment in various cancers remain largely unknown. Methods Generalized dysregulation of ZWILCH was observed through the whole transcriptome analysis in this study. The spatial transcriptome analysis was utilized to identify expressed regions of ZWILCH. Next, cells that mainly expressed ZWILCH in the tumor microenvironment were determined using the single-cell transcriptome analysis. Also, the “cellchat” R package was applied to estimate the effect of ZWILCH on malignant cell communication. Combining multiple analytic approaches including GSEA, GSVA, KEGG enrichment analysis, and Aucell, with TCPA functional protein data, Genome-wide CRISPR screening, potential functions of ZWILCH and the pathways in which ZWILCH participated were thoroughly exploited. Univariate Cox regression analysis calculated the association between ZWILCH and cancer patients’ adverse outcomes. Results ZWILCH is universally highly expressed in tumors. The spatial transcriptome analysis showed that ZWILCH overexpression comes from the tumoral region or mixed tumoral region. At the single-cell level, ZWILCH is chiefly expressed by malignant cells and proliferative T cells. The expression of ZWILCH mRNA is positively correlated with cell proliferation, repair of DNA damage, and cell cycle score. Plenty of metabolic pathways are inhibited in patients with high expression of ZWILCH. Moreover, after ZWILCH knockout, a large number of cancer cell lines are stagnated, inhibited, or died. Additionally, the malignant cells with positive expression of ZWILCH have a stronger ability for cell communication. In short, ZWILCH is meant to be a risk factor for clinical outcomes of multiple tumors. Conclusions ZWILCH is a promising therapeutic target that influences patient prognosis by participating in cell proliferation, cell communication, and reshaping the tumor microenvironment across different cancers. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02981-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73f14c1c066485285bab172ec02a8c4271d565b9" target='_blank'>
              Zwilch kinetochore protein affects the prognosis of cancer patients by participating in cell proliferation, enhancing cell communication, and reshaping the tumor microenvironment
              </a>
            </td>
          <td>
            Long Yao, Lianpo Liu, Jinsong Wu, Yunlong Huang, Renquan Zhang, Haoxue Zhang
          </td>
          <td>2025-07-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="HSD3B1 encodes an enzyme that catalyzes the conversion of adrenal precursors into potent sex steroids. A common germline variant (c.1100C) enhances this effect and is linked to breast cancer (BC) progression. As the HSD3B1 genotypes contribute to differences in local and adrenal steroid production, their transcriptional and phenotypic effects on cancers influenced by hormonal signaling such as BC and endometrial cancer (EC)—particularly in relation to menopausal status—remain unclear. We analyzed BC and EC sequenced from patients that received diagnostic tests in oncology clinics, and we determined the germline HSD3B1 c.1100 genotype (AA, AC, CC) from tumor DNA sequencing by using variant allele frequency, with inferred menopausal status assumed by age at molecular profiling. Whole-transcriptome RNA sequencing and gene set enrichment analysis showed that adrenal-permissive homozygous (CC) tumors in premenopausal ER + BC were enriched for hormone-related pathways, including Estrogen Response Early (NES ≈ +1.8). In premenopausal triple-negative BC, adrenal-restrictive homozygous (AA) tumors exhibited the elevated expression of immune and epithelial genes and the increased prevalence of MED12 alterations (AA 0.25% vs. CC 8%, p < 0.01). In endometrioid EC, CC tumors demonstrated the suppression of immune and proliferative pathways. Postmenopausal cases had higher progesterone receptor IHC positivity (AA 75% vs. CC 83%, p < 0.05) and numerically more frequent ESR1 copy number gains (AA 2.0% vs. CC 4.0%). Results highlight context-specific associations between germline HSD3B1 genotypes and tumor biology in BC and EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83fbb8500f0f6e5e010ceeaf4c87e837f593cf33" target='_blank'>
              HSD3B1 (c.1100C) Genotype Is Associated with Distinct Tumoral and Clinical Outcomes in Breast and Endometrial Cancers
              </a>
            </td>
          <td>
            Nikitha Vobugari, Allison Makovec, Samuel Kellen, Shayan S. Nazari, Andrew Elliott, Devin Schmeck, Aiden Deacon, Gabriella von Dohlen, Emily John, Pedro C. Barata, N. Agarwal, Melissa A. Geller, Britt K. Erickson, George W Sledge, J. Ostrander, Rana R McKay, Charles J. Ryan, Nima Sharifi, Emmanuel S. Antonarakis, Justin H. Hwang
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background The human genome contains approximately 3,200 near full-length autonomous human endogenous retroviral (HERV) genomes distributed across the 23 chromosomes. These autonomous HERV proviral genomes include long terminal repeats (LTRs) capable of promoting RNA transcription. In quiescent cells, most HERV loci remain transcriptionally silent. However, environmental changes, such as epigenetic remodeling of chromatin, can activate these silenced loci. Methods To study HERV reactivation, we previously analyzed autonomous HERV expression patterns in monocytes isolated from peripheral blood mononuclear cells (PBMCs) identified in single-cell RNA sequencing (scRNA-seq) databases using the Azimuth application. We developed a Window-based HERV Alignment (WHA) method, which analyzes aligned DNA sequences using sequential, non-overlapping windows of defined lengths. Samples were scored as positive (>= 9 good/usable windows) or negative (<= 8 good/usable windows). Results Using WHA, we established a control set from 31 normal individuals, with fewer than 8 windows at selected HERV loci. We analyzed scRNA-seq data from three studies of hospitalized COVID-19 patients and found distinct HERV expression patterns in monocytes. Unique patterns were also found in patients with influenza, Dengue virus, or sepsis. We next examined HERV expression at early (<7 days) and late (>14 days) timepoints post COVID-19 recovery and detected HERV loci in both groups. Analyzing 12 patients with post-acute sequelae of COVID-19 (PASC), we identified three HERV loci expressed in all patients. Some loci showed amplified numbers of good/usable windows, indicating longer transcripts and greater sequence depth. The most amplified locus was located within an intron of JAKMIP2, which, along with neighboring host genes, also showed increased transcription. Conclusion Previous studies have shown that viral infections, including COVID-19, influenza, and Dengue virus, as well as sepsis, can induce innate immune memory in monocytes through epigenetic remodeling of hematopoietic stem and myeloid precursor cells. The identification of co-amplified HERV loci and neighboring host gene transcripts in monocytes from PASC patients suggests expansion of epigenetically remodeled myeloid progenitors. The identification of these HERV-host gene patterns provides a foundation needed to understand the clinical features of patients with PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3890c566bb6e206114c5472e39818cdcd77102fc" target='_blank'>
              Amplification of select autonomous HERV loci and surrounding host gene transcription in monocytes from patients with post-acute sequelae of COVID-19
              </a>
            </td>
          <td>
            Hyunmin Koo, Casey D. Morrow
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Invasive lobular carcinoma (ILC) is the second most common histological subtype of breast cancer after invasive ductal carcinoma of no special type (NST), accounting for 10–15% of diagnoses. Despite the myriad molecular, histological and clinical differences between ILC and NST tumors, patients are treated in the same way, and although prognosis initially is good, ILC patients have poorer long-term outcomes. Understanding the differences between these two subtypes and identifying ILC-enriched therapeutic targets is necessary to improve patient care. Methods Human and mouse cancer-associated fibroblasts (CAFs), ILC cell lines and patient-derived organoids were used for in vitro and in vivo studies, including western blotting, migration, organotypic invasion assays and dissemination in zebrafish embryos. RNASeq was used to identify CAF and interleukin-6 (IL-6)-derived gene signatures. Bioinformatic analysis of public databases and immunohistochemical of human tumor microarrays was carried out. Results We identified IL-6 as a paracrine CAF-derived factor that activates Signal-Transducer-and-Activator-of-Transcription-3 (STAT3) in human and mouse ILC models. Analysis of human breast tumors showed that the IL-6/JAK/STAT3 pathway is enriched in ER + ILC compared to ER + NST. A 42-gene CAF dependent IL-6 gene signature and 64-gene consensus IL-6 gene signature were generated and were significantly enriched in ER + ILC, with many of the genes overexpressed in ILC tumors. IL-6 treatment suppressed downstream estrogen signaling and also led to the acquisition of a more mesenchymal-like phenotype associated with increased migration and invasion. Finally, IL-6 treatment significantly increased ILC cell dissemination following injection into zebrafish embryos. Conclusions CAF-derived IL-6 drives paracrine activation of the IL6/JAK/STAT3 signaling pathway which is enriched in ILC. This leads to the acquisition of pro-tumorigenic phenotypes, highlighting the pathway as a potential therapeutic target in ILC. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02074-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ff0735a08b68cfe9b8184d5f4b41c0bb6234a9" target='_blank'>
              Cancer-associated fibroblast driven paracrine IL-6/STAT3 signaling promotes migration and dissemination in invasive lobular carcinoma
              </a>
            </td>
          <td>
            Esme Bullock, Aleksandra Rozyczko, Sana Shabbir, Ifigenia Tsoupi, Adelaide I J Young, J. Travnickova, Laura Gómez-Cuadrado, Zeanap Mabruk, Giovana Carrasco, Elizabeth Morrow, K. Pennel, Pim Kloosterman, J. Houthuijzen, Jos Jonkers, L. Avalle, Valeria Poli, R. Iggo, Xue Xiao, Jingjing Guo, Xuan Zhu, Elizabeth Mallon, Joanne Edwards, E. E. Patton, Valerie G. Brunton
          </td>
          <td>2025-07-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Pediatric brain tumors, the most devastating cancers affecting children, are believed to originate from neural stem/progenitor cells in developing brain. In precise timing and specific regions during the brain development, chromatin deregulation plays crucial roles in redirecting normal neuronal differentiation pathways toward tumorigenesis. Indeed, epigenomic abnormalities are thought to be more important for brain tumor formation especially in children than adults, as pediatric brain tumors generally exhibit fewer genetic mutations compared to adult brain tumors. Given the small number of mutations, targeting such limited alterations involved in cancer epigenomes is expected to be more effective in pediatric brain tumors. The mechanisms of cancer epigenomes include mutation or dysregulation of chromatin remodelers, histone modifiers, histones themselves, and DNA methylation enzymes. Furthermore, genomic rearrangements and/or higher-order chromatin topology also contribute to these epigenomic mechanisms. These mechanisms are commonly observed in various types of pediatric brain tumors. However, alterations in chromatin regulatory factors differ across tumor types, reflecting the unique epigenetic landscapes shaped by their tumor origins. Accordingly, clarifying their functional similarities and differences across tumor types could offer valuable insights for finding new therapeutic strategies. Thus, this review article focuses on elucidating how pediatric brain tumors arise from epigenomic deregulation and what epigenetic molecules or mechanisms could serve as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a334571a5c8d012851148672d620a490ae61353" target='_blank'>
              Epigenetic modifications and their roles in pediatric brain tumor formation: emerging insights from chromatin dysregulation
              </a>
            </td>
          <td>
            Kento Kawata, Owen S. Chapman, Satoshi Narumi, Daisuke Kawauchi
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="VIRMA (also known as KIAA1429), as a core regulatory subunit of the m6A methyltransferase complex, plays a key role in tumorigenesis and progression by dynamically regulating RNA methylation modifications. Studies have shown that VIRMA is aberrantly overexpressed in more than 20 types of malignant tumors, including liver cancer, breast cancer, and lung cancer, and is significantly associated with chromosome 8q amplification and poor prognosis. Its mechanism of action involves regulating the expression of tumor-associated genes through both m6A-dependent and m6A-independent pathways, thereby promoting tumor proliferation, metastasis, and drug resistance. These findings suggest that VIRMA has the potential to serve as a pan-cancer diagnostic and prognostic biomarker. This review summarizes the role of VIRMA in malignant tumors from multiple perspectives and explores its potential applications in clinical diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1f0fb758f358894db60bb1aa6efc139d365cc4" target='_blank'>
              The Multifaceted Role of VIRMA, a Core Component of the Methyltransferase Complex, in Cancer and Cancer Therapy
              </a>
            </td>
          <td>
            Jinmeng Lu, Chengyu Zhang, Mengshuang Yin, Huili You, Chao Xiong, Jing Wu, Ying Gong, Zhangang Xiao, Jing Shen
          </td>
          <td>2025-06-22</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background Inflammatory breast cancer (IBC), of which HER2 + is the predominant subtype, is extremely aggressive and difficult to treat. Previous studies have suggested that targeting the tumour microenvironment (TME) may provide new directions for IBC diagnosis and treatment. Methods In this study, we used single-cell transcriptome technology (scRNA-seq) to investigate the molecular features of the TME of HER2 + IBC patients and performed a comprehensive and detailed comparison of the cellular components and molecular phenotypes of the TME between IBC patients and noninflammatory breast cancer (nIBC) patients to elucidate the cell types that are specifically enriched in the TME of IBC patients, as well as the molecular features that are responsible for the preferential remodelling of the cellular functional state in the TME. Results A total of 15,832 cells, including epithelial cells, endothelial cells, stromal cells, T cells, B cells, antibody secreting cells (ASCs) and myeloid cells, were obtained from tumour tissues from 3 HER2 + IBC patients for scRNA analysis. By comparing the TME with that of HER2 + nIBC patients in a public database, we found that the TME of HER2 + IBC patients had a greater level of lymphocyte infiltration than that of nIBC patients did, and an especially significant enrichment of ASCs (mainly plasmablasts or plasma cells). In the TME of HER2 + IBC patients, tumour-infiltrating T cells exhibited a dual molecular phenotype of high activation and high exhaustion, with tumour-infiltrating B cells preferring the extrafollicular developmental pathway, and tumour-infiltrating myeloid and mesenchymal cells exhibiting a greater immunosuppressive status. By performing a cellular interaction analysis, we revealed that PTN molecules were significantly overexpressed in HER2 + IBC tumour cells and that the cellular interactions mediated by these molecules were strongly correlated with the functional polarisation of the cellular components in the TME. We observed that HER2 + IBC tumour cells have an active interferon response and epithelial mesenchymal transition (EMT) signalling, and that their malignant process is strongly correlated with the inflammatory response. Moreover, we found that HER2 + IBC tumour-infiltrating B cells promoted necroptosis of endothelial cells through high expression of TNF, thus promoting inflammatory responses. Conclusion We found a strong correlation between high expression of PTN molecules in HER2 + IBC tumour cells and their highly invasive characteristics and highly immunosuppressive TME. These results suggest that HER2 + IBC tumour cells can promote an inflammatory response by upregulating the expression of TNF molecules in B cells via PTN molecules and that the enhanced inflammatory response in turn promotes tumour progression, a malignant cycle that shapes a more immunosuppressive TME. Therefore, diagnostic and therapeutic strategies targeting the PTN-TNF molecular axis may have considerable potential for development in HER2 + IBC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03454-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9c28eda21e6edafced575cd62b5c13be2df5e23" target='_blank'>
              Single-cell transcriptome analysis reveals the malignant characteristics of tumour cells and the immunosuppressive landscape in HER2-positive inflammatory breast cancer
              </a>
            </td>
          <td>
            Juan Huang, Yong-Gui Zhu, Wenjing Zeng, Yulong Zhang, Weizhi Xia, Fan Xia, Liyu Liu, Kuansong Wang, Yidi Guan, Taohong Shen, Bingjian Jiang, Lunquan Sun, Ayong Cao, Shouman Wang, Zhi Li
          </td>
          <td>2025-07-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In mammals, the binding of transcriptional activators leads to the repositioning of silent loci from the nuclear periphery to the nuclear interior. However, it remained unknown whether the same mechanism functions in Drosophila. Here, using FISH and DamID, we have shown that binding the GAL4 activator to the silent loci causes weakening of their interactions with the nuclear lamina and relocalization inside nuclei in Drosophila salivary gland cells and neurons. This mimics the removal from the nuclear periphery of a neuron-specific gene upon its activation in neurons. Salivary gland cells contain polytene chromosomes with mechanical properties, different from chromosomes of diploid cells, while neurons represent predominantly non-dividing cell type. Our results indicate a causal relationship between transcriptional activator binding and changes in the intranuclear position of loci in Drosophila. They also point to the similarity in general chromatin dynamics in mammals and Drosophila, thus strengthening the role of model organisms in studying genome architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a72e89c238e1e68a064b0436b325874b1c8e025b" target='_blank'>
              Binding of Transcriptional Activator to Silent Loci Causes Their Detachment from the Nuclear Lamina in Drosophila Neurons and Salivary Gland Cells
              </a>
            </td>
          <td>
            R. Simonov, O. Olenkina, V. Nenasheva, Yuri A. Abramov, Sergey A. Lavrov, A. Fedotova, Y. Y. Shevelyov
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaee95d8b33babc294494d923d1f29bffe920cba" target='_blank'>
              Gain of Function p53 mutant R273H confers distinct methylation profiles and consequent partial or full EMT states to colon tumour
              </a>
            </td>
          <td>
            H. Rani, Seemadri Subhadarshini, M. Jolly, V. Mahadevan
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5484870331a7d0dee6989fa1ddf5ea8e0b01c6cb" target='_blank'>
              Functional genomics and tumor microenvironment analysis reveal prognostic biological subtypes in Mantle cell lymphoma
              </a>
            </td>
          <td>
            Sunandini Sharma, Roshia Ali, Alyssa Bouska, Dylan Jochum, Meghana Kesireddy, Simeon Mahov, Joseph Lownik, Weiwei Zhang, Waseem G Lone, Mahfuza Afroz Soma, Alicia Gamboa, Vaishnavi Devarkonda, Dalia Elgamal, Atqiya Fariha, Adnan Mansoor, Douglas Stewart, Peter Martin, Brian K. Link, R. Advani, Paul M. Barr, A. Goy, A. Mehta, M. Kamdar, D. Stephens, Veronika Bachanova, L. Smith, Ryan D Morin, P. Pararajalingam, M. Lunning, Kai Fu, Dennis Wiesenberger, Wing C. Chan, J. Khoury, T. Greiner, J. Vose, Akil Merchant, Chengfeng Bi, J. Iqbal
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1387277252c00fad4b66b55a8349b13921b432" target='_blank'>
              MitoPerturb-Seq identifies common and gene-specific single-cell responses to mitochondrial DNA depletion and heteroplasmy
              </a>
            </td>
          <td>
            Stephen P. Burr, Kathryn Auckland, Angelos Glynos, Abhilesh S. Dhawanjewar, Wei Wei, Cameron Ryall, Antony M. Hynes-Allen, Malwina Prater, Matylda Sczaniecka-Clift, Julien Prudent, Patrick F. Chinnery, Jelle van den Ameele
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Glioma is a highly malignant primary tumor of the central nervous system, characterized by highly invasive behavior and a high recurrence rate. The current treatment options for gliomaoften have unsatisfactory outcomes. Replication protein A1 (RPA1), a component of the RPA heterotrimer, has been identified as a potential oncogene implicated in the development and clinical prognosis of various types of solid tumors, including liver cancer, nasopharyngeal carcinoma and gastrointestinal cancer. However, studies on the associations between RPA1 expression and glioma cell proliferation or patient prognosis are limited, and the underlying mechanisms remain unclear. The aim of the present study was to explore the expression of RPA1 in glioma and its clinical relevance in the clinicopathology and prognosis of glioma. In addition, the biological functions and signal transduction pathways mediated by RPA1 were predicted using bioinformatics analysis to provide basic theoretical support for further mechanistic research and in vivo experiments. The Cancer Genome Atlas (TCGA) data analysis, and reverse transcription-quantitative PCR, western blot analysis and immunohistochemical (IHC) staining were performed to analyze RPA1 expression in glioma cells or tissues of different grades. The associations between RPA1 expression and various pathological parameters related to tumor cell proliferation and patient prognosis were also investigated. Furthermore, the biological processes and signaling pathways influenced by RPA1-related target genes were predicted. The results revealed that RPA1 expression varies among different grades of gliomas, with higher World Health Organization (WHO) grades exhibiting higher RPA expression levels. Furthermore, both bioinformatics analysis and IHC staining results revealed that high RPA1 expression was associated with a shorter overall survival (OS). In addition, RPA1 expression exhibited a significant association with WHO glioma grade and key markers of malignancy, namely Ki-67 expression and p53 mutation status. Gene Ontology and Kyoto Encyclopedia of Genes and Genomes enrichment analyses suggested that RPA1 may promote the development of glioma through pathways including ‘DNA replication’, ‘Fanconi anemia pathway’, ‘mismatch repair’, ‘homologous recombination’, ‘nucleotide excision repair’ and ‘cell cycle’. In conclusion, RPA1 expression in glioma tissues and cells is positively associated with the degree of glioma malignancy, and high RPA1 expression is associated with a shorter OS in patients with glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb9154b84dfbf0461fd48c7dd062feb43da7f114" target='_blank'>
              Role and mechanism of RPA1 in the development and progression of glioma
              </a>
            </td>
          <td>
            Zhiming Zhao, Yu Sun, Ping Yin, Quan Zhang, Jianfu Zhang
          </td>
          <td>2025-06-26</td>
          <td>Experimental and Therapeutic Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell‐cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti‐tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non‐neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of Internal Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna C Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86b4088fda0a17a1413678d49be792807c01757" target='_blank'>
              Human TLS DNA polymerase: saviors or threats under replication stress?
              </a>
            </td>
          <td>
            Y. S. Rajpurohit, Mitu Lal, D. K. Sharma, Ishu Soni
          </td>
          <td>2025-05-23</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Organotropism results from the functional versatility of metastatic cancer cells to survive and proliferate in diverse microenvironments. This adaptivity can originate in clonal variation of the spreading tumor and is often empowered by epigenetic and molecular reprogramming of cell regulatory circuits. Related to organotropic colonization of metastatic sites are environmentally-sensitive, differential responses of cancer cells to therapeutic attack. Accordingly, understanding the organotropic profile of a cancer and probing the underlying driver mechanisms are of high clinical importance. However, determining systematically the organotropism of one cancer versus the organotropism of another cancer, potentially with the granularity of comparing the same cancer type between patients or tracking the evolution of a cancer in a single patient for the purpose of personalized treatment, has remained very challenging. It requires a host organism that allows observation of the spreading pattern over relatively short experimental times. Moreover, organotropic patterns often tend to be statistically weak and superimposed by experimental variation. Thus, an assay for organotropism must give access to statistical powers that can separate ‘meaningful heterogeneity’, i.e., heterogeneity that determines organotropism, from ‘meaningless heterogeneity’, i.e., heterogeneity that causes experimental noise. Here we describe an experimental workflow that leverages the physiological properties of zebrafish larvae for an imaging-based assessment of organotropic patterns over a time-frame of 3 days. The workflow incorporates computer vision pipelines to automatically integrate the stochastic spreading behavior of a particular cancer xenograft in tens to hundreds of larvae allowing subtle trends in the colonization of particular organs to emerge above random cell depositions throughout the host organism. We validate our approach with positive control experiments comparing the spreading patterns of a metastatic sarcoma against non-transformed fibroblasts and the spreading patterns of two melanoma cell lines with previously established differences in metastatic propensity. We then show that integration of the spreading pattern of xenografts in 40 – 50 larvae is necessary and sufficient to generate a Fish Metastatic Atlas page that is representative of the organotropism of a particular oncogenotype and experimental condition. Finally, we apply the power of this assay to determine the function of the EWSR1::FLI1 fusion oncogene and its transcriptional target SOX6 as plasticity factors that enhance the adaptive capacity of metastatic Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Throughout the cell cycle, DNA molecules convert between hierarchical intramolecular (cis) and intermolecular (trans) associations. Cohesin ATPase complexes produce both types of DNA associations which collectively are required for sister chromatid segregation, chromatin condensation, genomic architecture, gene transcription, and DNA repair. The mechanisms that regulate cohesin cis- and trans-activities, however, remain controversial. A popular model is that a regulatory complex (Pds5, Irr1/Scc3, and Rad61) sits atop a core ring-like complex (Mcd1/Scc1, Smc1, and Smc3), the latter of which exhibits the inherent ATPase activities responsible for producing cis-and trans-DNA conformations. Additional proteins transiently interact with cohesins to promote cohesin deposition onto DNA (Scc2 and Scc4) or stabilize cohesin-DNA binding (Eco1/Ctf7). Of these nine components, only RAD61 is non-essential. Recent findings, however, identified pairs of suppressor mutations that support the viability of cells individually deleted for either PDS5 or ECO1/CTF7 (herein ECO1). Intriguingly, CLN2 deletion is common in both suppressor pairs, suggesting that combined suppressor mutations may support the viability of cells co-deleted for both ECO1 and PDS5. These results further suggest that the addition of other suppressor mutations (such as ELG1 and RAD61) may support the viability of cells deleted of all auxiliary subunits - including IRR1/SCC3 (herein SCC3). Here, we test these predictions and report on novel gene deletion combinations required for cell cycle progression and cell viability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a5f10ca02abc4152347a73b1ca267645760052e" target='_blank'>
              Analysis of Combinatorial Cohesin Subunit Gene Deletions in Budding Yeast.
              </a>
            </td>
          <td>
            Grace Duke, R. Skibbens
          </td>
          <td>2025-06-10</td>
          <td>Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="DNA sequencing technologies are widely used to study tumor evolution within a cancer patient. However, analyses require various computational methods, including those to infer clone sequences (genotypes of cancer cell populations), clone frequencies within each tumor sample, clone phylogeny, mutational tree, dynamics of mutational signatures, and metastatic cell migration events. Therefore, we developed GenoPath, a streamlined pipeline of existing tools to perform tumor evolution analysis. We also developed and added tools to visualize results to assist interpretation and derive biological insights. We have illustrated GenoPath’s utility through a case study of tumor evolution using metastatic prostate cancer data. By reducing computational barriers, GenoPath broadens access to tumor evolution analysis. The software is available at https://github.com/SayakaMiura/GP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e039aed15ffae5155d36f38b8ec85975e0960dc5" target='_blank'>
              GenoPath: a pipeline to infer tumor clone composition, mutational history, and metastatic cell migration events from tumor DNA sequencing data
              </a>
            </td>
          <td>
            Ryan M. Tobin, Shikha Singh, Sudhir Kumar, Sayaka Miura
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cf680837c223860a295c7d973297324c3a3da0" target='_blank'>
              From predictive biomarker to therapeutic target: the dual role of SLFN11 in chemotherapy sensitivity.
              </a>
            </td>
          <td>
            Yue Feng, Yingze Li, Zhenhao Zhang, Yuxuan Dai, Xingchun Gou, Kejing Lao, Running Zhang
          </td>
          <td>2025-06-18</td>
          <td>Cancer chemotherapy and pharmacology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="At inception, eukaryotes solved the conflict of the physical dimensions of meter-sized linear chromatin fibers placed into micron-sized nuclei by employing two mechanisms: (i) fragmenting genomic DNA into sets of shorter segments, called chromosomes; (ii) compacting each chromosome by folding it into 3D chromatin arrays. Despite this physical segregation, eukaryotic genomes possess an intrinsic regulatory connectivity that facilitates remarkable chromosome-naive function-aligned integration and striking developmental precision in conveying genetic information from conception and embryogenesis to adulthood. In this contribution, gene ontology-guided proximity placement analyses of 8839 highly conserved retroviral Long Terminal Repeat (LTR) elements fixed in human genomes and linked to 5444 down-stream target genes revealed chromosome-naive compartmentalization and function-aligned integration along major developmental networks of the global genomic regulatory dominion (GRD), components of which operate during human embryogenesis, development, and adulthood. Evolution forged the GRD into a functionally consonant constellation of 26 genome-wide multimodular genomic regulatory networks (GRNs), each of which is defined by significant enrichment of numerous single gene ontology (GO)-specific traits. GRN coordinates appear scattered across chromosomes, occupying 5.5–15.09% of the human genome. Each GRN harbors 529–1486 regulatory LTRs derived from LTR7, MLT2A1, and MLT2A2 sequences, which appear quantitatively balanced according to their genome-wide abundance. GRNs govern the activities of between 199 and 805 down-stream target genes, including transcription factors, chromatin-state remodelers, signal-sensing and signal transduction mediators, enzymatic and receptor-binding effectors, intracellular complexes and extracellular matrix elements, and cell–cell adhesion molecules. GRNs consist of several hundred to thousands of smaller, GO enrichment-defined genomic regulatory modules (GRMs), which include from a dozen to hundreds of LTRs and down-stream target genes. GRMs appear to operate along specific phenotypic avenues on the timescale of an individual’s life-span, exerting profound effects on patterns of transcription, protein–protein interaction, developmental phenotypes, physiological traits, and pathological conditions in Modern Humans. Overall, this study identifies 69,573 statistically significant regulatory LTR-linked GRMs (binominal FDR q-value threshold of 0.001), including 27,601 GRMs validated by single GO-specific directed acyclic graph (DAG) analyses across six GO annotations. Collectively, the reported observations support the hypothesis that pan-primate endogenous retroviruses have made significant impacts on the evolution of the global regulatory connectivity of GRNs affecting the physiological traits and pathological conditions of Modern Humans, including phenotypes whose manifestations are considered human-specific.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3c484aae2c5b02482fb3324d011673d3da9e65b" target='_blank'>
              Genome-wide, chromosome-naïve compartmentalization of major developmental networks tailored by modular placement of human embryo-regulatory Long Terminal Repeats
              </a>
            </td>
          <td>
            G. Glinsky
          </td>
          <td>2025-06-12</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="R-loops, three-stranded RNA-DNA hybrid nucleic acid structures, are recognized for their roles in both physiological and pathological processes. Regulation of R-loops is critical for genome stability as disruption of R-loop homeostasis can lead to aberrant gene expression, replication stress, and DNA damage. Recent studies suggest that the RNA modification, N6-methyladenosine (m6A), can modify R-loops and the writers, erasers, and readers of m6A are involved in the dynamic regulation of R-loops. Here, we discuss the reported functions of various m6A regulatory proteins in relation to R-loops, highlighting their distinct roles in recognizing and modulating the formation, stability, and resolution of these structures. We further examine the functional implications of m6A and R-loop interaction in human diseases, with a particular emphasis on their roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178d222f0ed2c1781f83d81616226ea68d9cb6b9" target='_blank'>
              The Dynamic Interactions of m6A Modification and R-Loops: Implications for Genome Stability
              </a>
            </td>
          <td>
            Nicholas Kim, Hong Sun
          </td>
          <td>2025-06-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Ovarian cancer is one of the most common malignancies of the female reproductive system and is associated with poor prognosis. This study aimed to utilize single-cell RNA sequencing to investigate the heterogeneity of malignant epithelial cells in ovarian cancer, focusing on their potential functions and the implications for treatment and prognosis. Methods Single-cell RNA sequencing data were clustered using a single-cell transcriptome clustering method, and malignant epithelial cells were identified through copy number variation analysis. The interaction patterns between different malignant subpopulations and immune/stromal cells were analyzed using cell-to-cell communication analysis. A risk score (URS) model based on the UBE2C + epithelial subpopulation was then constructed through LASSO and multivariable Cox regression. High and low URS groups were compared in terms of tumor mutational burden (TMB), survival outcomes, and drug sensitivity. Finally, the role of GTF2F2 in ovarian cancer progression was validated through gene knockdown experiments in an ovarian cancer cell line (ES-2). Results Three major malignant epithelial cell subpopulations were identified (TMSB4X + Epi, TSC22D1 + Epi, and UBE2C + Epi). The UBE2C + Epi subpopulation exhibited higher stemness and greater invasive potential. The constructed URS model effectively stratified patients into high- and low-risk groups, with the high-risk group displaying a higher TMB level (p = 0.00011). Drug sensitivity predictions indicated that osimertinib, rapamycin, and dihydrorotenone might have stronger inhibitory effects in the high-risk group, whereas ERK inhibitors were more effective in the low-risk group. Functional assays demonstrated that GTF2F2 knockdown significantly suppressed ovarian cancer cell migration and invasion. Western blot analyses further showed elevated E-cadherin and reduced N-cadherin expression, suggesting that GTF2F2 may promote epithelial-mesenchymal transition (EMT). Conclusion The risk score model established in this study offers a novel framework for patient stratification and personalized therapy. Notably, the identification of the UBE2C + Epi subpopulation and key genes such as GTF2F2 highlights potential diagnostic and therapeutic targets, shedding light on the pathogenesis of ovarian cancer and paving the way for precision medicine approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01686-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519b55d3039bd908fc634a743e91a97b194c1fa0" target='_blank'>
              Single-cell RNA sequencing reveals the role of GTF2F2 in ovarian cancer oncogenesis and progression
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-05-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The cellular networks that maintain genome stability encompass numerous pathways involved in all aspects of nucleic acid metabolism. Through bioinformatic analysis, we identified the Zinc Finger CCCH-Type Containing 4 protein (ZC3H4), a suppressor of noncoding RNA (ncRNA) production, as a pivotal player in this system. Experimentally, ZC3H4 deficiency led to increased DNA damage, abnormal mitosis, and cellular senescence. Biochemical analysis and super-resolution microscopy revealed that the loss of ZC3H4 increased replication stress (RS)—a major driver of genome instability—by inducing a hypertranscription state that promoted R loop formation and transcription-replication conflicts (TRCs), both of which drive RS. Further bioinformatic analysis demonstrated that ZC3H4 preferentially binds to genomic regions prone to TRCs and R loops, where it suppresses ncRNA bursts, functioning as part of the Restrictor complex. Our findings identify ZC3H4 as a crucial factor in maintaining genome integrity, strategically positioned at the critical intersection of DNA and RNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2eceaa09e7e61059a9c40891a86a83f0e1a6b53" target='_blank'>
              ZC3H4 safeguards genome integrity by preventing transcription-replication conflicts at noncoding RNA loci
              </a>
            </td>
          <td>
            Yann Frey, Liana J Goehring, M. Haj, G. Róna, Carel Fijen, Michele Pagano, Tony T Huang, Eli Rothenberg, Y. Ziv, Yosef Shiloh
          </td>
          <td>2025-06-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary This study investigated circRNA_4083, which is a stable circular RNA which originates from the MSH3 gene of chicken. It is highly expressed in cardiac, pulmonary and renal tissues. Its biogenesis depends on flanking genomic sequences. CircRNA_4083 overexpression reduced apoptosis and enhanced cell proliferation, indicating its role in promoting cellular viability. Bioinformatics analysis indicates that circRNA_4083 interacts with RNA-binding molecules and regulates pathways that are crucial for genomic stability, possibly protecting the genome from damage. These findings underscore its importance in maintaining cellular health. By elucidating circRNA-driven mechanisms, this work advances the understanding of non-coding RNA functions in poultry biology, offering insights for developing strategies to improve disease resilience and productivity in agricultural practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c0e289cb6c7acbb75967e45fbe212b55b3a552a" target='_blank'>
              Mechanism of circRNA_4083 Circularization and Its Role in Regulating Cell Viability
              </a>
            </td>
          <td>
            Wenhao Li, Ting Yang, Haojie Wang, Hao Bai, Guobin Chang, Lingling Qiu
          </td>
          <td>2025-05-23</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Heritable gene activity variations that do not alter the underlying DNA sequence are known as epigenetic modifications. Mutations that disrupt genome structure and function are key drivers of oncogenesis. In addition to genetic mutations that cause direct disruptions in the DNA sequence, epigenetic changes can affect gene expression, which helps the development of cancerous traits. Changes in DNA methylation and histone mark patterns are the main drivers of these epigenetic modifications, as they encourage the growth and spread of tumors. In this review, we explore the substantial implications of epigenetic control on tumor genesis, metastatic behaviour, metabolic pathways, and the tumor microenvironment, delving into the intricacies of this intricate regulation. We pay special attention to the dysregulation at every stage of epigenetic modulation, which includes, but is not limited to, abnormalities in the enzymes that modify histones and methylate DNA, subunit loss or fusions in chromatin remodeling complexes, and disruptions in higher-order chromatin structure. We also highlight the development of inhibitors of medications targeted at epigenetic enzymes and summarize the abnormal roles of enzymes in DNA methylation, histone acetylation, and histone methylation during tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d006bb544ee4859c5808a78f45579e1cfd4a19" target='_blank'>
              Epigenetic regulators in cancer therapy and progression
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91ec8476ea04c15a90514f720adb0e97fd88bcc" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, Zhongqi Ge, Jaehyun Lee, Rishi Pasumarthi, Xiaohong Leng, Thomas Koehnke, O. Pasvolsky, Pranaya Raparla, Vinhkhang N. Nguyen, K. Maurer, Shuqiang Li, K. Livak, Emerie Danson, Bijal Thakkar, Elham Azizi, Robert J. Soiffer, S. Shukla, Ravi Majeti, Jerome Ritz, Catherine J. Wu, J. Molldrem, P. Bachireddy
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7409bece8a61b55b1fe618c705451021becc29" target='_blank'>
              Tissue-Specific Activation of Microhomology-Mediated End Joining with Age Reflects Dynamic Rewiring of DNA Repair in Rats
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: TIGD1 (Trigger Transposable Element Derived 1) is a recently identified oncogene with largely unexplored biological functions. Emerging evidence suggests its involvement in multiple cellular processes across cancer types. This study aimed to perform a comprehensive pan-cancer analysis of TIGD1 to evaluate its expression patterns, diagnostic utility, prognostic value, and association with immunotherapy response and drug resistance. Methods: Transcriptomic and clinical data from TCGA and GTEx were analyzed using various bioinformatic tools. Expression profiling, survival analysis, immune correlation studies, gene set enrichment, single-cell sequencing, and drug sensitivity assessments were performed. Results: TIGD1 was found to be significantly upregulated in various tumor types, with notably high expression in colon adenocarcinoma. Elevated TIGD1 expression was associated with poor prognosis in several cancers. TIGD1 levels correlated with key features of the tumor immune microenvironment, including immune checkpoint gene expression, TMB, and MSI, suggesting a role in modulating anti-tumor immunity. GSEA and single-cell analyses implicated TIGD1 in oncogenic signaling pathways. Furthermore, high TIGD1 expression was linked to resistance to several therapeutic agents, including Zoledronate, Dasatinib, and BLU-667. Conclusions: TIGD1 may serve as a promising diagnostic and prognostic biomarker, particularly in colon, gastric, liver, and lung cancers. Its strong associations with immune modulation and therapy resistance highlight its potential as a novel target for precision oncology and immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1163400219b2ecb1878714587256ba612b794cd1" target='_blank'>
              Beyond Transposons: TIGD1 as a Pan-Cancer Biomarker and Immune Modulator
              </a>
            </td>
          <td>
            Merve Gülşen Bal Albayrak, T. Korak, Gurler Akpinar, Murat Kasap
          </td>
          <td>2025-05-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Aging is a multifaceted process influenced by many elements. During cell division, the repetitive DNA sequences at the ends of chromosomes called telomeres protect them from degradation. Telomeres shorten alongside each cell division, eventually contributing to cellular senescence and aging. Telomerase as an enzyme has a role in the maintenance of telomere length. Reduced function of telomerase is linked to acceleration of aging and age-related diseases. By affecting cellular function, mutations in particular genes can cause aging. Genes involved in DNA repair, cellular metabolism, and inflammation play the key roles in this process. Accumulated mutations result in cellular dysfunction and age-related diseases over time. Epigenetic changes are the modifications that impact gene expression without altering the DNA sequence. Lifestyle factors (diet, exercise, stress) and environmental influences (toxins, trauma) can cause epigenetic alterations. DNA methylation as well as histone modifications are examples of epigenetic alterations. They influence how cells work and are essential to the aging process. Understanding these molecular mechanisms is essential for developing interventions to promote healthy aging and prevent age-related diseases. This paper explores the potential of CRISPR/Cas9 as a gene-editing tool to target these mechanisms and mitigate age-related conditions, ultimately enhancing longevity and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f9eeb8cd6bb47cacfd1747bd987a2823682b4c" target='_blank'>
              Applications of CRISPR-Cas9 in mitigating cellular senescence and age-related disease progression
              </a>
            </td>
          <td>
            Alireza Azani, Malihe Sharafi, Reyhaneh Doachi, Sama Akbarzadeh, Parsa Lorestani, Arash Haji Kamanaj Olia, Zahra Zahed, Hossein Gharedaghi, Haniyeh Ghasrsaz, Hassan Foroozand, Hossein Rahimi, Safa Tahmasebi, Qumars Behfar
          </td>
          <td>2025-07-08</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA polymerase delta (Pol δ) is a cornerstone of genomic stability, orchestrating DNA replication and repair through its catalytic subunit, POLD1. This subunit’s 3’–5’ exonuclease domain proofreads replication errors, ensuring fidelity. However, POLD1 mutations—particularly in this domain—disrupt this function, triggering genomic instability and a hypermutated state in cancers. This review delves into the multifaceted roles of POLD1 mutations, spotlighting their contributions to tumorigenesis and immunotherapy responses. Beyond their established link to syndromes like polymerase proofreading-associated polyposis (PPAP), these mutations unexpectedly enhance tumor immunogenicity in microsatellite-stable (MSS) tumors, previously considered largely resistant to immune checkpoint inhibitors (ICIs). By elevating tumor mutation burden and generating unique mutational signatures (e.g., SBS10d), POLD1 mutations sensitize MSS tumors to ICIs, challenging the dominance of microsatellite instability (MSI) as an immunotherapy predictor. Integrating structural insights, molecular mechanisms, and clinical data, this review positions POLD1 mutations as both a driver of cancer progression and a promising biomarker, redefining therapeutic possibilities in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d2442be2c75ad66ffe5186a1b712852b468974" target='_blank'>
              Beyond proofreading: POLD1 mutations as dynamic orchestrators of genomic instability and immune evasion in cancer
              </a>
            </td>
          <td>
            Huiqing Chen, Jiayu Wei, Qi Tang, Guohui Li, Yajing Zhou, Zhen Zhu
          </td>
          <td>2025-06-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CRISPR-Cas9 has democratized genome engineering due to its simplicity and efficacy. Adapted from a bacterial defense mechanism, CRISPR-Cas9 comprises the Cas9 endonuclease and a site-specific guide RNA. In vivo, the Cas9 ribonucleoprotein (RNP) can target specific genomic loci and generate double-strand breaks. Eukaryotic endogenous DNA repair mechanisms recognize the cut site and attempt to repair the DNA either by non-homologous end joining, which introduces insertions/deletions, resulting in a loss of reading frame in coding genes, or through homology-directed repair that maintains the reading frame. The latter approach allows the insertion of fluorescent reporter sequences in frame with protein-coding genes in order to monitor gene expression and protein dynamics in cells and whole organisms. Here, we provide a protocol for targeting endogenous genes to introduce sequences coding for fluorescent reporters in medaka (Oryzias latipes). The method is simple, robust, and efficient, thus facilitating straightforward organismal genome editing. Key features • Cloning free CRISPR/Cas9 tagging of endogenous genes with fluorescent reporter sequences. • Guidelines for designing CRISPR/Cas9 endogenous tagging experiments. • Straightforward generation of transgenic Medaka knock-in reporter lines. • Versatility with the use of Cas9 mRNA or protein and various fluorescent reporters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b5cb1a78e57b20fd3719b858a6cee4a591d69c8" target='_blank'>
              Cloning-Free Targeting of Endogenous Loci to Generate Fluorescent Reporters in Medaka
              </a>
            </td>
          <td>
            Simon Knoblich, Kiyoshi Naruse, Ali Seleit, Alexandre Paix
          </td>
          <td>2025-06-20</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is a common and aggressive malignancy. While significant advances have been made in the management of low-grade cancer, treatment of advanced HNSCC remains challenging. Here, we used a proteomic approach to find binding partners of the oncogene p63, the most frequently amplified transcription factor in HNSCC. We identified a zinc finger protein, ZNF148, which is coexpressed and physically binds p63 in carcinoma cells. Genome occupancy analyses identified a functional transcribed enhancer-derived RNA (eRNA) upstream of the CCND1 gene occupied by both factors. Mechanistically, p63 and ZNF148 control transcription of this eRNA, leading to overexpression of cyclin D1 to reinforce tumor cell proliferation. Importantly, this axis is specific for cancer cells and remains inactive in normal epithelial cells. The expression levels of these factors and the eRNA are positively correlated in cancer and associated with advanced stage and metastasis. Collectively, our data reveal a molecular pathway controlling HNSCC progression and identify potential selective targets for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5e810ce5ca8ccc51a55dbf7814b137367f41651" target='_blank'>
              p63 and ZNF148 cooperate to regulate head and neck squamous cell carcinoma.
              </a>
            </td>
          <td>
            R. Pecorari, Mara Mancini, Artem Smirnov, Giacomo Corleone, Flavia Novelli, A. Lena, Mariacristina Franzese Canonico, M. Montanaro, A. Cappello, A. Sacconi, Gabriella Misso, Michela Falco, Chiara Tammaro, F. Ciccosanti, Mauro Piacentini, M. Scimeca, M. Caraglia, G. Blandino, A. Mauriello, Maurizio Fanciulli, G. Melino, E. Candi
          </td>
          <td>2025-07-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="While it is straightforward to understand why most mutations affecting functional sequence are harmful, how genomic changes result in new beneficial traits is harder to understand. Domestication of transposable elements (TEs) is an important source of both new genes and new regulatory systems as, for their own propagation, TEs need to have transcription factor binding sites and functional products that predispose to their recruitment. But are such predispositions to gain-of-function sufficient? Here we consider the case of the endogenous retrovirus, HERVH. Knockdown data supports HERVH having roles in pluripotency, self-renewal and defence against transpositionally-active retroelements in the early human embryo. We clarify the pluripotent cell types associated with HERVH expression and, in the process, note a key unresolved issue, framed by the unwanted transcript hypothesis: how can some cell types have 2% of their transcripts being HERVH-derived but survive the multiplicity of cellular devices that suppress foreign transcripts, be this by transcriptional repression or post-transcriptional filtering? We note a common coupling between novelty generation and suppression evasion. For example, pluripotency-associated KLF4 binding is thought to compete with transcriptional suppressor binding. Similarly, HERVH has a strong splice site enabling efficient novel chimeric transcript formation, the resulting exon–intron junctions enabling evasion of the unwanted transcript filters that recognize low or absent intron presence. We conclude that to better understand domestication, a focus on predispositions to avoidance of unwanted transcripts filters, as well as predispositions to gain of functions, is necessary. The same insights will be valuable for transgene design (eg for gene therapy) and instructive of gain-of-function in tumours, as HERVH is known to be involved in onco-exaptation events. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00361-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1580e3fbff7b45d8d3d4ce0772f34b993e7ad4c0" target='_blank'>
              Co-option of an endogenous retrovirus (LTR7-HERVH) in early human embryogenesis: becoming useful and going unnoticed
              </a>
            </td>
          <td>
            Z. Izsvák, Jin Ma, Manvendra Singh, Laurence D. Hurst
          </td>
          <td>2025-07-05</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="DNA topoisomerases, particularly type II, are crucial for DNA processes, such as replication, transcription, and chromosome segregation, making them prime targets for cancer therapy. This review delves into the multifaceted mechanisms of action of type II topoisomerases, highlighting their essential roles beyond cancer progression. It explores recent advancements in screening and designing metallic complexes as inhibitors of topoisomerase II activity. Emphasizing the structural and functional diversity between alpha and beta isoforms, it elucidates their significance in DNA metabolism and genome integrity. Additionally, this review discusses the interplay of topoisomerase II with cellular components, underscoring its regulatory roles in gene expression. Insights into screening and design strategies for metallic complex inhibitors are provided, showcasing their therapeutic potential against cancer. Overall, this review highlights the importance of understanding topoisomerase II inhibition mechanisms and the versatility of metallic complexes in biomedical research, paving the way for novel therapeutic strategies and broader applications beyond cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a3a5eac5f306d411595288c7868532f4247f662" target='_blank'>
              Topoisomerase II Inhibition in Cancer: A Focus on Metal Complexes.
              </a>
            </td>
          <td>
            A. O. Akinyemi, Josias da Silveira Rocha, Gabriela Porto de Oliveira, Josenilton de Jesus Santos, B. Owoyemi, Fillipe V. Rocha
          </td>
          <td>2025-06-05</td>
          <td>Mini reviews in medicinal chemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b16c1cd80bd43a019c039162e9f1b0a15b5cf6d2" target='_blank'>
              Integrative Multi-Omics Analysis Identifies Nuclear Factor I as a Key Driver of Dysregulated Purine Metabolism in DIPG
              </a>
            </td>
          <td>
            I. Mersich, Sunny Congrove, Matthew Horchar, Roman Caceres, Ranjithmenon Muraleedharan, Janki M Desai, Pankaj B Desai, Larry Sallans, J. Reisz, Abby Grier, Matthew R. Hass, Omer Donmez, Cailing Yin, M. Weirauch, Leah C. Kottyan, Charles B. Stevenson, Claire X. Sun, P. D. de Blank, N. Pillay-Smiley, T. Hummel, N. Elumalai, Ali Tavassoli, Ron Firestein, T. Phoenix, A. D'alessandro, Biplab Dasgupta
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Cancer progression is a complex, multi-stage development process characterized by dynamic changes at the molecular level. Understanding these changes may provide new insights into tumorigenesis and potential therapeutic targets. This study focuses on the evolutionary transcriptomics of cancer, specifically analyzing the Transcriptome Age Index (TAI) across different pathological stages. By examining various cancers at four distinct pathological stages, we identify a significant «hourglass» pattern in TAI indices of ductal carcinoma of the breast, bladder carcinoma, and liver carcinoma, suggesting a conserved evolutionary trajectory during tumor development. The results reveal that early and late stages of these cancers exhibit higher TAI values, indicative of more novel gene expression, while intermediate stages show a dip in TAI, reflecting a more ancient evolutionary origin of expressed genes. This «hourglass» pattern underscores the evolutionary constraints and innovations at play during tumor progression. Our findings contribute to the growing body of evidence that evolutionary principles are deeply embedded in cancer biology, offering new perspectives on the dynamics of gene expression in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d899ba20c349660eb4cd72c9aa668578d579a2" target='_blank'>
              Evolutionary Transcriptomics of Cancer Development
              </a>
            </td>
          <td>
            Roman Ivanov, D. Afonnikov, Yury Matushkin, S. Lashin
          </td>
          <td>2025-05-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 A crucial bulwark of genome stability is the DNA repair system. The homologous recombination (HR) pathway is essential for the repair of the most harmful DNA lesions, like DNA double-strand breaks. Deficiencies in this pathway result in genomic instability and contribute to tumorigenesis. BRCA1 and BRCA2 (Breast Cancer type 1 and 2 susceptibility proteins) play critical roles in HR. Germline mutations in the BRCA1 and BRCA2 genes are responsible for nearly 50% of all familial breast and ovarian cancers. BRCA1/2-deficiencies coerce tumor cells to depend on alternative DNA repair pathways. Recent studies have demonstrated that inactivation of RAD52 protein induces lethality (synthetic lethality) in BRCA1/2-deficient cancer cells, without affecting normal cells. By exploiting this synthetic lethality relationship, targeting RAD52 with inhibitors may provide an exceptionally selective method for eliminating cancer cells. Focusing on DNA repair proteins involved in synthetically lethal relationships in cancer cells has become an important strategy in the development of specific cancer therapeutics. Our laboratory has developed small molecule inhibitors targeting RAD52. We have successfully created small molecule inhibitors of RAD52 (RAD52i) and have shown their ability to inhibit the biochemical activities RAD52 including DNA annealing and DNA strand exchange. Using the GFP-based recombination assays, we show that small molecule inhibitors of RAD52 disrupt the homologous DNA repair (HDR) and single strand annealing (SSA) repair pathways in human cells. Our compounds exhibited a notable reduction in cell viability in BRCA deficient cells. Furthermore, our small molecule RAD52i exhibited considerable efficacy in a mouse xenograft model cells markedly suppressing growth of human BRCA1-mutated cancer cells. We aim to develop small molecule RAD52i as novel cancer therapeutics.
 Citation Format: Shiva Ostadrahimi, Matthew J. Rossi, Sarah F. DiDomenico, Saiful Amin, Alexander V. Mazin. TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-12-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745e4e002b1607f75e1773159229e8749f23eb29" target='_blank'>
              Abstract P4-12-25: TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS
              </a>
            </td>
          <td>
            Shiva Ostadrahimi, Matthew Rossi, Sarah DiDomenico, Saiful Amin, Alexander Mazin
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epigenetics and genome science have become central to current molecular biology research. Among the key mechanisms ensuring genomic integrity is the silencing of transposable elements in germline cells, a process essential for fertility in both sexes. A pivotal component of this silencing machinery involves PIWI-interacting RNAs (piRNAs), a distinct class of small non-coding RNAs that regulate gene expression and suppress transposable elements at both the transcriptional and post-transcriptional levels. piRNAs function in concert with PIWI proteins, whose expression is critical for proper oogenesis, spermatogenesis, and early zygote development. Disruptions in piRNA or PIWI protein pathways not only impair germline function but also contribute to genome instability, unchecked cell proliferation, and aberrant epigenetic modifications, hallmarks of tumorigenesis. Emerging evidence links the dysregulation of the piRNA/PIWI axis to the development and progression of various cancers, including lung and colorectal carcinomas. This review highlights the fundamental roles of piRNAs and PIWI proteins in reproductive biology and their increasingly recognized relevance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e264ddb82fab7df947d4ae7f4949ecb2a0652c26" target='_blank'>
              piRNA-Mediated Maintenance of Genome Stability in Gametogenesis and Cancer
              </a>
            </td>
          <td>
            Martyna Zawalska, Maciej Tarnowski
          </td>
          <td>2025-06-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Nuclear atypia is associated with increased malignancy in numerous cancers, including glioblastoma (GBM). Here, we found that GBM stem cells display small nuclear size, prompting investigation of mechanisms underlying nuclear size regulation in the tumor hierarchy.


METHODS
We performed comparative gene expression and proteomics in GBM stem cells (GSCs) and neural stem cells (NSCs) to discover potential regulators of nuclear size. Through transcriptomic analysis, mass spectrometry, and pharmacologic inhibition, we interrogated the functional significance of nuclear size regulation.


RESULTS
GSCs were enriched for a nuclear sterol reductase, Lamin B Receptor (LBR). Targeting LBR increased nuclear size and decreased GSC viability and tumor initiation. Regulation of nuclear cholesterol synthesis underlaid LBR-dependency in GSCs. Loss of LBR or reduction of cholesterol levels induced double-strand DNA breaks (DSBs), activating P53-dependent DNA damage responses (DDR). The GSC proteomic LBR interactome revealed DDR mediators, including DEAD-box RNA helicase DDX5 that resolves R-loops at DSBs. Genetic targeting of LBR reduced the DDX5-R loop interaction, leading to increased R-loop formation rescued by cholesterol supplementation. Pharmacological sterol reductase inhibition mirrored genetic LBR targeting by reducing the DDX5-R loop interaction and increasing R-loops and DSBs. Targeting LBR genetically and pharmacologically inhibits GSC growth in vivo and synergizes with irradiation.


CONCLUSIONS
Stem-like GBM cells display reduced nuclear size, driven by nuclear cholesterol synthesis to regulate radiation responses, revealing a novel therapeutic paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac67f62707eaa5f9643989ac2e6171dc64366dea" target='_blank'>
              Nuclear cholesterol regulates nuclear size and DNA damage responses in cancer stem cells.
              </a>
            </td>
          <td>
            Tingting Duan, S. Taori, S. Bhargava, Sisi Lai, Cuiqing Zhong, Shira Yomtoubian, Huairui Yuan, Xujia Wu, Po Zhang, Tengfei Huang, Donghai Wang, Fanen Yuan, Daqi Li, Huan Li, Hailong Mi, Weichi Wu, Rui Wang, Ahmed Habib, Farrukh Hammed, F. Vendetti, Pascal O. Zinn, Christopher J. Bakkenist, Marlies Meisel, Qiulian Wu, Jeremy N. Rich
          </td>
          <td>2025-06-07</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Malignant rhabdoid tumor of the kidney (MRTK) is a rare renal tumor with poor prognosis. While germline mutations of SMARCB1 are considered to be the primary cause of MRTK, emerging evidence suggests that somatic epigenetic changes also play a vital role in the development and progression of MRTK. YTHDF1, an m6A reader protein, has been implicated in regulation of tumorigenesis by influencing RNA translation and stability in several adult cancers. However, the exploration of the role of YTHDF1 in pediatric cancer, especially MRTK, remains limited. Methods In this study, CRISPR/Cas9 was employed to knockout (KO) YTHDF1 in G401 cells. The impact of YTHDF1 on the cell growth and chemoresistance were assessed using CCK-8 assays. Western blot and qRT-PCR were used to determine the changes in ferroptosis marker gene expression. Additionally, 4D-label free quantitative proteomics was conducted to uncover alterations by YTHDF1 deletion. Results We observed that the deletion of YTHDF1 in the MRTK cell line led to a significant reduction in malignancy-associated characteristics, including decreased cell motility, invasive growth, and chemoresistance. Quantitative proteomic analysis revealed that the glutathione-related signaling pathway was notably affected by YTHDF1 KO. Specifically, YTHDF1 KO resulted in a reduction of both mRNA and protein levels of Glutathione S-Transferase Mu 2 (GSTM2), a phase II metabolizing enzyme responsible for conjugating glutathione to electrophilic compounds. The decrease in GSTM2 levels following YTHDF1 KO increased the susceptibility of MRTK cells to ferroptosis. Notably, overexpression of GSTM2 in YTHDF1 KO cells partially restored the oncogenic phenotype of MRTK cells, underscoring its role in MRTK progression. Conclusions In summary, our findings provide new insights into the molecular mechanisms driving MRTK progression, highlighting YTHDF1 and GSTM2 as potential therapeutic targets for this aggressive pediatric renal tumor. Graphical Abstract YTHDF1 is highly expressed in MRTK cells. YTHDF1 recognizes the m6A modification on GSTM2 mRNA, enhancing its stability. GSTM2 protects MRTK cells against ferroptosis independent of GPX4. Supplementary Information The online version contains supplementary material available at 10.1007/s10565-025-10049-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/248db6cfe415c7d8ca495a38efdbd0087162d544" target='_blank'>
              Loss of YTHDF1 suppresses the progression of malignant rhabdoid tumor of the kidney by regulating Glutathione S-transferase Mu 2 (GSTM2)
              </a>
            </td>
          <td>
            Qian-Wen Xiong, Yuntao Liu, Min He, Xiao-Die Shen, Manli Zhao, Shuang-Ai Liu, Gensheng Zhang, Qian Liu, Jinhu Wang, Wan-Xin Peng
          </td>
          <td>2025-06-07</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b5bfa4068fc673194fc62b48dc4a50c589763e7" target='_blank'>
              Ataxin-2 preserves oocyte genome integrity by promoting timely premeiotic DNA replication
              </a>
            </td>
          <td>
            V. L. Monteiro, Crystal Yu, Camilla Roselli, B. Bakthavachalu, Mani Ramaswami, T. Hurd
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background NSUN6 is a regulator of tRNA methylation that partially exists in the Golgi and centrosomes. It catalyzes the methylation of tRNACys and tRNAThr at the C72 site, which affects tRNA generation. However, its expression, prognosis, and immune invasion in pan-cancer have not been studied. This extensive research tapped into the TCGA database, gathering 33 carefully paired sets of cancer and nearby normal tissues, covering a broad spectrum of 33 different cancer types. Moreover, this study explored how NSUN6 expression relates to the presence of immune and stromal cells in the tumor’s intricate immune environment. To further understand NSUN6’s role, a detailed Gene Set Enrichment Analysis (GSEA) was conducted, revealing its enrichment patterns in various carcinoma subtypes and hinting at its involvement in tumor growth. Analysis of NSUN6 expression patterns revealed a significant increase in eight cancer types, including lung adenocarcinoma and pancreatic cancer (P < 0.001). Additionally, a reduction was observed in six cancer types, such as cholangiocarcinoma and thyroid cancer (P < 0.001). Survival analysis indicated that individuals with lung adenocarcinoma, pancreatic cancer, and other cancers expressing high levels of NSUN6 had improved prognosis. The association between NSUN6 expression and patient outcomes was evident, including overall survival, disease-specific survival, disease-free interval, and progression-free interval. Furthermore, individuals with high levels of NSUN6 expression showed significantly longer survival times than those with low levels (P < 0.05). The involvement of NSUN6 in immune infiltration was evident, GSEA showed a significant correlation between NSUN6 expression and the five most significant enrichment pathways in different tumors. The NSUN6 gene functions as an initial indicator for diagnosis in renal clear cell carcinoma, pancreatic cancer, lung adenocarcinoma, and low-grade brain glioma. There is an association between its elevated expression and the predictive outcome, as well as the immune system’s infiltration, in 33 varieties of cancer. This study is limited by its reliance on in silico analyses without experimental validation. Additionally, the use of publicly available datasets may introduce variability due to differences in data sources and platforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0727a8de06b6d8752b41beb8058ddf29b34aed" target='_blank'>
              NSUN6 expression correlates with prognosis and immune infiltration in human cancers based on pan-cancer analysis
              </a>
            </td>
          <td>
            Jie Gao, Yao Xu, Chan Tian, Qiuhong Zhang, Ming Zhang, H. Shan, Jie Shi, Zongjuan Ming, Shuanying Yang, Xia Yang
          </td>
          <td>2025-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: Previously, we found that an uncharacterized protein CXorf38 is significantly downregulated in human ZIP8-knockout (KO) cells. Given that ZIP8 regulates essential micronutrients linked to diseases including cancer, this study aims to characterize CXorf38 and evaluate its role in lung adenocarcinoma. Methods: iTRAQ-based proteomics was previously used to identify the abundance of proteins in ZIP8-KO cells. Cell proliferation and colony formation assays were used to examine the function of CXorf38 by overexpressing the gene in lung adenocarcinoma cell lines. Kaplan–Meier survival analysis was used to assess the prognostic value of CXorf38, while TCGA clinical database analysis was used to evaluate its expression in lung cancer tissues, particularly in smokers. Bioinformatics analyses (GO, KEGG, PPI, and ICI) were performed on CXorf38-coexpressed genes derived from patients with lung cancer. Results: CXorf38 overexpression suppressed lung cancer cell proliferation and colony formation, suggesting a tumor-suppressive role. Higher CXorf38 expression correlated with improved survival in patients with lung adenocarcinoma, but not in lung squamous cell carcinoma. Clinical data showed CXorf38 downregulation with lung cancer tissues of smokers, indicating a potential role in smoking-induced cancer progression and treatment. Functional analysis using bioinformatics linked CXorf38 to immune response regulation, suggesting involvement in the tumor immune microenvironment. Conclusions: Our study reveals for the first time that CXorf38 is a potential tumor suppressor, prognostic biomarker, and/or tumor immune regulator in lung adenocarcinoma—further research is warranted to explore its role in tumor immunity and its therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0de769da2616c4ecb2cadcb3f426fcabdecfbd13" target='_blank'>
              Chromosome X Open Reading Frame 38 (CXorf38) Is a Tumor Suppressor and Potential Prognostic Biomarker in Lung Adenocarcinoma: The First Characterization
              </a>
            </td>
          <td>
            Rui Yan, H. Tan, Na-Li Cai, Le Yu, Yan Gao, Yan-Ming Xu, Andy T. Y. Lau
          </td>
          <td>2025-06-01</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 What are the mechanisms underlying segmental aneuploidy (SA) in preimplantation human embryos, and to what extent is there concordance between germ cell layers?



 Segmental aneuploidy (SA) in IVF embryos predominantly arises from paternal meiotic errors, exhibiting deletion-duplications, inversions, and chromothripsis-like mechanisms, with rare SA concordance observed



 Segmental aneuploidy (SA) involves the gain or loss of chromosomal segments rather than entire chromosomes, leading to significant embryonic consequences such as implantation failure, spontaneous abortion, and congenital abnormalities. Identified in about 7% of aneuploid biopsies during preimplantation genetic testing for aneuploidy (PGT-A) and preimplantation genetic testing for structural rearrangements (PGT-SR), SA arises from chromosomal breakage and recombination during meiosis or early cleavage. These structural abnormalities disrupt gene dosage, potentially resulting in non-disjunction events or abnormal recombination, which affect normal embryonic development. Understanding SA mechanisms is crucial for improving outcomes in both natural conception and assisted reproductive technologies (ART).



 This single-centre retrospective study examined 101 blastocysts identified as aneuploid with segmental aneuploidy (SA) via a validated NGS-based PGT-A protocol. The protocol enables precise identification of the parental origin of SA (paternal or maternal) by integrating genotyping data with chromosomal copy number variation analysis. This method enhances understanding of SA mechanisms and their implications, contributing valuable insights into embryonic development and improving assisted reproductive technology (ART) outcomes.



 The participants in this study were couples undergoing IVF cycles with PGT-A from 2020–2024. Ethical approval was secured from IRAS (#294909) and HFEA licence (#R0208). Segmental aneuploidy (SA) was identified in embryos post-PGT-A. Participants provided cheek swabs for parental DNA analysis, and embryo biopsies included samples from the inner cell mass (ICM) and two trophectoderm (TE) biopsies. This approach enabled comprehensive analysis of SA origin and patterns in the context of preimplantation development.



 In our analysis of a cohort exceeding 100 cases of segmental aneuploidy (SA), we examined both two trophectoderm (TE) biopsies and the inner cell mass (ICM) from each embryo. We frequently observed complex SA aneuploidies characterized by combinations of deletions, duplications, inversions, isochromosomes, and ring chromosomes, often localized to a single chromosome, with a notable prevalence of paternal errors. Interestingly, while mosaicism typically suggests post-zygotic origins, evidence from our data indicates meiotic origins for some cases, supporting the concept of “SA rescue,” where specific chromosomes can be restored to euploidy—a phenomenon not commonly associated with whole chromosome aneuploidy.
 Furthermore, unlike the typical concordance seen in whole chromosome aneuploidy, perfect concordance of SA abnormalities among the three sampled regions was rare. We identified an “intrachromosomal effect,” where additional SA abnormalities on the same chromosome were maintained across the biopsies, possibly related to chromothripsis. These observed patterns bear resemblance to chromosomal instability seen in cancer, leading to the hypothesis that tumor-like mechanisms might influence SA development during preimplantation human embryogenesis. Overall, while the study’s design minimizes chance influences, variability in SA manifestations calls for cautious interpretation of these complex genetic phenomena.



 Limitations of this study include its retrospective and single-centre design, which may affect generalizability. The use of cheek swabs for parental DNA analysis might not capture full genetic variability, and biases could arise from sample handling and mosaicism. Further multicentre research is needed to validate the findings.



 This study helps us inform future PGT strategies, it highlights the need to screen for (and, where possible reduce) chromosome breakage/DNA damage in IVF cases. It provides novel insight into the mechanisms of chromosome behaviour in early human development. It introduces novel concepts hitherto unreported in this context.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62df6f66cade9f156346e5f1e524c58060a3734f" target='_blank'>
              P-557 Origin of segmental aneuploidy in preimplantation development: Studying >100 IVF embryos reveal novel mechanisms akin to cancer cells with a possible intrachromosomal effect
              </a>
            </td>
          <td>
            B. Al Hashimi, N. Macklon, S. SenGupta, T. Gordon, E. Linara-Demakakou, B. Raikundalia, K. Ahuja, D. Griffin
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The chromosomal passenger complex (CPC), comprising Survivin, Aurora B kinase, INCENP, and Borealin, is classically known for its essential functions during mitosis. However, recent findings expand the CPC's role beyond cell division, uncovering novel functions in replication stress response and genome stability maintenance. In our recent study by Falke et al. [1], we demonstrate that CPC components, particularly Survivin and Aurora B, contribute to the protection of stalled replication forks and facilitate translesion synthesis (TLS) - a specialized DNA damage tolerance mechanism. This regulation involves a functional interaction with proliferating cell nuclear antigen (PCNA), a central coordinator of DNA replication and repair. The work not only reshapes our understanding of CPC biology but also reveals a mechanistic link how tumor cells may exploit CPC activity to maintain replication under genotoxic stress. These findings open new avenues for targeting CPC-PCNA interactions in cancers characterized by elevated replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27440a8fa21c0d8a560b047855971e8e0f1ea441" target='_blank'>
              From mitosis to mutagenesis: Chromosomal passenger proteins at the crossroads of replication stress and cancer resilience
              </a>
            </td>
          <td>
            S. Knauer, Roland H. Stauber
          </td>
          <td>2025-07-07</td>
          <td>Cell Signaling</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Intra-tumor heterogeneity impacts disease progression and therapeutic resistance but remains poorly characterized by conventional histologic, immunophenotypic, and molecular approaches. Single-cell biophysical properties distinguish functional phenotypes complementary to these approaches, providing additional insight into cellular diversity. Here we link both buoyant mass and stiffness to gene expression to identify clinically relevant phenotypes within primary mantle cell lymphoma (MCL) cells, employing MCL as a model of biological and clinical diversity in human cancer. Linked measurements reveal that buoyant mass and stiffness characterize B-cell development states from naïve to plasma cell and correlate with expression of oncogenic B-cell receptor signaling genes such as BLK and CD79A. Additionally, changes in cell buoyant mass within primary patient specimens ex vivo correlate with sensitivity to Bruton’s Tyrosine Kinase inhibitors in vivo in MCL and chronic lymphocytic leukemia, another B-cell malignancy. These findings highlight the value of biophysical properties as biomarkers of response in pursuit of future precision therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222e5440bee40a1cfc03b8b975d31d717ee09614" target='_blank'>
              Integrating Single-Cell Biophysical and Transcriptomic Features to Resolve Functional Heterogeneity in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Ye Zhang, Lydie Debaize, Adam Langenbucher, Jenalyn Weekes, Ioulia Vogiatzi, Teemu P. Miettinen, Mingzeng Zhang, Emily Sumpena, Huiyun Liu, Sarah M. Duquette, L. Hackett, Jeremy Zhang, Sona Baghiyan, R. Redd, M. Aryee, M. Davids, Austin I. Kim, C. Ryan, David M. Weinstock, S. Manalis, M. Murakami
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2537688c884004a06a6382cdb89bfdc63b85ffff" target='_blank'>
              CRISPR-Based Gene Dependency Screens reveal Mechanism of BRAF Inhibitor Resistance in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            Shawn Noronha, Yue Liu, Gaga Geneti, Haojian Li, Xiaolin Wu, David Sun, Vaibhavi Gujar, Takashi Furusawa, Alexei Lobanov, Maggie Cam, Lipika R. Pal, N. Nair, Chi-Ping Day, E. Ruppin, Chandrayee Gosh, Jiangnan Hu, Suresh Kumar, T. Andresson, King Chan, Maura O’Neill, Raj Chari, Y. Pommier, J. Rivero, Urbain Weyemi, E. Kebebew, Myriem Boufraqech
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Background The study explores the role of non-coding ribonucleic acids (ncRNAs) and higher enhancer of zeste homolog 2 (EZH2) gene in breast cancer progression, examining how microRNA (miRNA) and long noncoding RNA (lncRNA) control EZH2, potentially influencing oncogene growth and treatment failure. Materials and methods The databases used in the study included Cyclebase 3.0 and CellTracer to determine EZH2’s role in cell cycle, Oncomine, OncoMX and The University of Alabama At Birmingham Cancer Data Analysis Portal (UALCAN) for Pan-cancer analysis, The Cancer Genomic Atlas Portal (TCGA Portal), Gene Expression Profiling Interactive Analysis (GEPIA2), OncoDB, CR2Cancer, Encyclopaedia of RNA Interactomes (ENCORI), and The Cancer Genome Altas Analyzer (TCGAnalyzeR v1.0) for differential expression analysis, CR2Cancer, OncoDB, MethMarkerDB, and Wanderer databases for epigenetic alteration analysis, Kaplan-Meier Plotter for survival analysis, Breast Cancer Gene Expression Miner (bc-GenExMiner v5.0) for hormone receptor analysis, Tumor-Immune System Interaction Database (TISIDB), Cancer Single Cell State Atlas (CancerSEA), TNMplot, DriverDBv4, and ENCORI for biological processes, cell cycle checkpoints and metastasis analysis, Enrichr, Tumor Immune Estimation Resource (TIMER 2.0), Gepia2 for transcription factor analysis, miRNet, Transcriptome Alterations in Cancer Omnibus (TACCO), and CancerMIRome for miRNA analysis, Enrichr, UALCAN, and ENCORI for lncRNA analysis. Results The EZH2 gene is overexpressed in breast cancer (BRCA) tumors, metastatic tissues, and circulating tumor cells, potentially leading to cancer progression. Patients with high EZH2 expression have shorter overall survival (OS), distant metastasis-free survival (DMFS), and relapse-free survival (RFS) compared to those with low expression. Estrogen receptot (ER)-negative BRCA tumors and PR-negative tumors have high EZH2 gene and eukaryotic transcription factor (E2F2) levels. The EZH2/E2F2 axis may assist ER/PR-negative BRCA by sponging homosapiens microRNA family (hsa-let-7b-5p) through lncRNA-thymopoietin antisense transcript 1 (TMPO-AS1). Overexpression of the EZH2 protein is associated with BRCA metastasis. Conclusion EZH2 overexpression in basal-like BRCA is mediated by a competing endogenous RNA (ceRNA) network and regulating their expression levels may facilitate better survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7050692d6924a95d1983e91c0ebe8eef26bc9ef7" target='_blank'>
              TMPO-AS1-hsa-let-7b-5p-EZH2-RNA network predicts poor survival in basal-like breast cancer patients
              </a>
            </td>
          <td>
            Prerna Vats, Bhavika Baweja, Sakshi Nirmal, Aditi Singh, Rajeev Nema
          </td>
          <td>2025-06-07</td>
          <td>Reports of Practical Oncology and Radiotherapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Personalized immunotherapy of pancreatic ductal adenocarcinoma (PDAC) through T-cell mediated targeting of tumor-specific, mutanome-encoded neoepitopes may offer new opportunities to combat this disease, in particular by countering recurrence after primary tumor resection. However, the sensitive and accurate calling of somatic mutations in PDAC tissue samples is compromised by the low tumor cell content. Moreover, the repertoire of immunogenic neoepitopes in PDAC is limited due to the low mutational load of the majority of these tumors. Methods We developed a workflow involving the combined analysis of next-generation DNA and RNA sequencing data from matched pairs of primary tumor samples and patient-derived xenograft models towards the enhanced detection of driver mutations as well as single nucleotide variants encoding potentially immunogenic T-cell neoepitopes. Subsequently, we immunized HLA/human T-cell receptor (TCR) locus-transgenic mice with synthetic peptides representing candidate neoepitopes, and molecularly cloned the genes encoding TCRs targeting these epitopes. Result Application of our pipeline resulted in the identification of greater numbers of non-synonymous mutations encoding candidate neoepitopes with increased confidence. Furthermore, we provide proof of concept for the successful isolation of HLA-restricted TCRs from humanized mice immunized with different neoepitopes, several of which would not have been selected based on mutanome analysis of PDAC tissue samples alone. These TCRs mediate specific T-cell reactivity against the tumor cells in which the corresponding mutations were identified. Conclusion Enhanced mutanome analysis and candidate neoepitope selection increase the likelihood of identifying therapeutically relevant neoepitopes, and thereby support the optimization of personalized immunotherapy for PDAC and other poorly immunogenic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74726107b1853e699ed8135ce18a77b1befecac" target='_blank'>
              Enhanced mutanome analysis towards the induction of neoepitope-reactive T-cell responses for personalized immunotherapy of pancreatic cancer
              </a>
            </td>
          <td>
            M. Volkmar, Dana Hoser, Claudia Lauenstein, Janina Rebmann, Agnes Hotz-Wagenblatt, Jan Rieger, I. Poschke, Jonas P. Becker, Angelika B. Riemer, M. Sprick, A. Trumpp, Oliver Strobel, Thomas Blankenstein, G. Willimsky, R. Offringa
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is the second most commonly diagnosed malignancy in men and is projected to cause approximately 35,250 deaths in 2024. The utility of total prostate-specific antigen (PSA) testing as a routine screening tool remains controversial due to limited specificity. Therefore, the identification of novel, noninvasive biomarkers is essential for improving early diagnosis. This study aimed to evaluate the methylation status of a specific CpG dinucleotide within the SPDEF (SAM-pointed domain-containing ETS transcription factor) gene promoter in blood leukocytes of PCa patients, using benign prostatic hyperplasia (BPH) samples as a control group. Methods Peripheral blood samples were collected from 360 men, including 180 diagnosed with PCa and 180 with BPH. A target CpG dinucleotide (cg11346722) in the SPDEF promoter was selected based on analysis of The Cancer Genome Atlas (TCGA) data. Methylation levels were assessed using methylation-sensitive restriction enzyme PCR (MSRE-PCR) and quantitative PCR (qPCR). Associations between methylation and clinical parameters—tumor stage (TS), histological grade, and total PSA levels—were analyzed. Results The mean methylation level at the SPDEF CpG site was significantly lower in PCa patients (hypomethylation: 92 ± 11.76%) compared to BPH controls (15.5 ± 15.12%) (p < 0.0001). Receiver operating characteristic (ROC) curve analysis demonstrated that SPDEF hypomethylation discriminated PCa from BPH with 98.3% sensitivity and 98.3% specificity at a < 55% methylation cutoff. A significant inverse correlation was observed between SPDEF methylation and both tumor stage (TS) and grade, whereas no correlation was found with total PSA levels. Conclusions Hypomethylation of a specific CpG dinucleotide in the SPDEF promoter may serve as a promising noninvasive blood-based biomarker for the early detection and clinical stratification of prostate cancer. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12894-025-01824-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b3d0cea675c5df05ed56b3e334c0ea9328b4fd5" target='_blank'>
              CpG dinucleotide methylation of the SPDEF gene as a blood-based epigenetic biomarker for prostate cancer diagnosis
              </a>
            </td>
          <td>
            Mohammad Vahid Ahmadianpour, S. Aleyasin
          </td>
          <td>2025-06-02</td>
          <td>BMC Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b6a64a6d2ad28ecff7ab0f31663d8f3fd11b6a" target='_blank'>
              A spatial atlas of chemoradiation therapy in pancreatic cancer identifies cellular and microenvironmental determinants of persister populations
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, Ariana Acevedo-Diaz, K. Rajapakshe, Galia Jacobson, Daniela Tovar, Jimin Min, Guangsheng Pei, Candise Tat, Ayush Suresh, Ching-Wei Tzeng, Matthew HG Katz, M. Bhutani, Huaming Wang, Robert A. Wolff, C. Haymaker, E. Ludmir, Huocong Huang, Xiongfeng Chen, Liang Li, Albert C Koong, Linghua Wang, N. Navin, Dadi Jiang, Ziyi Li, Anirban Maitra, Eugene J. Koay
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract The six-subunit origin recognition complex (ORC) loads excess MCM2–7 on chromosomes to promote initiation of DNA replication and is believed to be important for origin specification. Mapping of origins in cancer cell lines engineered to delete three of the subunits, ORC1, ORC2, or ORC5, shows that specific origins are still used and are mostly at the same sites in the genome as in wild-type cells. The few thousand origins that were upregulated in the absence of ORC suggest that GC/TA skewness and simple repeat sequences facilitate, but are not essential for, origin selection in the absence of the six-subunit ORC. Despite the lack of ORC, excess MCM2–7 is still loaded at comparable rates in G1 phase to license dormant origins and is also repeatedly loaded in the same S phase to permit re-replication. Thus, origin specification and excess MCM2–7 loading on origins do not require the six-subunit ORC in human cancer cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b9d06157a04f12f90653a1aa8763b312684dcdf" target='_blank'>
              Specific origin selection and excess functional MCM2–7 loading in ORC-deficient cells
              </a>
            </td>
          <td>
            Y. Shibata, Mihaela Peycheva, Etsuko Shibata, Daniel Malzl, Rushad Pavri, Anindya Dutta
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Aggressive and therapy-resistant glioblastoma is among the most lethal malignant tumors in humans. Complete surgical resection is often unachievable; therefore, combination chemoradiotherapy is used to target tumor cells residual beyond the resection margin. This approach induces DNA damage in tumor cells and activates the apoptosis pathway. Unfortunately, recurrence remains a major clinical challenge, frequently manifesting as more aggressive and treatmentresistant glioblastoma phenotypes. The DNA repair and damage response (DDR) pathways are critical for maintaining genome stability. While defects in these mechanisms contribute to oncogenesis, they also make tumor cells vulnerable to DNA-damaging therapy, as the cells become dependent on residual repair capacity. It is of paramount importance to understand the molecular components of these mechanisms and to identify potential therapeutic/pharmacological targets for improving outcomes in glioblastoma patients. A subpopulation of stem-like cells, designated as glioblastoma cancer stem cells (CSCs), has been identified as a critical factor in the initiation, maintenance, and recurrence of tumors. These cells exhibit therapy resistance due to enhanced DNA repair capacity. In addition, emerging evidence suggests a link between carbohydrate metabolism and DNA repair pathways, thereby revealing novel therapeutic vulnerabilities in glioblastoma. This review examines current strategies targeting DNA repair mechanisms in glioblastoma. We present a synopsis of recent advancements in research concerning the mechanisms and factors involved in the elimination of DNA damage induced by ionizing radiation and temozolomide (TMZ). Furthermore, we explore the potential of DNA repair pathway inhibitors under investigation in preclinical and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8edb7bc241371338e935ca9559370edd806d8316" target='_blank'>
              DNA Damage and Repair in Glioblastoma: Emerging Therapeutic Perspectives
              </a>
            </td>
          <td>
            I. Gareev, O. Beylerli, S. Roumiantsev
          </td>
          <td>2025-07-01</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Pituitary adenomas (PAs) are benign tumors in the pituitary gland. However, 30–40% of these tumors are invasive, complicating diagnosis and treatment. Invasive pituitary adenomas (IPAs) often respond poorly to conventional therapies, emphasizing the need for better diagnostic and therapeutic strategies. Understanding DNA methylation patterns in IPAs may reveal new biomarkers and therapeutic targets, leading to more effective management of this challenging disease. Methods Reduced representation bisulfite sequencing (RRBS) and RNA sequencing (RNA-seq) were performed on 129 samples from the Second Affiliated Hospital of Zhejiang University, including 69 tissue samples from invasive and non-invasive pituitary adenomas (NPA) and 60 blood samples from IPA, NPA and healthy individuals. Differentially methylated regions (DMRs) and differentially expressed genes (DEGs) were identified in tissues. Pearson correlation analysis was used to identify associations between DNA methylation status and gene expression, as well as the effect of methylation on gene expression at different sites. Blood samples were analyzed to detect DMRs and DEGs, correlating with tissue-derived findings. Finally, ROC analysis and a random forest model were used to identify biomarkers for discriminating invasive from non-invasive phenotypes. Results We identified 347 DMRs between IPA and NPA, of which 63% (219/347) were hypomethylated. Additionally, 543 mRNAs showed differential expression, with 350 upregulated and 193 downregulated. 17 genes demonstrated concurrent aberrant methylation and expression, primarily within introns, promoters, and CpG islands (CGIs). Notably, only protein tyrosine phosphatase receptor type T (PTPRT) exhibited a remarkably high correlation (r = 0.81) between its DNA methylation levels and mRNA expression levels. This correlation was observed within the intronic region/opensea of the gene's CGIs. Plasma sample analysis revealed 852 DMRs between IPA and NPA, with 52% (447/852) being hypomethylated. Three tumor tissue-derived blood biomarkers (MIR4535, SLC8A1-AS1, and TTC34) accurately discriminated between IPA and NPA patients with a combined AUC of 0.980. These markers also differentiated NPA from healthy controls, though with different methylation patterns. Conclusions The relationship between DNA methylation and gene expression is complex. Plasma-based DNA methylation markers can effectively discriminate between IPA and NPA, as well as between NPA and healthy individuals (N group). Supplementary Information The online version contains supplementary material available at 10.1186/s12916-025-04164-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63168fb628263eeafdc395f9c006371c823b818" target='_blank'>
              Molecular signatures of invasive and non-invasive pituitary adenomas: a comprehensive analysis of DNA methylation and gene expression
              </a>
            </td>
          <td>
            Yike Chen, Ningning Zhao, Jiahao Zhang, Xinyi Wu, Jian Huang, Xiaohui Xu, Feng Cai, Sheng Chen, Liyin Xu, Wei Yan, Yuan Hong, Yunfei Wang, Hui Ling, Jianxiong Ji, Gao Chen, Hongcang Gu, Jianmin Zhang, Qun Wu
          </td>
          <td>2025-07-01</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Congenital hypogonadotropic hypogonadism (CHH) is a rare endocrine disorder characterized by gonadal dysfunction attributed to impaired gonadotropin secretion. CHH is associated with approximately 60 genes including FGFR1. Nevertheless, the nucleotide variants of these genes are only related to less than half of the cases. Herein, we report a case of CHH caused by a novel mechanism. A 6-year-old boy presented with hypomasculinized genitalia, hyposmia, and syndactyly. Endocrine examinations showed impaired gonadotropin secretion. Short-read next-generation sequencing (NGS) identified the absence of mutations in the major causative genes for CHH. However, it detected an accumulation of discordant and split reads in a genomic region within the FGFR1 gene. Array-based comparative genomic hybridization did not detect copy-number abnormalities. Targeted long-read NGS and Sanger sequencing identified a de novo 333-bp insertion in exon 9 of the FGFR1 gene. A similarity search revealed that the insertion was an Alu element. This insertion caused a frameshift and resulted in premature termination (p. His409fsTer31). Further, it had several hallmarks of retrotransposition such as target site duplication, an endonuclease cleavage site-like motif, and a poly-A tail. The study results broadened the genetic basis of CHH that considered retrotransposon insertions. Importantly, this case emphasizes the need for additional genomic analyses in patients with CHH who had negative results on short-read NGS and array-based comparative genomic hybridization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/637624b99a101c0fcbfaf275613d36c05df12aa2" target='_blank'>
              De novo retrotransposon insertion into the FGFR1 gene in a boy with congenital hypogonadotropic hypogonadism: a case report
              </a>
            </td>
          <td>
            Kentaro Sawano, Keisuke Nagasaki, E. Suzuki, Yasuko Ogiwara, Ikuko Kageyama, Maki Fukami, Y. Kuroki
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Ribosomal DNA transcription is essential for ribosome biogenesis and the production of proteins. Using a combination of droplet digital PCR and deep bisulfite sequencing, we have quantified both the absolute number as well as the methylation level of individual rDNA transcription units (TU) in blood samples of 139 young healthy individuals and 141 sex- and age-matched individuals with unsolved syndromal developmental delay (DD), ranging from 0.02 to 18.4 years in age. There were no between-group differences in average promoter methylation, absolute copy number (CN), extreme CN, and hypomethylated (0-10%) presumably active CN. This largely excludes rDNA CN as a modulating factor in DD. The absolute CN in all 280 samples was 423.7 ± 108.4 (median 410, range 153 to 1,000) and the active CN was 175.0 ± 36.4 (median 174, range 70 to 376). Similar to adults, the absolute CN did not change from birth to sexual maturity but was strongly (Pearson ρ = 0.64, P < 0.001) correlated with promoter methylation. In contrast to adults, there was no significant correlation between age and promoter methylation and no age-related loss of active copies from birth to < 20 years. The number of completely unmethylated copies even significantly (Pearson ρ = 0.15; P = 0.01) increased during childhood and adolescence. Our results suggest that rDNA promoter methylation and the age-related loss of active rDNA TU, which are a hallmark of the aging process, start only after reaching sexual maturity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274c60d47b8d0482945a4f0864a98745972db01c" target='_blank'>
              rDNA copy number variation and methylation from birth to sexual maturity
              </a>
            </td>
          <td>
            Alina Michler, Sarah Kießling, Jana Durackova, R. Potabattula, Asuman Koparir, Thomas Haaf
          </td>
          <td>2025-06-16</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) accounts for twelve percent of all breast cancer cases, with a survival rate around ten percent lower than ER+/PR+ positive breast cancers. There are limited therapeutic options as these tumors do not respond to hormonal therapy or HER2-targeted treatments. We hypothesized that new insights into pathogenic mechanisms in TNBC can be obtained from studying epigenetic alterations through Hi-C (genome-wide chromosome conformation capture) data analysis.We developed a computational strategy that captured key properties of chromatin conformation while incorporating statistical measures of interaction significance. This model addresses limitations in Hi-C data analysis without relying on predefined features like TADs and compartments. We applied this model to Hi-C and RNA-seq data from TNBC patients, representing the data as multilayer networks to identify genome-wide properties of the TNBC 3D genome.Our network-based analysis revealed distinct chromatin interaction patterns in TNBC compared to healthy contralateral controls. Hi-C data can distinguish interaction patterns related to diseased phenotypes or interaction patterns with potential to exert regulatory effects instead of incidental contacts, but some apparently random interactions may also support important genome regulatory activities.Our findings demonstrate that network-based Hi-C analysis can capture the genome-wide complexity of chromatin interactions in TNBC. This integrative approach provides new insights into the epigenetic mechanisms underlying TNBC pathogenesis and contributes to the advancement of analysis methods for future investigations into novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2cde03b116eaa9a9ec374af95ff1672e86aa8b" target='_blank'>
              Integration of chromosome conformation and gene expression networks reveals regulatory mechanisms in triple negative breast cancer
              </a>
            </td>
          <td>
            Helena Reyes-Gopar, Keila Adonai Pérez-Fuentes, M. Bendall, Enrique Hernández-Lemus
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Maintaining genomic stability is essential for reducing the risk of carcinogenesis. Homologous recombination (HR) is a high‑fidelity DNA repair mechanism that addresses double‑strand breaks and interstrand crosslinks. The present review examined two key components of HR: RAD51, the eukaryotic recombinase and PALB2, a scaffolding protein. Their structural and functional roles are explored in the context of breast and ovarian cancer. RAD51 facilitates homology search and strand invasion, while PALB2 links BRCA1 and BRCA2, stabilizing RAD51 filaments. Mutations in these genes compromise HR, increasing susceptibility to various cancers and impacting treatment efficacy by impairing DNA repair. The present review discussed the clinical implications of RAD51 and PALB2 mutations, focusing on risk stratification, PARP inhibitor efficacy and emerging therapies. Additionally, it highlighted the potential of RAD51 and PALB2 as biomarkers and therapeutic targets, contributing to advances in personalized cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4cbc5968373fbc15750cedb90b63387fd5376bb" target='_blank'>
              RAD51 and PALB2 in precision oncology: Clinical implications for HRD associated breast and ovarian cancers (Review).
              </a>
            </td>
          <td>
            M. A. Kausar, K. Alshammari, Fahaad S Alenazi, Sadaf Anwar, Amany Mohammed Khalifa, Tarig A. N. Ginawi, Abdulaziz Asiri, M. Najm, S. Rabbani, Mohamed El-Tanani, Saumyatika Gantayat
          </td>
          <td>2025-07-03</td>
          <td>International journal of oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 The single-stranded DNA cytosine deaminase enzyme, APOBEC3B (A3B), has been implicated as a mutational driver in multiple human cancers1,2. Breast cancer in particular shows high levels of A3B expression and positive associations with poor clinical outcomes3. Recent studies have also demonstrated that human A3B expression in mice is carcinogenic with significantly shortened life expectancies4. The only genetic factor in breast cancer thus far associated with A3B expression and APOBEC signature single base substitutions is p53 mutation5. Therefore, here we test the hypothesis in vivo that human A3B expression will show pathlogical synergy in the form of accelerated mammary tumor penetrance upon combination with p53 haploinsufficiency due to a heterozygous deletion mutation spanning exons 2-10. Custom and commercially available animals were subjected to standard breeding practices to generate experimental (A3B, p53Delta exons 2-10/+) and control groups (p53Deltanexons 2-10/+). Surprisingly, in contrast to A3B-accelerated tumor development on a wildtype genetic background, A3B had no effect on the rates of tumor development in p53 heterozygous animals (median 13.5 months). The observed genetic epistasis was not due to a lack of A3B expression or activity, as evidenced by strong IHC positivity and a clear acculumlation of APOBEC signature single base substitution mutation in tumors. We conclude that A3B-accelerated tumor development requires full p53 function, and that a haploinsufficiency in p53 enables tumor cells (or their precursors) to better tolerate DNA damage lesions induced by A3B.
 Selected References:
 1. Petljak, M. et al. Addressing the benefits of inhibiting APOBEC3-dependent mutagenesis in cancer. Nat Genet 54, 1599-1608 (2022).
 2. Butler, K. & Banday, A. R. APOBEC3-mediated mutagenesis in cancer: causes, clinical significance and therapeutic potential. J Hematol Oncol 16, 31 (2023).
 3. Roelofs, P. A. et al. Clinical implications of APOBEC3-mediated mutagenesis in breast cancer. Clin Cancer Res 29, 1658-1669 (2023).
 4. Durfee, C. et al. Human APOBEC3B promotes tumor development in vivo including signature mutations and metastases. Cell Rep Med 4, 101211 (2023).
 5. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).\
 Citation Format: Joshua Proehl, Cameron Durfee, Yuan Zhao, Nuri Alpay Temiz, Reuben Harris. p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8de4891dd8d8ec2c399682371385c3645385994" target='_blank'>
              Abstract P3-06-26: p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo
              </a>
            </td>
          <td>
            Joshua Proehl, Cameron Durfee, Yuan Zhao, N. A. Temiz, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5f661d74703dc0b2d864c7015c79812dfe307ea" target='_blank'>
              Decoding the Transcriptional Complexity of the Human BRCA2 DNA Repair Gene Using Hybrid-seq.
              </a>
            </td>
          <td>
            P. Adamopoulos, Michaela A. Boti, Konstantina Athanasopoulou, P. Tsiakanikas, Glykeria N. Daneva, Andreas Scorilas
          </td>
          <td>2025-07-10</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies.We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration.Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration.Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health concern, impacting over half a million women annually. The primary cause is a persistent infection with hr-HPV, which disrupts various cellular processes crucial for normal function. This disruption leads to genetic instability, including changes in the expression of microRNAs and their corresponding host genes, with far-reaching consequences for cellular regulation. Researchers have widely utilized high-throughput technologies to analyze gene expression in cervical cancer, aiming to identify distinct molecular signatures of microRNAs and genes through genomic analysis. However, discrepancies among studies have been noted, possibly due to variations in sample collection, technological platforms, and data processing methods such as normalization and filtering. Therefore, it is essential to synthesize findings from diverse studies to comprehensively understand the molecular mechanisms of regulatory genetic networks involved in the initiation and progression of cervical cancer. This review examined the evidence detailing the role of microRNA signatures and their target genes in cervical carcinogenesis and disease advancement. The accumulated data suggest the presence of widespread regulatory genetic networks active in both precancerous and cancerous cervical cells, potentially acting as key drivers of this malignancy. Identifying these molecular genomic signatures could open new avenues for developing therapeutic strategies for cervical cancer, particularly in the realm of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5110a729c4003c4b384cf7e2d5409b9bf5d54dee" target='_blank'>
              Regulatory Genetic Networks by microRNAs: Exploring Genomic Signatures in Cervical Cancer
              </a>
            </td>
          <td>
            C. Pérez-Plasencia, Y. C. Orbe-Orihuela, Armando Méndez-Herrera, Jessica Deas, Claudia Gómez-Cerón, H. Jiménez-Wences, J. Ortiz-Ortiz, G. Fernández-Tilapa, Aldo F. Clemente-Soto, J. R. Parra-Unda, J. S. Velarde-Félix, M. Rodríguez-Dorantes, O. Peralta-Zaragoza
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 2519


 Background:
 Tumor heterogeneity and low mutational burden limits the availability of effective immunotherapy targets. Aberrant RNA-splicing (neojunctions) represents an underexplored yet promising source of neoantigens. To address this, we developed a neoantigen discovery platform (SNIPP) that characterizes a novel class of clonally-expressed, splicing-derived neoantigens. Furthermore, we validated the immunogenicity of these neoantigens by identifying specific TCRs that drive CD8+ T-cell-mediated tumor killing.
 Methods:
 SNIPP identified public neojunctions by analyzing TCGA RNA-seq data, selecting neojunctions with a positive sample rate (PSR) > 10% and filtering out those found in GTEx normal tissue RNA-seq data (PSR < 1%) across 12 cancer types. To characterize intratumorally conserved neojunctions, we performed maximally-distanced multi-site biopsies (
 n
 = 535) within glioma patients (
 n
 = 56) and generated RNA-seq data for each intratumoral site. Two independent algorithms were utilized to predict peptide processing likelihood and HLA-binding affinity of splicing-derived neoantigen candidates. Neoantigen-specific TCR sequences were identified via
 in vitro
 sensitization of PBMCs and subsequent 10x V(D)J scRNA-seq. These TCRs were transduced into CD8+ T-cells, which were tested downstream for immunogenicity and cytotoxicity against glioma cell lines.
 Results:
 Our pipeline identified 789 public neojunctions, including 32 neojunctions concurrently detected in transcriptomic and proteomic glioma datasets and confidently predicted to be presented by HLA-A*02:01. IVS and subsequent 10x V(D)J scRNA-seq identified TCR clonotypes reactive against neojunctions in
 RPL22
 (
 n
 = 7) and
 GNAS
 (
 n
 = 1), with the latter exhibiting high intratumoral conservation (detected in > 90% of spatially-mapped biopsies across 17/56 patients (26.78%)). TCR-transduced CD8+ T-cells recognized and were immunogenically activated and demonstrated cytotoxicity against endogenously processed and presented neoantigens in GBM and melanoma lines. Additionally, IDH1-mutant oligodendrogliomas exhibited significantly higher neojunction expression compared to IDH1-mutant astrocytomas and IDH1wt subtypes. Differential gene expression analysis (DESeq2) revealed reduced expression of splicing factors in oligodendrogliomas, attributed to their specific co-deletion of chromosomes 1p and 19q. CRISPRi-mediated knockdown of these splicing factors (e.g. SF3A3, SNRPD2) in IDH1wt glioma cells resulted in significantly increased expression of corresponding neojunctions.
 Conclusions:
 Our study highlights a novel class of neoantigens derived from tumor-wide aberrant RNA splicing. The SNIPP platform effectively identifies public intratumorally-conserved neojunctions with strong therapeutic potential. Furthermore, elevated neojunction expression in oligodendroglioma underscores the mechanistic link between dysregulated splicing factor expression and RNA splicing abnormalities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/476976fc09e028193d5d0a3658a6d5d31763d4d8" target='_blank'>
              Tumor-wide RNA splicing aberrations and their potential as therapeutic neoantigen targets.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O. Stevers, Iñaki Etxeberria, T. Nejo, Maggie Colton Cove, Lee-may Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Aaron A. Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideho Okada
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Objective: To undertake a literature review on the development of cervical cancer, emphasizing epidemiological data, human papillomavirus (HPV) infection in carcinogenesis, and the role of the XRCC1 gene in DNA repair, in order to provide valuable information for the medical and scientific community. The goal is to provide a valuable resource for clinical practice. Bibliographic Review: Cervical cancer ranks as the fourth most common malignancy worldwide. Despite known environmental, behavioral, and genetic risk factors, their management remains inadequate. Persistent infection with oncogenic HPV is a well-established risk factor; however, not all infected individuals develop cervical cancer, suggesting the involvement of additional mechanisms. Among these, polymorphisms in the XRCC1 gene have gained attention. XRCC1 encodes a scaffold protein essential for DNA repair coordination and cell cycle regulation. Three main XRCC1 polymorphisms—Arg194Trp, Arg280His, and Arg399Gln—are associated with impaired DNA repair, genomic instability, and increased cervical cancer susceptibility. Final Considerations: A deeper understanding of the genetic and cellular mechanisms underlying cervical cancer may facilitate the development of targeted medical interventions for women, contributing to improved quality of life reduced mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607876d22f013a85473f817b1c9f164dd5e62623" target='_blank'>
              Cervical cancer and XRCC1 gene polymorphisms
              </a>
            </td>
          <td>
            Andréia Michelle Alves Cunha de Alcântara, Ivan Barbosa Barros, Israel Santos da Silva, Karla Roberta Lemos Maul de Souza, Ivan Barbosa Barros, M. Maia, Paulo Roberto Eleutério de Souza
          </td>
          <td>2025-05-30</td>
          <td>Revista Eletrônica Acervo Saúde</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The CHEK2 gene serves a canonical role in the DNA damage response (DDR) pathway encoding the regulatory kinase CHK2 in the homologous recombination (HR) repair of double-strand breaks (DSB). Although CHEK2 is traditionally considered a tumor suppressor gene, recent studies suggest additional functions. Across several cohort studies, CHEK2 expression was negatively correlated with the efficacy of immune checkpoint inhibitors (ICI), which target the interaction between effector immune and tumor cells. This review explores two possible explanations for this observed phenomenon: the first relating to the canonical role of CHEK2, and the second introducing a novel role of the CHEK2 gene in immunomodulation of the tumor microenvironment (TME). DDR mutations have been implicated in increased levels of tumor mutation burden (TMB), often manifesting as neoepitope expression on the tumor cell surface recognized by effector immune cells. As a result, impaired DNA repair due to CHEK2 loss of function, either from germline deleterious variants or acquired mutations, results in the recruitment of CD8+ cytotoxic T-cells and subsequent efficacy of ICI treatment. However, functional loss of CHEK2 may be directly involved in potentiating the immune response through canonical inflammatory and anti-tumor pathways, acting through the cGAS-STING pathway. Although the exact mechanism by which CHEK2 modulates immune responses is still under investigation, combination therapy with CHEK1/2 inhibition and ICI immunotherapy has shown benefit in preclinical studies of several solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561bae3be7aafdc8f4a911392f292dcb271c31d1" target='_blank'>
              Beyond DNA damage response: Immunomodulatory attributes of CHEK2 in solid tumors
              </a>
            </td>
          <td>
            Helen Qian, Heba Ali, Vivekanudeep Karri, Justin T. Low, David M Ashley, Amy B. Heimberger, Lucy A. Godley, A. Sonabend, C. Dmello
          </td>
          <td>2025-06-10</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genome instability is a major force driving tumorigenesis. The ssDNA-binding protein complex CTC1-STN1-TEN1 (CST) plays a pivotal role in maintaining genome stability by countering replication stress, modulating DNA damage repair, and maintaining telomere integrity. Despite its well-documented role in genome maintenance, the involvement of CST in skin cancer development has yet to be investigated. We recently found that CST localizes at stalled DNA replication sites after UV exposure and may suppress the unwanted repriming activity, suggesting a potential role of CST in suppressing genome instability caused by UV damage. In this study, we first analyzed CST expression and alterations in cutaneous melanoma database and found that the CST genes are frequently altered in cutaneous melanoma and their expression is significantly downregulated in melanoma samples compared to normal tissues. We then generated a conditional knockout (cKO) mouse model with STN1 deficiency specifically in melanocytes to investigate its role in skin cancer formation. Upon chronic exposure to UV irradiation, STN1-deficient mice exhibit no obvious difference in melanoma incidence compared to control littermates, suggesting that STN1 downregulation in mature melanocytes has no significant effect on UV-induced skin cancer development in lab mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b040e5d51366f43fd311948a74f723a17b17afb" target='_blank'>
              In vivo investigation of STN1 downregulation in melanoma formation in adult mice following UV irradiation
              </a>
            </td>
          <td>
            Sara Knowles, Fang Wang, Maarten C Bosland, S. Gaddameedhi, Weihang Chai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Thyroid cancer (THCA) is a prevalent malignancy of the head and neck region, yet the mechanisms underlying its tumorigenesis and metastasis remain poorly understood. Given that Rho GTPase activating protein 36 (ARHGAP36) has been implicated in various cellular processes related to cancer progression, including cell migration and invasion, it represents a promising candidate for further investigation in THCA. To investigate the gene expression differences in ARHGAP36 between tumor and normal tissues, the GEPIA and UALCAN databases were utilized. Various factors were also evaluated, including sample types, cancer stages, demographics, histological characteristics and nodal status. The LinkedOmics database was used to constructed a co-expression network for ARHGAP36 in THCA. Gene Ontology (GO) process enrichment analysis for ARHGAP36-associated genes was conducted via Metascape. The TIMER and TISCH databases were employed to explore the relationships between ARHGAP36 and immune markers as well as cell clusters. Functional in vitro assays were performed to assess cellular behaviors such as proliferation, migration and apoptosis. The results indicated that ARHGAP36 expression was significantly elevated in THCA tissues compared with normal tissues. Co-expression analysis revealed significant links between ARHGAP36 and key genes, including IGSF1, DPYSL3, ZCCHC12, CD97, LOXL4, CST2 and WSCD. The enriched GO processes involved T-cell immunity, particularly highlighting the association between ARHGAP36 and CD4+ T cell infiltration. Notably, the downregulation of ARHGAP36 reduced tumor cell proliferation, migration and invasion while enhancing apoptosis. In conclusion, the findings of the present study indicate that ARHGAP36 plays a crucial role in facilitating immune evasion and promoting THCA progression. This underscores its potential as a diagnostic marker and therapeutic target in THCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88d6052ef0cc57d3e67b78040ccff1912bcbe365" target='_blank'>
              ARHGAP36 serves as a diagnostic and therapeutic marker that mediates immune escape and promotes thyroid cancer metastasis
              </a>
            </td>
          <td>
            Liyun Yang, Junzhi Liu, Yuhuan Gao, Shuixian Huang, Geping Wu
          </td>
          <td>2025-06-12</td>
          <td>Biomedical Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetics refers to chemical modifications of histone proteins and DNA that can regulate the expression of genes. The human epigenome is altered dynamically during cell differentiation and aging, and many diseases are associated with aberrant epigenome patterning. Recent advances in CRISPR have led to the development of programmable tools to edit epigenetic modifications at targeted genomic loci, enabling precise rewriting of epigenetic modifications in human cells. CRISPR-based epigenome editors rely on catalytically dead Cas9 coupled with epigenetic modifiers that ultimately result in programmed repression or activation of targeted genes in mammalian genomes. Unlike traditional genome editing methods, epigenome editing does not require DNA breaks or changes in the human genome sequence and thus serves as a safer alternative to control gene expression. In this protocol, we highlight two different methods to perform dCas9-mediated epigenome editing in human cell lines using plasmid DNA transfections and nucleofection of mRNAs encoding CRISPR epigenome editors. We demonstrate programmable epigenome editing to transiently repress genes using CRISPR interference (CRISPRi) and for silencing genes durably for many weeks using CRISPRoff, a fusion of dCas9 to the KRAB domain and de novo DNA methyltransferase complex. We also provide guidance on quantitative methods to measure successful epigenome editing of target genes and key considerations on which epigenome editing tool to use, depending on experimental criteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80b24490560fadbd4a02f2c118998b3d8a637ccd" target='_blank'>
              CRISPR Epigenome Editing in Human Cells using Plasmid DNA Transfection and mRNA Nucleofection Delivery.
              </a>
            </td>
          <td>
            Anna E. Christenson, Nikita S. Divekar, Mitzi G. Hernández Zamora, Carolyn D. Nguyen, James K. Nuñez
          </td>
          <td>2025-05-30</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Disorders of sex development (DSDs) are characterized by discrepancies among karyotype, the gonadal phenotype and gonadal anatomy following birth. Among these, 46, XY DSD is the most complex sub-type and a major cause of birth defects associated with sexual development. However, due to the considerable heterogeneity in pathogenic genes, numerous cases remain genetically undiagnosed. In the present study, a novel gain-of-function variant was identified in the mitogen-activated protein 3 kinase 1 (MAP3K1) gene, contributing to 46, XY DSD through the induction of abnormal cell apoptosis. Genetic analysis of a pediatric male patient and his family revealed a heterozygous MAP3K1 c.4445 G>A variant, resulting in an arginine-to-glutamine substitution. The variant site is highly conserved across species and the amino acid transition induced structural changes in the MAP3K1 protein. In vitro experiments demonstrated that the MAP3K1 c.4445 G>A variant markedly increased apoptotic signaling, leading to abnormal cell apoptosis, disruption of the cell cycle and reduced cell viability. Moreover, the variant exhibited increased levels of ERK1/2 and p38 phosphorylation, indicative of a gain-of-function effect. Subsequent analysis revealed increased expression of the testis-determining gene, SOX9 and reduced expression of the ovary-determining gene, FOXL2. Notably, alterations in gene expression were associated with the MAP3K1 c.4445 G>A variant, providing a mechanistic basis for the pathogenesis of 46, XY DSD. Collectively, these findings offer novel insights into the anti-apoptotic role of MAP3K1, advancing genetic diagnosis and pre-natal screening for individuals with DSDs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2799a19a1fd170b1fea72ad9ddd73906ece00960" target='_blank'>
              A novel variant in the MAP3K1 genomic locus reveals abnormal cell apoptosis as a potential pathogenic mechanism in 46, XY disorders of sex development
              </a>
            </td>
          <td>
            Yufu Lu, Sijia Wei, Shuang Wang, Jingzhi Zhang, Yongjie Xu, Changyudong Huang, Wei Pan, Zhengrong Wang
          </td>
          <td>2025-06-05</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Human endogenous retroviruses (HERVs) comprise genetic material from germ cells that were infected with ancient retroviruses during Pan troglodytes' evolutionary divergence. These "non-coding" repetitive elements now make up approximately 8% of the human genome. Once inserted in the genome, HERVs increase copy number via retrotransposition (a 'copy-and-paste' mechanism) and are involved key functions including transcriptional control and cellular fusion during placenta formation. Aberrant re-expression of HERV-associated genes initially silenced by host epigenetic mechanisms, including DNA methylation, has been linked to many diseases including cancer. Furthermore, in addition to regulating gene expression, including of neighboring genes, reversal of silencing of HERVs may also impact diverse biological functions through the LTRs, and gag, pol, and env proteins. Although previous studies, specifically in white populations, have documented the expression of HERV-K in breast cancer, there is a paucity of data on the expression and role of HERV-R in triple-negative breast cancer (TNBC) among rural West African women from Nigeria. Breast cancer remains the most frequently diagnosed cancer among women globally, with the highest mortality rates across sub-Saharan African nations compared with other countries. Advanced-stage disease at presentation, as well as limited access to treatment options, typically contribute to the poor overall survival rate of breast cancer in African countries including Nigeria. However, the incidence and mortality rates of TNBC in women with West African ancestry are alarmingly high, with an overall survival rate of three years in Zaria, Nigeria. To better understand the molecular basis underlying these high incidence and mortality rates, as well as identify prognostic biomarkers we assessed expression patterns of HERV-R and TNBC-associated genes from Formalin-Fixed Paraffin-Embedded (FFPE) samples of breast cancer, focusing on TNBC in Nigerian women from rural West Africa. To examine gene expression in this study, we used immunohistochemistry, immunoblot, and immunofluorescence, with results showing strong expression of the HERV-R env protein in all but one TNBC patient, as well as in other basal breast cancer subtypes. Our results also strongly suggest that HERV-R expression is associated with overexpression of oncogenes, including TMPRSS2, BRAF, CCND1, BAZ1B, KTM2D, and FOXA1. We found that patients with HERV-R expression have a poor prognosis with low survival. These findings imply that activation of HERV-R may play a role in regulating the expression of genes implicated in TNBC and may serve as a prognostic biomarker in breast cancer.
 Citation Format: Faruk Mohammed, Mohammed Daniyan, Ines Hosni, Halimatu Sadiya Musa, Sani Kamarudeen Owolabi, Adoke Kasimu Umar, Yawale Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie. Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-09-18.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d98da126d396eab075d52af30620900c11d04d" target='_blank'>
              Abstract P1-09-18: Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer
              </a>
            </td>
          <td>
            Faruk Mohammed, Mohammed Daniyan, Ines Hosni, H. S. Musa, S. K. Owolabi, A. Umar, Y. Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The IgLON family of tumor suppressor genes (TSG) impact a variety of cellular processes involved in cancer and non-cancer biology. OPCML is a member of this family and its inactivation is an important control point in oncogenesis and tumor growth. Here, we analyze RNA-Seq expression ratios in ovarian cancers from The Cancer Genome Atlas (TCGA) (189 subjects at Stage III) to identify genes that exhibit a cooperative survival impact (via Kaplan–Meier survival curves) with OPCML expression. Using enrichment analyses, we reconstruct functional pathway impacts revealing interactions of OPCML, and then validate these in independent cohorts of ovarian cancer. These results emphasize the role of OPCML’s regulation of receptor tyrosine kinase (RTK) signaling pathways (PI3K/AKT and MEK/ERK) while identifying three new potential RTK transcriptomic linkages to KIT, TEK, and ROS1 in ovarian cancer. We show that other known extracellular signaling receptor ligands are also transcriptionally linked to OPCML. Several key genes were validated in GEO datasets, including KIT and TEK. Considering the range of OPCML impacts evident in our analyses on both external membrane interactions and cytosolic signal transduction, we expand the understanding of OPCML’s broad cellular influences, demonstrating a multi-functional, pleiotropic, tumor suppressor, in keeping with prior published studies of OPCML function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95469dd0a784b5b2812df87071418bdff5bea009" target='_blank'>
              Bioinformatic RNA-Seq Functional Profiling of the Tumor Suppressor Gene OPCML in Ovarian Cancers: The Multifunctional, Pleiotropic Impacts of Having Three Ig Domains
              </a>
            </td>
          <td>
            Adam G. Marsh, Franziska Görtler, S. Hafizi, Hani Gabra
          </td>
          <td>2025-05-29</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d66d1c34039c0624a67bfae6ffcc0194c9308ee" target='_blank'>
              Bioelectric and Epigenetic Landscapes in Lateralized Breast Tumors Reveal Distinct Tumor Microenvironment Signatures
              </a>
            </td>
          <td>
            Sebastián Real, Sergio R. Laurito, Pablo Gonzalez, Oscar Bello, Joao Carvalho, María Roqué
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Three-prime repair exonuclease 1 (TREX1) is the major 3' to 5' DNA exonuclease in mammals and plays an essential role in preserving immune homeostasis by controlling cytosolic DNA sensing. By degrading excess self and foreign DNA, TREX1 limits aberrant activation of the cGAS-STING pathway and downstream type I interferon responses. Loss-of-function mutations in TREX1 underlie a spectrum of interferon-driven autoimmune and autoinflammatory syndromes, demonstrating its role as a key regulator of immune tolerance. Beyond autoimmunity, recent discoveries have uncovered critical roles for TREX1 in shaping tumor immunogenicity and modulating antiviral defense through regulation of DNA-sensing pathways. In this review, we summarize current insights into the evolutionary origin, structural mechanisms, and functional repertoire of TREX1 in innate immunity. We further discuss how dysregulation of TREX1 contributes to disease and highlight emerging strategies to therapeutically modulate TREX1 activity in cancer and interferonopathies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5f4176925aa63b0deb19594e07a23d1a3bf9c3" target='_blank'>
              TREX1 exonuclease in immunity and disease.
              </a>
            </td>
          <td>
            Zehua Shang, Lei Wang, Wen Zhou
          </td>
          <td>2025-07-08</td>
          <td>International immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db40ae083dc749116865ffafc0606bdc245362b3" target='_blank'>
              The penetrant chordoid glioma PRKCA mutation is an oncogenic gain-of-function kinase inactivation eliciting early onset chondrosarcoma in mice
              </a>
            </td>
          <td>
            V. Calleja, Jack C Henry, Mathias Cobbaut, Joanne Sewell, Karine Rizzoti, Francesca Houghton, Stefan Boeing, Nneka Anyanwu, Sunita Varsani-Brown, Thomas Snoeks, A. Suárez-Bonnet, S. Priestnall, N. Q. McDonald, A. Cameron, Peter J. Parker
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 e14558


 Background:
 DNA polymerases ε (
 POLE
 ) and δ1 (
 POLD1
 ) encode enzymes involved in DNA replication and proofreading. Somatic
 POLE
 /
 D1
 mutations occur in 1-3% of cancers. Loss-of-function alterations typically reside within the
 POLE
 /
 D1
 exonuclease domain and can result in high tumor mutation burden (TMB). Ultrahigh TMB (≥100 mut/Mb) is often associated with
 POLE/D1
 mutations, and results in near-universal responses to immune checkpoint inhibitors (ICI). Deciphering the contribution of
 POLE
 /
 D1
 mutations to TMB can be challenging, as these often co-occur with microsatellite instability (MSI-H) which also drives TMB.
 Methods:
 Samples were analyzed by NGS (NextSeq or NovaSeq) (Caris Life Sciences, Phx, AZ). TMB (totaled somatic mutations found per megabase pair) was evaluated in patients with
 POLE
 /
 POLD1
 /
 D2
 /
 D3
 /
 D4
 -altered cancers. MSI-H or microsatellite stable (MSS) status was determined by IHC and NGS. Cohorts were divided into pathogenic and likely pathogenic variants (i.e. exonuclease domain mutations) versus VUS alterations (i.e. variants outside the exonuclease domain). Differences in TMB between cohorts were assessed using Fisher’s exact test.
 Results:
 Pan-cancer, TMB ≥20 / ≥50 was identified in 10.9% / 3.4% of
 POLE
 VUS MSS cancers compared to 3.1% / 0.4% of MSS cancers with
 POLE
 /
 D1
 WT status (
 p
 < 0.0001). 0.94% of
 POLE
 VUS cancers had ultrahigh TMB versus 0.05% of MSS
 POLE
 /
 D1
 WT cancers (
 p
 < 0.0001). Similar rates were observed in MSS cancers with
 POLD1
 VUSs (Table). MSS cancers with pathogenic
 POLE
 variants exhibited variable rates of ultrahigh TMB depending on cancer type, ranging from 52.6% (ovarian) to 62.9% (endometrial) to 80.0% (colorectal). Notably, ultrahigh TMB occurred in 0.53%, 0.44%, and 0.44% of MSS patients with
 POLD2
 ,
 POLD3
 , and
 POLD4
 mutations (
 p
 < 0.0001), after excluding coexisting
 POLE
 /
 D1
 mutations.
 Conclusions:
 We propose that certain non-exonuclease
 POLE
 /
 D1
 mutations can result in loss-of-function and ultrahigh TMB in MSS patients. Further, tumor type appears to influence the impact of established
 POLE
 /
 D1
 driver mutations on TMB. Finally,
 POLD2
 -
 4
 mutations (not previously studied) may also contribute to genome instability and mutational load. These findings argue for deeper characterization of DNA polymerase mutations to identify patients that may benefit from ICI.






 POLE
 /
 D1
 WT + MSS[n (%)]


 POLE
 VUS Mutation + MSS[n (%)]


 POLE
 Pathogenic Mutation + MSS[n (%)]


 POLD1
 VUS Mutation + MSS[n (%)]


 POLD1
 Pathogenic Mutation + MSS[n (%)]





 Total
 139,842
 5,658
 557
 1,793
 11


 TMB ≥20
 4,378(3.1%)[ref]

 614(10.9%)
 p
 <0.0001


 529(95.0%)
 p
 <0.0001


 168(9.4%)
 p
 <0.0001


 5(45.5%)
 p
 <0.0001



 TMB ≥50
 586(0.42%)[ref]

 193(3.4%)
 p
 <0.0001


 467(83.8%)
 p
 <0.0001


 79(4.4%)
 p
 <0.0001


 2(18.2%)
 p
 =0.0009



 TMB ≥100
 73(0.05%)[ref]

 53(0.94%)
 p
 <0.0001


 343(61.6%)
 p
 <0.0001


 28(1.6%)
 p
 <0.0001


 1(9.1%)
 p
 =0.0058







 Evaluated by Fisher’s exact test comparing to respective POLE/D1 WT + MSS cohort.




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eb38b846148e42ca1249ce42b9e558a15a34574" target='_blank'>
              Potential impact of non-exonuclease
 POLE
 and
 POLD1
 mutations on high and ultrahigh TMB across cancers.
              </a>
            </td>
          <td>
            Alexander K. Tsai, S. Deshmukh, Sharon Wu, Dave S. B. Hoon, W. El-Deiry, Axel Grothey, Justin H Hwang, E. Antonarakis
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="In recent years, the incidence of cancer has increased, and early diagnosis and treatment are key to cancer treatment. Nucleolar and spindle-associated protein 1 (NUSAP1) is a crucial mitotic regulator. NUSAP1 binds to microtubules and mediates their attachment to chromosomes, ensuring the accurate distribution of genetic material to the two daughter cells. NUSAP1 is located on chromosome 15q15.1 and is typically expressed in proliferating cells, suggesting a potential oncogenic role. Studies have shown that NUSAP1 plays a significant role in various cancers, including breast cancer, cervical cancer, lung cancer, prostate cancer, and hepatocellular carcinoma. Furthermore, NUSAP1 may serve as a promising biomarker for cancer diagnosis and prognosis. In this review, we discuss the roles and mechanisms of NUSAP1 in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e17ed1053e2b63b1afa28756e4052dc1859c3cc0" target='_blank'>
              The role and therapeutic value of NUSAP1 in human cancers
              </a>
            </td>
          <td>
            Shenyuan Xu, Ding Nan, Ruiqi Liu, Chenting Liu, Yuefeng Hu, Luanluan Huang, Xiaoyan Chen, Hailong Sheng, Yanwei Lu, Haibo Zhang
          </td>
          <td>2025-07-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Alternative splicing (AS) is a pivotal post-transcriptional mechanism that expands the functional diversity of the proteome by enabling a single gene to generate multiple mRNA and protein isoforms. This process, which involves the differential inclusion or exclusion of exons and introns, is tightly regulated by splicing factors (SFs), such as serine/arginine-rich proteins (SRs), heterogeneous nuclear ribonucleoproteins (hnRNPs), and RNA-binding motif (RBM) proteins. These factors recognize specific sequences, including 5′ and 3′ splice sites and branch points, to ensure precise splicing. While AS is essential for normal cellular function, its dysregulation is increasingly implicated in cancer pathogenesis. Aberrant splicing can lead to the production of oncogenic isoforms that promote tumorigenesis, metastasis, and resistance to therapy. Furthermore, such abnormalities can cause the loss of tumor-suppressing activity, thereby contributing to cancer development. Importantly, abnormal AS events can generate neoantigens, which are presented on tumor cell surfaces via major histocompatibility complex (MHC) molecules, suggesting novel targets for cancer immunotherapy. Additionally, splice-switching oligonucleotides (SSOs) have shown promise as therapeutic agents because they modulate splicing patterns to restore normal gene function or induce tumor-suppressive isoforms. This review explores the mechanisms of AS dysregulation in cancer, its role in tumor progression, and its potential as a therapeutic target. We also discuss innovative technologies, such as high-throughput sequencing and computational approaches, that are revolutionizing the study of AS in cancer. Finally, we address the challenges and future prospects of targeting AS for personalized cancer therapies, emphasizing its potential in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c2739ebba4691f4fc04db8de970276d260f0dc2" target='_blank'>
              Alternative Splicing in Tumorigenesis and Cancer Therapy
              </a>
            </td>
          <td>
            Huiping Chen, Jingqun Tang, Juanjuan Xiang
          </td>
          <td>2025-05-29</td>
          <td>Biomolecules</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Genomic imprinting is the parent-of-origin specific monoallelic expression of genes that result from complex epigenetic interactions. It is often achieved by monoallelic 5-methylcytosine, resulting in the formation of differentially methylated regions (DMRs). These show a bias towards oocyte-derived methylation and survive reprogramming in the pre-implantation embryo. Imprinting is widespread in the human placenta. We have recently performed whole-genome screens for novel imprinted placenta-specific germline DMRs (gDMRs) by comparing methylomes of gametes, blastocysts and various somatic tissues, including placenta. We observe that, unlike conventional imprinting, for which methylation at gDMRs is observed in all tissues, placenta-specific imprinting is associated with transient gDMRs, present only in the pre-implantation embryo and extra-embryonic lineages. To expand the list of bona fide imprinted genes subject to placenta-specific imprinting, we reinvestigated our list of candidate loci and characterized two novel imprinted genes, PIK3R1 and G0S2, both of which display polymorphic imprinting. Interrogation of placenta single-cell RNA-seq datasets, as well as cell-type methylation profiles, revealed complex cell-type specificity. We further interrogated their methylation and expression in placental samples from complicated pregnancies, but failed to identify differences between intrauterine growth restricted or pre-eclamptic samples and controls, suggesting they are not involved in these conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b750b53f10506274107b5e1972c3a2fd8cd37694" target='_blank'>
              PIK3R1 and G0S2 are human placenta-specific imprinted genes associated with germline-inherited maternal DNA methylation
              </a>
            </td>
          <td>
            Dagne Daskeviciute, Becky Sainty, Louise Chappell-Maor, Caitlin Bone, Sarah Russell, Isabel Iglesias-Platas, Philippe Arnaud, Ana Monteagudo-Sánchez, Maxim V. C. Greenberg, Keran Chen, Africa Manerao-Azua, G. Perez de Nanclares, Jon Lartey, David Monk
          </td>
          <td>2025-06-26</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22bcb6bbbf4e6c64ab8d5a6fa755b5306451c07b" target='_blank'>
              EHMT1 mediates cellular motility in embryonal rhabdomyosarcoma by activating SOX8 expression.
              </a>
            </td>
          <td>
            Upasana Bajaj, Dipanwita Das, Jia Yu Leung, Nurul Fateha Binti Abdul Rashi, R. Taneja
          </td>
          <td>2025-06-04</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Understanding how the assembly of nascent mRNA into a ribonucleoprotein (mRNP) influences R-loop homeostasis is crucial for gaining insight into the cellular mechanisms that prevent genome instability. Here, we identify three RNA-binding proteins, Rie1, Rim4 and She2, whose expression levels are important to limit R-loop accumulation and, thus, to prevent DNA damage. Interestingly, Rim4 and She2 are overrepresented in CBP80-containing mRNPs formed in the absence of THO. In addition, we found that an excess of the RNA exosome component Dis3 impairs its function, promoting R-loops, particularly from non-coding RNAs, which cause genomic instability. Our results indicate that changes in the availability of different RBPs or RNAs, causes R-loop-mediated DNA damage in the cell. These results may help to understand the mechanism that promotes cancer, as several RBPs are overexpressed in different types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58254c8464c859b1341c1f452949c65b62663dc9" target='_blank'>
              Cellular imbalance of specific RNA-binding proteins associates with harmful R-loops
              </a>
            </td>
          <td>
            José Antonio Mérida-Cerro, Guillaume Chevreux, Benoît Palancade, Ana G. Rondón, A. Aguilera
          </td>
          <td>2025-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cell cycle structures vary significantly across cell types, which exhibit distinct phase compositions. Asynchronous DNA replication and dynamic cellular characteristics during the cell cycle result in considerable heterogeneity in DNA dosage, chromatin accessibility, methylation, and expression. Nonetheless, the consequences of cell cycle disruption in the interpretation of multi-omics data remain unclear. Here, we systematically assessed the influence of distinct cell phase structures on the interpretation of omics features in proliferating cells, and proposed solutions for each omics dataset. For copy number variation (CNV) calling, asynchronous replication timing (RT) interference induces false CNVs in cells with high S-phase ratio (SPR), which are significantly decreased following replication timing domain (RTD) correction. Similar noise is observed in the chromatin accessibility data. Moreover, for DNA methylation and transcriptomic analyses, cell cycle-sorted data outperformed direct comparison in elucidating the biological features of compared cells. Additionally, we established an integrated pipeline to identify differentially expressed genes (DEGs) after cell cycle phasing. Consequently, our study demonstrated extensive cell-cycle heterogeneity, warranting consideration in future studies involving cells with diverse cell-cycle structures. RTD correction or phase-specific comparison could reduce the influence of cell cycle composition on the analysis of the differences observed between stem and differentiated cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7f0e00dfe690501686ff54d8b38db35d315d13" target='_blank'>
              Mitigating Cell Cycle Effects in Multi-Omics Data: Solutions and Analytical Frameworks.
              </a>
            </td>
          <td>
            Rui Nie, Caihong Zheng, Likun Ren, Yue Teng, Yaoyu Sun, Lifei Wang, Junya Li, Jun Cai
          </td>
          <td>2025-05-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The pituitary tumor transforming gene 1 (PTTG1), discovered in 1997 by Pei and Melmed, takes part in cellular replication, cell cycle control, DNA repair mechanisms, organogenesis, metabolism regulation, cellular transformation, and senescence. Its biological actions include protein-protein interactions, modulation of gene transcription, and, other than intracellular and autocrine mechanisms, even paracrine activities. For the reasons mentioned above, PTTG1 stands out as a multifaceted regulator of cancer biology; it is involved in genomic and chromosomal instability, local invasiveness, neo-lymphangiogenesis, and metastatic spreading. In solid neoplasms, endocrine neoplasms, although deemed rare, have experienced a significant increase in diagnostic incidence in recent years. Endocrine cancers are still a major challenge in healthcare and research since several questions remain unanswered, even though researchers have made considerable efforts to uncover their causes. Twenty-seven years have passed since PTTG1's discovery, and several works have been published. However, only the tip of the iceberg has been unveiled. Herein, we review current knowledge of PTTG1's action in endocrine cancers, such as pituitary, thyroid, testicular, adrenal, pancreatic, and ovarian.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a57eb69f3187a01c5af405bf459aed1d53f3b1f4" target='_blank'>
              Pituitary tumor transforming gene 1 and endocrine cancers: an up-to-date review through history, current insights and future perspectives.
              </a>
            </td>
          <td>
            E. Vergani, Emanuela Teveroni, Francesca Mancini, F. Di Nicuolo, S. Raia, S. Chiloiro, F. Pierconti, Antonio Bianchi, A. M. Isidori, Alfredo Pontecorvi, Domenico Milardi
          </td>
          <td>2025-06-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common cancers, with an incidence that has increased significantly over the last 20 years. The World Health Organization (WHO), under its cancer program, identifies CRC as the third most prevalent cancer worldwide with a high mortality rate, especially in patients under the age of 50. Despite advances in surgery, chemotherapy, radiotherapy, and molecular targeted therapy, CRC shares a low survival rate and poor prognosis due to late diagnosis. To address these challenges, research into alternative diagnostic and therapeutic strategies has increasingly focused on molecular mechanisms, including noncoding RNAs (ncRNAs). Circular RNAs (circRNAs), a subclass of endogenous ncRNAs characterized by their covalently enclosed loop structure, demonstrate greater stability than linear RNAs, making them potential candidates for clinical applications. The circRNAs possess differential expressions in cancers and function as tumor suppressive or oncogenic activities. This review discusses the recent findings on circRNAs and their potential for biomarkers and therapeutic targets in CRC. As circRNAs can serve as miRNA sponges, increase or decrease mRNA expression, and function to regulate an oncogenic or tumor suppressor pathway, there may be the possibility for an early diagnosis, prognosis, or therapeutic role of circRNAs in CRC. Highlighting the clinical implications of circRNAs, this review discusses their potential to transform current CRC management strategies and address critical gaps in timely diagnosis and effective treatment. It also emphasized the need for further clinical research to validate their utility and translate these findings into practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db80f20f575f8c248f315abb0f46ec8b7f78d57b" target='_blank'>
              A review of circular RNAs in colorectal cancer: insights into biomarker discovery and therapeutic targeting
              </a>
            </td>
          <td>
            Sibin Nambidi, Karunakaran Bhuvaneswari Shruthi, Sujatha Padmanaban Kanimozhe, Antara Banerjee, S. Kondaveeti, A. Duttaroy, S. Pathak
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) greatly contribute to the growth, invasion, metastasis and drug resistance of neuroblastoma (NB). This study aimed to construct a CAF-related prognostic model and identify the immune status of patients with NB via single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq). Methods ScRNA-seq data of NB acquired from the Gene Expression Omnibus (GEO) database were used to identify cellular subpopulations. Bulk gene expression data were downloaded from GEO, The Cancer Genome Atlas (TCGA) and ArrayExpress databases and the prognostic model was constructed using univariate Cox and Least Absolute Shrinkage and Selection Operator (LASSO) analyses. Differences in immune infiltration, therapeutic responses and signaling pathways between the high- and low-risk groups were investigated. Finally, immunohistochemistry was performed to evaluate the protein expressions and a nomogram based on the risk signature and clinical characteristics was constructed. Results ScRNA-seq data of eight NB samples were integrated to identify 253 marker genes for CAF. An eight-gene prognostic CAF-related signature was established based on the GEO data. The CAF model was strongly associated with immune infiltration, drug response and active signaling pathways in tumors. Univariate and multivariate Cox regression analyses verified that the CAF model was as an independent prognostic indicator, and a nomogram integrating the clinical signature and CAF-related risk signature was constructed for clinical prediction. Conclusions The CAF-related signature can effectively predict the prognosis of NB and provide new genomic evidence for anti-CAF immunotherapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03023-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0c7a00465eed83a4f94e4167616471e1046f423" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis establishes a cancer associated fibroblast-related signature for predicting prognosis and therapeutic responses in neuroblastoma
              </a>
            </td>
          <td>
            Zhiyao Cao, Qi Wang, Yali Han, Jianwei Lin, Qi Wu, Chencheng Xu, Jingchun Lv, Lei Zhang, Hongxiang Gao, Dapeng Jiang
          </td>
          <td>2025-07-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The ASXL1 gene is one of the most frequently mutated genes in acute myeloid leukemia (AML). It is associated with signs of aggressiveness and adverse clinical outcomes. The aim of the current study was to analyze the genetic profile of ASXL1 gene mutations and its impact on the overall survival in AML patients from Pakistan. Thirty-eight well characterized AML patients were enrolled, and DNA sequencing of the ASXL1 was performed using the Illumina NextSeq500 next generation sequencing (NGS) system. Standard pipeline of bioinformatics tools was used to determine the mutational profile. The mutational profile of the enrolled AML patients was compared with that of 1000 Genomes project, and TCGA AML datasets. The analysis revealed 43 genetic variants in ASXL1 across the 38 AML patients (1.13 variant/patient). Eight rare variants were observed in exons 12, 13 of the ASXL1 gene. Notably, a recurrent rare nonsynonymous deleterious variant p.G1336S in exon 13 (NM_015338 transcript) was found in two patients (5.26%). The overall survival of the ASXL1+ (but TP53, FLT3, NPM1, EZH2, and WT1 negative) AML was shorter compared with the ASXL1- (p < 0.05). Further, the overall survival of current study ASXL1 + AML was found comparable with that of the TCGA AML. In conclusion, the non-silent mutations in ASXL1 were associated with lower survival in AML. Further studies with larger cohort are suggested for subsequent clinical implementation. Supplementary Information The online version contains supplementary material available at 10.1186/s12863-025-01324-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03f43436989187938c1f8f140ca42098c22feb4" target='_blank'>
              Genetic profile of ASXL1 gene in risk assessment in acute myeloid leukemia
              </a>
            </td>
          <td>
            Salem Ahmed Bamusa, Wardah Qureshi, Atia Gohar, Muhammad Irfan, I. Khan, Muhammad Shakeel
          </td>
          <td>2025-07-12</td>
          <td>BMC Genomic Data</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aim: Chemotherapy resistance, often linked to the development of resistance against genotoxic agents, is a major obstacle in cancer treatment. FoxM1, a transcription factor frequently overexpressed in malignancies such as breast cancer, is strongly associated with genotoxic therapy resistance. The aim of this study is to conduct a comparative analysis of the effects of thiostrepton (THIO), a FoxM1 inhibitor, on the DNA damage response in HUVEC cells (non-malignant) and MDA-MB-231 breast cancer cells (malignant) 
Materials and Methods: THIO's impact on cell viability were evaluated in both cell lines using the MTT assay. Oxidative DNA damage levels were measured with the 8-OHdG kit, and apoptosis was assessed using the Caspase 3 ELISA kit. The expression levels of DNA damage response genes (BRCA-1, DNAPKC, FOXM1, RAD51, MRE11 and XRCC1) were analyzed by RT-PCR. 
Results: MDA-MB-231 cells exhibited greater sensitivity to the cytotoxic effects of THIO than HUVEC cells. In HUVEC cells, THIO caused a significant increase in oxidative DNA damage, whereas no such effect was observed in MDA-MB-231 cell lines. Conversely, breast cancer cells showed a significant increase in Caspase 3 levels. RT-PCR results revealed a marked downregulation of DNA damage response genes, particularly BRCA-1, DNAPKC, MRE11, FOXM1, and XRCC1, in both cell types. 
Conclusion: THIO downregulates DNA damage response genes by inhibiting FoxM1 expression in both malignant and non-malignant cells. This effect suggests that THIO may enhance the sensitivity of breast cancer cells to therapy by disrupting DNA repair pathways, presenting potential for overcoming chemotherapy resistance. Further studies are required to validate these findings and explore THIO's therapeutic utility in breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0302ae4a15025cfb10555437aa0808dee744207" target='_blank'>
              FoxM1 inhibition by Thiostrepton downregulates DNA damage response genes, enhancing sensitivity of breast cancer cells to therapy
              </a>
            </td>
          <td>
            F. Demırtaş Korkmaz, Zekeriya Düzgün, Fadime Mutlu İçduygu
          </td>
          <td>2025-06-10</td>
          <td>Ege Tıp Dergisi</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ad49026046d30dd6909e4b9cd5b26991a13bef" target='_blank'>
              Tumor aneuploidy as a prognostic and predictive biomarker in immune checkpoint blockade.
              </a>
            </td>
          <td>
            Daniel Huang, L. F. Spurr, R. Weichselbaum, S. Pitroda
          </td>
          <td>2025-06-30</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cholangiocarcinoma is a highly heterogeneous malignant tumor, including intrahepatic cholangiocarcinoma, hepatoportal cholangiocarcinoma and distal cholangiocarcinoma. Its incidence is increasing worldwide and currently accounts for approximately 15% of all primary liver cancers and 3% of all gastrointestinal malignancies. There is a lack of early diagnostic methods for cholangiocarcinoma, and the overall treatment effect is poor, with a 5-year survival rate of less than 25%. New biomarkers are urgently needed in clinical practice to improve the current diagnosis and treatment status. Circulating tumor DNA (ctDNA) is DNA fragments released by tumor cells, which can show tumor-specific gene mutations (such as IDH1/2, FGFR2 fusion) and epigenetic modifications (such as abnormal methylation). With the rapid development of tumor liquid biopsy technology, ctDNA has been gradually applied in solid tumors such as lung cancer and colorectal cancer due to its high sensitivity and dynamic monitoring capabilities. This review systematically introduces ctDNA technology and its progress in early screening, early diagnosis, treatment response, and prognosis monitoring of cholangiocarcinoma. In addition, this review also summarizes the challenges and limitations of current ctDNA technology and analyzes future hot research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/582e6b4bb69a2b394d91aff713a3ea41c9e46004" target='_blank'>
              Circulating tumor DNA in cholangiocarcinoma: current clinical applications and future perspectives
              </a>
            </td>
          <td>
            Yi Wang, Yike Li, Zhong Liang, Yuqiao Zhang, Tong Li, Chenjun Tian, Jinyu Zhao, Boru Jin, Jie Cao, Yanyan Lin
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Loss of the Y chromosome (LOY) in peripheral blood mononuclear cells (PBMCs) is the most common somatic alteration in men and is associated with higher mortality from epithelial cancers1–3. In tumours, epithelial LOY is also associated with poor survival4–7. This raises several fundamental questions, such as why LOY in PBMCs drives cancer mortality and whether there is a relationship between LOY in PBMCs, PBMC-derived immune cells and cancer cells (and, if so, what its consequences are). We sought to answer these questions through a comprehensive pan-cancer analysis of bulk and single-cell RNA sequencing data from 29 human tumour types, along with autochthonous and syngeneic mouse models. In human and mouse tumours, malignant epithelial cells had the highest LOY prevalence, yet LOY was also present in tumour stromal and immune cells, with LOY in malignant epithelial cells predicting LOY in benign cells. LOY also correlated between paired tumour and PBMC samples from patients. Among benign cells, LOY induced the strongest shift in CD4+ and CD8+ T cells, with both showing transcriptomic signatures of immunosuppression. Furthermore, the magnitude of LOY in epithelial cells, CD4+ T cells and CD8+ T cells independently predicts survival, with tumours exhibiting concurrent epithelial and T cell LOY having the worst outcomes. Here we establish a model that links LOY in immune cells to LOY in malignant cells, which may explain in part why LOY in PBMCs is associated with increased cancer mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/521330e6a0acc4c997107eda98c17326ccf4fe70" target='_blank'>
              Concurrent loss of the Y chromosome in cancer and T cells impacts outcome
              </a>
            </td>
          <td>
            Xingyu Chen, Yiling Shen, Suhyeon Choi, Hany A Abdel-Hafiz, Mukta Basu, Lena Hoelzen, Martina Tufano, Saravana Kumar Kailasam Mani, Maryam Ranjpour, Jiani Zhu, V. K. Ramanujan, E. Koltsova, Vinicius F Calsavara, S. Knott, Dan Theodorescu
          </td>
          <td>2025-06-04</td>
          <td>Nature</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU-023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T > C in KKU-023, m.9493G > A in KKU-055, m.9172C > A in KKU-100, m.15024G > C in KKU-213A, m.12994G > A in KKU-452, and m.13406G > A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92494ba8290eb1c5fd7c0d865dc1a82ce3683d1" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines.
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, Poramate Klanrit, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, Hasaya Dokduang, W. Loilome
          </td>
          <td>2025-06-09</td>
          <td>PloS one</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e86dbb7bd01d6e90b441b62dcc070c2bc321518" target='_blank'>
              High-fidelity long-read sequencing reveals a complex RCCX locus at the single-nucleotide level in Korean patients with congenital adrenal hyperplasia
              </a>
            </td>
          <td>
            S. Kim, J.-H. Yoon, D. Kim, S. Park, G.-H. Kim,, , J.-H. Choi,, J. Kim, J. Kim
          </td>
          <td>2025-07-11</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac2fa84e64df92a9b47c54f45baf0d93294fea97" target='_blank'>
              Substrate heterogeneity promotes cancer cell dissemination
              </a>
            </td>
          <td>
            Zuzana Dunajova, Saren Tasciyan, Juraj Májek, J. Merrin, E. Sahai, M. Sixt, É. Hannezo
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="


 What is the prevalence of mitotic-origin aneuploidies in human blastocysts using a PGT platform capable of identifying cell division origin of aneuploidy? A single-center experience



 Although uncommon, some embryos are diagnosed with aneuploidies of mitotic origin only. Diagnosing aneuploidies without determining their cellular origin risks discarding embryos with potential viability.



 Chromosomal abnormalities in human embryos are a significant factor contributing to reproductive failure. Next-generation sequencing (NGS) platforms are widely accepted for evaluating chromosomal abnormalities, as they are copy number-focused technologies. However, in some cases, biopsies reveal a mix of normal and abnormal cells, referred to as mosaic embryos. The developmental potential of these mosaic embryos remains unclear, although successful outcomes, including normal live births, have been reported. Moreover, novel PGT approaches, such as high-density single nucleotide polymorphism (SNP) arrays, offer the capability to predict the cell division origin of aneuploidies, moving beyond a sole focus on chromosome copy numbers.



 This retrospective study analyzed 310 consecutive PGT-A biopsies obtained from 72 patients across 91 treatment cycles conducted between May 2023 and December 2024 at a private fertility clinic. Both autologous and donor cycles were included, and all blastocysts were evaluated using trophectoderm (TE) biopsy and DNA samples from the biological parents.



 Trophectoderm biopsies were obtained, with 5–10 cells collected for each sample. Parental DNA samples were also required to serve as reference material for the tested embryos. Trophectoderm biopsies and parental DNA were processed using SNP arrays to identify the cell division origin of aneuploidy, integrating both genotyping and copy-number data. Aneuploidies were classified by origin as maternal meiotic (from the oocyte), paternal meiotic (from the sperm), or mitotic (embryonic, arising post-zygotic division).



 Among the total embryos studied (n = 310), 44.8% (139/310) were identified as aneuploid. Of these, 34.8% (108/310) exhibited at least one maternal meiotic aneuploidy, with an average maternal age of 36.3 ± 4.4 years, consistent with the well-established correlation between advancing maternal age and chromosomal anomalies. Additionally, 7.7% (24/310) of the aneuploid embryos displayed paternal meiotic errors, while 8.7% (27/310) presented mitotic errors originating post-fertilization. Notably, 5.16% (16/310) of the embryos displayed exclusively mitotic-origin aneuploidies, underscoring the importance of mitotic anomalies in early embryonic development. Among the 139 abnormal embryos, 81.3% (113/139) exhibited whole chromosome aneuploidies, consistent with meiotic segregation errors involving entire chromosomes. Conversely, 13.67% (19/139) of the abnormal embryos exhibited segmental aneuploidies, involving duplications or deletions of specific chromosomal regions. A small subset of embryos, accounting for 5.04% (7/139), displayed karyotypes that included both whole chromosome aneuploidies and segmental abnormalities, reflecting the complexity of chromosomal errors in human embryos.



 This is a retrospective study with a relatively small number of embryos analyzed. Expanding the cohort of blastocysts and evaluating the clinical implications of aneuploidy origin prediction are necessary. Additionally, further research is needed to assess the reproductive potential of embryos with segmental chromosome aneuploidies.



 Identifying the cell division origin of aneuploidies provides deeper insights into embryonic chromosomal abnormalities and their developmental potential. This approach could improve clinical decision-making in assisted reproduction, enhancing embryo selection strategies and informing future studies on the implications of mitotic and segmental aneuploidies in human reproduction.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/839a948ebef49cd1b00902c78815394f99d075d1" target='_blank'>
              P-242 From Cell Division to Clinical Decisions: Unveiling the True Prevalence of Mitotic-Origin Aneuploidies in Preimplantation Embryos
              </a>
            </td>
          <td>
            M. J. Gomez Cuesta, D. Marin Chavarria, M. I. Gutierrez Estrada, M. Garcia Galicia, G. D. Cortes Ruiz, M. Alonso de Mendieta, C. Kandel Varsano, C. Lopez Valencia, E. Cervantes Bravo
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, R. Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Breast cancer is characterized as highly heterogenous and is a representative model to understand how molecular features of tumor biology determine therapeutic strategy. LINC01133 exhibits opposing expressing patterns across different breast cancer subtypes, yet its roles and mechanisms in ER+ breast cancer remain a loaded question. Methods The expression of LINC01133 was initially assessed utilizing a public dataset TCGA and subsequently validated within clinical samples through RT-qPCR and in situ hybridization (ISH). To determine the role of LINC01133, various assays, including colony formation, Transwell, 5-ethynyl-2′-deoxyuridine (EdU) labeling, and mouse xenograft experiments, were performed. Additionally, RNA immunoprecipitation (RIP), RNA pull-down, mass spectrometry (MS), and RNA stability assays were conducted to elucidate its mechanisms. Results LINC01133 was dramatically downregulated in ER+ breast cancer, which results in unfavorable prognosis. Functionally, LINC01133 inhibited migration and invasion in vitro and metastasis in vivo of ER+ breast cancer cells. Mechanistically, LINC01133 can directly interact with IGF2BP2 protein promoting its ubiquitination and degradation. The downregulation of LINC01133 was mediated by m6A modification, catalyzed by METTL3 and recognized by YTHDF2, causing half-life reduction and accelerated degradation of LINC01133. Conclusion Our findings revealed the downregulation of LINC01133 in ER+ breast cancer and provided novel insight to the role of METTL3/YTHDF2/LINC01133/IGF2BP2 axis in ER+ breast cancer, which might offer a novel perspective in the design and development of novel anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd1c654f899d20ee944e199054f4ac06718cd700" target='_blank'>
              The m6A modification of LINC01133 suppresses ER+ breast cancer progression by modulating IGF2BP2 protein stability via a ubiquitination-dependent mechanism
              </a>
            </td>
          <td>
            Mai-dong Li, Ben-jie Shan, Lei Wang, Shuang Gao, Li Hao, Hai-yang Yu, Yue-yin Pan
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35fd329abfeae45d934694f31530eb9db169304e" target='_blank'>
              A mouse model of PTEN Hamartoma Tumour Syndrome reveals that loss of the nuclear function of PTEN drives macrocephaly, lymphoid overgrowth, and late-onset cancer
              </a>
            </td>
          <td>
            Priyanka Tibarewal, V. Rathbone, Sarah E Conduit, G. Classen, Fiona M Black, Mohammad Amin Danesh, Georgia Constantinou, Zeinab Asgarian, Koujiro Tohyama, N. Kriplani, Virginia Alvarez Garcia, Elizabeth Foxall, Djenat Belarbi, M. Leverve, Wayne Pearce, Mahreen Adil, Zofia Varyova, Lucia Conde, Adriana Alves, G. Masson, Roger L. Williams, Adrienne M Flanagan, Javier Herrero, Katerina Stroud, M. Tischkowitz, K. Lachlan, Cheryl L Scudamore, Mark G H Scott, Nicholas R. Leslie, N. Kessaris, B. Vanhaesebroeck
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3115319084e3008e2ffcee186c2141b84b2e97e1" target='_blank'>
              Assembly and delivery of large DNA via chromosome elimination in yeast.
              </a>
            </td>
          <td>
            Yuan Ma, Yao Xiong, Jiayu Xu, Hui Xu, Zong-Heng Fu, Guang-Rong Zhao, Yi Wu, Yingjin Yuan
          </td>
          <td>2025-07-10</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Recent studies have revealed that polyclonality—where multiple distinct subclones cooperate during early tumor development—is a critical feature of tumor evolution, as demonstrated by Sadien et al. and Lu et al. in Nature (October 2024). These findings show that early polyclonal interactions can overcome fitness barriers, ultimately transitioning to monoclonality as dominant clones emerge. Understanding and targeting these interclonal dynamics offers new therapeutic opportunities. In this perspective, we outline how computational modeling and artificial intelligence (AI) tools can provide deeper insights into tumor polyclonality and identify actionable therapeutic strategies. By applying ligand–receptor interaction analysis, clonal trajectory reconstruction, network and pathway modeling, and spatial analysis, researchers can prioritize communication hubs, evolutionary bottlenecks, and microenvironmental niches that sustain tumor progression. These approaches, when integrated with experimental validation, offer a translational pathway from foundational discoveries to personalized cancer treatments aimed at disrupting cooperative subclonal ecosystems and preventing malignant progression. We commend the recent Nature publications, “Polyclonality overcomes fitness barriers in Apc-driven tumorigenesis” by Sadien et al. [1] and “Polyclonal-to-monoclonal transition in colorectal precancerous evolution” by Lu et al. [2], both featured on 2024 October 30. These groundbreaking studies employed distinct lineage tracing methods to investigate the origins and evolutionary dynamics of colorectal and intestinal tumorigenesis. Despite their different approaches, both studies reached convergent conclusions: Polyclonality plays a pivotal role in the early stages of tumor development, providing critical insights into how diverse cellular populations collaborate to overcome fitness barriers and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6580f8163812203e6792c0b0ae89c7c42827a3" target='_blank'>
              Artificial Intelligence-Powered Insights into Polyclonality and Tumor Evolution
              </a>
            </td>
          <td>
            Hong Zhao, T. Ideker, Stephen T C Wong
          </td>
          <td>2025-06-13</td>
          <td>Research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Numerous long noncoding RNAs (lncRNAs) are generated in response to external stimuli, but the scope and functions of such activity are not known. Here, this study provides insight into how the transcription of lncRNAs is connected to DNA damage response by identifying the lncRNA ZFAS1, which is required for cell cycle arrest, transcription regulation, and DNA repair. Mechanistically, ZFAS1 facilitates dynamic changes in hyperphosphorylated forms of the large subunit of RNA polymerase II (RNAPII) around transcription initiation sites by directly targeting the regulated genes. It is shown that extensive transcription shutdown and concomitant stimulated engagement of RNAPII-Ser2P are crucial for repair and cell survival upon genotoxic stress. Finally, ZFAS1 knockout in mice dampened nucleotide excision repair (NER) and led to kidney dysplasia. Overall, the findings extend the understanding of lncRNAs in DNA damage repair (DDR) and imply a protective role of lncRNA against DDR-deficient developmental disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd71da5d91ac1bdf35ea5bcddbb6ad4cb3db5ea" target='_blank'>
              IncRNA-ZFAS1, an Emerging Gate-Keeper in DNA Damage-Dependent Transcriptional Regulation.
              </a>
            </td>
          <td>
            Jiena Liu, Qing Lu, Zixuan Fan, Jiahui Lin, Nan He, Xin Zhang, Zhaoya Han, Tingting Zhu, Zhenzhen Wu, Yingying Xu, Yuming Wang
          </td>
          <td>2025-05-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 8624


 Background:
 Lung adenocarcinoma (LUAD), the most common lung cancer subtype, has poor survival rates and limited treatment options. Among its molecular drivers, loss of methylthioadenosine phosphorylase (MTAP) is linked to aggressive development and poor outcomes, yet its downstream effects remain unclear. MTAP is essential for purine biosynthesis and S-adenosyl methionine (SAM) production, a key methyl donor for epigenetic regulation. MTAP-loss leads to methylthioadenosine (MTA) accumulation, which inhibits methyltransferase activity and disrupts epigenetic control. However, the impact of MTAP deficiency on transcriptional and metabolic programming in LUAD is still largely unknown, highlighting a critical gap for targeted therapy. This study examines these molecular consequences to identify potential therapeutic vulnerabilities in MTAP-deficient LUAD.
 Methods:
 RNA-sequencing data for 510 LUAD samples were obtained from The Cancer Genome Atlas (TCGA) via cBioPortal. Samples were classified as MTAP-loss (n=64) or MTAP-normal (n=446) based on mutational data. Differential gene expression was performed using the DESeq2 package for R and the Benjamini-Hochberg procedure was used to control for false discovery rate. DNA methylation (Illumina Human Methylation 450k) was compared between MTAP-loss (n=46) and MTAP-normal (n=404) cohorts. Significantly altered probes were mapped to differentially expressed genes (DEGs). Pathway and gene ontology (GO) analyses were conducted using KEGG, GO, and EnrichR to identify dysregulated pathways.
 Results:
 DNA methylation analysis revealed 581 hypomethylated and only 51 hypermethylated probes in MTAP-loss samples, underscoring a global hypomethylation phenotype likely driven by MTA accumulation. We identified 343 differentially expressed, hypomethylated genes in MTAP-loss LUAD samples. Of these, upregulated genes were highly enriched in mitochondrial function and stress response pathways, whereas downregulated genes were linked to cell differentiation and developmental processes, suggesting an epigenetically driven metabolic reprogramming. Notably, these alterations may confer heightened cellular survival and adaptability under stress, while curtailing normal differentiation programs.
 Conclusions:
 Our findings indicate that MTAP loss in LUAD leads to a coordinated shift in DNA methylation and gene expression, promoting survival-focused metabolic and stress responses at the expense of normal regulatory pathways. These results highlight novel vulnerabilities in MTAP-deficient tumors and suggest potential targets for precision therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bac3c5fbe15b9c30b43fa429b943e30d1102b5e9" target='_blank'>
              Epigenetic and transcriptional consequences of MTAP-loss in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Swati Pothukuchi, Jihao Xu, Ssu-Wei Hsu, Ching-Hsien Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="PURPOSE
Telomere maintenance mechanisms including MYCN amplification (MYCN-A) (via upregulated telomerase reverse transcriptase [TERT] expression), TERT rearrangements (TERT-RA), and ATRX mutations confer neoplastic immortality in neuroblastoma (NB) cells. Clinical characterization of patients with NB harboring activating somatic TERT promoter point mutations (TERT-PM) in NB may improve stratification.


METHODS
To identify TERT-PM and TERT-RA, tumors were profiled by the Memorial Sloan Kettering Cancer Center-Integrated Mutation Profiling of Actionable Cancer Targets targeted next-generation sequencing platform and whole-genome sequencing, respectively. Associated clinical outcomes were studied.


RESULTS
TERT-PM and TERT-RA were detected in 17 of 603 (2.8%) and 31of 168 (18.4%) tumors from individual patients, respectively. The median age at diagnosis was 32 (range, 15-79) and 48 (range, 3-168) months for TERT-PM and TERT-RA, respectively. TERT-PM were located at canonical hotspots (C228T [16/17] and C250T [1/17]) and were concurrent with MYCN-A in 9 of 17 (53%). By contrast, MYCN-A occurred in 1 of 31 TERT-RA tumors. Most patients with TERT-PM had stage M (94%) and high-risk NB (HR-NB) (81%) at diagnosis. Twenty-eight patients with TERT-RA had HR-NB (90%). After risk appropriate therapy, complete response was achieved in 7 of 17 (41%) and 17 of 31 (55%) patients with TERT-PM and TERT-RA, respectively. The median progression-free (PFS) and overall survival (OS) were 9.7 ± 1.2 and 16.5 ± 2 months for patients with TERT-PM. Corresponding PFS/OS for patients with TERT-RA were 20 ± 6.4/56.8 ± 8.6 months (P = .015 for OS). Excluding MYCN-A status produced no significant survival difference (P > .1) between the two groups. CNS relapse occurred in 11 of 31 (35%) patients with TERT-RA versus 1 of 17 (6%) patients with TERT-PM.


CONCLUSION
TERT-PM is rarer than TERT-RA but both are associated with HR-NB and poor prognosis. MYCN-A frequently co-occurs with TERT-PM and might represent an ultra-high-risk subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/556a0902c3148a7408804dc2c0f9318f277250b6" target='_blank'>
              Clinical Significance of TERT Promoter Mutations in Neuroblastoma.
              </a>
            </td>
          <td>
            Natasha Persaud, G. Gundem, B. Kushner, Marc Ladanyi, Neerav Shukla, S. Modak
          </td>
          <td>2025-07-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Melanoma brain metastases (MBM) have a unique molecular profile compared to extracranial metastases (ECM). Description of the biological features and clinical outcomes of MBM will facilitate the design of rational therapies. Methods We examined the mutational landscape and gene expression profiles of MBM (74 patients) and ECM (34 patients) in paired patient samples from a previously published dataset with whole-exome sequencing (WES) and RNA sequencing (RNAseq) data from MD Anderson Cancer Center (MDACC). We also present findings from MBM from a new cohort of 14 patients from Duke University to strengthen investigation of somatic mutations and gene expression profiles. Gene Set Enrichment Analysis (GSEA) was used to compare paired MBM versus lymph node (LN) metastases and skin metastases. Relative immune cell abundance was inferred using deconvolution methods. Survival outcomes from craniotomy and associations with biological features, BRAF mutation status, and PTEN expression were assessed. Results GSEA found that autophagy signaling pathways are enriched in MBM versus LN and skin metastases. BRAF was the most frequently mutated clinically relevant gene in MBM and ECM, with NRAS and PTEN also frequently altered in MBM. The most strongly upregulated genes in autophagy pathways were glial fibrillary acidic protein (GFAP) and hemoglobin beta (HBB). An increased proportion of immune-suppressive M2 compared to tumor-suppressive M1 macrophages in MBM and ECM was identified. There was not sufficient evidence for an association between BRAF V600 mutation status or expression and overall survival (OS) from craniotomy. Conclusions The mutational landscape and gene expression of MBM from the Duke cohort resembled those previously reported in the MDACC cohort. Upregulation of autophagy pathways was observed in patient-matched MBM versus LN and skin metastases due to upregulation of two genes, GFAP and HBB. In MBM, higher M2:M1 ratio may contribute to a therapeutically relevant immune-suppressive tumor microenvironment (TME). Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02035-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd2acbbd5230dc401bd3ec3c002ad4d3ee572431" target='_blank'>
              A comprehensive, multi-center, immunogenomic analysis of melanoma brain metastases
              </a>
            </td>
          <td>
            L. B. Kennedy, Amanda E. D. Van Swearingen, Marissa R Lee, Layne W. Rogers, A. Sibley, Jeff Sheng, Dadong Zhang, Xiaodi Qin, E.S. Lipp, Swaminathan Kumar, Aron Y Joon, Pixu Shi, Michael A. Davies, K. Owzar, Carey K. Anders, April K S Salama
          </td>
          <td>2025-06-02</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Recent studies suggest that tail regeneration in lizards begins with a tumor-like stage usually termed regenerative blastema. Oncogenes and tumor suppressors are activated in blastema cells, resulting in a balanced cell proliferation that does not turn the blastema into a tumor. This outgrowth elongates forming new tissues and tail. We previously showed that physiological extracts from regenerating lizard tissues inhibit the growth of cancer cells in vitro within 2-4 days of administration, demonstrating that the growing lizard blastema contains regulatory molecules which can also influence human cancer cells. The molecules responsible for this inhibition were not identified in that initial study. In the present experimental study, after specific extractions of RNAs and/or proteins from the regenerating tail of lizard, we have confirmed the inhibition of breast cancer cell vitality in vitro within 2-3 days from their addition to the culture medium. Proteolysis or heat denaturation of proteins abolished the inhibitory effect. RNA delivered to breast cancer cells in vitro through lipid vesicles (liposomes) showed the highest inhibition of cancer cells vitality. Cell degeneration, detected by microscopy, revealed that RNA is more effective than proteins extracted from regenerating tissues. The present observations further suggest that RNAs coding for known tumor suppressor proteins, and non-coding RNAs that are highly expressed in the regenerating tail, may be key inhibitors (tumor suppressors) of blastema and cancer cell proliferation. The evolution of a mechanism for the self-remission of tumor growth in lizards remains uncertain, but continuing study of this reptile may help uncover natural mechanisms for tumor growth inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbbc4431106b7f4d802c168046ca986e622ee0cd" target='_blank'>
              RNA and proteins extracted from the regenerating tail of lizards determine inhibition of cancer cell proliferation.
              </a>
            </td>
          <td>
            Nicola Greco, Maurizio Onisto, Lorenzo Alibardi
          </td>
          <td>2025-06-13</td>
          <td>The International journal of developmental biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CHROMR is a primate-specific long noncoding RNA with emerging roles in homeostasis and pathophysiology. Elevated blood levels of CHROMR have been observed in patients with cardiovascular disease and several cancers, where it is correlated with poor clinical outcomes. Like many lncRNAs, CHROMR accumulates in both the nucleus and the cytoplasm, and it assumes distinct functions in each of these cellular compartments. In the nucleus, CHROMR sequesters a transcriptional repressor complex to activate interferon-stimulated gene expression and antiviral immunity. In the cytoplasm, CHROMR competitively inhibits microRNAs involved in cholesterol efflux and cell cycle regulation, thereby impacting gene pathways involved in reverse cholesterol transport, HDL biogenesis, and tumor growth. In this review, we detail the multifaceted functions of CHROMR in cholesterol metabolism, innate immunity, and cancer progression. We also explore the potential molecular mechanisms that govern its expression and dynamic subcellular localization, which may be key to its functional versatility. Advancing our understanding of the regulatory networks and cellular environments that shape CHROMR activity will be critical for assessing its promise as a therapeutic target and diagnostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e434fd00ac8710c07c05ad122a8af40ef9af54ab" target='_blank'>
              The Multifaceted Roles of CHROMR in Innate Immunity, Cancer, and Cholesterol Homeostasis
              </a>
            </td>
          <td>
            Emma R. Blaustein, Coen van Solingen
          </td>
          <td>2025-06-01</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (PIK3CA) plays a critical role in cell growth and survival. PIK3CA somatic mutations are linked to the PIK3CA-related overgrowth spectrum (PROS) syndrome. Cells from those patients appeared to be associated with a moderate but significant radiosensitivity. Mutations or amplifications in this gene are common in breast, colorectal, and lung cancers. Alterations in the PIK3CA gene, including amplification and mutation, are common in cancer, but their influence on radiotherapy is not yet fully understood. This report reveals a potential association between PIK3CA gene modifications and radiosensitivity (p < 0.05) deducted from 8 established colorectal cancer cell lines (HT-29, DLD-1, HCT-116, SW480, HCT-15, Colo-320, and LoVo). Meanwhile gene amplification (> 2) seems to be linked to increased radiation sensitivity, mutations appear to be associated with increased radioresistance in colorectal cancer cells. Leveraging this relationship, PIK3CA amplification and mutations could act as biomarker to pinpoint patients who might benefit from more personalized radiotherapy regimens. However, this association is preliminary and hypothesis-generating in view of the limited number of cases. Further studies are needed to confirm this conclusion. By uncovering the distinct mechanistic effects of these PIK3CA alterations on radiosensitivity phenotype in both normal and cancerous cells, researchers can lay the groundwork for tailored radiotherapy strategies in colorectal cancer. This insight could enhance treatment effectiveness while reducing side effects, ultimately leading to improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d2f05a3aac70c0b974b05ec72c5f8dff7113c5" target='_blank'>
              Effects of PIK3CA Gene Modifications on Radiosensitivity in Colorectal Cancer Cells.
              </a>
            </td>
          <td>
            Ghazi Alsbeih, K. Al-Hadyan
          </td>
          <td>2025-06-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosome segregation is essential for cellular proliferation. Unlike eukaryotes, bacteria lack cytoskeleton-based machinery to segregate their chromosomal DNA (nucleoid). The bacterial ParABS system segregates the duplicated chromosomal regions near the origin of replication. However, this function does not explain how bacterial cells partition the rest (bulk) of the chromosomal material. Furthermore, some bacteria, including Escherichia coli, lack a ParABS system. Yet, E. coli faithfully segregates nucleoids across various growth rates. Here, we provide theoretical and experimental evidence that polysome production during chromosomal gene expression helps compact, split, segregate, and position nucleoids in E. coli through nonequilibrium dynamics that depend on polysome synthesis, degradation (through mRNA decay), and exclusion from the DNA meshwork. These dynamics inherently couple chromosome segregation to biomass growth across nutritional conditions. Halting chromosomal gene expression and thus polysome production immediately stops sister nucleoid migration, while ensuing polysome depletion gradually reverses nucleoid segregation. Redirecting gene expression away from the chromosome and toward plasmids causes ectopic polysome accumulations that are sufficient to drive aberrant nucleoid dynamics. Cell width enlargement experiments suggest that limiting the exchange of polysomes across DNA-free regions ensures nucleoid segregation along the cell length. Our findings suggest a self-organizing mechanism for coupling nucleoid compaction and segregation to cell growth without the apparent requirement of regulatory molecules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38632c47e3d185d80d1fdc81e020f23db296ddb" target='_blank'>
              Nonequilibrium polysome dynamics promote chromosome segregation and its coupling to cell growth in Escherichia coli
              </a>
            </td>
          <td>
            Alexandros Papagiannakis, Q. Yu, Sander K. Govers, Wei-Hsiang Lin, N. Wingreen, C. Jacobs-Wagner
          </td>
          <td>2025-06-24</td>
          <td>eLife</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="DNA methylation is faithfully inherited during cell division, playing a crucial role in maintaining cellular identity. The process of DNA methylation maintenance relies on DNA methyltransferase DNMT1 and the ubiquitin E3 ligase UHRF1. UHRF1 facilitates the ubiquitination of both the replication factor PAF15 and histone H3, with each ubiquitin signal regulating replication-coupled and uncoupled DNA methylation maintenance, respectively. Over the past decades, advances in structural biology have significantly deepened our understanding of the molecular mechanisms governing DNA methylation maintenance. In particular, the emergence of cryo-electron microscopy (cryo-EM)-often referred to as the "Resolution Revolution"-has transformed many areas of biology, including epigenetics and chromatin biology. This review focuses on the structural mechanisms of DNA methylation maintenance, as revealed by the three-dimensional structures of key biomolecular complexes and discusses the potential development of inhibitors targeting DNA methylation maintenance factors based on structural insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4573541b6e9d478f35cbbab36494e717c68f84cd" target='_blank'>
              A Comprehensive Review of Structural Insights into DNA Methylation Maintenance.
              </a>
            </td>
          <td>
            Amika Kikuchi, K. Arita
          </td>
          <td>2025-06-05</td>
          <td>Genes & genetic systems</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 e15034


 Background:
 Endometrial cancer (EC) illustrates diverse nucleotide substitution and chromosomal re-arrangement signatures like other cancers. Namely, four molecular types are identified, replicative polymerases, microsatellite stable, copy number high/serous-like, and microsatellite instability characterized by mutations in mismatch repair genes. With advances in molecular techniques, early prediction of disease aggressiveness can be revealed using comprehension in ECs. Although few cell proliferative genes have been documented, a complete molecular landscaping is essential for effective treatment planning and decision-making. We report, the association between clinical disease status and corresponding molecular findings to establish the role of the latter in the early prediction of disease aggression in endometrial cancer.
 Methods:
 Retrospectively, we investigated the association of genomic mutational profiles of 40 endometrial cancer patients and their clinical disease status from PET scans and HPE reports. Next Generation Sequencing (NGS) test was performed using a comprehensive genomic panel (CGP) OncoIndx assay.
 Results:
 Amongst 40 patients, 65% (n = 26/40) of patients had progressive and metastatic disease from PET scans and HPE reports indicative of disease aggressiveness. Almost 80% (n = 20/26) of the aggressive disease cohort were detected to have a minimum of one genomic alteration. Their mutational profiles showed a diverse spectrum of pathway-related aberrations from cell proliferation, progression, double-strand DNA (dsDNA) repair, mismatch repair (MMR), and tumor suppression constituting a total of 83 genomic alterations detected in the whole cohort. Further, the cell proliferation and dsDNA repair pathways exhibited the highest mutational load of 57.8% (n = 48/83) and 14.5% (n = 12/83) respectively in the cohort. Likewise, tumor suppressor genes constituted the lowest frequency at 6% (n = 5/83).In addition, five patients were detected with high MSI scores and one patient with an intermediate MSI score of which two MSI-high patients were also found to harbor MMR gene mutations (MSH3:p.K383Rfs*32, p.D403H, p.E456* and p.E512*, MSH6:p.F1088Sfs*2).
 Conclusions:
 Using CGP, we observed mutational frequencies of cell proliferation and dsDNA repair pathways that act as crucial markers for disease aggression in EC. We also noted that high MSI scores were not always concordant with mutations in the classical MMR genes and vice versa. This also indicates the individual significance of estimating MSI scores in these patients for clinical decision-making. Since the identified landscape of mutations well correlated with the disease states categorized from their clinical findings, they could potentially transform into molecular attributes for early prediction of progression and metastasis aiding treatment planning and decision making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8df9baa418d967a1f642449b698f5fd35ecbbbc" target='_blank'>
              Mutational spectrum of cell proliferation genes as early predictive markers for aggressive disease in endometrial cancers.
              </a>
            </td>
          <td>
            Aarthi Ramesh, Rohit Ranade, Sandhya Iyer, Nidhi Patel, A. D'Souza, Sangeeta Prajapati, Bhagwat Jadhav, A. Vasudevan, A. Bharde, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer cell spheroids autonomously form in the ascites fluid and are considered a conduit for epithelial ovarian cancer metastasis within the peritoneal cavity. Spheroids are homotypic, avascular 3D structures that acquire resistance to anoikis to remain viable after cellular detachment. We used in vitro spheroid model systems to interrogate pathways critical for spheroid cell proliferation, distinct from those driving monolayer cancer cell proliferation. Using the 105C and KOC-7c human ovarian clear cell carcinoma (OCCC) cell lines, which have distinct proliferative phenotypes as spheroids but the same prototypical OCCC gene mutation profile of constitutively activated AKT signaling with the loss of ARID1A, we revealed therapeutic targets that efficiently kill cells in spheroids. RNA-seq analyses compared the transcriptome of 3-day monolayer and spheroid cells from these lines and identified the characteristics of dormant spheroid cell survival, which included the G2/M checkpoint, autophagy, and other stress pathways induced in 105C spheroids, in sharp contrast to the proliferating spheroid cells of the KOC-7c cell line. Next, we assessed levels of various G2/M checkpoint regulators and found a consistent reduction in steady-state levels of checkpoint regulators in dormant spheroid cells, but not proliferative spheroids. Our studies showed that proliferative spheroid cells were sensitive to Wee1 inhibition by AZD1775, but the dormant spheroid cells showed a degree of resistance to AZD1775, both in terms of EC50 values and spheroid reattachment abilities. Thus, we identified biomarkers of dormant spheroids, including the G2/M checkpoint regulators Wee1, Cdc25c, and PLK1, and showed that, when compared to proliferating spheroid cells, the transcriptome of dormant OCCC spheroids is a source of therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877fd898a1fb77cad11de61e359dcf933c175ace" target='_blank'>
              Transcriptomic Analyses of Ovarian Clear Cell Carcinoma Spheroids Reveal Distinct Proliferative Phenotypes and Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Bartlomiej Kolendowski, Sylvia Cheng, Yudith Ramos Valdés, Trevor G. Shepherd, G. DiMattia
          </td>
          <td>2025-05-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Mitotic chromosomes are considered to be universally folded as loop arrays across species and cell types. However, some studies suggest that features of mitotic chromosomes might be cell type or species specific. We previously reported that CTCF binding in human differentiated cell lines is lost in mitosis, whereas mitotic mouse embryonic stem cells (mESC) display prominent binding at a subset of CTCF sites. Here, we perform footprint ATAC-seq analyses of mESCs and somatic mouse and human cells confirming these findings. We then investigate roles of mitotically bookmarked CTCF in prometaphase chromosome organization by Hi-C. We do not find any remaining interphase structures such as TADs or loops at bookmarked CTCF sites in mESCs. This suggests that mitotic loop extruders condensin I and II are not blocked by CTCF, and thus that maintained CTCF binding does not alter mitotic chromosome folding. Lastly, we compare mitotic Hi-C data generated in this study in mouse with public data in human and chicken. We do not find any cell type-specific differences; however, we find a difference between species. The average genomic size of mitotic loops is smaller in chicken (200-300 kb), compared to human (400-600 kb) and especially mouse (1-1.5 mb). Interestingly, we find that this difference is correlated with the genomic length of q-arms in these species, a finding we confirm by microscopy measurements of chromosome compaction. This suggests that the dimensions of mitotic chromosomes can be modulated through control of loop size by condensins to facilitate species-appropriate shortening of chromosome arms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7587b4758e2a0d2dc6b16f50b764f5c00ce6820f" target='_blank'>
              Mitotic chromosomes harbor cell type and species-specific structural features within a universal loop array conformation.
              </a>
            </td>
          <td>
            Marlies E. Oomen, A. N. Fox, Inma Gonzalez, A. Molliex, Thaleia Papadopoulou, Pablo Navarro, J. Dekker
          </td>
          <td>2025-06-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="
 e15032


 Background:
 Homologous recombination deficiency (HRD) is an event triggered by impairment of the homologous recombination repair (HRR) pathway which acts on DNA double-strand breaks and interstrand cross-links (ICL). Although HRD has been shown to elevate the overall genomic instability in tumors inducing favorable responses to immune checkpoint inhibitors (ICI), its clinical utility is observed in specific cancer types like breast, ovarian, low-grade glioma, testicular germ cell tumors and kidney cancers. On the contrary, in many other cancer types, HRD is conducive to the development of immune-cold tumor microenvironment leading to unfavorable treatment responses to ICI. We present, HRD predictive scores with the corresponding IO biomarkers viz. microsatellite instability (MSI) and tumor mutation burden (TMB) pan-cancers.
 Methods:
 Retrospectively, we investigated the biomarker profile (HRD, MSI, TMB) and genomic (HRR pathway genes) profiles of 642 pan-cancer patients using a comprehensive genomic assay using OncoIndx.
 Results:
 Of the cohort of 642 pan-cancer patients, a total of 12.3% (n = 79/642) of patients were detected with high HRD scores, another 16.0% (n = 103/642) were detected with intermediate HRD scores and while 71.7% (n = 460/642) of patients showed low HRD scores. Among the cohorts of high, intermediate and low HRD scores, BRCA1/2 mutation-positive patients were 6.3% (n = 5/79), 5.8% (n = 6/103), and 5.2 (n = 24/460) respectively and HRR gene mutation-positive patients were 17.7% (n = 14/79), 25.2% (n = 26/103), 17.17% (n = 79/460) respectively. Most importantly, the immune-oncology biomarkers MSI and TMB scores were high only in 1.3% (n = 1/79) and 8.9% (n = 7/79) in HRD-high patients which were not significant compared to 1.7% (n = 8/460) and 10.4% (n = 48/460) observed in HRD-low patient cohort. Even in the cohort of patients with intermediate HRD scores, MSI and TMB did not vary significantly. Overall, no significant correlation was observed between high HRD scores vs BRCA1/2 mutations, HRR pathway gene mutations, MSI and TMB statuses. Thus, HRD scores could be predicted high, irrespective of all the associated biomarkers and pathway mutations. This indicates the utility of additional treatment options in BRCA1/2 wild-type or HRR pathway gene wild-type patients.
 Conclusions:
 Although HRD-high patients could be predictive of favorable PARPi and platinum-based therapies, their role in immune activation and ICI therapy response should be carefully considered. Even at a high score, HRD seems to be inducing non-inflammatory tumor microenvironment as opposed to triggering novel cell surface antigens and high TMB aiding in immunotherapy response. However, HRD as an individual biomarker alone adds immense therapeutic benefits even among BRCA-negative patients who could be potential for PARPi/platinum-based therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92c61b0baeee9c5a60d5b38acb29b5a9cb82e2f" target='_blank'>
              The non-canonical role of homologous recombination deficiency in the development of immune-cold tumor microenvironment.
              </a>
            </td>
          <td>
            A. Vasudevan, Aarthi Ramesh, Ankit Jain, Sandhya Iyer, Nidhi Patel, A. Joshi, M. K. Singhal, Sameer Shrirangwar, Chinmaya Kumar Pani, Ganapathi Bhat, V. Patil, Dr T. Raja, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad0e72d616aeb3ee8d41accd4d7a574cb580739" target='_blank'>
              A continuum reaction-diffusion model for spread of gene silencing in chromosomal inactivation
              </a>
            </td>
          <td>
            Shibashis Paul, Sha Sun, Karmella A. Haynes, Tian Hong
          </td>
          <td>2025-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 5072


 Background:
 STEAP
 1 and 2 (six-transmembrane epithelial antigen of prostate) are metalloreductase proteins involved in a variety of biologic processes.
 STEAP1/2
 are tumor-associated cell surface antigens highly expressed in prostate cancer (PC), although their role in cancer is poorly understood.
 STEAP1/2
 have emerged as successful targets for adoptive T-cell therapy trials for PC. We employed a multi-omics approach to investigate the molecular features associated with
 STEAP1
 and
 STEAP2
 expression in PC.
 Methods:
 NextGen Sequencing of DNA (592 genes or whole exome) and RNA (whole transcriptome) was performed for PC tumors (n = 7089) submitted to Caris Life Sciences (Phoenix, AZ). PC samples were stratified by
 STEAP1/2
 mRNA levels into top (high) and bottom quartile (low). Immune cell infiltration in the tumor microenvironment (TME) was inferred by quanTIseq. Transcriptomic signatures of androgen receptor signaling (AR), neuroendocrine classification (NEPC), and interferon gamma signaling (IFN) were calculated. Mann-Whitney U and X2/Fisher-Exact tests were applied where appropriate, with P-values adjusted for multiple comparisons (q< .05).
 Results:
 Of 7,089 samples, 63.2% were from the prostate; 11.7% from lymph node metastases (LNM); 7.3% from bone; and 17.8% from visceral/soft tissue metastases (V/STM).
 STEAP
 -1 and -2 were significantly correlated to each other (R= 0.90, p<.001), with significantly higher expression of
 STEAP1
 observed in primary prostate and LNMs, compared with reduced expression in V/STM (
 STEAP1
 TPM: 105.2 vs 140.6 vs 91.9 p<.001). Mutations in
 AR
 (3.8% v 1.9%),
 KDM6A
 (4.2% v 2.2%),
 SPOP
 (10.9% v 8.4%) and
 AR
 V7 (23.0% v 10.5%) were enriched in
 STEAP1
 high PC (each q<.01). Mutations in
 KDM6A
 (3.8% v 2.5%) and
 AR
 V7 (17.6% v 14.3%) were enriched in
 STEAP2
 high PC (each q<.05).
 STEAP1/2
 expression negatively correlated with TMB count (R =-0.03, p<.05) and IFN score (R = -0.26, p<.001). Concordantly, fewer proinflammatory immune cell fractions (M1 Macrophages, NK cells, CD4+/CD8+ T cells, myeloid dendritic cells) were observed within the TME of
 STEAP1/2
 high PC (p<.0001). However,
 STEAP1/2
 expression correlated positively with the AR signature (R = 0.39, p<.001) and androgen response pathways, while correlating negatively with the NEPC signature (R = -0.15, p<.001).
 Conclusions:
 PC tumors expressing high
 STEAP1/2
 display distinct genomic and transcriptomic profiles compared to
 STEAP1/2
 -low PC, and
 STEAP 1/2
 expression varies across sites of metastases. Immune biomarkers and immune cell infiltration data suggest that STEAP1/2 may be associated with a cold TME. The recent success of STEAP1-targeting T-cell redirecting therapies mechanisms by which adoptive T-cell strategies may overcome immunosuppressive factors within the TME. Ongoing development of T-cell immunotherapeutics targeting STEAP1 may account for the differential expression profiles in guiding patient selection and combination strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3db32deff1c58e26b5169abd853a15c53e234839" target='_blank'>
              Molecular characterization of STEAP1 and -2 in advanced prostate cancer.
              </a>
            </td>
          <td>
            Kevin K. Zarrabi, T. Adeyelu, A. Elliott, D. Geynisman, D. Oh, Carissa E. Chu, Rana R McKay, Nicholas A Zorko, E. Antonarakis, Lucia R Languino, Pedro C. Barata, D. Danila, Norm Smith, William Kevin Kelly
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Ovarian cancer ranks as the eighth most common malignancy in women globally. Early detection remains a critical challenge for improving ovarian cancer diagnosis and treatment outcomes. Our prior study demonstrated that ARHGAP40 downregulation in basal cell carcinoma is attributed to CpG island hypermethylation. However, the expression and methylation status of ARHGAP40 in ovarian cancer remain unexplored. Here, we investigated ARHGAP40 protein expression in normal fallopian tubes, ovarian benign tumors, borderline tumors, low-grade serous carcinoma (LGSC) and high-grade serous carcinomas (HGSC). Methylation analysis of the ARHGAP40 was performed using bisulfite sequencing PCR (BSP). MethyLight assays were developed to detect methylated circulating tumor DNA (ctDNA) fragments of ARHGAP40. IHC results revealed absent or weak ARHGAP40 protein expression in 93.8% (30/32) of HGSC, 11.1% (2/18) of borderline tumors, and in 14.3% (1/7) of LGSC cases. ARHGAP40 protein expression was robust expressed in all normal fallopian tubes (15/15) and benign ovarian tumors (8/8). CpG island hypermethylation in the ARHGAP40 promoter showed a strong inverse correlation with protein expression (P < 0.001). Furthermore, methylated ctDNA for ARHGAP40 was detected in 80.0% (4/5) of HGSC patients’ plasma, but not in benign tumor (0/3) and healthy controls (0/15). Our findings suggest that promoter hypermethylation may be a mechanism underlying ARHGAP40 silencing in HGSC. Detection of methylated ARHGAP40 ctDNA may serve as a noninvasive biomarker for early diagnosis and monitoring of HGSC. While our findings suggest the potential of methylated ARHGAP40 as an early diagnostic biomarker, the small sample size warrants validation in larger cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a13bc1c0523b1d5d86e6d21e3c7e8792952655" target='_blank'>
              Methylated ARHGAP40 DNA as a potential biomarker for early diagnosis in high-grade ovarian serous cancer
              </a>
            </td>
          <td>
            Jiaxin Chai, Dongni Leng, Shuwei Guo, Rusong Zhang, Jiandong Wang, Yun Gu, Qiu Rao
          </td>
          <td>2025-07-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Introduction: Epigenetic changes play a crucial role in cancer development. Among these, DNA methylation is one of the most significant due to its impact on gene expression and genome stability. Hyper-methylation generally suppresses transcription and hypo-methylation promotes it. In breast cancer research, tumor-adjacent tissue is often used as a reference for characterizing epigenetic alterations, but numerous studies suggest that the tissue adjacent to a tumor shares genomic characteristics with the tumor itself. The extent to which epigenetic profiles overlap between tumor, histologically normal tissues from the involved and uninvolved breast, and benign tissues from cancer-free individuals is unknown. Our study aims to address this gap by systematically comparing the DNA methylation profiles across these different tissue types. Methods: A total of 72 cancer patients were selected from Northwestern University, while data from 182 cancer-free women was obtained from tissue donated to the Komen Tissue Bank. From each individual cancer patient, four regional samples were collected: tumor tissue (TU), tumor-adjacent normal tissue (AN), benign tissue from the opposite quadrant of the involved breast (OQ), and benign tissue from the contralateral uninvolved breast (CUB). From cancer-free patients, normal breast tissue was collected (CFN). From these, we created a “tumor proximity axis”: CFN→CUB →OQ→AN→TU. Methylation profiles were assayed using Illumina’s Infinium Methylation EPIC v1.0 BeadChip. Differential methylation analysis was conducted in two sets of four pairwise comparisons: (1) comparing tumor samples to the four benign tissue categories, and (2) comparing tissue categories that are adjacent along the tumor proximity axis. The differentially methylated CpGs were analyzed for enrichment of transcription factor binding sites (TFBS) and genes. Results: Following data processing and quality control, there were 72 TU, 63 AN, 70 OQ, 72 CUB, and 182 CFN samples for analysis. Differential methylation analysis showed that TU samples had distinct methylation profiles, with more hypomethylation events compared to hypermethylation events relative to benign tissues. Case-benign tissues (AN, OQ, CUB) exhibited similar methylation profiles, distinct from CFN. Transcription factor binding site (TFBS) enrichment analysis revealed that case-benign samples, even those distant from tumor, i.e. CUB, showed breast cancer-related methylation changes. Hypomethylated sites in CUB compared to CFN were enriched for TF binding sites TP63, GATA3, ESR1, PR, AR, NR3C1, and GREB1. TU hypermethylation events were enriched for Polycomb-repressive complex 2 (PRC2) binding, including EZH2, SUZ12, and JARID2. ER+ and ER- tumors had distinct methylation profiles. Both ER+ and ER- tumors showed hypermethylation enrichment for PRC2-related binding motifs. However, hypomethylation events differed, with ER+ tumors enriched for hormone receptor-related pathways and ER- tumors enriched for hematopoiesis/immune-related pathways. We did not find any differential methylation between benign tissues from patients with ER+ vs. ER- tumors. Conclusions: DNA methylation changes profoundly at two points on the tumor proximity axis: CFN to CUB and AN to TU. Specifically, all case-benign tissues, including CUB, exhibited changes associated with breast cancer when compared to CFN. Although the methylation profiles of ER+ and ER- tumors differed, benign tissues showed no differences as a function of ER status of the tumor. These findings demonstrate that benign case tissue is not epigenetically “normal” and provide insights for further investigation into the process of tumorigenesis.
 Citation Format: Saya Dennis, Takahiro Tsukioki, Gannon Cottone, Wanding Zhou, Yuan Luo, Patricia A. Ganz, Mary E. Sehl, Seema Khan, Susan Clare. DNA methylation patterns are similar in benign tissue from ipsilateral and contralateral breast while different from matched breast cancer, and healthy controls [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr RF1-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17e532f44a3009f75c42b2bf7806776eed3af9ac" target='_blank'>
              Abstract RF1-05: DNA methylation patterns are similar in benign tissue from ipsilateral and contralateral breast while different from matched breast cancer, and healthy controls
              </a>
            </td>
          <td>
            Saya R. Dennis, Takahiro Tsukioki, Gannon Cottone, Wanding Zhou, Yuan Luo, Patricia A. Ganz, M. Sehl, Seema A. Khan, Susan E Clare
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The TP53 gene encodes the critical tumor suppressor p53, which is involved in the regulation of cell cycle control, apoptosis, and genomic stability. In cancer biology, p53 holds an elevated position because high mutation rates in the TP53 gene across human cancers lead to abnormal p53 protein. This would allow limitless replication of genetically damaged cells with malignant potential and so facilitate tumorigenesis. Normally referred to as the “guardian of the genome,” this protein assumes a central role in different cellular functions spanning metabolism through cell cycle checkpoints and apoptosis to DNA repair mechanisms. Its significance extends beyond tumor suppression because it controls metabolic pathways that are central to maintaining homeostasis under normal physiological conditions. The normal functions of p53 as a tumor suppressor are abrogated by mutations; indeed, many such mutant forms act as oncogenes themselves when expressed in cancer cells. The review paper presents information from recent scientific investigations on the chemical structure of p53 protein and its mutations with a discussion on implications for cancer treatment. It also integrates various research findings to clarify the clinical effects of the p53 gene, especially in breast cancer and myelodysplastic syndromes (MDS), and looks into changing treatment plans that focus on p53 problems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c0adda59bbabd91047006215c78f72a09302aad" target='_blank'>
              Structure, Function and Clinical Significance of P53 Protein: A Review Article
              </a>
            </td>
          <td>
            Farah mumtaz, Hadeel Nasser, Ruqaya Yahya Abd AL-Shaheed, Ali A. Al-fahham
          </td>
          <td>2025-05-28</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 11550


 Background:
 The development of resistance is the one of the reason for ineffective treatment of soft tissue sarcomas (STSs). The search for new molecular-biological targets, both for the personalized selection of novel targeted drugs and for predicting chemoresistance of STSs to individual antitumor agents and their combinations, is a pressing task.
 Methods:
 In this study, an experimental in vitro chemoresistance analysis, whole-genome sequencing, and single-cell sequencing of tumors in patients (N = 9) with undifferentiated pleomorphic sarcomas were performed. Chemotherapy resistance indices were determined for the obtained samples.
 Results:
 Whole-genome DNA sequencing yielded data on the type and frequency of somatic mutations in undifferentiated pleomorphic sarcomas, and driver genes of carcinogenesis were identified, among which the most frequently occurring were FCGBP, PARP4, TP53, RGPD3, PDE4DIP, and RB1. Correlations between the presence of genetic alterations and the response to chemotherapy in the in vitro test were sought. Thus, in samples resistant to the combination of doxorubicin and ifosfamide, the presence of mutations in the PEG3, USP8, NT5C3A, and WAS genes was characteristic. Bioinformatic analysis of single-cell transcriptome sequencing data, aimed at characterizing the population composition and transcriptomic landscape of undifferentiated pleomorphic sarcoma cells, revealed 15 normal and 8 tumor cell populations. Among the populations of normal cells, the following clusters were identified: M2 and M1 macrophages, T-cells, matrix-remodeling fibroblasts and myofibroblasts, monocytes, stromal cells, endothelial cells, and osteoblasts. Among tumor cells, clusters of proliferating cells, hypoxic cells, matrix-interacting cells, and tumor cells with a stem-like phenotype were identified. The most common clusters in all samples were M2 macrophages, endothelial cells, and one of the clusters of tumor cells interacting with the matrix. Comparison of the expression profiles of cells from patients with different responses to chemotherapy allowed identification and characterization of clusters associated with chemosensitivity and chemoresistance. These included clusters of macrophages, tumor cells, fibroblasts, and endothelial cells. In addition, genes common to all clusters associated with resistant samples were identified (ARGLU1, JUND, TNNT3, RHOB, CCNL2, LENG8, LUC7L3, KLF9, RSRP1, RNF213, SERPINE1, CDK5RAP3).
 Conclusions:
 The data obtained in this study will expand the understanding of the pathogenesis of undifferentiated pleomorphic sarcomas and the mechanisms underlying their development of chemotherapy resistance. In the future, this will serve as the basis for creating a test system to evaluate the expression levels of genes whose activation/repression is associated with the development of drug resistance in STSs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9452b21e788959623418046da69bbf46291bf598" target='_blank'>
              Molecular-genetic characteristics of soft tissue sarcomas associated with the development of chemotherapy resistance.
              </a>
            </td>
          <td>
            A. Tararykova, T. Fetisov, P. Shtompel, S. Khazanova, E. Trapeznikova, Maxim Menyailo, A. Ikonnikov, Elena E Kopantseva, A. Korobeynikova, B. Bokhyan, M. Yakubovskaya, K. Kirsanov
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Significance Here, we address a major limitation in the effectiveness of poly(ADP-ribose) polymerase inhibitors (PARPi) in breast cancer gene (BRCA)-mutant prostate cancer treatment: Only ~50% of patients respond despite clear genomic evidence of defective homologous recombination. Prior efforts to study PARPi resistance in prostate cancer have been plagued by the lack of suitable cell lines. We overcame this challenge using primary prostate organoids coupled with genome-wide CRISPR screening. The key finding is that loss-of-function mutations in the DNA prereplication complex (pre-RC) confer PARPi resistance. These genes map to chromosomal regions frequently lost in prostate cancer and could therefore serve as potential biomarkers of treatment response. Pharmacologic inhibition of geminin, a negative regulator of the pre-RC, can restore PARPi sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e95a3ba5883a1720490b54e4008aa7d7b0986c0b" target='_blank'>
              BRCA2 reversion mutation–independent resistance to PARP inhibition through impaired DNA prereplication complex function
              </a>
            </td>
          <td>
            Kyrie Pappas, Matteo Ferrari, Perianne Smith, S. Nandakumar, Zahra Khan, Serina B. Young, Justin Laclair, M. V. Russo, Emmet Huang-Hobbs, Nikolaus Schultz, W. Abida, Wouter Karthaus, M. Jasin, Charles L Sawyers
          </td>
          <td>2025-06-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="
 e17631


 Background:
 Poly (ADP-ribose) polymerase inhibitors (PARPi) has been studied to show positive efficacy in endometrial cancer (EC). However, rational candidates for precision treatment with PARPi need to be further explored. Recently, it has been reported that tumors with homologous recombination deficiency (HRD) are sensitive to PARPi. Here, we aimed to assess the relationship between HRD and molecular subgroups in EC.
 Methods:
 HRD statues and gene characteristics of 372 EC patients were retrospectively collected from August, 2022 to August, 2024. HRD score was calculated as the sum of loss of heterozygosity (LOH), telomeric allelic imbalance (TAI), and large-scale state transitions (LST) scores based on the next-generation sequencing (NGS) data.
 Results:
 Of the EC that underwent NGS testing, there were 5.38% (20/372) patients with
 BRCA1/2
 mutations, and the top two
 BRCA1/2
 mutated subgroups were microsatellite instability-high (MSI-H) (2.96%) and copy number low (CN-L) (1.88%) subgroups. However, there were 10.75% (40/372) patients with positive HRD, which is largely restricted to copy number high (CN-H) (4.57%), CN-L (2.96%) and MSI-H (2.96%) subgroup. Interestingly, based on HRD score with the cutoff of 30, there were mostly CN-H and CN-L subgroups and almost no
 POLE
 mutation and MSI-H subgroups in non
 BRCA1/2
 mutation and positive HRD patients.
 Conclusions:
 BRCA1/2
 mutations and positive HRD were seen in 5.38% and 10.75% EC patients, respectively. HRD score testing, based on LOH, TAI, and LST, may help select patients benefiting from PARPi with particular reference to CN-H and CN-L subgroups.


 Prevalence of HRD statues in different molecular subgroups of EC.





 HRD
 Non-HRD




 Molecular Subgroups
 All

 BRCA1/2
 mutation

 HRD score≥30




 POLE
 mut

 1(0.27%)
 1(0.27%)
 0(0%)
 8(2.15%)


 MSI-H
 11(2.96%)
 11(2.96%)
 0(0%)
 64(17.20%)


 CN-H
 17(4.57%)
 1(0.27%)
 16(4.30%)
 64(17.20%)


 CN-L
 11(2.96%)
 7(1.88%)
 4(1.08%)
 196(52.69%)






 HRD: Homologous recombination deficiency; EC: endometrial carcinomas;
 POLE
 mut:
 POLE
 mutations; MSI-H: microsatellite instability-high; CN-L: copy number low; CN-H: copy number high.




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66ff21a16e023a00431cd6937437785071b41ecc" target='_blank'>
              Prevalence of homologous recombination deficiency in different molecular subgroups of endometrial carcinomas.
              </a>
            </td>
          <td>
            Y. Rao, Wenwen Zhang, Na Zhang, Han Han, Yong Xiao, Ping Liu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor development is mainly marked by the gradual transformation of cells that acquire capacities such as sustained growth signaling, evasion of growth suppression, resistance to cell death, and induction of angiogenesis, achieving replicative immortality and activating invasion and metastasis. How different epigenetic alterations like m1A, m5C, and m6A contribute to tumor development is a field that still needs to be investigated. The immune modulators, CD70, CD80, and TIGIT, mainly regulate T-cell activation and consequently the immune evasion of tumors. Here, we explored the presence and the potential consequences of RNA modifications in these regulators in pan-cancer. Our findings highlight the critical role of the m6A, m5C, and m1A in regulating CD70, CD80, and TIGIT across multiple solid tumors. By combining epitranscriptomics data with functional enrichment and survival modeling, we show that RNA modification enzymes not only modulate immune-related gene expression but also serve as potential biomarkers for patient prognosis. By constructing a robust four-gene prognostic signature involving YTHDF3, RBM15B, IGF2BP2, and TRMT61A, we demonstrate that RNA modification profiles can accurately stratify patients into risk groups with distinct overall survival outcomes. The performance of this model across eight cancer types underscores the translational promise of epitranscriptomic markers in both mechanistic understanding and personalized oncology. Altogether, our study bridges the gap between the mechanistic regulation of immune checkpoints and their clinical utility, offering novel insights into how the epitranscriptome can be leveraged to improve cancer prognosis and potentially enhance immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d03e20bb526dd9e6903a8377f001f578b9d1d1d" target='_blank'>
              Epitranscriptomics Regulation of CD70, CD80, and TIGIT in Cancer Immunity
              </a>
            </td>
          <td>
            C. Rigopoulos, Marios Gkoris, Ilias Georgakopoulos-Soares, I. Boulalas, A. Zaravinos
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Canine lymphoma, a phenotypically and genetically heterogeneous disease, represents a significant proportion of canine cancers. We present a large-scale study of 238 dogs with lymphoma to better understand the genetic landscape of canine lymphoma, as well as the relationship to clinical outcomes. Using a targeted next-generation sequencing panel comprising 308 genes, we screened somatic and germline mutations in matched tumor and normal samples. Our findings revealed key associations between genetic alterations and lymphoma subtypes, with certain somatic variants linked to significant differences in response to common chemotherapy regimens. Recurrent mutations in genes such as KMT2C, KMT2D, NOTCH2, TRAF3, CCND1, ARID1A, CREBBP, and TP53 were observed, with TRAF3 mutations standing out for their significant association with prolonged progression-free survival and overall survival in B-cell lymphomas. In contrast, mutations in PIK3CD and CREBBP were associated with inferior outcomes in T-cell lymphomas, highlighting the immunophenotype-specific impact of genetic alterations on treatment responses. These findings support the integration of comprehensive genomic profiling in planning treatment strategies and optimizing clinical outcomes in canine lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e07f223413194847051e584c8f37b09d9f47c90d" target='_blank'>
              Identification of novel genetic mutations for the treatment prognostication of canine lymphoma
              </a>
            </td>
          <td>
            Josephine Tsang, Qi-Jing Yap, Sheena Kapoor, Jerry Cromarty, Sushmita Sen, Minji Kim, George Courcoubetis, Suhyeon Cho, Deanna Swartzfager, Stanley Park, Sungwon Lim, Ilona Holcomb, Jamin Koo
          </td>
          <td>2025-06-12</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction: Colorectal cancer (CRC) is a heterogeneous and multifactorial malignancy driven by a series of genetic and epigenetic alterations. In this field, telomere/telomerase dysfunction contributes to CRC carcinogenesis by impairing genomic stability and cellular replication. Objective: This study aimed to evaluate genetic and epigenetic alterations in CRC by examining mutation rates in the human telomerase reverse transcriptase (hTERT) promoter region, relative telomere length (RTL), hTERT gene expression, and DNA methylation in the TERT hypermethylated oncological region (THOR). Methods: A total of 45 CRC and 34 adjacent normal tissue samples from Moroccan patients were analyzed using molecular approaches, such as Sanger sequencing, quantitative PCR (qPCR), reverse transcription qPCR (RT-qPCR), and methylation-specific PCR (MSP). Results: No mutations in the hTERT promoter region were identified. However, hypermethylation in the THOR region was reported in 82.2% of CRC samples and 79.4% of adjacent normal tissues. High hTERT expression was detected in 50% of CRC patients. In addition, telomere length was significantly shorter (p=0.002) in cancerous tissues (1.41 [1.36 – 1.43]) compared to normal mucosa (1.559 [1.46 – 1.63]), with an RTL ratio less than 1 (0.90 [0.86 – 0.95]). No significant differences were found between clinicopathological features and hTERT expression, THOR methylation, or RTL, except for a significant correlation between THOR hypermethylation and smaller tumor size (p=0.017) and between THOR methylation and RTL in CRC tissues (p=0.034). Conclusion: These results suggest that telomere lengthening is crucial for CRC initiation and progression, and cancer cells tend to shorten telomeres to maintain the chromosomal instability (CIN) required for tumor progression. Further research is needed to elucidate the mechanisms underlying telomere shortening in CRC and understand the role of telomerase/telomere complex in CRC initiation and progression, which could provide new diagnostic, prognostic, and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e32cd6246bc493b71aa8671eea1af0f3ca4e1f7" target='_blank'>
              Exploring genetic and epigenetic alterations in hTERT gene in colorectal cancer
              </a>
            </td>
          <td>
            Meryem Jafari, Boutaina Addoum, A. Laraqui, Mohammed Oukabli, Ahmed Bounaim, Abdelmounim Ait Ali, Sidi Mohammed Bouchantouf, Noureddine Njoumi, Belkouchi Abdelkader, Fatima Zohra BenMoula, Hicham El Annaz, Khalid Ennibi, I. Chaoui, Youssef Bakri, Mohammed Attaleb
          </td>
          <td>2025-06-03</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Introduction Anoikis is a distinct form of programmed cell death, differing from classical apoptosis, and its role in malignant tumor progression, particularly in hepatocellular carcinoma (HCC), remains insufficiently understood. This study aims to elucidate the prognostic significance and therapeutic relevance of anoikis-related genes (ARGs) in HCC. Methods We systematically analyzed the expression, mutation, and copy number variation profiles of 27 known ARGs in HCC using public datasets. Unsupervised consensus clustering was performed to classify patients into anoikis subtypes. Weighted Gene Co-expression Network Analysis (WGCNA) identified hub gene modules, and LASSO Cox regression was applied to construct a prognostic risk score model. Correlations between the risk model and clinical outcomes, tumor microenvironment (TME) characteristics, and immunotherapy responses were evaluated. Single-cell RNA-seq and pan-cancer analyses were conducted to explore gene expression across cell types and cancer types. Finally, in vitro experiments were performed to validate the biological function of model genes. Results Two distinct anoikis subtypes with differing prognoses and TME features were identified in HCC. A two-gene prognostic model (TTC26 and TPX2) was developed, demonstrating robust performance in predicting patient outcomes. High-risk patients exhibited lower overall survival and distinct immune infiltration profiles. Pan-cancer analysis showed widespread dysregulation of TTC26 and TPX2. In vitro experiments confirmed that TTC26 promotes HCC cell proliferation, migration, and invasion. Discussion Our findings reveal that anoikis-related molecular classification is closely linked to HCC prognosis and immune landscape. The established prognostic model has potential clinical utility for risk stratification and treatment guidance. TTC26 may serve as a novel biomarker and therapeutic target in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31b65e8bef2ebab0cf113a808820c5250240ebd" target='_blank'>
              Identification of anoikis-related subtypes and a risk score prognosis model, the association with TME landscapes and therapeutic responses in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Xiangyu Zhai, Kecheng Li, Hailing Ding, Yanmei Wu, Xinlu Zhang, Hao Zhang, Huaxin Zhou, Chongzhong Liu, Zili Zhang, Bin Jin
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a889f8e2ddcb6a4b7bab00376e74479c448ebb" target='_blank'>
              Structural basis of Fanconi anemia pathway activation by FANCM.
              </a>
            </td>
          <td>
            R. Bythell-Douglas, Sylvie van Twest, Lara Abbouche, Elyse Dunn, Rachel J Coulthard, D. Briggs, Vincent J. Murphy, Xinxin Zhang, Winnie Tan, Sarah S Henrikus, Dongming Qian, Yin Wu, Jana Wolf, Laurent Rigoreau, S. Shakeel, Kathryn L Chapman, N. Q. McDonald, A. Deans
          </td>
          <td>2025-05-30</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f99ef5f46003666cce9bfbffdcaf4920a36fe552" target='_blank'>
              RNA G-quadruplexes: emerging regulators of gene expression and therapeutic targets.
              </a>
            </td>
          <td>
            Zukela Ruzi, Wei Zha, Huang Yuan Yuan, Jiaorui Liu
          </td>
          <td>2025-07-03</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the leading causes of death in women worldwide. Modern molecular medicine includes techniques like deoxyribonucleic acids (DNA) microarray and polymerase chain reaction (PCR); however, these techniques require specialized and expensive equipment to determine DNA concentration and purity. In breast cancer (BC), tumor growth promotes rapid cell replication, leading to errors and instability in DNA molecules. As a result, total DNA may exhibit differential degradation and changes in its electrical properties. DNA molecules possess an electronegative charge and implicit viscosity, so the number of double-helix structures and their electrical and physical properties are expected to alter volumetric bioimpedance based on DNA concentration and purity. In this study, a biosensor of BC DNA concentration and purity based on Electrical Bioimpedance Spectroscopy (EBiS) is proposed. Twenty DNA samples extracted from BC biopsies were quantified in concentration and purity by UV-Vis spectrophotometry and then evaluated through EBiS measurements from 100 Hz to 10 MHz Spearman correlation coefficients for EBiS measurements in the entire frequency range with respect to DNA concentration and purity were estimated through SPSS statistics software (Ver 21.0). The findings show impedance magnitude Z with a significant statistical correlation to DNA concentration and purity at 9.12 and 5.75 MHz, respectively (R 9.12 MHz = 0.936, R 5.75 MHz = 0.56, p < 0.05). The findings indicate the use of a biosensor based on EBiS measurement is a feasible technique to estimate BC DNA concentrations and purity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd040d5e3b25bafc8991d72e77f8f3858f9e013f" target='_blank'>
              Correlation of EBiS vs. Concentration and Purity of Total DNA in Breast Cancer Samples
              </a>
            </td>
          <td>
            Leticia Arias-González, J. Cruz-Ramos, N. Pérez-Vielma, Jacobo E. Mungia-Cervantes, Enrique Hernández-Sánchez, Marlon Rojas-López, Modesto Gomez-López
          </td>
          <td>2025-05-01</td>
          <td>Journal of Physics: Conference Series</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Two epigenetically labile subsets of genes that link embryonic environmental exposures with adult disease susceptibility are those that are imprinted and those with metastable epialleles. The expression of genes with metastable epialleles, like the agouti gene in Agouti viable yellow (Avy) mice, is highly variable between individuals but uniform in tissues within an individual. We used the Avy mouse to demonstrate that exposure to nutritional supplements, chemical toxicants, and low-dose ionizing radiation during embryogenesis alters adult disease susceptibility by modifying the epigenome. Genomic imprinting is a unique species-dependent epigenetic form of gene regulation that evolved approximately 150 million years ago in a common ancestor to Therian mammals. It resulted in monoallelic parent-of-origin-dependent gene silencing. Thus, imprinted genes are functionally haploid disease susceptibility loci, since only a single genetic or epigenetic event is required to alter their function. Expression of imprinted genes in the human genome is regulated by hemi-methylated imprint control regions (ICRs) in the human imprintome. Furthermore, human imprintome ICRs associated with chronic diseases (e.g., cancer, diabetes, and obesity) and behavioral disorders (e.g., autism, bipolar disorder, psychopathy, and schizophrenia) can now be identified with the use of cells from peripheral samples and the human imprintome array. The importance of metastable epialleles and imprinted genes in the etiology of environmentally induced human chronic diseases is discussed in this review.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960e9fe6811efda91a98ebc102150c171e03b920" target='_blank'>
              Epigenetics, human imprintome, and chronic diseases.
              </a>
            </td>
          <td>
            Randy L Jirtle
          </td>
          <td>2025-07-04</td>
          <td>Essays in biochemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Hepatocellular carcinoma (HCC) is a highly heterogeneous malignant tumor. Among the various epigenetic alterations involved in its pathogenesis, the N6-methyladenosine (m6A) modification of eukaryotic mRNA plays a pivotal role in tumor biology and holds significant potential for diagnostic and therapeutic applications. In this study, we identified and validated an 8-gene risk score that robustly predicts prognosis in HCC patients, achieving area under the ROC curve (AUC) values greater than 0.75 across both internal and external cohorts. A high-risk score was significantly associated with poorer clinical outcomes and reduced immune tolerance, while also indicating a potentially enhanced response to immunotherapy. Single-cell RNA sequencing revealed a higher proportion of T cells and a decreased presence of immunosuppressive T cells in the high-risk group, which may account for their improved responsiveness to immune checkpoint inhibitors. Finally, in vitro experiments and immunohistochemical staining showed the biological function of the key gene ANLN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea94ab482ecd557d67969f8038f92a2cc7e807c" target='_blank'>
              Stratification of Hepatocellular Carcinoma Using N6-Methyladenosine
              </a>
            </td>
          <td>
            Nan Wang, Jia-Xin Shi, M. Bartneck, E. Dahl, Junqing Wang
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 8602


 Background:
 MTAP
 loss leads to methylthioadenosine (MTA) accumulation, disrupting downstream metabolic pathways. Targeting synthetic lethal interactions with
 MTAP
 loss offers a promising therapeutic strategy. The DNA damage response (DDR) pathway, essential for genomic stability, is commonly dysregulated in lung cancer, impacting treatment response and prognosis. Emerging evidence has highlighted potential association between
 MTAP
 loss and mutations in DDR genes, suggesting that investigating their relationship is critical for uncovering disease mechanisms and identifying novel biomarkers and therapeutic targets.
 Methods:
 Hybridization capture sequencing with StarPanel NGS Assay (1,326 genes) was conducted on 2,258 Chinese lung cancer patients' tumor and matched peripheral blood samples. Then somatic and germline mutations, and TMB values of each sample were obtained. Based on a depth-based algorithm, differences between tissue and control samples in CNVs from the gene level were analyzed. Besides, we utilized shifts in germline heterozygous SNPs within the gene region to assist in determining if the loss was homozygous or heterozygous.
 Results:
 Homozygous and heterozygous deletions of
 MTAP
 were detected in 11.07% and 6.95% of all samples. Notably, 61.67% of these samples exhibited co-loss of
 MTAP
 ,
 CDKN2A
 and
 CDKN2B
 . The median TMB was significantly higher in samples with
 MTAP
 loss (3.85 mut/Mb) compared to those with intact
 MTAP
 (2.56 mut/Mb). In samples with homozygous
 MTAP
 loss, the top somatically co-altered genes were
 EGFR
 (76.00%),
 TP53
 (57.60%) and
 CDKN2A
 (52.80%). Gain-of-function (GoF) mutations were most prevalent in
 EGFR
 (56.80%),
 KRAS
 (12.00%) and
 MDM2
 (10.00%), while Loss-of-function (LoF) mutations were most common in
 TP53
 (50.40%),
 RBM10
 (11.60%) and
 PTEN
 (7.60%). Notably, LoF mutations in
 MTAP
 -loss samples showed a higher prevalence of DDR genes compared to
 MTAP
 -intact samples, including
 RAD50
 (2.40% vs. 0.00%,
 p
 < .0001) and
 POLQ
 (2.40% vs. 0.00%,
 p
 < .0001). Enrichment analysis further revealed LoF mutations unique to
 MTAP
 -loss samples were significantly enriched in the DDR pathway (
 p
 < .0001). For top germline pathogenic mutations, two DDR genes,
 RECQL4
 (0.80% vs. 0.38%,
 p
 = 0.29) and
 BRCA2
 (0.40% vs. 0.43%,
 p
 = 1), showed no significant differences between
 MTAP
 -loss and
 MTAP
 -intact samples.
 Conclusions:
 Potential association exists between
 MTAP
 loss and DDR pathway dysregulation, which may impact tumorigenic processes and therapeutic vulnerabilities. The enrichment of DDR-related LoF mutations indicates
 MTAP
 loss could exacerbate genomic instability by impairing DNA damage repair mechanisms, thereby increasing TMB and driving cancer progression with a distinct molecular profile. These vulnerabilities are primarily driven by somatic mutations, providing a rationale for exploring personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/305ad8ea528f00d8a04f7506987172e9d5a5207d" target='_blank'>
              Dysregulation of DNA damage repair in lung cancer driven by
 MTAP
 loss: Mechanistic insights and target discovery.
              </a>
            </td>
          <td>
            Bo Jiang, W. Liao, Xin Wang, Hongliang Hui, Yan-Gui Lin, Linjie Qiu, Dan Li, Min Luo, Haoran Miao, Fan Qiu, Yiqian Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 2633


 Background:
 HER2/neu mutations or amplifications, present in up to 23% of non-small cell lung cancers (NSCLCs), activate STAT3, promoting inflammation and suppressing adaptive immune responses. Elevated systemic inflammation-immune index (SII) in small cell lung cancer (SCLC) correlates with reduced response to PD-1/L1 immune checkpoint inhibitors (ICIs), leading to worse overall survival (OS) and progression-free survival (PFS). These findings highlight the need to mitigate inflammation in lung cancers to enhance ICI response and improve survival. Biomarker studies can identify genes that differentiate inflammatory and immune pathways, offering therapeutic insights.
 Methods:
 To identify genes regulating the HER2/neu oncogene, HER2/neu-overexpressing mice were crossed with genetically Diverse Outbred (DO) mice. The resulting DO F1 offspring were monitored for HER2/neu-expressing tumor onset and growth. Tumor phenotypes were associated with mouse haplotypes to identify quantitative trait loci (QTL) on chromosomes (chr) 2, 6, and X linked with aggressive HER2/neu tumors. Of the 35 candidate genes harboring protein-coding changes, homologous genes on human chr 1, 10, and X were analyzed using Caris Life Sciences datasets of NSCLC and SCLC: 25,143 primary NSCLC; 12,365 metastatic NSCLC; 621 primary SCLC; and 971 metastatic SCLC where primary biopsies are those taken from the lung and metastatic biopsies from sites other than the lung. Patient cohorts stratified by gene expression (top vs. bottom 50%) were correlated with OS, ICI response, and time on pembrolizumab or atezolizumab treatment (TOT).
 Results:
 Several homologous candidate genes correlated with lung cancer survival in both NSCLC and SCLC. Notably, high expression of
 LILRB4
 , a macrophage-specific checkpoint molecule, was associated with improved survival (HR 0.66–0.84,
 p
 < 0.001) across all lung cancer types and sites. Additionally, elevated
 TSC22D3
 , a glucocorticoid receptor-activated gene regulating anti-inflammatory pathways, including
 LILRB4
 expression, was positively associated with OS in metastatic NSCLC (HR 0.82,
 p
 < 0.00001) and SCLC (HR 0.77,
 p
 < 0.001). Expression of
 LILRB4
 and
 TSC22D3
 further enhanced survival in ICI-treated patients (SCLC: HR 0.72; NSCLC: HR 0.82;
 p
 < 0.001). In NSCLC, high
 LILRB4
 expression conferred an 18% improved TOT with pembrolizumab (HR 0.858,
 p
 < 0.0001).
 Conclusions:
 LILRB4
 and
 TSC22D3
 are key genes linked to improved survival outcomes in both SCLC and NSCLC, likely through their roles in mitigating inflammation. Their association with enhanced ICI responses underscores their potential as therapeutic targets. Future research will evaluate the relationship between
 LILRB4
 and
 TSC22D3
 expression and HER2/neu status in NSCLC and validate protein-level correlations in positive cells. Our long-term goal is to determine the therapeutic potential of
 LILRB4
 and
 TSC22D3
 in SCLC, NSCLC and HER2/neu-positive NSCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0e9d9066dedda6abe73bbff9473b3f5d6c28e8" target='_blank'>
              Effect of elevated expression of
 LILRB4
 and
 TSC22D3
 on survival in lung cancer.
              </a>
            </td>
          <td>
            B. Hrinczenko, T. Adeyelu, Wei-Zen Wei, A. Elliott, G. Bepler, A. Vanderwalde, B. Halmos, E. Antonarakis, Maryam B Lustberg, Jennifer Jacob
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background Oral squamous cell carcinoma (OSCC) is associated with poor prognosis due to extensive local invasiveness and lymph node metastasis, often leading to a significant decrease in aesthetics and function after surgery. Therefore, elucidation of the molecular mechanisms underlying OSCC is necessary for its early detection and treatment. N6-methyladenosine (m6A) modification of mRNA is the most common form of post-transcriptional RNA methylation and is often involved in the progression of cancer by regulating the expression of various genes. Recent studies reported the tumor-promoting effects of vir-like m6A methyltransferase associated (VIRMA, also termed KIAA1429), a novel molecule involved in m6A modification; however, its role in OSCC remains poorly understood. Methods In the present study, we determined the total m6A levels and VIRMA expression in OSCC using immunohistochemistry of tissue specimens and evaluated their association with clinicopathologic characteristics. We also performed gene expression analysis of VIRMA/KIAA1429 using public datasets. Results We found that the m6A levels were significantly higher in the OSCC specimens of patients with a more advanced clinical stage (P = 0.0063), lymph node metastasis (P = 0.0323), and venous invasion (P = 0.0380) compared to those without. The analysis of the public datasets revealed that VIRMA/KIAA1429 expression levels were higher in head and neck SCC than in normal mucosa, whereas immunohistochemistry revealed that VIRMA-expressing OSCC was associated with a significantly shorter disease-free survival (P = 0.0043) and was an independent poor prognostic factor (P = 0.0179). Conclusions These results highlight the potential utility of RNA methylation and VIRMA expression for the diagnosis and treatment of OSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4045080f537d11c412b6f7373ebe4ef6923638e2" target='_blank'>
              Total m6A RNA levels and VIRMA expression as potential diagnostic and prognostic markers in oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Kaori Shima, Yudai Shimojukkoku, Yasunobu Oku, Kanako Higashimoto, Takahiro Tsuchiyama, Yuka Kajiya, Miyako Kurihara-Shimomura, Tomonori Sasahira
          </td>
          <td>2025-07-03</td>
          <td>Diagnostic Pathology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 e20051


 Background:
 Histone lactylation, a newly identified epigenetic modification, has been extensively implicated in the initiation and progression of tumors. However, the precise role of histone lactylation in modulating tumor cell responses to T cell-mediated cytotoxicity remains insufficiently explored.
 Methods:
 To elucidate the adaptive changes tumor cells undergo in response to T cell-induced cytotoxicity within the tumor microenvironment (TME), we performed a comprehensive, genome-wide CRISPR-Cas9 screen. This approach facilitated the identification of genetic alterations in lung cancer cells co-cultured with cytotoxic T lymphocytes (CTLs) that promote their evasion of T cell-mediated destruction. Complementary RNA sequencing (RNA-seq) and chromatin immunoprecipitation sequencing (ChIP-seq) analyses were employed to investigate the molecular mechanisms by which tumor cells modulate their response to T cell-mediated killing, particularly following the deletion of the monocarboxylate transporter 1 (MCT1).
 Results:
 MCT1 deletion significantly heightened the cancer-cell susceptibility to CTL-mediated killing. Furthermore, the loss of MCT1 led to mitochondrial dysfunction within the tumor cells. Mechanistically, MCT1 knockout resulted in a reduction of histone H3K18 lactylation and downregulation of ferridoxin reductase (FDXR), which disrupted mitochondrial homeostasis and precipitated mitochondrial dysfunction. Notably, the combination of MCT1 inhibition with immune checkpoint blockade (ICB) therapy substantially enhanced ICB efficacy, prolonged survival in both lung cancer and malignant pleural effusion (MPE) murine models, and alleviated the immunosuppressive milieu within the TME.
 Conclusions:
 Our findings reveal a novel role for lactate metabolism in modulating mitochondrial dysfunction and anti-tumor immunity, with lactylation-mediated regulation of mitochondrial homeostasis serving as a critical mechanism. These results provide new insights into the epigenetic regulation of tumorigenesis and highlight potential immune-metabolic strategies for targeted lactate metabolism therapies in the treatment of lung cancer and MPE.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7369949ad290ca8642cb013b1fc65d9e0567a5" target='_blank'>
              MCT1-mediated histone lactylation modulates lung cancer cell sensitivity to T-cell–mediated killing by influencing mitochondrial dysfunction.
              </a>
            </td>
          <td>
            Q. Xue, W. Peng, Qiong Zhou
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="SET domain bifurcated histone methyltransferase 1 (SETDB1) is a histone H3 lysine 9 (H3K9) methyltransferase, functioning in transcriptional silencing of the transposable elements (TEs), endogenous retroviruses (ERVs), interferon-stimulated genes (ISGs), and immune cell-related molecules. SETDB1 collaborates with cofactors such as ATF7IP and TRIM28 and functions in concert with DNA methylation to maintain gene repression in both stem cells and differentiated somatic cells. Given its gene targets, recent studies have shown that SETDB1 is a critical immune checkpoint gene in both solid and hematological tumors. In this review, we first discuss the role of SETDB1 in gene regulation through its histone methyltransferase activity, including an overview of its structural features and key cofactors. We then highlight the lineage-specific roles of SETDB1 in both normal hematopoietic processes and hematological malignancies, emphasizing its function as an immune checkpoint molecule that suppresses natural killer (NK) cell-mediated antileukemia responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b622142a48ee47c5894f0bcff4fb87ea6d35f6c" target='_blank'>
              The Role of Histone Methyltransferase SETDB1 in Normal and Malignant Hematopoiesis.
              </a>
            </td>
          <td>
            Yu-Hsuan Chang, S. Goyama
          </td>
          <td>2025-06-04</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Introduction: BRCA1 promoter hypermethylation (BRCA1meth), which is a driver of homologous recombination deficiency (HRD), is observed in up to 25% of triple negative breast cancers (TNBCs) and ovarian carcinomas (OvCas). BRCA1meth is associated with worse outcome compared to BRCA1 mutations (Menghi et al., PMID: 35857626), and, when found in individuals without cancer (i.e. constitutional BRCA1meth), it is associated with increased risk of breast and ovarian cancer (Lonning, et al., PMID: 36074460). Current methodologies to determine the presence of BRCA1meth in a cancer sample require tissue biopsy and have not been adopted for standard clinical assessment. The GRAIL cell-free DNA (cfDNA)-based targeted methylation platform is a robust, biopsy-free, scalable assay that was optimized to distinguish cancer methylation patterns between different cancer signal origins. Here, we describe the GRAIL platform’s ability to detect and quantify BRCA1meth. Methods: As part of the CCGA study (NCT02889978), clinical data were recorded and plasma samples were collected and processed on GRAIL’s targeted methylation platform from 2,958 patients with cancer prior to any treatment, and from 2,790 individuals without cancer. Samples were evaluated from patients with a variety of cancers: breast, ovarian, cervical, uterine, lung, prostate, head and neck, kidney, bladder, stomach, esophageal, colorectal, and hematological malignancies. A dilution series of contrived samples was analyzed to establish analytical characteristics of BRCA1meth detection using the GRAIL assay. Results: When applied to contrived samples, the GRAIL platform was able to detect BRCA1meth at an LoD95 of <1% (as measured by BRCA1meth fraction = methylated molecules / total molecules). When applied to a cohort of 2,790 individuals without cancer, we found 4% of individuals harbored evidence of BRCA1meth in the fractional range of 0.4% to 37%, with the majority of individuals (81%) having <5% BRCA1meth fraction, similar to surveys using peripheral blood cell DNA (Lonning et al., PMID: 36074460). Consistent with recent data of constitutional BRCA1meth prevalence in umbilical cord blood of newborns (Nikolaienko, et al., PMID: 38053165), we see lower incidence of BRCA1meth in males without cancer (3%) compared to females without cancer (5%, p=0.01). BRCA1meth was significantly enriched in the cfDNA of patients with TNBC and OvCa compared to female individuals without cancer (17/112 (15%), adjusted p=0.001 for TNBC; 14/101 (14%), adjusted p=0.01 for OvCa), in concordance with previous assessments of BRCA1meth prevalence across different tumor types (Menghi et al., PMID: 35857626). In TNBC and OvCa, the per-sample BRCA1meth fraction ranged from 0% to 54% and was correlated with the methylation-based estimate of tumor fraction in both TNBC (R=0.68, p=0.0052) and OvCa (R=0.65, p=0.03). High BRCA1meth fractions (up to 16-21%) were observed in four individuals with uterine, colorectal, bladder, or kidney cancer. This suggests that BRCA1 deficiency may be implicated in these rare cases, which would imply HRD, making them candidates for specific treatments. Finally, across tumor types that affect both sexes, we found consistently lower percentages of BRCA1meth in male vs. female patients, reflecting the differential rates of constitutional BRCA1meth across the two sexes. This suggests that the dynamics of BRCA1 promoter methylation and its consequences for the development of cancer may differ by sex. Conclusions: This work shows that the GRAIL platform, as a single blood-based assay, can sensitively detect and quantify BRCA1meth from cfDNA, which may inform cancer susceptibility in non-cancer bearing individuals and may also have potential treatment selection implications for cancer patients.
 Citation Format: Francesca Menghi, Margaret Antonio, William Cance, Tracy Nance, Edison T. Liu. Quantification of Breast Cancer 1 gene (BRCA1) promoter methylation in individuals with and without cancer using cell-free DNA [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-03.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8ede0da8ea6d210ede4f24e81580ed41fdb9cdd" target='_blank'>
              Abstract P2-11-03: Quantification of Breast Cancer 1 gene (BRCA1) promoter methylation in individuals with and without cancer using cell-free DNA
              </a>
            </td>
          <td>
            F. Menghi, Margaret Antonio, William Cance, Tracy Nance, Edison T. Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Here we review the historical background and contemporary insights into genetic dominance, focusing on haploinsufficiency (HI), that is, when the function of only one allele of a gene is not enough to ensure a normal phenotype in a diploid organism. A related phenomenon is triplosensitivity, that is, pathogenic effects when there are three instead of two copies of some 'genes'. The importance of gene dosage issues was realized in humans when whole chromosomal abnormalities (aneuploidy) could be linked to clinical phenotypes such as Down, Edwards, and Patau syndromes. Subsequently, subtler chromosomal deletions and duplications have been shown to be responsible for many developmental syndromes. In several cases, a dosage‐sensitive gene mapping to the relevant regions has been implicated as causal. We delve into the mechanisms of HI, especially due to direct protein insufficiency and subunit imbalances in the context of multi‐subunit complexes. We show how the nonlinearity inherent to the relationship between genotype and phenotype is responsible for the dominance of the underlying genetic variants. We also explore why increased gene dosage can lead to abnormal phenotypes. Examples include trisomy or segmental genomic duplications in humans and oncogene amplification in cancers. Finally, we examine a few cases of genetic synergy, where the combined effect of two or more variants amplifies their individual effects, underlying a distinguishable phenotype. Further research is required to elucidate the dynamics of multicomponent interactions to unravel the mechanistic complexities of genetic dominance, inter‐gene interactions, and their implications for disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f53a61642f9ad382de56a8be421a6a2e4cf23bf4" target='_blank'>
              Gene Dosage Sensitivity and Human Genetic Diseases
              </a>
            </td>
          <td>
            R. Veitia, Johannes Zschocke, James A. Birchler
          </td>
          <td>2025-07-01</td>
          <td>Journal of Inherited Metabolic Disease</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="
 DNA repair processes are the foundation for genome integrity and survival, especially in extreme environments where DNA damage occurs more frequently and where archaea are found. Nevertheless, first-hand experimental information on repair pathways in archaea is scarce, and assignment of repair proteins is currently largely based on homology. We showed previously that DNA lesions induced by CRISPR-Cas self-targeting are repaired by microhomology mediated end joining (MMEJ). To identify proteins involved in the archaeal MMEJ pathway, we used deletion strains devoid of proteins assigned to the key steps of MMEJ, to examine changes in the repair outcome. In addition we used aphidicolin to inhibit the activity of the essential PolB1 protein. For the first time, we were thereby able to experimentally identify proteins involved in this repair pathway in the euryarchaeal model organism Haloferax volcanii. The present study confirms that Mre11, Rad50, Fen1, PolB1, LigA and LigN take part in MMEJ, as previously inferred. In addition, we show that Cas1 and Hel308a are also involved in the MMEJ pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5af42e41866e31a1c7b25ea6eee79e0cec5ab57c" target='_blank'>
              CRISPR-Cas induced self-targeting identifies key players in archaeal microhomology mediated end joining
              </a>
            </td>
          <td>
            A-L. Sailer, J. Wörtz, V. Smith, A. Stachler, F. Blau, M. Daratha, L-K Maier, T. Allers, A. Marchfelder
          </td>
          <td>2025-07-08</td>
          <td>microLife</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Nucleic acid amplification is a pivotal technology for gene investigation in molecular biology as well as an indispensable tool for nucleic acid-based analyses across various disciplines. However, the aerosolized amplicons render the subsequent reaction susceptible to carryover contamination, leading to serious false positive results, particularly in scenarios involving repeated amplification of identical sequences. Although the uracil-DNA glycosylase-based strategy can address this issue for natural DNA amplification, the uracil-containing DNA required for bisulfite conversion-based DNA methylation analysis fails to be analyzed because both aerosol and the intended template can be hydrolyzed. In this study, an innovative strategy for one-pot elimination of carryover contamination is established, applicable to both natural and uracil-containing DNA analyses using either thermal cycle-based or isotherm-based nucleic acid amplification. By incorporation of deoxyinosine triphosphate, hypoxanthine is inserted into amplicons, resulting in a different base composition from the original template. Hypoxanthine, a damaged base as the recognition site of endonuclease V, acts as a label for aerosol cleavage, while the intended template lacking hypoxanthine is unaffected. By systematic optimization, abundant aerosols are thoroughly hydrolyzed within a brief time. Despite the use of deoxyinosine triphosphate as an aberrant deoxyribonucleoside triphosphate, the amplification efficiency, sensitivity, and specificity are insusceptible. In addition, the hypoxanthine-containing amplicons can be analyzed not only for length identification by gel and capillary electrophoresis but also for sequence identification by sanger sequencing, pyrosequencing, and massively parallel sequencing. Moreover, the hypoxanthine-containing amplicons can be further used for cloning and restriction enzyme digestion, indicating the significant potential of this anti-aerosol strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5373512e1ee71db9407b5d72c170aac51f9f8cf9" target='_blank'>
              Elimination of Carryover Contamination during Nucleic Acid Amplification Based on Damaged Base Excision by DNA Repair Enzymes.
              </a>
            </td>
          <td>
            Xiaonan Liu, Wenjing Hu, Huyun Zhou, Jiaxing Zhang, Mengyang Zhao, Yali Cui, Jiangwei Yan
          </td>
          <td>2025-06-03</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the last two decades, significant improvements have been made in the understanding of the genomic and biological bases of acute lymphoblastic leukemia (ALL), resulting in enhanced genomic classification, more precise risk stratification and improved long-term outcomes. ALL is a hematologic malignancy defined by uncontrolled proliferation of immature B- or T- lymphoid blasts in the bone marrow, blood, and other extramedullary tissues. It affects most commonly children, representing the most common childhood cancer, but it also occurs in adults where outcome tends to be poorer compared to pediatric patients. A variety of genetic aberrations, including structural and numerical chromosome alterations, translocations generating fusion oncoproteins, cryptic genomic rearrangements, sequence mutations and genomic copy number changes, define multiple genomic subtypes, influence risk stratification and determine response to therapeutic strategies. In this review, we describe the updated genomic classification of ALL highlighting new biological insights and discussing their implications for prognostication and outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bf200ea2c50518790c91d4e35c6ba5760ccd542" target='_blank'>
              Genomic Subclassification and Risk Stratification of ALL.
              </a>
            </td>
          <td>
            Anna Østergaard, Ilaria Iacobucci
          </td>
          <td>2025-06-30</td>
          <td>Acta haematologica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc4d33cb171c95693c151dadaf1339c2b7a61c7" target='_blank'>
              SCLC-TumorMiner: A Directly Accessible Genomics Resource for Precision Oncology – Big Data for Small Cells
              </a>
            </td>
          <td>
            Fathi Elloumi, A. Dhall, Daiki Taniyama, Augustin Luna, Yasuhiro Arakawa, Sudhir Varma, Jeffrey Wang, Anisha Tehim, Mark Raffeld, Kenneth Aldape, Christophe E. Redon, Roshan Shresta, William C Reinhold, M. Aladjem, Jaydira Del Rivero, Nitin Roper, Anish Thomas, Y. Pommier
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Background Small cell lung cancer (SCLC) is a lethal lung malignancy and patients are often diagnosed with distant metastasis. Nearly all patients suffer from disease relapsing with inherent chemoresistance. Lack of targeted SCLC therapies further worsens disease outcomes, making it highly desirable to identify novel and effective therapeutic targets. Methods To search for potential therapeutic targets in SCLC, we analyzed publicly available single-cell and bulk RNA-sequencing (RNA-seq) data from normal, lung adenocarcinoma, and SCLC tumor tissues. To assess the targeting potential of FOXM1, we developed various in vitro models, including DOX-On-shFOXM1 (Tet-ON) inducible stable knockdown systems. Cisplatin resistant human and murine SCLC cell lines were generated to assess the role of FOXM1 in chemotherapy resistance. Immunoblotting, immunohistochemistry (IHC), and immuno-fluorescence were used to analyze the expression of FOXM1 and target proteins. ChIP-assay was used to study protein-gene interactions. Further, multicolor flow cytometry was employed to study the effect of FOXM1 inhibition on human T cells activation and differentiation. Subcutaneous xenograft and SCLC spontaneous (RPM: RB fl/fl ; TP53 fl/fl ; LSL-MYC T58A ) mouse models were used to evaluate the efficacy of FOXM1 inhibitors. Results Single-cell as well as bulk RNA-seq data revealed that FOXM1, an oncogenic transcription factor, is overexpressed in SCLC, and it was recapitulated in human and murine SCLC tissues and cell lines. Interestingly, chemo-resistant (CR) SCLC showed a substantially higher FOXM1 expression than naïve SCLC. Silencing FOXM1 genetically or pharmacologically by FOXM1 inhibitors revealed a marked reduction in cell viability, colony formation, migration and sphere formation in naïve and CR SCLC cells. Moreover, FOXM1 inhibition induced apoptosis and cell cycle arrest in SCLC cells. Furthermore, FOXM1 inhibition in combination with first-line platinum-based chemotherapy showed synergistic anticancer effects in both xenograft and RPM mouse models of SCLC. Our RNA-seq analysis revealed that FOXM1 inhibition altered the Aurora Kinase B (AURKB) signaling pathway, which is dysregulated in SCLC. Moreover, we found FOXM1 inhibition enhanced T cell activation and supported the differentiation of CD8 + cytotoxic T cells, and T cell-mediated killing of cancer cells. Conclusions Our study demonstrates that FOXM1 targeting using small molecule inhibitors has the potential to be a novel therapeutic strategy to combat SCLC progression including chemotherapeutic resistance and reshaping the anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cf5514bc8f1dd3db6ac08b065580f4ed8890ea4" target='_blank'>
              FOXM1 targeting alters AURKB activity and reshapes antitumor immunity to curb the progression of small cell lung cancer
              </a>
            </td>
          <td>
            Md Arafat Khan, Parvez Khan, Mahek Fatima, Asad Ur Rehman, Laiba Anwar, Z. Alsafwani, Aatiya Ahmad, Mohammad Ali Abbas Zaidi, Jesse L. Cox, Areem Zahid, Sameer Mohiuddin, Sung Hoon Kim, Juan A. Santamaria-Barria, Imayavaramban Lakshmanan, B. Katzenellenbogen, John A Katzenellenbogen, A. Ganti, Surinder K Batra, M. Nasser
          </td>
          <td>2025-07-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="Cervical cancer is the fourth most common malignant tumor in women, lacking specific and sensitive markers. TP53 induced glycolysis regulatory phosphatase (TIGAR) plays an important role in the initiation and progression of several solid tumors. This study aims to investigate the diagnostic and prognostic and immunotherapeutic role of TIGAR in cervical cancer. We observed that TIGAR expression was upregulated in cervical cancer cells and tissues using experiments and bioinformatics analyses. Moreover, elevated TIGAR expression was associated with poor prognosis using Kaplan–Meier survival analysis, and its diagnostic value was confirmed using the Receiver operating characteristic (ROC) curve. We also analyzed the correlation between TIGAR expression and immune infiltration and assessed its correlation with a clinical cohort of cervical cancer, and found that TIGAR expression was significantly associated with the infiltration levels of Foxp3 + Treg cells, CD11b + MDSC, CTLA4, and HAVCR2. TIGAR is a potential diagnostic and prognostic biomarker of cervical cancer and is expected to offer new possibilities for cervical cancer immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07201-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91786186f8b506b70f9791c3768dec5816483289" target='_blank'>
              TIGAR regulated by HPV E6 is correlated with disease stage, drug sensitivity, and immune microenvironment in cervical cancer
              </a>
            </td>
          <td>
            Yameng Liu, Yu Zhang, Hailong Zhang, Dianqin Xu, Xinzhu Zhou, Luhong Xie, Hongli Xi, Yurong Zhu, Hanlin Yang, Xiaoyu Zhu, Ji Ren, Shaoju Min, Yujie Tan
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c15cd4c65cc7acb2163700609aefaa2af8b33ee" target='_blank'>
              Integrating bulk and single-cell RNA sequencing reveals intratumor heterogeneity phenotypes and immune infiltration in pancreatic cancer.
              </a>
            </td>
          <td>
            Li Zhao, Yao Huang, Jiang Li, Rui Xie, Kaiming Li
          </td>
          <td>2025-07-13</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction: Anal squamous cell carcinoma (ASCC) is a rare but increasingly common gastrointestinal malignancy, mainly associated with oncogenic human papillomaviruses (HPVs). The role of non-coding RNAs (ncRNAs) in tumorigenesis is recognized, but the impact of viral ncRNAs on host gene expression remains unclear. Methods: We re-analyzed total RNA-Seq data from 70 anal biopsies: 31 low-grade squamous intraepithelial lesions (LGSIL), 16 high-grade SIL (HGSIL), and 23 ASCC cases. Microbial composition was assessed taxonomically. Novel viral miRNAs were predicted using vsRNAfinder and linked to host targets using TargetScan and expression correlation analyses. Results: Microbial profiling revealed significant differences in abundance, with Alphapapillomaviruses types 9, 10, and 14 enriched across lesion grades. We identified 90 novel viral miRNAs and 177 significant anti-correlated miRNA–mRNA interactions. Target genes were enriched in pathways related to cell cycle, epithelial–mesenchymal transition, lipid metabolism, immune modulation, and viral replication. Discussion: Our findings suggest that HPV-derived miRNAs, including those from low-risk types, may contribute to neoplastic transformation by modulating host regulatory networks. Conclusion: This study highlights viral miRNAs as potential drivers of HPV-related anal cancer and supports their utility as early biomarkers and therapeutic targets in ASCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911fe59aaa1647471be1a4e931c53c959630dc5" target='_blank'>
              Human Papillomavirus-Encoded microRNAs as Regulators of Human Gene Expression in Anal Squamous Cell Carcinoma: A Meta-Transcriptomics Study
              </a>
            </td>
          <td>
            Daniel J. García, Marco A. Pulpillo-Berrocal, José L. Ruiz, E. Andrés-León, L. Terrón-Camero
          </td>
          <td>2025-06-01</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f2b837ac11b06bf8917b835488e0f8a39575dc9" target='_blank'>
              Post-translational modifications in DNA damage repair: mechanisms underlying temozolomide resistance in glioblastoma.
              </a>
            </td>
          <td>
            Yike Chen, Kaikai Ding, Shuyu Zheng, Songting Gao, Xiaohui Xu, Haijian Wu, Fengqi Zhou, Yongjie Wang, Jinfang Xu, Chun Wang, C. Ling, Jing Xu, Lin Wang, Qun Wu, G. Giamas, Gao Chen, Jianmin Zhang, Chenggang Yi, Jianxiong Ji
          </td>
          <td>2025-05-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Plasmodium parasites, the causative agents of malaria, undergo crucial developments within the mosquito vector, initiated by the formation of male and female gametes. Male gametogenesis involves three rapid rounds of mitosis without nuclear or cell division, followed by a single round of DNA segregation and nuclear division during gamete budding. How the cell organizes the segregation of eight genomes from a single octoploid nucleus into eight haploid gametes is currently unknown. Here we discovered an atypical Arp2/3 complex in Plasmodium important for DNA segregation during male gametogenesis. Unlike the canonical Arp2/3 complex found in other eukaryotes, Plasmodium Arp2/3 localizes to endomitotic spindles and interacts with a kinetochore-associated protein. Disruption of Arp2/3 subunits or actin polymerization interferes with kinetochore–spindle association, causes the formation of subhaploid gametes, and blocks transmission. Our work identified an evolutionary divergent Arp2/3 complex in malaria parasites, provides insights into gametogenesis, and reveals potential targets for transmission-blocking interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a0eb8cd0287d7e4de3c5f68362eedb92788d67" target='_blank'>
              An atypical Arp2/3 complex is required for Plasmodium DNA segregation and malaria transmission
              </a>
            </td>
          <td>
            Franziska Hentzschel, David Jewanski, Yvonne Sokolowski, Pratika Agarwal, Anna Kraeft, Kolja Hildenbrand, Lilian P. Dorner, Mirko Singer, Matthias Marti, Friedrich Frischknecht
          </td>
          <td>2025-06-13</td>
          <td>Nature Microbiology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="DNA methylation is the best-known epigenetic mechanism regulating gene expression without altering the DNA sequence. Its counterpart, known as DNA demethylation, is equally important and enables the activation of previously silenced genes. DNA demethylation has attracted interest in the scientific community following the landmark discovery that Ten-Eleven Translocation (TET) proteins can convert 5-methylcytosine to 5-hydroxymethylcytosine. A growing body of research indicates that changes in TET protein levels and 5-hydroxymethylcytosine content are hallmarks of cancer. These epigenetic changes appear to play a critical role in the development of malignancies characterized by high levels of somatic mutations and genetic instability. Bladder cancer is among the most common cancers worldwide and, despite aggressive treatment, remains associated with high mortality and poor prognosis. The lack of reliable diagnostic and prognostic markers poses a significant challenge in its management, highlighting the urgent need for novel biomarkers to enable earlier diagnosis and more accurate prediction of clinical outcomes. This review examines epigenetic alterations associated with bladder cancer and their clinical implications. We focus on the impact of DNA methylation and demethylation on oncogene regulation, summarize scientific evidence supporting their role in bladder cancer development and progression, and briefly explore novel therapeutic strategies targeting those epigenetic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38660186524a060ff11f30a8ae4e9ad5b94b808c" target='_blank'>
              The role of DNA methylation and demethylation in bladder cancer: a focus on therapeutic strategies
              </a>
            </td>
          <td>
            Wiktoria Strasenburg, Jędrzej Borowczak, Daria Piątkowska, J. Jóźwicki, Dariusz Grzanka
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79b34bae8b1dbcf266da5a700548580b6368ded" target='_blank'>
              Bipartite chromatin recognition by Hop1 from two diverged holozoa
              </a>
            </td>
          <td>
            Alyssa A. Rodriguez, Alessandro E. Cirulli, Katie Chau, Justin Nguyen, Qiaozhen Ye, Kevin D. Corbett
          </td>
          <td>2025-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most aggressive and lethal brain tumor in adults and poses significant challenges to patient survival. This review provides a comprehensive exploration of the molecular and genetic landscape of GBM, focusing on key oncogenic drivers, such as epidermal growth factor receptor (EGFR), platelet-derived growth factor receptor (PDGFR), and the PI3K/AKT/mTOR pathway, which are critical for tumorigenesis and progression. We delve into the role of epigenetic alterations, including DNA methylation and histone modifications, in driving therapy resistance and tumor evolution. The tumor microenvironment is known for its pivotal role in immune evasion, with tumor-associated macrophages, myeloid-derived suppressor cells, and regulatory T cells creating an immunosuppressive niche that sustains GBM growth. Emerging therapies, such as immunotherapies, oncolytic viral therapies, extracellular vesicle-based approaches, and non-coding RNA interventions, are highlighted as promising avenues to disrupt GBM pathogenesis. Advances in precision medicine and innovative technologies, including electric field therapy and locoregional treatments, are discussed for their potential to overcome the blood‒brain barrier and treatment resistance. Additionally, this review underscores the importance of metabolic reprogramming, particularly hypoxia-driven adaptations and altered lipid metabolism, in fueling GBM progression and influencing the therapeutic response. The role of glioma stem cells in tumor recurrence and resistance is also emphasized, highlighting the need for targeted therapeutic approaches. By integrating molecular targeting, immune energetics, and technological advancements, this review outlines a multidisciplinary framework for improving GBM treatment outcomes. Ultimately, the convergence of genetic, metabolic, and immune-based strategies offers transformative potential in GBM management, paving the way for increased patient survival and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1445c096bc679931586564b48bbb15167d4847f8" target='_blank'>
              Glioblastoma at the crossroads: current understanding and future therapeutic horizons
              </a>
            </td>
          <td>
            Shilpi Singh, Devanjan Dey, Debashis Barik, Iteeshree Mohapatra, Stefan Kim, Mayur Sharma, Sujata Prasad, Peize Wang, Amar Singh, Gatikrushna Singh
          </td>
          <td>2025-07-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Loss-of-function (LoF) mutations frequently found in human cancers are generally intractable by classical small molecule inhibitor approaches. Among them are mutations affecting polycomb-group (PcG) epigenetic regulators, EZH2 and ASXL1 frequently found in haematological malignancies of myeloid or lymphoid lineage, and their concurrent mutations associates with particularly poor prognosis. While there is clear need to develop novel and effective treatments for these patients, the lack of appropriate disease models and mechanistic insights have significantly hindered the progresses. Here we show that genetic inactivation of Asxl1 and Ezh2 in murine haematopoietic stem/progenitor cells results in highly penetrant haematological malignancies as observed in corresponding human diseases. These PcG proteins regulate both coding and non-coding genomes, leading to marked reactivation of transposable elements (TEs) and DNA damage responses in PcG LoF mutated cells, which create a novel vulnerability for PARP inhibitors (PARPi)-induced synthetic lethality. Using both mouse models and primary patient samples, we demonstrate that Asxl1/Ezh2 mutated cells are highly sensitive to PARPi that induce excessive DNA damage and significantly extend disease latency. Intriguingly the observed PARPi-sensitivity can be specifically overridden by reverse transcriptase inhibitors that interrupt target-site primed reverse transcription (TPRT) and life cycle of TEs. This mechanism is contrastingly different from the current concept of BRCAness associated PARPi-induced synthetic lethality, which largely rely on deficient homologous recombination and is independent on reverse transcriptase inhibitors. Together, this study reveals a novel application and mechanism of PARPi-induced synthetic lethal targeting of blood cancers with reactivated TEs such as those carrying PcG epigenetic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5054befc76e369abbf5e3e565cd84241ccf574" target='_blank'>
              Transposable elements as novel therapeutic targets for PARPi-induced synthetic lethality in PcG-mutated blood cancer.
              </a>
            </td>
          <td>
            B. Zeisig, Chiou-Tsun Tsai, Clemence Virely, T. Fung, A. Akin, Estelle Troadec, Bo Jiao, Richard Thompson, Priscilla Nga Ieng Lau, Nanjun Li, Andriani Charalambous, Larissa Bomfim, Jennifer Lynch, Athina Georgiou, Robbert Hoogeboom, Claire Lynn, Si Yi Zhang, P. Patten, Cynthia Fisher, Anna Schuh, Seishi Ogawa, G. Mufti, Mohammad M Karimi, Chi Wai Eric So
          </td>
          <td>2025-06-25</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="By offering a precise, effective, and adaptable method for genome editing, CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) technology has completely transformed genetic biotechnology. Through a biochemical lens that highlights the molecular complexities behind its methods and discoveries, this review explores the complex field of CRISPR cloning and its emerging frontiers. We examine the molecular underpinnings of target identification, protospacer adjacent motif (PAM) specificity, and Cas nuclease activation, starting with the groundbreaking discovery of CRISPR-Cas systems in bacterial adaptive immunity. Advances in CRISPR-based cloning techniques, such as the creation of synthetic guide RNAs, Cas variants with specific functions, and high-fidelity and base-editing systems that reduce off-target effects, are also examined in the study. The molecular dynamics of DNA-RNA-protein interactions during gene targeting, activation of DNA repair pathways, and epigenetic state regulation are given special attention. We investigate cutting-edge uses beyond traditional genome editing, including RNA-targeting Cas proteins (like Cas13), CRISPR-based diagnostics and treatments, CRISPR interference (CRISPRi), and CRISPR activation (CRISPRa). Additionally covered is the incorporation of CRISPR with other biochemical technologies, such as gene drives, programmable transcription factors, and synthetic biology circuits. The field's increasing biochemical sophistication is highlighted by new developments like CRISPR-nanoparticle conjugates for targeted delivery and CRISPR screening platforms for functional genomics. In the end, this review emphasizes how a thorough biochemical comprehension of CRISPR systems is essential to maximizing their potential in environmental engineering, biotechnology, medicine, and agriculture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf5fde83fcfa701e2f38df12fc05920c20f45e1" target='_blank'>
              Exploring CRISPR Cloning and Beyond Through a Biochemical Lens in Genetic Biotechnology
              </a>
            </td>
          <td>
            Mariam Mariam, Khizra Ikhlaq, Zulaikha Abid, Fatima Sahar Uraf Maryam, Syed Qasim Mehmood, Ahtasham Ahmad, Sohrab Anwar Abbasi, Imran Ali Sani, Muhammad Adeel
          </td>
          <td>2025-07-02</td>
          <td>Scholars Academic Journal of Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has recently become a focus of cardiovascular disease (CVD) research as it is possibly involved in disease mechanisms and may function as a biomarker. The literature suggests that eccDNAs may modify gene expression patterns and contribute to the pathogenesis of diseases such as pulmonary arterial hypertension (PAH). Many eccDNAs have been reported to be upregulated in PAH patients with promisingly high diagnostic sensitivity and specificity values. The advent of eccDNA detection from plasma samples using high‐throughput sequencing technologies has opened new avenues for non‐invasive diagnosis. In this review, we focus on the role of eccDNA in CVDs, concentrating on its prospects for use as a biomarker of disease development and stage. However, further studies should be conducted to explicate the functional aspects of eccDNAs in cardiovascular health and disease. This work may eventually lead to the development of novel treatment options and improved clinical outcomes for CVD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/903ec3642b6c2ce94d9db06e20223312c1b439d8" target='_blank'>
              Extrachromosomal Circular DNA: Emerging Insights Into Cardiovascular Disease Mechanisms and Biomarker Potential
              </a>
            </td>
          <td>
            Saim Mahmood Khan, Jawairya Muhammad Hussain, Surraiya Riaz Mahmood Khan, Bushra Nadeem, Taniyat Naseem, Hammad Jawed, Aina Lashari
          </td>
          <td>2025-06-29</td>
          <td>iLABMED</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BRCA1-associated protein 1 (BAP1) and Enhancer of zeste homolog 2 (EZH2) proteins are associated with the Polycomb repressive complex (PRC), and both have key roles in cell cycle and DNA repair pathways, so variants in BAP1 and/or EZH2 could lead to tumorigenesis. Somatic and germline variants in these genes have been associated with uveal and cutaneous melanoma, mesothelioma, renal cell carcinoma (BAP1 loss-of-function variants), and lymphoma, leukemia, and triple-negative breast cancer (EZH2 gain-of-function variants). Loss of BAP1 function in the murine model has been shown to lead to an overexpression of EZH2 mRNA. Tazemetostat is a selective EZH2 inhibitor currently approved for the treatment of follicular lymphoma in the third line. The effects of tazemetostat have been studied in clinical trials for patients with BAP1 and EZH2 pathogenic variants with malignant mesothelioma and it has shown significant results when used in combination with poly (ADP-ribose) polymerase inhibitors on breast cancer cell lines. The interaction of BAP1 and EZH2 and the efficacy of tazemetostat show great potential for the future of cancer treatment for patients harboring variants in those genes due to the more aggressive nature of those tumors and the lack of targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79d6acaaed33cf0b6b7cca65a93fe16d0a41274" target='_blank'>
              BRCA1-Associated Protein 1 and Enhancer of Zeste Homolog 2: Pathway Interaction and Therapeutic Intervention in Breast Cancer, Mesothelioma, and Lymphoma.
              </a>
            </td>
          <td>
            Elizabeth Casey West, Jessica Evangelista, Annalisa Campanella, F. Lococo, Laura Sun, D. Ivankovic, Christopher L Farrell, Veronica Parker, Luigi Boccuto
          </td>
          <td>2025-06-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [7],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>