<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-20 09:25:41 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The tumor microenvironment (TME) is a complex network of interactions between malignant and host cells, yet its orchestration in advanced high-grade serous ovarian carcinoma (HGSC) remains poorly understood. We present a comprehensive single-cell spatial atlas of 280 metastatic HGSCs, integrating high-dimensional imaging, genomics, and transcriptomics. Using 929 single-cell maps, we identify distinct spatial domains associated with phenotypically heterogeneous cellular compositions, and demonstrate that immune cell co-infiltration at the tumor-stroma interface significantly influences clinical outcomes. To uncover the key drivers of the tumor ecosystem, we developed CEFIIRA (Cell Feature Importance Identification by RAndom forest), which identified tumor cell-intrinsic MHC-II expression as a critical predictor of prolonged survival, independent of clinicomolecular profiles. Validation with external datasets confirmed that MHC-II-expressing cancer cells drive immune infiltration and orchestrate spatial tumor-immune interactions. Our atlas offers novel insights into immune surveillance mechanisms across HGSC clinicomolecular groups, paving the way for improved therapeutic strategies and patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/684d76f91d1a8b1bc3e208feca47d927bec6a2ec" target='_blank'>
              Single-cell spatial atlas of high-grade serous ovarian cancer unveils MHC class II as a key driver of spatial tumor ecosystems and clinical outcomes
              </a>
            </td>
          <td>
            Fernando Pérez-Villatoro, Lilian van Wagensveld, Aleksandra Shabanova, Ada Junquera, Ziqi Kang, Iga Niemiec, Matias M. Falco, Ella Anttila, J. Casado, Eric Marcus, Essi Kahelin, Foteini Chamchougia, M. Salko, Saundarya Shah, Salvatore Russo, Jacopo Chiaro, Mikaela Grönholm, G. Sonke, K. K. van de Vijver, Rutgerus Fpm Kruitwagen, Maaike Avan der Aa, A. Virtanen, Vincenzo Cerullo, Anna Vähärautio, P. Sorger, Hugo M. Horlings, A. Färkkilä
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="In bladder cancer (BLCA), the cellular expression of lineage markers with predictive biomarker function can have important implications for tumor progression, treatment response, and survival. However, tumor heterogeneity and the coexistence of distinct tumor subpopulations can complicate the utility of using a primary, clinically assigned tumor signature. In this report, we have applied the reference atlas, Tabula Sapiens, and deep learning model, UniCell, to conduct in-depth transcriptional analysis of unique lineage marker-defined cell clusters in a carcinogen (BBN) induced bladder cancer model. UniCell deconvolution has identified tumor populations, including urothelial, adenocarcinoma, squamous cell carcinoma, and mesenchymal tumor populations, each with cell-intrinsic gene expression signatures relevant to human BLCA progression. The identified tumor clusters contain uniquely basal, luminal, stromal, or hybrid cells in cell lineage marker expression. To understand the significance of these populations during progression, we used trajectory and pseudo-time analysis to show that cells uniquely basal are plastic in lineage identity and can evolve to form tumor populations composed of other lineage marker-defined signatures. Finally, pathway and drug enrichment analysis of tumor cell clusters were used to identify therapeutics that may preferentially target the identified tumor cell populations. These data collectively define a molecular template that may uniquely profile molecular plasticity occurring during progression and response to therapy with important implications for human disease. Significance Tumor heterogeneity is a mechanism for treatment resistance. Our study defines unique tumor subpopulations having differential therapeutic sensitivities and potential for lineage plasticity. Our modeling may impact the treatment of BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e51b8c18f48b163ace1388c13e342c3f5124f692" target='_blank'>
              Mapping of Multilineage Tumor Cell Populations in Mouse Bladder Cancer
              </a>
            </td>
          <td>
            Nadine Schrode, Yang Hu, Haocheng Yu, M. Galsky, J. Sfakianos, R. Sebra, Kristin G Beaumont, David J Mulholland
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving rapidly as a pivotal technology in studying the biology of tumors and their associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. In this study, we used serial 5-m sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma tumor samples in tissue microarrays to compare the performance of the single cell ST platforms CosMx, MERFISH, and Xenium (uni/multi-modal) platforms in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx Digital Spatial Profiler, and hematoxylin and eosin staining data for the same samples. In addition to objective assessment of automatic cell segmentation and phenotyping, we performed pixel-resolution manual evaluation of phenotyping to carry out pathologically meaningful comparison between ST platforms. Our study detailed the intricate differences between the ST platforms, revealed the importance of parameters such as tissue age and probe design in determining the data quality, and suggested reliable workflows for accurate spatial profiling and molecular discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17805d448b42a38f2a1b50b0eb2fd17aaa264ce4" target='_blank'>
              Comparison of imaging-based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed, paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Nejla Ozirmak Lermi, Max Molina Ayala, Sharia D Hernandez, Wei Lu, Khaja B. Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, Jinzhuang Dou, Qingnan Liang, Rti Team, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchez-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L Gibbons, Ignacio I. Wistuba, C. Haymaker, Ken Chen, L. S. Solis Soto
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), a distinct subpopulation within tumors, are pivotal in driving treatment resistance and tumor recurrence, posing substantial challenges to conventional therapeutic strategies. Precise quantification and profiling of these cells are essential for improving cancer treatment outcomes. We present ACSCeND, an advanced deep neural network model accompanied by a robust workflow, specifically developed to quantify cellular compositions from bulk RNA-seq data, enabling accurate CSC profiling. By integrating bulk RNA-seq data with insights derived from single-cell RNA-seq datasets, ACSCeND effectively captures the diversity and hierarchical organization of tumor-resident cell states, alongside cell-specific gene expression profiles (GEPs). Compared to current tissue deconvolution models, ACSCeND exhibits superior performance, achieving significantly higher Concordance Correlation Coefficient (CCC) values and lower Root Mean Square Error (RMSE) across various pseudobulk and real-world bulk tissue samples. Application of ACSCeND to TCGA and PRECOG datasets reveals a strong association between CSC abundance and poorer disease-free survival outcomes, underscoring the clinical relevance of CSCs in cancer progression. Furthermore, cell-specific GEPs for distinct CSC states unveil novel molecular signatures and illuminate the origins of CSC-driven tumor heterogeneity. In summary, ACSCeND provides a powerful, scalable platform for high-throughput quantification of cellular compositions and distinct potency states within normal tissues as well as highly heterogeneous tissues, such as tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b3b0a8bb89725a847aae6e2af230a0666ba9d6" target='_blank'>
              Comprehensive Enumeration of Cancer Stem-like Cell Heterogeneity Using Deep Neural Network
              </a>
            </td>
          <td>
            Debojyoti Chowdhury, Shreyansh Priyadarshi, Sayan Biswas, Bhavesh Neekhra, Debayan Gupta, Shubhasis Haldar
          </td>
          <td>2024-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to random barcode sets. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fe68cd8b32acb61197d1f1f0a4cc878efa91c6" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Spatial profiling of tissues promises to elucidate tumor-microenvironment interactions and generate prognostic and predictive biomarkers. We analyzed single-cell, spatial data from three multiplex imaging technologies: cyclic immunofluorescence (CycIF) data we generated from 102 breast cancer patients with clinical follow-up, and publicly available imaging mass cytometry and multiplex ion-beam imaging datasets. Similar single-cell phenotyping results across imaging platforms enabled combined analysis of epithelial phenotypes to delineate prognostic subtypes among estrogen-receptor positive (ER+) patients. We utilized discovery and validation cohorts to identify biomarkers with prognostic value. Increased lymphocyte infiltration was independently associated with longer survival in triple-negative (TN) and high-proliferation ER+ breast tumors. An assessment of ten spatial analysis methods revealed robust spatial biomarkers. In ER+ disease, quiescent stromal cells close to tumor were abundant in good prognosis tumors, while tumor cell neighborhoods containing mixed fibroblast phenotypes were enriched in poor prognosis tumors. In TN disease, macrophage/tumor and B/T lymphocyte neighbors were enriched and lymphocytes were dispersed in good prognosis tumors, while tumor cell neighborhoods containing vimentin-positive fibroblasts were enriched in poor prognosis tumors. In conclusion, we generated comparable single-cell spatial proteomic data from several clinical cohorts to enable prognostic spatial biomarker identification and validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e86a33275ed2b41ff7e598b0d0e6585ea48b22" target='_blank'>
              Highly multiplexed imaging reveals prognostic immune and stromal spatial biomarkers in breast cancer.
              </a>
            </td>
          <td>
            Jennifer Eng, Elmar Bucher, Zhi Hu, Cameron R Walker, Tyler Risom, Michael Angelo, Paula I. Gonzalez-Ericsson, M. E. Sanders, A. B. Chakravarthy, Jennifer A Pietenpol, Summer L. Gibbs, Rosalie C. Sears, Koei Chin
          </td>
          <td>2025-01-14</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The organizational principles of nephronal segments are based on longstanding anatomical and physiological attributes that are closely linked to the homeostatic functions of the kidney. Novel molecular approaches have recently uncovered layers of deeper signatures and states in tubular cells that arise at various timepoints on the spectrum between health and disease. For example, a dedifferentiated state of proximal tubular cells with mesenchymal stemness markers is frequently seen after injury. The persistence of such a state is associated with failed repair. Here, we introduce a novel analytical pipeline applied to highly multiplexed spatial protein imaging to characterize proximal tubular subpopulations and neighborhoods in reference and disease human kidney tissue. The results were validated and extended through integration with spatial and single cell transcriptomics. We demonstrate that, in reference tissue, a large proportion of S1 and S2 proximal tubular epithelial cells express THY1, a mesenchymal stromal and stem cell marker that regulates differentiation. Kidney disease is associated with loss of THY1 and transition towards expression of PROM1, another stem cell marker shown recently to be linked to failed repair. We demonstrate that the trajectory of proximal tubular cells to THY1 expression is clearly distinct from that of PROM1, and that a state with PROM1 expression is associated with niches of inflammation. Our data support a model in which the interplay between THY1 and PROM1 expression in proximal tubules associates with their regenerative potential and marks the timeline of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e12887f03204d9141acb86ce8374e99f9dcbad" target='_blank'>
              Integration of spatial multiplexed protein imaging and transcriptomics in the human kidney tracks the regenerative potential timeline of proximal tubules
              </a>
            </td>
          <td>
            Mahla Asghari, Angela R. Sabo, Daria Barwinska, R. M. Ferreira, M. Ferkowicz, William S. Bowen, Ying-Hua Cheng, D. Gisch, Connor J. Gulbronson, Carrie L. Phillips, Katherine J. Kelly, T. Sutton, James C Williams, Miguel Vazquez, John O’Toole, Paul M. Palevsky, Sylvia E. Rosas, S. Waikar, Krzysztof Kiryluk, C. Parikh, Jeff Hodgins, P. Sarder, Ian H. de Boer, Jonathan Himmelfarb, Matthias Kretzler, Sanjay Jain, Michael T. Eadon, S. Winfree, Tarek M. El-Achkar, P. Dagher
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this gap, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from 2 patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells and we experimentally demonstrated that CCL18 increases neuroblastoma cell migration. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed the ALK ligand ALKAL2 and were predicted to communicate with neighboring ALK expressing cancer cells. We demonstrated a unique developmental pattern of adrenal medulla-specific expression of ALK and adrenocortical-specific expression of ALKAL2, suggesting a role of these signaling interactions in neuroblastoma carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cacb0dcc803b6c58db82e59f6d5ec533dad1808" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlson, A. Claeys, Eva Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Thyroid cancer progression from curable well-differentiated thyroid carcinoma to highly lethal anaplastic thyroid carcinoma is distinguished by tumor cell de-differentiation and recruitment of a robust stromal infiltrate. Combining an integrated thyroid cancer single-cell sequencing atlas with spatial transcriptomics and bulk RNA-sequencing, we define stromal cell subpopulations and tumor-stromal cross-talk occurring across the histologic and mutational spectrum of thyroid cancer. We identify distinct inflammatory and myofibroblastic cancer-associated fibroblast (iCAF and myCAF) populations and perivascular-like populations. The myCAF population is only found in malignant samples and is associated with tumor cell invasion, BRAFV600E mutation, lymph node metastasis, and disease progression. Tumor-adjacent myCAFs abut invasive tumor cells with a partial epithelial-to-mesenchymal phenotype. Tumor-distant iCAFs infiltrate inflammatory autoimmune thyroid lesions and anaplastic tumors. In summary, our study provides an integrated atlas of thyroid cancer fibroblast subtypes and spatial characterization at sites of tumor invasion and de-differentiation, defining the stromal reorganization central to disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b786943f0e4090faa02f89896c2ed59c657e0d0" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations
              </a>
            </td>
          <td>
            Matthew A. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey Caroland, Megan L. Tigue, H. Hartmann, J. Gallant, Courtney J. Phifer, Andres Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H Belcher, Ken S. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="In this study, we delved into the intricate cellular landscape and interactions within hepatocellular carcinoma (HCC) using single-cell RNA sequencing (scRNA-seq). Our analysis uncovered a diverse array of cell types present in HCC tumors, including endothelial cells, fibroblasts, hepatocytes, Kupffer cells, macrophages, neutrophils, NK cells, pericytes, and proliferating cells. By examining the expression profiles of key marker genes, we elucidated the distinct transcriptional signatures associated with each cell type. Notably, we observed upregulation of certain genes associated with tumor progression and metastasis, highlighting potential therapeutic targets. Additionally, our study revealed the relative proportions of each cell type within HCC tumors, shedding light on the complex cellular composition of these malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/516f83808b51104c5136da0299de6bce74fe34c4" target='_blank'>
              Single-Cell RNA Sequencing: Exploring the Cellular Landscape and Interactions in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Wenyi Wei
          </td>
          <td>2024-11-25</td>
          <td>Advances in Engineering Technology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The intra- and inter-tumoral heterogeneity of glioblastoma represents a significant therapeutic challenge, as well as difficulty in generating reliable models for in vitro studies. Historical 2D adherent cell lines do not recapitulate this complexity, whereas both patient-derived neurospheres (PDN) and organoids (PDO) demonstrate intra-tumoral heterogeneity. Here, we quantify the tumor cell composition from matched models established from the same primary tumor using a series of multi-omic interrogations. We find that both patient-derived models recapitulate the genomic, epigenomic and tumor cell heterogeneity of the primary tissue. Furthermore, single-nuclei RNA sequencing revealed a subset of organoids containing small numbers of non-malignant cells from neuron and immune cell lineages. Harnessing the intra-tumoral heterogeneity of PDN models, we reveal the impact of temozolomide chemotherapy on individual cell states, altering composition of tumors over time in response to treatment. Our data confirms that both patient-derived models recapitulate patient intra-tumoral heterogeneity providing a platform for tumor cell state refined therapeutic studies. Key Points Generation of matched patient-derived neurosphere and organoid models from resected GBM tissue Neurosphere models exhibit greater proliferative signatures Both patient-derived models recapitulate genomic and epigenomic features of the primary tissue Single-nuclei RNA sequencing reveals both models recapitulate intra-tumoral heterogeneity of the primary tissue Neurosphere models enable interrogation of therapeutic responses in the context of heterogeneity Importance of study Patient derived models can be powerful tools when they faithfully recapitulate the tumor tissue from which they are derived. In glioblastoma, patient derived neurospheres (PDN) and organoids (PDO) have both been used in studies, however the differences between these models and the recapitulation of patient heterogeneity remain to be fully characterized. To address this, we performed multi-omic profiling of PDN and PDO models generated from the same tumor tissue. We find that that across a range of data modalities, both model systems exhibit a high level of resemblance to tissue, and critically, maintain heterogeneity and tumor cell composition. The importance of modeling heterogeneity was demonstrated in PDN models, where temozolomide treatment specifically alters the abundance of MES-like and AC-like tumor cells. Our findings demonstrate that neurosphere and organoid models effectively preserve cellular heterogeneity, genomic alterations, methylation signatures and transcriptomic features, both highly suitable to model glioblastoma’s complex cellular landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c3caa2d109062b5ff4f7f9f68c99c0d6be111c" target='_blank'>
              Three-dimensional patient-derived models of glioblastoma retain intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Zachery Moore, Claire Storey, Daniel V. Brown, A. Valkovic, Montana Spiteri, Jasmine F. Pignatelli, Shannon J Oliver, Alana Fakhri, Kate Drummond, Seth Malinowski, Keith L. Ligon, Oliver M. Sieber, J. Whittle, S. Freytag, Sarah A. Best
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Recent advancements in spatial transcriptomics technologies have significantly enhanced resolution and throughput, underscoring an urgent need for systematic benchmarking. To address this, we collected clinical samples from three cancer types – colon adenocarcinoma, hepatocellular carcinoma, and ovarian cancer – and generated serial tissue sections for systematic evaluation. Using these uniformly processed samples, we generated spatial transcriptomics data across five high-throughput platforms with subcellular resolution: Stereo-seq v1.3, Visium HD FFPE, Visium HD FF, CosMx 6K, and Xenium 5K. To establish ground truth datasets, we profiled proteins from adjacent tissue sections corresponding to all five platforms using CODEX and performed single-cell RNA sequencing on the same samples. Leveraging manual cell segmentation and detailed annotations, we systematically assessed each platform’s performance across key metrics, including capture sensitivity, specificity, diffusion control, cell segmentation, cell annotation, spatial clustering, and transcript-protein alignment with adjacent CODEX. The uniformly generated, processed, and annotated multi-omics dataset is valuable for advancing computational method development and biological discoveries. The dataset is accessible via SPATCH, a user-friendly web server for visualization and download (http://spatch.pku-genomics.org/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a092f418d2d0937b601a2e81131e13ca731653a" target='_blank'>
              Systematic Benchmarking of High-Throughput Subcellular Spatial Transcriptomics Platforms
              </a>
            </td>
          <td>
            Pengfei Ren, Rui Zhang, Yunfeng Wang, Peng Zhang, Ce Luo, Suyan Wang, Xiaohong Li, Zongxu Zhang, Yanping Zhao, Yufeng He, Haorui Zhang, Yufeng Li, Zhidong Gao, Xiuping Zhang, Yahui Zhao, Zhihua Liu, Yuanguang Meng, Zhe Zhang, Zexian Zeng
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract High-dimensional, spatial single-cell technologies, such as CyTOF imaging mass cytometry (IMC), provide detailed information regarding locations of a large variety of cancer and immune cells in microscopic scales in tumor microarray slides obtained from patients prior to immune checkpoint inhibitor (ICI) therapy. An important question is how the initial spatial organization of these cells in the tumor microenvironment (TME) changes with time and regulates tumor growth and eventually outcomes as patients undergo ICI therapy. Utilizing IMC data of melanomas of patients who later underwent ICI therapy, we develop a spatially resolved interacting cell system model that is calibrated against patient response data to address the above question. We find that the tumor fate in these patients is determined by the spatial organization of activated CD8+ T cells, macrophages, and melanoma cells and the interplay between these cells that regulate exhaustion of CD8+ T cells. We find that fencing of tumor cell boundaries by exhausted CD8+ T cells is dynamically generated from the initial conditions that can play a protumor role. Furthermore, we find that specific spatial features such as co-clustering of activated CD8+ T cells and macrophages in the pretreatment samples determine the fate of the tumor progression, despite stochastic fluctuations and changes over the treatment course. Our framework enables the determination of mechanisms of interplay between a key subset of tumor and immune cells in the TME that regulate clinical response to ICIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9225e8a7e492a347e0673b0bb1568c995b67d250" target='_blank'>
              Spatial organization and stochastic fluctuations of immune cells impact clinical responsiveness to immunotherapy in melanoma patients
              </a>
            </td>
          <td>
            Giuseppe Giuliani, William Stewart, Zihai Li, Ciriyam Jayaprakash, Jayajit Das
          </td>
          <td>2024-11-26</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23b3d699dd705e1b450a74d7c903ab854deff777" target='_blank'>
              Increased spatial coupling of integrin and collagen IV in the immunoresistant clear-cell renal-cell carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, Taylor Peak, Oscar E. Ospina, N. Chakiryan, Anders E. Berglund, Paul A Stewart, Jonathan V Nguyen, C. M. Segura, Natasha L. Francis, P. R. Echevarria, J. Chahoud, Roger Li, Kenneth Y. Tsai, Jodi A. Balasi, Yamila Caraballo Peres, J. Dhillon, Lindsey A. Martinez, Warren E. Gloria, Nathan Schurman, Seang-Jae Kim, M. Gregory, James J Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-12-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="Uncontrolled proliferation is a hallmark of cancer, yet tumour cells can enter G0 arrest by halting the cell cycle reversibly (quiescence) or irreversibly (senescence) to survive under stress and in hostile tumour microenvironments (TME). G0 arrested cells contribute to drug tolerance, metastasis and recurrence, but their identification remains challenging due to their rarity and elusive regulatory pathways. Here, we quantify G0 arrest and proliferation decisions in single-cell and spatially profiled breast primary tumours to unveil adaptive responses driving immune compartmentalisation. We identify a genomically-constrained and prolonged G0 arrest state resembling that of dormant precursors of cancer progression. This state featured adaptive transcriptional reprogramming, including unfolded protein response stress, reduced mitochondrial translation and epithelial-mesenchymal plasticity enabled by semaphorin signalling. Spatial transcriptomics revealed G0 arrest pockets encapsulated by APOE+ lipid-associated macrophages, myofibroblastic CAFs and immature perivascular cells, suggestive of an immunosuppressive niche contrasting with cytotoxic environments in proliferative hotspots and displaying distinct drug sensitivities. To facilitate future research, we provide a foundation model capturing G0 arrest and proliferation with 94% accuracy in single cell data, available at https://github.com/secrierlab/G0-LM. Our findings provide new insights into the spatial organisation of cell cycle arrest in breast cancer, highlighting the role of G0 states in tumour heterogeneity and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d540e388314aece2599568858b5b0ad8004482" target='_blank'>
              Balancing tumour proliferation and sustained cell cycle arrest through proteostasis remodelling drives immune niche compartmentalisation in breast cancer
              </a>
            </td>
          <td>
            Cenk Celik, Eloise Withnell, Shi Pan, Tooki Chu, Johnathan Labbadia, M. Secrier
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioma is an aggressive brain tumor with a poor prognosis. Establishing an in vitro culture model that closely replicates the cellular composition and microenvironment of the original tumor has been challenging, limiting its clinical applications. Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib. Thus, this organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076fe0fc7a2a2145464370151771e2e7246f87bc" target='_blank'>
              GlioME: A Novel Organoid Model Retaining the Glioma Microenvironment for Personalized Drug Screening and Therapeutic Evaluation
              </a>
            </td>
          <td>
            Chengjun Zheng, Qiaodong Chen, Peng Wang, Delong Zhang, Zheng Fang, Yutong Feng, Jie Chen, Jiahong Chen, Yiwen Fu, Bao Yang, Qian Zhao, Fei Sun, Ying Zhang, Tao Jiang, Lemin Zheng, Zhaoshi Bao
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Dysregulation of intracellular signaling networks underpins cancer. However, a systems-level elucidation of how signaling networks within distinct cell subpopulations drive cancer progression in vivo has been unattainable due to technical limitations. We developed INSIGHT (INvestigating SIGnaling network of specific cell subpopulation in Heterogeneous Tissue), a new platform technology combining fluorescence-activated cell sorting with ultra-sensitive mass spectrometry to enable phosphoproteomic characterization of rare and discrete cell subpopulations from fixed tissues. We demonstrated the broad utility of INSIGHT by analyzing the oligodendroglial cell-specific signaling network in the mouse brain. We then applied INSIGHT to investigate the rare, disseminated tumor cell subpopulation in glioblastoma patient-derived xenograft models. INSIGHT uncovered a global rewiring of signaling networks with tumor cell dissemination, marked by a transition from proliferation-associated signaling in the primary tumor cells to signaling associated with postsynapse, neuronal migration, and ion homeostasis in disseminated tumor cells. We reveal interconnections between signaling circuitries within the networks, with numerous proteins, including GluA2, exhibiting altered phosphorylation without protein expression changes, emphasizing the role of post-translational modifications in glioblastoma dissemination. We validated key phosphorylation changes and inferred differentially active kinases with tumor spread to offer new systems-level insights into glioblastoma dissemination mechanisms in vivo. INSIGHT is generally applicable to a wide range of biological systems without genetic engineering and provides quantitative phosphorylation and protein expression data for selected cell subpopulations from heterogeneous tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/decaf507fd537acca35393bd50f3bedca402f1f6" target='_blank'>
              Uncovering the signaling networks of disseminated glioblastoma cells in vivo with INSIGHT
              </a>
            </td>
          <td>
            Ryuhjin Ahn, Alicia D. D’Souza, Lawrence Long, Yufei Cui, D. Burgenske, K. Bakken, Lauren L Ott, Brett L. Carlson, Grace Zhou, Tomer M. Yaron-Barir, Ishwar N. Kohale, Charles A. Whittaker, Cameron T. Flower, Jeffrey Wyckoff, A. Tuma, Jared L. Johnson, Jann Sarkaria, Forest M. White
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genetically engineered mouse models (GEMM) of cancer are a useful tool for exploring the development and biological composition of human tumors and, when combined with single-cell RNA-sequencing (scRNA-seq), provide a transcriptomic snapshot of cancer data to explore heterogeneity of cell states in an immunocompetent context. However, cross-species comparison often suffers from biological batch effect and inherent differences between mice and humans decreases the signal of biological insights that can be gleaned from these models. Here, we develop scVital, a computational tool that uses a variational autoencoder and discriminator to embed scRNA-seq data into a species-agnostic latent space to overcome batch effect and identify cell states shared between species. We introduce the latent space similarity (LSS) score, a new metric designed to evaluate batch correction accuracy by leveraging pre-labeled clusters for scoring instead of the current method of creating new clusters. Using this new metric, we demonstrate scVital performs comparably well relative to other deep learning algorithms and rapidly integrates scRNA-seq data of normal tissues across species with high fidelity. When applying scVital to pancreatic ductal adenocarcinoma or lung adenocarcinoma data from GEMMs and primary patient samples, scVital accurately aligns biologically similar cell states. In undifferentiated pleomorphic sarcoma, a test case with no a priori knowledge of cell state concordance between mouse and human, scVital identifies a previously unknown cell state that persists after chemotherapy and is shared by a GEMM and human patient-derived xenografts. These findings establish the utility of scVital in identifying conserved cell states across species to enhance the translational capabilities of mouse models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90f7ecfe362d789c123b157e906f4a995c4f86a3" target='_blank'>
              A deep-learning tool for species-agnostic integration of cancer cell states
              </a>
            </td>
          <td>
            Jonathan Rub, Jason E Chan, Carleigh Sussman, W. Tap, Samuel Singer, Tuomas Tammela, D. Betel
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="DICER1 syndrome predisposes children and young adults to tumor development across various organs. Many of these cancers are sarcomas, which uniquely express the RNase IIIb domain-deficient form of DICER1 and exhibit consistent histological and molecular similarities regardless of their anatomical origins. To uncover their cellular origin and developmental hierarchy, we established a lineage-traceable genetically engineered mouse model that allows for precise activation of Dicer1 mutations in Hic1+ mesenchymal stromal cells. This model resulted in the development of renal tumors closely mirroring human DICER1 sarcoma histologically and molecularly. Single-cell transcriptomics coupled with targeted spatial gene expression analysis revealed a Hic1+ progenitor population marked by Pdgfra, Dpt, and Mfap4, corresponding to universal fibroblasts of steady-state kidneys. These fibroblastic progenitors exhibit the capacity to undergo rhabdomyoblastic differentiation or transition to highly proliferative anaplastic sarcoma. Investigation of patient samples identified analogous cell states. This study uncovers a fibroblastic origin for DICER1 sarcoma and provides a faithful model for mechanistic investigation and therapeutic development for tumors within the rhabdomyosarcoma spectrum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7612b5f96fea7c628834a8ae0728263c8ee1372" target='_blank'>
              Spatial single cell transcriptomic analysis of a novel DICER1 Syndrome GEMM informs the cellular origin and developmental hierarchy of associated sarcomas
              </a>
            </td>
          <td>
            F. Kommoss, Joyce Zhang, Branden Lynch, S. Chen, J. Senz, Yana Moscovitz, Lesley A. Hill, Wilder Scott, Jonathan Bush, Kenneth S. Chen, Andreas von Deimling, William D. Foulkes, Gregg B. Morin, T. M. Underhill, Yemin Wang, D. Huntsman
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d97774290c1f515233beb8849c220d76ed9e57" target='_blank'>
              Unsupervised clustering reveals noncanonical myeloid cell subsets in the brain tumor microenvironment
              </a>
            </td>
          <td>
            Ismaïl Hermelo, Tuomo Virtanen, I. Salonen, R. Nätkin, Sofia Keitaanniemi, A. Tiihonen, Suvi Lehtipuro, Laura Kummola, Ella Raulamo, K. Nordfors, H. Haapasalo, M. Rauhala, J. Kesseli, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2025-01-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90bace8a9791e95b3eff7c16fd892d053802f89b" target='_blank'>
              Single-cell transcriptomes of dissecting the intra-tumoral heterogeneity of breast cancer microenvironment
              </a>
            </td>
          <td>
            Peixian Chen, Kaifeng Liang, Xiaofan Mao, Qiuyuan Wu, Zhiyan Chen, Yabin Jin, Kairong Lin, Tiancheng He, Shuqing Yang, Huiqi Huang, Guolin Ye, Juntao Gao, Dan Zhou, Zhihao Zeng
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Tumor morphological features from histology images are a cornerstone of clinical pathology, diagnostic biomarkers, and basic cancer biology research. Spatial transcriptomics, which provides spatially resolved gene expression profiles overlaid on histology images, offers a unique opportunity to integrate morphological and expression features, thereby deepening our understanding of tumor biology. However, spatial transcriptomics experiments with patient samples in either clinical trials or clinical care are costly and challenging, whereas histology images are generated routinely and available for many legacy prospective cohorts of disease progression and outcomes in well-annotated cohorts. Inferring spatial transcriptomics profiles computationally from these histology images would significantly expand our understanding of tumor biology, but paired data for training multi-modal spatial-histology models remains limited. Here, we tackle this challenge by incorporating performant foundation models pre-trained on massive datasets of pathology images and single-cell RNA-Seq, respectively, which provide useful embeddings to underpin multi-modal models. To this end, we developed PathOmCLIP, a model trained with contrastive loss to create a joint-embedding space between a histopathology foundation model and a single-cell RNA-seq foundation model. We incorporate a set transformer to gather localized neighborhood tumor architecture following contrastive training, which further enhances performance and is necessary to obtain robust results. We validate PathOmCLIP across five tumor types and achieve significant performance improvements in gene expression prediction tasks over other methods. PathOmCLIP can be applied to many archived histology images, unlocking valuable clinical information and facilitating new biomarker discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0858f170113a2543c26ceee12ee7ba475e2efa6" target='_blank'>
              PathOmCLIP: Connecting tumor histology with spatial gene expression via locally enhanced contrastive learning of Pathology and Single-cell foundation model
              </a>
            </td>
          <td>
            Yongju Lee, Xinhao Liu, Minsheng Hao, Tianyu Liu, Aviv Regev
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Human lung is a complex organ susceptible to various diseases. Single-cell transcriptomic studies provide rich data to targeting specific research questions. Here, we present uniLUNG, the largest lung transcriptomic cell atlas, comprising over 10 million cells across 20 disease states and healthy controls. We ensembled a universal hierarchical annotation framework and conducted a full benchmarking of data integration to define a standardized nomenclature and marker genes for lung cell types. Using uniLUNG, we identified Lym-monocyte and T-like B cells, new cell types in specific lung diseases, confirming their existence by comparing with external single-cell atlases. Additionally, we discovered the NSCLC-like SCLC subpopulation, a transitional malignant cell population associated with the transition from NSCLC to SCLC, which was validated and further characterized in spatial dimensions, revealing its complex role in tumour progression. Overall, uniLUNG represents a comprehensive range of human lung cell diversity, providing valuable data resources and a reliable foundation for lung single-cell research. HIGHLIGHTS The largest scRNA atlas for human lung covers 10 million cells from 20 lung states. A four-level universal cell annotation framework encompasses 120 lung cell types. Comprehensive benchmarking on 18 strategies guides data integration. Specific distribution of Lym-monocytes and T-like B cells in specific lung diseases. The NSCLC-like SCLC subpopulation in transitional events of malignant cells from NSCLC to SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707c7b05c0b869d7c0046d311738148202a1849b" target='_blank'>
              A universal single-cell transcriptomics atlas of human lung decodes multiple pulmonary diseases
              </a>
            </td>
          <td>
            Fanjie Wu, Wenhao Cai, Hai Tang, Shikang Zheng, Haiyue Zhang, Yixin Chen, Yutong Han, Dingli Zhou, Ruihan Wang, Mingli Ye, Renke You, Amin Chen, Jiaqi Li, Xuegong Zhang, Weizhong Li
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The ability of cancer cells to alter their identity, known as lineage plasticity, is crucial for tumor progression and therapy resistance. In lung adenocarcinoma (LUAD), tumor progression is characterized by a gradual loss of lineage fidelity and the emergence of non-pulmonary identity programs. This can lead to hybrid-identity (hybrid-ID) states in which developmentally incompatible identity programs are co-activated within individual cells. However, the molecular mechanisms underlying these identity shifts remain incompletely understood. Here, we identify the gastrointestinal (GI) transcriptional regulator HNF4α as a critical driver of tumor growth and proliferation in KRAS-driven LUAD. In LUAD cells that express the lung lineage specifier NKX2-1, HNF4α can induce a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also forms an aberrant protein complex with NKX2-1, which disrupts NKX2-1 localization and dampens pulmonary identity within hybrid-ID LUAD. Sustained signaling through the RAS/MEK pathway is critical for maintaining the hybrid-ID state. Moreover, RAS/MEK inhibition augments NKX2-1 chromatin binding at pulmonary-specific genes and induces resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion enhances sensitivity to pharmacologic KRASG12D inhibition. Collectively, our data show that co-expression of opposing lineage specifiers leads to a hybrid identity state that can drive tumor progression and dictate response to targeted therapy in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207f2a257f041c211d79bb19f39ca01a49097c42" target='_blank'>
              Opposing lineage specifiers induce a pro-tumor hybrid-identity state in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Rushmeen Tariq, Anna Waters, Kayla T. O'Toole, Eric L. Snyder
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma (GBM) cells are highly heterogeneous and invasive, leading to treatment resistance and relapse. However, the molecular regulation in and distal to tumors remains elusive. Here, we collected paired tissues from the tumor core (TC) and peritumoral brain (PTB) for integrated snRNA-seq and snATAC-seq analyses. Tumor cells infiltrating PTB from TC behave more like oligodendrocyte progenitor cells than astrocytes at the transcriptome level. Dual-omics analyses further suggest that the distal regulatory regions in the tumor genome and specific transcription factors are potential determinants of regional heterogeneity. Notably, while activator protein 1 (AP-1) is active in all GBM states, its activity declines from TC to PTB, with another transcription factor, BACH1, showing the opposite trend. Combined inhibition of AP-1 and BACH1 more efficiently attenuates the tumor progression in mice and prolongs survival than either single-target treatment. Together, our work reveals marked molecular alterations of infiltrated GBM cells and a synergy of combination therapy targeting intratumor heterogeneity in and distal to GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c193d462d0c2dbf4aee7a761498e3e53b84808" target='_blank'>
              Single-cell multi-omics sequencing uncovers region-specific plasticity of glioblastoma for complementary therapeutic targeting
              </a>
            </td>
          <td>
            Xin Wang, Qian Sun, Tianbin Liu, Haoran Lu, Xuyi Lin, Weiwen Wang, Yang Liu, Yunting Huang, Guodong Huang, Haixi Sun, Qianxue Chen, Junmin Wang, Daofeng Tian, Fan-En Yuan, Longqi Liu, Bo Wang, Ying Gu, Baohui Liu, Liang Chen
          </td>
          <td>2024-11-22</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The renal vasculature consists of highly specialized blood vessels that carry out distinct physiological functions. Defining their transcriptional signatures and tracing their developmental ontogeny has thus far been challenging due to a lack of regionally specific endothelial biomarkers. Here, we performed single nuclear RNA sequencing (snucRNA-Seq) to interrogate embryonic renal endothelial cells. We identified nine different endothelial cell types, and validated expression of novel candidates using multiplex RNAScope. We interrogate biological characteristics, including cell cycle stage, of developmental blood vessels with a variety of in vivo tools. Using genetic tools based on expression of tip cell marker Esm1, we performed detailed lineage tracing, demonstrating multi-clonal and multi-directional endothelial contribution to glomerular vasculature. Together, this study provides the first validated, tool-focused developmental atlas of the murine renal vasculature and elucidates several novel cellular mechanisms of nephron vascularization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e404560bedd479bc032952e4bd3bd630933dfdc5" target='_blank'>
              Zonal endothelial cell heterogeneity underlies murine renal vascular development
              </a>
            </td>
          <td>
            Peter M. Luo, Neha Ahuja, Christopher P. Chaney, Danielle Pi, Aleksandra Cwiek, Zaneta Markowska, C. Hiremath, Denise Marciano, Karen K. Hirschi, M. L. Iruela-Arispe, Thomas J. Carroll, Ondine Cleaver
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCCs) constitute one of the few cancer indications for which mortality rates continue to rise. While Notch signaling dictates a key progenitor lineage choice during development, its role in HCC has remained controversial. Using therapeutic antibodies targeting Notch ligands and receptors to screen over 40 patient-derived xenograft models, we here identify progenitor-like HCCs that crucially depend on a tumor-intrinsic JAG1-NOTCH2 signal. Inhibiting this signal induces tumor regressions by triggering progenitor-to-hepatocyte differentiation, the same cell fate-switch that Notch controls during development. Transcriptomic analysis places the responsive tumors within the well-characterized progenitor subclass, a poor prognostic group of highly proliferative tumors, providing a diagnostic method to enrich for Notch-dependent HCCs. Furthermore, single-cell RNA sequencing uncovers a heterogeneous population of tumor cells and reveals how Notch inhibition shifts cells from a mixed cholangiocyte-hepatocyte lineage to one resembling mature hepatocytes. Analyzing the underlying transcriptional programs brings molecular detail to this process by showing that Notch inhibition de-represses expression of CEBPA, which enables the activity of HNF4α, a hepatocyte lineage factor that is otherwise quiescent. We thus describe a compelling and targetable dependency in a poor-prognosis class of HCCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3764bfcf8bf124425bfb62c588b7e271cb0186df" target='_blank'>
              Notch signaling maintains a progenitor-like subclass of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kerstin Seidel, Robert Piskol, Thi Thu Thao Nguyen, Amy Shelton, Charisa L. Cottonham, Cecile C. de la Cruz, Joseph Castillo, Jesse Garcia, Udi Segal, Mark Merchant, Yeqing Angela Yang, Jasmine Chen, Musa Ahmed, Alexis Scherl, Rajesh Vij, Lluc Mosteiro, Yan Wu, Z. Modrusan, C. Metcalfe, Christopher W. Siebel
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Exosomes, functional biomarkers involved in cancer progression, have gained widespread attention for promoting tumor formation, growth, and metastasis. Current bulk exosome detections in bodily fluids enable cancer functional analysis, but average secretion levels from cell populations, losing parent cell information and ignoring exosome heterogeneity from diverse cell subgroups, necessitating an effective platform for analyzing single-cell exosome functional heterogeneity. Here, a high-throughput platform is presented, capable of efficient single-cell isolation and multi-color exosome phenotype analysis, as well as quantifying trace exosomes secreted by single cells. Photothermal-driven single-cell chips achieve significant single-cell isolation efficiency (≈97%) within 5 min, facilitating the ultra-high throughput single-cell exosome analysis. By conducting mass spectrometry and protein interaction of breast cancer exosome phenotypic proteins, key exosome phenotypes are identified. Tens of thousands of single cells from breast cancer cell lines, and clinical tissues are analyzed, revealing various subgroup differences. The study finds more CD44 and EGFR co-expressing exosome subgroups in breast cancer cell lines, while immune-evasion PD-L1 high-phenotype exosome subgroups are primarily presented in complex tumor microenvironments, especially in HER2-positive tissues. This platform offers powerful single-cell isolation, exosome quantification, and phenotypic analysis capabilities, making it a powerful tool for advancing single-cell exosome analysis in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e3aa2959506a656b04144e1a13a4a7656680a8" target='_blank'>
              Single-Cell Isolation Chip Integrated with Multicolor Barcode Array for High-Throughput Single-Cell Exosome Profiling in Tissue Samples.
              </a>
            </td>
          <td>
            Chao Wang, Yu Zhang, Jianbo Wang, Yunrui Han, Yihe Wang, Mingyuan Sun, Yanbo Liang, Miao Huang, Yang Yu, Huili Hu, Hong Liu, Lin Han
          </td>
          <td>2024-12-10</td>
          <td>Advanced materials</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Spatial transcriptomics offers unprecedented insights into the complex cellular landscapes of tissues, particularly in cancer research where understanding the tumor microenvironment is crucial. However, interpreting the vast and intricate data generated by this technology remains a significant challenge. This study explores the potential of Large Language Models (LLMs) to assist in the analysis and interpretation of spatial transcriptomic data from a murine melanoma tumor model. We first evaluated the performance of multiple LLM models in describing and quantifying spatial gene expression patterns. Our benchmarking revealed that spatial transcriptomics data interpretation proved challenging for most models, with only a few demonstrating sufficient capability for this complex task. Using Claude 3.5 Sonnet, which showed the highest accuracy in spot quantification and pattern recognition, we developed a systematic workflow for analyzing the tumor immune landscape. The model first assisted in identifying and prioritizing panels of M1 and M2 macrophage-associated markers through knowledge-driven scoring. It then demonstrated remarkable ability to integrate spatial expression data with extensive immunological knowledge, providing sophisticated interpretation of local immune organization. When analyzing individual tumor regions, the model identified coordinated immunosuppressive mechanisms including metabolic barriers and disrupted pro-inflammatory signaling cascades, findings that both aligned with and extended current understanding of tumor immunology. This study highlights the potential of LLMs as powerful assistive tools in spatial transcriptomics analysis, capable of combining advanced pattern recognition with extensive knowledge integration to enhance data interpretation. While significant development work remains to make such workflows scalable, our proof of concept demonstrates that LLMs can help accelerate the translation of spatial transcriptomics data into biological insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287bc0c305d5950e4e6f2901503d04b32465b5e" target='_blank'>
              Tackling the Complexity of Spatial Transcriptomics Data Interpretation with Large Language Models
              </a>
            </td>
          <td>
            Taushif Khan, Colleen M. Farley, John J. Wilson, Chih-Hao Chang, Damien Chaussabel
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cellular senescence features a durable exit from the cell cycle triggered by stress or carcinogens. The INK4 locus is inactivated in various cancers, yet in senescence, p16Ink4a is activated. Whether senescence is tumor-suppressing or -promoting remains a conundrum. We discovered an evolutionally-conserved Vertebrata INK4-homolog. This ink4ab triggers senescence upon oxidative- and/or carcinogenic-stress. Adult Ink4ab-deficient animals failed to activate senescence and developed spontaneous cancers. Combined Ink4ab and Tp53 deficiency revealed a reciprocal senescence and apoptosis regulation, controlling tumorigenesis, including retinoblastoma. INK4-hematopoietic-deficient mice exhibited p19Arf-dependent enhanced senescence-like phenotypes, uncontrolled cell proliferation, defective stem cell differentiation, and splenomegaly, with single-splenocytes spatially-enriched in senescence-associated secretory profiles. Our studies reveal the evolutionary origin of paradigms co-regulating senescence and tumor suppression and offer strategies to exploit these reciprocal pathways for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9243e085293669a418053332a11c9df19226137a" target='_blank'>
              Evolutionary conserved reciprocal senescence and tumor suppressor signals limit lifetime cancer
              </a>
            </td>
          <td>
            Liqiong Liu, Stephani Davis, Shamila Yusuff, Ann Strange, Yonghua Zhuang, Prasanna Vaddi, Kathleen Flaherty, Kelly Jara, Joseph Kramer, Christine Archer, Schuyler Lee, Bifeng Gao, A. van Bokhoven, Wei Wang, Sharon R. Pine, Tetsuya Nakamura, Hatim E. Sabaawy
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Low-grade endometrial stromal sarcoma (LG-ESS) is a rare uterine malignancy characterized by its complex tumor microenvironment (TME) and high recurrence rates, posing challenges to accurate prognosis and effective treatment. Identifying prognostic biomarkers is essential for improving patient stratification and guiding therapeutic strategies. Methods Using single-cell transcriptome analysis combined with H&E and multiplex immunofluorescence staining, we identified a subpopulation of tumor cells in LG-ESS and further validated the association of this subpopulation and its characteristic genes with LG-ESS prognosis by molecular characterization and bulk transcriptome data. Results Our analysis reveals multiple cellular subpopulations within the tumor tissue, particularly a tumor cell subpopulation among them which is associated with poor prognosis. Originating from normal stromal fibroblasts, this subpopulation appears to play a crucial role in TME remodeling, smooth muscle cell behavior, and potentially in tumorigenesis and metastasis. Of particular interest in this subpopulation is the highly expressed FGF12 gene, which is significantly associated with a shortened survival in ESS, highlighting its potential as a prognostic biomarker. Conclusion Our study reveals the complexity of TME within the LG-ESS and highlights the role that tumor cell subpopulations play in disease progression and patient prognosis. The identification of FGF12 as a prognostic biomarker suggests a new approach for the personalized treatment and prognosis monitoring of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e49616fef8441ab9c968f5c960e3e63e7204d31e" target='_blank'>
              Single-cell RNA-seq reveals FGF12 as a prognostic biomarker in low-grade endometrial stromal sarcoma
              </a>
            </td>
          <td>
            Yu Miao, Meng Dong, Qiyin Zhou, Julia Thiel, Na Li, Ying Cai, Dan Yuan, Haitao Wang, Su-Han Jin, Hua Yang, Jinjing Wang, Benjamin Frey, Udo S. Gaipl, Hu Ma, Jianghu Zhou
          </td>
          <td>2024-11-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Studying the spatiotemporal dynamics of cells in living organisms is a current frontier in bioimaging. Intravital Microscopy (IVM) provides direct, long-term observation of cell behavior in living animals, from tissue to sub-cellular resolution. Hence, IVM has become crucial for studying complex biological processes in motion and across scales, such as the immune response to pathogens and cancer. However, IVM data are typically kept in private repositories inaccessible to the scientific community, hampering large-scale analysis that aggregates data from multiple laboratories. To solve this issue, we introduce Immunemap, an atlas of immune cell motility based on an Open Data platform that provides access to over 58’000 single-cell tracks and 1’049’000 cell-centroid annotations from 360 videos in murine models. Leveraging Immunemap and unsupervised learning, we systematically analyzed cell trajectories, identifying four main patterns of cell migration in immune cells. Two patterns correspond to behaviors previously characterized: directed movement and arresting. However, we identified two other patterns, characterized by low directionality and twisted paths, often considered random migration. We show that the newly defined patterns can be subdivided into two distinct types: within small areas, suggesting a focused patrolling around one or a few cells, and over larger areas, indicative of a more extended tissue patrolling. Furthermore, we show that the percentage of cells displaying these motility patterns changes in response to immune stimuli. Altogether, Immunemap embraces the FAIR principles, promoting data reuse to extract novel insights from immune cell dynamics through an image-based systems biology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fcec1b17c98e94c83b787c02db01f7ce0422437" target='_blank'>
              Systematic analysis of immune cell motility leveraging Immunemap, an open intravital microscopy atlas
              </a>
            </td>
          <td>
            D. U. Pizzagalli, Pau Carrillo-Barberà, Elisa Palladino, Kevin Ceni, Benedikt Thelen, Alain Pulfer, Enrico Moscatello, R. F. Cabini, Johannes Textor, Inge M. N. Wortel, Rolf Krause, S. F. Gonzalez
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Computerized chest tomography (CT)-guided screening in populations at risk for lung cancer has increased the detection of preinvasive subsolid nodules, which progress to solid invasive adenocarcinoma. Despite the clinical significance, there is a lack of effective therapies for intercepting the progression of preinvasive to invasive adenocarcinoma. To uncover determinants of early disease emergence and progression, we used integrated single-cell approaches, including scRNA-seq, multiplexed imaging mass cytometry and spatial transcriptomics, to construct the first high-resolution map of the composition, lineage/functional states, developmental trajectories and multicellular crosstalk networks from microdissected non-solid (preinvasive) and solid compartments (invasive) of individual part-solid nodules. We found that early disease initiation and subsequent progression are associated with the evolution of immune-suppressive cellular phenotypes characterized by decreased cytotoxic CD8 T and NK cells, increased T cell exhaustion and accumulation of immunosuppressive regulatory T cells (Tregs) and M2-like macrophages expressing TREM2. Within Tregs, we identified a unique population of 4-1BB+ Treg subset enriched for the IL2-STAT5 suppressive pathway with transcription profiles supporting discrete metabolic alterations. Spatial analysis showed increased density of suppressive immune cells around tumor cells, increased exhaustion phenotype of both CD4 and CD8 T cells expressing chemokine CXCL13, and spatial micro-complex of endothelial and lymphocyte interactions within tertiary lymphoid structures. The single-cell architecture identifies determinants of early disease emergence and progression, which may be developed not only as diagnostic/prognostic biomarkers but also as targets for disease interception. Additionally, our dataset constitutes a valuable resource for the preinvasive lung cancer research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08ee8028223311574c20ea0561992916e40482b" target='_blank'>
              Acquisition of discrete immune suppressive barriers contributes to the initiation and progression of preinvasive to invasive human lung cancer
              </a>
            </td>
          <td>
            L. Yoffe, B. Bhinder, Sung Wook Kang, Haoran Zhang, Arshdeep Singh, H. Ravichandran, G. Markowitz, Mitchell T. Martin, Junbum Kim, Chen Zhang, O. Elemento, Wesley Tansey, Stewart Bates, T. McGraw, Alain Borczuk, Hyun-Sung Lee, Nasser K Altorki, Vivek Mittal
          </td>
          <td>2025-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background The adherent-to-suspension transition (AST) describes how solid tumor cells reprograms their anchorage dependency via defined hematopoietic transcriptional regulators to disseminate into circulating tumor cells (CTCs). It remains unclear, however, whether the dissemination and colonization processes require dynamic plasticity of AST factor expression in CTCs to permit the completion of the metastatic cascade. Methods We investigated AST factor-mediated adherent-suspension plasticity (ASP) in a newly developed cell-based dissemination assay. Here we identified a rare population of CTC-like cells spontaneously suspended from highly confluent breast cancer cells under chronic stress conditions that mimic the tumor microenvironment (TME). We investigated the oscillatory dynamics of AST factor expression in these CTC-like cells and their role in dissemination and colonization. Furthermore, we performed a single-cell transcriptomic analysis of matched primary tumors, CTCs, and metastatic lesions from newly enrolled de novo metastatic breast cancer patients, focusing on the plasticity of AST factor expression during the metastatic cascade in vivo. Results Using data obtained from the dissemination assay and de novo metastatic breast cancer patients, we have demonstrated the dynamic regulation of AST factor expression during the dissemination and colonization of CTCs both in vitro and in vivo. From our dissemination assay of breast cancer cell lines, we show that AST factor induction is required for the reprogramming of anchorage dependency and spontaneous detachment of CTC-like cells. We also show that subsequent AST factor suppression is required for the reattachment of CTCs during colonization. Single-cell transcriptome analyses in matched specimens of primary tumor, CTC, and metastatic lesions obtained from de novo metastatic breast cancer patients confirmed the oscillatory dynamics of AST factors and related gene signatures related to the adherent-suspension plasticity. Conclusion Our results show that the reprogramming of anchorage dependency in solid tumor cells via AST factor-mediated ASP plays a critical role in the dissemination and colonization of CTCs. These findings highlight the potential of targeting AST factors to develop effective anti-metastatic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24dbe2635c48e91654f5538698e4e2b76cbd216e" target='_blank'>
              Adherent-suspension plasticity promotes the dissemination and colonization of circulating tumor cells
              </a>
            </td>
          <td>
            Hyunbin D. Huh, Yujin Sub, Jee Hung Kim, Hannah Lee, Jongwook Oh, Dong Ki Lee, Jong Hwa Byun, Woo Yang Pyun, S. Bae, Sehyung Pak, Hai-Chon Lee, Yu Suk Choi, Junjeong Choi, Joon Jeong, H. Gee, Hyun Woo Park
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Tissue regeneration relies on the ability of cells to undergo de novo patterning. While tissue patterning has been viewed as the transition from initially identical un-patterned cells to an arrangement of different cell types, recent evidence suggests that initial heterogeneities between cells modulate tissue-scale pattern formation. Yet, how such heterogeneities arise and, thereafter, regulate cell type emergence in a population of cells is poorly understood. Using in vivo and in vitro mouse regenerative systems, we identify a critical tissue density that is required to induce heterogeneous inactivation of the mechanosensor YAP1. Experimental and biophysical approaches demonstrate that YAP1 cell-to-cell heterogeneity pre-patterns the first cell fate decision, via both chromatin remodelling and a supracellular feedback between FOXA1 and Delta-Notch signalling. This feedback motif induces cell fate bistability endowing memory to the system and the maintenance of patterns during homeostasis. These findings reveal a generalisable framework in which transient cell-to-cell heterogeneity, regulated by tissue-scale properties, serves as a critical control parameter for the emergence of cell fate and stable patterning during regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4e71565210ea1e9031332a79fd8f2875c11cbbd" target='_blank'>
              Cell heterogeneity and fate bistability drive tissue patterning during intestinal regeneration
              </a>
            </td>
          <td>
            C. Schwayer, S. Barbiero, D. B. Brückner, C. Baader, N. A. Repina, O. E. Diaz, L. C. Meylan, V. Kalck, S. Suppinger, Q. Yang, J. Schnabl, U. Kilik, J. G. Camp, B. Stockinger, M. Bühler, M. B. Stadler, É. Hannezo, P. Liberali
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332b50c1b783afb4c65a813ff9aedb52f38aaf88" target='_blank'>
              Spatial tumor immune heterogeneity facilitates subtype co-existence and therapy response in pancreatic cancer
              </a>
            </td>
          <td>
            Lukas Klein, M. Tu, Niklas Krebs, Laura Urbach, Daniela Grimm, Muhammad Umair Latif, Frederike Penz, Anna Blandau, Xueyan Wu, Rebecca Diya Samuel, Stefan Küffer, F. Wegwitz, N. Chan, Kazeera Aliar, F. Vyas, Uday Kishore, E. Hessmann, Andreas Trumpp, Elisa Espinet, A. Papantonis, Rama Khokha, V. Ellenrieder, Barbara T. Grünwald, Shiv K. Singh
          </td>
          <td>2025-01-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Understanding the phenotypic transitions of cancer cells is crucial for elucidating tumor progression mechanisms, particularly the transition from a non-invasive spheroid phenotype to an invasive network phenotype. We developed an agent-based model (ABM) using Compucell3D, an open-source biological simulation software, to investigate how varying biophysical and biochemical parameters influence emerging properties of cellular communities, including cell growth, division, and migration. Our focus was on cell-cell contact adhesion and matrix remodeling effects on cancer cell migration. We simplified enzymatic remodeling of the extracellular matrix and the subsequent enhancements to cellular chemotaxis or durotaxis as a combined effect of localized cellular secretion of a chemoattractant. By varying the chemoattractant secretion rate and contact adhesion energy, we simulated their effects on cellular behavior and driving the transition from a spheroid phenotype to a network phenotype. The model serves as a digital twin for 3D cancer cell culture, simulating cancer cell growth, division, and invasion over 1 week, validated against published data. The simulations track the emergent morphological and collective phenotype changes using key metrics such as cell circularity and invasion. Our findings indicate that increased chemoattractant secretion enhances the invasiveness of the collective cells, promoting the transition to a network phenotype. Additionally, changing cell-cell contact energy from a strong cell-cell adhesion to a weak cell-cell adhesion affects the compactness of the spheroids, resulting in lower circularity and increased collective cell invasion. Our work advances the understanding of tumor progression by providing insights into the biophysical mechanisms behind invasive cancer cell phenotypic transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6e2c13239072839fce8a0eaeadba76fadb7701" target='_blank'>
              Agent-based modeling reveals impacts of cell adhesion and matrix remodeling on cancer collective cell migration phenotypes
              </a>
            </td>
          <td>
            Temitope O. Benson, Mohammad Aminul Islam, Kailei Liu, Ashlee N. Ford Versypt
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The development of single-cell RNA-sequencing assays has transformed our understanding of cellular and tissue heterogeneity, yielding significant insights into disease biology and its underlying mechanisms. In this work, we describe icCITE-seq (intracellular cellular indexing of transcriptomes and epitopes), a scalable method that simultaneously measures surface and intracellular protein levels alongside gene expression across thousands of cells. We validate the specificity of intracellular staining and demonstrate the utility of this multi-omic approach in interrogating phenotypic cellular states through targeted genetic perturbations in primary human T cells. icCITE-seq enables systematic profiling of gene expression, coupled with cytoplasmic, nuclear and PTM epitopes, providing an integrated approach towards understanding cellular identity, complexity and disease regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d61b1b347f1ffdb53994830ef412c17da00b40" target='_blank'>
              Joint single-cell measurements of surface proteins, intracellular proteins and gene expression with icCITE-seq
              </a>
            </td>
          <td>
            Kelvin Y. Chen, Yusuke Takeshima, A. L. Zorzetto-Fernandes, Katsuhiro Makino, Tatsuya Kibayashi, Mayu Hata, Kenji Ichiyama, Josh Croteau, Kevin Taylor, Shimon Sakaguchi
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Identifying spatial domains is critical for understanding breast cancer tissue heterogeneity and providing insights into tumor progression. However, dropout events introduces computational challenges and the lack of transparency in methods such as graph neural networks limits their interpretability. This study aimed to decipher disease progression-related spatial domains in breast cancer spatial transcriptomics by developing the three graph regularized non-negative matrix factorization (TGR-NMF). A unitization strategy was proposed to mitigate the impact of dropout events on the computational process, enabling utilization of the complete gene expression count data. By integrating one gene expression neighbor topology and two spatial position neighbor topologies, TGR-NMF was developed for constructing an interpretable low-dimensional representation of spatial transcriptomic data. The progressive lesion area that can reveal the progression of breast cancer was uncovered through heterogeneity analysis. Moreover, several related pathogenic genes and signal pathways on this area were identified by using gene enrichment and cell communication analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744a8472d128321ec52aed7c2a2480cf5c10946b" target='_blank'>
              Deciphering progressive lesion areas in breast cancer spatial transcriptomics via TGR-NMF
              </a>
            </td>
          <td>
            Juntao Li, Shan Xiang, Dongqing Wei
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. A better understanding of the temporal dynamics and specific pathways leading into and out of the persister state is needed to identify strategies to prevent treatment failure. Using spatial transcriptomics in patient-derived xenograft models, we captured clonal lineage evolution during treatment. The persister state showed increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients. Phylogenetic tracing identified intrinsic and acquired resistance mechanisms (e.g., dual specific phosphatases, reticulon-4, and CDK2) and suggested specific temporal windows of potential therapeutic susceptibility. Deep learning-enabled analysis of histopathological slides revealed morphological features correlating with specific cell states, demonstrating that juxtaposition of transcriptomics and histological data enabled identification of phenotypically distinct populations from using imaging data alone. In summary, this study defined state change and lineage selection during melanoma treatment with spatiotemporal resolution, elucidating how choice and timing of therapeutic agents will impact the ability to eradicate resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d694b41718972dca1b5b8aa26a4498984d65d307" target='_blank'>
              Spatiotemporal Profiling Defines Persistence and Resistance Dynamics During Targeted Treatment of Melanoma.
              </a>
            </td>
          <td>
            Jill C. Rubinstein, Sergii Domanskyi, Todd B Sheridan, Brian Sanderson, SungHee Park, Jessica Kaster, Haiyin Li, Olga Anczuków, M. Herlyn, Jeffrey H. Chuang
          </td>
          <td>2024-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Background Thymomas are malignancies originating from the thymus, the primary lymphoid organ crucial for T cell development and selection. Normal thymopoiesis relies on physical and functional interactions between T-cell precursors and thymic stromal cells. Abnormalities in the frequency of T-cell developmental stages within dissociated thymoma tissue suggest that the tumour microenvironment (TME) substantially impacts thymopoiesis. Yet a detailed understanding of the precise cellular and spatial irregularities of thymoma is incomplete. This study aims to map the thymoma TME to analyse the cellular organization and interactions associated with T-cell development in the tissue. Methods Multiplexed immunofluorescence imaging detecting 28 distinct epitopes was employed to identify lymphoid and stromal cells in type B2 thymoma tissue and healthy human thymus of young and adult individuals. A computational pipeline for spatial profiling of microenvironments was developed that includes the accurate segmentation of complex-shaped stromal cells and the quantitative and qualitative analysis of cellular neighbourhoods and interactions. This pipeline was designed and validated to allow comparison of extracted spatial proteomic data across multiple tissue groups. Results Simultaneous characterisation of 57 cell-types and states was achieved for T-cells at various developmental stages, epithelial cells, hematopoietic antigen-presenting cells, and non-epithelial stromal cells. Detailed analyses of healthy tissue have been conducted on the cell-type composition and density, parenchymal distribution, tissue organization, and cell-cell interactions relevant to normal thymopoiesis. The density of T-cells as well as composition and protein expression of stromal populations in the TME substantially deviates from that observed in healthy tissue, particularly in the cortical compartments. Novel tissue niches have been identified within thymic histological compartments for both healthy and thymoma tissue, where well-defined regions of healthy thymi are either absent or reduced, in the TME, and where abnormalities in cellular interactions have been observed. Conclusions Our study established analytical methods allowing the mapping of type B2 thymoma TME at single-cell resolution as well as for high-throughput and in-depth analysis of spatial proteomic datasets. This investigation reveals region-specific spatial irregularities and abnormal cellular interactions within novel functional niches in the TME, contributing to an understanding of abnormalities of T-cell development in thymomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da72056429c6ea9f103a488ee050b57b4f824034" target='_blank'>
              AB056. Spatial analysis of T-cell development in type B2 thymoma tumour microenvironment at single-cell resolution
              </a>
            </td>
          <td>
            Kirolous S. Hanna, Fatima Dhalla, Andreas Tarcevski, M. Deadman, Pietro Zara, Bernard Liem, Rhona Taberham, Antonios Kourliouros, Dionisios Stavroulias, Adam E. Handel, M. I. Leite, Georg A. Hollander
          </td>
          <td>2024-12-01</td>
          <td>Mediastinum</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af37e793e7291e5ec4163cfb7481755bd374769" target='_blank'>
              Spatial transcriptomics reveals unique metabolic profile and key oncogenic regulators of cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Limin Zhou, Jiejie Liu, Peipei Yao, Xing Liu, Fei Chen, Yu Chen, Li Zhou, Chao Shen, You Zhou, Xin Du, Junbo Hu
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare and aggressive malignancy with limited treatment options, necessitating comprehensive molecular characterization to uncover actionable therapeutic targets. Here, we integrate bulk and single-cell transcriptomics, epigenomics, and DNA methylation profiling to construct a detailed molecular and cellular landscape of SNSCC. Our analysis identifies five transcriptionally and epigenetically distinct malignant subpopulations, including hypoxic (TC1) and proliferative (TC2) clusters, with TC1 showing significant association with poor clinical outcomes. We uncover a hypoxia-driven angiogenic axis, wherein TC1 cells secrete adrenomedullin (ADM), MIF, and VEGFA to activate endothelial tip cells, fostering tumor angiogenesis. Epigenetic reprogramming, characterized by alterations in DNA methylation and chromatin accessibility, underpins these transcriptional programs, with AP-1 transcription factors emerging as key regulators. Notably, ADM expression is epigenetically controlled, correlates with advanced clinical stage, and predicts reduced survival in head and neck cancer patients. This multi-omic study highlights the tumor heterogeneity and microenvironmental interactions driving SNSCC progression, revealing epigenetically regulated pathways as promising therapeutic targets for future intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f1bcee9ecccbc4aae5987d7312ce58acb94e80" target='_blank'>
              Multi-omic profiling identifies malignant subpopulations and a hypoxia-driven angiogenic axis as therapeutic vulnerabilities in sinonasal squamous cell carcinoma
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, Myeong S. Yu, Yoo-Sam Chung, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Recent developments in single-cell multi-omics technologies have provided the ability to identify diverse cell types and decipher key components of the tumor microenvironment (TME), leading to important advancements toward a much deeper understanding of how tumor microenvironment heterogeneity contributes to cancer progression and therapeutic resistance. These technologies are able to integrate data from molecular genomic, transcriptomic, proteomics, and metabolomics studies of cells at a single-cell resolution scale that give rise to the full cellular and molecular complexity in the TME. Understanding the complex and sometimes reciprocal relationships among cancer cells, CAFs, immune cells, and ECs has led to novel insights into their immense heterogeneity in functions, which can have important consequences on tumor behavior. In-depth studies have uncovered immune evasion mechanisms, including the exhaustion of T cells and metabolic reprogramming in response to hypoxia from cancer cells. Single-cell multi-omics also revealed resistance mechanisms, such as stromal cell-secreted factors and physical barriers in the extracellular matrix. Future studies examining specific metabolic pathways and targeting approaches to reduce the heterogeneity in the TME will likely lead to better outcomes with immunotherapies, drug delivery, etc., for cancer treatments. Future studies will incorporate multi-omics data, spatial relationships in tumor micro-environments, and their translation into personalized cancer therapies. This review emphasizes how single-cell multi-omics can provide insights into the cellular and molecular heterogeneity of the TME, revealing immune evasion mechanisms, metabolic reprogramming, and stromal cell influences. These insights aim to guide the development of personalized and targeted cancer therapies, highlighting the role of TME diversity in shaping tumor behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a6a26f346274035b31fd9f5e6d93385cd1777b" target='_blank'>
              Leveraging Single-Cell Multi-Omics to Decode Tumor Microenvironment Diversity and Therapeutic Resistance
              </a>
            </td>
          <td>
            Hussein Sabit, B. Arneth, Timothy M. Pawlik, Saimaa Abdel-Ghany, Aysha Ghazy, Rawan M. Abdelazeem, A. Alqosaibi, I. Al-Dhuayan, J. Almulhim, N. A. Alrabiah, Ahmed Hashash
          </td>
          <td>2025-01-10</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de8ac06c638611539d33f6ef5fdb316eabe9f02c" target='_blank'>
              Spatio-temporal transcriptomics of chromothriptic SHH-medulloblastoma identifies multiple genetic clones that resist treatment and drive relapse
              </a>
            </td>
          <td>
            Ilia Kats, Milena Simovic-Lorenz, Hannah Sophia Schreiber, Pooja Sant, Jan-Philipp Mallm, Verena Körber, Albert Li, Pravin Velmurugan, Sophie Heuer, Luisa Kües, Frauke Devens, M. Sill, M. Jugold, Mahmoud Moustafa, Amir Abdollahi, Frank Winkler, A. Korshunov, Stefan M Pfister, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Mismatch repair deficiency leads to high mutation rates and microsatellite instability (MSI-H), associated with immune infiltration and responsiveness to immunotherapies. In early stages, MSI-H tumors generally have a better prognosis and lower metastatic potential than microsatellite-stable (MSS) tumors, especially in colorectal cancer. However, in advanced stages, MSI-H tumors lose this survival advantage for reasons that remain unclear. We developed a syngeneic mouse model of MSI cancer by knocking out the MMR gene Msh2 in the metastatic 4T1 breast cancer cell line. This model mirrored genomic features of MSI-H cancers and showed reduction in metastatic incidence compared to their MSS counterparts. In MSI-H tumors, we observed an enrichment of immune gene-signatures that negatively correlated with metastasis incidence. Importantly, a hybrid epithelial-mesenchymal signature, related to aggressiveness was detected only in metastatic MSI-H tumors which may explain the worse outcomes after recurrence of MSI tumors compared to MSS. Interestingly, we identified immature myeloid cells at primary and metastatic sites in MSI-H tumor-bearing mice, suggesting that MMR deficiency elicits specific immune responses beyond T-cell activation. Significance A novel syngeneic mouse model of MSI cancer demonstrates that the immune system regulates MSI cancer cell dissemination, offering an important tool to model advanced stages of human MSI-driven disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4d06352a13d99fbaae03568dc6632e6da89c31" target='_blank'>
              Effect of MisMatch Repair Deficiency on metastasis occurrence modelized in a syngeneic mouse model
              </a>
            </td>
          <td>
            Pierre Laplante, Reginaldo Rosa, L. Nebot-Bral, Jordane Goulas, Sergey I. Nikolaev, Aymeric Silvin, P. Kannouche
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly disease. Over recent decades, a better understanding of the biological mechanisms implicated in its pathogenesis has led to the development of targeted therapies and immunotherapy, resulting in improvements in patient outcomes. To enhance our understanding of lung cancer tumor biology and advance towards precision oncology, we need a comprehensive tumor profile. In recent years, novel in situ spatial multiomics approaches have emerged, offering a more detailed view of the spatial location of tumor and tumor microenvironment cells. These advancements in in situ profiling may unveil further molecular and immune mechanisms in tumor biology that will lead to the discovery of biomarkers for treatment prediction and prognosis. In this review, we provide an overview of current and emerging pathology-based approaches for spatial immune profiling in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e58f728c302c9eea20cd5a44be684287b92352" target='_blank'>
              Novel Spatial Approaches to Dissect the Lung Cancer Immune Microenvironment
              </a>
            </td>
          <td>
            Idania Lubo, Sharia D Hernandez, I. Wistuba, L. S. Solis Soto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Despite substantial drug discovery investments, the lack of any significant therapeutic advancement in the treatment of glioblastoma (GBM) over the past two decades calls for more innovation in the identification of effective treatments. The inter-and intra-patient heterogeneity of GBM presents significant obstacles to effective clinical progression of novel treatments by contributing to tumour plasticity and rapid drug resistance that confounds contemporary target directed drug discovery strategies. Phenotypic drug screening is ideally suited to heterogeneous diseases, where targeting specific oncogenic drivers have been broadly ineffective. Our hypothesis is that a modern phenotypic led approach using disease relevant patient derived GBM stem cell systems will be the most productive approach to identifying new therapeutic targets, drug classes and future drug combinations that target the heterogeneity of GBM. In this study we incorporate a panel of patient derived GBM stem cell lines into an automated and unbiased ‘Cell Painting’ assay to quantify multiple GBM stem cell phenotypes. By screening several compound libraries at multiple concentrations across a panel of patient-derived GBM stem cells we provide the first comprehensive survey of distinct pharmacological classes and known druggable targets, including all clinically approved drug classes and oncology drug candidates upon multiple GBM stem cell phenotypes linked to cell proliferation, survival and differentiation. Our data set representing, 3866 compounds, 2.2million images and 64000 datapoints is the largest phenotypic screen carried out to date on a panel of patient-derived GBM stem cell models that we are aware off. We seek to identify agents and targets classes which engender potent activity across heterogenous GBM genotypes and phenotypes, in this study we further characterize two validated target classes, histone deacetylase inhibitors and cyclin dependent kinases that exert broad and potent effects on the phenotypic and transcriptomic profiles of GBM stem cells. Here we present all validated hit compounds and their target assignments for the GBM community to explore.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5b141f1935b93d7a4e33f68e3c71f2d2847678" target='_blank'>
              A comprehensive pharmacological survey across heterogeneous patient-derived GBM stem cell models
              </a>
            </td>
          <td>
            Richard J. R. Elliott, P. Nagle, M. Furqan, John C Dawson, Aoife McCarthy, A. Munro, C. Drake, Gillian M Morrison, Michael Marand, Daniel Ebner, Steven M. Pollard, V. Brunton, M. Frame, Neil O. Carragher
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Effective treatment of breast cancer remains a formidable challenge, partly due to our limited understanding of the complex microenvironmental factors that contribute to disease pathology. Among these factors are tissue-resident perivascular cells, which play crucial roles in shaping vascular basement membranes, maintaining vessel integrity, and communicating with adjacent endothelial cells. Despite their essential functions, perivascular cells have been relatively overlooked. Identifying them by immunostaining has been challenging due to their low abundance, inherent heterogeneity, and shared marker expression with other cell types. These challenges have hindered efforts to purify pericytes and generate primary cell models for studying their biology. Methods Using a recently developed FACS method, we successfully identified and purified each cell type from breast tissues, allowing us to deep-sequence their transcriptomes and generate primary cell models of each cell type—including pericytes. Here, we used these data to analyze cell-type-specific gene expression in tumors, which revealed a strong association between pericyte-specific genes and breast cancer patient mortality. To explore this association, we defined the heterogeneity of breast pericytes using single-cell RNA sequencing and identified a broad marker for visualizing perivascular cells in breast tumors. Results Remarkably, we discovered perivascular cells dissociated from vessels and emerged as a dominant mesenchymal cell type in a subset of breast tumors that contrasted with their normal perivascular location. Moreover, when we purified pericytes from the breast and cultured them alongside breast tumor cells, we discovered that they induced rapid tumor cell growth significantly greater than isogenic fibroblast controls. Discussion These findings identify perivascular cells as a key microenvironmental factor in breast cancer, highlighting the critical need for further research to explore their biology and identify specific stimulatory mechanisms that could be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920638149619a80dd215db038826c980f92e7f53" target='_blank'>
              Breast pericytes: a newly identified driver of tumor cell proliferation
              </a>
            </td>
          <td>
            Katelyn Del Toro, Y. Licon-Munoz, William Crabtree, Tristan Oper, Christine Robbins, William C. Hines
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f233ae172c7106f684ef62640304f7d51d73186" target='_blank'>
              Integrative mapping of human CD8+ T cells in inflammation and cancer.
              </a>
            </td>
          <td>
            Ziwei Xue, Lize Wu, Ruonan Tian, Bing Gao, Yu Zhao, Bing He, Di Sun, Bingkang Zhao, Yicheng Li, Kaixiang Zhu, Lie Wang, Jianhua Yao, Wanlu Liu, Linrong Lu
          </td>
          <td>2024-11-29</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Bone is a multifaceted tissue requiring orchestrated interplays of diverse cells within specialized microenvironments. Although significant progress has been made in understanding cellular and molecular mechanisms of component cells of bone, revealing their spatial organization and interactions in native bone tissue microenvironment is crucial for advancing precision medicine, as they govern fundamental signaling pathways and functional dependencies among various bone cells. In this study, we present the first integrative high-resolution map of human bone and bone marrow, using spatial and single-cell transcriptomics profiling from femoral tissue. This multi-modal approach discovered a novel bone formation-specialized niche enriched with osteoblastic lineage cells and fibroblasts and unveiled critical cell–cell communications and co-localization patterns between osteoblastic lineage cells and other cells. Furthermore, we discovered a novel spatial gradient of cellular composition, gene expression and signaling pathway activities radiating from the trabecular bone. This comprehensive atlas delineates the intricate bone cellular architecture and illuminates key molecular processes and dependencies among cells that coordinate bone metabolism. In sum, our study provides an essential reference for the field of bone biology and lays the foundation for advanced mechanistic studies and precision medicine approaches in bone-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de859f69ecf65ef803fd91ebf44b36784f4b9786" target='_blank'>
              Mapping the spatial atlas of the human bone tissue integrating spatial and single-cell transcriptomics
              </a>
            </td>
          <td>
            Weiqiang Lin, Yisu Li, Chuan Qiu, Binghao Zou, Yun Gong, Xiao Zhang, Di Tian, William Sherman, Fernando Sanchez, Di Wu, Kuan-Jui Su, Xinyi Xiao, Zhe Luo, Qing Tian, Yiping Chen, Hui Shen, Hongwen Deng
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Copy number variations (CNVs), the gain or loss of genomic regions, are associated with different diseases and cancer types, where they are related to tumor progression and treatment outcome. Single cell technologies offer new possibilities to measure CNVs in individual cells, allowing to assess population heterogeneity and to delineate subclonal structures. Single cell whole-genome sequencing is considered the gold-standard for the quantification of CNVs in single cells. However, the majority of existing single cell datasets interrogate gene expression, using scRNA-seq. Consequently, several computational approaches have been developed to identify CNVs from that data modality. Nevertheless, an independent benchmarking of these methods is lacking. We used 15 scRNA-seq datasets and evaluated six popular computational methods in their ability to recover the ground truth CNVs using a large set of performance metrics. Additionally, we explored whether they could correctly identify euploid cells, especially also in fully diploid samples, and subclonal structures in heterogeneous tumor samples. We discovered several dataset-specific factors that influence the performance of the methods, such as the dataset size and the number and type of CNVs in the analyzed sample. We found that the choice of the reference dataset can have a large impact on the performance. Methods which included additional allelic information from the scRNA-seq reads performed more robustly across scenarios, but at the cost of higher runtime. Furthermore, the methods differed substantially in their additional functionalities and resource requirements. We offer a benchmarking pipeline to help identify the optimal CNV calling method for newly generated scRNA-seq datasets, and to benchmark and improve new methods performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3f23333379961a358cfa4029d6bc3607c70188" target='_blank'>
              Benchmarking scRNA-seq copy number variation callers
              </a>
            </td>
          <td>
            Katharina T. Schmid, Aikaterini Symeonidi, Dmytro Hlushchenko, M. Richter, Maria Colomé-Tatché
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Cancer initiation and progression are controlled not only by intrinsic factors, such as genetic mutations and mesenchymal trait acquisition, but also by dynamic spatiotemporal shifts within the tumor microenvironment. These elements induce plasticity, and enhance invasiveness, metastasis, and treatment resistance, while promoting immune evasion through the reprogramming of the immune microenvironment. However, the molecular mechanisms driving this reprogramming remain unclear. In this review, we summarize the recent pivotal findings on pancreatic cancer, which is a highly malignant disease, and overview the roles of plasticity, metabolic reprogramming, and the creation of an immunosuppressive microenvironment in disease progression. We also present our research on two crucial signaling pathways, i.e., the dynamic interplay between mutational load and the tumor microenvironment, and the acquisition of cancer plasticity via inflammatory responses and metabolic reprogramming. A comprehensive understanding of these processes is expected to transform therapeutic strategies and improve the outcomes of pancreatic cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d6b2b31d56874111eca289fe4cec7a0df598ab" target='_blank'>
              Plasticity and Tumor Microenvironment in Pancreatic Cancer: Genetic, Metabolic, and Immune Perspectives
              </a>
            </td>
          <td>
            A. Hashimoto, S. Hashimoto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancers arising from dysregulation of generally operative signaling pathways are often tissue specific, but the mechanisms underlying this paradox are poorly understood. Based on striking cell-type specificity, we postulated that these mechanisms must operate early in cancer development and set out to study them in a model of von Hippel Lindau (VHL) disease. Biallelic mutation of the VHL ubiquitin ligase leads to constitutive activation of hypoxia inducible factors HIF1A and HIF2A and is generally a truncal event in clear cell renal carcinoma. We used an oncogenic tagging strategy in which VHL-mutant cells are marked by tdTomato, enabling their observation, retrieval, and analysis early after VHL-inactivation. Our findings reveal markedly different consequences of HIF1A and HIF2A activation, but that both contribute to renal cell-type specific consequences of VHL-inactivation in the kidney. Early involvement of HIF2A in promoting proliferation within proximal tubular epithelium supports therapeutic targeting of HIF2A early in VHL disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be97f9c0f0b00dada5534b4bd4b5d705fdb68b10" target='_blank'>
              HIFα isoform specific activities drive cell-type specificity of VHL-associated oncogenesis
              </a>
            </td>
          <td>
            Joana Darc Carola Correia Lima, Madeleine Hooker, Ran Li, Ayslan B. Barros, N. Masson, Christopher W. Pugh, D. Mole, Julie Adam, Peter J. Ratcliffe, Samvid Kurlekar
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Existing single-cell clustering methods are based on gene expressions that are susceptible to dropout events in single-cell RNA sequencing (scRNA-seq) data. To overcome this limitation, we proposed a pathway-based clustering method for single cells (scPathClus). scPathClus first transforms single-cell gene expression matrix into pathway enrichment matrix and generates its latent feature matrix. Based on the latent feature matrix, scPathClus clusters single cells using the method of community detection. Applying scPathClus to PDAC scRNA-seq datasets, we identified two types of cancer-associated fibroblasts (CAFs), termed csCAFs and gapCAFs, which highly expressed complement system and gap junction-related pathways, respectively. Spatial transcriptome analysis revealed that gapCAFs and csCAFs are located at cancer and non-cancer regions, respectively. Pseudotime analysis suggest a potential differentiation trajectory from csCAFs to gapCAFs. Bulk transcriptome analysis showed that gapCAFs-enriched tumors are more endowed with tumor-promoting characteristics and worse clinical outcomes, while csCAFs-enriched tumors confront stronger antitumor immune responses. Compared to established CAF subtyping methods, this method displays better prognostic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e5196bb420146fb22bc069bed5ff6f7bc8606d" target='_blank'>
              Pathway-based clustering identifies two subtypes of cancer-associated fibroblasts associated with distinct molecular and clinical features in pancreatic ductal carcinoma
              </a>
            </td>
          <td>
            Hongjing Ai, R. Nie, Xiaosheng Wang
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69505c7dd30d328b855b8e26bb2a91fa4c9f9efa" target='_blank'>
              Computer-aided analysis reveals metallothionein-positive cancer-associated fibroblasts promote angiogenesis in gastric adenocarcinoma
              </a>
            </td>
          <td>
            Xiaolong Jin, Yu Tian, Haoran Zhu, Yuewen Sun, Zhenxing Zhang
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="We develop Spatial Perturb-Seq, an in vivo CRISPR technology that interrogates multiple genes within single cells of intact tissues. We apply Spatial Perturb-Seq to screen risk genes for neurodegenerative diseases in the mouse brain, uncovering cell autonomous and cell-cell microenvironmental effects within the spatially intact tissue. Spatial Perturb-Seq functionally screens multiple genes in situ and in vivo, identifying candidate genes underlying dysregulated neuronal intercellular communication pathways in Lrp1 signalling and ephrin-Eph receptor interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/463cf687c456c72f8cdd5ea5f55255ae0d4d529a" target='_blank'>
              Spatial Perturb-Seq: Single-Cell Functional Genomics within Intact Tissue Architecture
              </a>
            </td>
          <td>
            Kimberle Shen, W. Seow, Choong Tat Keng, Daryl Lim Shern, Ke Guo, Amine Meliani, Irfan Muhammad Bin Hajis, Kok Hao Chen, W. Chew
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a leading cause of cancer-related deaths globally. The heterogeneity of the tumor microenvironment significantly influences patient prognosis, while the diversity of tumor cells shapes its unique characteristics. A comprehensive analysis of the molecular profile of tumor cells is crucial for identifying novel molecular targets for drug sensitivity analysis and for uncovering the pathophysiological mechanisms underlying CRC. Methods We utilized single-cell RNA sequencing technology to analyze 13 tissue samples from 4 CRC patients, identifying key cell types within the tumor microenvironment. Intercellular communication was assessed using CellChat, and a risk score model was developed based on eight prognostic genes to enhance patient stratification for immunotherapeutic approaches. Additionally, in vitro experiments were performed on DLX2, a gene strongly associated with poor prognosis, to validate its potential role as a therapeutic target in CRC progression. Results Eight major cell types were identified across the tissue samples. Within the tumor cell population, seven distinct subtypes were recognized, with the C0 FXYD5+ tumor cells subtype being significantly linked to cancer progression and poor prognosis. CellChat analysis indicated extensive communication among tumor cells, fibroblasts, and immune cells, underscoring the complexity of the tumor microenvironment. The risk score model demonstrated high accuracy in predicting 1-, 3-, and 5-year survival rates in CRC patients. Enrichment analysis revealed that the C0 FXYD5+ tumor cell subtype exhibited increased energy metabolism, protein synthesis, and oxidative phosphorylation, contributing to its aggressive behavior. In vitro experiments confirmed DLX2 as a critical gene associated with poor prognosis, suggesting its viability as a target for improving drug sensitivity. Conclusion In summary, this study advances our understanding of CRC progression by identifying critical tumor subtypes, molecular pathways, and prognostic markers that can inform innovative strategies for predicting and enhancing drug sensitivity. These findings hold promise for optimizing immunotherapeutic approaches and developing new targeted therapies, ultimately aiming to improve patient outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8647de1e0192ab64820f770599e15f3edf4689c3" target='_blank'>
              Innovative strategies to optimise colorectal cancer immunotherapy through molecular mechanism insights
              </a>
            </td>
          <td>
            Quanjun Lin, Zhiqiang Wang, Jue Wang, Ming Xu, Xinyi Zhang, Peng Sun, Yihang Yuan
          </td>
          <td>2024-12-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The cascade of events leading to tumor formation includes induction of a tumor supporting neovasculature, as a primary hallmark of cancer. Developing vasculature is difficult to evaluate in vivo but can be captured using microfluidic chip technology and patient derived cells. Herein, we established an on chip approach to investigate the mechanisms promoting tumor vascularization and vascular targeted therapies via co-culture of cancer spheroids and endothelial cells in a three dimensional environment. Methods: We investigated both, tumor neovascularization and therapy, via co-culture of human derived endothelial cells and adjacently localized metastatic renal cell carcinoma spheroids on a commercially available microfluidic chip system. Metastatic renal cell carcinoma spheroids adjacent to primary vessels model tumor, and induce vessels to sprout neovasculature towards the tumor. We monitored real time changes in vessel formation, probed the interactions of tumor and endothelial cells, and evaluated the role of important effectors in tumor vasculature. In addition to wild type endothelial cells, we evaluated endothelial cells that overexpress Prostate Specific Membrane Antigen (PSMA), that has emerged as a marker of tumor associated neovasculature. We characterized the process of neovascularization on the microfluidic chip stimulated by enhanced culture medium and the investigated metastatic renal cell carcinomas, and assessed endothelial cells responses to vascular targeted therapy with bevacizumab via confocal microscopy imaging. To emphasize the potential clinical relevance of metastatic renal cell carcinomas on chip, we compared therapy with bevacizumab on chip with an in vivo model of the same tumor. Results: Our model permitted real-time, high-resolution observation and assessment of tumor-induced angiogenesis, where endothelial cells sprouted towards the tumor and mimicked a vascular network. Bevacizumab, an antiangiogenic agent, disrupted interactions between vessels and tumors, destroying the vascular network. The on chip approach enabled assessment of endothelial cell biology, vessel's functionality, drug delivery, and molecular expression of PSMA. Conclusion: Observations in the vascularized tumor on chip permitted direct and conclusive quantification of vascular targeted therapies in weeks as opposed to months in a comparable animal model, and bridged the gap between in vitro and in vivo models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0cee76c44f4581ac79061d246894e755a88b77d" target='_blank'>
              Vascularized tumor on a microfluidic chip to study mechanisms promoting tumor neovascularization and vascular targeted therapies
              </a>
            </td>
          <td>
            Magdalena Skubal, B. M. Larney, N. Phung, Juan Carlos Desmaras, Abdul Vehab Dozic, A. Volpe, Anuja Ogirala, Camila Longo Machado, Jakob Djibankov, V. Ponomarev, J. Grimm
          </td>
          <td>2025-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Squamous cell carcinomas (SCC) are often preceded by potentially malignant precursor lesions, most of which remain benign. The terminal exhaustion phenotypes of effector T-cells and the accumulation of myeloid-derived suppressor cells (MDSC) have been thoroughly characterized in established SCC. However, it is unclear what precancerous lesions harbor a bona fide high risk for malignant transformation and how precancerous epithelial dysplasia drives the immune system to the point of no return. Here we show that expression of SRY-box transcription factor 2 (SOX2) in precancerous lesions imparts an irreversible risk that recruits suppressive myeloid cells by promoting the release of CCL2. We developed a unique genetically engineered mouse model (GEMM) to recapitulate the malignant transformation of epithelial dysplasia to SCC in the oral mucosa with high histologic and phenotypic fidelity. Using a combination of longitudinal human specimens and the Sox2-GEMM, we found that the myeloid cells in precancerous epithelial dysplasia exhibit a distinctive dichotomous profile featuring high levels of IL-1α-SLC2A1 and low levels of type-I interferon (IFN-I) signatures, which occurs before SCC emerges histologically. Brief priming of myeloid cells with IL-1α desensitizes them to IFN-I agonists and makes myeloid-derived suppressor cells (MDSC) even more suppressive of T-cell activation. Mechanistically, IL-1 activation represses the expression of DHHC3/7 enzymes, which are responsible for the palmitoylation of stimulator of interferon genes (STING). Early blockade of IL1 signaling using pharmacologic and genetic approaches similarly reduces MDSC and SLC2A1high myeloid cells, suppresses epithelial dysplasia transformation, and extends survival. This work establishes a previously unrecognized SOX2-CCL2-IL1 pathway that leads to irreversible immune escape when precancerous epithelial lesions transform.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68906ee8a14c680b07808e1d20602c4baa417802" target='_blank'>
              SOX2-induced IL1α-mediated immune suppression drives epithelial dysplasia malignant transformation
              </a>
            </td>
          <td>
            Hülya F. Taner, Wang Gong, Z. Fitzsimonds, Zaiye Li, Yuesong Wu, Yumin He, Kohei Okuyama, Wanqing Cheng, Jung Kuczura, Sashider Rajesh, Andriana Manousidaki, Shuo Feng, Miki Lee, Felipe Nör, E. Lanzel, S. Demehri, Peter J. Polverini, Jacques E. Nör, Thomas D. Wang, Jianwen Que, Haitao Wen, Yuying Xie, James J. Moon, Yu Leo Lei
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Single cell RNA-seq (scRNA-seq) technologies provide unprecedented resolution representing transcriptomics at the level of single cell. One of the biggest challenges in scRNA-seq data analysis is the cell type annotation, which is usually inferred by cell separation approaches. In-silico algorithms that accurately identify individual cell types in ongoing single-cell sequencing studies are crucial for unlocking cellular heterogeneity and understanding the biological basis of diseases. In this study, we focus on robustly identifying cell types in single-cell RNA sequencing data; we conduct a comparative analysis using methods established in biology, like Seurat, Leiden, and WGCNA, as well as Infomap, statistical inference via Stochastic Block Models (SBM), and single-cell Graph Neural Networks (scGNN). We also analyze preprocessing pipelines to identify and optimize key components in the process. Leveraging two independent datasets, PBMC and ROSMAP, we employ clustering algorithms on cell-cell networks derived from gene expression data. Our findings reveal that while clusters detected by WGCNA exhibit limited correspondence with cell types, those identified by multiresolution Infomap and Leiden, and SBM show a closer alignment, with Infomap standing out as a particularly effective approach. Infomap notably offers valuable insights for the precise characterization of cellular landscapes related to neurodegenration and immunology in scRNA-seq.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b8c009cdac4512de3acd925b5a468747284429" target='_blank'>
              Cell Type Differentiation Using Network Clustering Algorithms
              </a>
            </td>
          <td>
            Fatemeh Sadat Fatemi Nasrollahi, Filipi Nascimento Silva, Shiwei Liu, S. Chaudhuri, Meichen Yu, Juexin Wang, K. Nho, A. Saykin, David A. Bennett, Olaf Sporns, Santo Fortunato
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit enhanced resistance to standard of care treatments and have significantly worse overall survival (OS) compared to patients with classical subtype enriched tumors. It is important to develop genomic resources, enabling identification of novel putative targets in a statistically rigorous manner.


EXPERIMENTAL DESIGN
We compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 patient samples, aggregated from publicly available raw data. We mapped cell-type specific scRNAseq gene signatures in bulk RNAseq (n=744) and spatial transcriptomics (ST) (n=22) and performed validation using multiplex immunostaining.


RESULTS
Analysis of tumor cells from our scRNAseq atlas revealed nine distinct populations, two of which aligned with the basal subtype, correlating with worse OS in bulk RNAseq. Deconvolution identified one of the basal populations to be the predominant tumor subtype in non-dissociated ST tissues and in vitro tumor cell and patient-derived organoid lines. We discovered a novel enrichment and spatial association of CXCL10+ cancer associated fibroblasts (CAFs) with basal tumor cells. We identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, suggesting a relationship between these cell types in PDAC tumor microenvironment.


CONCLUSIONS
We show that our scRNAseq atlas (700,000 cells), integrated with ST data, has increased statistical power and is a powerful resource, allowing for expansion of current subtyping paradigms in PDAC. We uncovered a novel signaling niche marked by CXCL10+ CAFs and basal tumor cells that could be explored for future targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07269a9cb871b60ea57ca5b2a01a49e35b66cc9b" target='_blank'>
              Human pancreatic cancer single cell atlas reveals association of CXCL10+ fibroblasts and basal subtype tumor cells.
              </a>
            </td>
          <td>
            Ian M Loveless, Samantha B. Kemp, Kailee M Hartway, Jacob T. Mitchell, Yuesong Wu, Samuel D Zwernik, D. Salas-Escabillas, Sydney Brender, Madison George, Yetunde Makinwa, Thais Stockdale, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombwell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, E. Fertig, L. T. Kagohara, Brian Theisen, Howard C Crawford, Nina G. Steele
          </td>
          <td>2024-12-05</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Muscle-invasive bladder cancer (MIBC) is a prevalent cancer characterized by molecular and clinical heterogeneity. Assessing the spatial heterogeneity of the MIBC microenvironment is crucial to understand its clinical significance. Methods In this study, we used imaging mass cytometry (IMC) to assess the spatial heterogeneity of MIBC microenvironment across 185 regions of interest in 40 tissue samples. We focused on three primary parameters: tumor (T), leading-edge (L), and nontumor (N). Cell gating was performed using the Cytobank platform. We calculated the Euclidean distances between cells to determine cellular interactions and performed single-cell RNA sequencing (scRNA-seq) to explore the molecular characteristics and mechanisms underlying specific fibroblast (FB) clusters. scRNA-seq combined with spatial transcriptomics (ST) facilitated the identification of ligand–receptor (L–R) pairs that mediate interactions between specific FB clusters and endothelial cells. Machine learning algorithms were used to construct a prognostic gene signature. Results The microenvironments in the N, L, and T regions of MIBC exhibited spatial heterogeneity and regional diversity in their components. A distinct FB cluster located in the L region—identified as S3—is strongly associated with poor prognosis. IMC analyses demonstrated a close spatial association between S3 and endothelial cells, with S3-positive tumors exhibiting increased blood vessel density and altered vascular morphology. The expression of vascular endothelial growth factor receptor and active vascular sprouting were significant in S3-positive tumors. scRNA-seq and ST analyses indicated that the genes upregulated in S3 were associated with angiogenesis. NOTCH1–JAG2 signaling pathway was identified as a significant L–R pair specific to S3 and endothelial cell interactions. Further analysis indicated that YAP1 was a potential regulator of S3. Machine learning algorithms and Gene Set Variation Analysis were used to establish an S3-related gene signature that was associated with the poor prognosis of tumors including MIBC, mesothelioma, glioblastoma multiforme, lower-grade glioma, stomach adenocarcinoma, uveal melanoma, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, and lung squamous cell carcinoma. Conclusions We assessed the spatial landscape of the MIBC microenvironment and revealed a specific FB cluster with prognostic potential. These findings offer novel insights into the spatial heterogeneity of the MIBC microenvironment and highlight its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fcfdce6314523f58922cfd8b6a6e220595a1f7" target='_blank'>
              Spatially-resolved analyses of muscle invasive bladder cancer microenvironment unveil a distinct fibroblast cluster associated with prognosis
              </a>
            </td>
          <td>
            Chao Feng, Yaobang Wang, Wuyue Song, Tao Liu, Han Mo, Hui Liu, Shulin Wu, Zezu Qin, Zhenxing Wang, Yuting Tao, Liangyu He, Shaomei Tang, Yuanliang Xie, Qiuyan Wang, Tianyu Li
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Drug resistance and distant metastases are major contributors to mortality in colorectal cancer (CRC). Here we investigate mechanisms underlying acquired resistance to oxaliplatin, a first-line, standard-of-care CRC treatment. We generated oxaliplatin-resistant CRC tumor cells with clinically relevant dosing regimen, which displayed enhanced metastatic potential. Transcriptomic and phenotypic analyses revealed a critical function for cholesterol biogenesis in modulating TGF-β signaling activity, which in turn regulates SERPINE1 expression, a gene we identified as a key player in promoting drug resistance and metastasis. Additionally, we uncovered a SERPINE1-associated nine-gene expression signature, RESIST-M, that can predict overall and relapse-free survival (RFS) in clinical cohorts and is able to stratify patients into CMS4/iCMS3-fibrotic CRC-subtypes, underscoring its clinical utility. Using mouse tumor models, we provide further evidence that targeting SERPINE1 and cholesterol biogenesis can be viable approaches to re-sensitize the resistant pro-metastatic CRC cells to oxaliplatin. This study not only elucidates the molecular underpinnings of drug resistance and metastasis in primary CRC, but also offers prognostic and therapeutic strategies to guide clinical management of the disease. Significance This study reveals critical resources and insights on oxaliplatin resistance and metastasis in CRC via a novel TGF-β cholesterol axis. We generated improved oxaliplatin-resistant models that enabled identification of a prognostic SERPINE1-based gene signature to predict oxaliplatin resistance-induced metastasis in CRC. This gene signature derived from our models showed that the models can mimic CMS-4/iCMS-fibrotic-like metastatic CRC patients. We validated therapeutic candidates targeting CMS-4/iCMS-fibrotic-like metastatic CRC cells which can reverse drug resistance and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec01a84e2257e6ca8a1bf9c64a17d3d5c5ca0b06" target='_blank'>
              Modelling oxaliplatin resistance in colorectal cancer reveals a SERPINE1-based gene signature (RESIST-M) and therapeutic strategies for pro-metastatic CMS4 subtype
              </a>
            </td>
          <td>
            Stephen Qi Rong Wong, Mohua Das, Niranjan Shirgaonkar, Kenzom Tenzin, Huiwen Chua, Lin Xuan Chee, Sae Yeoh Ahpa, Astley Aruna Murugiah, Wei Yong Chua, Madelaine Skolastika Theardy, M. Thangavelu, Jane Vin Chan, Choon Kong Yap, Iain Bee Huat Tan, Petros K Tsantoulis, Sabine Tejpar, J. Loo, Ramanuj DasGupta
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Reduced therapy response in breast cancer has been correlated with heterogeneity in biomarker composition, expression level, and spatial distribution of cancer cells within a patient tumor. Thus, there is a need for models to replicate cell-cell, cell-stromal, and cell-microenvironment interactions during cancer progression. Traditional two-dimensional (2D) cell culture models are convenient but cannot adequately represent tumor microenvironment histological organization, in vivo 3D spatial/cellular context, and physiological relevance. Recently, three-dimensional (3D) in vitro tumor models have been shown to provide an improved platform for incorporating compositional and spatial heterogeneity and to better mimic the biological characteristics of patient tumors to assess drug response. Advances in 3D bioprinting have allowed the creation of more complex models with improved physiologic representation while controlling for reproducibility and accuracy. This review aims to summarize the advantages and challenges of current 3D in vitro models for evaluating therapy response in breast cancer, with a particular emphasis on 3D bioprinting, and addresses several key issues for future model development as well as their application to other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ccd5bcde9ada94a158d2c3401e2be4034200cf" target='_blank'>
              Modeling intratumor heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            E. McDonough, M. Barroso, Fiona Ginty, David T Corr
          </td>
          <td>2024-12-06</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Teratomas are a highly differentiated type of testicular germ cell tumors (TGCTs), the most common type of solid cancer in young men. Prominent inflammatory infiltrates are a hallmark of TGCTs, although their compositions and dynamics in teratomas remain elusive. Here, we reached out to characterize the infiltrating immune cells and their activation and polarization state by using high-throughput gene expression analysis of 129.MOLF-Chr19 mice that spontaneously develop testicular teratomas. We showed that inconspicuous testes without any apparent alterations in size or morphology can be clustered into three groups based on their expression of stemness and immune genes, supporting a model in which initial oncogenic transformation elicits a first wave of T-cell infiltration. Moderately and severely enlarged tumorous testes then displayed a progressive infiltration with T cells, monocytes/macrophages, and B cells. Importantly, T cells seem to adopt an inactive state caused by an overexpression of immune checkpoint molecules and the polarization of monocytes/macrophages to an anti-inflammatory phenotype. Our findings are supported by the analysis of metabolic gene expression, which unveiled alterations indicative of tumor growth and immune cell infiltration. Collectively, testicular teratomas, at least in mice, are characterized by a diverse inflammatory infiltrate containing T cells that putatively become inactivated, allowing the tumors to further grow. We believe that these findings may provide a rationale for the development of new immunomodulatory therapies for TGCTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b07bb71a6c00863ba07bb1169016cede3a797fe7" target='_blank'>
              Teratoma Development in 129.MOLF-Chr19 Mice Elicits Two Waves of Immune Cell Infiltration
              </a>
            </td>
          <td>
            Lucas Klaus, Sybille D. Reichardt, Maria Neif, Lutz Walter, F. Gayer, Holger M Reichardt
          </td>
          <td>2024-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite surgical resection, rigorous endoscopic surveillance, and immunotherapy with the Bacillus Calmette–Guérin (BCG) vaccine, 30% of high-risk bladder cancers recur, and 10% result in fatal outcomes within two years of diagnosis. The global shortage of BCG underscores the urgent need for alternative or complementary therapeutic strategies. To address this, we integrated transcriptomic profiling and targeted genomic sequencing to identify four consensus intrinsic subtypes of bladder cancer. Initially derived from bulk RNA profiling, these subtypes were further validated at the cellular and tissue-compartment levels using single-cell RNA sequencing and spatial transcriptomics. Notably, we identified a subtype of inflamed tumors with enhanced endogenous retroelement expression and increased commensal bacterial presence, which showed the highest responsiveness to BCG therapy. Additionally, we developed a machine learning-based model incorporating composite molecular features to predict recurrence risk, achieving a high accuracy (AUC = 0.90). Our findings establish a molecular precision framework for bladder cancer and nominate novel therapeutic targets to reduce reliance on BCG immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f0d9394677cb2bb295a5f2fe96a5d0c0fc0b84" target='_blank'>
              Genomic and transcriptomic profiling of high-risk bladder cancer reveals diverse molecular and microenvironment ecosystems
              </a>
            </td>
          <td>
            K. Meghani, Yanni Yu, N. Frydenlund, Erik Z. Li, B. Choy, S. Abdulkadir, J. Meeks
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) tumor cells and their interactions with the immune microenvironment, particularly at the leading-edge area, have been underexplored. This study employs single-cell RNA sequencing (scRNA-seq) and spatial transcriptome (ST) analysis on samples from the tumor core, adjacent non-tumorous tissue, and the leading-edge area of nine ICC patients. These findings indicate that tumor cells at the leading-edge area demonstrate enhanced proliferation and are tightly associated with the stroma, including endothelial cells and POSTN+ FAP+ fibroblasts. Notably, CD8+ T cells in this region exhibit a naive phenotype with low cytotoxicity and signs of exhaustion, likely due to compromised antigen presentation by antigen-presenting cells (APCs). The predominant CD8+ T cell subset, mucosal-associated invariant T (MAIT) cells, recruits SPP1+ macrophages within the stroma. This interaction, along with the presence of POSTN+ cancer-associated fibroblasts (CAFs) and endothelial cells, forms a unique "triad structure" that fosters tumor growth and ICC progression. The research highlights the intricate characteristics and interactions of ICC tumor cells in the leading-edge area, offering insights into potential therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4692f475931cf20c24dd17b389c955de7ede39" target='_blank'>
              Single-Cell and Spatial Transcriptomics Delineate the Microstructure and Immune Landscape of Intrahepatic Cholangiocarcinoma in the Leading-Edge Area.
              </a>
            </td>
          <td>
            Li Zuyin, Li Zhao, Cheng Qian, Changkun Zhang, Ma Delin, Jialing Hao, Chen Zhuomiaoyu, Li Yuzi, Zheng Jiaxi, Gao Jie, Zhu Jiye
          </td>
          <td>2024-12-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gamma-delta ( γδ ) T cells are a crucial component of the tumor immune microenvironment which are considered a promising potential therapeutic strategy and target. Increasing evidence suggests that these unique immune cells play significant roles across various cancers. However, γδ T cells are often regarded as having dual roles in tumors, and their influence on lung adenocarcinoma (LUAD) remains controversial. In this research, we employed a wild-ranging approach using multi-omics data to investigate the function of γδ T cells in LUAD. The abundance of γδ T cell infiltration is linked to a positive prognosis in lung adenocarcinoma. The tumor-inhibiting role of γδ T cells was played through intrinsic lineage evolution, acquiring cytotoxic functions and engaging in signal transduction with antigen-presenting cells. Furthermore, patients with higher γδ T cells infiltration abundance might be more favorable for immunotherapy. Lastly, we established a predictive model using CT images based on radiomics, providing a non-invasive strategy to assess γδ T cells infiltration in LUAD patients. These findings offer new insights and perspectives the personalized therapies of γδ T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ada3c2d9795ca814751eeb983c8b55cf90854b2" target='_blank'>
              Integrating of Radiomics and Single-cell Transcriptomics Uncovers the Crucial Role of γδ T Cells in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Ziyi Chen, Changqing Yang, Jiajing Wang, Baichuan Wang, Long He, Zhaoyi Liu, Yingxi Li, Tingting Qin, Peng Chen
          </td>
          <td>2025-01-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Epithelial-mesenchymal plasticity (EMP) is a cell-fate switching program that enables cells to adopt a spectrum of phenotypes ranging from epithelial (E) to mesenchymal (M) and hybrid E/M states. Hybrid E/M phenotypes are considered optimal for cancer metastasis due to their association with metastatic initiation and traits such as stemness, drug resistance, and collective migration. Boolean models of gene regulatory networks (GRNs) underlying EMP have provided valuable insights into the dynamics of E and M phenotypes. However, these models fail to capture hybrid phenotypes effectively, as Boolean frameworks allow only binary gene expression, while hybrid E/M cells exhibit partial expression of both E and M genes. To address this limitation, we modified a threshold-based Boolean formalism to incorporate intermediate gene expression levels. The resulting multilevel model reveals new hybrid steady states with partial expression of both E and M genes, expanding the phenotypic landscape beyond that captured by traditional Boolean models. These hybrid states exhibit lower frustration and higher abundance, indicating enhanced stability. By resolving degeneracy in the dynamics, we further demonstrate that hybrid states identified by the multilevel model are more stable. These findings suggest that adding minimal complexity to Boolean models can uncover previously hidden qualitative features of phenotypic landscapes governed by GRNs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29922a69bbdff28fdd118c9e80f2fc16659ff8c" target='_blank'>
              A multi-level formalism to capture the characteristics of hybrid phenotypes in Epithelial-Mesenchymal Plasticity
              </a>
            </td>
          <td>
            K. Hari, Shubham Tripathi, Vaibhav Anand, M. Jolly, Herbert Levine
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The adaptation of metastatic cells to their host tissue critically determines the pathogenicity of a cancer and therefore patient survival. Yet, it remains elusive to what extent the host environment drives gene expression programs in metastatic cells. Here we identify adaptive mechanisms that enable metastases to establish themselves in a novel tissue context. We performed single-cell RNA-sequencing on malignant and benign tissue samples from untreated donors with colorectal adenocarcinoma and liver metastasis to deduce tissue adaptive expression patterns. A novel computational approach identified genes and pathways that consistently adapted to the host tissue at the transition from the primary tumor to the paired metastasis across donors. This analysis revealed that the majority of expression changes in the metastasis reflect an expression signature reminiscent of benign liver epithelial cells. Cellular processes adapting to the liver environment include basic cellular functions such as energy metabolism, as well as tissue-specific pathways such as the regulation of lipid metabolism by PPAR-α. These adaptations potentially increase the pathogenicity of the metastatic cells and may provide new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189b4d35fab487ddefe0f36a861c9c03b2e18b5a" target='_blank'>
              Gene expression adaptation of metastases to their host tissue
              </a>
            </td>
          <td>
            L. Nagel, Marten C. Wenzel, Sascha Hoppe, Patrick S. Plum, Mohammad Karimpour, Marek Franitza, Roger Wahba, Marc Bludau, Christiane J. Bruns, Alexander Quaas, A. Beyer, Axel M. Hillmer
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) account for 0.01 to 2% of the total tumor mass; however, they play a key role in tumor progression, metastasis and resistance to current cancer therapies. The generation and maintenance of CSCs are usually linked to the epithelial–mesenchymal transition (EMT), a dynamic process involved in reprogramming cancer cells towards a more aggressive and motile phenotype with increased stemness potential. Cells that undergo an EMT process have shown to be more resistant to conventional chemo/radiotherapies. In this context, aldehyde dehydrogenase (ALDH) enzymes, known for their role in the cellular detoxification of aldehydes and enhancement of cell survival, are often upregulated in cancer cells, promoting their resistance to conventional cancer treatments. Indeed, high ALDH levels have become a hallmark biomarker of CSCs and are often used to isolate this sub-population from the more abundant cancer cell populations. Herein, we isolated human breast cancer epithelial cells with higher ALDH abundance (ALDHHigh) and compared them to those with low ALDH abundance (ALDHLow). ALDHHigh sub-populations exhibited more characteristic EMT biomarkers by adopting a more mesenchymal phenotype with increased stemness and enhanced migratory potential. Furthermore, ALDHHigh sub-populations displayed elevated senescent markers. Moreover, these cells also demonstrated higher levels of mitochondria DNA/mass, as well as greater mitochondrial and glycolytic metabolic function. Conversely, ALDHLow sub-populations showed a higher efficiency of mammosphere/colony formation and an increased proliferative capacity. Therefore, we demonstrated that these ALDH sub-populations have distinct characteristics, underscoring their role in EMT, the formation of tumors and the mechanisms of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c656cc9a0663a21dc7738f32f6803ef9c5938a5" target='_blank'>
              ALDHHigh Breast Cancer Stem Cells Exhibit a Mesenchymal–Senescent Hybrid Phenotype, with Elevated Metabolic and Migratory Activities
              </a>
            </td>
          <td>
            Luis Larrea Murillo, Conor J. Sugden, Bela Ozsvari, Zahra Moftakhar, Ghada S. Hassan, F. Sotgia, Michael P Lisanti
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed580d09e5d8a50632bdd53ac296d7f45feff0d1" target='_blank'>
              The microenvironment cell index is a novel indicator for the prognosis and therapeutic regimen selection of cancers
              </a>
            </td>
          <td>
            Xian-Yan Yang, Nian Chen, Qian Wen, Yu Zhou, Tao Zhang, Ji Zhou, Cheng-Hui Liang, , Xiao-Ya Wang, Qingmei Kang, Xiaomei Zheng, Xue-Jia Zhai, Hong-Ying Jiang, Tian-Hua Shen, Jin-Wei Xiao, Yu-Xin Zou, Yun Deng, Shuang Lin, Jiang-jie Duan, Jun Wang, Shicang Yu
          </td>
          <td>2025-01-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The complex interplay between metal abundance, transport mechanisms, cell distribution, and tumor progression-related biological pathways (e.g., metabolism, collagen remodeling) remains poorly understood. Traditionally, genes and metals have been studied in isolation, limiting insights into their interactions. Recent advances in spatial transcriptomics and elemental profiling now enable comprehensive exploration of tissue-wide metal-gene interactions, though integration remains challenging. In this proof-of-concept study, we investigated metal-dependent signaling within the tumor microenvironment of a unique colorectal cancer (CRC) tumor. We implemented a spatial multimodal workflow which integrated elemental imaging, gene expression, cellular composition, and histopathological features to uncover metals-related pathways through spatially resolved differential expression analysis. Preliminary findings revealed significant associations, for instance: elevated iron correlated with mesenchymal phenotypes located at the tumor's proliferative front, reflecting epithelial-to-mesenchymal transition pathways, and extracellular matrix remodeling. High concentrations of copper were predominantly localized in regions of active tumor growth and associated with upregulation of immune response genes. This proof-of-concept workflow demonstrates the feasibility of integrating elemental imaging with spatial transcriptomics to identify metals-based gene correlates. Future application of this workflow to larger patient cohorts will pave the way for expansive comparisons across the metallome and transcriptome, ultimately identifying novel targets for tumor progression biomarkers and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b7bf64840757cf5e603b33283a2afe6a77fb5a3" target='_blank'>
              Integration of Elemental Imaging and Spatial Transcriptomic Profiling for Proof-of-Concept Metals-Based Pathway Analysis of Colon Tumor Microenvironment
              </a>
            </td>
          <td>
            A. Srivastava, N. Shaik, Y. Lu, M. Chan, A. B. Diallo, S. Han, T. Punshon, B. P. Jackson, L. Vahdat, X. Liu, V. Mittal, K. S. Lau, J. Gui, L. Vaickus, J. Hoopes, F. Kolling, L. Perreard, J. Marotti, J. Levy
          </td>
          <td>2024-12-10</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Cryoablation induces antitumor immune responses. Spatial transcriptomic landscape technology has been used to determine the micron-level panoramic transcriptomics of tissue slices in situ.


METHODS
The effects of cryoablation on the immune microenvironment in non-small cell lung cancer (NSCLC) were explored by comparing the Whole Transcriptome Atlas (WTA) panel of immune cells before and after cryoablation using the spatial transcriptomic landscape.


RESULTS
The bioinformatics analysis showed that cryoablation significantly affected the WTA of immune cells, particularly genes related to cellular components, biological processes, molecular functions, proliferation and migration, and cytokine-cytokine receptor interaction signaling pathways.


CONCLUSIONS
The findings of this study suggest that cryoablation significantly impacts the biological functions of immune cells in the tumor microenvironment of NSCLC through multiple mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3a0230d879f0812d1beffd1bcd78bcdc486ebd" target='_blank'>
              Effect of cryoablation on the spatial transcriptomic landscape of the immune microenvironment in non-small cell lung cancer.
              </a>
            </td>
          <td>
            Jun Zhou, Shengxi Wang, Ming Liu, Zhaopei Li
          </td>
          <td>2024-12-01</td>
          <td>Journal of cancer research and therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d052ec02be986508482ae1f68379c54cb0ed96e8" target='_blank'>
              Cell-type specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell Painting images offer valuable insights into a cell’s state and enable many biological applications, but publicly available arrayed datasets only include hundreds of genes perturbed. The JUMP (Joint Undertaking in Morphological Profiling) Cell Painting Consortium perturbed roughly 75% of the protein-coding genome in human U-2 OS cells, generating a rich resource of single-cell images and extracted features. These profiles capture the phenotypic impacts of perturbing 15,243 human genes, including overexpressing 12,609 genes (using open reading frames, ORFs) and knocking out 7,975 genes (using CRISPR-Cas9). We mitigated technical artifacts by rigorously evaluating data processing options and validated the dataset’s robustness and biological relevance. Analysis of phenotypic profiles revealed novel gene clusters and functional relationships, including those associated with mitochondrial function, cancer, and neural processes. The JUMP Cell Painting genetic dataset is a valuable resource for exploring gene relationships and uncovering novel functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be825882a184a6b2ca3b2ad476751e43d3e0e20d" target='_blank'>
              Morphological map of under- and over-expression of genes in human cells
              </a>
            </td>
          <td>
            S. Chandrasekaran, Eric Alix, John Arevalo, Adriana Borowa, Patrick J. Byrne, William G. Charles, Zitong S. Chen, B. Cimini, Boxiong Deng, John G Doench, Jessica D. Ewald, Briana Fritchman, Colin J. Fuller, Jedidiah Gaetz, Amy Goodale, Marzieh Haghighi, Yu Han, Zahra Hanifehlou, Holger Hennig, Desiree Hernandez, Christina B. Jacob, Tim James, T. Jetka, Alexandr A. Kalinin, B. Komalo, Maria Kost-Alimova, Tomasz Krawiec, Brittany A. Marion, Glynn Martin, Nicola Jane McCarthy, Lisa Miller, Arne Monsees, Nikita Moshkov, Alán F. Muñoz, Arnaud Ogier, Magdalena Otrocka, Krzysztof Rataj, D. Root, Francesco Rubbo, Simon Scrace, Douglas W. Selinger, Rebecca Senft, Peter Sommer, A. Thibaudeau, Sarah Trisorus, Rahul Valiya Veettil, Will Van Trump, Sui Wang, Michał Warchoł, Erin Weisbart, A. Weiss, Michael Wiest, Agata Zaremba, Andrei Zinovyev, Shantanu Singh, A. Carpenter
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>66</td>
        </tr>

        <tr id="Spatial metabolomics holds great promise for unraveling the complexities of metabolic reprogramming in cancer, yet its development lags behind the rapid progress of spatial transcriptomics (ST). To bridge this gap and maximize the value of spatial gene expression datasets, we present Spatial Flux Balance Analysis (spFBA), a novel framework that transcends the limitations of existing gene set enrichment methods. The spFBA approach builds upon previous work designed for bulk and single-cell data to return Flux Enrichment Scores, up to the level of single reactions, that can distinguish their preferred directional usage retaining spatial resolution. spFBA integrates differential constraints on flux boundaries in steady-state metabolic modeling, informed by spatial gene expression, and utilizes corner-based flux sampling. We first tuned and validated spFBA using a publicly available 10x Visium ST dataset from renal cancer samples at the tumor-normal tissue interface. spFBA demonstrated the ability to recapitulate the tissue architecture in renal tumor sections, clearly delineating sustained metabolic growth in the tumor core compared to adjacent normal renal parenchyma. Furthermore, spFBA captured the Warburg effect, revealing distinct metabolic subpopulations within the tumor core, while highlighting lactate excretion as a hallmark across all subpopulations. We then applied spFBA to our own high-resolution ST datasets generated with stereo-seq from a colon cancer patient, including both primary tumor and liver metastases. Unlike the renal case, spFBA uncovered extensive areas of tumor cells consuming lactate in both primary and metastatic sites, with stromal regions in liver metastases producing lactate, suggesting a reverse Warburg effect at play. These findings highlight that cancer metabolic rewiring and the tumor composition of metabolic subpopulations depend highly on the tissue of origin and the spatial context within the target organ. spFBA emerges as a powerful tool to leverage the rapidly growing collection of spatial gene expression data, offering unprecedented insights into the intricate metabolic landscape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a4d090d655d07f06eab7f172bddcddf9fddf74f" target='_blank'>
              Spatial Flux Balance Analysis reveals tissue-of-origin and spatially dependent metabolic rewiring in renal and colorectal cancer
              </a>
            </td>
          <td>
            Davide Maspero, Giovanni Marteletto, Francesco Lapi, B. Galuzzi, Irene Ruano, Ben Vandenbosch, Ke Yin, Sabine Tejpar, Alex Graudenzi, Holger Heyn, Anna Pascual-Reguant, Chiara Damiani
          </td>
          <td>2024-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The metastasis of solid tumors hinges on cancer cells navigating through complex three-dimensional tissue environments, characterized by mechanical heterogeneity and biological diversity. This process is closely linked to the dynamic migration behavior exhibited by cancer cells, which dictates the invasiveness of tumors. In our study, we investigate tumor spheroids composed of breast cancer cells embedded in three-dimensional (3D) collagen matrices. Through a combination of quantitative experiments, artificial-intelligence-driven image processing, and mathematical modeling, we uncover rapid transitions in cell phenotypes and phenotype-dependent motility among disseminating cells originating from tumor spheroids. Persistent invasion leads to continuous remodeling of the extracellular matrix surrounding the spheroids, altering the landscape of migration phenotypes. Consequently, filopodial cells emerge as the predominant phenotype across diverse extracellular matrix conditions. Our findings unveil the complex mesoscale dynamics of invading tumor spheroids, shedding light on the complex interplay between migration phenotype plasticity, microenvironment remodeling, and cell motility within 3D extracellular matrices.




 Published by the American Physical Society
 2024


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afedef8827a246c2a67d180498da9a907634ab61" target='_blank'>
              Coupled Dynamics in Phenotype and Tissue Spaces Shape the Three-Dimensional Cancer Invasion
              </a>
            </td>
          <td>
            Austin Naylor, Maximilian Libmann, Izabel Raab, W.-J. Rappel, Bo Sun
          </td>
          <td>2024-12-24</td>
          <td>PRX Life</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5361264ba7fa5b03a05e9caae598d056bebbf71c" target='_blank'>
              Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Yilv Yan, Dongqing Sun, Junjie Hu, Yue Chen, Liangdong Sun, Huansha Yu, Yicheng Xiong, Zhida Huang, Haoran Xia, Xinsheng Zhu, D. Bian, Fenghuan Sun, Likun Hou, Chunyan Wu, Orion R Fan, Haiyang Hu, An Zeng, Lele Zhang, Yi Eve Sun, Chenfei Wang, Peng Zhang
          </td>
          <td>2024-12-10</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND
Growth hormone-secreting pituitary adenomas (GHPA) display diverse biological behaviors and clinical outcomes, necessitating the identification of tumor heterogeneity and prognostically relevant markers.


METHODS
In this study, we performed single-cell RNA sequencing (scRNA-seq) on 10 GHPA samples, four of which also underwent spatial transcriptome sequencing, and used scRNA-seq data from four normal pituitary samples as controls. Cell subtype characterization in GHPA was analyzed using multiple algorithms to identify malignant bias regulators, which were then validated using a clinical cohort.


RESULTS
We constructed the first single-cell and spatial transcriptome profiles of GHPA, which contained 87,862 cells and revealed 16 tumor cell subtypes. Among the tumor cells, we identified distinct developmental trajectories and three malignant-biased subtypes (PIT1_C05, PIT1_C06, and PIT1_C10). The spatial distribution characteristics of these malignant-biased cells may influence the growth characteristics and prognosis of GHPA. We screened specific regulatory transcription factors, including FOXO1, GTF2IRD1, and MAX. Clinical cohort validation indicated that FOXO1 might be associated with tumor invasion and progression, while high expression of MAX could result in poor endocrine outcomes.


CONCLUSION
GHPA exhibits rich heterogeneity and diverse cell subtypes, with specific transcription factors potentially regulating cell malignant bias, thereby influencing tumor characteristics and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a105784b2d156549c656072efae7d8dac904126" target='_blank'>
              Single-cell and spatial transcriptomic analyses reveal heterogeneity characteristics and specific cell subtype regulators in growth hormone-secreting pituitary adenomas.
              </a>
            </td>
          <td>
            Yu Zhang, Lifeng Wang, Xingcheng Yi, Xinyu Ma, Hongyu Wu, Mingzhao Zhang, Zhenqi Yang, Lizhen Ma, Zenghua Mi, Wei-jia Zhi, Cong Fu, Pinan Liu, Zhijun Yang
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A comprehensive understanding of tumor heterogeneity, tumor microenvironment and the mechanisms of drug resistance is fundamental to advancing breast cancer research. While single-cell RNA sequencing has resolved the issue of “temporal dynamic expression” of genes at the single-cell level, the lack of spatial information still prevents us from gaining a comprehensive understanding of breast cancer. The introduction and application of spatial transcriptomics addresses this limitation. As the annual technical method of 2020, spatial transcriptomics preserves the spatial location of tissues and resolves RNA-seq data to help localize and differentiate the active expression of functional genes within a specific tissue region, enabling the study of spatial location attributes of gene locations and cellular tissue environments. In the context of breast cancer, spatial transcriptomics can assist in the identification of novel breast cancer subtypes and spatially discriminative features that show promise for individualized precise treatment. This article summarized the key technical approaches, recent advances in spatial transcriptomics and its applications in breast cancer, and discusses the limitations of current spatial transcriptomics methods and the prospects for future development, with a view to advancing the application of this technology in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9484c611ac13b7e4460cc6b1b5eb2c6b25e4a9" target='_blank'>
              Spatial transcriptomics in breast cancer: providing insight into tumor heterogeneity and promoting individualized therapy
              </a>
            </td>
          <td>
            Junsha An, Yajie Lu, Yuxi Chen, Yuling Chen, Zhaokai Zhou, Jianping Chen, Cheng Peng, Ruizhen Huang, Fu Peng
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer evolution follows complex trajectories involving diverse genomic alterations and clonal dynamics, making it challenging to validate computational methods for single-cell DNA sequencing analysis. Here we present cancerSimCraft, a comprehensive framework for simulating cancer genome data at both clonal and single-cell resolution. cancerSimCraft combines deterministic rules with stochastic processes to model various genomic events including CNVs, SNVs, and WGDs. The framework enables integration of real cancer genome patterns with user-defined parameters, supporting customizable simulation designs that reflect both empirical data and theoretical models. Through systematic benchmarking, we demonstrate cancerSimCraft’s utility in evaluating computational methods under various conditions, particularly focusing on the impact of dataset size and parameter sensitivity. We further demonstrated its application in exploring clonal evolution and mutation patterns through controlled in silico experiments. These comprehensive simulation capabilities make cancerSimCraft a valuable resource for both computational method development and theoretical studies in cancer genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617e592c6c24284ee3c33b23e1a58a1555ca1d22" target='_blank'>
              cancerSimCraft: A Multi-resolution Cancer Genome Simulator with Comprehensive Ground Truth Tracking
              </a>
            </td>
          <td>
            Haijing Jin, Nicholas Navin, Ken Chen
          </td>
          <td>2024-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Thymic epithelial cells (TECs) consist of two distinct components: cortical TECs (cTECs) and medullary TECs (mTECs), which are crucial for the positive and negative selection of thymocytes, respectively. Among thymic epithelial tumors (TETs) originating from TECs, thymic carcinomas (TCs) exhibit the worst prognosis. While the cell of origin of TCs has long been enigmatic, our previous study has demonstrated that TCs reflect the signature of mTECs utilizing bioinformatics approaches and immunohistochemistry of autoimmune regulator (AIRE), a molecule specific to mTECs. However, the cellular heterogeneity of TCs relevant to their pathogenesis remains elusive. It motivated us to dissect the tumor heterogeneity in more detail. Methods To investigate the cancer ecosystem underlying TCs, we have conducted a comprehensive spatial transcriptome analysis on two cases of TCs. We also utilized the bulk RNA-seq dataset of TETs from The Cancer Genome Atlas project archive and single-cell RNA-seq datasets of normal TECs and TC from the public database, ensuring the validity of our findings. Results Our spatial transcriptome analysis identified the spatial distribution of key molecules associated with TCs. While the well-known TC markers, KIT and CD5, were broadly expressed across the tumor tissues, the expression of AIRE was sporadic, mirroring its distribution in the normal thymus. Unsupervised clustering also identified a distinct cluster with a high level of AIRE expression, and this cluster expressed genes relevant to AIRE-expressing mTECs in the normal thymus, suggesting that mTEC heterogeneity was maintained in TC to some extent. Furthermore, re-analysis of single-cell RNA-seq data on TC revealed the existence of unique tumor subpopulations resembling thymic myoid cells and post-AIRE mTECs. Our spatially aware analysis also highlighted a unique gene module localized at the margin of tumor nests. Conclusions Spatial transcriptomics has given us new insights into the biology of TCs, further suggesting the cellular heterogeneity in TCs associated with mTECs. These findings have significant implications for our understanding of TCs and their pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49d6f81c5ca890050ea45a0fe342b2a626e984cb" target='_blank'>
              AB028. Spatial transcriptomics to dissect the cellular heterogeneity in thymic carcinoma
              </a>
            </td>
          <td>
            Minoru Matsumoto, Yasuyo Saijo, Koichi Tsuneyama, T. Oya
          </td>
          <td>2024-12-01</td>
          <td>Mediastinum</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d643827078776ad2a6449f6a7b6358707684af74" target='_blank'>
              Cell-cell interactions mediating primary and metastatic breast cancer dormancy.
              </a>
            </td>
          <td>
            Nicholas A Lenart, Shreyas S Rao
          </td>
          <td>2024-11-25</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Hybrid epithelial/mesenchymal (E/M) cells are the most metastatic of carcinoma cell types. Hybrid E/M cells have both epithelial and mesenchymal properties reminiscent of chimeras, mythical creatures composed of different animals. The co-expression of epithelial and mesenchymal proteins is a defining feature of hybrid E/M cancer cells. In this review, we discuss the use of cytokeratin (epithelial) and vimentin (mesenchymal) intermediate filaments as markers in hybrid E/M cells. We describe the structures and functions of these filaments that are uniquely expressed in hybrid E/M cells and speculate on potential hybrid structures formed between vimentin and cytokeratin. We also describe cell-surface cytokeratin and vimentin and their functions and discuss their potential as predictive biomarkers and as targets for anti-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0a63915420ccb6800640641805de420f117dd3" target='_blank'>
              The Cancer Chimera: Impact of Vimentin and Cytokeratin Co-Expression in Hybrid Epithelial/Mesenchymal Cancer Cells on Tumor Plasticity and Metastasis
              </a>
            </td>
          <td>
            Nick A. Kuburich, Julia M. Kiselka, P. den Hollander, Andrew A. Karam, S. Mani
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa47d77b71a7337ad0ecaef200be807283fa4101" target='_blank'>
              Heterogeneity-driven phenotypic plasticity and treatment response in branched-organoid models of pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            A. Papargyriou, Mulham Najajreh, David P Cook, Carlo Maurer, S. Bärthel, H. Messal, Sakthi K Ravichandran, Till Richter, Moritz Knolle, Thomas Metzler, Akul R Shastri, R. Öllinger, Jacob Jasper, Laura Schmidleitner, Surui Wang, C. Schneeweis, H. Ishikawa-Ankerhold, T. Engleitner, Laura Mataite, Mariia Semina, Hussein Trabulssi, S. Lange, Aashreya Ravichandra, Maximilian Schuster, S. Mueller, K. Peschke, A. Schäfer, S. Dobiasch, S.E. Combs, Roland M. Schmid, Andreas R. Bausch, R. Braren, I. Heid, C. Scheel, Günter Schneider, Anja Zeigerer, Malte Luecken, K. Steiger, G. Kaissis, J. van Rheenen, F. Theis, Dieter Saur, R. Rad, Maximilian Reichert
          </td>
          <td>2024-12-10</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Despite recent therapeutic breakthroughs, cancer patients continue to face high recurrence and mortality rates due to treatment resistance. Cancer stem cells (CSCs), a subpopulation with self-renewal capabilities, are key drivers of refractive disease. This review explores the application of molecular imaging techniques, such as PET and SPECT, for the noninvasive detection of CSCs. By providing real-time monitoring of CSCs, these imaging methods have the potential to predict therapy resistance and guide personalized treatment approaches. Here, we cover the biological characteristics of CSCs, mechanisms of therapy resistance, and the identification and targeting of CSC-specific biomarkers with molecular imaging. Additionally, we address the challenges and opportunities for the clinical translation of CSC imaging, highlighting strategies where CSC imaging can be used to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c598784fb4bf91a45e00e3bbc55f45f0d4b538" target='_blank'>
              Molecular Imaging of Cancer Stem Cells and Their Role in Therapy Resistance
              </a>
            </td>
          <td>
            Sofia N Dos Santos, T. Witney
          </td>
          <td>2025-01-01</td>
          <td>Journal of Nuclear Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88ba70c1f7c9e0c6817db8a69bc543b13dfed9df" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics of esophageal squamous cell carcinoma with lymph node metastases.
              </a>
            </td>
          <td>
            Wei Guo, Bolun Zhou, Lizhou Dou, Lei Guo, Yong Li, J. Qin, Zhen Wang, Qilin Huai, Xuemin Xue, Yin Li, Jianming Ying, Qi Xue, Shugeng Gao, Jie He
          </td>
          <td>2025-01-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="ABSTRACT Macrophages are pivotal in driving breast tumor development, progression, and resistance to treatment, particularly in estrogen receptor-positive (ER+) tumors, where they infiltrate the tumor microenvironment (TME) influenced by cancer cell-secreted factors. By analyzing single-cell RNA sequencing data from 25 ER+ tumors, we elucidated interactions between cancer cells and macrophages, correlating macrophage density with epithelial cancer cell density. We identified that S100A11, a previously unexplored factor in macrophage-cancer crosstalk, predicts high macrophage density and poor outcomes in ER+ tumors. We found that recombinant S100A11 enhances macrophage infiltration and migration in a dose-dependent manner. Additionally, in a 3D matrix using a panel of three ER+ breast cancer cell lines, we showed that secreted S100A11 levels from cancer cells were associated with increased monocyte infiltration that subsequently differentiation toward macrophages. Genetic silencing of S100A11 in the S100A11-high T47D cancer cells reduced monocyte infiltration, consistent with results using a S100A11 blocking antibody in T47D cancer cells and in a clinically relevant patient-derived organoid model. Phenotypic analysis of macrophages cocultured with T47D cancer cells following S100A11 knockdown revealed lower expression of the immunosuppressive marker CD206, further underscoring the role of S100A11 as a paracrine regulator of pro-tumorigenic cancer-macrophage crosstalk. This study offers novel insights into the interplay between macrophages and cancer cells in ER+ breast tumors, highlighting S100A11 as a potential therapeutic target to modulate the macrophage-rich tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bd1881c70cca02440b0e461e40eba12027d0287" target='_blank'>
              Cancer-cell derived S100A11 promotes macrophage recruitment in ER+ breast cancer
              </a>
            </td>
          <td>
            Sanghoon Lee, Youngbin Cho, Yiting Li, Ruxuan Li, Angela Wong Lau, Matthew S Laird, Daniel B Brown, Priscilla McAuliffe, Adrian V. Lee, S. Oesterreich, Ioannis K. Zervantonakis, H. Osmanbeyoglu
          </td>
          <td>2024-11-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Abstract Glioma evolution is governed by a multitude of dynamic interactions between tumor cells and heterogenous neighboring, non-cancerous cells. This complex ecosystem, termed the tumor microenvironment (TME), includes diverse immune cell types that have gained increasing attention for their critical and paradoxical roles in tumor control and tumorigenesis. Recent work has revealed that the cellular composition and functional state of immune cells in the TME can evolve extensively depending on the tumor stage and intrinsic features of surrounding glioma cells. Concurrently, adaptations to the glioma cellular phenotype, including activation of various cellular states, occur in the context of these immune cell alterations. In this review, we summarize important features of the immune TME that play key roles during each stage of glioma progression, from initiation to immune escape, invasion and recurrence. Understanding the complex interplay between tumor and immune cells is critical for the development of effective immunotherapies for glioma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6197e8609e0a64b363d0e798255e45455eaf7c8" target='_blank'>
              Co-evolution of glioma and immune microenvironment
              </a>
            </td>
          <td>
            Mahmoud M Elguindy, Jacob S Young, Winson S Ho, R. Lu
          </td>
          <td>2024-12-03</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancer heterogeneity has been well understood based solely on tumor components. However, the tumor microenvironment, which plays a crucial role in cancer by regulating tumor cells’ differentiation, maturation, and malignant potential, remains under-explored. This study conducted high-throughput RNA-Seq analysis on fresh breast cancer tissues from each molecular subtype and a pure population of cancer-associated fibroblasts (CAFs) isolated from cancer tissue and adjacent normal parts. Based on their functions, we identified three populations of CAFs: a) immune response-related, b) ECM remodeling, and c) calcium/protein binding. Validation of differentially expressed genes among these CAF populations revealed subtype-specific correlations. For instance, CAFs expressing immune response-related genes were significantly enriched in Luminal A/B and TNBC (log +1-fold; p =0.00047), compared to Her-2 positive (log -2.3-fold; p =0.0026). ECM remodeling genes exhibited greater expression in Her-2 positive and TNBC (log +4 to +11-fold; p =7.87E-08) when compared to Luminal subtypes (log +1.5-fold to +6-fold; p =8.66E-05). Calcium binding genes displayed overall similar expression across breast cancer subtypes. Thus, this study identified and underscored the functional heterogeneity of CAFs within the tumor microenvironment across the molecular subtypes of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e870b934402b9a2b10afe0f5e6b9ec8fa60b89" target='_blank'>
              Deciphering Functional Heterogeneity of Cancer-Associated Fibroblasts Across Molecular Subtypes of Breast Cancer
              </a>
            </td>
          <td>
            D. Kashyap
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9cdfae4578e26b8066f9cf63fa62b59d86c2b7" target='_blank'>
              Exploring tumor microenvironment interactions and apoptosis pathways in NSCLC through spatial transcriptomics and machine learning.
              </a>
            </td>
          <td>
            Huimin Li, Yuheng Jiao, Yi Zhang, Junzhi Liu, Shuixian Huang
          </td>
          <td>2024-12-19</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) constitute the primary cellular component of the stroma in chordomas, characterized by an abundance of mucinous stromal elements, potentially facilitating their initiation and progression; however, this inference has yet to be fully confirmed. In this study, single-cell RNA sequencing (scRNA-seq), spatial transcriptomics (ST), bulk RNA-seq, multiplexed quantitative immunofluorescence (QIF), and in vivo and in vitro experiments were performed to determine the heterogeneity, spatial distribution, and clinical significance of CAFs in chordoma. ScRNA-seq was performed on 87,693 single cells derived from seven tumor samples and four control nucleus pulposus samples. A distinct CAF cluster distinguished by the upregulated expression of inflammatory genes and enriched functionality in activating inflammation-associated cells was identified. Pseudotime trajectory and cell communication analyses suggested that this inflammatory CAF (iCAF) subset originated from normal fibroblasts and interacted extensively with tumors and various other cell types. By integrating the scRNA-seq results with ST, the presence of iCAF in chordoma tissue was further confirmed, indicating their positioning at a distance from the tumor cells. Bulk RNA-seq data analysis from 126 patients revealed a correlation between iCAF signature scores, chordoma invasiveness, and poor prognosis. QIF validation involving an additional 116 patients found that although iCAFs were not in close proximity to tumor cells compared with other CAF subsets, their density correlated with malignant tumor phenotypes and adverse outcomes. In vivo and in vitro experiments further confirmed that iCAFs accelerate the malignant progression of chordomas. These findings could provide insights into the development of novel therapeutic strategies. © 2024 The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e399d12e04054b6320b0ebd4486fa4b22de699e6" target='_blank'>
              Multi-omics analysis unveils the role of inflammatory cancer-associated fibroblasts in chordoma progression.
              </a>
            </td>
          <td>
            Bo-Wen Zheng, Wei Guo
          </td>
          <td>2024-11-29</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor heterogeneity is a significant factor influencing cancer treatment effectiveness and can arise from genetic, epigenetic, and phenotypic variations among cancer cells. Understanding how tumor heterogeneity impacts tumor evolution and therapy response can lead to more effective treatments and improved patient outcomes. Traditional bulk genomic approaches fail to provide insights into cellular-level events, whereas single-cell RNA sequencing (scRNA-seq) offers transcriptomic analysis at the individual cell level, advancing our understanding of tumor growth, progression, and drug response. However, implementing single-cell approaches in clinical trials involves challenges, such as obtaining high-quality cells, technical variability, and the need for complex computational analysis. Effective implementation of single-cell genomics in clinical trials requires a collaborative “Team Medicine” approach, leveraging shared resources, expertise, and workflows. Here, we describe key technical considerations in implementing the collection of research biopsies and lessons learned from integrating scRNA-seq into City of Hope’s clinical trial design, highlighting collaborative efforts between computational and clinical teams across breast, brain, and ovarian cancer studies to understand the composition, phenotypic state, and underlying resistance mechanisms within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e501cb60571ffc07f7277f4aa067130d5278a3" target='_blank'>
              Single-Cell Transcriptomics Sheds Light on Tumor Evolution: Perspectives from City of Hope’s Clinical Trial Teams
              </a>
            </td>
          <td>
            Patrick A. Cosgrove, Andrea H. Bild, T. Dellinger, B. Badie, J. Portnow, Aritro Nath
          </td>
          <td>2024-12-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background: Microvascular invasion (MVI) is a critical prognostic factor in intrahepatic cholangiocarcinoma (ICC), strongly associated with postoperative recurrence. However, the phenotypic features and spatial organization of MVI remain inadequately understood. Methods: We performed a spatial transcriptomic analysis on 29,632 spots from six ICC samples, manually delineating MVI clusters using the cloupe software. Key biomarkers were identified and validated in an independent cohort of 135 ICC patients. Functional and survival analyses were conducted to assess clinical relevance, and cell-cell communication pathways were investigated. Results: MVI regions exhibited heightened proliferation, angiogenesis, and epithelial-mesenchymal transition, driven by increased expression of transcription factors SOX10, ZEB1, and SNAI2. CXCL5 and SLC6A14 were identified as potential MVI biomarkers and showed high expression in tumor-invasive areas. Serum CXCL5 demonstrated strong predictive power for vascular invasion (AUC = 0.92) and intrahepatic metastasis (AUC = 0.96). High expression of both CXCL5 and SLC6A14 was associated with the worst survival outcomes. MVI regions were enriched with immunosuppressive MRC1+ macrophages and exhibited elevated immune checkpoint expression, including HAVCR2 and TIGHT, indicative of immune resistance. Cell-cell communication analysis revealed CXCL5-CXCR2 and LGALS9-HAVCR2 as key ligand-receptor pairs contributing to the immunosuppressive microenvironment. Conclusions: This study identifies CXCL5 and SLC6A14 as key biomarkers of MVI, highlighting their roles in tumor proliferation, immune resistance, and poor clinical outcomes. These findings provide valuable insights into the spatial organization of MVI and its contribution to ICC progression, offering potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80f1aeb6ba61c9f3ee11c5dafb6bfea9349b09d" target='_blank'>
              Spatial analyses revealed CXCL5 and SLC6A14 as the markers of microvascular invasion in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            G. Fan, L. Dai, Tongji Xie, Lin Li, Le Tang, Xiaohong Han, Yuankai Shi
          </td>
          <td>2024-12-11</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In 2024, the United States was projected to experience 2 million new cancer diagnoses and approximately 611,720 cancer-related deaths, reflecting a broader global trend in which cancer cases are anticipated to exceed 35 million by 2050. This increasing burden highlights ongoing challenges in cancer treatment despite significant advances that have reduced cancer mortality by 31% since 1991. Key obstacles include the disease’s inherent heterogeneity and complexity, such as treatment resistance, cancer stem cells, and the multifaceted tumor microenvironment (TME). The TME—comprising various tumor and immune cells, blood vessels, and biochemical factors—plays a crucial role in tumor growth and resistance to therapies. Recent innovations in cancer treatment, particularly in the field of immuno-oncology, have leveraged insights into TME interactions. An emerging example is the FDA-approved therapy using tumor-infiltrating lymphocytes (TILs), demonstrating the potential of cell-based approaches in solid tumors. However, TIL therapy is just one of many strategies being explored. This review provides a comprehensive overview of the emerging field of immuno-oncology, focusing on how novel therapies targeting or harnessing components of the TME could enhance treatment efficacy and address persistent challenges in cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69efe2b80b4ba6460d216813b5cdf5408b15b18" target='_blank'>
              Targeting Cancer: Microenvironment and Immunotherapy Innovations
              </a>
            </td>
          <td>
            Irena Padzińska-Pruszyńska, B. Taciak, Łukasz Kiraga, A. Smolarska, M. Górczak, P. Kucharzewska, M. Kubiak, J. Szeliga, Agata Matejuk, M. Król
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1000f3a44e28fde956bb08113a825d4344bfa7a6" target='_blank'>
              Proteogenomic analysis reveals non-small cell lung cancer subtypes predicting chromosome instability, and tumor microenvironment
              </a>
            </td>
          <td>
            K. J. Song, Seunghyuk Choi, Kwoneel Kim, H. S. Hwang, E. Chang, Ji Soo Park, Seok Bo Shim, Seunghwan Choi, Yong Jin Heo, Woo Ju An, Dae Yeol Yang, Kyung-Cho Cho, W. Ji, Chang-Min Choi, Jae Cheol Lee, H. Kim, Jiyoung Yoo, Hee-Sung Ahn, Gang-Hee Lee, Chanwoong Hwa, Seoyeon Kim, Kyunggon Kim, Minsuh Kim, Eunok Paek, Seungjin Na, S. J. Jang, J. An, Kwang Pyo Kim
          </td>
          <td>2024-11-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185d03d5b622173898f4a312b550c66a7f8243e6" target='_blank'>
              Soft tissue sarcomas at the single-cell and spatial resolution: new markers and targets.
              </a>
            </td>
          <td>
            M. Menyailo, Elena E. Kopantseva, A. Khozyainova, Anastasia A. Korobeynikova, E. Denisov
          </td>
          <td>2024-11-24</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaacb86999aeed003c707b4000adb56d67bb6547" target='_blank'>
              Spatial transcriptome profiling identifies DTX3L and BST2 as key biomarkers in esophageal squamous cell carcinoma tumorigenesis
              </a>
            </td>
          <td>
            Rutao Li, Na Li, Qianqian Yang, Xing Tong, Wei Wang, Chang Li, Jun Zhao, Dong Jiang, Haitao Huang, Chen Fang, Kai Xie, Jiamin Yuan, Shaomu Chen, Guangbin Li, Haitao Luo, Zhibo Gao, Dongfang Wu, Xiaoli Cui, Wei Jiang, Lingchuan Guo, Haitao Ma, Yu Feng
          </td>
          <td>2024-12-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The resistance of tumors to immune checkpoint inhibitors (ICI) may be intricately linked to cellular senescence, although definitive clinical validation remains elusive. In this study, comprehensive pan-cancer scRNA-seq analyses identify fibroblasts as exhibiting the most pronounced levels of cellular senescence among tumor-associated cell populations. To elucidate this phenomenon, a fibroblast senescence-associated transcriptomic signature (FSS), which correlated strongly with protumorigenic signaling pathways and immune dysregulation that fosters tumor progression, is developed. Leveraging the FSS, the machine learning (ML) framework demonstrates exceptional accuracy in predicting ICI response and survival outcomes, achieving superior area under curve (AUC) values across validation, testing, and in-house cohorts. Strikingly, FSS consistently outperforms established signatures in predictive robustness across diverse cancer subtypes. From an integrative analysis of 17 CRISPR/Cas9 libraries, CDC6 emerges as a pivotal biomarker for pan-cancer ICI response and prognostic stratification. Mechanistically, experimental evidence reveals that CDC6 in tumor cells orchestrates fibroblast senescence via TGF-β1 secretion and oxidative stress, subsequently reprogramming the tumor microenvironment and modulating ICI response. These findings underscore the translational potential of targeting fibroblast senescence as a novel therapeutic strategy to mitigate immune resistance and enhance antitumor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83eb29659bd661ae208e4642398f8b3b7069e404" target='_blank'>
              A Distinguished Roadmap of Fibroblast Senescence in Predicting Immunotherapy Response and Prognosis Across Human Cancers.
              </a>
            </td>
          <td>
            Dongjie Chen, Pengyi Liu, Jiayu Lin, Longjun Zang, Yihao Liu, Shuyu Zhai, Xiongxiong Lu, Yuanchi Weng, Hongzhe Li
          </td>
          <td>2024-12-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary The landscape of lung cancer diagnosis and treatment has transformed over the past two decades, ushering in the era of precision medicine for lung cancer. In systemic therapy alone, 43 drugs have gained FDA approval since 2020, encompassing chemotherapy, molecularly targeted therapies, immunotherapy, and antibody–drug conjugates. These therapies have significantly improved patient survival and quality of life. Improved preclinical models have been crucial in driving these advancements. Increasingly, the impact of tumor heterogeneity, along with its interplay with the tumor microenvironment and immune system, has been recognized as critical in influencing responses to these agents. This review provides an overview of the key characteristics, advantages, and limitations of current in vitro and in vivo models used in functional precision lung cancer research of systemic therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e6df5f9238d9ad621f566e7e05cac7d4b8b345" target='_blank'>
              Preclinical Models for Functional Precision Lung Cancer Research
              </a>
            </td>
          <td>
            Jie-Zeng Yu, Zsofia Kiss, Weijie Ma, Ruqiang Liang, Tianhong Li
          </td>
          <td>2024-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Fibroblasts are present in all tissues and are crucial for maintaining tissue homeostasis. We previously identified fibroblasts marked by Dermatopontin (Dpt) but their role in supporting macrophage homeostasis remains unclear. Here, we generated novel mesenchymal lineage-restricted genetic tools to target Dpt expressing fibroblasts and elucidate their role in supporting skin macrophages. Transcriptional profiling, flow cytometry, and in situ hybridization uncovered two broad populations of F4/80-expressing skin macrophages, denoted by high expression of CD206 and CD64 (CD206hiCD64+), or CD11c. Targeted depletion of Dpt+ fibroblasts resulted in a profound loss of both macrophage populations. Conditional deletion of colony-stimulating factor-1 (Csf1) in Dpt+ fibroblasts revealed that CD206hiCD64+, and not CD11c+, macrophages are acutely dependent on fibroblast-derived Csf1, consistent with their higher expression of the Csf1 receptor. Following Csf1 deletion in Dpt+ fibroblasts, loss of CD206hiCD64+ macrophages were observed across the dermis, dermal white adipose tissue (dWAT), and adventitia, accompanied by a modest upregulation of fibroblast-related and extracellular matrix (ECM) genes and structural changes to the skin. Alterations to the skin network upon loss of fibroblast-derived Csf1 and CD206hiCD64+ macrophages led to a significant delay in wound healing. We also demonstrate the CSF1-CSF1R signaling pathway is functionally relevant in human systemic sclerosis, or scleroderma, as elevated levels of CSF1 produced by fibroblasts and an increased abundance of macrophages both correlate with disease severity. Our findings demonstrate the role of Dpt+ fibroblasts in regulating a Csf1-dependent macrophage niche in skin and orchestrating responses in injury and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db93e9ec31ae21d3580950e276b23572a594b17" target='_blank'>
              Dermatopontin-expressing fibroblasts mediate an essential skin macrophage niche
              </a>
            </td>
          <td>
            A. Vollmers, Sunny Z. Wu, Anthony Altieri, Erika E. McCartney, Hannah Bender, Wyne P. Lee, Juan Zhang, Crystal Hu, Salil Uttarwar, J. V. Vander Heiden, Christopher Davidson, Yein Chung, Willie Ortiz, Michael Long, Raymond Asuncion, Yeqing Angela Yang, Jean X. Jiang, Z. Modrusan, Akshay T. Krishnamurty, Wenxian Fu, Sören Müller, M. Buechler, Shannon J. Turley
          </td>
          <td>2024-11-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58121f921b0bd9812f510ea0edf4c0cb4c87aa52" target='_blank'>
              Spatial transcriptomic analysis drives PET imaging of tight junction protein expression in pancreatic cancer theranostics
              </a>
            </td>
          <td>
            James Wang, J. Seo, A. Kare, Martin Schneider, Mallesh Pandrala, Spencer K. Tumbale, M. Raie, Gokce Engudar, Nisi Zhang, Yutong Guo, Xiaoxu Zhong, Sofia Ferreira, Bo Wu, Laura D Attardi, G. Pratx, Andrei Iagaru, Ryan L. Brunsing, Gregory W Charville, Walter G. Park, Katherine W Ferrara
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Cell–cell communications (CCCs) involve signaling from multiple sender cells that collectively impact downstream functional processes in receiver cells. Currently, computational methods are lacking for quantifying the contribution of pairwise combinations of cell types to specific functional processes in receiver cells (e.g. target gene expression or cell states). This limitation has impeded understanding the underlying mechanisms of cancer progression and identifying potential therapeutic targets. Here, we proposed a deep learning-based method, scDCA, to decipher the dominant cell communication assembly (DCA) that have a higher impact on a particular functional event in receiver cells from single-cell RNA-seq data. Specifically, scDCA employed a multi-view graph convolution network to reconstruct the CCCs landscape at single-cell resolution, and then identified DCA by interpreting the model with the attention mechanism. Taking the samples from advanced renal cell carcinoma as a case study, the scDCA was successfully applied and validated in revealing the DCA affecting the crucial gene expression in immune cells. The scDCA was also applied and validated in revealing the DCA responsible for the variation of 14 typical functional states of malignant cells. Furthermore, the scDCA was applied and validated to explore the alteration of CCCs under clinical intervention by comparing the DCA for certain cytotoxic factors between patients with and without immunotherapy. In summary, scDCA provides a valuable and practical tool for deciphering the cell type combinations with the most dominant impact on a specific functional process of receiver cells, which is of great significance for precise cancer treatment. Our data and code are free available at a public GitHub repository: https://github.com/pengsl-lab/scDCA.git.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96c6383c760f6614481b5fab3bdbf070d3264811" target='_blank'>
              scDCA: deciphering the dominant cell communication assembly of downstream functional events from single-cell RNA-seq data
              </a>
            </td>
          <td>
            Boya Ji, Xiaoqi Wang, Xiang Wang, Liwen Xu, Shaoliang Peng
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286075cb8f68cf0d587bedcc36d60116c411594e" target='_blank'>
              Spatially resolved gene expression profiling of tumor microenvironment reveals key steps of lung adenocarcinoma development
              </a>
            </td>
          <td>
            Yuma Takano, Jun Suzuki, Kotaro Nomura, Gento Fujii, Junko Zenkoh, Hitomi Kawai, Yuta Kuze, Y. Kashima, Satoi Nagasawa, Yuka Nakamura, M. Kojima, K. Tsuchihara, Masahide Seki, A. Kanai, D. Matsubara, Takashi Kohno, Masayuki Noguchi, Akihiro Nakaya, Masahiro Tsuboi, G. Ishii, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-12-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Lineage plasticity and histologic transformation from prostate adenocarcinoma to neuroendocrine prostate cancer (NEPC) occurs in up to 15-20% of patients with castration-resistant prostate cancer (CRPC) as mechanism of treatment resistance and is associated with aggressive disease and poor prognosis. NEPC tumors typically display small cell carcinoma morphology with loss of androgen receptor (AR) expression and gain of neuroendocrine (NE) lineage markers. However, there is a spectrum of phenotypes that are observed during the lineage plasticity process, and the clinical significance of mixed histologies or those that co-express AR and NE markers or lack all markers is not well defined. Translational research studies investigating NEPC have used variable definitions making clinical trial design challenging. Here we discuss the diagnostic workup of metastatic biopsies to help guide the reproducible classification of phenotypic CRPC subtypes. We recommend classifying CRPC tumors based on histomorphology (adenocarcinoma, small cell carcinoma, poorly differentiated carcinoma, other morphologic variant, or mixed morphology) and immunohistochemical markers with a priority for AR, NKX3.1, INSM1, synaptophysin and cell proliferation based on Ki-67 positivity, with additional markers to be considered based on the clinical context. Ultimately, a unified workup of metastatic CRPC biopsies can improve clinical trial design and eventually practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ba90573d033c02c8e63a055556c5045dc3b6fc" target='_blank'>
              Framework for the Pathology Workup of Metastatic Castration-Resistant Prostate Cancer Biopsies.
              </a>
            </td>
          <td>
            Michael C Haffner, Michael J. Morris, C. C. Ding, Erolcan Sayar, Rohit Mehra, Brian D. Robinson, Lawrence D. True, Martin Gleave, Tamara L. Lotan, Rahul Aggarwal, Jiaoti Huang, Massimo Loda, Peter S Nelson, Mark A. Rubin, H. Beltran
          </td>
          <td>2024-11-26</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/602ca4326619720e872bb3125e86fd5844dc2d44" target='_blank'>
              A multiscale model of immune surveillance in micrometastases gives insights on cancer patient digital twins
              </a>
            </td>
          <td>
            Heber L Rocha, Boris Aguilar, Michael Getz, I. Shmulevich, P. Macklin
          </td>
          <td>2024-12-04</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>2</td>
          <td>83</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive subtype of breast cancer, with the worst prognosis among all subtypes. The impact of distinct cell subpopulations within the tumor microenvironment (TME) on TNBC patient prognosis has yet to be clarified. Methods Utilizing single-cell RNA sequencing (scRNA-seq) integrated with bulk RNA sequencing (bulk RNA-seq), we applied Cox regression models to compute hazard ratios, and cross-validated prognostic scoring using a GLMNET-based Cox model. Cell communication analysis was used to elucidate the potential mechanisms of CPVL and MSR1. Ultimately, RNA interference-mediated gene knockdown was utilized to validate the impact of specific genes on the polarization of tumor-associated macrophages (TAMs). Results Our findings revealed that the function of immune cells is more pivotal in prognosis, with TAMs showing the strongest correlation with TNBC patient outcomes, compared with other immune cells. Additionally, we identified CPVL and MSR1 as critical prognostic genes within TAMs, with CPVL expression positively correlated with favorable outcomes and MSR1 expression associated with poorer prognosis. Mechanistically, CPVL may contribute to favorable prognosis by inhibiting the SPP1-CD44 ligand-receptor and promoting CXCL9-CXCR3, C3-C3AR1 ligand-receptor, through which TAMs interact with other cells such as monocytes, neutrophils, and T cells. Moreover, cytokines including IL-18, IFNγR1, CCL20, and CCL2, along with complement-related gene like TREM2 and complement component CFD, may participate in the process of CPVL or MSR1 regulating macrophage polarization. Furthermore, RT-PCR experiments confirmed that CPVL is positively associated with M1-like TAM polarization, while MSR1 is linked to M2-like TAM polarization. Finally, the prognostic significance of these two genes is also validated in HER2-positive breast cancer subtypes. Conclusions CPVL and MSR1 are potential biomarkers for macrophage-mediated TNBC prognosis, suggesting the therapeutic potential of macrophage targeting in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af61ce193245e3b400806d4fcea4eed6dfe425f5" target='_blank'>
              Single-cell Atlas reveals core function of CPVL/MSR1 expressing macrophages in the prognosis of triple-negative breast cancer
              </a>
            </td>
          <td>
            Xinan Wang, Li Lin, Xue Zhang, Minghui Zhang, Zhuo Sun, Yichen Yang, Xiuna Zhang, Yonghui Yuan, Yong Zhang, Hao Chen, Ti Wen
          </td>
          <td>2024-12-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
A spatially resolved, niche-level analysis of tumour microenvironments (TME) can provide insights into cellular interactions and their functional impacts in gastric cancers (GC).


OBJECTIVE
Our goal was to translate the spatial organisation of GC ecosystems into a functional landscape of cellular interactions involving malignant, stromal and immune cells.


DESIGN
We performed spatial transcriptomics on nine primary GC samples using the Visium platform to delineate the transcriptional landscape and dynamics of malignant, stromal and immune cells within the GC tissue architecture, highlighting cellular crosstalks and their functional consequences in the TME.


RESULTS
GC spatial transcriptomes with substantial cellular heterogeneity were delineated into six regional compartments. Specifically, the fibroblast-enriched TME upregulates epithelial-to-mesenchymal transformation and immunosuppressive response in malignant and TME cells, respectively. Cell type-specific transcriptional dynamics revealed that malignant and endothelial cells promote the cellular proliferations of TME cells, whereas the fibroblasts and immune cells are associated with procancer and anticancer immunity, respectively. Ligand-receptor analysis revealed that CCL2-expressing fibroblasts promote the tumour progression via JAK-STAT3 signalling and inflammatory response in tumour-infiltrated macrophages. CCL2+ fibroblasts and STAT3-activated macrophages are co-localised and their co-abundance was associated with unfavourable prognosis. We experimentally validated that CCL2+ fibroblasts recruit myeloid cells and stimulate STAT3 activation in recruited macrophages. The development of immunosuppressive TME by CCL2+ fibroblasts were also validated in syngeneic mouse models.


CONCLUSION
GC spatial transcriptomes revealed functional cellular crosstalk involving multiple cell types among which the interaction between CCL2+ fibroblasts and STAT3-activated macrophages plays roles in establishing immune-suppressive GC TME with potential clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe72f4555eff64ae7e607a98dd2b2dd8cb471c6e" target='_blank'>
              Spatial dissection of tumour microenvironments in gastric cancers reveals the immunosuppressive crosstalk between CCL2+ fibroblasts and STAT3-activated macrophages.
              </a>
            </td>
          <td>
            Sung Hak Lee, Dagyeong Lee, Junyong Choi, Hye Jeong Oh, In-Hye Ham, Daeun Ryu, Seo-Yeong Lee, Dong-Jin Han, Sunmin Kim, Youngbeen Moon, In-Hye Song, K. Song, Hyeseong Lee, Seungho Lee, Hoon Hur, Tae-Min Kim
          </td>
          <td>2024-11-23</td>
          <td>Gut</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and malignant type of brain tumor in adults and no effective therapies exist to combat this disease. Surgical resection, radio- and temozolomide-based-chemotherapy are insufficient mainly due to inter- and intra-tumoral heterogeneity of GBM tumors, therefore personalized approaches are required. Additionally, diverse tumor microenvironment with a leading role of glioblastoma stem cells (GSCs) in hypoxic core underlie the aggressiveness and recurrence of GBM. However, the molecules and pathways which directly promote tumor aggressiveness together with the driving process of epithelial-to-mesenchymal transition (EMT) and the components of extracellular matrix (ECM), remain only partially covered with prior studies focused mainly on coding oncogenes and tumor suppressor genes. Over the past years, a special attention has been paid to non-coding, circular RNAs (circRNAs) differentially expressed in various cancers, including GBM. Despite high abundance and stability, the exact biological roles of most individual circRNAs have not been fully revealed, so far. Besides anti-cancer targets, circRNAs have been considered as essential biomarkers and stratifying criterion of malignancy. Thus, studies on functions of differentially expressed circRNAs in GBM are an emerging field to be explored. In herein study, the biological role of circRNA CLIP2 (circCLIP2) was found highly overexpressed in primary tumor tissues. Upon the knock-down of circCLIP2 in 2D and 3D in vitro models, functional tests performed clearly indicate a significant contribution of circCLIP2 to aggressive potential of GBM as showed by decreased rates of proliferation, migration and invasion. Additional studies indicated predominant expression of circCLIP2 in GSC fractions additionally elevated by hypoxic conditions. The increased rates of proliferation and migration as well as molecular analyses were also confirmed in in vivo settings where xenograft model was applied. The expression of EMT marker Snail2 and ECM-degrading enzyme metalloproteinase 9 (MMP9) were significantly downregulated in the tumors formed from circCLIP2 silenced cells. Taken together, these results indicate a strong contribution of circCLIP2 in the aggressive phenotype of GBM which is due to a modulation of GSC proliferative potential as well as migration and invasion via EMT together with the modulator of ECM composition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41e1172182008f639a17a4a3feb76b12f54727b" target='_blank'>
              Circular RNA - circCLIP2 is predominantly expressed in GSC niche and enhances glioblastoma aggressiveness via EMT and ECM signaling
              </a>
            </td>
          <td>
            J. Misiorek, Ż. Zarębska, K. Kuczyński, J. Latowska-Łysiak, A. Grabowska, P. Głodowicz, M. Sajek, Anna Karlik, Dorota Wronka, A. Barciszewska, Łukasz Przybył, K. Rolle
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ace1bf5ccc7bc1a835b1fd333db7f3f67e0b96" target='_blank'>
              Lessons learned from spatial transcriptomic analyses in clear-cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Jesper Jespersen, Cecilie Lindgaard, Laura Iisager, J. Ahrenfeldt, I. Lyskjær
          </td>
          <td>2025-01-09</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Dendritic cells (DCs) are promising targets for cancer immunotherapies because of their central role in the initiation and control of immune responses. The rare cDC1 population is of particular interest because of its remarkable ability to cross-present antigens (Ag) to CD8+ T cells, to promote Th1 cell polarization and NK cell activation and recruitment. However, the spatial organization and specific functions of cDC1s in response to immunotherapy remain to be clearly characterized in human tumors. Here, we implemented a multiplexed immunofluorescence analysis pipeline coupled with computational image analysis to determine the spatial organization of the cDC1 subset in a cohort of skin lesions from advanced melanoma patients treated with immune checkpoint inhibitors (ICI). For this, we performed a whole-slide image analysis of cDC1 infiltration and distribution as well as their spatial interactions with key immune partners such as CD8+ T cells and pDC according to the response of patients to ICI. We also analyzed LAMP3+-DC, which correspond to a mature subset of tumor-infiltrating DCs. Distance and cell network analyses demonstrated that cDC1s exhibited a scattered distribution compared to tumor-infiltrating pDCs and LAMP3+-DCs, which were preferentially organized in dense areas with high homotypic connections. Interestingly, the proximity and interactions between CD8+ T cells and cDC1s were positively associated with the response to ICI. In conclusion, our study unravels the complex spatial organization of cDC1s and their interactions with CD8+ T cells in melanoma patient lesions, shedding light on their pivotal role in shaping the response to ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10d121483bbab07523097b63187cdc94830056d" target='_blank'>
              The spatial organization of cDC1 with CD8+ T cells is critical for the response to immune checkpoint inhibitors in melanoma patients.
              </a>
            </td>
          <td>
            Elisa Gobbini, Margaux Hubert, A. Doffin, Anaïs Eberhardt, Léo Hermet, Danlin Li, Pierre Duplouye, Sarah Ghamry-Barrin, Justine Berthet, Valentin Benboubker, Maxime Grimont, C. Sakref, Jimmy Perrot, Garance Tondeur, Olivier Harou, Jonathan Lopez, Bertrand Dubois, Stéphane Dalle, Christophe Caux, J. Caramel, J. Valladeau-Guilemond
          </td>
          <td>2025-01-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80cae72455d0f13cb986669bd3a06f266bb4aab" target='_blank'>
              Spatial transcriptomics reveals substantial heterogeneity in triple-negative breast cancer with potential clinical implications
              </a>
            </td>
          <td>
            Xiaoxiao Wang, D. Venet, F. Lifrange, D. Larsimont, Mattia Rediti, L. Stenbeck, F. Dupont, G. Rouas, Andrea Joaquin Garcia, L. Craciun, L. Buisseret, Michail Ignatiadis, M. Carausu, N. Bhalla, Yuvarani Masarapu, Eva Gracia Villacampa, L. Franzén, Sami Saarenpää, L. Kvastad, K. Thrane, J. Lundeberg, F. Rothé, Christos Sotiriou
          </td>
          <td>2024-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="The growth and progression of a neoplasm is a multi–stage process mediated by interactions between tumor cells and their surrounding stroma including various cell types (fibroblasts, endothelial cells, immune cells), cell-free components (cytokines, extracellular vesicles and extracellular matrix), blood and lymphatic vessels that form the tumor microenvironment. During the progression of neoplasms, tumor cells can evade the immune response by suppressing or losing tumor antigens releasing immunosuppressive extracellular vesicles and exosomes, secreting various cytokines, chemokines and other factors, and begin to reprogram and remodel the surrounding stroma. The most relevant areas of TME study are cells and molecules of innate and acquired immunity exhibited both suppressive and stimulating activity against tumor cells due to their high heterogeneity and plasticity. Special attention is paid to immune control points, tumor microbiota, components of intercellular interaction as well as immunotherapy of tumor diseases.
In this paper, a systematic review of scientific publications in PubMed database devoted to the study of TME and published in the period from 2019 to 2024 was carried out. 2547 sources were studied and 68 articles were selected during the screening process. Search queries and keywords included «tumor microenvironment», «immune cells», «tumor», «oncogenesis», «metastasis». The formed group of scientific articles will allow to study in more detail the cellular and molecular
composition of the tumor microenvironment, as well as the main targets for the development of new approaches for malignant neoplasms diagnosis and therapy involving TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/568399709b478d3282ff1a127c4698c1e28c9300" target='_blank'>
              Tumor microenvironment: systematic review
              </a>
            </td>
          <td>
            A. V. Vialichka, D. B. Nizheharodava, Elizaveta M. Nazaranka, Bogdan A. Muzychenka, M. Zafranskaya
          </td>
          <td>2024-12-27</td>
          <td>Journal of the Belarusian State University. Ecology.</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Significance Our study provides functional evidence that metabolic reprogramming can directly impact cancer phenotypes and establishes proliferator-activated receptor gamma coactivator 1-alpha (PGC-1α)-induced mitochondrial metabolism as a driver of small cell neuroendocrine cancer (SCNC) progression and lineage determination. These mechanistic insights reveal common metabolic vulnerabilities across SCNCs originating from multiple tissues, opening additional avenues for pan-SCN cancer therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a586b3dcd3d3eae51970eccbac4cd9bc30b11e95" target='_blank'>
              PGC-1α drives small cell neuroendocrine cancer progression toward an ASCL1-expressing subtype with increased mitochondrial capacity
              </a>
            </td>
          <td>
            Grigor Varuzhanyan, Chia-Chun Chen, Jack Freeland, Tian He, Wendy Tran, Kai Song, Liang Wang, Donghui Cheng, Shili Xu, Gabriella A. Dibernardo, Favour N Esedebe, Vipul Bhatia, Mingqi Han, Evan R. Abt, Jung Wook Park, Sanaz Memarzadeh, David B. Shackelford, John K. Lee, Thomas G Graeber, O. Shirihai, Owen N. Witte
          </td>
          <td>2024-11-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c843d34d2321464470845817a613dfc68c436a" target='_blank'>
              Integrin α8 is a useful cell surface marker of alveolar lipofibroblasts
              </a>
            </td>
          <td>
            A. Fukada, Yasunori Enomoto, R. Horiguchi, Y. Aoshima, S. Meguro, H. Kawasaki, I. Kosugi, T. Fujisawa, N. Enomoto, Naoki Inui, T. Suda, T. Iwashita
          </td>
          <td>2025-01-13</td>
          <td>Respiratory Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49801de5fa9e55e176d4a57170a423c31755f306" target='_blank'>
              High-resolution, noninvasive single-cell lineage tracing in mice and humans based on DNA methylation epimutations.
              </a>
            </td>
          <td>
            Mengyang Chen, Ruijiang Fu, Yiqian Chen, Li Li, Shoucheng Wang
          </td>
          <td>2025-01-16</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) tend to acquire genetic aberrations upon culture in vitro. Common aberrations are mutations in the tumor suppressor TP53, suspected to confer a growth-advantage to the mutant cells. However, their full impact in the development of malignant features and safety of hPSCs for downstream applications is yet to be elucidated. Here, TP53 is knocked out in hPSCs using CRISPR-Cas9 and compared them with isogenic wild-type hPSCs and human germ cell tumor lines as models of malignancy. While no major changes in proliferation, pluripotency, and transcriptomic profiles are found, mutant lines display aberrations in some of the main chromosomal hotspots for genetic abnormalities in hPSCs. Additionally, enhanced clonogenic and anchorage-free growth, alongside resistance to chemotherapeutic compounds is observed. The results indicate that common TP53-depleting mutations in hPSCs, although potentially overlooked by standard analyses, can impact their behavior and safety in a clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b157fa230ecf42b51895b57dbccc944884bf0d8" target='_blank'>
              Depletion of TP53 in Human Pluripotent Stem Cells Triggers Malignant-Like Behavior.
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, T. F. Eleveld, Marnix van Soest, Sanne Hillenius, Dennis M. Timmerman, A. Gillis, B. Roelen, C L Mummery, Leendert H J Looijenga, Daniela C. F. Salvatori
          </td>
          <td>2025-01-06</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="To enhance understanding of cellular heterogeneity and disease from single-cell sequencing data, we introduce ResidPCA, a robust method for cell state identification that models cell type heterogeneity. Simulations demonstrate ResidPCA’s efficacy, particularly in complex scenarios, with its accuracy more than four times higher than conventional Principal Component Analysis (PCA) and over three times higher than Non-negative Matrix Factorization (NMF)-based methods in identifying states expressed across multiple cell types. In scRNA-seq data from light-stimulated mouse visual cortex cells, ResidPCA captures stimulus-driven variability with an accuracy more than five times higher than NMF methods. In single nucleus data from an Alzheimer’s disease cohort, ResidPCA identified 44 snATAC-based and 42 snRNA-based states. 30 snATAC states were significantly enriched for Alzheimer’s disease heritability and were often more significantly enriched than established cell types such as microglia. The ResidPCA-based snATAC state most significantly enriched for Alzheimer’s disease heritability further elucidates a recently identified mechanism involving the neuron-ODC-microglial axis. This state links early amyloid production in neurons and oligodendrocytes with later-stage microglial activation and immune response, driving Alzheimer’s disease progression. These results demonstrate ResidPCA’s ability to reveal additional biological variation in single-cell data and uncover disease-relevant cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca44f2bab9201756fbfd5e90a83f848209acc96f" target='_blank'>
              Discovery of disease-associated cellular states using ResidPCA in single-cell RNA and ATAC sequencing data
              </a>
            </td>
          <td>
            Shaye Carver, Kodi Taraszka, S. Groha, Alexander Gusev
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eeb1135b7c75ad3978d67f0a6864e002ffcff1b" target='_blank'>
              Comprehensive single-cell aging atlas of healthy mammary tissues reveals shared epigenomic and transcriptomic signatures of aging and cancer.
              </a>
            </td>
          <td>
            B. Angarola, Siddhartha Sharma, Neerja Katiyar, Hyeon Gu Kang, Djamel Nehar-Belaid, SungHee Park, Rachel Gott, Giray N Eryilmaz, Mark A. LaBarge, Karolina Palucka, Jeffrey H. Chuang, Ron Korstanje, D. Ucar, Olga Anczukόw
          </td>
          <td>2024-11-25</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Introduction Metastatic cancer affects millions of people worldwide annually and is the leading cause of cancer-related deaths. Most patients with metastatic disease are not eligible for surgical resection, and current therapeutic regimens have varying success rates, some with 5-year survival rates below 5%. Here, we test the hypothesis that metastatic cancer can be genetically targeted by exploiting single base substitution mutations unique to individual cells that occur as part of normal aging prior to transformation. These mutations are targetable because ~10% of them form novel tumor-specific “NGG” protospacer adjacent motif (PAM) sites targetable by CRISPR-Cas9. Methods Whole genome sequencing was performed on five rapid autopsy cases of patient-matched primary tumor, normal and metastatic tissue from pancreatic ductal adenocarcinoma decedents. CRISPR-Cas9 PAM targets were determined by bioinformatic tumor-normal subtraction for each patient and verified in metastatic samples by high-depth capture-based sequencing. Results We found that 90% of PAM targets were maintained between primary carcinomas and metastases overall. We identified rules that predict PAM loss or retention, where PAMs located in heterozygous regions in the primary tumor can be lost in metastases (private LOH), but PAMs occurring in regions of loss of heterozygosity (LOH) in the primary tumor were universally conserved in metastases. Conclusions Regions of truncal LOH are strongly retained in the presence of genetic instability and, therefore, represent genetic vulnerabilities in pancreatic adenocarcinomas. A CRISPR-based gene therapy approach targeting these regions may be a novel way to genetically target metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada3070ea15ff6901b235b693417e61df449621f" target='_blank'>
              Islands of genomic stability in the face of genetically unstable metastatic cancer
              </a>
            </td>
          <td>
            Kirsten Bowland, Jiaying Lai, Alyza M. Skaist, Yan Zhang, S. Teh, Nicholas J. Roberts, Elizabeth Thompson, Sarah Wheelan, Ralph H. Hruban, Rachel Karchin, Matthew H. Bailey, Christine A. Iacobuzio-Donahue, J. Eshleman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the past century, colorectal cancer (CRC) has become one of the most devastating cancers impacting the human population. To gain a deeper understanding of the molecular mechanisms driving this solid tumor, researchers have increasingly turned their attention to the tumor microenvironment (TME). Spatial transcriptomics and proteomics have emerged as a particularly powerful technology for deciphering the complexity of CRC tumors, given that the TME and its spatial organization are critical determinants of disease progression and treatment response. Spatial transcriptomics enables high-resolution mapping of the whole transcriptome. While spatial proteomics maps protein expression and function across tissue sections. Together, they provide a detailed view of the molecular landscape and cellular interactions within the TME. In this review, we delve into recent advances in spatial biology technologies applied to CRC research, highlighting both the methodologies and the challenges associated with their use, such as the substantial tissue heterogeneity characteristic of CRC. We also discuss the limitations of current approaches and the need for novel computational tools to manage and interpret these complex datasets. To conclude, we emphasize the importance of further developing and integrating spatial transcriptomics into CRC precision medicine strategies to enhance therapeutic targeting and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b317770be242dc222a3004766249a5b9de43dcc" target='_blank'>
              Current state and future prospects of spatial biology in colorectal cancer
              </a>
            </td>
          <td>
            Francisco G. Carranza, Fernando C. Diaz, Maria Ninova, Enrique Velazquez-Villarreal
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Clear cell renal cell carcinoma (ccRCC) is the most common histologic type of RCC. However, the spatial and functional heterogeneity of immunosuppressive cells and the mechanisms by which their interactions promote immunosuppression in the ccRCC have not been thoroughly investigated.


METHODS
To further investigate the cellular and regional heterogeneity of ccRCC, we analyzed single-cell and spatial transcriptome RNA sequencing data from four patients, which were obtained from samples from multiple regions, including the tumor core, tumor-normal interface, and distal normal tissue. On the basis, the findings were investigated in vitro using tissue and blood samples from 15 patients with ccRCC and validated in the broader samples on tissue microarrays.


RESULTS
In this study, we revealed previously unreported subsets of both stromal and immune cells, as well as mapped their spatial location at finer resolution. In addition, we validated the clusters of tumor cells after removing batch effects according to six characterized gene sets, including epithelial-mesenchymal transitionhigh clusters, metastatic clusters and proximal tubulehigh clusters. Importantly, we identified a special regulatory T (Treg) cell subpopulation that has the molecular characteristics of terminal effector Treg cells but expresses multiple cytokines, such as interleukin (IL)-1β and IL-18. This group of Treg cells has stronger immunosuppressive function and was associated with a worse prognosis in ccRCC cohorts. They were colocalized with MRC1 + FOLR2 + tumor-associated macrophages (TAMs) at the tumor-normal interface to form a positive feedback loop, maintaining a synergistic procarcinogenic effect. In addition, we traced the origin of IL-1β+ Treg cells and revealed that IL-18 can induce the expression of IL-1β in Treg cells via the ERK/NF-κB pathway.


CONCLUSIONS
We demonstrated a novel cancer-promoting Treg cell subset and its interactions with MRC1 + FOLR2 +TAMs, which provides new insight into Treg cell heterogeneity and potential therapeutic targets for ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2214f0f0d02f003fb8cfcf936f4f2873d564786b" target='_blank'>
              Spatial and single-cell transcriptomics reveal cellular heterogeneity and a novel cancer-promoting Treg cell subset in human clear-cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Xiyu Song, Yumeng Zhu, Wenwen Geng, J. Jiao, Hongjiao Liu, Ruo Chen, Qian He, Lijuan Wang, Xiuxuan Sun, Weijun Qin, Jiejie Geng, Zhinan Chen
          </td>
          <td>2025-01-01</td>
          <td>Journal for immunotherapy of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The role of cancer-associated pericytes (CAPs) in tumor microenvironment (TME) suggests that they are potential targets for cancer treatment. The mechanism of CAP heterogeneity in esophageal squamous cell carcinoma (ESCC) remains unclear, which has limited the development of treatments for tumors through CAPs. Therefore, a comprehensive understanding of the classification, function, cellular communication and spatial distribution of CAP subpopulations in ESCC is urgently needed.This study used large-sample single-cell transcriptome sequencing (scRNA-seq) data to investigate pericytes’ subpopulation characteristics, functions, upstream and downstream regulation and interactions with other components of the TME in the ESCC, and analyzed prognostically in conjunction with Bulk RNA-seq data. In addition, pericyte subpopulations were validated and their spatial distribution in the ESCC TME was observed by multiplex immunofluorescence. Drug prediction and molecular docking was further used to validate the medicinal value of drug targets.CAPs in the ESCC TME were found to be highly heterogeneous, and we identified six pericyte subtypes: c1_ARHGDIB, c2_BCAM, c3_LUM, c4_SOD2, c5_TYMS, and c6_KRT17, which have commonality in a part of their functions, and each of them has a major function to play, by having different strengths of interaction with different components in the TME. In addition, we found that c4_SOD2 was negatively correlated with prognosis, conversely, c5_TYMS was positively correlated with prognosis. The drug with a better effect on c5_TYMS was docetaxel (binding energy = -8.1, -8.7 kcal/mol); raloxifene may be more effective against c4_SOD2, although raloxifene has a slightly lower binding energy to SOD2 (-6.4 kcal/mol), it has a higher binding energy to PDGFRβ (-8.1 kcal/mol).The present study identified and discovered pericyte subpopulations that were significantly associated with prognosis, which provides new biomarkers for predicting patient prognosis and adds usable targets for immunotherapy, and it is also important for gaining insights into the composition of the TME in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e295485d04626d24ad4217e848a133499f9df967" target='_blank'>
              Single-cell sequencing analysis reveals cancer-associated pericyte subgroup in esophageal squamous cell carcinoma to predict prognosis
              </a>
            </td>
          <td>
            Kai Xiong, Bing Pan, Hao Fang, Ziyou Tao
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc9bd11ddbadc141e7404167a9d1237fb1c3a74" target='_blank'>
              Encompassing view of spatial and single-cell RNA sequencing renews the role of the microvasculature in human atherosclerosis.
              </a>
            </td>
          <td>
            Tore Bleckwehl, Anne Babler, Merel Tebens, Sidrah Maryam, Michael Nyberg, Markus Bosteen, Maurice Halder, Isaac Shaw, Susanne Fleig, Charles Pyke, Henning Hvid, Louise M. Voetmann, J. V. van Buul, J. Sluimer, Vivek Das, Simon J. Baumgart, R. Kramann, Sikander Hayat
          </td>
          <td>2024-12-23</td>
          <td>Nature cardiovascular research</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Defining consequential differences in intestinal epithelial stem cells in healthy humans versus those with inflammatory bowel disease (Crohn’s disease and ulcerative colitis) is essential for the development of much needed therapies to restore the epithelial barrier and maintain its fidelity. Employing single cell transcriptomic/epigenomic approaches and colonoid models from children and adults with Crohn’s disease led us to identify an inflammatory secretory progenitor (ISP) cell state present almost exclusively in patients with Crohn’s disease compared to control subjects. ISPs exhibit gene expression profiles consistent with normal secretory progenitor cells but concomitantly express a suite of distinguishing pro-inflammatory genes. Mechanistically, ISPs exhibit open chromatin and gene expression of ISP signature genes. While these ISP-specific genes are not expressed in intestinal stem cells, their chromatin is accessible in Crohn’s disease stem cells suggesting that ISP genes are epigenetically poised in stem cells and are transcriptionally activated in ISP cells in the presence of inflammatory stimuli. Consistently, Crohn’s disease colonoids exhibit sustained ISP gene expression that can be elicited further with pro-inflammatory cytokines or via co-culture with pro-inflammatory macrophages. In summary, we define differences in the epithelial stem and progenitor compartment of patients with Crohn’s disease suggesting aberrant stem cell differentiation and inflammatory gene expression arises during disease. HIGHLIGHTS We identify an inflammatory secretory progenitor cell state in endoscopically non-inflamed tissue from pediatric and adult patients with Crohn’s disease. Crohn’s disease epithelial stem and progenitor cells display increased chromatin accessibility preceding the emergence of inflammatory secretory progenitor cell states. Crohn’s disease colonoids from non-inflamed regions contain inflammatory secretory progenitor cells in the absence of inflammatory stimuli Inflammatory secretory progenitor cells expand in colonoid culture in response to cytokine stimulation or co-culture with pro-inflammatory macrophages">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1c97ff6cf153588b934b86c33ad5324b1f6d33" target='_blank'>
              An epigenetic basis for sustained inflammatory epithelial progenitor cell states in Crohn’s disease
              </a>
            </td>
          <td>
            Tatiana A. Karakasheva, Clara Morral Martinez, Yusen Zhou, Jingya Qui, Xinyi E. Chen, Gloria E. Soto, Shaneice K. Nettleford, Olivia T. Hix, Daana M. Roach, Alyssa M. Laguerta, Anusha Thadi, Rachael M. Edwards, Daniel Aleynick, Sarah Weinbrom, Elizaveta Borodyanskaya, Oliver Pickering, Marykate Fulton, Chia- Chen, Bella V. Peterson, Erik B. Hagen, Ian P. Yannuzzi, Zainab Haider, Zvi Cramer, Maire C Conrad, Ning Li, Meenakshi Bewtra, Yasin Uzun, Kai Tan, J. Kelsen, Andy J. Minn, Christopher Joachim Lengner, Kathryn E. Hamilton
          </td>
          <td>2024-11-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5daf8fc78edec4dc67cc0ac87030a2ee471c5078" target='_blank'>
              Cancer-associated fibroblasts: heterogeneity, tumorigenicity and therapeutic targets
              </a>
            </td>
          <td>
            Keke Lv, Tianlin He
          </td>
          <td>2024-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Malignant tumors can evade immune surveillance and elimination through multiple mechanisms, with the induction of immune cell dysfunction serving as a crucial strategy. Mounting evidence indicates that T cell senescence constitutes the primary mechanism underlying T cell dysfunction in acute myeloid leukemia (AML) and represents one of the potential causes of immunotherapy failure. AML usually progresses rapidly and is highly susceptible to drug resistance, thereby resulting in recurrence and patient mortality. Hence, disrupting the immune interface within the bone marrow microenvironment of AML has emerged as a critical objective for synergistically enhancing tumor immunotherapy. In this review, we summarize the general characteristics, distinctive phenotypes, and regulatory signaling networks of senescent T cells and highlight their potential clinical significance in the bone marrow microenvironment of AML. Additionally, we discuss potential therapeutic strategies for alleviating and reversing T cell senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b9ab03510c38b97ca7c2aca4b3d03ba9b0374e2" target='_blank'>
              Senescent T Cells: The Silent Culprit in Acute Myeloid Leukemia Progression?
              </a>
            </td>
          <td>
            Xiaolan Zhang, Lingbo Liu
          </td>
          <td>2024-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play pivotal roles in solid tumor initiation, growth, and immune evasion. However, the optimal biomimetic modeling conditions remain elusive. In this study, we investigated the effects of 2D and 3D culturing conditions on human primary CAFs integrated into a modular tumor microenvironment (TME). Using single-nucleus RNA sequencing (snRNAseq) and Proteomics’ Proximity Extension Assays, we characterized CAF transcriptomic profiles and cytokine levels. Remarkably, when cultured in 2D, CAFs exhibited a myofibroblast (myCAF) subtype, whereas in 3D tumor spheroid cultures, CAFs displayed a more inflammatory (iCAF) pathological state. By integrating single-cell gene expression data with functional interrogations of critical TME-related processes [natural killer (NK)-mediated tumor killing, monocyte migration, and macrophage differentiation], we were able to reconcile form with function. In 3D TME spheroid models, CAFs enhance cancer cell growth and immunologically shield cells from NK cell-mediated cytotoxicity, in striking contrast with their 2D TME counterparts. Notably, 3D CAF-secreted proteins manifest a more immunosuppressive profile by enhancing monocyte transendothelial migration and differentiation into M2-like tumor-associated macrophages (TAMs). Our findings reveal a more immunosuppressive and clinically relevant desmoplastic TME model that can be employed in industrial drug discovery campaigns to expand the cellular target range of chemotherapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b61425d19bdd4e9908dd233e505b35ad2127a3" target='_blank'>
              Unveiling immune cell response disparities in human primary cancer-associated fibroblasts between two- and three-dimensional cultures
              </a>
            </td>
          <td>
            Jian-Ping Yang, Nikhil Nitin Kulkarni, Masashi Yamaji, Tsubasa Shiraishi, Thang Pham, Han Do, Nicole Aiello, Michael Shaw, Toshihiro Nakamura, Akiko Abiru, N. Gavva, Shane R. Horman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Colonies of pure bacterial strains are highly dense cell structures that are organized in distinct and typical arrangements. The size, shape and variability of bacterial colonies are strongly dependent on the species and also influenced by environmental conditions. However, the spatial organization of individual cells is unknown for most strains. How specific heterogeneous cell types at different locations in a colony contribute to the overall structure and how they may influence the overall function of a colony is largely unknown. The study aims to investigate the local diversification of bacterial colony structures by introducing a local biopsy technique. The biopsied cells were analyzed by microbial flow cytometry and cytometric fingerprinting, which diversified the biopsied samples into many heterogeneous cell states. This two-stage resolution insight into colony structure was performed on five bacterial strains: Bacillus subtilis, Paenibacillus polymyxa, Kocuria rhizophila, Stenotrophomonas rhizophila, and Pseudomonas citronellolis. The effects of biopsy tool size (27G needle and 10 µL, 200 µL, 1000 µL pipette tips) and sampling location on the precision of the technique were tested by using both gates setting along Gaussian distributions of subpopulations and a grid gating tool as well as the t-distributed Stochastic Neighbor Embedding (t-SNE) method. The biopsy technique uncovered significant heterogeneity among the cells within bacterial colonies, identifying differences in cell cycle stages, the proportion of living and dead cells, and the abundance of spore types. Cells from different biopsy sites displayed distinct physiological states, revealing that colony structure is far more complex than previously understood. The technique’s precision depends on the biopsy tool size, dye equilibration, and cell handling, underscoring the importance of method calibration. The biopsy method, combined with cytometric fingerprinting, provided insights within only 15 to 45 minutes and is universally applicable. The study provides a high-resolution biopsy technique that explores the spatial distribution of cell types and their heterogeneous physiological cell states, allowing conclusions to be drawn from biopsy composition at different locations to overarching functions of the entire bacterial colony. This method also facilitates downstream analysis through further cell sorting, offering a powerful approach for future functional investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78643c1717987566c91442a72f5875dcccc93d25" target='_blank'>
              Bacterial colony biopsies: spatial discrimination of heterogeneous cell types by cytometric fingerprinting
              </a>
            </td>
          <td>
            Gorkhmaz Abbaszade, Kathrin Stückrath, Susann Müller
          </td>
          <td>2024-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT High-risk non-muscle-invasive bladder cancer (NMIBC) presents high recurrence and progression rates. Despite the use of Bacillus Calmette-Guérin gold-standard immunotherapy and the recent irruption of anti-PD-1/PD-L1 drugs, we are missing a comprehensive understanding of the tumor microenvironment (TME) that may help us find biomarkers associated to treatment outcome. Here, we prospectively analyzed TME composition and PD-L1 expression of tumor and non-tumoral tissue biopsies from 73 NMIBC patients and used scRNA-seq, transcriptomic cohorts and tissue micro-array to validate the prognostic value of cell types of interest. Compared to non-tumoral tissue, NMIBC presented microvascular alterations, increased cancer-associated fibroblast (CAF) and myofibroblast (myoCAF) presence, and varied immune cell distribution, such as increased macrophage infiltration. Heterogeneous PD-L1 expression was observed across subsets, with macrophages showing the highest expression levels, but cancer cells as the primary potential anti-PD-L1 binding targets. Unbiased analysis revealed that myoCAF and M2-like macrophages are specifically enriched in high-grade NMIBC tumors. The topological distribution of these two cell types changed as NMIBC progresses, as shown by immunofluorescence. Only myoCAFs were associated with higher rates of progression and recurrence in three independent cohorts (888 total patients), reaching prediction values comparable to transcriptomic classes, which we further validated using tissue micro-array. Our study provides a roadmap to establish the landscape of the NMIBC TME, highlighting myoCAFs as potential prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe97543b0d1de2e394926985ac37960e50a2bea" target='_blank'>
              Characterisation of the tumour microenvironment and PD-L1 granularity reveals the prognostic value of cancer-associated myofibroblasts in non-invasive bladder cancer
              </a>
            </td>
          <td>
            Carmen G. Cañizo, F. Guerrero-Ramos, M. Perez Escavy, I. Lodewijk, C. Suárez-Cabrera, L. Morales, Sandra P Nunes, E. Munera-Maravilla, Carolina Rubio, Rebeca Sánchez, Marta Rodríguez-Izquierdo, Jaime Martínez de Villarreal, Francisco X. Real, D. Castellano, Cristina Martín-Arriscado, David Lora Pablos, A. Rodríguez Antolín, M. Dueñas, J. Paramio, Victor G. Martínez
          </td>
          <td>2024-12-19</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Aging is a multifaceted process that manifests differently in various cell types and within the same cell type. Therefore, understanding age-related changes with single-cell resolution may illuminate the mechanistic details of tissue aging and offer targets for intervention. We utilized high-throughput single-cell RNA-sequencing (scRNA-seq) to analyze senescence-associated changes within the CD4 T cell compartment. First, we sequenced total human CD4 T cells from multiple donors, revealing that naïve CD4 T cells exhibited a less senescent state compared to memory cells. We focused on naïve CD4 T cells to determine if individual cells differ in expression of aging-associated markers. By sorting cells based on senescence-associated beta-galactosidase (B-gal) activity and applying scRNA-seq, we identified transcriptional signatures associated with more youthful and senescent states. Surprisingly, we discovered cells with lower B-gal expression comprised two distinct populations: one marked by high expression of HELIOS and low expression of SESN3, and another marked by high expression of BACH2 and low expression of TC2N. In comparison, 2 youthful subpopulation and senescence subpopulations differ in markers of immunological activation We then investigated if the composition of naive CD4 compartment changes with age, finding that the abundance of youthful subpopulations was reduced in the older donors’ CD4 populations. Thus, aging in the naïve CD4 compartment is associated with a shifted distribution between more youthful and more senescent cells. Moving forward, we will attempt to manipulate naïve CD4 T cells into a more youthful state by altering expression of the identified transcriptional signatures associated with youthful and senescent states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bab6ece44b0e3064c9cd597254aa9a3e1045d71" target='_blank'>
              DISTINCT CELL SUBTYPES WITHIN NAÏVE CD4 T CELLS COMPARTMENT SHAPE ITS AGING AND SENESCENCE
              </a>
            </td>
          <td>
            C. J. Kelly, Ashley Jeon, Jacob Rodriguez, Dina Prosser, Morgan Diegel, Nikolay Burnaevskiy
          </td>
          <td>2024-12-01</td>
          <td>Innovation in Aging</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55057507874b2d1db614aaeb55e2719a4aaa942d" target='_blank'>
              Metabolic rewiring in skin epidermis drives tolerance to oncogenic mutations.
              </a>
            </td>
          <td>
            Anupama Hemalatha, Zong-Hu Li, David G. Gonzalez, C. Matte-Martone, Karen Tai, Elizabeth Lathrop, Daniel Gil, S. Ganesan, Lauren E Gonzalez, Melissa C. Skala, Rachel J. Perry, Valentina Greco
          </td>
          <td>2025-01-06</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Ambiguous understanding of tumors and the tumor microenvironments (TMEs) hinders accurate diagnosis and available treatment for multifocal hepatocellular carcinoma (HCC) covering intrahepatic metastasis (IM) and multicentric occurrence (MO). Here, we characterized the diverse TMEs of IM and MO identified by whole-exome sequencing (WES) at single-cell resolution.


APPROACH AND RESULTS
We performed parallel WES and single-cell RNA sequencing (scRNA-seq) on twenty-three samples from seven patients to profile their TMEs when major results were validated by immunohistochemistry in the additional cohort. Integrative analysis of WES and scRNA-seq found that malignant cells in IM showed higher intra-tumor heterogeneity, stemness and more activated metabolism than those in MO. Tumors from IM shared similar TMEs while distinct TMEs were noticed in those from MO. Furthermore, CD20+ B cells, plasma cells and conventional type II dendritic cells (cDC2s) were decreased in IM relative to MO while T cells in IM exhibited a more terminally exhausted capacity with a higher proportion of proliferative/exhausted T cells than that in MO. Both CD20 and CD1C correlated with better prognosis in multifocal HCC. Additionally, MMP9+ tumor-associated macrophages were enriched across IM and MO which formed cellular niches with regulatory T cells (Tregs) and proliferative/exhausted T cells.


CONCLUSIONS
Our findings deeply decipher the heterogeneous TMEs between IM and MO, which provide a comprehensive landscape of multifocal HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2c7a45eee2228edb84e8693a219534a9ab548a" target='_blank'>
              Genomic and the tumor microenvironment heterogeneity in multifocal hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yongheng Yang, Qingqiang Ni, Hongguang Li, Jiuzheng Sun, Xia Zhou, Lingxin Qu, Liyuan Wang, Chuanzong Zhao, Xiaolu Zhang
          </td>
          <td>2024-12-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Hepatic arterial infusion chemotherapy (HAIC) has emerged as a promising treatment strategy for hepatocellular carcinoma (HCC), but a detailed understanding of the multicellular ecosystem after HAIC treatment is lacking. Here, we collected tumor samples from treatment-naïve primary and post-HAIC HCC, and integrated single-nucleus RNA sequencing with spatial transcriptomics to characterize the tumor ecosystem in the post-HAIC HCC. Increased fractions and enhanced cellular communication of CD4+ T, CD20+ B, and dendritic cell subtypes were identified in post-HAIC tumors. Moreover, it is substantiated that HAIC promoted tertiary lymphoid structures (TLS) formation, and addressed the roles of TLSs as spatial niches of cellular communication. Specifically, intermediate exhausted CD8+ T cells expressing Granzyme-K and PD-1 (PD-1+CD8+ Tex-int) expanded following HAIC and exhibited a functionally antitumor phenotype. PD-1+CD8+ Tex-int accumulated in the TLS vicinity and disseminated throughout the tumor microenvironment, demonstrating potential as an effective biomarker for HAIC-based treatment in HCC. This study provides valuable resources and biological insights in the cellular underpinnings of HAIC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62624d4b04f46d42e65710fe0c98f6d6afaa46d" target='_blank'>
              Single-Nucleus and Spatial Transcriptome Profiling Delineates the Multicellular Ecosystem in Hepatocellular Carcinoma After Hepatic Arterial Infusion Chemotherapy.
              </a>
            </td>
          <td>
            YeXing Huang, Zefeng Du, Zhicheng Lai, DongSheng Wen, Lichang Huang, Minke He, ZiChao Wu, Huifang Li, Hanyue Ouyang, Wenchao Wu, Anna Kan, Ming Shi
          </td>
          <td>2024-12-16</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc5afe786b47a61ca4c6c86b698a4b6f0aa1df3" target='_blank'>
              Characterizing tertiary lymphoid structures associated single-cell atlas in breast cancer patients
              </a>
            </td>
          <td>
            Xiaokai Fan, Daqin Feng, Donggui Wei, Anqi Li, Fangyi Wei, Shufang Deng, Muling Shen, Congzhi Qin, Yongjia Yu, Lun Liang
          </td>
          <td>2025-01-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47537700619a16e63c5ebabefc327d2e5284da8b" target='_blank'>
              Transcriptome analysis of novel B16 melanoma metastatic variants generated by serial intracarotid artery injection
              </a>
            </td>
          <td>
            Jenny C. Kienzler, Erick Contreras, J. Treger, Linda M. Liau, Geoffrey Owens, Robert M. Prins
          </td>
          <td>2025-01-16</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8124e4116b706bf406200aaca77b811cd619c61b" target='_blank'>
              Metabolically activated and highly polyfunctional intratumoral VISTA+ regulatory B cells are associated with tumor recurrence in early-stage NSCLC
              </a>
            </td>
          <td>
            D. Lo Tartaro, Beatrice Aramini, V. Masciale, Nikolaos Paschalidis, F. Lofaro, Anita Neroni, R. Borella, Elena Santacroce, Alin Liviu Ciobanu, A. V. Samarelli, Federica Boraldi, Daniela Quaglino, Alessandra Dubini, Michele Gaudio, G. Manzotti, F. Reggiani, Federica Torricelli, A. Ciarrocchi, A. Neri, Federica Bertolini, Massimo Dominici, Pier Luigi Filosso, F. Stella, L. Gibellini, S. De Biasi, Andrea Cossarizza
          </td>
          <td>2025-01-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is the third leading cause of cancer-related deaths with a 5-year survival rate of 13%. Surgical resection remains the only curative option as systemic therapies offer limited benefit. Poor response to chemotherapy and immunotherapy is due, in part, to the dense stroma and heterogeneous tumor microenvironment (TME). Opportunities to target the PDAC stroma may increase the effectiveness of existing or novel therapies. Current strategies targeting the stromal compartment within the PDAC TME primarily focus on degrading extracellular matrix or inhibiting stromal cell activity, angiogenesis, or hypoxic responses. In addition, extensive work has attempted to use immune targeting strategies to improve clinical outcomes. Preclinically, these strategies show promise, especially with the ability to alter the tumor ecosystem; however, when translated to the clinic, most of these trials have failed to improve overall patient outcomes. In this review, we catalog the heterogenous elements of the TME and discuss the potential of combination therapies that target the heterogeneity observed in the TME between patients and how molecular stratification could improve responses to targeted and combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254631c9190ee22fb9be4694a816eb9553b6afce" target='_blank'>
              Challenges and Opportunities in Targeting the Complex Pancreatic Tumor Microenvironment
              </a>
            </td>
          <td>
            Jennifer M. Finan, Yifei Guo, Shaun M Goodyear, Jonathan R. Brody
          </td>
          <td>2024-12-01</td>
          <td>JCO Oncology Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor microenvironment (TME) plays a crucial role in the oncogenesis, early distant metastasis, and limited immunotherapeutic responses of pancreatic ductal adenocarcinoma (PDAC). Myeloid cell populations, particularly tumor-associated macrophages (TAMs), located within this immunosuppressive niche, are pivotal in these processes. With aid of single cell RNA sequencing (scRNA-seq), multiple studies have successfully validated distinct macrophage subtypes pathological functions in different cancer types, yet the correlation of TAMs in hepatic metastatic and primary PDAC lesions remains insufficiently understood. In this study, we reanalyzed samples from both primary and hepatic metastatic PDACs to elucidate the functional conservation and variability of TAMs. Pseudotime trajectory inference, together with pseudobulk analysis based on subtype level, manifests TAMs undergo similar metabolism reprogramming and pro-inflammatory style transitions with specific signaling pathway activations. Compared to primary tumors, hypoxic conditions are alleviated, which largely depends on the physiological context and the preliminary ligand-receptor interactive network within the metastatic niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a56bbf6854596a35b27cf40482bd3f5c23fafab" target='_blank'>
              In-depth analysis of pathological function of tumor-associated macrophage populations in hepatic metastatic and primary PDAC
              </a>
            </td>
          <td>
            Fanyu Meng
          </td>
          <td>2024-12-12</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L. Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Macrophages are a highly plastic cell type that adopt distinct subtypes and functional states depending on environmental cues. These functional states can vary widely, with distinct macrophages capable of displaying opposing functions. We sought to understand how macrophage subtypes that exist on two ends of a spectrum influence the function of other cells. We used a co-culture system with primary human macrophages to probe the effects of macrophage subtypes on breast cancer cell proliferation. Our studies revealed a surprising phenotype in which both macrophage subtypes inhibited cancer cell proliferation compared to cancer cells alone. Of particular interest, using two different proliferation assays with two different breast cancer cell lines, we showed that differentiating macrophages into a "pro-tumor" subtype inhibited breast cancer cell proliferation. These findings are inconsistent with the prevailing interpretation that "pro-tumor" macrophages promote cancer cell proliferation and suggest a re-evaluation of how these interpretations are made. [Media: see text].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56750b0cd5723af97ed1329a64044bb68358f769" target='_blank'>
              Macrophage subtypes inhibit breast cancer proliferation in culture.
              </a>
            </td>
          <td>
            Sophia R S Varady, Daniel Greiner, Minna Roh-Johnson
          </td>
          <td>2024-11-27</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In this study, we delve into the intrinsic mechanisms of cell communication in hepatocellular carcinoma (HCC). Initially, employing single-cell sequencing, we analyze multiple malignant cell subpopulations and cancer-associated fibroblast (CAF) subpopulations, revealing their interplay through receptor-ligand interactions, with a particular focus on SPP1. Subsequently, employing unsupervised clustering analysis, we delineate two clusters, C1 and C2, and compare their infiltration characteristics using various tools and metrics, uncovering heightened cytotoxicity and overall invasion abundance in C1. Furthermore, our gene risk scoring model indicates heightened activity of the immune therapeutic pathway in C1. Lastly, employing a formulated scoring system, we stratify patients into high and low-risk groups, revealing notably poorer outcomes in the high-risk cohort on Kaplan-Meier curves. Risk scores exhibit a negative correlation with model genes and immune cell infiltration scores, indicating poor prognosis in the high-risk group. Further characterization elucidates the regulatory landscape of the high and low-risk groups across various signaling pathways. In addition, we used wet lab experiments to prove that ABCA1 plays a pro-oncogenic role in hepatocellular carcinoma cells by promoting proliferation, invasion, migration, and reducing apoptosis. In summary, these findings provide crucial insights, offering valuable clues and references for understanding HCC pathogenesis and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ebd5bca77500ddcf823c450cf7ed50074d3c6d" target='_blank'>
              ABCA1 promote tumor environment heterogeneity via epithelial mesenchymal transition in Huh7 and HepG2 liver cancer cell
              </a>
            </td>
          <td>
            Dinglai Yu, Fang Guo, Qiyu Zhang, Huajun Yu, Wenmin Wang, Yunzhi Chen
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d2a730533f465c1760e5f39354ae03ee2bf1e3c" target='_blank'>
              Spatial expression of fibroblast activation protein-α in clear cell renal cell carcinomas revealed by multiplex immunoprofiling analysis of the tumor microenvironment
              </a>
            </td>
          <td>
            G. Larrinaga, M. Redrado, Ana Loizaga-Iriarte, Amparo Pérez-Fernández, Aida Santos-Martín, Javier C. Angulo, José A Fernández, Alfonso Calvo, José I. López
          </td>
          <td>2025-01-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f22cb74c78b53ff77c895d167785c4abf72ee807" target='_blank'>
              Colorectal cancer patients-derived immunity-organoid platform unveils cancer-specific tissue markers associated with immunotherapy resistance
              </a>
            </td>
          <td>
            A. Esposito, A. Agostini, G. Quero, G. Piro, L. Priori, A. Caggiano, G. Scaglione, A. Battaglia, M. Calegari, L. Salvatore, M. Bensi, M. Maratta, A. Ceccarelli, G. Trovato, G. Genovese, E. Gurreri, S. Ascrizzi, M. Martini, C. Fiorillo, A. Fattorossi, F. De Sanctis, S. Ugel, V. Corbo, S. Alfieri, G. Tortora, C. Carbone
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde0e9abf6b50983ec6394f87b77c188d773d609" target='_blank'>
              YAP-driven malignant reprogramming of oral epithelial stem cells at single cell resolution
              </a>
            </td>
          <td>
            F. Faraji, Sydney I. Ramirez, Lauren M. Clubb, Kuniaki Sato, Valeria Burghi, Thomas S. Hoang, Adam Officer, Paola Y. Anguiano Quiroz, William M. G. Galloway, Zbigniew Mikulski, Kate Medetgul-Ernar, P. Marangoni, Kyle B Jones, Yuwei Cao, A. Molinolo, Kenneth Kim, Kanako Sakaguchi, Joseph A. Califano, Quinton Smith, Alon Goren, Ophir D. Klein, Pablo Tamayo, J. S. Gutkind
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="ABSTRACT Background Metastasis is the major cause of cancer‐related mortality. The premetastatic niche is a promising target for its prevention. However, the generality and cellular dynamics in premetastatic niche formation have remained unclear. Aims This study aimed to elucidate the generality and cellular dynamics in premetastatic niche formation. Materials and Methods We performed comprehensive flow cytometric analysis of lung and peripheral immune cells at three time points (early premetastatic, late premetastatic, and micrometastatic phases) for mice with subcutaneous implants of three types of cancer cells (breast cancer, lung cancer, or melanoma cells). The immuno‐cell profiles were then used to predict the metastatic phase by machine learning. Results We found a common pattern of changes in both lung and peripheral immune cell profiles across the three cancer types, including a decrease in the proportion of eosinophils in the early premetastatic phase, an increase in that of regulatory T cells in the late premetastatic phase, and an increase in that of polymorphonuclear myeloid‐derived suppressor cells and a decrease in that of B cells in the micrometastatic phase. Machine learning using immune cell profiles could predict the metastatic phase with approximately 75% accuracy. Discussion Validation of our findings in humans will require data on the presence or absence of micrometastases in patients and the accumulation of comprehensive and temporal information on immune cells. In addition, blood proteins, extracellular vesicles, DNA, RNA, or metabolites may be useful for more accurate prediction. Conclusion The discovery of generalities in premetastatic niche formation allow prediction of metastatic phase and provide a basis for the development of methods for early detection and prevention of cancer metastasis in a cancer type‐independent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f02612a5b53055050e50dcce6e173f615e1f3275" target='_blank'>
              Immune Cell Profiling Reveals a Common Pattern in Premetastatic Niche Formation Across Various Cancer Types
              </a>
            </td>
          <td>
            Shigeaki Sugiyama, Kanae Yumimoto, Keiichi I Nakayama
          </td>
          <td>2024-12-30</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life. Methods Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity. Results A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis. Conclusion By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b32d455915f14c9cee5327c8bd03503f5d7e8b33" target='_blank'>
              SNAI1 promotes epithelial-mesenchymal transition and maintains cancer stem cell-like properties in thymic epithelial tumors through the PIK3R2/p-EphA2 Axis
              </a>
            </td>
          <td>
            H. E, Lei Zhang, Zhenhua Yang, Long Xu, Tao Wang, Junhong Guo, Lang Xia, Juemin Yu, Heyong Wang, Y. She, Junqi Wu, Yue Zhao, Chang Chen, Deping Zhao
          </td>
          <td>2024-12-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) represent the main immune cell population in various brain malignancies. To elucidate their biological impact in the tumor microenvironment (TME) of meningiomas (MGMs), we assessed TAM numbers, activation state, malignancy- and survival-associated changes, as well as their association with tumor-infiltrating T lymphocytes (TILs). Methods TAM infiltration was analyzed in a multicenter cohort of 195 clinically well-annotated cases (follow-up >5 years, n=120 newly-diagnosed and n=75 recurrent MGMs) enriched for higher-grade MGMs. TAMs and M2-TAMs were quantified by tissue cytometry on whole-tumor sections. Further, we assessed levels of 27 cyto- and chemokines in a subset of tissues (n=46 cases), and re-analyzed our previously published T cell infiltration (n=94 cases) and expanded microarray (n=97 cases) datasets. Results Newly-diagnosed MGMs showed a substantial but highly heterogeneous TAM infiltration that was four times higher than for TILs. Anti-inflammatory M2-TAMs were increased in higher WHO grade tumors and in recurrent MGMs. Importantly, high M2-TAM infiltration was associated with poor progression-free survival independent of other prognostic confounders and even mitigated the beneficial prognostic effect of TIL infiltration. Additional cytokine, gene expression and pathway analyses corroborated the presence of an immunosuppressive niche in M2-TAM-enriched MGMs. Conclusions Altogether, higher numbers of TAMs and M2-TAMs appear to be a hallmark of clinically aggressive behavior in newly-diagnosed and recurrent MGMs. Unlike TILs, immunosuppressive TAMs seem to play a dominant negative role in the immunological landscape of MGMs, highlighting M2-TAMs to be an attractive treatment target for immunotherapeutic approaches. Translational Relevance of the Study Meningiomas (MGMs) are typically regarded as benign neoplasms, however there is a substantial proportion of clinically aggressive tumors that are refractory to standard treatment modalities and demand for the development of novel therapeutic approaches such as immunotherapy. This is the first comprehensive study reporting malignancy- and progression-associated changes of tumor-associated macrophages (TAMs), their polarization state, their association with tumor-infiltrating T lymphocytes (TILs), and their impact on patient survival in a large multicenter cohort of 195 tumors containing substantial numbers of clinically aggressive cases. Notably, we identified higher numbers of immunosuppressive M2-TAMs as an independent prognostic factor for poor survival, overriding the beneficial prognostic effects of TILs. Thus, our data highlight an important role of immunosuppressive M2-TAMs on tumor malignancy and progression, and further suggest targeting macrophages as a treatment strategy to improve the success of immunotherapeutic approaches in MGMs. Key points Meningiomas are highly infiltrated by immunosuppressive M2-TAMs. High M2-TAM numbers are an independent negative prognostic factor for patient survival. High TAM infiltration mitigates the beneficial prognostic impact of TILs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797056ddd8b0fcec8ccf003c922ab3488ed62e98" target='_blank'>
              Tumor-Associated Macrophages in Meningiomas: An Independent Prognostic Factor for Poor Survival Outperforming the Benefits of T cells
              </a>
            </td>
          <td>
            C. Lotsch, Fang Liu, R. Warta, G. Jungwirth, Carmen Rommel, Mandy Barthel, K. Lamszus, A. Kessler, Niels Grabe, Mario Loehr, R. Ketter, Christian Senft, Manfred Westphal, F. Sahm, Sandro Krieg, A. Unterberg, Matthias Simon, Andreas von Deimling, C. Herold-Mende
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Abstract Background Vein grafts are commonly employed in revascularisation surgery for multivessel coronary artery disease, yet neointimal hyperplasia (NIH) remains a critical impediment to the long‐term patency of these grafts. Despite this, effective methods to precisely identify and target interventions for the neointima are still inadequate. Methods In this study, Sprague–Dawley (SD) rats were used to establish an external jugular vein transplantation model, and the NIH pathophysiological process was tracked across 11 time points (0–35 days) using various histological stains. Spatial transcriptomics was performed on normal veins and 19‐day grafts to explore gene expression in neointimal regions. Immunohistochemical analysis identified neointima‐specific markers, while NIH progression was assessed in SD rats with four and a half LIM domains protein 1 (Fhl1) knockout and in human saphenous veins (HSV) with adenovirus‐mediated Fhl1 overexpression. Results Typical neointimal formation commenced by day 11 postgrafting and peaked at day 19. Neointimal cells originated from newly generated α‐SMA(+) repair cells located outside the grafted vein, displaying a hybrid fibroblast‐smooth muscle cell phenotype. Spatial transcriptomics identified stable and sustained Fhl1 expression within the neointima throughout the entire NIH phase. Systemic knockout of Fhl1 in SD rats via the phosphoinositide 3‐kinase pathway exacerbated graft inflammation, heightened cell proliferation, and accelerated NIH. Conversely, FHL1 overexpression in cultured HSV suppressed NIH. Conclusion These findings indicate that, following grafting into the arterial system, the newly formed repair cells external to the grafted vein play a pivotal role in NIH, with neointimal cells exhibiting stable and continuous Fhl1 expression. Fhl1 serves as a protective factor against NIH both in vivo and in HSV, likely due to its anti‐inflammatory and anti‐proliferative effects. Key points This study firstly used spatial transcriptomics technique to analyse the neointima and generated a specific neointimal transcriptomic atlas. Fhl1 exhibits specific and stable expression in the spatial region of the neointima. It has thus far the highest enrichment of expression in the neointima in NIH phases, suggesting that it is a prominent molecular biomarker of neointima. We generated rats with a Fhl1 deletion and found that insufficient Fhl1 expression caused an increase in the severity of vascular inflammation and proliferation during neointimal hyperplasia. Adenovirus‐mediated FHL1 overexpression in human saphenous vein have beneficial effects in preventing neointimal hyperplasia. These highlight its potential as a therapeutic target for mitigating vein graft failure associated with cardiovascular procedures. Spatial transcriptomics profiles and morphological observations demonstrated that a newly generated cell population outside the grafted vein with hybrid phenotype between SMCs and fibroblasts contributes to neointimal formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a9f540b21618eabf1f7a31ec78a413b6ce43cc9" target='_blank'>
              Fhl1, a new spatially specific protein, regulates vein graft neointimal hyperplasia
              </a>
            </td>
          <td>
            Chaoqun Wang, Jiantao Chen, Zicong Feng, B. Jian, Suiqing Huang, Kangni Feng, Haoliang Liu, Zhuoming Zhou, Ziyin Ye, Jing Lu, M. Liang, Zhongkai Wu
          </td>
          <td>2024-12-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Epithelial cancers are typically heterogeneous with primary prostate cancer being a typical example of histological and genomic variation. Prior studies of primary prostate cancer tumour genetics revealed extensive inter and intra-patient genomic tumour heterogeneity. Recent advances in machine learning have enabled the inference of ground-truth genomic single-nucleotide and copy number variant status from transcript data. While these inferred SNV and CNV states can be used to resolve clonal phylogenies, however, it is still unknown how faithfully transcript-based tumour phylogenies reconstruct ground truth DNA-based tumour phylogenies. We sought to study the accuracy of inferred-transcript to recapitulate DNA-based tumour phylogenies. We first performed in-silico comparisons of inferred and directly resolved SNV and CNV status, from single cancer cells, from three different cell lines. We found that inferred SNV phylogenies accurately recapitulate DNA phylogenies (entanglement = 0.097). We observed similar results in iCNV and CNV based phylogenies (entanglement = 0.11). Analysis of published prostate cancer DNA phylogenies and inferred CNV, SNV and transcript based phylogenies demonstrated phylogenetic concordance. Finally, a comparison of pseudo-bulked spatial transcriptomic data to adjacent sections with WGS data also demonstrated recapitulation of ground truth (entanglement = 0.35). These results suggest that transcript-based inferred phylogenies recapitulate conventional genomic phylogenies. Further work will need to be done to increase accuracy, genomic, and spatial resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d56f0a1a843a737c66bbddbc1e662cc71b163e" target='_blank'>
              Clonal phylogenies inferred from bulk, single cell, and spatial transcriptomic analysis of epithelial cancers
              </a>
            </td>
          <td>
            Andrew Erickson, Sandy Figiel, Timothy Rajakumar, Srinivasa Rao, Wencheng Yin, D. Doultsinos, Anette Magnussen, Reema Singh, Ninu Poulose, Richard J. Bryant, Olivier Cussenot, Freddie C Hamdy, Dan Woodcock, I. Mills, Alastair D. Lamb
          </td>
          <td>2025-01-03</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55719070e9f1cff38fbdc568fedbec8bc7d69e98" target='_blank'>
              Single-cell transcriptomic and spatial analysis reveal the immunosuppressive microenvironment in relapsed/refractory angioimmunoblastic T-cell lymphoma
              </a>
            </td>
          <td>
            Mengyan Zhu, Ning Li, Lei Fan, Rongrong Wu, Lei Cao, Yimin Ren, Chuanyang Lu, Lishen Zhang, Yun Cai, Yuzhu Shi, Zihan Lin, Xueying Lu, Jiayan Leng, Shiyang Zhong, Xingfei Hu, Bin Huang, Runheng Huang, Wanting Zhou, Diru Yao, Lingxiang Wu, Wei Wu, Quanzhong Liu, Peng Xia, Ruize Chen, W. Shi, Ruohan Zhang, Sali Lv, Chunling Wang, Liang Yu, Jianyong Li, Qianghu Wang, Kening Li, H. Jin
          </td>
          <td>2024-12-01</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/456d5e0f44d3a56861aef434239933c23d1853a0" target='_blank'>
              Single-cell RNA sequencing analysis reveals the dynamic changes in the tumor microenvironment during NMIBC recurrence.
              </a>
            </td>
          <td>
            Ziang Chen, Tianxiang Zhang, Weijian Li, Jiawei Hu, Y. Ou, Fangdie Ye, Jinhao Zhang, Haowen Jiang, Shenghua Liu
          </td>
          <td>2024-12-04</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Breast cancer continues to be a significant global health challenge due to its heterogeneity and propensity for therapeutic resistance. The current tumor, node, and metastasis (TNM) staging and molecular classification systems are limited in capturing the full biological complexity of breast cancer. Myeloid differentiation primary response protein 88 (MyD88), a key adaptor protein in inflammatory signaling pathways, has been implicated in various oncogenic processes. This integrative study combines bulk RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), and immunohistochemistry (IHC) to explore the prognostic significance and immunological implications of MyD88 in breast cancer. We discovered that elevated MyD88 expression is associated with improved patient outcomes and is linked to the immunological landscape of breast cancer, including immune cell infiltration and the expression of immune checkpoint genes. Furthermore, our analysis indicates that MyD88 high-expressing tumors are more sensitive to chemotherapeutic agents, suggesting its potential as a predictive biomarker for therapeutic response. A nomogram incorporating MyD88 expression with other clinical variables was developed to estimate survival probabilities, enhancing the model's predictive accuracy. Our findings highlight MyD88 as a promising candidate for personalized medicine approaches in breast cancer, warranting further investigation into its mechanistic role and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49b4bce9d9c2368a8693ac443c4ad7dde991d418" target='_blank'>
              Unveiling the Prognostic Power and Immune Landscape of MyD88 in Breast Cancer: an Integrative Bioinformatics and IHC Approach
              </a>
            </td>
          <td>
            Xin Yu, Qinfang Zhang, Bei Li, Shengrong Sun, Juanjuan Li, Wenge Li
          </td>
          <td>2025-01-01</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7713144e1809faa54b6475ca46362db89d138cc3" target='_blank'>
              In vitro model of retinoblastoma derived tumor and stromal cells for tumor microenvironment (TME) studies
              </a>
            </td>
          <td>
            Emily Alefeld, André Haase, Dario Van Meenen, Bettina Budeus, Oliver Dräger, Natalia Miroschnikov, Saskia Ting, Deniz Kanber, Eva Biewald, Nikolaos Bechrakis, Nicole Dünker, M. Busch
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e8f0b27ff9601215e0751c29f5ce6a0e2b2d75" target='_blank'>
              Metabolism and spatial transcription resolved heterogeneity of glutamine metabolism in cervical carcinoma
              </a>
            </td>
          <td>
            Qian Liu, Jiayu Zhu, Guzalinuer Abulizi, A. Hasim
          </td>
          <td>2024-12-06</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract We present the proteomic profiling of 79 bladder cancers, including treatment‐naïve non‐muscle‐invasive bladder cancer (NMIBC, n = 17), muscle‐invasive bladder cancer (MIBC, n = 51), and neoadjuvant‐treated MIBC (n = 11). Proteins were extracted from formalin‐fixed, paraffin‐embedded samples and analyzed using data‐independent acquisition, yielding >8,000 quantified proteins. MIBC, compared to NMIBC, shows an extracellular matrix (ECM) and immune response signature as well as alteration of the metabolic proteome together with concomitant depletion of proteins involved in cell–cell adhesion and lipid metabolism. Neoadjuvant treatment did not consistently impact the proteome of the residual tumor mass. NMIBC presents two proteomic subgroups that correlate with histological grade and feature signatures of cell adhesion or lipid/DNA metabolism. Treatment‐naïve MIBC presents three proteomic subgroups with resemblance to the basal‐squamous, stroma‐rich, or luminal subtypes and signatures of metabolism, immune functionality, or ECM. The metabolic subgroup presents an immune‐depleted microenvironment, whereas the ECM and immune subgroups are enriched for markers of M2‐like tumor‐associated macrophages and dendritic cells. Markers for natural killer cells are exclusive for the ECM subgroup, and markers for cytotoxic T cells are a hallmark of the immune subgroup. Endogenous proteolysis is increased in MIBC alongside upregulation of matrix metalloproteases, including MMP‐14. Genomic panel sequencing yielded the prototypical profile of prevalent FGRF3 alterations in NMIBC and TP53 alterations in MIBC. Tumor–stroma interactions of MIBC were investigated by proteomic analysis of patient‐derived xenografts, highlighting specific tumor and stroma contributions to the matrisome and tumor‐induced stromal proteome phenotypes. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b04379880ddec4790f247f8fb8cc011748cfe92" target='_blank'>
              Proteomic analysis of non‐muscle invasive and muscle invasive bladder cancer highlights distinct subgroups with metabolic, matrisomal, and immune hallmarks and emphasizes importance of the stromal compartment
              </a>
            </td>
          <td>
            Thien-Ly Julia Dinh, M. Rogg, M. Cosenza-Contreras, M. Li, Max Zirngibl, Nikolett Pintér, K. Kurowski, Frank Hause, Lena Pauli, Fiona Imberg, Alana Huynh, Marlene Schmid, Ievgen Glavinsky, Luisa Braun, Clara Van Wymersch, Luise Bergmann, Xenia Ungefug, Marion Kunz, Tilman Werner, Patrick Bernhard, Guadalupe Espadas, Eva Brombacher, Julia Schueler, Eduard Sabidó, Clemens Kreutz, C. Gratzke, Martin Werner, Markus Grabbert, P. Bronsert, Christoph Schell, Oliver Schilling
          </td>
          <td>2024-11-25</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ef6a160eddfbbad312082d26af8e03b167d27f" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance.
              </a>
            </td>
          <td>
            Michael J Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah R. Amend, James B. Hicks
          </td>
          <td>2024-11-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cancer, a prominent global health concern, involves the uncontrolled growth and dissemination of abnormal cells, forming tumors that can infiltrate nearby tissues and metastasize to distant sites. Early and precise tumor detection and treatments are crucial to impede their spread and mitigate disease progression. This study presents the development of a scoring system that harnesses single-cell RNA sequencing (scRNA-seq) data to predict the suitability of solid tumors, specifically glioblastoma (GBM) and lymphoma, for advanced cell therapies such as CAR-T, NK cell, and TIL therapies. By integrating key cellular factorsincluding tumor-infiltrating cells, immunosuppressive cells, and immune escape markersthis system provides a quantitative assessment of a tumors potential responsiveness to these therapies. Validation with patient-derived samples has demonstrated the system's potential to improve precision in therapy selection, offering a promising tool for personalized oncology. This research not only aims to enhance the effectiveness of cell therapies for cancer patients but also lays the groundwork for applying similar methodologies to other cancer types, advancing the integration of precision medicine in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa40086c8127d386e515ff58d140655d2ef491d" target='_blank'>
              A Scoring System that Leverages Single Cell RNA-Sequencing Data to Predict Cell Therapy Outcome in Solid Tumor Treatment
              </a>
            </td>
          <td>
            Yuwen Li
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="INTRODUCTION Alzheimer’s disease (AD), Dementia with Lewy bodies (DLB), and Parkinson’s disease (PD) represent a spectrum of neurodegenerative disorders (NDDs). Here, we performed the first direct comparison of their transcriptomic landscapes. METHODS We profiled the whole transcriptomes of NDD cortical tissue by snRNA-seq. We used computational analyses to identify common and distinct differentially expressed genes (DEGs), biological pathways, vulnerable and disease-driver cell subtypes, and alteration in cell-to-cell interactions. RESULTS The same vulnerable inhibitory neuron subtype was depleted in both AD and DLB. Potentially disease-driving neuronal cell subtypes were present in both PD and DLB. Cell-cell communication was predicted to be increased in AD but decreased in DLB and PD. DEGs were most commonly shared across NDDs within inhibitory neuron subtypes. Overall, we observed the greatest transcriptomic divergence between AD and PD, while DLB exhibited an intermediate transcriptomic signature. DISCUSSION These results help explain the clinicopathological spectrum of this group of NDDs and provide unique insights into the shared and distinct molecular mechanisms underlying the pathogenesis of NDDs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377545b1eb6117f0f10ff3c8fe9499887d75cc9d" target='_blank'>
              Comparative mapping of single-cell transcriptomic landscapes in neurodegenerative diseases
              </a>
            </td>
          <td>
            E. K. Shwab, Zhaohui Man, Daniel C. Gingerich, Julia E. Gamache, M. Garrett, G. Serrano, T. Beach, Gregory E. Crawford, Allison E Ashley-Koch, O. Chiba-Falek
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17b53b2d6763b6a8cc385c262233a02ff50e016f" target='_blank'>
              Tissue-resident immune cells: from defining characteristics to roles in diseases.
              </a>
            </td>
          <td>
            Jia Li, Chu Xiao, Chunxiang Li, Jie He
          </td>
          <td>2025-01-17</td>
          <td>Signal transduction and targeted therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer spheroids are spherical, three-dimensional (3D), in vitro assemblies of cancer cells, which are gaining importance as a useful model in cancer behavior studies. Designed to simulate key features of the in vivo tumor microenvironment, spheroids offer reliable insights for drug screening and testing applications. We observed contrasting phenotypes in 3D cervical cancer (CC) cultures. Thus, in this study, we compared the proteomes of 3D and traditional two-dimensional (2D) cultures of CC cell lines, HeLa, SiHa, and C33A. When cultured in in-house poly-(2-hydroxyethyl methacrylate)-coated plates under conditions suitable for 3D spheroid formation, these CC cell lines yielded spheroids exhibiting different features. Proteomic analysis of cells cultured in 2D and 3D cultures revealed similar protein profiles but remarkable differences in the expression levels of some proteins. In SiHa and C33A cells, the upregulation of key proteins required for spheroid formation was insufficient for the formation of compact spheroids. In contrast, HeLa cells could form compact spheroids because they upregulated the proteins, including cadherin-binding, cytoskeleton, and adhesion proteins, necessary for spheroid formation during the remodeling process. Overall, this study unravels the mechanisms underlying the formation of spheroids in the commonly used CC cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/815cc2d468c5e900a469f3a75fe62c8fb5f3a378" target='_blank'>
              Proteomic Analysis Reveals Cadherin, Actin, and Focal Adhesion Molecule-Mediated Formation of Cervical Cancer Spheroids
              </a>
            </td>
          <td>
            P. Molika, K. Leetanaporn, W. Chiangjong, Pongsakorn Choochuen, R. Navakanitworakul
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4843e60a35dd8061e986640d8a344bfa3d97b50" target='_blank'>
              Cancer Clones Revised
              </a>
            </td>
          <td>
            L. Laplane
          </td>
          <td>2024-12-23</td>
          <td>Biological Theory</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6984511f4cd3709e0746596a9a57c5a7832bd7" target='_blank'>
              Ovarian tumor cells gain competitive advantage by actively reducing the cellular fitness of microenvironment cells.
              </a>
            </td>
          <td>
            E. Madan, António M Palma, Vignesh Vudatha, Amit Kumar, P. Bhoopathi, Jochen Wilhelm, T. Bernaś, P. Martin, Gaurav Bilolikar, Aenya Gogna, M. L. Peixoto, Isabelle Dreier, Thais Fenz Araujo, Elena Garre, Anna Gustafsson, K. Dorayappan, Narsimha Mamidi, Zhaoyu Sun, Michail Yekelchyk, Davide Accardi, Amalie Lykke Olsen, Lin Lin, Asaf Ashkenazy Titelman, Michael Bianchi, Phil Jessmon, Elnaz Abbasi Farid, A. Pradhan, L. Neufeld, E. Yeini, Santanu Maji, Christopher J Pelham, Hyobin Kim, Daniel Oh, Hans Olav Rolfsnes, Rita C Marques, Amy Lu, Masaki Nagane, Sahil Chaudhary, Kartik Gupta, Keshav C Gogna, Ana Bigio, Karthikeya Bhoopathi, Padmanabhan Mannangatti, K. G. Achary, Javed Akhtar, S. Belião, Swadesh K. Das, I. Correia, Cláudia L da Silva, A. Fialho, M. Poellmann, Kaila Javius-Jones, Adam M Hawkridge, Sanya Pal, Kumari S Shree, E. Rakha, Sambhav Khurana, Gaoping Xiao, Dongyu Zhang, Arjun Rijal, Charles Lyons, Steven R. Grossman, David P. Turner, R. Pillappa, Karanvir Prakash, Gaurav Gupta, Gary L W G Robinson, Jennifer E Koblinski, Hongjun Wang, Gita Singh, Sujay Singh, Sagar Rayamajhi, M. Bacolod, Hope Richards, Sadia Sayeed, Katherine P Klein, David Chelmow, R. Satchi‐Fainaro, K. Selvendiran, Denise Connolly, F. Thorsen, Rolf Bjerkvig, K. Nephew, Michael O Idowu, M. Kühnel, Christopher Moskaluk, Seungpyo Hong, William L Redmond, Göran Landberg, Antonio Lopez-Beltran, Andrew Poklepovic, Arun J Sanyal, Paul B Fisher, George M. Church, U. Menon, Ronny Drapkin, Andrew K Godwin, Yonglun Luo, Maximilian Ackermann, A. Tzankov, Kirsten D. Mertz, Danny Jonigk, Allan Tsung, David Sidransky, Jose Trevino, Arturo P Saavedra, Robert Winn, Kyoung Jae Won, Eduardo Moreno, Rajan Gogna
          </td>
          <td>2024-12-09</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da415d9954dd04158cfcfaa83dc1880ca38c0523" target='_blank'>
              Single-cell and spatial transcriptomics reveal SPP1-CD44 signaling drives primary resistance to immune checkpoint inhibitors in RCC
              </a>
            </td>
          <td>
            Junfeng Zhang, Qingyan Peng, Jin Fan, Fuzhong Liu, Hongbo Chen, X. Bi, Shuai Yuan, Wei Jiang, Ting Pan, Kailing Li, Sihai Tan, Peng Chen
          </td>
          <td>2024-12-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="To elucidate aging-associated cellular population dynamics, we present PanSci, a single-cell transcriptome atlas profiling over 20 million cells from 623 mouse tissues across different life stages, sexes, and genotypes. This comprehensive dataset reveals more than 3,000 unique cellular states and over 200 aging-associated cell populations. Our panoramic analysis uncovered organ-, lineage-, and sex-specific shifts of cellular dynamics during lifespan progression. Moreover, we identify both systematic and organ-specific alterations in immune cell populations associated with aging. We further explored the regulatory roles of the immune system on aging and pinpointed specific age-related cell population expansions that are lymphocyte dependent. Our "cell-omics" strategy enhances comprehension of cellular aging and lays the groundwork for exploring the complex cellular regulatory networks in aging and aging-associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eb75182d99410329128884679de3564cd17ca18" target='_blank'>
              A panoramic view of cell population dynamics in mammalian aging.
              </a>
            </td>
          <td>
            Zehao Zhang, Chloe Schaefer, Weirong Jiang, Ziyu Lu, Jasper Lee, Andras Sziraki, Abdulraouf Abdulraouf, Brittney Wick, Maximilian Haeussler, Zhuoyan Li, Gesmira Molla, Rahul Satija, Wei Zhou, Junyue Cao
          </td>
          <td>2024-11-28</td>
          <td>Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="High-grade serous ovarian cancer, the most common and aggressive form of ovarian cancer, generally metastasises to visceral adipose tissues. In these tissues, the extracellular matrix through which ovarian cancer cells adhere and migrate is confined by the presence and preponderance of adipocytes. How cells migrate in this unique environment is not known, yet critical to understanding metastatic progression. To study these processes, we develop biomimetic organo-hydrogels that recreate structural and mechanical properties of human visceral adipose tissues. We show that ovarian cancer cells present invasive tropism towards organo-hydrogels, replicating the behaviour observed in native adipose tissues. This migration is facilitated by the mechanical anisotropy and microstructure of organo-hydrogels and adipose tissues, allowing the formation of cell force-induced migratory tracks, a process regulated by TGFβ in an MMP degradation-independent manner. These results highlight the contribution of adipocytes to tissue biophysical features as a key regulatory factor of ovarian cancer cell migration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca5d7b8d644f6db8906b24e33773411b1c21a68a" target='_blank'>
              The mechanical anisotropy of adipose tissues regulates ovarian cancer invasion
              </a>
            </td>
          <td>
            J. Gonzalez-Molina, Parisa Nabili, Daniele Marciano, Sara Abdelnabi, Okan Gultekin, M. Rasul, Yikun Zhang, Clémence Nadal, A. Chrysanthou, Twana Alkasalias, S. Salehi, Frances Balkwill, K. Lehti, Julien E. Gautrot
          </td>
          <td>2024-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Adenocarcinoma of the esophagogastric junction (AEGJ) is a highly aggressive tumor that frequently metastasizes to the liver. Understanding the cellular and molecular mechanisms that drive this process is essential for developing effective therapies.We employed single-cell RNA sequencing to analyze the tumor heterogeneity and microenvironmental landscape in patients with AEGJ liver metastases. This approach enabled us to characterize the diverse cell populations involved in the liver metastatic process.Our analysis revealed a significant involvement of fibroblasts and mural cells in AEGJ liver metastasis. We identified a specific fibroblast type in AEGJ liver metastasis and observed distinct gene expression patterns between adenocarcinoma of the esophagogastric junction and other stomach adenocarcinomas. Our study demonstrated high expression of the SFRP2 gene in pericyte cells during the liver metastasis of AEGJ. The incorporation of GEO, TCGA, and immunofluorescence staining of SFRP2 expression enhanced our study. High expression of SFRP2 in pericytes may influence vascular stability and angiogenesis through the Wnt pathway.Our study provides novel insights into the cellular interactions and molecular mechanisms that underlie AEGJ liver metastasis. Targeting the identified subtype of fibroblasts or influencing SFRP2 gene expression in pericytes may offer new therapeutic strategies for combating this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76cf1003bffe0097e28dee22754d6f4e46b48db" target='_blank'>
              Integrated single-cell RNA sequencing reveals the tumor heterogeneity and microenvironment landscape during liver metastasis in adenocarcinoma of esophagogastric junction
              </a>
            </td>
          <td>
            Junrui Xu, Ussama Sadiq, Wangruizhi Zhao, Hengbo Xia, Yiwei Liu, Renquan Zhang, Aman Xu
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c214c85284c0f7c6d4f85e6f9096b58eba34d42" target='_blank'>
              Dissecting tumor transcriptional heterogeneity from single-cell RNA-seq data by generalized binary covariance decomposition.
              </a>
            </td>
          <td>
            Yusha Liu, P. Carbonetto, Jason Willwerscheid, Scott A. Oakes, Kay F Macleod, Matthew Stephens
          </td>
          <td>2025-01-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="NPM1-mutated AML is one of the largest entities in international classification systems of myeloid neoplasms, which are based on integrating morphologic and clinical data with genomic data. Previous research, however, indicates that bulk transcriptomics-based subtyping may improve prognostication and therapy guidance. Here, we characterized the heterogeneity in NPM1-mutated AML by performing single-cell RNA-sequencing and spectral flow cytometry on 16 AML belonging to three distinct subtypes previously identified by bulk transcriptomics. Using single-cell expression profiling we generated a comprehensive atlas of NPM1-mutated AML, collectively reconstituting complete myelopoiesis. The three NPM1-mutated transcriptional subtypes showed consistent differences in the proportions of myeloid cell clusters with distinct patterns in lineage commitment and maturational arrest. In all samples, malignant cells were detected across different myeloid cell clusters, indicating that NPM1-mutated AML are heavily skewed but not fully arrested in myelopoiesis. Same-sample multi-color spectral flow cytometry recapitulated these skewing patterns, indicating that the three NPM1-mutated subtypes can be consistently identified across platforms. Moreover, our analyses highlighted differences in the abundance of rare hematopoietic stem cells suggesting that skewing occurs early in myelopoiesis. To conclude, by harnessing single-cell RNA-sequencing and spectral flow cytometry, we provide a detailed description of three distinct and reproducible patterns in lineage skewing in NPM1-mutated AML that may have potential relevance for prognosis and treatment of patients with NPM1-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d2c7d3f716516ca564f7acecb6aa3ff7231897" target='_blank'>
              Resolving inter- and intra-patient heterogeneity in NPM1-mutated AML at single-cell resolution
              </a>
            </td>
          <td>
            E. O. Karakaslar, E. Argiro, N. E. Struckman, R. H. Shirali, J. Severens, W. Honders, S. L. Kloet, H. Veelken, M. Reinders, M. Griffioen, E. B. van den Akker
          </td>
          <td>2024-12-11</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spatial transcriptomics, proteomics, and epigenomics are innovative technologies which offer an unparalleled resolution and wealth of data in understanding and the interpretation of cellular functions and interactions. These techniques allow researchers to investigate gene and protein expressions at an individual cell level, revealing cellular heterogeneity within, for instance, bioengineered tissues and classifying novel and rare cell populations that could be essential for the function of the tissues and in disease processes. It is possible to analyze thousands of cells simultaneously, which gives thorough insights into the transcriptomic view of complex tissues. Spatial transcriptomics combines gene expressions with spatial information, conserving tissue architecture and making the mapping of gene activity across different tissue regions possible. Despite recent advancements in these technologies, they face certain limitations. Single-cell transcriptomics can suffer from technical noise and dropout events, leading to incomplete data. Its applicability has been limited by the complexity of data integration and interpretation, although better resolution and tools for the interpretation of data are developing fast. Spatial proteomics and spatial epigenomics provide data on the distribution of proteins and the gene regulatory aspects in tissues, respectively. The disadvantages of these approaches include rather costly and time-consuming analyses. Nevertheless, combining these techniques promises a more comprehensive understanding of cell function and tissue organization, which can be predicted to be useful in achieving better knowledge of cell guidance in tissue-engineered constructs and a higher quality of tissue technology products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b910cb889d8cf7dd3b82978f6e3916ced3b58d85" target='_blank'>
              Spatial Transcriptomics, Proteomics, and Epigenomics as Tools in Tissue Engineering and Regenerative Medicine
              </a>
            </td>
          <td>
            M. Lammi, C. Qu
          </td>
          <td>2024-12-01</td>
          <td>Bioengineering</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Stromal cells are non‐hematopoietic cells that consist of endothelial cells and various mesenchymal cell populations. The composition of the stromal cell compartment is diverse in different organs. Numerous recent studies demonstrated that the lung environment contains heterogeneous mesenchymal stromal cell populations with distinctive genomic signatures and location preferences. Besides their role in supporting organ structure and remodeling tissue, mesenchymal stromal cells fulfill critical immune functions. These stromal cells show alterations during lung fibrosis and infectious disorders like COVID‐19 or flu infection. To date, their identification and isolation were challenging, and most information about their heterogeneity was derived from scRNAseq data. In this protocol, we describe an isolation, comprehensive flow cytometry assessment, and purification strategy for murine lung stromal cells. The described method is optimized for minimizing cell death while keeping a high level of cell purity. This protocol can be also used for ex‐vivo analysis of these cells in downstream functional assays. © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Isolation of stromal cells from murine lung tissue Basic Protocol 2: Flow cytometry assessment of lung stromal populations Basic Protocol 3: Purification of lung fibroblastic stromal cells Alternate Protocol: Positive selection of fibroblastic stromal cells">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768210ba3f920bfb05c8ef8764e7bd739b49227" target='_blank'>
              Isolation, Purification, and Comprehensive Flow Cytometry Assessment of Lung Stromal Cells
              </a>
            </td>
          <td>
            Sophia Rottmann, Veronika Lukacs-Kornek
          </td>
          <td>2024-12-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="There is an urgent need of precision medicine for ovarian cancer patients to identify patients who respond to chemotherapy and PARP inhibitors, a therapy targeting homologous recombination deficiency (HRD). Here we established a panel of 37 long-term patient-derived tumor organoids (PDTO) models of various histological subtypes from 224 patients and demonstrated that they mimic the histological and molecular characteristics of original tumors. Screening of chemotherapeutic drugs showed that PDTO exhibit heterogeneous responses, and that response of PDTO from high-grade serous ovarian carcinoma to carboplatin recapitulated patient response to first-line treatment. Additionally, the detection of HRD phenotype of PDTO using functional assay was associated with the results of the HRD test Genomic Instability Scar (GIScar). Although larger-scale investigations are needed to confirm the predictive potential of PDTO, these results provide further evidence of the potential interest of ovarian PDTO for functional precision medicine even if many challenges remain to be addressed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9373b0fdccc45bac208c5469698121e6a4da07c" target='_blank'>
              Long-term patient-derived ovarian cancer organoids closely recapitulate tumor of origin and clinical response
              </a>
            </td>
          <td>
            L. Thorel, E. Dolivet, P-M. Morice, R. Florent, J. Divoux, M. Perréard, L. Lecouflet, G. Desmartin, C. Marde Alagama, F. Giffard, A. Leconte, J. Lequesne, B. Clarisse, M. Briand, A. Traore, C. Villenet, JP. Meneboo, G. Babin, L. Gaichies, S. Martin-Françoise, JF. Le Brun, R. Rouzier, E. Brotin, C. Denoyelle, N. Vigneron, R. Leman, D. Vaur, L. Castera, C. Blanc-Fournier, N. Elie, B. Plancoulaine, F. Joly, M. Meryet-Figuiere, M. Figeac, L. weiswald, L. Poulain
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc37b95c1e16c6aa07dc838b1c35715a1b58b94d" target='_blank'>
              Transcription factors ASCL1 and OLIG2 drive glioblastoma initiation and co-regulate tumor cell types and migration
              </a>
            </td>
          <td>
            Bianca L Myers, Kathryn J. Brayer, Luis E Paez-Beltran, Estrella Villicana, Matthew S. Keith, Hideaki Suzuki, Jessie Newville, Rebekka H Anderson, Yunee Lo, Conner M Mertz, R. Kollipara, Mark D. Borromeo, Q. R. Lu, R. Bachoo, Jane E Johnson, T. Vue
          </td>
          <td>2024-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59e5b41f611d8bd26c1558060c6ef330913ca3b" target='_blank'>
              Pan-cancer drivers of metastasis
              </a>
            </td>
          <td>
            Ryan Lusby, Engin Demirdizen, Mohammed M A Inayatullah, Paramita Kundu, O. Maiques, Ziyi Zhang, Mikkel G Terp, Victoria Sanz-Moreno, Vijay K Tiwari
          </td>
          <td>2025-01-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/452c65f4f9faa535e4f661db53c177e8e8787cfc" target='_blank'>
              KANK1 promotes breast cancer development by compromising Scribble-mediated Hippo activation
              </a>
            </td>
          <td>
            S. S. Guo, Zhiying Liu, Guan M. Wang, Zhiqi Sun, Kaikai Yu, James P Fawcett, Reinhard Buettner, Bo Gao, Reinhard Fässler
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Aim: Lung adenocarcinoma (LUAD), the most prevalent subtype of non-small cell lung cancer (NSCLC), presents significant clinical challenges due to its high mortality and limited therapeutic options. The molecular heterogeneity and the development of therapeutic resistance further complicate treatment, underscoring the need for a more comprehensive understanding of its cellular and molecular characteristics. This study sought to delineate novel cellular subpopulations and molecular subtypes of LUAD, identify critical biomarkers, and explore potential therapeutic targets to enhance treatment efficacy and patient prognosis.
 Methods: An integrative multi-omics approach was employed to incorporate single-cell RNA sequencing (scRNA-seq), bulk transcriptomic analysis, and genome-wide association study (GWAS) data from multiple LUAD patient cohorts. Advanced computational approaches, including Bayesian deconvolution and machine learning algorithms, were used to comprehensively characterize the tumor microenvironment, classify LUAD subtypes, and develop a robust prognostic model.
 Results: Our analysis identified eleven distinct cellular subpopulations within LUAD, with epithelial cells predominating and exhibiting high mutation frequencies in Tumor Protein 53 (TP53) and Titin (TTN) genes. Two molecular subtypes of LUAD [consensus subtype (CS)1 and CS2] were identified, each showing distinct immune landscapes and clinical outcomes. The CS2 subtype, characterized by increased immune cell infiltration, demonstrated a more favorable prognosis and higher sensitivity to immunotherapy. Furthermore, a multi-omics-driven machine learning signature (MOMLS) identified ribonucleotide reductase M1 (RRM1) as a critical biomarker associated with chemotherapy response. Based on this model, several potential therapeutic agents targeting different subtypes were proposed.
 Conclusion: This study presents a comprehensive multi-omics framework for understanding the molecular complexity of LUAD, providing insights into cellular heterogeneity, molecular subtypes, and potential therapeutic targets. Differential sensitivity to immunotherapy among various cellular subpopulations was identified, paving the way for future immunotherapy-focused research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6451a0979890a61b3396616f5a7151f271c418" target='_blank'>
              Integrative multi-omics analysis for identifying novel therapeutic targets and predicting immunotherapy efficacy in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zilu Chen, Kun Mei, Foxing Tan, YuHeng Zhou, Haolin Du, Min Wang, Renjun Gu, Yan Huang
          </td>
          <td>2025-01-14</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="T cell exhaustion remains a significant barrier to immunotherapeutic success for many patients with solid tumors. Growing evidence suggests that enhanced survival and self-renewal properties of a stem-like precursor T cell population (Tpex) is correlated with a survival advantage in immunotherapy. In a recent study published in Science, Kang and colleagues find three epigenetic regulators commonly mutated in clonal hematopoiesis also control Tpex progression to exhaustion. By leveraging the finding that patients with enhanced survival in myelodysplastic syndrome (MDS) had T cell mutations in the ASXL1 gene, this study demonstrates that loss of ASXL1 in T cells preserves their stem-cell like properties of self-renewal and survival leading to increased anti-tumor responses when combined with immunotherapy in both mouse models and human cancers. These findings have significant implications for new therapeutic options that target epigenetic modifiers promoting exhaustion together with immune checkpoint blockade to improve response rates in patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9601064851c3c05e0025668607eb53660bf933ac" target='_blank'>
              Stayin' Alive: Targeting chromatin regulators of clonal hematopoiesis promotes CD8 T cell stemness.
              </a>
            </td>
          <td>
            Xingjian Qiu, Aaron Yang, Amanda C Poholek
          </td>
          <td>2024-11-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Canine oral melanoma is an aggressive cancer in dogs that may respond to new treatments designed to boost the immune system’s ability to fight the disease. In this study, we investigated whether mesenchymal stem cells derived from fat tissue could stimulate the immune system against this cancer. Mesenchymal stem cells are known to produce proteins called cytokines that help support immune functions. We tested whether combining these stem cells with another type of immune cell could inhibit cancer cell viability and migration. We also examined three key interleukins—2, 8, and 12—associated with immune responses to understand how they might reduce cancer cell growth and migration. Our findings showed that mesenchymal stem cells indeed reduced cancer cell migration and survival, although the effects varied depending on the cancer cell type, suggesting that some cells respond better than others. These results could aid in developing new, less invasive therapies that harness the immune system to fight cancer in dogs, potentially reducing the need for more aggressive treatments and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6dfb3b999d343db4306ba7210d51ed8bf9a10aa" target='_blank'>
              Canine Adipose-Derived Mesenchymal Stromal Cells Reduce Cell Viability and Migration of Metastatic Canine Oral Melanoma Cell Lines In Vitro
              </a>
            </td>
          <td>
            Fwu Shing Teng, Patrícia de Faria Lainetti, Mayara Simão Franzoni, Antonio Fernando Leis Filho, Cristina de Oliveira Massoco Salles Gomes, Renée Laufer-Amorim, Rogério Martins Amorim, C. Fonseca‐Alves
          </td>
          <td>2024-12-01</td>
          <td>Veterinary Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="BACKGROUND
To meet their high energy needs, tumor cells undergo aberrant metabolic reprogramming. A tumor cell may expertly modify its metabolic pathways and the differential expression of the genes for metabolic enzymes. The physiological requirements of the host tissue and the tumor cell of origin mostly dictate metabolic adaptation. Ameloblastoma (AB) is a benign odontogenic tumor of epithelial origin. Due to its unrestricted growth potential, local aggressiveness, and high likelihood of recurrence, this condition poses a significant risk to the patient's health. This study aimed to characterize the metabolic heterogeneity at single-cell resolution of AB.


METHODS
Single-cell RNA sequencing (scRNA-seq) was performed on 17,284 cells from three AB donors. Bioinformatic analysis was used to examine differentially expressed genes, subtypes, and regulatory mechanisms when combined with odontogenic keratocyst scRNA-seq data. Based on metabolic pathway gene sets, the metabolic landscape of AB tumor cells was examined.


RESULTS
Using scRNA-seq, we discovered that AB tumor cells had substantial heterogeneity. The biggest contributor to tumor cell metabolic characteristics is determined to be variation in mitochondrial programming and glycolysis. Surprisingly, hypoxia corresponds with both oxidative phosphorylation and glycolysis activity in AB tumor cells at the single-cell level.


CONCLUSION
This study presents a computational framework for defining metabolism using single-cell expression data and identifies oxidative phosphorylation and glycolysis as critical components of metabolism for AB tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c116028597b1c5eaf72b4b7ed869064e15002ef" target='_blank'>
              Metabolic Analysis of Tumor Cells Within Ameloblastoma at the Single-Cell Level.
              </a>
            </td>
          <td>
            Rui-Fang Li, Yi Zhao, Qi-Wen Man
          </td>
          <td>2024-12-31</td>
          <td>Oral diseases</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Glioblastoma (GBM) is an aggressive primary central nervous system tumor with poor survival outcomes and limited treatment options. In this study, we investigate the use of radiomic features derived from magnetic resonance imaging (MRI) scans to identify unique gene expression profiles in a cohort of patients with GBM. This study grouped patients based on radiomic features using a consensus clustering approach, which iteratively clusters patients to find robust and stable groups. We identified three clusters which yielded unique gene expression profiles. Significant differentially expressed genes previously associated with GBM prognosis and treatment sensitivity were identified in one cluster. In pathway enrichment analyses, genes upregulated in immune-related and DNA metabolism pathways and downregulated protein and histone deacetylation pathways were identified in the same cluster. Together, these findings suggest that consensus clustering of radiomic features may be a promising avenue for non-invasive characterization of molecular heterogeneity of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9a0c25b9c653744ad1d54a820df2dd878397385" target='_blank'>
              Radiomic Consensus Clustering in Glioblastoma and Association with Gene Expression Profiles
              </a>
            </td>
          <td>
            Tadeusz H. Wroblewski, Mert Karabacak, Carina Seah, Raymund L. Yong, Konstantinos Margetis
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT
Tumor-infiltrating lymphocytes (TILs) are key components of the tumor microenvironment (TME) and serve as prognostic markers for breast cancer. Patients with high TIL infiltration generally experience better clinical outcomes and extended survival compared to those with low TIL infiltration. However, as the TME is highly complex and TIL subtypes perform distinct biological functions, TILs may only provide an approximate indication of tumor immune status, potentially leading to biased prognostic results. Therefore, we reviewed the interactions between immune-infiltrating subtypes and tumor cells throughout the entire TME. By examining the antitumor or protumor effects of each TIL subtype, we aimed to better characterize the tumor immune landscape, offering more accurate and comprehensive insights for guiding triple-negative breast cancer (TNBC) treatment. In addition, this approach could lead to the development of new therapeutic targets, enabling tailored treatment strategies and precision medicine. Accumulating evidence suggests that the intestinal microbiome and its metabolites influence antitumor responses by modulating innate and adaptive immunity, with specific bacteria potentially serving as biomarkers for predicting clinical responses. Various studies have identified microorganisms in breast tissue, previously considered sterile, revealing differences in breast microbial composition between patients with breast cancer and controls, as well as associations between specific breast microorganisms and clinicopathologic features, including immune correlations. The aim of this review was to provide a more comprehensive set of prognostic markers for TNBC and to tap into potential-specific therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e0ddc5858a8a27ce545faa706c8e06b464e3d7" target='_blank'>
              Prognostic value of tumor-infiltrating lymphocyte subtypes and microorganisms in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Yating Shi, Zhi Guo, Qiang Wang, Huan Deng
          </td>
          <td>2024-12-01</td>
          <td>Journal of cancer research and therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
Small cell neuroendocrine carcinoma of the cervix (SCNECC) is a rare gynecologic malignant tumor, which is lack of systematic research. In order to investigate its molecular characteristics, origin and pathogenesis, single-cell transcriptome sequencing (scRNA-Seq) of SCNECC were performed for the first time, the cellular and molecular landscape was revealed and the key genes for clinical prognosis were screened.


METHODS
This article initially performed the scRNA-Seq on a tumor tissue sample from a SCNECC patient, combined with scRNA-Seq data from a healthy cervical tissue sample downloaded from public database, the single-cell transcriptome landscape was constructed. Then, we investigated the cell types, intratumoral heterogeneity, characteristics of tumor microenvironment and potential predictive markers of SCNECC.


RESULTS
We identified two malignant cell populations, tumor stem cells and malignant carcinoma cells, and revealed two tumor progression pathways of SCNECC. By analyzing gene expression levels in the pathophysiology of SCNECC, we found that the expression levels of ERBB4 and NRG1, as well as the expression profile of mTOR signaling pathway mediated by them, were significantly upregulated in malignant carcinoma cells. In addition, we also found that carcinoma cells were able to stimulate malignant cell proliferation through FN1 signaling pathway. The immune cells were in a stress state, with T cell depletion, macrophage polarization and mast cell glycolysis, these results suggested that carcinoma cells could interfere immune response and promote tumor escape through MIF, TGFb and other immunosuppressive related signaling pathways.


CONCLUSION
This study revealed the mechanism of genesis and progression in SCNECC and the related important signaling pathways, such as mTOR, and provided new insights into the treatment of SCNECC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e8ee7f62717b58fdef571a5207613833c5ee56" target='_blank'>
              Single-cell transcriptome sequencing and analysis provide a new approach for the treatment of small cell neuroendocrine carcinoma of the cervix.
              </a>
            </td>
          <td>
            Lewei He, Yuling Wu, Mingyi Lv, Jiyang Jiang, Yifei Li, Tao Guo, Zhenxin Fan
          </td>
          <td>2024-11-27</td>
          <td>Neuroendocrinology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The molecular features determining the risk of metachronous metastases in clear cell renal cell carcinoma (ccRCC) are poorly defined. Using a systematic tumor transcriptome deconvolution approach, we investigated the genomic and transcriptomic profiles of 192 ccRCC primary tumors with extended clinical follow-up to identify cancer and stromal cell molecular features associated with metastatic risk. At the genomic level, we identified a significantly higher frequency of copy number loss at 1p31-36 in primary tumors that later progressed with metastases. Tumor transcriptome deconvolution identified significant down-regulation of epithelial cell polarity, including PATJ (1p31), and fatty acid metabolism, including CYP4A11 (1p33), in cancer cells of tumors that developed metastatic progression. We developed and benchmarked a compact 5-feature predictive model (5G) that demonstrated improved accuracy over existing ccRCC gene signatures in the prediction of metachronous metastasis risk. Overall, our study highlights convergent genomic and transcriptomic alterations in chromosome 1p, driving dysregulation of epithelial cell polarity and fatty acid metabolism, as putative novel risk factors of metachronous metastasis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85898b20987df3b296ae9761d05ca0764e6d312" target='_blank'>
              Convergent genomic and molecular features predict risk of metachronous metastasis in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            M. Naeini, Mengyuan Pang, Neha Rohatgi, Sinem Kadioglu, Umesh Ghoshdastider, R. DiNatale, Roy Mano, A. Hakimi, A. Skanderup
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT Background Metastatic colorectal cancer (mCRC) is the main cause of CRC mortality, with limited treatment options. Although immunotherapy has benefited some cancer patients, mCRC typically lacks the molecular features that respond to this treatment. However, recent studies indicate that the immune microenvironment of mCRC may be modified to enhance the effect of immune checkpoint inhibitors. This study aimed to explore the metastatic tumor microenvironment (TME) by comparing cell populations in colorectal liver (CLM), lung (mLu), and peritoneal (PM) metastases. Methods RNA isolated from 20 CLM, 15 mLu, and 35 PM samples was subjected to mRNA sequencing and explored through TME deconvolution tools, consensus molecular subtyping (CMS), and differential gene expression and gene set enrichment analysis, with respect to the metastatic sites. Clinical data and KRAS/BRAF hotspot mutation status were also obtained for all the cases. Results The cell type fractions in the TME were relatively similar between the metastatic sites, except for cancer‐associated fibroblasts (CAFs), B cells, endothelial cells, and CD4+ T cells. Notably, PM showed enrichment for CAFs and endothelial cells, consistent with distinct pathways associated with metastatic growth and progression in the peritoneal cavity. PM with the mesenchymal subtype, CMS4, had increased CAFs, endothelial cells, and macrophages, along with up‐regulated genes related to TNF‐α signaling via NF‐κB, EMT, and angiogenesis. Conclusions Tumor samples from different metastatic sites exhibited a broadly similar TME in terms of immune cell composition, with some intriguing differences. Targeting CAF‐associated pathways, macrophages, and TNF‐α signaling through NR4A could represent potential novel therapeutic approaches in CMS4 PM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e496a24f62076b578b328b6a82488e3686ba6b6" target='_blank'>
              Enrichment of Cancer‐Associated Fibroblasts, Macrophages, and Up‐Regulated TNF‐α Signaling in the Tumor Microenvironment of CMS4 Colorectal Peritoneal Metastasis
              </a>
            </td>
          <td>
            Eirik Høye, Chakravarthi Kanduri, A. Torgunrud, Susanne Lorenz, Bjørn Edwin, S. Larsen, Å. Fretland, V. Dagenborg, K. Flatmark, Christin Lund-Andersen
          </td>
          <td>2024-12-30</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Tumor growth and progression involve coordinated regulation by internal, microenvironmental, and systemic signals and often display conspicuous sexual dimorphism. The mechanisms governing the integration and coordination of these signals, along with their sex-based differences, remain largely unknown. Using a Drosophila tumor model originating from nonreproductive tissue, we show that female-biased tumor growth involves multifaceted communications among tumor cells, hemocytes, and neuroendocrine insulin-producing cells (IPCs). Notch-active tumor cells recruit hemocytes carrying the tumor necrosis factor–α (TNF-α) homolog Eiger to the tumor microenvironment (TME), activating the c-Jun N-terminal kinase (JNK) pathway in tumor cells, instigating the sexually dimorphic up-regulation of cytokine Unpaired 2 (Upd2). Upd2, in turn, exerts a distal influence by modulating the release of a Drosophila insulin-like peptide (Dilp2) from IPCs. Dilp2 then activates the insulin signaling in the tumor, thereby fostering sexual-dimorphic tumor growth. Together, these findings reveal a relay mechanism involving the TME and systemic signals that collectively control the sexual dimorphism of tumor growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4504e9cb20bc72b2973f3610b4d0ff83ae5829a" target='_blank'>
              Sex-dimorphic tumor growth is regulated by tumor microenvironmental and systemic signals
              </a>
            </td>
          <td>
            Xianfeng Wang, Hongcun Bao, Yi-Chun Huang, Anindita Barua, Chun-Ming Lai, Jie Sun, Youfang Zhou, Fei Cong, Shangyu Gong, Chih-Hsuan Chang, Wu-Min Deng
          </td>
          <td>2024-12-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62a0b0314fc694a0d5ac1d3e0908ca758f9f880c" target='_blank'>
              Cancer-associated fibroblasts, tumor and radiotherapy: interactions in the tumor micro-environment
              </a>
            </td>
          <td>
            Kris Raaijmakers, G. Adema, Johan Bussink, M. Ansems
          </td>
          <td>2024-12-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcd5c512a1a95d505528decdc0fed35eccbfc51" target='_blank'>
              Insights from a multiscale framework on metabolic rate variation driving glioblastoma multiforme growth and invasion
              </a>
            </td>
          <td>
            M. Amereh, S. Shojaei, A. Seyfoori, Tavia Walsh, P. Dogra, V. Cristini, B. Nadler, Mohsen Akbari
          </td>
          <td>2024-11-25</td>
          <td>Communications Engineering</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="While optimizing cancer cell growth conditions, we discovered that cancer stem cells can generate second membranes outside the plasma membranes forming compartments separated from the extracellular matrix. The encapsulating membranes can extend and generate long cytocapsular tubes, wherein multiple cells can migrate. SILAC proteomics of the second cytocapsular membranes identified 400 membrane proteins, and a small subset of them are highly upregulated in cytocapsular cancers compared to normal tissues. The ATP-dependent calcium pump PMCA2 is one of the highest upregulated factors of the cytocapsular membrane, and antibodies serve as biomarkers for malignant tumors, as checked for 293 subtypes of cancers. Cytocapsular tumors have not been described before, possibly because the CC membranes do not exhibit epitopes targeted by conventional methods, and no efforts have been made to search for new cancer specific organelles. Antibodies against PMCA2 can now be used to map cancer evolution pathways in human bodies by comparisons of more than 12 000 annotated specimens from tissue banks worldwide. The current research reveals that the native malignant cancer cell is enclosed in a cytocapsular membrane. With the PMCA2 cancer biomarker available, the development of human cancers can be studied from cancer tissue banks and clinical cancer biopsies with a previously unknown diversity. The emerging knowledge on cancer-driving biomarkers opens doors for new routes in cancer diagnosis, surgery, therapy, and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4e5b795160ba02ecb2832797cada359373d1df" target='_blank'>
              Discovery of the Cytocapsular Membrane as Hallmark of Malignant Tumors.
              </a>
            </td>
          <td>
            Tingfang Yi, G. Wagner
          </td>
          <td>2024-12-16</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Neuroblastoma (NB) remains associated with high mortality and low initial response rate, especially for high-risk patients, thus warranting exploration of molecular markers for precision risk classifiers. Through integrating multiomics profiling, we identified a range of hub genes involved in cell cycle and associated with dismal prognosis and malignant cells. Single-cell transcriptome sequencing revealed that a subset of malignant cells, subcluster 1, characterized by high proliferation and dedifferentiation, was strongly correlated with the hub gene signature and orchestrated an immunosuppressive tumor microenvironment (TME). Furthermore, we constructed a robust malignant subcluster 1 related signature (MSRS), which was an independent prognostic factor and superior to other clinical characteristics and published signatures. Besides, TME differences conferred remarkably distinct therapeutic responses between high and low MSRS groups. Notably, polo-like kinase-1 (PLK1) was one of the most crucial contributors to MSRS and remarkably correlated with malignant subcluster 1, and PLK1 inhibition was effective for NB treatment as demonstrated by in silico analysis and in vitro experiments. Overall, our study constructs a novel molecular model to further guide the clinical classification and individualized treatment of NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33949844ac45f5e41b1ac34112dc1705f6129d2" target='_blank'>
              Multiomics Analysis Reveals Neuroblastoma Molecular Signature Predicting Risk Stratification and Tumor Microenvironment Differences.
              </a>
            </td>
          <td>
            Xing Zhou, Zhaokai Zhou, Xiaohan Qin, Jian Cheng, Yongcheng Fu, Yuanyuan Wang, Jingyue Wang, Pan Qin, Da Zhang
          </td>
          <td>2025-01-06</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor metastasis is a complex phenomenon that poses significant challenges to current cancer therapeutics. While the biochemical signaling involved in promoting motile phenotypes is well understood, the role of biomechanical interactions has recently begun to be incorporated into models of tumor cell migration. Specifically, we propose the unjamming transition, adapted from physical paradigms describing the behavior of granular materials, to better discern the transition toward an invasive phenotype. In this review, we introduce the jamming transition broadly and narrow our discussion to the different modes of 3D tumor cell migration that arise. Then we discuss the mechanical interactions between tumor cells and their neighbors, along with the interactions between tumor cells and the surrounding extracellular matrix. We center our discussion on the interactions that induce a motile state or unjamming transition in these contexts. By considering the interplay between biochemical and biomechanical signaling in tumor cell migration, we can advance our understanding of biomechanical control in cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e9eb9ae0500f0833efc931f23521b713ae0671" target='_blank'>
              Unjamming Transition as a Paradigm for Biomechanical Control of Cancer Metastasis.
              </a>
            </td>
          <td>
            Grace Cai, Nicole C Rodgers, Allen P. Liu
          </td>
          <td>2024-12-05</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4f1962ef1598f22c4d709373f72cc9202dbf33" target='_blank'>
              Mesenchymal stromal cells promote the formation of lung cancer organoids via Kindlin-2
              </a>
            </td>
          <td>
            Zhi-lin Sui, Xianxian Wu, Jiaxin Wang, ShihJye Tan, Chao Zhao, Zhentao Yu, Chuanyue Wu, Xiaoxia Wang, Ling Guo
          </td>
          <td>2025-01-09</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Nongenetic plasticity has emerged as a key driver of cancer drug resistance, yet its precise origins, nature, and consequences are not fully clarified. This review examines technological, computational, and conceptual developments in the nongenetic determinants of drug resistance. We begin by proposing refined definitions of cellular state, fate, and plasticity. We subsequently contextualize the findings from multimodal approaches to investigate plasticity, highlighting how new single-cell lineage-tracing methods provide opportunities for quantitatively capturing state transitions, their timescales and heritability, and how they contribute to resistance mechanisms. We also draw parallels with concepts from developmental biology and microbial persistence research. Next, we cover the role that computational approaches have played in revealing the otherwise latent patterns of heterogeneity that underlie plasticity from complex datasets. We conclude by emphasizing the need for standardized terminology in this rapidly evolving field and the path from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38fadd15ca85570a489d3058bda72d96573ce1d" target='_blank'>
              Navigating Nongenetic Plasticity in Cancer Drug Resistance
              </a>
            </td>
          <td>
            Hanxiao Sun, Nitu Kumari, Madeline E. Melzer, Vadim Backman, Yogesh Goyal
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="High-Grade Serous Ovarian Cancer (HGSOC) originates from fallopian tube (FT) precursors. However, the molecular changes that occur as precancerous lesions progress to HGSOC are not well understood. To address this, we integrated high-plex imaging and spatial transcriptomics to analyze human tissue samples at different stages of HGSOC development, including p53 signatures, serous tubal intraepithelial carcinomas (STIC), and invasive HGSOC. Our findings reveal immune modulating mechanisms within precursor epithelium, characterized by chromosomal instability, persistent interferon (IFN) signaling, and dysregulated innate and adaptive immunity. FT precursors display elevated expression of MHC-class I, including HLA-E, and IFN-stimulated genes, typically linked to later-stage tumorigenesis. These molecular alterations coincide with progressive shifts in the tumor microenvironment, transitioning from immune surveillance in early STICs to immune suppression in advanced STICs and cancer. These insights identify potential biomarkers and therapeutic targets for HGSOC interception and clarify the molecular transitions from precancer to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b491f761ec399a1258abdf4e675e3bc81f53e5e" target='_blank'>
              Multimodal Spatial Profiling Reveals Immune Suppression and Microenvironment Remodeling in Fallopian Tube Precursors to High-Grade Serous Ovarian Carcinoma.
              </a>
            </td>
          <td>
            Tanjina Kader, Jia-Ren Lin, C. Hug, S. Coy, Yu-An Chen, Ino de Bruijn, Natalie Shih, Euihye Jung, R. Pelletier, Mariana Lopez Leon, Gabriel Mingo, Dalia K Omran, Jong Suk Lee, Clarence Yapp, B. Satravada, Ritika Kundra, Yilin Xu, Sabrina Chan, Juliann B. Tefft, J. Muhlich, Sarah H Kim, Stefan M. Gysler, Judith Agudo, James R. Heath, N. Schultz, Charles Drescher, P. Sorger, Ronny Drapkin, S. Santagata
          </td>
          <td>2024-12-20</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a high-throughput transcriptomic approach with the power to identify rare cells, discover new cellular subclusters, and describe novel genes. scRNA-seq can simultaneously reveal dynamic shifts in cellular phenotypes and heterogeneities in cellular subtypes. Since the publication of the first protocol on scRNA-seq in 2009, this evolving technology has continued to improve, through the use of cell-specific barcodes, adoption of droplet-based systems, and development of advanced computational methods. Despite induction of the cellular stress response during the tissue dissociation process, scRNA-seq remains a popular technology, and commercially available scRNA-seq methods have been applied to the brain. Recent advances in spatial transcriptomics now allow the researcher to capture the positional context of transcriptional activity, strengthening our knowledge of cellular organization and cell-cell interactions in spatially intact tissues. A combination of spatial transcriptomic data with proteomic, metabolomic, or chromatin accessibility data is a promising direction for future research. Herein, we provide an overview of the workflow, data analyses methods, and pros and cons of scRNA-seq technology. We also summarize the latest achievements of scRNA-seq in stroke and acute traumatic brain injury, and describe future applications of scRNA-seq and spatial transcriptomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0277107d9508977e5eb0c209436914dd7332f9c2" target='_blank'>
              Single-cell RNA sequencing in stroke and traumatic brain injury: Current achievements, challenges, and future perspectives on transcriptomic profiling.
              </a>
            </td>
          <td>
            Ruyu Shi, Huaijun Chen, Wenting Zhang, R. Leak, Dequan Lou, Kong Chen, Jun Chen
          </td>
          <td>2024-12-09</td>
          <td>Journal of cerebral blood flow and metabolism : official journal of the International Society of Cerebral Blood Flow and Metabolism</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="The study by Grossmann and colleagues uses single-nucleus RNA sequencing in a cohort of matched high-risk neuroblastoma primary tumor samples, obtained from the same patient at diagnosis and definitive surgery, to identify persister cells that survive induction chemotherapy. These persister cells utilize mechanisms of chemoresistance that are both tumor-intrinsic and tumor-extrinsic, are highly dependent on the original genetic profile of the tumor, and represent novel, patient-specific targets to precisely inhibit chemoresistance and disease recurrence. See related article by Grossmann et.al., p. 2387.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17960d164122a139c932c5adc28d8d07f55f0399" target='_blank'>
              Characterization of Persister Cells Provides Insights into Mechanisms of Therapy Resistance in Neuroblastoma.
              </a>
            </td>
          <td>
            Amber B. Wolf, C. P. Reynolds, Eveline Barbieri
          </td>
          <td>2024-12-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, , Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5675284f659a316a50b22f64df578d9b883b676a" target='_blank'>
              Single-cell RNA sequencing highlights the unique tumor microenvironment of small cell neuroendocrine cervical carcinoma
              </a>
            </td>
          <td>
            Tianyou Wang, Li Zhang, Song Mei, Bo Wang, Jiaqi Liu, Weiping Yang, Jiongbo Liao, Chao Wang
          </td>
          <td>2025-01-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cervical cancer is the fourth most common cancer in women globally, and the main cause of the disease has been found to be ongoing HPV infection. Cervical cancer remains the primary cause of cancer-related death despite major improvements in screening and treatment approaches, especially in low- and middle-income nations. Therefore, it is crucial to investigate the tumor microenvironment in advanced cervical cancer in order to identify possible treatment targets.In order to better understand malignant cervical cancer epithelial cells (EPCs), this study used bulk RNA-seq data from UCSC in conjunction with single-cell RNA sequencing data from the ArrayExpress database. After putting quality control procedures into place, cell type identification and clustering analysis using the Seurat software were carried out. To clarify functional pathways, enrichment analysis and differential gene expression were carried out. The CIBERSORT and ESTIMATE R packages were used to evaluate the immune microenvironment characteristics, and univariate and multivariate Cox regression analyses were used to extract prognostic features. Furthermore, assessments of drug sensitivity and functional enrichment were carried out.Eight cell types were identified, with EPCs showing high proliferative and stemness features. Five EPC subpopulations were defined, with C1 NNMT+ CAEPCs driving tumor differentiation. A NNMT CAEPCs Risk Score (NCRS) model was developed, revealing a correlation between elevated NCRS scores and adverse patient outcomes characterized by immune evasion. In vitro experiments validated that the prognostic gene PLOD2 significantly enhances proliferation, migration, and invasion of cervical cancer cells.This investigation delineated eight cell types and five subpopulations of malignant EPCs in cervical cancer, establishing the C1 NNMT+ CAEPCs as a crucial therapeutic target. The NCRS model demonstrated its prognostic capability, indicating that higher scores are associated with poorer clinical outcomes. The validation of PLOD2 as a prognostic gene highlights its therapeutic potential, underscoring the critical need for integrating immunotherapy and targeted treatment strategies to enhance diagnostic and therapeutic approaches in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a0fda708b879e8e016206b2e92375962d615799" target='_blank'>
              Single-cell RNA sequencing and immune microenvironment analysis reveal PLOD2-driven malignant transformation in cervical cancer
              </a>
            </td>
          <td>
            Zhiheng Lin, Fengxin Wang, Renwu Yin, Shengnan Li, Yuquan Bai, Baofang Zhang, Chenlin Sui, Hengjie Cao, Dune Su, Lianwei Xu, Honghong Wang
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extranodal NK/T cell lymphoma (ENKTL) poses significant challenges in efficient treatment processes due to its aggressive nature and high recurrence rates. There is a critical need to develop a robust statistical model to predict treatment efficacy by dynamically quantifying biomarkers tailored to various stages of lymphoma. Recent analytics such as sequencing and microbiome tests have only been utilized to understand lymphoma progression and treatment response in clinical settings. However, these methods are limited by their quantitative analysis capabilities, long turnaround times, and lack of single‐cell resolution, which are essential for understanding the heterogeneous nature of lymphoma. In this study, we developed a deep learning‐enhanced image cytometry (DLIC) to investigate biophysical heterogeneities in peripheral blood mononuclear cells (PBMCs) from newly diagnosed (ND) ENKTL patients. We established a substantial cohort of 23 ND ENKTL patients, categorizing them into interim of treatment (n = 21) and end of treatment (n = 19) stages along their serial treatment timelines. Using a basic optical microscope and a commercial microchip, we analyzed over 270,000 single PBMCs in high‐throughput, profiling their size, eccentricity, and refractive index in a completely label‐free and quantified manner through AI‐based nanophotonic computation. We observed distinct heterogeneity variations in these three biophysical indicators across treatment stages and relapse statuses, revealing solid mechanistic correlations among the phenotypes. We established a three‐dimensional single‐cell distribution map for ENKTL patients and created a standard for quantifying the change in occupational volume. Leveraging this extensive database, DLIC offers on‐site analytics in clinical settings, facilitating treatment assessment and prognosis prediction through label‐free biophysical analysis of patient PBMCs, extracted directly without additional sample preparation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c894156a2be1f7a6af52e93d9d156786919efbed" target='_blank'>
              A simple approach to biophysical profiling of blood cells in extranodal NK/T cell lymphoma patients using deep learning‐integrated image cytometry
              </a>
            </td>
          <td>
            Seongcheol Park, S. Yoon, Youngho Song, Changyu Tian, Changi Baek, Hyunji Cho, Won Seog Kim, S. Kim, Soo-Yeon Cho
          </td>
          <td>2024-11-22</td>
          <td>BMEMat</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e14091bb37f5209a2b92498963038115c6336e4" target='_blank'>
              Epigenetic dysregulation in glioblastoma: potential pathways to precision medicine.
              </a>
            </td>
          <td>
            Vijeta Prakash, R. Gabrani
          </td>
          <td>2024-11-25</td>
          <td>Neurogenetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Metastasis refers to disseminating cancerous tumors from their primary site to distant locations inside the body. Cancer cells must go through a sequence of events called the “metastatic cascade” to develop metastases. Each stage necessitates a unique functional alteration. Cancer stem cells (CSCs) play a crucial role in tumor metastasis, but understanding their dynamic behavior and regulating mechanisms remains incomplete. This review explores the influence of liver CSCs on the biological processes that drive the spread and growth of cancer cells, as described by the “metastatic cascade” concept. Liver CSCs can spread to other organs by undergoing epithelial-mesenchymal transition (EMT). This alteration in the microenvironment facilitates cellular dissemination, immune surveillance evasion, dormancy induction, and subsequent reactivation. To effectively prevent and treat advanced hepatocellular carcinoma (HCC) metastases, it is crucial to understand the heterogeneity and features of liver CSCs involved in these processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc1dca4d49250af480b27fe8e674dc43a148d98" target='_blank'>
              Exploring the Role of Liver Cancer Stem Cells in Hepatocellular Carcinoma Metastasis
              </a>
            </td>
          <td>
            Maria Fatima, Maria Riaz, Muhammad Amjad, Muhammad Waqas, Muhammad Hashim Raza
          </td>
          <td>2024-11-22</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c8d295d83a8405bd4e51ee59b5847092a62eb6b" target='_blank'>
              snRNA-seq of human ovaries reveals heat shock proteins are associated with obesity related cancer risk
              </a>
            </td>
          <td>
            Yuhan Zhang, Yonghui Jiang, Yunhai Yu, Gengchen Feng, Zihe Zhao, Weihan Zhang, Shumin Li, Yimeng Li, Ziyi Yang, Xueqi Yan, Xueying Gao, Zi-Jiang Chen, Han Zhao, Shigang Zhao
          </td>
          <td>2024-11-26</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and aggressively progressing malignancy that presents considerable clinical challenges.Although chemotherapy can effectively manage symptoms during the earlystages of SCCE, its long-term effectiveness is notably limited, with theunderlying mechanisms remaining largely undefined. In this study, weemployed single-cell RNA sequencing (scRNA-seq) to analyze SCCE samplesfrom a single patient both before and after chemotherapy treatment. Our analysisrevealed significant cellular plasticity and alterations in the tumormicroenvironment’s cellular composition. Notably, we observed an increase intumor cell diversity coupled with reductions in T cells, B cells, and myeloid-likecells. The pre-treatment samples predominantly featured carcinoma cells in amiddle transitional state, while post-treatment samples exhibited an expandedpresence of cells in terminal, initial-to-terminal (IniTerm), and universally alteredstates. Further analysis highlighted dynamic interactions between tumor cells andimmune cells, with significant changes detected in key signaling pathways, suchas TIGIT-PVR and MDK-SDC4. This study elucidates the complex dynamics of cellplasticity in SCCE following chemotherapy, providing new insights and identifyingpotential therapeutic targets to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae84357d27c8ef8315335bde0ededde5ffcf7d8" target='_blank'>
              Single-cell RNA sequencing elucidates cellular plasticity in esophageal small cell carcinoma following chemotherapy treatment
              </a>
            </td>
          <td>
            Qinkai Zhang, Ziyu Gao, Ru Qiu, Jizhao Cao, Chunxiao Zhang, Wei Qin, Meiling Yang, Xinyue Wang, Ciqiu Yang, Jie Li, Dongyang Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 The heterogeneity of ovarian mesenchymal/stromal cells has just been revealed in both mice and humans. However, it remains unclear about the cellular development trace and the intercellular communication network in the whole life of the ovary. In the study, we integrated ours and published single-cell RNA sequencing data from E11.5 (embryonic day 11.5) until M12 (12-month-old) ovaries to show the dynamics of somatic cells along the developmental timeline. The intercellular crosstalk among somatic cell types was depicted with collagen signaling pathway as the most outgoing signals from stromal cells. We identified mesenchymal progenitor cells (CD24+) as the origin of stromal cells. Although their numbers decreased significantly in adults, the cells served as the major signal sender until ovarian senescence. Moreover, the ovarian injury could activate these stem cells and induce stroma remodeling in the aged ovary. Thus, mesenchymal progenitor cells may represent a new strategy to delay ovarian aging in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/738761dd66266e63f55d4f08fe8f7233527bdb3d" target='_blank'>
              Single-cell RNA sequencing reveals the intercellular crosstalk and the regulatory landscape of stromal cells during the whole life of the mouse ovary
              </a>
            </td>
          <td>
            Wan Jiang, Wenya Sun, Yue Peng, Hao Xu, Haonan Fan, Xin Jin, Yue Xiao, Yuxiang Wang, Pin Yang, Wenjie Shu, Jing Li
          </td>
          <td>2024-12-01</td>
          <td>Life Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background
Sarcomatoid hepatocellular carcinoma (HCC) is a rare histological subtype of HCC characterized by extremely poor prognosis; however, its molecular characterization has not been elucidated.


Methods
In this study, we conducted an integrated multiomics study of whole-exome sequencing, RNA-seq, spatial transcriptome, and immunohistochemical analyses of 28 paired sarcomatoid tumor components and conventional HCC components from 10 patients with sarcomatoid HCC, in order to identify frequently altered genes, infer the tumor subclonal architectures, track the genomic evolution, and delineate the transcriptional characteristics of sarcomatoid HCCs.


Results
Our results showed that the sarcomatoid HCCs had poor prognosis. The sarcomatoid tumor components and the conventional HCC components were derived from common ancestors, mostly accessing similar mutational processes. Clonal phylogenies demonstrated branched tumor evolution during sarcomatoid HCC development and progression. TP53 mutation commonly occurred at tumor initiation, whereas ARID2 mutation often occurred later. Transcriptome analyses revealed the epithelial-mesenchymal transition (EMT) and hypoxic phenotype in sarcomatoid tumor components, which were confirmed by immunohistochemical staining. Moreover, we identified ARID2 mutations in 70% (7/10) of patients with sarcomatoid HCC but only 1-5% of patients with non-sarcomatoid HCC. Biofunctional investigations revealed that inactivating mutation of ARID2 contributes to HCC growth and metastasis and induces EMT in a hypoxic microenvironment.


Conclusions
We offer a comprehensive description of the molecular basis for sarcomatoid HCC, and identify genomic alteration (ARID2 mutation) together with the tumor microenvironment (hypoxic microenvironment), that may contribute to the formation of the sarcomatoid tumor component through EMT, leading to sarcomatoid HCC development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/043615905cf4026d2c0bac526bd7ad774bf375ad" target='_blank'>
              Integrated Molecular Characterization of Sarcomatoid Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Rongrong Sun, Yu-Hang Ye, Ye Xu, Bo Wang, Si-yuan Pan, Ning Li, Long Chen, Jing-Yue Pan, Zhi-qiang Hu, Jia Fan, Zheng-jun Zhou, Jian Zhou, Chengli Song, Shaolai Zhou
          </td>
          <td>2024-12-10</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics (ST), a breakthrough technology, captures the complex structure and state of tissues through the spatial profiling of gene expression. A variety of ST technologies have now emerged, most prominently spot-based platforms such as Visium. Despite the widespread use of ST and its distinct data characteristics, the vast majority of studies continue to analyze ST data using algorithms originally designed for older technologies such as single-cell (SC) and bulk RNA-seq—particularly when identifying differentially expressed genes (DEGs). However, it remains unclear whether these algorithms are still valid or appropriate for ST data. Therefore, here, we sought to characterize the performance of these methods by constructing an in silico simulator of ST data with a controllable and known DEG ground truth. Surprisingly, our findings reveal little variation in the performance of classic DEG algorithms—all of which fail to accurately recapture known DEGs to significant levels. We further demonstrate that cellular heterogeneity within spots is a primary cause of this poor performance and propose a simple gene-selection scheme, based on prior knowledge of cell-type specificity, to overcome this. Notably, our approach outperforms existing data-driven methods designed specifically for ST data and offers improved DEG recovery and reliability rates. In summary, our work details a conceptual framework that can be used upstream, agnostically, of any DEG algorithm to improve the accuracy of ST analysis and any downstream findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4a404e375213c00046713f7a25aee3d20013003" target='_blank'>
              Cell-specific priors rescue differential gene expression in spatial spot-based technologies
              </a>
            </td>
          <td>
            Ornit Nahman, Timothy J Few-Cooper, S. Shen-Orr
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Osteosarcoma is the most common primary malignant bone tumor, predominantly affecting children and adolescents. Despite advances in conventional therapies like chemotherapy and surgical resection, the survival rate has remained stagnant due to therapeutic resistance and high rates of metastasis. The tumor microenvironment (TME), a complex network of cellular and non-cellular components, plays a pivotal role in osteosarcoma progression, metastasis, and treatment resistance. Understanding the dynamics within the TME is crucial for developing novel therapeutic strategies that can overcome these challenges. This review explores the key elements of the osteosarcoma TME, including immune cells, endothelial cells, cancer-associated fibroblasts (CAFs), and extracellular matrix (ECM). It examines the roles of cytokines, growth factors, and exosomes secreted by osteosarcoma cells in modifying the TME to foster tumor growth, evade immune surveillance, and promote angiogenesis. Furthermore, the review critically assesses current therapeutic approaches that target TME components, with a focus on disrupting the interactions between the tumor and its microenvironment. This analysis includes a review of clinical trials and preclinical studies evaluating TME-targeting therapies. The findings highlight that the TME actively contributes to osteosarcoma progression by promoting immunosuppression, angiogenesis, and metastasis, while also enhancing resistance to standard treatments such as chemotherapy and immune checkpoint inhibitors. Various therapeutic strategies targeting the TME, such as inhibiting angiogenesis, modulating immune responses, and disrupting CAF and ECM interactions, have shown promise in preclinical models. However, clinical outcomes remain variable, underscoring the complexity of the TME and the need for more comprehensive approaches. Targeting the TME represents a promising pathway to overcome therapeutic resistance in osteosarcoma. While significant progress has been made in understanding the role of the TME in tumor development and resistance mechanisms, further research is required to optimize TME-targeting therapies. A deeper comprehension of the intricate interactions between osteosarcoma cells and their microenvironment may lead to more effective, personalized treatments, improving clinical outcomes for patients with osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de846221faa142b04c72a554ab42e7526bfe9854" target='_blank'>
              Targeting the Tumor Microenvironment in Osteosarcoma: A Pathway to Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            Kaniga Pandi, B. Cheriyan, Rithika Sooriyaprakash, Pradhosh Sakthivel, Rajeshkumar Thirupathi
          </td>
          <td>2024-12-30</td>
          <td>Biomedical and Pharmacology Journal</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Identifying the genes capable of driving tumorigenesis in different tissues is one of the central goals of cancer genomics. Computational methods that exploit different signals of positive selection in the pattern of mutations observed in genes across tumors have been developed to this end. One such signal of positive selection is clustering of mutations in areas of the three-dimensional (3D) structure of the protein above the expectation under neutrality. Methods that exploit this signal have been hindered by the paucity in proteins with experimentally solved 3D structures covering their entire sequence. Here, we present Oncodrive3D, a computational method that by exploiting AlphaFold 2 structural models extends the identification of proteins with significant mutational 3D clusters to the entire human proteome. Oncodrive3D shows sensitivity and specificity on par with state-of-the-art cancer driver gene identification methods exploiting mutational clustering, and clearly outperforms them in computational efficiency. We demonstrate, through several examples, how significant mutational 3D clusters identified by Oncodrive3D in different known or potential cancer driver genes can reveal details about the mechanism of tumorigenesis in different cancer types and in clonal hematopoiesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/993a92b761f024500819f5e0414b063781f2b33d" target='_blank'>
              Oncodrive3D: Fast and accurate detection of structural clusters of somatic mutations under positive selection
              </a>
            </td>
          <td>
            Stefano Pellegrini, Ferran Muiños, Núria López-Bigas, A. González-Pérez
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cell and immunotherapy offer transformative potential for treating diseases like cancer and autoimmune disorders by modulating the immune system. The development of these therapies is resource-intensive, with the majority of drug candidates failing to progress beyond laboratory testing. While recent advances in machine learning have revolutionised areas such as protein engineering, applications in immunotherapy remain limited due to the scarcity of large-scale, standardised datasets and the complexity of cellular systems. In this work, we address these challenges by leveraging a high-throughput experimental platform to generate data suitable for fine-tuning protein language models. We demonstrate how models fine-tuned using a preference task show surprising correlations to biological assays, and how they can be leveraged for few-shot hit maturation in CARs. This proof-of-concept presents a novel pathway for applying ML to immunotherapy and could generalise to other therapeutic modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5caa57d1ac72291cb875562fc7f15a7e0da1704e" target='_blank'>
              Harnessing Preference Optimisation in Protein LMs for Hit Maturation in Cell Therapy
              </a>
            </td>
          <td>
            Katarzyna Janocha, Annabel Ling, Alice Godson, Yulia Lampi, Simon Bornschein, Nils Y. Hammerla
          </td>
          <td>2024-12-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="BACKGROUND
The majority of patients with prostate cancer (PCa) exhibit intrinsic resistance to immune checkpoint blockade (ICB) following radiotherapy (RT). This resistance is generally attributed to the limited antigen presentation of heterogeneous cells within tumors. Here, we aimed to isolate and characterize these diverse subgroups of tumor post-RT to understand the molecular mechanisms of their resistance to ICB.


METHODS
Single-cell RNA-sequencing (scRNA-seq) was used to profile senescent cancer cell clusters induced by RT in LNCaP cells. The expression and phosphorylation levels of ataxia telangiectasia and Rad3-related protein (ATR) were assessed by immunohistochemistry in clinical samples from patients with or without RT. Co-immunoprecipitation, mutagenesis, and Western blotting were used to measure the interactions between proteins. Xenograft experiments were performed to assess the tumor immune response in the mice.


RESULTS
We identified a subset of PCa cells that exhibited resistance to RT, characterized by a reduced antigen presentation capability, which enhanced their ability to evade immune detection and resist cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) blockade. scRNA-seq revealed that the senescent state was a transient phase of PCa cells post-RT, particularly in CTLA-4 blockade treatment-resistant cells. This state was marked by increased cytosolic ATR level. Cytosolic ATR phosphorylated CD86 in its cytosolic domain and enhanced the interaction between CD86 and its E3 ligase MARCH1 through electrostatic attraction. Depletion or inhibition of Atr increased the sensitivity to immune attack and improved responses to anti-Ctla-4 antibody treatment in a mouse model.


CONCLUSIONS
Our findings indicate that the activation of cytosolic ATR, which is associated with cellular senescence, impedes the effectiveness of combined RT and ICB treatments. This discovery may provide valuable insights for improving the efficacy of combined RT and ICB therapies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54111804377a9c0bbc1679bc34e9106d5b8ef9e5" target='_blank'>
              Radiotherapy-resistant prostate cancer cells escape immune checkpoint blockade through the senescence-related ataxia telangiectasia and Rad3-related protein.
              </a>
            </td>
          <td>
            Chenyi Shao, Yingyi Zhang, Hang Li, Jiajia Chen, Ting Huang, Jiaze Li, Simeng Wen, Sen Wang, Saijun Fan, Yu Zhao
          </td>
          <td>2024-12-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="BCR::ABL1 drives chronic myeloid leukemia (CML) disease and treatment, as revealed by the success of tyrosine kinase inhibitor (TKI) therapy. However, additional poorly characterized molecular pathways, acting as BCR::ABL1 independent mechanisms, play crucial roles in CML, contributing to leukemic stem cells (LSCs) persistence, TKI resistance and disease progression. Here, by combining high sensitive mass spectrometry (MS)-based phosphoproteomics with the SignalingProfiler pipeline, we obtained two signaling maps offering a comprehensive description of the BCR::ABL1 dependent and independent pro-survival signalling mechanisms. We leveraged these maps to unbiasedly and systematically discover therapeutic vulnerabilities, by implementing the Druggability Score computational algorithm. By this strategy, and in combination with in vitro and in ex vivo functional assays, we show a crucial role of acquired FLT3-dependency in resistant CML models. In conclusion, we reposition FLT3, one of the most frequently mutated drivers of acute leukemia, as a potential therapeutic target for TKI resistant CML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e945b75e65b40ef8d68e265cfc17bc10434dceea" target='_blank'>
              Phosphoproteomics reveals novel BCR::ABL1-independent mechanisms of resistance in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Valeria Bica, Veronica Venafra, Giorgia Massacci, Simone Graziosi, Sara Gualdi, G. Minnella, Federica Sorà, P. Chiusolo, Maria Elsa Brunetti, G. Napolitano, Massimo Breccia, Dimitrios Mougiakakos, Martin Böettcher, Thomas Fischer, Livia Perfetto, F. Sacco
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Cancer stem cells (CSCs) constitute a small and elusive subpopulation of cancer cells within a tumor mass and are characterized by stem cell properties. Reprogrammed CSCs exhibit similar capability to initiate tumor growth, metastasis, and chemo- and radio-resistance and have similar gene profiles to primary CSCs. However, the efficiency of cancer cell reprogramming remained relatively low. There is limited literature available on the reprogramming of lung cancer cells. Hence, in this study we have conducted reprogramming of human lung cancer cells towards more benign type of cells. Materials and methods The reprogramming process was carried out with the use of STEMCCA vector. We have investigated the gene expression profile of induced CSCs (iCSCs) using the microarray technique. Results The lung iCSCs demonstrate morphology characteristics of induced pluripotent stem cells (iPSCs) and gene expression profile that significantly differ from cells before reprogramming. We have also presented the elevated level of expression of genes associated with the cancer stemness and thus revealed new interesting CSC-like molecular markers. Conclusions These preliminary results demonstrated that the reprogramming process in vitro leads to the remarkable changes in cells at the gene level and has potential to be an alternative method of generating CSC-like cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5e391fa7a69486ab2113454ce0ec04bcbfac417" target='_blank'>
              Can the reprogrammed cancer cells serve as an alternative source of (induced) cancer stem cells?
              </a>
            </td>
          <td>
            E. Stelcer, Małgorzata Blatkiewicz, K. Jopek, W. Suchorska, Marcin Ruciński
          </td>
          <td>2024-12-04</td>
          <td>Reports of Practical Oncology and Radiotherapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Introduction Cancer-associated fibroblasts (CAFs) are a diverse group of cells that significantly contribute to reshaping the tumor microenvironment (TME), and no research has systematically explored the molecular landscapes of senescence related CAFs (senes CAF) in NB. Methods We utilized pan-cancer single cell and spatial transcriptomics analysis to identify the subpopulation of senes CAFs via senescence related genes, exploring its spatial distribution characteristics. Harnessing the maker genes with prognostic significance, we delineated the molecular landscapes of senes CAFs in bulk-seq data. We established the senes CAFs related signature (SCRS) by amalgamating 12 and 10 distinct machine learning (ML) algorithms to precisely diagnose stage 4 NB and to predict prognosis in NB. Based on risk scores calculated by prognostic SCRS, patients were categorized into high and low risk groups according to median risk score. We conducted comprehensive analysis between two risk groups, in terms of clinical applications, immune microenvironment, somatic mutations, immunotherapy, chemotherapy and single cell level. Ultimately, we explore the biological function of the hub gene JAK1 in pan-cancer multi-omics landscape. Results Through integrated analysis of pan-cancer spatial and single-cell transcriptomics data, we identified distinct functional subgroups of CAFs and characterized their spatial distribution patterns. With marker genes of senes CAF and leave-one-out cross-validation, we selected RF algorithm to establish diagnostic SCRS, and SuperPC algorithm to develop prognostic SCRS. SCRS demonstrated a stable predictive capability, outperforming the previously published NB signatures and clinic variables. We stratified NB patients into high and low risk group, which showed the low-risk group with a superior survival outcome, an abundant immune infiltration, a different mutation landscape, and an enhanced sensitivity to immunotherapy. Single cell analysis reveals biologically cellular variations underlying model genes of SCRS. Spatial transcriptomics delineated the molecular variant expressions of hub gene JAK1 in malignant cells across cancers, while immunohistochemistry validated the differential protein levels of JAK1 in NB. Conclusion Based on multi-omics analysis and ML algorithms, we successfully developed the SCRS to enable accurate diagnosis and prognostic stratification in NB, which shed light on molecular landscapes of senes CAF and clinical utilization of SCRS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/900ec2c5a181f8ad61d932465735fe6366505e3c" target='_blank'>
              Pan-cancer single cell and spatial transcriptomics analysis deciphers the molecular landscapes of senescence related cancer-associated fibroblasts and reveals its predictive value in neuroblastoma via integrated multi-omics analysis and machine learning
              </a>
            </td>
          <td>
            Shan Li, Junyi Luo, Junhong Liu, Dawei He
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The incidence of melanoma, a highly aggressive skin cancer, continues to increase worldwide, particularly among populations with lighter skin tones. The diagnostic challenge of melanoma lies in the absence of a distinctive clinical presentation, as its characteristics vary based on anatomical location, growth type, and histopathology. The melanoma-associated antigen (MAGE) gene family is differentially expressed in various human cancers, including melanoma. In this study, we explored the association between human MAGEA2 (hMAGEA2) expression and melanoma. Using a human melanoma tissue array, we confirmed that hMAGEA2 expression was higher in melanoma and metastatic melanoma than in normal tissues. Additionally, we used SK-MEL-5 and SK-MEL-28 cell lines to investigate the cellular and molecular mechanisms underlying melanoma progression and invasiveness. In SK-MEL-5 and SK-MEL-28 cells, hMAGEA2 overexpression accelerated cell proliferation. Conversely, the knockdown of hMAEGA2 reduced cell proliferation, colony formation, and migration significantly and induced arrest at the G2/M phase of the cell cycle. With respect to the molecular mechanism, the knockdown of hMAGEA2 decreased the phosphorylation of Akt, JNK, and p38 MAPK. Additionally, hMAGEA2 knockdown reduced tumor formation significantly at the in vivo level. Collectively, the robust correlation between hMAGEA2 and melanoma metastasis supports the potential utility of hMAGEA2 as both a diagnostic marker and novel therapeutic target for patients with melanoma metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2f210e7fd413579bc370d616c2d10236a1bb97e" target='_blank'>
              hMAGEA2 as a potential diagnostic and therapeutic target for melanoma progression and metastasis.
              </a>
            </td>
          <td>
            Junkoo Yi, C. Kim, Hyeonjin Kim, Eun Gyung Kim, Do Yoon Kim, Yonghun Sung, Kang Dong Liu, Seong-Kyoon Choi, Zaeyoung Ryoo, Song Park, Myoung-Ok Kim
          </td>
          <td>2024-11-24</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e846ee374ca9a20e78989a20462c74023e1477a" target='_blank'>
              Single-cell transcriptomics reveal the prognostic roles of epithelial and T cells and DNA methylation-based prognostic models in pancreatic cancer
              </a>
            </td>
          <td>
            Jing Du, Yaqian Zhao, Jie Dong, Peng Li, Yan Hu, Hailang Fan, Feifan Zhang, Lanlan Sun, Dake Zhang, Yuhua Zhang
          </td>
          <td>2024-12-21</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Atypical teratoid rhabdoid tumor (ATRT) is the most common malignant brain tumor in infants, and more than 60% of children with ATRT die from their tumor. ATRT is associated with mutational inactivation/deletion of SMARCB1, a member of the SWI/SNF chromatin remodeling complex, suggesting that epigenetic events play a critical role in tumor development and progression. Moreover, disruption of SWI/SNF allows unopposed activity of epigenetic repressors, which contribute to tumorigenicity. We therefore explored the role of the CoREST repressor complex in ATRT. Methods We evaluated the effects of the bifunctional LSD1/HDAC1/2 small molecule CoREST inhibitor, corin, on ATRT tumor cell growth, apoptosis, differentiation, gene expression and chromatin accessibility. Results We found that corin inhibited the growth of ATRT cells regardless of their epigenetic subgroup, and was associated with increased tumor cell apoptosis and differentiation. ATAC-seq showed increases in chromatin accessibility in corin-treated ATRT cells, with changes seen at genes associated with neuronal differentiation and synaptic function. RNA-seq confirmed increased expression of neuronal differentiation genes and decreased DNA replication/cell cycle-associated genes in ATRT cells treated with corin. Corin suppressed orthotopic ATRT tumor growth, leading to significant extension of lifespan. In addition, increased histone acetylation (H3K9ac, H3K27ac) and methylation (H3K4Me1) was seen in corin-treated ATRT orthotopic xenografts, consistent with on-target pharmacodynamics. Conclusion The CoREST inhibitor, corin, suppresses tumor growth, induces differentiation, and promotes apoptosis in ATRT, leading to significantly increased survival of mice bearing ATRT orthotopic xenografts. Our results suggest a potential application of CoREST complex inhibitors in patients with ATRT. Key Points CoREST complex inhibition by corin leads to decreased cell growth and increased apoptosis in ATRT Corin promotes chromatin accessibility and neuronal differentiation in ATRT Corin inhibits tumor growth and extends lifespan in ATRT animal models Importance of the Study Loss of function of SMARCB1 is a hallmark of ATRT which leads to dysfunction of the mammalian SWI/SNF complex and an inability to counteract epigenetic repressor complexes. The CoREST complex functions as a chromatin remodeling complex that represses neuronal differentiation genes during development. Inhibition of the CoREST complex by corin in ATRT leads to decreased tumor cell growth, induction of apoptosis and increased survival of mice bearing ATRT orthotopic xenografts. These changes are associated with increased chromatin accessibility and expression of genes associated with neuronal differentiation. Corin therefore reverses the primary block of differentiation that maintains a stem cell state in ATRT, which contributes to tumorigenesis. These studies significantly improve our understanding of how to therapeutically address the underlying epigenetic drivers of ATRT and support further development of corin for ATRT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024c492e649d7b4c64d548dac81278e98253c902" target='_blank'>
              The CoREST complex inhibitor, corin, leads to decreased tumor growth, increased cellular differentiation and extended lifespan in atypical teratoid rhabdoid tumor xenograft models
              </a>
            </td>
          <td>
            Anupa Geethadevi, Nikhil Vaidya, Robert J. Fisher, Tyler Findlay, Yiming Deng, Khoa Pham, Vikas Kumar, Samuel Beck, Calixto-Hope G Lucas, C. Eberhart, Jinchong Xu, Philip A. Cole, Jeffrey A. Rubens, Marianne Collard, Eric H. Raabe, R. Alani
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2db06d69ffc4c5099d43bf0ce0c03ec5dc4f4f2" target='_blank'>
              The characteristics and functional significance of disulfidptosis-related genes in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Haiqian Zhu, Chifeng Zhao, Haoran Zhu, Xuhui Xu, Conglin Hu, Zhenxing Zhang
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20bc7f7d6da280d06eb3817a53ab1d98805043fa" target='_blank'>
              The neuroepithelial origin of ovarian carcinomas explained through an epithelial-mesenchymal-ectodermal transition enhanced by cisplatin
              </a>
            </td>
          <td>
            D. Díaz-Carballo, Ayesha Safoor, Sahitya Saka, Adrien Noa-Bolaño, Flevy D’Souza, Jacqueline Klein, A. Acikelli, S. Malak, Udo Rahner, Amin T Turki, Anne Höppner, Annabelle Kamitz, Wanlu Song, Ye-Guang Chen, Lalitha Kamada, Andrea Tannapfel, Sebastian Brinkmann, Crista Ochsenfarth, Dirk Strumberg
          </td>
          <td>2024-11-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genome duplication, critical for cell survival and identity, requires precise origin activation to ensure accurate DNA replication and chromatin structure maintenance. In colorectal cancer cells, prolonged replication stress does not hinder cell proliferation, although the mechanisms driving cancer cell adaptability remain largely unclear. Here, we demonstrate that upon recovery, cancer cells are able to activate new replication origins in distinct domains, causing persistent changes in chromatin accessibility, nuclear morphology, and replication timing, ultimately promoting chromatin instability. Chromatin accessibility, particularly in promoter regions, increased following replication stress in a subset of cells, correlating with altered gene expression and nuclear expansion. Additionally, some genes with enhanced promoter accessibility displayed sustained expression changes, further suggesting a transcriptional shift linked to stress adaptation. Our findings reveal that colorectal cancer cells recover from severe replication stress through new origin activation, a mechanism that not only maintains cell proliferation under stress but may also accelerate tumour heterogeneity. This research underscores replication origin activation as a potential therapeutic target in combating cancer cell resilience. GRAPHICAL ABSTRACT MAIN HIGHLIGHTS Colorectal cancer cells under severe replication stress activate new replication origins upon recovery. The firing of new origins in distinct replication domains results in long-lasting changes in chromatin accessibility, especially in promoter regions, and is associated with alterations in nuclear morphology and replication timing. Increased chromatin accessibility in certain promoter regions correlates with prolonged gene expression changes. The capacity of CRC cells to recover from severe replication stress fosters chromatin instability and tumour heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04cc1950d705dd8a02a8a5d1f21cfa69a97d0c9c" target='_blank'>
              Adaptive replication origin activation alters chromatin dynamics and stability in cancer cells
              </a>
            </td>
          <td>
            Alejandro Pérez-Venteo, Fernando Unzueta, Sonia Feu, P. Llinàs-Arias, Caroline Mauvezin, Neus Agell
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/860ac194d88b87a4300486f4bf6c9542be5846a7" target='_blank'>
              Proteomic profiling reveals the significance of lipid metabolism in small cell lung cancer recurrence and metastasis
              </a>
            </td>
          <td>
            Haohua Zhu, Huiyang Shi, Jingyu Lu, Kai Zhu, Lin Yang, Lei Guo, Le Tang, Yuankai Shi, Xingsheng Hu
          </td>
          <td>2024-12-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioblastoma is a major unmet clinical need characterized by striking inter- and intra-tumoral heterogeneity and a population of glioblastoma stem cells (GSCs), conferring aggressiveness and therapy resistance. GSCs communicate through a network of tumor-tumor connections (TTCs), including nanotubes and microtubes, promoting tumor progression. However, very little is known about the mechanisms underlying TTC formation and overall GSC morphology. As GSCs closely resemble neural progenitor cells during neurodevelopment, we hypothesized that GSCs' morphological features affect tumor progression. We identified GSC morphology as a new layer of tumoral heterogeneity with important consequences on GSC proliferation. Strikingly, we showed that the neurodevelopmental morphoregulator ADD3 is sufficient and necessary for maintaining proper GSC morphology, TTC abundance, cell cycle progression, and chemoresistance, as well as required for cell survival. Remarkably, both the effects on cell morphology and proliferation depend on the stability of actin cytoskeleton. Hence, cell morphology and its regulators play a key role in tumor progression by mediating cell-cell communication. We thus propose that GSC morphological heterogeneity holds the potential to identify new therapeutic targets and diagnostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03788716ebc47d74596235fca4e0d4e1599b9c88" target='_blank'>
              Morphoregulatory ADD3 underlies glioblastoma growth and formation of tumor-tumor connections.
              </a>
            </td>
          <td>
            C. Barelli, F. Kaluthantrige Don, R. Iannuzzi, Stefania Faletti, I. Bertani, I. Osei, Simona Sorrentino, G. Villa, Viktoria Sokolova, Alberto Campione, Matteo R Minotti, G. Sicuri, Roberto Stefini, Francesco Iorio, N. Kalebic
          </td>
          <td>2024-11-26</td>
          <td>Life science alliance</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Canine cutaneous histiocytoma (CCH) is a tumor that originates from dermal Langerhans cells and affects particularly young dogs. The common spontaneous regression of CCH makes it an interesting model in comparative oncology research. Previous studies have indicated that anti-tumor immune responses may be involved, but details remain speculative to date. Here, we asked which specific immuno-oncological dynamics underlie spontaneous regression of CCH on mRNA and protein levels. QuantSeq 3′ mRNA sequencing with functional over-representation analysis and an nCounter RNA hybridization assay were employed on 21 formalin-fixed, paraffin-embedded CCH samples representing three different tumor stages (dataset information: GSE261387—Immuno-Oncologic Profiling by Stage-Dependent Transcriptome and Proteome Analyses of Spontaneously Regressing Canine Cutaneous Histiocytoma—OmicsDI). Nine additional samples were subjected to matrix-assisted laser desorption/ionization mass spectrometry imaging (MALDI-MSI). Surprisingly, only minor stage-specific differences were found. When we investigated expression of B7 family ligands and CD28 family receptors holding co-stimulatory and -inhibitory functions, respectively, we found a higher abundance of CD80, CD86, CTLA4 and CD28, which may trigger a balanced activation of lymphocyte-mediated immune responses. CD80 and CD86 expressing cells were further quantified by in situ hybridization and compared with data from three cases of canine histiocytic sarcoma (HS), a malignant tumor variant originating from antigen-presenting interstitial dendritic cells. A stage-specific increase of CD80 expressing cells was recorded in CCH from the tumor bottom to the top, while CD86 was continuously and homogenously expressed at high levels. Overall expression of CD80 in CCH was similar to that in HS (73.3 ± 37.4% vs 62.1 ± 46.4%), while significantly more CD86 expressing tumor cells were found in CCH (94.7 ± 10.3%) when compared to HS (57.6 ± 11.0%). Our data suggest that major immuno-oncological pathways are not regulated during regression of CCH on the mRNA or protein levels as detectable by the methods used. Instead, our data provide further evidence supporting previous hypotheses towards a role of immune stimulatory B7 family ligands and CD28 family receptors in the regression of CCH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db963ce30eb220ecdb0ce2dc04f12705a49bf7b8" target='_blank'>
              Immuno-oncologic profiling by stage-dependent transcriptome and proteome analyses of spontaneously regressing canine cutaneous histiocytoma
              </a>
            </td>
          <td>
            Alina K. Loriani Fard, Alexander Haake, Vladimir M. Jovanovic, S. Andreotti, Michael Hummel, Benjamin-Florian Hempel, Achim D. Gruber
          </td>
          <td>2024-11-26</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
Alcohol-associated hepatitis (AH) is the clinical manifestation of alcohol-associated liver disease (ALD). AH is a complex disease encompassing the dysregulation of many cells and cell subpopulations. This study used a hepatic spatial transcriptomic and proteomic approach (10X Genomics Visium) to identify hepatic cell populations and their associated transcriptomic and proteomic alterations in human AH.


METHODS
Formalin-fixed paraffin-embedded liver tissue from AH patients (n = 2) and non-ALD controls (donors) (n = 2) were used for Visium spatial transcriptomic and proteomic analysis.


RESULTS
AH cell clusters and cell markers were drastically different in regard to tissue pattern and number of cell types compared to non-ALD controls. Cholangiocytes, endothelial cells, macrophages, and stellate cells were more profuse in AH relative to non-ALD controls. Transcriptionally, proliferating cell nuclear antigen-positive (PCNA+) hepatocytes in AH more closely resembled cholangiocytes suggesting they were non-functional hepatocytes derived from cholangiocytes. Furthermore, mitochondria protein-coding genes were reduced in AH versus non-ALD control hepatocytes, suggesting reduced functionality and loss of regenerative mechanisms. Macrophages in AH exhibited elevated gene expression involved in exosomes as compared to non-ALD controls. The most upregulated macrophage genes observed in AH were those involved in exosome trafficking. Gene and protein signatures of disease-associated hepatocytes (ANXA2+/CXCL1+/CEACAM8+) were elevated in AH and could visually identify a pre-malignant lesion.


CONCLUSIONS
This study identified global cell type alterations in AH and distinct transcriptomic changes between AH and non-ALD controls. These findings characterize cellular plasticity and profuse transcriptomic and proteomic changes that are apparent in AH and contribute to the identification of novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2049f287b107fb48e735d44a0a77474e5846e8b" target='_blank'>
              Visium spatial transcriptomics and proteomics identifies novel hepatic cell populations and transcriptomic signatures of alcohol-associated hepatitis.
              </a>
            </td>
          <td>
            Tyler C. Gripshover, Rui S Treves, Eric C Rouchka, J. Chariker, Shirong Zheng, Elizabeth Hudson, Melissa L Smith, A. Singal, Craig J. McClain, Josiah E. Hardesty
          </td>
          <td>2024-11-26</td>
          <td>Alcohol, clinical & experimental research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most prevalent primary malignant brain tumor, remains challenging to treat due to extensive inter- and intra-tumor heterogeneity. This variability demands combination treatments to improve therapeutic outcomes. A significant obstacle in treating GBM is the expression of O6-methylguanine-DNA methyltransferase, a DNA repair enzyme that reduces the efficacy of the standard alkylating agent, temozolomide, in about 50% of patients. This underscores the need for novel, more targeted therapies. Our study investigates the metabolic–epigenetic impact of combining SN-38, a novel topoisomerase inhibitor inducing DNA double-strand breaks, with rabusertib, a checkpoint kinase 1 inhibitor. We identified this synergistic combination through high-throughput drug screening across a panel of GBM cell lines using a cancer drug library combined with SN-38. A secondary metabolic screening with the PEDS algorithm demonstrated a synergistic modulation of purine, one-carbon, and redox metabolism. Furthermore, the combined treatment led to the significant depletion of epigenetically relevant metabolites such as 5-methyl-cytosine, acetyl-lysine, and trimethyl-lysine. Reduced intermediates of the glutathione cycle indicated increased cellular stress following combinatorial treatment. Overall, the combination of SN-38 and rabusertib synergistically disrupts metabolites associated with epigenetic adaptations, leading to cytotoxicity independent of O6-methylguanine-DNA methyltransferase status, thereby underpinning this combination as a promising candidate for combinatorial therapy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed32abd0bdaa8ba2724fd42a8df45084943d6e1" target='_blank'>
              Targeting Metabolic and Epigenetic Vulnerabilities in Glioblastoma with SN-38 and Rabusertib Combination Therapy
              </a>
            </td>
          <td>
            J. Chiou, Valeria Impedovo, Yen Bao Huynh, Ruggiero Gorgoglione, Luiz O. F. Penalva, A. Lodi, Andrew J. Brenner, Stefano Tiziani
          </td>
          <td>2025-01-08</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cell phenotype underlies prostate cancer presentation and treatment resistance and can be regulated by epigenomic features. However, the osteotropic tendency of prostate cancer limits access to metastatic tissue, meaning most prior insights into prostate cancer chromatin biology are from preclinical models that do not fully represent disease complexity. Noninvasive chromatin immunoprecipitation of histones in plasma cell-free in humans may enable capture of disparate prostate cancer phenotypes. Here, we analyzed activating promoter- and enhancer-associated H3K4me2 from cfDNA in metastatic prostate cancer enriched for divergent patterns of metastasis and diverse clinical presentation. H3K4me2 density across prostate cancer genes, accessible chromatin, and lineage-defining transcription factor binding sites correlated strongly with circulating tumor DNA (ctDNA) fraction-demonstrating capture of prostate cancer-specific biology and informing the development of a statistical framework to adjust for ctDNA fraction. Chromatin hallmarks mirrored synchronously measured clinico-genomic features: bone versus liver-predominant disease, serum PSA, biopsy-confirmed histopathological subtype, and RB1 deletions convergently indicated phenotype segregation along an axis of differential androgen receptor activity and neuroendocrine identity. Detection of lineage switching after sequential progression on systemic therapy in select patients indicates potential utility for individualized resistance monitoring. Epigenomic footprints of metastasis-induced normal tissue destruction were evident in bulk cfDNA from two patients. Finally, a public epigenomic resource was generated using a distinct chromatin marker that has not been widely investigated in prostate cancer. These results provide insight into the adaptive molecular landscape of aggressive prostate cancer and endorse plasma cfDNA chromatin profiling as a biomarker source and biological discovery tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15869a6834cce04dbe66dca3a25682ba07c367af" target='_blank'>
              Plasma cell-free DNA chromatin immunoprecipitation profiling depicts phenotypic and clinical heterogeneity in advanced prostate cancer.
              </a>
            </td>
          <td>
            J. Sipola, Aslı D Munzur, Edmond M Kwan, Clara C Y Seo, Benjamin J Hauk, Karan Parekh, Yi Jou Ruby Liao, C. Bernales, G. Donnellan, Ingrid Bloise, Emily Fung, Sarah W. S. Ng, Gang Wang, G. Vandekerkhove, M. Nykter, M. Annala, C. Maurice-Dror, Kim Chi, C. Herberts, Alexander W Wyatt, David Y. Takeda
          </td>
          <td>2024-12-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The advent of single-cell transcriptomics has revolutionized our ability to analyze cellular heterogeneity and dynamics at a fine resolution, yet covering the vast array of potential perturbations remains challenging due to biological variability. To address this, we propose scCADE, a novel computational approach utilizing contrastive learning and an attention mechanism to decouple gene expression signatures and predict cellular responses to perturbations. scCADE excels in predicting responses in cells to perturbations observed in other cells but not yet seen in the target cells. Through rigorous ablation studies and validation across three datasets involving drug and gene editing perturbations, scCADE consistently outperformed existing methods, underscoring its efficacy and potential to advance genomics and personalized medicine by accurately forecasting responses to novel perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d36ceda28edb31805eb4f9032509797809b3ca7" target='_blank'>
              scCADE: A Superior Tool for Predicting Perturbation Responses in Single-Cell Gene Expression Using Contrastive Learning and Attention Mechanisms
              </a>
            </td>
          <td>
            Jingfeng Ou, Jiawei Li, Zhiliang Xia, Shurui Dai, Yulian Ding, Yan Guo, Limin Jiang, Jijun Tang
          </td>
          <td>2024-12-03</td>
          <td>2024 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer cell lines are the most common in-vitro models for the evaluation of anti-cancer drug sensitivities. Past studies have been conducted to decipher and characterize the pharmacogenomic feature of cell lines based on other omics data, such as genomic mutation data and whole-genome RNA sequencing (RNA-seq) profiles. In particular, proteomic data is also an essential component for the characterization of tumours. However, different from RNA-seq datasets rich in numerous transcriptome profiles of cancer cell lines and cell viability assay of drug responses, the pharmacogenomic protein quantifications are relatively scarce. With the availability of the recently enriched proteomic dataset ProCan-DepMapSanger, we systematically evaluated the interplays among genomic mutations, transcription, and protein expressions across cancer cell lines. In general, blood cancers have higher RNA-protein correlations than those in solid cancers. The differential expression analysis on protein data helped identify more expressional and functional impact of genomic mutations of cancer genes. We also integrated the proteomic map with drug molecular chemical features to construct a bi-modal machine learning model to infer the drug sensitivities of cancer cell lines. Our results demonstrated that protein quantifications can lead to better drug response prediction performance than the model trained on transcriptome profiles. In addition, integrating protein data with drug chemical features, represented as molecular graphs and learned by Graph Neural Network, outperformed the state-of-the-art model DeepOmicNet for drug response prediction in proteomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89e8ade30b207677f590882fddb023a8cd320d3" target='_blank'>
              Drug Response Modeling across Cancers: Proteomics vs. Transcriptomics
              </a>
            </td>
          <td>
            Zetian Zheng, Lei Huang, Fuzhou Wang, Linjing Liu, Jixiang Yu, Weidun Xie, Xingjian Chen, Xiangtao Li, Ka-Chun Wong
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff4ae0eaa9128342a0dedd747e19622d92c7f71" target='_blank'>
              FDFT1 maintains glioblastoma stem cells through activation of the Akt pathway
              </a>
            </td>
          <td>
            Hui Mo, Jiajia Shao, Zhun Li, Peiting Zeng, Xinke Yin, Yongsheng Huang, Peng Wang, Jianwei Liao
          </td>
          <td>2024-12-20</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, Martin Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Over the course of the last decade, genomic studies in the context of normal human hematopoiesis have provided new insights into the early pathogenesis of myeloproliferative neoplasms (MPN). A preclinical phase of MPN, termed clonal hematopoiesis (CH) was identified and subsequent lineage tracing studies revealed a multi-decade long time interval from acquisition of an MPN phenotypic driver mutation in a hematopoietic stem cell (HSC) to the development of overt MPN. Multiple germline variants associated with MPN risk have been identified through genome-wide association studies (GWAS) and in some cases functional interrogation of the impact of the variant has uncovered new insights into HSC biology and MPN development. Increasingly sophisticated methods to study clonal contributions to human hematopoiesis and measure HSC fitness have helped discern the biology underlying the tremendous clinical heterogeneity observed in MPN. Despite these advances, significant knowledge gaps remain particularly with respect to germline genetic contributors to both MPN pathogenesis and phenotypic diversity, as well as limitations in the ability to prospectively quantify rates of clonal expansion in individual MPN patients. Ultimately, we envisage a personalized approach to MPN care in the future, where an individualized genetic assessment can predict MPN trajectory and this information will be used to inform and guide therapy. MPN is particularly amenable to precision medicine strategies and our increased understanding of the evolution of MPN from normal blood stem cells provides a unique opportunity for early therapeutic intervention approaches and potentially MPN prevention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd917b44573f25d09202b5ca694e87a98f431b63" target='_blank'>
              Evolution of myeloproliferative neoplasms from normal blood stem cells.
              </a>
            </td>
          <td>
            S. Hormoz, Vijay G Sankaran, Ann Mullally
          </td>
          <td>2024-12-05</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancers exhibit a remarkable ability to develop resistance to a range of treatments, often resulting in relapse following first-line therapies and significantly worse outcomes for subsequent treatments. While our understanding of the mechanisms and dynamics of the emergence of resistance during cancer therapy continues to advance, questions remain about how to minimize the probability that resistance will evolve, thereby improving long-term patient outcomes. Here, we present an evolutionary simulation model of a clonal population of cells that can acquire resistance mutations to one or more treatments. We leverage this model to examine the efficacy of a two-strike “extinction therapy” protocol, in which two treatments are applied sequentially to first contract the population to a vulnerable state and then push it to extinction, and compare it to a combination therapy protocol. We investigate how factors such as the timing of the switch between the two strikes, the rate of emergence of resistant mutations, the doses of the applied drugs, the presence of cross-resistance, and whether resistance is a binary or a quantitative trait affect the outcome. Our results show that the timing of switching to the second strike has a marked effect on the likelihood of driving the cancer to extinction, and that extinction therapy outperforms combination therapy when cross-resistance is present. We conduct an in silico trial that reveals when and why a second strike will succeed or fail. Finally, we demonstrate that our conclusions hold whether we model resistance as a binary trait or as a quantitative, multi-locus trait.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9eae66855ec0b28076c2f378c35dec6bc945dff" target='_blank'>
              Evolutionary rescue model informs strategies for driving cancer cell populations to extinction
              </a>
            </td>
          <td>
            Amjad Dabi, Joel S. Brown, Robert A Gatenby, Corbin D. Jones, Daniel R. Schrider
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fc92ee9eeeb11fdc0010923fa936390ffdc668a" target='_blank'>
              Multi-omic and single-cell profiling of chromothriptic medulloblastoma reveals genomic and transcriptomic consequences of genome instability
              </a>
            </td>
          <td>
            Petr Smirnov, M. Przybilla, Milena Simovic-Lorenz, R. G. Parra, H. Susak, Manasi Ratnaparkhe, John K. L. Wong, V. Körber, Jan-Philipp Mallm, George Philippos, M. Sill, Thorsten Kolb, Rithu Kumar, Nicola Casiraghi, K. Okonechnikov, David R. Ghasemi, K. Maass, K. Pajtler, Anna Jauch, A. Korshunov, Thomas Höfer, M. Zapatka, Stefan M Pfister, W. Huber, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-23</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>109</td>
        </tr>

        <tr id="Odorant receptors (ORs), which constitute approximately 50% of all human G protein-coupled receptors, are increasingly recognized for their diverse roles beyond odor perception, including functions in various pathological conditions like brain diseases and cancers. However, the roles of ORs in glioblastoma (GBM), the most aggressive primary brain tumor with a median survival of only 15 months, remain largely unexplored. Here, we performed an integrated transcriptomic analysis combining The Cancer Genome Atlas RNA-seq and single-cell RNA sequencing data from GBM patients to uncover cell-type-specific roles of ORs within the tumor and its microenvironment. Our findings reveal that ORs display distinct expression patterns, with OR51E1 enriched in pericytes linked to vascular remodeling and angiogenesis, OR2B11 associated with tumor-associated macrophages supporting immunosuppressive phenotypes, and OR2L13 correlated with synaptic activity in recurrent tumors, potentially mediating treatment-induced neuronal adaptations. These results highlight ORs as potential therapeutic targets, offering new insights into their regulatory roles in GBM progression, immune modulation, and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0d78b05c85d74f15c35ba8438bef22107b35ac7" target='_blank'>
              Comprehensive Transcriptomic Analysis Reveals Cell-Type-Specific Roles of Human Odorant Receptors in Glioblastoma and the Tumor Microenvironment
              </a>
            </td>
          <td>
            Hee Jin Cho, Dong Jun Yeo, HeeWoong Yang, JaeHyung Koo
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b0dd1e192f0dce474e0a51a9c8fd0df15d21667" target='_blank'>
              Specialized pericyte subtypes in the pulmonary capillaries.
              </a>
            </td>
          <td>
            T. Klouda, Yunhye Kim, S. Baek, Mantu Bhaumik, Yan Li, Yu Liu, Joseph C Wu, Benjamin A Raby, V. D. J. Perez, Ke Yuan
          </td>
          <td>2025-01-13</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Motivation Many tumours show deficiencies in DNA damage response (DDR), which influence tumorigenesis and progression, but also expose vulnerabilities with therapeutic potential. Assessing which patients might benefit from DDR-targeting therapy requires knowledge of tumour DDR deficiency status, with mutational signatures reportedly better predictors than loss of function mutations in select genes. However, signatures are identified independently using unsupervised learning, and therefore not optimised to distinguish between different pathway or gene deficiencies. Results We propose SNMF, a supervised non-negative matrix factorisation that jointly optimises the learning of signatures: (1) shared across samples, and (2) predictive of DDR deficiency. We applied SNMF to mutation profiles of human induced pluripotent cell lines carrying gene knockouts linked to three DDR pathways. The SNMF model achieved high accuracy (0.971) and learned more complete signatures of the DDR status of a sample, further discerning distinct mechanisms within a pathway. Cell line SNMF signatures recapitulated tumour-derived COSMIC signatures and predicted DDR pathway deficiency of TCGA tumours with high recall, suggesting that SNMF-like models can leverage libraries of induced DDR deficiencies to decipher intricate DDR signatures underlying patient tumours. Availability https://github.com/joanagoncalveslab/SNMF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afb1aa490661811f25b1b0555092a4330771d6f8" target='_blank'>
              SNMF: Integrated Learning of Mutational Signatures and Prediction of DNA Repair Deficiencies
              </a>
            </td>
          <td>
            Sander Goossens, Y. Tepeli, Colm Seale, Joana P. Gonçalves
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f1f8dfa9377169d8a09f717c0d18ed8c6b2867" target='_blank'>
              Prospective validation of ORACLE, a clonal expression biomarker associated with survival of patients with lung adenocarcinoma.
              </a>
            </td>
          <td>
            D. Biswas, Yun-Hsin Liu, Javier Herrero, Yin Wu, David A Moore, Takahiro Karasaki, Kristiana Grigoriadis, Wei-Ting Lu, S. Veeriah, C. Naceur-Lombardelli, Neil Magno, S. Ward, A. Frankell, M. Hill, Emma C Colliver, Sophie de Carné Trécesson, Philip East, A. Malhi, Daniel M. Snell, Olga O’Neill, Daniel Leonce, J. Mattsson, A. Lindberg, P. Micke, Judit Moldvay, Z. Megyesfalvi, B. Dome, J. Fillinger, J. Nicod, Julian Downward, Zoltan Szallasi, A. Hackshaw, M. Jamal-Hanjani, N. Kanu, Nicolai J. Birkbak, C. Swanton
          </td>
          <td>2025-01-09</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Abstract The blood–brain barrier is often altered in glioblastoma (GBM) creating a blood–brain–tumor barrier (BBTB) composed of pericytes. The BBTB affects chemotherapy efficacy. However, the expression signatures of BBTB‐associated pericytes remain unclear. We aimed to identify BBTB‐associated pericytes in single‐cell RNA sequencing data of GBM using pericyte markers, a normal brain pericyte expression signature, and functional enrichment. We identified parathyroid hormone receptor‐1 (PTH1R) as a potential marker of pericytes associated with BBTB function. These pericytes interact with other cells in GBM mainly through extracellular matrix–integrin signaling pathways. Compared with normal pericytes, pericytes in GBM exhibited upregulation of several ECM genes (including collagen IV and FN1), and high expression levels of these genes were associated with a poor prognosis. Cell line experiments showed that PTH1R knockdown in pericytes increased collagen IV and FN1 expression levels. In mice models, the expression levels of PTH1R, collagen IV, and FN1 were consistent with these trends. Evans Blue leakage and IgG detection in the brain tissue suggested a negative correlation between PTH1R expression levels and blood–brain barrier function. Further, a risk model based on differentially expressed genes in PTH1R+ pericytes had predictive value for GBM, as validated using independent and in‐house cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee0728b81bf82d09f8e05a6d94f03f2e545717c1" target='_blank'>
              Single‐cell transcriptomic analysis of glioblastoma reveals pericytes contributing to the blood–brain–tumor barrier and tumor progression
              </a>
            </td>
          <td>
            Yuzhe Li, Changwu Wu, Xinmiao Long, Xiangyu Wang, Wei Gao, Kun-Kang Deng, Bo Xie, Sen Zhang, Minghua Wu, Qing Liu
          </td>
          <td>2024-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Recurrent therapy resistance is a major limitation in clinical efficacy and for the outcome of glioblastoma (GBM) patients, positioning GBM among the tumor types with the poorest survival outcomes. In this work, we dissected resistance mechanisms in GBM, which resulted in the identification of FGFR1 pathway as a major regulator of the signaling and metabolic rewiring associated with temozolomide (TMZ) resistance in GBM. Hence, we described a mechanism of resistance that operates at two major levels. First, a p53-mediated regulation of cell cycle inducing cell cycle arrest to allow DNA repair in response to TMZ. And second, a complete metabolic rewiring promoting lipid catabolism and preventing lipid peroxidation. Both the p53-mediated response and the metabolic adaptation are controlled by FGFR1, as inhibition of the FGFR1 pathway completely abolishes this signaling and metabolic reprograming, restoring sensitivity to TMZ. Our results also indicated a correlation of FGFR1 levels with poor prognosis in GBM patients, and validated the treatment of TMZ in combination with FGFR1 inhibitors as an efficient strategy to induce tumor cell death in pre-clinical animal models. This data position the receptor FGFR1 as a very promising candidate for evaluation in future clinical approaches to limit the development of therapy resistance to TMZ in GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba711301ae1507c841a26c809bbac2e4e721e807" target='_blank'>
              FGFR1 inhibition improves therapy efficacy and prevents metabolic adaptation associated with temozolomide resistance in glioblastoma
              </a>
            </td>
          <td>
            Laura Zarzuela, Ignacio G. López-Cepero, K. Rattigan, Elena Sánchez-Escabias, Macarena Morillo-Huesca, Ana Reina-Bando, Sara S. Oltra, J. M. Sierra-Párraga, María Ceballos-Chávez, Vivian Capilla-González, Gema Moreno-Bueno, P. S. Murdoch, JC Reyes, G. V. Helgason, Mercedes Tomé, Raúl V. Durán
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="While structural variants (SVs) are a clear sign of genomic instability, they have not been systematically quantified per patient. Therefore, the biological and clinical impact of high numbers of SVs in patients is unknown. We introduce tumor break load (TBL), defined as the sum of unbalanced SVs, as a measure for SV-associated genomic instability. Using pan-cancer data from TCGA, PCAWG, and CCLE, we show that a high TBL is associated with significant changes in gene expression in 26/31 cancer types that consistently involve upregulation of DNA damage repair and downregulation of immune response pathways. Patients with a high TBL show a higher risk of recurrence and shorter median survival times for 5/15 cancer types. Our data demonstrate that TBL is a biologically and clinically relevant feature of genomic instability that may aid patient prognostication and treatment stratification. For the datasets analyzed in this study, TBL has been made available in cBioPortal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26277db2501dbf3d4403083d453a9bedc8690ae8" target='_blank'>
              Tumor break load quantitates structural variant-associated genomic instability with biological and clinical relevance across cancers
              </a>
            </td>
          <td>
            S. Lakbir, Renske de Wit, Ino de Bruijn, Ritika Kundra, Ramyasree Madupuri, Jianjiong Gao, N. Schultz, Gerrit A. Meijer, Jaap Heringa, R. Fijneman, Sanne Abeln
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="The prognosis for muscle-invasive bladder cancer (MIBC) remains poor, and reliable prognostic markers have yet to be identified. Tertiary lymphoid structures (TLS) have been associated with favorable outcomes in certain cancers. However, the relationship between TLS and MIBC remains unclear. A multi-omics approach is utilized, leveraging single-cell RNA sequencing, spatial transcriptomics, bulk RNA sequencing, and immunohistochemistry, to investigate the roles of B cells and TLS in MIBC. These findings indicate that elevated levels of B cells and TLS correlate with improved prognoses in patients with MIBC, aligning with the robust antitumor immune responses observed in the TLS region. From a mechanistic perspective, CXCL13 serves as a critical cytokine for TLS formation in MIBC, primarily secreted by clonally expanded CXCL13+ T cells. This cytokine interacts with the CXCR5 receptor on NR4A2+ B cells, promoting TLS development. Plasma cells arising within the TLS microenvironment predominantly produce the IGHG antibody, potentially enhancing the phagocytic capabilities of C1QC+ macrophages. From an application standpoint, a TLS-specific gene signature is developed that effectively predicts outcomes in MIBC and other cancers. This study highlights the prognostic potential of TLS in MIBC and reveals immune mechanisms, offering insights for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a55189beb49b3fef971a608431673a12f1c24a25" target='_blank'>
              Tertiary Lymphoid Structures are Linked to Enhanced Antitumor Immunity and Better Prognosis in Muscle-Invasive Bladder Cancer.
              </a>
            </td>
          <td>
            Jiaxing Lin, Shan Jiang, Baoqiang Chen, Yiqing Du, C. Qin, Yuxuan Song, Yun Peng, Mengting Ding, Jilin Wu, Yihan Lin, Tao Xu
          </td>
          <td>2024-12-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Cancer research has historically focused on the Somatic Mutation Theory (SMT), viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking Cancer Evolution: From Genetic Mutations to Complex Information Systems in Tumor Reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="DNMT3A and TET2 are epigenetic regulator genes commonly mutated in age-related clonal hematopoiesis (CH). Despite having opposed epigenetic functions, these mutations are associated with increased all-cause mortality and a low risk for progression to hematological neoplasms. While individual impacts on the epigenome have been described using different model systems, the phenotypic complexity in humans remains to be elucidated. Here we make use of a natural inflammatory response occurring during coronavirus disease 2019 (COVID-19), to understand the association of these mutations with inflammatory morbidity (acute respiratory distress syndrome-ARDS) and mortality. We demonstrate the age-independent, negative impact of DNMT3A mutant CH on COVID-19-related ARDS and mortality. Using single cell (sc-) proteogenomics we show that DNMT3A mutations involve myeloid and lymphoid lineage cells. Using single cell multiomics sequencing, we identify cell-specific gene expression changes associated with DNMT3A mutations, along with significant epigenomic deregulation affecting enhancer accessibility, resulting in overexpression of IL32, a proinflammatory cytokine that can result in inflammasome activation in monocytes and macrophages. Finally, we show with single cell resolution that the loss of function of DNMT3A is directly associated with increased chromatin accessibility in mutant cells. Hence, we demonstrate the negative prognostic impact of DNMT3Amt CH on COVID-19 related ARDS and mortality. DNMT3Amt CH in the context of COVID-19, was associated with inflammatory transcriptional priming, resulting in overexpression of IL32. This overexpression was secondary to increased chromatic accessibility, specific to DNMT3Amt CH cells. DNMT3Amt CH can thus serve as a potential biomarker for adverse outcomes in COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be6f734acb174ae5d3944a4074bf5564fa474087" target='_blank'>
              Single cell multiomics reveal divergent effects of DNMT3A and TET2 mutant clonal hematopoiesis in inflammatory response.
              </a>
            </td>
          <td>
            Wazim Mohammed Ismail, Jenna A. Fernandez, M. Binder, T. Lasho, Minsuk Kim, Susan M. Geyer, Amelia Mazzone, C. Finke, A. Mangaonkar, Jeong-Heon Lee, Liguo Wang, Kwan Hyun Kim, Vernadette A. Simon, Fariborz Rakhshan Rohakthar, Amik Munankarmy, Seul Kee Byeon, S. Schwager, Jonathan J Harington, Melissa R. Snyder, Keith D Robertson, Akhilesh Pandey, Eric D. Wieben, Nicholas Chia, Alexandre Gaspar-Maia, Mirinal S Patnaik
          </td>
          <td>2024-12-04</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Clinical risk scores in chronic myeloid leukemia (CML) are inadequate for identifying chronic phase (CP) patients at high-risk of blast crisis (BC) progression. The lack of accurate predictive tests hamper timely interventions, including stem cell transplants, that are more effective in early disease. By interrogating a single cell atlas of primary imatinib resistance for a BC-like gene expression signature, we identify aberrant CD42A+ megakaryocytic and CD10+CD19+ lymphoid progenitor expansion, as well as STAT1- and IFNγ-related inflammatory programs, as consistent features in the bone marrow and peripheral blood of CP patients at high-risk of BC transformation. We develop multi-color flow cytometry-based tests (MFC) to detect these features in CP patients at the time of diagnosis. Validating our MFC panels on a combined Australia-Singapore cohort comprising 28 CP patients, including 14 who underwent transformation, we demonstrate their ability to detect 100% of CP patients who transform with no false positives. Our findings highlight small populations of inflamed hematopoietic stem and progenitor cells, present at CP diagnosis, as powerful harbingers of future BC transformation. The ability of MFC panels to detect these cells at diagnosis support their inclusion as accurate risk-assessment tools to improve management of high-risk patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/695b8c7eb368f693a97922122a5ffdc4d6cd3154" target='_blank'>
              Single-cell transcriptomics-guided development of flow cytometric tests predicting chronic myeloid leukemia blast crisis transformation at chronic phase diagnosis
              </a>
            </td>
          <td>
            Mengge Yu, Vaidehi Krishnan, Pavithra Shyamsundar, K. Lee, Florian Schmidt, Xi Ren, Charles T.H. Chuah, Shyam Prabhakar, V. Saunders, D. Yeung, N. Shanmuganathan, Agnes S. M. Yong, Timothy P. Hughes, S. T. Ong
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The ovary is one of the first organs to exhibit signs of aging, characterized by reduced tissue function, chronic inflammation, and fibrosis. Multinucleated giant cells (MNGCs), formed by macrophage fusion, typically occur in chronic immune pathologies, including infectious and non-infectious granulomas and the foreign body response1, but are also observed in the aging ovary2–4. The function and consequence of ovarian MNGCs remain unknown as their biological activity is highly context-dependent, and their large size has limited their isolation and analysis through technologies such as single-cell RNA sequencing. In this study, we define ovarian MNGCs through a deep analysis of their presence across age and species using advanced imaging technologies as well as their unique transcriptome using laser capture microdissection. MNGCs form complex interconnected networks that increase with age in both mouse and nonhuman primate ovaries. MNGCs are characterized by high Gpnmb expression, a putative marker of ovarian and non-ovarian MNGCs5,6. Pathway analysis highlighted functions in apoptotic cell clearance, lipid metabolism, proteolysis, immune processes, and increased oxidative phosphorylation and antioxidant activity. Thus, MNGCs have signatures related to degradative processes, immune function, and high metabolic activity. These processes were enriched in MNGCs compared to primary ovarian macrophages, suggesting discrete functionality. MNGCs express CD4 and colocalize with T-cells, which were enriched in regions of MNGCs, indicative of a close interaction between these immune cell types. These findings implicate MNGCs in modulation of the ovarian immune landscape during aging given their high penetrance and unique molecular signature that supports degradative and immune functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ded887bdf4a0069dbb82e21f70ee7c8e91b67f" target='_blank'>
              Multinucleated giant cells are hallmarks of ovarian aging with unique immune and degradation-associated molecular signatures
              </a>
            </td>
          <td>
            Aubrey Converse, Madeline J. Perry, Shweta S. Dipali, J. Isola, Emmett B. Kelly, Joseph M. Varberg, M. Zelinski, J. Gerton, Michael B. Stout, Michele T. Pritchard, Francesca E. Duncan
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background: Circulating rare cells participate in breast cancer evolution as systemic components of the disease and thus, are a source of theranostic information. Exploration of cancer-associated rare cells is in its infancy. Objectives: We aimed to investigate and classify abnormalities in the circulating rare cell population among early-stage breast cancer patients using fluorescence marker identification and cytomorphology. In addition, we sought to determine the dependency of these markers on the presence of tumors. Design: We evaluated the validity of a multi-rare-cell detection platform and demonstrated the utility of a specific rare cell subset as a novel approach to characterize the breast cancer system. Sampling was conducted both before and after tumor resection. Methods: Linearity of the Rarmax platform was established using a spike-in approach. The platform includes red blood cell lysis, leukocyte depletion and high-resolution fluorescence image recording. Rare cell analysis was conducted on 28 samples (before and after surgery) from 14 patients with breast cancer, 20 healthy volunteers and 9 noncancer control volunteers. In-depth identification of rare cells, including circulating tumor cells, endothelial-like cells, erythroblasts, and inflammation-associated cells, was performed using a phenotype and morphology-based classification system. Results: The platform performed linearly over a range of 5 to 950 spiked cells, with an average recovery of 84.6%. Circulating epithelial and endothelial-like cell subsets have been demonstrated to be associated with or independent of cancer with tumor presence. Furthermore, certain cell profile patterns may be associated with treatment-related adverse effects. The sensitivity in detecting tumor-presence and cancer-associated abnormality before surgery was 43% and 85.7%, respectively, and the specificity was 100% and 96.6%, respectively. Conclusion: This study supports the idea of a cancer-associated rare cell abnormality to represent tumor entities as well as systemic cancer. The latter is independent of the apparent clinical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dddc8e100310019b58c6626c61326f8e0d4cf2c" target='_blank'>
              Rare Cell Population Analysis in Early-Stage Breast Cancer Patients
              </a>
            </td>
          <td>
            Stefan Schreier, Prapaphan Budchart, S. Borwornpinyo, Lakkana Adireklarpwong, P. Chirappapha, W. Triampo, P. Lertsithichai
          </td>
          <td>2025-01-01</td>
          <td>Breast Cancer : Basic and Clinical Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb23096de05b742040245374a7585b16f9e2179" target='_blank'>
              Therapeutic implications and comprehensive insights into cellular senescence and aging in the tumor microenvironment of sarcoma
              </a>
            </td>
          <td>
            P. Zan, Yi Zhang, Yidong Zhu, Qingjing Chen, Zhengwei Duan, Yonghao Guan, Kaiyuan Liu, Anquan Shang, Zihua Li
          </td>
          <td>2025-01-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background Increasing evidence indicates that cancer stem cells (CSCs) and cancer stem-like cells form a special subpopulation of cells that are ubiquitous in tumors. These cells exhibit similar characteristics to those of normal stem cells in tissues; moreover, they are capable of self-renewal and differentiation, as well as high tumorigenicity and drug resistance. In prostate cancer (PCa), it is difficult to kill these cells using androgen signaling inhibitors and chemotherapy drugs. Consequently, the residual prostate cancer stem cells (PCSCs) mediate tumor recurrence and progression. Objective This review aims to provide a comprehensive and up-to-date overview of PCSCs, with a particular emphasis on potential therapeutic strategies targeting these cells. Methods After searching in PubMed and Embase databases using ‘prostate cancer’ and ‘cancer stem cells’ as keywords, studies related were compiled and examined. Results In this review, we detail the origin and characteristics of PCSCs, introduce the regulatory pathways closely related to CSC survival and stemness maintenance, and discuss the link between epithelial–mesenchymal transition, tumor microenvironment and tumor stemness. Furthermore, we introduce the currently available therapeutic strategies targeting CSCs, including signaling pathway inhibitors, anti-apoptotic protein inhibitors, microRNAs, nanomedicine, and immunotherapy. Lastly, we summarize the limitations of current CSC research and mention future research directions. Conclusion A deeper understanding of the regulatory network and molecular markers of PCSCs could facilitate the development of novel therapeutic strategies targeting these cells. Previous preclinical studies have demonstrated the potential of this treatment approach. In the future, this may offer alternative treatment options for PCa patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/537b50df573d4150f73440d71dd0d947241b969e" target='_blank'>
              Stemness regulation in prostate cancer: prostate cancer stem cells and targeted therapy
              </a>
            </td>
          <td>
            Hao Liang, Bin Zhou, Peixin Li, Xiaoyi Zhang, Shijie Zhang, Yaozhong Zhang, Shengwen Yao, Sifeng Qu, Jun Chen
          </td>
          <td>2024-12-23</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lobular carcinoma in situ (LCIS) is a precursor of invasive lobular carcinoma of the breast. LCIS cells lack cell-cell cohesion due to the loss of E-cadherin. LCIS cells grow in mammary lobules rather than in ducts. The etiology of this pattern, especially its dependence on cellular cohesion, is incompletely understood. We simulated passive intra-glandular scattering of carcinoma in situ (CIS) cells in an ultra-simplified hollow mold tissue replica (HMTR) and a discrete-time mathematical model featuring particles of variable sizes representing single cells (LCIS-like particles) or groups of cohesive carcinoma cells (DCIS-like particles). The HMTR features structures reminiscent of a mammary duct with associated lobules. The discrete mathematical model characterizes spatial redistribution over time and includes transition probabilities between ductal or lobular localizations. Redistribution of particles converged toward an equilibrium depending on particle size. Strikingly, equilibrium proportions depended on particle properties, which we also confirm in a continuous-time mathematical model that considers controlling lobular properties such as crowding. Particles of increasing size, representing CIS cells with proficient cohesion, showed increasingly higher equilibrium ductal proportions. Our investigations represent two conceptual abstractions implying a link between loss of cell-cell cohesion and lobular localization of LCIS, which provide a much-needed logical foundation for studying the connections between collective cell behavior and cancer development in breast tissues. In light of the findings from our simplified modeling approach, we discuss multiple avenues for near-future research that can address and evaluate the redistribution hypothesis mathematically and empirically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20e9057af1188f98a4f70cd23eb06e09c7352b5b" target='_blank'>
              Deficient cell-cell cohesion is linked with lobular localization in simplified models of lobular carcinoma in situ (LCIS)
              </a>
            </td>
          <td>
            Matthias Christgen, R. A. Caetano, M. Eisenburger, Arne Traulsen, P. Altrock
          </td>
          <td>2024-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Brain metastases are ten times more common than primary brain tumors and pose a significant clinical challenge. How brain metastatic tumor cells adapt to the unique and hostile brain microenvironment remains unclear. Astrocytes, the most abundant glial cells in the brain, are emerging as key mediators regulating the development of brain metastases. Initially anti-metastatic, astrocytes are reprogrammed by tumor-derived signals, transitioning into a pro-metastatic phenotype. Here, we review the roles of astrocytes in brain metastasis and describe the evidence for their phenotypic plasticity, the basis of astrocyte-tumor interactions, and potential therapeutic strategies targeting these processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af554e7b525b7c5d5e54e89384537ddfea3d74b6" target='_blank'>
              Astrocyte plasticity shapes brain metastasis progression through tumor-astrocyte crosstalk
              </a>
            </td>
          <td>
            Rupleen Kaur, Nalin Gupta, Andrew Dhawan
          </td>
          <td>2025-01-13</td>
          <td>Green Energy and Sustainability</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-06</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [11, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>