<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-03 09:32:55 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Kaposi’s sarcoma (KS) is a highly inflammatory, angiogenic tumor driven by Kaposi’s sarcoma-associated herpesvirus (KSHV), yet the origins of tumor cells and mechanisms of progression remain unclear. Here, we present the first spatial single-cell atlas of KS, profiling 256 samples across patch, plaque, and nodular lesions and normal controls. We identify CD34⁺ progenitor lymphatic endothelial cells (LECs) as the primary targets of KSHV, whose clonal expansion drives tumor growth. KSHV infection induces widespread cellular reprogramming across the tumor microenvironment, including LECs, vascular endothelial cells, fibroblasts, and macrophages, generating hybrid phenotypes that support angiogenesis, inflammation, and immune modulation. KSHV⁺ macrophages are enriched in tumor-proximal niches, further promoting a proangiogenic, immunosuppressive environment. Spatial analysis reveals evolving tumor-associated niches, with a core-to-periphery gradient correlating with infection, immune modulation, and cellular remodeling. We identify disease progression predictive signatures, offering mechanistic insights into KS pathogenesis and potential new therapeutic strategies by reprogramming the tumor microenvironment. Highlights ➣ CD34+ progenitor lymphatic endothelial cells are the primary KSHV target cells, with their clonal expansion driving Kaposi’s sarcoma growth. ➣ KSHV infection reprograms broad cell types into hybrid identities, and drives tumor-specific niche and vascular remodeling, endothelial plasticity, and immune modulation. ➣ KSHV-reprogrammed macrophages drive inflammation, angiogenesis, and immune modulation. ➣ Spatially resolved molecular and cellular signatures predict Kaposi’s sarcoma progression, offering novel therapeutic strategies targeting the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835b859927d90c9b0c5454b990e0f2840ed6f867" target='_blank'>
              Spatial Single-Cell Atlas Reveals KSHV-Driven Broad Cellular Reprogramming, Progenitor Expansion, Immune and Vascular Remodeling in Kaposi’s Sarcoma
              </a>
            </td>
          <td>
            Wen Meng, Arun Das, Harsh Sinha, Rana Naous, P. Bracci, Mike McGrath, Yufei Huang, Shou-Jiang Gao
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728733c32d8e17fbc34a5070c8b658170e0a8a22" target='_blank'>
              Identifying tissue states by spatial protein patterns related to chemotherapy response in triple-negative breast cancer
              </a>
            </td>
          <td>
            L.M. Luque, A. Khan, G. Torrisi, Tessa D. Green, D. Hardman, C. Owczarek, T. A. Phillips, D. Marks, M. Parsons, C. Sander, L. Schumacher
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Aggressive cancers, such as pancreatic ductal adenocarcinoma (PDAC), are often characterized by a complex and desmoplastic tumor microenvironment, a stroma rich supportive connective tissue composed primarily of extracellular matrix (ECM) and non-cancerous cells. Desmoplasia, a dense deposition of stroma, is a major reason for therapy resistance, acting both as a physical barrier that interferes with drug penetration and as a supportive niche that protects cancer cells through diverse mechanisms. Precise understanding of spatial cell interactions in stroma-rich tumors is essential for optimizing therapeutic responses. It enables detailed mapping of stromal-tumor interfaces, comprehensive cell phenotyping, and insights into changes in tissue architecture, improving assessment of drug responses. Recent advances in multiplexed immunofluorescence imaging have enabled the acquisition of large batches of whole-slide tumor images, but scalable and reproducible methods to analyze the spatial distribution of cell states relative to stromal regions remain limited. To address this gap, we developed an open-source computational pipeline that integrates QuPath, StarDist, and custom Python scripts to quantify biomarker expression at a single- and sub-cellular resolution across entire tumor sections. Our workflow includes: (i) automated nuclei segmentation using StarDist, (ii) machine learning-based cell classification using multiplexed marker expression, (iii) modeling of stromal regions based on fibronectin staining, (iv) sensitivity analyses on classification thresholds to ensure robustness across heterogeneous datasets, and (v) distance-based quantification of the proximity of each cell to the stromal border. To improve consistency across slides with variable staining intensities, we introduce a statistical strategy that translates classification thresholds by propagating a chosen reference percentile across the distribution of marker-related cell measurement in each image. We apply this approach to quantify spatial patterns of distribution of the phosphorylated form of the N-Myc downregulated gene 1 (NDRG1), a novel DNA repair protein that conveys signals from the ECM to the nucleus to maintain replication fork homeostasis, and a known cell proliferation marker Ki67 in fibronectin-defined stromal regions in PDAC xenografts. The pipeline is applicable for the analysis of markers of interest in stroma-rich tissues and is publicly available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b6327fb55ba3776af18a8a1265a4ec0c3c69a0b" target='_blank'>
              An image analysis pipeline to quantify the spatial distribution of cell markers in stroma-rich tumors
              </a>
            </td>
          <td>
            Antoine A. Ruzette, Nina Kozlova, Kayla A. Cruz, Taru Muranen, S. Nørrelykke
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) presents diverse histomorphological features within the tumor microenvironment (TME) that influence prognosis and response to immunotherapy. Leveraging contrastive learning, we developed an unbiased atlas of cell neighborhoods to systematically explore the LUAD microenvironment at the cellular scale and investigate how these neighborhoods combine to form histologic patterns. This multiscale approach enables a comprehensive understanding of both cell-specific interactions and broader histologic architecture in LUAD. Our analysis identified distinct histomorphological phenotype clusters of cellular neighborhoods (cn-HPCs) with prognostic significance. For example, cn-HPC 0 was associated with immune activation and favorable survival, while cn-HPC 23 was enriched in necrotic, immune-excluded regions and aligned with poorer outcomes. Through associations with immunophenotypes, co-expressed gene modules, and pathway enrichment, we found that cn-HPCs reflect underlying processes of immune modulation, cellular growth, and inflammation, offering insights into the functional landscape of LUAD. In an independent LUAD immunotherapy cohort, we found that cn-HPC 23 showed promising predictive value for treatment response. These findings suggest that fine-grained spatial profiling of the TME may offer complementary value for identifying patients more likely to benefit from immunotherapy. Taken together, our study highlights the potential of cn-HPCs to bridge histopathological patterns and clinical outcomes, supporting their relevance for prognosis and treatment stratification in LUAD. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17117-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732a50286d99e4889f11b9dfeef8dfb5de086217" target='_blank'>
              Contrastive learning uncovers cellular interactions and morphologies in the tumor microenvironment of lung adenocarcinoma linked to immunotherapy response
              </a>
            </td>
          <td>
            H. Le, N. Coudray, A. Yeaton, Sitharam Ramaswami, A. Karimkhan, Wei-Yi Cheng, James Cai, Tai-Hsien Ou Yang, S. Punekar, Luis Chiriboga, Vamsidhar Velcheti, Kwok-Kin Wong, D. Sterman, Andre L. Moreira, Harvey I. Pass, A. Tsirigos
          </td>
          <td>2025-10-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Tumors acquire a set of functional capabilities, known as the Hallmarks of Cancer, to thrive and grow. While extensive research has revealed how gene and protein expression networks contribute to these traits, the relationships between tumor cell states, spatial organization of immune and stromal cells in the tumor microenvironment, and tumor genetics, remains poorly understood. To address this, we constructed a cohort of primary tumor tissues from 117 pancreatic ductal adenocarcinoma (PDAC) patients. Using Multiplexed Ion-Beam Imaging (MIBI) with a 40-antibody panel, we mapped the spatial organization of major cell types and their phenotypes. We also conducted targeted genomic and bulk proteomic analyses to explore how spatial organization patterns align with tumor genetics and proteome profiles. We identified phenotypic subtypes of tumor epithelium that shape distinct microenvironmental niches: (i) Glycolytic tumors expressing proteins such as GLUT1, LDHA, MCT1 and HIF1α reside in hypoxic niches. They are encased by dense fibrotic stroma and lacking vasculature and immune cell infiltration. (ii) Tumor and metaplastic epithelium that exhibit an oxidative proteome profile, are highly enriched for ECAD and PD-L1 expression, and are situated near pancreatic epithelium and vasculature, with significant immune infiltration. (iii) Myeloid-interacting tumors expressing CXCL5, heavily infiltrated with neutrophils and characterized by low-collagen and high FAP positive fibroblasts. Notably, neutrophils accumulate within the lumen of tumor ducts where they exhibit increased apoptosis, DNA damage, and netosis. These highly neutrophil-infiltrated tumors, associate with poor disease outcome. The genetic landscape of PDAC influences these spatial and phenotypic patterns. Patients with TGFβ/SMAD pathway mutations, demonstrate an increased proportion of basal tumor cells accompanied by reduced lymphocytic infiltration. Similarly, distinct TP53 variants shape immune-cell infiltration: patients harboring missense mutations show increased lymphocytic infiltration compared to those with truncating mutations. These findings underscore the strong link between tumor genetics, phenotypic heterogeneity and cellular architecture of the PDAC microenvironment, offering new insights into collective tumor behavior.



 Ofer Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, Inti Zlobec, Tamar Geiger, Leeat Keren. Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B123.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb5d6211d2e33680561c9558492cf85ff16b33" target='_blank'>
              Abstract B123: Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization
              </a>
            </td>
          <td>
            O. Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, I. Zlobec, Tamar Geiger, Leeat Keren
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235c8c18e964369ee970e129fd3fe5bd3b1a1aa6" target='_blank'>
              In silico reconstruction of primary and metastatic tumor architecture using GIS-augmented spatial transcriptomics
              </a>
            </td>
          <td>
            Jin Young Yoo, Sabrina Akter, Qianying Zuo, Amir Kazemi, Wenjun Wu, Mahima Goel, Audrey Lam, Debapriya Dutta, Betsy Barnick, Melike Pekmezci, Maria Grosse Perdekamp, Aiman Soliman, Zeynep Madak-Erdogan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d523903df00c8d74634f25362c4c022807a0ad86" target='_blank'>
              Spatial proteomics for investigating solid tumor resistance mechanisms.
              </a>
            </td>
          <td>
            X. M. M. Zhou, A. D'Amiano, Charles Lu, Vrinda Madan, Sara Khoshniyati, J. Kollings, Noah E Sunshine, S. Surwase, Joel C. Sunshine
          </td>
          <td>2025-10-08</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract The heterogeneous nature of cell populations in human tumors is a major contributor to tumor evolution, including and perhaps most importantly in response to treatment. Here, we review current knowledge on tumor heterogeneity and cell state plasticity in small cell lung cancer (SCLC), a fast growing and highly metastatic form of lung cancer which develops rapid resistance to therapy. There is a pressing need to expand treatment options for patients with SCLC, which requires a better understanding of the mechanisms by which this disease is able to rapidly grow and evolve in response to therapy. Our current understanding points to epigenetic rather than genetic factors in defining major aspects of inter- and intra-tumoral heterogeneity in SCLC. SCLC is overall considered to be a neuroendocrine (NE) cancer type but SCLC tumors harbor a wide diversity of cancer cell states, including both NE and non-neuroendocrine (non-NE) states, defined by their mutually exclusive expression of a set of transcription factors such as ASCL1, NEUROD1, and POU2F3. The immune microenvironments of SCLC tumors also contain a great deal of heterogeneity. Here, we discuss the different SCLC cell states associated with their defining transcription factors, as well as the epigenetic mechanisms regulating the ability of SCLC cells to switch from one state to another. We further discuss how the composition of SCLC tumors and the surrounding immune cells may affect the response to chemotherapy and immunotherapy. Being able to control plasticity and heterogeneity in SCLC may in the future offer unique opportunities to improve treatment efficacy in this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f9c75cb099eaa87426731186cf92ba20e24f46a" target='_blank'>
              Tumor Heterogeneity and Plasticity in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Gina Duronio, Julien Sage
          </td>
          <td>2025-10-01</td>
          <td>Lung Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9320984fb7ca80afa8b2d49d724dd86390a378" target='_blank'>
              Transposable Elements and Homotypic Niches Drive Immune Dynamics and Resistance in Melanoma Epigenetic-based immunotherapy
              </a>
            </td>
          <td>
            Erika Ciervo, Francesco Ceccarelli, A. D. Di Giacomo, Piera Grisolia, A. Covre, Z. M. Besharat, Antonio De Falco, F. Caruso, Luigi Laezza, Luigi Ferraro, Gloria Mas Martin, M. F. Lofiego, Tommaso Sani, E. Ferretti, Yan Guo, Sean B. Holden, R. Mortarini, A. Anichini, M. Maio, T. Noviello, M. Ceccarelli
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            , , Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The spatial organization and composition of the tumor-immune microenvironment (TME) play a critical role in shaping the progression of many solid cancers, but the organization of the TME in primary prostate cancer (PCa) remains poorly characterized. We therefore profiled the abundance and spatial distributions of major cell types involved in adaptive immunity in 29 radical prostatectomy specimens stratified into high (HGG; n=14) and low Gleason-grade (LGG; n=15). Compared to LGG, HGG PCa exhibited significantly greater B and T cell infiltration with many immune cells organized into clusters, some of which resembled tertiary lymphoid structures (TLSs). In HGG tumors, these clusters were dense, symmetric, rich in PD-1+ T cells, and frequently proximate to the tumor compartment. LGG clusters were less well organized, and T cell depleted. Thus, a subset of high-grade PCa harbor organized immune clusters that may play a role in tumor control and contain therapeutically targetable T and B cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e5e762dad0168906d48c0967522a47db6d0ac" target='_blank'>
              Spatially organized lymphocytic microenvironments in high grade primary prostate tumors
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, Jia-Ren Lin, Brian Labadie, A. Atmakuri, Z. Maliga, Eamon Toye, K. Chaudagar, Madeleine S. Torcasso, Shannon Coy, G. Fanelli, Brigette Kobs, F. Socciarelli, Andréanne Gagné, Eliezer Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Spontaneously transformed mouse ovarian surface epithelial cell line ID8 generates a syngeneic mouse model, recapitulating the primary and metastatic stages of the malignancy hence, frequently used in pre-clinical evaluation. Due to inherent phenotypic plasticity, primary and metastatic tumors show contrasting phenotypic features dictated by the tissue niche. To address the tumor heterogeneity within the model we stratified it into a specific molecular subtype, which would provide an accurate setting for interpretation of experimental data. Stratification was performed on ID8 cell line and ID8 generated xenografts based on immunostaining, live cell imaging, histopathology, transcriptomics and proteomics. Tumor microenvironment (TME) investigation for each xenograft was done through flow cytometry analysis. ID8 cells present epithelial/mesenchymal (E/M) phenotype. Phenotypic plasticity retention in them causes class switching of ID8 cells in response to niche alteration, from E/M (in culture) to intermediate epithelial (iE) in orthotopic tumors while intermediate mesenchymal (iM) in subcutaneous tumors. Metastasis of ID8 primary tumor resulted into differentiated mesenchymal phenotype in ascites while differentiated epithelial phenotype in secondary intestinal tumors. Immune profiling of primary and secondary xenografts revealed differences in immune repertoire of tumors, highlighting subcutaneous tumor model to be inappropriate as compared to orthotopic model. Local tissue niche modulates the phenotype of tumors generated, transitioning through intermediate states towards a final commitment to either an epithelial or mesenchymal state that recapitulates the reported class switching in patient tumors. During disease progression, such heterogeneity of tumor cells at different niche(s) and their cross-talks with discrete TME at each site, will pose challenges for effective therapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18264-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20023e2da4c4317c3adb1e410526c772109c27da" target='_blank'>
              ID8 cells manifest phenotypic plasticity and molecular heterogeneity of high-grade serous ovarian cancer in response to the local tissue niche
              </a>
            </td>
          <td>
            Ritika Gupta, S. Shravan, Swapna Balaji Agwane, R. M. Bankar, S. Bapat
          </td>
          <td>2025-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has emerged as an advanced biological technology capable of resolving the complexity of cancer landscapes at single-cell resolution. Spatial transcriptomics(ST), as an innovative complementary approach, effectively compensates for the lack of spatial information inherent in scRNA-seq data. This review explores the rapidly evolving integration of scRNA-seq and ST and their transformative role in deciphering the tumor microenvironment (TME). We highlight how these technologies jointly uncover cellular heterogeneity, stromal-immune interactions, and spatial niches driving tumor progression and therapy resistance. Moving beyond previous reviews, we emphasize emerging computational strategies for data integration—including deconvolution and mapping approaches—and evaluate their applications in characterizing immune evasion, fibroblast diversity, and cell-cell communication networks. Ultimately, this review provides a forward-looking perspective on how spatial multi-omics are poised to advance precision oncology through spatially-informed biomarkers and diagnostic tools. We conclude that the full clinical potential of these technologies relies on closing the gap between analytical innovation and robust clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84801b1d17cdb40a9473ba25f6bac709b5c4c887" target='_blank'>
              Single-cell and spatial transcriptomics integration: new frontiers in tumor microenvironment and cellular communication
              </a>
            </td>
          <td>
            Wenxin Shi, Zhiqiang Zhang, Xiaotong Xu, Yanpeng Tian, Li Feng, Xianghua Huang, Yanfang Du, Zhongkang Li
          </td>
          <td>2025-10-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Quantitative investigation of how the spatial organization of cells within the tumor microenvironment associates with disease progression, patient outcomes, and that cell’s phenotypic state remains a key challenge in cancer biology. High-dimensional multiplexed imaging offers an opportunity to explore these relationships at single-cell resolution. Methods We developed a computational pipeline to quantify and analyze the neighborhood profiles of individual cells in multiplexed immunofluorescence images. The pipeline characterizes spatial co-localization patterns within the tumor microenvironment and applies interpretable supervised machine learning models, specifically orthogonal partial least squares analysis (OPLS), to identify spatial relationships predictive of cell states and clinical phenotypes. Results We applied this framework to a previously published non-small cell lung cancer (NSCLC) cohort across four applications. At the cellular level, we identified neighborhood features associated with lymphocyte activation states. At the tumor-immune interface, we demonstrated that the immune cell composition surrounding major histocompatibility complex class I-expressing (MHC I+) tumor cells could distinguish adenocarcinoma from squamous cell carcinoma. At the patient level, spatial features predicted tumor grade. Discussion By integrating cell-segmented imaging data with interpretable modeling, our pipeline reveals key spatial determinants of tumor biology. These findings generate testable mechanistic hypotheses about intercellular interactions and support the development of spatially informed prognostic and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a735226da4130f5ff72da463b9cc63eeddebebdb" target='_blank'>
              A flexible systems analysis pipeline for elucidating spatial relationships in the tumor microenvironment linked with cellular phenotypes and patient-level features
              </a>
            </td>
          <td>
            Gabriel F. Hanson, Kate A. Goundry, Remziye E. Wessel, Michael G Brown, Timothy N. J. Bullock, Sepideh Dolatshahi
          </td>
          <td>2025-09-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The heterogeneity and dynamic evolution of the tumor microenvironment (TME) are key drivers of tumor progression, immune evasion, and therapeutic resistance. Traditional transcriptomic approaches are limited by their reliance on population-averaged data and the lack of spatial context, making it challenging to fully dissect the complex interactions among tumor cells, immune infiltrates, and stromal components within the TME. In recent years, the synergistic advancement of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has revolutionized our ability to construct high-resolution cellular atlases and delineate spatially organized functional modules within the TME. These technologies have enabled the identification of novel cell subtypes, cell-cell communication pathways, and spatial niches associated with disease progression and therapy response. This review systematically compiles recent technological breakthroughs in the integration of single-cell and spatial multi-omics, highlights key discoveries in TME research, and discusses their transformative impact on the development of precision oncology. Additionally, it addresses current technical limitations and explores emerging strategies and future directions for advancing this rapidly evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90a0c36ae523e1a4fc23b5c8bc8013f692b5b483" target='_blank'>
              Refining Tumor Microenvironment Insights via Single-Cell and Spatial Transcriptomics for Precision Medicine
              </a>
            </td>
          <td>
            Xianbing Wang
          </td>
          <td>2025-09-05</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive disease with a dense fibrotic stroma and is often accompanied by chronic inflammation. Peritumoral inflammation is typically viewed as a reaction to nearby tumor growth. Here, we report that the inflamed pancreatic lobules are frequently invaded by tumor cells, forming a distinct, non-fibrotic tumor niche. Using a semi-supervised machine learning approach for annotations of clinical samples and multiplex protein profiling, we show that tumor cells at the invasion front are closely associated with acinar cells undergoing damage-induced changes, and with activated fibroblasts expressing markers of injury. The invaded lobules are linked to classical tumor phenotypes, in contrast to fibrotic areas where tumor cells display a more basal profile, highlighting microenvironment-dependent tumor subtype differences. In female mice, lobular invasion similarly aligns with the classical tumor phenotype. Together, our data reveal that pancreatic tumors colonize injured lobules, creating a unique niche that shapes tumor characteristics and contributes to disease biology. Peritumoural pancreatitis and lobular injury are commonly seen in pancreatic cancer. Here, the authors reveal interactions between invading tumour cells and inflammation-related lobular cells, linked to switching from a basal to classical tumor cell phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33fdbbbab057fd0e654aba5735a7c83d83362e84" target='_blank'>
              An injury-associated lobular microniche is associated with the classical tumor cell phenotype in pancreatic cancer
              </a>
            </td>
          <td>
            Sara Söderqvist, Annika Viljamaa, Natalie Geyer, Anna-Lena Keller, Kseniya Ruksha, Carina Strell, Neda Hekmati, Alexandra Niculae, J. Engstrand, E. Sparrelid, Caroline Salmén, Tânia D. F. Costa, Miao Zhao, Staffan Strömblad, Argyro Zacharouli, P. Ghorbani, Sara Harrizi, Yousra Hamidi, Olga Khorosjutina, Stefina Milanova, Bernhard Schmierer, Béla Bozóky, C. Fernández Moro, Marco Gerling
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c74617e8894674076d6153fb81e22ba9cb23b96" target='_blank'>
              Comparison of Human Melanoma Single Cell Profiles to Evolutionary Medicine Model Xiphophorus Provides Insights in Disease Control
              </a>
            </td>
          <td>
            Yanting Xing, Rachel Carrol, X. Maggs, Wes C Warren, Manfred Schartl, Yuan Lu
          </td>
          <td>2025-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2µm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b808c693c9788562c4b889cdeaf7fc68e251b" target='_blank'>
              Spatial transcriptomics of glioblastoma defines biologically and clinically significant reprogramming patterns across unique spatial microenvironments
              </a>
            </td>
          <td>
            V. Ravikumar, A. Maddox, Reva Kulkarni, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Tumor thrombus (TT) worsens prognosis and complicates surgery in renal cell carcinoma (RCC), yet its formation mechanisms remain unclear. Here, we perform integrative single-cell and spatial transcriptomic analyses on 71 tissues and 48 sections from RCC patients with or without TT. The cellular and spatial atlas reveals distinct TT-associated tumor microenvironment remodeling characterized by the enrichment of FAP+ fibroblasts. These FAP+ fibroblasts are spatially contiguous to aggressive cancer cells and promote their malignant phenotypes in vitro. Their abundance inversely correlates with functional NK cells, suggesting roles in tumor invasion and immune evasion. Furthermore, single-cell multiomics analysis identifies tumor pericytes as a source of FAP+ fibroblasts and delineates transcription factor dynamics underlying pericyte-fibroblast transition. Finally, high levels of FAP+ fibroblasts are associated with poor prognosis and predict a weaker response to anti-VEGF-based therapy. In conclusion, our study highlights FAP+ fibroblasts as drivers of aggressive RCC with TT, suggesting potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5472524a744e590345b74e85358cb939dc059d29" target='_blank'>
              FAP+ fibroblasts orchestrate tumor microenvironment remodeling in renal cell carcinoma with tumor thrombus
              </a>
            </td>
          <td>
            Jiacheng Ma, Yan Huang, Jie Chen, Yang Li, Rongyan Yao, Xiubin Li, Qiyang Liang, Xinran Chen, Cheng Peng, Kan Liu, Yuanjun Zhai, Xu Zhang, Xin Ma, Xiaowen Wang, Qingbo Huang, Fuchu He
          </td>
          <td>2025-10-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd7292fed094fd219af6750590d28db3dc5a363b" target='_blank'>
              Spatial organization of the tumor-immune microenvironment in ER-positive breast cancer: remodeling during treatment and associations with clinical response
              </a>
            </td>
          <td>
            Maria Aanesland Dahle, Jan-Lukas Førde, E. V. Egeland, A. Creason, Cameron Watson, Ø. Garred, L. Prasmickaite, G. Mælandsmo, Gordon B. Mills, O. Engebraaten, M. H. Haugen
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98126a0a8ef060e1442a8189ca2cc50b5b3d5b2e" target='_blank'>
              Integrated cell atlas and tumoroids chart pancreatic cancer therapeutic targets
              </a>
            </td>
          <td>
            Quan Xu, Ryo Okuda, Bruno Gjeta, Christoph Harmel, Marina Signer, Matilde Lucioli, Małgorzata Santel, M. Seimiya, Cinzia Esposito, Karolina Guja-Jarosz, Ashley Maynard, S. Morinaga, Y. Miyagi, Tomoyuki Yamaguchi, Y. Ueno, Salvatore Piscuoglio, Daniel J. Müller, Hideki Taniguichi, Barbara Treutlein, J. G. Camp
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly heterogeneous cancer with complex tumor–immune interactions. This heterogeneity, particularly in tumor and immune cells, complicates treatment and prognostic evaluation. Although recent studies have revealed distinct tumor cell states and immune dysfunction in HCC, the molecular basis underlying tumor aggressiveness remains poorly understood. A deeper understanding of the molecular and functional diversity of both tumor and immune cell populations, especially the identification of stem-like tumor subpopulations and immunosuppressive mechanisms, along with the development of robust prognostic biomarkers, is essential for advancing precision oncology and improving clinical outcomes. Methods We integrated three publicly available single-cell RNA sequencing (scRNA-seq) datasets from GEO to delineate the cellular architecture of the HCC tumor microenvironment. Unsupervised clustering and dimensionality reduction were employed to identify major cell types and tumor subpopulations. Functional annotation was performed using canonical markers, Monocle, CytoTRACE, and AUCell scoring. H2AFZ was identified as a candidate oncogene and validated through in vitro knockdown experiments. The interaction between T cell subsets and tumor subpopulations were further characterized. A prognostic risk model was constructed using LASSO regression. Results Six major cell types were identified in HCC TME. Tumor cells were subdivided into three distinct clusters: Tumor_C0, Tumor_C1 and Tumor_C2. Tumor_C2 showed the highest stemness, pro-metastatic activity and immunogenic cell death signatures. H2AFZ was highly expressed in Tumor_C2 and associated with poor prognosis. The knockdown of H2AFZ reduced H2A.Z protein levels, inhibited proliferation, invasion, and induced apoptosis. T cell analysis revealed five subpopulations. It was found that Tumor_C2 interacts with the proliferative and exhausted T cell subpopulations, suggesting a potential functional relationship between them. The prognostic model based on tumor_C2 transcriptomic features effectively stratified patient survival across multiple cohorts, with robust AUCs and Kaplan-Meier survival distinctions. Conclusion We identified a proliferative, stem-like tumor cell subpopulation (Tumor_C2) in HCC characterized by high H2AFZ expression, which drives tumor aggressiveness. T cell analysis revealed significant interactions with Tumor_C2. Moreover, a prognostic model based on Tumor_C2 features effectively stratified patient survival across multiple cohorts. Together, these findings highlight potential therapeutic targets for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7024320dd03972a4da66c548f19e56393e35ae99" target='_blank'>
              Single-cell transcriptomics identifies an H2AFZ-driven proliferative tumor subpopulation associated with poor prognosis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jihui Huo, Tao Yang, Kai Lei, Zeyao Wang, Zebin Chen, Qi Zhou
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Objectives Small cell lung cancer (SCLC) accounts for approximately 15% of lung tumors and is marked by aggressive growth and early metastatic spread. In this study, we used two SCLC mouse models with differing tumor mutation burdens (TMB). To investigate tumor composition, spatial architecture, and interactions with the surrounding microenvironment, we acquired multiplexed images of mouse lung tumors using imaging mass cytometry (IMC). These data build upon a previously published characterization of the mouse model. Data description After tumor detection, mice were assigned to one of five treatment groups. Lung tumor tissues were imaged with a 37-marker IMC panel designed to identify major cell types—tumor, immune, and structural—as well as their functional states. When possible, each tumor was sampled both at its center and border regions. Tumor masks in the form of binary images are provided to delineate tumor areas. Additional metadata include tumor onset and endpoint dates to support downstream correlation or predictive analyses based on the image data. This dataset offers a valuable resource for studying the histological and cellular complexity of SCLC in a genetically controlled mouse model across multiple therapeutic conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597aa5e8dfd6901b69ffa28e7837ffa96623cc96" target='_blank'>
              Imaging mass cytometry dataset of small-cell lung cancer tumors and tumor microenvironments
              </a>
            </td>
          <td>
            France Rose, O. Ibruli, Luca Lichius, M. Kiljan, Gokcen Gozum, Manoela Iannicelli Caiaffa, Jiali Cai, Li‐na Niu, Jan M. Herter, Holger Grüll, Reinhard Büttner, F. Beleggia, Graziella Bosco, Julie George, G. Herter-Sprie, H. C. Reinhardt, Katarzyna Bożek
          </td>
          <td>2025-09-08</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af62a3f15720a1c9a956b0f31df3cb0beb418ea" target='_blank'>
              Cell migration sculpts evolutionary dynamics favouring therapy resistance in lung cancer
              </a>
            </td>
          <td>
            A. Bhargava, Xiao Fu, Sasha Bailey, Yutaka Naito, Hamid Mohammadi, R. Hynds, S. Hooper, D. Biswas, A. Le Marois, Sunil Kumar, Yuriy Alexandrov, Paul French, James Mcginty, Paul A. Bates, C. Swanton, E. Sahai
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed2e2b6e5867f00f570022e3faf5971a5955554" target='_blank'>
              Highly multiplexed imaging recovers immune and metabolic niches in multiple myeloma associated with disease progression and bone involvement
              </a>
            </td>
          <td>
            Ingrid Aass Roseth, Lukas Hatscher, Chiara Schiller, Håkon Skjalg Selland Johnstuen, T. Slørdahl, Håkon Hov, D. Schapiro, T. Standal
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696e778534089692f9510070ab5b7ccf3c2a11f2" target='_blank'>
              Single-cell spatial mapping reveals dynamic bone marrow microarchitectural alterations and enhances clinical diagnostics in MDS
              </a>
            </td>
          <td>
            Ryan Nachman, Aleksandra Kopacz, Caitlin Unkenholz, Jian Chai, Arvin Ruiz, Itzel Valencia, Jeanne Jiang, F. Socciarelli, Jiwoon Park, Christopher E. Mason, Ling Zhang, D. Sallman, Gail J. Roboz, Pinkal M. Desai, Justin Kaner, Joshua A Fein, Monica L Guzman, Neal Lindeman, Amy Chadburn, Madhu M Ouseph, Paul Simonson, Julia Geyer, Giorgio Inghirami, Shahin Rafii, David Redmond, Sanjay Patel
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab36a15bd0fb0bbb24a4b0020944cbb88eef13f0" target='_blank'>
              OSDR2.0 infers microenvironment-driven cell-state transitions and population dynamics from a single spatial biopsy
              </a>
            </td>
          <td>
            Itay Ben Shalom, Jonathan Somer, Shoval Miyara, Avi Mayo, Shie Mannor, U. Alon
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64783d2fd1c1cb64aa22b77e49df40179825bbfc" target='_blank'>
              Inferring cellular communication through mapping cells in space using Tangram 2
              </a>
            </td>
          <td>
            Hejin Huang, Alma Andersson, Sunny Z. Wu, Gabriele Scalia, K. H. Hoi, Jenna Collier, Graham Heimberg, Hector Corrada-Bravo, Shreya Gaddam, Jan-Christian Huetter, Sören Müller, Shannon J. Turley, Dave Richmond, Aicha Bentaieb, Tommaso Biancalani
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is highly heterogeneous, with five-year survival rates dropping from $\sim$90% in localized disease to $\sim$15% with distant metastases. Disease progression is shaped not only by tumor-intrinsic alterations but also by the reorganization of the tumor microenvironment (TME). Metabolic, compositional, and spatial changes contribute to this progression, but considered individually they lack context and often fail as therapeutic targets. Understanding their coordination could reveal processes to alter the disease course. Here, we combined multiplexed ion beam imaging (MIBI) with machine learning to profile metabolic, functional and spatial states of 522 colorectal lesions with single-cell resolution. We observed recurrent stage-specific remodeling marked by a lymphoid-to-myeloid shift, stromal-cancer cooperation, and malignant metabolic shifts. Spatial organization of epithelial, stromal, and immune compartments provided stronger stratification of disease stage than tumor-intrinsic changes or bulk immune infiltration alone. To systematically model these coordinated changes, we condensed multimodal features into 10 latent factors of TME organization. These factors tracked disease progression, were conserved across cohorts, and revealed frequent multicellular metabolic niches and distinct, non-exclusive TME trajectories. Our framework MuVIcell exposes the elements that together drive CRC progression by grouping co-occurring changes across cell types and feature classes into coordinated multicellular programs. This creates a rational basis to therapeutically target TME reorganization. Importantly, the framework is scalable and flexible, offering a resource for studying multicellular organization in other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884e760c0de1e86aa10844e17508931cd572069b" target='_blank'>
              Robust multicellular programs dissect the complex tumor microenvironment and track disease progression in colorectal adenocarcinomas
              </a>
            </td>
          <td>
            Loan Vulliard, Teresa Glauner, Sven Truxa, Miray Cetin, Yu-Le Wu, R. Simon, Laura Behm, Jovan Tanevski, Julio Saez-Rodriguez, G. Sauter, Felix J. Hartmann
          </td>
          <td>2025-10-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df773895d5d3bb014e2dd4dc6276d432f2994491" target='_blank'>
              YAP1 defines an emergent, plastic population of relapsed small cell lung cancer
              </a>
            </td>
          <td>
            C. Stewart, K. Ramkumar, Runsheng Wang, Yuanxin Xi, Alexa J. Halliday, L. Diao, Qi Wang, Alejandra G Serrano, Sarah M. Groves, Simon Heeke, Azusa Tanimoto, L. Kaiser, Whitney E Lewis, Mukulika Bose, Pedro Da Rocha, L. Karacosta, Vito Quaranta, Jing Wang, Julie George, Luisa Maren, Solis Soto, Bingnan Zhang, J. Heymach, L. Byers, C. Gay
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03cc2aa2a287dcfe2c1cd1643687b026d41744b" target='_blank'>
              Single-cell and Spatial Omics Reveals Region-Specific Plasticity and Therapeutic Vulnerabilities in Metastatic High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Lai Man, Natalie Wu, Janet L. Oblinger, Dazhuan E Xin, Rohit Rao, Feng Zhang, Mike Adam, Oscar Lopez-Nunez, Sara Szabo, D. Allmen, Long-Sheng Chang, Brian D Weiss, 
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Owing to the complexity of TME components and the heterogeneity of cancer cells, the relationship between the niches of TME and prognosis in breast ductal carcinoma remains unknown. The staged characteristics of corresponding cancer cell behaviors are unclear. Our study aims to reveal spatial structures and specific cellular information of TME and cancer cells subgroups during the progression from DCIS to IDC and lymph node metastasis. Methods Single-cell sequencing, spatial transcriptomics, bulk RNA sequencing datasets were used to explore the changes in microenvironmental components and transcriptional programs of tumor cells during the progression of breast ductal carcinoma. Immunohistochemistry, multiplex immunofluorescence, flow cytometry cell cycle detection, invasion migration experiments, and WB imprinting were employed for validation. Results Analysis of TME cell type subsets revealed the accumulation of TEX, iTreg, and stress-phenotype TAM in the mammary gland in situ during the invasion process. Lymphatic metastases exhibited enrichment of nTregs and a more naïve-like CD8 T cell population. Spatial analysis and survival analysis showed that the spatial niches of CD4 TN and phagocytic-phenotype macrophages were associated with a favorable prognosis, and these niches were lost during disease progression. The proliferative subpopulation of breast ductal carcinoma was enriched in lymphatic metastatic tissues, expressing high levels of FAM111B and exhibiting intense TCA and oxidative phosphorylation metabolism. Silencing FAM111B led to cell cycle arrest, decreased invasion and migration abilities, and downregulation of core mediator genes for cuproptosis and disulfidptosis. Conclusions The stage-specific microenvironmental characteristics of breast ductal carcinoma correspond to some extent to the behavior of tumor cells. During the progression of ductal carcinoma in breast tissue, the establishment of an immunosuppressive microenvironment occurs. The microenvironmental spectrum at lymph node metastases differs somewhat, corresponding to a more enriched turnover of cancer cell proliferation and death. Inhibitors of FAM111B and inducers of cuproptosis and disulfidptosis may serve as potential therapeutic targets for proliferative subgroups. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07010-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfce9a8ff47f521350ed382190ba1004f756bb5c" target='_blank'>
              Spatiotemporal microenvironment landscape and malignant epithelial pattern transition in breast ductal carcinoma progression
              </a>
            </td>
          <td>
            Xifu Cheng, Wenjuan Zeng, Bingzhe Yin, Jiawei Gui, Hengbin Zhang, Zhenxing Lv, Simin Zhang, Yao Zhou
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/637f216aefe87456deedf3795555bf830c51467e" target='_blank'>
              A transcriptional map of human tonsil architecture: beyond the sum of (single cell) parts
              </a>
            </td>
          <td>
            Helena L. Crowell, L. Llaó-Cid, G. Frigola, Samuel Gunz, Irene Ruano, Patricia Lorden, Max Ruiz, Marta Kulis, J. Martín-Subero, Holger Heyn, E. Campo, A. Pascual-Reguant
          </td>
          <td>2025-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Background Metastatic renal cell carcinoma (mRCC) still lacks tissue-based biomarkers that predict benefit from first-line immune checkpoint inhibitor (ICI) regimens. High-resolution spatial transcriptomics can dissect the tumor microenvironment (TME) and reveal prognostic niches invisible to bulk assays, offering a path toward precision immunotherapy. Methods Formalin-fixed, paraffin-embedded samples from 12 therapy-naïve patients with mRCC treated with ICI-based combinations were profiled with 10x Genomics Visium and NanoString COSMx. Six patients contributed matched metastatic lesions. Cell-type deconvolution used robust cell type decomposition with a public single-cell RNA-seq reference. Clinical outcomes (mean follow-up 16.75 months) were correlated with spatial features. Validation employed Phase 2/3 JAVELIN Renal 101 as well as Checkmate 010/025 transcriptomic data. Results Spatial transcriptomics uncovered pronounced divergences between primary tumors and metastases in cellular composition and microarchitectural organization. Canonical RCC molecular subtypes displayed marked intrapatient and interpatient spatial heterogeneity, underscoring limitations of bulk classifications. Five recurrent immune niches emerged across samples. One niche—rich in macrophages and CD8+ T cells and defined by chemokine signaling—was closely associated with objective response in metastatic lesions yet absent or infrequent in matched primaries. A gene signature reflecting this niche stratified responders within the immuno-oncology (IO) -treated arms of JAVELIN 101 and CheckMate 010/025 trials. On the other hand, signatures derived from angiogenic tumor cell-rich niches defined responders within the comparator arms (tyrosine kinase inhibitor/everolimus) in these trials. Conclusion Metastasis-specific immune niches, rather than primary-tumor characteristics, appear decisive for ICI efficacy in mRCC. Spatial transcriptomic profiling can identify these clinically relevant microenvironments and generate transferable gene signatures, supporting biomarker-driven patient selection and trial design. Integrating spatial TME analysis into future studies may accelerate personalized immunotherapy strategies for mRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bedf8d3b886332325de79302aa00547b1e48f37" target='_blank'>
              Spatial transcriptomic profiling of metastatic renal cell carcinoma identifies chemokine-driven macrophage and CD8+ T-cell interactions predictive of immunotherapy response
              </a>
            </td>
          <td>
            Charis Kalogirou, M. Krebs, Andreas S Kunz, Oliver Hahn, Hubert Kübler, Marcel Schwinger, Arndt Hartmann, Astrid Schmieder, Markus Eckstein
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d4f1e5ea74f6620f6b4fbf1988331e79868e36" target='_blank'>
              Interpretable Deep Learning Reveals Biologically Relevant Spatial Gene Expression Patterns in Lung Tumors and their Microenvironment
              </a>
            </td>
          <td>
            Vibha R. Rao, Adrienne A Workman, Liang Lu, Xiaoying Liu, Shrey S. Sukhadia
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c81035972b65c627d1b19e7302e769d3ddd0b086" target='_blank'>
              A Single-Cell Atlas of the Breast Cancer Microenvironment Identifies Subtype-Specific Tumor-Immune Landscapes and Vulnerabilities
              </a>
            </td>
          <td>
            Liron Zisman Schachter, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35fce186f7fddf3cfc3ee6a30b68015192ce168e" target='_blank'>
              Exploiting TGF-β-mediated Stromal Programming in Homologous Recombination-Deficient Pancreatic Cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M Mummey, E. Zimmer, Dharini Srinivasan, A. Härle, Loic Moubri, Alica K Beutel, Rima Singh, M. Ekizce, M. K. Melzer, Yun Lee, Austin Silva, Luisa Härle, T. Engleitner, F. Arnold, M. Morawe, Benjamin K. Naggay, Juliane Schneider, Leonard Gilberg, Julia P. Mosler, Christina Ludwig, Chen Meng, M. Hirschenberger, V. Hunszinger, K. Kluck, Martina Kirchner, A. Volckmar, Mareike Wirth, M. S. Alhamdani, Jörg D. Hoheisel, J. Löhr, Thomas Seufferlein, Alireza Abaei, Ralf Kemkemer, Roland Rad, J. Budczies, M. Mulaw, Patrick C. Hermann, Mark Hänle, K. Sparrer, Christopher J. Halbrook, K. J. Gaulton, K. Steinestel, A. Stenzinger, Nelson Dusetti, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe40c75d4c6f7843c947dbae95bbc2601c4d486" target='_blank'>
              Multi-modal spatial characterization of tumor immune microenvironments identifies targetable inflammatory niches in diffuse large B cell lymphoma.
              </a>
            </td>
          <td>
            Yibo Dai, A. Kizhakeyil, Dai Chihara, Xubin Li, Yunhe Liu, T. P. Sainz Zuniga, Ashley Wilson, Jared Henderson, Daniil Vibe, Arman Petrosyants, Connor Jacobson, Alexander Sarachakov, K. Nomie, K. Kryukov, Aleksander Bagaev, Ayushi Chauhan, J. Westin, Christopher R. Flowers, F. Vega, Linghua Wang, Michael R Green
          </td>
          <td>2025-10-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) incidence is rising, and treatment remains challenging unless surgery is curative. Tumour heterogeneity contributes to resistance against both chemotherapy and immune checkpoint inhibitors, underscoring the need to better understand the complex tumour microenvironment (TME). While tumour models derived from cancer tissue from patients have advanced cancer research, they often fail to capture functional RCC heterogeneity and key TME components. We developed a 3D model system with a high success rate from resected tumour, retaining cancer, stromal, and immune cell populations. This system is fully compatible with advanced imaging technologies, including mass spectrometry imaging (MSI) and live-cell multiplex imaging. By integrating static spatial analysis with dynamic live-cell visualisation, our system provides unique insights into tumour heterogeneity, microenvironment metabolic crosstalk, and real-time cellular responses. Phenotypic characterization of the tumoroids showed strong histological resemblance to the original resected tissue, indicating that the tumoroids are reflective of the tumour in vivo and suitable as a representative model system. Additionally, DESI-MSI revealed distinct lipidomic profiles within patient-derived ccRCC tumoroids, capturing spatial metabolic heterogeneity reflective of the primary tissue. Lipid signatures varied across tumour regions, with phospholipid subclasses distinguishing epithelial, endothelial, and highly proliferative cell populations. Notably, non-clear cell regions exhibited reduced lipid droplet and fatty acid content, aligning with aggressive tumour phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc004e66880e4588257a07a21abe175bda8bb5d0" target='_blank'>
              Kidney tumoroid characterisation by spatial mass spectrometry with same-section multiplex immunofluorescence uncovers tumour microenvironment lipid signatures associated with aggressive tumour phenotypes
              </a>
            </td>
          <td>
            Hazem Abdullah, Greice M Zickuhr, I. Um, Alexander Laird, Peter Mullen, David J. Harrison, A. Dickson
          </td>
          <td>2025-09-16</td>
          <td>npj Imaging</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Understanding tumor heterogeneity at the resolution of individualized cell–cell communication networks (CCCNs) remains a major computational challenge in precision oncology. Existing inference methods largely rely on population-level correlation and thus fail to capture patient-specific signaling patterns across diverse cell types. To address this limitation, we developed an integrative computational framework combining the nested hierarchical Dirichlet process (nHDP) model for identifying hierarchically structured gene expression modules, with instance-specific Greedy Fast Causal Inference (iGFCI) for inferring individualized CCCNs (iCCCNs) in colorectal cancer (CRC). Applied to large-scale single-cell RNA-seq data from over 625,000 cells, our model successfully decomposed complex gene expression modules GEMs, potentially representing the cell lineage and cellular signaling states, and uncovered iCCCNs across detailed cell subtypes. We further used TCGA bulk RNA-seq data with survival data to validate the clinical relevance of these individualized gene expression module causal interactions, demonstrating their potential as robust prognostic signatures in CRC. Finally, we used principled causal inference methods to search for ligand-receptor pairs that mediate cell-cell communication. This framework enables mechanistic insights into immune evasion. Our computational method represents a significant advance toward realizing personalized oncology, enabling precise patient stratification and identification of actionable biomarkers for improved therapeutic targeting in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4e906c5fe00a93c967af7c5aabe8332c82ec34" target='_blank'>
              Inferring Personalized Cell-Cell Communication Networks in Colorectal Cancer with Individualized Causal Discovery
              </a>
            </td>
          <td>
            Aodong Qiu, Binfeng Lu, Gregory F. Cooper, Xinghua Lu, Lujia Chen
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96c08ee0ec338aa74bfce893ca0d187f48778d57" target='_blank'>
              Streamlining Spatial Transcriptomics for Human Kidney Tissue
              </a>
            </td>
          <td>
            Son Vo, Kieran Meadows, Han Do, Kevin R. Chapman, Graciela Andonegui, Daniel Muruve, Thang Pham, Justin Chun
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease with diverse outcomes, and intra-tumoral heterogeneity plays a significant role in both diagnosis and treatment. Despite its importance, the spatial distribution of intra-tumoral heterogeneity is not fully elucidated. Spatial transcriptomics has emerged as a promising tool to study the molecular mechanisms behind many diseases. It offers accurate measurements of RNA abundance, providing powerful tools to correlate the morphologies of cellular neighborhoods with localized gene expression patterns. However, the spot-based spatial transcriptomic tools, including the most widely used platform, Visium, do not achieve single-cell resolution readouts, which hinders data interpretability. In this study, we present a computational pathology image analysis pipeline (i.e., computational tissue annotation, CTA) that utilizes machine learning algorithms to accurately map tumor, stroma, and immune compartments within Visium-assayed tumor sections. Using a cohort of 23 breast tumor sections from four patients, we demonstrate that CTA can provide high-resolution annotations on the hematoxylin-and-eosin-stained images alongside the paired sequencing data, support the evaluation of deconvolution methods, deepen insights into intra-tumoral heterogeneity by increasing data analysis resolution, assist with spatially resolved intrinsic subtyping, and enhance the visualization of lymphocyte clones at single-cell resolution. The proposed pipeline provides valuable insights into the complex spatial architecture of breast cancer, contributing to more personalized diagnostics and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5fa100fe1b095f296661cac47143cc93bae15b" target='_blank'>
              Computational pathology annotation enhances the resolution and interpretation of breast cancer spatial transcriptomics data
              </a>
            </td>
          <td>
            Tianyi Li, Qiao Yang, B. Ács, E. Sifakis, Hosein Toosi, Camilla Engblom, K. Thrane, Qirong Lin, Jeff E Mold, Wenwen Sun, Ceren Boyaci, Sanna Steen, Jonas Frisén, Jens Lagergren, J. Lundeberg, Xinsong Chen, Johan Hartman
          </td>
          <td>2025-09-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9fa61bcf35b5d6c234bc5ce442e119522c741a" target='_blank'>
              Basal cell of origin resolves neuroendocrine-tuft lineage plasticity in cancer.
              </a>
            </td>
          <td>
            A. Ireland, Daniel A Xie, Sarah B Hawgood, Margaret W Barbier, Lisa Y. Zuo, Benjamin E Hanna, Scarlett Lucas-Randolph, Darren R Tyson, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Anish Thomas, Sonam Puri, C. Rudin, Joseph M Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2025-09-17</td>
          <td>Nature</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4feb4e5ba832919ee4cba91d9513cff2a02dce27" target='_blank'>
              Systematic Growth Factor Profiling Platform for 3D Tumor Models Reveals Estradiol-Responsive Cellular Mechanisms of Immunotherapy Resistance
              </a>
            </td>
          <td>
            Kwanghwan Lee, Minsung Kim, Si On Lim, Dong-Ju Shin, Yun Shin, Jung-Joo Choi, Maria Lee, Hyun Ju Kang, Jeong-Won Lee, Jin-Ku Lee
          </td>
          <td>2025-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c21e29753393c78c9a7147383291954089d02fd" target='_blank'>
              KDM5-driven transcriptional noise fuels plasticity-led awakening and relapse in paediatric cancer
              </a>
            </td>
          <td>
            Alejandro Allo Anido, Cecilia Roux, Emilia Chen, Siân Hamer, Abigail Shea, Ayeh Sadat Sadr, Christie English, Charlotte Butterworth, Harvey Che, Angella Bellini, Birgit Geoerger, Gudrun Schleiermacher, L. Chesler, Michael David Morgan, Alejandra Bruna
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386c9a2a1e499f753a4ef278dadc12b1134f2bac" target='_blank'>
              Disrupted astrocyte-endothelial crosstalk drives hemangioblastoma lesions in VHL disease
              </a>
            </td>
          <td>
            Macarena de Andrés-Laguillo, Irene Garcia-Gonzalez, Susana F Rocha, Aroa Garcia-Cabero, Sandra Ruiz-García, Luis Diago-Domingo, Aimane Danana, Lorena Cussó, Katrien de Bock, J. A. Enríquez, Rui Benedito
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d4b0eb929a81f46df1d1e2e57abece7f236b787" target='_blank'>
              PDAC tumor Immune system escape architecture: Analysis of transcriptomics of mechanisms that TME is tackling immune system in Pancreatic Ductal Adenocarcinoma Tumors
              </a>
            </td>
          <td>
            Abbas Zareei, Hossein Allahdadi
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Heterogeneity within the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment (TME), across tumor locations, and between patients largely contributes to poor treatment response and PDAC’s status as the fourth leading cause of cancer death in the United States. In order to identify suitable targets for innovative therapeutic interventions, it is necessary to understand and mechanistically test TME constituents, their infiltration, and their interaction dynamics in the context of primary and metastatic tumor locations using pre-clinical modeling. Therefore, we designed a novel 46-marker multiplex immunofluorescent staining panel for murine formalin-fixed paraffin-embedded (FFPE) tissues. This panel enables the spatial quantification of immune cell populations and activation/exhaustion states, epithelial-to-mesenchymal phenotypes, cancer-associated fibroblast heterogeneity, hematologic and lymphatic vessel infiltration, neuronal infiltration, and extracellular matrix deposition in the TME. We quantified the composition and spatial infiltration of the TME on matched murine primary and liver tumor samples from tandem orthotopic and hemi-spleen transplants of three distinct primary murine PDAC cell lines derived from the KPC model (KrasG12D; TP53R172H; PDX1-Cre; RosaEYFP). The resulting primary tumors from each cell line showed inter-tumoral heterogeneity as their distinct levels of immune infiltration had varying clustering patterns. Spatial analysis further revealed heterogeneous cancer cell MHC-I expression, immune cell exhaustion, and myeloid and lymphoid populations correlating with cell line. However, all liver nodules were encased by a barrier of immune cells separating the tumor from normal liver parenchyma irrespective of cell line. This observation reflects our recent spatial transcriptomic analysis of patient-derived matched liver and primary tumors, supporting the use of these pre-clinical models for future mechanistic investigations. Altogether, these data highlight the complexity of PDAC TMEs and the site-specific differences between primary and metastatic tumor composition.



 Christina R. Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert Welner, Julienne L. Carstens. High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B110.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335836491342c8b5df5a982f9e0790e87ffd1d16" target='_blank'>
              Abstract B110: High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity
              </a>
            </td>
          <td>
            Christina R Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a055b2099d39e961c79dc682be1c20375d7bba" target='_blank'>
              Tumor microenvironment response to neoadjuvant therapy in breast cancer: insights from single-cell and spatial omics.
              </a>
            </td>
          <td>
            Qingzhong Wu, Jiawei Yang, Da Zhang, Haowei Xu, Yuhang Yu, Yu Zheng
          </td>
          <td>2025-09-13</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by the coexistence of classical epithelial, basal, and mesenchymal cancer cell states. Targeting cancer cell plasticity—the ability of tumor cells to adapt to extrinsic pressures via non-genetic mechanisms—has emerged as a promising therapeutic strategy. In parallel, the immunosuppressive tumor microenvironment remains a major barrier to effective treatment in PDAC. Yet, whether specific cell states possess distinct functional properties that drive key tumor behaviors—such as phenotypic plasticity or immune evasion—remains poorly understood. Using genetically engineered mouse models driven by oncogenic Kras G12D and loss of Trp53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the "basal" signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we find that the basal state is a key driver of therapeutic resistance: rather than being eliminated by treatment, it persists through phenotypic adaptation in response to both KRAS-targeted therapies and cytotoxic chemotherapy. Co-elimination of the basal state alongside these therapies leads to deeper and more durable responses, underscoring its functional role in tumor persistence under therapy. In parallel, we observe that the basal state plays a central role in enforcing immunosuppression in PDAC tumors. Basal state ablation rapidly reshapes the tumor immune microenvironment, and we observe loss of cancer-associated fibroblasts, polarization of PDAC-resident macrophages toward a pro-inflammatory phenotype, and increased infiltration of T and NK cells. Cytokine profiling of basal-ablated tumors revealed a dramatic reduction in multiple immune-modulatory and immunosuppressive cytokines. Taken together, our data indicate that PDAC harbors a striking dependency on the basal state for both therapeutic resistance and immune evasion, and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.



 Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C. Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Zeynep C. Tarcan, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Dana Pe'er, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6879ea5062e58b2315581060e5992acf02efdbb" target='_blank'>
              Abstract B108: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Z. Tarcan, O. Basturk, R. Yaeger, James G Christensen, Yan Yan, E. de Stanchina, D. Pe’er, Tuomas Tammela
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4534f83212d853ecc1eca8aac314fc0a39255797" target='_blank'>
              Integrative Spatial Modelling of Cellular Plasticity using Graph Neural Networks and Geostatistics
              </a>
            </td>
          <td>
            Eloise Withnell, Cenk Celik, M. Secrier
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="It has become evident from decades of clinical trials that multimodal therapeutic approaches with focus on cell intrinsic and microenvironmental cues are needed to improve understanding and treat the rare, inoperable, and ultimately fatal diffuse intrinsic pontine glioma (DIPG), now categorized as a diffuse midline glioma. In this study we report the development and characterization of an in vitro system utilizing 3D Tumor Tissue Analogs (TTA), designed to replicate the intricate DIPG microenvironment. The innate ability of fluorescently labeled human brain endothelial cells, microglia, and patient-derived DIPG cell lines to self-assemble has been exploited to generate multicellular 3D TTAs that mimic tissue-like microstructures, enabling an in- depth exploration of the spatio-temporal dynamics between neoplastic and stromal cells. The 3D-TTA model recapitulates clinical patterns of DIPG growth, evidenced by resistance to chemotherapy, HDAC and proteasome inhibitors, as well as sensitization to the antibody-activated innate immune microenvironment including complement proteins and surrounding microglia. Multimodal fluorescence imaging platforms integrated with high-throughput omics revealed that alterations in tumor cell motility and growth in the 3D-TTA model compared to tumor cell only spheroids correlated with specific transcriptomic and proteomic changes. STAT3, ITGA5, LGALS1, SOD2, MVP, and CLIC1, associated with microenvironment signaling, DNA replication, and immune regulation, were identified as potential novel targets in the 3D model. The results indicate that the 3D TTA platform developed here represents a powerful tool for preclinical studies, paving the way for identification/validation of tissue specific biomarkers and novel drug targets, thus advancing disease management strategies for DIPG in children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b07b858c6fd1cb8139cba6229ad20ebbdfab9d" target='_blank'>
              Multicellular tumor-stromal interactions recapitulate aspects of therapeutic response and human oncogenic signaling in a 3D disease model for H3K27M-altered DIPG
              </a>
            </td>
          <td>
            Meenakshi Upreti, Astgik Petrosyan, M. Thornton, A. Hovsepyan, G. E. Fernandez, David S. Koos, S. Byrum, Samuel G Mackintosh, Jacob K Al-Husseini, Tania Porras, Joseph Ha, A. Tackett, Miqin Zhang, M. Johal, Anat Erdreich-Epstein, Susan Durham, Mark D Krieger, Ashley S Margol, Brendan H. Grubbs, T. Chambers, S. Asgharzadeh, Rex A. Moats, Peter A Chiarelli
          </td>
          <td>2025-09-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes
 TP53
 and
 RB1
 . However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yue Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is composed of a diverse and dynamic spectrum of cell types, cellular activities, and cell–cell interactions (CCI). Understanding the complex CCI within the TME is critical for advancing cancer treatment strategies, including modulating or predicting drug responses. Recent advances in omics technologies, including spatial transcriptomics and proteomics, have allowed improved mapping of CCI within the TME. The integration of omics insights from different platforms may facilitate the identification of novel biomarkers and therapeutic targets. This review discusses the latest computational methods for inferring CCIs from different omics data and various CCI and drug databases, emphasizing their applications in predicting drug responses. We also comprehensively summarize recent patents, clinical trials, and publications that leverage these cellular interactions to refine cancer treatment approaches. We believe that the integration of these CCI-focused technologies can improve personalized therapy for cancer patients, thereby optimizing treatment outcomes and paving the way for next-generation precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb307e16bbb2fc4780518da53d8301520562b427" target='_blank'>
              Cell–cell interactions as predictive and prognostic markers for drug responses in cancer
              </a>
            </td>
          <td>
            Xuehan Lu, Xiao Tan, Eun Ju Kim, Xinnan Jin, Meg L. Donovan, Jazmina L. Gonzalez Cruz, Zherui Xiong, Maria Reyes Becerra de los Reyes Becerra Perez, Jialei Gong, J. Monkman, Divya Agrawal, A. Kulasinghe, Quan Nguyen, Z. Tuong
          </td>
          <td>2025-10-08</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is classified into epithelial and mesenchymal subtypes based on transcriptional profiling, and these subtypes are associated with varied patient outcomes. Differences in the tumor microenvironment (TME), particularly in cancer-associated fibroblasts (CAFs), may contribute to this variation. scRNA-seq studies have observed heterogeneous CAF populations in PDAC, although the relationship between tumor subtype and CAF phenotype is not well understood. To investigate this relationship, we utilized epithelial and mesenchymal PDAC cell lines in a xenograft model to examine whether these subtypes influence CAF phenotypes.



 Xenograft models were generated by subcutaneously injecting BALB/c nude mice with PDAC cell lines of either the epithelial subtype (BxPC-3) or the mesenchymal subtype (MIA PaCa-2). Tumors were then harvested and dissociated for scRNA-seq to investigate the TME. Human tumor (graft) and murine stromal (host) cells were computationally separated using the Xenocell tool. Subsequent analyses focused on fibroblasts within the TME, which were subclustered to assess CAF heterogeneity between PDAC subtypes. To characterize transcriptional programs and identify CAF features, we performed high-dimensional weighted gene co-expression network analysis (hdWGCNA) and analyzed differentially expressed genes. Finally, spatial transcriptomics was conducted on tumor sections from BxPC-3 and MIA PaCa-2 xenografts to validate CAF clusters observed in scRNA-seq.



 Distinct CAF clusters were identified in the xenograft tumors, including LRRC15+ CAFs, RGS5+ CAFs, and inflammatory CAFs (iCAFs). LRRC15+ CAFs were predominant in tumors derived from the epithelial subtype (BxPC-3), while RGS5+ CAFs were enriched in tumors from the mesenchymal subtype (MIA PaCa-2). LRRC15+ CAFs exhibited a myofibroblastic CAF (myCAF) phenotype characterized by extracellular matrix production, with high expression of Lrrc15, Postn, and Fbln1. RGS5+ CAFs expressed canonical pericyte markers such as Rgs5 and Cox4i2, along with CAF-associated genes including Acta2. These CAF clusters also showed distinct expression patterns of collagens and secreted factors. In particular, Col11a1, Col12a1, and Wnt5a were upregulated in LRRC15+ CAFs, while Col4a1, Col10a1, and Tgfb1 were upregulated in RGS5+ CAFs. Spatial transcriptomics confirmed that gene modules associated with Lrrc15 and Rgs5, inferred from hdWGCNA, were differentially expressed between Acta2-positive stromal regions of epithelial and mesenchymal PDACs.



 These results suggest that epithelial and mesenchymal PDAC subtypes directly shape distinct CAF populations. Notably, the enrichment of RGS5+ CAFs in mesenchymal tumors may underlie their metastatic potential and poor clinical outcomes.



 Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, Jung Hyun Jo. Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B105.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020656bad62fdcf52b16552c5f7369030a0e1205" target='_blank'>
              Abstract B105: Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model
              </a>
            </td>
          <td>
            Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo-Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, J. Jo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) evolves within a highly interactive tumor microenvironment (TME) that shapes therapeutic response. We utilized mass cytometry to analyze over 10 million cells from 25 ESCC tumors, 24 adjacent nontumor tissues, and 23 peripheral blood samples, employing an extensive panel of 42 immune markers. The resulting atlas reveals a compartmentalized landscape with a reproducible paucity of CD4⁺ and CD8⁺ central memory T cells (TCM) in tumor sites. Reintroduction of patient-derived TCMs restored antitumor immunity in coculture assays, demonstrating their cytotoxic capacity in vitro and suggesting their potential relevance for future therapeutic exploration. Myeloid profiling identified PD-L1⁺ tumor-associated macrophages (TAMs) as correlates of clinical benefit; ex vivo PD-L1 blockade reprogrammed TAMs toward proinflammatory states, indicating pharmacological malleability. Notably, CD39⁺ tumor-infiltrating T cells were consistently associated with favorable prognosis and increased responsiveness to PD-1 blockade across cancer types. The functional inhibition of CD39 impaired cytotoxic T-cell activity, underscoring its dual role as a marker of immune dysfunction and a promising therapeutic target. Collectively, our findings provide a comprehensive immune landscape of ESCC, highlighting key immunological deficits and opportunities for targeted interventions. The insights gained underscore the potential of tailoring immunotherapies to the specific immune profiles of the TME, potentially revolutionizing treatment paradigms for ESCC patients. This study sets the stage for a more nuanced understanding and manipulation of the immune elements critical for optimizing cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c74ee06dce5a8743106e3e0c5ec6ea39727b2973" target='_blank'>
              Single-cell atlas of the esophageal squamous cell carcinoma immune ecosystem to predict immunotherapy response
              </a>
            </td>
          <td>
            Xiankai Chen, Yahui Zhao, Yuhao Wang, Xiliang Wang, Yuhao Liu, Zhihua Liu, Yin Li
          </td>
          <td>2025-10-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT The immune landscape of healthy prostate and its alterations during prostate cancer (PCa) progression remain poorly characterized. Using scRNA-sequencing and multiparametric flow-cytometry analysis, we comprehensively characterized immune cells in wild-type and PTEN(i)pe−/− mouse prostates, a model that closely recapitulates human PCa. PCa in PTEN(i)pe−/− is marked by the recruitment of tumor-associated neutrophils (TANs), which represent the dominant immune cell population and resolved into eight distinct states, Trem2+ tumor-associated-macrophages (TAMs), and exhausted CD8+ T cells. Trem2+ TAMs differ from the three main resident macrophage populations in the healthy prostate, exhibiting a strong metabolic and immunosuppressive signature, likely driven by the MIF/HIF1A-signaling axis. This study provides the first detailed characterization of immune cells in the healthy mouse prostate and reveals changes in the immune landscape associated with prostate cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb0c0592aa197777e78a6d1020c20a4157e6fc" target='_blank'>
              Characterization of the immune landscape in healthy mouse prostate and during prostate cancer progression
              </a>
            </td>
          <td>
            Despoina Pervizou, Joanna De Chiara, L. Spinelli, Maia Nestor-Martin, Lionel Chasson, Kateryna Len-Tayon, Darya Yanushko, Frédéric Fiore, Marc Bajénoff, Bernard Malissen, Daniel Metzger, Gilles Laverny, S. Henri
          </td>
          <td>2025-09-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastasis remains the principal cause of cancer-related mortality, yet its distribution across organs is far from random. Instead, tumor cells exhibit organotropism, a consistent preference for colonizing specific distant tissues, a phenomenon shaped by anatomical constraints, molecular crosstalk, and microenvironmental compatibility. Far beyond mere mechanical entrapment in vascular beds, metastatic dissemination reflects a coordinated interplay between tumor-intrinsic programs and organ-specific niches. Tumor-derived extracellular vesicles, cytokines, and matrix-remodeling enzymes actively precondition distant sites through pre-metastatic niche formation, creating permissive microenvironments primed for colonization. Simultaneously, tissue-specific immune landscapes, stromal compositions, and mechanical architectures determine the fate of disseminated tumor cells, whether they are eliminated, enter dormancy, or form macrometastases. Phenotypic plasticity, metabolic reprogramming, and immune evasion mechanisms equip subclones with the capacity to exploit these unique niches. Across cancer types, reproducible patterns of organotropic metastasis not only guide clinical surveillance and therapeutic stratification but also reveal vulnerabilities in the metastatic cascade. This review synthesizes emerging mechanistic insights across anatomical, immunological, and molecular domains to construct a comprehensive framework of organotropism, highlighting therapeutic opportunities to intercept metastasis at organ-specific checkpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7663dab540302726b6f7b6337c8bf27a6cf336" target='_blank'>
              Deciphering organotropism reveals therapeutic targets in metastasis
              </a>
            </td>
          <td>
            Jhommara Bautista, Andrés López-Cortés
          </td>
          <td>2025-10-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Biliary tract cancer (BTC) is a highly heterogeneous and aggressive gastrointestinal malignancy, marked by a high mortality rate and limited treatment efficacy. The primary contributing factors include the absence of reliable early detection methods, the anatomical intricacy of the biliary system, the inherently aggressive tumor biology, and the restricted effectiveness of systemic therapies. A profound understanding of molecular characteristics and clinically relevant emerging biomarkers is essential for advancing BTC treatment strategies. Recent developments in single-cell multi-omics technologies have enabled the analysis of genetic, transcriptomic, proteomic, and metabolomic data at the single-cell resolution, thereby uncovering the heterogeneity and complexity of tumor biology. These techniques provide critical insights into the diversity of immune cell populations within the tumor microenvironment (TME) and offer novel perspectives on tumor progression and potential therapeutic interventions. While single-cell technologies have significantly advanced the study of solid tumors, their application in BTC remains nascent, with a paucity of comprehensive reviews. This review systematically integrates single-cell genomics, transcriptomics, and epigenomics data to construct a cross-omics molecular atlas of BTC. It highlights the utility of single-cell multi-omics technologies in elucidating tumor heterogeneity, microenvironment remodeling, and clonal evolution in biliary tumors, while thoroughly analyzing their implications for clinical outcomes. Furthermore, this review explores personalized treatment strategies informed by single-cell technologies and underscores the significance of these technologies as indispensable tools for unraveling the complexity of BTC and fostering mechanism-based therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c48afe626e59e1df497d34aa627d14168d0c25d2" target='_blank'>
              Single-cell multi-omics in biliary tract cancers: decoding heterogeneity, microenvironment, and treatment strategies
              </a>
            </td>
          <td>
            Nannan Tang, Jiatong Li, Ao Gu, Meng‐Yao Li, Yingbin Liu
          </td>
          <td>2025-10-15</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. Results We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Conclusion Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00691-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Understanding the cellular origins and early evolutionary dynamics that drive the initiation of carcinogenesis is critical to advancing early detection and prevention strategies. By characterizing key molecular, cellular and niche events at the precancerous tipping point of early gastric cancer (EGC), we aimed to develop more precise screening tools and design targeted interventions to prevent malignant transformation at this stage. We utilized our AI models to integrate spatial multimodal data from nine EGC endoscopic submucosal dissection (ESD) samples (covering sequential stages from normal to cancer), construct a spatial-temporal profile of disease progression, and identify a critical tipping point (PMC_P) characterized by an immune-suppressive microenvironment during early cancer development. At this stage, inflammatory pit mucous cells with stemness (PMC_2) interact with fibroblasts via NAMPT\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\longrightarrow$$\end{document}⟶ITGA5/ITGB1 and with macrophages via AREG\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\longrightarrow$$\end{document}⟶EGFR/ERBB2 signaling, fostering cancer initiation. We established gastric precancerous cell lines and organoids to demonstrate that NAMPT and AREG promote cellular proliferation in vitro. Furthermore, in the transgenic CEA-SV40 mouse model, targeting AREG and/or NAMPT disrupted key cell interactions, inhibited the JAK-STAT, MAPK, and NFκB pathways, and reduced PD-L1 expression, which was also confirmed by western blot in vitro. These interventions delayed disease progression, reversed the immunosuppressive microenvironment, and prevented malignant transformation. Clinical validation was conducted using endoscopically resected EGC specimens. Our study provides a precise spatiotemporal depiction of EGC development and identifies novel diagnostic markers and therapeutic targets for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0410e9abe73bbef3e06750644cca2f27fd69ccca" target='_blank'>
              Spatiotemporal multi-omics analysis uncovers NAD-dependent immunosuppressive niche triggering early gastric cancer
              </a>
            </td>
          <td>
            Pingting Gao, Chunman Zuo, Wei Yuan, Jiabin Cai, Xiaoqiang Chai, Ruijie Gong, Jia Yu, Lu Yao, Wei Su, Zuqiang Liu, Shengli Lin, Yun Wang, Mingyan Cai, Lili Ma, Quanlin Li, Pinghong Zhou
          </td>
          <td>2025-09-22</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            A. Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a fundamental role in tumor progression. Cancer cells interact with their surroundings to establish a supportive niche through structural changes and paracrine signaling. Cells around transformed tumor cells contribute to cancer development, while infiltrating immune cells in this aggressive TME often become exhausted. Solid tumors, especially the most invasive types such as pancreatic ductal adenocarcinoma, are notably stiff mechanically, with cross-linking enzymes significantly affecting the survival of cancer cells in both primary tumors and metastatic sites. In this review, we highlight recent key contributions to the field, focusing on single-cell sequencing of stromal cells, which are increasingly seen as highly heterogeneous yet classifiable into distinct subtypes. These new insights enable the development of effective co-treatment approaches that could significantly enhance current and novel therapies against the most aggressive cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5047b2e8e1a1022055daa398bbad8bc791fe908c" target='_blank'>
              Microenvironment of Solid Tumors.
              </a>
            </td>
          <td>
            T. Korneenko, Nikolay B. Pestov, M. Shakhparonov, Nickolai A. Barlev
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Metastasis remains the leading cause of cancer-related death, yet the biological determinants that enable tumor cells to disseminate and colonize distant organs are incompletely understood. Emerging evidence identifies the microbiome, not merely as a bystander, but as an active architect of the metastatic cascade. Microbial communities residing in the gut, mucosal barriers, and within tumors shape metastatic progression by modulating immune surveillance, stromal remodeling, oncogenic signaling, and therapy response. Intratumoral and even intracellular microbes regulate epithelial–mesenchymal transition, angiogenesis, and immune escape, while gut-derived metabolites condition pre-metastatic niches and alter systemic immunity. Technological advances in spatial transcriptomics, single-cell multi-omics, and metagenomics have revealed a spatially organized, functionally integrated microbial ecosystem within tumors, challenging long-held assumptions of sterility in cancer biology. This review synthesizes five converging dimensions of this paradigm: microbial interactions in the metastatic tumor microenvironment; microbiome-mediated immunoediting and metastatic escape; the role of intratumoral and intracellular bacteria in dissemination; spatial-multi-omic approaches to map microbial niches; and microbial biomarkers predictive of metastasis and therapy outcomes. Collectively, these findings recast the microbiome as a critical and targetable determinant of metastasis. Deciphering the tumor–microbe–host triad holds transformative potential for biomarker development, therapeutic innovation, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15720d6be643a5b1bcd7bdc88b245af42661e241" target='_blank'>
              Microbial signatures in metastatic cancer
              </a>
            </td>
          <td>
            Jhommara Bautista, María Paula Fuentes-Yépez, Joseth Adatty-Molina, Andrés López-Cortés
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Medicine</td>
          <td>2</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02530b52b1f076b640bb9d3473208058e5838dbf" target='_blank'>
              Self-organization of tumor heterogeneity and plasticity
              </a>
            </td>
          <td>
            Carlos Pérez-González, D. Brückner, M. Di-Luoffo, Sophie Richon, R. Goswami, Meryem B Baghdadi, Florence Piastra-Facon, Neta Felsenthal, Réda Bouras, A. Fumagalli, Mirjam C van der Net, M. Gloerich, Salvatore Girardo, Jochen Guck, J. van Rheenen, J. Guillermet-Guibert, É. Hannezo, D. Vignjevic
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The tumor milieu is a dynamic ecosystem where immune cells, stromal cells, and tumor cells interact to influence tumor progression and anti-tumor immunity. Traditional experimental methods, limited to static in vitro or ex vivo analyses at specific time points, cannot fully capture the complexity and dynamic evolution of the tumor microenvironment (TME) in living organisms. Intravital microscopy (IVM), powered by advanced imaging technologies, precise labeling strategies, and optimized experimental approaches, enables real-time visualization of biological structures and cellular interactions within living animals. This review synthesizes findings from IVM-based research, focusing on the dynamic and transient interactions between tumor cells and other cell types, such as normal epithelial cells, immune cells, and stromal cells. It explores the nature of these interactions, their impact on tumor progression, and the outcomes of therapeutic interventions.Overall, we aim to provide a comprehensive resource that highlights the role of IVM in uncovering the dynamic cellular interplay within the TME and its implications for advancing tumor biological research and improving cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4692b6d4ed9fe84d83c4ff05fb8db709d7896889" target='_blank'>
              Visualizing cellular interactions: intravital imaging in tumor microenvironment
              </a>
            </td>
          <td>
            Shichao Li, Limei Liu, Juanjuan Shan, Cheng Qian
          </td>
          <td>2025-09-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Spatial tumour heterogeneity, which denotes the changes in cellular and molecular attributes across distinct locations within a tumour, significantly influences cancer diagnosis and treatment resistance. The heterogeneity of tumour cells inside a singular mass facilitates tumour development, metastasis, and the ineffectiveness of standard therapy. Comprehending the geographical distribution of tumour cells is crucial for formulating more efficient treatment regimens. Diverse methodologies are employed to investigate spatial heterogeneity, encompassing modern imaging techniques such as MRI, PET, and multiplexed imaging, alongside omics approaches including genomes, transcriptomics, and proteomics. These instruments offer insights into the tumour microenvironment and facilitate the identification of resistant subpopulations. The amalgamation of imaging and genomic data via radiogenomics has emerged as a viable methodology, providing an extensive perspective on the spatial and molecular intricacies of tumours. Principal findings reveal that spatial heterogeneity fosters medication resistance by establishing microenvironments characterised by varying oxygen levels, immunological infiltration, and genetic alterations, hence complicating the efficacy of monotherapy strategies. Hypoxic environments and immunological evasion significantly contribute to treatment resistance. Addressing geographical heterogeneity has the potential to enhance cancer treatments. By analysing the molecular and geographical characteristics of tumours, physicians can customise therapies more efficiently, minimising resistance and improving therapeutic results. This methodology signifies a vital advancement in precision medicine, providing more individualised and efficacious cancer therapies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691858c0b45e6f3db66c28fcbc5355f8258e6fea" target='_blank'>
              Spatial Tumor Heterogeneity: The Next Frontier in Understanding Cancer Resistance
              </a>
            </td>
          <td>
            Kwesi Egyin Taylor, Hycent Jacob, Tosin Ayodeji Oladosu, Godwin Kenechukwu Nwajiugo, Motunrayo Victoria Adigun, Markus Saerimam Nzunde, C. Ugo
          </td>
          <td>2025-10-18</td>
          <td>Oncology, Nuclear Medicine and Transplantology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d38817463882bea59fae7ae417ac7b28c60ff93" target='_blank'>
              Multi-omics evaluation of cell lines as models for metastatic prostate cancer
              </a>
            </td>
          <td>
            Xueying Liu, Weixing Yu, Xiuyuan Jin, Yugang Wang, Ke Liu
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic non-small cell lung cancer (mNSCLC) cells carry heterogeneity, not only among different subtypes but also within a single tumor. Most evidence suggests that mNSCLC exploits specific molecular drivers and mechanisms to maintain physiology, metabolism, and immune evasion during tumorigenesis. Genome-wide association studies also revealed particular mutations in the oncogenic drivers supporting tumor cell proliferation and survival, resulting in aggressive and drug-resistant phenotypes of mNSCLC. While significant progress has been made in understanding mNSCLC at the genetic and molecular levels, a considerable gap remains in understanding the dynamic interplay between intrinsic factors—particularly key tumor-associated cells—and tumor immune microenvironment (TIME) during metastasis. Hence, this review highlights histological and genetic characteristics, emphasizes the clinical relevance of metastasis, and the roles of tumor-associated cells in shaping the immunosuppressive tumor microenvironment (TME) in mNSCLC. Understanding these intricate features and mechanisms is crucial for identifying novel therapeutic targets and improving strategies to combat mNSCLC progression in diagnosed patients. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931263caf4b3388e1caa012daaf5acb9be29f764" target='_blank'>
              Unraveling the molecular-pathological characteristics and cellular complexity of the tumor immune microenvironment in metastatic non-small cell lung cancer
              </a>
            </td>
          <td>
            S. Bharadwaj, J. M. Mierzwicka, Lucie Vaňková, Petr Malý
          </td>
          <td>2025-09-29</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC.Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene.Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes.This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The global burden of cancer is rising, with treatment failures often due to the metastatic nature of late-stage malignancies. Circulating tumour cells (CTCs) are metastatic precursors shed from primary tumours, which survive in circulation, extravasate and colonise distant organs. The advent of high-throughput single-cell RNA sequencing (scRNA-seq) has revolutionised the investigation of transcriptomic landscape at single-cell resolution, enabling deep transcriptomic profiling, re-stratifying CTC subtypes and improving the detection of rare new subpopulations. Applications extend to understanding tumour microenvironments, characterising cellular heterogeneity, uncovering metastasis molecular mechanisms and improving prognosis and diagnostic strategies. A timeline of key milestones in CTC scRNA-seq research is also provided. Nevertheless, a knowledge gap remains due to unstandardised protocols and fragmented resources in CTC scRNA-seq research. We address this gap by proposing a 12-step CTC-specific scRNA-seq workflow to overcome methodological inconsistencies. This workflow spans the entire process from enrichment, single-cell sorting and sequencing to data pre-processing and downstream analyses, with a detailed compilation of data analysis tools. An in-depth discussion of the pros and cons of commonly used scRNA-seq tools is also included, specifically evaluating their suitability for CTC research. Additionally, emerging research frontiers, including the discovery of hybrid cells—fusion products of tumour and normal cells—and the integration of machine learning (ML) into scRNA-seq workflows, are explored. Future research should prioritise CTC scRNA-seq workflow standardisation, integrate ML-driven analysis and investigate rare and hybrid populations to advance metastasis research. This review supports these goals by guiding methods, informing tool selection and promoting data sharing for reproducibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d2d946c12a3009915b3b21748cb811af18ab2d4" target='_blank'>
              Single-cell RNA-sequencing of circulating tumour cells: A practical guide to workflow and translational applications
              </a>
            </td>
          <td>
            Francis Yew Fu Tieng, Learn-Han Lee, N. Ab Mutalib
          </td>
          <td>2025-10-06</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 The tumour microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) is comprised of heterogenous malignant epithelium and diverse immune infiltrates within a dense, fibrotic stroma. This multi-compartmental TME includes myeloid and lymphoid lineages, cancer-associated fibroblasts (CAFs), mural and endothelial cells, and tumour cells embedded in a rich extracellular matrix (ECM). Efforts to profile and perturb compartments or cellular phenotypes in isolation have often led to unintended adverse effects on neighbouring cells, compromising drug efficacy. This drives the need for an integrated, spatially-resolved understanding of the architecture and molecular features underlying the organization of the TME. To address this, we leveraged single-cell RNA sequencing (scRNAseq) data from eight distinct datasets to design three custom multiplexed imaging mass cytometry (IMC) panels against tumour, immune, and stromal targets. These were applied to three serial sections of a tissue microarray of 221 PDAC patients. Segmentation and clustering of 7.9 million cells revealed 83 distinct cell types and their functional states, including biophysical responses. Advanced image alignment across serial sections enabled the incorporation of TME-wide features and resolved eight spatially recurrent PDAC microenvironments neighbouring phenotypically distinct epithelial ducts. Two microenvironments, ECM-rich and immune-suppressed, were associated with worse overall survival and frequently co-occurred with a third, stiff matrix, characterized by stroma enriched in phospho-myosin light chain 2 expressing CAFs. In contrast, the immune infiltrated stroma microenvironment was linked to improved clinical outcomes and often co-occurred with CD105+ fibrovascular and immune infiltrated microenvironments. Matched whole-genome sequencing (n = 182/221) revealed that patients dominant in ECM-rich regions harboured 12p11.21 (KRAS) and 8q24.21 (MYC and POU5F1B) amplifications, and 17q22 (RNF43) deletions, and stiff matrix were associated with 13q33.3 (LATS2) deletion. To link spatial niches to proteomic signatures, we performed whole-slide IMC-guided laser capture microdissection and mass spectrometry, and further integrated differentially expressed proteins with transcriptomes from an external scRNAseq cohort (n = 163). ECM-rich and immune suppressed regions showed signatures of mechanically induced immune suppression, including NETosis and innate immune surveillance via pattern recognition receptors. Contrastingly, immune infiltrated microenvironments were enriched for cytotoxic activity and immune trafficking programs, while immune infiltrated stroma displayed signatures associated with oxidative stress responses. These findings define unique spatial ecosystems within the PDAC TME that are linked to specific genomic alterations, functional states and clinical outcomes, highlighting their therapeutic relevance and potential as targets to overcome the challenges of treating PDAC.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steven Gallinger, Hartland W. Jackson. Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B115.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e942fb4c97db1a6398873aeae226bf3656a085" target='_blank'>
              Abstract B115: Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Z. Lin, Edward L. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 PDAC is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells cope with poor access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells, receiving alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids from fibroblasts. However, the extent of metabolic support provided by other cell types in the PDAC TME is unexplored. We reasoned that metabolically cooperative cell types would be spatially positioned near cancer cells in these tumors, and that cooperative spatial arrangements would be enriched under conditions of relative hypovascularity. We first performed Xenium spatial transcriptomics on a cohort of 8 treatment-naive human PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. Niche analysis uncovered 17 recurrent cellular neighborhoods in the data, including an endothelial-enriched niche. We find that tumor-associated neutrophils form dense aggregates within malignant glands in tumors with low endothelial content, and form in tumor regions which are most distal from vasculature. In a separate cohort of 4 additional PDAC tumors, the expression of neutrophil chemokines CXCL1/2/3/5 was highest in a subpopulation of malignant cells distant from endothelium. These data suggest metabolic stress secondary to poor vascularity drives recruitment of neutrophils via the CXCL1/2/3/5 axis. Indeed, when exposed to Tumor Interstitial Fluid Media (TIFM), cancer cells upregulated the transcription and/or secretion of these cytokines and G-CSF. Since the formation of neutrophil aggregates is driven by cancer cell nutrient derangement, we hypothesized that cancer cells could cooperate metabolically with neutrophils to alleviate nutrient stress driven by poor vascularity. Neutrophil co-culture promoted cancer cell proliferation under TIFM, but not RPMI, while under lethal combination TIFM and serum starvation, co-culture enabled enhanced survival of cancer cells. Lastly, to understand what metabolic processes could be fueled by neutrophil aggregates in pancreatic cancer, we designed a fully custom spatial transcriptomics panel targeting ∼360 metabolic enzymes and profiled 4 additional PDAC cases, performing dispersed whole-transcriptome scFFPEseq on an adjacent tissue section. We integrated data across the spatial and dispersed modalities to enable spatial projection of whole-transcriptome-derived information. Cancer cells located near neutrophil aggregates had significantly enhanced expression of vitamin B6 salvage pathway, suggesting that one critical shared metabolite between neutrophils and cancer cells is the B6 precursor pyridoxine. In total, we have deployed multiple spatial and single cell RNA profiling technologies to uncover a novel node of metabolic crosstalk in pancreatic cancer between neutrophils and cancer cells. Future work is needed to confirm transfer of specific metabolites and the downstream sequelae of their provision for tumor growth.



 Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M Celeste. Simon. Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec70e5707c093ceceed418e17d0f9e9a3b6c60c" target='_blank'>
              Abstract B127: Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M. C. Simon
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are critical mediators of tumor-stroma crosstalk in pancreatic ductal adenocarcinoma (PDAC). While bulk and single-cell transcriptomic studies have uncovered CAF heterogeneity, the spatial organization of CAF subtypes and their physical interactions with tumor epithelial subtypes remain incompletely understood. Here, we integrate bulk, single-cell, and spatial transcriptomic datasets to systematically interrogate the molecular, cell, and spatial interactions between tumor-promoting CAFs (proCAFs), tumor-restraining CAFs (restCAFs), and basal-like versus classical tumor subtypes across multiple biological scales. Ligand-receptor analysis of single-cell RNA-seq data revealed robust signaling between proCAFs and basal-like tumor cells, particularly through integrin-mediated (COL11A1–ITGA11) pathways, suggesting subtype-specific stromal-epithelial crosstalk within distinct tumor microenvironments. Histopathologic validation via immunofluorescence staining confirmed that proCAF regions are preferentially located adjacent to tumor epithelium and are characterized by reactive, myxoid stroma, whereas restCAF regions exhibit a fibrous, hypocellular morphology. To further elucidate spatial relationships, we performed 10x Genomics Visium spatial transcriptomics on seven primary PDAC tumors. Application of the DeCAF classifier to Visium spots demonstrated a distinct radial gradient of proCAF expression, peaking at the tumor-stroma interface and diminishing with distance—indicative of spatial colocalization with basal-like programs. Notably, proCAF gene signatures were specifically enriched in stromal regions adjacent to basal-like tumor zones, while classical tumor regions were spatially associated with heterogeneous CAF compositions. This subtype-specific spatial organization and paracrine signaling may underlie the clinical observation that basal-like tumors are significantly more likely to co-occur with proCAF stroma (64.5%, p = 2.17e–5). Importantly, patients harboring both basal-like and proCAF subtypes exhibited the poorest clinical outcomes (median overall survival: 11.0 months), compared to those with classical/restCAF profiles (median overall survival: 30.4 months, p < 0.001). Collectively, these multidimensional data define spatially organized and functionally interactive tumor-stroma niches in PDAC. Our findings highlight a co-evolutionary relationship between proCAFs and basal-like tumor cells that contributes to an aggressive disease phenotype, and underscore the potential of spatially resolved tumor-stroma profiling for improving patient stratification and guiding therapeutic targeting of the tumor microenvironment.



 Xianlu Laura. Peng, Ian McCabe, Elena Kharitonova, Ryan Zhao, Changfei Luan, Arthi Hariharan, Jaewon Lee, Nancy Kren, Yi Xu, Joseph Kearney, Justin Su, Michelle LaBella, Susan Tsai, Brian Belt, Roheena Panni, David Linehan, Yuliya Pylayeva-Gupta, Alina Iuga, Naim Rashid, Jen Jen Yeh. Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B100.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6682e5a69c7a44f5aef0626bbff15428b91a16" target='_blank'>
              Abstract B100: Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications
              </a>
            </td>
          <td>
            X. Peng, Ian C McCabe, Elena V Kharitonova, Ryan T Zhao, C. Luan, Arthi Hariharan, Jaewon J Lee, Nancy P. Kren, Yi Xu, Joseph F Kearney, Justin Su, M. LaBella, Susan Tsai, Brian Belt, R. Panni, David Linehan, Y. Pylayeva-Gupta, Alina C Iuga, Naim U Rashid, J. J. Yeh
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in shaping immune responses and therapeutic outcomes in lung cancer, yet the diversity and functional specialization of tumor-associated macrophages (TAMs) remain poorly resolved. Here, we present a refined classification of TAM subtypes across large cohorts of cancer datasets using integrative analysis of single-cell RNA sequencing, spatial transcriptomics, and clinical datasets from lung adenocarcinoma and lung squamous cell carcinoma. By combining cell-based gene scoring with hierarchical classification, we defined 7 macrophage subtypes-each with distinct transcriptional programs and abundances. Notably, lipid-associated TAMs expand with disease progression and exhibit immunosuppressive and protumorigenic features, whereas tissue-resident macrophages decline. Spatial and survival analyses reveal that an increased lipid-associated to tissue-resident TAM ratio correlates with advanced disease and poor prognosis. Given that spatial transcriptomic assays rely on deconvolution techniques to infer cell type compositions, accurate gene expression signatures are essential, especially for fine-grained subpopulations of TAMs. Our refined subtype-specific signatures address this bottleneck and enhance the resolution of spatial mapping efforts. These findings offer new insights into macrophage heterogeneity and highlight lipid-associated TAMs as potential biomarkers and therapeutic targets in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e49c52a60b0a9da9a722e72cb8391f58dc458b" target='_blank'>
              Transcriptional states of lung cancer microenvironment reveal macrophage subtype dynamics linked to disease progression.
              </a>
            </td>
          <td>
            Duygu Keremitçi, Özlem Tuna, Aissa Houdjedj, Hilal Kazan, Yasin Kaymaz
          </td>
          <td>2025-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5994676ec22291e594d0de0ef39f5ccc905b6ab" target='_blank'>
              Integrative analysis of spatiotemporal transcriptomics delineates dynamic cell states in squamous tumorigenesis
              </a>
            </td>
          <td>
            Varsha Sreekanth, Timur Rusanov, Farah Yousef, Mark A Sanborn, Su Yeon Yeon, Vijayalakshmi Ananthanarayanan, Olga Karginova, Jalees Rehman, A. Salahudeen
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Differentiating between the two main subtypes of Inflammatory Bowel Disease (IBD): Crohn’s disease (CD) and ulcerative colitis (UC) is a persistent clinical challenge due to overlapping presentations. This study introduces a novel computational framework that employs spatial transcriptomics (ST) to create an explainable machine learning model for IBD classification. We analyzed ST data from the colonic mucosa of healthy controls (HC), UC, and CD patients. Using Non-negative Matrix Factorization (NMF), we first identified four recurring cellular niches, representing distinct functional microenvironments within the tissue. From these niches, we systematically engineered 44 features capturing three key aspects of tissue pathology: niche composition, neighborhood enrichment, and niche-gene signals. A multilayer perceptron (MLP) classifier trained on these features achieved an accuracy of 0.774 ± 0.161 for the more challenging three-class problem (HC, UC, and CD) and 0.916±0.118 in the two-class problem of distinguishing IBD from healthy tissue. Crucially, model explainability analysis revealed that disruptions in the spatial organization of niches were the strongest predictors of general inflammation, while the classification between UC and CD relied on specific niche-gene expression signatures. This work provides a robust, proof-of-concept pipeline that transforms descriptive spatial data into an accurate and explainable predictive tool, offering not only a potential new diagnostic paradigm but also deeper insights into the distinct biological mechanisms that drive IBD subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7e36d179c516512e6f46f88d6a580c0cc5c9f06" target='_blank'>
              Engineering Spatial and Molecular Features from Cellular Niches to Inform Predictions of Inflammatory Bowel Disease
              </a>
            </td>
          <td>
            M. J. T. Tan, Maria Kapetanaki, P. Benos
          </td>
          <td>2025-09-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4113c46dcadcc5e605fb92cfb97a958b41a5f2" target='_blank'>
              High-resolution spatial transcriptomics uncover epidermal-dermal divergences in Merkel cell carcinoma: spatial context reshapes the gene expression landscape.
              </a>
            </td>
          <td>
            Kuan Cheok Lei, Nalini Srinivas, Mitalee Chandra, Vahan Serobyan, Selma Ugurel, Daniel Hoffmann, T. Kervarrec, Weng-Onn Lui, Jürgen C. Becker
          </td>
          <td>2025-10-23</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74e30f87b2d3e252ed12332d86f09f04feb4c91" target='_blank'>
              CTDP: Identifying cell types associated with disease phenotypes using scRNA-seq data
              </a>
            </td>
          <td>
            Chonghui Liu, Murong Zhou, Zhidong Wang, Chen Yang, Yan Zhang, Zhongjun Jiang, Yu-Hang Yin, Zeyu Luo, Tianjiao Zhang, Guohua Wang, Lei Yuan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Tumor resident microbes are a well-recognized component of the tumor microenvironment. Microbial subcellular location within different tumor types along with their functionality remains to be determined. Bulk microbial profiling techniques, however, lack subcellular and spatial transcriptomic resolution. To address these limitations, we developed advanced quantitative fluorescent imaging methodologies that allowed for visualization of microbial cellular compartmentalization across different tumor types. Furthermore, we performed spatial microbial transcriptomics at the regional and single cell levels to interrogate microbial regulation of signaling within pancreatic cancer cells. To confirm presence of viable microbes within tumors, we performed multiplexed culturomics of patient tissue samples followed by Whole Genomic Sequencing (WGS) analysis. We used in vitro co-culture assays together with in vivo animal experiments for functional biological validation. Overall, our results identified several pathways under microbial regulation within cancer cells that can drive immune evasion through impaired antigen presentation. In summary, using multiple complimentary novel methodologies we characterize the microbial niche of tumors (MiNT) that uncovers microbial regulation of host cells signaling and reshaping of the surrounding tumor microenvironment.



 Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier Gomez, Steven B. Maron, Daniel Carvajal-Hausdorf, Erick M. Riquelme, Mark W. Hurd, Anirban Maitra, Luis Diaz, Ismet Sahin, Jared Burks, Michael P. Kim, Florencia McAllister. Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B011.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d435ae53244dd8930ff7ec47bc24ea3f53880a" target='_blank'>
              Abstract B011: Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment
              </a>
            </td>
          <td>
            Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier A Gomez, S. Maron, Daniel Carvajal-Hausdorf, Erick Riquelme, M. Hurd, Anirban Maitra, Luis A. Diaz, I. Sahin, Jared K. Burks, Michael P Kim, Florencia McAllister
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background/Objectives: Cervical Pap smears are routinely used to detect cellular abnormalities as a cervical cancer screening tool and to assess the presence of HPV for risk stratification of the disease. Here, we aimed to extend the applications of this sampling procedure by combining it with multicolor flow cytometry to characterize cell populations across cervical cancer disease stages. Methods: Cervical Pap smears from 30 patients with various disease stages ranging from normal to intraepithelial neoplasia up to treated cancers were analyzed as biofluids using multicolor flow cytometry. Individual samples were evaluated, and statistical analyses were performed over all sample stages. Cancer cell lines (CaSki, SiHa, HeLa, A549, U2OS) were examined as tumor cell controls. Results: Cervical biofluids were subdivided into cell populations according to their scattering properties and the expression of specific biomarkers: EpCAM and cytokeratin 8 for epithelial cells from tumors as well as healthy ectocervical and endocervical regions, and CD45 for immune cells. Discrimination of tumor cells was facilitated with cancer cell lines. Statistical analysis revealed that the composition of cell populations differs among disease stages, whereas treated cancer samples were consistently associated with a reduction in squamous epithelial cells and an increase in immune cells compared to normal samples. Conclusions: Herein, we identified the major cell populations in cervical biofluid samples and demonstrated that this method can detect changes in the cellular composition across different disease stages. This approach could be further exploited in cancer research and potentially serve as a companion diagnostic tool in tumor development, progression and during treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/565b70c9fdb10e2ce7ca145493da61430604f525" target='_blank'>
              Noninvasive Cell Population Profiling of Normal and Dysplastic Cervical Biofluids by Multicolor Flow Cytometry as a Promising Tool for Companion Diagnostics
              </a>
            </td>
          <td>
            Christoph Berger, Wolf Dietrich, Manuela Richter, Florian Kellner, Christian Kühne, Katharina Strasser
          </td>
          <td>2025-10-15</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. Methods We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. Results This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. Conclusions This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04173-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566fc9fd9c1a3c438b8fdc4f08cbb7f2ae09af61" target='_blank'>
              Joint analysis of expression and variation at single cell resolution by scVar
              </a>
            </td>
          <td>
            Wei Zhao, Fei Yang, Wenting Zong, Xinchang Zheng, Yiming Bao
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy with significant cellular heterogeneity, which poses a significant challenge for treatment. We integrated three single-cell RNA sequencing (scRNA-seq) datasets to construct an expression matrix, identify cell types, and assess copy number variation (CNV) scores, with a particular focus on ductal cells. Our study found that ductal cells are the primary source of malignant cells in PDAC and identified a high-malignancy subtype—ductal cell-3—where RACK1 was the most significantly differentially expressed gene. Through the analysis of public and clinical data, along with RT-qPCR and immunohistochemistry validation, we confirmed the high expression of RACK1 in PDAC tumors, which correlated with lymph node metastasis, clinical staging, histological differentiation, and prognosis. Further investigation revealed that RACK1 promotes PDAC cell proliferation, migration, and invasion. Sequencing analysis indicated that RACK1 inhibits the FOXO signaling pathway, promotes epithelial-mesenchymal transition (EMT), and facilitates the progression and metastasis of PDAC. This study provides new insights into the role of RACK1 in PDAC and its underlying mechanisms. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03679-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fac327fd52a6595ab14fc1ae00b1f2b7a0bcb1e" target='_blank'>
              Molecular mechanisms of RACK1-driven metastasis in pancreatic ductal adenocarcinoma revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Hangyu Liu, Tingxin Wang, Yuying Cui, Biao Zhang, Dong Shang, Huiyi Song
          </td>
          <td>2025-10-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pancreatic cancer, specifically pancreatic ductal adenocarcinoma (PDAC), is notorious for its aggressive nature and dismal prognosis, ranking as a leading cause of cancer-related mortality worldwide. Despite advancements in surgical techniques, chemotherapies, and novel therapeutic approaches, the overall survival rates remain bleak, primarily due to the complex and dynamic tumor microenvironment (TME) which fosters cancer progression and therapeutic resistance. Central to the TME are cancer-associated fibroblasts (CAFs), which exhibit significant heterogeneity and play multifaceted roles in tumor progression, ranging from remodeling the extracellular matrix to modulating immune responses. Recent breakthroughs in single-cell sequencing and spatial omics have unraveled the diverse functional states and spatial organization of various cell types within the TME, including distinct subtypes of CAFs which differ vastly in their influence on pancreatic cancer pathophysiology. This review delves into the intricate interactions between CAFs and pancreatic cancer cells, highlighting how metabolic reprogramming and immune regulation by CAFs contribute to the high malignancy of PDAC. By integrating single-cell and spatial resolution data, we offer new insights into the metabolic crosstalk and immune landscape remodeling driven by CAFs, establishing a foundation for targeting these stromal interactions in therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2add2bd80d02d372a76aab262dddc13647852fe6" target='_blank'>
              Metabolic and immune crosstalk between cancer-associated fibroblasts and pancreatic cancer cells
              </a>
            </td>
          <td>
            Qiyao Zhang, Z. Cao, Shangcheng Yan, Bingrong Chen, Hao Wu, Hongtao Cao, Chen Lin, Ziwen Liu
          </td>
          <td>2025-10-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Pancreatic cancer remains a lethal disease despite advances in surgery and systemic treatment in the last two decades, underscoring the urgent need to better understand its biological underpinnings. Despite remarkable advances in the molecular characterization of pancreatic ductal adenocarcinoma (PDAC), clinically actionable biomarkers remain scarce, and current treatment remains empiric. Transcriptomic subtypes such as “classical” and “basal-like” offer some prognostic value, but their ability to guide real-time treatment decisions is limited. In this review, we explore the limitations of current biomarker strategies, in particular subtype-based classifications, and argue for a functional reframing of biomarker development in PDAC, centered on patient-derived organoids (PDOs). We explore four key domains in which PDOs deepen our understanding of therapeutic response and resistance, namely, drug response phenotyping, modeling chemoresistance, incorporating tumor microenvironmental complexity through co-culture systems, and more functional profiling through proteomic and metabolomic approaches. Together, these applications move PDOs beyond static avatars of the tumor to dynamic platforms capable of capturing clinically relevant biology. As functional precision medicine gains traction, PDOs may offer a path to more tailored, responsive treatment strategies in a cancer where new options are urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243db33c2c4b51c3c745a2973296c1c4099759e6" target='_blank'>
              Functional Precision in Pancreatic Cancer: Redefining Biomarkers with Patient-Derived Organoids
              </a>
            </td>
          <td>
            C.A. Chew, Cheng Mun Wun, Yi Fang Lee, C.E. Chee, Khek Yu Ho, Glenn Kunnath Bonney
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Liver metastases represent a leading cause of mortality in pancreatic ductal adenocarcinoma (PDAC). Recent clinical evidence has revealed distinct immune ecosystems in the liver tumor microenvironment, yet mechanistic insight into the cellular and molecular mechanisms driving those diverse immune landscapes remains unclear. Here we report, computational cellular signaling analysis of single-cell RNA sequencing data from matched primary and liver metastatic samples isolated at early and late time points from the spontaneous genetically engineered mouse model KPC (Pdx1Cre; KrasLSL-G12D; Tp53LSL-R172H). This analysis uncovered several immunomodulatory axes between cancer cells-T cells and cancer cells- macrophages. Validation of these findings were performed using primary tumor cell lines, isolated from the same or littermate tumors as the scRNA-seq dataset. Bulk RNA-seq analysis of these cell lines showed a retention of cancer-derived ligands in culture. In vivo validation using the hemisplenic liver metastatic transplant model confirmed that cell lines isolated from primary tumors, which metastasized spontaneously to the liver, retained liver metastatic outgrowth potential. Flow cytometry and multiplex cytokine profiling of isolated cell lines, non-involved liver, and liver tumors revealed distinct immune population shifts within the metastatic niche and confirmed cancer cell derived immunomodulatory cytokines. This study highlights the interplay between cancer-intrinsic signaling and the immune landscape in PDAC liver metastases and supports future mechanistic exploration.



 Ayushi Mandloi, Jace Baines, Meet Patel, Christina R. Larson, Robert S. Welner, Julienne L. Carstens. Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A081.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fce5d9cdc25fe3d365bf9aa1c45eddedfbe9968" target='_blank'>
              Abstract A081: Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth
              </a>
            </td>
          <td>
            Ayushi Mandloi, Jace Baines, Meet Patel, Christina R Larson, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The ability of metastatic cancer cells to invade distant tissues requires them to cross a variety of tissue boundaries, each posing distinct structural and biochemical challenges. In particular, the boundary between dense, extracellular matrix (ECM)-rich tumor tissue and surrounding stromal tissue is associated with phenotypic changes in MDA-MB-231 breast cancer cells following transmigration, including increased invasiveness and aggressiveness. It remains unclear whether this transition arises from selective, permissive filtering of pre-existing subpopulations, such as cancer stem cells, or an instructive response of the entire cell population. Here, by combining single-cell migration analysis, heterogeneity analysis of cell proliferation, and computational modeling, it is demonstrated that tumor-tissue boundaries act as instructive interfaces. Using an established 3D fibrillar collagen I matrix model of interfaces, it is shown that all cells can transmigrate the interface with no evidence of selective filtering of subpopulations. Proliferation heterogeneity remains unchanged between transmigrated and non-transmigrated cells, further supporting an instructive mechanism. Simulations confirm that the interface instructively modulates cell behavior. These results indicate that tissue boundaries can reprogram cancer cell phenotypes, representing a potentially targetable mechanism in metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aace3b21099f4dc0398d90f3d8cd70a0bc5a03df" target='_blank'>
              Tumor-Tissue Boundaries as Instructive Interfaces in Breast Cancer Cell Invasion.
              </a>
            </td>
          <td>
            Cornelia Clemens, T. Zerjatke, Andrine Frank, Hannah Trampert, Nataliia Kotsiuba, I. Glauche, T. Pompe
          </td>
          <td>2025-09-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive disease with one of the lowest five-year survival rates of all cancers. Low grade, premalignant pancreatic intraepithelial neoplasias (PanINs) are common in the general population, but despite expression of mutant KRAS, the vast majority of these lesions will never progress to cancer. Understanding how additional genetic or epigenetic changes as well as changes to the tumor microenvironment contribute to PanIN to PDAC progression is critical and has the potential to reveal mechanisms of interception for this deadly disease. Our previous work identified the prolyl isomerase PIN1 as a critical factor in the tumor microenvironment for growth of a subset of pancreatic cancer cell lines. Loss of PIN1 in the tumor microenvironment restricted in vivo growth of multiple KPC cell lines. We identified distinct fibroblast phenotypes in the absence of microenvironmental PIN1 expression or function. These changes in fibroblast state drive significant changes in the extracellular matrix and growth factor expression, both of which impact the crosstalk to and behavior of the neoplastic cells. We are continuing to use conditional PIN1 knockout mice to interrogate the specific cell types responsible for the observed tumor-restrictive phenotype and are using single cell sequencing to better understand how changes to individual cell types affect multiple cell populations to inhibit tumor growth. In addition, we are comparing the fibroblast phenotypes observed in our mouse models to those found in tumor-promoting versus tumor-restrictive microenvironments in human tissues. These tissues include autopsy pancreata with low grade PanIN lesions as well as PDAC resection specimens with low and high grade PanINs. We are creating organoid and cell line models of epithelial and stromal cells from normal human pancreata to support complex co-culture engineered models of early disease progression. By integrating across mouse and human model systems, we strive to understand the crosstalk that permits progression of pancreatic cancer and to identify targets to intercept this process.



 Tana Gazdik, Madeline Kuhn, Chloe Bowman, Eric Carlson, Kyra Lindley, Ellen M. Langer. Interrogating microenvironmental changes that impact early pancreatic cancer development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4475acff7b0cbed38743a8484c9c3b1b2adb82c" target='_blank'>
              Abstract A031: Interrogating microenvironmental changes that impact early pancreatic cancer development
              </a>
            </td>
          <td>
            Tana Gazdik, Madeline R Kuhn, Chloe L Bowman, Eric J Carlson, Kyra A Lindley, Ellen M. Langer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genetically engineered mouse models have advanced cancer research but often fail to capture key features of certain human tumors. Rats, with distinct physiology and tumor biology, offer a powerful alternative, yet their use has been constrained by technical barriers to genome editing. Here, we report efficient somatic genome editing in rats, enabling both Indel and substitution mutations. We then apply this approach to model estrogen receptor (ER)-positive breast cancer, which accounts for ∼70% of human cases but remains poorly represented in mice. The resulting rat tumors reproduce hallmarks of human ER+ breast cancer, including ductal histology, hormone responsiveness, and immune-microenvironmental features. By contrast, identical genetic alterations in mice failed to yield ER+ tumors, underscoring critical species differences in tumorigenesis. Together, this work establishes a versatile platform for rapid generation of clinically relevant rat tumor models, opening new avenues to dissect tumor biology, therapeutic response, and immune interactions in previously inaccessible cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19abed2134c7009bf0720ace89b27aa8404fff98" target='_blank'>
              Rat somatic genome editing enables ER+ breast cancer modeling
              </a>
            </td>
          <td>
            Wen Bu, Tobie Lee, M. Anurag, Yunfeng Ding, Ruixin Xu, Lillian He, Alexandria Z. Bu, C. Nagi, Carolina Gutierrez, S. Hilsenbeck, Chonghui Cheng, Bing Zhang, Shixia Huang, Jianming Xu, C. K. Osborne, Arun Sreekumar, Eric Chang, Chad J. Creighton, X. Zhang, Yi Li
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide. However, the spatial and temporal dynamics underlying its development remain poorly characterized. This study employs spatial transcriptomics (ST) to investigate the progression of intestinal tumors in APC Min/+ mice across multiple time points. We identified distinct transcriptional profiles between tumor and normal tissues, resolving six major cell types through integrated dimensionality reduction and pathological annotation. Pseudo-time trajectory analysis revealed increased expression of MMP11 and MYL9 in later stages of tumor progression. Analysis of human CRC cohorts from the TCGA database further confirmed that high expression of these genes is associated with advanced clinical stages and promotes tumor proliferation and invasion. Temporal gene expression dynamics indicated enrichment of cancer-related pathways concurrent with suppression of lipid and amino acid metabolism. Notably, genes in the DEFA family were significantly upregulated in normal tissues compared to tumor tissues. Functional validation showed that DEFA3 inhibits colon cancer cell migration and proliferation in vitro. These demonstrate the value of ST in resolving spatiotemporal heterogeneity in CRC and identify both MMP11/MYL9 and DEFA3 as potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a441608f72da7375d3ddfdc034535ffa428b9ffc" target='_blank'>
              Investigating the Development of Colorectal Cancer Based on Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhaoyao Qi, Guoqing Gu, Huanwei Huang, Beile Lyu, Yibo Liu, Wei Wang, Xu Zha, Xicheng Liu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648afec99f4d26f113081261fc3d480301dc1dab" target='_blank'>
              Transcriptional reprogramming of tumor-infiltrating T cells during PD-1 blockade revealed through gene regulatory network and trajectory inference in squamous cell carcinoma
              </a>
            </td>
          <td>
            Roger Casals-Franch, Jordi Villà-Freixa, Lara Nonell, Adrián López García de Lomana
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) arises from precursor lesions, the most common of which are Pancreatic Intraepithelial Neoplasia (PanIN). PanINs are microscopic and cannot be detected in live individuals; as such, PanIN characterization has occurred prevalently in the context of cancer-bearing pancreata (adjacent normal) rather than in healthy tissue. Recently, through a unique collaboration with Gift of Life Michigan, we showed that PanIN prevalence is over 60% across age groups in donor healthy tissues, with high prevalence even in young individuals. Given PanIN prevalence and the rarity of pancreatic cancer, we hypothesized that the majority of PanIN is not fated to progress to malignancy. We postulated that a combination of neoplastic cell-intrinsic factors and components of the microenvironment controls progression to malignancy. We utilized spatial transcriptomics using the Visium platform (n = 14) and scRNASeq (n = 45) to dissect gene expression profiles and characterize spatial domains in donor healthy, adjacent-normal, and tumor tissues. We developed a spatial domain identification workflow that integrates spatial transcriptomics with the scRNASeq atlas, leveraging cell-type deconvolution, spatially informed clustering and pathology annotation. This approach allowed us to study the co-evolution of epithelial and stromal cells over the progression of pancreatic cancer. We were able to delineate distinct epithelial spatial domains—including Acinar, Ductal, and PanIN structures—in both donor and tumor tissues, as well as tumor-specific regions such as glandular tumor and poorly differentiated tumor. Additionally, we identified key stromal niches, including immune-rich regions such as tertiary lymphoid structures (TLS), plasma cell clusters, and macrophage-enriched zones, alongside non-immune fibrotic domains. We began by analyzing the epithelial domains and observed that while all acinar and ductal domains formed transcriptionally distinct clusters, neoplastic cells formed a continuum from sporadic healthy donor-associated PanINs to well- and poorly-differentiated tumors. We identified key genes that exhibit expression changes throughout the progression trajectory. Secondly, we analyzed characteristics of the stroma surrounding sporadic lesions, well-differentiated, and poorly-differentiated tumors. We observed key differences in fibroblasts between healthy donor-PanIN and tumorigenic regions, as fibroblasts surrounding donor-associated PanIN lacked known cancer-associated fibroblast (CAF) markers such as Alpha-Smooth Muscle Actin (α-SMA). We also observed distinct differences in immune cells, where there were more abundant plasma and CD8+ T-cells surrounding donor-associated PanIN, while tumorigenic regions showed macrophage enrichment. We then performed immunostaining and confocal imaging to validate the defined markers and specific cell populations. Our work defines the PanIN microenvironment as distinct from that of PDAC and maps the co-evolution of cancer cells and stroma over the course of carcinogenesis.



 Ahmed M. Elhossiny, Jude Okoye, Padma Kadiyala, Alexander Bray, Jamie Mills, Hannah Watkoske, Yaqing Zhang, Jiaqi Shi, Arvind Rao, Elana J. Fertig, Eileen S. Carpetner, Timothy Frankel, Marina Pasca di Magliano. Spatially Resolved Insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A052.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f4f9e15e43361f1e76f4bf1e3cc3e1d04354d0" target='_blank'>
              Abstract A052: Spatially Resolved Insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, J. Okoye, Padma Kadiyala, Alexander Bray, Jamie N. Mills, Hannah R Watkoske, Yaqing Zhang, Jiaqi Shi, Arvind Rao, E. Fertig, Eileen S. Carpetner, Timothy L. Frankel, Marina Pasca di Magliano
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d62c6090571c67ea1645ed1df6849f539cf798b" target='_blank'>
              Molecular and cellular dynamics of measurable residual disease progression in myelodysplastic syndromes
              </a>
            </td>
          <td>
            Yu-Hsiang Chen, Timothy Bi, Minhang Xu, Jaehyun Lee, , Mengchuan Zhuang, A. Velarde, D. McCurry, Hui Yang, G. Al-Atrash, Gabriela Rondon, E. Shpall, F. Jelloul, F. N. Cervoni-Curet, Sassine Ghanem, Celine Kerros, Priya Koppikar, J. Molldrem, G. Garcia-Manero, Ankit Patel, Pavan Bachireddy
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/466091ce59415bf8e8d644d95fdb8949a53fb96b" target='_blank'>
              Cancer-associated fibroblasts drive metabolic heterogeneity in KRAS-mutant colorectal cancer cells
              </a>
            </td>
          <td>
            Elizabeth Elton, Niki Tavakoli, Handan Cetin, Stacey D. Finley
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8e063a68a88a96c8de0cef9efb9254b78c4b6" target='_blank'>
              A cellular and spatial atlas of TP53-associated tissue remodeling defines a multicellular tumor ecosystem in lung adenocarcinoma.
              </a>
            </td>
          <td>
            William Zhao, Thinh Nguyen, Atharva Bhagwat, Akhil Kumar, Bruno Giotti, Benjamin Kepecs, Jason L. Weirather, N. Mahadevan, Åsa Segerstolpe, Komal Dolasia, Jamshid Abdul-Ghafar, Naomi Besson, Stephanie M Jones, Brian Y. Soong, Chendi Li, S. Vigneau, M. Slyper, Isaac Wakiro, Mei-ju Su, K. Helvie, Allison Frangieh, Judit Jané-Valbuena, Orr Ashenberg, Mark Awad, Asaf Rotem, Raphael Bueno, O. Rozenblatt-Rosen, K. Pfaff, Scott J. Rodig, Aaron N. Hata, Aviv Regev, B. Johnson, A. Tsankov
          </td>
          <td>2025-10-07</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Organoid technology has emerged as a revolutionary tool in cancer research, offering physiologically accurate, three-dimensional models that preserve the histoarchitecture, genetic stability, and phenotypic complexity of primary tumors. These self-organizing structures, derived from adult stem cells, induced pluripotent stem cells, or patient tumor biopsies, recapitulate critical aspects of tumor heterogeneity, clonal evolution, and microenvironmental interactions. Organoids serve as powerful systems for modeling tumor progression, assessing drug sensitivity and resistance, and guiding precision oncology strategies. Recent innovations have extended organoid capabilities beyond static culture systems. Integration with microfluidic organoid-on-chip platforms, high-throughput CRISPR-based functional genomics, and AI-driven phenotypic analytics has enhanced mechanistic insight and translational relevance. Co-culture systems incorporating immune, stromal, and endothelial components now permit dynamic modeling of tumor–host interactions, immunotherapeutic responses, and metastatic behavior. Comparative analyses with conventional platforms, 2D monolayers, spheroids, and patient-derived xenografts emphasize the superior fidelity and clinical potential of organoids. Despite these advances, several challenges remain, such as protocol variability, incomplete recapitulation of systemic physiology, and limitations in scalability, standardization, and regulatory alignment. Addressing these gaps with unified workflows, synthetic matrices, vascularized and innervated co-cultures, and GMP-compliant manufacturing will be crucial for clinical integration. Proactive engagement with regulatory frameworks and ethical guidelines will be pivotal to ensuring safe, responsible, and equitable clinical translation. With the convergence of bioengineering, multi-omics, and computational modeling, organoids are poised to become indispensable tools in next-generation oncology, driving mechanistic discovery, predictive diagnostics, and personalized therapy optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c3effe5e0ade80f8d6b3e4aa8e2c7994edd86eb" target='_blank'>
              Organoids as Next-Generation Models for Tumor Heterogeneity, Personalized Therapy, and Cancer Research: Advancements, Applications, and Future Directions
              </a>
            </td>
          <td>
            Ayush Madan, Ramandeep Saini, Nainci Dhiman, Shu-Hui Juan, M. K. Satapathy
          </td>
          <td>2025-10-08</td>
          <td>Organoids</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="One of the main characteristics of solid tumors, such as melanoma, is an acidic tumor microenvironment. Due to dysregulation of the cancer cell metabolism and an increased production of acidic metabolites, the tumor acidifies its microenvironment. We hypothesize that this has a strong impact on tumor heterogeneity and the formation of phenotypic subpopulations. Cell culture experiments are usually carried out at a physiological pH of 7.4. Here, we show that long-time acidosis results in the formation of a senescent subpopulation in melanoma cells. Interestingly, after reintroduction to physiological pH, these cells lose their senescence-associated attributes. We isolated this subpopulation using β-galactosidase-dependent C12FDG staining and FACS. Live cell imaging, microscopy, and RNA sequencing analysis revealed that these apparent senescent cells show an increased migratory activity. With this study, we demonstrate that the acidic tumor microenvironment drives tumor plasticity in melanoma and results in the formation of a small subpopulation, which promotes metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a15918dc412942e4628009e724e226c928f4b6e" target='_blank'>
              Acidic melanoma microenvironment selects for a senescence-like but also migratory-active subpopulation driving metastatic disease
              </a>
            </td>
          <td>
            Chafia Chiheb, Stefan Fischer, Zubeir El Ahmad, Ingmar Henz, Ines Böhme-Schäfer, Melanie Kappelmann‐Fenzl, A. Bosserhoff
          </td>
          <td>2025-10-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e57eb64936bf914f410576d3851ca5828a633294" target='_blank'>
              The proteomics and phosphoproteomics landscape of melanoma under T cell attack
              </a>
            </td>
          <td>
            G. Franciosa, Agnete W. P. Jensen, Ana Martínez-Val, Ilaria Piga, Marco Donia, J. Olsen
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance.
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0658c9ad7362921867b92e862ee8ac79e89e2e69" target='_blank'>
              Amoeboid-Mesenchymal Transition and the Proteolytic Control of Cancer Invasion Plasticity
              </a>
            </td>
          <td>
            Adam W. Olson, Jonathan Li, Xiao-Yan Li, Lana King, Kalins Banerjee, Atticus J. McCoy, M. Gondal, A. Chinnaiyan, D. El-Ashry, Evan T. Keller, Andrew J. Putnam, Stephen J. Weiss
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits profound spatial heterogeneity driving therapeutic resistance, while the role of cancer-associated fibroblasts (CAFs) in orchestrating immunosuppressive niches remains incompletely defined. This study integrates single-nucleus RNA sequencing (snRNA-seq) and spatial transcriptomics (stRNA-seq) to map the cellular and molecular landscape of HCC. snRNA-seq identifies key cell populations-including fibroblasts, T_NK cells, and endothelial cells-using canonical marker genes. Spatial transcriptomics maps gene expression across tumor regions (core, invasive front, stroma) via the robust cell type decomposition (RCTD) algorithm. Immunofluorescence validates collagen deposition and POSTN spatial distribution, confirming T-cell exclusion patterns. The analysis identifies hypoxic metabolic myofibroblasts (hmmyCAFs) as central regulators of the tumor microenvironment. hmmyCAFs enrich at the invasive front, forming collagen-rich barriers that physically exclude CD8⁺ T cells. Simultaneously, they secrete POSTN to suppress immune checkpoint signaling and drive hypoxia-mediated glycolytic reprogramming of T-cell metabolism. Clinically, hmmyCAF activity and POSTN expression correlate with reduced progression-free survival and immunotherapy resistance. This multimodal study defines hmmyCAFs as triple architects of physical immunosuppression, molecular regulation, and metabolic remodeling. By linking collagen remodeling, POSTN-mediated checkpoint inhibition, and hypoxia-driven metabolic reprogramming to clinical outcomes, hmmyCAFs and POSTN may serve as potential indicators for evaluating the efficacy of immunotherapy in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aba5ba124e6d224e40973627e744d5081476427" target='_blank'>
              Spatial Transcriptomics and snRNA-seq Expose CAF Niches Orchestrating Dual Stromal-Immune Barriers in Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Yingxue Li, Changxiang Huan, Haoting Sun, Wei Zhang, Zhen Guo, Chuanyu Li, Jia Yao, Zhiqi Zhang, Shizhe Yu, Qiongzhu Dong, Lunxiu Qin, Jinze Li, Lianqun Zhou
          </td>
          <td>2025-10-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a39d99b88fdac73c609342d555fba7b0f41448d" target='_blank'>
              Regulatory network and spatial modeling reveal cooperative mechanisms of resistance and immune escape in ER+ breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Sarthak Sahoo, M. Jolly, Jason T. George
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background Human papillomavirus (HPV)-associated oropharyngeal cancer (OPC) is increasing in prevalence, but the drivers of metastasis remain poorly understood, which impacts the ability to personalise management decisions. Much of the genomic research to date focuses on the HPV-negative population. Here, we utilise single-cell and spatial single-cell techniques to understand the drivers of metastasis. Methods Patients with HPV-positive OPC and cervical lymph node metastases treated with curative surgery had matched samples from the primary and lymph nodes collected for research. Single-cell RNA sequencing, single-cell spatial sequencing (Visium) and in-situ spatial platforms were performed. Cancer clones were delineated using inferred copy number variation. Expression phenotypes and interactions with the tumour microenvironment were compared between the metastasising and non-metastasising cancer clones. Results Individual cancer clones have varied abilities to metastasise and undergo clonal expansion in the lymph node, with only a subset of clones present in the primary expanding in the lymph node. Four mechanisms were identified as defining the metastatic phenotype, including protein translation adaptation, immunoproteasome dysfunction and immune evasion, suppression of the IFN immune response and cap-independent protein translation. Conclusions This research elucidates multiple mechanisms driving the expansion of cancer clones in HPV-positive oropharyngeal cancer. By detailing the roles of translational adaptation, immunoproteasome dysfunction, suppression of the interferon immune response and cap-independent protein translation, we provide insights into how these processes contribute to immune evasion and tumour survival. Supplementary Information The online version contains supplementary material available at 10.1186/s12916-025-04392-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fda352f2dd3f74c2b6cb392920d418565101f0" target='_blank'>
              Protein translation dysregulation and immune cell evasion mediated by IFN and immunoproteasome downregulation define metastatic clones in HPV-related cancer of the oropharynx
              </a>
            </td>
          <td>
            Venessa T. Chin, W. Muskovic, Rachael A. McCloy, Drew R Neavin, José Alquicira-Hernández, Himanshi Arora, A. Senabouth, Kavitha Krishna Sudhakar, Patricia Keith, Ellie Spenceley, Dominik Kaczorowski, Peter Floros, Brett Leavers, Julia Crawford, Richard Gallagher, Joseph E. Powell
          </td>
          <td>2025-10-14</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Biological systems exhibit emergent phenotypes that arise from the collective behavior of individual components, such as whole-organ functions that arise from the coordinated activity of its individual cells, or organism-level phenotypes that result from the functional interplay of collections of genes in the genome. We present CELLECTION, a deep learning framework that learns to associate subgroups of instances with different emergent phenotypes. We show CELLECTION enables interpretable predictions for heterogeneous tasks, including disease classification, identification of disease-associated cell subtypes, alignment of developmental stages between human model systems, and even predicting relative hand-wing indices across the avian lineage. CELLECTION therefore provides a scalable and flexible framework for identifying key cellular or genetic signatures underlying complex traits in development, disease, and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc6dcd93998e52fb62e6d82980c4261acb342888" target='_blank'>
              Predicting emergent phenotypes from single cell populations using CELLECTION
              </a>
            </td>
          <td>
            Hongru Hu, Siddhant Sanghi, Gerald Quon
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Highlights What are the main findings? Poor prognostic ligand–receptor pairs are localized in specific regions within glioblastoma tissues. What is the implication of the main finding? Our findings provide new insights into pathological cell–cell interactions and treatment for glioblastoma. Abstract Glioblastoma (GBM) is an aggressive and lethal malignant brain tumor. Cell–cell interactions (CCIs) in the tumor microenvironment, mediated by ligand–receptor (LR) pairs, are known to contribute to its poor prognosis. However, the prognostic influence of CCIs on patients with GBM and the spatial expression profiles of such LR pairs within tumor tissues remain incompletely understood. This study aimed to identify prognostic LR pairs in GBM and their intratumoral localization via multitranscriptomic analysis. The CCIs among GBM cells as well as between GBM and niche cells were comprehensively evaluated using 40,958 cells in single-cell RNA sequencing datasets. They were found to form intercellular networks in GBM by specific LR pairs, which were mainly implicated in extracellular matrix (ECM)-related biological processes. Survival analysis revealed that 13 LR pairs related to ECM biological processes contributed to poor prognosis (p < 0.05, and 95% confidence intervals > 1). Notably, our spatial transcriptomic analysis using three independent GBM cohorts revealed that the identified poor prognostic LR pairs were localized in specific regions within GBM tissues. Although the clinical importance of these LR pairs requires further investigation, our findings suggest potential therapeutic targets for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102a7c96b1dcd02ce81762c455b5d0717c9c4822" target='_blank'>
              Integrated Spatial and Single-Cell Transcriptomics Reveals Poor Prognostic Ligand–Receptor Pairs in Glioblastoma
              </a>
            </td>
          <td>
            Makoto Yoshimoto, Kengo Sugihara, Kazuya Tokumura, Shohei Tsuji, E. Hinoi
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The metastatic tumor microenvironment (TME) is a highly dynamic and heterogeneous ecosystem that plays a critical role in promoting cancer cell colonization, immune escape, and resistance to therapy. Recent advances in multi-omics technologies—including genomics, transcriptomics, epigenomics, proteomics, and metabolomics—have enabled a systems-level understanding of the molecular reprogramming that occurs in the TME following metastasis. In this review, we systematically summarize emerging findings from recent multi-omics studies that dissect cellular composition, signaling pathways, immune landscape, and metabolic rewiring within the metastatic TME. We highlight key molecular signatures and intercellular interactions that drive metastatic progression and therapy resistance. Furthermore, we discuss how integrative multi-omics data are being leveraged to identify actionable targets and to design novel immunotherapeutic and molecular precision strategies tailored to the metastatic niche. These insights provide a scientific rationale for the development of TME-targeted approaches in the treatment of advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1202cb3e5acfc52d10d5fc247d4fca996a2d6f2e" target='_blank'>
              Multi-omics analysis of the tumor microenvironment after metastasis: advancing toward personalized immunotherapy and molecular targeted strategies
              </a>
            </td>
          <td>
            Qing Gao, Huqiang Wu, Li Duan, Guanhua Lv, Dongmei Quan
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Metastatic pancreatic adenocarcinoma is the dominant clinical presentation with a grim 3% 5-year survival rate. Over 80% of metastatic disease occurs in the liver and has the poorest outcomes. Overall, there is little mechanistic understanding of what promotes liver metastatic outgrowth and organotropism. Our recent work using spatial transcriptomics on unique matched primary tumors and metastases revealed distinct cellular ecosystems, noting reduced desmoplasia, high proliferation, spatially constrained metabolism, and heightened pro-tumorigenic myeloid infiltration and T-cell dysfunction at the invasive border of liver metastases. However, moving beyond these observations requires establishing causal links between molecular drivers and metastatic competence. Causal studies are limited by current model systems, the spontaneous genetically engineered mouse models (GEMM) that form the basis of pre-clinical studies do not sufficiently model the clinical metastatic reality with robust matched pancreas and liver tumors, and are further hampered with inconsistent metastatic rates and unpredictable progression for timed analysis. Syngeneic transplants of GEMM-derived cancer lines into wild-type mice provide a rapid pre-clinical model system of liver disease. However, the liver-metastatic rates varies between cell lines, even with identical driving mutations. We suspected that the unexplored mechanisms driving these differences in metastatic outgrowth present an opportunity to understand critical biology. Here we report the development a consistent model system of matched pancreatic and liver tumors using syngeneic cell lines with high and low tropism for liver metastatic outgrowth in C57Bl/6 mice. These lines are transplanted at low cell numbers to better allow the evolution of the site-specific tumor microenvironment and provide a reliable model system to examine both cancer-cell intrinsic and site-specific microenvironmental factors dictating liver outgrowth. Our observations of comparable pancreatic growth, successful metastatic growth in other organs (peritoneum or lung), and micro-metastatic lesions in the liver at early time points, suggests the liver-tropic differences fall within the ability of these cells to successfully outgrow in the liver microenvironment, rather than in vivo proliferative or extravasation differences. Comparison of gene expression between high and low liver-tropic cell lines identified several immune regulatory genes and a general increase in lipid metabolism, consistent with our published patient data. Finally, spatial quantifications of these lesions using a novel 46-plex murine immunotyping panel on FFPE tissues show similar suppressive immune cells at the interface of the tumor and normal liver as observed in our patient data, but evolves across differing lesion size, suggestive of a spatiotemporal progression. Altogether, this model system provides a robust, efficient pre-clinical platform to dissect spatiotemporal drivers of liver metastatic disease.



 Christina R. Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, Ateeq M. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens. Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288b7e540e23f2b8aa282e16a6f01e2dc86d80d0" target='_blank'>
              Abstract A048: Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation
              </a>
            </td>
          <td>
            Christina R Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, A. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec585f40c1c2e8296bee75c248183641228fbd0" target='_blank'>
              Macrophages display spatiotemporal specificity in intrahepatic cholangiocarcinoma and drive tumour progression via OSM and THBS1
              </a>
            </td>
          <td>
            Tingjie Wang, Ruitao Long, Zhongyu Cheng, Mi Zhang, Bing Wei, Cong Lyu, Cuiyun Zhang, Guangbo Tang, Zihang Li, Lei Li, Yida Li, Xiaofei Zhu, Chao Shi, Ke Yang, Chuanrui Xu
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
Clonal hematopoiesis of indeterminate potential (CHIP) has been associated with adverse outcomes in some solid tumor settings, but its impact on breast cancer remains unclear. We sought to investigate the genotype-specific effects of CHIP on breast cancer outcomes and the tumor microenvironment.


EXPERIMENTAL DESIGN
We examined a retrospective cohort of 125 patients with breast cancer, using targeted sequencing to identify CHIP. Metastatic events were recorded, and distant metastasis-free survival probability was analyzed. In parallel, we developed chimeric mouse models of the two most mutated CHIP genes, DNMT3A and TET2. CHIP and control mice were orthotopically injected with syngeneic breast cancer cells. Tumor growth was measured, and immune infiltrate was profiled via mass cytometry.


RESULTS
CHIP was present in 18.4% of patients. High-burden CHIP and non-DNMT3A CHIP were associated with significantly shorter distant metastasis-free survival. In vivo, mice with Tet2-CHIP developed larger primary tumors and were more likely to experience lung metastasis, while Dnmt3a-CHIP did not differ from controls. The general immune subsets observed in both CHIP models were similar, but immunophenotyping revealed clonal expansion and immune cell subset skewing specific to the Tet2-CHIP model.


CONCLUSIONS
Our findings demonstrate a genotype-specific impact of CHIP on breast cancer across human and mouse data. Further, the chimeric mouse models we generated offer a clinically relevant tool to study solid tumors in a CHIP background. This work underscores the need for further functional studies and personalized risk assessment to clearly define the impact of various CHIP genotypes on breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ebb88f9db19ddf675e686f40d924b6b992289a8" target='_blank'>
              Clonal hematopoiesis of indeterminate potential influences breast cancer outcomes in a genotype-specific manner.
              </a>
            </td>
          <td>
            S. C. Reed, Chad R Potts, Leo Y. Luo, Brad A Davidson, Riley E. Bergman, Justin D J Kemp, Ethan K Fox, Bryce A Thomas, Lauren Ha, Vani Arora, Justin Cartailler, M. E. Sanders, Violeta Sanchez, Paula I. Gonzalez-Ericsson, S. Croessmann, Paula J Hurley, Alexander G. Bick, P. Ferrell, B. H. Park
          </td>
          <td>2025-09-19</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Chimeric antigen receptor T-cell (CAR-T) therapy has emerged as a breakthrough immunotherapy for multiple cancers, yet its efficacy remains limited in glioblastoma due to tumor heterogeneity, immune-evasion of the microenvironment, and rapid T-cell dysfunction. The aim of our study is to investigate the spatially differential expression of CAR-T targets in glioblastoma. We want to focus on differential expression of CAR-T targets in tumor cell subtypes, differences between tumor core and infiltration zone as well as reactive immune zones.



 We analyzed spatial transcriptomic data from 62 glioblastoma specimens using SPATA2 software, augmented by advanced computational tools such as scVI (data integration), Cell2Location (single-cell deconvolution), graph neural networks, gene set enrichment analysis (GSEA), and spatial annotation screening (SAS). CAR-T antigens were chosen from existing published clinical and pre-clinical CAR-T cell trials.



 Spatial mapping revealed a pronounced gradient of NPC-like cells with a higher abundance of NPC-like cells in the infiltration zone and resection margins—regions critical for residual disease persistence after resection. Clinical CAR-T targets (e.g., ERBB2, PTPRZ1) were highly expressed in AC-like and OPC-like dominated niches while nearly absent in NPC-rich areas, suggesting a blind spot for current CAR-T therapies. Single-cell deconvolution further correlated NPC-like dominance to increased T-cell infiltration, while GSEA and clonal dynamics analysis of the T-cell compartment suggested downregulated functioning subclones in NPC-rich spatial neighborhoods, underscoring potential immune evasion mechanisms in these regions.



 Our study exposes a fundamental limitation of current CAR-T designs: their inefficacy against NPC-like cells that dominate the infiltration zone and therefore the post-surgical glioblastoma ecosystem. By pinpointing spatial and phenotypic resistance hubs, we provide a roadmap for refining CAR-T targets to address this gap, potentially enhancing adjuvant immunotherapy for glioblastoma. Protein-level validation of key findings is currently ongoing to further substantiate the results.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe7508f42aeb973a2f44bba592abe40968ca576d" target='_blank'>
              P04.13.B SPATIAL HETEROGENEITY OF CAR-T ANTIGEN EXPRESSION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Y. Kong, J. Zhang, F. Sahm, G. Kastner, M. Mühlbauer, E. Grabis, P. Heinrichs, J. Straehle, R. Sankowski, R. Roelz, J. Beck, D. Heiland, N. Neidert
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma (PDAC) exhibits a huge range of genomic plasticity due to prevalent genomic instability across tumors. Longitudinal whole genome sequencing of patients at primary diagnosis and metastatic disease progression have shown KRAS copy number amplification and allelic imbalance as drivers of classical to basal-like phenotypic switch, resulting in propensity for whole genome duplication, genomic instability and aggressive metastasis. High-plex spatial proteomics have revealed that primary tumors are phenotypically mixed, but the relationship between intratumorally distinct phenotypes and their genomics remain unclear as existing genomics relies on bulk captured tumors. Therefore, this drives the need to spatially resolve the intra-tumoral genomic heterogeneity associated with distinct tumor phenotypes. We leveraged whole-slide high-plex Imaging Mass Cytometry (IMC) to stain a phenotypically mixed primary tumor with 41 different markers covering the classical (AGR2, TFF1, CEACAM6) and basal (TP63, KRT5, S100A2, CAV1) phenotype, pan-immune markers, epithelial transcription factors (GATA6, FOXA2, PDX1) and stroma. Bulk whole genome sequencing (30X) reveals that this tumor had undergone losses of chr 1p, 6p, 9, 17 and 8, in addition to 12p focal amplification and whole genome doubling. To address the association of spatial phenotypes to regional genomics, IMC guided laser capture microdissection (LCM) was performed on a serial section aligned to 4 classical (C1-4), 4 basal (B1-4) and 4 hybrid-staining (H1-4) regions, followed by independent shallow whole genome sequencing (sWGS) at 2X coverage for each captured region (n = 12). Quality control, alignment and variant calling was performed utilizing standard whole-genome sequencing workflow. Integer copy number was determined through 100kb bins and segmentation. A full range of ploidy solutions (2N, 3N and 4N) was fit for each region using Absolute Copy Number Estimation (ACE) and the best fit was used for each region. We observed that in region C2, losses in many chromosomes (1p, 6, 9, 17, 18) but had no arm level amplifications, indicating a mitotic defect. At region C3, we observed a small increase in the 12p segment harboring the KRAS locus, suggesting that a sub-clonal event had occurred resulting in amplification. At region H3, a phenotypically mixed region, a strong 12p amplification was observed along with widespread genomic instability. Finally, at region B1, the whole genome duplication is observed. Our initial pilot reveals a stepwise, orchestrated genome evolution process in a phenotypically mixed primary tumor which was not obvious in bulk whole genome sequencing, highlighting the need for spatially informed regional capture of tumors to uncover phenotypically and potentially clinically important genomic alterations.



 Tiak Ju Tan, Deni Van. Sechie, Ferris Nowlan, Noor Shakfa, Elizabeth Sunnucks, Sibyl Drissler, Jenn Gorman, Michael J. Geuenich, Chengxin Yu, Grannie O. Kane, Faiyaz Notta, Rob Grant, Julie Wilson, Steve Gallinger, Kieran R. Campbell, Hartland W. Jackson. Imaging mass cytometry guided shallow whole genome sequencing for phenotype informed regional copy number profiling in a pancreatic tumor [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B113.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e0d82bf993f8cb23419e03138a40de013a09be1" target='_blank'>
              Abstract B113: Imaging mass cytometry guided shallow whole genome sequencing for phenotype informed regional copy number profiling in a pancreatic tumor
              </a>
            </td>
          <td>
            Tiak Ju Tan, Deni Van. Sechie, Ferris Nowlan, Noor Shakfa, Elizabeth Sunnucks, Sibyl Drissler, Jennifer L. Gorman, Michael J. Geuenich, Chengxin Yu, Grannie O. Kane, F. Notta, Robert C. Grant, Julie M. Wilson, Stephen Gallinger, Kieran R. Campbell, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Cancer is a deadly disease affecting millions of people worldwide. Circulating tumor cells (CTCs) represent a critical link between primary malignancies and metastasis, acting as key players in cancer dissemination, progression, and recurrence. Although rare, CTCs offer a valuable, non-invasive window into tumor biology and the evolution of disease in patients. CTCs can exist as single cells in the circulation, but some are shed and travel in larger groups, referred to as CTC clusters. These clusters possess a greater oncogenic potential compared to individual CTCs. In this review, we aim to provide insight into the dynamic biological processes underlying CTC generation, biology, and survival, with a focus on epithelial-to-mesenchymal transition (EMT) and beyond like cancer stem cells (CSCs), cellular plasticity, and senescence. A crucial aspect of CTC biology is EMT, a process that imparts cancer cells with increased motility, invasiveness, resistance to apoptosis, and the ability to intravasate and evade the immune system. Beyond EMT the cancer cells show further plasticity, allowing epithelial tumor cells to adopt mesenchymal or hybrid phenotypes, which enables adaptation to a changing microenvironment and enhances therapy resistance. Moreover, a subset of cancer cells can acquire stem cell-like properties, including self-renewal and tumor-initiating capacity. EMT, along with processes such as dedifferentiation, contributes to the generation of cancer stem cells. In recent years, studies have also highlighted the complex and paradoxical role of senescence in CTC biology. While senescence typically results in permanent cell cycle arrest, in cancer cells it may be reversible and can promote tumor cell dormancy, immune evasion, and metastatic reactivation. By exploring the connections between CTCs, EMT, CSCs, plasticity, and senescence, we aim to shed light on the unique biology of CTCs, their metastatic potential, and their contributions to tumor heterogeneity. We hope that a better understanding of these processes will help advance the development of novel biomarkers and therapeutic targets for solid tumors beyond EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99a639294b1ce88e2f796dfc138b9bd8dbb8609" target='_blank'>
              Circulating tumor cells: indicators of cancer progression, plasticity and utility for therapies
              </a>
            </td>
          <td>
            Tamás Richárd Linkner, Z. Nagy, Alexandra Kalmár, Eszter Farkas, Fruzsina Bányai, Nikolett Szakállas, István Takács, Béla Molnár
          </td>
          <td>2025-10-20</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Recurrent/metastatic head and neck squamous cell carcinoma (HNSCC) is an aggressive malignancy with a significant unmet need for enhancing immunotherapy response given current modest efficacy. Here, we perform an in vivo CRISPR screen in an HNSCC mouse model to identify immune evasion genes. We identify several regulators of immune checkpoint blockade (ICB) response, including the ubiquitin C-terminal hydrolase 5 (UCHL5). Loss of Uchl5 in tumors increases CD8+ T cell infiltration and improved ICB responses. Uchl5 deficiency attenuates extracellular matrix (ECM) production and epithelial-mesenchymal-transition (EMT) transcriptional programs, which contribute to stromal desmoplasia, a histologic finding we describe as associated with reduced anti-PD1 response in human HNSCCs. COL17A1, a collagen highly and specifically expressed in HNSCC, mediates in part Uchl5-mediated immune evasion. Our findings suggest an unappreciated role for UCHL5 in promoting EMT in HNSCC and highlight ECM modulation as a strategy to improve immunotherapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fd6c2af8dceba0dfa5c4863b05099dccf3b0af2" target='_blank'>
              In vivo CRISPR screening in head and neck cancer reveals Uchl5 as an immunotherapy target
              </a>
            </td>
          <td>
            Cong Fu, R. Saddawi-Konefka, Jordan M. Chinai, Sarah Y. Kim, Ashwin V. Kammula, Jonathan J Perera, Aiping Jiang, Payal Tiwari, Erin N Kistler, Shiqi Tang, Sarah M Luna, Kayla J. Colvin, Juan Dubrot, Seth Anderson, Rachel A Fetterman, Cun Lan Chuong, Sarah K. Lane-Reticker, Collins K. Cheruiyot, Audrey J. Muscato, Zahra Alipour, Douglas R. Adkins, Gabriel K Griffin, Bradley E Bernstein, A. Egloff, Kathleen B. Yates, R. Chernock, J. Gutkind, R. Uppaluri, R. Manguso
          </td>
          <td>2025-09-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea7d79855ee47627896db43f29cfa10fcc98ce7" target='_blank'>
              Single-cell multiomic modelling of early metastatic events promoted by the extracellular matrix.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Jun Hyeong Lee, Junho Kang, Kyeonghui Kim, Youngwon Cho, Jihyun Park, Sang-Hyun Song, J. Kwon, Young-Joon Kim, Woong-Yang Park, Tae-You Kim, Jong-Eun Park, Pilnam Kim, Jung Kyoon Choi
          </td>
          <td>2025-09-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00f377a0017ffac5cfae6274b9416d7890857b80" target='_blank'>
              Type I interferon signaling instills divergent metastatic phenotypes and immunotherapy responses
              </a>
            </td>
          <td>
            Cort B. Breuer, Marcos Labrado, Grayson E. Rodriguez, K. C. Garcia, Nathan E. Reticker-Flynn
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485b5bdcf23d6358db284d68e8dde709a579bf6a" target='_blank'>
              Cell cycle dependent variation in endocytosis drives phenotypic diversity in M. tuberculosis
              </a>
            </td>
          <td>
            Neeraja Subhash, Sandhya Krishnan Radhakrishnan, Hitakshi Vijay, Neilay Bhalerao, Sahanawaz Molla, Anton Iyer, Shaon Chakrabarti, V. Sundaramurthy
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Cancer is typically defined by uncontrolled cellular proliferation; however, tumors also contain slower-cycling cell populations influenced by diverse environmental factors, such as nutrient deprivation, modifications in the extracellular matrix, and inflammatory stimuli. Reversible dormancy or quiescence plays a critical role in tissue and embryonic stem cell function and serves as a protective mechanism against inflammatory injury. In this study, a substantial proportion of quiescent cells, identified via an established live-cell reporter, were enriched from the tumor cell population using adoptive cell therapy models in orthotopic murine pancreatic tumors. These immune-evasive, quiescent cells exhibited increased tumorigenicity, chemoresistance, and stem-like properties. Remarkably, rather than possessing intrinsic resistance, these stem-like cancer cells acquired their immune-evasive capacity by establishing a protective niche within the tumor microenvironment. RNA sequencing of quiescent, immune-evasive cancer cells post-adoptive cell therapy highlighted epiregulin (EREG)—a secreted ligand of EGFR and ERBB4—as one of the most highly upregulated transcripts. Protein and single-cell transcriptomic analyses in humans show that EREG is highly expressed in slow-cycling cells, and its expression correlates with overall survival in patients with pancreatic ductal adenocarcinoma (PDAC). Therapeutic targeting of EREG in multiple murine tumor models resulted in the depletion of quiescent stem-like cells and enhanced outcomes with CAR T-cell therapy. Collectively, these findings identify EREG as a niche factor facilitating quiescent cell evasion of CAR T-cell therapy and indicate that disruption of this niche may enhance immunotherapeutic efficacy in PDAC by depleting quiescent stem-like cells and improving response rates.



 Bryan McClellan, Qiuju Wang, William Matsui. Epiregulin, a targetable niche-promoting factor, is utilized by quiescent stem-like cells within pancreatic tumors to facilitate resistance to CAR T-cell therapy [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B075.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/384b160fd23a7f7a4f2a4e4d75f492f5c07b69fd" target='_blank'>
              Abstract B075: Epiregulin, a targetable niche-promoting factor, is utilized by quiescent stem-like cells within pancreatic tumors to facilitate resistance to CAR T-cell therapy
              </a>
            </td>
          <td>
            B. McClellan, Qiuju Wang, William Matsui
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e655b3f34a47917eb0c1eab5cc8dbd65bd366140" target='_blank'>
              P2RY2 is a purinergic immune checkpoint linking extracellular ATP to immune evasion and adaptive resistance to immunotherapy
              </a>
            </td>
          <td>
            Zhaoqing Hu, Hitoshi Matsuo, Shangce Du, Cecilia Berzain Battioni, L. Jassowicz, Rafael Carretero, Melanie Sator-Schmitt, Xiyue Zhao, Beiping Miao, Cansu Eris, Helena Engel, Mohamed A.A. Mahmoud, Elke Laport, Yanling Xiao, Ilse Hofmann, Christel Herold-Mende, Chong Sun
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Myasthenia gravis (MG) presents a clinical challenge where autoantibody titers against the neuromuscular junction fail to predict disease severity or treatment response, and thymectomy provides inconsistent benefit despite removing the source of autoreactive B cells. We hypothesized that thymic B cells acquire survival mechanisms that bypass normal tolerance checkpoints, enabling persistence independent of antigen-specific selection. Using single-cell RNA sequencing, V(D)J repertoire analysis, and spatial transcriptomics from human thymic samples, representing the largest MG atlas to date (237,661 cells), we discovered a previously unrecognized tolerance checkpoint swap in MG pathogenesis. Pathological class-switched B cells within thymic germinal centers exhibited reduced antigen presentation capacity (decreased CD74, CD1C expression) and diminished CD40 co-stimulation while upregulating TNFRSF17 to engage BAFF survival signals. This shift from T cell-dependent activation to BAFF-driven survival allowed polyclonal autoreactive B cells to persist without stringent selection and seed peripheral sites. T follicular helper cells supported this reprogramming through increased TNFSF13B and CHGB expression, the latter enabling dopamine-mediated acceleration of B cell synapses. These findings resolve longstanding clinical paradoxes in MG and identify the BAFF-BCMA axis as an actionable therapeutic target. Measuring serum BAFF and soluble BCMA could provide superior biomarkers to antibody titers, while BAFF inhibition or CD40 agonism could restore normal tolerance checkpoints, offering new therapeutic strategies for MG and potentially other autoimmune diseases. One Sentence Summary Myasthenia gravis is characterized by a tolerance checkpoint swap where pathological B cells shift from requiring T cell help and antigen-specific selection to depending on BAFF for survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2ad3c1b676c0d2ac936461e48dfb4ab521985d1" target='_blank'>
              Myasthenia thymus reprograms class-switched B cells into BAFF-dependent survivors
              </a>
            </td>
          <td>
            Yuanqing Yan, Diego Avella Patino, Yimeng Zhao, Xin Wu, G. Budinger, Ankit Bharat
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) chemotherapy regimens are largely uniform, tailored mainly to patient fitness or rare genomic alterations. This strategy fails to capitalize on the disease’s substantial molecular heterogeneity, including variations in copy number alterations, immune infiltration, stromal composition, and cancer cell phenotypes. These features are not independent; they reflect interconnected biological processes and co-evolving pathways that shape tumour progression. Multi-omic integration is therefore essential—not only to capture the complexity of PDAC biology, but also to clarify the molecular basis of tumours classified as ‘intermediate’ by current stromal and cancer taxonomies, thereby enabling rational therapeutic targeting. To this end, we performed imaging mass cytometry on three serial sections of a PDAC tissue microarray (221 resected tumours, ∼4 cores each), generating >800 multiplexed images (40–43 channels) focused on epithelial, immune, or stromal biomarkers. We profiled 76 immune and stromal cell types and states (hypoxic, proliferative, apoptotic, under tension), as well as six tumour phenotypes defined by expression of epithelial transcription factors (GATA6, FOXA2, PDX1), classical markers (AGR2, TFF1, CEACAM6), basal markers (TP63, KRT5, S100A2, CAV1), and other PDAC-associated proteins (S100A4, MMP7, MUC16). These six cancer cell types captured the classical and basal PDAC signatures, along with four discrete “intermediate” states with distinct associations to stromal heterogeneity, RNA subtype (n = 92), tumour ploidy (n = 182), and patient outcome. These phenotypes are also detectable in unmatched single-cell RNA-seq data (n = 163), though with more overlap in marker expression. Using matched 30X whole genome sequencing (n = 182), we identified mutations and copy number alterations linked to shifts in cancer and stromal cell phenotypes, raising the question of which molecular axis best informs clinical prognosis. To address this, we applied a modified version of Stabl, a Lasso-based machine learning approach, to compare across omic layers and identify features most strongly associated with overall survival. Per-modality analysis showed that models incorporating omics outperformed those based on clinical features alone, with imaging data slightly outperforming genomics. Cross-omic integration revealed several prognostically relevant copy number aberrations, fibroblast phenotypes, and cancer cell states. By consolidating information on tumour phenotypes, stromal niches, and genomic alterations, this work aims to focus future drug discovery on the most clinically impactful molecular features in PDAC.



 Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steve Gallinger, Hartland W. Jackson. Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B116.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5cd553302915b28eaa98522e24327f9aa96cbdc" target='_blank'>
              Abstract B116: Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival
              </a>
            </td>
          <td>
            Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Edward L. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Highlights Single-cell & MR link ESCC immunotherapy to immune cells. Combined single-cell datasets yield more credible results. ESCC-immune cell discovery aids subsequent experimental validation. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03751-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cc89377f616ee4461240aafc742e5dad712a1f6" target='_blank'>
              Integrated single-cell and Mendelian randomization analyses: dissecting underlying causes of varied efficacy in immune neoadjuvant therapy for esophageal carcinoma
              </a>
            </td>
          <td>
            Jiaxin Li, Sibo Meng, Ying Zhou, Yufeng Cheng
          </td>
          <td>2025-10-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) cancer associated fibroblasts (CAFs) are a complex and heterogenous group of cells and are a key component of the tumour microenvironment (TME). Our understanding of CAF functional diversity has increased in recent years, however certain aspects of their biology remain elusive, such as the spatial organisation of CAFs, their interactions with other TME compartments, and their contribution to global TME architecture. Better understanding of these will significantly accelerate therapeutic development.



 We performed multiplex immunofluorescence on tissue microarrays of resected human pancreatic cancer (n=227) with an 8-plex CAF panel and a molecular subtyping panel. After assigning bulk cell type (CAF – FAP+CK-, Epithelium – FAP-CK+, Other – FAP-CK-), we integrated CAF morphology, spatial pseudotime analysis, and TME architectural scoring in a hierarchical manner (pixel > cell > neighborhood > tissue module > patient) to define architectural subtypes of PDAC with internal validation.



 Architectural subtypes are defined as follows (name, dominant CAF, median disease-specific survival): Cohesive subtype (iCAF, 40 months) Rich in dense, cohesive, and tightly-packed epithelium. Sparse, loose stroma with a heterogenous population of iCAF-like CAFs which exhibit migration away from epithelium on spatial pseudotime modelling. Inflammatory subtype (roundCAF, 31 months) Dominated by a rich infiltrate of presumed immune cells (FAP-CK-). Highly organised with low spatial entropy. Associated with the classical subtype. CAFs have “cold” marker expression, are round in shape with a smooth surface, and exhibit migration towards epithelium on spatial pseudotime modelling. Activated subtype (Enlarged myCAF, 17 months) Dense stroma infiltrated by large myCAF-like CAFs. High spatial entropy with loss of TME compartmentalisation. Immune excluded. Strongly associated with the squamous/basal-like subtype. Fragmented subtype (Stellate myCAF, 13 months) Loose, dis-cohesive epithelium infiltrated by a CAF-dense stroma rich in myCAF-like CAFs with stellate morphology. High CAF-epithelial cell spatial contact. Associated with poor histological differentiation and poorest prognosis. Hypothesised to represent impending or established (occult) metastasis.



 Morphospatial profiling reveals striking associations between CAF shape, function, and spatial niche, and adds crucial perspective to our understanding of CAF biology beyond current methods. Combined with spatial pseudotime analysis, it offers exciting insight into CAFs in transitory states between subtypes, allowing maximum leverage of datasets which lack a true temporal dimension. In addition, architectural subtyping reveals the complex and heterogenous spatial organisation of the pancreatic TME, and the co-evolution of CAFs with these organisational changes. Integrating this with histopathological analysis lays foundations for the development of novel spatial biomarkers of prognosis, therapeutic development, and treatment response.



 Adam Bryce, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, Catherine Ficken, Ian Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, Fieke Froeling, Stephan Dreyer, David Chang. Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B121.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76e85ad242be0ed691b8aeea89c3fdd7c71569d" target='_blank'>
              Abstract B121: Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            A. Bryce, Silvia Martinelli, R. Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, C. Ficken, Ian R Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, F. Froeling, S. Dreyer, David Chang
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Phenotypic heterogeneity - distinct molecular and behavioral variations within a population -significantly influences collective invasion and tumor progression. Here, we employ a molecular approach to explore how the underlying metabolic heterogeneity in non-small cell lung cancer (NSCLC) influences invasion and pack patterning. Assessing three-dimensional (3D) pack patterning revealed invasive heterogeneity across NSCLC cell lines and patient-derived samples. Flow cytometry identified IL13RA2 as a biomarker for invasive potential, enabling isolation of subpopulations with distinct invasive characteristics. By integrating a cell surface biomarker (IL13RA2±) with mitochondrial membrane potential (TMRM), we identified and isolated three distinct subpopulations. Two-dimensional (2D) analyses revealed differences in mitochondrial polarity and transcriptional programs associated with migration and oxygen-sensitivity. In 3D, these subpopulations invaded with distinct patterns, from contiguous circular packs to structured chains. Assessments under varied oxygen tension demonstrated that oxygen availability and subpopulation metabolism together influence collective invasion patterning. When recombined at ratios recapitulating the original population, both stochastic and opportunistic cooperative dynamics emerged, dependent on subpopulation composition and oxygen levels. Our molecular approach, integrating cell surface and metabolic characteristics, enables isolation of unique subpopulations and demonstrates that phenotypic and metabolic heterogeneity, population composition, and oxygen availability collectively pattern invasion packs and drive collective invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14710731ead2715e9cfc57e6f8afb7982d2087db" target='_blank'>
              Metabolic programming defines oxygen sensitive subpopulation hierarchies and patterning in collective invasion.
              </a>
            </td>
          <td>
            Veronika Y. Matsuk, Tala O. Khatib, Landon J. Marcus, Isaac E. Robinson, Yuan Liu, Janani K Pasupathy, Mala Shanmugam, Janna K Mouw, Adam I. Marcus
          </td>
          <td>2025-09-24</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b5111159e92c7f129ff30372c9b9934bfcdb2c" target='_blank'>
              Single-cell RNA sequencing and large-scale bulk combination with machine learning reveal gastric cancer-related macrophage heterogeneity
              </a>
            </td>
          <td>
            Saisai Gong, Yu peng Zhao, Sheng Yang, Tong Liu, Chengyun Li, Yun Zhou, Geyu Liang
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/584838fc3d0ebaa26117e1c12ed3d657762b5f85" target='_blank'>
              A histology foundation model for high-resolution spatial omics prediction, tumor detection, and, clustering of spatial transcriptomics
              </a>
            </td>
          <td>
            Ning Zhang, Yahui Long, Sibo Xia, Pengzhen Jia, Yujie Jin, Siqi Chen, Jinmiao Chen, Min Li
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BRCA1 mutation is closely linked to triple-negative breast cancer (TNBC), a subtype characterized by early onset, aggressive clinical course, and marked genomic instability. Although poly ADP ribose polymerase (PARP) inhibitors have expanded treatment options, resistance driven by cancer stem cells (CSCs) and incomplete interception of premalignant progression remain pressing challenges. In a recent study, Liu et al. developed a human induced pluripotent stem cell (iPSC)-based breast cancer model that follows BRCA1‑driven tumorigenesis from mammary differentiation to invasive disease and identify S100P as a BRCA1‑repressed effector associated with cancer stemness. These findings provide a human genetic context to probe early oncogenesis and nominate the S100P–RAGE axis as a precautionary—but promising—target for risk interception.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfe5bbd7fe04bb4260117969d5b0d5e59d56a2ae" target='_blank'>
              Deciphering BRCA1-driven tumorigenesis through an iPSC-based breast cancer model
              </a>
            </td>
          <td>
            Gaoxiang Zhu, Bing Zhao
          </td>
          <td>2025-10-01</td>
          <td>Cell Transplantation</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 We present a novel integration of data-driven single-cell analysis with mechanistic mathematical modeling to predict the impact of cancer-associated fibroblasts (CAF) on pancreatic ductal adenocarcinoma (PDAC) invasion. Bioinformatic analyses of high-throughput single-cell and spatial molecular assays give us access to human tumor gene expression at the time of measurement, enabling direct characterization of the human tumor microenvironment (TME), complementing our understanding of cell behavior gained from preclinical models. Notably, applying our Bayesian non-negative matrix factorization algorithm CoGAPS to single-cell RNA-seq data from PDAC allowed us to associate a gene program of concurrent epithelial-to-mesenchymal transition (EMT) and inflammation in epithelial cells with the presence of CAFs, validated through organoid co-culture experiments. We hypothesize that this gene program represents a mechanism by which CAFs promote invasive behavior in neoplastic cells over time. Testing this hypothesis requires inference of temporal changes, which is not possible with single-timepoint data collection in genomics data. In contrast to data-driven bioinformatics, mechanistic mathematical modeling can capture processes that happen over time and make predictions about a system. Agent-based models (ABMs) are one form of such mechanistic mathematical models that are well suited for temporal modeling of cellular phenotypes. We developed new software for ABMs that abstract cells into software agents with individual states and their own rules of behavior from single-cell and spatial molecular assays for temporal inference. We apply this technique to understand how CAFs in the PDAC microenvironment support the series of phenotype transitions comprising tumor progression and invasion, by encoding the CAF-induced ability of tumor cells to migrate and a cessation of proliferation in tumor cells, induced by CAF-secreted, non-diffusable substrate as inferred from our genomics analysis. While this provides a virtual framework to test the impact of varying CAF density on tumor cell invasion, it initially lacked integration with the spatial architecture of human tumors. We then apply the ABM to human PDAC spatial transcriptomics data to forecast tumor behavior over time in a spatially-informed setting, varying the hypothesized mechanisms. This revealed that a substrate-driven mechanism results in a broad mesenchymal-like zone surrounding epithelial-like outgrowths, while a cell contact-mediated mechanism results in a single-cell-wide boundary surrounding more significant outgrowth. When we compared these to real tumor compositions, this favors the explanation that secreted CAF substrate drives EMT in PDAC neoplastic cells, which we then validate in organoids with CAF conditioned media. Our work demonstrates how integrating ABMs with genomic and spatial data enables testing of mechanistic hypotheses about tumor-stroma interactions and provides deeper insights into PDAC microenvironments, potentially informing strategies to intercept disease progression.



 Jeanette I. Johnson, Daniel R. Bergman, Tamara Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Yoseph W. Dance, Jacob T . Mitchell, Elizabeth M. Jaffee, Neeha Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, Ashley L. Kiemen, Luciane T. Kagohara, Young H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O'Brien, Elana J. Fertig. Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d57a26173bbcb8bbded169c7c20718d2482d35" target='_blank'>
              Abstract A025: Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion
              </a>
            </td>
          <td>
            J. Johnson, Daniel R Bergman, T. Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Y. Dance, Jacob T. Mitchell, Elizabeth M. Jaffee, N. Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, A. Kiemen, Luciane T. Kagohara, Y. H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O’Brien, E. Fertig
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc219a64ec3d466c88c4dfd11649123286e2032b" target='_blank'>
              Cross-Species Insights from ART-D to Uncover Evolutionarily Conserved Oncogenic Mechanisms
              </a>
            </td>
          <td>
            Yifan Guo, Jiadong Zheng, Yixin Wu, Kun Zhao, Dian Lv, Wenhan Liu, Mengling Wang, Jin-Yu Lu, Wenyan Xu, Xianping Wang, Xianjue Ma
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer mortality, with minimal improvements in 5-year survival over the past decade. While PDAC originates from premalignant lesions, only a minority of these lesions advance further into malignancy. Distinguishing which lesions are likely to progress would transform enabling timely preventative surgical intervention, but remains a key challenge. In addition to the more well-characterized genetic mutations associated with PDAC onset, recent studies suggest that inflammation in the pancreas microenvironment promotes epithelial chromatin accessibility remodeling to increase their plasticity. This plasticity is thought to enhance cellular sensitivity in response to local signaling cues, which varies drastically across PDAC tumors. We propose that the cell lineage fate of the premalignant epithelial cells is heavily influenced by their surrounding microenvironment. To investigate this, we are using the CosMx spatial transcriptomics platform to profile cellular composition heterogeneity across pancreatic tissue cores from KrasLSL-G12D; Trp53LSL-R172H; Pdx1-Cre (KPC) mouse model of PDAC. By analyzing spatial heterogeneity within the tumor microenvironment, we aim to delineate how local microenvironmental differences might influence epithelial lineage fate decisions into PDAC. Our findings will help understand the factors driving progression from early lesions to invasive cancer, supporting efforts in early PDAC detection and informing clinical strategies for surgical resection.



 Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, Crispin J. Miller. Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B118.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb056397ddb7c8e4e0421ffdba57d9fda0a6959d" target='_blank'>
              Abstract B118: Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P Morton, Crispin J. Miller
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bb8a507427a3e0cfe9ea50f66dabfc18ac940f2" target='_blank'>
              Cancer Metastasis: Therapeutic Challenges and Opportunities.
              </a>
            </td>
          <td>
            Sri Sathya Sandilya Garemilla, Manisha Choudhary Kadambala, S. C. Gampa, Swetha Chinthala, S. Garimella
          </td>
          <td>2025-10-16</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The cellular plasticity of neuroblastoma is defined by a mixture of two major cell states, adrenergic and mesenchymal, which may contribute to therapy resistance. However, how neuroblastoma cells switch cellular states during therapy remains largely unknown, and how to eradicate neuroblastoma regardless of its cell state is a clinical challenge. To better understand the cellular plasticity of neuroblastoma in chemoresistance, we define the transcriptomic and epigenetic map of adrenergic and mesenchymal types of neuroblastomas using human and murine models treated with indisulam, a selective RBM39 degrader. We show that cancer cells not only undergo a bidirectional switch between adrenergic and mesenchymal states, but also acquire additional cellular states, reminiscent of the developmental pliancy of neural crest cells. These cell state alterations are coupled with epigenetic reprogramming and dependency switching of cell state–specific transcription factors, epigenetic modifiers, and targetable kinases. Through targeting RNA splicing, indisulam induces an inflammatory tumor microenvironment and enhances the anticancer activity of natural killer cells. The combination of indisulam with anti-GD2 immunotherapy results in a durable, complete response in high-risk transgenic neuroblastoma models, providing an innovative, rational therapeutic approach to eradicate tumor cells regardless of their potential to switch cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8bfb0658591ed4d276e6df844046df62c0e809" target='_blank'>
              RBM39 degrader invigorates innate immunity to eradicate neuroblastoma despite cancer cell plasticity
              </a>
            </td>
          <td>
            Shivendra Singh, Jie Fang, Hongjian Jin, Lee-Ann Van de Velde, Andrew Cortes, Jiani Chen, Sivaraman Natarajan, E. Poon, Qiong Wu, Christopher L. Morton, Mary A. Woolard, Waise Quarni, Jacob A Steele, Jon P Connelly, Liusheng He, Rebecca Thorne, Gregory Turner, Thomas Confer, Melissa Johnson, W. Caufield, Burgess Freeman, Timothy Lockey, A. Murphy, P. J. Murray, Takashi Owa, S. Pruett-Miller, Ruoning Wang, L. Chesler, Julie R. Park, A. Davidoff, John Easton, Xiang Chen, Paul G. Thomas, Jun Yang
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a common neoplasm that carries the risk of transformation into Richter’s syndrome (RS), a highly aggressive B cell lymphoma with poor prognosis. Limited availability of animal models and cell lines hinders understanding of transformation mechanisms. Addressing this gap, we established the first in silico dynamic model of the disease. Our methodology integrates mathematical logic modeling with experimental data to identify disease drivers, mechanisms, and potential therapeutic targets. We validated the model by comparing the model’s readout with experimental data from different biological levels, such as single-cell RNA sequencing analyses and a CLL/RS patient formalin-fixed paraffin-embedded (FFPE) tissue cohort. Our analyses identified BMI1 proto-oncogene and TP53 loss as key RS progression regulators. In addition, we performed an in silico target screening to identify promising target combinations in a personalized fashion. Through the synergy of mathematical modeling with experimental readouts, our model provides a complementary approach to investigate the process of CLL transformation to RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0084b1cbd03c2692e1100a97e2d57fe76c34d4" target='_blank'>
              CLL to Richter syndrome: Integrating network strategies with experiments elucidating disease drivers and personalized therapies
              </a>
            </td>
          <td>
            Julia Maier, J. Schwab, Silke D. Werle, Ralf Marienfeld, S. Stilgenbauer, Peter Möller, Nensi Ikonomi, Hans A. Kestler
          </td>
          <td>2025-09-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Alaska Native peoples experience high colorectal cancer (CRC) incidence and mortality rates, which Alaska tribal health organizations are working to address. To reduce the high rates, it is important to understand the biological mechanisms that drive risk and survival outcomes. The tumor microenvironment (TME) comprises diverse cell types, including cancer, stromal, and immune cells, whose composition and spatial organization may impact prognosis and disease progression. We performed spatial proteomics profiling on tissue microarrays from 97 Alaska Native patients with CRC using Akoya Biosciences’ PhenoCycler-Fusion system. We included 35 patients who died of CRC and 62 matched patients who did not die of CRC and lived at least as long as the patient they were matched to. Patients were matched on age at diagnosis, sex, tumor site, tumor stage, and year at diagnosis. We classified 2.90 million cells into 16 distinct cell types. We quantified immune and stromal cells as proportions of non-epithelial cells and epithelial cells as proportions of total cells. For spatial analysis, we identified eight cellular neighborhoods with distinct local cell composition and calculated spatial proximities between different cell types. We performed multivariable-adjusted logistic regression with generalized estimating equations to investigate associations of CRC-specific death with proportions of cell types and spatial proximities. We estimated odds ratios (OR) with 95% confidence intervals (CI) and calculated the false discovery rate (FDR) to account for multiple comparison using the Benjamin-Hochberg method. We found that higher proportions of stromal cells (OR for doubling of proportion=2.29, 95%CI=1.00-5.26) and higher proportion of M2 macrophages at the tumor center (OR=2.90, 95%CI=1.14-8.22) were associated with increased risks of CRC-specific death among 76 patients with microsatellite stable (MSS) tumors. We observed higher proportions of podoplanin (PDPN)-expressing stromal cells (OR=2.97 for doubling of proportion, 95%CI=1.28-7.08) and M1 macrophages (OR=2.42, 95%CI=1.05-5.69) within cellular neighborhoods predominantly composed of tumor epithelial cells among patients with CRC deaths compared to survivors, although these associations did not remain significant at FDR 0.05. Additionally, we observed that colocalization of PDPN-expressing stromal cells with CD8+ T cells, and colocalization of antigen-presenting cells with plasma cells, was associated with increased risk of CRC-specific death. In contrast, colocalization of B cells with stromal cells was associated with decreased risk (adjusted P=0.04 for all associations). Those associations remained significant among patients with MSS (adjusted P=0.03). This is the first study focusing on the TME cellular landscape among Alaska Native patients with CRC. Our results highlight the prognostic relevance of cellular composition and spatial organization within the TME, suggesting specific stromal and immune cell interactions as potential biomarkers for CRC outcomes.



 Hang Yin, Diana Redwood, Kimberly Smythe, Kristin L. Robinson, Daniel Jones, McGarry Houghton, Kevin C. Barry, Amanda L. Koehne, Elizabeth Donato, Cecilia Yeung, Minggang Lin, James J. Tiesinga, Tabitha A. Harrison, Sushma S. Thomas, Li Hsu, Jane C. Figueiredo, Li Li, Christopher I. Li, Ulrike Peters, Jeroen R. Huyghe. Spatial-resolved single-cell analysis of the tumor microenvironment in Alaska Native colorectal cancer patients reveals novel biomarkers for prognosis [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A087.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fe434533ff8a8160b5fcdfdf886e32d3184d6f4" target='_blank'>
              Abstract A087: Spatial-resolved single-cell analysis of the tumor microenvironment in Alaska Native colorectal cancer patients reveals novel biomarkers for prognosis
              </a>
            </td>
          <td>
            Hang Yin, Diana Redwood, K. Smythe, Kristin L. Robinson, Daniel Jones, Mcgarry Houghton, K. C. Barry, Amanda L Koehne, Elizabeth Donato, Cecilia Yeung, MingGang Lin, James Tiesinga, T. Harrison, Sushma Thomas, Li Hsu, Jane C Figueiredo, Li Li, Christopher I Li, Ulrike Peters, J. Huyghe
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1de125e5b8e76ca9fbd244fb7a42b1eef22672" target='_blank'>
              Mass cytometry data integration methods reveal rural-urban gradient of immune profiles across geography
              </a>
            </td>
          <td>
            Koen A. Stam, Jeremia J Pyuza, Mikhael D. Manurung, D. Tahapary, T. Supali, Marloes M A R van Dorst, A. Mahfouz, M. Charrout, S. Jochems, Marion H. König, Y. Kruize, Marouba Cisse, Ibrahima Diallo, Moustapha Mbow, M. Yazdanbakhsh, W. A. de Steenhuijsen Piters
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 Pancreatic intraepithelial neoplasms (PanINs), precursor lesions of pancreatic ductal adenocarcinoma (PDAC) that are abundant healthy individuals, rarely progress to cancer despite universally possessing many of the driver alterations associated with malignancy. Recent single cell profiling efforts of PanIN progression in vivo have described marked heterogeneity in chromatin accessibility and transcriptional identity within these neoplasias, but it remains unclear how discrete epigenetic factors play a role in shaping tumorigenesis, or what distinguishes rare neoplastic cells capable of seeding invasive carcinomas from their benign neighbors. Given that more than 80% of patients with PDAC present with unresectable or metastatic disease, further studies on the molecular features of PanIN progression are needed to enable methods of early detection and intervention. Unfortunately, owing to their microscopic size and challenges associated with the isolation of PanINs from surrounding normal epithelium and stroma, it is particularly difficult to address these knowledge gaps in vivo. By generating complex murine derived organoid models, we have built a system that allows us to faithfully reconstitute the evolution from benign PanIN to PDAC in vitro, as well as in vivo through orthotopic implantation into syngeneic mice. As they are easily perturbable, readily generate large amounts of biomass, and closely resemble the stages of PDAC initiation in vivo, these models can be leveraged to directly assay how chromatin remodelers influence malignant transformation. Further, pairing these organoids with our highly multiplexed, transcribable DNA barcoding system and single cell epi multiomics will enable us to study the evolutionary dynamics, chromatin landscapes, and transcriptional identities of clonal populations of PanIN organoids as they progress to cancer. Through these studies, we will gain unprecedented insights into the underpinnings of PDAC initiation, and thus may open up new avenues for the early detection of tumorigenesis.



 Connor J. Hennessey, Minh T. Than, Ben Z. Stanger, Sydney M. Shaffer. Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90b593e2ad7a725999b9d873ea37a620072eeba" target='_blank'>
              Abstract B106: Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models
              </a>
            </td>
          <td>
            Connor J. Hennessey, Minh T Than, Ben Z. Stanger, Sydney M. Shaffer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cced907718bd3a02f9c4423bb5be0323614fd5" target='_blank'>
              Long-term clonal analysis using stochastic models reveals heterogeneity and quiescence of hematopoietic stem cells
              </a>
            </td>
          <td>
            Yuri Garcia Vilela, L. Thielecke, A. C. Fassoni, I. Glauche
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is an epithelial cell cancer of the biliary tract. CCA can be further classified into intrahepatic cholangiocarcinoma (iCCA), perihilar cholangiocarcinoma (pCCA), and distal cholangiocarcinoma (dCCA) depending on the anatomical location. Until recently, the treatment for advanced CCA has remained highly reliant on chemotherapy, with Gemcitabine plus Cisplatin used in first-line treatment. Recent developments have led to the addition of immune checkpoint blockade (ICB) to the chemotherapy regimen, highlighting the promising potential of immunotherapies for CCA treatment. Despite these developments, most patients still do not benefit from current treatments, and response rates to ICB monotherapy remain modest. This underscores the need to develop more effective immunotherapeutic strategies. A major obstacle to this is the highly heterogenous nature of the disease. CCA tumors exhibit high inter-tumor heterogeneity in terms of anatomical locations, driver mutations, etiologies, and tumor microenvironment (TME) composition, making each patient immunologically distinct and difficult to benefit from a one-fit-all approach. There is a need to stratify patients according to individual disease status to identify immunotherapies and combination therapies that are most beneficial to them. Here we describe the different ways inter-tumor heterogeneity may arise in CCA, including stromal cell abundance, anatomical location, driver mutations, etiologies, TME profile, and tertiary lymphoid structure (TLS) presence. We also discuss what these factors mean to the immune microenvironment and their potential to be used as biomarkers. Careful stratification of patients is crucial in designing personalized medicine to improve survival outcomes and treatment efficacy for CCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb66e953d9482069c69df6565960e118d42cf3b5" target='_blank'>
              Stratifying Cholangiocarcinoma: Tumor Microenvironment, Molecular Drivers, and Novel Immunotherapeutic Approaches.
              </a>
            </td>
          <td>
            Cindy Xinqi Liu, Carmen Chak-Lui Wong
          </td>
          <td>2025-10-14</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Tertiary lymphoid structures (TLSs) are ectopic lymphoid aggregates that form in non-lymphoid organs, frequently observed in conditions such as cancer, autoimmune diseases, transplant rejection, and chronic inflammation. Growing evidence suggests that TLSs are beneficial for patients’ prognosis with higher TLS density generally correlating with improved therapeutic response and survival outcomes across malignancies and might serve as a novel therapeutic target for cancer immunotherapy. However, the correlation between TLSs and tumor development is still ambiguous. The exact timing of TLS formation during tumorigenesis and their dynamic evolution throughout tumor progression remain under investigation. Recent studies have identified potential strategies for inducing TLSs, but there remains a considerable distance from clinical application. More advanced techniques such as high-resolution spatial multi-omics technologies combined with big data analysis will benefit understanding the complex interactions within TLSs and developing novel immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d68cd72d4e06f1532031fad0eb22d59671e56af" target='_blank'>
              Orchestration of tertiary lymphoid structures: decoding developmental mechanisms for next-generation cancer immunotherapies
              </a>
            </td>
          <td>
            Wenqin Hao, Ying Zhao
          </td>
          <td>2025-09-12</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Despite advances in immunotherapy, durable responses in lung cancer remain limited to a subset of patients, underscoring the need for biomarkers capturing spatial immune-tumor interactions. Current methods, such as PD-L1 immunohistochemistry, suffer from sampling bias and fail to decode dynamic immune evasion mechanisms non-invasively. Methods We developed a radiomics framework integrating longitudinal tumor growth kinetics (log volume change rate, LVCR) with deep learning to: (1) delineate tumors via medical knowledge-guided segmentation; and (2) derive an Immune Evasion Score (IES) predicting immunosuppressive niches. The model employs immune-aware attention gates (IAAG) to prioritize regions associated with aggressive growth (high LVCR) and immune evasion. Results Validated on 420 CT scans, our approach achieved superior segmentation accuracy (Dice=0.7728 ± 0.03; HD95 = 9.8 ± 1.5 mm) over existing models. Critically, the IES predicted PD-L1 expression (AUC = 0.85; *p*<0.001) and CD8+ T-cell exclusion (*p*<0.01). High IES correlated with rapid immunotherapy progression (HR = 2.3, *p*=0.004), and spatial analysis confirmed 72.3% concordance between IAAG-prioritized regions and pathological PD-L1+ niches. Conclusion This work establishes a non-invasive paradigm for mapping immunosuppressive microenvironments, bridging precision radiotherapy with immunotherapy personalization. The IES provides a dynamic biomarker of immune evasion, potentially guiding patient stratification for checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7469972933375774141af2efc5f077d7dd79ab2e" target='_blank'>
              AI-based non-invasive profiling of the tumor immune microenvironment using longitudinal CT radiomics predicts immunotherapy response in lung cancer
              </a>
            </td>
          <td>
            Guan-Yuan Liu, Xiaoyan Zhang, Yutong He, Di Liang, Shaonan Xie, Ning Zhang, Nan Geng, Liwen Zhang, Yajie Huang, Fang Liu, Qingyi Liu
          </td>
          <td>2025-10-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8be9d1c5a1b99f57bdae54da32d36142c606a15" target='_blank'>
              A unified analysis of cell-type and trajectory-associated pathways in single-cell data using Phoenix
              </a>
            </td>
          <td>
            Yudit Halperin, D. Nachmani, Michal Rabani
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78a06fd992b15a6520a4d4a8e38c9a04a5e3b92" target='_blank'>
              Spatially-resolved Photoproximity Profiling of MYC Identifies a MYC-BAF Liability in Cancer Cells
              </a>
            </td>
          <td>
            Anthony J. Carlos, Shuyuan Huang, Dongbo Yang, Colin Swenson, Pratyasha Chakraborty, Shaopeng Yu, Benjamin D. Stein, Raymond E. Moellering
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43797c7834d20e5589904d0168347625fb0cad6e" target='_blank'>
              Unraveling cellular dynamic changes in tumor evolution induced by long-term low dose-rate radiation.
              </a>
            </td>
          <td>
            Wanshi Li, Weiwei Pei, Yiwei Wang, Jing Nie, Guang Hu, Yongsheng Zhang, Zhifei Cao, Zaozao Chen, Keyan Miao, Yuqi Chen, Likai Shi, Yingchu Dai, , Hailong Pei, C. Ye, Yasser F Ali, Wentao Hu, Wenying Yan, Guangming Zhou
          </td>
          <td>2025-09-05</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background/Aims
Patient-derived organoids (PDOs) are promising preclinical models that replicate critical tumor features. However, intratumoral heterogeneity challenges the clinical utility of PDOs, especially in capturing diverse tumor cell subpopulations.


Methods
Single-cell transcriptomics was used to analyze PDOs from distinct sites within a single gastric cancer tumor, aiming to assess their ability to reflect intratumoral heterogeneity.


Results
The PDOs displayed similarities in gene expression but also exhibited distinct profiles. Single-cell analysis of PDOs revealed upregulation of markers for neuroendocrine tumors, which was validated via immunohistochemistry staining of neuron-specific enolase in the primary tumor. Notably, heat shock proteins showed significant variability among the PDOs, impacting immune responses. Tumors with abundant heat shock proteins are reported to have increased cytotoxic T cell activity.


Conclusions
Intratumoral heterogeneity poses challenges for PDO-based models, highlighting the need for comprehensive assessment. Despite their limitations, PDOs offer valuable insights into precision medicine for patients with gastric cancer, aiding in the development of therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9154970e5ea83aed48cb2a2d344b23752aade163" target='_blank'>
              Patient-Derived Organoids from Multiple Sites of a Single Tumor Recapitulates Intratumoral Heterogeneity in Patients with Gastric Cancer.
              </a>
            </td>
          <td>
            Bo Kyung Yoon, Y. Bae, Yeonjin Je, Seyeon Joo, Yuna Kim, Su-Jin Shin, Sungsoon Fang, Jie-Hyun Kim
          </td>
          <td>2025-09-05</td>
          <td>Gut and liver</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults and associated with a poor prognosis and a median survival of approximately 15 months, largely due to the lack of effective therapies and inevitable recurrence. A key challenge is the spatial and cellular heterogeneity, evident on MRI as a necrotic center, a contrast-enhancing tumor core, and a non-enhancing, T2/FLAIR-hyperintense infiltration zone. The diffuse infiltration of tumor cells into surrounding brain tissue impedes complete surgical resection and drives relapse from persisting residual cells. While regional molecular differences have been well characterized, their functional implications - particularly drug sensitivities - remain insufficiently understood.



 To address this, we performed MRI- and 5-aminolevulinic acid (5-ALA)-guided tissue sampling from the tumor core and from the infiltration zone of patients with glioblastoma. We utilized pharmacoscopy—a single-cell, microscopy-based drug screening platform—to evaluate region- and tumor cell type-specific drug responses. Cancer cell types were defined based on differential expression of nestin (marking stem-like cells) and S100B (marking more differentiated tumor cells).



 Profiling a library of 59 drugs across region-specific tumor samples from 23 patients revealed pronounced heterogeneity in drug responses across patients, tumor regions and tumor cell states. To overcome this heterogeneity, we computationally predicted complementary drug pairs and validated 20 drug combinations in an independent cohort. The combinations demonstrated enhanced efficacy compared to single agents and effectively addressed glioblastoma’s spatial, cellular, and inter-patient heterogeneity. Notably, drug combinations of oncology and neuroactive drugs outperformed those involving agents from the same drug class (oncology + oncology or neuroactive + neuroactive). Ongoing mechanistical explorations reveal first insights into mechanisms of the complementary drug efficacy.



 Our findings underscore the limitations of monotherapies for patients with glioblastoma and highlight the potential of rationally designed drug combinations tailored to the tumor’s spatial and functional complexity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60062d1c4566c5af52ee0940de47f3ed99103b0" target='_blank'>
              OS12.5.A SPATIALLY RESOLVED FUNCTIONAL PROFILING FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, S. Lee, I. Sakic, R. Wegmann, J. Mena, F. Vasella, J. Friesen, L. Regli, E. Le Rhun, A. Zeitlberger, N. Tatari, G. Hutter, M. Neidert, M. Weller, B. Snijder, T. Weiss
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b221cace1b09faa76330c09ce1ca8391e4017e" target='_blank'>
              GLIS3 Marks a Neural-like Progenitor Cell State that Drives Metastasis in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Dennis Gong, Jimmy A. Guo, Jennifer Su, Sueda Cetinkaya, Connor J Hennessey, Patrick Yu, Ananya D. Jambhale, Peter L. Wang, Nicholas J Caldwell, Ashley Lam, Junning Wang, C. Shiau, Kevin S. Kapner, Laleh Abbassi, Seema Chugh, Shatavisha Dasgupta, Jonathan O Nowak, B. Wolpin, M. L. Zhang, Mari Mino-Kenudson, T. Jacks, Andrew J. Aguirre, William L. Hwang
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Patient-derived cancer cells (PDCCs) have emerged as a key strategy for advancing personalized cancer treatment. Unlike traditional cancer cell lines, PDCCs retain the genetic and phenotypic characteristics of the patient’s original tumor and can more accurately reflect tumor biology. This review explores recent advances in methods for culturing PDCCs, highlighting the role of these models in drug discovery and high-throughput screening of personalized therapeutic options. By establishing living models directly from patient tumors, PDCCs can more faithfully recapitulate tumor heterogeneity and microenvironmental features than traditional cell lines. These cultures bridge laboratory research and clinical reality, allowing functional testing of patients' cancer cells. Despite the promise of PDCCs, their culture remains fraught with challenges, including the extremely low number of cancer cells that can be obtained, difficulty maintaining tumor heterogeneity, low culture initiation success rates, and ethical considerations for using patient tissues. In addition, controversy remains regarding the reproducibility of results between different laboratories and patient samples. By examining the field’s current state, this review identifies gaps in the application of PDCCs, such as limited modeling capabilities for specific tumor types and the lack of comprehensive, scalable protocols for broad clinical use. This article discusses future directions, including integration with advanced microengineering and AI-driven analysis, which have the potential to overcome existing limitations and optimize PDCCs-based therapeutic strategies. PDCCs are expected to transform the future of cancer treatment as they ultimately provide more accurate drug testing and personalized medicine models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/350a1202d8af5d4a9d6b419293e7569e23ebec7c" target='_blank'>
              Advancements and challenges in culturing patient-derived cancer cells for personalized therapeutics
              </a>
            </td>
          <td>
            Yatian Fu, B. L. Khoo, Chwee Teck Lim
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Lab on a Chip Technologies</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Treatment of locally advanced unresectable pancreatic cancer remains a major clinical challenge due to pronounced heterogeneity and resistance to standard regimens. Increasing evidence highlights the critical role of the tumor microenvironment (TME) in shaping therapeutic response and driving drug resistance. In this minireview, we summarize recent advances in TME phenotyping and its potential to guide precision therapy. A four-dimensional framework integrating stromal, immune, genomic, and metabolic features has been proposed to better characterize TME heterogeneity. Preclinical and clinical studies indicate that strategies targeting the stroma, modulating immunity, or exploiting genomic vulnerabilities such as homologous recombination deficiency may enhance the efficacy of chemotherapy, immunotherapy, and targeted agents. Dynamic biomarkers, including circulating tumor DNA and carbohydrate antigen 19-9, also show promise for real-time therapy adaptation, although their clinical application remains limited. By synthesizing current evidence, we emphasize the importance of individualized treatment strategies that account for TME complexity. While encouraging, the translation of multiomics phenotyping and biomarker monitoring into routine clinical practice requires standardization, prospective validation, and integration of novel technologies. Future research should focus on establishing reproducible TME-guided models to enable dynamic and personalized therapy for patients with unresectable pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9ebed50c75b679e8298f471ae2f8ee5933e4197" target='_blank'>
              Tumor microenvironment phenotyping guides precision therapy in unresectable pancreatic cancer
              </a>
            </td>
          <td>
            Kai Zhao, Ming-Ming Xiao, Yong-Sheng Yang, Xiao Xiao
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Uveal melanoma (UM), the most common primary intraocular malignancy in adults, presents significant clinical challenges due to its high metastatic potential, pronounced hepatic tropism, and poor prognosis upon systemic dissemination. Despite established local therapies, nearly half of patients develop distant metastases, highlighting an urgent need for more effective systemic strategies. Recent advances in single-cell omics technologies (e.g., scRNA-seq, scATAC-seq, spatial transcriptomics) have revolutionized our understanding of UM pathobiology. These approaches have meticulously delineated the complex tumor heterogeneity, immunosuppressive microenvironment, and key molecular drivers—including novel macrophage subsets (e.g., immunosuppressive MΦ-C4), senescent endothelial cells, and non-canonical immune checkpoint expression—providing unprecedented resolution for identifying actionable therapeutic targets. Concurrently, innovations in materials science and biomedical engineering offer transformative opportunities for precision therapy. Engineered nanocarriers, biodegradable implants, and advanced gene therapy vectors (e.g., tropism-enhanced AAVs, CRISPR-Cas9 systems) enabled targeted drug delivery, sustained release, and genetic modulation tailored to the eye’s unique anatomy and immune privilege. This review synthesizes these converging frontiers, outlining how the integration of multi-omics insights with smart biomaterials can overcome current therapeutic limitations. We catalog emerging material-based platforms applicable to UM and summarize validated molecular targets (e.g., GNAQ/GNA11, YAP/TAZ, BAP1, c-Met, CXCR4). Furthermore, we propose an interdisciplinary paradigm spanning combinatorial targeted therapies, immunomodulation, minimally invasive devices (e.g., robotic radiosurgery), and engineered delivery systems. By bridging mechanistic discovery with translational engineering, this synergy holds significant promise for advancing precision medicine and improving clinical outcomes in UM, ultimately facilitating the transition from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0323cee7d6c89a49fe569ae452832b7a890d38bd" target='_blank'>
              Integrating single-cell omics and materials science for uveal melanoma: from mechanistic insights to precision therapeutics
              </a>
            </td>
          <td>
            Shouyong Fu, Changfei Li
          </td>
          <td>2025-09-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a peri-vascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment, including immune cells, astrocytes, and vascular cells.



 By analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes intimately associated with the endothelium as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape. Specifically, endothelial cells deprived of pericyte association upregulated Colony Stimulating Factor (CSF)-1, which in turn attracted peri-vascular tumor-associated macrophages polarized towards an immune-suppressive phenotype. In the absence of pericytes, the recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase c-Met on a subset of GBM cells harboring an extreme mesenchymal-like state. Indeed, orthotopic implantation of isolated c-Met-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype, compared to cells negative for c-Met.



 We infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a67e2a70cd46f28d9a303424953a15720ec17e" target='_blank'>
              P02.01.B PERICYTES ORCHESTRATE A TUMOR-SUPPRESSIVE MICROENVIRONMENT BY IMPINGING ON THE CROSSTALK BETWEEN MACROPHAGES AND TUMOR-INITIATING CELLS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, M. Safaee Talkhoncheh, B. Phung, E. Cordero, E. Johansson, R. Rosberg, G. B. Jönsson, A. Pietras, K. Pietras
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Internal organs are encased by a supportive epithelial monolayer of mesodermal origin, termed mesothelium. The nature, evolution and function of mesothelial cells, and their genetic regulation impacting disease development are insufficiently understood. Here, we generate a comprehensive organ-wide single-cell transcriptomic compendium of mesothelium across healthy and diseased mouse and human organs, delineating the evolution of conserved activated states of mesothelial cells in response to disease. We uncover genetic drives behind each cell state and reveal a conserved metabolic gate into multipotent proteolytic, inflammatory and fibrotic cell differentiation, in mouse and human. Using lung injury models in mice, in combination with mesothelial cell-specific viral approaches, we show that direct metabolic reprogramming using Ifi27l2a and Crip1 on organ surfaces, blocks multipotent differentiation and protects mouse lungs from fibrotic disease. These findings place mesothelial cells as cellular exemplars and gateway to fibrotic disease, opening translational approaches to subvert fibrosis across a range of clinical indications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32dcba7e7305488d54f2d30bf4e811162db8c3c5" target='_blank'>
              A mesothelial differentiation gateway drives fibrosis
              </a>
            </td>
          <td>
            Safwen Kadri, A. Fischer, Martin A Mück-Häusl, Wei Han, Amal Kadri, Yue Lin, Lin Yang, Shaoping Hu, Haifeng Ye, Pushkar Ramesh, M. Ansari, Herbert B. Schiller, H. Machens, Y. Rinkevich
          </td>
          <td>2025-09-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Vascular mimicry (VM) refers to the formation of vessel-like structures by tumor cells independent of endothelial cells. These VM channels connect to the host’s vascular network and are associated with aggressive tumors and poor patient prognosis. Most VM research has been conducted on melanoma, relying on patient-derived and mouse cell lines. In other solid tumors, VM studies rely on human cell lines, which have certain limitations for in vivo studies. Specifically, most in vivo VM research involving human cells uses subcutaneous mouse models that fail to recapitulate organ-specific tumor microenvironments. As the microenvironment is an essential driver of tumor vascularization, including VM, murine cell lines could facilitate VM investigations in syngeneic mouse models. Here, we present CT26 and KPC, well-characterized murine colorectal and pancreatic cancer cell lines, as cell models for VM investigations. Using in vitro cell-based assays, we demonstrate that CT26 and KPC undergo VM, a cell-intrinsic process that is enhanced by serum deprivation and exposure to hypoxia and is independent of tumor-secreted growth factors. Additionally, we demonstrate the importance of YAP/TAZ signaling in VM formation, as inhibition at non-cytotoxic concentrations attenuated VM formation. Remarkably, CA3, the most potent of the two inhibitors, significantly reduced cell proliferation in both cell lines at the IC50 concentration. This reduction in cell proliferation was associated with the induction of apoptosis in CT26 cells and changes in the cell cycle in both CT26 and KPC cells. Finally, dual YAP/TAZ knockdown in both cell lines significantly abrogated VM formation, validating our initial findings using inhibitors. These results show that CT26 and KPC cells undergo VM, and given their extensive use in cancer research, can be used to investigate VM in vivo using syngeneic models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec4fb2cfcbdb76b0840a4ef6bfa2d3738d1e6f07" target='_blank'>
              Murine Cell Line Models for Vascular Mimicry: The Role of YAP/TAZ Signaling
              </a>
            </td>
          <td>
            Matilde Righetti, Ana-Maria Primorac, J. T. Erler, V. Oria
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most common brain cancers diagnosed in children, often leading to life-long neurological impairments. Whilst targeted inhibitors exist, their effects are often short-lived, highlighting the urgent need for innovative treatments. Notably, some pLGG tumors spontaneously regress with age, suggesting that changes in the tumor microenvironment can arrest tumor growth. In this work, we aim to characterize which signaling interactions promote the growth of pLGG cells and how these interactions change dynamically with time. To achieve this, we have optimized a protocol for introducing pLGG mutations into developing mouse brains via in utero electroporation. This study revealed that brains harboring the common KIAA1549::BRAF fusion mutation have a striking phenotype characterized by increased astrocyte reactivity and myeloid cell infiltration. To better understand how signals derived from immune cells may impact pLGG cell fitness, we are additionally performing a high-throughput cytokine screen. We are using barcoded cytokine constructs tethered to the cell membrane to comprehensively screen the impact of cytokine signaling on the growth of neural stem cells engineered to overexpress common pLGG mutations. Together this work aims to unravel which signaling factors govern the age-dependent growth of pLGG tumors, which may reveal targetable aspects of the tumor microenvironment amenable to therapeutic intervention.



 Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy Chang, Ria Kedia, Michelle Boisvert, David Jones, Timothy Phoenix, Pratiti Bandopadhayay. Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f149e2a5701b9900d3e20ea1c251a5956b1b56d2" target='_blank'>
              Abstract B037: Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy H. Chang, Ria Kedia, Michelle Boisvert, David Jones, T. Phoenix, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Despite significant efforts, Diffuse Midline Gliomas (DMGs) remain incurable. Although promising results are obtained in preclinical studies, most approaches have failed to improve survival in these young patients. In this context, by integrating elements of the biophysical tumor microenvironment known to regulate the response to therapies, a new preclinical tool is developed to better evaluate the efficacy of antitumoral strategies. For this purpose, a novel DMG-on-Chip (DoC) is engineered composed of a 3D dense tumor disk embedded in an extracellular matrix, which is accessible for real-time monitoring using wide-field phase contrast or confocal fluorescence microscopy. By driving the oxygen supply within the chip solely in a radial manner, a hypoxia gradient is established that can be associated with changes in DMG cell phenotype, proliferation, and rewiring of metabolic and stress transcriptomic signatures. Finally, DoC is used to analyze the spatial heterogeneity of the response of DMG cell lines and patient-derived 3D cultures to treatments through cell segmentation. Altogether, these interdisciplinary characterizations validate the new tool as an interesting healthcare solution to understand how cell responses are modulated by biophysical or biochemical cues in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a25e403c641a7729cb13c93c687243726c53a1" target='_blank'>
              Novel Diffuse Midline Glioma-on-Chip Recapitulating Tumor Biophysical Microenvironment to Assess the Heterogeneity of Response to Therapies.
              </a>
            </td>
          <td>
            Lisa Terrassoux, Calypso Hazard, Agathe Laratte, Joanne Balsamelli, Mélanie Arcicasa, Elisa Fong, Léo Baland, Fabrice Soncin, Giovanni Cappello, C. Maurage, Sebastien Janel, Antony Bazir, Fabrizio Cleri, Anthony Treizebré, Eric Lartigau, Yousr Rekik, Samuel Meignan, Alessandro Furlan
          </td>
          <td>2025-09-29</td>
          <td>Small</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e8d2f1aaaa7f0e435a0bcfed4046ad25f2a01b" target='_blank'>
              Deciphering macrophage heterogeneity and factors driving M2 polarization in lung adenocarcinoma through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Meiling Sheng, Beiwei Yu, Qunzhi Wang, Yuanchao Xiao, Ming Ling, Xiaoming Wu
          </td>
          <td>2025-10-16</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="MYCN amplification without concurrent RB1 mutations characterizes a rare yet highly aggressive subtype of retinoblastoma; however, its precise developmental origins and therapeutic vulnerabilities remain incompletely understood. Here, we modeled this subtype by lentiviral-mediated MYCN overexpression in human pluripotent stem cell-derived retinal organoids, revealing a discrete developmental window (days 70–120) during which retinal progenitors showed heightened susceptibility to transformation. Tumors arising in this period exhibited robust proliferation, expressed SOX2, and lacked CRX, consistent with origin from primitive retinal progenitors. MYCN-overexpressing organoids generated stable cell lines that reproducibly gave rise to MYCN-driven tumors when xenografted into immunodeficient mice. Transcriptomic profiling demonstrated that MYCN-overexpressing organoids closely recapitulated molecular features of patient-derived MYCN-amplified retinoblastomas, particularly through activation of MYC/E2F and mTORC1 signaling pathways. Pharmacological screening further identified distinct therapeutic vulnerabilities, demonstrating distinct subtype-specific sensitivity of MYCN-driven cells to transcriptional inhibitors (THZ1, Flavopiridol) and the cell-cycle inhibitor Volasertib, indicative of a unique oncogene-addicted state compared to RB1-deficient retinoblastoma cells. Collectively, our study elucidates the developmental and molecular mechanisms underpinning MYCN-driven retinoblastoma, establishes a robust and clinically relevant human retinal organoid platform, and highlights targeted transcriptional inhibition as a promising therapeutic approach for this aggressive pediatric cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7456982cd84b09cab50502a3b5ab5c8e7a74c4cd" target='_blank'>
              Human Retinal Organoid Modeling Defines Developmental Window and Therapeutic Vulnerabilities in MYCN-Amplified Retinoblastoma
              </a>
            </td>
          <td>
            Jinkyu Park, Gang Cui, Jiyun Hong, H. Jeong, Minseok Han, Min Seok Choi, Jeong Ah Lim, Sanguk Han, Christopher Seungkyu Lee, Min Kim, Sangwoo Kim, Junwon Lee, S. Byeon
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38e3e2e1b934125c90539e7b6826f0bb1928fb1d" target='_blank'>
              Image-based phenotypic sorting of synthetic cells
              </a>
            </td>
          <td>
            Marijn van den Brink, Marlena Stam, Nico J. Claassens, C. Danelon
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains a highly lethal malignancy, in part due to its heterogeneous tumor microenvironment. Most PDAC tumors are poorly vascularized and embedded within densely fibrotic stroma; however, in prior computational analyses, we identified a subset of PDAC tumors with hyper-vascular features. In this study, we sought to determine the extent of vascular heterogeneity in PDAC and its functional consequences. Analysis of human tumor specimens revealed that approximately 10-20% of human tumors display features of hyper-vascularity. Using a panel of clonal murine cell lines that recapitulate this heterogeneity, we demonstrate that VEGFA-mediated angiogenesis does not account for the increased vessel density. Functionally, we further show that hyper-vascular tumors display enhanced sensitivity to both chemotherapy and anti-angiogenic therapy. These findings help explain the limited efficacy of VEGFA-targeted therapies in unselected PDAC cohorts and highlight vessel density as a potential biomarker for patient stratification.



 Samantha B. Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo I. Orlen, Samuel D. Zwernik, Kailee M. Hartway, Nina G. Steele, Agnieszka K. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger. Vascular heterogeneity influences treatment response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cec4b9af7cf61e156104d3e634ae5d707e4e9c" target='_blank'>
              Abstract A127: Vascular heterogeneity influences treatment response in pancreatic cancer
              </a>
            </td>
          <td>
            Samantha B Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo Orlen, Samuel D Zwernik, Kailee M Hartway, Nina G. Steele, A. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Merging evidence suggests that cellular fitness competition between malignant cells and the tumor microenvironment (TME) dictates tumor progression. We recently identified Flower isoforms as molecular “fitness fingerprints,” where cancer cells expressing Flower-Win gain dominance over surrounding cells marked with Flower-Lose. In this study, we uncover that loss of stromal fitness is not an inherent trait but rather a cancer-driven event. Tumor cells secrete a specific long non-coding RNA, Tu-Stroma, via exosomes, which reprograms splicing of the Flower locus in TME cells, enforcing Flower-Lose expression and thereby reducing stromal competitiveness. This cancer-induced shift establishes a permissive niche that accelerates growth, metastatic dissemination, and ultimately shapes host survival in ovarian cancer. Therapeutically, blockade of Flower using a fully humanized monoclonal antibody (Aenya) markedly suppressed tumor expansion and peritoneal spread in patient-derived xenograft models. Importantly, prophylactic administration preserved the integrity of intraperitoneal organs even in the presence of highly aggressive tumor cells. These findings highlight a previously unrecognized mechanism of cancer-driven stromal takeover and establish Flower as a first-in-class therapeutic target with broad implications for preventing metastatic colonization.



 Rajan G. Gogna. Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0864f00ed3ab31470a0c2e77e5565e26d50bb11a" target='_blank'>
              Abstract IA008: Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer
              </a>
            </td>
          <td>
            Rajan Gogna
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pre-invasive breast cancer, or ductal carcinoma in situ (DCIS), shares many morphologic and genomic features with invasive breast cancer, yet most DCIS tumors remain indolent over decades. In this study, we performed spatial analyses of somatic mutation patterns in 18 DCIS tumors to infer the underlying growth dynamics. Experimental data combined with mathematical models and Bayesian computation revealed a growth model that mimics the cellular dynamics of branching morphogenesis, which governs normal mammary gland development during puberty. Unlike a traditional model of clonal evolution, this non-competitive growth model allowed for extensive DCIS spread without the selective pressures of subclone competition that promote the emergence of invasive phenotypes. This finding provides a framework to understand the biological basis for the limited progression of DCIS and adds rationale for ongoing clinical efforts to reduce DCIS overtreatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56eaf0d13e80db0fb1f5ee2e2242518bc8cf64f7" target='_blank'>
              Spatially Discontinuous Mutation Topographies in Ductal Carcinoma in Situ Reveal Non-Competitive Growth Dynamics.
              </a>
            </td>
          <td>
            Marc D. Ryser, Matthew A Greenwald, Daniel L. Monyak, I. Sorribes, Lorraine M. King, Allison Hall, J. Geradts, Donald L Weaver, D. Mallo, Shannon T Holloway, Ethan Wu, Kevin A. Murgas, Robert B. West, Lars J. Grimm, Carlo C. Maley, Jeffrey R. Marks, Darryl Shibata, E. S. Hwang
          </td>
          <td>2025-10-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cellular senescence exerts context-dependent effects in cancer, functioning as both a tumor suppressor and promoter. Tumor suppression occurs through p53/p16-mediated cell cycle arrest, whereas tumor promotion is driven by the senescence-associated secretory phenotype (SASP), which reshapes the tumor microenvironment. SASP, comprising inflammatory cytokines such as IL-6 and IL-8 alongside matrix-remodeling factors, fosters immune evasion, angiogenesis and therapeutic resistance. Individual SASP components exert distinct effects on tumor progression across cancer types, which underscores the importance of context-specific analyses. For instance, IL-6 is associated with metastasis in breast cancer, whereas IL-8 is notably associated with therapy resistance in lung cancer. This heterogeneity highlights the need for personalized strategies targeting specific SASP factors. The primary aim of the present review is to systematically dissect the context-dependent mechanisms underlying cellular senescence in cancer including the heterogeneity of SASP and its cancer-type-specific roles, evaluate emerging senotherapeutic modalities, and discuss key challenges and future directions to guide precision oncology approaches. Recent advances in senotherapy, including senolytics such as dasatinib and quercetin, senomorphics and Traditional Chinese Medicine-derived agents such as resveratrol, aim to eliminate pathological senescence while preserving its beneficial roles. Nonetheless, key challenges persist, particularly in biomarker identification and optimizing combinations with immunotherapy. Future research can leverage single-cell technologies to dissect senescence heterogeneity, enabling the potential development of precision oncology approaches. The primary aim of the present review is to systematically dissect the context-dependent mechanisms underlying cellular senescence in cancer-including the heterogeneity of the SASP and its cancer-type-specific roles and evaluate emerging senotherapeutic modalities, and discuss key challenges and future directions to guide precision oncology approaches. To further advance this aim, future research can leverage single-cell technologies to dissect senescence heterogeneity at the cellular and molecular levels; this will help distinguish protective senescent populations from pathogenic ones, thereby enabling the potential development of precision oncology approaches tailored to tumor-specific senescence landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de14546306a02db7736f5e73f995779ddb1112f1" target='_blank'>
              Cellular senescence in cancer: Unveiling dual roles, tumor microenvironment dynamics and therapeutic innovations (Review)
              </a>
            </td>
          <td>
            Yatsu Lam, Jiaming Gu, Peihao Yin
          </td>
          <td>2025-10-14</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor microenvironment (TME) comprises neoplastic and stromal cells, and extracellular matrix elements, all engaging in a complex interplay that ultimately dictates tumorigenesis, cancer progression, and therapeutic response. While extensive research on the TME has been conducted in human oncology, data on its veterinary counterpart, particularly in feline mammary tumors (FMTs), are still scarce. In this review, we explore current understanding of feline mammary carcinoma (FMC) microenvironment, focusing on tumor necrosis, fibrosis, angiogenesis, adipose tissue tumor-associated inflammation, extracellular vesicles, and epithelial–mesenchymal transition (EMT) and their prognostic implications. In FMC, remodeling of collagen fibers, cancer-associated fibroblasts (CAFs), regulatory T cells (Tregs) and elevated serum leptin have been associated with poor prognosis, whereas stromal cytotoxic T cells correlate with more favorable outcomes. By contrast, findings on necrosis and pro-angiogenic factors remain inconsistent, and research on extracellular vesicles (EVs) is still in its early stages. This review presents insights from human breast cancer (HBC) that further support and elucidate the potential relevance of these TME components. As FMCs are highly aggressive tumors, a deeper understanding of their microenvironment could not only improve prognostic accuracy but also uncover novel therapeutic targets. Furthermore, due to their similarities, FMCs offer a potential valuable spontaneous model for HBC, particularly for the aggressive triple-negative phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acfc9c9171773b52e82ceb2afc7e5c11c59650bc" target='_blank'>
              Inside the Tumor: Decoding the Feline Mammary Tumor Microenvironment and Its Prognostic Value—A Review
              </a>
            </td>
          <td>
            J. Rodrigues-Jesus, Ana Canadas-Sousa, Hugo Vilhena, P. Dias-Pereira
          </td>
          <td>2025-10-08</td>
          <td>Veterinary Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Quantitative image analysis is essential for advancing stem cell biology, developmental studies, and drug discovery, yet most workflows still rely on manual or semi-quantitative scoring that is slow, subjective, and poorly scalable. A major challenge is converting complex colony morphologies into reproducible, high-dimensional datasets. To address this gap, we developed ColonyQuant, an open-source platform that integrates automated colony segmentation, alkaline phosphatase (AP) intensity quantification, morphometric profiling, and statistical classification into a single workflow. ColonyQuant computes per-colony functional readouts alongside comprehensive shape descriptors, capturing both staining intensity and structural features in a unified framework. Applied to embryonic stem cells (ESCs) treated with a selective KDM4 histone-demethylase inhibitor, ColonyQuant revealed dose-dependent reductions in colony area and integrated AP signal, together with systematic remodeling of morphometric metrics. Multivariate analyses robustly stratified treatment groups and identified intensity and solidity as principal features capturing dose-dependent colony responses. By transforming subjective scoring into objective, scalable, and biologically interpretable phenotyping, ColonyQuant provides a reproducible platform for stem cell research and high-content screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57357026b197a6c463b98555927514215f3da5a" target='_blank'>
              Automated Image-Based Profiling of Pluripotent Stem Cell Colonies
              </a>
            </td>
          <td>
            Rui Geng, Benjamin L. Kidder
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d8b0d319f8fcb0e9c8819010ea4b2ab4abcd49" target='_blank'>
              HIF-1α and RhoA Drive Enhanced Motility and Aerotaxis of Polyaneuploid Prostate Cancer Cells in Hypoxia
              </a>
            </td>
          <td>
            Noreen Hosny, Shengkai Li, S. Amend, Robert A. Gatenby, Kenneth J. Pienta, Joel Brown, Junle Qu, Robert H. Austin
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4a57be54cf710876a27f9e5a47455e094a8d69" target='_blank'>
              Tumors mimic the niche to inhibit neighboring stem cell differentiation
              </a>
            </td>
          <td>
            Yang Zhang, Yuejia Wang, Jinqiao Song, Lizhong Yan, Ziguang Wang, Dongze Song, Yudi Zhao, Shaowei Zhao
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Capillary malformation (CM) is a congenital vascular anomaly affecting the skin, mucosa, and brain, yet the understanding of its vascular pathogenesis remains limited. Methods We applied spatial whole-transcriptome profiling (GeoMx) and gene set enrichment analysis within CM lesions at single vasculature level. Differentially expressed genes were validated by immunofluorescence staining. Phosphoproteomics was profiled to uncover lesion- wide phosphorylation sites on proteins. Single-cell RNA sequencing was performed on CM- derived induced pluripotent stem cells (iPSCs) to determine differentiation trajectories of lesional vascular lineages. In silico gene perturbation was used to predict candidate genes for modulating vascular pathological progression, followed by functional validation in CM iPSC-derived endothelial cells (ECs) using a Tet-on system. Results A spatial transcriptomic atlas was constructed, and pathological landscape of individual CM vasculature was delineated. CM vessels exhibited hallmarks of endothelial-to-mesenchymal transition (EndMT), including disruption of adherens junctions (AJs), vascular identity transitions, and metabolic remodeling. Phosphoproteomics confirmed that differentially phosphorylated proteins were enriched in EndMT- and AJ-related pathways. Aberrant expression of venous transcriptional factor NR2F2 was observed in lesional ECs and correlated with progressive enlargement from capillaries to larger-caliber vessels containing multiple layers of smooth muscle cells (SMCs). In CM iPSCs, differentiation course yielded reduced ECs but increased SMCs. In silico knockout simulation predicted NR2F2 as a crucial regulator of facilitating SMC phenotype in CM. Consistently, enforced NR2F2 expression during iPSC differentiation suppressed endothelial markers while inducing SMC-associated genes. Conclusions Single CM vasculature displays pathological hallmarks characterized by EndMT and AJ disruption, leading to progressive vascular remodeling. NR2F2 functions as a central regulatory factor orchestrating the acquisition of the SMC phenotype, thereby representing a potential therapeutic target in CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1fa98536c86180270a8a6d151af2637cfd03767" target='_blank'>
              Single-vessel transcriptome map pathological landscapes and reveal NR2F2-mediated smooth muscle cell phenotype acquisition in capillary malformations
              </a>
            </td>
          <td>
            Vi Nguyen, Irving Mao, Siwuxie He, Isabella Castellanos, Mackenzie Azuero, Marcelo L Hochman, Yin Rong, L. Pernomian, Elliott H Chen, Harold I Friedman, Yan-Hua Chen, Qun Lu, Daping Fan, Camilla F Wenceslau, Dong-bao Chen, Stuart Nelson, Anil G Jegga, Yunguan Wang, Wenbin Tan
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de62dfbaf4b4553a5a57c81de9ba67688e29afbe" target='_blank'>
              Microphysiological systems for metastasis research: a stepwise approach.
              </a>
            </td>
          <td>
            Vira Sharko, Ignacio Ochoa, Estela Solanas
          </td>
          <td>2025-10-22</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b1a443d05c1334b2d0a0dff8460a523dfdb2d5" target='_blank'>
              A three-dimensional ex vivo model recapitulates in vivo features and drug resistance phenotypes in childhood acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Magdalini Kanari, Iria Jimenez Garcia, Fabio Steffen, L. Krattiger, Charles Bataclan, Wangjie Liu, Benjamin R. Simona, Bart Deplancke, O. Naveiras, M. Ehrbar, B. Bornhauser, Jean-Pierre Bourquin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>1</td>
          <td>46</td>
        </tr>

        <tr id="The awakening of dormant disseminated cancer cells appears to be responsible for the clinical relapses of patients whose primary tumors have been successfully cured months and even years earlier. In the present study, we demonstrate that dormant breast cancer cells lodged in the lungs reside in a highly mesenchymal, nonproliferative phenotypic state. The awakening of these cells is not triggered by a cancer cell-autonomous process. Instead, lung inflammation induced by the chemotherapeutic agent bleomycin effectively awakens dormant cancer cells, providing useful models for studying metastatic awakening. Mechanistically, the awakened cells shift from a highly mesenchymal to a quasi-mesenchymal phenotypic state in which they acquire tumorigenicity and proliferative ability. Once awakened, these cells can stably reside in this quasi-mesenchymal state and maintain their tumor-initiating ability, doing so without ongoing heterotypic signaling from the lung microenvironment. Epidermal growth factor receptor ligands released by the cells of the injured tissue microenvironment, including notably M2 type macrophages, promote dormant cancer cells to move toward this quasi-mesenchymal state, a transition that is critical for the awakening process. An understanding of the mechanisms of metastatic awakening may lead in the future to treatment strategies designed to prevent such awakening and resulting metastatic relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0978c66611eb319088a607dd5da99f325c7f34ff" target='_blank'>
              Inflammation awakens dormant cancer cells by modulating the epithelial-mesenchymal phenotypic state.
              </a>
            </td>
          <td>
            Jingwei Zhang, Jingwen Zhang, Longfei Han, Shiyi Wu, Jie Li, E. Eaton, Bingbing Yuan, Ferenc Reinhardt, Hao Li, Patrick C Strasser, Sunny Das, Joana Liu Donaher, Md Imtiaz Khalil, Haiping Jiang, Alexander Deuschel, Danni Lin, Carolin Sebastiany, Mariana Maranga, Salomé Shubitidze, Xiaofei Liu, Arthur W Lambert, Yun Zhang, Yana Liu, L. Sui, Sarah Elmiligy, Umberto Pozza, Rauf Günsay, Ranjan Mishra, José Velarde, Sonia Iyer, Whitney S. Henry, K. Weiskopf, Guihai Feng, Tobiloba E. Oni, R. Watnick, Xin Li, Robert A. Weinberg
          </td>
          <td>2025-09-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="


 Brain metastases often originate from circulating tumor cells (CTCs) that become arrested within cerebral microvessels. Recent findings indicate that this process induces localized hypoxia, which in turn upregulates angiogenic mediators such as Angiopoietin-2 (Ang-2), thereby promoting vascular remodeling and facilitating extravasation of tumor cells into the brain parenchyma. Nevertheless, the early transcriptional landscape of the adjacent brain tissue—the so-called pre-metastatic niche—remains poorly defined.



 To simulate early metastatic seeding, syngeneic murine lung and breast carcinoma cells were injected into the right carotid artery of immunocompetent mice. The contralateral, non-injected hemisphere served as an internal control to allow intra-animal comparisons, reduce biological variability, and minimize animal usage. Brain tissue was collected at 1 and 4 days post-injection, processed via formalin fixation and paraffin embedding, and subjected to histological evaluation. Comparative bulk RNA sequencing of tumor-bearing and control hemispheres was performed, complemented by high-plex spatially resolved single-cell transcriptomic profiling to detect localized gene expression patterns.



 Bulk RNA sequencing revealed no significant transcriptional differences between hemispheres. In contrast, spatial transcriptomic analysis uncovered robust, spatially restricted gene expression changes in the tumor-bearing hemisphere. These alterations involved genes associated with vascular remodeling, hypoxia-responsive pathways, and neuronal connectivity. Notably, spatial mapping demonstrated that the expression of several key transcripts peaked near sites of arrested tumor cells and declined with increasing distance, indicative of a highly localized, tumor-induced host response.



 This study reveals distinct, spatially confined transcriptional alterations that define the early pre-metastatic niche in the brain. By leveraging advanced spatial single-cell transcriptomics, we provide a powerful framework to dissect the molecular cues underlying early metastatic interactions within the native tissue microenvironment. These insights not only deepen our understanding of early brain metastasis biology but also inform the development of innovative strategies for therapeutic intervention at the pre-metastatic stage.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6ab71d6adf66b6036a2c3fb3715feb43793308" target='_blank'>
              P18.25.B SPATIALLY CONFINED TRANSCRIPTIONAL RESPONSES CHARACTERIZE THE EARLY PRE-METASTATIC NICHE IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            W. R. Hube, F. Schallerer, L. Bergmayr, R. Piecyk, T. Eska, K. J. Mueller, E. Nicolaishvili, L. Sevenich, K. Unger, P. Jurmeister, L. von Baumgarten, P. Harter
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal tumor. PDAC also shows poor response to many conventional treatments and is refractory to immunotherapy. We hypothesize that these challenges arise from the highly heterogeneous and immunosuppressive tumor microenvironment (TME) in PDAC. Therefore, to comprehend the players behind this immunosuppressive TME, we integrated 10 publicly available datasets into a robust PDAC scRNA atlas of 941.608 cells covering 201 patients (159 PDAC, 24 non-cancer, and 18 metastatic PDAC). The TME showed distinct composition for each tissue, predominating a fibro-inflammatory phenotype with infiltrating Tregs and dysfunctional CD8 T cells in primary PDACs. In contrast, adjacent normal tissue exhibited enrichment of tissue-resident memory CD8 T cells (p < 0.05). Metastatic PDAC was characterized by a reduction in stromal cells combined with myeloid and naive T cells infiltration. Comparison between treated and untreated PDAC revealed the enrichment of cancer-associated fibroblasts (CAFs) in treated patients. Besides, CD8 T cells from untreated patients exhibited higher levels of genes previously linked to immunotherapy response compared to treated patients. Non-negative matrix factorization (NMF) of the malignant cells unraveled 8 distinct metaprograms (MP). MP3 and MP7 overlapped with the well-known Classical subtype, whereas MP5 correlated with the Basal subtype. MP2 exhibited high expression of MHC-II genes and was enriched in metastatic lesions. Next, we investigated the tumor architecture using public spatial transcriptomic datasets. Spot-level deconvolution revealed 12 distinct cellular niches. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications.



 Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, Alessandro Bersch Osvaldt, Simone Marcia Dos Santos Machado, Eduardo Cremonese Filippi-Chiela, Cristina Bonorino. Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B126.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a30afbd64e95a911caaa3a76b2513c14671d0c3e" target='_blank'>
              Abstract B126: Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a696fa9baeadfe94d2a764547f5853c287c52816" target='_blank'>
              Hedgehog signaling drives glial cell plasticity and oncogenic reprogramming in gastroenteropancreatic neuroendocrine neoplasms
              </a>
            </td>
          <td>
            Suzann Duan, AnneLeigh Twer, Ateh Zinkeng, Ricky A. Sontz, Juanita L. Merchant
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 KRAS^G12D mutations represent a significant oncogenic driver, prevalent in highly aggressive malignancies such as pancreatic ductal adenocarcinoma (PDAC) and colorectal cancer (CRC). The recent development of direct KRAS^G12D inhibitors has ushered in a new era of targeted therapy, offering unprecedented opportunities to address these previously intractable cancers. This study aimed to explore and optimize therapeutic strategies by thoroughly evaluating the efficacy of MRTX1133, a novel selective non-covalent KRAS^G12D inhibitor, in conjunction with various immunotherapeutic modalities, particularly those involving natural killer (NK) cells.



 We utilized patient-derived xenograft and syngeneic murine cancer models to systematically characterize alterations of immune landscapes within tumor microenvironment (TME) upon KRAS^G12D inhibition. Comprehensive and in-depth assessment of immune cell function and single-cell level transcriptional activity was performed using flow cytometry and single-cell RNA sequencing analysis.



 Our findings reveal that MRTX1133 not only suppresses tumor progression but also profoundly enhances the activation of infiltrated natural killer (NK) cells within the tumor microenvironment. Functional depletion of NK cells partially, yet significantly, attenuated the anti-tumor activity of MRTX1133, underscoring the contribution of NK cells to the observed therapeutic benefit. Notably, KRAS^G12D inhibition led to a reduction in tumor-infiltrating Gr-1^+ myeloid-derived suppressor cells (MDSCs) and increased the proportion of effector NK cells. Furthermore, MRTX1133 facilitated greater intratumoral accumulation of adoptively transferred NK cells and counteracted tumor-induced systemic immunosuppression on NK cells. Combined treatment with MRTX1133 and NK cell immunotherapies produced synergistic anti-tumor effects and significantly extended survival in preclinical models. Mechanistically, ex vivo analyses demonstrated that MRTX1133 upregulates NK cell-activating ligands, including MICA/B and ULBPs, and suppresses the production of cytokines such as IL-6 and GM-CSF by cancer cells, thereby enhancing tumor cell susceptibility to NK cell-mediated cytotoxicity and reducing MDSC proliferation.



 Collectively, our results illuminate the interplay between KRAS^G12D inhibition and immuno-microenvironmental remodeling, highlighting rational combinations of targeted and NK cell-based immunotherapies as promising strategies to potentiate therapeutic responses in KRAS^G12D-driven cancers, potentially improving clinical outcomes for patients.



 Hongyuan Chen, Tuo Hu, Chao Xu; Liangjie Chi, Fangqin Xue, Chunbo He. KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aa888d3ef7c73bc5df1193380be3a3238a9fcd" target='_blank'>
              Abstract A016: KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity
              </a>
            </td>
          <td>
            Hongyuan Chen, Tuo Hu, Chao Xu, Liangjie Chi, Fangqin Xue, Chunbo He
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64fc817809e61b5f3b749c959f0a2daae4f889fc" target='_blank'>
              QUILPEN provides independent and label-free single-cell quantification of pigmentation dynamics and organelle content
              </a>
            </td>
          <td>
            Rebecca G. Zitnay, S. Alizada, Tarek E. Moustafa, Devin Lange, Luke Schreiber, Alexander Lex, Robert L. Judson-Torres, T. Zangle, R. Belote
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Single-cell molecular profiling has emerged as a cutting-edge technology in biological research, playing a pivotal role in elucidating fundamental life processes and cellular heterogeneity. However, traditional methods generally provide end point measurements and require cell disruption, posing challenges for long-term monitoring of living cell events, e.g., cancer therapy. Tumor Treating Fields (TTFields), a noninvasive therapeutic modality, exert inhibitory effects on tumor growth and invasion by applying low-intensity intermediate-frequency alternating electric fields to tumor regions, whose molecular mechanism on cancer physiology remains poorly understood. In this study, we propose a multifunctional microchip capable of continuous in situ monitoring of molecular signatures in living single cells. The platform integrates spatial cell positioning, single-cellular multimolecule tracking with surface-enhanced Raman scattering (SERS), and TTFields treatment functionalities on a microchip. Using multiplexed analysis of liver cancer cells treated with TTFields and chemotherapy drugs, we implement spatial-temporal multiomics profiling for cancer therapy. Temporal tracking of critical molecular events associated with glycolysis, genetic stability, and membrane integrity reveals significant differences between the groups with and without TTFields treatment. Causal network analysis of molecular profiles uncovers the underlying mechanisms through which TTFields modulate oncogenic pathways. Our work demonstrates the effectiveness of the proposed microchip platform for real-time monitoring of living cell molecular dynamics, offering a powerful tool for precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889ada64ff2d00892ebd5cde7eb385bb8b31694b" target='_blank'>
              Spatial-Temporal Multiomics Profiling of Living Single Cells on Chip for Cancer Therapy.
              </a>
            </td>
          <td>
            Kunru Yu, Sheng-Zong Chen, Rong Zhu
          </td>
          <td>2025-10-12</td>
          <td>ACS sensors</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains one of the most difficult solid tumors to treat, owing in part to its remarkable cell state plasticity - a property that enables tumor cells to resist therapy, evade differentiation, and adapt to selective pressures. While targeted therapies have succeeded in subsets of patients, most tumors persist and eventually progress. To pursue a fundamentally new therapeutic strategy, we conducted an unbiased chemical screen using a custom-designed dual reporter system designed to broadcast cell state dynamics in real time by fluorescently tagging the endogenous SOX9 (aberrant stem-like) and KRT20 (differentiated) loci. This screen targeted epigenetic regulators capable of supporting CRC plasticity and preventing differentiation. We discovered that selective inhibition of histone deacetylases HDAC1/2 promotes differentiation and reduces tumor growth across human and mouse CRC models. This effect was not a consequence of cytotoxicity but rather epigenetic reprogramming. Through mass spectrometry-based histone modification profiling, we identified H3K27ac and H3K9ac as key epigenetic marks enriched at HDAC1/2-bound loci. These marks correlated with enhanced chromatin accessibility and the upregulation of differentiation-associated genes. Degradation of EP300, a major acetyltransferase, reversed the phenotype, confirming the functional relevance of histone acetylation in mediating this differentiation response. Furthermore, a focused CRISPR screen uncovered DAPK3 as a previously unrecognized downstream effector, providing a mechanistic foothold for future biomarker development. Our findings introduce a paradigm shift in CRC therapy: rather than targeting plastic tumor cells for elimination, we aim to force them into a differentiated, less adaptive state, making them more vulnerable to conventional and targeted treatments. This reprogramming effect is trackable via specific histone acetylation patterns, which we propose as candidate predictive biomarkers of response. Based on these insights, we hypothesize that HDAC1/2 inhibitors, when combined with KRAS or EZH2 inhibitors and/or chemotherapy, could yield synergistic therapeutic benefit. We are actively evaluating these combinations in preclinical models. In sum, this work positions HDAC1/2 inhibition as a precision epigenetic strategy to suppress CRC plasticity through enforced differentiation. By integrating multiomic profiling, functional genomics, and novel reporter technologies, we offer a blueprint for targeting the epigenetic underpinnings of therapy resistance in gastrointestinal malignancies, one that prioritizes reprogramming over destruction.



 Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, Sandor Spisak, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James Cleary, Jun Qi, Nilay Sethi. Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1992691ab8c82d8cb17a698db10f779b389e986a" target='_blank'>
              Abstract C031: Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains largely refractory to therapy, due in part to the complex interplay between tumor cells and their microenvironment. Human antigen R (HuR/ELAVL1), a ubiquitously expressed RNA-binding protein, is emerging as an important regulator both of tumor-intrinsic and tumor-extrinsic pathways that govern PDAC progression. While the role of HuR in promoting cancer cell survival under stress is well established, recent studies reveal its broader role in shaping the tumor microenvironment (TME), including metabolic rewiring, stromal activation, angiogenesis, and immune modulation. In this review, we examine how tumor-intrinsic HuR drives epithelial-mesenchymal transition, stabilizes key transcripts involved in metabolic adaptation, and alters the tumor secretome to influence extracellular matrix deposition and fibroblast behavior. We further explore the role of HuR in regulating immune cell function and the immune landscape of PDAC. Notably, HuR-driven TME remodeling reinforces environmental stressors that further activate HuR, establishing a feed-forward loop that drives disease progression. These findings underscore HuR as a central regulator of the PDAC TME and therapeutic resistance, and thus, highlight its potential as a target in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1b261ccac3070698258b8350c58e4ccb52e4c4" target='_blank'>
              The Role of Tumor-Intrinsic HuR in Modeling the Pancreatic Tumor Microenvironment: Molecular Mechanisms and Therapeutic Opportunities.
              </a>
            </td>
          <td>
            Katherine R. Pelz, Kyle P Gribbin, Tess Sevetson, Jonathan R. Brody
          </td>
          <td>2025-10-13</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Renal cancer, particularly clear cell renal cell carcinoma (ccRCC), is characterized by significant intratumoral heterogeneity, which poses challenges for diagnosis and treatment. Single-cell sequencing (SCS) provides unprecedented insights into the cellular landscape of renal cancer, allowing for detailed characterization of tumor heterogeneity at the single-cell level. This review highlights how SCS has been instrumental in elucidating the origins of different renal cancer subtypes, understanding mechanisms of tumor initiation and progression, and dissecting the complex tumor microenvironment (TME). It discusses the identification of novel biomarkers and therapeutic targets, as well as the potential of SCS to inform personalized treatment strategies. The review also explores the integration of SCS with spatial omics technologies, which enhances the understanding of cellular interactions within their spatial context. Moreover, it addresses the challenges and future directions in applying SCS to clinical practice, emphasizing its significance in advancing renal cancer biology and improving clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03a5f0a7b414a81333643cfcf0b78e2ab4708d98" target='_blank'>
              Single-cell sequencing technology in renal cancer: insights into tumor biology and clinical application
              </a>
            </td>
          <td>
            Hanzhong Zhang, Ying Liu, Wenqiang Liu, Anqi Lin, Yu Fang, Le Qu, Xu Zhang, Peng Luo, Linhui Wang, Aimin Jiang
          </td>
          <td>2025-10-01</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86ecdda13a72d7b290cf8fe9dd99c8ce634923c0" target='_blank'>
              Galectin-9-driven immunosuppressive macrophage population shapes the lymph node metastatic microenvironment.
              </a>
            </td>
          <td>
            Yao Zhou, Xiaowei Liu, Guixiu Xiao, Genpeng Li, Xiujing He, Leyi Tang, Xueyan Wang, Yujia Lin, Jianyong Lei, Jing Jing, Hubing Shi
          </td>
          <td>2025-10-25</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="

 Prostate cancer (PCa) is a leading cause of male cancer mortality, with metastasis and immune evasion posing major therapeutic challenges. DOCK3, a guanine nucleotide exchange factor implicated in cytoskeletal dynamics, is poorly characterized in PCa. This study investigates DOCK3’s role in PCa metastasis and tumor immune microenvironment (TIME) remodeling.



 Multi-omics analyses integrated bulk RNA-seq from TCGA-PRAD (499 tumors/52 normals), scRNA-seq from GEO (45,325 cells), and genomic data. We performed: Differential expression analysis (DESeq2), Immune deconvolution (CIBERSORT,ssGSEA, xCell), WGCNA co-expression networks, Tumor mutational burden (TMB) assessment, Distant metastasis (M1 vs. M0) association studies, scRNA-seq clustering (Harmony/UMAP) and DE testing. Statistical significance thresholds: |log2FC|>1, padj<0.05.




 DOCK3 expression was found to be significantly elevated in metastatic (M1) tumors compared to primary (M0) tumors (p<0.05) and demonstrated a strong positive correlation with a higher tumor mutational burden (TMB) in metastatic samples (p<0.001). Cellular specificity analysis revealed that DOCK3 was exclusively and highly enriched within malignant epithelial and stromal cells, specifically in Cluster 6, where it exhibited a log2 fold-change of 9.13 (padj<1e-200) and was expressed in 54% of cells, compared to a negligible presence in all other clusters. In the tumor microenvironment, elevated DOCK3 expression was associated with a significant increase in cytotoxic immune infiltration, notably of CD8
 +
 T and Natural Killer cells, a finding consistently supported by multiple computational algorithms (all p<0.05). Clinically, a high level of DOCK3 was significantly associated with metastatic status (p<0.01), whereas high expression of CDKN3 was correlated with advanced disease features, including higher Gleason scores (3-5) and T-stage (T2-T4) (p<0.01). Furthermore, significant differences in immune infiltration patterns were observed between clusters. Pathway enrichment analysis of genes co-expressed with DOCK3, identified through the WGCNA Green Module, indicated significant involvement in biological processes such as cytoskeletal reorganization, muscle contraction, and metabolic pathways (FDR<0.01).




 DOCK3 drives PCa metastasis through cytoskeletal dynamics while paradoxically promoting an immunologically active microenvironment. Its tumor-specific expression and association with aggressive clinical features nominate DOCK3 as a novel biomarker for risk stratification and a promising therapeutic target for combinatorial immunotherapy in immunologically “cold” PCa.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fbd5c9d8d606fe242b51bce0cd05f9f0b2729c1" target='_blank'>
              DOCK3 orchestrates metastasis and immune microenvironment in prostate cancer
              </a>
            </td>
          <td>
            Jiaxue Han, Ming Zhang, Haipeng Zhou, Qiao Xiong, Xin Zhong, Ping Tan
          </td>
          <td>2025-10-22</td>
          <td>Frontiers in Urology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Understanding the functional impact of bacterial genetic diversity is crucial for linking pathogen variants to clinical outcomes. Here, we introduce a high-throughput cytological profiling pipeline optimized for Mycobacterium tuberculosis (Mtb) clinical strains, integrating OD-calibrated feature analysis and high-content microscopy. Our system quantifies single-bacterium morphological and physiological traits related to DNA replication, redox state, carbon metabolism, and cell envelope dynamics. Applied to 64 Mtb clinical isolates from lineages 1, 2, and 4, the approach revealed that cytological phenotypes recapitulate genetic relationships and exhibit both lineage- and density-dependent dynamics. Notably, we identified a link between a convergent “small cell” phenotype and a convergent ino1 mutation that is associated with the presence of an antisense transcript, suggesting a potential non-canonical regulatory mechanism under selection. In summary, we present a resource-efficient approach for mapping Mtb’s phenotypic landscape, uncovering cellular traits that underlie its evolution and providing new insights into the functional consequences of bacterial genetic diversity. IMPORTANCE Understanding how genetic variation in Mycobacterium tuberculosis (Mtb) shapes its physical traits is essential to unraveling the evolution of this global pathogen. Here, we introduce a systematically optimized, high-throughput imaging platform for the comprehensive characterization of Mtb clinical strains. We demonstrate that Mtb’s phenotypic manifestation is shaped by both genetic background and culture density. By accounting for these factors, our analysis linked distinct cellular dynamics to specific lineages, sublineages, and even single nucleotide variations. Notably, we linked a recurring mutation to a unique cell-shortening phenotype, finding that it potentially acts by creating a cryptic antisense transcript. This platform provides a powerful framework for systematically dissecting the physiological dynamics underlying Mtb evolution and identifying new therapeutic vulnerabilities of this deadly pathogen. Understanding how genetic variation in Mycobacterium tuberculosis (Mtb) shapes its physical traits is essential to unraveling the evolution of this global pathogen. Here, we introduce a systematically optimized, high-throughput imaging platform for the comprehensive characterization of Mtb clinical strains. We demonstrate that Mtb’s phenotypic manifestation is shaped by both genetic background and culture density. By accounting for these factors, our analysis linked distinct cellular dynamics to specific lineages, sublineages, and even single nucleotide variations. Notably, we linked a recurring mutation to a unique cell-shortening phenotype, finding that it potentially acts by creating a cryptic antisense transcript. This platform provides a powerful framework for systematically dissecting the physiological dynamics underlying Mtb evolution and identifying new therapeutic vulnerabilities of this deadly pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56fa61f881d5174f374abc12d00931b99537639c" target='_blank'>
              High-throughput cytological profiling uncovers genotype-phenotype associations in Mycobacterium tuberculosis clinical isolates
              </a>
            </td>
          <td>
            Qingyun Liu, Yue J Liu, Ruiyuan Liu, Peter H. Culviner, Xin Wang, I. D. Wolf, Ken Chen, Yiwang Chen, Yi Xiao, Guiming Zhang, Rongfeng Sun, Shoko Wakabayashi, Nicole C Howard, Mingyu Gan, Eric J. Rubin, Sarah M. Fortune, Junhao Zhu
          </td>
          <td>2025-10-09</td>
          <td>mSystems</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a0e9b33d2889e4173bb5bf055bf973eb4a22cb0" target='_blank'>
              Cell state dynamics during early stages of serous endometrial carcinoma
              </a>
            </td>
          <td>
            A. Flesken-Nikitin, Matalin G Pirtz, Christopher S Ashe, L. H. Ellenson, Anna Yemelyanova, Benjamin D Cosgrove, Alexander Yu. Nikitin
          </td>
          <td>2025-09-09</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND
Changes in the cyst microenvironment in Polycystic Kidney Disease (PKD) may drive progressive cyst formation. Bulk- and single-cell RNA sequencing have advanced our understanding of altered signaling; however, the lack of spatial information has limited our insights into local gene expression and cellular communication near cysts.


METHODS
We used wild-type and Pkd1-deficient mouse kidneys to generate 10x Genomics Visium Spatial Gene Expression datasets. Utilizing our single-cell mouse kidney atlas and single cell sequencing data for spot deconvolution, we enhanced resolution and estimated enriched cell types. We analyzed spatial gene expression patterns and used a cyst-centered analysis to identify cyst-associated gene signature. Cell communication near cysts was investigated, identifying key ligand-receptors. Prioritized key factors were validated in tissues.


RESULTS
We observed enrichment of fibroblasts, injury-repair-related cell types, and diverse immune populations in PKD. Injury-repair-related cells were exclusively observed in PKD, predominantly localized within immune cell-dense regions near cysts. These cells collectively contributed to the altered gene expression profile in PKD, including cyst-associated signature genes related to inflammatory processes. Analysis of cellular communication in less-inflamed regions around cysts revealed the involvement of multiple cell types. Key ligand-receptor interactions were associated with cytokine signaling, fibrosis, cellular development, and repair. These included Angpt2, C3, Csf1, Cxcl12, Il34, Gas6, Il16, Mdk, Mif, Ptn, Sfrp2, Spp1, Sdc1, Tnc, Tnfsf12, Wnt5a. In addition, ECM proteins implicated in immune response, ECM remodeling, cell adhesion, and cell signaling were identified, such as Adam9, Adam10, Col1a1, Col3a1, Col4a2, Lamb2, Lamc1, Efnb1, Efnb2, Thbs1, Thbs2, and Vcam1. IHC confirmed expression of Syndecan-1-Collagen IV, Midkine-Integrin β1, CSF-1, Pleiotrophin, and Tenascin-C in cystic kidneys.


CONCLUSIONS
Spatial transcriptomics in PKD revealed enrichment of (myo)fibroblasts, immune, and injury-repair-related cells near cysts, creating a (pro)inflammatory and (pro)fibrotic niche. Key ligand-receptor and ECM interactions were identified and validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecdcf5e98138976e4b0b6f2eafc40e1b84148be0" target='_blank'>
              Spatial Transcriptomics Reveals Injured Cells, Signature Genes, and Communication Patterns in the Cyst Microenvironment of Polycystic Kidney Disease.
              </a>
            </td>
          <td>
            Sevtap A Yasinoglu, Claudio Novella-Rausell, L. Wisse, K. Dijkstra, A. Mahfouz, Hans J. Baelde, DJM. Peters
          </td>
          <td>2025-10-10</td>
          <td>Journal of the American Society of Nephrology : JASN</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4ae2d5f1f6f184ad5a2b71dcc7e35b4feeb422f" target='_blank'>
              Functional biomaterials and machine learning approaches for phenotyping heterogeneous tumor cells and extracellular vesicles.
              </a>
            </td>
          <td>
            Rutwik Joshi, R. Ahmad, Karl Gardner, Hesaneh Ahmadi, Chau-Chyun Chen, Shannon Stott, Wei Li
          </td>
          <td>2025-09-18</td>
          <td>Biomaterials science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Cardiac issues following radiotherapy are increasingly prevalent among patients with thoracic cancer and coronary disease. However, the mechanisms underlying radiotherapy‐induced plaque instability and changes in plaque characteristics on imaging remain unclear. This study used single‐cell RNA sequencing to identify key features of vulnerable plaques following radiotherapy. Methods We applied dual‐mode synchronized optical coherence tomography‐intravascular ultrasound imaging in a rabbit model to reveal the characteristics of vulnerable plaques at various time points postradiotherapy. We then conducted single‐cell RNA sequencing on atherosclerotic lesions from ApoE −/− (apolipoprotein E−/−) mice across 3 stages: nonirradiated, early irradiation, and advanced irradiation. Bioinformatics was used to analyze cell populations, trajectories, and interaction differences among the groups. This was followed by pathological staining, flow cytometry, gene knockout, and antibody depletion. Results In vivo optical coherence tomography‐intravascular ultrasound imaging at the initial stage accurately evaluated radiation exposure's effect on the stability of rabbit carotid plaques. High‐throughput single‐cell transcriptomics mapped the radiotherapy‐induced immune system remodeling in atherosclerotic lesions, and identified a granzyme K+ CD8+ (cluster of differentiation 8+) T‐cell subpopulation. The infiltrating cells highly expressed the cytotoxic molecule GZMB (granzyme B) in the plaques. The absence of CD8+ T cells or GZMB ameliorated radiotherapy‐exacerbated atherosclerotic lesions. This demonstrates the pathogenic function of GZMK+ (granzyme K+) CD8+ cytotoxic T cells in promoting radiotherapy‐induced atherosclerotic lesion progression. Conclusions We identified characteristic arterial imaging changes resulting from radiation injury, accurately and dynamically assessing the effects of radiation exposure on plaque stability. Radiation exposure affected plaque stability by modulating GZMK+ CD8+ T‐cell activation and GZMB function, suggesting GZMB as a mediator of radio‐cardiovascular injury and a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ade88282c0b97eb135a560a965611a6206d8944e" target='_blank'>
              Single‐Cell RNA Sequencing Reveals Granzyme K+ CD8 + T Cells as a Target for Mitigating Plaque Instability Following Radiation
              </a>
            </td>
          <td>
            Manru Shi, Quan Lin, Shaohong Fang, Cheng Ma, Weiwei Xu, Di Sun, Naixin Wang, J. Cui, Zhonghua Tong, Yuwu Chen, Xing Luo, Sining Hu, Tuo Huang, Yunlong He, Yang Cao, Jiayin Liu, Jiaqi Li, Anxin Gu, Xiuwen Lan, Meng Chi, Chao Wang, Jiaqi Gu, Siqi Tian, Jiaao Liu, Zixuan Liu, Maomao Zhang, M. E
          </td>
          <td>2025-09-05</td>
          <td>Journal of the American Heart Association: Cardiovascular and Cerebrovascular Disease</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Approximately 40% of patients with pancreatic ductal adenocarcinoma (PDAC) receive no benefit from surgical resection and systemic chemotherapy and experience metastatic disease recurrence within 12 months. The extreme heterogeneity of the cells that populate PDAC tumors has been a significant impediment to identifying the cells that likely seed early recurrence. Previous studies show that metastatic spread is driven by specific cells with hybrid epithelial-mesenchymal characteristics, but despite the identification of numerous molecular features associated with epithelial, hybrid, and mesenchymal PDAC cancer cells, it has not been possible to specifically identify the cells that seed metastatic recurrence in human tissue. We have previously shown that PDAC cell glycan signatures are altered as a direct result of the metabolic and transcription changes that occur as PDAC cancer cells undergo EMT. Now we integrated the glycan signatures with known protein markers and morphological analyses using our pipeline for multiplexed immunofluorescence analysis to identify distinct subpopulations of epithelial, hybrid, and mesenchymal PDAC cells. In cell-level analyses of immunofluorescence images of 24 PDAC cell lines, these subpopulations revealed epithelial to hybrid to mesenchymal conversions along 3 distinct trajectories, which were defined by transcription factors and proteins previously associated with epithelial or mesenchymal states—TP53, TP63, GATA6, and MYC—and further distinguished by specific glycan signatures and protein markers of metabolism and cellular differentiation. The integrated transcription factor, glycan, and protein signatures identified the previously described classical and squamous PDAC subpopulations and defined a new signature for a fully mesenchymal subpopulation. The signatures differentiated between PDAC subpopulations in primary human tumors and in metastatic liver lesions and showed a patient-matched correspondence between the primary tumors and metastatic lesions in specific hybrid PDAC cells within distinct trajectories, potentially revealing the PDAC cells seeding metastatic recurrence. These results suggest that the development of mesenchymal and metastatic PDAC cells occurs through distinct, stepwise trajectories with divergent morphological, metabolic, molecular, and glycan-based signatures. These findings will/have the potential to inform the development of treatments that specifically target the PDAC cells that seed early metastatic recurrence.



 ChongFeng Gao, Zachary Klamer, Ali Moursy, Galen Hostetter, Paul M. Grandgenett, Peter Allen, Brian Haab. Resolving PDAC cell subpopulations that seed metastatic recurrence [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B125.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246925305285b4b397bb54f4914db32c1466cbe8" target='_blank'>
              Abstract B125: Resolving PDAC cell subpopulations that seed metastatic recurrence
              </a>
            </td>
          <td>
            Chongfeng Gao, Zachary L. Klamer, Ali Moursy, G. Hostetter, P. Grandgenett, Peter Allen, Brian Haab
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy driven by oncogenic KRAS and inflammation-induced cellular heterogeneity, yet the mechanisms underlying tumor-initiating cells (TICs) emergence and maintenance remain unclear. Our study establishes LY6D as a marker of persistent TICs that orchestrate pancreatic cancer progression across all disease stages. Single-cell RNA sequencing of inflammation-driven PDAC models reveals that LY6D+ TICs specifically originate from KRAS-mutant acinar-to-ductal metaplasia (ADM) lesions under inflammatory conditions, maintaining conserved stemness properties and exhibiting a distinctive oxidative phosphorylation (OXPHOS) dependency throughout tumor evolution. Moreover, genetic ablation of Ly6d in Kras G12D pancreata delays tumorigenesis, while forced Ly6d expression enhances tumorigenic potential and metastatic capability. Mechanistically, LY6D-despite lacking intracellular domains-scaffolds lipid raft-associated kinase networks and FOSL1-dependent epigenetic reprogramming to establish a ​stable pro-tumorigenic state. Clinically, LY6D+ cells are enriched in human PDAC and exhibit conserved stemness and epithelial-mesenchymal transition (EMT) properties. Strikingly, LY6D expression levels demonstrate PDAC-restricted prognostic power. Our work defines LY6D as pan-stage TICs marker linking cellular plasticity to PDAC initiation and progression, offering new avenues for early detection and interception of this lethal malignancy.



 Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue. LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B119.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f74adf609c9f9d814e1f1f768c348943a8c10b" target='_blank'>
              Abstract B119: LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Analyses of patient-derived cell lines have greatly enhanced discovery of molecular biomarkers and therapeutic targets. However, characterization of cellular morphological properties is limited. We studied cell morphologies of human pancreatic adenocarcinoma (PDAC) cell lines and their associations with drug sensitivity, gene expression, and functional properties. By integrating live cell and spatial messenger RNA imaging, we identified KRAS inhibitor–induced morphological changes specific for drug-resistant cells that correlated with gene expression changes. We then categorized a large panel of patient-derived PDAC cell lines into morphological and organizational subtypes and found differences in gene expression, therapeutic targeting potential, and metastatic proclivity. Patterns of cancer cell organization in human PDAC tissues stratified distinct gene expression signatures with clinical significance. In summary, we highlight the potential of cell morphological information in rapid, cost-effective assays to aid precision oncology efforts leveraging patient-derived in vitro models and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb047a3ab41c4e28c6535b371221906a6c927ab" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor matrix stiffness plays a critical role in cancer progression, metastasis, and therapy resistance. Although traditional biophysical methods have shed light on the impact of matrix stiffness on tumor behavior, these techniques are confined to measuring the physical properties of the tumors. In this study, we leveraged RNA-seq data to predict tumor matrix stiffness, aiming to reveal mechanical properties by molecular signatures across various cancer types. To this end, we systematically analyzed RNA-seq data from tumors of varying stiffness levels to identify stiffness-associated gene signatures. With these molecular signatures, we developed a computational model for predicting tumor matrix stiffness and further applied it to The Cancer Genome Atlas (TCGA) dataset. Our analysis revealed significant differences in the tumor microenvironment as well as immune response between soft and stiff tumor samples, suggesting that tumor rigidity impacts not only cellular behavior but also characteristics of the tumor microenvironment. These findings underscore the potential of RNA-based stiffness models to enhance our comprehension of tumor mechanics and cancer biology, thereby facilitating the development of innovative targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd0a56130a8bfa1d6781b487c86488f3edefb603" target='_blank'>
              Pan-cancer analysis reveals molecular signatures for predicting matrix stiffness in solid tumors.
              </a>
            </td>
          <td>
            Gongyu Tang, Xinyi Liu, Yuanxiang Li, Yunfei Ta, Minsu Cho, Hua Li, Xiaowei Wang
          </td>
          <td>2025-09-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/841868dae6381828134e074133437d8b76fe3212" target='_blank'>
              Bleb-based extravasation: conserved morphodynamics, divergent calcium control
              </a>
            </td>
          <td>
            Mizuki Morita, Manami Morimoto, Junichi Ikenouchi, Bertrand Pain, Yuji Atsuta, Yoshiki Hayashi, Takayuki Teramoto, Daisuke Saito
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most frequent brain tumors in children and comprise a heterogeneous group of tumors with different locations, histologic subtypes, ages at presentation, and clinical behavior. Tumors frequently respond to treatment with chemotherapy or radiation therapy, but they can regrow after a period of quiescence, requiring further therapy. Thus, a deeper understanding of the molecular processes involved in these tumors is required to develop therapeutic strategies that are effective against their disease mechanisms. To better understand the cellular behaviors of this heterogenous group of tumors, we have employed single-cell and single-nuclei RNA sequencing technologies to analyze a large-scale dataset (>250,000 cells) of pLGGs. Analysis of this data identified a heterogenous population of cell types and cell states, detecting mature and progenitor-like astrocytes and oligodendrocytes, as well as cells exhibiting senescence or cycling programs. Moreover, we identify a significant immune infiltrate, comprised primarily of microglia. In addition to heterogeneity within pLGG tumors, heterogeneity between LGG subtypes represents another layer that stratifies pLGG biology. We performed a compositional analysis of the cell types present in these tumors and compared transcription signatures and gene expression programs across shared cellular populations of histologically and genetically distinct pLGGs. Finally, we optimized our integration and batch correction analyses by using external 293T cells as spike in controls during our single-cell and single-nuclei data generation steps to determine the most suitable method for batch-effect removal. Our analysis of human pLGGs at the single-cell and single-nuclei resolution provides critical insight into the heterogenous biological activities that constitute these tumors.



 Michelle Boisvert, Ashwyn Perera, Alexandra L. Condurat, John Jeang, Jessica W. Tsai, Dana Novikov, Kevin Zhou, Madison Chacon, Jeromy DiGiacomo, Rushil Kumbhani, Dayle Wang, Michael D. Taylor, Jordan R. Hansford, Louise Ludlow, Nada Jabado, Keith L. Ligon, Rameen Beroukhim, Pratiti Bandopadhayay, David T.W. Jones. Understanding the transcriptional heterogeneity of pediatric low-grade gliomas and its implication for tumor pathophysiology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87fff9552d8445c379c8cd3282dd22472d8f4d25" target='_blank'>
              Abstract A014: Understanding the transcriptional heterogeneity of pediatric low-grade gliomas and its implication for tumor pathophysiology
              </a>
            </td>
          <td>
            Michelle Boisvert, Ashwyn A. Perera, Alexandra L. Condurat, John Jeang, Jessica W Tsai, Dana Novikov, Kevin N Zhou, Madison S. Chacon, Jeromy J. Digiacomo, Rushil Kumbhani, Dayle K. Wang, Michael D. Taylor, J. Hansford, Louise E. Ludlow, Nada Jabado, Keith L. Ligon, R. Beroukhim, P. Bandopadhayay, David Jones
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>151</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The most common cause of death in pediatric cancer patients is a treatment‐resistant tumor, compounded by metastatic spread, making surgery, radiation therapy, and chemotherapy unfeasible as curative treatment options. However, the mechanisms behind metastatic spread in pediatric tumors remain largely unexplored. We conducted whole‐genome copy number profiling on 171 primary tumor and metastases samples from 17 patients with neuroblastoma, Wilms tumor, or gonadal tumors, and performed targeted deep sequencing on a subset. Phylogenetic reconstruction enabled spatiotemporal tracking of subclones. In total, 11 of 17 patients displayed at least one metastasis arising earlier, defined as occurring before the most recent common ancestor in the primary tumor. In eight patients, metastatic spread was observed several times during tumor evolution, with different subclones from the same primary tumor having metastatic capability, even colonizing the same site. Strikingly, dissemination between metastases (intermetastatic spread) occurred in eight of nine patients with metastases in at least two different sites, indicating that this is a common phenomenon in pediatric malignancy. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24f7bd63ad2744ced5ab2ba8c285ffb7719db78" target='_blank'>
              Tracing metastatic spread in pediatric solid tumors using copy number and targeted deep sequencing
              </a>
            </td>
          <td>
            Natalie Andersson, Michele Ferro, Caroline Jansson, Subhayan Chattopadhyay, Jenny Karlsson, David Gisselsson
          </td>
          <td>2025-09-23</td>
          <td>The Journal of Pathology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma (PDA) has poor response to chemotherapy with a strong likelihood of cancer recurrence. While previous studies have examined patient samples collected before and after chemotherapy, few have analyzed matched samples from the same patients that allow us to deconvolute interpatient heterogeneity, a large limitation to PDA studies. Here, we study the effects of chemotherapy on tumor cells and their surrounding microenvironment using scRNA sequencing of matched pre- and post- treatment tissue biopsies from 5 patients. We utilized the Seurat v4 pipeline to observe transcriptomic differences and corresponding putative cellular interactions in both treatment states. Unsurprisingly, we found insufficient tumor epithelial cells post-treatment due to the intended cytotoxic effects of chemotherapy, leading us to focus on the tumor microenvironment where we had sufficient cells to analyze. We identified that immune cells in the TME are heterogeneously affected by chemotherapy treatment with patient specific differences, but we did identify a unanimous increase in CXCR4 in CD4 and CD8 positive T cells after treatment. Analysis of CAFs revealed that a subset of fibroblasts lose myofibroblastic features and increasingly acquire 'neural'-like features after treatment. This suggests that chemotherapy-altered fibroblasts may contribute to a pro-tumor microenvironment that facilitates recurrence through neuron–CAF–tumor interactions. Through inferred ligand-receptor pairs identified with the CellChat v2 pipeline, we observed that increased CXCL12 in these neural CAFs interacts with the increased CXCR4 on T-cells. We validated the expression of these genes in patients by staining human patient PDAC tissues (both treated and treatment naïve tissues) for neural genes that were highly enriched post-treatment in our single cell sequencing dataset. We have also begun inhibiting these genes of interest in CAFs and are exploring the effects of these ‘neural’-CAFs on patient-derived tumor organoids in culture.



 Aylin Z. Henstridge, Padma Kadiyala, Ahmed Elhoissiny, Jianhua Liu, Georgina Branch, Nandini Arya, Vaibhav Sahai, Nicole Peterson, Jorge Machicado, Richard Kwon, Allison Schulman, Erik Wamsteker, George Phillips, Martin Fernandez-Zapico, Mark Truty, Timothy Frankel, Fillip Bednar, Marina Pasca di Magliano, Costas Lyssiotis, Eileen S. Carpenter. An increase in neural cancer associated fibroblasts following exposure to chemotherapy in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B079.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e666941a86feb629e0cf1155d415645c6c810d" target='_blank'>
              Abstract B079: An increase in neural cancer associated fibroblasts following exposure to chemotherapy in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Aylin Z. Henstridge, Padma Kadiyala, Ahmed Elhoissiny, Jianhua Liu, Georgina Branch, Nandini Arya, Vaibhav Sahai, Nicole Peterson, Jorge D. Machicado, Richard S Kwon, Allison Schulman, Erik Wamsteker, George Phillips, Martín E Fernandez-Zapico, M. Truty, Timothy L. Frankel, Fillip Bednar, Marina Pasca di Magliano, C. Lyssiotis, Eileen S. Carpenter
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7db095a53d3537670668dae17c6ccb821695327" target='_blank'>
              Dissecting the epigenome dynamics in human immune cells upon viral and chemical exposure by multimodal single-cell profiling
              </a>
            </td>
          <td>
            Irem B Gündüz, Bei Wei, Derek C Chen, Wenliang Wang, Manoj Hariharan, Todd Norell, Timothy J. Broderick, M. T. McClain, Lisa L. Satterwhite, Thomas W. Burke, Elizabeth A. Petzold, Xiling Shen, Christopher W. Woods, Vance G. Fowler, F. Ruffin, Parinya Panuwet, Dana B. Barr, Aaron J. Wilk, Madeline J. Lee, Catherine Blish, Flora Castellino, Anna Maria Walley, Thomas G Evans, Joseph R Ecker, Fabian Müller, W.J. Greenleaf
          </td>
          <td>2025-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="This editorial discusses Alpsoy et al’s significant study of prognosis of pancreatic ductal adenocarcinoma (PDAC), which lacks histopathological markers. This study evaluated the synergistic prognolymphocytes. Peritumoral budding is significantly correlated with tumor volume, while intratumoral budding is closely related to lymph node metastasis. Peritumoral budding and intratumoral budding are confirmed as independent adverse prognostic factors, and their high levels of expression are associated with immature stromal phenotypes, suggesting the key role of epithelial-mesenchymal transition. These breakthrough findings provide a new multidimensional biomarker system for the prognostic assessment of PDAC, and promote the clinical transformation process of incorporating tumor budding indicators into the pathological reporting process. However, the complexity and spatiotemporal heterogeneity of the tumor microenvironment require us to go beyond traditional morphological analysis and move towards multiomics integration and dynamic monitoring. Through standardized pathological assessment, innovative treatment strategies and interdisciplinary collaboration, it is expected to transform tumor microenvironment-related markers into clinically applicable indicators, ultimately improving the treatment predicament of PDAC. This editorial intended to summarize relevant studies and share some of our views, in order to offer perspectives for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a304affe959bbde58f99c98545a1534cc99601" target='_blank'>
              Microenvironmental code of pancreatic ductal adenocarcinoma: The prognostic symphony of budding, matrix and lymphocytes
              </a>
            </td>
          <td>
            Qing-Qing Zhou, Yi-Meng Ren, Su-Ming Shi, Teng-Fei Liu
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5bbf3daa9e6f41e402a48c9bc2bd5b7f13849c" target='_blank'>
              Systematic elucidation and pharmacologic targeting on non-oncogene dependencies in imatinib-resistant gastrointestinal stromal tumor
              </a>
            </td>
          <td>
            Prabhjot S. Mundi, A. Grunn, A. Kojadinovic, Charles Karan, Ronald B. Realubit, Cristina I. Caescu, H. Hibshoosh, Mahalaxmi Aburi, , M.A. Ingham, D. Evans, S. Rothschild, Gary K. Schwartz, Andrea Califano
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) is the most common form of kidney cancer, leading to 179 000 cancer-related deaths annually. ccRCC initiating cells (CICs) are thought to drive tumor initiation, growth, therapy resistance, and metastasis, yet their molecular characteristics remain poorly defined. This study aims to identify putative CICs and their essential genes using a cell marker-agnostic strategy. Methods We performed a comprehensive analysis of ccRCC transcriptomes at single-cell resolution, developed patient-derived xenograft (PDX) and 3D patient-derived organoids (PDO) models of ccRCC, and conducted functional examinations in these models to investigate our findings. Results Computational modeling of tumor formation using single-cell RNA velocity analysis of five primary and metastatic ccRCC-PDXs revealed a minor cell population as the origin of other tumor cells, representing putative CICs. Pathway and network analyses suggested that a core network of proteins, conventionally known to regulate mitosis, are highly active in CICs and may be essential for their function. These proteins were expressed in PDOs and PDX-derived spheroids established following a CIC enrichment protocol. Spheroid cells exhibited higher tumorigenicity and colony formation ability than parental tumor cells, as confirmed by in-vivo injection in nude mice and in-vitro colony formation assays. Successive in vivo passaging confirmed the self-renewal capacity of spheroid-derived tumors. Pharmacological blockade of candidate proteins elicited dose-dependent inhibitory effect on spheroid and colony formation, with in-vivo validation showing that blocking these proteins significantly delayed tumor growth and more efficiently prevented tumor formation in mice. These results highlight the importance of these proteins in the cancer-initiating abilities of malignant cells. Interestingly, our results suggest that the identified target proteins may elicit their CIC-related function independently from their mitosis regulatory roles. Conclusions We identified and validated essential proteins in RCC-CICs, supported by single-cell transcriptome data and RCC spheroids and PDX models. Targeting the vulnerabilities of RCC-CICs, given their role in tumor initiation, progression, and therapy resistance, offers significant potential for developing new anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edac148fb846da9cb774550aaf710d22fbf68ef2" target='_blank'>
              64defining cancer initiating cells and their vulnerabilities in renal cell carcinoma
              </a>
            </td>
          <td>
            Zohreh Mehrjoo, H. Kuasne, Ariel Madrigal Aguirre, Ali Shahini, M. Annis, Anne-Marie Fortier, Tianyuan Lu, Larisa Morales Soto, Hong Zhao, D. Zuo, V. Pilon, M. Dankner, Tamiko Nishimura, Kevin Petrecca, Jonathan Spicer, Peter Siegel, Simon Tanguay, H. Najafabadi, Morag Park, Yasser Riazalhosseini
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce6be4d706b4653ff7064a56c5297b13de1df65" target='_blank'>
              Travelling Waves in Gene Expression: A Mathematical Model of Cell-State Dynamics in Melanoma
              </a>
            </td>
          <td>
            Charlotte Taylor Barca, Rotem Salmi-Leshem, Vishaka Gopalan, Sarah Woolner, Kerrie L. Marie, Gareth Wyn Jones, Oliver E. Jensen
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="PURPOSE
Medulloblastoma (MB) is a common pediatric brain tumor with distinct molecular subgroups, of which, Group 3 MB is associated with increased recurrence, metastatic potential and poor patient outcomes. Small molecule inhibitors targeting BMI1 have been shown to be efficacious against several types of malignant tumors, including pediatric MB. While our previously published in vivo study provides a promising proof-of-concept for the therapeutic targeting of BMI1 in Group 3 MB with small molecule inhibitor, it is not sufficient to eradicate the tumour.


EXPERIMENTAL DESIGN
In this study, following preclinical validation of BMI1 inhibitor PTC-596, DNA barcoding technology was leveraged to profile in vivo clonal dynamics of Group 3 MB in response to the established chemoradiotherapy regimen alone and in combination with PTC-596. Following demonstration of a small number of treatment-refractory clones we sought to identify potential druggable molecular vulnerabilities by utilizing phosphoproteomic profiling and genome-wide CRISPR screening.


RESULTS
By comparing the changes in phosphorylation pattern of key signaling kinases post PTC-596 treatment with the list of sensitizer genes from in vitro genome-wide CRISPR/Cas9 screen and to the essential genes in human neural stem cells (hNSCs), we identified several context-specific regulators of mTOR, AKT and PLK1 pathways. Subsequently, targeting the PI3K pathway with Enzastaurin was shown to be most meanable to synergistic targeting alongside BMI1 inhibition.


CONCLUSION
This work provides the foundation for clinical validation of small-molecule inhibitors synergistic with PTC-596 to improve the durability of remissions and extend survival of patients with treatment-refractory Group 3 MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254ad36a007c276e881f281d38c624b50e4a39ec" target='_blank'>
              Leveraging Medulloblastoma Clonal Dynamics to Overcome Treatment Resistance.
              </a>
            </td>
          <td>
            D. Bakhshinyan, Stefan Custers, Laura Escudero, Yujin Suk, Kevin R. Brown, Hardikkumar Patel, A. Adile, C. Chokshi, M. Shaikh, D. McKenna, M. Qazi, Kui Zhai, D. Tieu, Katherine Chan, Marla Weetal, C. Venugopal, Jason Moffat, Sheila K Singh
          </td>
          <td>2025-10-15</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c7c66e7f054de7552ccd4ec8d0186447c75c2a" target='_blank'>
              Single-cell Genomic Copy Number Evolution Reveals Frequent Loss of the Y chromosome in Esophageal Adenocarcinoma
              </a>
            </td>
          <td>
            Shiwei Yin, Na Zhong, Swati Agrawal, Bingru Feng, Sneha Pradhan, Nathan K. Jobalia, Maryam Vaziripour, Sampath K Poreddy, L. Kresty, Erin A Gillaspie, Nicholas E Dietz, W. Al-Refaie, K. Nandipati, Yusi Fu, Jun Xia
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Notch is a contact-dependent signaling pathway that plays critical roles in spatiotemporal patterning during embryogenesis. Beyond its role in development, Notch signaling regulates tissue patterning in adult organs in response to tissue injury or cancer. In the context of cancer, the role of Notch signaling has been extensively characterized in cancer cells, T cells, and endothelial cells. Our previous work has implicated Notch signaling in the myeloid and fibroblast compartments of the pancreatic tumor microenvironment (TME) in the tumor immune response, an area that has been previously understudied. In our current work we use genetically engineered mouse models to inhibit canonical Notch activation in fibroblasts (Pdgfra CreERT2/+, LSL-DNMAML) and myeloid cells (LysM Cre/+, LSL-DNMAML) separately. In both an orthotopic transplant and genetic model (Ptf1a FlpO/+;FSF-Kras G12D/+) of pancreatic cancer progression, inhibition of fibroblast-specific Notch activation had limited effects on disease progression and the transcriptional profile of the fibroblasts themselves. This is consistent with previous evidence of more limited Notch activation in pancreatic cancer-associated fibroblasts and suggests a primary role for fibroblasts in providing Notch ligand (JAG1) to neighboring cells. In contrast, inhibition of myeloid-specific Notch activation resulted in smaller orthotopic tumors and slowed progression in a genetic model of pancreatic neoplasia. These samples had reduced overall macrophage infiltration, and a secondary alteration of the fibroblast phenotypes was also seen. Collectively, these results suggest that Notch signaling in the myeloid compartment supports tumor progression and regulates multiple components of the pancreatic TME. Future studies will further interrogate how myeloid-specific Notch activation promotes disease progression and affects Notch signaling activation in adjacent compartments.



 Allison Bischoff, Wei Yan, Carlos Espinoza, Emily Lasse-Opsahl, Nur Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar. Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df0eac24a706ec3b2888b93172ea892748b9176" target='_blank'>
              Abstract A080: Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment
              </a>
            </td>
          <td>
            Allison C Bischoff, Wei Yan, Carlos E Espinoza, Emily L. Lasse-Opsahl, Nur M Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The immunosuppressive tumor microenvironment enables immune evasion through mechanisms beyond canonical immune checkpoints. While phosphatidylserine (PS) externalization coordinates apoptotic clearance under physiological conditions, tumors hijack this mechanism through apoptotic mimicry to subvert antitumor immunity. Here, we identify TMEM16F, a calcium-activated phospholipid scramblase, as a driver of tumor-intrinsic PS externalization. TMEM16F-mediated PS scrambling polarized macrophages to an immunosuppressive M2 phenotype, which promotes TGF-β1 secretion and regulatory T cell expansion to suppress cytotoxic lymphocytes. Genetic ablation of TMEM16F abolished PS exposure, systemically reprogrammed the tumor microenvironment and primary immune organs toward immune activation, and suppressed tumor growth across cancer models. Pharmacological scramblase inhibition produced these effects, demonstrating therapeutic potential. Our findings establish TMEM16F-dependent phospholipid scrambling as a critical immune evasion axis and propose targeting this pathway for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/849ef22c3310bf7f7e66cd2c040d4068fbae6b99" target='_blank'>
              TMEM16F phospholipid scramblase regulates tumorigenesis by modulating the tumor immune microenvironment.
              </a>
            </td>
          <td>
            Menghan Wu, Peishang Shi, Jianmin Huang, Xiaomin Ni, Ruijia Lai, Kun Cao, Mengya Cai, Hao Yu, Wenshan Zhao, Yang Zhang
          </td>
          <td>2025-10-16</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178ca173745db4823628df69a4e845e3404906a8" target='_blank'>
              Epigenetic modulation of CTCs as a novel therapeutic target in oncology.
              </a>
            </td>
          <td>
            Husni Farah, M. Abosaoda, Hayjaa Mohaisen Mousa, S. Jyothi, Priya Priyadarshini Nayak, J. Janney, Gurjant Singh, Ashish Singh Chauhan
          </td>
          <td>2025-10-11</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6971b52e8812bcb489dfce1326931d0b1e99d8ab" target='_blank'>
              Cell villages and Dirichlet modeling map human cell fitness genetics
              </a>
            </td>
          <td>
            Chloe Hanson, Tim Derebenskiy, Ana Rodriguez Vega, Yashika S. Kamte, Rachel G. Fox, Hannah Lambing, Patrick Allard, Harold Pimentel, Michael F. Wells
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lymph node (LN) lymphatic endothelial cells (LEC) actively acquire and archive foreign antigens. Here, we address questions of how LECs achieve durable antigen archiving and whether LECs with high levels of antigen express unique transcriptional programs. We use single cell sequencing in dissociated LN tissue and spatial transcriptomics to quantify antigen levels in LEC subsets and dendritic cell populations at multiple time points after immunization and determine that ceiling and floor LECs archive antigen for the longest duration. We identify, using spatial transcriptomics, antigen positive LEC-dendritic cell interactions. Using a prime-boost strategy we find increased antigen levels within LECs after a second immunization demonstrating that LEC antigen acquisition and archiving capacity can be improved over multiple exposures. Using machine learning we define a unique transcriptional program within archiving LECs that predicts LEC archiving capacity in independent mouse and human data sets. We test this modeling, showing we can predict lower levels of LEC antigen archiving in chikungunya virus-infected mice and demonstrate in vivo the accuracy of our prediction. Collectively, our findings establish unique properties of LECs and a defining transcriptional program for antigen archiving that can predict antigen archiving capacity in different disease states and organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382f59662225a03a1be36e91e9b0ab93cd6fd5d9" target='_blank'>
              A specific gene expression program underlies antigen archiving by lymphatic endothelial cells in mammalian lymph nodes
              </a>
            </td>
          <td>
            Ryan M. Sheridan, Thu A Doan, C. Lucas, Tadg S Forward, Ira Fleming, Valerie M Olsen, Abrianna M Qvale, Bennett J. Davenport, Kristen Zarrella, Michael G. Harbell, Aspen Uecker-Martin, Thomas E. Morrison, Jay R. Hesselberth, B. A. Tamburini
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Adoptive T cell therapies (ACT) have revolutionized the management of hematologic malignancies; however, their efficacy in solid tumors remains limited. Accumulating evidence implicates the tumor microenvironment (TME) - a highly complex and immunosuppressive niche as a major barrier to their effectiveness. In this review, we propose that the next generation of ACT will require a fundamental shift from a reductionist focus on T cell engineering alone to an integrated approach that considers the interactions between immune cells and the TME. A comprehensive literature review identified several emerging strategies to enhance the efficacy of ACT, including reprogramming tumor vasculature, repolarizing immunosuppressive myeloid and stromal cells, leveraging oncolytic viruses to remodel antigen presentation, inducing acute sterile inflammation, and targeting the physical properties of the extracellular matrix. While many of these approaches remain in early-stage development, some have already progressed to clinical trials, indicating their potential for clinical translation. Additionally, we found that conventional therapies, such as surgery, chemotherapy, and radiotherapy, can be strategically integrated with ACT to improve therapeutic outcomes. These findings highlight a shift in the field toward more integrative approaches. Future advances will likely depend on reprogramming the TME to support T cell persistence and functions. Addressing these interconnected challenges will require closer collaboration between immunology, oncology, and bioengineering disciplines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5845940292899bbb8a96147c6b79326179c978e6" target='_blank'>
              Reprogramming the tumor microenvironment to boost adoptive T cell therapy
              </a>
            </td>
          <td>
            Maria Fernanda Meza Pacheco, Lee-Hwa Tai
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The advent of RAS inhibitors in the clinic offers the possibility of revolutionized care for patients with pancreatic ductal adenocarcinoma (PDAC). While RAS(ON) multi-selective inhibitors like the investigational agent daraxonrasib (RMC-6236) and the related preclinical tool compound RMC-7977 are designed to target the KRAS-mutant malignant epithelium, their effects on the surrounding tumor stroma remain underexplored. Prior studies using genetically engineered mouse models (GEMMs) have shown that aberrant oncogenic KRAS signaling is critical for the establishment and maintenance of the tumor microenvironment. In recent years, multiple CAF subtypes have been identified based on gene expression profiles, spatial distribution, and cellular origin. Myofibroblastic CAFs (myCAFs), which are located near the malignant epithelium, are marked by active Sonic Hedgehog (SHH) signaling, while inflammatory CAFs (iCAFs) and antigen-presenting CAFs (apCAFs) reside more peripherally. Here, we investigate how pan-RAS-GTP inhibition by RMC-7977 reshapes the CAF landscape in PDAC and its impact on the extracellular matrix (ECM). We profiled CAF subtype prevalence and transcriptional programs using single-cell RNA sequencing (scRNA-seq) of GEMM-derived tumors treated in vivo with RMC-7977 across multiple timepoints. These findings were corroborated by immunostaining of murine and patient-derived tumor explants treated ex vivo for 2 to 4 days. Tumor stiffness following RMC-7977 treatment was assessed using nanoindentation, and mechanistic studies were performed in vitro using reporter fibroblasts. Preliminary data reveal that short-term RMC-7977 treatment reduces myCAF abundance and proliferation, while iCAF and apCAF populations expand. Notably, supplementation with exogenous SHH restores the original myCAF-rich phenotype even in the presence of RMC-7977. Pan-RAS-GTP inhibition also remodels the ECM, resulting in decreased tumor stiffness. These preclinical findings suggest that systemic pan-RAS-GTP inhibition alters not only the malignant epithelium but also the broader PDAC architecture through effects on stromal components. Understanding these stromal adaptations will enable improved assessment of tumor behavior and the development of more effective combination therapies.



 Allison C. Hess, Lorenzo Tomassoni, Kevin Muñoz Forti, Hun Jin Jeong, Deanna J. Besart, Tanner C. Dalton, Carmine F. Palermo, Stephen A. Sastra, Chang H. Lee, Andrea Califano, Simon Schwörer, Kenneth P. Olive, Marie C. Hasselluhn. RAS(ON) multi-selective inhibition remodels cancer-associated fibroblast subtypes and extracellular matrix in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B098.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80ccb92666321bba79e7b5a3b723ec7cf1794a5" target='_blank'>
              Abstract B098: RAS(ON) multi-selective inhibition remodels cancer-associated fibroblast subtypes and extracellular matrix in pancreatic cancer
              </a>
            </td>
          <td>
            Allison C. Hess, Lorenzo Tomassoni, Kevin Muñoz Forti, Hun Jin Jeong, Deanna J. Besart, Tanner C. Dalton, Carmine F. Palermo, Stephen A. Sastra, Chang H. Lee, Andrea Califano, Simon Schwörer, K. Olive, Marie C. Hasselluhn
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma is one of the deadliest solid tumors, with a two-year survival rate below 5%. Progress in developing effective treatments has been limited, largely due to the absence of physiologically relevant disease models that accurately recapitulate the tumor microenvironment. Such models are critical for understanding disease mechanisms and evaluating therapeutic strategies. Organoids have recently emerged as promising platforms for modeling various diseases, including pancreatic cancer, as they retain the structural and functional features of their tissue of origin. Moreover, CRISPR-Cas9 gene editing allows precise introduction of cancer-associated mutations, providing a powerful approach for generating genetically defined models.



 We established normal pancreatic organoids from human pancreatic tissue samples and validated their identity through qPCR and histological analyses, confirming their resemblance to native pancreatic epithelium. These organoids demonstrated stable growth over several passages and retained viability after cryopreservation and thawing. To model pancreatic cancer, we used CRISPR-Cas9 technology to introduce TP53 mutations, a key tumor suppressor gene altered in over 70% of pancreatic ductal adenocarcinoma (PDAC) cases. Following Nutlin-3a selection, successful TP53 knockout was confirmed by Sanger sequencing. The engineered TP53-knockout (KO) organoids were expanded and characterized to assess cancer-like phenotypes.



 Human-derived pancreatic organoids retained their structural and functional identity across passages and after thawing. Following TP53 gene editing, the TP53 KO organoids exhibited increased proliferation and morphological changes consistent with early tumorigenesis. Importantly, qPCR analysis revealed significant upregulation of cancer-associated markers, including MUC5AC, KRT19, and CEACAM5, in TP53-mutated organoids compared to controls, further supporting their malignant transformation. These findings represent a critical step toward building a genetically defined pancreatic cancer model that closely reflects human disease.



 We successfully generated a pancreatic cancer organoid model by editing TP53 in normal human pancreatic organoids. The TP53 KO organoids showed enhanced growth, morphological transformation, and elevated expression of key cancer markers, recapitulating early features of pancreatic cancer. This model mimics critical aspects of pancreatic cancer biology and provides a robust platform for studying tumor initiation, progression, and drug response. Unlike conventional animal models or stem-cell-derived systems, our approach offers a human-based, physiologically relevant, and reproducible system for pancreatic cancer research and therapeutic screening.



 Eden Demere Amare, Soraya SALAS-SILVA, Ji Hyun Shin, Dongho Choi. CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59cdfa7b3d8511b93672f4beff88185a825b61c7" target='_blank'>
              Abstract A013: CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation
              </a>
            </td>
          <td>
            Eden Demere Amare, Soraya SALAS-SILVA, Jihyun Shin, Dongho Choi
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="As a common malignant tumor, the heterogeneity of colorectal cancer plays an important role in tumor progression and treatment response. In recent years, the rapid development of single-cell transcriptomics and spatial transcriptomics technologies has provided new perspectives for resolving the heterogeneity of colorectal cancer. These techniques can reveal the complexity of cellular composition and their interactions in the tumor microenvironment, and thus facilitate a deeper understanding of tumor biology. However, in practical applications, researchers still face technical challenges such as data processing and result interpretation. The aim of this paper is to explore how to use artificial intelligence (AI) technology to enhance the research efficiency of single-cell and spatial transcriptomics, analyze the current research progress and its limitations, and explore how combining AI approaches can provide new ideas for decoding the heterogeneity of colorectal cancer, and ultimately provide theoretical basis and practical guidance for the clinical precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48a5aab5c189afed70eaf2f7feb72d37acadd46e" target='_blank'>
              Multidimensional decoding of colorectal cancer heterogeneity: Artificial intelligence-enabled precision exploration of single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Wen-Yu Luan, Qi Zhao, Zheng Zhang, Zhen-Xi Xu, Si-Xiang Lin, Yan-Dong Miao
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal malignancy characterized by a profoundly immunosuppressive tumor microenvironment. Within PDAC, tumor-associated macrophages (TAMs) are the most abundant immune cell population. Here, TAMs orchestrate the characteristic immunosuppression of the pancreatic tumor microenvironment, drive disease progression, and mediate therapeutic resistance in PDAC. Accordingly, successful development of approaches that reprogram or eliminate these pro-tumor immune populations have potential to improve patient outcomes. One of the most challenging aspects of developing approaches to modulate TAMs programming is the ability to perform mechanistic experiments in a system the recapitulates the complex pancreatic tumor microenvironment. We and others have leveraged primary bone marrow-derived macrophages (BMDMs) polarized by PDAC conditioned media as a discovery platform that has yielded translational findings. However, this system fails to account for the dynamic and reciprocal interactions between myeloid and cancer cells that occur within tumors. Accordingly, we sought to address if this model could be built into progressively more complex systems to assess if these better recapitulate TAMs using a systems biology approach. Using transcriptomic profiling, we compared different in vitro strategies, from direct co-cultures to conditioned media polarization, against TAMs harvested from orthotopic PDAC tumors. Our preliminary data show that allowing PDAC cells to engage in direct cellular contact and sustained paracrine signaling most closely resemble the transcriptomic profile of bona fide TAMs. Our further efforts to use these PDAC-macrophage culture models will be key to identifying relevant signaling pathways that can be mechanistically examined prior to more cumbersome in vivo validation. Collectively, our data can be used to facilitate more accurate and insightful investigations into PDAC immunology and the development of therapeutic targeting strategies.



 Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru. Dissecting myeloid - cancer communication networks in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B104.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad71226813c7f5d3d378a55b84523f40aab08ce1" target='_blank'>
              Abstract B104: Dissecting myeloid - cancer communication networks in pancreatic cancer
              </a>
            </td>
          <td>
            Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Immunotherapy remains ineffective in many solid tumors due to poor T-cell infiltration and a metabolically suppressive tumor microenvironment. A dual strategy combining low-level light (LLL) therapy with a nanoscale stimulator of interferon genes (STING) agonist formulation (nanoSTING@Mn) is presented to enhance immune activation and metabolic fitness for durable tumor immunity against T-cell lymphoma (EL4) model. NanoSTING@Mn, composed of ADU-S100 complexed with Mn2⁺ and encapsulated in biomimetic liposomes, potently activates the cGAS-STING pathway, induces a type I interferon response, and promotes lymphocyte infiltration. These monocytes polarize into M1 macrophages, suppressing regulatory T cells. Simultaneously, LLL photo-biomodulation reprograms mitochondrial metabolism in tumor-infiltrating CD8⁺ T and natural killer cells, restoring their durability and leading to complete local tumor eradication. This combination expands a distinct CD8⁺ T-cell subset with Tcf-1⁺ progenitor-exhausted features and elevated memory/effector gene expression, enhancing proliferation and cytotoxicity, as shown by single-cell RNA sequencing. Intranasal nanoSTING@Mn delivery mobilizes these LLL-revived T cells to the lung, where they differentiate into resident memory T cells and establish systemic antitumor immunity. Upon intravenous rechallenge, disseminated tumor cells are eliminated, preventing metastasis and ensuring long-term protection. This synergistic approach offers a scalable platform to boost immunotherapy efficacy and redefines immune-based metastasis prevention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e9ba5699caa736e818a9807d89458c43c8a2b62" target='_blank'>
              Light Metabolically Reprograms CD8+ T Cells to Potentiate STING-Driven Tumor Eradication and Prevent Metastasis.
              </a>
            </td>
          <td>
            Asmita Banstola, Shilin Gao, ZhengKun Zhang, Yan Dong, Prabhat Upadhyay, Quanwei Zhang, Yongli Li, Zuan-Tao Lin, Zhilong Wang, Mei X. Wu
          </td>
          <td>2025-10-23</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cell type–specific regulatory programs that drive type 1 diabetes (T1D) in the pancreas are poorly understood. Here, we performed single-nucleus multiomics and spatial transcriptomics in up to 32 nondiabetic (ND), autoantibody-positive (AAB+), and T1D pancreas donors. Genomic profiles from 853,005 cells mapped to 12 pancreatic cell types, including multiple exocrine subtypes. β, Acinar, and other cell types, and related cellular niches, had altered abundance and gene activity in T1D progression, including distinct pathways altered in AAB+ compared to T1D. We identified epigenomic drivers of gene activity in T1D and AAB+ which, combined with genetic association, revealed causal pathways of T1D risk including antigen presentation in β cells. Last, single-cell and spatial profiles together revealed widespread changes in cell-cell signaling in T1D including signals affecting β cell regulation. Overall, these results revealed drivers of T1D in the pancreas, which form the basis for therapeutic targets for disease prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cafcd5a288ad120848114a28181c932782050f5" target='_blank'>
              Single-cell multiome and spatial profiling reveals pancreas cell type–specific gene regulatory programs of type 1 diabetes progression
              </a>
            </td>
          <td>
            Rebecca L. Melton, Sara Jimenez, Weston S Elison, Luca Tucciarone, Abigail Howell, Gaowei Wang, Denise Berti, Elisha Beebe, Michael Miller, Chun Zeng, C. McGrail, Kennedy VanderStel, Katha Korgaonkar, R. Elgamal, Hannah M Mummey, Joshua Chiou, Emily Griffin, I. Kusmartseva, Mark Atkinson, Sebastian Preissl, F. Theis, Maike Sander, K. J. Gaulton
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Tumor-immune interactions are shaped by both antigenic heterogeneity and stochastic perturbations in the tumor microenvironment, yet the mathematical mechanisms underlying immune phase transitions remain poorly understood. We propose a four-compartment dynamical model that incorporates antigen accumulation and immune escape mutations. Bifurcation analysis reveals bistability between immune surveillance and immune escape states, providing a mechanistic explanation for heterogeneous immune outcomes during tumor progression. In the multistable regime, the stable manifold of a saddle point partitions the state space into distinct basins of attraction, determining the long-term fate of the system. We further analyze how stochastic fluctuations in the tumor microenvironment perturb these separatrices, potentially triggering irreversible state transitions. By characterizing the critical noise intensity and estimating the tipping time, we establish a mathematical framework for assessing noise-induced transitions. The model further predicts that increasing tumor cell death can improve system resilience to stochastic perturbations, whereas stronger immune pressure may facilitate immune escape-highlighting the nonlinear and non-monotonic nature of tumor-immune dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/358f26ec76382702bbe9f0bdc46b3b599d87dd6f" target='_blank'>
              Bistability and Noise-Induced Evasion in Tumor-Immune Dynamics with Antigen Accumulation and Immune Escape
              </a>
            </td>
          <td>
            Mengfan Tan, Shaoqing Chen, Chunjin Wei, Da Zhou
          </td>
          <td>2025-09-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs), central regulators of the tumor microenvironment (TME), are increasingly recognized as key orchestrators of the oncobiosphere—an integrated and dynamic multicellular system that sustains tumor progression and therapy resistance. CAFs not only support cancer stemness and self-renewal but also actively contribute to the formation of pre-metastatic niches. The bidirectional crosstalk between CAFs and tumor cells promotes the accumulation of a desmoplastic and immunosuppressive extracellular matrix (ECM), which serves as a major physical and immunological barrier against chemotherapy, targeted therapy, and immunotherapy. Therefore, overcoming CAF-mediated pseudo-resistance by dismantling this pathological architecture represents a promising approach for improving cancer treatment efficacy. Penetrium™, a novel niclosamide-based therapeutic formulated using an organic–inorganic nanohybrid delivery system. In a patient-derived pancreatic cancer organoid (PCO)–CAF co-culture model, which recapitulates key features of the human oncobiosphere, Penetrium™ selectively eliminated CAFs without affecting normal fibroblasts. This CAF-directed modulation restored gemcitabine sensitivity under previously resistant conditions and achieved complete ablation of CAF-integrated organoids. These findings provide the first ex vivo evidence that targeting CAFs and remodeling the stromal niche can overcome stromal-driven pseudo-resistance and re-enable effective chemotherapeutic response. The ability of Penetrium™ was further validated in in vivo syngeneic models of aggressive tumors—including triple-negative breast cancer (TNBC), non–small cell lung cancer (NSCLC), and pancreatic cancer. In these models, combination treatment with anti–PD-1 immune checkpoint inhibitors significantly enhanced intratumoral infiltration of cytotoxic immune cells, transforming the oncobiosphere from an immune-excluded (“cold”) to an immunologically active (“hot”) state. This immune activation was accompanied by functional disruption of the stromal architecture and enhanced tumor penetration of co-administered anti–PD-1 antibodies, indicating that CAF depletion alleviated desmoplastic and immunosuppressive barriers to immunotherapy. Together, these results demonstrate the potential of Penetrium™ as a tumor microenvironment–modulating agent that selectively eliminates CAFs to dismantle and reprogram the oncobiosphere. By targeting the structural basis of tumor resistance, Penetrium™ offers a novel strategy to overcome pseudo-resistance and augment the efficacy of immune checkpoint inhibitors and other anticancer modalities across diverse tumor types.



 Sue Jang, Sunki Lim, EunByule Choi, Sujong Kim, Jason Kim, Aizhan Abdildinova, Hailey Choi, Jin-Ho Choy, Geun-woo Jin. Disrupting the Oncobiosphere: CAF-Targeting Therapy with Penetrium™ Reverses Pseudo-Resistance in Tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C128.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e3047281674d0b39c09a46120c51f5e0fd63278" target='_blank'>
              Abstract C128: Disrupting the Oncobiosphere: CAF-Targeting Therapy with Penetrium™ Reverses Pseudo-Resistance in Tumors
              </a>
            </td>
          <td>
            Sue Jang, Sunki Lim, EunByule Choi, Sujong Kim, Jason Kim, Aizhan Abdildinova, Hailey Choi, Jin-Ho Choy, Geun-woo Jin
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Polycystic kidney disease (PKD) arises from mutations in cilia-associated genes, such as PKD1 and PKD2, expressed in renal epithelial cells, leading to progressive kidney dysfunction and end- stage kidney disease (ESKD). PKD patients exhibit significant heterogeneity in disease progression, largely due to genetic and environmental modifiers. Like patients, mouse models of PKD also exhibit significant heterogeneity with regards to the gene mutated, age of disease onset, and rate of disease progression. To elucidate the cellular and molecular consequences of these variables, we constructed an integrated single-cell and spatial transcriptomics atlas across mouse models of PKD, mapping changes in cell type composition, gene expression, and intercellular signaling networks across the whole atlas and within individual models. Consistently across models, single cell RNA sequencing (scRNAseq) data revealed increased Spp1 (osteopontin) expression and signaling from PKD-enriched clusters to Ly6clo monocytes. Global deletion of Spp1 in Pkd1RC/RC mice resulted in reduced cyst severity, improved kidney function, and reduced Ly6clo monocyte numbers, suggesting that SPP1 signaling to Ly6clo monocytes promotes PKD progression. We also created a freely available, searchable website (https://bmblx.bmi.osumc.edu/scPKD/) that can be used to identify cross- and intra-model specific changes in gene expression, guiding researchers to new therapeutic targets for treating PKD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c0ef30fff916db6f0ffbd15d9f601a5f6423bd0" target='_blank'>
              A cross model spatial and single-cell atlas reveals the conserved involvement of osteopontin in polycystic kidney disease
              </a>
            </td>
          <td>
            S. Miller, Hua Zhong, Weidong Wu, Audrey Cordova, Morgan E. Smith, A. Yashchenko, Zhang Li, D. Jafree, Chelsea N. Zimmerman, Christa I. DeVette, Vicki Do, Maya Hignite, Isabella G. Darby, Khalecha Bintha Ahmed, Yohan Park, Fariha Nusrat, Bibi Maryam, Sizhao Lu, Xiaoyan Li, Jenny R. Gipson, Xiaogang Li, David A. Long, M. Weiser-Evans, Bradley K. Yoder, Benjamin D. Cowley, Katharina Hopp, Jason R Stubbs, Qin Ma, A. Ma, Kurt A Zimmerman
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb8107340798d9682d8c6977b5d27a13471e822" target='_blank'>
              Integrated single-cell and spatial transcriptomics uncover the prognostic, epigenetic, and immunological roles of FANCC in low-grade glioma.
              </a>
            </td>
          <td>
            Zongye Zhang, Zhi Sha, Han Liu, Yusheng Chen, Zhendong Liu, Xingbo Cheng, Sujie Gu, Yanzheng Gao
          </td>
          <td>2025-09-09</td>
          <td>Neurological research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy characterized by a dense desmoplastic stroma and a nutrient-deprived tumor microenvironment (TME). These harsh conditions drive metabolic symbiosis, where cancer cells, stromal cells, and immune cells exchange metabolites to sustain tumor growth, therapy resistance, and immune evasion. Although MRTX1133 (KRASG12D inhibitor) has demonstrated promise in targeting the KRAS/MAPK pathway, responses differ throughout PDAC subtypes, and resistance has been reported. We treated PDAC cell lines with KRASG12D inhibitor (KRASi) and found a paradoxical increase in proliferation in some cell lines while suppressing growth in others, indicating the activation of compensatory survival mechanisms. In the sensitive cell lines, we found that KRASi disrupted growth, altered the metabolomic profile, and nutrient transport. We have identified SLC20A1, a phosphate transporter, as a novel KRAS-driven transporter. CRISPR/Cas9-mediated knockout of SLC20A1 disrupted the Warburg effect in PDAC cells. In conclusion, our data suggests that KRAS/MAPK inhibition exert robust control on nutrient transport as a mechanism to impair metabolism in KRAS-reliant PDAC cells.



 Harshita Nivrutti. Khedkar, Laiba Naz. Shiekh, Matthew Cheung, Don-Gerard Conde, Zeribe C. Nwosu. Oncogenic KRAS drives nutrient transport to support growth in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B020.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f683e1c06482d7e87be84283ced7cefbff45c117" target='_blank'>
              Abstract B020: Oncogenic KRAS drives nutrient transport to support growth in pancreatic cancer
              </a>
            </td>
          <td>
            H. N. Khedkar, Laiba Shiekh, Matthew Cheung, Don-Gerard Conde, Z. Nwosu
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb94fc6f75bdfeda4079ffba83972fab9d967bdb" target='_blank'>
              Radiotherapy drives lung metastasis in PDAC via a RhoGTPase signaling shift towards MRCK-dependency
              </a>
            </td>
          <td>
            KM McLay, M. Tesson, L. Dutton, Y. Sun, JD Misquitta, K. Stevenson, R. Corbyn, S. Lilla, L. McGarry, R. Shaw, Crispin J Miller, Leo M. Carlin, MF Olson, JP Morton, JL Birch
          </td>
          <td>2025-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Cancer can recur when a subset of tumor cells, termed persister cells, survive therapy and re-enter the cell cycle. The cellular lineages that give rise to persister cells and the mechanisms that confer the persister state remain poorly understood. Through single-cell multiomic profiling (snRNA-seq and snATAC-seq) on a cohort of (1) non-malignant fallopian tube, (2) treatment-naïve, and (3) neoadjuvant chemotherapy (NACT)-treated high-grade serous ovarian cancer (HGSOC) patient samples, we identified an epigenetic signature that defines the chemotherapy-tolerant persister state. The changes in chromatin accessibility characterizing the signature were identified in residual NACT tumors but are also present in treatment-naive samples from patients who later developed resistance. Furthermore, this epigenetic signature independently predicted chemotherapy response in patient-derived xenograft models of HGSOC and in a separate patient cohort. Cells enriched in the persister state arose from multiple lineages and displayed activation of oncogenic pathways, including altered stress responses, epithelial to mesenchymal transition, and changes to the cell cycle promoting quiescence. Finally, we identified a subset of genes that are epigenetically primed for expression before treatment and are upregulated after treatment. These findings suggest that an intrinsic epigenetic program primes tumor cells towards chemotherapy tolerance and reveal new vulnerabilities that could be exploited to delay or prevent cancer recurrence.



 Mihai G. Dumbrava, Wazim M. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. John Weroha, Alexandre Gaspar-Maia. Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a450af46a8e3f0b0c1a2127ce5282e5d4428e5a8" target='_blank'>
              Abstract PR008: Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells
              </a>
            </td>
          <td>
            Mihai G. Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Colon adenocarcinoma (COAD) is influenced by the tumor microenvironment, including immune cell dynamics, particularly M2 macrophages. Understanding these dynamics can offer insights into tumor progression and potential therapies. Methods We analyzed TCGA COAD data to quantify immune infiltration using TIMER, CIBERSORT, QUANTISEQ, MCPcounter, xCell, and EPIC methods. The ESTIMATE algorithm provided stromal and immune scores. WGCNA identified key gene modules correlated with immune infiltration. Prognostic significance of these genes was assessed using univariate Cox regression, Kaplan-Meier analysis, and nomogram construction. Single-cell RNA sequencing data from TISCH2 was used to explore gene expression correlations across cell types. We also examined correlations of LDOC1 with TMB, MSI, NEO, and tumor purity, and predicted immunotherapy responses using TIDE and Submap analyses. Drug sensitivity was analyzed using GDSC data. Results Significant variations in M2 macrophage levels were found between tumor and normal tissues. WGCNA identified three key gene modules. Univariate Cox regression highlighted DMPK and LDOC1 as significant prognostic genes. Pan-cancer analysis showed LDOC1 as a crucial prognostic marker across various cancers, with strong correlations to multiple pathways. Single-cell analysis revealed distinct expression patterns of DMPK and LDOC1. In COAD, LDOC1 expression correlated significantly with immune checkpoints and tumor microenvironment scores. TIDE and Submap analyses suggested better immunotherapy response in low LDOC1 groups. Drug sensitivity analysis identified eight potential therapeutic agents. Conclusion This study underscores the importance of M2 macrophages and LDOC1 in COAD, highlighting their potential as prognostic markers and therapeutic targets, paving the way for personalized treatments. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03698-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11e05862de6b52dfc8fd8583cebee00c133e3ea2" target='_blank'>
              Immune cell dynamics and prognostic role of M2 macrophages and LDOC1 in colon adenocarcinoma
              </a>
            </td>
          <td>
            Laibin Luo, Zhimin Xu, Hongwu Lin, Rong Huang
          </td>
          <td>2025-10-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The dynamic evolution of the immune tumor microenvironment (TME) during targeted therapy is a critical yet poorly understood determinant of treatment response and resistance. While most studies compare immune states before and after treatment, temporal immune changes during therapy remain largely uncharacterized, limiting development of effective combination strategies. Here, we investigated immune dynamics throughout targeted therapy using mouse melanoma models that recapitulate human therapeutic responses. Single-cell RNA sequencing (scRNA-seq) identified an inflection point where the inflamed TME during tumor regression, characterized by robust NK cell infiltration, transitions to an immune-excluded state upon onset of drug-tolerant residual disease. We uncovered a unique macrophage subset that orchestrates NK cell recruitment during regression. Depletion of these macrophages using LysM-cre;iDTR mice significantly reduced NK cell infiltration. Extending these findings to human cancer, longitudinal scRNA-seq analysis of melanoma and lung cancer patient samples revealed dynamic NK cell infiltration during targeted therapy, establishing a direct link between innate immune remodeling and treatment outcome. Unlike prior prognostic studies assessing immune states at single time points, our results provide mechanistic evidence of a temporal relationship between NK cell infiltration and therapeutic efficacy. Together, these findings position immune evolution as a driver of acquired resistance and identify macrophage–NK cell crosstalk as a therapeutically actionable axis to overcome immune exclusion and improve targeted therapy.



 Chia-Hsin Hsu, Andrew C. White. Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064d31f044dc7aac267bdcec71d797f7f005e82d" target='_blank'>
              Abstract A012: Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions
              </a>
            </td>
          <td>
            Chia-Hsin Hsu, Andrew C White
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Immunotherapies have shown significant clinical benefit in cancers with high unmet medical need. As next-generation bispecific antibodies are entering the clinic, early identification of patients likely to respond to immunotherapy remains both critical and challenging. We have previously demonstrated that our ex vivo 3D micro-tumor platform, which preserves the native tumor microenvironment (TME), is predictive for platinum-based chemotherapy and correlates with clinical immune biomarkers. Here, we show that the ex vivo platform also enables patient-specific profiling of drug sensitivity and immune activity for bispecific antibody ubamatamab.



 This study analyzed 14 tissue samples obtained from 12 ovarian cancer patients with primary, recurrent or progressive disease. Micro-tumors including the native TME were isolated from fresh ascites (n=12) or pleural fluid (n=2), seeded, and treated with pembrolizumab (single dose) and dose ranges of chemotherapeutics and ubamatamab (CD3xMUC16 bispecific T cell engager in Phase 1/2 clinical trials). Drug response was assessed through morphological features extracted from high-throughput 3D imaging. Sensitivity was defined by statistically significant immune cell expansion and micro-tumor killing. Supernatants collected after treatment were analyzed for secreted cytokines, chemokines and cytotoxic mediators using proximity extension assay (Olink) immune biomarker profiling for 5 patients. Quantification of immune cell populations was performed via immunofluorescence staining.



 Treatment with ubamatamab resulted in a significant expansion of immune cells in 43% (6/14) of ex vivo patient samples. Of these, 33% (2/6) also showed significant tumor killing, demonstrating complete functional responses. Responses to ubamatamab were reproducible in successive samples from the same patients, showing assay consistency. Baseline cytokine profiling confirmed substantial inter-patient heterogeneity, and patient samples with low baseline levels of CD8+ T cell cytokines showed ex vivo resistance to ubamatamab. Notably, responders with effective tumor killing exhibited robust and simultaneous induction of effector T cell cytokines and cytotoxic mediators (IFN-γ, IL-2, granzyme B, perforin) as well as IFN-γ-induced chemokines CXCL9 and CXCL11 upon ubamatamab treatment. Importantly, ex vivo sensitivity to ubamatamab did not align with responses to pembrolizumab or chemotherapy, indicating that this therapy may be effective in a distinct subset of patients.



 This study demonstrates the classification and functional characterization of patient-specific sensitivity to ubamatamab, a bispecific T cell engager in clinical development for ovarian cancer. Our ex vivo 3D micro-tumor platform facilitates quantification of immune responses and the discovery of predictive biomarkers to support patient stratification for emerging immunotherapies.



 Esmee Koedoot, Felix Behr, Timothy Sijsenaar, Léa Le Large, Cor D. de Kroon, Anne van Altena, Sander Bins, Dieudonné van der Meer, Willemijn Vader, Ingrid Boere, Nelleke Ottevanger, Judith Kroep. Bispecific antibody Ubamatamab induces personalized anti-tumor immune responses in an ex vivo platform for ovarian cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46429fd9066f1bc883507000c8c774ce1eb7d096" target='_blank'>
              Abstract A129: Bispecific antibody Ubamatamab induces personalized anti-tumor immune responses in an ex vivo platform for ovarian cancer
              </a>
            </td>
          <td>
            E. Koedoot, Felix Behr, Timothy J. Sijsenaar, Léa Le Large, C. D. de Kroon, A. V. van Altena, Sander Bins, Dieudonné J. van der Meer, W. Vader, Ingrid Boere, N. Ottevanger, J. Kroep
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Colorectal laterally spreading tumors (LSTs) are defined as non-protruding neoplasms exceeding 10 mm in diameter that grow primarily along the intestinal wall. The morphogenetic mechanisms and evolutionary trajectories of granular-type LSTs (LST-Gs) towards colorectal carcinoma remain unclear. Methods In this study, we investigate the transcriptional features of LST-Gs using single-cell RNA sequencing technology by comparing them with protruded-type adenomas (PAs) and normal mucosal tissues. Results Adenomatous LST-Gs harbor an epithelial cell population with metaplastic differentiation, which are almost absent in PAs. Cells with a high degree of differentiation in LST-G demonstrate enhanced immunogenicity and robust adhesion/junction interactions. Furthermore, LST-Gs show upregulated expression of molecular chaperones and metallothioneins compared to PAs, reflecting a more hostile microenvironment similar to that observed in carcinoma stages. These characteristics suggest that LST-G exhibits greater heterogeneity compared to the earliest colorectal adenomas, mirroring the progression from precancerous states to cancer. Notably, the Arp2/3 complex is significantly upregulated in highly differentiated LST-G cell populations, potentially facilitating cell migration along the basement membrane, which highlights the similarities in cell motility between adenomatous LST-G and normal mucosal epithelium as well as serrated polyps (SERs). Conclusion The differentiation state of cells within LST-G exhibits a close correlation with their diverse characteristics. Metaplastic differentiation, as a prominent feature at the transcriptional level, demonstrates significant associations with the genomic features, morphogenesis, and tumor progression of LST-G.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cfb702cab41be0ca69fc6ec0389c21e705f623" target='_blank'>
              Single-cell RNA profiling of colorectal granular-type laterally spreading tumor uncovers progression trajectory toward carcinoma and transcriptional signatures favoring lateral morphogenesis
              </a>
            </td>
          <td>
            Yueqing Gong, Yuxin Zhang, Xun Liu, Rongli Cui, Jingjing Lu, Jun Li, Fang Gu, Jing Zhang, Shigang Ding, Weiwei Fu
          </td>
          <td>2025-09-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3257e07d3eed982df5f02daf4c23bc618d1ee9c3" target='_blank'>
              A quantitative approach to discover nonlinear signaling linking combinatorial environments with cellular responses
              </a>
            </td>
          <td>
            Dmitry Kuchenov, Frederik Ziebell, Florian Salopiata, Alix Thomas, Mevlut Citir, Ursula Klingmueller, Wolfgang Huber, Carsten Schultz
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer with a dismal five-year survival rate of 13%. Most patients present with metastatic disease, and 80% undergoing surgery relapse within two years. Research on PDAC has traditionally focused on primary tumors, thus most treatment derives from primary tumor biology. Recent evidence unveiled significant differences in the tumor immune microenvironment (TIME) between primary and metastatic lesions. Understanding these differences is crucial, as immunotherapies are predominantly administered to patients with refractory metastatic PDAC, yet traditional treatment fails to account for organ specificity and patient TIME signatures. In this study, we wanted to know why the same cancer cell type in different organs would create different microenvironments in the pancreas, liver, and lungs. To do this, we investigated the TIME in early and late-stage metastatic cohorts by multiplex immunohistochemistry (mIHC), transcriptomic assays and spatial analysis. To profile the TIME of different immune and stromal cells, we employed spatial protein expression using mIHC platform for the early cohort. In total, we applied a 30-plex panel to 56 samples. Our analysis revealed a higher density of COL1A1+ fibroblasts and CD163+ macrophages in the liver compared to other sites. Additionally, liver metastases exhibited significantly fewer CD8+ T cells compared to other organs within the tumor core. These results confirmed the earlier studies regarding the TIME of metastatic lesions, while also showing more macrophage and fibrosis in the liver. In the late-stage cohort, similar trends of decreased immune cell presence and exclusion phenotypes were observed using mIHC and Visium platforms. To explore the underlying mechanisms, we performed transcriptomic assays on the immune compartment. Our findings indicated an enrichment of interferon-positive macrophage subsets, particularly in liver metastases compared to primary PDAC. Additionally, we observed a higher presence of exhaustive CD8+ T cells and regulatory T cells in primary sites compared to liver and lung metastases, suggesting that both primary PDAC and liver metastases possess an immunosuppressive environment. Furthermore, spatial proximity analysis demonstrated that CD8+ T cells were closer to tumor cells in lung metastases compared to liver and primary tumors, reinforcing the notion that lung metastases are more immune-active. Pathway analysis revealed downregulation of antigen presentation and lymphocyte proliferation signaling in liver T cells compared to those in the lungs. Notably, immune infiltration was observed in some liver samples, but the percentage was significantly lower compared to lung samples. This divergence in TIME modulation between organs suggests a significant impact on disease progression in PDAC patients. The significant difference in immuno-response across organ shows why it is critical to consider distinct immune landscapes of different organs and patient histories when developing tailored therapies for advanced PDAC patients.



 Yu-Lan Kao, Olivia M. Dres, Blake G. Sells, Varun Shenoy, Xiang Li, Kapur Dhami, Paul M. Grandgenett, Li Ding, Liang-I Kang, David G. DeNardo. Dynamics of tumor immune environment across organs in early and late-stage metastatic pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A084.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846e100bfc6ae1b16ca4e9e6b72859a2dc02155f" target='_blank'>
              Abstract A084: Dynamics of tumor immune environment across organs in early and late-stage metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Yu-Lan Kao, Olivia M Dres, Blake Sells, Varun V Shenoy, Xiang Li, Kapur B. Dhami, P. Grandgenett, Li Ding, Liang-I Kang, D. DeNardo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bed713649d4f8f7563c5fc42bffbb7a06297df9" target='_blank'>
              Cancer-driven neutrophil priming couples systemic epithelial regenerative programs with pre-metastatic niche formation
              </a>
            </td>
          <td>
            Nicolas Rabas, Victoria Bridgeman, C. Pospori, Marc Hennequart, Felipe S. Rodrigues, Ana Farias, Rute M M Ferreira, Konstantinos Axarli, Olamide Nduka, Probir Chakravarty, Nathalie Legrave, Izadora Furlani, I. Malanchi
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background/Objectives: Immune checkpoint blockade (ICB) has emerged as a promising therapeutic option for hepatocellular carcinoma (HCC), yet reliable biomarkers to predict clinical outcomes remain limited. Tumor-associated macrophages (TAMs) are increasingly recognized as key regulators of the tumor immune microenvironment. Methods: We interrogated a publicly available HCC single-cell RNA sequencing (scRNA-seq) dataset to characterize intratumoral immune cell subpopulations. Through unsupervised clustering and gene signature analysis, we identified a distinct subset of SPP1 (secreted phosphoprotein 1, also known as osteopontin) and BCL2A1 (Bcl-2-related protein A1) double-positive TAMs. Their abundance was quantified and associated with patient outcomes. Further independent HCC transcriptomic datasets with annotated PD1-based ICB response status were used for examination. Results: Across the discovery (GSE149614; n = 10) cohort, elevated expression of intratumoral SPP1+BCL2A1+ TAMs was identified in HCC. In the ICB datasets (GSE151530; n = 4), patients with high SPP1+BCL2A1+ TAM expression further exhibited significantly poorer responses to ICB therapy. Further, the validation cohort (GSE206325; n = 18) confirmed these findings accordingly. Notably, these TAMs were expressed thoroughly within the immunosuppressive T-cell microenvironment in non-responders but were distinctly expressed among the cytotoxic T-cell responses in responders. Conclusions: Our findings identify SPP1+BCL2A1+ TAMs as a poor prognostic biomarker in HCC patients undergoing ICB therapy. By promoting an immunosuppressive microenvironment, SPP1+BCL2A1+ TAMs, which are survival-advantaged, may represent both a predictive marker and a potential therapeutic target to enhance the efficacy of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed70e096110d906e0f3936bf566f10405872944" target='_blank'>
              Intratumoral SPP1+BCL2A1+ Tumor-Associated Macrophages Predict Poor Response to PD1 Blockade
              </a>
            </td>
          <td>
            , Yu-Ping Hung, Po-Chun Tseng, Rahmat Dani Satria, 
          </td>
          <td>2025-10-23</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer immunotherapy has revolutionized cancer treatment, offering durable responses in a subset of patients. Despite this progress, cancer cells have developed sophisticated mechanisms to evade immune surveillance, including genetic mutations, activation of immune escape pathways, alterations in the tumor microenvironment, and metabolic reprogramming. Additionally, the immune system’s ability to effectively target cancer cells can be compromised by immune tolerance and functional exhaustion. There is growing evidence that abnormal activation of cancer signaling pathways, such as the Wnt/β-catenin, Notch, JAK/STAT, p53, and PTEN pathways, acts independently or through crosstalk to form complex regulatory networks that influence tumor immune escape, the therapeutic response, and immune resistance. Consequently, elucidating the mechanisms by which these signaling pathways contribute to tumor immune escape is essential for optimizing current immunotherapy regimens and developing novel therapeutic strategies. Targeting these pathways in combination with immunotherapy represents a promising approach to overcome immune resistance and enhance treatment efficacy. The integration of pathway modulation with immunotherapeutic interventions may reveal new avenues for more effective and durable cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9142ff6a5cec86bc497f410ddb74aab33a5d7d49" target='_blank'>
              From intracellular drivers to immune modulators: emerging paradigms in oncogenic pathway-directed immunotherapy optimization
              </a>
            </td>
          <td>
            Dean Rao, Tiantian Wang, Chenan Fu, Yiming Luo, Junli Lu, Zhoubin Sun, Wei Yan, Huifang Liang, Limin Xia, Wenjie Huang
          </td>
          <td>2025-10-22</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Acquired treatment resistance in homologous recombination (HR) deficient high-grade serous ovarian cancer (HGSOC) commonly involves restoration of HR, often through secondary, somatic reversion mutations in HR genes. Our recent analysis of HR deficient, end-stage HGSOC found that most resistance mechanisms, including reversion mutations, are subclonal. The aim of this study is to investigate how the subclonal, reversion-negative cells persist during treatment. We are investigating single cell sequencing approaches that enable genotyping of individual cells, such that the phenotype of cells without reversions can be studied. In a pilot experiment we have used two HGSOC cell lines with known reversion mutations. To genotype individual cells and thus overcome the limitations of standard single-cell RNA sequencing (scRNA-seq), we combined short-read whole-transcriptome scRNA-seq with long-read, targeted single-cell cDNA sequencing on an Oxford Nanopore MinION using a 44-gene hybridization capture panel. A custom bioinformatics pipeline, incorporating Flexiplex for multiallelic codon resolution, annotated cell barcodes with their HR driver, reversion and TP53 mutations. Genotypes were then transferred to the short-read dataset following outlier and doublet filtering, and transcriptomic differences between cells with and without reversions were identified through differential gene expression analysis. We have repeated this protocol on a patient ascites sample. Prior short-read DNA sequencing from the cell lines and matched tumor samples defined the driver BRCA1 mutation, reversion events, pathogenic TP53 mutations, and whether there was loss of heterozygosity (LOH) at the BRCA1 loci. We successfully genotyped BRCA1 driver and reversion alleles in both cell lines in the long-read data. Rare cells heterozygous wildtype for the BRCA1 driver mutation were detected, suggesting either back-reversion events or subclones that escaped LOH - cells that were undetectable by bulk short-read methods. Cell genotypes were mapped to the short-read whole transcriptome data, and transcriptional programs that may support cell survival were identified in the non-reverted cells. Our approach demonstrates the utility of unbiased long-read sequencing for detecting BRCA1/2 reversions at single-cell resolution, including potential back-reversion subclones that are challenging to identify through bulk sequencing approaches. The differential gene expression results have identified potential resistance mechanisms. Ongoing work includes incorporation of fusion detection, structural-variant calling, and splice isoform analysis in our bioinformatics pipeline to further elucidate resistance pathways. We anticipate this work will result in an increased understanding of the resistance mechanisms that arise in HGSOC patients with HR deficiency, including identification of subclonal resistance mechanisms that cannot be detected through bulk sequencing approaches.



 Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne Willis, Joy Hendley, AOCS Study Group, Timothy Semple, Alicia Oshlack, Elizabeth L. Christie. Characterizing resistance at single cell resolution in HR deficient HGSOC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7b444080721cb95cb0b3c57900c2232de880b" target='_blank'>
              Abstract PR019: Characterizing resistance at single cell resolution in HR deficient HGSOC
              </a>
            </td>
          <td>
            Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne O. Willis, J. Hendley, Aocs Study Group, T. Semple, Alicia Oshlack, Elizabeth L. Christie
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Pancreatic cancer (PC) is characterized by a profoundly immunosuppressive tumor microenvironment and poor prognosis. Dendritic cells (DCs) are pivotal for antigen presentation and T-cell activation, yet their prognostic and mechanistic roles in PC remain incompletely defined. Methods This study performed weighted gene co-expression network analysis (WGCNA) on transcriptomic data from The Cancer Genome Atlas (TCGA) and two Gene Expression Omnibus cohorts (GSE62165, GSE85916) to identify DC–related gene modules. Consensus clustering based on these modules stratified patients into two immune phenotypes. A four-gene DC–related risk score (DCRS) was constructed using LASSO-Cox regression and validated in independent cohorts. Single-cell RNA sequencing data from 25 PC samples (GSE242230) were analyzed through cell clustering analysis, cell-cell communication analysis, and pathway-specific analysis. Functional assays following KCTD14 knockdown in CAPAN-1 and PANC-1 cell lines assessed its impact on proliferation, migration, invasion, and TNF signaling. Results WGCNA identified 130 overlapping DC–related genes enriched in immune pathways. Two DC–related patient clusters exhibited distinct overall survival (OS) (P < 0.05). The DCRS robustly stratified patients into high- and low-risk groups in both TCGA training and validation sets. DCRS demonstrated good predictive potential for OS and there is a significant difference in OS between the two groups of patients (P < 0.05). Single-cell analysis revealed KCTD14 enrichment in malignant epithelial cells and predicted its interaction with DCs via the TNF-TNFRSF1A axis. In vitro, KCTD14 knockdown significantly reduced PC cell proliferation, colony formation, migration, and invasion, and downregulated TNF-α and TNFRSF1A expression (P < 0.01). Conclusion We identified a novel DC–related gene signature that stratifies PC patients by prognosis and highlights KCTD14 as a novel immunomodulatory oncogene acting through the TNF-TNFR1 axis. Our findings provide a foundation for integrating DCRS into clinical risk assessment and for pursuing KCTD14/TNFR1-targeted therapies to overcome DC-mediated immune suppression in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c2e9b09f8e0021cbaaec894fa62c0adec907605" target='_blank'>
              Dendritic cell–related gene signature in pancreatic cancer stratifies patient subtypes and implicates a KCTD14–TNF signaling axis
              </a>
            </td>
          <td>
            Yuxin Liang, Yuheng Gu, Zilong Zhang, Deyuan Zhong, Hongtao Yan, Yuhao Su, Yahui Chen, Fei Wang, Zhengwei Leng, Xiaolun Huang
          </td>
          <td>2025-09-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the deadliest diseases, with a 5-year survival rate of only 13%. While rare mutations have been identified to drive PDAC therapy resistance, undiscovered potent, post-transcriptional regulatory mechanisms could be key to understanding this resistance. The RNA-binding protein HuR (ELAVL1) is known to play a critical role in PDAC oncogenesis by stabilizing mRNA transcripts involved in key oncogenic pathways. Our lab has shown that HuR is overexpressed in PDAC tumor and promotes multiple pro-survival signaling pathways. To study HuR-mediated PDAC progression in immune competent models that reflect the tumor microenvironment, we deleted ELAVL1 via CRISPR-Cas9 in Kras-p53 mutant-driven (KPC) murine PDAC cells and found that HuR-knockout (KO) KPC tumors grew more slowly compared to KPC wild-type (WT) tumors in orthotopic implantation settings. Using flow cytometry and multiplex immunohistochemistry, we found that HuR-WT tumors had significantly less T cell infiltration and greater endothelial cell infiltration compared to HuR-KO tumors. Deep transcriptomic and proteomic profiling revealed that HuR mediates the expression of multiple metabolic pathways and determines mRNA and protein cargo loading in secreted extracellular vesicles (EVs). In fact, when mice bearing HuR-KO tumors were injected with purified EVs from HuR-WT cells, both tumor size and endothelial cell function in leukocyte trafficking were rescued. In the context of T cell function, HuR directly enriched transcripts in the mTOR pathway, and inhibition of mTOR activity in HuR-WT tumor cells rescued T cell function in co-culture. Consistent with these observations linking tumor-intrinsic HuR to dysregulated endothelial and anti-tumor immune function, HuR-KO tumors were significantly more responsive to immune checkpoint blockade therapy as compared to WT-HuR tumors. Similarly, in a Kras-MYC driven mouse model, HuR-WT tumor cells metastasized at significantly higher rates to the liver compared to HuR-KO tumors. Finally, given the recent advancements in KRAS inhibitors, we tested their efficacy in these pre-clinical models, and surprisingly, we observed that HuR-KO tumors were up to >100-fold more resistant to KRAS inhibition than WT tumors. These observations could be explained by HuR expression and activity being closely linked to aberrant KRAS signaling, where loss of HuR leads to the reliance of compensatory pathways and drives PDAC cells away from a KRAS addiction phenotype, thus providing a setting of KRAS inhibitor resistance. Accordingly, surviving PDAC cells post-KRAS inhibitor treatment in vivo lost HuR expression compared to untreated tumors. Together, these findings highlight that HuR is a global post-transcriptional regulator that: 1) reshapes the tumor microenvironment to be immune evasive; 2) is involved in a feed-forward loop enhancing KRAS pathway activation; and 3) facilitates PDAC metastatic potential. Strategies to target HuR activity and use it as a potential biomarker could lead to improved therapeutic outcomes.



 Miffy Guo, Jennifer Finan, Hen Halamish, Katherine Pelz, Robert Eil, Rosalie Sears, Jonathan Brody. HuR drives immune evasion, metastases and KRAS addiction in pancreatic ductal adenocarcinoma: implications for IO-based and KRAS inhibitor therapies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ebd5486262a2dcbf20ec9a786ef9c644e158e4" target='_blank'>
              Abstract A027: HuR drives immune evasion, metastases and KRAS addiction in pancreatic ductal adenocarcinoma: implications for IO-based and KRAS inhibitor therapies
              </a>
            </td>
          <td>
            Miffy Guo, Jennifer M. Finan, Hen Halamish, Katherine R. Pelz, Robert Eil, Rosalie C. Sears, Jonathan R. Brody
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Loss-of-function mutations in STK11 (also known as LKB1) define a molecular subtype of non-small cell lung cancer that is resistant to immune checkpoint blockade therapy, partially owing to impaired T-cell infiltration and suppressed antigen presentation. Emerging evidence implicates epigenetic deregulation as a key driver of immune evasion in STK11-mutant tumors. In this issue of Cancer Research, Ahronian and colleagues identified the CoREST complex, via histone deacetylase 1, as a vulnerability that can be targeted pharmacologically to restore immune sensitivity in STK11-mutant tumors. The authors developed TNG260, a selective small-molecule CoREST inhibitor with favorable pharmacologic properties and reduced toxicity relative to pan-histone deacetylase inhibitors. In preclinical models of STK11-deficient non-small cell lung cancer, TNG260 reprograms the tumor epigenome to upregulate immune genes and synergizes with anti-PD-1 therapy to induce durable tumor regressions. Early clinical data from an ongoing trial (NCT05887492) show increased histone acetylation and CD8+ T-cell infiltration in patient tumors treated with TNG260 and pembrolizumab. This commentary places these findings in the broader context of epigenetic modulation of antitumor immune response. We also outline key questions for future investigation, including durability of immune response, tumor-type specificity, and biomarker strategies for patient selection and response monitoring. See related article by Ahronian et al., p. 3966.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44a0c3058248639746044326694c26992fc0c20" target='_blank'>
              Precision Epigenetic Reprogramming: CoREST Inhibition Sensitizes STK11-Mutant Tumors to Immune Checkpoint Blockade Therapy.
              </a>
            </td>
          <td>
            De-Chen Lin, Keyue Shen
          </td>
          <td>2025-10-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer therapy and cancer drug discovery and development have been historically focused on specific cancers (tissue/organ of origin). However, with advances in molecular biology and multi-omics of cancer, there is a trend to develop pan-cancer therapeutic modalities. In targeted therapy, pan-cancer strategies target common molecular alterations across different cancer types and specific cancer strategies are tailored to the unique biological characteristics of individual tumor types. Each approach offers distinct advantages and limitations, and understanding these differences is critical in the era of precision oncology. Targeting key molecular drivers in cancer has significantly changed drug development, allowing for broad-spectrum therapeutic strategies that address shared oncogenic pathways across various tumor types. Among these drivers, RAS, PCNA, and MDM2 have become critical targets due to their roles in a broad-spectrum of cancer biology, e.g., cell proliferation, survival, and genomic stability. Advances in molecularly guided therapies have led to promising approaches for disrupting these pathways, offering new opportunities for cancer treatment. Despite significant progress in the past, challenges such as drug resistance, tumor heterogeneity, and toxicity remain obstacles to widespread clinical success. This review explores the historical development, current advancements, and future directions of RAS, PCNA, and MDM2-targeted therapies, emphasizing their potential to reshape cancer treatment through pan-cancer approaches using biomarker-driven technologies, combination strategies, and next-generation inhibitors. These advancements pave the way for more effective and durable therapies across a wide range of malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fed71aa9bfb82d419fcc06f09785baacdca84ec" target='_blank'>
              Targeting MDM2, RAS, and PCNA for cancer targeted therapy: pan-cancer approaches vs. cancer-specific strategies
              </a>
            </td>
          <td>
            Wei Wang, Abigail Alley, Na Sun, Meheret Tadesse, Xinshi Wang, Ruiwen Zhang
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is characterized by stromal remodelling andstiffening, a process involving extracellular matrix (ECM) depositionand cross-linking, which forms a dense physical barrier that limits the penetration of chemotherapeutic agents and restricts immune cell-mediated cytotoxicity. In addition, the tumor stroma modulates nutrients availability and fosters cancer cell invasion and metastasis. Therefore, therapeutic strategies aimed at modulating the tumor stroma hold promise for overcoming some of the major challenges in pancreatic cancer treatment. However, most clinical trials of stromal-targeted therapies have not demonstrated meaningful improvements in overall survival, underscoring an urgent need to identify novel stromal targets and approaches that modulate the tumor microenviroment in ways that enhance therapeutic response and improve patient outcomes. Methods Transcriptomic datasets of PDAC sampleswere obtained from The Cancer Genome Atlas (TCGA). By integrating immune infiltration algorithms with weighted gene co-expression network analysis (WGCNA), this study aims to identify hub genes associated with cancer-associated fibroblasts (CAFs) infiltration in PDAC, explore the prognostic relevance of CAF infiltration, and construct a gene‑expression-based prognostic mode. Furthermore, we will employ a multi-modal approach incorporating magnetic resonance elastography, atomic force microscopy, single-cell RNA sequencing (scRNA-seq), histological assays, cell-based functional experiments, and animal models to identify candidate core genes involved in CAF infiltration and stromal stiffening in pancreatic cancer, and to assess the potential therapeutic relevance of targeting these genes. Results In PDAC, increased stromal stiffness and poor prognosis are both strongly linked to activation of CAFs. WGCNA identified six hub genes—ANTXR2, ANXA1, BCAT1, PLAU, TGFβ1, and ITGB5—which were used to construct a predictive model for estimating CAF abundance and forecasting PDAC prognosis. Single-cell RNA sequencing analysis showed that ITGB5 is highly expressed in CAFs. Both in vitro and in vivo experiments demonstrated that targeting ITGB5 in CAFs reduces tumor stiffness, thereby inhibiting the malignant progression of PDAC. These findings suggest a potential therapeutic benefit, although impact on clinical prognosis remains to be established. Conclusion The integration of magnetic resonance elastography, atomic force microscopy, and single-cell sequencing allows detailed research characterization of the biomechanical properties of PDAC. ITGB5 has been identified as an important regulator of the biomechanical features of PDAC. Targeting ITGB5 in CAFs may reduce the excessive stiffness of PDAC tissue and mitigate gemcitabine-associated stromal fibrosis, suggesting a potential strategy to improve treatment response. The effect on patient prognosis, however, requires confirmation in clinical studies. Trial registration This study is a prospective single-center clinical trial, which has been registered in the U.S. Clinical Trials Database (https://clinicaltrials.gov/study/NCT06526442) with the trial registration number NCT06526442. The registration date was October 9th, 2020. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07119-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303c7c686718b4d13d9fc453421d878d4dcfcaa1" target='_blank'>
              ITGB5-mediated biomechanical regulation in pancreatic ductal adenocarcinoma stroma impacts tumor progression and prognosis
              </a>
            </td>
          <td>
            Aoran Yang, Chengli Gu, Yining Liu, Baihe Luo, Yu Zeng, Minghui Zhou, Chunli Li, Chen Pan, Linxuan Wu, Zhiwen Hu, Xin Zhang, Feng Gao, Yu Shi
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Diffuse midline gliomas (DMGs) are universally fatal pediatric brain tumors with a median survival of just 11 months. Despite radiation therapy offering temporary benefit, there are no effective long-term treatments, and targeted therapies have shown minimal impact. The infiltrative nature of DMGs, combined with their critical brainstem location, underscores the urgent need to identify novel mechanisms driving tumor progression and resistance. Cellular senescence is a state of permanent cell-cycle arrest induced by stress and has traditionally been viewed as tumor-suppressive. However, recent evidence in adult cancers implicates senescent cells as active contributors to tumor growth, treatment resistance, and relapse through the secretion of pro-tumorigenic factors known as the senescence-associated secretory phenotype (SASP). While senescence has emerged as a hallmark of cancer in adults, its role in pediatric tumors such as DMGs remains largely unexplored. Using single-cell RNA sequencing of treatment-naïve DMG specimens, we identified rare populations of senescent glioma cells and microglia that would be undetectable using conventional approaches. Computational analyses revealed predicted paracrine interactions between senescent and non-senescent cells including pathways previously linked to tumor invasiveness and resistance. This study aims to define how senescent cells and their secretory programs shape the tumor microenvironment, promote DMG progression, and contribute to therapy resistance. By elucidating these mechanisms, we hope to uncover new therapeutic vulnerabilities in DMGs and expand our broader understanding of senescence as a driver of malignancy in pediatric cancers.



 Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya M. Rangaswamy, Claudia Kleinman, Pratiti Bandopadhayay. Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785f69efbe0f5fccc4e2b6c0500f58472565b0dc" target='_blank'>
              Abstract B040: Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance
              </a>
            </td>
          <td>
            Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya Rangaswamy, Claudia L Kleinman, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768ee203fcd258706f2498c5fcf4ab4120a2eda" target='_blank'>
              Somatic evolution of prostate cancer: mutation, selection, and epistasis across disease stages
              </a>
            </td>
          <td>
            Moein Rajaei, Alexander Yang, Kira A. Glasmacher, Christopher N. Cross, Nic Fisk, Elizabeth B. Perry, Jeff Mandell, S. Gaffney, Takafumi N. Yamaguchi, J. Livingstone, Jose Costa, Peter Humphrey, Vincent L. Cannataro, P. Boutros, Jeffrey P. Townsend
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb0d181b971ab9d5141b1475ee18608b8080d802" target='_blank'>
              Molecular Plasticity of T Cells Informs Their Possible Adaptation in 4T1 Tumors
              </a>
            </td>
          <td>
            Md. Iftehimul, Robert H. Newman, Scott H Harrison, Roshonda B. Jones, P. Muganda, Bryan L. Holloman, M. T. Hossain, C. J. Rorie, Misty D. Thomas, Joseph L. Graves, Howard L. Kaufman, Dipongkor Saha
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Penile squamous cell carcinoma (PSCC) is a malignancy characterized by a poor prognosis and lack of reliable biomarkers, presenting considerable therapeutic challenges. Tertiary lymphoid structures (TLSs) are critical modulators of antitumor immunity; however, the immunological dynamics, particularly the roles of B cells and their interactions with naive T cells in PSCC tissues, remain inadequately understood. This study integrates transcriptomic approaches, including spatial transcriptomics, single-cell sequencing (scRNA-seq), and bulk RNA sequencing (bulk RNA-seq), and immunohistochemistry to elucidate the immune architecture of and functional mechanisms within TLSs. The results reveal a positive correlation between TLS density and patient survival, with CD74⁺ B cells tending to be enriched during early TLS formation. These cells exhibit strong immune activation and a propensity to differentiate into plasma cells. By engaging with naive T cells through HLA-DRA via ligand-receptor interactions, CD74⁺ B cells activate transcription factors, including NFKB1, NFKB2, NFATC1, NFATC2, FOS, and RUNX1, in naive T cells, thereby enhancing the immune response. Consequently, CD74⁺ B cells represent a compelling biomarker for and therapeutic target of PSCC, offering profound insights into the immunological mechanisms that drive PSCC progression and response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f775dcc4d8e646b4cd53d88e405d27443da1f7f" target='_blank'>
              Spatial Transcriptional Dynamics of CD74⁺ B Cells in Tertiary Lymphoid Structures Drive Immune Evolution in Penile Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Ting Xue, Chuangzhong Deng, Jingya Liu, Ru Yan, Jing Li, Xiheng Hu, Xueying Li, Xiao Xiao, Jietian Jin, Hongzhen Tang, Desi Chen, Zihan Zuo, Yujie Liang, Dongbin Wang, Bonan Chen, Hui Han, Zaishang Li
          </td>
          <td>2025-10-20</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) is the most abundant internal RNA modification, orchestrated by writers, erasers, and readers. METTL14, a key component of the m6A methyltransferase complex, acts as a structural scaffold that ensures substrate recognition and modification precision. Beyond this canonical role, METTL14 regulates multiple biological processes, including chromatin remodeling, transcriptional activity, and senescence-associated signaling. Recent studies highlight its pivotal function in tumor immunity: METTL14 shapes T cell differentiation, CD8+ T cell activation, and the activity of macrophages and NK cells, thereby remodeling the tumor immune microenvironment. Moreover, METTL14 directly modulates immune checkpoint pathways by regulating PD-1 and PD-L1 expression, linking epitranscriptomic control with immune escape and therapeutic resistance. Aberrant METTL14 expression correlates with tumor progression and immune evasion, underscoring its potential as a predictive biomarker and therapeutic target. Targeting METTL14, alone or in combination with immune checkpoint inhibitors, may provide novel strategies to enhance immunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1847345d5b041c2cd26c401bdc866e8b5fa04449" target='_blank'>
              METTL14 in tumor immunity: epitranscriptomic regulation and therapeutic potential
              </a>
            </td>
          <td>
            Chunhong Li, Xiulin Jiang, Yixiao Yuan, Qiang Wang
          </td>
          <td>2025-10-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum.
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell research</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Single-cell perturbation dictionaries provide systematic measurements of how cells respond to genetic and chemical perturbations, and create the opportunity to assign causal interpretations to observational data. Here, we introduce RNA fingerprinting, a statistical framework that maps transcriptional responses from new experiments onto reference perturbation dictionaries. RNA fingerprinting learns denoised perturbation “fingerprints” from single-cell data, then probabilisti-cally assigns query cells to one or more candidate perturbations while accounting for uncertainty. We benchmark our method across ground-truth datasets, demonstrating accurate assignments at single-cell resolution, scalability to genome-wide screens, and the ability to resolve combinatorial perturbations. We demonstrate its broad utility across diverse biological settings: identifying context-specific regulators of p53 under ribosomal stress, characterizing drug mechanisms of action and dose-dependent off-target effects, and uncovering cytokine-driven B cell heterogeneity during secondary influenza infection in vivo. Together, these results establish RNA fingerprinting as a versatile framework for interpreting single-cell datasets by linking cellular states to the underlying perturbations which generated them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec6f6436c33f356603961f4c042e63aa8c6b591" target='_blank'>
              Mapping transcriptional responses to cellular perturbation dictionaries with RNA fingerprinting
              </a>
            </td>
          <td>
            Isabella N. Grabski, Junsuk Lee, John D Blair, Carol Dalgarno, Isabella Mascio, Alexandra Bradu, David A Knowles, R. Satija
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality, with genetic, epigenetic, and immune-dependent mechanisms contributing to disease pathogenesis. Preventive strategies are especially important for individuals with hereditary syndromes such as familial adenomatous polyposis (FAP), in which Adenomatous Polyposis Coli (APC) mutations drive early tumor formation. The polyposis in rat colon (Pirc) model, harboring an Apc mutation, faithfully recapitulates the development of adenomatous polyps, a precursor stage of CRC, in both the colon and duodenum. Here, we report the generation and characterization of primary cell cultures derived from 30 colonic and duodenal adenomas (benign tumors) from Pirc rats. These Pirc colon adenoma and duodenal adenoma cultures demonstrated consistent morphology, epithelial marker expression (E-cadherin and pan-cytokeratin), and robust 3D spheroid formation. Growth kinetics revealed a doubling time of 50 ± 4 hours. Notably, histone deacetylase and bromodomain inhibitors reduced cell viability and colony formation significantly, highlighting the synergistic potential in targeting deregulated epigenetic signatures as early-stage prevention strategies in FAP. These primary cultures, maintained between passages 5 and 10 to preserve phenotypic integrity, offer a valuable ex vivo model for early-stage CRC research and the screening of preventive agents. By bridging genetic susceptibility and translational prevention, this platform provides a novel and reproducible tool for advancing cancer interception.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6739c771e5dffc00116bad98344e26be1545833" target='_blank'>
              Development of Primary Colon and Duodenal Tumor Cultures from the Apc-mutant Pirc Rat for Screening Preventive Agents
              </a>
            </td>
          <td>
            Ahmed Muhsin, Sultan Neja, Nivedhitha Mohan, Sabeeta Kapoor, J. T. Perez, W. Dashwood, Asif Rashid, Roderick H. Dashwood, P. Rajendran
          </td>
          <td>2025-09-10</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9acc0f4914f12bf8600b5ee6d644cccba2b026" target='_blank'>
              ResNet50 and Single-Cell Multi-Omics analysis identify key cellular and molecular features in pediatric acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Taoli Suo, Jing Zhang, Yanli Chen, Hongyu Shi, Haiyan Lu
          </td>
          <td>2025-10-23</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intratumoral microbiota, a critical component of the tumor microenvironment, notably impacts tumor progression through various complex mechanisms such as metabolic regulation, immune system remodeling and genotoxicity. The present review focuses on eight prevalent solid tumors (breast, colorectal, lung, pancreatic, gastric, ovarian, prostate cancers and melanoma), detailing the intratumoral microbial compositional heterogeneity within these malignancies. The present review analyzes the heterogeneous carcinogenic mechanisms mediated by these microorganisms, including gene instability, immune microenvironment remodeling and metabolic intervention. The diagnostic value of microbial markers in liquid biopsy and in situ tissue detection is explored, and the potential for synergistic strategies combining microbial-targeted therapy and immunotherapy is discussed. Finally, the review suggests future research directions, such as spatiotemporal dynamic analysis and organoid-microorganism co-culture, offering new insights for precise cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09bce21033da16d36e409d313fbe6c3d34814bec" target='_blank'>
              Tumor-associated microbiota: Multi-cancer landscape, mechanistic insights and clinical translation (Review)
              </a>
            </td>
          <td>
            Jindi Yang, Yige Dong, Yi Chen, Hong Liang, Shengyu Rong, Zhe Liu, Qiulei Lang
          </td>
          <td>2025-09-16</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The morphogenesis of the primordial gut relies on signaling pathways such as Wnt, FGF, Notch, Hedgehog, and Hippo. Reciprocal crosstalk between the endoderm and mesoderm is integrated into the signaling pathways, resulting in craniocaudal patterning. These pathways are also involved in adult intestinal homeostasis including cell proliferation and specification of cell fate. Perturbations in this process can cause growth disturbances manifesting as adenomas, serrated lesions, and cancer. Significant differences have been observed between right and left colon cancers in the hindgut, and between the jejunoileum, appendix, and right colon in the midgut. The question is to what extent the embryology of the mid- and hindgut contributes to differences in the underlying tumor biology. This review examines the precursor lesions and consensus molecular subtypes (CMS) of colorectal cancer (CRC) to highlight the significance of embryology and tumor microenvironment (TME) in CRC. The three main precursor lesions, i.e., adenomas, serrated lesions, and inflammatory bowel disease-associated dysplasia, are linked to the CMS classification, which is based on transcriptomic profiling and clinical features. Both embryologic and micro-environmental underpinnings of the mid- and hindgut contribute to the differences in the tumors arising from them, and they may do so by recapitulating embryonic signaling cascades. This manifests in the range of CRC CMS and histologic cancer subtypes and in tumors that show multidirectional differentiation, the so-called stem cell carcinomas. Emerging evidence shows the limitations of CMS particularly in patients on systemic therapy who develop drug resistance. The focus is thus transitioning from CMS to specific components of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87a5db2d1894f9efd82dcfae525f5b0d6a5bde2f" target='_blank'>
              Embryonic Signaling Pathways Shape Colorectal Cancer Subtypes: Linking Gut Development to Tumor Biology
              </a>
            </td>
          <td>
            Kitty P. Toews, F. M. Auld, Terence N. Moyana
          </td>
          <td>2025-10-01</td>
          <td>Pathophysiology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human leukocyte antigen (HLA) polymorphism hinders off-the-shelf cell therapies. We developed REWRITE, a modular platform for iterative, scar-minimized genome writing of synthetic constructs >100 kb in human pluripotent stem cells (hPSCs). Using REWRITE, we deleted 105–209 kb of the HLA locus and installed synthetic 24 kb or 100 kb HLA haplotypes, and a 62 kb antigen-processing locus. This uncovered a persistent, heritable “epigenetic ghost” — an active state lingering despite genetic removal — whose resolution to a silenced default state is driven by native intergenic DNA. These loci restored inducible expression in key lineages, sparing cells from NK-mediated killing and establishing HLA-matched T-cell tolerance, enabling off-the-shelf cell therapies. REWRITE facilitates extensible programming of multigenic functions in allogeneic human cells — from immune design to genome architecture discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3662bdeb8a587004f28d0c02911c74aa245dde2b" target='_blank'>
              Human Genome REWRITE for Off-the-Shelf Stem Cells Reveals an “Epigenetic Ghost”
              </a>
            </td>
          <td>
            Serena F. Generoso, Sarah Levovitz, Susanna Jaramillo, Minjoo Kim, Sumanth Dara, Shean Fu Phen, Bryan Yi, Tomoki Yanagi, Thomas L. DesMarais, Neta Agmon, Megan S. Hogan, Leslie A. Mitchell, David M. Truong
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Cell identity, tightly regulated by transcription factors (TFs), is frequently disrupted in cancer, driving tumor progression and metastasis. In pancreatic ductal adenocarcinoma (PDAC), alterations in cell identity contribute to disease heterogeneity and therapy resistance. Integrated genomic, transcriptomic, and proteomic analyses have defined two major PDAC subtypes: classical and basal. Notably, recent evidence suggests that changes in subtype identity can enable resistance to targeted therapies. These subtypes are distinguished by relative levels of endodermal lineage-specifying TFs such as FOXA1/2 and HNF4α, which are maintained in the classical subtype but downregulated in the basal subtype. Understanding how these TFs shape cellular identity is critical for identifying therapeutic vulnerabilities and mechanisms of drug resistance. We have shown that HNF4⍺ promotes the classical subtype program of PDAC. Although FOXA1/2 and HNF4α regulate gene expression in normal tissues, it is still unclear whether such cooperation is necessary to drive the classical transcriptional program in PDAC. We hypothesize that FOXA1/2 and HNF4α cooperatively regulates the gene expression program that regulates the classical subtype program in PDAC. Using human cell lines and murine PDAC organoid models, we reconstituted and modulated TF expression to evaluate downstream gene expression and TF genomic binding via ChIP-seq. Conditional alleles of FoxA1/2 in our murine organoid lines reveal that loss of FOXA1/2 lead to a decrease in the classical program. When we overexpress HNF4α in FOXA1/2-null organoids, it leads to a decrease in organoid growth; therefore, illustrating the ability of HNF4α to regulate growth independent of FOXA1/2. In a human cell line, we find that co-expression of FOXA1/2 and HNF4a activates the classical B signature while FoxA2 alone activates the Classical A signature, indicating that FOXA2 and HNF4a activate subtype-specific transcriptional programs. Additionally, we demonstrate that the overexpression of HNF4a in a subcutaneous xenograft model confers resistance to KRASG12C inhibition, leading to an increase in tumor size and weight. Unexpectedly, we find that these TFs do not cooperatively regulate gene expression; rather, in the human cell line, FOXA2 interferes with HNF4α genomic binding, suggesting a competitive or inhibitory interaction. Overall, our work highlights how lineage-defining TFs govern PDAC cell identity and therapeutic response.



 Walter A. Orellana, Sydney N. Larsen, Eric L. Snyder. FOXA1/2 and HNF4α Regulate Molecular Subtype and Therapeutic Response in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B107.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10be5668747dfa95fe8d23992fe6617771de029e" target='_blank'>
              Abstract B107: FOXA1/2 and HNF4α Regulate Molecular Subtype and Therapeutic Response in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Walter A Orellana, Sydney N. Larsen, Eric L. Snyder
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is considered an immunologically cold cancer type. This perception is due to its rapid progression, strong presence of immunosuppressive cells and lack of response to current immunotherapies. However, we and others have shown that many patient PDAC tumors contain robust lymphocyte infiltration, aggregated in the form of tertiary lymphoid structures (TLS). The presence of TLS in PDAC is prognostic for long-term survival and in other cancer types, predictive of response to immunotherapy. Despite the clinical benefit to generating TLS in tumors, why they form in some PDAC patients but not others, remains unknown. The desmoplastic stroma in PDAC contributes to defective anti-tumor immunity largely due to myofibroblastic cancer associated fibroblasts (myCAF) induced by transforming growth factor-beta (TGFb). Other groups have demonstrated that mesenchymal cell phenotypes with lymphoid tissue organizing properties regulate TLS formation in other tumor types. In order to restore effective anti-tumor immunity for PDAC patients, CAF phenotypes must be reprogrammed to support rather than restrict intratumoral immune activity. Using a lymphotoxin beta receptor (LTBR) agonist we observed the induction of TLS-like aggregates in some murine PDAC tumor models, but not others. The CAF phenotypes of TLS-resistant models were predominantly myCAF while the TLS-permissive models were enriched for a VCAM1+ICAM1+ reticular-CAF (rCAF) subset. Induction of myCAF differentiation in vitro with TGFb1 silenced LTBR/TNFR mediated upregulation of CXCL13 and CCL19 chemokine expression and diminished B and T cell migration towards fibroblasts. Therapeutic TGFbR1 inhibition in tumor bearing mice abrogated these effects thereby allowing LTBR agonism to repolarize rCAF phenotypic programming associated with improved lymphocyte recruitment and T cell-dependent tumor control. In patient PDAC tumors, rCAF were spatially arranged encapsulating TLS while myCAF were located distally. These data indicate that myCAF-regulated PDAC stroma antagonizes acquisition of rCAF subsets critical for TLS formation but can be therapeutically remodeled to promote beneficial immune responses in PDAC tumors.



 Andrew J. Gunderson, Elijah Kirschstein, Olivia Harder, Jordan Krull, Madison Sikorski, Shrijan Khanal, Morgan Mack, Carl F. Ware, Elizabeth Evans, Michael J. Gough, Qin Ma, Wei Chen, Kristina Young. Myofibroblast programming blocks differentiation of immune enhancing fibroblastic reticular cells in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A094.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e530178f3d124403b585ae07a807d4c54e5bc663" target='_blank'>
              Abstract A094: Myofibroblast programming blocks differentiation of immune enhancing fibroblastic reticular cells in pancreatic cancer
              </a>
            </td>
          <td>
            Andrew J Gunderson, Elijah Kirschstein, Olivia Harder, Jordan Krull, Madison Sikorski, Shrijan Khanal, Morgan Mack, Carl F Ware, Elizabeth Evans, Michael J. Gough, Qin Ma, Wei Chen, Kristina Young
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/224fea21345ecef881ecf122aedc0be968301c73" target='_blank'>
              Leveraging Experimental Evolution to Extract Predictive Collateral Drug Response Signatures in Ewings Sarcoma
              </a>
            </td>
          <td>
            Kristi Lin-Rahardja, Ethan Canavan, Jacob G. Scott
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains one of the deadliest cancers, with a five-year survival rate of just 13%; a consequence of late diagnosis and limited therapeutic options. Surgical resection is the only potential cure, yet most patients are ineligible, and recurrence is frequent even after surgery. These challenges underscore the urgent need for advanced strategies. Given PDAC’s unique biology, integrating tumor microenvironment (TME) reprogramming with robust, status-informing theranostic markers represents a promising direction for future research. This is because a defining feature of PDAC is its dense, fibrotic stroma, driven by functional units of cancer-associated fibroblasts (CAFs) and their extracellular matrix (ECM). Unlike normal, tumor-suppressive fibroblast/ECM units, CAF/ECM functional units adopt an activated phenotype, often mediated by pathways such as TGF-β that alters integrin-dependent cell-ECM interactions, promotes tumor progression, and shapes drug responses through dynamic tumor-stroma signaling reciprocity. However, indiscriminate stromal ablation can paradoxically accelerate tumor growth, highlighting the need to precisely distinguish and modulate tumor-promoting versus tumor-suppressive CAF/ECM states. Recent evidence suggests that Netrin G1 (NetG1)-positive CAFs enhance glutamate and glutamine exchange with cancer and immune cells via the NetG1–NGL1 axis, mimicking synaptic communication and fostering both tumor growth and immunosuppression. Inhibition of this pathway has been shown to suppress PDAC progression in experimental models and the expression of these molecules in TME and cancer cells correlates with poor patient outcomes. Notably, although the clinical utility of NetrinG family members remains to be fully elucidated, recent analyses of publicly available transcriptional data reveal that high NetG2 expression, unlike published data for NetG1, correlates with improved survival. These findings suggest that additional NetrinG proteins could serve as modulators and status-informing markers of the TME, with the added potential to become new druggable TME function upholding targets. Here, we investigate the diverse, context-dependent roles of NetrinG proteins, including NGL1, NetG1, NGL2, NetG2, NGL3, and LAR, in modulating fibroblast states and the PDAC TME. Using optimized 3D fibroblast/ECM unit models and advanced imaging workflows that include Python-based analysis, FIJI, and AI-powered tools like StarDist, we dissect ECM architecture, nuclear features, and more. Our findings establish a research platform for unraveling the mechanistic and therapeutic potential of NetrinG proteins in PDAC and provide a basis for future studies aimed at validating their roles as potential theranostic targets.



 Aleksandr Dolskii, Olivia C. Williams, Tiffany Luong, Jaye Gardiner, Janusz Franco-Barraza, Mariia Dmitrieva, Sérgio A. Alcantara Dos Santos, Edna Cukierman. Stromal dynamics in PDAC: The role of NetrinG proteins in regulating fibroblast function [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A054.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a52f0b4ac58147aaae90bc85d2ba84286cfc5c90" target='_blank'>
              Abstract A054: Stromal dynamics in PDAC: The role of NetrinG proteins in regulating fibroblast function
              </a>
            </td>
          <td>
            Aleksandr Dolskii, Olivia C. Williams, T. Luong, Jaye C. Gardiner, J. Franco-Barraza, Mariia Dmitrieva, S. A. Alcântara dos Santos, Edna Cukierman
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive solid malignancy with poor overall prognosis and limited response to standard treatments. Growing interest in the modulation of DNA repair mechanisms, including the homologous recombination (HR) repair pathway, has opened new avenues for therapeutic development. BARD1 (BRCA1-Associated RING Domain 1) plays a complex role in tumor biology, functioning either as a tumor suppressor or as an oncogenic driver, depending on isoform expression, cellular context, and regulatory environment. In this review, we examine the dual roles of BARD1, focusing on its regulation and paradoxical activities in PDAC. We summarize evidence that BARD1 and BARD1 isoforms differentially affect DNA repair, apoptosis, and drug resistance and evaluate the therapeutic potential of targeting BARD1 and other DNA damage response (DDR) proteins. We also review ongoing clinical trials and investigational agents designed to exploit DDR vulnerabilities in PDAC. Together, these insights highlight BARD1’s context-dependent roles in PDAC and support continued efforts to profile BARD1 isoforms, clarify their functions, and leverage DDR pathways through precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c070f9d1ec5a160a1dcacf90475e9772e358bc" target='_blank'>
              BARD1: A Friend or Foe in Pancreatic Ductal Adenocarcinoma?
              </a>
            </td>
          <td>
            Lily Zekavat, Aditi Jain
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Osteosarcoma (OS) remains a challenging malignancy, particularly for metastatic cases, due to its heterogeneous genetic landscape characterized by lack of actionable oncogenic drivers and thus lack of personalized therapy. Though personalization of therapy based on functional assays using patient derived cells is emerging as a promising approach, it was not explored as a workflow for identifying patient specific drug cocktails. In this study we investigated a sequential drug screening approach for single and pair drugs which can be rationally used to identify 4 drug cocktails. We started with a systematic screening of 17 drugs from multiple classes across four osteosarcoma cell lines (U2OS, MG-63, SaOS-2, and K7M2), and observed differential drug responses among the cell lines. Interestingly, leading large language models (LLMs) failed to predict cell specific efficacy in OS cells while they were successful in KRAS mutation driven cells. Both SaOS-2 and K7M2 showed sensitivity to kinase inhibitors, particularly ponatinib, and we further explored their combinatorial therapeutic potential. We identified promising non-chemotherapeutic drug pairs, including trametinib-ponatinib and rapamycin-ganetespib and demonstrated potent synergistic effects. To mitigate dose limiting toxicities, drug pairs were formulated in polydopamine-stabilized nanoparticles and K7M2 murine models in vivo studies revealed that they preferably accumulated in tumors and were highly superior to the chemotherapeutic standard of care. Surprisingly, alternating administration of nanoparticle drug pairs was superior to concomitant regimen in both efficacy, survival and toxicity profiles. These findings strengthen a functional approach for combinatorial personalized treatment, potentially overcoming the limitations of current therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-20222-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a634be8214640979f220b1d8acd8f6cf52b7a021" target='_blank'>
              Functional personalized complex combination nano therapy for osteosarcoma
              </a>
            </td>
          <td>
            Orr Bar-Natan, Yuval Harris, Hagit Sason, Yosi Shamay
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the second leading cause of cancer-related deaths worldwide. Despite progress in screening and therapeutic strategies, a substantial proportion of patients are diagnosed at advanced stages or experience disease recurrence, both of which are associated with poor prognosis. Current standard-of-care treatments, including chemotherapy, targeted agents and immunotherapy, are often limited by the emergence of drug resistance or the immunologically ¨cold¨ nature of CRC. Immunotherapies failed to benefit the vast majority of CRC patients beyond the mismatch repair deficient (dMMR) subtype. These challenges highlight the need for more effective targeted immunotherapies capable of deep and durable responses. LGR5 (Leucine-rich repeat-containing G protein-coupled receptor 5) has high expression levels in CRC and other solid tumors, including gastric, pancreatic, and ovarian cancer. In CRC, LGR5 prevalence is high, and expression correlates with poor clinical outcomes. Physiologically, LGR5 expression is limited to adult stem cells, such as in intestinal crypts and hair follicles. In contrast, LGR5 is frequently overexpressed and broadly distributed in malignant tissues, making it a suitable candidate for therapeutic targeting. CDR609 is a first-in-class T cell engager (TCE) for the treatment of LGR5-expressing solid tumors, based on the same M-gager® format as CDR404 (Phase I, NCT06402201). It binds a specific epitope on LGR5 critical for facilitating immunological synapse formation and subsequent T cell activation. With an LGR5 monovalent affinity in the pM range, CDR609 shows very efficient cancer cell killing of LGR5+ cancer cell lines in co-cultures with human PBMCs. This is accompanied by upregulated expression of activation (CD69, CD25) and proliferation (Ki67) markers on CD8+ T cells. T cell activation was further confirmed by the presence of cytokines and cytolytic molecules detected in culture supernatants, including IFNγ and granzyme B. The excellent specificity profile was demonstrated by the lack of relevant T cell reactivity against various LGR5-negative cell lines, as well as normal human cells from different tissues. The biophysical properties of the molecule were optimized for an ideal developability profile. Stability and solubility measurements predict good shelf life and in vivo stability. In conclusion, CDR609 efficiently activates T cells and kills LGR5-positive cancer cells. It has the potential to generate robust immunological activity in patients with CRC and other cold tumors, as demonstrated in Phase I with the CDR404 M-gager® in patients with a variety of cold tumors. CDR609 also shows high specificity for LGR5, and an overall safe profile in the presence of normal human cells. Future studies will include in vivo efficacy testing and a broader range of cellular assays using ex vivo patient-derived tissues and cultured cells as part of the preclinical studies that will be included in the IND application.



 Athanasia Dasargyri, Sophie Barsin, Fabian Scheifele, Philipp Richle, Nagjie Alijaj, Anna Sobieraj, Romina Doerig, Hannes Merten, Martina Priola, Blaz Pavlovic, Marko Keric, David Schurter, Cedric Kiss, Ariadna Vilarrasa, Stephanie Jungmichel, Matteo Morotti, Swethajit Biswas, Christian Leisner, Leonardo Borras. CDR609: A First-in-Class LGR5-targeted T Cell Engager for treatment of Colorectal Cancer and other solid tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A126.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a61e5c10d8b8b32bf0ed5d930cd37ac9f4e81a5" target='_blank'>
              Abstract A126: CDR609: A First-in-Class LGR5-targeted T Cell Engager for treatment of Colorectal Cancer and other solid tumors
              </a>
            </td>
          <td>
            Athanasia Dasargyri, Sophie Barsin, F. Scheifele, P. Richle, Nagjie Alijaj, A. Sobieraj, R. Doerig, H. Merten, M. Priola, Blaz Pavlovic, Marko Keric, David Schurter, Cedric Kiss, A. Vilarrasa, S. Jungmichel, Matteo Morotti, Swethajit Biswas, Christian Leisner, Leonardo J. Borras
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Tumor-associated macrophages (TAMs) represent the main immune cell population in various brain malignancies, but their role in the meningioma (MGM) microenvironment remains to be elucidated.



 Here, we investigated TAM frequencies, activation state, survival-associated changes, and their association with tumor-infiltrating T-lymphocytes (TILs) in two independent study samples comprising altogether 680 MGMs. To this end, we performed tissue cytometry analyses, quantified tissue cytokine levels, and integrated previously-published TIL infiltration and microarray datasets in the discovery cohort (n=195 clinically well-annotated cases). This was complemented by a DNA methylation-based deconvolution approach to predict TAM and TIL infiltration rates and survival associations in an independent validation cohort of n=485 MGMs.



 Our findings revealed substantial but heterogeneous TAM infiltration in newly-diagnosed MGMs, with increased numbers of anti-inflammatory TAMs in clinically-aggressive tumors. Cytokine and transcriptome analyses corroborated the presence of an immunosuppressive niche in TAM-enriched MGMs. Importantly, high TAM infiltration was identified as an independent prognostic factor for poor survival, counteracting the beneficial prognostic effect of TILs. Furthermore, deconvolution analyses confirmed the opposing prognostic roles of TAMs and TILs in the validation cohort.



 Methylation-based deconvolution to explore the prognostic impact of TAM and TIL composition in menigiomas may serve as a potential biomarker for immunotherapeutic treatment strategies. Furthermore, our findings highlight pro-tumoral TAMs to be an attractive treatment target in MGMs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b541053e2ec078b9d9af0d16f05da69f5740e693" target='_blank'>
              OS13.5.A INTEGRATIVE MULTIOMICS ANALYSIS IDENTIFIES TUMOR-ASSOCIATED MACROPHAGES AS NOVEL INDEPENDENT BIOMARKER FOR POOR SURVIVAL IN MENINGIOMAS
              </a>
            </td>
          <td>
            C. Lotsch, R. Warta, F. Liu, G. Jungwirth, C. Rommel, K. Lamszus, A. Kessler, M. Löhr, R. Ketter, C. Senft, N. Maas, P. Sievers, M. Westphal, S. Krieg, A. Unterberg, M. Simon, A. von Deimling, F. Sahm, D. Raleigh, C. Herold-Mende
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is classified into two distinct subtypes, classical and basal. Basal PDACs (∼25% of PDACs) have the worst overall survival and are the only class to act as an independent poor prognostic factor. Therefore, there is an urgent need to understand basal PDAC development, identify biomarkers and find its unique vulnerabilities. We found high expression of the chromatin architectural protein, HMGA2 in the basal subtype of pancreatic cancer. This was across patient-derived cell lines, organoids and in RNA-seq and scRNA-seq patient datasets representing hundreds of patients. We also validated by multiplex immunohistochemistry (mIHC) in >500 diverse patient tissue microarray. We introduce a novel genetically engineered mouse model (GEMM) of basal pancreatic cancer. We find that in this model and in human orthotopic xenografts, HMGA2 is required for cancer growth and sufficient to drive a basal subtype. While HMGA2 is a chromatin factor, we have unexpectedly discovered that it plays an essential role in mRNA translation. Our data demonstrate that HMGA2 controls protein synthesis dynamics through reduced methylation of the protein phosphatase PP2A and subsequent activation of S6 Kinase, a key node of translation regulation. HMGA2 accomplishes this through inhibiting the expression of leucine carboxyl methyltransferase 1 (LCMT1). Importantly, loss of LCMT1 in the classical subtype is sufficient to increase mRNA translation and cause the acquisition of basal features. Thus, we describe a new mechanism by which lineage plasticity is regulated in pancreatic cancer. Most importantly, we demonstrate that high levels of HMGA2 in the basal subtype, and high levels of mRNA translation, render basal pancreatic cancer uniquely susceptible to inhibitors of protein synthesis. As such, we describe a new biomarker and subtype-specific treatment for basal pancreatic cancer.



 Sita Kugel, Stephanie Dobersch, Naomi Yamamoto, Aidan Schutter, Sarah M. Cavender, Tess M. Robertson, Gun Ho Jang, Annie N. Samraj, Faiyax Notta, Robert N. Eisenman, Andrew C. Hsieh. HMGA2 and protein leucine methylation drive pancreatic cancer lineage plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B111.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c6dd4619bfe9206ea9758d7086ed5b8a18e7ce8" target='_blank'>
              Abstract B111: HMGA2 and protein leucine methylation drive pancreatic cancer lineage plasticity
              </a>
            </td>
          <td>
            S. Kugel, Stephanie Dobersch, Naomi Yamamoto, Aidan Schutter, Sarah M. Cavender, Tess M. Robertson, Gung Ho Jang, Annie N Samraj, Faiyax Notta, Robert N Eisenman, Andrew C. Hsieh
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 To identify novel drivers of pancreatic ductal adenocarcinoma (PDAC) malignancy, we employed a systems biology approach called regulatory network analysis, which accurately infers the activity of transcription factors and other regulatory proteins based on the integrated expression of their positive and negative target genes. This highly validated approach enables the identification of the most hyper-activated and hyper-repressed regulatory proteins (“master regulators”) that drive phenotypic distinctions. We applied this technique to a set of 200 laser capture microdissected human PDAC samples as well as 45 low-grade precursors for which we had matched histopathological, clinical, and epidemiological annotation. We identified the master regulators associated with 4 malignancy phenotypes: tumor initiation, tumor progression, survival post resection, and association with KRAS activity. Integrating across these phenotypes, the top master regulator was BMAL2, a member of the PAS family. Although BMAL2 is classically associated to the circadian rhythm, gene set enrichment analysis highlighted a potential role in hypoxia response, a key phenotype in PDAC evolution. We previously demonstrated that PDAC in humans and in the genetically engineered “KPC” mouse model is hypovascularized and hypoperfused, and we have since shown that KPC tumors are profoundly hypoxic. Sequence conservation analysis of BMAL2 demonstrated close analogy to hypoxia-related genes, its activity is induced in response to it and inhibited following treatment with multiple RAF, MEK, and ERK inhibitors, validating its computationally inferred association with RAS activity. Strikingly, knockout of BMAL2 in human PDAC cells led to defects in viability and invasion in the setting of hypoxia. Metabolically, loss of BMAL2 impairs the ability to induce glycolysis - key metabolic pathway unregulated during hypoxia response - upon exposure to severe hypoxia. Surprisingly, knockout of BMAL2 in multiple PDAC cell lines led to a complete loss of HIF1A stabilization in response to hypoxia. By contrast, HIF2A was further upregulated under hypoxia in the setting BMAL2 loss. Additionally, metabolomic studies showed an overrepresentation of glycolysis metabolites and related metabolic pathways in hypoxic control conditions but to a lesser degree in BMAL2 knockout cells. Finally, preliminary in vivo experiments in cell line derived xenograft models displayed lower tumor growth rates in BMAL2 knock out cells. Together, our data indicates that BMAL2 loss blunts hypoxia response providing also a potential answer to the long-standing question of how these proteins are differentially regulated. Certainly, a connection between RAS signaling and glycolysis has long been apparent, but our findings provide a novel mechanistic understanding of the relationship between RAS and hypoxic metabolism.



 Alvaro Curiel Garcia, Carlo H. Maurer, Sam Holmstrom, Cristina Castillo, Pasquale Laise, Carmine F. Palermo, Steven A. Sastra, Anthony Andren, Li Zhang, Tessa Le Large, Irina Sagalovsky, Daniel R. Ross, Vilma Rosario, Kate Lu, Ethan Ferraiuolo, Nicholas Spinosa, Winston Wong, Kaitlin Shaw, John A. Chabot, Jeanine Genkinger, Hanina Hibshoosh, Gulam A. Manji, Alina C. Iuga, Roland M. Schmid Schmid, Michael A. Badgley, Kristen C. Johnson, Andrea Califano, Yatrik M. Shah, Costas A. Lyssiotis, Kenneth P. Olive. Ras-dependent activation of BMAL2 regulates hypoxic metabolism in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A047.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b200421d854e96ea48e923482c3749c3e10676" target='_blank'>
              Abstract A047: Ras-dependent activation of BMAL2 regulates hypoxic metabolism in pancreatic cancer
              </a>
            </td>
          <td>
            Á. C. García, Carlo Maurer, Sam Holmstrom, Cristina Castillo, Pasquale Laise, Carmine F. Palermo, Stephen A. Sastra, A. Andren, Li Zhang, Tessa Y. Le Large, Irina Sagalovsky, Daniel R. Ross, V. Rosario, K. Lu, Ethan Ferraiuolo, Nicholas Spinosa, W. Wong, K. Shaw, John A. Chabot, Jeanine Genkinger, H. Hibshoosh, G. Manji, Alina C Iuga, Roland M. Schmid Schmid, Michael A. Badgley, Kristen C. Johnson, Andrea Califano, Yatrik M. Shah, C. Lyssiotis, K. Olive
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c516594ae0175b41dbabb65d57d662cb8d6406" target='_blank'>
              Unveiling the role of the extracellular matrix in the osteosarcoma tumor microenvironment through integrated transcriptomics and experimental validation.
              </a>
            </td>
          <td>
            Yuyang Liu, Yuchen Han, Zixuan Guo, Yinglong Zhang, Xiuyuan Xu, Wenting Qi, Meng Xu, Jianxiong Li
          </td>
          <td>2025-10-07</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 PDAC is defined by a dense, immunosuppressive TME composed of ECM proteins, CAFs, and diverse immune cells. ChIP-seq on resected human PDAC samples identified MICAL2 as a super-enhancer-associated gene and its high expression was corelated with poor survival of patients. MICAL2, a flavin monooxygenase, promotes actin depolymerization and SRF transcription. This study investigates how tumor-intrinsic MICAL2 alters the PDAC TME. Orthotopic implantation of KPC-MICAL2 knockdown (M2KD) inducible cells led to reduced tumor growth. Immunofluorescence, flow cytometry, and atomic force microscopy showed M2KD tumors were more cellular, with marked reduction in PDPN+/α-SMA+ CAFs, collagen, and fibronectin, resulting in reduced stiffness. MICAL2 loss also reprogrammed CAF gene expression and increased infiltration of activated CD8+ T cells, skewing from an M2 to M1 macrophage phenotype. scRNA-seq revealed major shifts in immune populations, including increased cycling T and plasma cells. CD8+ T cell depletion reversed tumor suppression in both immune-hot and immune-cold syngeneic models, and adoptive transfer of MICAL2-KD tumor-infiltrating T cells suppressed the growth of control tumors. RNAScope showed reduced IL-1α, IL-6, TGF-β expression and decreased p38 and STAT3 phosphorylation in MICAL2-KD tumors. To test if MICAL2 regulates PDAC TME transcriptionally, we generated KPCM tumors and lines (by crossing PDX1-Cre; LSL-KRAS G12D/+; P53R172H/WT (KPC) with Mf/f mice) and reintroduced WT MICAL2 and MICAL2 lacking nuclear localization signal (DNLS) or FAD enzymatic domain in KPCM line. DNLS and FAD point mutant MICAL2 cells were deficient in tumor growth, had reduced stromal deposition and SRF activity, and increased CD8+ T cells. Anti-PD-1 significantly reduced M2KD tumor size and 50% of M2KD tumor-bearing mice had complete regression with anti-PD-1 and IL-1α antibodies. MICAL2-ASO treatment in human PDAC ex-vivo slice cultures increased T cell activation and M1 polarization. MICAL2 drives immunosuppression in PDAC and could be a potential therapeutic target.



 Bharti Garg, Evangeline Sari. Mose, Edgar Esparza, Jay Patel, Kevin Gulay Gulay, Sarah Sass, Alexei Martsinkovskiy, Asmina Courelli, Carrie Bishop, Gisselle Gonzalez, Adam Engler, Parag Katira, Vivien Ileana . Maltez, Herve Tiriac, Andrew M. Lowy. Actin cytoskeleton dynamics in tumor cells mediate Immune-suppressive microenvironment and sensitize pancreas tumors to PD-1 blockade therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a79cb674dc0d1eba8719f4642ee7a1f1f6ec638" target='_blank'>
              Abstract A109: Actin cytoskeleton dynamics in tumor cells mediate Immune-suppressive microenvironment and sensitize pancreas tumors to PD-1 blockade therapy
              </a>
            </td>
          <td>
            Bharti Garg, Evangeline S Mose, Edgar Esparza, Jay Patel, Kevin Gulay Gulay, Sarah Sass, A. Martsinkovskiy, Asmina Courelli, Carrie Bishop, Gisselle Gonzalez, Adam Engler, Parag Katira, Vivien Ileana . Maltez, Hervé Tiriac, A. Lowy
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>