<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-18 09:34:14 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76828c8834b0e2a09dc4aceaa26a03b54ca8e902" target='_blank'>
              Spatial tumour-immune ecosystems shape the efficacy of anti-PD1 immunotherapy in primary cutaneous melanoma
              </a>
            </td>
          <td>
            Félix Pham, Marion Dufeu, Valentin Benboubker, Maxime Grimont, Amélie Lhorisson, Justine Berthet, Marie Donzel, Raphaël Schneider, Laurie Tonon, A. Doffin, Félix Boivin, Simon Durand, Bertrand Dubois, Jonathan Lopez, Christophe Caux, J. Valladeau-Guilemond, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Early detection of melanoma through skin surveillance is critical for preventing metastatic progression. Primary cutaneous melanomas at early stage offer a unique opportunity to uncover fundamental mechanisms of tumor initiation, progression, and immune surveillance, but detailed spatial profiling of early disease remains limited. Here we integrate high-plex cyclic immunofluorescence (CyCIF) imaging, spatial transcriptomics, and conventional histology to identify factors associated with de-differentiation and dermal invasion in early-stage melanomas. We demonstrate a high level of variability from one primary cancer to the next, from one 100-300 cell microregion to the next within a single cancer, and from one cell to the next within a microregion. Intra-tumoral heterogeneity is influenced by local features of the microenvironment including proximity to T and myeloid cells and to perivascular environments. Thus, tumor plasticity and spatial heterogeneity arise early in melanoma development, potentially allowing for competition among multiple tumor states during the emergence of invasive disease. STATEMENT OF SIGNIFICANCE This study shows that tumor cell dedifferentiation and plasticity are locally patterned in primary human melanomas under the influence of microenvironment components such as myeloid cells and vasculature. As a result, neighborhoods in a single tumor can be as different from each other as in different tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bc89e4de14d8f34f2814e7cf735899a3c934813" target='_blank'>
              Spatial determinants of tumor cell dedifferentiation and plasticity in primary cutaneous melanoma
              </a>
            </td>
          <td>
            Tuulia Vallius, Yingxiao Shi, Edward Novikov, Shishir M Pant, R. Pelletier, Yu-An Chen, Juliann B. Tefft, Ajit Nirmal Johnson, Z. Maliga, Guihong Wan, George F. Murphy, S. Santagata, Yevgeniy R. Semenov, David Liu, C. G. Lian, P. Sorger
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a7baab96b7e0361b2e10254d2ab098fe969a9" target='_blank'>
              Tumor Cells Enriched for Interferon and Inflammatory Programs Pre-Exist in High Grade Serous Ovarian Cancer and are Proportionately Significantly Increased Post Chemotherapy
              </a>
            </td>
          <td>
            B. Winterhoff, Nomeda Girnius, Muyi Liu, Jaeyoung Kang, Mihir Shetty, M. A. Martinez-Gakidis, Weiyi Xu, Mayura Thomas, Linna Lahmadi, Aylin Z. Henstridge, Pilar Baldominos, Shobhana Talukdaer, Z. Chang, Jordan Mattson, Morgan Gruner, Sally A. Mullany, Peter Argenta, Gottfried E. Konecny, Dennis J. Slamon, Ronny Drapkin, Andrew C. Nelson, Timothy K. Starr, Joan S Brugge
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary While subcutaneous colorectal cancer (CRC) models are commonly employed in biological and preclinical studies, their cell morphological organization and molecular landscape are not yet fully understood. Using spatial transcriptomics, we identified two spatially distinct tumor regions demarcated by cancer-associated fibroblasts (CAFs), each region exhibiting unique molecular signatures and immune niche. Moreover, we uncovered specific cancer–stroma cell interactions that actively remodel the tumor microenvironment to facilitate immune evasion. These findings uncover the previously unrecognized spatial heterogeneity of the CRC mouse model and highlight the critical role of specialized tumor niches in driving tumor development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acb09b0c3fbf7dabf3a2ff1e9a531d5a3277e04" target='_blank'>
              Cancer-Associated Fibroblasts Establish Spatially Distinct Prognostic Niches in Subcutaneous Colorectal Cancer Mouse Model
              </a>
            </td>
          <td>
            Zhixian Lin, Jinmeng Wang, Yixin Ma, Yanan Zhu, Yuhan Li, Zhengtao Xiao, Wei Zhao
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6cfe1f46dc56e476a3d0be0d700257cc9ae2152" target='_blank'>
              Single-Cell and Spatial Profiling of Tumor Microenvironment Heterogeneity in Human Osteosarcomas
              </a>
            </td>
          <td>
            Xuejing zheng, Jin Yuan, Peng Jin, Xuejun Wang, Runlei Xu, Helin Feng, Xinxin Zhang, Yongjie Xie, Wence Wu
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Racial disparities in the clinical outcomes of triple-negative breast cancer (TNBC) have been well-documented, but the underlying biological mechanisms remain poorly understood. To investigate these disparities, we employed a multi-omic approach integrating imaging mass cytometry and spatial transcriptomics to characterize the tumor microenvironment (TME) in self-identified Black American (BA) and White American (WA) TNBC patients. Our analysis revealed that the TME in BA patients is marked by a network of endothelial cells, macrophages, and mesenchymal-like cells, which correlates with reduced patient survival. In contrast, the WA TNBC microenvironment is enriched in T-cells and neutrophils, indicative of T-cell exhaustion and suppressed immune responses. Ligand-receptor and pathway analyses further demonstrated that BA TNBC tumors exhibit a relatively “immune-cold” profile, while WA TNBC tumors display features of an “inflamed” TME, suggesting the evolution of a unique immunosuppressive mechanism. These findings provide insight into racially distinct tumor-promoting and immunosuppressive microenvironments, which may contribute to the observed differences in clinical outcomes among BA and WA TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2139c15b76d963542dd090ab4bb098c3eb9f6a" target='_blank'>
              Integrative spatial omics reveals distinct tumor-promoting multicellular niches and immunosuppressive mechanisms in Black American and White American patients with TNBC
              </a>
            </td>
          <td>
            Qian Zhu, Akhila Balasubramanian, Jaya Ruth Asirvatham, Megha Chatterjee, Badrajee Piyarathna, Jaspreet Kaur, Nada Mohamed, Ling Wu, Stacy Wang, Niloufar Pourfarrokh, P. D. Binsol, Mahak Bhargava, Uttam Rasaily, Yitian Xu, Junjun Zheng, D. Jebakumar, Arundhati Rao, Carolina Gutierrez, A. Omilian, Carl D. Morrison, Gokul M. Das, C. Ambrosone, Erin H. Seeley, Shu-hsia Chen, Yi Li, Eric Chang, Xiaoxian Li, Elizabeth Baker, Ritu Aneja, X. Zhang, Arun Sreekumar
          </td>
          <td>2025-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The tumor microenvironment is heterogeneous, structurally complex, and continually evolving, making it difficult to fully capture. Common dissociative techniques thoroughly characterize the heterogeneity of cellular populations but lack structural context. The recent boom in spatial analyses has exponentially accelerated our understanding of the structural complexity of these cellular populations. However, to understand the dynamics of cancer pathogenesis, we must assess this heterogeneity across space and time. In this review, we provide an overview of current dissociative, spatial, and temporal analysis strategies in addition to existing and prospective spatiotemporal techniques to illustrate how understanding the tumor microenvironment, focusing on dynamic immune-cancer cell interactions, across four dimensions will advance cancer research and its diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1f3339cce25c83c98dd45cea90c67e2a48bc606" target='_blank'>
              The tumor microenvironment across four dimensions: assessing space and time in cancer biology
              </a>
            </td>
          <td>
            Christina R. Larson, Ayushi Mandloi, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-06-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Transcriptome profiling of bladder cancer has revealed distinct basal-like and luminal-like molecular subtypes, which may be correlated with pathological subtypes of different patient outcomes. However, whether these molecular subtypes originate from the corresponding cell types in the normal urothelium and whether different cells of origin influence bladder cancer progression remain unclear. Here, we conducted cell-type-specific lineage tracing in CRISPR/Cas9-induced mouse bladder cancer models of Pten and Trp53 targeting. We show that although basal, intermediate, and superficial umbrella cells can all serve as the cell of origin for bladder cancer, transformed umbrella cells were gradually displaced by tumor cells from inner layers, particularly transformed basal cells, which had highest stemness. Histological and single cell RNA-sequencing data comparing basal- and intermediate-cell-induced bladder tumors revealed that basal-induced tumors displayed higher heterogeneity, and contained unique cell clusters including Krt14+Ki67+ highly proliferative basal cells, squamous cell carcinoma, and transitioning cells towards the Gata3+ luminal subtype. Trajectory analysis confirmed the cell lineage differentiation hierarchy uncovered in lineage tracing. Moreover, human bladder cancer molecular subtype signatures were highly enriched in mouse tumor cell clusters of the corresponding cell of origin, and a gene signature derived from the unique basal-induced clusters is predictive of worse patient outcome. Overall, our results support that the basal and luminal molecular subtypes of bladder cancer have the corresponding cells of origin as their basis, and that urothelial basal cells are intrinsically more competitive than intermediate and umbrella cells in generating aggressive bladder cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c74441168c668a9994b2523834450566ba1b30" target='_blank'>
              Intrinsic higher potency of basal urothelial cells to intermediate and umbrella cells as the cell of origin for bladder cancer
              </a>
            </td>
          <td>
            Chuan Yu, Nicholas Chu, Alexandra Aguirre, Jonathan Green, Qing Xie, Beatrice Knudsen, Zhu A. Wang
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b6a64a6d2ad28ecff7ab0f31663d8f3fd11b6a" target='_blank'>
              A spatial atlas of chemoradiation therapy in pancreatic cancer identifies cellular and microenvironmental determinants of persister populations
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, Ariana Acevedo-Diaz, K. Rajapakshe, Galia Jacobson, Daniela Tovar, Jimin Min, Guangsheng Pei, Candise Tat, Ayush Suresh, Ching-Wei Tzeng, Matthew HG Katz, M. Bhutani, Huaming Wang, Robert A. Wolff, C. Haymaker, E. Ludmir, Huocong Huang, Xiongfeng Chen, Liang Li, Albert C Koong, Linghua Wang, N. Navin, Dadi Jiang, Ziyi Li, Anirban Maitra, Eugene J. Koay
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe2aa8fe44537c3fe83f62a4d4acf8f8b14517" target='_blank'>
              STHELAR, a multi-tissue dataset linking spatial transcriptomics and histology for cell type annotation
              </a>
            </td>
          <td>
            Félicie Giraud-Sauveur, Quentin Blampey, Hakim Benkirane, A. Marinello, PH Cournède, Stergios Christodoulidis
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) has a poor survival rate due to late detection. PDAC arises from precursor microscopic lesions, termed pancreatic intraepithelial neoplasia (PanIN), that develop at least a decade before overt disease; this provides an opportunity to intercept PanIN-to-PDAC progression. However, immune interception strategies require full understanding of PanIN and PDAC cellular architecture. Surgical specimens containing PanIN and PDAC lesions from a unique cohort of 5 treatment-naive patients with PDAC were surveyed using spatial omics (proteomic and transcriptomic). Findings were corroborated by spatial proteomics of PanIN and PDAC from tamoxifen-inducible KPC mice. We uncovered the organization of lymphoid cells into tertiary lymphoid structures (TLSs) adjacent to PanIN lesions. These TLSs lacked CD21+CD23+ B cells compared with more mature TLSs near the PDAC border. PanINs harbored mostly CD4+ T cells, with fewer Tregs and exhausted T cells than PDAC. Peritumoral space was enriched with naive CD4+ and central memory T cells. These observations highlight the opportunity to modulate the immune microenvironment in PanINs before immune exclusion and immunosuppression emerge during progression into PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6b6b93e0faedffa7b598bebfd58275c143f057" target='_blank'>
              Spatial proteomics and transcriptomics reveal early immune cell organization in pancreatic intraepithelial neoplasia
              </a>
            </td>
          <td>
            Melissa R Lyman, Jacob T. Mitchell, Sidharth Raghavan, Luciane T. Kagohara, Amanda L. Huff, Saurav D Haldar, Sarah M. Shin, Samantha Guinn, Benjamin Barrett, Gabriella Longway, Alexei Hernandez, Erin M Coyne, Xuan Yuan, Lalitya Andaloori, Jiaying Lai, Yun Zhou Liu, Rachel Karchin, Anuj Gupta, A. Kiemen, André Forjaz, Denis Wirtz, Pei-Hsun Wu, Atul Deshpande, Jae W. Lee, T. Armstrong, Nilofer S. Azad, Jacquelyn W. Zimmerman, Laura D. Wood, Robert A Anders, Elizabeth D. Thompson, Elizabeth M. Jaffee, E. Fertig, W. Ho, N. Zaidi
          </td>
          <td>2025-06-26</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91860355badc3254098a406329f26559d939957e" target='_blank'>
              Optimised dissociation and multimodal profiling of prostate cancer stroma reveal fibromuscular cell heterogeneity with clinical correlates
              </a>
            </td>
          <td>
            E. Damisch, Elena Brunner, Lukas Nommensen, Lucy Neumann, Georgios Fotakis, Z. Trajanoski, S. Sopper, Georg Schäfer, M. Puhr, Isabel Heidegger, Marianna Kruithof-de Julio, Natalie Sampson
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47dca4509610b2f00715550942002b3b04fd28df" target='_blank'>
              Tracing colorectal malignancy transformation from cell to tissue scale
              </a>
            </td>
          <td>
            Helena L. Crowell, Irene Ruano, Zedong Hu, Yourae Hong, G. Caratù, Hubert Piessevaux, Ashley Heck, Rachel Liu, Max Walter, Megan Vandenberg, Kimberly Young, Dan McGuire, Evelyn Metzger, Margaret L Hoang, Joseph M. Beechem, Sabine Tejpar, A. Pascual-Reguant, Holger Heyn
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Xenium, a new spatial transcriptomics platform, enables subcellular-resolution profiling of complex tumor tissues. Despite the rich morphological information in histology images, extracting robust cell-level features and integrating them with spatial transcriptomics data remains a critical challenge. We introduce CellSymphony, a flexible multimodal framework that leverages foundation model-derived embeddings from both Xenium transcriptomic profiles and histology images at true single-cell resolution. By learning joint representations that fuse spatial gene expression with morphological context, CellSymphony achieves accurate cell type annotation and uncovers distinct microenvironmental niches across three cancer types. This work highlights the potential of foundation models and multimodal fusion for deciphering the physiological and phenotypic orchestration of cells within complex tissue ecosystems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0df7c5843203625e193cd9c5f8b65c48433cc0" target='_blank'>
              CellSymphony: Deciphering the molecular and phenotypic orchestration of cells with single-cell pathomics
              </a>
            </td>
          <td>
            Paul H. Acosta, Pingjun Chen, Simon P. Castillo, M. E. Salvatierra, Yinyin Yuan, Xiaoxi Pan
          </td>
          <td>2025-08-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="While pathology foundation models have transformed cancer image analysis, they often lack integration with molecular data at single-cell resolution, limiting their utility for precision oncology. Here, we present PAST, a pan-cancer single-cell foundation model trained on 20 million paired histopathology images and single-cell transcriptomes spanning multiple tumor types and tissue contexts. By jointly encoding cellular morphology and gene expression, PAST learns unified cross-modal representations that capture both spatial and molecular heterogeneity at the cellular level. This approach enables accurate prediction of single-cell gene expression, virtual molecular staining, and multimodal survival analysis directly from routine pathology slides. Across diverse cancers and downstream tasks, PAST consistently exceeds the performance of existing approaches, demonstrating robust generalizability and scalability. Our work establishes a new paradigm for pathology foundation models, providing a versatile tool for high-resolution spatial omics, mechanistic discovery, and precision cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9399a2bf1b84832c86d75a49ae3c9a7107309283" target='_blank'>
              PAST: A multimodal single-cell foundation model for histopathology and spatial transcriptomics in cancer
              </a>
            </td>
          <td>
            Changchun Yang, Haoyang Li, Yushuai Wu, Yilan Zhang, Yifeng Jiao, Yu Zhang, Rihan Huang, Yuan Cheng, Yuan Qi, Xin Guo, Xin Gao
          </td>
          <td>2025-07-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2fe9a625c4ba7b1dd23966948293739db34623" target='_blank'>
              In Situ Inference of Copy Number Variations in Image-Based Spatial Transcriptomics
              </a>
            </td>
          <td>
            Augusta Jensen, Helena L. Crowell, Anna Pascual Reguant, Irene Ruano, Sabine Tejpar, Holger Heyn, Mats Nilsson, Sergio Marco Salas
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary The tumor microenvironment (TME) is known to influence both disease progression and treatment outcomes in cancer. In this study, we analyzed high-dimensional imaging mass cytometry data from lung adenocarcinoma and triple-negative breast cancer to examine how the spatial positioning of immune cells relates to clinical outcomes. We developed a relative distance (RD) score that quantifies the proximity of immune cell pairs to cancer cells. This metric showed stronger associations with patient survival and treatment response than conventional features such as cell fraction or density. Our results suggest that spatial immune features may serve as robust biomarkers for patient stratification and personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1da098f6c41405a345327975831d3ae20b4e8e98" target='_blank'>
              Spatial Proximity of Immune Cell Pairs to Cancer Cells in the Tumor Microenvironment as Biomarkers for Patient Stratification
              </a>
            </td>
          <td>
            Jian-Rong Li, Xingxin Pan, Yupei Lin, Yanding Zhao, Yanhong Liu, Yong Li, Christopher I Amos, Chao Cheng
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bec286991f2affb3f9522a83843f9469ba7b89" target='_blank'>
              Developmental Plasticity and Stromal Co-option Shape a Pituitary Neuroendocrine Tumor Transcriptional Continuum
              </a>
            </td>
          <td>
            Robert C. Osorio, Jun Y. Oh, Jangham Jung, Alexander J. Ehrenberg, A. Saha, Meeki K. Lad, Harmon Khela, Nicole Brennick, P. Giannikopoulos, William W. Seeley, Lea T. Grinberg, Aaron Diaz, M. Aghi
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Cell-in-cell (CIC) structures, in which one cell is entirely engulfed by another, have been associated with poor outcomes in cancers. However, the mechanisms underlying this association remain poorly understood. We performed multiplex imaging of 56 cell identity, cell ‘state’ and cancer ‘hallmark’ proteins to characterise CICs, map their spatial interactions, and assess clinical associations across 444 tumour cores from 148 colorectal cancer patients, which contained over one million spatially resolved cells. We found that tumour regions containing CICs were associated with lower levels of cytotoxic T cells. We identified upregulated glucose metabolism as a consistent metabolic hallmark of CICs independent of cell type. Spatial analyses revealed that T cells adjacent to CICs underwent selective remodeling with distinct apoptotic and metabolic signatures. Finally, the presence of T cells within CIC neighbourhoods identified a subset of patients with improved survival. Our findings suggest that CICs may be a feature of metabolically competitive niches and a potential factor contributing to T-cell exclusion in tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bc73acfaddb58e731cdec2701f17feb2d562ab2" target='_blank'>
              Multiplex analysis of colorectal cancer tissue describes the composition, cell biology and spatial effects of cell-in-cell events and identifies a T cell-dependent prognostic signature
              </a>
            </td>
          <td>
            E. Bozkurt, Batuhan Kısakol, Mohammadreza Azimi, H. Dussmann, Sanghee Cho, E. McDonough, Joana Fay, Tony O’Grady, John P Burke, N. McCawley, Deborah A. McNamara, Daniel B. Longley, Fiona Ginty, Jochen H M Prehn
          </td>
          <td>2025-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Chronic respiratory diseases such as asthma and COPD involve interactions between multiple resident and immune cell types within bronchial airways, resulting in structural and functional changes. Thus cellular heterogeneity, arrangements and associated neighborhoods as well as interactions between cells and matrices represent intriguing yet challenging areas of study. Spatial phenotypic profiling facilitates exploration of these issues of the cellular microenvironment and identification of context-dependent cell-cell interactions. Utilizing spatial phenotyping, we interrogated the features and cellular landscape of lungs from non-asthmatics, asthmatics, and COPD in FFPE samples by developing a 10-plex antibody panel for the Akoya PhenoCycler®-Fusion system, focused on immune cells (CD45, CD3, CD4, CD8), proliferative cells (Ki67, PCNA), angiogenesis (CD34), epithelium (E-cadherin), smooth muscle (SMA) and extracellular matrix (collagen). We performed cell segmentation on multiplex immunofluorescence images and quantified marker intensity in each cell. Phenotypes were manually identified after normalization, integration, and clustering cells across samples. The composition, cell profiling, and distribution varied significantly between asthmatics and COPD compared to non-asthmatics emphasizing disease heterogeneity. Spatially agnostic analysis revealed that the matrix cluster was more abundant in COPD compared to non-asthmatics and asthmatics, consistent with a greater role for fibrosis. However, asthmatic patients had a higher proportion of unclassified and CD8 + clusters highlighting immune responses. Co-localization analysis showed near random distribution in non-asthmatics. But strong spatial interaction between T cells and other immune or matrix cells in asthma, and a higher avoidance of smooth muscle and immune cells, and of proliferative markers in both asthmatic and COPD. Niche analysis demonstrated different recurrent cell-cell interactions in asthmatic and COPD cohorts. In COPD, the matrix cell-enriched niche was more abundant, while in asthmatics, the unclassified cell-enriched niche was more prevalent compared to non-asthmatics. These findings provide insights into differential spatial organization of cells and tissues in asthma and COPD, with immune and epithelial mechanisms suggesting active inflammation and remodeling in asthma, but fibrotic processes in COPD, and potential role for vascular processes in both conditions. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12931-025-03315-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4fb506511da56b346daca646c35cd40c21b519d" target='_blank'>
              Spatial phenotyping of human bronchial airways in obstructive lung disease
              </a>
            </td>
          <td>
            L. Khalfaoui, R.M. Moore, J. Villasboas, K. R. Whitaker, B.C. Novotny, M. Thompson, C. Pabelick, Y. Prakash
          </td>
          <td>2025-07-02</td>
          <td>Respiratory Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="ABSTRACT Background Cutaneous squamous cell carcinoma (cSCC) and basal cell carcinoma (BCC) are the most prevalent types of nonmelanoma skin cancer (NMSC) and exhibit significant inter‐ and intra‐tumor heterogeneity. cSCC has a higher metastatic potential than BCC, accompanied by a considerable mortality rate. However, the detailed mechanisms of tumor evolution in cSCC have not yet been described. Methods We performed single‐cell RNA sequencing (scRNA‐seq) and T cell receptor (TCR) clonal analysis of skin biopsies from five BCCs, three squamous cell carcinomas in situ (SCCIS), and two invasive squamous cell carcinomas (SCC). Independent SCC specimens were used for spatial transcriptomic (ST) analysis using GeoMx Digital Spatial Profiler (DSP). Result Using scRNA‐seq, we analyzed a total of 117,663 cells. We distinguished cancer cells using copy number variation and identified SCC‐specific genes that potentially contribute to tumor progression. Analysis of tumor clones revealed SCC‐specific COL6A1+/ITGA5+ carcinoma cells which produce CXCL16. We also annotated CXCR6+ regulatory T cells (Tregs) which potentially move toward the tumor site by CXCL16, shaping the immunosuppressive TME. ST analysis supported these clones were located at the invasion site of SCC. Conclusion We suggest COL6A1 and ITGA5 promote the invasive and metastatic property of SCC. We also uncovered how SCC recruits Tregs via the CXCL16/CXCR6 axis to create a TME favorable for its survival. These molecules can be used as potential therapeutic targets for treatment of SCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10a0e1cd92069735154742961ad751b08397d911" target='_blank'>
              CXCL16 Producing Tumor Clones Are Shaping Immunosuppressive Microenvironment in Squamous Cell Carcinoma via CXCR6 Regulatory T Cell
              </a>
            </td>
          <td>
            H. Choi, Sunyoung Jung, Ki-Myo Kim, Mihyun Lee, Jun Ho Park, Sanha Hwang, Seung-Min Cha, Jade N Young, D. Poplausky, Hyunsung Nam, Nicholas Gulati, Chung-Gyu Park, Hyun Je Kim, Ji-Ung Park
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90f49812ba9bf54efb86922eb7c7255926faf91" target='_blank'>
              Mapping and reprogramming microenvironment-induced cell states in human disease using generative AI
              </a>
            </td>
          <td>
            Amir Akbarnejad, Lloyd Steele, D. Jafree, Sebastian Birk, M. Sallese, K. Rademaker, Adam Boxall, Ben Rumney, Catherine Tudor, Minal Patel, M. Prete, Stanislaw Makarchuk, Tong Li, Heather Stanley, A. Foster, Kenny Roberts, Andrew L. Trinh, C. E. Villa, G. Testa, S. Mahil, Arash Mehrjou, Catherine Smith, Sattar Vakili, M. Clatworthy, Thomas J. Mitchell, O. Bayraktar, M. Haniffa, M. Lotfollahi
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains among the most lethal cancers, with metastasis as the primary driver of mortality. While metastatic mechanisms are shared across malignancies, PDAC metastasis poses unique therapeutic challenges due to the presence of extensive tumor heterogeneity, desmoplasia, and immunosuppression — features that enable diverse migratory behaviors and therapeutic resistance. Recent advances have shown that metastatic progression in PDAC emerges from dynamic interactions between tumor cell–intrinsic and microenvironmental factors, each adapting to evolving stressors throughout the metastatic cascade. In the primary tumor, genomic instability and epigenetic reprogramming generate subclones with heightened invasive potential, while dense stromal reactions and myeloid-dominated immune suppression facilitate escape. During circulation, PDAC cells employ distinctive survival strategies through homotypic clustering and heterotypic interactions with blood components. At distant sites, PDAC cells adapt to organ-specific microenvironments through context-dependent metabolic and immune modulation, resulting in phenotypes that diverge from the primary tumor. In this Review, we examine how tumor-stroma crosstalk mechanisms shape metastatic progression in PDAC, provide a framework for understanding why conventional therapies often fail against metastatic disease, and highlight emerging opportunities for stage- and site-specific therapeutic interventions that target these unique adaptations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48f2d47124253bedbc482deeccc073d8dd0b354b" target='_blank'>
              Metastatic heterogeneity in pancreatic cancer: mechanisms and opportunities for targeted intervention
              </a>
            </td>
          <td>
            R. Maddipati
          </td>
          <td>2025-07-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Cancer therapy resistance (CTR) remains a significant challenge in oncology. Traditional methods like imaging, liquid biopsies and conventional omics analyses provide valuable insights, but lack the spatial resolution to fully characterise heterogeneity of tumour and the tumour microenvironment (TME). Recent advancements in spatial omics technologies offer unprecedented insights into the spatial organisation of tumours and TME. In this review, we summarise current methodologies for CTR research and highlight how spatial omics technologies and computational methods are revolutionising our understanding of CTR mechanisms. We also summarise recent studies leveraging spatial omics to uncover novel insights into CTR across various cancer types and therapies and discuss future opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baf8daf1a76c566edbbdf0a752cb4ca39d6eea13" target='_blank'>
              Cancer therapy resistance from a spatial‐omics perspective
              </a>
            </td>
          <td>
            Yinghao Zhang, Cheng Yang, Xi Chen, Liang Wu, Zhiyuan Yuan, Fan Zhang, Bin-Zhi Qian
          </td>
          <td>2025-07-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The organizational principles of nephronal segments are based on anatomical and physiological attributes that are linked to the homeostatic functions of the kidney. Recent molecular approaches have uncovered layers of deeper signatures and states in tubular cells that arise at various time points on the disease trajectory. Here, we introduce an analytical pipeline of multiplexed spatial protein imaging integrated with RNA expression to characterize proximal tubular subpopulations and neighborhoods in human kidney tissue. We demonstrate that, in reference tissue, a large proportion of S1 proximal tubular epithelial cells expresses thymus antigen 1 (THY1), a mesenchymal stromal and stem cell marker that regulates differentiation. Kidney disease is associated with loss of THY1 and transition toward expression of prominin 1 (PROM1), another stem cell marker recently linked to failed repair. Our data support a model in which the interplay between THY1 and PROM1 expression in proximal tubules associates with their regenerative potential and marks the timeline of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1923d134d598f27cb8990ba3881ccadfe262fa12" target='_blank'>
              Integration of spatial protein imaging and transcriptomics in the human kidney tracks the regenerative potential of proximal tubules
              </a>
            </td>
          <td>
            Mahla Asghari, Angela R. Sabo, Daria Barwinska, Ricardo Melo Ferreira, M. Ferkowicz, William S. Bowen, , D. Gisch, Connor J. Gulbronson, Carrie L. Phillips, Katherine J Kelly, Timothy A Sutton, James C Williams, Miguel Vazquez, John O’Toole, Paul M. Palevsky, Sylvia E. Rosas, S. Waikar, Krzysztof Kiryluk, C. Parikh, Jeffrey B. Hodgin, Pinaki Sarder, Ian H. de Boer, Jonathan Himmelfarb, Matthias Kretzler, Sanjay Jain, Michael T. Eadon, S. Winfree, Tarek M. El-Achkar, P. Dagher
          </td>
          <td>2025-08-15</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Background: Lung adenocarcinoma (LUAD) remains a leading cause of cancer-related mortality despite advances in treatments, necessitating more effective therapeutic strategies. Single-cell RNA sequencing (scRNA-seq) technology has revolutionized our ability to dissect the cellular complexity of cancers, which is often obscured in conventional bulk transcriptomic experiments. Methods: In this study, we performed an integrative analysis of scRNA-seq data from multiple LUAD patient cohorts to investigate cell-type-specific transcriptomic changes across disease stages. Clustering, lineage trajectory analysis, and transcriptional regulatory network reconstruction were employed to identify stage-specific gene markers and their upstream regulators. Additionally, we constructed intercellular communication networks to evaluate signaling changes within the tumor microenvironment (TME) during LUAD progression. Results: Our analysis revealed that epithelial cells from stage IV tumors exhibited a distinct transcriptional profile compared to earlier stages, a separation not observed in immune or stromal cell populations. We identified a panel of gene markers that differentiated epithelial cells across disease stages and effectively stratified patients into subgroups with distinct survival outcomes and TME compositions. Regulatory network analysis uncovered key transcription factors, including ATF3, ATF4, HSF1, KLF4, and NFIC, as potential upstream regulators of these stage-specific genes. Moreover, cell–cell communication analysis revealed a significant increase in signaling originating from epithelial cells and a concomitant decrease in immune-derived signals in late-stage LUAD. We identified several signaling pathways enriched in stage-specific crosstalk, including Wnt, PTN, and PDGF pathways, which may play critical roles in LUAD progression. Conclusions: This study provides a comprehensive single-cell resolution map of LUAD progression, highlighting epithelial-driven regulatory programs and dynamic intercellular communication within the TME. Our findings uncover novel molecular markers and regulatory mechanisms with potential prognostic and therapeutic value for more precise treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecbc245a8fbf12aaba1f14070430ba2f845cb23" target='_blank'>
              Single-Cell Transcriptomic Analysis Unveils Key Regulators and Signaling Pathways in Lung Adenocarcinoma Progression
              </a>
            </td>
          <td>
            Jialu Ma, Caleb McQuay, John Talburt, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2025-06-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Lymph node metastasis markedly worsens prognosis in lung adenocarcinoma (LUAD); however, the evolutionary dynamics and regulatory mechanisms underlying the heterogeneity of malignant epithelial cells during this process remain poorly understood and warrant comprehensive investigation. Methods We performed a comprehensive single-cell transcriptomic analysis of epithelial cells from 18 samples comprising normal lung tissue and lymph node metastases. Malignant epithelial cells were identified via inferred copy number variation (CNV) profiles. Key malignant subpopulations were further characterized through trajectory inference, cell–cell communication mapping, gene set variation analysis (GSVA), and reconstruction of transcription factor regulatory networks. To assess clinical relevance, we developed and validated a prognostic model—termed the EAS score—based on the transcriptional signatures of malignant epithelial subsets, using integrated data from multiple TCGA and GEO cohorts. The functional role of the hub gene SELENBP1 was experimentally validated through quantitative PCR (qPCR), Western blotting, immunohistochemistry (IHC), Transwell migration assays, colony formation assays, flow cytometry, ROS quantification, and subcutaneous tumorigenesis assays in vivo. Results Single-cell transcriptomic analysis identified four distinct malignant epithelial subtypes (Clusters 0–3), each characterized by unique patterns of CNV. Leveraging these defined cellular subpopulations, we constructed a highly accurate model for prognostication in LUAD, enabling reliable classification of patients based on clinical outcomes. Through detailed comparisons between groups with divergent prognostic risks, the study revealed notable differences across the tumor microenvironment (TME), including alterations in pathway activity, gene enrichment distributions, mutation profiles, and anticipated responses to immune checkpoint blockade. In addition, functional validation experiments confirmed that SELENBP1 plays a tumor-suppressive role, further supporting its relevance as a potential intervention target in LUAD. Conclusion This research provides insights into the evolutionary complexity and heterogeneity of malignant epithelial populations in lymph node metastatic sites of LUAD. It also presents a scoring system based on prognostic indicators, which serves as a reliable tool for forecasting patient survival outcomes. Moreover, the discovery of SELENBP1 as a candidate tumor suppressor emphasizes its importance in guiding both clinical risk categorization and the design of personalized treatment strategies for individuals classified as high-risk LUAD cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6a2796c042204c38d000db3a271f2db16f9cb8" target='_blank'>
              Single-cell transcriptomic profiling reveals the heterogeneity of epithelial cells in lung adenocarcinoma lymph node metastasis and develops a prognostic signature
              </a>
            </td>
          <td>
            Qiuqiao Mu, Han Zhang, Ying Shi, Mengli Xue, Jingxian Wang, Yun Ding, Lin Tan, Hui Yuan, Xin Li, Daqiang Sun
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cutaneous malignant melanoma is an extraordinarily aggressive and heterogeneous cancer that contains a small subpopulation of tumor stem cells (CSCs) responsible for tumor initiation, metastasis, and recurrence. Identification and characterization of CSCs in melanoma is challenging due to tumor heterogeneity and the lack of specific markers (CD271, ABCB5, ALDH, Nanog) and the ability of cells to dynamically change their phenotype. Phenotype-maintaining signaling pathways (Wnt/β-catenin, Notch, Hedgehog, HIF-1) promote self-renewal, treatment resistance, and epithelial–mesenchymal transitions. Tumor plasticity reflects the ability of differentiated cells to acquire stem-like traits and phenotypic flexibility under stress conditions. The interaction of CSCs with the tumor microenvironment accelerates disease progression: they induce the formation of cancer-associated fibroblasts (CAFs) and neo-angiogenesis, extracellular matrix remodeling, and recruitment of immunosuppressive cells, facilitating immune evasion. Emerging therapeutic strategies include immunotherapy (immune checkpoint inhibitors), epigenetic inhibitors, and nanotechnologies (targeted nanoparticles) for delivery of chemotherapeutic agents. Understanding the role of CSCs and tumor plasticity paves the way for more effective innovative therapies against melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624525e56a22d9cf1d979cf203e09ae4124349ea" target='_blank'>
              Cancer Stem Cells in Melanoma: Drivers of Tumor Plasticity and Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Adriana Sabău, A. Tinca, Raluca Niculescu, I. Cocuz, Andreea Raluca Cozac-Szöke, B. Lazar, D. Chiorean, C. Budin, O. S. Cotoi
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from archived, formalin-fixed, paraffin-embedded (FFPE) tissues from two patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed several oncogenic ligand-encoding genes (e.g. ALKAL2 and NRTN), which were predicted to communicate with neighboring cancer cells that expressed the corresponding receptors (e.g. ALK, RET). Several of these interactions were further validated experimentally and were shown to be clinically relevant. Collectively, our spatial analysis identifies multiple previously unrecognized signaling axes that may offer novel therapeutic options in neuroblastoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce7c8d7d28d1cfa3f77ec4f74044ee737c8a1da" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment in archived FFPE samples unveils novel paracrine interactions.
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlsson, A. Claeys, E. Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2025-08-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cce1fa72c7e2d6491d95989929c304afa244428" target='_blank'>
              Epithelial–mesenchymal cell state heterogeneity predetermines differential phospho-signaling responses to EGF stimulation
              </a>
            </td>
          <td>
            F. Kohane, Chantelle Johnstone, Daniel P Neumann, Ihuan Gunawan, Tim Huang, Fatemeh Vafaee, Christine L Chaffer, John G Lock
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) actively contribute to tumor development and treatment response. The interplay between tumor cells, immune cells, fibroblasts and blood vessels contribute to immune escape and drug resistance. Prior to treatment, higher tumor infiltrating lymphocytes correlate with better survival, while greater stromal content is linked to poor survival. Studying the composition and dynamics of the TME is essential for improving patient stratification, however a scalable tool for addressing this question is still lacking. Spatially resolved omics technologies allow for charting tissue architecture at the individual cell level, though large-scale studies remain challenging due to high expenses. In contrast, hematoxylin and eosin (H&E) slides are a cost-effective modality that provide rich morphological information for studying spatial biology. However, their use relies on pathologist interpretation. A key area of research in digital pathology has been automating cell (type) identification, predicting nuclei location and cell type in H&E slides. Existing deep learning models are limited by the quantity and diversity of training data, which requires pathologists to carefully annotate the location and identity of large volume of cells. To date, the largest dataset comprises approximately 200,000 cells, annotated with four cell types across 19 cancer types. We propose a novel approach of automated “molecular annotation”, where cell types and location on H&E slides are annotated with the aid of spatial proteomics, in place of pathologist annotation. Specifically, samples were profiled with both modalities. From the spatial omics modality, pixels were first segmented into cells, followed by cell clustering and cluster annotation based on molecular features of the cells. The location and identity of all cells on the tissue were then identified. These information were subsequently transferred to the H&E image by alignment at single-cell resolution, forming a dataset annotated with molecular ground truth. With a spatial omics dataset of two spatial proteomics slides from colorectal cancer patients, we obtained 160,000 annotated cells including 50,000 immune cells, 25,000 tumor cells, 12,600 stroma, 7,900 endothelial cells, among others. The size of this dataset is close to the largest annotated dataset publicly available to date. We then use this dataset to benchmark existing state-of-the-art deep-learning based cell type predictions models, as well as to fine-tune existing models for predicting cells in the colorectal tumor microenvironment. This proof-of-concept study aims to demonstrate the feasibility of molecular annotation approach. By including more spatial omics data, this approach can boost the performance of existing pre-trained models and enhance generalizability to specific tumor types. It opens up the opportunity to harness millions of cells for deep learning models to predict cell types on H&E slides, make AI models a cost-effective option for studying the TME.



 Siao-Han Wong, Benedikt Brors, Sonja Loges. Empowering AI-driven prediction of the tumor microenvironment from histopathology images via molecular annotation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B053.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62b30dcab5fc423419dfc7696f7849978493fc18" target='_blank'>
              Abstract B053: Empowering AI-driven prediction of the tumor microenvironment from histopathology images via molecular annotation
              </a>
            </td>
          <td>
            Siao-Han Wong, B. Brors, Sonja Loges
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Given the graft-versus-leukemia effect observed with allogeneic hematopoietic stem cell transplantation in refractory or relapsed acute myeloid leukemia (AML), immunotherapies have been explored in nontransplant settings. We applied a multiomic approach to examine bone marrow interactions in patients with AML treated with pembrolizumab and decitabine. Using extensively trained nuclear and membrane segmentation models, we achieved precise transcript assignment and deep learning–based image analysis. To address read-depth limitations, we integrated single-cell RNA sequencing with single-cell spatial transcriptomics from the same sample. Quantifying cell-cell distances at the edge level enabled more accurate tumor microenvironment analysis, revealing global and local immune cell enrichment near leukemia cells postpembrolizumab treatment, potentially linked to clinical response. Furthermore, ligand-receptor analysis indicated potential alterations in specific signaling pathways between leukemia and immune cells following immunotherapy treatment. These findings provide insights into immune interactions in AML and may inform therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0066efa7bc3f52352d23a57bb535219a1adb0cf1" target='_blank'>
              Single-cell spatial transcriptomics reveals immunotherapy-driven bone marrow niche remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R Herzog, Abigail Wong-Rolle, Laura W. Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann F Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine R Calvo, Christopher S. Hourigan, Kasper D Hansen, Chen Zhao
          </td>
          <td>2025-07-09</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third most common cancer in the world. Within the tumor microenvironment (TME), cancer cells are surrounded by diverse cell types including cancer-associated fibroblasts (CAFs) and immune cells. These neighboring cells can have a profound effect on prognosis and disease progression by providing local signals that impact tumor growth and metastasis. Prior work in the lab has identified via single cell RNA sequencing novel subtypes of cancer epithelial cells and enrichment of specific CAFs within the CRC TME. However, the locations and signaling interactions of these cells within the tumors remain poorly understood due to the loss of spatial information. In this study, we adopt a multi-facet approach to try to understand the CRC TME by utilizing a combination of single cell RNA sequencing and spatial omics technology. We first comprehensively characterized all the major cell types present in the CRC TME using single cell RNA sequencing, and with the knowledge gained, we set out to investigate the spatial localization of these cell types of interests and their interactions within the tumor architecture using the 10x Xenium Analyzer and merFISH. From our data, we identified subtypes of tumor epithelial cells that are uniquely enriched in CRC patients and observed distinct spatial organization of these cells within colonic glands across patient subgroups. In addition, we observed an enrichment of CAFs and immune cell populations at the tumor invasive margin, often accompanying regions of tumor budding. Further study of these tumor epithelial, CAF and immune subtypes and their spatial interactions may reveal new insights into CRC tumor biology and assist in the development of targeted therapies for CRC.



 Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, Nirmala Arul Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar. Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2896c139d399fbdf03bf5e11753408c9bfebdc83" target='_blank'>
              Abstract P19: Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics
              </a>
            </td>
          <td>
            Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, N. A. Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Individual cells within a given population exhibit striking variability in viral susceptibility, but it remains unknown whether this heterogeneity reflects memories encoded into the cellular lineage or true probabilistic variability. We used multi-color lineage tracing in a human primary organotypic skin model to reveal that viral resistance is encoded within specific cellular lineages. These lineages create distinct boundaries that block viral spread. Our lineage analyses in vitro confirmed that viral susceptibility exhibits strong heritability across cell generations, with siblings and cousins displaying remarkably similar infection outcomes. ATAC and proteomics profiling of resistant and susceptible clones revealed distinct epigenomic and proteomic states, with the transcription factor AP-1 emerging as a potential central regulator of lineage-encoded viral resistance. Inducing AP-1 activity with PMA rendered cells resistant to viral infection, suggesting a causative role in mediating resistance memory. Our findings demonstrate that antiviral resistance in human skin cells is encoded within cellular lineages and preserved through cell divisions, revealing how cell memory may shape infection dynamics and viral containment in tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb2bae8a5c0fba39a07b5a8201b6d06e217381d" target='_blank'>
              Lineage memory shapes viral resistance barriers in human skin
              </a>
            </td>
          <td>
            Laura C. Van Eyndhoven, Grant Kinsler, Jingchao Zhang, Aoife O’Farrell, Rim Abderrahim, Apurv Srivastav, Catherine G. Triandafillou, Todd M. Greco, Kenneth S. Zaret, I. Cristea, Megan H. Orzalli, N. Drayman, Abhyudai Singh, Arjun Raj
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Introduction Spatial transcriptomic analysis has proposed valuable insights into the behavior of tongue cancer. However, the specific cell types involved in chemically-induced carcinogenesis and the process of tumor development remain elusive. Methods We leveraged artificial intelligence (AI) algorithms and spatial transcriptomic sequencing to meticulously characterize the spatial and temporal evolution of 4-nitroquinoline-1-oxide (4NQO)-induced tongue carcinogenesis and intratumor heterogeneity. Results An AI classifier effectively categorized dysplastic tongue tissue into 13 distinct groups. Spatial transcriptomics identified 13 corresponding cellular subgroups with unique features within the lesion. Both methods successfully distinguished subtle muscle phenotype and genetic lineage variations induced by 4NQO, despite limited morphological differences. Evolutionary tree analysis revealed the dynamic appearance and disappearance of functionally and genetically diverse cell subgroups during the progression from epithelial dysplasia to in situ carcinoma and invasive cancer. Key findings include the identification of specific switch genes associated with tumor invasion and the revelation of significant intratumor heterogeneity. Discussion This spatial transcriptomic analysis of 4NQO-induced tongue cancer provides a detailed characterization of tumor evolution and heterogeneity. It elucidates critical aspects of tongue cancer cell behavior and identifies potential therapeutic targets (switch genes). These findings offer novel insights for improving the diagnosis and treatment of tongue cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679c71fb0ae1356ab18cbbcb3f0859a89370b05e" target='_blank'>
              Spatial transcriptomic analysis of 4NQO-induced tongue cancer revealed cellular lineage diversity and evolutionary trajectory
              </a>
            </td>
          <td>
            Feng Liu, Xiaojun Wu, Shaoqing Yu, Deqiang Cheng, Jun Pan, Xiaodong Wang, Guanzhen Yu, Chaofu Li
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d59997cf6c8e6c97ba758d271ce928cfa49100ba" target='_blank'>
              Transcriptomic Profiles from Normal and Tumor Tissue Samples Reveal Distinct Venule Populations and Novel Tumor Endothelial Cell Markers in Breast Cancer
              </a>
            </td>
          <td>
            K. Phoenix, Vijender Singh, Patrick A. Murphy, Kevin P. Claffey
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Integrating spatial transcriptomic data with immunofluorescence image data is challenging using existing tools due to their differences in spatial resolution. Immunofluorescence provides information about protein expression at the cellular or subcellular level, whereas spatial transcriptomic platforms typically rely on multicellular “spots” for RNA profiling. Our study coupled spatial transcriptomics of irradiated glioblastoma tissues with immunofluorescence for γH2AX, a marker of DNA damage within the nuclei of cells. We then compared gene expression in γH2AX-positive and negative regions within the tissue. There was significant interobserver variability in manual annotation of γH2AX positivity in multicellular spots by three different researchers (Kappa statistic = 0.345), despite all of them being familiar with γH2AX immunofluorescence and having predefined imaging parameters for annotation. This variability led to different researchers nominating different genes as being associated with DNA repair. To overcome this problem, we have developed a new tool using MATLAB. This tool performs “spot”-wise image analysis and uses researcher-defined parameters such as immunofluorescent marker intensity threshold and number of positive cells to annotate the “spots” as γH2AX positive or negative. The tissue with the most variability in manual annotation was annotated reproducibly by our MATLAB tool, leading to reproducible downstream analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36da1cf7d69ccada17068ecf953f32e30ee56bd" target='_blank'>
              A New Tool to Decrease Interobserver Variability in Biomarker Annotation in Solid Tumor Tissue for Spatial Transcriptomic Analysis
              </a>
            </td>
          <td>
            Sravya Palavalasa, Emily Baker, Jack Freeman, Aditri Gokul, Weihua Zhou, Dafydd Thomas, W. Al-Holou, Meredith A. Morgan, Theodore S. Lawrence, Daniel R Wahl
          </td>
          <td>2025-07-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Vestibular schwannoma (VS) is a benign tumor that can result in significant neurological and otological complications. The mechanisms underlying its development and spatial heterogeneity remain poorly understood. In this study, we analyzed single-cell RNA sequencing (scRNA-seq) data from three previously published vestibular schwannomas (VS1-3), along with spatial transcriptomics data from two additional specimens (VS_S1-2). Our results identified a VEGFA-enriched Schwann cell (SC) subtype in scRNA-seq data, which was validated by spatial transcriptomics. This subtype also exhibited a significant positive correlation with NOVhi SCs expression. These cells were centrally localized within tumor tissue. Furthermore, spatial analysis provided new insights into SC-stromal cell interactions, and we constructed a preliminary cellular atlas of VS tissues, enhancing our understanding of tumor growth dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a7b00297ec42d2e5217a894e786882ec23b539" target='_blank'>
              Integrating single-cell and spatial transcriptomics reveals the cellular heterogeneity of vestibular schwannoma
              </a>
            </td>
          <td>
            Wenqi Dong, Yuchen Jin, Lingkang Dong, Yumeng Jiang, Zhuangzhuang Li, Maoxiang Xu, Jingjing Wang, Feng Liu, Dongzhen Yu
          </td>
          <td>2025-07-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Lung adenocarcinoma (LUAD) exhibits significant cellular heterogeneity, yet the precise interactions between epithelial and stromal cells remain unclear. This study integrates single-cell and spatial transcriptomics to delineate tumor microenvironment dynamics, aiming to uncover key cellular subpopulations and their roles in LUAD progression. Methods We analyzed single-cell RNA sequencing (scRNA-seq) data from 21 LUAD patients and performed spatial transcriptomic deconvolution. Epithelial and fibroblast subpopulations were identified using Seurat and Harmony. Cell-cell communication was inferred via CellChat, while metabolic interactions were assessed using MEBOCOST. Copy number variation (CNV) analysis distinguished malignant cells, and trajectory inference mapped differentiation states. Spatial colocalization was examined via CellTrek. Prognostic signatures were derived from Cox regression, and a six-gene MCI score was validated using survival analysis. Results We identified eight epithelial (e.g., MUC21 + Epi, ASCL1 + Epi) and nine fibroblast subpopulations (e.g., Fb_IGFBP4, Fb_COL11A1), with tumor-enriched subsets showing elevated CNVs and metabolic crosstalk. Fb_IGFBP4 correlated with poor prognosis, while MUC21 + Epi exhibited amplified COL1A1/SDC4-mediated interactions with fibroblasts. Pathway analysis highlighted tumor-specific MK and collagen signaling between fibroblasts and epithelial cells, suggesting stromal-epithelial synergy drives progression. Spatial analysis revealed colocalization of epithelial and fibroblast subclusters in tumors, contrasting with normal tissue. The MCI score, derived from six genes (e.g., ADAM10, MARVELD1), independently predicted survival and stratified high-risk patients (AUC > 0.6). Conclusion This study identifies key stromal-epithelial subset interactions in LUAD, proposing prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06250-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e782303b020428f38f4b87de805bc2e91d348a09" target='_blank'>
              Decoding epithelial–fibroblast interactions in lung adenocarcinoma through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Jiajin Yang, Qiuping Xu, Yanjun Lu
          </td>
          <td>2025-07-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatoblastoma is the most common pediatric liver cancer, with the fastest rising incidence among childhood malignancies. Early genomic studies revealed that hepatoblastoma has the lowest mutational burden of any human cancer, however, recent advances in single-cell RNA-seq, multiomics, spatial transcriptomics, and functional genomics screenings have revealed substantial complexity. Diverse cellular subpopulations, divergent WNT signaling, key developmental pathways, and intricate interactions between the tumor cells and tumor immune microenvironment (TME) collectively shape tumor heterogeneity, disease progression, therapeutic responses, and genetic dependencies. Mosaic embryonic loss of heterozygosity (LOH) at chromosome 11p15.5 may be a hepatoblastoma-initiating event, as clonal expansion of 11p15.5 LOH occurs in adjacent normal liver tissue. A cholangiocyte-like subpopulation expresses FGF19, in a SOX4-dependent, paracrine manner, to drive the proliferation of neighboring embryonal hepatoblastoma cells. WNT-signaling dependent MDK promotes the immunosuppressive TME, which impairs immune cell infiltration. The TME may also be driven by islands of erythroblasts, which influence treatment resistance. Plasticity driven by changes in chromatin accessibility enables differentiation transition between hepatocytic and liver progenitor cell types, which is associated with treatment resistance. Here, we review recent findings in pediatric hepatoblastoma cells, tumor-associated cell types, and genetic dependencies that will serve to advance hepatoblastoma therapy. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361b9ebd3fa588c3f28e25ba36c91c45f4b07fb6" target='_blank'>
              From preneoplastic lesion to heterogenous tumor: recent insights into hepatoblastoma biology and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Yang, A. Davidoff, Andrew J. Murphy
          </td>
          <td>2025-07-19</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) plays a pivotal role in shaping therapeutic outcomes in lung cancer, with cancer-associated fibroblasts (CAFs) emerging as key mediators of drug resistance. In situ sequencing enables spatial resolution of tumor architecture, offering insight into heterogeneity and the influence of CAFs on tumor cell biology. However, directly translating these microenvironmental effects on tumor cell behavior into drug response predictions remains a major challenge. In this work, we leveraged spatial transcriptomics and in situ single-cell RNA sequencing to predict spatially resolved drug responses in non-small cell lung cancer (NSCLC) patient samples and investigated the impact of fibroblast density on therapeutic efficacy. We adapted our recently published scIDUC modeling approach—which integrates bulk RNA-seq from cancer cell lines with single-cell RNA-seq data to predict drug responses—for use with in situ data from eight NSCLC patients. Our spatial analysis revealed that tumor cells in high fibroblast-density regions exhibited distinct predicted drug sensitivities compared to those in low-fibroblast contexts. Notably, lapatinib—a dual EGFR/HER2 inhibitor—was consistently predicted to be less effective in tumor cells residing within fibroblast-rich niches. These predictions were validated experimentally using a co-culture model of CALU-3 lung cancer cells and IMR-90 fibroblasts. CALU-3 cells grown with fibroblasts, either in direct contact or separated by a transwell, showed significantly increased resistance to lapatinib compared to monoculture conditions, confirming our computational findings. Our study highlights the role of CAF-rich microenvironments in therapeutic response and proposes a CAF-informed framework for refining therapeutic strategies in NSCLC. By integrating spatial transcriptomics with predictive modeling, we provide a pathway to directly link microenvironmental context with functional drug response in tumors.



 Robert F. Gruener, Lilin Wang, Adam Lee, Weijie Zhang, R. Stephanie Huang. Spatially resolved drug response predictions reveal fibroblast-associated resistances in in situ lung cancer samples [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd4918af0e969655d588afdfb7b613e252a4d08" target='_blank'>
              Abstract A023: Spatially resolved drug response predictions reveal fibroblast-associated resistances in in situ lung cancer samples
              </a>
            </td>
          <td>
            Robert F. Gruener, Lilin Wang, Adam Lee, Weijie Zhang, R. S. Huang
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (cSCC), basal cell carcinoma (BCC), and melanoma, the three major types of skin cancer, account for over 70% of all cancer cases. Despite their prevalence, the skin cancer microenvironment remains poorly characterized, both in the outer skin layer where the cancer originates and at the deeper junctional and dermal layers into which it progresses. To address this, we integrated 12 complementary spatial single-cell technologies to construct orthogonally-validated cell signatures, spatial maps, and interactomes for cSCC, BCC, and melanoma. We comprehensively compared and integrated these spatial methods and provided practical guidelines on experimental design. Integrating four spatial transcriptomics platforms, we found keratinocyte cancer signatures, including six consistently validated gene markers. Spatial integration of transcriptomics, proteomics, and glycomics uncovered cancer communities enriched in melanocyte–fibroblast–T-cell colocalization with altered tyrosine and pyrimidine metabolism. Ligand-receptor analysis across >700 cell-type combinations and >1.5 million interactions highlighted key roles for CD44, integrins, and collagens, with CD44-FGF2 emerging as a potential therapeutic target. We consistently found differential interactions of melanocytes with fibroblasts and T-cells. We validated these interactions using Opal Polaris, RNAScope, and Proximal Ligation Assay. To integrate population-scale data, genetic association mapping in >500,000 individuals suggested SNPs enriched for spatial domains containing melanocytes, dysplastic keratinocytes, and fibroblasts, shedding light on functional mechanisms linking genetic heritability to cells within cancer tissue. This publicly available multiomics resource offers insights into the initiation and progression of the most lethal skin cancer (melanoma) and the most common forms (cSCC and BCC) and can be explored interactively at https://skincanceratlas.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87a9509055032cdb5b434f194c71651a4b05c7db" target='_blank'>
              Integrating 12 Spatial and Single Cell Technologies to Characterise Tumour Neighbourhoods and Cellular Interactions in three Skin Cancer Types
              </a>
            </td>
          <td>
            P. Prakrithi, Laura F Grice, Feng Zhang, Levi Hockey, Samuel X. Tan, Xiao Tan, Zherui Xiong, Onkar Mulay, Andrew Causer, Andrew Newman, Duy Pham, Guiyan Ni, Kelvin Tuong, Xinnan Jin, Eunju Kim, Minh Tran, Hani Vu, Nicholas M Muller, E. Killingbeck, M. Gregory, S. Teoh, Tuan Vo, Min Zhang, M. T. Landi, Kevin M. Brown, M. Iles, Zachary Reitz, K. Devitt, Liuliu Pan, A. Kulasinghe, Y. Kao, Michael Leon, Sarah Murphy, Hiromi Sato, J. G. Cruz, Snehlata Kumari, H. N. Luu, Sarah Warren, Chris L. D. McMillan, J. Henricson, Chris Anderson, David A Muller, Arun Everest-Dass, Blake O’Brien, M. Seviiri, Matthew H. Law, H. P. Soyer, Ian H. Frazer, Youngmi Kim, Mitchell S. Stark, Kiarash Khosrotehrani, Quan Nguyen
          </td>
          <td>2025-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) remains a lethal malignancy. Although immunochemotherapy regimens have improved patient survival rates, drug resistance still occurs in a significant subset of patients, highlighting the importance of elucidating the mechanisms within the tumor microenvironment. Here, we applied spatial single-cell transcriptomics to investigate the spatial characteristics of SCLC and their associations with immunochemotherapy resistance. By analyzing samples from 18 patients with extensive-stage SCLC, we identified two distinct epithelial cell subtypes: Epi-I and Epi-II. Epi-I exhibited high proliferative activity and was associated with treatment resistance and poor survival outcomes. In contrast, Epi-II showed more spatial contact with immune cells and was associated with treatment sensitivity. Further analysis uncovered a fascinating cellular transition paradigm, wherein Epi-I may be derived from Epi-II, with myeloid cells playing a facilitatory role in this transformation cascade. Specifically, within the spatial zone that was enriched with the Epi-II, epithelial cells may secrete MIF gene, which promoted the polarization of myeloid cells towards the M2 macrophages. The M2-polarized myeloid cells subsequently upregulated the expression of SPP1 that in turn triggered the activation of the PI3K-AKT signaling pathway in the adjacent epithelial cells, driving the conversion of Epi-II to Epi-I cells. Our findings revealed that the intricate crosstalk between epithelial and myeloid cells constitutes a pivotal resistance mechanism in SCLC, and targeting the SPP1/MIF pathway emerged as a promising strategy with the potential to enhance the treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0302ec6123f8cd6d62c8c99e2e99dcc7087718b" target='_blank'>
              Spatial transcriptomics reveals macrophage domestication by epithelial cells promotes immunotherapy resistance in small cell lung cancer
              </a>
            </td>
          <td>
            Yu Sun, Minghui Zhang, Yanbin Zhao, Yubo Yan, Lei Wang, Xuhui Liu, Songsong Xia, Bingbing Wang, Xiaoxin Zhang, Yan Wang
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Cholangiocarcinoma (CCA) is a highly heterogeneous malignancy, primarily comprising intrahepatic (iCCA) and extrahepatic (eCCA) subtypes. Reconciling the variability between iCCAs and eCCAs in clinical trials remains a challenge, largely due to the inadequate understanding of their shared and subtype-specific cellular heterogeneity. We aim to address this issue using single-cell and spatially resolved transcriptomic approaches. Methods We performed comprehensive single-cell RNA sequencing (scRNA-seq) by profiling 109,071 single cells from 28 samples, including chronic biliary inflammatory conditions (n = 7) and CCAs from different anatomical sites (n = 21). Findings were validated using external multi-omics datasets, tissue microarray cohort, spatial RNA in situ sequencing, CCA patient-derived organoids (PDOs), and mouse models. Results iCCAs and eCCAs exhibited distinct tumor ecosystems, with notable differences in cellular composition, diversity, and abundance across various cell types. Non-malignant epithelial cells displayed divergent precancer hallmarks from different biliary sites, with inflammatory extrahepatic bile ducts exhibiting early hijacking of the gastrointestinal metaplastic process. We identified seven meta-programs within cancer cells, mapped into four major subtypes. This subtyping was validated using external CCA cohorts and PDO models, distinguishing patients based on clinical outcomes and drug vulnerabilities. Specifically, iCCAs were associated with a senescent program, while eCCAs were enriched in an IFN-responsive program linked to adverse clinical outcomes and increased drug resistance. We identified a basal-like LY6D+ cancer cell subpopulation specific to eCCAs, which displayed significant stemness, drug resistance, and IFN-responsive features. This subpopulation was closely associated with an interferon-stimulated gene 15 (ISG15)-enriched mesenchymal and immune microenvironment. Functional assays demonstrated that ISG15 stimulation significantly boosted stemness, basal-like features, and drug resistance in eCCA cells, highlighting its pivotal role in sustaining the LY6D+ progenitor niches. Conclusion We present a comprehensive single-cell landscape of CCAs, uncovering the molecular heterogeneity between iCCA and eCCA subtypes. Transcriptomic subtyping of CCA cancer cells offers implications for clinical stratification and functional precision oncology. We identify basal-like epithelial progenitors and characterize their associated ISG15-enriched microenvironment in eCCAs. These findings hold significant promise for the development of novel prognostic biomarkers, therapeutic targets, and treatment strategies for CCAs. Supplementary Information The online version contains supplementary material available at 10.1186/s13045-025-01716-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fd09acd4eb4bccd6f0a7c5305db1cb0247e6ead" target='_blank'>
              Deciphering cholangiocarcinoma heterogeneity and specific progenitor cell niche of extrahepatic cholangiocarcinoma at single-cell resolution
              </a>
            </td>
          <td>
            Chunliang Liu, Xiang Wang, Erdong Liu, Yali Zong, Wenlong Yu, Youhai Jiang, Jianan Chen, Mingye Gu, Zhengyuan Meng, Jingfeng Li, Yang Liu, Yongjie Zhang, Jing Tang, Hongyang Wang, Jing Fu
          </td>
          <td>2025-06-23</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Background Neoadjuvant immune checkpoint blockade (nICB) has revolutionized cancer treatment, yet the underlying mechanisms of resistance in bladder cancer remain to be explored. Methods We conducted single-cell RNA sequencing (scRNA-seq) on peripheral blood mononuclear cells, tumor tissues, adjacent normal tissues, and metastatic lymph nodes from 2 nICB-naïve and 10 nICB-treated patients with bladder cancer (5 responders and 5 non-responders). Spatial RNA sequencing was performed on tumor slides from two responders and four non-responders. Findings were validated by multiplex immunohistochemistry, mice orthotopic bladder cancer model, and flow cytometry assays. Results nICB remodeled the tumor microenvironment of bladder cancer from both single-cell and spatial perspectives. scRNA-seq analysis revealed a significant increase in MYBL2hi cancer stem cells (CSCs) among non-responders. Analysis of the myeloid population showed that SPP1+ macrophages associated with angiogenesis were linked to CD8+ T cell exclusion. Further investigation into cell–cell communication revealed a propensity for bidirectional crosstalk between MYBL2hi CSCs and SPP1+ macrophages in non-responders. MYBL2hi CSCs derived CCL15, which bound to CCR1 and induced SPP1 upregulation in macrophages which reciprocally enhanced bladder cancer stemness and resistance to nICB through the SPP1-ITGα9β1 axis. Additionally, we identified an aged CCL3+ neutrophil population that interacted with SPP1+ macrophages through a positive feedback loop, contributing to nICB resistance. Finally, in vivo studies demonstrated that combined MYBL2 knockdown and SPP1 targeting synergistically enhanced ICB efficacy in bladder cancer. Conclusions Our research reveals transcriptomic characteristics associated with distinct therapeutic responses to nICB treatment, offering a foundation for optimizing personalized neoadjuvant strategies in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f561fa737d2808a2b787b34e9d1e658c2219648a" target='_blank'>
              Malevolent alliance of MYBL2hi cancer stem cell and SPP1+ macrophage confers resistance to neoadjuvant immunotherapy in bladder cancer
              </a>
            </td>
          <td>
            Hualin Chen, Zhaoheng Jin, Yueqiang Peng, Yingjie Li, Ziyi Li, Xuebin Zhang, Yi Xie, Jie Dong, Lin Ma, Zhigang Ji
          </td>
          <td>2025-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Accurately predicting which patients will respond to immune checkpoint blockade (ICB) remains a major challenge. Here, we present TIME_ACT, an unsupervised 66-gene transcriptomic signature of tumor immune activation derived from TCGA melanoma data. First, TIME_ACT scores accurately identify tumors with activated immune microenvironments across cancer types. Analysis of spatial features of the tumor microenvironment revealed that TIME_ACT-high regions exhibit dense lymphocyte infiltration near tumor cells, indicating localized immune activation. Second, in 15 anti-PD1 transcriptomic cohorts spanning six cancer types, TIME_ACT outperforms 22 established signatures and methods, achieving a mean AUC of 0.76 and a clinically meaningful mean odds ratio of 6.11. Thirdly, TIME_ACT scores can be accurately inferred from tumor histopathology slides. Finally, slide-inferred TIME_ACT scores predict ICB response across eight unseen cohorts, achieving a mean AUC of 0.72 and a mean odds ratio of 5.02. These findings establish TIME_ACT as a robust, pan-cancer, and low-cost predictor of ICB response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f083d96ace3eccacc20f8f51f00bb48faf2731" target='_blank'>
              Pan-cancer prediction of tumor immune activation and response to immune checkpoint blockade from tumor transcriptomics and histopathology
              </a>
            </td>
          <td>
            S. Mukherjee, Sumeet Patiyal, L.R. Pal, Tiangen Chang, Sumona Biswas, S. R. Dhruba, Amos Stemmer, Arashdeep Singh, Abbas Yousefi-Rad, Tien-Hua Chen, Binbin Wang, Denis Marino, Wonwoo Shon, Yuan Yuan, Mark Faries, Omid Hamid, Karen L. Reckamp, B. Waissengrin, Beatriz Ornelas, P. Chu, Salah Boudjadi, L. Ley, Dilara Akbulut, N. El Ahmar, S. Signoretti, D. Braun, Hyunjeong Joo, Hyungsoo Kim, A. Osipov, R. Figlin, J. Bar, Iris Barshack, Chi-Ping Day, Karine Sargsyan, A. Apolo, Kenneth D Aldape, Muh-Hwa Yang, Michael B. Atkins, Ze’ev A. Ronai, Danh-Tai Hoang, E. Ruppin
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Cancer progression remains a significant clinical challenge. Phenotypic adaptation by tumour cells results in disease heterogeneity, which drives treatment resistance and immune escape. T-cell immunotherapy, while effective at treating some cancer subtypes, can also fail due to limits on tumour immunogenicity or T-cell recognition. For example, one potential contributor to immune escape involves the density and alignment of the extracellular matrix (ECM) surrounding tumours, also known as tumour-associated collagen signature (TACS). However, the specific mechanisms by which aligned fibres contribute to decreased patient survival rates have not yet been decoupled. Here, we developed EVO-ACT (EVOlutionary agent-based cancer T-cell interaction), a two-dimensional agent-based modelling framework designed to investigate how different TACS architectures impact tumour evolution and dynamic interactions with CD8 ⁣+ T cells. Our results highlight that TACS-driven modulation of T-cell dynamics, combined with phenotypic adaptation, such as epithelial-to-mesenchymal transition, underlies differences in tumour immunogenicity and the application of our model can successfully recapitulate clinically observed breast cancer survival trends.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195f55d30b50b8b9b55b6c384b5267c2b8395881" target='_blank'>
              Distinct evolutionary patterns of tumour-immune escape and elimination determined by extracellular matrix architectures
              </a>
            </td>
          <td>
            Yijia Fan, Jason T. George
          </td>
          <td>2025-07-01</td>
          <td>Journal of the Royal Society Interface</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is characterized by high incidence and recurrence rates, presenting as an immune ‘cold’ tumor that exhibits a poor response to immunotherapy. The mechanisms underlying immune suppression and evasion within the tumor microenvironment (TME) of PCa remain inadequately understood. Methods A comprehensive analysis of the immune environment in PCa was conducted using combined single-cell and spatial transcriptomic approaches, encompassing samples from healthy tissue, adjacent normal tissue, and localized tumors. Cell abundance and polarization state analyses were performed to identify pivotal cellular populations. Spatial deconvolution techniques were employed to elucidate cell composition within its spatial context. Additionally, cell niche and spatial colocalization analyses were conducted to evaluate potential cellular interactions. Immune response enrichment analysis was utilized to assess cellular response states. In vivo and in vitro experiments were conducted to validate hypotheses. Results Data indicated a prevalent immunosuppressive state among CD8 T cells, accompanied by variations in cell abundance. Macrophages emerged as key regulators in recruiting CD8+ effector T cells and regulatory T cells (Tregs) into the TME, mediated by the CXCL12/CXCR4 axis. A spatial proximity relationship was established between CD8+ effector T cells and Tregs, suggesting Tregs directly influence CD8+ T cell function. Immune cell state analysis revealed interleukin-2 (IL-2) as a critical cytokine in reshaping the immune microenvironment, with Tregs competitively depleting IL-2 and mediating IL-2/STAT5 signaling to induce CD8+ effector T cell exhaustion. Treatment with CXCR4 inhibitor and IL-2 demonstrated significant antitumor effects and reversed immune dysfunction in both in vivo and in vitro experiments, with combined treatment exhibiting superior efficacy. Conclusion These findings elucidate the role of macrophages in mediating the CXCL12/CXCR4 axis to aggregate CD8+ effector T cells and Tregs, thereby influencing the TME. Furthermore, Tregs competitively deplete IL-2 and mediate IL-2/STAT5 signaling, leading to CD8+ effector T cells exhaustion and the establishment of an immunosuppressive microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363b3add731d9a14c149f45989beacb3b8971f64" target='_blank'>
              CXCL12/CXCR4 axis governs Treg spatial dominance over CD8+ T cells via IL-2 sequestration: a dual therapeutic target in prostate cancer
              </a>
            </td>
          <td>
            Junyi Li, Long Zhang, Ruoyang Liu, Changwen Xu, Huihui Tang, Yunfei Zou, Qingfei Cao, Weichao Huang
          </td>
          <td>2025-07-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Neoadjuvant therapy (NAT) has transformed cancer treatment by improving surgical outcomes and survival rates, yet resistance mechanisms across multiple cancer types remain unclear. This study aimed to decipher tumor ecosystem dynamics during NAT using cross-cancer single-cell sequencing data, focusing on identifying key mediators of immunosuppression and treatment resistance. Methods Single-cell RNA-sequencing (scRNA-seq) datasets from five solid tumors (esophageal squamous cell carcinoma, esophagogastric junction carcinoma, colorectal cancer, cervical cancer, and triple-negative breast cancer) were integrated. The data from these five cancer types underwent a rigorous process to standardize cell types across all datasets. Cell-cell communication analysis, Meta-Programs (MPs) via non-negative matrix factorization, and functional enrichment were performed. Immunohistochemistry (IHC) and Western blot validated S100A4 expression and PD-L1 induction in vitro. Results We constructed a single-cell map across cancer types and systematically characterized dynamic changes in tumor cells and diverse microenvironmental cell populations following neoadjuvant therapy, along with thier gene expression and pathway alterations. Our findings highlight that crosstalk between cancer-associated fibroblasts (CAFs) and tumor cells represents a critical determinant of neoadjuvant therapy resistance. Fibroblasts underwent significant state transitions post-treatment, marked by hypoxia-associated gene upregulation (e.g., S100A4) and immunosuppressive pathways. Meta-Programs (MPs) analysis identified a hypoxia-driven fibroblast state (MP5) containing S100A4 that correlated with treatment resistance. In vitro experiments, S100A4 co-localized with α-SMA + fibroblasts and directly induced PD-L1 expression in tumor cells, linking CAFs secreted S100A4 to immunosuppressive PD-L1 upregulation. Conclusion This cross-cancer single-cell atlas reveals S100A4, secreted by CAFs, as a conserved mediator of PD-L1 upregulation in tumor cells, driving immunosuppression and resistance to nICT. The atlas and mechanistic findings provide a rationale for targeting S100A4 to enhance treatment efficacy, pending validation in larger cohorts and mechanistic studies. This resource also supports the development of personalized, cross-cancer neoadjuvant strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/875f07738539c0aaf3118d5091d4cb483e59f11d" target='_blank'>
              Deciphering the tumor ecosystem dynamics undergoing immunochemotherapy therapy across multiple cancer types unveils the immunosuppressive role of S100A4 in fibroblasts by promoting PD-L1 expression in tumor cells
              </a>
            </td>
          <td>
            Bo Yang, Ruiji Chen, Mali Zu, Jie Yao, Hong Ren, Yi‐Shung Lin, Bo Zhang, Tianjiao Ji, Yang Liu
          </td>
          <td>2025-07-23</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metabolic crosstalk among diverse cellular populations contributes to shaping a competitive and symbiotic tumor microenvironment (TME) to influence cancer progression and immune responses, highlighting vulnerabilities that can be exploited for cancer therapy. Using a spatial multiomics platform to study the cell-specific metabolic spectrum in hepatocellular carcinoma (HCC), we map the metabolic interactions between different cells in the HCC TME and identify a unique tumor-immune-cancer-associated fibroblast (CAF) "interface" zone, where cell-cell interactions are enhanced and accompanied by significant upregulation of lactic acid and long-chain polyunsaturated fatty acids. Further combining single-cell mass spectrometry imaging of patient-derived tumor organoids, cocultured CAFs, and macrophages, we demonstrate that CAFs increase glycolysis and secrete lactic acid to the surrounding microenvironment to drive immunosuppressive macrophage M2 polarization. These findings facilitate the understanding of cancer-associated metabolic interactions in complex TME and provide clues for targeted clinical therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b292d4bb0787fa6280b9910112e99765dee292d" target='_blank'>
              Integrating spatial omics and single-cell mass spectrometry imaging reveals tumor-host metabolic interplay in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Panpan Chen, Haoyuan Geng, Bangzhen Ma, Yaqi Zhang, Zihan Zhu, Min Li, Shiping Chen, Xiao Wang, Chenglong Sun
          </td>
          <td>2025-07-29</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer remains a formidable global health burden owing to its persistently high incidence and mortality, highlighting the need to elucidate its underlying biological mechanisms. Such insights are crucial for refining diagnostic precision and therapeutic efficacy. Notably, human tumors represent highly heterogeneous ecosystems, comprising a dynamic interplay between malignant cells and non-malignant components, such as immune infiltrates and stromal elements. This cellular heterogeneity constitutes a major obstacle in tumor research. Recent technological advancements have enabled the development and application of novel tools for investigating tumor heterogeneity. In this context, single-cell sequencing technologies emerged as transformative tools for dissecting tumor architecture at a cellular resolution, offering unprecedented insights into the cellular diversity and molecular underpinnings of cancer. In particular, single-cell RNA sequencing (scRNA-seq) has been widely used to analyze the tumor microenvironment, particularly its immune components, thereby providing a valuable framework for understanding tumor-immune system interactions. However, the intrinsic heterogeneity of tumor cells has received comparatively less attention, with limited studies focusing on these malignant populations. Comprehensive profiling of tumor cells has become increasingly feasible as scRNA-seq technologies continue to evolve, offering higher throughput, increased cell capture efficiency, and more advanced analytical pipelines. Despite these advancements, current applications of scRNA-seq remain primarily focused on immune and stromal cell populations. Therefore, a paradigm shift is warranted, redirecting the investigative focus toward tumor cells and gaining deeper insights into tumorigenesis and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c3598e7911b47f7291bb4a9f0c4b83bd05a9bc" target='_blank'>
              Exploration and Application of Malignant Cell Heterogeneity Analysis with Single-Cell Transcriptome Sequencing Technology.
              </a>
            </td>
          <td>
            Wen Zhang, Tianliang Liu, Ting Liu, Yu Zhang, Zhen-guo Zhao, Xinxin Zhang, Xiaoyang Li, Jiasheng Zhong, Zhicheng Li, Shifu Chen, Libin Xu
          </td>
          <td>2025-07-25</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) exhibits substantial intertumoral heterogeneity, largely attributable to multiple tumor stem‐like cell populations, whose molecular identities and clinical significance remain incompletely defined. This study delineates tumor‐intrinsic stem‐like cell diversity and its prognostic implications through single‐cell transcriptomic profiling of 171,906 tumor epithelial cells (n = 152), integrated with bulk transcriptomic (n = 1389) and genomic (n = 1077) datasets. Functional validation was conducted via in vitro assays and multiplex immunofluorescence. A previously unrecognized lysosome‐associated transmembrane protein 4B‐positive (LAPTM4B+) stem‐like cell cluster was identified, distinct from the classical leucine‐rich repeat‐containing G‐protein coupled receptor 5‐positive (LGR5+) population. LAPTM4B+ cells exhibited MYC pathway activation and 8q chromosomal gains, with preferential enrichment in microsatellite‐stable, POLE wild‐type, and left‐sided tumors. Stratification based on LAPTM4B+/LGR5+ stem‐like cell ratios defined four CRC stem‐like subtypes (CSS), with CSS2 (LAPTM4B+‐dominant) associated with the poorest prognosis (HR = 2.31, p < 0.001). The combined expression of LAPTM4B and LGR5 demonstrated superior predictive power for CRC progression compared to either marker alone (AUC = 0.820 vs. 0.715/0.699), underscoring the synergistic influence of distinct stem‐like cell populations on patient outcomes. These findings provide novel insights into CRC heterogeneity and cooperative interactions among diverse stem‐like populations shaping disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/870d87b3057664fbe8a51f6ca3867c7fe15a2488" target='_blank'>
              Novel Lysosomal‐Associated Transmembrane Protein 4B‐Positive Stem‐Like Cell Subpopulation Characterizes High‐Risk Colorectal Cancer Subtypes
              </a>
            </td>
          <td>
            Yangyang Fang, Tian-Hao Fu, Ziqing Xiong, Qian Zhang, Wei Liu, Kuai Yu, Aiping Le
          </td>
          <td>2025-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most aggressive brain tumor in adults, is characterized by its infiltrative growth along the perivascular space. Mural cells (MCs), encompassing pericytes and smooth muscle cells, are multifunctional perivascular cells implicated in GBM progression. MCs not only facilitate vascular co-option but have also been suggested to contribute to the immunosuppressive tumor microenvironment, promoting tumor growth and migration. However, whether MC interactions with immune cells differ based on their proximity to the tumor remains unclear. Using single-cell RNA sequencing, we analyzed MC transcriptome profiles across distinct regions relative to the tumor mass in mouse and human GBM samples. Tumor-residing MCs exhibited profound phenotypic changes, showing upregulated gene expression and enhanced signaling activity toward immune cells, with region-specific ligand-receptor interactions. Conversely, border-residing MCs, despite their abundance, showed reduced activation and lacked distinct transcriptional profiles. These findings reveal spatially defined transcriptional heterogeneity in MCs within the GBM microenvironment, underscoring their dynamic role in the GBM microenvironment. This study provides novel insights into MC responses in GBM, identifying potential avenues for targeting MC-immune-cell interactions in therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34269e7ccaff0901b281158ca70f685dbde49243" target='_blank'>
              Pericytes change function depending on glioblastoma vicinity: emphasis on immune regulation.
              </a>
            </td>
          <td>
            C. Buizza, Robert Carlsson, Coralie Gamper, Gayatri Chitale, J. Bengzon, Gesine Paul
          </td>
          <td>2025-07-17</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d1b8aeaba5a454b04d6690efe6420aea37c43e" target='_blank'>
              Stochastic modelling of prostate progenitor architecture
              </a>
            </td>
          <td>
            Christo Morison, Esther Baena, Weini Huang
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The effectiveness of colorectal cancer (CRC) therapy is limited owing to the absence of treatments targeting drug-tolerant residual cancer cells. Although neoadjuvant therapy is effective, pathological examination of residual tumors has revealed the presence of small clusters of LGR5-positive cancer cells in the fibrous tissue. Here, we established a colorectal cancer cell line-derived organoid (CCD-organoid) regrowth model using a patient-derived cell line with cancer stem cell properties and demonstrated that it displayed the morphological characteristics of small clusters in clinical tissues. Time course analysis of single-cell RNA sequencing of the CCD-organoid regrowth model revealed various states and dynamic alterations within non-cycling cells. We identified subpopulations highly expressing protein translation-related genes RPL17 and EEF1G. To identify key signals for the transition of residual cancer cells to regrowth, we evaluated inhibitors targeting pathways such as the Wnt pathway, reactive oxygen species pathway, and RNA polymerase I pathway, highlighted in the single-cell RNA sequencing analysis. Only the polymerase I-inhibitor BMH-21 significantly reduced tumor growth both in vitro and in vivo, indicating the critical cell subpopulation driving recurrence. Our results demonstrate the possibility of a unique therapeutic target for CRC treatment targeting drug-tolerant residual cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa9b58cf763b8a32b666aa1599214c1a009f1d2" target='_blank'>
              Colorectal cancer cell line-derived organoid model with stem cell properties captures the regrowing state of residual cancer cells after neoadjuvant chemotherapy
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, T. Fujita, Atsuhiko Kato, Y. Zaitsu, Tomoko Jogo, Chie Kato, Takeshi Watanabe, Eri Hashimoto, Chiyoko Nishime, E. Fujii, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, H. Saeki, Hiroyuki Aburatani, Yoshihiko Maehara, Tatsumi Yamazaki
          </td>
          <td>2025-06-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Globally, gastric cancer (GC) stands as the fifth most prevalent form of malignant neoplasm and represents a significant contributor to mortality associated with oncological conditions. Despite advancements in therapeutic strategies for GC, the outcomes for patients with advanced stages of the disease continue to be unfavorable, largely due to tumor heterogeneity and the challenges posed by resistance to therapeutic agents. Metabolic reprogramming is pivotal in driving the advancement of GC, contributing to the development of resistance to pharmacological treatments and facilitating the cancer’s ability to evade immune surveillance. Developing multi-target comprehensive treatment strategies by integrating tumor microenvironment (TME) modulation holds promise for significantly improving therapeutic efficacy. Methods The study analyzed GC and identified key cell subtypes by integrating data derived from single-cell RNA-sequencing (scRNA-seq) alongside spatial transcriptomics information. Cell type identification was accomplished using the tool of Seurat, and the spatial distribution of cell types was revealed through the Robust Cell Type Decomposition technique. CellChat was used to analyze the interactions between key cell subtypes and other cells, and the “StLearn” package was employed to investigate spatial cell communication in depth. Additionally, the functional role of the key molecule ELK4 was validated through in vitro experiments. Results This research utilized scRNA-seq combined with spatial transcriptomics to comprehensively analyze GC, identifying the C1 NDUFAB1+ subtype, which exhibited high proliferative activity, metabolic reprogramming capabilities, and immune evasion properties. It was found that the C1 NDUFAB1+ subtype closely interacted with fibroblasts and pericytes via the PARs signaling pathway. Additionally, in vitro experiments confirmed that knockdown of ELK4 substantially curbed tumor cell proliferation, migration, and invasion. Conclusion This study revealed the main significance of the C1 NDUFAB1+ subtype in GC, elucidating its core mechanisms in tumor progression, metabolic reprogramming, and immune evasion. ELK4 was identified as a key regulatory factor that markedly enhanced the proliferation, migratory capacity, and invasive potential of tumor cells, while changes in the TME were a driving force behind immune suppression and drug resistance. The findings underscored the importance of developing specific therapeutic targets, targeting metabolic reprogramming, and overcoming immune evasion, providing new theoretical foundations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94360dc80f5310630bd93066b812e27d59cfa9fb" target='_blank'>
              Integrative single-cell and spatial transcriptomics uncover ELK4-mediated mechanisms in NDUFAB1+ tumor cells driving gastric cancer progression, metabolic reprogramming, and immune evasion
              </a>
            </td>
          <td>
            Yuwei Sun, Wenyang Nie, Zhikai Xiahou, Xiaojing Wang, Wenjia Liu, Zongkai Liu, Zhiheng Lin, Zhaidong Liu
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Osteosarcoma (OS) is the most common primary malignant bone tumour of childhood, yet five-year survival has plateaued at ~60–70% for localised disease and plunges below 30% once metastasis emerges. Formerly viewed as a cell-intrinsic neoplasm entombed in mineralised bone, OS is now understood as a spatially stratified ecosystem whose immune-evasion niches choreograph progression. Three-dimensional spatial transcriptomics (3-D ST) fuses barcode-based transcript capture with volumetric reconstruction, preserving x-, y- and z-axis context and exposing concentric C1QC necrotic belts, MCAM (melanoma cell-adhesion molecule, CD146) peri-vascular corridors, hypoxic glycolytic rims and therapy-induced tertiary-lymphoid islets that collectively sequester cytotoxic lymphocytes. Pre-clinical atlases link PD-L1 high SOX9 stem-like cells, LGALS3 macrophages and VEGFA-driven endothelial tips to chemoresistance and immune-checkpoint failure, while ligand–receptor inference highlights VEGFA–VEGFR2, CXCL12–CXCR4 and complement–CSF1R axes as actionable bottlenecks. Translational efforts already echo these insights: dual MCAM/VEGFR blockade collapses vascular gates, C1s or CSF1R antagonists dismantle necrotic-core “cold pockets”, and MCT1–POSTN combinations target lactate-stiffened stromal shells. By weaving methodological advances with emergent biology, this review crystallises how 3-D ST redefines OS immunopathology, sharpens biomarker discovery and accelerates spatially guided combination therapies. We aim to expose diagnostic blind spots, spotlight niche-directed interventions and chart a roadmap toward lifting the long-standing therapeutic ceiling in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c84eb34c63a5fd31899b587fca53cec9b872be8" target='_blank'>
              Deciphering spatially confined immune evasion niches in osteosarcoma with 3-D spatial transcriptomics: a literature review
              </a>
            </td>
          <td>
            Guangqiang Qiu, Yongcheng Tang, Junhui Zuo, Heng Wu, Yongxian Wan
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Triple-negative breast cancer is a highly aggressive disease subtype with limited treatment options. Despite the approval of targeted therapies such as PARP inhibitors, metastatic progression remains inevitable. Additionally, spatial arrangements, including immune exclusion zones and immune–stromal crosstalk, can profoundly influence therapeutic response. These observations underscore the need for more effective, individualized treatments that both target malignant cells and promote anti-tumor microenvironments. Toward that goal, we have developed a coordinated experimental-computational approach designed to enable identification of effective treatment strategies that consider the functional relationships between different cell types. First, to identify adaptive responses that may serve as therapeutic vulnerabilities, we treated C3TAg genetically engineered mouse model with a panel of clinically approved therapies then subjected these samples to single-cell RNAseq and multiplex tissue imaging. We observed that relatively short treatment durations lead to changes in the composition and spatial organization of tumors, suggesting adaptive responses that may be leveraged in combination treatment strategies. Notably, we observed changes in several macrophage populations, including changes to co-localization of M2-like macrophage and cancer-associated fibroblasts, highlighting stromal responses to therapeutic pressure. Second, we are using the resultant data to guide development and parameterization of an agent-based model that considers the functional relationships between different tumor components and can be used to predict effective therapeutic treatment strategies. We have developed analytical approaches to leverage our single-cell RNAseq data to estimate key model parameters operable in prioritized cell populations, including those related to proliferation, motility, antigen presentation, and cell-cell signaling. Initial model simulations suggest that spatial cell patterns are often not steady: immune cells dynamically aggregate, disperse, migrate, and reaggregate at new locations; macrophages infiltrated hypoxic and necrotic regions, where they polarized from an anti-tumor M1-like state to a pro-tumor M2-like state. These observations lay the groundwork for leveraging agent-based models for in silico testing of therapeutic combinations that can be prioritized for experimental validation. In total, our approach has the potential to rationally identify effective therapeutic strategies that lead to long-term durable control of tumors.



 Laura M. Heiser, Young Hwan Chang, Paul Macklin. Multiscale systems approach to target tumor ecosystem responses for therapeutic benefit [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f02bb56a8eb490bc79542639fae113efb7474f" target='_blank'>
              Abstract A027: Multiscale systems approach to target tumor ecosystem responses for therapeutic benefit
              </a>
            </td>
          <td>
            Laura M. Heiser, Young Hwan Chang, Paul Macklin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d71cf07bbb7f7d458f445b935e53d33e0b0d78bd" target='_blank'>
              Single-Cell Spatial Mapping of Human Kidney Development Reveals the Critical Role of the Local Microenvironment in Cell Fate Decisions
              </a>
            </td>
          <td>
            Jonathan Levinsohn, Samuel Grindel, Bernhard Dumoulin, Amin Abedini, A. Huang, Rena Levin-Klein, Boaz Weicz, Grace Rabinowitz, Andi M. Bergeson, Eunji Ha, Konstantin A Klötzer, Nancy R. Zhang, Paul M. Titchenell, Mingyao Li, Joo-Seop Park, Laura S. Finn, Kotaro Sasaki, P. Beckerman, Oren Pleniceanu, Alex J. Hughes, Katalin Susztak
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="While cervical cancer (CC) prognosis depends on tumor staging, the spatiotemporal evolution of tumor microenvironment (TME) heterogeneity during metastatic progression remains poorly characterized at single-cell resolution. We employed an integrative multi-omics approach, combining single-cell RNA sequencing (scRNA-seq; n = 11), spatial transcriptomics (ST), and bulk RNA-seq data from the TCGA-CESC cohort (n = 304), to systematically map TME remodeling across CC progression stages. scRNA-seq was performed on primary lesions from patients with localized (n = 3), regional (n = 4), and metastatic (n = 4) diseases, with in-depth analyses focusing on cellular characteristics, cell type composition alterations, functional changes, differentiation trajectories, and cell-cell interaction networks. These findings were further validated using spatial transcriptomics, bulk RNA-seq data, and multiple immunohistochemistry (mIHC) experiments. ScRNA-seq data revealed that the TME of the metastatic group displayed a distinct immunosuppressive phenotype. Three key subclusters closely linked to TME remodeling in this group were identified. Notably, a novel metastasis-associated epithelial subpopulation (Epi0_AGR2), characterized by both epithelial-mesenchymal transition (EMT) and chemokine secretory phenotypes, was discovered. Gene Set Variation Analysis (GSVA) revealed that transforming growth factor β (TGF-β) signaling activation served as its primary transcriptional driver. Additionally, a neutrophil subset with pro-tumor and immunosuppressive properties, as well as a cancer-associated fibroblasts (CAFs) subset that promoted angiogenesis, were enriched in the metastatic group. Cell-cell interaction analysis and spatial mapping further revealed the formation of coordinated Epi0-neutrophil-CAFs niches, which established TGF-β-CXCL1/2/8-OSM/OSMR feedforward loops. Importantly, a computational model derived from the TME metastatic niche signature demonstrated significant prognostic stratification in the TCGA cohort (HR = 2.5179, p = 0.0144). In all, this study provides the first comprehensive delineation of stage-specific TME dynamics in CC, revealing TGF-β-driven cellular cooperativity as a metastatic switch. The joint framework establishes a potential clinically translatable tool for precision prognosis and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac624640fa9f989701717857751510daeec92896" target='_blank'>
              Multi-Omics Analysis Reveals the transforming growth factor-β Signaling-Driven Multicellular Interactions with Prognostic Relevance in Cervical Cancer Progression
              </a>
            </td>
          <td>
            Yuhan Wang, Guang-xu Cao, Huimin Zeng, Yong Zhi, Meng-Xi Xu, Ying Wang, Min Liu, Yetian Ruan, Ka Yu Tse, Qingfeng Zhang, Jinli Gao, Zhiqiang Han, Fang Li
          </td>
          <td>2025-06-20</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Spread through air spaces (STAS) represents a novel invasion mechanism in adenocarcinoma that considerably influences lung cancer clinical outcomes; however, studies of its mechanisms at the spatial level are lacking. Methods We used the NanoString GeoMx digital spatial profiling (DSP) technology to conduct a spatial transcriptomic analysis of surgically resected tissues from non-small-cell lung cancer (NSCLC) patients with or without STAS. Results Compared with tumor nests in non-STAS patients, HLA-DRB5 and RASGRF1 were significantly less expressed in compartments of STAS, suggesting their inhibitory roles in the occurrence of STAS. Meanwhile, an increase in CD4 T memory cells and a decrease in B cells were observed in the tumor immune microenvironment of STAS. Furthermore, distinct molecular profiles were observed between tumor cells in tumor nests and in air spaces in STAS patients, which was highlighted by the elevated ITGA2 expression in the air spaces. These results were validated in an independent cohort by multiplex immunofluorescence stainings. Conclusion This study is the first to use DSP to analyze spatial transcriptomic profiles of NSCLC tumor nests and air space tumors, and it identifies potential module features that may be used for STAS identification and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/570168f77732c756800b064b7d546cfa69566936" target='_blank'>
              Spatial deciphering of the transcriptomic heterogeneity of tumor spread through air spaces in lung cancer
              </a>
            </td>
          <td>
            Wenhao Wang, Wenhao Zhou, Jingli Fan, Tao Jiang, Guang Yang, Congcong Song, Siwei Xu, Haitao Luo, Huining Liu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Nuclear protein in testis (NUT) carcinomas (NCs) are rare, clinically aggressive tumors with characteristic translocation involving NUTM1 and BRD4 genes. While NCs are generally characterized by undifferentiated basaloid cells with focal/abrupt squamous differentiation, tumoral heterogeneity remains unexplored. This may have therapeutic implications as NC frequently develops drug resistance and metastasis. Spatial transcriptomics (ST) sequencing technology emerged as an approach to elucidate tumoral heterogeneity by integrating cellular transcriptome and spatial distributions within tissues, providing insights into the interactions among different cell types and overall tissue genomic architecture.


METHODS
Three cases of sinonasal NC with formalin-fixed paraffin embedded tissue formed the study material. A representative region of each NC was subjected to 10X Genomics Visium Spatial Gene Expression analysis. Resulting data were analyzed in 10X Genomics Loupe Browser.


RESULTS
All cases demonstrated statistically distinct (graph-neural network based) transcriptomic clusters. These clusters were correlated with histologic characteristics by parsing all spots by region types based on tissue distribution including tumor, limited stroma, and normal epithelium. Cluster constitution and transcriptomic pattern were consistent with the morphologic features of each tissue region. NC#3 had a relatively low number of tumor clusters in comparison to NC#1 and NC#2.


CONCLUSIONS
Spatial transcriptomic profiling corresponds to histopathologic features in NC and yet highlights cell type diversity across tumors and in the case of NC#2, within a given tumor. Behavioral correlates are speculative, but NC#3 for which the patient had a rapidly lethal outcome showed a preponderance of neural gene expression which may be of interest in tumor cell progression/evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8f96bf0cc82c0a5a49b8e030eb8a17bb4a9e111" target='_blank'>
              Magnifying Glass on Sinonasal NUT Carcinoma Heterogeneity via Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Diana Bell, Garret W. Choby, M. Afkhami, E. Maghami, R. Ferrarotto, Carl Snyderman, Eric Wang, Ehab Y. Hanna, Raja Seethala
          </td>
          <td>2025-07-07</td>
          <td>Head & neck</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="ABSTRACT Intra‐tumoral heterogeneity poses a major challenge to treating and managing cancer patients. A characteristic feature of melanoma is its composition of cancer cells with typically heterogeneous content of melanin pigment, the production of which is a hallmark of normal melanocytic differentiation but of poorly understood consequence in melanoma cells, as prospective assessment of pigment heterogeneity in melanoma cells has been experimentally challenging. Here, we describe a novel flow cytometric method for high purity separation of viable melanoma cells based on their melanin content, exploiting the light scattering properties of melanin. By fluorescence‐activated cell sorting, we show that cells with low‐pigment content (LPCs) in melanoma cell lines and patient tumors are usually far more abundant than high‐pigment cells (HPCs) and have substantially increased potentials for colony formation in vitro and tumor formation in vivo. In RNAseq analysis, HPCs showed P53 activation and perturbed cell cycling, whereas LPCs displayed upregulation of MYC‐associated transcription and activated ribosome biogenesis. In proof‐of‐concept studies, the latter was targeted by topoisomerase 2 beta targeting with CX‐5461, which induced senescent HPC phenotypes and irreversible loss of clonogenic activity. These data indicate an ‘inverted pyramid’ hierarchical model of melanoma cell propagation wherein abundant LPCs frequently renew their own malignant potential to propagate disease but also infrequently generate HPCs that spontaneously lose this ability in a manner that might be exploited as an anti‐melanoma strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a7c2a2a2e8b632c604835e92877002fe7954605" target='_blank'>
              Prospective Isolation According to Melanin Pigment Content of Melanoma Cells With Heterogeneous Potentials for Disease Propagation
              </a>
            </td>
          <td>
            Clare Fedele, Gamze Kuser-Abali, Ralph Rossi, Peinan Zhao, Jason Li, M. Ameratunga, Pacman Szeto, Youfang Zhang, Miles C. Andrews, Mark Shackleton
          </td>
          <td>2025-06-30</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c2fd63e082e26fba9172752b056b5282ca5810" target='_blank'>
              Barcoded viral tracing identifies immunosuppressive astrocyte-glioma interactions.
              </a>
            </td>
          <td>
            Brian M Andersen, Camilo Faust Akl, Michael A. Wheeler, Zhaorong Li, M. Diebold, Michael Kilian, Joseph M Rone, Aditya Misra, Jessica E. Kenison, Joon-Hyuk Lee, Hong-Gyun Lee, C. M. Polonio, David Merrell, J. Weiss, Lillie Godinez, Gavin Piester, Tomer Illouz, Jessica J Ye, Arianna Ghia, Jazmin Martinez, Elizabeth N. Chung, Léna Srun, Daniel Farrenkopf, Lucas E. Flausino, Anton M Schüle, L. Sanmarco, Federico Giovannoni, Luca Fehrenbacher, M. Charabati, Cristina Gutiérrez-Vázquez, Margaret M Cusick, Prem Prabhakar, Connor C Bossi, Emily Lapinskas, Roni Nowarski, G. Getz, Keith L. Ligon, M. Prinz, E. Chiocca, David A. Reardon, Francisco J. Quintana
          </td>
          <td>2025-06-25</td>
          <td>Nature</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Current approaches to estimating cell trajectories, tumor progression dynamics, and cell population diversity of tumor microenvironment often depend on single-cell RNA sequencing, which is costly and resource intensive. To address this limitation, we developed an artificial intelligence (AI) model that leverages cell morphology features and histological spatial organization to classify tumor cell differentiation status, infer cell dynamic trajectories, and quantify tumor progression from hematoxylin and eosin (H&E)–stained whole-slide images. In three independent lung adenocarcinoma cohorts, our AI-based model accurately predicted cell differential status and provided quantifiable measures of tumor progression that were prognostic of patient survival. Spatial transcriptomic integrative analyses revealed cell components and gene signatures enriched in different cell differentiation statuses. Bulk transcriptomic analyses revealed that fast-progressing tumors exhibit up-regulated cell cycle pathways, while slow-progressing tumors retain characteristics of normal lung epithelium. This cost-effective method enables large-scale analysis of tumor progression dynamics using routinely collected pathology slides and provides insights into intratumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ccd7a11dd44adb135af6900f42639f7a9eb70bd" target='_blank'>
              Image-based inference of tumor cell trajectories enables large-scale cancer progression analysis
              </a>
            </td>
          <td>
            Yang Liu, Ling Cai, Ruichen Rong, Shidan Wang, Liwei Jia, Peiran Quan, Qin Zhou, Guan‑Di Xiao, Yang Xie
          </td>
          <td>2025-07-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Melanomas are genetically heterogeneous, displaying mitogen-activated protein kinase mutations and homozygous loss of tumor suppressor genes. Mouse models combining such mutations produce fast-growing tumors. In contrast, rare, slow-growing tumors arise in mice combining Braf activation with heterozygous loss of Pten. Here we show that similar tumors can arise in albino mice bearing only a Braf mutation. Incidence kinetics suggest a stochastic event underlies tumorigenesis in tumors that arise with only a Braf mutation, yet de novo mutations or structural variants that could explain the incidence of most tumors could not be found. Single-cell transcriptomics of tumors identify a cell type resembling “neural crest-like” cells in human and mouse melanomas. These exist in normal mouse skin, expand upon Braf activation, and persist through serial transplantation; analyses of gene expression suggest they serve as precursors of malignant cells. This state may serve as an intermediate on a slow path to malignancy that may provide a diagnostically and therapeutically important source of cellular heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a8b6d33ff26b499d62c39adf32521b06fa0db2" target='_blank'>
              Uncovering minimal pathways in melanoma initiation
              </a>
            </td>
          <td>
            Hui Xiao, Jessica Shiu, Chi-Fen Chen, Jie Wu, Peijie Zhou, Sahil S. Telang, Rolando Ruiz-Vega, Robert A Edwards, Qing Nie, A. D. Lander, Anand K. Ganesan
          </td>
          <td>2025-06-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Recent advancements in transcriptomic analysis, combined with single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics, have deepened our understanding of the tumor microenvironment and cellular heterogeneity, laying the groundwork for personalized therapies. The aim of this research is to explore the heterogeneity of tumor cells in colorectal cancer (CRC) and evaluate their prognostic value in different therapeutic contexts, emphasizing the impact of tumor cell heterogeneity on disease progression. Methods scRNA-seq alongside spatial transcriptomics was employed to analyze the heterogeneity of tumor cells in CRC, the spatial distribution of tumor cells, and their interactions with the microenvironment. Results We identified nine distinct tumor cell subtypes, with MLXIPL + neoplasm prevalent in advanced CRC, while ADH1C + and MUC2 + neoplasms were more common in early-stage CRC. MLXIPL + neoplasm was significantly associated with chemotherapy and targeted therapy efficacy. Analysis of spatial transcriptomics indicated that MLXIPL + neoplasm is located in the core area of the tumor cells. We developed a 13-gene prognostic signature (PS) using machine learning algorithm (StepCox backward), which predicts the prognosis of CRC patients. Furthermore, the patients with low PS score demonstrated higher immune cell infiltration and immune regulatory factors, suggesting enhanced immune surveillance and treatment response. Conclusions The findings highlight the critical role of tumor cell heterogeneity in CRC progression and treatment response, suggesting the need for personalized therapeutic strategies targeting different subpopulations. The constructed PS demonstrates significant clinical application value in predicting patient prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00805-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8065bcb7e9315d28e942e5a0a0b33a35e0d6f402" target='_blank'>
              Spatial transcriptomics and scRNA-seq: decoding tumor complexity and constructing prognostic models in colorectal cancer
              </a>
            </td>
          <td>
            Wei Song, Yatao Wang, Min Zhou, Fengqin Guo, Yanliang Liu
          </td>
          <td>2025-08-13</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Innate immune cells constitute the majority of the tumor microenvironment (TME), where they mediate both natural anti-tumor immunity and immunotherapy responses. While single-cell T- and B-cell receptor sequencing has provided fundamental insights into the clonal dynamics of human adaptive immunity, the lack of appropriate tools has precluded similar analysis of innate immune cells. Here, we describe a method that leverages somatic mitochondrial DNA (mtDNA) mutations to reconstruct clonal lineage relationships between single cells across cell types in native human tissues. We jointly sequenced single-cell transposase-accessible chromatin and mtDNA to profile n=124,958 cells from matched tumor, non-involved lung tissue (NILT), and peripheral blood of early-stage non-small cell lung cancer (NSCLC) patients, as well as n=93,757 cells from matched tumor and peripheral blood of ovarian cancer patients. Single-cell concomitant profiling of lineage and cell states of thousands of immune cells resolved clonality across cell types, tissue sites, and malignancies. Clonal tracing of innate immune cells demonstrates that TME-resident myeloid subsets, including macrophages and type 3 dendritic cells (DC3), are clonally linked to both circulating and tissue-infiltrating monocytes. Further, we identify distinct DC-biased and macrophage-biased myeloid clones, enriched in the tumor and NILT, respectively, and find that their circulating monocyte precursors exhibit distinct epigenetic profiles, suggesting that myeloid differentiation fate may be predetermined before TME infiltration. These results delineate the clonal pathways of intratumoral myeloid cell recruitment and differentiation in human cancer and suggest that remodeling of the tumor myeloid compartment may be peripherally programmed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3aaf4843ae398b98669a86ffe5560f59ef43af0" target='_blank'>
              Clonal lineage tracing of innate immune cells in human cancer
              </a>
            </td>
          <td>
            Vincent Liu, Katalin D. Sandor, Patrick K. Yan, Max Miao, Yajie Yin, Robert R. Stickels, Andy Y Chen, Kamir J. Hiam-Galvez, Jacob C. Gutierrez, Wenxi Zhang, Sairaj Sajjath, Raeline Valbuena, Steven Wang, Bence Dániel, Leif S. Ludwig, Brooke E Howitt, Caleb A. Lareau, Ansu Satpathy
          </td>
          <td>2025-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) lacks targeted therapies, leading to a poor prognosis. Younger patients with TNBC often present with aggressive disease and exhibit distinct tumor microenvironments (TME). We performed spatial profiling to understand the influence of menopausal status on the immune environment, tumor progression, and therapy response. Methods Eleven treatment-naive TNBC tumors were examined in epithelial and non-epithelial areas using digital spatial profiling to identify differentially expressed markers and pathways. Deconvolution methods were employed to identify immune cell subtypes, and the protein expression of T and B cells was confirmed. The prognostic utility of the identified genes and pathways was validated using the METABRIC and SCAN-B datasets. The expression of target genes was analyzed in the I-SPY 2 trial data to understand their influence on the response to specific therapies. Results Premenopausal tumors formed a distinct cluster characterized by the upregulation of cell activation and antigen presentation pathways. In contrast, T cell checkpoint, cancer antigen, and PI3K-AKT pathways were downregulated. External datasets validated these findings, showing a lower hazard and better prognosis for genes upregulated in premenopausal tumors. An enrichment of CD8 + T cells, endothelial cells, and monocytes was observed, along with increased intratumoral protein expression of CD8, CD4, and CD20. Premenopausal tumors demonstrated better responses to PARP and HSP90 inhibitors but showed lower sensitivity to Pembrolizumab and PI3K-AKT inhibitors compared to postmenopausal tumors in the I-SPY 2 trial data. Conclusion Our study underscores the importance of menopausal status in shaping the TME within TNBC, revealing distinct immune landscapes and therapy responses. These findings highlight the need for larger prospective studies to validate differential treatment strategies in younger patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06786-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c6a6cdaccc22fd9e9450a555636943aa5ed9adb" target='_blank'>
              Spatial profiling reveals unique immune microenvironment in premenopausal triple-negative breast cancer associated with therapy response
              </a>
            </td>
          <td>
            V. P. Nimbalkar, V. Snijesh, S. Rajarajan, C. Anupama, S. Mahalakshmi, Annie Alexander, Deshica Dechamma, Manju Moorthy, Gopalakrishna Ramaswamy, Rakesh Ramesh, B. Srinath, J. Prabhu
          </td>
          <td>2025-07-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Identifying functionally important cell states and structure within heterogeneous tumors remains a significant biological and computational challenge. Current clustering or trajectory-based models are ill-equipped to address the notion that cancer cells reside along a phenotypic continuum. We present Archetypal Analysis network (AAnet), a neural network that learns archetypal states within a phenotypic continuum in single-cell data. Unlike traditional archetypal analysis, AAnet learns archetypes in simplex-shaped neural network latent space. Using pre-clinical models and clinical breast cancers, AAnet resolves distinct cell states and processes, including cell proliferation, hypoxia, metabolism and immune interactions. Primary tumor archetypes are recapitulated in matched liver, lung and lymph node metastases. Spatial transcriptomics reveal archetypal organization within the tumor, and, intra-archetypal mirroring between cancer and adjacent stromal cells. AAnet identifies GLUT3 within the hypoxic archetype that proves critical for tumor growth and metastasis. AAnet is a powerful tool, capturing complex, functional cell states from multimodal data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4f4efe9bf4b85978ed408342a79a79722d40fd" target='_blank'>
              AAnet resolves a continuum of spatially-localized cell states to unveil intratumoral heterogeneity.
              </a>
            </td>
          <td>
            Aarthi Venkat, Scott E. Youlten, Beatriz P. San Juan, Carley A Purcell, Shabarni Gupta, Matthew Amodio, Daniel P Neumann, John G Lock, Anton E Westacott, Cerys S McCool, Daniel B. Burkhardt, Andrew Benz, A. Mollbrink, J. Lundeberg, David van Dijk, Jeff Holst, Leonard D Goldstein, Sarah Kummerfeld, Smita Krishnaswamy, Christine L Chaffer
          </td>
          <td>2025-06-24</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) was considered the most aggressive type of primary brain tumor, marked by poor clinical outcomes and a high tendency to relapse. The therapeutic efficacy of GBM was significantly compromised by tumor heterogeneity, dysregulated metabolic pathways, the formation of an immunosuppressive microenvironment, and treatment resistance. Therefore, multi-dimensional therapeutic strategies targeting GBM-specific molecular features, its intrinsic properties, and microenvironmental regulatory networks were considered to potentially provide new breakthroughs for overcoming treatment resistance in GBM. Methods We analyzed single-cell RNA sequencing (scRNA-seq) data processed with the Seurat package to accurately identify cell types. Spatial transcriptomics integrated Multimodal Intersection Analysis, TransferData, and Robust Cell Type Decomposition techniques to characterize the spatial distribution patterns of key cell subtypes. CellChat was employed to assess intercellular communication networks. Furthermore, in vitro experiments confirmed the main regulatory role of YEATS4 (key transcription factor of C2 PCLAF+ subtype) in GBM malignant progression. Results Through scRNA-seq, we identified the C2 PCLAF+ subtype in GBM and analyzed its molecular characteristics and functional role in tumor progression. This subtype exhibited a unique malignant phenotype, marked by significant proliferative activity, characteristic metabolic reprogramming, and dysregulated cell death regulation mechanisms. Spatial transcriptomics revealed its preferential localization within specific tumor niches. Furthermore, the C2 PCLAF+ subtype established a specific interaction with fibroblasts through the MDK-LRP1 ligand–receptor pair. Critically, silencing YEATS4 in vitro significantly inhibited GBM malignancy. Additionally, the prognostic risk score model based on the C2 PCLAF+ subtype demonstrated significant clinical translational value. Conclusion Our study systematically elucidated the malignant characteristics of the C2 PCLAF+ subtype and its molecular mechanisms driving GBM progression. This subtype promoted therapeutic resistance through unique metabolic reprogramming, MDK-LRP1-mediated microenvironmental interactions, and immunosuppressive properties. YEATS4 knockdown effectively suppressed malignant tumor behaviors, highlighting its therapeutic potential. These findings provided novel targeted intervention strategies to address GBM heterogeneity and treatment resistance, offering promising avenues for overcoming current therapeutic limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8a996fe1a4f26a1455e98643e994edaf34d934" target='_blank'>
              Single-cell and spatial atlas of glioblastoma heterogeneity: characterizing the PCLAF+ subtype and YEATS4’s oncogenic role
              </a>
            </td>
          <td>
            Siqi Ma, Yuwei Sun, Shaowei Zheng, Yilong Fu, Liangyu Wang, Dun Liu, Henan Jiao, Xuqiang Zhu, Xueyuan Li, Dongming Yan, Di Chen, Zi Ye
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rationale: Breast cancer recurrences and treatment failures can be attributed to intra-tumoral heterogeneity (ITH), which is characterize by the coexistence of diverse cellular states, including cancer stem cells (CSCs), within a single tumor.Recent insights suggest that ITH arises from non-genetic dynamics, enabling tumors to adapt and evolve into a therapy-tolerant state under treatment pressure. The aim of this work is to decipher the origin of persistent radiation tolerant cells (RTP) in breast tumors and to understand their mechanisms in order to find new strategies to avoid radiation resistance. Methods: To this aim,we developed a lineage tracing system and engineeredvarious breast cancer cell lines and patient-derived xenografts totracked radiation-induced cell plasticity. We combined lineage tracing with a unique RNAi screen under irradiation to identify and functionally validate the regulators of radio-induced cell plasticity. Results: We discovered that RTP cells, which possess CSC properties, emerge from radiotherapy-induced reprogramming of non-CSCs. From the combinatorial approach of the lineage tracing and the RNAi screen under irradiation, we then identified and functionally validated the LRP4/YAP axis as a crucial regulator of radio-induced cell plasticity. We further demonstrate that overexpression of LRP4 is common in residual disease post-treatment and is associated with breast tumors of poor prognosis. Conclusions: This work has demonstrated that the LRP4/YAP axis drives radioresistance by promoting the emergence of RTP cells through radiation-induced plasticity, and that modulation of the LRP4/YAP axis is a promising strategy for sensitizing breast cancers to radiotherapy, opening up a new avenue for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ee60b3470960d374218748b117281dcd5ba1d2a" target='_blank'>
              The LRP4/YAP axis drives the radiation-tolerant persister (RTP) cell state in breast cancer
              </a>
            </td>
          <td>
            Violaine Forissier, Julien Wicinski, Martin Castagné, Guillaume Pinna, Shuheng Lin, Anaïs Grandon, Caroline Bonnet, M. Macario, Remy Castellano, Simon Valdenaire, Julien Darréon, Agnès Tallet, C. Ginestier, E. Charafe-Jauffret
          </td>
          <td>2025-06-23</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Small cell lung cancer (SCLC) is a lethal lung malignancy and patients are often diagnosed with distant metastasis. Nearly all patients suffer from disease relapsing with inherent chemoresistance. Lack of targeted SCLC therapies further worsens disease outcomes, making it highly desirable to identify novel and effective therapeutic targets. Methods To search for potential therapeutic targets in SCLC, we analyzed publicly available single-cell and bulk RNA-sequencing (RNA-seq) data from normal, lung adenocarcinoma, and SCLC tumor tissues. To assess the targeting potential of FOXM1, we developed various in vitro models, including DOX-On-shFOXM1 (Tet-ON) inducible stable knockdown systems. Cisplatin resistant human and murine SCLC cell lines were generated to assess the role of FOXM1 in chemotherapy resistance. Immunoblotting, immunohistochemistry (IHC), and immuno-fluorescence were used to analyze the expression of FOXM1 and target proteins. ChIP-assay was used to study protein-gene interactions. Further, multicolor flow cytometry was employed to study the effect of FOXM1 inhibition on human T cells activation and differentiation. Subcutaneous xenograft and SCLC spontaneous (RPM: RBfl/fl;TP53fl/fl;LSL-MYCT58A) mouse models were used to evaluate the efficacy of FOXM1 inhibitors. Results Single-cell as well as bulk RNA-seq data revealed that FOXM1, an oncogenic transcription factor, is overexpressed in SCLC, and it was recapitulated in human and murine SCLC tissues and cell lines. Interestingly, chemo-resistant (CR) SCLC showed a substantially higher FOXM1 expression than naïve SCLC. Silencing FOXM1 genetically or pharmacologically by FOXM1 inhibitors revealed a marked reduction in cell viability, colony formation, migration and sphere formation in naïve and CR SCLC cells. Moreover, FOXM1 inhibition induced apoptosis and cell cycle arrest in SCLC cells. Furthermore, FOXM1 inhibition in combination with first-line platinum-based chemotherapy showed synergistic anticancer effects in both xenograft and RPM mouse models of SCLC. Our RNA-seq analysis revealed that FOXM1 inhibition altered the Aurora Kinase B (AURKB) signaling pathway, which is dysregulated in SCLC. Moreover, we found FOXM1 inhibition enhanced T cell activation and supported the differentiation of CD8 + cytotoxic T cells, and T cell-mediated killing of cancer cells. Conclusions Our study demonstrates that FOXM1 targeting using small molecule inhibitors has the potential to be a novel therapeutic strategy to combat SCLC progression including chemotherapeutic resistance and reshaping the anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cf5514bc8f1dd3db6ac08b065580f4ed8890ea4" target='_blank'>
              FOXM1 targeting alters AURKB activity and reshapes antitumor immunity to curb the progression of small cell lung cancer
              </a>
            </td>
          <td>
            Md Arafat Khan, Parvez Khan, Mahek Fatima, Asad Ur Rehman, Laiba Anwar, Z. Alsafwani, Aatiya Ahmad, Mohammad Ali Abbas Zaidi, Jesse L. Cox, Areem Zahid, Sameer Mohiuddin, Sung Hoon Kim, Juan A. Santamaria-Barria, Imayavaramban Lakshmanan, B. Katzenellenbogen, John A. Katzenellenbogen, A. Ganti, Surinder K. Batra, M. Nasser
          </td>
          <td>2025-07-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of kidney renal clear cell carcinoma (KIRC) exhibits complex dynamics among immune, stromal, and malignant cells that drive immune escape (IE) mechanisms and influence clinical outcomes. Through single-cell RNA sequencing and high-dimensional weighted gene co-expression network analysis (hdWGCNA), we identified a crucial IE-related gene module most strongly associated with KIRC progression. Partitioning Around Medoids (PAM) clustering delineated two distinct IE patterns, with pattern one demonstrating prolonged patient survival. Employing advanced machine learning (ML) algorithms, we identified Cathepsin S (CTSS) as the most pivotal tumor suppressor, with elevated CTSS expression consistently predicting improved survival across multiple independent cohorts. Functional analyses revealed significant enrichment of CTSS in immune-regulatory pathways, including B/T cell activation and inflammatory activity. Mutation profiling uncovered distinct genomic alterations in 5q35.3 among CTSS-high tumors, while drug response prediction identified eight potential therapeutic agents (e.g., Navitoclax, Ibrutinib) exhibiting enhanced efficacy in these patients. Notably, CTSS expression strongly correlated with immune cell infiltration and established immunotherapy biomarkers, supporting its dual role as both a prognostic indicator and predictor of immune response. This study provides mechanistic insights into IE in KIRC and positions CTSS as a promising biomarker and therapeutic target for precision immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03267-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6364c60b20a894510d01361e9f698e8acfc1b2ab" target='_blank'>
              CTSS in the tumor microenvironment links immune escape and immunotherapy sensitivity in kidney renal clear cell carcinoma
              </a>
            </td>
          <td>
            Hanjing Zhou, Jun Ying, Xuchun Xu, Jian Huang
          </td>
          <td>2025-07-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a significant global health challenge despite advancements in screening, HPV vaccination, and therapeutic strategies. Tumor heterogeneity, driven by epigenetic modifications, affects immune evasion, metastasis, and treatment response. Cancer-associated fibroblasts (CAFs) play a crucial role in CC progression and therapy resistance. Single-cell sequencing offers new insights but remains underutilized in CC research. This study integrates single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and deconvolution analysis to identify key genes and immunotherapy targets. By constructing a prognostic model and exploring the immune microenvironment, we aim to provide novel insights into CC pathogenesis and potential therapeutic strategies. Methods We utilized scRNA-seq, spatial transcriptomics, deconvolution analysis, and pseudotime trajectory mapping to delineate fibroblast subtypes within the tumor immune microenvironment (TIME) of CC. Functional annotations, differential gene expression profiling, cell–cell communication pathways, and transcription factor networks were systematically analyzed. A prognostic model based on bulk RNA-seq data was constructed and validated through survival analysis, with correlations to immune microenvironment characteristics. Functional experiments investigated the role of SDC1, a critical mediator of fibroblast-tumor crosstalk. Additionally, Fibroblast–tumor cell co-culture systems and functional assays were employed to investigate the paracrine role of SDC1. The CAF MYH11⁺ subpopulation was isolated via fluorescence-activated cell sorting (FACS). Multiplex immunofluorescence and immunohistochemical analyses were performed on both cultured cells and human cervical cancer tissue samples to characterize the spatial distribution and dynamic remodeling of MYH11 during stromal reorganization. Results Six distinct fibroblast subtypes were identified, including the C0 MYH11 + fibroblasts, which exhibited unique roles in stemness maintenance, metabolic activity, and immune regulation. Spatial and functional analyses revealed that the C0 subtype is central to tumor-fibroblast interactions, particularly through the MDK-SDC1 signaling axis. The prognostic model incorporating fibroblast-specific markers demonstrated robust predictive power for patient survival outcomes. Additionally, in vitro SDC1 knockdown significantly inhibited CC cell proliferation, migration, and invasion. Fibroblasts show spatially regulated heterogeneity, with activation markers enriched in the tumor zone and MYH11 highest in normal zones, indicating dynamic stromal remodeling. C0 MYH11 + CAF Promotes Tumor Cell Proliferation, Migration, and Inhibits Apoptosis via Soluble SDC1. Conclusion Our results illustrate, in some ways, the possible immunomodulatory and tumor supporting roles of CAFs in CC TIME and highlight the possibility that the MDK-SDC1 pathway is a promising therapeutic target. This study not only promotes a partially new understanding of temporal heterogeneity in CC, but also provides a possible reference base for the development of new biomarkers and immunotherapy approaches to improve clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03432-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d1387a936b1ef6d7e045f6c7bcff60d568a6f7" target='_blank'>
              Deciphering the tumor immune microenvironment: single-cell and spatial transcriptomic insights into cervical cancer fibroblasts
              </a>
            </td>
          <td>
            Zhiheng Lin, Youwei Zhou, Zhenran Liu, Wenyang Nie, Hengjie Cao, Shengnan Li, Xuanling Li, Lijun Zhu, Guangyao Lin, Yanyu Ding, Yi Jiang, Zuxi Gu, Lianwei Xu, Zhijie Zhao, Hua-Bao Cai
          </td>
          <td>2025-07-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>2</td>
          <td>9</td>
        </tr>

        <tr id="Background: Tumor-associated macrophages (TAMs) are critical regulators of the hepatocellular carcinoma (HCC) microenvironment, yet their epigenetic heterogeneity and regulatory programs remain poorly defined. Methods: We performed integrative analysis on single-cell RNA-seq and ATAC-seq profiling of HCC patients to dissect TAM subtypes at high resolution. By correlating chromatin accessibility with gene expression, we identified cell-type-specific candidate cis-regulatory elements (CREs). TAM subsets with prognostic significance were determined through integration with HCC clinical cohorts. Pseudotime and multi-regional analyses were used to uncover regulatory trajectories underlying macrophage phenotypic transitions. The identification framework of a super-enhancer (SE) was constructed, and potential therapeutic targets were prioritized using drug–gene interaction data. Results: We delineated the regulatory landscape of TAMs in HCC, revealing cell-type-specific chromatin accessibility patterns underlying TAM heterogeneity. The 65,342 CREs linked to gene expression were identified, with distal CREs contributing most to cell-type-specific regulation. Notably, SPP1+ TAMs were found to be enriched in tumor cores and associated with poor prognosis in HCC. Liver-resident Kupffer cells showed progressive loss of the core transcription factors SPIC and MAFB, suggesting a potential transition into SPP1+ TAMs under tumor pressure. We identified 133 SPP1+ TAM-specific SEs and constructed a TF–SE–target gene regulatory network. Notably, 13 target genes showed higher drug–gene interaction effects, highlighting their therapeutic potential. Conclusions: This study provides the chromatin accessibility map of TAMs in HCC and reveals how distal CRE-driven transcriptional programs shape TAM states. Our findings lay the foundation for understanding the epigenetic regulation of TAM heterogeneity and nominate potential targets for TAM-directed immunotherapy in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e02327a10ea72222cca675d378cb3f0e78ed3b3" target='_blank'>
              Integrated Single-Cell Analysis Dissects Regulatory Mechanisms Underlying Tumor-Associated Macrophage Plasticity in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yu Gu, Wenyong Zhu, Zhihui Zhang, Huiling Shu, Hao Huang, Xiao Sun
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e5808d86a5155048eee62dffb4fd9d18cc91c36" target='_blank'>
              AI-powered Deep Visual Proteomics reveals critical molecular transitions in pancreatic cancer precursors
              </a>
            </td>
          <td>
            Jimin Min, Lisa Schweizer, Gijs Zonderland, B. Selvanesan, Lukas Oldenburg, Seong-Woo Bae, Bong Jun Kim, Benjamin J. Swanson, Kelsey A. Klute, T. Caffrey, P. Grandgenett, Michael A. Hollingsworth, Ishani Ummat, Maximilian T. Strauss, A. Mund, Anirban Maitra
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Single-cell RNA sequencing captures static snapshots of gene expression but lacks the ability to track continuous gene expression dynamics over time. To overcome this limitation, we developed PROFET (Particle-based Reconstruction Of generative Force-matched Expression Trajectories), a computational framework that reconstructs continuous, nonlinear single-cell gene expression trajectories from sparsely sampled scRNA-seq data. PROFET first generates particle flows between time-stamped samples using a novel Lipschitz-regularized gradient flow approach and then learns a global vector field for trajectory reconstruction using neural force-matching. The framework was developed using synthetic data simulating cell state transitions and subsequently validated on both mouse and human in vitro datasets. We then deployed PROFET to investigate heterogeneity in treatment responses to palbociclib, a CDK4/6 inhibitor, in hormone receptor positive breast cancer. By comparing newly generated scRNA-seq data from a palbociclib-resistant breast cancer cell line with published patient-derived datasets, we identified a subpopulation of patient cells exhibiting profound phenotypic shifts in response to treatment, along with surface markers uniquely enriched in those cells. By recovering temporal information from static snapshots, PROFET enables inference of continuous single-cell expression trajectories, providing a powerful tool for dissecting the heterogeneity of cell state transitions in treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3148960cea2da72c3e81995d56cd3f9d3682ff6f" target='_blank'>
              PROFET Predicts Continuous Gene Expression Dynamics from scRNA-seq Data to Elucidate Heterogeneity of Cancer Treatment Responses
              </a>
            </td>
          <td>
            Yu-Chen Cheng, Hyemin Gu, Thomas O McDonald, Wenbo Wu, Shubham Tripathi, Cristina Guarducci, Douglas Russo, Daniel L. Abravanel, Madeline Bailey, Yue Wang, Yun Zhang, Yannis Pantazis, Herbert Levine, R. Jeselsohn, M. Katsoulakis, Franziska Michor
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Insufficient infiltration of cytotoxic T cells into solid tumors remains a critical obstacle in cancer immunotherapy. Despite extensive efforts to comprehend the mechanisms governing this limited infiltration, few studies have focused on the evolution of T cell motility behavior after co-culture. In this study, we combined quantitative cell trajectory analysis, computational modeling, and bulk/single-cell RNA sequencing to systematically characterize the impact of cell interactions. We reveal that in a 2.5D co-culture system with multiple cancer-cell clusters, cancer-specific T cells exhibit increased directional persistence, which facilitates their efficient searching of cancer-cell clusters. Additionally, these T cells form prolonged interactions with cancer cells, which is the most crucial factor for their accumulation on cancer-cell clusters. Furthermore, post-interaction, a cancer-cell subpopulation displays immunosuppressive traits, reducing T cell attractant expression, and undergoing epithelial-to-mesenchymal transition. These findings offer valuable insights into improving immunotherapy efficacy and tackling T cell infiltration challenges in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0a380330210bd455d4b1fb482cbeaabdb64b1d" target='_blank'>
              Deciphering antigen-specific T cell navigation tactics and cancer immune evasion in co-cultures
              </a>
            </td>
          <td>
            Xinyue Li, Taoli Jin, Lisha Wang, Senbin Wu, Yerong Liu, Ming Li, Weijing Han, Xuefei Li
          </td>
          <td>2025-07-31</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) progression is driven by a complex interplay of immunosuppressive and inflammatory mechanisms involving various cell types, functional states, and extracellular milieu. To unravel this intricate relationship, we took a systems biology approach using high resolution spatial distributions of multiple key markers in clinically diverse CRC tumors. We employed Multiplexed Ion Beam Imaging (MIBI-TOF) to capture 65 protein markers across 3,700 fields of view from 250 CRC tumors. This allowed us to deeply characterize the phenotypes and functional states of 5.7 million cells. To incorporate extracellular geography information, we developed a computational approach named CellExt. This quantified and modeled the gradated abundance of key structural and functional proteins found extracellularly, such as fibronectin, granzymeB and vimentin, from each cell boundary to a 40-micron distance in pixelwise steps. CellExt detected a distinct immunosuppressive modality characterized by extracellular PDL1, TNF-related apoptosis-inducing ligand (TRAIL), and matrix metalloproteinase-9 (MMP9). Investigating the cell types involved led us to map the potential receivers and emitters by linking extracellular trends to cellular enrichment and gene expression. We found that Pankeratin-high epithelial cells were acting as TRAIL receivers with downstream activation of non-canonical pro-tumorigenic signaling through S6 phosphorylation. In parallel, we found that T cell suppressive CD40+ antigen-presenting B cells with PD1 expression were acting as MMP9 emitters. Additionally, CellExt was able to identify two spatially segregated inflammatory modalities: IFNgamma cytokine with mucin protein, MUC5AC and prostaglandin synthase, cyclooxygenase-2 with extracellular matrix component, Collagen1A1. This suggests a strong inflammatory role for Collagen1A1 in CRC. We then contextualized our identified modalities by genetic background, tumor stage and CMS subtype of CRC tumors. Taken together, our approach revealed novel axes of immunosuppression and inflammation and puts forward potential extracellular targets for therapeutic intervention in specific CRC patient subsets.



 Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, Christine Fullaway, Sean Bendall, Iain Beehuat Tan, Shyam Prabhakar. Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P38.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18eea1ae1312b735cfb9925fbcce433bb07c964b" target='_blank'>
              Abstract P38: Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation
              </a>
            </td>
          <td>
            Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, C. Fullaway, Sean C. Bendall, Iain Beehuat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Understanding the interactions between specific cell types within tissue environments is essential for elucidating key biological processes, such as immune responses, cancer progression, inflammation, and development, in both physiological and pathological studies. The predominant methods for analyzing cell–cell interactions (CCI) rely primarily on statistical inference using mapping or network-based techniques. However, these approaches often struggle to prioritize meaningful interactions owing to the high sparsity and heterogeneity inherent in single-cell RNA sequencing (scRNA-seq) data, where small but biologically important differences can be easily overlooked. To overcome these limitations, we developed PriorCCI, a deep-learning framework that leverages a convolutional neural network (CNN) alongside Grad-CAM++, an explainable artificial intelligence algorithm. This study aims to provide a scalable, interpretable, and biologically meaningful framework for systematically identifying and prioritizing key ligand–receptor interactions between defined cell-type pairs from single-cell RNA-seq data, particularly in complex environments such as tumors. PriorCCI effectively prioritizes interactions between cancer and other cell types within the tumor microenvironment and accurately identifies biologically significant interactions related to angiogenesis. By providing a visual interpretation of gene-pair contributions, our approach enables robust inference of gene–gene interactions across distinct cell types from scRNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8d0cda95b391f509ed11c400e323a325085c153" target='_blank'>
              PriorCCI: Interpretable Deep Learning Framework for Identifying Key Ligand–Receptor Interactions Between Specific Cell Types from Single-Cell Transcriptomes
              </a>
            </td>
          <td>
            Hanbyeol Kim, Eunyoung Choi, Yujeong Shim, J. Kwon
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (TME) significantly impacts cancer progression, yet traditional animal models do not fully recapitulate the situation in humans. To address this, we developed tumor-derived precision lung slices (TD-PCLS), an ex vivo platform for studying the lung TME and evaluating therapies. TD-PCLS, viable for 8 to 10 days, preserve the heterogeneity and metabolic activity of primary tumors, as confirmed by seahorse analysis. Using multispectral FACS and phenocycler multiplex imaging, we spatially profiled TME components and cancer cell functionality. Additionally, TD-PCLS revealed patient-specific responses to chemo- and immunotherapies. To complement TD-PCLS, we established tumor-cell-seeded PCLS (TCS-PCLS) by introducing tumor and immune cells into healthy lung slices. This model highlighted macrophage-tumor interactions as critical for tumor cell proliferation, migration, and immune modulation. Together, these platforms provide a robust tool for lung cancer research, enabling precision medicine and advancing therapeutic discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6f776606f9d2df99cf55dbb6d93c418c75fd58c" target='_blank'>
              Living human lung slices for ex vivo modelling of lung cancer.
              </a>
            </td>
          <td>
            S. Mansouri, Annika Karger, Clemens Ruppert, Marc A Schneider, A. Weigert, Rajender Nandigama, Blerina Aliraj, Lisa Strotmann, A. V. Cherian, Diethard Pruefer, Peter Dorfmuller, Ludger Fink, I. Alkoudmani, S. Gattenlöhner, B. Eul, Andre Althoff, Peter Kleine, H. Winter, Andreas Guenther, A. Ghofrani, S. Pullamsetti, F. Grimminger, Werner Seeger, R. Savai
          </td>
          <td>2025-07-29</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) remains one of the most prevalent malignancies. A series of differentially expressed genes (DEGs) have been identified in tumor samples and play critical roles in modulating the characteristics of tumor cells. However, some DEGs are specifically expressed in the tumor microenvironment (TME) cells. The underlying mechanisms of the functional DEGs warrant comprehensive investigation to elucidate their contributions to tumor biology of NSCLC. Therefore, the primary goal of our study is to systematically investigate TME-related DEGs using NSCLC as a model. Methods DEG analysis was performed by comparing bulk transcriptomes of adjacent and tumor samples across 7 independent NSCLC cohorts. Expression pattern of these DEGs were annotated to specific cell types using a single-cell RNA sequencing (scRNA-seq) dataset from 13 NSCLC studies. Myeloid and stromal cells were re-clustered to achieve a detailed characterization of cell-cell interactions within the TME. Spatial co-localization of distinct subpopulations was validated by immunofluorescence staining and spatial transcriptomics (ST). Finally, functional relevance of these interactions was evaluated using a conditional knockout mouse model. Results A total of 82 overlapping DEGs were screened out using bulk transcriptomes across 7 NSCLC cohorts. After clustering the integrated 547,360 cells from 217 adjacent/tumor NSCLC samples with available scRNA-seq data, we observed that most of these DEGs were specifically expressed in epithelial, myeloid, and stromal cells. Notably, SPP1 ranks the top DEG and is specifically expressed in myeloid cells. The elevated SPP1 expression in bulk transcriptome was attributed to both enriched proportion of SPP1+ macrophages and increased expression level of SPP1 in such myeloid subcluster in NSCLC tumor, which was potentially regulated by several transcriptional factors (e.g., HIF1A, ATF5, and STAT1). Cell-cell communication analysis indicated FAP+ fibroblasts exhibited the strongest interaction with SPP1+ macrophages within TME. Multiple fluorescent staining demonstrated significantly increased SPP1+ macrophage and FAP+ fibroblast interactions in tumor compared to adjacent tissues. Nichnet analysis showed VCAN-ITGB1 ligand-receptor axis mediating this cellular interaction. ST analysis revealed that this interaction established an immune-excluding barrier surrounding the tumor core. In addition, immune cells (CD4+/CD8+ T and NK cell) were recruited to tumor edge. Consistently, macrophage-specific Spp1 knockout in a mouse model resulted in increased CD3+ and CD8+ T cell infiltration and reduced subcutaneous tumor size. Clinically, patients with FAPHighSPP1High expression in tumor exhibited significantly poor prognosis for immunotherapy treatment. Conclusions SPP1 ranks as the top DEG in NSCLC tumor attributed to its specific expression in SPP1+ macrophages, proportion of which increases in tumor and interact with FAP+ fibroblasts. Their interactions contribute to the formation of a tumor barrier that inhibits immune infiltration, reducing the immunotherapy efficacy in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5040bbe7618af6ffa532070665742270d9ea5641" target='_blank'>
              Single-cell and spatial transcriptomics profile the interaction of SPP1+ macrophages and FAP+ fibroblasts in non-small cell lung cancer
              </a>
            </td>
          <td>
            Minqin Xiao, Yiqi Deng, Hang Guo, Zhixiang Ren, Yajiao He, Xia Ren, Li-Bin Huang, Wei-Han Zhang, Hai-Ning Chen, Y. Shu, Fanxin Zeng, Yan Zhang, Heng Xu, Lanlan Wang
          </td>
          <td>2025-07-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d66d1c34039c0624a67bfae6ffcc0194c9308ee" target='_blank'>
              Bioelectric and Epigenetic Landscapes in Lateralized Breast Tumors Reveal Distinct Tumor Microenvironment Signatures
              </a>
            </td>
          <td>
            Sebastián Real, Sergio R. Laurito, Pablo Gonzalez, Oscar Bello, Joao Carvalho, María Roqué
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Globally, endometrial cancer continues to impact a significant number of women. Immunotherapy provides those suffering from advanced or relapsed disease hope, but an important barrier is still the absence of trustworthy predictive biomarkers. To tackle this challenge, single-cell sequencing and spatial transcriptomics (ST) are increasingly applied. In cervical cancers of the no specific molecular profile (NSMP) subtype accompanied by p53 mutations. In many cases, the tumor microenvironment (TME) in endometrial cancer exhibits strong immunosuppression or poor immune cell infiltration, often leading to worse clinical outcomes. Single-cell sequencing reveals cellular heterogeneity and helps identify potential therapeutic targets and predict treatment responses. Conversely, ST assists in determining biomarkers that influence the effectiveness of immunotherapy by capturing the spatial organization of tumors. When combined, these technologies allow for integrated multi-omics analysis that aids in the development of immunotherapies, prognostication, and diagnosis. But there are still moral and legal issues. Clinicians may be able to improve outcomes for patients who don’t respond well to current immunotherapies by utilizing these combined approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f087dd53dee17003d5211dac4cb4360cce4847" target='_blank'>
              Single-cell technologies and spatial transcriptomics: decoding immune low - response states in endometrial cancer
              </a>
            </td>
          <td>
            Yumeng Li, Hua Qiu, Zhenzhen Zhao, Fanghua Qi, Pingping Cai
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac106c26ebd96b1e15958a8ddb1a880a0c7c51a9" target='_blank'>
              Intratumoral Heterogeneity of Vimentin Modulates Nuclear Mechanotransduction, DNA Damage Response and Cancer Cell Survival
              </a>
            </td>
          <td>
            Infante Elvira, Terriac Emmanuel, Gelin Matthieu, Siegfried Hugo, Pereira David, Roca Vanessa, Varet Hugo, Khalilian Sara, Rietveld Reinier, Asnacios Atef, van Bodegraven Emma J, Etienne-Manneville Sandrine
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) continues to be a leading cause of cancer-related deaths globally, primarily due to its late diagnosis and the complex nature of its tumor microenvironment (TME). Within this environment, cancer-associated fibroblasts (CAFs) play a crucial role in regulating NSCLC progression and therapeutic resistance. Recent advancements in single-cell and spatial technologies have uncovered significant heterogeneity among CAFs, revealing distinct subpopulations with varying cellular origins, phenotypes, and functions. Besides, the role of small extracellular vesicles (sEVs) in facilitating bidirectional communication highlights functional importance of CAF derived sEVs in mediating the crosstalk between cancer and TME, implicating the potential of CAF-sEVs to be un-invasive diagnostic biomarkers for NSCLC. Despite such new insights, challenges remain exist, particularly concerning the mechanisms that drive CAF plasticity, the interactions between specific CAF subsets with cancer cells or immune cells, and the translational significance of CAF-sEVs. In this review, we summarize the latest advances in our understanding of CAF heterogeneity, the functional roles and clinical relevance of CAFs and their secreted sEVs in driving NSCLC, and the translational landscape of CAF-targeted strategies. By critically evaluating the most recent evidence, this review provides clinically relevant insights and highlights future directions for overcoming CAF-mediated barriers to therapy. Our aim is to facilitate the development of more precise, biomarker-driven, and safe approaches for targeting CAFs, ultimately improving personalized treatment and outcomes for patients with NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d1bb104cbbe26ccaae9f632691df643466beac" target='_blank'>
              Hidden forces: the impact of cancer-associated fibroblasts on non-small cell lung cancer development and therapy
              </a>
            </td>
          <td>
            Ziheng Wu, Meilin Luo, Huiyi Hu, Zhijun Jiang, Yinan Lu, Zhi-Jie Xiao
          </td>
          <td>2025-07-25</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0289ee010aab237793e43a5fa9c9591075ed88ec" target='_blank'>
              A clinically relevant morpho-molecular classification of lung neuroendocrine tumours
              </a>
            </td>
          <td>
            A. Sexton-Oates, Emilie Mathian, Noah Candeli, Yuliya Lim, C. Voegele, A. Génova, Laurane Mangé, Zhaozhi Li, Tijmen van Weert, L. Hillen, R. Blázquez-Encinas, A. González-Pérez, Maike L. Morrison, Eleonora Lauricella, L. Mangiante, Lisa Bonheme, L. Moonen, G. Absenger, Janine Altmuller, Cyril Dégletagne, Vincent Cahais, G. Centonze, Amélie Chabrier, C. Cuenin, F. Damiola, V. Montpréville, J. Deleuze, Anne-Marie C. Dingemans, É. Fadel, N. Gadot, A. Ghantous, Paolo Graziano, Paul Hofman, V. Hofman, A. Ibáñez-Costa, S. Lacomme, Núria López-Bigas, M. Lund-Iversen, Massimo Milione, L. Muscarella, S. Pedraza-Arévalo, C. Perrin, G. Planchard, H. Popper, L. Roz, A. Sparaneo, W. Buikhuisen, José van den Berg, M. Tesselaar, Jaehee Kim, Ernst Jan, M. Speel, S. Tabone-Eglinger, Thomas Walter, Gavin M. Wright, Justo P Castaño, Lara Chalabreysse, Liming Chen, Christophe Caux, Marco Volante, Nicolas Girard, J. Vignaud, Esther Conde, A. Mansuet-Lupo, Luka Brčić, Giuseppe Pelosi, Mauro Papotti, Sylvie Lantuéjoul, J. Derks, Talya Dayton, N. Alcala, M. Foll, L. Fernandez-Cuesta
          </td>
          <td>2025-07-18</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Microglial spatial heterogeneity remains a crucial yet not fully answered question in the context of potential cell-directed therapies for Alzheimer’s disease (AD). There is an unclear understanding of the dynamics of distinct microglia states adjacent to or far from amyloid-beta (Aβ) plaques and their contributions to neurodegenerative diseases. Here we combine multicolor fluorescence cell fate mapping, single-cell transcriptional analysis, epigenetic profiling, immunohistochemistry and computational modeling to comprehensively characterize the relation of plaque-associated microglia (PAM) and non-plaque-associated microglia (non-PAM) in a mouse model of AD. We show that non-PAM are a distinct and highly dynamic microglial state, transitioning to PAM after Aβ plaque deposition in female mice. Non-PAM modulate the cell population expansion in response to amyloid deposition and rapidly respond to environmental cues. Indeed, Csf1 signaling modulates non-PAM-to-PAM transition during disease progression. Our data suggest that microglia states and their dynamics between each other can have distinct contributions to disease, and they may be targeted for the treatment of AD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef9a3fe9de367ad9376482f5aa956c82f8113ee" target='_blank'>
              Response of spatially defined microglia states with distinct chromatin accessibility in a mouse model of Alzheimer’s disease
              </a>
            </td>
          <td>
            A. Ardura-Fabregat, Lance Fredrick Pahutan Bosch, Emile Wogram, Omar S. Mossad, R. Sankowski, Philipp Aktories, Lina Kieger, James Cook, Dilara Hasavci, Hatice Ulupinar, Daniel Brock, Fang Wang, Nicola Iovino, Samuel Wald, Sebastian Preissl, Bahtiyar Yilmaz, Daniel Schnepf, Andrew J. Macpherson, Thomas Blank, Katrin Kierdorf, M. Prinz
          </td>
          <td>2025-07-14</td>
          <td>Nature Neuroscience</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Up until present day, chimeric antigen receptor (CAR)-T cell therapy has only been approved for hematological malignancies, as CAR-T cells do not show comparable efficacy in solid tumors. Therefore, understanding the features of the tumor microenvironment (TME), is key to improve efficacy of adoptive cell therapies (ACTs) against solid tumors. In this context, robust workflows, which dissect the complex interactions between CAR-T cells and the TME are still lacking. To address this need, we have established an ex vivo workflow co-culturing tissue slices from patient tumor resections with CAR-T cells. The workflow is composed of assessing several complementary attributes, such as cytokine release via flow cytometry, quantification of cell infiltration into the tumor and assessment of the regions of the tissue slice the CAR-T cell infiltrate into by using the MACSima™ imaging cyclic staining technology. Using this workflow it is possible to observe the behavior of CAR-T cells within the tumor and its TME, their infiltration into distinct tumor compartments, as well as to dissect the underlying molecular mechanisms that drive T cell migration, thanks to MACSima™ multiplexing technology and its ability to image several markers at the same time. Assessment of ovarian carcinoma tissue slices revealed substantial release of specific cytokines and increased infiltration of T cells in the tumor areas when CAR-T cells were added to the tissue slices as compared to non-engineered T cells. The establishment of this novel approach will enable researchers to better characterize the interaction between CAR-T cells and the TME. Tissue slices present an intrinsic heterogeneity, which is indeed an advantage compared to other in vitro models but can turn itself into complex results interpretation. Therefore, we recommend that any conclusion derived from this assay should be verified with complementary models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80f902b452985b81dfcba4e7339eba4450d5adf" target='_blank'>
              Precision-cut tumor tissue slices, a novel tool to study the tumor microenvironment interactions with chimeric antigen receptor (CAR) T cells
              </a>
            </td>
          <td>
            Valeria Durante, Alina Wittwer, Benjamin Theek, Manuel Martinez-Osuna, E. Donnadieu, O. Hardt, Dominik Eckardt, A. Bosio, Sonja Schallenberg, Christoph Herbel
          </td>
          <td>2025-08-08</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Transcriptional Intratumoral heterogeneity (ITH) is a hallmark of aggressive cancers, yet how transcriptional ITH programs drive tumor metastasis and immune evasion in upper aerodigestive squamous cell carcinoma (UASCC) remains unclear. Through single-cell RNA sequencing analysis of UASCC cells and patient tumors, we uncovered a hybrid epithelial mesenchymal transition (hEMT) ITH program linked to metastatic dissemination. The transcription factor ETS1 was identified as a master regulator of the hEMT program, directly activating pro-metastatic genes and promoting distant spread in vivo. Unexpectedly, ETS1 also orchestrated an immune-cold tumor microenvironment by transcriptionally activating both STAT1 and PD-L1 (CD274) genes, suppressing T lymphocyte infiltration, and elevating immune checkpoint molecules. Clinically, ETS1-high tumors strongly correlated with poor survival and resistance to immune checkpoint blockade across multiple cohorts. Leveraging drug screens, we discovered that ETS1-high cancers are vulnerable to HSP90 inhibitors (e.g., Alvespimycin), which suppress ETS1 by disrupting HIF1A-mediated transcriptional activation. Together, our work reveals ETS1 as a dual driver of tumor distal metastasis and immune evasion in UASCC, while nominating HSP90 inhibition as a tailored treatment strategy for ETS1-driven tumors. These findings provide a roadmap for targeting aggressive ITH subsets and overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e629750fa34c7548f170b4ee72868f72fd5026a6" target='_blank'>
              ETS1 Orchestrates a Hybrid EMT Program Driving in vivo Metastasis and Immune Evasion
              </a>
            </td>
          <td>
            Benjamin Ziman, Talia Wenger, Chehyun Nam, Qian Yang, Daniel Arnaudov, Megha Sheth, Zhixuan Jing, Yuhao Pan, Joe Vargas, Yong Teng, Uttam K. Sinha, Y. Park, De-Chen Lin
          </td>
          <td>2025-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) are functionally diverse stromal regulators that orchestrate tumor progression, metastasis, and therapy resistance through dynamic crosstalk within the tumor microenvironment (TME). Recent advances in single‐cell multiomics and spatial transcriptomics have identified conserved CAF subtypes with distinct molecular signatures, spatial distributions, and context‐dependent roles, highlighting their dual capacity to promote immunosuppression or restrain tumor growth. However, therapeutic strategies struggle to reconcile this functional duality, hindering clinical translation. This review systematically categorizes CAF subtypes by origin, biomarkers, and TME‐specific functions, focusing on their roles in chemoresistance, maintenance of stemness, and formation of immunosuppressive niches. We evaluate emerging targeting approaches, including selective depletion of tumor‐promoting subsets (e.g., fibroblast activation protein+ CAFs), epigenetic reprogramming toward antitumor phenotypes, and inhibition of CXCL12/CXCR4 or transforming growth factor‐beta signaling pathways. Spatial multiomics‐driven combinatorial therapies, such as the synergistic use of CAFs and immune checkpoint inhibitors, are highlighted as strategies to overcome microenvironment‐driven resistance. By integrating CAF biology with translational advances, this work provides a roadmap for developing subtype‐specific biomarkers and precision stromal therapies, directly informing efforts to disrupt tumor‐stroma coevolution. Key concepts include spatial transcriptomics, stromal reprogramming, and tumor‐stroma coevolution, offering actionable insights for both mechanistic research and clinical innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d8f3a8f9589a88ea165cf34934eda0905c9005" target='_blank'>
              Cancer‐Associated Fibroblasts: Heterogeneity, Cancer Pathogenesis, and Therapeutic Targets
              </a>
            </td>
          <td>
            Yutao Li, Qingyun Liu, Xilin Jing, Yuqi Wang, Xiaohua Jia, Xing Yang, Kezhong Chen
          </td>
          <td>2025-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pediatric low-grade gliomas (pLGG) comprise 35% of all brain tumors. Despite favorable survival, patients experience significant morbidity from disease and treatments. A deeper understanding of pLGG biology is essential to identify novel, more effective, and less toxic therapies. We utilized single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and cytokine analyses to characterize and understand tumor and immune cell heterogeneity of pilocytic astrocytoma (PA) and ganglioglioma (GG). scRNA-seq revealed tumor and immune cells within the tumor microenvironment (TME). Tumor cell subsets include both progenitor and mature cell populations. Immune cells included myeloid and lymphocytic cells. There was a significant difference between the prevalence of two major myeloid subclusters between PA and GG. Bulk and single-cell cytokine analyses evaluated the immune cell signaling cascade with distinct immune phenotypes among tumor samples. KIAA1549-BRAF tumors appeared more immunogenic, secreting higher levels of immune cell activators and chemokines, compared to BRAF V600E tumors. Spatial transcriptomics revealed the differential gene expression of these chemokines and their location within the TME. A multi-pronged analysis demonstrated the complexity of the PA and GG TME and differences between genetic drivers that may influence their response to immunotherapy. Further investigation of immune cell infiltration and tumor-immune interactions is warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dc01e4fe40878b7885262fd79ea7216b56b949" target='_blank'>
              Multi-pronged analysis of pediatric low-grade glioma and ganglioglioma reveals a unique tumor microenvironment associated with BRAF alterations.
              </a>
            </td>
          <td>
            S. Zahedi, Kent Riemondy, Tian Liu, A. Griesinger, A. Donson, April A Apfelbaum, Rui Fu, Julian Grandvallet Contreras, M. Crespo, J. DeSisto, Madeline Groat, Emil Bratbak, Adam L Green, Todd C Hankinson, Michael Handler, Rajeev Vibhakar, Nicholas Willard, Nicholas K Foreman, Tzu Phang, Jean M. Mulcahy Levy
          </td>
          <td>2025-06-30</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ef3fe6c1c7dcea2e010f5575de7254a23fff3d7" target='_blank'>
              Spatial and single-cell transcriptomics landscape of adenomyosis
              </a>
            </td>
          <td>
            X. Yang, C. Li, J. He, B. Xie, L. Lv, J. Xu, Y. Lin, C. Zhou, M. Li, Z. Zeng, J. Tan, S. Chen, G. Cui, G. Liu, S. Suo, G. Peng, X. Liang
          </td>
          <td>2025-07-07</td>
          <td>None</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Precursor lesions like polyps and adenomas in the colon commonly precede colorectal cancer (CRC), advancing through the “normal-polyp-adenoma-carcinoma” sequence towards malignancy. Yet, the cellular heterogeneity and molecular mechanisms involved in CRC development remain inadequately characterized. Methods To understand the molecular mechanisms driving the onset and progression of CRC, we conducted a comprehensive analysis of ten clinical colorectal samples representing sequential pathological stages using single-cell RNA sequencing (scRNA-seq). Validation was performed through immunofluorescence and immunohistochemistry analyses in a separate human colorectal tissue cohort. Additional verification was carried out using bioinformatics analyses of public TCGA and GEO datasets. Results Our comprehensive analyses not only reveal the cellular diversity and transcriptomic differences throughout disease progression but also highlight the importance of leveraging ligand–receptor gene expression to distinguish various cell subtypes. Subsequent examination and validation with a larger sample cohort uncover the specific involvement of ligand–receptor genes, transcription factors, immunoglobulin genes, and heat shock genes in regulating immune responses and microenvironment changes during colorectal tumorigenesis. Conclusions Our extensive transcriptome dataset provides valuable insights and acts as a fundamental resource to deepen our understanding of the complex molecular landscape in CRC. This dataset facilitates improved diagnostic accuracy and the creation of more personalized therapeutic approaches. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06785-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e15cc889bc4d3cf181a1a8c15dbc72bc14a7b3" target='_blank'>
              A dynamic molecular landscape in colorectal cancer progression at single-cell resolution
              </a>
            </td>
          <td>
            Jianwen Sheng, Nianshuang Li, Shuai Li, Yuman Ye, Yuanhang Liao, Yupeng Zhang, Huan Chen, Mengxian Tu, Guping Zhong, Chaoliang Xiong, Pan Zheng, Yuting Lei, Zekun He, Xingxing He, Yao Zhang, Lei Wang, Xiaolin Gao, Yin Zhu, Jianping Liu, Huizhen Fan
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Understanding the molecular interactions between cells, tissues or organs is key to understanding the functioning of a biological system as a whole. Results Here, we propose crossWGCNA: a co-expression-based method that identifies highly interacting genes unbiasedly and that we employ to study stroma-epithelium communication in breast cancer. CrossWGCNA can be applied to bulk, single cell and spatial transcriptomics data. We validate it both in silico and experimentally, and we provide a fully documented R package allowing users to employ it. Conclusions The wide applicability and agnostic nature of our tool make it complementary to existing methods overcoming the limitations arising from strong baseline assumptions. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11747-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b943ef5ac17c56205d2bc6ae7f9f20263cb8cd8" target='_blank'>
              Cross-tissue gene expression interactions from bulk, single cell and spatial transcriptomics with crossWGCNA
              </a>
            </td>
          <td>
            Aurora Savino, R. Iannuzzi, L. Avalle, Andrea Lobascio, Francesco Iorio, Paolo Provero, Valeria Poli
          </td>
          <td>2025-07-01</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: Breast cancer, the most prevalent malignancy among women worldwide, exhibits significant heterogeneity, particularly in the tumor microenvironment (TME), which poses challenges for treatment. Spatial transcriptomics (ST) has emerged as a transformative technology, enabling gene expression analysis while preserving tissue spatial architecture. This provides unprecedented insights into tumor heterogeneity, cellular interactions, and disease mechanisms, offering a powerful tool for advancing breast cancer research and therapy. This review aims to synthesize the applications of ST in breast cancer research, focusing on its role in decoding tumor heterogeneity, characterizing the TME, elucidating progression and metastasis dynamics, and predicting therapeutic responses. We also explore how ST can bridge molecular profiling with clinical translation to enhance precision therapy. The key scientific concepts of review included the following: We summarize the technological advancements in ST, including imaging-based and sequencing-based methods, and their applications in breast cancer. Key findings highlight how ST resolves spatial heterogeneity across molecular subtypes and histological variants. ST reveals the dynamic interplay between tumor cells, immune cells, and stromal components, uncovering mechanisms of immune evasion, metabolic reprogramming, and therapeutic resistance. Additionally, ST identifies spatial prognostic markers and predicts responses to chemotherapy, targeted therapy, and immunotherapy. We propose that ST serves as a hub for integrating multi-omics data, offering a roadmap for precision oncology and personalized treatment strategies in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbbaea20f0037aedc680b32a7558f9b67f94b665" target='_blank'>
              Spatial Transcriptomics Decodes Breast Cancer Microenvironment Heterogeneity: From Multidimensional Dynamic Profiling to Precision Therapy Blueprint Construction
              </a>
            </td>
          <td>
            A. Ma, L. Xiang, Jingping Yuan, Qianwen Wang, Lina Zhao, Honglin Yan
          </td>
          <td>2025-07-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca80c0d5c8118b6afa306ae5816ee0be1c3263a" target='_blank'>
              Single-cell transcriptomics reveals hypoxia-driven iCAF_PLAU is associated with stemness and immunosuppression in anorectal malignant melanoma.
              </a>
            </td>
          <td>
            Hao Zhang, Lin Gan, Xiaofei Duan, Baojing Tuo, Hao Zhang, Senbo Liu, Yugui Lian, Enjie Liu, Zhenqiang Sun
          </td>
          <td>2025-07-04</td>
          <td>Journal of gastroenterology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the most abundant non-cancerous cell type in glioblastoma (GBM) and heavily influence GBM biology, contributing to tumor progression, therapeutic resistance, immune evasion, and neovascularization. Current in vitro models that utilize IL-4/IL-13 stimulation fail to capture the transcriptional and functional heterogeneity of TAMs observed in vivo. In this study, we utilize a serum-free indirect co-culture model with patient-derived xenolines to polarize primary human macrophages and characterize their molecular and functional phenotypes. We demonstrate that xenoline-polarized macrophages diverge from classical M1/M2 states and instead adopt transcriptional signatures reflective of TAM subsets identified from patients. Notably, macrophages polarized with the radiation-therapy selected xenoline, JX14P-RT, exhibited gene expression patterns enriched for interferon response and hypoxia, mirroring recurrent GBM samples. In contrast, JX14P TAMs showed enrichment in phagocytic gene sets. Functional validation of these phenotypes revealed discrepancies between the transcriptionally predicted and observed phenotypes, emphasizing the importance of integrating phenotypic validation in sequencing studies. Altogether, our findings establish xenoline-polarized macrophages as a useful alternative to traditional models that can be used to study immune-interactions in vitro. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02057-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f15f733124e62cedabe127db49de61b0e9f489" target='_blank'>
              Xenoline-polarized macrophages as an alternative in vitro model of tumor-associated macrophages in glioblastoma
              </a>
            </td>
          <td>
            Acta Neuropathologica Communications, Christopher D. Willey, Hasan Alrefai, Benjamin Lin, Amr Elkholy, Manoj Kumar, Taylor L. Schanel, Kevin J. Lee, Patricia H. Hicks, Joshua C. Anderson, Gao Guo, Eun-Young Erin Ahn, Ryan Miller
          </td>
          <td>2025-06-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938950b1958bbfaef501d56fa236f86c6e2c6141" target='_blank'>
              Multi-omic analysis of gallbladder cancer identifies distinct tumor microenvironments associated with disease progression.
              </a>
            </td>
          <td>
            Tao Zhou, Yanhong Wu, Shuai Li, Xinyao Qiu, Erdong Liu, Zhihua Xie, Xuebing Shi, Yani Zhang, Guosheng Ma, Wenbo Guo, Xiang Wang, Kaiting Wang, Xiaomeng Yao, Ji Hu, Siyun Shen, Shuai Yang, Xiaoqing Jiang, Jing Fu, Hongyang Wang, Jin Gu, Lei Chen
          </td>
          <td>2025-06-26</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>12</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a leading cause of cancer-related death, and reliable biomarkers for prognosis and treatment guidance remain limited. Membrane proteins play key roles in tumor progression and therapeutic response, yet their clinical utility in LUAD remains underexplored. We integrated scRNA-seq, spatial transcriptomics, and bulk RNA-seq datasets from multiple LUAD cohorts to identify cancer-specific membrane proteins derived from epithelial subpopulations. Based on these results, we constructed a prognostic signature, LCaMPS, and evaluated its predictive performance using multiple datasets. The expression of model genes was confirmed at both the bulk RNA and protein levels. Associations with the tumor microenvironment (TME) and drug sensitivity were further analyzed. A distinct LUAD-enriched epithelial cluster (Epi_c0) exhibiting hypoxic and EMT signatures was identified. 35 cancer-specific membrane proteins were defined, several of which, including TSPAN8, BACE2, and COX16, showed strong spatial localization within the tumor regions. LCaMPS, a 9-membrane gene-based prognostic model, stratified patient prognosis and predicted 5- and 10-year survival rates with high accuracy. High LCaMPS scores were associated with increased infiltration of neutrophils, endothelial cells, and fibroblasts in the TME and predicted higher sensitivity to 66 chemotherapeutic agents, including Gemcitabine and Sorafenib. Low-risk patients were predicted to respond better to drugs, such as Cisplatin and Parthenolide. This study highlights the importance of membrane expression patterns in LUAD at single-cell and spatial resolution. The LCaMPS model provides a robust prognostic and therapeutic stratification tool with potential applications in personalized cancer management. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-11105-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b97812f84c01b026525655a678dbe43cf10957" target='_blank'>
              A novel cancer-associated membrane signature predicts prognosis and therapeutic response for lung adenocarcinoma
              </a>
            </td>
          <td>
            Biao Tu, Jun Wu, Wei Zhang, Haitao Tang, Tenghui Dai, Bingfeng Xie
          </td>
          <td>2025-07-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most aggressive and treatment-resistant brain tumors, largely due to its profound intratumoral heterogeneity and immunosuppressive microenvironment. Various classifications of GBM subtypes were created based on transcriptional and methylation profiles. This effort, followed by the development of new technology such as single-nuclei sequencing (snRNAseq) and spatial transcriptomics, led to a better understanding of the glioma cells’ plasticity and their ability to transition between diverse cellular states. GBM cells can mimic neurodevelopmental programs to resemble oligodendrocyte or neural progenitor behavior and hitchhike the local neuronal network to support their growth. The tumor microenvironment, especially under hypoxic conditions, drives the tumor cell clonal selection, which then reshapes the immune cells’ functions. These adaptations contribute to immune evasion by progressively disabling T cell and myeloid cell functions, ultimately establishing a highly immunosuppressive tumor milieu. This complex and metabolically constrained environment poses a major barrier to effective antitumor immunity and limits the success of conventional therapies. Understanding the dynamic interactions between glioma cells and their microenvironment is essential for the development of more effective immunotherapies and rational combination strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0dda0986360e9c65e565d1dc755ad16cfd3932d" target='_blank'>
              Unraveling Glioblastoma Heterogeneity: Advancing Immunological Insights and Therapeutic Innovations
              </a>
            </td>
          <td>
            Joshua H. Liu, Maksym Horiachok, Santosh Guru, Cecile L. Maire
          </td>
          <td>2025-08-02</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Spatial transcriptomics (ST) enables high-resolution mapping of gene expression across tissue architecture and can reveal the organization of cellular states in complex tissues, such as the tumor microenvironment. Using ST data from glioblastoma (GBM) patients, a large recent study1 identified robust gene programs in a spatially oblivious manner and used them to assign cell states to spatial spots and assess cell-state spatial organization. The results characterized GBM tumors as generally spatially heterogeneous, with some hypoxia-associated structured regions. However, recent computational methods support analyses that more tightly integrate spatial context and gene expression to identify continuous gradients and rare niches in ST data. We hypothesized that applying such an approach may reveal additional key aspects of the spatial organization of transcriptional programs in GBM tumors.



 Based on our previous successes with traditional topic modeling in transcriptomic analysis, we focused on the recently published method Spatial Transcriptomics Analysis with topic Modeling to uncover spatial Patterns (STAMP)2. STAMP uses a deep generative model to perform spatially aware topic modeling that can account for replicates. Using STAMP, we reanalyzed the 26 ST samples (10x Visium, 60,000+ spots) from 17 patients in the GBM dataset1 to infer spatial topics or gene programs, which we then interpreted in the context of previous studies. We also extended the analysis framework to enable quantification and statistical assessment of the spatial coherence of continuous gene program weights.



 Consistent with previous studies, we identified gene programs associated with a hypoxic niche, as well as with precursor and differentiated endothelial, oligodendrocyte and neuronal cell types. However, our study also revealed a number of new gene programs with strongly spatially organized patterns of expression occurring in multiple patient samples. These findings indicate additional spatial axes that may capture important variation in and across GBM tumor microenvironments.



 Complementing a previous hypoxia-associated model of GBM spatial organization, our study characterized additional, spatially resolved functional processes, which may be clinically and therapeutically relevant. The results suggest that a spatially aware, non-categorical approach to ST analysis may drive discovery of previously hidden, spatially coherent, recurrent gene programs in the tumor microenvironment. This framework is broadly applicable for dissecting the transcriptionally governed spatial architecture of solid tumors.



 1Greenwald, A.C. et al. Integrative spatial analysis reveals a multi-layered organization of glioblastoma. Cell 187, 2485–2501.e26 (2024). 2ZHong, C., Ang, K.S. & Chen, J. Interpretable spatially aware dimension reduction of spatial transcriptomics with STAMP. Nat. Methods 21, 2072–2083 (2024).



 Joseph J. Sifakis, Sophia Madejski, Samantha J. Riesenfeld. Spatially aware transcriptomic topic modeling reveals novel signals of spatial organization in glioblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/953df0c913ecb019b9b2578d60ef3030541a85b4" target='_blank'>
              Abstract B050: Spatially aware transcriptomic topic modeling reveals novel signals of spatial organization in glioblastoma
              </a>
            </td>
          <td>
            Joseph J. Sifakis, Sophia Madejski, Samantha J Riesenfeld
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Recent studies have explored the composition of tumor microenvironment (TME) in diffuse large B-cell lymphoma (DLBCL) However, cell-to-cell interactions, along with the spatial organization of DLBCL TME and their impact on patient outcomes, have remained poorly characterized. We ap-plied multiplexed immunofluorescence, cell phenotyping, and neighborhood analysis to investigate 1,218,756 single cells in 99 samples from patients with primary DLBCL. We identified 17 cell pheno-types and 10 recurrent cellular neighborhoods (RCNs) across samples, subdividing DLBCLs into immune-poor areas and areas with diverse immune cell infiltrates. Avoidance of B cells and PD-1+ T cells was associated with less aggressive clinical characteristics and favorable survival. Like-wise, the proximity of CD8+ T cell-rich and immune-poor RCNs translated to favorable, and the proximity of PD-L1+ B cell-rich and CD8+ T cell-rich RCNs to unfavorable patient outcomes. Our findings provide insights into the spatial interactions and organization of DLBCL TME with implica-tions for patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8a36543383d1dc42cdb78f6b491ab4c213fbf1a" target='_blank'>
              Spatial Analysis of the Tumor Microenvironment in Diffuse Large B-Cell Lymphoma Reveals Clinically Relevant Cell Interactions and Recurrent Cellular Neighborhoods.
              </a>
            </td>
          <td>
            M. Autio, S. Leivonen, L. Meriranta, M. Karjalainen-Lindsberg, T. Pellinen, S. Leppä
          </td>
          <td>2025-08-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) tumours are classified by pathology into several distinct subtypes. Gene expression profiling has revealed transcriptional heterogeneity across NEPC, but this is rarely considered in the context of variation between pathologies. Diagnosis typically relies on immunohistochemical markers (CHGA, SYP, NCAM1) and genomic alterations in RB1, PTEN and TP53. We hypothesized that NEPC pathologies have unique transcriptional features. Single-cell RNA sequencing of 18 632 tumour cells from nine patient-derived xenograft models representing five pathologies (small-cell and large-cell neuroendocrine carcinomas, focal neuroendocrine differentiation (Focal NED), low-grade neuroendocrine and amphicrine) demonstrated pathway-specific enrichment. Focal NED and amphicrine tumours exhibited cellular subpopulations enriched for KRAS, IL2-STAT5 and TNF signalling pathways, absent in small- and large-cell carcinomas, which were instead enriched for Myc and E2F pathways. Furthermore, focal NED cells exhibited minimal clonal divergence from adjacent adenocarcinoma cells, while small cell carcinoma cells were clonally distinct. These data underscore significant transcriptional variation among NEPC pathologies, highlighting focal NED's unique biological context and its clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c9094a51a0844d2931b4655b974ca910515f91f" target='_blank'>
              Single-cell transcriptomics redefines focal neuroendocrine differentiation as a distinct prostate cancer pathology.
              </a>
            </td>
          <td>
            Rosalía Quezada Urban, S. Keerthikumar, A. Clark, Hong Wang, B. Phipson, Andrew Bakshi, Andrew Ryan, Heather Thorne, R. Taylor, Mitchell G. Lawrence, G. Risbridger, Roxanne Toivanen, D. L. Goode
          </td>
          <td>2025-07-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Metastasis causes about 90% of cancer deaths, but the anticipation of its occurrence from primary tumor biopsies remains an intractable clinical dilemma. Spatial organization and biochemical heterogeneity of TME are regulators of metastatic formation but are usually not interrogated by typical diagnostic approaches. To overcome this deficiency, we established a hybrid modeling approach that unites biopsy-derived proteomics, spatial statistical analysis, and machine learning to predict metastatic potential. Employing multiplexed proteomic imaging technologies like imaging mass cytometry, we profiled in situ protein expression patterns in the patient's biopsy tissue. These data were used to calculate spatial features like cell–cell interaction graphs, spatial proximity scores, and neighborhood-based immune signatures. Machine learning classifiers—graph neural networks and random forests—were trained to predict these spatial-biochemical signatures from recognized metastatic endpoints. Our model performed with excellent prediction capability (AUC ≈ 0.90) in a retrovalidation set across various solid cancers. Notably, we also found specific TME signatures that correlate with metastasis: dense foci of PD-L1⁺ macrophages and FOXP3⁺ regulatory T cells at the invasive edge were correlated with a higher risk of metastasis, and abundant CD8⁺ T cell infiltration at the invasive edge with lower progression. This integrated strategy provides a clinically feasible method for predicting metastasis from standard biopsy samples, which could potentially lead to earlier treatment and therapy stratification. Our results highlight the translational value of the combination of spatial proteomics and computational modeling towards improved prognostic precision in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc1c3111d9dc4060886bfbec6b57329f90c0fbb" target='_blank'>
              BIOCHEMICAL AND SPATIAL MODELING OF TUMOR MICROENVIRONMENT FROM BIOPSY PROTEOMICS TO PREDICT METASTATIC PROGRESSION
              </a>
            </td>
          <td>
            PEACE CHINONYEREM IKE, CHIKA GLORIA OKWUOGORI, THEOPHILUS IGWEBUIKE ILOEGBU, ONYEKA MILICENT ASUMAH, ONWUKWE OLUCHI BUCHI, AFOLAYAN OLUWATOSIN SUNDAY
          </td>
          <td>2025-07-10</td>
          <td>International Journal of Medical Biological and Pharmaceutical Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Formalin-fixed paraffin-embedded (FFPE) samples represent a vast, untapped resource for epigenomic research, yet molecular tools for deep analysis of these specimens remain limited. We introduce spatial FFPE-ATAC-seq, an approach for in situ profiling chromatin accessibility within archived tissues. This approach overcomes formalin-induced crosslinking challenges, allowing high-resolution mapping of chromatin landscapes while preserving tissue architecture. Applying spatial FFPE-ATAC-seq to mouse and human tissues, including brain and thymus, reveals intricate spatial organization and distinct cell types in alignment with tissue morphology. Integration with single-cell RNA sequencing validates the precision of our chromatin profiles in identifying key cell types and regulatory elements. We further apply this method to human melanoma, comprehensively characterizing chromatin accessibility across both tumor and non-tumor regions. This method significantly expands the toolkit for epigenomic research, unlocking the potential of an extensive collection of archived FFPE samples for studying gene regulation and disease mechanisms with spatial context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9a8ea3c46aa7a27b26189991ac0f0ce423df34a" target='_blank'>
              Spatial profiling of chromatin accessibility in formalin-fixed paraffin-embedded tissues
              </a>
            </td>
          <td>
            Pengfei Guo, Yufan Chen, Liran Mao, Angelysia Cardilla, Chin Nien Lee, Yan Cui, Dengge Jin, Yucong Hua, Xiaowei Xu, Yanxiang Deng
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Spatial genomics has revolutionized breast cancer research by mapping gene expression within the tumor microenvironment, offering unprecedented insights into the molecular and cellular interactions that drive tumor progression and therapeutic response across luminal A, luminal B, HER2-positive, and triple-negative subtypes. Unlike traditional genomic methods, spatial genomics preserves tissue architecture, revealing subtype-specific TME features, such as immune-rich regions in triple-negative breast cancer and stromal-dense environments in HER2-positive tumors. This review explores spatial genomics technologies, including Visium, GeoMx, and MERFISH, which enable precise TME analysis, highlighting their role in identifying biomarkers like PD-L1 and TGF-β for targeted therapies and immunotherapies. By elucidating TME-driven resistance mechanisms, such as hypoxia in triple-negative tumors or stromal remodeling in HER2-positive cases, spatial genomics informs clinical trials and combination therapies, enhancing patient stratification and treatment efficacy. Despite its promise, challenges including limited resolution, complex data analysis, and high costs hinder clinical adoption. Emerging technologies, artificial intelligence, and multi-omics integration offer solutions, promising higher precision and scalability. This review underscores spatial genomics’ transformative potential to bridge research and clinical practice, paving the way for personalized breast cancer therapies tailored to the dynamic TME, with future advancements poised to redefine treatment paradigms and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5822524fe77c324fb45f43e4adc21fca11863f06" target='_blank'>
              Unraveling tumor microenvironments with spatial genomics: Implications for precision breast cancer therapy
              </a>
            </td>
          <td>
            Amina Suleiman Kamsalem, Obianujunwa Mercy Ubah, Tobiloba Philip Olatokun, Regina Omodolapo Oluwaniyi, Olivia Ajifa Philips, Damilola Samuel Ojo-Omoniyi, Omolola Lolade Oluwasakinju
          </td>
          <td>2025-07-30</td>
          <td>International Journal of Life Science Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance poses a significant challenge in cancer therapy, contributing to rapid recurrence, disease progression, and high patient mortality. Despite its critical impact, few reliable predictors for cancer drug response or failure have been established for clinical application. Tumor heterogeneity and the tumor microenvironment (TME) are pivotal factors influencing cancer drug efficacy and resistance. Tumor heterogeneity leads to variable therapeutic responses among patients, while dynamic interactions between cancer cells and the TME enhance tumor survival and proliferation, underscoring the urgent need to identify cellular hallmarks for predicting drug response and resistance. Single-cell and spatial omics technologies provide high-resolution insights into gene expression at the individual cell level, capturing intercellular heterogeneity and revealing the underlying pathologies, mechanisms, and cellular interactions. This review delves into the principles, methodologies, and workflows of single-cell and spatial omics in cancer drug research, highlighting key hallmarks involving tumor heterogeneity, TME reprogramming, cell–cell interactions, metabolic modulation, and signaling pathway regulation in drug treatment at single-cell and spatial levels. Furthermore, we synthesize predictive cellular biomarkers for cancer drug response and resistance across 25 cancer types, paving the way for advancements in cancer precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af5a8672965af056452be6aaf8b56f1ff303b62" target='_blank'>
              Application of single-cell and spatial omics in deciphering cellular hallmarks of cancer drug response and resistance
              </a>
            </td>
          <td>
            Xiaoxia Cheng, Ting Peng, Tian Chu, Yiqun Yang, Jia Liu, Qinglei Gao, C. Cao, Juncheng Wei
          </td>
          <td>2025-07-02</td>
          <td>Journal of Hematology & Oncology</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="A
bstract
 Lung cancer remains a significant public health burden. One of the most personalized treatments uses a patient’s own tumor infiltrating lymphocytes (TILs), and TIL activity is also essential for immune checkpoint inhibitor (ICI) effectiveness. Responses to immunotherapies vary due to immune-suppressive tumor microenvironments (TMEs) and limited antigen presentation. In this study, we computationally examine cell-cell signaling and transcriptional activity using single-cell RNA sequencing of lung cancer treated by inhibiting methyltransferase EZH2. We show that EZH2 inhibition shifts the TME to immunogenic signaling patterns conducive to increased T cell response, including antigen presentation and homing. T cells also showed more stem-like phenotypes. Transcriptional activity was quieter with EZH2 inhibition but revealed better interferon response, altered myeloid and B cell differentiation, and apoptotic markers. Importantly, inferred EZH2 activity showed it could perform non-methyltransferase duties vital for T cell differentiation. These results indicate that EZH2 inhibition could improve immunotherapies for lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d5b3471193acbd21fb76beb7e0f92d1e7a78da" target='_blank'>
              Single-cell profiling of EZH2-mediated immune signaling perturbations in NSCLC
              </a>
            </td>
          <td>
            Daniel R. Plaugher, Avery R Childress, Christian M. Gosser, Dave-Preston Esoe, Jinpeng Liu, C. Brainson
          </td>
          <td>2025-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Pancreatic cancer (PC) is marked by extensive heterogeneity, posing significant challenges to effective treatment. The tumor microenvironment (TME), particularly cancer-associated fibroblasts (CAFs), plays a critical role in driving PC progression. However, the prognostic and functional contributions of distinct CAF subtypes remain inadequately understood. Here, we introduce a novel 7-gene risk model that not only robustly stratifies PC patients but also unveils the unique role of PHLDA1 as a key mediator in tumor-stroma crosstalk. Methods By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA sequencing data, we comprehensively characterized the heterogeneity of CAFs in PC. We identified five CAF subtypes and focused on matrix CAFs (mCAFs), which were strongly associated with poor prognosis. A 7-gene mCAF-associated risk model was constructed using advanced machine learning algorithms, and the biological significance of PHLDA1 was validated through co-culture experiments and pan-cancer analyses. Results Our multiomics analysis revealed that the novel 7-gene model (comprising USP36, KLF5, MT2A, KDM6B, PHLDA1, REL, and DDIT4) accurately predicts patient survival, immunotherapy response, and TME status. Notably, PHLDA1 was uniquely overexpressed in CAFs and correlated with the activation of key protumorigenic pathways, including EMT, KRAS, and TGF-β, underscoring its central role in modulating the crosstalk between CAFs and malignant ductal cells. Pan-cancer analysis further supported PHLDA1’s prognostic and immunomodulatory significance across multiple tumor types. Conclusion Our study presents a novel 7-gene prognostic model that significantly enhances risk stratification in PC and identifies PHLDA1+ CAFs as promising prognostic biomarkers and therapeutic targets. These findings provide new insights into the TME of PC and open avenues for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69308ca9c802c2244ef4487c21e12f41adc3c76e" target='_blank'>
              Integrated multiomics analysis identifies PHLDA1+ fibroblasts as prognostic biomarkers and mediators of biological functions in pancreatic cancer
              </a>
            </td>
          <td>
            Rui Wang, Guan-Hua Qin, Yifei Jiang, Fu-Xiang Chen, Zi-Han Wang, Linglin Ju, Lin Chen, Da Fu, En-Yu Liu, Su-Qing Zhang, Wei-Hua Cai
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Although the significance of intratumoral heterogeneity (ITH) has been widely acknowledged in various cancers, its role in pancreatic cancer (PC) remains underexplored and warrants further investigation. Methods Pancreatic cancer transcriptomic data were acquired from the TCGA and GEO databases. The DEPTH2 algorithm, in combination with differential expression analysis, was used to identify genes associated with intratumoral heterogeneity (ITH). We applied univariate Cox regression analysis and multiple machine learning techniques to establish a reliable prognostic model. Patients were then stratified according to their ITH scores, and differences between subgroups were examined through pathway enrichment analysis, immune cell infiltration profiling, and drug response prediction. Furthermore, we conducted subcellular localization and differential analysis of ITH using single-cell data, followed by cell-cell communication analysis to explore interaction relationships and identify key pathways. Results Patients exhibiting lower intratumoral heterogeneity (ITH) levels demonstrated poorer clinical outcomes. The constructed 11-gene signature successfully differentiated individuals into high- and low-risk categories with significant survival differences. Immune profiling revealed notable differences in immune cell composition between the two groups, with patients in the high ITH cohort exhibiting enhanced immune activation. Drug sensitivity analysis indicated a differential response to therapies, with high-risk patients more resistant to certain drugs. Single-cell RNA sequencing identified a greater ITH score in epithelial cells, highlighting key interactions, particularly involving Galectin signaling pathways. Conclusion Our results highlight intratumoral heterogeneity’s prognostic and therapeutic relevance in pancreatic cancer, suggesting its potential utility in guiding individualized treatment approaches. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03080-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c15cd4c65cc7acb2163700609aefaa2af8b33ee" target='_blank'>
              Integrating bulk and single-cell RNA sequencing reveals intratumor heterogeneity phenotypes and immune infiltration in pancreatic cancer
              </a>
            </td>
          <td>
            Li Zhao, Yao Huang, Jiang Li, Rui Xie, Kaiming Li
          </td>
          <td>2025-07-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4879912b1737fce64c962b382e72c481f6e838" target='_blank'>
              Spatial mapping of proteins and their activity states in cancer models by multiplex in situ PLA
              </a>
            </td>
          <td>
            Liza Löf, Bo Xu, Tanay Kumar Sinha, Caroline Dahlström, J. Vennberg, Tore Larsson Forssén, Axel Klaesson, C. Clausson, Xuan Wang, Ulla Strömberg-Olsson, M. Kamali-Moghaddam, C. Avenel, C. Wählby, Agata Zieba-Wicher, Ulf Landegren
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4bb0e4b1912ff07e936c216e052a79b324c47ba" target='_blank'>
              Mechanistic modeling and machine learning identifies optimum radiotherapy schedules to prevent treatment-induced metastasis
              </a>
            </td>
          <td>
            Christopher J Graser, Zhi Zhou, Manuel Schürch, Graydon Moorhead, Justin Dean, Cindy Lin, Jamie Dean, David E. Kozono, G. Lahav, Franziska Michor
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The dysregulated and fibrotic tumor microenvironment of hepatocellular carcinoma (HCC) delays diagnosis and presents many complex signals that drive disease progression. To better recapitulate this microenvironment, we have enhanced our established protein microarray platform by integrating design of experiments (DoE) methodology with high-throughput cell microarray screening. This innovative approach systematically interrogates the intricate roles of matrix stiffness (spanning healthy and fibrotic conditions), extracellular matrix (ECM) composition, and protein concentration, while simultaneously examining their interdependent interactions. By leveraging DoE principles, we were able to explore 117 unique microenvironments on a single microscope slide, ultimately generating a comprehensive dataset of 234 different microenvironments without compromising statistical rigor. Our enhanced screening system enabled the identification of unique microenvironmental interactions critically significant in dictating cellular responses, including adhesion, survival, proliferation, epithelial-to-mesenchymal transition, and drug resistance markers. Utilizing advanced statistical techniques such as linear models and principal component analysis, we characterized phenotypic clusters defined by precise microenvironmental cues. This work presents a robust, high-throughput microarray screening system that comprehensively explores the contributions of 9 physiologically relevant extracellular matrix proteins and matrix stiffness in modulating cellular behavior and disease progression through a methodologically sophisticated and statistically sound approach. .">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d6d43b14b77f59b187769ee40b324e60d7e33b" target='_blank'>
              Enhanced combinatorial analysis of tumor cell-ECM interactions using design-of-experiment optimized microarrays.
              </a>
            </td>
          <td>
            Hannah R C Kimmel, Allison L Paxhia, Zahra Adamji, Gregory H Underhill
          </td>
          <td>2025-07-24</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Primary cicatricial alopecia (PCA) causes irreversible hair loss due to immune-mediated hair follicle destruction. This study investigates follicle-targeted inflammation in lichen planopilaris (LPP), a major PCA subtype, to identify therapeutic targets. Methods Scalp samples from LPP, localized scleroderma (LS), and controls were analyzed using single-cell RNA sequencing and spatial transcriptomics. Cellular composition, spatial localization, and intercellular interactions were examined using differential gene expression and ligand-receptor analyses. Results CD8⁺ effector memory T cells (Tem) and macrophages infiltrated hair follicles in LPP, disrupting immune privilege and promoting scarring. Heightened interferon-γ (IFN-γ) signaling and STAT1 activation in Tem cells caused epithelial-mesenchymal transition (EMT) in hair follicle stem cells (HFSCs). Additionally, macrophage-secreted oncostatin M (OSM) impaired HFSC integrity. These mechanisms drive LPP’s inflammation and fibrosis. Conclusions Our findings identify interferon-γ and oncostatin M as key drivers of LPP pathogenesis, offering targets to reduce follicular scarring and preserve hair growth. Trial registration Not applicable. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01447-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93b14b62279f2fa64aedd48a02134af193bebc08" target='_blank'>
              Single-cell sequencing combined with spatial transcriptomics reveals the characteristics of follicle-targeted inflammation patterns in primary cicatricial alopecia
              </a>
            </td>
          <td>
            Qitao Chen, Yuqian Li, Qilin Zhu, Zhongming Li, Guanghui Shao, Yanjun Liu, Peixuan Jiang, Qiuwei Tao, Lili Shen, Jing Zhu, Linwei Wei, Yanhua Li, Xufeng Du
          </td>
          <td>2025-07-16</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and aggressive primary malignant brain tumor in adults, with limited survival outcomes due to tumor recurrence, mainly driven by GBM cell invasion and therapy resistance. Although temozolomide (TMZ) remains the standard-of-care chemotherapeutic, its long-term efficacy is often compromised by rapid emergence of acquired resistance, largely mediated by the DNA repair enzyme, methylguanine methyltransferase (MGMT). To investigate the interplay between tumor heterogeneity, drug resistance, and the extracellular matrix (ECM) microenvironment, we adapted a 3D methacrylamide-functionalized gelatin (GelMA) hydrogel model to study the behavior of mixed populations of TMZ-sensitive and TMZ-resistant GBM cells. Using both single-cell distributions and multicellular spheroids, we report the impact of heterogeneous cell populations and TMZ dosing regimens, including physiological, supraphysiological, and metronomic TMZ schedules, on drug response and migration. We show that the combination therapy of TMZ with an MGMT inhibitor, lomeguatrib, can modulate TMZ resistance in vitro. This hydrogel model enables systematic investigation of GBM heterogeneity, “go-or-grow” phenotypic plasticity, and therapeutic resistance in an ECM-rich microenvironment, offering a valuable platform for future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbe93715d79528e499ebbccc2bfd3c31f1cbde8" target='_blank'>
              Multicellular Model of Temozolomide Resistance in Glioblastoma Reveals Phenotypic Shifts in Drug Response and Migratory Potential
              </a>
            </td>
          <td>
            Victoria Kriuchkovskaia, Ela K. Eames, Sydney A McKee, Paul J. Hergenrother, Rebecca B. Riggins, Brendan A. C. Harley
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PDAC) is a rapidly progressing cancer that responds poorly to immunotherapies. Intratumoral tertiary lymphoid structures (TLS) have been associated with rare long-term PDAC survivors, but the role of TLS in PDAC and their spatial relationships within the context of the broader tumor microenvironment remain unknown. Herein, we report the generation of a spatial multi-omics atlas of PDAC tumors and tumor-adjacent lymph nodes from patients treated with combination neoadjuvant immunotherapies. Using machine learning-enabled hematoxylin and eosin image classification models, imaging mass cytometry, and unsupervised gene expression matrix factorization methods for spatial transcriptomics, we characterized cellular states within and adjacent to TLS spanning across distinct spatial niches and pathologic responses. Unsupervised learning identified TLS-specific spatial gene expression signatures that significantly associated with improved survival in PDAC patients. We identified spatial features of pathologic immune responses, including intratumoral TLS-associated B-cell maturation colocalizing with IgG dissemination and extracellular matrix remodeling. Our findings offer insights into the cellular and molecular landscape of TLS in PDACs during immunotherapy treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f63086cbccde700980caecaca85ce93b724792a" target='_blank'>
              Neoadjuvant immunotherapy promotes the formation of mature tertiary lymphoid structures in a remodeled pancreatic tumor microenvironment.
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, J. Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, G. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, E. Jaffee, Lei Zheng, W.J. Ho, Robert A. Anders, E. Fertig, L. T. Kagohara
          </td>
          <td>2025-08-15</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) has emerged as a powerful tool to map gene expression patterns to the local tissue structure in cancer, enabling unprecedented insights into cellular heterogeneity and tumour microenvironments. As the technology matures, developing new, spatially informed analytical frameworks will be essential to fully leverage its potential to elucidate the complex organisation and emerging properties of cancer tissues. Here, we highlight key challenges in cancer spatial transcriptomics, focusing on three emerging topics: (a) defining cell states, (b) delineating cellular niches and (c) integrating spatial data with other modalities that can pave the way towards clinical translation. We discuss multiple analytical approaches that are currently implemented or could be adapted in the future in order to tackle these challenges, including classical biostatistics methods as well as methods inherited from geospatial analytics or artificial intelligence. In the rapidly expanding landscape of ST, such methodologies lay the foundation for biological discoveries that conceptualise cancer as an evolving system of interconnected niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9031fde1ffd2ce70d2c223e1b5d79202f9418532" target='_blank'>
              Decrypting cancer's spatial code: from single cells to tissue niches.
              </a>
            </td>
          <td>
            Cenk Celik, Shi Pan, Eloise Withnell, Hou Wang Lam, M. Secrier
          </td>
          <td>2025-07-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer cell morphology reflects the dynamic interplay between genetic, epigenetic, and environmental factors that
drive tumor progression. This review explores the morphological evolution of cancer cells as a phenotypic manifestation of
underlying molecular dynamics, drawing on recent advances in evolutionary biology, cytoskeletal remodeling, computational
pathology, and tumor metabolism. Clonal evolution, both gradual and punctuated, generates morphological diversity that is
shaped by selective pressures from the tumor microenvironment and therapeutic interventions. Studies highlights the structural
and mechanical properties, such as reduced cellular stiffness and altered actin organization, are linked to metastatic potential
and cytoskeletal deregulation. Oncogenic signaling pathways (e.g., PI3K/AKT, Ras/MAPK) and metabolic reprogramming
further modulate cell shape and behavior. High-throughput imaging and machine learning have enabled quantification of both
static and dynamic morphological traits, correlating them with chromosomal instability and clinical outcomes. Morphological
heterogeneity has emerged as a powerful diagnostic and prognostic biomarker, with deep learning-derived metrics providing
scalable tools for cancer classification and risk stratification. Collectively, these insights underscore the clinical utility of
integrating morphological analysis with molecular profiling, offering new directions for personalized cancer diagnosis and
therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055067ac5bdde412ce5a95b6486dff3c99db46a4" target='_blank'>
              Morphology Matters: A Multidimensional Perspective on Cancer Cell Biology
              </a>
            </td>
          <td>
            Rose Rani
          </td>
          <td>2025-06-30</td>
          <td>International Journal for Research in Applied Science and Engineering Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutated oncogenic Kirsten rat sarcoma virus (KRAS) antigen is expressed in a large variety of cancers, including pancreatic, colorectal, and pulmonary cancers. The oncogenic KRAS mutations cause malignancies and are usually ubiquitously expressed by all cells of a tumor. The KRAS amino acid substitutions at the positions 12 or 13 are among the most frequent mutations in human cancers. Here, we developed immuno-oncotherapeutic non-integrative lentiviral vectors encoding a segment encompassing KRASG12D, either alone, or associated with antigen carriers. These carriers can improve the intracellular antigen routing to major histocompatibility complex presentation machineries or provide universal helper CD4+ epitopes. Immunotherapy with one of these vectors resulted in significant immune control of tumor growth in colorectal or pulmonary preclinical cancer models, in several murine genetic backgrounds. The antitumor effect was correlated with increased proportions of intra-tumoral hematopoietic cells and notably CD8+ T cells. Although this effect was partial, it was robust, reproducible and advantageously combinable with conventional chemotherapies and immunotherapies to improve antitumor protection. Therefore, this approach shows promise as an immuno-oncotherapy against KRAS-mediated malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61951127f6ba1a1f776a4ed744c835e44ef0de6c" target='_blank'>
              A lentiviral vector targeting a KRAS neoepitope for cancer immunotherapy
              </a>
            </td>
          <td>
            Anastasia Goloudina, Fabien le Chevalier, P. Authié, Sylvain Ciret, K. Nemirov, Ingrid Fert, F. Moncoq, B. Vesin, A. Noirat, Catherine Blanc, Yu Wei, Pierre Charneau, L. Majlessi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Despite substantial progress in discovering novel therapeutic targets and compounds, cancer hyperproliferative cells (CHCs) play a central role in driving tumor progression in gastric cancer (GC). High-resolution single-cell RNA sequencing (scRNA-seq) integrated with spatial transcriptomics sequencing (ST-seq) elucidating the role of the CHC microenvironment in driving GC progression remains lacking. Methods Through integrated analysis of 40 scRNA-seq samples and 6 ST samples from human gastric tissues spanning tumor progression, we delineate the phenotypic plasticity of tumor epithelium and characterize the transcriptional trajectory from normal gastric cells to CHCs, a process associated with RRM2 overexpression. Finally, we validated the analysis results through cellular and animal experiments. Results Our findings demonstrate that RRM2 knockdown triggers ferroptosis in GC cells by promoting lipid peroxidation and intracellular iron accumulation. We further screen a compound and RRM2 inhibitor, osalmid, which effectively suppresses tumor growth in GC cell xenografts by triggering ferroptosis, thus offering a new therapeutic option for GC. Conclusions This study elucidates the critical role of RRM2 + CHCs in gastric cancer development and progression, providing novel insights for targeted therapies. The discovery of osalmid as a potential therapeutic agent offers a promising new treatment strategy for GC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06847-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad62270a5ce806049ffe5312712ee483297a8c1" target='_blank'>
              Integration of scRNA-seq and ST-seq identifies hyperproliferative RRM2+ cells features and therapeutic targets in gastric cancer
              </a>
            </td>
          <td>
            Shuai Ping, Xiong Jia, Yanan Tian
          </td>
          <td>2025-07-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="As drug and natural product libraries expand, assays for assessing mechanisms of action (MoA) are increasingly critical. Performing cytological profiling using the Cell Painting (CP) assay enables image-based profiling of cellular states upon treatment, yet many bioactive compounds remain uncharacterized due to undetectable cellular effects under standard conditions. To address this, we combined drug dosing with cell activation using the protein kinase C (PKC) agonist phorbol myristate acetate (PMA). Profiling A549 lung cancer cells treated with 8,387 compounds at two concentrations (1 and 10 μM) in both resting and PMA-activated states allowed us to detect phenotypic effects for up to 40% of all screened compounds, effectively illuminating new phenotypic “dark space”. Over 1,000 compounds exhibited phenotypes exclusively under PMA activation, establishing its advantage for MoA studies. We introduce novel quality control measures for CP screens and demonstrate that integrating phenotypic signatures enhances MoA discovery. Notably, 2-methoxycinnamaldehyde clustered with glucocorticoid receptor modulators and induced nuclear translocation, emphasizing the power of this approach in uncovering novel drug mechanisms and, therefore, aiding in improving therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a69a88bdf0eb86999887480ebc0a5b9adfd4cbb0" target='_blank'>
              Cell painting in activated cells illuminates phenotypic dark space and uncovers novel drug mechanisms of action
              </a>
            </td>
          <td>
            Matylda Zietek, Akshar Lohith, Derfel Terciano, Beverley Marie Rabbitts, Aswad S. Khadilkar, John B MacMillan, Scott Lokey
          </td>
          <td>2025-06-23</td>
          <td>Research Square</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67b3f2125e256fc001c7ec119739d1e135a5d03" target='_blank'>
              A Mathematical Model of Persister Cell Plasticity and Its Impact on Adaptive Cancer Therapy
              </a>
            </td>
          <td>
            Adithya Mathews, D. Basanta
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Dendritic cells (DCs) are pivotal orchestrators of anti-tumor immunity. DC-based anti-tumor treatments are being actively developed, but effective clinical responses have not yet been achieved. Further exploration of DC heterogeneity in the tumor microenvironment (TME) and across cancer types could provide insights for developing DC-based immunotherapies. Here, we integrated single-cell RNA sequencing data of DCs from over 2,500 samples across 33 cancer types and established a comprehensive blueprint of human DCs. Several rare subsets of DCs infiltrated the tumors, including AXL+SIGLEC6+ (AS) DCs and Langerhans cell (LC)-like DCs, and displayed functional potentials marked with distinct transcriptomic characteristics. Computational analyses demonstrated that the LC-like subset could be an additional cellular origin of tumor-enriched LAMP3+ DCs and that distinct cellular origins are associated with the pleiotropic functional potentials of LAMP3+ DCs. Furthermore, this DC atlas enabled development of a machine learning model to guide DC annotation for subsequent single-cell analysis and prioritization of a valuable target for enhancing anti-tumor DC vaccination. This integrative resource provides a panoramic view to unravel the complexity of tumor-infiltrating DCs and offers valuable insights for developing therapies targeting DCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20198842496886b53a9ff7adf340fb98f1bc673" target='_blank'>
              Pan-cancer Analyses Refine the Single-Cell Portrait of Tumor-Infiltrating Dendritic Cells.
              </a>
            </td>
          <td>
            Tianyi Ma, Xiaojing Chu, Jinyu Wang, Xiangjie Li, Yu Zhang, Dan Tong, Wenbin Xu, Guohui Dang, Lu Qi, Yuhui Miao, Zemin Zhang, Si Cheng
          </td>
          <td>2025-07-31</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cutaneous melanoma is an aggressive cancer with an increasing incidence worldwide, highlighting the need for research into its pathogenesis. The tumor microenvironment (TME) plays a critical role in melanoma progression and consists of cellular components and an extracellular matrix (ECM) rich in cytokines and signaling molecules. The most abundant stromal cells within the TME are cancer-associated fibroblasts (CAFs), which remodel the ECM and modulate immune responses. Among immune cells, tumor-associated macrophages (TAMs) predominate, and their polarization toward the M2 phenotype supports tumor progression. Tumor-infiltrating lymphocytes (TILs) have diverse functions, including cytotoxic T-cells, helper T-cells that modulate immune response, B-cells forming tertiary lymphoid structures (TLS), and regulatory T-cells with immunosuppressive properties. Dendritic cells (DCs) also play a complex role in the TME. A notable subpopulation are mature regulatory dendritic cells (mregDCs), which contribute to immune evasion. All of these TME components may drive tumorigenesis. Advancements in melanoma treatment—including immunotherapy and targeted therapies—have significantly improved outcomes in advanced-stage disease. In parallel, emerging approaches targeting the tumor microenvironment and gut microbiome, as well as personalized strategies such as neoantigen vaccines and cell-based therapies, are under active investigation and may further enhance therapeutic efficacy in the near future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d1c207a44eb08e471b7d10318dde99844e3005" target='_blank'>
              Tumor Microenvironment in Melanoma—Characteristic and Clinical Implications
              </a>
            </td>
          <td>
            Hubert Sikorski, M. Żmijewski, Anna Piotrowska
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common primary brain cancer. It causes death mainly by local invasion via several routes, including infiltration of white matter tracts and penetration of perivascular spaces. However, the pathways that mediate these invasion routes are only partly known. Here, we conduct an integrative study to identify cell states and central drivers of route-specific invasion in GBM. Combining single-cell profiling and spatial protein detection in patient-derived xenograft models and clinical tumor samples, we demonstrate a close association between the differentiation state of GBM cells and their choice of invasion route. Computational modeling identifies ANXA1 as a driver of perivascular involvement in GBM cells with mesenchymal differentiation and the transcription factors RFX4 and HOPX as orchestrators of growth and differentiation in diffusely invading GBM cells. Ablation of these targets in tumor cells alters their invasion route, redistributes the cell states, and extends survival in xenografted mice. Our results define a close association between GBM cell differentiation states and invasion routes, identify functional biomarkers of route-specific invasion, and point toward targeted modulation of specific invasive cell states as a therapeutic strategy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd579d99eaa69724a2c2c47d55f912d2748b0a9" target='_blank'>
              The invasion phenotypes of glioblastoma depend on plastic and reprogrammable cell states
              </a>
            </td>
          <td>
            Milena Doroszko, Rebecka Stockgard, Irem Uppman, Josephine Heinold, Faidra Voukelatou, H. Mangukiya, T. Millner, Madeleine Skeppås, Mar Ballester Bravo, Ramy Elgendy, Maria Berglund, Ludmila Elfineh, C. Krona, Soumi Kundu, Katarzyna Koltowska, Silvia Marino, Ida Larsson, Sven Nelander
          </td>
          <td>2025-07-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND
Atypical teratoid rhabdoid tumors (ATRTs) are highly aggressive pediatric central nervous system tumors defined by the inactivation of the SMARCB1 gene. Despite the identification of three distinct molecular subtypes, each defined by unique clinical and molecular characteristics, no subtype-specific therapeutic strategies are currently available. This highlights an urgent need to deepen our understanding of the cellular heterogeneity and developmental origins of ATRTs.


METHODS
We generated a comprehensive single-nucleus transcriptomic atlas of ATRT samples, integrated it with single-nucleus ATAC-seq and spatial transcriptomics data, and validated our findings experimentally using patient-derived ATRT tumoroid models.


RESULTS
Our analyses revealed distinct subtype-specific differentiation trajectories, each resembling different brain progenitor lineages. We identified key transcription factors that appear to drive these developmental pathways. Furthermore, a shared cycling, intermediate precursor cell (IPC)-like cell population, interspersed throughout tumors, was consistently present within all ATRT samples. We demonstrate that these subtype-specific differentiation pathways can be pharmacologically manipulated in patient-derived ATRT tumoroids. By directing tumor cells along their respective subtype-specific trajectories, we were able to induce a shift toward more differentiated, non-proliferative states.


CONCLUSIONS
Collectively, our findings show that ATRTs recapitulate fetal brain signaling programs in a subtype-specific manner. This work provides a framework for understanding ATRT heterogeneity and supports the feasibility of maturation-based therapeutic strategies tailored to the molecular subtype of the tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f089b7cc6f0aae306ce7c5e4e459d6bc20cd73eb" target='_blank'>
              A cycling, progenitor-like cell population at the base of atypical teratoid rhabdoid tumor subtype differentiation trajectories.
              </a>
            </td>
          <td>
            Enrique Blanco-Carmona, Irene Paassen, Jiayou He, Jeff DeMartino, Annette Büllesbach, Nadia Anderson, Juliane L Buhl, A. Federico, Monika Mauermann, Mariël Brok, Karin Straathof, S. Behjati, Rajeev Vibhakar, A. Donson, Nicholas K Foreman, McKenzie L. Shaw, M. Frühwald, A. Korshunov, M. Hasselblatt, C. Thomas, Niels Franke, M. Kranendonk, E. Hoving, N. Jäger, P. Johann, Stefan M Pfister, Mariella G. Filbin, M. Kool, J. Drost
          </td>
          <td>2025-07-28</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common and deadly malignancies worldwide, with a particularly low 5-year survival rate in advanced patients. Immune cells in the tumor microenvironment, especially mast cells, play crucial roles in tumor initiation and progression. However, the dual role of mast cells in CRC remains poorly understood. Methods In this study, we used single-cell RNA sequencing (scRNA-seq), bulk RNA sequencing, and bioinformatics analyses to explore the heterogeneity of mast cell subpopulations in the CRC tumor microenvironment and their relationship with prognosis. We analyzed gene expression signatures associated with mast cell subpopulations derived from single-cell data of 40 CRC tumor samples and combined bulk RNA-seq data from HMU, GEO, and TCGA cohorts for prognostic prediction. Non-negative matrix factorization was used for clustering of mast cell subpopulations, followed by analysis of their specific gene markers, transcription factor activity, and biological pathways. Survival analysis and ROC curves were performed to assess their prognostic significance. Results Mast cells in the CRC tumor microenvironment were classified into three distinct subpopulations, each with unique gene markers and functional pathways. Mast cell subpopulations 1 and 3 were highly associated with pro-tumor pathways, while mast cell subpopulation 2 primarily exhibited anti-tumor immune regulatory characteristics. High expression of mast cell subpopulations 1 and 3 was associated with poor survival prognosis, while high expression of subpopulation 2 was linked to a better survival outcome. Key marker genes such as DNAJB1, SEMA7A, and XCR1 were identified as potential prognostic factors, with high expression of DNAJB1 and SEMA7A being significantly associated with poor prognosis, while high expression of XCR1 was linked to a favorable prognosis. Conclusion This study reveals the functional heterogeneity of mast cell subpopulations in the CRC tumor microenvironment and their differential roles in tumor progression. Identification of mast cell subpopulation-specific marker genes provides new molecular targets for clinical diagnosis, prognostic prediction, and personalized immunotherapy in CRC. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04119-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f36d01ec68db5890c180d0142a0a3d3d4bc0a598" target='_blank'>
              Diversity of mast cell subpopulations in the tumor microenvironment of colorectal cancer and their prognostic implications
              </a>
            </td>
          <td>
            Tian-yu Qiao, Chao Ding, Songtao Yu, Wenyang Li, Yonghou Zhao, Guiyu Wang
          </td>
          <td>2025-06-30</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Embryonal tumor with multilayered rosettes (ETMR) is a lethal embryonal brain tumor entity. To investigate the intratumoral heterogeneity and cellular communication in the tumor microenvironment (TME), we analyze in this work single-cell RNA sequencing of about 250,000 cells of primary human and murine ETMR, in vitro cultures, and a 3D forebrain organoid model of ETMR, supporting the main findings with immunohistochemistry and spatial transcriptomics of human tumors. We characterize three distinct malignant ETMR subpopulations - RG-like, NProg-like and NB-like - positioned within a putative neurodevelopmental hierarchy. We reveal PDGFRβ+ pericytes as key communication partners in the TME, contributing to stem cell signaling through extracellular matrix-mediated interactions with tumor cells. PDGF signaling is upregulated in chemoresistant RG-like cells in vivo and plays a role in recruiting pericytes to ETMR TME by finalizing a signaling cascade which promotes the differentiation of non-malignant radial glia cells, derived from our 3D model, into pericyte-like cells. Selective PDGFR-inhibition blocked the lineage differentiation into pericytes in vitro and reduced the tumor cell population in vivo. Targeting ETMR-pericyte interactions in the TME presents a promising therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/497d2ed06cfc62eb4cc72976940d7e3d7d08672f" target='_blank'>
              ETMR stem-like state and chemo-resistance are supported by perivascular cells at single-cell resolution
              </a>
            </td>
          <td>
            F. D. de Faria, Nicole C. Riedel, Daniel Münter, M. Interlandi, C. Göbel, L. Altendorf, Mathis Richter, V. Melcher, C. Thomas, Rajanya Roy, Melanie Schoof, Ivan Bedzhov, N. Moreno, M. Graf, M. Hotfilder, D. Holdhof, Wolfgang Hartmann, Ann-Katrin Bruns, A. Brentrup, F. Liesche-Starnecker, B. Maerkl, S. Sandmann, J. Varghese, M. Dugas, Pedro H Pinto, S. Balbach, I-Na Lu, Claudia Rossig, Oliver Soehnlein, Aysegül Canak, Martin Ebinger, Martin U Schuhmann, J. Schittenhelm, Michael F Frühwald, U. Schüller, T. Albert, Carolin Walter, Jan M Bruder, K. Kerl
          </td>
          <td>2025-06-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) represent a distinct subpopulation of tumor cells characterized by enlarged or multiple nuclei and aneuploidy. PGCCs are products of genomic instability, possessing cancer stem cell properties and exhibiting significant resistance to radiotherapy and chemotherapy. They can generate highly invasive daughter cells through asymmetric division, exhibiting epithelial-mesenchymal transition characteristics, and facilitating tumor recurrence and metastasis. In vivo, PGCCs with daughter cells in tumor tissue can migrate and infiltrate into the forefront stroma to form tumor budding, which are closely related to solid tumor recurrence, metastasis, and drug resistance. Studies have shown that inhibiting sphingolipid enzyme acid ceramidase or regulating autophagy can reduce the production of PGCCs with daughter cells. Under appropriate induction conditions, PGCCs with daughter cells can be induced to differentiate into benign tissues such as adipocytes, chondrocytes, and osteocytes, inhibiting their malignant proliferation and invasive destruction. This study reviewed the recent research developments regarding PGCCs, mainly explored the endogenous mechanisms of PGCCs formation and their malignant phenotype, as well as the process of tumor budding formation in vivo and potential therapeutic strategies targeting PGCCs. The main novelty of this study lies in exploring the translation of PGCCs basic research into the clinical pathological prognostic role of tumor budding, which can reveal the potential mechanism of PGCCs/tumor budding formation at the molecular level, providing theoretical basis for prognosis assessment, monitoring of recurrence and metastasis risks, as well as improving drug resistance and targeted therapy in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a3661c8ef6e9fb0fa2a5fd9bf5fffcb0f097f8" target='_blank'>
              Polyploid giant cancer cells and tumor budding: translation from basic research to clinical application
              </a>
            </td>
          <td>
            Peng Huang, Rong Wu, Zhimou Yang, Yuwei Li, Fei Fei, Yongjun Yu
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor characterized by an immunosuppressive microenvironment that importantly contributes to treatment resistance. Monocyte-derived macrophages (MDMs), which comprise approximately 50% of the cellular population within the GBM microenvironment, represent a major subset of tumor-associated macrophages. These cells drive tumor progression by promoting angiogenesis, immune evasion, and the phenotypic transformation of tumor cells. MDM infiltration is mediated by specific signaling pathways and regulated by the disruption of the blood–brain barrier and tumor-associated hypoxia. Recent technological advances have uncovered substantial heterogeneity among macrophages, including hypoxia-induced, lipid-metabolizing, phagocytic, and interferon-activated subtypes. This functional diversity is shaped by tumor-specific genetic alterations and metabolic reprogramming. Therapeutic approaches focusing on MDMs include inhibiting their recruitment, enhancing phagocytic activity, employing genetically engineered macrophage, and modulating metabolic pathways. While preclinical studies suggest that these approaches may improve efficacy when combined with immune checkpoint inhibitors, the dynamic spatiotemporal heterogeneity and adaptability of macrophages within the tumor microenvironment remain substantial therapeutic challenges. Future development in combination therapies, integrating single-cell multi-omics, spatial metabolic profiling, and targeted interventions, will be critical to precisely modulate MDMs, overcome immune tolerance, and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e73dae7727564fcd760697c5cdc2cc02393ac584" target='_blank'>
              New Insights into Monocyte-Derived Macrophages in Glioblastoma
              </a>
            </td>
          <td>
            Xuetong Li, Wei Gao, Xinmiao Long, Minghua Wu
          </td>
          <td>2025-07-23</td>
          <td>Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Immune checkpoint inhibitors show limited efficacy in tumors with low tumor mutational burden, partly due to insufficient neoantigen presentation. Methods We developed a novel approach for neoantigen identification using circulating tumor cells (CTCs) isolated via leukapheresis and flow cytometry. Peripheral blood mononuclear cells (PBMCs) were collected from 11 stage IV cancer patients and 2 healthy volunteers. CTCs were enriched by depleting CD45+ hematopoietic cells and selecting CD45−Vimentin+ cells, which were confirmed cytologically to contain malignant cells. Hematopoietic lineage analysis showed that over 50% of the CTC fraction consisted of non-hematopoietic cells. DNA extracted from both the CTC and normal hematopoietic fractions underwent exome sequencing. Neoantigens were identified using the Ancer® bioinformatics platform. Results In representative patients with gastric and salivary gland cancers, 94,636 and 46,423 CTCs were isolated, respectively. DNA yields were sufficient for exome sequencing without amplification or extensive cell culture. A total of 102 (patient with gastric cancer) and 108 (patient with salivary gland cancer) neoantigens were identified in each subject, including high-ranking T-cell epitopes derived from single nucleotide variants and frameshift mutations. According to the same procedures we could successfully identify a large number of neoantigens from the CTCs of all stage IV cancer patients. This confirms the feasibility of identifying individual patient-specific neoantigens from CTCs without requiring tumor biopsies. Conclusions This is the first study to demonstrate successful neoantigen identification using non-amplified CTCs isolated by apheresis and flow cytometry. The approach provides a minimally invasive, scalable alternative for neoantigen discovery and may better capture tumor heterogeneity compared to single-site biopsies. This method holds promise for enabling rapid, personalized immunotherapy strategies, including peptide vaccines, dendritic cell vaccines, and mRNA-based treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f2f611c004eda4f555a0f11de496ee09c9cb0f" target='_blank'>
              A method for identifying neoantigens through isolation of circulating tumor cells using apheresis among patients with advanced-stage cancer
              </a>
            </td>
          <td>
            Daiki Kobayashi, Takuya Kosumi, Queenie Lai Kwan Lam, Shigeharu Fujita, Yasuki Hijikata, Kaori Takeda, Tomoya Narita, Naomi Yamashita, Guilhem Richard, Anne S De Groot, Naohide Yamashita
          </td>
          <td>2025-07-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d94bb593ee184379443d2fb409c739871b84fe" target='_blank'>
              tugMedi: simulator of cancer-cell evolution for personalized medicine based on the genomic data of patients
              </a>
            </td>
          <td>
            Iurii S. Nagornov, E. Furukawa, M. Nagai, S. Yagishita, Tatsuhiro Shibata, M. Kato
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Invasive lobular carcinoma (ILC) is a common subtype of breast cancer that is defined in part by genetic loss of CDH1 caused by mutation or deletion, leading to loss of cell adhesion protein E-cadherin in >90% of ILC. Genetic loss of CDH1 is an early event in ILC oncogenesis, yet the mechanisms by which CDH1/E-cadherin acts as a tumor suppressor are not well understood. To study how early CDH1 loss drives ILC oncogenesis, we used a series of non-transformed human mammary epithelial cell (HMEC) models to target CDH1/E-cadherin, inhibiting extracellular E-cadherin signaling using antibodies versus modeling genetic CDH1 loss using siRNA or knockout via CRISPR/Cas9. Through transcriptome analyses across four HMEC models, we found that the mode of E-cadherin loss or suppression is critical for the subsequent phenotype. Antibody-mediated inhibition of cell-cell contacts induced gene signatures of epithelial-mesenchymal transition (EMT), consistent with the role of E-cadherin suppression during the EMT process. Conversely, genetic CDH1 loss – as in ILC oncogenesis – repressed EMT signatures, and instead remodeled gene expression toward a luminal epithelial phenotype. Using single cell transcriptomics and flow cytometry analyses of cell lineage markers, we found that genetic loss of CDH1 reprogrammed cells to a luminal progenitor-like phenotype. By isolating luminal versus basal cells prior to CDH1 knockout, we found that CDH1 loss led to remodeling of lineage identity in both populations, converging on a new lineage homeostasis with a luminal progenitor-like phenotype. Consistent with increased progenitor features, CDH1 loss enhanced proliferative capacity over the finite lifespan of the HMECs, highlighting a feature of early CDH1 loss that may contribute to clonal advantage during tumor initiation. Our findings support that inhibition of E-cadherin results in different transcriptional response compared to CDH1 loss, with the latter driving a transcriptional and phenotypic state characteristic of a luminal progenitor-like population, which offers new insight into early events in ILC oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55a904a9de6859d0ee1985b86b2f0ef97b6b1ec" target='_blank'>
              CDH1 loss remodels gene expression and lineage identity in human mammary epithelial cells
              </a>
            </td>
          <td>
            Maggie Musick, Chinasa A Ufondu, Carmen E. Rowland, Joseph L. Sottnik, Madeleine T. Shackleford, Camryn S. Nesiba, J. Ostrander, Matthew J. Sikora
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Neuroendocrine (NE) prostate cancer (NEPC) is an aggressive and lethal subtype of prostate cancer. It is typically characterized by the expression of NE markers and the loss of androgen receptor expression. De novo NEPC is rare, accounting for <2% of all prostate cancer cases at diagnosis. More commonly, NEPC arises from prostate adenocarcinoma following androgen deprivation therapy, with 20–25% of metastatic castration-resistant prostate cancers undergoing NE differentiation due to lineage plasticity. During this transition, pathways associated with epithelial-mesenchymal transition (EMT) and stemness are broadly activated, which is considered to be a key driver of NEPC's high metastatic potential, resistance to chemotherapy and radiotherapy and poor prognosis. EMT facilitates metastasis by enhancing cellular motility and invasiveness, while stemness properties contribute to post-metastatic colonization, immune evasion, therapy resistance and cellular dormancy. As manifestations of cellular plasticity, these processes share overlapping molecular mechanisms. Targeting key regulators within these pathways may offer promising therapeutic strategies for NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30582503232b9c92c62c32d091a4ab49e49ea5bf" target='_blank'>
              Widespread activation and critical role of EMT and stemness in the neuroendocrine differentiation of prostate cancer (Review)
              </a>
            </td>
          <td>
            Yun-Fan Li, Shuai Su, Yu Luo, Chengcheng Wei, Jingke He, Liangdong Song, Kun Han, Jue Wang, Xiangzhi Gan, Delin Wang
          </td>
          <td>2025-07-04</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd9e961ad1f345af55769759a723ac6d14212120" target='_blank'>
              Integrated genetic screening reveals FEN1 as a driver of stemness and temozolomide resistance in glioblastoma
              </a>
            </td>
          <td>
            Benjamin A Brakel, D. Mckenna, Anish Puri, V. Shaikh, Manoj Singh, Ali Saleh, Abdo-jose Tomajian, N. Mikolajewicz, Marcello Beltrami, Alisha Anand, P. Miletic, Kevin R. Brown, D. Tieu, W. Maich, Sabra K. Salim, Yujin Suk, M. Subapanditha, Deena M. A. Gendoo, C. Venugopal, Jason Moffat, Sachin Katyal, C. Chokshi, Sheila K Singh
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a highly heterogeneous disease, with castration-resistant prostate cancer (CRPC) and neuroendocrine prostate cancer (NEPC) representing its most aggressive and therapy-resistant forms. Emerging evidence indicates that lineage plasticity—driven by key transcription factors such as Octamer Binding Factor 4 (OCT4)—plays a crucial role in therapeutic resistance and disease progression. OCT4, in coordination with SOX2 and NANOG, acts as a master regulator of stemness and is frequently upregulated in prostate cancer stem cells (PCSCs). This upregulation contributes to tumor initiation, metastasis, and resistance to both androgen deprivation therapy (ADT) and chemotherapy. In this review, we explore the role of OCT4 in mediating lineage plasticity in prostate cancer, with particular emphasis on its involvement in treatment resistance and neuroendocrine differentiation. We also examine therapeutic strategies aimed at targeting OCT4 directly, such as microRNA-mediated suppression, small-molecule inhibitors, and suicide gene therapy, as well as indirect approaches that modulate OCT4 expression via FGFR and NF-κB signaling pathways. While these strategies offer promising avenues, challenges such as adaptive resistance and the intricate signaling networks within PCSCs remain significant hurdles. A deeper understanding of the molecular mechanisms underlying OCT4-driven plasticity may pave the way for novel therapeutic approaches and improved outcomes in advanced prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a8ce283789b3c96575337b7647df8d11de9b2ba" target='_blank'>
              Unlocking the Role of OCT4 in Cancer Lineage Plasticity: A Cross-Cancer Perspective with an Emphasis on Prostate Cancer
              </a>
            </td>
          <td>
            Mohammad Esfini Farahani, Yanquan Zhang, A. O. Akinyemi, Fatemeh Seilani, Md Rakibul Alam, Xiaoqi Liu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a3488612e1158a55dd4c8b299433b0f980348d5" target='_blank'>
              Molecular signature of human endometrial stem/progenitor cells at the single cell level
              </a>
            </td>
          <td>
            Harriet C. Fitzgerald, S. Mortlock, C. Filby, Fiona L. Cousins, Elizabeth Marquez-Garcia, K. A. Wyatt, Brett McKinnon, Luk Rombauts, J. Tsaltas, G. Montgomery, Caroline E Gargett
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Recent advances in single-cell technologies have enabled the creation of comprehensive cell atlases, reference maps of various cell types within organisms. Here we specifically focus on T cell atlases, which offer a detailed catalog of the adaptive immune system at single-cell resolution. As such, they capture cellular diversity, functional states, and spatial dynamics across tissues, developmental stages, and disease conditions. Given the central role of T cells in orchestrating immune responses, their dysregulation underpins autoimmune disorders, cancer progression and failed immunotherapies. Therefore, a unified T cell atlas is critical for decoding such disease mechanisms, identifying therapeutic targets, and advancing personalized treatments. In this article, we explore the latest advances in T cell atlases, describing breakthroughs in multi-omics technologies, spatial profiling and computational frameworks that resolve transcriptional, epigenetic and proteomic heterogeneity. We also address persistent challenges and highlight strategies to address these gaps. Finally, we discuss emerging frontiers set to reshape our understanding of T cell dynamics in both health and diseases. Together, these insights underscore the transformative potential of T cell atlases in reconstructing precision immunology and accelerating therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d0b0e5dda9a3845db88c70250149559b1315699" target='_blank'>
              Sketching T cell atlases in the single-cell era: challenges and recommendations.
              </a>
            </td>
          <td>
            Itana Bojović, António G. G. Sousa, S. Junttila, L. Elo
          </td>
          <td>2025-06-29</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The intrinsic complexity of biological processes often hides the role of dynamic microenvironmental cues in the development of pathological states. Microphysiological systems (MPSs) are emerging technological platforms that model in vitro dynamics of tissue-specific microenvironments, enabling a holistic understanding of pathophysiology. In our previous works, we engineered and used breast tumor MPS differing in matrix stiffness, pH, and fluid flow mimicking normal and tumor breast tissue. High-dimensional data using two distinctive human breast cell lines (i.e., MDA-MB-231, MCF-7), investigating cell proliferation, epithelial-to-mesenchymal transition (EMT), and breast cancer stem cell markers (B-CSC), were obtained from breast-specific microenvironments. Recognizing that the widespread adoption of MPS requires tailoring its complexity to application demands, we herein report an innovative machine-learning (ML)-based approach to analyze MPS data. This approach uses unsupervised k-means clustering and feature extraction to inform on key markers and specific microenvironments that distinguish invasive from non-invasive breast cell phenotypes. This data-driven approach streamlines future experimental design and emphasizes the translational potential of integrating MPS-derived insights with ML to refine prognostic tools and personalize therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d04f6b70ba36ca6bdecf1ece254a7da2cd8bdc7c" target='_blank'>
              Data-Driven Sustainable In Vitro Campaigns to Decipher Invasive Breast Cancer Features.
              </a>
            </td>
          <td>
            Lekha Shah, V. Breschi, Annalisa Tirella
          </td>
          <td>2025-07-25</td>
          <td>ACS biomaterials science & engineering</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d4e9d24e9c0c4b081e25cd4410c5971c24a9285" target='_blank'>
              Microenvironmental TGF-β is an early driver of NF1-associated tumour formation
              </a>
            </td>
          <td>
            Alex Power, Salome Stierli, Elizabeth Harford-Wright, Guillem Mòdol-Caballero, Emma Lloyd, Stella Kouloulia, Giulia Casal, Lara Krasinska, Omar Bouricha, Ilaria Napoli, Sara Ribeiro, C. Venturini, M. Clements, S. Parrinello, Alison C. Lloyd
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC), a prevalent and highly lethal malignancy, is notorious for its aggressive nature and inherent tendency to metastasize, posing significant challenges in clinical management and prognosis. Hypoxia, a pivotal characteristic of the tumor microenvironment (TME) in hepatocellular HCC, is intimately linked to disease progression and unfavorable patient outcomes, underscoring its critical role in shaping the malignant behavior of this cancer. Methods Our research leveraged single-cell RNA sequencing technology to dissect the heterogeneity of the HCC TME, focusing on hypoxia-related genes. To probe the effects of hypoxia on HCC invasion, we developed the Cell Hypoxia-Related Prognostic Feature (CHPF). We analyzed transcriptome data from the The Cancer Genome Atlas (TCGA) database and the GSE149614 dataset, employing computational methods such as UMAP, Weighted correlation network analysis (WGCNA), and CellChat to identify hypoxia cells, characterize cell subsets, and elucidate intercellular communications. Results Our analysis revealed significant heterogeneity in hypoxia cell populations within the HCC TME, with distinct expression patterns of hypoxia-related genes in neoplastic and immune cells. Our analysis revealed distinct hypoxia subpopulations within HCC, with significant overexpression of genes like MEG3, KLF6 and JUN in hypoxia cells. We identified a unique hypoxia subpopulation with high invasive potential and constructed a prognostic model based on H2-specific transcription factors including LRP10、MED8、NOL10、NOP58 and REXO4. The model demonstrated significant predictive value for lifespan of patients as verified in the TCGA dataset and an external validation group. Conclusion Key transcription factors like NOP58, MED8 play pivotal roles in hypoxia-induced HCC invasion and metastasis, and a predictive model based on these factors forecasts HCC survival. Our findings provide novel molecular markers and therapeutic targets for HCC, highlighting the importance of considering the hypoxia TME in diagnostic and treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03201-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85919f351fa84b022b25ebecfed4ffc0c988bac6" target='_blank'>
              Deciphering hypoxia’s role in hepatocellular carcinoma prognosis with single-cell approaches
              </a>
            </td>
          <td>
            Jun Liang, Di Chen, Huiyu Liang
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Metastasis is a hallmark of cancer and is responsible for the majority of cancer-related deaths. Evidence suggests that even a single cancer cell can spread and seed a secondary tumour. However, not all circulating tumour cells have this ability, which implies that dissemination and distal growth require adaptative mechanisms during circulation. Here we report that constriction during microcapillary transit will trigger reprogramming of melanoma cells to a tumorigenic cancer stem cell-like state. Using a microfluidic device mimicking physiological flow rates and gradual capillary narrowing, we showed that compression through narrow channels lead to cell and nuclear deformation, rapid changes in chromatin state and increased calcium handling through the mechanosensor PIEZO1. Within minutes of microcapillary transit, cells show increased regulation of transcripts associated with metabolic reprogramming and metastatic processes, which culminates in the adoption of cancer stem cell-like properties. Squeezed cells displayed elevated melanoma stem cell markers, increased propensity for trans-endothelium invasion, and characteristics of enhanced tumorigenicity in vitro and in vivo . Pharmacological disruption of channel activity inhibited the stem cell-like state, while the selective PIEZO1 activator Yoda1 primed this state irrespective of constriction. Finally, deletion of PIEZO1 led to complete abrogation of the constriction-induced stem cell-like state. Together, this work demonstrates that compressive forces during circulation can reprogram circulating cancer cells to tumorigenic stem cell-like states that are primed for extravasation and metastatic colonization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3810403f83f02e75d7eba1f1e769e5b2b7eed299" target='_blank'>
              Capillary constrictions prime cancer cell tumorigenicity through PIEZO1
              </a>
            </td>
          <td>
            G. Silvani, C. Kopecky, S. Romanazzo, V. Rodriguez, , E. Pandzic, J. G. Lock, C. Chaffer, K. Poole, K. A. Kilian
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Advances in treating primary breast tumors and extracranial metastases have inadvertently led to a rise in brain metastases, which are severe and carry bleak prognostic outcomes. Thus, there is an urgent need to investigate the cellular and molecular drivers of metastatic central nervous system (CNS) tumors. The tumor microenvironment (TME) is critically important for metastasis, as metastatic cells must alter their new environment to survive and grow. Our laboratory's recent studies have pinpointed platelet-derived growth factor receptor beta (PDGFRβ) signaling as a critical pathway in promoting breast cancer-associated brain metastases (BCBM). Key findings from this published work involved data emanating from a genetically engineered mouse model (GEMM) with hyperactive PDGFRβ-D849V in the mesenchymal lineage driven by Fsp1-cre. These mice exhibit increased BCBM when injected with murine mammary tumor cells secreting the ligand PDGFB, making them an effective model for studying BCBM. However, hyperactivity of PDGFRβ in these mice occurs in all mesenchymal cells. To investigate cell-specific PDGFRβ functions in the CNS, we analyzed publicly available single-cell RNA-seq data of the adult murine brain. This analysis identified astrocyte populations expressing both Pdgfrb and the Fsp1/S100A4 mesenchymal lineage marker. To probe the role of PDGFRβ in reactive astrocytes, we developed a Gfap-cre GEMM model crossed with our mutant PDGFRβ hyperactivation allele (Gfap-cre;Pdgfrb-D849V). These mice exhibit increased GFAP expression in the absence of tumor and decreased overall survival upon intracardiac injection of PDGFB secreting breast cancer cells. Furthermore, flow cytometry at day 10 post intracardiac injection indicates potential differences in the immune response of the brain TME compared to controls. Current work is focused on identifying specific transcriptomic changes in astrocyte populations due to both PDGFRβ hyperactivation as well as metastatic breast cancer colonization of the brain. Overall, this work aims to identify potential therapeutic targets to treat BCBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/065fae0e35727b10f37e95370e754abb9b46417c" target='_blank'>
              BSBM-08 ASTROCYTE PDGFRβ HYPERACTIVATION IN BREAST CANCER BRAIN METASTASIS
              </a>
            </td>
          <td>
            Nathaniel Grabinski, Johnathon Scheibel, Jesse Reardon, Paul Lee, Jonathan Godbout, S. Sizemore, Jessica Winter, Gina M. Sizemore
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Mechanisms of HIV persistence in tissues are distinct from that in the blood. Spatial transcriptomic profiling examines HIV-infected cells, surrounding neighborhoods, and tissue microenvironment in unprecedented resolution. Spatial profiling captures cytokine gradients, distances between HIV-infected cells and immune effectors (and their function versus exhaustion), and cell-cell interactions. We present an overview of spatial transcriptomic platforms and a workflow of quality controls, sanity check, and bioinformatic analysis.


RECENT FINDINGS
The selection of spatial profiling methods should base on the research question, resolution, breadth of coverage, the expression level of RNA of interest, tissue quality, and tissue size. Advanced spatial transcriptomic profiling can capture RNA molecules at high resolution (<1 μm) and thus enable near-single cell profiling at genome-wide (~20 000 genes) breadth. Specifically, poly-A-based mRNA capture can identify previously unknown targets, while targeted RNA capture increases sensitivity in low-quality tissues. In targeted capture, however, the increase in target numbers frequently decreases sensitivity. Coupling ATAC-seq, protein capture, and T cell receptor sequencing to spatial platforms is ongoing.


SUMMARY
Spatial transcriptomic profiling uncovers mechanisms of HIV persistence in tissues and informs therapeutic strategies. Investigators should ensure the rigor of analysis, validate findings, and avoid reporting signatures with unknown biological significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f09209df407344118779f936014e334d9c123d1" target='_blank'>
              Roadmap for spatial transcriptomics of HIV in tissues.
              </a>
            </td>
          <td>
            Qijie Guan, Amare Eshetu, Ya-Chi Ho
          </td>
          <td>2025-07-07</td>
          <td>Current opinion in HIV and AIDS</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Immune-based agents, especially Immune Checkpoint Inhibitors (ICI), are standard of care therapy in non-small cell lung cancers (NSCLC); however, a significant number of patient tumors fail to respond, or develop resistance. While target expression, mutation burden and oncogenic pathways impact responses, an established mechanism contributing to ICI therapy failure is evasion of T-cell responses via downregulation of human leukocyte antigen (HLA). Conversely, natural killer (NK) cells eQector function is enhanced in the absence of HLA, making NK cellular therapies an attractive option for ICI resistant tumors. Challenges for current NK cell therapies include failure to adequately infiltrate solid tumors and long-term persistence, which may be overcome by deploying NK-derived extracellular vesicles (NKEVs) as a personalized novel adoptive cellular therapeutic with cytotoxic effects. Methods In a human NSCLC cohort (n=10), we used single cell RNAseq and antibody labeling (CITEseq) to examine the immune cell landscape in peripheral immune cells (PBMCs) and tumors. NKEVs retrieved from patient NK cells were characterized with proteomics and bulk RNAseq, and EV functionality was assessed using primary tumor organoids. Results We identified circulating NK cell subsets, describing diQerences in cell composition, gene expression and signaling, related to time point, NSCLC subtype (adenocarcinoma, squamous cell), composition and tumor grade. Next, we examined the functional capabilities of patient NKEVs in organoid structures derived from primary tumor cells, finding that exposure to patient NKEVs resulted in a 40-45% decrease in organoid viability, and significantly lowered the cisplatin dose required to elicit cytotoxicity. In Nivolumab treated PBMC co-culture experiments, NKEV addition favorably shifted the organoid infiltrating immune population to significantly fewer CD4+ T cells and more CD56+ NK cells. Finally, we used the multi-omic characterization of NKEV molecular cargo to identify RNA transcripts and proteins associated with cytotoxic and immune recruiting functions. Conclusions This work demonstrated that NKEVs can be successfully harvested from patient derived, expanded NK cells, and highlights their heterogeneous cargo, and anti-tumor properties in combination with standard-of-care therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d000c1883c73b9666be10545d77653f5089b0cd0" target='_blank'>
              NK cell-derived extracellular vesicles enhance cytotoxicity and immune cell recruitment in non-small cell lung cancer
              </a>
            </td>
          <td>
            Joanna Palade, Eric Alsop, Nanyun Tang, J. Antone, Dorothy M. Paredes, Tithi Ghosh Halder, R. Soldi, Taylor Bargenquast, Gary Schwartz, Jennifer P. Finholt, George J. Snipes, Sunil Sharma, Michael E. Berens, T. Whitsett, K. Van Keuren-Jensen, R. Kelly
          </td>
          <td>2025-07-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Neuroblastomas are highly heterogeneous tumors originating from neural crest-derived cells destined to form the sympathetic nervous system. Nearly half of high-risk tumors present with amplification of the MYCN proto-oncogene. Here, we describe a Mycn-driven, transplantable, non-germline, genetically engineered mouse model (Mycn-nGEMM). Mycn-nGEMM tumors recapitulate the immune-evasive, macrophage-rich tumor microenvironment of high-risk, MYCN-amplified human neuroblastoma. Treatment of tumor-bearing mice with anti-PD-L1, but not anti-PD-1 or anti-CTLA-4, inhibited tumor growth, profoundly remodeling the tumor microenvironment by depleting anti-inflammatory macrophages and increasing T cell infiltration. Surprisingly, while tumor cells showed low expression of PD-L1, anti-inflammatory macrophages from both murine and human neuroblastoma expressed PD-L1. We identified cytokines, including macrophage migration inhibitory factor, secreted by the Mycn-nGEMM cancer cells that drive expression of PD-L1 on macrophages. Combining anti-PD-L1 with CD40 agonist antibodies further improved survival in Mycn-nGEMM mice, demonstrating the potential for myeloid-targeting immunotherapies to overcome inhibitory barriers in immune-evasive neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2aecad2de1b569da0f36a9e846ad42a57aa1628" target='_blank'>
              Myeloid-targeting immunotherapies overcome inhibitory barriers in immune-evasive neuroblastoma.
              </a>
            </td>
          <td>
            Marie Ménard, Hiroyuki Yoda, Nicole Nasholm, Megumi J Barata, Linyu Wang, Erin F. Simonds, Edbert D Lu, Shannon T. Wong-Michalak, Lauren McHenry, Alvin Farrel, Rebecca S. Kaufman, Vanessa Lopez, Rebekah J Kennedy, G. E. Fernandez, Hiroyuki Shimada, Liron D. Grossmann, S. Asgharzadeh, John M. Maris, W. Gustafson, William A Weiss
          </td>
          <td>2025-07-14</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Purpose Programmed cell death (PCD) mechanisms play crucial roles in cancer progression and treatment response. This study aims to develop a PCD scores prediction model to evaluate the prognosis of hepatocellular carcinoma (HCC) and elucidate the tumor microenvironment differences. Methods We analyzed transcriptomic data from 363 HCC patients in the TCGA database and 221 patients in the GEO database to develop a PCD prediction model. Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics sequencing (ST-seq) data from HCC patients were analyzed to investigate the tumor microenvironment and functional disparities. The oncogenic role of the key gene UBE2E1 in the model was explored in HCC through various in vitro experiments. Results Seventeen PCD-related genes were identified as significant prognostic indicators, forming the basis of our PCD prediction model. High-PCD scores correlated with poorer overall survival (OS) and exhibited significant predictive capabilities. scRNA-seq analysis revealed distinct tumor cell characteristics and immune microenvironment differences between high- and low-PCD groups. High-PCD tumors showed increased cell proliferation and malignancy-associated gene expression. T cells in high-PCD patients were more likely to be exhausted, with elevated expression of exhaustion markers. ST-seq data also confirmed these results. Among the genes associated with the PCD prognostic model, UBE2E1 was identified as a key oncogenic marker in HCC. Conclusions The PCD prediction model effectively predicts prognosis in HCC patients and reveals critical insights into the tumor microenvironment and immune cell exhaustion. This study underscores the potential of PCD-related biomarkers in guiding personalized treatment strategies for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88d93e2b7ad6d106b6fbcf29fd3ec02edc2e6591" target='_blank'>
              Integrative spatial and single-cell transcriptomics elucidate programmed cell death-driven tumor microenvironment dynamics in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kai Lei, Yutong Zhao, Shumin Li, Jiawei Liu, Wenhao Chen, Caihong Zhou, Yi Zhang, Jinmei Tan, Jian Wu, Qi Zhou, Jiehui Tan
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/058b330aef456f02cb84824c66751dcf434c0ff4" target='_blank'>
              Single-cell dissection of cervix and placenta reveal both novel and overlapping cell types
              </a>
            </td>
          <td>
            Margot van Riel, L. Lannoo, A. Pexsters, O. Tšuiko, I. Parijs, Irene Claes, Thierry Voet, Dirk Timmerman, J. R. Vermeesch
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We recovered tumor microenvironmental features previously described to associate with therapy response. We subsequently identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.



 Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andreanne Gagne, Samantha Hoffman, Kevin Bi, Breanna Titchen, Connor Hennessey, Joshua Remland, Matthew Carnes, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, Eliezer M. Van Allen. Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b559d25313fad22976c28c00d01803263f1319a3" target='_blank'>
              Abstract A046: Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda E. Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor Hennessey, Joshua Remland, Matthew Carnes, Erin Shannon, Sabrina Y. Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M Boland, A. Aguirre, Nilay S. Sethi, Valentina Boeva, Eliezer M Van Allen
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Innovative identification technologies for hematopoietic stem cells (HSCs) have expanded the scope of stem cell biology. Clinically, the functional quality of HSCs critically influences the safety and therapeutic efficacy of stem cell therapies. However, most analytical techniques capture only a single snapshot, disregarding the temporal context. A comprehensive understanding of the temporal heterogeneity of HSCs necessitates live-cell, real-time and non-invasive analysis. Here, we developed a prediction system for HSC diversity by integrating single-HSC ex vivo expansion technology with quantitative phase imaging (QPI)-driven machine learning. By analyzing the cellular kinetics of individual HSCs, we discovered previously undetectable diversity that snapshot analysis cannot resolve. The QPI-driven algorithm quantitatively evaluates stemness at the single-cell level and leverages temporal information to significantly improve prediction accuracy. This platform advances the field from snapshot-based identification of HSCs to dynamic, time-resolved prediction of their functional quality based on past cellular kinetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1e09a9f61aa5ec203755a5a99580bea38ed797" target='_blank'>
              Quantitative phase imaging with temporal kinetics predicts hematopoietic stem cell diversity
              </a>
            </td>
          <td>
            Takao Yogo, Yuichiro Iwamoto, H. J. Becker, Takaharu Kimura, Reiko Ishida, Ayano Sugiyama-Finnis, Tomomasa Yokomizo, Toshio Suda, Sadao Ota, Satoshi Yamazaki
          </td>
          <td>2025-07-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer immunotherapy is an innovative treatment approach that leverages the immune system to combat tumors, demonstrating significant therapeutic potential. In the past few years, single‐cell RNA sequencing (scRNA‐seq) has made significant progress in the field of cancer immunotherapy, enabling us to understand the complexity of anti‐tumor immune processes with unprecedented depth and precision, thereby facilitating the design of more effective immunotherapy strategies. This review aims to summarize the recent applications of advanced scRNA‐seq technologies in tumor immunology. First, we outline the most representative scRNA‐seq technologies with different technical principles, with a particular focus on single‐cell T cell receptor sequencing. Next, we describe how scRNA‐seq technology is applied to identify the cellular composition and phenotypes within the tumor microenvironment for the construction of immune cell atlas, and uncover key cell types and molecular mechanisms underlying treatment responses for developing novel immunotherapies. Finally, we address the current challenges and future prospects of scRNA‐seq technology in tumor immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caad9af0b2721a017882aaaa64cafb8d1119d63" target='_blank'>
              Advanced single‐cell RNA sequencing in tumor immunology
              </a>
            </td>
          <td>
            Yilong Liu, Xiyuan Yu, Zhipeng Han, Qiuyang Wei, Qi Niu, Chaoyong Yang, Jie P. Li, Zijian Guo, Huimin Zhang
          </td>
          <td>2025-06-24</td>
          <td>Interdisciplinary Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Despite extensive research efforts, glioblastoma (GBM) remains a deadly disease with poor prognosis. Although previous studies have identified various cell states within GBM tumors, the molecular mechanism underlying adaptive GBM cell plasticity induced by conventional therapy remains unclear.


METHODS
We used fluorescent reporters for proneural (PN) and mesenchymal (MES) subtypes to monitor GBM cell plasticity in real-time across multiple patient-derived cell lines. This approach revealed cells that concurrently expressed both proneural and mesenchymal markers. To investigate this unique hybrid population, we implemented a comprehensive methodological approach encompassing bulk and single-cell RNA sequencing, single-cell ChIP sequencing, nuclear proteomics, high-resolution imaging, orthotopic mouse models, clinical dataset analysis, and pharmacological and genetic techniques. This multifaceted strategy allowed us to gain functional and molecular insights into this distinct cellular population.


RESULTS
We showed that these hybrid cells are increased by conventional therapies, and are resistant to these therapies. At the molecular level, hybrid cells display significant alterations in chromatin structure and nuclear protein composition, elevated transcriptional activity, Myc activation, and improved transport between the nucleus and cytoplasm. Genetic and pharmaceutical inhibition of the nuclear import/export shuttling machinery, increased in hybrid cells, effectively suppressed adaptive GBM cell plasticity and hybrid identity, thereby enhancing the sensitivity of GBM cells to therapies.


CONCLUSION
Our results indicate that GBM hybrid cells play a crucial role in chemoradiation resistance. The nuclear transport machinery presents a potential therapeutic target for hybrid cells, offering a way to counteract the typical resistance to treatment observed in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0e616d882e7bd8159604ae82c04f4049ee875a" target='_blank'>
              Proneural-Mesenchymal hybrid glioblastoma cells are resistant to therapy and dependent on nuclear import.
              </a>
            </td>
          <td>
            G. Bourmeau, O. Anezo, Jeremy Raymond, A. Ballestín, C. Pichol-Thievend, J. Reveilles, Adrien Thomas, Lin Wang, Melanie Miranda, Eve Moutaux, S. Liva, V. Ribecco, L. Besse, F. Dingli, D. Loew, Céline Vallot, G. Gargiulo, V. Ravi, Kevin Joseph, G. Seano
          </td>
          <td>2025-07-08</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Objective: The aim of this study was to investigate the role and mechanism of CD79A+ B cells in mediating the microenvironment of breast cancer and the relationship with the prognosis of breast cancer. Methods: Single-cell RNA sequencing and bulk RNA sequencing analysis were combined to annotate breast cancer cell subtypes, perform cell communication and trajectory analysis. CD79A-related signature was constructed by LASSO and multivariate Cox analysis. CD79A+ B cell subsets in the tumor microenvironment were explored by immunoanalysis and multiple immunofluorescence analysis. Results: There were communication relationships between CD79A+ B cells and multiple cell types. A prognostic risk signature containing 6 genes was constructed by combining the TCGA dataset. The immune profile analysis showed that the low-risk group showed a higher immune response. In addition, multiple immunofluorescence analysis showed an attraction between CD79A+ B cells and tumor cells, and patients with high CD79A+ B cells expression had significantly higher survival rates. Conclusion: This study comprehensively explored the heterogeneity of CD79A+ B cells through transcriptome analysis and chromatin analysis, which contributes to an in-depth understanding of the function of CD79A+ B cells in biological processes as well as the molecular mechanism of breast carcinogenesis, providing a theoretical basis for treatment and prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/088e6783ffeb2259cf83f0eff8450fb916b96b21" target='_blank'>
              Single-Cell Sequence and Machine Learning Identify a CD79A+B Cells-Related Transcriptional Signature for Predicting Clinical Outcomes and Immune Microenvironment in Breast Cancer
              </a>
            </td>
          <td>
            Haihong Hu, Wendi Zhan, Hongxia Zhu, Bo Hao, Ting Yan, Jingdi Zhang, Siyu Wang, Taolan Zhang
          </td>
          <td>2025-07-26</td>
          <td>Cancer Informatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract While immune-modulating therapies which enhance antitumor T cell function have revolutionized cancer therapy, including for patients with brain metastases, the antigen specificity of intratumoral T cells remains poorly characterized. We performed droplet-based 5’ single-cell RNA sequencing and T cell receptor (TCR) sequencing to characterize the transcriptional profile and clonal repertoire of tumor-infiltrating T cells in resected tumors from patients with previously untreated non-small cell lung cancer (NSCLC) brain metastases (n=6). We identified a population of clonally-expanded, CXCL13+CD8+ T cells enriched for transcriptional signatures predictive of tumor reactivity. Using a high-throughput T cell antigen identification method developed by our group, we screened 164 TCRs from clonally-expanded and/or CXCL13+CD8+ T cells against 2,289 putative neoantigens in addition to 146 tumor-associated antigens. In total, we identified reactivities for 21% (n=35) of the TCRs screened. Consistent with recent studies, we observed that the transcriptional state of the T cell was more predictive of tumor reactivity than the degree of clonal expansion. Of TCR specificities identified, 25.7% (n=9) of TCRs were reactive to neoantigens derived from somatic single-nucleotide variants specific to each patient (private antigen). Intriguingly, 54.3% (n=19) of the TCRs were reactive to unmutated peptides expressed by many patients’ lung tumors (public antigens). In summary, we present a deep characterization of the antigenic reactivities of intratumoral T cells in NSCLC brain metastases. Our data are consistent with an emerging body of literature that has theorized that “missing reactivities” from neoantigen-centric studies arise from complex mechanisms of genetic and epigenetic dysregulation within tumors. Further characterization of these “public antigens” may provide the basis for novel antigen-directed T cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a71379070ce21b871f60cf702eab6f6054459a92" target='_blank'>
              TRLS-11 UNCOVERING ANTIGEN-SPECIFICITY OF INTRATUMORAL CD8+ T CELLS IN NON-SMALL CELL LUNG CANCER BRAIN METASTASES
              </a>
            </td>
          <td>
            Benjamin Lu, Berkay Yahsi, Jianlei Gu, Jack Coburn, Mithila Kasbe, Anna Arnal-Estapé, Kayla Sohn, Kalyn Whitehead, Mashwiyat Mosharraf, Don Nguyen, Hongyu Zhao, Liliana Lucca, K. Schalper, Veronica Chiang, David Hafler, Mark N. Lee
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="SPOCK1, a matricellular glycoprotein, has been implicated in tumor progression, metastasis, and the tumor immune microenvironment, yet its specific roles in breast cancer (BRCA) remain unclear. This study aimed to systematically explore the expression pattern, prognostic significance, mutation landscape, immune association, and spatial localization of SPOCK1 in breast cancer through integrated multi-omics analyses.Transcriptomic, genomic, and clinical data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) were utilized. Bulk RNA sequencing and single-cell RNA sequencing (scRNA-seq) analyses were conducted, including functional enrichment, immune infiltration assessments, mutation profiling, and transcription factor activity analysis. Multiplex immunohistochemistry (mIHC) was performed to validate the spatial distribution of SPOCK1+ cancer-associated fibroblasts (CAFs) within the tumor microenvironment. Statistical analyses were performed using R and GraphPad Prism.SPOCK1 was broadly overexpressed in multiple cancer types and significantly associated with poor prognosis in BRCA. High SPOCK1 expression correlated with immune checkpoint activation, enhanced immune infiltration, and enriched metastasis-related pathways such as epithelial–mesenchymal transition (EMT) and TGF-β signaling. Single-cell analysis identified CAFs as the primary cell population expressing SPOCK1, with spatial mIHC confirming their close proximity to tumor cells. Furthermore, SPOCK1-high CAFs exhibited stronger intercellular communications with malignant cells via collagen, fibronectin, and IGFBP signaling pathways, alongside distinct transcription factor and metabolic profiles. In breast cancer CAF cell lines with knockdown of ANXA2 we found that the expression of both SPOCK1 and IGF1 was reduced.SPOCK1 serves as a critical regulator of breast cancer progression, influencing tumor metastasis and reshaping the immune microenvironment via CAF-mediated mechanisms. These findings suggest that targeting SPOCK1+ CAFs could offer new therapeutic opportunities for breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6ec4d3a2efd20e6cfad0b54c715da9bed1f6fb" target='_blank'>
              SPOCK1 promotes the progression of breast cancer by modulating cancer-associated fibroblasts and exerts a synergistic effect with ANXA2
              </a>
            </td>
          <td>
            Yuan Jie, Xin Fei, Meng Fan
          </td>
          <td>2025-08-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Spatial transcriptomics (ST) is transforming our understanding of tumor heterogeneity by enabling high-resolution, location-specific mapping of gene expression across tumors and their microenvironment. However, the translational potential of spatial transcriptomics is still limited by its high cost, hindering the assembly of large patient cohorts needed for robust biomarker discovery. Here we present Path2Space, a deep learning approach that predicts spatial gene expression directly from histopathology slides. Trained on substantial breast cancer ST data, it robustly predicts the spatial expression of over 4,300 genes in independent validations, markedly outperforming existing ST predictors. Path2Space additionally accurately infers cell-type abundances in the tumor microenvironment (TME) based on the inferred ST data. Applied to more than a thousand breast tumor histopathology slides from the TCGA, Path2Space characterizes their TME on an unprecedented scale and identifies three new spatially-grounded breast cancer subgroups with distinct survival rates. Path2Space-inferred TME landscapes enable more accurate predictions of patients’ response to chemotherapy and trastuzumab directly from H&E slides than those obtained by existing established sequencing-based biomarkers. Path2Space thus offers a transformative, fast and cost-effective approach to robustly delineate the TME directly from their histopathology slides, facilitating the development of spatially-grounded biomarkers to advance precision oncology.



 Emma M. Campagnolo, Eldad D. Shulman, Roshan Lodha, Amos Stemmer, Peng Jiang, Carlos Caldas, Simon Knott, Danh-Tai Hoang, Kenneth Aldape, Eytan Ruppin. Path2Space: An AI approach for cancer biomarker discovery via histopathology inferred spatial transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B047.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d0f988faed3c86233b6f535df129f10c4acb357" target='_blank'>
              Abstract B047: Path2Space: An AI approach for cancer biomarker discovery via histopathology inferred spatial transcriptomics
              </a>
            </td>
          <td>
            Emma M. Campagnolo, E. Shulman, Roshan Lodha, Amos Stemmer, Peng Jiang, Carlos Caldas, Simon Knott, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2411017a831bd32082b22e6f4221bf8c31f61827" target='_blank'>
              ETS-guided iPSC-endothelial models recapitulate malaria pathogenesis
              </a>
            </td>
          <td>
            François Korbmacher, Hannah Fleckenstein, Rory K. M. Long, Patryk Poliński, Livia Piatti, Borja López-Gutiérrez, Alina Batzilla, Dennis Crusius, Vikas Trivedi, Miki Ebisuya, Maria Bernabeu
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94d39a50858ad986c811d60a70891ccb3b663404" target='_blank'>
              Single-cell spatial analysis with Xenium reveals anti-tumour responses of CXCL13 + T and CXCL9+ cells after radiotherapy combined with anti-PD-L1 therapy
              </a>
            </td>
          <td>
            Shunsuke A. Sakai, Hidekazu Oyoshi, M. Nakamura, T. Taki, Kotaro Nomura, H. Hojo, H. Hirata, A. Motegi, Yuka Nakamura, Junko Zenkoh, K. Aokage, Akira Hamada, M. Kojima, Takeshi Kuwata, K. Tsuchihara, T. Akimoto, J. Soh, Tetsuya Mitsudomi, Masahiro Tsuboi, Genichiro Ishii, Yutaka Suzuki, Ayako Suzuki, Riu Yamashita, S. Kageyama
          </td>
          <td>2025-07-16</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background To discover immune response-associated gene expressions related to brain metastases of skin melanomas, we analyzed a group of fifteen primary skin melanomas consisting of five tumors from patients who did not develop systemic metastases (NM); ten from patients with systemic metastasis of which five with (BM), and five without brain involvement (OOM). Regions of interest (ROIs) harboring tumor cells only; immune cells only; or immune cell infiltration in tumor cell regions, were separately analyzed. Methods we profiled the tumor immune microenvironment (TIME) of primary skin melanomas by using a panel of 77 immune response-related oligo-nucleotide targets in the GeoMx Digital Spatial profiler (Nanostring Technologies). Results Melanomas from patients who developed brain metastases contained significantly higher levels of (CD45+) immune cells and overexpressed fibronectin and CTLA-4 in all ROIs compared to the melanomas that had metastasized to other sites than the brain (p<0.05). Distant metastases were accompanied by lower expression of CD137 (4-1BB). Downregulation of VISTA, IDO1 and ICOS was associated with melanomas that gave rise to distant metastases including brain. Conclusion Lower expression of CD137 (4-1BB) is linked with the formation of distant metastases and the expressions of fibronectin and CTLA-4 herald the formation of brain metastasis in particular.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f922ad0f88efcd474799b2ee7597e58e29a1a3" target='_blank'>
              Overexpression of CTLA-4 and fibronectin, and lower expression of CD137 (4-1BB) is associated with brain metastasis of primary skin melanomas. An analysis of local immune response by digital spatial profiling
              </a>
            </td>
          <td>
            S. Najjary, Nur al Amery, Mengqi Huang, Disha Vadgama, Alexander C. J. van Akkooi, H. J. Gilhuis, Zineb Belcaid, Dana A. M. Mustafa, J. Kros, Pedro Berraondo, Fer. Aranda Vega
          </td>
          <td>2025-06-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Primary tumors constantly shed cancer cells into the circulation, yet only a fraction of these cells manages to give rise to metastatic tumors. Successful metastatic seeding and growth appears to depend on metabolic changes within cancer cells. Here, using a metabolism-focused CRISPR screen in a spontaneous metastasis model, we found that expression of the enzyme γ-butyrobetaine hydroxylase 1 (BBOX1) in a subpopulation of tumor cells in various carcinomas enables immune evasion. The metabolite carnitine produced by BBOX1 inhibited the small GTPase RhoA in natural killer (NK) cells, preventing immunological synapse formation and thereby protecting metastatic cells. Loss of BBOX1 in tumor cells promoted their destruction by NK cells in vitro and improved the efficacy of NK cell adoptive transfer therapy in vivo. These findings illustrate how BBOX1-positive tumor cells hijack carnitine production to evade immune surveillance during metastasis and propose BBOX1 as a potential metabolic checkpoint for anti-metastatic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83c565b424262f8f0a380c1e10b4fb97fa55b5b" target='_blank'>
              BBOX1 is a Metabolic Checkpoint that Mediates Metastatic Cancer Cell Evasion from Immunosurveillance by Natural Killer Cells.
              </a>
            </td>
          <td>
            Chuanjie Zhang, Hongchao He, Yi Gao, Jiawei Ding, Hai Huang, Yixun Liu, Hanqing Liu, Jun Xiao, Dan-Feng Xu, Zunguo Du, Zhenbo Zhang, Lechi Ye
          </td>
          <td>2025-07-31</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a high-mortality cancer characterized by its aggressive, treatment-resistant phenotype and a complex tumour microenvironment (TME) featuring significant hypoxia. Bulk transcriptomic analysis has identified the “classical” and “basal-like” transcriptional subtypes which have prognostic value; however, it is not well-established how microenvironmental heterogeneity contributes to these transcriptional signatures. Here, we exploited the TRACER platform to perform single cell transcriptome analysis of organoids at specific spatial locations to explore the effect of oxygen and other cell-generated microenvironmental gradients on organoid heterogeneity. We found that the microenvironmental gradients present in TRACER significantly impact the distribution of organoid transcriptional phenotypes and the enrichment of gene sets linked to cancer progression and treatment resistance. More significantly, we found that microenvironmental gradients, predominantly oxygen, drive changes in the expression of classical and basal-like transcriptional subtype gene signatures. This work suggests that hypoxia contributes to determining transcriptional subtypes in PDAC tumour cells independent of additional cells in the TME and broadly highlights the importance of considering microenvironmental gradients such as oxygen in organoid-based studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-98344-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d7b936c3248bb437b55ed176049e4ec9a8d43e" target='_blank'>
              Analysis of an engineered organoid model of pancreatic cancer identifies hypoxia as a contributing factor in determining transcriptional subtypes
              </a>
            </td>
          <td>
            Natalie Landon-Brace, Simon Latour, Brendan T. Innes, Michelle Nurse, Jose L. Cadavid, I. Co, Cassidy M. Tan, Ferris Nowlan, Sibil Drissler, F. Notta, H. Jackson, Gary D Bader, A. McGuigan
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Objectives This study aims to elucidate the complex molecular and cellular landscape of pancreatic ductal adenocarcinoma (PDAC) by identifying key regulatory non-coding RNAs (ncRNAs), hub protein-coding genes, and Intercellular communication pathways that may serve as prognostic biomarkers and therapeutic targets. Background Pancreatic cancer remains one of the deadliest malignancies worldwide, characterized by late diagnosis, limited treatment response, and poor prognosis. Among its histological subtypes, PDAC accounts for over 80% of cases and is defined by a highly fibrotic and immunosuppressive tumor microenvironment (TME). Methods We performed a comprehensive bioinformatics analysis integrating multiple transcriptomic datasets from the NCBI Gene Expression Omnibus (GEO), including mRNA, miRNA, lncRNA, and circRNA profiles from PDAC and adjacent normal tissues. Differential expression analysis was conducted using GEO2R, followed by functional enrichment via DAVID. Hub genes were identified from protein–protein interaction (PPI) networks constructed using STRING and validated using GEPIA2. A competing endogenous RNA (ceRNA) network was developed to investigate regulatory ncRNA–mRNA axes. To refine these findings, single-cell RNA-seq (scRNA-seq) data were analyzed to resolve the cellular origin of hub genes and ncRNAs, and CellChat was employed to model intercellular communication within the TME. Results We identified several dysregulated genes and ncRNAs implicated in key oncogenic pathways, including ECM remodeling, inflammation, and immune evasion. The ceRNA network highlighted functional interactions between circRNAs, lncRNAs, and miRNAs regulating key hub genes. Single-cell analysis revealed cell-type-specific expression of hub genes—e.g., FN1 and COL11A1 in fibroblasts, CXCL8 in macrophages, and ITGA3 in ductal cells—and uncovered a macrophage–endothelial CXCL8–ACKR1 signaling axis potentially driving tumor-associated angiogenesis. Moreover, correlations with immune cell infiltration and drug sensitivity further underscored the translational relevance of the identified molecular targets. Conclusion Our analysis combining bulk and single-cell transcriptomics provides a multi-scale view of PDAC pathogenesis. The findings highlight the interplay between ncRNAs, hub genes, and cellular crosstalk in shaping the tumor ecosystem and suggest novel targets for precision therapeutic intervention and biomarker development. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01815-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/588c78ae7417c5606a00168a20889c995ded56f7" target='_blank'>
              Combined bulk and single-cell transcriptomic analysis reveals cell-type-specific inflammatory crosstalk in pancreatic cancer
              </a>
            </td>
          <td>
            Ahmad Golestanifar, Mana Zakeri, Sahar Gohari-Lasaki, Hengameh Khedri, Mohammadreza Saberiyan
          </td>
          <td>2025-07-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a globally prevalent malignancy. This disease often progresses rapidly, resulting in many patients being diagnosed at a late stage, making early detection and intervention a major clinical challenge. Postoperative recurrence and metastasis rates remain significantly high, and no effective prevention strategies are currently available. Cancer-associated fibroblasts (CAFs) are essential components in the reorganization of the tumor microenvironment (TME), as they can modulate cancer cell proliferation, migration, invasion, and chemoresistance through diverse mechanisms or signaling pathways, including the release of cytokines, remodeling of the extracellular matrix, and the evasion of the immune response. This review offers a detailed overview of the cellular origins, subtype diversity, and functional differences among CAFs. In addition, it depicts the expression profiles of key markers in various CAF subtypes and clarifies essential signaling pathways and mechanisms of CAFs. Additionally, we discuss current and future therapeutic strategies targeting CAFs in the context of HCC. This review provides critical insights into future studies on novel therapeutic approaches for CAFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8451f01fad30bf68417ccee67bbd624a7786b63" target='_blank'>
              Cancer-associated fibroblasts in hepatocellular carcinoma: origins, heterogeneity, and therapeutic implications
              </a>
            </td>
          <td>
            Xin Shi, Weixiong Zhu, Jianpeng Zhang, Chuanlei Fan, Jing Zhang
          </td>
          <td>2025-07-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Although tumor cell heterogeneity between the tumor center (TC) and invasion front (IF) of oral squamous cell carcinoma (OSCC) is well documented, the morphological, molecular, and functional characteristics of cancer-associated fibroblasts (CAFs) in these regions remain poorly understood. Methods We examined hematoxylin and eosin (H&E)–stained OSCC sections to assess CAF morphology and correlation with patient prognosis. We then isolated paired CAFs from the tumor center (CAFTC) and invasion front (CAFIF) of four OSCC patients and compared their ECM-remodeling activity and pro-tumorigenic effects on OSCC cells. Furthermore, RNA sequencing identified differentially expressed genes between CAFTC and CAFIF. Finally, based on RNA-seq findings, we knocked down hyaluronan synthase 1 (HAS1) in CAFIF to evaluate its role in extracellular matrix (ECM) remodeling and tumor invasion. Results Compared to CAFTC, CAFIF exhibited a plump cell morphology and were associated with shorter disease-free survival. Functionally, CAFIF showed higher ECM-remodeling activity and more effective ability for promoting OSCC invasion and lymph node metastasis than CAFTC. RNA-seq identified HAS1 was significantly upregulated in CAFIF, promoting hyaluronic acid (HA) production and ECM remodeling. HAS1 knockdown in CAFIF diminished ECM remodeling and attenuated the ability of CAFIF to promoting OSCC invasion. Conclusion CAFIF with plump cell morphology showed pro-invasive abilities, driven in part by HAS1 overexpression and ECM remodeling, suggesting that targeting HAS1-driven ECM remodeling could be a promising therapeutic strategy. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03493-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6dfa996b747be7a23ea3bc4905b2cb5a5989f62" target='_blank'>
              HAS1high cancer associated fibroblasts located at the tumor invasion front zone promote oral squamous cell carcinoma invasion via ECM remodeling
              </a>
            </td>
          <td>
            Wanyong Jin, Qiuya Yu, Liyuan Yu, Ting Zhou, Xiren Wang, Wanqiu Lu, Xiaoxin Zhang, Liang Ding, Qingang Hu, Yanhong Ni
          </td>
          <td>2025-08-14</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is an aggressive subtype characterized by limited treatment options and poor prognosis. Recent evidence highlights the crucial role of cancer-associated fibroblasts (CAFs) in TNBC progression, yet their molecular characteristics remain incompletely understood. In this study, we performed a comprehensive analysis combining bioinformatics approaches with experimental validation to investigate CAF-related genes in TNBC. Using weighted gene co-expression network analysis (WGCNA) of TNBC samples from TCGA and METABRIC datasets, we identified 185 CAF-related genes significantly associated with extracellular matrix organization and TGF-β signaling pathways. Through rigorous statistical modeling, we developed a 3-gene prognostic signature (CERCAM, JAM3, PLAU) that effectively stratified TNBC patients into high- and low-risk groups with distinct survival outcomes. Importantly, we validated the functional role of PLAU, one of the signature genes, through in vitro and in vivo experiments. Results showed that CAF-derived PLAU played key role in the malignant behaviors of TNBC. Our findings provide new insights into CAF-mediated TNBC progression and suggest potential stromal targets for therapeutic intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03867-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1bb9fe2fe0c3e330d474111c7b5267eb0d6dbf" target='_blank'>
              Stromal PLAU mediates tumor progression and informs a novel therapeutic target in triple-negative breast cancer
              </a>
            </td>
          <td>
            Jun Zou, Jing-yao Zhang, Yu Li, Baowen Yuan, Yuanyi Wang, Yalong Qi, Qian Wang, Wan Qin, Xianglin Yuan, Binghe Xu
          </td>
          <td>2025-07-11</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Metaplastic breast cancer (MpBC) is defined by the presence of various morphological elements, typically biphasic, with epithelial (e.g. no-special type (NST), squamous) and mesenchymal (e.g. spindle, chondroid, osteoid) components. The established clonality of the different components favours an evolution model encompassing either a multipotent progenitor, or a linear metaplastic conversion. We used methylation profiling and showed that different morphologies have specific methylation profiles. Furthermore, our spatial transcriptomic approach, using 10× Genomics Visium and trajectory analysis, evidenced that spindle cells form a transition between the originating carcinoma of no-special type (NST) and pleomorphic regions, with osteoid differentiation likely to be an end-stage fate of the chondroid growth pattern, supporting the conversion model of lineage differentiation. We have also identified a series of master transcription factors likely to regulate these processes, and are significantly associated with metaplastic-like clinical features. This data further supports the conversion model of metaplasia and warrants functional analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077285caa17e07f895159bb1ece8c70f06837f91" target='_blank'>
              Integrated profiling of metaplastic breast cancer identifies putative master regulators of intratumoral heterogeneity
              </a>
            </td>
          <td>
            Yufan Feng, Albert Xiong, Onkar Mulay, Anna Sokolova, M. Lim, Benjamin Van Haeringen, Natasha McGuire, Xavier M. de Luca, Peter T Simpson, Quan Nguyen, S. Lakhani, A. M. McCart Reed
          </td>
          <td>2025-08-11</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Breast cancer (BC) remains a leading cause of cancer-related mortality in women, with complex mechanisms driving its initiation, progression, and resistance to therapy. In recent years, the tumor microenvironment (TME) has gained attention for its critical role in shaping tumor behavior, where small extracellular vesicles (small EVs) have emerged as key mediators of intercellular communication. These vesicles carry a diverse cargo of proteins, lipids, DNA, and various non-coding RNAs—such as miR-21, miR-155, and miR-1246—mirroring the molecular status of their originating cells. This review highlights the roles of small EVs in immune modulation, stromal remodelling, and metastatic niche formation, emphasizing their contribution to therapy resistance and immune evasion. We discuss recent updates on EV biogenesis, characterisation and isolation techniques, such as ultracentrifugation, immunoaffinity and microfluidic systems. We also critically evaluate their potential for clinical application and how well they conform to the MISEV2023 guidelines. Furthermore, we examine small EVs as diagnostic tools in liquid biopsies and compare them with conventional methods such as mammography and tissue biopsies. We also discuss organotropism mediated by small EV cargo (e.g., integrins α6β4, αvβ5) and the diagnostic potential of protein and lipid signatures (e.g., PD-L1, CD63, and exosomal lipidomics). Therapeutically, we explore engineered small EVs for drug delivery, gene modulation, and immune activation, addressing challenges of targeting efficiency, in vivo stability, immunogenicity, and clinical scalability. The review discusses ongoing clinical trials involving small EVs in BC and highlights key translational gaps between preclinical advances and clinical implementation. Finally, we explores how integrating artificial intelligence, single-cell transcriptomics, and multi-omics approaches can help overcome major challenges such as small EV heterogeneity and tracking limitations. Crucially, this integration enables a more tailored understanding of each patient’s tumor biology, reducing therapeutic failures by guiding more personalized and effective treatment strategies. Overall, small EVs represent a transformative tool in precision oncology, contingent on resolving key challenges in their clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbe25e9e7298ee7befa5cf726da413903d9b701b" target='_blank'>
              Small extracellular vesicles: messengers at the service of breast cancer agenda in the primary and distant microenvironments
              </a>
            </td>
          <td>
            Angela Galardi, Valentina Fogazzi, Claudia Tottone, M. Giussani, S. Pupa, G. Cosentino, M. Iorio
          </td>
          <td>2025-07-21</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Simple Summary Mass spectrometry imaging (MSI) lipidomics is a cutting-edge technique that maps the spatial distribution of hundreds of lipids within a tissue section. We used MSI to study the lipid patterns found in the human lung, in particular in two important pathological conditions: lung cancer and chronic obstructive pulmonary disease. We could associate differential lipid profiles with the main tissue types found in the lung thanks to the spatial resolution used: 25 μm. The main result in tumor tissue was the accumulation of lipids containing arachidonic acid, a precursor to molecules that actively participate in cell differentiation and cancer development. We also used public databases of lung cancer studies and established that lung cancer impacts two different families of lipids: phosphatidylinositol and sphingolipids. Overall, this study underscores the great potential of MSI lipidomics to provide underexplored insights into lung diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/458a25a979023f05c69b3c45f0c39ff9dee80d11" target='_blank'>
              Challenges and Advantages of Using Spatially Resolved Lipidomics to Assess the Pathological State of Human Lung Tissue
              </a>
            </td>
          <td>
            Ibai Calvo, Albert Maimó-Barceló, Jone Garate, Joan Bestard-Escalas, Sergio Scrimini, Jaume Sauleda, B. Cosío, José A. Fernández, Gwendolyn Barceló-Coblijn
          </td>
          <td>2025-06-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis and is often mediated by extrachromosomal DNA (ecDNA). EcDNA amplifications are associated with increased pathogenicity of cancer and poorer outcomes for patients. EcDNA can be detected accurately using fluorescence in situ hybridization (FISH) when cells are arrested in metaphase. However, the majority of cancer cells are non-mitotic and must be analyzed in interphase, where it is difficult to discern extrachromosomal amplifications from chromosomal amplifications. Thus, there is a need for methods that accurately predict oncogene amplification status from interphase cells. Here, we present interSeg, a deep learning-based tool to cytogenetically determine the amplification status as EC-amp, HSR-amp, or not amplified from interphase FISH images. We trained and validated interSeg on 652 images (40,446 nuclei). Tests on 215 cultured cell and tissue model images (9,733 nuclei) showed 89% and 97% accuracy at the nuclear and sample levels, respectively. The neuroblastoma patient tissue hold-out set (67 samples and 1,937 nuclei) also revealed 97% accuracy at the sample level in detecting the presence of focal amplification. In experimentally and computationally mixed images, interSeg accurately predicted the level of heterogeneity. The results showcase interSeg as an important method for analyzing oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8261c73bf75d788913526d1e389ee8b4f740f28c" target='_blank'>
              Inhibition of ACVR1 in Cancer-Associated Fibroblasts Suppresses Colorectal Cancer Cell Growth.
              </a>
            </td>
          <td>
            Shinya Kato, N. Miyoshi, S. Fujino, Masafumi Horie, Shinichi Yachida, Mitsunobu Takeda, Y. Sekido, Tsuyoshi Hata, Atsushi Hamabe, T. Ogino, Mamoru Uemura, Hirofumi Yamamoto, Masayoshi Yasui, M. Ohue, Y. Doki, H. Eguchi
          </td>
          <td>2025-07-31</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT Introduction Lung adenocarcinoma, the most common subtype of non‐small cell lung cancer, faces challenges such as drug resistance and tumor heterogeneity. N4‐acetylcytidine (ac4C) is an important RNA modification involved in cancer progression, but its role in lung adenocarcinoma remains unclear. Methods This study analyzed transcriptomic and single‐cell RNA sequencing data from public databases to investigate the expression and clinical significance of ac4C‐related genes in lung adenocarcinoma. Ten machine learning algorithms were applied to develop and validate an ac4C‐related gene signature (ARGSig) for prognosis prediction across multiple independent cohorts. Results Cells with high ac4C activity showed increased intercellular communication and activation of tumor‐associated pathways. The ARGSig model effectively stratified patients by survival outcomes and predicted sensitivity to immune checkpoint inhibitors and chemotherapy agents. Conclusion ac4C modification and its related genes play a critical role in lung adenocarcinoma development. The ARGSig model provides a promising molecular tool for prognosis evaluation and personalized treatment guidance in lung adenocarcinoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b6722e2c14c1989978a512d01d29289da459bd" target='_blank'>
              Integrating Single‐Cell Transcriptomics and Machine Learning to Define an ac4C Gene Signature in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Yuan Wang, Wei Su, Guangyao Zhou, Yijie Wang, Chunnuan Wu, Pengpeng Zhang, Lianmin Zhang
          </td>
          <td>2025-08-01</td>
          <td>Thoracic Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer invasion and subsequent metastasis to distant tissues occur when cancer cells lose cell–cell contact, develop a migrating phenotype, and invade the basement membrane (BM) and the extracellular matrix (ECM) to penetrate blood and lymphatic vessels. The identification of the mechanisms which induce the development from a ductal carcinoma in situ (DCIS) to a minimally invasive breast carcinoma (MIBC) is an emerging area of research in understanding tumor invasion and metastatic potential. To investigate the progression from DCIS to MIBC, we analyzed peritumoral collagen architecture using correlative scanning electron microscopy (SEM) on histological sections from human biopsies. In DCIS, the peritumoral collagen organizes into concentric lamellae (‘circular fibers’) parallel to the ducts. Within each lamella, type I collagen fibrils align in parallel, while neighboring lamellae show orthogonal fiber orientation. The concentric lamellar arrangement of collagen may physically constrain cancer cell migration, explaining the lack of visible tumor cell invasion into the peritumoral ECM in DCIS. A lamellar dissociation or the development of small inter fiber gaps allowed isolated breast cancer cell invasion and exosomes infiltration in the DCIS microenvironment. The radially arranged fibers observed in the peri-tumoral microenvironment of MIBC biopsies develop from a bending of the circular fibers of DCIS and drive a collective cancer cell invasion associated with an intense immune cell infiltrate. Type I collagen fibrils represent the peri-tumoral nano-environment which can play a mechanical role in regulating the development from DCIS to MIBC. Collectively, it is plausible to suggest that the ECM effectors implicated in breast cancer progression released by the interplay between cancer, stromal, and/or immune cells, and degrading inter fiber/fibril hydrophilic ECM components of the peritumoral ECM, may serve as key players in promoting the dissociation of the concentric collagen lamellae.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca42ff86f8ea9dc2e92ec4093986b47c8d0700cf" target='_blank'>
              Ultrastructural Changes of the Peri-Tumoral Collagen Fibers and Fibrils Array in Different Stages of Mammary Cancer Progression
              </a>
            </td>
          <td>
            Marco Franchi, Valentina Masola, Maurizio Onisto, Leonardo Franchi, Sylvia Mangani, V. Zolota, Z. Piperigkou, Nikos K. Karamanos
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Invasive lobular carcinoma (ILC) is the second most common histological subtype of breast cancer after invasive ductal carcinoma of no special type (NST), accounting for 10–15% of diagnoses. Despite the myriad molecular, histological and clinical differences between ILC and NST tumors, patients are treated in the same way, and although prognosis initially is good, ILC patients have poorer long-term outcomes. Understanding the differences between these two subtypes and identifying ILC-enriched therapeutic targets is necessary to improve patient care. Methods Human and mouse cancer-associated fibroblasts (CAFs), ILC cell lines and patient-derived organoids were used for in vitro and in vivo studies, including western blotting, migration, organotypic invasion assays and dissemination in zebrafish embryos. RNASeq was used to identify CAF and interleukin-6 (IL-6)-derived gene signatures. Bioinformatic analysis of public databases and immunohistochemical of human tumor microarrays was carried out. Results We identified IL-6 as a paracrine CAF-derived factor that activates Signal-Transducer-and-Activator-of-Transcription-3 (STAT3) in human and mouse ILC models. Analysis of human breast tumors showed that the IL-6/JAK/STAT3 pathway is enriched in ER + ILC compared to ER + NST. A 42-gene CAF dependent IL-6 gene signature and 64-gene consensus IL-6 gene signature were generated and were significantly enriched in ER + ILC, with many of the genes overexpressed in ILC tumors. IL-6 treatment suppressed downstream estrogen signaling and also led to the acquisition of a more mesenchymal-like phenotype associated with increased migration and invasion. Finally, IL-6 treatment significantly increased ILC cell dissemination following injection into zebrafish embryos. Conclusions CAF-derived IL-6 drives paracrine activation of the IL6/JAK/STAT3 signaling pathway which is enriched in ILC. This leads to the acquisition of pro-tumorigenic phenotypes, highlighting the pathway as a potential therapeutic target in ILC. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02074-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ff0735a08b68cfe9b8184d5f4b41c0bb6234a9" target='_blank'>
              Cancer-associated fibroblast driven paracrine IL-6/STAT3 signaling promotes migration and dissemination in invasive lobular carcinoma
              </a>
            </td>
          <td>
            Esme Bullock, Aleksandra Rozyczko, Sana Shabbir, Ifigenia Tsoupi, Adelaide I J Young, J. Travnickova, Laura Gómez-Cuadrado, Zeanap Mabruk, Giovana Carrasco, Elizabeth Morrow, K. Pennel, Pim Kloosterman, J. Houthuijzen, Jos Jonkers, L. Avalle, Valeria Poli, R. Iggo, Xue Xiao, Jingjing Guo, Xuan Zhu, Elizabeth Mallon, Joanne Edwards, E. E. Patton, Valerie G. Brunton
          </td>
          <td>2025-07-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Stromal cells are found in all tissues of the body. Among them, lymphoid stromal cells (LSCs) correspond to the cell subsets found in secondary and tertiary lymphoid organs. LSC heterogeneity has been characterized in depth in mice based on cell-fate mapping, high-resolution imaging and single-cell RNAseq analysis, and more recently in humans despite the difficulty of accessing these rare cell populations. At steady-state, LSCs organize discrete anatomical niches in lymphoid organs and orchestrate adaptive immune response. Studies of LSCs at the single cell level have identified a wide role for these cells in various pathological conditions, including solid tumors, autoimmune diseases, and lymphomas. In this review, we will discuss the diversity and plasticity of LSCs and LSC-like cells as well as their functions in pathological settings, with a focus on cancer and autoimmune diseases. Altogether, it highlights the importance of increasing our understanding of these cells, to use them as a target for novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae385fc73dd8ff4540908af0f79aa0db93c9021" target='_blank'>
              Lymphoid stroma in all its states
              </a>
            </td>
          <td>
            Nicolas Barbier, Valentin Isen, Karin Tarte, David Roulois
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Many studies have shown that pancreatic cancer is one of the cancers with extremely high mortality. The poor prognosis and lack of early diagnostic methods remain major challenges in the treatment of this cancer. In this article, in order to better detect the occurrence of pancreatic cancer, tools such as monocle3, singleR, harmony in R, and scanpy in Python were used to analyze the cells and genes of pancreatic cancer tissues in mice. By analyzing the data, the following results were obtained: T-cells in healthy PBMCs exhibited broader spatial dispersion than in PDAC tissues, suggesting tumor-driven immune surveillance impairment, while PDAC-associated macrophages displayed expanded distribution linked to pro-tumorigenic functions such as COL1A1-mediated ECM remodeling; Pseudotemporal trajectory analysis revealed myeloid progenitor bifurcation into monocytes/macrophages, with PDAC macrophages showing epigenetically silenced cytotoxic pathways such as suppressed GZMA/NKG7 and enhanced ribosomal biogenesis; Tissue-specific markers such as LCN2 in healthy and CTRB1/AMY2A in PDAC) and spatial co-localization of macrophages/tumor cells highlighted NOP53 as a dual-function hubinhibiting PI3K-AKT while activating p53and SPP1 as a paradoxical regulator of metastasis and antitumor immunity; Differential expression and GO enrichment analyses identified ribosomal biogenesis and cytoplasmic translation as PDAC-enriched pathways, contrasting with suppressed stress responses. Our spatial transcriptomic profiling further resolved elevated NOP53, CFB, and SPP1 expression gradients in PDAC tissues, proposing these as diagnostic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab94aa8cc2272f4a29c06caf0dcfe1338e28a0a0" target='_blank'>
              Spatiotemporal Transcriptomic Dissection of Tumor-Associated Macrophage Heterogeneity and Dual-Function Molecular Nodes in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Mengfei Li
          </td>
          <td>2025-06-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the ever-evolving landscape of cancer therapy, while cancer treatments such as chemotherapy, radiotherapy, and targeted therapy aim to eradicate malignant cells, they also inadvertently trigger cellular senescence in both cancerous and microenvironmental tissues. Therapy-induced senescence (TIS) can act as a barrier against tumor growth by halting cell proliferation in the short term, but the long-term persistence of therapy-induced senescent (TISnt) cells may pose a significant challenge in cancer management. Their distinct characteristics, like senescence-associated secretory phenotype (SASP), metabolic dysregulation, and immune evasion, make them exhibit remarkable heterogeneity to orchestrate the tumor microenvironment (TME), resulting in therapy resistance. However, how these TISnt cells functioning differently in cancer progression, and the intricate mechanisms by which they remodel the senescence-associated immunosuppressive microenvironment present challenges for improving anticancer therapy. Therefore, this review summarizes the heterogeneous TISnt cell phenotypes contributing to an accumulated senescent state, outlines their multidimensional interactions in the senescent microenvironment, and discusses current senescence-targeting strategies. Building on the current understanding of TIS, we propose potential avenues for improving TIS-targeting methodologies in the context of head and neck cancer, a representative heterogeneous malignancy, which can substantially enhance the efficacy of the “one-two punch” sequential treatment approach for head and neck cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378a355476ab29ee4164f4bc25cdfced24706c11" target='_blank'>
              Persistent accumulation of therapy-induced senescent cells: an obstacle to long-term cancer treatment efficacy
              </a>
            </td>
          <td>
            Jingjing Luo, Tongxu Sun, Zhenghui Liu, Yangfan Liu, Junjiang Liu, Shimeng Wang, Xueke Shi, Hongmei Zhou
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Oral Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="For years, valuable clinical samples preserved in formalin-fixed paraffin-embedded tissues were underutilized. However, with advanced spatial multiomics profiling tools, crucial information has become increasingly accessible. Integrating genomic data with spatial information has unveiled crucial insights into cellular activities, enhancing our comprehension of biology. Measuring cellular gene expression while capturing spatial context – including morphology and intercellular relationships – is vital for understanding both normal and diseased biological processes. To date, this approach has illuminated the mechanisms of complex diseases, such as cancer and has facilitated the discovery of biomarkers for early disease detection and new therapeutic targets, accelerating progress in cancer immunotherapies. Cutting-edge single-cell analysis tools are rapidly emerging as the gold standard for investigating intricate biological systems and medical specimens, fueling a multi-billion-dollar industry. Single-cell spatial research, in particular, is inherently cross-disciplinary and addresses questions that remain hidden when focusing solely on the genome or transcriptome of large cell populations. Leveraging advances in single-cell spatial profiling can offer insights into improving cancer immunotherapy and other modern medical treatments. This review will delve into the diverse applications of spatial profiling technology, showcasing examples that demonstrate its ability to provide a detailed picture of the underlying molecular and cellular mechanisms within cells. As a comprehensive reference, this review empowers researchers and industry leaders to harness single-cell and spatial omics for breakthroughs in biomedicine and translational science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33ed8c77341ba16d0b5a75b9927877b08daea9ef" target='_blank'>
              In situ spatial profiling: Gaining molecular and cellular insights within content and context
              </a>
            </td>
          <td>
            Shu-Ti Lin, Ian Molyneaux, Chen Yeh
          </td>
          <td>2025-07-10</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is a leading cause of cancer-related deaths worldwide. Despite advancements in treatment, prognosis for patients with advanced stages remains poor. Metabolic reprogramming in the tumor microenvironment, particularly abnormal glycolysis, plays a crucial role in immune evasion and treatment response. We collected nine single-cell datasets to create a single-cell atlas of CD8 + T cells from 89 NSCLC patients, revealing ten distinct states of these cells. We employed a multimodal data analysis approach, integrating bulk transcriptomics, single-cell transcriptomics, spatial transcriptomics, and proteomics. Using 117 machine learning models, we identified key genes associated with NSCLC metastasis. Notably, the StepCox[forward] + Lasso model was instrumental in pinpointing key genes that significantly impact disease prognosis. Our analysis revealed that LTB + LDHA + CD8 + T cells have a distinct metabolic and immune phenotype, characterized by enhanced glycolysis and elevated lactate production. This not only facilitates tumor cell migration and invasion but also impairs the cytotoxic function of CD8 + T cells. Furthermore, our machine learning models identified four key genes significantly associated with NSCLC metastasis: TBCD, PTPRC, LDHA, and ACTR2. Of these, high LDHA expression was strongly linked to poorer responses to immunotherapy and a higher risk of therapy resistance. LTB + LDHA + CD8 + T cells also reduced antitumor immune responses by inhibiting the secretion of effector molecules like GNLY. Additionally, elevated LDHA expression was associated with reduced CD8 + T cell infiltration, which further promotes tumor immune evasion. This study highlights the heterogeneity of CD8 + T cells in NSCLC, emphasizing the unique role of the LTB + CD8 + Tn subpopulation in metastasis. LDHA is identified as a critical key gene with a significant impact on immunotherapy outcomes, presenting a potential therapeutic target. These insights offer new biomarkers and targeted strategies for personalized immune therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e078cc67ce9052ada65f40295ba6296864dfb9f6" target='_blank'>
              Metabolic alterations driven by LDHA in CD8 + T cells promote immune evasion and therapy resistance in NSCLC
              </a>
            </td>
          <td>
            Hao Wen, Panpan Zhang, Juan Zhao, Yakui Liu, Lei Wan, Haoran Li, Jun Yi, Xinqiang Li
          </td>
          <td>2025-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e9ad63d60d3424441f26d5aee0efbd26af0b9f" target='_blank'>
              A single-cell atlas of ribosomal protein heterogeneity across human tissues reveals phenotypes of biological and clinical significance
              </a>
            </td>
          <td>
            Aishwarya Murali, Chartsiam Tipgomut, Anand D. Jeyasekharan, Himanshu Sinha
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="BACKGROUND
The effectiveness of PD-1/PD-L1 immune checkpoint blockade therapy in glioblastoma (GBM) is limited due to the tumor immunosuppressive microenvironment (TIME). Therefore, strategies of reprogramming TIME to a proinflammatory state offers a promising therapeutic approach.


METHODS
We applied bioinformatics analysis of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (stRNA-seq) to identify a significant accumulation of a cancer-associated fibroblasts (CAFs) subcluster with elevated LRRC15 expression in the nonresponders to anti-PD-1 therapy. Molecular mechanism of LRRC15 were functionally validated in vitro and in vivo.


RESULTS
These CAFs subcluster drive the infiltration of macrophages (Mφ) into the tumor microenvironment and promote their polarization toward the M2 phenotype. Deletion of Lrrc15 in CAFs significantly restrained tumor growth and prolonged survival in mouse models. Mechanistically, LRRC15 in CAFs promotes IL8 expression by activating the downstream FAK/SRC/NF-κB pathways, leading to Mφ migration and M2-like polarization. In turn, M2-like Mφs secrete TGF-β, which induces LRRC15 expression in CAFs via SMAD2-dependent transcriptional activation. Targeting CAFs subcluster with elevated LRRC15 expression in combination with anti-PD-1 treatment enhanced antitumor efficacy.


CONCLUSIONS
Our findings suggest that targeting LRRC15 may provide a novel strategy to augment anti-PD-1 therapy and overcome immunotherapy resistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1506f6c672d0a05f16addb76763bcd00e0bd9a3b" target='_blank'>
              CAF-derived LRRC15 orchestrates macrophage polarization and limits PD-1 immunotherapy efficacy in glioblastoma.
              </a>
            </td>
          <td>
            Feifei Luo, Y. Mei, Yanwen Li, Jiahao Yang, Shaoyan Xi, Endong Cao, Cong Shen, Dexiang Zhou, Peng Wang, Dong Zhou, Haiping Cai
          </td>
          <td>2025-06-26</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer, predominantly non-small cell lung cancer (NSCLC), remains a principal driver of cancer-related morbidity and mortality worldwide. Despite advancements in surgery, radiotherapy, chemotherapy, and targeted treatments, outcomes remain poor in advanced NSCLC. The tumor microenvironment (TME) exerts a critical influence on therapy responses. Within the TME, immune cells such as T and B lymphocytes, dendritic cells, myeloid-derived suppressor cells, tumor-associated macrophages, neutrophils, and natural killer cells can drive both pro- and anti-tumor processes. This review integrates their classification, phenotypic plasticity, and roles in NSCLC, highlighting key preclinical and clinical evidence while discussing pathogenesis, prognostic significance, and therapeutic potential. We also summarize the current immunotherapeutic strategies for advanced NSCLC, including first- or second-line regimens with immune checkpoint inhibitors alone or combined with chemotherapy, anti-angiogenic agents, or additional checkpoint inhibitors, and future directions. By elucidating the interplay between the NSCLC immune microenvironment and emerging immunotherapies, this review emphasizes the need for novel combination regimens and robust predictive biomarkers to improve clinical outcomes and extend survival in advanced NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d594d55b67aa26998978e57ef768648090cd4cd" target='_blank'>
              Tumor-infiltrating lymphocytes in NSCLC: from immune surveillance to immunotherapy
              </a>
            </td>
          <td>
            Haiyi Xue, Yilan Fan, Yi Li, Qian Zhao, Xuelu Zhang, Pei Zhao, Zhenjun Liu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The anti-tumor role of the immune system has long been associated with interferon-γ-mediated activation of immune cells and their ability to recognize and eliminate transformed cells. Fundamental principles of tumor immunoediting describe a dynamic interplay between the immune system and neoplastic cells, wherein immune pressure can paradoxically shape tumor evolution. Within this context, macrophages, natural killer cells, and T lymphocytes are central effectors of anti-tumor immunity. Traditionally, macrophages exhibiting M1 phenotype are characterized by high cytotoxic potential and considered important contributors to tumor eradication. In contrast, M2-polarized tumor-associated macrophages are associated with immune suppression and tumor progression. However, recent evidence challenges this binary paradigm. It is increasingly evident that M1 macrophages, while initially exerting anti-tumor effects, can also promote tumor progression by applying sustained cytotoxic pressure that selects for more malignant and immune-resistant tumor clones. This phenomenon represents an unexpected and overlooked contribution of cytotoxic macrophages to tumor progression. In this review, we examine the complex, context-dependent function of M1 macrophages and reassess current strategies aimed at enhancing their cytotoxicity. While such approaches may offer short-term benefits, they risk driving clonal selection of aggressive, immune-evasive tumor cells. Therefore, we propose a paradigm shift: instead of promoting M1 polarization alone, therapeutic strategies should consider the broader consequences of macrophage–tumor interactions. A nuanced understanding of macrophage plasticity and tumor dynamics is essential for designing effective immunotherapies. Recognizing the paradoxical role of M1 macrophages is critical to avoiding unintended support of tumor evolution and improving treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d17eeff6a62b58600242672f5016da3be414f80d" target='_blank'>
              M1 macrophages – unexpected contribution to tumor progression
              </a>
            </td>
          <td>
            Olga V. Kovaleva, M. Rashidova, V. V. Sinyov, O. S. Malashenko, Alexei N. Gratchev
          </td>
          <td>2025-07-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Skin melanoma is a malignant tumor that becomes increasingly difficult to treat when diagnosed late. Previously, we demonstrated that high levels of gelsolin (GSN) correlate with the advanced stages of cutaneous melanoma. GSN can be produced as various isoforms due to alternative splicing and differing start codon positions. To date, no studies have been conducted to determine whether GSN diverse isoforms are produced by melanoma cells in vivo and melanoma cell lines. Therefore, nothing is known about the role of specific GSN isoforms in skin melanoma biology. Methods We applied immunocytochemical staining to melanoma tissue samples to analyze the localization of GSN production within tumor samples. Additionally, we utilized bioinformatics analysis of transcript levels coding for selected GSN isoforms in publicly available gene transcript databases. To investigate the role of these GSN isoforms, we used the melanoma A375 cell line with GSN knockout to restore the production of only one GSN isoform at a time in these cells. We evaluated the modified cells' ability to migrate, spread, and regulate actin polymerization. We also tested the cells growing on laminin 1, a significant component of the basement membrane, and the melanoma microenvironment. Results We found that GSN expression produces three GSN isoforms in human melanoma cell lines: two cytosolic (B and C) and one secretory (A). Furthermore, we noted the presence of GSN both intracellularly and extracellularly in melanoma tumor samples, indicating that human melanoma cells produce diverse GSN isoforms in vivo. We discovered that cells producing GSN-A invade more efficiently, while cells producing GSN-C form the longest filipodia and migrate the best in 2D conditions. Both GSN-B and -C decrease the amount of filamentous actin. On the other hand, cells producing GSN-A and –B exhibit a lower proliferation rate. Finally, we observed that tumors formed by the clones expressing individual GSN isoforms do not grow in zebrafish embryos. Conclusions Overall, we demonstrate that GSN isoforms are produced as a mixture in melanoma cells and are not redundant in their function. Therefore, to support the well-being of melanoma cells, a mixture of GSN isoforms must be produced. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03876-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83b888f3d4df7ce78b6b13facfbbf8c12ca0fe86" target='_blank'>
              Skin melanoma cells produce diverse gelsolin (GSN) isoforms, which play non-redundant roles in cells’ proliferation and motility
              </a>
            </td>
          <td>
            Ewa Mazurkiewicz-Stanek, A. Makowiecka, Iryna Kopernyk, Michał Majkowski, A. Boguszewska-Czubara, Tomasz Trombik, Paweł Karpiński, Piotr Donizy, A. Mazur
          </td>
          <td>2025-07-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a common hematologic malignancy characterized by high tumor cell heterogeneity, which significantly impacts the clinical prognosis of patients. Angiogenesis and the molecular features of tumor cells play a critical role in tumor progression and drug resistance. This study aims to explore the impact of tumor cell heterogeneity and angiogenesis-related genes on the prognosis of MM. Methods We collected transcriptomic data and single-cell RNA sequencing data from MM patients through the Xena and GEO databases. The data were processed and analyzed using bioinformatics methods, including differential gene expression analysis, single-cell clustering, CNV analysis, transcription factor analysis, screening of angiogenesis-related genes, cell communication analysis, and immune infiltration analysis. Results Through integrative analysis of transcriptomic data and single-cell RNA sequencing data, we identified significant genomic copy number variations in the tumor cells of MM patients. Additionally, different tumor subgroups exhibited differences in angiogenic activity, gene expression, and tumor progression. Notably, high expression of the transcription factor cAMP-responsive element-binding protein 3-like 2 (CREB3L2) in the C1 subgroup was associated with the inhibition of angiogenesis and tumor cell proliferation and migration. Furthermore, the prognostic model based on angiogenesis and transcription factors demonstrated high accuracy in predicting the prognosis of MM patients. Conclusion This study highlights the critical roles of tumor cell heterogeneity and angiogenesis-related genes in MM. By constructing a prognostic model, it provides new theoretical insights for the precise diagnosis and personalized treatment of MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e6c99518d8d27afb5f7b1ce9d91ef059b7ce3d9" target='_blank'>
              Investigating the role of tumor cell heterogeneity and angiogenesis genes in the prognosis of multiple myeloma
              </a>
            </td>
          <td>
            Xue Qiao, Zhengrong Song, Li Geng, Lina Xing, Ying Wang
          </td>
          <td>2025-06-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Head and neck squamous cell carcinoma is a challenging cancer that often resists treatment due to its ability to escape the body’s immune defense. This review explains how the tumor’s surrounding environment—made up of immune cells, blood vessels, and structural proteins—helps the cancer grow, spread, and avoid immune attack. We aim to highlight how different components in this environment, including fibroblasts, suppressive immune cells, and genetic changes like TP53 mutations, contribute to disease progression. By understanding these mechanisms, researchers can develop more effective therapies that target not just the cancer cells, but also the supportive environment around them. This review may guide future research in creating personalized treatments and improving outcomes for patients with this aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec6e0af15a343b955ed9de45b18fba4fac507f2" target='_blank'>
              Immune Evasion in Head and Neck Squamous Cell Carcinoma: Roles of Cancer-Associated Fibroblasts, Immune Checkpoints, and TP53 Mutations in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Chung-Che Tsai, Yi-Chiung Hsu, Tin-Yi Chu, Po-Chih Hsu, Chan-Yen Kuo
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Breast cancer (BRCA) is characterised by complex cellular heterogeneity and differentiation hierarchies, which play a crucial role in bone metastasis and therapeutic resistance. However, existing classification systems remain inadequate in capturing these complexities, limiting their effectiveness in guiding treatment strategies. To address this gap, we integrated single‐cell RNA‐seq profiles, spatial transcriptomes, along with 1097 bulk RNA‐seq profiles of TCGA‐BRCA cohort to dissect the molecular landscape of BRCA. By performing UMAP analysis, we identified nine tumour clusters and three spatially distinct spot types (immune, stromal and malignant spots) and further delineated 11 differentiation states from 2493 malignant spots. Through clustering, monocle 2 pseudo‐time and prognostic analyses, we identified the prognostic BRCA cell fate‐related genes, then constructed a novel BRCA stratification system (four subtypes) with differential prognosis, biological plausibility and clinical significance. Also, least absolute shrinkage and selection operator (LASSO) regression analysis was performed for the BRCA cell fate‐related genes in constructing a prognostic model. The model has modest accuracy and accordance (AUC = 0.708), which could distinguish BRCA patients into high‐risk or low groups. With correlation analysis, regulation networks were constructed for different subtypes based on the key cell fate‐related genes, transcription factors, metastasis‐related pathways, immune components and so on, to investigate the regulatory relationships between primary BRCA and BRCA bone metastasis. Afterwards, we identified the most significant inhibitors (puromycin, MS‐275, megestrol, aesculetin) for bone metastatic BRCA, which might have potential translational significance. In all, we developed a novel molecular stratification system for BRCA based on the cell fate‐related markers of malignant cells, which offered strong translational potential for diagnosis, prognosis and personalised therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1838f6be81f1e41cf0fcf9935ae12d48d0ddb8c" target='_blank'>
              Molecular Subtypes and Risk Prediction Model Based on Malignant Cell Differentiation Trajectories in Breast Cancer
              </a>
            </td>
          <td>
            Penghui Yan, Hanlin Sun, Siqiao Wang, Runzhi Huang, Chaofeng Shi, Qihang Yang, Yibo Qiao, Haonan Wang, Deqian Kong, Jiwen Zhu, Yunqing Yang, Zongqiang Huang
          </td>
          <td>2025-08-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The mitotic inhibitor docetaxel (DTX) is often used to treat endocrine-refractory metastatic breast cancer, but initial responses are mitigated as patients eventually have disease progression. Using a cohort of ex vivo cultures of circulating tumor cells (CTCs) from patients with heavily pretreated breast cancer (n=18), we find two distinct patterns of DTX susceptibility, independent of clinical treatment history. In CTCs cultured from some patients, treatment with a single dose of DTX results in complete cell killing, associated with accumulation of non-viable polyploid (≥8N) cells arising from endomitosis. In others, a transient viable drug-tolerant persister (DTP) population emerges, ultimately enabling renewed proliferation of CTCs with preserved parental cell ploidy and DTX sensitivity. In these CTC cultures, efficient cell cycle exit generates a ≤4N drug-tolerant state dependent on CDKN1B (p27Kip1). Exposure to DTX triggers stabilization of CDKN1B through AKT-mediated phosphorylation at serine 10. Suppression of CDKN1B reduces the number of persister CTCs, increases ≥8N mitotic cells and abrogates regrowth after DTX exposure. Thus, CDKN1B-mediated suppression of endomitosis contributes to a reversible persister state following mitotic inhibitors in patient-derived treatment refractory breast cancer cells. Summary in bullets Transient DTX tolerant persister cells emerge in some patient-derived cultured CTCs. DTX-tolerant persisters restrict endoreduplication and polyploidy through CDKN1 (p27kip1). DTX exposure induces CDKN1B stabilization through AKT mediated phosphorylation at serine 10. Suppression of polyploidy underlies a drug tolerant persister state specific to mitotic inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c80ac0aa94dfd6f02a8336f8c8d7fde16ba668bb" target='_blank'>
              CDKN1B (p27/kip1) enhances drug-tolerant persister CTCs by restricting polyploidy following mitotic inhibitors
              </a>
            </td>
          <td>
            Elad Horwitz, Taronish D. Dubash, Annamaria Szabolcs, Ben S. Wittner, J. Kreuzer, Robert Morris, Aditya Bardia, Brian Chirn, Devon F. Wiley, Dante Che, Hunter C. Russel, Xcanda Ixchel Herrera Lopez, Douglas B. Fox, Ezgi Antmen, David T. Ting, Wilhelm Haas, Moshe Sade-Feldman, S. Maheswaran, Daniel A Haber
          </td>
          <td>2024-02-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="The use of CRISPR-Cas9 to engineer cancer cell lines has made it possible to precisely examine how cancer cells react to different drugs and therapies. Some of the key improvements are in the use of Mediator Complex Subunit 12 (MED12)-knockout cells to study cell resistance to BRAF inhibitors, CRISPR models of Epithelial-Mesenchymal Transition for breast cancer, and pharmacogenomic analysis in various cancer cell lines. CRISPR is used in immunotherapy to help Chimeric Antigen Receptor T (CAR-T) cells function better by disrupting the immune checkpoints like Programmed Cell Death Protein 1 (PD-1) and Cytotoxic T-lymphocyte- associated protein 4 (CTLA-4) and to adapt T cells to react with various antigens. As a result of these innovations, it is now possible to track how cancers like non-small cell lung cancer (NSCLC) and ovarian cancer evolve, change their epigenetic features, and find strategies to reverse their resistance. Moving forward, mixing AI analytics, single-cell multi-omics, patient-derived organoids, and CRISPR mechanisms will help improve precision oncology and speed up effective treatment planning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95f957c61adab36c7dec0eefa4bae427dbcc6b3" target='_blank'>
              CRISPR-Edited Cell Lines: A New Era in Functional Oncology Research.
              </a>
            </td>
          <td>
            Amita Joshi Rana, M.D.S.A.D.I.Q.U.E. Hussain, Ali Hanbashi, Faroq Kamli, Gyas Khan, Marwa Qadri, Saeed A Al-Qahtani, Mohammad Gayoor Khan, Sushil S. Burle, Vikas Jakhmola, Gaurav Gupta
          </td>
          <td>2025-08-13</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="OBJECTIVE
Adamantinomatous craniopharyngioma (ACP) is an epithelial tumor that occurs in the sellar region of the human brain. Despite resection, relapse is frequent with poor prognosis. To facilitate the development of new therapy for ACP, the authors examined the spatial distribution, cell of origin, and potential biological functions of leukemia inhibitory factor (LIF), an important stem cell self-renewal regulator, in a series of ACP tumors.


METHODS
The transcriptional sites of LIF and LIF receptor (LIFR) were determined by single-cell sequencing and space transcriptome analysis. LIF and LIFR distribution characteristics in different histopathological regions were detected with immunohistochemistry and immunofluorescence analysis. The relationships between the regional distributions of different tissues and tumor imaging characteristics, tumor cell stemness, cell proliferation, LIF expression, and patient prognosis were analyzed.


RESULTS
The authors' analysis of 39 ACPs detected LIF overexpression that was selectively enriched in cell clusters. In addition to the discovery of the stromal cells in the interstitial region, palisade epithelium, and stellate reticulum as the source cells of LIF production, the authors also revealed that LIFR was primarily generated by the cell clusters. Examination of differentially expressed genes between LIF-high and LIF-low ACP tumors indicated that the binding of LIF to LIFR may lead to the activation of the PI3K-AKT signaling pathway. Further analysis showed enrichment of LIF expression in β-catenin-positive cell clusters expressing stem cell markers of CD44, supporting its role in stem cell self-renewal. Integrated analysis with diagnostic imaging found higher level expression of LIF in cystic tumors than in solid tumors, displaying a trend toward poorer prognosis.


CONCLUSIONS
This study confirmed for the first time that LIF in ACP mainly originates from tumor microenvironment stroma. The authors' data suggest that future efforts should also include tumor stromal cells as a novel cellular and/or molecular target when developing new anticancer therapies against ACP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e38d490dae9950f240b893ac7de6f57947b7a0" target='_blank'>
              Use of integrated spatial transcriptomics and histopathological analysis in adamantinomatous craniopharyngiomas to identify stromal cells as a new cellular source of leukemia inhibitory factor.
              </a>
            </td>
          <td>
            Wenxin Hu, Chuan Zhao, Wenrong Zheng, Yi Lin, Ning Luo, Hongxing Liu, Xingfu Wang, Xueling Qi, Xianlong Wang, Xiao-Nan Li, Zhixiong Lin
          </td>
          <td>2025-08-01</td>
          <td>Journal of neurosurgery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT
Circulating tumor cells (CTCs), shed from the primary malignancies, are regarded as the "seeds" of tumor metastasis. They employ sophisticated strategies to evade immune detection during blood circulation by engaging with various blood components, ultimately facilitating their colonization in distant organs. Elucidating the mechanisms underlying CTC immune evasion may unlock novel immunotherapeutic strategies to prevent tumor metastasis. Recent advancements in CTC isolation and single-cell sequencing have provided insights into their complex microenvironment and immune evasion mechanisms. Many strategies for targeting either CTCs or their associated blood cells have been explored. Here, we systematically delineated the interaction network between CTCs and the diverse blood cells. By elucidating the microenvironmental and biological characteristics of CTCs, we summarize several potential immune evasion mechanisms, including immune checkpoint modulation, CTC clustering, platelet interactions, etc. Additionally, we highlight recent advances in intervention strategies targeting CTCs and the clinical application of CTCs in cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d94128a4ca71f38899bbd2a679a0555681c49d19" target='_blank'>
              Microenvironment crosstalk and immune evasion of circulating tumor cells: From mechanism to clinical significance.
              </a>
            </td>
          <td>
            Shi Chen, Sirui Zhang, Ying Shi, Xiaowei Liu
          </td>
          <td>2025-08-08</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND AND OBJECTIVES
Field cancerisation is the process that results in a group of cells acquiring some of the phenotypic changes of cancer prior to transformation into cancer. Clinically, an important challenge remains the ability to distinguish clonal lineages and microenvironments within cancerised fields that will remain indolent from those that will progress to malignant transformation. Spatial 'omics' can help us investigate genetic, epigenetic, transcriptomic, proteomic, and cellular microenvironments that transform normal cells into a cancerised field, and subsequently into cancer. In this review, we will discuss how spatial omics techniques have expanded our understanding of field cancerisation in prostate and bladder cancer, and the challenges associated with this research.


METHODS
We identified key articles relating to field cancerisation in bladder and prostate cancer. Special emphasis was placed on studies that used modern spatial profiling technologies and studies that were designed to investigate changes within normal tissue rather than simply using it as a control for tumour tissue.


RESULTS
Spatial omics research into field cancerisation has identified interesting early findings that have informed our understanding of: transformation of the benign epithelium and mechanisms of intra-prostatic clonal expansion for prostate cancer; clonal expansion within the normal urothelium; mutations that are unique to cancerised fields within the bladder; and how field cancerisation may prime the urothelium for cancer transformation.


CONCLUSIONS
Spatial omics profiling of field cancerisation can inform risk stratification and personalised treatment options. However, there are a number of challenges associated with the technologies that must be overcome before the potential of spatial omics can be fully realised in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957e9305f80e5b98aef1432b61f0ded66b1ee007" target='_blank'>
              What spatial omics is teaching us about field cancerisation in prostate and bladder cancer.
              </a>
            </td>
          <td>
            Matthew H V Byrne, T. Anbarasan, Lisa Browning, Dan Woodcock
          </td>
          <td>2025-06-25</td>
          <td>BJU international</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The growth of a tumor is tightly linked to the distribution of its cells along a continuum of activation states. Here, we systematically decode the activation state architecture (ASA) in a glioblastoma (GBM) patient cohort through comparison to adult murine neural stem cells. Modelling of these data forecasts how tumor cells organize to sustain growth and identifies the rate of activation as the main predictor of growth. Accordingly, patients with a higher quiescence fraction exhibit improved outcomes. Further, DNA methylation arrays enable ASA-related patient stratification. Comparison of healthy and malignant gene expression dynamics reveals dysregulation of the Wnt-antagonist SFRP1 at the quiescence to activation transition. SFRP1 overexpression renders GBM quiescent and increases the overall survival of tumor-bearing mice. Surprisingly, it does so through reprogramming the tumor’s stem-like methylome into an astrocyte-like one. Our findings offer a framework for patient stratification with prognostic value, biomarker identification, and therapeutic avenues to halt GBM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dee763118a03533523bd5d7836ea98324ad3594" target='_blank'>
              Cross-species comparison reveals therapeutic vulnerabilities halting glioblastoma progression
              </a>
            </td>
          <td>
            Leo Carl Foerster, Oguzhan Kaya, Valentin Wüst, Diana-Patricia Danciu, Vuslat Akcay, Milica Bekavac, Kevin Chris Ziegler, Nina Stinchcombe, Anna Tang, S. Kleber, Joceyln Tang, Jan Brunken, Irene Lois-Bermejo, Noelia Gesteira-Perez, Xiujian Ma, Ahmed Sadik, P. Le, Kevin Petrecca, Christiane A Opitz, Haikun Liu, C. R. Wirtz, Angela Goncalves, A. Marciniak-Czochra, Simon Anders, A. Martin-Villalba
          </td>
          <td>2025-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related mortality globally, with non-small cell lung cancer (NSCLC) representing 85% of cases. Advances in treatment modalities, including stereotactic radiation therapy, have improved outcomes. However, possible synergistic effects of these therapies remain underexplored at the molecular level. This study investigated high-dose radiation-induced proteomic changes in lung adenocarcinoma cell line HCC-44 grown adherently and cell line A549, grown as adherent cells or spheroids. Our hypothesis was that proteins upregulated by 10 Gy irradiation serve as resistance drivers in cancerous cells and can thus represent potential therapeutic targets. The label-free mass spectrometry revealed distinct proteomic responses to 10 Gy irradiation, varying by cell line and culturing conditions. Differentially expressed proteins elevated in the irradiated samples included ephrin type-A receptor 2 (EPHA2) in adherent cells and insulin-like growth factor 2 receptor, tetraspanin 3 as well as cathepsin D (CTSD) in spheroids. The validation of these targets was carried out via Western blot, immunofluorescence, viability assay and spheroid formation assay which demonstrated that irradiation sensitized adherent NSCLC cells to EPHA2 and CTSD inhibitors. These findings underscore the potential of integrating radiation and targeted therapies in NSCLC treatment, and highlight EPHA2 as a promising candidate for future therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-09285-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b394ae32bb3fba2e6d32961ffafe24b8e9f20a6" target='_blank'>
              Proteome changes associated with effect of high dose single-fractionation radiation on lung adenocarcinoma cell lines
              </a>
            </td>
          <td>
            Elina Leis, Vijayachitra Modhukur, H. Lust, M. Vardja, Merilin Saarma, Ivar Ilves, Margaret Pütsepp, Sergei Kopanchuk, Jana Jaal, D. Lavogina
          </td>
          <td>2025-07-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Cervical cancer’s tumor microenvironment (TME) was composed of a diverse array of immune cells that significantly influence tumor progression and response to treatment. Recent advancements in multi-omics and single-cell sequencing had provided valuable insights into the cellular heterogeneity and immune landscape of the TME, revealing critical interactions that shape tumor behavior and therapy outcomes. Method This study used multi-omics and single-cell sequencing to explore the immune landscape, cellular heterogeneity, and drug sensitivity in cervical cancer, focused on tumor subtypes and their interactions with immune cells, and aimed to understand therapy responses. Results The research presented a thorough single-cell analysis of cervical cancer, identified distinct tumor epithelial cell (EPC) subtypes, and explored their roles in tumor progression, immune evasion, and therapeutic response. It underscored the potential of tumor EPCs as valuable biomarkers for prognosis and as targets for personalized treatment approaches. Conclusion The immune landscape of cervical cancer and its interaction with tumor endothelial progenitor cells played crucial roles in determining the tumor’s progression and response to therapy. The classification of tumor subtypes based on immune characteristics and drug sensitivity was critical for personalized treatment. The identification of TSPAN1 as key biomarkers provided insight into tumor biology and potential therapeutic targets. Our findings emphasized the need for combining immune checkpoint modulation with precise drug sensitivity analysis to optimize treatment strategies, particularly in advanced cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db53e3e78ba135d282fa510797c202842bdafc26" target='_blank'>
              Revolutionizing cervical cancer treatment: single-cell sequencing of TSPAN1+ tumor EPCs and immune checkpoints to assess drug sensitivity and optimize therapy
              </a>
            </td>
          <td>
            Yumeng Li, Zhiheng Lin, Guangyao Lin, Zhijie Zhao, Zhikai Xiahou, Pingping Cai
          </td>
          <td>2025-07-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Compelling evidences have manifested that breast cancer cells prefer to metastasize to certain distant organs, including brain, lung, bone and liver. According to the canonical “seed and soil” theory, this prominent biological behavior, termed as metastatic organotropism, involves intricate interactions between breast cancer cells (the “seeds”) and specific residents in the tumor microenvironment (the “soil”), initiating from pre-metastatic niche formation to metastatic outgrowth. Recently, multifaceted heterogeneity of tissue-specific macrophages (TSMs) and their roles played in organotropic metastases of breast cancer are incrementally unveiled. Herein, we decipher multiple diversities of TSMs, including evolvement, profiles, functions and metabolic characteristics under different polarization states. Further, we elaborate on bidirectional effects of TSMs on metastatic organotropism of breast cancer (both to the “seeds” and “soil”), and unearth underlying signaling pathways based on updated mechanistic researches. Lastly, we compile a series of clinical trials, hoping to illuminate promising TSM-targeting therapies against breast cancer organotropic metastases. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ed79b9316a9fed3f14b3bb67e4e61ce4c40709" target='_blank'>
              Heterogeneous tissue-specific macrophages orchestrate metastatic organotropism of breast cancer: implications for promising therapeutics
              </a>
            </td>
          <td>
            Cenzhu Wang, Pinchao Fan, Yu Zhou, Meican Ma, Haowei Zhong, Lei Liu, Qin Chen, Kun Xu
          </td>
          <td>2025-06-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The vertebrate central nervous system is enveloped by the meninges, consisting of the pia, arachnoid, and dura layers. The arachnoid is hypothesised to give rise to the most common primary intracranial tumours, meningiomas. However, molecular evidence supporting this hypothesis is lacking. There are no effective medical therapies to treat meningiomas that are resistant to local interventions, encumbered by our limited understanding of their cellular origin. To address this limitation in our understanding of meningioma biology, we generated a comprehensive reference single cell and spatial transcriptomic atlas of human fetal meninges at post-conceptional weeks 5-13. We found that the meningeal layers develop concurrently, and identified an inner CDH1-positive dura cell layer expressing tight junction genes consistent with barrier function. We show that transcriptionally, meningioma cells resemble dura-lineage cells, and that common meningioma driver genes were expressed preferentially in the dura lineage. Our findings suggest that meningiomas originate from dura lineage cells. HIGHLIGHTS scRNA-seq and spatial transcriptomics reveals architecture of human fetal meninges development Meningeal layers are formed concurrently, by a gradual refinement of cell states A CDH1+ inner dura sublayer expresses tight junction genes like the arachnoid barrier Meningioma tumours likely originate from inner dura lineage cells, not arachnoid">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f22fec9d525d243d381da89dbcc0ee1ca3f50309" target='_blank'>
              Cell atlas of the developing human meninges reveals a dura origin of meningioma
              </a>
            </td>
          <td>
            Elin Vinsland, Sergio Marco Salas, Ivana Kapustová, Lijuan Hu, S. Webb, Xiaofei Li, Xiaolin He, Mats Nilsson, M. Haniffa, Roger A. Barker, Oscar Persson, D. Raleigh, Erik Sundström, P. Lönnerberg, Sten Linnarsson
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4144c066603d461af67a1609df3300fe3c044e71" target='_blank'>
              Single-cell and spatial transcriptomics reveal mechanisms of radioresistance and immune escape in recurrent nasopharyngeal carcinoma.
              </a>
            </td>
          <td>
            Rui You, Qunlun Shen, Chao Lin, Kangning Dong, Xiao Liu, Hanshi Xu, Wanming Hu, Yulong Xie, Ruoqi Xie, Xiaoyi Song, Chunliu Huang, Jinhui Wu, Tao Yu, Huifeng Li, Zining Wang, Keming Chen, Xiong Zou, Peiyu Huang, Yijun Hua, Youping Liu, Tianliang Xia, Shihua Zhang, Mingyuan Chen
          </td>
          <td>2025-07-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Gastric cancer (GC) is characterized by heterogeneous tumor microenvironment (TME) with various cell types contributing to disease progression and patient outcomes. This study aims to dissect the single-cell transcriptomic landscape of GC, highlighting the role of tumor-associated macrophages (TAMs) and establishing a novel prognostic signature based on high oxidative phosphorylation (OXPHOS) macrophages. Methods Single-cell sequencing data from paired GC and normal stomach tissues, obtained from the GEO database (GSE184198), were processed to reveal cellular heterogeneity and identify TAM subsets with high OXPHOS activity. Using the TCGA STAD dataset, survival analyses were conducted on 435 GC patients to establish a high-OXPHOS-macrophage-related prognostic signature. Results We identified eight distinct cell types within the GC TME, indicating significant cellular heterogeneity. Macrophages, particularly TAMs, were found in greater numbers in tumor tissue, with the C3 macrophage subset exhibiting the highest OXPHOS score. A 19-gene high-OXPHOS-macrophage-related prognostic signature was constructed, stratifying patients into different risk categories with significant survival differences (P<0.05). NPC2, LY96, and TPP1 were identified as key macrophage-expressed markers, correlating with prognosis. Cell communication analysis revealed increased interaction in tumor tissues, especially involving NPC2, LY96, and TPP1 positive macrophages, which facilitated tumorigenesis and immune evasion. Conclusion The high-OXPHOS-macrophage-related prognostic signature derived from scRNA-seq data provides valuable insights into GC patient stratification. NPC2, LY96, and TPP1, highly expressed in TAMs, were implicated in promoting tumor growth and immune escape, offering potential targets for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f9b92dd366657dc4590a10fb71fe958b8515ec7" target='_blank'>
              Prognostic implications of high- OXPHOS macrophages in gastric cancer: a single-cell transcriptomics and tumor microenvironment communication study
              </a>
            </td>
          <td>
            Ziyuan Lin, Yunyu Xu, Xiaohe Xu, Xinjun Lin, Lin Cheng, Qifeng Zhao
          </td>
          <td>2025-06-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Investigating the heterogeneous responses of individual cancer cells to chemotherapeutic drugs is crucial for deciphering the mechanisms of cancer drug resistance. In recent years, single-cell proteomics has demonstrated its significant capability in exploring drug response in cancer cells. Meanwhile, there are increasing reports suggesting that the cellular morphology is potentially associated with drug resistance. However, integrating the single-cell proteomic results with morphological information remains challenging. Here, we present a morphology-aware single-cell proteomic analysis (Morp-SCP) platform to precisely capture cells of interest with real-time and high-resolution imaging, and to conduct deep proteomic analysis at the single-cell level, providing multidimensional information on the target single cells. The Morp-SCP platform was applied for exploring the time-dependent proteomic alterations of human nonsmall cell lung cancer cells (A549) upon cisplatin exposure. Subpopulations of drug-resistant A549 cells were identified, which exhibited distinct proteomic and morphological patterns when resisting cell death induced by cisplatin exposure. By revealing the proteomics-morphology relationship, the Morp-SCP platform offers an effective strategy to provide insights into the heterogeneity of drug resistance at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/439c211eca558887357b434ba49c5af9cf3c9266" target='_blank'>
              Integrated Single-Cell Proteomic and Morphometric Analysis Reveals Heterogeneous Drug-Resistant Subpopulations.
              </a>
            </td>
          <td>
            Qian-Xi Fan, Yi-Rong Jiang, Jian-Bo Chen, Yi-Xue Chen, Qin-Qin Xu, Jie Wu, Huifeng Wang, Yi Yang, Jian-Zhang Pan, Qun Fang
          </td>
          <td>2025-07-07</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) occurs as the third most common cancer with high mortality across the world. Understanding the intratumoral immune cell heterogeneity and their responses to various therapies is crucial for enhancing patient outcomes. This study aimed to characterise and evaluate the immune microenvironment landscapes of CRC shaped by different therapies including CD73 inhibitor, PD‐1 blockade and photothermal therapy (PTT). Our investigation revealed that three therapies could commonly modulate the down‐regulation of Treg, M2 macrophage and Ptprj+ G4 granulocyte, up‐regulation of effector/memory T cell, M1 macorphage and Hilpda+ G1 granulocyte. Moreover, we identified the uniquely dis‐regulated cell types and pathway activities response to each therapy, such as CD73 inhibitor enriched more Cd8+ memory and central memory (CM) cell, PD‐1 blockade with more Cd8+ CTL and Cxcl3+ G2 granulocyte, and PTT with more Cd8+ effector memory and Rethlg+ G3 granulocyte cell. These responses disordered the glycolysis, angiogenesis, phagocytosis functions and cellular communication to reshape the CRC tumour immune microenvironment. We provide the detail insights into the intratumoral immunomodulation preferences of CRC mice treated with CD73 inhibitor, PD‐1 blockade and PTT therapies, which might contribute to the ongoing development of more effective anticancer strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ef48ff72788aaf2bb81620f30c069f2cf46018" target='_blank'>
              Characterising and Evaluating the Immune Microenvironment Landscapes of Colorectal Cancer Shaped by Different Therapies
              </a>
            </td>
          <td>
            Chen Zhou, Yifan Wang, Yuanyuan Li, Weitao Zhang, Yunmeng Bai
          </td>
          <td>2025-07-16</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Ovarian cancer (OC) remains the most lethal gynecologic malignancy in the United States due to its late diagnosis, aggressive nature, and poor responsiveness to existing therapies. Dissecting the molecular mechanisms and identifying molecular drivers of aggressiveness and therapy resistance is critical for devising new therapies and improving patient outcomes. Methods MYB expression was evaluated in a panel of OC cell lines by immunoblotting. Gain and loss of function studies were performed by developing stable control and forced-MYB-expressing and -silenced cell lines, respectively. Functional assays included growth kinetics, clonogenicity, cell cycle, live-dead cell measurements, and annexin-V staining, followed by flow cytometry, migration and invasion assays, and MTT assays following drug treatment. Gene expression profiling was done using the nanoString PanCancer Progression panel. Chromatin immunoprecipitation (ChIP) was performed to confirm MYB binding to the responsive gene promoter, followed by siRNA-mediated silencing to establish the intermediary role in potentiating the downstream effects. Results Low to high MYB expression was reported in all OC cell lines, with negligible expression reported in normal ovarian surface epithelial cells. MYB expression was significantly higher in aggressive (SKOV3-ip) and chemoresistant (A2780-CP) OC cell lines compared to the parental (SKOV3 and A2780) cells. Functional assays in MYB-overexpressing and -silenced OC cell lines demonstrated a role of MYB overexpression in increased cell proliferation, survival, migration, invasion, EMT, and chemoresistance. nanoString analysis and comparison of transcriptomic data of MYB-silenced SKOV3-ip and MYB-overexpressing SKOV3 cells with their respective control cells identified MYB-dependent genes. Interestingly, these target genes showed a limited overlap between cell lines, suggesting a cell-specific MYB-regulated gene regulation. AKT3 was consistently identified as a common MYB-regulated gene in multiple OC cell lines and confirmed as a direct transcriptional MYB target through confirmation of MYB binding to its promoter. Pathway analysis using the MYB-regulated transcriptomic data also identified PI3K/Akt signaling to be activated in MYB-overexpressing cells. siRNA-mediated silencing of AKT3 confirmed its role in potentiating the oncogenic actions of MYB in OC cells. Conclusion MYB/AKT3 axis drives ovarian cancer growth, aggressiveness, and chemoresistance, highlighting its potential as a therapeutic target in ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01761-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f923e08057dabd7b0590daa1c6d7ce8ebf7838e" target='_blank'>
              MYB/AKT3 axis is a key driver of ovarian cancer growth, aggressiveness, and chemoresistance
              </a>
            </td>
          <td>
            K. Vikramdeo, Orlandric Miree, Shashi Anand, Amod Sharma, S. K. Srivastava, Seema Singh, Rodney P. Rocconi, A. Singh
          </td>
          <td>2025-08-11</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="A compelling body of research has confirmed the critical role of the tumor microenvironment (TME) in the growth and aggressiveness of breast cancer (BC). Cancer-associated fibroblasts (CAF), the predominant cell component of the TME, significantly influences its behavior through complex crosstalk with cancer and other stromal cells in response to tumor-derived activating signals. Murine models are extensively used to investigate the mechanisms underlying CAF's role in BC progression, as they allow for the evaluation of the endogenous TME and the assessment of immune system involvement using syngeneic xenograft studies. Here, we present a set of methods for culturing BC-derived CAFs, manipulating gene expression, and evaluating their pro-tumorigenic functions using both in vitro and in vivo approaches. We also describe protocols for CAF gene expression profiling and transcriptomic analysis, alongside a computational procedure for inferring transcriptional signatures in primary tumors using publicly available bulk RNA-seq data. Additionally, we introduce a user-friendly, web-based tool for analyzing the transcriptomes of stromal and epithelial components from laser capture micro-dissected BC tumors. Ultimately, these methods can provide valuable insights into the molecular mechanisms driving CAF-tumor interactions and hold potential for identifying novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f94a97738d3f7846e0a2b6fcec7f408e4d3b367d" target='_blank'>
              Cancer-Associated Fibroblasts from Mouse Mammary Tumors as Tools for Molecular and Computational Studies.
              </a>
            </td>
          <td>
            Daniele Viavattene, Aurora Savino, Somayeh Mirzaaghaei, Andrea Lobascio, Valeria Poli, L. Avalle
          </td>
          <td>2025-07-03</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Oral cancer, a formidable public health challenge, demands innovative approaches for early detection and effective treatment. This review explores the transformative role of transcriptomics in oral cancer research, leveraging cutting-edge technologies to decode the RNA landscape of tumors. We delve into RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), long non-coding RNA sequencing (lncRNA-seq), microarray analysis, and small RNA profiling, showcasing their unique contributions to unraveling the complexities of oral cancer. By mapping gene expression patterns, identifying biomarkers, and understanding tumor heterogeneity at an unprecedented resolution, transcriptomics offers powerful insights into the molecular mechanisms driving oral cancer. Furthermore, we highlight how these approaches pave the way for personalized medicine, enabling precise therapeutic interventions and innovative treatment strategies. The review also addresses the future directions and challenges in the field, emphasizing the need for advanced computational tools and interdisciplinary collaboration to fully harness the potential of transcriptomics in transforming oral cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11589d7a62e5f0b71eace88c7076fdf589259998" target='_blank'>
              Role of transcriptomics in the study of oral cancer
              </a>
            </td>
          <td>
            P. Natarajan, V. Umapathy
          </td>
          <td>2025-07-28</td>
          <td>Frontiers in Oral Health</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT In most cancers, T lymphocytes comprise an essential cellular component of the non-neoplastic microenvironment, where they have the capacity to both suppress and support tumor growth. One specialized T lymphocyte population is the CD8+ exhausted T cell, which has been intensely studied as an actionable therapeutic target. Unfortunately, there is currently no uniformly accepted classification scheme for these specialized T cells. To provide a potential model for classifying CD8+ exhausted T cells, we leveraged single cell transcriptomic analysis of a diverse collection of both human (n = 8) and mouse (n = 4) cancers to identify unique subpopulations shared across tumor types and species. By integrating data from both human and mouse cancer studies, as well as previously described CD8+ exhausted T cell subsets, we provide an integrated framework to characterize the heterogeneity of exhausted CD8+ T cells. As such, one of these subpopulations (cluster C1) increases following immune checkpoint inhibitor treatment in the setting of cancer in mice and patients. Taken together, this proposed classification scheme may be useful for the design and interpretation of current and future immune-based therapy studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9bb9b37e719dff3c4a062c99fe8116dd65c7d0" target='_blank'>
              Pan-cancer single cell transcriptomic clustering reveals heterogeneous CD8+ exhausted T cell populations with different immune checkpoint inhibitor responses
              </a>
            </td>
          <td>
            Rui Mu, Rasha Barakat, David H. Gutmann
          </td>
          <td>2025-08-03</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d135048b3a2d9e0d80a5651add7d7f82d46f7f4a" target='_blank'>
              Innate immunity and the NF-κB pathway control prostate stem cell plasticity, reprogramming and tumor initiation.
              </a>
            </td>
          <td>
            Chen Jiang, Yura Song, S. Rorive, Justine Allard, Elisavet Tika, Zahra Zahedi, C. Dubois, Isabelle Salmon, Alejandro Sifrim, Cédric Blanpain
          </td>
          <td>2025-06-23</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are responsible for colorectal cancer (CRC) chemoresistance, recurrence, and metastasis. Therefore, identifying molecular stemness targets that are involved in tumor growth is crucial for effective treatment. Here, we performed an extensive in vitro and in vivo molecular and functional characterization, revealing the pivotal role of the lysine methyltransferase SET and MYND Domain Containing 3 (SMYD3) in colorectal cancer stem cell (CRC-SC) biology. Specifically, we showed that SMYD3 interacts with and methylates c-MYC at K158 and K163, thereby modulating its transcriptional activity, which is implicated in stemness and colorectal malignancy. Our in vitro data suggest that SMYD3 pharmacological inhibition or its stable genetic ablation affects the clonogenic and self-renewal potential of patient-derived CRC-SCs and organoids by altering their molecular signature. Moreover, we found that SMYD3 stable knock-out or pharmacological inhibition drastically reduces CRC tumorigenicity in vivo and CRC-SC metastatic potential. Overall, our findings identify SMYD3 as a promising therapeutic target acting directly on c-MYC, with potential implications for countering CRC-SC proliferation and metastatic dissemination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/508ee1e83935ec4b0c34a584bea5279ee9131213" target='_blank'>
              Tailoring a novel colorectal cancer stem cell-targeted therapy by inhibiting the SMYD3/c-MYC axis
              </a>
            </td>
          <td>
            Martina Lepore Signorile, Elisabetta Di Nicola, G. Forte, P. Sanese, C. Fasano, Vittoria Disciglio, Katia De Marco, Marialaura Latrofa, Loris De Cecco, Marica Ficorilli, Marta Lucchetta, E. Torchia, Chiara Dossena, Giusy Bianco, Vito Spilotro, C. Ferroni, Nicoletta Labarile, R. Armentano, Francesco Albano, A. Mestice, Gianluigi Gigante, V. Lantone, G. Lantone, Leonardo Vincenti, Alberto Del Rio, G. Varchi, V. Grossi, Cristiano Simone
          </td>
          <td>2025-06-30</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is the most prevalent subtype of lung cancer, presenting significant challenges in treatment and prognostic prediction. Despite advancements in therapeutic approaches in recent years, personalized medicine has not yet achieved a notable breakthrough. Given the poor prognosis of patients, there is an urgent need to enhance the ability for precise prediction. Mitochondria play a crucial role in the metabolism and energy production of cancer cells, yet their specific impact in lung adenocarcinoma warrants further investigation. This study leveraged data from the TCGA and GEO databases to stratify 515 lung adenocarcinoma patients into two distinct subtypes based on mitochondrial-related genes. We systematically evaluated survival outcomes and biological pathway activities between subtypes, characterized their immune infiltration profiles, and developed a prognostic model using subtype-specific differentially expressed genes. Drug sensitivity disparities were further assessed. Single-cell RNA sequencing data were analyzed using an XGBoost classifier to delineate cell-type heterogeneity across subtypes at single-cell resolution. In LUAD, we identified two distinct subtypes. One subtype exhibited active mitochondrial metabolism, which was associated with poor prognosis and higher tumor purity. Moreover, this subtype showed greater sensitivity to Osimertinib. Further single-cell analysis revealed that this subtype was characterized by substantial macrophage infiltration, potentially promoting tumor progression through the NF-κB signaling pathway. Overall, our study identified novel LUAD subtypes and provided new insights into the clinical treatment of LUAD. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07982-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08aa1cb8c92b7c61ac76cf9ab60fdfeab09318a6" target='_blank'>
              Mitochondria-related gene-based molecular subtypes of lung adenocarcinoma and their prognostic implications
              </a>
            </td>
          <td>
            Ziyi Zhanghuang, Fei Xie, Xuemei Ma, Jinfeng Chen
          </td>
          <td>2025-07-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Background Triple‐negative breast cancer (TNBC) poses significant challenges due to its aggressive nature and lack of targeted therapies. Understanding the cellular behaviors of TNBC is crucial for developing effective treatments. Aims This study aims to compare the morphological characteristics of non‐tumorigenic MCF10A and aggressive MDA‐MB‐231 TNBC cell lines using advanced analytical techniques. Methods and Results Advanced techniques such as Principal Component Analysis (PCA), t‐Distributed Stochastic Neighbor Embedding (t‐SNE), and digital holographic microscopy were utilized. Cellular features such as area, migration, motility, irregularity, and optical thickness were thoroughly analyzed over time. Our results revealed significant morphological differences between the MCF10A and MDA‐MB‐231 cell lines. Specifically, MDA‐MB‐231 cells displayed enhanced motility and a smaller, more variable size, attributes that may facilitate their invasive potential. In contrast, MCF10A cells exhibited larger sizes and more regular migration patterns, suggesting stability in structured tissue environments. Additionally, temporal analysis highlighted consistent phenotypic behaviors over time, with MDA‐MB‐231 cells demonstrating higher optical thickness and irregularity, indicating potential structural complexities associated with malignant transformation. Correlative analysis further confirmed these results by revealing connections between cell size, motility, and optical properties crucial for understanding cell behavior within their microenvironment. Conclusion The profound differences in cellular dynamics between MCF10A and MDA‐MB‐231 cell lines underscore the unique adaptive mechanisms of TNBC cells. Our study provides valuable insights into the cellular foundations of TNBC aggressiveness, offering a foundation for future research aimed at understanding the mechanistic underpinnings of TNBC progression and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9f97f859a9a924b82dc01461c45eed54b60c098" target='_blank'>
              Dissecting Morphological and Functional Dynamics of Non‐Tumorigenic and Triple‐Negative Breast Cancer Cell Lines Using PCA and t‐SNE Analysis
              </a>
            </td>
          <td>
            Yazan A. Almahdi, Eva R Schwark, Aidan J. McLaughlin, Besa Xhabija
          </td>
          <td>2025-06-27</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cervical cancer (CC) is the fourth commonest malignant tumor among women worldwide and is characterized by high heterogeneity and a complex ecosystem. A comprehensive understanding of the heterogeneity of tumors and the tumor microenvironment (TME) is crucial for effective CC management. Single-cell RNA sequencing (scRNA-seq) is a powerful tool that can be employed to unveil the heterogeneity of tumors and the TME, as well as to elucidate the evolutionary trajectories of tumors. Spatial transcriptomics (ST) technology, on the other hand, can address the complexity and diversity of the spatial microenvironment of tumors, thereby compensating for the limitations of scRNA-seq. As emerging technologies, both scRNA-seq and ST are increasingly being utilized in CC research. In this review, we summarized the latest advancements in scRNA-seq and ST for CC, with a focus on investigating tumor heterogeneity, the TME, tumor evolutionary trajectories, treatment resistance mechanisms, and potential therapeutic targets. These insights collectively contribute to the development of more effective treatment and prevention strategies for CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f5b3526e206bb4dcf755c55607c83c389e4ccb" target='_blank'>
              Applications and Prospects of Single-Cell RNA Sequencing and Spatial Transcriptomics in Cervical Cancer
              </a>
            </td>
          <td>
            Yifu Wang, Li Yang, Yunzhi Liu, Huangrong Ma, Miaoying Cai, Chunyu Liang, Li Zhang, Zezhuo Su, Zhiyuan Xu
          </td>
          <td>2025-06-26</td>
          <td>BioMed Research International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Interstitial lung diseases (ILDs) encompass a diverse group of pulmonary disorders, with progressive fibrosis leading to poor prognosis. Here we aimed to identify key molecules involved in progressive fibrosis across various ILDs, using spatial transcriptomics (ST). ST analysis (Visium) was performed on lung cryobiopsy specimens from five patients with various ILDs. Two cases, rich in young fibrotic lesions, as defined by fibroblastic foci and destructive alveolar organization, were selected for spatial high‐dimensional weighted gene coexpression network analysis (hdWGCNA) to identify key gene networks with biological significance in active fibrosis. We utilized public single‐cell RNA sequencing datasets of various ILDs, performed enrichment analysis and trajectory‐based differential expression analysis, and quantified cell–cell communication to evaluate the involvement of the spatially extracted module in fibrosis. Immunohistochemical staining of the extracted molecules was performed. Using hdWGCNA, we identified a distinct gene module (the SM2 module) enriched in young fibrotic lesions. The SM2 module was characterized by distinct features of fibroblast activation that were represented across various lesions. Key hub genes within this module, including COL1A2, COL3A1, COL1A1, and SPARC, formed a robust coexpression network. Immunohistochemical staining showed that SPARC, a component of the SM2 module, was highly expressed in young fibrotic lesions, but not in old scarring lesions, across various ILDs. To assess the prognostic significance of SPARC immunohistochemical expression, we extended our analysis to a cohort of 71 patients with unclassifiable ILDs (uILDs), a particularly heterogeneous subtype with unclear pathogenesis and limited treatment options. Higher SPARC levels in the upper, lower, or both lung lobes in uILD were significantly associated with poor overall survival. In summary, an integrated cross‐disease approach using ST revealed key gene expression patterns central to active fibrosis and successfully identified SPARC as a potentially beneficial prognostic marker. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9a55a6327413c2017bbf7b10baf6a8049b2a4f0" target='_blank'>
              Spatial transcriptomics identifies SPARC as a prognostic marker in interstitial lung diseases
              </a>
            </td>
          <td>
            T. Niitsu, T. Kataoka, K. Fukushima, D. Motooka, Shigeyuki Shichino, Yayoi Natsume-Kitatani, H. Kitamura, Takashi Niwa, T. Baba, D. Okuzaki, Atsushi Kumanogoh, Shizuo Akira, K. Okudela, Takashi Ogura
          </td>
          <td>2025-07-28</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Purpose The heterogeneity of immune cells is a critical manifestation of gastric cancer (GC) heterogeneity and significantly contributes to immune therapy resistance. Although previous studies have focused on the roles of specific myeloid cells and exhausted CD8+ T cells in immune resistance, the immune cell interaction network and its spatiotemporal distribution in GC immune resistance remain underexplored. Methods This study integrated multiple GC single-cell RNA sequencing, spatial transcriptomics, bulk-RNA sequencing, and single-cell immunotherapy datasets of our cohort (NFHGC Cohort). Methods such as single-cell subpopulation identification, transcriptomic analysis, spatial colocalization, cell communication network analysis and tissue immunofluorescence of gastric cancer were employed to investigate immune cell interactions and their molecular mechanisms in immune resistance. Results By leveraging a comprehensive approach that integrates single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq profiles, we identified 20 immune subsets with potential prognostic and therapeutic implications. Our findings suggest a stromal immunosuppressive network orchestrated by Macro_SPP1/C1QC macrophages and CD8_Tex_C1 T cells, which may form a barrier impeding antitumor immunity. Macrophage-derived MIF signaling appears to drive immunosuppression via the MIF-CD74/CXCR4/CD44 axis. Based on these observations, we developed a preliminary TME classification system using a gene signature derived from barrier-associated immune cell markers and unsupervised clustering. Conclusions Our study identified a potential stromal immunosuppressive barrier in gastric cancer, driven by Macro_SPP1/C1QC macrophages and CD8_Tex_C1 T cells, which may contribute to immune dysfunction and therapy resistance. Molecular subtyping based on this barrier’s presence could inform personalized immune therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1cf3d75b61cafecd69ad5d518d7d286742cb1e5" target='_blank'>
              Identification of a stromal immunosuppressive barrier orchestrated by SPP1+/C1QC+ macrophages and CD8+ exhausted T cells driving gastric cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Guichuang Ma, Xiaohan Liu, Qinrui Jiang, Shaowei Li, Qijing Wu, B. Liang, F. Sun, Chunhui Gu, Wangjun Liao, Zhi-Huan Zhang, Min Shi, Qiong Huang
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary Soft tissue sarcomas (STSs) are rare cancers that respond poorly to current treatments. As the immune system is increasingly recognized as a key player in cancer control, we investigate a specific type of immune cell—natural killer (NK) cells. We analyzed NK cells from blood samples of STS patients and healthy individuals. NK cells from STS patients were less functional, showing reduced degranulation and lower production of IFNγ, both essential for killing cancer cells. We also found alterations in key surface markers, which may explain their impaired function. Among these, CD27 and NKp44 were notably different between patients and healthy individuals and showed strong potential to distinguish between these groups. Higher CD27 levels were associated with NK cells that are less effective at killing cancer cells. Overall, our findings suggest that NK cells from STS patients may have a reduced capacity to fight cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14bb47c660c210aea5cd2c30473df6b270829f44" target='_blank'>
              Comprehensive Receptor Repertoire and Functional Analysis of Peripheral NK Cells in Soft Tissue Sarcoma Patients
              </a>
            </td>
          <td>
            L. M. Sousa, J. Almeida, Tânia Fortes-Andrade, Patrícia Couceiro, Joana Rodrigues, Rúben Fonseca, M. Santos-Rosa, Paulo Freitas-Tavares, J. Casanova, Paulo Rodrigues-Santos
          </td>
          <td>2025-07-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Melanoma is the most aggressive form of skin cancer, and despite significant therapeutic advances over the past decade, a substantial number of patients still progress to a fatal outcome. The initiation and progression of melanoma are strongly influenced by interactions between melanoma cells and other components of the tumor microenvironment (TME). In this review, we focus on the interplay between fibroblasts resident in the tumor microenvironment and tumor cells. In particular, we examine the molecular mechanisms through which melanoma cells induce the transformation of resident fibroblasts into their active form, known as cancer-associated fibroblasts (CAFs). We also explore the role of CAFs in shaping the melanoma microenvironment (MME) and in organizing the pre-metastatic niche, a specialized microenvironment that forms in distant organs or tissues to support the survival and expansion of metastatic melanoma cells. Finally, we discuss emerging therapeutic strategies aimed at disrupting the interactions between CAFs, melanoma cells, and other components of the tumor microenvironment to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d9f3ff9ff2ce42f0f728f6171249249cc7ed33f" target='_blank'>
              The Role of Fibroblasts in Melanoma Development: From Tumor Microenvironment Remodeling to Pre-Metastatic Niche Formation
              </a>
            </td>
          <td>
            Immacolata Belviso, Raffaele Pastore, Aldo Mileo, Emiliano Del Genio, Stefania Boccia, Stefano Palermi, C. Sellitto, M. L. Motti
          </td>
          <td>2025-06-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Genetic and cell biological analyses of this rare glioblastoma (GBM) case with rapidly progressive pleural metastasis revealed essential roles of epithelial-mesenchymal transition (EMT) and glioma stem-like cells (GSCs) in extraneural metastasis. Contrary to the common prediction, the metastatic lesion shared 73% of mutations and two fusion genes with the primary but not the recurrent tumor, indicating that extraneural metastasis occurs early in GBM progression. Functional assays using primary tumor-derived GSCs confirmed EMT-enhanced migratory behaviors. EMT and genetically driven early tumor seedings are crucial for extraneural GBM metastasis, emphasizing the need for further investigation into EMT pathways and GSC biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e9c844456cae0a033c5748e4caed704ed136585" target='_blank'>
              Epithelial-Mesenchymal Transition and Genetically Driven Early Tumor Seedings in Extraneural Metastasis of Glioblastoma.
              </a>
            </td>
          <td>
            Seisaku Kanayama, Kentaro Watanabe, Minoru Tanaka, Junko Takita, T. Todo
          </td>
          <td>2025-06-20</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="ABSTRACT Fusobacterium nucleatum (Fn) is commonly enriched in colorectal cancer (CRC) and associated with poor outcomes, though its mechanisms remain unclear. Our study investigated how Fn affects the tumor microenvironment through single-cell transcriptomic analyses of 42 CRC patient tissues, comparing Fn-positive and Fn-negative tumors. We discovered that Fn impairs IgA plasma cell development and secretory IgA (sIgA) production by disrupting communication with tumor-associated macrophages. Additional experiments in germ-free mice, together with our re-analysis of a publicly available single-cell RNA-seq data set from a CRC mouse model with an intact gut microbiome–both models having been orally gavaged with Fn–jointly validated the causal role of Fn in impairing sIgA induction. We identified a dysregulated IgA maturation (IGAM) module in Fn-positive patients, indicating compromised mucosal immunity and increased bacterial infiltration. This IGAM signature effectively stratified Fn-positive patients, suggesting potential for targeted therapeutic approaches. Our findings reveal that Fn disrupts sIgA production, increasing tumor microbial burden and worsening prognosis through chronic inflammation in Fn-positive CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b983088c09af7c7c8c02b6b79dd03932133b600d" target='_blank'>
              Secretory IgA dysfunction underlies poor prognosis in Fusobacterium-infected colorectal cancer
              </a>
            </td>
          <td>
            Ilseok Choi, Kyung-A Kim, Sang Cheol Kim, Donghwan Park, Ki Taek Nam, Junha Cha, S. Baek, Junha Cha, Hye-Yeong Jo, Minsun Jung, Melody Y. Zeng, Irina Matei, Susan Bullman, Joong Bae Ahn, Yoon Dae Han, Han Sang Kim, Insuk Lee
          </td>
          <td>2025-07-16</td>
          <td>Gut Microbes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Natural killer (NK) cells and T cells play crucial roles in liver metastatic cancer, particularly through their cytotoxic effects on tumor cells. Although existing evidence suggests a functional network of mutual regulation between NK and T cells, the nature of their interaction and its role in liver metastasis remain elusive. In this study, we analyzed single-cell transcriptomics of liver metastases and adjacent tissues from nasopharyngeal carcinoma (NPC), thyroid carcinoma (THCA), breast cancer (BC), colorectal cancer (CRC) and cervical cancer (CESC) to uncover the NK-T cell interaction network and its functional implications. In adjacent tissues, we observed increased infiltration of CD8+ NKT-like cells, γδT cells, and NK cells. The interaction between CD8+ NKT-like cells, CD8+ T cells, γδT cells, and NK cells were enhanced, with stronger signals associated with T and NK cell functions. Notably, CD8+ NKT-like cells, CD8+ T cells, and γδT cells exhibited an increased capacity to activate NK cells. These T cell subsets promoted NK cell anti-tumor activity via CD48-CD244 and TNF-TNFR signaling pathways, which in turn activated the ERK, JNK, and MAPK cascades. Our findings provide valuable insights into the NK-T cell interaction network in liver metastatic cancer, highlighting its potential as a therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-06241-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d630bd4e5bebe82c7ea2d9533a6c99b8e57bfc4" target='_blank'>
              Single-cell transcriptomics analysis reveals a disrupted NK-T cell interaction network in liver metastatic cancer
              </a>
            </td>
          <td>
            Xiaoshuang Wang, Zhongen Wu, Yan Zhou, Dehua Yang, Qingtong Zhou, Di Zhu, Mingwei Wang, Lu Wang
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Cell type-specific transcriptional heterogeneity in embryonic mouse skin is well-documented, but few studies have investigated the regulatory mechanisms. Here, we present high-throughput single-cell chromatin accessibility and transcriptome sequencing (HT-scCAT-seq), a method that simultaneously profiles transcriptome and chromatin accessibility. We utilized HT-scCAT-seq to dissect the gene regulatory mechanism governing epidermal stratification, periderm terminal differentiation, and fibroblast specification. Results By linking chromatin accessibility to gene expression, we identify candidate cis-regulatory elements (cCREs) and their target genes which are crucial for dermal and epidermal development. We describe cells with similar gene expression profiles that exhibit distinct chromatin accessibility statuses during periderm terminal differentiation. Finally, we characterize the underlying lineage-determining transcription factors and demonstrate that ALX4 and RUNX2 are candidate transcription factors regulators of the dermal papilla lineage development through in silico perturbation analysis and CUT&Tag experiment. Conclusions Overall, HT-scCAT-seq represents a powerful tool for unraveling the spatiotemporal dynamics of gene regulation in single cells. Our results advance the understanding of embryonic skin development while providing a scalable framework for investigating regulatory mechanisms across diverse biological systems and disease contexts. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03652-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/747c145f394c82a8a3bbf273f6afa4e007611092" target='_blank'>
              Deciphering gene regulatory programs in mouse embryonic skin through single-cell multiomics analysis
              </a>
            </td>
          <td>
            Q. Deng, Pengfei Cai, Yingjie Luo, Zhongjin Zhang, Wen Ma, Zijie Huang, Xiaoya Chen, Shijie Hao, Weiguang Ma, Jiangshan Xu, Chunqing Wang, Mengnan Cheng, Xiumei Lin, Ru Zhou, Shanshan Duan, Junjie Chen, Ronghai Li, Xuyang Shi, Chang Liu, Liang Wu, Peng Gao, Jianting Li, Xiao Yang, Xiangdong Wang, Jun Xie, Longqi Liu, Yue Yuan, Chuanyu Liu
          </td>
          <td>2025-07-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Non-melanoma skin cancer (NMSC), comprising basal cell carcinoma (BCC) and cutaneous squamous cell carcinoma (cSCC), represents the most common type of cancer worldwide, particularly among Caucasians. While BCC is locally invasive with minimal metastatic potential, cSCC is a highly aggressive tumor with a significant potential for metastasis, particularly in elderly populations. Tumor development and progression and the metastasis of cSCC are influenced by a complex interplay between tumor cells and the tumor microenvironment. Recent research highlights the importance of various immune cell subsets, including T cells, tumor-associated macrophages (TAMs), and dendritic cells, in influencing tumor progression, immune evasion, and treatment resistance. This review outlines key regulatory mechanisms in the immune tumor microenvironment (TME) of cSCC and explores the role of cytokines, immune checkpoints, and stromal interactions. We further discuss the relevance of three-dimensional (3D) in vitro models such as spheroids, organoids, and tumor-on-chip systems as tools to mimic immune–tumor interactions with higher physiological relevance, such as macrophage activation and polarization against cSCC cells. Globally, 3D models offer new opportunities for immunotherapy screening and mechanistic studies. Understanding the immune landscape in cSCC through advanced modeling techniques holds strong clinical potential for improving diagnostic and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423c3e60a81f05412f1191c356ba2acb90d2d9a5" target='_blank'>
              Exploring the Complexity of Cutaneous Squamous CellCarcinoma Microenvironment: Focus on Immune Cell Roles by Novel 3D In Vitro Models
              </a>
            </td>
          <td>
            M. Quadri, Marco Iuliano, Paolo Rosa, G. Mangino, Elisabetta Palazzo
          </td>
          <td>2025-07-23</td>
          <td>Life</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Background/Aim: Tongue squamous cell carcinoma (TSCC), a highly aggressive subtype of head and neck cancers, is characterized by frequent lymphatic metastasis and poor prognosis. Recently, we showed that lymphatic vessel endothelial hyaluronan receptor 1 (LYVE-1) is involved in TSCC progression, yet the underlying molecular mechanisms remain unclear. Materials and Methods: CRISPR/Cas9 gene editing was employed to generate LYVE1 knockout (KO) TSCC cell lines. Single-cell clones were isolated, screened, and validated through sequencing and Inference of CRISPR Edits (ICE) analysis and qRT-PCR. RNA sequencing was performed on LYVE1 KO and wild-type (WT) cells to identify differentially expressed genes (DEGs). Bioinformatic analyses, including Gene Ontology (GO) enrichment and protein-protein interaction (PPI) network mapping, were conducted to explore affected pathways. Finally, network topology was examined using NetworkAnalyzer and cytoHubba plugins. Results: Transcriptomic analysis revealed significant down-regulation of pro-metastatic pathways, including epithelial-mesenchymal transition (EMT), extracellular matrix remodeling, and immune modulation. DEG analysis identified 263 genes, with key down-regulated targets such as WNT5A, TGFB2, and MMP2, and up-regulation of tumor-suppressive genes including PTGS2. GO and PPI analyses highlighted LYVE1’s pivotal role in regulating cell adhesion, migration, and immune response. Conclusion: LYVE1 KO reduces TSCC invasive potential by disrupting EMT and tumor-stroma interactions, aligning with previous experimental findings. These results suggest LYVE1 as a critical driver of metastasis, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff3c633bd83b5225f79493e53059643bbd3c2790" target='_blank'>
              CRISPR/Cas9-mediated Knockout of LYVE1 In Human Tongue Cancer Cells Reveals Transcriptomic Changes in Metastasis-associated Pathways
              </a>
            </td>
          <td>
            Sini Karinen, Johanna FORERO-RODRÍGUEZ, Annika Järvinen, Kari K Eklund, Gonçalo Barreto, A. Salem
          </td>
          <td>2025-06-26</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The tumor ecosystem heterogeneity of papillary thyroid carcinoma (PTC) is poorly characterized in children and young adult patients (CAYA-PTC). In this study, single-cell RNA sequencing is used to profile transcriptomes from the paratumor and tumor tissues of 11 patients. Compared to adult, CD4T_Tfh and CD8T_Tex cells are significantly more prevalent in CAYA-PTC patients. Three phenotypes are identified within the thyrocytes through differentiation trajectory analysis, including normal, BRAF-like, and Fusion-like. Notably, the data reveal that CAYA-PTC patients lack the "mild-state (BRAF-like)" malignant thyrocyte population. This variation in differentiation states indicates that PTC cells in CAYA patients rapidly develop into invasive and metastatic forms, whereas in adult patients, this progression occurs gradually over a longer period. Additionally, extracellular matrix cancer-associated fibroblasts (emCAFs_LAMP5) interact with endothelial cells and thyrocytes, promoting tumor angiogenesis and metastasis more prominently in CAYA patients. Fibroblast activation protein (FAP) expression is high in emCAF_LAMP5 and positively correlated with LAMP5 in CAYA-PTC tissues. Consequently, 68Ga-FAPI-PET emerges as a promising diagnostic method for CAYA patients who are not effectively diagnosed by traditional 18F-FDG-PET. Collectively, the findings provide insight into the CAYA-PTC ecosystem that suggests distinct diagnostic, prognostic, and therapeutic implications compared to adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce5c40a8e8a8fbc86d0cfd5509e3eb16f2ff46d" target='_blank'>
              Single-Cell RNA Sequencing Reveals the Heterogeneity in Differentiation Trajectory and Tumor Microenvironment Leading to More Aggressive Phenotypes of Papillary Thyroid Cancer in Children and Young Adult Patients.
              </a>
            </td>
          <td>
            K. Guo, Lingyi Yang, Zirui Huang, Kai Qian, Yuan Shi, Mengjia Fei, Yuan Feng, Zixian Chen, Ming Qiu, Linglin Tang, Anqi Li, Zhibao Lv, Jiangbin Liu, Zhou Chen, Kuiran Dong, Zhuoying Wang
          </td>
          <td>2025-07-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background: Several studies have demonstrated the occurrence of secondary tumors as a rare but significant complication of chimeric antigen receptor T (CAR-T) cell therapy, underscoring the need for a detailed investigation. Given the limited variety of secondary tumor types reported to date, a comprehensive characterization of the various secondary tumors arising after CAR-T therapy is essential to understand the associated risks and to define the role of the immune microenvironment in malignant transformation. This study aims to characterize the immune microenvironment of a newly identified secondary tumor post-CAR-T therapy, to clarify its pathogenesis and potential therapeutic targets. Methods: In this study, the bone marrow (BM) samples were collected by aspiration from the primary and secondary tumors before and after CD19 CAR-T treatment. The CD45+ BM cells were enriched with human CD45 microbeads. The CD45+ cells were then sent for 10× genomics single-cell RNA sequencing (scRNA-seq) to identify cell populations. The Cell Ranger pipeline and CellChat were used for detailed analysis. Results: In this study, a rare type of secondary chronic myelomonocytic leukemia (CMML) were reported in a patient with diffuse large B-cell lymphoma (DLBCL) who had previously received CD19 CAR-T therapy. The scRNA-seq analysis revealed increased inflammatory cytokines, chemokines, and an immunosuppressive state of monocytes/macrophages, which may impair cytotoxic activity in both T and natural killer (NK) cells in secondary CMML before treatment. In contrast, their cytotoxicity was restored in secondary CMML after treatment. Conclusions: This finding delineates a previously unrecognized type of secondary tumor, CMML, after CAR-T therapy and provide a framework for defining the immune microenvironment of secondary tumor occurrence after CAR-T therapy. In addition, the results provide a rationale for targeting macrophages to improve treatment strategies for CMML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f62036ee9eb9eb457e3406f8e167f834148a9998" target='_blank'>
              Decoding the immune microenvironment of secondary chronic myelomonocytic leukemia due to diffuse large B-cell lymphoma with CD19 CAR-T failure by single-cell RNA-sequencing
              </a>
            </td>
          <td>
            Xudong Li, Hong Huang, Fang Wang, Mengjia Li, Binglei Zhang, Jianxiang Shi, Yuke Liu, Mengya Gao, Mingxia Sun, Haixia Cao, Danfeng Zhang, Na Shen, Weijie Cao, Zhilei Bian, Haizhou Xing, Wei Li, Linping Xu, Shiyu Zuo, Yongping Song
          </td>
          <td>2025-07-07</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Drug discovery and testing are both time-consuming and costly. Leveraging single-cell transcriptomic data, we developed a machine learning framework to model patient cells using scRNA-seq data, aiming to infer gene regulatory networks (GRNs) that drive cell states and plasticity. Our computational algorithm predicts cellular responses to drug perturbations at the transcriptional level, even without experimental testing. We utilized this algorithm to identify drug candidates that could shift anti-inflammatory macrophages to a pro-inflammatory state for remodeling the tumor microenvironment, reactivate exhausted CD8 T cells, and convert immunosuppressive CD4 Treg cells into central memory CD4 cells to enhance anti-tumor activity. The predicted outcomes were validated experimentally through scRNA-seq, flow cytometry, and cytokine profiling. In human hepatocellular carcinoma (HCC), tumor infiltrating lymphocytes (TILs) treated with the predicted drugs showed a decreased FoxP3+ CD4 Treg population compared to untreated control TILs from the same HCC (n = 3). Separately, TILs treated with another set of predicted drugs showed higher levels of IFNG+ CD8 T cells than untreated controls (n = 3). Lastly, in human THP1 macrophages differentiated to an anti-inflammatory state, treatment resulted in increased levels of TNFa and IL1b secretion compared to untreated controls, indicating a phenotypic shift towards a pro-inflammatory state. We then tested the same drug on tumor associated macrophages (TAMs) from HCC and showed an increased CD64+ HLA-DR+ macrophage subset upon treatment. scRNA-seq analysis further revealed key pathways induced by the drugs, that resulted in the desired change in immune phenotype, offering insight into immune cell function and plasticity. In conclusion, this study introduced a machine-learning driven approach to identify candidate small molecules from scRNA-seq data that are capable of modulating macrophage and T cell activation, providing new avenues for targeted interventions in cancer.



 N. Jannah M. Nasir, John F. Ouyang, Valerie Chew, Owen Rackham. Applying Machine Learning to Identify Enhancers of Immune Cell Functionality for Targeted Cancer Therapies [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P54.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb6e81b3bc27e9df1055642ea570cd3c3f46a059" target='_blank'>
              Abstract P54: Applying Machine Learning to Identify Enhancers of Immune Cell Functionality for Targeted Cancer Therapies
              </a>
            </td>
          <td>
            N. J. M. Nasir, John F. Ouyang, Valerie Chew, Owen Rackham
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (Tumor Microenvironment, TME) is a core regulatory factor in the occurrence, development, and treatment resistance of tumors. Macrophages, as key immune cell components in the TME, have a profound impact on the tumor process (Visser and Joyce in Cancer Cell 41:374–403, 2023). This review aims to systematically elucidate the characteristics and functional differences of macrophage polarization into M1 and M2 phenotypes within the TME. Additionally, it endeavors to dissect the regulatory mechanisms by which metabolic products, cytokines, and extracellular matrix components secreted by tumor cells modulate macrophage polarization (Wang et al. in Mol Cancer 23:268, 2024). Moreover, the metabolic reprogramming of tumorassociated macrophage (TAM) is a core mechanism for their functional shift, and intervening in metabolic pathways holds promise for reprogramming TAMs to inhibit tumor progression (Jin et al. in Nat Cancer 6:239–252, 2025). Within the TME, macrophages can be polarized into classically activated M1 and alternatively activated M2 types (Ge and Wu in Zhongguo Fei Ai Za Zhi 26:228–237, 2023). Accumulating evidence indicates that classically activated M1 macrophages orchestrate anti-tumor immunity by secreting pro-inflammatory cytokines, including tumor necrosis factor-α (TNF-α) and interleukin-12 (IL-12), which collectively activate cytotoxic T lymphocyte responses, induce tumor cell apoptosis, and enhance immune surveillance (Luo et al. in Front Immunol 15:1352946, 2024). In contrast, M2 macrophages are induced in the TME and promote tumor angiogenesis, immune evasion, tumor cell proliferation, and metastasis by secreting factors such as vascular endothelial growth factor (VEGF) and transforming growth factor-beta (TGF-β) (Wang et al. in NPJ Precis Oncol 8:31, 2024). Therefore, in-depth research on the mechanisms of macrophage polarization in the tumor microenvironment provides an important basis for developing new tumor immunotherapy strategies and has significant clinical translational value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323da979d84b0f4625128d836a1d59f140970818" target='_blank'>
              Advances in the regulation of macrophage polarization by the tumor microenvironment
              </a>
            </td>
          <td>
            Minzhi Peng, Yuan Zhu, Yi Hu, Jianping Wen, Weiguo Huang
          </td>
          <td>2025-08-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Lung cancer stands as one of the most prevalent malignancies globally, with its high morbidity and mortality intimately associated with its inconspicuous early symptoms, the advanced stages at diagnosis, and resistance to conventional therapeutic interventions. Exosomes, serving as pivotal mediators of intercellular communication, carry biomolecules that play crucial roles in tumorigenesis, progression, and metastasis, holding promise as novel targets for early diagnosis, prognostic evaluation, and treatment of lung cancer. Methods In this study, lung cancer-related datasets were obtained from the GEO database and TCGA. Through differential gene analysis, enrichment analysis, immune infiltration analysis, and drug regulatory analysis, exosome-associated genes pertinent to lung cancer were screened and identified. Results The research revealed significant downregulation of CRYAB, CAV1, HYAL1, and TUBB6 genes in lung cancer tissues, whereas SERINC2, PAICS, SLC2A1, and BIRC5 genes were markedly upregulated. These genes were predominantly enriched in biological processes such as cell migration, oxidative stress response, and cell cycle regulation, as well as in KEGG pathways like the IL-17 signaling pathway. Immune infiltration analysis demonstrated a high correlation between these genes and the infiltration levels of various immune cells. Furthermore, through drug-gene enrichment analysis and molecular docking experiments, significant correlations were found between drugs such as celecoxib and some exosome-related genes, with interaction targets existing between these drugs and CAV1, SLC2A1, and BIRC5 genes. Conclusion This study unveils the expression characteristics and biological significance of exosome-associated genes in lung cancer. The differential expression of these genes not only offers potential biomarkers for early diagnosis of lung cancer but also lays a foundation for further research into their biological functions in this disease. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03173-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34b7ac84cc89c2118e1e8ef700dcc6c2b9b5300f" target='_blank'>
              Expression characteristics and biological significance of exosome-related genes in lung cancer
              </a>
            </td>
          <td>
            Qixiang Zhong, Yujie Zhao
          </td>
          <td>2025-07-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Cancer is a leading cause of death worldwide, and researchers are continuously working to uncover its drivers and new therapeutic targets. In this review, we explain the role of the oncogenic protein CREPT that is overexpressed in the vast majority of cancers, leading to poor overall survival. CREPT ablation has shown sustained tumor regression in vitro and in mouse models. CREPT activation mediates cancer progression by enhancing the cell cycle by promoting chromatin loop formation. Thus, selectively targeting CREPT expression may provide alternative strategies for inhibiting abnormal cell division, metastasis, and tumor growth, and may improve the efficacy of the currently available therapies in combination, and will be key to reversing cancerous growth in patients with CREPT-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c17bae866b8a91dd27eb2fc5ec68639a8ba51279" target='_blank'>
              Disrupting Cell Cycle Machinery: CREPT Is an Emerging Target in Cancer Therapy
              </a>
            </td>
          <td>
            Umar Farooq, Jun Li, Zhijie Chang
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Heterotopic ossification (HO) is a pathological process where bone forms in extraskeletal tissues, often occurring as a complication of tissue repair following injury. This condition can lead to movement limitations, pain, and functional impairment. However, the underlying pathomechanisms remain poorly understood. This study aims to elucidate key biomolecular networks involved in HO through a comprehensive multi‐omics analysis. Single‐cell, bulk, and spatial transcriptome datasets were obtained from the Gene Expression Omnibus (GEO) database. Migrasome score analysis identified a critical cell subtype associated with HO. Key genes were identified through high‐dimensional weighted gene co‐expression network analysis (hdWGCNA), machine learning, and dataset validation from clinical samples. Then we analyzed immune infiltration, microRNA (miRNA) networks, co‐expression networks, transcription factor (TF) regulatory networks, and signaling pathways to investigate potential regulatory mechanisms of HO. Spatial transcriptomics revealed the spatial patterns of cell subpopulation distribution and key molecule expression. Experimental validation further confirmed the expression patterns of key molecules in HO. As a result, we identified mesenchymal lineage cells (MLin) as the key migrasome‐associated cell subtype and determined peptidylprolyl isomerase B (Ppib) and transgelin (Tagln) as the key molecules. We constructed a regulatory network of these biomolecules and clarified their spatial distribution. Notably, the expression of Ppib and Tagln is temporally correlated with HO progression. Collectively, the identification of Ppib and Tagln, along with the construction of key biomolecular networks, facilitates the discovery of novel biomarkers for HO, offering promising potential for the development of preventive and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187a62a039287d77739831d7976ebd089c5fdedf" target='_blank'>
              Deciphering the Regulatory Networks of the Migrasome‐Associated Cell Subpopulation in Heterotopic Ossification via Multi‐Omics Analysis
              </a>
            </td>
          <td>
            Guanzhi Li, Xiao Deng, Tong Li, Yuchen Liu, Lei Tan, Kairui Zhang, Bin Yu
          </td>
          <td>2025-06-20</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Spatial data acquisition technologies enable high-throughput quantification of molecular expression in tissue sections maintaining spatial context information. However, performing downstream analysis on a whole tissue section requires the alignment and integration of multiple tissue slices. This is a nontrivial task due to tissue heterogeneity and plasticity. Although manual solutions exist, they are time-consuming and require technical expertise. Hence, automated and robust alignment and integration of multiple slices within and across datasets, individuals, and experiments becomes essential. This study aims to (i) present a comprehensive review of methodologies for spatial transcriptomics (ST) data alignment and integration, (ii) explain the problem, its scope and challenges, and (iii) propose a general pipeline. We review 24 tools addressing multi-slice ST alignment and integration, and tackling key challenges through downstream validation. Tools focusing solely on single-slice ST analyses or multi-omics integration are excluded. We categorize these approaches by methodology (statistical mapping, image processing and registration, and graph-based) in accordance with the generalized pipeline. We evaluate their strengths, limitations, and real-world applications based on task scope and their potential to advance biological insights. Despite improved spatial resolution and 3D tissue reconstruction, significant challenges persist in achieving robust alignment and integration across heterogeneous tissue slices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3892a4fa97322a200f44eb50a41bddeae483fdbe" target='_blank'>
              A comprehensive review of spatial transcriptomics data alignment and integration
              </a>
            </td>
          <td>
            M. Khan, Suzan Arslanturk, Sorin Draghici
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recent advancements have enabled tissue samples to be profiled at the unprecedented level of detail of a single cell. Analysis of this data has enabled discoveries that are relevant to understanding disease and developing therapeutics. Large-scale profiling efforts are underway which aim to generate 'atlas' resources that catalog cellular archetypes including biomarkers and spatial locations. While the problem of cellular data visualization is not new, the size, resolution, and heterogeneity of single-cell atlas datasets presents challenges and opportunities. We survey the usage of visualization to interpret single-cell atlas datasets by assessing over 1,800 figure panels from 45 biological publications. We intend for this report to serve as a foundational resource for the visualization community as atlas-scale single-cell datasets are emerging rapidly with aims of advancing our understanding of biological function in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796b3d170cc4e2945a302ca6861debacf36381a0" target='_blank'>
              The State of Single-Cell Atlas Data Visualization in the Biological Literature.
              </a>
            </td>
          <td>
            Mark S. Keller, Eric Morth, Thomas C. Smits, Simon Warchol, Grace Guo, Qianwen Wang, Robert Krueger, Hanspeter Pfister, Nils Gehlenborg
          </td>
          <td>2025-06-27</td>
          <td>IEEE computer graphics and applications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae991021621725bf1b34e349540819c402a22de9" target='_blank'>
              A Ligand-Centered Framework for γδ T Cell Activation in Colorectal Cancer Revealed by Single-Cell and Transformer-Based Perturbation
              </a>
            </td>
          <td>
            Ran Ran, Douglas K. Brubaker
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a subset of cells within tumors that exhibit stem cell-like characteristics, including the ability to self-renew and differentiate. CSCs are the cause of carcinogenesis and tumorigenesis. The expression of cell surface markers, which varies linked to the kind of tumor, is utilized to recognize CSCs. An essential part of tumor invasion and metastasis is played by CSCs. Numerous investigations have been carried out to find distinguished markers and different phenotypes of CSCs, which are especially crucial for identifying and separating this subset of cells. It was discovered that the regulation of CSCs involves a multitude of signaling pathways. These cells are determined by their ability to self-renewal pathways such as Wnt/β-catenin, JAK/STAT3, PTEN/PI3-K/Akt, and Hedgehog, their surface biomarkers, and their resistance to many drugs. Aberrant activation of these signaling pathways is associated with cell growth. Thus, focusing on CSCs is seen to be a viable anti-cancer treatment approach. It is encouraging that CSCs' self-renewal pathways present a viable target for changing their survival tactics and limiting their capacity to proliferate tumors. This study highlights the characterization and investigation of CSC self-renewal pathways, also discusses potential targeted therapy for CSC, and gives a summary of the significant factors and pathways that adjust CSC formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/441bf7be356d1c2d3ba019982154a866827815ef" target='_blank'>
              Navigating the Molecular Signaling: Deciphering Cancer Stem Cell Self-Renewal Pathways
              </a>
            </td>
          <td>
            Seyed Nasser Hoseinian, Mohammad Saeedi, Mohammad Erfan Saravani, Sepideh Zenoozi, F. Mehranfar, Amin Pouyan
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular and Cellular Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor adaptability relies on the ability of cancer cells to dedifferentiate and acquire stem-like features, fueling therapeutic resistance and metastasis. Differentiation therapy aims to reprogram tumor cells into more mature, less aggressive states to counteract this plasticity. Here, we identify cannabinoid receptor 2 (CB2R) as a novel therapeutic target that promotes sustained differentiation in breast cancer. Using tumor-derived organoids from both mouse models and patient biopsies, we show that brief, low-dose exposure to phytogenic or synthetic CB2R ligands induces a basal-to-luminal switch, suppresses stemness, and reduces invasiveness and self-renewal. These phenotypic changes are associated with decreased tumor initiation and aggressiveness in vivo . Transcriptomic profiling reveals that CB2R activation initiates transient chromatin remodeling and epigenetic reprogramming, resulting in a stably differentiated state. Importantly, CB2R-driven differentiation sensitizes tumor cells to tamoxifen, enabling lower therapeutic doses with improved efficacy-supporting the principles of adaptive therapy aimed at long-term disease control. Our findings position CB2R modulation as a promising non-cytotoxic strategy to restrict cancer plasticity and enhance the effectiveness of existing breast cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff04e163a7aa41f773b08a8fee437665a57ee718" target='_blank'>
              CB2R-induced differentiation epigenetically restrains cancer plasticity enabling adaptive therapy.
              </a>
            </td>
          <td>
            Nuria G. Martínez-Illescas, Gemma Pérez-García, María Rubert-Hernández, Camila Quezada-Gutiérrez, Estrella Martín-Zapater, Luis M Alonso-Colmenar, Carmen Hernández, Ángela Zarco-Cuadrillero, Eduardo Caleiras, Miguel Muñoz-Ruiz, Ana Payo-Payo, Laura Frías-Aldeguer, Cristina Sánchez, Mercedes Balcells-Camps, E R Edelman, María Salazar-Roa
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Nuclear morphology is a defining cellular feature, differing based on cell type, tissue type, and species. In healthy cells, nuclear morphology is generally tightly regulated and maintained; however, dynamic changes in nuclear morphology are observed under certain conditions, for instance in early embryos and in some immune cells. Deviations in normal nuclear morphology are linked to numerous diseases, including most cancers and premature aging syndromes. Many regulators of nuclear morphology have been identified, encompassing both intranuclear, cytoplasmic, and extracellular factors. Of note, recent studies have converged on chromatin and chromatin-associated proteins as key determinants of nuclear morphology and dynamics. In this review we discuss how the chromatin landscape regulates nuclear morphology in both normal and diseased cellular states. Additionally, we highlight emerging technologies that promise to bridge critical gaps in our understanding of nuclear morphology, including new approaches to probe nuclear structure and the use of synthetic cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adc2c44f577e2c2679b995d4aca9623ddc124e59" target='_blank'>
              How the chromatin landscape influences nuclear morphology
              </a>
            </td>
          <td>
            Sourabh Sengupta, Haritha Prabha, Daniel L. Levy
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
. The ability of cancer cells to disseminate from the primary tumor and form metastatic lesions frequently leads to fatal outcomes. Recently, however, it has been recognized that this process is driven by complex interactions between the cancer and the neighboring cells, and, overall, made possible by a supportive tumor microenvironment. The emergence of high-throughput technologies is expected to bring much needed clarity to unraveling the players and intricate communication pathways that promote metastatic progression.


AREAS COVERED
In this report, the impact of mass spectrometry and proteomic technologies on deciphering the cross-talk between cancer and tumor microenvironment cells is discussed. Focus is placed on the role of cell-membrane and secretome proteins as the main enablers of this cross-talk, and on the challenges presented by metastatic tumors that evolve in the brain. Future prospects are assessed in the context of recent biology, technology, and data analysis breakthroughs.


EXPERT OPINION
Advancements in high-throughput proteomic technologies, complemented by the exciting potential of new disease model systems and data processing abilities of artificial intelligence, are expected to bring groundbreaking progress in deciphering the fundamental biological mechanisms that support cancer behavior and metastatic development, revealing novel therapeutic targets, and guiding innovative intervention approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad606f697b154e1d0963d77bb80a6ca95fd8151f" target='_blank'>
              Proteomic technologies for profiling cell-membrane/secretome interactions in brain metastatic cancer progression.
              </a>
            </td>
          <td>
            I. Lazar
          </td>
          <td>2025-07-17</td>
          <td>Expert review of proteomics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Periodontitis, a chronic inflammatory condition of the periodontium, is associated with over 60 systemic diseases. Despite advancements, precision medicine approaches have had limited success, emphasizing the need for deeper insights into cellular subpopulations and structural immunity, particularly gingival keratinocytes. This study employs autoencoder models and data augmentation techniques to explore the transcriptomic diversity of gingival keratinocytes at the single-cell level. Single-cell RNA sequencing data from GSE266897 were processed using the Scanpy library, with quality control implemented to filter cells based on predefined metrics. Clustering was performed using principal component analysis (PCA) and k-nearest neighbor (KNN) algorithms. Marker gene identification and differential expression analysis were used to characterize cell clusters. Visualization techniques, including UMAP, heatmaps, dot plots, and violin plots, provided insights into gene expression patterns. The autoencoder architecture featured an encoder reducing input size to 256 units with ReLU activation, a bottleneck layer, and a decoder restoring data dimensions. The basic Autoencoder (AE) demonstrated superior performance, achieving the lowest loss (0.725), the highest accuracy (0.695), and minimal false positives. The Test-Time Augmentation AE also performed robustly, achieving an F1 score of 0.642 and an AUC-ROC of 0.800. The Basic AE effectively modeled RNA-seq data complexity compared to Variational and Denoising Autoencoders. This study highlights advanced computational techniques to investigate gingival keratinocytes’ transcriptomic diversity, revealing distinct subpopulations and differential gene expression profiles. These findings underscore the active role of keratinocytes in periodontal health and inflammatory responses, contributing to precision medicine approaches in periodontology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54234971a14ce3e8edfa17d85d926435fc3b536" target='_blank'>
              Leveraging autoencoder models and data augmentation to uncover transcriptomic diversity of gingival keratinocytes in single cell analysis
              </a>
            </td>
          <td>
            P. K. Yadalam, P. Natarajan, Carlos M. Ardila
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Ovarian cancer (OV) is the deadliest gynecologic malignancy owing to its late diagnosis and high metastatic propensity. Current biomarkers lack sufficient sensitivity and specificity for the detection of early-stage cancer. To address this gap, we integrated single‐cell transcriptomic profiling of tumor tissues with analysis of circulating exosomal RNA, aiming to uncover candidate markers that reflect tumor heterogeneity and metastatic potential and that may serve as sensitive, non‐invasive diagnostics. Methods We integrated single-cell RNA sequencing (scRNA-seq) data from primary tumors and metastatic lesions with bulk tissue transcriptomes and plasma-derived exosomal RNA sequencing (RNA-seq). Differentially expressed genes (DEGs) shared across tumor cells, metastatic subpopulations, and exosomes were identified through intersection analysis. Candidate genes were validated in clinical specimens using qPCR and immunohistochemistry. We then applied ten machine learning algorithm to exosomal transcriptomic data to evaluate diagnostic performance and identify the optimal classifier. Tumor cell differentiation states were evaluated using CytoTRACE, and intercellular communication was analyzed with CellChat. Results Intersection analysis highlighted 52 overlapping DEGs, of which SCNN1A and EFNA1 emerged as the top prognostic indicators. Both genes were significantly upregulated in tumor tissues, metastatic foci, and plasma exosomes (P < 0.01). The exosome-based Adaboost model had an area under the curve of 0.955 in an independent test cohort. Single-cell subcluster analyses revealed high SCNN1A/EFNA1 expression correlated with stem-like differentiation states and enriched pathways associated with immune evasion and adhesion. CellChat analysis demonstrated that highly differentiated tumor cells extensively engaged with fibroblasts and endothelial cells, implying their role in niche formation. Conclusions By coupling single-cell, bulk tissue, and exosomal transcriptomics, we elucidated the key molecular drivers of OV metastasis and established SCNN1A and EFNA1 as promising non-invasive biomarkers. This multi-omics framework provides an effective strategy for early detection and a better understanding of metastatic progression in OV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0c71607fd05b63fb6eb088e5cf81be9d264179" target='_blank'>
              Integrative single-cell and exosomal multi-omics uncovers SCNN1A and EFNA1 as non-invasive biomarkers and drivers of ovarian cancer metastasis
              </a>
            </td>
          <td>
            Liping Tang, Dong Pang, Chengbang Wang, Jiali Lin, Shaohua Chen, Jiangchun Wu, Junqi Cui
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tumor microenvironment (TME) plays an important role in tumorigenesis, development, metastasis and drug sensitivity, but little is known about it in lung squamous cell carcinoma (LUSC). Here, the RNA-sequencing data, clinical and survival data of patients with LUSC in The Cancer Genome Atlas and six independent datasets were collected. Based on the unsupervised clustering of knowledge-based functional gene expression signatures, LUSC was classified into four subtypes. Cluster1 and cluster3 exhibited substantial tumor immune infiltration, suggesting a better response to immunotherapy. Relatively worse survival was observed in cluster4, probably due to higher angiogenesis. Besides, differentially expressed genes in cluster1, cluster2 and cluster3 were prominently enriched in immune-related pathways, while extracellular matrix-related pathways were enriched for cluster4. Genomic data analyses showed significant variations in tumor mutational burden and mutational frequency of several genes, such as TP53, among the four subtypes. Additionally, the four subtypes exhibited heterogeneity in the sensitivity of commonly used chemotherapy drugs for lung cancer and the intratumor microbiome profile. Finally, a prognostic model was developed and its performance and generalization ability were independently validated in multiple datasets. Overall, our study advances the understanding of the TME in LUSC and proposes a prognostic model that facilitates clinical decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad5b9219c70b502bb25fefa3777b3f5d8d780eb" target='_blank'>
              Multi-dimensional characterization of the tumor microenvironment profiles in lung squamous cell carcinoma.
              </a>
            </td>
          <td>
            Yi Zhou, Lixun Chai, Yuyao Wang, Hongguang Zhang
          </td>
          <td>2025-06-25</td>
          <td>Physiological genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play immunosuppressive roles in the tumor microenvironment (TME). Specifically, they reportedly act as physical barriers preventing immune cell infiltration. However, the spatial relationships between CAFs and cancer cells in antitumor immunity remain unknown. In this study, we established three-dimensional (3D) constructs, in which the spatial relationships were controlled using a 3D bioprinter. Using these models, we found that the mixed distribution of fibroblasts (FBs) and cancer cells suppressed the antitumor immunity more than the surrounding distribution of FBs as physical barriers. The 3D construct with mixed distribution promoted transforming growth factor (TGF)-β and periostin (encoded by Postn gene) crosstalk, resulting in immunosuppression. Postn-knockdown in FBs decreased the TGF-β production in the mixed 3D construct and activated antitumor immunity both in vitro and in vivo. Clinically, patients with head and neck cancer or lung cancer showing mixed distribution of α-smooth muscle actin+ myofibroblast-like CAFs exhibited worse prognosis after programmed death protein-1 blockade therapies, and lower CD8+ T cell infiltration than those that had CAFs surrounding cancer cells. Overall, our findings suggest that the close interactions of CAFs and cancer cells facilitate immunosuppression, rather than the physical barriers created by CAFs, highlighting their potential as biomarkers and therapeutic targets for cancer immunotherapies based on spatial relationships. Furthermore, this study highlights the beneficial applications of 3D bioprinters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ca705d8cf5eb81819b79af3fbcdff5bc29b929" target='_blank'>
              Close Spatial Interactions between Cancer Cells and Cancer-Associated Fibroblasts Suppress Antitumor Immunity.
              </a>
            </td>
          <td>
            Y. Naoi, Yumi Inukai, Tomoka Izumikawa, Joji Nagasaki, Takamasa Ishino, Yuki Ueda, Yin Min Thu, Miho Fujiwara, Takahiro Baba, Go Makimoto, K. Suzawa, Kazuhiro Okada, Ken-Ichi Yamamoto, Masakiyo Sakaguchi, Shuta Tomida, Y. Maeda, Shinichi Toyooka, Mizuo Ando, Yosuke Togashi
          </td>
          <td>2025-07-01</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common malignant tumors with high incidence and mortality. The challenge remains to construct reliable prognostic prediction models and to further elucidate the key molecular mechanisms of tumor progression. To address this, we performed WGCNA based on 120 immune cell expression profiles from GEO sources to obtain a collection of monocytes/macrophages-related genes. The prognostic model was constructed by univariate survival analysis and LASSO regression analysis. Then, the prognostic model was validated by Multivariate Cox regression, Kaplan-Meier survival analysis and ROC analysis. In this prognostic model, we identified that PLIN2 has a potential value for CRC prognosis. PLIN2 expression in monocytes/macrophages was verified by scRNA-seq datasets and spatial transcriptome datasets, and PLIN2 was found to promote macrophage transformation to M2 subtype. Clinical specimens and tissue microarrays confirmed the differential expression and prognostic value of PLIN2 in CRC patients. Functional experiments demonstrated that PLIN2 gene overexpression promoted the proliferation, migration and invasion of CRC cells and significantly facilitated tumor growth in vivo. Mechanistically, we revealed that CD36 is a potential downstream target gene of PLIN2. The CD36 inhibitor Sulfo-N-succinimidyl Oleate significantly reversed PLIN2-induced proliferation, migration, invasion, and EMT activity of CRC cells in vitro and in vivo. Immunoprecipitation and immunofluorescence experiments confirmed that PLIN2 could interact with CD36. PLIN2 stabilized CD36 protein expression by inhibiting the proteasomal degradation pathway, thereby promoting CD36-mediated EMT activity. Overall, our study highlights that the PLIN2/CD36 axis regulates EMT activity and CRC progression, suggesting that interventions in this signaling pathway may offer a promising therapeutic approach to CRC progression. Schematic diagram elucidating the role of PLIN2 in CRC by Figdraw. FA is transported into the cell via CD36-mediated endocytosis. In CRC cells, PLIN2 promotes stability of CD36 and interacts with CD36 to activate the EMT process. However, the CD36 inhibitor SSO inhibits the binding of FAs to CD36 and attenuates its endocytosis, thereby reversing the PLIN2-mediated EMT process. Ultimately, the PLIN2-induced enhancement of CRC cell proliferation, migration, and invasion is attenuated by the CD36 inhibitor SSO.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e0a7e18e5633e2450668612a5920001ab2297dc" target='_blank'>
              PLIN2 promotes colorectal cancer progression through CD36-mediated epithelial-mesenchymal transition
              </a>
            </td>
          <td>
            Fan Yang, Ying Li, Xue Shang, Yun Zhu, Wenting Hou, Yi Liu, Qing Hua, Zhirong Sun
          </td>
          <td>2025-07-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Oropharyngeal candidiasis (OPC) caused by the fungus Candida albicans triggers a robust inflammatory response that rapidly eliminates the bulk of the fungal load in the oral mucosa. While pro-inflammatory responses to OPC have been extensively studied, little is known about the counterbalance of immunity in the infection milieu that mitigates inflammatory damage. We employed 10× Visium Spatial Transcriptomics, a next-generation technology that preserves the spatial integrity of infected tongue tissues, enabling high-resolution mapping of the host microenvironment during Candida infection in a murine OPC model. This approach provided a view of cellular interactions and immune dynamics, revealing intricate crosstalk between distinct immune cell populations. Our findings highlight a previously underappreciated role of the type-2 immune response and M2 macrophages in maintaining tissue homeostasis, and the predicted role of platelet degranulation in facilitating disease resolution. Finally, we identified a novel family of small proline-rich antimicrobial proteins with potent antifungal activity. These molecules emerge as promising therapeutic candidates, offering a new avenue for combating OPC. IMPORTANCE Oropharyngeal candidiasis (OPC), a fungal infection caused by Candida albicans, affects individuals with weakened immune systems. Our study used spatial transcriptomics, a cutting-edge technology that preserves tissue architecture while mapping immune interactions at high resolution. This approach allowed us to uncover previously unrecognized cellular crosstalk and regulatory pathways that shape the host response to OPC. We discovered that platelets, beyond their role in clotting, play a key role in antifungal defense. Additionally, M2 macrophages are important for resistance to OPC. Most notably, we identified a new family of antimicrobial proteins with strong antifungal properties, presenting promising therapeutic potential. By uncovering these overlooked immune mechanisms, our research findings may lead to better treatments for OPC, particularly in immunocompromised individuals. Oropharyngeal candidiasis (OPC), a fungal infection caused by Candida albicans, affects individuals with weakened immune systems. Our study used spatial transcriptomics, a cutting-edge technology that preserves tissue architecture while mapping immune interactions at high resolution. This approach allowed us to uncover previously unrecognized cellular crosstalk and regulatory pathways that shape the host response to OPC. We discovered that platelets, beyond their role in clotting, play a key role in antifungal defense. Additionally, M2 macrophages are important for resistance to OPC. Most notably, we identified a new family of antimicrobial proteins with strong antifungal properties, presenting promising therapeutic potential. By uncovering these overlooked immune mechanisms, our research findings may lead to better treatments for OPC, particularly in immunocompromised individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/901da1fcdc673e05f0604138f1185396e69199f8" target='_blank'>
              A spatial transcriptomic atlas of the host response to oropharyngeal candidiasis
              </a>
            </td>
          <td>
            Sunna Nabeela, Hayden McSwiggin, R. Magalhães, Eliciane Mattos, Mohammad Mannan, John T. Dillon, Ashley Barbarino, E. Youssef, Shakti Singh, Wei Yan, Ashraf S. Ibrahim, Heather R. Conti, Priya Uppuluri
          </td>
          <td>2025-06-30</td>
          <td>mBio</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumor heterogeneity and the dynamic evolution of tumor immune microenvironment (TIME) contribute to therapeutic resistance and poor clinical prognosis. To elucidate this mechanism, we first established a murine tumor evolution model (TEM) and systematically identified evolutionary core genes demonstrating progressive alterations during evolution. Subsequently, we developed a single-cell TEM through integrative analysis of hepatocellular carcinoma (HCC) clinical specimens (n=10) with external cohorts (n=11), enabling dynamic characterization of tumor-immune interactions during evolution, while addressing ethical challenges associated with obtaining tumor tissues from multiple stages in a single patient. Through TEMs analyses, we identified a contrasting glucose metabolism pattern between malignant cells and CD8+ T cells during tumor evolution. Mechanistically, glucose metabolic dominance triggers NSUN2 upregulation in tumor cells, where this functional RNA methyltransferase stabilizes key glycolytic transcripts (GLUT1, HK2, PFKM) through mRNA methylation. The NSUN2-mediated GLUT1 stabilization enhances the competitive advantage of tumor cells in glucose acquisition, creating a positive feedback loop that accelerates malignancy and exacerbates CD8+ T cell dysfunction. Building on these insights, we designed a dual-targeting strategy combining GLUT1/NSUN2 axis inhibitor WZB117 with PD-L1 blockade, which synergistically suppressed tumor evolution and reversed immunosuppression in preclinical models, suggesting a novel synergistic therapeutic strategy for treatment-resistant HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b96376e6c37993b534f4d68956c9aa4ba9723b03" target='_blank'>
              The glucose sensor NSUN2-m5C modification regulates tumor-immune glucose metabolism reprogramming to drive hepatocellular carcinoma evolution
              </a>
            </td>
          <td>
            Jing He, Boqiang Liu, Weijun Zhao, Hao Shen, Yi Wang, Weiqi Li, Chenqi Jin, Yifan Wang, Xiujun Cai, Liang Shi
          </td>
          <td>2025-07-11</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Human endogenous retroviruses (HERVs) are remnants of ancient viral infections and comprise 6–8% of the human genome. Their biological functions in cancer remain poorly understood, especially in glioblastoma, the most common and deadly primary brain cancer in adults. Prior studies on HERV expression in glioblastoma have yielded conflicting results. Here, we employed orthogonal computational pipelines to address these limitations. Results Locus-specific analysis revealed marked heterogeneity of transcription among HERVs within the same clade. We found that individual HERV proviruses are more than twice as likely to be underexpressed in glioblastoma than overexpressed and that most differentially expressed HERVs are exonized within transcripts. These HERVs are enriched in the 3’-terminal exons of transcripts, associated with alternative polyadenylation and contribute conserved polyadenylation signals. We identified HERV expression patterns associated with glioblastoma subtypes and anatomic features. We also identified three proviruses or proviral fragments of particular interest including one associated with survival and one exonized within a currently unannotated cancer-specific transcript. Among the most recently integrated clade of HERVs, excluding solo-LTRs, only the HML-2 provirus at 1q22 is overexpressed in glioblastoma. We report previously undescribed transcripts incorporating this provirus that may encode several proteins. Conclusions This study represents the first systematic exploration of the heterogeneity of HERV expression between anatomic features of any cancer. It shows that exaptation of HERV polyadenylation signals and HERV-associated APA are widespread in the human transcriptome and identifies critical structural information regarding the HML-2 1q22 provirus transcript, which has been the focus of several recent analyses. Our findings underscore the importance of locus-specific and anatomic feature-specific HERV analysis and suggest structural and functional roles for HERVs in glioblastoma-associated transcripts. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00365-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae712fda8b06db59cbe0784641111444d4db678e" target='_blank'>
              In-depth analysis of endogenous retrovirus expression in glioblastoma
              </a>
            </td>
          <td>
            Daniel A. Murimi-Worstell, Irene B. Murimi-Worstell, Farrah M. Roy, A. Burn, Michael Freeman, Ralph B. Puchalski, John M. Coffin
          </td>
          <td>2025-07-16</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bad13afb77ed20c24d4a2ca39dbd49aeea916cc3" target='_blank'>
              ODFormer: a Virtual Organoid for Predicting Personalized Therapeutic Responses in Pancreatic Cancer
              </a>
            </td>
          <td>
            Jing Xu, Xiaolu Yang, Yunguang Li, Huan Wang, Yikai Li, Shijie Tang, Shiwei Guo, Jiawei Zou, Xiaohan Shi, Zexi Chen, Chaoliang Zhong, Penghao Li, W. Jing, Kai-lian Zheng, Xinqian Wu, Dong Gao, Luonan Chen, Gang Jin
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The testis-specific protein Y-linked (TSPY) is a male-specific cancer-testis antigen specifically expressed in germ cells of the testis under normal conditions and various cancers, particularly in hepatocellular carcinoma (HCC), under oncogenic conditions. It binds to cyclin B and exacerbates the cyclin B-CDK1 phosphorylation of factors important for mitotic/meiotic divisions. To determine if such TSPY proliferative actions could contribute to various male-biases in liver cancer, TSPY transgene was expressed in an oncogene-induced preclinical mouse model of HCC, using the hydrodynamic tail vein injection strategy. The results showed that TSPY expression suppressed tumor cell growth at early stage but could evolve to resume oncogenic progression at late stage in this mouse model. Transcriptome and bioinformatic analyses demonstrated that significant immune and inflammatory responses were activated in early stage of the cancer, resulting in elimination of positive tumor cells. Significant TSPY antibodies were present in the sera of positive mice, similar to the presence of autoantibodies in the sera of patients positive for TSPY in their tumors. Flow cytometry and cellular protein fractionation analyses of positive tumor cells showed that TSPY protein could be mislocalized on the cell surface and likely be responsible for the humoral immunity. Additional studies demonstrated that TSPY peptides were produced and could form complexes with MHC-I molecules and be presented on the cell surface, thereby eliciting robust cytotoxic T cell responses and killing of positive tumor cells. Importantly these immune responses diminished and TSPY could exacerbate oncogenic growth at late stage. These findings suggest that as a male-specific cancer-testis antigen, TSPY is extremely immunogenic capable of eliciting robust immune and inflammatory responses at the early stage and is a significant candidate for development of immunotherapeutics, such as therapeutic cancer vaccine and antibody-drug conjugates, in treatments of hepatocellular carcinoma in men. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01432-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/385f798a691d59fcc01fce449e9f2fc212a4f4f7" target='_blank'>
              The human testis-specific protein Y-linked (TSPY) is a male-specific cancer-testis antigen capable of eliciting significant immune responses and elimination of positive tumor cells in hepatocellular carcinoma
              </a>
            </td>
          <td>
            T. Kido, Yun-Fai Chris Lau
          </td>
          <td>2025-06-25</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="INTRODUCTION
Osteosarcoma, a highly aggressive bone tumor, continues to pose significant treatment challenges despite advances in molecular research. Traditional therapeutic strategies have largely relied on targeting genetic alterations of tumor genes or signaling pathways, but these approaches have been less effective in clinical settings due to the complex biology.


AREAS COVERED
Recent insights into the molecular landscape of osteosarcoma have revealed key mechanisms of therapeutic resistance, including tumor plasticity, immune evasion, and metabolic reprogramming. The interaction between the tumor and its microenvironment, such as mechanical stress, hypoxia, and extracellular matrix composition, leads to spatially distinct regions with varying drug sensitivities.


EXPERT OPINION
We highlight three key shifts in understanding osteosarcoma biology: 'target plasticity' driven by evolving tumor dynamics, the importance of mechanical signaling at the tumor-bone interface, and the potential of multi-omics platforms for real-time monitoring and personalized treatment. We propose a new therapeutic framework that integrates these advances to overcome resistance mechanisms. By focusing on epigenetic reprogramming, immune resetting, and mechanopharmacological approaches, we envision a more comprehensive strategy for osteosarcoma treatment that goes beyond traditional single-target therapies. The success of these strategies will depend on integrating spatially informed, time-resolved treatment regimens, guided by advanced molecular and computational technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ffde0bbfbd90bb0bf5e018161ce107f39f7740f" target='_blank'>
              Assessing the current molecular understanding of therapeutic targets in osteosarcoma.
              </a>
            </td>
          <td>
            Chao Zhang, Ji-Lan Yang
          </td>
          <td>2025-08-07</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background The unique stress response state of T cells (Tstr) is found in various cancers and correlates with altered Lung Adenocarcinoma (LUAD) tumor microenvironment and immunotherapy outcomes. However, fewer studies have been described on the relationship between Tstr and LUAD, and its mechanism of action in the pathogenesis of LUAD needs to be further investigated. Methods The categorization and validation of cluster types for Tstr signature genes were systematically carried out using transcriptomic data sourced from the TCGA and GEO databases. The focus was on elucidating distinct pharmacological characteristics, tumor microenvironmental traits, and immunotherapeutic benefits across various subtypes. Moreover, a predictive model was developed through Least Absolute Shrinkage and Selection Operator (LASSO) regression analysis, and subsequent cellular experiments confirmed the pivotal role of CREG2 in driving LUAD cell proliferation and migration. Results Tstr signature genes represented by the heat shock protein family are highly expressed at the end of T-cell differentiation and affect cellular communication. Patients with the C1 subtype have a poorer prognosis and high expression of HSPA1A and HSPA1B, which exhibit cold tumor characteristics and define them as important Tstr subtypes. CREG2 is positioned as a promising biological indicator as it plays an important part in enhancing the migratory and proliferative capacity of LUAD cells. Discussion This study reveals the characteristics of different Tstr types. A predictive model was built to forecast the prognosis of LUAD patients based on the DEGs of the subtype. The exploration delved into the bio-function of CREG2 in the progression of LUAD. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03170-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77dda53b77f8e92a63f02ef3e1d805a06740b60e" target='_blank'>
              Integration of single cell and bulk transcriptomes identifies T cell stress subtypes in LUAD
              </a>
            </td>
          <td>
            Shengping Min, Linfeng Pan, Xinyu Zhang, Huili Chen, Lixuan Qiu, Xinyu Wang, Yiluo Xie, Kai Zhang, Qiang Zhang, Chaoqun Lian, Jing Zhang
          </td>
          <td>2025-07-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The bone marrow microenvironment is intimately linked to the biology that underpins the development and progression of multiple myeloma. However, the complex cellular and molecular features that form bone marrow niches are poorly defined. Here, we used subcellular spatial transcriptomics to profile the expression of 5,001 genes in human bone marrow in the context of multiple myeloma. Using this approach, we explored the plasma cell and stroma ecosystem in bone marrow trephines from 21 individuals, including 7 with pre-malignant disease and 10 with newly diagnosed multiple myeloma. Using spatial transcriptomics in conjunction with an optimised trephine biobanking methodology, we could resolve major components of the human bone marrow microenvironment and reliably characterise distinct plasma cell populations in samples from healthy, pre-malignant disease and active myeloma. When plasma cells were visualised in the context of location, we detected spatially restricted subpopulations of plasma cells in five of ten newly diagnosed myeloma trephines. Surprisingly, the composition of haematopoietic and stromal microenvironments varied significantly between newly diagnosed myeloma trephines. Furthermore, these differences in microenvironments were also observed within trephines that had spatially restricted plasma cell subpopulations. Thus, these data are not consistent with the hypothesis that a universal bone marrow microenvironment supports the expansion of malignant plasma cells in myeloma. Instead, we propose that myeloma subpopulations form distinct microenvironments and can vary between both patients and spatial location.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb6467d36212cc2446a47ddf3e54593d16e09483" target='_blank'>
              Profiling the spatial architecture of multiple myeloma in human bone marrow trephine biopsy specimens with spatial transcriptomics.
              </a>
            </td>
          <td>
            R. K. Yip, Jeremy Er, Lei Qin, Quoc Hoang Nguyen, Allan Motyer, Joel S. Rimes, Amanda Light, Ruvimbo D. Mishi, Ling Ling, Casey J. A. Anttila, Ellen Tsui, D. Amann-Zalcenstein, Mark R. Dowling, K. Rogers, R. Bowden, Yunshun Chen, Simon J Harrison, Edwin D Hawkins
          </td>
          <td>2025-07-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Malignant glioma is a highly aggressive brain tumor characterized by frequent recurrence, poor prognosis, and limited responsiveness to standard therapies. Glioblastoma multiforme, the most common and aggressive subtype, further complicates treatment due to its infiltrative nature, genetic heterogeneity, the protective blood-brain barrier, and an immunosuppressive microenvironment. Despite aggressive treatment strategies such as surgical resection combined with chemoradiotherapy, the median survival for malignant glioma patients remains low, highlighting the urgent need for more effective therapeutic approaches. Oncolytic virotherapy (OVT), a dual-modality approach that combines immunotherapy and biotherapy, has emerged as a promising alternative. Oncolytic viruses (OVs) can replicate continuously, disseminate within the tumor, and stimulate anti-tumor immunity, offering distinct advantages in targeting invasive and immunologically “cold” malignant glioma. However, the efficacy of OVT in clinical trials remains unsatisfactory, particularly in single-agent regimens. This limitation is primarily attributed to the viruses’ limited replication efficiency, suboptimal immune induction, premature clearance by antiviral immune responses, and the blood-brain barrier, which impedes effective intracranial delivery. Thus, further optimization of viral modifications, delivery systems, and treatment regimens is critical to enhancing therapeutic potency before OVT can become a standard therapy for malignant glioma. This review systematically summarizes current strategies for enhancing OVs, including genetic engineering, chemical functionalization, and carrier-based delivery. Furthermore, it highlights combination therapies that aim to synergistically enhance therapeutic efficacy through chemotherapy, radiotherapy, and immunotherapy. Finally, the review emphasizes recent clinical trials leveraging these enhancement strategies, aiming to offer novel insights for translating OVs from research to clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f823dd6016cb2ab73e24869fff2ae51d94c3fc49" target='_blank'>
              Optimizing Oncolytic Virotherapy for Malignant Glioma: From Bench to Bedside
              </a>
            </td>
          <td>
            Weihua Jiang, Yeqing Tian, Huafen Gu, Wenqing Guan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Management and Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune evasion represents a significant challenge in oncology. It allows tumors to evade immune surveillance and destruction, thereby complicating therapeutic interventions and contributing to suboptimal patient outcomes. This review addresses the critical need to understand how cancers evade immune surveillance. It aims to provide a comprehensive overview of strategies of tumors to escape immune detection by examining tumor-induced immune suppression, immune checkpoint regulation, and genetic and epigenetic influences. Moreover, it explores the dynamic role of the tumor microenvironment (TME) in fostering immune resistance and highlights the impact of metabolic reprogramming on immune suppression. Additionally, this review focuses on how tumor heterogeneity influences immune evasion and discusses the limitations of current immunotherapies. The role of key signaling pathways, including programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte–associated antigen 4 (CTLA-4), transforming growth factor-β (TGF-β), nuclear factor kappa-light-chain-enhancer of activated B cells (NF-κB), and cyclic GMP-AMP synthase-stimulator of interferon genes (cGAS–STING) is analyzed to elucidate their contributions to immune escape. Emphasizing the complexities of immune evasion, this review underscores the importance of personalized approaches and the integration of multi-omics data to combat therapeutic resistance. Furthermore, it discusses novel and emerging therapeutic strategies, such as bispecific antibodies, oncolytic viruses, and nanotechnology-driven immunotherapies, showcasing innovative avenues in cancer treatment. The significance of this review lies in its potential to guide future research and innovations in immunotherapy, ultimately improving patient outcomes and advancing our understanding of cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b9bd056fe3296a46c5f79e0e0f8c410fd536528" target='_blank'>
              Immune evasion in cancer: mechanisms and cutting-edge therapeutic approaches
              </a>
            </td>
          <td>
            Muhammad Tufail, Canhua Jiang, Ning Li
          </td>
          <td>2025-07-31</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Oxaliplatin-based neoadjuvant chemotherapy (NAC) is the standard treatment for advanced colorectal cancer (CRC), yet resistance to NAC poses a significant clinical challenge. Methods To investigate the mechanisms of chemoresistance, we analyzed single-cell RNA sequencing (scRNA-seq) data from CRC patients undergoing NAC. Comprehensive analyses, including InferCNV, differentially expressed genes analysis, pathway enrichment, cell communication, and SCENIC were performed. High-throughput drug screening identified potential therapeutic candidates targeting chemoresistance pathways, and the efficacy of targeting the KLF5/PI3K/AKT axis in combination with oxaliplatin was explored in animal models. Results NAC effectively reduced tumor burden and enhanced T_NK cell infiltration in responsive tumors. Notably, NAC-resistant cell clusters exhibited activation of fatty acid-related metabolic pathways and demonstrated limited immune infiltration. Transcriptional analysis identified KLF5 as a potential driver of chemotherapy resistance. Based on these findings, we developed a KLF5 regulon-associated risk score model with significant potential for predicting CRC patient prognosis. Mechanistically, KLF5 activation of the PI3K/AKT pathway conferred chemoresistance in CRC cells. Through high-throughput screening, GDC-0941, a PI3K/AKT inhibitor, emerged as a promising therapeutic agent that synergistically enhanced oxaliplatin efficacy and overcame resistance in preclinical models. Conclusions Targeting the KLF5/PI3K/AKT axis may enhance chemotherapy efficacy and overcome drug resistance in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89694e65cc06e36cf7753e13edc503c27ae0798" target='_blank'>
              Targeting the KLF5/PI3K/AKT axis as a therapeutic strategy to overcome neoadjuvant chemoresistance in colorectal cancer
              </a>
            </td>
          <td>
            Meiling Gao, Jiahao Qian, Ping Xia, Wancheng Liu, Yunjia Jiang, Yuchang Xia, Xin Yao, Q. Jiao, Minggang Wei
          </td>
          <td>2025-07-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c43e8f31b705481e167f098065b8bddd3cfad835" target='_blank'>
              Unraveling the role of perineural invasion in cancer progression across multiple tumor types.
              </a>
            </td>
          <td>
            Muqtada Shaikh, Sanket Shirodkar, G. Doshi
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Macrophages often pervade solid tumors, and clusters of macrophages sometimes associate with longer survival of patients. However, clustering mechanisms and impacts on key functions such as phagocytosis remain obscure. Here, under conditions that maximize cancer cell phagocytosis within cohesive tumors, we uncover pathways that favor dynamic clusters and find a colocalization of tumor-intrusive pseudopodia which we term "intrudopodia." Cluster formation over hours on low-adhesion substrates occurs after macrophage induction to a state colloquially referred to as M1 after exposure to interferons and T cell-derived cytokines. Clusters prove fluid on timescales of minutes and also sort from interleukin-4-treated, so-called M2 macrophages that tend to disperse. M1 macrophages upregulate specific cell-cell adhesion receptors but suppress actomyosin contractility, with both pathways contributing to cluster formation. Decreased cortical tension was not only reflected in a low level of nuclear lamin-A that downregulates cytoskeletal targets of serum response factor and tends to soften the nucleus but was also predicted to unleash pseudopodia. Macrophage neighbors in tumor spheroids indeed coextend intrudopodia between cancer cell junctions-at least when phagocytosis conditions are maximized. Intrudopodia from neighbors help detach and individualize cancer cells for rapid engulfment. Juxtaposition of a macrophage cluster with tumor cell nests defines a broad interface that minimizes cancer cell nearest neighbor interactions and maximizes coordination of macrophage intrudopodia. Cooperative phagocytosis thus overcomes solid tumor cohesion-and might explain why the macrophage clustering factor ITGAL associates with patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a55b2dafa57fa725379d7dc71b70c66988362113" target='_blank'>
              Clustered macrophages cooperate to eliminate tumors via coordinated intrudopodia.
              </a>
            </td>
          <td>
            L. Dooling, Alişya A Anlaş, Michael P. Tobin, Nicholas M. Ontko, Tristan Marchena, Maximilian Wang, Jason C. Andrechak, D.E. Discher
          </td>
          <td>2025-07-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Osteosarcoma, a prevalent malignant bone tumor in pediatric and adolescent populations, is characterized by a markedly diverse tumor microenvironment (TME) that complicates therapeutic intervention. Among the key components of the TME, M2 macrophages contribute to immune suppression, tumor progression, and resistance to therapy. This study aims to explore the immune microenvironment of osteosarcoma and identify key immunosuppressive components using integrated transcriptomic analysis.



 We integrated scRNA-seq (GSE152048) and bulk RNA-seq (TARGET) data to characterize the osteosarcoma tumor microenvironment. Key analyses included cell clustering and annotation (Seurat), pseudotime trajectory (Monocle 2), cell-cell communication (CellChat), and M2 macrophage deconvolution (CIBERSORT, BayesPrism). Gene modules were identified via WGCNA, and drug sensitivity was predicted using oncoPredict. Key genes were validated by qRT-PCR and Western blot.



 Our integrated analysis revealed that M2 macrophage infiltration was significantly associated with an immunosuppressive tumor microenvironment in osteosarcoma. Weighted gene co-expression network analysis (WGCNA) identified gene modules highly correlated with M2 macrophage abundance and immune evasion signatures. Reduced M2 infiltration was linked to altered transcriptional patterns and increased predicted sensitivity to Etoposide and Doxorubicin. Two immune-related genes, PTK2B and PTPRC, were significantly upregulated in osteosarcoma samples and validated by RT-qPCR and Western blotting in U2OS cells.



 This study demonstrates that M2 macrophage infiltration is a defining feature of the osteosarcoma immune microenvironment and is associated with immune suppression. Through integrative transcriptomic analysis, PTK2B and PTPRC were identified as key immune-related genes linked to M2 macrophage abundance and patient prognosis. These results establish a transcriptional basis for the immunosuppressive phenotype in osteosarcoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c9aeb21260637fca67e1110200fbde1ea1d4d" target='_blank'>
              Integrated RNA sequencing reveals tumor microenvironment heterogeneity and immunosuppressive role of M2 macrophages in osteosarcoma
              </a>
            </td>
          <td>
            Zhijun Ding, Yi Li, Cheng Zhong
          </td>
          <td>2025-08-01</td>
          <td>Oncologie</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Glioma, a tumor derived from neuroglial cells of the central nervous system, is characterized by its high malignancy, frequent recurrence, and high mortality rate. Recent advances in exosome research related to gliomas have propelled this emerging field as a pivotal area of clinical neuroscience investigation. Despite extensive focus on glioma exosome research, a systematic visual analysis of this field remains lacking. Using bibliometric analysis, this study maps the scientific landscape of glioma exosome research to identify current hotspots and emerging trends in neuro-oncology. Methods This bibliometric analysis extracted glioma exosome publications from the Web of Science Core Collection(WoSCC) spanning 2005–2025. Following systematic screening, 209 articles constituted the analytic corpus. Using VOSviewer(VOS), Scimago Graphica, and CiteSpace, we mapped contributions across geographic, institutional, authorship, and conceptual domains. This visualization delineates the field’s intellectual structure, identifies current research foci, and projects emerging trends in neuro-oncology. Results Annual publication volume on glioma exosomes has increased significantly since 2019. China and the United States are the predominant contributors, collectively driving field advancement. Leading institutions include Shandong University, Shandong Provincial Key Laboratory of Brain Function Reconstruction, and Capital Medical University, whose productivity metrics reflect substantial influence. Notably, Chinese institutions comprise 90% of the top 10 productive entities. Key investigators Li Gang and Xue Hao demonstrate exceptional scholarly output, advancing the field through high-impact research. NEURO-ONCOLOGY emerges as the premier journal, evidenced by its dominant citation frequency. Thematic analysis reveals three conceptual domains: Exosome biogenesis in gliomas, Glioblastoma extracellular vesicle signatures, and Exosome mediated drug delivery systems-representing current priorities and evolving research frontiers. Conclusion Exosomes have attracted considerable attention in glioma research, establishing themselves as a cutting edge thematic focus. This study applies bibliometric rigor to map research trajectories and conceptual evolution in glioma exosomes. The analysis establishes foundational frameworks that prioritize future investigation domains while objectively quantifying the field’s translational potential. These evidence-based insights will accelerate mechanistic discovery and clinical translation in neuro-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce5f890f1f8b8ce61524a09398cf2f3457fe7ab4" target='_blank'>
              Evolution of glioma exosome research: emerging trends and global collaborations (2005–2025)
              </a>
            </td>
          <td>
            Yijun Zeng, Ting Liu, Shixue Jing, Yongfeng Wang, Jihong He, Hui Cao, Xinlian Liu, Lushun Zhang
          </td>
          <td>2025-07-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Liquid biopsy offers a promising, minimally invasive approach for cancer detection by providing valuable insights into tumor biology. Among the various sources used in liquid biopsy, platelets and their RNA stand out as a unique diagnostic tool, reflecting the body’s systemic response to cancer. However, during laboratory platelet extraction known as platelet washing, white blood cells (WBCs) retention can occur, potentially confounding platelet-derived RNA sequencing data.



 To address this challenge, we developed a method to quantify WBC enrichment in platelet RNA-seq datasets. Using the largest publicly available dataset (GEO GSE183635), which includes 2,351 samples from cancer patients, healthy donors, and individuals with benign, non-cancer conditions, we identified 3 sample clusters based on leukocyte marker levels. This allowed us to differentiate two entirely distinct sets of samples, with the lowest and the highest WBC retention, containing 377 matched samples in each set. We then assessed how the leukocyte presence influences the performance of machine learning models for cancer classification for the low and high leukocyte subgroup.



 Across the full test set (n=453), our model achieved an area under the curve (AUC) of 0.94. Interestingly, in the low-leukocyte subset (n=229), the AUC slightly decreased to 0.93, whereas in the high-leukocyte subset (n=224), it increased to 0.95.



 These results suggest that while leukocyte retention may slightly enhance classification performance, platelets alone provide substantial diagnostic value. Overall, our findings highlight the strong potential of platelet-based liquid biopsy and reveal how understanding leukocyte retention can further refine cancer detection strategies.



 Michał Sieczczynski, Krzysztof Pastuszak, Anna J. Zaczek, Matthew T. Rondina, Sjors GJG. in 't Veld, Myron G. Best, Anna Supernat. Decoding the Leukocyte Effect: How Cell Retention Shapes ML Outcomes in Platelet RNA-based Cancer Detection [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B034.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1028fbb68d5dca2426241f5c4eaf3adb3c531ac0" target='_blank'>
              Abstract B034: Decoding the Leukocyte Effect: How Cell Retention Shapes ML Outcomes in Platelet RNA-based Cancer Detection
              </a>
            </td>
          <td>
            Michal Sieczczynski, Krzysztof Pastuszak, A. Żaczek, Matt Rondina, S. I. '. In 't Veld, M. Best, A. Supernat
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b31f4399101f9b6167477a72fd46a9e859a5855" target='_blank'>
              Novel Computational Analysis Identifies Cytotoxic Lymphocyte-to-Monocyte Balance in Tumors as a Predictor of Recurrence-Free Survival in Colorectal Carcinoma.
              </a>
            </td>
          <td>
            Manuel Fernandez, L. Todeschini, Bridget P Keenan, David Rosenberg, Sophia Hernandez, M. Zampese, Guilin Qiao, T. Pollini, A. Maker
          </td>
          <td>2025-06-21</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Drug resistance significantly impairs the prognosis of patients with gastric cancer (GC). As a key player in the acquisition of drug resistance, understanding the detailed evolution of the GC tumor microenvironment (TME) is crucial for improving therapeutic effectiveness. The clinical significance of markers related to cancer-associated fibroblast (CAF) proliferation and extracellular matrix remodeling were analyzed using public databases and immunohistochemistry staining on an in-house cohort of patients with GC. Combining this with single-cell RNA sequencing data revealed the expression patterns of all candidate markers, highlighting POSTN due to its pronounced upregulation in CAFs and its strong correlation with poor prognosis in patients with GC. Mechanistically, POSTN is directly regulated by the YAP1/TEAD1 co-transcription factor and is demonstrated to play significant roles in fibroblast proliferation and migration processes. The present study underscores POSTN as a promising marker for predicting GC prognosis and a powerful regulator that can augment the tumorigenic phenotypes of CAFs, providing a potential target to mitigate CAF-originated drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3a85ca4b39083cd7faf716f5d06621406b8862a" target='_blank'>
              POSTN is exclusively activated in cancer-associated fibroblasts and leads to unfavorable prognosis of patients with gastric cancer
              </a>
            </td>
          <td>
            Ching Hei To, Fuda Xie, Peiyao Yu, Jialin Wu, Yang Lyu, Bonan Chen, Tiejun Feng, Hoi Wing Leung, Wei Kang
          </td>
          <td>2025-07-08</td>
          <td>Molecular and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Purpose: The tumor microenvironment (TME) is difficult to model in an in vivo cancer research setting. This study leveraged intratumor microdosing using comparative in vivo oncology (CIVO) with spatial profiling to evaluate the effects of the stimulator of interferon genes agonist dazostinag, alone or with chemotherapy, on cellular responses within the native TME of intact human tumors. Patients and Methods: This phase 0 study enrolled adult patients with head and neck squamous cell carcinoma (HNSCC) planned for surgical intervention. Intratumoral microdose injections of dazostinag (maximum dose: 1.68 μg in a 0.05 mg/mL solution), alone or combined with various chemotherapies, were delivered via CIVO to tumors 24, 48, 72, or 96 hours prior to resection. Each tumor sample was prepared for analysis using IHC and ISH. Analysis of the microdosed tumors using the GeoMx Digital Spatial Profiler and CosMx Spatial Molecular Imager was performed in one patient. Results: Type 1 IFN signaling was induced with dazostinag alone and in combination with chemotherapy from multiple cell types within the TME, including immune cells. Dazostinag also shifted the polarization of macrophages from an immune-suppressive phenotype to a proinflammatory phenotype at 24 hours after injection. Enrichment of cytotoxic T cells was observed in regions of localized dazostinag exposure, coinciding with increased chemokine (CXCL9) expression. Based on cleaved caspase-3, an apoptosis marker, dazostinag plus chemotherapy increased cellular apoptosis relative to either drug alone. Conclusions: Utilizing CIVO and spatial profiling technology, dazostinag alone and combined with chemotherapy promoted an early proinflammatory response and enhanced chemotherapy-mediated cell death in the native TME of intact human HNSCC tumors. Significance: The CIVO approach demonstrates that dazostinag alone and combined with chemotherapy promotes both proapoptotic and early proinflammatory responses in the native TME of intact human HNSCC tumors, providing clinical evidence for an on-target mechanism of action and rationale for further clinical investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56e7554c8e7ac27dada98b874c3acbefe052a196" target='_blank'>
              In Situ Proinflammatory Effects of Dazostinag Alone or with Chemotherapy on the Tumor Microenvironment of Patients with Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Richard C Gregory, N. Lineberry, Alexander Parent, Karthik Rajasekaran, Thomas J Ow, Cherie-Ann Nathan, B. Hatton, Wendy Jenkins, M. Grenley, Connor Burns, Angela Merrell, Jason P Frazier, Jonathan M J Derry, E. Beirne, R. Klinghoffer
          </td>
          <td>2025-07-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="ABSTRACT Background Cervical cancer remains a global public health challenge, particularly in regions with limited access to screening and vaccination. While high‐risk HPV infection is the primary cause, the genetic and molecular mechanisms driving cervical cancer progression are not fully understood. Objective This study integrates Mendelian randomization (MR) and single‐cell RNA sequencing (scRNA‐seq) to identify causal eQTL‐related genes and explore their roles in tumorigenesis. Functional experiments were conducted to validate key findings. Methods MR analysis identified eQTL‐related genes with significant causal associations with cervical cancer. Functional enrichment and Gene Set Variation Analysis (GSVA) revealed their involvement in key pathways. scRNA‐seq explored cell‐specific expression patterns and immune cell infiltration in the tumor microenvironment (TME). In vitro experiments, including qRT‐PCR, siRNA knockdown, migration, proliferation, and colony formation assays, validated the biological roles of pivotal genes. Results A total of 307 eQTL‐related genes were identified, enriched in pathways such as Th17 cell differentiation, TNF, and IL‐17 signaling. scRNA‐seq revealed cell‐specific expression of key genes, including FUOM, which was elevated in cervical cancer cells. FUOM knockdown significantly reduced cell proliferation (by 37%, p < 0.001), migration (by 43%, p < 0.001), and colony formation (by 62%, p < 0.001). Regulatory analysis identified miRNAs as upstream modulators of these genes. Conclusion This study identifies FUOM as a novel driver gene in cervical cancer progression and highlights its role in tumorigenesis and immune modulation. These findings provide insights into potential biomarkers and therapeutic targets, offering a foundation for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f2af3648a1778b7bc1979bb8e9f7cee9b957bcb" target='_blank'>
              Integrative Genomic and Functional Approaches Identify FUOM as a Key Driver and Therapeutic Target in Cervical Cancer
              </a>
            </td>
          <td>
            Wenzhi Jiao, Shanshan Liu, Jianwei Shi, Minmin Yu
          </td>
          <td>2025-08-01</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9d6d4cae3e0a9086b4188f54060ec83ad93a702" target='_blank'>
              Integrating scRNA-seq and spatial transcriptomics to explore the implication of G6PD on immune microenvironment in lymphatic metastasis of breast cancer.
              </a>
            </td>
          <td>
            Hongsen Liu, Mengting Chen, Bo Hong, Ruijin Liang, Lijie Fan, Yun Qian
          </td>
          <td>2025-07-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Steatotic liver disease-related hepatocellular carcinoma (SLD-HCC), a rising global challenge, is characterised by unique tumour microenvironment (TME) adaptations.


OBJECTIVE
This study investigates the immune microenvironment and interactions driving immunosuppression and potential resistance to immunotherapy in SLD-HCC.


DESIGN
We employed single-cell transcriptomics, cytometry by time-of-flight (CyTOF) and two independent spatial transcriptomics platforms to study the TME of 22 SLD-HCC and 31 non-SLD-HCC cases. Findings were further validated using multiplex immunohistochemistry in an independent cohort of 103 patients, an HCC model and an immunotherapy-treated patient cohort to evaluate clinical relevance.


RESULTS
Our findings revealed significant alterations in immune and lipid metabolism pathways, particularly in regulatory T cells (Tregs) and cancer-associated fibroblasts (CAFs), suggesting distinct cellular adaptations to a high-fat TME and general immunosuppression. CyTOF revealed a cold, immunosuppressive TME with reduced CD8+ T cells and increased Tregs. Spatial transcriptomics further highlighted distinct Treg-CAF clusters localised at tumour margins, suggesting a spatially organised immunosuppressive niche. Mechanistically, tumour necrosis factor superfamily member 14 (TNFSF14)-tumour necrosis factor receptor superfamily member 14 (TNFRSF14)-mediated Treg-CAF interaction was identified as a critical driver of immunotherapy resistance in SLD-HCC. Blocking TNFRSF14 in an HCC model fed with a high-fat diet resulted in reduced Tregs, increased active CD8+ and memory CD4+ T cells, and a synergistic effect with anti-programmed cell death protein 1 therapy to enhance antitumour immunity and overcome immunotherapy resistance in SLD-HCC.


CONCLUSION
This study uncovers critical immune and metabolic adaptations in SLD-HCC, identifying TNFSF14-TNFRSF14 signalling as a key driver of immunotherapy resistance. Targeting this signalling axis enhances antitumour immunity and improves immunotherapy efficacy, offering a promising therapeutic strategy for SLD-HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8945e27ac60921d9362c3d051e5dc0848a2a4ff0" target='_blank'>
              Targeting Treg-fibroblast interaction to enhance immunotherapy in steatotic liver disease-related hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Aldo Prawira, Hang Xu, Yu Mei, N. J. M. Nasir, Marie J Y Reolo, Masayuki Otsuka, W. Leow, Mohammad Rahbari, Ziyao Chen, M. Weerasooriya, Liyana Bte Abdullah, Jiawei Wu, S. Hazirah, Martin Wasser, A. Chung, Brian K. P. Goh, Pierce Chow, Salvatore Albani, Joycelyn Lee, T. Lim, Weiwei Zhai, Y. Dan, George Bb Goh, Mathias F Heikenwälder, Yongliang Zhang, Ramanuj Dasgupta, Wai Fei David Tai, Haiyan Liu, Jinmiao Chen, Valerie Chew
          </td>
          <td>2025-07-22</td>
          <td>Gut</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="PLXNA3 is a transmembrane protein essential for axon guidance and synapse formation, gaining attention in cancer research for its roles in cell migration, invasion, and signal transduction. However, its biological significance in breast cancer remains underexplored. Here, bulk RNA sequencing and survival analysis revealed that elevated PLXNA3 levels were correlated with increased malignancy and poorer survival in breast cancer. Single-cell RNA sequencing demonstrated that PLXNA3 was predominantly enriched in tumor cell clusters within the tumor immune microenvironment, rather than in immune cell clusters. In vitro experiments further demonstrated that knockdown and overexpression of PLXNA3 significantly altered the proliferative, invasive, and migratory capacities of breast cancer cells. Additionally, the overexpression of PLXNA3 was negatively correlated with immune activation status and served as a predictor of poor response to anti-PD1 immunotherapy. Our findings suggested that high expression of PLXNA3 is associated with poor prognosis in breast cancer and plays a crucial role in cancer immunity, making it a promising novel target for intervention in breast cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13843-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f126a6deb0547875700a586994249ce5224fe17" target='_blank'>
              Comprehensive analysis of the PLXNA3 gene on prognosis and immune characteristics in breast cancer
              </a>
            </td>
          <td>
            Wen Sun, Yifan Xu, Qi Qi, Weiya Zhang, H. Pan, Guangshun Sun, Shui Wang, Wenbin Zhou
          </td>
          <td>2025-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal cancers, with an average survival time of only six months following diagnosis, even with currently available therapies. Thus, PDAC represents a significant therapeutic challenge, necessitating a deeper understanding of its biology and tumor microenvironment (TME) to develop more effective treatments and improve patient outcomes. Here, we report that the expression of Eukaryotic Elongation Factor-2 Kinase (eEF2K) is associated with shorter patient survival and demonstrate that eEF2K signaling is critical for the PDAC tumor growth and regulated by the TME. Furthermore, in vivo targeted genetic inhibition of eEF2K suppressed tumor growth in two different PDAC mouse models, reduced tumor-associated macrophages (TAMs), and induced marked apoptosis in tumor tissues without any signs of toxicity. Our data suggest that eEF2K knockdown diminishes the activity of the AXL receptor tyrosine kinase and reduces the expression of macrophage-derived factors, such as Monocyte Chemoattractant Protein-1 (MCP1), along with the Gas6/AXL signaling pathway in PDAC cells. Additionally, analysis of the NCI-TCGA PDAC patient database further showed that eEF2K expression, in the presence of TAM markers, correlates with even shorter patient survival. TAM-released factors, such as MCP1, Gas6, and exosomes, induce eEF2K expression in PDAC cells, as well as the activity of AXL, SRC, VEGF, Snail, and MMP2, contributing to epithelial-to-mesenchymal transition (EMT), invasion, metastasis, and angiogenesis. In conclusion, our findings reveal for the first time that eEF2K is a critical oncogenic driver of PDAC tumor growth and thus targeting eEF2K represents a promising and novel therapeutic strategy for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca2a458a943c8611c08480c527e7133171db725" target='_blank'>
              eEF2K is a poor prognostic factor and novel molecular target in pancreatic cancer: regulating tumor growth and progression via the tumor microenvironment
              </a>
            </td>
          <td>
            Didem Karakaş, Ahmed Ashour, H. Mokhlis, N. Kahraman, R. Bayraktar, S. Dilmac, N. Kabil, M. Erdoğan, E. Dere, E. Ulukaya, B. Ozpolat
          </td>
          <td>2025-07-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Lung cancer remains a leading cause of cancer-related mortality due to its capacity for silent metastasis and the significant challenges in achieving effective treatment. Currently, targeted therapies and chemotherapies are the primary options for advanced or inoperable lung cancer; however, their efficacy is often undermined by the cancer's ability to develop resistance through both genetic and non-genetic mechanisms. This review explores recent advances in understanding metabolic reprogramming in non-small cell lung cancer (NSCLC), focusing on its critical role in cancer progression. NSCLC cells exhibit heterogeneous activation of metabolic pathways influenced by their oncogenic mutations. Notably, their metabolic phenotypes evolve in response to environmental stressors and therapeutic pressures. Moreover, NSCLC cells engage in metabolic crosstalk with their microenvironment to enhance survival, leveraging distinct metabolic adaptations at both primary and metastatic sites. Despite extensive preclinical studies evaluating novel therapeutic strategies targeting these metabolic pathways, many have failed in clinical trials due to severe adverse effects. This is because the targeted pathways are crucial not only for cancer cells but also for normal cellular functions. Future research must prioritize approaches that selectively disrupt cancer-specific metabolic regulation to improve therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a854e0477bbe5fbd82c8f4b77d0195460cff4ca2" target='_blank'>
              The Emerging Roles of Metabolic Reprogramming in Non-Small Cell Lung Cancer Progression.
              </a>
            </td>
          <td>
            Beatriz P Peixoto, Rebecca A Clague, Joshua P Reddy, Hiromi I Wettersten
          </td>
          <td>2025-07-08</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intrinsic radiotherapy resistance remains a persistent challenge to the treatment of cervical cancer (CC), as the underlying mechanisms remain largely elusive. In this study, proteomic profiling was combined with in vitro and in vivo experiments to clarify the potential mechanisms driving radiotherapy resistance. Initially, Orbitrap Astral-based proteomic profiling of clinical specimens revealed markedly decreased UBE2C expression in radioresistant tumors. Subsequent functional validation with cellular and animal models of UBE2C deficiency revealed that UBE2C is critical for preserving radiosensitivity and its knockdown directly induced the radioresistance of CC. Furthermore, proteomic analysis of UBE2C-knockdown CC cell lines, combined with multilayer intersection analysis of the clinical proteomic data, revealed that the glycolysis-associated protein F264 is potentially regulated by UBE2C. Mechanistically, F264 regulates tumor cell dormancy to evade radiation-induced damage. Finally, immunohistochemical analysis of an independent clinical cohort confirmed aberrant F264 protein expression patterns. Therefore, we hypothesize that altered UBE2C protein levels during cervical carcinogenesis modulate F264 expression, which partially activates tumor dormancy mechanisms, ultimately contributing to the intrinsic radiotherapy resistance observed in a subset of CC patients during initial treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66da9933252c89bbef332f34d82e18fbe63053e1" target='_blank'>
              Deficiency of the UBE2C/F264 Axis Underlies Intrinsic Radioresistance in Cervical Carcinoma.
              </a>
            </td>
          <td>
            Maowei Ni, Danying Wan, Mengni Li, Junzhou Wu, Juan Ni, Hanmei Lou, Shengjie Zhang, Z. Yin
          </td>
          <td>2025-08-01</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Meningioma is the most prevalent primary brain tumor with extensive intra-tumoral heterogeneity and high recurrence rates, particularly in high-grade meningiomas. Despite advancements in understanding the molecular underpinnings of meningiomas, the longitudinal evolutionary trajectory and cellular diversity of recurrent tumors remain elusive. In this study, we perform single-nuclei sequencing of matched primary and recurrent meningiomas to explore the dynamic transcriptional heterogeneity and evolutionary trajectory of meningioma tumor cells, as well as their molecular interactions with tumor-associated immune cells that shape the complex milieu of the meningioma microenvironment. Our findings reveal that both primary and recurrent meningiomas constitute diverse cellular compositions and hierarchies, where recurrent tumor cells are characterized by enrichments of cell cycle activities and proliferative kinetics. Integrative RNA velocity and latent time uncover divergent transcriptional trajectories in recurrent tumors, demonstrating multidirectional transitions with the dominance of COL6A3, which confers higher risk vulnerability and treatment resistance. Tumor microenvironment analysis further reveals enrichments of COL6A3-mediated interactions between immunosuppressive macrophages and tumor cells in recurrent meningiomas. Collectively, these results provide profound insights into the complex evolutionary process of meningiomas and suggest potential therapeutic strategies for the treatment of recurrent tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46967d3ec0f9b8d5fcb467d9f78f81fb62b39f73" target='_blank'>
              Single-cell analysis reveals a longitudinal trajectory of meningioma evolution and heterogeneity
              </a>
            </td>
          <td>
            Ji Yoon Lee, Eun Jung Lee, Bo Yeon Seo, Jiwon Kim, Youjin Song, Dayoung Lee, Namsung Moon, Harim Koo, Chul-Kee Park, Min-Sung Kim, Serk In Park, Do-Hyun Nam, D. Kong, Jason K. Sa
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Liver metastasis remains a major cause of death in colorectal cancer (CRC). Interactions between tumor cells and components of the tumor microenvironment (TME) are integral to the progression of cancer cell migration and metastatic dissemination. Investigating these cellular dynamics is essential for identifying actionable therapeutic targets. Methods This study employed bioinformatics analysis, IHC, WB, and murine in vivo imaging to delineate the role of SPOCD1 in CRC. Additional in vitro co-culture systems, immunofluorescence, and RNA-seq were utilized to determine how SPOCD1 modulated epithelial-mesenchymal transition (EMT) through cancer-associated fibroblasts (CAFs). Mechanistic insight into SPOCD1-mediated transcriptional regulation of LAMA4 was obtained via dual-luciferase reporter assays, ChIP-qPCR, and Co-IP. Results SPOCD1 was found to be markedly upregulated in CRC cells and exhibited potent pro-metastatic activity in vivo. Integration of external datasets with experimental validation revealed a strong correlation between SPOCD1 expression and CAFs infiltration in the TME. Further analyses demonstrated that SPOCD1 enhanced CXCL12 expression in CAFs via upregulation of LAMA4, enabling CXCL12 to engage CXCR4 on CRC cells and activate EMT signaling, thereby driving metastasis. This signaling axis was effectively interrupted by CXCR4 inhibitors. Mechanistically, SPOCD1 promoted LAMA4 expression through DNMT1 recruitment, facilitating DNA methylation-dependent transcriptional regulation. Conclusion This study delineates a SPOCD1-centered interaction network between CRC cells and CAFs in colorectal cancer liver metastasis (CRLM), offering novel molecular candidates for therapeutic intervention in CRLM. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06863-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d8174ccdaba2a7e9f4b9613ed271a53eaeae82" target='_blank'>
              Cancer cell SPOCD1 promotes colorectal cancer liver metastasis by activating the CXCL12/CXCR4 signaling pathway in cancer-associated fibroblasts
              </a>
            </td>
          <td>
            Geng Peng, Lin Zhong, Lina Luo, Yongle Ju, Yan Lu, L. Ng, D. Foo, Fu Cheng, Luji Huang, M. Ouyang
          </td>
          <td>2025-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d00a92d8c57a156fa42fb869149b3fe4641e0fb9" target='_blank'>
              Single-cell and spatial transcriptomics of stricturing Crohn's disease highlights a fibrosis-associated network.
              </a>
            </td>
          <td>
            Lingjia Kong, Sathish Subramanian, Åsa Segerstolpe, Vy Tran, Angela Shih, Grace T Carter, H. Kunitake, Shaina W. Twardus, Jasmine Li, Shivam Gandhi, M. E. Kaper, Christy Cauley, Eric J Chen, C. Porter, T. Delorey, L. Bordeianou, Rocco Ricciardi, Ashwin N. Ananthakrishnan, Helena Lau, Daniel B. Graham, Richard Hodin, Jacques Deguine, Christopher S Smillie, Ramnik J. Xavier
          </td>
          <td>2025-06-25</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793beb6dae04857b331a1472ade96cbcdfbcb83e" target='_blank'>
              Integrated protocol for single-cell RNA sequencing, organoid culture, and RNA-protein spatial analysis in gastroesophageal tissues
              </a>
            </td>
          <td>
            Naveen Kumar, Pon Ganish Prakash, Rajendra Kumar Gurumurthy, Cindrilla Chumduri
          </td>
          <td>2025-07-07</td>
          <td>BMC Methods</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tongue Squamous Cell Carcinoma (TSCC) represents a significant subtype of malignant oral cancer, characterized by a heterogeneous tumor microenvironment (TME). The tongue, a complex muscular organ, naturally presents an initial microenvironment that is largely inhospitable to the initiation and progression of TSCC. However, advanced-stage TSCC exhibits a pronounced accumulation of cancer-associated fibroblasts (CAFs), indicative of a drastic microenvironmental transformation. In this study, through comprehensive analysis combining cell model assessments and single-cell RNA sequencing data from a 4-NQO-induced TSCC mouse model, along with a lineage tracing-in-transplant assay, we elucidate and confirm the process of transdifferentiation whereby tongue muscle cells (TMCs) convert into CAFs in the TSCC context. Furthermore, we demonstrate that targeting TSCC with an IL-17a inhibitor offers a viable strategy to inhibit the reprogramming of TMCs into CAFs. Additionally, our research also identifies four critical marker genes involved in the transdifferentiation of TMCs to CAFs, Thbs1, Crabp1, Ifi205, and Cxcl5. Collectively, these findings delineate a mechanism through which TSCC cells induce the transdifferentiation of TMCs into CAFs, thereby transforming the cancer-suppressive microenvironment of the tongue into one that is conducive to TSCC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/666df9c3fc7fca0534d4dd3d77f31ac2e7399d3f" target='_blank'>
              Transdifferentiation of tongue muscle cells into cancer-associated fibroblasts in response to tongue squamous cell carcinoma
              </a>
            </td>
          <td>
            Weifan Lin, Wu-hua Huang, Shiqiang Mei, Xiangwan Lu, Junheng Zheng, Hua Wang, Lingling Zheng, Yan Zhang
          </td>
          <td>2025-07-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Transcriptome profiling by RNA sequencing (RNAseq) can provide insightful information on the molecular processes underlying disease development and progression. Although fresh tissue represents the preferred source material for RNAseq, here, we investigated the feasibility of applying RNAseq analysis to single 10 μm thick formalin-fixed and paraffin-embedded (FFPE) lung slides from the lungs of control and bleomycin (BLM)-treated mice. This approach aims at providing spatial-oriented transcriptomic data, that can be integrated with in vivo and ex vivo readouts obtained on the same sample, as a way to enhance the mechanistic information and biomarker/target discovery potential of preclinical models of fibrotic lung diseases. Methods RNAseq analysis was conducted on individual FFPE slides from the lungs of both controls and BLM-treated mice. The results were initially validated by comparison with publicly available bulk data from fresh-frozen (FF) mouse tissues, both untreated and BLM-treated, as well as human idiopathic pulmonary fibrosis (IPF) biopsies. Unsupervised cluster analysis was performed on Differentially Expressed Genes (DEGs) distinguishing untreated and BLM-treated fibrotic lung samples. For each sample, Pearson correlation analysis was used to compare expression levels of individual gene clusters with Ashcroft Scores and aeration compartments quantitatively assessed on the matched 2D micro-CT coronal slice. Results Over 90% of annotated genes within the FFPE dataset were shared with gene signatures retrieved from FF bulk datasets. Differentially modulated gene clusters were mainly found to be associated with extracellular matrix (ECM) organization, tissue remodeling, and inflammatory response pathways. For each sample, expression levels of individual gene clusters were highly correlated with 2D histology readouts and aeration compartments determined on matched 2D coronal slices by micro-CT imaging. Conclusions FFPE lung tissue represents a valuable alternative to fresh tissue for RNAseq analysis, allowing to achieve a more precise, spatially oriented picture of pulmonary disease development. This approach is thus instrumental to a better characterization of the molecular changes associated to each sample. It can also contribute to a more informed interpretation of histology and micro-CT imaging data, paving the way to the identification of translationally relevant biomarkers as well as novel candidate targets for the development of more effective therapeutic interventions. Supplementary Information The online version contains supplementary material available at 10.1186/s12931-025-03300-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca3f0c49a6eeb93406bc59871126d70b0cd7e03" target='_blank'>
              Spatial transcriptomic and morpho-functional information derived from single mouse FFPE slides allows in-depth fingerprinting of lung fibrosis
              </a>
            </td>
          <td>
            Erica Ferrini, Costanza Bonfini, Giovanna Marchese, Martina Buccardi, M. Zoboli, P. Faccioli, N. Sverzellati, G. Villetti, Simone Ottonello, Maria Ravo, F. F. Stellari
          </td>
          <td>2025-07-02</td>
          <td>Respiratory Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Chemotherapy is an integral part of the clinical management of triple negative breast cancer (TNBC), however, development of chemoresistance occurs frequently. Tumor sensitivity to treatment is known to be strongly influenced by the immune microenvironment, signifying the predictive potential of immune alterations. How tumor cells that acquire resistance may subsequently modulate the immune microenvironment it is still not well described. Here, we investigated immunomodulation in the context of acquired chemoresistance in TNBC, focusing on the role of the secretome. Methods Bulk RNA sequencing and multiplex cytokine profiling were performed on paclitaxel-resistant and -sensitive isogenic variants of TNBC cells to reveal resistance-associated secretome alterations. The immunomodulatory influence of the tumor cell secretome was investigated by exploring its effect on monocytes, macrophages (MΦs) and T cells derived from healthy blood donors. The influence on the immune cell phenotype and activity was evaluated by measuring molecular markers and performing functional assays. To validate the clinical relevance, we utilized longitudinal -omics data from breast cancer patients refractory to standard chemotherapy in the NeoAva clinical trial. CIBERSORT was applied to transcriptomics data to infer MΦ and T cell abundance in individual tumors upon treatment. To evaluate their association with the secretome profiles, patient-matched serum cytokine data were used. Results The acquisition of chemoresistance was accompanied by enhanced secretion of cytokines. Subsequently, the resistant cell secretome affected the abundance, phenotype and activity of immune cells. Specifically, it potentiated the recruitment of monocytes, facilitated the polarization of MΦs towards the immunosuppressive M2-like phenotype, and attenuated the activation of CD8+ T cells. Data from the NeoAva clinical cohort validated the enrichment of M2 MΦs and/or the depletion of M1 MΦs after treatment in the majority of residual tumors. The MΦ-associated changes counteracted CD8+ T cell abundance and were partially associated with the cytokine-enriched secretome. Conclusion Development of chemoresistance in BC is associated with alterations in the tumor secretome, which impairs immune activation and facilitates immunosuppression. Knowledge on the immune microenvironment in residual tumors after standard chemotherapy could aid in selecting rational treatment options for this group of patients. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02082-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b23b4587f443f9967a9b8d5e80c45be19dc3ce" target='_blank'>
              Chemoresistant tumor cell secretome potentiates immune suppression in triple negative breast cancer
              </a>
            </td>
          <td>
            Eleni Skourti, Kotryna Seip, N. Mensali, S. Jabeen, S. Juell, Inger Øynebråten, S. Pettersen, O. Engebraaten, Alexandre Corthay, E. Inderberg, Helmut Dolznig, G. Mælandsmo, E. V. Egeland, L. Prasmickaite
          </td>
          <td>2025-07-14</td>
          <td>Breast Cancer Research : BCR</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="Background Parkinson’s disease (PD) is a progressive neurodegenerative disorder marked by motor dysfunction and dopaminergic neuron loss. Although its genetic and molecular underpinnings have been increasingly studied, the pathways driving PD progression remain unclear. The integrated stress response (ISR), a conserved mechanism activated by cellular stress, has been linked to several neurological diseases, but its role in PD and the key ISR-related genes involved are still poorly understood. Materials and Methods We used publicly available transcriptomic data from GEO, including single-cell RNA sequencing and spatial transcriptomics, to identify ISR-related genes involved in PD progression. ISR scores were compared across brain cell types, and differentially expressed genes in microglia were further screened using Lasso regression and random forest algorithms. Enrichment analyses (GSEA and GSVA) revealed their involvement in immune-related pathways. CIBERSORT was applied to assess immune cell infiltration, while spatial transcriptomics mapped the regional expression of key genes. Finally, DDIT4 expression was validated in PD cell and mouse models. Results We identified four key ISR-related genes (DDIT4, GNA13, HSPA1B, and SLC7A5) that were differentially expressed in PD microglia. Functional enrichment analysis revealed that these genes were predominantly involved in immune-related signaling pathways, including JAK-STAT, NF-κB, and Notch, suggesting their potential role in regulating neuroinflammation. Spatial transcriptomics revealed distinct regional expression patterns of these ISR-related genes across brain tissues. In vitro and in vivo experiments confirmed the upregulation of DDIT4 in PD models, and its silencing alleviated neurotoxicity and reduced α-synuclein aggregation, highlighting its potential role in PD pathogenesis. Conclusion This study provides new insights into the molecular mechanisms of PD and highlights DDIT4 as a promising therapeutic target. Its regulatory role in immune signaling and cellular stress pathways may offer novel avenues for clinical intervention and personalized treatment strategies in PD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64254e7f6c0bdf74b643b97c54382169e29a9cf5" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Identifies ISR-Related Genes Driving Immune Regulation in Parkinson’s Disease
              </a>
            </td>
          <td>
            Hua Jiang, Xiaotian Zhang, Shengyu Feng, Wei Feng
          </td>
          <td>2025-07-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Infectious diseases remain a leading cause of mortality worldwide. The pathogenesis that comprises infection, focal inflammation, and immuno-response, typically occurs in one or multiple organs or tissues. Analysis of the molecular composition of affected tissues with their spatial context is pivotal to elucidate the underlying disease mechanisms and to develop accurate diagnostic strategies. In recent years, mass spectrometry imaging (MSI) technology has achieved significant advancements and has emerged as an powerful tool for tissue-based molecular exploration with high molecular specificity and spatial resolution. Although MSI has been rapidly adopted in numerous branches of biomedical research, its application in the field of infectious diseases research is still in its early stages. With this in mind, this review aims to familiarize infectious disease researchers with the advantages and diverse applications of MSI. Additionally, we delineate several existing technical challenges, application pitfalls, and the potential solutions to overcome these challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9a1612ce608f4a73a5eafeadf22ccb4d8e58376" target='_blank'>
              Mass Spectrometry Imaging: Revolutionizing Molecular Insights in Infectious Diseases Research
              </a>
            </td>
          <td>
            Minmin Zhang, Xiao Wang, Xiaoling Su, Aidiya Yimamu, Lanjuan Li, Zeyu Sun
          </td>
          <td>2025-06-30</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Twenty percent of follicular lymphoma (FL) patients relapse early with poor outcomes; however, the molecular mechanisms underlying this aggressive behavior are unknown. Using a multi-omics approach, we show that FL patients with elevated IRF4 expression (IRF4hi) have increased transformation risk, dysregulated immune signaling, and a suppressive tumor microenvironment. Loss- and gain-of-function experiments in IRF4hi lymphoma cells, along with chromatin profiling, demonstrate that IRF4 impairs their interaction with T cells by repressing antigen presentation and co-receptor gene modules, while promoting the expression of cytokines that antagonize TFH and Treg functions. Additionally, IRF4 rewires tumor metabolism which restricts glucose availability to immune cells. Silencing of IRF4 inhibits tumor cell growth and restores immune surveillance mechanisms, thus representing a promising target for therapy. Our data suggest that IRF4hi lymphoma cells co-opt a developmental mechanism used to exit the germinal center response in promoting a more aggressive cancer via engagement of multiple immune-evasive mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eac0e65fc642810489570608cecc61617b03f9" target='_blank'>
              IRF4 promotes immune evasion and shapes the tumor microenvironment in Follicular Lymphoma.
              </a>
            </td>
          <td>
            Surendra Dasari, K. Wenzl, Geoffrey M. Nelson, Emmanuel Contreras Guzman, Zhiquan Wang, L. Chartier, Zhi-Zhang Yang, J. Villasboas, Joshua Olson, P. Mukherjee, Vaishali Bhardwaj, Xinyi Tang, B. Negaard, Johannes L Zakrzewski, Rebecca L King, Sarah Huet, B. Tesson, Matt J. Maurer, Franck Morschhauser, Grzegorz S. Nowakowski, Karen Adelman, Harinder Singh, L. Pasqualucci, M. Shlomchik, A. Novak, Stephen M Ansell, P. Mondello
          </td>
          <td>2025-07-16</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Background Chemoresistance and recurrence following treatment are the greatest impediments to the prognosis of glioblastoma (GBM). Increasing evidence indicates that cancer-associated fibroblasts (CAFs) play a significant role in the progression of glioblastoma. Nevertheless, the role and source of CAFs in recurrent and chemotherapy-resistant GBMs still remain ambiguous. Methods Spatial transcriptome (ST) sequencing was conducted on the tissue microarray encompassing primary and recurrent glioma samples in order to characterize the cellular composition. Subsequently, the infiltration of CAFs in our formerly established in vivo temozolomide (TMZ)-resistant model was inspected through immunohistochemical staining. Additionally, we carried out RNA-seq and label-free quantitation (LFQ) proteomics on HCMECs co-cultured with TMZ-sensitive (TMZ-S) or TMZ-resistant (TMZ-R) cells to explore the mechanism. Results This investigation revealed that CAFs and astrocytes are enriched in recurrent GBM, and this phenotype is associated with the expression of extracellular matrix (ECM) proteins associated with COL1A1 and FN1 deposition. Further investigations revealed that tenascin-C (TNC) and filamin C (FLNC), which potentially mediate endothelial-to-mesenchymal transition (EndMT), are the predominant factors that induce the deposition of ECM proteins in the resistance-promoting microenvironment. Additionally, the natural product punicalin (PNC) was found to downregulate EndMT-related proteins, multidrug resistance-associated membrane proteins, and collagen-related proteins by targeting TNC and FLNC, thereby increasing the susceptibility of temozolomide (TMZ)-resistant cells to chemotherapeutic agents both in vitro and in vivo. Conclusion These discoveries indicate that TNC and FLNC induced EndMT was a key resource of CAFs and targeting TNC and FLNC to inhibit EndMT and the collagen pathway is a promising tactic for reversing drug resistance in tumours. The development of combined chemotherapeutic strategies based on the features of tumour microenvironment endothelial cells and ECM deposition has high potential clinical value in increasing the efficacy of tumour treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06743-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90f87d542212bc590894a08468baa2631e0e5527" target='_blank'>
              Spatial transcriptomics and multi-omics reveal relapse and resistance mechanisms of EndMT-derived CAFs mediated by TNC and FLNC in glioblastoma
              </a>
            </td>
          <td>
            Li Ji, Die Xia, Yu Zhou, Yaling Hu, Zhenkun Yang, Ying Yin, Jingjing Wang, Bo Zhang, Lingli Gong, Koukou Li, Jian Zou, Mei Wang
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="FOXA1 is altered in 10 to 40% of prostate cancers, yet its oncogenic mechanisms remain uncharacterized in vivo. We developed knock-in mouse models representing distinct classes of FOXA1 mutations. Histopathological and multi-omic analyses of prostate tissues and organoids revealed that Class 1 mutations, in conjunction with p53 inactivation, drive androgen-dependent adenocarcinomas through co-activation of mTORC1/2 and oncogenic AR signaling stemming from chimeric AR-half enhancers. In contrast, Class 2 mutations induce intra-luminal plasticity by reprogramming differentiated luminal cells into a progenitor-like state through activation of KLF5 and AP-1 neo-enhancer circuitries, which enables enhanced survival and proliferation even under castrate androgen levels. Our findings establish FOXA1 as a multifaceted oncogene, with distinct mutational classes divergently evolving to drive prostate tumorigenesis or therapy-resistant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877a995b94079d1a62d50951659ac85f4910e75e" target='_blank'>
              Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity.
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, J. Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin P. Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2025-06-26</td>
          <td>Science</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c28d4f03f5665c2fc1d48e04f627236eb7626fd" target='_blank'>
              Transcriptomic entropy reveals tissue-specific patterns in aging and predicts cancer progression
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, José Pedro Castro, P. Galante
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The contribution of the nervous system to the tumor microenvironment and the importance of neural invasion as a route for cancer dissemination are being increasingly recognized. Interactions of cancer cells with neurons can promote their invasion around and into nerves, a feature of many cancers with poor clinical outcomes. In vitro models to study reciprocal interactions between neurons and cancer cells provide valuable tools for understanding cancer spread and identifying approaches to mitigate it. Here, we describe a protocol for murine dorsal root ganglia (DRG) isolation and the establishment of both whole mount and dissociated monolayer cultures that can be used to visualize neuron morphology and neurite outgrowth over time. Whole DRGs mounted in Matrigel preserve nerve architecture and responses to stimuli in a heterogeneous environment more similar to the in vivo nerve, while dissociated nerve cultures allow assessment of direct cell-cell interactions more closely. Once DRG cultures are established, cancer cells can be added to generate co-cultures that can be used to visualize changes in neurite outgrowth and nerve morphology in response to cancer cells. Growth or motility of cancer cells in response to nerve-derived signals over time or under conditions of growth stimulation or inhibition can be assessed, as well as visualizing the effects of direct contact between cancer cells and nerve extensions. As both co-culture models can be generated simultaneously, this protocol provides a more comprehensive view of the impact of cancer-neuron interactions and facilitates comparisons of treatment conditions and integration of information from the cellular level and whole ganglia. This protocol will facilitate the study of nerve-tumor interactions and can be used for a wide range of applications, including studies of cell signaling, drug screening, or study of the heterogeneity of the tumor-nerve environment and the mechanisms of tumor dissemination along nerves.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed277735f673ca49473d2865d2f8280ae46af9b" target='_blank'>
              Establishing In Vitro Models of Dorsal Root Ganglia Culture: Complementary Approaches for Investigating Cancer-Nerve Crosstalk.
              </a>
            </td>
          <td>
            Larissa C B Oliveira, Erica R Pereira, Brandy D. Hyndman, Bryanna Thomson, Juliana M Serpeloni, Lois M Mulligan
          </td>
          <td>2025-07-11</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 The intricate relationship between inflammation and cancer progression has been well-documented, with chronic inflammation playing a pivotal role in tumor development and drug resistance. This project investigates how cancer-intrinsic inflammation influences drug response, aiming to identify key regulators that could serve as therapeutic targets to overcome drug resistance in colorectal cancer (CRC). We hypothesize that distinct inflammation types within cancer cells modulate their response to chemotherapy, potentially driving resistance. Our study is structured around two specific aims: First, we investigate the inflammation-type-dependent drug response using single-cell barcoded colon cancer cells. Our preliminary studies demonstrated that CRC cells in an inflamed microenvironment, particularly those with STING pathway activation, exhibit distinct responses to 5-FU treatment compared to non-inflamed cells. This finding underscores the need to consider inflammation as a critical factor in cancer treatment strategies. By characterizing the response of inflamed and non-inflamed cells to various drug treatments at single-cell resolution, we aim to identify the specific inflammation-driven mechanisms that contribute to drug resistance. Second, we focus on the characterization and validation of key regulators identified in inflamed CRC cells. We will validate candidate genes through functional assays, including gene knockdown experiments, to uncover the molecular pathways through which inflammation modulates drug response. By linking the distinct inflammatory states within cancer cells to drug resistance, we aim to identify novel biomarkers and therapeutic targets that could enhance the effectiveness of existing treatments. Our research has the potential to significantly impact the treatment of CRC by providing a deeper understanding of how inflammation drives drug resistance, ultimately guiding the development of more effective, personalized therapies for patients with inflamed tumors. This approach addresses the urgent need for targeted strategies to overcome the challenges posed by inflammation-associated drug resistance in cancer therapy.



 Kerem Fidan, Semih Can Akincilar, Tim Stuart, Vinay Tergaonkar. Deciphering Inflammation-Driven Drug Resistance Mechanisms in Colorectal Cancer through Single-Cell Barcoding [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P71.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb480f230d054f0fd849886c3b02be9cc7e768af" target='_blank'>
              Abstract P71: Deciphering Inflammation-Driven Drug Resistance Mechanisms in Colorectal Cancer through Single-Cell Barcoding
              </a>
            </td>
          <td>
            Kerem Fidan, S. C. Akıncılar, Tim Stuart, Vinay Tergaonkar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer represents a significant global health hazard marked by elevated morbidity and mortality rates. Furthermore, the majority of tumor therapies encounter challenges, including metastasis, recurrence, and drug resistance. Consequently, it is essential to identify a specific and efficient tumor treatment approach. In recent years, the ongoing investigation and comprehension of tumors have led to significant attention towards cancer stem cells (CSCs). CSCs can facilitate tumor progression via self-renewal, differentiation capabilities, and multidrug resistance. Their function as a fundamental contributor to tumor heterogeneity, drug resistance, recurrence, and metastasis has emerged as a significant focus in cancer therapy research. In recent years, microRNAs (miRNAs) have been identified as crucial post-transcriptional regulators in biological processes, including chemosensitivity, self-renewal, apoptosis, invasion, and metastasis of cancer stem cells (CSCs). This paper systematically reviews the molecular mechanisms through which miRNAs influence the characteristics of cancer stem cells by targeting essential genes (e.g., SOX2, EGFR, c-Met) and modulating signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and PI3K/Akt. Furthermore, we investigated the viability of miRNAs as non-invasive biomarkers for cancer diagnosis and prognosis evaluation, examined the similarities and attributes of pivotal miRNAs in modulating cancer stem cell functionality, and deliberated on therapeutic approaches stemming from miRNA regulation of cancer stem cell activity. We anticipate that this research will yield novel insights into targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c085b3dd2e8cad808f86e5932aff3c84c64bec4" target='_blank'>
              Regulatory Functions of microRNAs in Cancer Stem Cells: Mechanism, Facts, and Perspectives
              </a>
            </td>
          <td>
            Xingmei Mao, Sixue Peng, Yan Lu, Linjiang Song
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND
Primary central nervous system lymphoma (PCNSL) is a highly aggressive subtype of non-Hodgkin lymphoma that is confined to the central nervous system. PCNSL is associated with a poor 5-year survival rate of 30-40%, partly due to a high recurrence rate of 60%. A comprehensive understanding of the molecular signatures and mechanisms underlying drug resistance in PCNSL is crucial, as it has significant implications for therapeutic strategies.


METHODS
A cohort of 56 newly diagnosed PCNSL patients was generated. The tumor specimens were obtained by biopsy. A combined approach, including single-cell transcriptomics and B-cell receptor sequencing, transcriptome-informed multiplex immunohistochemistry, and ex-vivo drug response assays, was applied to reveal the transcriptional and immune microenvironment landscape of PCNSL.


RESULTS
We identify four main B cell subtypes, each characterized by distinct transcriptomic profiles, from malignant B cells in PCNSL. Among these subtypes, plasmablast-like lymphoma cells (PBLCs), characterized by upregulated genes typically associated with plasma cell differentiation, comprise 1.3%-8.1% of malignant B cells and correlate with poor prognosis in PCNSL. PBLCs exhibit reduced expression of CD20, Bruton tyrosine kinase, and FAS, resulting in drug resistance and immune evasion. In addition, PBLCs elevate the expression of critical transcription factors, including XBP1 and PRDM1, to maintain their phenotype. Clinical drugs, such as Lenalidomide, demonstrate potential cytotoxic effects on PBLCs.


CONCLUSIONS
These findings highlight PBLCs as a distinct subtype of malignant B cells that plays a critical role in the multidrug resistance of PCNSL and reveal the molecular signature of PBLCs that can be targeted for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17a9a2cca6ca455e6dde63a6892673c09fa8c29d" target='_blank'>
              Plasmablast-like lymphoma cells as a distinct subpopulation confers multidrug resistance in PCNSL.
              </a>
            </td>
          <td>
            Feng Liang, Xiaotao Zhang, An Ping, Yichen Gu, Ziyang Jin, Huaming Li, Lingxiao Lu, Fan Yang, Junjie Wang, Jianan Lu, Feng Zhou, Jinfang Xu, Jingsen Chen, Junming Zhu, Fengqiang Liu, Jianmin Zhang, Ligen Shi
          </td>
          <td>2025-06-21</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a heterogeneous illness influenced by intricate tumor-immune interactions and characterized by a dismal prognosis. Macrophage-mediated immunological signaling facilitates tumor proliferation and may associate inflammation in the tumor microenvironment (TME) of CRC with negative outcomes. Notwithstanding therapeutic advancements, resistance to treatment remains a significant obstacle. scRNA-seq offers comprehensive insights into the immune signaling network and immunological dynamics inside the CRC’s TME.We integrated scRNA-seq data from GEO with extensive RNA-seq data from TCGA to elucidate immunological signaling and dynamic cellular variation in the TME of CRC. The analyses encompassed quality control via Seurat, InferCNV, Monocle, CellChat, and SCENIC, differential gene expression, inference of copy number variation (CNV), pseudo time trajectories, and intercellular communication. Prognostic modeling was conducted using Cox regression and LASSO. Immune infiltration and drug sensitivity were evaluated by CIBERSORT, ESTIMATE, xCell, TIDE, and pRRophetic. Functional validation encompassed siRNA knockdown, qRT-PCR, Western blot analysis, and cellular assessment in CRC cell lines.We discovered four categories of tumor cells exhibiting variations in cell cycle, stemness, and differentiation. The MKI67+ subpopulation exhibited a heightened dynamic cell state and engaged with macrophages via the MIF-(CD74+CD44) axis to facilitate immunological signaling. HMGA1 is a crucial transcription factor in this fraction, and its knockdown impedes CRC cell proliferation, motility, and invasion. The cancer model utilizing the MKI67+ TCs subpopulation (MTRS) successfully classified patient survival and linked with immune infiltration patterns and medication responses. Enrichment analysis revealed tumor-promoting and immunological signaling networks. Correlation scores suggest that this subpopulation may be linked to inflammation and immunosuppression inside the TME.Our research indicates that the C2 MKI67+ TCs subpopulation is a key driver of immune signal transduction in CRC TME, which may induce inflammatory responses through interaction with macrophages, thereby leading to adverse consequences such as cardiac dysfunction. HMGA1 represents a viable target for immunotherapy, and our cancer model derived from this subpopulation offers prognostic significance and direction for immunotherapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b53660e6d491a7fc5588044fc19d5176db5aae" target='_blank'>
              Single-cell technology reveals the crosstalk between tumor cells and immune cells: driving immune signal transduction and inflammation-mediated cardiac dysfunction in the tumor microenvironment of colorectal cancer
              </a>
            </td>
          <td>
            Wenyang Nie, Wangzheqi Zhang, Zhikai Xiahou, Yuxuan Meng, Yuhang Liu, Jingwen Zhang, Zhen Wang, Yong Wang
          </td>
          <td>2025-08-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Allogeneic hematopoietic stem cell transplantation (allo-HSCT) is a critical therapeutic strategy for acute myeloid leukemia (AML). However, relapse remains a major challenge, with limited effective options for managing post-transplant recurrence. Tumor cells escape immune surveillance by establishing a complex, immunosuppressive tumor microenvironment (TME), which plays a central role in immune evasion and disease relapse. Understanding the TME’s impact on immune cells and their functional alterations is therefore of significant clinical importance. Methods Utilizing a non-irradiated AML mouse model and bioinformatic analysis, we systematically analyzed T-cell dynamics during AML progression and identified abnormal metabolic activation in CD8+ T cells at early disease stages, which correlated with leukemia cell immune evasion. Result We found that when major histocompatibility complex (MHC) molecules on AML cells were haplomatched with immune cells, CD8+ T cells exhibited robust cytokine and cytotoxic activation, retaining immune profiles similar to those of normal T cells. However, under the same tumor burden, T cells, particularly CD8+ T cells, showed dramatic metabolic gene activation in MHC-matched leukemic mice. This metabolic activation led to rapid exhaustion of CD8+ T cells, accelerating leukemia progression. Based on these activated metabolic genes, we established a metabolism-associated genes (MAGs) scoring system. Validation using public datasets revealed that a high MAG score is associated with poor prognosis in AML patients. Conclusions Our findings provide molecular evidence supporting the stronger graft-versus-leukemia effects observed in HLA haploidentical HSCT. Moreover, they highlight that abnormal metabolic activation in CD8+ T cells during early AML progression contributes to rapid disease advancement and poor prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06833-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f61f68f92f2fe1df37b3baf378ddd109cecd4ec" target='_blank'>
              Abnormal metabolic activation of CD8+ T cells correlates with poor prognosis in acute myeloid leukemia
              </a>
            </td>
          <td>
            Bixia Wang, Jianing Tang, Yang Zhou, Qianqian Huang, Xinya Jiang, Fangqing Zhang, Shuang Fan, Qi Zhang, Jingrui Zhou, Yu Wang, Xiao-Jun Huang, Huidong Guo
          </td>
          <td>2025-07-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a170c771c60fa0f48c58e00c2cf28bfa97b61abe" target='_blank'>
              Multipotent lineage potential in B cell acute lymphoblastic leukemia is associated with distinct cellular origins and clinical features.
              </a>
            </td>
          <td>
            Ilaria Iacobucci, Andy G X Zeng, Qingsong Gao, Laura Garcia-Prat, Pradyuamma Baviskar, Sayyam Shah, Alexander Murison, V. Voisin, M. Chan-Seng-Yue, Cheng Cheng, C. Qu, Colin Bailey, Matthew Lear, Matthew T. Witkowski, Xin Zhou, Airen Zaldivar Peraza, Karishma Gangwani, Anjali S Advani, S. Luger, M. Litzow, J. M. Rowe, E. Paietta, Wendy Stock, J. Dick, C. Mullighan
          </td>
          <td>2025-06-27</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>105</td>
        </tr>

        <tr id="


 Spatial omics provides three types of information: location, gene expression, and morphology. While current algorithms typically integrate gene expression with location, they often overlook morphology. Chronic Myelomonocytic Leukemia (CMML) is a rare blood cancer characterized by sustained peripheral blood monocytosis, bone marrow dysplasia, and overlapping features of both myelodysplastic syndromes (MDS) and myeloproliferative neoplasms (MPN). A higher level of monoblasts, which are an immature monocytes with round nuclei, is a known risk factor in CMML, but the underlying biology and markers remain unclear. In this study, we developed a novel framework that integrates gene expression and morphology to investigate morphogenomic phenotypes in 12 CMML patients and 8 healthy controls. Recognizing that most existing spatial omics workflows are designed for adherent cells and are unsuitable for suspension cells like bone marrow mononuclear cells (BMMC), we first developed a protocol for immobilizing suspension cells. This was followed by MERFISH spatial RNA analysis of 492 genes across approximately 25,000 cells in a 16 mm2 region of interest over 34 hours. We then introduced a Morphology and Expression Data Optimal Clustering (MEDOC) algorithm, which uses adaptive Fourier-domain alignment and graph clustering to analyze cell shapes. In healthy BMMC, MEDOC analysis identified genes correlated with cell shape, findings that were substantiated by single-cell RNA sequencing data, confirming their association with nuclear shape. Leveraging this shape marker, we successfully identified a promonocyte population, a monoblast subtype that was previously difficult to distinguish. Furthermore, MEDOC analysis in CMML patients revealed two distinct nuclear shapes in mature leukemic monocytes: one resembling a normal kidney bean shape and another characterized by dysmorphic features. Thus, our framework offers a powerful tool to uncover the underlying biology of clinically relevant malignant cell morphologies and to enhance CMML patient stratification.



 Jagadish Sankaran, Ignasius Joanito, Giovani Claresta Wijaya, Ziyue Chen, Joseph Lee, Vaidehi Krishnan, Than Hein, Shyam Prabhakar. INTERGRATING CELL SHAPE AND SPATIAL EXPRESSION IDENTIFIES MORPHOGENOMICS STATES IN LEUKEMIA [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P08.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f4c00943719edf91d07a5ae18534128f66314a" target='_blank'>
              Abstract P08: INTERGRATING CELL SHAPE AND SPATIAL EXPRESSION IDENTIFIES MORPHOGENOMICS STATES IN LEUKEMIA
              </a>
            </td>
          <td>
            Jagadish Sankaran, Ignasius Joanito, Giovani Claresta Wijaya, Ziyue Chen, Joseph Lee, Vaidehi Krishnan, Than Hein, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>