<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-06 09:26:34 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The tumor microenvironment (TME) is a complex network of interactions between malignant and host cells, yet its orchestration in advanced high-grade serous ovarian carcinoma (HGSC) remains poorly understood. We present a comprehensive single-cell spatial atlas of 280 metastatic HGSCs, integrating high-dimensional imaging, genomics, and transcriptomics. Using 929 single-cell maps, we identify distinct spatial domains associated with phenotypically heterogeneous cellular compositions, and demonstrate that immune cell co-infiltration at the tumor-stroma interface significantly influences clinical outcomes. To uncover the key drivers of the tumor ecosystem, we developed CEFIIRA (Cell Feature Importance Identification by RAndom forest), which identified tumor cell-intrinsic MHC-II expression as a critical predictor of prolonged survival, independent of clinicomolecular profiles. Validation with external datasets confirmed that MHC-II-expressing cancer cells drive immune infiltration and orchestrate spatial tumor-immune interactions. Our atlas offers novel insights into immune surveillance mechanisms across HGSC clinicomolecular groups, paving the way for improved therapeutic strategies and patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/684d76f91d1a8b1bc3e208feca47d927bec6a2ec" target='_blank'>
              Single-cell spatial atlas of high-grade serous ovarian cancer unveils MHC class II as a key driver of spatial tumor ecosystems and clinical outcomes
              </a>
            </td>
          <td>
            Fernando Pérez-Villatoro, Lilian van Wagensveld, Aleksandra Shabanova, Ada Junquera, Ziqi Kang, Iga Niemiec, Matias M. Falco, Ella Anttila, J. Casado, Eric Marcus, Essi Kahelin, Foteini Chamchougia, M. Salko, Saundarya Shah, Salvatore Russo, Jacopo Chiaro, Mikaela Grönholm, G. Sonke, K. K. van de Vijver, Rutgerus Fpm Kruitwagen, Maaike Avan der Aa, A. Virtanen, Vincenzo Cerullo, Anna Vähärautio, P. Sorger, Hugo M. Horlings, A. Färkkilä
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Neuroendocrine and tuft cells are rare, chemosensory epithelial lineages defined by expression of ASCL1 and POU2F3 transcription factors, respectively1,2. Neuroendocrine cancers, including small cell lung cancer (SCLC), frequently display tuft-like subsets, a feature linked to poor patient outcomes3–13. The mechanisms driving neuroendocrine–tuft tumour heterogeneity, and the origins of tuft-like cancers are unknown. Using multiple genetically-engineered animal models of SCLC, we demonstrate that a basal cell of origin (but not the accepted neuroendocrine origin) generates neuroendocrine–tuft-like tumours that highly recapitulate human SCLC. Single-cell clonal analyses of basal-derived SCLC further uncovers unexpected transcriptional states and lineage trajectories underlying neuroendocrine–tuft plasticity. Uniquely in basal cells, introduction of genetic alterations enriched in human tuft-like SCLC, including high MYC, PTEN loss, and ASCL1 suppression, cooperate to promote tuft-like tumours. Transcriptomics of 944 human SCLCs reveal a basal-like subset and a tuft-ionocyte-like state that altogether demonstrate remarkable conservation between cancer states and normal basal cell injury response mechanisms14–18. Together, these data suggest that the basal cell is a plausible origin for SCLC and other neuroendocrine-tuft cancers that can explain neuroendocrine–tuft heterogeneity—offering new insights for targeting lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a9f4695f47fd01e98ecf709150d28f3a1f7f97" target='_blank'>
              Basal cell of origin resolves neuroendocrine–tuft lineage plasticity in cancer
              </a>
            </td>
          <td>
            A. Ireland, Sarah B. Hawgood, Daniel A. Xie, Margaret W. Barbier, Scarlett Lucas-Randolph, Darren R. Tyson, Lisa Y. Zuo, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Sonam Puri, Charles M. Rudin, J. Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="We conducted a spatial analysis using imaging mass cytometry applied to stage III colorectal adenocarcinomas. This study used multiplexed markers to distinguish individual cells and their spatial organization from 52 colorectal cancers. We determined the landscape features of cellular spatial features in the CRC tumor microenvironment. This spatial single-cell analysis identified 10 unique cell phenotypes in the tumor microenvironment that included stromal and immune cells with a subset which had a proliferative phenotype. These special features included spatial neighborhood interactions between single cells as well as different tissue niches, especially the tumor infiltrating lymphocyte regions. We applied a robust statistical analysis to identify significant correlations of cell features with phenotypes such as microsatellite instability or recurrence. We determined that microsatellite stable (MSS) colorectal cancers had an increased risk of recurrence if they had the following features: 1) a low level of stromal tumor-infiltrating lymphocytes, and 2) low interactions between CD4+ T cells and stromal cells. Our results point to the utility of spatial single-cell interaction analysis in defining novel features of the tumor immune microenvironments and providing useful clinical cell-related spatial biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc76d98a517403a79b6bbdaa67daa9d1d6c7537" target='_blank'>
              The single-cell spatial landscape of stage III colorectal cancers
              </a>
            </td>
          <td>
            Andrew Su, Hojoon Lee, Minh Tran, Richard Cruz, A. Sathe, Xiangqi Bai, Ignacio A. Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, D. Haslem, David Jones, Lincoln Nadauld, Quan H Nguyen, Hanlee P. Ji, Terence Rhodes
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="In bladder cancer (BLCA), the cellular expression of lineage markers with predictive biomarker function can have important implications for tumor progression, treatment response, and survival. However, tumor heterogeneity and the coexistence of distinct tumor subpopulations can complicate the utility of using a primary, clinically assigned tumor signature. In this report, we have applied the reference atlas, Tabula Sapiens, and deep learning model, UniCell, to conduct in-depth transcriptional analysis of unique lineage marker-defined cell clusters in a carcinogen (BBN) induced bladder cancer model. UniCell deconvolution has identified tumor populations, including urothelial, adenocarcinoma, squamous cell carcinoma, and mesenchymal tumor populations, each with cell-intrinsic gene expression signatures relevant to human BLCA progression. The identified tumor clusters contain uniquely basal, luminal, stromal, or hybrid cells in cell lineage marker expression. To understand the significance of these populations during progression, we used trajectory and pseudo-time analysis to show that cells uniquely basal are plastic in lineage identity and can evolve to form tumor populations composed of other lineage marker-defined signatures. Finally, pathway and drug enrichment analysis of tumor cell clusters were used to identify therapeutics that may preferentially target the identified tumor cell populations. These data collectively define a molecular template that may uniquely profile molecular plasticity occurring during progression and response to therapy with important implications for human disease. Significance Tumor heterogeneity is a mechanism for treatment resistance. Our study defines unique tumor subpopulations having differential therapeutic sensitivities and potential for lineage plasticity. Our modeling may impact the treatment of BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e51b8c18f48b163ace1388c13e342c3f5124f692" target='_blank'>
              Mapping of Multilineage Tumor Cell Populations in Mouse Bladder Cancer
              </a>
            </td>
          <td>
            Nadine Schrode, Yang Hu, Haocheng Yu, M. Galsky, J. Sfakianos, R. Sebra, Kristin G Beaumont, David J Mulholland
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving rapidly as a pivotal technology in studying the biology of tumors and their associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. In this study, we used serial 5-m sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma tumor samples in tissue microarrays to compare the performance of the single cell ST platforms CosMx, MERFISH, and Xenium (uni/multi-modal) platforms in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx Digital Spatial Profiler, and hematoxylin and eosin staining data for the same samples. In addition to objective assessment of automatic cell segmentation and phenotyping, we performed pixel-resolution manual evaluation of phenotyping to carry out pathologically meaningful comparison between ST platforms. Our study detailed the intricate differences between the ST platforms, revealed the importance of parameters such as tissue age and probe design in determining the data quality, and suggested reliable workflows for accurate spatial profiling and molecular discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17805d448b42a38f2a1b50b0eb2fd17aaa264ce4" target='_blank'>
              Comparison of imaging-based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed, paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Nejla Ozirmak Lermi, Max Molina Ayala, Sharia D Hernandez, Wei Lu, Khaja B. Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, Jinzhuang Dou, Qingnan Liang, Rti Team, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchez-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L Gibbons, Ignacio I. Wistuba, C. Haymaker, Ken Chen, L. S. Solis Soto
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Recent studies have identified a unique subtype of cancer-associated fibroblasts (CAFs) termed antigen-presenting CAFs (apCAFs), which remain the least understood CAF subtype. To gain a comprehensive understanding of the origin and function apCAFs, we construct a fibroblast molecular atlas across 14 types of solid tumors. Our integration study unexpectedly reveals two distinct apCAF lineages present in most cancer types: one associated with mesothelial-like cells and the other with fibrocytes. Using a high-resolution single-cell spatial imaging platform, we characterize the spatial niches of these apCAF lineages. We find that mesothelial-like apCAFs are located near cancer cells, while fibrocyte-like apCAFs are associated with tertiary lymphoid structures. Additionally, we discover that both apCAF lineages can up-regulate the secreted protein SPP1, which facilitates primary tumor formation and peritoneal metastasis. Taken together, this study offers an unprecedented resolution in analyzing apCAF lineages and their spatial niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fa0ff2ff7820fdabd0fa0f4d3421c72dfbfc1" target='_blank'>
              Single-cell resolution spatial analysis of antigen-presenting cancer-associated fibroblast niches
              </a>
            </td>
          <td>
            Xiongfeng Chen, Zhuan Zhou, Zeynep Yazgan, Luyu Xie, Francesca Rossi, Yang Liu, Bo Zhang, Patricio M. Polanco, Herbert J. Zeh, Alex C. Kim, Huocong Huang
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), a distinct subpopulation within tumors, are pivotal in driving treatment resistance and tumor recurrence, posing substantial challenges to conventional therapeutic strategies. Precise quantification and profiling of these cells are essential for improving cancer treatment outcomes. We present ACSCeND, an advanced deep neural network model accompanied by a robust workflow, specifically developed to quantify cellular compositions from bulk RNA-seq data, enabling accurate CSC profiling. By integrating bulk RNA-seq data with insights derived from single-cell RNA-seq datasets, ACSCeND effectively captures the diversity and hierarchical organization of tumor-resident cell states, alongside cell-specific gene expression profiles (GEPs). Compared to current tissue deconvolution models, ACSCeND exhibits superior performance, achieving significantly higher Concordance Correlation Coefficient (CCC) values and lower Root Mean Square Error (RMSE) across various pseudobulk and real-world bulk tissue samples. Application of ACSCeND to TCGA and PRECOG datasets reveals a strong association between CSC abundance and poorer disease-free survival outcomes, underscoring the clinical relevance of CSCs in cancer progression. Furthermore, cell-specific GEPs for distinct CSC states unveil novel molecular signatures and illuminate the origins of CSC-driven tumor heterogeneity. In summary, ACSCeND provides a powerful, scalable platform for high-throughput quantification of cellular compositions and distinct potency states within normal tissues as well as highly heterogeneous tissues, such as tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b3b0a8bb89725a847aae6e2af230a0666ba9d6" target='_blank'>
              Comprehensive Enumeration of Cancer Stem-like Cell Heterogeneity Using Deep Neural Network
              </a>
            </td>
          <td>
            Debojyoti Chowdhury, Shreyansh Priyadarshi, Sayan Biswas, Bhavesh Neekhra, Debayan Gupta, Shubhasis Haldar
          </td>
          <td>2024-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="In rectal cancer, where part of the patients undergoes chemoradiotherapy, there is a need for improved pretreatment biomarkers applicable to biopsies. Tumor budding (TB) is a biomarker used in colon cancer, and due to its link to epithelial-mesenchymal transition (EMT), is hypothesized to be a potential marker for therapy resistance. Assessment of the utility of tumor buds in rectal biopsies is challenging due to their rarity. As EMT-related processes are also seen in other morphological features beyond tumor buds, we investigated EMT in tumor tissue including morphological features such as tumor cluster size and fibril-like structures. To do so, we leveraged a cohort of colon cancer whole-slide images and another cohort consisting of rectal cancer biopsies, visualized using hyperplex immunofluorescence to identify tumor and EMT-associated proteins. We built a custom image analysis pipeline to detect and segment tumor buds and other morphological features and correlated them with molecular expression intensities. We found strong correlations of EMT up-regulation and morphological transition states, both at the invasive margin and the tumor center. We furthermore observed a link between morpho-molecular transitions and histological growth patterns, which in turn can inform novel biomarkers. Finally, quantification of these morpho-molecular transition states in rectal biopsies showed their impact on survival after neoadjuvant chemoradiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f452e9eb79df45de63d0362c848378c46029149c" target='_blank'>
              Morpho-molecular features of Epithelial Mesenchymal Transition associate with clinical outcome in patients with rectal cancer
              </a>
            </td>
          <td>
            Mauro Gwerder, Cansaran Saygili Demir, Hannah L Williams, A. Lugli, Cristina Graham Martinez, Joanna Kowal, Amjad Khan, Philipp Kirchner, Thibaud Koessler, Martin D. Berger, Martin Weigert, I. Zlobec
          </td>
          <td>2024-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The organizational principles of nephronal segments are based on longstanding anatomical and physiological attributes that are closely linked to the homeostatic functions of the kidney. Novel molecular approaches have recently uncovered layers of deeper signatures and states in tubular cells that arise at various timepoints on the spectrum between health and disease. For example, a dedifferentiated state of proximal tubular cells with mesenchymal stemness markers is frequently seen after injury. The persistence of such a state is associated with failed repair. Here, we introduce a novel analytical pipeline applied to highly multiplexed spatial protein imaging to characterize proximal tubular subpopulations and neighborhoods in reference and disease human kidney tissue. The results were validated and extended through integration with spatial and single cell transcriptomics. We demonstrate that, in reference tissue, a large proportion of S1 and S2 proximal tubular epithelial cells express THY1, a mesenchymal stromal and stem cell marker that regulates differentiation. Kidney disease is associated with loss of THY1 and transition towards expression of PROM1, another stem cell marker shown recently to be linked to failed repair. We demonstrate that the trajectory of proximal tubular cells to THY1 expression is clearly distinct from that of PROM1, and that a state with PROM1 expression is associated with niches of inflammation. Our data support a model in which the interplay between THY1 and PROM1 expression in proximal tubules associates with their regenerative potential and marks the timeline of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e12887f03204d9141acb86ce8374e99f9dcbad" target='_blank'>
              Integration of spatial multiplexed protein imaging and transcriptomics in the human kidney tracks the regenerative potential timeline of proximal tubules
              </a>
            </td>
          <td>
            Mahla Asghari, Angela R. Sabo, Daria Barwinska, R. M. Ferreira, M. Ferkowicz, William S. Bowen, Ying-Hua Cheng, D. Gisch, Connor J. Gulbronson, Carrie L. Phillips, Katherine J. Kelly, T. Sutton, James C Williams, Miguel Vazquez, John O’Toole, Paul M. Palevsky, Sylvia E. Rosas, S. Waikar, Krzysztof Kiryluk, C. Parikh, Jeff Hodgins, P. Sarder, Ian H. de Boer, Jonathan Himmelfarb, Matthias Kretzler, Sanjay Jain, Michael T. Eadon, S. Winfree, Tarek M. El-Achkar, P. Dagher
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4258aac43fd62290da08aca5103948ba5a168475" target='_blank'>
              Multimodal single cell-resolved spatial proteomics reveal pancreatic tumor heterogeneity
              </a>
            </td>
          <td>
            Yanfen Xu, Xi Wang, Yuan Li, Yiheng Mao, Yiran Su, Yize Mao, Yun Yang, Weina Gao, Changying Fu, Wendong Chen, Xueting Ye, Fuchao Liang, Panzhu Bai, Ying Sun, Shengping Li, Ruilian Xu, Ruijun Tian
          </td>
          <td>2024-11-21</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this gap, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from 2 patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells and we experimentally demonstrated that CCL18 increases neuroblastoma cell migration. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed the ALK ligand ALKAL2 and were predicted to communicate with neighboring ALK expressing cancer cells. We demonstrated a unique developmental pattern of adrenal medulla-specific expression of ALK and adrenocortical-specific expression of ALKAL2, suggesting a role of these signaling interactions in neuroblastoma carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cacb0dcc803b6c58db82e59f6d5ec533dad1808" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlson, A. Claeys, Eva Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third leading cause of cancer mortality in the United States. Familial adenomatous polyposis (FAP) is a hereditary syndrome that raises the risk of developing CRC, with total colectomy as the only effective prevention. Even though FAP is rare (0.5% of all CRC cases), this disease model is well suited for studying the early stages of malignant transformation as patients form many polyps reflective of pre-cancer states. In order to spatially profile and analyze the pre-cancer and tumor microenvironment, we have performed single-cell multiplexed imaging for 52 samples: 12 normal mucosa,16 FAP mucosa,18 FAP polyps, 2 FAP adenocarcinoma, and 4 sporadic colorectal cancer (CRCs) using Co-detection by Indexing (CODEX) imaging platform. The data revealed significant changes in cell type composition occurring in early stage polyps and during the malignant transformation of polyps to CRC. We observe a decrease in CD4+/CD8+ T cell ratio and M1/M2 macrophage ratio along the FAP disease continuum. Advanced dysplastic polyps show a higher population of cancer associated fibroblasts (CAFs), which likely alter the pre-cancer microenvironment. Within polyps and CRCs, we observe strong nuclear expression of beta-catenin and higher number neo-angiogenesis events, unlike FAP mucosa and normal colon counterparts. We identify an increase in cancer stem cells (CSCs) within the glandular crypts of the FAP polyps and also detect Tregs, tumor associated macrophages (TAMs) and vascular endothelial cells supporting CSC survival and proliferation. We detect a potential immunosuppressive microenvironment within the tumor ‘nest’ of FAP adenocarcinoma samples, where tumor cells tend to segregate and remain distant from the invading immune cells. TAMs were found to infiltrate the tumor area, along with angiogenesis and tumor proliferation. CAFs were found to be enriched near the inflammatory region within polyps and CRCs and may have several roles in supporting tumor growth. Neighborhood analyses between adjacent FAP mucosa and FAP polyps show significant differences in spatial location of cells based on functionality. For example, in FAP mucosa, naive CD4+ T cells alone tend to localize near the fibroblast within the stromal compartment. However, in FAP polyp, CD4+T cells colocalize with the macrophages for T cell activation. Our data are expected to serve as a useful resource for understanding the early stages of neogenesis and the pre-cancer microenvironment, which may benefit early detection, therapeutic intervention and future prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f34b5e9edbfa348b570295378a92ebadba11703" target='_blank'>
              Single-cell spatial mapping reveals alteration of cell type composition and tissue microenvironment during early colorectal cancer formation
              </a>
            </td>
          <td>
            Tuhin K. Guha, Edward D. Esplin, Aaron M. Horning, R. Chiu, Kristina Paul, Annika K. Weimer, Winston R. Becker, Rozelle Laquindanum, M. Mills, D. G. Esplin, Jeanne Shen, Emma Monte, Shannon M. White, Thomas V. Karathanos, Daniel L. Cotter, Joanna Bi, U. Ladabaum, Teri A. Longacre, Christina Curtis, William J. Greenleaf, James M Ford, Michael P. Snyder
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER.
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Carcinogenesis is characterized not only by the uncontrolled growth of malignant cells but also by the disruption of the normal balance of cellular processes and intercellular interactions in the microenvironment that overcome the constraints of tissue homeostasis and support malignant growth. We profiled benign pulmonary dysplasia, carcinoma in situ, and invasive lung carcinomas at single-cell resolution to identify composite changes in cellular processes, signaling, and interactions among tumor-immune-stromal cells in the microenvironment in progressively advanced disease stages. We developed OncoTerrain, a hyperparameter-tuned model that captured synergistic multimodal signatures of an increasingly perturbed microenvironment in malignant disease stages and identified composite microenvironmental changes that supported cancer hallmarks. Key cancer-related changes in transcriptional states, cellular processes, and intercellular interactions involving immune, fibroblast, and stromal cell types preceded tumor initiation and were often synergistic. The microenvironment of increasingly malignant tissues was characterized by immune avoidance, ECM remodeling, and altered cell mobility. There were changes in cell states in fibroblasts, macrophages, and their inter-cellular interactions with other cell types, whereas T-cell activation occurred late. The in-situ carcinomas showed variations in the composite microenvironmental states that corroborated their pathology, which was not apparent at the genome level. A subset of those harbored populations of tumor and non-tumor cells with aggressive characteristics in some but not all aspects of hallmarks of carcinogenesis in the lung. We suspect that the variation in the coordination of microenvironmental cues may influence why some but not all in-situ tumors progress to the advanced stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/437dc7b34021d21e799bbd0ca24c7b8aa0ec942c" target='_blank'>
              Disruptive changes in tissue microenvironment prime oncogenic processes at different stages of carcinogenesis in lung
              </a>
            </td>
          <td>
            Vignesh Venkat, Xiaoju Hu, Antara Biswas, Ankit Saxena, Jyoti Malhotra, Gregory Reidlinger, Subhajyoti De
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In this study, we delved into the intricate cellular landscape and interactions within hepatocellular carcinoma (HCC) using single-cell RNA sequencing (scRNA-seq). Our analysis uncovered a diverse array of cell types present in HCC tumors, including endothelial cells, fibroblasts, hepatocytes, Kupffer cells, macrophages, neutrophils, NK cells, pericytes, and proliferating cells. By examining the expression profiles of key marker genes, we elucidated the distinct transcriptional signatures associated with each cell type. Notably, we observed upregulation of certain genes associated with tumor progression and metastasis, highlighting potential therapeutic targets. Additionally, our study revealed the relative proportions of each cell type within HCC tumors, shedding light on the complex cellular composition of these malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/516f83808b51104c5136da0299de6bce74fe34c4" target='_blank'>
              Single-Cell RNA Sequencing: Exploring the Cellular Landscape and Interactions in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Wenyi Wei
          </td>
          <td>2024-11-25</td>
          <td>Advances in Engineering Technology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Although immune checkpoint inhibitors (ICIs) have revolutionized melanoma treatment, many individuals either show no response or eventually develop acquired resistance. This variability could be linked to differences in the tumor immune microenvironment, pre-existing drug-resistant cells or treatment induced changes in melanoma cell states. Using single cell RNA sequencing (scRNA-seq) coupled with heritable barcodes, this study aims to lineage trace changes in melanoma transcriptional cell states and map changes in the tumor immune microenvironment to uncover if preexisting and/or therapy-induced melanoma transcriptional cell states lead to resistance to ICIs. In this study, we used YUMMER1.7 mouse melanoma cells which were transduced with barcodes under conditions that allow delivery of a single unique barcode to each cell, this served as a cell lineage tag. Each barcode is heritable and stably transcribed into RNA molecules so that individual barcoded cells can be matched with a gene expression profile from scRNA-seq outputs. The barcoded cells were subcutaneously injected into C57BL/6 mice, and once tumors were established, mice were treated with 3 cycles of anti-PD-1 and anti-CTLA-4 therapy. Lineage tracing and scRNA-seq analyses were performed on tumors harvested prior to treatment, early on treatment (Day 6), during minimal residual disease (Day 13) and upon relapse. Early on treatment (Day 6)- two tumour groups were harvested, those that responded well and others that only partially responded. Our results showed that during the early on treatment phase (Day 6) tumors that responded well and those that only partially responded displayed distinct barcodes. Additionally, prevalent barcoded cells identified at the minimal residual disease phase were also dominant upon relapse. Analysis of the transcriptional states is underway to determine if ICI therapy induces transcriptional heterogeneity in sensitive and tolerant cells and if a particular or several transcriptional states lead to relapse. This study will potentially identify predictive response biomarkers and vulnerabilities of ICI naïve and resistant cells that could be targeted to improve outcomes for melanoma patients. We will also identify neoadjuvant approaches, to remove the inherent heterogeneity within the tumor cell population, facilitating a more uniform sensitivity to ICIs.
 Citation Format: Reem Saleh, Riyaben Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen Sheppard. Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2046528ce162da3c656f0facbde8bbcbfa6faf03" target='_blank'>
              Abstract C044: Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance
              </a>
            </td>
          <td>
            Reem Saleh, Riyaben P. Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen E. Sheppard
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="T cell therapies, such as chimeric antigen receptor (CAR) T cells and T cell receptor (TCR) T cells, are a growing class of anti-cancer treatments. However, expansion to novel indications and beyond last-line treatment requires engineering cells’ dynamic population behaviors. Here we develop the tools for cellular behavior analysis of T cells from live-cell imaging, a common and inexpensive experimental setup used to evaluate engineered T cells. We first develop a state-of-the-art segmentation and tracking pipeline, Caliban, based on human-in-the-loop deep learning. We then build the Occident pipeline to collect a catalog of phenotypes that characterize cell populations, morphology, movement, and interactions in co-cultures of modified T cells and antigen-presenting tumor cells. We use Caliban and Occident to interrogate how interactions between T cells and cancer cells differ when beneficial knock-outs of RASA2 and CUL5 are introduced into TCR T cells. We apply spatiotemporal models to quantify T cell recruitment and proliferation after interactions with cancer cells. We discover that, compared to a safe harbor knockout control, RASA2 knockout T cells have longer interaction times with cancer cells leading to greater T cell activation and killing efficacy, while CUL5 knockout T cells have increased proliferation rates leading to greater numbers of T cells for hunting. Together, segmentation and tracking from Caliban and phenotype quantification from Occident enable cellular behavior analysis to better engineer T cell therapies for improved cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e025cf1fccd7422ab1c920631722b6d643225bf" target='_blank'>
              Cellular behavior analysis from live-cell imaging of TCR T cell–cancer cell interactions
              </a>
            </td>
          <td>
            Archit Verma, Changhua Yu, Stefanie Bachl, Ivan Lopez, Morgan Schwartz, Erick Moen, Nupura Kale, Carter Ching, Geneva Miller, Tom Dougherty, Edward Pao, William Graf, Carl Ward, Siddhartha Jena, Alex Marson, J. Carnevale, David Van Valen, Barbara E. Engelhardt
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Melanoma, a highly metastatic skin cancer, exhibits variations in prognosis and response to therapy based on the site of metastasis. Despite the success of immunotherapy and targeted therapies in melanoma, over half of metastatic melanoma patients will experience disease progression due to therapy resistance. The heterogeneity and plasticity of melanoma cells contribute to metastatic dissemination and therapy resistance. Our aim is to determine whether distinct clones and/or their transcriptional cell states can predict the formation of tumors in various organs and assess how these clones change over time. To identify melanoma clones across different metastatic sites, NOD scid gamma (NSG) mice and C57BL/6 mice were injected subcutaneously, intravenously or intracranially with the same pool of cells of barcoded luciferase expressing YUMMER1.7 murine melanoma cells. Transduction conditions ensured that one DNA barcode integrated into cell genomes at one barcode per cell, serving as a lineage tag. Bioluminescence imaging was performed once a week to monitor tumor growth of mice injected intravenously and intracranially. Subcutaneous tumors were measured by calliper. Mice were euthanized at different time points; day 8, 15, 22, 29 post-implantation and at ethical endpoints. Tumors were harvested and DNA sequencing was performed to identify barcodes expressed by the tumor cells. All mice developed tumors, with 100% penetrance in NSG mice. However, in C57BL/6 mice, 10% of mice intravenously injected and 27% of subcutaneously implanted mice showed complete lesion regression, suggesting that the immune system may be responsible for tumor regression. Analysis of barcodes allowed us to assess the heterogeneity of melanoma tumors at different metastatic sites and their evolution over time. Lineage tracing and clonal heterogeneity will be evaluated using the state of art technology, SPLINTR (Single-cell Profiling and LINeage Tracing), enabling us to match a cells evolution with changes in transcriptional states. Barcode analysis performed before implanting the cells and, at different timepoints in subcutaneous and lung tumors showed that dominant subclones at the baseline were also dominant in subcutaneous and lung tumors in immunocompromised mice. In contrast, dominant subclones in immunocompetent mice was those present in lower proportion at baseline. Additionally, greater variability of subclones was observed especially in lung and brain tumors, across immunocompetent mice, likely as a mechanism of resistance, enabling these tumors to overcome immunoediting. Understanding the variability of clonality between different metastatic sites and over time will improve our comprehension of the role of different subclones in organ- specific metastasis and their transcriptional cell states.
 Citation Format: Veronica L. Aedo Lopez, Reem Saleh, Benjamin Blyth, Xin Du, Dane Vassiliadis, Katie Fennell, Davide Moi, Roberta Mazzieri, Riccardo Dolcetti, Karen E. Sheppard, Grant A. McArthur. Intra- and inter-tumoral heterogeneity of melanoma across different metastatic sites [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd114bc5004ce77e46072db707d1b3ec32d44c22" target='_blank'>
              Abstract B001: Intra- and inter-tumoral heterogeneity of melanoma across different metastatic sites
              </a>
            </td>
          <td>
            Veronica L. Aedo Lopez, Reem Saleh, Benjamin Blyth, Xin Du, Dane Vassiliadis, Katie Fennell, D. Moi, Roberta Mazzieri, Riccardo Dolcetti, Karen E. Sheppard, Grant A. McArthur
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract High-dimensional, spatial single-cell technologies, such as CyTOF imaging mass cytometry (IMC), provide detailed information regarding locations of a large variety of cancer and immune cells in microscopic scales in tumor microarray slides obtained from patients prior to immune checkpoint inhibitor (ICI) therapy. An important question is how the initial spatial organization of these cells in the tumor microenvironment (TME) changes with time and regulates tumor growth and eventually outcomes as patients undergo ICI therapy. Utilizing IMC data of melanomas of patients who later underwent ICI therapy, we develop a spatially resolved interacting cell system model that is calibrated against patient response data to address the above question. We find that the tumor fate in these patients is determined by the spatial organization of activated CD8+ T cells, macrophages, and melanoma cells and the interplay between these cells that regulate exhaustion of CD8+ T cells. We find that fencing of tumor cell boundaries by exhausted CD8+ T cells is dynamically generated from the initial conditions that can play a protumor role. Furthermore, we find that specific spatial features such as co-clustering of activated CD8+ T cells and macrophages in the pretreatment samples determine the fate of the tumor progression, despite stochastic fluctuations and changes over the treatment course. Our framework enables the determination of mechanisms of interplay between a key subset of tumor and immune cells in the TME that regulate clinical response to ICIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9225e8a7e492a347e0673b0bb1568c995b67d250" target='_blank'>
              Spatial organization and stochastic fluctuations of immune cells impact clinical responsiveness to immunotherapy in melanoma patients
              </a>
            </td>
          <td>
            Giuseppe Giuliani, William Stewart, Zihai Li, Ciriyam Jayaprakash, Jayajit Das
          </td>
          <td>2024-11-26</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Inadequate response to androgen deprivation therapy (ADT) frequently arises in prostate cancer, driven by cellular mechanisms that remain poorly understood. Here, we integrated single-cell RNA sequencing, single-cell multiomics, and spatial transcriptomics to define the transcriptional, epigenetic, and spatial basis of cell identity and castration response in the mouse prostate. Leveraging these data along with a meta-analysis of human prostates and prostate cancer, we identified cellular orthologs and key determinants of ADT response and resistance. Our findings reveal that mouse prostates harbor lobe-specific luminal epithelial cell types distinguished by unique gene regulatory modules and anatomically defined androgen-responsive transcriptional programs, indicative of divergent developmental origins. Androgen-insensitive, stem-like epithelial populations - resembling human club and hillock cells - are notably enriched in the urethra and ventral prostate but are rare in other lobes. Within the ventral prostate, we also uncovered two additional androgen-responsive luminal epithelial cell types, marked by Pbsn or Spink1 expression, which align with human luminal subsets and may define the origin of distinct prostate cancer subtypes. Castration profoundly reshaped luminal epithelial transcriptomes, with castration-resistant luminal epithelial cells activating stress-responsive and stemness programs. These transcriptional signatures are enriched in tumor cells from ADT-treated and castration-resistant prostate cancer patients, underscoring their likely role in driving treatment resistance. Collectively, our comprehensive cellular atlas of the mouse prostate illuminates the importance of lobe-specific contexts for prostate cancer modeling and reveals potential therapeutic targets to counter castration resistance. Significance Statement Androgen deprivation therapy is a mainstay in prostate cancer treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer new directions for prostate cancer modeling and pave the way toward novel therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e36c2b5c9a5688d815f8fbf4d281fd5d7c15cd" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>166</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease of the bone marrow (medullary) but can also involve extramedullary tissues. While single cell dynamics of AML in suspension are previously explored, a comprehensive spatial transcriptomic assessment in AML remain underexplored. Here, we used Visium spatial transcriptomics to resolve medullary and extramedullary AML environments. We reveal spatial co-localization of monocytes and granulocyte-monocyte progenitors with leukemic populations in the bone marrow, sharing molecular signatures with extramedullary sites. Cell-cell communication via the CXCL12- CXCR4 axis correlated with PI3K/AKT/mTOR signaling in high inflammatory niches. Trans- differentiation states were concentrated in AML-infiltrated regions, with committed-like AML populations present in inflammatory niches and away from the trabeculae, while primitive-like AML cells localized near the endosteal niche. We validated these findings in GeoMx-based Digital Spatial profiling (DSP). Our study applied multimodal spatial transcriptomic approaches to characterize the spatial hierarchy and microenvironmental dynamics of AML differentiation states. We also demonstrated the feasibility of applying Visium-based spatial transcriptomics in decalcified bone tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6a7bd964d0d9d4b4af6c8ceee9fbf7068b314fb" target='_blank'>
              Spatial Transcriptomics Reveals Inflammation and Trans-differentiation States of Acute Myeloid Leukemia in Extramedullary and Medullary Tissues
              </a>
            </td>
          <td>
            Enes Dasdemir, Ivo Veletic, Christopher P. Ly, Andres E. Quesada, Christopher D. Pacheco, F. Jelloul, Pamela Borges, Sreyashi Basu, Sonali Jindal, Zhiqiang Wang, Alexander Lazar, Khalida M Wani, D. Antunes, P. Reville, Preethi H. Gunaratne, R. Tower, Padmanee Sharma, Hussein A. Abbas
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23b3d699dd705e1b450a74d7c903ab854deff777" target='_blank'>
              Increased spatial coupling of integrin and collagen IV in the immunoresistant clear-cell renal-cell carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, Taylor Peak, Oscar E. Ospina, N. Chakiryan, Anders E Berglund, Paul A Stewart, Jonathan V Nguyen, C. M. Segura, Natasha L. Francis, P. R. Echevarria, J. Chahoud, Roger Li, Kenneth Y. Tsai, Jodi A. Balasi, Yamila Caraballo Peres, J. Dhillon, Lindsey A. Martinez, Warren E. Gloria, Nathan Schurman, Seang-Jae Kim, M. Gregory, James J Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-12-05</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Immune cells determine the role of the tumor microenvironment during tumor progression, either suppressing tumor formation or promoting tumorigenesis. We analyzed the profile of immune cells in the tumor microenvironment of control mouse skins and skin tumors at the single-cell level. We identified 15 CD45+ immune cell clusters, which broadly represent the most functionally characterized immune cell types including macrophages, Langerhans cells (LC), conventional type 1 dendritic cells (cDC1), conventional type 2 dendritic cells (cDC2), migratory/mature dendritic cells (mDC), dendritic epidermal T cells (DETC), dermal γδ T cells (γδT), T cells, regulatory T cells (Tregs), natural killer cells (NK), type 2 innate lymphoid cells (ILC2), neutrophils (Neu), mast cells (Mast), and two proliferating populations (Prolif.1 and Prolif.2). Skin tumor progression reprogramed immune cells and led to a marked increase in the relative percentages of macrophages, cDC2, mDC, Tregs, and Neu. Macrophages, the largest cell cluster of immune cells in skin tumors. In addition, macrophages emerged as the predominant communication ‘hub’ in skin tumors, highlighting the importance of macrophages during skin tumor progression. In contrast, other immune cell clusters decreased during skin tumor progression, including DETC, γδT, ILC2, and LC. In addition, skin tumor progression dramatically upregulated Jak2/Stat3 expression and the interferon response across various immune cell clusters. Further, skin tumor progression activated T cells and NK cells indicated by elevated expression of IFN-γ and Granzyme B in skin tumors. Meanwhile, a pronounced infiltration of M2-macrophages and Tregs in skin tumors created an immunosuppressive microenvironment, consistent with the elevated expression of the Stat3 pathway in skin tumors. In summary, our study elucidates the immune cell landscape of epidermal neoplasms, offering a comprehensive understanding of the immune response during skin tumor progression and providing new insights into cancer immune evasion mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e55f00e717854abc6b07c45a9ea29834680a1b82" target='_blank'>
              Single-Cell Profiling Reveals Global Immune Responses during the Progression of Murine Epidermal Neoplasms
              </a>
            </td>
          <td>
            Xiying Fan, Tonya M. Brunetti, Kelsey Jackson, Dennis R. Roop
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains an untreatable disease. Understanding GBM’s infiltrative biology at the resection margin is limited, despite causing disease recurrence and progression. To address this, we generated a high-throughput single-nucleus (sn)RNA-seq and snATAC-seq multi-omic dataset from six tumors with distinct genomic drivers and combined it with spatial transcriptomics to characterize the unique molecular phenotype of GBM near the margin. By contrasting GBM-specific biology in matching “Core” vs. “Margin” dissections, we define unique, shared “GBM infiltration” and chromatin accessibility signatures near the margin. We prioritize EGFR as a top differentially expressed and accessible “Margin” marker across GBM subtypes, show its dynamic expression along a core-to-margin infiltration trajectory, and validate its role in migration through CRISPR/Cas9 deletion in two patient-derived models. ChIP-seq studies furthermore corroborate preferential TEAD1 binding at EGFR’s accessible regulatory elements. This validated multi-omic dataset enables further studies into tumor and microenvironment biology in the context of residual GBM disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d032da4813c4397d215d1b17d17829c10107f9f" target='_blank'>
              Spatial multi-omics defines a shared glioblastoma infiltrative signature at the resection margin
              </a>
            </td>
          <td>
            Balagopal Pai, S. Ramos, W. Cheng, T. Joshi, Gabrielle Price, Jessica Tomé-García, G. Nudelman, S. Shroff, K. Beaumont, Yong Raymund, R. Sebra, E. Zaslavsky, N. Tsankova
          </td>
          <td>2024-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="The extracellular matrix (ECM) forms the scaffold in which cells reside and interact. The composition of this scaffold guides the development of local immune responses and tissue function. With the advent of multiplexed spatial imaging methodologies, investigating the intricacies of cellular spatial organisation are more accessible than ever. However, the relationship between cellular organisation and ECM composition has been broadly overlooked. Using imaging mass cytometry, we investigated the association between cellular niches and their surrounding matrix environment during allergic airway inflammation in two commonly used mouse strains. By first classifying cells according to their canonical intracellular markers and then by developing a novel analysis pipeline to independently characterise a cells ECM environment, we integrated analysis of both intracellular and extracellular data. Applying this methodology to three distinct tissue regions we reveal disparate and restricted responses. Recruited neutrophils were dispersed within the alveolar parenchyma, alongside a loss of alveolar type I cells and an expansion of alveolar type II cells. This activated parenchyma was associated with increased proximity to hyaluronan and chondroitin sulphate. In contrast, infiltrating CD11b+ and MHCII+ cells accumulated in the adventitial cuff and aligned with an expansion of the subepithelial layer. This expanded subepithelial region was enriched for closely interacting stromal and CD11b+ immune cells which overlaid regions enriched for type-I and type-III collagen. The cell-cell and cell-matrix interactions identified here will provide a greater understanding of the mechanisms and regulation of allergic disease progression across different inbred mouse strains and provide specific pathways to target aspects of remodelling during allergic pathology. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc2c9f01204b9f7ab59404926bfc53010da8fafb" target='_blank'>
              Extracellular matrix phenotyping by imaging mass cytometry defines distinct cellular matrix environments associated with allergic airway inflammation
              </a>
            </td>
          <td>
            J. Parkinson, M. Ghafoor, R. Dodd, H. E. Tompkins, M. Fergie, M. Rattray, J. Allen, T. E. Sutherland
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7e21f7327e57fd7a5664b9a5fd090102737fcd" target='_blank'>
              scStateDynamics: deciphering the drug-responsive tumor cell state dynamics by modeling single-cell level expression changes
              </a>
            </td>
          <td>
            Wenbo Guo, Xinqi Li, Dongfang Wang, Nan Yan, Qifan Hu, Fan Yang, Xuegong Zhang, Jianhua Yao, Jin Gu
          </td>
          <td>2024-11-21</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a fatal adult solid tumour with median overall survival of 18-20 months post-diagnosis; contributing factors to therapeutic inefficacy include acquisition of genomic alterations post-therapy, immune evasion, deregulated hypervascularization, and tumor microenvironmental factors such as hypoxia. Combined with extensive inter- and intra-tumoral heterogeneity at bulk and single-cell level, hypoxia contributes to a gradient of molecular alterations that are specific to different cell populations that make up tumour bulk and reside in specific niches. Hitherto, high-dimensional histopathologic analyses of hypoxic regions within GBM tissue have not been performed. We took a combined spatial and single-cell proteomic profiling approach to investigate the histopathologic features of hypoxia by leveraging a unique clinical study wherein the exogenous hypoxia marker, pimonidazole (PIMO), was administered to GBM-patients preoperatively. Tissue specimens were subjected to imaging mass cytometry, high-resolution imaging, and serial immunohistochemistry using a panel of markers associated with cellular hallmarks of hypoxia, metabolism, proliferation, stemness, angiogenesis, and immune cell types. Our findings showed that PIMO staining is associated with histopathologic features of hypoxia and correlates with specific metabolic, immune, and stemness markers in GBM; specific lymphocyte populations were depleted from hypoxic regions alongside alterations in macrophagic and microglial landscape in a niche-specific manner; hypoxia reduced the proportion of proliferating glioma initiating cells and altered the proliferative, transcriptional, and translational capacity of different cell populations; microvessel density was reduced in hypoxic microenvironment. Cytometry by time-of-flight further validated the altered proportions of specific immune cell types enriched in hypoxic populations of GBM microenvironment. Our study is the first to report use of PIMO to interrogate spatial, single-cell, and phenotypic architecture associated with tissue hypoxia and altered expression of biomarkers associated with hypoxia/glycolysis, immune infiltration, proliferation, and stemness. Identification of targetable biomarkers and mediators of hypoxia-driven habitats in GBM may provide direction for future immunotherapeutic research.
 Citation Format: Shreya Gandhi, Sheila Mansouri, Olivia Singh, Mark Zaidi, Ronald Wu, Andrew Gao, Bradley Wouters, Gelareh Zadeh. Spatial and single-cell proteomic landscaping of the hypoxic microenvironment in glioblastoma [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B069.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5960e28aa4e015a793e4ede112dd73ea174cad4c" target='_blank'>
              Abstract B069: Spatial and single-cell proteomic landscaping of the hypoxic microenvironment in glioblastoma
              </a>
            </td>
          <td>
            Shreya Gandhi, S. Mansouri, O. Singh, M. Zaidi, Ronald Wu, A. Gao, Bradley Wouters, G. Zadeh
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Glioma is an aggressive brain tumor with a poor prognosis. Establishing an in vitro culture model that closely replicates the cellular composition and microenvironment of the original tumor has been challenging, limiting its clinical applications. Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib. Thus, this organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076fe0fc7a2a2145464370151771e2e7246f87bc" target='_blank'>
              GlioME: A Novel Organoid Model Retaining the Glioma Microenvironment for Personalized Drug Screening and Therapeutic Evaluation
              </a>
            </td>
          <td>
            Chengjun Zheng, Qiaodong Chen, Peng Wang, Delong Zhang, Zheng Fang, Yutong Feng, Jie Chen, Jiahong Chen, Yiwen Fu, Bao Yang, Qian Zhao, Fei Sun, Ying Zhang, Tao Jiang, Lemin Zheng, Zhaoshi Bao
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Dysregulation of intracellular signaling networks underpins cancer. However, a systems-level elucidation of how signaling networks within distinct cell subpopulations drive cancer progression in vivo has been unattainable due to technical limitations. We developed INSIGHT (INvestigating SIGnaling network of specific cell subpopulation in Heterogeneous Tissue), a new platform technology combining fluorescence-activated cell sorting with ultra-sensitive mass spectrometry to enable phosphoproteomic characterization of rare and discrete cell subpopulations from fixed tissues. We demonstrated the broad utility of INSIGHT by analyzing the oligodendroglial cell-specific signaling network in the mouse brain. We then applied INSIGHT to investigate the rare, disseminated tumor cell subpopulation in glioblastoma patient-derived xenograft models. INSIGHT uncovered a global rewiring of signaling networks with tumor cell dissemination, marked by a transition from proliferation-associated signaling in the primary tumor cells to signaling associated with postsynapse, neuronal migration, and ion homeostasis in disseminated tumor cells. We reveal interconnections between signaling circuitries within the networks, with numerous proteins, including GluA2, exhibiting altered phosphorylation without protein expression changes, emphasizing the role of post-translational modifications in glioblastoma dissemination. We validated key phosphorylation changes and inferred differentially active kinases with tumor spread to offer new systems-level insights into glioblastoma dissemination mechanisms in vivo. INSIGHT is generally applicable to a wide range of biological systems without genetic engineering and provides quantitative phosphorylation and protein expression data for selected cell subpopulations from heterogeneous tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/decaf507fd537acca35393bd50f3bedca402f1f6" target='_blank'>
              Uncovering the signaling networks of disseminated glioblastoma cells in vivo with INSIGHT
              </a>
            </td>
          <td>
            Ryuhjin Ahn, Alicia D. D’Souza, Lawrence Long, Yufei Cui, D. Burgenske, K. Bakken, Lauren L Ott, Brett L. Carlson, Grace Zhou, Tomer M. Yaron-Barir, Ishwar N. Kohale, Charles A. Whittaker, Cameron T. Flower, Jeffrey Wyckoff, A. Tuma, Jared L. Johnson, Jann Sarkaria, Forest M. White
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="DICER1 syndrome predisposes children and young adults to tumor development across various organs. Many of these cancers are sarcomas, which uniquely express the RNase IIIb domain-deficient form of DICER1 and exhibit consistent histological and molecular similarities regardless of their anatomical origins. To uncover their cellular origin and developmental hierarchy, we established a lineage-traceable genetically engineered mouse model that allows for precise activation of Dicer1 mutations in Hic1+ mesenchymal stromal cells. This model resulted in the development of renal tumors closely mirroring human DICER1 sarcoma histologically and molecularly. Single-cell transcriptomics coupled with targeted spatial gene expression analysis revealed a Hic1+ progenitor population marked by Pdgfra, Dpt, and Mfap4, corresponding to universal fibroblasts of steady-state kidneys. These fibroblastic progenitors exhibit the capacity to undergo rhabdomyoblastic differentiation or transition to highly proliferative anaplastic sarcoma. Investigation of patient samples identified analogous cell states. This study uncovers a fibroblastic origin for DICER1 sarcoma and provides a faithful model for mechanistic investigation and therapeutic development for tumors within the rhabdomyosarcoma spectrum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7612b5f96fea7c628834a8ae0728263c8ee1372" target='_blank'>
              Spatial single cell transcriptomic analysis of a novel DICER1 Syndrome GEMM informs the cellular origin and developmental hierarchy of associated sarcomas
              </a>
            </td>
          <td>
            F. Kommoss, Joyce Zhang, Branden Lynch, S. Chen, J. Senz, Yana Moscovitz, Lesley A. Hill, Wilder Scott, Jonathan Bush, Kenneth S. Chen, Andreas von Deimling, William D. Foulkes, Gregg B. Morin, T. M. Underhill, Yemin Wang, D. Huntsman
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d97774290c1f515233beb8849c220d76ed9e57" target='_blank'>
              Unsupervised clustering reveals noncanonical myeloid cell subsets in the brain tumor microenvironment
              </a>
            </td>
          <td>
            Ismaïl Hermelo, Tuomo Virtanen, I. Salonen, R. Nätkin, Sofia Keitaanniemi, A. Tiihonen, Suvi Lehtipuro, Laura Kummola, Ella Raulamo, K. Nordfors, H. Haapasalo, M. Rauhala, J. Kesseli, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2025-01-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background & Aims Imaging-based, single-cell spatial transcriptomics (iSCST) using formalin-fixed, paraffin-embedded (FFPE) tissue could transform translational research by retaining all tissue cell subsets and spatial locations while enabling the analysis of archived specimens. We aimed to develop a robust framework for applying iSCST to archived clinical FFPE mucosal biopsies from patients with inflammatory bowel disease (IBD). Methods We performed a comprehensive benchmarking comparison of three iSCST platforms capable of analyzing FFPE specimens. We analyzed FFPE mucosal biopsies (n=57) up to 5 years old from non-IBD controls (HC; n=9) and patients with ulcerative colitis (UC;n=11). After platform-specific cell segmentation, we applied a uniform data processing pipeline to all datasets, including transcript detection, cell annotation, differential gene expression, and neighborhood enrichment. Transcriptomic signatures identified with iSCST were validated using external, publicly available bulk transcriptomic datasets. Results A custom 290-plex Xenium gene panel exhibited the highest sensitivity and specificity for transcript detection, enabling precise identification and quantification of diverse cell subsets and differentially expressed genes across cell types and disease states. We mapped transcriptionally distinct fibroblast subsets to discrete spatial locations and identified inflammation-associated fibroblasts (IAFs) and monocytes as a colitis-associated cellular neighborhood. We also identified signatures associated with Vedolizumab (VDZ) responsiveness. VDZ non-responders were characterized by an IAF-monocyte transcriptional signature, while responders exhibited enrichment of epithelial gene sets. Conclusions Our optimized iSCST framework for archived FFPE biopsies provides unique advantages for assessing the role of colitis-associated cellular networks in routinely collected clinical samples. FFPE-based biomarkers could integrate with existing clinical workflows and potentially aid in risk-stratifying patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ae9c6b7cdc0c99eb4a02b078cfde661c5962b30" target='_blank'>
              Single-cell spatial transcriptomics of fixed, paraffin-embedded biopsies reveals colitis-associated cell networks
              </a>
            </td>
          <td>
            Elvira Mennillo, Madison L. Lotstein, Gyehyun Lee, Vrinda Johri, Christina Ekstrand, Jessica Tsui, Julian Hou, Donna E. Leet, Jun Yan He, U. Mahadevan, W. Eckalbar, David Y. Oh, G. Fragiadakis, Michael G. Kattah, A. Combes
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a multi-clonal disease, existing as a milieu of clones with unique but related genotypes as initiating clones acquire subsequent mutations. However, bulk sequencing cannot fully capture AML clonal architecture or the clonal evolution that occurs as patients undergo therapy. To interrogate clonal evolution, we performed simultaneous single cell molecular profiling and immunophenotyping on 43 samples from 32 NPM1-mutant AML patients at different stages of disease. Here we show that diagnosis and relapsed AML samples display similar clonal architecture patterns, but signaling mutations can drive increased clonal diversity specifically at relapse. We uncovered unique genotype-immunophenotype relationships regardless of disease state, suggesting leukemic lineage trajectories can be hard-wired by the mutations present. Analysis of longitudinal samples from patients on therapy identified dynamic clonal, transcriptomic, and immunophenotypic changes. Our studies provide resolved understanding of leukemic clonal evolution and the relationships between genotype and cell state in leukemia biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3ca51b5a9370b4c12038c29c1cdd25d0324a8d0" target='_blank'>
              Genotype-immunophenotype relationships in NPM1-mutant AML clonal evolution uncovered by single cell multiomic analysis
              </a>
            </td>
          <td>
            Morgan Drucker, Darren Lee, Xuan Zhang, Bailee N. Kain, Michael Bowman, D. Nicolet, Zhe Wang, R. Stone, K. Mrózek, Andrew J Carroll, D. Starczynowski, R. L. Levine, John C. Byrd, Ann-Kathrin Eisfeld, Nathan Salomonis, H. L. Grimes, Robert L Bowman, Linde A. Miles
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90bace8a9791e95b3eff7c16fd892d053802f89b" target='_blank'>
              Single-cell transcriptomes of dissecting the intra-tumoral heterogeneity of breast cancer microenvironment
              </a>
            </td>
          <td>
            Peixian Chen, Kaifeng Liang, Xiaofan Mao, Qiuyuan Wu, Zhiyan Chen, Yabin Jin, Kairong Lin, Tiancheng He, Shuqing Yang, Huiqi Huang, Guolin Ye, Juntao Gao, Dan Zhou, Zhihao Zeng
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1266e9e933529788695d2086bf906381800e149" target='_blank'>
              Single-cell and bulk transcriptome analysis reveals tumor cell heterogeneity and underlying molecular program in uveal melanoma
              </a>
            </td>
          <td>
            Ke Li, Jingzhe Huang, Ying Tan, Jie Sun, Meng Zhou
          </td>
          <td>2024-11-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Spatial transcriptomics, situated at the intersection of genomics and spatial biology, offers profound insights into the spatial organization of gene expression within tissues. However, its potential has been constrained by either limited resolution or throughput. While single-cell RNA-seq allows for in-depth profiling of cellular gene expression, the crucial spatial information is often sacrificed during sample collection. In a groundbreaking fusion of these two techniques, our research introduces the glmSMA computational algorithm. This innovative approach aims to predict cell locations by integrating scRNA-seq data with spatial-omics reference atlases. The essence of glmSMA lies in formulating cell mapping as a convex problem, strategically minimizing differences between cellular expression profiles and location expression profiles through L1 and Generalized L2 regularization. Our algorithm has undergone rigorous testing across diverse tissues, including mouse brain, drosophila embryo, and human PDAC samples. The compelling results validate glmSMA’s efficacy, demonstrating its capability to faithfully recapitulate spatial gene expression and anatomical structures. This marks a significant stride forward in overcoming the limitations of existing spatial transcriptomic techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f235303646474952e82316a6231392881105104" target='_blank'>
              A network regularized linear model to infer spatial expression pattern for single cell
              </a>
            </td>
          <td>
            Chaohao Gu, Hu Chen, Zhandong Liu
          </td>
          <td>2024-11-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human lung is a complex organ susceptible to various diseases. Single-cell transcriptomic studies provide rich data to targeting specific research questions. Here, we present uniLUNG, the largest lung transcriptomic cell atlas, comprising over 10 million cells across 20 disease states and healthy controls. We ensembled a universal hierarchical annotation framework and conducted a full benchmarking of data integration to define a standardized nomenclature and marker genes for lung cell types. Using uniLUNG, we identified Lym-monocyte and T-like B cells, new cell types in specific lung diseases, confirming their existence by comparing with external single-cell atlases. Additionally, we discovered the NSCLC-like SCLC subpopulation, a transitional malignant cell population associated with the transition from NSCLC to SCLC, which was validated and further characterized in spatial dimensions, revealing its complex role in tumour progression. Overall, uniLUNG represents a comprehensive range of human lung cell diversity, providing valuable data resources and a reliable foundation for lung single-cell research. HIGHLIGHTS The largest scRNA atlas for human lung covers 10 million cells from 20 lung states. A four-level universal cell annotation framework encompasses 120 lung cell types. Comprehensive benchmarking on 18 strategies guides data integration. Specific distribution of Lym-monocytes and T-like B cells in specific lung diseases. The NSCLC-like SCLC subpopulation in transitional events of malignant cells from NSCLC to SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707c7b05c0b869d7c0046d311738148202a1849b" target='_blank'>
              A universal single-cell transcriptomics atlas of human lung decodes multiple pulmonary diseases
              </a>
            </td>
          <td>
            Fanjie Wu, Wenhao Cai, Hai Tang, Shikang Zheng, Haiyue Zhang, Yixin Chen, Yutong Han, Dingli Zhou, Ruihan Wang, Mingli Ye, Renke You, Amin Chen, Jiaqi Li, Xuegong Zhang, Weizhong Li
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The ability of cancer cells to alter their identity, known as lineage plasticity, is crucial for tumor progression and therapy resistance. In lung adenocarcinoma (LUAD), tumor progression is characterized by a gradual loss of lineage fidelity and the emergence of non-pulmonary identity programs. This can lead to hybrid-identity (hybrid-ID) states in which developmentally incompatible identity programs are co-activated within individual cells. However, the molecular mechanisms underlying these identity shifts remain incompletely understood. Here, we identify the gastrointestinal (GI) transcriptional regulator HNF4α as a critical driver of tumor growth and proliferation in KRAS-driven LUAD. In LUAD cells that express the lung lineage specifier NKX2-1, HNF4α can induce a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also forms an aberrant protein complex with NKX2-1, which disrupts NKX2-1 localization and dampens pulmonary identity within hybrid-ID LUAD. Sustained signaling through the RAS/MEK pathway is critical for maintaining the hybrid-ID state. Moreover, RAS/MEK inhibition augments NKX2-1 chromatin binding at pulmonary-specific genes and induces resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion enhances sensitivity to pharmacologic KRASG12D inhibition. Collectively, our data show that co-expression of opposing lineage specifiers leads to a hybrid identity state that can drive tumor progression and dictate response to targeted therapy in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207f2a257f041c211d79bb19f39ca01a49097c42" target='_blank'>
              Opposing lineage specifiers induce a pro-tumor hybrid-identity state in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Rushmeen Tariq, Anna Waters, Kayla T. O'Toole, Eric L. Snyder
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma (GBM) cells are highly heterogeneous and invasive, leading to treatment resistance and relapse. However, the molecular regulation in and distal to tumors remains elusive. Here, we collected paired tissues from the tumor core (TC) and peritumoral brain (PTB) for integrated snRNA-seq and snATAC-seq analyses. Tumor cells infiltrating PTB from TC behave more like oligodendrocyte progenitor cells than astrocytes at the transcriptome level. Dual-omics analyses further suggest that the distal regulatory regions in the tumor genome and specific transcription factors are potential determinants of regional heterogeneity. Notably, while activator protein 1 (AP-1) is active in all GBM states, its activity declines from TC to PTB, with another transcription factor, BACH1, showing the opposite trend. Combined inhibition of AP-1 and BACH1 more efficiently attenuates the tumor progression in mice and prolongs survival than either single-target treatment. Together, our work reveals marked molecular alterations of infiltrated GBM cells and a synergy of combination therapy targeting intratumor heterogeneity in and distal to GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c193d462d0c2dbf4aee7a761498e3e53b84808" target='_blank'>
              Single-cell multi-omics sequencing uncovers region-specific plasticity of glioblastoma for complementary therapeutic targeting
              </a>
            </td>
          <td>
            Xin Wang, Qian Sun, Tianbin Liu, Haoran Lu, Xuyi Lin, Weiwen Wang, Yang Liu, Yunting Huang, Guodong Huang, Haixi Sun, Qianxue Chen, Junmin Wang, Daofeng Tian, Fan-En Yuan, Longqi Liu, Bo Wang, Ying Gu, Baohui Liu, Liang Chen
          </td>
          <td>2024-11-22</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Liquid biopsies present a promising, minimally invasive procedure through which clinicians can assess and monitor disease status by interrogating circulating tumor cell (CTC) and immune cell phenotypes. In the case of Small-cell lung cancer (SCLC) – a type of lung cancer characterized by aggressive growth, dismal prognosis and limited treatment options - leveraging liquid biopsies and CTCs for evaluating disease status would be of great benefit, given that biopsies are scarce. Furthermore, SCLC CTC numbers are higher compared to other cancers, presenting a unique opportunity to utilize them for assessing SCLC therapy resistance and identifying new targetable markers. In this study, we leverage mass cytometry (i.e. CyTOF) to phenotype SCLC CTC phenotypes in longitudinal clinical specimens to assess therapy resistance at a personalized level. By using CyTOF - a multiplex cytometric approach that enables interrogation of 30-50 protein markers per single cell – we show that we are able to identify and analyze a significant number of CTCs and their heterogeneous expression profiles. To achieve this, we first optimized a CyTOF panel of ∼25-30 antibodies by using SCLC cell lines and blood specimens from healthy donors and SCLC patients. We show that by using this approach, we are able to identify and phenotype SCLC CTCs by using the combination of CD45 (negative expression) and CD56 positive expression with further validation achieved by TTF1 and SCLC subtype transcription factor expression (NEUROD1, POU2F3 and ASCL1). This approach avoids enriching for specific CTC epithelial phenotypes that express surface markers that fluctuate during epithelial- mesenchymal transition (EMT) (e.g. EpCAM, a common marker for isolating CTCs). For parallel immune profiling we assessed among others CD8/CD4 (T cells) and CD11b/HLA-DR (Myeloid-derived suppressor cells, MDSCs) expression. We then proceeded to interrogate CTC phenotypic differences in the naïve vs relapsed setting as well as dynamic changes in longitudinal blood specimens from SCLC patients by evaluating markers that describe immune suppression activity (PD-L1) and EMT status (E-Cadherin, Vimentin, MUC1, Twist and Slug). Importantly, we show that with our approach we can track in real time therapeutic targets that are currently being tested in clinical trials or have been recently FDA approved (e.g. pYAP and DLL3) and found that MUC1 and pYAP are significantly increased in CTCs detected in blood specimens of SCLC relapsed patients. Our immune profiling showed an increase of MDSC percentages and a decrease in the CD4/CD8 T cell ratio in the relapsed setting. Finally, when we used PHENOSTAMP, a previously developed reference EMT map for assessing EMT phenotypes in lung cancer clinical specimens, we observed an increase of EMT heterogeneity in CTCs in relapsed patients. Our study highlights the translational power of our approach in utilizing CyTOF for longitudinally tracking CTCs directly in SCLC patient liquid biopsies towards assessing therapy response and resistance at a personalized level.
 Citation Format: Loukia G Karacosta, Sayantan Bhattacharyya, Allison Stewart, Ashley Victorian, Ester F Lujan, Shafqat Ehsan, Subin Kim, Alberto Duarte, Runsheng Wang, Benedict Anchang, Jing Wang, Lauren Byers. Leveraging mass cytometry for phenotyping CTCs in SCLC liquid biopsies: Tracking therapy resistance at a personalized level [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr PR005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/780e8baaf5c56b4ff2d0f86de6218a0a6ecd6ad1" target='_blank'>
              Abstract PR005: Leveraging mass cytometry for phenotyping CTCs in SCLC liquid biopsies: Tracking therapy resistance at a personalized level
              </a>
            </td>
          <td>
            L. Karacosta, Sayantan Bhattacharyya, A. Stewart, Ashley Victorian, Ester F Lujan, Shafqat F. Ehsan, Subin Kim, Alberto Duarte, Runsheng Wang, Benedict Anchang, Jing Wang, Lauren E. Byers
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCCs) constitute one of the few cancer indications for which mortality rates continue to rise. While Notch signaling dictates a key progenitor lineage choice during development, its role in HCC has remained controversial. Using therapeutic antibodies targeting Notch ligands and receptors to screen over 40 patient-derived xenograft models, we here identify progenitor-like HCCs that crucially depend on a tumor-intrinsic JAG1-NOTCH2 signal. Inhibiting this signal induces tumor regressions by triggering progenitor-to-hepatocyte differentiation, the same cell fate-switch that Notch controls during development. Transcriptomic analysis places the responsive tumors within the well-characterized progenitor subclass, a poor prognostic group of highly proliferative tumors, providing a diagnostic method to enrich for Notch-dependent HCCs. Furthermore, single-cell RNA sequencing uncovers a heterogeneous population of tumor cells and reveals how Notch inhibition shifts cells from a mixed cholangiocyte-hepatocyte lineage to one resembling mature hepatocytes. Analyzing the underlying transcriptional programs brings molecular detail to this process by showing that Notch inhibition de-represses expression of CEBPA, which enables the activity of HNF4α, a hepatocyte lineage factor that is otherwise quiescent. We thus describe a compelling and targetable dependency in a poor-prognosis class of HCCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3764bfcf8bf124425bfb62c588b7e271cb0186df" target='_blank'>
              Notch signaling maintains a progenitor-like subclass of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kerstin Seidel, Robert Piskol, Thi Thu Thao Nguyen, Amy Shelton, Charisa L. Cottonham, Cecile C. de la Cruz, Joseph Castillo, Jesse Garcia, Udi Segal, Mark Merchant, Yeqing Angela Yang, Jasmine Chen, Musa Ahmed, Alexis Scherl, Rajesh Vij, Lluc Mosteiro, Yan Wu, Z. Modrusan, C. Metcalfe, Christopher W. Siebel
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Spatial transcriptomics offers unprecedented insights into the complex cellular landscapes of tissues, particularly in cancer research where understanding the tumor microenvironment is crucial. However, interpreting the vast and intricate data generated by this technology remains a significant challenge. This study explores the potential of Large Language Models (LLMs) to assist in the analysis and interpretation of spatial transcriptomic data from a murine melanoma tumor model. We first evaluated the performance of multiple LLM models in describing and quantifying spatial gene expression patterns. Our benchmarking revealed that spatial transcriptomics data interpretation proved challenging for most models, with only a few demonstrating sufficient capability for this complex task. Using Claude 3.5 Sonnet, which showed the highest accuracy in spot quantification and pattern recognition, we developed a systematic workflow for analyzing the tumor immune landscape. The model first assisted in identifying and prioritizing panels of M1 and M2 macrophage-associated markers through knowledge-driven scoring. It then demonstrated remarkable ability to integrate spatial expression data with extensive immunological knowledge, providing sophisticated interpretation of local immune organization. When analyzing individual tumor regions, the model identified coordinated immunosuppressive mechanisms including metabolic barriers and disrupted pro-inflammatory signaling cascades, findings that both aligned with and extended current understanding of tumor immunology. This study highlights the potential of LLMs as powerful assistive tools in spatial transcriptomics analysis, capable of combining advanced pattern recognition with extensive knowledge integration to enhance data interpretation. While significant development work remains to make such workflows scalable, our proof of concept demonstrates that LLMs can help accelerate the translation of spatial transcriptomics data into biological insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287bc0c305d5950e4e6f2901503d04b32465b5e" target='_blank'>
              Tackling the Complexity of Spatial Transcriptomics Data Interpretation with Large Language Models
              </a>
            </td>
          <td>
            Taushif Khan, Colleen M. Farley, John J. Wilson, Chih-Hao Chang, Damien Chaussabel
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Low-grade endometrial stromal sarcoma (LG-ESS) is a rare uterine malignancy characterized by its complex tumor microenvironment (TME) and high recurrence rates, posing challenges to accurate prognosis and effective treatment. Identifying prognostic biomarkers is essential for improving patient stratification and guiding therapeutic strategies. Methods Using single-cell transcriptome analysis combined with H&E and multiplex immunofluorescence staining, we identified a subpopulation of tumor cells in LG-ESS and further validated the association of this subpopulation and its characteristic genes with LG-ESS prognosis by molecular characterization and bulk transcriptome data. Results Our analysis reveals multiple cellular subpopulations within the tumor tissue, particularly a tumor cell subpopulation among them which is associated with poor prognosis. Originating from normal stromal fibroblasts, this subpopulation appears to play a crucial role in TME remodeling, smooth muscle cell behavior, and potentially in tumorigenesis and metastasis. Of particular interest in this subpopulation is the highly expressed FGF12 gene, which is significantly associated with a shortened survival in ESS, highlighting its potential as a prognostic biomarker. Conclusion Our study reveals the complexity of TME within the LG-ESS and highlights the role that tumor cell subpopulations play in disease progression and patient prognosis. The identification of FGF12 as a prognostic biomarker suggests a new approach for the personalized treatment and prognosis monitoring of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e49616fef8441ab9c968f5c960e3e63e7204d31e" target='_blank'>
              Single-cell RNA-seq reveals FGF12 as a prognostic biomarker in low-grade endometrial stromal sarcoma
              </a>
            </td>
          <td>
            Yu Miao, Meng Dong, Qiyin Zhou, Julia Thiel, Na Li, Ying Cai, Dan Yuan, Haitao Wang, Su-Han Jin, Hua Yang, Jinjing Wang, Benjamin Frey, Udo S. Gaipl, Hu Ma, Jianghu Zhou
          </td>
          <td>2024-11-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background The adherent-to-suspension transition (AST) describes how solid tumor cells reprograms their anchorage dependency via defined hematopoietic transcriptional regulators to disseminate into circulating tumor cells (CTCs). It remains unclear, however, whether the dissemination and colonization processes require dynamic plasticity of AST factor expression in CTCs to permit the completion of the metastatic cascade. Methods We investigated AST factor-mediated adherent-suspension plasticity (ASP) in a newly developed cell-based dissemination assay. Here we identified a rare population of CTC-like cells spontaneously suspended from highly confluent breast cancer cells under chronic stress conditions that mimic the tumor microenvironment (TME). We investigated the oscillatory dynamics of AST factor expression in these CTC-like cells and their role in dissemination and colonization. Furthermore, we performed a single-cell transcriptomic analysis of matched primary tumors, CTCs, and metastatic lesions from newly enrolled de novo metastatic breast cancer patients, focusing on the plasticity of AST factor expression during the metastatic cascade in vivo. Results Using data obtained from the dissemination assay and de novo metastatic breast cancer patients, we have demonstrated the dynamic regulation of AST factor expression during the dissemination and colonization of CTCs both in vitro and in vivo. From our dissemination assay of breast cancer cell lines, we show that AST factor induction is required for the reprogramming of anchorage dependency and spontaneous detachment of CTC-like cells. We also show that subsequent AST factor suppression is required for the reattachment of CTCs during colonization. Single-cell transcriptome analyses in matched specimens of primary tumor, CTC, and metastatic lesions obtained from de novo metastatic breast cancer patients confirmed the oscillatory dynamics of AST factors and related gene signatures related to the adherent-suspension plasticity. Conclusion Our results show that the reprogramming of anchorage dependency in solid tumor cells via AST factor-mediated ASP plays a critical role in the dissemination and colonization of CTCs. These findings highlight the potential of targeting AST factors to develop effective anti-metastatic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24dbe2635c48e91654f5538698e4e2b76cbd216e" target='_blank'>
              Adherent-suspension plasticity promotes the dissemination and colonization of circulating tumor cells
              </a>
            </td>
          <td>
            Hyunbin D. Huh, Yujin Sub, Jee Hung Kim, Hannah Lee, Jongwook Oh, Dong Ki Lee, Jong Hwa Byun, Woo Yang Pyun, S. Bae, Sehyung Pak, Hai-Chon Lee, Yu Suk Choi, Junjeong Choi, Joon Jeong, H. Gee, Hyun Woo Park
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Immune checkpoint inhibitors such as anti-PD-1 antibodies (aPD1) can be effective in treating advanced cancers. However, many patients do not respond and the mechanisms underlying these differences remain incompletely understood. In this study, we profile a cohort of patients with locally-advanced or metastatic basal cell carcinoma undergoing aPD-1 therapy using single-cell RNA sequencing, high-definition spatial transcriptomics in tumors and draining lymph nodes, and spatial immunoreceptor profiling, with long-term clinical follow-up. We find that successful responses to PD-1 inhibition are characterized by an induction of B-cell receptor (BCR) clonal diversity after treatment initiation. These induced BCR clones spatially co-localize with T-cell clones, facilitate their activation, and traffic alongside them between tumor and draining lymph nodes to enhance tumor clearance. Furthermore, we validated aPD1-induced BCR diversity as a predictor of clinical response in a larger cohort of glioblastoma, melanoma, and head and neck squamous cell carcinoma patients, suggesting that this is a generalizable predictor of treatment response across many types of cancers. We discover that pre-treatment tumors harbor a characteristic gene expression signature that portends a higher probability of inducing BCR clonal diversity after aPD-1 therapy, and we develop a machine learning model that predicts PD-1-induced BCR clonal diversity from baseline tumor RNA sequencing. These findings underscore a dynamic role of B cell diversity during immunotherapy, highlighting its importance as a prognostic marker and a potential target for intervention in non-responders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c13f381588d31078af3cc3cb23453357caf14bf3" target='_blank'>
              Induced B-Cell Receptor Diversity Predicts PD-1 Blockade Immunotherapy Response
              </a>
            </td>
          <td>
            Yonglu Che, Jinwoo Lee, Farah Abou-Taleb, Kerri E. Rieger, Ansuman T. Satpathy, A. L. S. Chang, Howard Y. Chang
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e01bd46496bf7ca9aab2045ddff751c6da9e8f4" target='_blank'>
              Tumor microenvironment remodeling after neoadjuvant chemoradiotherapy in local advanced rectal cancer revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Wen-jing Su, Yuhang Ling, Xiaodong Yang, Yong Wu, Chungen Xing
          </td>
          <td>2024-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) comprises diverse cell types whose interactions are crucial for cancer progression. Traditional bulk tissue analyses mask cell-type-specific epigenetic changes that may drive disease progression. We used DNA methylation data and a reference- based cell-type deconvolution algorithm — hierarchical tumor immune microenvironment epigenetic deconvolution (HiTIMED) — to quantify cell types in the TME. Then, we employed CellDMC, a statistical interaction testing framework that can use cell type proportions in conjunction with differential methylation analysis to identify cell-specific differential methylation. Genome-scale DNA methylation profiles from breast cancer tissue (n=609) and normal breast tissue (n=230) samples were accessed in GEO and GTEx. Our novel approach leverages bulk DNA methylation data to infer cell-type-specific epigenetic landscapes, circumventing challenges and limitations associated with direct single-cell methylation profiling, such as high costs and technical variability. Reference-based computational tools allow us to dissect complex tissue compositions and reveal distinct methylation patterns and pathways that may be unique to individual cell types, providing insights that are not discernible from traditional bulk tumor analysis approaches. Compared with nontumor normal samples, we identified significant DNA methylation alterations (FDR≤0.05) across tumor, stromal, endothelial, epithelial, lymphocytes, and myeloid cells in breast tumors. Tumor cells exhibited the highest number, 8,510, differentially methylated cytosines (DMCs) with 1,966 of these unique to tumor cells. Stromal, endothelial, and epithelial cells also showed considerable methylation alterations, with a total of 530, 440, 339 DMCs, respectively, with 5 DMCs unique to epithelial cells. Despite lower immune cell proportions in our samples, we identified 104 lymphocyte specific DMCs and 185 myeloid specific DMCs, with 14 and 20 DMCs unique to these cell types respectively. The unique DMCs identified in each cell type suggest distinct epigenetic landscapes that may drive specific cellular functions and interactions within the TME. Importantly, Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analysis of the annotated cell-type-specific DMCs revealed distinct pathways that were not apparent in the pathway analysis of bulk tumor versus normal tissue, underscoring the value of examining epigenetic alterations at the cell-type level. Our findings highlight the opportunity to leverage DNA-based cell typing with DNA methylation data to discern cell-specific molecular alterations. By focusing on cell-type-specific alterations, we can better understand the epigenetic mechanisms driving TME crosstalk, such as the activation of stromal cells and the modulation of immune responses by tumor cells. Our findings underscore the need for targeted therapeutic strategies that consider the unique epigenetic landscapes of individual cell types within the TME, offering new avenues for improving cancer treatment outcomes.
 Citation Format: Barbara Karakyriakou, Brock C. Christensen, Lucas A. Salas. Identification of cell-specific DNA methylation alterations in the breast tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae2a87bb3a64a66fe890b4d4d581317faf06571" target='_blank'>
              Abstract B033: Identification of cell-specific DNA methylation alterations in the breast tumor microenvironment
              </a>
            </td>
          <td>
            Barbara Karakyriakou, Brock C. Christensen, Lucas A Salas
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e6a9e6b43769b80db465f9dd9f678e6c40ef5c" target='_blank'>
              Single cell and spatial transcriptomics highlight the interaction of club-like cells with immunosuppressive myeloid cells in prostate cancer
              </a>
            </td>
          <td>
            A. Kiviaho, Sini K. Eerola, H. Kallio, M. K. Andersen, Miina Hoikka, A. Tiihonen, I. Salonen, Xander Spotbeen, A. Giesen, C. T. Parker, S. Taavitsainen, Olli Hantula, Mikael Marttinen, Ismaïl Hermelo, Mazlina Ismail, Elise Midtbust, M. Wess, W. Devlies, Abhibhav Sharma, S. Krossa, Tomi Häkkinen, Ebrahim Afyounian, K. Vandereyken, Sam Kint, J. Kesseli, T. Tolonen, T.L.J. Tammela, T. Viset, Øystein Størkersen, G. Giskeødegård, M. B. Rye, T. Murtola, A. Erickson, Leena Latonen, G. Bova, Ian G. Mills, S. Joniau, J. Swinnen, Thierry Voet, T. Mirtti, Gerhardt Attard, Frank Claessens, T. Visakorpi, K. Rautajoki, May-Britt Tessem, Alfonso Urbanucci, M. Nykter
          </td>
          <td>2024-11-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. A better understanding of the temporal dynamics and specific pathways leading into and out of the persister state is needed to identify strategies to prevent treatment failure. Using spatial transcriptomics in patient-derived xenograft models, we captured clonal lineage evolution during treatment. The persister state showed increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients. Phylogenetic tracing identified intrinsic and acquired resistance mechanisms (e.g., dual specific phosphatases, reticulon-4, and CDK2) and suggested specific temporal windows of potential therapeutic susceptibility. Deep learning-enabled analysis of histopathological slides revealed morphological features correlating with specific cell states, demonstrating that juxtaposition of transcriptomics and histological data enabled identification of phenotypically distinct populations from using imaging data alone. In summary, this study defined state change and lineage selection during melanoma treatment with spatiotemporal resolution, elucidating how choice and timing of therapeutic agents will impact the ability to eradicate resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d694b41718972dca1b5b8aa26a4498984d65d307" target='_blank'>
              Spatiotemporal Profiling Defines Persistence and Resistance Dynamics During Targeted Treatment of Melanoma.
              </a>
            </td>
          <td>
            Jill C. Rubinstein, Sergii Domanskyi, Todd B Sheridan, Brian Sanderson, SungHee Park, Jessica Kaster, Haiyin Li, Olga Anczuków, M. Herlyn, Jeffrey H. Chuang
          </td>
          <td>2024-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, Mirjam Judith Feldkamp, C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af37e793e7291e5ec4163cfb7481755bd374769" target='_blank'>
              Spatial transcriptomics reveals unique metabolic profile and key oncogenic regulators of cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Limin Zhou, Jiejie Liu, Peipei Yao, Xing Liu, Fei Chen, Yu Chen, Li Zhou, Chao Shen, You Zhou, Xin Du, Junbo Hu
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Epithelial-to-mesenchymal transition (EMT) is a reversible process during which epithelial cells lose their epithelial characteristics and acquire mesenchymal features, that are important for tumor initiation, progression, stemness, metastasis, and therapy resistance. Different EMT states, each with distinct functional properties, have been identified across various tumor models. These EMT states reside within specific physical niches, characterized by unique stromal cell populations. Understanding the cellular interactions within the tumor microenvironment (TME) and their influence on EMT could lead to novel therapeutic strategies. To understand how the TME regulates the distinct tumor states, we performed single-cell RNA sequencing of the tumor cells and their TME in mouse cancer models presenting EMT. We identified distinct subsets of cancer-associated fibroblasts (CAFs), macrophages, neutrophils, endothelial cells, pericytes, adipocytes, dendritic cells, mast cells, Schwann cells, and T cells in different tumor types. To unravel how TME-tumor cell interactions regulates EMT, we used different bioinformatic approaches to identify ligand-receptor pairs that are likely to organize the tumor niche and to regulate tumor transition states. In vitro treatment of tumor cells with some of these ligands demonstrated their EMT-promoting effects. These findings provide novel mechanistic insights on the crosstalk between tumor cells and the surrounding TME, that may have important implications for anti-cancer therapy.
 Citation Format: Cédric Blanpain. Regulation of EMT tumor states by stromal cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr IA019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ded367efe26987fbf64cb06b553ba3a3b5b7a19" target='_blank'>
              Abstract IA019: Regulation of EMT tumor states by stromal cells
              </a>
            </td>
          <td>
            Cédric Blanpain
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e3304563df16fc68d3eeffdd08dc060638480f4" target='_blank'>
              Single-cell transcriptomic landscape deciphers olfactory neuroblastoma subtypes and intra-tumoral heterogeneity.
              </a>
            </td>
          <td>
            Jingyi Yang, Xiaole Song, Huankang Zhang, Quan Liu, Ruoyan Wei, Luo Guo, Cuncun Yuan, Fu Chen, K. Xue, Yuting Lai, Li Wang, Junfeng Shi, Chengle Zhou, Juan Wang, Yingxuan Yu, Qibing Mei, Li Hu, Huan Wang, Chen Zhang, Qianqian Zhang, Houyong Li, Ye Gu, Weidong Zhao, Huapeng Yu, Jingjing Wang, Zhuo-fu Liu, Han Li, S. Zheng, Juan Liu, Lu Yang, Wanpeng Li, Rui Xu, Jiani Chen, Yumin Zhou, Xian-kui Cheng, Yiqun Yu, Dehui Wang, Xicai Sun, Hongmeng Yu
          </td>
          <td>2024-11-14</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de8ac06c638611539d33f6ef5fdb316eabe9f02c" target='_blank'>
              Spatio-temporal transcriptomics of chromothriptic SHH-medulloblastoma identifies multiple genetic clones that resist treatment and drive relapse
              </a>
            </td>
          <td>
            Ilia Kats, Milena Simovic-Lorenz, Hannah Sophia Schreiber, Pooja Sant, Jan-Philipp Mallm, Verena Körber, Albert Li, Pravin Velmurugan, Sophie Heuer, Luisa Kües, Frauke Devens, M. Sill, M. Jugold, Mahmoud Moustafa, Amir Abdollahi, Frank Winkler, A. Korshunov, Stefan M Pfister, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220a065de9e40d9837ef9539b8c30437e0e009d" target='_blank'>
              Advanced single-cell and spatial analysis with high-multiplex characterization of circulating tumor cells and tumor tissue in prostate cancer: Unveiling resistance mechanisms with the CoDuCo in situ assay
              </a>
            </td>
          <td>
            Lilli Bonstingl, Margret Zinnegger, Katja Sallinger, Karin Pankratz, Christin-Therese Müller, Elisabeth Pritz, Corinna Odar, C. Skofler, Christine Ulz, Lisa Oberauner-Wappis, Anatol Borrás-Cherrier, Višnja Somođi, Ellen Heitzer, T. Kroneis, T. Bauernhofer, Amin El-Heliebi
          </td>
          <td>2024-11-16</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly disease. Over recent decades, a better understanding of the biological mechanisms implicated in its pathogenesis has led to the development of targeted therapies and immunotherapy, resulting in improvements in patient outcomes. To enhance our understanding of lung cancer tumor biology and advance towards precision oncology, we need a comprehensive tumor profile. In recent years, novel in situ spatial multiomics approaches have emerged, offering a more detailed view of the spatial location of tumor and tumor microenvironment cells. These advancements in in situ profiling may unveil further molecular and immune mechanisms in tumor biology that will lead to the discovery of biomarkers for treatment prediction and prognosis. In this review, we provide an overview of current and emerging pathology-based approaches for spatial immune profiling in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e58f728c302c9eea20cd5a44be684287b92352" target='_blank'>
              Novel Spatial Approaches to Dissect the Lung Cancer Immune Microenvironment
              </a>
            </td>
          <td>
            Idania Lubo, Sharia D Hernandez, I. Wistuba, L. S. Solis Soto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Breast tumors comprise several sub-populations of cells, so-called “clones”, with specific cellular and molecular characteristics. Some cancer clones can be more ‘aggressive’ than others in particular tumor microenvironments (TMEs), contributing to metastatic progression and drug resistance. Understanding the specific genes/pathways deregulated in these aggressive cancer cells is essential for developing new tailored treatment strategies. To accomplish this, we optimized the use of a lentiviral-based barcoding technique (Berthelet, J., et.al., Sci Adv 2021, Serrano, A., et.al., Nat Rev Cancer 2022) where cancer cells from cell lines and patient-derived xenografts are tagged with optical barcodes and transplanted into the mammary fat pads of immunodeficient mice. Upon resection of tumors and collection of metastatic organs at the experimental endpoint, we analyzed clonal fate and fitness using flow cytometry and imaging. The subsequent molecular characterization of individual clones was determined by bulk and single-cell RNA sequencing. We also leverage the use of these barcoded models to investigate the mechanisms of innate and acquired resistance to Sacituzumab Govitecan (SG), a drug recently approved for the treatment of patients with advanced breast cancer. Our studies revealed unique gene signatures and pathways associated with preferential organ tropism (in the lungs, liver and brain TMEs) and SG resistance. This comprehensive analysis, currently validated using autopsy samples and patient databases, will support the design of new predictive markers and targeted therapies for metastatic breast cancer.
 Citation Format: Sreeja Reddy Gadipally, Jean Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, Caroline Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, Charafe Jauffret Emmanuelle, Melissa Davis, Robin L. Anderson, Luciano Martelotto, Belinda Yeo, Delphine Merino. Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528d6539434045b291c3a0faa67fa6d3c29fbc80" target='_blank'>
              Abstract B005: Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance
              </a>
            </td>
          <td>
            Sreeja Gadipally, J. Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, C. Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, C. Emmanuelle, Melissa J Davis, Robin L Anderson, Luciano Martelotto, B. Yeo, Delphine Merino
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Tumor microenvironments (TMEs) encompass complex ecosystems of cancer cells, infiltrating immune cells, and diverse cell types. Intercellular and intracellular signals within the TME significantly influence cancer progression and therapeutic outcomes. Although computational tools are available to study TME interactions, explicitly modeling tumor progression across different cancer types remains a challenge. Methods This study introduces a comprehensive framework utilizing single-cell RNA sequencing (scRNA-seq) data within a multilayer network model, designed to investigate molecular changes across glioma progression stages. The heterogeneous, multilayered network model replicates the hierarchical structure of biological systems, from genetic building blocks to cellular functions and phenotypic manifestations. Results Applying this framework to glioma scRNA-seq data allowed complex network analysis of different cancer stages, revealing significant ligand‒receptor interactions and key ligand‒receptor-transcription factor (TF) axes, along with their associated biological pathways. Differential network analysis between grade III and grade IV glioma highlighted the most critical nodes and edges involved in interaction rewiring. Pathway enrichment analysis identified four essential genes—PDGFA (ligand), PDGFRA (receptor), CREB1 (TF), and PLAT (target gene)—involved in the Receptor Tyrosine Kinases (RTK) signaling pathway, which plays a pivotal role in glioma progression from grade III to grade IV. Discussion These genes emerged as significant features for machine learning in predicting glioma progression stages, achieving 87% accuracy and 93% AUC in a 3-year survival prediction through Kaplan-Meier analysis. This framework provides deeper insights into the cellular machinery of glioma, revealing key molecular relationships that may inform prognosis and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d112e50918401623433091b937b2d39ad5c4da" target='_blank'>
              Integrative analysis of single-cell transcriptomic and multilayer signaling networks in glioma reveal tumor progression stage
              </a>
            </td>
          <td>
            Fereshteh Fallah Atanaki, L. Mirsadeghi, Mohsen Riahi Manesh, K. Kavousi
          </td>
          <td>2024-11-13</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction Effective treatment of breast cancer remains a formidable challenge, partly due to our limited understanding of the complex microenvironmental factors that contribute to disease pathology. Among these factors are tissue-resident perivascular cells, which play crucial roles in shaping vascular basement membranes, maintaining vessel integrity, and communicating with adjacent endothelial cells. Despite their essential functions, perivascular cells have been relatively overlooked. Identifying them by immunostaining has been challenging due to their low abundance, inherent heterogeneity, and shared marker expression with other cell types. These challenges have hindered efforts to purify pericytes and generate primary cell models for studying their biology. Methods Using a recently developed FACS method, we successfully identified and purified each cell type from breast tissues, allowing us to deep-sequence their transcriptomes and generate primary cell models of each cell type—including pericytes. Here, we used these data to analyze cell-type-specific gene expression in tumors, which revealed a strong association between pericyte-specific genes and breast cancer patient mortality. To explore this association, we defined the heterogeneity of breast pericytes using single-cell RNA sequencing and identified a broad marker for visualizing perivascular cells in breast tumors. Results Remarkably, we discovered perivascular cells dissociated from vessels and emerged as a dominant mesenchymal cell type in a subset of breast tumors that contrasted with their normal perivascular location. Moreover, when we purified pericytes from the breast and cultured them alongside breast tumor cells, we discovered that they induced rapid tumor cell growth significantly greater than isogenic fibroblast controls. Discussion These findings identify perivascular cells as a key microenvironmental factor in breast cancer, highlighting the critical need for further research to explore their biology and identify specific stimulatory mechanisms that could be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920638149619a80dd215db038826c980f92e7f53" target='_blank'>
              Breast pericytes: a newly identified driver of tumor cell proliferation
              </a>
            </td>
          <td>
            Katelyn Del Toro, Y. Licon-Munoz, William Crabtree, Tristan Oper, Christine Robbins, William C. Hines
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f233ae172c7106f684ef62640304f7d51d73186" target='_blank'>
              Integrative mapping of human CD8+ T cells in inflammation and cancer.
              </a>
            </td>
          <td>
            Ziwei Xue, Lize Wu, Ruonan Tian, Bing Gao, Yu Zhao, Bing He, Di Sun, Bingkang Zhao, Yicheng Li, Kaixiang Zhu, Lie Wang, Jianhua Yao, Wanlu Liu, Linrong Lu
          </td>
          <td>2024-11-29</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The dynamic interplay between tumor cells and γδT cells within the tumor microenvironment (TME) significantly influences disease progression and immunotherapy outcome. Here, we delved into the modulation of γδT-cell activation by tumor cell ligands CD112 and CD155, which interact with the activating receptor DNAM-1 on γδT cells. Spatial and single cell RNA sequencing (scRNA-seq), as well as spatial metabolome analysis, from neuroblastoma (NB) revealed that the expression levels and localization of CD112 and CD155 varied across and within tumors, correlating with differentiation status, metabolic pathways, and ultimately disease prognosis and patient survival. Both in vivo tumor xenograft experiments and in vitro co-culture experiments demonstrated that a high CD112/CD155 expression ratio in tumors enhanced γδT-cell-mediated cytotoxicity, while a low-ratio fostered tumor resistance. Mechanistically, CD112 sustained DNAM-1-mediated γδT-cell activation, whereas CD155 downregulated DNAM-1 expression via TRIM21-mediated ubiquitin proteasomal degradation. By interacting with tumor cells differentially expressing CD112 and CD155, intratumoral γδT cells exhibited varying degrees of activation and DNAM-1 expression, representing three major functional subsets. This study underscores the complexity of tumor-immune crosstalk, offering insights into how tumor heterogeneity shapes the immune landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0f304618ec33893cbfa169baffb49b70570809" target='_blank'>
              Spatial and Single Cell Analyses Reveal Heterogeneity of DNAM-1 Receptor-Ligand Interactions that Instructs Intratumoral γδT-Cell Activity.
              </a>
            </td>
          <td>
            Xiaolin Wang, Hui Wang, Zhengjing Lu, Xiangjun Liu, Wenjia Chai, Wei Wang, Jun Feng, Shen Yang, Wei Yang, Haiyan Cheng, Chenghao Chen, Shihan Zhang, N. Sun, Qiaoyin Liu, Qiliang Li, Wenqi Song, Fang Jin, Qi Zeng, Shengcai Wang, Yan Su, Huanmin Wang, Xinxin Ni, Jingang Gui
          </td>
          <td>2024-11-08</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Background & Objective: Prostate cancer (PCa) cells frequently disseminate to bone where they can remain dormant for extended periods. The precise cellular and molecular mechanisms responsible remain largely unknown. The advent of spatial transcriptomics provides a unique opportunity to study key bone microenvironment-dormant PCa cell interactions in situ. Here, we developed a novel immunocompetent in vivo model of PCa dormancy in bone and employed the technology to address this major gap in knowledge. Methods: We established an in vitro protocol to induce dormancy in multiple PCa cell lines, including the C57BL/6 murine PCa cell line, RM1. Isogenic active and dormant PCA cells underwent bulk RNA Seq. We used an intra- iliac artery (IIA) approach to allow niche-specific seeding of dormant PCa within the bone. Spatial transcriptomics (10X Visium) was performed on tissue sections and R packages (Seurat & CellChat) were used for data analysis. Results: We validated in vitro PCa dormancy via cell cycle analysis, membrane dye retention and ERK/p38 ratio. RNASeq analysis of dormant PCa cells (vs. active) revealed upregulation of immunomodulatory genes such as, GDF15, BDNF, A2M, VGF, NLRP1, and CD55. C57BL/6 immunocompetent mice were inoculated with active or dormant RM1 cells via IIA. Majority of active RM1 mice reached endpoint within 15 days. In contrast, mice bearing dormant RM1 (D-RM1) had extended survival and were removed from study after 30 days. Histological analysis revealed solitary D-RM1 (panCK+/Ki67-/CC3-) on endosteal surfaces compared to macrometastases (high Ki67+) in the active RM1 group. Using IIA PBS injected tumor naïve bones as a control, we applied spatial transcriptomics (10X Visium) to interrogate the microenvironment surrounding active and dormant RM1 PCa cells. Bioinformatic analyses demonstrated higher expression of B cell markers (CD19, CD79a, CD79b, Ighm, and Pax5) and enhanced B cell activation/antigen phagocytosis signature in the D- RM1 group compared to other groups. We validated the presence of CD19 B-cells in proximity to the D-RM1 cells in subsequent tissue sections. In contrast, B cells were located less frequently and to the periphery of active RM1 metastases. Inference analysis using CellChat also identified a significant and distinct signaling pattern between CD55 and CD97 in the B cell enriched clusters in D-RM1 group. CD55/CD97 interaction is a noted mediator of B cell homing. These data suggest a potential role for distinct populations of B cells during the establishment of PCa dormancy in bone. Conclusion: Spatial transcriptomic analysis of the dormant PCa bone microenvironment reveals an enrichment of CD19+ B cells with a distinct signaling pattern. Given their known roles in immune tolerance we posit that infiltrating B cells may play a role in protecting and sustaining newly disseminated dormant PCa cells from immune mediated elimination.
 Citation Format: Mostafa M. Nasr, Ryan T. Bishop, Haley du Bois, Tao Li, Jeremy S. Frieling, Conor C. Lynch. Spatial transcriptomics reveals a distinct B cell signature in the dormant prostate cancer bone microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1da73f9c20af7ebfb3e86b110c704de45b40046e" target='_blank'>
              Abstract B044: Spatial transcriptomics reveals a distinct B cell signature in the dormant prostate cancer bone microenvironment
              </a>
            </td>
          <td>
            Mostafa M. Nasr, Ryan T Bishop, Haley du Bois, Tao Li, Jeremy S. Frieling, Conor C Lynch
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Existing single-cell clustering methods are based on gene expressions that are susceptible to dropout events in single-cell RNA sequencing (scRNA-seq) data. To overcome this limitation, we proposed a pathway-based clustering method for single cells (scPathClus). scPathClus first transforms single-cell gene expression matrix into pathway enrichment matrix and generates its latent feature matrix. Based on the latent feature matrix, scPathClus clusters single cells using the method of community detection. Applying scPathClus to PDAC scRNA-seq datasets, we identified two types of cancer-associated fibroblasts (CAFs), termed csCAFs and gapCAFs, which highly expressed complement system and gap junction-related pathways, respectively. Spatial transcriptome analysis revealed that gapCAFs and csCAFs are located at cancer and non-cancer regions, respectively. Pseudotime analysis suggest a potential differentiation trajectory from csCAFs to gapCAFs. Bulk transcriptome analysis showed that gapCAFs-enriched tumors are more endowed with tumor-promoting characteristics and worse clinical outcomes, while csCAFs-enriched tumors confront stronger antitumor immune responses. Compared to established CAF subtyping methods, this method displays better prognostic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e5196bb420146fb22bc069bed5ff6f7bc8606d" target='_blank'>
              Pathway-based clustering identifies two subtypes of cancer-associated fibroblasts associated with distinct molecular and clinical features in pancreatic ductal carcinoma
              </a>
            </td>
          <td>
            Hongjing Ai, R. Nie, Xiaosheng Wang
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa9e49413db565400c66d264638af2b81a7c480" target='_blank'>
              Single cell-spatial transcriptomics and bulk multi-omics analysis of heterogeneity and ecosystems in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jiazhou Ye, Yan Lin, Zhiling Liao, Xing Gao, Cheng Lu, Lu Lu, Julu Huang, Xi Huang, Shilin Huang, Hongping Yu, Tao Bai, Jie Chen, Xiaobo Wang, Mingzhi Xie, Min Luo, Jinyan Zhang, Feixiang Wu, Guo-bin Wu, Liang Ma, Bangde Xiang, Lequn Li, Yongqiang Li, Xiaoling Luo, R. Liang
          </td>
          <td>2024-11-15</td>
          <td>NPJ Precision Oncology</td>
          <td>2</td>
          <td>19</td>
        </tr>

        <tr id="Despite thorough characterizations of cellular compositions within the breast tumor microenvironment (TME), their implications for disease progression and patient prognosis are still poorly understood. Unraveling these effects is vital for identifying potential targets to improve treatment outcomes. In this study, we devised an explainable machine learning (XML) pipeline to scrutinize the associations between TME cellular constituents and relapse-free survival (RFS). By applying our pipeline to estimated cell fractions in the METABRIC and TCGA datasets and comparing these results with associations to pathological complete response (pCR) after neoadjuvant chemotherapy (NAC), we created a comprehensive catalog of the TME’s role based on 5000 patient samples. Our findings reveal an unexpected dichotomy in which macrophages correlate positively with pCR but negatively with RFS, particularly within estrogen receptor-positive (ER+) and Luminal A and B (LumA/B) cancer subtypes. We show that this pattern is driven by heterogeneity in breast tumors characterized by increasing levels of macrophage infiltration. Through imaging mass cytometry (IMC) analysis, we discovered that macrophages tend to accumulate in the vicinity of HLA-ABChi epithelial cells as their frequency increases in tumor tissues and also express elevated levels of HLA-ABC protein. Combining IMC with single-cell RNA sequencing (scRNA-seq) data, we uncovered a significant association between these HLA-ABChi macrophages and regulatory and exhausted T cells (TReg and TEx), suggesting their involvement in immune suppression, likely by creating a chronically activated immunosuppressive TME. Subsequent cell-cell communication analysis predicted interactions between HLA-ABChi macrophages and TEx cells via the ligands SIGLEC9, ALCAM, and CSF1, and with TReg cells through APP, ANGPTL4, and SIGLEC9 signaling. Considering the clinical relevance of macrophages in ER+ (LumA/B) subtypes, our research enhances the characterization of macrophage-driven immune suppression in these tumors and identifies potential targets for immunomodulatory strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ef5aa61ac57e0a39d6c5657cd0986b7fc829d8" target='_blank'>
              Integrated multiomics analysis unveils how macrophages drive immune suppression in breast tumors and affect clinical outcomes
              </a>
            </td>
          <td>
            Y. Azimzade, Mads Haugland Haugen, Vessela N Kristensen, A. Frigessi, Alvaro Köhn-Luque
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69505c7dd30d328b855b8e26bb2a91fa4c9f9efa" target='_blank'>
              Computer-aided analysis reveals metallothionein-positive cancer-associated fibroblasts promote angiogenesis in gastric adenocarcinoma
              </a>
            </td>
          <td>
            Xiaolong Jin, Yu Tian, Haoran Zhu, Yuewen Sun, Zhenxing Zhang
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tubo-ovarian high-grade serous carcinoma (HGSC) is the most lethal gynecological malignancy and frequently responds to platinum-based chemotherapy because of common genetic and somatic impairment of DNA damage repair (DDR) pathways. The mechanisms of clinical platinum resistance are diverse and poorly molecularly defined. Consequently, there are no biomarkers or medicines that improve patient outcomes. Herein we use single cell mass cytometry (CyTOF) to systematically evaluate the phosphorylation and abundance of proteins known to participate in the DNA damage response (DDR). Single cell analyses of highly characterized HGSC cell lines that phenocopy human patients show that cells with comparable levels of intranuclear platinum, a proxy for carboplatin uptake, undergo different cell fates. Unsupervised analyses revealed a continuum of DDR responses. Decompositional methods were used to identify eight distinct protein modules of carboplatin resistance and sensitivity at single cell resolution. CyTOF profiling of primary and secondary platinum-resistance patient models shows that a complex DDR sensitivity module is strongly associated with response, suggesting it as a potential tool to clinically characterize complex drug resistance phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1700b92ee74968435b1e4e6c96fff15f73c54ccf" target='_blank'>
              Coordinated protein modules define DNA damage responses to carboplatin at single cell resolution in human ovarian carcinoma models
              </a>
            </td>
          <td>
            Jacob S. Bedia, Ying-Wen Huang, Antonio Delgado Gonzalez, Veronica D. Gonzalez, I. Funingana, Zainab Rahil, Alyssa Mike, Alexis Lowber, M. Vias, Alan Ashworth, James D. Brenton, W. Fantl
          </td>
          <td>2024-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a leading cause of cancer-related deaths globally. The heterogeneity of the tumor microenvironment significantly influences patient prognosis, while the diversity of tumor cells shapes its unique characteristics. A comprehensive analysis of the molecular profile of tumor cells is crucial for identifying novel molecular targets for drug sensitivity analysis and for uncovering the pathophysiological mechanisms underlying CRC. Methods We utilized single-cell RNA sequencing technology to analyze 13 tissue samples from 4 CRC patients, identifying key cell types within the tumor microenvironment. Intercellular communication was assessed using CellChat, and a risk score model was developed based on eight prognostic genes to enhance patient stratification for immunotherapeutic approaches. Additionally, in vitro experiments were performed on DLX2, a gene strongly associated with poor prognosis, to validate its potential role as a therapeutic target in CRC progression. Results Eight major cell types were identified across the tissue samples. Within the tumor cell population, seven distinct subtypes were recognized, with the C0 FXYD5+ tumor cells subtype being significantly linked to cancer progression and poor prognosis. CellChat analysis indicated extensive communication among tumor cells, fibroblasts, and immune cells, underscoring the complexity of the tumor microenvironment. The risk score model demonstrated high accuracy in predicting 1-, 3-, and 5-year survival rates in CRC patients. Enrichment analysis revealed that the C0 FXYD5+ tumor cell subtype exhibited increased energy metabolism, protein synthesis, and oxidative phosphorylation, contributing to its aggressive behavior. In vitro experiments confirmed DLX2 as a critical gene associated with poor prognosis, suggesting its viability as a target for improving drug sensitivity. Conclusion In summary, this study advances our understanding of CRC progression by identifying critical tumor subtypes, molecular pathways, and prognostic markers that can inform innovative strategies for predicting and enhancing drug sensitivity. These findings hold promise for optimizing immunotherapeutic approaches and developing new targeted therapies, ultimately aiming to improve patient outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8647de1e0192ab64820f770599e15f3edf4689c3" target='_blank'>
              Innovative strategies to optimise colorectal cancer immunotherapy through molecular mechanism insights
              </a>
            </td>
          <td>
            Quanjun Lin, Zhiqiang Wang, Jue Wang, Ming Xu, Xinyi Zhang, Peng Sun, Yihang Yuan
          </td>
          <td>2024-12-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit enhanced resistance to standard of care treatments and have significantly worse overall survival (OS) compared to patients with classical subtype enriched tumors. It is important to develop genomic resources, enabling identification of novel putative targets in a statistically rigorous manner.


EXPERIMENTAL DESIGN
We compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 patient samples, aggregated from publicly available raw data. We mapped cell-type specific scRNAseq gene signatures in bulk RNAseq (n=744) and spatial transcriptomics (ST) (n=22) and performed validation using multiplex immunostaining.


RESULTS
Analysis of tumor cells from our scRNAseq atlas revealed nine distinct populations, two of which aligned with the basal subtype, correlating with worse OS in bulk RNAseq. Deconvolution identified one of the basal populations to be the predominant tumor subtype in non-dissociated ST tissues and in vitro tumor cell and patient-derived organoid lines. We discovered a novel enrichment and spatial association of CXCL10+ cancer associated fibroblasts (CAFs) with basal tumor cells. We identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, suggesting a relationship between these cell types in PDAC tumor microenvironment.


CONCLUSIONS
We show that our scRNAseq atlas (700,000 cells), integrated with ST data, has increased statistical power and is a powerful resource, allowing for expansion of current subtyping paradigms in PDAC. We uncovered a novel signaling niche marked by CXCL10+ CAFs and basal tumor cells that could be explored for future targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07269a9cb871b60ea57ca5b2a01a49e35b66cc9b" target='_blank'>
              Human pancreatic cancer single cell atlas reveals association of CXCL10+ fibroblasts and basal subtype tumor cells.
              </a>
            </td>
          <td>
            Ian M Loveless, Samantha B. Kemp, Kailee M Hartway, Jacob T. Mitchell, Yuesong Wu, Samuel D Zwernik, D. Salas-Escabillas, Sydney Brender, Madison George, Yetunde Makinwa, Thais Stockdale, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombwell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, E. Fertig, L. T. Kagohara, Brian Theisen, Howard C Crawford, Nina G. Steele
          </td>
          <td>2024-12-05</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Muscle-invasive bladder cancer (MIBC) is a prevalent cancer characterized by molecular and clinical heterogeneity. Assessing the spatial heterogeneity of the MIBC microenvironment is crucial to understand its clinical significance. Methods In this study, we used imaging mass cytometry (IMC) to assess the spatial heterogeneity of MIBC microenvironment across 185 regions of interest in 40 tissue samples. We focused on three primary parameters: tumor (T), leading-edge (L), and nontumor (N). Cell gating was performed using the Cytobank platform. We calculated the Euclidean distances between cells to determine cellular interactions and performed single-cell RNA sequencing (scRNA-seq) to explore the molecular characteristics and mechanisms underlying specific fibroblast (FB) clusters. scRNA-seq combined with spatial transcriptomics (ST) facilitated the identification of ligand–receptor (L–R) pairs that mediate interactions between specific FB clusters and endothelial cells. Machine learning algorithms were used to construct a prognostic gene signature. Results The microenvironments in the N, L, and T regions of MIBC exhibited spatial heterogeneity and regional diversity in their components. A distinct FB cluster located in the L region—identified as S3—is strongly associated with poor prognosis. IMC analyses demonstrated a close spatial association between S3 and endothelial cells, with S3-positive tumors exhibiting increased blood vessel density and altered vascular morphology. The expression of vascular endothelial growth factor receptor and active vascular sprouting were significant in S3-positive tumors. scRNA-seq and ST analyses indicated that the genes upregulated in S3 were associated with angiogenesis. NOTCH1–JAG2 signaling pathway was identified as a significant L–R pair specific to S3 and endothelial cell interactions. Further analysis indicated that YAP1 was a potential regulator of S3. Machine learning algorithms and Gene Set Variation Analysis were used to establish an S3-related gene signature that was associated with the poor prognosis of tumors including MIBC, mesothelioma, glioblastoma multiforme, lower-grade glioma, stomach adenocarcinoma, uveal melanoma, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, and lung squamous cell carcinoma. Conclusions We assessed the spatial landscape of the MIBC microenvironment and revealed a specific FB cluster with prognostic potential. These findings offer novel insights into the spatial heterogeneity of the MIBC microenvironment and highlight its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fcfdce6314523f58922cfd8b6a6e220595a1f7" target='_blank'>
              Spatially-resolved analyses of muscle invasive bladder cancer microenvironment unveil a distinct fibroblast cluster associated with prognosis
              </a>
            </td>
          <td>
            Chao Feng, Yaobang Wang, Wuyue Song, Tao Liu, Han Mo, Hui Liu, Shulin Wu, Zezu Qin, Zhenxing Wang, Yuting Tao, Liangyu He, Shaomei Tang, Yuanliang Xie, Qiuyan Wang, Tianyu Li
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Immune checkpoint inhibitors have revolutionized cancer therapy by harnessing the body’s immune system to eliminate tumor cells. However, only a subset of patients responds to treatment. Understanding why only some patients respond remains a critical challenge in cancer research due to the complexity and variability of cellular composition within the tumor microenvironment. Our study presents PRECISE (Predicting therapy Response through Extraction of Cells and genes from Immune Single-cell Expression data), a pipeline that leverages single-cell RNA sequencing data and machine learning techniques to predict ICI responses, while maintaining the richness of single-cell information and ensuring interpretability of the results. Results The PRECISE pipeline implements gene and cell-filtering approaches for optimizing treatment prediction. Utilizing the XGBoost algorithm for predicting patient response to ICI on a dataset of melanoma-infiltrated immune cell achieved an initial Area Under the Curve (AUC) of 0.84. This signal is further improved to 0.89 after Boruta feature selection, revealing an 11-gene predictive signature. Investigation of these genes through SHAP values identified various gene-pair interactions with non-linear conditional effects on predictions. Furthermore, a novel reinforcement learning framework implemented in PRECISE reveals non-predictive single cells that detracts the model’s performance. Altogether, the identified gene- and cell-based signatures demonstrates high prediction power across independent datasets, including lung, breast, brain, and skin cancers. Conclusions Our approach demonstrates the potential of supervised machine learning and reinforcement learning to enhance the understanding of cancer immunity and improve the prediction of treatment responses using single-cell data. Understanding patients’ response to immune checkpoint inhibitors (ICIs) is a critical challenge in cancer research, given the complexity and variability of immune interactions within the tumor microenvironment. Our study leverages single-cell RNA sequencing data and aims to utilize machine learning techniques to predict immunotherapy responses, while maintaining the richness of single-cell information and ensuring interpretability of the results. Using a dataset of melanoma immune cells, we conducted thorough preprocessing and applied the XGBoost algorithm in a leave-one-out cross-validation fashion to predict sample response. By labeling cells according to their sample’s response and aggregating predictions, we achieved an initial AUC score of 0.84. This score was improved to 0.89 with the application of Boruta feature selection, which identified key predictive genes, leading to an 11-gene predictive signature. Further analysis pinpointed T cell clusters as significant contributors to immune response. Utilizing SHAP values provided deeper insights into gene behaviors, interactions, and their effects on the model’s predictions. Additionally, a novel reinforcement learning model was developed for single-cell level prediction and characterization, allowing us to identify and analyze the most predictive cells for response and non-response. Our approach demonstrates the potential of sophisticated computational methods to enhance our understanding of cancer immunity and improve the prediction of treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdff0e265c13d348620c08428ea6a43d804a4e06" target='_blank'>
              Uncovering Predictive Gene and Cellular Signatures for Checkpoint Immunotherapy Response through Machine Learning Analysis of Immune Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff90b5963386b78b07e8f87307689a97c4a0bc10" target='_blank'>
              iFlpMosaics enable the multispectral barcoding and high-throughput comparative analysis of mutant and wild-type cells.
              </a>
            </td>
          <td>
            Irene Garcia-Gonzalez, Stefano Gambera, Susana F Rocha, Alvaro Regano, Lourdes García-Ortega, Mariya Lytvyn, Luis Diago-Domingo, Maria S. Sanchez-Muñoz, Aroa Garcia-Cabero, Ivana Zagorac, Wen Luo, Macarena De Andrés-Laguillo, Macarena Fernández-Chacón, Verónica Casquero-García, F. F. Lunella, C. Torroja, F. Sánchez-Cabo, Rui Benedito
          </td>
          <td>2024-12-13</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a rare malignancy disproportionately affecting adolescents and young adults with no standard of care. FLC is characterized by thick stroma, which has long suggested an important role of the tumor microenvironment. Over the past decade, several studies have revealed aberrant markers and pathways in FLC. However, a significant drawback of these efforts is that they were conducted on bulk tumor samples. Consequently, identities and roles of distinct cell types within the tumor milieu, and the patterns of intercellular communication, have yet to be explored. In this study we unveil cell-type specific gene signatures, transcription factor networks, and super-enhancers in FLC using a multi-omics strategy that leverages both single-nucleus ATAC-seq and single-nucleus RNA-seq. We also infer completely rewired cell-to-cell communication patterns in FLC including signaling mediated by SPP1-CD44, MIF-ACKR3, GDF15-TGFBR2, and FGF7-FGFR. Finally, we validate findings with loss-of-function studies in several models including patient tissue slices, identifying vulnerabilities that merit further investigation as candidate therapeutic targets in FLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5969e5a104cf835bc8976096535f834d7534207" target='_blank'>
              Single-cell multi-omic analysis of fibrolamellar carcinoma reveals rewired cell-to-cell communication patterns and unique vulnerabilities
              </a>
            </td>
          <td>
            Alaa R. Farghli, Marina Chan, Marc S. Sherman, L. K. Dickerson, Bo Shui, M. Nukaya, Andreas Stephanou, Rosanna K. Ma, Brian J Pepe-Mooney, Colton J. Smith, Donald Long, Paul R Munn, Adrian McNairn, Jennifer K. Grenier, Michael Karski, Sean M. Ronnekleiv-Kelly, V. Pillarisetty, Wolfram Goessling, Taranjit S. Gujral, K. Vakili, Praveen Sethupathy
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Drug resistance and distant metastases are major contributors to mortality in colorectal cancer (CRC). Here we investigate mechanisms underlying acquired resistance to oxaliplatin, a first-line, standard-of-care CRC treatment. We generated oxaliplatin-resistant CRC tumor cells with clinically relevant dosing regimen, which displayed enhanced metastatic potential. Transcriptomic and phenotypic analyses revealed a critical function for cholesterol biogenesis in modulating TGF-β signaling activity, which in turn regulates SERPINE1 expression, a gene we identified as a key player in promoting drug resistance and metastasis. Additionally, we uncovered a SERPINE1-associated nine-gene expression signature, RESIST-M, that can predict overall and relapse-free survival (RFS) in clinical cohorts and is able to stratify patients into CMS4/iCMS3-fibrotic CRC-subtypes, underscoring its clinical utility. Using mouse tumor models, we provide further evidence that targeting SERPINE1 and cholesterol biogenesis can be viable approaches to re-sensitize the resistant pro-metastatic CRC cells to oxaliplatin. This study not only elucidates the molecular underpinnings of drug resistance and metastasis in primary CRC, but also offers prognostic and therapeutic strategies to guide clinical management of the disease. Significance This study reveals critical resources and insights on oxaliplatin resistance and metastasis in CRC via a novel TGF-β cholesterol axis. We generated improved oxaliplatin-resistant models that enabled identification of a prognostic SERPINE1-based gene signature to predict oxaliplatin resistance-induced metastasis in CRC. This gene signature derived from our models showed that the models can mimic CMS-4/iCMS-fibrotic-like metastatic CRC patients. We validated therapeutic candidates targeting CMS-4/iCMS-fibrotic-like metastatic CRC cells which can reverse drug resistance and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec01a84e2257e6ca8a1bf9c64a17d3d5c5ca0b06" target='_blank'>
              Modelling oxaliplatin resistance in colorectal cancer reveals a SERPINE1-based gene signature (RESIST-M) and therapeutic strategies for pro-metastatic CMS4 subtype
              </a>
            </td>
          <td>
            Stephen Qi Rong Wong, Mohua Das, Niranjan Shirgaonkar, Kenzom Tenzin, Huiwen Chua, Lin Xuan Chee, Sae Yeoh Ahpa, Astley Aruna Murugiah, Wei Yong Chua, Madelaine Skolastika Theardy, M. Thangavelu, Jane Vin Chan, Choon Kong Yap, Iain Bee Huat Tan, Petros K Tsantoulis, Sabine Tejpar, J. Loo, Ramanuj DasGupta
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Reduced therapy response in breast cancer has been correlated with heterogeneity in biomarker composition, expression level, and spatial distribution of cancer cells within a patient tumor. Thus, there is a need for models to replicate cell-cell, cell-stromal, and cell-microenvironment interactions during cancer progression. Traditional two-dimensional (2D) cell culture models are convenient but cannot adequately represent tumor microenvironment histological organization, in vivo 3D spatial/cellular context, and physiological relevance. Recently, three-dimensional (3D) in vitro tumor models have been shown to provide an improved platform for incorporating compositional and spatial heterogeneity and to better mimic the biological characteristics of patient tumors to assess drug response. Advances in 3D bioprinting have allowed the creation of more complex models with improved physiologic representation while controlling for reproducibility and accuracy. This review aims to summarize the advantages and challenges of current 3D in vitro models for evaluating therapy response in breast cancer, with a particular emphasis on 3D bioprinting, and addresses several key issues for future model development as well as their application to other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ccd5bcde9ada94a158d2c3401e2be4034200cf" target='_blank'>
              Modeling intratumor heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            E. McDonough, M. Barroso, Fiona Ginty, David T Corr
          </td>
          <td>2024-12-06</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin D. Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Teratomas are a highly differentiated type of testicular germ cell tumors (TGCTs), the most common type of solid cancer in young men. Prominent inflammatory infiltrates are a hallmark of TGCTs, although their compositions and dynamics in teratomas remain elusive. Here, we reached out to characterize the infiltrating immune cells and their activation and polarization state by using high-throughput gene expression analysis of 129.MOLF-Chr19 mice that spontaneously develop testicular teratomas. We showed that inconspicuous testes without any apparent alterations in size or morphology can be clustered into three groups based on their expression of stemness and immune genes, supporting a model in which initial oncogenic transformation elicits a first wave of T-cell infiltration. Moderately and severely enlarged tumorous testes then displayed a progressive infiltration with T cells, monocytes/macrophages, and B cells. Importantly, T cells seem to adopt an inactive state caused by an overexpression of immune checkpoint molecules and the polarization of monocytes/macrophages to an anti-inflammatory phenotype. Our findings are supported by the analysis of metabolic gene expression, which unveiled alterations indicative of tumor growth and immune cell infiltration. Collectively, testicular teratomas, at least in mice, are characterized by a diverse inflammatory infiltrate containing T cells that putatively become inactivated, allowing the tumors to further grow. We believe that these findings may provide a rationale for the development of new immunomodulatory therapies for TGCTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b07bb71a6c00863ba07bb1169016cede3a797fe7" target='_blank'>
              Teratoma Development in 129.MOLF-Chr19 Mice Elicits Two Waves of Immune Cell Infiltration
              </a>
            </td>
          <td>
            Lucas Klaus, Sybille D. Reichardt, Maria Neif, Lutz Walter, F. Gayer, Holger M Reichardt
          </td>
          <td>2024-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite surgical resection, rigorous endoscopic surveillance, and immunotherapy with the Bacillus Calmette–Guérin (BCG) vaccine, 30% of high-risk bladder cancers recur, and 10% result in fatal outcomes within two years of diagnosis. The global shortage of BCG underscores the urgent need for alternative or complementary therapeutic strategies. To address this, we integrated transcriptomic profiling and targeted genomic sequencing to identify four consensus intrinsic subtypes of bladder cancer. Initially derived from bulk RNA profiling, these subtypes were further validated at the cellular and tissue-compartment levels using single-cell RNA sequencing and spatial transcriptomics. Notably, we identified a subtype of inflamed tumors with enhanced endogenous retroelement expression and increased commensal bacterial presence, which showed the highest responsiveness to BCG therapy. Additionally, we developed a machine learning-based model incorporating composite molecular features to predict recurrence risk, achieving a high accuracy (AUC = 0.90). Our findings establish a molecular precision framework for bladder cancer and nominate novel therapeutic targets to reduce reliance on BCG immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f0d9394677cb2bb295a5f2fe96a5d0c0fc0b84" target='_blank'>
              Genomic and transcriptomic profiling of high-risk bladder cancer reveals diverse molecular and microenvironment ecosystems
              </a>
            </td>
          <td>
            K. Meghani, Yanni Yu, N. Frydenlund, Erik Z. Li, B. Choy, S. Abdulkadir, J. Meeks
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) tumor cells and their interactions with the immune microenvironment, particularly at the leading-edge area, have been underexplored. This study employs single-cell RNA sequencing (scRNA-seq) and spatial transcriptome (ST) analysis on samples from the tumor core, adjacent non-tumorous tissue, and the leading-edge area of nine ICC patients. These findings indicate that tumor cells at the leading-edge area demonstrate enhanced proliferation and are tightly associated with the stroma, including endothelial cells and POSTN+ FAP+ fibroblasts. Notably, CD8+ T cells in this region exhibit a naive phenotype with low cytotoxicity and signs of exhaustion, likely due to compromised antigen presentation by antigen-presenting cells (APCs). The predominant CD8+ T cell subset, mucosal-associated invariant T (MAIT) cells, recruits SPP1+ macrophages within the stroma. This interaction, along with the presence of POSTN+ cancer-associated fibroblasts (CAFs) and endothelial cells, forms a unique "triad structure" that fosters tumor growth and ICC progression. The research highlights the intricate characteristics and interactions of ICC tumor cells in the leading-edge area, offering insights into potential therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4692f475931cf20c24dd17b389c955de7ede39" target='_blank'>
              Single-Cell and Spatial Transcriptomics Delineate the Microstructure and Immune Landscape of Intrahepatic Cholangiocarcinoma in the Leading-Edge Area.
              </a>
            </td>
          <td>
            Li Zuyin, Li Zhao, Cheng Qian, Changkun Zhang, Ma Delin, Jialing Hao, Chen Zhuomiaoyu, Li Yuzi, Zheng Jiaxi, Gao Jie, Zhu Jiye
          </td>
          <td>2024-12-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, , Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The adaptation of metastatic cells to their host tissue critically determines the pathogenicity of a cancer and therefore patient survival. Yet, it remains elusive to what extent the host environment drives gene expression programs in metastatic cells. Here we identify adaptive mechanisms that enable metastases to establish themselves in a novel tissue context. We performed single-cell RNA-sequencing on malignant and benign tissue samples from untreated donors with colorectal adenocarcinoma and liver metastasis to deduce tissue adaptive expression patterns. A novel computational approach identified genes and pathways that consistently adapted to the host tissue at the transition from the primary tumor to the paired metastasis across donors. This analysis revealed that the majority of expression changes in the metastasis reflect an expression signature reminiscent of benign liver epithelial cells. Cellular processes adapting to the liver environment include basic cellular functions such as energy metabolism, as well as tissue-specific pathways such as the regulation of lipid metabolism by PPAR-α. These adaptations potentially increase the pathogenicity of the metastatic cells and may provide new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189b4d35fab487ddefe0f36a861c9c03b2e18b5a" target='_blank'>
              Gene expression adaptation of metastases to their host tissue
              </a>
            </td>
          <td>
            L. Nagel, Marten C. Wenzel, Sascha Hoppe, Patrick S. Plum, Mohammad Karimpour, Marek Franitza, Roger Wahba, Marc Bludau, Christiane J. Bruns, Alexander Quaas, A. Beyer, Axel M. Hillmer
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Rationale:
 Smooth muscle cell (SMC) plasticity and phenotypic switching play prominent roles in the pathogenesis of multiple diseases like atherosclerosis and pulmonary hypertension, but their role in tumorigenesis is unknown.


 Objective:
 To investigate SMC diversity and plasticity in tumor angiogenesis and its underlying molecular drivers.


 Methods and Results:
 Here we use SMC-specific lineage-tracing mouse models and single cell RNA sequencing (scRNA-seq) to observe the phenotypic diversity of SMCs participating in tumor vascularization. We find that around 10% of SMCs adopt a phenotype which includes features traditionally associated with macrophage-like cells, suggesting they may contribute to the immune profile of the tumor microenvironment (TME). These cells are transcriptionally similar to ‘resolution phase’ M2b macrophages, which have been described to have a role in inflammation resolution. Signaling from bone marrow stromal antigen 2 (BST2) on the surface of tumor cells to PIRA2 on SMC promotes this phenotypic transition, leading to a SMC phenotype that is more proliferative, migratory, and phagocytic. Knockdown of BST2 in the tumor significantly decreases the transition towards a macrophage-like phenotype, and the cells that were able to transition had a comparatively higher inflammatory signal typically associated with anti-tumor effect.


 Conclusions:
 As BST2 is known to be a poor prognostic marker in multiple cancers where it is associated with an M2 macrophage-skewed TME, these studies suggest that phenotypically switched SMCs may have a previously unidentified role in this immunosuppressive milieu. The novel concept of SMC contribution to the immune landscape suggests manipulation of SMC phenotypic switching as an approach to altering cancer pathogenesis. Further translational work is needed to understand how this phenotypic switch could influence response to anti-cancer agents and if targeted inhibition of SMC plasticity would be therapeutically beneficial.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/049a7db1d4940952698e1c93fbbc4e08101322fd" target='_blank'>
              Abstract 4134531: BST2 induces vascular smooth muscle cell plasticity and phenotype switching during cancer progression
              </a>
            </td>
          <td>
            Caitlin F. Bell, Richard A. Baylis, Wei Feng Ma, Clint Miller, Nick J. Leeper
          </td>
          <td>2024-11-12</td>
          <td>Circulation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Aggressive human prostate tumors traverse a contractile smooth muscle pseudo-capsule, in a process termed extracapsular extension (ECE), defining the pT3 pathologic stage associating with increased biochemical recurrence, bone metastases, and cancer-specific mortality. While T3 lesions can be detected by non-invasive mpMRI imaging, how the tumor invades into and through the contractile muscle is unknown. Using a mouse xenograft model system of ECE, we investigated the transcriptomic consequences of human tumor invasion into and through the contractile smooth muscle of the mouse diaphragm. Both human and mouse gene transcripts were documented and analyzed. Tumor cells were intraperitoneally injected into male NSG mice and 6 weeks later, three tumor compartments (pre-invasive, muscle-invasive, and super-invasive cells that had reached the superior diaphragm surface) were analyzed for differential bulk human and mouse gene expression. Whole genome transcriptomic sequencing and GO pathway analysis revealed that approximately 414 human genes were differentially expressed between the non- invasive, muscle-resident, and super-invasive tumor populations and at least 5 enriched pathways were involved. The unique expression gene patterns of muscle-resident tumor cells were reversible when compared to the pre-invasive and super-invasive expression patterns. Significant increases in g-H2AX and nuclear deformation were observed in the muscle-resident tumor clusters as compared to the non-invasive tumor mass. No differences in tumor cell proliferation, as detected by Ki67 staining, were found between the tumor clusters in the three compartments. Immunohistochemistry staining for a damage response cytokeratin (KRT6A) and integrin (CD49f) revealed heterogeneity within the muscle resident tumors as compared to the non- invasive cells. Taken together, these data suggest a hostile contractile muscle environment elicits specific responses by the invading tumor. Single cell analysis within each of the tumor compartments is currently underway to define the spatial heterogeneity of gene expression in invasive tumor cell clusters within the contractile muscle. A dynamic reciprocity of the tumor/muscle microenvironment is suggested by the analysis of the bulk transcriptomic sequencing demonstrating a significant increase in mouse muscle bio-synthetic gene transcription as a consequence of the human tumor penetrating the tissue. Taken together, these data indicate that human tumors, during the act of traversing the contractile muscle layer, respond to this unique microenvironment by transiently altering transcription, while sustaining nuclear damage which results in the reprogramming of the muscle. This new information suggests that novel tumor or muscle biomarkers might indicate early muscle invasion events and assist new high-resolution image analysis for precision diagnostic and/or therapeutic decisions. (Supported in part by P30 CA23074; F30 CA143924, UACC Team Science Award).
 Citation Format: Kendra D Marr, Beatrice S Knudsen, Rafael Sainz, Kelvin W Pond, Noel E Warfel, Belinda E Sun, Anne E Cress. Tumor muscle invasion promotes tumor heterogeneity and normal muscle reprogramming [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d695d2e17a8bb732f87fee7797bcc07c303850" target='_blank'>
              Abstract A055: Tumor muscle invasion promotes tumor heterogeneity and normal muscle reprogramming
              </a>
            </td>
          <td>
            Kendra D. Marr, Beatrice S. Knudsen, Rafael Sainz, Kelvin W Pond, Noel E. Warfel, Belinda E Sun, A. Cress
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d907c1f70a7e162db16222c621c3aec6a63202a" target='_blank'>
              Integrated single-cell analysis reveals heterogeneity and therapeutic insights in osteosarcoma
              </a>
            </td>
          <td>
            Dongan He, Xiaoqian Che, Haiming Zhang, Jiandong Guo, Lei Cai, Jian Li, Jinxi Zhang, Xin Jin, Jianfeng Wang
          </td>
          <td>2024-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d052ec02be986508482ae1f68379c54cb0ed96e8" target='_blank'>
              Cell-type specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acquired resistance is one of the major causes of failure of standard therapies in cancer patients. Chemotherapeutic agents are still widely used and the understanding of the mechanisms leading to secondary resistance to these molecules are still puzzling. Recently, the role of the tumor immune microenvironment has been recognized. Among the cells potentially involved, macrophages seem to be the perfect culprits. In a previous work, we have shown that hypoxic macrophages are able to provide strong protection against 5-fluorouracil, a first-line chemotherapeutic agent in digestive cancers. In the present work, we use mathematical modeling to explore the spatiotemporal aspects of the treatment-induced organization of the tumor environment. Based on analytical and numerical analysis, we propose that macrophage-driven protection against chemotherapy under treatment does not rely solely on biochemical degradation, but is enhanced by the emergence of spatially structured chemotherapeutic protective niches. This work paves the way for the development of new therapeutic strategies that rely on targeting the spatial organization of tumors as a way to control treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f1862b2c528215ed84b22678834153abe2795d7" target='_blank'>
              Spatial organization of mediated-macrophage chemoprotective niches in solid tumors: A mathematical analysis
              </a>
            </td>
          <td>
            William Dangelser, A. Stéphanou, Arnaud Millet
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Pericytes are crucial for the development, stabilization, and functional regulation of microvasculature, especially in the retina. In diabetic retinopathy (DR), early loss of pericytes is a key event that drives microvascular dysfunction. Despite their critical role, the mechanisms underlying the functional heterogeneity of pericytes in DR remain poorly understood, impeding the development of effective therapeutic strategies. Methods We employed single-cell RNA sequencing to construct a comprehensive single- cell atlas of non-diabetic and diabetic retinas. Using bioinformatic clustering and subcluster analysis, we identified a specific pericyte subcluster associated with diabetic microvascular complications. Differential gene expression analysis and immunofluorescence validation highlighted PTTG1 as a potential key regulator of pericyte dysfunction. To investigate its functional role, we emplyed CRISPR/Cas9 and adenoviral vectors to modulate PTTG1 expression in vitro and in vivo. Combined transcriptomic and metabolomic approaches were used to explore the mechanistic pathways through which PTTG1 influences pericyte biology and vascular function. Results We identified a novel pericyte subcluster characterized by elevated expression of PTTG1, which was strongly correlated with diabetic microvascular dysfunction. Silencing PTTG1 using CRISPR/Cas9 and siRNA in vitro mitigated pericyte dysfunction under high- glucose conditions. Targeted knockdown of PTTG1 using viral vectors improved retinal vascular integrity and reduced neovascularization in diabetic mice. Transcriptomic and untargeted metabolomic analyses revealed that PTTG1 knockdown reprogrammed pericyte energy metabolism by modulating glycolysis pathway genes, reducing oxidative stress, and restoring pericyte function, ultimately alleviating microvascular dysfunction in DR. Conclusions PTTG1 plays a critical role in regulating pericyte dysfunction and maintaining vascular homeostasis in diabetic retinopathy. By modulating key metabolic pathways and pericyte phenotypes, PTTG1 represents a promising therapeutic target for treating diabetic microvascular complications. These insights offer a novel molecular framework for developing targeted therapies aimed at restoring retinal vascular health in diabetic patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d2cfb264dd76b8c3dd1f5b9db0dce030fd7e6a3" target='_blank'>
              PTTG1-Mediated Pericyte Dysfunction Drives Diabetes-Induced Microvascular Dysfunction
              </a>
            </td>
          <td>
            Linyu Zhang, Ling Ren, Jingyue Zhang, Min Xia, Xiaosa Li, Mudi Yao, Fanfei Ma, Chang Jiang, Jin Yao, Biao Yan
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef9e868d681b765f550dc5f5febbd2934a88e0fb" target='_blank'>
              Single-cell RNA sequencing reveals the landscape of the cellular ecosystem of primary hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zeli Yin, Yilin Song, Liming Wang
          </td>
          <td>2024-11-14</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5361264ba7fa5b03a05e9caae598d056bebbf71c" target='_blank'>
              Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Yilv Yan, Dongqing Sun, Junjie Hu, Yue Chen, Liangdong Sun, Huansha Yu, Yicheng Xiong, Zhida Huang, Haoran Xia, Xinsheng Zhu, D. Bian, Fenghuan Sun, Likun Hou, Chunyan Wu, Orion R Fan, Haiyang Hu, An Zeng, Lele Zhang, Yi Eve Sun, Chenfei Wang, Peng Zhang
          </td>
          <td>2024-12-10</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="
 Metastatic disease is the common cause of death in colorectal cancer. To better understand the cellular landscape of colorectal metastasis, we profiled primary tumors and liver, lung, and peritoneal metastases using single-cell RNA sequencing. Our analysis revealed a subset of tumor epithelial cells with a highly plastic transcription profile and, inversely, low expression of stemness markers. Tumor cells with plasticity markers were spatially enriched at the tumor- stroma interface with exposure to tissue microenvironmental factors. Cell-free malignant ascites, which recapitulates the tumor microenvironment, impaired expression of proliferation and stem- like markers in favor of plasticity markers and enhanced migration of tumor cells. Furthermore, Co-culture of colorectal tumor cells with CD11b+ macrophages isolated from malignant ascites from patients was sufficient to recapitulate tumor cell plasticity. Profiling of tumor-associated macrophages identified enrichment for a lipid-handling features and high spatial correlation with plastic tumor cells. Taken together, we demonstrate that tumor-associated macrophages contribute to disease progression through development of pro-malignant features.
 Citation Format: Matthew A. Cottam, Muhammad B. Mirza, Emily N. Arner, Marium Siddiqui, Clare Lipscombe, Zaid Hatem, Katy E. Beckermann, Kamran Idrees. Regulatory macrophages contribute to tumor cell plasticity in colorectal cancer metastases [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38b2321f9b6c18e29ebc7faf6bc4bebdd5ef49f8" target='_blank'>
              Abstract B019: Regulatory macrophages contribute to tumor cell plasticity in colorectal cancer metastases
              </a>
            </td>
          <td>
            Matthew A. Cottam, M. Mirza, Emily N. Arner, Marium Siddiqui, Clare Lipscombe, Zaid Hatem, Katy E. Beckermann, K. Idrees
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="A comprehensive understanding of tumor heterogeneity, tumor microenvironment and the mechanisms of drug resistance is fundamental to advancing breast cancer research. While single-cell RNA sequencing has resolved the issue of “temporal dynamic expression” of genes at the single-cell level, the lack of spatial information still prevents us from gaining a comprehensive understanding of breast cancer. The introduction and application of spatial transcriptomics addresses this limitation. As the annual technical method of 2020, spatial transcriptomics preserves the spatial location of tissues and resolves RNA-seq data to help localize and differentiate the active expression of functional genes within a specific tissue region, enabling the study of spatial location attributes of gene locations and cellular tissue environments. In the context of breast cancer, spatial transcriptomics can assist in the identification of novel breast cancer subtypes and spatially discriminative features that show promise for individualized precise treatment. This article summarized the key technical approaches, recent advances in spatial transcriptomics and its applications in breast cancer, and discusses the limitations of current spatial transcriptomics methods and the prospects for future development, with a view to advancing the application of this technology in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9484c611ac13b7e4460cc6b1b5eb2c6b25e4a9" target='_blank'>
              Spatial transcriptomics in breast cancer: providing insight into tumor heterogeneity and promoting individualized therapy
              </a>
            </td>
          <td>
            Junsha An, Yajie Lu, Yuxi Chen, Yuling Chen, Zhaokai Zhou, Jianping Chen, Cheng Peng, Ruizhen Huang, Fu Peng
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Cancer-associated fibroblasts (CAFs) are attractive therapeutic targets for cancer due to their abundance in the tumor microenvironment (TME) and extensive pro- and antitumorigenic functions. However, formulating CAF-targeting strategies remains challenging due to the complexity of CAF biology. CAFs specialize into phenotypically and functionally distinct subtypes that may respond differently to specific treatments. Thus, designing CAF-targeting therapies necessitates a thorough understanding of CAF heterogeneity and preclinical model systems that can reliably prototype CAF biology observed in tumors. However, CAF heterogeneity is still not well-understood for many solid tumors, including the stroma-rich pancreatic ductal adenocarcinoma (PDAC), and the translatability of stroma models has not been comprehensively evaluated. In this work, we developed in vitro cell lines of tumor-CAF cocultures using primary CAFs derived from PDAC and used single-cell transcriptomics and CRISPR perturbation to investigate CAF heterogeneity and subtype-specific responses. We found that coculturing primary CAFs with the BxPC3 PDAC cell line could capture the tumor-CAF crosstalks underlying CAF activation. We characterized eight subtypes, including myofibroblastic, inflammatory, extracellular matrix (ECM)-like, and interferon CAFs, which could be preserved with immortalization. We also delineated the trajectories of CAF subtype interconvertibility, highlighting inflammatory CAFs as the likely initial diseased state and ECM CAFs as a “reservoir” that seeded the later states. By correlating the in vitro subtypes with clinical data of immunotherapy responses, we established that the in vitro cell lines could model the immuno-modulatory and immunoresistant phenotypes of CAFs. Finally, using an immortalized tumor-CAF coculture line, we performed Perturb-seq single-cell perturbation to perturb several stroma genes of interest, including TGFBR1, SPATS2L, AEBP1, and PTGS1, and evaluated the subtype-specific effects of the perturbations. We found that perturbing TGFBR1 resulted in the expected eradication of myofibroblastic CAFs, while the other perturbations resulted in subtype-specific modulations of metabolic and extracellular matrix related pathways. Overall, our findings underscored that in vitro tumor-CAF coculture models could serve as a translatable system that effectively captured CAF biology and heterogeneity observed in tumors and highlighted some similarities between PDAC CAFs and previously characterized breast cancer CAFs. We also demonstrated that single-cell perturbation could be a useful approach for interrogating the distinct responses of different CAF subtypes to perturbations.
 Citation Format: Elysia Saputra, Shamsudheen Karuthedath Vellarikkal, Lixia Li, Hong Sun, Suchitra Natarajan, Federica Piccioni, Alex M. Tamburino, Xin Yu, Aleksandra K. Olow. Characterization and CRISPR perturbation of cancer-associated fibroblast heterogeneity in vitro models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d13335b76ec58b7c353073a2851e7fff51991a" target='_blank'>
              Abstract B009: Characterization and CRISPR perturbation of cancer-associated fibroblast heterogeneity in vitro models
              </a>
            </td>
          <td>
            Elysia Saputra, S. Vellarikkal, Lixia Li, Hong Sun, Suchitra Natarajan, Federica Piccioni, Alex M. Tamburino, Xin Yu, Aleksandra K. Olow
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Recent drug discovery breakthroughs led to the approval of KRASG12C inhibitors in lung adenocarcinoma (LUAD). Unfortunately, clinical responses remain limited due to rapid resistance onset. Proteolysis-targeting chimeras (PROTACs) have emerged as promising alternatives to traditional inhibition. However, there is limited mechanistic understanding of KRAS degradation in vivo. Here, we developed a preclinical LUAD mouse model and demonstrated that targeted oncogenic KRAS degradation induces rapid tumor regression. Transcriptional, histological, and immunophenotypic analyses revealed a substantial remodeling of the tumor microenvironment. Notably, disease relapse observed during long-term degrader treatment stems from proteolysis machinery dysregulation, indicating resistance mechanisms distinct from those reported upon KRAS inhibition. Our findings highlight the therapeutic potential of KRAS degradation in LUAD, offering insights into cell-intrinsic and extrinsic mechanisms driving durable antitumor responses and supporting further clinical exploration. SIGNIFICANCE Given the short duration of the clinical responses to KRAS inhibitors, complementary therapies are a dire medical need. Our preclinical findings endorse KRAS degradation as a therapeutic alternative in LUAD, where cell-intrinsic and extrinsic mechanisms drive tumor regression and durable therapeutic responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/259221e90e5e8be79572d797a84901e029d36ed2" target='_blank'>
              Targeted KRASG12V degradation elicits efficient and durable lung adenocarcinoma regression in vivo
              </a>
            </td>
          <td>
            Alberto Martín, Inés M. García-Pérez, Sonia San José, Pep Rojo, Carlos Riego-Mejías, Cristina Teodosio, Bárbara MG. Barbosa, Carolina Sánchez-Zarzalejo, Ignasi Folch-I-Casanovas, Antonia Odena Caballol, Sònia Jarió, Rodrigo Entrialgo, M. Nokin, Diana Loa, Elizabeth Guruceaga, Camille Stephan-Otto Attolini, Chiara Ambrogio, Alberto Villanueva, Silvestre Vicent, Antoni Riera, David Santamaría, Cristina Mayor-Ruiz
          </td>
          <td>2024-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Cancer progression poses a clinical hurdle due to tumor heterogeneity that contributes to treatment resistance and immune evasion. While the adaptive immune system can effectively eliminate cancer in certain instances, tumor escape remains challenging. The immune microenvironment plays an important role in this process. One unanswered question is the role of ECM geometry, also known as Tumor Associated Collagen Signatures (TACS), and its effects on tumor immune recognition. In solid malignancies, ECM geometry has been associated with disease stage and T cell infiltration. Specifically, empirically observed ECM topologies are frequently categorized based on fiber arrangement: random fibers (TACS1), circumferentially aligned fibers (TACS2), and radially arranged fibers (TACS3). Even though a clear negative correlation between TACS and patient survival has been established, the specific roles and extent of TACS in T cell-driven cancer evolution remain uncharacterized. The precise mechanisms underlying how TACS influences cell movement are still not fully elucidated, with divergent opinions on whether and how the ECM mediates immune cell infiltration. At present, we currently lack a physical model relating the impact of TACS on the spatial co-evolution between an adaptive immune repertoire and a heterogeneous population of evading cancer cells. Here, we developed the EVO-ACT (EVOlutionary Agent-based Cancer T cell interaction) model, which is based on the Gillespie algorithm, to study the effects of TACS on tumor evolution and dynamical tumor-T cell interactions. Our analysis of tumor-T cell interactions across three TACS types reveals distinct migration patterns, with TACS3 showing the highest efficiency, TACS1 intermediate, and TACS2 the lowest. TACS impacts T cells more than tumors, with stronger chemokine gradients aiding T cell infiltration. Our model suggests that despite TACS2's limited efficiency, it cannot fully impede T cell infiltration. Varied migration efficiencies lead to diverse T cell infiltration patterns and immunoediting levels. TACS3 shows a benefit to immune recognition and higher survival, contrary to lower survival in TACS3 patients observed clinically in breast cancer. Our results indicate that only after introducing phenotypic changes, such as Epithelial-Mesenchymal Transition (EMT), and noting that EMT occurs exclusively after the TACS2-TACS3 progression, can our findings successfully explain the clinically observed trends. Considering mesenchymal tumor cells' ability to upregulate PD-1/PD-L1, we explore TACS-specific tumor evolution's impact on PD-1/PD-L1 inhibitor responses and patient survival. We observe higher tumor escape rates and lower survival in TACS3 compared to TACS2. Thus, TACS3 alone does not account for lower survival in breast cancer; TACS3-associated late-stage cancer byproducts like EMT and elevated checkpoint expression significantly decrease survival.
 Citation Format: Yijia Fan, Jason Tomas George. ECM architecture drives distinct evolutionary patterns of tumor immune escape and elimination in breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A031.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52d9e3eca63d450bccf177de70595f8faf5e6be" target='_blank'>
              Abstract A031: ECM architecture drives distinct evolutionary patterns of tumor immune escape and elimination in breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Jason Tomas George
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Growth hormone-secreting pituitary adenomas (GHPA) display diverse biological behaviors and clinical outcomes, necessitating the identification of tumor heterogeneity and prognostically relevant markers.



 In this study, we performed single-cell RNA sequencing (scRNA-seq) on 10 GHPA samples, four of which also underwent spatial transcriptome sequencing, and used scRNA-seq data from four normal pituitary samples as controls. Cell subtype characterization in GHPA was analyzed using multiple algorithms to identify malignant bias regulators, which were then validated using a clinical cohort.



 We constructed the first single-cell and spatial transcriptome profiles of GHPA, which contained 87,862 cells and revealed 16 tumor cell subtypes. Among the tumor cells, we identified distinct developmental trajectories and three malignant-biased subtypes (PIT1_C05, PIT1_C06, and PIT1_C10). The spatial distribution characteristics of these malignant-biased cells may influence the growth characteristics and prognosis of GHPA. We screened specific regulatory transcription factors, including FOXO1, GTF2IRD1, and MAX. Clinical cohort validation indicated that FOXO1 might be associated with tumor invasion and progression, while high expression of MAX could result in poor endocrine outcomes.



 GHPA exhibits rich heterogeneity and diverse cell subtypes, with specific transcription factors potentially regulating cell malignant bias, thereby influencing tumor characteristics and prognosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a105784b2d156549c656072efae7d8dac904126" target='_blank'>
              Single-cell and spatial transcriptomic analyses reveal heterogeneity characteristics and specific cell subtype regulators in growth hormone-secreting pituitary adenomas
              </a>
            </td>
          <td>
            Yu Zhang, Lifeng Wang, Xingcheng Yi, Xinyu Ma, Hongyu Wu, Mingzhao Zhang, Zhenqi Yang, Lizhen Ma, Zenghua Mi, Wei-jia Zhi, Cong Fu, Pinan Liu, Zhijun Yang
          </td>
          <td>2024-12-31</td>
          <td>International Journal of Surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc23d3d27f11aa0ad92c216de43831771b9dae4" target='_blank'>
              Differential cell architecture and microenvironmental responses of pretumoral and tumoral cellular models exposed to coverslip-induced hypoxia.
              </a>
            </td>
          <td>
            Magdalena Millán, Felipe Parietti, Florencia Lamela, M. C. De Rossi, Belén Benítez, Valeria Levi, Manoela Domingues, R. Bologna-Molina, M. Arocena, J. Hochmann
          </td>
          <td>2025-01-03</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Tumor is a complex and dynamic ecosystem formed by the interaction of numerous diverse cells types and the microenvironments they inhabit. Determining how cellular states change and develop distinct cellular communities in response to the tumor microenvironment is critical to understanding cancer progression. Tumour-associated macrophages (TAMs) are an important component of the tumour microenvironment and play a crucial role in cancer progression. This study was designed to identify cell-state-specific M2 macrophage markers associated with gastric cancer (GC) prognosis through integrative analysis of single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq data using a machine learning framework named EcoTyper. Results The results showed that TAMs were classified into M1 macrophages, M2 macrophages, monocytes, undefined macrophages and dendritic cells, with M2 macrophages predominating. EcoTyper assigned macrophages to different cell states and ecotypes. A total of 168 cell-state-specific M2 macrophage markers were obtained by integrative analysis of scRNA-seq and bulk RNA-seq data. These markers could categorize GC patients into two clusters (clusters A and B) with different survival and M2 macrophages infiltration abundance. Cell adhesion molecules, cytokine-cytokine receptor interaction, JAK/STAT pathway, MAPK pathway were significantly enriched in cluster A, which had worse survival and higher M2 macrophages infiltration. Conclusion In conclusion, this study profiles a single-cell atlas of intratumor heterogeneity and defines the cell states and ecotypes of TAMs in GC. Furthermore, we have identified prognostically relevant cell-state-specific M2 macrophage markers. These findings provide novel insights into the tumor ecosystem and cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d8a2fb2bd7548dde928bcb962ded6e0380b6c3" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA-Sequencing Based on EcoTyper Machine Learning Framework Identifies Cell-State-Specific M2 Macrophage Markers Associated with Gastric Cancer Prognosis
              </a>
            </td>
          <td>
            A-Kao Zhu, Guang-Yao Li, Fang-Ci Chen, Jia-Qi Shan, Yu-Qiang Shan, Chen-Xi Lv, Zhi-Qiang Zhu, Yi-Ren He, Lu-Lu Zhai
          </td>
          <td>2024-12-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d643827078776ad2a6449f6a7b6358707684af74" target='_blank'>
              Cell-cell interactions mediating primary and metastatic breast cancer dormancy.
              </a>
            </td>
          <td>
            Nicholas A Lenart, Shreyas S Rao
          </td>
          <td>2024-11-25</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Hybrid epithelial/mesenchymal (E/M) cells are the most metastatic of carcinoma cell types. Hybrid E/M cells have both epithelial and mesenchymal properties reminiscent of chimeras, mythical creatures composed of different animals. The co-expression of epithelial and mesenchymal proteins is a defining feature of hybrid E/M cancer cells. In this review, we discuss the use of cytokeratin (epithelial) and vimentin (mesenchymal) intermediate filaments as markers in hybrid E/M cells. We describe the structures and functions of these filaments that are uniquely expressed in hybrid E/M cells and speculate on potential hybrid structures formed between vimentin and cytokeratin. We also describe cell-surface cytokeratin and vimentin and their functions and discuss their potential as predictive biomarkers and as targets for anti-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0a63915420ccb6800640641805de420f117dd3" target='_blank'>
              The Cancer Chimera: Impact of Vimentin and Cytokeratin Co-Expression in Hybrid Epithelial/Mesenchymal Cancer Cells on Tumor Plasticity and Metastasis
              </a>
            </td>
          <td>
            Nick A. Kuburich, Julia M. Kiselka, P. den Hollander, Andrew A. Karam, S. Mani
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The development of T cell receptor-engineered T cells (TCR-T) targeting intracellular antigens is a promising strategy for treating solid tumors; however, the mechanisms underlying their effectiveness remain poorly understood. In this study, we employed advanced techniques to investigate the functional state of T cells engineered with retroviral vectors to express a TCR specific for the NY-ESO-1 157-165 peptide in the HLA-A*02:01 context. Flow cytometry revealed a predominance of naïve T cells. Gene expression profiling using NanoString technology revealed upregulation of genes encoding chemokine receptors CCR2 and CCR5, indicating enhanced migration towards tumor sites. In the SK-Mel-37 xenograft model, these transduced T cells achieved complete tumor eradication. Furthermore, single-cell RNA sequencing (scRNA-seq) conducted 14 days post-TCR T cell infusion provided a comprehensive analysis of the in vivo adaptation of these cells, identifying a distinct subset of CD8+ effector T cells with an NK cell-like gene expression profile. Our findings indicate that NY-ESO-1 TCR-transduced T cells have the potential to mediate dual antitumor effects through both antigen-independent NK-like and antigen-specific CTL-like responses. This study underscores the potential of NY-ESO-1 TCR-T cells as potent tumor-eradicating agents, highlighting the importance of harnessing their versatile functional capabilities to refine and enhance therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c35da049d73d1a38187e30a25bcb51a1113dcd7" target='_blank'>
              Decoding NY-ESO-1 TCR T cells: transcriptomic insights reveal dual mechanisms of tumor targeting in a melanoma murine xenograft model
              </a>
            </td>
          <td>
            A. Alsalloum, S. Alrhmoun, O. Perik-Zavosdkaia, M. Fisher, M. Volynets, J. Lopatnikova, R. Perik-Zavodskii, J. Shevchenko, Julia Philippova, Olga Solovieva, Evgenii Zavjalov, V. Kurilin, H. Shiku, A. Silkov, S. Sennikov
          </td>
          <td>2024-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa47d77b71a7337ad0ecaef200be807283fa4101" target='_blank'>
              Heterogeneity-driven phenotypic plasticity and treatment response in branched-organoid models of pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            A. Papargyriou, Mulham Najajreh, David P Cook, Carlo Maurer, S. Bärthel, H. Messal, Sakthi K Ravichandran, Till Richter, Moritz Knolle, Thomas Metzler, Akul R Shastri, R. Öllinger, Jacob Jasper, Laura Schmidleitner, Surui Wang, C. Schneeweis, H. Ishikawa-Ankerhold, T. Engleitner, Laura Mataite, Mariia Semina, Hussein Trabulssi, S. Lange, Aashreya Ravichandra, Maximilian Schuster, S. Mueller, K. Peschke, A. Schäfer, S. Dobiasch, S.E. Combs, Roland M. Schmid, Andreas R. Bausch, R. Braren, I. Heid, C. Scheel, Günter Schneider, Anja Zeigerer, Malte Luecken, K. Steiger, G. Kaissis, J. van Rheenen, F. Theis, Dieter Saur, R. Rad, Maximilian Reichert
          </td>
          <td>2024-12-10</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f53257f56f2a57ba1bd1234a1ea057e49c3bd2" target='_blank'>
              Single-cell RNA-seq analysis of cancer-endothelial cell interactions in primary tumor and peritoneal metastasis from a single patient with colorectal cancer
              </a>
            </td>
          <td>
            Yuri Sakimoto, Kohei Kumegawa, S. Matsui, T. Yamaguchi, T. Mukai, K. Okabayashi, Seiichi Mori, Yuko Kitagawa, T. Akiyoshi, Reo Maruyama
          </td>
          <td>2024-11-18</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88ba70c1f7c9e0c6817db8a69bc543b13dfed9df" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics of esophageal squamous cell carcinoma with lymph node metastases.
              </a>
            </td>
          <td>
            Wei Guo, Bolun Zhou, Lizhou Dou, Lei Guo, Yong Li, J. Qin, Zhen Wang, Qilin Huai, Xuemin Xue, Yin Li, Jianming Ying, Qi Xue, Shugeng Gao, Jie He
          </td>
          <td>2025-01-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="ABSTRACT Macrophages are pivotal in driving breast tumor development, progression, and resistance to treatment, particularly in estrogen receptor-positive (ER+) tumors, where they infiltrate the tumor microenvironment (TME) influenced by cancer cell-secreted factors. By analyzing single-cell RNA sequencing data from 25 ER+ tumors, we elucidated interactions between cancer cells and macrophages, correlating macrophage density with epithelial cancer cell density. We identified that S100A11, a previously unexplored factor in macrophage-cancer crosstalk, predicts high macrophage density and poor outcomes in ER+ tumors. We found that recombinant S100A11 enhances macrophage infiltration and migration in a dose-dependent manner. Additionally, in a 3D matrix using a panel of three ER+ breast cancer cell lines, we showed that secreted S100A11 levels from cancer cells were associated with increased monocyte infiltration that subsequently differentiation toward macrophages. Genetic silencing of S100A11 in the S100A11-high T47D cancer cells reduced monocyte infiltration, consistent with results using a S100A11 blocking antibody in T47D cancer cells and in a clinically relevant patient-derived organoid model. Phenotypic analysis of macrophages cocultured with T47D cancer cells following S100A11 knockdown revealed lower expression of the immunosuppressive marker CD206, further underscoring the role of S100A11 as a paracrine regulator of pro-tumorigenic cancer-macrophage crosstalk. This study offers novel insights into the interplay between macrophages and cancer cells in ER+ breast tumors, highlighting S100A11 as a potential therapeutic target to modulate the macrophage-rich tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bd1881c70cca02440b0e461e40eba12027d0287" target='_blank'>
              Cancer-cell derived S100A11 promotes macrophage recruitment in ER+ breast cancer
              </a>
            </td>
          <td>
            Sanghoon Lee, Youngbin Cho, Yiting Li, Ruxuan Li, Angela Wong Lau, Matthew S Laird, Daniel B Brown, Priscilla McAuliffe, Adrian V. Lee, S. Oesterreich, Ioannis K. Zervantonakis, H. Osmanbeyoglu
          </td>
          <td>2024-11-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Spatial transcriptomics is revolutionizing our understanding of cellular dynamics and interactions in the tumor microenvironment (TME). By mapping distinct cellular populations and their interactions, we can gain valuable insights into tumor behavior. To investigate these dynamics in breast cancer, we conducted Xenium In-situ spatial transcriptomics on twenty formalin-fixed paraffin-embedded (FFPE) samples taken from eight patients undergoing neoadjuvant chemotherapy. These samples have been taken pre-treatment, mid-treatment and post-treatment from four cases with pathological complete response (pCR), and four cases with residual disease (RD) at time of surgery. Our analyses have found distinct cellular composition changes between pCR and RD cases. Patients with RD demonstrated an increase in endothelial, perivascular-like (PVL) and epithelial cells from pre- to post-treatment. This was accompanied by a decrease in myeloid and malignant epithelial cells at mid-treatment. In contrast, patients with pCR demonstrated increases in cancer-associated fibroblasts (CAFs) and myeloid cells, with reductions in plasmablasts and cancer epithelial cells at mid-treatment. This suggests that an increase in CAF and myeloid cells observed between pre-treatment and mid-treatment samples may be predictive of response. We performed ligand-receptor (LR) analyses, which demonstrated diverse myeloid interactions in RD cases and diverse CAF interactions in pCR cases in pre-treatment samples. Mid-treatment samples exhibited a decrease in interactions for CAFs, myeloid and T cells in pCR and RD cases. The most frequently identified CAF LR interactions, as senders, were between TGFB1/TGFB3 and TGFBR1/TGFBR2, while the most frequently identified myeloid LR interactions were between PDCD1LG2 - PDCD1 (encoding PDL2 & PD1), CCL8 - ACKR1 (encoding Atypical Chemokine Receptor 1) and CCL2 - ACKR1. Our findings indicate that the dynamic changes in cellular composition and interactions within the TME differs between responders and non-responders. To further explore these differences, we plan to perform spatially constrained analyses to identify unique cellular niches that may serve as potential biomarkers for informing novel therapeutic strategies.
 Citation Format: Jeremy Mo, Hanyun Zhang, Kate Harvey, Alexander Swarbrick. Spatial transcriptomics unveils disparities in tumor microenvironment dynamics between breast cancer responders and non-responders [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d85f0bd80cb01173f30a8b1b70f6e0a4a64eb5" target='_blank'>
              Abstract C040: Spatial transcriptomics unveils disparities in tumor microenvironment dynamics between breast cancer responders and non-responders
              </a>
            </td>
          <td>
            Jeremy Mo, Hanyun Zhang, K. Harvey, Alexander Swarbrick
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="CDK4/6 inhibitors such as palbociclib block cell cycle progression and improve outcomes for many ER+/HER2- breast cancer patients. Unfortunately, many patients are initially resistant to the drug or develop resistance over time in part due to heterogeneity among individual tumor cells. To better understand these mechanisms of resistance, we used multiplex, single-cell imaging to profile cell cycle proteins in ER+ breast tumor cells under increasing palbociclib concentrations. We then applied spherical principal component analysis (SPCA), a dimensionality reduction method that leverages the inherently cyclical nature of the high-dimensional imaging data, to look for changes in cell cycle behavior in resistant cells. SPCA characterizes data as a hypersphere and provides a framework for visualizing and quantifying differences in cell cycles across treatment-induced perturbations. The hypersphere representations revealed shifts in the mean cell state and population heterogeneity. SPCA validated expected trends of CDK4/6 inhibitor response such as decreased expression of proliferation markers (Ki67, pRB), but also revealed potential mechanisms of resistance including increased expression of cyclin D1 and CDK2. Understanding the molecular mechanisms that allow treated tumor cells to evade arrest is critical for identifying targets of future therapies. Ultimately, we seek to further SPCA as a tool of precision medicine, targeting treatments by individual tumors, and extending this computational framework to interpret other cyclical biological processes represented by high-dimensional data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57de25c3330f7fce9f46a524989a6ef4e9398a4c" target='_blank'>
              Spherical Manifolds Capture Drug-Induced Changes in Tumor Cell Cycle Behavior
              </a>
            </td>
          <td>
            Olivia Wen, Samuel C Wolff, Wayne Stallaert, Didong Li, Jeremy E. Purvis, Tarek M. Zikry
          </td>
          <td>2024-11-21</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor heterogeneity is a significant factor influencing cancer treatment effectiveness and can arise from genetic, epigenetic, and phenotypic variations among cancer cells. Understanding how tumor heterogeneity impacts tumor evolution and therapy response can lead to more effective treatments and improved patient outcomes. Traditional bulk genomic approaches fail to provide insights into cellular-level events, whereas single-cell RNA sequencing (scRNA-seq) offers transcriptomic analysis at the individual cell level, advancing our understanding of tumor growth, progression, and drug response. However, implementing single-cell approaches in clinical trials involves challenges, such as obtaining high-quality cells, technical variability, and the need for complex computational analysis. Effective implementation of single-cell genomics in clinical trials requires a collaborative “Team Medicine” approach, leveraging shared resources, expertise, and workflows. Here, we describe key technical considerations in implementing the collection of research biopsies and lessons learned from integrating scRNA-seq into City of Hope’s clinical trial design, highlighting collaborative efforts between computational and clinical teams across breast, brain, and ovarian cancer studies to understand the composition, phenotypic state, and underlying resistance mechanisms within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e501cb60571ffc07f7277f4aa067130d5278a3" target='_blank'>
              Single-Cell Transcriptomics Sheds Light on Tumor Evolution: Perspectives from City of Hope’s Clinical Trial Teams
              </a>
            </td>
          <td>
            Patrick A. Cosgrove, Andrea H. Bild, T. Dellinger, B. Badie, J. Portnow, Aritro Nath
          </td>
          <td>2024-12-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Breast cancer has the highest global incidence among all cancers, affecting more than 2 million individuals annually. Despite the availability of new drugs and novel treatment combinations, it is postulated that the incidence and mortality of breast cancer will rise by 40.8% and 51.9% respectively by 2040. Such dire statistics are associated with the clonal evolution of cancer cells that leads to therapeutic resistance and consequent relapse in breast cancer patients. On the other hand, the tumor microenvironment (TME) comprising of tumor cells, cancer-associated immune cells, re-programmed stromal cells, and the extracellular matrix (ECM) creates an immunosuppressive niche facilitating immune evasion. This review focuses on a critical cellular component of the tumor microenvironment, the tumor-associated macrophages (TAMs) in breast cancer immunotherapy. Macrophages are inherently plastic and can convert from an anti-tumor M1 phenotype to a pro-tumor M2 phenotype based on microenvironmental cues. Cancer cells facilitate these cues, allowing the tumor-associated macrophages to gain M2 phenotype and mediate immune evasion. Therefore, knowledge of the distinct role of tumor-associated macrophages in immune evasion can help design therapeutics such as engineered macrophages, M2 targeting drugs, and novel macrophage-mediated drug delivery strategies for long-term survival in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6e4305fc222b5a36786a300bfa76f97eae0c28" target='_blank'>
              Reprogramming the tumor microenvironment – macrophages emerge as key players in breast cancer immunotherapy
              </a>
            </td>
          <td>
            Ana Sami, Afsheen Raza
          </td>
          <td>2024-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc282ab7c49fa5f1e3334e1737972fbc08f7f900" target='_blank'>
              Single cell analysis identified a basal cell transition state associated with the development and progression of bladder cancer
              </a>
            </td>
          <td>
            Yang Li, Pengjie Shi, Yuhong Ding, Zhipeng Yao, Lilong Liu, Junyi Hu, Zhenghao Liu, Jinxu Li, Ke Chen, Yaxin Hou
          </td>
          <td>2024-11-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to adapt to and exploit their microenvironment for sustained growth. The liver is a common site of metastasis, but the interactions between tumor cells and hepatocytes remain poorly understood. In the context of liver metastasis, these interactions play a crucial role in promoting tumor survival and progression. This study leverages multiomics coverage of the microenvironment via liquid chromatography and high-resolution, high-mass accuracy mass spectrometry-based untargeted metabolomics, 13C-stable isotope tracing, and RNA sequencing to uncover the metabolic impact of co-localized primary hepatocytes and a colon adenocarcinoma cell line, SW480, using a 2D co-culture model. Metabolic profiling revealed disrupted Warburg metabolism with an 80% decrease in glucose consumption and 94% decrease in lactate production by hepatocyte-SW480 co-cultures relative to SW480 control cultures. Decreased glucose consumption was coupled with alterations in glutamine and ketone body metabolism, suggesting a possible fuel switch upon co-culturing. Further, integrated multiomic analysis indicates that disruptions in metabolic pathways, including nucleoside biosynthesis, amino acids, and TCA cycle, correlate with altered SW480 transcriptional profiles and highlight the importance of redox homeostasis in tumor adaptation. Finally, these findings were replicated in 3-dimensional microtissue organoids. Taken together, these studies support a bioinformatic approach to study metabolic crosstalk and discovery of potential therapeutic targets in preclinical models of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8536f19d4a9f802b3a9ad739c338954491fd6245" target='_blank'>
              Deciphering Colorectal Cancer-Hepatocyte Interactions: A Multiomic Platform for Interrogation of Metabolic Crosstalk in the Liver-Tumor Microenvironment
              </a>
            </td>
          <td>
            Alisa B. Nelson, Lyndsay E. Reese, Elizabeth Rono, Eric D. Queathem, Yinjie Qiu, Braedan M. McCluskey, Alexandra Crampton, Eric Conniff, Katherine A Cummins, Ella Boytim, Senali Dansou, Justin H. Hwang, S. Safo, Patrycja Puchalska, David K. Wood, K. Schwertfeger, Peter A. Crawford
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81af71d49f35f4ca08b391cf52780be4b4e726a9" target='_blank'>
              Integrated single-cell analysis reveals distinct epigenetic-regulated cancer cell states and a heterogeneity-guided core signature in tamoxifen-resistant breast cancer
              </a>
            </td>
          <td>
            Kun Fang, A. Ohihoin, Tianxiang Liu, Lavanya Choppavarapu, Bakhtiyor Nosirov, Qianben Wang, Xue-Zhong Yu, S. Kamaraju, Gustavo W. Leone, Victor X Jin
          </td>
          <td>2024-11-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e7cfabf5bb0faa15589259ab10869fb550df73" target='_blank'>
              Divergent WNT signaling and drug sensitivity profiles within hepatoblastoma tumors and organoids
              </a>
            </td>
          <td>
            T. A. Kluiver, Yuyan Lu, Stephanie A Schubert, Lianne Kraaier, F. Ringnalda, P. Lijnzaad, Jeff DeMartino, Wouter L Megchelenbrink, Vicky Amo-Addae, Selma Eising, F. D. de Faria, Daniel Münter, M. van de Wetering, K. Kerl, E. Duiker, Marius C van den Heuvel, Vincent E. de Meijer, R. D. de Kleine, J. J. Molenaar, T. Margaritis, Hendrik G. Stunnenberg, R. R. de Krijger, J. Zsiros, H. Clevers, Weng Chuan Peng
          </td>
          <td>2024-11-20</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0c7c17d016e29c5fa8d5c2c935484d2301a05e" target='_blank'>
              Anaplastic thyroid carcinoma: vimentin segregates at the invasive front of tumors in a murine xenograft model.
              </a>
            </td>
          <td>
            Alessandro Miraglia, L. Giannotti, F. De Nuccio, A. S. Treglia, Michele Maffia, D. Lofrumento, Bruno Di Jeso, G. Nicolardi
          </td>
          <td>2024-11-18</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Summary Haematopoietic stem and progenitor cell (HSPC) autologous gene therapies are promising treatment for a variety of blood disorders. Investigation of the long‐term HSPC clonal dynamics and other measures of safety and durability following lentiviral‐mediated gene therapies in predictive models are crucial for assessing risks and benefits in order to inform decisions regarding wider utilization. We established an autologous lentivirally barcoded HSPC transplantation model in rhesus macaque (RM), a model offering insights into haematopoiesis and gene therapies with direct relevance to human. Healthy young adult RMs underwent total body irradiation, followed by transplantation of autologous HSPCs transduced with a lentiviral vector containing a diverse genetic barcode library, uniquely labelling individual HSPCs and their progeny. With up to 131 months of follow‐up, we now report quantitative clonal dynamics, characterizing the number, diversity, stability and lineage bias of hundreds of thousands of HSPC clones tracked in five RMs. We documented long‐term stable and multi‐lineage output from a highly polyclonal pool of HSPCs. Clonal succession after stable haematopoietic reconstitution was minimal. There was no evidence for accelerated acquisition of acquired somatic mutations following autologous lentivirally transduced HSPC transplantation. Our results provide relevant insights into long‐term HSPC behaviours in vivo following transplantation and gene therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/572ce898b59869c94f73c33f1fe01408d0f30361" target='_blank'>
              Long‐term tracking of haematopoietic clonal dynamics and mutations in non‐human primate undergoing transplantation of lentivirally barcoded haematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Rohan V. Hosuru, Jack Yang, Yifan Zhou, A. Gin, T. Hayal, So Gun Hong, Cynthia E. Dunbar, Chuanfeng Wu
          </td>
          <td>2024-11-10</td>
          <td>British Journal of Haematology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Triple negative breast cancer (TNBC) that fails to respond to neoadjuvant chemotherapy (NACT) can be lethal. Developing effective strategies to eradicate chemoresistant disease requires experimental models that recapitulate the heterogeneity characteristic of TNBC. To that end, we established a biobank of 92 orthotopic patient-derived xenograft (PDX) models of TNBC from the tumors of 75 patients enrolled in the ARTEMIS clinical trial (NCT02276443) at MD Anderson Cancer Center, including 12 longitudinal sets generated from serial patient biopsies collected throughout NACT and from metastatic disease. Models were established from both chemosensitive and chemoresistant tumors, and nearly 30% of PDX models were capable of lung metastasis. Comprehensive molecular profiling demonstrated conservation of genomes and transcriptomes between patient and corresponding PDX tumors, with representation of all major transcriptional subtypes. Transcriptional changes observed in the longitudinal PDX models highlight dysregulation in pathways associated with DNA integrity, extracellular matrix interactions, the ubiquitin-proteasome system, epigenetics, and inflammatory signaling. These alterations reveal a complex network of adaptations associated with chemoresistance. This PDX biobank provides a valuable resource for tackling the most pressing issues facing the clinical management of TNBC, namely chemoresistance and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4f115ab3999452774821dd883be5f67205d676c" target='_blank'>
              An annotated biobank of triple negative breast cancer patient-derived xenografts featuring treatment-naïve and longitudinal samples throughout neoadjuvant chemotherapy
              </a>
            </td>
          <td>
            Amanda L. Rinkenbaugh, Yuan Qi, Shirong Cai, Jiansu Shao, F. Hancock, Sabrina Jeter-Jones, Xiaomei Zhang, Emily Powell, Lei Huo, R. Lau, Chunxiao Fu, R. Gould, P. den Hollander, E. Ravenberg, J. White, G. Rauch, Banu Arun, C. Yam, Alastair M. Thompson, Gloria V. Echeverria, S. Moulder, W. Symmans, Jeffrey T. Chang, H. Piwnica-Worms
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Inherited genetic variants contribute to Barrett’s esophagus (BE) and esophageal adenocarcinoma (EAC) but it is unknown which cell types are involved in this process. We performed single cell RNA-sequencing of BE, EAC and paired normal tissues and integrated data of a genome-wide association study to determine cell type-specific genetic risk and cellular processes that contribute to BE and EAC. The analysis revealed that EAC development is driven to a greater extent by local cellular processes than BE development. One cell type of BE origin (BE-EAC) and cellular processes that control the differentiation of columnar cells are of particular relevance for EAC development. Further, specific subtypes of fibroblasts and endothelial cells contribute to BE and EAC development, while dendritic cells and CD4+ memory T cells contribute exclusively to BE development. The diagnostic use of markers characterizing the identified cell types and cellular processes should be explored in future for EAC prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cab9925c507d28dd37c045256d8f4a2182543fb5" target='_blank'>
              Single cell analysis of Barrett’s esophagus and carcinoma reveals cell types conferring risk via genetic predisposition
              </a>
            </td>
          <td>
            Marten C. Wenzel, Ann-Sophie Giel, Patrick S. Plum, Sascha Hoppe, Marek Franitza, Christoph Jonas, Pouria Dasmeh, René Thieme, Yue Zhao, Dominik Heider, C. Palles, R. Fitzgerald, Christiane J Bruns, Reinhard Buettner, Alexander Quaas, Ines Gockel, Carlo Maj, Seung-Hun Chon, Johannes Schumacher, Axel M. Hillmer
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Uveal melanoma (UM), arising from melanocytes in the choroid, accounts for 3% to 5% of all melanocytic tumors and over 70% of intraocular malignancies. Despite effective local treatments, metastasis remains a significant challenge, with more than half of patients developing metastatic disease within ten years. Conventional therapies often yield poor outcomes, highlighting the urgent need for novel therapeutic strategies to enhance survival and prognosis for UM patients. Methods We conducted a detailed analysis of the GSE139829 dataset, focusing on scRNA-seq data from eight primary UM patients and three with metastatic disease. Through clustering and marker gene expression analyses, we identified distinct subtypes of UM tumor cells and examined their transcriptional, metabolic, and intercellular communication profiles. We developed a novel prognostic model, PCOLCE TCs Risk Score (PTRS), centered on the C5 PCOLCE+ tumor cells, which was validated through in vitro functional assays. Additionally, we performed immune infiltration and metabolic pathway analyses to elucidate tumor-immune interactions and their clinical significance. Results We identified eight distinct cell types in UM and classified tumor subpopulations into six subgroups. The C5 PCOLCE+ TCs subpopulation was highlighted as crucial in UM malignancy, demonstrating high differentiation potential and a significant role in tumor progression. CellChat analysis revealed substantial communication between C5 PCOLCE+ TCs and fibroblasts, suggesting their involvement in tumor growth and extracellular matrix remodeling. Metabolic pathway analysis indicated enhanced oxidative phosphorylation and glutathione metabolism in this subpopulation. Additionally, we developed a PTRS model based on C5 PCOLCE+ TCs, identifying CITED1 as a high-risk gene that promotes UM cell proliferation, invasion, and migration in vitro. Conclusion This study provides insights into UM metastasis via single-cell analysis, identifying C5 PCOLCE+ TCs as key malignancy drivers associated with oxidative phosphorylation and immune interactions. Our PTRS model highlights CITED1 as a high-risk gene that promotes UM cell proliferation, paving the way for new prognostic models and therapeutic targets to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960495014c1bd91ad32e09912185659c46495106" target='_blank'>
              Single-cell RNA sequencing reveals key molecular drivers and immune landscape in uveal melanoma: implications for targeted therapy and prognostic modeling
              </a>
            </td>
          <td>
            Zeyu Song, Wenwen Shao, Zhikai Xiahou, Yue Xu, Xiaofeng Zhang
          </td>
          <td>2024-11-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Despite the ability to induce strong clinical responses, therapies that target mutational drivers of oncogenic signaling are not curative in advanced cancers, as a subset of cancer cells is capable of surviving therapy and evolving resistance. While therapy resistance is commonly viewed as a cell-intrinsic phenomenon, multiple factors produced by tumors can also confer strong therapy resistance. In contrast to the detailed elucidation of molecular mediators of cell-intrinsic and stroma-mediated (SM) resistance, the relative importance of cell-intrinsic and SM resistance to in vivo therapeutic responses remains poorly defined. To address this gap of knowledge, we used a well-characterized experimental model of ALK+ lung cancer H3122 cell line that exemplifies a duality observed across common models of targetable cancers. Under standard stroma-free in vitro cultures, H3122 cells can survive under therapeutically relevant concentrations of ALK inhibitors (ALKi), eventually acquiring cell-intrinsic therapy resistance. At the same time, co-culture with stromal fibroblasts drastically reduces the sensitivity of H3122 cells to ALKi, indicating the relevance of SM resistance. To understand the impact of SM resistance on in vivo responses, we sought to identify the mechanism(s) responsible for the therapy-protective effects of stromal fibroblasts, subsequently interrogating the impact of the disruption of this mechanism on in vivo therapeutic responses. Our in vitro studies with a large panel of primary stromal fibroblast isolates identified the HGF-cMET axis as the major mechanism responsible for ALKi desensitization. Surprisingly, xenograft validation studies demonstrated a weak effect of HGF-cMET modulation. Histological analyses of tumor tissues revealed that this relative weakness is attributable to a strong, spatially limited HGF-independent component of SM resistance. Mechanistic follow-up demonstrated that the HGF-independent component of SM resistance integrates the effect of multiple well-known juxtacrine and paracrine-acting mechanisms. Whereas the multifactorial nature of stroma-mediated resistance prevented a clear assessment of its relative contribution to in vivo therapy responses, our spatial analyses indicate that SM resistance dominates the ability of tumors to avoid therapeutic elimination. Surprisingly, we found that SM resistance was also a substantial contributor to tumor relapse, indicating that in vivo therapy resistance can integrate both cell-intrinsic and cell-extrinsic effects. This duality of therapy resistance and the multifactorial underpinning of both cell-intrinsic and stroma-mediated resistance present a challenge to therapeutic strategies focused on identifying and targeting specific resistance mechanisms. Our studies suggest that this limitation can be overcome by shifting the therapeutic focus towards shared orthogonal therapeutic sensitivities of tumor cells within residual disease.
 Citation Format: Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara Henry, Viktoriya Marusyk, Chandler Gatenbee, Marilyn Bui, Jacob Scott, Philipp M. Altrock, Eric Haura, Alexander R.A. Anderson, David Basanta, Andriy Marusyk. Spatially limited stroma-mediated resistance potentiates targeted therapy resistance through an integration of multiple juxtacrine and paracrine mechanisms [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C035.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943bc0d90e7f0f13d1f5d2038bd76c206372d768" target='_blank'>
              Abstract C035: Spatially limited stroma-mediated resistance potentiates targeted therapy resistance through an integration of multiple juxtacrine and paracrine mechanisms
              </a>
            </td>
          <td>
            Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara-Chinua N Henry, Viktoriya Marusyk, Chandler D. Gatenbee, Marilyn M Bui, Jacob G. Scott, Philipp M. Altrock, E. Haura, Alexander R. A. Anderson, D. Basanta, A. Marusyk
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) is associated with a high risk of distant metastasis, particularly to the lung and liver. Both intrinsic characteristics of cancer cells and the tumor microenvironment (TME) influence TNBC growth and metastasis. The TME, comprising stromal, immune, and endothelial cells, exhibits heterogeneity both between and within cell types. Identifying the molecular characteristics of TME cells that affect cancer progression is crucial. We aim to investigate this using single-cell RNA sequencing of patient-derived xenograft (PDX) models to explore TME cells and genes involved in TNBC metastasis. We established PDX models by transplanting tumor tissues from 26 TNBC patients into immunodeficient mice. These models were categorized based on their metastatic potential and patient outcomes: non-metastatic (n=5) and metastatic (n=4). Single-cell RNA sequencing of nine PDX tumors revealed significant differences in gene expression of murine stromal cells between metastatic and non-metastatic models. We identified ten genes (Serpinb2, Spp1, Tnc, Thbs1, Timp1, Il11, Mt2, Crabp1, Cck, Mt1) that were highly expressed in the stromal cells of metastatic PDX models. To test if TNBC cells influence these fibroblast genes, we exposed NIH3T3 fibroblasts to conditioned media from the 4T1 TNBC cell line. In three independent experiments, only Crabp1 expression in NIH3T3 cells was consistently increased by 4T1- conditioned media. CRABP1, a retinoic acid-binding protein, is known for its role in differentiation and proliferation by regulating MAPK signaling. Although Crabp1-positive cancer-associated fibroblasts have been noted, its role in TNBC metastasis was previously unexplored. We investigated CRABP1 function using a CRABP1 knockdown NIH3T3 cell line. CRABP1 knockdown led to reduced invasion and migration of 4T1 cells in trans-well assays and decreased invasiveness of 4T1 spheroids in a collagen matrix. Additionally, CRABP1 knockdown NIH3T3 cells showed reduced proliferation in vitro and fewer alpha-SMA positive cancer-associated fibroblasts in co-injected 4T1 tumors in BALB/C mice. In summary, CRABP1, identified through single-cell RNA sequencing of stromal cells in TNBC PDX models, is a potential regulator of TNBC metastasis, affecting cancer cell migration, invasion, and fibroblast proliferation.
 Citation Format: Woohang Heo, Yujeong Her, Sieun Yang, Dakyung Lee, Rokhyun Kim, Jong-Il Kim, Hyeong-Gon Moon. Single cell RNA sequencing of triple negative breast cancer patient-derived xenograft model identifies CRABP1 of cancer-associated fibroblast as a key regulator of breast cancer metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f55fcf53482b3293e557961f04bd5df55bfd4945" target='_blank'>
              Abstract A034: Single cell RNA sequencing of triple negative breast cancer patient-derived xenograft model identifies CRABP1 of cancer-associated fibroblast as a key regulator of breast cancer metastasis
              </a>
            </td>
          <td>
            W. Heo, Y. Her, Sieun Yang, Dakyung Lee, Rokhyun Kim, Jong-Il Kim, H. Moon
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1000f3a44e28fde956bb08113a825d4344bfa7a6" target='_blank'>
              Proteogenomic analysis reveals non-small cell lung cancer subtypes predicting chromosome instability, and tumor microenvironment
              </a>
            </td>
          <td>
            K. J. Song, Seunghyuk Choi, Kwoneel Kim, H. S. Hwang, E. Chang, Ji Soo Park, Seok Bo Shim, Seunghwan Choi, Yong Jin Heo, Woo Ju An, Dae Yeol Yang, Kyung-Cho Cho, W. Ji, Chang-Min Choi, Jae Cheol Lee, H. Kim, Jiyoung Yoo, Hee-Sung Ahn, Gang-Hee Lee, Chanwoong Hwa, Seoyeon Kim, Kyunggon Kim, Minsuh Kim, Eunok Paek, Seungjin Na, S. J. Jang, J. An, Kwang Pyo Kim
          </td>
          <td>2024-11-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 The tumor microenvironment plays a crucial role in cancer progression and treatment response. Systemically administered metabolic cancer therapies target not only malignant but also stromal cells, including immune cells - the only stromal population previously addressed in this context. However, how such treatments impact other non-malignant cell types in tumors remains poorly understood. Here we show that inhibition of de novo pyrimidine synthesis in endothelial cells accelerates tumor growth and alters tumor immune repertoire. We found that whole-body de novo pyrimidine synthesis deficiency in mice, caused by the inducible whole-body ablation of DHODH, accelerates the growth rate of orthotopic lung tumors. Single-cell transcriptomic analysis of tumor-bearing lungs revealed that DHODH deficiency in the stroma impacts multiple cell populations, including immune and, surprisingly, endothelial cells. To explore the endothelial-specific effects, we generated a mouse model with inducible DHODH deficiency restricted to endothelial cells. The endothelium-specific model recapitulates the accelerated lung tumor growth observed in the whole-body DHODH deficiency model. Single-cell transcriptomics analysis of tumor-bearing lungs in the endothelium-specific model pointed to changes in the immune repertoire, particularly an enrichment of monocytes. We confirmed these results on the protein level using spectral flow cytometry, and we are currently in the process of uncovering the mechanism by which endothelial deficiency of pyrimidine synthesis affects the immune landscape of tumors. Inhibitors of pyrimidine de novo synthesis have been tested in clinical trials but failed due to their low efficiency. Our findings indicate that systemic treatment targeting pyrimidine synthesis may be hampered by its pro-tumorigenic effects in the endothelium, highlighting the unexpected role of endothelial metabolism in this context.
 Citation Format: Petra Hyrossova, Isidora Milisav, Silvia Novais, Mirko Milosevic, Jakub Rohlena, Katerina Rohlenova. Endothelial pyrimidine synthesis deficiency promotes tumor growth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae32fff34fa2e0903b1542a8baef25671307eb63" target='_blank'>
              Abstract A018: Endothelial pyrimidine synthesis deficiency promotes tumor growth
              </a>
            </td>
          <td>
            Petra Hyrossova, Isidora Milisav, S. Novais, Mirko Milosevic, Jakub Rohlena, Katerina Rohlenova
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Lung Adenocarcinoma (LUAD) leads in cancer deaths worldwide. Immune Checkpoint blockade (ICB) therapy has improved patient survival of some subtypes (like Kras oncogene-driven; loss of tumor-suppressor p53– “KP”), but not others (Kras oncogene-driven; loss of LKB1– “KL”). Some studies suggest the poorly inflamed KL tumor environment reduces T cell infiltration. However, the impact of other immune mechanisms/cell types on overall ICB-responsiveness remains understudied. Tertiary lymphoid structures (TLSs) are T/B cell aggregates organized by various cell types that form within inflamed non-lymphoid organs. Recent studies have strongly correlated tumor-associated TLSs (TA-TLSs) with positive clinical outcomes and are predictive of patient responders to immunotherapy in many solid tumor types. Thus, understanding the mechanistic factors that sustain mature TA-TLSs will provide more clarity on what molecularly entails an optimal immune response within an ICB-treated patient. To that end, a major advancement has been our knowledge of TA-TLS phenotypic markers driven through developments in multiplex immunofluorescence imaging of 2D tissue sections. However, tissue sectioning destroys the organ architecture, providing limited spatial information over a small field of view. Therefore, non-destructive 3D fluorescent imaging of whole organs, and lightsheet fluorescence microscopy (LSFM) can provide a novel and broader global view of multicellular structures, like TA-TLSs within the whole organ. By using the Blaze ultramicroscope, we imaged whole lung lobes from our novel autochthonous neoantigen-induced lung adenocarcinoma cancer models of KP and KL mice based on our previously published immunogenic KP x NINJA (inversion-induced joined neoantigen) model, which relies on Cre- recombinase activity dependent activation of KrasLox-Stop-Lox (LSL)G12D and deletion of p53fl/fl. Then, doxycycline and tamoxifen administration lead to lung-specific promoter [club cell secretory protein (CCSP)-rtTA] dependent expression of NINJA. We have also developed novel image segmentation methods using machine learning tools in Imaris 10.2 to identify tumors and immune cell clusters, enabling the quantification of the distance and volume of observable tumors and TLSs in the entire organ, which to our knowledge has not been previously attempted. Using this novel imaging and quantification pipeline, we recently discovered that while KP-NINJA lungs produce many TA-TLSs, the KL tumors (compared to KP) lack organized TA-TLSs, and that intriguingly, B-cells are sparsely sequestered near the main bronchus, away from tumors, resembling small bronchi-associated lymphoid structures (BALTs) that typically form in inflamed lungs. We are concurrently doing 2D multiplex cyclic-immunofluorescence imaging to characterize the variety of other cell types present in TLSs of KP-NINJA vs KL-NINJA and more closely understand the molecular cues sustaining TA-TLSs, providing clarity on what entails an “optimal” immune response within an ICB-treated patient.
 Citation Format: Advait Jeevanandam, Kelli Connolly, Nikhil Joshi. Spatial study of tertiary lymphoid structures in lung adenocarcinoma using 3D lightsheet whole-organ Imaging and machine learning-based quantification [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93e250b90708e399aa85957d7d601a70aaa9b96c" target='_blank'>
              Abstract B032: Spatial study of tertiary lymphoid structures in lung adenocarcinoma using 3D lightsheet whole-organ Imaging and machine learning-based quantification
              </a>
            </td>
          <td>
            Advait Jeevanandam, Kelli A. Connolly, Nikhil S. Joshi
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185d03d5b622173898f4a312b550c66a7f8243e6" target='_blank'>
              Soft tissue sarcomas at the single-cell and spatial resolution: new markers and targets.
              </a>
            </td>
          <td>
            M. Menyailo, Elena E. Kopantseva, A. Khozyainova, Anastasia A. Korobeynikova, E. Denisov
          </td>
          <td>2024-11-24</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaacb86999aeed003c707b4000adb56d67bb6547" target='_blank'>
              Spatial transcriptome profiling identifies DTX3L and BST2 as key biomarkers in esophageal squamous cell carcinoma tumorigenesis
              </a>
            </td>
          <td>
            Rutao Li, Na Li, Qianqian Yang, Xing Tong, Wei Wang, Chang Li, Jun Zhao, Dong Jiang, Haitao Huang, Chen Fang, Kai Xie, Jiamin Yuan, Shaomu Chen, Guangbin Li, Haitao Luo, Zhibo Gao, Dongfang Wu, Xiaoli Cui, Wei Jiang, Lingchuan Guo, Haitao Ma, Yu Feng
          </td>
          <td>2024-12-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1df19dd7f1af96f97d3c64e2cfe897bb19c7e2" target='_blank'>
              Multiomics integration and machine learning reveal prognostic programmed cell death signatures in gastric cancer
              </a>
            </td>
          <td>
            Zihao Bai, Hao Wang, Jingru Han, Jia An, Zhaocong Yang, Xuming Mo
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The resistance of tumors to immune checkpoint inhibitors (ICI) may be intricately linked to cellular senescence, although definitive clinical validation remains elusive. In this study, comprehensive pan-cancer scRNA-seq analyses identify fibroblasts as exhibiting the most pronounced levels of cellular senescence among tumor-associated cell populations. To elucidate this phenomenon, a fibroblast senescence-associated transcriptomic signature (FSS), which correlated strongly with protumorigenic signaling pathways and immune dysregulation that fosters tumor progression, is developed. Leveraging the FSS, the machine learning (ML) framework demonstrates exceptional accuracy in predicting ICI response and survival outcomes, achieving superior area under curve (AUC) values across validation, testing, and in-house cohorts. Strikingly, FSS consistently outperforms established signatures in predictive robustness across diverse cancer subtypes. From an integrative analysis of 17 CRISPR/Cas9 libraries, CDC6 emerges as a pivotal biomarker for pan-cancer ICI response and prognostic stratification. Mechanistically, experimental evidence reveals that CDC6 in tumor cells orchestrates fibroblast senescence via TGF-β1 secretion and oxidative stress, subsequently reprogramming the tumor microenvironment and modulating ICI response. These findings underscore the translational potential of targeting fibroblast senescence as a novel therapeutic strategy to mitigate immune resistance and enhance antitumor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83eb29659bd661ae208e4642398f8b3b7069e404" target='_blank'>
              A Distinguished Roadmap of Fibroblast Senescence in Predicting Immunotherapy Response and Prognosis Across Human Cancers.
              </a>
            </td>
          <td>
            Dongjie Chen, Pengyi Liu, Jiayu Lin, Longjun Zang, Yihao Liu, Shuyu Zhai, Xiongxiong Lu, Yuanchi Weng, Hongzhe Li
          </td>
          <td>2024-12-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Anaplastic Thyroid Cancer (ATC) is an aggressive form of cancer with poor prognosis, heavily influenced by its tumor immune microenvironment (TIME). Understanding the cellular and gene expression dynamics within the TIME is crucial for developing targeted therapies. This study analyzes the immune microenvironment of ATC and Papillary Thyroid Cancer (PTC) using single-cell RNA sequencing (scRNA-seq). We performed a comprehensive scRNA-seq analysis on ATC and PTC samples, incorporating cell type annotation, marker gene identification, and clustering based on gene expression. A specific focus was on the prevalence and biomarkers of Pre-Exhausted CD8+ T cells in ATC, utilizing the single-cell tumor immune atlas for immune cell characterization. The scRNA-seq analysis identified distinct immune cell populations and differentially expressed genes in ATC and PTC samples. Notably, Pre-Exhausted CD8+ T cells were found to be prevalent in ATC datasets. Additional immunofluorescence staining and co-culture experiments with the ATC cell line identified GNLY, a member of the saposin-like protein family as a potential biomarker for Pre-Exhausted CD8+ T cells in ATC. This study provides valuable insights into the immune landscape of ATC, emphasizing the prevalence of Pre-Exhausted CD8+ T cells and identifying GNLY as a potential biomarker. Understanding the TIME composition and the role of specific immune cells in cancer progression can inform the development of targeted immunotherapies for ATC. Future research should explore the functional implications of GNLY and other identified biomarkers in modulating the immune response in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bee3b0c483d956dde8850d164e4f812a4b30008" target='_blank'>
              Persistent pre-exhausted CD8+ T cells shape the tumor immune microenvironment in anaplastic thyroid cancer.
              </a>
            </td>
          <td>
            X. Ruan, Mei Tao, Yuanxing Dong, Linfei Hu, Guangwei Xu, Chuanxiang Hu, Yue Huang, Yuqi Wang, Jialong Yu, Wei Luo, Ming Gao, Min Zhao, Xiangqian Zheng
          </td>
          <td>2025-01-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Lung adenocarcinoma (LUAD) is the leading cause of cancer-related mortality, largely due to its high prevalence rate and limited therapeutic options. Immunotherapy remains one of the most promising therapeutic avenues, yet further research is necessary to enhance therapy effectiveness for a broader range of patients. Central to the mechanism of immunotherapy is the detection of tumor neoantigens by cytotoxic T cells, which trigger the elimination of tumor cells. However, it is largely unknown how the early interaction between T cells and neoantigen-expressing tumor cells affects the acquisition of cancer hallmarks and tumor progression. To study this, our lab has developed the iNversion-INduced Joined neoAntigen (NINJA) genetically engineered mouse model in which we can initiate expression of a neoantigen in Kras-mutant, Trp53-null, tdTomato-expressing lung adenocarcinoma cells (KPT-NINJA). This system enables us to analyze tumor initiation, progression, and advanced LUAD development over 20 weeks in both neoantigen-expressing and non-expressing tumors. We defined cancer hallmark acquisition and tumor progression in our model by using a combination of multiplex immunofluorescence and computational algorithms to quantify tumor size, immune infiltration, metastasis, and expression of cancer molecular markers p-ERK and Ki67. We isolated tumor cell and immune cell subsets using neural networks to segment cells and tumor structures and used the colocalization of molecular markers to confirm the identity of various cell types. Our initial data validated that within immunogenic tumors there is an increase of T cell infiltration in tumors and have found immunogenic tumors are associated with overall smaller tumors at early time points. Interestingly, these tumors are associated with a decrease in p-ERK expression, suggesting immune infiltration is negatively correlated with p-ERK expression and tumor size. Together, these data indicate that increased immunogenic pressure may be disrupting early tumor proliferation and hallmark progression, shaping the trajectory of development at a very early stage. We are also correlating other molecular markers of tumor progression with immune infiltration, such as Ki67 expression, which we will colocalize with the other findings to construct a model of how tumors acquire hallmark capabilities from an immunogenic perspective. By isolating the mechanism of early tumor progression in immunogenic contexts, we can begin to predict disease trajectory. This will contribute to a better understanding of the mechanisms behind immune evasion and may uncover crucial insights to broaden immune therapy efficacy.
 Citation Format: Ishan Bansal, Jennifer Loza, Brian G. Hunt, Kelli Connolly, Srividhya Venkatesan, Nikhil Joshi. Hallmarks of progression in early neoantigen-expressing lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B013.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6754e5bab1083c1c562c8d6c1058076e47cfd9e" target='_blank'>
              Abstract B013: Hallmarks of progression in early neoantigen-expressing lung adenocarcinoma
              </a>
            </td>
          <td>
            Ishan Bansal, Jennifer L. Loza, Brian G. Hunt, Kelli A. Connolly, Srividhya Venkatesan, Nikhil S. Joshi
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Metastatic bladder cancer (BLCA) is an aggressive disease complicated by the emergence of variant histological subtypes. The UW/Fred Hutch metastatic BLCA rapid autopsy program is a first-of-its-kind program collecting cell-free DNA (cfDNA) and matched normal tissue, primary tumor, and metastatic tumor samples from advanced BLCA patients, with a focus on these variant subtype tumors. This dataset, consisting of 20 patients with up to five metastases per patient, is a unique opportunity to assess how cfDNA is able to capture heterogeneity across BLCA. We assessed both inter-patient heterogeneity between BLCA subtypes and inter-tumoral heterogeneity within each patient using epigenetic and genomic profiling. Using nucleosome profiling, we identified the activity of lineage markers and transcription factors that distinguish healthy individuals from BLCA patients, and between BLCA subtypes. Furthermore, we conducted a detailed analysis of whole genome (30X) and targeted (4,000X) cfDNA sequencing in their ability to capture the evolutionary history of tumor mutations. Detection of tumor mutations by cfDNA is highly dependent on clonality, with >90% of founder tumor mutations captured by either sequencing method, while capture is decreased for either subclonal mutations or those private to a single metastasis. Importantly, deleterious mutations in BLCA driver genes had close to 100% detection using targeted panel sequencing, even for subclonal mutations. These findings lay the groundwork for the use of cfDNA in clinical BLCA decision-making, including tracking the emergence of pathogenic, targetable mutations and variant subtypes.
 Citation Format: Samantha L Schuster, Pushpa Itagi, Sonali Arora, Patricia C Galipeau, Thomas W Persse, Michael Yang, Allie Kreitman, Alan Min, Funda Vakar-Lopez, John K Lee, Petros Grivas, Robert B Montgomery, Jonathan L Wright, Andrew C Hsieh, Hung- Ming Lam, Gavin Ha. Cell-free DNA captures inter- and intra-patient heterogeneity in advanced bladder cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7eb84ec36801f20a2ac043ba0422da74b9249b7" target='_blank'>
              Abstract B040: Cell-free DNA captures inter- and intra-patient heterogeneity in advanced bladder cancer
              </a>
            </td>
          <td>
            Samantha L Schuster, P. Itagi, Sonali Arora, Patricia Galipeau, Thomas Persse, Michael Yang, Allie Kreitman, Alan Min, F. Vakar‐López, John K. Lee, P. Grivas, R. Montgomery, Jonathan L. Wright, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Despite the pivotal role of cytotoxic T lymphocytes (CTLs) in anti-tumor immunity, a substantial proportion of CTL-rich hepatocellular carcinoma (HCC) patients experience early relapse or immunotherapy resistance. However, spatial immune variations impacting the heterogeneous clinical outcomes of CTL-rich HCCs remain poorly understood. Here, we compared the single-cell and spatial landscapes of 20 CTL-rich HCCs with distinct prognoses using multiplexed in situ staining and validated the prognostic value of myeloid spatial patterns in a cohort of 386 patients. Random forest and Cox regression models identified macrophage aggregation as a distinctive spatial pattern characterizing a subset of CTL-rich HCCs with an immunosuppressive microenvironment and poor prognosis. Integrated analysis of single-cell and spatial transcriptomics, combined with in situ staining validation, revealed that spatial aggregation enhanced pro-tumoral macrophage reprogramming in HCCs, marked by lipid metabolism orientation, M2-like polarization, and increased adjacent CTL exhaustion. This spatial effect on macrophage reprogramming was replicated in HCC-conditioned human macrophage cultures, which showed an enhanced capability to suppress CTLs. Notably, increased macrophage aggregation was associated with higher response rates to anti-PD-1 immunotherapy. These findings suggest that the spatial distribution of macrophages is a biomarker of their functional diversities and microenvironment status, which holds prognostic and therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0a3e561fbd74941c09e71f727074ba04c903f0f" target='_blank'>
              Spatial Organization of Macrophages in CTL Rich Hepatocellular Carcinoma Influences CTL Antitumor Activity.
              </a>
            </td>
          <td>
            Yulan Weng, Lu Wang, Yuting Wang, Junyu Xu, Xiaoli Fan, Shufeng Luo, Qiaomin Hua, Jing Xu, Gaoteng Liu, , Chang-An Zhao, Dong-Ming Kuang, Chong Wu, Limin Zheng
          </td>
          <td>2025-01-02</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286075cb8f68cf0d587bedcc36d60116c411594e" target='_blank'>
              Spatially resolved gene expression profiling of tumor microenvironment reveals key steps of lung adenocarcinoma development
              </a>
            </td>
          <td>
            Yuma Takano, Jun Suzuki, Kotaro Nomura, Gento Fujii, Junko Zenkoh, Hitomi Kawai, Yuta Kuze, Y. Kashima, Satoi Nagasawa, Yuka Nakamura, M. Kojima, K. Tsuchihara, Masahide Seki, A. Kanai, D. Matsubara, Takashi Kohno, Masayuki Noguchi, Akihiro Nakaya, Masahiro Tsuboi, G. Ishii, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-12-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Lineage plasticity and histologic transformation from prostate adenocarcinoma to neuroendocrine prostate cancer (NEPC) occurs in up to 15-20% of patients with castration-resistant prostate cancer (CRPC) as mechanism of treatment resistance and is associated with aggressive disease and poor prognosis. NEPC tumors typically display small cell carcinoma morphology with loss of androgen receptor (AR) expression and gain of neuroendocrine (NE) lineage markers. However, there is a spectrum of phenotypes that are observed during the lineage plasticity process, and the clinical significance of mixed histologies or those that co-express AR and NE markers or lack all markers is not well defined. Translational research studies investigating NEPC have used variable definitions making clinical trial design challenging. Here we discuss the diagnostic workup of metastatic biopsies to help guide the reproducible classification of phenotypic CRPC subtypes. We recommend classifying CRPC tumors based on histomorphology (adenocarcinoma, small cell carcinoma, poorly differentiated carcinoma, other morphologic variant, or mixed morphology) and immunohistochemical markers with a priority for AR, NKX3.1, INSM1, synaptophysin and cell proliferation based on Ki-67 positivity, with additional markers to be considered based on the clinical context. Ultimately, a unified workup of metastatic CRPC biopsies can improve clinical trial design and eventually practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ba90573d033c02c8e63a055556c5045dc3b6fc" target='_blank'>
              Framework for the Pathology Workup of Metastatic Castration-Resistant Prostate Cancer Biopsies.
              </a>
            </td>
          <td>
            Michael C Haffner, Michael J. Morris, C. C. Ding, Erolcan Sayar, Rohit Mehra, Brian D. Robinson, Lawrence D. True, Martin Gleave, Tamara L. Lotan, Rahul Aggarwal, Jiaoti Huang, Massimo Loda, Peter S Nelson, Mark A. Rubin, H. Beltran
          </td>
          <td>2024-11-26</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/602ca4326619720e872bb3125e86fd5844dc2d44" target='_blank'>
              A multiscale model of immune surveillance in micrometastases gives insights on cancer patient digital twins
              </a>
            </td>
          <td>
            Heber L Rocha, Boris Aguilar, Michael Getz, I. Shmulevich, P. Macklin
          </td>
          <td>2024-12-04</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>2</td>
          <td>82</td>
        </tr>

        <tr id="
 Although various effective anti-cancer treatments have become available over the last decades, resistance to all available therapies remains the major cause of death of cancer patients with disseminated tumors. Striking examples are patients with triple-negative breast cancer (TNBC), which are frequently defective in the repair of DNA double strand breaks, e.g. due to loss of BRCA1 function. Because of this defect, the patients initially respond very well to DNA damage-inducing chemotherapy. Unfortunately, disseminated tumors are usually not eradicated and resistant tumor cells are eventually selected from residual primary or metastatic tumor sites. It is therefore crucial to understand the molecular mechanisms underlying the drug tolerance of the residual tumor cells. To study residual disease, we used the K14cre;Brca1 F/F ;p53 F/F (KB1P) mouse model for hereditary breast cancer, which provides the unique opportunity to explore and target those mechanisms in a fully immunocompetent model. The mammary tumors that spontaneously develop highly resemble their human counterparts, both morphologically and in their therapy response. For example, tumors shrink in response to poly(ADP-ribose) polymerase inhibition, platinum-based treatment or the commonly used doxorubicin, docetaxel and cyclophosphamide (TAC) combination therapy. But despite repeated drug sensitivity, the KB1P mammary tumors are not eradicated, not even by a frequent dosing schedule. By combining single-cell RNA sequencing, spatial transcriptomics and imaging mass cytometry, we dissected the intratumoral heterogeneity and alterations in the tumor microenvironment of residual disease. We identified specific subpopulations of tumor cells that have a survival benefit after treatment, and these occurred together with an altered microenvironment characterized by a highly reactive stroma infiltrated with both anti-inflammatory and pro-tumoral immune cells. Interestingly, those structural and transcriptional changes are reversed in the relapsed tumors, highlighting the plasticity of drug tolerance. To investigate the mechanisms of the altered tumor-stroma interactions that are relevant for drug tolerance, we have designed a custom-made CRISPR/Cas9 library based on differential gene expression of the residual tumor cells. This library enables us to functionally test relevant mechanisms of drug tolerance in vivo and we expect that these analyses will provide useful insights into the tumor-stroma interactions that contribute to residual disease. Taken together, our detailed analyses demonstrate the substantial remodeling of tumor cells in their microenvironment upon treatment. To develop new therapeutic approaches to eradicate drug-tolerant tumor cells and thereby circumvent tumor relapse, it is essential to better understand the heterogeneous cellular composition of residual tumors in its spatial context. With this project, we hope to provide comprehensive data to develop better therapeutic strategies that target the tumor-stroma interaction and eradicate residual tumors.
 Citation Format: Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, Ismar Klebic, Sven Rottenberg. Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecf546e6bc5a8fa717c4c87fcb379e8ed3399d7" target='_blank'>
              Abstract C034: Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors
              </a>
            </td>
          <td>
            Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, I. Klebic, Sven Rottenberg
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Our research aims to understand the adaptive, ergo potentially metastatic, responses of prostate cancer to changing microenvironments. Emerging evidence implicates a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study revealed that PACC presence in patient prostate tumors at the time of radical prostatectomy was predictive of future metastasis. To test for a causative relationship between PACC state biology and metastasis in prostate cancer, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) from animal models. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Specifically, we applied this approach to analysis of subcutaneous, caudal artery, and intracardiac murine models. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. Furthermore, in vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites. Additional in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence of pro-metastatic mechanisms specific to the PACC state. Implications: Considering that many anti-cancer agents induce the PACC state, our data positions the increased metastatic competency of PACC state cells as an important unforeseen ramification of neoadjuvant regimens, which may help explain clinical correlations between chemotherapy and metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c130a571d99ddcd76ddd1df86bc68f827f48ad3" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic.
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T A Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, S. Amend
          </td>
          <td>2024-12-10</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="BACKGROUND
A spatially resolved, niche-level analysis of tumour microenvironments (TME) can provide insights into cellular interactions and their functional impacts in gastric cancers (GC).


OBJECTIVE
Our goal was to translate the spatial organisation of GC ecosystems into a functional landscape of cellular interactions involving malignant, stromal and immune cells.


DESIGN
We performed spatial transcriptomics on nine primary GC samples using the Visium platform to delineate the transcriptional landscape and dynamics of malignant, stromal and immune cells within the GC tissue architecture, highlighting cellular crosstalks and their functional consequences in the TME.


RESULTS
GC spatial transcriptomes with substantial cellular heterogeneity were delineated into six regional compartments. Specifically, the fibroblast-enriched TME upregulates epithelial-to-mesenchymal transformation and immunosuppressive response in malignant and TME cells, respectively. Cell type-specific transcriptional dynamics revealed that malignant and endothelial cells promote the cellular proliferations of TME cells, whereas the fibroblasts and immune cells are associated with procancer and anticancer immunity, respectively. Ligand-receptor analysis revealed that CCL2-expressing fibroblasts promote the tumour progression via JAK-STAT3 signalling and inflammatory response in tumour-infiltrated macrophages. CCL2+ fibroblasts and STAT3-activated macrophages are co-localised and their co-abundance was associated with unfavourable prognosis. We experimentally validated that CCL2+ fibroblasts recruit myeloid cells and stimulate STAT3 activation in recruited macrophages. The development of immunosuppressive TME by CCL2+ fibroblasts were also validated in syngeneic mouse models.


CONCLUSION
GC spatial transcriptomes revealed functional cellular crosstalk involving multiple cell types among which the interaction between CCL2+ fibroblasts and STAT3-activated macrophages plays roles in establishing immune-suppressive GC TME with potential clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe72f4555eff64ae7e607a98dd2b2dd8cb471c6e" target='_blank'>
              Spatial dissection of tumour microenvironments in gastric cancers reveals the immunosuppressive crosstalk between CCL2+ fibroblasts and STAT3-activated macrophages.
              </a>
            </td>
          <td>
            Sung Hak Lee, Dagyeong Lee, Junyong Choi, Hye Jeong Oh, In-Hye Ham, Daeun Ryu, Seo-Yeong Lee, Dong-Jin Han, Sunmin Kim, Youngbeen Moon, In-Hye Song, K. Song, Hyeseong Lee, Seungho Lee, Hoon Hur, Tae-Min Kim
          </td>
          <td>2024-11-23</td>
          <td>Gut</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Cancer arises through an evolutionary process in which somatic mutations, including single nucleotide variants (SNVs) and copy number aberrations (CNAs), drive the development of a malignant, heterogeneous tumor. Reconstructing this evolutionary history from sequencing data is critical for understanding the order in which mutations are acquired and the dynamic interplay between different types of alterations. Advances in modern whole genome single-cell sequencing now enable the accurate inference of copy number profiles in individual cells. However, the low sequencing coverage of these low pass sequencing technologies poses a challenge for reliably inferring the presence or absence of SNVs within tumor cells, limiting the ability to simultaneously study the evolutionary relationships between SNVs and CNAs. In this work, we introduce a novel tumor phylogeny inference method, Pharming, that jointly infers the evolutionary histories of SNVs and CNAs. Our key insight is to leverage the high accuracy of copy number inference methods and the fact that SNVs co-occur in regions with CNAs in order to enable more precise tumor phylogeny reconstruction for both alteration types. We demonstrate via simulations that Pharming outperforms state-of-the-art single-modality tumor phylogeny inference methods. Additionally, we apply Pharming to a triple-negative breast cancer case, achieving high-resolution, joint reconstruction of CNA and SNV evolution, including the de novo detection of a clonal whole-genome duplication event. Thus, Pharming offers the potential for more comprehensive and detailed tumor phylogeny inference for high-throughput, low-coverage single-cell DNA sequencing technologies compared to existing approaches. Availability https://github.com/elkebir-group/Pharming">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d64f3b951403db46ae5a215e58c613919dd9b93c" target='_blank'>
              Pharming: Joint Clonal Tree Reconstruction of SNV and CNA Evolution from Single-cell DNA Sequencing of Tumors
              </a>
            </td>
          <td>
            Leah L. Weber, Anna Hart, Idoia Ochoa, M. El-Kebir
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and malignant type of brain tumor in adults and no effective therapies exist to combat this disease. Surgical resection, radio- and temozolomide-based-chemotherapy are insufficient mainly due to inter- and intra-tumoral heterogeneity of GBM tumors, therefore personalized approaches are required. Additionally, diverse tumor microenvironment with a leading role of glioblastoma stem cells (GSCs) in hypoxic core underlie the aggressiveness and recurrence of GBM. However, the molecules and pathways which directly promote tumor aggressiveness together with the driving process of epithelial-to-mesenchymal transition (EMT) and the components of extracellular matrix (ECM), remain only partially covered with prior studies focused mainly on coding oncogenes and tumor suppressor genes. Over the past years, a special attention has been paid to non-coding, circular RNAs (circRNAs) differentially expressed in various cancers, including GBM. Despite high abundance and stability, the exact biological roles of most individual circRNAs have not been fully revealed, so far. Besides anti-cancer targets, circRNAs have been considered as essential biomarkers and stratifying criterion of malignancy. Thus, studies on functions of differentially expressed circRNAs in GBM are an emerging field to be explored. In herein study, the biological role of circRNA CLIP2 (circCLIP2) was found highly overexpressed in primary tumor tissues. Upon the knock-down of circCLIP2 in 2D and 3D in vitro models, functional tests performed clearly indicate a significant contribution of circCLIP2 to aggressive potential of GBM as showed by decreased rates of proliferation, migration and invasion. Additional studies indicated predominant expression of circCLIP2 in GSC fractions additionally elevated by hypoxic conditions. The increased rates of proliferation and migration as well as molecular analyses were also confirmed in in vivo settings where xenograft model was applied. The expression of EMT marker Snail2 and ECM-degrading enzyme metalloproteinase 9 (MMP9) were significantly downregulated in the tumors formed from circCLIP2 silenced cells. Taken together, these results indicate a strong contribution of circCLIP2 in the aggressive phenotype of GBM which is due to a modulation of GSC proliferative potential as well as migration and invasion via EMT together with the modulator of ECM composition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41e1172182008f639a17a4a3feb76b12f54727b" target='_blank'>
              Circular RNA - circCLIP2 is predominantly expressed in GSC niche and enhances glioblastoma aggressiveness via EMT and ECM signaling
              </a>
            </td>
          <td>
            J. Misiorek, Ż. Zarębska, K. Kuczyński, J. Latowska-Łysiak, A. Grabowska, P. Głodowicz, M. Sajek, Anna Karlik, Dorota Wronka, A. Barciszewska, Łukasz Przybył, K. Rolle
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Vγ9Vδ2T cells have the unique ability to recognize a broad range of malignant transformed cells. The tumor targeting event involving BTN2A1 and BTN3A1 dimers on the tumor cell surface is critical, leading to full activation of the TCR. Although the molecular mechanisms governing TCR engagement and T cell activation are well-characterized, the role of Vγ9Vδ2 T cells in cancer immune surveillance remains to be fully elucidated, particularly the mechanisms that enable these cells to discriminate between healthy and malignant cells at an early stage of malignant transformation. We employed two independent, genetically engineered step-wise mutagenesis models of human colorectal and breast cancer that mimic the transformation steps leading to tumor formation. We demonstrate that various single oncogenic mutations introduced into healthy organoids or cells, are sufficient to upregulate surface expressed BTN2A1 and enable Vγ9Vδ2 TCR binding to tumor cells. However, full activation of T cells through a Vγ9Vδ2TCR required additional subsequent phosphorylation of juxtamembrane (JTM) amino acids of BTN3A1, leading to the activating heterodimerization of BTN2A1 and 3A1. Using a protein interactome mapping pipeline, we identified PHLDB2, SYNJ2 and CARMIL1 as key players in controlling these delicate dual surface dynamics of BTN2A1 and 3A1 during early transformation. This mode of action allowed Vγ9Vδ2TCR T cells to control tumors in vitro and in vivo, emphasizing the crucial role of these molecules from early mutagenesis, to advanced cancer stages, and highlighting the therapeutic potential of a Vγ9Vδ2TCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae97086c70f2495b91617599cd421e999f208f8" target='_blank'>
              Sensitivity to Vγ9Vδ2TCR T cells is imprinted after single mutations during early oncogenesis
              </a>
            </td>
          <td>
            A. Cleven, A. Meringa, P. Brazda, Domenico Fascì, T. Koorman, Tineke Aarts, Inez Johanna, D. Beringer, P. Hernández-López, S. Heijhuurs, Tomohiro Mizutani, Sangho Lim, M. Huismans, Jochem Bernink, David Vargas Diaz, Wei Wu, Esther San Jose, Jelle Schipper, Nikos Tsakirakis, Lauren Hoorens van Heyningen, Annick Nouwens, Lucrezia Gatti, T. Straetemans, Hugo Snippert, Jeanine Roodhart, P. Derksen, J. Drost, M. Altelaar, Albert J. R. Heck, Hans Clevers, Juergen Kuball, Z. Sebestyén
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) treatment is hindered by a dearth of representative mouse GBM preclinical models in immunocompetent mice. Here, we characterized five murine GBM stem-like cell (mGSC) models derived from lentivirus-induced tumors in transgenic mice that are driven by the activation of the Nf1-Ras signaling pathway and inactivation of Tp53.



 MGSC lines (005, RIG, NF53, C1, and C3) were cultured as spheres in serum-free stem cell media. Whole exome sequencing (WES) was employed to quantify single nucleotide polymorphisms (SNPs). Stem cell properties were characterized by stemness in vitro and tumorigenicity after intracerebral implantation in C57BL/6 mice. Tumor phenotypes and the immune microenvironment were characterized by immunohistochemistry, flow cytometry, and RNA sequencing.



 WES revealed a large variation in coding sequence SNPs across mGSC lines (~20-fold), likely influenced by the mixed backgrounds of the parental mice. MGSCs exhibited variable clonogenic sphere formation and CD133 expression levels. In vivo, they consistently initiated lethal malignant gliomas, with median survival ranging from 29 to 82 days, and showed strong CD44 expression and variable invasiveness. The tumor microenvironment featured an abundance of CD68+ macrophages and uniform high PD-L1+ myeloid cells, while T cell infiltration varied among the models, with low mutation burden C1 and C3 exhibiting fewer tumor-infiltrating T cells.



 Upon orthotopic implantation in immunocompetent mice, mGSCs generate tumors characteristic of human GBM. Despite similar strategies to generate these mGSCs, they exhibited a range of phenotypes and immune profiles in mGSC-derived orthotopic tumors. These mGSCs provide new preclinical GBM models for developing GBM immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85632e943cebc73850dcb33bb8236cdd57ebd815" target='_blank'>
              immunocompetent murine glioblastoma stem-like cell models exhibiting distinct phenotypes
              </a>
            </td>
          <td>
            Kimia Kardani, S. Ghouse, Muzammil Arif Din Abdul Jabbar, Namita Rajasubramanian, Judit Sánchez Gil, A. Stemmer-Rachamimov, Yasushi Soda, R. Martuza, Toshiro Hara, Hiroaki Wakimoto, S. Rabkin
          </td>
          <td>2024-12-07</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is the most prevalent malignant brain tumor, with an average survival time of 14 to 20 months. Its capacity to invade brain parenchyma leads to the failure of conventional treatments and subsequent tumor recurrence. Recent studies have explored new therapeutic strategies using a chemoattracting gradient to attract GBM cells into a soft hydrogel trap where they can be exposed to higher doses of radiation or chemotherapy. It has been demonstrated in vitro under static conditions, that nanoparticles (NPs) encapsulating the chemoattractant CXCL12 can create a gradient to attract GBM cell. However, GBM cell invasion is also largely dependent on interstitial fluid flow (IFF). In the present study, a custom-made in vitro 3D model with indirect perfusion to mimic IFF at flow rates of 0.5 μL/min and 3 μL/min was used to examine the invasive behavior of F98-rodent-derived and U87-human-derived GBM cells. This model simulated IFF and CXCL12 gradient within an alginate:matrigel-based hydrogel mimicking brain parenchyma. Findings revealed that CXCL12 (1600 ng/mL) released from NPs significantly increased the migration of F98 GBM cells after 72 hours under IFF conditions at both 0.5 and 3 μL/min. In contrast, U87 GBM cells required a higher CXCL12 concentration (2400 ng/mL) and longer incubation time for migration (120 hours). Unlike the F98 cells, U87 GBM cells showed a CXCL12 dose-dependent proliferation. Semi-quantitative qPCR showed higher CXCR4 mRNA levels in F98 cells than in U87 cells. CXCL12 significantly increased intracellular calcium levels via CXCR4 activation, with a 2.3-fold rise in F98 cells compared to U87, consistent with observed cell behavior during perfusion. This highlights the combined influence of IFF and CXCL12 on cell migration, dependent on cell line. This 3D dynamic model is a valuable tool to analyze parameters like interstitial fluid flow (IFF) and chemokine gradients, influenced by GBM tumor diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6bd1496f5c1fbe6eb347fa559aed0f8ce3defbb" target='_blank'>
              CXCL12 impact on glioblastoma cells behaviors under dynamic culture conditions: Insights for developing new therapeutic approaches
              </a>
            </td>
          <td>
            Wiam El Kheir, Sahar Naasri, Bernard Marcos, Nick Virgilio, Benoit Paquette, Nathalie Faucheux, Marc-Antoine Lauzon
          </td>
          <td>2024-12-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Introduction: Tumors systemically remodel the immune system during metastasis. Developing anti-metastatic immunotherapies that target these tumor-immune interactions to make the metastatic niche hostile colonization would represent a paradigm shift in cancer treatment. However, traditional high-throughput screening (HTS) platforms that use simple read-outs and contrived in vitro systems are ill-suited to identify such therapies. In contrast, HTS platforms that accurately model the metastatic niche and use high-dimensional read-outs such as multiplexed single-cell RNA-sequencing (scRNA-seq) can accurately profile nuanced perturbation responses in individual immune cell types and therefore have the potential to discover anti-metastatic immunotherapies. In this study, we describe the development of an ex vivo lung tissue culture HTS platform that preserves immune gene expression signatures observed in the in vivo metastatic niche and is amenable to single-cell chemical transcriptomic analysis using MULTI- seq (McGinnis et al., Nature Methods, 2019). Inspired by our recent description of myeloid cell TLR-NFκB inflammation during breast cancer lung metastasis (McGinnis et al., Cancer Cell, 2024), we then performed the first anti-metastatic immunotherapy drug screen and identified TLR-NFκB inhibitors that effectively operate on all desired myeloid cell types in the lung metastatic niche. Methods and Results: Building upon existing techniques for culturing precision-cut lung slices (Wu et al., Journal of Experimental Medicine, 2024) and patient- derived tumor fragments (Voabil et al., Nature Medicine, 2021), we established and optimized an ex vivo lung tissue culture system that enables 48-hour culturing of lung slices isolated from 4T1 tumor-bearing mice. scRNA-seq profiling revealed successful capture of all relevant immune cell types and retention of metastasis-associated gene expression programs (e.g., myeloid TLR-NFκB inflammation and neutrophil degranulation) following culture. We then scaled our platform to address how myeloid cells in the lung metastatic niche respond to perturbations targeting every component of the canonical TLR-NFκB signaling cascade. Cell-type-specific perturbation modeling and quantification of TLR-NFκB signaling inhibition identified compounds that optimally perturb all intended myeloid cell types (e.g., tissue-resident macrophages, neutrophils, and monocytes) and represent prioritized candidates for future in vivo validation studies to assess impact on metastatic disease progression. Conclusions: In this study, we describe the development and application of a novel HTS platform that couples ex vivo lung tissue cultures with single-cell chemical transcriptomics to identify anti-metastatic immunotherapy candidates. We demonstrate how our platform can identify TLR-NFκB signaling inhibitors that optimally operate in the lung metastatic niche, paving the way for future interrogation of additional metastasis-associated immune cell signaling pathways in different disease and tissue backgrounds.
 Citation Format: Chris McGinnis, Winnie Yao, Ansuman Satpathy. Anti-metastatic immunotherapy discovery using ex vivo lung tissue cultures and high-throughput single-cell chemical transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr PR017.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abfcae45218f099638f6a7c32802b5781110ec64" target='_blank'>
              Abstract PR017: Anti-metastatic immunotherapy discovery using ex vivo lung tissue cultures and high-throughput single-cell chemical transcriptomics
              </a>
            </td>
          <td>
            Chris McGinnis, Winnie Yao, Ansuman T. Satpathy
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Lung cancer is the leading cause of cancer mortality, and despite improvements in treatment, tumors typically respond incompletely and resume growth after acquisition of drug resistance. Recent completion of a neoadjuvant osimertinib Phase II trial treating patients with surgically resectable stage I-IIIA EGFR-mutated non-small cell lung cancer (EGFRm NSCLC) (NCT03433469) (Blakely et. Al, JCO 2024) has highlighted the importance of further identifying non-genomic mechanisms of persistence and resistance to targeted therapy. Here we analyzed these patient samples to identify transcriptionally regulated signaling patterns enriched at Residual Disease (RD) after neoadjuvant osimertinib treatment compared to treatment naïve samples (TN), as well as signaling patterns enriched at RD in patients who ultimately develop disease recurrence after surgical resection compared to those who remain disease-free. Spatially resolved transcriptomic sequencing was performed, using the 10X Genomics Visium platform, on 36 tissue sections (n=9 TN, n=19 RD, n=4 Progressive Disease (PD), n=4 Tumor Adjacent Normal), from 25 patients. TN and PD samples were from standard-of-care surgical resections. 18 of 19 RD samples were from NCT03433469. The Visium array spots that tile the sequencing capture area are 55 µm in diameter, and thus can contain an estimated 1-10 cells within each array spot. After quality control, 91,582 array spots remained for downstream analysis. We identified high confidence tumor array spots annotated both as “Cancer” by a board-certified pathologist and with tumor characteristic copy number variations. Through dimensionality reduction clustering, we identified 10 major array spot clusters, including the high confidence tumor array spot cluster. These clusters were further subset into 44 high resolution clusters of array spot niches, 32 of which were shared across at least nine patients. Because array spots are not single-cell resolution, these niches represent spots with similar mixed cell type composition and transcriptional signatures. We identified significant increases in the CCL14+ Endothelial Cell Niche, CXCL2+ Endothelial Cell Niche, Lymphocytes + Endothelial Niche, and CXCL12+ Fibroblast Niche in RD samples compared to TN. We identified a cell signaling program defined by spatially co-expressed ligand-receptor pairs associated with focal adhesion, enriched in PD samples. This same pattern was enriched in RD samples from patients who had recurrent disease after surgical resection compared to RD samples from patients who remained disease free. These data describe an increase in immune trafficking in RD samples, as well as identify an EGFR- alternative signaling pathway correlating with osimertinib resistance. Identification of spatially resolved, differential architectural organization and signaling patterns in RD and PD states will expand our understanding of targeted therapy resistance in EGFRm NSCLC, with the potential to identify additional clinically actionable targets.
 Citation Format: Whitney Tamaki, Daniel L. Kerr, Wei Wu, Grant Eilers, Anatoly Urisman, Yu-Ting Chou, Philippe Gui, Shigeki Nanjo, Johannes R. Kratz, David M. Jablons, Trever G. Bivona, Collin M. Blakely. Identification of cell-cell signaling programs across the tumor-tumor microenvironment ecosystem and associated with clinical status in neoadjuvant osimertinib treated patient samples revealed by spatial profiling [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00831e7253b0b6e70e43b596e9e7ff252fbadae" target='_blank'>
              Abstract C025: Identification of cell-cell signaling programs across the tumor-tumor microenvironment ecosystem and associated with clinical status in neoadjuvant osimertinib treated patient samples revealed by spatial profiling
              </a>
            </td>
          <td>
            W. Tamaki, D. Kerr, Wei Wu, Grant Eilers, A. Urisman, Yu-Ting Chou, Philippe Gui, Shigeki Nanjo, J. Kratz, D. Jablons, T. Bivona, Collin M Blakely
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="A noticeable characteristic of pancreatic ductal adenocarcinoma (PDAC) tumors is a dense tumor microenvironment with abundant and dense, desmoplastic stroma woven tightly with both cellular and matrix components. The high stromal density is associated with higher intratumor pressures which, until the last decade, was largely assumed to be tumor protective, confirmed by early studies demonstrating that altering the stroma was effective in genetically engineered models of PDAC. However, clinical trials using these approaches have been disappointing. There is increasing recognition that stroma heterogeneity is much greater than initially thought with an explosion of investigation into cancer-associated fibroblast (CAF) subpopulations led by experimental and single-cell transcriptomic studies. This review summarizes and attempts to harmonize the current transcriptomic data of CAF subpopulations. Understanding the heterogeneity of CAFs, the matrix, and other tumor microenvironment features will be critical to developing effective therapeutic approaches. Identifying model systems that best recapitulate the clinical behavior and treatment response of human PDAC will be important. Examining subpopulations as defined by clinical outcome will remain a critical step in defining clinically impactful CAF subtypes in larger clinical cohorts. The future of precision oncology in PDAC will depend on the integration of precision tumor epithelial and precision stroma approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a50e456e62579aeacb26fe2456d06aace9ecba04" target='_blank'>
              CAFomics: convergence to translation for precision stroma approaches.
              </a>
            </td>
          <td>
            Ian C McCabe, X. Peng, Joseph F Kearney, Jen Jen Yeh
          </td>
          <td>2024-11-08</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Somatic genomic rearrangements are hallmarks of cancer. Complex genomic rearrangements (CGRs) involving multiple intertwined structural alterations are often present in tumor genomes. CGRs frequently harbor oncogenic drivers, but their genomic architectures and etiologies are poorly understood. We used deep-coverage optical mapping technology to profile the genomic landscapes of normal lung tissues, benign pulmonary lesions, carcinoma in situ, and advanced carcinomas to examine the patterns of genome disorganization and instability in different stages of carcinogenesis in lung. Large rearrangements and CGRs were prevalent in the carcinomas. We developed omcplR to resolve the architecture of CGRs and predict their genesis from optical mapping data using genome-graph concept. We found that CGRs often arose from hierarchical combinations of multiple, localized simple structural variations, and harbored allelic heterogeneity at the affected loci. Rearrangement patterns and associated genomic signatures suggested that chromoanasynthesis was a likely prevalent mechanism driving complex genomic rearrangements. The early rearrangement junctions in intra-chromosomal CGRs were usually localized within the same chromatin domains, but the late junctions in advanced tumors had more heterogeneous contexts suggesting progressive organizational heterogeneity. The CGRs, especially the late events therein were under positive selection. A composite signature of genomic alterations including the CGRs captured the trajectory of progressive genomic disorganization and instability with carcinogenesis in lung and underscored the extent of genomic structural heterogeneity among the in-situ tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0b28163735146a5869b061b33efdd0283e9308" target='_blank'>
              Deconstructing evolutionary histories of complex genomic rearrangements in lung malignancies
              </a>
            </td>
          <td>
            Xiaoju Hu, Vignesh Venkat, Gregory R Riedlinger, Zhiyuan Shen, Jyoti Malhotra, Subhajyoti De
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Glioblastoma IDH wild type (GBM IDH wt) has a poor prognosis and a strongly associated with inflammatory processes. Inflammatory molecules generate positive feedback with tumor cells fueling tumor growth as well as recruitment of immune cells that promote aggressiveness. Although the role of many inflammatory molecules is well known, there are many macromolecules, such as the S100A proteins, whose role is only now beginning to be established. Methods: Using RNA-seq, bioinformatics tools and a cohort of glioma patients to validate the results, we have analysed the inflammatory processes involved in glioma. Transcriptional profiles were also used to define biological processes of relevance to specific S100A proteins. Finally, we characterized the relevant immune populations with an IHC analysis and transcriptional profiling. Results: We have noted an increased expression of S100A in GBM IDH wt compared to gliomas IDH mutants. This allowed us to analyse the involvement of different members of the family, such as S100A9, A11 and A13 as possible regulators of inflammatory processes in the GBM-IDH wt microenvironment. Thus, we observed that S100A9 is located in hypoxic areas linked to the function of neutrophils, S100A11 is found in vascular areas associated with the function of perivascular pericytes and macrophages, and finally, S100A13 which is related to the dysfunction of microglia. Conclusion: Our findings define different functions for S100A9, A11 and A13 proteins that are associated with the architecture of the glioblastoma microenvironment and define its progression. Moreover, these alterations can be reversed by the RAGE inhibitor, Azeliragon which is in a phase I/II clinical trial NCT05635734.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a6e8cb3b6c9ae0be7b8e990303379bc4a5f91c1" target='_blank'>
              S100A proteins show a spatial distribution of inflammation associated with the glioblastoma microenvironment architecture
              </a>
            </td>
          <td>
            Blanca Cómitre-Mariano, Berta Segura-Collar, Gabriel Vellila-Alonso, Rubén Contreras, Aurelio Henandez-Lain, Manuel Valiente, Juan M. Sepulveda, Stephen Garrett Marcus, Guillermo García-Posadas, Luis Jiménez-Roldán, Á. Pérez-Núñez, R. Gargini
          </td>
          <td>2025-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80cae72455d0f13cb986669bd3a06f266bb4aab" target='_blank'>
              Spatial transcriptomics reveals substantial heterogeneity in triple-negative breast cancer with potential clinical implications
              </a>
            </td>
          <td>
            Xiaoxiao Wang, D. Venet, F. Lifrange, D. Larsimont, Mattia Rediti, L. Stenbeck, F. Dupont, G. Rouas, Andrea Joaquin Garcia, L. Craciun, L. Buisseret, Michail Ignatiadis, M. Carausu, N. Bhalla, Yuvarani Masarapu, Eva Gracia Villacampa, L. Franzén, Sami Saarenpää, L. Kvastad, K. Thrane, J. Lundeberg, F. Rothé, Christos Sotiriou
          </td>
          <td>2024-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Abstract Background Perioperative chemotherapy is the standard of care for patients with locally advanced gastric and gastroesophageal junction cancer. Recent evidence demonstrated the addition of programmed cell death protein 1 (PD-1) inhibitors enhanced therapeutic efficacy. However, the mechanisms of response and resistance remain largely undefined. A detailed multiomic investigation is essential to elucidate these mechanisms. Methods We performed whole-exome sequencing, whole-transcriptome sequencing, multiplex immunofluorescence and single-cell RNA sequencing on matched pretreatment and post-treatment samples from 30 patients enrolled in an investigator-initiated Phase 2 clinical trial (NCT04908566). All patients received neoadjuvant PD-1 inhibitors in combination with chemotherapy. A major pathologic response (MPR) was defined as the presence of no more than 10% residual viable tumor cells following treatment. Results Before treatment, the positive ratio of CD3+T cells in both the tumor parenchyma and stroma was significantly higher in the non-MPR group compared with the MPR group (p=0.042 and p=0.013, respectively). Least absolute shrinkage and selection operator regression was employed for feature gene selection and 13 genes were ultimately used to construct a predictive model for identifying MPR after surgery. The model exhibited a perfect area under curve (AUC) of 1.000 (95% CI: 1.000 to 1.000, p<0.001). Post-treatment analysis revealed a significant increase in CD3+T cells, CD8+T cells and NK cells in the tumor stroma of MPR patients. In the tumor parenchyma, aside from a marked increase in CD8+T cells and NK cells, a notable reduction in macrophage was also observed (all p<0.05). Importantly, forkheadbox protein 3 (FOXP3), the principal marker for regulatory T cells (Treg) cells, showed a significant decrease during treatment in MPR patients. FOXP3 expression in the non-MPR group was significantly higher than in the MPR group (p=0.0056) after treatment. Furthermore, single-cell RNA sequencing analysis confirmed that nearly all Treg cells were derived from the non-MPR group. Conclusions Our study highlights the critical role of dynamic changes within the tumor immune microenvironment in predicting the efficacy of neoadjuvant combined immunochemotherapy. We examined the disparities between MPR/non-MPR groups, shedding light on potential mechanisms of immune response and suppression. In addition to bolstering cytotoxic immune responses, specifically targeting Treg cells may be crucial for enhancing treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd7e41ce76cedb5c64fc740c519b9826d4d1968" target='_blank'>
              Multiomics reveals tumor microenvironment remodeling in locally advanced gastric and gastroesophageal junction cancer following neoadjuvant immunotherapy and chemotherapy
              </a>
            </td>
          <td>
            Z. Ji, Xia Wang, Jiaqi Xin, Lijun Ma, Duo Zuo, Hong-li Li, Lan Su, Xinze Lv, S. Ge, Le Zhang, Yong Liu, Yanhui Zhang, Tingting Ding, Ting Deng, Y. Ba, Rui Liu
          </td>
          <td>2024-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Precision medicine for paediatric and adult cancers that includes drug sensitivity profiling, can identify effective therapies for individual patients. However, obtaining adequate biopsy samples for high-throughput (HTP) screening remains challenging, with tumours needing to be expanded in culture or patient-derived xenografts – this is time-consuming and often unsuccessful. Herein, we have developed paediatric patient-derived tumour models using an engineered extracellular matrix (ECM) tissue mimic hydrogel system and HTP 3D bioprinting. Gene expression analysis from neuroblastoma and sarcoma patients identified key components of the ECM in these tumour types. Engineered hydrogels with ECM-mimic peptides were used to create patient-specific tumour organoids, modelling tumour growth conditions. Expanded tumour organoids recapitulated the genetic and phenotypic characteristics of the original tumours and retained tumourgenicity. Screening of these models identified individualised drug sensitivities. Our approach offers a timely and clinically relevant technology platform for precision medicine in paediatric cancers, potentially transforming preclinical testing across cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea0198b891b0eff96e2184508189a30fce8af925" target='_blank'>
              Engineered paediatric tumours retain maintains tumour genotype and phenotype for precision medicine
              </a>
            </td>
          <td>
            MoonSun Jung, Valentina Poltavets, Joanna Skhinas, G. Tax, Alvin Kamili, Angela (Jinhan) Xie, Sarah Ghamrawi, Philipp Graber, Jie Mao, Marie Wong-Erasmus, Louise Cui, Kathleen Kimpton, Pooja Venkat, Chelsea Mayoh, Emmy D G Fleuren, Ashleigh M. Fordham, Zara A Barger, John Grady, David M. Thomas, Eric Y Du, Mark J. Cowley, Andrew J. Gifford, Jamie I. Fletcher, Loretta M S Lau, M. E. M. Dolman, J. J. Gooding, Maria Kavallaris
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Introduction The extracellular matrix (ECM) supports tumor progression by influencing tumor cell migration and invasion. This study examines the link between peritumoral ECM morphology and five-year recurrence risk in TNM Stage II colon cancer, using quantitative whole-slide ECM imaging. We hypothesize that loose ECM regions are associated with increased recurrence risk due to enhanced tumor budding (TB) or poorly differentiated clusters (PDC). Methods In a case-control study of 100 TNM Stage II colon cancer patients (25 with recurrence and 75 controls matched by lymph node sampling and tumor extent), Picrosirius red-stained sections were imaged to quantify ten ECM parameters across 798 regions of interest (ROIs). Conditional logistic regression assessed associations between ECM morphologies, TB, PDCs, and recurrence. Results Unsupervised clustering identified three ECM morphologies: dense fibrous, loose sparse, and complex tortuous. Dense fibrous ECM correlated strongly with recurrence (aOR 9.43, 95% CI 3.29-29.30, p < 0.001), while loose sparse and complex tortuous ECMs were associated with reduced recurrence risk (aOR 0.33, 95% CI 0.11-0.91, p = 0.040, and aOR 0.14, 95% CI 0.02-0.53, p = 0.012, respectively). TB was highest in loose sparse ECM (mean 9.3), and PDCs were highest in dense fibrous ECM (mean 5.5). Discussion Our findings suggest that ECM morphology, particularly dense fibrous ECM, predicts recurrence in Stage II colon cancer, highlighting ECM profiling as a promising tool for patient stratification beyond traditional staging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10cd804b89bbca0cdd6f6ca0c5c57995b707a846" target='_blank'>
              Whole-Slide ECM Imaging Reveals Dense Fibrous Matrix as a High-Risk Factor for Recurrence in Stage II Colon Cancer
              </a>
            </td>
          <td>
            C. Ravensbergen, V.S. Colaco, H. Putter, A. Crobach, J. Boonstra, J.H.N. Lindeman, R. Tollenaar, W. Mesker
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 Introduction: Properties of tumor angiogenesis impact on tumor growth and composition of the tumor microenvironment. Tumor vessels are composed of pericytes and other mural cells. Detailed characterization of the cellular composition of the microvasculature can suggest functional vessel and case properties relevant for outcome, response to treatment and immune surveillance. Results: Based on single cell RNA-seq from three human stage II/III colon cancers and pilot in-situ multi-antibody staining of six tumors two main clusters of perivascular cells were identified; B1 (MCAM+, MYH11-) and B2 (MCAM+, MYH11+) cells. B1 cells displayed pericyte features including expression of RGS5, whereas B2 cells showed high expression of smooth muscle cell markers like ACTG2 and Desmin. Desmin-high B2 cells were predominantly located in muscular layers, whereas Desmin-low/MYH11-high B2 cells showed perivascular enrichment. An extended novel analyses workflow of digital image analysis was used to profile peri-endothelial composition in 19 intervals; 11 1-µm intervals from 0 to 10 µm and 8 5-µm intervals from 10 to 50 µm from endothelial areas. Each of these expansion areas was quantitated regarding density of PDGFRB+/- cells including B1/B2 cells, and a MYH11+/MCAM- cells. Density of macrophage subsets (CD68-positive subsets defined by CD163 and CD11c) were also quantitated. In general, B1 cells showed stronger spatial association with endothelial cells than B2 cells, and this spatial association was stronger for the PDGFRB+ B1 subset. Survival analysis indicated that prognostic relevance was strongest when PDGFRB+ (favorable prognosis) cells were closest to endothelial cells. Regarding perivascular macrophage density, M0 and transit M1/M2 macrophages were predominantly enriched in the peri-endothelial regions, whereas density of M1 and M2 increased gradually as distance from vessels increased. M2 density was overall associated with poor prognosis, whereas high density of other macrophage subsets overall was associated with good prognosis. Ongoing studies are using a clustering-based approach to identify vessel subsets, and prognosis associations are being consolidated in additional cohorts. Summary: This study provides a novel approach for high resolution profiling of perivascular status in CRC, and possibly other tumor types, that could be of specific importance to identify tumor features associated with response to angiogenesis-directed therapies.
 Citation Format: Linglong( 凌 龙 ) Huang( 黄 ), Mercedes Herrera, Jonas Sjölund, Vladimir Chocoloff, Simon Joost, Rasul M. Tabiev, Lina Wik Leiss, Carina Strell, Luis Nunes, Artur Mezheyeuski, David Edler, Anna Martling, Fredrik Pontén, Bengt Glimelius, Tobias Sjöblom, Maria Kasper, Kristian Pietras, Arne Östman. Mapping the vascular walls of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a41b99632bcf5e6a6bbd94c1d54966c1ecf742fe" target='_blank'>
              Abstract B029: Mapping the vascular walls of colorectal cancer
              </a>
            </td>
          <td>
            Linglong(凌龙) Huang( 黄 ), Mercedes Herrera, Jonas Sjölund, Vladimir Chocoloff, S. Joost, Rasul M. Tabiev, Lina Wik Leiss, Carina Strell, Luis Nunes, A. Mezheyeuski, D. Edler, A. Martling, Fredrik Pontén, B. Glimelius, T. Sjöblom, Maria Kasper, Kristian Pietras, Arne Östman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eeb1135b7c75ad3978d67f0a6864e002ffcff1b" target='_blank'>
              Comprehensive single-cell aging atlas of healthy mammary tissues reveals shared epigenomic and transcriptomic signatures of aging and cancer.
              </a>
            </td>
          <td>
            B. Angarola, Siddhartha Sharma, Neerja Katiyar, Hyeon Gu Kang, Djamel Nehar-Belaid, SungHee Park, Rachel Gott, Giray N Eryilmaz, Mark A. LaBarge, Karolina Palucka, Jeffrey H. Chuang, Ron Korstanje, D. Ucar, Olga Anczukόw
          </td>
          <td>2024-11-25</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Uterine dendritic cells (uDCs) are critical for endometrial function, yet their origin, molecular characteristics, and specific roles during the pre- and post-implantation periods in the human endometrium remain largely unknown. The complexity of the endometrial environment makes defining the contributions of uDCs subtypes challenging. We hypothesize that distinct uDC subsets carry out specialized functions, and that resident progenitor DCs generate these subtypes. Employing single-cell RNA sequencing on uterine tissues collected across different menstrual phases and during early pregnancy, we identify several uDCs subtypes, including resident progenitor DCs. CITE-seq was performed on endometrial single-cell suspensions to link surface protein expression with key genes identified by the RNAseq analysis. Our analysis revealed the developmental trajectory of the uDCs along with the distinct functional roles of each uDC subtype, including immune regulation, antigen presentation, and creating a conducive environment for embryo implantation. This study provides a comprehensive characterization of uDCs, serving as a foundational reference for future studies for better understanding female reproductive disorders such as infertility and pregnancy complications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db155caad60945980a15a5d8664e0fa0d705609" target='_blank'>
              Decoding Functional and Developmental Trajectories of Tissue-Resident Uterine Dendritic Cells Through Integrative Omics
              </a>
            </td>
          <td>
            Gil Mor, Aditi Singh, Jing Yang, Nicholas K Adzibolosu, Songchen Cai, Elana Kauf, Lingtao Yang, Qiyuan Li, Hanjie Li, Alexandra Werner, Siddharth Parthasarathy, Jiahui Ding, Jared M. Fortier, Marta Rodriguez- Garcia, L. Diao
          </td>
          <td>2024-11-14</td>
          <td>Research Square</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Despite recent breakthroughs in spatial biology, either imaging-based or sequencing-based spatial transcriptomics is largely equivalent to spatial messenger RNA (mRNA) expression quantification. An RNA molecule experiences a complex life cycle involving transcription, splicing, maturation, translation, and degradation . It is highly desirable to profile all these RNA species spanning their life cycle to explore the biology of RNA at genome scale and cellular level. Furthermore, formalin-fixed paraffin-embedded (FFPE) tissues are essential in clinical practice, being the backbone of human disease histopathological diagnoses . Pathology departments have accrued vast collections of FFPE blocks over time, creating a rich, yet underutilized compendium of materials that, accompanied by clinical data, stands as a treasure trove for human biology and translational research . In this evolving landscape, we present “Pathology-compatible Deterministic Barcoding in Tissue” (Patho-DBiT) not only enabling spatial full-coverage base-by-base whole transcriptome sequencing but also crafted to address the challenges of clinically archived FFPE tissues. Patho-DBiT integrates in situ polyadenylation, microfluidic in tissue barcoding, and computational innovations to decode rich RNA biology inherent in FFPE samples. The platform capitalizes on RNA fragmentation naturally occurring in FFPE specimens and appends poly(A) tails to a broad spectrum of RNA species, thereby overcoming traditional barriers associated with FFPE samples and even outperforming the assays conducted with frozen tissues. Patho-DBiT permits spatial co-profiling of gene expression and alternative splicing, unveiling region-specific isoforms in the mouse brain. High-sensitivity transcriptomics is constructed from 5-year archived T-cell lymphoma tissues, with cross-validation conducted using super-resolution spatial phenotyping technology (CODEX). Furthermore, genome-scale single nucleotide RNA variants (SNVs) are captured to distinguish malignant subclones from non-malignant cells in human B-cell lymphomas, dissecting spatial clonal architectures based on both SNVs and copy number variation (CNV) profiles. Patho-DBiT also enables spatially resolved co-profiling of large and small RNAs, facilitating the analysis of a microRNA-mRNA regulatory network and tRNA utilization within clinical biopsies and elucidating their roles in tumorigenesis. With superior intronic read capture efficiency, Patho-DBiT spatially mapped RNA splicing dynamics associated with the developmental trajectory of tumor B cells. High resolution Patho-DBiT with a 10-μm spot size reveals the heterogeneities of human lymphomas within a spatial neighborhood and traces the spatiotemporal molecular kinetics driving tumorprogression at the cellular level. Patho-DBiT represents a first-of-its-kind technology, enabling the spatial exploration of rich RNA biology in FFPE tissues to aid in clinical pathology research.
 Citation Format: Zhiliang Bai, Dingyao Zhang, Yan Gao, Bo Tao, Mingyao Li, Yi Xing, Jun Lu, Mina Xu, Rong Fan. Patho-DBiT: Spatially exploring RNA biology in archival formalin-fixed paraffin- embedded tissues [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr A022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e489d2580983206cf2150a52be6f3f7c8e540a5e" target='_blank'>
              Abstract A022: Patho-DBiT: Spatially exploring RNA biology in archival formalin-fixed paraffin- embedded tissues
              </a>
            </td>
          <td>
            Zhiliang Bai, Dingyao Zhang, Yan Gao, Bo Tao, Mingyao Li, Yi Xing, Jun Lu, Mina L. Xu, Rong Fan
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Despite promising results with immune checkpoint blockade (ICB) therapy, brain metastases (BrM) remain a deadly complication for cancer patients. Understanding the highly specialized brain metastatic tumor microenvironment (BrTME) is crucial for identifying determinants of response, however, progress has been hindered by limited access to patient samples and scarcity of relevant preclinical models. Here, we developed two mouse melanoma BrM models by intracardiac injection of cells derived from our previously established M4-B2905 melanoma mouse cell line. We characterized their mutational landscape and performed single-cell phenotypic and transcriptomic analysis, demonstrating that these models recapitulate the cellular and molecular features of human melanoma BrMs. Comparative and interactome analysis of our models revealed key factors contributing to ICB response and resistance. We found that the responsive model (BR1) was characterized by tumor cells that polarize microglia toward reactive states via inflammatory programs and response to IFN signaling. These microglia express high levels of antigen presentation and immunostimulatory molecules, triggering T cell recruitment and activation and promoting ICB therapy response. In contrast, in the resistant model (BR3), the tumor cells express neurological molecular signatures and ligands that sustain microglia homeostasis. This results in poor T cell infiltration predominantly composed of naïve cells, evading immune reaction and promoting ICB resistance. While we showed that systemic ICB therapy induced BrTME changes in both models, differences between responder and resistant models were maintained, emphasizing the importance of tumor cell-intrinsic programs in dictating the BrTME. We validated the translational relevance of our findings and demonstrated that BR1 signatures positively correlate with T cell infiltration and are associated with improved patient outcomes, whereas BR3 signatures negatively correlate with T cells and are found in patients with worse prognosis. Here we address an important challenge in the field by providing clinically relevant BrM models that recapitulate both the cellular and molecular features of human disease and the variability of ICB response seen in patients. We further reveal mechanistic insights into BrM ICB response and identify potential therapeutic targets to modulate the BrTME.
 Citation Format: Amelie Daugherty-Lopes, Eva Perez-Guijarro, Vishaka Gopalan, Jessica Rappaport, Quanyi Chen, April Huang, Khiem C. Lam, Sung Chin, Jessica Ebersole, Emily Wu, Gabriel Needle, Isabella Church, George Kyriakopoulos, Shaojun Xie, Yongmei Zaho, Charli Gruen, Antonella Sassano, Romina E Araya, Andres Thorkelsson, Cari Smith, Maxwell P. Lee, Sridhar Hannenhalli, Chi-Ping Day, Glenn Merlino, Romina S. Goldszmid, Shaojun Xie. Dissecting the brain metastatic microenvironment to uncover immune and molecular correlates of response to immunotherapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83bdd8b5e937692077f180146aa529c30245669c" target='_blank'>
              Abstract A002: Dissecting the brain metastatic microenvironment to uncover immune and molecular correlates of response to immunotherapy
              </a>
            </td>
          <td>
            Amélie Daugherty-Lopès, Eva Pérez-Guijarro, Vishaka Gopalan, Jessica Rappaport, Quanyi Chen, April Huang, Khiem C. Lam, Sung Chin, Jessica Ebersole, Emily Wu, Gabe Needle, Isabella Church, George Kyriakopoulos, Shaojun Xie, Yongmei Zaho, Charli Gruen, Antonella Sassano, Romina E. Araya, Andres Thorkelsson, Cari Smith, Maxwell P. Lee, Sridhar Hannenhalli, Chi-Ping Day, Glenn Merlino, R. Goldszmid, Shaojun Xie
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Recurrence and metastasis are the major issues for papillary thyroid cancer (PTC). Current morphological and molecular classification systems are not satisfied for PTC diagnosis due to lacking variant-specific morphological criteria and high signal-to-noise in mutation-based diagnosis, respectively. Importantly, intratumor heterogeneity is largely lost in current molecular classification system, which can be resolved by single cell RNA sequencing (scRNA-seq). However, scRNA-seq loses spatial information and morphological features. Herein, scRNA-seq is integrated and spatially-resolved transcriptomics (SRT) to elaborate the mechanisms underlying the spatial heterogeneity, malignancy and metastasis of PTCs by associating transcriptome and local morphology. This results demonstrated that PTC cells evolved with multiple routes, driven by the enhanced aerobic metabolism and the suppressed mRNA translation and protein synthesis and the involvement of cell-cell interaction. Two curated malignant and metastatic footprints can discriminate PTC cells from normal thyrocytes. Ferroptosis resistance contributed to PTC evolution. This results will advance the knowledge of intratumor spatial heterogeneity and evolution of PTCs at spatial and single-cell levels, and propose better diagnostic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/870c25c5831bd8c1742183d121d62d6970a89470" target='_blank'>
              Spatial and Single-Cell Transcriptomics Unraveled Spatial Evolution of Papillary Thyroid Cancer.
              </a>
            </td>
          <td>
            Guangzhe Zheng, Shaobo Chen, Wanqi Ma, Quanshu Wang, Li Sun, Changwen Zhang, Ge Chen, Shuping Zhang, Shuguang Chen
          </td>
          <td>2024-11-14</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Metastatic cancer affects millions of people worldwide annually and is the leading cause of cancer-related deaths. Most patients with metastatic disease are not eligible for surgical resection, and current therapeutic regimens have varying success rates, some with 5-year survival rates below 5%. Here, we test the hypothesis that metastatic cancer can be genetically targeted by exploiting single base substitution mutations unique to individual cells that occur as part of normal aging prior to transformation. These mutations are targetable because ~10% of them form novel tumor-specific “NGG” protospacer adjacent motif (PAM) sites targetable by CRISPR-Cas9. Methods Whole genome sequencing was performed on five rapid autopsy cases of patient-matched primary tumor, normal and metastatic tissue from pancreatic ductal adenocarcinoma decedents. CRISPR-Cas9 PAM targets were determined by bioinformatic tumor-normal subtraction for each patient and verified in metastatic samples by high-depth capture-based sequencing. Results We found that 90% of PAM targets were maintained between primary carcinomas and metastases overall. We identified rules that predict PAM loss or retention, where PAMs located in heterozygous regions in the primary tumor can be lost in metastases (private LOH), but PAMs occurring in regions of loss of heterozygosity (LOH) in the primary tumor were universally conserved in metastases. Conclusions Regions of truncal LOH are strongly retained in the presence of genetic instability and, therefore, represent genetic vulnerabilities in pancreatic adenocarcinomas. A CRISPR-based gene therapy approach targeting these regions may be a novel way to genetically target metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada3070ea15ff6901b235b693417e61df449621f" target='_blank'>
              Islands of genomic stability in the face of genetically unstable metastatic cancer
              </a>
            </td>
          <td>
            Kirsten Bowland, Jiaying Lai, Alyza M. Skaist, Yan Zhang, S. Teh, Nicholas J. Roberts, Elizabeth Thompson, Sarah Wheelan, Ralph H. Hruban, Rachel Karchin, Matthew H Bailey, Christine A. Iacobuzio-Donahue, J. Eshleman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="BACKGROUND
Prostate cancer's complex interplay with the immune microenvironment prompted an investigation into immune-related pathogenic mechanisms and potential therapeutic targets.


METHODS
Within the GSE176031 data set, Seurat meticulously dissected single-cell profiles from radical prostatectomy patients. Leveraging CellMarker and SingleR cell identities were precisely annotated. Then, monocle traced pseudotime trajectories, illuminating cellular paths, complemented by CellChat's insights into intricate intercellular communications. Furthermore, mendelian randomization (MR) robustly substantiated causal associations within prostate cancer contexts.


RESULTS
Employing single-cell analysis on intraoperative tumor and normal tissue, we identified 15 distinct cell types, notably observing a significant T cell reduction in tumor samples. Intercellular communication analysis revealed multiple pathways between epithelial cells and T cells, highlighting interleukin (IL)-IL7R-IL2RG interactions. IL7R, crucial in T cell apoptosis, showed differential expression across T cell development stages. Patients with IL7 amplification had poorer outcomes (p < 0.05), supported by MR in two cohorts (ieu-b-4809 cohort: odds ratio [OR] = 1.005, p = 0.002, 95% confidence interval [CI] [1.002-1.008]; ebi-a-GCST90018905: OR = 1.063, p = 0.032, 95% CI [1.005-1.125]), confirming IL7 as a prostate cancer risk factor.


CONCLUSIONS
These findings suggest T cell depletion via IL7-IL7R signaling may drive prostate cancer progression, offering novel therapeutic insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bfc520d1668fd03c8682647e804ec854987049b" target='_blank'>
              IL7 as a Risk Factor for Prostate Cancer: Implications for T Cell Apoptosis and Infiltration in the Tumor Microenvironment.
              </a>
            </td>
          <td>
            Enyang He, Yaowen Li, Rui Zhao, Qinyan Kong, Yi Shao, Cong Wang, Baoqun Liu, Yvhang Jiang, Qian Liu, Hualei Cui
          </td>
          <td>2024-11-26</td>
          <td>The Prostate</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Introduction:
 Lung pericytes (PCs) are mural cells in close contact with endothelial cells (ECs) in the microvasculature. A significant challenge in investigating PC biology is largely due to the absence of a unique cell marker, making it difficult to distinguish them from other mural cell populations. The identification of such a marker would allow investigators to describe the role of PCs in various diseases including pulmonary arterial hypertension.


 Hypothesis:
 We hypothesize that HIG1 hypoxia-inducible domain family member 1B (
 Higd1b
 ) is exclusively expressed in lung PCs, which contributes to hypoxia (Hx)-induced vascular remodeling.


 Methods:
 We utilized single-cell RNA sequence (scRNA-seq) databases, spatial transcriptomics, and RNAscope from human and murine lungs to identify a PC-specific cell marker and compare the gene expression between PC subtypes. We utilized Cre-LoxP and CRISPR technology to construct a novel tamoxifen-inducible
 Higd1b-CreERT2
 knock-in mouse model and performed lineage-tracing studies to describe the role of PC subtypes in Hx-induced pulmonary hypertension (PH).


 Results:
 ScRNA-seq analysis from the lungs of humans and mice identified
 Higd1b
 as a specific gene marker for PCs whose expression is absent in other mural cell populations. Validation with a reporter mouse line
 Higd1b-CreERT2::R26-tdT
 confirmed Higd1b-Cre+ cells specifically label PCs and no other mural cells (
 Fig 1A
 ). Lineage tracing in the Hx-induced murine model of PH demonstrated the accumulation of PCs in the muscularized distal arterioles (
 Fig 1B
 ). Through scRNA-seq and immunofluorescence validation, we identified two
 HIGD1B
 + PC subtypes that exist in the pulmonary capillary: Type 1 PCs, which are quiescent in the capillaries, and Type 2 PCs exhibit multipotent cell-like properties and accumulate in the arterioles after exposure to Hx. Furthermore, we found Type 2 PCs transited into SMC-like cells via the upregulation of Vimentin, contributing to vascular remodeling in Hx-induced PH.


 Conclusion:
 We identified
 Higd1b
 as a unique marker for PCs and generated a novel
 Higd1b-CreERT2
 mouse that specifically labels PCs in the lungs. The discovery of PC-subtype specialization advances our understanding of lung pericyte biology and PC’s contribution to capillary remodeling under pathological conditions.



">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2b85b3938ef69167bee8a1177fee15de401d03c" target='_blank'>
              Abstract 4116949: Specialized Pericyte Subtypes in the Pulmonary Capillary
              </a>
            </td>
          <td>
            T. Klouda, Yunhye Kim, Han-Seong Baek, Mantu Bhaumik, Yu Liu, Tiffany Liu, Jianwen Que, Joseph Wu, Benjamin A Raby, Vinicio A Dejesus, Ke Yuan
          </td>
          <td>2024-11-12</td>
          <td>Circulation</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/150e626daa762108097ae88ebcf26c0674670dd5" target='_blank'>
              Isolation and identification of patient-derived liver cancer stem cells and development of personalized treatment strategies
              </a>
            </td>
          <td>
            Tingting Guo, Shuai Zhang, Weiping Zeng, Yan Liang, Jinghe Xie, Shoupei Liu, Yaqi Qiu, Ying Fu, Y. Ou, Keqiang Ma, Bailin Wang, Weili Gu, Yuyou Duan
          </td>
          <td>2024-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Background: Prostate cancer is a prevalent malignancy in men, with treatment resistance posing a significant challenge to successful cure. Acquired Resistance to Therapy (ART) arises from two main components: (1) Tumor Microenvironment (TME) and (2) mutations or epigenetic changes at gene level within the tumor cells themselves. While most cancer therapies target the intrinsic biology of cancer cells, growing evidence indicates that these treatments can also harm the stromal cells within TME, potentially driving ART. The biology of ART may vary based on the types of treatments, cell types, and organ locations. Thus, this study aims to dissect ART components driven by TME-mediated cellular processes in a comprehensive manner. Hypothesis: We propose that cancer therapies induce stress responses in non-cancerous cells within the TME. These stress responses involve the secretion of cytokines, mediators, and growth factors, which we term Stress Response Secretory Programs (SRSPs). We further hypothesize that targeting SRSPs could overcome ART that helps to improve treatment specificity and efficacy. Materials and Methods: We employed comprehensive single-cell profiling of common metastatic sites included liver, prostate, lung, bone, lymph node and spleen in preclinical models to identify secreted proteins and dysregulated pathways caused by genotoxic therapies. In vitro, we evaluated treatment resistance in murine prostate epithelial cell lines (Myc-CAP) by exposing them to conditioned media (CM) from irradiated murine fibroblasts (F4M2), using cell growth assays. Results: Single-cell RNA sequencing of organ samples revealed significant expressions of SRSPs and increased inflammatory responses in liver and prostate fibroblasts as well as other stromal cells treated with carboplatin. In vitro, CM from irradiated F4M2 cells enhanced cell proliferation and survival of Myc-CAP cells in response to docetaxel treatment. Conclusion: Our findings highlight the crucial role of SRSPs in mediating ART through interactions within the TME, which alter tumor cell phenotypes. Targeting these SRSPs holds potential for overcoming treatment resistance and improving therapeutic outcomes, underscoring the need to consider TME complexity in cancer treatment strategies.
 Citation Format: Tony Lok Heng Chu, Armand Bankhead, Ilsa Coleman, Sander Frank, Peter S. Nelson, Tarana Arman. Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c940709273c954da8cb4bd8042d98f46649ce8ee" target='_blank'>
              Abstract C033: Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment
              </a>
            </td>
          <td>
            Tony Lok Heng Chu, Armand Bankhead, Ilsa M Coleman, Sander Frank, Peter S. Nelson, Tarana Arman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/346f0b34e37ff4a92b4ba0dc4976ab70716a2ff8" target='_blank'>
              Single-cell multiomics reveals simvastatin inhibits pan-cancer epithelial-mesenchymal transition via the MEK/ERK pathway in XBP1+ mast cells
              </a>
            </td>
          <td>
            Sen Lin, Huimin Zhang, Ruiqi Zhao, Zhulin Wu, Weiqing Zhang, Mengjiao Yu, Bei Zhang, Lanyue Ma, Danfei Li, Lisheng Peng, Weijun Luo
          </td>
          <td>2024-11-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the past century, colorectal cancer (CRC) has become one of the most devastating cancers impacting the human population. To gain a deeper understanding of the molecular mechanisms driving this solid tumor, researchers have increasingly turned their attention to the tumor microenvironment (TME). Spatial transcriptomics and proteomics have emerged as a particularly powerful technology for deciphering the complexity of CRC tumors, given that the TME and its spatial organization are critical determinants of disease progression and treatment response. Spatial transcriptomics enables high-resolution mapping of the whole transcriptome. While spatial proteomics maps protein expression and function across tissue sections. Together, they provide a detailed view of the molecular landscape and cellular interactions within the TME. In this review, we delve into recent advances in spatial biology technologies applied to CRC research, highlighting both the methodologies and the challenges associated with their use, such as the substantial tissue heterogeneity characteristic of CRC. We also discuss the limitations of current approaches and the need for novel computational tools to manage and interpret these complex datasets. To conclude, we emphasize the importance of further developing and integrating spatial transcriptomics into CRC precision medicine strategies to enhance therapeutic targeting and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b317770be242dc222a3004766249a5b9de43dcc" target='_blank'>
              Current state and future prospects of spatial biology in colorectal cancer
              </a>
            </td>
          <td>
            Francisco G. Carranza, Fernando C. Diaz, Maria Ninova, Enrique Velazquez-Villarreal
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Melanoma, a highly aggressive and life-threatening cancer, poses a significant therapeutic challenge, particularly for patients with distant metastasis, who face a 5-year survival rate of only approximately 22.5%. This underscores the urgent need for innovative therapeutic approaches. Emerging evidence suggests that sensory and parasympathetic innervation within the tumor microenvironment (TME) plays a crucial role in driving metastasis and immune evasion, partly through the promotion of M2 macrophages and cancer stem cells. To uncover the mechanisms behind tumor-specific innervation in metastatic melanoma, we employed a systems biology approach integrating bulk RNAseq from diverse melanoma cell lines, TCGA-SKCM data analysis, single-cell RNAseq, and spatial transcriptomics. This comprehensive analysis led us to identify a previously uncharacterized gene regulatory network (GRN) orchestrating tumor innervation. This GRN is driven by axon guidance cues (Netrin, Slit, Semaphorin), their receptors (DCC, Robo, Neuropilin, ITGB1), and transcription factors (RAS, RND1, TFAP2, CAMKII). We further discovered that the pioneer factor FOXC2, known for its association with metastasis, plays a critical role in opening the promoters of innervation-specific genes. Additionally, we identified Ephrin and CXCR4-mediated crosstalk between neurons and tumor cells. Machine learning models validated the accuracy of this GRN in predicting melanoma metastasis, and a neural network-based model confirmed the enrichment of nerve cells near metastatic tumor cells within the TME in TCGA immunohistochemistry data. The functional relevance of this network was further supported by CRISPR-mediated knockout studies, which disrupted tumor-specific signaling pathways. Notably, we also observed the enrichment of genes linked to mRNA processing bodies and neuron-specific membrane-less condensates in metastatic melanoma. These findings highlight the critical role of neuronal infiltration in melanoma metastasis. Currently, therapeutic strategies targeting this aspect of the TME remain limited. Our identified gene set offers potential as a metastatic predictor, and targeting the neural component of the TME represents a promising new therapeutic avenue for melanoma.
 Citation Format: Anjana Goli, Subhajit Dutta. Deciphering the neurogenic tumor microenvironment: A novel gene network driving innervation in metastatic melanoma offers prognostic and therapeutic insights [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a5d79cdb084a3fb556d9c17280ebbf7e5cef77" target='_blank'>
              Abstract B024: Deciphering the neurogenic tumor microenvironment: A novel gene network driving innervation in metastatic melanoma offers prognostic and therapeutic insights
              </a>
            </td>
          <td>
            Anjana Goli, Subhajit Dutta
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b9a78f9c795229afb402fbd56b363bc8e608d7" target='_blank'>
              Normal tissue transcriptional signatures for tumor-type-agnostic phenotype prediction
              </a>
            </td>
          <td>
            Corey Weistuch, Kevin A. Murgas, Jiening Zhu, Larry Norton, Ken A. Dill, Allen R. Tannenbaum, Joseph O. Deasy
          </td>
          <td>2024-11-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Circulating Tumor Cells (CTCs) hold the promise of comprehensive yet noninvasive molecular characterization of cancer cells during the course of tumor evolution and progression. To dissect the epigenetic profile of prostate CTCs, we undertook paired DNA methylation and RNA sequencing of single CTCs isolated from blood specimens of patients with prostate cancer. Across the genome, our study identified 40 core Partial Methylation Domains (PMDs) that arise early in tumorigenesis and are shared by all individual CTCs across multiple patients. Key among these is a single genomic locus harboring multiple interferon-inducible genes along with all the CD1 gene family members, implicated in innate immunity. Based on mouse models, we propose that early biallelic silencing of multiple genes residing within a single PMD may contribute to the escape from immune surveillance in prostate and potentially other cancers. Studies of CTC heterogeneity within individual patients are limited by the very small number of cells that can be isolated from standard 10ml blood volumes. To enable such analyses, we applied a new high throughput microfluidic platform, capable of processing an entire leukapheresis product, interrogating 2-3 L of blood volume equivalent. Applying this technology to patients with metastatic prostate cancer, we successfully enriched thousands of CTCs from individual cases, enabling the characterization of intra-patient heterogeneity at the level of cell morphology and marker expression, chromosome copy number variation and exome-wide genotyping, and transcriptionally-defined tumor cell subpopulations. Such high-volume microfluidic enrichment of CTCs constitutes a new dimension in liquid biopsies.
 Citation Format: Daniel A Haber, Shyamala Maheswaran, Hongshan Guo, Joanna Vuille, Shih-Bo Huang, Ben S Wittner, Lecia V Sequist, Richard J Lee, Patricia AR Brunker, David T Miyamoto, Avanish Mishra, Mehmet Toner. Tumor cell-based liquid biopsy to characterize prostate cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr IA005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe44dae01b6a504aeda595e4d36a826d45670615" target='_blank'>
              Abstract IA005: Tumor cell-based liquid biopsy to characterize prostate cancer
              </a>
            </td>
          <td>
            D. A. Haber, S. Maheswaran, Hongshan Guo, Joanna A. Vuille, Shih-Bo Huang, Ben S. Wittner, L. Sequist, Richard J. Lee, Patricia A. R. Brunker, D. Miyamoto, Avanish Mishra, Mehmet Toner
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="Previous studies revealed that tumor-associated macrophages/microglia (TAMs) promoted glioma invasiveness during tumor progression and after radiotherapy. However, the communication of TAMs with tumor cells remains unclear. This study aimed to examine the role of small extracellular vesicles (sEVs) derived from TAMs in TAMs-mediated brain tumor invasion. This study utilized BV2 and RAW264.7 cell lines representing resident and infiltrating macrophages, respectively, to unveil their effect on tumor cells. Purified sEVs from BV2 and RAW264.7 were validated by nanoparticle track analysis (NTA), transmission electron microscopy (TEM), and western blotting for sEV markers. The effect of sEVs on the murine astrocytoma tumor cell line ALTS1C1 was examined on cell proliferation, migration, and gene expression. The results showed that ALTS1C1 cells effectively engulfed sEVs purified from BV2 and RAW264.7. Only BV2-derived sEVs promoted cell proliferation and were dose-dependent. Further, morphological changes in ALTS1C1 cells were observed after incubation with BV2-derived sEVs, which was associated with enhancing cell migration. BV2-mediated glioma proliferation and mobility were related to the upregulation of vascular endothelial growth factor (VEGF) and downregulation of death effector domain-containing protein (DEDD) gene expression. This study demonstrates the distinct function of sEVs of resident macrophages on glioma cell invasion and reveals the mechanism underlying microglia-mediated tumor progression. These findings suggested resident microglia is the potential therapeutic target for TAMs-induced brain tumor invasiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f062fb3413e27a9de43358674b8a4c6d1e771176" target='_blank'>
              Distinct roles of small extracellular vesicles from resident and infiltrating macrophages on glioma growth and mobility
              </a>
            </td>
          <td>
            Chu-Hsuan Chen, Sheng-Yun Hsu, Wen-Jui Yu, Chi-Shiun Chiang, Ching-Fang Yu
          </td>
          <td>2025-01-01</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Desmosomes are transmembrane protein complexes critical for cell-cell adhesion in epithelial tissues and other cell types. In skin cancers like melanoma, epithelial cells called keratinocytes are the predominant cells in the primary melanoma microenvironment, yet it remains largely unknown whether this cell type is subject to genetic alterations during melanoma development and its role in early melanoma progression. In this study, using an integrative analysis of tumor mutations and protein biophysical interactions, we identify a high frequency of genetic alterations in desmosome-related genes in human cancers, with the highest occurrences in cutaneous melanoma. Notably, over 70% of cutaneous melanoma cases exhibit mutations in desmosome genes and desmosome gene mutations are associated with lowered expression of the desmosome complex in melanoma. Using spatial transcriptomics and immunohistochemistry analyses, we find that the decrease in desmosome gene expression predominantly occurs in keratinocytes within the tumor microenvironment rather than in the melanoma cells themselves. To study the functional significance of this alteration in melanoma development, we used a human melanoma/keratinocyte co-culture system using primary melanoma cell lines. Our studies show that knockdown of desmosome genes in keratinocytes markedly increased proliferation of adjacent melanoma cells in melanoma/keratinocyte co-cultures. Additionally, melanoma cell proliferation is enhanced when exposed to media preconditioned by desmosome-deficient keratinocytes. These observations suggest that the gradual accumulation of mutations in desmosome genes within neighboring keratinocytes may create a conducive environment that primes melanoma cells for neoplastic transformation. This study underscores the critical role of genetic alterations in the tumor microenvironment in melanoma progression and highlights the potential for targeting desmosome-related pathways in therapeutic strategies against melanoma.
 Citation Format: Mohita Tagore. Desmosome mutations in the microenvironment regulate melanoma proliferation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A042.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cede4aac61d054a3a470265c4e2db5a458ea7c19" target='_blank'>
              Abstract A042: Desmosome mutations in the microenvironment regulate melanoma proliferation
              </a>
            </td>
          <td>
            Mohita M. Tagore
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5daf8fc78edec4dc67cc0ac87030a2ee471c5078" target='_blank'>
              Cancer-associated fibroblasts: heterogeneity, tumorigenicity and therapeutic targets
              </a>
            </td>
          <td>
            Keke Lv, Tianlin He
          </td>
          <td>2024-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Ewing sarcoma (EwS) is an aggressive bone and soft tissue tumor and EwS patients often succumb to their disease years after initial treatment due to relapses at metastatic sites. Novel therapeutic strategies are needed that can successfully eliminate microscopic residual disease foci that persist at the end of primary therapy. Extracellular matrix (ECM) proteins in the tumor microenvironment (TME) provide critical pro-survival and pro-invasion signals to tumor cells. Our published and unpublished data show that the glycoprotein tenascin-C (TNC) is enriched in EwS metastases and that TNC and other pro-tumorigenic ECM proteins are deposited by subpopulations of CAF-like tumor cells that are activated in response to tumor and TME-derived TGF-beta ligands. TNC is abundantly produced by multiple human tumors of epithelial and non-epithelial lineage but is otherwise rarely expressed outside of development and wound healing. Importantly, TNC is specifically enriched in metastatic lesions where it has been implicated as a mediator of metastatic competence and treatment resistance. In the current work we have tested whether the TNC-rich TME of disseminated EwS tumor foci could be leveraged to direct therapies to sites of residual micrometastatic disease. To achieve this, we generated monovalent and bivalent anti-human TNC VHHs (hTNC-VHH) using a mammalian expression platform. Camelid-derived hTNC-VHH sequences were sourced from published literature. VHHs are single domain heavy chain only antibody fragments with higher stability and tissue penetration than conventional monoclonal antibodies. Using EwS tumor spheroids in collagen-rich 3D culture, hTNC-VHH penetrated dense ECM matrices and bound through multiple cell layers in under 10 minutes. hTNC-VHH accumulated in TNC-positive but not TNC-knockout EwS spheroids and were retained beyond 96 hours. To assess their potential to drive protein and immune therapies to TNC-rich TMEs, we fused hTNC-VHH to immune modulating CD3- and CD28-activating VHHs. These hTNC-VHH-CD3/CD28-conjugates promoted immune cell activation and tumor cell death in PBMC-tumor co-culture assays as determined by CD25 flow cytometry and fluorescence microscopy, respectively. To assess in vivo localization, fluorescently-tagged hTNC-VHH were intravenously injected into mice that had been xenografted with EwS cells. Tissue microscopy confirmed selective binding of hTNC-VHH to small lung and liver EwS micrometastases in less than 4 hours. VHHs were not retained in non-tumor bearing organs including the brain and heart or in tumor-free lung and liver regions. Experiments are ongoing to assess anti-tumor efficacy and pharmacodynamics of hTNC-VHH conjugates in EwS xenografts. Together these findings suggest that the ECM-remodeling properties of CAF-like EwS cells can be exploited to recruit novel ECM-targeting protein therapeutics to micrometastases. If successful, this innovative approach could eradicate microscopic residual disease and prevent metastatic EwS recurrence.
 Citation Format: Emma D. Wrenn, Jason P. Price, Raymond O. Ruff, Nicolas M. Garcia, James M. Olson, Elizabeth R. Lawlor. Leveraging ECM deposition by CAF-like Ewing sarcoma tumor cells to target micrometastases with matrix-binding VHHs [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C050.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9942d5319f0b477046d50c6a8b105f9899b2930d" target='_blank'>
              Abstract C050: Leveraging ECM deposition by CAF-like Ewing sarcoma tumor cells to target micrometastases with matrix-binding VHHs
              </a>
            </td>
          <td>
            Emma D. Wrenn, Jason P. Price, Raymond O. Ruff, Nicolas M Garcia, James M. Olson, Elizabeth R. Lawlor
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play pivotal roles in solid tumor initiation, growth, and immune evasion. However, the optimal biomimetic modeling conditions remain elusive. In this study, we investigated the effects of 2D and 3D culturing conditions on human primary CAFs integrated into a modular tumor microenvironment (TME). Using single-nucleus RNA sequencing (snRNAseq) and Proteomics’ Proximity Extension Assays, we characterized CAF transcriptomic profiles and cytokine levels. Remarkably, when cultured in 2D, CAFs exhibited a myofibroblast (myCAF) subtype, whereas in 3D tumor spheroid cultures, CAFs displayed a more inflammatory (iCAF) pathological state. By integrating single-cell gene expression data with functional interrogations of critical TME-related processes [natural killer (NK)-mediated tumor killing, monocyte migration, and macrophage differentiation], we were able to reconcile form with function. In 3D TME spheroid models, CAFs enhance cancer cell growth and immunologically shield cells from NK cell-mediated cytotoxicity, in striking contrast with their 2D TME counterparts. Notably, 3D CAF-secreted proteins manifest a more immunosuppressive profile by enhancing monocyte transendothelial migration and differentiation into M2-like tumor-associated macrophages (TAMs). Our findings reveal a more immunosuppressive and clinically relevant desmoplastic TME model that can be employed in industrial drug discovery campaigns to expand the cellular target range of chemotherapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b61425d19bdd4e9908dd233e505b35ad2127a3" target='_blank'>
              Unveiling immune cell response disparities in human primary cancer-associated fibroblasts between two- and three-dimensional cultures
              </a>
            </td>
          <td>
            Jian-Ping Yang, Nikhil Nitin Kulkarni, Masashi Yamaji, Tsubasa Shiraishi, Thang Pham, Han Do, Nicole Aiello, Michael Shaw, Toshihiro Nakamura, Akiko Abiru, N. Gavva, Shane R. Horman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="ABSTRACT High-risk non-muscle-invasive bladder cancer (NMIBC) presents high recurrence and progression rates. Despite the use of Bacillus Calmette-Guérin gold-standard immunotherapy and the recent irruption of anti-PD-1/PD-L1 drugs, we are missing a comprehensive understanding of the tumor microenvironment (TME) that may help us find biomarkers associated to treatment outcome. Here, we prospectively analyzed TME composition and PD-L1 expression of tumor and non-tumoral tissue biopsies from 73 NMIBC patients and used scRNA-seq, transcriptomic cohorts and tissue micro-array to validate the prognostic value of cell types of interest. Compared to non-tumoral tissue, NMIBC presented microvascular alterations, increased cancer-associated fibroblast (CAF) and myofibroblast (myoCAF) presence, and varied immune cell distribution, such as increased macrophage infiltration. Heterogeneous PD-L1 expression was observed across subsets, with macrophages showing the highest expression levels, but cancer cells as the primary potential anti-PD-L1 binding targets. Unbiased analysis revealed that myoCAF and M2-like macrophages are specifically enriched in high-grade NMIBC tumors. The topological distribution of these two cell types changed as NMIBC progresses, as shown by immunofluorescence. Only myoCAFs were associated with higher rates of progression and recurrence in three independent cohorts (888 total patients), reaching prediction values comparable to transcriptomic classes, which we further validated using tissue micro-array. Our study provides a roadmap to establish the landscape of the NMIBC TME, highlighting myoCAFs as potential prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe97543b0d1de2e394926985ac37960e50a2bea" target='_blank'>
              Characterisation of the tumour microenvironment and PD-L1 granularity reveals the prognostic value of cancer-associated myofibroblasts in non-invasive bladder cancer
              </a>
            </td>
          <td>
            Carmen G. Cañizo, F. Guerrero-Ramos, M. Perez Escavy, I. Lodewijk, C. Suárez-Cabrera, L. Morales, Sandra P Nunes, E. Munera-Maravilla, Carolina Rubio, Rebeca Sánchez, Marta Rodríguez-Izquierdo, Jaime Martínez de Villarreal, Francisco X. Real, D. Castellano, Cristina Martín-Arriscado, David Lora Pablos, A. Rodríguez Antolín, M. Dueñas, J. Paramio, Victor G. Martínez
          </td>
          <td>2024-12-19</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Aging is a multifaceted process that manifests differently in various cell types and within the same cell type. Therefore, understanding age-related changes with single-cell resolution may illuminate the mechanistic details of tissue aging and offer targets for intervention. We utilized high-throughput single-cell RNA-sequencing (scRNA-seq) to analyze senescence-associated changes within the CD4 T cell compartment. First, we sequenced total human CD4 T cells from multiple donors, revealing that naïve CD4 T cells exhibited a less senescent state compared to memory cells. We focused on naïve CD4 T cells to determine if individual cells differ in expression of aging-associated markers. By sorting cells based on senescence-associated beta-galactosidase (B-gal) activity and applying scRNA-seq, we identified transcriptional signatures associated with more youthful and senescent states. Surprisingly, we discovered cells with lower B-gal expression comprised two distinct populations: one marked by high expression of HELIOS and low expression of SESN3, and another marked by high expression of BACH2 and low expression of TC2N. In comparison, 2 youthful subpopulation and senescence subpopulations differ in markers of immunological activation We then investigated if the composition of naive CD4 compartment changes with age, finding that the abundance of youthful subpopulations was reduced in the older donors’ CD4 populations. Thus, aging in the naïve CD4 compartment is associated with a shifted distribution between more youthful and more senescent cells. Moving forward, we will attempt to manipulate naïve CD4 T cells into a more youthful state by altering expression of the identified transcriptional signatures associated with youthful and senescent states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bab6ece44b0e3064c9cd597254aa9a3e1045d71" target='_blank'>
              DISTINCT CELL SUBTYPES WITHIN NAÏVE CD4 T CELLS COMPARTMENT SHAPE ITS AGING AND SENESCENCE
              </a>
            </td>
          <td>
            C. J. Kelly, Ashley Jeon, Jacob Rodriguez, Dina Prosser, Morgan Diegel, Nikolay Burnaevskiy
          </td>
          <td>2024-12-01</td>
          <td>Innovation in Aging</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3b8dd5bbe35e524e752fa9c462148784357715a" target='_blank'>
              Integrative ensemble modelling of cetuximab sensitivity in colorectal cancer patient-derived xenografts
              </a>
            </td>
          <td>
            Umberto Perron, Elena Grassi, Aikaterini Chatzipli, Marco Viviani, Emre Karakoc, Lucia Trastulla, Lorenzo M. Brochier, C. Isella, E. Zanella, Hagen Klett, Ivan Molineris, Julia Schueler, M. Esteller, Enzo Médico, Nathalie Conte, Ultan McDermott, L. Trusolino, A. Bertotti, F. Iorio
          </td>
          <td>2024-11-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Ambiguous understanding of tumors and the tumor microenvironments (TMEs) hinders accurate diagnosis and available treatment for multifocal hepatocellular carcinoma (HCC) covering intrahepatic metastasis (IM) and multicentric occurrence (MO). Here, we characterized the diverse TMEs of IM and MO identified by whole-exome sequencing (WES) at single-cell resolution.


APPROACH AND RESULTS
We performed parallel WES and single-cell RNA sequencing (scRNA-seq) on twenty-three samples from seven patients to profile their TMEs when major results were validated by immunohistochemistry in the additional cohort. Integrative analysis of WES and scRNA-seq found that malignant cells in IM showed higher intra-tumor heterogeneity, stemness and more activated metabolism than those in MO. Tumors from IM shared similar TMEs while distinct TMEs were noticed in those from MO. Furthermore, CD20+ B cells, plasma cells and conventional type II dendritic cells (cDC2s) were decreased in IM relative to MO while T cells in IM exhibited a more terminally exhausted capacity with a higher proportion of proliferative/exhausted T cells than that in MO. Both CD20 and CD1C correlated with better prognosis in multifocal HCC. Additionally, MMP9+ tumor-associated macrophages were enriched across IM and MO which formed cellular niches with regulatory T cells (Tregs) and proliferative/exhausted T cells.


CONCLUSIONS
Our findings deeply decipher the heterogeneous TMEs between IM and MO, which provide a comprehensive landscape of multifocal HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2c7a45eee2228edb84e8693a219534a9ab548a" target='_blank'>
              Genomic and the tumor microenvironment heterogeneity in multifocal hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yongheng Yang, Qingqiang Ni, Hongguang Li, Jiuzheng Sun, Xia Zhou, Lingxin Qu, Liyuan Wang, Chuanzong Zhao, Xiaolu Zhang
          </td>
          <td>2024-12-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma IDH wild type (GB), the most common malignant primary brain tumor, is characterized by rapid proliferation, extensive infiltration into surrounding brain tissue, and significant resistance to current therapies. Median survival is only 15 months despite extensive clinical efforts. The tumor microenvironment (TME) in GB is highly specialized, supporting the tumor’s aggressive behavior and its ability to evade conventional treatments. One critical component is the aberrant vascular network that complicates the delivery of chemotherapy across the blood–brain barrier. Antiangiogenic therapies emerged as a promising option but have shown limited efficacy in extending the survival of these patients. Comprehension of the complex vascular network of GB may be a key to overcoming the limitations of current therapies. Pericytes are gaining recognition within the context of the TME. These mural cells are essential for vascular integrity and may contribute to tumor progression and therapeutic resistance. Although their role has been evidenced in other tumors, they remain underexplored in GB. Pericytes are known to respond to tumor hypoxia and interact with vascular endothelia, influencing responses to DNA damage and antiangiogenic treatments. They actively regulate not only angiogenesis but also the different vasculogenic strategies for tumor neovascularization. Additionally, they affect leukocyte trafficking and tumor-associated macrophages. This review aims to integrate the various functions controlled by pericytes to favor deeper investigation into their actionable potential. Pericytes may represent a promising target for novel therapeutic strategies in order to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835c71d63c5567604bb678c8f9634fc408771476" target='_blank'>
              Pericytes in Glioblastoma: Hidden Regulators of Tumor Vasculature and Therapy Resistance
              </a>
            </td>
          <td>
            Irene Salazar-Saura, María Pinilla-Sala, J. Megías, Lara Navarro, Esther Roselló-Sastre, T. San-Miguel
          </td>
          <td>2024-12-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25ea97844e3ac53280202c1ab1e25a03d1cb6ccc" target='_blank'>
              Tailoring glioblastoma treatment based on longitudinal analysis of post-surgical tumor microenvironment
              </a>
            </td>
          <td>
            C. Bastiancich, E. Snacel-Fazy, Samantha Fernandez, Stéphane Robert, Roberta Stacchini, Léa Plantureux, Sébastien Boissonneau, Benoit Testud, Benjamin Guillet, F. Debarbieux, H. Luche, D. Figarella-Branger, Marie-Anne Estève, E. Tabouret, A. Tchoghandjian
          </td>
          <td>2024-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Hepatic arterial infusion chemotherapy (HAIC) has emerged as a promising treatment strategy for hepatocellular carcinoma (HCC), but a detailed understanding of the multicellular ecosystem after HAIC treatment is lacking. Here, we collected tumor samples from treatment-naïve primary and post-HAIC HCC, and integrated single-nucleus RNA sequencing with spatial transcriptomics to characterize the tumor ecosystem in the post-HAIC HCC. Increased fractions and enhanced cellular communication of CD4+ T, CD20+ B, and dendritic cell subtypes were identified in post-HAIC tumors. Moreover, it is substantiated that HAIC promoted tertiary lymphoid structures (TLS) formation, and addressed the roles of TLSs as spatial niches of cellular communication. Specifically, intermediate exhausted CD8+ T cells expressing Granzyme-K and PD-1 (PD-1+CD8+ Tex-int) expanded following HAIC and exhibited a functionally antitumor phenotype. PD-1+CD8+ Tex-int accumulated in the TLS vicinity and disseminated throughout the tumor microenvironment, demonstrating potential as an effective biomarker for HAIC-based treatment in HCC. This study provides valuable resources and biological insights in the cellular underpinnings of HAIC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62624d4b04f46d42e65710fe0c98f6d6afaa46d" target='_blank'>
              Single-Nucleus and Spatial Transcriptome Profiling Delineates the Multicellular Ecosystem in Hepatocellular Carcinoma After Hepatic Arterial Infusion Chemotherapy.
              </a>
            </td>
          <td>
            YeXing Huang, Zefeng Du, Zhicheng Lai, DongSheng Wen, Lichang Huang, Minke He, ZiChao Wu, Huifang Li, Hanyue Ouyang, Wenchao Wu, Anna Kan, Ming Shi
          </td>
          <td>2024-12-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b26feb24c74a70e25f404994931c40c73dd0fe64" target='_blank'>
              Deciphering molecular landscape of breast cancer progression and insights from functional genomics and therapeutic explorations followed by in vitro validation
              </a>
            </td>
          <td>
            Bushra Khan, Rowaid M. Qahwaji, Mashael S. Alfaifi, Tanwir Athar, Abdullah Khan, Mohammad Mobashir, I. Ashankyty, K. Imtiyaz, Areej Alahmadi, M. M. Rizvi
          </td>
          <td>2024-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aaea8ef96bcf757be9b672ce6a26fbe4a6d0c91" target='_blank'>
              Integration of single-cell sequencing and drug sensitivity profiling reveals an 11-gene prognostic model for liver cancer
              </a>
            </td>
          <td>
            Qunfang Zhou, Jingqiang Wu, Jiaxin Bei, Zixuan Zhai, Xiuzhen Chen, Wei Liang, Jing Meng, Mingyu Liu
          </td>
          <td>2024-11-25</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d49c3b89849fb97fd894e63e495575849c7b7545" target='_blank'>
              A multilineage screen identifies actionable synthetic lethal interactions in human cancers.
              </a>
            </td>
          <td>
            Samson H. Fong, Brent M Kuenzi, Nicole M Mattson, John Lee, Kyle S. Sanchez, A. Bojorquez-Gomez, Kyle Ford, Brenton P. Munson, Katherine Licon, Sarah Bergendahl, John Paul Shen, J. Kreisberg, Prashant Mali, Jeffrey H Hager, Michael A White, T. Ideker
          </td>
          <td>2024-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="The vomeronasal organ (VNO) detects signaling molecules that often prompt innate behaviors such as aggression and reproduction. Vomeronasal sensory neurons, classified into apical and basal lineages based on receptor expression, have a limited lifespan and are continuously replaced from a common stem cell niche. Using a combination of single-cell RNA sequencing data, immunofluorescence staining, and lineage tracing, we identified CXCR4 expression in proliferative stem cells and the basal neuronal lineage. Mice with a conditional knockout of Cxcr4 showed an increased number of SOX2-positive proliferative stem cells and enhanced basal neuronal lineage maturation. In addition, computational gene perturbation analysis revealed 87 transcription factors that may contribute to neurogenesis, among which SOX2. Conditional knockout of Cxcr4 did not only disturb neuronal maturation, but also affected non-neuronal cell types, resulting in a decrease of basal lamina lining quiescent stem cells and an increase in sustentacular support cells. Together, these findings enhance our understanding how a common pool of stem cells can give rise to different cell types of the VNO, highlighting the distinct role of CXCR4 in this process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17d61a169986ba05c46333384d4d3c1c24808357" target='_blank'>
              Stem cell expressed CXCR4 regulates tissue composition in the vomeronasal organ.
              </a>
            </td>
          <td>
            André Dietz, Katja Senf, Eva M Neuhaus
          </td>
          <td>2024-12-06</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 We recently described orphan non-coding RNAs (oncRNAs) as a class of cancer-specific small RNAs with the potential to play functional roles in breast cancer progression1. Here, we report a systematic search to annotate and characterize cancer-emergent oncRNAs across 32 tumor types. We also leverage large-scale in vivo genetic screens in xenografted mice to functionally identify driver oncRNAs in multiple tumor types. We have not only discovered a large repertoire of oncRNAs, but also found that their presence and absence represent a digital molecular barcode that faithfully captures the types and subtypes of cancer. Importantly, we discovered that this molecular barcode is partially accessible from the cell-free space as some oncRNAs are secreted by cancer cells. In a large retrospective study across 192 breast cancer patients, we showed that oncRNAs can be reliably detected in the blood and that changes in the cell-free oncRNA burden captures both short-term and long-term clinical outcomes upon completion of a neoadjuvant chemotherapy regimen.
 Citation Format: Hani Goodarzi. Systematic Discovery and Annotation of Cancer Emergent Orphan non-coding RNAs in human Cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr I004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842b60b16b6659e409e6c2de70590b3fbdb8fdc2" target='_blank'>
              Abstract I004: Systematic Discovery and Annotation of Cancer Emergent Orphan non-coding RNAs in human Cancers
              </a>
            </td>
          <td>
            Hani Goodarzi
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is the third leading cause of cancer-related deaths with a 5-year survival rate of 13%. Surgical resection remains the only curative option as systemic therapies offer limited benefit. Poor response to chemotherapy and immunotherapy is due, in part, to the dense stroma and heterogeneous tumor microenvironment (TME). Opportunities to target the PDAC stroma may increase the effectiveness of existing or novel therapies. Current strategies targeting the stromal compartment within the PDAC TME primarily focus on degrading extracellular matrix or inhibiting stromal cell activity, angiogenesis, or hypoxic responses. In addition, extensive work has attempted to use immune targeting strategies to improve clinical outcomes. Preclinically, these strategies show promise, especially with the ability to alter the tumor ecosystem; however, when translated to the clinic, most of these trials have failed to improve overall patient outcomes. In this review, we catalog the heterogenous elements of the TME and discuss the potential of combination therapies that target the heterogeneity observed in the TME between patients and how molecular stratification could improve responses to targeted and combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254631c9190ee22fb9be4694a816eb9553b6afce" target='_blank'>
              Challenges and Opportunities in Targeting the Complex Pancreatic Tumor Microenvironment
              </a>
            </td>
          <td>
            Jennifer M. Finan, Yifei Guo, Shaun M Goodyear, Jonathan R. Brody
          </td>
          <td>2024-12-01</td>
          <td>JCO Oncology Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor microenvironment (TME) plays a crucial role in the oncogenesis, early distant metastasis, and limited immunotherapeutic responses of pancreatic ductal adenocarcinoma (PDAC). Myeloid cell populations, particularly tumor-associated macrophages (TAMs), located within this immunosuppressive niche, are pivotal in these processes. With aid of single cell RNA sequencing (scRNA-seq), multiple studies have successfully validated distinct macrophage subtypes pathological functions in different cancer types, yet the correlation of TAMs in hepatic metastatic and primary PDAC lesions remains insufficiently understood. In this study, we reanalyzed samples from both primary and hepatic metastatic PDACs to elucidate the functional conservation and variability of TAMs. Pseudotime trajectory inference, together with pseudobulk analysis based on subtype level, manifests TAMs undergo similar metabolism reprogramming and pro-inflammatory style transitions with specific signaling pathway activations. Compared to primary tumors, hypoxic conditions are alleviated, which largely depends on the physiological context and the preliminary ligand-receptor interactive network within the metastatic niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a56bbf6854596a35b27cf40482bd3f5c23fafab" target='_blank'>
              In-depth analysis of pathological function of tumor-associated macrophage populations in hepatic metastatic and primary PDAC
              </a>
            </td>
          <td>
            Fanyu Meng
          </td>
          <td>2024-12-12</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e013dd21a317acebc6bbd04a30655def3c28bb68" target='_blank'>
              Decorin as a key marker of desmoplastic cancer-associated fibroblasts mediating first-line immune checkpoint blockade resistance in metastatic gastric cancer.
              </a>
            </td>
          <td>
            Ki Tae Kim, Min Hee Lee, Su-Jin Shin, In Cho, Jung Cheol Kuk, Jina Yun, Yoon Young Choi
          </td>
          <td>2024-11-09</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metastatic progression is responsible for the majority of cancer-related deaths. A better understanding of the underlying molecular mechanisms driving metastasis therefore remains of utmost importance. Here we take efforts to excavate mutant driver factors of metastases in the level of signaling pathways. We introduce two models, EntCDP and ModSDP, to detect the similarity and specificity of pathways in a pan-cancer context. The simulation studies confirm their feasibility and identification accuracy. Using mutation profiles of 17 cancer types with primary and metastatic cohorts from MSK-IMPACT, we apply the two models to investigate the transition of signaling systems from primary to metastatic tumors mainly from five perspectives. 1) We initially perform a comprehensive comparison of 15 primary-metastatic cancer pairs. Nearly all shared common driver genes or pathways, while specific driver gene sets of primary or/and metastatic tumors can be seen for almost each pair, except for breast cancer. 2) We use ModSDP to identify the specificity of metastases with the same primary site and highlight calcium signaling in breast cancer with bone metastasis, FoxO signaling for melanoma/lung lymph node metastasis, etc. 3) Oppositely, we use EntCDP to identify common factors yielding three frequently metastatic sites (liver, lung and brain), and recognized cushing syndrome, microRNAs in cancer and MAPK signaling pathway, separately. 4) Regarding high-tropism metastatic patterns, we investigated the relationship between the metastatic tumor and primary tumors in both locations. In the typical pattern of colorectal cancer metastasis to the liver, we detected hepatic signals in primary colorectal cancer. 5) Finally, we focused on the common and specific characteristics of relevant cancer types, such as gender-related cancers and gastrointestinal cancers. Study on gynecologic tumors suggest endocrine hormone change and virus infection as their common risk factors, while for male we highlight hedgehog signaling and EPH-related receptors as invasive potential of prostate cancer. Multiple interesting findings revealed by this study may be helpful for the understanding of the extent of signal changes during tumor metastasis. We expect that this study will provide a valuable resource for transforming our knowledge about signals in cancer metastasis into alternative clinical practice for advanced patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facca2ecc611248ea30b446f5773d2c64fea06c3" target='_blank'>
              Pan-cancer study on transition of signaling systems from primary to metastatic tumors
              </a>
            </td>
          <td>
            Wenjia Zhou, Junhua Zhang
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Metastasis in breast cancer is a major challenge for prognosis and effective treatment, primarily because cancer cells can create an immune-suppressive microenvironment that supports their progression and spread. The physical properties of cancer cells significantly influence their metastatic potential and their interaction with the environment. Identifying targets that impact both the physical characteristics of cancer cells, and their microenvironment is crucial for tackling metastatic disease. Interferon-stimulated gene 15 (ISG15), a ubiquitin-like protein, is often overexpressed in breast cancer tissues and is associated with poor prognosis, higher tumor grade, and increased invasiveness. Despite these associations, the precise role of ISG15 and its post-translational modification, ISGylation, in cancer cell behavior and interactions with their microenvironment remains unclear. To explore this, we interrogated the biophysical role of ISG15 by using mouse modeling and in vitro loss of function approaches. We utilized whole- body ISG15-deficient mice and conducted a series of experiments to assess the impact of ISG15 on cancer cell properties and their surrounding environment. Our preliminary results indicate that cells lacking ISG15 exhibit altered physical traits, such as increased stiffness and size, which may influence their metastatic behavior. Additionally, targeting ISG15 or disrupting ISGylation enhances the susceptibility of cancer cells to immune cell-mediated destruction. In ISG15- deficient mice, we observed significantly reduced tumor growth and metastasis compared to wild-type controls. These findings suggest that ISG15 modulates critical aspects of cancer cell behavior and their interactions with the microenvironment, contributing to immune evasion and metastatic spread. Our research highlights ISG15 as a promising therapeutic target and provides a foundation for developing new strategies to prevent and treat metastatic cancer
 Citation Format: Mohamed Haloul, Priya Shah, Vinay Pai, Faruk Hossen, James James Lee, Ekrem Emrah Er. ISG15 modulate cancer cell mechanics and metastasis in breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B027.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a28aa9d49c850dc48c11b47ab53305430122a856" target='_blank'>
              Abstract B027: ISG15 modulate cancer cell mechanics and metastasis in breast cancer
              </a>
            </td>
          <td>
            Mohamed Haloul, Priya Shah, Vinay Pai, Faruk Hossen, James James Lee, E. E. Er
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The transcriptional heterogeneity of tumor microenvironment (TME) cells is a crucial factor driving the diversity of cellular response to drug treatment and resistance. Therefore, characterizing the cells associated with drug treatment and resistance will help us understand therapeutic mechanisms, discover new therapeutic targets and facilitate precision medicine. Here, we describe a database, scDrugAct (http://bio-bigdata.hrbmu.edu.cn/scDrugAct/), which aims to establish connections among drugs, genes and cells and dissect the impact of TME cellular heterogeneity on drug action and resistance at single-cell resolution. ScDrugAct is curated with drug-cell connections between 3838 223 cells across 34 cancer types and 13 857 drugs and identifies 17 274 drug perturbation/resistance-related genes and 276 559 associations between >10 000 drugs and 53 cell types. ScDrugAct also provides multiple flexible tools to retrieve and analyze connections among drugs, genes and cells; the distribution and developmental trajectories of drug-associated cells within the TME; functional features affecting the heterogeneity of cellular responses to drug perturbation and drug resistance; the cell-specific drug-related gene network; and drug-drug similarities. ScDrugAct serves as an important resource for investigating the impact of the cellular heterogeneity of the TME on drug therapies and can help researchers understand the mechanisms of action and resistance of drugs, as well as discover therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/043f2ce1e460a880b52e0d2bd7d4a4783b1e445d" target='_blank'>
              ScDrugAct: a comprehensive database to dissect tumor microenvironment cell heterogeneity contributing to drug action and resistance across human cancers.
              </a>
            </td>
          <td>
            Yanjun Xu, Yifang Zhang, Kaiyue Song, Jiaqi Liu, Rui Zhao, Xiaomeng Zhang, Liying Pei, Mengyue Li, Zhe Chen, Chunlong Zhang, Peng Wang, Feng Li
          </td>
          <td>2024-11-11</td>
          <td>Nucleic acids research</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Fatty acid binding protein 7 (FABP7) is a multifunctional chaperone involved in lipid metabolism and signaling. It is primarily expressed in astrocytes and neural stem cells (NSCs), as well as their derived malignant glioma cells within the central nervous system. Despite growing evidence for FABP7’s tumor-intrinsic onco-metabolic functions, its mechanistic role in regulating the brain tumor immune microenvironment (TIME) and its impact on prognosis at the molecular level remain incompletely understood. Utilizing combined transcriptome profiling and pan-cancer analysis approaches, we report that FABP7 mediates the expression of multiple onco-immune drivers, collectively impacting tumor immunity and clinical outcomes across brain cancer subtypes. An analysis of a single-cell expression atlas revealed that FABP7 is predominantly expressed in the glial lineage and malignant cell populations in gliomas, with nuclear localization in their parental NSCs. Pathway and gene enrichment analysis of RNA sequencing data from wild-type (WT) and Fabp7-knockout (KO) mouse brains, alongside control (CTL) and FABP7-overexpressing (FABP7 OV) human astrocytes, revealed a more pronounced effect of FABP7 levels on multiple cancer-associated pathways. Notably, genes linked to brain cancer progression and tumor immunity (ENO1, MUC1, COL5A1, and IL11) were significantly downregulated (>2-fold) in KO brain tissue but were upregulated in FABP7 OV astrocytes. Furthermore, an analysis of data from The Cancer Genome Atlas (TCGA) showed robust correlations between the expression of these factors, as well as FABP7, and established glioma oncogenes (EGFR, BRAF, NF1, PDGFRA, IDH1), with stronger associations seen in low-grade glioma (LGG) than in glioblastoma (GBM). TIME profiling also revealed that the expression of FABP7 and the genes that it modulates was significantly associated with prognosis and survival, particularly in LGG patients, by influencing the infiltration of immunosuppressive cell populations within tumors. Overall, our findings suggest that FABP7 acts as an intracellular regulator of pro-tumor immunomodulatory genes, exerting a synergistic effect on the TIME and clinical outcomes in brain cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15939c78a79b5a83578829fc7d538cf232f24932" target='_blank'>
              Integrated Transcriptome Profiling and Pan-Cancer Analyses Reveal Oncogenic Networks and Tumor-Immune Modulatory Roles for FABP7 in Brain Cancers
              </a>
            </td>
          <td>
            Yool Lee, Carlos C. Flores, Micah Lefton, Sukanya Bhoumik, Yuji Owada, Jason R. Gerstner
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 The awakening of dormant disseminated cancer cells is likely responsible for the clinical relapses of patients whose primary tumors have been cured months and even years earlier. In the present study, we demonstrate that dormant breast cancer cells lodged in the lungs reside in a highly mesenchymal, non-proliferative phenotypic state. The awakening of these cells does not occur by a cancer cell-autonomous process. Instead, inflammation and wound healing of the surrounding tissue microenvironment causes them to shift from a highly mesenchymal to a quasi-mesenchymal phenotypic state in which they acquire stemness and proliferative ability. Once awakened, these cells can stably reside in this quasi-mesenchymal state and maintain their stemness, doing so without ongoing heterotypic signaling from the lung microenvironment. EGFR ligands released by the cells of the injured tissue microenvironment, including notably M2 type macrophages, promote dormant cancer cells to move toward this quasi-mesenchymal state, a transition that is essential for the awakening process. An understanding of the mechanisms of metastatic awakening may lead in the future to treatment strategies designed to prevent such awakening and resulting metastatic relapse.
 Citation Format: Jingwei Zhang, Robert Weinberg. Awakened dormant cancer cells undergo highly mesenchymal to quasi- mesenchymal transition and acquire stemness [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8beb7f3a092f66cbe81343c9f59c0b804bcf8a" target='_blank'>
              Abstract C018: Awakened dormant cancer cells undergo highly mesenchymal to quasi- mesenchymal transition and acquire stemness
              </a>
            </td>
          <td>
            Jingwei Zhang, Robert Weinberg
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this study, we delve into the intrinsic mechanisms of cell communication in hepatocellular carcinoma (HCC). Initially, employing single-cell sequencing, we analyze multiple malignant cell subpopulations and cancer-associated fibroblast (CAF) subpopulations, revealing their interplay through receptor-ligand interactions, with a particular focus on SPP1. Subsequently, employing unsupervised clustering analysis, we delineate two clusters, C1 and C2, and compare their infiltration characteristics using various tools and metrics, uncovering heightened cytotoxicity and overall invasion abundance in C1. Furthermore, our gene risk scoring model indicates heightened activity of the immune therapeutic pathway in C1. Lastly, employing a formulated scoring system, we stratify patients into high and low-risk groups, revealing notably poorer outcomes in the high-risk cohort on Kaplan-Meier curves. Risk scores exhibit a negative correlation with model genes and immune cell infiltration scores, indicating poor prognosis in the high-risk group. Further characterization elucidates the regulatory landscape of the high and low-risk groups across various signaling pathways. In addition, we used wet lab experiments to prove that ABCA1 plays a pro-oncogenic role in hepatocellular carcinoma cells by promoting proliferation, invasion, migration, and reducing apoptosis. In summary, these findings provide crucial insights, offering valuable clues and references for understanding HCC pathogenesis and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ebd5bca77500ddcf823c450cf7ed50074d3c6d" target='_blank'>
              ABCA1 promote tumor environment heterogeneity via epithelial mesenchymal transition in Huh7 and HepG2 liver cancer cell
              </a>
            </td>
          <td>
            Dinglai Yu, Fang Guo, Qiyu Zhang, Huajun Yu, Wenmin Wang, Yunzhi Chen
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) can drive tumor initiation, progression and resistance in some of the most aggressive cancers and is emerging as a promising anti-cancer target. However, detection currently requires costly whole-genome sequencing (WGS) or labor-intensive cytogenetic or FISH imaging, limiting its application in routine clinical diagnosis. To overcome this, we developed ecPath (ecDNA from histopathology), a computational method for predicting ecDNA status from routinely available hematoxylin and eosin (H&E) images. ecPath implements a deep-learning method we call transcriptomics-guided learning, which utilizes both transcriptomics and H&E images during the training phase to enable successful ecDNA prediction from H&E images alone, a task not achievable with models trained on H&E images only. It is trained on more than 6,000 tumor whole-slide images from the TCGA cohort with the best performance in predicting ecDNA status in brain and stomach tumors (average AUC=0.78). ecPath revealed that ecDNA-positive tumors are enriched with pleomorphic, larger and high-density nuclei. Testing in an independent cohort, ecPath predicted ecDNA status of 985 pediatric brain tumor patients with an AUC of 0.72. Finally, we applied ecPath to identify ecDNA-positive tumors in the TCGA cohort for which no WGS data were available. Like WGS-based ecDNA-positive labels, the predicted ecDNA-positive status also identify poor prognoses for low grade glioma patients. These results demonstrate that ecPath enables the detection of ecDNA from routinely available H&E imaging alone and help nominate aggressive tumors with ecDNA to study and target it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ffb85a2375bcb28d5b3da1f1dc6239edbfdf5f" target='_blank'>
              ecPath detects ecDNA in tumors from histopathology images
              </a>
            </td>
          <td>
            Mudra Choudhury, Lihe Liu, Anamika Yadav, Owen S Chapman, Zahra Ahmadi, Raneen Younis, Chinmay Sharma, Navansh Goel, S. Sridhar, Rishaan Kenkre, Aditi Dutta, Shanqing Wang, E. Shulman, S. R. Dhruba, Danh-Tai Hoang, Kevin M. Tharp, Megan R Paul, Denise Malicki, K. Yip, E. Ruppin, Lukas Chavez, Sanju Sinha
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="It is important to systematically identify tumor suppressor genes (TSGs) to improve our understanding of tumorigenesis and develop strategies for early diagnosis and mitigating disease progression. In the present study, we used an in vivo genome-wide clustered regularly interspaced short palindromic repeats (CRISPR)-CRISPR-associated protein 9 (Cas9) screen and identified FPS/FES-related (FER) as a TSG. Single-cell RNA sequencing (scRNA-seq) revealed that normal cells with low FER expression exhibited elevated malignant transformation potential and stemness properties. FER knockout promoted the tumorigenic transformation, characterized by high colony-forming efficiency and suspension growth ability, acquired tumorigenicity in vivo, increased metabolic activity, dedifferentiation properties, and immune evasion. Moreover, analysis revealed that low FER expression tumors share molecular phenotypes with FER knockout cells, suggesting the consistent role of FER in tumor initiation and progression. Taken together, our findings not only provide insights into the essential role of FER as a tumor suppressor in tumor initiation and progression but also highlight its potential as a target for future clinical diagnosis. © 2024 The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d0ef168e6b6a500c7fdbd949d7e67ca980d8052" target='_blank'>
              CRISPR-Cas9 screening identifies the role of FER as a tumor suppressor.
              </a>
            </td>
          <td>
            Jiaqi Wang, Ran Yang, Feng-Po Wang, Junlei Zhang, Yutong Dong, Jiangjun Wang, Meng Yu, Yixiao Xu, Lianlian Liu, Yuda Cheng, Chen Zhang, Yi Yang, Wubin Yang, Jiali Wang, Guangxing Chen, Yi Huang, Yanping Tian, Rui Jian, Bing Ni, Wei Wu, Y. Ruan
          </td>
          <td>2024-12-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is characterized by recurrent somatic mutations in epigenetic regulators, which stratify patients into clinically significant subgroups with distinct prognoses and treatment responses. However, the cell type-specific epigenetic landscape of RCC—broadly and in the context of these mutations—is incompletely understood. To investigate these open questions, we integrated single nucleus ATAC sequencing data from RCC tumors across four independent cohorts. In clear cell RCC tumors, we identified four shared malignant epigenetic programs related to angiogenesis, proximal tubule-like features, interferon (IFN) signaling, and one that lacked distinct genomic regions with increased accessibility. Among the mutated epigenetic regulators, BAP1 mutation exhibited the most significant impact on chromatin accessibility in tumor cells, and the associated epigenetic changes were linked to IFN response. We identify multiple potential sources of elevated IFN signaling in these lesions, such as increased immune infiltration and increased accessibility and expression of an IFN-associated ERV, ERV3-16A3_LTR. We find that the expression of ERV3-16A3_LTR may itself be a negative prognostic biomarker in ccRCC. Our findings highlight the convergence of malignant epigenetic programs across ccRCC tumors and suggest that BAP1 loss, potentially through ERV3-16A3_LTR dysregulation, is associated with an IFN response-high epigenetic program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f60a362700f5a511019adc2819b352bb15feca9" target='_blank'>
              Single-cell epigenetic profiling reveals an interferon response-high program associated with BAP1 deficiency in kidney cancer
              </a>
            </td>
          <td>
            Sabrina Y. Camp, M. X. He, Michael S. Cuoco, E. Saad, Erica Pimenta, Kevin Meli, Z. Bakouny, C. Labaki, Breanna M. Titchen, Yun Jee Kang, Jack Horst, Rachel Trowbridge, Erin Shannon, K. Helvie, A. Thorner, S. Vigneau, Angie Mayorga, Jahnavi Kodali, Hannah Lachmayr, Meredith Bemus, Jihye Park, T. Choueiri, Kevin Bi, E. V. Van Allen
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Mucinous colorectal carcinoma (MUC CRC) dissemination into the tumor stroma and metastasis to multiple organs, including the peritoneum, is associated with poor prognosis. Disseminating MUC CRCs exhibit either a conventional ‘apical-in’ or an inverted ‘apical-out’ polarity phenotype that influence patient outcome. Identifying the mechanisms controlling MUC CRC polarity is critical to understand disease progression. Here, we analyze patient-derived MUC CRC xenografts, with apical-in or apical-out polarity, ex vivo or within collagen gels to mimic the peritumoral stroma. Single-cell analyses reveal α2β1-integrin as a key collagen-binding receptor in these models. Collagen–α2β1-integrin interaction activates Src and ERK/MAPK signaling and upregulates the expression of SorLA, an endosomal sorting receptor. SorLA supports apical-in polarity and carcinoma-stroma interactions by promoting integrin recycling to the plasma membrane and HER2/HER3 expression through a positive feedback mechanism. Accordingly, we observe positive correlation between HER2, HER3 and SorLA in patient samples with the highest HER2 expression in apical-in-presenting tissues. Treatment of tumor spheres with clinically relevant HER2/HER3-targeting antibodies reverts sphere polarity and impedes collagen remodeling and adhesion to mouse peritoneum. This SorLA—integrin—HER2/HER3 signaling axis may represent a basis for MUC CRC-patient stratification and shed light on other carcinomas with similar apical-out phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/284740ed86720e8dd61c88e27e9b05faf5479ca5" target='_blank'>
              Signaling downstream of tumor-stroma interaction regulates mucinous colorectal adenocarcinoma apicobasal polarity
              </a>
            </td>
          <td>
            Nicolas Pasquier, A. Isomursu, Hellyeh Hamidi, J. Mathieu, Jouni Härkönen, Gautier Follain, Christophe Desterke, Zoé Fusilier, Junel Solis, Irina Belaya, Pasi Kankaanpää, Valeria Barresi, J. Cartry, Sabrina Bedja, Fanny Jaulin, Johanna Ivaska
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells aggregate to form clusters, which have enhanced stem-like properties and metastasis potential. However, the molecular mechanisms underlying the formation of cancer stem cell cluster-like structures with acquisition of stronger invasion and metastasis abilities remain unclear. Micropapillary carcinoma (MPC) is a subpopulation of small, merulioid, inverted, nonfibrous vascular clusters floating in the stroma present in a range of solid malignant tumors and characterized by frequent vascular/lymphatic vessel invasion and lymph node metastasis. Our results showed that these cell clusters exhibit a stem cell phenotype, supporting the premise that MPC may serve as a promising solid tumor model for studying invasion and metastasis of cancer stem cell clusters. In this review, we discuss the latest advances in MPC research and targeted therapy, focusing on analysis of their stem-like characteristics, mapping their multiomics characteristics, and elucidating the vascular and immune microenvironment of MPC. The existing MPC organoid model was employed to explore potential breakthroughs in targeted therapy and immunotherapy for cancer stem cell clusters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/465f3c6afb9220f958e9f34e5f679f5bbb6538be" target='_blank'>
              Micropapillary structure: A natural tumor collective invasion model with enhanced stem-like properties.
              </a>
            </td>
          <td>
            Sisi Li, Shuangshu Gao, Ling Qin, Caixia Ding, Jinghui Qu, Yifei Cui, Lixia Qiang, Shengjie Yin, Xiaoyu Zheng, Hongxue Meng
          </td>
          <td>2024-11-20</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Introduction: Survival rates in advanced head and neck cancer (HNC) remain poor with no clinically approved biomarkers to predict treatment efficacy. HNC is a highly heterogenous disease and the lack of available models that capture patient heterogeneity and the complexity of the tumor microenvironment (TME) contributes to these poor patient outcomes. Microphysiological systems (MPS) are complex multicellular in vitro models that can recapitulate three-dimensional organ structure and function. The low cell number requirements for MPS permits the creation of patient-specific TME models, where all the cell types can be isolated from a single piece of tumor tissue. These patient-specific MPS represent a promising strategy for elucidating the contribution of the TME to treatment resistance, predicting the best treatment for each patient and improving patient outcomes in HNC. Here, we report a patient-specific MPS that contains multiple TME compartments and recapitulates patient heterogeneity. Methods: Tumor tissue was collected from patients with oral cavity carcinoma undergoing standard of care surgery followed by radiation or chemoradiation. Tissue from each patient was digested and tumor infiltrating leukocytes (TIL) were isolated using CD45 magnetic beads, while epithelial cells and fibroblasts were expanded in culture using specific media formulations. Tissue was also snap frozen for single cell RNA sequencing analysis. To build the MPS, a custom designed microfluidic device was filled with a hydrogel matrix containing patient-matched TIL, fibroblasts and epithelial spheroids to mimic the solid tumor. Lumens molded from the hydrogel were seeded with blood or lymphatic endothelial cells to create vasculature. Multiple replicate MPS can be built from each patient, which are cultured in an optimized media formulation to support the multiple primary cell types. Patient-specific MPS were treated with radiation or cisplatin + radiation to mimic the treatment the patient received. Multiple orthogonal endpoints were measured including tumor cell migration and viability, cytokine secretion and angiogenesis. Single cell sequencing was also performed on the original tumor tissue and matched MPS. Results: We have successfully created patient-specific models of the HNC TME. Single cell RNA sequencing analysis of the MPS suggests that cellular heterogeneity is being retained. Patient characteristics varied between MPS with some patient TME MPS showing highly migratory tumor cells or angiogenic responses. Treatment responses were measured for each patient-specific MPS and ongoing work will characterize the contributions of the TME to treatment resistance. These data demonstrate the feasibility of using patient-specific MPS to investigate the contribution of the TME to treatment response and resistance and lays important groundwork for using MPS to identify patient treatment responses.
 Citation Format: Adeel Ahmed, Marcos Lares, Fauzan Ahmed, Adam R. Burr, Paul M. Harari, David J. Beebe, Sheena C. Kerr. Engineering the tumor microenvironment: Patient-specific microphysiological systems of the head and neck cancer tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5867639a61561e0c0780d2ce0be172eff5b37091" target='_blank'>
              Abstract B034: Engineering the tumor microenvironment: Patient-specific microphysiological systems of the head and neck cancer tumor microenvironment
              </a>
            </td>
          <td>
            Adeel Ahmed, Marcos Lares, Fauzan Ahmed, Adam R. Burr, Paul M. Harari, David J. Beebe, S. Kerr
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Background Metastasis is the major cause of cancer‐related mortality. The premetastatic niche is a promising target for its prevention. However, the generality and cellular dynamics in premetastatic niche formation have remained unclear. Aims This study aimed to elucidate the generality and cellular dynamics in premetastatic niche formation. Materials and Methods We performed comprehensive flow cytometric analysis of lung and peripheral immune cells at three time points (early premetastatic, late premetastatic, and micrometastatic phases) for mice with subcutaneous implants of three types of cancer cells (breast cancer, lung cancer, or melanoma cells). The immuno‐cell profiles were then used to predict the metastatic phase by machine learning. Results We found a common pattern of changes in both lung and peripheral immune cell profiles across the three cancer types, including a decrease in the proportion of eosinophils in the early premetastatic phase, an increase in that of regulatory T cells in the late premetastatic phase, and an increase in that of polymorphonuclear myeloid‐derived suppressor cells and a decrease in that of B cells in the micrometastatic phase. Machine learning using immune cell profiles could predict the metastatic phase with approximately 75% accuracy. Discussion Validation of our findings in humans will require data on the presence or absence of micrometastases in patients and the accumulation of comprehensive and temporal information on immune cells. In addition, blood proteins, extracellular vesicles, DNA, RNA, or metabolites may be useful for more accurate prediction. Conclusion The discovery of generalities in premetastatic niche formation allow prediction of metastatic phase and provide a basis for the development of methods for early detection and prevention of cancer metastasis in a cancer type‐independent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f02612a5b53055050e50dcce6e173f615e1f3275" target='_blank'>
              Immune Cell Profiling Reveals a Common Pattern in Premetastatic Niche Formation Across Various Cancer Types
              </a>
            </td>
          <td>
            Shigeaki Sugiyama, Kanae Yumimoto, Keiichi I Nakayama
          </td>
          <td>2024-12-30</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life. Methods Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity. Results A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis. Conclusion By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d4c9ead4b1e7f5435e924e22c66461019ad235" target='_blank'>
              Targeting of TAMs: can we be more clever than cancer cells?
              </a>
            </td>
          <td>
            J. Kzhyshkowska, Jiaxin Shen, I. Larionova
          </td>
          <td>2024-11-08</td>
          <td>Cellular and Molecular Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) represent the main immune cell population in various brain malignancies. To elucidate their biological impact in the tumor microenvironment (TME) of meningiomas (MGMs), we assessed TAM numbers, activation state, malignancy- and survival-associated changes, as well as their association with tumor-infiltrating T lymphocytes (TILs). Methods TAM infiltration was analyzed in a multicenter cohort of 195 clinically well-annotated cases (follow-up >5 years, n=120 newly-diagnosed and n=75 recurrent MGMs) enriched for higher-grade MGMs. TAMs and M2-TAMs were quantified by tissue cytometry on whole-tumor sections. Further, we assessed levels of 27 cyto- and chemokines in a subset of tissues (n=46 cases), and re-analyzed our previously published T cell infiltration (n=94 cases) and expanded microarray (n=97 cases) datasets. Results Newly-diagnosed MGMs showed a substantial but highly heterogeneous TAM infiltration that was four times higher than for TILs. Anti-inflammatory M2-TAMs were increased in higher WHO grade tumors and in recurrent MGMs. Importantly, high M2-TAM infiltration was associated with poor progression-free survival independent of other prognostic confounders and even mitigated the beneficial prognostic effect of TIL infiltration. Additional cytokine, gene expression and pathway analyses corroborated the presence of an immunosuppressive niche in M2-TAM-enriched MGMs. Conclusions Altogether, higher numbers of TAMs and M2-TAMs appear to be a hallmark of clinically aggressive behavior in newly-diagnosed and recurrent MGMs. Unlike TILs, immunosuppressive TAMs seem to play a dominant negative role in the immunological landscape of MGMs, highlighting M2-TAMs to be an attractive treatment target for immunotherapeutic approaches. Translational Relevance of the Study Meningiomas (MGMs) are typically regarded as benign neoplasms, however there is a substantial proportion of clinically aggressive tumors that are refractory to standard treatment modalities and demand for the development of novel therapeutic approaches such as immunotherapy. This is the first comprehensive study reporting malignancy- and progression-associated changes of tumor-associated macrophages (TAMs), their polarization state, their association with tumor-infiltrating T lymphocytes (TILs), and their impact on patient survival in a large multicenter cohort of 195 tumors containing substantial numbers of clinically aggressive cases. Notably, we identified higher numbers of immunosuppressive M2-TAMs as an independent prognostic factor for poor survival, overriding the beneficial prognostic effects of TILs. Thus, our data highlight an important role of immunosuppressive M2-TAMs on tumor malignancy and progression, and further suggest targeting macrophages as a treatment strategy to improve the success of immunotherapeutic approaches in MGMs. Key points Meningiomas are highly infiltrated by immunosuppressive M2-TAMs. High M2-TAM numbers are an independent negative prognostic factor for patient survival. High TAM infiltration mitigates the beneficial prognostic impact of TILs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797056ddd8b0fcec8ccf003c922ab3488ed62e98" target='_blank'>
              Tumor-Associated Macrophages in Meningiomas: An Independent Prognostic Factor for Poor Survival Outperforming the Benefits of T cells
              </a>
            </td>
          <td>
            C. Lotsch, Fang Liu, R. Warta, G. Jungwirth, Carmen Rommel, Mandy Barthel, K. Lamszus, A. Kessler, Niels Grabe, Mario Loehr, R. Ketter, Christian Senft, Manfred Westphal, F. Sahm, Sandro Krieg, A. Unterberg, Matthias Simon, Andreas von Deimling, C. Herold-Mende
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55719070e9f1cff38fbdc568fedbec8bc7d69e98" target='_blank'>
              Single-cell transcriptomic and spatial analysis reveal the immunosuppressive microenvironment in relapsed/refractory angioimmunoblastic T-cell lymphoma
              </a>
            </td>
          <td>
            Mengyan Zhu, Ning Li, Lei Fan, Rongrong Wu, Lei Cao, Yimin Ren, Chuanyang Lu, Lishen Zhang, Yun Cai, Yuzhu Shi, Zihan Lin, Xueying Lu, Jiayan Leng, Shiyang Zhong, Xingfei Hu, Bin Huang, Runheng Huang, Wanting Zhou, Diru Yao, Lingxiang Wu, Wei Wu, Quanzhong Liu, Peng Xia, Ruize Chen, W. Shi, Ruohan Zhang, Sali Lv, Chunling Wang, Liang Yu, Jianyong Li, Qianghu Wang, Kening Li, H. Jin
          </td>
          <td>2024-12-01</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/456d5e0f44d3a56861aef434239933c23d1853a0" target='_blank'>
              Single-cell RNA sequencing analysis reveals the dynamic changes in the tumor microenvironment during NMIBC recurrence.
              </a>
            </td>
          <td>
            Ziang Chen, Tianxiang Zhang, Weijian Li, Jiawei Hu, Y. Ou, Fangdie Ye, Jinhao Zhang, Haowen Jiang, Shenghua Liu
          </td>
          <td>2024-12-04</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ef6a160eddfbbad312082d26af8e03b167d27f" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance.
              </a>
            </td>
          <td>
            Michael J Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah R. Amend, James B. Hicks
          </td>
          <td>2024-11-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2461a2661414e1e6ff8b37117c56d2cfd9c5c7b" target='_blank'>
              Cell hiding in colorectal cancer: correlation with response to chemotherapy in vitro and in vivo
              </a>
            </td>
          <td>
            I. Druzhkova, A. Potapov, N. Ignatova, M. Bugrova, I. Shchechkin, M. Lukina, L. Shimolina, E. Kolesnikova, M. Shirmanova, E. Zagaynova
          </td>
          <td>2024-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer, a prominent global health concern, involves the uncontrolled growth and dissemination of abnormal cells, forming tumors that can infiltrate nearby tissues and metastasize to distant sites. Early and precise tumor detection and treatments are crucial to impede their spread and mitigate disease progression. This study presents the development of a scoring system that harnesses single-cell RNA sequencing (scRNA-seq) data to predict the suitability of solid tumors, specifically glioblastoma (GBM) and lymphoma, for advanced cell therapies such as CAR-T, NK cell, and TIL therapies. By integrating key cellular factorsincluding tumor-infiltrating cells, immunosuppressive cells, and immune escape markersthis system provides a quantitative assessment of a tumors potential responsiveness to these therapies. Validation with patient-derived samples has demonstrated the system's potential to improve precision in therapy selection, offering a promising tool for personalized oncology. This research not only aims to enhance the effectiveness of cell therapies for cancer patients but also lays the groundwork for applying similar methodologies to other cancer types, advancing the integration of precision medicine in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa40086c8127d386e515ff58d140655d2ef491d" target='_blank'>
              A Scoring System that Leverages Single Cell RNA-Sequencing Data to Predict Cell Therapy Outcome in Solid Tumor Treatment
              </a>
            </td>
          <td>
            Yuwen Li
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Background Human glioblastoma multiforme (GBM) is a highly aggressive tumor with insufficient therapies available. Especially, novel concepts of immune therapies fail due to a complex immunosuppressive microenvironment, high mutational rates, and inter-patient variations. The intratumoral heterogeneity is currently not sufficiently investigated. Methods Biopsies from six different locations were taken in a cohort of 16 GBM patients who underwent surgery. The tissue slides were analyzed utilizing high-content imaging microscopy and algorithm-based image quantification. Several immune markers for macrophage and microglia subpopulations were investigated. Flow cytometry was used to validate key results. Besides the surface marker, cytokines were measured and categorized based on their heterogenicity and overall expression. Results M2-like antigens, including CD204, CD163, Arg1, and CSF1R, showed comparatively higher expression, with GFAP displaying the least intratumoral heterogeneity. In contrast, anti-tumor-macrophage-like antigens, such as PSGL-1, CD16, CD68, and MHC-II, exhibited low overall expression and concurrent high intratumoral heterogeneity. CD16 and PSGL-1 were the most heterogeneous antigens. High expression levels were observed for cytokines IL-6, VEGF, and CCL-2. VILIP-a was revealed to differentiate most in principle component analysis. Cytokines with the lowest overall expression, such as TGF-β1, β-NGF, TNF-α, and TREM1, showed low intratumoral heterogeneity, in contrast to βNGF, TNF-α, and IL-18, which displayed high heterogeneity despite low expression. Conclusion The study showed high intratumoral heterogeneity in GBM, emphasizing the need for a more detailed understanding of the tumor microenvironment. The described findings could be essential for future personalized treatment strategies and the implementation of reliable diagnostics in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f548e838e4c2baead447a4f25f35e61c6a9cee9" target='_blank'>
              Markers of tumor-associated macrophages and microglia exhibit high intratumoral heterogeneity in human glioblastoma tissue
              </a>
            </td>
          <td>
            Mikael Ispirjan, Sascha Marx, Eric Freund, Steffen K. Fleck, J. Baldauf, Karl Roessler, Henry W. S. Schroeder, S. Bekeschus
          </td>
          <td>2024-11-10</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell mass spectrometry (SCMS) is an emerging tool for studying cell heterogeneity according to variation of molecular species in single cells. Although it has become increasingly common to employ machine learning models in SCMS data analysis, such as the classification of cell phenotypes, the existing machine learning models often suffer from low adaptability and transferability. In addition, SCMS studies of rare cells can be restricted by limited number of cell samples. To overcome these limitations, we performed SCMS analyses of melanoma cancer cell lines with two phenotypes (i.e., primary and metastatic cells). We then developed a meta-learning-based model, MetaPhenotype, that can be trained using a small amount of SCMS data to accurately classify cells into primary or metastatic phenotypes. Our results show that compared with standard transfer learning models, MetaPhenotype can rapidly predict and achieve a high accuracy of over 90% with fewer new training samples. Overall, our work opens the possibility of accurate cell phenotype classification based on fewer SCMS samples, thus lowering the demand for sample acquisition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e084118ab6942b4246273acb060f9d4fd94e6e5a" target='_blank'>
              MetaPhenotype: A Transferable Meta-Learning Model for Single-Cell Mass Spectrometry-Based Cell Phenotype Prediction Using Limited Number of Cells.
              </a>
            </td>
          <td>
            Songyuan Yao, Tra D Nguyen, Yunpeng Lan, Wen Yang, Dan Chen, Yihan Shao, Zhibo Yang
          </td>
          <td>2024-11-21</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8645f518787a0ca20fba12896526944d30b5806b" target='_blank'>
              Development of patient-derived lymphomoids with preserved tumor architecture for lymphoma therapy screening
              </a>
            </td>
          <td>
            A. Santamaria-Martínez, J. Epiney, Divyanshu Srivastava, Daniele Tavernari, Marco Varrone, Dina Milowich, Igor Letovanec, Thorsten Krueger, Rafael Duran, Giovanni Ciriello, A. Cairoli, E. Oricchio
          </td>
          <td>2024-12-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a complex clonal disorder characterized by clinical, genetic, metabolomic, and epigenetic heterogeneity resulting in the uncontrolled proliferation of aberrant blood-forming precursor cells. Despite advancements in the understanding of the genetic, metabolic, and epigenetic landscape of AML, it remains a significant therapeutic challenge. Functional profiling techniques, such as BH3 profiling (BP), gene expression profiling (GEP), proteomics, metabolomics, drug sensitivity/resistance testing (DSRT), CRISPR/Cas9, and RNAi screens offer valuable insights into the functional behavior of leukemia cells. BP evaluates the mitochondrial response to pro-apoptotic BH3 peptides, determining a cell’s apoptotic threshold and its reliance on specific anti-apoptotic proteins. This knowledge can pinpoint vulnerabilities in the mitochondria-mediated apoptotic pathway in leukemia cells, potentially informing treatment strategies and predicting therapeutic responses. GEP, particularly RNA sequencing, evaluates the transcriptomic landscape and identifies gene expression alterations specific to AML subtypes. Proteomics and metabolomics, utilizing mass spectrometry and nuclear magnetic resonance (NMR), provide a detailed view of the active proteins and metabolic pathways in leukemia cells. DSRT involves exposing leukemia cells to a panel of chemotherapeutic and targeted agents to assess their sensitivity or resistance profiles and potentially guide personalized treatment strategies. CRISPR/Cas9 and RNAi screens enable systematic disruption of genes to ascertain their roles in leukemia cell survival and proliferation. These techniques facilitate precise disease subtyping, uncover novel biomarkers and therapeutic targets, and provide a deeper understanding of drug-resistance mechanisms. Recent studies utilizing functional profiling have identified specific mutations and gene signatures associated with aggressive AML subtypes, aberrant signaling pathways, and potential opportunities for drug repurposing. The integration of multi-omics approaches, advances in single-cell sequencing, and artificial intelligence is expected to refine the precision of functional profiling and ultimately improve patient outcomes in AML. This review highlights the diverse landscape of functional profiling methods and emphasizes their respective advantages and limitations. It highlights select successes in how these methods have further advanced our understanding of AML biology, identifies druggable targets that have improved outcomes, delineates challenges associated with these techniques, and provides a prospective view of the future where these techniques are likely to be increasingly incorporated into the routine care of patients with AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d60f51fb21da6cf836930cd187d2acd3d680d13" target='_blank'>
              Decoding Acute Myeloid Leukemia: A Clinician’s Guide to Functional Profiling
              </a>
            </td>
          <td>
            Prasad Iyer, S. Jasdanwala, Yuhan Wang, K. Bhatia, Shruti Bhatt
          </td>
          <td>2024-11-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy characterized by the clonal expansion of myeloid precursor cells. Advances in genomic profiling have enhanced our understanding of AML pathogenesis, leading to the identification of recurrent mutations, including TP53, FLT3, MUC4, RAS, and IDH1/2. These mutations significantly influence treatment response and prognosis, with TP53 mutations conferring poor outcomes and resistance to conventional therapies. Despite the advent of venetoclax-based regimens, resistance mechanisms persist, necessitating the development of novel therapeutic strategies. This study aims to investigate the efficacy of drug combinations for the treatment of AML using both in vitro AML cell lines and an in vivo zebrafish embryo xenograft model. Specifically, we focus on two drug combinations; the pan-RAF inhibitor LY3009120 combined with the mTOR inhibitor sapanisertib (designated as LS), and the JAK1/2 inhibitor ruxolitinib combined with the ERK inhibitor ulixertinib (designated as RU). The study integrates real-time cell viability assays, xenograft imaging, and genomic analyses to assess drug efficacy and explore correlations between treatment responses and mutational profiles, particularly TP53, FLT3, and MUC4 mutations. Both combinations, LS and RU, demonstrated superior efficacy compared to venetoclax-based treatments in reducing cell viability across AML cell lines. The LS combination showed significant reductions in cell viability in MOLM16 and SKM cells, while RU exhibited comparable efficacy with lower toxicity profiles. In zebrafish embryos, the LS combination effectively inhibited the proliferation of xenografted human AML cells, as evidenced by decreased fluorescence signals, indicating cell death. The RU combination also disrupted survival signaling pathways, showing promise as a therapeutic strategy. Furthermore, a correlation was identified between drug response and mutational profiles, with TP53, FLT3 and MUC4 mutations significantly influencing sensitivity to the LS and RU combinations. These findings support the further development of LS and RU as effective alternatives to current clinical regimens, with implications for personalized AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e928f03f37069a717618c39ee33259e9551d5324" target='_blank'>
              Targeting AML Resistance with LY3009120-Sapanisertib and Ruxolitinib-Ulixertinib Combinations Demonstrate Superior Efficacy in FLT3, TP53, and MUC4 mutations
              </a>
            </td>
          <td>
            Elham Gholizadeh, Ehsan Zangene, A. Parsons, M. Kontro, Caroline A. Heckman, Mohieddin Jafari
          </td>
          <td>2024-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer spheroids are spherical, three-dimensional (3D), in vitro assemblies of cancer cells, which are gaining importance as a useful model in cancer behavior studies. Designed to simulate key features of the in vivo tumor microenvironment, spheroids offer reliable insights for drug screening and testing applications. We observed contrasting phenotypes in 3D cervical cancer (CC) cultures. Thus, in this study, we compared the proteomes of 3D and traditional two-dimensional (2D) cultures of CC cell lines, HeLa, SiHa, and C33A. When cultured in in-house poly-(2-hydroxyethyl methacrylate)-coated plates under conditions suitable for 3D spheroid formation, these CC cell lines yielded spheroids exhibiting different features. Proteomic analysis of cells cultured in 2D and 3D cultures revealed similar protein profiles but remarkable differences in the expression levels of some proteins. In SiHa and C33A cells, the upregulation of key proteins required for spheroid formation was insufficient for the formation of compact spheroids. In contrast, HeLa cells could form compact spheroids because they upregulated the proteins, including cadherin-binding, cytoskeleton, and adhesion proteins, necessary for spheroid formation during the remodeling process. Overall, this study unravels the mechanisms underlying the formation of spheroids in the commonly used CC cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/815cc2d468c5e900a469f3a75fe62c8fb5f3a378" target='_blank'>
              Proteomic Analysis Reveals Cadherin, Actin, and Focal Adhesion Molecule-Mediated Formation of Cervical Cancer Spheroids
              </a>
            </td>
          <td>
            P. Molika, K. Leetanaporn, W. Chiangjong, Pongsakorn Choochuen, R. Navakanitworakul
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da415d9954dd04158cfcfaa83dc1880ca38c0523" target='_blank'>
              Single-cell and spatial transcriptomics reveal SPP1-CD44 signaling drives primary resistance to immune checkpoint inhibitors in RCC
              </a>
            </td>
          <td>
            Junfeng Zhang, Qingyan Peng, Jin Fan, Fuzhong Liu, Hongbo Chen, X. Bi, Shuai Yuan, Wei Jiang, Ting Pan, Kailing Li, Sihai Tan, Peng Chen
          </td>
          <td>2024-12-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The present review provides a comprehensive overview of the current state of in vitro cancer studies, focusing on recent advancements and ongoing in cell culture models and analyses techniques. Cancer cells grow in a complex and dynamic environment, interacting with various cellular components, such as stromal cells, cancer‐associated fibroblasts, immune cells, and the extracellular matrix (ECM). The ECM provides structural support and unique characteristics essential for tumorigenesis. Accurately modeling this intricate tumor microenvironment and precisely analyzing cell–cell and cell–ECM interactions are crucial for understanding cancer progression and therapeutic responses. Consequently, oncology research is advancing toward a) three‐dimensional tumor models, b) single‐cell level analyses, and c) live‐cell analyses. This review aims to elucidate current knowledge in this field, emphasizing the benefits these innovative approaches offer over traditional two‐dimensional models, bulk analyses, and endpoint measurements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a44db0896c6a9b4ac78194bb1c17e5d4557f6db" target='_blank'>
              Advancements in Cancer Research: 3D Models, Single‐Cell, and Live‐Cell Techniques for Better Insights
              </a>
            </td>
          <td>
            Federica Carnevali, S. Forciniti, V. Onesto, Anna Chiara Siciliano, Helena Iuele, Giuliana Grasso, A. F. da Cruz, Ilaria Serra, N. Depalma, Stefano D'Ugo, P. Piscitelli, M. Spampinato, Giuseppe Gigli, Carolina de Camargo de Oliveira, L. L. del Mercato
          </td>
          <td>2024-11-09</td>
          <td>Advanced Therapeutics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c214c85284c0f7c6d4f85e6f9096b58eba34d42" target='_blank'>
              Dissecting tumor transcriptional heterogeneity from single-cell RNA-seq data by generalized binary covariance decomposition.
              </a>
            </td>
          <td>
            Yusha Liu, P. Carbonetto, Jason Willwerscheid, Scott A. Oakes, Kay F Macleod, Matthew Stephens
          </td>
          <td>2025-01-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) is now the first and second leading cause of cancer-related deaths in men and women under 50, respectively, upending prior predictions. While many risk factors have been identified, the etiology of early onset disease is poorly understood, and it is unclear why some benign polyps progress to CRC while most do not. The genetic drivers of colon tumor initiation and progression to malignancy are not sufficient to drive disease on their own, as evidenced by the near ubiquitous presence of oncogenic mutations within morphologically “normal” adult tissues like the colon. This raises several fundamental questions: 1) what are the pioneering molecular and cellular events underlying benign-to-malignant transition following oncogenic mutation? 2) Is this sequence of events predetermined by cancer genetics or are additional (micro)environmental stimuli necessary? 3) How does the benign-to-malignant transition reshape tumor clonal architecture and is this immunogenic? We have begun to address these questions using a combination of rare human colon polyps with evidence of early malignant transition and innovative new mouse models that enable control of stepwise colon cancer progression. These autochthonous models combine in vivo CRISPR-Cas9 and an inducible “split-Cre” system to decouple focal tumor initiation in the distal colon (via Apc knockout) from spatiotemporal control of secondary driver events of malignant progression (e.g., Kras G12D, Trp53 knockout) in rare cells of benign adenomas. We have validated the in vivo functionality of these models, generating early “intramucosal carcinoma” and “adenocarcinoma-in-adenoma”—physiologically relevant and elusive stages of human colon cancer development that have never been reliably captured in model systems to date. Comparing these tumors with spatial transcriptomic data from human specimens, we report distinct transcriptional programs of early progression, a unique stromal reaction associated with early progression, and restructuring of glandular architecture and the stem cell niche in early carcinoma. By delineating the earliest events underlying cancer progression, this work promises to inform new strategies for early detection and prevention. We anticipate that the models will also be particularly valuable in functionally interrogating host and environmental factors that impact risk of early onset cancer.
 Citation Format: Peter M.K. Westcott, Yihan Qin, Daniel Zhang, Nikita Persaud, Zakeria Aminzada, Nischal Bhandari, Colin McLaughlin, William Rideout III, Santiago Naranjo, Song Han, Rodrigo Romero, Claire Regan, Jonathan Preall, Semir Beyaz, Tyler Jacks, Zhen Zhao, Sepideh Gholami. Early changes to the colon tumor microenvironment during benign-to- malignant transition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr PR016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/accd8252a49f53325d258371518d1ee2669b74af" target='_blank'>
              Abstract PR016: Early changes to the colon tumor microenvironment during benign-to- malignant transition
              </a>
            </td>
          <td>
            P. Westcott, Yihan Qin, Daniel Zhang, Nikita Persaud, Zakeria Aminzada, Nischal Bhandari, Colin McLaughlin, William Rideout, S. Naranjo, Song Han, Rodrigo Romero, C. Regan, Jonathan Preall, Semir Beyaz, Tyler Jacks, Zhen Zhao, Sepideh Gholami
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The inter- and intra-tumor heterogeneity of triple negative breast cancers (TNBC), which is reflected in diverse drug responses, interplays with tumor evolution. Here, we developed a preclinical experimental and analytical framework using treatment-naive TNBC patient-derived tumor xenografts (PDTX) to test their predictive value in personalized cancer treatment approaches. Patients and their matched PDTX exhibited concordant drug responses to neoadjuvant therapy using two trial designs and dosing schedules. This platform enabled analysis of non-genetic mechanisms involved in relapse dynamics. Treatment resulted in permanent phenotypic changes with functional and therapeutic consequences. High throughput drug screening methods in ex vivo patient derived tumor xenograft cells (PDTCs) revealed patient-specific drug response changes dependent on first-line therapy. This was validated in vivo, as exemplified by a change in olaparib sensitivity in tumors previously treated with clinically relevant cycles of standard-of-care chemotherapy. In summary, PDTXs provide a robust tool to test patient drug responses and therapeutic regimens and to model evolutionary trajectories. However, high inter-model variability and permanent non-genomic transcriptional changes constrain their use for personalized cancer therapy. This work highlights important considerations associated with preclinical drug response modeling and potential uses of the platform to identify efficacious and preferential sequential therapeutic regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8c0bbe2cdb3aa66895b4352d80e8131c8d6526" target='_blank'>
              Modeling Drug Responses and Evolutionary Dynamics using Patient-Derived Xenografts Reveals Precision Medicine Strategies for Triple Negative Breast Cancer.
              </a>
            </td>
          <td>
            A. Shea, Y. Eyal-Lubling, Daniel Guerrero-Romero, Raquel Manzano Garcia, W. Greenwood, Martin O’Reilly, D. Georgopoulou, Maurizio Callari, G. Lerda, Sophia Wix, A. Giovannetti, Riccardo Masina, E. Esmaeilishirazifard, Wei Cope, Alistair G. Martin, Ai Nagano, Lisa Young, Steven Kupczak, Yi Cheng, H. Bardwell, E. Provenzano, J. Kane, Jonathan Lay, L. Grybowicz, K. McAdam, Carlos Caldas, Jean Abraham, O. Rueda, Alejandra Bruna
          </td>
          <td>2024-11-08</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background: Targeted therapies (e.g., ibrutinib) have markedly improved chronic lymphocytic leukemia (CLL) management; however, ~20% of patients experience disease relapse, suggesting the inadequate depth and durability of these front-line strategies. Moreover, immunotherapeutic success in CLL has been stifled by its pro-tumor microenvironment milieu and low mutational burden, cultivating poor antigenicity and limited ability to generate anti-tumor immunity through adaptive immune cell engagement. Previously, we have demonstrated how a three-carbon-linker spirocyclic dimer (SpiD3) promotes futile activation of the unfolded protein response (UPR) in CLL cells through immense misfolded-protein mimicry, culminating in insurmountable ER stress and programmed CLL cell death. Method: Herein, we used flow cytometry and cell-based assays to capture the kinetics and magnitude of SpiD3-induced damage-associated molecular patterns (DAMPs) in CLL cell lines and primary samples. Result: SpiD3 treatment, in vitro and in vivo, demonstrated the capacity to propagate immunogenic cell death through emissions of classically immunogenic DAMPs (CALR, ATP, HMGB1) and establish a chemotactic gradient for bone marrow-derived dendritic cells. Conclusions: Thus, this study supports future investigation into the relationship between novel therapeutics, manners of cancer cell death, and their contributions to adaptive immune cell engagement as a means for improving anti-cancer therapy in CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6bb6049f43b4446b0fae7febaccea77debb8ea" target='_blank'>
              Immunogenic Cell Death Traits Emitted from Chronic Lymphocytic Leukemia Cells Following Treatment with a Novel Anti-Cancer Agent, SpiD3
              </a>
            </td>
          <td>
            Elizabeth Schmitz, Abigail Ridout, Audrey L. Smith, Alexandria P. Eiken, Sydney A. Skupa, Erin M. Drengler, Sarbjit Singh, Sandeep Rana, A. Natarajan, Dalia El-Gamal
          </td>
          <td>2024-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="NPM1-mutated AML is one of the largest entities in international classification systems of myeloid neoplasms, which are based on integrating morphologic and clinical data with genomic data. Previous research, however, indicates that bulk transcriptomics-based subtyping may improve prognostication and therapy guidance. Here, we characterized the heterogeneity in NPM1-mutated AML by performing single-cell RNA-sequencing and spectral flow cytometry on 16 AML belonging to three distinct subtypes previously identified by bulk transcriptomics. Using single-cell expression profiling we generated a comprehensive atlas of NPM1-mutated AML, collectively reconstituting complete myelopoiesis. The three NPM1-mutated transcriptional subtypes showed consistent differences in the proportions of myeloid cell clusters with distinct patterns in lineage commitment and maturational arrest. In all samples, malignant cells were detected across different myeloid cell clusters, indicating that NPM1-mutated AML are heavily skewed but not fully arrested in myelopoiesis. Same-sample multi-color spectral flow cytometry recapitulated these skewing patterns, indicating that the three NPM1-mutated subtypes can be consistently identified across platforms. Moreover, our analyses highlighted differences in the abundance of rare hematopoietic stem cells suggesting that skewing occurs early in myelopoiesis. To conclude, by harnessing single-cell RNA-sequencing and spectral flow cytometry, we provide a detailed description of three distinct and reproducible patterns in lineage skewing in NPM1-mutated AML that may have potential relevance for prognosis and treatment of patients with NPM1-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d2c7d3f716516ca564f7acecb6aa3ff7231897" target='_blank'>
              Resolving inter- and intra-patient heterogeneity in NPM1-mutated AML at single-cell resolution
              </a>
            </td>
          <td>
            E. O. Karakaslar, E. Argiro, N. E. Struckman, R. H. Shirali, J. Severens, W. Honders, S. L. Kloet, H. Veelken, M. Reinders, M. Griffioen, E. B. van den Akker
          </td>
          <td>2024-12-11</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spatial transcriptomics, proteomics, and epigenomics are innovative technologies which offer an unparalleled resolution and wealth of data in understanding and the interpretation of cellular functions and interactions. These techniques allow researchers to investigate gene and protein expressions at an individual cell level, revealing cellular heterogeneity within, for instance, bioengineered tissues and classifying novel and rare cell populations that could be essential for the function of the tissues and in disease processes. It is possible to analyze thousands of cells simultaneously, which gives thorough insights into the transcriptomic view of complex tissues. Spatial transcriptomics combines gene expressions with spatial information, conserving tissue architecture and making the mapping of gene activity across different tissue regions possible. Despite recent advancements in these technologies, they face certain limitations. Single-cell transcriptomics can suffer from technical noise and dropout events, leading to incomplete data. Its applicability has been limited by the complexity of data integration and interpretation, although better resolution and tools for the interpretation of data are developing fast. Spatial proteomics and spatial epigenomics provide data on the distribution of proteins and the gene regulatory aspects in tissues, respectively. The disadvantages of these approaches include rather costly and time-consuming analyses. Nevertheless, combining these techniques promises a more comprehensive understanding of cell function and tissue organization, which can be predicted to be useful in achieving better knowledge of cell guidance in tissue-engineered constructs and a higher quality of tissue technology products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b910cb889d8cf7dd3b82978f6e3916ced3b58d85" target='_blank'>
              Spatial Transcriptomics, Proteomics, and Epigenomics as Tools in Tissue Engineering and Regenerative Medicine
              </a>
            </td>
          <td>
            M. Lammi, C. Qu
          </td>
          <td>2024-12-01</td>
          <td>Bioengineering</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Genetic modifications using CRISPR-Cas9 have revolutionized cancer research and other pre-clinical studies. Exceptionally, these efficient tools are inadequate in a few disease models and cell lines due to the aberrant differentiation states and the accumulation of excessive somatic mutations that compromise the robustness of viral gene delivery and stable transduction. A couple of B lymphoma cell lines fall into this category where lentiviral transfection becomes inefficient and exhibits variable efficiency. Additionally, lentiviral delivery requires high biosafety levels. To address this challenge, we have developed a two-step strategy that supports CRISPR-Cas9 through lentivirus and murine ecotropic γ-retrovirus. By engineering B lymphoma cell lines to express Cas9 and mCat-1, a specific receptor for ecotropic retroviruses, we enable efficient and safe gene editing through ecotropic γ-retrovirus. We demonstrate the efficacy of this method by generating IgM-deficient B lymphoma cell lines. This innovative approach simplifies protocols, enhances accessibility, and paves the way for standardized gene manipulation of B cell lymphoma models for molecular cell biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a2c45cfa5f916fccda93d3f9c47decafd7f279" target='_blank'>
              Facilitating Gene Editing in Human Lymphoma Cells Using Murine Ecotropic γ-Retroviruses
              </a>
            </td>
          <td>
            Manish Kumar, Eva Gentner-Göbel, Palash Chandra Maity
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Stromal cells are non‐hematopoietic cells that consist of endothelial cells and various mesenchymal cell populations. The composition of the stromal cell compartment is diverse in different organs. Numerous recent studies demonstrated that the lung environment contains heterogeneous mesenchymal stromal cell populations with distinctive genomic signatures and location preferences. Besides their role in supporting organ structure and remodeling tissue, mesenchymal stromal cells fulfill critical immune functions. These stromal cells show alterations during lung fibrosis and infectious disorders like COVID‐19 or flu infection. To date, their identification and isolation were challenging, and most information about their heterogeneity was derived from scRNAseq data. In this protocol, we describe an isolation, comprehensive flow cytometry assessment, and purification strategy for murine lung stromal cells. The described method is optimized for minimizing cell death while keeping a high level of cell purity. This protocol can be also used for ex‐vivo analysis of these cells in downstream functional assays. © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Isolation of stromal cells from murine lung tissue Basic Protocol 2: Flow cytometry assessment of lung stromal populations Basic Protocol 3: Purification of lung fibroblastic stromal cells Alternate Protocol: Positive selection of fibroblastic stromal cells">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768210ba3f920bfb05c8ef8764e7bd739b49227" target='_blank'>
              Isolation, Purification, and Comprehensive Flow Cytometry Assessment of Lung Stromal Cells
              </a>
            </td>
          <td>
            Sophia Rottmann, Veronika Lukacs-Kornek
          </td>
          <td>2024-12-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4422b0759e87ed388f1ba9e5bcbfed55cb314237" target='_blank'>
              Spatial transcriptomics reveals strong association between SFRP4 and extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            M. K. Andersen, S. Krossa, Elise Midtbust, Christine A. Pedersen, M. Wess, T. S. Høiem, T. Viset, Øystein Størkersen, Ingunn Nervik, E. Sandsmark, H. Bertilsson, G. Giskeødegård, M. B. Rye, May-Britt Tessem
          </td>
          <td>2024-11-08</td>
          <td>Communications Biology</td>
          <td>4</td>
          <td>26</td>
        </tr>

        <tr id="ABSTRACT The tumor microbiome, a complex community of microbes found in tumors, has been found to be linked to cancer development, progression, and treatment outcome. However, it remains a bottleneck in distangling the relationship between the tumor microbiome and host gene expressions in tumor microenvironment, as well as their concert effects on patient survival. In this study, we aimed to decode this complex relationship by developing ASD-cancer (autoencoder-based subtypes detector for cancer), a semi-supervised deep learning framework that could extract survival-related features from tumor microbiome and transcriptome data, and identify patients’ survival subtypes. By using tissue samples from The Cancer Genome Atlas database, we identified two statistically distinct survival subtypes across all 20 types of cancer Our framework provided improved risk stratification (e.g., for liver hepatocellular carcinoma, [LIHC], log-rank test, P = 8.12E−6) compared to PCA (e.g., for LIHC, log-rank test, P = 0.87), predicted survival subtypes accurately, and identified biomarkers for survival subtypes. Additionally, we identified potential interactions between microbes and host genes that may play roles in survival. For instance, in LIHC, Arcobacter, Methylocella, and Isoptericola may regulate host survival through interactions with host genes enriched in the HIF-1 signaling pathway, indicating these species as potential therapy targets. Further experiments on validation data sets have also supported these patterns. Collectively, ASD-cancer has enabled accurate survival subtyping and biomarker discovery, which could facilitate personalized treatment for broad-spectrum types of cancers. IMPORTANCE Unraveling the intricate relationship between the tumor microbiome, host gene expressions, and their collective impact on cancer outcomes is paramount for advancing personalized treatment strategies. Our study introduces ASD-cancer, a cutting-edge autoencoder-based subtype detector. ASD-cancer decodes the complexities within the tumor microenvironment, successfully identifying distinct survival subtypes across 20 cancer types. Its superior risk stratification, demonstrated by significant improvements over traditional methods like principal component analysis, holds promise for refining patient prognosis. Accurate survival subtype predictions, biomarker discovery, and insights into microbe-host gene interactions elevate ASD-cancer as a powerful tool for advancing precision medicine. These findings not only contribute to a deeper understanding of the tumor microenvironment but also open avenues for personalized interventions across diverse cancer types, underscoring the transformative potential of ASD-cancer in shaping the future of cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bb9bb246cc99199c3ecec679e18a3d2f3b8f074" target='_blank'>
              Deep learning enabled integration of tumor microenvironment microbial profiles and host gene expressions for interpretable survival subtyping in diverse types of cancers
              </a>
            </td>
          <td>
            Haohong Zhang, Xinghao Xiong, Mingyue Cheng, Lei Ji, Kang Ning
          </td>
          <td>2024-11-20</td>
          <td>mSystems</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59e5b41f611d8bd26c1558060c6ef330913ca3b" target='_blank'>
              Pan-cancer drivers of metastasis
              </a>
            </td>
          <td>
            Ryan Lusby, Engin Demirdizen, Mohammed Inayatullah, Paramita Kundu, O. Maiques, Ziyi Zhang, Mikkel G Terp, Victoria Sanz-Moreno, Vijay K Tiwari
          </td>
          <td>2025-01-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer.
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-02</td>
          <td>Cell research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="
 Background:
 Understanding the intricate dynamics of lung microcirculation and capillary endothelial cells (ECs) is crucial for shedding light on both normal physiological processes and the pathogenesis of pulmonary diseases. Recent advancements in single-cell transcriptomics (scRNA-seq) have unveiled distinct lung EC populations, including aerocytes and general capillary cells (gCaps).


 Hypothesis:
 We hypothesized that gCaps demonstrate more extensive diversity than previously understood, marked by distinct molecular and functional profiles.


 Methods:
 We utilized scRNA-seq and computational analysis on enriched populations of lung ECs. The mitostress Seahorse assay was conducted to study metabolic function. Electric cell-substrate impedance sensing (ECIS) assessed cell growth, and the Matrigel assay evaluated angiogenic potential of isolated ECs.


 Results:
 scRNA-seq analysis identified five novel gCap populations, each with unique molecular signatures and functions. Notably, two populations characterized by the expression of ion transporters Scn7a (Na+) and Clic4 (Cl−) are crucial for establishing arterial-to-vein zonation and enhancing the capillary barrier. We identified "root" cells (Flot1+), exhibiting the highest G2M and S scores, essential for the mitotic regeneration and repair of neighboring ECs. Depletion of Flot1+ cells significantly impaired EC proliferation, highlighting their critical role in cell repair. Furthermore, we discovered that gCaps aiding in the venous transition were characterized by the expression of Lingo2. Additionally, we identified a population of detached gCaps marked by an elevated expression of Fabp4, metabolically active genes, and tip-cell markers, suggesting a key role in regulating angiogenesis. The Matrigel sprouting assay confirmed that Fabp4+ gCaps significantly contribute to angiogenesis. Metabolic assays indicated that both Clic4+ and Fabp4+ cells exhibited increased mitochondrial function. Finally, western blot analysis across various EC junction and signaling proteins in isolated gCaps showed significant heterogeneity, further illustrating the complex nature of these cell populations.


 Conclusion:
 Our study highlights the diverse roles and molecular signatures of lung capillary ECs, enhancing our understanding of their contributions to both normal lung function and pulmonary diseases. These insights also offer valuable direction for developing therapeutic strategies aimed at addressing cardiovascular disorders.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7342ec242764aeb67a6add3cfe72a3a31b11119" target='_blank'>
              Abstract 4140265: Single Cell Transcriptomics Unveils Novel Lung Capillary Endothelial Cell Populations and Their Crucial Roles
              </a>
            </td>
          <td>
            J. James, A. Dekan, D. Bharti, M. Valuparampil Varghese, Sedat Kacar, M. Niihori, O. Rafikova, R. Rafikov
          </td>
          <td>2024-11-12</td>
          <td>Circulation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="DNA methylation can give rise to robust biomarkers of aging, yet most studies profile it at the bulk tissue level, which masks cell type-specific alterations that may follow distinct aging trajectories. Long-read sequencing technology enables methylation profiling of extended DNA fragments, which allows mapping to their cell type of origin. In this study, we introduce a framework for evaluating cell type-specific aging using long-read sequencing data, without the need for cell sorting. Leveraging cell type-specific methylation patterns, we map long-read fragments to individual cell types and generate cell type-specific methylation profiles, which are used as input to a newly developed probabilistic aging model, LongReadAge, capable of predicting epigenetic age at the cell-type level. We apply LongReadAge to track aging of myeloid cells and lymphocytes from bulk leukocyte data as well as circulating cell-free DNA, demonstrating robust performance in predicting age despite limited shared features across samples. This approach provides a novel method for profiling the dynamics of epigenetic aging at cell-type resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5709038484acf3d57f1f45cf8c79fcec1017a895" target='_blank'>
              Profiling Epigenetic Aging at Cell-Type Resolution Through Long-Read Sequencing
              </a>
            </td>
          <td>
            A. Eames, M. Moqri, J. Poganik, V. N. Gladyshev
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c426370b1fe4fbd062526de11c77d9b155f25f" target='_blank'>
              Single-cell RNA sequencing reveals immune microenvironment niche transitions during the invasive and metastatic processes of ground-glass nodules and part-solid nodules in lung adenocarcinoma
              </a>
            </td>
          <td>
            Yi-Feng Ren, Qiong Ma, Xiao Zeng, Chun-Xia Huang, Jia-Li Ren, Fang Li, Jia-Jing Tong, Jia-Wei He, Yang Zhong, Shi-Yan Tan, Hua Jiang, Long-Fei Zhang, Heng-zhou Lai, Ping Xiao, Zhuang Xiang, Peng Wu, Li-Ting You, Wei Shi, Xi Fu, Chuan Zheng, Feng-ming You
          </td>
          <td>2024-11-23</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Introduction. During metastasis, tumor cells interact extensively with endogenous cells in distant organs. These interactions change the behavior of these surrounding cell populations, often inducing a more pro-tumor microenvironment. The lung is one of the most common sites of breast cancer (BC) metastasis, and in the lung, alveolar epithelial cells are the most common cell type. My data suggest that the lung alveolar epithelium contributes to metastatic outgrowth, and I hypothesize that BC lung micrometastases activate surrounding lung epithelial cells which, in turn, support the outgrowth of BC metastases within the lung. The overall purpose of these studies is to identify factors secreted by resident lung epithelial cells that could be used as metastasis-specific therapeutic targets and/or agents. Methods. To test this hypothesis, I utilized immunocompetent preclinical BC metastasis models to study lung metastatic outgrowth, the stage at which most patients are diagnosed with metastatic disease. A custom imaging panel was developed to quantify lung wound repair, and single-cell RNA-sequencing (scRNAseq) was performed on mouse lungs with high or low metastatic burden to identify genes in the lung epithelium that produce potentially targetable pro-metastatic secreted factors. No-contact co-culture experiments were used to directly study reciprocal paracrine interactions between lung type II alveolar epithelial (AT2) and BC cells. Results. A wound repair-related phenotype, characterized by chronic inflammation, developed within the lung microenvironment during metastatic outgrowth, including an increase in the number and activation of AT2 cells surrounding metastases as they grow. Single-cell RNA-sequencing of mouse lungs with a high vs. low metastatic burden showed that metastatic outgrowth significantly changed AT2 gene expression resulting in a modified secretome. No-contact co-culture experiments indicated that TNBC cells directly alter AT2 gene expression, while AT2 secreted factors promoted TNBC growth. I investigated the possible mechanism(s) responsible for these effects and discovered that AT2 pulmonary surfactant protein and lipid levels are altered by BC-derived secreted factors. Interestingly, treatment with the naturally derived calf surfactant Infasurf, which contains native surfactant proteins/lipids and is FDA-approved for use in premature infants, inhibited BC proliferation. Conclusion. Low levels of surfactant are commonly associated with pulmonary disease and lung cancer. My BC lung metastasis data suggest that something similar may be occurring in the lungs during metastatic outgrowth. Overall, my studies demonstrate the potential for targeting lung epithelial cells in the metastatic microenvironment, in addition to directly targeting malignant cells, as an effective way to treat and manage BC lung metastases.
 Citation Format: Jessica L. Christenson, Nicole S. Spoelstra, Jennifer K. Richer. Reciprocal activation of breast cancer metastases and the lung epithelium during metastatic outgrowth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b54efc3b640295c6631376423e51f0df8729df6" target='_blank'>
              Abstract A001: Reciprocal activation of breast cancer metastases and the lung epithelium during metastatic outgrowth
              </a>
            </td>
          <td>
            Jessica L. Christenson, Nicole S Spoelstra, Jennifer K Richer
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Within the tumor microenvironment (TME), cancer-associated fibroblasts (CAFs) were shown to be an active and pivotal cell population, supporting many protumorigenic mechanisms. Podoplanin (PDPN)-positive CAFs are of special interest since their abundance correlated with a worse prognosis for patients of different cancer entities, including malignant melanoma. In this study, we applied a loss-of-function approach in an in vivo mouse melanoma model to evaluate the contribution of CAF-specific PDPN expression to melanoma formation and progression. Surprisingly, despite its prominent expression in CAFs deletion of PDPN in this cell type did neither affect the onset, nor growth of MM tumors. These data imply that PDPN expression in CAFs represents a biomarker for poor prognosis but does not serve as a useful target for stroma-directed therapy of malignant melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3666add015c6558ff2185b82cf13e728aaa3557e" target='_blank'>
              CAF Specific Expression of Podoplanin May Be Dispensable for the Malignancy of Malignant Melanoma.
              </a>
            </td>
          <td>
            Saskia Tauch, Bettina Kast, S. Lohr, Lowis Kemm, Melanie Sator‐Schmitt, N. Gengenbacher, H. Augustin, Peter Angel
          </td>
          <td>2024-11-08</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62a0b0314fc694a0d5ac1d3e0908ca758f9f880c" target='_blank'>
              Cancer-associated fibroblasts, tumor and radiotherapy: interactions in the tumor micro-environment
              </a>
            </td>
          <td>
            Kris Raaijmakers, G. Adema, Johan Bussink, M. Ansems
          </td>
          <td>2024-12-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Phenotypic plasticity is a hallmark of cancer and increasingly realized as a mechanism of resistance to androgen receptor (AR)-targeted therapy. Now that many prostate cancer (PCa) patients are treated upfront with AR-targeted agents, it's critical to identify actionable mechanisms that drive phenotypic plasticity, to prevent the emergence of resistance. We showed that loss of tristetraprolin (TTP, gene ZFP36) increased NF-κB activation, and was associated with higher rates of aggressive disease and early recurrence in primary PCa. We also examined the clinical and biological impact of ZFP36 loss with co-loss of PTEN, a known driver of PCa. Analysis of multiple independent primary PCa cohorts demonstrated that PTEN and ZFP36 co-loss was associated with increased recurrence risk. Engineering prostate-specific Zfp36 deletion in vivo, induced prostatic intraepithelial neoplasia, and, with Pten co-deletion, resulted in rapid progression to castration-resistant adenocarcinoma. Zfp36 loss altered the cell state driven by Pten loss, demonstrated by enrichment of EMT, inflammation, TNFα/NF-κB, IL6-JAK/STAT3 gene sets. Additionally, our work revealed that ZFP36 loss also induced enrichment of multiple gene sets involved in mononuclear cell migration, chemotaxis, and proliferation. Use of the NF-κB inhibitor, dimethylaminoparthenolide (DMAPT) induced marked therapeutic responses in tumors with PTEN and ZFP36 co-loss and reversed castration resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a8debd5a52234a8e6e2eecf0d0e1cd5849b25a" target='_blank'>
              Low tristetraprolin expression activates phenotypic plasticity and primes transition to lethal prostate cancer in mice.
              </a>
            </td>
          <td>
            Katherine L. Morel, Beatriz German, Anis A Hamid, Jagpreet S Nanda, S. Linder, A. Bergman, Henk van der Poel, Ingrid Hofland, E. Bekers, S. Trostel, Deborah L. Burkhart, S. Wilkinson, Anson T. Ku, Minhyung Kim, Jina Kim, Duanduan Ma, Jasmine T. Plummer, Sungyong You, Xiaofeng A. Su, W. Zwart, A. Sowalsky, Christopher J. Sweeney, Leigh Ellis
          </td>
          <td>2024-11-19</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb40aeff67c4162245cfc687f8adca56b59144a" target='_blank'>
              Integrated molecular and functional characterization of the intrinsic apoptotic machinery identifies therapeutic vulnerabilities in glioma
              </a>
            </td>
          <td>
            E. Fernández, Wilson X Mai, Kai Song, Nicholas A. Bayley, Jiyoon Kim, Henan Zhu, Marissa S Pioso, Pauline Young, Cassidy Andrasz, Dimitri Cadet, Linda M Liau, Gang Li, William Yong, Fausto J Rodriguez, Scott J. Dixon, Andrew J Souers, Jingyi Jessica Li, Thomas Graeber, T. Cloughesy, David A. Nathanson
          </td>
          <td>2024-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Hypoxia is significantly associated with cancer progression and treatment outcomes. Nevertheless, the precise molecular mechanisms underlying the hypoxia-induced immunosuppressive microenvironment in high-grade serous ovarian cancer (HGSOC) are still not fully understood. Methods By analyzing five independent transcriptomic datasets, we investigated the effect of hypoxia on prognosis and tumor microenvironment (TME) in HGSOC. The hypoxia levels and the intercellular communication signaling pathways were studied by using single-cell analysis. Furthermore, the Hypoxia-TME classifier was developed and then validated in the multiple HGSOC datasets. In addition, we also investigated the prognostic significance, genetic variations, signaling pathways, and the potential for immunotherapy benefits in different Hypoxia-TME subgroups. Results Hypoxia was identified as a crucial risk factor in HGSOC, and strongly correlated with an immunosuppressive microenvironment characterized by alterations in the composition and distribution of immune cells. Single-cell analysis elucidated the heterogeneity inherent within the TME in HGSOC, and demonstrated an association between the hypoxic TME and fibroblasts as well as macrophages. CellChat analysis identified SPP1-CD44 and CXCL12-CXCR4 as the principal signaling axes through which macrophages and fibroblasts interact with T cells, respectively. Moreover, a personalized Hypoxia-TME classifier was constructed and validated through the integration of the hypoxia (18 genes) and TME (7 immune cells) scores. It was observed that patients in the Hypoxialow/TMEhigh subgroup displayed a significantly better prognosis than other subgroups. Different subgroups exhibited unique genomic alterations and variations in signaling pathway differences, including TGF-β and Wnt/β-catenin pathways, which are closely associated with various biological functions. Finally, our results indicated that patients in the Hypoxialow/TMEhigh subgroup exhibit a better response to immunotherapy, suggesting the potential utility of the Hypoxia-TME classifier as a new biomarker in HGSOC. Conclusion Our study revealed hypoxia-induced immunosuppressive microenvironment, and developed Hypoxia-TME classifier to distinguish the prognosis, immune characteristics, and potential benefits of immunotherapy in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20687a43e7305e0122a3a8346c6bbe4322680deb" target='_blank'>
              Integrated analysis of single-cell and bulk transcriptome reveals hypoxia-induced immunosuppressive microenvironment to predict immunotherapy response in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Qingshan Chen, Yue Zhang, Chao Wang, Hui Ding, Liqun Chi
          </td>
          <td>2024-11-13</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24be616e06c75b57b6ca8151f14f1e70c2c9348f" target='_blank'>
              PIK3CA mutation fortifies molecular determinants for immune signaling in vascular cancers.
              </a>
            </td>
          <td>
            Donghee Lee, Emma C Kozurek, Md Abdullah, Ethan J Wong, Rong Li, Z. Liu, Hai Dang Nguyen, Erin Dickerson, Jong Hyuk Kim
          </td>
          <td>2024-12-21</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Nongenetic plasticity has emerged as a key driver of cancer drug resistance, yet its precise origins, nature, and consequences are not fully clarified. This review examines technological, computational, and conceptual developments in the nongenetic determinants of drug resistance. We begin by proposing refined definitions of cellular state, fate, and plasticity. We subsequently contextualize the findings from multimodal approaches to investigate plasticity, highlighting how new single-cell lineage-tracing methods provide opportunities for quantitatively capturing state transitions, their timescales and heritability, and how they contribute to resistance mechanisms. We also draw parallels with concepts from developmental biology and microbial persistence research. Next, we cover the role that computational approaches have played in revealing the otherwise latent patterns of heterogeneity that underlie plasticity from complex datasets. We conclude by emphasizing the need for standardized terminology in this rapidly evolving field and the path from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38fadd15ca85570a489d3058bda72d96573ce1d" target='_blank'>
              Navigating Nongenetic Plasticity in Cancer Drug Resistance
              </a>
            </td>
          <td>
            Hanxiao Sun, Nitu Kumari, Madeline E. Melzer, Vadim Backman, Yogesh Goyal
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Some cancers such as glioblastoma (GBM), show minimal response to medical interventions, often only capable of mitigating tumor growth or alleviating symptoms. High metabolic activity in the tumor microenvironment marked by immune responses and hypoxia, is a crucial factor driving tumor progression. The many developments in mass spectrometry (MS) over the last decades have provided a pivotal tool for studying proteins, along with their posttranslational modifications. It is known that the proteomic landscape of GBM comprises a wide range of proteins involved in cell proliferation, survival, migration, and immune evasion. Combination of MS imaging and microscopy has potential to reveal the spatial and molecular characteristics of pathological tissue sections. Moreover, integration of MS in the surgical process in form of techniques such as DESI-MS or rapid evaporative ionization MS has been shown as an effective tool for rapid measurement of metabolite profiles, providing detailed information within seconds. In immunotherapy-related research, MS plays an indispensable role in detection and targeting of cancer antigens which serve as a base for antigen-specific therapies. In this review, we aim to provide detailed information on molecular profile in GBM and to discuss recent MS advances and their clinical benefits for targeting this aggressive disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ad806685fa50b021dffd830b867c2367ab7a25" target='_blank'>
              Mass Spectrometry Advances in Analysis of Glioblastoma.
              </a>
            </td>
          <td>
            Sofian Al Shboul, Ashita Singh, R. Kobetić, David R. Goodlett, P. M. Brennan, Ted Hupp, I. Dapic
          </td>
          <td>2024-11-11</td>
          <td>Mass spectrometry reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND
The pathogenesis of psoriasis, an inflammatory skin disease, is incompletely understood. Growing evidence substantiates the involvement of stromal cells in the inflammatory process.


OBJECTIVES
To investigate the roles of stromal cells, including fibroblasts, vascular endothelial cells (VECs) and smooth muscle cells (VSMCs), in the psoriatic inflammatory microenvironment and the possible underlying mechanisms involved.


METHODS
This study employed combination of single-cell, spatial transcriptome and bulk RNA sequencing using lesional and nonlesional skin samples from patients with psoriasis vulgaris (PV) and healthy skin samples from unaffected individuals.


RESULTS
Through the analysis of transcriptome from 364,098 single cells, we uncovered WNT5A+ fibroblasts, ITIH5+ VECs and VCAN+ VSMCs with the significantly increased cell proportions in the papillary dermis of lesional skin. We defined eight unique subclusters of fibroblasts in the skin and observed a shift of WIF1+ fibroblasts towards WNT5A+ fibroblasts, with abnormal activation of the non-canonical Wnt signaling pathway and increased capabilities of angiogenesis and pro-inflammatory. For the microvascular cells, VSMCs could undergo phenotypic transformation from a contractile phenotype to a synthetic phenotype in the development of psoriatic inflammation. ITIH5+ VECs and VCAN+ VSMCs were identified with an essential role in regulating angiogenesis and vascular remodeling involved in the mechanism of psoriatic pathological changes. Ligand receptor analyses demonstrated WNT5A+ fibroblasts were extensively implicated in interactions with various cell types in skin, especially with ITIH5+ VECs and VCAN+ VSMCs within the papillary dermis.


CONCLUSIONS
Interactions of stromal cells in the papillary dermis were identified as possible pathogenic elements in psoriasis vulgaris. Improving the inflammatory microenvironment by targeting stromal cells might be a potential treatment strategy for psoriasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1eb8edc16b87584906a5c6fdfa4b1295d5ed3a" target='_blank'>
              Multitranscriptome analysis revealed that stromal cells in the papillary dermis promote angiogenesis in psoriasis vulgaris.
              </a>
            </td>
          <td>
            Bo Zhang, Junpu Mei, Qijun Liao, Shan Zhou, He Huang, Hui Liu, Xiaoli Xu, Yafen Yu, Chao Wu, Wenjun Wang, Weining Hu, Tingting Zhu, Yin Zhang, Mengyun Chen, Caihong Zhu, Mengjun Yu, Jinping Gao, Xianfa Tang, Xiawei Liu, Ze Guo, Xiaodong Zheng, Zhuang Wen, Gang Chen, Lili Tang, Xiaoyan Ding, Hui Cheng, Yang Li, Hongyan Wang, Hui Li, Yangrui Zhang, Xing Fan, Rouxi Chen, Zherou Rong, Ping Liu, Shengxiu Liu, Zhen Yue, Peiguang Wang, Zhiming Cai, Min Gao, Zaixing Wang, Xiaodong Fang, Fusheng Zhou, Huayang Tang
          </td>
          <td>2024-11-21</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
Despite recent advances in the biology of IDH-wildtype glioblastoma, it remains a devastating disease with median survival of less than 2 years. However, the molecular underpinnings of the heterogeneous response to the current standard-of-care treatment regimen consisting of maximal safe resection, adjuvant radiation, and chemotherapy with temozolomide remain unknown.


METHODS
Comprehensive histopathologic, genomic, and epigenomic evaluation of paired initial and recurrent glioblastoma specimens from 106 patients was performed to investigate the molecular evolution and cellular phenotypes underlying differential treatment responses.


RESULTS
While TERT promoter mutation and CDKN2A homozygous deletion were early events during gliomagenesis shared by initial and recurrent tumors, most other recurrent genetic alterations (eg, EGFR, PTEN, and NF1) were commonly private to initial or recurrent tumors indicating acquisition later during clonal evolution. Furthermore, glioblastomas exhibited heterogeneous epigenomic evolution with subsets becoming more globally hypermethylated, hypomethylated, or remaining stable. Glioblastoma that underwent sarcomatous transformation had shorter interval to recurrence and were significantly enriched in NF1, TP53, and RB1 alterations and the mesenchymal epigenetic class. Patients who developed somatic hypermutation following temozolomide treatment had significantly longer interval to disease recurrence and prolonged overall survival, and increased methylation at 4 specific CpG sites in the promoter region of MGMT was significantly associated with this development of hypermutation. Finally, an epigenomic evolution signature incorporating change in DNA methylation levels across 347 critical CpG sites was developed that significantly correlated with clinical outcomes.


CONCLUSIONS
Glioblastoma undergoes heterogeneous genetic, epigenetic, and cellular evolution that underlies prognostically different treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8c70c5b87e0a8ebd1138c27ed60e44214d7859" target='_blank'>
              Longitudinal multimodal profiling of IDH-wildtype glioblastoma reveals the molecular evolution and cellular phenotypes underlying prognostically different treatment responses.
              </a>
            </td>
          <td>
            Calixto-Hope G Lucas, N. Al-Adli, Jacob S. Young, Rohit Gupta, R. Morshed, Jasper Wu, Ajay Ravindranathan, Anny Shai, N. Oberheim Bush, Jennie W Taylor, John de Groot, J. Villanueva‐Meyer, M. Pekmezci, Arie Perry, Andrew W Bollen, P. Theodosopoulos, M. Aghi, E. Chang, Shawn L Hervey-Jumper, D. Raleigh, Annette M Molinaro, Joseph F Costello, Aaron A. Diaz, Jennifer L. Clarke, N. Butowski, Joanna J. Phillips, Susan M Chang, Mitchell S. Berger, D. A. Solomon
          </td>
          <td>2024-11-19</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="The study by Grossmann and colleagues uses single-nucleus RNA sequencing in a cohort of matched high-risk neuroblastoma primary tumor samples, obtained from the same patient at diagnosis and definitive surgery, to identify persister cells that survive induction chemotherapy. These persister cells utilize mechanisms of chemoresistance that are both tumor-intrinsic and tumor-extrinsic, are highly dependent on the original genetic profile of the tumor, and represent novel, patient-specific targets to precisely inhibit chemoresistance and disease recurrence. See related article by Grossmann et.al., p. 2387.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17960d164122a139c932c5adc28d8d07f55f0399" target='_blank'>
              Characterization of Persister Cells Provides Insights into Mechanisms of Therapy Resistance in Neuroblastoma.
              </a>
            </td>
          <td>
            Amber B. Wolf, C. P. Reynolds, Eveline Barbieri
          </td>
          <td>2024-12-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite recent advances, improvements to long-term survival in metastatic carcinomas, such as pancreatic or ovarian cancer, remain limited. Current therapies suppress growth-promoting biochemical signals, ablate cells expressing tumor-associated antigens, or promote adaptive immunity to tumor neoantigens. However, these approaches are limited by toxicity to normal cells using the same signaling pathways or expressing the same antigens, or by the low frequency of neoantigens in most carcinomas. Here, we report a fundamentally different strategy for designing safer and more effective anti-cancer therapies through the sensing of cancer-driving biochemical signals and their rewiring to virotherapeutic activation. Specifically, we rationally engineer a RNA vector to self-replicate and cause cytotoxicity in cancer cells exhibiting hyperactive HER2 (ErbB2), but not in normal cells with normal HER2 signaling. Compared to a widely tested virotherapeutic from the same vector family, our hyperactive ErbB2-restricted RNA vector (HERV) demonstrated lower toxicity and greater activity against metastatic HER2-positive ovarian cancer in mice, extending survival independently of tumor antigenicity. Most importantly, HERV synergizes with standard-of-care chemotherapy against ovarian cancer metastases in vivo, with 43% of combination-treated subjects surviving for months beyond subjects treated to either therapy alone. Taken together, these results introduce rewiring of cancer-driving signaling pathways to virotherapeutic activation as a strategy for more specific and effective cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf89a175114fc9ad8dc2a85a68772e9b4013140e" target='_blank'>
              Rewiring oncogenic signaling to RNA vector replication for the treatment of metastatic cancer
              </a>
            </td>
          <td>
            Xinzhi Zou, Cynthia Zhao, Kevin T. Beier, Chil-Yong Kang, Michael Z. Lin
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Intrapatient tumor heterogeneity contributes to immunotherapy and targeted therapy resistance in cutaneous metastatic melanoma patients. More than 50% of melanoma patients develop brain metastases (MBM) during disease progressionand these patients have a very poor prognosis with a median overall survival of 4 months. To assess how tumors persist through treatments and adapt to and rely on organ-specific microenvironments, we utilized samples from a patient who had received anti-CTLA4, BRAFi/MEKi, and BRAFi/MEKi+CDK4/6i treatments during the LOGIC2 clinical trial. Eight short-term melanoma cultures (STMCs) were established from different metastatic site biopsies at chronological time points throughout the patient’s treatment and subsequent post-mortem resection of the bone, ovary, and brain metastatic sites. While the initial treatment naïve metastatic tumor in the breast was BRAF V600E mutant, NRAS Q61 mutations were acquired after treatment with CDK4/6i in the bone and ovary tumors. We performed single-cell RNA sequencing and identified that each STMC consists of 2-10 different subpopulations. By cell viability assay, we determined all STMCs were resistant to CDK4/6i, however had varying degrees of response to BRAFi and MEKi in vitro. Two STMCs (MM150604, MM150859) were sensitive to both BRAFi and MEKi. We identified HER3 was upregulated in the sensitive STMCs and more specifically, HER3 expression was elevated in the MBM STMC and in other MBM patient tumors in publicly available datasets. In publicly available data of the MBM tumor microenvironment (TME), neurons produce NRG1 and NRG2 while astrocytes and pericytes produce NRG2, exogenous sources of NRGs in the MBM TME that act as ligands for the HER3 receptor. Treatment with recombinant NRG1 partially rescued the growth of MBM cells and restored MAPK and AKT-mTOR signaling in the presence of BRAFi/MEKi suggesting that the brain microenvironment may play a role in drug tolerance. This novel finding demonstrates a unique role of HER3 in MBM and targeting the NRG-HER3 signaling axis could have therapeutic potential in melanoma patients with active brain metastases.
 Citation Format: Haley P. Wilson, Glenn L. Mersky, Jelan I. Haj, Jessica L.F. Teh, Phil F. Cheng, Signe Caksa, Manal Mustafa, Isabella Trachtenberg, Francis J. Waltrich Jr, Casey D. Stefanski, Vivian Chua, Dan A. Erkes, Mitchell P. Levesque, Reinhard Dummer, Timothy J. Purwin, Claudia Capparelli, Andrew E. Aplin. Intrapatient heterogeneity and targeting microenvironment protection mechanisms in melanoma metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C048.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44bfe1a0d7f48e3ce31e054b1efd8507580f7dfc" target='_blank'>
              Abstract C048: Intrapatient heterogeneity and targeting microenvironment protection mechanisms in melanoma metastasis
              </a>
            </td>
          <td>
            Haley P Wilson, Glenn L Mersky, Jelan I Haj, J. Teh, Phil F Cheng, Signe Caksa, Manal U Mustafa, Isabella V Trachtenberg, Francis J. Waltrich, Casey D Stefanski, Vivian Chua, Dan A Erkes, Mitchell Levesque, R. Dummer, Timothy J. Purwin, Claudia Capparelli, Andrew E. Aplin
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f2150cfbdcf9bf561de7bbb85cf14321344131" target='_blank'>
              Extrajunctional CLDN10 cooperates with LAT1 and accelerates clear cell renal cell carcinoma progression
              </a>
            </td>
          <td>
            Akifumi Onagi, K. Sugimoto, M. Kobayashi, Yumi Sato, Yasuyuki Kobayashi, Kei Yaginuma, Satoru Meguro, Seiji Hoshi, Jyunya Hata, Yuko Hashimoto, Yoshiyuki Kojima, Hideki Chiba
          </td>
          <td>2024-12-05</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0adb15ba326745aaf69f2834817afe756ee03a2" target='_blank'>
              Integrated proteomics and metabolomics analyses reveal new insights into the antitumor effects of valproic acid plus simvastatin combination in a prostate cancer xenograft model associated with downmodulation of YAP/TAZ signaling
              </a>
            </td>
          <td>
            Federica Iannelli, Rita Lombardi, S. Costantini, M. Roca, L. Addi, F. Bruzzese, E. Di Gennaro, Alfredo Budillon, B. Pucci
          </td>
          <td>2024-11-16</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaa35d32ecdf58360fe2158aa64cfa3b17ec1726" target='_blank'>
              Evolving cell states and oncogenic drivers during the progression of IDH-mutant gliomas.
              </a>
            </td>
          <td>
            Jingyi Wu, L. G. Gonzalez Castro, S. Battaglia, C. E. El Farran, J. D’Antonio, Tyler E. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-11-21</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="KRAS is the most frequently mutated oncogene in lung adenocarcinoma, with G12C and G12V being the most predominant forms. Recent breakthroughs in KRASG12C inhibitors have transformed the clinical management of patients with G12C mutation and advanced our understanding of its function. However, little is known about the targeted disruption of KRASG12V, partly due to a lack of specific inhibitors. Here, we leverage the degradation tag (dTAG) system to develop a KRASG12V transgenic mouse model. We explore the therapeutic potential of KRASG12V degradation and characterize its impact on the tumor microenvironment (TME). Our study reveals that degrading KRASG12V abolishes lung and pancreatic tumors in mice and causes a robust inhibition of KRAS-regulated cancer intrinsic signaling. Importantly, targeted degradation of KRASG12V reprograms the TME towards a stimulatory milieu and drives antitumor immunity, elicited mainly by effector and cytotoxic CD8+ T cells. Our work provides important insights into the impact of degrading KRASG12V on both tumor progression and immune response, highlighting degraders as a powerful strategy for targeting KRAS mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7557f24725afcb0b1e2df48764b1f6cd0b9ab54" target='_blank'>
              Targeted degradation of oncogenic KRASG12V triggers antitumor immunity in lung cancer models.
              </a>
            </td>
          <td>
            Dezhi Li, Ke Geng, Yuan Hao, Jiajia Gu, Saurav Kumar, Annabel T Olson, Christina C. Kuismi, Hye Mi Kim, Yuanwang Pan, Fiona Sherman, Asia M. Williams, Yiting Li, Fei Li, Ting Chen, C. Thakurdin, Michela Ranieri, Mary Meynardie, Daniel S. Levin, Janaye Stephens, Alison Chafitz, Joy Chen, Mia S Donald-Paladino, Jaylen M Powell, Ze-Yan Zhang, Wei Chen, Magdalena Ploszaj, Han Han, S. Gu, Tinghu Zhang, Baoli Hu, B. Nacev, Medard Ernest Kaiza, Alice H. Berger, Xuerui Wang, Jing Li, Xuejiao Sun, Yang Liu, Xiaoyang Zhang, T. Bruno, N. Gray, Behnam Nabet, Kwok-Kin Wong, Hua Zhang
          </td>
          <td>2024-12-24</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Oral squamous cell carcinoma (OSCC) is a prevalent type of head and neck cancer with a high mortality rate, partly due to late diagnosis often accompanied by metastasis. Cachexia, a complex syndrome marked by severe weight loss and muscle wasting, frequently accompanies advanced cancers including OSCC. Notably, 50-70% of OSCC patients exhibit cachexia at diagnosis, which persists post-treatment, hinting at a metastatic contribution to this syndrome. One significant challenge in studying cancer cachexia is the lack of well-characterized pre-clinical models that accurately reflect human disease. Traditional mouse models often involve subcutaneously transplanted cells, which fail to mimic the natural tumor-host interactions and metastatic progression seen in patients. These models do not typically exhibit metastasis and differ significantly from the genetic and molecular landscape of human cancers. To address these limitations, we developed a syngeneic mouse model by orthotopically transplanting MOC2 murine OSCC cells into the tongues of B6J mice, thereby closely mimicking the tumor microenvironment and metastatic behavior of human OSCC. By enhancing metastasis with a palm-enriched high-fat diet, we observed progressive tumor growth, metastasis, and body weight loss, independent of food intake reduction, indicating metabolic dysregulation rather than anorexia. Comprehensive analyses, including single-cell RNA sequencing (scRNAseq) of immune cells and transcriptomic and metabolomic profiling of tumor and host tissues, identified significant gene expression changes associated with metastasis and cachexia. Enhanced metastasis correlated with lipid metabolism changes in lung metastases, suggesting that metastatic cells exploit lipids for growth. Further, immune profiling revealed neutrophils' metabolic adaptation in metastasis, with implications for both tumor progression and cachexia. Targeting key metabolic and inflammatory pathways in tumor and immune cells holds potential for mitigating metastasis and cachexia. Early-stage serum metabolomics indicated systemic metabolic shifts, emphasizing the need to understand the interplay between tumor metastasis and host metabolism. These findings underscore the complex mechanisms underlying OSCC-induced cachexia, and the need for new targeted therapeutic strategies to disrupt this lethal synergy between cancer progression and metabolic deterioration of the host.
 Citation Format: Blanca Majem, Josefina Martin, Liudmila Shevkova, Claudia Bigas, Gloria Pascual, Salvador Aznar-Benitah. Cross-talk between metastatic cells and host systems: Neutrophil metabolic adaptation, immune profiling, and systemic metabolic shifts in tumor progression and cachexia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr PR008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c1eb32f102c37cc14f504819a073149f1e255e" target='_blank'>
              Abstract PR008: Cross-talk between metastatic cells and host systems: Neutrophil metabolic adaptation, immune profiling, and systemic metabolic shifts in tumor progression and cachexia
              </a>
            </td>
          <td>
            Blanca Majem, Joseﬁna Martin, Liudmila Shevkova, Claudia Bigas, Gloria Pascual, Salvador Aznar-Benitah
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Gastric intestinal metaplasia (IM) is a precancerous stage spanning a morphological spectrum that is poorly represented by human cell line models.


OBJECTIVE
We aim to establish and characterise human IM cell models to better understand IM progression along the cancer spectrum.


DESIGN
A large human gastric IM organoid (IMO) cohort (n=28), their clonal derivatives and normal gastric organoids (n=42) for comparison were established. Comprehensive multi-omics profiling and functional characterisation were performed.


RESULTS
Single-cell transcriptomes revealed IMO cells spanning a spectrum from hybrid gastric/intestinal to advanced intestinal differentiation. Their lineage trajectories connected different cycling and quiescent stem and progenitors, highlighting differences in gastric to IM transition and the potential origin of IM from STMN1 cycling isthmus stem cells. Hybrid IMOs showed impaired differentiation potential, high lineage plasticity beyond gastric or intestinal fates and reactivation of a fetal gene programme.Cell populations in gastric IM and cancer tissues were highly similar to those derived from IMOs and exhibited a fetal signature. Genomically, IMOs showed elevated mutation burden, frequent chromosome 20 gain and epigenetic deregulation of many intestinal and gastric genes. Functionally, IMOs were FGF10 independent and showed downregulated FGFR2. Several IMOs exhibited a cell-matrix adhesion independent subpopulation that displayed chromosome 20 gain but lacked key cancer driver mutations, potentially representing the earliest neoplastic precursor of IM-induced gastric cancer.


CONCLUSIONS
Overall, our IMO biobank captured the heterogeneous nature of IM, revealing mechanistic insights on IM pathogenesis and progression, offering an ideal platform for studying early gastric neoplastic transformation and chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10191e1bc78f541e4ae8577c359e64a33ed296da" target='_blank'>
              Divergent lineage trajectories and genetic landscapes in human gastric intestinal metaplasia organoids associated with early neoplastic progression.
              </a>
            </td>
          <td>
            Sarah S K Yue, Yin Tong, H. Siu, S. Ho, S. Law, W. Tsui, D. Chan, Yuanhua Huang, Annie S Y Chan, Shui Wa Yun, Ho Sang Hui, Jee-Eun Choi, Matthew S S Hsu, Frank P L Lai, April S Chan, S. Yuen, Hans Clevers, Suet Yi Leung, H. Y. Yan
          </td>
          <td>2024-11-20</td>
          <td>Gut</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is well known for its immune suppressive role, especially in solid tumors which are characterized by a thick, dense stroma. Apart from cell-cell interactions and biochemical signals, the tumor stroma is also characterized by its distinct mechanical properties, which are dictated by the composition and architecture of its extracellular matrix (ECM). Cancer-associated fibroblasts (CAFs) are the main producers and remodelers of the stromal ECM, and their heterogeneity has recently become a focus of intense research. This review describes recent findings highlighting CAF subtypes and their specific functions, as well as the development of 3D models to study tumor stroma mechanics in vitro. Finally, we discuss the quantitative techniques used to measure tissue mechanical properties at different scales. Given the diagnostic and prognostic value of stroma stiffness and composition, and the recent development of anti-tumor therapeutic strategies targeting the stroma, understanding and measuring tumor stroma mechanical properties has never been more timely or relevant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2c4fc7f9d47ee39dc0046b9648f8d487ad2a2f3" target='_blank'>
              Understanding and measuring mechanical signals in the tumor stroma.
              </a>
            </td>
          <td>
            Fàtima de la Jara Ortiz, Chiara Cimmino, Maurizio Ventre, Alessandra Cambi
          </td>
          <td>2024-11-10</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive subtype of breast cancer, with the worst prognosis among all subtypes. The impact of distinct cell subpopulations within the tumor microenvironment (TME) on TNBC patient prognosis has yet to be clarified.Utilizing single-cell RNA sequencing (scRNA-seq) integrated with bulk RNA sequencing (bulk RNA-seq), we applied Cox regression models to compute hazard ratios, and cross-validated prognostic scoring using a GLMNET-based Cox model. Cell communication analysis was used to elucidate the potential mechanisms of CPVL and MSR1. Ultimately, RNA interference-mediated gene knockdown was utilized to validate the impact of specific genes on the polarization of tumor-associated macrophages (TAMs).Our findings revealed that the function of immune cells is more pivotal in prognosis, with TAMs showing the strongest correlation with TNBC patient outcomes, compared with other immune cells. Additionally, we identified CPVL and MSR1 as critical prognostic genes within TAMs, with CPVL expression positively correlated with favorable outcomes and MSR1 expression associated with poorer prognosis. Mechanistically, CPVL may contribute to favorable prognosis by inhibiting the SPP1-CD44 ligand-receptor and promoting CXCL9-CXCR3, C3-C3AR1 ligand-receptor, through which TAMs interact with other cells such as monocytes, neutrophils, and T cells. Moreover, cytokines including IL-18, IFNγR1, CCL20, and CCL2, along with complement-related gene like TREM2 and complement component CFD, may participate in the process of CPVL or MSR1 regulating macrophage polarization. Furthermore, RT-PCR experiments confirmed that CPVL is positively associated with M1-like TAM polarization, while MSR1 is linked to M2-like TAM polarization. Finally, the prognostic significance of these two genes is also validated in HER2-positive breast cancer subtypes.CPVL and MSR1 are potential biomarkers for macrophage-mediated TNBC prognosis, suggesting the therapeutic potential of macrophage targeting in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af61ce193245e3b400806d4fcea4eed6dfe425f5" target='_blank'>
              Single-cell Atlas reveals core function of CPVL/MSR1 expressing macrophages in the prognosis of triple-negative breast cancer
              </a>
            </td>
          <td>
            Xinan Wang, Li Lin, Xue Zhang, Minghui Zhang, Zhuo Sun, Yichen Yang, Xiuna Zhang, Yonghui Yuan, Yong Zhang, Hao Chen, Ti Wen
          </td>
          <td>2024-12-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Metastatic disease represents a complex series of interaction between tumors and their surrounding microenvironment. Here we utilize model systems to define organ specific transcriptional signatures present in metastatic seeding. We have identified unique epigenetic programing necessary for the colonization of the liver or the lung by colorectal cancer. We have shown that manipulation of master regulators of these signatures in vitro and in vivo changes the metastatic potential of the tumor. In specific, we have found that native cytokines within the liver and lung are engaged by the metastatic tumor cells to promote this reprograming. In the case of the liver, we have identified hepatocyte released CCL2 acts on metastasizing tumors to promote colonization. Through in vitro and in vivo models, we showed that CCL2 engagement drives upregulation of the transcription factor TCF7 and subsequent enhancer reprograming to promote survival of the metastatic cells in the liver. In the lung, we have identified additional, unique secreted cytokines which lead to the upregulation of other TCF family members. Together this data identifies an important signaling nodes in colorectal cancer. Here we propose a model in which secreted factors by the host organ, such as CCL2, reprogram genes within the tumor cell to promote growth and survival of the metastatic cell in foreign microenvironments such as the lung or liver. Targeting these proteins may kill existing metastatic lesions as well as block new metastases from forming which will be essential in improving the outcomes of patients with oligometastatic and premetastatic disease.
 Citation Format: Charlie Niesen, Nathan Wu, Jonathan Rennhack. Microenvironmentally released cytokines mediate organ specific transcriptional profiles for metastatic colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c74823f81b8e3a451e0d9b48e3071c6de71312d7" target='_blank'>
              Abstract A008: Microenvironmentally released cytokines mediate organ specific transcriptional profiles for metastatic colorectal cancer
              </a>
            </td>
          <td>
            Charlie Niesen, Nathan Wu, Jonathan Rennhack
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Glioblastoma (GBM) is an aggressive primary brain tumor characterized by high treatment resistance and an extremely high rate of recurrence. To understand the mechanisms of treatment resistance and develop novel therapeutic approaches, we prepared 11 patient-derived human GBM cells and 5 mouse GBM cells from genetically engineered mouse models. We performed repeated irradiation for them to analyze the mechanisms of acquired resistance. During repeated irradiation, GBM cells enhanced radio- and chemo-resistance, increased invasiveness, and elevated ability to establish an immunosuppressive microenvironment. We identified upregulation of Collagen type I and III (COL I & III) as crucial for this adaptive resistance. COL I & III, rarely expressed in normal brain tissue but highly expressed in resistant GBM, emerged as potential markers for targeting therapy-resistant GBM tumors. To apply these findings to treatment, we focused on von Willebrand factor (vWF), which selectively binds to expose COL I & III in abnormal tissues. We developed CBD-IL-12 by conjugating vWF’s collagen-binding domain (CBD) into the immunostimulatory cytokine IL-12. In mouse models of therapy-resistant GBM, intravenously administered CBD-IL-12 accumulated specifically in tumor sites, without accumulating in normal brain tissue or other healthy organs. CBD-IL-12 enhanced dendritic cell recruitment, increased CD8+ T cell infiltration, and improved the immunosuppressive tumor environment, resulting in tumor suppression and significantly prolonged survival. Our findings suggest that extracellular matrix proteins have tumor-specific and abundant presence in the tumor microenvironment, indicating their potential application in targeted therapy. We demonstrated that therapy-resistant GBM cells exhibited upregulated expression of COL I & III, and CBD can be utilized to selectively deliver cytokines to tumor sites. This CBD-based immunotherapy approach shows promise in transforming the treatment landscape for recurrent GBM, offering a novel strategy to combat this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ce5c5d4d533b9da0d99dffd0cf1999877f2db23" target='_blank'>
              IL-4 IMMUNOTHERAPY TARGETING THE UNIQUE EXTRACELLULAR MATRIX IN RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Satoru Osuka
          </td>
          <td>2024-11-29</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cancers exhibit a remarkable ability to develop resistance to a range of treatments, often resulting in relapse following first-line therapies and significantly worse outcomes for subsequent treatments. While our understanding of the mechanisms and dynamics of the emergence of resistance during cancer therapy continues to advance, many questions remain about which treatment strategies can minimize the probability that resistance will evolve, thereby improving long-term patient outcomes. In this study, we present an evolutionary simulation model of a clonal population of cells that can acquire resistance mutations to one or more treatments. We then leverage this model to examine the efficacy of a two-strike “extinction therapy” protocol—in which two treatments are applied sequentially in an effort to first contract the population to a vulnerable state and then push it to extinction—in comparison to that of a combination therapy protocol. We investigate the impact of parameters such as the timing of the switch between the two strikes, the rate of emergence of resistant mutations, the dose of the applied drugs, the presence of cross-resistance, and whether resistance is a binary or a quantitative trait. Our results indicate that the timing of switching from the first to the second strike has a marked effect on the likelihood of driving the population to extinction, and that extinction therapy outperforms combination therapy when cross-resistance is present. We conduct an in silico trial that reveals more detailed insight into when and why a second strike will succeed or fail. Finally, we demonstrate that modeling resistance as a quantitative rather than binary trait does not change our overall conclusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f738c3dddcab46b26ff2c97c3888fefb1e9d2858" target='_blank'>
              Evolutionary rescue model informs strategies for driving cancer cell populations to extinction
              </a>
            </td>
          <td>
            Amjad Dabi, Joel S. Brown, Robert A Gatenby, Corbin D. Jones, Daniel R. Schrider
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="BACKGROUND
The majority of patients with prostate cancer (PCa) exhibit intrinsic resistance to immune checkpoint blockade (ICB) following radiotherapy (RT). This resistance is generally attributed to the limited antigen presentation of heterogeneous cells within tumors. Here, we aimed to isolate and characterize these diverse subgroups of tumor post-RT to understand the molecular mechanisms of their resistance to ICB.


METHODS
Single-cell RNA-sequencing (scRNA-seq) was used to profile senescent cancer cell clusters induced by RT in LNCaP cells. The expression and phosphorylation levels of ataxia telangiectasia and Rad3-related protein (ATR) were assessed by immunohistochemistry in clinical samples from patients with or without RT. Co-immunoprecipitation, mutagenesis, and Western blotting were used to measure the interactions between proteins. Xenograft experiments were performed to assess the tumor immune response in the mice.


RESULTS
We identified a subset of PCa cells that exhibited resistance to RT, characterized by a reduced antigen presentation capability, which enhanced their ability to evade immune detection and resist cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) blockade. scRNA-seq revealed that the senescent state was a transient phase of PCa cells post-RT, particularly in CTLA-4 blockade treatment-resistant cells. This state was marked by increased cytosolic ATR level. Cytosolic ATR phosphorylated CD86 in its cytosolic domain and enhanced the interaction between CD86 and its E3 ligase MARCH1 through electrostatic attraction. Depletion or inhibition of Atr increased the sensitivity to immune attack and improved responses to anti-Ctla-4 antibody treatment in a mouse model.


CONCLUSIONS
Our findings indicate that the activation of cytosolic ATR, which is associated with cellular senescence, impedes the effectiveness of combined RT and ICB treatments. This discovery may provide valuable insights for improving the efficacy of combined RT and ICB therapies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54111804377a9c0bbc1679bc34e9106d5b8ef9e5" target='_blank'>
              Radiotherapy-resistant prostate cancer cells escape immune checkpoint blockade through the senescence-related ataxia telangiectasia and Rad3-related protein.
              </a>
            </td>
          <td>
            Chenyi Shao, Yingyi Zhang, Hang Li, Jiajia Chen, Ting Huang, Jiaze Li, Simeng Wen, Sen Wang, Saijun Fan, Yu Zhao
          </td>
          <td>2024-12-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction Macrophages and T cells play crucial roles in liver physiology, but their functional diversity in hepatocellular carcinoma (HCC) remains largely unknown. Methods Two bulk RNA-sequencing (RNA-seq) cohorts for HCC were analyzed using gene co-expression network analysis. Key gene modules and networks were mapped to single-cell RNA-sequencing (scRNA-seq) data of HCC. Cell type fraction of bulk RNA-seq data was estimated by deconvolution approach using single-cell RNA-sequencing data as a reference. Survival analysis was carried out to estimate the prognosis of different immune cell types in bulk RNA-seq cohorts. Cell-cell interaction analysis was performed to identify potential links between immune cell types in HCC. Results In this study, we analyzed RNA-seq data from two large-scale HCC cohorts, revealing a major and consensus gene co-expression cluster with significant implications for immunosuppression. Notably, these genes exhibited higher enrichment in liver macrophages than T cells, as confirmed by scRNA-seq data from HCC patients. Integrative analysis of bulk and single-cell RNA-seq data pinpointed SPP1 + macrophages as an unfavorable cell type, while VCAN + macrophages, C1QA + macrophages, and CD8 + T cells were associated with a more favorable prognosis for HCC patients. Subsequent scRNA-seq investigations and in vitro experiments elucidated that SPP1, predominantly secreted by SPP1 + macrophages, inhibits CD8 + T cell proliferation. Finally, targeting SPP1 in tumor-associated macrophages through inhibition led to a shift towards a favorable phenotype. Discussion This study underpins the potential of SPP1 as a translational target in immunotherapy for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d7601947ef3a8c3102650d556bda64571da461a" target='_blank'>
              Identification of SPP1 + macrophages as an immune suppressor in hepatocellular carcinoma using single-cell and bulk transcriptomics
              </a>
            </td>
          <td>
            Han Jin, Woonghee Kim, Meng Yuan, Xiangyu Li, Hong Yang, Mengzhen Li, Mengnan Shi, Hasan Turkez, Mathias Uhlén, Cheng Zhang, Adil Mardinoglu
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b996ba4302eed98cf594d704cd4db5866f47b20" target='_blank'>
              Circulating tumor cell plasticity determines breast cancer therapy resistance via neuregulin 1-HER3 signaling.
              </a>
            </td>
          <td>
            Roberto Würth, E. Donato, Laura Michel, Massimo Saini, Lisa Becker, Tasneem Cheytan, Daria Doncevic, Tobias Messmer, E. Gutjahr, Rebecca Weber, C. Klein, Hamed Alborzinia, Umut Yildiz, V. Vogel, M. Hlevnjak, Polina Kozyulina, Sarah-Jane Neuberth, Paul Schwerd-Kleine, Sevinç Jakab, Nicole Pfarr, A. Angeles, Astrid K Laut, Darja Karpova, M. Falcone, O. Hardt, Benjamin Theek, Celina V. Wagner, M. Becker, Sabine Wagner, Martina Haselmayr, Anita Schmitt, Carsten Müller-Tidow, S. Riethdorf, Klaus Pantel, M. Zapatka, Holger Sültmann, Carl Herrmann, V. Thewes, Peter Lichter, A. Schneeweiss, M. Sprick, A. Trumpp
          </td>
          <td>2025-01-03</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Introduction Cancer-associated fibroblasts (CAFs) are a diverse group of cells that significantly contribute to reshaping the tumor microenvironment (TME), and no research has systematically explored the molecular landscapes of senescence related CAFs (senes CAF) in NB. Methods We utilized pan-cancer single cell and spatial transcriptomics analysis to identify the subpopulation of senes CAFs via senescence related genes, exploring its spatial distribution characteristics. Harnessing the maker genes with prognostic significance, we delineated the molecular landscapes of senes CAFs in bulk-seq data. We established the senes CAFs related signature (SCRS) by amalgamating 12 and 10 distinct machine learning (ML) algorithms to precisely diagnose stage 4 NB and to predict prognosis in NB. Based on risk scores calculated by prognostic SCRS, patients were categorized into high and low risk groups according to median risk score. We conducted comprehensive analysis between two risk groups, in terms of clinical applications, immune microenvironment, somatic mutations, immunotherapy, chemotherapy and single cell level. Ultimately, we explore the biological function of the hub gene JAK1 in pan-cancer multi-omics landscape. Results Through integrated analysis of pan-cancer spatial and single-cell transcriptomics data, we identified distinct functional subgroups of CAFs and characterized their spatial distribution patterns. With marker genes of senes CAF and leave-one-out cross-validation, we selected RF algorithm to establish diagnostic SCRS, and SuperPC algorithm to develop prognostic SCRS. SCRS demonstrated a stable predictive capability, outperforming the previously published NB signatures and clinic variables. We stratified NB patients into high and low risk group, which showed the low-risk group with a superior survival outcome, an abundant immune infiltration, a different mutation landscape, and an enhanced sensitivity to immunotherapy. Single cell analysis reveals biologically cellular variations underlying model genes of SCRS. Spatial transcriptomics delineated the molecular variant expressions of hub gene JAK1 in malignant cells across cancers, while immunohistochemistry validated the differential protein levels of JAK1 in NB. Conclusion Based on multi-omics analysis and ML algorithms, we successfully developed the SCRS to enable accurate diagnosis and prognostic stratification in NB, which shed light on molecular landscapes of senes CAF and clinical utilization of SCRS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/900ec2c5a181f8ad61d932465735fe6366505e3c" target='_blank'>
              Pan-cancer single cell and spatial transcriptomics analysis deciphers the molecular landscapes of senescence related cancer-associated fibroblasts and reveals its predictive value in neuroblastoma via integrated multi-omics analysis and machine learning
              </a>
            </td>
          <td>
            Shan Li, Junyi Luo, Junhong Liu, Dawei He
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Invasive lobular carcinoma (ILC) is the most common special histological subtype of breast cancer, after No Special Type (NST, ductal carcinoma/IDC), and affects 10-15% of all breast cancer patients. ILC is characterized by E-cadherin loss, Estrogen Receptor (ER) positivity, and a distinct single file invasive growth pattern associated with a unique pattern of metastasis. ILC metastasizes to common breast cancer sites like bone and lung, but ILC also spreads to gynecologic tissues, the peritoneum, gastrointestinal tract and central nervous system - with enrichment for orbital and leptomeningeal metastases. In vitro and in vivo models of ILC metastasis have not been extensively developed. Models of spontaneous metastasis from ER+ breast cancer overall are critically limited; ER+ ILC metastases models present only micro-metastatic disease with extremely long latency (>9 months). Together these issues limit research on the unique nature of metastatic ILC. We leveraged mammary intraductal (MIND) xenografting with two ER+ ILC cell lines (MM134 and 44PE) and confirmed prior reports that these lines spontaneously metastasize to bone, brain, ovary, uterus, lung, and adrenal glands with up to 100% incidence. Importantly, we find that supplementation with low dose estradiol (E2) in drinking water rapidly advances metastatic progression, but not primary tumor growth. Using ex vivo bioluminescence imaging (BLI), we find BLI+ lesions are detectable as early as 90 days. At 150 days post-challenge, 100% of mice have BLI+ metastases at multiple sites. Beyond 22 weeks, 100% of mice present with symptomatic and ultimately lethal CNS metastases. BLI+ macro-metastatic lesions were isolated and expanded ex vivo as organotrophic cell line variants. Analysis of RNA-seq data is ongoing to evaluate 1) common factors enriched in metastatic ILC, and 2) organotrophic factors and pathways enriched in specific metastatic sites. In patients with ILC, bone metastases are typically the first site of metastatic development, occur at a higher incidence versus NST, and are associated with increased morbidity. Bone metastases from ILC patients are more osteogenic compared to the characteristically osteolytic lesions of NST. To examine how ILC cells may promote osteoblastic responses, we prepared ILC and NST conditioned media (CM) and applied it to differentiating osteoblasts (OB). We observed increased OB differentiation and increased mineral deposition with ILC CM compared to NST CM, supporting that ILC cell lines have a distinct impact on the bone microenvironment which may manifest in vivo yet be defined in vitro. Collectively, our observations support that ER+ ILC cell lines can recapitulate key features of clinical disease progression, including spontaneous metastasis mimicking clinical phenotypes. We are leveraging these models to better understand the ILC metastatic cascade and adaptation to unique metastatic niches, and to identify shared and unique attributes of ILC metastases that can be exploited for advancing therapeutics.
 Citation Format: Joseph L. Sottnik, Matthew J. Sikora. Development and characterization of novel models of lobular breast cancer metastases [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d088f59a61acb161e790ad277b46fc6a05ff1de" target='_blank'>
              Abstract A010: Development and characterization of novel models of lobular breast cancer metastases
              </a>
            </td>
          <td>
            Joseph L. Sottnik, Matthew J. Sikora
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Background: Metastatic prostate cancer is an androgen receptor (AR) driven disease for which multiple AR targeted therapies are FDA approved, leading to improvements in patient outcomes. However, development of treatment resistance remains universal, driven by AR alterations as well as lineage state transitions that bypass AR signaling and culminate in neuroendocrine prostate cancer (NEPC) with poor prognosis. Liquid biopsies could enable longitudinal molecular analysis to identify such lineage transitions, but bulk RNA sequencing of circulating tumor cells (CTCs) has been limited by the challenge of isolating sufficiently pure samples for global signaling pathway analysis. We developed a novel approach to CTC purification that results in purity comparable to tissue biopsies and performed the first large-scale CTC RNA-seq study in a multi-institutional cohort of patients with metastatic prostate cancer. Using this approach, we identified multiple prostate cancer lineage states associated with prognosis. Methods: 273 blood samples were collected from 117 patients with histologically confirmed metastatic prostate cancer treated at the University of Wisconsin Carbone Cancer Center, William S. Middleton Memorial Veterans Hospital, UC San Diego Moores Cancer Center and Dana Farber Cancer Institute. CTCs were isolated with our automated microfluidic technology integrating negative and positive selection for CTC enrichment. CTCs were captured immunomagnetically followed by RNA isolation on chip and RNA sequencing. A modification of the ESTIMATE algorithm was used to infer sample tumor purity, and CIBERSORTx was used to infer immune content. Overall survival (OS) was defined as date of death or last contact relative to first CTC sample collection. Results: 146 samples from 70 patients met our >50% purity threshold for gene expression pathway analysis. Single sample pathway analysis identified four CTC transcriptional phenotypes: luminal A (high AR signaling, intermediate proliferation), luminal B (high AR signaling, high proliferation), low proliferation, and neuroendocrine. Compared to patients with low CTC burden/low tumor purity (median OS NR, n=63), patients with luminal A (n=21) and low proliferation (n=12) phenotypes had similar survival, while patients with luminal B (n=18) and NE (n=3) had markedly shorter survival (LumB: median OS 6 months, HR 9.1, log rank p<0.0001, NE: median OS 3.7 months, HR 11.8, log rank p=0.0019). Conclusion: We report the largest CTC RNA sequencing cohort of patients with metastatic prostate cancer and demonstrate that CTC RNA-seq identifies prostate cancer lineage states mirroring those described in tissue profiling. In addition to an NE phenotype concordant with tissue histology and associated with markedly inferior overall survival, we identified a more common luminal B CTC subtype defined by persistent AR signaling and high proliferation and associated with poor prognosis comparable to NEPC.
 Citation Format: Marina N Sharifi, Jamie M Sperger, Amy K Taylor, Katharine E Tippins, Shannon R Reese, Viridiana Carreno, Katherine R Kaufmann, Alex H Chang, Luke A Nunamaker, Charlotte Linebarger, Kyle T Helzer, Matthew Bootsma, Grace C Blitzer, John Floberg, David Kosoff, Rana R McKay, Xiao X Wei, Shuang G Zhao, Joshua M Lang. High purity CTC RNA sequencing identifies poor prognosis lineage states in castrate resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr PR010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e0c2f62cdccd306207d39b4e5d134a71a27ab85" target='_blank'>
              Abstract PR010: High purity CTC RNA sequencing identifies poor prognosis lineage states in castrate resistant prostate cancer
              </a>
            </td>
          <td>
            Marina N. Sharifi, J. Sperger, Amy K. Taylor, Katharine E Tippins, Shannon R. Reese, Viridiana Carreno, K. Kaufmann, Alex H. Chang, Luke A Nunamaker, Charlotte Linebarger, K. Helzer, M. Bootsma, G. Blitzer, John Floberg, D. Kosoff, Rana R McKay, Xiao X Wei, Shuang G. Zhao, Joshua M Lang
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d4ffa627a25fa61de5497018c98a78bbf8db341" target='_blank'>
              Application of deep learning models on single-cell RNA sequencing analysis uncovers novel markers of double negative T cells
              </a>
            </td>
          <td>
            Tian Xu, Qin Xu, Ran Lu, David Oakland, Song Li, Liwu Li, Christopher M Reilly, Xin Luo
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract We performed comprehensive histopathologic, genomic, and epigenomic evaluation of paired initial and recurrent glioblastoma specimens from 106 patients to investigate the molecular evolution and cellular phenotypes underlying differential treatment responses. While TERT promoter mutation and CDKN2A homozygous deletion were early events during gliomagenesis shared by initial and recurrent tumors, most other recurrent genetic alterations (e.g., EGFR, PTEN, NF1) were commonly private to initial or recurrent tumors indicating acquisition later during clonal evolution. Further, glioblastomas exhibited heterogeneous epigenomic evolution with subsets becoming more globally hypermethylated, hypomethylated, or remaining stable. Glioblastoma that underwent sarcomatous transformation at recurrence had inferior survival and were significantly enriched in NF1, TP53, and RB1 alterations and the Mesenchymal epigenetic class. Patients who developed somatic hypermutation following temozolomide treatment had significantly longer interval to disease recurrence and prolonged overall survival, and hypermethylation at four specific CpG sites in the promoter region of MGMT was significantly associated with this development of hypermutation. Finally, an epigenomic evolution signature incorporating change in DNA methylation levels across 347 critical CpG sites was developed that significantly correlated with clinical outcomes. This talk will describe how GBM undergoes heterogeneous genetic, epigenetic, and cellular evolution that underlies prognostically different treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/704f3941c3c62c19e62495b9dd1824a7c49e3146" target='_blank'>
              IL-1 IDHWT GLIOBLASTOMA LONGITUDINAL MOLECULAR EVOLUTION
              </a>
            </td>
          <td>
            C. G. Lucas, N. Al-Adli, J. S. Young, R. Gupta, R. Morshed, J. Wu, A. Ravindranthan, A. Shai, N. Oberheim Bush, J. W. Taylor, J. de Groot, J. Villanueva-Meyer, M. Pekmezci, A. Perry, A. W. Bolen, P. Theodosopoulos, M. Aghi, E. Chang, S. Hervey-Jumper, D. R. Raleigh, A. M. Molinaro, J. Costello, A. A. Diaz, J. Clarke, N. Butowski, J. J. Phillips, S. M. Chang, M. S. Berger, D. A. Solomon
          </td>
          <td>2024-11-29</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Resistance to BRAF and MAPK inhibitors is a significant challenge in melanoma treatment, driven by adaptive and acquired mechanisms that allow tumour cells to evade therapy. Here, we examined early signalling responses to single and combined BRAF and MAPK inhibition in a BRAFV600E, drug-sensitive melanoma cell line and a drug-resistant ARID1A-knockout (KO) derivative. ARID1A, frequently mutated in melanoma, is associated with resistance and immune evasion. Using an innovative systems biology approach that integrates transcriptomics, proteomics, phosphoproteomics, and functional kinomics through matrix factorization and network analysis, we identified key signalling alterations and resistance mechanisms. We found that ARID1A-KO cells exhibited transcriptional rewiring, sustaining MAPK1/3 and JNK activity post-treatment, bypassing feedback sensitivity observed in parental cells. This rewiring suppressed PRKD1 activation, increased JUN activity—a central resistance network node—and disrupted PKC dynamics through elevated basal RTKs (e.g., EGFR, ROS1) and Ephrin receptor activity post-treatment. ARID1A mutations also reduced HLA-related protein expression and enriched extracellular matrix components, potentially limiting immune infiltration and reducing immunotherapy efficacy. Our graph-theoretical multi-omics approach uncovered novel resistance-associated signalling pathways, identifying PRKD1, JUN, and NCK1 as critical nodes. While receptor activation redundancies complicate single-target therapies, they also present opportunities for combination strategies. This study highlights ARID1A’s role in reshaping signalling and immune interactions, offering new insights into melanoma resistance mechanisms. By identifying actionable targets, including JUN and immune pathways, we provide a foundation for developing integrated therapeutic strategies to overcome resistance in BRAF/MAPK inhibitor-treated melanoma. One sentence summary This study reveals how ARID1A-mediated transcriptional rewiring drives resistance to MAPK inhibitors in melanoma by altering signalling pathways, immune interactions, and receptor dynamics, highlighting potential targets for combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bdb990bb1257653b7e587028f560dd7ef66b129" target='_blank'>
              ARID1A-induced transcriptional reprogramming rewires signalling responses to drug treatment in melanoma
              </a>
            </td>
          <td>
            Charlie George Barker, Sumana Sharma, Ana Mafalda Santos, Konstantinos-Stylianos Nikolakopoulos, Athanassios D. Velentzas, Franziska Völlmy, A. Minia, V. Pliaka, M. Altelaar, Gavin J Wright, Leonidas G. Alexopoulos, D. Stravopodis, E. Petsalaki
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Background: Androgen-targeted therapies are a mainstay of treatment for metastatic prostate cancer and have significantly improved patient outcomes. However, development of treatment resistance remains universal, occurring through androgen receptor (AR) alterations driving constitutive AR signaling, or lineage state transitions that bypass AR entirely and culminate in a small cell/neuroendocrine phenotype (NEPC). Cell surface targeted therapies, including antibody drug conjugates and targeted radioligand therapies, represent a new therapeutic class that has shown promise for treatment of androgen-resistant metastatic prostate cancer (mCRPC). Circulating tumor cells (CTCs) present an accessible source of tumor material to identify expression of known and novel targets for therapeutic development, as well as the unique potential for longitudinal profiling to understand the evolution of target expression and association with clinical resistance to cell surface targeted therapies. We have developed a CTC platform allowing for gene expression and protein profiling of cell surface targets on CTCs. Methods: Live CTCs were isolated with our automated microfluidic technology integrating negative and positive selection for CTC enrichment with a cohort of 273 samples. CTCs were captured immunomagnetically followed by RNA isolation on chip and RNA-seq, or protein staining on chip for single cell fluorescent quantification of prostate adenocarcinoma and NEPC cell surface targets including PSMA, TROP2, B7H3 and DLL3. Results: CTC RNA sequencing can be used to understand the expression of cell surface targets across different CTC phenotypes. In CTCs from patient with mCRPC, canonical prostate adenocarcinoma cell surface targets STEAP1, KLK2 and FOLH1 and epithelial cell surface target TACSTD2 had the highest expression, while prostate adenocarcinoma target STEAP2, pan-tumor target ERBB2, and tumor immune checkpoint cell surface protein CD276 (B7H3) had intermediate expression. Expression of DLL3 and SSTR2, targets associated with neuroendocrine differentiation, was lowest, though still detected in a subset of CRPC CTCs. Comparison of expression levels between CRPC and NEPC CTCs demonstrated lower expression of most adenocarcinoma cell surface targets in NEPC CTCs as expected, and a trend towards higher expression of DLL3 and SSTR2. Cell surface expression is being assessed for proteins including Trop-2, PSMA and DLL3. Evaluation of DLL3 at the protein level in a subset of patients including both CRPC (n=13) and NEPC (n=12) confirmed that DLL3 was detected in both adenocarcinoma and neuroendocrine CTCs, with a median of 45.8% DLL3-positive CTCs per sample. Conclusions: Biomarkers are needed to better predict response and resistance to targeted therapies. CTC RNAseq and single CTC phenotyping hold potential for predictive and on-treatment biomarkers of response to cell surface targeted therapies. Further study of mechanisms driving response and resistance can be evaluated with these biomarkers.
 Citation Format: Jamie M. Sperger, Marina N. Sharifi, Katherine R. Kaufmann, Matthew L. Bootsma, Shannon R. Reese, Viridiana Carreno, Alex H. Chang, Luke A. Nunamaker A. Nunamaker, Charlotte Linebarger, Amy K. Taylor, Katherine E Tippins, Kyle T. Helzer, Shuang G. Zhao, Joshua M Lang. Molecular assessment of cell surface targets using integrated circulating tumor cell (CTC) RNAseq and single CTC phenotyping [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B042.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c55829197c0d705fe0f09e3fa114fffa4e135df" target='_blank'>
              Abstract B042: Molecular assessment of cell surface targets using integrated circulating tumor cell (CTC) RNAseq and single CTC phenotyping
              </a>
            </td>
          <td>
            J. Sperger, Marina N. Sharifi, K. Kaufmann, M. Bootsma, Shannon R. Reese, Viridiana Carreno, Alex H. Chang, Luke A Nunamaker, A. Nunamaker, Charlotte Linebarger, Amy K. Taylor, Katherine E Tippins, K. Helzer, Shuang G. Zhao, Joshua M Lang
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ea22d01029a12589e7d1611f97b97b5a3d769f" target='_blank'>
              Novel integrated multiomics analysis reveals a key role for integrin beta-like 1 in wound scarring.
              </a>
            </td>
          <td>
            Sangeun Kim, Ryota Noda, Yu-Chen Liu, Y. Nakajima, Shoichiro Kameoka, Daisuke Motooka, S. Mizuno, Satoru Takahashi, Kento Takaya, Takehiko Murase, K. Ikematsu, Katsiaryna Tratsiakova, Takahiro Motoyama, Masahiro Nakashima, Kazuo Kishi, Paul Martin, S. Seno, D. Okuzaki, Ryoichi Mori
          </td>
          <td>2024-11-18</td>
          <td>EMBO reports</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="
 Age is the single largest risk factor for the development of cancer, but how age impacts the molecular mechanisms that drive cancer remain poorly understood. While it is clear that age-related accumulation of cell autonomous mutations contributes to tumorigenesis, the central role age-related changes in the tumor microenvironment play in the transformation process is becoming more fully appreciated. Underscoring the importance of an aged microenvironment in cancer development are findings that senescent fibroblasts, which accumulate with age, directly stimulate preneoplastic and neoplastic cell growth and tumor progression. Investigations into how senescent fibroblasts promote tumorigenesis revealed that they express a plethora of growth factors, extracellular matrix remodeling enzymes, chemokines, and cytokines collectively referred to as the senescence associated secretory phenotype (SASP). Chemotherapy induces similar changes that can negatively impact a patient’s quality of life. We will discuss how these changes impact tumor progression and therapy-induced bone loss.
 Citation Format: Sheila A. Stewart. Age-related stromal changes drive breast cancer tumor progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr IA027.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a25dfba7fd100b42c0d41c8f8cf69167a46b14" target='_blank'>
              Abstract IA027: Age-related stromal changes drive breast cancer tumor progression
              </a>
            </td>
          <td>
            Sheila A Stewart
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: This research study aimed to assess the performance of a research use only (RUO) immunofluorescence (IF) assay targeting programmed death-ligand 1 (PD-L1) evolution in epithelial and/or mesenchymal circulating tumor cells (CTCs) isolated from blood using Parsortix® microfluidic technology. Upregulation of PD-L1 enables cancer cells to evade the host immune response. Assessment of PD-L1 status in the tumor, as determined by traditional tissue biopsy, can indicate if immunotherapy has the potential to be an effective treatment. However, PD-L1 expression in tumor biopsies may not reflect metastatic sites in their entirety and can become outdated during tumor evolution, as the process cannot be repeated due its invasive nature. Liquid biopsy offers a minimally invasive option for dynamic testing of PD-L1 in CTCs as patients undergo treatment. Methods: Analytical linearity, and specificity and sensitivity were assessed by spiking HCC1954 and Hs 578T cancer cell lines into the blood samples of 34 healthy volunteers. The assay was then evaluated in samples from 47 metastatic lung cancer patients, among which 32 had a known PD-L1 tissue status (19 positive, 13 negative). Up to six successive draws were taken for each patient, collected into Streck Cell-Free DNA blood collection tubes. ANGLE’s Parsortix® instruments were used to isolate spiked cancer cells or CTCs from blood samples based on their size and deformability. The CTC- enriched harvests were processed onto ANGLE’s CellKeepTM slides and IF stained with ANGLE’s Portrait® PD-L1 RUO assay for PD-L1 identification on CTCs. Slides were imaged on a BioView automated imaging system. Results: Analytical specificity of PD-L1 was 98% and analytical sensitivity was 81%. Analytical linearity (R2=0.91) was shown over a 0–500 cells range. In metastatic lung cancer patients, ≥1 CTC was identified in at least one draw for 91% of donors, with 55% of those donors exhibiting ≥1 PD-L1 positive CTC. A majority (81%) of donors exhibited a positive correlation between PD-L1 tissue status and CTC expression. However, the remaining 19% of donors exhibited discordance with a PD-L1 negative tissue status but PD-L1 positivity identified in ≥1 CTC. Of the PD-L1 positive CTCs identified, 98% expressed mesenchymal markers. Dynamic change was shown in all but one donor, with PD-L1 status of the CTCs changing over time. Conclusions: This study demonstrated the ability to determine PD-L1 status in CTCs from the blood of metastatic lung cancer patients and, subject to further study, the future potential to develop dynamic PD-L1 testing for advancement of more personalized cancer treatments. Patients with a PD-L1 negative tissue status exhibited PD-L1 positive CTCs, demonstrating the feasibility of a more repeatable and accurate assessment of PD-L1 status than can be achieved through a traditional tissue biopsy.
 Citation Format: Morgan Spode, Chloe Goodwin, Aarabhi Varatharajah, Aaron Cottingham, Elisha Duhig, Laura Kaja, David Greaves, Mariacristina Ciccioli, Anne-Sophie Pailhes-Jimenez. Investigating PD-L1 status in circulating tumor cells isolated from blood samples of lung cancer patients [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr A038.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df86246ea3d3196ce89a2f9b0641b71cf9edc35d" target='_blank'>
              Abstract A038: Investigating PD-L1 status in circulating tumor cells isolated from blood samples of lung cancer patients
              </a>
            </td>
          <td>
            Morgan Spode, Chloe Goodwin, Aarabhi Varatharajah, A. Cottingham, Elisha Duhig, Laura Kaja, David Greaves, M. Ciccioli, A. Pailhes-Jimenez
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="According to the principles of Evolutionary Cancer Cell Biology (ECCB), cancer stem cells (CSCs) do not derive from normal stem cells; rather, they originate from a distinct functional phenotype of germline cells characterized by asymmetric cell division (ACD). This phenotype proliferates through ACD, producing self-renewing cells alongside non-proliferating daughter cells with CSC qualities. ECCB posits that CSCs do not proliferate. Similar to protists, there exists a close reciprocal relationship between sister cells that collectively form a germline and stem cells. These sister cells perform complementary roles: proliferating cancer germline cells generate stem cells, whereas the non-proliferating CSCs give rise to progenitor cells for the formation of new germline clones. ECCB distinguishes between primary CSCs, which are associated with carcinogenesis and primary tumors, and secondary CSCs, which are linked to metastases. This unicellular stem cell system is homologous to that of parasitic protists, such as amebae. Both CSCs and amebae stem cells are produced by an oxygen-sensitive germline and are vulnerable to damage when oxygen levels exceed 6.0% (germline hyperoxia), as elevated oxygen concentrations can harm the germline genome. Germline cells that lose their stemness quality continue to cycle through defective symmetric cell divisions (DSCD). However, to restore functionality, the DSCD genome must be repaired through hyperpolyploidization. This process occurs in native polyploid giant cancer cells, which are homologous to the multinucleated genome repair structures found in protists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22fd9b25c7727535490357470b16f77d5c7c81c2" target='_blank'>
              Reevaluating cancer stem cells and polyploid giant cancer cells from the evolutionary cancer cell biology perspective
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2024-11-21</td>
          <td>Cancer Plus</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a hematological malignancy characterized by the clonal proliferation of malignant plasma cells within the bone marrow. The disease’s complexity is underpinned by a variety of genetic and molecular abnormalities that drive its progression. Methods This review was conducted through a state-of-The-art literature search, primarily utilizing PubMed to gather peer-reviewed articles. We focused on the most comprehensive and cited studies to ensure a thorough understanding of the genetic and molecular landscapes of MM. Results We detail primary and secondary alterations such as translocations, hyperdiploidy, single nucleotide variants (SNVs), copy number alterations (CNAs), gene fusions, epigenetic modifications, non-coding RNAs, germline predisposing variants, and the influence of the tumor microenvironment (TME). Our analysis highlights the heterogeneity of MM and the challenges it poses in treatment and prognosis, emphasizing the distinction between driver mutations, which actively contribute to oncogenesis, and passenger mutations, which arise due to genomic instability and do not contribute to disease progression. Conclusion & Future Perspectives We report key controversies and challenges in defining the genetic drivers of MM, and examine their implications for future therapeutic strategies. We discuss the importance of systems biology approaches in understanding the dependencies and interactions among these alterations, particularly highlighting the impact of double and triple-hit scenarios on disease outcomes. By advancing our understanding of the molecular drivers and their interactions, this review sets the stage for novel therapeutic targets and strategies, ultimately aiming to improve clinical outcomes in MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec433a1270b41378c5571d49631b7d920ddc502b" target='_blank'>
              The Genetic and Molecular Drivers of Multiple Myeloma: Current Insights, Clinical Implications, and the Path Forward
              </a>
            </td>
          <td>
            Meghana Ram, Molly Fraser, Júnia Vieira dos Santos, R. Tasakis, Ariana Islam, Jannah Abo-Donia, Samir Parekh, A. Laganà
          </td>
          <td>2024-12-01</td>
          <td>Pharmacogenomics and Personalized Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract
 Round cell soft tissue tumors (RCSTTs) are a diverse group of aggressive malignancies that share overlapping clinical, morphological, and immunohistochemical features, posing significant diagnostic challenges. Recent advances in molecular pathology have revolutionized the understanding and classification of these tumors by identifying tumor-specific genetic alterations. This review highlights the role of molecular insights in complementing histopathology and immunohistochemistry to achieve precise diagnosis, risk stratification, and therapeutic decision-making. Key subtypes, including Ewing sarcoma, CIC- and BCOR-rearranged sarcomas, alveolar rhabdomyosarcoma, desmoplastic small round cell tumor, and extraskeletal myxoid chondrosarcoma, are discussed with emphasis on their molecular signatures and clinical significance. The advent of techniques such as fluorescence in situ hybridization (FISH), reverse transcriptase-polymerase chain reaction (RT-PCR), and next-generation sequencing (NGS) has enabled detection of genetic fusions and aberrations critical for tumor identification and targeted therapies. Integration of molecular pathology with traditional diagnostic approaches is essential for improving diagnostic accuracy, prognostication, and therapeutic strategies, paving the way for personalized medicine in the management of RCSTTs. Full form of ABBREVIATIONS are provided at the end.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4697a589f2a69b32b9998c7bb05dd3c97ac1fa8" target='_blank'>
              Beyond Immunuhistochemistry: Molecular Insights Into Round Cell Soft Tissue Tumors
              </a>
            </td>
          <td>
            Prashant Kumar Madoori, Sujatha Tejavat
          </td>
          <td>2024-12-30</td>
          <td>Perspectives in Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [15, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>