<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-07 09:32:03 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c056a7bc97d38004639e645962dc9ff9c3cfad" target='_blank'>
              Modeling cellular influence delineates functionally relevant cellular neighborhoods in primary and metastatic pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yeonju Cho, Jae W. Lee, Sarah M. Shin, Alexei Hernandez, Xuan Yuan, Jowaly Schneider, Jody E. Hooper, Laura D. Wood, Elizabeth M. Jaffee, Atul Deshpande, W. Ho
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is characterized by high malignancy and early propensity for metastasis, and modest response to immunotherapy due to the immunosuppressive microenvironment. Surgical intervention has shown benefits in treating early-stage SCLC. However, most patients experience recurrence after surgery. The factors associated with relapse free survival in these patients remain unclear. We collected operation specimens from ten early-stage SCLC patients (N0M0), conducted long-term follow-up, and grouped them based on disease status. Subsequently, we performed a retrospective analysis using single-cell spatial imaging mass cytometry to explore the characteristics of tumor cells and differences in the tumor microenvironment, especially the single-cell constitute of immune cells, between the two groups. We found that, in early-stage SCLC, tumor cells display pronounced heterogeneity, both intra-group and inter-group. Patients with early recurrence are characterized by a distinct subpopulation of tumor cells with high Ki-67 expression. Non-relapse patients demonstrate better infiltration of M1 macrophages and stromal cells. Neighborhood analysis suggested that positive interactions between macrophages, stromal cells, and T cells with tumor cells may benefit patient prognosis. Additionally, recurrent tumor cells might enhance their metastatic capacity and remodel the microenvironment through upregulation of GranzymeB or reduction of c-Myc expression. In conclusion, SCLC tumor cells demonstrate tumor heterogeneity and microenvironmental changes in the early clinical stages. A higher proportion of M1 macrophages is associated with prolonged postoperative survival in early-stage SCLC patients. This research provides novel insights and evidence for treating and preventing postoperative recurrence in SCLC. Supplementary Information The online version contains supplementary material available at 10.1007/s10565-025-10056-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1755c9b716477f70da2c740f51d015cee2a36b" target='_blank'>
              Single-cell spatial proteomics of non-relapse small cell lung cancer identifies tumor microenvironment determinants of survival
              </a>
            </td>
          <td>
            Yin Li, L. Xia, Hui Wang, Xinghao Ai, Ying Wang, Qingquan Luo, Yuchen Han, Shun Lu, Xinghua Cheng
          </td>
          <td>2025-06-17</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Over the past years, cancer research has transitioned from a ‘cancer cell-centered’ focus to a more integrative view of tumors as dynamic ecosystems. This paradigm shift emphasizes the tumor microenvironment (TME) as a complex network of interacting cellular and acellular components, where tumor cells orchestrate a supportive environment that facilitates progression, metastasis, and immune evasion. Understanding the spatial organization of these components within the TME is crucial, as the positioning and interactions between cancerous and non-cancerous cells significantly influence tumor behavior and therapy response. Spatial proteomics has emerged as a powerful tool for TME analysis, enabling the detection and quantification of proteins within intact tissue architecture at subcellular resolution. This approach provides insights into cellular interactions, signaling pathways, and functional states, facilitating the discovery of novel biomarkers and therapeutic targets linked to specific tissue regions and cellular contexts. Translating spatial proteomics into clinical practice requires overcoming challenges related to technology refinement, standardization of workflows, and adaptation to routine pathology settings. Melanoma is an aggressive, highly immunogenic malignancy with variable response rates to existing immunotherapies. Given that over half of patients treated with immune checkpoint inhibitors (ICIs) fail to respond or experience disease progression, the identification of novel biomarkers and therapeutic targets to enhance current therapies is urgently required. Spatial imaging technologies are increasingly being utilized to dissect the complex interplay between stroma, melanoma, and immune cell types within the TME to address this need. This review examines key spatial proteomics methods, their applications in melanoma biology, and associated image analysis pipelines. We highlight the current limitations, and future directions, emphasizing the potential for clinical translation to guide personalized treatment strategies, inform prognosis, and predict therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1eb1624eaa05c3beff7c464020ea5f0d816374c" target='_blank'>
              Spatial proteomics of the tumor microenvironment in melanoma: current insights and future directions
              </a>
            </td>
          <td>
            Chiara Bungaro, Michele Guida, Benedetta Apollonio
          </td>
          <td>2025-05-15</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bc89e4de14d8f34f2814e7cf735899a3c934813" target='_blank'>
              Spatial determinants of tumor cell dedifferentiation and plasticity in primary cutaneous melanoma
              </a>
            </td>
          <td>
            Tuulia Vallius, Yingxiao Shi, Edward Novikov, Shishir M Pant, R. Pelletier, Yu-An Chen, Juliann B. Tefft, Ajit Nirmal Johnson, Z. Maliga, Guihong Wan, George F. Murphy, S. Santagata, Yevgeniy R. Semenov, David Liu, C. G. Lian, P. Sorger
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a7baab96b7e0361b2e10254d2ab098fe969a9" target='_blank'>
              Tumor Cells Enriched for Interferon and Inflammatory Programs Pre-Exist in High Grade Serous Ovarian Cancer and are Proportionately Significantly Increased Post Chemotherapy
              </a>
            </td>
          <td>
            B. Winterhoff, Nomeda Girnius, Muyi Liu, Jaeyoung Kang, Mihir Shetty, M. A. Martinez-Gakidis, Weiyi Xu, Mayura Thomas, Linna Lahmadi, Aylin Z. Henstridge, Pilar Baldominos, Shobhana Talukdaer, Z. Chang, Jordan Mattson, Morgan Gruner, Sally A. Mullany, Peter Argenta, Gottfried E. Konecny, Dennis J. Slamon, Ronny Drapkin, Andrew C. Nelson, Timothy K. Starr, Joan S Brugge
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="A comprehensive understanding of cellular behavior and response to the tumor microenvironment (TME) in colorectal cancer (CRC) remains elusive. Here, we introduce the high-definition Visium spatial transcriptomic technology (Visium HD) and investigate formalin-fixed paraffin-embedded human CRC samples (n = 5). We demonstrate the high sensitivity, single-cell-scale resolution and spatial accuracy of Visium HD, generating a highly refined whole-transcriptome spatial profile of CRC samples. We identify transcriptomically distinct macrophage subpopulations in different spatial niches with potential pro-tumor and anti-tumor functions via interactions with tumor and T cells. In situ gene expression analysis validates our findings and localizes a clonally expanded T cell population close to macrophages with anti-tumor features. Our study demonstrates the power of high-resolution spatial technologies to understand cellular interactions in the TME and paves the way for larger studies that will unravel mechanisms and biomarkers of CRC biology, improving diagnosis and disease management strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f42a49dad6477b432ac875de1bc87a28316b76b" target='_blank'>
              High-definition spatial transcriptomic profiling of immune cell populations in colorectal cancer
              </a>
            </td>
          <td>
            Michelli F Oliveira, J. P. Romero, Meii Chung, Stephen R. Williams, Andrew D. Gottscho, Anushka Gupta, Susan E. Pilipauskas, Seayar Mohabbat, Nandhini Raman, David Sukovich, David Patterson, Sarah E B Taylor
          </td>
          <td>2025-06-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b426c5f702b08645ac569d89b1da888e84c491" target='_blank'>
              Deciphering the cellular tumor microenvironment landscape in salivary gland carcinomas using multiplexed imaging mass cytometry
              </a>
            </td>
          <td>
            M. Mayer, L. Nachtsheim, L. Jansen, P. Wolber, M. Schmiel, A. Quaas, J. P. Klussmann, C. Arolt
          </td>
          <td>2025-05-13</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Identification of the consensus molecular subtypes (CMS) opened significant potential for understanding the tumor biology and intertumoral heterogeneity of colorectal cancer (CRC). However, molecular subtyping in CRC traditionally relies on bulk transcriptomics, therefore, lacks spatial and single-cell level aspect. Methods We constructed tissue microarrays using tumor cores from 222 CRC patients. Arrays were stained and imaged using 54 cell identity and cancer hallmark markers, delivering spatially resolved protein profiles of >2 million cells. RNA sequencing data and CMS classification were also available for these patients. After segmentation of cancer, stromal and immune cells, we investigated intratumoral heterogeneity within CMS subtypes using spatially resolved single-cell protein profiling (>2 million cells). We compared cell types, their spatial organization and their expression of cancer hallmark-related proteins in CMS 1-4 subtypes. Results We revealed tissue atlases illustrating the cell types/states, spatial heterogeneity, cellular neighborhoods, cellular network, and single-cell protein profiles of CMS tumors. CMS1 tumors had more CD3+, CD8+, and PD1+ immune cells that were found in the epithelial layer frequently. CMS1 was also associated with higher levels of metabolic reprogramming markers such as upregulated glycolysis. CMS2 showed immune segregation, reactive stroma patterns and higher levels of apoptotic and proliferative signaling proteins. CMS3 exhibited clustered cancer cells with high RIP3 levels, suggesting a pro-inflammatory microenvironment. CMS4 displayed stromal-centric and immune-evasive tumors characterized by decreased HLA-1 levels and regulatory T-cell exclusion from epithelium. Conclusion We present a spatial protein atlas of CRC at single-cell resolution and demonstrate novel aspects of CMS tumour structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd0aeac5c1de86e47bfdfaa33c8809067d3de4e" target='_blank'>
              High-Resolution Spatial Proteomics Characterises Colorectal Cancer Consensus Molecular Subtypes
              </a>
            </td>
          <td>
            Batuhan Kısakol, Anna Matveeva, Sanghee Cho, Mohammadreza Azimi, A. Lindner, M. Salvucci, E. McDonough, Joanna Fay, Tony O’Grady, N. McCawley, John P Burke, Deborah A. McNamara, John Graf, Simon McDade, Daniel B. Longley, Fiona Ginty, J. Prehn
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer cells display highly heterogeneous and plastic states in glioblastoma, an incurable brain tumour. However, how these malignant states arise and whether they follow defined cellular trajectories across tumours is poorly understood. Here, we generated a deep single cell and spatial multi-omic atlas of human glioblastoma that pairs transcriptomic, epigenomic and genomic profiling of 12 tumours across multiple regions. We identify that glioblastoma heterogeneity is driven by spatially-patterned transitions of cancer cells from developmental-like states towards those defined by a glial injury response and hypoxia. This cellular trajectory regionalises tumours into distinct tissue niches and manifests in a molecularly conserved manner across tumours as well as genetically distinct tumour subclones. Moreover, using a new deep learning framework to map cancer cell states jointly with clones in situ, we show that tumour subclones are finely spatially intermixed through glioblastoma tissue niches. Finally, we show that this cancer cell trajectory is intimately linked to myeloid heterogeneity and unfolds across regionalised myeloid signalling environments. Our findings define a stereotyped trajectory of cancer cells in glioblastoma and unify glioblastoma tumour heterogeneity into a tractable cellular and tissue framework.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a25f85ea327b0edecce4b687c0e133b5fa0f6d5" target='_blank'>
              A spatiotemporal cancer cell trajectory underlies glioblastoma heterogeneity
              </a>
            </td>
          <td>
            Grant de Jong, Fani Memi, Tannia Gracia, Olga Lazareva, Oliver Gould, Alexander Aivazidis, Manas Dave, Qianqian Zhang, Melanie Jensen, A. Rifaioglu, Joao D. Barros-Silva, Sabine Eckert, Di Zhou, Yvette Wood, Elizabeth Tuck, Sezgin Er, Henry Marshall, Kenny Roberts, Andrew L. Trinh, Shreya Rai, Tyler Shaw, Agnes Oszlanczi, Hayden Powell, R. Petryszak, Zoi Katsirea, Irfan Mamun, I. Mulas, Annelies Quaegebeur, Mayen Briggs, Stanislaw Makarchuk, Jessica Cox, Jimmy Tsz Hang Lee, Laura Rueda, Manu Saraswat, Harry J C J Bulstrode, Adam Young, Minal Patel, Tarryn Porter, E. Prigmore, Moritz Mall, Julio Saez-Rodriguez, James Briscoe, David H. Rowitch, Richard Mair, Sam Behjati, O. Stegle, O. Bayraktar
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4028f5dcc0e86189184980c13bb56279010a29ab" target='_blank'>
              Clonal lineage tracing and parallel multiomics profiling reveal transcriptional diversification induced by ARID1A deficiency
              </a>
            </td>
          <td>
            Kenichi Miyata, Liying Yang, Yuxiao Yang, Kohei Kumegawa, R. Maruyama
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6cfe1f46dc56e476a3d0be0d700257cc9ae2152" target='_blank'>
              Single-Cell and Spatial Profiling of Tumor Microenvironment Heterogeneity in Human Osteosarcomas
              </a>
            </td>
          <td>
            Xuejing zheng, Jin Yuan, Peng Jin, Xuejun Wang, Runlei Xu, Helin Feng, Xinxin Zhang, Yongjie Xie, Wence Wu
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Background Breast cancer is a highly heterogeneous disease where variations of biomarker expression may exist between individual foci of a cancer (intra-tumoral heterogeneity). The extent of variation of biomarker expression in the cancer cells, distribution of cell types in the local tumor microenvironment and their spatial arrangement could impact on diagnosis, treatment planning and subsequent response to treatment. Methods Using quantitative multiplex immunofluorescence (MxIF) imaging, we assessed the level of variations in biomarker expression levels among individual cells, density of cell cluster groups and spatial arrangement of immune subsets from regions sampled from 38 multi-focal breast cancers that were processed using whole-mount histopathology techniques. Molecular profiling was conducted to determine the intrinsic molecular subtype of each analysed region. Results A subset of cancers (34.2%) showed intra-tumoral regions with more than one molecular subtype classification. High levels of intra-tumoral variations in biomarker expression levels were observed in the majority of cancers studied, particularly in Luminal A cancers. HER2 expression quantified with MxIF did not correlate well with HER2 gene expression, nor with clinical HER2 scores. Unsupervised clustering revealed the presence of various cell clusters with unique IHC4 protein co-expression patterns and the composition of these clusters were mostly similar among intra-tumoral regions. MxIF with immune markers and image patch analysis classified immune niche phenotypes and the prevalence of each phenotype in breast cancer subtypes was illustrated. Conclusions Our work illustrates the extent of spatial heterogeneity in biomarker expression and immune phenotypes, and highlights the importance of a comprehensive spatial assessment of the disease for prognosis and treatment planning. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02038-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185f08263c4f5dcd967322d5656a07ab8cc0083f" target='_blank'>
              Intra-tumoral spatial heterogeneity in breast cancer quantified using high-dimensional protein multiplexing and single cell phenotyping
              </a>
            </td>
          <td>
            Alison M. Cheung, Dan Wang, M. Quintayo, Yulia Yerofeyeva, Melanie Spears, John M. S. Bartlett, Lincoln Stein, Jane Bayani, Martin J Yaffe
          </td>
          <td>2025-05-21</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="IDH-wildtype glioblastoma (GBM) is an aggressive brain tumor with poor survival and few therapeutic options. Transcriptionally-defined cell states coexist in GBM and occupy defined regions of the tumor. Evidence indicates that GBM cell states are plastic, but it remains unclear if they are determined by the underlying epigenetic state and/or by microenvironmental factors. Here, we present spatially-resolved epigenomic profiling of human GBM tissues that implicate chromatin structure as a key enabler of cell plasticity. We report two epigenetically-defined and spatially-nested tumor niches. Each niche activates short-range molecular signals to maintain its own state and, surprisingly, long-range signals to reinforce the state of the neighboring niche. The position of a cell along this gradient-like system of opposing signals determines its likelihood to be in one state or the other. Our results reveal an intrinsic system for cell plasticity that is encoded in the chromatin profiles of two adjacent niches that dot the topological architecture of GBM in cartesian space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc7a8f38a5d53281f1873999c12438505ab4d573" target='_blank'>
              Spatial epigenomic niches underlie glioblastoma cell state plasticity
              </a>
            </td>
          <td>
            Sam Kint, S. T. Younes, Shuozhen Bao, Gretchen Long, David Wouters, Erin Stephenson, Xing Lou, Mei Zhong, Di Zhang, Graham Su, Archibald Enninful, Mingyu Yang, Huey-Miin Chen, K. Ellestad, Colleen Anderson, Jennifer Moliterno, John J. Kelly, Jennifer A. Chan, Alejandro Sifrim, Jiangbing Zhou, Ana Nikolic, Rong Fan, Marco Gallo
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b6a64a6d2ad28ecff7ab0f31663d8f3fd11b6a" target='_blank'>
              A spatial atlas of chemoradiation therapy in pancreatic cancer identifies cellular and microenvironmental determinants of persister populations
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, Ariana Acevedo-Diaz, K. Rajapakshe, Galia Jacobson, Daniela Tovar, Jimin Min, Guangsheng Pei, Candise Tat, Ayush Suresh, Ching-Wei Tzeng, Matthew HG Katz, M. Bhutani, Huaming Wang, Robert A. Wolff, C. Haymaker, E. Ludmir, Huocong Huang, Xiongfeng Chen, Liang Li, Albert C Koong, Linghua Wang, N. Navin, Dadi Jiang, Ziyi Li, Anirban Maitra, Eugene J. Koay
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1dc109a3a473a99af4a77b675644d925cf4104" target='_blank'>
              Oncogenic and tumor-suppressive forces converge on a progenitor-orchestrated niche to shape early tumorigenesis
              </a>
            </td>
          <td>
            José Reyes, Isabella S. Del Priore, Andrea C. Chaikovsky, Nikhita Pasnuri, Ahmed M. Elhossiny, Tobias Krause, Andrew Moorman, Catherine Snopkowski, Meril Takizawa, Cassandra Burdziak, Nalin Ratnayeke, Ignas Masillioni, Yu-jui Ho, R. Chaligné, P. Romesser, A. Filliol, T. Nawy, John P. Morris, Zhen Zhao, M. P. di Magliano, Direna Alonso-Curbelo, D. Pe’er, Scott W. Lowe
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ba430604b98f8f47f7bf5f84f30b7530579a6d" target='_blank'>
              Spatial Biology and Organoid Technologies Reveal a Potential Therapy-Resistant Cancer Stem Cell Population in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Pritha Adhikary, Jayati Chakrabarti, Jiang Wang, Ugonna Ezuma-Igwe, Xi Sun, Willian C. Bastian, Nicolas Braissand, Megan P. Corbett, Eugene F. Douglass, Sangmin Kim, Rohan Kanakamedala, Abigail G. Branch, Saptarshi Mallick, Meenu Priya Resmi, Payton Stevens, Marco Padilla-Rodriguez, Maga Sanchez, Gregory Lawrence Beatty, James Griffin, Petros G. Nikolinankos, Kellen Chen, Taylor Riall, Syed A. Ahmad, Rachna Shroff, Frédéric Hollande, Davendra P. S. Sohal, Yana Zavros
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47dca4509610b2f00715550942002b3b04fd28df" target='_blank'>
              Tracing colorectal malignancy transformation from cell to tissue scale
              </a>
            </td>
          <td>
            Helena L. Crowell, Irene Ruano, Zedong Hu, Yourae Hong, G. Caratù, Hubert Piessevaux, Ashley Heck, Rachel Liu, Max Walter, Megan Vandenberg, Kimberly Young, Dan McGuire, Evelyn Metzger, Margaret L Hoang, Joseph M. Beechem, Sabine Tejpar, A. Pascual-Reguant, Holger Heyn
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9da57971dc8950b12d4ca5ccbc0969a63be6c27" target='_blank'>
              Comparison of QuPath and HALO platforms for analysis of the tumor microenvironment in prostate cancer
              </a>
            </td>
          <td>
            Wei Zhang, Qin Zhou, Jonathan V. Nguyen, Erika Egal, Qian Yang, Michael R. Freeman, S. Hu-Lieskovan, Gita Suneja, Anna Coghill, Beatrice S Knudsen
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Colorectal cancer (CRC), including both microsatellite instability (MSI) and microsatellite stability (MSS) subtypes, frequently exhibits intrinsic resistance to immunotherapy. However, the spatial tumor microenvironment (TME) and its role in distinguishing immunotherapy responders from non-responders remain poorly understood. In this study, spatial multiomics, including imaging mass cytometry (n = 50 in-house), spatial proteomics (n = 50 in-house), and spatial transcriptomics (n = 9 in-house), were employed to elucidate the spatial TME of metastatic CRC (mCRC) patients receiving immunotherapy. These methodologies were integrated with single-cell RNA sequencing (scRNA-seq), bulk RNA-seq, and bulk proteomics for comprehensive analysis and validation. A spatial immune atlas containing 314,774 cells was constructed. We found that C1QC+ resident tissue macrophages (RTMs) were more abundant in responders regardless of microsatellite status. Co-localization of C1QC+ RTMs with CD4+ T cells was observed in responders, and MHC-II expression facilitated their interaction. In contrast, cancer-associated fibroblasts inhibited this interaction in non-responders. Moreover, whole genome screening identified key genes involved in antigen presentation in C1QC+ RTMs. Hence, our study highlights the importance of spatial immune mapping in revealing the complex spatial topology of CRC and corresponding immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb9f523addd61c5cd3fcf55f0e488cedb552f07" target='_blank'>
              Spatially resolved C1QC+ macrophage-CD4+ T cell niche in colorectal cancer microenvironment: implications for immunotherapy response
              </a>
            </td>
          <td>
            Hangyu Zhang, Libing Hong, Zhen Dong, Shan Xin, Bo Lin, Jinlin Cheng, Weihong Tian, Bin Li, Jing Wang, Xiaoyan Liu, Chuan Liu, Yuzhi Jin, Yanzhi Feng, Ge Su, Xuqi Sun, Qiqi Liu, Xiaomeng Dai, Yang Gao, Zhou Tong, Lulu Liu, Xudong Zhu, Yi Zheng, Peng Zhao, Tiannan Guo, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-07-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The spatial proteomic profiling of complex tissues is essential for investigating cellular function in physiological and pathological states. However, the imbalance among resolution, protein coverage, and expense precludes their systematic application to analyze whole tissue sections in an unbiased manner and with high resolution. Here, we introduce panoramic spatial enhanced resolution proteomics (PSERP), a method that combines tissue expansion, automated sample segmentation, and tryptic digestion with high-throughput proteomic profiling. The PSERP approach facilitates rapid quantitative profiling of proteomic spatial variability in whole tissue sections at sub-millimeter resolution. We demonstrated the utility of this method for determining the streamlined large-scale spatial proteomic features of gliomas. Specifically, we profiled spatial proteomic features for nine glioma samples across three different mutation types (IDH1-WT/EGFR-mutant, IDH1-mutant, and IDH1/EGFR-double-WT gliomas) at sub-millimeter resolution (corresponding to a total of 2,230 voxels). The results revealed over 10,000 proteins identified in a single slide, which helps us to portray the diverse proteins and pathways with spatial abundance patterns in the context of tumor heterogeneity and cellular features. Our spatial proteomic data revealed distinctive proteomic features of malignant and non-malignant tumor regions and depicted the distribution of proteins from tumor centers to tumor borders and non-malignant tumor regions. Through integrative analysis with single-cell transcriptomic data, we elucidated the cellular composition and cell–cell communications in a spatial context. Our PSERP also includes a spatially resolved tumor-specific peptidome identification workflow that not only enables us to elucidate the spatial expression patterns of tumor-specific peptides in glioma samples with different genomic types but also provides us with opportunities to select combinations of tumor-specific mutational peptides whose expression could cover the maximum tumor regions for future immune therapies. We further demonstrated that combining tumor-specific peptides might enhance the efficacy of immunotherapy in both patient-derived cell (PDC) and patient-derived xenograft (PDX) models. PSERP efficiently retains precise spatial proteomic information within the tissue context and provides a deeper understanding of tissue biology and pathology at the molecular level. Supplementary Information The online version contains supplementary material available at 10.1186/s13045-025-01710-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c4be1dba037f0ddb640334fdf336656ba41adb" target='_blank'>
              Panoramic spatial enhanced resolution proteomics (PSERP) reveals tumor architecture and heterogeneity in gliomas
              </a>
            </td>
          <td>
            Ziyan Xu, Yunzhi Wang, Tao Xie, Rongkui Luo, Heng-Li Ni, Hang Xiang, Shaoshuai Tang, S. Tan, Rundong Fang, Peng Ran, Qiao Zhang, Xiaomeng Xu, Shan Tian, Fuchu He, Wenjun Yang, Chen Ding
          </td>
          <td>2025-05-26</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd09b27d0d8fe6fa80c372bc8924f459f7a4d33" target='_blank'>
              MOSAIC: Intra-tumoral heterogeneity characterization through large-scale spatial and cell-resolved multi-omics profiling
              </a>
            </td>
          <td>
            Caroline Hoffmann
          </td>
          <td>2025-05-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f703b1371a985c2aa75a6f16b3b87971f363e77b" target='_blank'>
              Spatial analysis reveals a novel inflammatory tumor transition state which promotes a macrophage-driven induction of sarcomatoid renal cell carcinoma
              </a>
            </td>
          <td>
            Allison M. May, Suguru Kadomoto, Claire Williams, Alex C. Soupir, Stephanie The, J. McGue, Tyler Robinson, G. Shelley, Mitchell T Hayes, Brooke L. Fridley, Jodi A. Balasi, Paolo M. Ramos Echevarria, J. Dhillon, S. Nallandhighal, Satwik Acharyya, Liying Chen, Jessica Aldous, Nathan Schurman, Veera Baladandayuthapani, Timothy L. Frankel, S. Salami, Brandon J. Manley, R. Mehra, Aaron M. Udager, Evan T. Keller
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive skin cancer with high propensity for metastasis, caused by Merkel-cell-polyomavirus (MCPyV), or chronic UV-light-exposure. How MCPyV spatially modulates immune responses within the tumor microenvironment and how such are linked to patient outcomes remains unknown. We interrogated the cellular and transcriptional landscapes of 60 MCC-patients using a combination of multiplex proteomics, in-situ RNA-hybridization, and spatially oriented transcriptomics. We identified a spatial co-enrichment of activated CD8+ T-cells and CXCL9+PD-L1+ macrophages at the invasive front of virus-positive MCC. This spatial immune response pattern was conserved in another virus-positive tumor, HPV+ head-and-neck cancer. Importantly, we show that virus-negativity correlated with high risk of metastasis through low CD8+ T-cell infiltration and the enrichment of cancer-associated-fibroblasts at the tumor boundary. By contrast, responses to immune-checkpoint blockade (ICB) were independent of viral-status but correlated with the presence of a B-cell-enriched spatial contexts. Our work is the first to reveal distinct immune-response patterns between virus-positive and virus-negative MCC and their impact on metastasis and ICB-response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28305c5ea574406c24e96fc8bdd4ca5602d07afe" target='_blank'>
              Spatially organized inflammatory myeloid-CD8+ T cell aggregates linked to Merkel-cell Polyomavirus driven Reorganization of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Maximilian Haist, M. Matusiak, Yuqi Tan, S. Zimmer, H. Stege, Tim N. Kempchen, S. Mitschke, Pauline Chu, B. Weidenthaler-Barth, Graham L. Barlow, F. Rogall, Antonio Delgado Gonzalez, M. Baertsch, Silvia Albertini, M. Gariglio, C. Borgogna, Y. Goltsev, S. Grabbe, John W. Hickey, Garry P. Nolan
          </td>
          <td>2025-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d980820189344192705dd2c34f2a4d48d18931d" target='_blank'>
              An atlas of gene programs in non-small cell lung cancer delineates novel patterns of lineage vacillation, plasticity, and aggressiveness
              </a>
            </td>
          <td>
            Rui Hong, Pascal Klöckner, Sarah A. Mazzilli, Avrum E. Spira, J. Beane, Masanao Yajima, Joshua D. Campbell
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bec286991f2affb3f9522a83843f9469ba7b89" target='_blank'>
              Developmental Plasticity and Stromal Co-option Shape a Pituitary Neuroendocrine Tumor Transcriptional Continuum
              </a>
            </td>
          <td>
            Robert C. Osorio, Jun Y. Oh, Jangham Jung, Alexander J. Ehrenberg, A. Saha, Meeki K. Lad, Harmon Khela, Nicole Brennick, P. Giannikopoulos, William W. Seeley, Lea T. Grinberg, Aaron Diaz, M. Aghi
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/432419f2032b46caaaf3eb1aa445ee0f4fa38314" target='_blank'>
              Clonal tracing with somatic epimutations reveals dynamics of blood ageing.
              </a>
            </td>
          <td>
            Michael Scherer, Indranil Singh, M. Braun, Chelsea Szu-Tu, Pedro Sanchez Sanchez, Dominik Lindenhofer, N. A. Jakobsen, Verena Körber, Michael Kardorff, Lena Nitsch, Pauline Kautz, Julia Rühle, Agostina Bianchi, Luca Cozzuto, R. Frömel, Sergi Beneyto-Calabuig, Caleb A Lareau, Ansuman T. Satpathy, Renée Beekman, Lars M Steinmetz, S. Raffel, Leif S. Ludwig, Paresh Vyas, Alejo E. Rodriguez-Fraticelli, Lars Velten
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="
 8540


 Background:
 The liver is a frequent site of metastasis and carries a poor prognosis in patients with non-small cell (NSCLC) and small cell lung cancer (SCLC). Patients with liver metastases (LM) derive limited benefit from immune checkpoint inhibitors (ICI), due to hepatic myeloid derived suppressor cell (MDSC) mediated T cell elimination. Here, we used imaging mass cytometry (IMC) to perform single cell, highly multiplexed, analysis of LM and primary lung tumors to investigate how vascular endothelial growth factor (VEGF) influences T cell depletion within the tumor immune microenvironment (TIME) of LM.
 Methods:
 We comprehensively characterizedthe TIME in LM and primary lung tumors in 21 patients with NSCLC or SCLC using IMC. A panel of 40 antibodies was assembled to interrogate immune subsets and VEGF pathway markers. Each antibody was conjugated to a unique metal isotope. After validation, the antibody cocktail was used to stain the biopsies. Tissue images were segmented using Mesmer, and hierarchical clustering was applied to single-cell expression data to identify phenotypes. Similar clustering of cell neighbor profiles was applied to obtain spatial motifs. Phenotypic and motif frequencies, together with functional expression across phenotypes, were compiled from all samples and compared across conditions.
 Results:
 Initial visualization of the raw, unsegmented data revealed higher infiltration of CD8
 +
 and CD4
 +
 T cells in LM compared to the lung TIME. After segmentation, marker expression heatmaps uncovered complex cell-cell interaction ecosystems. The liver samples were enriched with M2 macrophages (CD163
 +
 ), MDSC (CD11b
 +
 ), and proliferative endothelial cells (CD105
 +
 ) whereas the lung samples were enriched in tumor cells (TTF1
 +
 for NSCLC and INSM1
 +
 /synaptophysin
 +
 for SCLC) and T cells (CD4
 +
 , CD8
 +
 ). Spatial neighborhood profiling of NSCLC liver tissues identified 12 neighborhood types, showing a general trend of mutual exclusivity between MDSCs and CD4
 +
 /CD8
 +
 T cells across neighborhoods. Notably, CD8
 +
 T cells in MDSC-enriched neighborhoods exhibited consistently higher FAS expression, a key apoptotic marker. Heterogenous FAS and VEGF signaling across neighborhoods suggested a mixed immune-suppressive and vascularized response in the liver TIME, supporting VEGF’s role in mediating MDSC-driven hepatic CD8
 +
 T cell depletion in patients with LM.
 Conclusions:
 Our findings highlight significant differences in the TIME between LM and primary lung tumors, with LM demonstrating a more immunosuppressive and VEGF-enriched milieu. The spatial association of MDSCs with CD8
 +
 T cells, along with elevated FAS expression and VEGF signaling suggests a mechanistic role for VEGF in driving immune evasion within liver tumors. These results underscore the potential of VEGF blockade as a therapeutic strategy to overcome T cell suppression and improve ICI efficacy in patients with lung cancer metastatic to liver (NCT05588388, PI Sankar).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59e054bf8906de7ff5d9b26427c709d2d7fa221a" target='_blank'>
              Immune landscape of liver metastases in advanced lung cancer.
              </a>
            </td>
          <td>
            K. Sankar, Simeon Mahov, A. Villamejor, A. Walts, Akil A. Merchant, Karen L. Reckamp, Neil A. Bhowmick
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5b1b0582e2b72f3132558990be0699903a9036" target='_blank'>
              CLONEID: A Framework for Monitoring and Steering Subclonal Dynamics
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Vural Tagal, Tao Li, Saeed S. Alahmari, Jackson Cole, Daniel Hannaby, Xiaoqing Yu, Konstantin Maksin, Andrew Schultz, Hojoon Lee, Issam ElNaqa, S. Eschrich, Janine M Lupo, Hanlee P. Ji, Aaron A Diaz, Noemi Andor
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3c75d8fc63308ed8697666b49a18333c902334" target='_blank'>
              A Single-cell and Spatially Resolved Cell Atlas of Human Esophageal Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Yong Shi, Ke An, Yu Qi, Xinhan Zhang, Yueqin Wang, Xuran Zhang, Shaoxuan Zhou, Ouwen Li, Yanan Song, Jiayi Zhou, Yue Du, Mingyang Hou, Yun-Gui Yang, QuanC. Kan, Xin Tian
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Image-based single-cell transcriptomics can identify diverse cell types within intact tissues. However, in adaptive immunity, V(D)J recombination generates unique immune receptors within cells of the same type, leading to important functional variation that is not yet defined by these methods. Here we introduce B-cell-receptor multiplexed error robust fluorescence in situ hybridization (BCR-MERFISH), which distinguishes plasma cell clones based on V-gene usage in combination with transcriptome profiling. We demonstrate that BCR-MERFISH accurately identifies V-gene usage in cell culture and in mice with restricted or native plasma cell diversity. We then use BCR-MERFISH to reveal the microbiota-dependent changes in plasma cell abundance, clonal diversity, and public clonotype usage in the mouse gut and the non-uniform distribution of plasma cell clones along the mouse ileum. As tissue context is an essential modulator of plasma cell dynamics, we anticipate that BCR-MERFISH may offer new insights into a wide range of immunological questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04bc662f55dd7766902cd268f9ec1ec3c80d4c7" target='_blank'>
              In situ profiling of plasma cell clonality with image-based single-cell transcriptomics
              </a>
            </td>
          <td>
            Evan Yang, Jose Aceves-Salvador, C. Castrillon, Uli Herrmann, E. Akama‐Garren, M. Carroll, Jeffrey R. Moffitt
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pituitary neuroendocrine tumors (PitNETs) can be invasive or aggressive, yet the mechanisms behind these behaviors remain poorly understood, impeding treatment advancements. Here, we integrat single-cell RNA sequencing and spatial transcriptomics, analyzing over 177,000 cells and 35,000 spots across 57 tissue samples. This comprehensive approach facilitates the identification of PitNETs tumor populations and characterizes the reconfiguration of the tumor microenvironment (TME) as PitNETs progress and invade. We trace the trajectory of TPIT-lineage PitNETs and identify an aggressive tumor cluster marked by elevated p53-mediated proliferation and a higher Trouillas classification, both associated with tumor progression. Additionally, we document the heterogeneity of immune stromal cells within PitNETs, particularly noting the enrichment of SPP1+ tumor associated macrophages (TAMs) in invasive tumors. These TAMs facilitate tumor invasion through the SPP1-ITGAV/ITGB1 signaling pathway. Our in-depth single-cell and spatial analysis of PitNETs uncovers the molecular dynamics within the TME, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1b01507bfc7f0aa4eb306817808e84f0ce1133" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal tumor heterogeneity and immune remodeling involved in pituitary neuroendocrine tumor progression
              </a>
            </td>
          <td>
            Wan Su, Zhang Ye, Jifang Liu, K. Deng, Jinghua Liu, Huijuan Zhu, Lian Duan, Chen Shi, Linjie Wang, Yuxing Zhao, Fengying Gong, Yi Zhang, Bo Hou, Hui You, Feng Feng, Qing Ling, Yu Xiao, Yongdong Guo, Wenyi Fan, Sumei Zhang, Zixin Zhang, Xiaomin Hu, Yong Yao, Chunhong Zheng, Lin Lu
          </td>
          <td>2025-05-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90f49812ba9bf54efb86922eb7c7255926faf91" target='_blank'>
              Mapping and reprogramming microenvironment-induced cell states in human disease using generative AI
              </a>
            </td>
          <td>
            Amir Akbarnejad, Lloyd Steele, D. Jafree, Sebastian Birk, M. Sallese, K. Rademaker, Adam Boxall, Ben Rumney, Catherine Tudor, Minal Patel, M. Prete, Stanislaw Makarchuk, Tong Li, Heather Stanley, A. Foster, Kenny Roberts, Andrew L. Trinh, C. E. Villa, G. Testa, S. Mahil, Arash Mehrjou, Catherine Smith, Sattar Vakili, M. Clatworthy, Thomas J. Mitchell, O. Bayraktar, M. Haniffa, M. Lotfollahi
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1554b6f0212ba05651f81096c5d01da1508bdc40" target='_blank'>
              Cell simulation as cell segmentation.
              </a>
            </td>
          <td>
            Daniel C Jones, Anna E. Elz, Azadeh Hadadianpour, H. Ryu, David R. Glass, Evan W. Newell
          </td>
          <td>2025-05-22</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Recent studies have revealed that polyclonality—where multiple distinct subclones cooperate during early tumor development—is a critical feature of tumor evolution, as demonstrated by Sadien et al. and Lu et al. in Nature (October 2024). These findings show that early polyclonal interactions can overcome fitness barriers, ultimately transitioning to monoclonality as dominant clones emerge. Understanding and targeting these interclonal dynamics offers new therapeutic opportunities. In this perspective, we outline how computational modeling and artificial intelligence (AI) tools can provide deeper insights into tumor polyclonality and identify actionable therapeutic strategies. By applying ligand–receptor interaction analysis, clonal trajectory reconstruction, network and pathway modeling, and spatial analysis, researchers can prioritize communication hubs, evolutionary bottlenecks, and microenvironmental niches that sustain tumor progression. These approaches, when integrated with experimental validation, offer a translational pathway from foundational discoveries to personalized cancer treatments aimed at disrupting cooperative subclonal ecosystems and preventing malignant progression. We commend the recent Nature publications, “Polyclonality overcomes fitness barriers in Apc-driven tumorigenesis” by Sadien et al. [1] and “Polyclonal-to-monoclonal transition in colorectal precancerous evolution” by Lu et al. [2], both featured on 2024 October 30. These groundbreaking studies employed distinct lineage tracing methods to investigate the origins and evolutionary dynamics of colorectal and intestinal tumorigenesis. Despite their different approaches, both studies reached convergent conclusions: Polyclonality plays a pivotal role in the early stages of tumor development, providing critical insights into how diverse cellular populations collaborate to overcome fitness barriers and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6580f8163812203e6792c0b0ae89c7c42827a3" target='_blank'>
              Artificial Intelligence-Powered Insights into Polyclonality and Tumor Evolution
              </a>
            </td>
          <td>
            Hong Zhao, T. Ideker, Stephen Wong
          </td>
          <td>2025-06-13</td>
          <td>Research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edded8a4572e2f0a421a8d1ab432a7fe9035dbf5" target='_blank'>
              A Human Tumor-Immune Organoid Model of Glioblastoma
              </a>
            </td>
          <td>
            Shivani Baisiwala, Elisa Fazzari, Matthew Li, A. Martija, Daria J. Azizad, Lu Sun, Gilbert Herrera, Trinh L Phan, Amber Monteleone, David A. Nathanson, Anthony C. Wang, Won Kim, R. Everson, Kunal S. Patel, Linda M. Liau, Robert M Prins, A. Bhaduri
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Allogeneic hematopoietic cell transplantation (alloHCT) is curative for various hematologic diseases but often leads to acute graft-versus-host disease (GVHD), a potentially life-threatening complication. We leverage GVHD as a uniquely tractable disease model to dissect complex T-cell–mediated pathology in 27 alloHCT recipients. We integrate pre-transplant identification of alloreactive T-cells with longitudinal tracking across blood and gut, using mixed lymphocyte reaction-based clonal “fingerprinting”, TCR clonotyping, single-cell RNA/TCR sequencing, and spatial transcriptomics. Using DecompTCR, a novel computational tool for longitudinal TCR analysis, we uncover clonal expansion programs linked to GVHD severity and TCR features. Multi-omics profiling of gut biopsies reveals enrichment and clonal expansion of CD8⁺ effector and ZNF683(Hobit)⁺ resident memory T-cells, cytolytic remodeling of regulatory and unconventional T-cells, and localization of CD8⁺ effector T-cells near intestinal stem cells in crypt loss regions. This framework defines dynamic immune circuit rewiring and phenotypic plasticity with implications for biomarkers and therapies. Graphical Abstract Highlights Persistent expansion of diverse alloreactive T cell clones is a hallmark of severe GVHD DecompTCR reveals dynamic clonal expansion programs linked to GVHD severity and clinical outcome CD8+ T cell clones exhibit phenotypic plasticity in vivo across intestinal tissue compartments in GVHD High-resolution spatial profiling shows CD8+ effector T cells localize near intestinal stem cell niches and drive epithelial injury in GVHD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/101476c161e31296c716a3225abae509503d5e46" target='_blank'>
              Spatiotemporal Single-Cell Analysis Reveals T Cell Clonal Dynamics and Phenotypic Plasticity in Human Graft-versus-Host Disease
              </a>
            </td>
          <td>
            Lingting Shi, Ajna Uzuni, X. Wang, Michael Pressler, David W. Harle, Shami Chakrabarti, Rodney Macedo, Kirubel Belay, Christian A. Gordillo, Erik Raps, Jia Yi Zhang, Achille Nazaret, J. L. Fan, Yinuo Jin, Xumin Shen, Joshua Fuller, Tamjeed Azad, Jessie Huang, Pranik Chainani, Julian A. Abrams, Armando del Portillo, Markus Y. Mapara, M. Alhamar, Megan Sykes, José L. McFaline-Figueroa, Elham Azizi, Ran Reshef
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) is emerging as a powerful technology that transforms our understanding of thyroid cancer by offering a spatial context of gene expression within the tumor tissue. In this review, we synthesize the recent applications of ST in thyroid cancer research, with a particular focus on the heterogeneity of the tumor microenvironment, tumor evolution, and cellular interactions. Studies have leveraged the spatial information provided by ST to map distinct cell types and expression patterns of genes and pathways across the different regions of thyroid cancer samples. The spatial context also allows a closer examination of invasion and metastasis, especially through the dysregulation at the tumor leading edge. Additionally, signaling pathways are inferred at a more accurate level through the spatial proximity of ligands and receptors. We also discuss the limitations that need to be overcome, including technical limitations like low resolution and sequencing depth, the need for high-quality samples, and complex data handling processes, and suggest future directions for a wider and more efficient application of ST in advancing personalized treatment of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3b3022cb800cbef42ede8e65012209b885afa4" target='_blank'>
              Spatial Transcriptomics in Thyroid Cancer: Applications, Limitations, and Future Perspectives
              </a>
            </td>
          <td>
            Chaerim Song, Hye-Ji Park, Man S. Kim
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Developing high-resolution reference maps of disease-susceptible spatial niches is a critical step to mitigating the profound effects of lung disease. Here, we present an integrated multimodal single-nucleus human lung atlas (snHLA) profiling 746,047 nuclei from 49 mapped lung blocks spanning clinically relevant distal airways, alveoli, and interstitium across 11 healthy adults. Integrating snRNA-seq and SNARE-seq2, which co-assays chromatin accessibility and gene expression from the same nucleus, we resolved 70 molecularly distinct populations and captured 332,846 accessible chromatin regions, nominating new transcriptional regulators of human lung cell diversity. Spatial transcriptomics using MERFISH mapped 25 cell populations across 7 structural neighborhoods and multiplexed immunofluorescence localized cell subtypes and distal airway-defining protein markers, expanding and validating distinct lung structure-specific cell populations. This open access snHLA and companion Cell Type and Marker Gene Dictionary with anatomically aligned nomenclature delivers a foundational resource at an unprecedented resolution to interrogate the origins of lung pathophysiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/085ae835dd14cf9157e86d76e8875d20b81277fe" target='_blank'>
              A Multimodal Spatial and Epigenomic Atlas of Human Adult Lung Topography
              </a>
            </td>
          <td>
            T. Duong, D. Diep, Kimberly Y. Conklin, Indy Bui, J. Purkerson, Eric Boone, Jacqueline Olness, Sahil Patel, Beverly Peng, Colin Kern, Zoey Zhao, Ravi S. Misra, H. Huyck, J. Verheyden, Zea Borok, Yun Zhang, R. Scheuermann, Quan Zhu, Gail Deutsch, Jim Hagood, Xin Sun, Kun Zhang, Gloria S. Pryhuber
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d59997cf6c8e6c97ba758d271ce928cfa49100ba" target='_blank'>
              Transcriptomic Profiles from Normal and Tumor Tissue Samples Reveal Distinct Venule Populations and Novel Tumor Endothelial Cell Markers in Breast Cancer
              </a>
            </td>
          <td>
            K. Phoenix, Vijender Singh, Patrick A. Murphy, Kevin P. Claffey
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c75666ffc8a05530da0c0fa726109747688e9c42" target='_blank'>
              Single-cell chromatin landscape and DNA methylation patterns reveal shared molecular programs in human tumor and non-tumor tissue CCR8+ Treg cells
              </a>
            </td>
          <td>
            K. Braband, Tamara Kaufmann, Niklas Beumer, Malte Simon, Sara Salome Helbich, Delia Mihaela Mihoc, Morten Voss, A. Nedwed, K. Bauer, Jan-Philipp Mallm, Tanja Ziesmann, Ute Distler, Stephan Tenzer, Christoph Eckert, Michael Volkmar, D. Weichenhan, Marion Bähr, C. Plass, Matthias Linke, S. Diederich, Matthias Klein, T. Bopp, Hansjoerg Schild, Andreas Henkel, Edoardo Filippi, Federico Marini, B. Brors, Matthias M Gaida, Michael Delacher
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e14574


 Background:
 Understanding cellular interactions within the tumor microenvironment (TME) is essential for elucidating disease progression and advancing immunotherapy. The TME is a complex ecosystem composed of cells with dysregulated metabolism immune response and signaling pathways, which in turn influence tumor development and treatment response. Multiplexed assessment serves as an important tool for clinical oncology. The simultaneous readout of multiple processes can provide biological insights and elucidate disease mechanism. Imaging Mass Cytometry (IMC) is a spatial biology imaging technique that utilizes CyTOF technology and enables deep characterization of the diversity and complexity of the TME. IMC technology offers scalable, high-throughput acquisition while generating high-quality data with true dynamic range of signal without amplification or fluorescence-based limitations such as spectral overlap and autofluorescence.
 Methods:
 To study cellular processes and their roles in tumor progression, we utilized the Human Cell Metabolism and Human Cell Signaling Panels to investigate energy production, cellular homeostasis and mitogenic signaling pathways. We then mapped these processes to the types of cells in the TME by using the Human Immuno-Oncology IMC Panel and the Human T Cell Exhaustion IMC Panel or the Maxpar Neuro Phenotyping IMC Panel Kit to characterize immune cell and neurological phenotypes in detail. We first acquired data using Preview Mode to assess the whole tissue, followed by higher-resolution imaging of selected regions of interest using Cell Mode or of the whole tissue section using Tissue Mode.
 Results:
 Our data analysis revealed significant insights into the spatial organization and metabolic profile of cells across cancer tissues. Elevated glycolysis and mTOR pathway activation suggested adaptations to hypoxia and anabolic growth in tumor areas, while interactions between fibroblasts and immune cells highlighted crosstalk within the TME. Our Neuro Phenotyping Panel was used to reveal potential for immune response in glioblastoma. Unsupervised pixel clustering and hierarchical clustering using MCD SmartViewer highlighted metabolic activity and activation of signaling pathways within tumor regions.
 Conclusions:
 Comprehensive spatial biology profiling using the IMC approach highlights the interconnected roles these pathways play in promoting tumor survival and resistance to therapies. These findings, which illuminate the metabolic and signaling heterogeneity of the TME, are crucial for developing future prognostic assessments and have the potential to guide more effective, personalized cancer therapies. For Research Use Only. Not for use in diagnostic procedures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b16079bb313271e366d31125c0f7c2617d3a11a" target='_blank'>
              Spatial profiling of solid tumor microenvironment using imaging mass cytometry with high-plex panels.
              </a>
            </td>
          <td>
            T. Pfister, J. Y. Chwee, Nick Zabinyakov, Qanber Raza, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Solid cancers are complex 'ecosystems' comprised of diverse cell types and extracellular molecules, where heterotypic interactions significantly influence disease etiology and therapeutic response. However, our current understanding of tumor microenvironments (TMEs) remains incomplete, hindering the development and implementation of novel TME-targeted drugs. To maximize cancer therapeutic development we require a systems-level understanding of the malignant, stromal, and immune states that define the tumor and determine treatment response. In their recent study, Liu and colleagues took a new approach to resolving the complexity of stromal heterogeneity. They leveraged extensive single-cell spatial multi-omics datasets across various cancer types and platforms to identify four conserved spatial Cancer-Associated Fibroblast (CAF) subtypes, classified by their spatial organization and cellular neighborhoods. Their work expands upon prior efforts to develop a CAF taxonomy, which primarily relied on single-cell RNA-Sequencing (scRNA-Seq) and yielded a multitude of classification systems. This study advances our understanding of CAF biology by establishing a link between spatial context and CAF identity across diverse tumor types. Departing from recent single-cell transcriptomics studies that employed a marker-based approach for sub-state identification, Liu and colleagues conducted de novo discovery of CAF subtypes using spatial neighborhood information alone. By positioning spatial organization as the defining axis of CAF heterogeneity, this research redefines CAF classification and provides a new framework for exploring the rules governing tumor ecosystems and developing novel ecosystem-based therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940389ee957141f2833e0d9de53711c0ead3a30c" target='_blank'>
              Pan-Cancer Spatial Profiling Reveals Conserved Subtypes and Niches of Cancer-Associated Fibroblasts.
              </a>
            </td>
          <td>
            Hani Jieun Kim, Travis Ruan, Alex Swarbrick
          </td>
          <td>2025-05-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Disease heterogeneity across a diverse patient cohort poses challenges to cancer drug development due to inter-patient variability in treatment responses. However, current preclinical models fail to depict inter-patient tumor heterogeneity, leading to a high failure rate when translating preclinical leads into clinical successes. Methods: We integrated the expression profiles of prostate cancer (PC) lines and castration-resistant PC (CRPC) patient tumors to identify cell-lines that transcriptomically match distinct tumor subtypes in a clinical cohort. Representative cell-lines were co-cultured to create “mixed-cell” models depicting inter-patient heterogeneity in CRPC, which were employed to assess drug combinations. Results: When drug combinations previously tested in CRPC clinical cohorts, were assessed to establish proof-of-concept, in-vitro responses measured in our models concurred with their known clinical efficacy. Additionally, novel drug combinations computationally predicted to be efficacious in heterogeneous tumors, were evaluated. They demonstrated preclinical efficacy in the mixed-cell models, suggesting they will likely benefit heterogeneous patient cohorts. Furthermore, we showed that the current practice of screening cell-lines/xenografts separately and aggregating their responses, failed to detect their efficacy. Conclusions: We believe that the application of our models will enhance the accuracy of preclinical drug assessment, thereby improving the success rate of subsequent clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4d639da8c86d125f39c0d18727d1ba6b4439891" target='_blank'>
              Novel mixed cancer-cell models designed to capture inter-patient tumor heterogeneity for accurate evaluation of drug combinations
              </a>
            </td>
          <td>
            Sampreeti Jena, Daniel Kim, Adam Lee, Weijie Zhang, Kevin Zhan, Yingming Li, S. Dehm, R. S. Huang
          </td>
          <td>2025-05-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ac7f496e193e8ea2a2dad30237740bdee230d9" target='_blank'>
              Multiscale modeling predicts dependence of mesenchymally transitioned tumor niche fitness on cell-cell and cell-matrix adhesions
              </a>
            </td>
          <td>
            C. V. Sai Prasanna, M. Jolly, R. Bhat
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a622937af13dfe0cc2c7b54d0cc8b36683ac30e" target='_blank'>
              Rictor CRISPR Gene Editing by Lipid Nanoparticle Delivery Stimulates Anti-tumor Immunity in Breast Cancer Liver Metastasis Model
              </a>
            </td>
          <td>
            Yaqoob Ali, Tyler Galbraith, Nourhan Abdelfattah, A. Ziemys, Thomas Wong, C. Hashimoto, M. Faisal, Xu Qian, Harlan Cook, Tej Pandita, Yitian Xu, Roberto Rosato, Shu-hsia Chen, Kyuson Yun, Fransisca Leonard
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e526c65686bd001de0faa1340763d5a1c493080f" target='_blank'>
              Cancer systems immunology reveals myeloid—T cell interactions and B cell activation mediate response to checkpoint inhibition in metastatic breast cancer
              </a>
            </td>
          <td>
            Edgar Gonzalez, Jesse Kreger, Yingtong Liu, Xiaojun Wu, Arianna Barbetta, A. Baugh, Batul Al-zubeidy, Julie Jang, Sarah M. Shin, Matthew Jacobo, Vered Stearns, Roisin Connolly, Won Ho, Juliet Emamaullee, Adam L. MacLean, Evanthia T. Roussos Torres
          </td>
          <td>2025-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity. GBM tumors contain neoplastic cells within a spectrum of different transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we developed a novel framework, ISOSCELES (Inferred cell Sensitivity Operating on the integration of Single-Cell Expression and L1000 Expression Signatures), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identified compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states. We validated the significance of these findings in vitro, ex vivo, and in vivo, and identified a novel combination of an OLIG2 inhibitor and Depatux-M for GBM. Our studies suggest that ISOSCELES identifies cell states sensitive and resistant to targeted therapies in GBM and that it can be applied to identify new synergistic combinations. Highlights Integration of GBM single-cell RNA sequencing data with L1000-derived drug response signatures facilitates clustering of tumor cells and small molecules on cell-drug connectivity. Cell-drug connectivity predicts the identities of drug-sensitive and resistant cell states. In silico perturbation analysis using cell-drug connectivity predicts drug-induced changes in the cell-drug connectivity landscape in vivo. In silico perturbation analysis to predict drug-induced changes in the tumor cell-drug connectivity landscape predicts drug combinations that synergize in vivo to extend survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25613f3a85e6691e66bbf8705e07ffcebf8c6cbf" target='_blank'>
              DRUG AND SINGLE-CELL GENE EXPRESSION INTEGRATION IDENTIFIES SENSITIVE AND RESISTANT GLIOBLASTOMA CELL POPULATIONS
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, Ricardo J Komotar, M. I. De La Fuente, Gregory Stein, Alexander Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, N. Ayad
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83b5ca4a7a4ab998ed122fece632645d240cae73" target='_blank'>
              Decoding Plasticity Regulators and Transition Trajectories in Glioblastoma with Single-cell Multiomics
              </a>
            </td>
          <td>
            Manu Saraswat, Laura Rueda-Gensini, Elisa Heinzelmann, Tannia Gracia, Fani Memi, Grant de Jong, Jannes Straub, Cedar Schloo, D. Hoffmann, E. Jung, Tim Kindinger, Bettina Weigel, Bryce Lim, S. Weil, Oliver Gould, Richard Mair, Katharina Mikulik, Martin Rohbeck, W. Wick, F. Winkler, O. Bayraktar, O. Stegle, Moritz Mall
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Lymphomas are highly heterogeneous malignancies characterized by diverse genetic, epigenetic, and phenotypic profiles, which complicate diagnosis, prognosis, and treatment. Traditional bulk sequencing methods mask the complexity of intratumoral variation, often overlooking rare subclones that may drive disease progression and therapeutic resistance. Single-cell genomics has emerged as a transformative approach to decipher tumor heterogeneity at unprecedented resolution. This technology enables the dissection of individual cellular populations within lymphomas, offering insights into clonal evolution, transcriptional diversity, and microenvironmental interactions. By applying single-cell RNA sequencing (scRNA-seq), chromatin accessibility assays (scATAC-seq), and single-cell DNA sequencing (scDNA-seq), researchers can unravel lineage relationships, identify resistant subpopulations, and track dynamic changes in response to therapy. In lymphomas such as diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL), and mantle cell lymphoma (MCL), single-cell approaches have revealed distinct malignant and non-malignant cell states that correlate with treatment outcomes. Moreover, integrating single-cell data with spatial transcriptomics and immune profiling enhances the understanding of the tumor microenvironment, including immune evasion mechanisms. These insights can inform personalized treatment strategies, identify novel therapeutic targets, and enable early detection of relapse. Despite technical challenges such as data complexity, sample viability, and cost, the application of single-cell genomics in lymphoma research is rapidly advancing. Future directions include multi-omics integration, real-time patient monitoring, and clinical translation of predictive biomarkers. This review underscores the pivotal role of single-cell genomics in resolving tumor heterogeneity and predicting therapeutic resistance, positioning it as a cornerstone for next-generation precision oncology in lymphomas,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/438856abbab164e715372bfe3f2a6c07df9594ec" target='_blank'>
              Role of mutational signatures and clonality assessments in tailoring targeted therapies for lymphoma
              </a>
            </td>
          <td>
            Andoh Sheikh Atta-ullah, Uche Philip, Sampson Janice Candy
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4036914e070de84fceb3196161df4ff74c7602d" target='_blank'>
              Predicting targeted- and immunotherapeutic response outcomes in melanoma with single-cell Raman Spectroscopy and AI
              </a>
            </td>
          <td>
            Kai Chang, Mamatha Serasanambati, Baba Ogunlade, Hsiu-Ju Hsu, James Agolia, Ariel Stiber, Jeffrey Gu, Saurabh Sharma, Jay Chadokiya, Amanda Gonçalves, Fareeha Safir, Nhat Vu, Daniel Delitto, Amanda R. Kirane, Jennifer A. Dionne
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Spatial transcriptomics technologies have emerged as powerful tools for understanding cellular identity and function within the natural spatial context of tissues. Traditional transcriptomics techniques, such as bulk and single-cell RNA sequencing, lose this spatial information, which is critical for addressing many biological questions. Here, we present a protocol for high-resolution spatial transcriptomics using fixed frozen mouse lung sections mounted on 10X Genomics Xenium slides. This method integrates multiplexed fluorescent in situ hybridization (FISH) with high-throughput imaging to reveal the spatial distribution of mRNA molecules in lung tissue sections, allowing detailed analysis of gene expression changes in a mouse model of pulmonary hypertension (PH). We compared two tissue preparation methods, fixed frozen and fresh frozen, for compatibility with the Xenium platform. Our fixed frozen approach, utilizing a free-floating technique to mount thin lung sections onto Xenium slides at room temperature, preserved tissue integrity and maximized the imaging area, resulting in high-fidelity spatial transcriptomics data. Using a predesigned 379-gene mouse panel, we identified 40 major lung cell types. We detected key cellular changes in PH, including an increase in arterial endothelial cells (AECs) and fibroblasts, alongside a reduction in capillary endothelial cells (CAP1 and CAP2). Through differential gene expression analysis, we observed markers of endothelial-to-mesenchymal transition and fibroblast activation in PH lungs. High-resolution spatial mapping further confirmed increased arterialization in the distal microvasculature. These findings underscore the utility of spatial transcriptomics in preserving the native tissue architecture and enhancing our understanding of cellular heterogeneity in disease. Our protocol provides a reliable method for integrating spatial and transcriptomic data using fixed frozen lung tissues, offering significant potential for future studies in complex diseases such as PH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/399fddf9db865d1e3c47fe6ffcc8cfaa55ffe72d" target='_blank'>
              Enhanced Spatial Transcriptomics Analysis of Mouse Lung Tissues Reveals Cell-Specific Gene Expression Changes Associated with Pulmonary Hypertension
              </a>
            </td>
          <td>
            Hanqiu Zhao, Xiaokuang Ma, Peng Chen, Bin Liu, Jing Wei, John Zhang, Ankit A. Desai, Andrea L. Frump, Olga Rafikova, Michael B Fallon, Shenfeng Qiu, Zhiyu Dai
          </td>
          <td>2025-05-15</td>
          <td>Journal of respiratory biology and translational medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis M. Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Breast cancer brain metastases (BCBM) affect nearly 90,000 patients annually in the United States and carry a significant risk of mortality. As metastatic lesions develop, the unique milieu of the brain microenvironment shapes disease progression and therapeutic response. Among resident brain cells, astrocytes are both the most common, and are increasingly recognized as key regulators of this process, yet their precise role remains poorly defined. Here, we present a hybrid agent-based model (ABM) to simulate tumor–astrocyte interactions on a two-dimensional lattice. In our model, metastatic tumor cells induce phenotypic reprogramming of astrocytes from an antito a pro-metastatic state, thereby enhancing tumor proliferation. We systematically evaluate how variations in astrocyte density, spatial distribution, and chemotherapy impact tumor expansion and spatial morphology, quantified by fractal dimension, lacunarity, and eccentricity. Our simulations reveal that astrocyte reprogramming accelerates tumor progression and contributes to increased morphological complexity and chemotherapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331a7ee541e9213b60f3df2d93320f1bf057f75f" target='_blank'>
              Astrocyte reprogramming drives tumor progression and chemotherapy resistance in agent-based models of breast cancer brain metastases
              </a>
            </td>
          <td>
            Rupleen Kaur, R. Barker-Clarke, Andrew Dhawan
          </td>
          <td>2025-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Organotropism results from the functional versatility of metastatic cancer cells to survive and proliferate in diverse microenvironments. This adaptivity can originate in clonal variation of the spreading tumor and is often empowered by epigenetic and molecular reprogramming of cell regulatory circuits. Related to organotropic colonization of metastatic sites are environmentally-sensitive, differential responses of cancer cells to therapeutic attack. Accordingly, understanding the organotropic profile of a cancer and probing the underlying driver mechanisms are of high clinical importance. However, determining systematically the organotropism of one cancer versus the organotropism of another cancer, potentially with the granularity of comparing the same cancer type between patients or tracking the evolution of a cancer in a single patient for the purpose of personalized treatment, has remained very challenging. It requires a host organism that allows observation of the spreading pattern over relatively short experimental times. Moreover, organotropic patterns often tend to be statistically weak and superimposed by experimental variation. Thus, an assay for organotropism must give access to statistical powers that can separate ‘meaningful heterogeneity’, i.e., heterogeneity that determines organotropism, from ‘meaningless heterogeneity’, i.e., heterogeneity that causes experimental noise. Here we describe an experimental workflow that leverages the physiological properties of zebrafish larvae for an imaging-based assessment of organotropic patterns over a time-frame of 3 days. The workflow incorporates computer vision pipelines to automatically integrate the stochastic spreading behavior of a particular cancer xenograft in tens to hundreds of larvae allowing subtle trends in the colonization of particular organs to emerge above random cell depositions throughout the host organism. We validate our approach with positive control experiments comparing the spreading patterns of a metastatic sarcoma against non-transformed fibroblasts and the spreading patterns of two melanoma cell lines with previously established differences in metastatic propensity. We then show that integration of the spreading pattern of xenografts in 40 – 50 larvae is necessary and sufficient to generate a Fish Metastatic Atlas page that is representative of the organotropism of a particular oncogenotype and experimental condition. Finally, we apply the power of this assay to determine the function of the EWSR1::FLI1 fusion oncogene and its transcriptional target SOX6 as plasticity factors that enhance the adaptive capacity of metastatic Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acab5ca78a81e16038185b4373a584725e52ba21" target='_blank'>
              Low-branching vessel architecture shapes immune cell niches and predicts immune responses in renal cancer
              </a>
            </td>
          <td>
            M. Toma, Yangping Li, Melina Kehl, Tim Kempchen, L. Esser, Katharina Baschun, S. Leonardelli, Thomas Pinetz, R. Turiello, M. Yong, Natalie Pelusi, Guillermo Altamirano-Escobedo, Eduardo Bayro-Corrochano, Sebastian Kadzik, M. Eckstein, Lukas Flatz, Fabian Hörst, J. Kleesiek, J. Saal, Glen Kristiansen, Manuel Ritter, J. Ellinger, Viktor Grünwald, Niklas Klümper, Alexander Effland, M. Hölzel
          </td>
          <td>2025-06-06</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c327cba87285dc890d1f7b106ff92f090a08a250" target='_blank'>
              Single-Cell Analysis of Meningiomas Reveals Mutation-Associated Tumor and Immune Cell Gene Expression Programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus I Anzaldua-Campos, Gregory J Zipfel, Albert H. Kim, J. Dowling, Eric C. Leuthardt, Joshua W. Osbun, A. Vellimana, Michael R. Chicoine, Gavin P. Dunn, Allegra A. Petti
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Chromosomal instability (CIN), a characteristic feature of esophageal adenocarcinoma (EAC), drives tumor aggressiveness and therapy resistance, presenting an intractable problem in cancer treatment. CIN leads to constitutive stimulation of the innate immune cGAS–STING pathway, which has been typically linked to anti-tumor immunity. However, despite the high CIN burden in EAC, the cGAS– STING pathway remains largely intact. To address this paradox, we developed novel esophageal cancer models, including a CIN-isogenic model, discovering myeloid-attracting chemokines – with the chemokine CXCL8 (IL-8) as a prominent hit – as conserved CIN-driven targets in EAC. Using high-resolution multiplexed immunofluorescence microscopy, we quantified the extent of ongoing cGAS-activating CIN in human EAC tumors by measuring cGAS-positive micronuclei in tumor cells, validated by orthogonal whole-genome sequencing-based CIN metrics. By coupling in situ CIN assessment with single-nucleus RNA sequencing and multiplex immunophenotypic profiling, we found tumor cell-intrinsic innate immune activation and intratumoral myeloid cell inflammation as phenotypic consequences of CIN in EAC. Additionally, we identified increased tumor cell-intrinsic CXCL8 expression in CINhigh EAC, accounting for the inflammatory tumor microenvironment. Using a novel signature of CIN, termed CINMN, which captures ongoing CIN-associated gene expression, we confirm poor patient outcomes in CINhigh tumors with signs of aberrantly rewired cGAS–STING pathway signaling. Together, our findings help explain the counterintuitive maintenance and expression of cGAS–STING pathway components in aggressive, CINhigh tumors and emphasize the need to understand the contribution of CIN to the shaping of a pro-tumor immune landscape. Therapeutic strategies aimed at disrupting the cGAS-driven inflammation axis may be instrumental in improving patient outcomes in this aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dfd8088150f447fa336f3b6ae36838000571864" target='_blank'>
              Chromosomal instability shapes the tumor microenvironment of esophageal adenocarcinoma via a cGAS–chemokine–myeloid axis
              </a>
            </td>
          <td>
            Bruno Beernaert, Rose L Jady-Clark, Parin Shah, Erik Ramon‐Gil, Nora M Lawson, Zack D. Brodtman, Somnath Tagore, Frederik Stihler, Alfie S Carter, Shannique Clarke, Tong Liu, Winston Zhu, Erkin Erdal, Alistair Easton, Leticia Campo, Molly Browne, Stephen Ash, Nicola Waddell, Thomas Crosby, Simon R. Lord, Derek A Mann, Ignacio Melero, Carlos E. De Andrea, Andrea E. Tijhuis, F. Foijer, Ester M. Hammond, Kadir C Akdemir, J. Leslie, Benjamin Izar, Eileen E Parkes
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is one of the most lethal cancers for humans. HCC is highly heterogeneous. In this study, we performed ultra-depth (∼1 million reads per spot) sequencing of 6,320 spatial transcriptomes on a case of HCC. Sixteen distinct spatial expression clusters were identified. Each of these clusters was spatially contiguous and had distinct gene expression patterns. In contrast, benign liver tissues showed minimal heterogeneity in terms of gene expression. Numerous immune cell-enriched spots were identified in both HCC and benign liver regions. Cells adjacent to these immune cell-enriched spots showed significant alterations in their gene expression patterns. Interestingly, the responses of HCC cells to the nearby immune cells were significantly more intense and broader, while the responses of benign liver cells to immune cells were somewhat narrow and muted, suggesting an innate difference in immune cell activities towards HCC cells in comparison with benign liver cells. However, cell-cell interaction analyses showed significant immune evasion by HCC cancer cells. When standard-depth sequencing was performed, significant numbers of genes and pathways that were associated with these changes disappeared. Qualitative differences in some pathways were also found. These results suggest that deep spatial sequencing may help to uncover previously unidentified mechanisms of liver cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa11a255e72a72322fc9f248cef1483a72dd27ef" target='_blank'>
              Deep spatial sequencing revealing differential immune responses in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yan-Ping Yu, Caroline Obert, Baoguo Ren, Marielle Krivit, Kyle Metcalfe, Jia-Jun Liu, Tuval Ben-Yehezkel, Silvia Liu, Jian-Hua Luo
          </td>
          <td>2025-06-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca2ada71e9dda717c7f2fb533cc95887911976c" target='_blank'>
              SMARCA4 is essential for early-stage tumor development but its loss promotes late-stage cancer progression in small-cell lung cancer
              </a>
            </td>
          <td>
            Nicole Kirk, Jin Ng, Kate-Lin Ly, Young Ho Ban, Godfrey Dzhivhuho, Jinho Jang, Kyung-Pil Ko, M. Kareta, Jae-Ll Park, Anthony N. Karnezis, Anish Thomas, K. Sutherland, Kwon-Sik Park
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 8539


 Background:
 Immune checkpoint inhibitors have greatly improved outcomes in advanced non-small cell lung cancer (NSCLC). However, patients with EGFR mutant NSCLC have a poor response to immune checkpoint inhibitor therapy. Emerging evidence suggests that an immunosuppressive tumor microenvironment plays an important role in this setting, however, we still lack fundamental knowledge about tumor endothelial cell biology. We hypothesized that oncogene specific changes in the expression of immune-related genes in tumor endothelial cells account for differences in the tumor microenvironment and efficacy of immune checkpoint inhibitor therapy.
 Methods:
 We utilized spatial transcriptomics (GeoMx Digital Spatial Profiling) on resected tumor tissue of EGFR mutant (n = 5) and KRAS mutant (n = 5) NSCLC patients to investigate the transcriptional signature of tumor, endothelial and stromal cells. Additionally, we used NicheNet to explore intercellular communication between tumor, endothelial, and stromal cells. Immune gene set enrichment analysis scores were calculated using ESTIMATE. Immune cell type proportions were estimated using CIBERSORT.
 Results:
 Using spatial transcriptomics, we dissected the tumor microenvironment into tumor, stromal and endothelial compartments. By analyzing predicted cellular communication, we found that tumor and stromal cells primarily affect an interferon-related gene signature in tumor endothelial cells. Notable differentially expressed interferon-related genes, that were significantly up-regulated in KRAS mutant and down-regulated in EGFR mutant NSCLC patients, included CXCL9, STAT1, WARS1, IRF1 and ICAM1. In the stromal compartment, immune gene set enrichment analysis scores were significantly lower in EGFR mutant than KRAS mutant NSCLC (median, 355 vs. 713, P = 0.026) indicating an immunosuppressive tumor microenvironment. We observed substantial heterogeneity while exploring the cellular landscape of the stromal compartment in EGFR mutant and KRAS mutant NSCLC. Notably, we found a significantly decreased proportion of pro-inflammatory macrophages in the stromal compartment of EGFR compared to KRAS mutant NSCLC (P = 0.006).
 Conclusions:
 We identified distinct interferon-related gene signatures in tumor endothelial cells of patients with EGFR and KRAS mutantNSCLC associated with variations in the cellular composition of the tumor microenvironment. This may provide a better understanding for the development of spatial biomarkers to identify which oncogene-driven NSCLC patients are most likely to benefit from immune checkpoint inhibitor therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3c21a018e08000029baaeb3ba8f18554c44ba0f" target='_blank'>
              Spatial transcriptomic profiling of the tumor microenvironment in EGFR and KRAS mutant non-small cell lung cancer.
              </a>
            </td>
          <td>
            F. J. Bolte, Nick Natale, Samantha DiBenedetto, Margaret Moore, E. Stelow, Richard Hall, D. Gioeli
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background: Gallbladder cancer (GBC) is a rare but aggressive malignancy, often detected late due to early asymptomatic stages. Understanding cellular and molecular changes from normal tissue to high-grade intraepithelial neoplasia (HGIN) and invasive GBC is vital for identifying early biomarkers and therapeutic targets. Methods: We performed single-cell RNA sequencing on 98,113 cells derived from 2 normal adjacent tissues (NAT), 2 HGIN, and 6 GBC samples. The cellular diversity and heterogeneity, particularly within epithelial and immune cell populations in NAT–HGIN–GBC, were investigated utilizing single-cell RNA sequencing, bulk RNA sequencing (bulk RNA-seq), and 10 machine learning methodologies. Furthermore, the intercellular crosstalk between epithelial cells and tumor immune microenvironment cells was examined and validated through multiplex immunofluorescence staining. Results: The constructed cell atlas elucidated alterations in the immune landscape across various states of NAT–HGIN–GBC, highlighting a more pronounced inhibitory immune microenvironment in GBC. The epithelial subtype TOP2A+ Epi is markedly elevated in GBC and is correlated with a poor prognosis. Key genes associated with this subtype may include GMNN, CYTOR, KLK6, and BIRC5. Similarly, immunosuppressive macrophages, identified as TOP2A+ Macro, also increase along the NAT–HGIN–GBC sequence and are linked to reduced patient survival. Furthermore, TOP2A+ Macro and CD8+ exhausted T cells (CD8+ Tex) engage in intercellular communication with epithelial TOP2A+Epi cells via the TWEAK/FN14 signaling pathway, thereby promoting tumor progression and immune evasion in GBC. The findings were further corroborated through multiplex immunofluorescence staining conducted on specimens from patients. Conclusions: This study elucidates significant alteration in the cellular ecosystems and intercellular signaling within the tumor immune microenvironment across the NAT–HGIN–GBC sequence. It identifies TOP2A, TWEAK, and FN14 as potential biomarkers and therapeutic targets for GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f8ecb5a2d68898a868d59ee319a91eaeca48bb" target='_blank'>
              Single-cell analyses unravel ecosystem dynamics and intercellular crosstalk during gallbladder cancer malignant transformation
              </a>
            </td>
          <td>
            Zhaobin He, Jianqiang Cao, Xiqiang Wang, Shengbiao Yang, Huijie Gao, Yongzhe Yu, Zequn Di, Cheng Peng
          </td>
          <td>2025-05-16</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d1b8aeaba5a454b04d6690efe6420aea37c43e" target='_blank'>
              Stochastic modelling of prostate progenitor architecture
              </a>
            </td>
          <td>
            Christo Morison, Esther Baena, Weini Huang
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intra-tumor heterogeneity impacts disease progression and therapeutic resistance but remains poorly characterized by conventional histologic, immunophenotypic, and molecular approaches. Single-cell biophysical properties distinguish functional phenotypes complementary to these approaches, providing additional insight into cellular diversity. Here we link both buoyant mass and stiffness to gene expression to identify clinically relevant phenotypes within primary mantle cell lymphoma (MCL) cells, employing MCL as a model of biological and clinical diversity in human cancer. Linked measurements reveal that buoyant mass and stiffness characterize B-cell development states from naïve to plasma cell and correlate with expression of oncogenic B-cell receptor signaling genes such as BLK and CD79A. Additionally, changes in cell buoyant mass within primary patient specimens ex vivo correlate with sensitivity to Bruton’s Tyrosine Kinase inhibitors in vivo in MCL and chronic lymphocytic leukemia, another B-cell malignancy. These findings highlight the value of biophysical properties as biomarkers of response in pursuit of future precision therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222e5440bee40a1cfc03b8b975d31d717ee09614" target='_blank'>
              Integrating Single-Cell Biophysical and Transcriptomic Features to Resolve Functional Heterogeneity in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Ye Zhang, Lydie Debaize, Adam Langenbucher, Jenalyn Weekes, Ioulia Vogiatzi, Teemu P. Miettinen, Mingzeng Zhang, Emily Sumpena, Huiyun Liu, Sarah M. Duquette, L. Hackett, Jeremy Zhang, Sona Baghiyan, R. Redd, M. Aryee, M. Davids, Austin I. Kim, C. Ryan, David M. Weinstock, S. Manalis, M. Murakami
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1ca3901ac698b79cb29b7a3967fd687d4414a1" target='_blank'>
              Multiomic profiling of hypoxic glioblastoma stem cells reveals expansion of subpopulations with distinct epigenetic and CNV profiles
              </a>
            </td>
          <td>
            Adrianne Corseri, Travis Moore, Nicole Szczepanski, Hyeyeon Hwang, Alan Zdon, Gurkan Yardimci Galip, Nikos Tapinos
          </td>
          <td>2025-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The effectiveness of colorectal cancer (CRC) therapy is limited owing to the absence of treatments targeting drug-tolerant residual cancer cells. Although neoadjuvant therapy is effective, pathological examination of residual tumors has revealed the presence of small clusters of LGR5-positive cancer cells in the fibrous tissue. Here, we established a colorectal cancer cell line-derived organoid (CCD-organoid) regrowth model using a patient-derived cell line with cancer stem cell properties and demonstrated that it displayed the morphological characteristics of small clusters in clinical tissues. Time course analysis of single-cell RNA sequencing of the CCD-organoid regrowth model revealed various states and dynamic alterations within non-cycling cells. We identified subpopulations highly expressing protein translation-related genes RPL17 and EEF1G. To identify key signals for the transition of residual cancer cells to regrowth, we evaluated inhibitors targeting pathways such as the Wnt pathway, reactive oxygen species pathway, and RNA polymerase I pathway, highlighted in the single-cell RNA sequencing analysis. Only the polymerase I-inhibitor BMH-21 significantly reduced tumor growth both in vitro and in vivo, indicating the critical cell subpopulation driving recurrence. Our results demonstrate the possibility of a unique therapeutic target for CRC treatment targeting drug-tolerant residual cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa9b58cf763b8a32b666aa1599214c1a009f1d2" target='_blank'>
              Colorectal cancer cell line-derived organoid model with stem cell properties captures the regrowing state of residual cancer cells after neoadjuvant chemotherapy
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, T. Fujita, Atsuhiko Kato, Y. Zaitsu, Tomoko Jogo, Chie Kato, Takeshi Watanabe, Eri Hashimoto, Chiyoko Nishime, E. Fujii, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, H. Saeki, Hiroyuki Aburatani, Yoshihiko Maehara, Tatsumi Yamazaki
          </td>
          <td>2025-06-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="ABSTRACT Background Breast cancer is a highly heterogeneous disease, characterized by tumor and nontumor cells at various cell states. Ecotyper is an innovative machine learning framework that quantifies the tumor microenvironment and delineates the tumor ecosystem, demonstrating clinical significance. However, further validation is needed in breast cancer. Methods Ecotyper was applied to identify multiple cellular states and tumor ecotypes using large‐scale breast cancer bulk sequencing data, followed by a detailed analysis of their associations with clinical classification, molecular subtypes, survival prognosis, and immunotherapy response. Identified subtypes were further characterized using single‐cell and spatial data sets to reveal molecular profiles. Results In a comprehensive analysis of 6578 breast cancer samples from four data sets, Ecotyper identified 69 cellular states and 10 tumor ecotypes. Of these, 37 cellular states significantly correlated with overall survival. Notably, specific states within epithelial cells, macrophages/monocytes, and fibroblasts were linked to a worse prognosis. CE2 abundance was identified as the most significant marker indicating unfavorable prognosis and was further validated in an additional data set of 116 HER2‐negative patients. These biomarkers also indicated the efficacy of neoadjuvant immunotherapy in breast cancer. CE2‐high cancers were characterized by an abundance of basal‐like epithelial cells, scant lymphocytic infiltration, and activation of hypoxia signaling. Single‐cell analysis showed that CE2‐high areas were rich in SPP1‐positive tumor‐associated macrophages(TAM), basal‐like epithelial cells, and hypoxic cancer‐associated fibroblasts(CAF). Spatially, these regions were often peripheral in triple‐negative breast cancer, adjacent to fibrotic/necrotic zones. Multiplex immunofluorescence confirmed the enrichment of SPP1+CD68+TAM and HIF1A+SMA+CAF in hypoxic triple‐negative breast cancer (TNBC) regions. Conclusions Ecotyper identified novel biomarkers for breast cancer prognosis and treatment prediction. The CE2‐high region may represent a hypoxic immune‐suppressive niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed777dbf5c26fef9472ca8bd002f079780d98595" target='_blank'>
              The Identification of Novel Prognostic and Predictive Biomarkers in Breast Cancer via the Elucidation of Tumor Ecotypes Using Ecotyper
              </a>
            </td>
          <td>
            Feng Du, J. Ju, F. Zheng, Song-Lin Gao, Peng Yuan
          </td>
          <td>2025-05-27</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Fibroblasts play essential roles in cancer progression, exhibiting activation states that can either promote or inhibit tumor growth. Understanding these differential activation states is critical for targeting the tumor microenvironment (TME) in cancer therapy. However, traditional molecular markers used to identify cancer-associated fibroblasts are limited by their co-expression across multiple fibroblast subtypes, making it difficult to distinguish specific activation states. Morphological and motility characteristics of fibroblasts reflect their underlying gene expression patterns and activation states, making these features valuable descriptors of fibroblast behavior. This study proposes an artificial intelligence-based classification framework to identify and characterize differentially activated fibroblasts by analyzing their morphodynamic and motile features. We extract these features from label-free live-cell imaging data of fibroblasts co-cultured with breast cancer cell lines using deep learning and machine learning algorithms. Our findings show that morphodynamic and motile features offer robust insights into fibroblast activation states, complementing molecular markers and overcoming their limitations. This biophysical state-based cellular classification framework provides a novel, comprehensive approach for characterizing fibroblast activation, with significant potential for advancing our understanding of the TME and informing targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1693652b380d377e2983bb1084b665960d61177d" target='_blank'>
              Classification of differentially activated groups of fibroblasts using morphodynamic and motile features
              </a>
            </td>
          <td>
            Minwoo Kang, Chanhong Min, Somayadineshraj Devarasou, Jennifer H. Shin
          </td>
          <td>2025-05-15</td>
          <td>APL Bioengineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd1f3cec19d8b2ce70a881702528c91746b1cae" target='_blank'>
              CellLENS enables cross-domain information fusion for enhanced cell population delineation in single-cell spatial omics data.
              </a>
            </td>
          <td>
            Bokai Zhu, Sheng Gao, Shuxiao Chen, Yuchen Wang, Jason Yeung, Yunhao Bai, Amy Y. Huang, Y. Yeo, Guanrui Liao, Shulin Mao, Z. G. Jiang, Scott J Rodig, Ka-Chun Wong, Alex K. Shalek, GP Nolan, Sizun Jiang, Zongming Ma
          </td>
          <td>2025-05-22</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Pancreatic adenosquamous cancer (PASC) is an extremely rare subtype of pancreatic cancer characterized by a poorer prognosis and higher likelihood of metastasis compared with the more prevalent pancreatic ductal adenocarcinoma (PDAC). Although genomic changes during PASC tumorigenesis have been documented, the corresponding alterations in the tumor immune microenvironment (TIME) remain inadequately elucidated. Therefore, this study aims to analyze the immune landscape of PASC by employing multiplex immunohistochemistry (mIHC) and digital image analysis. Methods In this study, we analyzed four independent cohorts comprising 120 patients with PASC and 386 patients with PDAC. We employed mIHC to quantify three in situ panels of immuno-oncology-related biomarkers at subcellular resolution. We then used five samples to perform laser capture microdissection, RNA sequencing, and whole-exome sequencing to explore the underlying mechanisms of the compartment-specific immune phenotypes in PASC. Results Our findings revealed a more immunosuppressive TIME in PASC compared with PDAC, characterized by a decreased abundance of T cells. Immune cell types indicative of enhanced immune surveillance, including cytotoxic and memory T cells and antigen-experienced T cells, were present at significantly lower densities in PASC compared with PDAC. Conversely, some immunosuppressive macrophage phenotypes exhibited increased distribution in PASC. Immunosuppressive immune cells (ICs) were abundant, enriched within stromal regions, highly heterogeneous across tumors, and exhibited distinct distributions between squamous cell (SQC) and adenocarcinoma (ADC) compartments in PASC. Notably, the TIME of SQC compartments harbored more exhausted T cells compared with synchronous ADC compartments, indicating attenuated immune surveillance during squamous transformation. Transcriptomic profiling of microdissected SQC and ADC regions revealed immune exhaustion signatures and downregulated T-cell differentiation pathways in SQC compartments, alongside altered antigen presentation machinery and elevated tumor mutational burden, suggesting squamous-specific tumor-associated antigens with potential immunotherapeutic relevance. Beyond differences in IC density, we observed closer spatial proximity of CD45RO+ and PD-1+CD3+CD8+ T cells to tumor cells within 10, 20 and 30 µm ranges in PASC compared with PDAC, with variations by histological subregion. Furthermore, we found distinct expression patterns of the programmed cell death protein-1 (PD-1)/programmed death ligand 1 (PD-L1) and T-cell immunoreceptor with immunoglobulin and the ITIM domain (TIGIT)/CD155 axes in the PASC TIME associated with survival outcomes. Notably, TIGIT+CD8+ T cells and CD155+ CD68+macrophages, along with their proximity to tumor cells, served as independent prognostic indicators. These findings were validated in an independent cohort study. Conclusion Our study advances the understanding of PASC by providing updated insights into its immunoenvironmental features. These findings underscore the potential of targeting immune checkpoint pathways, particularly the TIGIT/CD155 and PD-1/PD-L1 axes, as a therapeutic strategy for PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d936da9d89fc333e1aeb534f40e042c28a5208e" target='_blank'>
              Attenuated immune surveillance during squamous cell transformation of pancreatic adenosquamous cancer defines new therapeutic opportunity for cancer interception
              </a>
            </td>
          <td>
            Xinyuan Chen, Shanyue Sun, Shuofeng Li, Shuangni Yu, Jie Chen, Xian-liang Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35658443146621f3cb5a98587b659629d87dcfdc" target='_blank'>
              Longitudinal multi-omic profiling uncovers immune escape and predictors of response in multiple myeloma
              </a>
            </td>
          <td>
            Denis Ohlstrom, William C. Pilcher, Marina E. Michaud, Chaitanya R. Acharya, Sarthak Satpathy, Edgar Gonzalez-Kozlova, R. Jayasinghe, Katherine E. Ferguson, Hope L. Mumme, Shivani Nanda, Yizhe Song, Sowmitri Karthikeya Siddhartha Mantrala, D. Karagkouni, Jessica Schulman, Nick Pabustan, Junia Vieira Dos Santos, Daniel W. Sherbenou, J. Keats, Alex Gout, S. Foltz, A. Laganà, T. Kourelis, Ravi Vij, M. Dhodapkar, D. Avigan, H. Cho, L. Baughn, Ajay Nooka, S. Lonial, Shaji K. Kumar, Mehmet K Samur, Ioannis S. Vlachos, Li Ding, Sacha Gnjatic, George Mulligan, Manoj K. Bhasin
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Quiescence (reversible cell-cycle arrest) and senescence (irreversible arrest) are challenging to distinguish due to a lack of specific biomarkers, yet both arise simultaneously after chemotherapy. While senescence suppresses tumors by limiting proliferation and recruiting the immune system, quiescent cancer cells evade future therapies and may resume proliferation. Here, we pair time-lapse imaging of cell-cycle dynamics with single-cell RNA-sequencing after etoposide treatment to differentiate these states, linking heterogeneous cell-cycle phenotypes to the transcriptomic landscape. We identify diverse senescent types (senotypes) and link them to two arrest pathways – a gradual path arising after a standard mitosis-to-G0 transition, and a rarer but direct path driven by a mitotic slip. Using pseudotime trajectory analysis, we find that senescent phenotypes begin to manifest early and gradually along the first trajectory, even in shallow quiescent cells. These data support a model wherein, following chemotherapy, quiescence and senescence exist on a continuum of cell-cycle withdrawal at a transcriptome-wide level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e448f12e8844e81bc7406643cd94547023bd4f4c" target='_blank'>
              Single-cell RNA sequencing reveals distinct senotypes and a quiescence-senescence continuum at the transcriptome level following chemotherapy
              </a>
            </td>
          <td>
            Brianna Fernandez, Victor J. Passanisi, Humza M. Ashraf, Sabrina L. Spencer
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Metastatic hormone-naïve prostate cancer (mHNPC) is an infrequent form of this tumor type that is characterized by metastasis at the time of diagnosis and accounts for up to 50% of prostate cancer-related deaths. Despite the extensive characterization of localized and metastatic castration-resistant prostate cancer, the molecular characteristics of mHNPC remain largely unexplored. Results Here, we provide the first extensive transcriptomics characterization of primary tumor specimens from patients with mHNPC. We generate discovery and validation bulk and single-cell RNA-seq datasets and perform integrative computational analysis in combination with experimental studies. Our results provide unprecedented evidence of the distinctive transcriptional profile of mHNPC and identify stroma remodeling as a predominant feature of these tumors. Importantly, we discover a central role for the SRY-box transcription factor 11 (SOX11) in triggering a heterotypic communication that is associated with the acquisition of metastatic properties. Conclusions Our study will constitute an invaluable resource for a profound understanding of mHNPC that can influence patient management. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03623-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bec724fe7758a24264be0a981015a60432a2227" target='_blank'>
              Transcriptional analysis of metastatic hormone-naïve prostate cancer primary tumor biopsies reveals a relevant role for SOX11 in prostate cancer cell dissemination
              </a>
            </td>
          <td>
            Natalia Martín-Martín, Saioa Garcia-Longarte, Jon Corres-Mendizabal, Uxue Lazcano, I. Astobiza, Laura Bozal-Basterra, Nicolas Herranz, Hielke van Splunder, Onintza Carlevaris, Mikel Pujana-Vaquerizo, María Teresa Blasco, Ana M Aransay, A. Rosino, J. Tudela, Daniel Jimenez, Alberto Martinez, Andrei Salca, Aida Santos-Martín, Sofía Rey, Aitziber Ugalde-Olano, David Gonzalo, M. Graupera, R. Gomis, J. Mateo, Miguel Unda, E. González-Billalabeitia, Ana Loizaga-Iriarte, Isabel Mendizabal, Arkaitz Carracedo
          </td>
          <td>2025-06-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78beb6714803e0c6118158e2cf27d989400224a0" target='_blank'>
              Single Cell RNA Sequencing Reveals Gene Expression Continuums Along the Spatial Hierarchy of the Pulmonary Circulation
              </a>
            </td>
          <td>
            S. Sveiven, C. Knutsen, Fabio Zanini, David N Cornfield, C. Alvira
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract The tumor microenvironment (TME) significantly influences cancer prognosis and therapeutic outcomes, yet its composition remains highly heterogeneous, and currently, no highly accessible, high‐throughput method exists to define it. To address this complexity, the TMEclassifier, a machine‐learning tool that classifies cancers into three distinct subtypes: immune Exclusive (IE), immune Suppressive (IS), and immune Activated (IA), is developed. Bulk RNA sequencing categorizes patient samples by TME subtype, and in vivo mouse model validates TME subtype differences and differential responses to immunotherapy. The IE subtype is marked by high stromal cell abundance, associated with aggressive cancer phenotypes. The IS subtype features myeloid‐derived suppressor cell infiltration, intensifying immunosuppression. In contrast, the IA subtype, often linked to EBV/MSI, exhibits robust T‐cell presence and improved immunotherapy response. Single‐cell RNA sequencing is applied to explore TME cellular heterogeneity, and in vivo experiments demonstrate that targeting IL‐1 counteracts immunosuppression of IS subtype and markedly improves its responsiveness to immunotherapy. TMEclassifier predictions are validated in this prospective gastric cancer cohort (TIMES‐001) and other diverse cohorts. This classifier could effectively stratify patients, guiding personalized immunotherapeutic strategies to enhance precision and overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e59035a4145d3d05ce3897eefa444405fb68dd" target='_blank'>
              Immunotyping the Tumor Microenvironment Reveals Molecular Heterogeneity for Personalized Immunotherapy in Cancer
              </a>
            </td>
          <td>
            D. Zeng, Yunfang Yu, Wenjun Qiu, Q. Ou, Qianqian Mao, Luyang Jiang, Jianhua Wu, Jiani Wu, Huiyan Luo, Peng Luo, Wenchao Gu, N. Huang, S. Zheng, Shaowei Li, Yonghong Lai, Xiatong Huang, Yiran Fang, Qiongzhi Zhao, R. Zhou, Huiying Sun, Wei Zhang, J. Bin, Yulin Liao, Masami Yamamoto, Tetsuya Tsukamoto, Sachiyo Nomura, Min Shi, Wangjun Liao
          </td>
          <td>2025-05-28</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Melanomas are genetically heterogeneous, displaying mitogen-activated protein kinase mutations and homozygous loss of tumor suppressor genes. Mouse models combining such mutations produce fast-growing tumors. In contrast, rare, slow-growing tumors arise in mice combining Braf activation with heterozygous loss of Pten. Here we show that similar tumors can arise in albino mice bearing only a Braf mutation. Incidence kinetics suggest a stochastic event underlies tumorigenesis in tumors that arise with only a Braf mutation, yet de novo mutations or structural variants that could explain the incidence of most tumors could not be found. Single-cell transcriptomics of tumors identify a cell type resembling “neural crest-like” cells in human and mouse melanomas. These exist in normal mouse skin, expand upon Braf activation, and persist through serial transplantation; analyses of gene expression suggest they serve as precursors of malignant cells. This state may serve as an intermediate on a slow path to malignancy that may provide a diagnostically and therapeutically important source of cellular heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a8b6d33ff26b499d62c39adf32521b06fa0db2" target='_blank'>
              Uncovering minimal pathways in melanoma initiation
              </a>
            </td>
          <td>
            Hui Xiao, Jessica Shiu, Chi-Fen Chen, Jie Wu, Peijie Zhou, Sahil S. Telang, Rolando Ruiz-Vega, Robert A Edwards, Qing Nie, A. D. Lander, Anand K. Ganesan
          </td>
          <td>2025-06-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afbfafb6e752991df103b290fb3be622c923aa5e" target='_blank'>
              Immune Spatial Organization Predicts Metastasis Risk in Aggressive Localized Prostate Cancer
              </a>
            </td>
          <td>
            David D. Yang, Aya Abdelnaser, Jeremiah Wala, Alexander J Haas, Alfred A Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Jihye Park, Martin T King, P. L. Nguyen, T. Choueiri, A. Tewari, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="
 e13119


 Background:
 The tumor microenvironment (TME) of primary breast tumors (PTs) and breast-to-liver metastases shows heterogeneous infiltration and interactions between tumor, immune, and stromal cells. Hepatic metastases exhibit a more immunosuppressive TME with fewer cytotoxic T cells (CTLs) relative to PTs, which may contribute to increased tumor growth and resistance to systemic therapies. To investigate the immune cell infiltration and immune-tumor interactions in PT and liver metastases, we utilized whole transcriptome analysis and immunofluorescence-based protein expression as part of spatial TME profiling.
 Methods:
 We identified 13 chemo-naïve PTs and 3 breast-to-liver metastases. Formalin-fixed paraffin-embedded (FFPE) samples were used for hematoxylin and eosin (H&E) staining and spatial transcriptomics (ST) profiling with the 10X Genomics Visium CytAssist and Visium HD platforms. The H&E slides were manually annotated in collaboration with a board-certified pathologist, and ST data were deconvoluted using breast-specific scRNA-sequenced references and canonical marker labeling.
 Results:
 Our analysis of the 13 PTs and 3 breast-to-liver metastases revealed significant heterogeneity in tumor clusters, immune cell infiltration, gene set enrichments, and gene signatures between primary and visceral TMEs. Unsupervised clustering of hormone receptor-positive (HR+) HER2-negative (HER2) samples demonstrated considerable intratumoral and intertumoral variability, with differences in the expression of tumor markers (EPCAM, ER, PR, ERBB2, KRT14, KRT18), hypoxia-related signatures (HIF1a), and pro-angiogenic drivers (VEGF). Breast-to-liver metastases showed higher expression of pathways involved in proliferation, angiogenesis, and anti-apoptosis compared to primary tumors. Hepatic metastases were enriched for HALLMARK gene sets related to proliferation and metabolic pathways, including PI3K/AKT/mTOR, mTORC1 signaling, E2F targets, MYC targets, G2M checkpoint, mitotic spindle, glycolysis, oxidative phosphorylation, and fatty acid metabolism when compared to primary tumors. In contrast, primary tumors were enriched for immune-related pathways (TNF-α, interferon-γ, inflammatory response, IL-6 JAK/STAT3), epithelial-mesenchymal transition (EMT), myogenesis, and angiogenesis gene signatures, which are consistent with their metastatic potential. Cellular profiling of liver lesions revealed higher myeloid suppressor cells (TAMs and MDSCs): CD8
 +
 ratio with shorter cell-cell distances relative to PTs suggesting a key role in immune evasion, tumor progression, and therapeutic resistance.
 Conclusions:
 ST data from primary and metastatic breast cancer samples revealed notable tumor heterogeneity, characterized by highly active metabolic and proliferative pathways, along with a higher abundance of myeloid cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/389d6dee563fb2269c25d3de12654b70850dddf7" target='_blank'>
              Spatial transcriptomics and functional mapping of the suppressive TME in primary breast tumor and breast-to-liver metastases.
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Cheol Park, Matthew Jacobo, Marqus Dela Cruz, Michelle Li, Sabrina Carrel, Elexa Rallos, Michael Press, Evanthia T. Roussos Torres
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mitochondrial (MT) mutations serve as natural genetic markers for inferring clonal relationships using single cell sequencing data. However, the fundamental challenge of MT mutation-based lineage tracing is automated identification of informative MT mutations. Here, we introduced an open-source computational algorithm called “MitoTracer”, which accurately identified clonally informative MT mutations and inferred evolutionary lineage from scRNA-seq or scATAC-seq samples. We benchmarked MitoTracer using the ground-truth experimental lineage sequencing data and demonstrated its superior performance over the existing methods measured by high sensitivity and specificity. MitoTracer is compatible with multiple single cell sequencing platforms. Its application to a cancer evolution dataset revealed the genes related to primary BRAF-inhibitor resistance from scRNA-seq data of BRAF-mutated cancer cells. Overall, our work provided a valuable tool for capturing real informative MT mutations and tracing the lineages among cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b015ab9e9a2db2d9420366fceee8ad8d28fa830" target='_blank'>
              MitoTracer facilitates the identification of informative mitochondrial mutations for precise lineage reconstruction
              </a>
            </td>
          <td>
            Xuexin Yu, Jing Hu, Yuhao Tan, Mingyao Pan, Hongyi Zhang, Bo Li
          </td>
          <td>2025-06-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Colon cancer (CC) is one of the most common and lethal cancers worldwide, with rising incidence rates in both developed and developing countries. Although advances in treatments such as surgery, chemotherapy, and targeted therapies have been made, prognosis for advanced colon cancer, particularly with metastasis, remains poor. Recent studies highlight the significant role of post-transcriptional modifications like acetylation in cancer biology, affecting processes like gene transcription, metabolism, and tumor progression. Methods This study applied multi-omics analyses, including single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and Mendelian randomization. Data were obtained from public datasets like GSE132465, UCSC Xena, and GeneCards. We focused on acetylation-related genes, specifically NAT10 and GNE, using scoring methods, cell–cell interaction models, and survival analyses to investigate their role in colon cancer development, metastasis, and immune evasion. Results This study identifies that NAT10 is highly expressed in epithelial cells of colorectal cancer (CC) and is closely associated with tumor progression and metastasis. Single-cell RNA sequencing analysis revealed that NAT10-positive epithelial cells exhibited strong interactions with myeloid cells and T cells, with significant differences in cell–cell communication (p < 0.05). Based-on-summary-data Mendelian randomization (SMR) analysis further supports a causal relationship between NAT10 and colorectal cancer. In the MR analysis, a significant positive correlation was observed between NAT10 and colorectal cancer risk using summary data from genome-wide association studies (GWAS) and expression quantitative trait loci (eQTL) studies (β_SMR = 0.004, p_SMR = 0.041, p_HEIDI = 0.737). These findings suggest that NAT10 may serve as a pathogenic factor in colorectal cancer development, providing additional genetic evidence that links this acetylation-related gene to colorectal cancer. Survival analysis further demonstrated that NAT10-positive epithelial cells are associated with poorer prognosis. In the TCGA dataset, patients with NAT10-positive epithelial cells exhibited a significantly shorter disease-free survival (DFS) (p = 0.012). Unlike GNE-positive cells, NAT10-positive epithelial cells exhibited immune escape characteristics, and TIDE analysis indicated that NAT10-positive epithelial cells were associated with a lower response to immune checkpoint blockade therapy (p = 1.3e−5), suggesting that they may impair the efficacy of immunotherapy by promoting immune evasion. In contrast, GNE was also significantly expressed in epithelial cells of colorectal cancer, but its role differs from that of NAT10. GNE-positive epithelial cells demonstrated strong communication with immune cells, particularly in interactions between myeloid cells and T cells through receptor-ligand pairs. Despite the important role of GNE-positive epithelial cells in the tumor microenvironment, their association with immune escape is weaker compared to NAT10. Survival analysis revealed that GNE-positive epithelial cells were associated with a better prognosis (p = 0.015). In the TCGA dataset, patients with GNE-positive epithelial cells displayed longer disease-free survival (DFS), contrary to the results from the SMR analysis. Conclusions Leveraging SMR and multi-omics analysis, this study highlights the significant role of acetylation-related genes, particularly NAT10, in colon cancer. The findings suggest that acetylation modifications in epithelial cells contribute to immune evasion and cancer progression. NAT10 could serve as a promising biomarker and therapeutic target for early diagnosis and targeted therapy, offering new avenues for improving colon cancer treatment and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37348c834755949c2631ebe0e9fc050634d0c0ce" target='_blank'>
              Deciphering the role of acetylation-related gene NAT10 in colon cancer progression and immune evasion: implications for overcoming drug resistance
              </a>
            </td>
          <td>
            Xuancheng Zhou, Xun Sang, Lai Jiang, Shengke Zhang, Chenglu Jiang, Yuheng Gu, Yiping Fu, Guanhu Yang, Jieying Zhang, Hao Chi, Binbin Wang, Xiaolin Zhong
          </td>
          <td>2025-05-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Objective To investigate the metabolic remodeling and functional specialization of immune cells within the tumor microenvironment (TME) of bone tumors, including Ewing’s sarcoma, osteosarcoma, and giant cell tumor of bone, through high-resolution single-cell RNA sequencing (scRNA-seq) analysis. Methods Immune cells were isolated from 13 bone tumor samples and profiled via scRNA-seq to delineate cellular compositions, metabolic adaptations, and intercellular communication networks. Differential gene expression analysis, metabolic pathway enrichment, and pseudotime trajectory inference were employed to characterize functional states and differentiation processes of immune cell subsets. Results We identified 12 major immune cell clusters with distinct functional and metabolic characteristics. Naïve T cells exhibited amino acid metabolism-dependent activation potential, whereas NK cells relied on lipid metabolism and the TCA cycle for cytotoxic activity. Macrophage subsets demonstrated functional divergence: C06 macrophages adopted lipid metabolism to facilitate immunosuppression and tissue repair, while C04 macrophages displayed pro-inflammatory characteristics associated with complement activation. Intercellular signaling analysis revealed FN1 as a central regulator of immune coordination, governing cell adhesion, migration, and homeostasis within the TME. Conclusion This study provides novel insights into the metabolic and functional plasticity of immune cells in bone tumor TMEs, underscoring the critical role of metabolic remodeling in immune regulation. Our findings highlight potential therapeutic targets for modulating immune cell function and offering new avenues to improve treatment outcomes for patients with bone tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06346-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e0d19e0f285642428880ab85bf21db7c2bbae62" target='_blank'>
              Single-cell transcriptomics reveals metabolic remodeling and functional specialization in the immune microenvironment of bone tumors
              </a>
            </td>
          <td>
            Jun Chen, Na Cui, Shaokai He, Chun-Yan Xia, Wei-Qing Li
          </td>
          <td>2025-05-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to chemotherapy of pancreatic ductal adenocarcinoma (PDAC) is largely driven by intratumoral heterogeneity (ITH) due to tumor cell plasticity and clonal diversity. In order to develop novel strategies to overcome this defined mechanism of resistance, tools to monitor and quantify ITH in a rapid and scalable fashion are needed urgently. Here, we employed label-free digital holographic microscopy (DHM) to characterize ITH in PDAC. We established a robust experimental and machine learning analysis pipeline to perform single cell phenotyping based on DHM-derived phase images of PDAC cells in suspension. Importantly, we are able to detect dynamic changes in tumor cell differentiation and heterogeneity of distinct PDAC subtypes upon induction of epithelial-to-mesenchymal transition and under treatment-imposed pressure in murine and patient-derived model systems. This platform allows us to assess phenotypic ITH in PDAC on a single cell level in real-time. Implementing this technology into the clinical workflow has the potential to fundamentally increase our understanding of tumor heterogeneity during evolution and treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b97958c7f42c79b39377ba5cf9b12106537163c" target='_blank'>
              Label-free single cell phenotyping to determine tumor cell heterogeneity in pancreatic cancer in real-time.
              </a>
            </td>
          <td>
            Katja Wittenzellner, M. Lengl, S. Röhrl, Carlo Maurer, C. Klenk, A. Papargyriou, Laura Schmidleitner, Nicole Kabella, Akul R Shastri, D. Fresacher, Farid Harb, Nawal Hafez, S. Bärthel, Daniele Lucarelli, Carmen Escorial-Iriarte, F. Orben, R. Öllinger, Ellen Emken, L. Fricke, Joanna Madej, Patrick Wustrow, I. Demir, Helmut Friess, Tobias Lahmer, Roland M. Schmid, Roland Rad, Günter Schneider, Bernhard Kuster, Dieter Saur, O. Hayden, Klaus Diepold, Maximilian Reichert
          </td>
          <td>2025-05-27</td>
          <td>JCI insight</td>
          <td>1</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac2fa84e64df92a9b47c54f45baf0d93294fea97" target='_blank'>
              Substrate heterogeneity promotes cancer cell dissemination
              </a>
            </td>
          <td>
            Zuzana Dunajova, Saren Tasciyan, Juraj Májek, J. Merrin, E. Sahai, M. Sixt, É. Hannezo
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 e20030


 Background:
 Tumoral heterogeneity has been widely acknowledged to influence clinical outcome of lung adenocarcinoma (LUAD). This study investigated spatial gene profiling of different pathological subtypes of early-stage invasive LUAD with Stereo-seq transcriptomics technique. The tumor microenvironment (TME) was decoded with cellular and molecular events at perspectives of tumoral activities and immune activities.
 Methods:
 We collected spatial RNA profiles of nine Stereo-seq chips (captured area of 1 x1 cm) from 4 invasive LUAD patients. Pathologist identified histological subtypes including lepidic adenocarcinoma (LP), acinar adenocarcinoma (AC) and other invasive adenocarcinoma like papillary adenocarcinoma (IA) on adjacent H&E images. Spatial data matrix was obtained based on pathological annotation accordingly. Spatially resolved gene expression of each subtypes was obtain based on pathological annotation to conduct high resolution analyses. TME functional units were calculated by analysis of cellular neighborhoods (CNs) based on cell compositions within resolution of bin 100 (window of 50um x 50um). Sub-populations of tumoral cells were stratified by molecular functions (GSEA and GSVA score) and stemness (CytoTRACE).
 Results:
 All pathological subtypes shared the identical 10 heterogeneous CNs and 5 tumoral clusters (TM) with various propotions. Tumor cluster 0 (TM0) and TM3 were identified as the malignant population with significant high stemness and significant high activities of MTORC1 signaling, TGF-BETA signaling, PI3K-ATK signaling and hypoxia. AC and IA were mainly composed of TM0 and TM3. Compared to TM0, TM3 had higher EMT activity. We further identified two immune-related CNs (CN4 and CN8) with significant high GSVA score of Interferon-Alpha response, Interferon-Gamma response and Inflammatory response. Spatial association showed that the lymphoid-cells-dominated CN4 was enriched in LP and IA and nearly absent in AC. Meanwhile, the myeloid-cells-dominated CN8 was enriched in AC. GSEA score showed that immune activity of CN4 was more intense in IA when compared to LP. Notably, TM0 mainly present in CN8, and TM3 mainly present in CN4. Therefore, the LP was speculated to reach an inert and statical status. TME of AC was featured as colocalization of myeloid cells and tumor cells with high stemness. TME of IA was featured as colocalization of lymphoid cells and tumor cells with high EMT activity.
 Conclusions:
 This is the first study to delineate the homogeneity of tumor cells and heterogeneity of immune cells among different pathological types of NSCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344f597f0ff9f98416e718719f81b290a2438fe0" target='_blank'>
              Decoding spatial transcriptomics-based heterogeneity of TME in early-stage invasive pulmonary adenocarcinoma.
              </a>
            </td>
          <td>
            Xin Liu, Jingyi Tian, Z. Shang, C. Xiang, Fei Teng, Jiajun Zhang, Yuchen Han
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a significant global health challenge despite advancements in screening, HPV vaccination, and therapeutic strategies. Tumor heterogeneity, driven by epigenetic modifications, affects immune evasion, metastasis, and treatment response. Cancer-associated fibroblasts (CAFs) play a crucial role in CC progression and therapy resistance. Single-cell sequencing offers new insights but remains underutilized in CC research. This study integrates single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and deconvolution analysis to identify key genes and immunotherapy targets. By constructing a prognostic model and exploring the immune microenvironment, we aim to provide novel insights into CC pathogenesis and potential therapeutic strategies. Methods We utilized scRNA-seq, spatial transcriptomics, deconvolution analysis, and pseudotime trajectory mapping to delineate fibroblast subtypes within the tumor immune microenvironment (TIME) of CC. Functional annotations, differential gene expression profiling, cell–cell communication pathways, and transcription factor networks were systematically analyzed. A prognostic model based on bulk RNA-seq data was constructed and validated through survival analysis, with correlations to immune microenvironment characteristics. Functional experiments investigated the role of SDC1, a critical mediator of fibroblast-tumor crosstalk. Additionally, Fibroblast–tumor cell co-culture systems and functional assays were employed to investigate the paracrine role of SDC1. The CAF MYH11⁺ subpopulation was isolated via fluorescence-activated cell sorting (FACS). Multiplex immunofluorescence and immunohistochemical analyses were performed on both cultured cells and human cervical cancer tissue samples to characterize the spatial distribution and dynamic remodeling of MYH11 during stromal reorganization. Results Six distinct fibroblast subtypes were identified, including the C0 MYH11 + fibroblasts, which exhibited unique roles in stemness maintenance, metabolic activity, and immune regulation. Spatial and functional analyses revealed that the C0 subtype is central to tumor-fibroblast interactions, particularly through the MDK-SDC1 signaling axis. The prognostic model incorporating fibroblast-specific markers demonstrated robust predictive power for patient survival outcomes. Additionally, in vitro SDC1 knockdown significantly inhibited CC cell proliferation, migration, and invasion. Fibroblasts show spatially regulated heterogeneity, with activation markers enriched in the tumor zone and MYH11 highest in normal zones, indicating dynamic stromal remodeling. C0 MYH11 + CAF Promotes Tumor Cell Proliferation, Migration, and Inhibits Apoptosis via Soluble SDC1. Conclusion Our results illustrate, in some ways, the possible immunomodulatory and tumor supporting roles of CAFs in CC TIME and highlight the possibility that the MDK-SDC1 pathway is a promising therapeutic target. This study not only promotes a partially new understanding of temporal heterogeneity in CC, but also provides a possible reference base for the development of new biomarkers and immunotherapy approaches to improve clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03432-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d1387a936b1ef6d7e045f6c7bcff60d568a6f7" target='_blank'>
              Deciphering the tumor immune microenvironment: single-cell and spatial transcriptomic insights into cervical cancer fibroblasts
              </a>
            </td>
          <td>
            Zhiheng Lin, Youwei Zhou, Zhenran Liu, Wenyang Nie, Hengjie Cao, Shengnan Li, Xuanling Li, Lijun Zhu, Guangyao Lin, Yanyu Ding, Yi Jiang, Zuxi Gu, Lianwei Xu, Zhijie Zhao, Huabao Cai
          </td>
          <td>2025-07-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) presents substantial heterogeneity and unpredictability in its progression. Despite therapeutic advancements, mortality from advanced PCa remains a significant challenge. Understanding the intercellular communication within the tumor microenvironment (TME) is critical for uncovering mechanisms driving tumorigenesis and identifying novel therapeutic targets. Methods We employed an integrative approach combining bulk RNA sequencing, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics to investigate interactions between FAP+ fibroblasts and tumor-associated macrophages in PCa. Key findings were validated using immunohistochemical and immunofluorescence staining techniques. Results Analysis of 23,519 scRNA-seq data from 23 prostate samples revealed a pronounced accumulation of FAP+ fibroblasts in tumor tissues. Spatial transcriptomics and bulk RNA sequencing demonstrated strong associations between FAP+ fibroblasts and SPP1+ macrophages. Notably, tumor-specific intercellular signaling pathways, such as CSF1/CSF1R and CXCL/ACKR1, were identified, highlighting their potential role in fostering an immunosuppressive TME. Conclusion Our findings unveil a distinct pattern of crosstalk between FAP+ fibroblasts and SPP1+ macrophages in PCa, shedding light on potential therapeutic targets for advanced PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d000b5dbafb928d866718b2a6f1d1452be0aec5" target='_blank'>
              Intercellular communication between FAP+ fibroblasts and SPP1+ macrophages in prostate cancer via multi-omics
              </a>
            </td>
          <td>
            Tingting Wu, Xinyu Li, Fei Zheng, Hanchao Liu, Yang Yu
          </td>
          <td>2025-05-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9faa1273d62a3ded4dd4ed42a59e3c9f4c050ab5" target='_blank'>
              Spatial omics technology potentially promotes the progress of tumor immunotherapy.
              </a>
            </td>
          <td>
            Zhen Lan, Yuanyuan Yang, Lingling Li, Chaoguan Wang, Zhenqiang Sun, Qiming Wang, Yang Liu
          </td>
          <td>2025-06-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background The dynamics of single-nucleotide variants (SNVs) play a critical role in understanding tumor development, yet their influence on shaping tumor microenvironments remains largely unexplored. Spatial transcriptomics offers an opportunity to map SNVs within the tumor context, potentially uncovering new insights into tumor microenvironment dynamics. Results This study developed SpatialSNV for identifying effective SNVs across tumor sections using multiple spatial transcriptomics platforms. The analysis revealed that SNVs reflect regional tumor evolutionary traces and extend beyond RNA expression changes. The tumor margins exhibited a distinct mutational profile, with novel SNVs diminishing in a distance-dependent manner from the tumor boundary. These mutations were significantly linked to inflammatory and hypoxic microenvironments. Furthermore, spatially correlated SNV groups were identified, exhibiting distinct spatial patterns and implicating specific roles in tumor–immune system crosstalk. Among these, critical SNVs such as S100A11L40P in colorectal cancer were identified as tumor region–specific mutations. This mutation, located within exonic nonsynonymous regions, may produce neoantigens presented by HLAs, marking it as a potential therapeutic target. Conclusions SpatialSNV represents a promising framework for unraveling the mechanisms underlying tumor–immune crosstalk within the tumor microenvironment by leveraging spatial transcriptomics and SNV-based tissue domain characterization. This approach is designed to be scalable, integrative, and adaptable, making it accessible to researchers aiming to explore tumor heterogeneity and identify therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32eb52bad2a391e12368299093f1326ca08dc55e" target='_blank'>
              SpatialSNV: A novel method for identifying and analyzing spatially resolved SNVs in tumor microenvironments
              </a>
            </td>
          <td>
            Yi Liu, Fan Zhu, Xinxing Li, Xiangyu Guan, Yong Hou, Yu Feng, Xuan Dong, Young Li
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is a clinically and molecularly heterogeneous disease. While there is increasing recognition of diverse tumor phenotypes across patients, less is known about the molecular and phenotypic heterogeneity present within an individual. In this study, we aimed to define the patterns, extent, and consequences of inter- and intra-tumoral heterogeneity in lethal prostate cancer. By combining and integrating in situ tissue-based and sequencing approaches, we analyzed over 630 tumor samples from 52 mPC patients. Our efforts revealed phenotypic heterogeneity at the patient, metastasis, and cellular levels. We observed that intra-patient, inter-tumoral molecular subtype heterogeneity was common in mPC and showed associations with genomic and clinical features. Additionally, cellular proliferation rates varied within a given patient across molecular subtypes and anatomic sites. Single-cell sequencing studies revealed features of morphologically and molecularly divergent tumor cell populations within a single metastatic site. These data provide a deeper insight into the complex patterns of tumoral heterogeneity in mPC with implications for clinical management and the future development of diagnostic and therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb925628a7789a4c45633ccff7c788aadf4bd5ff" target='_blank'>
              Patterns of intra- and inter-tumor phenotypic heterogeneity in lethal prostate cancer.
              </a>
            </td>
          <td>
            Martine P. Roudier, Roman Gulati, Erolcan Sayar, Radhika A Patel, Micah Tratt, Helen Richards, Paloma Cejas, Miguel Muñoz Gomez, Xintao Qiu, Yingtian Xie, B. Hanratty, Samir Zaidi, Jimmy L. Zhao, M. Adil, Chitvan Mittal, Yibai Zhao, R. Dumpit, Ilsa Coleman, J. Low, Thomas W Persse, Patricia Galipeau, John K. Lee, Maria S Tretiakova, Meagan Chambers, F. Vakar‐López, Lawrence D. True, Marie E Perrone, Hung-Ming Lam, Lori Kollath, Chien-Kuang C. Ding, Stephanie Harmon, Heather H. Cheng, Evan Y. Yu, R. Montgomery, Jessica E. Hawley, Daniel W. Lin, Eva Corey, M. Schweizer, Manu Setty, G. Ha, Charles Sawyers, C. Morrissey, Henry Long, Peter S Nelson, Michael C. Haffner
          </td>
          <td>2025-06-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Globally, gastric cancer (GC) stands as the fifth most prevalent form of malignant neoplasm and represents a significant contributor to mortality associated with oncological conditions. Despite advancements in therapeutic strategies for GC, the outcomes for patients with advanced stages of the disease continue to be unfavorable, largely due to tumor heterogeneity and the challenges posed by resistance to therapeutic agents. Metabolic reprogramming is pivotal in driving the advancement of GC, contributing to the development of resistance to pharmacological treatments and facilitating the cancer’s ability to evade immune surveillance. Developing multi-target comprehensive treatment strategies by integrating tumor microenvironment (TME) modulation holds promise for significantly improving therapeutic efficacy.The study analyzed GC and identified key cell subtypes by integrating data derived from single-cell RNA-sequencing (scRNA-seq) alongside spatial transcriptomics information. Cell type identification was accomplished using the tool of Seurat, and the spatial distribution of cell types was revealed through the Robust Cell Type Decomposition technique. CellChat was used to analyze the interactions between key cell subtypes and other cells, and the “StLearn” package was employed to investigate spatial cell communication in depth. Additionally, the functional role of the key molecule ELK4 was validated through in vitro experiments.This research utilized scRNA-seq combined with spatial transcriptomics to comprehensively analyze GC, identifying the C1 NDUFAB1+ subtype, which exhibited high proliferative activity, metabolic reprogramming capabilities, and immune evasion properties. It was found that the C1 NDUFAB1+ subtype closely interacted with fibroblasts and pericytes via the PARs signaling pathway. Additionally, in vitro experiments confirmed that knockdown of ELK4 substantially curbed tumor cell proliferation, migration, and invasion.This study revealed the main significance of the C1 NDUFAB1+ subtype in GC, elucidating its core mechanisms in tumor progression, metabolic reprogramming, and immune evasion. ELK4 was identified as a key regulatory factor that markedly enhanced the proliferation, migratory capacity, and invasive potential of tumor cells, while changes in the TME were a driving force behind immune suppression and drug resistance. The findings underscored the importance of developing specific therapeutic targets, targeting metabolic reprogramming, and overcoming immune evasion, providing new theoretical foundations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94360dc80f5310630bd93066b812e27d59cfa9fb" target='_blank'>
              Integrative single-cell and spatial transcriptomics uncover ELK4-mediated mechanisms in NDUFAB1+ tumor cells driving gastric cancer progression, metabolic reprogramming, and immune evasion
              </a>
            </td>
          <td>
            Yuwei Sun, Wenyang Nie, Zhikai Xiahou, Xiaojing Wang, Wenjia Liu, Zongkai Liu, Zhiheng Lin, Zhaidong Liu
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer cell spheroids autonomously form in the ascites fluid and are considered a conduit for epithelial ovarian cancer metastasis within the peritoneal cavity. Spheroids are homotypic, avascular 3D structures that acquire resistance to anoikis to remain viable after cellular detachment. We used in vitro spheroid model systems to interrogate pathways critical for spheroid cell proliferation, distinct from those driving monolayer cancer cell proliferation. Using the 105C and KOC-7c human ovarian clear cell carcinoma (OCCC) cell lines, which have distinct proliferative phenotypes as spheroids but the same prototypical OCCC gene mutation profile of constitutively activated AKT signaling with the loss of ARID1A, we revealed therapeutic targets that efficiently kill cells in spheroids. RNA-seq analyses compared the transcriptome of 3-day monolayer and spheroid cells from these lines and identified the characteristics of dormant spheroid cell survival, which included the G2/M checkpoint, autophagy, and other stress pathways induced in 105C spheroids, in sharp contrast to the proliferating spheroid cells of the KOC-7c cell line. Next, we assessed levels of various G2/M checkpoint regulators and found a consistent reduction in steady-state levels of checkpoint regulators in dormant spheroid cells, but not proliferative spheroids. Our studies showed that proliferative spheroid cells were sensitive to Wee1 inhibition by AZD1775, but the dormant spheroid cells showed a degree of resistance to AZD1775, both in terms of EC50 values and spheroid reattachment abilities. Thus, we identified biomarkers of dormant spheroids, including the G2/M checkpoint regulators Wee1, Cdc25c, and PLK1, and showed that, when compared to proliferating spheroid cells, the transcriptome of dormant OCCC spheroids is a source of therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877fd898a1fb77cad11de61e359dcf933c175ace" target='_blank'>
              Transcriptomic Analyses of Ovarian Clear Cell Carcinoma Spheroids Reveal Distinct Proliferative Phenotypes and Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Bartlomiej Kolendowski, Sylvia Cheng, Yudith Ramos Valdés, Trevor G. Shepherd, G. DiMattia
          </td>
          <td>2025-05-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Myofibroblastic cancer-associated fibroblasts (CAF) in tumor stroma serves as an independent poor prognostic indicator, supporting higher stemness in oral cancer; however, the underlying biology is not fully comprehended. Here, we have explored the crucial role of Tunica Interna Endothelial Cell Kinase (Tie2/TEK) signaling in transition and maintenance of myofibroblastic phenotype of CAFs, and as possible link with the poor prognosis of head and neck squamous cell carcinoma (HNSCC) patients. Methods Bulk and single cell RNA-sequencing (scRNAseq) methods and in-depth bioinformatic analysis were applied for CAF and cancer cells co-culture for studying molecular relationships. In vitro 3D-spheroid-forming ability, expression of stemness markers, in vivo tumor formation ability in zebrafish embryo and syngeneic mouse allografts formation was conducted to test stemness, upon targeting CAF-specific Tie2 activity by gene silencing or with small molecule inhibitor. Immunohistochemistry analysis was performed to locate the distribution of Tie2 and αSMA in primary tumors of oral carcinoma. Prognosis in HNSCC patient cohort from The Cancer Genome Atlas (TCGA) study was analysed based on single sample gene set enrichment score (ssGSEA) and Kaplan–Meier analysis. Results Autocrine or exogenous TGFβ-induction in CAF led to the recruitment of histone deacetylase 2 (HDAC2) on the promoter of Tie2-antagonist, Angiopoietin-2 (ANGPT2), resulting in its downregulation, leading to phosphorylation of Tie2 (Y992) and subsequent activation of SRC (Y418). This led to SRC/ROCK mediated αSMA-positive stress-fiber formation with gain of myofibroblast phenotype. The CAF-specific Tie2-signaling was responsible for producing embryonic-like cell state in co-cultured cancer cells; with enhanced tumor initiating ability. Tie2 activity in CAF exerted the dynamic gene expression reprogramming, with the upregulation of ‘cell migration’ and downregulation of ‘protein biosynthesis’ related gene-regulatory-network modules in malignant cells. The AUCell scores calculated for gene signatures derived from these modules showed significant concordance in independently reported scRNAseq studies of HNSCC tumors and significant association with poor prognosis in HNSCC patient cohort. Conclusions CAF-specific Tie2 activity may serve as direct stromal-target against cancer cell plasticity leading to poor prognosis of oral cancer patients. Overall, our work has provided wider applicability of Tie2-specific functions in tumor biology, along with its known role in endothelial cell-specific function. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/973b044801c6bab58b14790142170e897c97d9cf" target='_blank'>
              Tie2 activity in cancer associated myofibroblasts serves as novel target against reprogramming of cancer cells to embryonic-like cell state and associated poor prognosis in oral carcinoma patients
              </a>
            </td>
          <td>
            Paromita Mitra, Uday Saha, Kingsly Joshua Stephen, Priyanka Prasad, Subhashree Jena, Ankit Kumar Patel, H. Bv, Santosh Kumar Mondal, S. Kurkalang, Sumitava Roy, Arnab Ghosh, Shantanu Saha Roy, J. Das Sarma, N. Biswas, Moulinath Acharya, R. Sharan, P. Arun, M. Jolly, Arindam Maitra, Sandeep Singh
          </td>
          <td>2025-05-10</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 11508


 Background:
 Leiomyosarcoma (LMS) is a smooth muscle-derived tumor with significant heterogeneity and limited treatment options for recurrent/metastatic disease. A lack of targetable driver mutations and prognostic and predictive biomarkers have hampered the care of patients with LMS. There is a great need to better understand the biology of LMS and develop novel therapeutics. Advances in single cell RNA sequencing (scRNA-seq) have allowed for better understanding of intratumoral heterogeneity in diverse cancer subtypes. However, tissue dissociation during this process leads to loss of spatial context. Spatial gene expression analysis builds upon scRNA-seq and has the potential to yield information about tissue organization, cell-cell interactions, niches, and cell states. To date there have been limited application of spatial transcriptomics to sarcoma.
 Methods:
 We performed single nucleus multiome (snRNA-seq and snATAC-seq) on a cohort of 16 primary, untreated LMS samples including 12 soft tissue (STLMS) and 4 uterine (ULMS) tumors. We then designed a custom 480-gene panel using the differentially expressed genes from clusters identified in snRNA-seq data to be able to identify spatial relationships between these clusters and to assess these clusters on a larger scale. We utilized the 10x Genomics Xenium platform. This was applied to LMS tissue microarrays (TMAs) comprising a total of 326 tissue cores from 127 unique patients. Matched primary and metastatic samples from the same patient were available for 33 patients.
 Results:
 Analysis of scRNAseq data identified 2 distinct subtypes: a dedifferentiated subtype with mesenchymal features (MES) and a differentiated subtype with enrichment of smooth muscle cell markers (SMC). Integration of chromatin accessibility data from snATACseq showed enrichment of nuclear factor I (NFI) transcription factor (TF) motifs in the MES and AP-1 motifs in the SMC group. Whole genome sequencing did not reveal an obvious genomic etiology for these subtypes. Spatial transcriptomics was able to identify these 2 subtypes in a larger cohort of tumors. Consistent with snRNAseq data, we find that most tumors had almost exclusively either MES or SMC cells. We assessed spatial relationships between these subtypes and infiltrating immune cells. This revealed an enrichment in immunosuppressive macrophages and exhausted T cells in MES tumors compared to SMC tumors. Analysis of matched primary and metastatic tumors demonstrated that the subtype (MES or SMC) generally remains consistent between different sites of disease.
 Conclusions:
 We identify 2 novel LMS subtypes (MES and SMC) driven by distinct TFs. Spatial transcriptomic analysis confirmed the presence of these 2 subtypes in a larger cohort and demonstrated that MES tumors are associated with a more immunosuppressive tumor microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/170994843a3a0b5e571cb27c29ebdaf0b414317f" target='_blank'>
              Spatial transcriptomic profiling from over 300 leiomyosarcoma samples.
              </a>
            </td>
          <td>
            Ryan A Denu, Zhao Zheng, Veena Kochat, Yingda Jiang, William I. Padron, D. Ingram, Khalida M Wani, Larissa A Meyer, P. Soliman, R. Ratan, Alexander J. Lazar, E. Keung, Kunal Rai, E. N. Nassif Haddad
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2702678fe4bf9d99269192e19243ce5bd9b25ff1" target='_blank'>
              Patient-Derived Three-Dimensional Lung Tumor Models to Evaluate Response to Immunotherapy
              </a>
            </td>
          <td>
            Kayla F. Goliwas, Kenneth P. Hough, Sruti Sivan, Sierra L. Single, Sameer S. Deshmukh, Joel L. Berry, Maya Khalil, Benjamin Wei, Yanis Boumber, Mohammad Athar, Aakash Desai, S. Ponnazhagan, James M. Donahue, J. Deshane
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="This study explores the inflammatory response observed in pancreata and pancreatic lymph node (pLN) samples obtained throughout the natural history of type 1 diabetes (T1D) including non-diabetic individuals and non-diabetic autoantibody positive individuals with high susceptibility using spatial transcriptomics (ST). Integration of ST with public single-cell RNA sequencing data enabled interrogation of transcriptional alterations in T1D pathogenesis across both tissues and cellular scales. In the T1D pancreas, we observed global upregulation of multiple inflammation-associated transcripts, including regenerating islet-derived (REG) family genes, complement factor 3 (C3), SOD2, and OLFM4, and highlighted cellular candidates potentially contributing to these signatures. Within the T1D pLN, we observed spatially restricted upregulation of lymphotoxin-β (LTB) alongside follicular dendritic cell (FDC)-associated transcripts including FDCSP, CLU, and FCER2. Collectively, these findings highlight distinct inflammation signatures in the pancreas and regional pLN which can help inform the development of future therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d86fa210fd0a0e42f81c1f4633973c64cff969b8" target='_blank'>
              Spatial transcriptomics from pancreas and local draining lymph node tissue reveals a lymphotoxin-β signature in human type 1 diabetes
              </a>
            </td>
          <td>
            Miguel A. Medina-Serpas, Maigan A. Brusko, Gregory J. Golden, Martha Campbell-Thompson, Trevor Rogers, A. Posgai, Eline T. Luning Prak, Chengyang Liu, Klaus H. Kaestner, Ali Naji, Michael R. Betts, Lauren M McIntyre, M. Atkinson, Todd M. Brusko
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Simple Summary Ductal carcinoma in situ is a prevalent non-invasive form of breast cancer that, while confined to the milk ducts, carries the potential to progress to invasive disease. With increased detection due to advanced imaging and widespread screening, there is growing concern about the ability to accurately predict which cases are likely to become aggressive. This review aims to deepen the understanding of the biological mechanisms that influence DCIS progression, focusing on the roles of the tumor microenvironment, immune response, and molecular alterations. By synthesizing the contribution of myoepithelial cells, tumor-infiltrating lymphocytes, and key molecular markers, the review seeks to enhance risk stratification methods. These insights could support the development of more precise, personalized treatment strategies that effectively prevent cancer progression while minimizing unnecessary interventions in patients with low-risk DCIS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76422452bac8ca5523021cf11ee79191c7cd5786" target='_blank'>
              DCIS Progression and the Tumor Microenvironment: Molecular Insights and Prognostic Challenges
              </a>
            </td>
          <td>
            Karolina Prajzendanc
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Neuroblastoma, the most common pediatric extracranial solid tumor, has heterogeneous clinical outcomes ranging from malignant progression to spontaneous regression. With the highest frequency of the elusive spontaneous regression, low-risk INSS Stage 4S neuroblastoma represents an ideal model for mechanistic investigation. Spontaneous regression is often accompanied by tumor differentiation, but the mechanisms underlying this process remain largely unclear. Methods Single-nucleus transcriptomics (snRNA-seq) data of neuroblastoma samples were obtained from the Synapse repository to investigate the composition of heterogeneous tumor cell clusters. The feature of the Stage 4S-specific tumor cell subpopulation was revealed through differential expression analysis, pathway enrichment analysis and pseudotime analysis, followed by clinical significance validation on public cohort datasets. The biological function of secreted SLIT3 was validated using multiple in vitro models, including recombinant protein treatment, conditioned medium treatment, and cell lines coculture, to confirm the intratumoral crosstalk effect. Orthotopic and subcutaneous xenograft models were established to verify SLIT3's in vivo function. Cellular bulk RNA-seq analysis was performed with or without SLIT3 recombinant protein treatment to discover the downstream pathways activated by SLIT3, followed by validation with specific pathway inhibitors. Results Analysis of snRNA-seq revealed a distinct subpopulation of tumor cells within INSS Stage 4S neuroblastoma, characterized by a spontaneous regression-like program progressing toward differentiation. Activated SLIT-ROBO signaling was found in the Stage 4S-specific tumor cell subpopulation, which strongly correlated with favorable prognosis. Further investigation into the secreted ligands in SLIT-ROBO related pathways revealed that SLIT3 displayed the most potent enrichment in Stage 4S tumors and the strongest differentiation-inducing effect. In vitro experiments using recombinant SLIT3 protein, conditioned medium, and cell lines coculture consistently demonstrated the capacity of SLIT3 to induce neuroblastoma cell differentiation via intratumoral crosstalk, as evidenced by increased neurite outgrowth and elevated expression of neuronal differentiation markers. Both orthotopic xenograft and subcutaneous xenograft models demonstrated that SLIT3 expression suppressed tumor growth, leading to in vivo tumor differentiation. Mechanistically, PLCβ/PKC signaling mediates the SLIT3-induced neuroblastoma cell differentiation. Conclusions Stage 4S-specific tumor cell subpopulation exhibits a spontaneous regression-like program, from which SLIT3 mediates intratumoral crosstalk and promotes neuroblastoma differentiation via PLCβ/PKC signaling. These findings provide new insights into the mechanism of spontaneous regression in neuroblastoma and offer novel therapeutic targets for differentiation-based treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06621-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc7b67c542baf51616516c8722d1f59d34fe27f7" target='_blank'>
              SLIT3-mediated intratumoral crosstalk induces neuroblastoma differentiation via a spontaneous regression-like program
              </a>
            </td>
          <td>
            Meiling Liu, Dekang Lv, Wenjing Yan, Yi Wu, Shulan Wang, Luoxuan Wang, Jie Lei, Deshun Zeng, Zifeng Wang, Fang Liu, Bing Deng, Quentin Liu, Bin He, M. Yan
          </td>
          <td>2025-05-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The tumor microenvironment is heterogeneous, structurally complex, and continually evolving, making it difficult to fully capture. Common dissociative techniques thoroughly characterize the heterogeneity of cellular populations but lack structural context. The recent boom in spatial analyses has exponentially accelerated our understanding of the structural complexity of these cellular populations. However, to understand the dynamics of cancer pathogenesis, we must assess this heterogeneity across space and time. In this review, we provide an overview of current dissociative, spatial, and temporal analysis strategies in addition to existing and prospective spatiotemporal techniques to illustrate how understanding the tumor microenvironment, focusing on dynamic immune-cancer cell interactions, across four dimensions will advance cancer research and its diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1f3339cce25c83c98dd45cea90c67e2a48bc606" target='_blank'>
              The tumor microenvironment across four dimensions: assessing space and time in cancer biology
              </a>
            </td>
          <td>
            Christina R. Larson, Ayushi Mandloi, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-06-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Angiosarcoma, a rare and highly aggressive malignancy originating from vascular endothelial cells, is characterized by its rapid progression, high invasiveness, and poor prognosis. Due to the limited understanding of its tumor microenvironment (TME) and the absence of effective treatments, further research is essential to elucidate its pathogenic mechanisms and improve therapeutic strategies. Objective This study aims to characterize the cellular heterogeneity and unique TME of primary breast angiosarcoma using single-cell RNA sequencing (scRNA-seq), to identify potential therapeutic targets and improve clinical outcomes. Methods Tumor samples were obtained from a patient with bilateral primary breast angiosarcoma and two patients with invasive breast cancer. Single-cell RNA sequencing (scRNA-seq) was conducted to capture the transcriptomic profiles of individual cells within the tumor samples. Following stringent quality control, a total of 31,771 cells were analyzed using comprehensive bioinformatics approaches. Cell populations were identified and classified into distinct cell types, and differential gene expression analysis was performed to explore key signaling pathways. Functional enrichment analysis was used to identify pathways related to tumor progression and immune evasion. Additionally, cell-cell communication networks were mapped to understand interactions within the TME, with a focus on pathways that may serve as therapeutic targets. Results The scRNA-seq analysis revealed significant differences in the distribution of perivascular cells, fibroblasts, T cells, endothelial cells, and myeloid cells in breast angiosarcoma compared to invasive breast cancer. Key pathways enriched in angiosarcoma samples included growth factor binding, platelet-derived growth factor binding, and ribosome biogenesis, with abnormal expression of several ribosomal proteins. Notably, genes such as FAT4, KDR, FN1, and KIT were highly expressed in angiosarcoma endothelial cells, correlating with poor prognosis. Cell communication analysis highlighted the CXCL12-CXCR4 axis as a crucial mediator of the TME in angiosarcoma. Conclusion This study provides critical insights into the TME of primary breast angiosarcoma, highlighting potential molecular targets and pathways for therapeutic intervention. These findings may inform the development of more effective treatment strategies for this rare and challenging tumor type. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02022-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61afd778df17e939f7a978e09a799d8e740668a2" target='_blank'>
              Dissecting the tumor microenvironment in primary breast angiosarcoma: insights from single-cell RNA sequencing
              </a>
            </td>
          <td>
            Peikai Ding, Shengbin Pei, Yi Zhai, Zheng Qu, Yazhe Yang, Xiaolong Feng, Qiang Liu, Xiangyu Wang, Wenxiang Zhang, Zhongzhao Wang, Xiangyi Kong, Jing Wang, Yi Fang
          </td>
          <td>2025-06-05</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61913974e99e15ccb9e61a055d93cfcb623fd1c0" target='_blank'>
              Matrix structure and microenvironment dynamics correlate with chemotherapy response in ovarian cancer
              </a>
            </td>
          <td>
            Florian Laforêts, Panoraia Kotantaki, Samar Elorbany, Joseph J Hartlebury, Joash D Joy, Beatrice Malacrida, Rachel C Bryan-Ravenscroft, Chiara Berlato, Erica Di Federico, John F Marshall, R. Manchanda, W. Jarolimek, Lara Perryman, E. Maniati, Frances R Balkwill
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Medullary thyroid carcinoma (MTC) is a rare and aggressive thyroid cancer with a challenging prognosis. While the immune microenvironment plays a crucial role in cancer progression, its role in MTC remains underexplored compared to more common thyroid cancers. Methods this study investigates the immune landscape of MTC by systematically evaluating immune cell infiltration and expression of immune markers across various tissue topographies. We utilized advanced immunohistochemical techniques to analyze tissue samples from 24 MTC patients, focusing on the tumor core, interface with healthy tissue, adjacent normal thyroid tissue, and lymph node metastases. Results our findings reveal a distinct immune profile with increased CD3+, CD4+, CD8+ and CD20+ lymphocytes in normal tissues adjacent to tumors and a notable presence of granzyme B+ cells in the tumor interface, particularly in patients with structural disease. Additionally, we observed a significant enrichment of mast cells in metastatic tissues. Discussion these results highlight the complex and spatially dependent immune landscape of MTC, suggesting implications for targeted immunotherapy. This study provides novel insights into the immune microenvironment of MTC and emphasizes the need for further research to elucidate its impact on disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ab2d28852a497483bb6cba642f6c18c52966ea" target='_blank'>
              Spatial immune profiling reveals distinct microenvironments in medullary thyroid carcinoma
              </a>
            </td>
          <td>
            Maria Eduarda de Castro, Gustavo Forlin de Siqueira, Jean Ferrante Mariano, M. Kizys, L. Ceolin, Fernando Augusto Soares, Rodrigo Natal, Humberto Carneiro, Rodrigo Nalio Ramos, Laura Sterian Ward, Niels Olsen Saraiva Camara, Cleber Pinto Camacho, Flavia de Oliveira Facuri Valente, Susan C. Lindsey, Diego Dias dos Santos, Cristiane Damas Gil, João Roberto Maciel Martins, Rui Monteiro de Barros Maciel, L. L. Cunha
          </td>
          <td>2025-05-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2fc80562262fc23fc64cea425e831c1c998be8" target='_blank'>
              Functional Spatial Mapping of the Tumour Immune Microenvironment In Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Sam Legg, Emma Wagner, Christopher J. Applebee, Amanda R. Kirane, Julian Padget, Banafshé Larijani
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Pancreatic cancer (PC) is a highly aggressive pancreatic malignant tumor with poor prognosis due to its complex tumor microenvironment (TME) and metastatic potential. Fibroblasts play an important role in tumor progression and metastasis by remodeling the extracellular matrix (ECM) and influencing the immune response. This study explored migrasome-associated fibroblasts, especially TSPAN4 and ITGA5, as key regulators of pancreatic cancer metastasis, aiming to provide new ideas for therapeutic strategies targeting TME. Methods We employed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to analyze pancreatic cancer tissues. Data from the GEO and TCGA databases were integrated and processed using batch correction techniques. Single-cell data were analyzed with Seurat and Monocle for dimensionality reduction and pseudotime trajectory analysis. Cell communication was assessed using CellCall and CellChat. Spatial transcriptomic analysis was conducted with RCTD and MISTy tools to investigate cell interactions within the TME. Additionally, gene enrichment, deconvolution, and prognostic analyses were performed, alongside experimental validation through siRNA transfection, qRT-PCR, and various functional assays to investigate the role of TSPAN4 in metastasis. Results Our results underscore the critical role of TSPAN4+ fibroblasts in pancreatic cancer. These fibroblasts were predominantly located at the tumor periphery and exhibited elevated migrasome gene expression, which was associated with enhanced ECM remodeling and immune suppression. Intercellular communication analysis revealed that TSPAN4+ fibroblasts engaged in extensive interactions with immune cells, such as macrophages and endothelial cells, facilitating metastasis and immune evasion. Moreover, the high expression of immune checkpoint markers indicated their involvement in modulating the immune response. Conclusion TSPAN4+ fibroblasts are key regulators of pancreatic cancer progression, contributing to metastasis, immune suppression, and ECM remodeling. Targeting these fibroblasts represents a promising therapeutic strategy to improve clinical outcomes and enhance the effectiveness of immunotherapies in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3280d6d2c8de28d9483ff92142917bd9b0c11e4" target='_blank'>
              TSPAN4+ fibroblasts coordinate metastatic niche assembly through migrasome-driven metabolic reprogramming and stromal-immune crosstalk in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Qingwen Hu, Jiali Chen, Yang Liu, Haiqing Chen, Haotian Lai, Lai Jiang, Xuancheng Zhou, Shengke Zhang, Jinbang Huang, Hao Chi, Bo Li, Xiaolin Zhong
          </td>
          <td>2025-05-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cervical cancer represents a significant global health challenge, with complex cellular and molecular mechanisms driving its progression from HPV infection to invasive malignancy. This study employed an integrated approach combining single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (stRNA-seq) to comprehensively characterize the tumor microenvironment (TME) across different stages of cervical cancer development. Through analysis of samples from normal cervix, HPV-infected normal cervix, high-grade squamous intraepithelial lesions (HSIL), and invasive cervical cancer, we identified distinct cellular populations and their dynamic changes during disease progression. Our findings revealed significant heterogeneity in immune cell populations, particularly highlighting the role of SPP1+ macrophages that were substantially enriched in cervical cancer compared to precancerous and normal tissues. Cell-cell communication networks and spatial mapping demonstrated that SPP1+ macrophages interact extensively with immune cells through the SPP1-CD44 signaling axis. This interaction contributes to an immunosuppressive microenvironment through modulation of T cell function and promotion of tumor cell survival. Furthermore, high expression of SPP1 correlated with advanced tumor stages and poor overall survival in cervical cancer patients, highlighting its potential as a prognostic biomarker. Our comprehensive characterization of the cellular landscape and intercellular communication networks in cervical cancer progression provides valuable insights for the development of targeted therapeutic strategies aimed at modulating the TME, particularly through disruption of the SPP1-CD44 axis. These findings establish a foundation for more effective personalized approaches to improve clinical outcomes in cervical cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87cbb4dd12be6f1664aacd1df3e0b1efa6fe30d7" target='_blank'>
              Deciphering the cellular and molecular landscape of cervical cancer progression through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Peng Xia, Juanhong Zhou, Rong Shen, Degui Wang
          </td>
          <td>2025-05-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Microwave thermochemotherapy (MTC) has been applied to treat lip squamous cell carcinoma (LSCC), but a deeper understanding of its therapeutic mechanisms and molecular biology is needed. To address this, we used single-cell transcriptomics (scRNA-seq) and spatial transcriptomics (ST) to highlight the pivotal role of tumor-associated neutrophils (TANs) among tumor-infiltrating immune cells and their therapeutic response to MTC. MNDA+ TANs with anti-tumor activity (N1-phenotype) are found to be abundantly infiltrated by MTC with benefit of increased blood perfusion, and these TANs are characterized by enhanced cytotoxicity, ameliorated hypoxia, and upregulated IL1B, activating T&NK cells and fibroblasts via IL1B-IL1R. In this highly anti-tumor immunogenic and hypoxia-reversed microenvironment under MTC, fibroblasts accumulated in the tumor front (TF) can recruit N1-TANs via CXCL2-CXCR2 and clear N2-TANs (pro-tumor phenotype) via CXCL12-CXCR4, which results in the aggregation of N1-TANs and extracellular matrix (ECM) deposition. In addition, we construct an N1-TANs marker, MX2, which positively correlates with better prognosis in LSCC patients, and employ deep learning techniques to predict expression of MX2 from hematoxylin-eosin (H&E)-stained images so as to conveniently guide decision making in clinical practice. Collectively, our findings demonstrate that the N1-TANs/fibroblasts defense wall formed in response to MTC effectively combat LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2560362b63b6e3eb203f753fb2eef7cc7ed8926f" target='_blank'>
              Single-cell and spatial transcriptomics reveals an anti-tumor neutrophil subgroup in microwave thermochemotherapy-treated lip cancer
              </a>
            </td>
          <td>
            Bingjun Chen, Huayang Fan, Xin Pang, Zeliang Shen, Rui Gao, Haofan Wang, Zhenwei Yu, Tianjiao Li, Mao Li, Yaling Tang, Xinhua Liang
          </td>
          <td>2025-05-13</td>
          <td>International Journal of Oral Science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Pediatric low-grade gliomas (pLGG) comprise 35% of all brain tumors. Despite favorable survival, patients experience significant morbidity from disease and treatments. A deeper understanding of pLGG biology is essential to identify novel, more effective, and less toxic therapies. We utilized single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and cytokine analyses to characterize and understand tumor and immune cell heterogeneity of pilocytic astrocytoma (PA) and ganglioglioma (GG). scRNA-seq revealed tumor and immune cells within the tumor microenvironment (TME). Tumor cell subsets include both progenitor and mature cell populations. Immune cells included myeloid and lymphocytic cells. There was a significant difference between the prevalence of two major myeloid subclusters between PA and GG. Bulk and single-cell cytokine analyses evaluated the immune cell signaling cascade with distinct immune phenotypes among tumor samples. KIAA1549-BRAF tumors appeared more immunogenic, secreting higher levels of immune cell activators and chemokines, compared to BRAF V600E tumors. Spatial transcriptomics revealed the differential gene expression of these chemokines and their location within the TME. A multi-pronged analysis demonstrated the complexity of the PA and GG TME and differences between genetic drivers that may influence their response to immunotherapy. Further investigation of immune cell infiltration and tumor-immune interactions is warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dc01e4fe40878b7885262fd79ea7216b56b949" target='_blank'>
              Multi-pronged analysis of pediatric low-grade glioma and ganglioglioma reveals a unique tumor microenvironment associated with BRAF alterations.
              </a>
            </td>
          <td>
            S. Zahedi, Kent Riemondy, Tian Liu, A. Griesinger, A. Donson, April A Apfelbaum, Rui Fu, Julian Grandvallet Contreras, M. Crespo, J. DeSisto, Madeline Groat, Emil Bratbak, Adam L Green, Todd C Hankinson, Michael Handler, Rajeev Vibhakar, Nicholas Willard, Nicholas K Foreman, Tzu Phang, Jean M. Mulcahy Levy
          </td>
          <td>2025-06-30</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Traditional gene expression deconvolution methods assess a limited number of cell types, therefore do not capture the full complexity of the tumor microenvironment (TME). Here, we integrate nine deconvolution tools to assess 79 TME cell types in 10,592 tumors across 33 different cancer types, creating the most comprehensive analysis of the TME. In total, we found 41 patterns of immune infiltration and stroma profiles, identifying heterogeneous yet unique TME portraits for each cancer and several new findings. Our findings indicate that leukocytes play a major role in distinguishing various tumor types, and that a shared immune-rich TME cluster predicts better survival in bladder cancer for luminal and basal squamous subtypes, as well as in melanoma for RAS-hotspot subtypes. Our detailed deconvolution and mutational correlation analyses uncover 35 therapeutic target and candidate response biomarkers hypotheses (including CASP8 and RAS pathway genes).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b1e17d9e760b6c7495c8a6aea0db799ec80899" target='_blank'>
              Pan-cancer immune and stromal deconvolution predicts clinical outcomes and mutation profiles
              </a>
            </td>
          <td>
            B. Bhinder, Verena Friedl, Sunantha Sethuraman, Davide Risso, Kami E. Chiotti, R. J. Mashl, Kyle Ellrott, Jordan A. Lee, , K. Gyan, Aditya Deshpande, Marcin Imielinski, R. Bareja, Josh Stuart, Myron Peto, K. Hoadley, Alexander J. Lazar, A. Cherniack, Jingchun Zhu, Shaolong Cao, Mark Rubin, Wenyi Wang, Oliver F. Bathe, N. Robine, Li Ding, Peter W. Laird, Wanding Zhou, Hui Shen, V. Thorsson, Jen Jen Yeh, Matthew H. Bailey, Daniel Cui Zhou, , M. Goldman, Yongsheng Li, Anil Korkut, Nidhi Sahni, D. N. Hayes, Michael K. A. Mensah, Ina Felau, Anab Kemal, Samantha J. Caesar-Johnson, John A. Demchok, Liming Yang, M. Ferguson, R. Tarnuzzer, Zhining Wang, J. Zenklusen, Adam Alana Andrea Andrew Angela Benjamin Benjamin Brian Margolin Weinstein Sboner Blair Brooks Berman Raph, Adam Margolin, Alana Weinstein, A. Sboner, Andrew Blair, Angela Brooks, Benjamin Berman, Benjamin Raphael, Brian Craft, Dante S. Bortone, David Heimain, David L Gibbs, David Haan, D. Betel, Duncan McColl, Emek Demir, Faeze Brahman, F. Farshidfar, Gaddy Getz, Galen Gao, Gordon Robertson, Huy Dinh, Hyo Young Choi, I. Shmulevich, Ioannis Anastopoulous, Jasmine Yang, Jeff Damrauer, Ken Chen, Lauren Sanders, Lee Cooper, , Matt Reyna, Mike Noble, Mohammed El-Kebir, Molly Zhang, Nicole Yeager, Paul Little, Richard Moffitt, S. Meier, Teresa Swatloski, T. Knijnenburg, Thomas Matthew, Vicky Chen, Vlado Uzunangelov, Xinghua Lu, Yige Wu, Yulia Newton, Zeya Wang, Paul T. Spellman, O. Elemento
          </td>
          <td>2025-07-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5484870331a7d0dee6989fa1ddf5ea8e0b01c6cb" target='_blank'>
              Functional genomics and tumor microenvironment analysis reveal prognostic biological subtypes in Mantle cell lymphoma
              </a>
            </td>
          <td>
            Sunandini Sharma, Roshia Ali, Alyssa Bouska, Dylan Jochum, Meghana Kesireddy, Simeon Mahov, Joseph Lownik, Weiwei Zhang, Waseem G Lone, Mahfuza Afroz Soma, Alicia Gamboa, Vaishnavi Devarkonda, Dalia Elgamal, Atqiya Fariha, Adnan Mansoor, Douglas Stewart, Peter Martin, Brian K. Link, R. Advani, Paul M. Barr, A. Goy, A. Mehta, M. Kamdar, D. Stephens, Veronika Bachanova, L. Smith, Ryan D Morin, P. Pararajalingam, M. Lunning, Kai Fu, Dennis Wiesenberger, Wing C. Chan, J. Khoury, T. Greiner, J. Vose, Akil Merchant, Chengfeng Bi, J. Iqbal
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="
 Cancer treatment options including small molecules, antibody-drug conjugates (ADCs), CAR T-cells, and combination therapies are rapidly expanding to improve cancer patient survival. However, efficacy of targeted therapies is limited by interpatient and intratumoral heterogeneity in breast cancer, and may be further exacerbated by dysregulated tumor microenvironments. For example, studies suggested that changes in tumor and breast tissue microenvironment in obesity (e.g. inflammation, increased growth factors) could lead to worsened prognosis that could be related to therapy resistance. Systematic characterization of the cellular effects of emerging drug candidates, particularly across different tumor microenvironments, is challenging, as it needs to be surveyed in diverse cancer contexts beyond just a handful of cancer cell lines. To overcome this challenge, we created PRISM (Profiling Relative Inhibition Simultaneously in Mixtures), an effective drug discovery screening platform with over 900 genetically distinct cancer cell lines of diverse lineages, to rapidly test therapeutics in a high-throughput manner. These cell lines, including 33 different breast cancer cell lines covering diverse cell types and subgroups, carry a specific 24 nucleotide “barcode” tag and have rich baseline genomic and functional characterization. They are first pooled into 20-25 cell pools of different lineages with similar growth rates, then subsequently cultured together with or without drugs. The relative barcode quantity read out by NextGen sequencing determines cell line responses to drugs. This pooled approach significantly reduces the time and the resources required for drug screening. Importantly, utilizing the rich cell line characterization to interpret viability profiles enables the identification of drivers of differential sensitivity and potential biomarkers of compound response. Furthermore, we created a breast cancer cell specific pool to investigate how media conditions mirroring different tumor microenvironments [e.g. media filtration, addition of insulin (0.01 mg/mL) or EGF (0.01 ug/mL)] can impact drug effects on breast cancer cells. In the PRISM 900+ cell panel treated with ADCs conjugated to different payloads that target HER2, there was a clear association between cell line sensitivity and ERBB2 (encoding HER2) gene expression, with the strongest effects on viability observed in a subset of cell lines overexpressing ERBB2. Upon further comparison of biomarker profiles across different ADCs, we established relationships between payload resistance and ABCB1 efflux pumps emerged for a subset of ADC payloads. Using the breast cancer cell specific pool, our data showed that estrogen receptor (ER) degraders fulvestrant and vepdegestrant (ARV-471) had significantly greater effects in reducing viability of ER+ versus ER- breast cancer cell lines. Media filtering reduced the drug effects of fulvestrant but not ARV-471, suggesting that ARV-471 efficacy may be less sensitive to the tumor microenvironment than fulvestrant. Lapatinib (HER2/neu and EGF receptor inhibitor) was more effective in killing HER2+ than HER2- negative breast cancer cells, with cells cultured in EGF-spiked media being more sensitive to lapatinib, whereas media filtration diminished its effects. We did not observe breast cancer subgroup specific killing by alpelisib (PI3K inhibitor) or olaparib (PARP inhibitor), but breast cancer cells became less sensitive to alpelisib when cultured in either insulin- or EGF-spiked media, and less sensitive to olaparib in EGF-spiked media. Overall, our study highlights how the PRISM 900+ cell panel can provide insights into drug specificity, cancer subtype selectivity, and to uncover clinically relevant targets. The breast cancer specific pool can further increase the throughput of understanding how drug responses may be altered in different tumor microenvironments in breast cancer.
 Citation Format: Blanche C. Ip, Jillian N. Eskra, Tenzin Sangpo, Ashish Bino George, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5ec03b1ee18de91282ff62b13d5cb24a1236003" target='_blank'>
              Abstract P4-10-26: PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells
              </a>
            </td>
          <td>
            Blanche C. Ip, Jillian N. Eskra, Tenzing Sangpo, Ashish Bino George, M. Ronan, M. Rees, Jennifer A. Roth
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 2520


 Background:
 Despite the transformational impact of immune checkpoint blockade, many cancer patients do not experience long-term survival. T cells with innate immune signatures can secrete inflammatory cytokines/chemokines and deliver potent cytotoxic signals potentially ideal for tumor immunity. The novel double-stranded RNA sensor RIG-I agonist SLR14 improved the control of murine melanoma. We tested the hypothesis that SLR14 transforms T cells to a cytotoxic state in immunologically “cold” human tumor specimens.
 Methods:
 We developed an approach, called PERCEPT, to directly test the response of patient tumor and immune samples to novel and established therapies
 ex vivo
 using perturbational single-cell RNA sequencing (Table). We obtained 9 surgical resections from primary or metastatic melanoma and Merkel Cell Carcinoma (MCC) tumors and lymph node metastases and made suspension replicates of tumor and infiltrating immune cell co-cultures. We stimulated for 42-48 hours (Table). We Flourescently Activated Cell Sorted live cells and then barcoded for multiplexed single-cell sequencing using 10x scRNAseq. We used CINEMA-OT to identify factors associated with response and resistance to the perturbations tested. We developed and validated CRISPR-KO MCC and small cell lung cancer (SCLC) cell lines, and co-cultured with CD14+ monocytes or monocyte-derived DCs.
 Results:
 Stimulation with RIG-I agonist SLR14 induced expression beyond canonical IFN-stimulated genes in tumor cells, NK cells, and T cells. SLR14 stimulates tumor-infiltrating T cells into antiviral states in tumor-immune co-cultures and primes
 in vitro
 T-cell production of IFNγ. However, MCC immune infiltrate responsiveness to IFN or SLR14 was notably decreased compared to the melanoma samples, and perturbational computational analyses with CINEMA-OT identified the cytokine midkine (MDK) associated with nonresponse in MCC. Knockout of MDK restored response to IFN and SLR14 by MCC and SCLC tumor cell lines, as well as co-cultured CD14+ monocytes or monocyte-derived DCs.
 Conclusions:
 Our approach revealed that midkine, a multifunctional cytokine, suppresses innate immune sensing of IFN and SLR14 in both tumor and immune cells, disrupting the tumor immunity cycle at multiple points. We show that this effect, while comparatively infrequent in melanoma, is pronounced in MCC and SCLC. Our study thus uses a direct assessment of patient tumor and immune samples to identify a novel resistance mechanism enriched in neuroendocrine tumors MCC and SCLC.




 Therapeutic class
 Stimulation
 Target
 Clinical development




 Immune checkpoint inhibitor
 αPD-1
 PD-1
 Standard of care


 Cytokine
 IFNγIFNβ
 IFNGRIFNAR
 Early-phase clinical trials orPre-clinical


 Innate immune agonist
 Poly(I:C)-NTPoly(I:C)-TADU-S100SLR14
 TLR3MDA5/TIG-I/TLR3STINGRIG-I


 Combination
 αPD-1 + IFNβ
 PD-1/IFNAR




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/404b41f4f104c4cf7c170db6748ecf8ec9a5f2f1" target='_blank'>
              Perturbational single-cell RNA sequencing of patient tumors in Merkel cell and small cell lung carcinomas.
              </a>
            </td>
          <td>
            Curtis J. Perry, Alexander Frey, Yuewei Fei, P. Costa, Jason Z Wang, Sina Ghadermarzi, Daniel Levine, Asuka Koda, Amin Nassar, Min Ding, Yunan Nie, Therese Cordero-Dumit, Arnaud Augert, David van Djik, K. Olino, J. Ishizuka
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Emerging spatial profiling technologies have revolutionized our understanding of how tissue architecture shapes disease progression, yet the contribution of cellular diversity remains underexplored. Here, Ding and colleagues introduce multiomics and ecological spatial analysis (MESA), an ecology-inspired framework that integrates spatial and single-cell expression data to quantify tissue diversity across multiple scales. MESA both identifies distinct cellular neighborhoods and computes a variety of diversity metrics alongside the identification of diversity "hotspots". Applied to human tonsil tissue, MESA revealed previously undetected germinal center organization, while in spleen tissue of a murine lupus model, MESA highlights increasing cellular diversity with disease progression. Importantly, diversity hotspots do not correspond to conventional compartments identified by existing methods, presenting an orthogonal metric of spatial organization. In colorectal cancer, MESA's diversity metrics outperformed established subtypes at predicting patient survival, while in hepatocellular carcinoma, multi-omic integration identified significantly more ligand-receptor interactions between immune cells compared to single-modality analysis. This work establishes cellular diversity within tissues as a critical correlate of disease progression and underscores the value of multi-omic integration in spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eb774cd8ef924c70b07f117ccf0af3bd80ea8f6" target='_blank'>
              Cells keep diverse company in diseased tissues.
              </a>
            </td>
          <td>
            Kieran R. Campbell, A. Goeva
          </td>
          <td>2025-05-16</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378e143896f70bf1c18f50eea249337f3e7bfc52" target='_blank'>
              Single-Cell RNA Sequencing and Inferred Protein Activity Analysis Reveal a Distinct Tumor Phenotype in Early-Onset Colorectal Cancer Patients
              </a>
            </td>
          <td>
            David A. Cohen, Aleksandar Z Obradovic, Luka Jovanovic, Yosuke Mitani, Tianxia Li, Soobeom Lee, Dechokyab De, Casidhe-Nicole R Bethancourt, Alice Shin, Karen Dunbar, Francesco Cambuli, Christopher Lengner, Ning Li, Caitlin Rogers, Jennifer Ho, Viktoria Jakubikova, Emma Blystone, Yoanna S. Pumpalova, Beatrice Dionigi, Sanja Vickovic, Chia-Wei Cheng, Joel Gabre
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Understanding the heterogeneity of Rheumatoid Arthritis (RA) and identifying therapeutic targets remain challenging using traditional bulk transcriptomics alone, as it lacks the spatial and protein-level resolution needed to fully capture disease and tissue complexities. In this study, we applied Laser Capture Microdissection (LCM) coupled with mass spectrometry-based proteomics to analyze histopathological niches of the RA synovium, enabling the identification of protein expression profiles of the diseased synovial lining and sublining microenvironments compared to their healthy counterparts. In this respect, key pathogenetic RA proteins like membrane proteins (TYROBP, AOC3, SLC16A3, TCIRG1, and NCEH1), and extracellular matrix (ECM) proteins (PLOD2, OGN, and LUM) showed different expression patterns in diseased synovium compartments. To enhance our understanding of cellular dynamics within the dissected regions, we further integrated the proteomic dataset with single-cell RNA sequencing (scRNA-seq), and deduced cell type enrichment, including T cells, fibroblasts, NK cells, myeloid cells, B cells, and synovial endothelial cells. By combining high-resolution spatial proteomics and transcriptomic analyses, we provide novel insights into the molecular mechanisms driving RA, and highlight potential protein targets for therapeutic intervention. This integrative approach offers a more comprehensive view of RA synovial pathology, and mitigates the limitations of traditional bulk transcriptomics in target discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92de30759136153ce270d6dfee3613164882bb0" target='_blank'>
              Integrative Spatial Proteomics and Single-Cell RNA Sequencing Unveil Molecular Complexity in Rheumatoid Arthritis for Novel Therapeutic Targeting
              </a>
            </td>
          <td>
            Xue Wang, Fei Wang, Archana S. Iyer, Heather Knight, Lori J. Duggan, Yingli Yang, Liang Jin, Baoliang Cui, Yupeng He, Jan Schejbal, Lucy A. Phillips, Bohdan P. Harvey, Sílvia Sisó, Yu Tian
          </td>
          <td>2025-05-22</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca80c0d5c8118b6afa306ae5816ee0be1c3263a" target='_blank'>
              Single-cell transcriptomics reveals hypoxia-driven iCAF_PLAU is associated with stemness and immunosuppression in anorectal malignant melanoma.
              </a>
            </td>
          <td>
            Hao Zhang, Lin Gan, Xiaofei Duan, Baojing Tuo, Hao Zhang, Senbo Liu, Yugui Lian, Enjie Liu, Zhenqiang Sun
          </td>
          <td>2025-07-04</td>
          <td>Journal of gastroenterology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the most abundant non-cancerous cell type in glioblastoma (GBM) and heavily influence GBM biology, contributing to tumor progression, therapeutic resistance, immune evasion, and neovascularization. Current in vitro models that utilize IL-4/IL-13 stimulation fail to capture the transcriptional and functional heterogeneity of TAMs observed in vivo. In this study, we utilize a serum-free indirect co-culture model with patient-derived xenolines to polarize primary human macrophages and characterize their molecular and functional phenotypes. We demonstrate that xenoline-polarized macrophages diverge from classical M1/M2 states and instead adopt transcriptional signatures reflective of TAM subsets identified from patients. Notably, macrophages polarized with the radiation-therapy selected xenoline, JX14P-RT, exhibited gene expression patterns enriched for interferon response and hypoxia, mirroring recurrent GBM samples. In contrast, JX14P TAMs showed enrichment in phagocytic gene sets. Functional validation of these phenotypes revealed discrepancies between the transcriptionally predicted and observed phenotypes, emphasizing the importance of integrating phenotypic validation in sequencing studies. Altogether, our findings establish xenoline-polarized macrophages as a useful alternative to traditional models that can be used to study immune-interactions in vitro. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02057-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f15f733124e62cedabe127db49de61b0e9f489" target='_blank'>
              Xenoline-polarized macrophages as an alternative in vitro model of tumor-associated macrophages in glioblastoma
              </a>
            </td>
          <td>
            Acta Neuropathologica Communications, Christopher D. Willey, Hasan Alrefai, Benjamin Lin, Amr Elkholy, Manoj Kumar, Taylor L. Schanel, Kevin J. Lee, Patricia H. Hicks, Joshua C. Anderson, Gao Guo, Eun-Young Erin Ahn, Ryan Miller
          </td>
          <td>2025-06-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e17536


 Background:
 High-grade cervical intraepithelial neoplasia (HG-CIN) affects 200,000 U.S. women annually, with rising prevalence among those aged 30 & older, many of whom missed the opportunity for HPV vaccination. Treatment remains predominantly surgical, with a notable absence of pharmacologic interventions. Clinical trial endpoints rely on histological regression, but standard microscopy cannot accurately distinguish between regressing and progressing lesions. Compelling evidence underscores the tumor microenvironment’s (TME) critical role in driving immune suppression and malignant transformation. Leveraging a novel spatial imaging platform, this study provides high-resolution molecular insights into HG-CIN directly from routine H&E-stained slides, enabling precise characterization of immune marker distributions and their implications for disease progression. This technology has the potential to transform tissue analysis and significantly impact drug development.
 Methods:
 Sixteen formalin-fixed paraffin-embedded (FFPE) tissues were obtained from the health system’s Biospecimen Repository, with twelve used to optimize the sequencing protocol directly from H&E slides. Sample was categorized as CIN3, benign, or micro-invasive cervical mucosa based on the history, Pap smear/HPV test results, and an expert review by a Gynecologic Pathologist. Library generation and sequencing were conducted using Visium Gene Expression (both SD & HD platforms) kits for FFPE (10x Genomics). Specific genetic markers for immunosuppressive cell activities were highlighted to demonstrate its differential presence in CIN3 vs normal cervical tissue.
 Results:
 Sixteen FFPE samples were analyzed, with 11 used to optimize sequencing protocols. Three CIN3, three normal, and one micro-invasive cervical cancer sample revealed distinct tumor, stromal, and immune microenvironments. CIN3 tissues had elevated Tregs markers (FOXP3, CD4), MDSCs (CD11b, CD33, HLA-DR), and the Treg trafficking molecules CCR7 and TGF-β within stromal regions. Additionally, CIN3 had high gene expression for CDKN2A, which encodes the immunosuppressive ligand PDL1.
 Conclusions:
 The CIN and cancer microenvironment demonstrated strong immunosuppressive markers across immune, stromal, and epithelial compartments, aligning with key known features of tumorigenesis and progression. This spatial imaging platform & optimized workflow captures biologically relevant data directly from standard routinely stained H&E slide, achieving near single-cell resolution, enabling histopathologic comparison to gene expression data. Our study revealed unique immune suppression profiles in CIN3 linked to HPV progression, highlighting the platform’s potential to transform tissue analysis and advance drug development applications.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65b12713b597d9f017570b06219fcd74b91e4800" target='_blank'>
              Spatial transcriptomics for profiling the microenvironment of cervical precancer progression from routine H&E slides: Applications for pharmaceutical development.
              </a>
            </td>
          <td>
            Mario R Castellanos, C. Regan, Seema Khutti, Kai Wang, M. Dhar, Jonathan Preall
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cell plasticity is a hallmark of cancer, enabling tumor cells to acquire multiple phenotypes responsible for tumor progression, metastasis, and therapy resistance. In this issue of the JCI, Kawai and colleagues leveraged genetically engineered mouse models (GEMM) of pancreatic ductal adenocarcinoma (PDAC) to demonstrate that loss of Pbrm1, a member of the SWI/SNF complex, drives dedifferentiation and aggressive tumor features. Pbrm1 loss activated a program of epithelial-to-mesenchymal transition (EMT) and allowed the emergence of poorly differentiated histologies that are commonly associated with high recurrence rate and dismal prognosis. These findings reveal the role of the SWI/SNF complex during PDAC evolution in maintaining cell identity and restraining the progression of this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51cf4e0b352b84ce0ee22909d350d6fe7531e126" target='_blank'>
              Switching on the evolutionary potential of pancreatic cancer: the tumor suppressor functions of PBRM1
              </a>
            </td>
          <td>
            Luigi Perelli, G. Genovese
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Non–small cell lung cancer (NSCLC) constitutes over 80% of lung cancer cases and remains a leading cause of cancer-related mortality worldwide. Despite the advent of immune checkpoint inhibitors, their efficacy is limited to 27 to 45% of patients. Identifying likely treatment responders is essential for optimizing healthcare and improving quality of life. We generated multiplex immunofluorescence (mIF) images, histopathology, and RNA sequencing data from human NSCLC tissues. Through the analysis of mIF images, we characterized the spatial organization of 1.5 million cells based on the expression levels for 33 biomarkers. To enable large-scale characterization of tumor microenvironments, we developed NucSegAI, a deep learning model for automated nuclear segmentation and cellular classification in histology images. With this model, we analyzed the morphological, textural, and topological phenotypes of 45.6 million cells across 119 whole-slide images. Through unsupervised phenotype discovery, we identified specific lymphocyte phenotypes predictive of immunotherapy response. Our findings can improve patient stratification and guide selection of effective therapeutic regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebcac8871256527456ef7fc38ff937674e4c6cc4" target='_blank'>
              Single-cell multimodal analysis reveals tumor microenvironment predictive of treatment response in non–small cell lung cancer
              </a>
            </td>
          <td>
            Yuanning Zheng, Christoph Sadée, Michael Ozawa, B. Howitt, Olivier Gevaert
          </td>
          <td>2025-05-23</td>
          <td>Science Advances</td>
          <td>2</td>
          <td>9</td>
        </tr>

        <tr id="The advent of the omics era has facilitated the identification of precise biomarkers for cancer progression, revealing a broader diversity of macrophage phenotypes beyond the traditional M1/M2 classification. Folate receptor 2 (FOLR2)-positive macrophages, co-expressing markers such as mannose receptor C-Type 1 (MRC1/CD206) and lymphatic vessel endothelial hyaluronan receptor 1(LYVE1), are an embryonically derived subset typically found around blood vessels in both tumor stroma and normal tissues. Despite FOLR2’s longstanding association with anti-inflammatory, immunosuppressive macrophages in tumors, its precise role in cancer progression remains unclear. Recent studies suggest that FOLR2+ macrophages can either promote or inhibit cancer progression, depending on their multifaceted roles in the tumor microenvironment. This review provides a comprehensive overview of the biological features, functional roles, molecular mechanisms, and therapeutic potential of FOLR2+ macrophages in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a55317e456e1c334fa4e34ea5c90a5105fdddf" target='_blank'>
              FOLR2+ macrophages in cancer: allies or enemies
              </a>
            </td>
          <td>
            Decheng Jiang, Ruiling Xiao, Jialu Bai, Xing Wang, Yuan Fang, Jianzhou Liu, Xiaohan Wang, Ming Cui, Lei You, Yupei Zhao
          </td>
          <td>2025-06-02</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938950b1958bbfaef501d56fa236f86c6e2c6141" target='_blank'>
              Multi-omic analysis of gallbladder cancer identifies distinct tumor microenvironments associated with disease progression.
              </a>
            </td>
          <td>
            Tao Zhou, Yanhong Wu, Shuai Li, Xinyao Qiu, Erdong Liu, Zhihua Xie, Xuebing Shi, Yani Zhang, Guosheng Ma, Wenbo Guo, Xiang Wang, Kaiting Wang, Xiaomeng Yao, Ji Hu, Siyun Shen, Shuai Yang, Xiaoqing Jiang, Jing Fu, Hongyang Wang, Jin Gu, Lei Chen
          </td>
          <td>2025-06-26</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54dd2ec057c7a25df4bed262e9ae15c8f28acaf6" target='_blank'>
              Mapping the Transcriptional Landscape of Drug Responses in Primary Human Cells Using High-Throughput DRUG-seq
              </a>
            </td>
          <td>
            Lauren Baugh, Sébastien Vigneau, Srijani Sridhar, Sarah Boswell, George Pilitsis, John Bradley, Olga Allen, Laura M. Rand, Amanda Riccardi, Kelsey Roach, Kevin Lema, Ayla Ergun
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f4e542b3c6d086c581d428766c82b0185e38fd" target='_blank'>
              Intratumoral heterogeneity in microsatellite instability status at single cell resolution
              </a>
            </td>
          <td>
            Harrison Anthony, C. Seoighe
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a992d80c35fbb9812f59aed82ae176945f099f6c" target='_blank'>
              The Aging Microenvironment as a Determinant of Immune Exclusion and Metastatic Fate in Pancreatic Cancer
              </a>
            </td>
          <td>
            Priyanka Gupta, Rabi Murad, Li Ling, Yijuan Zhang, Karen Duong-Polk, Swetha Maganti, C. Galapate, C. Commisso
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Lung metastasis remains a leading cause of cancer mortality, driven not only by primary tumors but crucially by dynamic interactions within the premetastatic niche (PMN). Emerging evidence reveals that organ‐specific PMN formation precedes tumor cell arrival, creating an immunosuppressive microenvironment conducive to metastatic colonization. Primary tumor‐derived factors interact with lung resident cells, promoting the recruitment of bone marrow‐derived cells and creating a favorable microenvironment for the migrating tumor cells. Despite progress, systematic integration of PMN's multilayered regulatory mechanisms—from mechanism to clinical application remains lacking. The review summarizes recent studies investigating the effects of PMN in the entire process of lung metastasis. Our discussion revolves primarily around extracellular vesicles, extracellular matrix remodeling, proinflammation, immunosuppression, angiogenesis, and metabolic reprogramming. We emphasize the impact of various treatment strategies for primary tumors on the PMN in clinical practice, while also summarizing current diagnostic methods and exploring new therapeutic possibilities in the PMN. Finally, we propose a roadmap for future investigations into the lung PMN, highlighting prioritized research axes that bridge mechanistic discoveries with bench‐to‐bedside translation of PMN‐targeted interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0871bd03da6cd8fd6aae769050e9eb31d43909dc" target='_blank'>
              Targeted Lung Premetastasis Niche: Mechanisms, Strategies, and Application
              </a>
            </td>
          <td>
            Chenghao Cao, Di Lu, Huigang Li, Shen Pan, Jianyong Zhuo, Zuyuan Lin, Chiyu He, Peiru Zhang, Songmin Ying, Xuyong Wei, Shusen Zheng, Zhe Yang, Xiao Xu
          </td>
          <td>2025-06-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Background The heterogeneity of colorectal cancer (CRC) and its complex immune microenvironment pose significant challenges for treatment. Understanding the cellular composition and dynamic changes is essential for uncovering mechanisms of tumour progression. Methods To investigate the cellular heterogeneity and immune microenvironment of CRC, identifying critical subpopulations, functional pathways, and prognostic biomarkers, single-cell transcriptomic data from 41 CRC samples across four datasets were integrated. Bioinformatic analyses identified cellular subpopulations, cell communication networks, and prognostic biomarkers. The expression patterns, clinical significance and biological function of TUBB were validated in vitro. Results A distinct epithelial subpopulation with proliferative and invasive features was identified, promoting tumour progression by resisting apoptosis and remodelling the extracellular matrix. ActMono, a terminal state of myeloid cells, was enriched in tumours and linked to disease progression. Cell communication analysis highlighted galectin signalling in immune regulation. A prognostic model (CRS) based on secretory immune cell-related genes identified TUBB as a key molecule influencing the cell cycle and extracellular matrix remodelling, with its expression patterns, clinical significance and biological effects validated in vitro. Conclusion This study reveals critical subpopulations, signalling pathways, and biomarkers in CRC, providing insights into tumour progression and potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e4b3d5d75559211564431db384708f82801552" target='_blank'>
              Single-cell sequencing reveals dysregulated cell type perturbations and critical mediator communication remodelling in colorectal cancer
              </a>
            </td>
          <td>
            Chengyuan Xu, Siqi Zhang, Zhouyu Zhang, Luoqiu Zhang, Bin Sun, Zicheng Yu, Hailong Liu
          </td>
          <td>2025-06-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="During cancer development, mutations promote gene expression changes that cause transformation. Leukemia is frequently associated with aberrant HOXA expression driven by translocations in nucleoporin genes or KMT2A, and mutations in NPM1. How disparate mutations converge on this regulatory pathway is not understood. Here we demonstrate that mutant NPM1 (NPM1c) forms nuclear condensates in multiple human cell lines, mouse models, and primary patient samples. We show NPM1c phase separation is necessary and sufficient to coordinate the recruitment of NUP98 and KMT2A to condensates. Through extensive mutagenesis and pharmacological destabilization of phase separation, we find that NPM1c condensates are necessary for regulating gene expression, promoting in vivo expansion, and maintaining the undifferentiated leukemic state. Finally, we show that nucleoporin and KMT2A fusion proteins form condensates that are biophysically indistinguishable from NPM1c condensates. Together, these data define a new condensate underlying leukemias that we term coordinating bodies (C-bodies), and propose C-bodies as a therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0742ebc47874d0a681b5ec4cb75157b2b57840b5" target='_blank'>
              Nuclear Phase Separation Drives NPM1-mutant Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Gandhar K. Datar, Elmira Khabusheva, Archish Anand, Joshua Beale, Marwa Sadek, Chun-Wei Chen, Evdokiia Potolitsyna, Nayara Alcantara-Contessoto, Guangyuan Liu, Josephine De La Fuente, Christina Dollinger, A. Guzman, Alejandra G Martell, Katharina Wohlan, A. Maiti, N. Short, S. S. Yi, V. Andresen, Bjørn T. Gjertsen, B. Falini, R. E. Rau, L. Brunetti, Nidhi Sahni, M. Goodell, Joshua A. Riback
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of prostate cancer that strongly correlates with metastatic burden and appears prominently in both primary cancer and metastatic disease. Low Gleason score primary prostate tumors display pervasive centrosome loss, a known mechanistic driver of CIN, that disrupts normal spindle assembly and increases mitotic errors. Previously, we found that transient depletion of centrosomes in immortalized, non-tumorigenic prostate epithelial cells (PrEC) induced a burst of CIN, generating cell lines capable of forming xenograft tumors. Here, we use a multi-omics approach to identify the oncogenic alterations caused by transient centrosome loss. By integrating genomic and transcriptomic data of the prostate lines, we identified a consensus set of focal copy-number variations (CNVs) induced by centrosome loss in cultured cells that are also detectable within a subset of samples from a prostate cancer patient cohort. Using this CNV signature, we were able to derive a unique transcriptomic signature from prostate cancer patient samples that showed strong predictive value for adverse clinical outcomes. In summary, our experimental system uses centrosome loss to promote a punctuated burst of genomic crisis that is characteristic of genome evolution during prostate cancer progression. Consequently, this prostate cancer model produced recurrent structural variations that are detectable in patient samples and associate with worse outcomes. Graphic Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7263ae00a48684dacddf5d2cc713b37a7dad7c68" target='_blank'>
              Transient centrosome loss in cultured prostate epithelial cells induces chromosomal instability to produce an oncogenic genotype that correlates with poor clinical outcomes
              </a>
            </td>
          <td>
            Jiawen Yang, Diogo de Oliveira Pessoa, John M. Ryniawec, Matthew R. Coope, Daniel W. Buster, Emily Loertscher, Mengdie Wang, Chen Chen, A. Cress, Gregory C Rogers, Megha Padi
          </td>
          <td>2025-05-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Esophageal squamous cell carcinoma (ESCC) is a highly heterogeneous malignancy. Traditional clinical staging systems have limited value in prognostic evaluation and treatment guidance, often failing to capture the profound impact of intratumoral diversity on patient outcomes. Single-cell RNA sequencing (scRNA-seq) provides new perspectives on the cellular makeup and conditions in tumor tissues, with promising implications for functional classification and personalized therapeutic strategies in esophageal cancer. Methods In this study, we integrated scRNA-seq data with bulk RNA-seq profiles from esophageal cancer tissues to construct a comprehensive cellular atlas, focusing on the transcriptional characteristics of epithelial cells. Malignant epithelial cells were identified based on copy number variation (CNV) features using inferCNV analysis. Their developmental states and regulatory mechanisms were further characterized via transcription factor activity inference (SCENIC) and pseudotime trajectory analysis (Monocle). Based on marker genes of malignant epithelial subpopulations, we developed a multi-gene risk scoring model using data from the TCGA and GEO (GSE53624) cohorts. The model’s predictive value for immune landscape, mutational features, and drug sensitivity was also evaluated. Additionally, qRT-PCR was conducted to quantify the expression levels of model genes in ESCC tissue samples, further evaluating their biological relevance. Functional roles of the key gene HMGB3 were validated in vitro through CCK-8 proliferation assays, Transwell invasion assays, and colony formation assays following gene knockdown in ESCC cell lines. Results At the single-cell level, we identified ten major cell types and six distinct malignant epithelial subclusters, which exhibited pronounced heterogeneity in cell cycle states, transcriptional regulatory networks, and differentiation trajectories. High CNV scores and the enrichment of specific transcription factors (e.g., FOXC1, E2F1, RUNX1) suggested a proliferative and immune-evasive phenotype. A six-gene prognostic model (HMGB3, CHORDC1, CTSD, BTG2, MT1E, PHYHD1) showed strong predictive accuracy for overall survival in the TCGA and GSE53624 cohorts. Furthermore, the risk score showed a significant correlation with an immunosuppressive tumor microenvironment, increased tumor purity, and the activation of certain immune-related pathways. Analysis of drug sensitivity suggests that patients classified as low-risk could respond better to various targeted therapies and chemotherapeutic agents, underscoring their potential clinical relevance. Functional assays revealed that HMGB3 knockdown markedly suppressed ESCC cell proliferation, invasion, and colony formation, supporting its oncogenic role in esophageal cancer progression. Conclusion This study systematically characterized epithelial cell heterogeneity in esophageal cancer at single-cell resolution and established a risk model based on malignant epithelial markers that effectively predicts prognosis, immune status, and potential drug response. Combined with experimental validation, our findings highlight the pivotal role of HMGB3 in ESCC progression and provide a novel theoretical and practical framework for functional tumor classification and individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70d92eb766fe9a932637e37c8c15e4650793e3d8" target='_blank'>
              Malignant epithelial cell marker–driven risk signature enables precise stratification in esophageal cancer
              </a>
            </td>
          <td>
            Hao Zhang, Shizhao Cheng, Yijun Xu
          </td>
          <td>2025-05-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance poses a significant challenge in cancer therapy, contributing to rapid recurrence, disease progression, and high patient mortality. Despite its critical impact, few reliable predictors for cancer drug response or failure have been established for clinical application. Tumor heterogeneity and the tumor microenvironment (TME) are pivotal factors influencing cancer drug efficacy and resistance. Tumor heterogeneity leads to variable therapeutic responses among patients, while dynamic interactions between cancer cells and the TME enhance tumor survival and proliferation, underscoring the urgent need to identify cellular hallmarks for predicting drug response and resistance. Single-cell and spatial omics technologies provide high-resolution insights into gene expression at the individual cell level, capturing intercellular heterogeneity and revealing the underlying pathologies, mechanisms, and cellular interactions. This review delves into the principles, methodologies, and workflows of single-cell and spatial omics in cancer drug research, highlighting key hallmarks involving tumor heterogeneity, TME reprogramming, cell–cell interactions, metabolic modulation, and signaling pathway regulation in drug treatment at single-cell and spatial levels. Furthermore, we synthesize predictive cellular biomarkers for cancer drug response and resistance across 25 cancer types, paving the way for advancements in cancer precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af5a8672965af056452be6aaf8b56f1ff303b62" target='_blank'>
              Application of single-cell and spatial omics in deciphering cellular hallmarks of cancer drug response and resistance
              </a>
            </td>
          <td>
            Xiaoxia Cheng, Ting Peng, Tian Chu, Yiqun Yang, Jia Liu, Qinglei Gao, Canhui Cao, Juncheng Wei
          </td>
          <td>2025-07-02</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0e9134dab7b134dd165787785bd9a7f1f9b1843" target='_blank'>
              Deep learning inference of cell type-specific gene expression from breast tumor histopathology
              </a>
            </td>
          <td>
            Andrew Wang, S. R. Dhruba, Kun Wang, Emma M. Campagnolo, E. Shulman, E. Ruppin
          </td>
          <td>2025-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background
 The tumor microenvironment and biomarkers play a pivotal role in breast cancer research, yet there remains a pressing need for effective biomarkers. This study focuses on identifying a novel IGKC+ T Cell subpopulation and its related biomarkers to pave the way for innovative targeted therapies and improved clinical outcomes.


Methods
We first performed single-cell RNA sequencing (scRNA-seq) analysis to characterize immune cell heterogeneity within the tumor microenvironment, leading to the identification of series cell subpopulation. Then, by performing univariate analysis to correlate cell proportions with patient prognosis, we identified a novel IGKC+ T cell subpopulation. Next, we applied bulk RNA-seq deconvolution algorithms to estimate the abundance of this subpopulation across breast cancer cohorts. Differential expression analysis and weighted gene co-expression network analysis (WGCNA) were employed to identify genes associated with the IGKC+ T cell population. To pinpoint key regulatory genes, we applied machine learning algorithms. Based on the hub genes identified, we constructed a prognostic risk model and developed a nomogram to aid clinical decision-making. Immune infiltration patterns were further assessed in high- vs low-risk groups defined by the model. Finally, functional validation was performed through overexpression of BCL2L14 in vitro, and downstream signaling pathways were examined.


Results
We identified the novel IGKC+ T cell subpopulation and core genes. Machine learning pinpointed BCL2L14, IGHD, MAPT-AS1, NT5DC4, and TNIP3 as key regulators of breast cancer progression in this subpopulation. The model stratified patients into high- and low-risk groups, with high-risk patients showing worse prognosis and weaker immune infiltration. Overexpression of BCL2L14 was experimentally demonstrated to accelerate breast cancer progression, linked to enhanced phosphorylation of the NF-κB pathway.


Conclusion
Our results underscore BCL2L14 as a potential driver within the novel T-cell subpopulation and a critical biomarker for breast cancer diagnosis. These findings provide a basis for developing advanced diagnostic tools and targeted therapies, which may ultimately enhance patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/899ab08619ae94e923463082f4db3808d873da0a" target='_blank'>
              Integration of Bulk and Single-Cell Transcriptomics Reveals BCL2L14 as a Novel IGKC+ T Cell-Associated Therapeutic Target in Breast Cancer.
              </a>
            </td>
          <td>
            Jiaming He, Aiman Akhtar, Jing Li, Qiang Wei, Yang Yuan, Jianhua Ran, Yongping Ma, Dilong Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of inflammation research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Renal cell carcinoma is characterized by a poor prognosis. Recently, renal cell carcinoma has been recognized as a metabolic disease associated with fatty acid metabolic reprogramming, although in-depth studies on this topic are still lacking. We found that fatty acid metabolism reprogramming in renal cell carcinoma is primarily characterized by high expression of FABP1. FABP1 + tumors significantly impact survival and display distinct differentiation trajectories compared to other tumor subclusters. They show elevated expression of angiogenesis and cell migration signals, with PLG-PLAT-mediated interactions with endothelial cells notably enhanced. Spatial transcriptomics show a prominent co-localization of FABP1 + tumors with endothelial cells, and their spatial distribution closely aligns with that of PLAT + endothelial cells. FABP1 + tumors exhibit a unique pattern in spatial transcriptomics, enriched in Extracellular Matrix and angiogenesis-related pathways. Through receptor-ligand interaction analysis, a novel PLG-PLAT functional axis was found between tumor epithelial cells and endothelial cells. Based on results of experiments, we infer that FABP1 + tumors can promote plasmin-related tumor angiogenesis by triggering the PLG-PLAT signaling axis. Finally, utilizing preclinical models, we suggest that targeting the FABP1-PLG-PLAT axis may serve as promising strategy enhancing the sensitivity of Tyrosine Kinase Inhibitor therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02377-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27626d5d72e69d9402b111fc64dabcbee4819f07" target='_blank'>
              Single-cell multi-omics reveals that FABP1 + renal cell carcinoma drive tumor angiogenesis through the PLG-PLAT axis under fatty acid reprogramming
              </a>
            </td>
          <td>
            Yiqiu Wang, Yingchun Liang, Min Li, Jiayi Lu, Sian Zhou, Yaoyu Yu, Changwei Yang, Xinhuang Hou
          </td>
          <td>2025-06-16</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gliomas, particularly glioblastomas, are highly malignant brain tumors with high recurrence rates and poor prognosis. Despite advances in treatment, recurrence remains a major challenge. Epithelial-mesenchymal transition (EMT) plays a key role in tumor invasion and recurrence. This study explores the transcriptional and regulatory mechanisms driving glioma recurrence, focusing on mesenchymal-like (MES-like) subpopulations. Single-nucleus RNA sequencing was performed on 52 IDH wild-type GBM specimens, including 26 primary and 26 recurrent tumors. Spatial transcriptomics data were also incorporated. Tumor subpopulations were identified through gene regulatory network analysis, copy number variation detection, and nonnegative matrix factorization. Functional validation was conducted using gene knockdown experiments, followed by xenograft studies. We discovered novel MES-like subpopulations in recurrent GBM enriched with EMT-related genes like EGR1 and SERPINE1. These subpopulations exhibited increased transcriptional activity and were associated with poor prognosis and invasiveness. Knockdown of SERPINE1 significantly reduced cell proliferation and migration. Spatial transcriptomics showed MES-like cells concentrated at the tumor margins, highlighting their role in invasion and recurrence. MES-like subpopulations, driven by EGR1 and SERPINE1, are critical in GBM. Targeting these regulators could offer new therapeutic strategies to reduce glioma recurrence and improve outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02036-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f7432f6a50e6a5d830f429844d915dd10fcd2" target='_blank'>
              Single-cell transcriptome sequencing reveals new epithelial-stromal associated mesenchymal-like subsets in recurrent gliomas
              </a>
            </td>
          <td>
            Jinwei Li, Shengrong Long, Yang Zhang, Shuangqi Yu, Hongyu Xu, Rui Liang, Quan Liu, Jinnan Zhang, Xiang Li, Yixin Fu, Tao Xin, Yinyan Wang
          </td>
          <td>2025-06-07</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb25978847c5553a988089ffa28339e8dff51f1" target='_blank'>
              Single-Cell profiling reveals targetable malignant T-cell subtypes and immune evasion pathways in Sezary Syndrome
              </a>
            </td>
          <td>
            M.S.F Beth A. Childs, Ph.D Eslam A. Elghonaimy, Praveen, Ramakrishnan Geethakumari, Kiran A. Kumar, J. F. Merola, M. D. Heather W. Goff, M. Todd A Aguilera, Ph.D Todd
          </td>
          <td>2025-05-21</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Treatment-induced neuroendocrine prostate cancer (t-NEPC) is a lethal, castration-resistant subtype of prostate cancer. While t-NEPC typically arises from adenocarcinoma through neuroendocrine transdifferentiation after androgen pathway inhibition, the temporal dynamics and molecular drivers of this process remain poorly understood. Here, utilizing the first-in-field patient-derived xenograft (PDX) model of adenocarcinoma-to-NEPC transdifferentiation (LTL331/331R), we performed longitudinal single-cell transcriptomic sequencing (scRNA-seq) across seven timepoints spanning pre-castration to relapsed NEPC. Our analysis demonstrated 15 distinct cell clusters, including twelve adenocarcinoma clusters and two NEPC clusters (ASCL1high/FOXA2low and ASCL1low/FOXA2high clusters). Notably, we revealed a newly-discovered, early intermediate transitional cell state during t-NEPC development distinguished by epithelial-mesenchymal transition (EMT), stem cell-related, metabolically active, and HDAC-associated regulatory signatures. Analysis of this intermediate transitional cluster led to the identification of RUNX1T1 as a pivotal transcriptional regulator of NEPC transdifferentiation. Functionally, RUNX1T1 overexpression promoted AR pathway inhibition (ARPI) -induced NE transdifferentiation and increased resistance to ARPI treatment in prostate adenocarcinoma. RUNX1T1 knockdown reverses the NE transdifferentiation, inhibits NEPC cell proliferation and induces apoptosis, and cell cycle arrest. In summary, this study identifies a critical intermediate transitional cell state during t-NEPC development and reveals the heterogeneity of terminal NEPC, offering new insights into NEPC biology and emphasizing the importance of early intervention. Moreover, the discovery of RUNX1T1 as a key early driver active in both the initial and terminal phases of NEPC progression presents promising opportunities for therapeutic intervention. Highlights Longitudinal single-cell RNA sequencing of treatment-induced adeno-to-NE transdifferentiation model revealed an intermediate transitional cell state in NEPC development and progression. The sequential emergence of ASCL1high/FOXA2low and ASCL1low/FOXA2high NEPC subclusters indicates a temporal evolution of NEPC and highlights their role in contributing to NEPC heterogeneity. RUNX1T1, a transcriptional regulator showing increased expression in both intermediate cell state and NEPCs, is identified as an early driver of NEPC development. RUNX1T1 plays important functional roles in promoting adeno-to-NE transdifferentiation in early phase of NEPC development and maintaining NE phenotype and aggressiveness in terminal NEPC. Targeting RUNX1T1 or its complex could offer new therapeutic strategies for NEPC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3dfda20cdaed4a7ffd5a497959ce67fe25256c9" target='_blank'>
              Longitudinal single-cell analysis reveals RUNX1T1 as an early driver in treatment-induced neuroendocrine transdifferentiation
              </a>
            </td>
          <td>
            Yuchao Ni, Dong Lin, Mingchen Shi, Yen-Yi Lin, Hui Xue, Xin Dong, Liang Liu, F. Sar, Rebecca Wu, Tunc Morova, A. Haegert, Robert Bell, Xinyao Pang, Adam Classen, Yu Wang, Junru Chen, Stanislav Volik, S. L. Bihan, Nathan A. Lack, Christopher Ong, Gang Wang, H. Zeng, Colin C. Collins, Yuzhuo Wang
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Patients with high-grade serous ovarian cancer (HGSOC) typically present with widespread metastasis, obscuring a temporal understanding of tumor-immune dynamics. To address this, we perform multi-site global proteomics alongside matched immunohistochemistry (IHC) for CD4⁺ and CD8⁺ tumor-infiltrating lymphocytes (TILs) in patient samples. We order the protein expression profiles using an unbiased pseudotime analysis, recapitulating clinical observations of metastatic progression, and providing a framework to explore tumor-immune dynamics from localized to metastatic disease. Metastatic progression correlates with immune cell infiltration, the recruitment of regulatory T cells (Tregs) to counterbalance γδ T cell abundance, and an increased abundance of exhausted CD8⁺ T cells. The accumulation of Tregs at metastatic sites correlates with SNX8 expression, a critical regulator of the STING pathway. In early-stage tumors, keratin-expressing cancer cells recruit Tregs via MHC class II, fostering an inflammatory phenotype with limited IFNγ production and non-clonally expanded T cells. Together, our findings reveal novel mechanisms of immune escape associated with both localized disease and metastatic progression in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5284b73444d2932c5ad492644667337328f9e74" target='_blank'>
              A temporal model of tumor-immune dynamics during the metastatic progression of high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Donagh Egan, Kate Glennon, Ann Treacy, Aurelie Fabre, Janet Mccormack, Salisha Hill, Ryan Morrison, Vadim Zhernovkov, Daniel C Liebler, Walter Kolch, Donal J Brennan
          </td>
          <td>2025-06-16</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Salivary gland cancers are rare, diverse malignancies characterized by poor response to immunotherapy. The tumor immune environment in these cancers remains poorly understood. To address this, we perform an integrative analysis of the tumor immune microenvironment in a large cohort of advanced salivary gland cancer samples. Most tumors exhibit low immune activity with limited immune cell infiltration. Inflammation is linked to higher tumor mutational burden in non-adenoid cystic carcinoma histologies. Subtype specific expression of immune checkpoints is identified with prominent expression of VTCN1 in luminal-like cells within adenoid cystic carcinoma. Macrophages with immunosuppressive properties dominate the immune microenvironment across subtypes. Responses to immunotherapy are limited and associated with a higher ratio of T-cells relative to macrophages in individual cases, warranting further investigation. Here, we show an immunosuppressive environment in salivary gland cancers and identify subtype-specific immune vulnerabilities that could inform tailored therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5d16057c5cf3334337ef5c688188d79ab5741" target='_blank'>
              A macrophage-predominant immunosuppressive microenvironment and therapeutic vulnerabilities in advanced salivary gland cancer
              </a>
            </td>
          <td>
            Erika Zuljan, B. von der Emde, I. Piwonski, Ana Pestana, K. Klinghammer, A. Mock, Peter Horak, Christoph Heining, Frederick Klauschen, Ina Pretzell, M. Boerries, Christian H. Brandts, S. Kreutzfeldt, Maria-Veronica Teleanu, D. Huebschmann, L. Morris, M. Heiland, U. Keller, T. Conrad, H. Glimm, S. Fröhling, Sebastian Ochsenreither, Ulrich Keilholz, Eric Blanc, D. Beule, D. Rieke
          </td>
          <td>2025-06-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 e15561


 Background:
 Colorectal cancer (CRC), the third most prevalent cancer globally, remains a major cause of cancer-related mortality. Genomic and epigenomic plasticity during CRC progression, particularly metastasis, poses significant challenges to effective treatment. Patient-derived organoids (PDOs) reflect the heterogeneous genetic and morphological composition of cancer cells, enabling ex vivo disease modeling. This study investigates the epigenomic makeup of human primary and metastatic CRC to advance precision epigenetic therapies.
 Methods:
 We analyzed 45 organoid models, including 40 CRC organoids (from primary tumors and liver/lung metastases) and 5 generated from healthy colonic tissue. Using integrative epigenomic profiling, including chromatin accessibility, chromatin state mapping (via six histone modifications), and 3D genome conformation, we constructed a comprehensive CRC epigenomic landscape.
 Results:
 Our analysis of CRC patient-derived organoids revealed a core set of accessible genomic regions shared across all organoids, irrespective of tumor location or metastatic status. This conserved chromatin signature highlights common regulatory pathways in CRC during disease progression. Unsupervised clustering based on epigenomic features revealed four distinct CRC subtypes, encompassing both primary and metastatic models. Notably, organoids derived from liver and lung metastases did not cluster by metastatic site, underscoring epigenomic convergence. Subtype D comprised 57.1% (8/14) of primary tumors, while subtypes B and C contained 47.3% (9/19) of liver metastases and 42.8% (3/7) of lung metastases, respectively. KRAS mutation status further stratified subtypes, with wild-type KRAS observed exclusively in subtypes A and B, while KRAS-mutant variants were enriched in subtypes C (6/7) and D (4/10). Histone modification patterns, defined by six marks (H3K4me1, H3K4me3, H3K27me3, H3K27ac, H3K36me3, and H3K9me3) revealed subtype-specific regulatory chromatin landscapes. Three-dimensional chromatin interactions were mapped, and genome-wide interaction contacts are used to identify various chromatin structures that may influence metastatic capacity.
 Conclusions:
 This study highlights distinct and shared epigenomic signatures in CRC PDOs, offering novel insights into tumor progression and metastasis. Integrative chromatin analysis provides a framework for identifying epigenetic biomarkers and epigenome targeting strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66760aad28585c2f41842514d0645e6e13f51968" target='_blank'>
              Deciphering colorectal cancer progression through integrative epigenomic analysis for precision therapeutics.
              </a>
            </td>
          <td>
            Ghazaleh Tavallaee, Amin Nooranikhojasteh, Elias Orouji
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Although the clinical significance of plasma cells within tumors has been recognized, studies on the development of plasma cells and the characteristics of the antibodies they secrete within the tumor microenvironment remain limited. We investigated the properties of plasma cells within cancer tissues using single-cell RNA and single cell B cell receptor sequencing. We characterized plasma cells exhibiting clonal expansion and synthesized the antibodies produced by these cells, confirming the clinical relevance of immunoglobulin H (IGH) isotypes. Plasma cells comprised approximately 5% of the total immune cell population within the tumor; clonal expansion was more prevalent in plasma cells than in B cells. Among plasma cells, the most frequent immunoglobulin isotype was IGHG1 and IGKC. We synthesized six recombinant antibodies, including those from the largest clonal plasma cells. Two antibodies that formed clones showed membranous staining in cancer cells. The cancer cells that metastasized to the lymph node showed a loss of expression as observed by immunohistochemistry. Analysis of bulk RNA sequencing data from 1078 patients with breast cancer revealed that tumor-infiltrating plasma cells expressing IGHG1 were associated with favorable prognoses. These tumors exhibited increased B cell receptor diversity, immunogenic mutation, and intratumoral heterogeneity. This study suggests the potential for discovering cancer-associated antibodies derived from intratumoral plasma cells. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04045-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54a09d3573d6929df29bef067a1cea30d57f996f" target='_blank'>
              Recombinant antibodies from clonally expanded cancer-associated plasma cells
              </a>
            </td>
          <td>
            Wonkyung Kim, Ji-Hye Oh, Chae Won Park, Hyori Kim, Young Gwang Kang, Da Eun Oh, H. Lee, Ji Hun Kim, Chang Ohk Sung
          </td>
          <td>2025-05-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Analysis of post-translational modifications (PTMs) of proteins can provide new insight, beyond that obtained from analysis of protein levels, for understanding the tumor microenvironment (TME). The characteristics of PTMs in immune cells, along with their spatial distribution, have not been comprehensively integrated, which impedes our understanding of the complexity and heterogeneity of the TME in hepatocellular carcinoma (HCC). Herein, we used a strategy that combines antibodies for specific PTMs with mass cytometry and mass spectrometry technologies to identify PTMs at single-cell resolution. We found that the phosphorylation status of M2 macrophages was substantially altered in tumor tissues from patients with hepatitis B virus (HBV)-positive HCC. Utilizing the expression profiles of site-specific phospho-HSP27, STAT1, and TRIM28, we classified M2 macrophages into four distinct subtypes: M2-P0 (absence of any of the three phospho-proteins), M2-P1 (presence of one of the three phospho-proteins), M2-P2 (presence of two of the three phospho-proteins), and M2-P3 (presence of all three phospho-proteins). The spatial relationships and functional characteristics of these M2 macrophage subpopulations were assessed using single-cell PTM (scPTM) omics. The abundance of the M2-P2 and M2-P3 subtypes was closely associated with an immunosuppressive TME and responsiveness to immunotherapy in HBV+ HCC. Overall, this study introduces a scPTM omics approach that uncovers subtypes of macrophages associated with immunotherapeutic responses in HBV+ HCC and provides valuable insights into the immunosuppressive TME of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/122793fb7689acfc8cdc967dfa3fecfbbdbf0958" target='_blank'>
              Single-cell analysis of post-translational modifications identifies immunosuppressive macrophage subtypes in the HBV-positive hepatocellular carcinoma microenvironment.
              </a>
            </td>
          <td>
            Huakan Zhao, Ran Ren, Xi Zhang, Mengtao Zhan, Jinwei Cui, Jun Zhang, Xi Liu, Lei Wu, Yu Chen, Yu Zhou, Yang Xiao, Jiangang Zhang, Yang Chen, Lu Zheng, Bing Sun, Yongsheng Li
          </td>
          <td>2025-06-02</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 7070


 Background:
 Follicular lymphoma (FL) exhibits considerable variability in biological features and clinical trajectories, compounded by a complex tumor microenvironment (TME) populated with nonmalignant immune cells. In this study, we comprehensively characterized FL by integrating multi-omics data that considered both tumor cells and microenvironmental components. This analysis revealed four subtypes of FL, each exhibiting distinct biological characteristics and differing clinical behaviors.
 Methods:
 Our study included 1,203 samples, 53 bulk transcriptomic samples from our center, 1,150 samples from public databases, and five single-cell RNA sequencing (scRNA-seq) cohorts. The Ecotyper algorithm was employed to identify immune cell states. Unsupervised clustering analysis using non-negative matrix factorization (NMF) was performed to identify distinct FL subtypes based on cellular infiltration.
 Results:
 A total of 30 unique cell states were identified from nine annotated cell populations (B cells, plasma cells, CD4 T cells, CD8 T cells, Tregs, NK cells, Tfh, monocytes/macrophages, and dendritic cells). Validation was performed across three independent scRNA-seq and bulk transcriptomic cohorts to ensure robustness. Four distinct B cell states were identified, each exhibiting unique gene expression profiles and varying prognostic implications. B cell S2 and S3 were associated with adverse outcomes. Notably, PRDM15 in B cell S2 had the most significant prognostic impact, remaining an independent adverse prognostic factor even after adjustment for m7-FLIPI. Mutational profile analysis demonstrated distinct mutation patterns across B cell states: mutations in CCND3, SPEN, and ARID1A were enriched in B cell S1; CREBBP and STAT6 mutations were prominent in B cell S2; and IRF8 and BCL2 mutations were observed in B cell S4, while no significant mutation enrichment was detected in B cell S3. Similarly, immune cells of the same type yet in different states exhibited functional and clinical heterogeneity. Four subtypes of follicular lymphoma characterized by distinct tumor cell states and varying immune infiltration were identified through unsupervised clustering analysis. FLE1 exhibits characteristics of a "cold tumor" with a high abundance of B cell S2. In contrast, FLE2 and FLE3 show moderate immune infiltration, while FLE4 is marked by abundant immune infiltration and elevated expression of immunosuppressive checkpoint molecules, resembling an "inflammatory" tumor phenotype. Prognostic analysis indicates that FLE1 has the most unfavorable prognosis, followed by FLE4.
 Conclusions:
 Our study stratifies FL patients based on the heterogeneity of tumor cells and the immune microenvironment, proposing an immune-based classification for FL. Targeting CREBBP and PRDM15 may offer promising new strategies for the clinical management of patients exhibiting a "cold tumor" phenotype.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0fb822abee7b71f9392d723669dd1a087479f12" target='_blank'>
              Clinical and biological subtypes of follicular lymphoma revealed by tumor and immune cell states.
              </a>
            </td>
          <td>
            Hengqi Liu, Yingfang Feng, Ning Zhang, Cong Sun, Ning Zhang, Xianhuo Wang, Huilai Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is crucial for the onset, development, and resistance to treatment of lung cancer. The tumor microenvironment consisting of a complex array of immune cells, fibroblasts, endothelial cells, extracellular matrix elements, and signaling molecules, facilitates tumor growth and spread while inhibiting the body’s antitumor immune response. In lung cancer, tumor-associated macrophages, cancer-associated fibroblasts, mast cells, and dendritic cells interact through cytokines, chemokines, growth factors, and matrix metalloproteinases to create an immunosuppressive and proangiogenic milieu. Hypoxic conditions within the TME further enhance cancer cell adaptability through hypoxia-inducible factors (HIFs), promoting epithelial–mesenchymal transition, immune evasion, and metastasis. Moreover, miRNAs have emerged as key regulators of gene expression within the TME, offering novel insights into tumor behavior and potential therapeutic targets. Targeting dynamic interactions within the TME, particularly through the modulation of immune responses, angiogenesis, and stromal remodeling, offers promising avenues for precision pharmacological approaches. This review covers the current understanding of the lung TME, highlighting its impact on cancer pathophysiology and treatment strategies. Understanding and therapeutically reprogramming the TME may pave the way for personalized and more effective interventions for lung cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/704551a1fc0a296571f2de743b996534680087ce" target='_blank'>
              Tumor Microenvironment: An Emerging Landscape for Lung Cancer Therapy
              </a>
            </td>
          <td>
            S. Sohag, Sharmin Nur Toma, Md. Al-Imran Imon, Maiweilan Maihemuti, Famim Ahmed, Mst. Afsana Mimi, Imran Mahmud, Md. Mahmudul Hasan
          </td>
          <td>2025-06-30</td>
          <td>Future Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (ICC) is characterized by high malignancy, and its global incidence is predicted to continue to increase over the past decades. However, the mechanisms underlying ICC pathogenesis and progression remain unclear.


APPROACH AND RESULTS
The training cohort consisted of single-cell sequencing of 12 treatment-naïve ICC samples and spatial transcriptomics of four ICC samples. The validation cohort comprised of RNA-seq data from 87 ICC tumor samples. Finally, we validated our findings via multiplex immunofluorescence, organoids, and mice models both in vivo and in vitro. We found significant heterogeneity within the tumor microenvironment (TME) of ICC patients. ICC cells were classified into five molecular subtypes, and we revealed that aspartate beta-hydroxylase (ASPH) was a marker gene for invasion subtypes. We then selected cepharanthine (CEP) as an ASPH inhibitor that effectively suppressed tumor progression. Regarding the ICC lymphatic metastasis mechanism, we found that tumor cells in N1 lymph nodes highly expressed tumor-specific MHC-II molecules but lacked co-stimulatory factors CD80/CD86, inducing a state of anergy in CD4+ T cells, which might facilitate ICC immune evasion.


CONCLUSIONS
The TME of ICC was heterogeneous. ASPH markedly enhanced ICC invasion The ASPH inhibitor CEP significantly inhibits ICC progression and may serve as a targeted therapeutic drug for ICC. Tumor cells in N1 lymph nodes demonstrate high expression of tumor-specific MHC-II molecules, but silencing of co-stimulatory factors such as CD80/CD86 induces CD4+ T cells into an anergic state. Our study indicated that ASPH and MHC-II may serve as novel therapeutic targets for ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2012ee56dc72ff219a9edcce829b5a1b74d80284" target='_blank'>
              Tumor microenvironment heterogeneity and progression mechanisms in intrahepatic cholangiocarcinoma: A study based on single-cell and spatial transcriptomic sequencing.
              </a>
            </td>
          <td>
            Fengwei Li, Yao Li, Lishan Wang, Lei Xu, Hui Xue, Wenxin Wei, Yong Xia, Lei Wang, Feng Shen, Kui Wang
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer cell morphology reflects the dynamic interplay between genetic, epigenetic, and environmental factors that
drive tumor progression. This review explores the morphological evolution of cancer cells as a phenotypic manifestation of
underlying molecular dynamics, drawing on recent advances in evolutionary biology, cytoskeletal remodeling, computational
pathology, and tumor metabolism. Clonal evolution, both gradual and punctuated, generates morphological diversity that is
shaped by selective pressures from the tumor microenvironment and therapeutic interventions. Studies highlights the structural
and mechanical properties, such as reduced cellular stiffness and altered actin organization, are linked to metastatic potential
and cytoskeletal deregulation. Oncogenic signaling pathways (e.g., PI3K/AKT, Ras/MAPK) and metabolic reprogramming
further modulate cell shape and behavior. High-throughput imaging and machine learning have enabled quantification of both
static and dynamic morphological traits, correlating them with chromosomal instability and clinical outcomes. Morphological
heterogeneity has emerged as a powerful diagnostic and prognostic biomarker, with deep learning-derived metrics providing
scalable tools for cancer classification and risk stratification. Collectively, these insights underscore the clinical utility of
integrating morphological analysis with molecular profiling, offering new directions for personalized cancer diagnosis and
therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055067ac5bdde412ce5a95b6486dff3c99db46a4" target='_blank'>
              Morphology Matters: A Multidimensional Perspective on Cancer Cell Biology
              </a>
            </td>
          <td>
            Rose Rani
          </td>
          <td>2025-06-30</td>
          <td>International Journal for Research in Applied Science and Engineering Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525bc26ccdba95190ad1d88938dd65bd9ec49925" target='_blank'>
              Cell painting in activated cells illuminates phenotypic dark space and uncovers novel drug mechanisms of action
              </a>
            </td>
          <td>
            Matylda Zietek, Akshar Lohith, Derfel Terciano, Beverley Marie Rabbitts, Aswad S. Khadilkar, John B MacMillan, Scott Lokey
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mutated oncogenic Kirsten rat sarcoma virus (KRAS) antigen is expressed in a large variety of cancers, including pancreatic, colorectal, and pulmonary cancers. The oncogenic KRAS mutations cause malignancies and are usually ubiquitously expressed by all cells of a tumor. The KRAS amino acid substitutions at the positions 12 or 13 are among the most frequent mutations in human cancers. Here, we developed immuno-oncotherapeutic non-integrative lentiviral vectors encoding a segment encompassing KRASG12D, either alone, or associated with antigen carriers. These carriers can improve the intracellular antigen routing to major histocompatibility complex presentation machineries or provide universal helper CD4+ epitopes. Immunotherapy with one of these vectors resulted in significant immune control of tumor growth in colorectal or pulmonary preclinical cancer models, in several murine genetic backgrounds. The antitumor effect was correlated with increased proportions of intra-tumoral hematopoietic cells and notably CD8+ T cells. Although this effect was partial, it was robust, reproducible and advantageously combinable with conventional chemotherapies and immunotherapies to improve antitumor protection. Therefore, this approach shows promise as an immuno-oncotherapy against KRAS-mediated malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61951127f6ba1a1f776a4ed744c835e44ef0de6c" target='_blank'>
              A lentiviral vector targeting a KRAS neoepitope for cancer immunotherapy
              </a>
            </td>
          <td>
            Anastasia Goloudina, Fabien le Chevalier, P. Authié, Sylvain Ciret, K. Nemirov, Ingrid Fert, F. Moncoq, B. Vesin, A. Noirat, Catherine Blanc, Yu Wei, Pierre Charneau, L. Majlessi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67b3f2125e256fc001c7ec119739d1e135a5d03" target='_blank'>
              A Mathematical Model of Persister Cell Plasticity and Its Impact on Adaptive Cancer Therapy
              </a>
            </td>
          <td>
            Adithya Mathews, D. Basanta
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Targeting personalized tumor neoantigens is a promising strategy in immuno-oncotherapy, tumor heterogeneity requires that adaptive immunity be effectively induced against a broad spectrum of neoantigens for a significant anti-tumor effect. We developed several non-integrative lentiviral vectors encoding optimized immunogen that induce robust T cell responses against neoepitopes derived from murine colorectal tumors. Incorporating proteasome-targeting sequences and 4-alanine spacers between neoepitopes enhanced responses to both dominant and subdominant neoepitopes. Using longer natural sequences encompassing minimal epitopes further improved immunogenicity. These vectors drove complete regression of MC38 tumors expressing the selected neoepitopes and sustained immune memory to prevent relapse. Additionally, these vectors synergized with anti-programmed cell death protein 1 (PD1) therapy to inhibit wild-type MC38 tumor growth. Variant allele frequency tracking demonstrated T cells eradicated neoantigen-positive cells without affecting negative ones. Our results validate lentiviral vectors for personalized neoepitope therapy and underscore the need for diverse neoantigens in immunotherapy against tumor mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cc98109769dae2c9eb97fb1704cc3bfecac639d" target='_blank'>
              Optimized lentiviral backbone induces robust and diverse T cell immunity against neoantigens to counteract tumor heterogeneity
              </a>
            </td>
          <td>
            B. Vesin, Ingrid Fert, A. Noirat, P. Authié, Sylvain Ciret, Catherine Blanc, Yakov Vitrenko, Pierre Charneau, L. Majlessi, François Anna
          </td>
          <td>2025-07-04</td>
          <td>NPJ Vaccines</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cell heterogeneity is a universal feature of life. Although biological processes affected by cell-to-cell variation are manifold, from developmental plasticity to tumour heterogeneity and differential drug responses, the sources of cell heterogeneity remain largely unclear1,2. Mutational and epigenetic signatures from cancer (epi)genomics are powerful for deducing processes that shaped cancer genome evolution3–5. However, retrospective analyses face difficulties in resolving how cellular heterogeneity emerges and is propagated to subsequent cell generations. Here, we used multigenerational single-cell tracking based on endogenously labelled proteins and custom-designed computational tools to elucidate how oncogenic perturbations induce sister cell asymmetry and phenotypic heterogeneity. Dual CRISPR-based genome editing enabled simultaneous tracking of DNA replication patterns and heritable endogenous DNA lesions. Cell lineage trees of up to four generations were tracked in asynchronously growing cells, and time-resolved lineage analyses were combined with end-point measurements of cell cycle and DNA damage markers through iterative staining. Besides revealing replication and repair dynamics, damage inheritance and emergence of sister cell heterogeneity across multiple cell generations, through combination with single-cell transcriptomics, we delineate how common oncogenic events trigger multiple routes towards polyploidization with distinct outcomes for genome integrity. Our study provides a framework to dissect phenotypic plasticity at the single-cell level and sheds light onto cellular processes that may resemble early events during cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8a5871308fcb63b1e32d25ce00cce571108d0" target='_blank'>
              Multigenerational cell tracking of DNA replication and heritable DNA damage
              </a>
            </td>
          <td>
            Andreas Panagopoulos, Merula Stout, Sinan Kılıç, Peter Leary, Julia Vornberger, Virginia Pasti, A. Galarreta, Aleksandra Lezaja, Kyra Kirschenbühler, Ralph Imhof, H. Rehrauer, Urs Ziegler, Matthias Altmeyer
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="
 e16452


 Background:
 Liver metastasis is a leading cause of death amongst localized pancreatic ductal adenocarcinoma (PDAC) patients following surgical resection. Yet, there are no adjuvant therapies currently available targeting liver colonization in PDAC. We are hindered in our ability to develop such therapies by an incomplete understanding of PDAC liver metastasis' molecular mechanisms.
 Methods:
 To address this challenge, we labeled primary mouse PDAC subclones with DNA barcodes to characterize their pre-metastatic state using ATAC-seq and RNA-seq and determine their relative
 in vivo
 liver metastasis potential in competition assays in immunocompetent hosts.
 Results:
 We identified a gene signature separating metastasis-high and metastasis-low subclones orthogonal to the normal-to-PDAC and classical-to-basal axes. The metastasis-high subclones feature activation of inflammation-related genes and high NF-κB and Zeb/Snail family activity and the metastasis-low subclones feature activation of neuroendocrine, motility, and Wnt pathway genes and high CDX2 and HOXA13 activity. In a functional screen, we validated novel mediators of PDAC liver metastasis related to inflammation, including the NF-κB targets
 Fos
 and
 Il23a
 , and beyond inflammation including
 Myo1b
 and
 Tmem40
 . We scored human PDAC tumors for our signature of metastatic potential from mouse and found that metastases have higher scores than primary tumors. Moreover, primary tumors with higher scores are associated with worse prognosis. We also found that our metastatic potential signature is enriched in other human carcinomas, suggesting that it is conserved across epithelial malignancies.
 Conclusions:
 This study establishes a model of clonal evolution during PDAC liver metastasis under immune selective pressure, revealing novel functional regulators amenable to therapeutic intervention in the adjuvant setting. Furthermore, it provides a method for scoring human carcinomas based on metastatic potential that may have prognostic and predictive value across multiple cancer subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d54beefd4d0e886101bb6db8b7a938594843dac" target='_blank'>
              Effect of clonal competition model of PDAC liver metastasis on a pan-carcinoma gene signature of metastatic potential.
              </a>
            </td>
          <td>
            Jesse S Handler, Zijie Li, Rachel K Dveirin, Jessica Lin, Weixiang Fang, James Forsmo, Hani Goodarzi, E. Fertig, Reza Kalhor
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy; however, no validated treatments are currently available. Clinically, the tumor position, head and uncinate process (HU), or body and tail (BT) of the pancreas are vital for surgical strategies; however, fundamental research has seldom revealed the heterogeneous tumor microenvironment (TME) among different types of PDAC. Here, we applied multicohort single-cell and spatial RNA-seq methods together with patient-derived organoid models to reveal the TME heterogeneity between HU and BT PDAC. Osteopontin, encoded by SPP1, is secreted by vessel endothelial cells in BT PDAC and is associated with increased tumor burden. The number of tumor cells marked by CRABP2 was lower in BT PDAC and was identified as a prognostic marker of overall survival, as well as CD8+ T-cell infiltration. The expression of CRABP2 was also validated to be downregulated in BT PDAC in patient-derived organoid models. Overall, we profiled the heterogeneous PDAC TME between HU and BT PDAC, which could provide novel insight into the relationships between clinical characteristics and TME molecular research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077490657a2feb3d7ea606c6fed7a402a475e6c0" target='_blank'>
              SPP1+ Venous Endothelial Cells and CRABP2+ Tumor Cells Contribute to Favorable Body and Tail Pancreatic Ductal Adenocarcinoma Tumor Microenvironment.
              </a>
            </td>
          <td>
            Yueze Liu, Yifan Fu, Tao Liu, Jun Wang, Zeyu Zhang, Yanan Shi, Zhe Cao, Gang Yang, Hao Chen, W. Luo, Jinxin Tao, Yuanyang Wang, Guihu Weng, Menggang Zhang, Liyuan Ye, Jianchun Xiao, J. Qiu, Taiping Zhang, Hua Huang
          </td>
          <td>2025-06-11</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Alterations in glycolysis play a crucial role in cancer cells, influencing tumor aggressiveness and therapeutic effect, particularly in pancreatic adenocarcinoma (PAAD). However, the specific glycolysis-related genes involved in PAAD progression remain poorly understood. This study established glycolysis-related molecular subtypes with distinct survival outcomes using TCGA datasets. The favorable prognosis subtype exhibited enhanced immune infiltration and an activated tumor microenvironment. A glycolysis prognostic model effectively predicted PAAD survival, correlating with global glycolytic pathways, and AUCell evaluated neutrophil communication networks of models. Functional validation demonstrated that ENO1/PGM2L1 co-expression promoted tumor proliferation, migration, invasion, and glycolytic flux in vitro, while accelerating xenograft growth in vivo. Conversely, their knockdown suppressed malignancy. Our study demonstrated that the glycolytic prognostic risk model serves as a reliable tool for prognosis and prediction of PAAD progression. ENO1 and PGM2L1 emerge as key risk factors promoting the malignant progression of PAAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbf7a9ccf491c81be3110e0f15ea439fc392789" target='_blank'>
              Integrative Single-Cell and Bulk RNA Sequencing Identifies a Glycolysis-Related Prognostic Signature for Predicting Prognosis in Pancreatic Cancer
              </a>
            </td>
          <td>
            Nan Wu, Chong Zhou, Xu Yan, Ziang Liu, Ruohan Jiang, Yuzhou Luo, Ping Jiang, Yu Mu, Shan Xiao, Xien Huang, Yun-Yun Zhou, Donglin Sun, Yan Jin
          </td>
          <td>2025-05-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Locally advanced esophageal adenocarcinoma remains difficult to treat and the ecological and evolutionary dynamics responsible for resistance and recurrence are incompletely understood. Here, we performed longitudinal multiomic analysis of patients with esophageal adenocarcinoma in the MEMORI trial. Multi-region multi-timepoint whole-exome and paired transcriptome sequencing was performed on 27 patients before, during and after neoadjuvant treatment. We found major transcriptomic changes during treatment with upregulation of immune, stromal and oncogenic pathways. Genetic data revealed that clonal sweeps through treatment were rare. Imaging mass cytometry and T cell receptor sequencing revealed remodeling of the tumor microenvironment during treatment. The presence of genetic immune escape, a less-cytotoxic T cell phenotype and a lack of clonal T cell expansions were linked to poor treatment response. In summary, there were widespread transcriptional and environmental changes through treatment, with limited clonal replacement, suggestive of phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286e0cfb0095dd021b03614405167d3960808941" target='_blank'>
              Evolutionary and immune microenvironment dynamics during neoadjuvant treatment of esophageal adenocarcinoma
              </a>
            </td>
          <td>
            M. Barroux, J. Househam, E. Lakatos, T. Ronel, Ann-Marie Baker, Henrike Salié, M. Mossner, Kane Smith, C. Kimberley, S. Nowinski, Alison Berner, V. Gunasri, Martin Borgmann, S. Liffers, Marnix Jansen, G. Caravagna, K. Steiger, J. Slotta-Huspenina, Wilko Weichert, Luis Zapata, Eleftheria Giota, Sylvie Lorenzen, M. Alberstmeier, Benjamin M Chain, H. Friess, B. Bengsch, Roland M. Schmid, J. Siveke, M. Quante, Trevor A. Graham
          </td>
          <td>2025-05-14</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common and deadly malignancies worldwide, with a particularly low 5-year survival rate in advanced patients. Immune cells in the tumor microenvironment, especially mast cells, play crucial roles in tumor initiation and progression. However, the dual role of mast cells in CRC remains poorly understood. Methods In this study, we used single-cell RNA sequencing (scRNA-seq), bulk RNA sequencing, and bioinformatics analyses to explore the heterogeneity of mast cell subpopulations in the CRC tumor microenvironment and their relationship with prognosis. We analyzed gene expression signatures associated with mast cell subpopulations derived from single-cell data of 40 CRC tumor samples and combined bulk RNA-seq data from HMU, GEO, and TCGA cohorts for prognostic prediction. Non-negative matrix factorization was used for clustering of mast cell subpopulations, followed by analysis of their specific gene markers, transcription factor activity, and biological pathways. Survival analysis and ROC curves were performed to assess their prognostic significance. Results Mast cells in the CRC tumor microenvironment were classified into three distinct subpopulations, each with unique gene markers and functional pathways. Mast cell subpopulations 1 and 3 were highly associated with pro-tumor pathways, while mast cell subpopulation 2 primarily exhibited anti-tumor immune regulatory characteristics. High expression of mast cell subpopulations 1 and 3 was associated with poor survival prognosis, while high expression of subpopulation 2 was linked to a better survival outcome. Key marker genes such as DNAJB1, SEMA7A, and XCR1 were identified as potential prognostic factors, with high expression of DNAJB1 and SEMA7A being significantly associated with poor prognosis, while high expression of XCR1 was linked to a favorable prognosis. Conclusion This study reveals the functional heterogeneity of mast cell subpopulations in the CRC tumor microenvironment and their differential roles in tumor progression. Identification of mast cell subpopulation-specific marker genes provides new molecular targets for clinical diagnosis, prognostic prediction, and personalized immunotherapy in CRC. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04119-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f36d01ec68db5890c180d0142a0a3d3d4bc0a598" target='_blank'>
              Diversity of mast cell subpopulations in the tumor microenvironment of colorectal cancer and their prognostic implications
              </a>
            </td>
          <td>
            Tian-yu Qiao, Chao Ding, Songtao Yu, Wenyang Li, Yonghou Zhao, Guiyu Wang
          </td>
          <td>2025-06-30</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, Muhammad T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Health outcomes for people with HIV (PWH) have improved significantly with combination antiretroviral therapy (ART), yet the risk of lung cancer remains elevated. While a single case cannot establish causality, we describe here an investigation of a 74-year-old male PWH with de novo high-grade neuroendocrine small cell lung carcinoma. To investigate the potential contribution of HIV to cancer development, we performed HIV integration site sequencing on blood, tumor, and non-tumor tissue samples from the patient. We analyzed integration site distribution, clonal expansion, and associated gene disruption. Phosphatase and Tensin Homolog (PTEN) expression was evaluated using immunofluorescence and microscopy. A total of 174 unique HIV integration sites were identified, with 29.9% (52/174) located in clonally expanded cells. The most frequent integration site in clonally expanded cells was within the PTEN gene, representing 4.2% to 16.7% of all HIV-infected cells across samples. PTEN expression was markedly reduced in tumor regions relative to non-tumor tissue. Areas positive for HIV p24 antigen showed minimal PTEN expression. These findings suggest that HIV integration into the PTEN gene, coupled with clonal expansion of HIV-infected cells, may impair anti-tumor immune responses and promote cancer progression in PWH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7464c79465c61b74241bf20c790d2e5443d3adb7" target='_blank'>
              HIV Integration into the PTEN Gene and Its Tumor Microenvironment Implications for Lung Cancer
              </a>
            </td>
          <td>
            , Elizabeth F. Rowland, Sara Gianella, Sandip Pravin Patel, S. Solso, Cheryl Dullano, Robert Deiss, Daria Wells, C. Ignacio, G. Caballero, M. Porrachia, Collin Kieffer, A. Chaillon
          </td>
          <td>2025-07-04</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a complex and dynamic ecosystem crucial for cancer development and progression. Within this intricate milieu, T‐cells constitute a heterogeneous population and serve as a cornerstone of antitumor immunity. Notably, T‐cells can rapidly transition across a wide spectrum of phenotypic and functional states within the disrupted TME. Despite the crucial role of T‐cells in cancer immunity, a comprehensive understanding of their plasticity within the TME remains limited. In this review, we delve into the functional plasticity and spatial distribution of T‐cells in response to diverse microenvironmental conditions. Additionally, we review the plasticity of T‐cell functional states during conventional therapies, highlighting their potential to enhance or limit therapeutic outcomes. Finally, we propose innovative therapeutic approaches that leverage T‐cell plasticity to enhance clinical efficacy by regulating the immune response within the TME. By providing insights into the dynamics of T‐cell behavior, this review highlights the promising potential of targeting T‐cell plasticity as an immuno‐sensitizer to refine therapeutic strategies and overcome current challenges in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ba464623e6386604e59c25089e4ff9051f3936" target='_blank'>
              Unlocking T‐Cell Plasticity in the Tumor Microenvironment: Implications for Cancer Progression and Therapeutic Strategies
              </a>
            </td>
          <td>
            Xiao-Hong Ding, Xue‐Pei Li, Fenfang Chen, Han Wang, Yizhou Jiang
          </td>
          <td>2025-05-24</td>
          <td>MedComm – Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Globally, endometrial cancer continues to impact a significant number of women. Immunotherapy provides those suffering from advanced or relapsed disease hope, but an important barrier is still the absence of trustworthy predictive biomarkers. To tackle this challenge, single-cell sequencing and spatial transcriptomics (ST) are increasingly applied. In cervical cancers of the no specific molecular profile (NSMP) subtype accompanied by p53 mutations. In many cases, the tumor microenvironment (TME) in endometrial cancer exhibits strong immunosuppression or poor immune cell infiltration, often leading to worse clinical outcomes. Single-cell sequencing reveals cellular heterogeneity and helps identify potential therapeutic targets and predict treatment responses. Conversely, ST assists in determining biomarkers that influence the effectiveness of immunotherapy by capturing the spatial organization of tumors. When combined, these technologies allow for integrated multi-omics analysis that aids in the development of immunotherapies, prognostication, and diagnosis. But there are still moral and legal issues. Clinicians may be able to improve outcomes for patients who don’t respond well to current immunotherapies by utilizing these combined approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f087dd53dee17003d5211dac4cb4360cce4847" target='_blank'>
              Single-cell technologies and spatial transcriptomics: decoding immune low - response states in endometrial cancer
              </a>
            </td>
          <td>
            Yumeng Li, Hua Qiu, Zhenzhen Zhao, Fanghua Qi, Pingping Cai
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease, with tumors capable of adapting to shifting conditions, making the development of effective personalized therapies particularly challenging. Patient-derived models, such as patient-derived organoids (PDOs) and circulating tumor cell (CTC) cultures, have emerged as powerful tools for investigating intra- and inter-tumor heterogeneity. These models largely retain the genetic, phenotypic, and microenvironmental features of the original tumors, providing valuable insights into disease progression, drug response, and resistance mechanisms. Furthermore, by enabling tumors’ spatiotemporal molecular profiling, PDOs and CTCs offer a dynamic approach to assess treatment efficacy over time. However, to fully capture the complexity of breast cancer heterogeneity, it is required to develop models from multiple tumor and blood samples collected throughout the course of treatment. This review explores the potential of integrating PDOs and CTC models to better understand intra-tumor heterogeneity while addressing key challenges in developing patient-derived models that accurately recapitulate patients’ tumors to advance personalized care. The integration of PDOs and CTCs could represent a paradigm shift in the personalized management of metastatic breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a82abf6024e151676efff540be9bde6be013c6" target='_blank'>
              Deciphering Breast Tumor Heterogeneity Through Patient-Derived Organoids and Circulating Tumor Cells
              </a>
            </td>
          <td>
            Benedetta Policastro, Nikoline Nissen, Carla L. Alves
          </td>
          <td>2025-06-25</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the most abundant non-cancerous cell type in glioblastoma (GBM) and heavily influence GBM biology, contributing to tumor progression, therapeutic resistance, immune evasion, and neovascularization. Current in vitro models that utilize IL-4/IL-13 stimulation fail to capture the transcriptional and functional heterogeneity of TAMs observed in vivo. In this study, we utilize a serum-free indirect co-culture model with patient-derived xenolines to polarize primary human macrophages and characterize their molecular and functional phenotypes. We demonstrate that xenoline-polarized macrophages diverge from classical M1/M2 states and instead adopt transcriptional signatures reflective of TAM subsets identified from patients. Notably, macrophages polarized with the radiation-therapy selected xenoline, JX14P-RT, exhibited gene expression patterns enriched for interferon response and hypoxia, mirroring recurrent GBM samples. In contrast, JX14P TAMs showed enrichment in phagocytic gene sets. Functional validation of these phenotypes revealed discrepancies between the transcriptionally predicted and observed phenotypes, emphasizing the importance of integrating phenotypic validation in sequencing studies. Altogether, our findings establish xenoline-polarized macrophages as a more physiologically relevant alternative to traditional models, offering a useful model for studying tumor-immune interaction in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e59e91c3e7e604dd7d80dd0333a4e13f653a5b4e" target='_blank'>
              Xenoline-polarized macrophages as a physiologically relevant in vitro model of tumor-associated macrophages in glioblastoma
              </a>
            </td>
          <td>
            Hasan Alrefai, Benjamin Lin, Amr Elkohly, Manoj Kumar, Taylor L. Schanel, Kevin J. Lee, Patricia H. Hicks, Joshua C. Anderson, Gao Guo, Eun-Young Erin Ahn, C. R. Miller, Christopher D. Willey
          </td>
          <td>2025-05-09</td>
          <td>Research Square</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME) that plays a pivotal role in tumor initiation, progression, and immune evasion. Recent advancements have highlighted the intricate interplay between immune cell infiltration patterns, immune checkpoint dysregulation, and metabolic reprogramming in driving HNSCC immune escape. Despite these insights, significant challenges remain, including the incomplete understanding of specific immune evasion pathways and the lack of personalized therapeutic strategies. To address these gaps, this review introduces a novel “Trinity” regulatory network of immune evasion in HNSCC, encompassing: (1) metabolic reprogramming-mediated immune checkpoint modulation, (2) stromal cell-driven immune dysfunction, and (3) epigenetic remodeling fostering immune tolerance. This framework provides a theoretical foundation for the development of multi-targeted combination therapies and offers innovative strategies to overcome immune evasion. Additionally, this review systematically synthesizes the current understanding of the relationship between the HNSCC microenvironment and immune escape, with a focus on emerging immunotherapeutic approaches such as PD-1/PD-L1 inhibitors and CAR-T cell therapy. Leveraging cutting-edge single-cell sequencing and spatial transcriptomics, we elucidate the spatiotemporal heterogeneity of the HNSCC immune landscape and propose a new paradigm of “lineage plasticity-driven immune adaptation.” These insights not only advance our understanding of HNSCC biology but also pave the way for the development of precision immunotherapies aimed at improving patient survival and quality of life. By integrating multidisciplinary perspectives, this work underscores the importance of targeting the TME to achieve durable clinical responses and overcome immunotherapy resistance in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87d51f5bbe548cb50db3cd5b470a30e09259eacc" target='_blank'>
              Unraveling the immune evasion mechanisms in the tumor microenvironment of head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xuri Zhao, Yaping Zhu, Youya He, Weiyan Gu, Qi Zhou, Bei Jin, Shenguo Chen, Haisheng Lin
          </td>
          <td>2025-05-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastatic cancer, the primary cause of cancer mortality, frequently exhibits heightened dependence on certain transcription factors (TFs), which serve as master regulators of oncogenic signaling yet are often untargetable by small molecules. Selective Cell in ORgan Targeting (SCORT) nanoparticles are developed for precise CRISPR/Cas13d mRNA and gRNA delivery to metastatic cancer cells in vivo, aiming to knock down the undruggable oncogenic TF HoxB13. In prostate cancer liver metastasis models driven by HoxB13, repeated systemic SCORT‐Cas13d‐gHoxB13 treatment significantly decreases HoxB13 expression, reduces metastasis, and extends mouse survival. Prolonged treatment shows no significant impact on major organ function, histology or immune markers. Mechanistically, SCORT‐Cas13d‐gHoxB13 treatment suppresses metastatic tumor proliferation and angiogenesis while promoting apoptosis by regulating multiple gene pathways. Unexpectedly, it inhibits the non‐canonical, EMT‐independent oncogenic function of Snail. These findings suggest that SCORT‐Cas13d‐gHoxB13 can effectively and safely target the undruggable HoxB13 in metastatic prostate cancer, positioning CRISPR/Cas13d as a potential treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1f6eea06defd172595a1854baf946894bc6d693" target='_blank'>
              SCORT–Cas13d Nanotherapy Precisely Targets the ‘Undruggable’ Transcription Factor HoxB13 in Metastatic Prostate Cancer In Vivo
              </a>
            </td>
          <td>
            Zhifen Cui, Furong Huang, Kun Fang, Jingyue Yan, Yuebao Zhang, Diana D. Kang, Yufan Zhou, Yue Zhao, Jeffrey Everitt, W. Hankey, Andrew J Armstrong, Jiaoti Huang, Hongyan Wang, Victor X Jin, Yizhou Dong, Qianben Wang
          </td>
          <td>2025-05-11</td>
          <td>Advanced Science</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c9003a002bf9a6c00cceca68fcf223699d2693" target='_blank'>
              Single-cell multi-omic integration analysis prioritizes druggable genes and reveals cell-type-specific causal effects in glioblastomagenesis
              </a>
            </td>
          <td>
            Y.-F. Huang, K.-L. Huang
          </td>
          <td>2025-07-01</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor diseases are characterized by high interindividual and intratumoral heterogeneity (ITH). The development and progression of neoplasms outline complex networks of extracellular and cellular signals that have yet to be fully elucidated. This narrative review provides a comprehensive overview of the literature related to the cellular and molecular mechanisms underlying the heterogeneity of the tumor mass. Furthermore, it examines the possible role of the tumor microenvironment in the development and support of the neoplasm, in order to highlight its potential in the construction of a diagnostic–therapeutic approach to precision medicine. Many authors underline the importance of the tumor microenvironment (TME) as it actively takes part in the growth of the neoplastic mass and in the formation of metastases and in the acquisition of resistance to anticancer drugs. In specific body districts, the ideal conditions occur for the TME establishment, particularly the inflammatory state, the recruitment of cell types, the release of specific cytokines and growth factors, hypoxic conditions. These components actively intervene by enabling tumor progression and construction of physical barriers shaped by the extracellular matrix that contribute to forming peripheral tolerance by intervention of myeloid precursors and the polarization of M2 macrophages. In recent years, ITH and the TME have assumed an important position in cancer research and pharmacology as they enable understanding the dense network of communication existing between the neoplasm and the surrounding environment, and to monitor and deepen the effects of drugs with a view to develop increasingly precise and effective therapies. In the last decade, knowledge of TME has been exploited to produce targeted molecular agents (inhibitory small molecules, monoclonal antibodies, gene therapy). Nonetheless, the bibliography shows the need to study ITH through new prognostic and predictive biomarkers (e.g., ctDNA and CTCs) and to increase its basic biology knowledge. Precision medicine is a new opportunity in the treatment of oncological diseases that is transforming the development of new drug approaches and their clinical use. Biology and biotechnologies are providing the bases for this revolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9eef0c89f58575a3286340faed6565d85cc9b7" target='_blank'>
              Microenvironment and Tumor Heterogeneity as Pharmacological Targets in Precision Oncology
              </a>
            </td>
          <td>
            S. Tonello, Roberta Rolla, Paolo Amedeo Tillio, PP. Sainaghi, Donato Colangelo
          </td>
          <td>2025-06-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Breast cancer exhibits heterogeneity at various levels, from subtype differences between patients (inter-patient heterogeneity) to diverse malignant cell compositions, variability in hormone receptor (HR) expression, and cellular makeup within single breast cancer samples (intra-tumour heterogeneity). Despite the availability of numerous effective therapies, many patients still experience incomplete treatment responses and subsequent relapse. These adverse outcomes are often due to the critical but frequently overlooked factor of cellular heterogeneity.
 To gain deeper insights into intra-tumour heterogeneity, we applied single-cell technologies to a cohort of 250 primary, untreated breast cancers. We optimized tissue cryopreservation methods, eliminating the need for fresh sample processing, and developed multiplex tissue profiling techniques. These cost-efficient processes improve handling of small tissue sizes, such as biopsies, and reduce batch effects. To ensure accurate and reliable data processing, we created a scalable computational workflow that includes benchmarked methods for sample SNP-demultiplexing, doublet detection, high-resolution cell annotation, and cellular integration. Additionally, we extended our existing methods to study the cellular heterogeneity of breast cancers.
 Our method, scSubtyper, examines the phenotypic differences between malignant cells within tumours by comparing each single cell to features associated with different molecular subtypes and assigning each cell to one of these subtypes. Our previous study and preliminary results from this project revealed that over 90% of samples exhibit a mix of malignant cells of different subtypes, and 50% of samples contain cells with characteristics of all subtypes, demonstrating that cellular heterogeneity exists not only between malignant cells but also within malignant cells of a tumour. Our second approach, known as ecotyping, assesses patterns of cell type frequencies across samples and groups them based on the similarity of these co-occurrences. Preliminary results have revealed the existence of five ecotypes that lack significant associations with clinical subtypes. Application of the same approach exclusively within the HR-positive samples identified four ecotypes characterized by distinct abundances of immune and stromal cells. This analysis revealed that ecotypes are not simple surrogates for clinical and molecular subtypes, but their presence could influence different responses to treatment.
 Together, our high-throughput tissue processing and computational approaches to study intra-tumour heterogeneity are now being applied to our large, well-annotated clinical cohort. Supported by preliminary results, we hypothesize that this study will play a vital role in optimizing breast cancer patient stratification to improve treatment management and outcomes.
 Citation Format: Beata Kiedik, Daniel L. Roden, Kate Harvey, Ghamdan Al-Eryani, Sunny Z. Wu, Mun N. Hui, Sandra O’Toole, Elgene Lim, Charles M. Perou, Alexander Swarbrick. Exploring cellular heterogeneity of localised breast cancers [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-09-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/355a116d2a0ae027b0d9081aa9023a8465a9971c" target='_blank'>
              Abstract P4-09-19: Exploring cellular heterogeneity of localised breast cancers
              </a>
            </td>
          <td>
            Beata Kiedik, D. Roden, K. Harvey, G. Al-Eryani, Sunny Z. Wu, M. Hui, Sandra O’Toole, Elgene Lim, C. Perou, Alex Swarbrick
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Simple Summary Single-label immunohistochemistry (sIHC) is used in infectious disease studies to detect etiological agents, and to quantify and spatially localize antigens in lesions. While sIHC is limited to staining one antigen per tissue section, multiplex immunohistochemistry (mIHC) offers significantly more detailed information and is more cost- and time-efficient than repeated sIHC. Given that swine lymph node pathology involves variations in T and B lymphocytes and macrophages, this study aims to apply mIHC to visualize simultaneously and on the same slide their spatial distribution and variation across the three main regions of the swine lymph node: follicles, interfollicular areas and medulla-like tissue. We demonstrated the use of bright-field mIHC for in situ analysis of spatial cellular distribution in swine lymph nodes during PCV2-SD and PRRSV lymphadenopathy, revealing distinct lymphocyte distributions compared to reactive lymphoid hyperplasia. The study highlights the potential of mIHC as a valuable tool for understanding and studying disease dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c47de3fe690520b75f0437e0eae36becf35b80" target='_blank'>
              Bright-Field Multiplex Immunohistochemistry in Swine PCV2 and PRRSV Lymphadenopathies
              </a>
            </td>
          <td>
            G. D’Annunzio, L. V. Muscatello, Chiara Tugnoli, Stefano Pesaro, Andrea Luppi, Michelangelo Fiorentino, T. Franceschini, Alessia Grillini, G. Rugna, Giuseppe Sarli, L. Mandrioli
          </td>
          <td>2025-06-01</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The intratumoral microbiota, as an important component of the tumor microenvironment, is increasingly recognized as a key factor in regulating responses to cancer immunotherapy. Recent studies have revealed that the intratumoral microbiota is not uniformly distributed but instead exhibits significant spatial heterogeneity, with its distribution patterns influenced by factors such as tumor anatomy, local immune status, and therapeutic interventions. This spatial heterogeneity not only alters the interactions between microbes and the host immune system but may also reshape the immunogenic and immunosuppressive landscapes of tumors. The enrichment or depletion of microbiota in different tumor regions can influence immune cell infiltration patterns, metabolic pathway activities, and immune checkpoint molecule expression, thereby driving the development of resistance to immunotherapy. Moreover, certain bacterial metabolites form concentration gradients between the tumor core and margins, thereby regulating immune cell function. Therefore, understanding and manipulating the spatial distribution of intratumoral microbiota, particularly in resistant patients, holds promise for developing new strategies to overcome immunotherapy resistance. In the future, precise modulation strategies targeting microbial spatial heterogeneity, such as engineered bacterial vectors, probiotic combinations, and phage therapy, may open new avenues for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58f37c6ba2761018518110c9bed4886530b224c2" target='_blank'>
              Spatial Heterogeneity of Intratumoral Microbiota: A New Frontier in Cancer Immunotherapy Resistance
              </a>
            </td>
          <td>
            Qiwen Tan, Xiongjing Cao, Falong Zou, Hanwenchen Wang, Lijuan Xiong, Shenghe Deng
          </td>
          <td>2025-05-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA), a neoplasm arising from biliary epithelial cells, is particularly widespread in Southeast Asia, with northeastern Thailand exhibiting the greatest prevalence attributed to Opisthorchis viverrini infection. This malignancy exhibits considerable molecular heterogeneity, leading to therapeutic resistance and recurrence. Comprehending its molecular mechanisms is essential for enhancing diagnostic and treatment approaches. Our research utilized multi-region LC-MS/MS proteomic analysis to investigate intratumor heterogeneity (ITH) in CCA. We examined 52 tumor areas and 13 neighboring tissues from 13 patients, concentrating on protein profiling, pathway analysis, differential protein expression, and the identification of shared and unique protein signatures. The findings indicated considerable inter-patient proteome variability, characterized by markedly distinct protein expressions among individuals, aligning with prior cancer research. Intra-tumor heterogeneity was apparent, with merely 18 proteins common to all tumor areas and patients, underscoring the intricacy of CCA. Significantly, the common proteins were associated with metabolic reprogramming and oxidative stress pathways, indicating possible indicators and therapeutic targets. This work highlights the significant proteome variability in CCA at both intra-tumor and inter-patient levels, underscoring the necessity for customized therapeutic approaches to tackle the disease’s complexity and improve treatment outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-02713-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afe6a3a1d14e85f2bd0571da7c1e456fa657e91b" target='_blank'>
              Proteomic profiling reveals common and region-specific protein signatures underlying tumor heterogeneity in cholangiocarcinoma
              </a>
            </td>
          <td>
            Sirinya Sitthirak, S. Roytrakul, Arporn Wangwiwatsin, N. Namwat, Poramate Klanrit, Hasaya Dokduang, P. Sa-Ngiamwibool, Attapol Titapan, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2025-05-18</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Cellular heterogeneity within cancer tissues determines cancer progression and treatment response. Single‐cell RNA sequencing (scRNA‐seq) has provided a powerful approach for investigating the cellular heterogeneity of both cancer cells and stroma cells in the tumor microenvironment. However, the common practice to characterize cell identity based on the similarity of their gene expression profiles may not really indicate distinct cellular populations with unique roles. Generally, the cell identity and function are orchestrated by the expression of given specific genes tightly regulated by transcription factors (TFs). Therefore, deciphering TF activity is essential for gaining a better understanding of the uniqueness and functionality of each cell type. Herein, metaTF, a computational framework designed to infer TF activity in scRNA‐seq data, is introduced and existing methods are outperformed for estimating TF activity. It presents the improved effectiveness in characterizing cell identity during mouse hematopoietic stem cell development. Furthermore, metaTF provides a superior characterization of the functional identity of breast cancer epithelial cells, and identifies a novel subset of neural‐regulated T cells within the tumor immune microenvironment, which potentially activates BCL6 in response to neural‐related signals. Overall, metaTF enables robust TF activity analysis from scRNA‐seq data, significantly enhancing the characterization of cell identity and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/310aebb046f221f9691b3d34683db0bbb309e243" target='_blank'>
              Accurate Transcription Factor Activity Inference to Decipher Cell Identity from Single‐Cell Transcriptomic Data with MetaTF
              </a>
            </td>
          <td>
            Yongfei Hu, Yuanyuan Zhu, Guangjue Tang, Ming Shan, Puwen Tan, Ying Yi, Xiyuan Zhang, Man Liu, Xinyu Li, Le Wu, Jia Chen, Hailong Zheng, Yan Huang, Zhuan Li, Xiaobo Li, Dong Wang
          </td>
          <td>2025-05-21</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Drug sensitivity is a major determinant influencing the efficiency of cancer treatment, and its evaluation faces critical challenges due to the cancer cell heterogeneity. Herein, a droplet-driven compartmentalization array (SCellEV chip) is presented to uncover single-cell trastuzumab sensitivity. Leveraging the differential flow resistance principle, the chip enables the flexible generation of single droplets orderly situated at designated spots. The single droplet includes parallel hydrodynamic microtraps empowers efficient compartments for single cell capture, and culture, and employs aptCD63-functionalized microbeads to immune-affinity capture single cell extracellular vesicles (EVs). The pairing capture of single cells and their EVs using single beads allows simultaneous identification of EVs and their original cells. The protein profiles (HER2 and CD63) and anti-cancer drug challenge revealed the cellular/EVs secretion heterogeneity at the single cell level and cell susceptibility to trastuzumab, further confirming that this chip achieves discriminating and predicting single breast cancer cells with different trastuzumab sensitivity using single cell/EVs profiling strategy. Furthermore, this analysis process retains the cell viability of measured cells to promote downstream detection integration. Overall, the SCellEV chip offers new avenues for single-cell analysis and exhibits great potential for cancer diagnostic and targeted treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcd22f45999fb009740a6f20d00e3273f8e6d22" target='_blank'>
              A Microfluidic Droplet Array Promotes Trastuzumab Sensitivity Exploration of Single Breast Cancer Cells.
              </a>
            </td>
          <td>
            Junjie Bai, Xuan Zhang, Xing Wei, Hai-Yan Li, Ya-Nan Zhao, Ze-Jun Wang, Ting Yang, Jian-Hua Wang, Mingli Chen
          </td>
          <td>2025-06-16</td>
          <td>Small</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="This study presents a pioneering exploration into the role of aggrephagy-related genes (ARGs) in glioblastoma (GB), a kind of malignant tumor which is highly invasive and resistant to a series of therapy. Utilizing single-cell sequencing to dissect their influence on the tumor microenvironment (TME) and tumorigenesis. By applying non-negative matrix factorization for dimensionality reduction and clustering of single-cell data, distinct cellular subtypes within the TME influenced by ARGs were identified, uncovering their functions and interactions. The investigation extends to validating the prognostic significance of ARGs and their potential in predicting immunotherapy outcomes. Molecular docking analysis of key ARGs further highlights TUBA1C and UBB as promising therapeutic targets, offering novel insights into GB’s complex biology and suggesting a targeted approach for therapy, which is characterized by some crucial pathways in our analysis, including PI3k-akt and TGF-beta pathways. This comprehensive single-cell level examination not only advances our understanding of aggrephagy’s role in GB but also proposes new avenues for prognosis and treatment strategies, emphasizing the critical impact of ARGs on the TME and GB progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d160ce3578d46173e199ce3a22ee88e96cde97" target='_blank'>
              Integrating multi-omics data to identify the role of Aggrephagy-related genes in tumor microenvironment and key tumorigenesis factors of GB from the perspective of single-cell sequencing
              </a>
            </td>
          <td>
            Zipei Chen, Shengke Zhang, Chenglu Jiang, Lai Jiang, Haiqing Chen, Jinbang Huang, Jie Liu, Guanhu Yang, Xiufang Luo, Hao Chi, Jiangping Fu
          </td>
          <td>2025-05-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer remains a significant global health burden due to its high morbidity and mortality. Oncogene-targeted therapy and immunotherapy have markedly improved the 5-year survival rate in the patients with advanced or metastatic tumors compared to outcomes in the era of chemotherapy/radiation. Nevertheless, the majority of patients remain incurable. Initial therapies eliminate the bulk of tumor cells, yet residual populations termed drug-tolerant persister cells (DTPs) survive, regenerate tumor and even drive distant metastases. Notably, DTPs frequently render tumor cross-resistance, a detrimental phenomenon observed in the patients with suboptimal responses to subsequent therapies. Analogous to species evolution, DTPs emerge as adaptative products at the cellular level, instigated by integrated intracellular stress responses to therapeutic pressures. These cells exhibit profound heterogeneity and adaptability shaped by the intricate feedforward loops among tumor cells, surrounding microenvironments and host ecology, which vary across tumor types and therapeutic regimens. In this review, we revisit the concept of DTPs, with a focus on their generation process upon targeted therapy or immunotherapy. We dissect the critical phenotypes and molecule mechanisms underlying DTPs to therapy from multiple aspects, including intracellular events, intercellular crosstalk and the distant ecologic pre-metastatic niches. We further spotlight therapeutic strategies to target DTP vulnerabilities, including synthetic lethality approaches, adaptive dosing regimens informed by mathematical modeling, and immune-mediated eradication. Additionally, we highlight synergistic interventions such as lifestyle modifications (e.g., exercise, stress reduction) to suppress pro-tumorigenic inflammation. By integrating mechanistic insights with translational perspectives, this work bridges the gap between DTP biology and clinical strategies, aiming for optimal efficacy and preventing relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a66d9e3af1fc508d0ed83941eccffc587ef2ea" target='_blank'>
              Drug-induced tolerant persisters in tumor: mechanism, vulnerability and perspective implication for clinical treatment
              </a>
            </td>
          <td>
            Shujie Liu, Anfeng Jiang, Faqing Tang, Minghao Duan, Bin Li
          </td>
          <td>2025-05-24</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Background Melanoma is a highly heterogeneous malignancy with diverse molecular and clinical behaviors. A precise molecular classification is critical for improving prognostic assessment and guiding personalized therapy. Methods We performed an integrative multi-omics analysis of skin cutaneous melanoma using data from The Cancer Genome Atlas (TCGA) and validated our findings in independent cohorts. Multi-layered data, including transcriptomic, genomic, epigenetic, and immune landscape profiles, were analyzed using unsupervised clustering and machine learning approaches to define molecular subtypes. Functional assays and in silico drug screening were employed to explore subtype-specific vulnerabilities. Results Three robust molecular subtypes (CS1, CS2, CS3) were identified, each with distinct genomic alterations, tumor microenvironment characteristics, and clinical outcomes. The CS2 subtype was immunologically “hot,” characterized by high tumor mutational burden (TMB), elevated neoantigen load, strong immune infiltration, and activated IFN-γ signaling. CS2 tumors showed significant enrichment of immune checkpoint gene expression and were associated with favorable response to anti-PD-1 therapy in external validation cohorts. In contrast, CS1 and CS3 were immunologically “cold” with immune exclusion, high chromosomal instability, and activation of oncogenic pathways linked to immune evasion. Transcriptomic drug sensitivity modeling suggested that CS1 and CS3 may benefit from HSP90 or MEK inhibitors. Moreover, COL11A2 was identified as a subtype-enriched oncogenic driver predominantly expressed in CS1/CS3, and its silencing impaired tumor cell proliferation, invasion, and epithelial–mesenchymal transition (EMT) features. Conclusions This study presents a refined multi-omics classification of melanoma that reveals biologically and clinically distinct subtypes with divergent immune and therapeutic profiles. It offers a framework for subtype-specific treatment strategies, and identifies COL11A2 as a potential target in immune-cold melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47ce450bfa8af83563781e72095e944188c7b621" target='_blank'>
              Multi-omics-based subtyping of melanoma suggests distinct immune and targeted therapy strategies
              </a>
            </td>
          <td>
            Changchang Li, Xiaoqiong Lin, Jinhui Wang, Qiaochu Zhou, Fangfang Feng, Jie Xu
          </td>
          <td>2025-06-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumorigenesis and metastasis of solid tumors are coupled to profound biophysical changes that alter cancer cells’ mechanobiology, critically impacting metastatic progression. In particular, cell stiffness determines the ability of cancer cells to invade surrounding tissues, withstand shear fluid stress and evade immune surveillance. Here, we summarize the biological factors, pathological factors, and therapeutic modalities that affect the mechanobiology of cancer cells. We focus on clinically utilized chemotherapeutics and targeted therapies that show direct and indirect modulation of cancer cells’ stiffness and discuss how these treatments can be used in combination with other treatment modalities to improve patient outcomes. Finally, we list the outstanding challenges in the field and provide a perspective on expanding the clinical utilization of experimental therapeutics that can act as “mechanotherapeutics” by regulating mechanobiology of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48dea033308f0896b776a8c120aaa366f1ffaad8" target='_blank'>
              Targeting cancer cell stiffness and metastasis with clinical therapeutics
              </a>
            </td>
          <td>
            Alexa M. Gajda, Raymundo Rodríguez-López, E. E. Er
          </td>
          <td>2025-06-11</td>
          <td>Clinical & Experimental Metastasis</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Microsatellite stable colorectal cancer liver metastases (MSS-CRLM) resist immune checkpoint inhibitors due to their immunosuppressive tumor microenvironment (TME) and low mutation burden (TMB). A personalized neoantigen vaccine, Neo-CRCVAS, using whole-exome and RNA sequencing of murine MSS-CRC cells, comprising 7 immunogenic neoantigen peptides with Poly(I:C) is developed and combined with regorafenib as a novel therapy (RegoNeo). In MSS-CRLM mouse models, RegoNeo significantly enhanced tumor regression and survival while establishing durable immune memory. Single-cell RNA and TCR sequencing revealed that RegoNeo expanded a distinct Rgs2⁺CD8⁺ T cell population with strong cytotoxic activity and TCR clonal expansion. These Rgs2⁺CD8⁺ T cells, enriched for neoantigen-specific T cells, demonstrated potent tumor-killing capabilities in both mouse models and patient-derived organoids. The findings establish RegoNeo as a promising personalized immunotherapy that reprograms the immunosuppressive tumor microenvironment by increasing Rgs2⁺CD8⁺ T cell infiltration, highlighting both the treatment approach and this specific T cell subset as potential therapeutic targets for MSS-CRLM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af05deb80fcefae47795b520ff5725ad06de81ba" target='_blank'>
              Personalized Neoantigen Vaccine plus Regorafenib Increases Rgs2⁺CD8⁺ T Cells Infiltration and Reprograms the Tumor Microenvironment in Microsatellite Stable Colorectal Cancer Liver Metastases.
              </a>
            </td>
          <td>
            Hengkai Chen, Bin Chen, Yuanfeng Yang, Shoufeng Li, Huajun Cai, Zhicheng Zhuang, Yong Wu, Yuan Gao, Yupeng Chen, Xing Liu, Guoxian Guan, J. Zhuang
          </td>
          <td>2025-06-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="DNA sequencing technologies are widely used to study tumor evolution within a cancer patient. However, analyses require various computational methods, including those to infer clone sequences (genotypes of cancer cell populations), clone frequencies within each tumor sample, clone phylogeny, mutational tree, dynamics of mutational signatures, and metastatic cell migration events. Therefore, we developed GenoPath, a streamlined pipeline of existing tools to perform tumor evolution analysis. We also developed and added tools to visualize results to assist interpretation and derive biological insights. We have illustrated GenoPath’s utility through a case study of tumor evolution using metastatic prostate cancer data. By reducing computational barriers, GenoPath broadens access to tumor evolution analysis. The software is available at https://github.com/SayakaMiura/GP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e039aed15ffae5155d36f38b8ec85975e0960dc5" target='_blank'>
              GenoPath: a pipeline to infer tumor clone composition, mutational history, and metastatic cell migration events from tumor DNA sequencing data
              </a>
            </td>
          <td>
            Ryan M. Tobin, Shikha Singh, Sudhir Kumar, Sayaka Miura
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e17615


 Background:
 Uterine carcinosarcoma (UCS) is a rare gynecologic malignancy that has poor clinical outcomes and few targeted therapies. This aggressive variant of endometrial carcinoma has a uniquely biphasic appearance and morphology of metastases from these tumors can be carcinoma, sarcoma, or carcinosarcoma. The tumor immune microenvironment (TIME) plays a crucial role in the behavior of cancers. Recent advances in single cell spatial transcriptomics allow for greater insight into the TIME of these aggressive tumors.
 Methods:
 Ten samples from four patients with multiple matched longitudinal samples underwent single-cell spatial transcriptomic (scST) profiling (Nanostring CosMx 6k). Standard segmentation and QC filters were applied. Cell typing was performed with confirmation by expert pathologist annotation of tissue regions. Normalized gene expression was compared within the clusters based on clinicopathologic characteristics.
 Results:
 scST profiling of 80,843 high-quality cells revealed 38 distinct clusters (11 carcinoma, 12 sarcoma, 9 immune, 7 benign mesenchymal). Tumor clusters were largely patient-specific with tumors demonstrating diverse transcriptional heterogeneity while metastatic tumors were composed of several less diverse clones. While immune cell clusters represented cells from all patients, there was significant patient and sample heterogeneity in the proportions of immune cell subsets observed. Macrophages were the most abundant immune cell type in all samples, plasma cells were common in one patient's samples, and rare T cells were found only in specific cellular niches. Transcriptomic features of intra-tumoral macrophages were assessed based on site (primary versus distant metastasis) and temporal (naïve versus treated with chemotherapy) groupings. Compared to tumor associated macrophages (TAM) in primary tumors, TAMs in distant metastases were more likely to have M2-like features with higher expression of
 CXCL2, CD163, CD209,
 and
 CD44
 (P<0.005). TAMs residing in chemotherapy-treated tumors had higher expression of
 IGF2, CXCL2, CXCL1, CD163, CD209, CXCL16, CD24
 ,
 CD44, CD93
 (P<0.005). Upregulation of pro-angiogenic pathways (i.e.
 VEGFA
 ) was common in both distant metastases and post-chemotherapy UCS TAMs.
 Conclusions:
 Uterine carcinosarcoma is a highly heterogeneous tumor both with respect to tumor cell and immune microenvironment features. This work highlights transcriptomic characteristics in macrophages in the UCS microenvironment. Notably, macrophages in distant metastases and in patients with prior chemotherapy were found to have more M2 TAM-like macrophages than primary untreated tumors. These results provide rationale for both TAM-based immunotherapy and anti-angiogenic approaches in UCS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ce5dd582063cc9e8b6f92d0391909010dfa4c1" target='_blank'>
              Single cell characterization of uterine carcinosarcoma and analysis of transcriptomic shifts in tumor-associated macrophages in metastatic and chemotherapy-treated tumors.
              </a>
            </td>
          <td>
            C. Nief, Sabrina Zdravkovic, Sahar Nasr, Phoebe Hammer, Brooke Howitt
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) play pivotal roles in tumor progression and metastasis by mediating intercellular communication within the tumor microenvironment. In this study, we identified two novel EX cargo proteins—UBE2NL and HIST2H3PS2—derived from highly aggressive epithelial ovarian cancer (EOC) cells and mesenchymal-type ovarian stromal progenitor cells (MSC-OCSPCs) but absent in less aggressive SKOV3 cells. Quantitative proteomic profiling via LC-MS/MS and TCGA-integrated analysis revealed that high expression of these genes correlated with advanced tumor stages and poor overall survival in EOC, and high HIST2H3PS2 expression predicted poor survival in endometrial cancer (EC). Functionally, UBE2NL and HIST2H3PS2 overexpression promoted EOC cell invasiveness, which was further enhanced by EX-mediated autocrine and paracrine effects. In contrast, the knockdown of UBE2NL reduced cell invasiveness and prolonged mouse survival in vivo. Moreover, HIST2H3PS2-enriched EXs significantly increased peritoneal dissemination and ascites in murine models. These findings suggest that EX-packaged UBE2NL and HIST2H3PS2 drive tumor aggressiveness and metastasis in gynecologic cancers, highlighting their potential as prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab408f1a8dd6876e3050f8f8e9c0cda2bf511c6" target='_blank'>
              Extracellular-Vesicle-Associated UBE2NL and HIST2H3PS2 Promote Tumor Aggressiveness and Metastasis in Gynecologic Cancer
              </a>
            </td>
          <td>
            Chih-Ming Ho, T. Yen, Tzu-Hao Chang, Shih-Hung Huang
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The extracellular matrix (ECM), closely linked to the dynamic changes in the tumor microenvironment (TME), plays a critical role in modulating tumor immunity. The dual role of the ECM in tumor progression, encompassing both promotion and inhibition, is attributed to its components influencing immune cell activation, migration, and infiltration. This mechanism is intricately connected with the efficacy of immunotherapies. Currently, there is limited understanding of how ECM remodeling spatially and temporally coordinates with immune checkpoint inhibitors (ICIs) or adoptive cell therapies. Furthermore, strategies to selectively target pathological ECM components while preserving their homeostatic functions urgently require systematic investigation. In this review, we summarize current findings on the interplay between ECM and tumor immune regulation, with a particular focus on how key ECM components contribute to immune modulation. Furthermore, we discuss emerging strategies targeting ECM-related mechanisms to enhance the efficacy of immunotherapies, including approaches that remodel the ECM to improve immune infiltration and strategies that synergize with existing immunotherapies. By integrating these insights, we provide a perspective on leveraging ECM-targeted interventions to overcome immune evasion and optimize cancer immunotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5953fd16008b5e87f07c27dd9323238a1a805dce" target='_blank'>
              Extracellular matrix dynamics in tumor immunoregulation: from tumor microenvironment to immunotherapy
              </a>
            </td>
          <td>
            Qin Hu, Yifei Zhu, Jie Mei, Ying Liu, Guoren Zhou
          </td>
          <td>2025-06-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 DNA methylation, an essential epigenetic modification, plays a critical role in establishing and maintaining cellular identities by regulating gene expression. In cancer, aberrant DNA methylation patterns contribute to tumor development and progression, yet the cell-type-specific methylation alterations within the tumor microenvironment (TME) remain poorly understood. In breast cancer, a highly heterogeneous malignancy, endothelial cells within the TME (TECs) are key players in tumor angiogenesis and progression. Methylation changes detected in bulk tumor analyses, however, reflect signals from multiple cell types present in the TME, making it challenging to pinpoint cell-type-specific epigenetic changes. Recently, methods have been developed to using DNA methylation to deconvolute cell type proportions which can in turn be used to attribute methylation alterations to specific cell types in the TME. A reference-based cell-type deconvolution algorithm, the Hierarchical Tumor Immune Microenvironment Epigenetic Deconvolution (HiTIMED) method, allows accurate quantification of cell type proportions within the TME. We accessed DNA methylation array data from 609 breast tumors and 331 nontumor normal breast tissues from the Gene Expression Omnibus (GEO) and the Genotype-Tissue Expression (GTEx) portal. To these DNA methylation profiles, we applied HiTIMED and measured tumor and non-tumor cell proportions. Then, applying a statistical interaction model called CellDMC we leveraged the cell type proportion data to identify differential methylation specific to cell types in the breast cancer TME, including TECs and tumor cells. We identified 1897 tumor-cell-specific and 180 TEC-specific differentially methylated cytosines (DMCs) in cancer compared to normal breast tissue (FDR ≤ 1e-10). Importantly, in the cell-type unadjusted differential analysis, tumors exhibited hypermethylation compared to normal tissues; however, in the tumor-cell-specific analysis, tumor cells were hypomethylated relative to normal samples. By applying these advanced techniques to breast cancer samples, our study unveils cell-type-specific DNA methylation changes that are pivotal in driving the pro-tumorigenic environment. These findings not only enhance our understanding of the molecular mechanisms underlying breast cancer progression but also highlight potential therapeutic targets within the TME. The integration of HiTIMED and CellDMC represents a significant methodological advancement in cancer epigenetics, offering a more precise and insightful approach to studying the complex interplay between different cell types within the TME.
 Citation Format: Barbara Barbara, Lucas A. Salas, Brock C. Christensen. Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-29.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ae964d40b279e042709ea22a740c1da93f0fc" target='_blank'>
              Abstract P5-06-29: Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution
              </a>
            </td>
          <td>
            Barbara Barbara, Lucas A Salas, Brock C. Christensen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e14635


 Background:
 Recent advances in computational pathology have revolutionized the analysis of histopathology images, enabling precise and rapid clinical outcome predictions. However, a significant challenge persists in translating computational insights into biologically meaningful data that informs clinical decisions. Human-interpretable image features (HIFs) have emerged as a solution, offering a detailed view of the tumor microenvironment (TME). In this study, we explored how HIFs derived from high-resolution histopathology images can predict the expression of genes associated with poor survival outcomes in stomach adenocarcinoma (STAD), aiming to bridge the gap between computational pathology and actionable clinical insights.
 Methods:
 Whole-slide histopathology images of STAD from The Cancer Genome Atlas were analyzed. Expert pathologists annotated labels were used to identify tissue types like cancer, stroma, necrosis, and normal tissue and cell types, including cancer cells, lymphocytes, macrophages, plasma cells, and fibroblasts. Using these annotations, convolutional neural networks were trained to classify these tissue and cell types, from which HIFs were extracted reflecting the biological composition of the TME, such as cell density ratios relative to surrounding tissues. We then identified genes with strong correlations to HIFs (ρ > 0.5) for further analysis.
 Results:
 Our results revealed high expression of four genes
 ABCA6, ABCA8, ADAM33,
 and
 ADAMTS10
 that were significantly associated with poor survival outcomes in STAD (p < 0.01). Each gene demonstrated a strong correlation (ρ > 0.5 and p = < 0.01) with specific HIFs.
 ABCA6
 and
 ABCA8
 , genes involved in lipid transport, are linked to a high fibroblast-to-stroma density ratio. This stromal-dominant environment is a hallmark of therapy resistance and poor outcomes.
 ADAM33
 and
 ADAMTS10
 , proteases involved in extracellular matrix remodeling, were strongly correlated (ρ > 0.5a and p = < 0.01) with a high macrophage-to-stroma density ratio. This feature represents an immunosuppressive TME, often seen in aggressive and invasive cancers.
 Conclusions:
 These findings underscore the power of HIFs to reveal key TME characteristics. A stromal-dominant TME, defined by elevated
 ABCA6
 and
 ABCA8
 , and an immunosuppressive, macrophage-rich environment associated with increased
 ADAM33
 and
 ADAMTS10
 , serve as markers of disease aggressiveness and poor prognosis. By integrating these insights into AI-driven models, clinicians may identify high-risk patients earlier in their treatment journey, enabling personalized treatment strategies to improve outcomes for STAD patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebad59e7c9b25a675a77d956eb6f9b7c2d45acd0" target='_blank'>
              Linking gene expression to tumor microenvironment using H&E features in stomach adenocarcinoma.
              </a>
            </td>
          <td>
            S. H. Bukhari, F. Vempalli, J. D. Warren, Shyam Aggarwal, Aditya Sarin, Mandeep Singh Malhotra, Rakesh K. Yadav, M. Macha, A. Bhat, Harry Lander, Tariq Masoodi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background: Lung adenocarcinoma (LUAD) remains a leading cause of cancer-related mortality despite advances in treatments, necessitating more effective therapeutic strategies. Single-cell RNA sequencing (scRNA-seq) technology has revolutionized our ability to dissect the cellular complexity of cancers, which is often obscured in conventional bulk transcriptomic experiments. Methods: In this study, we performed an integrative analysis of scRNA-seq data from multiple LUAD patient cohorts to investigate cell-type-specific transcriptomic changes across disease stages. Clustering, lineage trajectory analysis, and transcriptional regulatory network reconstruction were employed to identify stage-specific gene markers and their upstream regulators. Additionally, we constructed intercellular communication networks to evaluate signaling changes within the tumor microenvironment (TME) during LUAD progression. Results: Our analysis revealed that epithelial cells from stage IV tumors exhibited a distinct transcriptional profile compared to earlier stages, a separation not observed in immune or stromal cell populations. We identified a panel of gene markers that differentiated epithelial cells across disease stages and effectively stratified patients into subgroups with distinct survival outcomes and TME compositions. Regulatory network analysis uncovered key transcription factors, including ATF3, ATF4, HSF1, KLF4, and NFIC, as potential upstream regulators of these stage-specific genes. Moreover, cell–cell communication analysis revealed a significant increase in signaling originating from epithelial cells and a concomitant decrease in immune-derived signals in late-stage LUAD. We identified several signaling pathways enriched in stage-specific crosstalk, including Wnt, PTN, and PDGF pathways, which may play critical roles in LUAD progression. Conclusions: This study provides a comprehensive single-cell resolution map of LUAD progression, highlighting epithelial-driven regulatory programs and dynamic intercellular communication within the TME. Our findings uncover novel molecular markers and regulatory mechanisms with potential prognostic and therapeutic value for more precise treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecbc245a8fbf12aaba1f14070430ba2f845cb23" target='_blank'>
              Single-Cell Transcriptomic Analysis Unveils Key Regulators and Signaling Pathways in Lung Adenocarcinoma Progression
              </a>
            </td>
          <td>
            Jialu Ma, Caleb McQuay, John Talburt, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2025-06-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2688e0962e702419956c9e4e0d9f4727ce79a529" target='_blank'>
              A polyomavirus-positive Merkel cell carcinoma mouse model supports a unified cancer origin
              </a>
            </td>
          <td>
            Wendy Yang, Sara Contente, Sarah Rahman
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c490b5674b718496318b2118c9f55620a929ba49" target='_blank'>
              Dictionary of human intestinal organoid responses to secreted niche factors at single cell resolution
              </a>
            </td>
          <td>
            Meghan M. Capeling, Bob Chen, Kazeera Aliar, Elisa Penna, Veronica Ibarra Lopez, Conrad Foo, Sandra Rost, Loryn Holokai, Xinming Tong, Devan Phillips, Caden W. Sweet, Jing Li, Sharmila Chatterjee, Elizabeth Skippington, Z. Modrusan, Lisa M. McGinnis, Runmin Wei, M. Keir, O. Rozenblatt-Rosen, Michelle B. Chen
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="
 3593


 Background:
 Variation in response to radiotherapy for the treatment of rectal cancer is likely due to heterogeneity in the tumour microenvironment. However, to date, no reliable predictive biomarkers of response are in clinical use and the mechanisms underlying response are unknown. Tertiary lymphoid structures (TLS), which are ectopic lymphoid aggregates found in the tumour microenvironment, have been linked to response to immunotherapy, but little is known about their role in radiotherapy. Here, we aimed to explore the potential of lymphocytes and tertiary lymphoid structures as predictive biomarkers of response to radiotherapy and profile the tumour immune microenvironment in the context of response to radiotherapy.
 Methods:
 For this study, we accessed pre-treatment biopsies from 20 rectal cancer patients with known pathological response to long-course chemoradiotherapy (LCCRT). We selected regions of interest based on immunohistological identification of tumour and lymphocytic infiltrate in formalin-fixed paraffin-embedded tissue. We performed targeted proteomic profiling of 87 immuno-oncology proteins using the Nanostring GeoMx Digital Spatial Profiler to quantify protein expression with spatial resolution within regions of interest, including TLSs, in the tumour microenvironment.
 Results:
 Unsupervised clustering based on normalised protein expression showed a clear separation between the complete responders to LCCRT and all other tumours, and this separation is driven by differences in T cells within TLSs (CD3+). Differentially expressed proteins within CD3+ aggregates include depletion of the natural killer cell marker, CD56 and increased expression of the apoptosis marker, cleaved caspase 9. The distribution of TLS-tumour distance was also significantly different between response groups.
 Conclusions:
 The study highlights the role of TLSs in modulating the immunogenic landscape of the tumour microenvironment in rectal cancer, likely influencing the response to radiotherapy. Spatially resolved proteomic analyses identifies potential biomarkers for radiotherapy response and underscores the importance of profiling tumour-immune microenvironment complexity when stratifying patients for therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38e831504bf073567c720a00c7fb61ae0355f499" target='_blank'>
              Digital spatial profiling: Mapping tumor responses to radiotherapy in rectal cancer.
              </a>
            </td>
          <td>
            R. Purcell, Adèle Hegoburu, Mohammad Amer, Arielle Sulit, Frank Frizelle
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129265cf9a8d9036012e909d0315f3224ece47d6" target='_blank'>
              The epigenetic regulator TRIM24 controls melanoma cell dedifferentiation and resistance to treatment in melanoma
              </a>
            </td>
          <td>
            Simon Durand, Félix Boivin, Roxane M. Pommier, L. Barbollat-Boutrand, Maxime Grimont, Félix Pham, Marion Dufeu, Eric Cumunel, Raphaël Schneider, Anthony Ferrari, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor development is mainly marked by the gradual transformation of cells that acquire capacities such as sustained growth signaling, evasion of growth suppression, resistance to cell death, and induction of angiogenesis, achieving replicative immortality and activating invasion and metastasis. How different epigenetic alterations like m1A, m5C, and m6A contribute to tumor development is a field that still needs to be investigated. The immune modulators, CD70, CD80, and TIGIT, mainly regulate T-cell activation and consequently the immune evasion of tumors. Here, we explored the presence and the potential consequences of RNA modifications in these regulators in pan-cancer. Our findings highlight the critical role of the m6A, m5C, and m1A in regulating CD70, CD80, and TIGIT across multiple solid tumors. By combining epitranscriptomics data with functional enrichment and survival modeling, we show that RNA modification enzymes not only modulate immune-related gene expression but also serve as potential biomarkers for patient prognosis. By constructing a robust four-gene prognostic signature involving YTHDF3, RBM15B, IGF2BP2, and TRMT61A, we demonstrate that RNA modification profiles can accurately stratify patients into risk groups with distinct overall survival outcomes. The performance of this model across eight cancer types underscores the translational promise of epitranscriptomic markers in both mechanistic understanding and personalized oncology. Altogether, our study bridges the gap between the mechanistic regulation of immune checkpoints and their clinical utility, offering novel insights into how the epitranscriptome can be leveraged to improve cancer prognosis and potentially enhance immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d03e20bb526dd9e6903a8377f001f578b9d1d1d" target='_blank'>
              Epitranscriptomics Regulation of CD70, CD80, and TIGIT in Cancer Immunity
              </a>
            </td>
          <td>
            C. Rigopoulos, Marios Gkoris, Ilias Georgakopoulos-Soares, I. Boulalas, A. Zaravinos
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb1152ca394367d1b31e6130a4964a702584963" target='_blank'>
              Multiscale Modeling Uncovers Macrophage Infiltration and TNF-α Signaling Networks for Targeting in Inflammatory Breast Cancer Tumor Emboli
              </a>
            </td>
          <td>
            Pritha Pai, Christophe Van Berckelaer, S. V. van Laere, Alexandra Bennion, Theresa Charity, Jinming Yang, F. Bertucci, Peter van Dam, Gregory M Palmer, Shannon McCall, Luc Y. Dirix, Naoto T. Ueno, G. Devi
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in shaping immunometabolism in prostate cancer, influencing disease progression and therapeutic response. This review examines the dynamic interactions between tumor cells and immune cells within the prostate cancer TME, focusing on how metabolic reprogramming of both tumor and immune cells drives immunosuppression. Key immune players, including T-cells, macrophages, and myeloid-derived suppressor cells, undergo metabolic adaptations influenced by hypoxia, nutrient deprivation, and signaling from tumor cells. Additionally, we discuss the metabolic pathways involved, such as glycolysis and oxidative phosphorylation, and how these processes are exploited by cancer cells to evade immune surveillance. Furthermore, this review highlights potential therapeutic strategies targeting immunometabolism, including metabolic inhibitors and their combination with immunotherapies. A deeper understanding of the complex role of immunometabolism in prostate cancer will not only provide insights into the tumor’s immune evasion mechanisms but also facilitate the development of novel treatment approaches that enhance the efficacy of current therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd6031bfb5363d860488050852357492fe22b41c" target='_blank'>
              Role of Tumor Microenvironment in Prostate Cancer Immunometabolism
              </a>
            </td>
          <td>
            Yutao Wang, Yiming Chen, Jianfeng Wang
          </td>
          <td>2025-06-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2411017a831bd32082b22e6f4221bf8c31f61827" target='_blank'>
              ETS-guided iPSC-endothelial models recapitulate malaria pathogenesis
              </a>
            </td>
          <td>
            François Korbmacher, Hannah Fleckenstein, Rory K. M. Long, Patryk Poliński, Livia Piatti, Borja López-Gutiérrez, Alina Batzilla, Dennis Crusius, Vikas Trivedi, Miki Ebisuya, Maria Bernabeu
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer progression and therapeutic resistance are propelled by the remarkable plasticity of signaling networks, which dynamically rewire under selective pressures to maintain proliferation, enable immune evasion and promote metastasis. Despite advances in precision oncology, the dynamic crosstalk between tumor cells, non-coding genomes and the microenvironment continues to undermine treatment efficacy. This call for submissions, Revolutionizing Cancer Treatment: Navigating the Intricate Landscape of Cellular Signaling Networks, seeks cutting-edge research that dissects these adaptive mechanisms through innovative technologies - from single-cell multi-omics and spatial transcriptomics to AI-powered network modeling. We welcome studies leveraging physiomimetic models (e.g., organoids, 3D-bioprinted ecosystems) to decode tumor heterogeneity, as well as translational work targeting emergent vulnerabilities at the intersection of epigenetics, metabolic reprogramming and stromal interactions. By integrating systems biology with computational and experimental approaches, this collection aims to catalyze the design of adaptive therapies that outmaneuver cancer's evolutionary resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba13faeaf95516c4bb5cbd68aeea6df92e61f127" target='_blank'>
              Revolutionizing cancer treatment: Navigating the intricate landscape of cellular signaling networks.
              </a>
            </td>
          <td>
            Hao Zhang
          </td>
          <td>2025-05-26</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f8652ac4eec6ab6f4d4e47c902764e61cc0510" target='_blank'>
              Taurine from tumour niche drives glycolysis to promote leukaemogenesis.
              </a>
            </td>
          <td>
            Sonali Sharma, Benjamin J. Rodems, Cameron D Baker, Christina M. Kaszuba, Edgardo I. Franco, Bradley R. Smith, Takashi Ito, Kyle Swovick, Kevin A. Welle, Yi Zhang, Philip Rock, Francisco A Chaves, Sina Ghaemmaghami, L. Calvi, Archan Ganguly, W. R. Burack, Michael W. Becker, Jane L. Liesveld, Paul S. Brookes, Joshua C. Munger, Craig T Jordan, John M Ashton, Jeevisha Bajaj
          </td>
          <td>2025-05-14</td>
          <td>Nature</td>
          <td>4</td>
          <td>37</td>
        </tr>

        <tr id="Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies.We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration.Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration.Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recently, contrary to attacking cancer cells, the tumor microenvironment (TME) with genomic stability and vulnerable nature has emerged as promising therapeutic targets in hepatocellular carcinoma (HCC). Within TME ecosystem, tumor-associated macrophages (TAMs) play a pivotal role in tumor evasion and progression of HCC. However, their clinical and therapeutic implications remain unexplored.Utilizing a large-scale sc-RNA seq dataset, a landscape of HCC cellular ecosystem was depicted. Based on previous literature, an effectively differential TAMs subset classification was identified. Gene variations was extracted through trajectory analysis and then unsupervised clustering was conducted within RNA-seq data. Subsequently, survival analysis, specific pathway enrichment as well as hub regulatory network analysis were performed. Additionally, the immune cell infiltration and genomic variations were evaluated between clusters. Drug sensitivity and underlying therapeutic molecular were also explored. Through multiple immunofluorescence, our findings were verified.Herein, integrating single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq data, we established a novel TAM classification system based on mutually exclusive SPP1 and FOLR2 signatures. According to the TAM trajectory genes, unsupervised clustering stratified HCC into three distinct clusters. Cluster 3 (C3), which is characterized by metabolic dysregulation and immunosuppressive TME, exhibited the poorest prognosis among the three groups. Hub network analysis of C3 further indicated its characteristic dysregulation of liver-specific metabolism. SPP1 was identified as a key signature of C3, which contributed to suppressing the infiltration of CD8+ T cells. Therapeutic evaluation revealed that C3 were sensitive to chemotherapy and tyrosine kinase inhibitors, while those C1 and C2 were more suitable for immunotherapy. Drug screening identified potential therapeutic compounds for each cluster.This study redefines the heterogeneity of TAMs beyond the M1/M2 paradigm, linking the TAMs trajectory genes to HCC patient stratification. SPP1 blockade emerged as a strategy for counteracting immunosuppression, and cluster-specific therapies may optimize the management of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d85b5a9a5d2466c00caeea207b03b8145b6f159" target='_blank'>
              SPP1+ tumor-associated macrophages define a high-risk subgroup and inform personalized therapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Wei-Xuan Xu, Yanjun Ye, Jia-Lin Chen, Xin-Ying Guo, Chen Li, Juan Luo, Lin-Bin Lu, Xiong Chen
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) presents significant clinical challenges owing to its dense stroma and complex tumor microenvironment (TME). In this study, large-scale single-cell transcriptomics and spatial transcriptomics (ST) were integrated to dissect the heterogeneity of fibroblasts and their crosstalk with epithelial cells, with a focus on key ligand-receptor interactions. Eight distinct fibroblast subpopulations were identified, among which extracellular matrix (ECM)-remodeling fibroblasts were particularly enriched in tumor tissues and associated with poor prognosis. ECM-remodeling fibroblasts were located at the terminal stage of the fibroblast pseudotime trajectory, and SOX11 was identified as a key transcription factor in this subpopulation. Further analyses revealed that ECM-remodeling fibroblasts can interact with epithelial cells through the POSTN-ITGAV/ITGB5 ligand-receptor axis, a critical pathway that promotes tumor progression. Clinical analyses demonstrated a strong correlation between POSTN expression and poor prognosis in patients with PDAC. Mechanistically, POSTN interacts with integrin ITGAV/ITGB5 on tumor cells, activating the PI3K/AKT/β-catenin pathway and promoting epithelial-mesenchymal transition (EMT) phenotype. Pharmacological inhibition of the POSTN-integrin axis partially reversed these malignant traits, highlighting its potential as a therapeutic target. This study provides new insights into fibroblast heterogeneity and its role in PDAC progression, emphasizing the POSTN-ITGAV/ITGB5 axis as a promising target for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e1d45940647abe0d0d6b914abc94702cee2e0ff" target='_blank'>
              Integrative Single-Cell and Spatial Transcriptomics Analysis Reveals ECM-remodeling Cancer-associated Fibroblast-Derived POSTN as a Key Mediator in Pancreatic Ductal Adenocarcinoma Progression
              </a>
            </td>
          <td>
            Yifan Wu, Shuquan Li, Hao Yu, Shan Zhang, Lianghua Yan, Xiao-ya Guan, Wei Xu, Zhen Wang, Ang Lv, Xiuyun Tian, Chunyi Hao, Jianhui Wu
          </td>
          <td>2025-05-27</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Human skin is not a uniform organ but a mosaic of anatomically distinct niches, with each site finely tuned to unique environmental demands and immune pressures. Yet, the molecular determinants that define these regional identities and their relationship to site-specific vulnerability to inflammatory disease remain poorly understood. Here, we generate a high-resolution single-cell atlas of human skin, profiling 274,834 cells from 96 healthy samples across 7 anatomically distinct sites (acral, arm, axilla, back, face, leg and scalp). Our analysis reveals striking region-specific transcriptional and cellular networks, uncovering how local immune-stromal crosstalk governs tissue homeostasis and underpins anatomical susceptibility to distinct inflammatory diseases such as such as systemic lupus erythematosus (SLE), atopic dermatitis (AD), and psoriasis. These findings illuminate the tissue-intrinsic foundations of regional immune identity and provide a blueprint/resource for the development of precision therapies tailored to the distinct immunological microenvironments of specific anatomical skin sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0efad5522b30037ec342a6195a031267736175c" target='_blank'>
              A Transcriptomic Atlas of Healthy Human Skin Links Regional Identity to Inflammatory Disease
              </a>
            </td>
          <td>
            Sahiti Marella, Rachael Bogle, Jennifer Fox, L. Tsoi, Xianying Xing, Yiqian Gu, J. Kirma, M. Sarkar, Vincent van Drongelen, Benjamin Klein, Jeff H. Kozlow, Paul W. Harms, Katherine A. Gallagher, S.Chakravarthy Naik, Vito W. Rebecca, Bogi Andersen, Mio Nakamura, J. Kahlenberg, R. Modlin, A. Billi, Johann E. Gudjonsson
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968347e5e4a23139233feb67cc1b20faf82efda9" target='_blank'>
              Modeling Bistable Dynamics Arising from Macrophage-Tumor Interactions in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Hwayeon Ryu, Susanna Röblitz, K. Larripa, A. Frank
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Embryonal tumor with multilayered rosettes (ETMR) is a pediatric brain tumor with dismal prognosis. Characteristic alterations of the chromosome 19 microRNA cluster (C19MC) are observed in most ETMR; however, the ramifications of C19MC activation and the complex cellular architecture of ETMR remain understudied. Here we analyze 11 ETMR samples from patients using single-cell transcriptomics and multiplexed spatial imaging. We reveal a spatially distinct cellular hierarchy that spans highly proliferative neural stem-like cells and more differentiated neuron-like cells. C19MC is predominantly expressed in stem-like cells and controls a transcriptional network governing stemness and lineage commitment, as resolved by genome-wide analysis of microRNA-mRNA binding. Systematic analysis of receptor–ligand interactions between malignant cell types reveals fibroblast growth factor receptor and Notch signaling as oncogenic pathways that can be successfully targeted in preclinical models and in one patient with ETMR. Our study provides fundamental insights into ETMR pathobiology and a powerful rationale for more effective targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02046d3dbdf78b8dbf1ae4df369d0858d2ab96f2" target='_blank'>
              Cellular hierarchies of embryonal tumors with multilayered rosettes are shaped by oncogenic microRNAs and receptor–ligand interactions
              </a>
            </td>
          <td>
            Alexander Beck, Lisa Gabler-Pamer, G. Alencastro Veiga Cruzeiro, S. Lambo, Bernhard Englinger, McKenzie L. Shaw, Olivia A Hack, Ilon Liu, Rebecca D. Haase, Carlos A O de Biagi, A. Baumgartner, Andrezza Do Nascimento Silva, M. Klenner, Pia Freidel, Jochen Herms, Louisa von Baumgarten, Joerg-Christian Tonn, N. Thon, K. Bruckner, S. Madlener, L. Mayr, D. Senfter, A. Peyrl, I. Slavc, D. Lötsch, C. Dorfer, Rene Geyregger, N. Amberg, C. Haberler, Norman L Mack, Benjamin Schwalm, Stefan M. Pfister, Andrey Korshunov, Lissa C Baird, Edward Yang, Susan N Chi, S. Alexandrescu, J. Gojo, Marcel Kool, Volker Hovestadt, Mariella G. Filbin
          </td>
          <td>2025-05-26</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a high-mortality cancer characterized by its aggressive, treatment-resistant phenotype and a complex tumour microenvironment (TME) featuring significant hypoxia. Bulk transcriptomic analysis has identified the “classical” and “basal-like” transcriptional subtypes which have prognostic value; however, it is not well-established how microenvironmental heterogeneity contributes to these transcriptional signatures. Here, we exploited the TRACER platform to perform single cell transcriptome analysis of organoids at specific spatial locations to explore the effect of oxygen and other cell-generated microenvironmental gradients on organoid heterogeneity. We found that the microenvironmental gradients present in TRACER significantly impact the distribution of organoid transcriptional phenotypes and the enrichment of gene sets linked to cancer progression and treatment resistance. More significantly, we found that microenvironmental gradients, predominantly oxygen, drive changes in the expression of classical and basal-like transcriptional subtype gene signatures. This work suggests that hypoxia contributes to determining transcriptional subtypes in PDAC tumour cells independent of additional cells in the TME and broadly highlights the importance of considering microenvironmental gradients such as oxygen in organoid-based studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-98344-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d7b936c3248bb437b55ed176049e4ec9a8d43e" target='_blank'>
              Analysis of an engineered organoid model of pancreatic cancer identifies hypoxia as a contributing factor in determining transcriptional subtypes
              </a>
            </td>
          <td>
            Natalie Landon-Brace, Simon Latour, Brendan T. Innes, Michelle Nurse, Jose L. Cadavid, I. Co, Cassidy M Tan, Ferris Nowlan, Sibil Drissler, F. Notta, H. Jackson, Gary D Bader, A. McGuigan
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Type I collagen (Collagen I, Col I), a principal component of the tumor extracellular matrix (ECM) that accounts for 80% of total ECM collagen, has emerged as a crucial factor in tumor biology. Recent research has found that Col I has influenced tumor growth, invasion, migration, and prognosis by forming a physical barrier and interacting with components in the tumor microenvironment (TME). Pancreatic ductal adenocarcinoma (PDAC) is characterized by a prominent stromal microenvironment with remarkable cellular and spatial heterogeneity, which significantly impacts the biology of the disease and its resistance to treatment. As research on the influence of ECM on tumor behavior progresses, the role of Col I in tumor occurrence and development has also been gradually expounded. Unlike the pro-tumorigenic role caused by excessive deposition in other cancers, PDAC features a dense stroma with spatial heterogeneity, where Col I deposition exhibits paradoxical pro- and anti-tumor effects: Cancer-associated fibroblasts secrete heterotrimeric Col I (α1/α1/α2), which initially suppresses tumor growth by recruiting M1 macrophages, while tumor cells produce homotrimeric Col I (α1/α1/α1, 2–3% of total Col I) to promote immune evasion via CXCL5 secretion and CD8+ T cell reduction. Mechanistically, Col I activates TGF-β/SMAD3 signaling to drive epithelial–mesenchymal transition (EMT) and enhances matrix stiffness, increasing tumor cell migration by 60–80% in 3D cell culture models. Thus, understanding the complex interactions between Col I and other microenvironmental components may enhance its anti-tumor action while suppressing its pro-tumor activity, potentially improving PDAC patients’ survival. In this review, we discuss the current research progress regarding the function of Col I in PDAC tumors and summarize its contributions to PDAC progression, thus providing a basis for follow-up research and potential targets for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ceb222174824b91b91f72e2fc4e082f366f425a" target='_blank'>
              The type I collagen paradox in PDAC progression: microenvironmental protector turned tumor accomplice
              </a>
            </td>
          <td>
            Qian Li, Cheng Yang, Junqiang Li, Ronglin Wang, Jie Min, Yang Song, Haichuan Su
          </td>
          <td>2025-07-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 4155


 Background:
 The tumor microenvironment (TME) that surrounds pancreatic ductal adenocarcinoma (PDAC) is a multi-faceted and dynamic ecosystem in which stromal fibroblasts communicate with cancer cells to mediate tumor growth, metastasis, and chemotherapy resistance. It is well recognized that both cancer-associated fibroblasts (CAFs) and its non-cellular, fibrotic components within the TME can foster a protumorigenic environment for PDAC. However, we still lack a comprehensive understanding of the precise mechanisms in which this dense, fibrotic matrix can help drive malignant behaviors. Here, we reveal a novel mechanism in which the zinc-finger transcription factor GLI2 regulates type 1 collagen expression within CAFs and how the soluble variant of this collagen promotes irinotecan chemoresistance.
 Methods:
 We leveraged transcriptomic data from The Cancer Genome Atlas, International Cancer Genome Consortium, and Clinical Proteomic Tumor Analysis Consortium to evaluate GLI2 expression and stromal content of human PDAC tumors through bulk RNA-sequencing deconvolution. Using single-nucleus RNA sequencing of human PDAC tumors, we validated the association of GLI2 expression and stromal matrix constituents in CAFs. Chromatin immunoprecipitation assays in human CAFs confirmed GLI2 binding at the COL1A1 promoter. Through RNAi-based inactivation of GLI2, we determined how loss of GLI2 impacts regulation of type 1 collagen. Additionally, we conducted MTT assays to assess tumor viability in response to irinotecan treatment.
 Results:
 Transcriptomic analysis revealed that GLI2 is highly enriched in CAFs and strongly correlated with stromal fibrosis compared to other non-tumor cell constituents within the TME. We have shown that GLI2 directly binds to the promoter of COL1A1, a key component of type 1 collagen, in CAFs and regulates its transcription in a manner dependent on TGFβ1 signaling. Interestingly, PDAC tumors exposed to type 1 collagen show increased expression of pro-tumorigenic pathways involved in inflammation, EGR signaling, cytokine-receptor interactions, and irinotecan resistance. We further validate that human PDAC cells pre-treated with collagen can confer chemoresistance to irinotecan with viability assays.
 Conclusions:
 Taken together, our study demonstrates a novel mechanism in which GLI2 regulates the secretion of collagen within CAFs, which in turn can enable PDAC to acquire resistance to standard-of-care treatments. These findings highlight not only the oncogenic functions for CAFs and their fibrotic secretome, but also open new avenues in which therapeutic targeting of the TME may provide clinical benefit for PDAC patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3a2065caeb9cc2540ae0418d70044e4a839709" target='_blank'>
              Functional role of GLI2 in cancer-associated fibroblasts for modulation of the fibrotic tumor microenvironment within pancreatic cancer.
              </a>
            </td>
          <td>
            John Y. Kwon, Renzo E Vera, Maite G. Fernadez-Barrena, Jose M. Falero, Roberto G. Chaparro, Matthew D. Ross, M. D. Toruner, Murat Toruner, Ezequiel J Tolosa, Luciana L Almada, Huocong Huang, Rolf A. Brekken, Martin E Fernandez-Zapico
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Gastroenteropancreatic neuroendocrine tumors (GEP-NETs) are a family of tumors that arise throughout the gastrointestinal tract. These tumors are heterogeneous, with complex clinical symptoms and tumor behaviors, and demonstrate rising incidence rates worldwide. In addition to their nature, GEP-NETs possess limited diagnostic and therapeutic options, which results in poor survival rates for patients with metastatic tumors. Given these findings, a further analysis of these tumors’ biology is needed to determine new therapeutic strategies. The tumor microenvironment (TME) consists of several residual cell populations and non-cellular components whose altered behavior creates a tumor-supportive niche. Studies from other cancers demonstrate the TME’s significance in tumor initiation, progression, and spread. In this review, we discuss efforts to characterize the TME in GEP-NETs. Preliminary studies of the immune system in GEP-NETs have led to several major clinical trials, with limited success. Efforts to target signaling crosstalk between cancer-associated fibroblasts, vascular endothelial cells, and tumor cells has led to major discoveries and multiple approved therapies. Finally, alterations to the extracellular matrix may lead towards an improved understanding of GEP-NET development, behavior, and improved detection methods. While research has rapidly expanded our knowledge within the last decade, further work is needed to bring our understanding of the GEP-NET TME in line with other rare cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5602e7167401420da4dc63b32444d080749f374d" target='_blank'>
              The Role of the Tumor Microenvironment in Gastroenteropancreatic Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Srujana V. Yellapragada, Steven D. Forsythe, James P. Madigan, Samira M. Sadowski
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7be43f99c39d72183ee247e245dd3934afb3150" target='_blank'>
              Decoding meningioma prognosis with multi-omics: macrophage diversity, immune-CNV interplay, and novel SPP1-targeted strategies.
              </a>
            </td>
          <td>
            Hailang Fan, Xiaojie Li, Yaqian Zhao, Lairong Song, Lanlan Sun, Zhen Wu, Junting Zhang, Haikun Zhang, Feifan Zhang, Yingdi Huang, Dake Zhang, Liang Wang
          </td>
          <td>2025-06-16</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In vitro hematopoiesis systems can be used to define mechanisms for blood cell formation and function, produce cell therapeutics, and model blood cell contributions to systemic disease. Hematopoietic progenitor cell (HPC) production remains inefficient, precluded by knowledge gaps related to specification and morphogenesis of specialized hemogenic endothelial cells, which undergo an endothelial-to-hematopoietic transition (EHT) to form HPCs. We elected to define changes in gene expression and chromatin organization during HPC formation to reveal regulatory mechanisms. Using paired single cell RNA/ATAC sequencing together with Hi-C, we profiled cells before and after EHT. Pathway analysis and pseudotime inferences confirmed a continuum of stromal and endothelial cells undergoing development into HE cells and lineage-based HPCs in vitro. In these cell types, we characterize cis-regulatory elements and transcriptional regulatory activities that facilitate EHT and HPC homeostasis, including for SNAI1, SOX17, TGFβ, STAT4, as well as for GFI1b and KLF1 in megakaryocyte- and erythroid-biased progenitors, respectively. We then leveraged our insights into chromatin organization among in vitro-derived cells to assess enrichments corresponding to human trait variation reported in human genome wide association studies. HPCs revealed locus enrichment for quantitative blood traits and autoimmune disease predisposition, which were particularly enriched in myeloid- and lymphoid-biased populations. Stromal and endothelial cells from our in vitro cultures were specifically enriched for accessible chromatin at blood pressure loci. Our findings reveal genes and mechanisms governing in vitro hematopoietic development and blood cell-related disease pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4774749a5a1117ef1a36110f9d8a3ed3849f1988" target='_blank'>
              Multimodal analysis of in vitro hematopoiesis reveals blood cell-specific genetic impacts on complex disease traits
              </a>
            </td>
          <td>
            Rong Qiu, Khanh B. Trang, Carson Shalaby, J. Pippin, James Garifallou, Struan F. A. Grant, Christopher S. Thom
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c087be1ee8d5c3fb0e227ebb941d840bf16ba32a" target='_blank'>
              Residual Breast Cancer Cells Co-opt SOX5-driven Endochondral Ossification to Maintain Dormancy
              </a>
            </td>
          <td>
            Amulya Sreekumar, Eric Blankemeyer, Christopher J. Sterner, Tien-Chi Pan, D. Pant, Sarah Acolatse, Hamza Turkistani, G. Belka, Sean D. Carlin, C. Assenmacher, M. Sellmyer, David A. Mankoff, Lewis A. Chodosh
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Ewing sarcoma (ES) is a rare and aggressive pediatric bone malignancy with poor prognosis, driven by therapy-resistant tumor microenvironments (TME). The TME plays a critical role in tumor progression through a complex and dynamic network of reciprocal interactions among immune cells (dysfunctional T cells, immunosuppressive macrophages), stromal components (cancer-associated fibroblasts), and tumor cells. These interactions collectively shape the immune landscape, promote immune evasion, and contribute to therapeutic resistance. Identifying reliable prognostic markers remains a critical challenge. Methods Here we performed an integrated single-cell RNA sequencing, WGCNA, and bulk RNA-seq analyses to investigate tumor-immune interactions. Differentially expressed genes (DEGs) intersected with T cell markers identified a total of 174 T cell-associated genes. Functional enrichment analysis and molecular subtyping were performed to explore immune-related pathways. A prognostic model based on CLEC11A, BDP1, and ID3 was constructed using Cox regression and validated in external datasets. Immune infiltration was assessed using the CIBERSORT algorithm. Results T cell marker analyses revealed key roles in pathways such as PI3K-Akt signaling and immune modulation. Molecular subtyping identified two clusters with distinct immune microenvironments: Cluster C1 (immunosuppressive phenotype and poorer prognosis) and Cluster C2 (functionally active immune profile associated with better prognosis). The prognostic model demonstrated high predictive accuracy for 1-, 3-, and 5-year survival (AUC: 0.85, 0.82, 0.78). Additionally, a higher tumor mutation burden (TMB) with low survival rate has been observed in High-risk group. Immune infiltration analysis showed higher CD8+ T cell and dendritic cell activity and immune checkpoint expression in low-risk groups. Experimental validation demonstrated that ID3 silencing inhibited tumor cell proliferation and induced cell cycle arrest in ES cell lines. Conclusion Together, our study identified CLEC11A, BDP1, and ID3 as key T cell associated prognostic markers and developed a validated model to predict survival outcomes in ES. Insights into T cell markers and tumor-immune dynamics offer promising advances in prognostic assessment and immunotherapy for ES. Furthermore, the role of ID3 in immune evasion and tumor proliferation underscores its potential as a therapeutic target, providing new avenues for immune checkpoint regulation and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61b182bc7d5b374e78a6d8b617b48c4b427c14cb" target='_blank'>
              Integrative analysis of T cell-associated markers in Ewing sarcoma reveals prognostic signatures and immune dynamics
              </a>
            </td>
          <td>
            U. Saddozai, Chenxu Liu, Fei Yan, Zhendong Lu, M. B. Khawar, Muhammad Usman Akbar, Saadullah Khattak, Haibo Sun, Ping Yang
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Gastric cancer (GC) is among the most lethal human malignancies with limited treatment options. Cell-cell interactions within the tumor microenvironment (TME) can promote tumor growth, yet their therapeutic value has not been fully explored. Here, bulk RNA-seq, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics (ST) were integrated to analyze the heterogeneity of GC microenvironment. Tumor-specific GREM1+ fibroblasts and SPP1+ macrophages were significantly enriched in GC tissues and were involved in immunosuppression, inflammation regulation, and tumor progression. We then indicated that GREM1+ fibroblasts and SPP1+ macrophages were positively correlated in 12 independent GC datasets and validated their close localization by multiplex immunohistochemistry staining and spatial transcriptomics. Patients with both high GREM1+ fibroblasts and SPP1+ macrophages exhibited significantly shorter OS and showed enrichment of tumor-associated pathways. Our results demonstrated the complex interactions between GREM1+ fibroblasts and SPP1+ macrophages, which may serve as a potential therapeutic target for future treatment of GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad0abacc337130966d37d3605e0104e32183302" target='_blank'>
              Multi-omics analyses reveal interactions between GREM1+ fibroblasts and SPP1+ macrophages in gastric cancer
              </a>
            </td>
          <td>
            L. Qiu, Xiao Zhao, Sheng Yao, Yang Fei, Y. Gong, Zishan Zhou, Shunchang Jiao, Jianming Xu
          </td>
          <td>2025-06-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Cancer cells achieve replicative immortality through telomere maintenance mechanisms (TMMs), primarily via telomerase activation or alternative lengthening of telomeres (ALT). Sarcomas frequently employ the ALT pathway, which traditionally correlates with adverse clinical outcomes. However, chondrosarcomas represent a unique context where the role and prognostic significance of ALT remain largely unexplored. Methods We performed comprehensive analyses of single-cell RNA-sequencing data from patients with chondrosarcoma and integrated this with 90 bulk RNA-seq datasets. This approach enabled detailed characterization of TMM at single-cell resolution, identification of ALT-specific signatures, and evaluation of the tumor microenvironment in chondrosarcomas. Results Patients with ALT-like chondrosarcomas exhibited significantly improved survival compared to those with non-ALT-like chondrosarcomas. Analysis of the tumor immune microenvironment revealed distinct metabolic and immune landscapes between the ALT-like and non-ALT-like groups. Single-cell analysis showed that high-entropy stem-like cells in high-grade chondrosarcomas predominantly adopted telomerase activation over the ALT pathway as their TMM. Additionally, we identified a 100-gene signature that reliably distinguishes ALT-like chondrosarcomas, providing a robust molecular marker for classification and prognosis. Conclusions Our study reveals ALT-like state as a marker of favorable prognosis in chondrosarcomas—contrasting with its typically adverse implications in other sarcomas. We establish a robust 100-gene signature that reliably identifies ALT-like chondrosarcomas and characterize their distinct immune microenvironment profile. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881425c49c7bf310ad569e7f1a7bdeb0d0fff86f" target='_blank'>
              Alternative lengthening of telomeres confers favorable prognosis in chondrosarcomas
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jin-Hong Kim, Yi-Jun Kim
          </td>
          <td>2025-05-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The stromal tumor environment (STE) plays a vital yet often underrecognized role in cancer progression, shaping tumor development. It influences complex interactions among stromal cells, immune cells, extracellular matrix (ECM) components, and signaling molecules. Stromal cells regulate integrin signaling and are involved in remodeling the tumor microenvironment, driving critical processes such as cell migration, survival, angiogenesis, and proliferation. Advances in understanding the STE are creating new opportunities for early cancer detection and treatment. Innovations in molecular imaging, particularly for melanoma, are transforming cancer diagnostics by providing deeper insights into tumor biology, cellular interactions, and angiogenesis, paving the way for more personalized and timely therapeutic interventions. Among these advancements, in vivo skin fluorescent imaging (SFI) stands out as a powerful tool, offering real-time molecular insights at the point of care with the ability to improve patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/887153b98db08b0a71265bb4aa6af40acad38403" target='_blank'>
              The Role of Stromal Tumor Environment in Advancing Skin Cancer Diagnosis and Treatment.
              </a>
            </td>
          <td>
            Frank Jing, Ani Pazhava, Catherine Shachaf, A. Meves
          </td>
          <td>2025-05-14</td>
          <td>International journal of dermatology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer cells alter their mechanical properties in response to the rigidity of their environment. Here, we explored the implications of this environmental mechanosensing for anti-tumor immunosurveillance using single cell biophysical profiling and metastasis models. Cancer cells stiffened in more rigid environments, a biophysical change that sensitized them to cytotoxic lymphocytes. In immunodeficient mice, this behavior manifested in the outgrowth of stiffer metastatic cells in the rigid bone than in the soft lung, while in immunocompetent hosts, it led to preferential elimination of stiffer cancer cells and suppression of bone metastasis. Environmentally-induced cell stiffening and immune sensitization both required Osteopontin, a secreted glycoprotein that is upregulated during bone colonization. Analysis of patient metastases spanning mechanically distinct tissues revealed associations between environmental rigidity, immune infiltration, and cancer cell stiffness consistent with mechanically driven immunosurveillance. These results demonstrate how environmental mechanosensing modulates anti-tumor immunity and suggest a mechanoimmunological basis for metastatic site selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36919956b3fc6d216b0503cde81fa82cac7cb9cf" target='_blank'>
              Mechanoimmunological Control of Metastatic Site Selection
              </a>
            </td>
          <td>
            Yassmin Elbanna, M. Tello-Lafoz, Aliya Holland, Ye Zhang, Jun-Goo Kwak, Zhenghan Wang, Alexandrina M. Yakimov, Myra Dada, Samuel Vayner, Sarah M. Duquette, Young Hun Kim, T. Bale, Benjamin Y. Winer, Kenny K. H. Yu, J. Massagué, Jungwoo Lee, O. Barzilai, S. Manalis, Morgan Huse
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Breast cancer is the most prevalent form of malignant tumor that frequently metastasizes to axillary lymph nodes (LNs). Nonetheless, the precise mechanisms underlying alterations in the tumor microenvironment (TME) in LN metastasis in breast cancer remain poorly understood. Single-cell RNA sequencing of 28 LN samples from 23 patients is performed, and a comprehensive landscape of the entire ecosystem is generated. Ten major cell types are identified, with the subclusters of each major cell type exhibiting diverse characteristics. Furthermore, multiple signatures are collected to evaluate the key components of the subclusters using multi-omics methodologies. This study finds that myCAFs may hasten LN metastasis, and observed a notable increase in APOE+ macrophages and a higher proportion of exhausted CD8+ T cells, contributing to the immunosuppressive TME. Moreover, cancer cells in metastatic lesions exhibited diverse expression patterns linked to proliferation, metastasis, oxidative phosphorylation, hypoxia, and interferon responses. Using multi-omics approaches and experimental validations, it determines that GLO1 can promote lymphatic angiogenesis, metastasis, and inhibit the proteasomal degradation of GSS, thereby maintaining intracellular glutathione (GSH) and reactive oxygen species (ROS) balance. Collectively, the study offers novel perspectives on the microenvironment remodeling of breast cancer LN metastases, suggesting that GLO1 may be a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f92d409a651118984d17df6fe54f30f3acdf6d65" target='_blank'>
              Paracrine Orchestration of Tumor Microenvironment Remodeling Induced by GLO1 Potentiates Lymph Node Metastasis in Breast Cancer.
              </a>
            </td>
          <td>
            Jindong Xie, Wenjian Liu, Xinpei Deng, Huan Wang, Xueqi Ou, Xin An, Min-yi Situ, An-Kui Yang, Chuan Peng, Rongfang He, Yi Xie, Hailin Tang, Yuman Chen, Jie-Ying Liang, Ruonan Shao, Weikai Xiao, Shaoquan Zheng
          </td>
          <td>2025-06-10</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 e20051


 Background:
 Histone lactylation, a newly identified epigenetic modification, has been extensively implicated in the initiation and progression of tumors. However, the precise role of histone lactylation in modulating tumor cell responses to T cell-mediated cytotoxicity remains insufficiently explored.
 Methods:
 To elucidate the adaptive changes tumor cells undergo in response to T cell-induced cytotoxicity within the tumor microenvironment (TME), we performed a comprehensive, genome-wide CRISPR-Cas9 screen. This approach facilitated the identification of genetic alterations in lung cancer cells co-cultured with cytotoxic T lymphocytes (CTLs) that promote their evasion of T cell-mediated destruction. Complementary RNA sequencing (RNA-seq) and chromatin immunoprecipitation sequencing (ChIP-seq) analyses were employed to investigate the molecular mechanisms by which tumor cells modulate their response to T cell-mediated killing, particularly following the deletion of the monocarboxylate transporter 1 (MCT1).
 Results:
 MCT1 deletion significantly heightened the cancer-cell susceptibility to CTL-mediated killing. Furthermore, the loss of MCT1 led to mitochondrial dysfunction within the tumor cells. Mechanistically, MCT1 knockout resulted in a reduction of histone H3K18 lactylation and downregulation of ferridoxin reductase (FDXR), which disrupted mitochondrial homeostasis and precipitated mitochondrial dysfunction. Notably, the combination of MCT1 inhibition with immune checkpoint blockade (ICB) therapy substantially enhanced ICB efficacy, prolonged survival in both lung cancer and malignant pleural effusion (MPE) murine models, and alleviated the immunosuppressive milieu within the TME.
 Conclusions:
 Our findings reveal a novel role for lactate metabolism in modulating mitochondrial dysfunction and anti-tumor immunity, with lactylation-mediated regulation of mitochondrial homeostasis serving as a critical mechanism. These results provide new insights into the epigenetic regulation of tumorigenesis and highlight potential immune-metabolic strategies for targeted lactate metabolism therapies in the treatment of lung cancer and MPE.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7369949ad290ca8642cb013b1fc65d9e0567a5" target='_blank'>
              MCT1-mediated histone lactylation modulates lung cancer cell sensitivity to T-cell–mediated killing by influencing mitochondrial dysfunction.
              </a>
            </td>
          <td>
            Q. Xue, W. Peng, Qiong Zhou
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Many studies have shown that pancreatic cancer is one of the cancers with extremely high mortality. The poor prognosis and lack of early diagnostic methods remain major challenges in the treatment of this cancer. In this article, in order to better detect the occurrence of pancreatic cancer, tools such as monocle3, singleR, harmony in R, and scanpy in Python were used to analyze the cells and genes of pancreatic cancer tissues in mice. By analyzing the data, the following results were obtained: T-cells in healthy PBMCs exhibited broader spatial dispersion than in PDAC tissues, suggesting tumor-driven immune surveillance impairment, while PDAC-associated macrophages displayed expanded distribution linked to pro-tumorigenic functions such as COL1A1-mediated ECM remodeling; Pseudotemporal trajectory analysis revealed myeloid progenitor bifurcation into monocytes/macrophages, with PDAC macrophages showing epigenetically silenced cytotoxic pathways such as suppressed GZMA/NKG7 and enhanced ribosomal biogenesis; Tissue-specific markers such as LCN2 in healthy and CTRB1/AMY2A in PDAC) and spatial co-localization of macrophages/tumor cells highlighted NOP53 as a dual-function hubinhibiting PI3K-AKT while activating p53and SPP1 as a paradoxical regulator of metastasis and antitumor immunity; Differential expression and GO enrichment analyses identified ribosomal biogenesis and cytoplasmic translation as PDAC-enriched pathways, contrasting with suppressed stress responses. Our spatial transcriptomic profiling further resolved elevated NOP53, CFB, and SPP1 expression gradients in PDAC tissues, proposing these as diagnostic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab94aa8cc2272f4a29c06caf0dcfe1338e28a0a0" target='_blank'>
              Spatiotemporal Transcriptomic Dissection of Tumor-Associated Macrophage Heterogeneity and Dual-Function Molecular Nodes in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Mengfei Li
          </td>
          <td>2025-06-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79b9ccad6349bc919fa38a076a3195bd6f23bab" target='_blank'>
              Loss of colonic fidelity enables multilineage plasticity and metastasis.
              </a>
            </td>
          <td>
            Patrizia Cammareri, Michela Raponi, Yourae Hong, C. Billard, Nat Peckett, Yujia Zhu, Fausto D Velez-Bravo, Nicholas T Younger, Donnchadh S Dunican, S. Ö. Pohl, Aslıhan Bastem Akan, Nora J Doleschall, John Falconer, M. White, Jean Quinn, K. Pennel, Roberta Garau, Sudhir B. Malla, P. Dunne, Richard R Meehan, Owen J. Sansom, J. Edwards, Malcolm G Dunlop, Farhat V N Din, Sabine Tejpar, Colin W. Steele, K. B. Myant
          </td>
          <td>2025-06-04</td>
          <td>Nature</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 e14585


 Background:
 Immune checkpoint blockade (ICB) targeting PD-1/PD-L1 is crucial in treating breast cancer. Despite clinical success, the mechanisms by which ICB therapies reshape the tumor microenvironment (TME) and enhance anti-tumor activity remain unclear. Recent studies reveal that anti-PD-1 therapies broadly alter TME cells beyond PD-1+ T cells, highlighting extensive cell-cell communication (CCC) as a key factor. Understanding CCC changes in response to anti-PD-1 therapy can illuminate its mechanisms of action (MOA) and TME dynamics. This study applies causal inference methodology to investigate CCC between T cells and non-T cells in the TME of breast cancer patients receiving anti-PD-1 therapy.
 Methods:
 We analyzed single-cell RNA-seq data from 31 breast cancer patients pre- and on-treatment with anti-PD-1 (pembrolizumab) reported by Bassez et al., 2021. We identified differentially expressed genes (DEGs) across cell types. We applied the instrumental variable method to identify causal relationships between signals in T-cell and non-T-cell DEGs to uncover treatment-induced CCC. We further searched for ligand-receptor pairs mediating CCC and identified gene expression modules (GEMs) regulated by these ligand-receptors. Finally, we constructed a CCC network from T to non-T cells.
 Results:
 Anti-PD1 treatment induced broad gene expression changes in diverse cell populations. Major T cell pathways influenced by anti-PD-1 therapy include NF-κB, interferon-γ and interleukins. CD4
 +
 and CD8
 +
 exhausted cells, expressing high levels of
 PDCD1
 (encoding PD-1), exhibited distinct activated pathways. We identified the CCC network from T cells to non-T cells via ligand-receptor interactions. For example, anti-PD1 treatment activated cellular stress, apoptosis, and pro-inflammatory cytokine signaling pathways in T cells, altering the expression of a GEM that included
 TSC22D3
 and
 TXNIP
 . This initiated signaling to myeloid cells via
 RPS19
 -
 C5AR1
 , leading to NF-κB activation. CD4+ exhausted T cells primarily signaled to monocytes, enhancing helper functions to recruit and activate monocytes, thereby promoting immune regulation and amplification. In contrast, CD8+ exhausted T cells primarily interacted with macrophages, intensifying cytotoxic responses that facilitated effective tumor-cell killing.
 Conclusions:
 Our analyses provide insights into the dynamic interplay of cells within the TMEs during anti-PD-1 therapy. These findings could facilitate the identification of new biomarkers for predicting heterogeneous treatment responses to anti-PD-1 regimens, potentially enhancing the design and customization of immunotherapeutic strategies for breast cancer patients. Finally, this study demonstrated the utility of causal inference methodologies for mechanistic studies of CCC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74715a58ffb2aed0b55ed5b7dec461d0fe94b624" target='_blank'>
              Use of causal modeling to uncover cell-cell communication dynamics in the tumor microenvironment during anti-PD-1 therapy in breast cancer patients.
              </a>
            </td>
          <td>
            Aodong Qiu, Han Zhang, Joseph Ramsey, Boyang Sun, Bryan Andrews, Kun Zhang, Gregory F. Cooper, Lujia Chen, Xinghua Lu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a major malignancy threatening women’s health, with high-grade squamous intraepithelial lesions playing a critical role in the progression toward CC. Exploring the molecular characteristics of epithelial cells (EPCs) as high-stage intraepithelial neoplasia evolves into CC is essential for the development of effective targeted drugs for cervical cancer. Single-cell RNA sequencing technology can fully understand the immune response at each molecular level, providing new ideas and directions for the precise treatment of CC. Methods Single-cell RNA sequencing was employed to comprehensively map EPCs characteristics. The differentiation trajectory of EPCs was inferred using Slingshot, while enrichment analysis highlighted the biological functions of EPCs. Cellchat visualized cell-cell interactions, and SCENIC was used to infer transcription factor regulatory networks in EPCs. CCK-8, colony formation, and EDU experiments were used to verify cell proliferation changes. Scratch assays and transwell assays were used to verify cell migration and invasion. Results A distinct EPCs subpopulation with high TOP2A expression was identified, predominantly originating from tumor tissues. This subpopulation exhibited disrupted mitosis and cell cycle regulation, along with features of high proliferation, high energy metabolism, and matrix plasticity. It played a key role in shaping the tumor microenvironment via the LAMC1-(ITGA3-ITGB1) signaling pathway. FOXM1, a key transcription factor in this cell subpopulation, significantly inhibited the proliferation and invasion of cervical cancer cells. Conclusion Through in-depth analysis of EPCs, this study provides promising insights and potential therapeutic targets for precision targeted treatment strategies for CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dee7774b4a74cbb3d72e5ad05823a4742cd929c3" target='_blank'>
              Epithelial cells with high TOP2A expression promote cervical cancer progression by regulating the transcription factor FOXM1
              </a>
            </td>
          <td>
            Weisha Sun, Lu Chen, Xiaolin Feng
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Biliary tract cancers (BTCs) are aggressive malignancies encompassing intrahepatic and extrahepatic cholangiocarcinoma, gallbladder carcinoma, and ampullary carcinoma. Here, we report integrative analysis of 63 BTC cell lines via multi-omics and genome-scale CRISPR screens. We identify widespread EGFR dependency in BTC, alongside dependencies selective to anatomic subtypes. Additionally, we delineate strategies to overcome therapeutic resistance, with combined EGFR inhibition potentiating targeting of KRAS-mutant and FGFR2-fusion-driven models, and SHP2 inhibition effective in the latter context. Clustering RNA/protein expression and dependencies data revealed functional relationships transcending single-gene alterations, with biliary, squamous, or dual biliary/hepatocyte lineage signatures stratifying BTC models. These subtypes exhibit distinct dependency profiles- including cell fate transcription factors GRHL2, TP63, and HNF1B, respectively- and demonstrate prognostic significance in patient samples. Potential subtype-specific targetable vulnerabilities include Integrin-a3 and the detoxification enzyme UXS1. This cell line atlas reveals therapeutic targets in molecularly-defined BTCs, unveils disease subtypes, and provides a resource for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/156ff64109bed8af48870c13d716792fd5799d42" target='_blank'>
              Generation of a biliary tract cancer cell line atlas identifies molecular subtypes and therapeutic targets.
              </a>
            </td>
          <td>
            Vindhya Vijay, Negin Karisani, Lei Shi, Yu-Han Hung, Phuong Vu, Prabhat Kattel, Lauren Kenney, J. Merritt, Ramzi Adil, Qibiao Wu, Yuanli Zhen, Robert Morris, J. Kreuzer, Meena Kathiresan, Xcanda Ixchel Herrera Lopez, Haley Ellis, Ilaria Gritti, Lilian Lecorgne, Ines Farag, Alexandra Popa, William Shen, Hiroyuki Kato, Qin Xu, E. Balasooriya, Meng-Ju Wu, Jinkai Wan, Hiroshi Kondo, Saireudee Chaturantabut, Srivatsan Raghavan, Matthew D. Hall, S. Patnaik, Min Shen, R. Kelley, James M Cleary, Michael S Lawrence, D. Root, Krushna C Patra, Vanessa S Silveira, Cyril H. Benes, Vikram Deshpande, D. Juric, William R. Sellers, Cristina R Ferrone, W. Haas, Francisca Vazquez, Gaddy Getz, N. Bardeesy
          </td>
          <td>2025-05-12</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="M2-polarized tumor-associated macrophages (TAMs) are a key factor contributing to the poor prognosis of pancreatic ductal adenocarcinoma (PDAC). While various factors within the tumor microenvironment (TME) drive their formation, the role of PDAC-derived exosomes in this process remains unclear. We aim to clarify the regulatory impacts of tumor-derived exosomes to TAMs. After the intratumoral injection to subcutaneous tumor of C57BL/6 mice, we demonstrated PDAC-derived exosomes exacerbate PDAC progression, accompanied with upregulated M2 phenotype of TAMs and unaffected proliferation signatures. Through intratumoral injection model and multi-Omics analyses, we identified CCT6A as a novel tumor-derived exosomal protein, bridging TAMs M2 polarization and PDAC prognosis. Co-culture with exosomes derived from CCT6Ahigh PDAC leads to greater M2 phenotype of TAMs via PI3K-AKT signaling. According to proteomics data, chemokines’ abundance reduces over tenfold once exosomal CCT6A absence, including CXCL1, CXCL3, CCL20 and CCL5, whose interaction with CCT6A in PDAC cells was confirmed by interactomics data. Moreover, we found silencing CCT6A abrogated the antagonism effects of CD47 antibody immunotherapy. Our findings implied that the subunit of the T-complex protein Ring Complex (TRiC) CCT6A serves as a matchmaker during exosome-mediated chemokines transfer from PDAC to TAMs. Silencing CCT6A effectively sensitized PDAC to CD47 antibody immunotherapy in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a89a81a03a62a72732095044f5f03c32275ff639" target='_blank'>
              Tumor-derived exosomal CCT6A serves as a matchmaker introducing chemokines to tumor-associated macrophages in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Tianyin Ma, W. Guo, Minghe Zhang, Wenzhi He, Cairang Dongzhi, Xiangdong Gongye, Peng Xia, Yibo Chai, Zhang Chen, Yimin Zhu, Chengmin Qu, Jie Liu, Zhiyong Yang, Weijie Ma, Ming‐Hung Tian, Yufeng Yuan
          </td>
          <td>2025-05-15</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
Atherosclerosis involves complex interactions between lipids, immune cells, vascular smooth muscle cells, and fibroblasts within the arterial wall. While significant advances in single-cell technologies have shed light on the roles of immune cells and vascular smooth muscle cells in plaque development, fibroblasts remain underexplored, leaving critical gaps in understanding their contributions to disease progression and plaque stability. Comprehensive characterization of fibroblast phenotypes in atherosclerosis is essential to unravel their diverse functions and to distinguish between subsets that may play protective versus pathogenic roles in the disease process.


METHODS
Here, we utilized cellular indexing of transcriptomes and epitopes by sequencing to comprehensively profile fibroblast diversity in a mouse model of atherosclerosis. Mice were fed an atherogenic diet for 0, 8, 19, and 26 weeks, representing distinct stages of disease progression, enabling a detailed phenotypic characterization of fibroblasts throughout the course of atherosclerosis development.


RESULTS
We identified 4 distinct fibroblast subpopulations, including a myofibroblast population closely resembling vascular smooth muscle cell-derived chondromyocytes. The proportions of these fibroblast subsets exhibited a modest decline as atherosclerosis progressed. Through multimodal analysis, we identified CD26 as a highly expressed and specific marker for one of these fibroblast subpopulations, distinguishing it from other subsets. Using a combination of flow cytometry and immunohistochemistry, we demonstrated that CD26+ fibroblasts predominantly reside in the adventitia of healthy arteries. During atherosclerosis progression, these cells expand into the intima and primarily localize within the fibrous cap of the lesion.


CONCLUSIONS
Our multiomic analysis highlights the phenotypic diversity and dynamic changes of fibroblasts during atherosclerosis progression. Among these, CD26+ fibroblasts emerge as a distinct subpopulation that expands within atherosclerotic lesions and may play a critical role in promoting plaque stability through their migration into the fibrous cap.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af796b09d1dc7b9ff103a4aea099fa62c9a6120" target='_blank'>
              Single-Cell Multimodal Profiling Reveals a Novel CD26+ Fibroblast Subpopulation in Atherosclerosis.
              </a>
            </td>
          <td>
            A. Bashore, Johana Coronel, Chenyi Xue, Lucie Y. Zhu, Muredach P. Reilly
          </td>
          <td>2025-06-12</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ovarian cancer (OC) is a leading cause of death of gynecological cancers in women. Poor patient response to treatment highlights the need to better understand how the tumor microenvironment affects OC progression. Growing evidence indicates the crucial role of non-cancerous components, such as cancer-associated fibroblasts, in establishing a complex network of cellular and molecular interactions, influencing cancer progression and response to treatment. Therefore, in this study, we sought to characterize the impact of fibroblasts on OC cell behavior and drug response. Using both direct and indirect cell co-culture systems, we observed distinct changes in cancer cell proliferation, morphology, and secretome in the presence of fibroblasts. Furthermore, an imaging-based high-throughput drug screen of 528 oncology compounds revealed multiple drugs that showed altered efficacy in the co-culture conditions, demonstrating the role of fibroblasts in driving cancer cell resistance to treatment. Most importantly, our data identified the two drug combinations of Birinapant or Vorinostat with Carboplatin as promising treatments, exploiting the altered cancer cell phenotype in co-cultures. These findings were supported by the increased sensitivity of ex vivo cultures to these combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec16a3d9c76cc2ee8b3abe1252f39e100b5c4acc" target='_blank'>
              Systematic profiling of cancer-fibroblast interactions reveals drug combinations in ovarian cancer.
              </a>
            </td>
          <td>
            Greta Gudoityte, Osheen Sharma, Laura Leuenberger, E. Wallin, Josefin Fernebro, P. Östling, Rebecka Bergström, Johan Lindberg, U. Joneborg, Olli-P. Kallioniemi, B. Seashore-Ludlow
          </td>
          <td>2025-05-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The evolution of isocitrate dehydrogenase (IDH)-wildtype glioblastoma (GBM) after standard-of-care therapy remains poorly understood. Here we analyzed matched primary and recurrent GBMs from 59 patients using single-nucleus RNA sequencing and bulk DNA sequencing, assessing the longitudinal evolution of the GBM ecosystem across layers of cellular and molecular heterogeneity. The most consistent change was a lower malignant cell fraction at recurrence and a reciprocal increase in glial and neuronal cell types in the tumor microenvironment (TME). The predominant malignant cell state differed between most matched pairs, but no states were exclusive or highly enriched in either time point, nor was there a consistent longitudinal trajectory across the cohort. Nevertheless, specific trajectories were enriched in subsets of patients. Changes in malignant state abundances mirrored changes in TME composition and baseline profiles, reflecting the co-evolution of the GBM ecosystem. Our study provides a blueprint of GBM’s diverse longitudinal trajectories and highlights the treatment and TME modifiers that shape them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0990e85585d1a285c19cfc504ce05c954e344b" target='_blank'>
              Deciphering the longitudinal trajectories of glioblastoma ecosystems by integrative single-cell genomics
              </a>
            </td>
          <td>
            Avishay Spitzer, K. Johnson, M. Nomura, L. Garofano, Djamel Nehar-Belaid, Noam Galili Darnell, Alissa C. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, Simona Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, Jong Bae Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, Simone P Niclou, Labeeba Nusrat, Sorcha Kellet, Sunit Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, Marc Sanson, S. Tanaka, Nobuhito Saito, D.M. Ashley, Stephen T Keir, Keith L. Ligon, Jason T Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-05-09</td>
          <td>Nature Genetics</td>
          <td>5</td>
          <td>62</td>
        </tr>

        <tr id="Abrogating the immunoevasive role of the tumor immune microenvironment (TIME) represents a critical yet still elusive challenge in cancer treatment. Progress in this area has been hampered by both technological limitations and incomplete understanding of TIME-dependent immunoevasion mechanisms. We hypothesize that the immune-evasive role of TIME subpopulations—including regulatory T cells, cancer-associated fibroblasts, and tumor-associated macrophages—is critically mediated by hyperconnected Master Regulator Checkpoint (MRC) modules whose aberrant activity, as induced by paracrine signals, can be abrogated or modulated either genetically or pharmacologically. MRCs are primarily composed of transcription and co-transcription factors, acting downstream of surface receptors and signal transduction cascades to control the transcriptional identity and, ultimately, the phenotype of individual TIME subpopulations. Pharmacological inhibition of subpopulation-specific MRC proteins can thus help reprogram the TIME and potentially abrogate or modulate its immunosuppressive state. This paradigm shift, away from single ligand/receptor targeting, is supported by recent algorithmic, experimental, and clinical advances allowing systematic identification of MRCs and their pharmacological modulators using systems immunology-based approaches. Refocusing the deployment of existing tools and experimental methods that have proven successful in tumor cell contexts to identify and validate MRC-targeting agents capable of remodeling the immunosuppressive cell states of the tumor microenvironment can potentially pave the road to novel combination therapy synergizing with immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/992372e5de760c711f403d3ef7aa2e616073ce3d" target='_blank'>
              Systematic identification and targeting of master regulator checkpoints (MRC) governing tumor microenvironment-mediated immune evasion
              </a>
            </td>
          <td>
            Pasquale Laise, G. Bosker, Mariana Babor, Xiaoyun Sun, Stuart Andrews, Lorenzo Tomassoni, Andrea Califano, Mariano J. Alvarez
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Ovarian cancer is one of the most common malignancies of the female reproductive system and is associated with poor prognosis. This study aimed to utilize single-cell RNA sequencing to investigate the heterogeneity of malignant epithelial cells in ovarian cancer, focusing on their potential functions and the implications for treatment and prognosis. Methods Single-cell RNA sequencing data were clustered using a single-cell transcriptome clustering method, and malignant epithelial cells were identified through copy number variation analysis. The interaction patterns between different malignant subpopulations and immune/stromal cells were analyzed using cell-to-cell communication analysis. A risk score (URS) model based on the UBE2C + epithelial subpopulation was then constructed through LASSO and multivariable Cox regression. High and low URS groups were compared in terms of tumor mutational burden (TMB), survival outcomes, and drug sensitivity. Finally, the role of GTF2F2 in ovarian cancer progression was validated through gene knockdown experiments in an ovarian cancer cell line (ES-2). Results Three major malignant epithelial cell subpopulations were identified (TMSB4X + Epi, TSC22D1 + Epi, and UBE2C + Epi). The UBE2C + Epi subpopulation exhibited higher stemness and greater invasive potential. The constructed URS model effectively stratified patients into high- and low-risk groups, with the high-risk group displaying a higher TMB level (p = 0.00011). Drug sensitivity predictions indicated that osimertinib, rapamycin, and dihydrorotenone might have stronger inhibitory effects in the high-risk group, whereas ERK inhibitors were more effective in the low-risk group. Functional assays demonstrated that GTF2F2 knockdown significantly suppressed ovarian cancer cell migration and invasion. Western blot analyses further showed elevated E-cadherin and reduced N-cadherin expression, suggesting that GTF2F2 may promote epithelial-mesenchymal transition (EMT). Conclusion The risk score model established in this study offers a novel framework for patient stratification and personalized therapy. Notably, the identification of the UBE2C + Epi subpopulation and key genes such as GTF2F2 highlights potential diagnostic and therapeutic targets, shedding light on the pathogenesis of ovarian cancer and paving the way for precision medicine approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01686-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519b55d3039bd908fc634a743e91a97b194c1fa0" target='_blank'>
              Single-cell RNA sequencing reveals the role of GTF2F2 in ovarian cancer oncogenesis and progression
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-05-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Radiotherapy resistance represents a critical aspect of cancer treatment, and molecular characterization is needed to explore the pathways and mechanisms involved. DNA repair, hypoxia, metabolic reprogramming, apoptosis, tumor microenvironment modulation, and activation of cancer stem cells are the primary mechanisms that regulate radioresistance, and understanding their complex interactions is essential for planning the correct therapeutic strategy. Proteomics has emerged as a key approach in precision medicine to study tumor heterogeneity and treatment response in cancer patients. The integration of mass spectrometry-based techniques with bioinformatics has enabled high-throughput, quantitative analyses to identify biomarkers, pathways, and new potential therapeutic targets. This review highlights recent advances in proteomic technologies and their application in identifying biomarkers predictive of radiosensitivity and radioresistance in different tumors, including head and neck, breast, lung, and prostate cancers. Sample variability, data interpretation, and the translation of findings into clinical practice remain challenging elements of proteomics. However, technological advancements support its application in a wide range of topics, allowing a comprehensive approach to radiobiology, which helps overcome radiation resistance. Ultimately, incorporating proteomics into the radiotherapy workflow offers significant potential for enhancing treatment efficacy, minimizing toxicity, and guiding precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c57bb12b0c55c3df25a4ed39ad02e8ef8a7a5810" target='_blank'>
              Deciphering Radiotherapy Resistance: A Proteomic Perspective
              </a>
            </td>
          <td>
            D. Perico, P. Mauri
          </td>
          <td>2025-06-01</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metabolic dysfunction-associated steatotic liver disease (MASLD) has limited treatments, and cell type-specific regulatory networks driving MASLD represent therapeutic avenues. We assayed five transcriptomic and epigenomic modalities in 2.4M cells from 86 livers across MASLD stages. Integrating modalities increased annotation of the genome in liver cell types several-fold over previous catalogs. We identified cell type regulatory networks of MASLD progression, including distinct hepatocyte networks driving MASL and mild and severe fibrosis MASH. Our single cell atlas annotated 88% of MASH-associated loci, including a third affecting hepatocyte regulation which we linked to distal target genes. Finally, we characterized hepatocyte heterogeneity, including MASH-enriched populations with altered repression, localization, and signaling. Overall, our results provide high-resolution maps of liver cell types and revealed novel targets for anti-MASH therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7d8151962524268bafd46f3bbc5c2abb6789115" target='_blank'>
              Single cell multiomics reveals drivers of metabolic dysfunction-associated steatohepatitis
              </a>
            </td>
          <td>
            Weston S Elison, Lei Chang, Yang Xie, Charlene Miciano, Qian Yang, Hannah M Mummey, Ryan Lancione, Sierra Corban, Sadatsugu Sakane, J. Lucero, S. Mamde, Hyun Young Kim, Matthew J Kim, Rebecca L. Melton, Luca Tucciarone, Audrey Lie, Timothy Loe, Tanmayi Vashist, Kelsey Dang, R. Elgamal, Daofeng Li, Melissa Vu, Elie N. Farah, Chad Seng, Jovina Djulamsah, Bing Yang, Justin Buchanan, Michael Miller, Mai Tran, J. Birrueta, Neil C Chi, Ting Wang, Agnieszka D’Antonio-Chronowska, Allen Wang, T. Kisseleva, David A. Brenner, Bing Ren, K. J. Gaulton
          </td>
          <td>2025-05-11</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f23b04434bb73a39ffc142be7c6ab3896c53a0" target='_blank'>
              Longitudinal single cell RNA-sequencing reveals evolution of micro- and macro-states in chronic myeloid leukemia
              </a>
            </td>
          <td>
            D. Frankhouser, Dandan Zhao, Yu-Hsuan Fu, Anupam Dey, Ziang Chen, Jihyun Irizarry, Jennifer Rangel Ambriz, S. Branciamore, Denis O’Meally, Lucy Ghoda, Jeffery M. Trent, Stephen J Forman, Adam L. Maclean, Ya-Huei Kuo, Bin Zhang, Russell C. Rockne, Guido Marcucci
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is the complex ecosystem surrounding a tumor. It is a complex and continuously evolving entity. The TME comprises a variety of complex components, such as cancer cells, immune cells, stromal cells, and extracellular matrix, that precisely regulate the interaction of tumor cells with other components, allowing tumor cells to continue to metastasize, escape immune surveillance, resist apoptosis, and proliferate. Researchers believe that the TME is not just a silent bystander but instead an active promoter of cancer progression. At the onset of tumor growth, a dynamic and reciprocal relationship is developed between cancer cells and components of the tumor microenvironment; this relationship supports cancer cell survival, local invasion, and metastatic dissemination. In this article, we discuss the role of the various components of the TME in the progression and metastasis of cancer, the pathways influenced by the TME, and the drugs that can treat tumors by targeting the components of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9869eb44a45ce4947a2fc2cac8a08a25b3dc82" target='_blank'>
              The Role of the Tumor Microenvironment in Cancer Progression and Metastasis
              </a>
            </td>
          <td>
            Oseghale, Ikalo David, Ikokwu, Godwin Mmaduabuchi, Ugoagwu, Kingsley Ugonna, Elizabeth Anuoluwa Akintayo, Matthew Gyang, Okiki Nosiru Quadri, Catherine Rono, O. Ralph-Okhiria, Odunvbun Praise
          </td>
          <td>2025-05-29</td>
          <td>Journal of Cancer and Tumor International</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="
 e20113


 Background:
 Small cell lung cancer (SCLC) is a highly lethal malignancy, and there is an urgent need for more effective therapies. Antibody-drug conjugates (ADCs), which combine the precision of monoclonal antibodies with the cytotoxic power of small molecules, represent a promising approach. Despite extensive research, the relationship between ADC target overexpression and the transcriptional profile of SCLC remains unclear. Our study aimed to elucidate how ADC target expression correlates with transcriptional subtypes of SCLC, potentially guiding the selection of ADCs based on molecular profiling.
 Methods:
 RNA-seq and tandem mass spectrometry data for 107 patients with matched SCLC tumors and normal adjacent lung tissue were obtained from the CPTAC SCLC cohort. SCLC subtypes were defined according to expression of four transcription factors: achaete-scute homologue 1 (ASCL1), neurogenic differentiation factor 1 (NeuroD1), yes-associated protein 1 (YAP1) and POU class 2 homeobox 3 (POU2F3). ADC target expression for DLL3, SEZ6, B7-H3 (
 CD276
 ), and Trop2 (
 TACSTD2
 ) were assessed at the transcriptomic and proteomic level. The correlation between transcriptional subtype and ADC target expression was determined using Pearson’s correlation coefficient.
 Results:
 In 107 SCLC tumors, subtypes corresponded to 66 (61.6%) ASCL1, 21 (19.6%) NeuroD1, 3 (2.8%) YAP1, and 17 (15.9%) POU2F3. The ASCL1 subtype was positively correlated with DLL3 (R
 2
 =0.93, p=9E-87) and SEZ6 (R
 2
 =0.85, p=2.9E-57), and negatively correlated with TROP2 expression (R
 2
 =-0.72, p=1.5E-34). The NEUROD1 subtype showed moderate positive correlation with DLL3 (R
 2
 =0.52, p=4.8E-14) and SEZ6 (R
 2
 =0.58, p=3.6E-18), and a moderate negative correlation with TROP2 (R
 2
 =-0.52, p=1.6E-14). The POU2F3 subtype did not demonstrate strong positive or negative correlations with ADC targets. The YAP1 subtype was negatively correlated DLL3 (R
 2
 =-0.72, p=1.1E-32) and SEZ6 (R
 2
 =-0.71, p=9.5E-33), and positively correlated with Trop2 expression (R
 2
 =0.85, P=7.5E-61).
 Conclusions:
 Our findings reveal that molecular subtypes of SCLC exhibit unique patterns of ADC target expression. These profiles are crucial for further research and could lead to the development of subtype-specific treatments that improve clinical outcomes for patients with SCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34167c0dda56aaaeb23c8d98646f057387899f8a" target='_blank'>
              Proteogenomic characterization of antibody-drug conjugate target expression profiles in transcriptional subtypes of small cell lung cancer.
              </a>
            </td>
          <td>
            Edwin Lin, Mitch Singstock, Sanad Alhushki, Sameer S. Deshmukh, Changde Cheng, Kayla F. Goliwas, J. Deshane, A. R. Naqash, Fawzi Abu Rous, Rajat Thawani, Aakash Desai
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="BACKGROUND AND OBJECTIVES
Field cancerisation is the process that results in a group of cells acquiring some of the phenotypic changes of cancer prior to transformation into cancer. Clinically, an important challenge remains the ability to distinguish clonal lineages and microenvironments within cancerised fields that will remain indolent from those that will progress to malignant transformation. Spatial 'omics' can help us investigate genetic, epigenetic, transcriptomic, proteomic, and cellular microenvironments that transform normal cells into a cancerised field, and subsequently into cancer. In this review, we will discuss how spatial omics techniques have expanded our understanding of field cancerisation in prostate and bladder cancer, and the challenges associated with this research.


METHODS
We identified key articles relating to field cancerisation in bladder and prostate cancer. Special emphasis was placed on studies that used modern spatial profiling technologies and studies that were designed to investigate changes within normal tissue rather than simply using it as a control for tumour tissue.


RESULTS
Spatial omics research into field cancerisation has identified interesting early findings that have informed our understanding of: transformation of the benign epithelium and mechanisms of intra-prostatic clonal expansion for prostate cancer; clonal expansion within the normal urothelium; mutations that are unique to cancerised fields within the bladder; and how field cancerisation may prime the urothelium for cancer transformation.


CONCLUSIONS
Spatial omics profiling of field cancerisation can inform risk stratification and personalised treatment options. However, there are a number of challenges associated with the technologies that must be overcome before the potential of spatial omics can be fully realised in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957e9305f80e5b98aef1432b61f0ded66b1ee007" target='_blank'>
              What spatial omics is teaching us about field cancerisation in prostate and bladder cancer.
              </a>
            </td>
          <td>
            Matthew H V Byrne, T. Anbarasan, Lisa Browning, Dan Woodcock
          </td>
          <td>2025-06-25</td>
          <td>BJU international</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Reactivation of transposable elements (TEs) in somatic tissues, particularly of LINE-1, is associated with disease by causing gene mutations and DNA damage. Previous work has shown that the PIWI pathway is crucial for TE suppression in the germline. However, the status and function of this pathway has not been well characterized in differentiated somatic cells and there is lack of consensus on the role of the pathway in somatic tumorigenesis. To shed light on this conundrum, we examined the PIWI pathway in colon cancer through a combination of bioinformatic analyses and cell-based assays. Shifted Weighted Annotation Network (SWAN) analysis revealed that the pathway experiences significant allelic losses in colon cancer and that PIWIL2, the main catalytic component of the pathway responsible for TE silencing, experiences the highest percent deletions. PIWIL2 is downregulated in colon tumors of advanced stage, nodal metastasis, and in certain subtypes, correlating with poor survival, while it is even downregulated in ulcerative colitis, an inflammatory bowel disease that predisposes to colon cancer. Knockout studies in colon epithelial Caco2 cells show that PIWIL2 depletion leads to increased anchorage-independent growth, increased LINE-1 levels and activity, and in DNA damage, altogether highlighting a tumor-suppressing role of PIWIL2 in the colon. SUMMARY STATEMENT This study investigates the PIWI-piRNA pathway in colon cancer using a nuanced bioinformatic and cell-based approach, linking the downregulation of PIWIL2 to disease progression, transposable element activation and DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6553455e0dd4bcf44406169f133238204316d7c1" target='_blank'>
              PIWIL2 downregulation in colon cancer promotes transposon activity and pro-tumorigenic phenotypes
              </a>
            </td>
          <td>
            Alyssa Risner, J. Nair-Menon, Abhinav Cheedipudi, Joe R Delaney, Vamsi K Gangaraju, A. Kourtidis
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Compelling evidences have manifested that breast cancer cells prefer to metastasize to certain distant organs, including brain, lung, bone and liver. According to the canonical “seed and soil” theory, this prominent biological behavior, termed as metastatic organotropism, involves intricate interactions between breast cancer cells (the “seeds”) and specific residents in the tumor microenvironment (the “soil”), initiating from pre-metastatic niche formation to metastatic outgrowth. Recently, multifaceted heterogeneity of tissue-specific macrophages (TSMs) and their roles played in organotropic metastases of breast cancer are incrementally unveiled. Herein, we decipher multiple diversities of TSMs, including evolvement, profiles, functions and metabolic characteristics under different polarization states. Further, we elaborate on bidirectional effects of TSMs on metastatic organotropism of breast cancer (both to the “seeds” and “soil”), and unearth underlying signaling pathways based on updated mechanistic researches. Lastly, we compile a series of clinical trials, hoping to illuminate promising TSM-targeting therapies against breast cancer organotropic metastases. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ed79b9316a9fed3f14b3bb67e4e61ce4c40709" target='_blank'>
              Heterogeneous tissue-specific macrophages orchestrate metastatic organotropism of breast cancer: implications for promising therapeutics
              </a>
            </td>
          <td>
            Cenzhu Wang, Pinchao Fan, Yu Zhou, Meican Ma, Haowei Zhong, Lei Liu, Qin Chen, Kun Xu
          </td>
          <td>2025-06-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy, a hallmark of cancer, leads to widespread changes in chromosome copy number, altering the abundance of hundreds or thousands of proteins. How-ever, evidence suggests that levels of proteins encoded on affected chromosomes are often buffered toward their abundances observed in diploid cells. Despite its preval-ence, the molecular mechanisms driving this protein dosage compensation remain largely unknown. It is unclear whether all proteins are buffered to a similar degree, what factors determine buffering, and whether dosage compensation varies across different cell lines or tumor types. Moreover, its potential adaptive advantage and therapeutic relevance remain unexplored. Here, we established a novel approach to quantify protein dosage buffering in a gene copy number-dependent manner, show-ing that dosage compensation is widespread but variable in cancer cell lines and in vivo tumor samples. By developing multifactorial machine learning models, we identify mean gene dependency, protein complex participation, haploinsufficiency, and mRNA decay as key predictors of buffering. We also show that dosage com-pensation can affect oncogenic potential and that higher buffering correlates with reduced proteotoxic stress and increased drug resistance. These findings highlight protein dosage compensation as a crucial regulatory mechanism and a potential therapeutic target in aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63d2bd35509beaab88a25ac6ce826723195385e" target='_blank'>
              Explainable Machine Learning Identifies Factors for Dosage Compensation in Aneuploid Human Cancer Cells
              </a>
            </td>
          <td>
            Erik Marcel Heller, Karen Barthel, M. Räschle, Klaske M. Schukken, J. Sheltzer, Zuzana Storchová
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) have been identified as important mediators of cancer metastasis, especially in the establishment of organ-specific metastatic niches. These membranous vesicles secreted by tumor cells release diverse bioactive cargo, including proteins, nucleic acids, and lipids, thereby allowing for intercellular communication and microenvironment modulation. Recent evidence demonstrates that EVs can also contribute to the formation of pre-metastatic niches by reprogramming immune cells, modifying the stromal environment, and inducing epithelial-mesenchymal transition (EMT) to promote metastatic colonization. In this review, we describe the molecular mechanism of organotropic metastasis orchestrated by EVs, with special emphasis on immune modulation and tumor microenvironment reprogramming. We also explore the potential of EVs as biomarkers for early detection of metastasis and as potential therapeutic targets for combating metastatic progression. Dissociating EV species and their influence on tumor dissemination will undoubtedly pave the way for implementing novel anti-cancer strategies to intercept tumor dissemination at its very early stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33188b75e18abb76a4279a05b9e5d1c239a98267" target='_blank'>
              Beyond boundaries: exploring the role of extracellular vesicles in organ-specific metastasis in solid tumors
              </a>
            </td>
          <td>
            Goutham Nidhi, Vikas Yadav, Tejveer Singh, Deepika Sharma, Monica Bohot, S. Satapathy
          </td>
          <td>2025-06-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) has a poor survival rate due to late detection. PDAC arises from precursor microscopic lesions, termed pancreatic intraepithelial neoplasia (PanIN), that develop at least a decade before overt disease--this provides an opportunity to intercept PanIN-to-PDAC progression. However, immune interception strategies require full understanding of PanIN and PDAC cellular architecture. Surgical specimens containing PanIN and PDAC lesions from a unique cohort of five treatment-naïve patients with PDAC were surveyed using spatial-omics (proteomic and transcriptomic). Findings were corroborated by spatial proteomics of PanIN and PDAC from tamoxifen-inducible KPC (tiKPC) mice. We uncovered the organization of lymphoid cells into tertiary lymphoid structures (TLSs) adjacent to PanIN lesions. These TLSs lacked CD21+CD23+ B cells compared to more mature TLSs near the PDAC border. PanINs harbored mostly CD4+ T cells with fewer Tregs and exhausted T cells than PDAC. Peri-tumoral space was enriched with naïve CD4+ and central memory T cells. These observations highlight the opportunity to modulate the immune microenvironment in PanINs before immune exclusion and immunosuppression emerge during progression into PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6b6b93e0faedffa7b598bebfd58275c143f057" target='_blank'>
              Spatial Proteomics and Transcriptomics Reveal Early Immune Cell Organization in Pancreatic Intraepithelial Neoplasia.
              </a>
            </td>
          <td>
            Melissa R Lyman, Jacob T. Mitchell, Sidharth Raghavan, Luciane T. Kagohara, Amanda L. Huff, Saurav D Haldar, Sarah M. Shin, Samantha Guinn, Benjamin Barrett, Gabriella Longway, Alexei Hernandez, Erin M Coyne, Xuan Yuan, Lalitya Andaloori, Jiaying Lai, Yun Zhou Liu, Rachel Karchin, Anuj Gupta, A. Kiemen, André Forjaz, Denis Wirtz, Pei-Hsun Wu, Atul Deshpande, Jae W. Lee, T. Armstrong, Nilofer S. Azad, Jacquelyn W. Zimmerman, Laura D. Wood, Robert A Anders, Elizabeth D. Thompson, Elizabeth M. Jaffee, E. Fertig, W. Ho, N. Zaidi
          </td>
          <td>2025-06-26</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36016009dd0f8056e799887b3ce94bc9a6296493" target='_blank'>
              Ovarian cancer, neutrophil hitchhiking, and NETs: unraveling their role in pathogenesis and management.
              </a>
            </td>
          <td>
            Shahrzad Amirlatifi, Shabnam Forouzin, Elahe Sadati, Somayeh Moradpanah, Ramina Fazeli, Paria Panahinia, Nasim Zarifi, Sania Rahimi
          </td>
          <td>2025-06-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e20009


 Background:
 Osimertinib resistance in patients with EGFR-driven non-small cell lung cancer (NSCLC) contributes to mortality. Targeting bypass alterations have had limited efficacy. Here, we explored the role of CD105 (endoglin) expression, a co-receptor for bone morphogenetic protein (BMP) signaling as a factor of osimertinib treatment.
 Methods:
 Sixty-six NSCLC tumors, 44 of which had EGFR mutations were subjected to imaging mass cytometry to identify CD105 expression status as a predictor of clinical outcomes. To study the effects of EGFR and CD105 disruption two osimertinib-resistant EGFR-mutant cell lines were developed for subsequent characterization by single cell RNA sequencing. The promoter chromatin status and glycolytic state of the lines were evaluated by ATAC sequencing and Single Cell ENergetIc metabolism by profiling Translation inHibition analysis (SCENITH), respectively.
 Results:
 There was an inverse correlation between the expression CD105 and overall survival in treatment-naïve patients (
 P
 = 0.0071). Modeling osimertinib resistance highlighted the emergence of a persister cell population expressing tumor dormancy features. Further, the persister cells were found to have elevated CD105 levels with responsivity to the neutralizing antibody, carotuximab (ENV105), in restoring epigenetic and metabolic reprogramming to support osimertinib sensitivity. The carotuximab mechanism of action was through the re-engagement of EGFR signaling by potentiating physical CD105/EGFR interaction. Combination of osimertinib and carotuximab significantly reduced tumor growth in two independent NSCLC mouse models (
 P
 = 0.0056 and
 P =
 0.0008), where either single agent had little benefit.
 Conclusions:
 CD105 targeting provides a novel mutation-independent strategy to overcome resistance in EGFR-mutant NSCLC. Based on the efficacy of carotuximab capacity to limit tumor dormancy and restored osimertinib sensitivity, a Phase 1 trial (NCT05401110) investigating the combination therapy is currently accruing subjects.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/403b9eb716a17d3e6aa13d9c70998a95aef3047d" target='_blank'>
              Effect of carotuximab (ENV105)-mediated CD105 blockade on osimertinib sensitivity in EGFR-mutant non-small cell lung cancer persister cells.
              </a>
            </td>
          <td>
            Neil A. Bhowmick, Sandrine Billet, B. Waissengrin, K. Sankar, E. Posadas, Sungyong You, Jean Lopategui, Karen L. Reckamp, Manish Thiruvalluvan
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83fa54668a1bde6cb09830187a5dd62f51b3deb5" target='_blank'>
              Glioblastoma shift from bulk to infiltrative growth is guided by plexin-B2-mediated microglia alignment in invasive niches.
              </a>
            </td>
          <td>
            Sangjo Kang, Mary E. Ughetta, Jack Y Zhang, Valerie Marallano, Anirudh Sattiraju, Theodore Hannah, Shalaka Wahane, Aarthi Ramakrishnan, Molly E. Estill, N. Tsankova, Li Shen, A. Tsankov, R. Friedel, Hongyan Zou
          </td>
          <td>2025-05-29</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 e16171


 Background:
 Hepatocellular carcinoma (HCC), accounting for 70%-90% of primary liver cancers, often involves portal vein tumor thrombus (PVTT), a significant adverse prognostic factor linked to worsened liver function and treatment challenges. This study uses multi-omics analysis to investigate the tumor immune microenvironment in HCC with PVTT, focusing on the role of cancer-associated fibroblasts (CAFs) in PVTT formation and identifying potential therapeutic targets to improve clinical outcomes.
 Methods:
 We utilized nCounter technology, single-cell RNA sequencing (scRNA-seq), digital spatial profiling (DSP), and Olink proteomics to investigate intra-tumoral spatial heterogeneity from transcriptomics, proteomics, and immunology perspectives. Key molecules promoting PVTT formation were identified via differentially expressed genes (DEGs) analysis, spatial expression and single-cell analysis. Identified Acarbose as a potential drug for treating PVTT was conducted using 3D QSAR, pharmacophore modeling, and molecular dynamics simulations. The clinical potential of Acarbose targeting NID1 in PVTT treatment was further validated through cell proliferation assays, immune-functional mouse models, immunofluorescence, and real-world clinical data analysis.
 Results:
 A comprehensive multi-omics analysis of 85 samples from 47 HCC patients revealed that myofibroblast-associated cells (myCAFs) dominate the tumor microenvironment (TME) of PVTT, shaping a tumor immune-suppressive environment. The macrophage-to-myofibroblast transition (MMT) induced myCAF formation, establishing an immune barrier in the primary tumor. A cross-analysis of DEGs from DSP data and scRNA-seq identified Nidogen1 (NID1) as a key gene driving PVTT formation. NID1 was upregulated in MMT cells and myCAFs within the PVTT TME, contributing to immunosuppression and PVTT formation. Computational virtual screening identified acarbose as a potential therapeutic agent. When combined with tislelizumab, acarbose significantly enhanced T cell-mediated tumor cytotoxicity, inhibited tumor growth, and reduced NID1 expression, demonstrating superior efficacy compared to anti-PD-1 monotherapy, both in vitro and in vivo. Real-world data showed that long-term use of acarbose in diabetic HCC patients reduced NID1 levels and prevented PVTT formation, supporting acarbose as a promising strategy to target NID1 and inhibit PVTT.
 Conclusions:
 Our findings demonstrate the critical role of myCAFs in shaping the tumor immune-suppressive environment that promotes PVTT formation. Targeting NID1, a key driver of this process, with acarbose effectively disrupts the myCAF-mediated immune barrier and synergistically enhances the efficacy of immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e255ac4af9555248e208900e14562fa04e3faced" target='_blank'>
              Multi-omics investigation for the tumor immune microenvironment and spatial heterogeneity in hepatocellular carcinoma with PVTT: MyCAFs and NID1 in immune evasion.
              </a>
            </td>
          <td>
            Tao Han, Zhi Zhu, Heming Li, Kai Li
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The intricate tumor microenvironment (TME) poses a significant barrier to effective cancer immunotherapy, requiring innovative strategies to model and address its challenges. Traditional models, such as 2D cultures and animal studies, often fail to capture the TME's dynamic, multicellular, and spatially complex nature, limiting their predictive power for therapeutic outcomes. To overcome these limitations, this review examines innovative microphysiological systems (MPS) that enhance the understanding of tumor‐immune interactions and pave the way for more effective immunotherapeutic strategies. First the complex features of the TME and its key players are detailed, elaborating on their dynamic interplay with tumors. Importantly, it is highlighted how these components contribute to treatment resistance, offering crucial insights into therapeutic failures. Then, state‐of‐the‐art 3D in vitro organ‐on‐chip (OoC) models are presented that faithfully recapitulate the TME, incorporating patient‐derived tumors to enhance clinical relevance. These advanced systems not only overcome the limitations of traditional animal models and 2D cultures but also provide a robust platform for assessing and improving immunotherapeutic regimens. By bridging the gap between bench and bedside, MPS promises to accelerate the development of novel, more effective immunotherapies for solid tumors, potentially transforming cancer treatment in the near future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6697512c635b75cb87b2b32aa131010d9c074648" target='_blank'>
              Microphysiological Systems in Cancer Research: Advancing Immunotherapy through Tumor Microenvironment‐Integrated Organ‐On‐Chip Models
              </a>
            </td>
          <td>
            Fabiana Moresi, Francesco Noto, Jyothsna Vasudevan, Xavier Bisteau, Giulia Adriani, Andrea Pavesi
          </td>
          <td>2025-07-04</td>
          <td>Advanced Therapeutics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cervical cancer (CC) is the fourth commonest malignant tumor among women worldwide and is characterized by high heterogeneity and a complex ecosystem. A comprehensive understanding of the heterogeneity of tumors and the tumor microenvironment (TME) is crucial for effective CC management. Single-cell RNA sequencing (scRNA-seq) is a powerful tool that can be employed to unveil the heterogeneity of tumors and the TME, as well as to elucidate the evolutionary trajectories of tumors. Spatial transcriptomics (ST) technology, on the other hand, can address the complexity and diversity of the spatial microenvironment of tumors, thereby compensating for the limitations of scRNA-seq. As emerging technologies, both scRNA-seq and ST are increasingly being utilized in CC research. In this review, we summarized the latest advancements in scRNA-seq and ST for CC, with a focus on investigating tumor heterogeneity, the TME, tumor evolutionary trajectories, treatment resistance mechanisms, and potential therapeutic targets. These insights collectively contribute to the development of more effective treatment and prevention strategies for CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f5b3526e206bb4dcf755c55607c83c389e4ccb" target='_blank'>
              Applications and Prospects of Single-Cell RNA Sequencing and Spatial Transcriptomics in Cervical Cancer
              </a>
            </td>
          <td>
            Yifu Wang, Li Yang, Yunzhi Liu, Huangrong Ma, Miaoying Cai, Chunyu Liang, Li Zhang, Zezhuo Su, Zhiyuan Xu
          </td>
          <td>2025-06-26</td>
          <td>BioMed Research International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Stem cell plasticity is crucial for rapidly proliferating tissues, such as the intestine, to cope with microenvironmental stress and maintain homeostasis. This framework may be employed by cancer stem cells (CSCs) to interact with the tumor microenvironment (TME) and give rise to drug resistance, tumor recurrence, and metastasis. Here, we consider colorectal cancer (CRC) as an example to review the technological and conceptual advances in microenvironmental regulation of cancer stemness. Some unique features of CRC, including reprogramming of stromal cells, bacterial niche alteration, and metabolic interplay, were summarized to illustrate how the TME affects the plasticity of CSCs. The key pathways and molecular mechanisms involved in metastasis and drug resistance are reviewed to obtain a deeper understanding of how CSCs may be manipulated for better outcomes in CRC treatment. These findings advance our knowledge of tumor heterogeneity and may provide insights into the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e85fc5484d5fb7be8f2f1af589f1c3b9e327f305" target='_blank'>
              Microenvironmental Regulation of Colorectal Cancer Stemness: Technological Advances, Molecular Basis, and Therapeutic Targets
              </a>
            </td>
          <td>
            Hui Zhao, Qilin Li, Jichao Qin
          </td>
          <td>2025-06-26</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e16331


 Background:
 Cholangiocarcinoma (CCA) describes a malignancy originating from the epithelial cells of the bile ducts. While rare, this malignancy often has an unfortunate outlook given its advanced stage at presentation. Development of CCA likely depends upon a gradual accumulation of genetic mutations. These mutations in-turn, transition precursor lesions, biliary intraepithelial neoplasia and intraductal papillary mucinous neoplasm, into the cancer that is CCA. The tumor microenvironment has been increasingly studied and likely plays a big role in the pathogenesis of CCA. A variety of cells contribute to said environment, ranging from regulatory T cells(Treg) to natural killer(NK) cells. The role of the NK cell is being increasingly studied in the context of CCA as it is one of the most abundant immune cells in the tumor microenvironment and likely plays a large role in the immune response to cancer cells.
 Methods:
 Our sample set included 36 patients from a publicly available RNA sequencing dataset from MD Anderson. RNA sequences were extracted and then input into a Stanford based application, CIBERSORT. This illustrates the impact different immune cells have on a tumor microenvironment. The immune cells included are the following: B-cells naïve, B-cells memory, plasma cells, CD8 T-cells, CD4 T-cells, T-cells follicular, T-cells regulatory, natural killer activated cells, natural killer resting cells, monocytes, macrophages (M0, M1, M2), activated dendritic cells, resting dendritic cells, activated mast cells, resting mast cells, eosinophils, and neutrophils. CIBERSORT uses immune cell fractions as the marker of immune cell impact. These fractions were then analyzed in a statistical tool, Radiant by Shiny, that computes a linear regression describing the relationship between immune cells and impact on overall survival.
 Results:
 Natural killer cells in the tumor microenvironment have a strong, positive impact on overall survival (p = 0.044). The coefficient of correlation was +198.
 Conclusions:
 Results of our analysis illustrate the strong and positive relationship that NK cells have on the overall survival of CCA. This is likely due to the NK cell’s ability to promote cell death through various pathways, such as the Fas-L and TNF apoptotic pathways. More specifically, it has been shown that cholangiocytes may have the ability to present lipid antigens to NK cells thus activating an alternate killing pathway making the NK cell a crucial player in the tumor microenvironment. This, coupled with being one of the most abundant cells in the tumor microenvironment in CCA, opens the door for the NK cell as an potential target for future treatment options. Understanding these specific interactions between NK cells and tumor cells proves to be essential for developing novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a1da82ab42c7f1091a653ec8bd73313150dda1f" target='_blank'>
              The tumor microenvironment in cholangiocarcinoma: Impact of natural killer cells on overall survival.
              </a>
            </td>
          <td>
            Morgan Kiryakoza, Lorin Sallan, Paul F Hanona, Neil Jairath, Daniel Ezekwudo, George Howard, I. Jaiyesimi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Pancreatic cancer (PC) is marked by extensive heterogeneity, posing significant challenges to effective treatment. The tumor microenvironment (TME), particularly cancer-associated fibroblasts (CAFs), plays a critical role in driving PC progression. However, the prognostic and functional contributions of distinct CAF subtypes remain inadequately understood. Here, we introduce a novel 7-gene risk model that not only robustly stratifies PC patients but also unveils the unique role of PHLDA1 as a key mediator in tumor-stroma crosstalk.By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA sequencing data, we comprehensively characterized the heterogeneity of CAFs in PC. We identified five CAF subtypes and focused on matrix CAFs (mCAFs), which were strongly associated with poor prognosis. A 7-gene mCAF-associated risk model was constructed using advanced machine learning algorithms, and the biological significance of PHLDA1 was validated through co-culture experiments and pan-cancer analyses.Our multiomics analysis revealed that the novel 7-gene model (comprising USP36, KLF5, MT2A, KDM6B, PHLDA1, REL, and DDIT4) accurately predicts patient survival, immunotherapy response, and TME status. Notably, PHLDA1 was uniquely overexpressed in CAFs and correlated with the activation of key protumorigenic pathways, including EMT, KRAS, and TGF-β, underscoring its central role in modulating the crosstalk between CAFs and malignant ductal cells. Pan-cancer analysis further supported PHLDA1’s prognostic and immunomodulatory significance across multiple tumor types.Our study presents a novel 7-gene prognostic model that significantly enhances risk stratification in PC and identifies PHLDA1+ CAFs as promising prognostic biomarkers and therapeutic targets. These findings provide new insights into the TME of PC and open avenues for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69308ca9c802c2244ef4487c21e12f41adc3c76e" target='_blank'>
              Integrated multiomics analysis identifies PHLDA1+ fibroblasts as prognostic biomarkers and mediators of biological functions in pancreatic cancer
              </a>
            </td>
          <td>
            Rui Wang, Guan-Hua Qin, Yifei Jiang, Fu-Xiang Chen, Zi-Han Wang, Linglin Ju, Lin Chen, Da Fu, En-Yu Liu, Su-Qing Zhang, 
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1093733a8ade229255066e064b631c0ae1860e" target='_blank'>
              ImmunoFusion: A Unified Platform for Investigating RNA-seq-Derived Gene Fusions in Cancer and Immunotherapy
              </a>
            </td>
          <td>
            YanKun Zhao, Shixiang Wang, Shensuo Li, Minjun Chen, Su-Han Jin, U. Gaipl, Hu Ma, Jian-Guo Zhou
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Objectives The tumorigenic progression of Lung adenocarcinoma (LUAD), the predominant NSCLC subtype, is predominantly driven by co-occurring mutations in KRAS proto-oncogene (KRAS)/Tumor protein p53 (TP53). However, their impact on tumor microenvironment (TME) heterogeneity, particularly neutrophil dynamics, remains poorly understood. This present study aims to elucidate how KRAS/TP53 mutations reprogram the TME and develop a neutrophil-centric prognostic signature for LUAD. Methods Leveraging single-cell RNA sequencing data and transcriptome data, neutrophil subpopulations were identified using Seurat and CellChat R packages, with trajectory analysis via Monocle2 R package. High-dimensional weighted gene co-expression network analysis (hdWGCNA), univariate Cox regression, and least absolute shrinkage and selection operator (LASSO) regression analyses were employed to generate a prognostic signature. Functional validation included Ras homolog family member V (RHOV) knockdown in A549/H1299 cells using siRNA, were assessed by cell counting kit 8 (CCK8) assay, wound healing assay, and transwell assay. Results KRAS/TP53-mutated LUAD exhibited increased neutrophil infiltration, particularly IS MUT subtypes with enhanced OSM/CALCR/IL-1 signaling. A five-gene prognostic signature (MS4A1, ANLN, FAM83A, RHOV, KRT6A) stratified patients into high- and low-risk groups with divergent overall survival in the TCGA-LUAD cohort (p < 0.0001). AUCs achieved 0.73, 0.70, and 0.66 at 1-, 3-, and 5-year, respectively. External validation in immunotherapy cohorts (IMvigor210, GSE78220) confirmed the fine predictive capability of the prognostic signature in predicting treatment response. An integrated prognostic nomogram combining clinicopathological features and risk score further improved its clinical utility. Pseudotime analysis found that RHOV was essential for the growth of lung epithelial cells. RHOV knockdown significantly reduced the proliferation, migration, and invasion capabilities of A549/H1299 cells in vitro. Conclusion KRAS/TP53 mutations may drive neutrophil heterogeneity in the TME of LUAD, addressing prognostic and therapeutic value. The five-gene signature and RHOV targeting offer translational relevance for risk stratification and therapy. These findings bridge genomic alterations with TME remodeling, advancing precision oncology in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872ec4a305d8c2a95a643b848b88e9570d7d811b" target='_blank'>
              Single-Cell Transcriptomic Profiling Reveals KRAS/TP53-Driven Neutrophil Reprogramming in Luad: A Multi-Gene Prognostic Model and Therapeutic Targeting of RHOV
              </a>
            </td>
          <td>
            Yinghui Ye, Yulou Luo, Yutian Sun, Yujie Zhang, Jiaxin Lin, Ziling Yang, Anping Xu, Bei Xue
          </td>
          <td>2025-05-29</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by significant heterogeneity and a poor prognosis. M2 macrophages (M2) play a crucial role in the microenvironment of PDAC, yet their exact functional significance remains poorly understood. This study sought to explore the potential communication and clinical implications of M2 in PDAC. The differentiation pathway of M2 and their interaction with malignant PDAC cells were analyzed using single-cell RNA sequencing (scRNA-seq). Additionally, a prognostic signature related to M2 marker genes was constructed and analyzed using bulk RNA sequencing. Three distinct differentiation states of M2 were characterized, each exhibiting unique transcription factor expression profiles and signaling pathway activations. Potential intercellular communication between M2, PDAC malignant cells, and fibroblasts was initially observed at the level of computer analysis. The prognostic model was identified through the LASSO algorithm. Patients stratified by risk level displayed significant variations in prognostic indicators such as signaling pathway activity, gene mutation frequency, immune cell infiltration, and drug sensitivity. These findings are of course preliminary, and future in vitro/in vivo experiments will be needed to knock out these signature genes or ligand-receptor pairs of genes in macrophages to validate their role in macrophage development. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02838-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/273ead8bc63462bd01d5d5a63b43ec9833ef2540" target='_blank'>
              Establishment of M2 macrophage-related prognostic signature in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Hui Chen, Jingyang Yin, Xianxing Wang, Renpei Xia, Fanbo Qin, Zhuo Li, Jiali Yang, Changjiang Liu, Shixiang Guo, Huaizhi Wang
          </td>
          <td>2025-06-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Background: Metastatic breast cancer (MBC) is considered an incurable disease and understanding the molecular mechanisms at the basis of tumor progression has become mandatory. Circulating tumor cells (CTCs) constitute a population of rare cells that are shed in the bloodstream by primary and metastatic tumors and migrate towards distant organs. The molecular characterization of CTCs is a powerful tool to better investigate the mechanisms underlying metastasis. In this study, we combine phenotypic and transcriptional profiling of CTCs isolated from MBC patients to infer the mechanisms involved in their biology, also in relation to the metastatic site and their phenotypic features. Methods: Blood samples for CTC analysis were collected from luminal MBC patients recruited in the KENDO study (Protocol code: IRST174.19); CTCs were enriched using the RosetteSep CTC Enrichment Cocktail and frozen until downstream analyses as viable cells. To validate the antibody staining for phenotypic analysis, tests were conducted using the cancer cell line SKBR-3. In brief, SKBR-3 cells were spiked in peripheral blood of a healthy volunteer and enriched using RosetteSep CTC Enrichment Cocktail. Enriched SKBR-3 and patients’ CTCs were incubated with antibodies targeting CD45 (leukocyte marker) and epithelial markers (E-tag; EpCAM, E-cadherin) for phenotypic analysis using the DEPArray NxT platform. For transcriptomic profiling, single CTCs were isolated as single, viable cells. Libraries were prepared using the QIAseq UPX 3’ Transcriptome Kit and sequenced using MiSeq on V3-150 cycles cartridges. Bioinformatic and statistical analyses were carried out using the CLC Genomic Workbench. Results: Through feasibility analysis conducted on SKBR-3 cells, we demonstrate that our workflow involving phenotypic analysis on DEPArray NxT is an efficient tool to identify cancer cells (CD45-/E-tag+) and white blood cells (WBCs; CD45+/E-tag-). In MBC samples, we detected different cell populations: WBCs (CD45+/E-tag-), CTCs (CD45-/E-tag+), and dual-positive CTCs (CD45+/E-tag+; dpCTCs). Transcriptomic profiling was successfully performed on 37 single CTCs isolated from 7 patients. The transcriptional profile of CTCs is consistent with breast malignancy for different databases including ClinVar2019, DisGeNet, and Human Phenotype Ontology. Moreover, Gene Set Enrichment analysis highlighted pathways associated with synapse organization and calcium channel activity, suggesting the implication of the interplay between cancer cells and neurons in metastasis. Compared to a commercial RNA, CTCs express at high levels transcripts associated with cancer dissemination and genome architecture (i.e., STAG2 and H2AFZ). In CTCs from patients with bone metastasis we detected the expression of genes associated with metastatic osteotropism (i.e., S100A4, VAPA, HMGB1), while CTCs from a single patient with gastric metastasis expressed genes associated with gastric cancer metastasis and cell redox homeostasis. Transcriptional profiling of dpCTCs revealed the expression of transcripts encoding for members of the CD47/SIRPα axis, supporting their macrophage-cancer cell fusion origin, and increased glycogen biosynthetic and metabolic activity. Conclusions: Our data suggest the potential of CTC based transcriptomics in providing new insights on the clinical-biological evaluation of MBC and its mechanisms underlying metastatic cascade and organotropism. In addition, our data provide hints for the characterization of dpCTCs, a poorly studied population whose role in metastasis is still far to be fully elucidated.
 Citation Format: Michela Palleschi, Tania Rossi, Sara Bandini, Michele Zanoni, Michela Cortesi, Erika Bandini, Andrea Rocca, Giulia Gallerani, Ivan Vannini, Meropi Plousiou, Lorenzo Gerratana, Giovanni Tallini, Giovanni Martinelli, Ugo De Giorgi, Paola Ulivi. Transcriptomic and phenotypic profiling of circulating tumor cells from metastatic breast cancer patients [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-01-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48cb1aa4534473e2256f4e5d0f28171830899d25" target='_blank'>
              Abstract P3-01-17: Transcriptomic and phenotypic profiling of circulating tumor cells from metastatic breast cancer patients
              </a>
            </td>
          <td>
            M. Palleschi, T. Rossi, Sara Bandini, Michele Zanoni, M. Cortesi, E. Bandini, Andrea Rocca, G. Gallerani, Ivan Vannini, Meropi Plousiou, L. Gerratana, Giovanni Tallini, Giovanni Martinelli, U. De Giorgi, Paola Ulivi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer research faces significant biological, technological, and systemic limitations that hinder the development of effective therapies and improved patient outcomes. Traditional preclinical models, such as 2D and 3D cell cultures, murine xenografts, and organoids, often fail to reflect the complexity of human tumor architecture, microenvironment, and immune interactions. This discrepancy results in promising laboratory findings not always translating effectively into clinical success. A core obstacle is tumor heterogeneity, characterized by diverse genetic, epigenetic, and phenotypic variations within tumors, which complicates treatment strategies and contributes to drug resistance. Hereditary malignancies and cancer stem cells contribute strongly to generating this complex panorama. Current early detection technologies lack sufficient sensitivity and specificity, impeding timely diagnosis. The tumor microenvironment, with its intricate interactions and resistance-promoting factors, further promotes treatment failure. Additionally, we only partially understand the biological processes driving metastasis, limiting therapeutic advances. Overcoming these barriers involves not only the use of new methodological approaches and advanced technologies, but also requires a cultural effort by researchers. Many cancer studies are still essentially observational. While acknowledging their significance, it is crucial to recognize the shift from deterministic to indeterministic paradigms in biomedicine over the past two to three decades, a transition facilitated by systems biology. It has opened the doors of deep metabolism where the functional processes that control and regulate cancer progression operate. Beyond biological barriers, systemic challenges include limited funding, regulatory complexities, and disparities in cancer care access across different populations. These socio-economic factors exacerbate research stagnation and hinder the translation of scientific innovations into clinical practice. Overcoming these obstacles requires multidisciplinary collaborations, advanced modeling techniques that better emulate human cancer, and innovative technologies for early detection and targeted therapy. Strategic policy initiatives must address systemic barriers, promoting health equity and sustainable research funding. While the complexity of cancer biology and systemic challenges are formidable, ongoing scientific progress and collaborative efforts inspire hope for breakthroughs that can transform cancer diagnosis, treatment, and survival outcomes worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/479bf9e891ac0e6fa811bc608566229cbe810997" target='_blank'>
              Overcoming Barriers in Cancer Biology Research: Current Limitations and Solutions
              </a>
            </td>
          <td>
            Giovanni Colonna
          </td>
          <td>2025-06-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) are essential components of the tumor microenvironment (TME), contributing to tumorigenesis, progression, and resistance to therapy. However, the functional diversity of CAF subpopulations and their role in tumor progression and patient prognosis remain poorly understood. This study aims to explore CAF heterogeneity and their functional roles in the TME using single-cell RNA sequencing (scRNA-seq) and multi-omics data analysis. Methods scRNA-seq data were analyzed to cluster CAF subpopulations in the TME, with key genes identified through functional annotation. Differentially expressed genes were analyzed, and prognostic genes were selected via Cox and LASSO regression. A risk score model (RiskScore) was developed for survival prediction and immune therapy sensitivity evaluation. Core CAF genes were examined using siRNA interference, qPCR, and Western blotting. Drug sensitivity was assessed to explore the clinical relevance of these genes. Results Four CAF subpopulations (CAF-0, CAF-1, CAF-2, CAF-3) were identified, revealing differences in key tumor-associated signaling pathways (e.g., MYC, WNT, TGF-β). Thirteen core genes related to prognosis were identified, and a RiskScore model was developed, showing significantly worse survival rates for high-risk patients (p < 0.001) and features of immune suppression, including increased M0 macrophage infiltration. Drug sensitivity analysis indicated that core genes (e.g., KLRB1, MAP1B) were linked to drug sensitivity, suggesting potential biomarkers for targeted therapy. Experimental validation showed that knockdown of the HIP1R gene significantly reduced tumor cell expression, confirming its critical role in tumor development. Conclusion This study offers a comprehensive analysis of CAF heterogeneity and its impact on TME, patient prognosis, and drug sensitivity. The developed RiskScore model provides theoretical support for personalized treatment based on CAF-related genes, offering new insights into CAF-driven tumor progression and potential targets for precision oncology and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5c1ce9d5dedbb8252961d9100d080e0187631a3" target='_blank'>
              Therapeutic implications of cancer-associated fibroblast heterogeneity: insights from single-cell and multi-omics analysis
              </a>
            </td>
          <td>
            Yumin Wang, Yi Ding, Haolin Liu, Zhongyou Xia, Guoqiang Liao, Shicheng Fan, JunXiong Li, JingBo Qin, PinYao Liang, Peng Gu, Xiaodong Liu, Runan Dong
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2e39c5159dc340646cc3bcfb136e1f500d00e4" target='_blank'>
              Reelin-LRP8 signaling mediates brain dissemination of breast cancer cells via abluminal migration.
              </a>
            </td>
          <td>
            Haofeng Huang, Min Zhang, Enyu Huang, Yongfang Zhao, Xiaoyu Li, Pu Qiu, Cairui Li, Jiahua Tao, Yuanqi Zhang, Lianxiang Luo, Guozhu Ning, Ceshi Chen, Jingjing Zhang
          </td>
          <td>2025-06-12</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cf265ca3e7fb2004f37001431683cd2d9283383" target='_blank'>
              The invadosome of stem-like cancer cells: metastasis machinery and targetable vulnerability.
              </a>
            </td>
          <td>
            Kelvin K Tsai
          </td>
          <td>2025-06-04</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Penile squamous cell carcinoma (PSCC) is a highly aggressive malignancy without effective treatment due to limited knowledge of its development and tumor microenvironment (TME). In this study, single-nucleus RNA sequencing (snRNA-seq) and high-resolution spatial transcriptomics are employed to comprehensively investigate the development trajectories and the TME. The results revealed that PSCC cells mimicked the differentiation and tissue organization of normal penile epithelium, independent of the human papillomavirus (HPV) infection status. Notably, a spatial subtype, Tum_1, appeared at early stage of tumor differentiation and in tumor-normal boundary regions. This subtype exhibited enhanced basal-like and stemness features and showed high LAMC2 expression, which activated laminin-integrin signaling via ITGA6/ITGB4, promoting tumor invasiveness. Furthermore, the results indicated that HPV-positive basal stem-like neoplasms dampened the immune function of T cells and macrophages, promoting an immunosuppressive environment that facilitates tumor progression. Supporting this, the patients with head and neck squamous cell carcinoma and lung squamous cell carcinoma who have high expression of HPV-positive Tum_1 signatures derived greater benefit from PD-1 blockade therapy. In summary, the findings provide a comprehensive spatial landscape of the PSCC TME and suggest potential treatment approaches targeting laminin-integrin interaction and immunotherapy, especially in HPV-positive patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c44a03a89b4201962b2248fcc19db445771e54" target='_blank'>
              Single-Cell and Spatial Transcriptomic Profiling of Penile Squamous Cell Carcinoma Reveals Dynamics of Tumor Differentiation and Immune Microenvironment.
              </a>
            </td>
          <td>
            Hongjian Song, Zhichao Tong, Guixiang Xie, Yaowei Li, Yubo Zhao, Fengpu Fan, Zongzheng Yang, Qing Shi, Qian Zhang, Feng Wen, Hongxue Meng, Haonan Li, Pengyu Guo, Dayong Hou, Zhenwei Zhang, Zhihao Yin, Ziyi Liu, Jianwei Wang, Peng Zhang, Peng Dai, Changfu Li, Lichen Teng, Yangyang Xu, Dexin Ding, Tao Xu, Jianzhang Li, Yongsheng Chen, Yu Qiu, Xiaowei Hu, Wentao Liu, Liang Wu, Roman Nawroth, Guibo Li, Hang Su, Ziqing Deng, Ziqi Wang, Di Wang, Wanhai Xu
          </td>
          <td>2025-06-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41cd39016a4b2c876a2210fd9227cfa83893338" target='_blank'>
              Unraveling Tyrosine-Kinase Inhibitor Resistance in NSCLC Cells via Same-cell Measurement of RNA, Protein, and Morphological Responses
              </a>
            </td>
          <td>
            Vivian Dien, Connor Thompson, Tristin Rammel, Jaime Roquero, Marielle Krivit, Grace Yeo, Daniel Honigfort, Pazinah Bhadha, Colin Brown-Greeves, Mariam Dawood, Keerthana Elango, Hyukjin J. Kwon, Tyler Lopez, Alvin Lo, Meghna Manjunath, Pin Ren, Ben Song, Mitch Sundkamp, Ramreddy Tippana, Wilfred Van Ijcken, Jennifer Wong, Jane Yao, Da Zhou, Sinan Arslan, Matthew Kellinger, Amirali Kia, S. Kruglyak, Michael Previte, Molly He
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GB) is the most aggressive tumor of the central nervous system (CNS), accounting for almost 80% of all primary brain tumors. Despite standard-of-care consisting of surgical resection, when possible, adjuvant radiotherapy (RT) and chemotherapy with Temozolomide (TMZ), GB remains highly fatal, with an estimated recurrence rate of over 90% and a median overall survival (OS) of around 15 months from diagnosis. Several factors contribute to such poor patient outcome, including a unique myeloid-rich tumor microenvironment (TME) that confers immunosuppression and therapeutic resistance. Multi-omics, single-cell transcriptomics and multi-modal spatial analyses of GB are unraveling the diversity of brain myeloid cells, including activated microglia, border-associated macrophages (BAM), and monocyte-derived glioma-associated macrophages (GAM), instructed by ontogeny, spatial distribution, cell-cell interactions and response to metabolic cues in the TME. In this review, we elaborate on the heterogeneity and plasticity of myeloid cells in GB and discuss the promise and challenges for rational therapeutic targeting of GAMs in GB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d921ef730c3240f4439955cc403f795493cbb47c" target='_blank'>
              Rational therapeutic targeting of myeloid cells in glioblastoma: challenges and perspectives
              </a>
            </td>
          <td>
            Faruk Akay, Maya Saleh
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In isocitrate dehydrogenase wildtype glioblastoma (GBM), cellular heterogeneity across and within tumors may drive therapeutic resistance. Here we analyzed 121 primary and recurrent GBM samples from 59 patients using single-nucleus RNA sequencing and bulk tumor DNA sequencing to characterize GBM transcriptional heterogeneity. First, GBMs can be classified by their broad cellular composition, encompassing malignant and nonmalignant cell types. Second, in each cell type we describe the diversity of cellular states and their pathway activation, particularly an expanded set of malignant cell states, including glial progenitor cell-like, neuronal-like and cilia-like. Third, the remaining variation between GBMs highlights three baseline gene expression programs. These three layers of heterogeneity are interrelated and partially associated with specific genetic aberrations, thereby defining three stereotypic GBM ecosystems. This work provides an unparalleled view of the multilayered transcriptional architecture of GBM. How this architecture evolves during disease progression is addressed in the companion manuscript by Spitzer et al.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c45eaa99273988e47319564458ae28d00342e00" target='_blank'>
              The multilayered transcriptional architecture of glioblastoma ecosystems
              </a>
            </td>
          <td>
            M. Nomura, Avishay Spitzer, K. Johnson, L. Garofano, Djamel Nehar-Belaid, Noam Galili Darnell, Alissa C. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, Simona Migliozzi, L. G. Gonzalez Castro, Tamrin ChowdhFury, N. Robine, C. Reeves, Jong Bae Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, Simone P Niclou, Labeeba Nusrat, Sorcha Kellet, Sunit Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, Marc Sanson, S. Tanaka, Nobuhito Saito, D.M. Ashley, Stephen T Keir, Keith L. Ligon, Jason T Huse, W. Yung, A. Lasorella, R. Verhaak, A. Iavarone, M. Suvà, I. Tirosh
          </td>
          <td>2025-05-09</td>
          <td>Nature Genetics</td>
          <td>3</td>
          <td>62</td>
        </tr>

        <tr id="Cancer metastasis constitutes a multifactorial phenomenon that continues to confound therapeutic strategies. The biochemical signals governing motile phenotypes have been extensively characterized, but mechanobiological interactions have only recently been integrated into cancer cell motility models and remain less well elucidated. The identification of the biochemically and mechanically controlled epithelial–mesenchymal transition (EMT) of cancer cells, which occurs either completely or partially, has led to a major breakthrough and a universal phenomenon in cancers. In addition, a relatively new theory based on mechanobiological aspects called “jamming-to-unjamming transition” is being proposed to explain the transition of cancer cells to an invasive phenotype. The latter transition may help to better understand the different types of 3D migration and invasion of cancer cells. Similarly to EMT, the transition from jamming to unjamming seems to be controlled by molecular and physical factors, including cell mechanics and mechanical cues from the extracellular matrix (ECM) of the tumor microenvironment (TME). It is challenging to grasp the distinctions between the transition from jamming to unjamming and EMT, as they appear to be the same at first glance. However, upon closer examination, the two transitions are quite separate. Moreover, it is still unclear whether both transitions may act synergistically. This review highlights the most important breakthroughs in the transition from jamming to unjamming, with a focus on mechanobiology and extracellular environmental aspects, and it compares them with those of EMT. In addition, the impact of the TME, such as ECM scaffold and cancer-associated fibroblasts (CAFs) on the jamming-to-unjamming transition is discussed. Finally, the research frontiers and future directions in the field of mechanobiological research in cancer metastasis are outlined.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/356465968ad0ebd4b69565d6368d70dc587c326c" target='_blank'>
              ECM Mechanics Control Jamming-to-Unjamming Transition of Cancer Cells
              </a>
            </td>
          <td>
            C. Mierke
          </td>
          <td>2025-06-20</td>
          <td>Cells</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Gastric cancer remains a leading cause of cancer-related mortality worldwide, driven significantly by gastric cancer stem cells (GCSCs), a unique subset of cells characterized by self-renewal, multipotency, and therapy resistance. This review synthesizes recent advancements in understanding GCSCs, focusing on their definition, characteristics, identification, isolation and role in tumor growth, metastasis, and resistance to conventional treatments. Key surface markers such as CD44, CD133, leucine rich repeat containing G protein-coupled receptor 5 (LGR5), and aquaporin 5 (AQP5) are critical for GCSC identification, with their expression linked to poor prognosis and tumor progression. The review summarizes mechanisms like epithelial-mesenchymal transition (EMT) and Wnt/β-catenin signaling that enhance GCSC stemness and malignancy. It also highlights emerging targeted therapies, including antibody-based approaches including CD44v6-specific antibodies, inhibitors like sulfasalazine, and novel immunological strategies such as chimeric antigen receptor T (CAR-T) cell therapy, aimed at eradicating GCSCs. By elucidating the molecular and functional roles of GCSCs, this review underscores GCSCs as therapeutic targets and provides a foundation for developing precision medicine strategies to improve gastric cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cacd867c51c08c4c5178ffbc92262edb25abafe2" target='_blank'>
              Current insights into gastric cancer stem cells: Mechanisms and therapeutic prospects
              </a>
            </td>
          <td>
            Chang-Yong Yin
          </td>
          <td>2025-05-30</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fdc708d82395eddf5ef9f62e5ba5f1c540963b1" target='_blank'>
              Single-cell and bulk transcriptome analysis identifies B-cell subpopulations and associated cancer subtypes with distinct clinical and molecular characteristics.
              </a>
            </td>
          <td>
            Yin A He, Li Zhao, Yufen Zheng, Xiaosheng Wang
          </td>
          <td>2025-06-17</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, largely due to its profoundly immunosuppressive tumor microenvironment (TME) and intrinsic resistance to conventional therapies. Radiotherapy (RT), traditionally valued for its cytotoxic effects, has recently been recognized for its immunomodulatory potential. This mini-review explores the multifaceted interactions between RT and the PDAC immune microenvironment, highlighting mechanisms such as induction of immunogenic cell death, enhancement of antigen presentation, modulation of cytokine and chemokine profiles, and upregulation of immune checkpoint molecules. These effects may transform immunologically “cold” tumors into “hot” ones, providing a rationale for combination strategies with immunotherapy. However, the dense desmoplastic stroma, abundance of regulatory T cells, and myeloid-derived suppressor cells within PDAC present substantial challenges that hinder effective immune activation. Advances in single-cell and spatial transcriptomic technologies offer new opportunities to better characterize the TME and guide personalized treatment strategies. By synthesizing mechanistic insights and clinical evidence, this review underscores the potential of integrating RT with immunotherapy to overcome resistance mechanisms and improve therapeutic outcomes in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97772762d5bc3d1c76c3496afb2362f063af01af" target='_blank'>
              Radiotherapy and immune microenvironment crosstalk in pancreatic cancer: a comprehensive review of current insights and future directions
              </a>
            </td>
          <td>
            Yucheng Xu, Jingjing Chen, Yudong Qiu, Juan Du
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Targeted therapy interventions using tyrosine kinase inhibitors (TKIs) provide encouraging treatment responses in patients with ALK-rearranged lung adenocarcinomas, yet resistance occurs almost inevitably. In addition to tumor cell-intrinsic resistance mechanisms, accumulating evidence suggests that cancer-associated fibroblasts (CAFs) within the tumor microenvironment contribute to therapy resistance. This study aimed to investigate CAF-driven molecular networks that shape the therapeutic susceptibility of ALK-driven lung adenocarcinoma cells. Methods Three-dimensional (3D) spheroid co-cultures comprising ALK-rearranged lung adenocarcinoma cells and CAFs were utilized to model the tumor microenvironment. Single-cell RNA sequencing was performed to uncover transcriptional differences between TKI-treated homotypic and heterotypic spheroids. Functional assays assessed the effects of CAF-conditioned medium and CAF-secreted factors on tumor cell survival, proliferation, lipid metabolism, and downstream AKT signaling. The therapeutic potential of targeting metabolic vulnerabilities was evaluated using pharmacological inhibition of lipid metabolism and by ferroptosis induction. Results CAFs significantly diminished the apoptotic response of lung tumor cells to ALK inhibitors while simultaneously enhancing their proliferative capacity. Single-cell RNA sequencing identified lipogenesis-associated genes as a key transcriptional difference between TKI-treated homotypic and heterotypic lung tumor spheroids. CAF-conditioned medium and the CAF-secreted factors HGF and NRG1 activated AKT signaling in 3D-cultured ALK-rearranged lung tumor cells, leading to increased de novo lipogenesis and suppression of lipid peroxidation. These metabolic adaptations were critical for promoting tumor cell survival and fostering therapy resistance. Notably, both dual inhibition of ALK and the lipid-regulatory factor SREBP-1, as well as co-treatment with ferroptosis inducers such as erastin or RSL3, effectively disrupted the CAF-driven metabolic-supportive niche and restored sensitivity of resistant lung tumor spheroids to ALK inhibition. Conclusions This study highlights a critical role for CAFs in mediating resistance to ALK-TKIs by reprogramming lipid metabolism in ALK-rearranged lung cancer cells. It suggests that targeting these metabolic vulnerabilities, particularly through inhibition of lipid metabolism or induction of ferroptosis, could provide a novel therapeutic approach to overcome resistance and improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40170-025-00400-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a02a6efa6497a2bcee01f9f0743c25e6ad43b8" target='_blank'>
              Cancer-associated fibroblasts promote drug resistance in ALK-driven lung adenocarcinoma cells by upregulating lipid biosynthesis
              </a>
            </td>
          <td>
            Ann-Kathrin Daum, Lisa Schlicker, M. Schneider, T. Muley, Ursula Klingmüller, Almut Schulze, Michael Thomas, P. Christopoulos, Holger Sültmann
          </td>
          <td>2025-06-16</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Clinically aggressive lactotroph pituitary neuroendocrine tumors (PitNET) are invasive tumors with an unusually rapid growth rate despite maximally tolerated doses of dopamine agonist (DA). We aimed to unravel the molecular heterogeneity of lactotroph PitNET and to identify biomarkers of aggressiveness and resistance to pharmacological treatment. A total of 13 patients harboring DA-resistant lactotroph PitNET were included in this study. Visium Spatial Transcriptomics (ST), whole transcriptome sequencing (WTS), and whole exome sequencing (WES) were performed in tumors from 4 of these patients; WTS and WES was carried out in 5; tumors from two patients underwent ST and WES and tumors from two other patients underwent only ST. Tumors were classified as null or partial responders according to their response to DA treatment. The eight PitNET analyzed by ST exhibited significant intratumoral heterogeneity, with clones showing alterations in PI3K/AKT and lipid metabolism pathways, particularly inositol phosphate, glycerophospholipid, and sphingolipid metabolism. The cell-cell communication analysis showed FGF-FGFR ligand receptor interaction whilst the transcription factors RXRA and CREM showed participation in both groups. A trajectory exploration was performed by including all PitNET together in a single analysis to determine whether there was a tendency or molecular pathway showing a differentiation pattern that would guide the transition from a partially responsive PitNET to a completely unresponsive one. We did not observe any such pattern. All of these findings were corroborated in the cohort of DA-resistant PitNETs in which only bulk WTS and WES were performed. The bulk WTS corroborated lipid metabolism and PI3K-AKT pathway alteration in PitNET, whereas the WES showed only SF3β1 and TP53 variants in one tumor each. Our work suggests that the PI3K/AKT pathway may constitute a molecular target at which to aim therapeutic strategies designed to treat aggressive and DA-resistant lactotroph PitNET. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02025-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9584b357aff2d73647fc85b3bbd6a84a02ea98" target='_blank'>
              Spatial transcriptomics reveal PI3K-AKT and metabolic alterations in aggressive, treatment-resistant lactotroph pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            F. Martinez-Mendoza, Sergio Andonegui-Elguera, E. Sosa-Eroza, Erick Gomez-Apo, Aurea Escobar-España, Carolina González-Torres, Javier Gaytán-Cervantes, Alam Palma-Guzman, Hugo Torres-Flores, Alberto Moscona-Nissan, S. Hinojosa-Alvarez, J. Hernández-Perez, R. Chavez-Santoscoy, G. Guinto, Gerardo Y. Guinto-Nishimura, B. López-Félix, Erick U. Zepeda-Fernandez, Erick M. Estrada-Estrada, V. Correa-Correa, P. A. González-Zavala, Marco A. Asenscio-Montiel, M. A. García-Vargas, Emmanuel Cantu-Chavez, Rocío Arreola-Rosales, Keiko Taniguchi-Ponciano, D. Marrero-Rodríguez, Moises Mercado
          </td>
          <td>2025-05-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Matrix metalloproteinase 1 plays a pivotal role in tumor biology and immune modulation through its enzymatic remodeling of the extracellular matrix, facilitating tumor progression. In this study, we utilized large-scale single-cell RNA sequencing and spatial transcriptomics to investigate MMP1 expression, its cellular localization, and its impact on tumor progression and immune modulation. Our findings reveal that MMP1 expression is elevated in various tumor types and is strongly correlated with metastatic potential. High MMP1 expression was associated with increased activity in epithelial–mesenchymal transition signaling and TNFα/NF-κB pathways, which are known to promote tumor progression. Furthermore, MMP1+ malignant cells exhibited significant interactions with immune cells, particularly macrophages and CD8+ T cells. MMP1 expression correlated with enhanced macrophage infiltration and impaired CD8+ T-cell function, contributing to an immunosuppressive tumor microenvironment. Notably, the CXCL16-CXCR6 and ANXA1-FPR3 signaling axes were identified as key mediators of these interactions. Inhibition of MMP1 in vitro demonstrated reduced cell invasion, stemness, and proliferation, while increasing reactive oxygen species levels and promoting apoptosis. Our findings position MMP1 as a key player in the “tumor-immune” vicious cycle and a promising therapeutic target to enhance anti-tumor responses and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d0c741845c10c07b47cb268ec3e101fb8935759" target='_blank'>
              Decoding the impact of MMP1+ malignant subsets on tumor-immune interactions: insights from single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Da-Ming Xu, Ling-Xiao Chen, Ting Xue, Xiao-Yu Zhuang, Li-Chao Wei, Hui Han, Miao Mo
          </td>
          <td>2025-05-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="For over a century, scientists reported the disruption of normal nuclear shape and size in cancer. These changes have long been used as tools for diagnosis and staging of malignancies. However, to date, the mechanisms underlying these aberrant nuclear phenotypes and their biological significance remain poorly understood. Using a model of pancreatic ductal adenocarcinoma (PDAC), the major histological subtypes of pancreatic cancer, we found oncogenic mutant KRAS reduces nuclear size. Transcriptomic and protein expression analysis of mutant KRAS-expressing PDAC cells revealed differential levels of several nuclear envelope-associated genes. Further analysis demonstrated the nuclear lamina protein, Emerin (EMD), acted downstream of KRAS to mediate nuclear size reduction in PDAC. Analysis of human PDAC samples showed that increased EMD expression associates with reduced nuclear size. Finally, in vivo genetic depletion of EMD in a mutant KRAS-driven PDAC model resulted in an increased nuclear size and a reduced incidence of poorly differentiated PDAC. Thus, our data provides evidence of a novel mechanism underlying nuclear size regulation and its impact in PDAC carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fad3a87beb812ed2828574a6cdacbebbce33710" target='_blank'>
              Emerin is an effector of oncogenic KRAS-driven nuclear dynamics in pancreatic cancer.
              </a>
            </td>
          <td>
            Luis F Flores, David L Marks, Renzo E Vera, Ashley N Sigafoos, Ezequiel J Tolosa, Luciana L Almada, David R Pease, M. D. Toruner, Brian Chang, Brooke R Tader, Kayla C LaRue-Nolan, Ryan M. Carr, Rondell P Graham, Catherine E Hagen, Matthew R Brown, Aleksey V Matveyenko, Katherine L Wilson, David W Dawson, Christopher L. Pin, Kyle J Roux, Martin E Fernandez-Zapico
          </td>
          <td>2025-06-10</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f88d93c45659d05d4c112ca60db918cd868e7fbf" target='_blank'>
              Signalomics for molecular tumor boards and precision oncology of breast and gynecological cancers.
              </a>
            </td>
          <td>
            Tatiana V Denisenko, Anna E. Ivanova, A. Koval, Denis N. Silachev, Lee Jia, G. Sukhikh, Vladimir L. Katanaev
          </td>
          <td>2025-06-09</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6381edb3d3b2ef9cddaaae818d10d5852bbd41f" target='_blank'>
              Detecting and quantifying clonal selection in somatic stem cells.
              </a>
            </td>
          <td>
            Verena Körber, N. A. Jakobsen, N. Ansari-Pour, Rachel Moore, Nina Claudino, M. Metzner, Eva Thielecke, Franziska Esau, B. Usukhbayar, Mirian Angulo Salazar, Simon Newman, B. Kendrick, Adrian H Taylor, Rasheed Afinowi-Luitz, R. Gundle, B. Watkins, Kim Wheway, D. Beazley, S. Dakin, Antony Palmer, Andrew J. Carr, Paresh Vyas, Thomas Höfer
          </td>
          <td>2025-07-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Small-cell lung cancer (SCLC) is the most lethal type of lung cancer, characterized by rapid evolution from chemosensitivity to chemoresistance and limited treatment options. However, the mechanisms underlying this evolution remain poorly understood. Here, we show that Retinoid X receptor γ (RXRγ) is uniquely overexpressed in chemo-resistant SCLC tumors, and that RXRγ serves as an essential factor driving chemoresistance in SCLC. RXRγ forms phase-separated droplets with LSD1 in the nucleus, which enhances RXRγ-mediated gene transcription activity and reprograms gene expression, promoting tumor stemness and metastasis, and eventually driving SCLC chemoresistance. In turn, RXRγ antagonist disrupts RXRγ-LSD1 interaction, reducing their binding to the target gene locus, markedly suppressing the expression of the RXRγ target gene network. Finally, RXRγ antagonists strongly suppress tumor growth and metastasis and restore SCLC vulnerability to chemotherapy in multiple preclinical SCLC models, resulting in a substantial extension of survival in mouse models. Thus, these results establish RXRγ as a key player in SCLC by phase separation and as a potential therapeutic target for this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc6976505863353b005498741bc1fbef13303de9" target='_blank'>
              Phase separation of RXRγ drives tumor chemoresistance and represents a therapeutic target for small-cell lung cancer.
              </a>
            </td>
          <td>
            Hong Wang, Jie Huang, Zhenhua Zhang, Yana An, Huizi Sun, Jiang-liang Chen, Weineng Feng, Hao Duan, Yonggao Mou, Yuanxiang Wang, Peiqing Liu, Huihao Zhou, Hong-Wu Chen, Jian Zhang, Xiaoyun Lu, Junjian Wang
          </td>
          <td>2025-05-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Pyroptosis is a form of programmed cell death characterized by inflammasome activation and the release of inflammatory cytokines, which induce a strong immune response. Unlike apoptosis, pyroptosis can elicit potent immune stimulation, potentially playing a crucial role in anti-tumor immunity. However, it may also promote tumor progression by altering the tumor microenvironment and facilitating immune evasion. This study investigates pyroptosis-related gene expression in hepatocellular carcinoma (HCC), with a focus on identifying key genes that influence prognosis and tumor microenvironment dynamics. Methods Single-cell RNA sequencing (scRNA-seq) data from 10 HCC patients were obtained from the GEO database (GSE149614), along with spatial transcriptomic data and bulk RNA-seq data from TCGA. We performed data processing and quality control using the Seurat package and applied machine learning techniques, including LASSO regression, to identify key pyroptosis-related genes. Functional analyses, including Gene Ontology (GO), KEGG, and GSVA, were conducted to explore biological pathways. Pyroptosis levels were quantified across cell types, and survival analysis was performed to evaluate prognostic impacts. Cell communication and immune infiltration were also assessed to understand the tumor microenvironment. Results We identified CHMP4B as a key pyroptosis-related gene in HCC, significantly associated with poor prognosis. High CHMP4B expression was correlated with shorter overall survival (OS) and disease-free survival (DFS). Functional enrichment analysis showed that CHMP4B is involved in cell cycle regulation, DNA repair, and cytoskeletal organization. Spatial transcriptomics revealed heterogeneous CHMP4B expression in the tumor microenvironment, with higher levels found in advanced tumor stages. Moreover, high CHMP4B expression was associated with increased infiltration of immunosuppressive cells, such as monocytes and macrophages, and upregulation of immune checkpoint molecules (PD-L1, CTLA4), suggesting its role in promoting immune evasion. Conclusions Our findings highlight CHMP4B as a critical regulator of pyroptosis in HCC, influencing tumor progression and immune modulation. High CHMP4B expression may facilitate the development of an immunosuppressive microenvironment, enabling immune escape and tumor growth. The study underscores CHMP4B’s potential as a prognostic biomarker and therapeutic target in HCC. However, the limited sample size calls for further validation using larger datasets and multi-omics approaches, such as proteomics and metabolomics, to fully elucidate its functional role in HCC pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db1e712eb15666ba81f87054115a23e3c5a7897" target='_blank'>
              Pyroptosis drives tumor progression and immune evasion in Hepatocellular Carcinoma: a single-cell and spatial transcriptomic study
              </a>
            </td>
          <td>
            Siyu Tao, Yunhao Yang
          </td>
          <td>2025-05-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite sharing >98% genomic similarity, humans are more likely to develop cancers than our closest living ancestors, the nonhuman primates. Here, we unexpectedly discover that, unlike chimpanzee and other primates, a critical embryonic development, immune homeostasis, and general cell-death regulator protein called Fas Ligand (FasL) contains a Pro153-Ser153 evolutionary substitution in humans. The latter renders human FasL preferentially susceptible to cleavage by plasmin, an overly elevated protease in solid tumors. Since FasL-mediated killing of tumor cells by activated T-lymphocytes and chimeric antigen receptor T-cells (CAR-T) is critical for therapeutic efficacy, we find that elevated plasmin levels in certain ovarian tumors interfere with the T-lymphocyte-expressed FasL death signaling. Either targeted inhibition or blocking plasmin accessibility to membrane FasL rescues the FasL cell-death function of activated T-lymphocytes in response to immune-checkpoint receptor targeting antibodies. These findings of evolutionary significance highlight that elevated plasmin in metastatic tumors potentially contributes to differential outcomes of T-cell-based immunotherapies in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c85bd33f93a168afeeb2c17f47d255562fb6e17f" target='_blank'>
              Evolutionary regulation of human Fas ligand (CD95L) by plasmin in solid cancer immunotherapy
              </a>
            </td>
          <td>
            Brice E. N. Wamba, Tanmoy Mondal, Francis Freenor V, Mehr Shaheed, Oliver Pang, Daniel Bedinger, Patrick Legembre, Laurent Devel, Sanchita Bhatnagar, G. Leiserowitz, Jogender Tushir-Singh
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33a0442e636fb2e64ac9c51076d57138856ca6c8" target='_blank'>
              Mathematical Modeling of Cancer Cell Invasion of Tissue: Development of an in silico Organotypic Assay
              </a>
            </td>
          <td>
            Vivi Andasari, Alf Gerisch, A. South, Mark A. J. Chaplain
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, largely due to its pronounced cellular heterogeneity, therapy resistance, and metastatic propensity, traits driven by cancer stem cells (CSCs). Once thought to be a rare, static subpopulation, CSCs are now recognized as a dynamic, plastic cell state influenced by both intrinsic and extrinsic factors. In this review, we explore the “inner” (genetic, cellular, and metabolic) and “outer” (microenvironmental and macroenvironmental) determinants that govern CSC identity, emergence, and evolution in PDAC. We focus on how acinar and ductal cell plasticity, in combination with oncogenic mutations (e.g., KRAS, TP53), contribute to tumor initiation and CSC generation, how genomic instability further fuels CSC heterogeneity and adaptability, and we highlight how micro and macro TME properties, metabolism and external factors regulate CSC behavior afterwards. Altogether, CSCs exist at the nexus of genetic chaos and environmental cues. Understanding this complex interplay is key to developing therapies that not only eradicate CSCs but also prevent their reemergence. While still far from clinical application, disrupting CSC-supportive networks should be a research and clinical focus for improving outcomes for PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08e2a3f4aedf4a7111928c29cee984777f2e319" target='_blank'>
              Inner and Outer Determinants of Pancreatic Cancer Stem Cells
              </a>
            </td>
          <td>
            J. C. López-Gil, Diego Navarro, B. Sainz
          </td>
          <td>2025-06-16</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/503cca6deb2a91fe468e2c781a6fa64738c62b22" target='_blank'>
              Drug resistance and tumor heterogeneity: cells and ensembles
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-05-31</td>
          <td>Biophysical Reviews</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="Liver diseases encompass a wide range of etiologies and involve highly heterogeneous cellular environments, yet the specific cellular states through which genetic risk contributes to disease remain incompletely understood. In this study, we integrated genome-wide association study (GWAS) data from six liver diseases and two metabolic traits with transcriptomic profiles of approximately 168,000 human liver cells at single-cell resolution, using the single-cell disease relevance score (scDRS) approach. Our results revealed that disease-associated genetic signals are predominantly localized to non-parenchymal cells—particularly liver sinusoidal endothelial cells (LSECs), cholangiocytes, and specific subsets of lymphocytes. Notably, we identified marked intra-cell-type heterogeneity, with disease associations confined to specific subpopulations exhibiting immune activation or stress-responsive transcriptional programs. For example, autoimmune and viral liver diseases were linked to immunologically active LSECs and cholangiocytes, whereas their metabolically active counterparts showed no enrichment. These findings highlight the necessity of resolving liver cell complexity to uncover the functional basis of genetic risk and suggest that susceptibility to liver disease is driven by specialized cell states within broader cellular categories. Our study provides a refined cellular map of liver disease susceptibility, offering new perspectives for understanding pathogenic mechanisms and informing targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/192d524a339350f75aa0e4bcaad1a36c461732e3" target='_blank'>
              Dissecting the Cellular Heterogeneity Underlying Liver Diseases Through the Integration of GWASs and Single-Cell RNA Sequencing
              </a>
            </td>
          <td>
            Miao Zhou, Meng Liu, Chao Xue
          </td>
          <td>2025-06-27</td>
          <td>Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Tumor resistance is the primary reason for treatment failure in patients with cancer, while oncolytic viruses (OVs), as a novel therapy, have been rapidly advancing through clinical evaluation and are typically assessed in recurrent tumors that are refractory to standard chemotherapy. However, whether the adaptive process that fosters chemotherapy resistance influences the efficacy of OV therapy is unknown. Methods We analyzed chemo-resistant colorectal cancer (CRC) using in vitro, in vivo, and patient-derived organoid models to assess sensitivity to OVs. Through RNA sequencing analysis and immunohistochemistry were performed in clinical samples that indicated TANK-binding kinase 1 (TBK1) expression. Using single-cell RNA sequencing, flow cytometry, and in vivo neutralization assays to demonstrate that the combination of TBK1 inhibitor (TBK1i) and OVs reprograms the tumor immune microenvironment, particularly by activating natural killer (NK) cells. Through RNA sequencing analysis, we identified intercellular cell adhesion molecule-1 (ICAM-1) as a potential target responsible for NK cell activation. Subsequently, we designed and conducted rescue experiments, both in vitro and in vivo, to validate the influence of ICAM-1 on NK cell activity. Results We demonstrated that chemo-resistant CRC showed decreased sensitivity to the OV in vitro, in vivo, and patient-derived organoids. Further investigation revealed aberrant activation of TBK1 in chemo-resistant CRC, which mediated the activation of the type I interferon pathway and impaired viral replication. TBK1 inhibition enhanced intratumor viral replication and direct oncolysis effect in vitro and augmented the antitumor immunity elicited by OVs in vivo. Immune cell profiles presented that OV/TBK1i combination reshaped the tumor microenvironment and especially activated the NK cell response. Immune cell depletion studies demonstrated that NK cells were required for the synergistic therapeutic activity of the OV/TBK1i combination. Mechanistically, TBK1 inhibition synergized with VSVΔ51 to increase ICAM1 expression in a RIPK1-dependent manner, promoting NK cell-mediated tumor killing. Conclusion This study presents a promising approach for treating chemo-resistant CRC by combining OVs and TBK1i.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad16bd40220efd0dd4a029d8c16e8f0ba36e528" target='_blank'>
              Targeting TBK1 potentiates oncolytic virotherapy via amplifying ICAM1-mediated NK cell immunity in chemo-resistant colorectal cancer
              </a>
            </td>
          <td>
            Xin Guo, Huolun Feng, Zhihui Xi, Ji Zhou, Zuda Huang, Jieqing Guo, Jiabin Zheng, Zejian Lyu, Yongfeng Liu, Jianlong Zhou, Yucheng Zhang, Yuhan Zhang, Yong Li, Fan Xing
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Gallbladder cancer (GBC) is a highly aggressive malignancy with a propensity for lymph node metastasis (LNM), which significantly worsens prognosis. This review explores the molecular mechanisms underlying LNM in GBC, focusing on the roles of vascular endothelial growth factors (VEGFs), chemokines, cancer-associated fibroblasts (CAFs), tumor-associated macrophages (TAMs), hypoxia-inducible factors (HIFs), and non-coding RNAs (ncRNAs) in shaping the tumor microenvironment (TME). Unique features of GBC, such as its bile-rich microenvironment and hypoxia-driven lymphangiogenesis, are highlighted. We discuss how these factors promote lymphangiogenesis, immune evasion, and extracellular matrix (ECM) remodeling, collectively facilitating LNM. Potential therapeutic targets, including VEGF-C/D pathways, matrix metalloproteinase (MMP) inhibitors, and immune-modulating therapies, are also reviewed. Future research integrating single-cell omics and patient-derived organoid models is essential for advancing precision medicine in GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74624732c38b10967280fe625753908628ae6ef1" target='_blank'>
              Molecular Mechanisms of Lymph Node Metastasis in Gallbladder Cancer: Insights into the Tumor Microenvironment
              </a>
            </td>
          <td>
            Qingyu Tang, Yichen Guan, Yubo Ma, Qi Li, Zhimin Geng
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="This study aims to investigate metabolic reprogramming heterogeneity in hepatocellular carcinoma (HCC) cells and identify novel therapeutic targets for HCC treatment. Single-cell RNA sequencing data from public databases were used to analyze the TME of HCC and reveal the characteristics of different cell subsets, including mononuclear phagocytes, epithelial cells, endothelial cells, NK/T cells, B cells, and unknown cells. The analysis revealed that these cell subsets play their own unique roles in tumor progression and immune escape. Analysis of copy number variations (CNVs) was performed on tumor-derived epithelial cells, with the epithelial cells in Cluster 3 subgroup showing the highest CNV levels. Gene Ontology (GO) enrichment analysis revealed that these cell subsets were involved in a variety of biological processes such as immune response, cell communication, and metabolic pathways, which were consistent with their functional roles. Pseudotemporal analysis further delineated the malignant trajectory of HCC cells, with Cluster 3 exhibiting enhanced phosphatidylinositol metabolism, suggesting a critical role for metabolic reprogramming in tumor invasion and proliferation. Furthermore, a diagnostic model incorporating metabolic reprogramming-associated gene signatures was established, which effectively distinguished HCC from normal tissues. Among these signatures, splicing factor 3a subunit 3 (SF3A3) was identified as both diagnostic and independent prognostic biomarker. Mechanistically, SF3A3 knockdown in HCC cell lines significantly suppressed proliferation, migration, PI3K/AKT signaling, and EMT marker expression, thereby demonstrating its role in driving HCC aggressiveness. In conclusion, these findings elucidate novel molecular characteristics of HCC based on metabolic reprogramming, while establishing SF3A3 as a promising multi-faceted target for HCC diagnosis, prognostic assessment, and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37f071868689d97b4d0d4b6a9e7cefb9b68e3b5" target='_blank'>
              Exploring the level of metabolic reprogramming and the role of prognostic factor SF3A3 in hepatocellular carcinoma through integrated single-cell landscape analysis
              </a>
            </td>
          <td>
            Wanshuo Wei, Yuan Gan, Xindan Zhang, Yumo Chen, Zengfeng Huang, Shuhan Wang, Xiaomei Xie, Yongle Li, Pengtao Qin, Lihe Jiang
          </td>
          <td>2025-05-27</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [27],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>