<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-29 09:39:23 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Melanoma outcomes have dramatically improved over the past decade, but some patients still experience disease recurrence, particularly those who present at later stage of disease. Prior studies identified an altered immune microenvironment in the sentinel lymph node (SLN) including the presence of dysfunctional CD8 T cells and CD4 T regulatory cells (Tregs), but the spatial organization of the SLN and the interactions of individual cell types have not been extensively studied. To understand how spatial organization of immune cells in the SLN is related to outcomes, we performed spatial proteomic profiling of Stage I and II melanoma SLN to generate an atlas of cell types. Following multiplexed immunofluorescence imaging, deep learning-based segmentation and clustering, we identified 33 subsets of T cells, B cells and Tumor cells. Lymphoid region analyses established a foundational spatial map of the melanoma SLN, revealing higher regional frequencies of activated and memory CD4 T cells in Stage I SLN and consistent positioning of T cells at functional spatial locations. To better understand cellular interactions, we evaluated the immediate neighbors around each index cell using Effect Size Interaction mapping (ESI-map), a novel computational toolkit for understanding spatial interactions. Nearest-neighbor analyses across biological replicates revealed rewiring in cell-cell interacting pairs across disease progression and patient outcomes, particularly with respect to exhausted TOX+ CD8 T cells. Stage II patients who experienced disease recurrence had notable enrichment of cellular interactions between Tregs and TOX+ CD8 T cells that was associated with reduced expression of granzyme B and interferon-γ that was most pronounced in exhausted CD8 T cells. Together, these data demonstrate that the specific spatial interactions between Tregs and exhausted CD8 T cells in the SLN provide a novel immune signature for understanding melanoma outcomes. One Line Summary Treg-CD8 T cell interactions in the sentinel lymph node predict recurrence outcomes in Stage II SLN+ melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927407a38e9474aa6152c45daf9e0743472dc6ee" target='_blank'>
              Cellular interactions in the sentinel lymph node predict melanoma recurrence
              </a>
            </td>
          <td>
            Sabrina M. Solis, Yang Yang, Yee Hoon Foong, Anushka Dheer, K. Ma, M. S. Farooq, Shira G. Rosenberg, Sophie L. Gray-Gaillard, G. Karakousis, Xiaowei Xu, Rami S. Vanguri, Alexander C. Huang, R. Herati
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d2c494a1cd657524f51f9577b21d3baa396ea7" target='_blank'>
              Spatiotemporal regulation of cell cycle states within the complex tumor microenvironment
              </a>
            </td>
          <td>
            Gianni Zanardelli, Olivier Tassy, Maulik K. Nariya, Nacho Molina
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial transcriptomics has emerged as a transformative technology in biomedical research, offering unprecedented insights into gene and protein expression within their native tissue context. Unlike conventional bulk or single-cell sequencing approaches, spatial omics has the advantage of preserving the spatial structure of tissues, allowing researchers to directly map molecular information onto histological structures. This review provides an overview of the current state of spatial omics technologies, highlighting their application in cancer research. Spatial omics has enabled detailed characterization of the tumor microenvironment (TME), revealing spatial heterogeneity, immune cell infiltration patterns, and complex mechanisms of tumor progression and therapy resistance across various cancer types. The review covers future directions, including artificial intelligence–driven analytics, improved standardization, and cost reduction to accelerate clinical translation. Ultimately, spatial omics is poised to play a central role in precision oncology, enabling a deeper understanding of tumor biology and informing more effective individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a77ce136fd5fd93d240b66bc2895fcb841db1de" target='_blank'>
              Spatial omics: applications and utility in profiling the tumor microenvironment
              </a>
            </td>
          <td>
            Ji-Eun See, Sarah Barlow, Wani Arjumand, Hannah DuBose, Felipe Segato Dezem, Jasmine T. Plummer
          </td>
          <td>2025-12-01</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad9383618813c7fb20bfa893bf5368770766ebe" target='_blank'>
              Analysis of intratumoral immune heterogeneity reveals spatial organisation of immunosuppression in breast cancer
              </a>
            </td>
          <td>
            Christina Metoikidou, P. Bonté, Vadim K. Karnaukhov, Laëtitia Lesage, L. Djerroudi, André Nicolas, P. Sirven, A. Vincent-Salomon, Olivier Lantz, Emanuela Romano, Sebastián Amigorena
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Protein spatial imaging, rooted in spatial proteomics, enables the localization of proteins within their native environment, thus preserving structural and functional context. Its integration into cancer research has provided insights into tumor progression, especially in breast cancer, where ductal carcinoma in situ (DCIS) may transition into invasive breast cancer (IBC). Multiplexed ion beam imaging by time-of-flight (MIBI-TOF) has emerged as a powerful method to visualize protein distribution, tumor–stroma interactions, and immune cell dynamics. Evidence indicates that the tumor microenvironment (TME)—including stromal architecture, extracellular matrix (ECM), and immune cell infiltration—plays a central role in malignant progression. Here, we highlight recent advances in protein spatial imaging, discuss optimization strategies, and examine its implications for breast cancer progression and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a7eef8cc6668ecc3d6de49f721002ee2756192" target='_blank'>
              Spatial Proteomics Imaging to Decode Disease Progression: A Perspective on Tumor Microenvironments
              </a>
            </td>
          <td>
            Haonan Zhang
          </td>
          <td>2025-11-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="While advances in single-cell genomics have helped to chart the cellular components of tumor ecosystems, it has been more challenging to characterize their specific spatial organization and functional interactions. Here, we combine single-cell RNA-seq, spatial transcriptomics by Slide-seq, and in situ multiplex RNA analysis to create a detailed spatial map of healthy and dysplastic colon cellular ecosystems and their association with disease progression. We profiled inducible genetic CRC mouse models that recapitulate key features of human CRC, assigned cell types and epithelial expression programs to spatial tissue locations in tumors, and computationally used them to identify the regional features spanning different cells in the same spatial niche. We find that tumors were organized in cellular neighborhoods, each with a distinct composition of cell subtypes, expression programs, and local cellular interactions. Comparing to scRNA-seq and bulk RNA-seq data from human CRC, we find that both cell composition and layout features were conserved between the species, with mouse neighborhoods correlating with malignancy and clinical outcome in human patient tumors, highlighting the relevance of our findings to human disease. Our work offers a comprehensive framework that is applicable across various tissues, tumors, and disease conditions, with tools for the extrapolation of findings from experimental mouse models to human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af74f8f0f1a2428bab07b6671a3e436ab63d622f" target='_blank'>
              Spatially defined multicellular functional units in colorectal cancer revealed from single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            I. Avraham-Davidi, Simon Mages, J. Klughammer, Noa Moriel, Shinya Imada, Matan Hofree, Evan Murray, Jonathan Chen, Karin Pelka, Arnav Mehta, Genevieve M Boland, T. Delorey, Leah Caplan, Danielle Dionne, Robert Strasser, Jana Lalakova, Anežka Niesnerová, Hao Xu, Morgane Rouault, I. Tirosh, Hacohen Nir, Fei Chen, Omer Yilmaz, Jatin Roper, O. Rozenblatt-Rosen, Mor Nitzan, A. Regev
          </td>
          <td>2025-12-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>197</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f7dcdd3510de0385797859e77f339a9cbb0192e" target='_blank'>
              Multidimensional Single-Cell Transcriptomic Profiling of Uterine Leiomyosarcomas Identifies Molecular Subtypes with Distinct Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Maria Korah, J. Agolia, Biren Reddy, R. Flojo, Amanda Gonçalves, Lilin Wang, Wesley Bobst, Brooke Liang, Kaylin A. Yip, Rosyli Reveron-Thornton, Chuner Guo, Robert Gruener, Andrea E. Delitto, Peter Y. Xie, Angela D. Tabora, Beatrice J. Sun, Sara K. Daniel, Babak Litkouhi, Oliver Dorigo, Nam Q. Bui, Minggui Pan, E. Moding, A. Kalbasi, Amanda R. Kirane, George A. Poultsides, Jeffrey A. Norton, Aaron M. Newman, G. Charville, K. Ganjoo, Deshka S. Foster, R. Stephanie Huang, Byrne Lee, Michael T. Longaker, Daniel J. Delitto
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Integration of scRNA-seq data from millions of cells revealed a high diversity of cell types in the healthy and diseased human lung. In a large and complex organ, constantly exposed to external agents, it is crucial to understand the influence of lung tissue topography or external factors on gene expression variability within cell types. Here, we apply three spatial transcriptomics approaches, to: (i) localize the majority of lung cell types, including rare epithelial cells within the tissue topography, (ii) describe consistent anatomical and regional gene expression variability within and across cell types, and (iii) reveal distinct cellular neighborhoods in specific anatomical regions and examine gene expression variations in them. We thus provide a spatially resolved tissue reference atlas in three representative regions of the healthy human lung. We further demonstrate its utility by defining previously unknown imbalances of epithelial cell type compositions in chronic obstructive pulmonary disease lungs. Our topographic atlas enables a precise description of characteristic regional cellular responses upon experimental perturbations or during disease progression. Using multiplexed spatially resolved transcriptomic approaches, the authors generate a topographic atlas of the healthy adult lung with location-related gene expression variability or neighbourhood changes as exemplified by COPD patient sample analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe80d855be30beaf44c2810f98505d59172a3d0" target='_blank'>
              Spatial single-cell atlas reveals regional variations in healthy and diseased human lung
              </a>
            </td>
          <td>
            A. Firsova, Sergio Marco Salas, L. B. Kuemmerle, X. Abalo, A. Sountoulidis, L. Larsson, Krishnaa T. Mahbubani, Jonas Theelke, Žaneta Andrusivová, Leire Alonso Galicia, A. Liontos, Tamás Balassa, Ferenc Kovács, Peter Horvath, Yuexin Chen, Janine Gote-Schniering, M. Stoleriu, Jürgen Behr, Kerstin B. Meyer, W. Timens, Herbert B. Schiller, Malte D. Luecken, F. Theis, J. Lundeberg, Mats Nilsson, M. Nawijn, Christos Samakovlis
          </td>
          <td>2025-11-05</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf21fbcae37403e9935121738032d46b9b3bab3c" target='_blank'>
              ProMPt: A modular preclinical platform for functional modelling of prostate cancer heterogeneity and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Nicole Pandell, Jacob Househam, Matteo Tartagni, Archana Thankamony, Florian Gabel, Roberto Rota, Elizabeth Flittner, B. Gurel, I. Figueiredo, G. Seed, A. Neeb, Mary Chol, W. Yuan, J. Clohessy, C. Tape, A. Sharp, J. D. De Bono, Marco Bezzi
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076f7d290241ef13795e81746d2f0f634b36afeb" target='_blank'>
              Star-Motifs: Revealing Single-Cell Spatiotypes from Routine Histology Using Star-Convex Neighborhoods
              </a>
            </td>
          <td>
            G. N. Gunesli, Muhammad Dawood, Lawrence S. Young, F. Minhas, S. E. A. Raza, N. Rajpoot
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: Intratumour heterogeneity (ITH) is one of the key characteristics of cancer and is closely associated with patient prognosis, treatment resistance, and tumor metastasis. Nevertheless, the study of ITH in hepatocellular carcinoma (HCC) remains limited. Methods: The present study elucidated the influence of ITH on the tumor microenvironment (TME) in HCC. We applied Non-negative Matrix Factorization (NMF) analysis to a cohort of 78 single-cell RNA sequencing (scRNA-seq) HCC samples to systematically characterize ITH. Furthermore, by integrating spatial transcriptomics (ST) data from five HCC patients, we comprehensively analyzed the spatial organization and functional properties of distinct niches within HCC. We conducted a detailed analysis of the cell-type co-localization relationships within the TME and constructed a comprehensive atlas of HCC spatial organization. Results: We observed a co-localization relationship between hypoxia tumor cells, plasmalemma vesicle-associated protein (PLVAP+) endothelial cells (EC), and vascular endothelial growth factor A (VEGFA+) cancer-associated fibroblasts (CAF), suggesting a key role for hypoxia tumor cells in VEGFA+ CAF transformation and tumor angiogenesis. We identified a unique boundary region enriched with dendritic cells1 (DC1), interferon-expressing tumor cells, lymphatic EC, C–X–C Motif Chemokine Ligand 10 (CXCL10+) macrophages (Mac), and secreted phosphoprotein 1 (SPP1+) Mac located between the tumor-infiltrating immune cells and tumor regions. Furthermore, we found that CXCL10+ Mac and SPP1+ Mac, despite co-localizing in the boundary region, exhibit distinct functions, which may be attributed to their unique spatial locations, with the former being closer to the immune-infiltrated region and the latter more proximal to the tumor area. Conclusions: Our study highlights the critical role of spatial interactions between tumor cells and the microenvironment in HCC. The findings offer new insights into ITH and underscore the importance of spatial organization in understanding cancer biology and designing future precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e826ac6739dc6ba19b8b336aa7292a375efd8d3e" target='_blank'>
              Integrative Analysis of Single-Cell and Spatial Transcriptomics Reveals Intratumor Heterogeneity Shaping the Tumor Microenvironment in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-12-01</td>
          <td>Livers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Risk stratification remains a critical challenge in non-small cell lung cancer patients for optimal therapy selection. In this study, we develop an artificial intelligence-powered spatial cellomics approach that combines histology, multiplex immunofluorescence imaging and multimodal machine learning to characterize the complex cellular relationships of 43 cell phenotypes in the tumor microenvironment in a real-world retrospective cohort of 1168 non-small cell lung cancer patients from two large German cancer centers. The model identifies cell niches associated with survival and achieves a 14% and 47% improvement in risk stratification in the two main non-small cell lung cancer subtypes, lung adenocarcinoma and squamous cell carcinoma, respectively, combining niche patterns with conventional cancer staging. Our results show that complex immune cell niche patterns identify potentially undertreated high-risk patients qualifying for adjuvant therapy. Our approach highlights the potential of artificial intelligence powered multiplex imaging analyses to better understand the contribution of the tumor microenvironment to cancer progression and to improve risk stratification and treatment selection in non-small cell lung cancer. Risk stratification in non-small cell lung cancer (NSCLC) remains challenging. By combining multiplex immunofluorescence, H&E histology, and AI, the study identifies spatial “cell-niche” patterns that enhance survival prediction beyond UICC8 staging. These patterns reclassify many stage I patients as high risk, revealing potentially undertreated cases and establishing spatial tumor microenvironment features as clinically actionable biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53bd1a5c6021c6b38d3074a4cf442e836ada177" target='_blank'>
              AI-powered spatial cell phenomics enhances risk stratification in non-small cell lung cancer
              </a>
            </td>
          <td>
            Simon Schallenberg, Gabriel Dernbach, Sharon Ruane, Philipp Jurmeister, Cornelius Böhm, Kai Standvoss, Sandip Ghosh, M. Frentsch, M. Dragomir, Philipp Keyl, Corinna Friedrich, Il-Kang Na, Sabine Merkelbach-Bruse, A. Quaas, N. Frost, Kyrill Boschung, Winfried Randerath, G. Schlachtenberger, Matthias Heldwein, Ulrich Keilholz, Khosro Hekmat, Jens Rückert, R. Buettner, Angela Vasaturo, David Horst, Lukas Ruff, M. Alber, Klaus-Robert Müller, Frederick Klauschen
          </td>
          <td>2025-11-03</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern, and a high relapse rate. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, where malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, we identify pericytes as the most active paracrine signaling hub within the tumor parenchyma. Their depletion through genetic engineering results in accelerated tumor progression and shortened survival. Mechanistic studies reveal that pericyte deficiency remodels the endothelium and impacts the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, the pericyte-deprived endothelium recruits perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages express Hepatocyte Growth Factor, which reinforces activation of its receptor tyrosine kinase MET on GBM cells harboring a pronounced mesenchymal subtype driven by the key phenotypic regulator Fosl1. Indeed, orthotopic implantation of MET-expressing GBM cells corroborates their superior tumor-initiating and invasive capabilities. Thus, pericytes represent critical modulators of GBM development by orchestrating a tumor-suppressive microenvironment, highlighting the importance of their preservation in therapy. Improved understanding of the tumor ecosystem in glioblastoma is critical for developing new treatment strategies for the disease. Here, the authors identify pericytes as an active paracrine signaling hub within the tumor parenchyma that orchestrates a tumor-suppressive microenvironment, suggesting pericyte preservation as a key feature of future therapeutic regimens for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09a05790107e087a632df06714ceec3176e20b4" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, Mehrnaz Safaee Talkhoncheh, B. Phung, E. Cordero, R. Rosberg, E. Johansson, G. Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumors are composed of cellular populations with distinct genotypes and phenotypes, which dynamically evolve over time and during treatment. This process is known as clonal evolution, and it is difficult to reveal fine-scale clonal structure with traditional bulk sequencing. Although single-cell genome sequencing could enable reconstruction of tumor clonal evolution, it remains technically challenging and the number of single cells profiled is generally insufficient due to high cost. To address this issue, we developed scClone, a computational toolkit that integrates variant detection and genotype inference for single-cell RNA-seq (scRNA-seq) and spatial transcriptomic data. It further provides interactive visualization of clonal structure and dynamic evolution. scClone addresses key limitations inherent to scRNA-seq, such as expression drop-out and allelic imbalance, and incorporates cell type or state annotation with mutational signature analysis to enable comprehensive profiling of tumor heterogeneity. scClone demonstrated robust performance across multiple datasets—generated from both full-length and fragmented RNA sequencing—by accurately reproducing mutation profiles and resolving clonal mixtures in myeloma, hepatocellular carcinoma and pancreatic cancer. Additionally, scClone has been applied to spatial transcriptomics, enabling the delineation of clonal structures within histological sections from ovarian cancer and cutaneous squamous cell carcinoma. In summary, our results demonstrate that scClone can extract genetic information from scRNA-seq datasets, thereby successfully establishing genotype–phenotype associations at the single-cell level and providing insights into the clonal evolution of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d470f131086238e06c10a786c9bfc7fb86a41116" target='_blank'>
              De Novo Detection of Clonal Structure and Evolution in Single-Cell and Spatial Transcriptomes
              </a>
            </td>
          <td>
            Shihao Bai, Xianbin Su, Ziyao Chen, Ze-Guang Han
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Mouse models are pivotal for translational research, enabling the detection of therapeutic-induced immune modifications within the tumor microenvironment (TME). We utilized a 40-marker panel targeting tumor and immune markers to investigate the spatial protein composition of the whole mouse embryo, mouse brain and mouse glioblastoma (GBM) tumor sections. Imaging Mass Cytometry™ (IMC™) technology was employed to capture the expression of 40 markers simultaneously while preserving tissue integrity and enabling a flexible workflow without autofluorescence artifacts. Data analysis revealed distinct spatial patterns of tumor and immune markers across the GBM, identifying key features such as necrotic cores, regions of high immune infiltration, proliferating tumor cells and diverse populations of infiltrating immune cells. The wide detection range of expression provided by IMC technology enabled visualization and analysis of clusters exhibiting varying levels of key GBM markers, suggesting a probable dual origin of the GBM, coexistence of pro- and anti-tumorigenic immune responses and overall extreme heterogeneity that can only be visualized with high-plex antibody panels. This approach demonstrates the capability of IMC technology to reveal the complex cellular and structural spatial landscape of the TME in GBM, providing valuable insights into tumor biology and potential therapeutic targets for patient stratification. For Research Use Only. Not for use in diagnostic procedures.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d885a41b02f8da25eb39b11abd7a0e0030ab5b6" target='_blank'>
              Characterization of cellular diversity and immune landscape in Mouse Glioblastoma using whole slide imaging modes of high-plex Imaging Mass Cytometry 2619
              </a>
            </td>
          <td>
            Qanber Raza, Nick Zabinyakov, T. Pfister, Nikesh Parsotam, David Howell, Liang Lim, James Pemberton, Yun-Jyh Chwee, C. Loh
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34b247d052e44baf977776483c7e044dbe0ae158" target='_blank'>
              Mechanisms of tumor persistence in metastatic melanoma following successful immunotherapy
              </a>
            </td>
          <td>
            Yingxiao Shi, Z. Maliga, Tuulia Vallius, Shishir M. Pant, R. Pelletier, Brigette Kobs, Solanky Priyanka, Yiwen He, Eliezer Van Allen, S. Santagata, P. Ott, C. G. Lian, Elizabeth I. Buchbinder, David Liu, P. Sorger
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Colorectal cancer (CRC) brain metastases have a poor prognosis and limited treatment options, including resistance to radiation therapy. Little is known about the molecular and cellular mechanisms that enable CRC tumor cells to adapt to the brain and establish a supportive tumor microenvironment. To address this gap we used spatial transcriptomics to analyze 51 CRC brain metastases. A subset had matched primary colon tumors and longitudinally paired metastatic resections before and after radiation treatment. We identified the critical spatial cellular features of the tumor epithelium and the surrounding tumor microenvironment that support metastatic growth in the brain. CRC brain metastases developed a stromal microenvironment with abundant fibroblasts and tumor-associated macrophages. A fibroblast–macrophage cellular neighborhood promoted angiogenesis, extracellular matrix remodeling, and immune suppression. Tumor cells showed local adaptations. In endothelial-rich regions, they were proliferative whereas in macrophage-rich regions, they were more differentiated and immune evasive. Compared with paired primary tumors, CRC brain metastases showed increased chromosomal instability, with activation of RNA-processing, stress response, and junctional remodeling pathways. After radiation treatment, resistant clones had increased epithelial–mesenchymal transition, while the immunosuppressive stroma remained intact. We identified tumor-derived MIF, GDF15, PRSS3 and SEMA3C ligands and macrophage-derived SPP1 that have the potential to affect multiple cell types in the metastatic niche. These ligand-receptor interactions drive angiogenesis, stromal activation and immune suppression. In a macrophage–tumor–fibroblast co-culture model, knockout of SPP1 in macrophages led to reduced expression of lipid-metabolism related genes and disrupted tumor-promoting interactions. Together, these results indicate that CRC growth in the brain is sustained by a specific cellular organization with immunosuppressive multicellular interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121b9f0cf30a6fb662f64416fefc808a183dcc03" target='_blank'>
              Colorectal cancer relies on an immunosuppressive cellular topography and genomic adaptations for establishing brain metastases
              </a>
            </td>
          <td>
            A. Sathe, Mengrui Zhang, Xiangqi Bai, Ji In Kang, Rithika Meka, Huiyun Sun, Susan M. Grimes, Aparajita Khan, Mingen Liu, A. Luksik, M. Lim, Claudia K. Petrisch, Christopher M. Jackson, Hannes Vogel, Jeanne Shen, M. Gephart, Summer S. Han, Hanlee P. Ji
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 A major barrier in understanding adult diffuse gliomas lies in the inability to track tumor evolution in real time. To address this, we employed a novel method known as continuous lineage tracing, which integrates CRISPR/Cas9-based expressed DNA barcoding with single-cell RNA sequencing, enabling a phylogenetic approach to studying tumor development. Patient-derived glioma-initiating cell lines were engineered with expressed barcodes targeted by CRISPR/Cas9 and implanted into mice to create intracranial xenografts. Tumors underwent single-cell RNA sequencing; expressed barcodes were used to infer clonal relationships, and transcriptomic profiles enabled cell state classification. Phylogenetic lineage trees were reconstructed using lineage inference algorithms to characterize cell fitness, expansion, and plasticity. Our analysis uncovered extensive intra-clonal cell state heterogeneity, indicating active phenotype switching prior to therapy. We identified consistent transcriptional programs associated with tumor engraftment and in vivo clonal advantage. Lineage tracing revealed gene expression signatures linked to key phenotypes: fitness, enriched for neural-mesenchymal and injury-response pathways; expansion, associated with RNA splicing; and plasticity, correlated with cell cycle and DNA repair programs. Glioma stem cells appeared to span a transcriptional continuum from undifferentiated, high-fitness states to more differentiated, low-fitness states, with high-fitness cells potentially representing transitional phenotypes. We then validated these findings in a cohort of 185 surgically resected gliomas with matched bulk RNA and proteomic data. Phylogenetic gene signatures differed markedly between IDH-mutant and IDH-wildtype tumors. When stratified by tumor type, both fitness and expansion signatures were significantly associated with overall survival in GBM (median 12.6 months, p=0.041) and oligodendrogliomas (median 66 months, p=0.027). This study demonstrates the utility of continuous lineage tracing to reconstruct tumor evolution and identify transcriptional programs linked to tumor growth and prognosis. Our approach provides a powerful framework for dissecting glioma biology and identifying potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d67c2559a40c369b3a39bc8fe0aad62196164a" target='_blank'>
              STEM-24. SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS, EXPANSION AND PLASTICITY AXES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Ajisebutu, Xiaojun Fan, Chloe Gui, Jeff Liu, Vikas Patil, A. Landry, Y. Ellenbogen, L. Yefet, Phooja Persaud, F. Nassiri, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Neuroblastoma (NB), the most common extracranial solid tumor in children, is characterized by remarkable cellular heterogeneity and clinical variability ranging from spontaneous regression to aggressive progression and relapse. Despite advances in multimodal therapies, including surgery, chemotherapy, radiotherapy, differentiation therapy, and immunotherapy—treatment resistance remains the principal barrier to improving survival in high-risk patients. Recent single-cell and spatial multi-omics studies have revolutionized our understanding of NB by revealing its developmental origins, lineage hierarchy, and adaptive evolution under therapeutic pressure. These technologies have delineated distinct cellular states along an adrenergic–mesenchymal continuum and uncovered the dynamic interplay between tumor cells and their microenvironment. Genetic instability, epigenetic reprogramming, and metabolic plasticity cooperate with immune and stromal remodeling to drive tumor persistence and relapse. At the molecular level, mechanisms such as MYCN-driven chromatin remodeling, super-enhancer reorganization, bypass signaling activation, quiescent persister programs, immune checkpoint engagement, and metabolic rewiring collectively enable therapeutic escape. Importantly, these processes are reversible, highlighting tumor plasticity as both a hallmark and a potential vulnerability of NB. Integrating single-cell transcriptomics, epigenomics, and spatial profiling provides an unprecedented framework to map resistance evolution, identify lineage-specific vulnerabilities, and guide rational combination strategies. Targeting epigenetic regulators, metabolic checkpoints, and immune suppressive networks in a temporally coordinated manner holds promise for converting NB from an adaptive to a controllable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance
              </a>
            </td>
          <td>
            Guoqian He, Si-Jia He, Xiao-yu Jing, Yi-Ling Dai, Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a5fb32298b06f7826592aac1dafed7fb246a78" target='_blank'>
              Genotype-phenotype correlations and de novo induction of cancer stem cells in Wilms tumor initiation
              </a>
            </td>
          <td>
            N. S. Pop, D. Koot, C.M. Brouwers, M. Linssen, J. Claassens, C.W.J. Cartlidge, D.D. Özdemir, K. S. Dolt, P. Hohenstein
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdb3fc96051ec080fed6bcf88bde02391e145c9" target='_blank'>
              Integrative spatial multi-omics reveals prognostic tumor niches in female genital tumors
              </a>
            </td>
          <td>
            Tian Xu, Siyu Lin, Yuling Wu, Yufeng He, Zongxu Zhang, Yanping Zhao, Peng Zhang, Luyang Zhao, Wen Yang, M. Ye, Rui Zhang, Liang Wen, Pengfei Ren, Ce Luo, Zhe Zhang, Zexian Zeng, Yuanguang Meng
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Spatial context is a fundamental driver of intra-tumor heterogeneity in glioblastoma (GBM), shaping the molecular landscape, cellular composition, and therapeutic vulnerabilities. While single-cell profiling has illuminated the diversity of GBM cell states, a critical gap remains in understanding how local microenvironments orchestrate gene expression programs and cellular plasticity within intact tumors. To address this, we analyzed spatial transcriptomics data from 44 GBM specimens, comprising over 128,000 tissue spots, to systematically map the biological programs and cellular architecture of distinct tumor niches. Leveraging machine learning, we derived a unified batch-adjusted data embedding that captured the local tissue context of each spot across all slides, allowing the study of spatially continuous rewiring of molecular programs. This approach enabled us to identify ligand–receptor interactions and transcription factors (TF) with spatially localized activity across tissue slides. Using cross-correlation metrics, we identified multiple TF modules specifically active in certain environments. Notably, the perivascular region exhibited the most diverse transcriptional activity, with multiple TF modules associated with tumor growth, division, and migration processes. Additional modules were uniquely enriched at the leading edge and hypoxic regions, as well as at the tumor–necrosis interface. Importantly, we identified a unique module active at the tumor core–leading edge interface, with corresponding spots enriched for alternative splicing processes, suggesting this could be a core driver of the invasive tumor phenotype. Dynamic gene network inference along a continuous hypoxic gradient uncovered context-specific hub genes and TFs that facilitate tumor metabolic adaptation and cell state shifts. These findings underscore the immense spatial heterogeneity of GBM, characterized by context-specific molecular rewiring of tumor cells driven by complex cellular interactions. Our study not only reveals new vulnerabilities but also highlights the need for spatially targeted combination therapies to disrupt the adaptive spectrum of cancer cell states in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fddeac5d22fe2de68ff3fa47ac318949e6c8ee0" target='_blank'>
              TMIC-118. Identifying spatially-localized biological programs and molecular vulnerabilities in glioblastomas
              </a>
            </td>
          <td>
            V. Ravikumar, Reva Kulkarni, A. Maddox, Arushi Tripathy, Emre Kiziltug, Dah-Luen Huang, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb4486dfc086ad91b487a1c5269e719293310a3" target='_blank'>
              Machine perfusion and single-cell spatial transcriptome mapping identifies novel immune escape mechanisms in colorectal cancer liver metastasis
              </a>
            </td>
          <td>
            Peter Kok-Ting Wan, Ranchu Cheng, David Johnson, Carl Lee, Shihong Wu, Areeb Mian, Ahmet Hazini, Sorayya Moradi, Kate Friesen, Flurin Caviezel, Syed Hussain Abbas, Hatem Sadik, Adil Lakha, Keaton Jones, Girishkumar Kumaran, M. Bottomley, Rob Jones, Mark C. Coles, R. Bashford-Rogers, Constantin Coussios, Len Seymour, Robert Carlisle, Kerry Fisher, Alex Gordon-Weeks
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cervical cancer ranks as the fourth most common malignancy among women worldwide, with approximately 604,000 new cases and 342,000 deaths annually. Despite advances in HPV vaccination and screening programs, significant challenges remain in diagnostic accuracy, therapeutic efficacy, and prognostic assessment. Traditional bulk RNA sequencing methods average gene expression data across entire tissue specimens, masking cellular diversity and spatial architectural patterns that influence tumor progression. Telomere maintenance mechanisms play crucial roles in cervical cancer development, with over 90% of cervical cancers exhibiting telomerase reactivation. This study integrated single-cell RNA sequencing and spatial transcriptomics technologies to construct the first comprehensive spatial molecular atlas of cervical cancer. Single-cell RNA sequencing analysis utilized public datasets from the GEO database with systematic quality control through a multidimensional assessment framework. Spatial transcriptomics analysis employed the 10x Genomics Visium platform, successfully identifying 38 distinct cellular neighborhoods (N1-N38). Spatial expression patterns of 10 key genes were analyzed, including epithelial markers (MUC1, CDH1, KRT16), stromal markers (COL1A1, COMP, DCN), and immune markers (CD3G, FCGR1A). In vitro validation was performed using HeLa and SiHa cell lines. Multiple distinct cell subpopulations and 38 cellular neighborhoods with unique molecular characteristics were successfully identified. MKI67 demonstrated spatial heterogeneity with proliferative “hotspots”; COMP was primarily expressed in stromal regions and participated in tumor-stroma interactions; KRT16 exhibited patterns reflecting epithelial differentiation gradients. Different neighborhoods showed enrichment for various cell types, including immune hotspots, stromal-rich regions, and epithelial-dominant areas. Immunoglobulin-related genes (IGLC2, IGHG1, IGHG2, etc.) displayed unique spatial expression characteristics. Quantitative PCR validation confirmed differential expression patterns between cell lines. This study established the first comprehensive spatial transcriptomic atlas of cervical cancer, revealing unprecedented insights into tumor microenvironment organization and cellular spatial relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a6a98dd06253f9d1bf4fd4787fb52f0b0c3d2f1" target='_blank'>
              Spatial transcriptomic landscape and cellular neighborhood heterogeneity in cervical cancer: integrative single-cell and spatial RNA sequencing analysis
              </a>
            </td>
          <td>
            Dieyi Mo, Huana Cao, Shaojuan Lai, Jinding Luo, Weijie Ye
          </td>
          <td>2025-11-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Background: Mature B-cell neoplasms display densely packed and morphologically heterogeneous cells, often complicating tissue-level analysis. Recent studies have revealed the complexity of the tumor microenvironment (TME) in these malignancies (Roider, Nature Cell Biology 2024; Colombo, Blood Advances 2022), providing insights into prognosis and treatment response. Spatial profiling platforms such as imaging mass cytometry (IMC) enable single-cell, in situ analysis using multiplexed antibody panels, but accurate and interpretable segmentation remains a challenge in lymphoid tissues where cells frequently overlap, deform, and exhibit non-canonical marker profiles. Common tools, including DeepCell Mesmer and Cellpose, assume round, non-overlapping cells with clear boundaries, a simplification that often fails in DLBCL, FL, and cHL (Dufva et al. 2020). These limitations can yield biologically misleading segmentations and restrict precise TME characterization.
 Methods: To address this gap, we developed SegMO (Segmentation and Morphological feature Optimization), an unsupervised, annotation-free segmentation framework compatible with multiple spatial proteomics platforms. SegMO integrates nucleus-anchored seeding, probabilistic modeling of marker expression, cluster-guided refinement, and a reinforcement learning (RL) module to produce biologically realistic, soft whole-cell masks that allow partial overlaps and irregular shapes. Protein expression is modeled with flexible statistical distributions to accommodate partial or co-expressed markers, and cells are clustered by expression features before iterative mask refinement to improve marker purity and spatial coherence. The RL model optimizes boundaries through pixel-level adjustments to maximize cell fidelity, recover low-signal regions, and capture realistic morphology.
 Results: We applied SegMO to 24 IMC samples (3 tonsil, 2 spleen, 9 brain, 10 lymphoma) stained with 43 protein markers, from a larger cohort representing 76 individuals (DLBCL, FL, cHL, and healthy donors). SegMO segmented ~74,000 cells and grouped them into biologically distinct clusters based on marker-driven statistical modeling. Compared to Mesmer, SegMO improved nuclear signal integrity (0.92 vs. 0.90, P < 1×10⁻⁵), detected fewer weak-signal nuclei (<0.1% vs. 2.5%, P < 10⁻⁴), and generated more accurate boundaries in high-density and overlapping regions. Soft segmentation preserved subtle morphological features such as protrusions, compression, elongation, and partial overlaps that rigid models did not capture.
 Morphological profiling showed that Cluster 7, enriched in tumor tissues and likely neoplastic B cells, had one of the largest average cell sizes (mean = 139.95 pixels²) and highest nuclear ratios (mean = 0.65), both significantly greater than Cluster 4 (fibroblasts; mean = 97.6, ratio = 0.59) and Cluster 5 (CD4⁺ T cells; mean = 101.6, ratio = 0.56) from healthy tissue (ANOVA P < 1×10⁻⁶³ for size, P < 1×10⁻⁹¹ for ratio; Tukey HSD P < 0.001). Cluster 1, composed of CD4⁺CD7⁺ lymphocytes, was enriched in the GCB subtype (Wilcoxon P = 0.006), with intermediate size (mean = 127.70), high nuclear ratios (mean = 0.61), and irregular shapes (elongation, asymmetry, or compression) reflecting activation, mobility, and spatial pressure in the TME. These patterns were not detected by standard segmentation tools.
 Beyond cell-level statistics, SegMO enabled spatial visualization of marker-defined clusters and uncovered architectural features including immune–tumor colocalization, irregular clustering, and localized crowding in aggressive lymphomas. The RL module further improved segmentation in low-signal or noisy regions, recovering masks otherwise lost or poorly defined.
 Conclusion: SegMO is a robust, versatile, annotation-free AI framework for single-cell segmentation in multiplex tissue imaging of hematologic samples. By integrating probabilistic modeling, cluster-guided refinement, and reinforcement learning, SegMO produces biologically realistic boundaries in complex TMEs, enhancing both morphological fidelity and marker signal accuracy. This capability supports high-resolution spatial and morphological analysis, enabling precise characterization of tumor-associated features, immune interactions, and proliferative states in lymphomas and other hematologic malignancies. Full-cohort analysis with integrated RL, revealing additional morphological patterns and TME phenotypes, will be presented.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3006a1cfec74931bbbc57b898d74d8cc5186f85" target='_blank'>
              Distinct morphological features of malignant cells in large B-cell lymphoma identified by soft-segmentation in imaging-based spatial proteomics
              </a>
            </td>
          <td>
            Junming Shi, T. Noordenbos, M. Ijsselsteijn, Noel F.C.C. de Miranda, J. Vermaat, Mohammad Shahrokh Esfahani
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, M.-R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Introduction Breast cancer (BC) prognosis is fundamentally dictated by its pronounced molecular and cellular heterogeneity. While single-cell RNA sequencing (scRNA-seq) has detailed the tumor microenvironment (TME) cellular composition, a critical knowledge gap persists regarding the vascular heterogeneity of tumor endothelial cells (TECs) across different BC subtypes. Current TEC classification remains conventional, obscuring whether subtype-specific TECs possess unique transcriptomic signatures that influence treatment response, particularly to anti-angiogenic therapies. Given the pivotal role of tumor vasculature in proliferation and metastasis, and the existing research bias towards Triple-Negative BC (TNBC), a comprehensive investigation into the complexity of the vascular system is warranted. Furthermore, as lymph node metastasis (LNMT) is a major determinant of cancer mortality, distinguishing the endothelial and immune cell subtypes within LNMT microenvironments from primary tumors is essential for clarifying pathogenesis. Methods To address this, we integrated scRNA-seq data from primary BC tumors, ER_LN and healthy tissues, generating a robust dataset of 98,000 cells (~ 12 samples). Employing single-cell transcriptomics, spatial transcriptomics, and immunohistochemistry, we precisely delineated the endothelial cell (EC) heterogeneity within matched tumor and peri-tumoral tissues and scrutinized their intricate interactions with immune populations. Results We identified two previously uncharacterized, tumor-enriched endothelial cell subtypes, designated EC4 and EC5, which demonstrate subtype-specific functional adaptations and prognostic significance. EC4 cells, highly prevalent across BC, are principally characterized by antigen presentation, immune cell recruitment, and pro-inflammatory signaling. Conversely, EC5 cells, also enriched in BC, exhibit robust extracellular matrix (ECM) remodeling and potent tumor angiogenesis. We further characterized the functional divergence of EC4 and EC5 within ER tumors relative to HER2 tumors and ER_LN metastases. Our analysis reveals conserved endothelial programming mechanisms across BC subtypes, coexisting with distinct TME-driven transcriptional adaptations. Importantly, interactome analysis highlighted novel and subtype-specific communications between these novel EC subsets and immune cells, particularly CD8+T cells and macrophages. Experimental validation demonstrates that ECs overexpressing APP can mediate the M2 polarization of macrophages, underscoring diverse immunomodulatory roles for EC subsets across different BC contexts. Conclusions These findings offer critical and granular insights into the complex interplay between novel EC subtypes and the immune microenvironment in BC progression and metastasis. We establish that ECs are active and heterogeneous modulators of the TME, identifying specific therapeutic vulnerabilities within the tumor vasculature and providing a foundational blueprint for developing future precision immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67936270be24af1f65608417190d3a4e352d6fe3" target='_blank'>
              Single-cell profiling identifies heterogeneity of the immune microenvironment in healthy, primary and lymph node metastatic BC
              </a>
            </td>
          <td>
            Chuyu Liu, Feng Zeng, Shanshan Gao, Nengying Zhang, Yulan Cai
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Cancer is not composed of a single cell type but rather a complex community of cancer cells, immune cells, and supporting stromal cells that communicate with each other. These cellular conversations shape how each cancer grows, spreads, and responds to treatment. In this study, we compared single-cell sequencing data from seven different human cancers to explore how these cell interactions differ among tumor types. We found that pancreatic cancer contains many neutrophils, a type of immune cell that interacts mainly with other immune cells, while liver cancer lacks the usual fibroblast support cells. In contrast, esophageal and breast cancers were rich in fibroblasts that send growth signals to tumor cells, and thyroid cancer retained genes that may slow tumor progression. These differences help explain why some cancers behave more aggressively than others. Our findings provide a clearer picture of how the tumor environment varies among cancers and may guide the development of new strategies to treat solid tumors by targeting their surrounding cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95ce9b5897ed438ba40418ac1eb1c9c140e7a59a" target='_blank'>
              Comparative Oncology: Cross-Sectional Single-Cell Transcriptomic Profiling of the Tumor Microenvironment Across Seven Human Cancers
              </a>
            </td>
          <td>
            Riku Okamoto, Kota Okuno, Akiko Watanabe, Kanako Naito, Hiroyuki Minoura, Shumpei Shibaki, Kyonosuke Ikemura, Keiko Oki, Yu Kuroda, Shiori Fujino, Yusuke Nie, Nobuyuki Nishizawa, Eiichiro Watanabe, Mariko Kikuchi, Koshi Kumagai, Takahiro Yamanashi, Hiroshi Katoh, Hajime Takayasu, Takeo Sato, Takafumi Sangai, Yusuke Kumamoto, T. Naitoh, Naoki Hiki, Keishi Yamashita
          </td>
          <td>2025-10-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have demonstrated prolonged efficacy in certain melanoma patients, yet a significant portion of patients do not experience clinical improvement, with the mechanisms underlying this resistance still not fully understood. Using established cell markers, we partitioned the single‐cell transcriptome into clusters, finding a notable link between NK cells and patient response to immunotherapy. We further identified four distinct subpopulations of NK cells, profiling marker gene sets and unique biological functions associated with each subpopulation. This analysis provides insights into the trajectory of NK cell development and differentiation, along with identifying the transcription factors driving these processes. The study pinpointed NK cluster 01 as pivotal in influencing patient sensitivity and prognosis during immunotherapy. Single‐cell transcriptome and spatial transcriptomics (ST) analysis revealed the proximity of NK cluster 01 cells to melanoma cells, hinting at a potential regulation of cell‐cell interaction via the IFN‐II signaling pathway network. ST analysis revealed the spatial arrangement and interaction of NK cluster 01 cells with melanoma cells. This study explores the feasibility of targeting NK cluster 01 cells with small molecule drugs via molecular docking, offering a promising approach to bolster the clinical utility of NK cell therapy. We comprehensively analyze the heterogeneity of NK cells within melanoma, elucidate the potential regulatory interactions between NK cells and other microenvironmental components, and establish a basis for the future clinical utilization of distinct NK cell subsets as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbdc1aaba32b7600dd3221ae61c5e31ee96bd3" target='_blank'>
              Integrating Single‐Cell and Spatial Transcriptomics Reveals NK Cell Subpopulations Associated With Immunotherapy for Melanoma
              </a>
            </td>
          <td>
            Zhicheng Hu, Yongfei Chen, Hao Yang, Qiuming Pan, Hongrui Li, Shuting Li, Junxi Wang, Yudi Huang, Guanglong Huang, Shanqiang Qu
          </td>
          <td>2025-12-01</td>
          <td>Smart Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related morbidity and mortality worldwide. Tumor sidedness, distinguishing left- (LCRC) and right-sided (RCRC) cancers, has emerged as a critical clinical determinant, influencing patient prognosis and therapeutic response. However, the cellular and molecular mechanisms underlying these differences remain poorly understood. Recent advances in single-cell RNA sequencing (sc-seq) provide high-resolution insights into CRC heterogeneity, revealing distinct tumor, immune, and stromal cell populations and their context-specific interactions. In this review, we synthesize sc-seq studies that dissect the molecular programs driving progression, therapy resistance, and metastasis in CRC. We highlight malignant subclusters characterized by metabolic reprogramming and spatially organized oncogenic signaling; specialized immune cell states, including macrophage subsets, exhausted T cells, and mast cells, that shape tumor immunity; and stromal elements such as cancer-associated fibroblasts and endothelial tip cells that remodel the extracellular matrix, promote angiogenesis, and foster immune evasion. Importantly, sc-seq demonstrates that LCRC and RCRC represent distinct multicellular ecosystems with differential immune recruitment and stromal signaling, underscoring the need for sidedness-informed therapeutic strategies. We propose that future interventions should target cell-cell communication networks and spatially defined tumor–microenvironment interactions to overcome heterogeneity and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b9e3bca233f1317bcd2e2e43fb69a5fedfb984" target='_blank'>
              Sidedness matters: single-cell perspectives on left- and right-sided colorectal cancer
              </a>
            </td>
          <td>
            M. Yanova, Diana Maltseva, Alexander Tonevitsky
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a05de91b68f189c5a7a8fad28c43d535bcdc834" target='_blank'>
              A spatial atlas of colorectal cancer reveals the influence of stromal niches on tumour differentiation
              </a>
            </td>
          <td>
            Andrew D. Pattison, Rebekah M. Engel, Wing Hei Chan, Spencer Greatorex, David Nickless, Liam Skinner, Camilla Cohen, Julie Hickey, Christine Georges, Anne L. Fletcher, P. McMurrick, Thierry Jardé, Lochlan Fennell, Helen E. Abud
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor vascular endothelial cells are essential constituents of the tumor microenvironment, responsible for delivering oxygen and nutrients that are vital for tumor growth and proliferation. As a hallmark of cancer progression, abnormal tumor vasculature contributes to tumor development through multiple mechanisms. Although anti-angiogenic therapies are widely used in the treatment of various cancers, the intrinsic heterogeneity of endothelial cells poses significant challenges regarding therapeutic efficacy. Therefore, further investigation into the heterogeneity of tumor endothelial cells is of paramount importance. The rapid advancement of single-cell sequencing technologies in recent years has facilitated the detailed characterization of heterogeneity among tumor endothelial cells at the single-cell level, thereby fostering a more precise understanding of the functional roles of individual cells within the tumor microenvironment. This technology has become an indispensable tool for investigating the heterogeneity of tumor endothelial cells, offering insights that could inform the refinement of future cancer treatments. In this review, we synthesize findings from the field of single-cell omics to elucidate the heterogeneous characteristics of tumor endothelial cells. We analyze recent advancements in single-cell technology used in the study of tumor cell heterogeneity in terms of both commonalities and distinctive features, covering aspects at the gene and cellular levels. In this review, we provide an overview of recent applications of single-cell sequencing technology in analyzing tumor endothelial cell heterogeneity, offering insights into the development of precise tumor therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a50144e46bbcc6cadd8fab9df5c73d57f4ff4f3" target='_blank'>
              Research Progress Regarding the Use of Single-Cell Sequencing Technology in Analyzing Tumor Endothelial Cell Pathophysiology
              </a>
            </td>
          <td>
            Shu Zhao, S. Liu, Wenxin Shao, Dong Liu
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Thyroid cancer, the most common endocrine malignancy, is characterized by a unique and complex tumor microenvironment (TME). To unravel the high tumor heterogeneity and molecular mechanisms driving cancer progression, we performed single-cell RNA sequencing (scRNA-seq) analysis, enabling a comprehensive exploration of cellular diversity and molecular dynamics at single-cell resolution. We employed Principal Component Analysis (PCA) and Uniform Manifold Approximation and Projection (UMAP) for dimensionality reduction and subsequent identification of cellular clusters. Differential gene expression analysis across subclusters was conducted using the FindAllMarkers function, while the DoHeatmap function was utilized to visualize the distribution of differentially expressed genes. The AUCell algorithm was applied to evaluate pathway enrichment within specific cell subtypes. To decipher cellular communication networks, we integrated the CellChat and NicheNet algorithms, which revealed intricate intercellular signaling interactions. Finally, multiplex immunohistochemistry (mIHC) was performed to validate key cellular interactions identified in silico. By analyzing 405,077 single cells from 50 thyroid cancer samples (including papillary, anaplastic, and metastatic tumors) and 14 normal thyroid tissues, we identified four major cellular subpopulations through unbiased clustering based on gene expression patterns and representative cellular markers. The TME was found to encompass diverse immune, endothelial, and mesenchymal cell subtypes, including novel populations such as CD4 + HSPA1A + T cells. Functional pathway enrichment analysis highlighted the roles of abundant cell types in tumor progression. Cell-cell communication analysis uncovered potential immunotherapeutic targets and revealed critical crosstalk among hub niche cells, including APOE+ macrophages, EMT-like cancer-associated fibroblasts (CAFs), and RBP7+ endothelial cells. These findings were further validated by multiplex immunohistochemistry, confirming the spatial organization and interactions of these cell populations within the TME. Our study provides a comprehensive single-cell transcriptomic atlas of thyroid cancer, offering profound insights into tumor heterogeneity, the functional roles of key niche cells, and potential biomarkers for anticancer therapy. These findings not only enhance our understanding of thyroid cancer biology but also pave the way for the development of novel therapeutic strategies targeting the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b009366b4ed17ce1b669fc0314c882a610b52822" target='_blank'>
              Comprehensive single-cell RNA analysis reveals intertumoral microenvironment heterogeneity and hub niche of carcinogenesis in thyroid cancer
              </a>
            </td>
          <td>
            Hao Xu, Junjie Ma, Nanjun Li, Haoxiang Xuan, Shijie Lin, Yunuo Shen, Lu Yin, Ping Wang, Qianhui Xu
          </td>
          <td>2025-11-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) arises from heterogeneous precursor lesions, including intraductal papillary mucinous neoplasms (IPMNs), but the features distinguishing indolent from progressive lesions remain unclear. We performed an integrative analysis of transcriptomic, genomic, and microenvironmental profiles of IPMNs to define multi-omic phenotypes. Using transfer learning, we projected IPMN-derived transcriptional programs onto spatial transcriptomic datasets from IPMNs and pancreatic intraepithelial neoplasias (PanINs). We identified two major phenotypes: one associated with cancer-associated fibroblasts and epithelial-to-mesenchymal transition, shared across IPMN, PanIN, and PDAC; and a second, glycolysis-enriched phenotype with a unique somatic mutation profile specific to IPMN. Spatial mapping further revealed grade-specific enrichment of transcriptional programs and distinct interactions with stromal and immune subtypes, underscoring the role of the precancer microenvironment in progression. These findings establish multi-omic phenotypes that unify genetic, transcriptional, and microenvironmental heterogeneity, providing a framework for distinguishing progressive from indolent precancers and a web-based public atlas for future exploration of these data and transcriptional phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9701f2eee2cdef87945b689296ec7c72133df8a9" target='_blank'>
              Integrative analysis of genomic and transcriptomic data informs precancer progression in the pancreas
              </a>
            </td>
          <td>
            Kathleen Noller, Jiaying Lai, Daniel Lesperance, Ricky S. Adkins, Ahmed M. Elhossiny, Paola A. Guerrero, K. Rajapakshe, Anirban Maitra, M. Giglio, A. Mahurkar, Owen R. White, Marina Pasca di Magliano, Michael F. Ochs, Luciane T. Kagohara, Laura Wood, Rachel Karchin, E. Fertig
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Recent single-cell studies implicate the glutamatergic lineage of the rhombic lip (RL) as the origin of group (G) 3/4 medulloblastomas (MBs) and a stalled differentiation program as underlying their malignant transformation. It is unknown, however, if G3/4 MBs exhibit cellular plasticity as has been observed in glioma. The effect of therapy on cellular hierarchies in G3/4 MBs is unknown. We profiled N=57 G3/4 MB frozen specimens obtained from the University of California, San Francisco Brain Tumor Center and the Children’s Brain Tumor Network biobanks via single-nucleus RNA and epigenetic sequencing. This cohort included 43 diagnostic and 14 patient-matched recurrent specimens. We profiled N=32 FFPE specimens via Xenium single-cell spatial transcriptomics. We performed genetic lineage tracing in intracranial implants of G3 MB cell lines derived from primary and metastatic disease using engineered barcodes, and in clinical specimens using endogenous mutations. Remarkably, xenografts from primary tumors produced similar cell types, in similar proportions to those found in treatment-naive human MBs. Moreover, the distribution of cell types found in xenografts was conserved at the level of individual clones, indicating cell-intrinsic regulation of tumor composition. Single-cell phylogenetic analysis demonstrated significant rates of dedifferentiation events from unipolar brush-like cells to RL-progenitor phenotypes. This plasticity was increased in recurrent specimens and in metastatic models compared to models and samples derived from primary disease. Samples with higher plasticity exhibited greater spatial heterogeneity, while samples with lower rates of dedifferentiation had more structured spatial organization. These studies show that G3/4 MB cellular composition is homeostatic in treatment-naive disease and conserved at the level of individual clones. Yet, these tumors exhibit plasticity that is exacerbated by therapy and affects spatial organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7150b972b545d28e4de15c0584606a43a9a4dc" target='_blank'>
              STEM-18. GENETIC LINEAGE TRACING OF MEDULLOBLASTOMA UNDER THERAPY WITH SPATIAL CONTEXT
              </a>
            </td>
          <td>
            Bohyeon Yu, Jangham Jung, Abhinav Jain, Emanuele Rosatti, Joanna Phillips, Aaron A. Diaz
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Spatial proteomics can capture cell organization, providing deep insights into diseases and catalyze significant advancements in clinical diagnosis, treatment, and outcomes. The tumor microenvironment (TME) is an organized signaling network consisting of extracellular matrix (ECM), tumor cells, stromal cells, and immune cells. Tissue histopathology, particularly immunohistochemistry (IHC), remains the most frequently used modality to assess the TME. However, despite advances in histopathology, the complexity of the TME often leads to the loss of critical information about tumor function and clinical outcomes. New technologies for high-dimensional protein mapping within intact tissues, such as multiplexed ion beam imaging by time of flight (MIBI-TOF), are increasingly employed to reveal the functional encoding of the TME. MIBI uses secondary ion mass spectrometry to image metal-tagged antibodies in tissue samples. This allows for the simultaneous quantification of dozens of antibodies, even those present in low abundance, at a subcellular level within a single scan, offering precise insights into complex biological processes. We have utilized spatial omics tools to investigate immune tolerance in diverse contexts such as human neuropathology, peanut allergy immunotherapy, pulmonary arterial hypertension, and cancer. Here, we demonstrate the application of MIBI in large-scale cancer clinical trials to analyze panels of ∼30 biomarkers.



 2U24CA224309



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9b866f97f205cb2e4a97a39048acf0269a7554b" target='_blank'>
              Application of Spatial Proteomics for Comprehensive Immune Monitoring in Cancer Clinical Trials 9251
              </a>
            </td>
          <td>
            S. Kowli, Alex Kong, C. Fullaway, Mina Pichavant, Sean C. Bendall, Michael R. Angelo, H. T. Maecker
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Deciphering the spatial biology of the tumor microenvironment (TME) can inform researchers about the likelihood of therapeutic success. Of particular interest is the spatial organization of immune cells, especially T cells, which can undergo exhaustion resulting in reduced effector functions due to expression of inhibitory markers such as PD-1. Imaging Mass Cytometry™ (IMC™) technology is a quantitative multiplexed spatial imaging technique for assessment of 40-plus biomarkers simultaneously on the same tissue slide without signal amplification or autofluorescence interference. We used IMC technology to investigate the spatial distribution and phenotypic characteristics of immune cells, focusing on exhausted T cells in the TME of multiple cancers. Various IMC imaging modes revealed striking heterogeneity of the TME, including clusters of lymphocytes at the tumor margins, indicative of immune surveillance. Pixel-clustering and single-cell analyses delineated distinct tumor areas based on extent of T cell exhaustion, immune cell density, cell proliferation and stromal components. These findings illustrate the power of IMC technology to elucidate the complex spatial landscape of the TME, which may aid in developing prognostic assessments and guide personalized therapeutic strategies for cancer treatments. For Research Use Only. Not for use in diagnostic procedures.



 Tumor Immunology: Checkpoints, Prevention, and Treatment (TIPT)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b22f9c7f1b4911c7dde0700548e2e19989a2ee" target='_blank'>
              High-plex spatial analysis of T cell exhaustion using Imaging Mass Cytomerty for immuno-oncology studies 2622
              </a>
            </td>
          <td>
            Qanber Raza, T. Pfister, Yun-Jyh Chwee, Nikesh Parsotam, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are key regulators of the tumor microenvironment, yet their spatial organization and immunomodulatory functions in H. pylori-associated gastric cancer (GC) remain incompletely understood. We profiled formalin-fixed paraffin-embedded (FFPE) tumors from 71 GC patients using spatial transcriptomics and integrated single-cell RNA-seq from three independent cohorts (China, USA, and Singapore). CAF-immune cell colocalization was quantified by neighborhood enrichment and aggregation index score. Ligand-receptor inference and trajectory analysis resolved CAF signaling and state transitions. To delineate post-transcriptional control, ARE-motif scanning and expression correlations were combined with laser-assisted crosslinking and immunoprecipitation sequencing (LACE-seq) to nominate ZFP36 targets. Immune contexture and prognostic associations were evaluated using CIBERSORT-ABS and Kaplan-Meier analyses in The Cancer Genome Atlas (TCGA) and Asian Cancer Research Group (ACRG) cohorts. We first defined the spatial distributions of the four CAF subtypes reported in prior studies and found that their immune associations varied across histologic and infection-defined GC subtypes. In H. pylori-positive tumors, THBS1⁺ CAFs were spatially enriched near regulatory T cells (Tregs) and were associated with local immunosuppression through WNT5-FZD interactions. In parallel, ZFP36 bound AU-rich elements within the FN1 3′ untranslated region (3'UTR), destabilizing FN1 mRNA and thereby diminishing FN1⁺ CAF-mediated cytotoxic T lymphocyte (CTL) activation. Together, these axes promoted Treg accumulation and suppression of CTL activation. These findings reveal infection-associated stromal programs that shape the immune landscape in GC and highlight CAF-directed pathways as potential therapeutic targets in H. pylori-associated GC. H. pylori-induced CAF-mediated immunosuppression in GC. 1. H. pylori infection triggers transcriptional reprogramming of CAFs within the tumor microenvironment of GC. 2. The CAF subset marked by THBS1 is spatially enriched adjacent to Tregs and facilitates their recruitment via WNT5-FZD ligand-receptor signaling. 3. The RNA-binding protein ZFP36 binds AU-rich elements in the FN1 3’UTR, destabilizing FN1 mRNA and thereby attenuating CTL activation. 4. The THBS1-WNT5 axis stabilizes Tregs, whereas the ZFP36-FN1 axis limits CTL activation, converging to drive local immunosuppression in H. pylori-positive GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f1c559549d2edda98547851b085fedd496597a" target='_blank'>
              Spatial and functional dissection of cancer-associated fibroblasts-mediated immune modulation in H. pylori-associated gastric cancer
              </a>
            </td>
          <td>
            Bonan Chen, Hongzhen Tang, Xiaohong Zheng, Fuda Xie, Peiyao Yu, Yang Lyu, Tiejun Feng, Jialin Wu, Jingya Liu, Yi Xu, A. H. Cheung, Canbin Fang, Zhangding Wang, Shouyu Wang, Justin Chak Ting Cheung, Yujuan Dong, Ruoxi Tian, Yigan Zhang, Cheng Lu, C. Wong, Jun Yu, William K. K. Wu, Elke Burgermeister, M. Tong, Fengbin Zhang, Wei Kang, Kam Tong Leung, K. To
          </td>
          <td>2025-11-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Metastases represent one of the hardest obstacles in cancer treatment, accounting for many cancer-related deaths. Understanding the mechanisms that drive disease progression is essential to improve patient outcomes and develop more effective therapeutic strategies. This phenomenon appears to be elicited by dormant cancer cells (DCCs), which can persist undetected for extended periods of time, entering a non-proliferative, hibernation-like state that confers resistance to conventional therapies and facilitates immune evasion. Owing to their status as a rare and energy-restricted population, combined with the limitations of current medical imaging, DCCs often evade early detection, hindering timely intervention and effective clinical management. Consequently, a critical need exists to develop high-resolution detection systems and identify specific DCC targetable biomarkers. Here, a comprehensive overview of the current understanding of DCCs is reported, with a focus on recent advancements in experimental strategies for their identification and tracking, as well as therapeutic approaches currently under clinical investigation aimed at targeting these elusive cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9d13b10d52e65e6fb3d5e89c185ee227dee670" target='_blank'>
              Decoding cancer dormancy: integrative genomic, phenotypic and live-cell imaging analysis to reveal the hidden cancer cell reservoir
              </a>
            </td>
          <td>
            G. Porcelli, Caterina D’Accardo, Francesca Angeloro, Martina Cucchiara, Paola Bianca, Vincenzo Davide Pantina, Narges Roozafzay, Chiara Modica, Miriam Gaggianesi, Sebastiano Di Bella, G. Stassi, Alice Turdo, M. Todaro
          </td>
          <td>2025-11-12</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ffa550baca76e714d9b657ce67086b4b77a9db" target='_blank'>
              Deciphering Tumor Microenvironment Dynamics in Tumorigenesis and Lymph Node Metastasis of Esophageal Squamous Cell Carcinoma using Single-cell RNA Sequencing
              </a>
            </td>
          <td>
            Hansoll Na, T. Hong, Chung Lee, Y. Yang, Ha Eun Kim, Byung Jo Park, Hyun Ki Kim, Dae Joon Kim
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cell state plasticity drives metastasis and therapy resistance in cancers. In melanoma, these behaviors map onto a melanocytic-to-mesenchymal-like continuum regulated by AP-1 transcription factors. However, how the AP-1 network encodes a limited set of discrete states, why their distributions vary across tumors, and what drives phenotypically consequential AP-1 state transitions remain unclear. We develop a mechanistic ODE model of the AP-1 network capturing their dimerization-controlled, co-regulated, competitive interactions. Calibrated to heterogeneous single-cell data across genetically diverse melanoma populations and combined with statistical learning, the model reveals network features explaining population-specific AP-1 state distributions. These features correlate with MAPK activity across tumor lines and with variability within clones, linking MAPK signaling to AP-1 states. The model predicts and experiments validate adaptive AP-1 reconfiguration under MAPK inhibition, inducing a dedifferentiated, therapy-resistant state that can be blocked by model-guided AP-1 perturbations. These results establish AP-1 as a configurable network and provide a computational framework for predicting and modulating AP-1 driven cell state plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80053bfbfe501e1acc3e16cb2b130e2f5342e9b1" target='_blank'>
              Cell state plasticity emerging from co-regulated, competitive, and configurable interactions within the AP-1 network
              </a>
            </td>
          <td>
            Yonatan N. Degefu, Magda Bujnowska, Douglas G. Baumann, Mohammad Fallahi-Sichani
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The spontaneous regression of cancer lesions demonstrates the potential of immune surveillance; yet, these transient events have remained inaccessible to systematic study. Using longitudinal intravital microscopy in a carcinogen-induced model of head and neck cancer, we tracked premalignant lesions within the same animals for 24 weeks at single-cell resolution. This approach revealed three trajectories: progression, stability, or regression, and enabled dissection of the immune dynamics underlying each fate. Lesion outcome was dictated by the spatial organization of myeloid-derived antigen-presenting cells: regressing lesions were characterized by dense clusters of myeloid-derived cells associated with CXCL9+/CXCL10+ expression and T cell recruitment, whereas progressing lesions displayed sparse, non-clustered infiltration. Remarkably, transient myeloid clusters arose prior to any detectable lesion formation and consistently marked regions that would later develop into premalignant lesions. These findings identify spatiotemporal myeloid organization as an early determinant of tumor fate and provide a mechanistic framework for predicting and intercepting cancer at its inception. Summary Sentence Early myeloid architecture dictates cancer fate: dense CXCL9⁺/CXCL10⁺ clusters with T-cell enrichment accompany regression, whereas sparse infiltration predicts progression. Transient pre-lesional myeloid clusters emerge at future tumor sites, revealing immune organization as an early determinant of malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59ee0ed38702dcef1f68ff3c8ff2d5ef5e4fc717" target='_blank'>
              Longitudinal Imaging of the Premalignant Tumor Microenvironment Reveals Transient Myeloid States Predictive of Tumor Fate
              </a>
            </td>
          <td>
            Thomas D. Madsen, Desu Chen, Maria Hernandez, Sarah M. Hammoudeh, Marco Heydecker, Emily Chen, Salma Abu-Elnaj, Noemi Kedei, Weiye Wang, Roberto Weigert
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is the 4th leading cause of cancer death in the United States as of 2024. Its tumor microenvironment (TME) contains heterogenous inflammatory and immunosuppressive areas due to varying immunophenotypes relating to immune cell infiltration and exhaustion. CD4+ T helper cells can contribute to these dynamics, but defined roles of some subsets in PDAC remain debatable. We hypothesize that effector CD4+ T cell subsets spatially influence levels of inflammation in the PDAC TME to promote tumor metastasis. To investigate these dynamics, we designed a novel sequential multiplex immunofluorescent panel for murine FFPE slides consisting of 50 markers to spatially capture immune cells with activation/exhaustion states, epithelial-to-mesenchymal phenotypes, vessel infiltration, and stromal deposition in the TME. Using FFPE slides also improves preservation of structural morphology and translation to human tumor samples. Here, we applied the panel to primary murine PDAC tumors and liver metastases compared to normal tissues, revealing heterogenous MHC-I presentation from primary tumor cells, variations in CD4+ subsets in the primary tumor, and increased lymphocyte infiltration in liver metastases, validating our panel with recent spatial and dissociative multimodal publications. Ultimately, we will apply the panel to mice deficient in specific CD4+ subsets to better elucidate the spatial role of effector T helper cells in PDAC metastases.



 Supported by UAB Hematology & Oncology Startup Fund; Hirshberg Foundation for Pancreatic Cancer Reasearch



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d46f7f6b06d7c343cfa41f8977a3f99ebd086c73" target='_blank'>
              High-dimensional spatial analysis of murine tumor microenvironment highlights the heterogeneity of primary and metastatic pancreatic cancer immune landscape 3822
              </a>
            </td>
          <td>
            Christina R Larson, Ayushi Mandloi, Jace Baines, Richard Laube, Alana M. Franceski, R. Welner, Julienne Carstens
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c38b6d1e04676ebe5c874a1af9317e2b5d3d46" target='_blank'>
              Single-cell Transcriptomic Profiling of Pancreatic Ductal Adenocarcinoma: Epithelial Reprogramming and Systemic Immune Exhaustion
              </a>
            </td>
          <td>
            Qinyu Xu
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Despite the use of immunotherapy in esophageal squamous cell carcinoma (ESCC), treatment failure occurs occasionally in patients, yet the underlying mechanisms remain poorly understood. Methods: We conducted large-scale single-cell RNA sequencing (scRNA-seq) data analysis, which integrated seven independent datasets from 192 ESCC patients to yield over 440,000 high-quality single cells, to systematically characterize the tumor microenvironment (TME) landscape during ESCC progression and immunotherapy response. Additionally, we performed high-resolution spatial transcriptomics (stRNA-seq) using the 10x Visium HD platform on paired pre- and post-treatment tissues from two patients (one immunotherapy responder and one non-responder), which enhanced the findings from the scRNA-seq data and mapped therapy-induced TME at the spatial level. Multiplex immunohistochemistry was employed based on seven patients to confirm distinct patterns of intercellular crosstalk underlying differential therapeutic outcomes. Results: In scRNA-seq data, we found that B lineage cells were reduced during ESCC progression but were enriched in immunotherapy-resistant patients. Further analysis of malignant ESCC cells suggested that immunotherapy resistance might be associated with a subpopulation of tumor cells exhibiting aberrantly elevated cholesterol biosynthesis. Cell communication analysis of scRNA-seq and stRNA-seq data collectively revealed that immunotherapy resistance was linked to cellular crosstalk between cholesterol-biosynthetic tumor cells and germinal center (GC) B cells within tertiary lymphoid structures. Notably, single-cell, spatial data, and multiplex immunohistochemistry demonstrated that cholesterol biosynthesis-associated ESCC cells express elevated levels of MIF. This disrupts GC reactions by competing with the CXCL12-CXCR4 signaling axis via MIF-CXCR4 interactions, thereby impairing B cell-mediated immunity. Conclusions: MIF+ tumor cells in GCs may be a biomarker for predicting immunotherapy resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8a1ee6ff72d48f3e16c31d484505ab587cac16" target='_blank'>
              MIF-expressing tumor cells mediate immunotherapeutic resistance in esophageal squamous cell carcinoma.
              </a>
            </td>
          <td>
            Jing Song, Xiaomei Song, Yue Xie, Hong Guo, Yupeng Cun
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Somatic evolution drives cancer progression and therapy resistance. Underlying phenotypic progressions is the development of marked clonal heterogeneity in most cancers. However, deciphering the downstream molecular effects of clonal heterogeneity in primary cancer has been limited due to the inability to isolate subclones by standard methods. While single-cell RNA-sequencing (scRNA-seq) provides high-resolution mapping of heterogeneous cell states, it cannot link transcriptional programs to cancer-driving mutations in these individual cells. We and others developed methods to detect genotypes and their transcriptional outputs in single cells (e.g., Genotyping of Transcriptomes; Nature, 2019). However, these approaches have technical limitations to broadly profile numerous mutations, which is nonetheless required for clonally complex neoplasms. The existing approaches are also restricted to fresh or frozen samples, limiting the use of the widely available pathology formalin fixed paraffin embedded (FFPE) tissues.
 To address these challenges, we developed Genotyping of Transcriptomes for Multiple Targets and Sample Types, GoT-Multi, a next generation single-cell multi-omics method compatible with FFPE tissues that integrates multiplexed genotyping with transcriptomic profiling. GoT-Multi adapts a probe-based scRNA-seq method (Flex Fixed RNA Profiling, 10x Genomics) by including custom mutation-specific probes along with the standard transcriptional probes. Furthermore, we developed an ensemble-based machine learning pipeline, GoT-Multi-ML, that optimizes the genotyping calls by denoising technical artifacts. GoT-Multi-ML leverages multiple machine learning models to learn the non-linear patterns of the genotyping data features (e.g., reads per genotyping transcript, GC content of probes), agnostic to the whole transcriptomic data. We validated GoT-Multi via a cell line-mixing experiment using SK-BR-3 and MCF-7 cell lines. GoT-Multi detected the targeted mutations (HIST1H1C A24T and S100A10 A77V) in the expected cell line (MCF-7) with an average genotyping rate of 72% and accuracy of 97%.To define the impact of clonal evolution on phenotypic progression, we focused on a prototype of cancer evolution, the progression of chronic lymphocytic leukemia (CLL) to therapy-resistant large B-cell lymphoma (LBCL) called Richter Transformation (RT). We applied GoT-Multi to a unique cohort of primary nodal cryopreserved samples containing both CLL and LBCL components (n = 5). We targeted 18 mutations (1-6 targets per sample) with a 98% median accuracy. We observed heterogeneous cancer cell states, including proliferating, stress response and inflammatory (n = 51,465 cells). LBCL was enriched in proliferating and cyclin D2-elevated states, whereas CLL was associated with a stress response signature. GoT-Multi enabled us to reconstruct the clonal architectures, up to 4 distinct subclones per lymphoma, and their associated cell states. Differential gene expression between mutated and wildtype cells revealed that genetically heterogenous subclones, including those with SRRM2 and JUNB mutations and therapy resistance-associated mutations in PLCG2 and BTK, displayed enrichment in inflammatory cell states, upregulating TNF and interferon signaling genes while suppressing cell cycle-related genes. Conversely, other subclones with mutations in, e.g., IRF8, POU2F2, PRKDC were associated with MYC activation and/or enhanced proliferation. These findings suggested that heterogeneous genotypes may converge on similar downstream transcriptional cell states. In addition, TNF itself was overexpressed in the proliferative subclones, suggesting that TNF production from these subclones may enhance inflammatory signaling of the TNF-responsive subclones, indicative of crosstalk between subclones. Finally, we demonstrated the applicability of GoT-Multi to pathology archived FFPE tissues. We profiled 9 mutations with an accuracy of 98% in an FFPE RT sample (n = 2,140 nuclei), allowing us to detect 5 subclones and their associated cell states. In summary, co-mapping of clonal and cell state heterogeneity at single cell resolution through GoT-Multi suggested that heterogeneous subclonal genotypes may converge on similar downstream oncogenic pathways to enhance overall tumor fitness. We envision that the broad applicability of GoT-Multi may help uncover the molecular underpinnings of cancer progression and therapy-resistance across oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/954b694c778f5e54689dba8e5834b825525f28c2" target='_blank'>
              Co-mapping clonal and transcriptional heterogeneity in somatic evolution via got-multi.
              </a>
            </td>
          <td>
            Minwoo Pak, M. Saurty-Seerunghen, Kellie Wise, Tsega-Ab Abera, Chhiring Lama, Neelang Parghi, Xiaotian Sun, Qi Gao, Liming Bao, Mikhail Roshal, John N Allan, Richard Furman, Luciano G. Martelotto, Anna S. Nam
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most common histological subtype of renal cancer and remains a clinical challenge due to its frequent resistance to therapy and poor prognosis in advanced stages. Apoptosis, a fundamental tumor-suppressive mechanism, exhibits paradoxical roles in cancer, wherein apoptotic tumor cells can also contribute to immunosuppression and tumor progression. However, the spatial dynamics, transcriptional heterogeneity, and prognostic relevance of apoptosis-related gene programs in ccRCC remain poorly defined. We performed an integrative analysis combining single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and summary-based Mendelian randomization (SMR) to dissect apoptosis-related malignant cell states in ccRCC. Cancer cells were stratified based on apoptosis gene signatures and CASP9 expression. Cell–cell communication was assessed using CellChat and spatial interaction networks were constructed using RCTD and mistyR. SMR was employed to link genetically regulated CASP9 expression with renal cancer risk. A CASP9-associated prognostic model was developed using LASSO Cox regression and DeepSurv on TCGA and E-MTAB-1980 cohorts. We identified transcriptionally and spatially distinct apoptosis-high and apoptosis-low malignant cell subpopulations. Apoptosis-high tumor cells, characterized by elevated CASP9 expression, preferentially localized near macrophage-enriched stromal regions and exhibited stronger spatial clustering. Ligand–receptor modeling revealed directional signaling via the SPP1–CD44 axis between CASP9-high cancer cells and macrophages. SMR analysis provided genetic evidence supporting CASP9 as a causal gene for renal cancer. CASP9-high cells demonstrated distinct developmental trajectories and formed multicellular spatial modules with macrophages and cycling cells. A five-gene apoptosis-related signature derived from CASP9-stratified tumor cells robustly predicted patient survival across both training and validation cohorts. Low-risk patients exhibited enriched immune infiltration, increased immune checkpoint expression, and enhanced immune pathway activity. Our study reveals that apoptosis, particularly CASP9-driven programs, defines a spatially organized, immunosuppressive malignant cell state in ccRCC. CASP9 acts as both a genetic driver and spatial regulator of tumor–macrophage interactions, contributing to disease progression. The CASP9-associated risk model demonstrates strong prognostic utility and highlights apoptosis as a promising therapeutic axis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22570e710b314543b7842267551dd125b21247d1" target='_blank'>
              Integrative SMR and single cell & spatial analysis reveals the spatial heterogeneity and prognostic value of CASP9-mediated apoptotic pathways in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Zijie Yu, Xinghan Yan, Wenchuan Shao, Lingchen Cai, Da Zhong, Xiyi Wei, Ninghong Song
          </td>
          <td>2025-11-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline G. Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, K. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran C. Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, K. L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is one of the most aggressive brain tumors because its cells spread widely through the brain, making complete removal impossible. We implanted fluorescently labeled glioma stem-like cells (GSCs) into the mouse brain and observed their infiltrative growth using advanced 3D imaging. Within 6–10 weeks, tumors caused about 15% brain swelling and showed a strong preference for invading the corpus callosum, a major white matter tract. By measuring Sox2-positive cells, we found very high densities in the tumor bulk (over 4000 cells/mm2) but much lower levels at the invasive margin (about 30 cells/mm2) and a clear gradient in the corpus callosum (3010 to 1.5 cells/mm2 across 1 mm). Using CLARITY tissue clearing combined with light-sheet microscopy, we visualized how tumor cells infiltrate along blood vessels and brain structures, adopting distinct shapes and strategies. Single-cell gene analysis of the invading corpus callosum cells identified a 7-gene “invasion signature” (NES, CCND1, GUSB, NOTCH1, E2F1, EGFR, TGFB1), which was linked to worse survival and may represent new therapeutic targets. Abstract Background/Objectives: High-grade gliomas (HGGs), including glioblastomas, are among the most aggressive brain tumors due to their high intratumoral heterogeneity and extensive infiltration. Glioma stem-like cells (GSCs) frequently invade along white matter tracts such as the corpus callosum, but the molecular programs driving this region-specific invasion remain poorly defined. The aim of this study was to identify transcriptional signatures associated with GSC infiltration into the corpus callosum. Methods: We established an orthotopic xenograft model by implanting fluorescently labeled human GSCs into nude mouse brains. Tumor growth and invasion patterns were assessed using tissue clearing, light-sheet fluorescence microscopy, and histological analyses. To characterize region-specific molecular profiles, we performed microfluidic-based single-cell RNA expression analysis of 48 invasion- and stemness-related genes in cells isolated from the tumor bulk (TB) and corpus callosum (CC). Results: By six weeks post-implantation, GSCs displayed marked tropism for the corpus callosum, with distinct infiltration patterns captured by three-dimensional imaging. Single-cell gene expression profiling revealed significant differences in 7 of the 48 genes (14.6%) between TB- and CC-derived GSCs. These genes—NES, CCND1, GUSB, NOTCH1, E2F1, EGFR, and TGFB1—collectively defined a “corpus callosum invasion signature” (CC-Iv). CC-derived cells showed a unimodal, high-expression profile of CC-Iv genes, whereas TB cells exhibited bimodal distributions, suggesting heterogeneous transcriptional states. Importantly, higher CC-Iv expression correlated with worse survival in patients with low-grade gliomas. Conclusions: This multimodal approach identified a corpus callosum-specific invasion signature in glioma stem-like cells, revealing how local microenvironmental cues shape transcriptional reprogramming during infiltration. These findings provide new insights into the spatial heterogeneity of gliomas and highlight potential molecular targets for therapies designed to limit tumor spread through white matter tracts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561590a9b7ca738409c5933b00e64b421edbb629" target='_blank'>
              Innovative In Vivo Imaging and Single Cell Expression from Tumor Bulk and Corpus Callosum Reveal Glioma Stem Cells with Unique Regulatory Programs
              </a>
            </td>
          <td>
            Natália dos Santos, Aline Aquino, Friedrich Preusser, Fabio Rojas Rusak, E. H. Jandrey, Miyuki Uno, T. K. Furuya, C. Lancellotti, M. Maldaun, Roger Chammas, Stephan Preibisch, A. Camargo, C. Masotti, Erico Tosoni Costa
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma.
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) invasion is shaped by dynamic clonal evolution within the spatial and cellular architecture of the brain. To investigate this process, we integrated multiparametric MRI, whole-exome sequencing (WES), bulk RNA-seq and single-nucleus RNA-seq (snRNA-seq) from 69 IDH-wildtype GBM patients, using spatially resolved biopsies from contrast-enhancing (CE) core tumor and non-enhancing (NE) invasive regions. Phylogenetic analysis revealed distinct clonal architectures. CE-enriched clones carried truncal mutations in conanical drivers of cell cycle and RTK signaling (TP53, PTEN, EGFR, PDGFRA/B), and private alterations in DNA repair genes (CDH11, PAX7, RAD51B). In contrast, NE-enriched clones exhibited private mutations in MAPK and Hippo pathways components (STAG2, BRAF, MAP2K3, CTNNA1), suggesting region-specific evolutionary trajectories. Furthermore, snRNAseq cell-cell communication analysis uncovered enriched ligand-receptor signaling in NE regions between malignant cells and brain-resident populations, including PTN—PTPRZ1 (myeloid), NRG-ERBB4 and SLIT-ROBO (neuronal, astrocytic, and oligodendrocytic). Bulk RNAseq deconvolution confirmed increased representation of neurons and glial cells in biopsies harboring NE-specific clones. These findings define a unique population of GBM clones localized at the infiltrative margin, marked by private MAPK/Hippo mutations and enhanced intracellular communication with the surrounding brain microenvironment. The recurrent engagement of neural-glial signaling pathways suggests that these NE-associated clones co-opt normal cellular networks to support invasion, immune evasion, and persistence following surgical resection. This study establishes a mechanistic link between spatial clonal evolution and tumor-host communication at the GBM infiltrative region and provide a framework for targeting cell-cell signaling circuits that sustain invasive tumor ecosystems and escape therapeutic treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee4b73c1f939dc48d12b3b70a8dbe5ab434d2210" target='_blank'>
              CSIG-31. REDEFINING THE GLIOBLASTOMA RESECTION MARGIN THROUGH SPATIAL CLONAL AND CELLULAR ECOSYSTEM ANALYSIS
              </a>
            </td>
          <td>
            Angad S. Beniwal, Junlin Wang, Xiaoling Lu, C. Sereduk, S. Migliozzi, L. Garofano, C. Plaisier, Leland Hu, R. Barajas, Junwen Wang, N. Tran
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Understanding the evolutionary dynamics of clonal populations is essential for uncovering the principles of development, disease progression, and therapeutic resistance. Recent advances in single-cell lineage tracing and transcriptomics enable such analyses by combining heritable barcodes with cell-state information. Here, we present SCOUT (single-cell Ornstein–Uhlenbeck trees), a framework that models gene expression dynamics along single-cell lineage trees using Ornstein–Uhlenbeck processes to distinguish neutral drift from selective pressure. Using simulations, we demonstrate that SCOUT accurately classifies genes based on their underlying evolutionary models. We further validate SCOUT in Caenorhabditis elegans development, identifying biological processes under selection across distinct developmental contexts. Finally, we apply SCOUT to a lung adenocarcinoma xenograft model, revealing key regulators of metastatic progression and tumor microenvironmental adaptation. By integrating lineage and transcriptomic data, SCOUT provides a powerful evolutionary lens for dissecting the forces that shape cell fate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1a07c3c29336ff660b3f4e1f28115198c90579" target='_blank'>
              SCOUT: Ornstein–Uhlenbeck modelling of gene expression evolution on single-cell lineage trees
              </a>
            </td>
          <td>
            Hannah Stuart, Aaron McKenna
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Resistance to immune checkpoint blockade (ICB) remains a major challenge in lung adenocarcinoma (LUAD), with stromal mechanisms underlying CD8⁺ T cell exhaustion still poorly understood. By integrating single-cell, bulk, and spatial transcriptomic datasets using EcoTyper, we identified a distinct immunosuppressive ecotype, EC10, enriched for TGF-β signaling and epithelial–mesenchymal transition. EC10 exhibited spatial co-localization of fibroblasts, malignant epithelial cells, and exhausted CD8+ T cells, and was consistently associated with immune exclusion, poor progression-free survival, and elevated TIDE scores across four ICB-treated LUAD cohorts. Cell–cell communication analyses revealed a dominant TGFB1–SERPINE1 signaling axis originating from fibroblasts, linking stromal remodeling to T cell dysfunction. In contrast, EC12 represented an inflamed, ICB-responsive state enriched in interferon signaling. These findings define EC10 as a spatially organized, fibroblast-driven immunosuppressive ecosystem predictive of ICB resistance, and highlight the therapeutic potential of targeting the TGF-β axis in LUAD. This research identifies a fibroblast-driven immunosuppressive ecosystem in lung adenocarcinoma, linking TGF-β signaling to immune resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130525b051ffbdab87e5edbec0e75b95564de9c8" target='_blank'>
              Fibroblast TGF-β signaling defines spatial tumor ecosystems linked to immune checkpoint blockade resistance
              </a>
            </td>
          <td>
            Seo-Young Lee, Youngjoo Lee
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Background Angioimmunoblastic T cell lymphoma (AITL) is a subtype of peripheral T-cell lymphoma with a poor prognosis. Pathologically, AITL is characterized by the proliferation of diverse stromal cells (SCs), including blood endothelial cells (BECs) and follicular dendritic cells (FDCs). However, the tumor histology exhibits considerable heterogeneity, both between and within patients. It is hypothesized that these SCs form a supportive tumor microenvironment, and their histological heterogeneity may influence the clinical course. We previously reported that single-cell RNA sequencing (scRNA-seq) of SCs in lymph nodes (LNs) identified 30 distinct SC subclusters (Nat Cell Biol, 2022). However, the specific SC subclusters involved in AITL tumorigenesis remain undefined.
 Aims We aimed to characterize the SC heterogeneity in AITL microenvironments by employing scRNA-seq and spatial multi-omics at single-cell resolution.
 Methods Non-hematopoietic cells were isolated from 10 AITL and 5 normal LNs using magnetic- and fluorescence-activated cell sorting, followed by scRNA-seq. Based on the scRNA-seq data of SCs and immune cells, we designed a 289-gene panel for spatial transcriptomics (ST) using Xenium in situ, and a 50-protein panel for spatial proteomics (SP) using PhenoCycler fusion targeting immune cell and SC subclusters. These assays were applied to 27 formalin-fixed paraffin-embedded AITL samples. Whole exome sequencing (WES) was successfully performed on 26 of the 27 samples. We analyzed a bulk RNA-seq dataset of 97 AITL to validate the prognostic potential of gene expression patterns.
 Results First, we analyzed scRNA-seq data from 49,363 stroma cells and identified 30 SC subclusters consistent with our prior findings. Notably, transitional BECs between capillary BECs and activated high endothelial venules (C-aHEVs) expressed markers of both capillary BECs and HEVs, and were further subdivided into C-aHEV1 and C-aHEV2. Both tip cells, which contribute to angiogenesis, and C-aHEV2 were increased in AITL samples. Differentially expressed genes analysis revealed that both C-aHEV1 and C-aHEV2 highly expressed ACKR1. Notably, C-aHEV1 exhibited high expression of chemokines such as CXCL10 and CXCL12, whereas C-aHEV2 expressed genes encoding extracellular matrix proteins. Interaction analysis revealed that the CCL5-ACKR1 axis, which is known as a regulator of leukocyte migration, was activated between CD8-positive T cells and C-aHEVs. Furthermore, the COL15A1-integrin and SPARC-ENG axis, both of which have been reported to promote angiogenesis, were specifically enriched between C-aHEV2 and other HEVs.
 ST analysis of 2,020,192 cells identified 10 BEC subclusters, 5 non-endothelial SC subclusters, and 12 immune/tumor cell subclusters. Spatial proximity measurement revealed that C-aHEV1 was the BEC subcluster most closely associated with tumor cells. Spatial niche analysis revealed that C-aHEV1, together with FDCs, formed a distinct tumor-associated niche (CFT niche). On the other hand, C-aHEV2 was localized farther from the tumor than C-aHEV1, and formed a niche in conjunction with other HEV subclusters (HEV niche). The CFT niche-dense area and the HEV niche-dense area were often in close proximity, and the densities of CXCR4, encoding areceptor for CXCL12, and LAG3 and CXCR6, serving as T-cell exhaustion markers, were high in the CFT niche.
 SP analysis detected 6,172,072 cells and identified SCs containing 5 BEC components (artery, capillary, C-aHEV, HEV, and vein), tumor cells, and immune cell subtypes. C-aHEVs tended to be located closer to tumor cells than other BECs and formed a unified vascular niche in conjunction with HEV.
 WES analysis revealed the recurrent G17V RHOA mutations in 19 samples. Notably, samples harboring G17V RHOA mutations exhibited significantly higher densities of C-aHEV2 cells.
 Bulk RNA-seq analysis revealed that patients with high expression of C-aHEVs and other HEV-related signature exhibited poorer survival outcomes.
 Summary/ Conclusion By integrating scRNA-seq and spatial data, we identified C-aHEVs as key components in shaping the AITL microenvironment through the recruitment of immune cells. As the gene expression level, C-aHEVs can be subdivided into C-aHEV1 and C-aHEV2, which exhibited distinct properties. Further investigation is needed to validate the mechanisms by which these BEC clusters interact with tumor cells and contribute to tumorigenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/087d4af2ea5c2629eb9fea9276a3b1bcc2d2dd51" target='_blank'>
              Distinct blood endothelial cells shape spatial niches in angioimmunoblastic T-cell lymphoma
              </a>
            </td>
          <td>
            Yuki Shimizu, Manabu Fujisawa, Yoshiaki Abe, K. Makishima, S. Suma, Y. Suehara, K. Hattori, Tatsuhiro Sakamoto, Kyohei Sano, A. Tsubosaka, Miwako Kakiuchi, D. Kaji, Ayako Suzuki, Kensuke Usuki, Kosei Matsue, Yasunori Ota, Shigeru Chiba, S. Ishikawa, Yutaka Suzuki, Christian Steidl, M. Sakata-Yanagimoto
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8015c113e13d1290a915b03f174abb55236e475" target='_blank'>
              Longitudinal analysis of matched patient biospecimens reveals Neural Reprogramming of Cancer-Associated Fibroblasts Following Chemotherapy in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Aylin Z. Henstridge, Nandini Arya, Ahmed M. Elhossiny, Padma Kadiyala, Georgina Branch, Vaibhav Sahai, Nicole Peterson, Jorge D. Machicado, Richard S. Kwon, Allison Schulman, Erik Wamsteker, George M. Philips, Stacy Menees, Jonathan Xia, Arvind Rao, Jiaqi Shi, Timothy L. Frankel, Filip Bednar, Marina Pasca di Magliano, C. Lyssiotis, M. Truty, Martin Fernandez-Zapico, Eileen S. Carpenter
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="


 Hematopoiesis occurs in the complex microenvironment of the bone marrow. This includes interactions between hematopoietic cells themselves as well as interactions with non-hematopoietic cells such as stromal cells, endothelial cells, adipocytes, bone cells, and neural cells, as well as extracellular matrix materials. Given this complex milieu, it is critical to study hematopoietic regulation in the context of its native tissue architecture. However, a lack of methods to simultaneously identify cell types at high granularity and examine their spatial organization, inter-cellular signaling, and cellular states has impeded efforts to further understand hematopoietic regulation in health and disease. Recently, a suite of spatial transcriptomic technologies has emerged enabling such an analysis. Among these, Multiplex Error Robust Fluorescence In Situ Hybridization (MERFISH) is a leading platform. MERFISH is a massively multiplexed single-molecule RNA imaging method built upon single-molecule fluorescence in situ hybridization (smFISH). MERFISH multiplexes smFISH by assigning to each RNA a unique binary barcode physically encoded into a library of nucleic acid encoding probes, which is read-out through sequential rounds of smFISH measurements on the same sample. Critically, MERFISH utilizes error-robust and error-correcting encoding methods which allow this technique to image hundreds to thousands of RNAs with a high detection efficiency, a low false positive rate, and a large dynamic range. MERFISH has been applied to multiple tissues such as the brain and intestine yielding novel insights into tissue organization and signaling. A major obstacle to applying MERFISH, or other RNA-based methods, to the blood system has been the high RNase activity of hematopoietic tissues, resulting in poor RNA integrity. Here we present a new method of bone marrow preparation that prevents RNA degradation and enables high quality bone marrow MERFISH data acquisition. With this method and a 1,022 gene library we spatially mapped ~1.5 million cells from 28 mouse femoral cross sections, identifying 53 distinct cell clusters. This dataset embeds well with published dissociated single cell RNAseq datasets, verifying capture of all expected cell types. Importantly, it also includes cell types that do not dissociate well such as mesenchymal stromal cells, osteoclasts, eosinophils, mature megakaryocytes, and rare hematopoietic stem cells. With this dataset, we quantified the frequency and distance of all cell-cell contacts and find that bone marrow microenvironments are well described by a statistical model of cell contact frequency. We find that two cell types—sinusoidal endothelial cells (SECs) and mesenchymal stromal cells (MSCs)—exist in a spectrum of transcriptional states that can be defined by a pseudotime axis. The spatial intersection of high pseudotime SECs and MSCs coincides with HSC niches, indicating that specific signaling from a subset of these cells facilitates hematopoiesis. Conversely, the intersection of low pseudotime SECs and MSCs coincides with increased density of natural killer, dendritic, T-, and plasma B-cells, indicating that subsets of these cells facilitate immune microenvironments. To understand the factors that regulate these microenvironments, we interrogated our MERFISH measurements and integrated scRNAseq datasets and find a series of candidate factors secreted by MSCs and SECs that may facilitate bone marrow niche formation. To understand the restructuring of bone marrow during aging, we compared MERFISH measurements of femoral cross sections from 8-, 40-, and 90-week mice. In aged mice, we find a structural breakdown of hematopoietic organization correlating with mouse age. These results are consistent with age-associated chronic inflammation and provide a step toward understanding aging related bone marrow dysfunction. Overall, this study provides new insights into bone marrow single cell spatial organization and a new tool that should greatly facilitate further understanding bone marrow niches in health and disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638434b03a57ff13fc4e0f106290fada913449e3" target='_blank'>
              Single cell spatial transcriptomics reveals age-dependent cellular organization of mouse bone marrow
              </a>
            </td>
          <td>
            Christian Reardon-Lochbaum, Paolo Cadinu, Hao Zhang, Jeffrey R. Moffitt, Alan B. Cantor
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Identifying drivers of cancer progression to guide treatment selection is hindered by our limited understanding of tumor heterogeneity and its impact on tumor evolution. Here, we delineate the phenotypic variability across ~300,000 cells collected from multiple tumor loci in primary prostate and matched locoregional metastases using single-cell chromatin accessibility and gene expression sequencing. We find inter-patient heterogeneity to be confined to malignant populations. Within individual tumor loci, we see phenotypic heterogeneity among malignant cell populations despite a shared clonal genotypic architecture. We also observe that malignant cell populations disseminating to locoregional lymph nodes mirror the clonal architecture and phenotypic heterogeneity across primary tumor loci, while shifting from canonical prostate-cancer states to non-canonical inflammatory-like states. Our findings suggest a bottleneck imposed during the dissemination process, funneling prostate cancer cells toward an inflammatory-like cell state. These insights into the interplay between phenotypic identity and clonal architecture refine our understanding of prostate cancer progression and suggest that convergence of cancer cells towards an inflammatory-like state underlies dissemination to lymph nodes, offering a critical framework for future studies into prostate cancer metastatic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b9d84066de3071bee8ea7d75cdd27aba04e0e02" target='_blank'>
              Prostate cancer cells converge to an inflammatory-like state upon metastatic dissemination
              </a>
            </td>
          <td>
            Tina Keshavarzian, K. Furlano, Giacomo Grillo, Lisanne Mout, C. Arlidge, Faizan Hasan, Ankita Nand, Migle Mikutenaite, Evdoxia Karadoulama, Ashish Goyal, Elisabeth L Pezzuto, Jessica Heilmann, Jakub Sykora, S. Minner, Thorsten Schlomm, G. Sauter, R. Simon, H. He, Joachim Weischenfeldt, C. Plass, Clarissa Gerhäuser, M. Lupien
          </td>
          <td>2025-12-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="HPV status is a key determinant of prognosis and treatment response in head and neck squamous cell carcinoma (HNSCC). To investigate how HPV influences the tumor-immune-stromal landscape, we performed high-dimensional spatial profiling, including its impact on spatial organization, tertiary lymphoid structures (TLSs), and spatially organized cellular neighborhoods. Tumor biopsies from HNSCC patients (n = 16; 7 HPV-positive, 9 HPV-negative) were stained with a multiplex immunofluorescence (mIF) panel focused on immune profiling. A deep learning-based analysis pipeline enabled the identification and phenotypic state profiling of 14 cell types. Tissues were segmented into four distinct tumor regions, and spatial neighborhoods and TLSs were identified and analyzed for differential cellular composition, activation states, and spatial interactions between HPV-positive and HPV-negative tumors. HPV-positive and HPV-negative tumors differ in their tumor microenvironment (TME) composition, tumor cell state and spatial organization. The TME of HPV-positive tumors exhibited a greater abundance of activated lymphocytes, B- and T-cell-enriched spatial neighborhoods, and PD-1–PD-L1 interactions within the tumor area, whereas HPV-negative tumors were dominated by fibroblast- and macrophage-rich niches. T- cells in HPV-positive tumors showed greater activation across neighborhoods and areas, while in HPV-negative tumors T- cells demonstrated enrichment of exhaustion and terminal differentiation markers such as PD-1 and CD57. HPV-positive tumor cells had increased IDO1, HLA-DR, and Ki67 positivity, whereas HPV-negative tumor cells were more frequently CD44 positive, reflecting a more stem-like phenotype. Importantly, TLSs in HPV-positive tumors were located closer to the tumor area and enriched in activated immune cells, including ICOS+ CD4 T- cells, memory T- cells, and CD21+ B- cells. In contrast, TLSs in HPV-negative tumors were more distant and enriched for immunosuppressive populations such as PD-1+/PD-L1+ Tregs and macrophages. HPV status defines distinct spatial immune architectures in HNSCC. HPV-positive tumors harbor immune-activating TLSs and cellular neighborhoods that support antitumor immunity, whereas HPV-negative tumors exhibit suppressive niches and stromal dominance. These findings highlight TLSs, particularly their proximity and composition, as key features of the HPV-stratified TME and potential biomarkers for immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec5a252b8e081f817dffff9e37e6b264cbbcf5" target='_blank'>
              Spatial profiling of HPV-stratified head and neck squamous cell carcinoma reveals distinct immune niches and microenvironmental architectures
              </a>
            </td>
          <td>
            E. Markovits, Dmytro Klymyshyn, Roni Froumine, Hailing Zong, M. Mints, Sangeetha Mahadevan, Kenneth Bloom, Jamie G. Bates, Gareth J. Thomas, Lauri Diehl, Oscar Puig, Abhishek Aggarwal
          </td>
          <td>2025-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Background:
 Follicular lymphoma (FL) is a germinal center–derived B cell malignancy characterized by the clonal expansion of a dominant malignant B cell population. While prior studies have identified transcriptional subtypes across patients, most analyses have treated B cells within individual tumors as a transcriptionally uniform population. However, this assumption has not been systematically tested using high-resolution, clonotype-resolved transcriptomic data. The broader landscape of intratumoral B cells—including those not belonging to the dominant clone—remains poorly defined, and their roles in disease biology are not well understood. These unexpanded, intratumoral B (ITB) cells may include residual non-malignant populations, early-stage malignant cells, or immune-modulating cells with distinct roles in the tumor microenvironment. It is also unknown whether their presence varies across patients or correlates with clinical features or immune microenvironmental interactions.
 Methods:
 We performed integrated single-cell 5′ RNA sequencing and B cell receptor (BCR) repertoire sequencing on 7 newly diagnosed FL 1-3A fresh-frozen lymph node biopsies (Dasari et al., Blood Cancer Discov 2025). Using a high-confidence approach that incorporates heavy and light chain pairing, V and J gene usage, and CDR3 nucleotide junction identity, we classified B cells into two groups: (1) cells belonging to the dominant expanded clone (FL B cells) and (2) all other B cells with detectable and non-expanded BCRs (ITB). These clonal identities were then integrated with single-cell transcriptomic profiles using Seurat and Immcantation pipelines. Differential expression analysis was conducted using edgeR to compare FL B cells to ITB cells across all FL biopsies.
 Results:
 After quality filtering, a total of 44,637 single cells were retained across all 7 samples (minimum per sample: 3,847 cells), including malignant and non-malignant B cells as well as diverse tumor microenvironment populations. The proportion of B cells assigned to the dominant FL clone varied substantially across tumors, ranging from 22% to 89%, indicating that some tumors contain a substantial population of ITB cells. In one tumor, we identified two expanded malignant B cell clones comprising 40.6% and 34.4% of all cells. These clones exhibited non-overlapping V and J gene usage (IGHV3-11/IGHJ5 vs. IGHV3-72/IGHJ3), distinct CDR3 junctions, and different immunoglobulin isotypes (IGHG1 vs. IGM). BCL2 was expressed in most cells within both clones (Clone 1: 87.8%, Clone 2: 67.0%), with mean log-normalized expression levels of 1.79 and 0.95, respectively. Both clones formed distinct UMAP clusters and showed transcriptional divergence from ITBs, with 453 differentially expressed genes (DEGs) identified for Clone 1 and 98 for Clone 2 (adjusted p<0.01, |log₂FC|>1, min.pct≥0.25). Additionally, 748 DEGs were identified between the two clones. This evidence of clonal divergence within a single FL tumor reinforces the need to reconsider intratumoral homogeneity in FL and suggests that multiple malignant clones may sometimes coexist.
 When comparing FL clones to ITB cells across tumors we found that FL clones exhibited a transcriptional program enriched for proliferation and cell cycle regulators, including MKI67 (p<0.005), E2F1 (p<0.0001), CDKN2A (p<0.0001), and CDKN2B (p<0.01), as well as elevated expression of AICDA (p<0.01), EZH2 (p<0.0001), and PD-L1 (p<0.05). As expected, the malignant cells also had higher expression of BCL2 (p<0.01) and CD10/MME (p<0.0001). In contrast, ITB cells demonstrated relatively higher expression of SELL (p<0.0001) and TNFRSF14 (p<0.05), which were significantly downregulated in the FL clone.
 Conclusions:
 Together, these findings challenge the prevailing notion that FL tumors are defined by a single homogeneous malignant B cell population. Instead, our data support a model in which multiple transcriptionally and functionally distinct B cell populations coexist within tumors, including dominant clones and a diverse population of ITB cells. This is, to our knowledge, the first study to comprehensively characterize these populations using integrated, lineage-aware single-cell multiomic profiling across FL patients. These insights may ultimately inform new strategies for understanding tumor evolution, immune evasion, and therapeutic targeting in follicular lymphoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2df969e2ff2052fc6b44c14d817620d6381a9b7" target='_blank'>
              Intratumoral B cell diversity in follicular lymphoma revealed by single-cell paired transcriptome and BCR sequencing
              </a>
            </td>
          <td>
            Miranda S. Siminski, J. Walker, Joseph Novak, A. Dropik, Zilin Xianyu, Zhi-Zhang Yang, M. Maurer, Thomas M Habermann, Rebecca King, T. Witzig, S. Ansell, J. Cerhan, P. Mondello, Anne J Novak
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The tumor microenvironment (TME) has emerged as a promising source of prognostic biomarkers. To fully leverage its potential, analysis methods must capture complex interactions between different cell types. We propose HiGINE -- a hierarchical graph-based approach to predict patient survival (short vs. long) from TME characterization in multiplex immunofluorescence (mIF) images and enhance risk stratification in lung cancer. Our model encodes both local and global inter-relations in cell neighborhoods, incorporating information about cell types and morphology. Multimodal fusion, aggregating cancer stage with mIF-derived features, further boosts performance. We validate HiGINE on two public datasets, demonstrating improved risk stratification, robustness, and generalizability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d207f0757528c944b801713e062bc7358ff14382" target='_blank'>
              From Cells to Survival: Hierarchical Analysis of Cell Inter-Relations in Multiplex Microscopy for Lung Cancer Prognosis
              </a>
            </td>
          <td>
            Olle Edgren Schullerqvist, Jens Baumann, Joakim Lindblad, Love Nordling, Artur Mezheyeuski, Patrick Micke, Natavsa Sladoje
          </td>
          <td>2025-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Human breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of human genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This epithelial–mesenchymal transition (EMT) compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cancer cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d470a8773036dd5ad3cf2b42848d5f3ff555882" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial to mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. P. McCord
          </td>
          <td>2025-11-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 H3K27 altered diffuse midline gliomas (DMG) are aggressive brain tumors that arise centrally in the brain. These tumors are not amenable to surgery and no curative therapy exists. DMG exhibits profound heterogeneity, with tumor cells displaying dynamic and interchangeable cellular states that recapitulate aspects of normal development. In parallel, the tumor microenvironment (TME) is highly diverse, further contributing to the complexity of the disease. A better understanding of the DMG ecosystem is needed for therapeutic target identification. In this study, publicly available single cell RNA seq data of DMG (Jessa et al. Nat. Genet. 2022) were analyzed to define key molecular characteristics of DMG. Xenium spatial transcriptomics and multiplex immunohistochemistry were performed on FFPE patient samples collected at diagnosis to study the tumor and TME cells in their native spatial context. In DMG, both the glioma cellular composition and the heterogeneity of the TME varied based on the anatomical location of the tumor. Tumors located in the pons exhibited a high proportion of astrocyte-like glioma cells. In contrast, thalamic DMG showed a greater abundance of oligodendrocyte-like tumor cells, suggesting a regional bias in cellular differentiation states. Furthermore, the TME in thalamic tumors was notably enriched with immunotolerant macrophage populations, expressing molecules associated with an anti-inflammatory, pro-tumoral phenotype. With spatial profiling, distinct tumor niches were identified, showing heterogeneity among different samples and underscoring the complex and spatially organized nature of both tumor cell states and the surrounding microenvironment in DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/408581e61f93671e8e7b1c9fc7f6b7a6669a068a" target='_blank'>
              EPCO-24. DISSECTING THE ECOSYSTEM OF PEDIATRIC H3K27 ALTERED DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Julie Morscio, Chiara Caprioli, C. González-Blas, Pierluigi Calò, Katy Vandereyken, Thierry Voet, R. Sciot, Sandra Jacobs, F. De Smet
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23889452eb3a744641d2c79b511978437ffd39c" target='_blank'>
              A new genetically engineered transplant model of glioma recapitulates key phenotypes of low- and high-grade gliomas
              </a>
            </td>
          <td>
            D. Forsythe, Shannon J. Oliver, A. Valkovic, Danielle Lewthwaite, Germaine Uys, Brittany Koning, David Eccles, L. Satgunaseelan, I. Hermans, S. Freytag, Jim R Whittle, Sarah A. Best, Melanie J. McConnell
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PAAD) metastasis is driven by complex tumor–microenvironment interactions. Here, we integrated single-cell and bulk transcriptomic analyses of 104,855 cells from 10 patients to delineate the cellular and molecular landscape of primary versus metastatic PAAD. We identified metastasis-associated epithelial (LMO7⁺, TOP2A⁺, PIGR⁺), fibroblast (IGKC⁺, RGS5⁺), and M2-like macrophage (APOE⁺, CD14⁺, FOLR2⁺, SPP1⁺) subpopulations, validated via bulk deconvolution. Functional analyses revealed upregulated Wnt signaling, epithelial–mesenchymal transition, and angiogenesis in metastatic epithelial and fibroblast compartments. Intercellular communication analysis highlighted SPP1-mediated macrophage–epithelial/fibroblast crosstalk involving key receptor–ligand pairs, contributing to immune suppression and metastatic niche formation. Integrating gene expression and cell proportions, we developed a prognostic model with high predictive accuracy (C-index > 0.85), stratifying patients into risk groups with distinct immune landscapes. Furthermore, PTK6 was identified as a driver of PAAD proliferation, migration, and invasion. Collectively, our study elucidates TME-driven mechanisms of PAAD metastasis, identifies prognostic and therapeutic targets, and provides a framework for precision intervention. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01974-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759c03413ef4b1106fa2a25c75713885026e6884" target='_blank'>
              Single-cell transcriptomic analysis reveals key cell types and pathogenic genes associated with metastasis in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Shijun Shen, Zhiqiang Li, Hong Yang, Xinwei Zhang, Gang Chen, Chengzhou Pa
          </td>
          <td>2025-11-29</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is an aggressive brain tumor with limited treatment options, and the tumor microenvironment plays a major role in its progression. Among stromal cells, cancer-associated fibroblasts (CAFs) are known to influence tumor growth and immune responses, but their diversity in glioblastoma has not been well understood. In this study, we analyzed both bulk and single-cell RNA sequencing data to identify CAF subtype-associated patterns within glioblastoma. We discovered four CAF populations—immune, inflammatory, antigen-presenting, and myofibroblastic—each associated with poor patient survival and unique immune and metabolic features. We also identified Spleen Tyrosine Kinase (SYK) as a key regulatory gene shared across these CAF subtypes. Our findings highlight the importance of stromal heterogeneity in glioblastoma and suggest that targeting CAF-related signaling, including SYK, may offer new therapeutic opportunities for this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77eebcce441ee2b3c1609920849b71d1de5f2a30" target='_blank'>
              Dissecting CAF Heterogeneity in Glioblastoma Reveals Prognostic Subtypes and a Central Regulatory Role for Spleen Tyrosine Kinase (SYK)
              </a>
            </td>
          <td>
            Ji-Yong Sung, K. Hwang
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 The diffusely infiltrative growth pattern of glioblastoma (GBM) is a major obstacle to effective therapy. A complex network of tumor intrinsic and microenvironmental features interact to shape the landscape of this growth pattern, but deciphering the molecular substrates of these interactions remains challenging. While single cell RNA sequencing has reshaped our understanding of GBM heterogeneity and uncovered cellular phenotypes of many cellular populations in these tumors, the spatial context of this heterogeneity could not be assessed with earlier technologies. By contrast, spatial transcriptomics platforms have the potential to map this transcriptome-wide information in situ. We recently reported the use of this technology to develop a layered model of spatial architecture in GBM and showed that hypoxia represents a key organizing feature of GBM heterogeneity. However, the limited resolution of this platform precluded assessment of whole-transcriptome features at truly cellular resolution. In this work, we now present our findings utilizing the next generation of this technology and showcase the ability to map whole transcriptome data in spatial context at true single cell resolution in primary patient GBM samples. We explore heterogeneous GBM phenotypes in the context of GBM spatial microarchitecture. Using copy number inference, we demonstrate the ability to assess clonal heterogeneity in situ and map the pattern of invasion of unique GBM clones. Finally, we discuss our progress to discover organizing principles underlying distinct patterns of GBM cell invasion. Taken together, we demonstrate the potential of single cell spatial transcriptomics to decipher the molecular heterogeneity of GBM microenvironment in situ and uncover the biology driving the aggressive infiltrative growth of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad815e1440b141545b8a8eee62737b92edef994" target='_blank'>
              TMIC-68. Deciphering the spatial architecture of transcriptional heterogeneity in glioblastoma at single cell resolution
              </a>
            </td>
          <td>
            C. Mount, Zebin Wen, Alissa C. Greenwald, S. Kovatsis, Jack Lu, D. Gerovasilis, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e493ac7b6aab5ae1ab712cb247fef1395ea0b1a" target='_blank'>
              Quantitative profiling of intratumor immune heterogeneity identifies loss of immune diversity as a hallmark of cancer progression.
              </a>
            </td>
          <td>
            Qiqi Lu, Jian-qin Luo, Chia-Hao Tung, Xiaosheng Wang, Zhongming Zhao
          </td>
          <td>2025-12-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Age is a dominant risk factor for all major breast cancer subtypes. However, the mechanisms by which aging influences tumor development remain unclear. Using a novel mouse model whereby breast cancer is induced in situ in young and old wild-type mice via intraductal delivery of a lentivirus encoding the HER2/neu oncogene, we found that old mice exhibited a higher oncogene-induced tumor burden than young mice. Old tumor cells showed reduced expression of interferon-related genes, particularly the T cell-recruiting chemokines Cxcl9 and Cxcl10, linked to their altered chromatin accessibility. CXCL9/10 expression also declined with age in human HER2+ tumors. Correspondingly, old tumors exhibited fewer T cells within tumor lesions. Targeted interventions showed that decreased expression of Cxcl9/10 is responsible for reduced T cell infiltration and weakened anti-tumor immunity. These results show how aged tumor cells are impaired in their recruitment of immune cells, leading to a defective anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38011df7fcbd774b6090832583a2578934863ea" target='_blank'>
              Aging alters tumor cell - T cell crosstalk to promote breast cancer progression
              </a>
            </td>
          <td>
            Shanshan Yin, Kay T Yeung, S. Mamde, Kate Lin, Armin Gandhi, Xue Lei, Andrew Davis, Rouven Arnold, Jing Yang, P. Adams
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 The human brain contains a rich milieu of extracellular matrix (ECM) components that promote normal physiology and are often dysregulated in pathologies including the malignant cancer glioblastoma (GBM). We have used in situ single-cell spatial transcriptomic platforms to map the expression patterns of nearly 400 ECM-related genes in multiple GBM samples (IDH1 wild type). Our analysis identifies at least four different GBM cell populations with unique ECM expression profiles that show spatial enrichment in distinct intratumor regions. Spatial mapping also demonstrates largely non-overlapping expression signatures of various ECM components in stromal cell types, particularly in GBM vascular endothelial cells and reactive microglia/macrophages. Comparisons of GBM versus lower-grade II and III astrocytoma samples (IDH1 R132H) identifies differential expression of key ECM components, including elevated levels of select ECM glycoproteins (IGFBP2 and MGP) and ECM-affiliated proteins (ANXA1 and ANXA2). In addition, we detect spatially enriched expression of COL8A1 (collagen), LUM (proteoglycan), and POSTN (ECM glycoprotein) in perivascular stromal cells in GBM but not in lower grade tumors. Computational analysis of putative ligand-receptor interactions reveals novel communication networks between cancer cells and stromal components, particularly in regions of GBM microvascular proliferation and pseudopalisading necrosis. In summary, this comprehensive spatial map provides new insights into ECM control of tumor initiation and progression and identifies potential therapeutic targets within the GBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e40eadeccedca2699836ba1458071cfd535dcbfb" target='_blank'>
              TMIC-33. Single Cell Spatial Profiling of Matrisome Gene Expression in Glioblastoma
              </a>
            </td>
          <td>
            Joseph H. McCarty, Arpan De, Santiago A Forero, Ali Pirani, John E. Morales
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01df2c37dabe824163d0ae6c9689244efcafc8f6" target='_blank'>
              Multi-scale Network Topology Analysis Reveals Metabolic Reprogramming and Therapeutic Vulnerabilities in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Handan Cetin, P. Bogdan, S. Mumenthaler, Stacey D. Finley
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy, poised to become the second leading cause of cancer-related deaths in North America. The paucity of neoantigen presentation in PDAC has been hypothesized to promote the development of an “immunologically-cold” tumor microenvironment (TME), driving poor clinical responses to immunotherapy. Despite the abundant immune infiltrate observed in PDAC, T cells display a limited cytotoxic capacity to kill malignant tumor cells which further diminishes throughout disease progression. Utilizing single cell RNA-sequencing coupled to T cell receptor profiling, we have characterized T cell populations in clinical specimens of inflammatory pre-malignant intraductal papillary mucinous neoplasia (IPMN) lesions and advanced stage PDAC tumors. Striking phenotypic and clonal dynamic differences were observed in both CD8 and CD4 T cell populations between IPMN and PDAC specimens which underly their distinct functional capabilities in the evolving TME. The spatial interactions and microenvironmental neighborhoods of neoantigen-reactive T cells in PDAC have yet to be explored in-depth and hold unexploited therapeutic potential. Utilizing the 10X Genomics Xenium platform with custom probes designed to hybridize with neo-antigen-specific T cell clones, we are interrogating the cellular interactions which drive T cell clonal expansion to identify therapeutically relevant vulnerabilities in PDAC malignant progression.



 Supported by National Insitutes of Health, Canadian Institutes of Health Research, and Princess Margaret Cancer Center Catalyst Grant.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f6400ccbc9c89325532880266eae31b90968879" target='_blank'>
              Defining molecular determinants of T cell spatial localization and clonal response throughout PDAC malignant progression 3030
              </a>
            </td>
          <td>
            Matthew Bianca, Sara Lamorte, Teresa Ciudad, Cristiane Naffah de Souza, Rene Quevedo, Tracy Mcgaha
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4726aa2358332b95021c3fdf2dd12bc437304fe5" target='_blank'>
              Targeting CREB remodels the immune microenvironment to enhance immunotherapy responses in pancreatic cancer
              </a>
            </td>
          <td>
            S. Mehra, Supriya Srinivasan, Sudhakar Jinka, Varunkumar Krishnamoorthy, Vineet Kumar Gupta, Vanessa Garrido, Anna Bianchi, Andrew Adams, Manan Patel, H. Amirian, Luis A Nivelo, Karthik Rajkumar, Rimpi Khurana, Edmond W Box, Yuguang Ban, Oliver G McDonald, J. Datta, Kathleen E. DelGiorno, Nipun B Merchant, Sangeeta Goswami, A. Dosch, Nagaraj S. Nagathihalli
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f0d1b1f44b4f72bd47f0ccdb51001e256798703" target='_blank'>
              A Unified Single-Cell Atlas of HNSCC: Uncovering HPV and Sex Variability in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Cristina Conde Lopez, Divyasree Marripati, M. J. Besso, Mareike Roscher, Rui Han, W. W. Hadiwikarta, M. Elkabets, Jochen Hess, Ina Kurth
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Pulmonary pleomorphic carcinoma (PPC) is a rare subtype of lung cancer that comprises both epithelial and sarcomatoid components. The molecular basis of PPC, including the cellular dynamics of its components, remains largely unknown. To elucidate potential therapeutic targets for PPC, we perform a multi-omics analysis incorporating digital spatial profiling and single-cell RNA sequencing (scRNA-seq). PPC exhibits diverse driver gene alterations, including MET exon 14 skipping mutation (METex14) and ALK fusion. In spatial transcriptomics, MET gene and protein are overexpressed exclusively within the epithelial component and not in the sarcomatoid component, even in patients harboring METex14. Epithelial-mesenchymal transition (EMT)-related transcriptional changes, along with extracellular matrix (ECM) remodeling between the epithelial and sarcomatoid components, are observed. scRNA-seq identifies cell populations within the epithelial component that contribute to the malignant transformation and differentiation of the sarcomatoid component. They are characterized by an intermediate EMT state with ECM remodeling signature, suggesting their potential as novel therapeutic targets for PPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f421d642675df6e6bdc9b51c7d9b23970c7bec20" target='_blank'>
              Single-cell and spatial transcriptomic characterization of pulmonary pleomorphic carcinoma
              </a>
            </td>
          <td>
            Atsushi Matsuoka, K. Shien, Shuta Tomida, Masayoshi Ohki, Kazuya Hisamatsu, Ryota Fujiwara, Kosei Ishimura, Ryunosuke Fujii, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Ryo Yoshichika, Fumiaki Mukohara, Mao Yoshikawa, Yuma Fukumoto, K. Suzawa, Y. Tomioka, Shin Tanaka, Kentaroh Miyoshi, Mikio Okazaki, S. Sugimoto, Yusuke Otani, Atsushi Tanaka, Hirofumi Inoue, Yosuke Togashi, Hidetaka Yamamoto, D. Ennishi, Shinichi Toyooka
          </td>
          <td>2025-12-16</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b0c553f54a8eda4d3dc1abe42f3db69a41cfe1" target='_blank'>
              Single-cell and spatial transcriptomic profiling reveals distinct tumor microenvironment dynamics in cervical adenocarcinoma and squamous cell carcinoma.
              </a>
            </td>
          <td>
            Yaqin Wu, Bei Wei, Zhentong Wei, Hongyan Wang, Yunyun Liu, Dongdong Xu, Xiaokang Li, Liyuan Guo, Huaiwu Lu, Huiling Shang
          </td>
          <td>2025-12-19</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Super-enhancers (SEs) are clusters of enhancers with potent regulatory capabilities. They play a crucial role in shaping cellular identity and driving the progression of various diseases, including cancer. SEs exhibit significant heterogeneity across different cell types and cancer subtypes. Analysis of SE landscapes can recapitulate existing classification systems and unveil novel SE-driven epigenetic subtypes. In this review, we summarized the latest advancements in cancer subtype identification based on SE-related characteristics, outlined the typical analytical workflows adopted in such studies, and explored the biological and clinical significance of SE-driven subtypes. Furthermore, we discussed the field’s key challenges and emerging technologies to highlight future research directions. SE analysis provides a robust framework for dissecting cancer heterogeneity. This approach offers novel epigenetic perspectives and support for the realization of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/285d6877af1dc808910fb0a7890277cebd5d2976" target='_blank'>
              Decoding cancer heterogeneity through super-enhancer landscapes: from subtype discovery to therapeutic opportunity
              </a>
            </td>
          <td>
            Shuyang Cai, Zhenchang Wang, Xiao Sun, Hao Huang
          </td>
          <td>2025-11-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumor immune microenvironment plays a critical role in tumor progression and responses to immunotherapy. Nevertheless, its cellular complexity and heterogeneity remain incompletely understood. In this study, we employed high-resolution single-cell RNA sequencing on CD45+ immune cells isolated from ten syngeneic murine tumor models, representing seven distinct cancer types under treatment-naïve conditions, thereby enabling a comprehensive profiling of tumor-infiltrating immune cells. We identified seven principal immune cell populations and provided an in-depth characterization of T cells, NK/innate lymphoid cells, dendritic cells, monocytes/macrophages, and neutrophils. Cross-species analyses further delineated conserved immune cell states and transcriptomic features within the T cell and monocyte/macrophage compartments that are shared across syngeneic models and human tumors. To investigate the functional relevance of the predominant monocyte/macrophage compartment and the notable presence of neutrophils in syngeneic tumors, we evaluated responses to anti-PD-1 therapy across various models and analyzed the enrichment of monocyte/macrophage subsets in tumors that responded to treatment. Furthermore, we conducted neutrophil depletion experiments using anti-Ly6G antibodies, administered both as monotherapy and in combination with PD-1 blockade. Remarkably, an interferon-stimulated gene-high (ISGhigh) monocyte subset was significantly enriched in models responsive to anti-PD-1 therapy. Neutrophil depletion resulted in variable antitumor effects across models but failed to enhance the efficacy of PD-1 blockade. In summary, our single-cell profiling offered a detailed atlas of the immune microenvironment across multiple syngeneic mouse tumor models, thereby enabling rational model selection for immuno-oncology studies. We uncovered an ISGhigh monocyte subset enriched in anti-PD-1 responsive models, and showed the context-dependent effects of neutrophil depletion on tumor immunity and immunotherapy, underscoring the heterogeneity and functional divergence of immune cell sublineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f925e5b4fe351a72672a38a0505d78e0be84e52" target='_blank'>
              Single-cell atlas of the tumor immune microenvironment across syngeneic murine models
              </a>
            </td>
          <td>
            Jia Wang, Bin Jiang, Minjuan Deng, Han Yan, Peilin Zhang, Wei Jin, Zhirong Shen
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05c188c0c6e2f128d8772ff9b6be6a1353be8fc1" target='_blank'>
              A Hybrid Multiscale Model for Predicting CAR-T Therapy Outcomes in Solid Tumors
              </a>
            </td>
          <td>
            M. Nikmaneshi, Lance L. Munn
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Endometrial cancer is regarded as one of the most prevalent malignancies in women globally. Despite the advancements brought by multiple therapeutic strategies, the efficacy and effectiveness of treatment still appear to be diminished. Therefore, this accentuates the importance of understanding the tumour microenvironment (TME). Conventional methods, including immunohistochemistry (IHC), along with recently emerged techniques like single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST), have rejuvenated the notion of deciphering TME. From deconstructing tumour heterogeneity, identifying cell populations that play significant roles in treatment response, to discovering key biomarkers and therapeutic targets, these technologies lay the base for innovations. Importantly, this is the first comprehensive review that brings together scRNA-seq, ST, and IHC in Endometrial cancer. Merging these methods is geared towards creating a better grasp of the dynamics and interactions between diverse cells and the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bef0a0a3a7a3aad177138a9aca3da36e10bf85b" target='_blank'>
              Navigating tumour microenvironment in endometrial carcinoma: a comprehensive review integrating immunohistochemistry, single-cell RNA-sequencing and spatial transcriptomics
              </a>
            </td>
          <td>
            Oishee Mondal, Blessy Kiruba, Sajitha Lulu Sudhakaran, V. Sundararajan
          </td>
          <td>2025-11-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The spatial organization of the tumor microenvironment (TME) encodes the selective pressures that drive cancer evolution, but many tools focus on local cell contacts or single-modality readouts. We developed Tumor Landscape Analysis (TLA), a modular framework that borrows from landscape ecology and spatial statistics to quantify heterogeneity, fragmentation, and diversity across scales, from single cells to whole-slide ecosystems.



 TLA ingests whole-cell or nuclear centroids and region-level segmentations to compute (i) point statistics such as nearest-neighbor distance, Ripley’s H, and Morisita-Horn colocalization; (ii) kernel-smoothed local spatial profiles including Getis-Ord Gi* hot and cold spots; and (iii) fragmentation, shape, and diversity metrics such as patch density, contagion, shape index, and Shannon entropy. We also introduce Local Microenvironments (LMEs), unsupervised ecological niches defined by joint cell-type abundance and spatial mixing. Pilot analyses were performed in ductal carcinoma in situ (DCIS; n=353 samples, 175 patients) with cell-level maps of epithelial, lymphocyte, and fibroblast populations, and in pancreatic ductal adenocarcinoma (PDAC; n=203 resected cases after neoadjuvant therapy) using cancer and stroma segmentations.



 In DCIS, spatial relationships rather than raw abundance associated with recurrence risk. Shorter lymphocyte-to-fibroblast minimum nearest-neighbor distance and lower Ripley’s H, indicating greater intermixing, correlated with earlier recurrence independent of cell counts. This highlights immune–stromal proximity as a candidate eco-evolutionary driver of persistence. In PDAC, tissue-level fragmentation captured prognostic architecture after therapy. Two shape-based signatures emerged: (1) stromal uniformity, where lower standard deviation of stromal patch shape index associated with improved disease-free survival, consistent with a more constrained, less reactive stroma; and (2) tumor geometric complexity, where higher mean cancer patch shape index, reflecting more irregular or dendritic nests, associated with lower recurrence risk, whereas compact nests predicted worse outcomes. Across datasets, LME maps localized niche phenotypes, such as mixed-immune or segmented tumor habitats, that align with hypothesized selection landscapes and provide inputs for fragmentation analysis.



 TLA operationalizes an eco-evolutionary view of cancer by unifying interpretable, multiscale spatial metrics in a single workflow that works with any rasterized data input. By quantifying how habitats, edges, and mixing patterns are organized, TLA reveals niche structures and architectural motifs linked to clinical outcomes and complements foundation models and modality-specific pipelines with theory-grounded, clinically translatable descriptors. Ongoing work integrates TLA features with molecular data and outcomes to derive robust biomarkers and to inform adaptive, niche-targeted strategies in precision oncology.



 Ryan M. Carr, Luis Cisneros, Merih Deniz. Toruner, Martin E. Fernandez-Zapico, Carlo Maley. Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f5d8ed56157756f83dcf8a730cae9e8b632c46" target='_blank'>
              Abstract A015: Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments
              </a>
            </td>
          <td>
            Ryan M. Carr, L. Cisneros, M. D. Toruner, Martín E Fernandez-Zapico, Carlo C. Maley
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Hepatocellular carcinoma (HCC) is a deadly and highly complex cancer. To better understand what makes it aggressive, we investigated the key genetic switches, known as transcription factors (TFs), that control tumor cell behavior. By analyzing vast amounts of genetic data from single cells, we identified a core network of nine TFs that drive HCC progression. We found that these TFs orchestrate a shift in tumor cells from a state of rapid growth to a more invasive and metabolically adapted state. This change also reshapes the tumor’s surroundings, creating an environment that suppresses the immune system, largely by recruiting and activating specific immune cells called SPP1+ macrophages. This coordinated action between tumor cells and immune cells promotes cancer growth and immune evasion. Our findings highlight this TF–macrophage axis as a promising new target for developing more effective HCC therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989aaab1ae1f0d6fa871b18d9e4cad44d18f7594" target='_blank'>
              Transcription Factor Networks Drive Tumor Progression and Immune Microenvironment Remodeling in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Sang Hoon Lee, Juwon Ahn, Wonbin Choi, Jina Kim, Joon Yeon Hwang, Jae-Hwan Kim, Hyaekang Kim, Woori Kwak
          </td>
          <td>2025-11-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Spatial Multiomics allows for context-based study of the tumor microenvironment (TME). TME consists of diverse cells that collectively influence tumor progression and immune activation. CD8 T-cells, essential for antitumor immunity, often become exhausted in the TME. PD1+ TCF1+ stem-like CD8 T-cells retains differentiation potential and can be rejuvenated through cytokine signaling and immune checkpoint modulation. Here we introduce a workflow for RNAscope™ Multiplex Fluorescent v2 assay combining RNA in-situ hybridization (ISH) and protein immunofluorescence to study cell marker (CD8, PD1) with RNA targets (TNFA, TCF7, IFNG) in a tumor microarray (TMA). The Manual Multiplex Protease-Free workflow enables simultaneous RNA and protein visualization without enzymatic disruption, preserving antigen integrity and tissue morphology. Using this Multiomics assay, we can study CD8 T-cell differentiation and activation. Co-detection of RNA transcripts and proteins provides insights into specific T cell phenotypes and their activation/exhaustion status. Data reveal robust co-expression patterns of these targets, particularly in the tumor-cell neighboring stromal and immune cells in breast, cervical and stomach cancers, reflecting immune activation and exhaustion across distinct TME niches. This methodology lays a foundation for refining immunotherapeutic strategies targeting PD1+ TCF7+ CD8 T-cells, enhancing effector functions, and advancing precision oncology interventions.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bea7d4ce3faf01c213d46ecb8b271a789ba794e6" target='_blank'>
              Spatial multiomics reveals T-cell activation and exhaustion states in the tumor microenvironment 3446
              </a>
            </td>
          <td>
            Steve Zhou, Yifan Wang, Debia Wakhloo, A. Dikshit, Li-chong Wang, Maithreyan Srinivasan
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) poses a great challenge for therapeutics development due to its heterogeneity and the lack of immunocompetent models that faithfully recapitulate human GBM. We sought to generate and characterize syngeneic mouse models for major GBM subtypes.



 Postnatal electroporation using the PiggyBac transposon/transposase for overexpression (OE) and CRISPR/Cas9 for knockout (KO) was used to generate models with common GBM alterations including EGFRvIII-OE/Cdkn2a-KO/Pten-KO, Nf1-KO/Pten-KO/p53-KO, and RasV12-OE/Cdk4-OE/p53-KO. Model characterization was done by HE staining, immunofluorescence staining, RNA sequencing and flow cytometry.



 All models displayed histopathological and molecular features of GBM such as pseudopalisiding necrosis, microvascular proliferation, and cellular atypia, as well as Gfap and Nestin expression. On the transcriptomic level, Nf1/Pten/p53 tumors showed enrichment for the human mesenchymal signature, while RasV12/Cdk4/p53 and EGFRvIII/Cdkn2a/Pten tumors resembled the classical and proneural signatures. Each tumor was comprised of four cellular states, i.e., astrocytic, mesenchymal, OPC-like, and NPC-like, suggestive of intratumoral heterogeneity. On examining the microenvironment, EGFRvIII/Cdkn2a/Pten tumors had a higher relative abundance of T cells (48.6% vs 6.3%, p=0.0084, 48.6% vs 8.8%, p=0.0376,), NK cells, (1.2% vs 0.4%, p=0.1425, 1.2% vs 0.19%, p=0.0315) and dendritic cells (7.5% vs 0.4%, p=0.0272) than the Nf1/Pten/p53 and RasV12/Cdk4/p53 models. RasV12/Cdk4/p53 tumors had greater infiltration of monocyte-derived macrophages than Nf1/Pten/p53 (35.9% vs 4.18%, p=0.0012) and EGFRvIII/Cdkn2a/Pten (35.9% vs 10.4%, p=0.0620) tumors as well as an elevated CD4+/CD8+ T cell ratio. In the Nf1/Pten/p53 model, microglia were the most abundant cells within the CD45+ compartment, surpassing levels observed in EGFRvIII/Cdkn2a/Pten (60.2% vs 4.6%, p=0.0548) and RasV12/Cdk4/p53 (60.2%vs7.5%, p=0.0564) tumors.



 We developed an array of syngeneic mouse models of GBM that recapitulate human inter- and intra- tumoral heterogeneity. Future studies will define the functional states of immune cells in the microenvironment and evaluate the impact of immune cell composition on the response to immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9fb466bb8d2e828409bfc681044d64171c2f26" target='_blank'>
              TMOD-40. Genetic drivers shape tumor phenotype and composition of the tumor microenvironment in syngeneic mouse models of glioblastoma
              </a>
            </td>
          <td>
            J. Pyczek, Shannon Snelling, Xueqing Lun, Bo Young Ahn, Heewon Seo, Miranda Yu, Daniela F. Quail, A. Morrissy, Jennifer A Chan
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Clear cell renal cell carcinoma (ccRCC) exhibits significant heterogeneity due to morphological changes and tumor microenvironment dynamics, influencing systemic therapy responses. While the role of high-mobility group AT-hook 2 (HMGA2) in tumor progression has been implicated in other cancers, its significance in ccRCC remains unclear. This study investigates the role of HMGA2 in these processes and its clinical impact. Methods Spatial transcriptomics (ST) was performed on primary ccRCC samples to investigate expression trajectories associated with HMGA2 expression and morphological evolution. In metastatic ccRCC cohorts treated with systemic therapy, immunohistochemistry and bulk RNA sequencing data were analyzed to evaluate molecular and clinical features in relation to HMGA2. Single-cell RNA sequencing (scRNA-seq) data were used to explore immune cell populations and their interactions. Based on these findings, multiplex immunohistochemistry (mIHC) assessed spatial distribution, cell–cell interactions, and pathological responses of key immune populations. Results HMGA2 expression was associated with aggressive morphological patterns, such as solid sheets and rhabdoid/sarcomatoid. ST revealed a progressive increase in HMGA2 expression along the morphological trajectory, marked by a shift from clear to eosinophilic cytoplasm, with eccentric nuclei and prominent nucleoli, and loss of vascular architecture. HMGA2-high tumors exhibited aggressive phenotypes driven by cell cycle, epithelial-mesenchymal transition, and inflammatory signaling pathways. Clinically, patients with high HMGA2 had worse progression-free survival but responded better to immune checkpoint inhibitor combination (Combo-ICI) therapy than to tyrosine kinase inhibitor monotherapy. To assess the immune landscape, scRNA-seq data revealed that HMGA2-high tumors were enriched with progenitor exhausted CD8+ T cells (Tpex), along with increased frequencies of conventional dendritic cell type 1 (cDC1) and inflammatory cDC type 2, which were found to interact with Tpex via ICAM-1. mIHC confirmed that Tpex were enriched among Combo-ICI responders in HMGA2-high tumors, with higher densities and closer proximity to ICAM-1+ cDC1. Conclusions These findings suggest that dynamic HMGA2 expression contributes to morphological evolution and modulates immune responses through enhanced Tpex–cDCs engagement, serving as a potential marker for systemic therapy response in ccRCC. However, additional experimental studies are required to validate these mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c12fc33545db9ae9db8d540caf1522acd61b8f02" target='_blank'>
              HMGA2 links morphological evolution and microenvironment dynamics to systemic therapy response in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            T. Nakamoto, Takashi Yoshida, C. Ohe, Yoshiki Yasukochi, Naho Atsumi, Takeshi Sano, K. Higasa, Katsunori Uchida, Koji Tsuta, Hidefumi Kinoshita
          </td>
          <td>2025-11-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Tumor heterogeneity is one of the key factors leading to diagnostic errors, difficulty in identifying subtle cellular differences, and the occurrence of immune escape. Although single-cell RNA sequencing (scRNA-seq) has elucidated the molecular characteristics and heterogeneity at the single-cell level, it fails to retain spatial location information. In contrast, spatial transcriptomics (ST) technology can simultaneously acquire gene expression information and cellular localization, providing a unique perspective for studying the tumor microenvironment, cell-cell interactions, and other aspects. The integrated application of scRNA-seq and ST technologies enables the analysis of tumor heterogeneity from both molecular and spatial dimensions, offering comprehensive information support for precision medicine. The complementary advantages of these two technologies not only overcome the limitations of traditional methods but also promote the research on complex mechanisms such as tumor heterogeneity and the tumor microenvironment, accelerating the translational process of clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9514d53fae21da5a677d774e552caceeac1e73" target='_blank'>
              Research Progress in Single-Cell RNA Sequencing and Spatial Transcriptomics for Precision Oncology
              </a>
            </td>
          <td>
            Minxin Mo
          </td>
          <td>2025-12-18</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Immunotherapy has transformed cancer treatment, highlighting the importance of effective antitumor immunity to fight cancer. However, its success in pediatric cancer remains limited, underscoring the urgent need to identify new immunotherapeutic targets. In this study, we explored the clinical relevance of B cells and tertiary lymphoid structures (TLS) in neuroblastoma (NB), a pediatric tumor with a heterogeneous immune landscape. Methods We analyzed 87 treatment-naïve NB specimens, spanning both localized and metastatic disease previously characterized for T-cell and dendritic cell (DC) infiltration. B cells were detected by immunohistochemistry, and plasma cells were quantified using multiple immunofluorescence. Spatial organization and functional status of immune cells within TLSs were assessed by imaging mass cytometry using a 29-antibody panel. In parallel, gene expression profiles were obtained through NanoString PanCancer Immune Profiling and further validated using publicly available bulk and single-cell RNA-sequencing data from untreated and treated NB samples. These transcriptomic datasets were used to support protein-level findings and to identify prognostic gene signatures. Results B-cell infiltration in NB tumors strongly correlated with the presence of T cells and DCs at both protein and transcriptomic levels, and was associated with improved prognosis. Similar to other solid tumors, B cells in NB were either scattered throughout the tumor or organized into TLSs of varying maturity. Spatial proteomic and transcriptomic analyses revealed that localized tumors often contain mature TLSs, with functional B cells able to antigen presentation and immunoglobulin expression, alongside high cytotoxic T cells. In contrast, metastatic tumors primarily exhibited immature TLSs, with evidence of B-cell and T-cell dysfunction. Importantly, we identified gene signatures associated with B cells and TLSs that not only predicted survival in NB but were also prognostic in multiple adult cancers. Conclusions Our findings highlight a central role for B cells and TLSs in shaping the immune microenvironment of NB. Their presence and maturation status are linked to clinical outcome, suggesting their potential as prognostic biomarkers and targets for novel immunotherapeutic strategies in pediatric oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dadde0b86ab40cec87507bbdba192005334b0d" target='_blank'>
              Mapping B cells and the immune landscape of tertiary lymphoid structures reveals their clinical impact in neuroblastoma
              </a>
            </td>
          <td>
            O. Melaiu, M. Chierici, Paula Gragera, Nicolò Lazzaro, L. L. Petrilli, J. Wienke, Francisca J. Bergsma, B. M. Verhoeven, C. De Stefanis, Valentina D’Oria, M. C. Benedetti, Giovanni Barillari, R. Alaggio, M. A. De Ioris, M. Vinci, Ninib Baryawno, R. Carsetti, G. Jurman, Jan J. Molenaar, Franco Locatelli, D. Fruci
          </td>
          <td>2025-11-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c14f73e77a88fcb7e1149959d53563fad60dc97" target='_blank'>
              Liquid Crystal-like self-organization of Glioblastoma Prevents cell density induced migratory arrest
              </a>
            </td>
          <td>
            Urszula Hohmann, Chalid Ghadban, J. Prell, Christian Strauss, Faramarz Dehghani, T. Hohmann
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Renal cell carcinoma (RCC) patients exhibit variable responses to immune checkpoint blockade therapies. Identifying reliable biomarkers to predict response remains a critical challenge. Tumor-infiltrating T cells (TILs) are implicated in immunotherapy outcomes, but their phenotypic diversity and spatial distribution within the tumor microenvironment are poorly understood. In this study, we performed multi-omic profiling of matched tumor, adjacent normal kidney, and peripheral blood samples with high-resolution spatial transcriptomics was conducted on formalin-fixed paraffin-embedded samples. We identified distinct subsets of CD4+CD8+ double-positive (DP) and CD4-CD8- double-negative (DN) T cells, each with unique gene expression signatures and TCR repertoires. DP T cells exhibited an exhausted phenotype, marked by co-inhibitory receptor expression and clonal TCR expansion, suggesting chronic antigen exposure. DN T cells displayed a heterogeneous profile, expressing both cytotoxic and regulatory molecules, alongside with diverse TCR repertoires, indicating potential tumor-specific reactivity. Spatial transcriptomics revealed distinct localization patterns of DP and DN T cells within the tumor microenvironment. This study provides a comprehensive analysis of TILs in RCC, highlighting the heterogeneity and functional specialization of DP and DN T cells.



 Supported by Sungkyunkwan University Start-up Fund 2024-1239-000-1; National Research Foundation of Korea RS-2024-00405650



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9479919a0539b9bff5772f4839a4d6f9ebb91a8" target='_blank'>
              Single-cell landscape of tumor-infiltrating T cells in renal cell carcinoma reveals novel insights into double-positive/double-negative T cells 2441
              </a>
            </td>
          <td>
            Jaeyoun Park, Jae Hyun Kim, Heeju Ryu
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains one of the leading causes of cancer-related morbidity and mortality in men worldwide, and therapeutic resistance—particularly to immunotherapy—continues to limit clinical efficacy. Mounting evidence has positioned the immunosuppressive tumor microenvironment (TME) as a core driver of disease progression and a formidable barrier to effective immune-based interventions. In this review, we present a comprehensive and multi-dimensional analysis of the cellular, stromal, and molecular constituents of the immunosuppressive TME in PCa, highlighting its significant heterogeneity and context-dependent functions. We emphasize recent breakthrough insights obtained through single-cell RNA sequencing (scRNA-seq), spatial multi-omics, and high-dimensional imaging technologies, which are redefining our understanding of tumor–immune–stromal interactions. Based on these mechanistic findings, we examine precision therapeutic strategies aimed at remodeling the TME, including combinatorial immune checkpoint blockade, metabolic reprogramming, cytokine network regulation, and advanced nanomedicine-based delivery systems. Finally, we discuss translational opportunities and future research directions, underscoring the necessity of integrating advanced omics technologies with biomarker-driven clinical trial design to enable individualized, precision interventions and improve survival outcomes for patients with PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce05a3966abc2d6663f2f9fc06215e6d78446ea" target='_blank'>
              Tumor microenvironment-mediated immune evasion and resistance in prostate cancer: mechanisms, cross-talk, and therapeutic opportunities
              </a>
            </td>
          <td>
            Mingfeng Li, L. Kwantwi, Cuimei Wang, Qin Xiao
          </td>
          <td>2025-11-26</td>
          <td>Clinical and Experimental Medicine</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="
 Tumor progression is driven not only by the complex milieu of transcriptional cell states of distinct lineages that populate the tumor microenvironment (TME) but also by the interactions these states forge with one another. Systematically charting the landscape of such prognostic cell-state interactions (CSIs) remains difficult as single-cell datasets are small and rarely linked to clinical outcomes, whereas bulk profiles obscure cellular context. We developed CSI-TME, a computational pipeline that adapts the concept of synthetic lethality to infer prognostic cell-state interactions by directly utilizing the large cohorts of bulk transcriptomic datasets. We applied CSI-TME to IDH-mutant gliomas, deriving a highly reproducible cell-state interaction network (CSIN) that is predominantly pro-tumor and differentially activated in IDH-mut astrocytoma versus oligodendroglioma. Malignant cells within the CSIN resemble multiple neuronal lineages including, astrocyte-like and oligodendrocyte-progenitor-like program, and reveal key interactions between glioma stem cells and T cells. CSIN stratifies patient response to immune-checkpoint blockade therapy. Roughly 20% of interactions involve direct ligand-receptor communication, and their participating cell states significantly co-localize in spatial-transcriptomic datasets, most notably for a pro-tumorigenic interaction between tip-like endothelial cells and hypoxic malignant cells involving multiple ligand-receptor interactions. Integration with somatic-mutation profiles reveals that many interactions in CSIN exert anti-tumor effect early-on in glioma but shift toward pro-tumor roles as the disease progresses. Taken together, CSI-TME provides a unique way of identifying the prognostic and clinically relevant cell state interactions without the need for extensive single cell datasets and provides a framework for prioritizing therapeutic ligand-receptor targets, patient stratification, while providing new insight into how interactions among the cell states shape the TME in IDH-mutant glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3146cb53ad109ab0b9473e62c6d93d13f03288e3" target='_blank'>
              TMIC-77. Identification of Clinically Relevant Cell State Interactions in the Tumor Microenvironment of IDH-Mut Glioma
              </a>
            </td>
          <td>
            Arashdeep Singh, Bharati Mehani, Vishaka Gopalan, Ken Aldape, Sridhar Hannenhalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive and lethal adult brain tumor. The cellular heterogeneity within the tumor microenvironment (TME) plays a critical role in the complexity of treatment and poor survival. GBM is typically classified into 3 molecular subtypes—Classical, Mesenchymal, and Proneural—associated with EGFR, NF1, and PDGFRA genetic drivers, respectively. Our team previously generated human GBM-specific driver mutations using RCAS/tv-a gene transfer system to create immunocompetent genetically engineered mouse models (GEMMs) of murine GBM (mGBM) that share the same cell-of-origin. While we and others have shown that these GEMMs closely mimic myeloid cell composition and bulk expression profiles observed in human GBM (hGBM) subtypes, it is critical to identify species-specific similarities and differences in tumor microenvironment (TME) heterogeneity at cellular resolution. Hence, we reasoned that single-cell characterization of mGBM models and comparison to hGBM would highlight the cross-species similarities/differences and enable investigators to identify and validate the most relevant human target candidates. Towards this goal, we performed single-cell RNA sequencing (scRNA-seq), multicolor flow cytometry, and in-depth computational analysis on EGFR-, Nf1-, and PDGFB-driven GEMMs to dissect the effect of common GBM genetic drivers on the TME. To assess the human relevance, we compared our findings to bulk RNA-seq data from human GBM tumors with mutually exclusive EGFRvIII deletion, NF1 loss of function mutation, or PDGFRA amplification on chromosome 4. Our analysis revealed that PDGFB-driven tumors were more proliferative and enriched for Wnt signaling interactions, while EGFRvIII-driven tumors showed an elevated interferon signaling response. Moreover, Nf1-silenced tumors displayed higher myeloid abundance, myeloid immunosuppressive interactions involving Osteopontin, Treg infiltration, and expression of immune checkpoint molecule Ctla4. Overall, we established a human-mouse analytical platform for genotype-aware target discovery and validation, which offers promising new avenues for more effective, personalized treatments in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edea93ae20a51718ffae751a4a4ae63a990d752e" target='_blank'>
              TMIC-24. Single-cell decoding of the genotype-immunophenotype relationship in glioblastoma
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, D. Hambardzumyan, A. Tsankov
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Cancers are characterized by genomic instability events such as aneuploidy, chromosomal arm imbalances, and segmental copy number changes. These genomic features frequently define different subclones within a tumor. Single-cell DNA sequencing (scDNA-seq) identifies these large-scale genomic alterations that define subclonal features. However, scDNA-seq does not provide biological phenotypic information on individual subclones. Single-cell RNA sequencing (scRNA-seq) offers biological information but is less accurate in discovering genomic instability events. We developed a computational framework, scAlign, for integrating scRNA-seq and scDNA-seq from the same specimen and define subclonal cellular phenotypes at the resolution of individual cells. Subclones were defined by aneuploidy and chromosomal arm imbalance among primary and metastatic cancers. Using the cells in the G0/G1 phase, the extensive cellular sampling from both assays characterized the subclonal architecture of these cancers. The scDNA-seq provided a ground truth for copy number-based subclones. From the scRNA-seq data, the epithelial cells in G0/G1 were identified and assigned to specific subclones by the scAlign based on gene dosage. Afterward, we determined the differential gene expression and biological pathway activities of specific clones. Overall, integrative multi-omics analysis of single-cell datasets is more informative than any individual genomic modality, provides deep insights into intratumoral heterogeneity and reveals subclonal biology. scAlign is available at https://github.com/XQBai/scAlign.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91a097601e22d4ca33e4ce8206425906a9876da" target='_blank'>
              Single-cell aneuploidy and chromosomal arm imbalances define subclones with divergent transcriptomic phenotypes
              </a>
            </td>
          <td>
            Xiangqi Bai, Billy T. Lau, A. Sathe, Susan M. Grimes, Alison Almeda-Nostine, Hanlee P. Ji
          </td>
          <td>2025-10-31</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is an incurable and aggressive brain cancer marked by profound intra-tumoral heterogeneity. Malignant cells exist in four core transcriptional states: stem-like (OPC-like and NPC-like) and more differentiated astrocytic (AC-like) or mesenchymal (MES-like) states. Using spatial transcriptomics, we previously identified a layered tumor structure of these states, with hypoxia emerging as an organizing driver, implicating oxygen availability as a central factor in cell state dynamics. However, how these cellular states interact with each other and with the tumor microenvironment remains incompletely understood.



 To uncover state-specific vulnerabilities for reducing GBM cell state diversity, we conducted a small-molecule screen in gliomasphere models using over 1,600 cysteine-reactive covalent inhibitors, assessing their effects on cell states via RNA-seq. We then employed the Sonar metabolic reporter to visualize NAD+ and NADH levels in gliomaspheres. Through experiments with orthotopic xenografts in mice and co-culture with human cortical organoids, we characterized distinct redox profiles across GBM cell states by single-cell and bulk RNA-seq. Spatial single-cell transcriptomics and mass spectrometry imaging (MSI) metabolomics on adjacent sections of frozen patient samples were used to identify preferential metabolic programs in specific cell states.



 Our small-molecule screen identified compounds that promote differentiation and are enriched for NRF2 activation, a master regulator of the antioxidant response. Spatial single-cell transcriptomics and mass spectrometry metabolomics experiments confirmed that MES-like cells exhibit a heightened glycolytic signature, whereas stem-like states preferentially depend on oxidative phosphorylation (OXPHOS) for their energetic needs. Finally, pharmacologic perturbations revealed that mitochondrial and glycolysis inhibition selectively deplete different cell state populations in gliomaspheres.



 Our findings provide deeper insight into the metabolic underpinnings of GBM cell states and support the rationale for combinatorial therapies aimed at reducing intratumoral heterogeneity that exploit their divergent dependencies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e25475481794c097c1d8706111cced1dc0e56d4b" target='_blank'>
              TMET-30. Uncovering the metabolic programs underlying malignant cell state heterogeneity in glioblastoma
              </a>
            </td>
          <td>
            Alexander Jucht, Maolin Ge, Rony Chanoch-Myers, Florian Ruiz, Gerard Baquer, Alissa C. Greenwald, Min Yang, Toshiro Hara, Yilin Fan, Channing Pooley, D. Gerovasilis, Charles Couturier, Nathalie Y R Agar, I. Tirosh, L. Bar-Peled, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Molecular heterogeneity in head and neck squamous cell carcinoma (HNSCC) is well recognized, yet existing subtype frameworks remain largely descriptive and have not translated into therapeutic decision-making. Here, we establish a mechanistic platform that converts transcriptomic diversity into drug-actionable tumor states. Integrating multi-cohort RNA-seq from 727 tumors across five independent datasets, genome-scale CRISPR dependency maps, and pharmacologic screening, we define distinct tumor survival circuits across HPV-negative HNSCC and nominate subtype-matched therapeutic strategies. These circuits encompass a proliferative axis (MYC, MET/FAK, inflammatory and translational programs), an epithelial-differentiated/adhesion program, an EMT-like state with stromal activation, and mitochondrial/oxidative metabolic states, each mapping to selective liabilities (e.g., mitotic/autophagy control, ERBB/PI3K and cadherin signaling, OXPHOS/mitochondrial translation, and G2/M-integrin-Notch pathways, respectively). We then develop a transcriptomic predictor of EGFR-inhibitor response using machine learning and validate it in prospectively collected, fresh patient-derived 3D microtumors. The resulting 13-gene signature identifies erlotinib-responsive tumors (R = 0.93) and maps biologically to an epithelial-differentiated state, outperforming EGFR expression alone. Our study establishes a subtype-to-dependency-to-therapy framework, enabling precision stratification and providing a clinically feasible path for prospective biomarker deployment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29377070a0d6fb125a75d4b7cf89275cb55faab" target='_blank'>
              Subtype-Specific Dependencies and Drug Vulnerabilities Enable Precision Therapeutics in Head and Neck Cancer
              </a>
            </td>
          <td>
            Joel M Vaz, Songli Zhu, Mateo Useche, Lauren Shih, Emily Marchiano, Slobodan Beronja, Bruce E Clurman, Cristina Rodriguez, Brittany Barber, Marina Chan, Taranjit S. Gujral
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdc2f4fd014e0601edcc3f78406fd8afaf2448ea" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors.
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, D. Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson A. Weir, O. Ashenberg, Geraldine S Pinkus, Scott J. Rodig, Caroline Uhler, Evan Z. Macosko, Margaret Shipp, A. Louissaint, Fei Chen, Todd R Golub
          </td>
          <td>2025-12-20</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4e23f0e2eb9b451cd70e81ca9b082a2f8b6469" target='_blank'>
              Advancements in single-cell sequencing for cervical cancer research.
              </a>
            </td>
          <td>
            Congli Pu, Biyuan Xing, Shujie Wang, Zhao Liu, Yingchao Zhao
          </td>
          <td>2025-11-06</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d59ad194fb04ebf384288815f79948f2c11cf9f" target='_blank'>
              Humanized glioblastoma patient-derived orthotopic xenografts recreate a locally immunosuppressed human immune ecosystem amenable to immunotherapeutic modulation
              </a>
            </td>
          <td>
            P. M. Moreno-Sanchez, A. Oudin, Batuhan Kısakol, Heiko Dussmann, E. Klein, V. Baus, Camille Rolin, Carole Seguin-Devaux, Mahsa Rezaeipour, Aurélie Poli, A. Michelucci, J. Paggetti, E. Moussay, Jochen H. M. Prehn, Simone P. Niclou, A. Gołȩbiewska
          </td>
          <td>2025-11-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Low-grade gliomas (LGGs) exhibit low overall mutational burden and an immunosuppressive microenvironment, contributing to immunotherapy resistance. Intratumoral heterogeneity (ITH) further complicates the targeting of tumor-specific antigens (TSAs), yet its influence on the antigenic and immune landscapes in LGG remains understudied, particularly due to reliance on single-biopsy analyses.



 We performed exome and transcriptome sequencing on 70 spatially mapped biopsies from seven IDH-mutant Grade II astrocytoma patients. Tumor purity, immune deconvolution, and neoantigen prediction were conducted to assess regional immune variation and antigenic burden. In one patient (P375), neoantigen-specific CD8+ T cells were isolated using barcoded peptide-HLA multimers. Reactive T-cell receptor (TCR) clonotypes were identified via single-cell TCR sequencing and functionally validated in Jurkat76 and CD8+ T cells.



 Spatial profiling of whole tumors revealed most mutations were biopsy-specific, with a steep drop in mutations shared across multiple regions. RNA-seq–based unsupervised clustering with xCell and DESeq2 revealed two distinct immune microenvironments: immune-hot regions with elevated immune infiltration, and immune-cold regions with minimal immune presence. Notably, mutation-derived n-mer peptides from immune-hot regions showed enhanced predicted immunogenicity relative to those from immune-cold areas. In one patient (P375), neoantigen-specific CD8⁺ TCR clonotypes were identified targeting PRMT5 mutations found in immune-hot regions. A mutant PRMT5-specific TCR demonstrated robust, antigen-dependent activation and dose-dependent cytotoxicity in vitro. This TCR showed high specificity toward the mutant peptide with minimal cross-reactivity to the wild-type sequence. Furthermore, engineered CD8⁺ T cells effectively killed glioma cells endogenously expressing the full-length mutant PRMT5, supporting its candidacy as a viable immunotherapeutic target.



 Our findings reveal significant 3D spatial and immunologic heterogeneity in low-grade gliomas and demonstrate that neoantigens such as mutant PRMT5 can be selectively targeted by highly specific, cytotoxic TCR-engineered CD8⁺ T cells. These results support the potential of personalized TCR-based immunotherapy for low-grade glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fa3c48068f63d55f21f844232ecea3f66c8ffb8" target='_blank'>
              EXTH-52. Spatially Resolved Antigenic Landscapes In Low-grade Glioma Reveal Targets For Personalized Immunotherapy
              </a>
            </td>
          <td>
            Darwin W. Kwok, Michael Zhang, Cliff Wang, Maggie Colton Cove, Chibo Hong, Nicholas O. Stevers, Benjamin Lerman, S. Lakshmanachetty, Tyler Borrman, Zheng Pan, Benjamin Yuen, Songming Peng, Diana Nguyen, Michael Martin, S. Hilz, Joanna Phillips, Anny Shai, N. Bush, S. Hervey-Jumper, Michael McDermott, S. Mandl, H. Okada, Joseph F Costello
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor fibrosis is recognized as a malignant hallmark in various solid tumors; however, the clinical importance and associated molecular characteristics of tumor fibrosis in liver metastases (LM) from colorectal cancer (CRLM) remain poorly understood. Here we show that patients with CRLM whose liver metastases (LM) exhibited tumor fibrosis (Fibrosis+ LM) had significantly worse progression-free survival (P = 0.025) and overall survival (P = 0.008). Single-cell RNA sequencing revealed that the tumor microenvironment of the Fibrosis+ LM was characterized by T cells with an exhausted phenotype, macrophages displaying a profibrotic and suppressive phenotype and fibrosis-promoting fibroblasts. Further investigation highlighted the pivotal role of VCAN_eCAF in remodeling the tumor fibrosis in the tumor microenvironment of Fibrosis+ LM, emphasizing potential targetable interactions such as FGF23 or FGF3-FGFR1. Validation through multiplex immunohistochemistry/immunofluorescence and spatial transcriptomics supported these findings. Here we present a comprehensive single-cell atlas of tumor fibrosis in LM, revealing the intricate multicellular environment and molecular features associated with it. These insights deepen our understanding of tumor fibrosis mechanisms and inform improved clinical diagnosis and treatment strategies. The liver is a common site for cancer spread, especially from colorectal cancer. This study looks at how fibrosis in liver metastases affects treatment and survival. Researchers studied 471 patients with liver metastases from colorectal cancer. They examined tissue samples to see how fibrosis impacts survival and used advanced techniques such as single-cell RNA sequencing to understand the tumor environment better. They found that patients with more fibrosis had worse survival rates. The study also revealed that fibrosis changes the tumor environment, making it harder for the immune system to fight cancer. This includes an increase in certain immune cells that suppress the body’s defense against tumors. The researchers suggest that targeting fibrosis could improve treatment outcomes. In the future, therapies that reduce fibrosis might help enhance the effectiveness of cancer treatments and improve patient survival. This summary was initially drafted using artificial intelligence, then revised and fact-checked by the author.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253b52aabc9f910e8e84f8a906624aeebc79b3e8" target='_blank'>
              Single-cell transcriptomic profiling reveals liver fibrosis in colorectal cancer liver metastasis
              </a>
            </td>
          <td>
            Yiqiao Deng, Chengyao Guo, Xiaomeng Liu, Xin Li, Jianmei Liu, Wenjie Liu, Jinghua Chen, Zhen Huang, Ye‐fan Zhang, Xinyu Bi, Jian‐jun Zhao, Jianguo Zhou, Zhi-yu Li, Hongliang Wu, Baocai Xing, Qichen Chen, Hong Zhao
          </td>
          <td>2025-11-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables simultaneous mapping of tissue morphology and spatially resolved gene expression, offering unique opportunities to study tumor microenvironment heterogeneity. Here, we introduce a computational framework that predicts spatial pathway activity directly from hematoxylin-and-eosin-stained histology images at microscale resolution 55 and 100 um. Using image features derived from a computational pathology foundation model, we found that TGFb signaling was the most accurately predicted pathway across three independent breast and lung cancer ST datasets. In 87-88% of reliably predicted cases, the resulting spatial TGFb activity maps reflected the expected contrast between tumor and adjacent non-tumor regions, consistent with the known role of TGFb in regulating interactions within the tumor microenvironment. Notably, linear and nonlinear predictive models performed similarly, suggesting that image features may relate to pathway activity in a predominantly linear fashion or that nonlinear structure is small relative to measurement noise. These findings demonstrate that features extracted from routine histopathology may recover spatially coherent and biologically interpretable pathway patterns, offering a scalable strategy for integrating image-based inference with ST information in tumor microenvironment studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29b8964f22d869e7cd99fbb783574fbfe821242b" target='_blank'>
              Digital Modeling of Spatial Pathway Activity from Histology Reveals Tumor Microenvironment Heterogeneity
              </a>
            </td>
          <td>
            Ling Liao, Changhuei Yang, Maxim Artyomov, Mark Watson, Adam Kepecs, Haowen Zhou, Alexey Sergushichev, R. Cote
          </td>
          <td>2025-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72005e917b50c2e5c4b78fb42f30196a93fe8580" target='_blank'>
              Transcriptional Profiling of Commonly Used Liver Cancer Cell Lines Reveals Disease-Specific Modeling Potential and Authentication Concerns
              </a>
            </td>
          <td>
            Nima Sarfaraz, Gail Rosen, Trevor Winkler, Risha Patel, Srinivas Somarowthu, Michael Bouchard
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Although non-muscle-invasive bladder cancer (NMIBC) frequently recurs and progresses despite BCG therapy, cellular mechanisms behind this remain unclear. Understanding the dynamics within the tumor microenvironment (TME) and identifying pathways associated with BCG resistance are crucial for improving NMIBC treatment. To elucidate transcriptional dynamics and key cellular interactions in the TME, we performed longitudinal single-nucleus RNA sequencing of 9 NMIBC cases with three pairs of treatment-naïve and disease progression samples. The resulting 58,037 nuclei were subject to cell type and subtyping annotations along with cell–cell interaction analyses. Our analysis revealed marked interpatient heterogeneity in malignant cells, accompanied by copy number alterations associated with disease progression. TGF-β signaling emerged as a key pathway, showing gradual enrichment from pre-treatment to post-progression samples. Within the tumor microenvironment, we identified distinct subtypes including LAMP3 + dendritic cells and iCAF fibroblasts that followed unique pseudotime trajectories linked to progression. Cell–cell interaction analysis further highlighted critical ligand–receptor pairs such as DSC2–DSG2 and ENG–BMPR2, which were associated with poor clinical outcomes. These interactions may represent hypothesis-generating candidates that warrant further validation. This study highlights key cellular and molecular mechanisms underlying bladder cancer progression Post-BCG treatment. Through single-nucleus RNA sequencing, we identified critical TME subtypes, TGF-β signaling involvement, and crucial ligand-receptor interactions such as DSC2-DSG2 and ENG-BMPR2, which may serve as preliminary biomarkers requiring confirmation in larger independent cohorts. These findings offer valuable insights into enhancement of treatment strategies and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b994449406e351724039c2b4ed65a466872b5c" target='_blank'>
              TGF-β signaling and tumor microenvironment dynamics in bladder cancer progression post-BCG therapy: a longitudinal single-nucleus RNA-seq study
              </a>
            </td>
          <td>
            Seo-Young Lee, Yun-Hee Lee, Tae-Min Kim, U-Syn Ha
          </td>
          <td>2025-11-10</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Classic Hodgkin lymphoma (CHL) is unique among most malignancies as the malignant Hodgkin and Reed–Sternberg (HRS) cells (0.1-5%) are vastly outnumbered by a heterogeneous population of reactive, non-neoplastic cells within the tumor-microenvironment (TME). CHL exhibits a bimodal age distribution, with peaks in adolescents and young adults (AYAs) aged 15-39 years and older adults (over 50 years). Although a previous bulk gene expression-based study (Johnston et al., Blood 2022) suggested differences in the TME transcriptional profiles between pediatric CHL and adult CHL, the study did not take into account for key factors such as cellular interactions, specific expression features of HRS cells, or the spatial architecture of the TME. A deeper understanding of the age-related TME ecosystem is essential for advancing our knowledge of the unique pathogenesis of pediatric CHL and for developing biomarker-driven targeted therapies. In this study, we aim to elucidate distinct TME ecosystems in pediatric CHL by applying single-cell and spatial technology.



 We performed single-nuclei RNA sequencing (snRNA-seq) on formalin-fixed, paraffin-embedded (FFPE) tissue samples from a total of 27 patients, including 11 adult (19-39 years) and 8 pediatric HL patients (10-16 years) with Epstein-Barr virus (EBV)-negative nodular sclerosis HL, as well as 4 adult and 4 pediatric reactive lymph nodes (RLN) serving as normal controls. We merged the expression data from all cells and used the louvain clustering algorithm to identify major cell types and functional cellular subsets and defined each immune cell population based on marker expression. We also performed spatial transcriptomics using the CosMx™ Spatial Molecular Imager with the human 6K discovery panel, applied to tissue microarray (TMA) from 75 pediatric CHL patients (3-18 years) enrolled in the Children's Oncology Group (COG) AHOD0031 trial. We utilized transcriptomic signatures of cell types identified by snRNA-seq to annotate major cell types in CosMx data by a label transfer approach. For each cell type in the TME, we calculated a ‘spatial score’ (Aoki et al., J Clin Oncol 2024), spatial cell enrichment score of a given cell type as the distance to the five nearest neighbor cells, capped at the spatial interaction range (50µm).



 We analyzed 119,335 cell transcriptomes after quality control filtering in the snRNA-seq data. Unsupervised clustering revealed 20 phenotypically distinct clusters. When comparing age-related distributions of immune cell phenotypes, CD8+ T cells, NK cells, naïve T cells, CD4+ T cells and B cells were significantly more predominant in pediatric CHL compared to adult CHL (p < 0.001). In contrast, myeloid cells including monocytes, macrophages, and dendritic cells, were more enriched in adult CHL than in pediatric CHL (p < 0.001). In particular, we identified a subcluster, myeloid-C2, characterized by high CXCL13and CD68expression, which represent the most distinct population in adult CHL.
 Among the immune populations enriched in the TME of pediatric CHL, we further investigated the CD8+ T cell subsets, as the role of CD8+ T cells in CHL remains incompletely understood. We first performed differential gene expression analysis between cells from pediatric and adult CHL samples within the CD8+ T cell cluster, identifying CCL5 as the one of the most up-regulated genes in pediatric CHL. We further identified a pediatric HL enriched subcluster, CD8-C2, characterized by high CCL5 and LAG3 expression. The CD8-C2 cluster exhibited high expression of cytotoxicity, IFN response, and exhaustion signatures. Notably, the cell-to-cell communication tool, Cell-Chat, revealed a significant interaction between the CCL5+ (CD8-C2) and CCR4(HRS cells) axis (p<0.001). CCR4+ HRS cells were significantly more enriched in pediatric CHL compared to adult CHL (p < 0.001).
 To validate these findings, we analyzed CosMx data comprising 557,742 cells. By calculating a spatial score, we confirmed that CCR4+ HRS cells were significantly more proximal to CCL5+ CD8+T (CD8-C2) cells (p=0.02), but not to CCL5- CD8+T cells.



 Multimodal transcriptional and spatial profiling reveals a distinct TME in pediatric CHL. We identified the interaction of CCR4+ HRS cells with ligand-expressing CCL5+ CD8+ T cells as a prominent crosstalk axis in pediatric CHL, with potential Iimplications for novel therapeutic approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f7d22fa537349a2f45b4bca8f24230406a0d3b4" target='_blank'>
              Multimodal single-cell and spatial analyses reveal a distinct immune ecosystem in pediatric classic Hodgkin lymphoma
              </a>
            </td>
          <td>
            Shinichiro Oshima, Yifan Yin, N. Dakhallah, Rose Chami, Bo Ngan, A. Punnett, Hang Yin, Isabella Kong, Shinya Rai, A. Telenius, R. Kridel, Federico Gaiti, David W Scott, Kara Kelly, Debra Friedman, S. Castellino, L. Roth, Christian Steidl, T. Aoki
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract Aim Glioblastoma (GBM) exhibits pronounced cellular plasticity that contributes to therapeutic resistance. A facet of this adaptability is neuronal mimicry, whereby tumor cells acquire neuron-like features to engage with neural circuits. The coherence and biological significance of this process remain uncertain. Methods Spatial transcriptomic and proteomic datasets from Greenwald et al. [1] were analyzed alongside healthy cortical tissue from Ravi et al. [2] as a non-malignant reference. A permutation-based framework identified tumor–neuronal (TN) cells across Healthy, Core, Infiltrative, and Bulk regions, with stability verified by resampling and Jaccard overlap (CV < 10%). Results TN cells were enriched in Bulk (OR ≈ 70, p < 10−10) and Infiltrative (OR ≈ 7, p < 10−10) regions relative to Healthy. Integrative analyses revealed a synaptic-like transcriptional program marked by SNAP25, SYT1, and STMN2 (logFC ≈ 1–1.5), enriched for vesicle cycling and cytoskeletal remodeling pathways. Neuronal subtype markers (SLC17A7, GABRA1) were depleted, while malignant markers (CLU, GFAP, VIM) persisted. Single-cell proteomics confirmed co-localization of neuronal and tumor proteins within clustered, double-positive cells. Conclusion This study delineates a reproducible hybrid state with partial neuronal characteristics, suggesting that neuronal mimicry represents an adaptive, rather than lineage-converted, trajectory of GBM that may inform strategies to disrupt tumor–neuron crosstalk. References 1. Greenwald AC, et al. ‘Integrative spatial analysis reveals a multi-layered organization of glioblastoma.’ Cell 2024;187(10):2485–2501.e26. 2. Ravi V, Will P, Kueckelhaus J, et al. ‘Spatially resolved multi-omics deciphers bidirectional tumor-host interdependence in glioblastoma.’ Dryad 2025;10.5061/dryad.h70rxwdmj.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1722c03dee89d4877aa24eca4527f645a5b89d83" target='_blank'>
              Spatial multi-omics characterizes neuronal mimicry as a coherent hybrid state in glioblastoma
              </a>
            </td>
          <td>
            Claire Yuan, Jake Y Chen
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Despite the remarkable success of targeted therapy, the emergence of acquired resistance remains a major obstacle to cure acute myeloid leukemia (AML) patients. While existing studies have predominantly focused on genetic heterogeneity as a driver for therapy resistance, recent evidence suggests that non-genetic persister state is equally important, but such studies are lacking in AML. Here, we define the temporal transcriptome and chromatin landscape along with lineage trajectory of individual genetically identical AML persister clones to existing anti-AML therapeutics.
 To model persister state in AML, we utilized watermelon library (Oren et al., 2021), that expresses lineage barcode with inducible Histone-2B-mCherry to trace clonal evolution and cell division. We first evaluated whether slow-cycling, quiescent persisters emerge in AML lines after exposure to standard therapy. Our data showed the emergence of slow cycling persisters with cytarabine (AraC) in all 4 cell lines, and gilteritinib (GLT) in MOLM-13 and MV4-11 cells while BH3 mimetics (venetoclax (VEN), S63845 and navitoclax) had fast-cycling persisters. Integrating proliferative-index profiles with cell cycle profiles depicted extensive heterogeneity in AML persisters imposed by drug target rather than genetic state. Drug holiday experiments exhibited a reversible drug tolerance state in both slow and fast-cycling persisters. Longitudinal drug sensitivity and BH3 profiling assays showed that persisters emerged with adaptation to reduced mitochondrial priming that reversed after drug holiday. However, transitioning from persister to stable resistant cells resulted in a sustained loss in apoptotic priming, mirroring observations made in PDX models and primary tumors.
 To understand trajectory of persister emergence and underlying mechanisms, we performed single-cell multiomics on drug-naïve and persister populations (day 9) in MV4-11 (GLT), and OCI-AML2 (VEN and AraC). We expanded 10,000 unique barcoded cells and administered the drug treatment to derive persisters. Using lineage barcode analysis, we report significant reduction in the dominant lineage of day 0 and survival of multiple small lineages indicating an acquired state of persistence than selection of pre-existing resistant clone in both the lines. Further trajectory analysis defined three fates of the lineages as: expanding (log2FC>1), shrinking (log2FC <1) and stable. AraC persisters showed higher expanding and stable lineages, while VEN and GLT persisters resulted in higher shrinking and stable clones, revealing heterogeneity in lineage expansion. We next asked whether persisters resulting from different therapies arised from the same parental lineage. To test this, we performed lineage tracing on VEN and AraC persisters in OCI-AML2 and compared surviving lineages at Day 9 vs Day 0 cells. This analysis identified 52 parental lineages that were able to produce both VEN and AraC persisters in AML2, providing evidence of branched evolution of persisters from the same parental clone.
 We finally defined transcriptional and epigenetic heterogeneity in baseline and persister cells by performing UMAP on single-cell RNAseq and ATACseq data. We identified persisters to all 3 drugs clusters distinctly from their drug naïve counterparts at both transcriptome and chromatin accessibility. Next, we investigated differentially expressed pathways in persisters and observed higher inflammatory response pathways such as TNFα signaling in GLT, enrichment of MYC targets and oxidative phosphorylation in VEN and high mTORC1 signalling in AraC persisters. We observed higher chromatin accessibility in GLT and VEN whereas condensed chromatin in AraC persisters. We next asked which motifs are highly enriched in each persister and observed the enrichment of various KLF and SP family of motifs in each persister driving multiple gene signatures in each persister type. We also observed enrichment of LSC-primed and cDC like cells in GLT persisters but with AraC and VEN persisters a uniform pattern of all states was observed, indicated stemness may not be common mechanism to persister evolution.
 Collectively, our integrated single-cell multiomics and lineage tracing approach reveals that AML persisters emerge through stochastic drug-specific, non-genetic adaptive mechanisms characterized by distinct transcriptional and chromatin remodeling programs rather than deterministic Darwinian selection of pre-existing resistant cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b912c83b14f52bda36c8d81fafb30ac8e86a906c" target='_blank'>
              Dissection of non-genetic resistance mechanisms by lineage tracing and single cell multiomics reveals drug-specific persister programs in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            K. Bhatia, Xiao Xian Lin, Ankit Jana, Emanuelle I. Grody, Ryan Jeremy Lim Jing Wei, Shawn Koh Jia Jun, Elisha Teh Jia Yu, Cin thet Kyi, Prasad Iyer, Greg Kellogg, Yogesh Goyal, Lawrence Boise, Yaara Oren, Shruti Bhatt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Advances in our understanding of the tumor microenvironment (TME) will help inform the development of next generation immunotherapies and engineered immune-cell therapies. Spatially characterizing immune cells with cell marker protein and activation marker RNA is crucial. To address this, we have developed a high throughput RNAscope™ assay on the BOND RX for same slide RNA+protein detection. We utilized the new protease-free RNAscope™ Multiomic LS assay that can detect up to 6 RNA and protein targets on the same slide with TSA-based amplification strategy that offers a strong signal boost. Here, we demonstrate the use of our pre-conjugated antibody panel and a some unconjugated primary antibodies in multiple tumor tissues. Infiltrating Cytotoxic T cell lymphocytes were detected using CD8, GZMB and IFNG co-expression. Regulatory T cells were detected using CD4, FOXP3 co-expression. PanCK was used as a tumor marker to delineate tumor from stroma. Similarly, tumor associated macrophages were detected using CD163, CD68, IL10 and IL-1B expression. We were also able to identify distinct M1 and M2 macrophage populations in the tumor samples. The RNAscope Multiomic LS assay was successfully able to interrogate different subpopulation of immune cells with high quality RNA and protein signal while maintaining tissue morphology. The high throughput capability with strong signal boosting technology makes this assay an ideal tool for identifying diagnostic and therapeutic biomarkers.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b2afa7ff60daccddf4d93319b5feabb08eb8786" target='_blank'>
              Identifying immune cell phenotypes and their function by using spatial multiomics 3124
              </a>
            </td>
          <td>
            A. Dikshit, Debia Wakhloo, Julia S. Yu, Sonali A Deshpande
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) exhibits early systemic dissemination that precedes clinical detection, challenging the classical model of metastasis as a late evolutionary event. Mounting evidence indicates that molecular and cellular programs enabling invasion and distant colonization emerge at preinvasive or early carcinoma stages. This mini-review synthesizes recent advances defining how tumor-intrinsic mechanisms, such as KRAS-driven basal extrusion, epithelial–mesenchymal plasticity and chromosomal instability, cooperate with microenvironmental cues to promote early metastatic competence. The desmoplastic stroma, rich in fibroblasts, inflammatory mediators and aligned extracellular matrix, provides both structural and biochemical support for tumor-cell escape. Additionally, neural and immune interactions, including chemokine and cytokine signaling, facilitate perineural invasion and systemic pre-metastatic niche (PMN) conditioning via extracellular vesicles and cytokine networks. Recognizing PDAC as a systemic disease from inception reframes therapeutic priorities: early systemic therapy and biomarker-guided patient stratification may intercept occult dissemination. We propose that integrating mechanistic insights on tumor–microenvironment crosstalk with perioperative and liquid-biopsy-driven clinical strategies could redefine early intervention in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92dd387af76159f510d779a0044bf468159245fa" target='_blank'>
              The molecular logic of early metastasis in pancreatic cancer: crosstalk between tumor and microenvironment
              </a>
            </td>
          <td>
            Andrea Costamagna, Noemi Ghiglione, M. Martini
          </td>
          <td>2025-11-28</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Pediatric cancers are often driven by unique alterations suggesting mechanistic ties to underlying developmental programs. Ependymoma (EPN) is an example of a brain cancer subtype driven by gene fusions involving ZFTA-RELA (ZR) that are exclusive to this disease. This leads to our hypothesis that specific chromatin states in developmental lineage programs are at risk of oncogenic transformation, and that these chromatin signatures persist during tumor development and govern tumor cell heterogeneity. To address this question, we constructed an integrated single-nucleus Multiome (snRNA-seq and snATAC-seq) atlas of the developing mouse forebrain and compared it with snMultiome analysis of ZR-driven mouse and human EPN. We identified specific developmental lineage programs present in glial progenitor cells enriched with the Plagl family transcription factors; the precise DNA binding sites of ZFTA fusion oncoproteins. In both mouse and human EPN, ZR expression maintains accessibility of developmental epigenomic programs leading to persistent cell proliferation and tumorigenicity. Cross-species analysis of mouse and human EPN revealed significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that established a putative tumor cell hierarchy. These findings were expanded by applying in vivo lineage barcode tracing methods in mouse EPN, which revealed dominant cell lineages that aggressively expanded and recapitulated the entire tumor cell diversity. Our findings unravel the intersection between developmental and oncogenic epigenomic states critical for ZR driven brain cancer. These findings shape our understanding of the earliest stages of epigenetic programs in brain cancer, and how cancer drivers such as ZR intersect with developmental programs to establish tumor cell heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76bc0f8a27b22de067c5d347cc54b71b6de69b8d" target='_blank'>
              EPCO-12. DOMINANT MALIGNANT LINEAGE LEVERAGE RESTRICTED EPIGENOMIC PROGRAMS TO DRIVE EPENDYMOMA DEVELOPMENT
              </a>
            </td>
          <td>
            Hua Sun, A. Kardian, S. Ippagunta, N. Laboe, Hsiao-Chi Chen, Jiangshan Zhan, Erik Emanus, S. Varadharajan, T. Zheng, B. Holcomb, Jon P. Connelly, Yong-Dong Wang, K. Lowe, Hyun Kyoung Lee, S. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background Gastric cancer progression involves complex interactions among tumor cells, immune components, and stromal elements within the tumor microenvironment. However, a comprehensive understanding of cellular heterogeneity, spatial organization, and cell-cell communication in gastric cancer remains incomplete. Methods Single-cell RNA sequencing was performed on 252, 399 cells from six tissue types, spanning gastritis, intestinal metaplasia, primary tumors, adjacent normal tissue, and metastatic lesions. Integration with spatial transcriptomics enabled spatial mapping of cellular interactions. Pseudotime, cell-cell communication, and transcriptional heterogeneity analyses were conducted. Tumor stage-associated gene modules were identified using Weighted Gene Co-expression Network Analysis (WGCNA) of The Cancer Genome Atlas (TCGA) data. Finally, a deep learning-based prognostic model was developed and externally validated. Results Our analysis revealed dynamic remodeling of the tumor microenvironment during gastric cancer progression, characterized by the expansion of dysfunctional CD8+ T cells, pro-tumorigenic fibroblasts (e.g., ITGBL1+, PI16+, and ITLN1+), and altered myeloid populations. Stromal-immune crosstalk, particularly fibroblast-driven immunosuppressive signaling, was prominent. Spatial transcriptomics revealed the colocalization of immune and stromal cells, supporting spatially organized cellular interactions. WGCNA identified a gene module (657 genes) associated with T cell, myeloid, and stromal alterations, as well as tumor stage. A deep learning model based on this gene set accurately stratified patients according to survival in both TCGA and independent validation cohorts. Risk scores were correlated with clinical features, including tumor stage and therapeutic response. Conclusions Our integrative single-cell, spatial, and computational analysis provides a high-resolution map of gastric cancer microenvironment remodeling. We identified key stromal and immune subpopulations, extensive cellular communication networks, and spatial structures that collectively drive tumor progression and metastasis. The derived gene signature and prognostic model have the potential for clinical risk stratification and therapeutic targeting in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f147ce2dccf2934265aaef857f3e2b638aa838fb" target='_blank'>
              Integrative multi-omics analysis of gastric cancer evolution from precancerous lesions to metastasis identifies a deep learning-based prognostic model
              </a>
            </td>
          <td>
            Yulin Ren, Xiaoyan Zhang, Ke Li, Shuning Xu, L. Qiao, Qun Li, Cheng Zhang, Ying Liu
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11151ef60d8479a0c0f55b160c123009b32bf93" target='_blank'>
              Single-cell transcriptomic analysis reveals the heterogeneity and functional characteristics of macrophage subpopulations in colon cancer.
              </a>
            </td>
          <td>
            Jiazheng Zhao, Baochun Wang, Xiao Li, Chunting Wei, Yi Min, Dayong Wang
          </td>
          <td>2025-11-23</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a powerful technique for understanding cellular diversity, but processing large patient cohorts to identify phenotype-associated cell populations remains challenging. Here, scPER (Estimating cell Proportions using single-cell RNA-seq Reference), a rigorous approach combining adversarial autoencoder and extreme gradient boosting to estimate tumor microenvironment cell compositions and identify phenotype-associated subclusters for bulk RNA-seq samples. Integrating scRNA-seq datasets from diverse studies, scPER constructed comprehensive reference panels and disentangled confounders from true signals. scPER achieved superior accuracy in cellular proportion estimation compared to CIBERSORTx, BayesPrism, Scaden, MuSiC, SCDC, DeSide and ReCIDE. It showed high accuracy in predicting metastatic melanoma immunotherapy response and identified a critical T cell subcluster expressing FCRL3 and SLAMF7. In metastatic urothelial cancer, scPER predicted TGFβ-mediated inhibition of CD4 naïve T cells to diminish PD-L1 checkpoint blockade efficacy. scPER enables robust integration of scRNA-seq datasets to estimate cellular proportions across tumors and identify clinically relevant cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8390e37b5363dcdab22134564264151d0e0926d0" target='_blank'>
              scPER: A Rigorous Computational Approach to Determine Cellular Subtypes in Tumors Aligned With Cancer Phenotypes From Total RNA Sequencing.
              </a>
            </td>
          <td>
            Bingrui Li, Xiaobo Zhou, R. Kalluri
          </td>
          <td>2025-11-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badf7545e441ad3330c93dcdb325456db2158597" target='_blank'>
              Spatial Dynamics of Tumor Cell Plasticity in Lung Adenocarcinoma Revealed by Region-Specific Single Cell Transcriptomics.
              </a>
            </td>
          <td>
            A. Khozyainova, Maxim E. Menyailo, Vera G. Subrakova, Dmitry M. Loos, P. Iamshchikov, Anton A Fedorov, M. S. Tretyakova, Ustinia A Bokova, Evgeny O Rodionov, Sergey V Miller, Evgeny B Topolnitsky, D. V. Piankov, Sergei A Korostelev, Evgeny V. Denisov
          </td>
          <td>2025-12-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Y. Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, R. Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, M. Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="


 Introduction Primary central nervous system lymphoma (PCNSL) is an aggressive B-cell lymphoma exhibiting unique central nervous system (CNS) tropism, and high recurrence rates despite sharing morphological and molecular features with systemic Diffuse Large B-Cell Lymphoma (DLBCL). Given the unique immune landscape of the CNS and the critical role of macrophages in neural tissue, we hypothesized that PCNSL may be sustained by a CNS-specific macrophage program distinct from DLBCL, representing a targetable mechanism underlying immune evasion, CNS confinement, and treatment resistance. While tumor-associated macrophages (TAMs), particularly CD163+ M2-like macrophages, are known to be enriched in PCNSL tumor microenvironment (TME), prior spatial studies have focused predominantly on tumor-intrinsic features and T-cell dysfunction, leaving macrophage organization and functional programming poorly characterized.
 Methods We employed cutting-edge spatial multi-omics using four complementary platforms to comprehensively profile macrophage heterogeneity. Formalin-fixed paraffin-embedded (FFPE) samples in TMA format (26 PCNSL, 89 DLBCL) were analyzed using a Xenium 380-gene immuno-oncology panel. GeoMx digital spatial profiling whole transcriptome analysis (DSP-WTA) was performed in 82 cases (17 PCNSL, 65 DLBCL) using CD3, CD20, and CD68 cell masks. Macrophage signatures identified from DSP were validated across independent single-cell RNA sequencing (scRNA-seq) datasets (PCNSL N=28, DLBCL N=17, reactive lymph nodes N=2), distinguishing microglia from monocyte-derived macrophages. CellScape high-plex imaging was used to confirm phenotypes at the protein level and assess spatial proximity and interactions in 24 PCNSL and 5 tonsil samples using 40 architecture, immune and macrophage markers.
 Results Xenium spatial profiling revealed significantly higher macrophage abundance in PCNSL versus DLBCL, confirmed by DSP-WTA (p=0.0002). DSP further demonstrated that PCNSL TAMs upregulate immunosuppressive genes (CRYAB, SPP1) while downregulating T-cell recruitment genes (CCL19, IGSF6), with enrichment of glycolysis, cholesterol homeostasis, and peroxisome pathways. The CXCL9:SPP1 expression ratio, a validated macrophage polarity biomarker of prognosis in cancer, was significantly reduced in PCNSL macrophages across both DSP and scRNA-seq datasets (p=0.027).
 Of specific interest, differentially expressed gene (DEG) projection and BayesPrism deconvolution of CD68+ regions of interest (ROIs) revealed enrichment of TREM2+ macrophages (p=0.00066) and elevated GPNMB+ lipid-laden macrophage (LLM) signatures (p=0.004) in PCNSL. scRNA-seq confirmed higher LLM signatures in PCNSL versus DLBCL (p=0.0052) and lymph nodes (p=1.2e-06), identifying a monocyte-derived subset enriched in cholesterol metabolism and high-density lipoprotein (HDL) binding pathways. Cell-cell communication analysis revealed enhanced interactions between LLMs and regulatory T-cells via secreted phosphoprotein 1 (SPP1), apolipoprotein E (APOE), and intercellular adhesion molecule 1 (ICAM1) signaling axes. CellScape imaging confirmed the presence of GPNMB+CD163+CD274+ macrophages in PCNSL, with spatial analysis showing that LLM-T cell proximity correlated with treatment response.
 Conclusion We define a CNS-specific macrophage program characterized by TREM2+ GPNMB+ lipid-laden macrophages that may foster CNS tropism in PNCSL, along with immune evasion through metabolic reprogramming and regulatory T-cell activation. This first comprehensive spatial multi-omics characterization of macrophage heterogeneity distinguishing PCNSL from DLBCL identifies TREM2, SPP1 and lipid metabolism as potential therapeutic targets for macrophage-directed immunotherapy in this disease of unmet clinical need.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f03873c92164563d0cf6a99e4fda6b810ff22d" target='_blank'>
              Spatial multi-omics reveal distinct immunosuppressive lipid-laden macrophages in primary CNS lymphoma compared to systemic DLBCL
              </a>
            </td>
          <td>
            Liang Hong, Min Liu, Shruti Sridhar, C. Ong, Brooks Jennifer, F. Hamberger, B. Lane, Daniel Sanchez, Ranga Sudharshan, Ashley P. Tsang, Y. Peng, Chartsiam Tipgomut, P. Jaynes, O. Braubach, Evan Keller, Arvind Rao, S. de Mel, Joanne Lee, Jayalakshmi S., Qiang Pan-Hammarstrom, C. Tan, Susan Swee Shan, Siok‑Bian Ng, Claudio Tripodo, Anand D. Jeyasekharan
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background: Tertiary Lymphoid Structures (TLS) are organized aggregates of immune cells that form in non-lymphoid tissues in response to chronic inflammation, including cancer. In melanoma, TLS presence within the tumor microenvironment is associated with a pre-existing anti-tumor immunity and linked to improved immunotherapy responses. However, their role in predicting recurrence, particularly in early-stage melanoma, remains unclear.Methods: We conducted a retrospective analysis of archival primary tumor samples from 65 patients with stage I-II melanoma and negative sentinel lymph node (Table 1). Tumor immune infiltrates and TLS were initially assessed using four-marker immunohistochemistry (IHC) panels targeting CD20, CD4, CD8, and Podoplanin (PDPN), a marker of stromal cells. TLS were objectively identified using the HDBSCAN AI-based clustering algorithm. Tumor samples were further analyzed for high-dimensional phenotyping and spatial analysis using a 17 markers panel with hyperplexed immunofluorescence imaging (HIFI). Associations between TLS features and clinical outcomes were evaluated.Results: IHC revealed that 88% of patients had peritumoral T cell infiltration, and TLS were detected in 54% of cases. TLS-positive patients had improved disease-free survival (DFS, p=0.2) and overall survival (OS, p=0.2) (Figure 1A, B). To investigate further, HIFI was performed on 8 patients (4 recurrent, 4 non-recurrent), revealing 13 distinct cell types: stromal (n=6), tumor (n=2), and immune (n=5) (Figure 1C). Spatial analysis identified 9 recurrent spatial modules (SM), each representing a distinct pattern of interaction between cell types. One SM, enriched for PD-L1⁺ stromal cells, was significantly associated with recurrence (p=0.019) (Figure 1D).Conclusions: TLS presence associates with improved DFS and OS in early-stage melanoma. However, beyond their mere presence, TLS spatial organization and local stromal-immune context could influence clinical outcomes. Our data demonstrated that a specific spatial module—defined by the proximity of PD-L1⁺ stromal cells—is associated with disease recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/978d2823a740581a2a5f2a0fdbf9de6949d49e50" target='_blank'>
              13 | Spatial contexture of tertiary lymphoid structures as a predictor of recurrence in early-stage melanoma
              </a>
            </td>
          <td>
            Italian Melanoma Intergroup
          </td>
          <td>2025-12-05</td>
          <td>Dermatology Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BC) is the most common malignancy among women, with its progression and prognosis significantly influenced by the tumor microenvironment (TME). Age-related differences in TME composition lead to distinct tumor behaviors: young patients (≤ 40 years) exhibit aggressive tumors, while elderly patients (> 70 years) experience immunosenescence and reduced therapy responses. We performed single-cell RNA sequencing (scRNA-seq) analysis on tumors from 10 breast cancer patients (5 ≤ 40 years, 5 ≥ 70 years), encompassing 33,664 high-quality cells. After cell annotation and batch correction, malignant epithelial cells were identified using inferCNV. We applied pseudotime trajectory analysis, pathway enrichment, and cell–cell communication profiling to investigate age-specific TME dynamics. Survival relevance was assessed using a GEO cohort (GSE20685) of young breast cancer patients, and immunohistochemical staining was performed on clinical tumor and fibroadenoma tissues to validate protein-level expression of key ISGs. In young patients, malignant epithelial cells showed gradual upregulation of interferon-stimulated genes (ISGs) such as IFI44, IFI44L, IFIT1, and IFIT3 along the pseudotime trajectory, suggesting their involvement in early tumorigenesis. High expression of these ISGs was significantly associated with poor overall survival in a young BC cohort (GSE20685). Immunohistochemical validation further confirmed elevated IFIT3 protein levels in young tumor tissues. In contrast, elderly patients had a TME enriched in macrophages and fibroblasts, with activation of immunosuppressive pathways (e.g., SPP1, COMPLEMENT). Our integrative analysis identifies ISGs as key transcriptional drivers of tumorigenesis in young breast cancer, with potential prognostic and therapeutic value. Despite limited sample size, the combination of single-cell transcriptomics, clinical survival data, and protein-level validation provides robust evidence of age-specific TME remodeling. These findings support the development of age-tailored immunotherapy strategies targeting interferon signaling in young patients and immune checkpoint pathways (e.g., LAG3, CTLA4) in elderly individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81afa54ab4f065b06519a2062a7ca2a21beb87a4" target='_blank'>
              Single-cell RNA sequencing reveals age-related heterogeneity in the tumor microenvironment of breast cancer patients
              </a>
            </td>
          <td>
            Zihao Li, Ji Feng, Mengyang An, Shuming Kou, Zengren Zhao, Jingjing Yu, Bo Liu, Fangjian Shang
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Aberrant tumor glycosylation can alter immune recognition; however, the specific influence of glycan-lectin interactions on tumor progression remains poorly understood. Here, we identify the C-type lectin receptor CD301b (encoded by Mgl2) as a regulator of immune activity within the breast tumor microenvironment (TME). Using a murine triple-negative breast cancer model, we demonstrate that tumors expressing the Tn glycoantigen grow more rapidly, and this growth is facilitated by CD301b+ immune cells. Depletion or genetic loss of CD301b markedly suppressed tumor growth, indicating that CD301b promotes tumor progression through myeloid-tumor interactions. Phenotypic analyses revealed that CD301b+ cells within tumors are type 2 conventional dendritic cells (cDC2s), a subset known to influence immune polarization. Single-cell RNA sequencing of human breast cancers showed that the human ortholog CLEC10A is expressed in cDC2-like dendritic cells and select macrophage subsets, suggesting a conserved role for CD301+ myeloid populations. Transcriptomic profiling of tumors developed in Mgl2-deficient mice revealed a shift toward an inflammatory, immune-activated state consistent with enhanced antitumor immunity. Together, these findings establish a link between tumor glycosylation and lectin signaling of myeloid cells, highlighting CD301b as a potential target for reprogramming the tumor immune microenvironment in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401b567312697ac0756ae3899ebeedb84ca70dd0" target='_blank'>
              CD301b lectin expression in the breast tumor microenvironment augments tumor growth
              </a>
            </td>
          <td>
            A. Ozdilek, A. Paschall, Zahra Nawaz, Afaq M. M. Niyas, F. Avci
          </td>
          <td>2025-12-03</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Pediatric-type high-grade gliomas (HGGs) are aggressive brain tumors characterized by rapid growth, infiltrative behavior, and significant therapy resistance. One major challenge in treating HGGs involves the interplay of tumor cells to their microenvironment; specifically the blood-tumor barrier (BTB). Our studies combined histologic and transcriptomic evaluations of immunodeficient rodents intracortically injected with H3K27-altered and H3 WT glioma cells. Multiplex immunohistochemical staining provided temporospatial analysis of known cellular markers involved in BBB formation, including pericytes, endothelial tight-junction proteins, and myeloid cell markers. Regions of interest included the tumor, peritumor (+250µM), and distant (>250µM) areas. Analysis was conducted with the use of QuPath and cell segmentation performed using Mesmer; with expression proportions quantified for each cell/protein marker. Utilization of vessel isolation and nuclei extraction for sc-RNAseq (VINE-seq) allowed us to specifically explore the molecular landscape of vasculature within the tumor microenvironment. All rodent models were untreated and evaluated early to late post-intracranial injection (D20-50). Preliminary results in H3-K27-altered glioma models reveal elevated tight junctions within distal regions compared to the tumor (p<0.0001); demonstrating an intact BTB furthest from tumor bulk. Within the tumor, high Iba1+ macrophages (46%) and low neuronal (1%) expression was present (p<0.0001), while the intratumoral vasculature demonstrated the highest localization of glioma stem cells, and PDGFRβ+ during late disease progression (p<0.0001). VINE-seq identified unique cell-clusters in the tumor bearing vs. contralateral hemisphere at middle and late timepoints (D40-50). Ongoing VINE-seq studies will identify single cell expression changes both regionally and temporally with correlation to multiplex staining. These findings of blood-tumor barrier expression profiles among glioma cells offer a glimpse of changes for both the cellular and molecular landscape with both region and time specificity. Our research underscores the importance of understanding these interactions related to restrictions of CNS treatment and pediatric-type HGG progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3676d783a5588622cda56cd645fcf0d5acc71b" target='_blank'>
              TMIC-08. Multiplex immunofluorescent imaging and VINE-seq demonstrates the temporospatial complexity of the blood-tumor-barrier in pediatric-type diffuse high grade glioma
              </a>
            </td>
          <td>
            Jatia Mills, Minhye Kwak, Sanghee Lim, D. Maric, Kory Johnson, Trevor McKee, Sadhana Jackson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) exhibits substantial genetic and functional heterogeneities. We generated clonal spheroids (CS) from HNSCC cell lines and patient tumor cells, uncovering distinct functional differences. Hyperproliferative spheroids (HRPS) showed increased proliferation and tumorigenic potential, whereas hypoproliferative spheroids (HOPS) demonstrated enhanced migration and resistance to cisplatin and radiation. In patient tumor-derived spheroids, spheroid size distribution analysis across various TNM stages indicated that early-stage tumors (T2N0M0) predominantly formed HRPS, while later-stage tumors (T4N2M0) exhibited a higher prevalence of HOPS. Single-cell RNA sequencing (scRNA-seq) identified six clusters in the HOPS and four in the HRPS, each defined by unique gene expression profiles and pathway enrichment. HOPS were enriched with fibroblast, mesenchymal, and keratinizing epithelial cells, indicating a metastatic phenotype. They also expressed genes associated with stemness (KRT15 and ALDH3A1). Conversely, HRPS was characterized by proliferative genes, including TOP2A, AURKA, CKS2, AREG, and ANLN, suggesting an advanced HNSCC proliferative signature. Despite being generated from only two cells/well, the various clusters in both spheroid types mirrored the diverse tumor microenvironment in HNSCC, including neoplastic, epithelial, basal, fibroblast, and mesenchymal cells. Notably, the gene expression profiles of Cluster 3 in the HOPS and Cluster 1 in the HRPS closely matched the expression patterns observed in the HNSCC-TCGA dataset. These clusters also displayed a high transcriptional correlation between patient tumors and their xenografts, reinforcing the clonal nature of HNSCC’s genetic and functional diversity of HNSCC. This study provides a deeper understanding of the genetic and functional diversity of HNSCC tumors, which can aid in the development of more precise diagnostic tools by identifying distinct tumor subtypes. Distinguishing between hyper- and hypoproliferative spheroids could improve prognostic accuracy, helping to predict tumor behavior and patient outcomes. Moreover, the identification of specific gene expression profiles associated with proliferation, metastasis, and treatment resistance may guide personalized therapeutic strategies, enabling targeted treatment based on the unique tumor characteristics of individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/942cca58b9f76ca4806f68d559d87541dfee33c1" target='_blank'>
              Uncovering functional divergence and cellular clusters with specific gene signatures in HNSCC clonal spheroids
              </a>
            </td>
          <td>
            Jyoti Pandey, Md. Zubbair Malik, Ritis K Shyanti, Palak Parashar, Praveen K. Kujur, Deepali Mishra, Dhanir Tailor, Jee Min Lee, T. Kataria, Deepak Gupta, Hitesh Verma, Anshu Yadav, S. Kateriya, Sanjay V. Malhotra, Vibha Tandon, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Background and objectives Recent molecular landscape studies characterizing lung cancer mostly utilize single tumor regions, ignoring tumor heterogeneity and evolutionary patterns. To address this issue, the TRAcking NSCLC Evolution through therapy (TRACERx) Consortium established a multiregion sampling of Western lung cancer to investigate its longitudinal evolutionary dynamics. Due to ancestral differences, there are still major gaps in understanding oriental lung cancer. Methods We have established a prospective cohort of patients with non-small cell lung cancer (NSCLC) named Chinese lung cancer evolution and microenvironment tracking under therapy (CLEVER) study. Discussion By acquiring comprehensive genetic variation data and clonal events from multiple regions of the tumor, we aim to decipher the molecular evolution code of Chinese lung cancer. The analysis of cell composition and spatial structure further characterizes the remodeling effects of tumor evolution and neoadjuvant therapy on the tumor microenvironment, helping to identify important markers that influence treatment effectiveness and recurrence. By monitoring mutations using a personalized minimal residual disease (MRD) panel from tumors at baseline, we can trace evolutionary significance and clonality, aiding in therapy decisionmaking. The CLEVER study significantly contributes to the development of personalized treatment strategies for lung cancer patients, enhancing outcomes and tailoring interventions to individual patient profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f2f25eef8688466fb037f4fd6a8d16c5aeeb96" target='_blank'>
              Protocol for Chinese lung cancer evolution and microenvironment tracking under therapy study
              </a>
            </td>
          <td>
            Wenxiang Wang, Yue He, Xiaoqiu Yuan, Lin Weng, Jing Bai, Xu Liu, Li Han, Jun Xu, Shuai Wang, Shanshan Lu, Jian Bai, Hui Quan, Qingyun Liu, Xinrui Li, Yuxuan Xiao, Yun Wang, Qingna Zhang, Na Zhou, Yanyan Hou, Hao Li, Y. Li, Fan Yang, Jun Wang, Kezhong Chen
          </td>
          <td>2025-12-01</td>
          <td>Journal of Translational Internal Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0127956ee657e81733c7622d2b48c0407a2cab1" target='_blank'>
              Cell heterogeneity contributes to the variable response of HIV-1 to latency reversing agents
              </a>
            </td>
          <td>
            Rachel Topno, Hussein Karaki, Flavia Mazzarda, Oriane Pourcelot, Manon Philippe, K. Zibara, Ovidiu Radulescu, Edouard Bertrand
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe18ec72d329e63e66769b08f8cae7347edc672" target='_blank'>
              Prognosis prediction using autophagy gene expression in osteosarcoma
              </a>
            </td>
          <td>
            Qing Liu, Cong Zhou, Ke Xu, Qikui Zhu, Junqin Lu, Chun Zhang, Wukun Xie, Guojiu Fang, Xue Zhou Chen, Dasheng Tian, Juehua Jing, Yize Li, Hongsheng Wang, Wenxiu Duan, Yihui Bi
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Current therapeutic strategies for endometrial cancer are mainly based on aggressive histological types and molecular subtypes. However, ignoring the spatial distribution of immune/stromal cells fails to account for the heterogeneity of the local tumour microenvironment, leading to biased prediction of treatment response. The goal of precision medicine is to delineate the biological characteristics of local functional units based on molecular labelling, which adequately reflects spatially adaptive changes during treatment or metastasis. Single-cell resolution analysis of 40 endometrial cancer cases across four molecular subtypes was performed using imaging mass cytometry (IMC) to quantify the frequency, spatial distribution, and intercellular crosstalk of distinct immune and stromal cell populations. These ecosystem-level features were systematically correlated with clinical features and outcomes, including treatment response and survival. We further identified CD90 + clusters as key regulators of macrophage polarization and T-cell infiltration dynamics, with flow cytometry used to validate their functional role in tumour subtype specification and microenvironmental remodelling. Finally, machine learning-based spatial phenotyping was employed to construct molecular subtype-specific signatures and a highly accurate recurrence prediction model for high-risk endometrial cancer. Single-cell profiling revealed that CD90 + clusters constitute a critical immunomodulatory component within the tumour microenvironment, demonstrating significant enrichment in macrophage differentiation pathways and serving as key mediators of intercellular signalling networks. Furthermore, computational models integrating functional molecular signatures with cell–cell interaction profiles demonstrated high predictive accuracy for both molecular subtyping and recurrence risk stratification in patients with endometrial carcinoma. Our study establishes a spatial eco-context framework for molecular subtypes of endometrial cancer by integrating single-cell spatial multiomics data. This approach enables high-resolution mapping of tumour-immune-stromal interaction networks and reveals novel targets for personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63a8a4865fa920849f9dc4e8439a41c6d062126a" target='_blank'>
              Spatial ecostructural modelling of endometrial cancer identifies the key role of CD90 + CD105 + endothelial cells in tumour heterogeneity and predicts disease recurrence
              </a>
            </td>
          <td>
            Di Wu, Cinian Lv, Zhifeng Yan, Luyang Zhao, Lian Li, M. Ye, Mingyang Wang, Qingzhi Zhai, Nan Wang, Zheng Wang, Yuanguang Meng, Mingxia Li
          </td>
          <td>2025-11-17</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 The spatial dynamics of B cells are crucial during immune response, shaping clonal evolution and affinity maturation. B cell evolution can by studied by reconstructing phylogenetic lineage trees, which represent B cell receptor (BCR) mutations among sampled cells. SpatialVDJ sequencing is a recently developed technique that pairs spatial RNA-sequencing with long-read, spatially resolved BCR data on a 2D slice of tissue. Currently, no methods exist to analyze the movement of B cell lineages across 2D coordinates. We introduce GIBLE (Geographic Inference of B Cell Lineage Evolution), a novel computational framework that reconstructs spatial locations of unobserved, ancestral B cells and quantifies movement across the tissue slice. We used GIBLE to understand B cell maturation in ectopic germinal centers in thymic tissue from patients with myasthenia gravis, an autoimmune disorder in which autoreactive B cells produce antibodies against the neuromuscular junction. GIBLE provides a robust and widely applicable framework for integrating spatial and phylogenetic data to trace B cell evolution during immune responses using SpatialVDJ data.



 Supported by NIH/NCI 5T32CA134286.



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171685d301f0d9a0ae908798444d8439b632feea" target='_blank'>
              Resolving B cell migration from SpatialVDJ sequencing with GIBLE 3612
              </a>
            </td>
          <td>
            Hunter J. Melton, Chinmay M. Vaidya, Jessie J. Fielding, Sherry Wu, Fred W. Kolling, Kevin C. O’Connor, Kenneth B. Hoehn
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Lung cancer is a highly heterogeneous disease, and the tumor microenvironment (TME) characteristics are closely related to disease progression and treatment outcomes. We elucidated the cell type-specific transcriptome landscape of cancer cells and the effect of the TME on lung adenocarcinoma (LUAD). Methods Single-cell RNA sequencing of the LUAD tissues and matched adjacent normal tissues of three patients in the early-stage via histopathological and immunohistochemical confirmation. Results The results revealed the landscape of 68,579 cells in LUAD microenvironment, and highly heterogeneous AT2 cells are crucial source of lung epithelial cell carcinogenesis. Genes KRT81, SPP1, PCDH7, SLC2A1, and TET1 were significantly upregulated in tumor tissues and associated with poor prognosis and survival, providing insights for exploring lung cancer biomarkers in future studies. Trajectory analysis identified ERBB4, SEMA4A, GCNT2 and SOX4 as key factors in AT2 cells that may promote cell proliferation, migration, or epithelial-mesenchymal transition (EMT) during the progression of lung adenocarcinoma (LUAD). We elucidated the transcription factor–target gene regulatory network involving NKX2-1 and TEAD1 in malignant tumor cells derived from AT2 cells. The findings indicated that malignant AT2 cells regulate communication between epithelial and immune cells in the TME by predicted FN1-CD44 and CADM1-CADM1 ligand-receptor interactions, which ultimately suppresses the host immune response. Discussion This comprehensive single-cell analysis increases our understanding of AT2 cells molecular and dynamics of metastatic lung cancer. In summary, single-cell RNA profiling of LUAD offers valuable prognostic insights based on AT2 cell types and identifies potential biomarkers for therapeutic responses, aiding the future development of LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e15868fa220169dfea67c353cad3603481040a" target='_blank'>
              Single-cell RNA sequencing reveals heterogeneity among AT2 epithelial cells in the lung adenocarcinoma microenvironment
              </a>
            </td>
          <td>
            Zimeng Cao, Xuefeng Bai, Jing Li, Jinru Cai, Gang Cao, Yuncai Xiao, Wei Zhou
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8a1f1ccbeb7cda31923d2bd498d2b7cb3a690e5" target='_blank'>
              Metabolic zonation and characterization of tissue slices with spatial transcriptomics
              </a>
            </td>
          <td>
            Oier Etxezarreta Arrastoa, A. Pirona, Allon Wagner, N. Yosef
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) represents a significant global health challenge due to its molecular heterogeneity and the immunosuppressive nature of its tumor microenvironment (TME). Tumor-associated macrophages (TAMs), particularly the M2-polarized subset, are crucial in HCC progression, contributing to immune evasion, angiogenesis, and metastasis. However, the mechanisms driving TAM reprogramming and their metabolic interactions within the TME remain poorly understood. In this study, we applied single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to identify a novel metabolic-immune axis in HCC. Our findings revealed increased oxidative phosphorylation (OXPHOS) in tumor cells, which correlated with enhanced ribosomal activity in M2 macrophages, suggesting a metabolic coupling between the two. Tumor cells, under elevated OXPHOS, secrete factors that recruit macrophages and promote M2 polarization, thereby reinforcing immune suppression. Further analysis of The Cancer Genome Atlas (TCGA) dataset led to the development of a prognostic model based on OXPHOS-related genes, showing strong predictive power across multiple cohorts. These results suggest a novel OXPHOS-ribosome-M2 polarization axis and offer potential therapeutic targets for immunometabolic therapies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d98d705964e6108c751cf9dce67e7948e856f2d" target='_blank'>
              M2 macrophages and tumor cells engage in a metabolic feedback loop to drive HCC progression
              </a>
            </td>
          <td>
            Yimin Nong, Xiaolong Chen, Baoxin Zhang, Zhenying Lin, Wuriniletu, Chengyou Du, Qiang Liu
          </td>
          <td>2025-11-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Brain metastasis is a major contributor to melanoma-related mortality, arising from both genomic alterations and dynamic adaptations to the brain microenvironment. Tumor cells must cross the blood–brain barrier and exploit brain-specific signaling and metabolic landscapes to establish metastases. Brain metastases display molecular and phenotypic programs distinct from other organs, reflecting pre-existing and acquired mutations during progression. We analyzed primary melanoma, metastatic melanoma, and metastatic brain tumors genomic datasets to identify mutated proteins and signaling pathways that are common to or distinct among these tumor types. Our comprehensive computational analyses uncovered key pathways and proteins associated with brain metastasis. Brain metastases exhibit unique mutational landscapes and convergent alterations in neurotrophin, MAPK, and PI3K/AKT signaling compared with primary and metastatic melanoma. Proteins especially frequent in brain metastases pathways support survival, proliferation, and adaptation in the brain microenvironment. Pharmacology-wise, rational strategies such as combining c-KIT and VEGFR inhibitors can suppress converging oncogenic pathways and mitigate resistance in advanced melanoma. Co-targeting neurotrophin receptors with MAPK pathway members, particularly in combination with BRAF inhibitors, appears a promising approach for treating melanoma brain metastases. Our findings underscore the molecular heterogeneity of melanoma brain metastases, shaped by interconnected signaling networks, and reveal therapeutic targets that may diverge from those in other metastatic sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d4ecead949fc8a270a159656433a0bf5e5ad895" target='_blank'>
              Cellular Mechanisms Underlying Melanoma Brain Metastasis
              </a>
            </td>
          <td>
            B. R. Yavuz, Hyunbum Jang, Ruth Nussinov
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="


 Glioblastoma exhibits regional heterogeneity, with distinct cellular phenotypes and molecular features between contrast-enhancing (CE) solid tumor regions and non-enhancing (NE) infiltrative regions. Understanding functional repercussions of this heterogeneity is important.



 Image guided biopsies of CE and NE regions of newly diagnosed IDH wild-type glioblastoma were obtained. Tissues were analyzed using single cell sequencing to identify glioma stem cell populations using gene set enrichment analysis and established gene signatures. Cells were treated with temozolomide (TMZ, 1000 μM), paclitaxel (PTX, 50 μM), topotecan (TPT, 50 μM), vincristine (VCR, 50 μM), and doxorubicin (DOX, 20 μM) for 48 hours. Cell viability was measured using a luminescence-based assay to assess regional differences in chemotherapeutic response.



 Matched samples from 5 patients were sequenced and identified NE regions as harboring elevated levels of glioma stem-like cells using multiple established signatures. Multiple cell markers including several HOX genes, PROM1, F3, ID4, PAUPAR were significantly upregulated in NE tumor cells (FC ≥ 4.0; FDR < 0.05). Samples from NE regions grew slower (p<0.001) than CE regions at the 1-week mark, but were able to recapitulate confluent tumor in cortical organoid and xenograft models by 4 weeks. In matched samples from 3 patients, there was significantly reduced chemotherapeutic efficacy in NE regions (CE viability vs NE viability, p = 0.17) in the various chemotherapeutic agents tested.



 NE infiltrating regions of glioblastoma maintain high proportions of cells that have characteristics of glioma stem cells transcriptionally. These regions have slow initial growth in in vitro and in vivo models, but are able to recapitulate solid tumor properties. Likely related to this quiescent property, NE regions are more resistant to chemotherapeutic agents.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49847b7486480b2128b07d57d35417c0ebd3d28" target='_blank'>
              DDDR-05. Non-enhancing regions of Glioblastoma Harbor Quiescent Stem Like Cells that Contribute to Chemotherapy Resistance
              </a>
            </td>
          <td>
            Mahmoud I Youssef, Travis Perryman, Stuart Harper, Amani Carson, Lindsey A Dudley, Eliana S Oduro, Alondra Delgadillo, Shail Patel, Nish Sud, Harley Kornblum, Kunal S. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary Cancer stem cells (CSCs) have been suggested to be the subset of cancer cells that drive tumor growth and progression and fuel the tumor with heterogeneous populations of cells, including those resistant to therapy. Different proteins have been proposed as biomarkers of CSCs in oral squamous cell carcinoma (OSCC). In this study, we simultaneously investigated multiple putative biomarkers of CSCs that have been previously proposed in OSCC, with the aim of identifying a pure CSC subpopulation. Our results suggest that several stem cell subpopulations may co-exist in a tumor, each having an impact on different clinical parameters. The cell subpopulations identified by use of different CSC markers were found to be dynamic and able to switch their protein markers over time. The inter-tumoral diversity of cancer stem cells demonstrated here raises novel challenges in terms of their use as predictive biomarkers and as efficient therapeutic targets for OSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dc7fec448accd93f20cdf3e8cf88a44c781b603" target='_blank'>
              Identification of Phenotypically Distinct Cancer Stem Cell Subpopulations in Oral Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            T. Osman, Oddveig G. Rikardsen, M. Teh, D. Sapkota, Kristina Xiao Liang, E. Neppelberg, Adrian Biddle, Ian C. Mackenzie, Lars Uhlin-Hansen, A. C. Johannessen, D. E. Costea
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Gliomas are lethal malignancies composed of heterogeneous and dynamic subpopulations of cellular states that resemble both normal neurodevelopmental cell types and adaptive responses to stress within the tumor microenvironment (TME). While cellular plasticity enables gliomas to survive environmental pressures, the mechanisms driving glioma state dynamics remain unclear. To explore how glioma state heterogeneity functionally and metabolically interacts with the brain TME, we conducted a multi-omic analysis across 392 glioma specimens, including patient tumors, orthotopic xenografts and gliomasphere cultures using bulk RNA/whole-exome sequencing and lipidomic profiling. Single-cell transcriptome sequencing of a diverse panel of glioma tumors (n = 21) integrated with bulk tumor transcriptome profiling revealed a spectrum of cellular identities associated with distinct lipidomic profiles. Comparison of matched patient and derived models showed that the non-native environments, such as gliomasphere culture, restricted glioma state diversity and enriched for states characterized by elevated de novo fatty acid synthesis. Across environmental contexts, glioma state plasticity was coupled to lipid metabolic plasticity, enabling tumors to concordantly remodel their state composition and lipid profiles in response to environmental constraints. Of note, certain tumors exhibited limited state and metabolic plasticity – specifically those enriched for states within oligodendroglial and neuronal lineages. These gliomas demonstrated reduced de novo lipid synthesis capacity, and a concordant increased reliance on exogenous lipid scavenging within the brain microenvironment for survival. Together, these results link glioma heterogeneity and plasticity to lipid metabolic reprogramming, and highlight how distinct cellular state profiles result in environment-dependent metabolic liabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149479392e37d2670e1ddd1d4b3e1788684e5fcb" target='_blank'>
              EPCO-63. LIPID METABOLIC HETEROGENEITY ACROSS GLIOMA CELLULAR STATES REVEALS ENVIRONMENTAL DEPENDENCIES
              </a>
            </td>
          <td>
            Nicholas A. Bayley, Jenna K. Minami, Cassidy Andrasz, C. Tse, Jennifer Salinas, Dimitri Cadet, A. Bhaduri, T. Cloughesy, Thomas G. Graeber, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="


 Leukemia evolution is driven by a complex interplay between somatic mutations and epigenetic changes. However, our understanding of these processes is limited by a lack of scalable and cost-effective methods to co-assay genetic and epigenetic changes in single cells. Here, we present scGEM-seq (single-cell Genotyping and Methylome sequencing), a flexible multi-omics platform that permits simultaneous characterization of genetic variants and DNA methylation profiles from single cells. scGEM-seq combines three core innovations that can be accomplished at ~3% the cost ($0.03 per cell) of competing technologies (>$1 per cell in snmC-seq3 and SciMETv2): (1) single-cell capture with semi-permeable capsules (Atrandi Biosciences), enabling multi-step enzymatic reactions to be performed efficiently with minimal loss of DNA; (2) a novel split-pool strategy (HexaDecimal barcoding) that efficiently generates ~1 million unique cell barcodes that are compatible with bisulfite-like enzymatic conversions; and (3) flexible library preparation that can be adapted for genome-wide or targeted sequencing.
 In cell-mixing studies, we show that scGEM-seq accurately resolves cell populations based on DNA methylation profiles and genetics. At modest sequencing depth (0.001X per cell for whole-genome DNA libraries, 15734.4±12123.1 CpGs per cell for methylation libraries), scGEM-seq resolved mouse and human cell lines at expected ratios (1:1:1 ratio of GM12878, K562 and YAC1 cells) with low doublet rates (~2.4% using species genome alignment). scGEM-seq also resolved human cancer cell lines of distinct lineages (1:1:1 ratio of SW480, A375 and OCIAML3 cells) according to methylation and genetics (cell-specific SNVs and copy-number changes) with modest sequencing depth (0.002X per cell for whole-genome DNA libraries, 20242.6±12808.2 CpGs per cell for methylation libraries) and low doublet rates (2.88% using cell-specific SNVs).
 To enhance genotyping efficiency, we also combined scGEM-seq with two different target enrichment strategies: hybrid capture (Pan-cancer hybridization panel, IDT; n=127 genes) and amplicon sequencing. In mixed human cancer cell lines, scGEM-seq achieved consistent single-cell genotyping for known cancer genes and hotspot mutations, including a mean of 13.23±5.30% of cells for hybrid capture and >98% of cells for amplicon sequencing. To determine the sensitivity of scGEM-seq for genotyping of rare cells, we prepared a cell mixture containing varying proportions of human cancer cell lines (GM12878: 92.85%, SW480: 5%, A375: 1.5%, OCIAML3: 0.5%, K562, 0.15%). scGEM-seq coupled with amplicon sequencing for known cell-specific variants recovered the expected cell proportions down to ~0.1-0.2%, representing fewer than 2 malignant cells per 1000 (SW480: 302/6294 cells, 4.80%; A375: 88/6294 cells, 1.40%; OCIAML3: 30/6294 cells, 0.48%; K562: 11/6294 cells, 0.17%).
 Finally, we applied scGEM-seq to primary bone marrow samples from 4 patients with myeloid neoplasms (MDS and AML) representing a range of blast counts (47%, 52%, 10% and 16%). Here, scGEM-seq revealed distinct populations of normal, premalignant, and malignant cells, including subclonal leukemia cells with KIT D816V (47/777 cells, 6.05%) and T417_D419delinsL (72/700 cells, 10.29%) mutations in a patient with inv(16) AML, and clonal (premalignant) hematopoietic cells harboring a DNMT3A S770L mutation (251/558 cells, 44.98%) and subclonal malignant blasts with distinct KRAS G12C (92/385 cells, 23.9%) and G12D (4/381 cells, 1.05%) mutations.
 In conclusion, scGEM-seq provides a flexible, cost-effective, and high-resolution method for elucidating the complex interplay between epigenetic and genetic cell states in cancer cells, including within rare cell populations. We envision multiple applications for scGEM-seq for the study of leukemia and other blood cancers, including to characterize (epi)clonal heterogeneity and dynamics, identify minimal residual disease, and capture rare stem/progenitor populations or drug-resistant clones that emerge post therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da31bde36d790d58830d2a574d2fd507a69d8f" target='_blank'>
              Single-cell genotyping and methylome sequencing (scGEM-seq) resolves cell-states and clonal hierarchies in leukemia
              </a>
            </td>
          <td>
            Kyung Lock Kim, Maria R. Capilla-Guerra, Evan C Chen, Andrew Lane, Gabriel Griffin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Phenotypic plasticity plays a key role in cancer progression and metastasis, enabling cancer cells to adapt and evolve, but precisely how distinct axes governing phenotypic plasticity interact to shape tumour progression and patient outcomes remains unclear. We investigated five major interconnected axes of plasticity in ER-positive (ER+) breast cancer: Metabolic Reprogramming, Epithelial-to-Mesenchymal Plasticity (EMP), Luminal–Basal (Lineage) Switching, Stemness, and Drug-resistance using network dynamics simulations, integrative bulk and single-cell transcriptomic analyses and patient survival analyses. We show that these axes are not independent but drive one another, forming two mutually inhibiting ‘teams’ of nodes enabling specified cellular behaviour. One team (favouring high glycolysis, stem-like, basal-like, mesenchymal/hybrid and tamoxifen-resistant phenotype) was found to be associated with aggressive progression and worse survival. On the other hand, the opposing team (favouring high oxidative phosphorylation, non-stem-like, luminal-like, epithelial and tamoxifen-sensitive phenotype) correlated with better outcomes. Importantly, altering one axis of plasticity often drove coordinated responses along other axes and vice versa. Our findings establish phenotypic plasticity in cancer as a coordinated, multi-axis dynamical process, thus suggesting novel strategies to disrupt systems-level reprogramming enabling metastasis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acaf4b9f2872701e5c44aad69b0ee83629b9228" target='_blank'>
              Interconnected axes of phenotypic plasticity drive coordinated cellular behaviour and worse clinical outcomes in breast cancer
              </a>
            </td>
          <td>
            Ritesh Kumar Meena, Yijia Fan, Soundharya Ramu, Anjaney J. Pandey, Abhinav Kannan, Yashvita Subramanian, Yukta Subramanian, Jason T. George, M. Jolly
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Metastatic outgrowth requires that cancer cells delaminate from the primary tumor, intravasate, survive in circulation, extravasate, migrate to, and proliferate at a distal site. Recurrent genetic drivers of metastasis remain elusive, suggesting that unlike the early steps of oncogenesis, metastasis drivers may be variable. We develop a framework for identifying metastasis regulators using CRISPR/Cas9-based screening in a genetically defined organoid model of colorectal adenocarcinoma. We conduct in vitro screens for invasion and migration alongside orthotopic, in vivo screens for gain of metastasis in a syngeneic mouse model. We identify CTNNA1 and BCL2L13 as bona fide metastasis-specific suppressors which do not confer any selective advantage in primary tumors. CTNNA1 loss promotes cell invasion and migration, and BCL2L13 loss promotes anchorage-independent survival and non-cell-autonomous changes to macrophage polarization. This study demonstrates proof of principle that large-scale genetic screening can be performed in tumor-organoid models in vivo and identifies novel regulators of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33c9f8937cc2d0d69608a2acde03a6c6afb3fab8" target='_blank'>
              Forward genetic screening in engineered colorectal cancer organoids identifies regulators of metastasis.
              </a>
            </td>
          <td>
            Xin Wang, Zvi Cramer, N. A. Leu, Keara Monaghan, K. Durning, S. Adams-Tzivelekidis, Joshua Rhoades, Jonathan Heintz, Yuhua Tian, Joshua Rico, Diego Mendez, R. Petroni, Austin King, Melissa S Kim, Rina Matsuda, Olivia Hanselman, Alice E. Shin, María F Carrera Rodríguez, Igor E. Brodsky, Anil Rustgi, Ning Li, Christopher Lengner, M. Andres Blanco
          </td>
          <td>2025-11-11</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Genomic alterations in glioblastoma (GB) drive distinct transcriptional subtypes, yet how they shape the tumour microenvironment (TME) architecture remains poorly understood. Here, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss, and PDGFB overexpression in Tp53−/− Nestin-tv-a mice, that recapitulated classical, mesenchymal, and proneural transcriptomic GB subtypes. We integrated single-cell RNA sequencing with regional (NanoString GeoMx) and single cell (10x Visium HD) spatial transcriptomics and proteomics data to acquire an in-depth characterization of the TME organization. At single cell level, NF1- and EGFRvIII-driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human GBs that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1043a078420592595b843047c99776c531341d4b" target='_blank'>
              TMIC-04. Genetic drives promote distinct microenvironment organization in glioblastoma
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is characterized by molecular and phenotypic heterogeneity. With increasing guideline‐driven use of metastatic biopsies, more mPC specimens are being evaluated in surgical pathology. However, unlike localized prostate cancer, no standardized framework currently exists to guide the diagnostic workup of metastatic biopsies or reliably determine phenotypic subtypes. While many mPCs retain conventional acinar features, a growing subset exhibits phenotypic plasticity – including loss of prostate epithelial identity and emergence of neuroendocrine or other divergent lineages. This phenotypic diversity often occurs in castration‐resistant prostate cancer as a mechanism of resistance to chronic androgen receptor pathway inhibition and is characterized by genomic alterations and epigenetic reprogramming. This review outlines the histologic and molecular spectrum of mPC and proposes a practical, pathology‐informed diagnostic approach integrating morphologic assessment and immunohistochemistry. Adoption of a standardized diagnostic framework and multidisciplinary integration will be useful for employing precision oncology in advanced mPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de38a46f132d3e7836729ee1ba6aae01604d174" target='_blank'>
              Molecular subtypes of metastatic prostate cancer: from pathophysiology to diagnosis
              </a>
            </td>
          <td>
            J. M. Dsouza, Erolcan Sayar, M. Schweizer, Stephanie Harmon, C. Morrissey, H. Beltran, Peter S. Nelson, Liang Cheng, C. C. Ding, Michael C. Haffner
          </td>
          <td>2025-12-12</td>
          <td>Histopathology</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="


 Human diseases affect various cell types within the local niche, but the multicellular responses of human diseases have not been collectively assessed across organs. Here, we categorize multicellular responses of human diseases based on a remapped human cell atlas, incorporating 12 million single-cell transcriptome profiles acquired from an automated data collection pipeline. Additional curation expands the collection up to 46 million cells, covering 160 disease conditions across 49 distinct organ identities. From the re-aligned atlas of single-cell transcriptome data, we constructed a hierarchical cell classification framework comprising 32 major cell types and 165 cell subtype annotations. The universal cell annotation framework enabled abundance-based clustering of the blood samples from various disease contexts, which rendered 10 different blood phenotypes including a Covid-19-specific interferon response enriched phenotype. Further, systematic profiling of disease and organ association patterns of cellular subtype reveals archetypes of human diseases repeatedly found across organs, including a multicellular phenotype highly specific to the tumor microenvironment. Collectively, our study provides foundational resources for cross-tissue cell type annotation and presents a comprehensive multicellular overview of disease-associated responses across human tissues.



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2903ab4171a794e8922ec87b95153f45613d70f3" target='_blank'>
              Cross-tissue integration of human cell identifies common cellular phenotypes across human diseases 2329
              </a>
            </td>
          <td>
            Seong-Kun Kim, Jong-Eun Park
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is marked by an immunologically inert tumor microenvironment (TME), which may explain why immunotherapies have generally failed to improve survival. While TME features have been linked to clinical outcomes, prior studies were limited by small sample sizes and narrow cell type resolution, precluding a full understanding of TME multicellular dynamics. To address this, we assembled 1,979 clinically annotated GBM RNA-seq profiles and benchmarked six deconvolution algorithms using pseudobulk data from a large GBM single-cell RNA-seq dataset. Bayesian deconvolution (InstaPrism) emerged as a top performer and was used to characterize the TME across 39 cell states. We then stratified tumors by their dominant malignant cell state (AC-like, MES-like, NPC-like, or OPC-like). We identified lymphoid-dominant and myeloid-dominant immune compositions within each GBM subtype. We demonstrate that lymphoid-dominant, immune-activated tumors associate with significantly improved overall survival within the aggressive MES-like group, but not within other groups. Immune activation was defined by a conserved stress-inflammatory module consisting of general inflammatory activation, tissue remodeling, and stress. In tumors dominated by MES-like cell states, immune cells within the TME were driven by robust interferon alpha and gamma gene signatures. Sobel mediation analysis pinpointed Stimulator of interferon genes (STING) upregulation as a potential upstream orchestrator within this group. Interestingly, STING activation showed no beneficial effect in AC-like or NPC-like tumors and was detrimental in OPC-like GBM, highlighting a group-specific biology. To our knowledge, this represents the most comprehensive analysis of the GBM TME to date in terms of patient number (n=1,979) and TME cell type plexy (n=39). The findings offer new understanding of GBM multicellular organization and how different cell types coordinate within the tumor. These data may help design rational combinations of immunomodulatory therapies and guide patient selection for next-generation clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91fc74608f0fb0d307e1af10e8fcfdf8bbe8b02e" target='_blank'>
              TMIC-63. Microenvironment archetypes in 1,979 glioblastomas reveal cell state-specific drivers of immune recruitment and survival benefit
              </a>
            </td>
          <td>
            Kevin J. Tu, Zachary J. Reitman
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Understanding CD8+ tissue-resident memory T cells (TRM) spatial characteristics in hepatocellular carcinoma (HCC) is challenging, and clarifying the spatial feature changes following immunotherapy represents an urgent research gap. Methods This study employs a multi-omics approach to analyze the spatial distribution and intercellular interactions of TRM cells in HCC tissues using radiomics, single-cell sequencing, and multiplex immunofluorescence histochemistry (m-IHC). Results Our results show that the number of CD8+ TRM cells in HCC increases following immunotherapy. Furthermore, after dividing tumor tissues into the tumor core (TC), invasion margin (IM), and normal tissue (N), a increase in CD8+ TRM cells from the IM to the TC can be observed. Consistent with the results of single-cell sequencing analysis, this change in spatial characteristics may be associated with the interactions between CD8+ TRM cells and CD68+ cells. Discussion Immunotherapy can modify the spatial characteristics of CD8+ TRM cells via regulating their crosstalk with other immune cells, and the spatial distribution of CD8+ TRM cells in the HCC tumor microenvironment (TME) correlates with immune checkpoint blockade (ICB) therapeutic efficacy. Clarifying the mechanisms of action of immunotherapeutic drugs and developing a non-invasive radiomics model to predict CD8⁺ TRM cell dynamics will facilitate the clinical management of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/384dbc51e0f9efcd6dc85b035d8ba5b7b014c96f" target='_blank'>
              Multi-omics spatial characteristics of CD8+TRM cells in hepatocellular carcinoma and immunotherapy response prediction
              </a>
            </td>
          <td>
            Shipeng Li, Ju Ma, Xinqiang Li, Bingbing Qiao, Liancai Wang, Deyu Li, Hua Zhou, Haixu Xu, Jinzhen Cai
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Spatial studies of the tumor microenvironment are crucial for understanding the complex interactions between different cell types and how these interactions contribute to disease progression. This poster presents a streamlined approach to spatial analysis of the tumor microenvironment in brain tissues, focusing on astrocytoma and oligodendroglioma. Additionally, the methodology was applied to Parkinson’s Disease versus healthy brain tissue to observe disease-specific alterations. We employed primary antibody conjugates to perform immunohistochemistry (IHC) on human tissue samples, utilizing an 8-plex panel of markers. The study incorporated IHC-validated antibody conjugates supplemented by an antibody labeling kit for disease-specific markers, facilitating high spatial resolution imaging without the need for lengthy workflows commonly associated with secondary amplification or cyclic stripping methods. Our findings demonstrate a significant increase in astrocytes and oligodendrocytes in the tumor tissue, as revealed by the increased presence of GFAP and Olig2, respectively, compared to healthy controls. The use of the EVOS S1000 imaging system made image capture and analysis remarkably straightforward, further simplifying the workflow. This approach provides clear advantages over traditional methods, highlighting its potential for broad application in immunological and neuropathological studies. For Research Use Only. Not for use in diagnostic procedures.



 Thermo Fisher Scientific



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957e86629e50682381d4aded4eed8bfc06834bb3" target='_blank'>
              Streamlined spatial analysis of the tumor microenvironment and associated disease states using multiplex immunohistochemistry 9329
              </a>
            </td>
          <td>
            Mae Voeun, C. Langsdorf
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e088dcd4f395d6c7df482c6c35fcef0137dc41" target='_blank'>
              An epigenetic switch in vascular phenotype augments anti-tumor immunity
              </a>
            </td>
          <td>
            Dae Joong Kim, Mitchell T McGinty, S. Anandh, C. Riedstra, Yuvraj Sethi, Melanie R. Rutkowski, A. C. Dudley
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Over 95% of primary central nervous system lymphoma (PCNSL) cases are histologically classified as diffuse large B-cell lymphoma (DLBCL), but present markedly poorer prognosis than systemic DLBCL. This discrepancy may stem from its high molecular heterogeneity, immune privilege, inhibitory tumor microenvironment (TME) and dilemma in drug delivery. Current understanding in the tumor ecosystem of PCNSL remains limited. This study employed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic (ST) analyses to decode tumor microenvironmental features of PCNSL.
 We performed scRNA-seq of PCNSL tumor samples and reactive lymphoid hyperplasia samples, and additionally conducted ST on PCNSL tumor samples. Unsupervised clustering of B cells identified 4 malignant B cell (mBc) subpopulations with apparently transcriptional and spatial discrepancy. Pseudotime trajectory analysis revealed that mBc3 represented the origin, which evolved into mBc4 and mBc1 and ultimately differentiated into mBc2. To assess the clinical relevance of these subsets, we applied BayesPrism for deconvolution of bulk RNA-seq data from DLBCL cohorts, presenting mBc2 subset was significantly correlated to poor prognosis (p< 0.001). Kyoto Encyclopedia of Genes and Genomes pathway analysis and scMetabolism analyses indicated that mBc2 exhibited a strong oxidative phosphorylation (OXPHOS) metabolic signature. ST analyses of tumor specimens further confirmed the overlap of mBc2 localization and the regions with elevated OXPHOS activity.
 Based on its high OXPHOS characteristics, we further explored its molecular expression features in an attempt to identify potential therapeutic targets. COX7B, a structural subunit of mitochondrial respiratory chain complex IV, was identified as an mBc2 marker through Venn analysis, specifically overexpressed in the mBc2 subset, the terminal stage of developmental trajectory. In vitroassays demonstrated that COX7B expression is upregulated in activated B-cell-like DLBCL cell lines, and its knockdown significantly reduced cell viability and induced apoptosis, supporting its potential as a therapeutic target.
 The previous studies have shown that clonally expanded CD8+T cells in PCNSL undergo a dynamic transition from a pre-exhausted state to an exhausted state, exhibiting more pronounced T cell exhaustion features compared to systemic DLBCL, suggesting the presence of an immunosuppressive TME in PCNSL. However, the specific mechanisms driving T cell exhaustion and immune suppression remain incompletely understood. Recognizing the critical role of T cells in the TME and in determining therapeutic response, we further investigated the immunosuppressive landscape of T cells and regulatory mechanisms mediated by tumor cells. T cell cluster analysis demonstrated a substantial enrichment of exhausted T cells (exhTc) and regulatory T cells (TregCD4) in PCNSL. Key exhausted molecules (TIGIT, HAVCR2/TIM-3, LAG3, CTLA4, and PDCD1/PD-1) were notably upregulated. ST confirmed high expression of HAVCR2/TIM-3 in T-cell-enriched regions. Importantly, the ligands for HAVCR2/TIM-3, CTLA4 and CEACAM1 were found to be upregulated in malignant B cells, with CD80 (CTLA4 ligands) particularly enriched in B-cell regions of spatial slices. These findings suggested active involvement of mBc in shaping the immunosuppressive TME. To explore intercellular signaling, we used CellChat to construct the communication network within the PCNSL microenvironment. mBc subsets showed stronger interactions with exhTc and TregCD4 compared to other TME cell types, primarily mediated by the macrophage migration inhibitory factor (MIF) signaling pathway. CD74+CXCR4 and CD74+CD44 were identified as key ligand-receptor pairs in this pathway. Analysis of spatial slices revealed elevated expression of CXCR4 and CD74 in tumor regions, further indicating widespread activation of the MIF pathway. These findings suggest that mBc subsets mediate the immunosuppression of exhTc and TregCD4 via MIF signaling.
 In conclusion, this study reveals the intratumoral heterogeneity and immunosuppressive microenvironment of PCNSL, identifies the high-risk COX7B⁺ mBc2 subset, and indicates COX7B as a potential therapeutic target. Additionally, we clarify the key role of MIF signaling in immune suppression, offering new directions for targeted and immune therapies in PCNSL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f0ba610cbed6b065d81ef84a88bd658adbfc55a" target='_blank'>
              Single-cell and spatial transcriptomics analyses decipher malignant B cell heterogeneity and immunosuppressive microenvironment in primary central nervous system lymphoma
              </a>
            </td>
          <td>
            Cong Wang, T. Lu, Xiangrong Zheng, Ran Kong, Mei Ding, Xiaohui Sui, Xiangxiang Zhou
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common type of primary liver cancer. Tumor heterogeneity is a major obstacle to effective treatment and is poorly understood using traditional bulk sequencing methods. This review highlights the transformative role of single-cell and multi-omics technologies in determining the cellular and molecular complexities of HCC. We summarize recent advances in single-cell transcriptomics, epigenomics, multi-omics, and spatial transcriptomics platforms, emphasizing their applications in characterizing tumor subclones, cancer-associated fibroblast-immune interactions, circulating tumor cells, and immune-resistant phenotypes. Spatial approaches have revealed the architecture of cancer stem cell niches and tertiary lymphoid structures, providing unprecedented insights into tumor organization and microenvironmental crosstalk. Although still in their early stages, clinical trials have begun to incorporate these technologies, underscoring their translational potential. Single-cell and spatial omics have reshaped HCC research by enabling high-resolution profiling of tumor ecosystems and driving the discovery of biomarkers, therapeutic targets, and strategies for patient stratification. However, high cost, technical expertise, and limited accessibility, particularly in resource-constrained settings, are major barriers to its widespread adoption. Addressing these challenges is critical for translating these powerful approaches into clinical practice and for advancing precision medicine for the treatment of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0abad77c5fcae33c11d7d851ad3eb337d3881a5" target='_blank'>
              Exploring Single-Cell and Multi-Omics Technologies and Their Role in Unravelling Tumor Heterogeneity of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Charmi Jyotishi, Suresh Prajapati, Mansi Patel, Reeshu Gupta
          </td>
          <td>2025-12-17</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 The World Health Organization provides clear criteria for diagnosing IDH1-wildtype glioblastoma (GBM), relying on traditional approaches to define pathoetiological features, genomic mutations, copy number variation, and aberrant expression of prognostic genes. While these methods guide diagnosis, they may lack the resolution to fully capture tumor heterogeneity, therapeutic response, or outcomes. To address this, we applied leading-edge technologies to 14 primary and recurrent tumors to add granularity to current diagnostic metrics and clinical classification. Aiming to better inform tumor subtyping and improve patient care, this integrative approach enabled us to identify malignant gene programs and localize prognostically relevant features at both single-cell and spatial levels. We analyzed 14 GBM tumors from 6 patients using spatial in situ sequencing (ISS) and single-nuclei RNA-sequencing. A custom ISS gene panel enabled spatial mapping of tumor-associated cell types and state markers, while matched snRNA-seq from adjacent tumor tissue provided transcriptome-wide profiles. One patient contributed 5 serially collected recurrent tumors, enabling longitudinal analysis of tumor evolution. Profiling these tumors using snRNA-seq identified canonical copy number alterations (+7/–10) and ligand-receptor interaction analysis showed an evocative ANXA1 signaling axis progress across recurrent tumors, which suggests possible therapeutic opportunities. Finally, the dataset enabled subcellular and spatial localization of the prognostic GBM biomarker, EGFRvIII, being enriched in oligodendrocyte progenitor (OPC)-like tumor cells, which provides exciting evidence of these cancer cell subtypes as the putative cell of origin for GBM. By extending beyond conventional diagnostic benchmarks, our molecular profiling reveals new insight into GBM biology that may support more refined tumor subtyping and prognostic assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb079e9e9eb8a8e1b502483557634499fe4e750" target='_blank'>
              BIOM-72. INTEGRATIVE SINGLE-NUCLEI SPATIAL TRANSCRIPTOMIC PROFILING OF PRIMARY AND RECURRENT HUMAN IDH-WILDTYPE GLIOBLASTOMA REVEALS CELL-SPECIFIC EGFRVIII EXPRESSION AND ANXA1 SIGNALING AXIS
              </a>
            </td>
          <td>
            Lauren Powers, Vaibhav Jain, Michael Aksu, Emily Hocke, Alan Smith, Stephanie Arvai, Ellora Haukefrers, Kevin Stevenson, Khooshbu Patel, Diane Satterfield, Elizabeth Thomas, Karen Abramson, Giselle López, R. McLendon, David M. Ashley, Simon Gregory
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The development of novel therapeutics for deadly diseases such as glioblastoma (GBM) is bottlenecked by poor preclinical models. GBM is the most common and deadliest primary brain tumor in adults, with an average prognosis of 12–15 months, primarily due to its high cellular heterogeneity and treatment resistance from GBM stem cells. The advancement of in vitro models into organoids, three-dimensional tissue-like modeling systems, has been a promising approach to improving translational medicine for GBM. However, the critical tradeoff between technical convenience and physiological relevance threatens the integrity and reproducibility of GBM organoid (GBO) biomanufacturing. This comprehensive review breaks down and discusses the key features of GBM tumor microenvironment (TME), traces the advancement of in vitro models from two-dimensional cultures to three-dimensional stem cell-derived GBOs, evaluates the process through an engineering perspective (genetic, biochemical, biophysical, and process engineering), and addresses critical translational gaps. Reviewing trends over the last fifteen years in biomanufacturing approaches to GBOs revealed fundamental oversights that address previous review focuses on the limitations of organoids (i.e., maturity, vasculature, and immune defense). To summarize, GBO’s translational gap and reproducibility challenges are rooted in the prioritization of technical convenience over physiological relevance. To achieve clinical relevance, future GBO development must focus on transitioning to fully defined components (excluding animal-derived ECM), developing sufficiently large-sized constructs to recapitulate the full TME, and integrating non-destructive and enhanced functional readouts of the GBOs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e994f4d6a33fd45263cff1ae3c39866677c30f73" target='_blank'>
              Bioengineering Stem Cell-Derived Glioblastoma Organoids: A Comprehensive Review
              </a>
            </td>
          <td>
            Alexandra D. Avera, Yonghyun Kim
          </td>
          <td>2025-12-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Tracking small-molecule distribution in heterogeneous cell samples at single-cell resolution remains a major analytical challenge. Here, we present a tellurophene-functionalized analogue of the proteasome inhibitor Carfilzomib (TeCar) whose distribution can be followed by mass cytometric (MC) quantification while preserving target engagement and cytotoxicity. Structural and biochemical analyses confirm that TeCar binds the proteasome in a mode comparable to the clinically approved parent compound. Using MC, we demonstrate selective TeCar accumulation in malignant over immune cells within mixed populations, with cancer cells exhibiting 15 to 30-fold higher uptake. Tellurium signal correlates with proteasomal activity, and differential labeling among immune subsets reveals functional heterogeneity not captured by transcriptomics alone. These findings establish tellurophene tagging as a minimally perturbing and broadly applicable strategy for functional distribution studies at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297f01c1c4f8d9a53fbe0c9fdab22012abff3851" target='_blank'>
              Tellurophene-Tagged Carfilzomib Enables Single-Cell Mass Cytometric Mapping of Proteasome Activity.
              </a>
            </td>
          <td>
            Nicole Potter, A. Eddenden, Aleksandra Fomina, Anuroopa Dinesh, H. Jackson, A. McGuigan, M. Groll, Mark Nitz
          </td>
          <td>2025-11-18</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Effective antitumor immunity ultimately depends on the priming and activation of tumor-specific cytotoxic CD8+ T cells; however, the role of intratumoral cell-cell immune interactions remains incompletely understood. Recent work has revealed that the temporospatial colocalization of dendritic cells (DCs), helper T cells (Th), and cytotoxic T lymphocytes (CTL) within the tumor immune microenvironment following immune checkpoint blockade correlates with clinical response. Herein, we report the integration of more than one million spatially resolved single-cell profiles across six spatial proteomic and transcriptomic assays, which demonstrated that DC:Th:CTL three-cell-type clusters were common even in immunotherapy-naïve and highly desmoplastic tumors, such as fibrolamellar carcinoma and pancreatic ductal adenocarcinoma (PDAC). We found that these immune triads were enriched for functionally important type 1 conventional DC, mature DCs enriched in immunoregulatory molecules (mregDC), CXCL13+ Th, and GZMK+ effector CTL phenotypes. Subsequent multiplex immunofluorescence imaging of more than 450 primary PDAC tumors showed that the density of antigen-presenting cell (APC):Th:CTL three-cell-type clusters was correlated with intratumoral T-cell clonal expansion and improved overall survival. These findings suggest that DC:Th:CTL triads are conserved across solid tumors and highlight the importance of intratumoral spatial niches in mediating endogenous antitumor immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e981c34b939efc05dce5ef8fec4e92e8da12dbd" target='_blank'>
              Intratumoral three-cell-type clusters are a conserved feature of endogenous antitumor immunity.
              </a>
            </td>
          <td>
            Sheela R. Damle, J. A. Carter, K. E. Goodsell, Jose M B Pineda, L. K. Dickerson, Xiuyun Jiang, Jacqueline L Mudd, Thomas Walsh, H. Kenerson, Jack Cernak, S. S. B. Chauhan, Emily Beirne, Sujata Jana, Amanda L. Koehne, Kiran R Vij, Marianna B Ruzinova, R. Yeung, Shreeram Akilesh, Eric A. Collisson, Ryan C Fields, I. N. Crispe, Venu G. Pillarisetty
          </td>
          <td>2025-11-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7358fd5e226c097b2efb1a3758a0fef198cc402b" target='_blank'>
              Single cell spatial profiling of the matrisome identifies region-specific adhesion and signaling networks in glioblastoma.
              </a>
            </td>
          <td>
            Arpan De, Santiago A Forero, Ali Pirani, John E. Morales, Marisol De La Fuente-Granada, Sumod Sebastian, Jason T Huse, Leomar Ballester, J.S. Weinberg, Frederick F. Lang, Kadir C. Akdemir, Joseph H. McCarty
          </td>
          <td>2025-12-09</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a resilient subpopulation within tumors, capable of driving progression, metastasis, and recurrence. One mechanism that enables this plasticity is asymmetric cell division (ACD), a process by which CSCs generate both a self-renewing stem cell and a differentiated daughter cell. While traditionally associated with tissue development, ACD is now recognized as a dynamic and regulated feature of cancer biology, particularly in response to stress conditions such as hypoxia and radiation. In this review, we provide a comprehensive and mechanistic synthesis of how intrinsic factors such as polarity complexes, cell fate determinants (CFDs), spindle orientation cooperate with extrinsic cues from the tumor microenvironment to orchestrate ACD in CSCs. We explore how this process contributes to tumor heterogeneity, therapy resistance, and the emergence of quiescent, drug-tolerant CSCs across multiple malignancies, including brain, breast, colorectal, and hematologic cancers. Importantly, we highlight recent efforts to pharmacologically disrupt or redirect ACD using inhibitors of NOTCH, WNT, AURORA kinases, and MSI1, presenting ACD as a therapeutic vulnerability rather than a static trait. By shifting the focus from CSC markers to division mode, this review introduces a novel conceptual framework for targeting tumor hierarchy and plasticity. Understanding and manipulating ACD offers a promising frontier in precision oncology - one where altering the balance of cell fate decisions could limit relapse, reduce intratumoral complexity, and enhance long-term treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911960284166687ce9462f6da53a77bba5e7d79" target='_blank'>
              Targeting asymmetric division in cancer stem cells: implications for relapse and resistance
              </a>
            </td>
          <td>
            Mona Latifi, Mahsa Sani, F. Sani
          </td>
          <td>2025-12-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Introduction
 Classical Hodgkin Lymphoma (cHL) is one of the most manageable human cancers. The early identification of patients at risk of relapse following front-line therapy, currently represents an unsolved clinical need. cHL is a complex ecosystem characterized by an intricate and mutual interaction between an extensively dominant tumor microenvironment (TME) and malignant Hodgkin Reed-Sternberg (HRS) cells; disease progression likely reflects some innate features of this organization. Nevertheless, even the most advanced prognostic models fail to fully account for the clinical heterogeneity of the disease. A major limitation lies in the inherent difficulty of profiling HRS cells, given their rarity and unique distribution within the tumor parenchyma. In this study, we employed an integrated multidimensional transcriptomic analysis to explore the peculiar architecture of the cHL tumor ecosystem and its potential relationship with clinical progression and patient outcomes.
 Methods
 Patient cohort: A retrospective training cohort of untreated cHL patients (N=155), age < 65 years and including all disease stages, was selected. Patients were classified as either relapsed/refractory (R/R+) (N=31) or non-relapsed (R/R-) (N=124) based on a minimum follow-up of 3 years.
 A morphology-guided spatial transcriptomics approach was used to analyze the transcriptional profile of HRS and TME components in surgical lymph node resections of a matched R/R-:R/R+ case-control subset of the cohort. A total of 118 Areas of Interest (AOIs) were collected, 56 from the TME (CD45+/CD30-) and 62 from HRS-rich regions (CD30+). Digital barcoding profiling was used to assess the expression of 770 immune-related genes across the entire training cohort. Cox regression analysis was used to identify genes significantly associated with progression-free survival (PFS). IHC was used to confirm the validity of the derived model.
 Results
 Morphology-guided profiling of HRS cells showed significant differences between R/R+ and R/R- cHLs. Unsupervised clustering based on the transcriptional profile of HRS cells strongly segregated AOIs according to clinical classification. Topographical organization of HRS within the tumor parenchyma, along with paracrine signaling, accounted for HRS differential properties in the two groups. The TME in R/R+ cHLs exhibited an immunosuppressed phenotype, defined by the exclusion of non-tumoral B-cells and overrepresentation of immunosuppressive populations (M2 Macrophages, Tregs and Mast cells). This phenotype was associated with altered immune-modulatory signaling by HRS cells, involving inhibition of stimulatory mediators such as chemotactic cytokines (CXCL9, CXCL10, etc) and IFN signaling and the upregulation of inhibitory molecules including CCL17 (encoding for TARC) and IL13. An inverse correlation was observed between the presence of non-tumoral B-cells and immunosuppressive populations. Digital barcoding gene expression analysis on the entire cohort (N=155) confirmed this model and identified a non-tumoral B-cell specific gene signature (N=18) that significantly correlated with improved PFS and stratified patients according to risk of progression (HR 0.23 95% CI 0.06-0.87; p=0.03)
 Conclusion
 This study provides, one of the first molecular characterization of HRS cells and of their interplay with the surrounding TME, uncovering mechanisms that drive immune escape and disease progression. Notably, our findings reveal a previously unrecognized protective role of non-malignant B cells residing in the TME of cHL, highlighting their potential utility as biomarkers for risk stratification in cHL. Finally, our data demonstrate that integrating high-resolution transcriptional profiling with morphology enables a more precise understanding of cHL complexity, capturing not only intrinsic variability but also the reciprocal spatial organization between cell populations, which significantly shape the biological properties of the entire tumor ecosystem.
 REFERENCES
 1. Scott, DW &Gascoyne, RD The tumour microenvironment in B cell lymphomas. Nat. Rev. Cancer 14, 517–534 (2014)
 2. Aoki, T. et al. Single-cell transcriptome analysis reveals disease-defining t-cell subsets in the tumor microenvironment of classic hodgkin lymphoma. Cancer Discov.10, 406–421 (2020)
 3. Aoki T. et al. Spatially Resolved Tumor Microenvironment Predicts Treatment Outcomes in Relapsed/Refractory Hodgkin Lymphoma. J Clin Oncol. doi: 10.1200/JCO.23.01115. (2024)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e25acab9e1dc7d742c876b8b8b809c99b9c1c2e9" target='_blank'>
              Morphology guided spatial transcriptomics to study topography of Hodgkin reed sternberg cells interaction with tumor microenvironment and its impact in defining risk of progression in classical Hodgkin lymphoma.
              </a>
            </td>
          <td>
            B. Donati, T. Lazic, Riccardo Valli, R. Durmo, Attilio Gennaro, Cristian Ascione, Pierluigi Di Chiaro, A. Ruffini, M. Gotti, Luca Arcaini, Erica Travaglino, Marco Paulli, Francesco Merli, A. Ciarrocchi, S. Luminari
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains a highly aggressive brain tumor with limited therapeutic options and significant immune evasion. Tumor-associated regulatory T cells (Tregs) and macrophages (TAMs) are central orchestrators of the GBM immune microenvironment; however, the dynamics and subtype-specific interactions between these populations during disease progression are still poorly understood. In this study, we utilized complementary mesenchymal and proneural murine GBM models to examine the differentiation trajectories and functional crosstalk of Tregs at single-cell resolution. Single-cell RNA sequencing (scRNA-seq) of tumor-infiltrating Tregs revealed that mesenchymal GBM promoted the expansion of an immunosuppressive Treg population characterized by elevated CCR8 (C-C chemokine receptor type 8), GITR (Glucocorticoid-Induced TNFR-Related protein), and IL1R2 (Interleukin 1 Receptor Type 2), which was less pronounced in proneural GBM. Trajectory analysis uncovered enhanced proneural-to-mesenchymal transition signatures, indicating dynamic Treg adaptation to evolving tumor states. Validation in human spatial transcriptomic GBM datasets revealed that the murine tumor Treg signature was enriched within mesenchymal regions and closely associated with myeloid-dense microenvironments. By leveraging a time-course scRNA-seq dataset from the mesenchymal GBM model, we observed a progressive expansion of the mesenchymal-like Treg signature paralleling tumor evolution and mesenchymal transition. Notably, CellChat-based cell–cell communication analysis demonstrated that Treg–myeloid interactions were not only enhanced over time but also that Treg subsets differentially engaged with immunosuppressive TAMs through unique ligand–receptor pairs, underscoring the dynamic and reciprocal adaptation between regulatory and myeloid compartments. Collectively, our study delineates a model in which GBM progression is shaped by the evolving interplay of Treg and myeloid programs, providing a framework for targeting context-specific immunosuppressive circuits in future immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a27d9dd70e278b9475e8a73470ac145030c21c" target='_blank'>
              IMMU-86. Dynamic Treg-myeloid crosstalk dictates glioblastoma progression
              </a>
            </td>
          <td>
            Sadashib Ghosh, Jeffrey M. Grimes, Stephen Opoku, J. Leavenworth
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="


 Adoptive T cell therapies represent a major advance for patients with otherwise-refractory blood cancers. Yet patient responses can vary widely, due in part to the inherent heterogeneity of the manufactured T cell product. Certain T cell subsets/phenotypes in the infusion product can predict clinical efficacy, but it is unclear how and where these desirable cells arise during therapeutic T cell expansion. We investigated the contribution and fate of multiple T cell subsets (naive, central-memory, effector-memory) expanded alone versus together in a pool towards the final T cell product. We used lentivirus-based proteomic barcodes and mass cytometry to follow these populations over time. When expanded together in a pool, all three subsets displayed a greater degree of terminal differentiation and effector cytokine production than when expanded alone. However, only naive CD8 T cells expanded in the pool showed enhanced degranulation activity (CD107a). Meanwhile, pooled naive and effector-memory T cells, but not central-memory T cells, exhibited greater glycolytic activity and oxidative phosphorylation as measured by single-cell metabolic regulome profiling. This coincided with a restoration of their cell cycle progression by the end of an 11-day expansion, compared to their counterparts expanded alone which gradually returned to G0/G1 over time. Overall, these results suggest that pooled expansion may only be beneficial for certain subsets for T cell therapies.



 Tumor Immunology: Checkpoints, Prevention, and Treatment (TIPT)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c5adabc46309c5649e66cbe749b8ff0e825e8e8" target='_blank'>
              Fate mapping of human T cell subsets during expansion for T cell therapy 3264
              </a>
            </td>
          <td>
            A. Lam
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Secreted phosphoprotein 1 (SPP1+) macrophages are a recurrent and functionally critical immune cell subset across multiple cancer types. They drive adverse clinical outcomes by promoting immunosuppression, tumor invasion, metastasis, and therapy resistance. Given their prevalence and pivotal role, SPP1+ macrophages have become a major focus in cancer immunology and a promising target for therapeutic development. SPP1+ macrophages have been identified in a wide range of human malignancies through single-cell RNA sequencing and spatial transcriptomics studies. Their differentiation and maintenance are strongly influenced by reciprocal cellular interactions and hypoxic conditions within the tumor microenvironment (TME). Within the tumor microenvironment (TME), SPP1+ macrophages promote tumor progression by interacting with cancer-associated fibroblasts (CAFs) and helping to form a physical barrier that restricts immune cell infiltration into the tumor core. Specifically, they impair the recruitment of CD8+ T cells and promote T cell exhaustion (TEX). In this review, we focus on recent advances in understanding the differentiation of SPP1 macrophages in hypoxic tumor microenvironment and the role of SPP1+ macrophages in immunosuppression and their therapeutic implications in cancer. Targeting this subset of macrophages has emerged as a highly promising therapeutic strategy, with several approaches demonstrating encouraging results in preclinical models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c7489149a391c5b3e442dad42f10334718f488" target='_blank'>
              SPP1+ macrophages in tumor immunosuppression: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Juanjuan Wang, Ya Wang, Yuqing Liu, Rongcun Yang
          </td>
          <td>2025-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Intratumoral heterogeneity supports cancer progression and metastasis while limiting therapeutic efficacy. Comprehensive biological characterization of heterogeneous subclones in breast cancer may hold promise in battling against this deadly disease. Using single-cell transcriptomic analysis and in situ profiling of primary tumors, metastases, and circulating tumor cells from multiple patients with breast cancer, we identified a unique EPCAM+ CD68+ TREM2+ tumor subpopulation, likely resulting from the fusion of tumor cells and lipid-associated macrophages (LAM). The presence of these tumor-LAM fusion cells in the blood or in distinct metastatic sites was significantly correlated with metastatic progression. Stable fusion clonal lines established in vitro exhibited substantially enhanced proliferation, tumor initiation, and metastasis formation in mice. Integrative molecular and functional analyses revealed a critical role for SNX10 in mediating tumor-LAM fusion. Mechanistically, SNX10 physically interacted with the phospholipid scramblase ANO6 and maintained its protein stability by suppressing proteasome-mediated degradation. Furthermore, the transcription factor CBX3 directly targeted SNX10 promoter and modulated its expression. Fusion cells accumulated abundant intracellular lipid droplets and were highly sensitive to simvastatin treatment in vitro and in vivo. Together, this study uncovered that CBX3-SNX10-ANO6 signaling facilitates generation of an aggressive tumor-LAM fusion cell subpopulation that promotes metastasis, revealing an alternative metastatic mechanism and exposing putative therapeutic vulnerabilities.


SIGNIFICANCE
Single-cell transcriptomic profiling combined with functional and clinical validation identifies fusion of tumor cells and lipid-associated macrophages mediated by the CBX3-SNX10-ANO6 axis as a potentially targetable mechanism driving cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4271e1aeed14ff3c28d665da96c160262a128c0a" target='_blank'>
              Fusion of Tumor Cells with Lipid-Associated Macrophages Drives Metastatic Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Yixin Cheng, Guanyin Huang, Xuefei Liu, Chun Wu, Jingru Lian, Jing Cao, Lianhui Duan, Boxi Zhao, Yufan Yang, Siqi Chen, Guanqiao Li, Liping Wu, Shuqian Zheng, Wenjing Wang, Tianyu Zha, Yang Yang, Yilin Lu, Jian Zhang, Hao Yu, Jun Tan, Feiqiu Wen, Zhenyu He, Xin Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Spatial biology is rapidly emerging as a powerful tool for deepening our knowledge of the complex, dynamic nature of tissue-specific immune responses. It involves investigating the interactions between lymphoid cells, antigen-presenting cells, and non-lymphoid cells within tissues. Multiplex immunohistochemistry (IHC) and immunofluorescence (IF) are ideal techniques for characterizing the spatial aspects of immune responses. To streamline accurate marker detection, emerging automated staining and imaging platforms require qualified antibodies with platform-specific and proven specificity and sensitivity. Using IHC antibodies from R&D SystemsTM, we used a systematic process to qualify antibodies for use on the Lunaphore COMET™ platform in murine lymphoid tissue. COMETTM offers a novel sequential immunofluorescence (seqIF™) approach that utilizes cycles of staining, imaging, and elution with unconjugated primary antibodies and fluorescently labeled secondary antibodies. In this study, we distinguish important cell types, including pan-cytokeratin+ epithelial cells, CD45+ hematopoietic cells, and CD31+ endothelial cells. Simultaneously, we can further characterize hematopoietic cells using phenotypic markers, including Ki67, the regulatory transcription factor FoxP3, and the activation marker CD86. This approach provides a powerful platform for characterizing tissue-specific immune responses, differentiating disease states, and identifying potential therapeutic targets.



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cafcb0d97c16e1cc46bb1d13d4f71af1e136bee6" target='_blank'>
              Spatial Distribution and Characterization of Mouse Lymphoid Tissue Using Fully Automated Multiplex Immunofluorescence on COMETTM 3909
              </a>
            </td>
          <td>
            Ruha Adelkar, Emily K Cartwright, A. Kalyuzhny, Jodi Hagen, Ashley Oliver, Brandon Wagner
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Diffuse midline gliomas (DMGs) are universally fatal pediatric brain tumours defined by their midline localization, early onset, and distinct mutational landscape. The H3.3K27M mutation is found in approximately 80% of DMGs, marking it as a key initiating event. However, H3.3K27M alone does not induce brain tumour formation in experimental models, suggesting that its oncogenic potential may be linked to specific developmental windows and susceptible precursor populations. Identifying the cellular states that are vulnerable to transformation is critical to understanding DMG initiation. To establish a reference framework for these vulnerable states, we generated a single-cell RNA sequencing atlas of normal hindbrain development. Using a Sox2-eGFP reporter mouse and FACS-based enrichment of Sox2+ precursors, we profiled over 120,000 cells across ten developmental stages, capturing critical transitions in neurogenesis and gliogenesis. To reveal regional differences, we integrated the atlas with a similarly generated forebrain atlas, providing a comprehensive map of neurodevelopmental trajectories where DMG mutations might exert their effects. Building on this developmental reference, we utilized inducible mouse models to introduce H3.3K27M mutations in precursors and employ lineage tracing to track their fate over time. When combined with loss of Trp53 and overexpression of PDGFRa, alterations prevalent in DMG, mice develop tumours that resemble the human disease. H3.3K27M-driven tumours consistently localize to the pons, in contrast to H3 wild-type tumours, which occur more variably across the forebrain and hindbrain. Tracking these pontine tumours using serial MRI revealed that tumours arise well before symptom onset and exhibit growth dynamics that vary by location and timing. Earlier-arising tumours are more aggressive and tend to occur in posterior locations. Together, our integrated approach combining lineage tracing, tumour modelling, and a developmental atlas provides a powerful platform to dissect the early events in DMG pathogenesis and identify developmental vulnerabilities that may be targeted for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0a1a4d1a35cbc94502034b6562096dc3dd8efc6" target='_blank'>
              STEM-27. TRACING THE ORIGINS OF H3.3K27M MUTANT DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Caitlin Herrington, Evan Lubanszky, Quang Trinh, Luiza Lopes Pontual, Jannine Forst, Cynthia Hawkins, Peter Dirks
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba29f656998508469f49c53f9437fe122dadd98" target='_blank'>
              DYRK1A regulates cancer cell and cancer-associated fibroblast secretomes to foster an immunosuppressive microenvironment in pancreatic cancer
              </a>
            </td>
          <td>
            Silvia Pascual-Sabater, Sushmita Varhadi, Chiara Di Vona, Marta Celma, Cristina Fillat, S. de la Luna
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 The tumor microenvironment is highly heterogeneous, composed of diverse cell types with distinct phenotypes and characteristics. Gaining insights into these differences is essential for advancing cancer immunotherapy and research. Reliable results demand high-quality samples and standardized processing, however, generating single-cell suspensions from solid tumors is challenging due to variations in tumor type, size, texture, and developmental stage. Balancing enzymatic and mechanical dissociation is critical to preserve cell viability and yield. We developed the STEMprep™ Mouse Tumor Dissociation Kit for use with the STEMprep™ Automated Tissue Dissociation System, featuring a user-friendly instrument and specially designed sample tubes. This system includes an optimized enzyme formulation and a single protocol that works across diverse tumor types. Our findings demonstrate consistently high cell viability and yield (cells/mg tissue) across tumors: soft B16 melanoma (91.2 ± 3.3% and 1.5E4 ± 7.4E3 [n = 15]), medium-firm CT26 colon carcinoma (82.9 ± 6.1% and 2.6E4 ± 1.3E4 [n = 9]), and firm 4T1 mammary carcinoma (88.5 ± 4.0% and 2.7E4 ± 1.7E4 [n = 4]). This method maintains cell integrity and epitope availability for downstream analysis. Furthermore, STEMprep™-processed samples are compatible with EasySep™ magnetic cell isolation technology, and are functional in T cell suppression assays, enhancing tumor research workflows with efficiency, consistency, and reliability.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15528cd65eafa51df7aab4672196a35521811783" target='_blank'>
              Streamlined mouse tumor processing with STEMprep™: Automated, Efficient, and Reliable 3260
              </a>
            </td>
          <td>
            F. Antignano, G. Poon, So Ra Lee, Jenna Thompson, A. Chenery, Alice Liang, Payam Zachkani, Andrew Nobles, Bilal El yassem, Chris Ryan, Marc Delorme, Rodrigo Martins, Martin O’Keane, A. Kokaji, A. Eaves, Sharon Louis
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) contains metabolically distinct cell lineages: treatment-sensitive cells (TSCs) dependent on glycolysis and treatment-resistant persister cells (TRPCs) reliant on lipid metabolism. Our integrative systems biology approach, combining spatial multi-omics and targeted analyses, uncovers lipid acquisition in TRPCs as a pivotal driver of GBM progression which is supported by a retrospective study of statin-treated patients, providing clinical evidence for the therapeutic potential of targeting lipid metabolism in GBM patients stratified based on high TRPC signatures.



 We applied spatial transcriptomics and proteomics combined with 3D reconstruction of the tumor microenvironment to map TRPC- and TSC-enriched regions. Immunohistochemically stained tissues were analyzed using CellProfiler and our custom SNAQ algorithm for single-cell neighborhood analysis. Lipid transfer from Tumor-Associated Myeloid Cells (TAMCs) to TRPCs was tracked via flow cytometry, holotomography, and time-lapse imaging. Genetic and pharmacological targeting of CCR2, CSF1R, and FABP3 was performed in vitro and in vivo. Finally, a retrospective analysis of 4,085 GBM patients assessed the impact of statin treatment in relation to TRPC signatures.



 TRPCs exploit recruited TAMCs to acquire lipids, supporting their survival and immune evasion. These interactions are mediated by CCL2/7–CCR2 and CSF1–CSF1R signaling. Disruption of TRPC-TAM lipid homeostasis significantly remodeled the immune microenvironment, reduced tumor growth, and extended survival in preclinical models. Patients with high TRPC signatures typically had poor outcomes; however, retrospective analysis revealed that statin treatment was associated with improved survival in this subgroup. These findings position lipid trafficking as a metabolic vulnerability in GBM and highlight the therapeutic relevance of targeting TRPC–TAMC interactions.



 Our study reveals lipid trafficking to TRPCs as a key mechanism driving GBM progression. Targeting TAMC-mediated lipid transfer disrupts immune suppression and tumor growth. Transcriptomic profiling may guide patient stratification, identifying those likely to benefit from lipid metabolism-targeted therapies such as statins.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6655254013c2e00179a9a7336a71e7fa3aca627" target='_blank'>
              TMIC-83. Targeting Lipid Metabolism in GBM: A Systems Biology Strategy to Overcome Treatment-Resistant Persister Cells and Their Myeloid Cell-Mediated Support
              </a>
            </td>
          <td>
            Avirup Chakraborty, Changlin Yang, Diana Feier, Aryeh Silver, Nyla T. Searl, Miruna Anica, Avinash Pittu, Illeana West, Olusegun O. Sobanjo, Ethan D. Hodge, Mia K. Engelbart, Srabasti Sengupta, Nagheme Thomas, C. V. von Roemeling, Maryam Rahman, Jiangping Huang, Jeffrey K Harrison, D. Mitchell, Matthew R. Sarkisian, L. Deleyrolle
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Pancreatic neuroendocrine tumors (panNETs) undergo high-grade (HG) transformation often, with the liver being a common site for metastasis​. Surgical resection being considered as long-term outcomes for selected patients. However, recurrence and grade transformation remain potential significant challenges, with recurrence rates approaching 50–95%.1,2 Despite extensive whole exome analysis and construction of phylogenetic trees, we were unable to identify either a consistent genetic driver or any specific patterns of tumor transformation.2 We are also concerned that some of our treatments may inadvertently contribute to grade transformation. To assess the treatment effects, we performed pre- and post- trans arterial embolization bulk-transcriptome analysis, which revealed cell population shifts in panNETs that may be linked to grade transformation.4To explore cellular shifts within the liver tumor microenvironment during panNET progression, we performed single-cell RNA sequencing. The resulting data was processed using Seurat.v5. The clusters of panNET cells, fibroblasts, hepatocytes, and immune cells were annotated by known markers. Additionally, we distinguished malignant cells from a reference of immune cells, fibroblasts, and hepatocytes by InferCNV. To understand the tumor progression, we performed trajectory analysis using monocle3. Differential metabolic activity between the panNETs grades were assessed using the scMetabolism package. Cell-cell communication and related ligand-receptor pairs were inferred using CellChatDB. Our results revealed distinct transcriptional trajectories, with pseudotime analysis demonstrating a trajectory of tumor evolution from low-grade (LG) to HG states. HG tumors were enriched for proliferative and aggressive subclusters compared to LG cells. Metabolic profiling further highlighted grade-specific adaptations, with HG tumors showing enhanced oxidative phosphorylation and TCA cycle across tumor cells, whereas LG tumors displayed relatively lower metabolic activity. Notably, immune cells were found metabolically active in LG via glycolysis. Conversely, HG panNETs exhibit glucose depletion in immune cells and less active microenvironment. The CellChat analysis revealed distinct cellular communication patterns in panNET grade. For instance, in HG tumors, signaling was primarily mediated between fibroblasts and panNET cells, with Glu-(SLC1A2+GLS)-GRM7 as the dominant ligand–receptor pair. In LG tumors, major interactions occurred among immune cells, hepatocytes, and panNET cells, with Glu-(SLC1A3+GLS)-GRIK2 representing the highest contributor. The results suggest that metabolic rewiring, altered intercellular signaling, and transcriptional plasticity underlie progression from LG to HG disease.
 Reference:
 1. Spolverato etal J Gastrointest Surg. 2017, 21(12):2039-2047
 2. Mayo etal Ann Surg Onc. 2011, 18(13):3657-65.
 3. Singh et al. Early ... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium. 2023.
 4. Singh et al. Post... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium.2023



 Himanshu N. Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jungeui Hong, Nitya Raj, Olca Basturk, Zeynep Tarcan, Christine A. Iacobuzio, Etay Ziv. Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f19a2d46016dbf4a94424ac84c3f39682a28b2d" target='_blank'>
              Abstract B025: Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors
              </a>
            </td>
          <td>
            Himanshu Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jung-Pyo Hong, Nitya Raj, O. Basturk, Z. Tarcan, Christine A. Iacobuzio, E. Ziv
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Gliomas and brain metastases are associated with poor prognosis, necessitating a deeper understanding of brain tumor biology and the development of effective therapeutic strategies. Although our group and others have demonstrated microbial presence in various tumors, recent controversies regarding cancer-type-specific intratumoral microbiota emphasize the importance of rigorous, orthogonal validation. This prospective, multi-institutional study included a total of 243 samples from 221 patients, comprising 168 glioma and brain metastases samples and 75 non-cancerous or tumor-adjacent tissues. Using stringent fluorescence in situ hybridization, immunohistochemistry and high-resolution spatial imaging, we detected intracellular bacterial 16S rRNA and lipopolysaccharides in both glioma and brain metastases samples, localized to tumor, immune and stromal cells. Custom 16S and metagenomic sequencing workflows identified taxa associated with intratumoral bacterial signals in the tumor microenvironment; however, standard culture methods did not yield readily cultivable microbiota. Spatial analyses revealed significant correlations between bacterial 16S signals and antimicrobial and immunometabolic signatures at regional, neighborhood and cellular levels. Furthermore, intratumoral 16S bacterial signals showed sequence overlap with matched oral and gut microbiota, suggesting a possible connection with distant communities. Together, these findings introduce microbial elements as a component of the brain tumor microenvironment and lay the foundation for future mechanistic and translational studies. A combination of high-resolution spatial imaging, spatial proteomics and transcriptional data reveals sparse and heterogeneous bacterial signals in gliomas and brain metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae0d1782181d2f2125dbd26294df4180d5bf9d82" target='_blank'>
              Microbial signals in primary and metastatic brain tumors
              </a>
            </td>
          <td>
            Golnaz Morad, Ashish V Damania, Brenda Melendez, Bharat Singh, Fabiana J. Veguilla, Rebecca A. Soto, Yasmine M. Hoballah, P. Sahasrabhojane, M. Wong, Mona M. Ahmed, Rene N. Rico, Kaitlyn N. Lewis, Khalida M Wani, D. Shamsutdinova, Rossana N. Lazcano Segura, D. Ingram, Eric A. Goethe, Abderrahman Day, Ivonne I Flores, Lauren K McDaniel, M. Chelvanambi, Sarah B. Johnson, Florentia Dimitriou, Pravesh Gupta, Shivangi Oberai, M. Zal, Phoebe Doss, Mohamed A Jamal, Eiko Hayase, Chetna Wathoo, Lisa M. Norberg, Stephanie L. Jenkins, Sara Nass, J. Gumin, L. Long, Jing Yang, Gina R. Bradley, M. Bekal, A. Dono, P. Pichardo-Rojas, Samuel W. Andrewes, Leomar Ballester, J. Losh, Jiyong Liang, Longfei Huo, Douglas Nielsen, Brittany C Parker Kerrigan, Priscilla K. Brastianos, N. Fowlkes, Chia-Chi Chang, Robert R. Jenq, C. Gomez-Manzano, Jason T Huse, Michael A. Davies, A. Lazar, Krishna P. Bhat, Nitin Tandon, Y. Esquenazi, Christine B. Peterson, Vinay K. Puduvalli, Frederick F. Lang, Christopher D. Johnston, Susan Bullman, N. Ajami, S. Ferguson, J. Wargo
          </td>
          <td>2025-11-01</td>
          <td>Nature Medicine</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive primary brain tumor, characterized by profound intratumoral heterogeneity. Recent studies have identified neural stem cells (NSCs) in the subventricular zone (SVZ) as potential cells-of-origin harboring early driver mutations. However, the stepwise transformation of these rare, mutation-bearing NSCs into heterogeneous tumor populations remains poorly understood. In this study, using a spontaneous somatic mouse model that recapitulates human gliomagenesis, as well as tumor-free SVZ tissues from GBM patients, we identified transcriptionally distinct precancerous cells harboring oncogenic mutations. These cells emerge through oligodendrocyte progenitor-like lineage specification and exhibit aberrant transcriptional programs, including dysregulated translation and extracellular matrix remodeling. Single-cell transcriptomic analysis revealed transitional cell states and progressive acquisition of gliomagenic programs, leading to the emergence of molecularly diverse malignant cell populations. Our findings demonstrate that these OPC-like precancerous cells serve as a critical intermediate in GBM evolution and a driver of intratumoral molecular heterogeneity. This work provides a single-cell atlas of early gliomagenesis and identifies a potential window for therapeutic intervention targeting precancerous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f90544542d0fd8ecacde0e88ce21d50dda39f62" target='_blank'>
              EPCO-19. PRECANCEROUS OPC-LIKE CELLS DRIVE GLIOBLASTOMA EVOLUTION AND SHAPE INTRATUMORAL HETEROGENEITY
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, J. Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, Jeong Ho Lee
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e555b898cb0281329d778a02e7fea08e0c7964" target='_blank'>
              Single cell transcriptional perturbome in pluripotent stem cell models.
              </a>
            </td>
          <td>
            Elisa Balmas, M. Ratto, Kirsten E. Snijders, Silvia Becca, Carla Liaci, Irene Ricca, Giorgio R Merlo, R. Calogero, Luca Alessandrì, Sasha Mendjan, Alessandro Bertero
          </td>
          <td>2025-12-10</td>
          <td>Molecular systems biology</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="
 Brain metastases (BM) from lung adenocarcinoma represent a formidable clinical challenge due to limited therapeutic efficacy. To gain insight into complex tumor microenvironment (TME) of BM, we analyzed single-cell RNA-sequencing data comprising 66,769 cells from primary tumors (PTs; 44,355 cells) and matched BM lesions (22,414 cells). Our analysis revealed a profound shift in cellular composition during metastasis: while T cells predominated PT (44.12%), tumor cells became the most abundant population in BM (55.17%), with corresponding reductions in T cells (27.57%) and macrophages (19.3%). Cell–cell communication analysis identified fibroblasts as central mediators in both PT and BM, with a transition from collagen signaling in PT to thrombospondin signaling in BM, facilitating interactions with tumor cells, B cells, and oligodendrocytes. Ingenuity Pathway Analysis revealed enrichment of collagen-related genes in fibroblasts (MFAP4, COL15A1, FBLN1, FBLN2) and Th1/Th2 pathway genes in T cells (IKZF1, SH2D1A, RASGRP1, SKAP1). Subclustering uncovered seven distinct fibroblasts subpopulations and a notable shift from PT-enriched C0 to BM-enriched C1, suggesting dynamic phenotypic adaptation. Pseudotime trajectory analysis further clarified fibroblasts differentiation states. T-cell subclustering revealed a significant increase in immunosuppressive regulatory T cells (84.1%) and proliferating CD4+ T cells (10.5%) in BM, highlighting mechanisms of immune evasion. COL15A1 expression correlated with poor patient survival and emerged as a promising therapeutic candidate, showing increased sensitivity to Dabrafenib, Trametinib, Piperlongumine, Cabozantinib, Staurosporine, and Vemurafenib. Additional gene drug associations were observed, with FBLN1, RASGRP1, MFAP4, SH2D1A, and IKZF1 linked to potential sensitivity to Erlotinib, Afatinib, Fluorouracil, and Brefeldin A. These findings provide a high-resolution map of the BM TME and highlight COL15A1 and related pathways as potential targets for precision therapy in patients with BM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc3a5b204d80d9cd5d66d31480f9cf9dd3274795" target='_blank'>
              TMIC-44. Fibroblast reprogramming and immune evasion define the tumor niche in brain metastases
              </a>
            </td>
          <td>
            Sanskriti Dey, J. Chuang, Chih-Yang Wang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Introduction : Chronic graft-versus-host disease (GVHD) is a leading cause of late non-relapse mortality following allogeneic hematopoietic cell transplantation, yet spatial biomarkers capable of predicting outcomes remain lacking. Minor salivary glands (MSGs) are anatomically accessible organs that can be easily accessed by fast and minor invasive biopsies and reflect systemic immune perturbations, making them ideal for spatial profiling of immune diseases. We applied STARComm, a spatial multiomics framework that identifies Multicellular Interaction Modules (MCIMs), which represent tissue microenvironments where multiple cell types engage in coordinated signaling. MCIMs capture clinically relevant hubs of immune communication and tissue remodeling, enabling spatial resolution of pathogenic processes in GVHD.
 Methods : Thirty-six MSG tissues were collected from patients enrolled between 2018 and 2022 at the Hospital das Clínicas, University of São Paulo. All patients had undergone allogeneic bone marrow transplantation and subsequently received MSG biopsies as part of clinical follow-up for chronic GVHD assessment. The cohort included healthy controls (n=8), GVHD survivors (n=18), and patients who died from GVHD-related complications (n=10), with detailed clinical metadata. Spatial multiomic profiling was performed on the same histological section using a dual-modality approach: spatial transcriptomics with Xenium (10x Genomics) using a targeted 280-gene panel, and spatial proteomics with Phenocycler Fusion (Akoya Biosciences) using a 36-antibody multiplex panel. MCIMs were computed via STARComm by integrating local communication density maps across spatial grids. Cell types were annotated with TACIT, and differences in MCIM composition were compared across groups. Prognostic associations were modeled using Cox regression. To prioritize therapeutic targets, we applied SpatialDrug2Cell, a single-cell-based pipeline for inferring druggable dependencies, to spatially segmented cell populations, adapting it for high-resolution in situ data.
 Results:A total of 842,154 spatially resolved cells were analyzed across all samples. Receptor–ligand interactions were inferred from 17 curated gene pairs selected for biological relevance to GVHD and compatibility with both transcriptomic and proteomic detection, based on expression specificity and spatial proximity. Nine MCIMs were identified. In healthy MSGs, MCIMs reflected structured stromal, epithelial, and vascular signaling. GVHD samples exhibited marked reorganization, with MCIMs-0, -2, and -6 significantly enriched (p<0.001), particularly within TLS-like periductal niches. MCIM-0, dominant in fatal cases, exhibited dense B and T cell, T and T cell, and stromal interactions, with high spatial density of CXCL12-CXCR4 and CCL5-SDC4 signaling, primarily transcribed by fibroblasts and pericytes. In contrast, MCIM-3, enriched in fibroblast–endothelial interactions and B cell regulatory networks, was more abundant in survivors. In multivariate analysis, MCIMs-0 and -6 were independently associated with increased mortality (HR=2.08 and 2.83), while MCIM-3 showed a protective trend (HR=0.51). SpatialDrug2Cell, applied to MCIM-localized cell populations and spatially restricted therapeutic targets, highlighting candidate agents such as rituximab, atezolizumab, and plerixafor.Conclusions:Our analysis establishes spatially resolved MCIMs as prognostic biomarkers in chronic GVHD. The emergence of high-risk communication modules in MSGs, particularly in patients with fatal outcomes, reveals tissue-localized immune architectures associated with disease progression. By integrating STARComm with adapted drug inference tools like SpatialDrug2Cell, we identify clinically relevant, spatially anchored therapeutic targets in routinely sampled tissues. This framework may enable mechanism-based stratification and therapeutic guidance in GVHD and other immune-mediated diseases
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1daff8734fed12743fa31dcf5eea76ecab05128e" target='_blank'>
              Spatial communication modules identify prognostic cell-cell signaling hubs in chronic graft-versus-host disease
              </a>
            </td>
          <td>
            Bruno Fernandes Matuck, Khoa L. A. Huynh, Deiziane de Souza, G. Fatobene, Luiz Valente Soares Junior, L. F. Ferraz da Silva, Marisa Dolhnikoff, V. Rocha, Jinze Liu, Kevin Byrd
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer is a highly heterogeneous disease, with many cancers containing multiple distinct subclones. While subclones are often seen as competitors (survival of the fittest), intratumor heterogeneity can also offer direct benefits to the tumor through cooperation between different clones. This has important clinical implications, as interdependent populations may present therapeutic vulnerabilities. Here, we review existing evidence for clonal cooperativity to address key questions and outline future developments based on six overarching principles: (a) secreted factors are important mediators of clonal cooperation; (b) (very) small subclones can significantly affect tumor behavior; (c) both genetic and nongenetic heterogeneity are substrates for cooperation; (d) nonmalignant cells from the tumor microenvironment can act as cooperating partners; (e) clonal cooperation occurs throughout different stages of cancer, from premalignancy to metastasis; and (f) clonal cooperation can promote therapy resistance by protecting otherwise sensitive populations. Together, these principles suggest clonal cooperation as an important mechanism in cancer. Lastly, we discuss how novel technological developments could address remaining gaps to open up new therapeutic strategies that exploit clonal cooperativity by targeting the tumor's weakest link.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb39ce21ba07d76bffe91c932b77c8c8a7e6988d" target='_blank'>
              Strength through diversity: how cancers thrive when clones cooperate.
              </a>
            </td>
          <td>
            Marije C. Kuiken, Maartje Witsen, E. Voest, K. Dijkstra
          </td>
          <td>2025-11-09</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) progression and recurrence are often driven by therapy-resistant cellular states emerging within hypoxic tumor niches. While hypoxia is a hallmark of GBM, the spatial transcriptomic heterogeneity that underpins resistance in these regions remains poorly characterized. Here, we employ pimonidazole labeling in conjunction with high-resolution spatial and single cell transcriptomics across 29 primary, treatment-naive human GBM samples, syngeneic mouse models, and patient-derived glioma-initiating cells exposed to hypoxia to define the hypoxia-specific tumor microenvironment and its associated cellular programs. We reveal spatially organized sub-niches within hypoxic regions, each supporting diverse cellular programs yet coexisting tumor lineage states with minimal transitions between cell states. Phylogenetic reconstruction based on inferred copy number variations demonstrates consistent evolutionary trajectories from non-hypoxic to hypoxic tumor loci. Specific copy number alterations were enriched in hypoxic regions and associated with immune infiltration and poor prognosis in MGMT-methylated tumors. Cross-species analyses confirmed the conservation of hypoxia-associated gene programs and spatial architectures. Together, our study delineates the cellular, evolutionary, and spatial complexity of hypoxia in GBM, identifies hypoxia-induced genomic alterations that may promote tumor fitness, and offers insights into new therapeutic vulnerabilities within hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec812f9197d64f5c26991c527a5afc02eb1748" target='_blank'>
              TMIC-96. Integrated multi-omics reveals distinct hypoxic sub-niches and associated epigenomic and genomic alterations in glioblastoma
              </a>
            </td>
          <td>
            S. Mansouri, Jefferey Liu, Vikas Patil, Ruth Isserlin, Phoebe Lombard, Ronald Wu, Julio Sosa, S. Gandhi, Y. Ellenbogen, O. Singh, Hafsah Ali, C. Velásquez, Bradly Wouters, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Tumor microenvironments are complex and host diverse cell types and interactions. In-depth characterization of these environments requires detecting multiple biomarkers within the same sample, which is limited by traditional fluorescent immunohistochemistry (IHC) due to spectral constraints. Further, many high-plex IHC methods are time-consuming or require multiple imaging rounds. Here we developed a one-step multiplex labeling approach using a cocktail of primary antibody conjugates combined with spectral unmixing, enabling the simultaneous detection of 8 biomarkers and a nuclear counterstain. This streamlined approach offers high-plex biomarker detection within a timeframe similar to traditional single-antibody IHC workflows. Using the EVOS S1000 Spatial Imaging System and quantitative image analysis software, we identified multiple distinct myeloid (CD68+), lymphoid (CD20+, CD4+, FOXP3+, CD8+), and proliferating (Ki67+) cell subpopulations in normal and tumor colon tissue samples and compared their abundance and spatial distribution in relation to tissue architecture markers for smooth muscle actin and pan-cytokeratin. Additionally, we analyzed the cellular composition of a tertiary lymphoid structure in the colon tumor, believed to play a role in the immune response to colorectal cancer. These data highlight the advantages of high-plex biomarker detection for phenotyping complex tumor microenvironments using our streamlined labeling workflow and spectral unmixing approach.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d287e09a1ed64337a052860f72c1d3cae8bc8f" target='_blank'>
              Cellular phenotyping of tumor microenvironments with a high-plex fluorescent labeling and spectral unmixing approach 9333
              </a>
            </td>
          <td>
            Austin Harvey, Oggie Golub
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Pineoblastoma (PB) is a clinically aggressive embryonal central nervous system (CNS) tumor composed of distinct molecular subgroups. The cellular and biological basis of PB remains poorly defined, limiting the development of more effective therapeutic strategies. To identify the cellular composition, origins, and drivers of PB, we derived single-cell transcriptomes from patient-derived pineal tumors (n=38) and integrated malignant cells with a novel transcriptional atlas of mouse pineal gland development (E13-P21). Application of a series of computational strategies pinpointed transient, cycling pinealocyte progenitors as the likely cellular origin of PB, irrespective of molecular subgroup. To functionally validate these findings, we generated lineage-specific genetically engineered mouse (GEM) models representing distinct PB subgroups with high temporal, anatomic, and phenotypic fidelity. Transcriptomic analysis further substantiated the accuracy of novel GEM models which converged on cell-cycle dysregulation, highlighting a core pathogenic mechanism across PB subgroups. Relative to the developing pineal gland, photoreceptor transcription factors were aberrantly activated in PB GEM models, suggesting photoreceptor involvement in tumor initiation and/or maintenance. Multi-omic analysis across a large range of diverse CNS cancers revealed an oncogenic photoreceptor program specific to PB, retinoblastoma (RETB), and Group 3 medulloblastoma (G3-MB). This shared transcriptional program was active within respective cellular origins, establishing a unified mechanistic basis across these anatomically distinct CNS tumors. CRISPR-based functional studies confirmed selective dependencies of photoreceptor transcription factors (NRL, CRX, OTX2) in PB, RETB and G3-MB, demonstrating the essentiality of this photoreceptor program. These discoveries not only resolve longstanding uncertainties regarding PB pathogenesis but also highlight conserved molecular signatures and therapeutic vulnerabilities spanning different pediatric CNS malignancies. Our results underscore the potential of targeting developmentally regulated programs and master transcription factors as novel therapeutic strategies for PB, RETB, and G3-MB, offering critical insights and preclinical models for future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cee7231cd5056fbbc7fcab0e2a1e377781c0c82" target='_blank'>
              EPCO-05. AN ONCOGENIC PHOTORECEPTOR PROGRAM UNIFIES DISTINCT CENTRAL NERVOUS SYSTEM TUMORS
              </a>
            </td>
          <td>
            B. Gudenas, S. Ahmad, Bernhard Englinger, Anthony P Y Liu, Miao-yun Zhao, Leena Paul, Jennifer L. Hadley, Hong Lin, Yiran Li, Melissa Batts, Priya Mittal, Stephanie C Wu, Sara Lewis, Katie Han, T. Soliman, Laure Janke, David Meredith, E. Pfaff, J. Gojo, Jennifer Cotter, P. Klimo, F. Boop, Amar Gajjar, G. Robinson, G. Rosén, S. Alexandrescu, David T. W. Jones, Brent A. Orr, F. Swartling, M. Filbin, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Abstract Background Early tumour vascular invasion contributes to cancer progression. Tip cells, a subset of tumour endothelial cells, significantly decline after anti‐angiogenic therapy. However, their behaviour and the roles of their signature genes during early invasion are incompletely understood. Methods This study employed single‐cell transcriptomic analysis and 10x Genomics Visium spatial transcriptomics on fresh lung tissues from patients with pulmonary nodules and from KrasG12D (K) and KrasG12DTgfbr2−/− (KT) mice. The role of plasma vesicle‐associated protein (PLVAP), a tip cell marker, was further examined using survival databases, immunofluorescence, in vitro co‐culture, cell migration, invasion assays and endothelial tube formation. Results Tip cell proportions were elevated in early‐stage lung adenocarcinoma (LUAD) tissues and KT mice, with evidence suggesting they arise from capillaries type I. PLVAP expression was enriched in tumour endothelial cells, induced by TGFβ1, and negatively correlated with patient prognosis. Functionally, PLVAP promoted endothelial cell invasion, migration and angiogenesis, and regulated tumour cell invasiveness. Intercellular analysis revealed that some tip cells also expressed TGFβ1, which may act on adjacent tumour cells to enhance invasion during early tumour development. Conclusion Tip cells increased during early LUAD progression and likely evolved from capillaries type I. Their marker PLVAP was associated with poor prognosis and pro‐invasive endothelial behaviour. Tumour‐secreted TGFβ1 upregulated PLVAP in endothelial cells, promoting angiogenesis and tumour invasion. Additionally, tip‐cell‐derived TGFβ1 may further stimulate tumour aggressiveness, highlighting a reciprocal interaction that contributes to early tumour progression. Key points Tip cells expand during early LUAD progression and likely originate from capillary type I endothelial cells. Tumour‐derived TGFβ1 induces PLVAP expression in endothelial cells, linking tumour signals to vascular activation. PLVAP enhances endothelial cell migration, invasion and angiogenic capacity. Endothelial PLVAP promotes tumour cell invasiveness, revealing a reciprocal endothelial‐tumour interaction that drives early tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79021428fe4068225ddb196bfe79a293a09eade5" target='_blank'>
              PLVAP mediates the regulation of the tumour microenvironment in early‐stage lung adenocarcinoma
              </a>
            </td>
          <td>
            Linshan Xie, Mengting Sun, Yujie Zheng, Zezhong Yu, Hui Kong, Jinjie Yu, Shaohua Lu, Yong Zhang, Jie Hu, Hongyi Xin, Jian Zhou, Xiangdong Wang, Charles A Powell, Fred R Hirsch, Chunxue Bai, Yuanlin Song, Jun Yin, Dawei Yang
          </td>
          <td>2025-12-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="NSCLC accounts for approximately 85% of lung cancers, and Kirsten rat sarcoma (KRAS) is the most commonly mutated gene in NSCLC. Although management of NSCLC has advanced markedly in recent years, its heterogeneity and plasticity remain incompletely understood. Single-cell RNA sequencing (scRNA-seq) has become an essential tool for dissecting cellular heterogeneity in complex tissues such as the lung. However, generating high-quality single-cell suspensions from tumor-bearing lung tissue is challenging, particularly during tumor formation when the tissue becomes fibrotic and heterogeneous. A key requirement for successful scRNA-seq is the preparation of viable single cells with minimal processing-induced stress or damage. Here, we present a detailed step-by-step protocol for isolating high-quality single-cell suspensions from both healthy and tumor-bearing mouse lung tissue in a KrasG12D; tdTomato reporter mouse model. The procedure includes tissue perfusion, controlled enzymatic digestion, gentle mechanical dissociation, red blood cell lysis, filtration, and viability assessment, and achieves >85% viable cells prior to scRNA-seq. Finally, the tdTomato+ tumor cells are efficiently isolated by FACS and can be detected as distinct clusters by scRNA-seq using this protocol. SUMMARY We present a detailed protocol for isolating high-quality single-cell suspensions from both healthy and tumor-bearing murine lung tissue. This protocol can be applied to scRNA-seq, flow cytometry analysis, and primary cell isolation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05709be045bd3ac955b0ab18505a25294de0610f" target='_blank'>
              Optimized protocol for processing murine tumor-bearing lung tissue for flow cytometry and single cell RNA-sequencing
              </a>
            </td>
          <td>
            Sicong Wang, Anastasiia Ivanova, Helen P. Makarenkova, Katja A. Lamia
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Intracranial germ cell tumors (IGCTs) encompass several histologic subtypes, each characterized by distinct morphologies and varying therapeutic outcomes, yet their immune landscape remains poorly understood. To elucidate the immune microenvironment associated with these tumors, we first obtained four samples with paired spatial transcriptomics (Visium HD) and proteomics (CODEX), as well as two additional samples with only spatial transcriptomics data to study the immune microenvironment in different histologic regions. Our observations revealed a diverse immune pattern across IGCT tissue types. Notably, we observed the remarkable presence of tertiary lymphoid structure (TLS) within the IGCTs. Specifically, intratumoral TLS was most abundant and mature in teratoma, germinoma, and their respective components in mixed GCT, whereas yolk sac tumor (YST) exhibited no intratumoral TLS, with only a few peripheral TLS noted. To confirm and extend these findings in additional cases, we analyzed IGCT bulk RNA-seq data, including mature teratomas, immature teratomas (IMT), germinomas, YSTs, and mixed GCTs. Our analysis indicated that germinoma, mixed GCT, and teratoma exhibited both the highest TLS signature scores and the highest abundance of T cells and B cells, while YST and IMT had the least. Additionally, we noticed that the high abundance of B cells and T cells was driven by multiple clones rather than oligoclonal expansion. These findings provide valuable insights into the immune landscape of IGCTs, highlighting the potential for targeted immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/460b9c1e95b60feb342540a44365c08c4e3019a4" target='_blank'>
              TMIC-56. Spatial multi-omics data reveals immune landscape and tertiary lymphoid structures in intracranial germ cell tumors
              </a>
            </td>
          <td>
            Ran Liu, Dong Song, Jihong Tang, Yimeng Qiao, Huiyuan Chen, R. Chai, Jiguang Wang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is characterized by its highly aggressive phenotype and dismal outcome. Despite the benefit of adding immune checkpoint blockade to standard chemotherapy, tumors acquire the ability to evade immunosurveillance and develop resistance. To investigate these underlying mechanisms, we perform high-dimensional profiling of human and murine SCLC specimens. In matched primary and metastatic human samples, we observe MHC-I loss in metastases, highlighting its role in immune evasion. Correspondingly, silencing MHC-I in SCLC cells drastically reduces immune infiltration and promotes metastasis in mice. Using mass spectrometry and phospho-tyrosine kinase analyses, we identify ERBB2 signaling as a suppressor of MHC-I and driver of immune-modulatory transcripts. Mechanistically, genetic and pharmacologic blockade of ERBB2 induces MHC-I in a STING-dependent manner and prevents immune evasion in autochthonous murine SCLC. Strikingly, combining ERBB2 inhibition with anti-PD-1 elicits profound synergistic responses in preclinical models, suggesting this combination for future clinical trials in SCLC patients. While the emergence of immune checkpoint inhibitors has improved outcomes in patients with small cell lung cancer (SCLC), tumour that develop means of immune evasion become resistant. Here, the authors report that ERBB2 signalling induces loss of MHC Class I expression and subsequently immune evasion in preclinical models of SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f612388efd2f0a9c1ebc8d3e60e6b396ae989beb" target='_blank'>
              ERBB2 signaling drives immune cell evasion and resistance against immunotherapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Meder, C. Orschel, C. Bouchez, R. Gholamipoorfard, C. V. Orschel, David Stahl, Christoph Kreer, Mirjam Koker, Marieke Nill, Ilayda G Kocak, Ka-Won Noh, Xinlei Zhao, Leon Ullrich, B. Häupl, Josefine Jakob, Marie-Lisa Eich, Alexandra Florin, Holger Grüll, J. Wolf, Filippo Beleggia, R. Buettner, T. Oellerich, Florian Klein, J. Brägelmann, H. C. Reinhardt, Nima Abedpour, R. Ullrich
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) and pancreatic ductal adenocarcinoma (PDAC) are highly lethal cancers marked by profound epigenetic and metabolic reprogramming. Among the candidate biomarkers, the DNA methyltransferase DNMT3A and the guanine monophosphate synthetase (GMPS) have emerged as potential prognostic drivers, yet their roles across tumor contexts remain unclear. Here, we demonstrate the application of KMplotter to interrogated pan-cancer transcriptomic and survival datasets encompassing over 7000 patients, complemented by expression profiling of normal, tumor, and metastatic tissues, and integrated tumor microenvironment (TME) analyses. Elevated DNMT3A and GMPS expression correlated with worse overall survival in HCC, particularly in Asian patients, while in PDAC, high DNMT3A but low GMPS expression predicted favorable outcomes. Both genes were consistently upregulated in tumors relative to normal tissues, with further increases in metastatic HCC. Immune deconvolution revealed that DNMT3A was linked to Th2/Treg-enriched niches, whereas GMPS overexpression coincided with high mutational burden or stromal enrichment, fostering immunosuppressive microenvironments. Comparative analysis of toll-like receptor signatures highlighted divergent antigen-sensing pathways, with HCC reflecting viral-driven immune exhaustion and PDAC showing self-antigen–associated signaling. Collectively, these findings position DNMT3A and GMPS as context-dependent biomarkers that integrate metabolic and immune cues to shape prognosis in liver and pancreatic cancer, offering mechanistic insight and translational relevance for patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c71a88a2f1a7eea4d4abf0ff75d340e4a7fe137" target='_blank'>
              Comparative Tumor Microenvironment Analysis for HCC and PDAC Using KMplotter
              </a>
            </td>
          <td>
            Wen-Han Chang, Drashya Shah, Scott Myers, Michael Potts, Sanjive Qazi, Vuong Trieu
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related mortality worldwide, with tumor recurrence and metastasis posing significant challenges despite advances in targeted therapies and immunotherapy. Cellular dormancy, a reversible, quiescent state marked by cell cycle arrest, has emerged as a key driver of therapeutic resistance and disease relapse, particularly in small-cell lung cancer (SCLC) and non-small cell lung cancer (NSCLC). Multiple mechanisms, including autophagy, stress-adaptive signaling, microenvironmental cues, and epigenetic dysregulation, have been implicated in the regulation of dormancy and long-term cell survival. Among these, epigenetic modifications such as DNA methylation, histone modifications, and non-coding RNAs (ncRNAs) play pivotal roles in maintaining dormancy by repressing proliferative gene expression programs. Increasing evidence suggests that dormant tumor cells harbor distinct epigenomic signatures, which may serve as predictive biomarkers for minimal residual disease (MRD) and relapse risk. This review summarizes current advances in understanding the epigenetic regulation of cellular dormancy in lung cancer, with a particular emphasis on the interplay between epigenetic modifiers and oncogenic signaling pathways. Furthermore, emerging molecular targets and associated therapeutic agents currently under clinical evaluation are presented, emphasizing how a deeper understanding of the epigenetic landscape governing dormancy may inform the development of novel interventions to improve long-term clinical outcomes in lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f78e8931704f8d1c6b0d604cdcde3728bfe1b4a" target='_blank'>
              Unlocking Lung Cancer Cell Dormancy: An Epigenetic Perspective
              </a>
            </td>
          <td>
            F. Fabrizio
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 The introduction of bowel cancer screening has led to an increase in the detection and removal of pre-malignant adenomas, however many of these would never progress to colorectal cancer (CRC) within a patient’s lifetime. This implies a key evolutionary bottleneck at the transition from adenoma to carcinoma, with immune surveillance likely playing a central role in suppressing the outgrowth of invasive cells. Here we analysed a cohort of “ca-in-ads” (adenomas with a small focus of cancer) to provide a unique snapshot of the invasive transition. We combined genomics, spatial transcriptomics, digital pathology and multiplex imaging, concurrently deriving the distribution of tumour clones, their gene expression and their interplay with the immune microenvironment. We performed multi-region low-coverage whole genome sequencing (lcWGS) and T-cell receptor sequencing (using FUME-TCRseq, a novel method developed in our lab) on archival ca-in-ads from 40 patients. For a subset of cases we performed spatial transcriptomics using the 10x Genomics Visium platform, including custom probes to detect T-cell clonotypes of interest. We applied a deep learning cell classifier to haematoxylin and eosin (H&E) stained sections to determine the abundance and spatial distribution of eight cell types, with detailed immunophenotyping on selected cases (using a 50-marker panel on Akoya’s PhenoCycler-Fusion platform). We found a significant increase in copy number alterations in carcinoma regions relative to adenomas, and this was accompanied by a large shift in the composition of the local T-cell repertoire. TCR sequencing revealed expanded T-cell clonotypes that were adenoma-specific or carcinoma-specific, and we validated the spatial exclusivity of these using a custom Visium assay. Although overall lymphocyte density remained unchanged, digital pathology and cyclic immunofluorescence revealed significantly increased infiltration of macrophage and neutrophil populations in regions of invasive cancer. Taken together our analysis suggests a striking reshaping of tumour-immune dynamics at the transition to an invasive phenotype in CRC. Understanding what governs this invasive shift could highlight new avenues for cancer prevention or identify individuals at risk of CRC.



 Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, Nick Trahearn, Holly Eggington, Miangela M. Lacle, Manuel Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham. Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b4fa90cafae68daf434680b6837e68209ae59d6" target='_blank'>
              Abstract PR004: Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer
              </a>
            </td>
          <td>
            Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, N. Trahearn, Holly Eggington, M. Laclé, M. Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) and diffuse midline glioma (DMG) are two of the most aggressive brain tumors, marked by significant intratumoral heterogeneity and transcriptional plasticity, which drive resistance to therapy. These tumors share similar transcriptional programs, though the specific genes associated with these programs differ. Transitions between transcriptional states, particularly to the mesenchymal state, which is associated with greater resistance to treatment, complicate treatment strategies. A significant challenge in glioma research is the inability to dynamically track these state transitions in vitro and in vivo. To address this challenge, we introduce CRISPR-TAG reverse genetics, a novel approach that enables real-time monitoring of key transcriptional state markers in glioblastoma and diffuse midline glioma cell lines. Using CRISPR-TAG, we insert a fluorescent reporter sequence at the 3’ end of target genes, linking the fluorescent signal to endogenous gene expression. This allows us to dynamically track changes in gene expression as cells transition between states. We have successfully tagged 30 key cell state markers in patient-derived GBM and DMG cell lines, providing a robust model to investigate the upstream regulators of cell state transitions. By integrating CRISPR-TAG with global CRISPR screens and high-content pharmacological screens, we uncover genetic dependencies that underlie glioma plasticity. For example, we identify modulators of the mesenchymal cell state, including ANXA1 expression, which serves as a critical marker of this resistant state. Additionally, we demonstrate how our method allows in vivo tracking and modulation of cell state transitions in explant cultures, with AI-assisted analysis providing detailed insights into these transitions. In conclusion, CRISPR-TAG reverse genetics lays a critical foundation for building a high-resolution map of glioma state regulators, paving the way for targeted therapeutic strategies. By enabling the precise modulation of cell states, this method offers the potential to complement and enhance existing treatment modalities for GBM and DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eaf38521e14ac09d468437f1dbddc6b4207f994" target='_blank'>
              CNSC-29. RESOLVING THE REGULATION OF GLIOBLASTOMA CELL STATES BY CRISPR-TAG REVERSE GENETICS
              </a>
            </td>
          <td>
            Josephine Heinold, Madeleine Skeppås, Filippa Kruse, Clara Nordquist, Irem Uppman, Sven Nelander
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) serve as a promising biomarker for tracking cancer metastasis, progression, and recurrence. Liquid biopsy techniques centered on CTC detection have demonstrated considerable potential due to their non-invasive nature and ability to provide real-time monitoring of tumor dynamics. However, conventional bulk CTC analyses fail to capture the intrinsic heterogeneity among CTC populations, obscuring crucial insights into tumor biology. Single-cell RNA sequencing (scRNA-seq) enables high-resolution characterization of CTC heterogeneity, offering new opportunities for precision oncology and mechanistic studies of tumor progression. Despite these advantages, the existing methodologies for single-CTC sequencing tend to suffer from inefficiencies, including low recovery rates, labor-intensive workflows, and contamination risks associated with multiple manual handling steps. To address these limitations, we present an integrated microfluidic protocol that consolidates CTC enrichment, purification, and single-cell sequencing into a unified workflow. The method employs dynamically controlled magnetic capture within a herringbone-structured chip, where vortex mixing and cumulative immunomagnetic bead binding enable robust, high-throughput CTC isolation with minimal cell damage. Subsequent purification using a leukocyte antibody-coated microfluidic chip effectively removes the non-target cells, further enhancing CTC purity through negative selection. Finally, a high-precision single-cell sequencing chip, designed based on differential flow resistance principles, facilitates efficient single-cell capture and pairing with uniquely barcoded microbeads. This novel platform overcomes the limitations of Poisson distribution-based methods, improving CTC utilization while minimizing microbead consumption and sequencing costs. Our integrated protocol significantly enhances CTC capture efficiency, purity, and single-cell sequencing throughput, making it well-suited for clinical applications and large-scale cancer research. By enabling a more precise and scalable analysis of CTC heterogeneity, this method has the potential to refine early cancer diagnosis, treatment monitoring, and mechanistic studies of metastasis, ultimately advancing the field of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9babafe8a10523678f47d2a96e859b16fafc3f98" target='_blank'>
              Microfluidics-based High-throughput Circulating Tumor Cell Sorting and Single-cell Sequencing Technology.
              </a>
            </td>
          <td>
            Zheyu Liu, Ye Tian, Zian Zhang, Lingling Wu, Liang Dong, Wei Xue
          </td>
          <td>2025-11-14</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, , Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background The most aggressive type of skin cancer, melanoma, has a high prevalence of metastases and a poor prognosis. Despite advancements in treatment, drug resistance and tumor microenvironment heterogeneity, especially involving neuro-immune interactions, continue to exist. The goal of this study is to uncover the cellular heterogeneity of melanoma in order to pinpoint molecular targets and tumor-promoting subtypes. Methods Melanoma single-cell RNA sequencing data came from GEO. Twelve cell types were discovered after Harmony batch effect adjustment and Seurat quality control; melanoma cells were subtyped. Functional expression of differential genes was examined using gene ontology and gene set enrichment. Cytotrace measured subtype differentiation potency. PySCENIC revealed transcription factor regulatory networks, and CellChat predicted intercellular communication between malignant cell subtypes and other cell types. Functional experiments with A375 and MEWo cell lines—lentiviral shRNA knockdown, CCK-8 proliferation, wound healing, transwell migration, and flow cytometry apoptotic assays—confirmed the analysis. Results Single-cell RNA sequencing was utilized to analyze melanoma cell subtypes and their interactions in the tumor microenvironment. C2 IGFBP3+, which had the lowest CytoTRACE2 score and was enriched in late tumor stages, affected melanoma development. This subtype expressed prominent immunomodulatory pathways. The C2 subtype mediate signaling to fibroblasts/pericytes via the PROS pathway and myeloid/plasmacytoid dendritic cells via the MHC-II pathway. The C2 subtype was strongly linked with FOSL1 expression, which may affect gene transcription and illness progression. FOSL1 knockdown significantly increased apoptosis and decreased melanoma cell motility and proliferation in vitro. Conclusion We identified immunoregulatory C2 IGFBP3+ melanoma cell subtypes in our investigation, and FOSL1 was a crucial transcription factor that aided in cell migration, proliferation, and survival. Because the C2 subtype involves the MHC-II and PROS signaling pathways, which have been shown to have roles in neuroimmunology, neuroinflammation, and pain regulation, it may serve as a hub for neuro-immune interactions in the tumor microenvironment. Precision treatments for melanoma may be advanced by focusing on the FOSL1 or C2 subtype pathways, which may assist in overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34e116063cb6903b41d6e715ba8448ec0e1c090" target='_blank'>
              Single-cell transcriptomics identifies FOSL1-regulated IGFBP3+ melanoma subtype as a neuro-immunoregulatory signaling hub facilitating tumor progression
              </a>
            </td>
          <td>
            Wenjia Ge, Ziwei Zhang, Wenjie Chen, Zhijie Zhao, Hua-Bao Cai, Yantao Ding, Jin Xu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cda57de19bafd2e6a7a2966bef4b6684b824ac08" target='_blank'>
              Morphological Landscape Mapping Decodes Pathological Heterogeneity and Proteomic Programs in HCC
              </a>
            </td>
          <td>
            X. Wang, Y. Cheng, X. Peng, J. Li, J. Han, Z. Yang, G.-Y. Ding, H. Zhao, P. Qu, Q. Sun, Z. Chang, T. Zou, B. Wang, T. Huang, Z. Cheng, S. Guo, S. Ma, J. Wang, C. Liu, X. Zhang, S. Zhang, L. Tian, H. Han, L. Ai, L. Wang, X.-Y. Wang, J. Zhou, J. Fan, Q. Gao, H. Zhou, X. Liu, J. Shi
          </td>
          <td>2025-12-02</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="In this study, we analyzed publicly available scRNA-seq data from primary tumor tissues of 10 Retinoblastoma (RB) patients to explore tumor microenvironment (TME) heterogeneity. Our findings revealed distinct subpopulations of cone precursor (CP) cells, with higher proportions in invasive RB. Differential gene expression and pathway analysis highlighted functional diversity among CP subpopulations, with CP4 showing elevated TGF-β signaling in invasive RB. Cell–cell interaction analysis further identified rewiring of communication networks, with increased fibroblast–CP interactions in invasive tumors. Bulk RNA-seq identified two molecular subtypes, with subtype 1 showing an immunosuppressive TME. Finally, DOK7 was identified as a key gene associated with invasion, with functional assays confirming its role in promoting tumor progression. These results provide valuable insights into RB progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0641facdea378a09ac72da42fc0862a4720147" target='_blank'>
              Comprehensive analysis of single-cell and bulk RNA sequencing uncover tumor microenvironment diversity in invasive Retinoblastoma
              </a>
            </td>
          <td>
            Wei Wan, Xiaoqian Chen, Hongli Liu, Ling Yang, Pingping Huang
          </td>
          <td>2025-11-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Effective T cell immunity against solid tumors requires not only functional capacity but also targeted trafficking to tumor sites. Effector T cells must engage in a multi-step adhesion process with endothelial cells (ECs) to extravasate from the bloodstream into tissues. While these interactions are well-characterized in venular endothelial cells (VECs) of benign tissues, they remain poorly understood in malignant tumors. Tumor vasculature is known for its irregular morphology and disrupted organization. Using single-cell RNA-seq and spatial transcriptomics, we show that some tumor ECs still maintain a venular transcriptional profile. Intratumoral VECs exhibit an immature phenotype with reduced adhesion molecule expression, whereas marginal VECs are activated and closely associated with infiltrating T cells. Employing atypical chemokine receptor 1 (ACKR1) as a novel VEC marker, we find that VECs are enriched in ‘hot’ tumors with high T cell infiltration but are depleted in ‘cold’ tumors with low infiltration. This trend is observed across heterotopic and orthotopic murine models and human samples. Notably, ACKR1+ VECs predominantly localize at tumor margins, corresponding to sites where adoptively transferred T cells preferentially home, suggesting that efficient T cell infiltration relies on these VECs. These findings highlight that peritumoral VEC abundance and functional maturity are essential for effective T cell infiltration into tumors.



 NIH-NIAID AI112521



 Cellular Adhesion, Migration, and Inflammation (CAM)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87145ede5d2ec550ed35385389b9611388e31a6a" target='_blank'>
              Peritumoral venular endothelial cells modulate T cell infiltration into solid tumors 4763
              </a>
            </td>
          <td>
            Changwei Peng, Paris Pallis, Munir M. Mosaheb, Stephanie K. Dougan, Ulrich H. von Andrian
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven by heterogeneous tumor cell populations and microenvironmental interactions. However, a comprehensive single-cell atlas across patients that captures this heterogeneity, and its clinical implications has been lacking. Such an atlas could reveal rare tumor subpopulations that underpin disease aggressiveness and offer new prognostic biomarkers or therapeutic targets. We integrated two high-quality single-cell RNA sequencing datasets from 29 CRC patients to construct a cellular atlas encompassing immune, stromal and epithelial compartments. Malignant epithelial cells were distinguished via inferCNV-based copy number alteration analysis and reclustered, yielding seven transcriptionally distinct malignant subpopulations. One malignant epithelial cluster, marked by high expression of the long non-coding RNA ELFN1-AS1, exhibited the highest stemness signature. From this stem-like cluster, we derived a four-gene prognostic model (RPL21, GAL, ELFN1-AS1, AIF1L). In the TCGA-COAD cohort, this model stratified patients into high-risk and low-risk groups with significantly different survival outcomes, with the high-risk group experiencing significantly worse survival. High-risk tumors were enriched for metabolic and translational pathways and displayed distinct immune and genomic features. AIF1L was identified as a hub gene within the signature, and its knockdown in CRC cells enhanced migration and invasion, functionally validating its role in tumor progression. Our study provides a high-resolution single-cell atlas of CRC and identifies a previously unrecognized stem-like tumor cell subpopulation with prognostic significance. These findings highlight novel prognostic biomarkers and suggest potential therapeutic targets (such as AIF1L) that could inform patient stratification and the development of targeted therapies for CRC. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-15241-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3d35184693bc48dd0fe85be6bfc55a03d6e4e7" target='_blank'>
              Single-cell RNA sequencing defines prognostic subtypes and identifies AIF1L as a therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Gao Li, Dingxue Wang, Guo Chen
          </td>
          <td>2025-12-26</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The tumor immune microenvironment (TiME) accommodates significant transcriptomic and proteomic heterogeneity, and understanding this heterogeneity is key to the development of cancer treatments. Most analytical approaches provide only one type of biological readout and/or lack spatial information needed to decipher TiME complexity. We developed a new spatial multi-omics toolset to comprehensively study the TiME transcriptomic and proteomic readouts with single cell resolution. We used the CellScape™ Precise Spatial Proteomics platform with enhanced photobleaching in cyclic immunofluorescence (EpicIF) technology to detect 12+ RNA targets via HCR™ Gold RNA-FISH and 24+ protein markers via multiplex immunofluorescence (mIF) on the same formalin-fixed paraffin-embedded (FFPE) tissue sample. RNA-FISH target probes were added to the tissue sample after antigen retrieval, and tissues were then mounted in CellScape fluidic chambers to enable on-instrument staining, imaging, and signal removal. The RNA-FISH assay was followed by a 24-plex mIF assay containing antibodies against common immune-oncology biomarkers. Multiomic data from human tonsil, colon and breast tissues showed high conformity between RNA-FISH and mIF assays on the CellScape platform. The combined assay revealed differential biomarker expression profiles associated with immune-regulatory processes in the tumor samples and provided detailed insight into immune and inflammatory processes of the TiME.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5daaa251cd6bb773ad5618931f13decbb4ef34a" target='_blank'>
              Spatial multi-omic analysis of the tumor microenvironment using high-plex HCR RNA and proteomic immune profiling on the CellScape platform 4057
              </a>
            </td>
          <td>
            Matthew H. Ingalls, J. Boog, Daniel Jimenez Sanchez, Adam Maddox, Aneesh Acharya, Cameron Earl, Harry Choi, A. Christians, O. Braubach
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Introduction: Large B-cell lymphomas (LBCL) exhibit intra-patient biologic heterogeneity. Single-site biopsies limit comprehensive assessment of this complexity, which circulating tumor DNA (ctDNA) overcomes by detecting mutations from distinct anatomical sites. We have previously shown anatomical genomic heterogeneity can be quantified by integrating baseline ctDNA and tissue profiles (Goldstein, ASH 2024). Though aberrant somatic hypermutation (aSHM) drives lymphomagenesis, its contribution to anatomical heterogeneity, clonal evolution, and outcomes is poorly understood. We validate the clinical value of anatomical genomic heterogeneity in diverse aggressive lymphomas, delineate its relation to aSHM, and then propose a DLBCL clonal evolution model.
 Methods: We studied 588 specimens from 196 patients across 3 cohorts with comprehensive genotyping of baseline plasma and matched tumor tissues. The Discovery Cohort included 66 LBCL patients from a multinational cohort receiving front-line chemoimmunotherapy with pre-treatment tumor, germline and plasma genotyping by CAPP-Seq (Kurtz, JCO 2018). We developed the Metric Of Spatial Anatomic Intra-tumoral genomic Complexity (MOSAIC) as the root mean square error of a linear regression model of mutant allele frequencies (log[VAF]) in ctDNA versus those shared in tissue.
 After defining a MOSAIC threshold in our Discovery Cohort, two validation cohorts with pre-treatment tumor, germline and plasma samples and treated with front-line chemoimmunotherapy were used: 97 LBCL patients from the HOVON-902 study profiled by PhasED-seq (Wang, ASCO 2025) and 33 MCL patients from the LyMa trial profiled by CAPP-Seq (Tessoulin, 18-ICML).
 To elucidate clonal evolution, we explored the molecular associations and relationship of aberrant somatic hypermutation (aSHM) with anatomic heterogeneity and outcomes in LBCL. We defined aSHM rate as the fraction of ctDNA mutations occurring within canonical aSHM targets (Schmitz, NEJM 2017). We measured clonal architecture (sciclone) and molecular subtypes (LymphGen) and performed survival analysis (Kaplan-Meier; Cox models).
 Results: In our Discovery Cohort, 100% had SNVs unique to plasma, with median 15% of plasma mutations absent in paired tissue. The median MOSAIC was 0.235 (threshold for validation) with range 0.1-0.4. Cases with >1 subclone in plasma had higher MOSAIC (median 0.26 vs 0.19,p=0.002). MOSAIC did not correlate with age, COO, histological subtype, Stage, or IPI. However, high MOSAIC was associated with worse PFS (HR=4.3,p=0.01) and OS (HR=12.7,p=0.02). MOSAIC did not correlate with ctDNA burden and remained prognostic independent of ctDNA levels.
 In the MCL LyMa Validation Cohort, median MOSAIC was 0.28 (IQR 0.13-0.39). MOSAIC correlated with ctDNA burden (p=0.05) and LDH (p=0.01), but not with leukemic status, blastoid morphology, or Ki67. High MOSAIC predicted worse PFS (HR 13.6,p=0.01), independent of LDH and ctDNA level.
 In the LBCL HOVON-902 Validation Cohort, median MOSAIC was 0.21 (IQR 0.17-0.28) with 38% classified as high. BN2, EZB & A53 tumors by LymphGen had higher MOSAIC (median 0.26), while MCD had lower scores (median 0.18). High MOSAIC was again associated with inferior PFS (HR=2.3,p=0.04) and OS (HR=3.4,p=0.02) and higher end of treatment MRD+ rates (28% vs 12%).
 TP53 and P2RY8 mutations were associated with high MOSAIC patients. GCB tumors showed numerically higher aSHM rates, though no significant differences in aSHM rates by histology or stage were found. Lower aSHM rate correlated with higher MOSAIC (r= -0.4,p=0.001) and worse PFS (HR 4.4,p=0.02). Patients with both high MOSAIC and low aSHM rate (38%) had 5-year PFS 43% (HR 5.6,p=0.001) and OS 57% (HR 9.8,p=0.003), vs 87 and 93% respectively. In 80%of patients, median plasma VAF for aSHM exceeded non-aSHM mutations, indicating aSHM occurs early in lymphomagenesis. High MOSAIC patients had higher ratios of median aSHM to non-aSHM VAFs, consistent with ongoing subclonal evolution via non-aSHM mutations.
 Conclusions: We describe and validate a novel biomarker measuring anatomical genomic heterogeneity, integrating plasma and tissue profiles, that independently predicts prognosis for front-line chemoimmunotherapy. We show that aSHM aberrations occur early in clonal evolution and likely initiate lymphomagenesis. Low aSHM rates and high anatomical heterogeneity in B-cell lymphomas define a high-risk subgroup prone to treatment resistance via anatomical clonal divergence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166a4d4cafdabd9dbc2d4b4b470b779c06222e42" target='_blank'>
              Spatial anatomical genomic heterogeneity and aberrant somatic hypermutation define clonal evolution pathways that predict treatment resistance in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            J. Goldstein, Steven Wang, B. Tessoulin, C. Liu, F. Scherer, S. Le Gouill, Martine E D Chamuleau, M. Nijland, Maximilian Diehn, David Kurtz, A. Alizadeh
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="
 The H3K27M mutation inhibits PRC2 activity resulting in a global hypomethylation and concomitant increase in acetylation of H3K27 wildtype histones. Such epigenetic alterations force diffuse midline glioma (DMG) cells to retain a highly proliferative stem-like cell state. However, single cell transcriptomics revealed that some cancer cells, despite harboring the H3K27M mutation, can exit the cell cycle and acquire features of mature glial-like cells, providing rationale for differentiation therapies. Here, we sought to dissect the epigenetic landscape of these distinct cancer cell states within DMG to better understand how tumor cell identity might be modulated for therapeutic benefit. To do so, we profiled the histone posttranslational modifications landscape and genome accessibility in both stem- as well as glial-like DMG cells. We observed that the more mature glial-like cells overturn the oncogenic activity of the histone mutation on H3K27 marks, and consequently carry higher levels of the repressive histone mark H3K27me3 and lower levels of the activating mark H3K27ac. Next, we questioned whether H3K27 acetylation loss would reciprocally drive DMG cell differentiation. Intriguingly, our epigenetically-focused CRISPR screen identified the histone acetyltransferases responsible for acetylating H3K27, EP300/CBP, as essential genes for the continuous proliferation of stem-like DMG cells. In agreement, DMG cells are selectively sensitive to EP300/CBP inhibition or degradation, unlike human astrocytes and H3G34R/V high-grade glioma cells. Interestingly, targeting EP300/CBP reverts the epigenetic reprograming caused by H3K27M by not only depleting H3K27ac but also rescuing H3K27me3 levels. Finally, ATAC and single cell RNA sequencing revealed that the epigenetic changes caused by EP300/CBP targeting shift DMG stem-like cells towards more differentiated glial-like cell states. In summary, we show that distinct DMG transcriptomic cell states reflect underlying epigenetic heterogeneity which can be exploited for treatment. In particular, we demonstrate that EP300/CBP are promising therapeutic targets for differentiation therapy in this universally fatal glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38acecf50f0c665a725bfac7f8d20ecf3c8a39a3" target='_blank'>
              EPCO-64. EXPLOITING EPIGENETIC CELL STATE HETEROGENEITY TOWARDS DIFFERENTIATION THERAPY IN H3K27M-ALTERED DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Joana G. Marques, Matthew Mullaly, C. B. Junior, C. Cascio, Andrezza Do Nascimento Silva, Qi Liu, Pruthvi Gowda, N. Danial, Jun Qi, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 Diffuse midline glioma (DMG) is an incurable pediatric brain tumor, with radiotherapy offering only transient benefit before inevitable recurrence. While recent studies have revealed the cellular complexity of DMG, the relationship between intratumoral heterogeneity and disease progression — particularly under therapeutic pressure — remains poorly understood. In this study, we integrate longitudinal single-cell- and spatial transcriptomics with single-cell chromatin accessibility profiling and radiomics to investigate how distinct DMG cell states and microenvironmental components evolve in response to treatment, contributing to resistance and the emergence of therapy-persistent niches. We applied this multi-omic approach to a unique cohort of 10 matched diagnostic and autopsy DMG samples obtained from children enrolled in the PNOC023 clinical trial. Spatial transcriptomic profiling revealed that post-treatment autopsy tissues showed consistent enrichment of oligodendrocyte progenitor cell-like malignant populations, suggesting that this stem-like subpopulation is selectively retained or expanded following radiotherapy in DMG. Moreover, integration of single-cell RNA and ATAC sequencing with cerebrospinal fluid microRNA analyses further revealed molecular signatures of resistance and uncovered minimally invasive biomarkers of treatment response. To complement our molecular analyses, we applied radiomic profiling of longitudinal MRI scans to non-invasively monitor changes in tumor heterogeneity over the course of treatment in this patient cohort. In parallel, we developed a data-driven mathematical model using patient-derived DMG xenografts to simulate tumor cell responses to radiotherapy and generate quantitative predictions of preclinical treatment outcomes. We employed this model to explore alternative radiation fractionation schemes and identify optimal dosing strategies that most effectively deplete therapy-resistant tumor cell populations in DMG. Together, these integrated spatial and molecular datasets provide a high-resolution framework for understanding how radiotherapy remodels the DMG tumor ecosystem. By identifying persistent cellular populations and spatial niches that survive treatment, this work lays the groundwork for future precision therapies aimed at targeting the cellular reservoirs that drive DMG recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4395d6f286b7b33609966b9d19ccc4d41894fd90" target='_blank'>
              RBIO-16. Mapping the Spatiotemporal Evolution of Diffuse Midline Glioma Under Radiotherapy
              </a>
            </td>
          <td>
            Costanza Lo Cascio, Shashank Katiyar, Simone Bruno, Carlos A O De Biagi Junior, Danielle Bitterman, Joana G. Marques, Sina Neyazi, Sharmistha Pal, David Kozono, Benjamin H. Kann, Charles Stiles, Sabine Mueller, Franziska Michor, D. Haas-Kogan, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a highly malignant pediatric brain tumor of the cerebellum characterized by extensive molecular, biological, and clinical heterogeneity. Current genetically engineered mouse models have only partially replicated this diversity, and harbor species-specific limitations owing to fundamental differences between mouse and human development. Therapeutic strategies that accommodate MB heterogeneity are required to improve patient outcomes but necessitate biologically faithful models that are predominantly lacking. To this end, we have established human pluripotent stem cell-derived cerebellar organoids that recapitulate early human cerebellar development and enable deep mechanistic studies into MB oncogenesis. Transcriptomic profiling and immunophenotyping of our organoids throughout developmental timepoints revealed cellular compositions and differentiation trajectories reminiscent of cerebellar development in vivo. As our cerebellar organoids faithfully reproduce putative lineages of origin for distinct MB subgroups, we’ve endogenously Cre-tagged stem cell lines and generated various expression constructs to manipulate known driver genes in specific cellular compartments to generate accurate human cerebellar organoid-based disease models. We furthered the utility of this ex vivo human system by establishing a cerebellar organoid/patient-derived xenograft (PDX) co-culture platform that fosters the sustainability of MB PDXs in culture. Upon co-culturing with cerebellar organoids, we observed sustained viability and proliferation of MB PDXs, both of which are unprecedented in standard in vitro cultures. Further, we observed differential electrophysiological properties in PDX-harboring organoids compared to standalone organoids and identified putative signaling pathways crucial for cellular crosstalk between tumor and their supporting microenvironment. Our ongoing multi-faceted MB modeling strategies leveraging cerebellar organoids engender new experimental opportunities otherwise incompatible with pre-existing approaches, providing an efficient platform for advancing disease biology and identifying therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9c6a6fee26a49ce1740bbf42bd5633cf13397a" target='_blank'>
              TMOD-13. Human pluripotent stem cell-derived cerebellar organoids as a robust platform for studying medulloblastoma pathogenesis
              </a>
            </td>
          <td>
            Stephanie C Wu, Rachel Xu, Yiran Li, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="


 Renal cell carcinoma (RCC) is responsive to immunotherapy despite having a low tumor mutational burden. Interestingly, the degree of CD8+ T cell infiltration in RCC negatively correlates with outcomes, suggesting that a deeper understanding of the tumor microenvironment may provide critical insight to help improve clinical outcomes.



 We conducted a multi-omics approach including flow cytometry, multiplex immunohistochemistry, and single-cell RNA-seq to compare immune cell composition, gene expression, spatial distribution, and cell-cell interactions among a cohort of primary breast cancer, head and neck squamous cell carcinoma, non-small cell lung cancer, and RCC specimens.



 Flow cytometry analysis of tumor-infiltrating lymphocytes (TILs) revealed increased exhausted CD8 + (Tex) and CD4+CD8+ double positive (DP) T cells and decreased CD4+ T cells including regulatory T (Treg) cells in RCC samples compared to other tumor types. scRNA-seq analysis of TILs confirmed these results and showed Tex cells as a major population of CD8+ T cells. Using biomarkers for tumor-reactive cells including 4-1BB, CD39, and CD103 along with a gene signature to identify neo-antigen reactive T cells, we found that the Tex population contained putative tumor-reactive T cells and were not merely bystander cells.



 Exhausted CD8+ and DP T cells in RCC contain tumor-reactive T cells and may represent a novel cell population capable of targeting RCC.



 Funding: Funding was provided by a sponsored research grant from GlaxoSmithKline, the Providence Portland Medical Foundation, and the M. J. Murdock Charitable Trust.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3623ddf9190f1765df9cf05747187e1bf4ab0a38" target='_blank'>
              Multi-cancer comparative multi-omics approach reveals a permissive tumor microenvironment in renal cell carcinoma through accumulation of exhausted CD8+ and CD4+CD8+ double positive T cells 2661
              </a>
            </td>
          <td>
            M. Janghorban, Y. Koguchi, Noah D Simons, Wesley Rosales, V. Rajamanickam, Tanisha L. Christie, William L. Miller, Iliana Gonzalez, Zhaoyu Sun, Yaping Wu, Lakhvir Kaur, Brendan D. Curti, N. Yanamandra, Johanna Kaufmann, B. Bernard, Thomas Duhen, Eric Tran, William L Redmond
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61684838fc3c53a5cd2329af114682e03471ba75" target='_blank'>
              Single-Cell Mendelian Randomization Identifies Cell-Type Specific Genetic Drivers of Lung Cancer Subtypes
              </a>
            </td>
          <td>
            Q. Yang, C. Yan, X.-f. Wang
          </td>
          <td>2025-12-02</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer is a highly heterogeneous disease, driven by genomic and transcriptional changes that impact disease progression and metastatic potential. The interplay between clonal evolution, transcriptional plasticity, and tumour microenvironment is, however, poorly understood. Here, we leverage and integrate single-nuclei RNA sequencing and whole-genome sequencing from 43 spatially distinct tumour samples from five patients with locally advanced prostate cancer to reconstruct clonal evolution trajectories and transcriptional changes driving metastasis at single-cell resolution. We find extensive clonal heterogeneity, including both monophyletic and polyphyletic metastatic dissemination, and ongoing clonal evolution in the primary tumour after metastatic spread. Metastatic seeding converges on disease trajectories involving both genomic and transcriptional changes, including androgen receptor independence and activation of estrogen-, WNT- and JAK-STAT- pathway activity, in spatially distinct areas. Our findings suggest an intricate interplay between clonal evolution and cellular plasticity driving metastatic seeding and point toward more integrative prognostic markers for improved patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea384ac37d8c4c2acd2bd5fc9ab62cffd7ea81c" target='_blank'>
              Clonal evolution and transcriptional plasticity shape metastatic dissemination routes in prostate cancer
              </a>
            </td>
          <td>
            Migle Mikutenaite, Evdoxia Karadoulama, F. Favero, A. Locallo, Francisco German Rodriguez Gonzalez, Daria Kiriy, Tina Keshavarzian, K. Furlano, M. Graefen, Robert G. Bristow, Clarissa Gerhäuser, C. Plass, G. Sauter, R. Simon, M. Lupien, S. Minner, Thorsten Schlomm, Joachim Weischenfeldt
          </td>
          <td>2025-12-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is the most aggressive primary brain tumor in adults, marked by rapid progression and poor prognosis. GBM may also harbor a complex virome, defined here as the total repertoire of viral elements within tumors, encompassing both exogenous viruses and reactivated endogenous retroviruses. While viral signatures have been observed in other cancers, their transcriptional activity in GBM and their distribution across tumor cells and extracellular compartments remain largely unexplored. Methods We analyzed 45 long- and small-RNA libraries from glioma stem-like cells, their extracellular fractions (exosomes, microvesicles, ribonucleoprotein complexes), and normal brain controls (BioProject PRJNA360129). Taxonomic profiling and functional annotation were performed using established pipelines (MetaPhlAn4, BLASTn, UniProtKB, InterPro), with fresh media controls to exclude background contamination. To validate discriminative power, viral RNA profiles were classified using machine learning -Random Forest models trained on nine species identified as glioblastoma-enriched by LEfSe. Results We detected ~3.41 million viral reads, with >99% from GBM cells and extracellular compartments and <1% from normal brain. Reads spanned 36 species across 11 phyla. Nine species were consistently enriched in glioblastoma: Saimiriine gammaherpesvirus 2, Human endogenous retrovirus K, Camelpox virus, Pestivirus A, Macacine gammaherpesvirus 5, Avian myelocytomatosis virus, Rabbit fibroma virus, Omegapapillomavirus 1, and Cyprinid herpesvirus 2 with genes mapping to oncogenic pathways. Random Forest models showed long-RNA profiles perfectly discriminated against tumor from normal samples (AUC = 1.0), whereas small-RNA profiles carried limited signal (AUC = 0.5). Tumor–extracellular comparisons revealed strong compartment-specific long-RNA signatures (AUC = 1.0), with partial overlap between tumor cells and exosomes in small-RNA data (AUC = 0.75). Conclusion Glioblastoma contains a distinct viral transcriptome; long-RNA profiles show strong diagnostic power, and compartment-specific signatures suggest biomarker potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03aa5d91d0d223768b844afb632280b82ecb5f9" target='_blank'>
              The glioblastoma virome across tumor and extracellular compartments
              </a>
            </td>
          <td>
            M. Ghorbani, H. Karimi, Farnoosh Solati, Nazanin Anbarestani, Nooshin Khoshdoozmasouleh
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common and deadliest primary brain tumor, with no effective therapies and a longstanding record of failed phase 3 clinical trials—often without mechanistic insight into why promising approaches fall short. To address this critical gap, we implemented a novel on-therapy, longitudinal, and multi-regional tissue sampling strategy in patients treated with rQNestin34.5v2, an engineered oncolytic biologic based on herpes simplex virus 1, to explore how tumor and immune ecosystems evolve during treatment. Using high-resolution spatial transcriptomics (Xenium) and targeted spatial proteomics (CyCIF), we profiled over 80 spatially and temporally distinct samples from six patients, capturing over half a million immune, stromal, and malignant cells. This unique dataset enabled unprecedented resolution of the dynamic interactions between tumor and immune compartments in situ over the course of therapy. Our preliminary analysis revealed discrete, virus-associated lymphoid-like structures that formed early after treatment initiation. These structures were enriched in T cells, B cells, and myeloid cells, and were associated with localized reductions in tumor cell density. Over time, we observed a shift from an initially inflamed microenvironment to a more immunosuppressive state. By the time of radiographic progression, these lymphoid structures had largely dissipated—underscoring the importance of on-treatment sampling in capturing transient, treatment-responsive immune architectures. While these findings are preliminary, they suggest a transient immune structure linked to early therapeutic response, whose collapse may contribute to treatment resistance. Our study illustrates how integrative, on-therapy spatial profiling can begin to reveal mechanisms of response and resistance in GBM, providing a framework for refining future immunotherapeutic strategies in brain tumors and other treatment-refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf60db76ebc26534935de49d68b0018aa310569" target='_blank'>
              IMMU-67. On-Treatment Spatiotemporal Profiling of Glioblastoma During Oncolytic ImmunoTherapy
              </a>
            </td>
          <td>
            Charles Couturier, Jennifer Gantchev, Gerard Baquer, Peter Zhang, Alicia D’Souza, Michael S Regan, Kaitlyn Bootz, J. Bonney, Md Amin Hossain, G. Teamlab, Atul Deshpande, Forest M. White, Kenny K. H. Yu, E. Chiocca, Nathalie Y R Agar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains among the most aggressive and intractable human cancers, driven by profound intratumoral heterogeneity, dynamic cellular plasticity, and robust mechanisms of immune escape. To overcome these barriers, we developed an integrated discovery and engineering pipeline that combines single-nucleus RNA sequencing (snRNA-seq), antibody generation through both immunization and generative AI design, yeast-based directed evolution, and rationally engineered CAR-T cell architectures. We first assembled a high-resolution snRNA-seq atlas spanning malignant GBM, adult brain, and developmental brain tissues. Cross-comparative analysis revealed PTPRZ1 as a consistently expressed, tumor-enriched surface antigen with minimal expression in normal adult tissues. Further analysis across developmental datasets confirmed PTPRZ1 expression during neurodevelopment, particularly in glioma stem cells and oligodendrocyte progenitor-like states, and showed retention of expression across all major GBM differentiation trajectories. These findings suggest that PTPRZ1 marks a persistent cellular state within the glioma hierarchy, making it a stable and tractable immunotherapeutic target despite GBM’s plasticity. To generate high-affinity binders against PTPRZ1, we pursued two parallel strategies: (1) phage display from immunized hosts followed by affinity maturation via yeast-based directed evolution, and (2) de novo nanobody design using structure-aware generative models trained on antibody sequence and structural datasets. Candidate binders were refined through site-directed mutagenesis to enhance specificity and binding kinetics. Given the known challenges of targeting PTPRZ1, including extracellular domain shedding and glycosylation-dependent surface stability, we engineered a suite of TCR-mimetic CAR-T cells incorporating modular costimulatory domains to enable tunable activation thresholds and improved resistance to exhaustion. These constructs emulate core features of physiological TCR signaling, offering increased antigen sensitivity, functional persistence, and reduced tonic signaling in hostile tumor contexts. This work establishes a modular platform for precision antigen discovery and CAR-T cell engineering in GBM and validates PTPRZ1 as a compelling, developmentally anchored immunotherapeutic target. More broadly, it offers a scalable roadmap for rational design of next-generation cellular therapies against solid tumors that evade conventional targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/693e783962995ec2b156273884091139270f0e5f" target='_blank'>
              EXTH-68. Targeting Tumor Heterogeneity And Plasticity in Glioblastoma With Novel CAR T Cells Targeting PTPRZ1
              </a>
            </td>
          <td>
            Aditya Mohan, Barbara Lipes, Charla Gentry, Rushil Yerabelli, Kisha Patel, Bhairavy J Puviindran, E. Blandford, Ariel Gonzalez, Daniel S. Wilkinson, John H. Sampson, Michael Gunn, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Clonal hematopoiesis of indeterminate potential (CHIP) is a precursor state characterized by expansion of blood clones carrying somatic mutations that originate in the hematopoietic stem cells (HSCs). Individuals with CHIP have a 10-fold increased risk of developing myeloid malignancies such as myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). Progression of CHIP to MDS/AML takes several decades and hence targeting the CHIP clones during this period provides a viable strategy to prevent MDS/AML. Despite its importance, we still don't understand the mechanisms of CHIP predisposition and its progression. Recent genome wide association studies (GWAS) have identified several non-coding genetic loci that are strongly associated with CHIP, but the causal variants remain unknown because most of the variants are found in non-coding genomic regions. We have previously shown thatmodulation of gene regulatory elements in HSCsunderlies genetic risk of myeloid malignancies; whether this is also true for premalignant state such as CHIP is currently unknown.
 We hypothesize that germline CHIP risk variants in non-coding loci affect enhancer elements in HSCs. In order to identify these variants we utilized a high throughput approach termed Massively Parallel Reporter Assay (MPRA) to simultaneously screen for regulatory effects of 1,374 non-coding variants within 24 GWAS loci from a comprehensive study of CHIP risk on ~600,000 individuals. In the MPRA approach, we synthesized pools of barcoded lentiviral constructs harboring non-coding variants (risk and non-risk alleles) and surrounding genomic sequence (250bp) upstream of a minimal promoter driving a reporter gene. Sequencing of the plasmid pool confirmed an uniform distribution of barcodes with a median of around 50 barcodes per variant. Since enhancers show cell-type specific activity, we next searched for the appropriate cell-type to perform the MPRA. As the disease relevant population in CHIP is the HSCs, which is rare, we searched for alternative cell types. We identified MUTZ-3, a AML cell line that maintains a self-renewing CD34+ population and is dependent on HSC transcription factors. We confirmed that MUTZ-3 cells have trans-acting factors similar to primary HSPCs, by testing several validated HSPC enhancers including PU.1 upstream regulatory element (URE) and GATA2 inv(3) enhancer using individual reporter assays.
 We performed our lentiviral MPRA screen using a library of ~73,000 constructs in CD34+ fraction of MUTZ-3 cells. 48 hours post-infection we lysed the cells for DNA and RNA and sequenced the barcodes by NextSeq. The experiments were highly reproducible and all 5 replicates correlated well with each other at both the DNA and RNA barcode level. When we measure the enhancer activity by ratio of RNA to DNA barcode read counts, positive controls in the pool (PU.1 URE, GATA2 enhancer) showed higher activity compared to negative ORF controls. The activity of test constructs that overlap with hematopoietic ATAC peaks was higher that non-overlapping constructs indicating MPRA can identify endogenous enhancers. We identified several hits that showed allele specific reporter activity based on CHIP variant (Ref and Alt allele).
 We prioritized rs17834140, a variant identified by the MPRA for further validation studies. The variant is located within the 4th intron of the MSI2 gene in an hematopoietic ATAC peak and MPRA and individual reporter assays indicate enhancer activity and the risk allele increases MSI2 expression. We further confirmed endogenous enhancer activity by both CRISPR/Cas9 deletions and CRISPR inactivation experiments in human hematopoietic cells. To test whether MSI2 overexpression can also contribute to clonal expansion of CHIP mutant HSCs, we infected LSK cells from TET2 knockout mice and wildtype controls with the MSI2 overexpressing and control lentiviral vectors and observed an increased in vitro serial colony replating ability of TET2 cells upon MSI2 upregulation.
 Our study paves the way to systematically assess non-coding risk variants associated with CHIP risk. We expect the MPRA approach will help to draw out key unknown myeloid neoplasm predisposition genes and their regulatory elements that are influenced by the germline variants.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691b91de07b0de85cfcd28321c80391149dbd376" target='_blank'>
              Assessing germline noncoding risk variants impacting clonal hematopoiesis using a massively parallel reporter assay
              </a>
            </td>
          <td>
            Jessica Jeejan, Trieu Nguyen, S. Kales, Chie Yanase, Dominika Kwasniak, Aruna Shekhar Nannivala Chandra Shekhar, Gowri Poigaialwar, Richard Voit, Keisuke Ito, Ryan Tewhey, Satish Nandakumar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 The immune surveillance hypothesis posits that the immune system controls pre-cancerous cells prior to tumor formation. However, direct visualization of the process has been nearly impossible, because pre-cancerous cells are difficult to identify in conventional cancer models. To address this technical gap, our lab established a unique mouse model called MADM (Mosaic Analysis with Double Markers) that generates rare mutant cells with unequivocal GFP labeling. In a MADM model for triple negative breast cancer, we have discovered the specific and prominent presence of T/B cell-dominated immune aggregates adjacent to premalignant mutant ducts. Characterization via flow cytometry and spatial staining has revealed these aggregates are heterogenous in size, composition, and organization, and tend to dissipate as malignant tumors form. Interestingly, a significant portion of T cells in these aggregates are CD4+ Foxp3+ Tregs, which typically play an important role in preventing excess inflammation. We hypothesize that Tregs regulate immune surveillance against premalignancy to prevent excess inflammation/autoimmunity, but in doing so also prevent eradication of premalignant cells. We are testing this hypothesis using Treg ablation during premalignancy and expect that Treg ablated mice will show increased T/B cell presence and activation in the mammary gland and decreased premalignant cell expansion.



 Supported by the Basser Center for BRCA; University of Virginia (UVA) Pinn Scholarship; NIH/NCI R01-CA256199; UVA Cancer Center Training Grant



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e28dd574c9b2b053c06b1da358808d0a549f6508" target='_blank'>
              Uncovering the role of Tregs in immune surveillance against cancer with a mouse model that reveals premalignancy at single cell resolution 3379
              </a>
            </td>
          <td>
            Elaina K. Ball, Xian Zhou, Jianhao Zeng, Alexys Riddick, Hui Zong
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5dbf13eb9670595afcc13d733cac2504d66e7a" target='_blank'>
              Single-cell aggrephagy landscape delineates intercellular communication of tumor microenvironment associated with ovarian cancer progression and immunotherapy.
              </a>
            </td>
          <td>
            Feng Fang, Jiemiao Chen, Xinran Li, Cao Zhou, Yilan Dai, Yuanyuan Chen, Xin Yang, Yong Xie, Fujie Qin, Huijiao Fu
          </td>
          <td>2025-11-24</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Meningiomas represent the most prevalent primary intracranial tumors, with the majority classified as WHO grade 1, typically benign tumors exhibiting slow growth. In contrast, WHO grade 2 and 3 meningiomas display increased histopathological aggressiveness, characterized by increased growth, recurrence rates, invasiveness and poorer prognoses. Emerging evidence suggests that tumor microenvironment (TME) plays a critical role in tumor aggressiveness; however, its contribution to meningioma pathobiology is poorly understood. This study aims to establish a patient derived orthotopic xenograft (PDOX) meningioma model to elucidate the role of the TME in driving tumor aggressiveness and to identify novel therapeutic targets. To this end, primary tumor cells derived from meningiomas of various WHO grades were isolated and grown as monolayer cultures, spheroids, and organoids. Additionally, the immortalized meningioma WHO Grade 3 IOMM-Lee cell line was used to represent high-grade tumors. To investigate tumor formation and TME, these cells were implanted intracranially into NMRI nude mice. Tumor development was confirmed via MRI. Subsequently, whole brains were collected either 6-9 months post-inoculation or upon symptom onset, followed by sectioning and analysis through immunohistochemistry. Moreover, tissue samples are being collected for spatial transcriptomics to further investigate molecular profiles. Preliminary analysis of brains implanted with IOMM-Lee cells revealed a 100% tumor take rate, with high Ki67 expression, indicative of significant proliferation. A notable presence of microglia and macrophages (IBA1, CD204) was observed, suggesting involvement of the TME. Current efforts focus on expanding the cohort by including more patients with different grades of meningiomas. This model will provide valuable insights into the cellular composition, heterogeneity, and tumor-immune cell interactions. Ultimately, it will uncover potential therapeutic targets for meningioma and enable their functional validation. This project is supported by the European Union’s Horizon 2020 research and innovation programme under the Marie Sklodowska-Curie grant agreement No. 945322 (LEAD) and Lundbeck Foundation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3b2bddbecba83931920b06f1f448eed8ac49c4e" target='_blank'>
              TMIC-74. A patient-derived orthotopic meningioma model for interrogation of the tumor microenvironment
              </a>
            </td>
          <td>
            Pelin Gulizar Ersan, Claes Ottzen Laurentiussen, J. Haslund-Vinding, Josephine Budde, J. Lange, Mikael Håndstad, Shania Lashkari, N. Hammouda, Karina Stræde, Nadia Hajib, M. Andersen, Bo Halle, F. Poulsen, Andreas Kjaer, Frederik Vilhardt, T. Mathiesen, B. Kristensen
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (iCCA) is a highly metastatic tumor type, and iCCA with intrahepatic metastasis is associated with poor prognosis. Previous studies have confirmed that the tumor microenvironment (TME) is closely related to tumor metastasis. However, the state of the TME during iCCA liver metastasis remains unknown.


APPROACH AND RESULTS
We profiled 28 specimens from 16 patients with iCCA (with and without intrahepatic metastasis) using integrated single-cell and spatial transcriptomics, bulk RNA sequencing, whole-exome sequencing, and in vivo and in vitro functional experiments to characterize the specific TME changes during iCCA liver metastasis. we identified the metastatic tumor cells, COL3A1+ epithelial cells, and revealed that COL3A1+ epithelial cells-endothelial-to-mesenchymal transition cells crosstalk promoted tumor invasion by inducing mesenchymal transformation of endothelial cells, while COL3A1+ epithelial cells-VEGFA+ CCL4+ neutrophils crosstalk promoted tumor colonization by forming neutrophil extracellular traps.


CONCLUSIONS
We revealed the key biological mechanisms involved in the invasion and colonization phases of iCCA liver metastasis. Moreover, we provide valuable data for understanding iCCA liver metastasis and a possible avenue for the treatment of advanced iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd684325d487cd1d69724768d37142d24e7c6b41" target='_blank'>
              Spatiotemporal landscape of intrahepatic cholangiocarcinoma liver metastasis at the single-cell level.
              </a>
            </td>
          <td>
            Ziyu Xun, Hao Wang, Zhengfeng Xuan, Kexu Xiong, Na Zhang, J. Long, Huishan Sun, Yiran Li, Chengpei Zhu, M. Piao, Ting Zhang, Longhao Zhang, Shuofeng Li, Chengjie Li, Jiongyuan Li, Boyu Sun, Zi-Qi Zhou, Shanshan Wang, Ziyue Huang, Kai Liu, Yang Tan, Xiaohua Shi, Xiaobo Yang, Hanping Wang, Ling Lu, Haitao Zhao
          </td>
          <td>2025-12-10</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Cancer arises from genetic and epigenetic alterations that reprogram gene expression networks, driving unchecked proliferation. Glioblastoma (GBM), the most common and lethal primary brain tumor, has a median survival of about one year and a dismal 5% five-year survival rate. Despite extensive research, effective treatments remain elusive, underscoring the need for a deeper understanding of GBM biology. Although several genetic alterations have been implicated in gliomagenesis, driving genetic factors responsible for GBM are still obscure, and how these genetic changes interact with epigenetic dysregulation is poorly understood. Here, we identify CHD5, a gene located in the frequently deleted 1p36 chromosomal region, as recurrently deleted in over 20% of GBM cases and further downregulated epigenetically. To investigate the role of CHD5 in gliomagenesis, we generated a glioma-prone mouse model carrying conditional alleles of Pten, Trp53, and Chd5, and employed engineered neural stem cells (NSCs)—a major GBM cell-of-origin—to induce tumor formation. Loss of Chd5 impaired NSC differentiation, enhanced proliferation, and drove in vivo gliomagenesis. Mechanistically, Chd5 forms a NuRD complex that binds the Myc promoter and super-enhancers to regulate its expression. Chd5 loss leads to chromatin accessibility changes, Myc upregulation, and aggressive tumor development. Importantly, reactivating CHD5 in human GBM xenografts significantly extended survival. These findings establish CHD5 as a tumor suppressor in gliomagenesis, offer therapeutic insights for cancers with 1p36 deletions, and present a robust NSC-based platform for functional genetic studies and modeling GBM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccecf93321da1d6d66641061e1225be0f52f0874" target='_blank'>
              EPCO-68. CHD5 IS A TUMOR SUPPRESSOR IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Xueqin Sun, A. Mills
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Over 75% of PTCL-NOS patients fail to achieve long-term control with current therapies, highlighting the need for novel therapeutic targets. Genetic and non-genetic drivers in cancer cells require external cues from other cancer cells and from the lymphoma microenvironment (LME), which enable cancer cell adaptation to metabolic stress, immune mediated stress and treatment exposure. The resulting signaling crosstalk propels the co-evolution of cancer and LME cells with emergence of diverse cancer-supporting cell states. Identifying these and their maintenance mechanisms may therefore reveal novel therapeutic targets.
 We characterized the LME of PTCL using transcriptional signature deconvolution and multiparametric imaging in 845 patient samples. We identified four biologically and clinically distinct categories named Depleted (DP), B-cell rich (BR), Mesenchymal (MS) and Inflammatory (IN) based on the proportion of different LME cells. RNA-seq analysis showed that SMA+ cancer associated fibroblasts (CAF) are present in DP, MS and IN LMEs each with distinct phenotypes. To functionally analyze the contribution of CAFs to PTCL maintenance, we established multiple patient-derived tumor xenografts (PDXs) that retain the same LME categories of primary tumors. We then tested lymphoma cell survival in co-cultures of CAFs and PTCL cells from matched (e.g., IN-CAF + IN-PTCL) and unmatched PDXs (e.g., IN-CAF + DP-PTCL). Matched co-cultures sustained lymphoma cell proliferation to a higher extent than unmatched, independently of LME subtype. This suggests that CAF are distinctly reprogrammed within each LME.
 We hypothesize that pro-tumoral CAF phenotypes require adaptive gene expression in coordination with lymphoma cells' phenotypes. RNA-Seq of matched IN-CAF co-cultured with IN-PTCL revealed activation of pathways like “Transcription”, “DNA replication” and “Translation” including the upregulation of CDK7. Although CDK7 plays a minor role in cell cycle progression, its phosphorylation of the RNA polymerase II as part of the TFIIH complex is critical in activating and coordinating “cell state”-defining transcription.
 To assess CDK7's role in establishing lymphoma-supportive CAF states, IN-PTCL PDX mice were treated with the covalent selective CDK7 inhibitor YKL-5-124 for 12 h followed by transcriptomic and proteomic profiling of IN-PTCL and IN-CAFs. In IN-CAFs, CDK7 inhibition altered gene and protein expression of several collagens and cytokines, including COL1A1, COL1A2, MMP8, and CXCL12. Ligand-receptor analysis identified CXCL12-CXCR4 as the key CDK7-dependent interaction between IN-CAFs and IN-PTCL cells. IN-PTCL cells showed activation of DNA damage repair pathways, including upregulation of ALYREF and EIF4E, likely as compensatory mechanisms for decreased CDK7-dependent transcription. Since these may represent acquired therapeutic vulnerabilities, we conducted viability screenings in co-cultures of IN-PTCL and IN-CAF isolated from YKL-5-124 (and vehicle) tretated PDX mice. Cells were subsequently exposed for 48 h, individually and in co-cultures, to a drug library containing 40 clinical-phase compounds. This approach led to the identification of 14 compounds with synergistic anti-lymphoma effects. Among these, ten drugs including the XPO1 inhibitor Selinexor, the HSP90 inhibitor PU-H71 and the translation inhibitor Omacetaxine exhibited enhanced anti-lymphoma activity specifically in the co-culture setting, whereas four compounds showed reduced efficacy. This was not seen in mismatched co-cultures and indicates a potential protective role of CAFs and, importantly, that this mechanism is CDK7 dependent.
 To address the limitations of PDX models established in immunodeficient mice with limited ability to capture the immune-mediated effects of drug treatments, we developed an IN-PTCL syngeneic model by intrasplenic implantation of the murine T-cell lymphoma cell line EL4. Treatment with YKL-5-124 significantly reduced the tumor burden compared to vehicles. Cytokine profiling (immunoblotting array) and ELISA of culture medium of YKL-5-124 treated EL4-CAFs showed reduced levels of pro-inflammatory cytokines such as CXCL12, IL6, CCL2, ICAM-1, and CXCL1 indicating that CDK7 is necessary in establishing pro-lymphomagenic IN-CAF phenotypes.
 In conclusion, CDK7 activity is required to sustain specific pro-tumoral crosstalk interactions of matched CAFs and PTCL cells, which can be exploited therapeutically.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386e7a67f6fb5e2e6ca873d1aaf4d401efdcd89c" target='_blank'>
              CDK7 enables adaptive gene transcription required by stromal cells to support distinct T-cell lymphoma phenotypes and represents a novel microenvironment-directed therapeutic target
              </a>
            </td>
          <td>
            K. Kouidri, D. Fiore, M. Revuelta, N. Di Siervi, Federico Ruiz Moreno, Subhashini Joshi, C. Kayembe, Abigail Taylor, G. Inghirami, L. Cerchietti, N. Zamponi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 For glioblastoma (GBM), like other solid tumors, intratumoral heterogeneity is likely a critical factor in promoting tumor growth, homeostasis, and therapeutic response. Recent applications of single-cell genomics techniques have revealed the scope and nature of GBM cell diversity. However, these techniques do not allow for the experimental manipulation of these same tumor subpopulations, raising a critical barrier to understanding tumor biology and developing effective therapies. To overcome this barrier, we developed endogenous gene reporter systems to enable the isolation, characterization, and manipulation of key GBM subpopulations directly in patient tumor isolates. This approach utilizes streamlined techniques for gene editing in GBM stem-like cells (GSCs), which eliminate the need for the cloning of individual cells. As proof of concept, we applied this approach to two genes in hGSC isolates: OLIG1 (GBM-OPC marker) and SOX9 (GSC/NSC stem cell homeostasis). We targeted the 3’ exon of each gene so that EGFP or RFP are in-frame with the upstream exon but separated by a 2A ribosome skip sequence. We then employed these GSC reporter isolates to comprehensively screen human transcription factors using a CRISPR-Cas9 library and LV-ORF library (1836 genes and 3548 isoforms) to identify transcription factors that activate or repress OLIG1 and SOX9. From these screens we identified 15 genes necessary and sufficient for regulating OLIG1 promoter activity, including known and novel regulators. These included SOX9 itself, which we find is a key factor linking OPC identify with self-renewal of GSCs. In addition, we identified dozens of transcription factor isoforms (validated by long read RNA-seq) that impact reporter activity, as well as ectopic factors not normally expressed in our GSCs which, nonetheless, significantly alter OLIG1 and SOX9 expression. At this meeting, we report results of these screens and present a functional genetic map of factors affecting the GBM-OPC cell identity, self-renewal, and plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9966806d724ae473e21ea19603f70a36860465" target='_blank'>
              STEM-26. COMPREHENSIVE FUNCTIONAL GENOMIC SCREENS REVEAL HUMAN TRANSCRIPTION FACTORS REQUIRED TO MAINTAIN GLIOBLASTOMA OLIGODENDROCYTE PROGENITOR CELL-LIKE IDENTITY
              </a>
            </td>
          <td>
            Anca B. Mihalas, Kelly Mitchell, Sonali Arora, Colette Felton, Mikey Witkowski, Aiden Abadilla, Daniel A. Kuppers, Angela Brooks, Alice Berger, Patrick J. Paddison
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Brain metastases are frequent and often lethal complications of advanced cancers. Microglia, resident immune cells of the brain, are known to exert both anti-tumor and pro-tumor functions in late-stage metastases; however, their response during the initial outgrowth of metastatic lesions is not well characterized. Understanding how heterogeneous microglial subgroups are regulated in the developing tumor microenvironment could pave the way for therapeutic strategies to eliminate metastatic tumors at an early stage. In this study, we used a combination of in vivo fate map imaging, single-cell RNA sequencing, and a holographic photoconversion-based technique (Opto-omics) to track tumor fate and early microglial responses over time in the same animals during colonization of disseminated tumor cells. The microglial population was transcriptionally and morphologically heterogeneous, comprising both pro- and anti-tumor subsets. Genetic and pharmacological perturbations revealed that microglial phenotypes could be shifted by inhibiting TGF-β signaling or by deleting the tumor cell surface antigens CD24a and CD47. These findings reveal targetable plasticity in early-stage microglial responses to brain metastasis and suggest that harnessing pro-phagocytic microglial states may offer a therapeutic window before systemic immunosuppression becomes dominant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1778e4e2ee1302e3145504143572e313e56713fc" target='_blank'>
              Microglia Display Heterogeneous Initial Responses to Disseminated Tumor Cells.
              </a>
            </td>
          <td>
            T. Tsuji, Haruka Hirose, Daisuke Sugiyama, Mariko Shindo, Rahadian Yudo Hartantyo, Yutaro Saito, Tsuyako Tatematsu, Shouta Sugio, M. Sanbo, Masumi Hirabayashi, Yasuhiro Kojima, Jun Koseki, K. Hosoya, Hiroshi Yoshida, Tatsuya Ogimoto, Y. Yasuda, Kentaro Hashimoto, Hitomi Ajimizu, Yuichi Sakamori, H. Yoshida, Noritaka Sano, Masahiro Tanji, Hiroaki Ito, Kazuhiro Terada, M. Hamaji, T. Menju, Hiroyuki Konishi, S. Kumagai, Cyrus M. Ghajar, Daisuke Kato, Hiroshi Date, Akihiko Yoshizawa, Yoshiki Arakawa, H. Ozasa, A. Moorhouse, Teppei Shimamura, H. Nishikawa, Toyohiro Hirai, Hiroaki Wake
          </td>
          <td>2025-12-10</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/128cca3d1edd8a7877ad22a89e79521ff40bea75" target='_blank'>
              Single-cell sequencing in bladder cancer: new insights from tumor cell diversity to individualized treatment strategies.
              </a>
            </td>
          <td>
            Cen Liufu, Shuai Ye, Cong Yin, Tao Zhu, Jiahao Jiang, Yan Wang, Bentao Shi
          </td>
          <td>2025-12-03</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a leading male malignancy worldwide, with metabolic reprogramming being a critical hallmark of its progression. Extracellular vesicles (EVs) derived from tissues directly reflect the tumor microenvironment, offering unique insights into cancer pathophysiology that are unattainable through cell line or biofluid-derived EVs. However, the functional roles of tissue-derived EVs in PCa metabolism remain poorly understood. Leveraging our expertise in murine PCa model establishment and EV isolation from prostate tissue, this study aimed to characterize functional differences between PCa and normal prostate tissue via proteomic analysis of tissue-derived sEVs. We orthotopically implanted luciferase-labeled PCa cells into nude mice to establish an in situ PCa model, confirmed tumor formation via in vivo imaging, and harvested tissues after 4 weeks. sEVs were isolated using ultracentrifugation combined with an iodixanol density cushion and characterized by transmission electron microscopy, nanoparticle tracking analysis, and protein marker profiling. Proteomic analysis identified 28 upregulated and 24 downregulated proteins in PCa-derived sEVs compared to normal controls. Subcellular localization revealed enrichment in the cytoplasm, while pathway analysis highlighted significant involvement in metabolic processes, particularly glycolysis, amino acid biogenesis, carbon metabolism, and pyruvate metabolism. Our study establishes a robust method for isolating prostate tissue sEVs and provides the first evidence that PCa tissue-derived sEVs exhibit profound metabolic pathway alterations. These findings enhance our understanding of PCa progression mechanisms and may facilitate the development of novel diagnostic biomarkers and therapeutic strategies targeting metabolic dysregulation in PCa. SUMMARY: In this study, we created a method to isolate prostate tissue small EVs, based on our knowledge of the murine prostate cancer model building. Our data suggested that prostate tissue small EVs proteins significantly changed in many metabolism pathways, such as Glycolysis, Biogenesis of amino acids, Carbon metabolism and Pyruvate metabolism. In this study, we are the first to report prostate tissue-derived EVs proteins enriched in alterations of cancer metabolism. These differential proteins in PCa tissue EVs reflect metabolic changes in PCa and may provide insights into the development of early diagnostic biomarkers or novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c976599295f2bbc1baf63bb2a61d05088e5e38" target='_blank'>
              Proteomics of Prostate Cancer Tissue Small Extracellular Vesicles Reveal Alteration of Metabolism.
              </a>
            </td>
          <td>
            Lijuan Yu, Ting Ding, R. Olofsson Bagge, Lei Zheng, Xiaoke Hao
          </td>
          <td>2025-11-17</td>
          <td>Proteomics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background CD47 is a “don’t eat me” signal that is overexpressed in tumors to evade phagocytosis by tumor associated macrophages (TAM). Investigational agents targeting CD47, such as magrolimab, aim to induce phagocytosis of tumor cells by TAMs. Previously, two key TAM subsets have been identified: C1QC TAMs, which display pro-phagocytic activity, and SPP1 TAMs that express pro-angiogenic markers. We characterize CD47 expression and its relationship with tumor macrophages in solid tumor samples. Patients and methods Resectable tumors from head and neck squamous cell carcinoma (n=36) (HNSCC), breast cancer (n=37) (BC), and colorectal cancer (n=36) (CRC) were evaluated for CD47 expression by immunohistochemistry (IHC), two multiplex immunofluorescence panels were used to characterize TAM markers and T cell markers. RNA sequencing was also performed. Results CD47 protein expression was higher on tumor cells compared to stromal cells across tumor indications tested. Although CRC had the lowest prevalence for CD47 expression in primary tumors, we observed a marked increase in CD47 expression in CRC liver metastases. We developed an SPP1 TAM gene signature and validated a C1QC TAM gene signature to estimate TAM abundances from bulk RNA-Seq data. In the TAM mIF analysis, HNSCC had the highest macrophage density of the indications tested. We observed a positive correlation between a higher C1QC: SPP1 TAM ratio and macrophage phenotype and tumor T cell density. C1QC macrophage signatures correlate with tumor CD47 protein expression in BC and HNSCC samples, suggesting interplay between them. Conclusions We characterized CD47 expression across key solid tumor indications being evaluated clinically using anti-CD47 blockade agents: HNSCC, breast cancer and CRC. Using a CD47 IHC assay, we identified HNSCC as an indication with the highest CD47 expression. In addition, we quantified tumor macrophages using multiplex immunofluorescence (mIF) and determined that HNSCC had the highest density of TAMs. Compared to relatively low CD47 expression in primary CRC tumors, CRC liver metastases had very high CD47 expression. Quantification of TAM signatures and CD47 expression represent key biomarkers to monitor in patient samples during exploration of CD47-blockade agents in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31725c03ca6311073138a838325de55dcf4fe468" target='_blank'>
              CD47 expression in solid tumors correlates with phagocytic tumor-associated macrophage gene signature
              </a>
            </td>
          <td>
            Nicholas van Buuren, Mengshu Xu, Yi Zhang, Paola Correa, Shiva Zaboli, Azadeh Madjidi, Christina Moon, Szu-Wen Liu, Ruidong Li, Kai Sun, Shahed Iqbal, Abhishek Aggarwal, Min Wang, Li Li, Jared M Odegard, Kelli Boyd
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is a highly aggressive malignancy with a heterogeneous molecular landscape. This study investigates the interplay between mRNA subtypes and the tumor immune microenvironment (TIME) in HGSOC. We analyzed transcriptomic data to characterize four mRNA subtypes: immunoreactive, differentiated, proliferative, and mesenchymal. Using DESeq2 and WGCNA, we identified gene modules associated with clinical traits and immune cell infiltration. The immunoreactive subtype exhibited significant enrichment of M1/M2 macrophages, Th1 cells, and B cells, along with LAIR-1, FCRL5, and IRF4 expression. The mesenchymal subtype was associated with fibroblasts, while the proliferative subtype showed strong ovarian stromal cell involvement. Our findings reveal distinct immune landscapes associated with each mRNA subtype, suggesting potential therapeutic targets and prognostic markers for HGSOC.



 NIH/NCI U54CA118638; Chan Zuckerberg Initiative CZIF2002-007045



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aa36312916ae96eb8236af900b44415c56ff2fa" target='_blank'>
              Transcriptomic subtypes and immune infiltration in high-grade serous ovarian cancer 4644
              </a>
            </td>
          <td>
            Kaylin M. Carey, C. Young, Alexis J Clark, Eric D Dammer, Rajesh Singh, J. Lillard
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Due to their intrinsic heterogeneity and plasticity, pediatric brain tumors present highly complex clinical challenges. To provide insight into the molecular scene of these tumors, we generated spatial transcriptomics data from 19 distinct patients, 7 of which suffered a relapse of the disease. In this cohort, spanning 59 tissue sections across 8 diagnoses and 4 tumor grades, we recovered diagnosis-specific gene expression patterns that could be linked to processes characteristic of developmental stages. We also identified between 2 to 4 spatial archetypal niches per section, which could then be related to 11 main biological themes, and to distinct celltype-like transcriptomic signatures. For example, we noted strong spatial correlations between developmental archetypes and oligodendrocyte lineage signal in pilocytic astrocytoma. Lastly, we utilised spatially inferred copy-number variants for the profiling of relapse-associated tentative tumor clones. These clones were then related via spatial geographical analysis to regions with strong blood vessel signatures. Overall, these results provide vital details into the progression and maintenance of pediatric brain tumors, offering novel edges to be exploited in personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f69864dd9511e02d9a815817c172642be7190a7d" target='_blank'>
              Surveying the molecular landscape of pediatric brain tumors via spatially resolved transcriptomics
              </a>
            </td>
          <td>
            Javier Escudero Morlanes, Timo-Pekka K Lehto, L. Larsson, L. A. Galicia, A. Mollbrink, Alia Shamikh, Elisa Basmaci, Gabriela Prochazka, T. de Ståhl, Johanna Sandgren, F. Taylan, Bianca Tesi, Ann Nordgren, Andrew Erickson, Alastair D Lamb, Klas Blomgren, Monica Nistér, J. Lundeberg, R. Mirzazadeh, L. Kvastad
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="


 Introduction: Subclonal diversity in multiple myeloma (MM)—defined by genetically distinct populations of malignant plasma cells—plays a pivotal role in disease progression, therapeutic resistance, and relapse. Characterizing subclone-specific transcriptomic profiles, copy number alterations, somatic variants, and their interactions with the immune microenvironment is critical for understanding tumor heterogeneity and identifying novel therapeutic targets. In this study, we analyzed smoldering multiple myeloma (SMM) and newly diagnosed multiple myeloma (NDMM) samples to explore the genomic landscape of subclones and their crosstalk with immune cells.
 Methods: CD138+ scRNA-seq and scATAC-seq multiomic data from SMM patients (n=10) and NDMM patients (n=21) were obtained from the published study dbGAP phs003220 (PMC11099140). Matched CD138- scRNA-seq data of immune cells were sourced from Sudha et al., IMS 2023. Subclones for each tumor sample were identified using inferCNV (v1.8.1) and a custom-built integration pipeline that utilized both scRNA-seq and scATAC-seq data. Cell-cell communications between subclones and immune cell types were identified using CellChat (v1.6.0) using the ligand-receptor interactions from CellChatDB. Copy Number Abnormality (CNA) profiles of subclones from single cell were compared with whole genome sequencing (WGS) data (phs003220) and similarity scores were computed. Due to differing technologies (RNA-seq vs WGS) and sequencing depth (single cell vs. bulk), this score reflects how well single-cell multiomics captures major chromosomal changes seen in WGS. Subclone-to-immune cell interactions were quantified to identify statistically significant chromosomal abnormalities that change their prevalence. Survival analysis was performed on the most common MM abnormalities to identify if subclonal co-occurrence influences prognosis. For each subclone, BAM files were generated, and somatic variants (SNVs + Indels) were called using Strelka2 with WGS control samples from the same patient.
 Results: We identified 23 subclones in SMM with a mean of 2.30 (min= 1; max=4); and 79 subclones in NDMM with a mean of 3.59 (min=1; max=8). Subclone and WGS CNAs were comparable to one another with mean Jaccard index (JI) of 0.57 for SMM and 0.69 for NDMM. Samples with low JI tended to have more subclones and more complex CNAs as measured through ploidy (correlation coefficient (r) = -0.069 in SMM and -0.177 in NDMM). NDMM subclones tended to have more CNAs than in SMM. From the IU cohort, Del13 was observed in 60% of the SMM and 45% of the NDMM patients. Gain1q was identified in 50% of the SMM and NDMM patients. Patients with co-occurring Gain1q and Del13q in the same subclone had significantly shorter progression-free survival (PFS) than patients with at least one of these alterations (median PFS 285 days vs 1194 days; P = 0.004). Subclones with Gain1q had a lower average percentage of interactions with CD8T cells (Gain1q=25.435, Norm1q=33.26; P=0.005). In NDMM subclones with Gain1q, there was a lower percentage of MHC-I pathway interactions with CD8T cells (P=0.001) and MHC-II pathway interactions with CD4T cells (P=0.001). Subclones with Del13q showed a higher percentage of MIF pathway (MIF-CD74/CXCR4) interactions in NDMM. SMM subclones with Del13p tended to have a lower percentage of MHC-I interactions in CD8T, CD4T and NK cells. Gain6p subclones had a higher percentage of MK pathway (MDK-NCL/ITGA4+ITGB1) interactions in SMM subclones. Somatic variants were identified for each subclone from the scATAC-seq. When compared with WGS somatic variants in NDMM, a median of 253 variants was found both subclones and WGS while 2752 were unique to subclones. Numerous pathogenic somatic variants were identified in the myeloma driver genes from subclones.Conclusions: The identification of subclone-specific immune interaction networks provides new opportunities for targeted therapeutic intervention. These results establish subclonal CNA profiling as both a prognostic tool and guide for precision immunotherapy in multiple myeloma, particularly for high-risk disease characterized by specific genomic alterations co-occurring in subclones. This study underscores the importance of single-cell analysis to fully capture the clinically relevant genomic heterogeneity that drives disease progression and treatment resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4387fcdd106b6a1f94375e3c3e641c2dbb1f9a" target='_blank'>
              Subclonal genomic heterogeneity shapes immune crosstalk and clinical outcomes in multiple myeloma
              </a>
            </td>
          <td>
            Parvathi Sudha, Sylvia Robertson, Sihong Li, Sean D. McCabe, Kelvin Lee, R. Abonour, Brian Walker, A. Suvannasankha, Travis S. Johnson
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abstract Tumor tissues are composed of malignant subclones with diverse genetic profiles. Reconstructing the evolutionary trajectory of these subclones is crucial for understanding how tumors acquire malignant traits. However, current approaches to subclonal tree reconstruction are limited either by their reliance on single-cell DNA sequencing (scDNA-seq) that involve a small number of cells and thus yield low-resolution results, or using single-cell RNA sequencing (scRNA-seq) data, which despite including larger cell populations, remain susceptible to bias from high dropout rates and technical noise. Here, we introduce CluVar, an autoencoder-based framework for inferring the phylogeny of cancer subclones from scRNA-seq data using mutation profile analysis. To address the extensive missing variant information inherent in scRNA-seq datasets, CluVar incorporates a customized loss function and multiple hidden layers optimized for clustering. CluVar demonstrated superior performance in reconstructing phylogenetic trees of cancer subclones under a range of erroneous conditions. When applied to cancer scRNA-seq data, the phylogenetic tree predicted using CluVar aligned well with the transcriptomic profiles. These findings highlight its utility for tracing evolutionary trajectories and identifying novel variants associated with cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68f982657d8bf3e302b1cdc665cd1c0c69419b27" target='_blank'>
              CluVar: clustering of variants using autoencoder for inferring cancer subclones from single cell RNA sequencing data
              </a>
            </td>
          <td>
            Chae Won Kim, Heewon Park, Dohyeon Kim, Yuchang Seong, Minhae Kwon, Junil Kim
          </td>
          <td>2025-11-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gastric cancer (GC) remains a major global health burden with persistently high mortality despite therapeutic advances. Accumulating evidence highlights the pivotal role of tumor-associated macrophages (TAMs) in orchestrating gastric tumor progression through immune suppression, angiogenesis, extracellular matrix remodeling, and metastasis. Within the tumor microenvironment (TME), TAMs exhibit functional plasticity, often polarizing toward an M2-like phenotype that promotes immunosuppression and tumorigenicity. These cells actively participate in immune evasion via immune checkpoint expression and cytokine-mediated T cell inhibition, while also facilitating lymphovascular invasion and chemoresistance through exosome-mediated crosstalk. The density and phenotype of TAMs have been associated with prognosis and therapeutic response in GC. Recent studies have proposed TAMs as promising targets for therapy, with strategies focusing on depleting M2 subsets, reprogramming toward M1 phenotypes, and blocking TAM-driven oncogenic signaling. Targeted interventions, including MENK, paclitaxel, and NF-κB inhibitors, have shown potential in preclinical models. This review comprehensively discusses the mechanistic roles of TAMs in GC and evaluates emerging TAM-targeted therapeutic strategies that may enhance the efficacy of immunotherapy and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/913bc3f4af4f9d7fed06d178714e6cfa582ea213" target='_blank'>
              Tumor associated macrophages in gastric cancer dual roles in immune evasion and clinical implications for targeted therapy
              </a>
            </td>
          <td>
            Chang Wang, Xu Fan, Xiaomeng Sun, Yi Xu, Yanna Sun, Jidong Liu
          </td>
          <td>2025-12-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCC) with multinodular morphology were typically polyclonal and exhibited extensive heterogeneity. We explored the multiple-layer heterogeneity of HCC utilizing single-cell multiomics sequencing together with multiregional sampling. We found that Confluent Multi-Nodular samples exhibit more heterogeneous immune landscapes characterized by increased transcriptome heterogeneity and more complex immune-related interactions compared to Single Nodular (SN) samples. We identified differential DNA methylation patterns among distinct tumor foci in a subset of liver cancer patients. Global DNA hypomethylation in cancer cells predominantly occurs in partially methylated domains. We predicted and validated two genes, GADD45A and SNHG6, as potential drivers of DNA hypomethylation. Lastly, we demonstrated that DNA methylation distance serves as a more robust metric than gene expression distance for reconstructing tumor evolutionary trajectories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eb1f83715ac49e3d497001d0455b6e605a23122" target='_blank'>
              Interrogating subclonal heterogeneity of liver cancer with single-cell multi-omics analysis
              </a>
            </td>
          <td>
            Liwen Zhang, Dandan Hu, Xing Peng, Dan Su, Lei Chang, Yi Guo, Kai Tian, Qingpei Huang, Enze Deng, Zhuojie Su, Longxi Li, Xiong Xiong, Weiyuan Ma, Hongjian Qi, Yaojun Zhang, Yanxiao Zhang, Xiaoying Fan
          </td>
          <td>2025-11-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Despite decades of concerted research and clinical efforts, patient outcomes in glioblastoma (GBM) remain dismal. While chimeric antigen receptor (CAR)- T cell therapies in GBM have produced individual reports of efficacy, they have yet to meaningfully alter standard of care. Tumor heterogeneity, both inter- and intra-patient, likely contributes to these variable responses. Single-cell RNA sequencing (scRNA-seq) conducted in our lab has revealed that this transcriptional heterogeneity is organized along distinct cellular state axes. We sought to overcome this heterogeneity by designing CAR-T cells directed against these cellular states. We hypothesize that CAR-T cells targeting a defined GBM cell state will lead to a depletion of those populations. We report the development of a panel of cell state-targeting CAR-T cell candidates engineered for target-specific activity in vitro. To model the three-dimensional tumor environment, we developed patient-derived GBM organoids that recapitulate the cell state landscape seen in primary tumors. We treated GBM organoids with these cell state-targeting CAR-T cells and assessed cell states by scRNA seq. Unexpectedly, we observed convergent transcriptome shifts to a new cell state with overlapping features of mesenchymal (MES) and interferon-induced signatures. To target this dynamic, we designed a novel CAR-T cell directed against this population and observed feedforward target enrichment in response to therapy. We anticipate that our findings will inform rational design of translational CAR-T cell candidates for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae77281b5ad33a2d2f93d13dc0c418dca38401a5" target='_blank'>
              EXTH-37. Evaluating cell state-targeting CAR-T cells in glioblastoma
              </a>
            </td>
          <td>
            S. Kovatsis, C. Mount, E. Boxer, Jun Zhong, S. Dumont, D. Gerovasilis, Jack Lu, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spatial transcriptomics, which deals with tissue architecture in genetic investigation, is an innovative technique for examining cell heterogeneity and tissue organization. This review emphasizes major approaches, include spatially resolved transcriptome methods, immunohistochemistry as well as in situ hybridization, all of which permit the mapping of RNA molecules in their native tissue environment. These methods have proven essential in achieving our understanding of biological events such as tumor evolution, progression of cancer, and cancer tumor stem cell detection. Spatial transcriptomics, the study of patterns of gene expression in space, reveals the intricate nature of the tumor microenvironment (TME) and its effect on cancer biology. Although it delivers insight on the cellular connections that underlie disease, the significance of spatial transcriptomics in multiple organs has expanded.

Although its immense potential, there are still difficulties to be conquered, particularly within the areas of analysis of data, spatial resolution, and integration with other omics data. To be able to fully comprehend the complexities of tissues biology and ailments, this review additionally tackles future potential avenues, including the necessity for greater multiplexing, enhanced resolution, and the combination of functional genomics. With this synthesis, we intend provide an extensive summary of the state of spatial transcriptomics currently and demonstrate that it possesses the potential to improve precision medicine, cancer research, and our understanding of broader biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579b9e172c2769b4ad99d0da42fa24b04f1ee980" target='_blank'>
              Spatial Transcriptomics in Cancer Research: Methods and Applications
              </a>
            </td>
          <td>
            P. K. Kumar, K. M. Rao, Syed Mohiuddin Quadri, Bagadam Aparna, V. U. M. rao
          </td>
          <td>2025-12-18</td>
          <td>International Journal of Current Science Research and Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Why does gastric cancer frequently recur despite aggressive treatment? The answer may lie in a small but stubborn population of cells hidden deep within the tumor, the cancer stem cells (CSCs). These cells possess self-renewal capacity, resist conventional therapies and drive relapse and metastasis, making them critical determinants of patient outcomes. Their persistence highlights the urgent need to understand the molecular mechanisms sustaining their survival. One emerging explanation comes from the field of epitranscriptomics, which investigates chemical modifications of RNA that regulate gene expression post-transcriptionally. Among these modifications, N6-methyladenosine (m6A) has attracted particular attention. It serves as a dynamic regulator of RNA metabolism, controlling stability, translation and splicing, and has been shown to promote stemness and aggressiveness in gastric cancer stem cells (GCSCs). Dissecting the m6A-dependent regulatory networks in CSCs paves the way for novel diagnostic biomarkers and therapeutic strategies designed to eliminate the root of gastric cancer recurrence and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57fa6c528cff78e5be882e6c5182cbc5cde84094" target='_blank'>
              The Epitranscriptomic Landscape of Gastric Cancer Stem Cells: The Emerging Role of m6A RNA Modifications
              </a>
            </td>
          <td>
            Diana Pádua, P. Mesquita, Raquel Almeida
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b6297b1b489e23bcfae1eb1b6a372257f881b60" target='_blank'>
              Retargeted oncolytic viruses engineered to remodel the tumor microenvironment for glioblastoma immunotherapy.
              </a>
            </td>
          <td>
            Federico Giovannoni, Craig A. Strathdee, Camilo Faust Akl, Brian M. Andersen, Zhaorong Li, Hong-Gyun Lee, M. Torti, Joseph M. Rone, Pere Duart-Abadia, Martina Molgora, Linxing Kong, Michael Floyd, Jian Teng, Yulia Gyulakian, Peter Grezsik, Terry Farkaly, A. Denslow, Sonia Feau, Irene Rodriguez-Sanchez, Judy Jacques, Marco Colonna, Edward M Kennedy, Tooba A. Cheema, Lorena Lerner, Christophe Quéva, F. Quintana
          </td>
          <td>2025-12-04</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Gliomas carry a high disease burden, particularly the aggressive IDH-wildtype subtype. However, some IDH-mutant gliomas can transition from a less aggressive G-CIMP-High (GCH) state to a more malignant G-CIMP-Low (GCL) phenotype, mimicking IDH-wildtype tumors. Understanding the cellular and molecular mechanisms behind this progression is vital for developing effective therapies.



 We propose that single-nuclei resolution analysis can reveal distinct neoplastic cell subtypes driving the GCH-to-GCL transition through activation of stemness- and proliferation-related gene programs.



 We analyzed 22 glioma specimens from 15 patients (including matched primary and recurrent tumors) using single-nuclei RNA- and ATAC-sequencing. After quality control and neoplastic cell identification via somatic CNVs, we applied non-negative matrix factorization (NMF) to identify cell subclusters based on gene expression patterns. We further characterized these clusters using gene set enrichment, protein-protein interaction networks, developmental trajectory analysis, and integration with external RNA-seq datasets. Chromatin accessibility was assessed via snATAC-seq to identify regulatory elements and transcription factor activity.



 We identified six major neoplastic cell subtypes: Astrocytic-Support, Pro-neural Housekeepers, Proteo-synthetic Cells, Mitotic Proliferators, Neural Guidance & Growth Cells, and Mesenchymal-mutant Cells. Each showed distinct gene expression signatures, pathway enrichments, and survival associations. Two subtypes—Mitotic Proliferators and Mesenchymal-mutant Cells—were linked to high-grade tumors and poor patient outcomes. Chromatin accessibility patterns corroborated transcriptional findings, highlighting differential TF regulation in aggressive subtypes.



 This study reveals two aggressive GCL-related neoplastic cell subtypes with distinct stem-like and proliferative features. These findings provide insight into glioma progression mechanisms and uncover potential molecular targets for therapeutic intervention in IDH-mutant gliomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd05421cc91e3cb0411e5625733c706631851341" target='_blank'>
              EPCO-21. SINGLE-NUCLEI ANALYSIS IDENTIFIES NOVEL NEOPLASTIC IDH MUTANT ASTROCYTOMA PROLIFERATIVE AND STEMNESS PHENOTYPES DRIVING POOR PATIENT OUTCOMES
              </a>
            </td>
          <td>
            H. Noushmehr, G. Herrgott, L. Garofano, Ian Lee, J. Snyder, T. Walbert, A. Castro, A. Iavarone
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/027f986afc66cae82f4bb168ec0c6120d17b62d0" target='_blank'>
              Transgelin defines pro-tumorigenic cancer-associated fibroblasts in pancreatic cancer.
              </a>
            </td>
          <td>
            Xingxing Wang, Keiko Shinjo, Kohei Kumegawa, R. Maruyama, Shinji Mii, Yukihiro Shiraki, Tastunori Nishimura, Y. Murofushi, Miho M. Suzuki, T. Kabasawa, Mitsuru Futakuchi, A. Kanai, Yutaka Suzuki, Atsushi Enomoto, Yutaka Kondo
          </td>
          <td>2025-12-16</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive brain tumor, characterized by its ability to manipulate the tumor microenvironment (TME) for sustained tumor growth and recurrence. Astrocytes, the most abundant glial cells, adopt reactive phenotypes upon interacting with glioma cells. While tumor-associated astrocytes drive GBM proliferation, immunosuppression, and treatment resistance, the mechanisms by which cancer cells reprogram astrocytes remain poorly defined. Our previous studies showed that mitochondria transfer from astrocytes to GBM cells enhances tumorigenicity, but the role of this communication pathway in astrocyte biology remains unclear. Utilizing co-cultures of patient-derived GBM cells (PDCs) expressing mito-GFP and hTERT-immortalized human astrocytes, we identified mitochondria transfer to astrocytes in a GBM cell type-dependent manner, with rates as high as ~40% in some models. Targeted sequencing of unique mtDNA variants additionally revealed transfer of PDC mtDNA to co-cultured astrocytes. Initial in-vivo experiments demonstrated that co-implantation of astrocytes with PDCs accelerated tumorigenesis in immunocompromised mice. To assess the relevance of mitochondria transfer in the tumor microenvironment, we subsequently implanted mitochondria-labelled PDCs and demonstrated that tumor-associated astrocytes contained GBM-derived mitochondria. Functionally, acquisition of GBM mitochondria promoted astrocyte proliferation, increased cellular reactive oxygen species (ROS), and enhanced secretion of the chemokine CCL2, which is implicated in the recruitment of immunosuppressive myeloid cells in GBM. Astrocyte proliferation and increased CCL2 were orthogonally validated in vitro through addition of cell-free PDC mitochondria. Mechanistic investigations further revealed that tumor necrosis factor (TNF) enhanced GBM-to-astrocyte mitochondria transfer. These findings elucidate a poorly understood, fundamental process by which mitochondria transfer promotes astrocyte reactivity and supports an immunosuppressive tumor microenvironment. Future studies will explore the molecular drivers of GBM mitochondria transfer and astrocyte reprogramming to identify potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38e6ce2ad2a7d383a88613cd241a04afb6cf153" target='_blank'>
              TMIC-36. Intercellular mitochondria transfer drives immunosuppressive astrocyte reprogramming in glioblastoma
              </a>
            </td>
          <td>
            Jonathan Mitchell, B. León, O. T. Pumar, Asmita Pathak, Ogechukwu Mbegbu, Yi-An Chen, Floris P Barthel, Irem Karaman, Carolina De La Pena, A. Lasorella, A. Iavarone, Defne Bayik, Dionysios C. Watson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="


 Antiretroviral therapy (ART) has transformed HIV into a manageable chronic condition. Stopping ART causes rapid viral rebound, revealing complex and latent tissue reservoirs that hinder efforts for a cure. Understanding the biological, cellular, and tissue-specific factors involved is key to overcoming this challenge. In-situ spatial biology and multiplex histology technology can provide novel insights into the spatial immune and anatomical dysregulation driven by HIV. We performed in-situ multiplex-spatial proteomic analysis using the Nanostring CosMx Human 64-plex Immuno-Oncology Protein Panel on fresh-frozen spleen and gut tissues from two people with HIV (PWH) (detectable and undetectable) from the Last Gift Rapid Research Autopsy Cohort. Data quality control and normalization was processed with the AtoMx pipeline. Despite viremia status, we found similar total protein expression in the spleen and colon tissue. Cell type annotation using the CELESTA pipeline revealed that the viremic donor had more vascular smooth muscle cells in the colon, while the aviremic donor had more epithelial and fibroblast cells. Immune profiling showed higher lectin specific protein expressions in immune cells of the aviremic donor, particularly in dendritic and epithelial cells in the colon compared to the viremic donor. These preliminary findings highlight tissue-specific immune and cellular changes in PWH, with future studies planned to compare these changes to those in people without HIV.



 Supported by NIH/NIAID AI-164559



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5645976e541610bcd1dcd387cc992098aa48a1ae" target='_blank'>
              Spatial biology discovery approaches to dissect in situ modification of site-specific tissue HIV reservoirs 2542
              </a>
            </td>
          <td>
            Stephen T. Yeung, P. Moar, Vanessa Gomez-Moreno, Davey Smith, A. Chaillon, S. Gianella, L. Ndhlovu
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Immune system control is a principal hurdle in cancer evolution. The temporal dynamics of immune evasion remain incompletely characterized, and how immune-mediated selection interrelates with epigenome alteration is unclear. Here we infer the genome- and epigenome-driven evolutionary dynamics of tumor-immune coevolution within primary colorectal cancers (CRCs). We utilize a multiregion multiomic dataset of matched genome, transcriptome and chromatin accessibility profiling from 495 single glands (from 29 CRCs) supplemented with high-resolution spatially resolved neoantigen sequencing data and multiplexed imaging of the tumor microenvironment from 82 microbiopsies within 11 CRCs. Somatic chromatin accessibility alterations contribute to accessibility loss of antigen-presenting genes and silencing of neoantigens. Immune escape and exclusion occur at the outset of CRC formation, and later intratumoral differences in immuno-editing are negligible or exclusive to sites of invasion. Collectively, immune evasion in CRC follows a ‘Big Bang’ evolutionary pattern, whereby it is acquired close to transformation and defines subsequent cancer-immune evolution. This study nominates immune escape as an early event in colorectal cancer and shows how this can be driven through both genetic and epigenetic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b298787dd108fa4c080b3a88642ab2a59626f9" target='_blank'>
              Epigenetically driven and early immune evasion in colorectal cancer evolution
              </a>
            </td>
          <td>
            E. Lakatos, V. Gunasri, L. Zapata, J. Househam, Timon Heide, Nick Trahearn, O. Swinyard, Luis H Cisneros, Claire Lynn, M. Mossner, C. Kimberley, Inma Spiteri, George D. Cresswell, Gerard Llibre-Palomar, M. Mitchison, Carlo C. Maley, Marnix Jansen, M. Rodriguez-Justo, John Bridgewater, Ann-Marie Baker, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2025-11-05</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="The biological heterogeneity between left- and right-sided colorectal cancer (CRC) poses a significant clinical challenge and the underlying regulatory mechanisms remain elusive. As an emerging hallmark of cancer, the contribution of circadian rhythm disruption (CRD) to this side-specific heterogeneity is unclear. By integrating single-cell and spatial transcriptomic analyses, this study shows that tumor cells in right-sided CRC exhibit significantly higher CRD scores and identifies the emergence of a NONO-positive tumor-cell subpopulation (NONO⁺ TC) as a key molecular feature of this phenotype. Spatial analysis further confirms that this NONO⁺ TC forms a tightly co-localized microenvironment with fibroblasts in situ. Notably, cell–cell communication analyses indicate that NONO expression does not augment the signal-sending capacity of tumor cells but instead reprograms them into highly efficient signal receivers. These augmented incoming signals predominantly originate from the more pro-tumorigenic myofibroblastic cancer-associated fibroblast (myCAF) subtype, indicating that NONO⁺ TC are particularly sensitive to malignant stromal inputs. In conclusion, our study delineates the dual role of NONO as both a circadian regulator and a pro-tumorigenic signaling hub. By enhancing tumor-cell reception of CAF-derived signals, NONO links CRD to the formation of NONO⁺ TC niches and a malignant microenvironment in right-sided CRC, providing new mechanistic insight into the spatial heterogeneity of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241b9e1e30a34229f91641257f9f4e9436ac45df" target='_blank'>
              NONO links circadian rhythm disruption and enhanced tumor-fibroblast crosstalk in right-sided colorectal cancer
              </a>
            </td>
          <td>
            Zhi-Hao Shang, Qin-Chang Zhang, Song-yang Xi, Kai Chen, Shao-Bo Guo, Zhou Zhou, Xin-Zhuo Zhan, Yun-Xia Wu, Xin-Yi Li, Hai-Bo Cheng, Xue-Jun Song, Gui-Hua Tian
          </td>
          <td>2025-10-31</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progression involves dynamic remodeling of the tumor microenvironment (TME). However, the stage-specific dynamics of immune and stromal cell remodeling throughout LUAD progression remain incompletely understood. Here, the study systematically profiles the cellular composition and transcriptional states across multiple LUAD stages, integrating early-stage patient specimens with publicly available datasets encompassing advanced-stage disease. The analysis reveals a marked stage-dependent shift from a proliferative and immune-activated microenvironment in early LUAD to a hypoxia-enriched and immunosuppressive landscape in advanced disease. A distinct hypoxia-adapted epithelial tumor cell subpopulation (C5), exhibiting transcriptional features of metastasis, invasion, and hypoxia, and poor prognosis, is identified. Advanced LUAD featured immunosuppressive LGMN⁺ macrophages and STAT1-driven exhausted CD8⁺ T cells. FKBP11⁺ plasma B cells exhibited exhaustion-linked metabolic changes. POSTN⁺ CAFs emerged as central mediators of extracellular matrix (ECM) remodeling and immune exclusion. Collectively, the findings reveal a model of hypoxia-driven functional convergence, in which distinct TME components co-evolve toward phenotypes that collectively promote immune evasion, matrix remodeling, and tumor progression. These findings may provide insights into stage-specific cellular dynamics and highlight promising therapeutic targets for precision immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8876e01109cf8f024442bf59e35e9ed246f97f" target='_blank'>
              Characterizing Stage-Specific Cellular Dynamics and Microenvironmental Remodeling in Lung Adenocarcinoma by Single-Cell RNA Sequencing.
              </a>
            </td>
          <td>
            Bomiao Qing, Xiaolan Li, Xiang He, Junyi Wang, Yi Yang, Manling Jiang, Bingbing Yan, Lei Zhang, Anying Xiong, Q. Ran, Guoping Li
          </td>
          <td>2025-11-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26bc66a34e745a67c0425c16be84d04301c9194" target='_blank'>
              Multimodal analysis reveals potential association of CDH13 with endothelial cells and its overexpression in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Ke-Jun Wu, Li-Hua Yang, Dong-Ming Li, Rong-Quan He, Di-Yuan Qin, Shi-De Li, Jian-Di Li, Y. Dang, Ming-Jie Li, Qi Li, Jian-jun Li, Lu Zhang, Han He, Ji-Feng He, Gang Chen, Xiao-Bo Jiang
          </td>
          <td>2025-12-12</td>
          <td>European journal of medical research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Understanding the mechanisms underlying key agricultural traits remains a central challenge in crop research, but recent advances in technologies are providing powerful tools to address this issue. Among these, single-cell and spatial transcriptomics have revealed tissue heterogeneity and spatial organization, offering unique insights into cellular gene expression dynamics and the coordinated activity of multiple cell types. These approaches help uncover how specific cell types contribute to agricultural traits and refine candidate loci lists through integration with trait-associated loci. Additionally, single-cell and spatial transcriptomics have the potential to serve as cell-level readout platforms integrating cellular perturbations, enabling high-throughput discovery of causal relationships between genotype and gene expression at the cellular level in plants. Successful implementation will accelerate the identification of key genetic variants for crop improvement. Here we review lessons learned from application of single-cell screening in mammalian cells, highlight major technical and biological barriers to its use in plants, and outline potential strategies to overcome these challenges. Together, the widespread application and integration of single-cell and spatial transcriptomics with other technologies enable not only the descriptive cataloging of cell states but also the causal interrogation of sequence functions and regulatory networks at cell-type resolution, ultimately advancing gene function studies and accelerating crop improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff7551fbd396347f03bdd7b21419c8e91773900" target='_blank'>
              Decoding crops one cell at a time: from cell atlases to single-cell genetics.
              </a>
            </td>
          <td>
            Hao Zhang, Robert J. Schmitz
          </td>
          <td>2025-11-25</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) exhibits spatial and molecular heterogeneity. Understanding region-specific transcriptional programs within distinct tumor microenvironments is crucial for identifying niche-targeted therapeutic strategies. This study uses the Ivy Glioblastoma Atlas Project (Ivy GAP) to characterize transcriptional variation across histologically annotated tumor regions. We analyzed annotated bulk RNA-seq data from Ivy GAP, covering five tumor regions: cellular tumor (CT, n=111), infiltrating tumor (IT, n=24), leading edge (LE, n=19), microvascular proliferation (MVP, n=50), and perinecrotic zone (PNZ, n=65). Differential gene expression (DGE) analysis was performed using CT as reference. Enrichment analysis was conducted using Gene Ontology and KEGG databases. Prognostic significance of differentially expressed genes (DEG) was assessed using CGGA-GBM dataset for survival analysis. Finally, we performed a pseudo-spatial analysis by plotting normalized Fragments Per Kilobase of transcript per Million mapped fragments (FPKM) expression levels of DEG genes across tumor regions to infer expression dynamics across distinct regions. Each region exhibited distinct transcriptional profiles. IT and LE were enriched for synaptic transmission, neuronal signaling, calcium signaling and excitable membrane potential genes; MVP for angiogenesis and extracellular matrix remodeling; and PNZ for hypoxia-induced metabolic adaptation. Several genes, including CACNG3 (20.5 vs 13.6 m, p=0.03), HCN1 (19.8 vs 13.9 m, p=0.02), SULT4A1(19.2 vs 13.9 m, p=0.03), and SYCE1 (21.7 vs 13.6 m, p=0.0006), were strongly associated with survival in CGGA-GBM dataset. Pseudo-spatial mapping allowed visualization of region-specific gene expression gradients. This integrative analysis reveals spatially distinct molecular landscapes in GBM and identifies prognostic gene markers linked to specific tumoral microenvironments. Our findings define region-specific transcriptional programs in GBM, offering mechanistic insights into tumor microenvironment heterogeneity and nominating candidate genes for future functional validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eee57357cd7906869206ea1e359f6b55d3036c56" target='_blank'>
              BIOM-86. MAPPING REGION-SPECIFIC TRANSCRIPTOMIC PROGRAMS IN GLIOBLASTOMA FROM HISTOLOGICALLY ANNOTATED RNA-SEQ DATA
              </a>
            </td>
          <td>
            Biprojit Nath, S. Bobholz, A. Lowman, S. Duenweg, Benjamin Chao, F. Kyereme, Jennifer Connelly, E. K. Mrachak, Max O. Krucoff, Peter S LaViolette
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Recently, it has become widely recognized that culturing cancer cells in vitro in small, 3D aggregates known as tumor spheroids provides a more physiologically relevant model of in vivo tumor behavior compared to 2D monolayer cultures. Dynamic optical coherence tomography (dOCT) is a non-invasive imaging modality that, by analyzing temporal fluctuations in the light scattered from biological tissue, does not require exogenous contrast agents to visualize and quantify cellular activity within 3D cell cultures. However, recent volumetric dOCT studies have encountered challenges due to low acquisition speeds. In this study, we present morphological and dynamic analyses of prostate tumor spheroid growth over a two-week longitudinal period, utilizing volumetric imaging with a line-field dOCT platform. Our method clearly differentiated between active cellular metabolism in live spheroids and the lack of activity in spheroids fixed with formaldehyde. Quantitative validation of the dynamic signal was conducted using the Alamar Blue proliferation assay, while qualitative validation was provided by live/dead fluorescence microscopy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51715bb05893c4f00c959588b59a96a3be80d8a0" target='_blank'>
              Longitudinal investigation of prostate tumor spheroid proliferation with dynamic line-field optical coherence tomography
              </a>
            </td>
          <td>
            Stephanie Swanson, Keyu Chen, Elahe Cheraghi, E. Osei, K. Bizheva
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad6284204cf8fc0e7c47d8fa667f142a64e34e5" target='_blank'>
              Machine learning identifies TIME subtypes linking EGFR mutations and immune states in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Zetian Gong, Mingjun Du, Ying Li, Bicheng Ye, Yuming Huang, Hui Gong, Wei Wang, Liang Chen, Zongli Ding, Pengpeng Zhang
          </td>
          <td>2025-11-26</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Breast cancer remains a leading cause of cancer-related death in women worldwide. Emerging evidence highlights the central roles of breast cancer stem cells (BCSCs) and circulating tumor cells (CTCs) in tumor initiation, progression, therapeutic resistance, and metastasis. BCSCs self-renew and drive intertumoral heterogeneity, while CTCs disseminate from primary tumors into the bloodstream, seeding distant sites. These populations share molecular features, including stemness and epithelial-mesenchymal transition markers, supporting the concept that a subset of CTCs acquires stem-like traits, enhancing metastatic potential and resistance to standard therapies. This review synthesizes current knowledge on BCSC molecular programs, key signaling pathways (e.g., Wnt, Notch, Hedgehog, Janus kinase/signal transducer and activator of transcription), and microenvironmental interactions that sustain stemness. It also examines mechanisms of CTC intravasation, state-dependent detection strategies, and their diagnostic and prognostic utility. We further highlight the adaptive plasticity of cancer stem cell-like CTCs, their contributions to drug resistance, and opportunities to target these phenotypes for personalized treatment. Clarifying the biological links between BCSCs and CTCs could enable earlier detection of hidden metastasis and inform combination therapies aimed at both stemness and dissemination. As multimodal detection improves and functional profiling matures, integrating BCSC/CTC analyses into routine care may refine risk stratification and guide individualized management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5bcd7f8dc81f11c35c61485cd16b591fdbe679b" target='_blank'>
              Breast cancer stem cells and circulating tumor cells: Dual drivers of progression and relapse
              </a>
            </td>
          <td>
            Zahra Azizi, B. Er Urgancı, I. Acikbas
          </td>
          <td>2025-12-26</td>
          <td>World Journal of Stem Cells</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Glioblastoma progression features extensive dissemination of cancer cells throughout the brain. Limited by neuroimaging resolution and lack of specific markers, malignant cell infiltration is clinically undetectable. This makes complete tumor removal impossible and relapse inevitable. During their exodus from the tumor core, glioblastoma cells are traversing distinct brain territories influencing their phenotypic cell state. To better understand the cellular and molecular dynamics of glioblastoma invasion, we profiled glioblastoma patient avatars and clinical samples using total, single-cell and spatial transcriptomics. In addition to gene expression data, active intracellular pathways of glioblastoma invasion were screened with high throughput activated kinase assays (Pamgene). We identified that human glioblastoma stem cells exhibiting proneural (PN) and neuron-progenitor like (NPC) transcriptomic traits in vitro formed highly infiltrative and connected xenografts in patient avatars. Consistent with this, PN/NPC glioblastoma stem cells cultured on primary hippocampal neurons exhibited higher connectivity and motility compared to mesenchymal-like cells. Leveraging single-cell and spatial transcriptome analytics of avatar and patient samples, we performed cell clustering, annotation, pseudotime and RNA velocity analyses to associate glioblastoma phenotypic cell states to cellular functions. We found that invasive glioblastoma cells re-activated genes and pathways related to neuron progenitor maturation when transitioning from the tumor core to the brain microenvironment. To verify whether invasive PN/NPC glioblastoma cell integration would alter brain functionality, we performed preclinical live cerebral blood volume recordings using noninvasive brain functional ultrasounds (fUS). As glioblastomas progressed, we detected a gradual alteration of brain functional connectivity. Modifications happened in brain areas featuring minimal, yet consistent glioblastoma cell colonization, identified by post-mortem pathology analyses. Altogether, our data shed light on the importance of glioblastoma cell adaptability and brain integration during the metastatic progression. They also provide a framework for detecting invisible clinical features such as tumor invasion, focusing on brain functional connectivity assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6de15ca728123a2849468be8f7f288fcf64e2b6b" target='_blank'>
              ANGI-14. GLIOBLASTOMA PROGRESSION FROM TRANSCRIPTOMIC REGULATION TO BRAIN FUNCTIONAL CONNECTIVITY
              </a>
            </td>
          <td>
            Madeleine Lackman, C. Bergström, Anaïs Virenque, Kato Herman, Aarti Venkatesan, Juli Udayani, Takashi Namba, Analiz Rodriguez, E. Castrén, P. Laakkonen, Sinem Karaman, V. Le Joncour
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Prostate cancer commonly presents as multifocal disease, with distinct tumours that are often genetically independent. Metastases, however, typically derive from a single dominant clone, highlighting the clinical challenge of identifying which lesion drives progression. Focal therapy, an emerging treatment strategy, aims to target only the most aggressive tumour focus or foci, sparing normal prostate structures and reducing morbidity. Yet, the lack of consistent genomic drivers in primary tumours complicates efforts to identify the focus most likely to progress. While prostate cancers are known to harbour widespread methylation alterations, the extent to which these epigenetic changes track with clonal origin remains poorly understood. To address this, we performed multiregional epigenomic sequencing of 189 prostate tissue samples, including 109 tumour foci and 80 non-tumour regions from 44 patients (2-8 tumour samples per case). Differential methylation analysis revealed widespread alterations distinguishing benign from tumour tissue, including a core ‘epigenetic trunk’ of >20,000 recurrently altered CpGs shared across cases. DNA variants called from non-methylation sites of the epigenomic sequencing closely matched variants from WGS in cases with WGS available. Phylogenetic trees constructed from DNA variants and phyloepigenetic trees from methylation sequencing revealed striking patterns. Some cases lacked shared DNA variants across all tumour foci, indicating a polyclonal tumour origin, yet these genetically unrelated foci shared extensive methylation alterations. This suggests widespread methylation convergence occurs in clonally distinct tumour foci. To dissect clonal relationships further, we applied two orthogonal lineage tracing approaches using the methylation sequencing data: fluctuating CpGs (sites that change stochastically over time) and highly entropic 8-mer ‘methylation barcodes’ within the protocadherin gene cluster. Fluctuating CpGs showed concordant states across tumours that shared DNA sequence variants, while the methylation barcodes likewise revealed high similarity among such tumours. Together, these methods support the independent evolutionary origins of polyclonal prostate tumour foci. Pathway analysis of recurrently altered methylation loci highlighted consistent enrichment in epithelial–mesenchymal transition, MYC signalling, DNA damage response, and hormonal signalling across cases, suggesting functional convergence at the pathway level despite divergent genetic origins, consistent with selective pressures acting on shared programmes. Together, these findings demonstrate that DNA methylation alterations represent fundamental and recurrent events in prostate cancer, more consistent than DNA sequence variants across multifocal disease. Lineage-tracing analyses further highlight that methylation processes record tumour evolutionary history, encoding both clonal relationships and convergent biology. These observations may inform methylation-based biomarkers and the rational selection of lesions for focal therapy.



 Tamsin J. Robb, Melissa Cheung, Rajbir Batra, Henson Lee Yu, John C. Thomas, Anne Warren, Andrew Lynch, Daniel Brewer, David Wedge, CRUK-ICGC Prostate Cancer Group, Charles Massie, Harveer Dev. Widespread methylation convergence in clonally distinct foci of multifocal prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90717932598b7c081fea25f61e12e100d9757059" target='_blank'>
              Abstract B003: Widespread methylation convergence in clonally distinct foci of multifocal prostate cancer
              </a>
            </td>
          <td>
            Tamsin J. Robb, Melissa Cheung, Rajbir Batra, Henson Lee Yu, John C. Thomas, Anne Y Warren, Andrew Lynch, Daniel Brewer, D. Wedge, Charles Massie, H. Dev
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Objectives FAM49B has been shown to promote proliferation and metastasis of colorectal cancer (CRC) by stabilizing MYC through phosphorylation of NEK9; however, its role in shaping the immune suppressive tumor microenvironment (TME), particularly in macrophage polarization, remains unclear. Methods We applied multi-omics approaches to study CRC by integrating 33 scRNA-seq samples from 16 CRC patients, 2 paired spatial transcriptomics (ST) samples, and bulk RNA data to characterize malignant epithelial cells (High_FAM49B_EP) and tumor-associated macrophages (TAMs). Functional validation of FAM49B was conducted via knockdown experiments and proteomics analysis. Results A High_FAM49B_EP subpopulation was identified in primary tumors (PT) and liver metastases (LM), exhibiting elevated MYC signaling and association with poor prognosis. TAMs showed spatial heterogeneity: M1-like CXCL3+ TAMs predominated in PT, whereas M2-like SPP1+ TAMs were enriched in LM. CellChat analysis revealed that High_FAM49B_EP activated macrophage polarization through the MDK–NCL signaling axis. Pseudotime trajectory analysis confirmed differentiation from CXCL3+ to SPP1+ TAMs driven by upregulation of NCL. Spatial mapping showed co-localization of MDK+ epithelial cells with NCL+ TAMs in the immunosuppressive microenvironment. FAM49B knockdown significantly inhibited MDK expression and disrupted ECM–receptor interactions. Conclusions FAM49B promotes immunosuppressive TME formation by mediating TAM polarization via the MDK–NCL axis, suggesting the FAM49B–MDK–NCL pathway as a potential therapeutic target for CRC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8878570ce8b97d296e4580e17e3836990c936b1" target='_blank'>
              Single-cell and spatial transcriptomics integration reveals FAM49B promotes tumor-associated macrophages polarization in colorectal cancer via the MK pathway
              </a>
            </td>
          <td>
            Tianyu Liu, Quchen Ding, Jin Gou, Chen Lu, Xingming Lu, Jiatong Chen, Y. E, Lianhong Li, Chongguo Zhang, Xiaojuan Zhu, Chunzhao Yu, Xiagang Luo
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Diffuse midline gliomas (DMGs) have uniformly poor survival outcomes despite initial responses to radiation therapy (RT). Functional genomic and high throughput drug screens have identified promising therapeutic targets for DMGs, but these efforts have largely been performed in cell cultures, thereby missing targets that are selectively vulnerable within in vivo tumor environments. Here, we develop a genome-wide CRISPR interference (CRISPRi) platform to screen essential and radiation sensitizing genetic targets in DMG xenografts. We demonstrated the feasibility of genome-wide in vivo screens by introducing a lentiviral library of 22,580 inert barcodes prior to orthotopic transplantation of SJ-DIPG7 cells. Individual tumors yielded only 55-77% of this library after 3-4 weeks of intracranial growth. However, in silico pooling of two or more tumors enabled >88% genome-wide coverage. We then performed in vivo CRISPRi screens using a dual-sgRNA genome-wide library comprising 21,554 gene-targeting vectors and 1,026 non-targeting controls. SJ-DIPG7 stably expressing dCas9-Zim3 were infected with CRISPRi libraries and intracranially transplanted to immunocompromised mice. Animals were treated with control or RT (2 Gy x 5 fractions), a dose that extended survival from SJ-DIPG7 xenografts but not sufficient for cure. After in silico pooling of 2-3 mice, 93% of barcoded sgRNAs were recovered and we identified 340 genes that diminished in vivo growth upon knockdown (mean log2 fold-change -5.58;FDR 2%). These genes included 145 known essential genes from DepMap, supporting the validity of our method, and were enriched for proteasome components. We also identified 108 radiation sensitizing targets (mean log fold-change -6.02;FDR 7%) including DNA repair (TOPBP1, PPP4C), metabolism (CNDP1, LDHA, OGDH), and neuronal function (LSAMP, EFNA1). We validated the essential phenotype of the proteasome component PSMA3. Our work establishes the feasibility of genome-wide in vivo functional genomics for systematically identifying therapeutic vulnerabilities of DMGs, enabling the discovery of combination treatments for patients living with DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c58f1e437f00caa915d46128ea9259f60b4c0f9" target='_blank'>
              DDDR-47. Development of in vivo CRISPR interference screens in diffuse midline glioma xenografts to identify genetic vulnerabilities controlling responses to radiation therapy
              </a>
            </td>
          <td>
            I-Ching Wang, C. Zou, Akane Hubbard, Sasha Sengelmann, T. Ozawa, D. Raleigh, Christopher L. Tinkle, A. Shelat, S. J. Liu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Prostate cancer (PC) remains among the leading causes of cancer-related deaths with a better understanding of the mechanisms underlying its pathogenesis urgently needed. The synthesis of complement component C3 by tumor-infiltrating immune cells in the tumor microenvironment (TME) facilitates tumor progression via immunosuppression and angiogenesis support, denoting TME C3 as a therapeutic target. However, the role of C3 expressed by tumor cells themselves during oncogenesis remain inadequately characterized. Using data from multiple clinical PC cohorts we found that, surprisingly, C3 transcript and protein levels were significantly reduced in malignantly transformed regions when compared to benign regions of human PC tissues. C3 silencing in human benign primary prostate epithelial cells (PECs) caused uncontrolled proliferation while provision of intracellular C3a to the aggressive PC cell line DU145 reduced tumor burden in a xenograft mouse model, confirming the unexpected tumor-suppressing role of C3. Mechanistically, steady-state, endogenous C3 expression, C3a generation, and -receptor engagement in PECs maintains normal cell turnover through restraint of a PI3Kd-dependent MYC program upstream of ribosomal biogenesis. Collectively, these findings ascribe a non-traditional, tumor-suppressor function to cell intrinsic C3 and advocate for dissecting the context-specific roles of complement in malignancies to better inform on complement-targeted cancer therapies.



 NIH: ZIA/hl006223, ZIA/DK075149, R35GM138283 Swiss National Science Foundation: PCEFP3_194618



 Immune Mechanisms of Human Disease (HUM)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020832dd81a765c03d9c1ee584f3bf5577bab7a2" target='_blank'>
              Cell-autonomous complement C3 activity in prostate epithelial cells suppresses malignant transformation through tonic restrain of the MYC-tumorigenesis program 3135
              </a>
            </td>
          <td>
            Natalia Kunz, Jack A. Bibby, Nicolas S. Merle, Sumeyra Kartal, Hidekazu Yamamoto, Luopin Wang, Md Taj, Jianping Ma, Martin Kolev, Jonas Lotscher, Tamara Kellerhals, Maria L Balmer, Ashalia Chandra, Christoph Hess, Behdad Afzali, Prokar Dasgupta, Erin E. West, A. Sowalsky, Claudia Kemper
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) shape the tumor microenvironment of primary breast tumors to promote tumor progression and therapy resistance. While the lung is a top metastatic site in breast cancer, the origins of lung metastasis-associated fibroblasts and their influence on disseminating tumor cell outgrowth and chemoresistance are poorly understood. Here, we demonstrate the applicability of 2-dimensional and 3-dimensional cocultures of primary human lung fibroblasts (LF) and breast cancer cells (BCC) as models of tumor-stromal interactions in lung metastatic breast cancer. Using these models, we find that BCC lines representing clinically relevant molecular subtypes differentially induce CAF-like phenotypes in primary LFs corresponding with their propensity for lung metastasis. Furthermore, we identify a mechanism by which juxtacrine signaling from LFs to triple negative breast cancer (TNBC) cells promotes expansion of prognostic dormant-like cell subpopulations and instigates autophagy-dependent therapy resistance via integrin and Janus kinase1/2 signaling. A high content kinase inhibitor compound library screen using this model identifies vacuolar protein sorting 34 as a therapeutic vulnerability unique to BCC-LF interaction whose inhibition can resensitize TNBC cells to chemotherapy and relieve LF-mediated extrinsic therapy resistance. Therefore, we propose coculture of primary human LFs and BCC as a reductionist model of interactions between tumor cells and lung-resident stroma and a tool for therapeutic and mechanistic discovery in lung metastatic breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f189b128eada5790139b7643e3b3334624b2c8d" target='_blank'>
              Breast cancer cell coculture induces normal lung fibroblast transition to CAFs, promoting tumor cell dormancy and therapy resistance.
              </a>
            </td>
          <td>
            Marika L Klosowski, Kathryn E Cronise, Eric P Palmer, Kelly McAuliffe, Claire Stratton, Kelsie Sparks, Kendall T Malmstrom, Gwyneth Knott-Byars, Qiong Zhou, Hector Esquer, Daniel V. LaBarbera, Daniel P Regan
          </td>
          <td>2025-11-21</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 The relevance of the spatial context of the increasing number of potential biomarkers within the tumor microenvironment (TME) has resulted in a need for reliable and accurate multiplex immunohistochemical (mIHC) assays. The SignalStar mIHC assay from Cell Signaling Technology is a novel approach that was utilized to amplify and visualize up to 8 targets within the same formalin fixed, paraffin-embedded (FFPE) tissue section without the need for cycling of antibodies or fluorophore deposition. Here, we compare each target in an 8-plex panel consisting of T-cell exhaustion markers to the canonical chromogenic IHC assay. We also demonstrate the value of amplification when detecting lower abundance functional markers by comparing the percent positivity of SignalStar amplified targets to that achieved with direct and indirect immunofluorescence. Furthermore, we highlight the flexibility of this assay via the usage of a SignalStar secondary antibody, which allows for the incorporation of any IHC-validated rabbit antibody into a panel. Lastly, we also compare the SignalStar assay to other mIHC assays with respect to signal intensity and percent cell positivity. Ultimately, our data demonstrates that the SignalStar mIHC assay is a powerful tool that accurately characterizes the complex TME, allowing researchers to better understand immunotherapeutic approaches and other treatment strategies.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5388ac320b15789afa2366c0bfc8f1b720971bda" target='_blank'>
              Spatial analysis of the tumor microenvironment can be achieved via the novel SignalStarTM multiplex immunohistochemistry assay 3651
              </a>
            </td>
          <td>
            J. Ziello, Derek Papalegis, Lily Vu, Sasha Tkachev, Gabriella Spang, Sarah R Klein
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are increasingly recognized as critical contributors to the limited efficacy of immunotherapy, particularly through the establishment of immune-excluded tumor microenvironments. However, the functional heterogeneity and spatial ecological roles of CAFs in melanoma remain poorly characterized. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq data to systematically dissect CAF molecular programs in melanoma using weighted gene co-expression network analysis (WGCNA). Spatial distribution, CellChat, and ligand–receptor interaction analyses were applied to construct CAF-centered immune exclusion networks. A functional scoring model (riskScore) based on the CAF-M1 module was developed to evaluate clinical relevance. CAF was spatially co-localized with endothelial cells, and engaged in strong crosstalk with multiple immune cell types via pathways such as COLLAGEN–integrin and THBS1–CD47. The derived riskScore model demonstrated robust prognostic value across TCGA-SKCM cohort. Through spatial aggregation and multifaceted immune signaling, CAF constructs a multilayered immune-exclusion network in melanoma, linking stromal remodeling to immune evasion. These findings offer novel insights into CAF-driven immune resistance and may inform future stratified immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca83751e1c8ec5203b2f4b9c36b4cadfbfd1df19" target='_blank'>
              Exploration of the roles of CAFs in melanoma based on single-cell transcriptomics and spatial transcriptomics
              </a>
            </td>
          <td>
            Sheng Hong, Yuhan Zhao, Anni Hu, Yizhong Zhang, Yuming Li, Yan Rong, Zukai Li, Wenzhuo Su, Lude Zhu
          </td>
          <td>2025-11-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 As cancer therapies improve and patients live longer, the incidence of brain metastases (BrMets) continues to rise. Among women with metastatic breast cancer, 10-15% develop BrMets, with rates as high as 30% in HER2+ and 50% in triple-negative. BrMets, represent highly evolved, therapy-resistant tumors with a median survival of 10 months and accompanied by severe neurologic decline. BrMets often emerge years after initial diagnosis, suggesting that metastatic cells must acquire brain-specific traits to survive. This process may involve transcriptional convergence—where tumor cells adopt brain-like programs, and brain-resident cells exhibit tumor-associated changes. Despite its clinical relevance, the molecular basis of this convergence remains poorly defined. Decoding, and targeting, these shared gene programs could reveal new therapeutic vulnerabilities.



 We performed spatial transcriptomic profiling on 235 patient tissue cores (BrMets, adjacent normal brain, primary breast tumors, and non-cancer brain). Using NanoString GeoMx, DSP, we measured the expression of 18,677 RNAs across 450 spatially-defined regions enriched for tumor, immune, and brain-resident cells. To interrogate transcriptional convergence, we developed the Equivalent Expression Index, a novel statistical tool to detect genes with biologically meaningful expression similarity across distinct cell populations.



 Our integrated analysis revealed two prognostic gene signatures. The Metastasis-Induced Brain Shared (MIBS-9) signature, comprising brain-like genes upregulated in tumor cells, was associated with longer patient survival. Conversely, the Adjacent brain-Resident Cancer Shared (ARCS-81) signature, defined by tumor-like gene expression in brain-resident cells, correlated with shorter survival. These signatures were externally validated and enriched for pathways related to neuronal remodeling, immune evasion, cellular communication, and cancer/metastasis.



 By coupling spatial transcriptomics with a novel equivalence-based algorithm, we uncover distinct, clinically relevant gene programs that mediate bidirectional adaptations between tumor and brain-resident cells. These findings illuminate key mechanisms of metastatic outgrowth and offer new targets for disrupting the breast cancer BrMet microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c6b99afe4369f11ea5994698b0e8b275a5df02" target='_blank'>
              TMIC-89. Spatial transcriptomics reveals tumor–brain expression convergence and identifies prognostic gene signatures in breast cancer brain metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, Christine Yeh, Pablo Nunez Perez, Giuseppe Barisano, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia Plevritis, M. Gephart
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains among the most aggressive cancers, due to its limited therapeutic options and poor prognosis. Even with the adoption of total neoadjuvant therapy (TNT) and potentially curative surgical resection, PDAC relapse rates remain unacceptably high. Current systems for evaluating tumor response are valuable but limited, as they fail to capture the ecological complexity and broader architecture of the tumor microenvironment (TME). This results in uncertainty regarding the prognosis and recurrence risk of patients. To address this and to capture the complexity of the PDAC TME, we developed the Tumor Landscape Analysis (TLA). Using digitized H&E slides from 203 PDAC patients who received TNT and surgical resection at our institution, we applied a machine-learning pipeline to segment tumor and stromal compartments. We then quantified the TME architecture using ecological frameworks to assess the disease in multiple scales: individual patches, whole-slide landscapes, and aggregated patient-level summaries. Our analysis revealed that higher-order spatial features are independently prognostic of disease-free survival (DFS). Two statistical models showed different but complementary spatial characteristics that predict disease risk. In the first model, patients with more irregular cancer patch shapes and lower variability in stromal shape had significantly better DFS, with high-risk patients relapsing in a median of 7.2 months compared to 11.6 months in the low-risk group (HR=1.82, p=0.0003). In the second model, smaller stromal patch areas combined with greater tumor-stroma edge density identified patients at elevated risk, with high-risk individuals showing a median DFS of 8.0 months versus 15.7 months in low-risk patients (HR=1.73, p=0.0007). Importantly, these associations persisted even after adjustment for the T and N stage, showing that spatial features provide unique information beyond traditional pathology measures. These models help visualize PDAC as a comprehensive ecosystem that carries clinical significance. By quantifying tumor conformation and configuration, our models reveal prognostic information not available through CAP grading, percent tumor burden, or routine cell counts. This study suggests that tumor-stroma interactions and their spatial organizations are potentially biologically important for recurrence. We present a new, ecologically inspired approach to digital pathology for PDAC that not only improves our ability to predict recurrence and survival in PDAC but also sheds light on cancer as a complex, evolving ecosystem.



 Merih D. Toruner, Luis H. Cisneros, Miranda Lin, Andrea Maraone, Carlo Maley, Paul Dizona, Qian Shi, Martin E. Fernandez-Zapico, Ryan M. Carr. Ecological landscape analysis of the tumor microenvironment predicts recurrence risk in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d7729b6a5dc9c1d5cb651f1abed4a42008bec5" target='_blank'>
              Abstract B027: Ecological landscape analysis of the tumor microenvironment predicts recurrence risk in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            M. D. Toruner, L. Cisneros, Miranda Lin, Andrea V. Maraone, Carlo C. Maley, Paul Dizona, Qian Shi, Martín E Fernandez-Zapico, Ryan M. Carr
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fb5388d0bd0362161f91d2111550a5aa12010f" target='_blank'>
              Inference of clonal hematopoiesis using the collective behaviour of DNA methylation states
              </a>
            </td>
          <td>
            Samuel. J. C. Crofts, Caleb M. Grenko, Neil A. Robertson, Kristina Kirschner, Tamir Chandra
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 TERT promoter mutations are the most frequent single nucleotide variants (SNVs) in glioblastoma and many other cancers. These hotspot mutations provide novel sites for transcription factor binding, activating expression of TERT gene and leading to telomere elongation and cancer cell immortalization. Moreover, these SNVs disrupt the highly structured G-quadruplex region of TERT promoter, impacting chromatin organization more broadly. Since epigenetic cell state is highly dynamic and regulated by cues from the tumor microenvironment, studying these changes in freshly resected tumor tissue is highly relevant. To enable such studies in glioblastoma, we developed STAR-FACS, a method allowing for cell sorting based on an SNV, that can be applied to freshly dissociated human tumor tissue. The single cell suspension is fixed, and a mutation-specific PCR is performed in the intact cells. Incubation of the cells with a labeled probe that hybridizes to the PCR product renders the cells fluorescent and amenable for cell sorting. Using TERTp WT and MUT glioblastoma cell lines and neurospheres, we demonstrated that STAR-FACS protocol is compatible with downstream profiling of the sorted cells by bulk and single cell transcriptomics and epigenetic profiling with CUT&Tag. We also show that TERT promoter mutation-based cancer cell enrichment can be performed on freshly dissociated glioblastoma tissue. STAR-FACS allowed us to compare expression profiles of TERT promoter mutant glioblastoma cells accumulating 5-ALA with 5-ALA- cancer cells from the same tumor, which would be challenging with standard methods. Our mutation-based cell sorting method provides a novel, inexpensive tool for studies linking the effect of subclonal non-coding SNVs on transcriptomic and epigenetic heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297a12bdc543566d0e93d050acddeb7d7c6644de" target='_blank'>
              EPCO-50. TERT PROMOTER MUTATION-BASED CELL SORTING FOR SINGLE CELL TRANSCRIPTOMIC AND EPIGENETIC PROFILING OF GLIOBLASTOMA CELLS
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, O. Szentirmai, M. Janiszewska
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represent a complex ecosystem of intertwined interconnections between various cell types. Through diverse mechanisms of intercellular communication, GBM cells can modulate the behavior of both malignant and non-tumor cells, promoting cancer progression. One of the key drivers of GBM plasticity and adaptability is extrachromosomal DNA (ecDNA). ecDNA allows GBM cells to rapidly and reversibly modify their genome and by thousands of folds increase the expression of critical oncogenes. However, virtually nothing is known about the role of ecDNA in intercellular communication. Using qPCR and immunofluorescent staining combined with fluorescent in situ hybridization, we demonstrated that ecDNA can be transported between different populations of GBM cells via tumor microtubes in vitro. The efficiency of this transfer substantially differs for various ecDNAs and for GBM cells derived from different patients. Our results suggest that recipient GBM cells can maintain exogenous ecDNA for up to five weeks. Further experiments with iPSC-derived cerebral organoids implanted with ecDNA-containing GBM cells revealed ecDNA transfer between GBM and non-tumor cells. To confirm that this process occurs in patients, we performed single-cell RNA sequencing and whole-genome sequencing of malignant and adjacent non-transformed brain tissues. Integrated analysis of mutations in DNA and RNA, copy number variations, and gene expression patterns confirmed that a substantial fraction of non-cancer cells within the tumor microenvironment harbor ecDNA. We hypothesize that ecDNA transfer between GBM cells can accelerate cancer evolution by disseminating the most beneficial ecDNA molecules within the tumor, thereby bypassing the bottleneck of clonal competition. On the other hand, ecDNA transfer from GBM to non-tumor cells can theoretically have a profound impact on the properties of recipient normal cells, altering their phenotypes to promote further tumor progression or even turning them into seeds for tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2992457556b240d36672234fd746fe0938a62d29" target='_blank'>
              TMIC-26. Intercellular transfer of extrachromosomal DNA in glioblastoma
              </a>
            </td>
          <td>
            Marat Pavlyukov, Anastasia Kazakova, Lindsey A Dudley, Yuji Nakano, Soniya Bastola, Kunal S. Patel, Harley Kornblum
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Immune surveillance is believed to eradicate pre-cancerous cells to guard against cancer. However, its mechanisms remain unclear, as pre-cancerous cells are difficult to detect. To gain a glimpse into immune surveillance, our lab developed a genetically engineered mouse model recapitulating human triple negative breast cancer (TNBC), which generates rare, p53-Brca1 mutant cells with unequivocal GFP labeling. Guided by the visualization of premalignant cells, we observed tertiary lymphoid structure (TLS)-like immune aggregates near mutant ducts long before tumor formation. GeoMx-based spatial profiling showed not only activation signatures in T and B cells but also elevated interferon response genes in mutant ducts associated with immune aggregates when compared to those free of immune infiltration. FTY720 treatment at premalignancy, which blocks T and B cell egress from lymph nodes, led to increased mutant cells expansion and shortened tumor latency, demonstrating the restraint of mutant cell growth by the immune system. To clarify the roles of specific immune cells, we plan to deplete CD4 T, CD8 T, and B cells at premalignancy. We will also conduct longitudinal studies on T and B cells along the tumor evolution process from premalignancy to malignancy to understand how immune surveillance eventually falters. Taken together, our studies are poised to elucidate key mechanisms of immune surveillance in the premalignant stage of TNBC.



 Supported by NIH R01-CA256199; Basser Center for BRCA; the Pinn Scholarship (UVA); UVA Cancer Center Spatial Biology Funding; UVA Cancer Center Training Grant to X.Z.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ee7201e884081fced81f5a6fa79a355f932afc" target='_blank'>
              First glimpse of immune surveillance during premalignant progression of triple negative breast cancer 3378
              </a>
            </td>
          <td>
            Xian Zhou, Jianhao Zeng, Elaina Kiral Ball, Alexys Riddick, Pat Pramoonjago, Victor Engelhard, Hui Zong
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Introduction: The prevalence of epigenetic regulator mutations in clonal hematopoiesis and hematologic malignancies highlights the necessity of understanding how mutations in these regulators affect the fate potential of hematopoietic stem and progenitor cells (HSPCs) at a fundamental level. Utilizing single-cell multi-omics data, including paired transcriptome and chromatin accessibility (10X Multiome) and paired histone post-translational modification profiles and RNA polymerase II profiling (sciCUT&Tag + sciCUTAC) of human bone marrow mononuclear cells from healthy donors, we have identified regulatory dynamics of cell fate gain and loss unique to single lineages and quantify the loss of the stem cell program over a given fate transition.These tools will unravel the complex regulatory landscape of priming and lineage biases in healthy hematopoiesis and provide a pathway for investigating the specific mechanisms by which clinically observed epigenetic regulator mutations disrupt HSPC fate decisions in hematologic diseases.
 Methods: To construct a combined low-dimensional representation of chromatin accessibility scATAC-seq and sciCUTAC data were processed using shared genomic features using Seurat (Hao et al. 2021) and Signac (Stuart et al. 2021), then integrated using Seurat IntegrateData(). A shared pseudotime axis could then be computed using Palantir (Setty et al. 2019). To quantify epigenetic mark enrichment and chromatin accessibility at the gene level, gene scores were computed using ArchR (Granja et al. 2021). Using these scores, regulatory activity trends are fit for individual genes using Gaussian process regression (Otto et al. 2024).
 Results: Co-occurring H3K27me3 depletion, transcription, chromatin accessibility, and H3K27ac enrichment are restricted to the B cell lineage for PAX5 and EBF1, both critical master regulators of B lymphopoiesis. Comparable patterns of repression and de-repression at key transcription factor loci are detectable in the myeloid and erythroid lineages. The pseudotime ordering constructed from combined scATAC-seq and sciCUTAC data allows elucidation of the relative ordering of changes in histone mark enrichment, chromatin accessibility, and gene expression throughout hematopoiesis.
 Conclusions: Preliminary results from this work demonstrate the potential of co-analyzing more than two single-cell data modalities simultaneously, without the need for the development of novel techniques that can measure three or more modalities paired in single cells. The observed epigenetic activity at key master regulators, including PAX5 and EBF1 in B lymphopoiesis, defines a signature of lineage-specific gene activation during hematopoiesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f39fbc2f6b34120c4aaa60d8bbea50ff4cc6bd5" target='_blank'>
              Detecting lineage-specific regulatory activity and stemness retention in single-cell multiomics of human hematopoiesis
              </a>
            </td>
          <td>
            Cailin Jordan, Dominik J. Otto, Jacob Greene, S. Henikoff, Manu Setty
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Recent evidence demonstrates the existence of tissue resident memory B cells (BRM) in the lungs,where they provide long-lived protection against microbial infection,but it is unknown if analogous cells contribute to anti-cancer immunity.To dissect the dynamics and functional role of resident B cells in cancer progression,we used a cutaneous extracellular vesicle (EV) vaccination strategy to deliver DEL and OVA antigens locally.Adoptive transfer of antigen-specific B (Hy10,recognizing DEL) and T (OTII,recognizing OVA) cells allowed us to establish Hy10 BRM in specific localized regions of skin.Flow cytometric profiling revealed that these Hy10 BRM were enriched in expression of CD38,CD73,CXCR3, and CCR6 consistent with a phenotype associated with B cell memory and tissue residency.Challenging mice with MOC2 squamous cell carcinoma cells expressing OVA/DEL demonstrated that the presence of Hy10 BRM provided localized protection against tumor growth.Ongoing experiments aim to apply single cell RNA sequencing to transcriptionally profile human and murine tumor infiltrating BRM signatures.In addition,we are dissecting the mechanism through which BRM mediate anti-cancer immunity using both in vivo and in vitro models.Altogether,our results indicate that BRM can establish in skin and that such cells are primed for cancer immunosurveillance and killing.These findings expand our understanding of BRM protection role against cancer, and open new immunotherapies for targeting B cells.



 Melanoma Research Alliance, 1253470; Elsa U. Pardee Foundation; Knight Cancer Institute, OHSU



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d9cbb1331da0f1c0e6abf8fafefd43fcd0c044" target='_blank'>
              Tissue resident memory B cells mediate local anti-cancer immunity 3448
              </a>
            </td>
          <td>
            Abrar Samiea, Zihan Guo, F. Pucci, Joshua M. Moreau
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95f4955996275b1364c480a5de0b28f26332655d" target='_blank'>
              Structural basis for constitutive activation of oncogenic gp130 mutants in human inflammatory hepatocellular adenomas
              </a>
            </td>
          <td>
            Yi Zhou, Jing Cao, T. Ramlall, Luke L. McGoldrick, Jennifer Jones, Annan S. I. Cook, Raymond Leidich, Mark W. Sleeman, William C. Olson, P. Stevis, Matthew C. Franklin
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72d62689c5dd46733c3aa071e7330253553893df" target='_blank'>
              IRF4 is a master regulator of multinucleated giant cell formation
              </a>
            </td>
          <td>
            M. Hofmann, G. Heller, M. Kieler, L. Musiejovsky, Lukas Kremp, Martina Kerndl, P. Ettel, Birgit Niederreiter, Lucia Quemada Garrido, Anahita Sedighi, Thomas Krausgruber, Leonhard X. Heinz, Axel Dietschmann, David Voehringer, Bart Everts, Thomas Weichhart, Omar Sharif, Stephan Blüml, Gernot Schabbauer
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Background Despite recent therapeutic advances, 5-year survival for AML in adults remains ~35% (SEER estimate). Immunotherapies have shown limited efficacy in AML largely due to the lack of surface antigens that clearly distinguish malignant cells from normal hematopoietic counterparts. Conventional transcriptomics and proteomics have only found targets lacking clear distinction between AML, myeloid cells, and hematopoietic stem and progenitor cells (HSPCs). Here, we employed high-resolution surface proteomics of primary leukemia bone marrow aspirates to identify both tumor-enriched canonical surface proteins as well as a new class of “non-canonical” surface protein targets, i.e. typically intracellular proteins aberrantly presented on the AML cell surface.
 Methods We used periodate-based cell surface-capture (CSC) and lysine-targeting (sulfo-NHS) labeling strategies, analyzed with both DDA and DIA proteomic methods, to profile surface proteins in newly diagnosed (n=35), relapsed (n=25), and remission (n=16) primary AML samples. An independent cohort (n=38) was used for target validation, with assessment in peripheral blood mononuclear cells and HSPCs for off-tumor expression. Plasma membrane localization of non-canonical targets was validated using flow cytometry and Western blotting. Interactors of canonical and non-canonical membrane proteins were evaluated through a recently-described multiscale photocatalytic proximity labeling strategy (Lin et al., Science (2024)). For promising AML-specific target DC-SIGN/CD209 we synthesized an MMAE-based antibody-drug conjugate (ADC).
 Results Canonical cell surfaceome analysis revealed a distinctive expression of potential targets including CD209, CD300LB, LILRA1 amongst >150 significantly upregulated surface proteins in AML as compared to remission samples. Single-cell RNA-seq analysis of publicly available datasets supported AML specificity of these targets. Flow cytometry validated expression of CD209 across 5 cell line models and primary AML (n=15), with protein expression independent of genomic alterations. In non-canonical surfaceome analysis, we identified a preponderance of RNA binding proteins, consistent with our recent findings in other cell lines (Perr et al, Cell (2025)). However, we also identified many Golgi-, ER- and Mitochondrial-annotated proteins as possible AML-specific cell surface antigens given absence in remission sample surface proteomic data. Nine of these non-canonical surface antigens were confirmed to be surface-localized on AML cells by flow cytometry (n=13). Flow cytometry confirmed particularly strong expression of GORASP2 on the AML cell surface, whereas HSPC flow cytometry showed no expression of either CD209 or GORASP2. Further in silico analysis, Western blotting, and super-resolution imaging confirmed localized clusters of GORASP2, a canonical Golgi body protein, on the AML plasma membrane. Multiscale photocatalytic-based proximity labeling proteomics validated that GORASP2 interacted with a similar set of canonical membrane proteins as known surface antigens ITGB2 and CD209. An antibody-drug conjugate developed against CD209 demonstrated strong preclinical efficacy both in vitro and in vivo versus AML cell line models. The in vitro ADCcytotoxic assay targeting CD209 showed prominent efficacy with IC50 ranging 1.5nM to 3nM in various AML cell lines. Ex vivo primary AML cytotoxicity assays showed killing in an antigen dependent manner. Treatment with CD209 ADC in Nomo-1 AML disseminated murine model suppressed tumor growth and improved overall survival compared to controls.
 Conclusion Our study provides a comprehensive surfaceome landscape of AML, identifying both canonical and non-canonical membrane proteins as promising immunotherapy targets. The successful validation of these targets across multiple AML lines and primary samples, with absent expression on healthy cells or HSPCs, validates their potential as AML-specific immunotherapy targets. Preclinical efficacy of a CD209-targeting ADC underscores the ability to develop novel AML-selective therapies based on this data. This work will also drive fundamental investigation into new areas of AML tumor biology and surface protein trafficking.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1caaac1aa97579ab2821341a6a42048579774426" target='_blank'>
              Unbiased in-depth surfaceome profiling of Acute Myeloid Leukemia reveals novel immunotherapy targets beyond canonical membrane proteins
              </a>
            </td>
          <td>
            Abhilash Barpanda, Szu-Ying Chen, Benson George, Trenton PetersClarke, Peiyuan Chai, Vipul Kumar, Sujata Walunj, Amit Prabhakar, Haley Johnshon, Snehal Ganjave, Rohit Bhadoria, Sanjana Prudhvi, B. Patiño-Escobar, Adila Izgutdina, Marko Wiita, Rucha Deo, Amrik S Kang, Sham Rampersaud, V. Steri, H. Geng, Aaron C. Logan, Kevin Leung, J. Ostrem, Benjamin Huang, Akiko Hata, Andrej Šali, James Wells, Ryan T Flynn, Arun Wiita
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) is a highly complex multicellular system and has been shown to play a fundamental role in dictating tumorigenesis. High expression of regulatory surface molecules, cytokines, and chemokines in fibroblast niches within lung tumor stroma, paired with their colocalization with Treg cells, suggests cancer-associated fibroblast (CAF) niches may alter the functional response of Treg cells and influence anti-tumor immunity. Here, utilizing a transcriptomic approach, we have uncovered a distinct CAF population – which we term “imCAFs” – that is predominantly localized to tumor-derived tissue samples and is enriched for immunomodulatory gene signatures, T cell ligands, and chemoattractants. This population expresses a unique set of cell surface molecules that permit their identification and isolation from lung tumor tissue. Interestingly, CXCL9, a well-established T cell chemoattractant, was among the top differentially expressed genes in the imCAF population. We were able to demonstrate that TME Treg cells preferentially migrate to CXCL9 gradients, and colocalize with CXCL9-producing imCAFs in lung tumors. CXCL9-producing imCAFs attract a unique subset of Treg cells that highly upregulate the CXCL9 receptor, CXCR3. This subset of Treg cells is functionally distinct from CXCR3-negative Treg cells. Our analyses suggest that imCAFs play a critical role in dictating the recruitment and function of a specific subset of Treg cells to the TME in lung tumors.



 Supported by NIH/NHLBI R01HL148718; NIH/NIAID T32AI106711.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/909bf82020ae6c71525db0b4290dae269c849d42" target='_blank'>
              Immunomodulatory cancer-associated fibroblasts influence Treg cell dynamics in lung cancer 2712
              </a>
            </td>
          <td>
            Nicholas Arpaia, Olivia R. Ringham
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Background The immune microenvironment of the three most common gynaecological malignancies—tubo‐ovarian cancer, endometrial cancer and cervical cancer—has not been systematically studied, limiting clinical application. Methods This study analyses 272 389 CD45+ immune cells by integrating publicly available single‐cell RNA sequencing (scRNA‐seq) data from 111 tumour and non‐malignant tissue samples. We identified distinct subsets within immune cells: 11 for monocytes/macrophages, six for CD4 T cells, eight for CD8 T cells and five for B cells, detailing their distribution, prevalence and distinct functions. Results A pro‐angiogenic macrophage subset linked to NF‐κB signalling was associated with worse clinical outcomes and an interferon‐primed macrophage subset correlated with improved survival by recruiting T cells through CXCL9/10/11 secretion, as confirmed by multi‐colour immunohistochemistry. T cells exhibited dynamic roles in tubo‐ovarian cancer, with CD8 Tex cells contributing to immune dysfunction and poor prognosis, while CD8 Trm cells in early‐stage tumours supported immune surveillance. Additionally, we identified co‐stimulatory and co‐inhibitory receptor interactions and classified distinct B cell subsets with varying prognostic implications. Conclusions This comprehensive analysis of the tumour immune microenvironment in gynaecological malignancies provides new insights into immune cell composition and function offering potential for optimising immunotherapies and improving clinical outcomes in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/490064a2fa8b085af77b6abab69c28754ca9996b" target='_blank'>
              Single‐cell landscape of the tumour immune microenvironment in human gynaecologic malignancies
              </a>
            </td>
          <td>
            Simin Yin, Sen Li, Mengyan Tu, Junfen Xu
          </td>
          <td>2025-11-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Colorectal cancer (CRC) often occurs in combination with sarcopenia (SP), which affects the prognosis of CRC, but the histopathological mechanisms underlying this process are unclear. Single-cell transcriptome technology provides a tool for resolving multi-organ pathological interactions, and previous studies have lacked direct molecular evidence of a trans-tissue microenvironment. To address this, we applied an integrated analytical approach. The integration of single-cell transcriptome data from 16 CRC tumor tissues (scRNA-seq) and 17 SP muscle tissues (snRNA-seq) was used to systematically resolve heterogeneity across tissue microenvironments using Harmony to correct for batch effects and combining cellular annotations, differential expression analyses, and CellChat communication modeling. Our analysis revealed that CRC and SP share 3 core cell types, namely, endothelial cells, fibroblasts and monocytes, but they show distinct transcriptional and functional differentiation in different tissues. CRC cells are enriched in the inflammatory, angiogenic and matrix remodeling pathways, accompanied by the upregulation of hypoxia-associated genes, whereas SP is biased toward muscle structural maintenance, metabolic regulation and repair functions. The MIF-CD74, COLLAGEN and THBS-CD47 pathways were active in CRC, whereas SDC4 and CD36-related signals were dominant in SP. VWF, PDGFRA and FCN1 were stably expressed in both groups, suggesting the existence of a conserved homeostatic regulatory network. In conclusion, this study reveals the cross-tissue functional heterogeneity of cell types shared by CRC and SP, suggests a set of potential pervasive regulators, and provides new perspectives for understanding the systemic effects of tumors and the microenvironmental mechanisms of sarcopenia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28aac551ab88d08b24ed6a70bb80afdcde46ff4f" target='_blank'>
              Single-cell transcriptome analysis elucidates the microenvironmental interactions between colorectal cancer and sarcopenia
              </a>
            </td>
          <td>
            Tihong Qiu, Xiaoying Wang, Jie Li, Yiping Guan, Hongyou Wen
          </td>
          <td>2025-11-28</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) play crucial and important role in cancer dynamics by affecting homeostasis, immunosuppression, and angiogenesis within the tumor microenvironment. Using single-cell transcriptomics, we constructed a comprehensive atlas of myeloid cell populations across healthy and pan-cancer tissues that revealed heterogeneity. Our analysis suggested that TAMs may arise from two distinct origins: C1QC+ TAMs, which likely are derived from resident tissue macrophages, SPP1+ TAMs and ISG15+ TAMs, which appear to originate from circulating monocytes. Regarding immature myeloid-derived suppressor cells (MDSCs), we highlighted THBS1+ MDSCs and their descendants, SPP1+ TAMs, as key contributors to tumor progression, immunosuppression, and angiogenesis. We proposed a dichotomous model for TAMs, in which C1QC+ TAMs are associated with better patient outcomes, whereas the THBS1+ MDSCs - SPP1+ TAMs lineage correlates with poorer survival and unfavorable response to immunotherapy. This study offers insight into the complex interactions among monocyte-macrophage subtypes and sheds light on TAM heterogeneity and its implications for cancer progression and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a07f8d41863dfbb5389df0779f0c2e6e40c36fc" target='_blank'>
              Pan-cancer single-cell RNA sequencing analysis refines multi-origin monocyte and macrophage lineages.
              </a>
            </td>
          <td>
            Truc Do Thanh Nguyen, Andrew J Lee, Hyun Jung Park, Nameeta Shah, Bayrta Mandzhieva, Dong-Sup Lee, Inkyung Jung, Woong-Yang Park
          </td>
          <td>2025-11-13</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related deaths worldwide. Existing therapeutic options have limited efficacy, particularly for lung squamous cell carcinoma (LUSC), underscoring the critical need for the identification of new therapeutic targets. We previously demonstrated that the Transmembrane Serine Protease TMPRSS11B promotes the transformation of human bronchial epithelial cells and enhances lactate export from LUSC cells. Here, we evaluate the impact of TMPRSS11B activity on the host immune system and the tumor microenvironment (TME). Tmprss11b depletion significantly reduces tumor burden in immunocompetent mice and triggers an infiltration of immune cells. RNA FISH analysis and spatial transcriptomics in the autochthonous Rosa26-Sox2-Ires-GfpLSL/LSL; Nkx2-1fl/fl; Lkb1fl/fl (SNL) model reveal an enrichment of Tmprss11b expression in LUSC tumors, specifically in Krt13+ hillock-like cells. Furthermore, utilizing ultra-pH-sensitive nanoparticle imaging and metabolite analysis, we identify regions of acidification, elevated lactate, and enrichment of immunosuppressive (M2-like) macrophages in LUSC tumors. These results demonstrate that TMPRSS11B promotes an acidified and immunosuppressive TME and nominate this enzyme as a therapeutic target in LUSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edde3e6259df0a2e4e7c324d76c028ecfb0de34e" target='_blank'>
              TMPRSS11B promotes an acidified microenvironment and immune suppression in squamous lung cancer
              </a>
            </td>
          <td>
            Hari Shankar Sunil, Jean R. Clemenceau, Anthony Grichuk, Isabel Barnfather, S. R. Nakkireddy, Luke Izzo, Q. Feng, William Hartnett, Bret M. Evers, Lisa Thomas, I. Subramaniyan, Li Li, William C Putnam, Steven Hepensteil, Jingfei Zhu, Barrett L. Updegraff, John D. Minna, R. DeBerardinis, Tae Hyun Hwang, Jinming Gao, T. Oliver, Kathryn A. O’Donnell
          </td>
          <td>2025-11-10</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 IDH-mutant astrocytomas often progress from low-grade to higher-grade tumors, through incompletely defined mechanisms. The neuronal microenvironments through direct and paracrine-mediated signaling drive glioblastoma progression; therefore, we hypothesized its role in IDH-mutant astrocytoma evolution. In this study, we integrate sixteen patient-matched newly diagnosed and recurrent astrocytomas (CNS WHO grades 2–4), which were interrogated using spatial RNA sequencing, multiplexed immunofluorescence, and three-dimensional co-cultures comprising mouse cortical neurospheres (cNS) and patient-derived astrocytoma organoids (GliA; n = 17). Harmony-integrated UMAP of 10,189 spatial transcriptomes identified ten gene-expression programs across 8 IDH-mutant astrocytomas. CNS WHO grade 2 and grade 3 tumors shared overlapping transcriptomic profiles, whereas grade 4 tumors formed a distinct cluster. A 28-gene IDH-mutant glioma neural connectivity score, characterized by TSP1, NDRG2, and Cx43, correlated with overall survival. Immunofluorescent staining confirmed overexpression of TSP1, NDRG2, and Cx43 in cortex-infiltrative and dense tumor regions. Functional interrogation using novel in vitro co-culture model systems revealed a grade-specific increase in Ki67⁺ proliferating tumor cells throughout the tumor, not restricted to areas of glioma-neuron integration. This was quantified across 7 patient-matched GliA–cNS co-cultures (n = 172 regions) and GliA monocultures (n = 193 regions). Proliferation in GliA co-cultures decayed radially with increasing distance from the neuron–tumor interface, demonstrating that astrocytomas are most proliferative in close proximity to neurons. Furthermore, GliA-cNS co-culture demonstrated no increase in neuronal firing rates and synchrony, as quantified across 22 GliA–cNS co-cultures from 5 patients and 20 cNS monocultures. IDH-mutant astrocytomas display uniform, grade-dependent synaptic remodeling, revealing potential therapeutic targets and treatments for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6edd08f2509905cf56e834334ff1638c7882e2f" target='_blank'>
              CNSC-98. SYNAPTIC REMODELING DRIVES IDH-MUTANT ASTROCYTOMA PROGRESSION AND EVOLUTION
              </a>
            </td>
          <td>
            S. Oten, K. Mirchia, T. Picart, M. Nguyen, V. Ambati, H. Vasudevan, Jacob Young, Jennie W Taylor, S. Krishna, D. Brang, Joanna Phillips, A. Perry, Mitchell S. Berger, S. Chang, J. D. de Groot, S. Hervey-Jumper, D. Raleigh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 . Chronic lymphocytic leukemia (CLL) cells bidirectionally interacts with the tumor microenvironment (TME). Leukemic cells establish within lymph nodes (LNs) a pro-tumoral niche that drives disease progression and therapy resistance. A deeper understanding of dynamic CLL-TME modifications is essential to identify novel markers and actionable cell-to-cell interactions for more specific targeted therapies. We applied Digital Spatial Profiling (DSP) to characterize tumor and immune cells, including T lymphocytes and macrophages, in CLL-infiltrated LNs, keeping distinct microanatomical areas present within proliferation centers (PCs) and from those pertinent to the surrounding non-PC.



 Seventeen FFPE CLL-LNs were grouped into 3 clinical subsets: A (n=6) indolent CLL; B (n=4) CLL requiring treatment within 6 months after LN biopsy; C (n=7) relapse/progressive CLL. Spatial transcriptomics was performed using the GeoMx® Digital Spatial Profiler. Region of interest (ROIs) were identified and selected based on multiplexed immunofluorescence with anti-CD3 anti-CD68 and MUM1 for the identification of PCs T cells, macrophages, and PC neoplastic B cells, respectively. ROIs inside and outside PCs were selected for each cell type.
 Normalized transcriptomic matrices were generated for each cell segment, and Gene Ontology (GO) analysis was performed on pre-ranked differentially expressed genes (DEGs; *p*-value < 0.05, |log₂ fold change| > 0). Pathways were selected based on an FDR Q-value threshold (Q < 0.05), NOM p-value (< 0.05) and ranked by Normalized Enrichment Score (NES).



 DSP was performed on a total of 295 ROIs, comprising 88 CD3+ ROIs, 106 CD68+ ROIs, 101 PC neoplastic B cells ROIs and distributed within or outside PCs. First, transcriptomic profile of pooled ROIs within and outside PC were compared for each cell type. As expected CLL ROIs within PC showed upregulation of MUM1, MYC, and PCNA genes, thus providing evidence of ongoing leukemic cell proliferation. CD3⁺ T lymphocytes inside PCs exhibited enrichment in pathways associated with proliferation and pro-inflammation, and upregulation of STAT1, NF-kB1, and CCL19 genes compared to non-PCs ROIs. As for macrophages, they showed enrichment in genes associated with oxidative stress (SOD1, TRX2), chromatin remodelling and inflammatory pathways, and showed upregulation of the checkpoint molecule LAG3.
 Our subsequent analysis focused on comparing cell-specific transcriptomic pattern associated with the three distinct clinical subgroups. In group A, CD3+ T lymphocytes retain an immunological, activated, and homeostatic-like transcriptional profile, with enrichment of pathways related to positive immune system process, cell activation and signalling, along with upregulation of key genes including TNF, IL17D, STAT5B, NF-kB1, IFNA16. In group B, CD3+ T lymphocytes displayed upregulation of genes like ENO1, AKT2, PTPN1 associated with metabolism and cellular activation, vesicle trafficking, mitochondrial activity, and chromatin remodelling. In group C, a defective T cell status became apparent (enriched pathways: apoptosis, downregulation of cell function, motility, negative transcription, with main genes including CR1, MYB, IL3RA, IL12A, TNFRSF21). CD68+ macrophages showed an increased functional state in group A, with enriched pathways related to immunity and motility. However, in more aggressive phases of the disease (groups B and C) they showed enrichment in cell activation, chromatin remodelling, vesicle trafficking and mitochondrial activity. In group A, tumor cells were enriched in B cell activation, antigen-receptor mediated signalling pathway, and immune response regulating signalling pathways. In group B, we observed pathways associated with cell cycle, chromatin remodelling, vesicle trafficking and organelles, and motility while, in the group C, vesicle trafficking and organelles, along with inflammation-related pathways, were prominent.



 Our findings demonstrate that within CLL-LNs, both TME and leukemic cells activate distinct transcriptional programs based on their location inside or outside the anatomically defined MUM1+ PCs, which function as proliferative hubs. Moreover, our transcriptomic analysis reveals that, as CLL clinically progresses, tumor and TME co-evolve, shifting from a more physiological condition of immunological activity, toward an increasingly dysfunctional and immunologically altered status.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60177207e0e4c3f2a43d8f1489bc64ee39f2a20c" target='_blank'>
              Spatial transcriptomic analysis of chronic lymphocytic leukemia lymph nodes reveals tumor microenvironment evolution during disease progression
              </a>
            </td>
          <td>
            M. C. Vegliante, Benedetta Apollonio, A. Bucci, Giulia Notarpietro, S. A. Pappagallo, P. Mondelli, Rossana Daprile, A. Scattone, M. Frenquelli, A. Guarini, Giancarlo Castellano, A. Ciarrocchi, B. Donati, M. Ponzoni, P. Ghia, C. Minoia
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) progression is shaped by the tumor microenvironment, particularly tumor-associated macrophages (TAMs), which often adopt immunosuppressive functions. CD300e, a myeloid receptor involved in immune regulation, has an uncharacterized role in CRC. Methods Functional studies were conducted in azoxymethane/dextran sodium sulfate and MC38 murine CRC models using CD300e systemic and myeloid-specific CD300e knockout mice, and adoptive transfer experiments assessed macrophage-intrinsic effects. Human studies included analysis of CD300e expression in matched tumor and normal tissue from patients with CRC and in vitro co-culture of patient-derived colon tumor organoids with monocytes to study CD300e induction and TAM polarization. Results In vivo, CD300e deficiency led to reduced tumor burden, enhanced major histocompatibility complex expression on TAMs, and improved T-cell responses. CD300e-deficient macrophages exhibited increased phagocytic activity, antigen presentation, and support for T-cell proliferation and cytotoxicity. Adoptive transfer confirmed that macrophage-intrinsic CD300e expression is sufficient to suppress T-cell function and promote tumor growth. In patients with CRC, CD300e is selectively upregulated in tumor-infiltrating monocytes and macrophages, driving a suppressive phenotype marked by impaired antigen presentation. Tumor-derived signals in vitro induce CD300e expression and promote a protumorigenic macrophage profile. Conclusions Our findings identify CD300e as a critical regulator of macrophage-mediated immune suppression in CRC and a potential target for reprogramming TAMs to enhance immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c56c500ab6751223d28c46c0f7327dc521a283" target='_blank'>
              CD300e is a driver of the immunosuppressive tumor microenvironment and colorectal cancer progression via macrophage reprogramming
              </a>
            </td>
          <td>
            Annica Barizza, Stefania Vassallo, L. Masatti, Mattia Laffranchi, Sofia Giacometti, S. Lonardi, Mattia Bugatti, Sara Coletta, C. Della Bella, M. D’Elios, Simone Pizzini, A. Rosato, W. Vermi, Matteo Fassan, G. Spolverato, Silvano Sozzani, E. Calura, R. Sommaggio, Gaia Codolo
          </td>
          <td>2025-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="


 INTRODUCTION Monoclonal gammopathies –including MGUS, smoldering multiple myeloma (SMM), and multiple myeloma (MM)– are driven by clonal expansion of malignant plasma cells. Understanding their clonal architecture and the genomic landscape at single-cell resolution is essential to unravel disease progression and guide treatment strategies, particularly in the context of novel immunotherapies. In this study, single-cell DNA sequencing, was combined with surface protein profiling, to comprehensively dissect somatic mutations, copy number variations (CNVs), clonotypes, and immunophenotypes across disease stages and treatment responses. This novel single cell multi-omics approach provides a powerful perspective on the genomic and functional heterogeneity of plasma cell disorders, offering key insights for precision medicine and therapeutic stratification.
 METHODS CD138+ were isolated from bone marrow aspirates of 11 patients (21 samples) with MGUS, SMM, or MM using AutoMACS sorting. Patient samples included diagnostic, progression, relapse, and treatment-resistant stages. Single-cell DNA sequencing was performed using a panel of 839 amplicons, combined with surface protein profiling via a 20-antibody cocktail. Clonal architecture was inferred by integrating somatic mutations, clonotyping and CNVs. Surface protein expression was used to correlate genotypic features with targetable phenotypes at single cell resolution.
 RESULTS Clonal complexity and genomic architecture showed substantial heterogeneity across disease stages and therapeutic contexts. In early disease, such as MGUS and SMM, some cases already exhibited complex CNV profiles or early driver mutations, indicating subclinical clonal evolution. Several patients displayed stable CNV landscapes over time, while mutational profiles fluctuated, suggesting a selection or suppression of cell-specific subclones rather than wholesale genomic shifts.
 In relapsed/refractory multiple myeloma, branched clonal evolution was frequently observed, with sequential acquisition of mutations affecting genes such as TP53, FRG1, and TRAF2 in specific cells. In some cases, dual-clone coexistence with distinct genomic and immunoglobulin rearrangement profiles was evident, reflecting convergent tumor evolution. Furthermore, certain patients exhibited a distinct pattern of surface antibody expression, indicating the presence of plasma cells co-expressing both major therapeutic targets in MM, GPRC5D and BCMA, while other plasma cell populations expressed only one or neither of these targets.
 A key finding emerged when integrating genomic data with clinical outcomes: patients harboring monoallelic deletions and/or mutations in GPRC5D or BCMA (TNFRSF17) consistently showed limited or no clinical response to corresponding targeted therapies. As an example, one patient with biallelic inactivation of GPRC5D in near 90% of the cells prior to TALQUETAMAB treatment, failed to respond, correlating with absent membrane expression of the target. Conversely, a separate patient who progressed on anti-GPRC5D therapy demonstrated a favorable clinical response to BCMA-directed CAR-T cells, consistent with preserved BCMA expression and copy number in all the cells.
 These findings indicate that the genomic integrity and surface expression of immunotherapeutic targets strongly influence treatment efficacy. Importantly, discrepancies between gene copy number and surface protein levels — such as cases with BCMA amplification but heterogeneous expression — highlight post-transcriptional regulation as an additional layer of resistance.
 CONCLUSIONS Single-cell DNA and protein co-profiling in monoclonal gammopathies provides critical insights into clonal evolution, therapeutic resistance, and the dynamic expression of immunotherapeutic targets. Specifically, GPRC5D genomic alterations and reduced protein expression may compromise response to targeted therapies, highlighting the need for functional biomarker screening. Our data support the clinical relevance of integrating single-cell genomics in the management of plasma cell dyscrasias to tailor therapy and anticipate resistance. Our results suggest that performingsingle-cell genomic and proteomic profiling prior to treatment initiation may improve therapeutic selection and predict response, particularly in the era of targeted immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b66dd32e338c623da193862d4d414630dc6f82e" target='_blank'>
              Single-cell DNA profiling unveils clonal evolution and identifies mechanisms of therapeutic resistance and progression in monoclonal gammopathies
              </a>
            </td>
          <td>
            B. Ariceta, P. Aguirre-Ruiz, Andrea Castro, D. Alignani, Saioa Vicente, A. Lopez, A. Alfonso-Piérola, L. Tamariz-Amador, Jesús F San-Miguel, P. Rodríguez-Otero, Felipe Prósper
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 Tumors represent highly dynamic systems that rely on the integration of numerous signaling pathways to drive uncontrolled growth and adapt to selective pressures, including those induced by anti-cancer therapies. Glioblastoma (GBM), the most common primary malignant brain tumor, is characterized by a high degree of heterogeneity, including the presence of therapeutically resistant cancer stem cells that drive tumor recurrence after therapy. A frequent genetic modification in GBM is the amplification of the proto-oncogene MYC, which is essential for GBM cancer stem cell proliferation and tumor initiation. However, targeting cancer stem cells by inhibiting MYC has been challenging due to its intrinsically disordered structure as a transcription factor and its multifaceted roles in cell biology. The identification of upstream mechanisms driving MYC expression in GBM may provide additional opportunities for therapeutic development. Here, we provide the first evidence for a non-gap junctional role of connexin 43 (Cx43) in driving MYC expression in patient-derived GBM cancer stem cells. Knockdown of Cx43 reduced GBM cell growth, survival, and tumor initiation through a reduction in expression of MYC and MYC target genes. Cx43 depletion also reduced expression of WNK lysine-deficient protein kinase 1 (WNK1), a kinase shown to be essential for MYC expression in other systems. WNK1 depletion phenocopied Cx43 knockdown and reduced GBM cell growth, tumor initiation, and MYC expression. Together, these results define a novel WNK-MYC signaling network downstream of Cx43 that promotes tumor growth and cancer stem cell phenotypes in GBM and may lead to improved therapies for GBM by targeting this axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f4480478312773b234eefda0633c46b41b15ddc" target='_blank'>
              CSIG-09. A CX43-WNK1-C-MYC SIGNALING AXIS DRIVES GLIOBLASTOMA CANCER STEM CELL GROWTH AND SURVIVAL
              </a>
            </td>
          <td>
            E. Mulkearns-Hubert, Nicole Hajdari, Aymerick Gaboriau, D. Silver, Andrew Dhawan, Justin D. Lathia
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Purpose Monocytes and macrophages are recognized as predominant immune populations in human glioblastoma, where they play vital roles in tumor progression. Despite their established significance, the heterogeneity of these cells—particularly within the monocyte compartment—remains incompletely characterized. Methods We comprehensively used scRNA-seq, spatial transcriptome sequencing combined with immunofluorescence and T cell co-culture assays to illuminate the heterogeneity and function of monocyte in glioblastoma. Results In this study, from the perspective of ligand-receptor networks, we have identified and characterized three distinct glioblastoma subtypes. Single-cell RNA-seq analysis further revealed that the C3 subtype with bad prognosis exhibited a higher proportion of S100A9high monocytes. Spatial transcriptomics combined with immunofluorescence assays demonstrated that these S100A9high monocytes were spatially adjacent to M2 macrophages, exhausted CD8+ T cells, and endothelial cells. In vitro T cell co-culture assays revealed S100A9high monocyte produced elevated levels of the immunosuppressive cytokine IL-10, reactive oxygen species (ROS) and inducible nitric oxide synthase (iNOS), all of which might impair T cell function and immune response. Notably, elevated abundance of S100A9high monocyte correlated with poor patient prognosis. Conclusion In summary, our results deciphered the heterogeneity of monocytes in glioblastoma and identified a novel poor prognosis-associated monocyte subset, S100A9high monocytes, which foster an immunosuppressive, pro-tumorigenic microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b56f421b008f9fd171284dfd7fddcbd5bc0f429e" target='_blank'>
              Decoding the Prognosis-Related S100A9high Monocyte in Glioblastoma Using Single-Cell and Spatial Transcriptome Sequencing
              </a>
            </td>
          <td>
            Xiucan Li, Ying Qin, Pengfei Gao, Huixue Wang, Yanling Liu, Lian Ren, Dongdong Wu, Xueyuan Heng
          </td>
          <td>2025-11-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a highly dynamic network shaped by immune, vascular, and stromal interactions, further modulated by extracellular factors. The high tumor heterogeneity complicates treatment due to evolving resistance. Biodynamic imaging (BDI), a 3D coherence-gated holography technique, quantifies intracellular motion to assess phenotypic responses to therapy. We applied BDI to prostate cancer (PCa) treated with IL-27-based combinations to evaluate the dynamic responses of immunotherapy. This study is the first to integrate BDI with RNA-seq to correlate physiological changes with gene expression. Using a subcutaneous PCa model, tumors were treated ex vivo with chemotherapy and immunomodulatory agents. BDI enabled capturing motion frequency shifts (0.01–12.5 Hz), while RNA-seq revealed the molecular changes in the tumors. IL-27 showed potential to modulate the immunologically “cold” TME and enhance therapeutic efficacy. This integrative approach offers a novel platform for evaluating combination strategies in PCa and may inform more effective treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1439e864183bad47fa0679ec246c930f10dfa35f" target='_blank'>
              Exploring tumor dynamics and responses of prostate cancer to IL-27 based treatment combinations through biodynamic imaging and RNA sequencing analyses
              </a>
            </td>
          <td>
            Shreya Kumar, Dawith Lim, Harish Kothandaraman, Nadia Atallah-Lanman, David Nolte, John Turek, Marxa L. Figueiredo
          </td>
          <td>2025-11-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The efficacy of immunotherapy in bladder cancer (BCa) is highly influenced by tumor heterogeneity, underscoring the urgent need for mechanistic insights and precise molecular subtyping. Adenosine-to-inosine RNA editing (AIRE), a critical post-transcriptional mechanism driving protein diversity, offers a promising lens for decoding the immune microenvironment in BCa. In this study, we established an integrative framework combining molecular, imaging, and pathological data to identify and validate a novel immune-related BCa subtype. Based on multi-center transcriptomic datasets, we employed NMF, WGCNA, and Lasso algorithms to classify tumors into two distinct immune subtypes characterized by different levels of immune cell infiltration and divergent responses to immunotherapy. Mechanistic validation revealed that ANXA1 and RASAL2 may contribute to immunosuppressive niche formation via the TNF-NFκB-CXCL8 axis, potentially acting as molecular triggers for subtype differentiation. Furthermore, radiomic and histopathological analyses at the imaging and tissue levels supported the molecular findings, providing orthogonal validation of subtype-specific characteristics. Collectively, this study introduces an AIRE-based immune molecular subtype of BCa and systematically characterizes it across multiple dimensions. By achieving a closed-loop from identification to validation and clinical translation, our findings offer a new perspective for patients most likely to benefit from immunotherapy and advancing precision treatment strategies in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89065f12cff37f2644ca2e30f67834c8758ac184" target='_blank'>
              Integrative molecular–radiopathomic characterization of AIRE-driven immune subtypes in bladder cancer
              </a>
            </td>
          <td>
            Ji Liu, Shuwen Sun, Yadong Guo, Zhuoran Gu, Hong Wang, Yang Yu, Shiyu Mao, Yong Wei, Zongtai Zheng, Bing Shen, Xudong Yao
          </td>
          <td>2025-12-14</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="This study addresses the limited mechanistic understanding behind medical imaging for tumor microenvironment (TME) assessment. We developed a novel framework that analyzes tumor imaging heterogeneity index (TIHI)-correlated genes to uncover underlying TME biology and therapeutic vulnerabilities. DCE-MRI and mRNA data from 987 high-risk breast cancer patients in the I-SPY2 trial, together with mRNA data from 508 patients in GSE25066, were analyzed. TIHI-associated genes were identified via Pearson correlation, clustered via weighted gene co-expression network analysis (WGCNA), and subgroups were defined via non-negative matrix factorization (NMF). The clinical relevance of the image-to-gene comprehensive (I2G-C) subtype defined by subgroups was assessed using logistic regression and Cox analysis. I2G-C comprised four clusters with distinct immune and replication/repair functions. It further stratified receptor, PAM50, and RPS5 subtypes. The “immune+/replication+” was more likely to achieve pathological complete response (pCR) (OR = 2.587, p < 0.001), while the “immune−/replication−” was the least likely to achieve pCR (OR = 0.402, p < 0.001). The “immune+/replication+” showed sensitivity to pembrolizumab (OR = 10.192, p < 0.001) and veliparib/carboplatin (OR = 5.184, p = 0.006), while “immune-/replication-” responded poorly to pembrolizumab (OR = 0.086, p < 0.001). Additionally, “immune+/replication-” had the best distant recurrence-free survival (DRFS), whereas “immune-/replication+” had the worst (log-rank p = 6 × 10−4, HR = 5.45). By linking imaging heterogeneity directly to molecular subtypes and therapeutic response, this framework provides a robust, non-invasive surrogate for genomic profiling and a strategic tool for personalized neoadjuvant therapy selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaac41c7947552aff84e1e324fe7769979665178" target='_blank'>
              Tumor Imaging Heterogeneity Index-Inspired Insights into the Unveiling Tumor Microenvironment of Breast Cancer
              </a>
            </td>
          <td>
            Qingpei Lai, Xinzhi Teng, Jiang Zhang, Xinyu Zhang, Yufeng Jiang, Yao Pu, Peixin Yu, Wen Li, Tian Li, J. Cai, Ge Ren
          </td>
          <td>2025-11-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Background:
 Chronic lymphocytic leukemia (CLL) is a heterogeneous B-cell malignancy characterized by clonal expansion and immune dysfunction. Although peripheral blood (PB) is frequently used for genomic and transcriptomic profiling, it may not fully capture the biological activity of the disease. The lymph node (LN) microenvironment plays a central role in promoting malignant cell survival, proliferation, and immune evasion. Genomic alterations such as TP53, NOTCH1, and SF3B1 mutations are associated with adverse outcomes, but it remains unclear how these mutations interact with the local immune contexts across compartments. We investigated how clonal architecture and immune composition differ between PB and LN, and how these features relate to evolutionary dynamics.
 Methods:
 We performed whole exome sequencing (WES) and bulk RNA sequencing on paired LN biopsies and CD19⁺-purified PB samples from 26 patients with CLL. We selected samples from patients with known CLL driver mutations. Somatic variants, copy number alterations, and clonal architecture were inferred using SuperFreq, and cancer cell fraction (CCF) was used to define clonal distribution. Variants were filtered to include only those with >3 alternate reads, >20 total reads, a variant allele frequency (VAF) >0.02, and classified as non-synonymous mutations. CCF was considered equivalent to VAF if >0.6, otherwise CCF was calculated as VAF * 2. CCF was adjusted by dividing CCF by the ancestral or founding clone CCF as a proxy for tumor cell fraction. Clones were considered compartmentalized if the absolute CCF difference between LN and PB exceeded 0.25. Exome alteration was calculated by summing the length of exonic targets falling in copy-number segments with an absolute log2 copy-ratio >0.2 and dividing by the total captured exome length. LN immune composition was estimated from bulk RNA-seq data using CIBERSORTx.
 Results:
 Of the 26 patients, 12 were treatment-naive and 18 were IGHV unmutated. In the PB, mutational frequency was as follows: 46.1% NOTCH1, 42.3% TP53, and 26.9% SF3B1. Oncoplot analysis revealed the presence of multi-hit alterations and a predominance of missense and frameshift variants.
 Analysis of paired PB and LN samples revealed notable heterogeneity in clonal architecture. Using SuperFreq, we identified between 2-6 clones per patient (median: 4). The median number of subclones did not differ between LN and PB (4 vs. 4), treatment naive vs. relapsed/refractory status (4 vs. 4) or between IGHV unmutated vs mutated (4.5 vs. 4). We evaluated genomic instability and found significantly higher exome alteration in patients who were IGHV unmutated (P = .03). Clonal compartmentalization (CCF difference of |.25|) was observed in 18 of 26 (69%) patients, with 5 patients showing clonal evolution in the LN (28%), 6 in the PB (33%), and 7 with distinct clones evolving in both LN and PB (39%).
 SF3B1 mutations were common in clones with CCF trending higher in the LN compared to PB (median ΔCCF 0.16, P = .11). Interestingly, CCF of NOTCH1 mutations was balanced across compartments (median ΔCCF = -0.02; P = .36), and CCFof TP53 mutations was numerically slightly lower in LN than PB (median ΔCCF –0.05; P = .97).
 LN samples with stable clonal populations had higher proportions of CD4⁺ naïve T cells compared to those with evolving clones (P = .02), but no other significant changes in T cell populations were noted. Patients with NOTCH1 or SF3B1 mutations had reduced dendritic cell abundance in LN (P = .04 and P = .06, respectively) compared to those with TP53 mutations or no known driver mutations.
 Conclusions:
 This study demonstrates distinct clonal and immune features between PB and LN compartments in CLL, a heterogeneous clonal architecture across compartments. Differences in clonal composition across LN sites within individual patients suggest that localized immune contexts play a role in clonal selection. Evolving LN clones were more common in LNs with relative decreases in CD4⁺ T cell subsets, and the presence of common driver mutations was associated with reductions in dendritic cell content. Together these observations suggest an interplay of supportive vs restrictive roles of the LN TME in CLL progression. These findings highlight the added biological insight provided by LN sampling and suggest that PB alone may not fully capture the clonal heterogeneity of CLL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121dea9d74142004efb29c8ec3326a61ca23f8a3" target='_blank'>
              Role of the immune microenvironment in the clonal evolution of chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            Victoria Rogness, Bharath Dileep Kumar, Chingiz Underbayev, Pia Nierman, Clare Sun, Christine Gruessner, A. Wiestner
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly lethal malignancy with poor prognosis due to its resistance to conventional chemotherapy and limited number of targetable mutations. The recurrent loss-of-function mutation of the tumor suppressor ARID1A represents a key vulnerability potentially exploitable via synthetic lethality. To advance research in this area, we successfully established and characterized two novel CCA cell lines, SiSP-K01 and SiSP-K05, derived from moderately differentiated intrahepatic CCA. Both models were confirmed to carry ARID1A heterozygous loss and complete protein depletion, authentically reproducing the target genotype. Comprehensive molecular and phenotypic profiling (including karyotype, growth kinetics, 100% in vivo tumorigenesis, and tumor marker expression) confirmed their fidelity to the patient tissue and CCA lineage. Importantly, exome and transcriptome analysis not only validated the cell lines but also revealed an associated derangement of the PI3K/AKT signaling pathway. Based on this molecular finding, we performed drug screening and demonstrated that these ARID1A-deficient CCA cell lines are hypersensitive to inhibition by both PI3K/AKT and PARP inhibitors. The SiSP-K01 and SiSP-K05 cell lines are therefore critical new preclinical models suitable for functional studies of ARID1A deficiency and the development of targeted therapies for CCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef0e9dfbc0a68397e20306d980742627a07920fa" target='_blank'>
              Establishment of novel cholangiocarcinoma cell lines with ARID1A deficiency and preclinical validation of synthetic lethality therapies
              </a>
            </td>
          <td>
            Sunisa Prasopporn, G. Sittithumcharee, Jantappapa Chanthercrob, Somchai Limsrichamrern, Arada Hirankitti, Pimkanya More-krong, Sakda Sathirareuangchai, A. Chaiboonchoe, Somponnat Sampattavanich, Seiji Okada, Siwanon Jirawatnotai
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a highly aggressive brain cancer with limited treatment options and poor long-term survival. Adoptive cellular therapy (ACT) has shown promise in targeting GBM, but current strategies often rely on tumor-derived antigens, limiting their applicability in patients with inoperable tumors. Moreover, GBM recurrence is driven by tumor heterogeneity and glioma stem cells (GSCs) that evade standard therapies. Radial glial cells (RGCs), neurodevelopmental progenitors with transcriptional similarity to GSCs, express developmental antigens (DAs) also found in GBM, suggesting their potential as a tumor-independent, renewable antigen source. To explore this, we used single-cell RNA sequencing to profile RGCs from postnatal day 4 murine cortices and identified a subset of antigens shared with tumors from the KR158B GBM model. Using our O.R.A.N. antigen prediction platform, we screened these for immunogenicity and developed an RGC-derived cDNA antigen library for use in ACT. Polyclonal T cells generated from dendritic cells pulsed with RGC RNA showed robust IFNγ and TNFα release in response to cognate antigen and selectively lysed tumor cells in vitro. Further, RGC T cells were significantly better than total tumor RNA (ttRNA) T cells at killing CD133-enriched tumor cells, indicating a preference of RGC T cells for stemness markers. In GBM-bearing mice, ACT using RGC T cells significantly improved survival (p = 0.0058), achieving outcomes comparable to ttRNA ACT, without overt neurotoxicity or systemic inflammation. In parallel human studies, RGC-derived antigens were tested against patient-derived GBM xenografts. Although RGC T cells targeted a smaller fraction of tumor cells compared to ttRNA T cells, they showed enriched recognition of GSC-like populations in 3D bioprinted immune-tumor assays. These findings support the utility of RGCs as a tumor-independent antigen source capable of driving anti-GBM responses. This renewable library enables ACT even in cases where tumor resection is not possible and provides a strategy for targeting treatment-resistant GSC subpopulations early in disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa6bde2a5f253652f6585d9d9a61d9726319ee48" target='_blank'>
              CNSC-02. BORN TO KILL: NEURODEVELOPMENTAL CELLS AS RENEWABLE TARGETS IN T CELL THERAPY FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            C. Love, John Figg, D. Jin, A. Reid, Illeana West, M. Englebart, C. Francis, Jeffery Drake, D. Mitchell, C. Flores
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [13, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>