<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-29 09:30:58 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Aggressive cancers, such as pancreatic ductal adenocarcinoma (PDAC), are often characterized by a complex and desmoplastic tumor microenvironment, a stroma rich supportive connective tissue composed primarily of extracellular matrix (ECM) and non-cancerous cells. Desmoplasia, a dense deposition of stroma, is a major reason for therapy resistance, acting both as a physical barrier that interferes with drug penetration and as a supportive niche that protects cancer cells through diverse mechanisms. Precise understanding of spatial cell interactions in stroma-rich tumors is essential for optimizing therapeutic responses. It enables detailed mapping of stromal-tumor interfaces, comprehensive cell phenotyping, and insights into changes in tissue architecture, improving assessment of drug responses. Recent advances in multiplexed immunofluorescence imaging have enabled the acquisition of large batches of whole-slide tumor images, but scalable and reproducible methods to analyze the spatial distribution of cell states relative to stromal regions remain limited. To address this gap, we developed an open-source computational pipeline that integrates QuPath, StarDist, and custom Python scripts to quantify biomarker expression at a single- and sub-cellular resolution across entire tumor sections. Our workflow includes: (i) automated nuclei segmentation using StarDist, (ii) machine learning-based cell classification using multiplexed marker expression, (iii) modeling of stromal regions based on fibronectin staining, (iv) sensitivity analyses on classification thresholds to ensure robustness across heterogeneous datasets, and (v) distance-based quantification of the proximity of each cell to the stromal border. To improve consistency across slides with variable staining intensities, we introduce a statistical strategy that translates classification thresholds by propagating a chosen reference percentile across the distribution of marker-related cell measurement in each image. We apply this approach to quantify spatial patterns of distribution of the phosphorylated form of the N-Myc downregulated gene 1 (NDRG1), a novel DNA repair protein that conveys signals from the ECM to the nucleus to maintain replication fork homeostasis, and a known cell proliferation marker Ki67 in fibronectin-defined stromal regions in PDAC xenografts. The pipeline is applicable for the analysis of markers of interest in stroma-rich tissues and is publicly available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b6327fb55ba3776af18a8a1265a4ec0c3c69a0b" target='_blank'>
              An image analysis pipeline to quantify the spatial distribution of cell markers in stroma-rich tumors
              </a>
            </td>
          <td>
            Antoine A. Ruzette, Nina Kozlova, Kayla A. Cruz, Taru Muranen, S. Nørrelykke
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer immunotherapy has revolutionized clinical oncology; however, the inherent complexity and heterogeneity of cancer present substantial challenges to achieving broad therapeutic efficacy. Tumor heterogeneity manifests not only among different patients but also within individual tumors, further complicating personalized treatment approaches. Single-cell sequencing technologies encompassing genomics, transcriptomics, epigenomics, proteomics, and spatial omics have significantly enhanced our ability to dissect tumor heterogeneity at single-cell resolution with multi-layered depth. These approaches have illuminated tumor biology, immune escape mechanisms, treatment resistance, and patient-specific immune response mechanisms, thereby substantially advancing precision oncology strategies. This review systematically examines recent advances in single-cell multi-omics technologies across various cancer research areas, emphasizing their transformative impacts on understanding tumor heterogeneity, immunotherapy, minimal residual disease monitoring, and neoantigen discovery. Additionally, we discuss current technical and analytical limitations and unresolved questions associated with single-cell technologies. We anticipate single-cell multi-omics technologies will become central to precision oncology, facilitating truly personalized therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3005de124c813e8dd9ae4675ad8451e32f8f1a34" target='_blank'>
              Single-cell multi-omics in cancer immunotherapy: from tumor heterogeneity to personalized precision treatment
              </a>
            </td>
          <td>
            Jiayuan Le, Yating Dian, Deze Zhao, Ziyu Guo, Zehao Luo, Xiang Chen, Furong Zeng, Guangtong Deng
          </td>
          <td>2025-08-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b1a84299cf4bd86e16928f5b0911b15d09c6d9" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z. Lin, Dylan Schaff, Sydney M. Shaffer, Drew M Pardoll, Christina Jackson, Nancy R. Zhang
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Summary Intratumoral heterogeneity, originating from genetic, epigenetic, and phenotypic cellular diversity, is pervasive in cancer. As these heterogeneous states employ diverse mechanisms to promote tumor progression, metastasis, and therapy resistance, emergence of cancer heterogeneity is one of the most significant barriers to curative treatment. Here we leverage deep learning approaches to develop a high-throughput image-analysis paradigm with subcellular resolution that quantifies and predicts colorectal cancer (CRC) patient outcome based on tissue architecture and nuclear morphology. We further combine this approach with spatial transcriptomics, multiplex immunohistochemistry, and patient-derived organoids to uncover a dynamic, co-evolutionary relationship between tumor architecture and cell states. We identify clinically relevant architectural interfaces in CRC tissue that diversify cellular identities by favoring distinct cancer stem cell states and thus promote evolution of tumor heterogeneity. Specifically, tissue fragmentation and associated compressive forces promote loss of classic stem cell signature and acquisition of fetal/regenerative stem cell states, which initially emerges as a hybrid state with features of epithelial-to-mesenchyme (EMT) transition. Additional tumor stroma communication then diversifies these states into distinct stem cell and EMT states at the invasive margins of tumors. Reciprocally, Wnt signaling state of tumor cells tunes their responsiveness to tissue architecture. Collectively, this work uncovers a feedback loop between cell states and tissue architecture that drives cancer heterogeneity and cell state diversification. Machine learning-based analyses harness this co-dependency, independently of mutational status or cancer stage, to predict patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8d83b8caf4ac682c803318ebb577d7836c261e" target='_blank'>
              Colorectal cancer heterogeneity co-evolves with tumor architecture to determine disease outcome
              </a>
            </td>
          <td>
            Fabien Bertillot, Solène Hervé, Nelly Drobjazko, Noemi Kedei, Mona El Gendi, Maria O Hernandez, Kai Kruse, E. Erkan, Mirjam I. Binner, Karolina Punovuori, T. Pellinen, M. Ahtiainen, Jan Böhm, J. Mecklin, Caj Haglund, Jaana Hagström, Mike Nikolaev, N. Gjorevski, Matthias Lütolf, Toni T Seppälä, Sara A. Wickström, Yekaterina A. Miroshnikova
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Intrinsic genetic alterations and dynamic transcriptional changes contribute to the heterogeneity of solid tumors. Lung adenocarcinoma (LUAD) is characterized by its significant histological, cellular and molecular heterogeneity. The present study aimed to study the spatial transcriptomics of primary LUAD with initial hopes to decipher molecular characteristics of subtype transitions in LUAD progression, offering new insights for novel therapeutic strategies. Spatial transcriptomics libraries were first generated from tumor samples collected from patients with LUAD who underwent surgical resection in The Fourth Hospital of Hebei Medical University in 2022 and were sequenced using Illumina NovaSeq 6000 system. The processed data were analyzed for differential gene expressions and networks, and were annotated according to cell type, spatial ligand-receptor interaction and trajectory inference. Our analysis revealed 34 annotated cell types, with cancer-associated fibroblasts (CAFs) being the most abundant, playing a crucial role in tumor microenvironment remodeling and prognosis. We noted significant spatial correlations between various immune cells and found that different histological subtypes displayed unique cell composition profiles, particularly in the micropapillary subtype, which exhibited higher macrophage proportions and distinct gene expression pathways related to extracellular matrix organization and receptor tyrosine kinase signaling. Additionally, we explored the dedifferentiation states within these subtypes, identifying that region with higher dedifferentiation scores corresponded to increased tumor invasiveness and potential drug resistance. Our findings demonstrate dynamic biological changes and dedifferentiation states of tumor subtypes during the progression process. This study reveals important biological processes in tumor development and may offer valuable guidance for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a252a3c50c2377c8fca1733740eaa8f4897e893" target='_blank'>
              Spatial transcriptomics uncovers immune-cell plasticity and dedifferentiation signatures in aggressive lung adenocarcinoma subtypes
              </a>
            </td>
          <td>
            Huiyan Deng, Qingyi Liu, Ziqiang Tian, Guan-Yuan Liu, Shize Wang, Di Liang, Yun Wang, Yaqing Han, Shaonan Xie
          </td>
          <td>2025-08-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cell plasticity, defined as the ability of tumor cells to reversibly adopt distinct functional states, plays a central role in tumor heterogeneity, therapy resistance, and disease relapse. This process enables cells to enter stem-like, dormant, or drug-tolerant persister states in response to treatment or environmental stress without undergoing genetic changes. Such reversible transitions complicate and limit current treatments. Conventional cancer models often fail to capture the complexities of these adaptive states. In contrast, patient-derived tumor organoids (PDOs), which retain the cellular diversity and structure of primary tumors, provide a unique system for investigating plasticity. This review describes how PDOs can model cellular plasticity, such as the emergence of drug-tolerant persister cells and the interconversion between cancer stem cell states across multiple tumor types. We particularly focused on colorectal cancer organoids, for which research on the mechanism of plasticity is the most advanced. Combined with single-cell analysis, lineage tracing, and functional assays, PDOs can help identify the molecular pathways that control plasticity. Understanding these mechanisms is important for developing therapies to prevent treatment failure and control disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd4e7e497e61cac197e5ff80d3e09c68cd18937b" target='_blank'>
              Patient-Derived Tumor Organoids to Model Cancer Cell Plasticity and Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            R. Coppo, Masahiro Inoue
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, Gregory Schuyler Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a molecularly diverse form of lung cancer characterized by distinct oncogenic driver mutations that influence both tumour biology and clinical outcomes. Understanding the interplay between these oncogenic drivers and the tumour microenvironment (TME) is crucial for improving therapeutic strategies and patient management. Here, we investigate the impact of driver mutations on the composition and spatial architecture of the TME in LUAD. Using imaging mass cytometry (IMC), we analyse tumour samples from 157 LUAD patients, integrating genomic and clinical data to link specific mutations with tumour characteristics. Unique patterns are associated with mutated KRAS and EGFR tumours with TP53 co-mutations, suggesting these co-mutations reshape the TME and promote resistance to tyrosine kinase inhibitors (TKIs). Overall, our findings highlight the complex interplay between oncogenic driver mutations and the TME in LUAD, underscoring the importance of integrating genomic and cellular data to understand the underlying tumour behaviour and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05eac26faa3f5fa852d857cbc84d4c9c3228ae87" target='_blank'>
              Oncogenic driver mutations underlie the spatial tumour immune landscape of non-small cell lung cancer
              </a>
            </td>
          <td>
            Saskia Hartner, H. Abolfathi, M. Rezanejad, Bridget Liu, E. Karimi, Dakota Rogers, Mark Sorin, Samuel Doré, Lysanne Desharnais, M. Orain, William Enlow, A. Gagné, Yuhong Wei, Y. Bossé, D. Quail, Philippe Joubert, L. Walsh
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Simple Summary Spatial transcriptomics is an advanced technology that allows for the study of gene activity in cancer tissues while preserving the original tissue structure. Unlike traditional methods that either analyze samples without location information or examine individual cells removed from their natural environment, spatial transcriptomic technology maps where genes are active within intact tumor tissue. This approach enables a greater understanding of how different areas of a tumor vary, how cancer cells interact with surrounding healthy tissue, and how cells are organized spatially within the tumor at high resolution. As spatial technology becomes more widely used, it may improve cancer diagnosis and treatment by helping to create more personalized and effective therapies for individual patients. This review outlines the current and future role of spatial transcriptomics methodologies in the context of genitourinary cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa8ee089aa47992263f8b69f0d5705b49d03f797" target='_blank'>
              Current Role and Future Frontiers of Spatial Transcriptomics in Genitourinary Cancers
              </a>
            </td>
          <td>
            Firas Hatoum, Adnan Fazili, Justin W Miller, Xuefeng Wang, Xiaoqing Yu, Xin Lu, Jeffrey S. Johnson, Philippe E. Spiess, J. Chahoud
          </td>
          <td>2025-08-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Xenium, a new spatial transcriptomics platform, enables subcellular-resolution profiling of complex tumor tissues. Despite the rich morphological information in histology images, extracting robust cell-level features and integrating them with spatial transcriptomics data remains a critical challenge. We introduce CellSymphony, a flexible multimodal framework that leverages foundation model-derived embeddings from both Xenium transcriptomic profiles and histology images at true single-cell resolution. By learning joint representations that fuse spatial gene expression with morphological context, CellSymphony achieves accurate cell type annotation and uncovers distinct microenvironmental niches across three cancer types. This work highlights the potential of foundation models and multimodal fusion for deciphering the physiological and phenotypic orchestration of cells within complex tissue ecosystems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0df7c5843203625e193cd9c5f8b65c48433cc0" target='_blank'>
              CellSymphony: Deciphering the molecular and phenotypic orchestration of cells with single-cell pathomics
              </a>
            </td>
          <td>
            Paul H. Acosta, Pingjun Chen, Simon P. Castillo, M. E. Salvatierra, Yinyin Yuan, Xiaoxi Pan
          </td>
          <td>2025-08-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cellular anatomy and signaling vary across niches, which can induce gradated gene expressions in subpopulations of cells. Such spatial transcriptomic gradient (STG) makes a significant source of intra-tumor heterogeneity. We present Local Spatial Gradient Inference (LSGI), a computational framework that systematically identifies spatial locations with prominent, interpretable STGs from spatial transcriptomic (ST) data. We demonstrate LSGI in tumor ST datasets and identify pan-cancer and tumor-type specific pathways with gradated patterns, highlighting the ones related to spatial transcriptional intratumoral heterogeneity. LSGI enables interpretable STG analysis, which can reveal novel insights in tumor biology from the increasingly reported tumor ST datasets. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03716-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c2919ee7022acd5a65c198642bf710ee21f0432" target='_blank'>
              LSGI: interpretable spatial gradient analysis for spatial transcriptomics data
              </a>
            </td>
          <td>
            Qingnan Liang, Luisa M. Solis Soto, C. Haymaker, Ken Chen
          </td>
          <td>2025-08-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="ABSTRACT Objective To illustrate a new barcoded orthotopic patient‐derived xenograft (PDX) mouse model where one can investigate phenotypic effects of single‐cell level gene manipulation in a pooled format. To address some concerns of current PDX mouse models of head and neck squamous cell carcinoma (HNSCC): (1) genomic evolution with passage by generating high‐purity cancer cells, which can also be utilized for other downstream applications, including cell culture‐based studies, and (2) cost‐effectiveness of current PDX models. Methods Two‐millimeter tumor cubes from nine patients were implanted into immunodeficient mouse flanks subcutaneously. Purified tumor cells were obtained from subcutaneous xenografts. Various numbers of purified tumor cells were then injected into the lingual tissue of immunodeficient mice, and the lowest amount of cells needed to achieve a 100% orthotopic engraftment rate were identified. Clonal stability was tested using a lentiviral barcoding system. The orthotopic PDXs' genetic landscapes were characterized using whole exome sequencing. Results This approach yielded an overall engraftment rate of 88.9%. The purification process increased cancer cell purity from 34% to 92%. Lingual injection of 100,000 purified tumor cells achieved a 100% orthotopic engraftment rate from purified subcutaneous PDX tumor cells while maintaining clonal and genetic stability. Conclusion Our study presents a barcoded orthotopic patient‐derived xenograft model for head and neck squamous cell carcinoma with clonal stability. This model provides a way to study phenotypic effects of single cell level gene manipulation in a pooled format. The method can be adapted for in vitro work as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/770f65963b9ed4389a6e68906eb989805afcd015" target='_blank'>
              Barcoded Orthotopic Patient‐Derived Head & Neck Squamous Cell Carcinoma Model Demonstrating Clonal Stability and Maintenance of Cancer Driver Mutational Landscape
              </a>
            </td>
          <td>
            Peiran Zhou, Claire B Mills, Zhao Ming Dong, Brittany R Barber, Slobodan Beronja
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite advances in cancer therapies, treatment responses remain highly variable due to the complexity and heterogeneity of the tumor microenvironment. The tumor microenvironment comprises malignant, immune, stromal, and endothelial cells, along with extracellular matrix and soluble factors, all organized into spatially distinct communities that evolve dynamically throughout tumor progression and therapy. These spatial structures orchestrate tumor behavior, immune evasion, and drug resistance. Recent breakthroughs in spatial omics technologies, including spatial transcriptomics and spatial proteomics, have enabled high-resolution, multiplexed mapping of tissue architecture and molecular characteristics. These technologies provide valuable insights into how the spatial organization of cells and signaling networks within the tumor microenvironment influences therapeutic efficacy. Notably, specific structures, such as tertiary lymphoid structures, fibroblast-mediated stromal barriers, and vascular heterogeneity have been identified as spatial determinants of treatment response. By delineating cellular communities and their interactions, spatial omics technologies can reduce intratumoral complexity into clinically interpretable modules. This review summarizes the diversity of these spatial structures and their relationships with treatment outcomes in immunotherapy, chemotherapy, radiotherapy, and targeted therapy. In addition, it highlights present challenges in data integration, analytical standardization, and functional validation, and discusses future directions for incorporating spatial omics technologies into precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7572818b2484643c196593baace3df887c52839e" target='_blank'>
              Structural cell communities in the tumor microenvironment: Spatial determinants of therapeutic response
              </a>
            </td>
          <td>
            Xuan Cui, Y. Ni, Xia Lei, Lan Zhao, Cheng Qian, Juanjuan Shan
          </td>
          <td>2025-08-14</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Owing to the complexity of TME components and the heterogeneity of cancer cells, the relationship between the niches of TME and prognosis in breast ductal carcinoma remains unknown. The staged characteristics of corresponding cancer cell behaviors are unclear. Our study aims to reveal spatial structures and specific cellular information of TME and cancer cells subgroups during the progression from DCIS to IDC and lymph node metastasis. Methods Single-cell sequencing, spatial transcriptomics, bulk RNA sequencing datasets were used to explore the changes in microenvironmental components and transcriptional programs of tumor cells during the progression of breast ductal carcinoma. Immunohistochemistry, multiplex immunofluorescence, flow cytometry cell cycle detection, invasion migration experiments, and WB imprinting were employed for validation. Results Analysis of TME cell type subsets revealed the accumulation of TEX, iTreg, and stress-phenotype TAM in the mammary gland in situ during the invasion process. Lymphatic metastases exhibited enrichment of nTregs and a more naïve-like CD8 T cell population. Spatial analysis and survival analysis showed that the spatial niches of CD4 TN and phagocytic-phenotype macrophages were associated with a favorable prognosis, and these niches were lost during disease progression. The proliferative subpopulation of breast ductal carcinoma was enriched in lymphatic metastatic tissues, expressing high levels of FAM111B and exhibiting intense TCA and oxidative phosphorylation metabolism. Silencing FAM111B led to cell cycle arrest, decreased invasion and migration abilities, and downregulation of core mediator genes for cuproptosis and disulfidptosis. Conclusions The stage-specific microenvironmental characteristics of breast ductal carcinoma correspond to some extent to the behavior of tumor cells. During the progression of ductal carcinoma in breast tissue, the establishment of an immunosuppressive microenvironment occurs. The microenvironmental spectrum at lymph node metastases differs somewhat, corresponding to a more enriched turnover of cancer cell proliferation and death. Inhibitors of FAM111B and inducers of cuproptosis and disulfidptosis may serve as potential therapeutic targets for proliferative subgroups. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07010-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfce9a8ff47f521350ed382190ba1004f756bb5c" target='_blank'>
              Spatiotemporal microenvironment landscape and malignant epithelial pattern transition in breast ductal carcinoma progression
              </a>
            </td>
          <td>
            Xifu Cheng, Wenjuan Zeng, Bingzhe Yin, Jiawei Gui, Hengbin Zhang, Zhenxing Lv, Simin Zhang, Yao Zhou
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background The presence of tertiary lymphoid structures (TLS) in solid tumors, including Merkel cell carcinoma (MCC), is associated with a better prognosis and a better response to immunotherapy with immune checkpoint inhibition (ICI). The detailed mechanisms by which TLS influence antitumor immune responses are only partially understood. Methods Clinically annotated tumor tissues of 27 patients with MCC were obtained prior to ICI therapy. Tumor samples were subjected to transcriptomic and multiplex immuno-visual profiling, T-cell receptor (TCR) clonotype mapping, as well as—in selected cases—spatial transcriptomics to comprehensively characterize the tumor immune microenvironment. Results Weighted gene co-expression network analysis (WGCNA) of transcriptomic data in combination with topological overlap measures indicated a higher abundance of TLS in tumors of patients with MCC responding to ICI therapy. This concept was substantiated through immunomorphological analyses, revealing mature B-cell follicle-like structures characterized by high endothelial venules (HEVs). Further supporting HEVs as critical entry points for naïve T cells, the presence of TLS was correlated with a pronounced infiltration of CD4+ and CD8+ T cells, exhibiting both naïve and central memory phenotypes. The TCR repertoire of these infiltrates exhibited enhanced richness and diversity with a pronounced reactivity toward Merkel cell polyomavirus-derived T-cell epitopes. Spatially resolved RNA and V(D)J sequencing revealed the expression of genes associated with T-cell recruitment within TLS, alongside the presence of naïve and central memory T-cell markers. Notably, individual clonally expanded TCR transcripts were detected both within TLS and among tumor-infiltrating lymphocytes. The latter were associated with low expression of memory cell markers and high expression of effector cell markers. Additionally, a spatial gradient in the expression of genes linked to immune stress in MCC cells—such as those involved in the interferon-γ response and antigen processing and presentation machinery—originated in proximity to the TLS. Conclusion Our findings are consistent with a key role of TLS in shaping immune interactions within the MCC microenvironment, driving the recruitment of diverse tumor-reactive T cells. These insights hold promise for advancing immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13de7f3d541927495612436f916863e4107c4925" target='_blank'>
              Tertiary lymphoid structures in Merkel cell carcinoma facilitate naïve and central memory T-cell infiltration linked to immunotherapy response
              </a>
            </td>
          <td>
            Nalini Srinivas, Ivelina Spassova, Kuan Cheok Lei, Jiwei Gao, María José Pino, Giovanni Giglio, S. Kitanovski, Mazdak Dalkoohi, E. Livingstone, Ulrike Leiter, Peter Mohr, T. Gambichler, Ingo Stoffels, Selma Ugurel, Phyllis Fung-Yi Cheung, Camilla Engblom, Weng-Onn Lui, Jürgen C. Becker
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality, with tumor heterogeneity and drug resistance posing significant challenges to treatment. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq deconvolution to analyze BRCA samples. Our analysis identified 15 major cell clusters, including neoplastic epithelial, immune, stromal, and endothelial populations. Notably, low-grade tumors showed enriched subtypes, such as CXCR4+ fibroblasts, IGKC+ myeloid cells, and CLU+ endothelial cells, with distinct spatial localization and immune-modulatory functions. These subtypes were paradoxically linked to reduced immunotherapy responsiveness, despite their association with favorable clinical features. High-grade tumors exhibited reprogrammed intercellular communication, with expanded MDK and Galectin signaling. Bulk RNA-seq deconvolution further supported the prognostic significance of low-grade-enriched subtypes. Our findings highlight the heterogeneity of the tumor microenvironment and provide new insights into immune evasion and therapeutic resistance in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57925849e8c34466fe3132e9855985812e402540" target='_blank'>
              Machine learning analysis reveals tumor heterogeneity and stromal-immune niches in breast cancer
              </a>
            </td>
          <td>
            Junjie Kuang, Guofang Zhong, Linfeng Zhao, Xia Yuan, Yundong Zhou, Jun Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Current cancer therapies often fail due to tumor heterogeneity and rapid resistance evolution. A new evolutionary framework, ‘selection for function,’ proposes that tumor progression is driven by group phenotypic composition (GPC) and its interaction with the microenvironment, not by individual cell traits. This perspective opens new therapeutic avenues: targeting the tumor’s functional networks rather than individual cells. Real-time tracking of GPC changes could inform adaptive treatments, delaying progression and resistance. By integrating evolutionary and ecological principles with conventional therapies, this strategy aims to transform cancer from a fatal to a manageable chronic disease. Crucially, it does not necessarily require new drugs but offers a way to repurpose existing therapies to impair a tumor’s evolutionary potential. By steering tumor evolution toward less aggressive states, this approach could improve prognosis and long-term patient survival compared to current methods. We argue that leveraging GPC dynamics represents a critical, yet underexplored, opportunity in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53fd3b887f1a92efe08b517e2a13e55225513a1" target='_blank'>
              Leveraging selection for function in tumor evolution: System-level cancer therapies
              </a>
            </td>
          <td>
            F. Thomas, Jean-Pascal Capp, A. Dujon, A. Marusyk, K. Asselin, M. Campone, Pascal Pujol, Catherine Alix-Panabières, Benjamin Roche, B. Újvári, Robert Gatenby, A. Nedelcu
          </td>
          <td>2025-08-18</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Purpose Microsatellite instability-high colorectal cancer is characterized by hypermutated genomes and high neoantigen loads, yet a significant proportion of patients exhibit resistance to immune checkpoint blockade. This study aims to investigate tumor cell functional heterogeneity and its role in immune evasion. Patients and Methods We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA sequencing from microsatellite instability-high colorectal cancer patients. After quality control, normalization, and clustering, malignant epithelial subpopulations were identified through copy number variation analysis and non-negative matrix factorization. Functional characterization employed gene set enrichment analysis. Spatial transcriptomics clarified immune cell and tumor subpopulation localization, and survival analyses assessed prognostic implications. Results We identified a glycolysis-enriched tumor subpopulation (MP2) that co-localized with immunosuppressive niches marked by Treg accumulation, effector T-cell depletion, and FOLR2+ tumor-associated macrophages. MP2-high tumors were associated with immune checkpoint blockade resistance and poor prognosis. Mechanistically, MP2 cells secreted lactate, promoting Treg differentiation and macrophage polarization toward an immunosuppressive phenotype. Spatial transcriptomics revealed the precise organization of these lactate-rich, immune-excluded niches within tumors. Conclusion These findings establish tumor cell-intrinsic glycolysis as a key driver of immune evasion in microsatellite instability-high colorectal cancer and propose metabolic targeting as a strategy to overcome immune checkpoint blockade resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773cdb1598a2f7975562a7cc2f1fda279ad89676" target='_blank'>
              Glycolysis-Driven Immune Evasion in Microsatellite Instability-High Colorectal Cancer: An Integrated Single-Cell and Spatial Transcriptomics Study
              </a>
            </td>
          <td>
            Chenchen Li, Peicong Cai, Hengda Zeng, Jianxia Li, Huabin Hu, Jianwei Zhang, Zehua Wu, Ge Qin, Yanhong Deng
          </td>
          <td>2025-09-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract High‐risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single‐cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this, we profiled untreated and chemotherapy‐treated high‐risk neuroblastomas from archived, formalin‐fixed, paraffin‐embedded (FFPE) tissues from two patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p‐gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical‐like cells expressed several oncogenic ligand‐encoding genes (e.g. ALKAL2 and NRTN), which were predicted to communicate with neighboring cancer cells that expressed the corresponding receptors (e.g. ALK, RET). Several of these interactions were further validated experimentally and were shown to be clinically relevant. Collectively, our spatial analysis identifies multiple previously unrecognized signaling axes that may offer novel therapeutic options in neuroblastoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce7c8d7d28d1cfa3f77ec4f74044ee737c8a1da" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment in archived FFPE samples unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlsson, A. Claeys, E. Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2025-08-08</td>
          <td>The Journal of Pathology</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Background Cutaneous squamous cell carcinoma (cSCC) and basal cell carcinoma (BCC) are the most prevalent types of nonmelanoma skin cancer (NMSC) and exhibit significant inter‐ and intra‐tumor heterogeneity. cSCC has a higher metastatic potential than BCC, accompanied by a considerable mortality rate. However, the detailed mechanisms of tumor evolution in cSCC have not yet been described. Methods We performed single‐cell RNA sequencing (scRNA‐seq) and T cell receptor (TCR) clonal analysis of skin biopsies from five BCCs, three squamous cell carcinomas in situ (SCCIS), and two invasive squamous cell carcinomas (SCC). Independent SCC specimens were used for spatial transcriptomic (ST) analysis using GeoMx Digital Spatial Profiler (DSP). Result Using scRNA‐seq, we analyzed a total of 117,663 cells. We distinguished cancer cells using copy number variation and identified SCC‐specific genes that potentially contribute to tumor progression. Analysis of tumor clones revealed SCC‐specific COL6A1+/ITGA5+ carcinoma cells which produce CXCL16. We also annotated CXCR6+ regulatory T cells (Tregs) which potentially move toward the tumor site by CXCL16, shaping the immunosuppressive TME. ST analysis supported these clones were located at the invasion site of SCC. Conclusion We suggest COL6A1 and ITGA5 promote the invasive and metastatic property of SCC. We also uncovered how SCC recruits Tregs via the CXCL16/CXCR6 axis to create a TME favorable for its survival. These molecules can be used as potential therapeutic targets for treatment of SCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10a0e1cd92069735154742961ad751b08397d911" target='_blank'>
              CXCL16 Producing Tumor Clones Are Shaping Immunosuppressive Microenvironment in Squamous Cell Carcinoma via CXCR6 Regulatory T Cell
              </a>
            </td>
          <td>
            Hyun Seung Choi, Sunyoung Jung, Ki-Myo Kim, Mihyun Lee, Jun Ho Park, Sanha Hwang, Seung-Min Cha, Jade N. Young, D. Poplausky, Hyunsung Nam, Nicholas Gulati, Chung-Gyu Park, Hyun Je Kim, Ji-Ung Park
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease with diverse outcomes, and intra-tumoral heterogeneity plays a significant role in both diagnosis and treatment. Despite its importance, the spatial distribution of intra-tumoral heterogeneity is not fully elucidated. Spatial transcriptomics has emerged as a promising tool to study the molecular mechanisms behind many diseases. It offers accurate measurements of RNA abundance, providing powerful tools to correlate the morphologies of cellular neighborhoods with localized gene expression patterns. However, the spot-based spatial transcriptomic tools, including the most widely used platform, Visium, do not achieve single-cell resolution readouts, which hinders data interpretability. In this study, we present a computational pathology image analysis pipeline (i.e., computational tissue annotation, CTA) that utilizes machine learning algorithms to accurately map tumor, stroma, and immune compartments within Visium-assayed tumor sections. Using a cohort of 23 breast tumor sections from four patients, we demonstrate that CTA can provide high-resolution annotations on the hematoxylin-and-eosin-stained images alongside the paired sequencing data, support the evaluation of deconvolution methods, deepen insights into intra-tumoral heterogeneity by increasing data analysis resolution, assist with spatially resolved intrinsic subtyping, and enhance the visualization of lymphocyte clones at single-cell resolution. The proposed pipeline provides valuable insights into the complex spatial architecture of breast cancer, contributing to more personalized diagnostics and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5fa100fe1b095f296661cac47143cc93bae15b" target='_blank'>
              Computational pathology annotation enhances the resolution and interpretation of breast cancer spatial transcriptomics data
              </a>
            </td>
          <td>
            Tianyi Li, Qiao Yang, B. Ács, E. Sifakis, Hosein Toosi, Camilla Engblom, K. Thrane, Qirong Lin, Jeff E Mold, Wenwen Sun, Ceren Boyaci, Sanna Steen, Jonas Frisén, Jens Lagergren, J. Lundeberg, Xinsong Chen, Johan Hartman
          </td>
          <td>2025-09-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Tumor heterogeneity in metastatic prostate cancer (mPC) is well established, but comprehensive characterization using routine sampling remains challenging. Autopsy-based research addresses this obstacle by enabling broad tissue collection within individual patients after treatment. In this issue of the JCI, Roudier et al. analyzed samples from a mPC research autopsy cohort, revealing extensive inter- and intratumor heterogeneity across patients and at the cellular level. The authors associated this variability with genomic, phenotypic, and clinical features and explored the importance of tumors expressing both androgen receptor and neuroendocrine markers. Their findings demonstrate heterogeneity across metastatic sites that may influence treatment response and clinical outcomes, informing future therapeutic strategies in mPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1162fb7a103df269563289c6dd21cc5f28aaf11" target='_blank'>
              Uncovering phenotypic heterogeneity through research autopsy in lethal prostate cancer
              </a>
            </td>
          <td>
            Sylvie S.W. Chan, Osvaldas Vainauskas, Gerhardt Attard
          </td>
          <td>2025-08-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The organizational principles of nephronal segments are based on anatomical and physiological attributes that are linked to the homeostatic functions of the kidney. Recent molecular approaches have uncovered layers of deeper signatures and states in tubular cells that arise at various time points on the disease trajectory. Here, we introduce an analytical pipeline of multiplexed spatial protein imaging integrated with RNA expression to characterize proximal tubular subpopulations and neighborhoods in human kidney tissue. We demonstrate that, in reference tissue, a large proportion of S1 proximal tubular epithelial cells expresses thymus antigen 1 (THY1), a mesenchymal stromal and stem cell marker that regulates differentiation. Kidney disease is associated with loss of THY1 and transition toward expression of prominin 1 (PROM1), another stem cell marker recently linked to failed repair. Our data support a model in which the interplay between THY1 and PROM1 expression in proximal tubules associates with their regenerative potential and marks the timeline of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1923d134d598f27cb8990ba3881ccadfe262fa12" target='_blank'>
              Integration of spatial protein imaging and transcriptomics in the human kidney tracks the regenerative potential of proximal tubules
              </a>
            </td>
          <td>
            Mahla Asghari, Angela R. Sabo, Daria Barwinska, Ricardo Melo Ferreira, M. Ferkowicz, William S. Bowen, Ying-Hua Cheng, D. Gisch, Connor J. Gulbronson, Carrie L. Phillips, Katherine J. Kelly, T. Sutton, James C Williams, Miguel A Vazquez, John O'Toole, Paul M. Palevsky, Sylvia E. Rosas, S. Waikar, Krzysztof Kiryluk, C. Parikh, Jeffrey B. Hodgin, Pinaki Sarder, Ian H. de Boer, Jonathan Himmelfarb, Matthias Kretzler, Sanjay Jain, Michael T. Eadon, S. Winfree, T. El-Achkar, P. Dagher
          </td>
          <td>2025-08-15</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is characterized by exceedingly high rates of metastatic progression, with the liver representing the most common site of distant spread. Here, we established a platform for multisite immune profiling of human PDAC encompassing the tumor, peripheral circulation, and premetastatic liver, to more comprehensively study how various immune subsets might contribute to patient outcomes. Methods Tumor, liver, and blood samples were obtained from patients undergoing resection for non-metastatic PDAC. Derived immune cells underwent paired single-cell RNA and TCR sequencing. Immune composition, cell-type functional profiles, and T cell clonal expansion patterns were evaluated across tissue sites. Results In total, 106,539 immune cells were sequenced, of which 85,748 met criteria for analysis. We identified 32 cell populations, of which seven demonstrated significant enrichment within a particular tissue, highlighting that this workflow possesses the granularity needed for identifying potential future biomarkers. Functional profiling revealed tissue-specific differences in cell phenotypes. This included terminally differentiated exhausted CD8 T cells within the tumor, highly active Tregs within the premetastatic liver and tumor, and M1 versus M2 polarization of liver and tumor macrophage populations, respectively. Within the tumor, expanded Treg clones were uniquely abundant, and while expanded clones could be tracked to the blood and premetastatic liver, many of these mapped back to known viral antigens. Leveraging previously validated gene sets, we show how these can be applied to predict the tumor reactivity of intratumoral T cells using transcriptional signatures. We demonstrated a high degree of concordance between multiple independent signatures and tracked high-priority TCRs within the blood and liver. Conclusion This study demonstrates the feasibility of a platform, which has already been implemented into ongoing clinical protocols, for immune profiling of human PDAC across the sites most relevant to metastatic progression. Future applications of this work can monitor immune populations throughout metastatic progression to build a temporal database of immune phenotypes and track association with clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04146-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9fc3615c15a691adb382d776e1bcc8d09f65d7" target='_blank'>
              A platform for multisite immune profiling of premetastatic pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Elishama N. Kanu, Ashley Fletcher, Jiayin Bao, Ethan S Agritelley, Julia Button, A.M. Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, M. Lidsky, S. Zani, D. Blazer, Peter J Allen, Zhicheng Ji, Frank J. Lowery, Sri Krishna, Nicholas D. Klemen, Daniel P. Nussbaum, Erika J. Crosby
          </td>
          <td>2025-08-23</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9fa61bcf35b5d6c234bc5ce442e119522c741a" target='_blank'>
              Basal cell of origin resolves neuroendocrine-tuft lineage plasticity in cancer.
              </a>
            </td>
          <td>
            A. Ireland, Daniel A Xie, Sarah B Hawgood, Margaret W Barbier, Lisa Y. Zuo, Benjamin E Hanna, Scarlett Lucas-Randolph, Darren R Tyson, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Anish Thomas, Sonam Puri, C. Rudin, Joseph M Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2025-09-17</td>
          <td>Nature</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="High-grade serous carcinoma (HGSC) is the most common ovarian cancer subtype, typically diagnosed at late stages with poor prognosis. Understanding early molecular events driving HGSC progression is crucial for timely detection and development of effective treatment strategies. We performed and integrated spatial cell-type resolved proteomics and paired transcriptomics across 25 women with precursor lesions of the fallopian tube and/or HGSC. Epithelial cell signatures revealed early activation of SUMOylation machinery, increased ATR and Wnt signaling, and enhanced MHC-I antigen presentation along the disease trajectory. The stroma exhibited extracellular matrix remodeling and interferon-mediated inflammation. Serous tubal intraepithelial carcinomas (STICs) in cancer patients contained a pro-coagulative signature and reduced APOA1/2 compared to STICs in individuals without cancer. We functionally established important roles of epithelial-derived TRIP13 and SUMOylation, and cancer-associated fibroblast-derived SULF1 and BGN in HGSC progression. These findings provide unique molecular insights into HGSC pathogenesis and identify potential new therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44488614a89cae7d4f29ffc136362eb7dd32e0db" target='_blank'>
              Integration of cell-type resolved spatial proteomics and transcriptomics reveals novel mechanisms in early ovarian cancer
              </a>
            </td>
          <td>
            Andreas Metousis, Hilary A Kenny, Aasa Shimizu, L. Schweizer, S. Ben-Moshe, A. Bilecz, Rahul Krishnan, Jingwen Zhang, Isabel Alcazar, Lucy Kelliher, Mallika Ravi, Tejas Samantaray, Sabrina Richter, Yan Li, Jiying Wang, Sophia Steigerwald, F. Theis, Florian A. Rosenberger, T. Nordmann, S. Yamada, Ricardo R. Lastra, M. Mann, E. Lengyel
          </td>
          <td>2025-08-28</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third most common cancer in the world. Within the tumor microenvironment (TME), cancer cells are surrounded by diverse cell types including cancer-associated fibroblasts (CAFs) and immune cells. These neighboring cells can have a profound effect on prognosis and disease progression by providing local signals that impact tumor growth and metastasis. Prior work in the lab has identified via single cell RNA sequencing novel subtypes of cancer epithelial cells and enrichment of specific CAFs within the CRC TME. However, the locations and signaling interactions of these cells within the tumors remain poorly understood due to the loss of spatial information. In this study, we adopt a multi-facet approach to try to understand the CRC TME by utilizing a combination of single cell RNA sequencing and spatial omics technology. We first comprehensively characterized all the major cell types present in the CRC TME using single cell RNA sequencing, and with the knowledge gained, we set out to investigate the spatial localization of these cell types of interests and their interactions within the tumor architecture using the 10x Xenium Analyzer and merFISH. From our data, we identified subtypes of tumor epithelial cells that are uniquely enriched in CRC patients and observed distinct spatial organization of these cells within colonic glands across patient subgroups. In addition, we observed an enrichment of CAFs and immune cell populations at the tumor invasive margin, often accompanying regions of tumor budding. Further study of these tumor epithelial, CAF and immune subtypes and their spatial interactions may reveal new insights into CRC tumor biology and assist in the development of targeted therapies for CRC.



 Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, Nirmala Arul Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar. Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2896c139d399fbdf03bf5e11753408c9bfebdc83" target='_blank'>
              Abstract P19: Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics
              </a>
            </td>
          <td>
            Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, N. A. Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="It has become evident from decades of clinical trials that multimodal therapeutic approaches with focus on cell intrinsic and microenvironmental cues are needed to improve understanding and treat the rare, inoperable, and ultimately fatal diffuse intrinsic pontine glioma (DIPG), now categorized as a diffuse midline glioma. In this study we report the development and characterization of an in vitro system utilizing 3D Tumor Tissue Analogs (TTA), designed to replicate the intricate DIPG microenvironment. The innate ability of fluorescently labeled human brain endothelial cells, microglia, and patient-derived DIPG cell lines to self-assemble has been exploited to generate multicellular 3D TTAs that mimic tissue-like microstructures, enabling an in- depth exploration of the spatio-temporal dynamics between neoplastic and stromal cells. The 3D-TTA model recapitulates clinical patterns of DIPG growth, evidenced by resistance to chemotherapy, HDAC and proteasome inhibitors, as well as sensitization to the antibody-activated innate immune microenvironment including complement proteins and surrounding microglia. Multimodal fluorescence imaging platforms integrated with high-throughput omics revealed that alterations in tumor cell motility and growth in the 3D-TTA model compared to tumor cell only spheroids correlated with specific transcriptomic and proteomic changes. STAT3, ITGA5, LGALS1, SOD2, MVP, and CLIC1, associated with microenvironment signaling, DNA replication, and immune regulation, were identified as potential novel targets in the 3D model. The results indicate that the 3D TTA platform developed here represents a powerful tool for preclinical studies, paving the way for identification/validation of tissue specific biomarkers and novel drug targets, thus advancing disease management strategies for DIPG in children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b07b858c6fd1cb8139cba6229ad20ebbdfab9d" target='_blank'>
              Multicellular tumor-stromal interactions recapitulate aspects of therapeutic response and human oncogenic signaling in a 3D disease model for H3K27M-altered DIPG
              </a>
            </td>
          <td>
            Meenakshi Upreti, Astgik Petrosyan, M. Thornton, A. Hovsepyan, G. E. Fernandez, David S. Koos, S. Byrum, Samuel G Mackintosh, Jacob K Al-Husseini, Tania Porras, Joseph Ha, A. Tackett, Miqin Zhang, M. Johal, Anat Erdreich-Epstein, Susan Durham, Mark D Krieger, Ashley S Margol, Brendan H. Grubbs, T. Chambers, S. Asgharzadeh, Rex A. Moats, Peter A Chiarelli
          </td>
          <td>2025-09-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey Pass, Dafna Bar Sagi, Itai Yanai, Kwok-Kin Wong
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most aggressive and treatment-resistant brain tumors, largely due to its profound intratumoral heterogeneity and immunosuppressive microenvironment. Various classifications of GBM subtypes were created based on transcriptional and methylation profiles. This effort, followed by the development of new technology such as single-nuclei sequencing (snRNAseq) and spatial transcriptomics, led to a better understanding of the glioma cells’ plasticity and their ability to transition between diverse cellular states. GBM cells can mimic neurodevelopmental programs to resemble oligodendrocyte or neural progenitor behavior and hitchhike the local neuronal network to support their growth. The tumor microenvironment, especially under hypoxic conditions, drives the tumor cell clonal selection, which then reshapes the immune cells’ functions. These adaptations contribute to immune evasion by progressively disabling T cell and myeloid cell functions, ultimately establishing a highly immunosuppressive tumor milieu. This complex and metabolically constrained environment poses a major barrier to effective antitumor immunity and limits the success of conventional therapies. Understanding the dynamic interactions between glioma cells and their microenvironment is essential for the development of more effective immunotherapies and rational combination strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0dda0986360e9c65e565d1dc755ad16cfd3932d" target='_blank'>
              Unraveling Glioblastoma Heterogeneity: Advancing Immunological Insights and Therapeutic Innovations
              </a>
            </td>
          <td>
            Joshua H. Liu, Maksym Horiachok, Santosh Guru, Cecile L. Maire
          </td>
          <td>2025-08-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Immunotherapy has been used in the clinical management of TNBC. While BRCA1 mutations are associated with immunotherapy response, the therapeutic outcomes in TNBC patients are not promising. Methods This study integrated spatial, single-cell, and bulk RNA-seq data to explore the role of BRCA1 in reshaping the TNBC microenvironment. Through multi-scale analysis, phenotype changes and potential biomarkers in cancer-associated fibroblasts (CAF) were identified. To validate these findings at the protein level, we employed high-resolution, label-free proteomics sequencing in our in-house cohort, providing critical real-world validation. A predictive system for response to ICIs was constructed through the step-by-step machine learning pipeline. Results Compared to BRCA1 mutant patients, BRCA1 wild-type patients experienced increased T-cell exhaustion and dendritic cell tolerance. We identified a MEG3+ pre-CAF subgroup via pseudo-time analysis. Moreover, ISG15 may serve as an immunoregulatory biomarker, and the proposed predictive model demonstrated potential in forecasting immunotherapy response, although further validation is needed. Conclusions This study highlighted the cellular heterogeneity of TNBC and identified ISG15 as a candidate biomarker potentially associated with treatment response. The ISG15-based predictive system might provide a robust framework for predicting ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125312e82f26c9a44c81f1fdf19e7985af609a32" target='_blank'>
              Resolving tumor microenvironment heterogeneity to forecast immunotherapy response in triple-negative breast cancer through multi-scale analysis
              </a>
            </td>
          <td>
            Shihao Sun, Shuang Chen, Kaiyuan Li, Ge Zhang, Nan Wang, Yijia Xu, Xinxing Wang, Jiangrui Chi, Lin Li, Yi Sun
          </td>
          <td>2025-08-19</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. Results We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Conclusion Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00691-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of breast cancer is a complex ecosystem, in which cancer-associated fibroblasts (CAFs), as the most abundant stromal cell type, meticulously construct an ecological niche that supports tumor growth through mechanisms including extracellular matrix (ECM) remodeling, secretion of bioactive factors, and interactions with neighboring cells. High-resolution technologies, including single-cell sequencing and spatial transcriptomics, have revealed the high heterogeneity, functional diversity, and spatial distribution within the CAF population. Significant differences exist in the interactions between distinct CAF subpopulations and immune cells. Through complex crosstalk with the immune system, they collaboratively establish an immunosuppressive network, becoming a core driving force for tumor immune escape. This review focuses on the latest research advances in heterogeneous subpopulations of CAFs within the breast cancer microenvironment, delves into how the complex bidirectional crosstalk between different CAF subpopulations and immune cells collaboratively shapes the tumor immune microenvironment (TIME), and summarizes various CAF-based therapeutic strategies for breast cancer, aiming to provide critical theoretical basis and novel therapeutic perspectives for the clinical translation of CAF heterogeneity research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e74899699742a2ae95fa97ca3627d6bdb366035" target='_blank'>
              Crosstalk between heterogeneous cancer-associated fibroblast subpopulations and the immune system in breast cancer: key players and promising therapeutic targets
              </a>
            </td>
          <td>
            Lu Liu, Weijun Wan, Yilin Chang, Luoquan Ao, Yan Xu, Xiang Xu
          </td>
          <td>2025-09-01</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background T-cell activation and clonal expansion are essential to effective immunotherapy responses in non-small cell lung cancer (NSCLC). The distribution of T-cell clones may offer insights into immunogenic mechanisms and imply potential prognostic and predictive information. Methods We analyzed α/β T-cell receptor (TCR) clonality using RNA-sequencing of bulk frozen tumor tissue from 182 patients with NSCLC. The data was integrated with molecular and clinical characteristics, extensive in situ imaging, and spatial sequencing of the tumor immune microenvironment. TCR clonality was also determined in an independent cohort of nine patients with immune checkpoint-treated NSCLC. Results TCR clonality (Gini index) patterns ranged from high T-cell clone diversity with high evenness (low Gini index) to clonal dominance with low evenness (high Gini index). Generally, TCR clonality in cancer was lower than in matched normal lung parenchyma distant from the tumor (p=0.021). The TCR clonality distribution between adenocarcinoma and squamous cell carcinoma was similar; however, smokers showed a higher Gini index. While in the operated patient with NSCLC cohort, TCR clonality was not prognostic, in an immune checkpoint inhibitor-treated cohort, high TCR clonality was associated with better therapy response (p=0.016) and prolonged survival (p=0.003, median survival 13.8 vs 2.9 months). On the genomic level, a higher Gini index correlated strongly with a lower frequency of epidermal growth factor receptor (EGFR) and adenomatous polypsis coli (APC) gene mutations, but a higher frequency of P53 mutations, and a higher tumor mutation burden. In-depth characterization of the tumor tissue revealed that high TCR clonality was associated with an activated, inflamed tumor phenotype (PRF1, GZMA, GZMB, INFG) with exhaustion signatures (LAG3, TIGIT, IDO1, PD-1, PD-L1). Correspondingly, PD-1+, CD3+, CD8A+, CD163+, and CD138+immune cells infiltrated cancer tissue with high TCR clonality. In situ sequencing recovered single dominant T-cell clones within the patient tumor tissue, which were predominantly of the CD8 subtype and localized closer to tumor cells. Conclusion Our robust analysis pipeline characterized diverse TCR repertoires linked to distinct genotypes and immunologic tumor phenotypes. The spatial clustering of expanded T-cell clones and their association with immunological activation underscores a functional, clinically relevant immune response, particularly in patients with NSCLC treated with checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10cb95082ff80184987c55803cfe4d35d428a707" target='_blank'>
              Spatial TCR clonality and clonal expansion in the in situ microenvironment of non-small cell lung cancer
              </a>
            </td>
          <td>
            Hui Yu, Anastasia Magoulopoulou, Rose-Marie Amini, Maria Chatzinikolaou, Masafumi Horie, A. Lindberg, Artur Mezheyeuski, M. Backman, Andreas Metousis, H. Brunnström, M. Marincevic, J. Botling, J. Mattsson, K. Kärre, Karin Leandersson, Mats Nilsson, Carina Strell, P. Micke
          </td>
          <td>2025-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="ABSTRACT Single‐cell RNA sequencing (scRNA‐seq) has transformed our understanding of tumours by enabling high‐resolution profiling of their cellular composition. Traditionally perceived as masses of homogeneous cancer cells, tumours are now recognised as complex ecosystems shaped by the tumour microenvironment (TME), which includes diverse immune cells, cancer‐associated fibroblasts and extracellular matrix components. scRNA‐seq has revealed remarkable heterogeneity within the TME, identifying novel or rare immune cell subsets and delineating their dynamic functional states. In particular, it has illuminated intercellular signalling networks and temporal cell‐state transitions that drive tumour progression and immune evasion. Moreover, the integration of scRNA‐seq data with clinical information has highlighted its potential in improving patient stratification, biomarker discovery and therapeutic target identification. Here, we systematically summarise recent advances in applying scRNA‐seq to dissect the TME, discuss the implications of these findings for immunotherapy resistance and precision oncology, and outline future opportunities for integrating scRNA‐seq with emerging technologies to develop more effective and personalised cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7bf7b9a7f8397821baf87f91e3e07f64fb492a" target='_blank'>
              Tumour Single‐Cell Bioinformatics: From Immune Profiling to Molecular Dynamics
              </a>
            </td>
          <td>
            Jane Siu Fan Li, Zoey Zeyuan Ji, Aaron Qi Zhang, C. Ng, Guibin Qiao, Dongmei Zhang, Chunjie Li, Qing Zhang, Ka-Fai To, P. M. Tang
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Lung cancer, a leading cause of cancer death, displays profound histologic and molecular heterogeneity across adenocarcinoma, squamous, and small‐cell types. Clinically, tumours can shift between these states, reflecting lineage plasticity—the reprogramming of differentiated cells to alternate identities. Pre‐existing genomic/epigenomic diversity and microenvironmental cues supply the substrates and pressures for plasticity from disease onset. This review anchors plasticity within normal lung development to clarify how fate programs are co‐opted to drive progression, immune escape, therapy resistance, and invasion. Main text Focusing on the intricate interplay between lineage dysregulation and tumour progression in lung cancer, this review integrates insights from lung tissue development to explore the pivotal molecules and mechanisms driving lineage plasticity, alterations and migration during lung carcinogenesis and progression. Recent research findings on lung cancer lineage plasticity are synthesised, shedding light on the role of transcriptional and epigenetic regulators in disrupting tumour lineages. Particular emphasis is placed on how tumour microenvironmental factors, such as hypoxia, stromal cells and immune cells, reshape tumour cellular profiles by modulating the epigenomic landscape. Furthermore, this review specifically discusses the impact of epidermal growth factor receptor (EGFR) and KRAS mutations on lung cancer progression and the consequent immune escape mechanisms they engender. Importantly, we highlight that lineage regulation persists throughout tumour development, from the early onset of lung adenocarcinoma (LUAD) to its progression through late‐stage dedifferentiation and metastasis. We evaluate the implications of these factors on treatment resistance in lung cancer and focus on innovative therapeutic strategies targeting lineage plasticity. Conclusions Lineage plasticity spans the entire course of lung cancer, from early tumorigenesis through metastasis to treatment resistance. Lineage transitions that occur during tumour progression arise from specific combinations of genomic and epigenetic alterations and are further shaped by microenvironmental forces such as hypoxia, stromal remodeling, and immune pressure. By summarising current research advancements, we aim to provide new insights for future lung cancer research and to promote the development of more effective therapeutic interventions. Key points Lineage plasticity runs through the entire process of lung cancer progression and drug resistance, and drives early tumorigenesis via lineage imbalance. Certain driver mutations have lineage‐restricted tumorigenic potential, requiring lineage reprogramming for tumor initiation. Lineage transitions in lung cancer require specific genomic and epigenetic alterations. Lineage plasticity insights provide a mechanistic framework linking lung cancer origin, evolution, and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d07aa8707388affcb2b558f5da5a8327303f46a8" target='_blank'>
              Persistent lineage plasticity driving lung cancer development and progression
              </a>
            </td>
          <td>
            Fanchen Meng, Jianyu Li, Zhijun Xia, Qinglin Wang, Qinhong Sun, Siwei Wang, Lin Xu, Rong Yin
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Summary Kaposi’s sarcoma (KS) is a highly inflammatory, angiogenic tumor driven by Kaposi’s sarcoma-associated herpesvirus (KSHV), yet the origins of tumor cells and mechanisms of progression remain unclear. Here, we present the first spatial single-cell atlas of KS, profiling 256 samples across patch, plaque, and nodular lesions and normal controls. We identify CD34+ progenitor lymphatic endothelial cells (LECs) as the primary targets of KSHV, whose clonal expansion drives tumor growth. KSHV infection induces widespread cellular reprogramming across the tumor microenvironment, including LECs, vascular endothelial cells, fibroblasts, and macrophages, generating hybrid phenotypes that support angiogenesis, inflammation, and immune modulation. KSHV+ macrophages are enriched in tumor-proximal niches, further promoting a proangiogenic, immunosuppressive environment. Spatial analysis reveals evolving tumor-associated niches, with a core-to-periphery gradient correlating with infection, immune modulation, and cellular remodeling. We identify disease progression predictive signatures, offering mechanistic insights into KS pathogenesis and potential new therapeutic strategies by reprogramming the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835b859927d90c9b0c5454b990e0f2840ed6f867" target='_blank'>
              Spatial Single-Cell Atlas Reveals KSHV-Driven Broad Cellular Reprogramming, Progenitor Expansion, Immune and Vascular Remodeling in Kaposi’s Sarcoma
              </a>
            </td>
          <td>
            Wen Meng, Arun Das, Harsh Sinha, Rana Naous, P. Bracci, Mike McGrath, Yufei Huang, Shou-Jiang Gao
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling.
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Circulating tumor cells (CTCs) are the primary mediators of metastatic disease. After leaving the tumor, they can travel through the blood alone or with cluster partners to aid them on their way to distant metastatic sites. While CTCs are critical diagnostic tools for monitoring cancer progression, the biology underlying their behavior is obscured by their rarity and heterogeneity. To unravel the complexity of their actions, we undertook a large-scale longitudinal study to determine the factors that affect their presence, clustering, and their impacts on disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ef013e40043bfc40c0608d2cc2f97f3380a6366" target='_blank'>
              Comprehensive Longitudinal Linear Mixed Modeling of CTCs Illuminates the Role of Trop2, EpCAM, and CD45 in CTC Clustering and Metastasis
              </a>
            </td>
          <td>
            Seth D. Merkley, Huining Kang, U. Brown-Glaberman, Dario Marchetti
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background and objective Melanoma exhibits profound biological complexity, driven by immune evasion, phenotypic plasticity, and resistance to therapy. While programmed cell death (PCD) shapes tumor–immune interactions, its mechanistic landscape in melanoma remains incompletely defined. This study aims to comprehensively characterize PCD-related signatures and their associations with tumor heterogeneity, prognosis, and immunotherapeutic outcomes. Methods Single-cell RNA sequencing data from melanoma cohorts (cutaneous and acral subtypes) were used to assess PCD activity via AUCell-based scoring across major cell types. Cell-type–specific analyses examined heterogeneity, metabolic dependencies, and pathway correlations. Intercellular communication was analyzed using CellChat. Bulk RNA sequencing data were then integrated to identify PCD-related gene signatures, and machine learning models (LASSO, Ridge, XGBoost) were applied to develop a prognostic model. Immune infiltration, immunogenomic correlations, and immunotherapy responses were further evaluated using ESTIMATE, CIBERSORT, TMB, IPS, and external ICB-treated cohorts. Results Among all cell types, melanoma cells exhibited the highest PCD activation, with disulfidptosis, immunogenic cell death (ICD), and autosis being the most prominent. High PCD activity was linked to advanced clinical stage, lymphatic metastasis, and poor prognosis. Melanoma subpopulations with hyperactivated PCD displayed elevated copy number variation (CNV) burden, enhanced fibroblast/endothelial interactions, and invasive transcriptional profiles. A 15-gene prognostic signature was developed, effectively stratifying survival and immunotherapy response across multiple cohorts. Low-risk tumors demonstrated favorable immune infiltration (CD8+ T cells, M1 macrophages), higher tumor mutational burden (TMB), and greater immunogenicity, while high-risk tumors exhibited immune exclusion, cancer-associated fibroblast (CAF) enrichment, and adverse mutations. Conclusion This study highlights the functional and clinical significance of PCD heterogeneity in melanoma and provides a validated prognostic model for patient stratification and therapeutic decision-making. These findings underscore the potential of targeting PCD dynamics as a novel approach in melanoma management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2d7ead94be26e504d9669468ee3ae02d2938b23" target='_blank'>
              Programmed cell death-driven remodeling of the melanoma microenvironment enables prognostic stratification and therapeutic prediction
              </a>
            </td>
          <td>
            Bo Hu, Shengnan Chai, Xuan Li, Qiang Zhang, Mei Jin, Long Zhang
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mixed neuroendocrine–nonneuroendocrine neoplasms (MiNEN) are usually highly aggressive tumors characterized by marked histological heterogeneity, most commonly represented by mixed adenocarcinoma and poorly differentiated neuroendocrine carcinoma (NEC). However, beyond morphological observations, the biological basis and implications of this heterogeneity remain incompletely understood. In this study, we combined component-specific next-generation sequencing and spatial transcriptomics to investigate three mixed adenocarcinoma-NEC cases from different anatomical sites (ileocecal, ovarian, gastric), tracing tumor progression from precursor lesions to invasive NEC. Genomic analyses revealed a shared trunk of driver mutations across all tumor compartments, confirming their clonal origin, while also uncovering additional compartment-specific alterations. Spatial transcriptomics, together with gene set enrichment analysis (GSEA), revealed distinct transcriptional profiles aligned with histologically annotated compartments (e.g., adenocarcinoma, NEC, precursor). In NECs, GSEA consistently showed downregulation of immune-related pathways and upregulation of proliferation-associated pathways compared to non-neuroendocrine tumor areas. Moreover, distinct transcriptomic subclusters were identified within morphologically homogeneous NEC regions in two of the three cases. These subclusters exhibited significant differences in immune regulation, proliferation signaling, and cell-cycle control, and were associated with divergent predicted chemotherapy-response signatures, suggesting clinically relevant implications for treatment sensitivity and resistance. In summary, our findings indicate that despite a shared clonal origin, MiNEN develop distinct genetic and transcriptomic features across tumor compartments. The inconsistent presence of transcriptomic subclusters within morphologically similar regions underscores the complexity of intratumoral heterogeneity in these aggressive neoplasms. By connecting morphological and molecular layers of tumor architecture, spatial profiling may aid in translating biological complexity into more targeted clinical strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12022-025-09869-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d940265ab2a62d1a11097a7fc51ef43cc25274" target='_blank'>
              Exploring Intratumoral Heterogeneity in Mixed Neuroendocrine-Nonneuroendocrine Neoplasms with Spatial Transcriptomics: Even More Diverse Than Anticipated
              </a>
            </td>
          <td>
            Annika Weiß, Julia Teply-Szymanski, Maxime Schmitt, S. Foersch, P. Jank, Joscha Griger, Uwe Wagner, Detlef K Bartsch, Carsten Denkert, Moritz Jesinghaus
          </td>
          <td>2025-08-20</td>
          <td>Endocrine Pathology</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f3009a3214b83a250848d6ef5905a4fb9e1a3db" target='_blank'>
              Unraveling the single cell spatial landscapes of melanoma brain metastases
              </a>
            </td>
          <td>
            A. Manukyan, K. Peters, H. Radbruch, F. Roßner, A. Lehmann, K. Kleo, T. Conrad, J. Altmueller, I. Plumbon, A. Akalin, K. Juerchott, E. Wyler, M. Landthaler, J. Radke, T. Redmer
          </td>
          <td>2025-09-02</td>
          <td>None</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Continuous passaging of cancer cell lines can drive phenotypic and genotypic divergence, potentially compromising the reliability of such models. In this study, we show that two late-passage strains (S1 and S2) of ovarian cancer cell line SKOV3, although authenticated via short tandem repeat (STR) profiling as identical, exhibit substantial differences in morphology, transcriptomic signatures, ability to form 3D cultures and chemotherapeutic responses. Notably, S1 formed compact 3D spheroids and exhibited enhanced epithelial-mesenchymal transition (EMT) pathway activity, whereas S2 displayed a more proliferative, MYC-driven phenotype with larger spheroid structures requiring higher seeding densities. Transcriptomic analysis revealed pathways associated with hypoxia, EMT and angiogenesis in 3D culture, highlighting the complexity introduced by dimensionality in tumour modelling. Critically, S1 showed higher sensitivity to doxorubicin than S2 (IC50 of 0.12 µM versus 1.28 µM, P=0.0001), indicating how clonal evolution can confound drug-response assays. Ultimately, our findings suggest that although STR profiling remains essential for cell line authentication, functionally distinct subpopulations can arise and coexist within the same culture, and their isolation may reveal divergent phenotypes that compromise reproducibility in preclinical cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85b19ad530ecaca77ba40f9ce705c8944938d900" target='_blank'>
              Extended passaging of the SKOV3 ovarian cancer cell line leads to two phenotypically different strains
              </a>
            </td>
          <td>
            Eglė Žymantaitė, M. Gabrielaitė, V. Pašukonienė, A. Mlynska
          </td>
          <td>2025-08-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2μm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Gliosarcoma (GS) is a rare variant of glioblastoma (GBM), characterized by biphasic glial and sarcomatous histology and poor prognosis. Despite its distinct clinical features, GS remains underrepresented in glioma research due to the lack of biologically faithful and experimentally tractable models. Methods We established patient-derived gliosarcoma organoids (GSOs) from freshly resected tumors using a suspension culture system without enzymatic dissociation. Histological, molecular, and functional properties were evaluated using H&E staining, immunohistochemistry, whole-exome sequencing, 3D invasion assays, and single-cell RNA sequencing (scRNA-seq). Drug response assays were performed using temozolomide and ANA-12 (NTRK2 inhibitor). Results GSOs preserved key histological features, genetic alterations, and tumor microenvironmental cell populations from the original tumors. They stable growth and viability over extended culture periods and maintained integrity following cryopreservation and recovery, supporting their long-term utility. 3D invasion assays further revealed infiltrative behavior, consistent with the aggressive nature of GS. Histological and molecular analyses revealed that GSOs retained glial and mesenchymal differentiation, diverse non-malignant stromal cells, and case-specific somatic alterations. Comparative scRNA-seq revealed distinct transcriptional programs: GSOs were enriched for fibroblast-like and oligodendrocyte progenitor-like states, while glioblastoma organoids (GBOs) displayed astrocyte-like differentiation and high connectivity signatures. Functional assays confirmed consistent sensitivity of GSOs to temozolomide, and selective therapeutic response to NTRK2 inhibition was observed in the GSO harboring an NTRK2 alteration, supporting the utility for genomic-context-guided therapy. Conclusion Together, GSOs constitute a tractable and translationally relevant model that faithfully reflects the cellular complexity, genetic landscape, and tumor-specific heterogeneity. This model addresses a critical gap in GS biology and supports its integration into precision oncology. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06952-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6998dea80dedd424ac624050d4245e9964a32790" target='_blank'>
              Tumor microenvironment-preserving gliosarcoma organoids as an in vitro preclinical platform: a comparative analysis with glioblastoma models
              </a>
            </td>
          <td>
            Junseong Park, Dokyeong Kim, Hyeon-Chun Park, Min-Gyoung Park, Songzi Zhang, Okcho Na, Youn Soo Lee, Minho Lee, Stephan Ahn, Yeun-Jun Chung
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Central nervous system (CNS) diseases, a leading cause of global disability and mortality, encompass a wide range of brain disorders such as stroke, Alzheimer's disease, Parkinson's disease, and so on. These diseases are characterized by dynamic cellular heterogeneity and disrupted intercellular crosstalk, yet their molecular drivers remain incompletely resolved. Single‐cell RNA sequencing (scRNA‐seq) dissects transcriptional diversity at cellular resolution, while spatial transcriptomics (ST) maps niche‐specific interactions within tissue architecture—complementary approaches that have revealed disease‐associated subpopulations, neural–glial communication, and microenvironmental remodeling. However, standalone omics layers inadequately capture the genetic, epigenetic, and functional cascades underlying CNS pathologies. Here, we highlight the transformative potential of integrating scRNA‐seq and ST with multiomic profiling to delineate spatially orchestrated molecular networks. Such multiomic convergence enables systematic deconstruction of molecular mechanisms and intercellular communication across disease progression. By correlating these signatures with clinical phenotypes, this strategy accelerates biomarker discovery, patient stratification, and therapeutic target identification. We further discuss challenges in data harmonization, subcellular spatial resolution, and computational scalability that must be addressed to realize personalized CNS medicine. This synthesis advocates for interdisciplinary frameworks to translate multiomic insights into mechanistically grounded diagnostics and therapies, ultimately bridging the gap between molecular discovery and precision clinical intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea44b4c63fcfa5d6fa0d523c87a18b28fd4f249" target='_blank'>
              Spatiotemporal Dynamics of Central Nervous System Diseases: Advancing Translational Neuropathology via Single‐Cell and Spatial Multiomics
              </a>
            </td>
          <td>
            Mingkai Xia, Quan Liu, Wenli Zhang, Jinwen Ge, Zhigang Mei
          </td>
          <td>2025-08-19</td>
          <td>MedComm</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in shaping immune responses and therapeutic outcomes in lung cancer, yet the diversity and functional specialization of tumor-associated macrophages (TAMs) remain poorly resolved. Here, we present a refined classification of TAM subtypes across large cohorts of cancer datasets using integrative analysis of single-cell RNA sequencing, spatial transcriptomics, and clinical datasets from lung adenocarcinoma and lung squamous cell carcinoma. By combining cell-based gene scoring with hierarchical classification, we defined 7 macrophage subtypes-each with distinct transcriptional programs and abundances. Notably, lipid-associated TAMs expand with disease progression and exhibit immunosuppressive and protumorigenic features, whereas tissue-resident macrophages decline. Spatial and survival analyses reveal that an increased lipid-associated to tissue-resident TAM ratio correlates with advanced disease and poor prognosis. Given that spatial transcriptomic assays rely on deconvolution techniques to infer cell type compositions, accurate gene expression signatures are essential, especially for fine-grained subpopulations of TAMs. Our refined subtype-specific signatures address this bottleneck and enhance the resolution of spatial mapping efforts. These findings offer new insights into macrophage heterogeneity and highlight lipid-associated TAMs as potential biomarkers and therapeutic targets in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e49c52a60b0a9da9a722e72cb8391f58dc458b" target='_blank'>
              Transcriptional states of lung cancer microenvironment reveal macrophage subtype dynamics linked to disease progression.
              </a>
            </td>
          <td>
            Duygu Keremitçi, Özlem Tuna, Aissa Houdjedj, Hilal Kazan, Yasin Kaymaz
          </td>
          <td>2025-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="SUMMARY Ovarian high-grade serous carcinoma (HGSC) is characterized by extensive intra-peritoneal dissemination and tumor heterogeneity. In the metastatic cascade, tumors utilize several transcriptional programs to translocate and survive in distant tissues. Here, we analyzed multi-modal, real-world data from 350 tumor samples across 160 patients with HGSC to identify transcriptional programs that drive intra-peritoneal metastasis and heterogeneity. We identified nine transcriptional programs, including those regulating epithelial-mesenchymal transition and immune modulation and cytoskeletal reorganization, which shape distinct metastatic trajectories to solid and ascitic environments and are associated to treatment response. Our results reveal pronounced intra-patient transcriptional heterogeneity, which in some cases surpassed inter-patient heterogeneity, highlighting the importance of multi-site sampling for accurate prognostication and combinatorial treatments in HGSC. Our extensive characterization offers novel insights into intra-peritoneal metastasis with significant prognostic implications, reveals histomorphological biomarkers for patient stratification and paves the way for innovative therapeutic strategies aimed at impairing cancer cell adaptability and limiting metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32c6dc18825437bb12a5a64bda09f91c2c6d329" target='_blank'>
              Multi-modal characterization of transcriptional programs that drive metastatic cascades to solid sites and ascites in ovarian cancer
              </a>
            </td>
          <td>
            Kaiyang Zhang, Essi Kahelin, G. Marchi, Oskari Lehtonen, S. Salloum, K. Lavikka, Yilin Li, Felix Dietlein, A. Lahtinen, J. Oikkonen, S. Hietanen, J. Hynninen, Antti Häkkinen, A. Virtanen, S. Hautaniemi
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Introduction Recent advancements in transcriptomic analysis, combined with single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics, have deepened our understanding of the tumor microenvironment and cellular heterogeneity, laying the groundwork for personalized therapies. The aim of this research is to explore the heterogeneity of tumor cells in colorectal cancer (CRC) and evaluate their prognostic value in different therapeutic contexts, emphasizing the impact of tumor cell heterogeneity on disease progression. Methods scRNA-seq alongside spatial transcriptomics was employed to analyze the heterogeneity of tumor cells in CRC, the spatial distribution of tumor cells, and their interactions with the microenvironment. Results We identified nine distinct tumor cell subtypes, with MLXIPL + neoplasm prevalent in advanced CRC, while ADH1C + and MUC2 + neoplasms were more common in early-stage CRC. MLXIPL + neoplasm was significantly associated with chemotherapy and targeted therapy efficacy. Analysis of spatial transcriptomics indicated that MLXIPL + neoplasm is located in the core area of the tumor cells. We developed a 13-gene prognostic signature (PS) using machine learning algorithm (StepCox backward), which predicts the prognosis of CRC patients. Furthermore, the patients with low PS score demonstrated higher immune cell infiltration and immune regulatory factors, suggesting enhanced immune surveillance and treatment response. Conclusions The findings highlight the critical role of tumor cell heterogeneity in CRC progression and treatment response, suggesting the need for personalized therapeutic strategies targeting different subpopulations. The constructed PS demonstrates significant clinical application value in predicting patient prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00805-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8065bcb7e9315d28e942e5a0a0b33a35e0d6f402" target='_blank'>
              Spatial transcriptomics and scRNA-seq: decoding tumor complexity and constructing prognostic models in colorectal cancer
              </a>
            </td>
          <td>
            Wei Song, Yatao Wang, Min Zhou, Fengqin Guo, Yanliang Liu
          </td>
          <td>2025-08-13</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer characterized by a lack of hormone receptors, making it challenging to treat. Methods We generated a comprehensive spatial cell atlas of TNBC using a multi-omics integration approach that combined single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics. This integration allowed us to characterize the spatial microenvironment and map the cell-type-specific distributions in TNBC tissues. Results Our analysis revealed significant heterogeneity in cell types and spatial distribution, with normal regions enriched in insulin resistance functions, whereas cancerous regions displayed diverse cell populations, including immune cells, cancer-associated fibroblasts (CAFs), and mesenchymal cells. By constructing transcription factor (TF) regulatory networks, we identified TFF3, RARG, GRHL1, RORC, and KLF5 as critical regulators of epithelial cells, whereas EMX2, TWIST1, TWIST2, NFATC4, and HOXC6 were found to play essential roles in mesenchymal cells. Immunohistochemical validation supported the involvement of these TFs in TNBC. Further analysis of receptor-ligand interactions highlighted the roles of KNG1_BDKRB2 and NRG1_ERBB4 signaling in promoting tumor aggression, suggesting potential therapeutic targets. GO enrichment analysis revealed overlapping pathways between epithelial and mesenchymal cells, focusing on migration, signaling, and development, indicating that the shared regulatory mechanisms contribute to cancer progression. Conclusion Our findings provide new insights into the TNBC microenvironment, emphasizing the complex spatial interactions between different cell types and highlighting key regulatory pathways that could be targeted for future therapeutic interventions. This spatial cell atlas lays the foundation for further exploration of tumor microenvironment dynamics and precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8697e1d4f457ce1a990f5f037a0dd30fbd923ee5" target='_blank'>
              Multi-omics integration uncovers key transcriptional regulators in triple-negative breast cancer spatial heterogeneity
              </a>
            </td>
          <td>
            Ning Zhang, Ye Zhang, Rui-Fei Yang, Hai-Qing Chu, Abdur Rehman, Lu-Yu Yang, Ya-Yu Li, Xin Zhou, Shou-Ping Gong, Hui-Ling Cao
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Meningiomas are common brain tumors, but a subset behaves aggressively and resists current treatments. This review proposes a developmentally informed framework showing that where a meningioma arises in the skull mirrors its embryologic origin (neural crest vs. mesoderm) and imprints distinct genetic, epigenetic, and immune programs that shape tumor behavior and treatment response. This lineage “stamp” helps explain why convexity and skull-base tumors differ and highlights actionable circuits, NF2/YAP–TAZ, PI3K/AKT, TRAF7-linked pathways. We integrate spatial genomics and single-cell data to map tumor location and molecular class to rational therapies, including CDK4/6, PI3K/AKT, YAP/TEAD, and epigenetic inhibitors, with emerging strategies such as synthetic-lethal combinations, CRISPR-guided target discovery, and degrader (PROTAC) approaches for high-grade and recurrent disease. Our goal is a practical roadmap that links anatomy and biology to precision treatment and trial design, improving outcomes by matching the right therapy to the right meningioma at the right site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3274f2ff22ffc0fdf55125037e12d854830b1a18" target='_blank'>
              Spatially Encoded Oncogenesis and Transcriptional Plasticity in Meningioma: Drivers of Therapeutic Resistance and Opportunities for Targeted Intervention
              </a>
            </td>
          <td>
            Matthew A Abikenari, Amit Regev, Brandon Bergsneider, Vratko Himič, Shreyas Annagiri, Lily H Kim, Ravi Medikonda, John Choi, S. Jeyaretna, Daniel M. Fountain, Michael Lim
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Purpose The dysregulated immune microenvironment represents a key pathogenic driver in psoriatic lesions. However, the intricate cellular and molecular interactions underlying psoriasis remain incompletely elucidated. Therefore, we aim to employ integrated multi-omics approaches to characterize the immune microenvironment and pathogenic niche in psoriasis, thereby elucidating the cellular and molecular mechanisms of disease pathogenesis. Methods Integrated Single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA sequencing (RNA-seq) data to explore the heterogeneity of stromal cells and immune cells in psoriatic lesions and the complex spatial niches formed between them. Enrichment analysis, intercellular communication analysis, and spatial co-localization analysis were used to investigate the transcriptional changes and distribution characteristics of each cell type in the lesions of psoriasis patients. Results Using scRNA-seq, we identified a novel CD4+ tissue-resident memory T cell (TRM) subset that is exclusively present in lesional skin of psoriasis patients but absent in healthy skin. These cells exhibit elevated expression of genes including IL17RA, IL22, PD1 (PDCD1), CXCR6, ITGAE, CD69, TNFRSF9, TNFRSF4, IL7R, CD4, and STAT3. Additionally, we discovered a novel microvascular endothelial cell subset, designated Venous endo2, which highly expresses CD93, ACKR1, ICAM1, VCAM1, IL15, SELE, and SELP, while also overlapping with high endothelial venule (HEV)-associated transcriptional signatures. Integrated analysis of scRNA-seq and spatial transcriptomics further revealed strong spatial co-localization of Venous endo2 with fibroblast activation protein-positive fibroblasts (FAP+ Fbs), T cells, and antigen-presenting cells (APCs) in Psoriasis lesions—a pattern not observed in healthy control skin. Conclusion Through integrated multi-omics analysis, we identified a potential pathogenic niche in psoriasis patients, composed of Venous endo2, FAP+ Fbs, T cells, and APCs. This structure resembles tertiary lymphoid structures (TLS), suggesting a functional parallel in disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d8eeb3b234a229f3a9f78e22b42dca0befb2a94" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals a Pathological Niche Formed by FAP+ Fibroblasts, Immune, and Endothelial Cells in Psoriatic Lesions
              </a>
            </td>
          <td>
            Wuwei Zhuang, Qi Zhang, Qingtao Kong, Yun Hui, Jie Shen, Chen Zhang, Hong Sang, Qiao Ye
          </td>
          <td>2025-09-01</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila.
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0c307663da680bc20dfe3893c95285de2664ca" target='_blank'>
              Spatiotemporal analyses of the pan-cancer single-cell landscape reveal widespread profibrotic ecotypes associated with tumor immunity.
              </a>
            </td>
          <td>
            Ya Han, Lele Zhang, Dongqing Sun, Guang-xu Cao, Yuting Wang, Jiali Yue, Junjie Hu, Zhonghua Dong, Fang Li, Taiwen Li, Peng Zhang, Qiu Wu, Chenfei Wang
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. M. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4hiBPTFhiE-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17102-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28caecb5ae60429cc1067831e07cb74217728059" target='_blank'>
              Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Xiaowei Gu, Fuzhong Liu, Junfeng Zhang, Nigeerayi Nuermaimaiti, Wenjia Guo, Xiaogang Dong
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic cancer remains a lethal disease despite advances in surgery and systemic treatment in the last two decades, underscoring the urgent need to better understand its biological underpinnings. Despite remarkable advances in the molecular characterization of pancreatic ductal adenocarcinoma (PDAC), clinically actionable biomarkers remain scarce, and current treatment remains empiric. Transcriptomic subtypes such as “classical” and “basal-like” offer some prognostic value, but their ability to guide real-time treatment decisions is limited. In this review, we explore the limitations of current biomarker strategies, in particular subtype-based classifications, and argue for a functional reframing of biomarker development in PDAC, centered on patient-derived organoids (PDOs). We explore four key domains in which PDOs deepen our understanding of therapeutic response and resistance, namely, drug response phenotyping, modeling chemoresistance, incorporating tumor microenvironmental complexity through co-culture systems, and more functional profiling through proteomic and metabolomic approaches. Together, these applications move PDOs beyond static avatars of the tumor to dynamic platforms capable of capturing clinically relevant biology. As functional precision medicine gains traction, PDOs may offer a path to more tailored, responsive treatment strategies in a cancer where new options are urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243db33c2c4b51c3c745a2973296c1c4099759e6" target='_blank'>
              Functional Precision in Pancreatic Cancer: Redefining Biomarkers with Patient-Derived Organoids
              </a>
            </td>
          <td>
            C.A. Chew, Cheng Mun Wun, Yi Fang Lee, C.E. Chee, Khek Yu Ho, Glenn Kunnath Bonney
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Uterine leiomyosarcoma (ULSA) is a highly aggressive gynecologic malignancy characterized by early metastasis, profound immunosuppression, and resistance to conventional therapies, including immune checkpoint blockade (ICB). The intricate tumor microenvironment (TME) and cellular heterogeneity driving its progression and therapy resistance remain poorly defined. Methods We performed single-cell RNA sequencing (scRNA-seq) on metastatic lesions (pelvic cavity, rectum, peritoneum, bladder) from a treatment-naïve ULSA patient and compared them to normal uterine myometrium, MMM (n=5). Integrated analyses included cellular composition mapping, copy number variation (CNV) assessment, pseudotemporal trajectory reconstruction, cell-cell communication inference, functional enrichment, and validation via multiplex immunofluorescence (mpIF). Survival correlations were assessed using the TCGA-SARC cohort. Results In this study, the main finding is that the tumor microenvironment (TME) has a strong immunosuppressive effect. Firstly, its characteristic is exhausted CD8+T cells. This study found that as time progresses, the initial cell markers (CCR7, MAL) gradually disappear, while the exhaustion markers (LAG3, HAVCR2, TIGIT) are enriched. This is associated with poor prognosis. Secondly, the M2-polarized macrophages are mainly composed of M2-like tumor-associated macrophages (TAMs) with tumor-promoting characteristics (CD163, FTH1, FTL, TIMP1), and there is a polarization from M1 to M2. Finally, the immature, tumor-promoting N2 neutrophils (CD15+EDARADD+) enriched in the metastatic foci are associated with poor prognosis. The cell communication involves the interaction of MIF-(CD74+CD44) between T/B cells, as well as the role of the CXCL8 signaling axis in promoting angiogenesis, TAM polarization, and immunosuppression. Conclusion For the first time, a comprehensive single-cell map of ULSA was constructed, depicting a metastasis-susceptible cell subset (U11-EDARADD) and an extremely immunosuppressed tumor microenvironment dominated by depleted CD8+T cells, M2 macrophages and N2 neutrophils. These features shed light on the underlying mechanisms of chemotherapy resistance and immunotherapy failure. The biomarkers identified here (EDARADD, CLDN10, TMIGD2) as well as the dysregulated pathways (TGF-β, angiogenesis, MIF signaling) provide possible targets for future development of combined immunotherapy strategies against this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b01f016db1d6adc66dea5efac6b72384978f8f" target='_blank'>
              Single-cell profiling delineates the tumor microenvironment and immunological networks in patient-derived uterine leiomyosarcoma
              </a>
            </td>
          <td>
            Yi Guo, Dongsheng Shen, Yuhang Xiao, Chenghao Wu, Meiyi Chen, Lina Yang, Huaifang Li, Xiaowen Tong, Rujun Chen, Fang Li
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide. However, the spatial and temporal dynamics underlying its development remain poorly characterized. This study employs spatial transcriptomics (ST) to investigate the progression of intestinal tumors in APC Min/+ mice across multiple time points. We identified distinct transcriptional profiles between tumor and normal tissues, resolving six major cell types through integrated dimensionality reduction and pathological annotation. Pseudo-time trajectory analysis revealed increased expression of MMP11 and MYL9 in later stages of tumor progression. Analysis of human CRC cohorts from the TCGA database further confirmed that high expression of these genes is associated with advanced clinical stages and promotes tumor proliferation and invasion. Temporal gene expression dynamics indicated enrichment of cancer-related pathways concurrent with suppression of lipid and amino acid metabolism. Notably, genes in the DEFA family were significantly upregulated in normal tissues compared to tumor tissues. Functional validation showed that DEFA3 inhibits colon cancer cell migration and proliferation in vitro. These demonstrate the value of ST in resolving spatiotemporal heterogeneity in CRC and identify both MMP11/MYL9 and DEFA3 as potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a441608f72da7375d3ddfdc034535ffa428b9ffc" target='_blank'>
              Investigating the Development of Colorectal Cancer Based on Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhaoyao Qi, Guoqing Gu, Huanwei Huang, Beile Lyu, Yibo Liu, Wei Wang, Xu Zha, Xicheng Liu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Summary/Abstract Glioblastoma stem cell (GSC) cultures are initiated from glioblastoma (GBM) surgical resection tissue. They can capture and propagate key GBM primary tumor molecular and cellular features. We have deeply characterized four isocitrate dehydrogenase (IDH)-expressing (or IDH+) GSC cultures from unrelated adults to serve as cellular models for the majority of adult primary GBM. We demonstrate that GSC cultures can be continuously propagated in defined, serum-free media and 5% oxygen without requiring specialized growth substrates; have well-defined genomic and mtDNA variants and gene/protein expression profiles; and highly reproducible dose-survival curves when treated with the GBM standard-of-care therapies of ionizing radiation (IR) and temozolomide (TMZ). We also illustrate how expressed lentiviral barcodes, mtDNA variants and single cell gene expression profiling can be used to define and track cellular heterogeneity over 40 days after IR treatment. These well-characterized IDH+ GSC cultures can support many high throughput in vitro assay formats, including xenograft, organoid and other GBM disease modeling protocols. They should prove a useful resource to better understand GBM biology, and to identify new and more effective GBM therapies and treatment regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b852883dfd3891f1795ac6382d7546d2b8963f5d" target='_blank'>
              Cellular heterogeneity and therapeutic response profiling of human IDH+ glioma stem cell cultures
              </a>
            </td>
          <td>
            N. Chambwe, Scott R. Kennedy, Brendan F. Kohrn, Pavlo Lazarchuk, Mario Leutert, Guangrong Qin, Bahar Tercan, M. Sanchez-Contreras, Weiliang Tang, Jerome J. Graber, Patrick J. Paddison, Judit Villén, Ilya Shmulevich, Raymond J Monnat
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="High-grade serous tubo-ovarian cancer (HGSC) is marked by substantial inter- and intra-tumor heterogeneity. The tumor microenvironments (TME) of HGSC show pronounced variability in cellular make-up across metastatic sites, which is linked to poorer patient outcomes. The influence of cellular composition on therapy sensitivity, including chemotherapy and targeted treatments, has not been thoroughly investigated. In this study, we examined the premise that the variations in cellular composition can forecast drug efficacy. Using a high-throughput 3D in vitro tumoroid model, we assessed the drug responses of 23 distinct cellular configurations of tumoroids comprised of OVCAR3 HGSC cells, mesenchymal stem cells, HUVEC endothelial cells, and U937 monocytes to an assortment of five therapeutic agents, including carboplatin and paclitaxel. We identified that the overall pooled viability in response to these five drugs was highest among tumoroid compositions that contained a large number of myeloid cells, whereas the most sensitive tumoroids to these agents were comprised of only cancer cells. Additionally, we found that the “mesenchymal tumoroids” containing 400 or more mesenchymal stem cells were more sensitive to carboplatin than paclitaxel. By amalgamating our experimental findings with random forest machine learning algorithms, we assessed the influence of TME cellular composition on treatment reactions. Our findings reveal notable disparities in drug responses correlated with tumoroid composition, underscoring the significance of cellular diversity within the TME as a predictor of therapeutic outcomes. This research establishes a foundation for employing human tumoroids with varied cellular composition as a method to delve into the roles of stromal, immune, and other TME cell types in enhancing cancer cell susceptibility to various treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49e1984d98981fa778612856dadda5f531075bc3" target='_blank'>
              Cracking the code: predicting tumor microenvironment enabled chemoresistance with machine learning in the human tumoroid models
              </a>
            </td>
          <td>
            Michael Bregenzer, P. Mehta, Kathleen M Burkhard, Geeta Mehta
          </td>
          <td>2025-08-19</td>
          <td>Npj Biomedical Innovations</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Significance Androgen deprivation therapy (ADT) is a mainstay in prostate cancer (PCa) treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer directions for PCa modeling and pave the way toward therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b171b1d175becacb6af2e7a071be6d1fa5ee752f" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2025-08-25</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>169</td>
        </tr>

        <tr id="The tumor microenvironment drives many malignant features of pancreatic ductal adenocarcinoma (PDAC). The fibroblasts within pancreatic tumors promote tissue remodeling, immune suppression, and resistance to therapy. However, the interactions between stromal populations and pancreatic cancer cells are less understood in the liver, the most frequent site of PDAC metastasis. To address this, we employ single cell transcriptomics to compare primary pancreatic vs. liver PDAC lesions. Here, we identify the expression of hepatocyte growth factor (HGF) in fibroblasts and its receptor MET in cancer cells are both markedly increased in the PDAC liver niche. Using functional assays, we validate that mitogenic MET signaling is activated in PDAC cells by liver-derived fibroblasts. Importantly, the inhibition of MET signaling leads to reduced tumor growth in immune competent mouse models. Collectively, our data demonstrates that liver stromal-epithelial crosstalk networks engage in signaling pathways distinct from primary pancreatic tumors, highlighting opportunities to develop new treatments for metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa94d2601cca690cf0f25d219abda02e990ca8c" target='_blank'>
              Comparative Analysis of Primary and Liver Fibroblasts Reveals MET as a Potent Target in Pancreatic Cancer Metastasis
              </a>
            </td>
          <td>
            Rima Singh, Natalie Yousefian, Walker M. Allen, Cecily Anaraki, Alica K Beutel, Sabrina Calderon, Ian M. Loveless, Oliver G. McDonald, Jennifer P Morton, David Imagawa, Zeljka Jutric, Thomas F. Martinez, Nina G. Steele, Christopher J Halbrook
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a tumor microenvironment (TME) composed of a dense extracellular matrix, cancer-associated fibroblasts (CAFs), vasculature, neural elements, and immune cell populations. This complex network promotes tumor proliferation, invasion, metastasis, and resistance to immunotherapy and chemotherapy. The microenvironmental characteristics of the various PDAC subtypes are discussed in this review. And we examines the role of cancer cells in the TME, highlighting their ability to manipulate stromal components to serve as collaborators in tumor progression. Furthermore, we explored the formation mechanism of the immunosuppressive microenvironment in PDAC, paying attention on Inflammation and intrinsic genetic alterations, the regulatory effect of metabolic reprogramming, the contribution of CAFs and the role of immune cells in cancer cell metastasis. This review shows the role of soluble molecules and exosomes in facilitating PDAC progression and immune evasion within the microenvironment. In conclusion, we outline the novel therapeutic strategies that focus on the interaction between cancer cells and their microenvironment, with the objective of offering new insights for future precision medical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e8295ec7fa4434bd00f48ee0333dca0ffe9c46" target='_blank'>
              Domestication and feedback: bidirectional hijacking in pancreatic ductal adenocarcinoma microenvironment
              </a>
            </td>
          <td>
            Yuxi Qiao, Haobo Yin, Yong Zhang, Jingdong Zhang, Qian Dong
          </td>
          <td>2025-08-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) progression is driven by a complex interplay of immunosuppressive and inflammatory mechanisms involving various cell types, functional states, and extracellular milieu. To unravel this intricate relationship, we took a systems biology approach using high resolution spatial distributions of multiple key markers in clinically diverse CRC tumors. We employed Multiplexed Ion Beam Imaging (MIBI-TOF) to capture 65 protein markers across 3,700 fields of view from 250 CRC tumors. This allowed us to deeply characterize the phenotypes and functional states of 5.7 million cells. To incorporate extracellular geography information, we developed a computational approach named CellExt. This quantified and modeled the gradated abundance of key structural and functional proteins found extracellularly, such as fibronectin, granzymeB and vimentin, from each cell boundary to a 40-micron distance in pixelwise steps. CellExt detected a distinct immunosuppressive modality characterized by extracellular PDL1, TNF-related apoptosis-inducing ligand (TRAIL), and matrix metalloproteinase-9 (MMP9). Investigating the cell types involved led us to map the potential receivers and emitters by linking extracellular trends to cellular enrichment and gene expression. We found that Pankeratin-high epithelial cells were acting as TRAIL receivers with downstream activation of non-canonical pro-tumorigenic signaling through S6 phosphorylation. In parallel, we found that T cell suppressive CD40+ antigen-presenting B cells with PD1 expression were acting as MMP9 emitters. Additionally, CellExt was able to identify two spatially segregated inflammatory modalities: IFNgamma cytokine with mucin protein, MUC5AC and prostaglandin synthase, cyclooxygenase-2 with extracellular matrix component, Collagen1A1. This suggests a strong inflammatory role for Collagen1A1 in CRC. We then contextualized our identified modalities by genetic background, tumor stage and CMS subtype of CRC tumors. Taken together, our approach revealed novel axes of immunosuppression and inflammation and puts forward potential extracellular targets for therapeutic intervention in specific CRC patient subsets.



 Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, Christine Fullaway, Sean Bendall, Iain Beehuat Tan, Shyam Prabhakar. Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P38.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18eea1ae1312b735cfb9925fbcce433bb07c964b" target='_blank'>
              Abstract P38: Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation
              </a>
            </td>
          <td>
            Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, C. Fullaway, Sean C. Bendall, Iain Beehuat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The tumor microenvironment (TME) consists of tumor cells, stromal cells, infiltrating immune cells and non-cellular components such as extracellular matrix, blood vessels and a wide variety of secreted proteins. Evidence shows that beyond supporting tumor growth, the TME also promotes tumor cell proliferation and invasion and contributes to treatment resistance, ultimately affecting patient prognosis. Cell-to-cell communication within the TME is driven by secreted proteins such as cytokines, chemokines, growth factors and interferons, which are produced not only by tumor cells but also by various stromal cells and immune cells. These proteins form a complex signaling network that promotes tumor cell proliferation and invasion and enables tumors to evade innate and adaptive immune responses. Antibody arrays are a technology that can simultaneously screen hundreds of secreted proteins in complex biological samples, aiding in the exploration of this complex signaling network. By combining high-throughput multiplex immunoassays such as antibody arrays with cellular and molecular biology techniques, researchers have uncovered complex regulatory mechanisms of cytokine networks within the TME. The present review summarized recent findings on the communication between tumor cells and the TME, as well as key secreted proteins essential for tumor progression and the development of therapeutic resistance. In addition, it discusses how high-throughput antibody arrays contribute to our understanding of regulatory networks of secreted proteins in the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b84b1200ba5d18e4e2abf23d107f5f3c2ed9e0f6" target='_blank'>
              Unraveling tumor cell-tumor microenvironment crosstalk through antibody array technologies (Review)
              </a>
            </td>
          <td>
            Yanlin Wang, Shuhong Luo, Hua Dong, Ruo-Pan Huang
          </td>
          <td>2025-08-18</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Retinoblastoma is the most common intraocular malignant tumor in childhood. Although current treatments offer a high survival rate, treatment toxicity, tumor relapse, and treatment resistance require a deeper understanding of the disease mechanisms to develop adapted therapies. Microscopically, this tumor is characterized by different states of differentiation and proliferation, ranging from poorly differentiated to well-differentiated retinoblastoma. Retinocytoma, on the other hand, is a benign non-proliferative tumor. Recent next-generation multi-omics analyses classified retinoblastoma tumors in subtypes 1 or 2, subtype 2 presenting a later age of onset, more genetic alterations, and higher metastatic potential. In parallel, several single-cell transcriptomics studies demonstrated intratumoral heterogeneity. However, mapping the different cell populations directly on the tumor and comparing histological features and molecular subtypes remains an unmet need. Methods Spatial transcriptomics was used to characterize a primary enucleated case with two histologically distinct areas. The whole transcriptomic profile of sixteen regions of interest, covering the two differentiation patterns of the tumor and the non-tumoral retina, was analyzed. Results The clustering of the regions of interest based on whole transcriptome data correlated with the histological description: cluster 1 (6 regions of interest) corresponded to highly differentiated areas and cluster 2 (6 regions of interest) to poorly differentiated areas. They showed enrichment for phototransduction and proliferation respectively, confirmed by immunohistochemistry for markers of these pathways. The publicly available molecular signatures of the two retinoblastoma subtypes categorized our regions of interest into two groups, which correlated perfectly with histological observation and transcriptomic profiles. Further analysis of the expression of specific senescence markers in the well-differentiated area did not support the diagnosis of retinocytoma as it did not confirm the expected up-regulation seen in this tumor type. Conclusions This study demonstrates a strong correlation between histological observation and molecular profiling, representing the first mapping of retinoblastoma composed of adjacent molecular subtypes 1 and 2. It also highlights the diagnostic ambiguity between retinocytoma and well-differentiated non-proliferative subtype 1 retinoblastoma and emphasizes the need for specific biomarkers to differentiate these two tumor types. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14814-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599058c9510122a1b04d9c9b60c21f33e8817b4e" target='_blank'>
              Spatial transcriptomics of retinoblastoma: a visual window on intra-patient heterogeneity
              </a>
            </td>
          <td>
            A. Moulin, J. Thevenet, L. Mazzeo, S. Tissot, C. Stathopoulos, F. L. Munier, A. Berger
          </td>
          <td>2025-09-02</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Trachea squamous cell carcinoma (TSCC) is a subtype of lung cancer. A thorough investigation of the tumor microenvironment of TSCC is crucial for the development of cancer therapeutics and predicting clinical responses. In this study, we utilized single-cell RNA sequencing to analyze seven TSCC samples (including five malignant and two non-malignant samples) and obtained 70,682 high-quality cells. Based on the expression levels of marker genes, we identified 7 major cell types within the samples. By comparing malignant samples that received chemotherapy with those that did not, we identified critical transcriptional regulators responsible for T cell state transition in response to chemotherapy. Additionally, we found specific transcriptional regulators and differentially expressed genes between malignant and non-malignant groups. We identified more particularly abundant specifical intercellular communication in the malignant sample group and that may significantly influence the progression and spread of cancerous cells. Overall, our study provides the first single-cell atlas that comprehensively explains TSCC development and chemotherapy effects, thereby laying a new molecular foundation for therapeutic research in TSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f4d5fafc26e824adc29e95477c2a4bfd6ebf074" target='_blank'>
              Comprehensive single-cell sequencing reveals the tumor microenvironment and tumor-specific characteristics in trachea squamous cell carcinoma
              </a>
            </td>
          <td>
            HongWu Wang, Hongli Li, Heng Zou, Han Meng, Yan Liu, Chengjun Ban, Weixia Yu, Miao Cheng, Jun Teng
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Medullary thyroid carcinoma (MTC) is a rare neuroendocrine tumor arising from parafollicular C-cells in the thyroid. Unlike other neuroendocrine tumors, knowledge about the MTC tumor microenvironment and its role in therapy resistance remains limited, mainly due to the lack of reliable in vitro models. We hypothesized that, as observed in other cancer types, MTC cells may exhibit a degree of phenotypic plasticity that could be modulated by various microenvironmental factors.



 The primary aim of this study was to establish patient-derived MTC in vitro models capable of growing under different three-dimensional (3D) culture conditions while retaining the ability to shift toward a more mesenchymal, stem-like phenotype.



 We successfully established eight patient-derived primary MTC cell lines from tumors with different genetic backgrounds and variable degrees of aggressiveness. These lines were characterized in terms of phenotype and responses to different microenvironmental stimuli and targeted therapies through Western Blot, confocal microscopy, ELISA assays, ELDA, RT-qPCR, and proliferation assays.



 Notably, the cells maintained their plasticity over extended passaging when cultured under stem-promoting conditions. In differentiation media, they preserved the ability to secrete high levels of classic MTC markers, such as calcitonin and chromogranin A. Under 3D culture conditions, all cell lines showed a significant increase in stem and neuroendocrine precursor markers, including SOX2, OCT4, TBB3 and FOXA1, along with spatially dependent shifts in Epithelial-to- Mesenchymal Transition (EMT) markers E-cadherin and Vimentin. These phenotypic variations partially recapitulated the molecular alterations observed in MTC tissue samples. Moreover, changes in culture conditions—such as inducing a pseudo- hypoxic state or culturing in non-adherent environments—profoundly affected both cell phenotype and sensitivity to therapeutic agents currently available for MTC treatment, including Pralsetinib, Selpercatinib, Cabozantinib, Lenvatinib, and Vandetanib.



 In conclusion, we present here the generation of a new valid patient- derived in vitro model that can provide new opportunities to explore MTC biology complexity and will allow future translational studies, including those involved in uncovering the mechanisms underlying the insurgence of therapy resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0c689353bda0dac4479e679a592360fdc7492a4" target='_blank'>
              OC48 - Generation of patient-derived in vitro models to unravel medullary thyroid cancer microenvironment and therapy resistance
              </a>
            </td>
          <td>
            V. Ghiandai, V. Cirello, G. Gazzano, G. Dionigi, C. Colombo, L. Fugazzola, L. Persani, E. S. Grassi
          </td>
          <td>2025-09-01</td>
          <td>European Journal of Endocrinology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Gliomas represent the most frequent primary brain tumors in adults and are marked by considerable molecular and clinical heterogeneity. The 2021 WHO classification of central nervous system tumors identifies three major adult-type diffuse glioma subtypes: astrocytoma IDH-mutant, oligodendroglioma IDHmutant with 1p/19q-codeletion, and glioblastoma IDH-wildtype.1 Despite ongoing advances, treatment resistance and poor prognosis remain major challenges, largely due to intratumoral heterogeneity and the complexity of the tumor microenvironment.2 A promising yet underexplored avenue for patient-centered glioma profiling is the tumor secretome, which includes soluble factors, extracellular vesicles, and metabolites released by both neoplastic and stromal cells.3 Existing studies rely on in vitro models that fail to recapitulate the complexity of tumor-host interactions. To bridge this gap, we established an innovative, standardized ex vivo protocol for secretome isolation directly from glioma tissue. The protocol preserves microenvironmental cues from the operating room to laboratory analysis, ensuring spontaneous secretome release under controlled conditions. Feasibility and performance were assessed in a preliminary cohort of seven adult diffuse glioma patients. Transmission electron microscopy confirmed tissue structural integrity. Proteomic profiling identified 84 cancer-associated proteins within the secretome. Additionally, metabolomic analysis revealed a distinctive signature of small molecules. Notably, glioblastoma IDH-wildtype samples exhibited increased lactate levels, consistent with hypoxia-driven metabolism, and decreased N-acetylaspartate, indicating neuronal loss. Overall, this ex vivo secretome protocol offers a robust and physiologically relevant platform for glioma characterization, with potential applications in biomarker discovery, patient stratification, and the development of personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea6096ea6e8da3b1feda45c80168b298c559e43" target='_blank'>
              P64 | EX VIVO EXTRACTION OF GLIOMA SECRETOME: A NOVEL MICROENVIRONMENT-BASED PROTOCOL FOR METABOLIC AND MOLECULAR PROFILING
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Background and Aims The relationship between chronic liver disease and liver cancer remains poorly understood, and treatment options for advanced liver disease remain limited. This study aims to elucidate the dynamic evolution of cellular and molecular alterations from normal liver to diseased liver. Methods Single‐cell RNA sequencing was performed to profile the dynamic cellular variations from normal liver to diseased liver. Bioinformatics analyses were employed to assess tumour heterogeneity and the evolution of tumour microenvironment. Molecular biology validations were performed to investigate the findings. Results Through single‐cell RNA sequencing, we constructed a human liver landscape consisting of more than 130 000 single cells, from normal, cirrhotic tissue, primary liver cancer and paired adjacent tissues. We profiled both inter‐ and intra‐tumour heterogeneity of hepatocellular carcinoma and intrahepatic cholangiocarcinoma. A dual role of the pro‐fibrosis and pro‐carcinogenesis FABP5high macrophage subset was shown to be conserved throughout the progression of liver fibrosis and carcinogenesis; markers, spatial localization, origin, and functional pathways associated with FABP5high macrophages were identified through in vitro and in vivo validation. Metabolic reprogramming of these conserved FABP5high macrophages is implicated in the formation of cirrhotic and cancerous niches. Additionally, we also identified a subset of RGCC+COL4A1+ endothelial cells enriched in HCC tissues, which might experience endothelial‐mesenchymal transition and orchestrate the proliferation and invasion of cancer cells via angiogenesis. Conclusions Our findings provide insights into the evolution from chronic liver disease to liver cancer. These insights will contribute to the development of novel cell subset‐based therapeutics based on immunotherapy and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec8b08f4e01c5f59d84102994dc20c75fd98f8a8" target='_blank'>
              A Conserved FABP5high Macrophage Subset Promotes Fibrosis and Carcinogenesis in Advanced Liver Disease
              </a>
            </td>
          <td>
            Wenhao Ge, Lifeng Ma, Yu Huang, W. E., Shujing Lai, Bijun Cui, Jingjing Wang, Yutian Ji, Daiyuan Liu, Song Ye, Yang Kong, Jiarong Zhou, Chengxuan Yu, Yingsheng Wu, Qiang Sun, Dongkai Zhou, Wei Zhu, Yuan Ding, Xiaoping Han, Guoji Guo, Weilin Wang
          </td>
          <td>2025-08-28</td>
          <td>Liver International</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="PURPOSE
Clonal hematopoiesis of indeterminate potential (CHIP) has been associated with adverse outcomes in some solid tumor settings, but its impact on breast cancer remains unclear. We sought to investigate the genotype-specific effects of CHIP on breast cancer outcomes and the tumor microenvironment.


EXPERIMENTAL DESIGN
We examined a retrospective cohort of 125 patients with breast cancer, using targeted sequencing to identify CHIP. Metastatic events were recorded, and distant metastasis-free survival probability was analyzed. In parallel, we developed chimeric mouse models of the two most mutated CHIP genes, DNMT3A and TET2. CHIP and control mice were orthotopically injected with syngeneic breast cancer cells. Tumor growth was measured, and immune infiltrate was profiled via mass cytometry.


RESULTS
CHIP was present in 18.4% of patients. High-burden CHIP and non-DNMT3A CHIP were associated with significantly shorter distant metastasis-free survival. In vivo, mice with Tet2-CHIP developed larger primary tumors and were more likely to experience lung metastasis, while Dnmt3a-CHIP did not differ from controls. The general immune subsets observed in both CHIP models were similar, but immunophenotyping revealed clonal expansion and immune cell subset skewing specific to the Tet2-CHIP model.


CONCLUSIONS
Our findings demonstrate a genotype-specific impact of CHIP on breast cancer across human and mouse data. Further, the chimeric mouse models we generated offer a clinically relevant tool to study solid tumors in a CHIP background. This work underscores the need for further functional studies and personalized risk assessment to clearly define the impact of various CHIP genotypes on breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ebb88f9db19ddf675e686f40d924b6b992289a8" target='_blank'>
              Clonal hematopoiesis of indeterminate potential influences breast cancer outcomes in a genotype-specific manner.
              </a>
            </td>
          <td>
            S. C. Reed, Chad R Potts, Leo Luo, Brad A Davidson, Riley E. Bergman, Justin D J Kemp, Ethan K Fox, Bryce A Thomas, Lauren Ha, Vani Arora, Justin Cartailler, M. E. Sanders, Violeta Sanchez, Paula I. Gonzalez-Ericsson, S. Croessmann, Paula J Hurley, Alexander G. Bick, P. Ferrell, B. H. Park
          </td>
          <td>2025-09-19</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Background Mucosa-associated invariant T (MAIT) cells represent a unique population of innate-like T lymphocytes capable of detecting non-peptide antigens in the context of monomorphic antigen-presenting molecules. Due to their abundance in barrier tissues, reactivity to local inflammatory cues, and cytotoxic and regulatory functions, MAIT cells are poised to shape the dynamics of various tumor microenvironments. Growing evidence suggests that MAIT cells can exert protumor and/or antitumor effects in cancers arising from or metastasizing to mucosal tissues. However, MAIT cell roles in bladder cancer (BCa) remains unclear. Methods To begin to identify MAIT cells in BCa, we stained bladder tumor biopsies for T cell receptor (TCR) Vα7.2+ cells. We then refined a human MAIT cell signature, which enabled us to interrogate a bulk RNA sequencing dataset and conduct correlation analyses linking intratumoral MAIT cell abundance and mortality from BCa. To extend our work to an in vivo setting, we employed a clinically relevant mouse model in which Mr1+/+ B6-MAITCAST (MAIT-sufficient) and Mr1−/− B6-MAITCAST (MAIT-deficient) mice were exposed to N-butyl-N-(4-hydroxybutyl)nitrosamine, a chemical carcinogen associated with tobacco smoke. In additional experiments, MAIT cells were functionally removed through acetyl-6-formylpterin (Ac-6-FP) administration. Effector and regulatory cell types were phenotyped by flow cytometry, and BCa tumor burden and progression were assessed by MRI and/or H&E and Ki67 staining. Results TCR Vα7.2+ cells were readily detectable in several BCa biopsies, and our bioinformatic analyses correlated heavier MAIT cell presence in BCa tumors with poorer overall survival. Similarly, we found higher tumor burdens in Mr1+/+ B6-MAITCAST mice than in Mr1−/− or Ac-6-FP-treated animals. Bladder MAIT cells from tumor-bearing mice exhibited phenotypic MAIT17 bias based on transcription factors they harbored along with increased interleukin-17A and tumor necrosis factor-α production capacities upon stimulation. Finally, FoxP3+ regulatory T (Treg) cell frequencies were elevated in Mr1+/+ mouse bladder tumors, likely contributing to an immunosuppressive tumor microenvironment, a finding that could be recapitulated in our transcriptomic studies on human BCa. Conclusions MAIT cells are abundant in BCa tumor microenvironments where they potentiate Treg cell accumulation and play protumor roles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/904ce037c95ea168f087ef993f969eaddff0121a" target='_blank'>
              MAIT cells promote cancer progression and regulatory T cell accumulation in bladder tumor microenvironment
              </a>
            </td>
          <td>
            Inbar A Habaz, Xuejin Ou, Nicole I Wang, Angela Schincaglia, Valeryia Shydlouskaya, Agetha Mahendran, F. Martinez-Santiesteban, Dylan Doolabi, Yige Bao, Ahmad R Movasseghi, Julius Haruna, Nicholas E. Power, Timothy J Scholl, Melissa J Huynh, Alp Sener, S. M. Haeryfar, Dr S M Mansour Haeryfar
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Papillary thyroid carcinoma (PTC) is the most prevalent form of thyroid cancer, yet its cellular heterogeneity and prognostic determinants remain poorly defined. Here, we integrate two single-cell RNA sequencing datasets comprising 20 human thyroid samples to construct a high-resolution cellular atlas of PTC. We identify 29 distinct cellular subpopulations and delineate their composition, dynamics, and interactions in healthy versus tumor tissues. Notably, epithelial and monocyte populations were markedly expanded in PTC, whereas adaptive immune subsets such as B and T cells were diminished. Cell–cell communication analysis revealed enhanced intercellular signaling in the tumor microenvironment, with epithelial and endothelial cells receiving the strongest inputs. Among monocyte-specific transcriptional signatures, we identified 65 prognostic genes via univariate Cox analysis. A LASSO-derived 14-gene risk score robustly stratified patient outcomes, with CSGALNACT1 emerging as a key epithelial-specific, independent prognostic gene. Pseudotime analysis further supported its role in epithelial cell differentiation. Functional validation demonstrated that CSGALNACT1 promotes proliferation in PTC cell lines, suggesting a potential oncogenic function. Immune deconvolution across risk groups revealed substantial divergence in innate and adaptive immune infiltration, indicating a close interplay between tumor-intrinsic transcriptional programs and immune microenvironment remodeling. Collectively, our study provides a comprehensive single-cell framework for PTC, identifies a clinically relevant risk model, and highlights CSGALNACT1 as a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62fcd940763c0c7abf39d3fd346b8a069b2e5dc" target='_blank'>
              Comprehensive analysis of the papillary thyroid carcinoma identifies CSGALNACT1 as a proliferation driver and prognostic biomarker
              </a>
            </td>
          <td>
            Yu Zhao, Juan Zhang, Yong Yao, Linna Yu
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea7d79855ee47627896db43f29cfa10fcc98ce7" target='_blank'>
              Single-cell multiomic modelling of early metastatic events promoted by the extracellular matrix.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Jun Hyeong Lee, Junho Kang, Kyeonghui Kim, Youngwon Cho, Jihyun Park, Sang-Hyun Song, J. Kwon, Young-Joon Kim, Woong-Yang Park, Tae-You Kim, Jong-Eun Park, Pilnam Kim, Jung Kyoon Choi
          </td>
          <td>2025-09-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Recent studies have explored the composition of tumor microenvironment (TME) in diffuse large B-cell lymphoma (DLBCL) However, cell-to-cell interactions, along with the spatial organization of DLBCL TME and their impact on patient outcomes, have remained poorly characterized. We ap-plied multiplexed immunofluorescence, cell phenotyping, and neighborhood analysis to investigate 1,218,756 single cells in 99 samples from patients with primary DLBCL. We identified 17 cell pheno-types and 10 recurrent cellular neighborhoods (RCNs) across samples, subdividing DLBCLs into immune-poor areas and areas with diverse immune cell infiltrates. Avoidance of B cells and PD-1+ T cells was associated with less aggressive clinical characteristics and favorable survival. Like-wise, the proximity of CD8+ T cell-rich and immune-poor RCNs translated to favorable, and the proximity of PD-L1+ B cell-rich and CD8+ T cell-rich RCNs to unfavorable patient outcomes. Our findings provide insights into the spatial interactions and organization of DLBCL TME with implica-tions for patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8a36543383d1dc42cdb78f6b491ab4c213fbf1a" target='_blank'>
              Spatial Analysis of the Tumor Microenvironment in Diffuse Large B-Cell Lymphoma Reveals Clinically Relevant Cell Interactions and Recurrent Cellular Neighborhoods.
              </a>
            </td>
          <td>
            M. Autio, S. Leivonen, L. Meriranta, M. Karjalainen-Lindsberg, T. Pellinen, S. Leppä
          </td>
          <td>2025-08-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Despite the availability of numerous approved immunotherapies for various cancers, durable progression-free survival remains relatively uncommon among patients with advanced cancer. As research into immunotherapy intensifies, the heterogeneity and complexity of the tumor microenvironment (TME) have emerged as critical determinants of treatment response and a major obstacle to understanding tumor resistance mechanisms. Recent advances in spatially resolved transcriptomics (SRT) enable transcriptome-wide measurement of gene expression while preserving essential spatial information, which supports the characterization of the features of the TME. This review outlines key challenges in cancer immunotherapy and emphasizes the importance of the TME in determining therapeutic efficacy. SRT strategies suitable is systematically introduced for TME profiling, comparing their relative advantages and limitations. Additionally, bioinformatics tools and methods used to reconstruct spatial tissue patterns are discussed from SRT data. Furthermore, how SRT is explored can enhance the understanding of the complexity and heterogeneity of the TME, facilitate the identification of novel therapeutic targets, and advance biomarker discovery to refine personalized treatment strategies for diverse cancer patients. Finally, future perspectives on the application of SRT in cancer immunotherapy and addressing the associated challenges are provided.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53e788cfc2c6f7fa7f419ffa32b647da6da2a8f3" target='_blank'>
              Spatially Resolved Transcriptomics: Revealing Tumor Microenvironment Heterogeneity to Advance Cancer Immunotherapy.
              </a>
            </td>
          <td>
            Dan Liu, Lu Xiao, Yuyuan Wu, Chensirong Yue, Mengmeng Li, Yuanpei Sun, Chao Yang
          </td>
          <td>2025-09-04</td>
          <td>Small methods</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Background γδ T cells possess unique immunological features including tissue tropism, major histocompatibility complex-independent antigen recognition, and hybrid T/natural killer cell properties that make them promising candidates for cancer immunotherapy. However, the therapeutic potential of Vδ1 γδ T cells, particularly when engineered with chimeric antigen receptors (CARs), remains underexplored in solid tumors such as pancreatic cancer (PC), largely due to their low abundance in peripheral blood and challenges in ex vivo expansion. This study aims to directly compare the preclinical safety and efficacy among CAR-engineered Vδ1 γδ T cells, Vδ2 γδ T cells, and conventional αβ T cells. Methods We developed a CAR targeting prostate stem cell antigen (PSCA) and engineered it into human blood-derived Vδ1 γδ T cells. These PSCA CAR-Vδ1 T cells were evaluated for antitumor activity against PSCA-expressing pancreatic tumor cells using both in vitro cytotoxicity assays and in vivo xenograft mouse models. Comparative analyses were conducted using PSCA CAR-Vδ1 γδ T cells, PSCA CAR-Vδ2 γδ T cells, and PSCA CAR-αβ T cells. Safety assessments of these CAR-engineered T cell subsets were conducted to evaluate graft-versus-host disease (GvHD) and cytokine release syndrome (CRS), while exhaustion profiles were assessed using single-cell RNA sequencing. Results Our study demonstrated that PSCA CAR-Vδ1 γδ T cells, PSCA CAR-Vδ2 γδ T cells, and PSCA CAR-αβ T cells exhibited similar antitumor efficacy. However, PSCA CAR-Vδ1 T cells did not exhibit GvHD or CRS compared with CAR-αβ T cells. PSCA CAR-Vδ1 T cells also had lower scores for exhaustion when compared with PSCA CAR-Vδ2 T cells by single-cell transcriptomic analysis. Conclusion PSCA CAR-Vδ1 γδ T cells represent a potent and safe therapeutic modality for PSCA+ PC, with efficacy comparable to other CAR T cell types and potential advantages in safety and persistence. These findings support further development of CAR-Vδ1 T cell immunotherapy for solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32f1cca28d11253460c5ca6b0458064057000da9" target='_blank'>
              Comparative efficacy and safety of PSCA CAR-engineered Vδ1 γδ T cells for immunotherapy of pancreatic cancer
              </a>
            </td>
          <td>
            Zhixin Li, Ningyuan Liu, Zahir Shah, Lewei Jin, L. Tian, Xiaolei Sun, Jianying Zhang, Zhiyao Li, M. Caligiuri, Jianhua Yu
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chemoresistance is a major challenge in the treatment of triple-negative breast cancer (TNBC). Multicellular spheroids are an attractive platform for investigating chemoresistance in TNBC, as they replicate the cues of the tumour microenvironment in vivo. We conducted a comprehensive literature search to summarise the multifactorial and interlinked mechanisms driving chemoresistance in TNBC spheroids. These mechanisms include spatial heterogeneity, hypoxia, extracellular matrix remodelling, tumour–stroma crosstalk, drug efflux, apoptotic resistance, and cancer stem cell signalling. Strategies for overcoming chemoresistance in TNBC spheroids include nanocarrier systems to overcome spatial diffusion limitations, pathway inhibition, and targeting tumour–microenvironment interactions. Despite their advantages, some spheroid models face challenges such as low reproducibility, a lack of heterogeneity, variability in size and shape, limited vascularisation, and constraints in long-term culture. Advanced culturing platforms such as clinostat bioreactors allow for extended culture periods, enabling mature spheroid drug testing. Furthermore, advanced analytical techniques provide spatially resolved spheroid data. These multifactorial and interlinked mechanisms reflect the tumour microenvironment in vivo that spheroids recapitulate, rendering them valuable models for studying chemoresistance. The incorporation of stromal components and advanced analytical workflows will enhance the utility and translational relevance of spheroids as reliable preclinical models for drug discovery in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdb7ab453824dd32a274fa7bdf4060e455f56d16" target='_blank'>
              Utility of Multicellular Spheroids for Investigating Mechanisms of Chemoresistance in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            KN Ncube, Iman van den Bout, Clarissa Willers, C. Gouws, W. Cordier
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive and heterogeneous brain tumor. Glioma stem-like cells (GSCs) play a central role in tumor progression, therapeutic resistance, and recurrence. Although immune cells are known to shape the GBM microenvironment, the impact of antigen-presenting-cell (APC) signature genes on tumor-intrinsic phenotypes remains underexplored. We analyzed both bulk- and single-cell RNA sequencing datasets of GBM to investigate the association between APC gene expression and tumor-cell states, including stemness and metabolic reprogramming. Signature scores were computed using curated gene sets related to APC activity, KEGG metabolic pathways, and cancer hallmark pathways. Protein–protein interaction (PPI) networks were constructed to examine the links between immune regulators and metabolic programs. The high expression of APC-related genes, such as HLA-DRA, CD74, CD80, CD86, and CIITA, was associated with lower stemness signatures and enhanced inflammatory signaling. These APC-high states (mean difference = –0.43, adjusted p < 0.001) also showed a shift in metabolic activity, with decreased oxidative phosphorylation and increased lipid and steroid metabolism. This pattern suggests coordinated changes in immune activity and metabolic status. Furthermore, TNF-α and other inflammatory markers were more highly expressed in the less stem-like tumor cells, indicating a possible role of inflammation in promoting differentiation. Our findings revealed that elevated APC gene signatures are associated with more differentiated and metabolically specialized GBM cell states. These transcriptional features may also reflect greater immunogenicity and inflammation sensitivity. The APC metabolic signature may serve as a useful biomarker to identify GBM subpopulations with reduced stemness and increased immune engagement, offering potential therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf0983476db2efab2251dc274b40344cf2a61c3" target='_blank'>
              Elevated Antigen-Presenting-Cell Signature Genes Predict Stemness and Metabolic Reprogramming States in Glioblastoma
              </a>
            </td>
          <td>
            Ji-Yong Sung, K. Hwang
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) is one of the leading cause of cancer-associated mortality globally. Its tumor microenvironment consists of various types of cells, which have a great impact on patient response and prognosis. The present study used scRNA-seq technology to investigate gene expression pattern of various cell types in NSCLC and their correlation with immune microenvironment. Methods We acquired NSCLC single-cell RNA sequencing data from GSE117570 and employed UMAP clustering analysis to explore the major cell subpopulations. Additionally, we explored the differential expression of important genes among various cell types and their associations with immune cell infiltration, scores of tumor microenvironment. Results Our results exhibited the heterogenicity of cell types in NSCLC and also indicated more than 60 genes whose expression displayed significant differences between cell groups, such as AP1S1, BTK, FUCA1, NDEL14, TMEM106B and UNC13D. Expression of these genes correlated with immunocyte infiltration, tumor microenvironment scores, indicating their potential roles in tumor progression and therapeutics. Conclusion Our single-cell transcriptomic dissection of NSCLC highlights cellular heterogeneity as well as potential biomarkers associated with immune microenvironment. The findings provide a foundation for the rational development of biomarker-driven personalized treatment and therapy by extending the cell-type specificity of current designs of NSCLC therapeutics. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03494-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfe159d40376ac64bc4970c70d60376979b77ad" target='_blank'>
              Single-cell RNA-seq reveals gene expression heterogeneity in NSCLC and its link to the immune microenvironment
              </a>
            </td>
          <td>
            Jinzhuo Ren, Hua Zhang, Haiyong Wu
          </td>
          <td>2025-09-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="PURPOSE
The Tumor Microenvironment (TME) has a vital role in cancer survival and progression, and may play roles in drug resistance and immune escape. To date, few studies have detailed the TME of Ewing sarcoma (EwS).


EXPERIMENTAL DESIGN
We performed spatially resolved transcriptomics of primary treatment-naïve EwS tumor biopsies from patients with or without clinical metastasis, complemented by high-plex spatial proteomic analysis.


RESULTS
We discovered greater stromal enrichment in localized EwS primary tumors compared to metastasis-associated EwS primary tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumor role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling connectivity, suggesting a potential immune-evasive mechanism.


CONCLUSIONS
In addition to the immune recruitment role of tumor-associated stroma, our findings provide spatial insight into the TME of EwS and provide a rationale for the preclinical investigation of MIF as a potential target for Ewing sarcoma immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f9c77e0153cfc111b2db6ef0b69632696638603" target='_blank'>
              Spatial profiling identifies tumor-associated stroma enrichment and MIF as potential immunotherapy targets in primary Ewing sarcomas.
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-09-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Multiple myeloma (MM) is characterized by extensive intratumoral heterogeneity and complex interactions within the bone marrow microenvironment, yet the cellular and molecular drivers of treatment resistance remain poorly defined. Protein tyrosine phosphatase receptor gamma (PTPRG) has emerged as a candidate tumor suppressor in various malignancies by antagonizing proliferative and survival signaling, but its functional and prognostic relevance in MM has not been established.We analyzed 103,171 single‐cell transcriptomes from 18 MM samples (10 optimal responders [OR] and 8 suboptimal responders [SOR] to bortezomib–melphalan–prednisone) to investigate cell‐type composition, malignant plasma cell subclusters, and tumor–microenvironment crosstalk. InferCNV was used to distinguish malignant plasma cells, which were further reclustered and correlated with bulk prognostic phenotypes. Differential expression, pathway enrichment, transcription‐factor activity, pseudotime trajectory, and ligand–receptor interaction analyses were performed. Finally, bulk datasets (GSE9782, GSE2658, MMRF-CoMMpass) and in vitro knockdown assays in U266 and NCI-H929 cells were used to validate the prognostic and functional role of PTPRG.Eleven major cell types were annotated, with plasma cells, T/NK cells, and CD14+ monocytes predominating; SOR samples exhibited an expanded plasma‐cell fraction and reduced T/NK, CD14+ monocyte, pre-B, and HSPC populations. Among 35,944 malignant plasma cells, five subclusters were defined; one subcluster (MalPlasma3) was enriched in SOR samples and harbored 93.1% of cells associated with poor survival. MalPlasma3 and “worse‐survival” cells showed activation of stemness, E2F/MYC targets, and G2M checkpoint pathways, driven by transcription factors E2F8, E2F7, FOXM1, E2F1, and TIMELESS. Pseudotime analysis revealed a bifurcating differentiation toward a resistant phenotype, accompanied by upregulation of cell‐cycle and proliferation modules. In the OR group, enhanced cytotoxic features in NK, effector, and naïve T cells, along with IGF1–IGF1R and IFNG–IFNGR signaling, suggested a supportive microenvironment. In contrast to the known role as a tumor suppressor in solid and hematologic cancers, our integrative analyses identified PTPRG among seven stemness‐related genes upregulated in MalPlasma3 and poor‐survival cells, which was echoed in the observed reduced cell viability and increased apoptosis in MM cell lines following siRNA‐mediated PTPRG knockdown.This single‐cell multi‐omic dissection implicates a proliferative, stem‐like MalPlasma3 subcluster and identified PTPRG as a key mediator of drug resistance and poor outcome in MM, offering novel prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda63458271c1d78bb1529cdfe4855855ca64701" target='_blank'>
              Single-cell profiling uncovers PTPRG-driven stemness in malignant plasma cells and signatures of treatment failure in multiple myeloma
              </a>
            </td>
          <td>
            Jiewen Tan, Jinman Zhong, Yueping He, Yunman Xu, Chang Chen, Dan Xiong
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a common neoplasm that carries the risk of transformation into Richter’s syndrome (RS), a highly aggressive B cell lymphoma with poor prognosis. Limited availability of animal models and cell lines hinders understanding of transformation mechanisms. Addressing this gap, we established the first in silico dynamic model of the disease. Our methodology integrates mathematical logic modeling with experimental data to identify disease drivers, mechanisms, and potential therapeutic targets. We validated the model by comparing the model’s readout with experimental data from different biological levels, such as single-cell RNA sequencing analyses and a CLL/RS patient formalin-fixed paraffin-embedded (FFPE) tissue cohort. Our analyses identified BMI1 proto-oncogene and TP53 loss as key RS progression regulators. In addition, we performed an in silico target screening to identify promising target combinations in a personalized fashion. Through the synergy of mathematical modeling with experimental readouts, our model provides a complementary approach to investigate the process of CLL transformation to RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0084b1cbd03c2692e1100a97e2d57fe76c34d4" target='_blank'>
              CLL to Richter syndrome: Integrating network strategies with experiments elucidating disease drivers and personalized therapies
              </a>
            </td>
          <td>
            Julia Maier, J. Schwab, Silke D. Werle, Ralf Marienfeld, S. Stilgenbauer, Peter Möller, Nensi Ikonomi, Hans A. Kestler
          </td>
          <td>2025-09-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Background Penile squamous cell carcinoma (PSCC) is a rare yet potentially lethal malignancy, often resulting in devastating disfigurement, with a 5-year survival rate of only ∼50%. Human papillomavirus (HPV) infection is implicated in approximately half of PSCC cases and is associated with improved clinical outcomes; however, the underlying mechanisms remain poorly understood. Methods To elucidate HPV-associated differences in the tumor microenvironment, we performed single-cell RNA sequencing on tumors from 11 treatment-naïve PSCC patients, analyzing a total of 52 980 single cells. Unsupervised clustering identified 49 distinct cellular clusters across immune and stromal compartments. Results HPV-positive tumors exhibited an increased abundance of mast cells and a reduction in the proliferative macrophages subpopulation compared to HPV-negative tumors. Notably, CD8+ T cells in HPV-positive PSCC expressed lower levels of immune checkpoint molecules, suggesting a less exhausted immune state. Conversely, TIGIT and its ligands were significantly enriched in HPV-negative tumors, potentially fostering an immunosuppressive niche. Conclusion Collectively, our study delineates the single-cell landscape of PSCC and highlights distinct tumor microenvironment remodeling associated with HPV status, suggesting that the reduced immunosuppression in HPV-positive tumors may underlie their more favorable prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d170eff0ce2ddd0035f0702533123e38640014" target='_blank'>
              Single-cell transcriptomic profiling reveals distinct tumor microenvironments in HPV-associated penile squamous cell carcinoma
              </a>
            </td>
          <td>
            Lianbang Zhu, Deyun Shen, Jiahao Zhou, Chen Cheng, Zhiyao Xu, Yong Liang, Wen Pan, Tao Tao
          </td>
          <td>2025-09-01</td>
          <td>Precision Clinical Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Systems biology approaches have been applied to prostate cancer to model how individual cellular and molecular components interact to influence cancer development, progression, and treatment responses. The integration of multi-omic experimental data with computational models has provided insights into the molecular characteristics of prostate cancer and emerging treatment strategies that have the potential to improve patient outcomes. Here, we highlight recent advancements that have emerged from systems modeling in prostate cancer. These include descriptions of the molecular landscape of prostate cancer and how genomic alterations inform computational models of disease progression, how evolutionary processes give rise to mechanisms of therapeutic resistance, and the development of innovative treatment strategies such as adaptive therapy. We also highlight current challenges in prostate cancer that can be addressed through systems biology approaches. These include tumor heterogeneity, poor immunotherapy response, a paucity of experimental model systems, and the ongoing translation of computational models for clinical decision making. Leveraging systems biology approaches has the potential to lead to a better understanding of the disease and better patient outcomes in the treatment of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88aeffc4990ae5d3cf0820d80475621089e3b441" target='_blank'>
              Systems biology successes and areas for opportunity in prostate cancer
              </a>
            </td>
          <td>
            Michael V Orman, Laura S Graham, Scott D. Cramer, James C. Costello
          </td>
          <td>2025-08-01</td>
          <td>Endocrine-Related Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Thyroid cancer (TC) is now among the fastest-growing solid tumours, yet therapeutic gains remain limited for poorly differentiated, anaplastic and medullary variants whose median survivals are measured in months. Once guided chiefly by histology and single-gene assays, immunotherapy is being reshaped by single-cell profiling, which exposes the cellular mosaics that arbitrate response and resistance. Dissection of more than 150–000 tumour- and immune-cell transcriptomes has uncovered follicular-like, partial EMT-like and dedifferentiated thyrocyte states embedded within ‘hot’ (CD8hi IFN-γhi), ‘cold’ (CD8lo) and ‘excluded’ (stroma-walled) immune niches; these phenotypes correlate with PD-1/LAG-3 expression, macrophage polarisation and radio-iodine refractoriness. Functional studies reveal that SPP1–CD44 and GAS6–AXL crosstalk licenses epithelial–mesenchymal transition while VSIG4+ macrophages blunt cytotoxic T-cell activity, collectively undermining checkpoint blockade. Spatial transcriptomics corroborates these insights, mapping PD-L1-high tumour islets millimetres from CXCL13-rich tertiary lymphoid structures, whereas CITE-seq quantifies actionable checkpoints and cytokine receptors across patient biopsies. Emerging therapeutics mirror this granular knowledge: combinatorial PD-1 + LAG-3 inhibition, CSF-1R-directed macrophage re-programming and TSH-receptor-targeted CAR-T cells are advancing through early-phase trials, while ex-vivo single-cell pharmacotyping aligns drug cocktails with an individual’s tumour ecosystem. Early lenvatinib-pembrolizumab or selpercatinib-nivolumab trials show ~40% ORR but grade-3 hypertension >60%, prompting staggered-start designs. These advances sharpen pathogenetic resolution, refine patient selection and accelerate translational pipeline design. By integrating single-cell biology, immunology and endocrine oncology, this review identifies diagnostic blind spots, spotlights drug-repurposing opportunities and charts a roadmap toward personalised immunotherapeutic strategies capable of improving outcomes across the diverse spectrum of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6d6beff9a786418bc13757a09189c54eb993ef" target='_blank'>
              Personalised immunotherapy strategies informed by single cell profiling in thyroid cancer: a mini review
              </a>
            </td>
          <td>
            Ruyu Chen, Zhimin Wang, Xueying Chen, Yangling Huang, Fengnuan Zhang, Tianshu Gao
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a055b2099d39e961c79dc682be1c20375d7bba" target='_blank'>
              Tumor microenvironment response to neoadjuvant therapy in breast cancer: insights from single-cell and spatial omics.
              </a>
            </td>
          <td>
            Qingzhong Wu, Jiawei Yang, Da Zhang, Haowei Xu, Yuhang Yu, Yu Zheng
          </td>
          <td>2025-09-13</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Midkine (MDK), a multifunctional growth factor, has been implicated in promoting tumor progression, yet its role in glioblastoma (GBM) remains insufficiently characterized. To investigate MDK’s function in glioma, we integrated four RNA-Seq datasets into a harmonized cohort of 1,017 adult gliomas, including 256 GBM samples. We complemented this with freshly collected human GBM tissues and matched primary cell cultures to evaluate MDK expression and secretion patterns, further contextualized using single-cell RNA-Seq. Finally, we tested the impact of GBM-derived MDK on macrophage secretome composition to validate our in silico observations. We found that MDK expression increases with tumor grade in IDHwildtype gliomas, accompanied by a shift in isoform proportions favoring the canonical MDK transcript. High MDK expression was associated with poor prognosis specifically in GBM, where the MDKhigh subgroup comprised 75% of cases. MDKhigh GBMs exhibited a distinctive multiomic signature, including elevated chemokine and cytokine expression. Functionally, GBM-derived MDK induced macrophages to secrete multiple cytokines and chemokines, suggesting its role in reshaping the tumor microenvironment. Our findings reveal MDK’s previously underappreciated role in GBM aggressiveness and immune modulation, underscoring its potential as a biomarker and actionable therapeutic target for most GBM patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16253-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5822ef3a00105861887610dcb7da2c42fc7f87" target='_blank'>
              Coordinated changes in midkine expression and midkine-associated multiomic profile in glioma microenvironment
              </a>
            </td>
          <td>
            Mieszko Lachota, K. Zielniok, Agata Gozdz, Patrycja Szpak, Ilona Kalaszczyńska, Radosław Zagożdżon
          </td>
          <td>2025-08-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Tumor‐associated macrophages (TAMs) are prominent constituents of solid tumors, and their prevalence is often associated with poor clinical outcomes. These highly adaptable immune cells undergo dynamic functional changes within the immunosuppressive tumor microenvironment (TME), engaging in reciprocal interactions with malignant cells. This bidirectional communication facilitates concurrent phenotypic transformation: tumor cells shift toward invasive mesenchymal states, whereas TAMs develop immunosuppressive, pro‐tumorigenic traits. Increasing evidence highlights metabolic reprogramming, characterized by dysregulation of lipid metabolism, amino acid utilization, and glycolytic activity, as the fundamental molecular basis orchestrating this pathological symbiosis. However, a comprehensive understanding of how metabolic reprogramming specifically coordinates the mutual polarization of tumor cells and TAMs is lacking. This review thoroughly examines the molecular mechanisms governing this co‐polarization process, detailing critical transcriptional regulators, essential signaling pathways, and the maintenance of adaptive phenotypes within the TME. Furthermore, this review critically assesses promising therapeutic strategies aimed at disrupting this alliance, including the use of metabolically targeted agents, engineered chimeric antigen receptor macrophages, and TAM‐selective nanoparticle delivery systems. These insights provide a crucial foundation for the development of next‐generation cancer immunotherapies focused on reprogramming pathological polarization dynamics to overcome treatment resistance and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7946cf3691363a365260b427222e670775d573c5" target='_blank'>
              Polarization of Tumor Cells and Tumor‐Associated Macrophages: Molecular Mechanisms and Therapeutic Targets
              </a>
            </td>
          <td>
            Guohao Wei, Bin Li, Mengyang Huang, Mengyao Lv, Zihui Liang, Chuandong Zhu, Lilin Ge, Jing Chen
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f63086cbccde700980caecaca85ce93b724792a" target='_blank'>
              Neoadjuvant immunotherapy promotes the formation of mature tertiary lymphoid structures in a remodeled pancreatic tumor microenvironment.
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, J. Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, Genevieve L. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, E. Jaffee, Lei Zheng, W.J. Ho, Robert A. Anders, E. Fertig, L. T. Kagohara
          </td>
          <td>2025-08-15</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Objective
Advanced gastric cancer remains highly refractory to therapy, with limited immunotherapy efficacy due to tumor microenvironment heterogeneity. Primary cilia, microtubule-based organelles involved in tumor progression, remain insufficiently explored in gastric cancer. This study aimed to define primary cilia subtypes and establish prognostic signatures for personalized treatment strategies.


Methods
Bulk transcriptomic data from over 1,500 gastric cancer samples were integrated to define distinct primary cilia subtypes. A primary ciliary phenotype-associated signature (PCS) was established using a multi-machine learning survival framework incorporating ten algorithms. The prognostic predictive value and immunotherapy response prediction capability of PCS were validated across multiple independent cohorts. Single-cell RNA sequencing analysis was performed to identify cellular populations associated with high-PCS phenotype. Causal weighted gene co-expression network analysis (WGCNA) was employed to identify driving factors, followed by functional validation through cell culture experiments and xenograft models.


Results
Two distinct primary cilia subtypes were identified and validated across all cohorts, with C2 patients exhibiting significantly worse overall survival compared to C1 patients. PCS demonstrated robust predictive value for both prognosis and immunotherapy response, with superior accuracy compared to existing models across multiple validation cohorts. High-PCS patients showed reduced tumor purity, increased stromal cell infiltration, and poor response to immunotherapy. Single-cell analysis revealed that fibroblasts had the highest PCS scores and identified a novel secreted modular calcium-binding protein 2 (SMOC2)high myofibroblastic cancer-associated fibroblast (mCAF) population as the key driver of high-PCS phenotype. Functional experiments confirmed that SMOC2 knockdown significantly suppressed gastric cancer cell proliferation, migration, and invasion, while promoting mCAF-to-inflammatory cancer-associated fibroblasts (iCAF) transition.


Conclusions
PCS serves as a robust prognostic biomarker for gastric cancer patients. Additionally, targeting SMOC2 high mCAFs represents a potential therapeutic strategy for patients with high-PCS gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8325aa5e13eb71b8edb3f16c8e7ba4db0f0ba8" target='_blank'>
              SMOC2 high myofibroblastic cancer-associated fibroblast drives primary cilia-associated tumor microenvironment remodeling and poor prognosis in gastric cancer.
              </a>
            </td>
          <td>
            Qiqi Xu, Changjiang Yang, Jingyuan Ning, Yunze Niu, Xuesong Zhao, Long Zhao, Caihong Wang, Shan Wang, Yingjiang Ye, Zhanlong Shen
          </td>
          <td>2025-08-30</td>
          <td>Chinese journal of cancer research = Chung-kuo yen cheng yen chiu</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Prostate cancer is the most common non-cutaneous cancer among men in the United States. Most prostate cancers are driven by androgen receptor (AR) signaling, but there are an increasing number of cases that lose AR and gain neuroendocrine (NE) features (AR−/NE+) or lack both (AR−/NE−). These latter subtypes are particularly aggressive and lethal. Extracellular vesicles (EVs) have shown great potential as biomarkers for noninvasive liquid biopsy assays, as EVs contain biomolecules from their cells of origin. Here, we used a shotgun proteomics approach with mass spectrometry to interrogate the global proteome of EVs isolated from prostate cancer cell lines reflecting diverse clinical subtypes, including AR−/NE+ and AR−/NE− models. We identified 3,952 EV proteins, which clustered largely by tumor subtype and provided enough proteomic coverage to derive classic gene signatures of AR or NE identity that are of high relevance for prostate cancer prognostication. EVs isolated from AR+ cells displayed high levels of proteins regulated by AR and mTOR signaling. EVs isolated from AR−/NE+ cells contained known NE markers such as SYP and CHGA, whereas EVs from AR−/NE− models were enriched in basal cell markers and proteins that regulate epithelial-to-mesenchymal transition (EMT). We integrated our cell line data with recently published EV proteomics data from 27 advanced prostate cancer patients and found 2,733 overlapping proteins including cell surface markers relevant to prostate cancer, AR activity indicators, and proteins enriched in specific subtypes (AR+, AR−/NE−, AR−/NE+). This approach is especially promising for rare cancer subtypes, such as prostate cancers that lose AR−related features and gain NE features, so as to optimize the use of these liquid biopsy samples for clinical decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3560bbe96c84bfbb639bc1eefd901706a0b01ad" target='_blank'>
              Proteomic profiling of extracellular vesicles distinguishes prostate cancer molecular subtypes
              </a>
            </td>
          <td>
            Megan L Ludwig, Ali T Arafa, Saasha Vinoo, Jason C Jones, Abderrahman Day, Hannah E Bergom, Z. Sychev, A. Horrmann, Nicholas M Levinson, S. Dehm, Emmanuel S. Antonarakis, Justin Hwang, J. Drake
          </td>
          <td>2025-08-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Fibroblasts are critical mediators of tumor progression and metastasis; however, their heterogeneity and specific functions in the context of colorectal cancer (CRC) liver metastases remain incompletely understood.We performed single-cell RNA sequencing (scRNA-seq) analysis on relevant tissue samples. Subsequently, we validated our findings using immunofluorescence assays on histological slides from primary CRC with liver metastasis (mCC) and liver metastatic tumors (mLC). Further investigations included transcriptomic profiling, pseudotime trajectory analysis, multiplex immunofluorescence staining, and cell-cell communication analysis.Our scRNA-seq analysis identified a distinct PRELP-positive cancer-associated fibroblast (CAF) subtype that is associated with liver metastasis and tumor progression. These PRELP+ CAFs were predominantly enriched in mLC, less abundant in mCC, and rare in non-metastatic CRC (nCC). This distribution was confirmed by immunofluorescence. Transcriptomically, PRELP+ CAFs exhibit a unique signature defined by extracellular matrix components (e.g., PRELP, COLEC11, ITGBL1) and the activation of pro-tumor pathways such as TGF-β and Wnt signaling. Pseudotime analysis indicated they represent a terminal fibroblast differentiation state. Spatially, they colocalize with immune cells (T cells, B cells, plasma cells), and communication analysis suggests they foster an immunosuppressive microenvironment via APP-CD74 and collagen-CD44 signaling, thereby promoting immune evasion. The transcription factors NR2F2, JUN, and JUND were identified as key regulators of this CAF subtype.These findings provide crucial new insights into fibroblast heterogeneity within CRC liver metastases. We characterize PRELP+ CAFs as a specialized, terminally differentiated fibroblast population that contributes to immunosuppression and tumor progression, highlighting them as a potential therapeutic target for inhibiting metastatic advancement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62bf8481f437e703e061482f9031f26e35dec625" target='_blank'>
              Characterization of the phenotype and function of PRELP+ fibroblast subtype in liver metastatic colorectal cancer
              </a>
            </td>
          <td>
            Yuting Dai, Xingying Huang, Min Sun, Shiyu Zhang, Weiqiang Yu, Kongwang Hu, Qiang Wu, Qingfa Wu
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053a87c896e7b2228f019eac1392494ac26a840e" target='_blank'>
              A spatial map of MAPK-activated immunosuppressive myeloid populations in pediatric low-grade glioma.
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Romain Sigaud, Evan Puligandla, Bridget Liu, E. Karimi, Alva Annett, M. Rezanejad, Wajih Jawhar, Robert Taylor, Yi Cao, S. Schmid, F. Selt, P. Hernáiz Driever, Svea Horn, P. Sievers, M. Prinz, M. Glatzel, Christian Mawrin, C. Hartmann, C. Monoranu, Liza Konnikova, Felix Sahm, Stefan M. Pfister, Olaf Witt, David T. W. Jones, A. Koch, Claudia L Kleinman, David Capper, L. Walsh, Nada Jabado, T. Milde
          </td>
          <td>2025-09-15</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Intratumor heterogeneity is a hallmark of cancer, enabling subpopulations of cells to evade therapy, adapt to immune attack, and thrive in diverse microenvironments. While retrospective genomic and epigenomic analyses have mapped the large-scale histories of tumor evolution, they cannot capture the rapid, dynamic changes in cell state that occur as individual cells divide. Panagopoulos and colleagues recently developed a cellular platform to monitor the role of transient replication stress in real time, tracking sister cells as they divide and replicate. The authors use these techniques to show that daughter and granddaughter cells can inherit very different states, often leading to further cellular instability. This work broadens our understanding of how diverse cell states arise from oncogenic stress and how cellular heterogeneity emerges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beadadba41d03604ec2c3cef748b8a0844e1482c" target='_blank'>
              Generations in Flux: Tracking the Birth of Cellular Heterogeneity in Cancer.
              </a>
            </td>
          <td>
            Aaron McKenna
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite notable advances with immune checkpoint inhibitors (ICIs) in esophageal squamous cell carcinoma (ESCC), their clinical efficacy remains limited, largely due to CD8⁺T cell exhaustion. Among these, progenitor exhausted T cells (Tpex) represent a key subset with stem cell–like features that sustain durable anti-tumor immunity.We applied multi-color immunohistochemistry (mIHC) to determine the spatial distribution and clinical significance of Tpex cells within the tumor microenvironment (TME) of ESCC. Publicly available single-cell RNA sequencing (scRNA-seq) datasets were further analyzed to characterize Tpex cell phenotypes, differentiation trajectories, and intercellular communication networks.Tpex cells constituted a distinct subset of infiltrating CD8⁺T cells and represented a transitional stage of the exhaustion continuum. A higher degree of Tpex infiltration was significantly associated with improved overall survival in ESCC patients. Moreover, scRNA-seq data from patients treated with PD-1 blockade revealed that responders harbored markedly enriched Tpex populations compared with non-responders.Our findings identify Tpex cells as a critical prognostic and immunologically active CD8⁺T cell subset in ESCC. Their abundance and functional engagement are closely associated with favorable clinical outcomes and response to PD-1 blockade. Furthermore, their stem cell-like properties may be pivotal in shaping durable anti-tumor immunity and could provide novel therapeutic targets to enhance the efficacy of PD-1-based immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08c5eac76128741742f8a923e04cbd692a6d7928" target='_blank'>
              Progenitor-exhausted T cell as prognostic indicator in esophageal squamous cell carcinoma: illuminating their key contribution to tumor immunity
              </a>
            </td>
          <td>
            Yi Liu, Hongwei Jiang, Zhang Fang, Bin Xu, Junjun Chen, Xiao Zheng, Renhao Geng, Lujun Chen
          </td>
          <td>2025-09-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract BACKGROUND A major barrier in our understanding of glioblastoma (GBM) is the difficulty in tracking tumour development in real-time. We adopted a novel approach that combines the principles of classic cellular barcoding with CRISPR/Cas-9 technology and single-cell RNA sequencing known as continuous lineage tracing to apply a phylogenetic approach to studying tumour development. METHODS Patient derived glioma initiating cell lines were engineered with expressed DNA barcodes with CRISPR/Cas-9 targets and subsequently implanted to create an intracranial xenograft model. Tumors were sent for single cell RNA sequencing; clonal relationships were surmised through identification of expressed barcodes, and cells were characterized by their transcriptional profiles. Phylogenetic lineage trees were created utilizing lineage reconstructive algorithms to define cell fitness and expansion. RESULTS Our work has revealed a significant amount of intra-clonal cell state heterogeneity, suggesting that tumour cells engage in phenotype switching prior to therapeutic intervention. We defined a consistent transcriptional pattern for tumour engraftment and in vivo clonal advantage. Phylogenetic lineage trees allowed us to define gene signatures of both cell fitness and expansion, which correlate strongly with published neural-mesenchymal and developmental-injury response phenotypes. GBMs exist along a transcriptional gradient between undifferentiated but “high-fit” cells and terminally differentiated, “low-fit” cells: cells with highly fitness appear to represent cells undergoing cell state transition. CONCLUSION We successfully employed a novel lineage tracing technique in GBM, creating a powerful tool for real-time tracing of tumour growth through the analysis of a unique set of highly detailed transcriptional data with associated clonal and phylogenetic relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51825ee39f0efc909dc234d3cc75f48f8c3aa8f6" target='_blank'>
              56 SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS AND EXPANSION AXES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Ajisebutu, F. Fan, Chloe Gui, Y. Ellenbogen, A. Landry, Jeff Liu, Q. Wei, G. Zadeh
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The 3D organization of the genome plays a critical role in regulating gene expression, maintaining cellular identity, and mediating responses to environmental cues. Advances in super-resolution microscopy and genomic technologies have enabled unprecedented insights into chromatin architecture at nanoscale resolution. However, the complexity and volume of data generated by these techniques necessitate innovative computational strategies for effective analysis and interpretation. In this review, we explore the transformative role of deep learning in the analysis of 3D genome organization, highlighting how deep learning models are being leveraged to enhance image reconstruction, segmentation, and dynamic tracking in chromatin research. We provide an overview of deep learning-enhanced methodologies that significantly improve spatial and temporal resolution of images, with a special focus on single-molecule localization microscopy. Furthermore, we discuss deep learning’s contribution to segmentation accuracy, and its application in single-particle tracking for dissecting chromatin dynamics at the single-cell level. These advances are complemented by frameworks that enable multimodal integration and interpretability, pushing the boundaries of chromatin biology into clinical diagnostics and personalized medicine. Finally, we discuss emerging clinical applications where deep learning models, based on chromatin imaging, aid in disease stratification, drug response prediction, and early cancer detection. We also address the challenges of data sparsity, model interpretability and propose future directions to decode genome function with higher precision and impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33be16632331d98ff0665f21e81c0af12092a8a7" target='_blank'>
              Deep learning in chromatin organization: from super-resolution microscopy to clinical applications
              </a>
            </td>
          <td>
            Mikhail Rotkevich, Carlotta Viana, M. V. Neguembor, M. Cosma
          </td>
          <td>2025-08-29</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary This systematic review explores the role of tumor-infiltrating B lymphocytes (TIBLs) in the tumor microenvironment (TME) of colorectal cancer, addressing a gap in the immuno-oncology literature. A comprehensive literature search spanning 2000 to 2025 identified 32 eligible studies involving 5766 patients, in accordance with PRISMA guidelines. The findings revealed multiple B cell subsets, including plasma cells, memory B cells, and regulatory B cells. These subsets exhibited distinct roles and prognostic implications in the progression of colon and rectal cancer. The review also highlighted methodological variability in TIBL identification across studies. Furthermore, it identified immunological pathways through which TIBLs may influence carcinogenesis, suggesting their potential as novel immunotherapy targets. Overall, the review emphasizes the prognostic and therapeutic relevance of TIBLs, supporting their future use as biomarkers for personalized treatment strategies in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56323f27757ccb518fba0bc56470e1399b4e8b61" target='_blank'>
              The Role of Tumor-Infiltrating B Lymphocytes in Colorectal Cancer Patients: A Systematic Review of Immune Landscape Evolution
              </a>
            </td>
          <td>
            G. Fagarasan, V. Făgărăsan, V. Bintintan, G. Dindelegan
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Neuroblastomas encompass malignant cells with varying degrees of differentiation, ranging from adrenergic (adr) cells resembling the sympathoadrenal lineage to undifferentiated, stem-cell-like mesenchymal (mes) cancer cells. Relapsed neuroblastomas, which often have mesenchymal features, have a poor prognosis and respond less to anticancer therapies, necessitating the development of novel treatment strategies. To identify novel treatment options, we analyzed the sensitivity of 91 pediatric cell models, including patient-derived tumoroid cultures, to a drug library of 76 anti-cancer drugs at clinically relevant concentrations. This included 24 three-dimensionally cultured neuroblastoma cell lines representing the range of mesenchymal to adrenergic subtypes. High-throughput ATP-based luminescence measurements were compared to high-content confocal imaging. With machine learning-supported imaging analysis, we focused on changes in the lysosomal compartment as a marker for therapy-induced senescence and assessed the basal lysosomal levels in a subset of untreated mesenchymal versus adrenergic cells. We correlated these findings with pathway activity signatures based on bulk RNA and scRNAseq. Comprehensive image-based synergy screens with spheroid cultures validated the combined effects of selected drugs on proliferation and cytotoxicity. Mesenchymal models presented high basal lysosomal levels correlating with senescence-associated secretory phenotype (SASP) and sphingolipid metabolism pathways. Chemotherapy treatment further increased lysosome numbers, indicative of therapy-induced senescence. Furthermore, the mesenchymal subtypes correlated with MAPK activity and sensitivity to MAPK pathway inhibitors. Lysosomal and SASP signaling is druggable by inhibitors of lysosomal acid sphingomyelinase (SLMi) or senolytics, including BCL2-family inhibitors. Especially the sequential combination of MEK inhibitors (MEKi) with BCL2-family inhibitors was the most effective on relapsed neuroblastoma cell lines. Gene expression analysis of 223 patient samples, drug sensitivity profiling of five patient-derived fresh tissue cultures, and in vivo zebrafish embryo neuroblastoma xenograft models confirmed these findings. Inhibition of MAPK signaling in combination with BCL2-family inhibitors is a novel treatment option for patients suffering from relapsed neuroblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0577cfd3155a7a66d89cff76645a273955a12c7e" target='_blank'>
              High content-imaging drug synergy screening identifies specific senescence-related vulnerabilities of mesenchymal neuroblastomas
              </a>
            </td>
          <td>
            Sonja Herter, Marta Emperador, Kyriaki Smyrilli, Daniela Kocher, Simay Celikyürekli, Constantia Zeiser, Xenia F. Gerloff, Sina Kreth, K. Henrich, Kendra K. Maaß, Johanna Rettenmeier, T. Grünewald, H. Peterziel, Frank Westermann, A. Hamacher-Brady, Olaf Witt, Ina Oehme
          </td>
          <td>2025-08-25</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="ABSTRACT Cancer cell evasion of therapy is a highly adaptive process that undermines the efficacy of many treatment strategies. A significant milestone in the study of these mechanisms has been the advent of pooled CRISPR knockout screens, which enable high-throughput, genome-wide interrogations of tumor dependencies and synthetic lethal interactions, advancing our understanding of how cancer cells adapt to and evade therapies. However, the utility of this approach diminishes when applied to dynamic biological contexts, where processes are transient and sensitivity to routine cell culture manipulations that introduce noise and limit meaningful discoveries. To overcome these limitations, we present RESTRICT-seq, a next-generation pooled screening methodology that restricts Cas9 nuclear activation in controlled, repeated cycles. By confining Cas9 catalytic activity to strict temporal windows, RESTRICT-seq mitigates undesired fitness penalties that routinely accumulate throughout pooled screens. When benchmarked against conventional pooled screens and standard inducible protocols, RESTRICT-seq revealed significantly fewer divergent cell clones and increased signal-to-noise ratio, overcoming a key limitation of traditional methods. Leveraging RESTRICT-seq, we conducted a comprehensive functional survey of the druggable mammalian epigenome, uncovering several elusive epigenetic drivers of treatment resistance in cutaneous squamous cell carcinoma (cSCC). This revealed PAK1 as a previously unrecognized mediator of cSCC resistance in human and mouse SCC, offering new insights into a prognostic marker and therapeutic target of high clinical significance. Our findings establish RESTRICT-seq as a powerful tool for extending the applicability of pooled CRISPR screens to dynamic and previously intractable biological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ccdefb21a27947026277d5009da2011b09160aa" target='_blank'>
              RESTRICT-seq enables time-gated CRISPR screens and uncovers novel epigenetic dependencies of SCC resistance
              </a>
            </td>
          <td>
            Selahattin Can Ozcan, Dreyton G Amador, Justin Anthony Powers, Ashley G. Njiru, Zahra Ansari, Yvon Woappi
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Interpatient tumor heterogeneity manifests as multimodal distributions across genomic, transcriptomic, and microenvironmental profiles. This fundamentally violates the unimodal assumption of conventional machine learning models, impairing immune checkpoint blockade (ICB) response prediction. To resolve this limitation, we propose a heterogeneity-optimized framework that applies K-means clustering to stratify patients into biologically distinct hot-tumor and cold-tumor subgroups, demonstrating superiority over hierarchical/DBSCAN clustering. Subsequently, heterogeneity-optimized predictive models–a support vector machine for hot-tumor subtypes and a random forest for cold-tumor subtypes–were developed utilizing seven heterogeneity-associated biomarkers to circumvent unimodal constraints. The proposed model significantly enhances ICB response prediction in melanoma, NSCLC, other cancer types, and pan-cancer datasets, achieving a mean accuracy gain of at least 1.24% compared to 11 baseline methods. This performance improvement was consistently validated in an independent external cohort. Furthermore, the approach enables biologically interpretable precision immunotherapy by explicitly modeling multimodal heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea2e7a884fb718694551d16dff86f3e29cf9498" target='_blank'>
              Heterogeneity-optimized method for predicting immune checkpoint blockade response
              </a>
            </td>
          <td>
            Juan Liang, Qihang Guo, Shan Xiang, Yun Wang, Juntao Li
          </td>
          <td>2025-09-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Uveal melanoma (UM), the most common primary intraocular malignancy in adults, presents significant clinical challenges due to its high metastatic potential, pronounced hepatic tropism, and poor prognosis upon systemic dissemination. Despite established local therapies, nearly half of patients develop distant metastases, highlighting an urgent need for more effective systemic strategies. Recent advances in single-cell omics technologies (e.g., scRNA-seq, scATAC-seq, spatial transcriptomics) have revolutionized our understanding of UM pathobiology. These approaches have meticulously delineated the complex tumor heterogeneity, immunosuppressive microenvironment, and key molecular drivers—including novel macrophage subsets (e.g., immunosuppressive MΦ-C4), senescent endothelial cells, and non-canonical immune checkpoint expression—providing unprecedented resolution for identifying actionable therapeutic targets. Concurrently, innovations in materials science and biomedical engineering offer transformative opportunities for precision therapy. Engineered nanocarriers, biodegradable implants, and advanced gene therapy vectors (e.g., tropism-enhanced AAVs, CRISPR-Cas9 systems) enabled targeted drug delivery, sustained release, and genetic modulation tailored to the eye’s unique anatomy and immune privilege. This review synthesizes these converging frontiers, outlining how the integration of multi-omics insights with smart biomaterials can overcome current therapeutic limitations. We catalog emerging material-based platforms applicable to UM and summarize validated molecular targets (e.g., GNAQ/GNA11, YAP/TAZ, BAP1, c-Met, CXCR4). Furthermore, we propose an interdisciplinary paradigm spanning combinatorial targeted therapies, immunomodulation, minimally invasive devices (e.g., robotic radiosurgery), and engineered delivery systems. By bridging mechanistic discovery with translational engineering, this synergy holds significant promise for advancing precision medicine and improving clinical outcomes in UM, ultimately facilitating the transition from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0323cee7d6c89a49fe569ae452832b7a890d38bd" target='_blank'>
              Integrating single-cell omics and materials science for uveal melanoma: from mechanistic insights to precision therapeutics
              </a>
            </td>
          <td>
            Shouyong Fu, Changfei Li
          </td>
          <td>2025-09-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c0faec4d04ddee86bb924ee79c08f0dc4ced64" target='_blank'>
              Cancer-associated fibroblasts drive lung adenocarcinoma progression via THBS2-mediated epithelial-mesenchymal transition.
              </a>
            </td>
          <td>
            Yuqing Ren, Ruijie Ming, Anning Zuo, Shutong Liu, Yuhao Ba, Yuyuan Zhang, Yukang Chen, Teng Pan, Peng Luo, Quan Cheng, Jinhai Deng, Yi Yue, Hui-Xian Xu, Siyuan Weng, Xinwei Han, Dongdong Zhou, Zaoqu Liu
          </td>
          <td>2025-09-19</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ABSTRACT Multiple myeloma (MM) orchestrates a profound disruption of immune balance within the bone marrow (BM) microenvironment, driving disease progression and therapeutic resistance. To better understand these complex immune dynamics, we used high-dimensional mass cytometry (CyTOF) profiling to comprehensively characterize the immune landscape of the BM across different stages of myeloma progression, including MGUS (n = 16), smoldering MM (SMM; n = 25), and active MM, both newly diagnosed (n = 43) and relapsed/refractory (n = 104). Our analysis revealed substantial immune remodeling during disease progression, characterized by adaptive immune suppression and extensive infiltration of innate immune populations. Transformation from MGUS to SMM was marked by significant alterations in central and effector memory T helper cells, effector cytotoxic T cells, and an enrichment of monocytic and neutrophil subsets. Active MM stages were further distinguished by increased expansion of myeloid and monocytic lineages, alongside a pronounced reduction in progenitor and transitional B cells. Correspondence analysis demonstrated that specific immune profiles were significantly associated with clinical outcomes, including progression-free survival and overall survival. This study highlights the potential of CyTOF-based molecular profiling to unravel the intricate immune dynamics of the BM microenvironment across MM disease stages, enhancing our understanding of MM pathogenesis and providing a foundation for identifying prognostic biomarkers and tailoring precision immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7c28c48facee0fbd69c0cc410926fbb87e89302" target='_blank'>
              CyTOF profiling of bone marrow immune dynamics across myeloma stages
              </a>
            </td>
          <td>
            Danka Cholujová, G. Beke, Ľuboš Kľučár, Ľ. Drgoňa, Zuzana Valuskova, M. Leiba, E. Kastritis, David M. Dorfman, Kenneth C. Anderson, J. Jakubikova
          </td>
          <td>2025-08-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) is a critical determinant of tumor progression and therapeutic response in gliomas. While pediatric gliomas have historically been treated using strategies derived from management of adult gliomas, emerging evidence reveals that pediatric gliomas possess a unique TME. The pediatric TME is distinct, characterized not only by differences in cellular composition, but a lower mutational burden, diminished neoantigen presentation, and heightened immunosuppressive activity. The unique immune landscape, developmental trajectories, and immune escape mechanisms in the pediatric TME create barriers to effective therapy. Recent studies show promising results in novel and advanced therapeutic strategies, highlighting the potential for innovative immunotherapeutic approaches. Advances in methodologies for modeling the TME, including computational approaches and animal-based models, provide new insights into pediatric glioma biology. Utilization of computational models may provide opportunities to predict tumor response to specific therapies and tailor immunotherapy regimes to individuals, allowing for personalized care. Leveraging the unique features of the pediatric TME offers an opportunity to overcome current barriers to immunotherapy and develop more effective, age- and tumor-specific treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d12c4d7c16cb4f880257014b09a937ace0732f5a" target='_blank'>
              Targeting the Tumor Microenvironment in Pediatric Gliomas: Advances and Future Directions in Immunotherapy
              </a>
            </td>
          <td>
            Cheyenne Ahamed, Lam Nguyen, Cayley S Brock, Ayla Farzamnia, Pierrick Millet, Keisaku Sato, Kevin K Kumar
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) constitute a heterogeneous population of cells within the tumor microenvironment and are associated with cancer development and drug resistance. The absence of a universal classification for CAFs hinders their research and therapeutic targeting. To define CAF phenotypes, we developed patient-derived cell cultures of breast cancer (BC) and validated and characterized four distinct CAF subtypes (S1–S4) by Costa’s classification. Three out of five primary cell cultures of BC demonstrated different functional features rather than fixed cellular states due to the plasticity of the CAF phenotype. CAF crosstalk with cancer cells supported their survival in the presence of anticancer drugs. Based on the analysis of the cytotoxic effect of doxorubicin, cisplatin and tamoxifen, it was demonstrated that CAF-S4 and CAF-S1 cells were sensitive to the action of all drugs investigated, despite the fact that they possessed different mechanisms of action. CAF-S2 cells exhibited the highest level of resistance to the antitumour agents. Homotypic and heterotypic spheroids with CAFs could be used to model the fibrotic area of BC in vitro. The patient-derived cell cultures of CAFs formed spheroids. Hypoxia-activated CAF-S4 have been shown to stimulate the metastatic potential of triple-negative BC cells in a heterotypic spheroid model. Consequently, this study could be a starting point for the development of novel therapeutic strategies that target CAFs and their interactions with cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca2b5723346e98367fd01163fd140d5d1bf9c665" target='_blank'>
              Exploring the Heterogeneity of Cancer-Associated Fibroblasts via Development of Patient-Derived Cell Culture of Breast Cancer
              </a>
            </td>
          <td>
            Anna Ilyina, A.A. Leonteva, Ekaterina Berezutskaya, M.M. Abdurakhmanova, M. Ermakov, S. Mishinov, E. Kuligina, Sergey Vladimirov, Maria Bogachek, Vladimir Richter, A. Nushtaeva
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is complex and dynamic, and changes significantly with tumour progression. Studying the evolving state of T cells, especially tumour-specific subsets, has become feasible. However, the roles of exhausted T cells (Tex) and pre-exhausted tissue-resident memory T cells (pf-Trm), which emerge after prolonged antigen stimulation, remain unclear. Using single-cell sequencing data, we analyzed the immune landscape of patients with colorectal cancer (CRC) across clinical stages to quantify the abundance of T cell subtypes. Functional enrichment analysis revealed that early stage Tex cells retained some functionality, whereas advanced stage Tex cells showed a significant functional loss. Early stage pf-Trm cells actively participate in immune surveillance and antigen presentation, whereas advanced stage pf-Trm cells exhibit reduced functions. Flow cytometry analysis of clinical cohorts was used to measure the proportions of Tex and pf-Trm. Elevated levels of PD-1 and Tim-3 have been detected in TILs from CRC patients. Data from The Cancer Genome Atlas (TCGA) linked high Tex levels to poor prognosis in CRC, while pf-Trm correlated with better outcomes in early CRC but worse outcomes in advanced CRC due to functional exhaustion. Thus, Tex and pf-Trm cells may serve as prognostic biomarkers, and Tim-3 and CD103 may be promising targets for immune checkpoint inhibitors. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-14409-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08110fc857bf45d38a2c72215e91f6b78e6d67e2" target='_blank'>
              Role of T cell exhaustion and tissue-resident memory T cells in the expression and prognosis of colorectal cancer
              </a>
            </td>
          <td>
            Han Wu, Pei-Wen Fan, Ya-Ning Feng, Cheng Chang, Ting Gui, Jia-bei Meng, Ruo-Zheng Wang
          </td>
          <td>2025-08-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77d17a800b951a14fec481a0422bbf1125414c43" target='_blank'>
              NOX1 and NPY1R mark regional colon stem cell populations that serve as cancer origins in vivo.
              </a>
            </td>
          <td>
            Maxime Gasnier, Tanysha Chi-Ying Chen, Swathi Yada, Sowmya Sagiraju, Yusuke Yoshikawa, Stefano Perna, H. Lim, Bernett Lee, Nick Barker
          </td>
          <td>2025-09-02</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Advances in treating primary breast tumors and extracranial metastases have inadvertently led to a rise in brain metastases, which are severe and carry bleak prognostic outcomes. Thus, there is an urgent need to investigate the cellular and molecular drivers of metastatic central nervous system (CNS) tumors. The tumor microenvironment (TME) is critically important for metastasis, as metastatic cells must alter their new environment to survive and grow. Our laboratory's recent studies have pinpointed platelet-derived growth factor receptor beta (PDGFRβ) signaling as a critical pathway in promoting breast cancer-associated brain metastases (BCBM). Key findings from this published work involved data emanating from a genetically engineered mouse model (GEMM) with hyperactive PDGFRβ-D849V in the mesenchymal lineage driven by Fsp1-cre. These mice exhibit increased BCBM when injected with murine mammary tumor cells secreting the ligand PDGFB, making them an effective model for studying BCBM. However, hyperactivity of PDGFRβ in these mice occurs in all mesenchymal cells. To investigate cell-specific PDGFRβ functions in the CNS, we analyzed publicly available single-cell RNA-seq data of the adult murine brain. This analysis identified astrocyte populations expressing both Pdgfrb and the Fsp1/S100A4 mesenchymal lineage marker. To probe the role of PDGFRβ in reactive astrocytes, we developed a Gfap-cre GEMM model crossed with our mutant PDGFRβ hyperactivation allele (Gfap-cre;Pdgfrb-D849V). These mice exhibit increased GFAP expression in the absence of tumor and decreased overall survival upon intracardiac injection of PDGFB secreting breast cancer cells. Furthermore, flow cytometry at day 10 post intracardiac injection indicates potential differences in the immune response of the brain TME compared to controls. Current work is focused on identifying specific transcriptomic changes in astrocyte populations due to both PDGFRβ hyperactivation as well as metastatic breast cancer colonization of the brain. Overall, this work aims to identify potential therapeutic targets to treat BCBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/065fae0e35727b10f37e95370e754abb9b46417c" target='_blank'>
              BSBM-08 ASTROCYTE PDGFRβ HYPERACTIVATION IN BREAST CANCER BRAIN METASTASIS
              </a>
            </td>
          <td>
            Nathaniel S Grabinski, Johnathon Scheibel, Jesse Reardon, Paul Lee, Jonathan Godbout, S. Sizemore, Jessica Winter, Gina M. Sizemore
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background/Objectives: Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with poor prognosis. T cell exhaustion (TEX) is a key factor in tumor immune evasion and therapeutic resistance. In this study, we integrated single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data to characterize TEX-related transcriptional features in HCC. Methods: We first computed TEX scores for each sample using a curated 65-gene signature and classified them into high-TEX and low-TEX groups by the median score. Differentially expressed genes were identified separately in scRNA-seq and bulk RNA-seq data, then intersected to retain shared candidates. A 26-gene prognostic signature was derived from these candidates via univariate Cox and LASSO regression analysis. Results: The high-TEX group exhibited increased expression of immune checkpoint molecules and antigen presentation molecules, suggesting a tumor microenvironment that is more immunosuppressive but potentially more responsive to immunotherapy. Functional enrichment analysis and protein–protein interaction (PPI) network construction further validated the roles of these genes in immune regulation and tumor progression. Conclusions: This study provides a comprehensive characterization of the TEX landscape in HCC and identifies a robust gene signature associated with prognosis and immune infiltration. These findings highlight the potential of targeting TEX-related genes for personalized immunotherapeutic strategies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a1103280627c731b735d281df63ab3e636c5d0" target='_blank'>
              Bulk and Single-Cell Transcriptomes Reveal Exhausted Signature in Prognosis of Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Ruixin Chun, Haisen Ni, Ziyi Zhao, Chunlong Zhang
          </td>
          <td>2025-08-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor-derived extracellular vesicles (TEVs) are increasingly recognized as key mediators of intercellular communication between cancer cells and their environment, a process crucial for tumor progression. TEVs can act locally on neighboring cells or travel long distances to impact remote tissues, thereby promoting tumor growth, cell invasion, pre-metastatic niche formation, and ultimately, metastasis. Despite significant insights into the molecular mechanisms by which TEVs shape the tumor microenvironment (TME) and induce pro-metastatic effects in recipient cells, many questions remain unanswered. Recent studies suggest that caveolae, invaginations of the plasma membrane with critical roles in cellular mechanics, may play an important role in TEV-mediated metastatic trait acquisition by cancer cells. The presence of caveolin-1 (Cav1) in EVs supports its involvement in EV dynamics, including biogenesis, secretion and uptake by recipient cells. Further research into the role of Cav1 in EV-mediated cancer progression could pave the way for improved diagnostic tools and novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea652a5ff93c1b98b26dd7f3de3802cfa6293377" target='_blank'>
              The role of Caveolin-1 in tumor-derived extracellular vesicle biology and its implications
              </a>
            </td>
          <td>
            Satish Kailasam Mani, Christophe Lamaze, Cristian Saquel
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an extremely aggressive disease, and standard of care therapies have failed to yield significant clinical benefit, with invasive surgery being the only curative treatment for patients with early-stage disease. Tumor-associated glycans in pancreatic cancer have direct effects on the survival and propagation of the tumor proper and contribute to an immunosuppressed tumor microenvironment. The existence of a “tumor glycocode” in PDAC and the role of hypersialylation in this cancer have been hugely underscored. Through this initial understanding, significant strides have been made in the field of glycosylation-mediated immune regulation, uncovering glyco-immune checkpoints that facilitate tumor progression in PDAC and other malignancies. Here, we describe the specific roles of glycan-binding proteins, such as C-type lectin receptors, Siglecs, and Galectins, in generating and promoting immunosuppression, exacerbating survival outcomes, and dampening therapeutic efficacy. We illustrate the scale of glycan-mediated regulation of homeostatic immune responses and how cancer glycans facilitate dampened anti-tumor immunity through the major histocompatibility complex (MHC) and the enhanced expression of immune checkpoints, such as PD-L1 and CTLA-4. A wide array of glycan-targeted therapies against PDAC in the clinic, including monoclonal antibodies, enzymes, and vaccines, has been described. With the help of new glycosylation signatures identified and techniques that allow us to reach single-cell resolution, we can target glycans and generate strategies to activate the immune system against PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53240b990cbc102d6003368890068dd413e8c9bd" target='_blank'>
              Unraveling the glyco-immunity nexus in pancreatic cancer
              </a>
            </td>
          <td>
            Christabelle Rajesh, Richard D. Cummings, P. Radhakrishnan
          </td>
          <td>2025-08-04</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive blood cancer in which disease initiation and relapse are driven by leukemic cells with stem-like properties, known as leukemic stem cells (LSCs). The LSC compartment is highly heterogenous and this contributes to differences in therapy response. This heterogeneity is determined by genetic and nongenetic factors including somatic mutations, the cell of origin, transcriptional and epigenetic states as well as phenotypic plasticity. While this complicates the identification and eradication of LSCs, it also presents an opportunity to tailor therapeutic strategies to the phenotypic and functional states of LSCs present in a patient, exploiting their specific vulnerabilities. The emergence of single-cell multiomics technologies has transformed our ability to dissect cellular heterogeneity in AML, enabling simultaneous interrogation of genomic, transcriptomic, epigenomic and proteomic layers and providing high-resolution molecular snapshots of individual cells. In this review, we discuss causes and consequences of LSC heterogeneity, highlight advances in single-cell multiomics technologies to resolve it and outline how they can address shortcomings in our understanding of LSC heterogeneity and plasticity to revolutionize diagnostics and disease monitoring of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07654c74db7467e7588089114a2ee5e54a7d1bf9" target='_blank'>
              Resolving leukemic stem cell heterogeneity and plasticity with single-cell multiomics.
              </a>
            </td>
          <td>
            Frank Y Huang, Andreas Trumpp, Patrick Stelmach
          </td>
          <td>2025-08-01</td>
          <td>Seminars in hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most common brain cancers diagnosed in children, often leading to life-long neurological impairments. Whilst targeted inhibitors exist, their effects are often short-lived, highlighting the urgent need for innovative treatments. Notably, some pLGG tumors spontaneously regress with age, suggesting that changes in the tumor microenvironment can arrest tumor growth. In this work, we aim to characterize which signaling interactions promote the growth of pLGG cells and how these interactions change dynamically with time. To achieve this, we have optimized a protocol for introducing pLGG mutations into developing mouse brains via in utero electroporation. This study revealed that brains harboring the common KIAA1549::BRAF fusion mutation have a striking phenotype characterized by increased astrocyte reactivity and myeloid cell infiltration. To better understand how signals derived from immune cells may impact pLGG cell fitness, we are additionally performing a high-throughput cytokine screen. We are using barcoded cytokine constructs tethered to the cell membrane to comprehensively screen the impact of cytokine signaling on the growth of neural stem cells engineered to overexpress common pLGG mutations. Together this work aims to unravel which signaling factors govern the age-dependent growth of pLGG tumors, which may reveal targetable aspects of the tumor microenvironment amenable to therapeutic intervention.



 Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy Chang, Ria Kedia, Michelle Boisvert, David Jones, Timothy Phoenix, Pratiti Bandopadhayay. Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f149e2a5701b9900d3e20ea1c251a5956b1b56d2" target='_blank'>
              Abstract B037: Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy Chang, Ria Kedia, Michelle Boisvert, David Jones, T. Phoenix, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background and purpose The tumor microenvironment (TME) is widely acknowledged as a pivotal regulator of cancer progression. However, the dualistic role of tertiary lymphoid structures (TLSs), which serve as critical immune hubs within the TME, remains incompletely characterized, particularly with respect to their context-dependent capacity to either inhibit or facilitate tumor development. This review aims to synthesize current understanding of the complex interactions between stromal cells and TLSs, addressing existing gaps in mechanistic insight and exploring therapeutic avenues to exploit TLS plasticity. Key reviewed topics The current study critically reviews the mechanisms by which stromal components, including cancer-associated fibroblasts and endothelial cells, contribute to TLS neogenesis through chemokine-mediated recruitment of lymphocytes. Furthermore, it highlights the dual functional roles of TLSs as sites of both anti-tumor immune activation and immunosuppression, notably via the enrichment of regulatory T cells. The clinical implications of mature TLS presence, particularly their association with improved patient prognosis and enhanced therapeutic responsiveness, are also analyzed. Main conclusions TLSs demonstrate a bifunctional nature, wherein their spatial organization and dynamic interactions with stromal elements dictate the balance between immune activation and tolerance within the TME. While mature TLSs are generally correlated with favorable clinical outcomes, their potential to foster immunosuppressive microenvironments necessitates the development of precision-targeted interventions. The interplay between stromal cells and TLSs represents a promising therapeutic axis for modulating the tumor immune milieu. Future perspectives Future research should prioritize strategies aimed at promoting TLS maturation, disrupting immunosuppressive niches, and integrating TLS-modulating agents with existing immunotherapeutic regimens to enhance clinical efficacy. Additionally, the identification of robust biomarkers reflective of TLS functional states and the rigorous validation of stromal-targeted therapies within combinatorial treatment frameworks are imperative for advancing translational applications. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69c13fc16e53261c020d877469541bad918b8156" target='_blank'>
              In-depth insight into tumor-infiltrating stromal cells linked to tertiary lymphoid structures and their prospective function in cancer immunotherapy
              </a>
            </td>
          <td>
            Maedeh Radandish, Niloofar Mashhadi, Amir Hossein Aghayan, Motahareh Taghizadeh, Sara Salehianfard, Sheida Yahyazadeh, Omid Vakili, Somayeh Igder
          </td>
          <td>2025-08-10</td>
          <td>Experimental Hematology & Oncology</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="T follicular helper (Tfh) cells are central to the adaptive immune response and exhibit remarkable functional diversity and plasticity. The complex nature of Tfh cell populations, inconsistent findings across experimental systems and potential differences across species have fueled ongoing debate regarding core regulatory pathways that govern Tfh differentiation. Many studies have experimentally investigated individual proteins and circuits involved in Tfh differentiation in limited contexts, each providing only a partial understanding of the process. To address this, we adopted a novel multi-scale network systems approach that incorporates both regulatory and protein-protein interactions. Our approach integrates diverse data types, captures regulation across multiple levels of immune system organization, and recapitulates known drivers. Further, we discover a core Tfh gene set that is conserved across tissue types and disease contexts, and is consistent across data modalities - bulk, single-cell and spatial. While components of this set have been individually reported, a novel aspect of our work lies in the discovery, characterization, and connectivity of this core signature using a single unbiased approach. Using this method, we also uncover a novel function of IL-12, a molecule with reported conflicting functions, in the regulation of Tfh differentiation. Notably, we find that, in both humans and mice, IL-12 is permissive for the differentiation of Tfh precursors, but blocks subsequent differentiation into GC Tfh cells. Overall, this work elucidates novel networks with unexplored roles in governing Tfh cell differentiation across species and tissues, paving the way for novel -therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cdef08e00798c3da4a3f940c2d60360ad82720" target='_blank'>
              A unified network systems approach uncovers a core novel program underlying T follicular helper cell differentiation
              </a>
            </td>
          <td>
            Alisa A. Omelchenko, Syed A. Rahman, Vinayak V. Viswanadham, Grace J. Yuen, Perla M. Del Rio Estrada, Valentino D’Onofrio, Yijia Chen, Na Sun, Hamid Mattoo, Chinmay G. Varma, Gonzalo Salgado, Maribel S. Nava, Lady C. Ruiz, Dafne D. Rivera, S. A. Rios, Sudhir P. Kasturi, Susan P. Ribeiro, M. Shlomchik, Amanda C. Poholek, Shiv S. Pillai, Rebecca A. Elsner, V. Mahajan, Jishnu Das
          </td>
          <td>2025-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) is a global health challenge shaped by a complex tumor microenvironment (TME), where myeloid‐driven inflammatory and immunosuppressive processes play central roles. Understanding the spatial and dynamic interactions within the TME is essential for uncovering novel regulatory mechanisms. We integrated spatial transcriptomics, single‐cell RNA‐sequencing, and network pharmacology to develop a comprehensive analytical framework for deconstructing cellular and molecular interactions across spatial domains within the TME. Using this approach, we investigated the potential mechanisms by which the Sparganii Rhizoma‐Curcumae Rhizoma (SCP) Chinese medicine (TCM) herbal pair may influence primary CRC and liver metastases. Our analysis suggests that in primary CRC, SCP may affect neutrophil‐enriched domains through modulation of CEACAM‐related signaling, while in liver metastases, it may influence macrophage‐dominated domains associated with APP signaling. These findings highlight possible domain‐specific molecular interactions and suggest that SCP may regulate intra‐ and inter‐domain signaling patterns relevant to TME remodeling. This study establishes a novel integrative methodology to investigate spatial domain‐specific mechanisms within the TME. Our findings reveal the multi‐dimensional effects of SCP, providing a precision‐targeted, TME‐centric therapeutic strategy for CRC. The framework aligns with TCM principles and offers new avenues for personalized cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64108e2457f88f143ed78e3504c4067d48fd77bb" target='_blank'>
              Spatial Domain‐Based Approach to Analyze the Mechanism of Sparganii Rhizoma‐Curcumae Rhizoma Pair in the Treatment of Colorectal Cancer
              </a>
            </td>
          <td>
            Yuan-Jie Liu, Qian-wen Ye, Yi‐lan Jin, Qian Zhang, Xi Zou, Jie-pin Li
          </td>
          <td>2025-08-01</td>
          <td>Food Science & Nutrition</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Functional cellular heterogeneity in tumours often underlies incomplete response to therapy and relapse. Previously, we demonstrated that the growth of the paediatric brain malignancy, sonic hedgehog subgroup medulloblastoma, is rooted in a dysregulated developmental hierarchy, the apex of which is defined by characteristically quiescent SOX2+ stem-like cells. Integrating gene expression and chromatin accessibility patterns in distinct cellular compartments, we identify the transcription factor Olig2 as regulating the stem cell fate transition from quiescence to activation, driving the generation of downstream neoplastic progenitors. Inactivation of Olig2 blocks stem cell activation and tumour output. Targeting this rare OLIG2-driven proliferative programme with a small molecule inhibitor, CT-179, dramatically attenuates early tumour formation and tumour regrowth post-therapy, and significantly increases median survival in vivo. We demonstrate that targeting transition from quiescence to proliferation at the level of the tumorigenic cell could be a pivotal medulloblastoma treatment strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27908f257548bd97b59f7a3c88e807b37fb0d43b" target='_blank'>
              38 OLIG2 MEDIATES A RARE TARGETABLE STEM CELL FATE TRANSITION IN SONIC HEDGEHOG MEDULLOBLASTOMA
              </a>
            </td>
          <td>
            Kinjal Desai, Siyi Wanggou, Erika Luis, Heather Whetstone, Chunying Yu, Robert J. Vanner, Hayden J. Selvadurai, Lilian Lee, Jinchu Vijay, J. E. Jaramillo, Jerry Fan, P. Guilhamon, M. Kushida, Xuejun Li, Gregory Stein, Santosh Kesari, Benjamin D. Simons, Xi Huang, Peter B. Dirks
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Brain metastases (BM) critically reduce breast cancer (BC) patients’ survival. Extravasation is pivotal for BM development, but the underlying events at the blood-brain barrier (BBB) remain elusive. We aimed to unravel the players and mechanisms governing BC cells (BCCs)-BBB interaction. For that, mixed cultures of human brain microvascular endothelial cells (HBMECs), mimicking the BBB, and brain-tropic triple-negative BCCs (MDA-MB-231 Br4), or non-brain-tropic (MDA-MB-231) or non-metastatic cells (MCF-7) were established. Temporal and spatial analysis of BCCs-BBB interactions (live-cell imaging automated microscopy), and assessments of endothelial-to-mesenchymal transition (EndMT) markers, transcription factors, cytoskeletal proteins, and morphology (immunocytochemistry) were performed. BBB integrity (permeability, transendothelial electrical resistance) and endothelial migration (wound-healing) were also assessed. Our results revealed that contrasting with non-metastatic and non-brain-tropic cells, BCCs quickly developed an invasive, migratory phenotype, characterized by invadopodium formation and reduced roundness. Spatial analysis showed different positioning of BCCs relative to the BBB endothelium over time, with 14% of BCCs transmigrated after 3 h, compromising BBB integrity through endothelial holes, reduced tightness, and increased permeability. Prior to transmigration, alterations in adhesion markers (E-selectin, ICAM-1, CD24, CD34, β3-integrin, Sialyl-Lewis X) were observed. EndMT was also evident by decreased endothelial (β-catenin and pan cytokeratin) and increased mesenchymal (vimentin, neuronal-cadherin, Slug, ZEB1) markers, elongation (RhoA, α-SMA), nuclear deformation, and migratory capacity. Caveolin-1 silencing in HBMEC decreased BCCs transmigration. This study reveals significant BBB phenotypic and structural changes, facilitating both paracellular and transcellular BCCs transmigration. These findings provide advanced understanding of BCCs trafficking across the BBB, aiding strategy development to prevent extravasation and BM. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01985-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58375842dd28bad8d863cc4ae660178921a26a5b" target='_blank'>
              Unveiling the intricate dynamics of the interplay between triple-negative breast cancer cells and the blood-brain barrier endothelium
              </a>
            </td>
          <td>
            Ana Rita Garcia, Ana Rita Vaz, Rui Malhó, Hugo M. Botelho, Inês Figueira, M. A. Brito
          </td>
          <td>2025-08-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b1a443d05c1334b2d0a0dff8460a523dfdb2d5" target='_blank'>
              A three-dimensional ex vivo model recapitulates in vivo features and drug resistance phenotypes in childhood acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Magdalini Kanari, Iria Jimenez Garcia, Fabio Steffen, L. Krattiger, Charles Bataclan, Wangjie Liu, Benjamin R. Simona, Bart Deplancke, O. Naveiras, M. Ehrbar, B. Bornhauser, Jean-Pierre Bourquin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Lysosomes are critical organelles that act as degradation centers and signaling hubs within cells, playing a significant role in various cellular processes and human diseases, including cancer. However, the extent to which they influence the heterogeneity and clinical outcomes of ovarian cancer (OC) remains inadequately understood. In this study, we used consensus clustering to identify two distinct lysosome-related clusters (LCs) in OC by analyzing the expression profiles of OC patients from The Cancer Genome Atlas (TCGA) database. Further analyses revealed the functional characteristics and immune landscapes of these subgroups, providing valuable insights into the tumor immune microenvironment (TIME) and tumor responses to immunotherapy. Additionally, we developed and validated a prognostic model based on differentially expressed genes (DEGs) between the two LCs, demonstrating its effectiveness in predicting patient prognosis, TIME characteristics, and immunotherapy potential in OC. A further investigation explored the relationship between lysosome-associated risk scores, IC50 values of standard antitumor agents, and the expression levels of prognostic genes. Finally, in vitro experiments showed that inhibiting CRHR1, a lysosome-associated prognostic gene, significantly reduced OC cell proliferation, invasion, and migration. In conclusion, our study establishes a novel lysosome-based classification and prognostic framework for OC, offering a practical tool to predict clinical outcomes and guide personalized immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18271-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a65a1574799d7c11e4a3b68fa4b0f39f93a926" target='_blank'>
              Lysosome-related molecular subtypes reveal prognostic signatures and immunotherapeutic implications in ovarian cancer
              </a>
            </td>
          <td>
            Zhou Fang, Yunqing Liu, Yujie Cui, Ke Cao, Yiling Han, Zhijie Gao
          </td>
          <td>2025-09-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) is a common characteristic of human cancers, which occurs most frequently in ovarian and breast cancers. The unique genetic vulnerabilities, coupled with the synthetic lethality effect of Poly-(ADP-ribose) polymerase (PARP) inhibitors (PARPi), provide a promising opportunity for targeting HRD cancers. However, only a few HRD patients benefit from PARPi monotherapy, and it is therefore imperative to explore effective combination therapies for HRD tumors. Growing evidence has underscored the distinct tumor microenvironment (TME) landscape of HRD cancers. Immune activation and immune suppression co-exist in a dynamic balance during the development of HRD cancers. At the late stage, however, negative immune regulation predominates, resulting in the formation of an immunosuppressive microenvironment. This intricate network reprograms cancer biology in multiple aspects and serves as a potential target for cancer treatment. In this review, we briefly outline the current HRD tests and genetic characteristics of HRD cancers, focusing on breast, ovarian, pancreatic, and prostate cancers. We then summarize the interactions and crosstalk between immune cells and cancer cells, as well as various signaling pathways within the TME of HRD cancers. Additionally, we highlight recent advances in combining PARP inhibitors with immunotherapies in preclinical models and clinical trials of HRD cancers. This review provides valuable insights and perspectives into the distinct landscape of TME in HRD cancers, and offers a rationale for expanding the application of this combined therapeutic approach to a broader range of HRD cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befa4ba7c7bbd20ccb4cc8148dd06bbf9c57c776" target='_blank'>
              The distinct landscape of tumor immune microenvironment in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Qiuyang Xu, Yuanjia Wen, Taiyuan Huang, Huayi Li, Xingzhe Liu, Shen-nan Shi, Wenjian Gong, Gordon B. Mills, Ding Ma, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-08-20</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions that drive progression and pathobiology. The aim of this study was to understand how these spatial patterns and interactions evolve with treatment.


METHODS
To explore these relationships, we employed imaging mass cytometry to measure the expression of 34 protein markers, enabling the identification of GBM-specific cell types and their interactions at single-cell protein level in paired primary (pre-treatment) and recurrent (post-treatment) GBM samples from five patients.


RESULTS
We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes.


CONCLUSIONS
These findings provide insight into treatment-induced shifts in GBM's cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fcab07cdc7f1fdc819d108cb3fdc13fdcffb099" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment.
              </a>
            </td>
          <td>
            Shoaib Ajaib, Joshua Winter-Luke, Richard J Digby, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F Stead
          </td>
          <td>2025-09-01</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The draining lymph node (LN) is the most frequent and often first site of cancer metastasis. Although endobronchial ultrasound-guided transbronchial needle aspiration (EBUS-TBNA) is frequently performed as a standard practice in lung cancer diagnosis and staging, its diagnostic accuracy remains modest, primarily due to the minuscule sample size of needle aspirates. With the advent of single-cell technologies, we comprehensively analyzed the immune cell repertoire in a series of EBUS-TBNA samples. Intrathoracic LN samples from 18 subjects with pathologically confirmed metastasis and four controls without evidence of metastasis were compared using single-cell RNA sequencing and mass cytometry analyses. We found that immune cell composition and gene expression patterns differed markedly between metastatic and control LNs. In particular, metastatic LNs contained relatively more APOE-high myeloid cells, with the latter exhibiting significant transcriptional derangement and a powerful intercellular interaction signature. Additionally, CD8 T cells in metastatic LNs demonstrated a unique exhausted phenotype. In conclusion, immune cell phenotypes and gene expression patterns from EBUS-TBNA samples can be leveraged to advance our understanding of cancer immunology and may have independent diagnostic value when malignant cells fail to be identified on histopathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc0ebe3d726c4d02e68690d2768e9f2e7fdb0cc" target='_blank'>
              APOE-high myeloid cells are uniquely associated with metastatic intrathoracic lymph nodes obtained by EBUS-TBNA in primary lung cancer
              </a>
            </td>
          <td>
            S. Kim, Minhyung Kim, Ki-Hyun Kim, Jacob D Eccles, Bumseo Baek, Catherine Dell, Kevin Haas, Christopher M. Kapp, Christian Ascoli, Sungyong You, Gye Young Park
          </td>
          <td>2025-08-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) are ectopic lymphoid aggregates often found in chronic inflammatory conditions, including cancer. These structures, which share many cellular and functional features with secondary lymphoid organs, can profoundly influence the tumor microenvironment by promoting local anti-tumor immune activation. TLSs have been observed in various cancers, including melanoma, and are associated with improved responses to immunotherapy and clinical outcomes. However, our understanding of the molecular mechanisms underlying TLS formation and function remains incomplete. This review summarizes the current findings on TLSs in human melanoma, drawing from multiple studies to provide an updated overview. We discuss the cellular composition, spatial distribution, and genetic signatures of TLSs at different stages of melanoma pathogenesis and in subtypes including acral, uveal, and desmoplastic melanoma. Additionally, we examine the influence of tumor mutational burden (TMB) and complement activation on TLS formation, as well as the role of TLSs in immune checkpoint inhibitor therapy. We also highlight the potential of TLSs as indicators for disease progression and treatment response, and review preclinical strategies aimed at inducing TLSs to improve therapeutic outcomes. This synthesis aims to support ongoing research into the role of TLSs in melanoma immunobiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd5d327c6ce7e444f0890b04b9badc06dcf384c" target='_blank'>
              Tertiary Lymphoid Structures in Human Melanoma: Molecular Mechanisms and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Gelare Ghajar-Rahimi, Ishika Patel, Nabiha Yusuf
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cell signaling plays a critical role in regulating cellular state, yet uncovering regulators of signaling pathways and understanding their molecular consequences remains challenging. Here, we present an iterative experimental and computational framework to identify and characterize regulators of signaling proteins, using the mTOR marker phosphorylated RPS6 (pRPS6) as a case study. We present a customized workflow that uses the 10x Flex assay to jointly profile intracellular protein levels, transcriptomes, and CRISPR perturbations in single cells. We use this to generate a “glossary” dataset of paired protein–RNA measurements across targeted perturbations, which we leverage to train a predictive model of pRPS6 levels based solely on transcriptomic data. Applying this model to a genome-wide Perturb-seq dataset enables in silico screening for pRPS6 and nominates novel regulators of mTOR signaling. Experimental validation confirms these predictions and reveals mechanistic diversity among hits, including changes in signaling output driven by anabolic activity, cellular proliferation and multiple stress pathways. Our work demonstrates how integrated experimental and computational approaches provide a scalable framework for multimodal phenotyping and discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a589887aa331a2fbdb712cfa97ab0770a36bd6c" target='_blank'>
              Iterative, multimodal, and scalable single-cell profiling for discovery and characterization of signaling regulators
              </a>
            </td>
          <td>
            John D Blair, Alexandra Bradu, Carol Dalgarno, Isabella N. Grabski, R. Satija
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are cells shed from tumors into the bloodstream, providing a unique source of information on tumor biology. CTCs can be collected through liquid biopsy, offering a noninvasive option to track cancer progression and treatment responses. Whereas ctDNA gives valuable insights into genomic alterations, CTCs allow for the direct analysis of cellular protein and RNA expression, essential for identifying targetable markers and resistance mechanisms. Resistance to cancer therapies is often due to genetic and phenotypic adaptations within tumor cells. CTC analysis can help track these adaptations, providing real-time insights into evolving resistance mechanisms and suggesting alternative therapies.


SIGNIFICANCE
Enumeration and molecular characterization of CTCs provide unique insights into the biology of cancer metastasis as well as clinically relevant information on tumor evolution, risk assessment, and therapy response in individual patients with solid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80509a0d0dd4086ca6871a4474226159831e914e" target='_blank'>
              From Discovery to Diagnosis: A Perspective for Circulating Tumor Cells in Personalized Oncology.
              </a>
            </td>
          <td>
            C. Alix-Panabières, Klaus Pantel
          </td>
          <td>2025-08-18</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="


 KRAS^G12D mutations represent a significant oncogenic driver, prevalent in highly aggressive malignancies such as pancreatic ductal adenocarcinoma (PDAC) and colorectal cancer (CRC). The recent development of direct KRAS^G12D inhibitors has ushered in a new era of targeted therapy, offering unprecedented opportunities to address these previously intractable cancers. This study aimed to explore and optimize therapeutic strategies by thoroughly evaluating the efficacy of MRTX1133, a novel selective non-covalent KRAS^G12D inhibitor, in conjunction with various immunotherapeutic modalities, particularly those involving natural killer (NK) cells.



 We utilized patient-derived xenograft and syngeneic murine cancer models to systematically characterize alterations of immune landscapes within tumor microenvironment (TME) upon KRAS^G12D inhibition. Comprehensive and in-depth assessment of immune cell function and single-cell level transcriptional activity was performed using flow cytometry and single-cell RNA sequencing analysis.



 Our findings reveal that MRTX1133 not only suppresses tumor progression but also profoundly enhances the activation of infiltrated natural killer (NK) cells within the tumor microenvironment. Functional depletion of NK cells partially, yet significantly, attenuated the anti-tumor activity of MRTX1133, underscoring the contribution of NK cells to the observed therapeutic benefit. Notably, KRAS^G12D inhibition led to a reduction in tumor-infiltrating Gr-1^+ myeloid-derived suppressor cells (MDSCs) and increased the proportion of effector NK cells. Furthermore, MRTX1133 facilitated greater intratumoral accumulation of adoptively transferred NK cells and counteracted tumor-induced systemic immunosuppression on NK cells. Combined treatment with MRTX1133 and NK cell immunotherapies produced synergistic anti-tumor effects and significantly extended survival in preclinical models. Mechanistically, ex vivo analyses demonstrated that MRTX1133 upregulates NK cell-activating ligands, including MICA/B and ULBPs, and suppresses the production of cytokines such as IL-6 and GM-CSF by cancer cells, thereby enhancing tumor cell susceptibility to NK cell-mediated cytotoxicity and reducing MDSC proliferation.



 Collectively, our results illuminate the interplay between KRAS^G12D inhibition and immuno-microenvironmental remodeling, highlighting rational combinations of targeted and NK cell-based immunotherapies as promising strategies to potentiate therapeutic responses in KRAS^G12D-driven cancers, potentially improving clinical outcomes for patients.



 Hongyuan Chen, Tuo Hu, Chao Xu; Liangjie Chi, Fangqin Xue, Chunbo He. KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aa888d3ef7c73bc5df1193380be3a3238a9fcd" target='_blank'>
              Abstract A016: KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity
              </a>
            </td>
          <td>
            Hongyuan Chen, Tuo Hu, Chao Xu, Liangjie Chi, Fangqin Xue, Chunbo He
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive and lethal brain tumor with very poor prognosis despite recent progress in multimodal treatments. Within glioma tissue, various niche cells such as macrophages and neutrophils form a unique glioma immune microenvironment (GIME) by interacting with heterogenous cancer cells, and this has been implicated in disease progression and responsiveness to immunomodulatory therapies. This study explores novel potential prognostic markers associated with the GIME using integrated bioinformatics analyses, including single-cell RNA-sequencing (scRNA-seq), and spatial transcriptome (ST) datasets of clinical GBM specimens. We first identified 42 genes as being associated with poor prognosis in GBM from 5 different cohorts, GBM vs. nontumor tissue, grade IV vs. grade II gliomas, isocitrate dehydrogenase (IDH)-wild-type vs. IDH-mutant variants, mesenchymal vs. proneural and classical subtypes, and hazard ratio for overall survival. Among these, 32 genes were positively correlated with ESTIMATEScore, infiltration of various immune cell types, expression of known immune-related genes, and representative immune-associated biological signals. On scRNA-seq analysis, 7 genes were relatively concentrated in tumor-associated macrophages rather than in malignant cells. ST analysis revealed that Collagen beta(1-O)galactosyltransferase 1 (COLGALT1), Integrin subunit beta 2 (ITGB2), and Myosin light chain 12A (MYL12A) were distributed in the interface between the tumor and the peritumoral area, overlapping with the expression of representative immune-related genes. These findings support the potential of COLGALT1, ITGB2 and MYL12A as biomarkers for predicting the prognosis and immune responses of GBM, which can help in the development of potential immunotherapeutic strategies for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76a83bcafe86bab51686dcebc0d98499aabea491" target='_blank'>
              Potential Prognostic Markers for Glioblastoma Associated with the Glioma Immune Microenvironment.
              </a>
            </td>
          <td>
            Kazuya Tokumura, E. Hinoi
          </td>
          <td>2025-09-01</td>
          <td>Biological & pharmaceutical bulletin</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) remains a major challenge in oncology, necessitating the identification of new therapeutic targets. This study aimed to explore the CRC microenvironment and uncover potential targets using single‐cell RNA sequencing (scRNA‐seq). Methods Single‐cell RNA sequencing data from GEO (GSE164522, GSE132465, and GSE144735) were integrated and stratified based on CRC clinical stages and tumor grades. Cell clustering analysis identified distinct cellular subsets, while survival and functional enrichment analyses evaluated the prognostic significance and biological activity of differentially expressed genes. Additionally, transcription factor activity and pseudotime trajectory analyses elucidated cellular dynamics during CRC progression. Cell–cell communication analysis identified key ligand‐receptor interactions involved in immune regulation. Finally, multiplex immunofluorescence was used to validate the expression of key markers and cells in CRC tissues from patients. Results In the low differentiated colorectal adenocarcinoma (LDCA) grade of CRC, the proportion of T cells significantly increased, accompanied by the enrichment of Treg cells and the upregulation of genes associated with immunosuppression, proliferation, and differentiation. Notably, TGFβ1+ Tregs significantly expanded in LDCA, accumulating in late tumor grades with enhanced immunosuppressive capacity and self‐proliferation. Pseudotime analysis confirmed their dominance in late CRC, reinforcing immunosuppression through signal amplification and sustained expansion. Concurrently, CD8+ T cells in the LDCA grade exhibited a progressive loss of cytotoxic function, transitioning into an exhausted state while concurrently activating apoptotic pathways, leading to a profound impairment of antitumor immunity. Cell–cell communication analysis further demonstrated that TGFβ1+ Treg exhibits the strongest interactions with CD8+ T cells, with KLRB1‐CLEC2D, LGALS9‐HAVCR2, and TNFSF10‐TNFRSF10B emerging as pivotal ligand‐receptor pairs displaying significantly enhanced signaling in the LDCA grade. Conclusion Collectively, TGFβ1+ Treg may mediate CD8+ T cell dysfunction through these ligand‐receptor interactions, accelerating T cell exhaustion and apoptosis, thereby fostering a profoundly immunosuppressive tumor microenvironment that ultimately drives CRC immune evasion and malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b126f27787b7f75c899e3f95b02d5186b9c098e5" target='_blank'>
              Integrated Single‐Cell RNA‐Seq Reveals Immunosuppressive Mechanisms of Treg Cell Differentiation and Tumor Microenvironment Interactions in Colorectal Cancer
              </a>
            </td>
          <td>
            Dongwei Li, Jingya Liu, Lixian Yu, Xina Luo, Xiaomei Wu, Ronghui Chen
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Tumor heterogeneity impacts invasive behaviors, treatment response, and clinical outcomes in triple-negative breast cancer (TNBC). However, this heterogeneity remains incompletely characterized. This study aims to utilize multi-scale data to investigate inter-tumoral heterogeneity and identify potential TNBC biomarkers.Single-cell RNA expression profiles were analyzed using R packages. Specifically, the infercnv, Pyscenic, GeneNMF, SCP, Vector, CellChat, and hdWGCNA packages were employed to identify malignant cells and characterize heterogeneity in transcription factors, metaprograms, lineage evolution, developmental trajectories, cell–cell interactions, and co-expression networks. Bulk RNA datasets were incorporated to assess the prognostic value of cell clusters and candidate genes. G Protein Subunit Alpha 15 (GNA15) expression was determined via reverse transcription–quantitative PCR (RT–qPCR) and immunohistochemistry. Cell functional assays were performed to evaluate proliferation, migration, and invasion capabilities.A total of 14,335 malignant cells were isolated from epithelial cells across 15 single-cell RNA samples. Six tumor cell clusters were identified, which exhibited distinct prognoses, biological functions, driver transcription factors, and co-expression networks. Notably, the S2 cluster demonstrated association with multiple malignancy-related pathways and inferior survival outcomes. GNA15 emerged as the S2 cluster hub gene. In vitro experiments confirmed that GNA15 knockdown significantly attenuated proliferation, migration, and invasion in TNBC cell lines.Our study comprehensively delineated TNBC tumor cell heterogeneity and established the critical role of GNA15 in TNBC progression. These findings enhance the understanding of TNBC heterogeneity and provide a theoretical foundation for TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e149422e4d37b7c64ca2e5cbee9d7c8957b702e6" target='_blank'>
              Multiomics profiling identifies the poor prognostic role of a tumor cluster with GNA15 overexpression in triple-negative breast cancer
              </a>
            </td>
          <td>
            Guixin Wang, Jun Cao, Chenglu Lu, Yu-Peng Cao, Shuo Wang, Ziyi Chen, Zhaohui Chen, Yingxi Li, Yue Yu, Yao Tian, Xin Wang
          </td>
          <td>2025-09-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Metaplastic breast cancer (MpBC) is defined by the presence of various morphological elements, typically biphasic, with epithelial (e.g. no-special type (NST), squamous) and mesenchymal (e.g. spindle, chondroid, osteoid) components. The established clonality of the different components favours an evolution model encompassing either a multipotent progenitor, or a linear metaplastic conversion. We used methylation profiling and showed that different morphologies have specific methylation profiles. Furthermore, our spatial transcriptomic approach, using 10× Genomics Visium and trajectory analysis, evidenced that spindle cells form a transition between the originating carcinoma of no-special type (NST) and pleomorphic regions, with osteoid differentiation likely to be an end-stage fate of the chondroid growth pattern, supporting the conversion model of lineage differentiation. We have also identified a series of master transcription factors likely to regulate these processes, and are significantly associated with metaplastic-like clinical features. This data further supports the conversion model of metaplasia and warrants functional analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077285caa17e07f895159bb1ece8c70f06837f91" target='_blank'>
              Integrated profiling of metaplastic breast cancer identifies putative master regulators of intratumoral heterogeneity
              </a>
            </td>
          <td>
            Yufan Feng, Albert Xiong, Onkar Mulay, Anna Sokolova, M. Lim, Benjamin Van Haeringen, Natasha McGuire, Xavier M. de Luca, Peter T Simpson, Quan Nguyen, S. Lakhani, A. M. McCart Reed
          </td>
          <td>2025-08-11</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4146020be8bb663490d3534c28c485f753647c10" target='_blank'>
              Machine learning integration of bulk and single-cell RNA-seq data reveals glycolytic heterogeneity in colorectal cancer.
              </a>
            </td>
          <td>
            Yuanyuan Du, Zefeng Miao, Peng Li, Dan Feng, Mulin Liu, Aifang Ji, Shijun Li
          </td>
          <td>2025-08-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Glioblastoma is a highly aggressive and devastating brain malignancy with dismal prognosis and extremely limited therapeutic options. Identification of prognostic biomarkers and therapeutic targets from multi‐omics data is critical for improving patient outcomes. In this study, we investigated the clinical significance of cellular heterogeneity and super‐enhancer‐driven regulatory networks, which are critically implicated in glioblastoma progression and treatment resistance. We first performed scRNA‐seq to dissect tumour microenvironment heterogeneity, identifying 16 distinct cell clusters, including astrocytes, macrophages, and CD8+ T cells. CellChat analysis revealed key intercellular signalling pathways, with astrocytes and macrophages acting as central communication hubs. To integrate bulk RNA sequencing data, we applied the Scissor algorithm to identify survival‐associated cell states. By combining single‐cell and bulk transcriptomic data, we uncovered 642 survival‐related genes, including QKI and RBM47, which robustly predicted patient survival and immunotherapy response. Furthermore, WGCNA analysis identified seven co‐expression modules and super enhancer‐regulated networks orchestrated by transcription factors (RFX2, RFX4) and hub genes (NEAT1, CFLAR). These networks stratified patients into high‐ and low‐risk groups with significant survival differences. Collectively, our findings elucidate the intricate interplay between cellular heterogeneity and super enhancer‐driven gene regulation in glioblastoma, providing a translational framework for targeting oncogenic hubs and modulating microenvironment interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7616587b7549f42937faeb12de6e3dcf343225e7" target='_blank'>
              Integration of Single‐Cell RNA and Bulk RNA Sequencing Reveals Cellular Heterogeneity and Identifies Survival‐Associated Regulatory Networks in Glioblastoma
              </a>
            </td>
          <td>
            Zijun Xu, Bohan Xi, Jiaming Huang, Liqiang Zhang, Sifu Cui, Xianwei Wang, Dong Chen, Shupeng Li
          </td>
          <td>2025-08-13</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT In humans, cell-cell communication orchestrates tissue organization, immune coordination, and repair, yet spatially mapping these interactions remains a challenge for biology. We introduce STARComm, a scalable-interpretable computational method that identifies Multicellular Communication Interaction Modules (MCIMs) by detecting spatially co-located receptor-ligand activity from high-plex spatial transcriptomics in 2D and 3D. Applied to an atlas of >14million cells across 8 cancers, STARComm revealed 24 conserved and tumor-specific MCIMs, including a fibro-immune module with targetable axes linked to immune exclusion and immunotherapy resistance. In chronic graft-versus-host disease, STARComm identified three salivary gland MCIMs predictive of patient death and two druggable axes ( CXCL12-CXCR4 , CCL5-SDC4 ), both with FDA-approved therapeutics. STARComm demonstrated that peripheral tissue profiling can forecast fatality nearly 3 years in advance using minor salivary glands. By enabling scalable biomarker discovery, drug targeting, and spatially resolved precision profiling, STARComm bridges the gap between spatial biology and clinical translation, advancing the field of spatial medicine. SUMMARY Despite major advances in spatial biology, no framework has yet linked spatially resolved intercellular communication networks, independent of cell types, to clinical outcomes in human disease. Here, we present STARComm, a scalable method that identifies Multicellular Interaction MCIMs (MCIMs). Applying STARComm to minor salivary gland biopsies from patients with chronic graft-versus-host disease (GVHD), we identify MCIMs that not only distinguish healthy from diseased tissue but also stratify patient survival. High-risk MCIMs are enriched for actionable immune and stromal pathways, including those targetable with existing therapies. These findings establish the first outcome-linked spatial communication framework in any human disease and highlight the translational potential of oral tissues as minimally invasive platforms for real-time immune diagnostics, prognostic modeling, and therapeutic screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a093b13aff7d0f5c2b6d575cc2d04f2b86e47ba" target='_blank'>
              STARComm Scalably Detects Emergent Modules of Spatial Cell-Cell Communication in Inflammation and Cancer
              </a>
            </td>
          <td>
            Khoa Huynh, B. Matuck, Deiziane de Souza, XiuYu Zhang, Giancarlo Fatobene, Luiz Alberto Valente Soares Júnior, L. F. Ferraz da Silva, V. Rocha, K. Byrd, Jinze Liu
          </td>
          <td>2025-08-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract In its first decade, The Journal of Pathology: Clinical Research has become a leading source of translational studies advancing molecular diagnostics in cancer, particularly in urologic pathology. This commentary highlights recent contributions that collectively place precision oncology at the forefront of pathology research. One review examines cancer stem cells in renal cell carcinoma, emphasizing the complexity of cellular plasticity and the tumor microenvironment in driving resistance and recurrence. In prostate cancer, epithelial‐to‐mesenchymal transition (EMT) regulators, including Twist, Slug, and Snail, are identified as synergistic markers of poor prognosis, linked to hypoxia and invasiveness. Another review details the integration of homologous recombination repair gene testing into clinical workflows, supporting targeted treatment strategies with poly (ADP‐ribose) polymerase inhibitors. In pediatric oncology, TP53 alterations in Wilms tumor are shown to occur beyond anaplastic cases, expanding their prognostic significance. Advances in molecular subtyping are also demonstrated in bladder cancer, where transcriptomic profiling could enable tailored neoadjuvant therapy. In clear cell renal cell carcinoma, re‐evaluation of a prognostic model revealed that while necrosis or sarcomatoid differentiation correlated with poor outcomes, only DNA methylation markers improved prognostic accuracy, underscoring their utility for biopsy‐based risk stratification. Finally, digital spatial profiling of sarcomatoid urothelial carcinoma reveals an immunosuppressive microenvironment with CD163‐positive cells, implicating them in EMT and aggressive phenotype. Together, these studies highlight the transformative role of integrated molecular diagnostics in guiding individualized therapies and improving outcomes in urologic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9269338c6f4616fe82344276fa3a9645f1fc73c2" target='_blank'>
              Translational milestones in urologic pathology: integrating molecular diagnostics across cancer types
              </a>
            </td>
          <td>
            Andres Matoso, A. Acosta
          </td>
          <td>2025-09-01</td>
          <td>The Journal of Pathology: Clinical Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioma represents the most prevalent primary tumors of the central nervous system, originating from glial cells. Cancer stem cells have the ability to extensively proliferate, self-renew, and form colonies, which contribute to tumorigenesis. Studies have found a population of cells within glioblastoma exhibiting characteristics similar to those of cancer cells, termed glioma stem cells (GSCs). GSCs have two distinct phenotypes: mesenchymal subtype (MES) and proneural subtype (PN). Despite the vital role of these subtypes in glioma biology, there is a significant lack of comprehensive reviews focusing on the regulatory mechanisms underlying each phenotype.This review integrates emerging insights into the regulatory mechanisms underlying GSCs plasticity, with dedicated analysis of novel pathways governing PN and MES phenotypes and their dynamic transition. By examining these critical elements, we aim to contribute to the development of novel therapeutic strategies in the ongoing fight against gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50da6e5130555aeaeba55f2b83f42650df5fbf68" target='_blank'>
              Phenotypic variations in glioma stem cells: regulatory mechanisms and implications for therapeutic strategies
              </a>
            </td>
          <td>
            Guofeng Tian, Yifu Song, Yaochuan Zhang, Liang Kan, Ana Hou, Sheng Han
          </td>
          <td>2025-09-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Osteosarcoma (OS) is the most common primary bone malignancy in children and adolescents, which is also considered an aggressive disease due to its rapid growth rate, ability to metastasize early, and complex and heterogeneous tumor microenvironment (TME). Although we are developing improved surgical and chemotherapeutic approaches, the presence of metastatic or recurrent disease is still detrimental to the patient’s outcome. Major advances in understanding the molecular mechanisms of OS are needed to substantially improve outcomes for patients being treated for OS. This review integrates new data on the molecular biology, pathophysiology, and immune landscape of OS, as well as introducing salient areas of tumorigenesis underpinning these findings, such as chromothripsis; kataegis; cancer stem cell dynamics; and updated genetic, epigenetic, and glycosylation modifiers. In addition, we review promising biomarkers, diagnostic platforms, and treatments, including immunotherapy, targeted small molecule inhibitors, and nanomedicine. Using genomic techniques, we have defined OS for its significant genomic instability due to TP53 and RB1 mutations, chromosomal rearrangements, and aberrant glycosylation. The TME is also characterized as immunosuppressive and populated by tumor-associated macrophages, myeloid-derived suppressor cells, and regulatory T cells, ultimately inhibiting immune checkpoint inhibitors. Emerging fields such as glycomics and epigenetics, as well as stem cell biology, have defined promising biomarkers and targets. Preclinical studies have identified that glycan-directed CAR therapies could be possible, as well as metabolic inhibitors and 3D tumor models, which presented some preclinical success and could allow for tumoral specificity and enhanced efficacy. OS is a biologically and clinically complex disease; however, advances in exploring the molecular and immunologic landscape of OS present new opportunities in biomarkers and the development of new treatment options with adjunctive care. Successful treatments in the future will require personalized, multi-targeted approaches to account for tumor heterogeneity and immune evasion. This will help us turn the corner in providing improved outcomes for patients with this resilient malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d810402edaf93a3babd4fc4b0a6f7de04e75bb" target='_blank'>
              Molecular and Glycosylation Pathways in Osteosarcoma: Tumor Microenvironment and Emerging Strategies Toward Personalized Oncology
              </a>
            </td>
          <td>
            G. Iacobescu, A. Corlatescu, H. Costin, Razvan Spiridonica, M. Popa, C. Cirstoiu
          </td>
          <td>2025-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Bladder cancer (BLCA) continues to be a significant cause of cancer mortality in the urinary tract, with therapeutic resistance representing a major barrier to improving patient outcomes. Within the tumor microenvironment (TME), cancer-associated fibroblasts (CAFs) are pivotal drivers of BLCA progression, contributing to immune evasion and therapy resistance. This study leverages single-cell analysis to delineate CAF subclusters and explore the immune characteristics of CAFs-based BLCA classification. Materials and methods Signal-cell RNA sequencing (scRNA-seq) datasets were used to identify CAF subpopulations in BLCA, and bulk RNA-seq datasets were used to construct CAFs-based BLCA classification. Next, we comprehensively explored the distinct heterogeneity and characteristics for four CAFs-based BLCA subtypes. Moreover, machine learning algorithms were applied to identify novel potential targets for each subtype, and experimentally validate their effects. Results This study identified CAFs closely associated with BLCA development based on scRNA-seq datasets. Through further systematic clustering and functional analysis of CAFs, we successfully identified 10 distinct CAF sub-clusters, including PSCA+ Pericyte, ISG15+ Pericyte, ACTA2+ Smooth muscle cell (SMC), ACTG2+ SMC, CCL21+ inflammatory Pericyte, CD74+ apCAF, STMN1+ pCAF, CXCL14+ mCAF, APOD+ iCAF, CFD+ iCAF. The study identified four pCAFs-based BLCA distinct subtypes with different molecular, functional, and immunologic characteristics. C3 exhibited an immune-rich subtype accompanied by poor clinical prognosis, cell death pathway enrichment, higher expression of MHC molecules and co-stimulatory/co-inhibitory molecules. Conversely, C4 subtype has a smaller number of patients and an optimal prognosis, associated with lower levels of cell death pathway enrichment, lower frequency of tumor mutations, and an “immune desert” TME. C1 is mainly enriched in metabolism-related pathways, and C2 is mainly enriched in the activation of genome instability pathways, accompanied by more frequent mutations and higher Atezolizumab response. Furthermore, this study identified potential target genes or prognostic markers for each subtype. Conclusion Various heterogeneous CAF subgroups exist in BLCA, which is closely associated with the development of BLCA. This study identified a promising platform for understanding heterogeneity of CAFs-based BLCA subtypes, providing novel insights into the intricate molecular mechanisms of BLCA. Potential target genes for each subtype provide a basis for diagnosis and screening of BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3695d526f4a124539104737c20e126464559121b" target='_blank'>
              Characterization of cancer-related fibroblasts in bladder cancer and construction of CAFs-based bladder cancer classification: insights from single-cell and multi-omics analysis
              </a>
            </td>
          <td>
            Zhaokai Zhou, Yajun Chen, Zhan Wang, Shuai Yang, Zhengrui Li, Run Shi, Ruizhi Wang, Kui Liu, Xiaojuan Tang, Qi Li, Ran Xu
          </td>
          <td>2025-09-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfc80059f8767909d0c8bd930a6c33c0803bbb" target='_blank'>
              Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers.
              </a>
            </td>
          <td>
            Rishaan Kenkre, Jon D. Larson, Owen S. Chapman, J. Luebeck, Yan Yuen Lo, Megan Paul, Wenshu Zhang, V. Bafna, Robert J Wechsler-Reya, Lukas Chavez
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor draining lymph nodes (TDLNs), as secondary lymphoid organs, are pivotal in initiating and regulating adaptive immune responses. Historically, TDLNs were recognized primarily as metastasis gateways in cancer, promoting radical dissection to prevent recurrence. However, emerging preclinical studies reveals their critical role in orchestrating systemic anti-tumor immune responses during cancer therapy, highlighting the dilemma of balancing lymph nodes (LNs) preservation with metastasis control. This review traces the evolving understanding of TDLN biology in oncology, from the era of radical LN dissection to multi-omics-driven insights, and synthesizes their dual roles as immune hubs and metastatic niches across first-line clinical therapies (e.g., immunotherapy, radiotherapy, chemotherapy, targeted therapy, etc.). We further propose the concept of “Lymph Node Multi-modal Protective Research (LNMPR)”, emphasizing the prospective value of integrating multi-omics technologies, including spatial transcriptomics, single-cell profiling, and imaging, to decode LN immune dynamics and optimize therapeutic responses. By bridging mechanistic insights with clinical strategies, LN-centric immune modulation may open up a new path for precise tumor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ddb9cb8a82e08359fde743e98ea142d35dd6bc8" target='_blank'>
              From trash to treasure: tumor draining lymph nodes as a multi-omics goldmine in cancer therapy
              </a>
            </td>
          <td>
            Yina Li, Zihan Chen, Zhikun Guo, Jiangnan Yu, Jianan Lu, Lei Wang, Qian Zhou
          </td>
          <td>2025-08-19</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24a4c1ebff0edca5f86402765983e14a517f49f7" target='_blank'>
              Form Follows Function: A New Paradigm of Pancreatic Cancer Progression.
              </a>
            </td>
          <td>
            William Gasper, Giada Pontecorvi, Zhikai Chi, S. Enrico, Lily C. Goodman, Haneen Amarneh, Evelyn Mazzarelli, Joshua Thomas, Lucia Zhang, Rachel Handloser, Seth LeCates, A. L. Tomescu, Francesca Rossi, Myla Sykes, Anjali Bhatt, Nicholas Fazio, Alessandro Bifolco, Nafeesah Fatimah, Austin Seamann, Cheryl M Lewis, Shelby Melton, Herbert J. Zeh, Megan B. Wachsmann, Dario Ghersi, Matteo Ligorio
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Recent advances in single-cell studies have unveiled the extensive transcriptomic heterogeneity of cancer-associated fibroblasts (CAFs), with each subset potentially playing unique roles in the tumor microenvironment. Despite these insights, it remains unclear how to culture different subsets of CAFs in vitro while preserving their in vivo phenotype. The inherent plasticity of CAFs, characterized by their ability to dynamically alter their phenotype in response to various environmental stimuli, presents significant challenges in inducing and maintaining specific CAF states in vitro. In this study, we investigated the effects of cell shape and confinement on two-dimensional culture substrates on the transcriptomic profile and phenotype of cultured CAFs. By inducing cytoskeletal rearrangement and activating mechanotransductory pathways through micropatterning of polyacrylamide hydrogels, we observed significant shape- and confinement-dependent changes in cell morphology. Our results showed that micropatterned CAFs exhibited phenotypic shifts towards more myofibroblastic, desmoplastic and inflammatory subsets. Additionally, micropatterning allowed for control over a range of CAF-specific markers and pathways, significantly altering their transcriptome landscape and behavior. Lastly, we demonstrated that micropatterned and non-micropatterned CAFs responded differently to anti-cancer drugs, underscoring the importance of phenotype-oriented therapy that accounts for CAF plasticity and regulatory networks. Control over CAF morphology through micropatterning presents a unique opportunity to establish highly robust CAF phenotypes in vitro, thereby facilitating a deeper understanding of CAF plasticity and heterogeneity. This approach also holds promise for the development of novel therapeutic targets, as it allows for the precise manipulation of CAF states to better model and study their interactions within the tumor microenvironment.



 Aleksandr Mitriashkin, Josephine Yu Yan Yap , N. Gopalakrishna Iyer , Gianluca Grenci, Eliza Li Shan Fong. Cell confinement by micropatterning induces phenotypic changes in cancer-associated fibroblasts [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr LT03.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7eb0c539e2fb60deec8ba647c172fbc76cfd279" target='_blank'>
              Abstract LT03: Cell confinement by micropatterning induces phenotypic changes in cancer-associated fibroblasts
              </a>
            </td>
          <td>
            Aleksandr Mitriashkin, Josephine Yu Yan Yap, N. G. Iyer, Gianluca Grenci, Eliza Li Shan Fong
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Clonally established tumor cell lines often do not recapitulate the behavior of cells in tumors. The sequencing of a whole tumor tissue may not uncover transcriptome profiles induced by the interactions of all different cell types within a tumor. Interferons for instance have a vast number of binding sites in their target genes. Access to the DNA binding sites is determined by the epigenomic state of each different cell type within a tumor mass. To understand how genes such as interferons appear to have both tumor-promoting and tumor-inhibiting functions, single-cell transcript analysis was performed in the breast cancer tissue of HER2+ (epidermal growth factor receptor 2) patients. We identified that potential antagonistic oncogenic activities of cells can be due to diverse expression patterns of genes with pleiotropic functions. Molecular pathways both known and novel were identified and were similar with those previously identified for patients with rheumatoid arthritis. Our study demonstrates the efficacy in using single-cell transcript analysis to gain insight into genes with apparent contradictory or paradoxical roles in oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261a79861db7f4bc06df15b1b4671db2e425c476" target='_blank'>
              Single-Cell mRNA Analysis for the Identification of Molecular Pathways of IRF1 in HER2+ Breast Cancer
              </a>
            </td>
          <td>
            Laura Vilardo, P. Pelucchi, Antonia Brindisi, E. Abeni, E. Piscitelli, E. Mosca, G. Bertalot, Mira Palizban, Theodoros Karnavas, A. Gritzapis, Ioannis Misitzis, Martin Götte, Ileana Zucchi, R. Reinbold
          </td>
          <td>2025-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Regulating tumor invasion and metastasis is pivotal for improving cancer patient prognosis. While cell migration is a key factor in these processes, the non-targeted effects of chemoradiotherapy on cell motility remain poorly understood. In this study, we employed HeLa-FUCCI cells—a cervical cancer-derived HeLa cell line integrated with the Fluorescent Ubiquitination-Based Cell Cycle Indicator (FUCCI) probe, enabling the visualization of cell cycle phases—to investigate the radiation-induced impacts, including non-targeted effects, on cell migration. To create irradiated (In-field) and non-irradiated (out-of-field) regions, half of the culture dish was shielded with a lead block during irradiation. Cells were then exposed to 2 Gy X-rays, with or without cisplatin. Following irradiation, the cells were subjected to time-lapse imaging at 15 min intervals for 24 h, and the acquired data were analyzed using cell segmentation and tracking algorithms, Cellpose 2.0 and TrackMate 7. Without cisplatin, the migration velocity and total distance traveled of Out-of-field cells were significantly reduced compared to controls, suggesting a suppressive bystander signal. In contrast, with cisplatin treatment, these parameters significantly increased in both In-field and Out-of-field cells. This suggests that chemoradiotherapy may inadvertently enhance tumor cell motility outside the target volume, a critical finding with significant implications for therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/186add6c9ae96976df3f063bad12c6cb1e871668" target='_blank'>
              Deep Learning-Based Imaging Analysis Reveals Radiation-Induced Bystander Effects on Cancer Cell Migration and the Modulation by Cisplatin
              </a>
            </td>
          <td>
            Ryosuke Seino, Hisanori Fukunaga
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Effective identification of oncogenic mutations is essential for diagnosis, forecasting resistance, and metastasis in remission. It is required for an optimal drug regimen. We develop a framework to discover mutations that co-exist in different oncoproteins, and those that are excluded, likely encoding oncogene-induced senescence. First, mapping the proteins onto pathways assists combinatorial drug selections and helps to detect metastases. Second, it provides the molecular basis for synthetic lethality, to date investigated at the genome level. Our pan-cancer profiles of ~60,000 tumor sequences, detect 3424 co-existing tumor-specific mutations. Mapping them onto pathways indicates that they preferentially promote specific primary tumors. We uncover metastatic mutations and provide metastatic breast-cancer markers. This work not only clarifies the mechanistic basis of intratumor mutational diversity but usefully reveals markers for metastasis in patients’ genomes and introduces a novel computational framework for detecting metastasis based on tumor mutational profiles. Mapping the mutations onto pathways provides an invaluable metastasis-targeting resource, guiding drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8dbee9272fd01bd4b589963df2977d50caa0e8" target='_blank'>
              Mutations in tumor signaling, metastases, and synthetic lethality establish distinct patterns
              </a>
            </td>
          <td>
            B. R. Yavuz, Uğur Şahin, Hyunbum Jang, Ruth Nussinov, Nurcan Tunçbağ
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4ae2d5f1f6f184ad5a2b71dcc7e35b4feeb422f" target='_blank'>
              Functional biomaterials and machine learning approaches for phenotyping heterogeneous tumor cells and extracellular vesicles.
              </a>
            </td>
          <td>
            Rutwik Joshi, R. Ahmad, Karl Gardner, Hesaneh Ahmadi, Chau-Chyun Chen, Shannon L Stott, Wei Li
          </td>
          <td>2025-09-18</td>
          <td>Biomaterials science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95f957c61adab36c7dec0eefa4bae427dbcc6b3" target='_blank'>
              CRISPR-Edited Cell Lines: A New Era in Functional Oncology Research.
              </a>
            </td>
          <td>
            Amita Joshi Rana, M.D.S.A.D.I.Q.U.E. Hussain, Ali Hanbashi, Faroq Kamli, Gyas Khan, Marwa Qadri, Saeed A Al-Qahtani, Mohammad Gayoor Khan, Sushil S. Burle, Vikas Jakhmola, Gaurav Gupta
          </td>
          <td>2025-08-13</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="OBJECTIVE
Adamantinomatous craniopharyngioma (ACP) is an epithelial tumor that occurs in the sellar region of the human brain. Despite resection, relapse is frequent with poor prognosis. To facilitate the development of new therapy for ACP, the authors examined the spatial distribution, cell of origin, and potential biological functions of leukemia inhibitory factor (LIF), an important stem cell self-renewal regulator, in a series of ACP tumors.


METHODS
The transcriptional sites of LIF and LIF receptor (LIFR) were determined by single-cell sequencing and space transcriptome analysis. LIF and LIFR distribution characteristics in different histopathological regions were detected with immunohistochemistry and immunofluorescence analysis. The relationships between the regional distributions of different tissues and tumor imaging characteristics, tumor cell stemness, cell proliferation, LIF expression, and patient prognosis were analyzed.


RESULTS
The authors' analysis of 39 ACPs detected LIF overexpression that was selectively enriched in cell clusters. In addition to the discovery of the stromal cells in the interstitial region, palisade epithelium, and stellate reticulum as the source cells of LIF production, the authors also revealed that LIFR was primarily generated by the cell clusters. Examination of differentially expressed genes between LIF-high and LIF-low ACP tumors indicated that the binding of LIF to LIFR may lead to the activation of the PI3K-AKT signaling pathway. Further analysis showed enrichment of LIF expression in β-catenin-positive cell clusters expressing stem cell markers of CD44, supporting its role in stem cell self-renewal. Integrated analysis with diagnostic imaging found higher level expression of LIF in cystic tumors than in solid tumors, displaying a trend toward poorer prognosis.


CONCLUSIONS
This study confirmed for the first time that LIF in ACP mainly originates from tumor microenvironment stroma. The authors' data suggest that future efforts should also include tumor stromal cells as a novel cellular and/or molecular target when developing new anticancer therapies against ACP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e38d490dae9950f240b893ac7de6f57947b7a0" target='_blank'>
              Use of integrated spatial transcriptomics and histopathological analysis in adamantinomatous craniopharyngiomas to identify stromal cells as a new cellular source of leukemia inhibitory factor.
              </a>
            </td>
          <td>
            Wenxin Hu, Chuan Zhao, Wenrong Zheng, Yi Lin, Ning Luo, Hongxing Liu, Xingfu Wang, Xueling Qi, Xianlong Wang, Xiao-Nan Li, Zhixiong Lin
          </td>
          <td>2025-08-01</td>
          <td>Journal of neurosurgery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Melanoma brain metastasis (MBM) leads to significant morbidity and mortality. Building on previous findings that WNT5A regulates melanoma metastasis and dormancy in extracranial niches, we hypothesize that WNT5A signaling facilitates melanoma invasion across the blood-brain barrier, while its downregulation drives intracranial metastatic proliferation. To explore this, we conducted functional studies using human melanoma cell lines and an organotypic mouse brain xenograft system. Overexpression or pharmacologic induction of WNT5A increased melanoma migration and invasion toward brain explant conditioned media in transwell assays. Wholemount confocal imaging of brain explant cultures revealed distinct tumor colonization patterns mirroring clinical dormancy and proliferative outgrowth depending on WNT5A level: melanoma overexpressing WNT5A formed single-cell dispersions perivascularly, whereas silencing WNT5A in syngeneic cells produced proliferative clusters. Exogenous recombinant WNT5A also induced G1 cell cycle arrest in FUCCI-transduced melanoma cells, supporting its role in maintaining dormancy. Conversely, the WNT5A antagonist SFRP1 drove melanoma toward increased tumor size and accelerated tumor development. Interestingly, immunohistochemical analysis of postmortem human brain tissue demonstrated robust SFRP1 expression in cerebral endothelium and neurons, indicating a potential source promoting metastatic outgrowth. Integrin profiling revealed elevated expression of adhesion proteins in invasive melanoma compared to WNT5A-silenced syngeneic lines, suggesting an association with enhanced metastatic potential and tumor extravasation. WNT5A overexpression resulted in greater endothelial and extracellular matrix engagement by adhesion assay and live-cell tracking. These interactions were recapitulated in ex vivo xenografts, with WNT5A-high melanoma adhering to endothelial basement laminin. Together, these findings highlight the importance of WNT5A regulation through the metastatic cascade, from endothelial adhesion and establishment of a perivascular dormancy niche to metastatic outgrowth. Further integration of transcriptomics and proteomics on mouse models and clinical MBM samples aim to uncover key modulators of WNT signaling and associated pathways in MBM progression which might be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a305149b00eaf3bc4069c98fdc0c3ac940583e00" target='_blank'>
              BSBM-02 WNT5A MEDIATES MELANOMA BRAIN METASTASIS
              </a>
            </td>
          <td>
            Kevin Zhang, Denys Balandin, M. R. Rocha, Vania Wang, Fan Huang, Cheyenne Palm, Alexis E Carey, Ritvik Koya, Jordina Rincon-Torroella, C. Eberhart, Calixto-Hope G. Lucas, A. Weeraratna
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Developing patient-derived organoids (PDOs) from metastatic breast cancers offers opportunities for functional drug screening and biomarker discovery, yet their establishment remains technically demanding. To test this methodology for generating PDOs from CNS breast cancer metastases, we successfully derived a PDO from a solid CNS metastasis. The PDOs continued to express proliferative markers and Estrogen Receptor during passaging, demonstrating the feasibility of this approach. We next sought to generate PDOs from breast cancer leptomeningeal disease (LMD), isolating metastatic breast cancer cells from the cerebrospinal fluid (CSF) of affected patients who consented to donate tissue for research. We compared previously reported Type 1 and Type 2 medium formulations for growing breast cancer organoids, as well as a recently described Ocello medium that incorporates conditioned medium from leptomeningeal cells. In our initial experience, we found that LMD organoids could survive in Ocello medium but exhibited slow or no expansion in culture. For comparison, we also tested the growth of metastatic cancers that had spread to other fluid-filled compartments, leading to malignant effusions in the pleural and peritoneal spaces. In contrast to LMD, we found that PDOs from pleural effusions and ascites exhibited rapid expansion of epithelial components for the majority of cases, expressed expected breast cancer markers, and maintained intratumoral heterogeneity. We also demonstrate the ability to isolate T cells from malignant fluid for co-culturing with PDOs to test immunotherapy agents. These results underscore both the potential and limitations of current ex vivo approaches in modeling metastatic disease within fluid compartments. While PDOs from pleural effusions and ascites show promise, refining culture conditions for LMD remains critical. Our future studies will aim to optimize ex vivo expansion by testing additional modifications to culture conditions and continuing the slow expansion of existing viable cells. We will also incorporate in vivo growth efforts using mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dbee1e52c3cc4078fa31acf650940fe9ad8392a" target='_blank'>
              PRCA-03 PATIENT-DERIVED ORGANOIDS FOR MODELING CENTRAL NERVOUS SYSTEM (CNS) METASTATIC BREAST CANCER
              </a>
            </td>
          <td>
            M. Ebadi, Lisa Ni, David P Sheppard, Molly H Blau, Erica L Peters, Indira Dastan, Larissa Muller, Sara Wade, Fei Gao, S. Braunstein, Jonathan T Yang, V. Venur, L. M. Halasz
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="For over sixty years, blood researchers have been counting clones with every tool at their disposal. Inspired by phage and fly geneticists, Till and McCulloch irradiated mice to induce chromosomal aberrations. Using this labeling strategy, they demonstrated that different types of blood cells shared the same mutation in every spleen colony, thereby proving the existence of hematopoietic stem cells. Since their breakthrough, technological advances have enabled researchers to quantify hematopoiesis at single-cell resolution in increasingly complex samples across both mice and humans. With these modern sophisticated lineage tracing methods, our foundational understanding of the blood system is being reshaped. For instance, we now interpret hematopoietic architecture as arising from stem and progenitor cells of diverse developmental origins, each with distinct fate biases encoded by unique regulatory states. Interacting with this regulatory layer, genetic mutations and epimutations arise, expanding clonally and becoming pervasive with age. Together, clonal heterogeneity and age-driven clonal selection may underlie the perplexing diversity of therapy responses in cancer and beyond. As these paradigm-shifting insights gain traction, clonal tracing is being adopted across dozens of biological and clinical studies. Here, we review the modern toolbox of clonal tracking technologies, with a focus on next-generation sequencing-based approaches, and provide a practical guide for matching specific research questions with optimal experimental strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca9cbce80016ba1b72cbad0fc7cc9413794f150" target='_blank'>
              Clonal tracing of blood stem cells across mouse and human lifespans.
              </a>
            </td>
          <td>
            Alejo E. Rodriguez-Fraticelli
          </td>
          <td>2025-09-22</td>
          <td>Blood</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) occurs heterogeneously among carcinoma cells to promote chemoresistance. Identifying the signaling pathways involved will nominate drug combinations to promote chemoresponse, but cell population-level studies can be misleading, and single-cell transcriptomics are limited to indirect ontology-based inferences. To understand EMT heterogeneity at a signaling protein level, we combined iterative indirect immunofluorescence imaging of pancreas cancer cells and tumors and mutual information (MI) analysis. Focusing first on mitogen-activated protein kinase pathways, MI indicated that cell-to-cell variation in ERK activity determined EMT heterogeneity in response to different growth factors and chemotherapeutics but that JNK compensated when MEK was inhibited. Population-level models could not capture these experimentally validated MI inferences. The dominant role of ERK was consistently indicated by MI even when the analysis was expanded to include seven potential EMT-regulating signaling nodes. More generally, this work provides an approach for studying multivariate signaling-phenotype relationships based on protein measurements in any setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3bfa194502b5ce8ba1fe00ed578bfccff400f9" target='_blank'>
              The kinase ERK plays a conserved dominant role in the heterogeneity of epithelial-mesenchymal transition in pancreatic cancer cells.
              </a>
            </td>
          <td>
            Michelle C. Barbeau, Brooke A. Brown, S. Adair, Todd W. Bauer, Matthew J. Lazzara
          </td>
          <td>2025-08-12</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer therapies are limited by tumor heterogeneity, complex tumor microenvironments (TME), and treatment resistance. Traditional 2D cell cultures cannot replicate tumor 3D architecture and dynamic interactions, reducing clinical relevance. Organoid-on-a-Chip (OoC) technology overcomes these gaps by integrating microfluidics, tissue engineering, and cell biology to create physiologically accurate 3D models. This platform simulates TME dynamics—including vascularization and multi-organ interactions—surpassing static conventional models. Key advancements: (1) Three development phases: basic 3D culture (2009–2015), multi-organ coupling (2016–2020), and clinical translation (2021–present); (2) FDA Modernization Act 2.0 (2022) enabling OoC data as sole preclinical evidence for clinical trials; (3) Patient-derived organoids (PDOs) retaining parental tumors’ features with >87% drug-response accuracy in colorectal cancer. Vascularized tumor chips further study angiogenic dynamics and drug efficacy. While OoC excels in drug screening, toxicity testing, and personalized oncology, challenges persist in simulating systemic immune responses. Advancing multi-organ integration and policy alignment remains critical to replace animal models and advance precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ae0a214bc39ef2fcf7935f22b0933c555122daa" target='_blank'>
              Application and development of Organ-on-a-Chip technology in cancer therapy
              </a>
            </td>
          <td>
            Ling xiao Wang, Shu ling Liu, Ning Wu
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human induced pluripotent stem cells (iPSCs) are gaining momentum as a powerful starting material in cell therapy. To fully harness their potential, CRISPR technology permits endogenous gene modifications as well as the introduction of advanced features, to increase the immune compatibility of the cells or insert suicide genes for enhancing therapeutic safety, for instance. However, genetic manipulation of iPSCs, in particular the generation of knock-in lines, remains relatively inefficient. Conventional mitigation strategies, such as enriching for positive cells using antibiotic selection or complex instrumentation, may, however, cause conflicts with good manufacturing practice (GMP) requirements. To address this challenge, we have systematically optimized a basic gene editing procedure using both Cas9 and Cas12a-based ribonucleoprotein (RNP) complexes. Based on the sequential delivery of RNPs and donor plasmids as a critical hallmark, this virus-free approach permits knock-ins of full-length transgenes at above 30% efficiency, while readily identifying positive clones through random screening at small scale. We exemplify these advances by creating and characterizing homozygous iPSC lines depleted of HLA class I and carrying an inducible caspase-9 suicide gene. Isolated clones from independent GMP iPSC lines retained genomic integrity, differentiation capability, and functionality of the safety switch in the differentiated state. This improved methodology will form a flexible platform for custom gene editing universally applicable both in basic iPSC research and therapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17876-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09eb4a0dc63d8980b4874abede67f3864afc2133" target='_blank'>
              Sequential factor delivery enables efficient workflow for universal gene editing in clinical grade iPS cells
              </a>
            </td>
          <td>
            Thomas Berger, Elitsa Borisova, A. Gamerschlag, Daniel Terheyden-Keighley, Soraia Martins, Boris Greber
          </td>
          <td>2025-09-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The growth of a tumor is tightly linked to the distribution of its cells along a continuum of activation states. Here, we systematically decode the activation state architecture (ASA) in a glioblastoma (GBM) patient cohort through comparison to adult murine neural stem cells. Modelling of these data forecasts how tumor cells organize to sustain growth and identifies the rate of activation as the main predictor of growth. Accordingly, patients with a higher quiescence fraction exhibit improved outcomes. Further, DNA methylation arrays enable ASA-related patient stratification. Comparison of healthy and malignant gene expression dynamics reveals dysregulation of the Wnt-antagonist SFRP1 at the quiescence to activation transition. SFRP1 overexpression renders GBM quiescent and increases the overall survival of tumor-bearing mice. Surprisingly, it does so through reprogramming the tumor’s stem-like methylome into an astrocyte-like one. Our findings offer a framework for patient stratification with prognostic value, biomarker identification, and therapeutic avenues to halt GBM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dee763118a03533523bd5d7836ea98324ad3594" target='_blank'>
              Cross-species comparison reveals therapeutic vulnerabilities halting glioblastoma progression
              </a>
            </td>
          <td>
            Leo Carl Foerster, Oguzhan Kaya, Valentin Wüst, Diana-Patricia Danciu, Vuslat Akcay, Milica Bekavac, Kevin Chris Ziegler, Nina Stinchcombe, Anna Tang, S. Kleber, Joceyln Tang, Jan Brunken, Irene Lois-Bermejo, Noelia Gesteira-Perez, Xiujian Ma, Ahmed Sadik, P. Le, Kevin Petrecca, Christiane A Opitz, Haikun Liu, C. R. Wirtz, Angela Goncalves, A. Marciniak-Czochra, Simon Anders, A. Martin-Villalba
          </td>
          <td>2025-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="PURPOSE
With surgery being the only potential cure for pancreatic cancer, high-risk premalignant pancreatic lesions often go unnoticed by palpation or white light visualization, leading to recurrence. We asked whether near-infrared fluorescence imaging of tumor-associated inflammation could identify high-risk premalignant lesions, leveraging the tumor microenvironment (TME) as a sentinel of local disease and, thus, enhance surgery outcomes.


EXPERIMENTAL DESIGN
Fluorescence-guided surgery was performed on genetically engineered mice (Ptf1a-Cre; LSL-KrasG12D/+; Smad4flox/flox [KSC]) at discrete stages of disease progression, histologically confirmed high-risk, premalignant lesions in postnatal mice to locally advanced pancreatic tumors in adults, using the imaging agent V-1520, a translocator protein (TSPO) ligand. Age-matched wild-type littermates were used as controls, while Ptf1a-Cre; LSL-KrasG12D/+(KC) mice modeled pancreatitis and precursors of low penetrance. Localization of V-1520 and tumor-associated macrophages amongst the TME was detected by immunofluorescence imaging.


RESULTS
V-1520 exhibited robust accumulation in the pancreata of KSC mice from the early postnatal stage. Increased accumulation was observed in the pancreata of adolescent- and adult-aged mice with greater ductal lesion and stromal burden. Confocal microscopy of ex vivo pancreas specimens co-localized V-1520 accumulation primarily with CD68-expressing macrophage in KSC mice. Unlike the pancreata of KSC mice, accumulation of V-1520 did not exceed background levels in the pancreata of KC mice with pancreatitis.


CONCLUSION
V-1520 exhibited differential accumulation in pancreatic cancer-associated inflammation compared to pancreatitis. Given the robust tracer uptake in tissues associated with early yet high-risk lesions, we envision V-1520 could enhance surgical resection and reduce the potential for recurrence from residual disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e1713383f0cb9b8f04471c53342e9b58b0edaa9" target='_blank'>
              Preclinical Fluorescence-Guided Imaging Leveraging Surrounding Sentinel Tumor Microenvironment Identifies High-Risk Premalignant Pancreatic Lesions.
              </a>
            </td>
          <td>
            Shilpa Sharma, Xiaoxia Wen, Jianbo Wang, Beibei Huang, Denise Hernandez, Cong-Dat Pham, Zhiwen Liu, S. J. Lin, Aiko Yamaguchi, Dimitra K. Georgiou, R. P. Coll, H. C. Manning
          </td>
          <td>2025-08-25</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabe0cbcb8c826f9af8109abd06e0ae3e7cf0a6a" target='_blank'>
              Patient-derived organoids reveal hypoxia-driven plasticity and therapeutic vulnerabilities in pheochromocytomas and paragangliomas.
              </a>
            </td>
          <td>
            Xiaojing Wang, Maite Calucho, Cynthia M Estrada-Zuniga, Mehwish Noureen, Hector Gonzalez-Cantú, Samantha Mossuto, Jonathan N Levi, Bethany Landry, Qianjin Guo, Kailee A Rutherford, Zi-Ming Cheng, Andreanne Sannajust, Sapna Raghunathan, Sanskriti Balaji, Viviane Nascimento da Conceicao, Summer Norris, S. Y. Munawar, H. Nguyen, Jacob De Rosa, G. Tomlinson, James X Wu, Michael W. Yeh, M. Livhits, Anand Vaidya, James Powers, Ron Lechan, Art Tischler, Miao Zhang, Paul Graham, Camilo Jimenez, M. Peitzsch, Nicole Bechmann, Graeme Eisenhofer, Yanli Ding, Mio Kitano, A. Soragni, P. Dahia
          </td>
          <td>2025-08-23</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Recent advancements in single-cell sequencing technology have reshaped our understanding of cellular processes. While in the realm of biological research, the understanding of the underlying physical and chemical mechanisms from single-cell omics stands as a promising frontier, yet it is still not quite adequately explored. This knowledge gap stems from the complexities of mapping nonequilibrium physical and chemical principles onto the heterogeneous and complex dynamics of cellular functions. Herein, several key biological processes are highlighted to embark on this challenging journey of harnessing the power of single-cell omics to elucidate the nonequilibrium physical and chemical basis of various biological cell functions, including cell cycle, cell differentiation/reprogramming, cancer progression and metastasis, and embryonic dynamic development. This perspective presents breakthrough insights into cell division and differentiation, highlighting the nonequilibrium landscape and flux as the driving forces governing the cellular function which can be quantified through the single-cell omics data, providing new insights into cellular plasticity and fate determination. In addition, it provides possible early warning signals of cancer formation and metastasis based on omics data. Venturing into the wonders of dynamical development, it shows the uncovered nonequilibrium physicochemical mechanisms determined by the dynamical landscape-flux of embryogenesis from time-resolved single-cell data. This perspective further offers an outlook on challenges and opportunities for the integrations of spatiotemporal multiomics and nonequilibrium physical and chemical theories, in anticipation of a more comprehensive and in-depth understanding of the myriad processes of life. Hence, this perspective summarizes key advances in this emerging field and points to the next opportunities and challenges to fully integrate the potential of single-cell biotechnology and physical chemistry theory in life science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0321194975efe56e0cc49e03ac6617ecc83ce7ca" target='_blank'>
              Quantifying Landscape and Flux from Single-Cell Omics: Unraveling the Physical Mechanisms of Cell Function
              </a>
            </td>
          <td>
            Ligang Zhu, Jin Wang
          </td>
          <td>2025-08-07</td>
          <td>JACS Au</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Emerging evidence links alterations in the tumor microbiome to therapeutic responses to immunotherapy. Alterations in β-catenin are among the most frequently observed oncogenic drivers of hepatocarcinogenesis and are associated with T-cell exclusion. However, their effect on the immune cell environment and microbiome in hepatocellular cancer is not well understood. We hypothesized that β-catenin could modulate the immune microenvironment through alterations in the secretome and release of extracellular vesicles (EV) that mediate tumor and immune cell interactions and increase tumor growth within regions with attenuated immune activity and reduced microbial diversity. Methods We used a synthetic transgenic murine model of β-catenin-driven hepatocarcinogenesis to analyze microbiome composition, diversity, and immune cell profiles in vivo. Tumor and stool samples were collected from mice with early- or late-stage hepatocellular carcinoma and were used for profiling. Results The microbiome associated with intrahepatic tumors differs from that in non-tumoral regions, normal liver tissues, and gut tissues. Constitutive β-catenin expression modulates lipopolysaccharide-mediated signaling in macrophages and alters the secretion of immunomodulatory chemokines and cytokines. Tumoral immune cell profiles differed from those in hepatic tissues. EV-mediated signaling between immune cells and epithelial cells with mutant β-catenin and immune cells modulates immune cell populations in vitro and in vivo. Conclusion Mutations in β-catenin can drive immune responses through EV-based tumor cell-immune cell interactions to modulate both the tumor microflora and immune microenvironment. A potential strategy to augment responses to immunotherapy for hepatocellular cancer could target these interactions to restore microbial diversity or immune cell infiltration within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d73e045384f38112d9502014ce3fbc6f907c47" target='_blank'>
              Beta-Catenin Mutations Can Impact on the Interplay Between Tumor and Immune Cells and Hepatic Microbiota in Hepatocellular Cancer
              </a>
            </td>
          <td>
            Yu Ota, Julia Driscoll, Anneliese Hill, Irene K Yan, Tushar Patel
          </td>
          <td>2025-08-01</td>
          <td>Journal of Hepatocellular Carcinoma</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) display remarkable functional heterogeneity, yet the molecular mechanisms driving their diverse phenotypes remain elusive. Using CRISPR screens in primary macrophages, we identified tumor-derived factors, including lactic acid, PGE2, and GM-CSF, as key modulators of TAM polarization. These factors interact synergistically and antagonistically to shape distinct TAM phenotypes that are highly conserved across human cancers. Mechanistically, lactic acid and PGE2 jointly induce angiogenic gene programs while suppressing GM-CSF-driven MHC-II expression at the chromatin level, creating mutually exclusive distributions of proangiogenic and MHC-II+ TAMs, which are differentially localized to specific spatial niches in the tumor microenvironment. Furthermore, we showed that shifting TAMs to an interferon-responsive phenotype, triggered by Adar inactivation, significantly promotes the infiltration of effector CD8+ T cells through specific receptor-ligand interactions. These findings uncover a conserved mechanism of TAM polarization and offer insights into therapeutic strategies for TAM reprogramming to potentiate cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cb6bdb7d04c92080704c1d38a2b724c54141d29" target='_blank'>
              Functional genetic screens reveal key pathways instructing the molecular phenotypes of tumor-associated macrophages.
              </a>
            </td>
          <td>
            Youxue Lu, Ce Luo, Lanxiang Huang, Genyi Wu, Lihan Zhong, Jieyu Chu, Fubing Wang, Zexian Zeng, Deng Pan
          </td>
          <td>2025-09-04</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Osteosarcoma represents the most common principal malignant bone tumor that predominantly appears among teenagers and children. While multimodal treatment methods have greatly evolved with time, survival for recurrent or metastatic disease remains low due to the resistance that accumulates during treatment. Increasing evidence identifies the tumor microenvironment (TME), in particular cancer-associated fibroblasts (CAFs), as playing an important role in imposing immune suppression, enhancing tumor aggressiveness, and mediating resistance toward immunotherapy and chemotherapy. This article gives an overview of the derivation, phenotypic heterogeneity, and mechanisms of action of CAFs during osteosarcoma, such as facilitating immune escape, survival signaling, drug efflux, regulation of genes through exosomes, and inhibiting ferroptosis. Furthermore, we present existing and new treatment methods that are centered on CAFs, such as suppression of the paracrine pathway (e.g., IL-6/STAT3, TGF-β), depletion of CAFs lineages by targeting fibroblast activation protein (FAP), and conversion toward tumor-restraining CAFs. Other methods that are gaining popularity are targeting CAFs-releasing exosomes and metabolic liabilities. By shedding light on CAFs-based methods for imposing resistance and trying targeted treatments, this review offers insights into novel therapeutic combinations that can overcome treatment barriers and improve survival outcomes in osteosarcoma regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edf6237fc3513d36396d6dcdecd8243a643d651f" target='_blank'>
              Cancer-associated fibroblasts in osteosarcoma: key players in immune escape and targeted therapy
              </a>
            </td>
          <td>
            Junjie Fan, Yujie Jin, Feng Lv, Weidong Wu, Li Sun, Churong Wang
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (iCCA) is a highly aggressive and heterogeneous malignancy characterized by marked resistance to standard chemotherapy and poor prognosis. While the advent of immunotherapy has revolutionized the management of several solid tumors, including melanoma, breast cancer, and non-small cell lung cancer, its efficacy in iCCA remains limited. Recent clinical trials have demonstrated the efficacy of durvalumab in combination with chemotherapy for iCCA, leading to its approval as a first-line treatment. However, overall response rates remain low, largely due to its immunosuppressive tumor immune microenvironment (TIME). The immune-cold nature of iCCA is typified by a dominant presence of immunosuppressive cell populations, including M2-polarized tumor-associated macrophages, myeloid-derived suppressor cells, and T regulatory cells. In addition, traditional biomarkers such as PD-L1 expression, tumor mutational burden, and microsatellite instability have shown limited predictive value in iCCA, highlighting the need for novel biomarkers and immunotherapeutic strategies. Emerging approaches aimed at reprogramming the TIME, including combination therapies targeting suppressive cells, stromal remodeling, and novel immune effectors like CAR-T and cancer vaccines, hold significant promise for enhancing therapeutic efficacy. This review summarizes the distinct features of iCCA TIME, key mechanisms of immune evasion, current challenges, and future directions to overcome immune resistance, with the aim of developing personalized immunotherapies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1167c3442bd04894cf43de06826ab2ab3325aec4" target='_blank'>
              Immune Landscape of Intrahepatic Cholangiocarcinoma: Evasion and Therapeutic Insights
              </a>
            </td>
          <td>
            Nunzia Porro, Elena Spínola-Lasso, F. Marra, A. Gentilini
          </td>
          <td>2025-09-17</td>
          <td>Immuno</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/781db8d3b9951c3d4f119253d55c69b0ec7522ab" target='_blank'>
              Hybrid In Vivo Breast Cancer Model Reveals Transcriptomic Insights into Cancer Progression with Age.
              </a>
            </td>
          <td>
            Lauren Hawthorne, Gokhan Bahcecioglu, Jun Yang, Erin Howe, Emilija Aleksandrovic, Siyuan Zhang, P. Zorlutuna
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT In most cancers, T lymphocytes comprise an essential cellular component of the non-neoplastic microenvironment, where they have the capacity to both suppress and support tumor growth. One specialized T lymphocyte population is the CD8+ exhausted T cell, which has been intensely studied as an actionable therapeutic target. Unfortunately, there is currently no uniformly accepted classification scheme for these specialized T cells. To provide a potential model for classifying CD8+ exhausted T cells, we leveraged single cell transcriptomic analysis of a diverse collection of both human (n = 8) and mouse (n = 4) cancers to identify unique subpopulations shared across tumor types and species. By integrating data from both human and mouse cancer studies, as well as previously described CD8+ exhausted T cell subsets, we provide an integrated framework to characterize the heterogeneity of exhausted CD8+ T cells. As such, one of these subpopulations (cluster C1) increases following immune checkpoint inhibitor treatment in the setting of cancer in mice and patients. Taken together, this proposed classification scheme may be useful for the design and interpretation of current and future immune-based therapy studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9bb9b37e719dff3c4a062c99fe8116dd65c7d0" target='_blank'>
              Pan-cancer single cell transcriptomic clustering reveals heterogeneous CD8+ exhausted T cell populations with different immune checkpoint inhibitor responses
              </a>
            </td>
          <td>
            Rui Mu, Rasha Barakat, David H. Gutmann
          </td>
          <td>2025-08-03</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: The number one cause of cancer-related deaths across the globe is metastatic spread of malignant tumors. To determine therapeutic targets, it is important to understand cellular and genetic changes that facilitate invasion, dissemination, and colonization. Although this field can be improved through genomics, the interactions between tumor-intrinsic alterations and the microenvironment are not fully outlined, and thus, current metastasis-preventive approaches are limited. 
Objective: The purpose of the study was to examine cellular changes and genetic mutations related to tumor metastasis, to examine individual patient features, and to examine clinical correlations that point to possible areas of therapeutic intervention. 
Study Design: A retrospective observational study. 
Place and Duration of study: Pakistan Institute of Medical Sciences, PIMS, Islamabad, Pakistan. Jan 2024 to December 2024 
Methods: Review of 100 patients with histologically proven malignant tumors was a retrospective observational study. Institutional records were used to collect demographic and clinical data. Recurrent mutations in TP53, KRAS, and PIK3CA of tumor tissue samples and epithelial-mesenchymal transition markers were assessed by immunohistochemistry. Statistical analyses comprised descriptive statistics, calculation of mean ages and standard deviation and comparison of groups using Student t-test with a significance level set to p < 0.05. 
Results: 100 patients (52 males and 48 females) were used as the cohort. The mean age was 59.4 years +- 11.2 SD. Genomic findings identified TP53 mutations in 45 percent of patients, KRAS mutations in 28 percent and PIK3CA mutations in 18 percent. Loss of E-cadherin expression was also found in 36 percent of samples and was associated with greater invasiveness. Analysis of the metastatic burden revealed that patients who had both TP53 and KRAS mutations had a much greater metastatic burden than patients without (p = 0.018). Clinical associations suggested that genetic changes and epithelial-mesenchymal transition signatures operated as a combined effect on progression, which underscored a suggestion that molecular heterogeneity is the basis of metastatic potential. 
Conclusion: We have shown that genetic mutations, together with cellular reprogramming (epithelial-mesenchymal transition) contribute to metastasis. Combined TP53 and KRAS mutations and their relationship with metastatic progression are correlated, which is important to identify potential risk stratification biomarkers. The combination of molecular and cellular analysis provides informative tools to develop therapies that will help intercept metastatic progression, which ultimately will raise the prognosis and survival rates in patients with progressive malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a13e9e21cb70713f15ecb05360a37152fe398f" target='_blank'>
              Analyzing The Cellular and Genetic Alterations That Contribute to The Metastatic Spread of Malignant Tumors
              </a>
            </td>
          <td>
            Rakhshanda Shamim, Zaid Waqar, Adeel Haidar, Ammar Niazi, Muhammad Mustafa Sadiq, Bushra Anam Ali
          </td>
          <td>2025-09-23</td>
          <td>Journal of Neonatal Surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Brain metastases are a significant complication of non-small-cell lung cancer (NSCLC), contributing to high morbidity and mortality rates. The introduction of immune checkpoint inhibitors (ICIs) has opened new therapeutic avenues for patients with NSCLC, including those with brain metastases. However, the distinct microenvironment of the brain presents unique challenges to the effectiveness of these treatments. This review examines the mechanisms by which ICIs impact brain metastases from NSCLC, with particular focus on immune cell trafficking across the blood–brain barrier (BBB), tumor microenvironment modulation, and transcriptomic evolution of brain-tropic tumor clones. Unlike prior reviews, we integrate emerging data from single-cell and spatial transcriptomic studies, BBB disruption mechanisms, and the tumor-supportive role of brain-resident glia. We also critically evaluate key clinical trials and real-world evidence, highlighting differences in ICI efficacy across patient subgroups and therapeutic contexts. Additionally, we address the evolving role of surgical resection, stereotactic radiosurgery, and cerebrospinal-fluid-based biomarkers in optimizing ICI-based treatment strategies. This synthesis provides a comprehensive, mechanistic, and clinically relevant framework for improving outcomes in patients with NSCLC brain metastases treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c6ed65bc2f6ca5f07b82a4924c158eebc0c9a9" target='_blank'>
              Immune Checkpoint Inhibition in Patients with Brain Metastases from Non-Small-Cell Lung Cancer: Emerging Mechanisms and Personalized Clinical Strategies
              </a>
            </td>
          <td>
            Nicola J. Nasser, K. Sindhu, Loor Nasser, Z. Shafaee, Joshua Li, Lucas Resende Salgado, Baoqing Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma is the most prevalent primary brain malignancy, distinguished by its highly invasive behavior and exceptionally high rates of recurrence. Conventional radiation therapy, which employs uniform treatment margins, fails to account for patient-specific anatomical and biological factors that critically influence tumor cell migration. To address this limitation, numerous computational models of glioblastoma growth have been developed, enabling generation of tumor cell distribution maps extending beyond radiographically visible regions and thus informing more precise treatment strategies. However, despite encouraging preliminary findings, the clinical adoption of these growth models remains limited. To bridge this translational gap and accelerate both model development and clinical validation, we introduce PREDICT-GBM, a comprehensive integrated pipeline and dataset for modeling and evaluation. This platform enables systematic benchmarking of state-of-the-art tumor growth models using an expert-curated clinical dataset comprising 255 subjects with complete tumor segmentations and tissue characterization maps. Our analysis demonstrates that personalized radiation treatment plans derived from tumor growth predictions achieved superior recurrence coverage compared to conventional uniform margin approaches for two of the evaluated models. This work establishes a robust platform for advancing and systematically evaluating cutting-edge tumor growth modeling approaches, with the ultimate goal of facilitating clinical translation and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3497012bad0fd14f83f8c07402640538840f77c2" target='_blank'>
              PREDICT-GBM: Platform for Robust Evaluation and Development of Individualized Computational Tumor Models in Glioblastoma
              </a>
            </td>
          <td>
            L. Zimmer, J. Weidner, M. Balcerak, F. Kofler, I. Ezhov, B. Menze, B. Wiestler
          </td>
          <td>2025-09-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Over the past decade, patient-derived organoids (PDOs) have emerged as powerful in vitro models that closely recapitulate the histological, genetic, and functional features of their parental primary tissues, representing a ground-breaking tool for cancer research and precision medicine. This advancement has led to the development of living PDO biobanks, collections of organoids derived from a wide range of tumor types and patient populations, which serve as essential platforms for drug screening, biomarker discovery, and functional genomics. The classification and global distribution of these biobanks reflect a growing international effort to standardize protocols and broaden accessibility, supporting both basic and translational research. While their relevance to personalized medicine is increasingly recognized, the establishment and maintenance of PDO biobanks remain technically demanding, particularly in terms of optimizing long-term culture conditions, preserving sample viability, and mimicking the tumor microenvironment. In this context, this review provides an overview of the classification and worldwide distribution of tumor and paired healthy tissue-specific PDO biobanks, explores their translational applications, highlights recent advances in culture systems and media formulations, and discusses current challenges and future perspectives for their integration into clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a9d583775632944bea1fb022512dcc7a1ca6c16" target='_blank'>
              Patient-Derived Organoid Biobanks for Translational Research and Precision Medicine: Challenges and Future Perspectives
              </a>
            </td>
          <td>
            F. J. Di Paola, G. Calafato, P. Piccaluga, G. Tallini, Kerry Jane Rhoden
          </td>
          <td>2025-08-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>2</td>
          <td>57</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Lu and colleagues identified profound microbial heterogeneity across individual tumor nodules in multifocal hepatocellular carcinoma, revealing that specific bacterial consortia enrich in metastatic clones and drive epithelial-mesenchymal transition and immune suppression. Their study places the intratumoral microbiome alongside genomic and transcriptomic diversity as a key modulator of intrahepatic tumor evolution. See related article by Lu et al., p. 1630.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3eb39fed6f77266430fee7056b5354580c4710f" target='_blank'>
              A New Dimension in Tumor Heterogeneity: Intratumoral Bacteria Impact Multifocal Hepatocellular Carcinoma Development.
              </a>
            </td>
          <td>
            A. Dzutsev, Giorgio Trinchieri
          </td>
          <td>2025-08-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is a frequently observed phenomenon in aging individuals without apparent illness and exhibits an increased prevalence in cancer patients. Mechanistic studies indicate that mutant immune cells alter the tumor microenvironment, leading to increased inflammation, blood vessel formation, and immune cell exhaustion. Paradoxically, these changes also preserve stem-like T-cell pools that can be utilized by immunotherapy. CH may be incidentally detected in patients whose solid tumors are profiled by next-generation sequencing. Clinically, CH confers higher risks of therapy-related myeloid neoplasms, cardiovascular and inflammatory toxicities, and context-specific changes in treatment efficacy. Moreover, tumorinfiltrating CH independently shortens survival. Two validated risk scores can inform the risk for myeloid malignancy, yet surveillance, cardiometabolic management, and regimen selection still primarily rely on expert consensus. Because CH may be discovered incidentally, rigorous confirmation of variant origin when CH is suspected is essential to avoid misdirected therapy. We propose a pragmatic approach: confirm CH with paired blood sequencing when feasible; integrate high-risk features into risk stratification, counseling, and monitoring for cytopenias and cardiovascular events; and prefer less genotoxic regimens when the oncologic benefit is comparable. Early trials blocking interleukin-1β suggest that targeting inflammation driven by CH may improve outcomes in patients with solid tumors. Prospective studies informed by mutation analysis and tracking clonal changes and inflammatory markers are needed to determine if routine CH assessment can be integrated into precision oncology to improve outcomes for patients with solid tumors and CH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48a6d6136853e6a95a267ec620d529e97551fcc6" target='_blank'>
              What's hidden in plain sight? Impact of clonal hematopoiesis on the risk and progression of non-hematologic cancers.
              </a>
            </td>
          <td>
            José C Martínez, Catherine C Coombs
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb8107340798d9682d8c6977b5d27a13471e822" target='_blank'>
              Integrated single-cell and spatial transcriptomics uncover the prognostic, epigenetic, and immunological roles of FANCC in low-grade glioma.
              </a>
            </td>
          <td>
            Zongye Zhang, Zhi Sha, Han Liu, Yusheng Chen, Zhendong Liu, Xingbo Cheng, Sujie Gu, Yanzheng Gao
          </td>
          <td>2025-09-09</td>
          <td>Neurological research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In prostate cancer, it is recognized that adenocarcinoma can transdifferentiate into neuroendocrine prostate cancer (NEPC) owing to lineage plasticity; however, transdifferentiation into other histological types remains uncertain. We present a case of a patient who underwent surgery for adenocarcinoma, which later recurred as prostate carcinosarcoma. Genomic analysis revealed a TMPRSS2-ERG fusion, confirming a common clonal origin and transdifferentiation from adenocarcinoma to carcinosarcoma. Additionally, we identified a frameshift mutation in TP53 and the loss of PTEN and RB1. Transcriptome analysis revealed enriched epithelial-mesenchymal transition and immune-related pathways, a pattern distinct from both adenocarcinoma and NEPC. To our knowledge, this is the first report that comprehensively evaluated the clonal origin of the rare prostate carcinosarcoma and characterized it using genomic and transcriptomic sequencing. It enhances our understanding of prostate cancer lineage plasticity and highlights the importance of developing novel therapies specifically targeted at prostate carcinosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc3dc165d184c74305476d6dbd5e449b98486ca" target='_blank'>
              Case Report: Genomic insights into prostate adenocarcinoma transdifferentiation to carcinosarcoma due to lineage plasticity
              </a>
            </td>
          <td>
            Tomohiro Fukui, Arinobu Fukunaga, Yuki Teramoto, Maki Fujiwara, Kensuke Hikami, T. Sunada, Kei Mizuno, Yuki Kita, Takayuki Sumiyoshi, Takayuki Goto, R. Saito, Takashi Kobayashi, S. Akamatsu
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Mitotic catastrophe (MC) is a well-recognized endogenous mechanism of tumor cell death, characterized as a delayed cell death process associated with aberrant mitosis. However, its prognostic significance in the context of intratumoral heterogeneity in esophageal squamous cell carcinoma (ESCC) remains largely unexplored. Methods We performed an in-depth analysis of single-cell RNA sequencing (scRNA-seq) data from ESCC obtained from the Gene Expression Omnibus (GEO) database. MC scores for individual cells were calculated using the AddModuleScore function, and T cell specific gene modules were identified via the high-dimensional weighted gene co-expression network analysis (hdWGCNA) framework. To further elucidate the developmental trajectories and intercellular interactions of T cells, pseudotime analysis and cell-cell communication inference were conducted. A prognostic risk model was then constructed using three machine learning algorithms combined with multivariate Cox regression analysis. Following risk stratification, we performed immune infiltration profiling, drug sensitivity analysis, and molecular docking to comprehensively assess the functional implications of the risk model in ESCC. Based on preliminary results from quantitative Real-time PCR (qRT-PCR) and Western blotting (WB), we selected the hub gene SLF2 for functional validation using wound healing, Cell Counting Kit-8 (CCK-8) assay, Transwell, and colony formation assays. Results Based on T cell mitotic catastrophe associated genes (MCAGs) and utilizing machine learning algorithms, we established a robust prognostic risk model for ESCC. The model demonstrated excellent stratification capability in predicting patient outcomes and effectively revealed the heterogeneity of the tumor immune microenvironment (TIME) and drug sensitivity. Furthermore, functional experiments confirmed that knockdown of the hub gene SLF2 significantly inhibited the migration, invasion, and proliferation of ESCC cells. Conclusion The prognostic model based on MCAGs we developed serves as an effective tool for predicting outcomes in ESCC.T cell-specific MCAGs drive intratumoral heterogeneity in ESCC, serving as potential prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00815-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4482ff5484bb99ce98963d648871eb4d8f4fb2bb" target='_blank'>
              Integrative single-cell and bulk transcriptomic analysis reveals the landscape of T cell mitotic catastrophe associated genes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Shuang Li, Zheng Tao, Nan Wang, Yazhou Liu, Kai Xie, Haitao Ma
          </td>
          <td>2025-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Variations in DNA methylation within the DNA damage response (DDR) mechanism could have significant implications for glioma prognosis and immune responses. This study aimed to explore the global DNA methylation landscape of DDR genes in gliomas and identify key epigenetically regulated genes influencing glioma biology and immunity. Methods This study incorporated a range of public and local glioma datasets. Multiple clinical, bioinformatic, and in vitro experimental analyses were conducted to explore clinical and biological aspects. Results Global DNA methylation variations in DDR genes correlated with distinct glioma prognoses, with five CpGs identified as potent predictors. Hierarchical clustering and a risk-score model based on these CpGs unveiled immune-related prognostic subgroups in glioblastomas (GBMs) and lower-grade gliomas (LGGs). NSUN5, epigenetically regulated by one of these CpGs, highlighted the biological significance of the DDR CpG panel. In vitro, NSUN5 displayed tumor-suppressor-like activities in GBM cells, but clinically, it was an unfavorable prognostic marker. Depletion of NSUN5 shifted cytosolic DNA sensing from a STING-dependent (cGAS-STING) pathway to a STING-independent (DNA-PK-HSPA8) pathway, leading to a delayed but more robust type I interferon (IFN) response in GBM cells and enhancing microglial M1 polarization and chemotaxis. This may partially account for the functional discrepancy of NSUN5 observed between experimental and clinical contexts. Conclusion This study highlights the complex interplay between DNA methylation, the DDR mechanism, cytosolic DNA sensing, and glioma immunity. These findings may inspire novel strategies for DNA sensing-based immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06360-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc54757ac8e99a0979bc781abac88c0c6433dc38" target='_blank'>
              DNA methylation variations of DNA damage response in glioblastoma: NSUN5 modulates tumor-intrinsic cytosolic DNA-sensing and microglial behavior
              </a>
            </td>
          <td>
            An-An Yin, Yan-Qiu Yao, Yi-Feng Liu, Yu-Sha Ji, Yalong He, Tian-Chi Ma, Wen-Heng Guo, A. Etcheverry, M. Aubry, Jean Mosser, Wei Lin, Yuan-Ming Wu, Kun Chen
          </td>
          <td>2025-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Despite the clear genetic evidence linking serous tubal intraepithelial carcinoma (STIC) and high-grade serous ovarian carcinoma (HGSOC), the specific conditions and events that promote the progression of STIC lesions into invasive disease remain poorly understood.



 As a critical initial step, we have assembled a cohort of incidental p53 signatures (p53.I), incidental STIC (STIC.I), and STIC with concurrent HGSOC (STIC.C). We have performed extensive multi-modal analysis using multiplexed tissue imaging and spatial transcriptomics to identify features of the immune system that play a vital role in the early steps of HGSOC development. We have processed 43 specimens using highly multiplexed tissue imaging at single-cell resolution (cyclic immunofluorescence, CyCIF), and 35 specimens for micro-regional spatial whole transcriptomics using the GeoMx (Nanostring) on over 500 pathologist-annotated regions of interest.



 We showed a temporal evolution of HGSOC, including the activation of the interferon (IFN) signaling pathway underpinning the progression from the very early stage, p53.I. Additional cancer hallmark pathways, such as Transforming Growth Factor Receptor-β, Hypoxia, were induced in the tumor microenvironment (TME), including STIC.C. One of the mechanisms of the early induction of IFN might be induced by DNA damage response and chromosomal instability, potentially activating the cGAS-STING pathway. We validated IFN activation at the early stage of HGSOC by tissue imaging. For instance, more epithelial cells expressing both HLA-A as well as HLA-E, p-TBK1 and p-STAT3 with disease progression than FT regardless of BRCA status (p<0.001, GLMMs). We found IFN signaling to be persistent at STIC.C and further upregulated in the tumor. To model the evolution of the cancer ecosystem, we then looked into immune composition at both protein and RNA levels. Our data suggested active immune surveillance and potential elimination for p53 signatures. Immune surveillance was potentially further increased at STIC.I by an increased activated cDC1, antigen-presenting cells (APCs), and >30% activated (PD1+) cytotoxic (CD8+ CD103-) and TRM with some level of exhaustion (LAG3+). STIC.C, presumably resembling late clonal expansion, had less intra-tumoral NK and cDC1 in both protein and RNA levels. There was a gradual decline in total CD8+ T cells with HGSOC progression, while 30-40% expressed Ki67/PD1 or LAG3 in cancer, indicating immune editing. Exhaustion state was further shown with HAVCR2 expression (TIM3). We found that one of the contributors of T cells exhaustion was chronic IFN.



 Epithelial cells with IFN activation seem to be under a natural selection with HSGSOC progression. One of the selective advantages might be the upregulation of MHC-class I. HLA-E upregulation by tumor cells was shown to be immune suppressive by reducing NK cell surveillance. Along with the gradual accumulation of cancer hallmark pathways, an emergence of immune suppressive mechanisms seems to arise at the very early precursor stage.



 Tanjina Kader, Jia-Ren Lin, Clemens Hug, Shannon Coy, Yu-An Chen, Ino de. Bruijin, Clarence Yapp, Jeremy L. Muhlich, Nikolaus Schultz, Charles W. Drescher, Peter K. Sorger, Ronny Drapkin, Sandro Santagata. Immune suppression in the fallopian tube: Insights from the pre-cancer atlas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bfa669bc19f1cf6d070d242569736aa782cd91a" target='_blank'>
              Abstract IA009: Immune suppression in the fallopian tube: Insights from the pre-cancer atlas
              </a>
            </td>
          <td>
            Tanjina Kader, Jia-Ren Lin, Clemens Hug, S. Coy, Yu-An Chen, Ino de. Bruijin, Clarence Yapp, J. Muhlich, N. Schultz, Charles W Drescher, P. Sorger, Ronny Drapkin, S. Santagata
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is a highly vascularized and immunogenic malignancy with a complex tumor microenvironment (TME) that shapes disease progression and therapeutic resistance. Despite advances in immune checkpoint inhibitors (ICIs) and targeted therapies, clinical responses remain heterogeneous, underscoring the need for a deeper understanding of RCC immunobiology. This review comprehensively examines the immunosuppressive TME of RCC, emphasizing the roles of cytotoxic and immunosuppressive immune cells, carcinoma-associated fibroblasts (CAFs), abnormal vasculature, and extracellular matrix (ECM) remodeling in fostering immune evasion. This review summarized emerging biomarkers—including PD-L1 expression, tumor mutational burden (TMB), gene mutations, and immune-based subtypes—that may predict ICI response. Furthermore, we evaluate current immunotherapeutic strategies, such as ICIs, combination therapies, and novel approaches targeting immunosuppressive cells and metabolic pathways. While combination therapies have improved outcomes, challenges like toxicity and resistance persist, necessitating biomarker-driven patient stratification and optimized treatment sequencing. Future directions should focus on deciphering TME heterogeneity and developing precision immunotherapy strategies to enhance clinical efficacy in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c635f92429f0a0ad81f2c837971a966305d3f506" target='_blank'>
              Characteristics of the tumor microenvironment and potential immunotherapy strategies in renal cell carcinoma
              </a>
            </td>
          <td>
            Hui Wen, Shi Zheng, Xiaoqin Zhu, Ling Wang, Dongping Chen
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b038428041ef5d754c8d9cbe5398a0911b1f648" target='_blank'>
              Fusion-driven oncogenic programs shape the immune landscape in translocation renal cell carcinoma.
              </a>
            </td>
          <td>
            Prathyusha Konda, Cary N. Weiss, Yantong Cui, Sayed Matar, Jinyu Wang, Yasmin L Nabil, Riva Deodhar, Jiao Li, J. Horst, Sabrina Y. Camp, A. B. Sheshdeh, Jonathan L. Hecht, David J. Einstein, Yashika Rustagi, Anwesha Nag, A. Thorner, Cheng-Zhong Zhang, E. V. Van Allen, S. Signoretti, T. Choueiri, Srinivas R Viswanathan
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Lineage plasticity, a critical hallmark of cancer progression, enables tumor cells to evade inhibition of primary oncogenic pathways through histologic transformation. This adaptive process, driven by stemness-associated features and epigenetic reprogramming, poses significant challenges in treatment. Using non-small cell lung cancer and prostate cancer as models, we examine the utility of tissue and liquid biopsies in detecting histologic transformations and tailoring treatments to specific subtypes, which has profound clinical implications, potentially improving outcomes in patients with advanced, therapy-resistant disease. We also discuss emerging therapeutic strategies, including novel molecular targets, and address ongoing clinical challenges in managing treatment-emergent histologic transformation.


SIGNIFICANCE
The advent of highly effective molecularly targeted therapies results in increased recognition of treatment-emergent histologic transformation. This review not only summarizes current evidence on diagnosis and management of lineage plasticity but also explores therapeutic strategies under study, outlining a framework for clinical translation and successful drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49dc4debf7b809a5797b2dd12ac58569636b53ca" target='_blank'>
              Histologic Transformation in Cancer: The Path for Clinical Translation.
              </a>
            </td>
          <td>
            Ioannis A. Vathiotis, Avisek Banerjee, Utsav Sen, Felix C Saalfeld, Debdatta Halder, A. Charpidou, Konstantin Syrigos, H. Horinouchi, Ashutosh K Tewari, Mark A Dawson, M. Wermke, Triparna Sen
          </td>
          <td>2025-09-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background The epidermal growth factor receptor (EGFR) plays a significant role in vasculogenic mimicry (VM), a process by which aggressive cancer cells within hypoxic solid tumors form blood vessel-like structures independent of endothelial cells. Mostly attributed to cancer stem cells (CSC), VM is strongly associated with chemoresistance and poor prognosis in glioblastomas (GBMs). The trafficking of EGFR from the plasma membrane is in part regulated by Sortilin (SORT1), a type I membrane glycoprotein with receptor sorting functions. Notably, EGFR and SORT1 have been reported to also localize within exosomes. This study questions whether the EGFR/SORT1 interplay impacts in vitro VM in human GBM-derived cell models. Methods In silico analysis was used to compare GBM to healthy brain tissue transcripts levels. cDNA arrays and RT-qPCR were performed to assess transcript levels in U87, U118, U138, and U251 GBM cells. Immunoblotting was used to assess protein expression levels. In vitro 3D VM was performed on Cultrex, while real-time chemotaxis was measured using the xCELLigence system. Transient gene silencing was achieved using sequence-specific siRNA. Results Increased levels of EGFR, along with tetraspanins exosomal markers CD9, CD63, and CD81 were observed in GBM. Interaction predictions further identified EGFR as a central hub linking SORT1 to exosomal biomarkers. Elevated SORT1 protein and gene expression were found in human stage IV GBM-derived U87, U118 and U251 cell lines. This was consistent with cDNA array analyses from clinically annotated GBM demonstrating higher SORT1 levels in stage III-IV compared to stage I-II tumors. Pharmacological inhibition of SORT1 function by AF38469 or siRNA-mediated transient silencing of SORT1 abrogated chemotactic cell migration and reduced in vitro VM in U87 and U251 GBM cells. SORT1 silencing prevented EGFR gene and protein expression upon in vitro VM, as well as reduced chemoresistance (ABCB1, ABCB5) and CSC (PROM1/CD133, SOX2) molecular signature. Conclusions These findings underscore the complex interplay of the EGFR/SORT1 axis in regulating VM and in driving to the transcriptional regulation of chemoresistance and CSC molecular signatures of GBM associated with VM. These insights could inspire novel therapeutic strategies targeting the EGFR/SORT1 complex in GBM. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14789-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a6caf1a83d258c60ec2d74b78b8590a89ca03b1" target='_blank'>
              A SORT1/EGFR molecular interplay as a prognostic biomarker in glioblastoma vasculogenic mimicry: implications in the transcriptional regulation of cancer stemness and drug resistance
              </a>
            </td>
          <td>
            Marie-Eve Roy, Alain Zgheib, N. Eliopoulos, B. Annabi
          </td>
          <td>2025-08-20</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c99e7f8f3247a328ab6891e4e20208ff1f21334e" target='_blank'>
              RIG-I-dependent tumor-intrinsic type I interferon signaling restricts growth in breast cancer 3D culture.
              </a>
            </td>
          <td>
            Katherine Liu, Lise Mangiante, Roni Levin-Konigsberg, Cristina Sotomayor-Vivas, Wenting Yang, Kaitlyn Spees, Zhicheng Ma, Jennifer L. Caswell-Jin, Christina Curtis, M. Bassik
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Background Colorectal laterally spreading tumors (LSTs) are defined as non-protruding neoplasms exceeding 10 mm in diameter that grow primarily along the intestinal wall. The morphogenetic mechanisms and evolutionary trajectories of granular-type LSTs (LST-Gs) towards colorectal carcinoma remain unclear. Methods In this study, we investigate the transcriptional features of LST-Gs using single-cell RNA sequencing technology by comparing them with protruded-type adenomas (PAs) and normal mucosal tissues. Results Adenomatous LST-Gs harbor an epithelial cell population with metaplastic differentiation, which are almost absent in PAs. Cells with a high degree of differentiation in LST-G demonstrate enhanced immunogenicity and robust adhesion/junction interactions. Furthermore, LST-Gs show upregulated expression of molecular chaperones and metallothioneins compared to PAs, reflecting a more hostile microenvironment similar to that observed in carcinoma stages. These characteristics suggest that LST-G exhibits greater heterogeneity compared to the earliest colorectal adenomas, mirroring the progression from precancerous states to cancer. Notably, the Arp2/3 complex is significantly upregulated in highly differentiated LST-G cell populations, potentially facilitating cell migration along the basement membrane, which highlights the similarities in cell motility between adenomatous LST-G and normal mucosal epithelium as well as serrated polyps (SERs). Conclusion The differentiation state of cells within LST-G exhibits a close correlation with their diverse characteristics. Metaplastic differentiation, as a prominent feature at the transcriptional level, demonstrates significant associations with the genomic features, morphogenesis, and tumor progression of LST-G.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cfb702cab41be0ca69fc6ec0389c21e705f623" target='_blank'>
              Single-cell RNA profiling of colorectal granular-type laterally spreading tumor uncovers progression trajectory toward carcinoma and transcriptional signatures favoring lateral morphogenesis
              </a>
            </td>
          <td>
            Yueqing Gong, Yuxin Zhang, Xun Liu, Rongli Cui, Jingjing Lu, Jun Li, Fang Gu, Jing Zhang, Shigang Ding, Weiwei Fu
          </td>
          <td>2025-09-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b9bfbc1d5b01f6c556fa697a0ffa5a580ef80a3" target='_blank'>
              Extracellular matrix protein 1 in cancer: multifaceted roles in tumor progression, prognosis, and therapeutic targeting.
              </a>
            </td>
          <td>
            Jue Wang, Qionglian Huang, Hanjuan Ning, Weiwei Liu, Xianghui Han
          </td>
          <td>2025-09-25</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The 16th annual Frontiers in Cancer Science conference convened leading experts to discuss the latest developments in cancer research. Key research themes included mechanisms of treatment resistance and innovative strategies to target resistant cancer cells, metabolic plasticity and its therapeutic vulnerabilities, modulation of the tumor microenvironment to enhance therapeutic efficacy, recent advances in immunotherapies and engineered immune cells, and strategies to overcome tumor immune evasion. The conference also highlighted the development of advanced spatial transcriptomic technologies as a powerful tool to decipher tumor heterogeneity and identify novel therapeutic targets. Additionally, the transformative potential of artificial intelligence and machine learning was explored in optimizing therapy selection, refining prognostication models, and improving patient outcomes, with a focus on advancing personalized, cost-effective cancer care. These collective insights underscore the rapid progress in the field and the potential for translating these discoveries into effective clinical interventions, marking a significant step toward addressing the complexities of cancer biology and enhancing patient care worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69fa9e8af00a427e4ba71206e84cf03208b94606" target='_blank'>
              Charting New Paths in Cancer Research: Insights from the Frontiers in Cancer Science Conference 2024.
              </a>
            </td>
          <td>
            Giang Le Minh, S. S. Lucky, S. Pervaiz, Kristijan Ramadan, J. Yeong, S. T. Ong, Vaidehi Krishnan, C. F. Cheok, W. Chng, S. de Mel, Xiaomeng Wang, Anand D. Jeyasekharan, Chartsiam Tipgomut, L. Kong, K. Sabapathy, Cheng Ean Chee, Koji Itahana, R. Taneja, Soo-Chin Lee
          </td>
          <td>2025-08-01</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Background Immunotherapy has progressively gained prominence as a cornerstone therapeutic modality across diverse oncological contexts, with its clinical efficacy intricately linked to the dynamic interactions between the tumor microenvironment (TME) and neoplastic cells. Central to this paradigm is angiogenesis—a quintessential hallmark of cancer—which not only sustains tumor growth but also orchestrates immunomodulatory networks within the TME, thereby profoundly influencing therapeutic responsiveness. However, in the field of bladder cancer (BC), the relationship among angiogenesis and prognosis, immunotherapy response, and immune cell infiltration remains to be further explored. Methods To systematically uncover this relationship, we carried out an exhaustive assessment of 36 genes linked to angiogenesis (AAGs) and explored the relationship between angiogenesis and transcript, prognostic outcomes, as well as the infiltration of immune cells. By constructing an AAG_score, we quantified the angiogenic subtype characteristics of each patient. Subsequently, we evaluated the value of these characteristics in foreseeing BC prognosis and the response of treatment, and concurrently analyzed the performance of AAGs in diffuse large B-cell lymphoma for comparative study. Through RT-qPCR, CCK8 and other experiments, we verified the role of NID2 in bladder cancer. Results This study explored different types of AAGs mutations in BC samples at the genetic level and elucidated the expression patterns of AAGs. Through in-depth analysis, we identified two distinct molecular subpopulations and found significant associations between AAG mutations and patients’ clinicopathological features, prognosis, and invasive TME. We found that patients with low AAG_score showed increased microsatellite instability, high mutation tendency, and immune motivation, and had a better prognosis. NID2 plays a role in promoting proliferation in bladder cancer and evaluate in diffuse large B-cell lymphoma. In addition, our study found a significant correlation between index of cancer stem cell and AAG_score in drug sensitivity. Conclusion In summary, our study successfully identified prognosis-related AAG characteristics in BC patients. These characteristics not only contribute to a clearer understanding of TME properties but also provide an important basis for exploring more effective immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e0c3a38e058744faab2e7b9e1bb3c40f92cc4d" target='_blank'>
              Angiogenesis-related gene NID2 profiling and immune infiltration in bladder cancer: prognostic implications and immunotherapy response
              </a>
            </td>
          <td>
            Jinxia Cao, Wang Zheng
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background: Anoikis is a form of programmed cell death triggered by the detachment of cells from the extracellular matrix. Anoikis resistance represents a critical factor in tumor metastasis, and elucidating the mechanisms by which epithelial cancer cells evade this process may provide a molecular insight for effectively targeting metastatic progression. Methods: Presented here are an experimental workflow and a detailed protocol to examine anoikis sensitivity in tumor cells both in vitro and in vivo. We described a detachment-induced anoikis model, a three-dimensional spheroid culture system, and an in vivo circulating tumor cell assay, by using the human lung carcinoma cell line A549 as a model system. We detailed the cell culture conditions, materials, and sample preparation, and the evaluation and quantification of anoikis. Together, these methods provide a comprehensive approach for investigating anoikis resistance. Conclusion: This protocol offers valuable insights into the mechanisms underlying anoikis resistance and may facilitate the identification of novel therapeutic targets for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44845c475c780a2f717aad7ae39727b62d7d7bbc" target='_blank'>
              An experimental workflow for investigating anoikis resistance in cancer metastasis
              </a>
            </td>
          <td>
            Xue Han, Yipan Zheng, Xiaohui Si, Zhe Liu
          </td>
          <td>2025-08-05</td>
          <td>Journal of Biological Methods</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/704c0384d0353e0bc729ca17a587131359cbb749" target='_blank'>
              TCR-engineered T cells targeting a shared β-catenin mutation eradicate solid tumors.
              </a>
            </td>
          <td>
            Maria Stadheim Eggebø, Julia Heinzelbecker, Heyilimu Palashati, Nicholas Chandler, T. Tran, Yingqian Li, Weiwen Yang, Maarja Laos, Isaac Blaas, E. Rustad, R. C. Bollineni, Marina Delic-Sarac, Fridtjof Lund-Johansen, M. M. Nielsen, Johanna Olweus
          </td>
          <td>2025-08-27</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Despite advances in medicine, cancer remains one of the foremost global health concerns. Conventional treatments like surgery, radiotherapy, and chemotherapy have advanced with the emergence of targeted and immunotherapy approaches. However, therapeutic resistance and relapse remain major barriers to long-term success in cancer treatment, often driven by cancer stem cells (CSCs). These rare, resilient cells can survive therapy and drive tumour regrowth, urging deeper investigation into the mechanisms underlying their persistence. CSCs express ion channels typical of excitable tissues, which, beyond electrophysiology, critically regulate CSC fate. However, the underlying regulatory mechanisms of these channels in CSCs remain largely unexplored and poorly understood. Nevertheless, the therapeutic potential of targeting CSC ion channels is immense, as it offers a powerful strategy to disrupt vital signalling pathways involved in numerous pathological conditions. In this review, we explore the diverse repertoire of ion channels expressed in CSCs and highlight recent mechanistic insights into how these channels modulate CSC behaviours, dynamics, and functions. We present a concise overview of ion channel-mediated CSC regulation, emphasizing their potential as novel diagnostic markers and therapeutic targets, and identifying key areas for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74bc88d91ef29f0fa1f654c524e84092e0b4e373" target='_blank'>
              Deciphering the Role of Functional Ion Channels in Cancer Stem Cells (CSCs) and Their Therapeutic Implications
              </a>
            </td>
          <td>
            Krishna Samanta, Gali Sri Venkata Sai Rishma Reddy, Neeraj Kumar Sharma, Pulak Kar
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The dynamic evolution of the immune tumor microenvironment (TME) during targeted therapy is a critical yet poorly understood determinant of treatment response and resistance. While most studies compare immune states before and after treatment, temporal immune changes during therapy remain largely uncharacterized, limiting development of effective combination strategies. Here, we investigated immune dynamics throughout targeted therapy using mouse melanoma models that recapitulate human therapeutic responses. Single-cell RNA sequencing (scRNA-seq) identified an inflection point where the inflamed TME during tumor regression, characterized by robust NK cell infiltration, transitions to an immune-excluded state upon onset of drug-tolerant residual disease. We uncovered a unique macrophage subset that orchestrates NK cell recruitment during regression. Depletion of these macrophages using LysM-cre;iDTR mice significantly reduced NK cell infiltration. Extending these findings to human cancer, longitudinal scRNA-seq analysis of melanoma and lung cancer patient samples revealed dynamic NK cell infiltration during targeted therapy, establishing a direct link between innate immune remodeling and treatment outcome. Unlike prior prognostic studies assessing immune states at single time points, our results provide mechanistic evidence of a temporal relationship between NK cell infiltration and therapeutic efficacy. Together, these findings position immune evolution as a driver of acquired resistance and identify macrophage–NK cell crosstalk as a therapeutically actionable axis to overcome immune exclusion and improve targeted therapy.



 Chia-Hsin Hsu, Andrew C. White. Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064d31f044dc7aac267bdcec71d797f7f005e82d" target='_blank'>
              Abstract A012: Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions
              </a>
            </td>
          <td>
            , 
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Therapies for diffuse glioma induce DNA damage response (DDR), and strategies to exploit DDR defects are active areas of investigation. While global DNA methylation profiling effectively classifies gliomas into subtypes, the epigenetic and gene expression patterns of DDR genes, and their contribution to tumor classification and outcomes, have yet to be fully elucidated. Thus, dissecting the DDR epigenetics, gene expression, and single-cell heterogeneity may reveal key molecular characteristics, refine prognosis, and identify novel treatment strategies and resistance mechanisms.


METHODS
We characterized DDR epigenetics and gene expression of TCGA glioblastomas (GBM) and low-grade gliomas (LGG). Single-cell protein analysis by imaging mass cytometry (IMC) was performed on a separate cohort of 118 diffuse gliomas.


RESULTS
Analysis of TCGA cohorts revealed two DDR methylation groups that correlated with IDH mutation status and previously reported molecular groups. DDR transcription profiling further classified tumors into four groups. Those with high DDR transcription across pathways were linked to poor survival independent of IDH or MGMT status, and potentially improved prognostication beyond established biomarkers. Single-cell characterization of a separate cohort revealed intratumoral DDR diversity and identified proliferative tumor cells with high DDR protein expression across pathways that are associated with unfavorable grade and survival.


CONCLUSIONS
Tumor-level epigenetic and transcriptional DDR signatures alone can distinguish molecular-defined diagnosis and outcomes of gliomas beyond established biomarkers. A higher abundance of glioma cells with high DDR effector expression across pathways is associated with poor survival. Thus, clinical assessment of pan-DDR expression may inform prognosis and identify potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b0fbeaa6fa3878bda82552131f1d944fc87086" target='_blank'>
              DNA damage response profile distinguishes poor-acting gliomas with shared methylome signatures.
              </a>
            </td>
          <td>
            Nalin Leelatian, Charu Singh, Richard Bouffard, R. Sundaram, Kirsten E Diggins, William Sullivan, Sateja Paradkar, Zeynep E. Erson Omay, B. Mobley, Susan E. Gueble, Juan C Vasquez, R. Bindra
          </td>
          <td>2025-08-27</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Intratumoral heterogeneity remains a major barrier to durable cancer therapies, largely driven by the persistence of cancer stem cells (CSCs). In this study, we employed an integrated, multi-scale approach to investigate how melanoma CSCs respond to siRNA-mediated silencing of three key regulatory genes: KLF4, SHH, and HIF1α. Using a combination of morphological, molecular, spectroscopic, and elemental analyses, we explored structural and biochemical consequences of gene knockdown. Gene silencing resulted in significant changes in cell shape and size, reduced F-actin organization, and decreased PFN1 expression, indicating a loss of stem-like properties. ATR-FTIR spectroscopy revealed shifts in biomolecular composition, notably a reduction in amide III intensity and an increase in lipid ester content. SEM-EDS point-based elemental analysis revealed SEM-EDS point-based elemental analysis revealed relative differences in carbon and nitrogen levels between selected central and peripheral regions of silenced and control cells, at the micron-scale working depth, reflecting broader elemental distribution trends rather than precise subcellular compartmentalization. XPS analysis further confirmed these differences, providing additional insights into the elemental composition of the cellular surface. The integration of FTIR spectroscopy into this study highlights the potential of infrared spectroscopy as a powerful tool in cancer research. These findings demonstrate that targeting critical regulatory pathways induces cytoskeletal and biochemical remodelling in melanoma CSCs, offering a multi-dimensional perspective on cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5835a6c3f340e01657d8039c9562c5a0b45cfc" target='_blank'>
              Integrated spectroscopic and morphological analyses reveal cellular shifts in gene-silenced melanoma CSCs
              </a>
            </td>
          <td>
            Berrin Ozdil, Günnur Güler, E. Ataman, H. Aktug
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Malate dehydrogenase 1 (MDH1), an NAD(H)-dependent isoenzyme, is a key component of the malate-aspartate shuttle (MAS). A significant association has been observed between MDH1 expression and various characteristics of the tumor microenvironment across different cancer types. Methods This study provides comprehensive pan-cancer analyses exploring the expression patterns, clinical and pathological correlations, genetic alterations, immunogenomic profiles, single-cell dynamics, alternative splicing signatures, and pharmacological sensitivities related to MDH1. Drug sensitivity profiling and molecular docking techniques have been employed to identify potential anti-cancer compounds targeting MDH1. Experiments have also been conducted to investigate the biological function of MDH1 in lung adenocarcinoma (LUAD) and to confirm the interaction between MDH1 and macrophages using immunofluorescence assays. Results MDH1 expression levels are elevated across a wide range of malignancies, and overexpression of MDH1 was consistently linked to poor prognosis in multiple cancer subtypes. Moreover, MDH1 expression shows complex correlations with various immune cell populations, particularly macrophages, and cohort analysis of both bulk and pan-cancer single-cell immunotherapy data suggest that MDH1 could serve as a predictive marker for immunotherapy responses. Moreover, knockdown of MDH1 suppresses macrophage invasion. To investigate the role of MDH1 in LUAD cells, a potential inhibitor of MDH1 was identified, BI-2536, and has been confirmed to impact MDH1 activity and impede the growth of LUAD cells. Conclusion Our findings indicate that MDH1 may serve as a potential prognostic marker and a promising target for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3176f0dff692eaebf853d9454461d636ffecd8e7" target='_blank'>
              Panoramic view of MDH1: driving cancer progression and shaping the tumor immune microenvironment
              </a>
            </td>
          <td>
            Yunchen Lou, Yunwei Lou, Yao Cheng, Beining Xu, Hanbin Chen, Yinwei Dai
          </td>
          <td>2025-08-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Prostate cancer accounts for approximately 1.5 million new diagnoses and 400,000 deaths every year worldwide, and demographic projections indicate a near-doubling of both figures by 2040. Despite existing treatments, 10–20% of patients eventually progress to metastatic castration-resistant disease (mCRPC). The median overall survival (OS) after progression to mCPRC drops to 24 months, and efficacy drops severely after each additional line of treatment. Omics platforms have reached advanced levels and enable the acquisition of high-resolution large datasets that can provide insights into the molecular mechanisms underlying PCa pathology. Genomics, especially DDR (DNA damage response) gene alterations, detected via tissue and/or circulating tumor DNA, efficiently guides therapy in advanced prostate cancer. Given recent developments, we have performed a comprehensive literature search to cover recent research and clinical trial reports (over the last five years) that integrate omics along three converging trajectories in therapeutic development: (i) predicting response to approved agents with demonstrated survival benefits, (ii) stratifying patients to receive therapies in clinical trials, (iii) guiding drug development as part of drug repurposing frameworks. Collectively, this review is intended to serve as a comprehensive resource of recent advancements in omics-guided therapies for advanced prostate cancer, a clinical setting with existing clinical needs and poor outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a974f7234fe451d49d30d3e981f74c6e7ec7c75" target='_blank'>
              Omics-Mediated Treatment for Advanced Prostate Cancer: Moving Towards Precision Oncology
              </a>
            </td>
          <td>
            Yasra Fatima, Kirubel Nigusu Jobre, Enrique Gómez-Gómez, B. Małkiewicz, A. Vlahou, M. Mokou, H. Mischak, M. Frantzi, Vera Jankowski
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor stroma can be actively shaped by tumor cells to adopt pro-tumorigenic properties, making it a focal point of cancer research. Among the stromal components, cancer-associated fibroblasts (CAFs) -- often the most abundant cell type in the tumor microenvironment (TME) -- play a pivotal role in promoting cancer progression through their interactions with tumor cells and other stromal components. Elucidating the mechanisms behind these interactions requires robust methods for isolating and characterizing CAFs, which can also lay the foundation for developing novel CAF-targeted cancer therapies. This study presents a method for isolating fibroblasts from both the tumor and the adjacent normal regions of radical prostatectomy specimens obtained from prostate cancer (PCa) patients. This approach enables the concurrent isolation of CAFs and their normal counterparts (NFs) from the same individual, providing a paired experimental system. Detailed protocols are provided for the maintenance of primary CAFs and NFs for downstream in vitro analyses, as well as for their immortalization to facilitate long-term studies. We also describe functional assays designed to compare the effects of CAFs and NFs on cancer cell behaviors, including proliferation, migration, and anchorage-independent growth. Two complementary approaches are detailed: treatment of cancer cells with fibroblast-derived conditioned media (CM), and direct co-culture of fibroblasts with tumor cells. Together, these methodologies represent a comprehensive toolkit for investigating the dynamic interplay between CAFs and tumor cells, not only in PCa but potentially across a range of human cancers. Moreover, molecular characterization of these interactions may reveal key mediators of CAF-driven oncogenesis, offering promising targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2d8dc58b820e48882f34f8df6b8e5bd76919a2" target='_blank'>
              Isolation, Culture, and Characterization of Prostate Cancer-Associated Fibroblasts.
              </a>
            </td>
          <td>
            Somayeh Mirzaaghaei, L. Avalle, Chiara Verrengia, Jahnavi Srivatsa, Laura Conti, Marta Gai, Chiara Fiameni, Paolo Gontero, Umberto Mortara, Luca Molinaro, Mauro Papotti, Valeria Poli
          </td>
          <td>2025-08-01</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Metastasis remains a leading cause of morbidity and mortality in patients diagnosed with cancer. A variety of in vitro and in vivo approaches have been employed to study the individual steps of the metastatic cascade. However, these methodologies are sometimes limited in their ability to recapitulate the biological complexity and heterogeneity of human tumor biology. As a result, significant knowledge gaps still exist regarding the development, growth and evolution of treatment resistance in metastatic tumors. In this Perspective, we discuss the benefits and drawbacks of current, widely used techniques to model metastatic disease. We also highlight novel approaches utilized in recent studies to confront the limitations posed by classic modeling techniques. Ultimately, we provide suggestions for ensuring scientific rigor and reproducibility in metastasis studies, and we propose key areas of focus for developing next-generation models of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10d2e8655ad96014a93526cd27ef8255240f4d2b" target='_blank'>
              Modeling metastasis – leveraging novel tools to streamline discovery in advanced cancer
              </a>
            </td>
          <td>
            Nicole M Eskow, Eva Hernando
          </td>
          <td>2025-08-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lymphoid neoplasms (LNs) are heterogeneous malignancies arising from lymphoid cells, displaying diverse clinical and molecular features. Although LNs are collectively frequent, individual subtypes are rare, posing challenges for genetic association studies. Indeed, genome-wide association studies (GWAS) explained only a fraction of the heritability. Shared genetic susceptibility and overlapping risk factors suggest a partially common etiology across subtypes. We employed a multi-trait GWAS strategy to improve discovery power by leveraging pleiotropy among LN subtypes. We defined LN phenoclusters based on cell of origin, somatic mutation profiles, and approved therapeutic agents. Using data from three large cohorts—the UK Biobank, Million Veteran Program, and FinnGen—we analyzed 31,937 LN cases and 1.2 million controls across 8 individual subtypes and 7 phenoclusters. We replicated the novel associations in two independent cohorts (All of Us and the Prostate, Lung, Colorectal, and Ovarian Cancer Screening Trial) with 2892 LN cases and 165,791 controls. We identified 76 genome-wide significant loci for individual subtypes or subtype clusters, including 20 novel associations. We identified the subtypes contributing to each locus, putative candidate causal variants, and genes underlying the associations, and found enrichment of specific cell types, biological processes, and drugs associated with LN risk genes. Overall, this study identified new LN genetic risk loci and candidate genes, providing insights that may inform novel therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8ac41083338a114514b280145d44117043358f" target='_blank'>
              Clustering of lymphoid neoplasms by cell of origin, somatic mutation and drug usage profiles: a multi-trait genome-wide association study
              </a>
            </td>
          <td>
            Murat Güler, F. Canzian
          </td>
          <td>2025-08-29</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="The convergence and interplay of pyroptosis, apoptosis, and necroptosis have led to the conceptualization of PANoptosis, an innovative paradigm of inflammatory programmed cell death. Characterized by the hierarchical assembly and activation of the PANoptosome, PANoptosis operates through tightly orchestrated signaling hubs and is intricately linked to organelle functionality. Accumulating evidence underscores its pivotal role in diverse oncogenic processes, positioning PANoptosis as a compelling frontier for antitumor therapeutic exploration. This review delineates the mechanistic underpinnings of PANoptosis, synthesizes its established contributions to tumor progression, and examines its dynamic crosstalk with the tumor immune microenvironment (TIME). Notably, we highlight recent breakthroughs in PANoptosis-driven immunotherapeutic strategies. We further propose that targeting PANoptosis to reprogram TIME represents a transformative approach in oncology, shifting the research paradigm from unimodal cell death regulation to multidimensional intervention. This perspective not only advances fundamental understanding but also holds significant promise for clinical translation, heralding a new era in cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707a3f9d7d1d3e0ffa5a09384319308519598436" target='_blank'>
              A potential strategy to rebuild the tumor immune microenvironment: PANoptosis
              </a>
            </td>
          <td>
            Siyu Wu, Boyan Tian, Xueying Pang, Bowen Sui
          </td>
          <td>2025-08-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aim: Growing evidence points to non-genetic mechanisms underlying long-term resistance to cancer therapies. These mechanisms involve pre-existing or therapy-induced transcriptional cell states that confer resistance. However, the relationship between early transcriptional responses to treatment and the eventual emergence of resistant states remains poorly understood. Furthermore, it is unclear whether such early resistance-associated transcriptional responses are evolutionarily conserved. In this study, we examine the similarity between early transcriptional responses and long-term resistant states, assess their clinical relevance, and explore their evolutionary conservation across species. Methods: We integrated datasets on early drug responses and long-term resistance from multiple cancer cell lines, bacteria, and yeast to identify early transcriptional changes predictive of long-term resistance and assess their evolutionary conservation. Using genome-wide CRISPR-Cas9 knockout screens, we evaluated the impact of genes associated with resistant transcriptional states on drug sensitivity. Clinical datasets were analyzed to explore the prognostic value of the identified resistance-associated gene signatures. Results: We found that transcriptional states observed in drug-naive cells and shortly after treatment overlapped with those seen in fully resistant populations. Some of these shared features appear to be evolutionarily conserved. Knockout of genes marking resistant states sensitized ovarian cancer cells to Prexasertib. Moreover, early resistance gene signatures effectively distinguished therapy responders from non-responders in multiple clinical cancer trials and differentiated premalignant breast lesions that progressed to malignancy from those that remained benign. Conclusion: Early cellular transcriptional responses to therapy exhibit key similarities to fully resistant states across different drugs, cancer types, and species. Gene signatures defining these early resistance states have prognostic value in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4c9b8cf4452549a749a1c9bb16bff08ecee299" target='_blank'>
              Resistance signatures manifested in early drug response across cancer types and species
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Arashdeep Singh, MM Gottesman, Sridhar Hannenhalli
          </td>
          <td>2025-08-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark, with a pervasive presence across cancers. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate the behaviors and functions of ecDNA. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction, whereas we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA. Significance: ecDNA is an essential factor in cancer progression. We found that a group of cancer cells with ecDNA is selectively depleted after lentiviral infection. This finding provides promise for ecDNA-specific targeting, suggests the need for caution in using lentivirus, and offers alternative ways to study ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb47b6253f4e8e1db726d8195373534a90d452c" target='_blank'>
              Selective Depletion of Cancer Cells with Extrachromosomal DNA via Lentiviral Infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, Emmanuel E Korsah, Noah A Dusseau, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Background To explore the differences in tumor immune microenvironment between different KRAS mutation subtypes, and to provide a new direction for clinical immunotherapy of different subtypes of colorectal cancer. Methods We examined the spatial distribution of common inflammatory cell markers of CD8 + T cells, CD4 + T cells, natural killer cells, and B cells and macrophages in 55 colorectal cancer patients with different KRAS gene phenotypes using immunohistochemistry and panoramic scanning. We analyzed the relationship between inflammatory cells and clinical information. TCGA and String online databases were used to analyze the relationship between KRAS mutation subtypes and inflammatory cell markers. Results We observed that significant differences in the spatial distribution of the tumor invasion front and the immune cell infiltration within the tumor. Colorectal cancer patients with different KRAS mutations showed different immune infiltration, and the main cells with differences were FOXP3 regulatory T cells and M2 macrophages. Furthermore, these two cell types were strongly associated with the prognosis of KRAS wild-type and KRAS mutant colorectal cancer. Conclusions The complexity of the immune microenvironment cannot be explained by infiltration of specific cell types alone, but may arise from cell-to-cell interactions or changes in the proportion of different immune cells. However, the infiltration of FOXP3 cells and M2 macrophages probably accounts for the differences between KRAS mutant subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03be9a5c418e895db04c9d1a928c80c7d3a288ec" target='_blank'>
              Analysis of the immune microenvironment in colorectal cancer with different KRAS gene subtypes
              </a>
            </td>
          <td>
            Jing Huang, Qian Gong, Qingshu Li, M. Xiao, Ming Li, Shuxian Zhang, Yalan Wang, Yi Tang
          </td>
          <td>2025-08-04</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The metastatic tumor microenvironment (TME) is a highly dynamic and heterogeneous ecosystem that plays a critical role in promoting cancer cell colonization, immune escape, and resistance to therapy. Recent advances in multi-omics technologies—including genomics, transcriptomics, epigenomics, proteomics, and metabolomics—have enabled a systems-level understanding of the molecular reprogramming that occurs in the TME following metastasis. In this review, we systematically summarize emerging findings from recent multi-omics studies that dissect cellular composition, signaling pathways, immune landscape, and metabolic rewiring within the metastatic TME. We highlight key molecular signatures and intercellular interactions that drive metastatic progression and therapy resistance. Furthermore, we discuss how integrative multi-omics data are being leveraged to identify actionable targets and to design novel immunotherapeutic and molecular precision strategies tailored to the metastatic niche. These insights provide a scientific rationale for the development of TME-targeted approaches in the treatment of advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1202cb3e5acfc52d10d5fc247d4fca996a2d6f2e" target='_blank'>
              Multi-omics analysis of the tumor microenvironment after metastasis: advancing toward personalized immunotherapy and molecular targeted strategies
              </a>
            </td>
          <td>
            Qing Gao, Huqiang Wu, Li Duan, Guanhua Lv, Dongmei Quan
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Discovery of cancer driver pathways is essential for targeted therapies, since these pathways govern tumor progression and treatment resistance. However, their context-specific patterns across populations remain poorly understood. Leveraging pan-cancer genomic data, we apply our two models, EntCDP and ModSDP, to perform stratified analyses from four perspectives: region, tumor type, age group, and risk factors. Our results reveal the regional biases in perturbed pathways, such as PI3K-Akt in Chinese patients and GPCR in American patients with bladder cancer. Subtype comparisons highlight the mTOR signaling in lung adenocarcinoma and the FoxO signaling in lung squamous cell carcinoma. Pediatric-adult comparisons emphasize the enrichment of Ras signaling in pediatric acute myeloid leukemia and PAK signaling in pediatric glioblastoma, respectively. Risk factor associations further link Notch-mediated pathways to alcohol consumption and CDKN-regulated pathways to obesity-related cancers. Our findings demonstrate the utility of stratified driver pathway analysis in uncovering common and specific mechanisms, which can help prioritize context-aware therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e088118e277d04d3b9c6fe00f85dd703f5be1ff1" target='_blank'>
              Multi-context modeling of driver pathways reveals common and specific mechanisms across 23 cancer types
              </a>
            </td>
          <td>
            Wenjia Zhou, Junhua Zhang
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Wilms tumor (WT), the most common pediatric renal malignancy, exhibits a relatively low mutational burden compared to adult cancers, which hinders the development of targeted therapies. To elucidate the molecular landscape of WT, we perform integrative proteomic, phosphoproteomic, transcriptomic, and whole-exome sequencing analyses of WT and normal kidney tissue adjacent to tumor. Our multi-omics approach uncovers prognostic genetic alterations, distinct molecular subgroups, immune microenvironment features, and potential biomarkers and therapeutic targets. Proteome- and transcriptome-based stratification identifies three molecular subgroups with unique signatures, correlating with different histopathological subtypes and putative cellular origins at different stages of embryonic kidney development. Notably, we identify EHMT2 as a promising prognostic biomarker and therapeutic target associated with epigenetic regulation and Wnt/β-catenin pathway. In this work, we provide a comprehensive molecular characterization of WT, offering valuable insights into its pathogenesis and a foundational resource for future therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3be681b81ddbc1d74bdc72b4011124ef4e90944" target='_blank'>
              Integrative proteogenomic characterization of Wilms tumor
              </a>
            </td>
          <td>
            Cheng Cheng, Li Zhang, Xiaofeng Chang, Kai Chen, Tian He, Jia Shi, Fan Lv, Lijia Pan, Yangkun Wu, Qianqian Cheng, Dong Ren, Y. Guo, Weiping Zhang, Huanmin Wang, Tieliu Shi, Jing Li, Xinxin Ni, Yeming Wu, Yaqiong Jin, Zhixiang Wu
          </td>
          <td>2025-08-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of chronic inflammation-associated cancers (CIACs) is shaped by cycles of injury and maladaptive repair, yet the principles organizing fibrotic stroma in these tumors remain unclear. Here, we applied the concept of hot versus cold fibrosis, originally credentialed in non-cancerous fibrosis of heart and kidney, to lung squamous cell carcinoma (LUSC), a prototypical CIAC. Single-cell transcriptomics of matched tumor and adjacent-normal tissue from 16 treatment-naive LUSC patients identified a cold fibrotic architecture in the LUSC TME: cancer-associated fibroblasts (CAFs) expanded and adopted myofibroblast and stress-response states, while macrophages were depleted. This macrophage-poor, CAF-rich stroma was maintained by CAF autocrine growth factor loops, including TIMP1, INHBA, TGFB1, and GMFB. In parallel, the immune compartment exhibited a hot tumor phenotype with abundant T and B cells, forming spatially distinct but molecularly engaged networks with CAFs. CAF gene programs typifying cold fibrosis in LUSC were conserved in other CIACs, including esophageal and gastric adenocarcinomas. These results redefine desmoplastic regions of tumors through the lens of a non-cancer fibrosis model, demonstrating that conserved stromal circuits constitute therapeutic vulnerabilities in CIACs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ed36e6ea62839a2c887c4f2b23f5bf7ed74c7d" target='_blank'>
              A fibroblast-centric network drives cold fibrosis in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Shoval Miyara, Shachaf Frenkel, Avi Mayo, Philippe Gascard, Michael K. Strasser, David Gibbs, Eviatar Weizman, Itay Ben Shalom, Yan Stein, Deng Pan, Joseph A. Caruso, V. Sangwan, Nicholas Bertos, Julie Bérubé, Sophie Camilleri-Broet, Spyridon Oikonomopoulos, H. Djambazian, K. Umansky, Jacob Elkahal, Shimrit Mayer, Pierre Fiset, Jiannis Ragoussis, Miri Adler, Eldad Tzahor, Sui Huang, Lorenzo Ferri, T. Tlsty, Ruth Scherz-Shouval, U. Alon
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Metastatic liver cancer (MLC) remains a leading cause of cancer-related mortality due to the liver’s unique immunotolerant microenvironment and high vascularization. Key mechanisms involve KC-mediated fibronectin deposition, neutrophil extracellular traps (NETs), and MDSC-driven T-cell exhaustion. Clinically, therapeutic strategies targeting the tumor microenvironment (TME) such as CSF1R inhibition, CCR2/CCR5 blockade, and CD40 agonism show promise in preclinical and early-phase trials, especially when combined with immunotherapy. However, challenges remain in overcoming systemic immunosuppression. This review summarizes the dual roles of hepatic immune cells including Kupffer cells (KCs), neutrophils, and myeloid-derived suppressor cells (MDSCs) in either suppressing or promoting metastatic colonization. We elucidate how the liver’s immunological balance, governed by innate and adaptive responses, shifts toward immunosuppression during metastasis, fostering a pro-tumor niche. This synthesis of immunological insights underscores the potential of TME-modulating therapies to improve outcomes in MLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106592c4224525a57d0ce8f76cd85d021982d4ff" target='_blank'>
              Immune microenvironment regulation and clinical immunotherapy strategies of metastatic liver cancer
              </a>
            </td>
          <td>
            Dan Liu, Mingzhu Li, Ying Liang, Fang Xu, Runtian Li, Yang Sun
          </td>
          <td>2025-09-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ffde0bbfbd90bb0bf5e018161ce107f39f7740f" target='_blank'>
              Assessing the current molecular understanding of therapeutic targets in osteosarcoma.
              </a>
            </td>
          <td>
            Chao Zhang, Ji-Lan Yang
          </td>
          <td>2025-08-07</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d43ce8f81cf0d90a92d73349eebd57717f9bf249" target='_blank'>
              Decoding adenomyosis pathogenesis using an assembloid model.
              </a>
            </td>
          <td>
            Yiliang Xu, Tao Cheng, Jianzhang Wang, Zhiruo Qiu, Xiaohong Guan, Yayuan Yu, Jiacheng Shen, Fangfang Xu, Xiaohong Jiang, Dandan Bai, Mingzhu Wang, Shuyan Mei, Hong Wang, Xiaocui Xu, Li Wu, Shaorong Gao, Xuan Che
          </td>
          <td>2025-08-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Background The progression and prognosis of early‐stage lung adenocarcinoma are closely associated with histologic subtypes, yet the presence of mixed histologic patterns often complicates prognostic assessment. Currently, the correlation between molecular and histologic features remains poorly understood. Methods Formalin‐fixed paraffin‐embedded (FFPE) samples were collected from patients with primary early‐stage lung adenocarcinoma, encompassing three histologic subtypes: well‐differentiated, moderately differentiated, and poorly differentiated. The GeoMx Digital Spatial Profiler platform was utilized to obtain spatial transcriptomic profiling. Regions of interest were carefully selected and further subdivided into three categories of areas of interest, specifically epithelial cell‐enriched regions, macrophage‐enriched regions, and other regions. Multiplex immunofluorescence (mIF) assays were employed to validate the obtained results. Results Distinct molecular characteristics were identified in tumor epithelial‐ and macrophage‐enriched compartments spanning well‐differentiated to poorly differentiated tumors. In poorly differentiated tumors, we observed enrichment of pathways related to humoral immune response, complement activation regulation, and extracellular matrix receptor interaction pathways, all of which are significantly associated with poorer prognosis. We integrated these pathways to develop a composite molecular signature that strongly correlate with adverse prognosis. Conclusions Our results provide new insights into the link between molecular and histologic subtypes in mixed‐type lung adenocarcinomas. Specifically, the identified molecular signatures offer potential biomarkers for predicting disease progression and prognosis, thus facilitating more precise and personalized therapeutic approaches. Key points Poorly differentiated components in mixed‐type early‐stage lung adenocarcinoma (LUAD) are characterized by enrichment of humoral immune response, complementactivation regulation, and extracellular matrix receptor interaction pathways, which areassociated with worse prognosis. A composite molecular signature integrating the keypathways strongly correlates with adverse clinical outcomes, serving as apotential prognostic biomarker. Digital spatial transcriptomics reveals spatially resolvedmolecular heterogeneity in tumor epithelial‐ and macrophage‐enrichedcompartments, bridging the gap between histologic subtypes and molecularmechanisms in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec35bb82932a3767cb2f73edc45c10c5b4e72637" target='_blank'>
              Spatial transcriptomic analysis across histological subtypes reveals molecular heterogeneity and prognostic markers in early‐stage lung adenocarcinoma
              </a>
            </td>
          <td>
            Hua Geng, Wenhao Zhou, Haitao Luo, Jiaqian Wang, Shixiong Li, Congcong Song, Yujie Zhao, Meilin Xu
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most lethal primary brain tumor, owing to profound intratumoral heterogeneity and the limited efficacy of standard treatments. The mesenchymal (MES) molecular subtype is particularly aggressive, exhibiting heightened invasiveness, therapy resistance, and dismal patient survival compared with the proneural (PN) subtype. Emerging evidence implicates the High Mobility Group Box 1 (HMGB1) protein and its cognate receptor, the Receptor for Advanced Glycation End Products (RAGE), as drivers of malignant progression, yet their contribution to the PN-to-MES transition is incompletely defined. We integrated transcriptomic analyses of TCGA-GBM and TCGA-LGG cohorts with immunohistochemistry on in-house patient specimens. Functional studies in patient-derived and established GBM cell lines included migration and invasion assays, tumorsphere formation assays, shRNA knockdowns, and Seahorse XF metabolic profiling to interrogate the HMGB1-RAGE axis. HMGB1 and RAGE expression was markedly elevated in MES GBM tissues and cell lines. Importantly, higher HMGB1 expression correlated with shortened overall survival (p < 0.009). HMGB1 silencing curtailed cell motility and downregulated core epithelial-to-mesenchymal transition markers (N-cadherin, Snail). RAGE knockdown diminished tumorsphere formation efficiency and reduced transcription of stemness genes (OCT4), underscoring its role in sustaining tumor-initiating capacity. Metabolically, HMGB1/RAGE activation boosted both mitochondrial respiration and glycolysis, conferring the bioenergetic flexibility characteristic of MES GBM. The HMGB1-RAGE signaling axis orchestrates mesenchymal identity, invasiveness, stem cell-like properties, and metabolic reprogramming in GBM. Targeting this pathway may disrupt the PN-to-MES transition, mitigate therapeutic resistance, and ultimately improve outcomes for glioblastoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0797dca5650557366d41b1e14b170fc6c641df8f" target='_blank'>
              The HMGB1-RAGE Axis Drives the Proneural-to-Mesenchymal Transition and Aggressiveness in Glioblastoma
              </a>
            </td>
          <td>
            , , Vijesh Kumar Yadav, Iat-Hang Fong, Heng-Wei Liu, Chien-Min Lin
          </td>
          <td>2025-09-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Macrophage polarization plays a pivotal role in shaping the tumor microenvironment and influencing rectal cancer progression. However, the metabolic and prognostic regulators governing this process remain largely undefined. Methods We constructed a macrophage polarization gene signature (MPGS) by integrating weighted gene co-expression network analysis (WGCNA) with multiple machine learning algorithms across two independent cohorts: 363 rectal cancer samples from GSE87211 and 177 samples from The Cancer Genome Atlas (TCGA). The prognostic performance of MPGS was evaluated across rectal and multiple other cancer types. Functional analyses, single-cell RNA sequencing, immunohistochemistry of clinical specimens, and in vitro cellular assays were employed to investigate the role of the MPGS hub gene, PYGM, in tumor biology and immune modulation. Results The MPGS exhibited robust prognostic capability and effectively predicted responses to immunotherapy and various chemotherapeutic agents. Both MPGS and its central metabolic component, PYGM, were closely linked to M2 macrophage infiltration, immunosuppressive tumor microenvironments, and poor clinical outcomes in rectal adenocarcinoma. Single-cell transcriptomic analysis revealed that malignant epithelial cells with elevated PYGM expression are metabolically active and closely interact with M2 macrophages. Clinical tissue analyses and functional assays confirmed that PYGM is upregulated in rectal cancer and promotes tumor cell proliferation, migration, and M2 macrophage polarization. Conclusions This study firstly highlights PYGM as a key metabolic and immunological regulator in rectal cancer, with significant prognostic and therapeutic implications. MPGS and PYGM may serve as novel biomarkers for risk stratification and guide personalized treatment strategies in patients with rectal adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8198c80f5a26115add8dc0453b94b75563056daf" target='_blank'>
              Machine learning identifies PYGM as a macrophage polarization–linked metabolic biomarker in rectal cancer prognosis
              </a>
            </td>
          <td>
            Chengyuan Xu, Siqi Zhang, Bin Sun, Zicheng Yu, Hailong Liu
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated fibroblasts (CAFs) regulate tumorigenesis, tumor cell proliferation, and metastasis via secreting related regulatory factors; however, the evidence for CAFs in regulating development of upper tract urothelial carcinoma (UTUC) remains unclear. Here, by utilizing single-cell RNA sequencing (scRNA-seq), single-nucleus RNA sequencing (snRNA-seq), SpaTial enhanced resolution omics-sequencing (Stereo-seq), and UTUC immunofluorescence chip cohort to construct the first comprehensive microenvironmental atlas of CAFs, we investigated the roles of CAFs in UTUC progression. Through hierarchical clustering and the copy number variation (CNV) scores of UTUC epithelial cells, we first classified tumor epithelial cells into high-malignant, medium-malignant, and low-malignant potential categories. We found that the myofibroblastic CAFs1 (myCAFs1) and myCAFs2 subclusters exhibited significant interaction signals with all three types of epithelial cells, among which high-malignant epithelial cells (HMECs) exhibited pronounced communication signals with CAFs, and FN1 and COL1A1 generated by CAFs played critical roles in this process, suggesting that the progression of UTUC may be attributed to the activation of tumor cells by CAFs. Both myCAFs1 and myCAFs2 simultaneously affect bladder urothelial carcinoma (BUC) prognosis, with the risk model showing good consistency across cohorts. The study constructs a multi-omics landscape of UTUC and identify common prognostic markers shared with BUC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/281dda7d54e63559872a9e12f7b32f74d4425ae4" target='_blank'>
              The tumor-associated fibroblasts regulate urothelial carcinoma progression.
              </a>
            </td>
          <td>
            Yu Xiao, Junfeng Yang, Mengjie Sun, Yongfu Li, Qinyin Liu, Jinjun Leng, Maolin Yang, Jinrui Wang, Hongju Li, Caifeng Yang, Changfen Luo, Jiahong Li, Longli Kang, Fen Huang, Yanhong Yu, Chuanmao Zhang
          </td>
          <td>2025-09-19</td>
          <td>Journal of molecular cell biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Osteosarcoma, a prevalent malignant bone tumor in pediatric and adolescent populations, is characterized by a markedly diverse tumor microenvironment (TME) that complicates therapeutic intervention. Among the key components of the TME, M2 macrophages contribute to immune suppression, tumor progression, and resistance to therapy. This study aims to explore the immune microenvironment of osteosarcoma and identify key immunosuppressive components using integrated transcriptomic analysis.



 We integrated scRNA-seq (GSE152048) and bulk RNA-seq (TARGET) data to characterize the osteosarcoma tumor microenvironment. Key analyses included cell clustering and annotation (Seurat), pseudotime trajectory (Monocle 2), cell-cell communication (CellChat), and M2 macrophage deconvolution (CIBERSORT, BayesPrism). Gene modules were identified via WGCNA, and drug sensitivity was predicted using oncoPredict. Key genes were validated by qRT-PCR and Western blot.



 Our integrated analysis revealed that M2 macrophage infiltration was significantly associated with an immunosuppressive tumor microenvironment in osteosarcoma. Weighted gene co-expression network analysis (WGCNA) identified gene modules highly correlated with M2 macrophage abundance and immune evasion signatures. Reduced M2 infiltration was linked to altered transcriptional patterns and increased predicted sensitivity to Etoposide and Doxorubicin. Two immune-related genes, PTK2B and PTPRC, were significantly upregulated in osteosarcoma samples and validated by RT-qPCR and Western blotting in U2OS cells.



 This study demonstrates that M2 macrophage infiltration is a defining feature of the osteosarcoma immune microenvironment and is associated with immune suppression. Through integrative transcriptomic analysis, PTK2B and PTPRC were identified as key immune-related genes linked to M2 macrophage abundance and patient prognosis. These results establish a transcriptional basis for the immunosuppressive phenotype in osteosarcoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c9aeb21260637fca67e1110200fbde1ea1d4d" target='_blank'>
              Integrated RNA sequencing reveals tumor microenvironment heterogeneity and immunosuppressive role of M2 macrophages in osteosarcoma
              </a>
            </td>
          <td>
            Zhijun Ding, Yi Li, Cheng Zhong
          </td>
          <td>2025-08-01</td>
          <td>Oncologie</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chimeric Antigen Receptor (CAR) T cell therapy has achieved high response rates in patients with relapsed or refractory hematologic malignancies. However, comparable efficacy in solid tumors remains limited, partly due to poor CAR T cell persistence and immune-mediated rejection. A major contributor, which has hampered the clinical efficacy of CAR T cells in clinical practice, is the immunogenicity of the murine-derived single-chain variable fragments (scFvs) commonly used in CAR constructs. Cell and humoral immune responses to the murine parts of CARs have been implicated in CAR T cell rejection. Here, we describe the generation and in vitro characterization of humanized CAR T cells targeting prostate-specific membrane antigen (PSMA) on prostate cancer cells, based on two distinct murine scFvs (A5 and D7). Humanization improved the germinality index and successfully preserved CAR surface expression. Functional assays demonstrated that humanized PSMA-CAR T cells retained antigen-specific binding, activation and cytotoxicity, differentiation, exhaustion and cytokine secretion profiles comparable to their murine counterparts. These results support the feasibility of humanization as a strategy to reduce immunogenicity without compromising CAR T cell capabilities, providing a foundation for further in vivo validation in solid tumor settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b7b7f0b2036a28a4056352443879cab64d41410" target='_blank'>
              Engineering of Humanized PSMA-Directed CAR T Cells for Potent and Specific Elimination of Prostate Cancer Cells
              </a>
            </td>
          <td>
            Tobias D. Deller, Jamal Alzubi, Laura Mosti, Marie Peschers, C. Gratzke, Phillipp Wolf, T. Cathomen
          </td>
          <td>2025-08-28</td>
          <td>Cells</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The process of wound healing is regulated by complex interactions between different cell types across space and time. Through the profiling of individual cells within their complex environment, single-cell transcriptomics methods enable the investigation of cellular heterogeneity, cell communication networks, and cell-cell interactions involved in the wound healing process. However, many single-cell analysis tools are run within a computer coding environment, and their more widespread use by wound healing scientists is thwarted by the apparent lack of bioinformatics expertise. Therefore, a step-by-step workflow is presented showing how to use a graphical coding environment called RStudio to perform a basic single-cell analysis of a temporal mouse excisional skin wound healing dataset. This visual and guided protocol will enable scientists with no bioinformatics background to download a previously published wound healing dataset, perform critical quality control steps, run a standard single-cell analysis workflow including dataset visualizations and cell type annotations using Seurat, run cell subtype analyses, run module scoring analyses, run cell-cell interaction analyses using CellChat, and perform integrative analyses of multiple datasets using Seurat. Narrative explanations are provided for each step in the protocol and graphical results from every line of code are presented to safely guide the user through the workflow. The goal of this visual introduction to a single-cell analysis pipeline is to enable more wound healing scientists to use bioinformatics tools directly in their own laboratories in order to facilitate deeper analyses of their own single-cell datasets as well as more widespread re-analyses of previously published single-cell datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01aaa9ce25d59a7c9eba1c49987e0404e67db6a5" target='_blank'>
              Using R, Seurat, and CellChat to Analyze a Single-Cell Transcriptomics Dataset of Mouse Skin Wound Healing.
              </a>
            </td>
          <td>
            Shalyn Keiser, Nichole Botello, Ethan Cruz, Mateusz S. Wietecha
          </td>
          <td>2025-08-01</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intratumoral microbiota, a critical component of the tumor microenvironment, notably impacts tumor progression through various complex mechanisms such as metabolic regulation, immune system remodeling and genotoxicity. The present review focuses on eight prevalent solid tumors (breast, colorectal, lung, pancreatic, gastric, ovarian, prostate cancers and melanoma), detailing the intratumoral microbial compositional heterogeneity within these malignancies. The present review analyzes the heterogeneous carcinogenic mechanisms mediated by these microorganisms, including gene instability, immune microenvironment remodeling and metabolic intervention. The diagnostic value of microbial markers in liquid biopsy and in situ tissue detection is explored, and the potential for synergistic strategies combining microbial-targeted therapy and immunotherapy is discussed. Finally, the review suggests future research directions, such as spatiotemporal dynamic analysis and organoid-microorganism co-culture, offering new insights for precise cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09bce21033da16d36e409d313fbe6c3d34814bec" target='_blank'>
              Tumor-associated microbiota: Multi-cancer landscape, mechanistic insights and clinical translation (Review)
              </a>
            </td>
          <td>
            Jindi Yang, Yige Dong, Yi Chen, Hong Liang, Shengyu Rong, Zhe Liu, Qiulei Lang
          </td>
          <td>2025-09-16</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glutamine metabolic reprogramming is a hallmark of tumor progression and is highly correlated with poor clinical outcomes. The excessive uptake of glutamine by tumor cells is a key factor contributing to widespread invasion, metastasis, and immune suppression. SLC38A2, an amino acid transporter widely expressed on the surface of tumor cells, has not been thoroughly studied regarding its function and prognostic significance in tumor progression. Our objective is to employ bioinformatics methods to conduct a comprehensive and in-depth analysis of SLC38A2 across various cancers, aiming to elucidate its role and prognostic value in tumor biology.By comprehensively incorporating gene expression and clinical data from the TCGA tumor database, GTEx database, Human Protein Atlas, and GEO database, we analyzed the expression profile, mutations, and established prognostic models for SLC38A2 across various cancers. Additionally, we investigated the enrichment of SLC38A2 at the single-cell level in 12 types of cancer and analyzed its temporal expression patterns in different cell subgroups in breast and pancreatic cancer. We also studied the correlation between SLC38A2 and glutathione metabolism.Compared to normal tissues, SLC38A2 exhibits significant differential expression in 15 types of cancer and serves as a prognostic risk factor in BRCA (HR = 1.597, p < 0.05), LUAD (HR = 1.650, p < 0.01), MESO (HR = 2.007, p < 0.05), and PAAD (HR = 1.761, p < 0.05), while acting as a protective factor in KIRC (HR = 0.625, p < 0.05). Furthermore, SLC38A2 is positively correlated with tumor and stromal cells, negatively correlated with immune cell infiltration, and associated with immune exhaustion. In BRCA, SLC38A2 is highly expressed during early differentiation of malignant and stromal cells, and enriched in late differentiation of immune cells. Moreover, the expression of SLC38A2 shows a general positive correlation with glutathione metabolism genes in BRCA, LUAD, MESO, and PAAD, demonstrating diagnostic value.SLC38A2 shows widespread changes in expression patterns within tumor tissues, making it an effective diagnostic and prognostic biomarker. It is enriched in malignant cells and tumor-infiltrating stromal cells, while negatively correlated with the infiltration of many cells involved in anti-tumor immunity. Targeting SLC38A2 presents a viable therapeutic strategy by inhibiting glutamine competition and relieving immune suppression in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8cdb69c6df340eda3f029c78020f687d6995d39" target='_blank'>
              Panoramic analysis of the biological function and clinical value of SLC38A2 in human cancers: a study based on pan-cancer and single-cell analysis
              </a>
            </td>
          <td>
            Mao Liao, Yuqing Rao, Molan Li, Jiayang Guo, Kun Guo, Kaiyue Li, Rui Zheng, Yifan Liu, Qianyi Wang, Manni Wang, Duo Chen, Meng Zhang, Yongfeng Wang, Yanzong Zhao, Sheng Li
          </td>
          <td>2025-09-17</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive solid malignancy with poor overall prognosis and limited response to standard treatments. Growing interest in the modulation of DNA repair mechanisms, including the homologous recombination (HR) repair pathway, has opened new avenues for therapeutic development. BARD1 (BRCA1-Associated RING Domain 1) plays a complex role in tumor biology, functioning either as a tumor suppressor or as an oncogenic driver, depending on isoform expression, cellular context, and regulatory environment. In this review, we examine the dual roles of BARD1, focusing on its regulation and paradoxical activities in PDAC. We summarize evidence that BARD1 and BARD1 isoforms differentially affect DNA repair, apoptosis, and drug resistance and evaluate the therapeutic potential of targeting BARD1 and other DNA damage response (DDR) proteins. We also review ongoing clinical trials and investigational agents designed to exploit DDR vulnerabilities in PDAC. Together, these insights highlight BARD1’s context-dependent roles in PDAC and support continued efforts to profile BARD1 isoforms, clarify their functions, and leverage DDR pathways through precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c070f9d1ec5a160a1dcacf90475e9772e358bc" target='_blank'>
              BARD1: A Friend or Foe in Pancreatic Ductal Adenocarcinoma?
              </a>
            </td>
          <td>
            Lily Zekavat, Aditi Jain
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ce04753f787ce77194145424676b248b0e063cc" target='_blank'>
              Advancements in the Metabolic Profiling of Three-Dimensional Brain Tumor Spheroids for Drug Screening.
              </a>
            </td>
          <td>
            Lijing Yang, Xiaojuan Ma, Decao Yang, Jiagui Song, Jianling Yang, Yan Sun, Yan Wang, Lixiang Xue
          </td>
          <td>2025-09-05</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Uveal melanoma, the most common eye cancer in adults, remains limited to surgical intervention and chemotherapy, with a dismal survival rate that has not improved in over 50 years. To address this therapeutic impasse, we systematically analyzed public gene expression, RNAi, and CRISPR knockout datasets and identified RASGRP3 as an essential gene specifically for uveal melanoma. RasGRP3 is uniquely overexpressed and essential for survival in uveal melanoma cells, but dispensable in healthy cells. RasGRP3 remains “undruggable” due to its intracellular localization and lack of targetable binding pockets. To overcome this, we developed a CRISPR-Cas13d RNA-targeting therapeutic that specifically knocks down RasGRP3 mRNA. This Cas13d-based therapeutic mediates selective uveal melanoma killing through two synergistic mechanisms: (i) direct silencing of the essential RasGRP3 transcript, and (ii) collateral RNA degradation triggered by the cleavage of overexpressed RasGRP3. When delivered via optimized lipid nanoparticles encoding Cas13d mRNA and guide RNA, this strategy eliminated >97% of uveal melanoma cells while sparing healthy cells, including retinal pigment epithelial cells. This approach outperformed conventional Cas9 and siRNA methods in potency without inducing permanent genomic alterations. Our findings establish a RNA-targeting therapeutic for uveal melanoma and a framework for Cas13d-based interventions against broad “undruggable” cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bb413d9ad08e88d74f39a8ba7b106b39795e58" target='_blank'>
              CRISPR-Cas13d-Mediated Targeting of a Context-Specific Essential Gene Enables Selective Elimination of Uveal Melanoma
              </a>
            </td>
          <td>
            Daniel Stauber, Lucas Sosnick, Yitong Ma, Sopida Pimcharoen, Atip Lawanprasert, Niren Murthy, David Myung, L. S. Qi
          </td>
          <td>2025-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Somatic mutations are desirable targets for selective cancer elimination, yet most are found within noncoding regions. We have adapted CRISPR-Cas9 as a novel, cancer-specific killing strategy by targeting the subset of somatic mutations that create protospacer adjacent motifs (PAMs), which have evolutionally allowed bacterial cells to distinguish between self and non-self DNA for Cas9-induced double strand breaks (DSBs). We hypothesized that somatic 5’-NGG-3’ PAMs arising from base substitutions (e.g. NCG→NGG) could serve as CRISPR-Cas9 targets for selective cancer elimination through the induction of multiple DSBs. Whole genome sequencing (WGS) of paired tumor-normal (T-N) samples from three PC patients were put through PAMfinder (software for somatic PAM discovery). On average, 417 somatic PAMs per PC case were found (>99% of them in non-exons) and 97% of the selected subset were Sanger confirmed. We then analysed T-N samples from ICGC which revealed medians of 455 somatic PAMs per tumor in two PC cohorts (N=174 combined), 2790 in lung (N=29), and 3236 in esophageal (N=388) cancer cohorts. We performed γH2AX staining on PC cell lines transduced with 4-9 sgRNAs designed from our PAM discovery approach, and observed more foci with increased number of sgRNAs in the target cell line but not in the non-target cell line. Deep sequencing and WGS on clones transduced with cancer-specific sgRNAs also demonstrated on-target activity in the target cell line but not in either the patient’s normal cells or an irrelevant cancer cell line. No off-target events were detected. To test selective toxicity in vitro and in vivo, we transduced 4-9 sgRNAs into PC co-cultures and found 69-99% selective cell death of three target cell lines through flow cytometry. Finally, using mouse xenograft models, 92% weight reduction (N=10, P=0.0004) of tumors transduced with a pool of 9 cancer-specific sgRNAs was achieved compared to tumors transduced with non-targeting sgRNAs (N=8).



 Selina Shiqing K. Teh, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Akhil Kotwal, Hong Liang, Sarah Wheelan, Chien-Fu Hung, Michael Goldstein, Nicholas J. Roberts, and James R. Eshleman. CRISPR-Cas9 as a Novel Cancer Gene Therapy Approach against Pancreatic Cancers [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr LT02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b50e15f1763d831356c9974d95d4708f8806a3e" target='_blank'>
              Abstract LT02: CRISPR-Cas9 as a Novel Cancer Gene Therapy Approach against Pancreatic Cancers
              </a>
            </td>
          <td>
            S. Teh, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Akhil Kotwal, Hong Liang, Sarah Wheelan, Chien-Fu Hung, M. Goldstein, Nicholas J. Roberts, James R. Eshleman
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Data from cell viability assays, which measure cumulative division and death events in a population and reflect substantial cellular heterogeneity, are widely available. However, interpreting such data with mechanistic computational models is hindered because direct model/data comparison is often muddled. We developed an algorithm that tracks simulated division and death events in mechanistically-detailed single-cell lineages to enable such a model/data comparison and suggest causes of cell-cell drug response variability. Using our previously developed model of mammalian single-cell proliferation and death signaling, we simulated drug dose response experiments for four targeted anti-cancer drugs (alpelisib, neratinib, trametinib and palbociclib) and compared them to experimental data. Simulations are consistent with data for strong growth inhibition by trametinib (MEK inhibitor) and overall lack of efficacy for alpelisib (PI-3K inhibitor), but are inconsistent with data for palbociclib (CDK4/6 inhibitor) and neratinib (EGFR inhibitor). Model/data inconsistencies suggest that (i) the importance of CDK4/6 for driving the cell cycle may be overestimated, and (ii) the cellular balance between basal (tonic) and ligand-induced signaling is a critical determinant of receptor inhibitor response. Simulations show subpopulations of rapidly and slowly dividing cells in both control and drug-treated conditions. Variations in mother cells prior to drug treatment impinging on ERK pathway activity are associated with the rapidly dividing phenotype and trametinib resistance. This work lays a foundation for the application of mechanistic modeling to large-scale cell viability datasets and better understanding determinants of cellular heterogeneity in drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91b34cacec4c400be097c53ff9cbf10582c4cad0" target='_blank'>
              Mechanistic modeling of cell viability assays with in silico lineage tracing
              </a>
            </td>
          <td>
            Arnab Mutsuddy, Jonah R. Huggins, Aurore K. Amrit, Atalanta Manuela Harley-Gasaway, Cemal Erdem, Evan T Jones, Olivia G Laurine, Jon Calhoun, Marc R. Birtwistle
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="This review article explores the role of immuno-radiotherapy in the context of genome instability and tumour evolution. Genomic changes in tumours exist in a delicate balance with the immune system, offering evolutionary pathways to adapt and grow but risking provoking an immune response. Rapid developments across both immunotherapy and radiotherapy have raised questions about the potential benefits combination therapy, and how best to identify ideal treatment populations. Here we discuss foundational studies of genomic instability and tumour evolution, how these paradigms translate into immune surveillance and evasion, and subsequently go on to explore recent preclinical and clinical studies of both treatment modalities. Understanding how cancers evolve in the context of the immune system could provide a key insight in delivering better therapies that could overcome treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4d64cec361f889e2be65100e62ef8d285da2f26" target='_blank'>
              Tumour Evolution Driving Genome Instability, Immune Interactions, and Response to Radiotherapy
              </a>
            </td>
          <td>
            G. Seed, France Truong, Rahma Riahi, Ben O’Leary
          </td>
          <td>2025-06-09</td>
          <td>Cancer journal (Sudbury, Mass.)</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background Cellular senescence plays a significant role in tumorigenesis and tumor progression. Substantial evidence indicates that senescence occurs in cancer-associated fibroblasts (CAFs), the predominant stromal component within the tumor microenvironment (TME), which profoundly impacts tumor biology. However, despite growing evidence of stromal cell involvement in cancer progression, the specific mechanisms and clinical implications of senescent CAFs (SCAFs) in hepatocellular carcinoma (HCC) have not been fully elucidated. Methods The senescence signature was utilized to evaluate the senescence status of cell types within the TME of HCC using the GSE149614 dataset. The CytoTRACE and cell-cell communication analysis were used to find the correlation between cancer stemness and SCAFs. A risk prediction model associated with SCAFs was constructed to investigate potential mechanisms by which SCAFs promote tumor progression. Single-cell RNA sequencing data was used to identify senescent CAF-related genes. Gene expression and clinical data for HCC were obtained from the Cancer Genome Atlas (TCGA), International Cancer Genome Consortium (ICGC), and National Omics Data Encyclopedia (NODE) databases. Using four machine-learning algorithms, crucial genes were identified to develop a CAF-senescence-related risk model, predicting prognosis, cancer stemness, immune infiltration, tumor mutation burden, and therapeutic responses in HCC patients. Next, we explored the role of Collagen Triple Helix Repeat Containing-1 (CTHRC1) in cancer stemness using both in vitro and in vivo experiments. Through various functional experiments, we elucidated the downstream signaling pathways of CTHRC1. Additionally, chromatin immunoprecipitation experiments were used to verify that key transcription factors bind to the CTHRC1 promoter region. Results CAFs exhibited high senescence status and a strong correlation with cancer stemness in HCC. A novel CAF-senescence-score (CSscore) prognostic model was established for HCC based on 10 genes: CTHRC1, SERPINE1, RNF11, ENG, MARCKSL1, ASAP1, FHL3, LAMB1, CD151, and OLFML2B. The survival prediction performance was validated on TCGA, ICGC, and NODE cohorts. Immune analysis revealed that the CSscore was positively correlated with immunosuppressive immune cell populations, including M2 macrophages and regulatory T cells. Conversely, a negative correlation was observed between the CSscore and anti-tumor immune cells such as CD8 + T cells, dendritic cells, and B cells HCC patients with a low CSscore had a lower tumor mutation burden and showed improved responsiveness to immunotherapy and transarterial chemoembolization. In vitro experiments and bioinformatics analysis further revealed that CTHRC1 was significantly elevated in SCAFs promoted cancer stemness and metastasis via the SRY-box transcription factor 4 (SOX4)-CTHRC1-Notch1 axis in HCC. Conclusion Our study revealed that SCAFs were strongly correlated with cancer stemness in HCC. A novel machine learning model based on senescent CAF-related genes was constructed to accurately predict prognosis in HCC patients. Furthermore, CTHRC1 was identified as a novel prognostic and therapeutic biomarker to predict poor prognosis in HCC and promote cancer stemness and metastasis through the Notch signaling pathway, with its expression being transcriptionally regulated by SOX4. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02369-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff7c6095fcfbd5cb01c70f78e80c25fa1a454693" target='_blank'>
              Senescent fibroblasts secrete CTHRC1 to promote cancer stemness in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hai Huang, Wang Peng, Qiaodan Zhou, Yuchong Zhao, Luyao Liu, H. Cui, Jingwen Liang, Mengdie Cao, Wei Chen, Ronghua Wang, Shiru Chen, S. Xiong, Bin Cheng, Shuya Bai
          </td>
          <td>2025-08-25</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Brain metastases (BM) represent a signiﬁcant unmet clinical need with poor survival and limited drug thera- pies. Recent evidence suggests a subset of BM have a more locally invasive phenotype. We sought to determine whether the tumour leading edge and tumour microenvironment (TME) differed between BM of different phe- notypes using spatial transcriptomics.



 Image guided surgical sampling of the tumour margin in operated BM cases using a previously described method (“Geo-Tagging”). Tumour microarray from FFPE blocks assembled using 2mm dots in triplicate. Spatial transcriptomics analysis using Nanostring GeoMX platform and immune-oncology panel (86 markers). Mart1 (melanoma BM) or CK7 (breast and lung adenocarcinoma BM) was used to delineate the tumour, GFAP for per- itumoral brain (TME) and CD45 for immune compartment. Annotation of tissue with clinical data for overall survival, local recurrence, primary cancer and assessment of BM leading edge using established criteria – dif- fuse or border.



 22 cases were analyzed in triplicate, 5 breast cancer, 8 metastatic melanoma and 9 lung adenocarcinoma. Data was analysed using established bioinformatics pipelines in R. As expected there were signiﬁcant differences in gene expression between all BM of different primary cancers but not in the portion of T cells, macrophages and NK cells. PDL1 was upregulated in the tumour compartment and HLA-E downregulated in the TME com- partment of BM without a clear border. Locally recurrent BM signiﬁcantly overexpressed a number of genes previously established as important in BM, including for adhesion (ICAM1, EPCAM), microglia-tumour interac- tion (STAT3) and local invasion (CCND1, VEGFA).



 BM have different phenotypes that are dependent on both tumour and local microenvironmental factors with the future potential for therapeutic targeting to modify tumour behaviour and reduce local invasion and recur- rence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef6992d8a9fcf5e2f41d2333a8c8118644a30147" target='_blank'>
              INVESTIGATING THE TUMOUR MICROENVIRONMENT FOR BRAIN METASTASES USING SPATIAL TRANSCRIPTOMICS AND SURGICALLY GUIDED SAMPLING
              </a>
            </td>
          <td>
            R. Zakaria, Michael Jenkinson, Ian Scott, Frances Greaney Davies, Michael Chvanov, Jamie Soul
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4006665d3be664d3d5517dbddb8bef8da7462da" target='_blank'>
              High-Resolution Spatial Transcriptomics Reveals Fibroblast and Neuroimmune Microenvironments in Endometriosis Lesions.
              </a>
            </td>
          <td>
            Caroline M Haney, Elaheh Alizadeh, Meryl Sullivan, Joshua Lee, Jasmina Kuljancic, W. Flynn, Paul Robson, Brian S. White, Danielle E Luciano, Elise T. Courtois
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Polymerase epsilon mutant (POLE-mut) endometrial cancers (EC) are characterized by a near 100% disease-specific survival rate, even when treated by surgery alone. This spectacular survival, combined with the ultramutated genome and high level of neoantigens in these tumors, indicates a substantial degree of immune control in preventing disease spread and recurrence. Although these features are intriguing, the immune infiltration of POLE-mut EC has predominantly been confined to immunohistochemistry studies. Here, we used state of the art single-cell RNA and TCR sequencing to characterize the immune landscape of POLE-mutant ECs. Moreover, we uniquely analyzed patient blood samples taken two to eight years after curative treatment to assess formation of long-term immune memory in circulation. We identified specialized tumor-infiltrating myeloid subsets at different stages of maturation, an array of lymphocytes ranging from immature to cytotoxic and adaptive natural killer (NK) as well as tumor-reactive exhausted and effector T cells, contributing to a highly inflammatory anti-tumor response. Remarkably, our analysis of blood samples taken years after curative treatment uncovered the presence of tumor-reactive T cell clones that matched the primary tumor. This indicates the formation of systemic long-term memory immune responses in POLE-mut EC survivors. Our study highlights the distinctive immunogenicity of POLE-mut EC and identifies key features associated with persistent anti-tumor immunity that may contribute to prolonged, relapse-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0198b259b8d5d685a1ba64c108955b1306a25438" target='_blank'>
              Spatiotemporal immune landscape and long-term immune memory in POLE-mutant endometrial cancer at the single-cell level.
              </a>
            </td>
          <td>
            K. Brummel, M. Requesens, N. van Rooij, H. Workel, F. Eggink, A. Plat, R. Wardenaar, D. Spierings, F. Foijer, David N. Church, Joost Bart, H. Nijman, M. de Bruyn
          </td>
          <td>2025-08-29</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c8f319f11687277fd179aa2df80ba7f003d32d" target='_blank'>
              Loss of the Y chromosome drives cancer metabolic reprogramming.
              </a>
            </td>
          <td>
            Boopathi Subramaniyan, Bipul R. Acharya, Prince Anand, Saravana Kumar Kailasam Mani, Saswat Mohapatra, Xingyu Chen, Fangyuan Qu, Yesai S Fstkchyan, Corazon Gutierrez, Hany A Abdel-Hafiz, James C. Costello, Dan Theodorescu
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: The classical ''seed and soil'' hypothesis suggests that metastatic spread is shaped by tumor-intrinsic traits (''seeds'') and the organ-specific microenvironment (''soil''). We expand this concept to explain lesion-level therapeutic responses and phenotypic variability across metastatic cancers. Methods: We analyzed 55,220 lesions from 6,087 patients enrolled in 20 clinical trials across six cancer types. Using nonlinear mixed-effects modeling, we estimated lesion-specific parameters: regression rate (kkill), progression rate (kge), and resistant fraction (Fx). Multivariable Cox models, adjusted for cancer type, treatment modality, and clinical covariates, were used to assess organ-specific response patterns. Two key microenvironmental features--Vascular Perfusion and Leakiness Index (VaPLI) and tissue immune tolerance--were evaluated as predictors of lesion-level phenotypes. Results: Treatment response dynamics varied significantly across metastatic sites, even within the same cancer type and therapy. Lesion-level responses reflected a strong seed-soil interaction, influenced by treatment modality. Liver metastases showed high initial regression but rapid progression, while bone lesions, especially in prostate cancer, exhibited more stable responses. VaPLI and immune tolerance status were significant predictors of lesion behavior. Conclusions: Our findings support an expanded seed-soil framework that incorporates physiological and therapeutic context, enabling site-aware treatment strategies and refined patient selection for metastatic cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f8120d9184a5579062858ffd9ded5984b24c995" target='_blank'>
              A Pan-Cancer Analysis of Lesion-Level Treatment Response to Extend the ‘Seed and Soil’ Paradigm
              </a>
            </td>
          <td>
            Dilley Ian, Jiawei Zhou, Quefeng Li, Yanguang Cao
          </td>
          <td>2025-09-18</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9220ff17576dcea92752853f5d558e4780ff95e5" target='_blank'>
              CRISPR screening by AAV episome-sequencing (CrAAVe-seq): a scalable cell-type-specific in vivo platform uncovers neuronal essential genes.
              </a>
            </td>
          <td>
            Biswarathan Ramani, Indigo V. L. Rose, N. Teyssier, Andrew Pan, Spencer Danner-Bocks, Tanya Sanghal, Lin Yadanar, Ruilin Tian, Keran Ma, J. Palop, Martin Kampmann
          </td>
          <td>2025-08-22</td>
          <td>Nature neuroscience</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Simple Summary Breast cancer is a complex disease influenced not only by genes but also by the way cancer cells generate and use energy. One rare change is the loss of a gene called fumarate hydratase (FH), which normally helps cells produce energy efficiently. We examine how this change influences how tumors grow and respond to treatment. By studying thousands of breast cancer samples, we found that tumors missing FH tend to create a special environment that helps them grow and survive, especially by encouraging new blood vessel formation. We also describe a patient with this FH change who had a remarkable and long-lasting response to a therapy that blocks blood vessel growth (known as anti-VEGF treatment). These findings suggest that identifying this rare energy-related alteration may help clinicians determine which patients are most likely to benefit from this therapy. Such insights could contribute to advancing more personalized approaches to breast cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c214b499120e8d0fc3e3396e45d4d3eb6e6c018" target='_blank'>
              Revealing the Angiogenic Signature of FH-Deficient Breast Cancer: Genomic Profiling and Clinical Implications
              </a>
            </td>
          <td>
            L. A. Sinberger, Noa Keren-Khadmy, Assaf Goldberg, T. Peretz-Yablonski, A. Sonnenblick, Mali Salmon-Divon
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Significant group-based variations in prostate cancer (PCa) outcomes represents a critical healthcare challenge in the USA. African American (AA) men face the highest incidence of PCa and are twice as likely to succumb to their disease compared to their Caucasian American (CA) counterparts. This disparity stems from a complex interplay involving social, environmental, and biological factors. While inequitable access to quality healthcare is a key contributor, even in settings with equal care, AA patients more often present with advanced disease at diagnosis, suggesting unique tumor biology. While some understanding of the epigenomic landscape has been recently identified, the full extent of global chromatin remodeling contribution towards differences in health outcomes across population groups remains poorly understood. To better elucidate how chromatin remodeling and gene regulation influences PCa development across CA and AA population groups, we employed single-cell spatial transposase-accessible chromatin technology followed by sequencing (scATAC-seq) and H3K27ac spatial-CUT&Tag using samples from men who underwent prostatectomy at the Center for Prostate Disease Research/Walter-Reed National Military Medical Center. Our current analysis revealed significantly greater chromatin accessibility in AA men compared to CA men. Leveraging a machine learning-based computational cell typing method called Cellcano to annotate scATACseq data, enabled the identification of unique cell types and distinct gene score signatures enriched in AA samples compared to CA counterparts. Furthermore, AA demonstrated significant lower accessible chromatin associated with CCCTC-binding factor (CTCF) which is a critical zinc finger protein crucial for regulating gene expression, epigenetic marks, and maintaining 3-dimensional genome structure. We have validated this finding by CTCF immunohistochemistry in an independent TMA that includes 99 patients with distribution of CA and AA prostatectomy samples. Moreover, integration of H3K27ac profiling revealed several differentially regulated enhancer regions near specific genes across samples. Collectively, our current findings provide critical insights into the divergent chromatin accessibility profiles between AA and CA men with localized PCa, which likely underlie distinct transcriptional responses that can determine therapeutic resistance and tumor progression. These data shed light on the complex epigenomic mechanisms driving the group-based variations in PCa outcomes and may inform the development of personalized treatment strategies to address this critical healthcare challenge.



 Beatriz German Falcon, Kun-Lin Ho, Tri Truong, Jennifer Garbarino, Jiaying Chen, Amina Ali, William Azadze, Sally Elsamanoudi, Colin Ng, Gregory Chesnut, Leigh Ellis. Analysis of the chromatin accessibility landscape in patients with localized prostate cancer [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C087.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f758d1a482aa1ee0457d64a79f2bbfa79c84e92e" target='_blank'>
              Abstract C087: Analysis of the chromatin accessibility landscape in patients with localized prostate cancer
              </a>
            </td>
          <td>
            Beatriz German Falcon, Kun-Lin Ho, Tri Truong, Jennifer Garbarino, Jiaying Chen, Amina Ali, William Azadze, S. Elsamanoudi, Colin Ng, Gregory T. Chesnut, Leigh Ellis
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background A key to understanding cancer is to determine the impact on the cellular pathways caused by the repertoire of DNA changes accrued in a cancer cell. Exploring the interactions between genomic aberrations and the expressed transcriptome can not only improve our understanding of the disease but also identify potential therapeutic approaches. Results Using random forest models, we successfully identified transcriptional patterns associated with the loss of wild-type activity in cancer-related genes across various tumour types. While genes like TP53 and CDKN2A exhibited unique pan-cancer transcriptional patterns, others like ATRX, BRAF, and NRAS showed tumour-type-specific expression patterns. We also observed that genes like AR and ERBB4 did not lead to strong detectable patterns in the transcriptome when disrupted. Our investigation has also led to the identification of genes highly associated with transcriptional patterns. For instance, DRG2 emerged as the top contributor in classification of ATRX alterations in lower-grade gliomas and was significantly downregulated in ATRX mutant tumours. Additionally, transcriptional features important in classification of PTEN aberrations, such as CDCA8, AURKA, and CDC20, were found to be closely related to PTEN function. Conclusions Our findings demonstrate the utility of machine learning in interpretation of cancer genomic data and provide new avenues for development of targeted therapies tailored to individual patients with cancer. Our analysis on the transcriptome revealed genes with expression levels strongly correlated with alterations in cancer-related genes. Additionally, we identified AURKA inhibitors as potential therapeutic option for tumours with alterations in tumour suppressors like FBXW7 or NSD1. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02339-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e1f512af72f3e98e7eac17d6531fcc9f244b2e0" target='_blank'>
              Transcriptional patterns of cancer-related genes in primary and metastatic tumours revealed by machine learning
              </a>
            </td>
          <td>
            Faeze Keshavarz-Rahaghi, E. Pleasance, Steven J M Jones
          </td>
          <td>2025-08-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Lysosome-dependent cell death (LDCD) is an emerging form of regulated cell death with critical implications in tumor development, immune modulation, and therapy responsiveness. However, the role of LDCD-related genes in colon adenocarcinoma (COAD) remains poorly understood. We comprehensively analyzed LDCD-related gene expression profiles using transcriptomic data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus databases (GEO). Unsupervised clustering was performed to identify molecular subtypes. A prognostic signature was developed using LASSO and Cox regression analyses. Immune infiltration characteristics and immunotherapy responses were assessed via multiple algorithms. Single-cell RNA sequencing (scRNA-seq) analysis was conducted to explore the cellular distribution of LDCD genes. Functional assays, including colony formation, Transwell migration, and western blotting, were performed to validate the role of key LDCD regulators in COAD cell line. LDCD-related genes were differentially expressed in COAD tissues and stratified patients into two distinct subtypes with significant differences in survival, immune infiltration, and biological pathways. A five-gene prognostic signature demonstrated robust predictive ability for overall survival and was associated with tumor immune microenvironment characteristics. High-risk patients showed distinct immune checkpoint profiles and predicted sensitivity to immune checkpoint blockade therapy. scRNA-seq analysis revealed cell type-specific expression patterns of LDCD genes. In vitro experiments confirmed that modulation of a key LDCD regulator, SLC11A1, affected the proliferation and invasiveness of COAD cells. Moreover, the impact of SLC11A1 on apoptosis and intracellular ROS level confirmed the potential association between SLC11A1 and COAD cell apoptosis. Our study reveals the prognostic value and immunological relevance of LDCD-related genes in COAD. These findings provide new insights into molecular classification and may aid in the development of personalized therapeutic strategies targeting LDCD pathways in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18267-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9901f6ba7f419afbbf7d8a4c7ff078253e88b294" target='_blank'>
              Comprehensive characterization of lysosome-dependent cell death reveals prognostic significance and immune landscape in colon adenocarcinoma
              </a>
            </td>
          <td>
            Zhenhua Liu, Lianming Cai, B. Liao, Baowen Wang, T. Xia, Xiangxin Wu
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2174bd5047fff23ba3520fb452837eee05b0da" target='_blank'>
              Systematic discovery of CRISPR-boosted CAR T cell immunotherapies.
              </a>
            </td>
          <td>
            Paul Datlinger, Eugenia V Pankevich, Cosmas D Arnold, Nicole Pranckevicius, Jenny Lin, Daria Romanovskaia, Moritz Schaefer, Francesco Piras, Anne-Christine Orts, A. Nemc, Paulina N Biesaga, Michelle Chan, Theresa Neuwirth, Artem V Artemov, Wentao Li, Sabrina Ladstätter, Thomas Krausgruber, Christoph Bock
          </td>
          <td>2025-09-24</td>
          <td>Nature</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Current breast organoid models, primarily based on self-assembly, face limitations in accurately mimicking the complex stages of tissue development, notably in replicating detailed tissue architectures and cellular diversity. Here, we demonstrate that a solitary human breast stem cell can, within a controlled hydrogel extracellular matrix environment, autonomously generate niche signals that drive directed organogenesis, producing organoids with mesenchymal and parenchymal components. This system surpasses traditional models by forming complex, heterogeneous ductal-lobular structures akin to those in native human breast tissue with the emergence of mesenchyme-like stroma. Utilizing long-term, high-resolution live imaging, we quantitatively analyze the dynamic tissue development and morphogenesis, capturing cellular behaviors crucial for understanding both development and disease. The inclusion of patient-derived cells from diverse biological backgrounds, including pre- and post-menopausal donors, significantly enhances the model's applicability. Our findings underscore that this controlled organogenesis approach represents a major leap over existing methods, offering a robust platform for probing the intricacies of human breast tissue development and its responses to environmental cues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b9ecb36488fcd624dc04a9daeb222c114b25fd3" target='_blank'>
              Single-cell organogenesis captures complex breast tissue formation in 3D.
              </a>
            </td>
          <td>
            G. Rauner, Nicole Traugh, Colin Trepicchio, Meadow E. Parrish, Kenan Mushayandebvu, Charlotte Kuperwasser
          </td>
          <td>2025-08-06</td>
          <td>Development</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H.A. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ObjectiveTo investigate the mechanism of tumor-associated macrophages (TAM) in the invasive migration of lung adenocarcinoma (LUAD) cells.MethodsThe single-cell sequencing data of lung adenocarcinoma (GSE131907) were initially analyzed. Kaplan-Meier curve analysis, univariate as well as multivariate Cox analysis, and immunofluorescence staining were performed. The analysis of fibroblast-macrophage interactions using Single-cell CellChat revealed their relationship. Subsequently, we screened and validated the target proteins in macrophages that interact with SEMA3C. The effects of these interactions on lung cancer cell migration and invasion were evaluated in vitro through Western blot analysis to assess phenotypic changes in macrophages, as well as through Transwell migration and invasion assays.ResultsSEMA3C was predominantly expressed in fibroblasts of patients with high-grade lung adenocarcinoma at high levels. SEMA3C exhibited independent prognostic significance in determining the overall survival outcome among individuals diagnosed with lung adenocarcinoma. Lung adenocarcinoma fibroblasts had elevated SEMA3C. CellChat demonstrated enhanced interactions between TAM as well as T cells. A high expression of vascular non-inflammatory molecule 1 (VNN1) in fibroblast macrophages during Stage II-III, and this elevated VNN1 was also an independent prognostic factor. The interaction between cancer-associated fibroblasts (CAFs) and VNN1 on macrophage membranes mediated by SEMA3C. Furthermore, these experiments demonstrated that SEMA3C regulates the polarization of TAM through VNN1, thereby influencing lung cancer.ConclusionThe phenotype of TAM is regulated by SEMA3C, which in turn influences the migration as well as invasion of lung cancer cells through VNN1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca3b18f77cbea75c9377998f4675506eee37ae90" target='_blank'>
              SEMA3C regulates tumor-associated macrophage phenotype and influences lung cancer cell migration and invasion through VNN1.
              </a>
            </td>
          <td>
            Renlong Liu, Jie Lian, Tiebo Yang, Tao Sun, Chunyang Wang, Yan Yan
          </td>
          <td>2025-08-01</td>
          <td>Human & experimental toxicology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations in numerous genes contribute to human cancer, with different oncogenic lesions prevalent in different cancer types. However, the malignant phenotype is simple, characterized by unrestricted cell growth, invasion, and often metastasis. One possible hypothesis explaining this dichotomy is that cancer genes regulate common targets, which then function as master regulators of essential cancer phenotypes. To identify mechanisms that drive the most fundamental feature shared by all tumors-unrestricted cell proliferation-we used a multiomic approach, which identified translation and ribosome biogenesis as common targets of major oncogenic pathways across cancer types. Proteomic analysis of tumors and functional studies of cell cultures established nucleolar and coiled-body phosphoprotein 1 as a key node, whose convergent regulation, both transcriptionally and posttranslationally, is critical for tumor cell proliferation. Our results indicate that lineage-specific oncogenic pathways regulate the same set of targets for growth control, revealing key downstream nodes that could be targeted for therapy or chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e15820af1f4a4174736d844aaa4b18b01cd6fd" target='_blank'>
              Diverse oncogenes use common mechanisms to drive growth of major forms of human cancer.
              </a>
            </td>
          <td>
            O. Kauko, Mikko Turunen, Päivi Pihlajamaa, Antti Häkkinen, R. Queiroz, M. Pääkkönen, S. Ventelä, Massimilliano Gaetani, Susanna L. Lundström, Antonio Murgia, Biswajyoti Sahu, J. Routila, Gong-Hong Wei, H. Irjala, J.L. Griffin, Kathryn S. Lilley, Teemu Kivioja, S. Hautaniemi, J. Taipale
          </td>
          <td>2025-08-22</td>
          <td>Science advances</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Abstract Brain metastasis occurs in up to 40% of patients with non-small cell lung cancer (NSCLC). Considerable genomic heterogeneity exists between the primary lung tumor and respective brain metastasis; however, the identity of the genes capable of driving brain metastasis is incompletely understood. Here, we carried out an in vivo genome-wide CRISPR activation (CRISPRa) screen to identify molecular drivers of brain metastasis from an orthotopic NSCLC patient-derived xenograft model. We discovered activating expression of the Alzheimer’s disease associated β-site amyloid precursor protein cleaving enzyme 1 (BACE1) led to a significant increase in brain metastasis. Furthermore, genetic and pharmacological inhibition of BACE1 blocked NSCLC brain metastasis. Mechanistically, we identified BACE1 acts through its novel substrate EGFR to drive this metastatic phenotype. Together, our data highlights the power of in vivo CRISPR screening to unveil novel molecular drivers and potential therapeutic targets of NSCLC brain metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f345106700d366f8dea4f4646de89615abfeab1" target='_blank'>
              PRCA-06 GENOME-WIDE IN VIVO CRISPR ACTIVATION SCREEN IDENTIFIES BACE1 AS A THERAPEUTIC VULNERABILITY OF LUNG CANCER BRAIN METASTASIS
              </a>
            </td>
          <td>
            Shawn C Chafe, Kui Zhai, Nikoo Aghaei, P. Miletic, Zhi Huang, Kevin R. Brown, Daniel Mobilio, Daniel Young, Yujin Suk, Shan Grewal, D. Mckenna, Zahra Alizada, Agata M Kieliszek, Fred Lam, Laura Escudero, Qian Huang, A. Huebner, Jack Lu, Patrick Ang, Alisha Anand, Stefan Custers, Erika Apel, Sarah Slassi, Benjamin A Brakel, Jongmyung Kim, James K C Liu, B. Bassey-Archibong, Rober Abdo, Y. Shargall, Jianwei Lu, J. Cutz, Qi Zhang, Shawn Li, C. Venugopal, Rob Hynds, A. Dufour, Jason Moffat, Charles Swanton, Shideng Bao, Sheila K Singh
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Spatial omics provides three types of information: location, gene expression, and morphology. While current algorithms typically integrate gene expression with location, they often overlook morphology. Chronic Myelomonocytic Leukemia (CMML) is a rare blood cancer characterized by sustained peripheral blood monocytosis, bone marrow dysplasia, and overlapping features of both myelodysplastic syndromes (MDS) and myeloproliferative neoplasms (MPN). A higher level of monoblasts, which are an immature monocytes with round nuclei, is a known risk factor in CMML, but the underlying biology and markers remain unclear. In this study, we developed a novel framework that integrates gene expression and morphology to investigate morphogenomic phenotypes in 12 CMML patients and 8 healthy controls. Recognizing that most existing spatial omics workflows are designed for adherent cells and are unsuitable for suspension cells like bone marrow mononuclear cells (BMMC), we first developed a protocol for immobilizing suspension cells. This was followed by MERFISH spatial RNA analysis of 492 genes across approximately 25,000 cells in a 16 mm2 region of interest over 34 hours. We then introduced a Morphology and Expression Data Optimal Clustering (MEDOC) algorithm, which uses adaptive Fourier-domain alignment and graph clustering to analyze cell shapes. In healthy BMMC, MEDOC analysis identified genes correlated with cell shape, findings that were substantiated by single-cell RNA sequencing data, confirming their association with nuclear shape. Leveraging this shape marker, we successfully identified a promonocyte population, a monoblast subtype that was previously difficult to distinguish. Furthermore, MEDOC analysis in CMML patients revealed two distinct nuclear shapes in mature leukemic monocytes: one resembling a normal kidney bean shape and another characterized by dysmorphic features. Thus, our framework offers a powerful tool to uncover the underlying biology of clinically relevant malignant cell morphologies and to enhance CMML patient stratification.



 Jagadish Sankaran, Ignasius Joanito, Giovani Claresta Wijaya, Ziyue Chen, Joseph Lee, Vaidehi Krishnan, Than Hein, Shyam Prabhakar. INTERGRATING CELL SHAPE AND SPATIAL EXPRESSION IDENTIFIES MORPHOGENOMICS STATES IN LEUKEMIA [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P08.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f4c00943719edf91d07a5ae18534128f66314a" target='_blank'>
              Abstract P08: INTERGRATING CELL SHAPE AND SPATIAL EXPRESSION IDENTIFIES MORPHOGENOMICS STATES IN LEUKEMIA
              </a>
            </td>
          <td>
            Jagadish Sankaran, Ignasius Joanito, Giovani Claresta Wijaya, Ziyue Chen, Joseph Lee, Vaidehi Krishnan, Than Hein, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Adoptive cell transfer (ACT) of engineered T cells is effective against B-cell malignancies but has faltered against solid tumors due to the immunosuppressive tumor microenvironment (TME). FASL and TGFβ are key mediators of T cell dysfunction in the TME and overexpressing dominant negative (dn) forms of their receptors in T cells increases anti-tumor efficacy in solid tumor models. However, an approach which directly targets the endogenous genes would be more amenable to multiplex editing and reduce competition with WT alleles. Here, we employ base editing (BE) in primary human T cells to install naturally occurring dominant negative FAS and TGFβR2 mutations. In vitro survival and proliferation assays demonstrate that BE T cells are resistant to pro-apoptotic and anti-proliferative effects of FAS and TGFβ signaling. CAR-T cells with BE-installed dn TGFβR2 or dn FAS exhibit improvements in cytotoxicity, while dn TGFβR2 CAR T demonstrate increased persistence and reduced expression of phenotypic markers of exhaustion compared to controls. Moreover, BE-engineered dn CAR T outperform lentiviral-engineered cDNA over expression counterparts in several functional assays. Considering the efficiency of BE and its amenability for multiplex editing, our novel approach lends itself to engineering strategies necessary to overcome T cell dysfunction in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7621f41d6df04230d16f2a2ff42337f01e7d9f5" target='_blank'>
              Installation of Dominant-Negative Mutations in FAS and TGFβR2 via Base Editing in Primary T Cells
              </a>
            </td>
          <td>
            B. Wick, Mitchell G. Kluesner, Nicholas J. Slipek, Joseph G. Skeate, Ethan M. Niemeyer, Beau R. Webber, B. Moriarity
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="ABSTRACT Cervical cancer, a prevalent gynecological malignancy, is primarily associated with human papillomavirus (HPV) infection. However, some cases display distinct molecular alterations beyond HPV, significantly impacting the tumor immune microenvironment (TIME) and posing therapeutic challenges. Among these molecular changes, NTRK gene fusions have emerged as critical drivers of tumor progression, invasiveness, and poor prognosis. This review highlights the role of NTRK fusions in cervical cancer oncogenesis, examining their effects on cellular signaling pathways and their interplay with HPV status in shaping TIME. The relationship between HPV infection and NTRK fusions is explored, revealing their combined influence on disease progression. Additionally, the potential of NTRK‐targeted therapies in precision oncology is discussed, emphasizing their promise as treatment options for cervical cancer. By addressing the molecular, immune, and clinical complexities of cervical cancer, this review provides valuable insights into advancing research and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/410638efee6638f05664c1c84150841fa419270e" target='_blank'>
              Role of NTRK Fusion Genes in the Tumor Immune Microenvironment of HPV (+/−) Cervical Cancer
              </a>
            </td>
          <td>
            Qiongying Wang, Chan Zhang, Shijia Liu, Wangshu Li, Wenjuan Wei, Aziz Ur Rehman Aziz, Han Lu, Daqing Wang
          </td>
          <td>2025-08-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lineage tracing techniques have been developed rapidly in the past decades by employing new genetic engineering tools. However, due to their invasive nature, these are difficult to apply to humans. Although endogenous DNA mutations can be used for in vivo lineage tracing in humans, their extremely low mutation rate presents substantial technical challenges. Epimutations on DNA methylation happen at a rate of about 0.001 per CpG site per division. Such rich and stable information enables high‐resolution, noninvasive lineage tracing in humans, as recently achieved with both MethylTree and EPI‐Clone. MethylTree is a computational innovation that accurately predicts cell lineages from single‐cell DNA methylation data, be it genome‐wide or targeted. EPI‐Clone is a targeted approach that requires careful CpG panel selection for specific tissues, which has been validated in blood. In this review, we present an overview of related historical studies, discuss the development of both MethylTree and EPI‐Clone, and compare these two approaches. Although EPI‐Clone is more scalable and cheaper, MethylTree has a higher resolution and works directly across different tissues. We demonstrate here that MethylTree also works well with EPI‐Clone data, thus providing a unified solution for epimutation‐based lineage tracing. Finally, we highlight the advantages of epimutation‐based lineage tracing, discuss future directions for tool development, and touch on considerations in biological applications. Epimutation‐based lineage tracing opens up an exciting avenue for noninvasive lineage tracing in humans across many biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/122d672a85ec23d09a8ffcbcd67f33fdfb64e9df" target='_blank'>
              DNA methylation meets lineage tracing: History, recent progress, and future directions
              </a>
            </td>
          <td>
            Ruijiang Fu, Mengyang Chen, Shoucheng Wang
          </td>
          <td>2025-09-21</td>
          <td>Quantitative Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pediatric papillary thyroid carcinoma (PPTC) has exhibited a progressive increase in incidence in recent years, characterized by heightened biological aggressiveness relative to adult papillary thyroid carcinoma (APTC). Nevertheless, the molecular mechanisms governing PPTC-specific pathobiology remain elusive. Through high-resolution single-cell RNA sequencing analysis of 90234 cellular transcriptomes from 4 PPTC and 9 APTC clinical specimens, a phenotypically distinct cellular subpopulation (ITGA2hi-PTC cells) mechanistically responsible for PPTC's distinct clinical behavior, is identified. By integrating PPTC-derived organoid models with in vivo functional validation and multiplex immunohistochemistry, it is demonstrated that ITGA2 orchestrates dual oncogenic pathways: 1) augmentation of glycolytic flux and 2) induction of M2 macrophage polarization. These synergistic mechanisms fundamentally drive PPTC oncogenesis and metastatic progression. Cross-validation across independent clinical cohorts consistently confirms the translational significance of these findings. Our multi-omics characterization of PPTC-specific cellular ecosystems and signaling cascades establishes a mechanistic framework for advancing diagnostic precision and targeted therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976a540c94cb69fb55d14150dadd38b429ede85" target='_blank'>
              Single-Cell Transcriptomics Reveals ITGA2-Mediated Metabolic Reprogramming and Immune Crosstalk in Pediatric Thyroid Carcinogenesis.
              </a>
            </td>
          <td>
            Zhi-Jun Zhan, Ning Li, Yan Sun, Lu Chen, , Yun-Han Shan, , Zhe-Cheng Li, , Neng Tang, Shi Chang, Peng Huang
          </td>
          <td>2025-09-23</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a heterogeneous illness influenced by intricate tumor-immune interactions and characterized by a dismal prognosis. Macrophage-mediated immunological signaling facilitates tumor proliferation and may associate inflammation in the tumor microenvironment (TME) of CRC with negative outcomes. Notwithstanding therapeutic advancements, resistance to treatment remains a significant obstacle. scRNA-seq offers comprehensive insights into the immune signaling network and immunological dynamics inside the CRC’s TME. Methods We integrated scRNA-seq data from GEO with extensive RNA-seq data from TCGA to elucidate immunological signaling and dynamic cellular variation in the TME of CRC. The analyses encompassed quality control via Seurat, InferCNV, Monocle, CellChat, and SCENIC, differential gene expression, inference of copy number variation (CNV), pseudo time trajectories, and intercellular communication. Prognostic modeling was conducted using Cox regression and LASSO. Immune infiltration and drug sensitivity were evaluated by CIBERSORT, ESTIMATE, xCell, TIDE, and pRRophetic. Functional validation encompassed siRNA knockdown, qRT-PCR, Western blot analysis, and cellular assessment in CRC cell lines. Results We discovered four categories of tumor cells exhibiting variations in cell cycle, stemness, and differentiation. The MKI67 + subpopulation exhibited a heightened dynamic cell state and engaged with macrophages via the MIF-(CD74+CD44) axis to facilitate immunological signaling. HMGA1 is a crucial transcription factor in this fraction, and its knockdown impedes CRC cell proliferation, motility, and invasion. The cancer model utilizing the MKI67 + TCs subpopulation (MTRS) successfully classified patient survival and linked with immune infiltration patterns and medication responses. Enrichment analysis revealed tumor-promoting and immunological signaling networks. Correlation scores suggest that this subpopulation may be linked to inflammation and immunosuppression inside the TME. Conclusion Our research indicates that the C2 MKI67 + TCs subpopulation is a key driver of immune signal transduction in CRC TME, which may induce inflammatory responses through interaction with macrophages, thereby leading to adverse consequences such as cardiac dysfunction. HMGA1 represents a viable target for immunotherapy, and our cancer model derived from this subpopulation offers prognostic significance and direction for immunotherapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b53660e6d491a7fc5588044fc19d5176db5aae" target='_blank'>
              Single-cell technology reveals the crosstalk between tumor cells and immune cells: driving immune signal transduction and inflammation-mediated cardiac dysfunction in the tumor microenvironment of colorectal cancer
              </a>
            </td>
          <td>
            Wenyang Nie, Wangzheqi Zhang, Zhikai Xiahou, Yuxuan Meng, Yuhang Liu, Jingwen Zhang, Zhen Wang, Yong Wang
          </td>
          <td>2025-08-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.


SIGNIFICANCE
ecDNA-driven tumor genome evolution provides a major challenge to curative cancer therapies. We demonstrate that ecDNA copy-number dynamics drives treatment resistance by promoting oncogene dosage-dependent phenotypic heterogeneity in MYCN-amplified cancers. Exploiting phenotype-specific vulnerabilities of ecDNA cells, therefore, presents a powerful strategy to overcome treatment resistance. See related article by Korsah, p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c185029056945edb11eefccadb52a90bcea86a" target='_blank'>
              Does neoadjuvant chemotherapy bias the transcriptomic profile of platinum-sensitive advanced high grade serous ovarian cancer patients towards a resistant phenotype? Findings from a gene expression meta-analysis
              </a>
            </td>
          <td>
            Durga Prasan, Unnati Raut, HK Madhumathi, Gangotri Siddappa, LJ Bonney, Amritha Suresh, S. Dhar
          </td>
          <td>2025-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The human body contains ~27–36 trillion cells of up to 10,000 cell types (CTs) within a volume of ~62–120 liters (males) and 52–89 liters (females). The Human Reference Atlas (HRA) v2.3 provides a quantitative 3D framework of CTs across 73 reference organs and 1,283 3D anatomical structures (ASs). The HRA Cell Type Population (HRApop) effort quantifies CTs per AS using high-quality single-cell (sc) data processed through scalable, reproducible workflows and cell type annotation (CTann) tools. HRApop v1.0 includes reference CT populations for 73 ASs (112 when sex-specific) using 662 datasets spatially registered to 230 locations across 17 organs (31 when sex-specific). For 558 sc-transcriptomics datasets (11,042,750 cells), CTs and biomarker expression were computed using Azimuth, CellTypist, and popV. To test generalizability, 104 sc-proteomics datasets (16,576,863 cells) were integrated. In total, HRApop includes 27,619,613 cells. HRApop can be used to predict (1) CT populations for 3D volumes in the human body and (2) the spatial origin of a tissue block, given a CT population. Data and code are at cns-iu.github.io/hra-cell-type-populations-supporting-information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebb2aca86dc56659eee58275de4ce878d6c3dc83" target='_blank'>
              Constructing and Using Cell Type Populations of the Human Reference Atlas
              </a>
            </td>
          <td>
            Andreas Bueckle, B. Herr, Lu Chen, Daniel Bolin, Danial Qaurooni, Michael Ginda, Yashvardhan Jain, Aleix Puig-Barbe, Kristin Ardlie, Fusheng Wang, Katy Börner
          </td>
          <td>2025-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7804cba30cd408c643b898725452b0298732929b" target='_blank'>
              A Spatial and Temporal Transcriptomic Atlas of Mouse Intestinal Regeneration.
              </a>
            </td>
          <td>
            Rachel Ofer, Tyler Tran, Ping He, Xia Qiu, Xinyu Hu, Emmanuel Zachariah, Curtis Krier, A. Saxena, Jiekun Yang, Michael Verzi
          </td>
          <td>2025-08-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS) are aggressive hematologic malignancies with poor outcomes. Leukemia stem and progenitor cells (LSPCs) are a subset of cells within the bulk tumor thought to be responsible for initiating disease and causing relapse. LSPCs evade chemotherapy partially due to their quiescent state. Therefore, studying this cell subpopulation is critical to identify new disease targets that can better the outcomes of AML/MDS patients. The use of RNAi and genetic approaches is technically challenging in primary leukemia cells and particularly LSPCs. Overcoming this technical hurdle could greatly expand the breath of pre-clinical and mechanistic examination of LSPCs. In this study, we describe a methodology to efficiently introduce siRNA, resulting in effective gene knock down in LSPCs. After isolation of LSPCs from primary patient samples, we showed that electroporation of these cells does not affect cell viability significantly. Further, using siGLO green transfection indicator, we show that RNA is introduced into the nucleus of these cells. siRNA transfection leads to efficient knockdown of target genes and has subsequent biological relevant activity. We have identified a method to effectively knock down genes in leukemia stem and progenitor cells, opening up new avenues to examine LSPC biology in human specimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ea32c9417556ae81a575d59c7de2abcdb20dbe5" target='_blank'>
              siRNA Mediated Genetic Perturbation of Primary Human Leukemia Stem and Progenitor Cells
              </a>
            </td>
          <td>
            Anagha Inguva, Hunter Tolison, Jeremy Rahkola, Craig T Jordan, Courtney Jones, Maria Amaya
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Multiple myeloma (MM) is a malignancy marked by uncontrolled plasma cell proliferation, immune evasion, and drug resistance. Despite advances in treatment, the disease remains incurable due to relapses and drug resistance. This study aims to investigate the molecular and cellular interactions within the myeloma microenvironment using single-cell RNA sequencing (scRNA-seq), Mendelian randomization (MR), and pathway analysis to uncover therapeutic targets.


METHODS
We used scRNA-seq to analyze tumor, immune, and stromal cell interactions in MM. Data were processed using Seurat for clustering, dimensionality reduction, and cell-type annotation. Gene Set Variation Analysis (GSVA) and cell-cell interaction analysis were performed to identify signaling pathways involved in disease progression. Two-sample MR was applied to explore causal relationships between genetic variants and gene expression. Finally, molecular docking simulations were used to identify potential small molecule modulators of key proteins involved in MM.


RESULTS
We identified significant immune-related signaling pathways, including PI3K-AKT-mTOR, WNT-β-catenin, and TGF-β, upregulated in immune cells within the MM microenvironment. Genes such as HLA-C, CTSS, and LRRFIP1 showed positive causal relationships with MM, while SHISA5 and ISG15 exhibited protective roles. Cell communication analysis revealed key ligand-receptor interactions between immune and tumor cells. Molecular docking identified promising small molecules like actein and aflatoxin B1 targeting ISG15 and TAGLN2.


CONCLUSIONS
This study reveals key genetic drivers and immune modulation mechanisms in MM. Targeting immune-related pathways, such as PI3K-AKT-mTOR and WNT-β-catenin, and small molecules targeting ISG15 and TAGLN2 could offer new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ddcfc1e5a8f4f32f9fd598c516ebb1c5e5f270" target='_blank'>
              Single-cell RNA sequencing reveals immune regulatory mechanisms and molecular therapeutic strategies in the microenvironment of multiple myeloma.
              </a>
            </td>
          <td>
            Qiang Su, Kehan Long, Mozili Adu, Meijun Jiang, Qiaochu Li, Xufeng Wan, Jian Cao, Yan Yue, Shuoyuan Li, Zhendong Ying, Qibin Liu, Chaoqun You, Zhuang Zhang, Duan Wang
          </td>
          <td>2025-09-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly lethal, aggressive malignancy. Little is known about the evolutionary trajectories of HCC and how clinical decision-making could be informed based on biopsies of the initial tumour. Here, we report the whole-exome sequencing of a unique series of resected HCC tumours and matched recurrences. This cohort included patients who received a liver transplant and who were immunosuppressed at time of recurrence, in comparison to patients who underwent liver resection for HCC and immunocompetent at time of recurrence, therefore facilitating analyses of immune selection in driving evolutionary divergence. We find extensive evolutionary divergence between baseline and recurrent tumours, with the majority of mutations in our cohort being private, in the process informing sampling guidelines for precision oncology in this disease. We also find no evidence that immunosuppression relaxes immune selection pressures, given the absence of a genomic footprint reflecting the presentation of neoantigens or altered dynamics of genomic evolution. We attribute this to the presence of genetic lesions that confer the capabilities of immune evasion in these tumours prior to transplantation, and then validate the link between immune selection pressures and the emergence of these lesions in publicly available HCC datasets. Our findings point to HCC as a cancer with extensive evolutionary divergence over time, partly defined by an irreversible, genetically determined trajectory towards immune escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9c9a24625027fee25b759620a813b537ff8ac7" target='_blank'>
              Evolutionary dynamics of recurrent hepatocellular carcinoma under divergent immune selection pressures
              </a>
            </td>
          <td>
            A. Chakravarthy, Elisa Pasini, Xun Zhao, Jeffrey To, Shu Yi (Roxana) Shen, Sandra Fischer, A. Ghanekar, Arndt Vogel, Robert C. Grant, Jennifer J Knox, Gonzalo Sapisochin, Gregory J. Gores, Daniel De Carvalho, Mamatha Bhat
          </td>
          <td>2025-08-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract The interaction between intratumoral microbiome and the tumor microenvironment (TME) has furthered our understanding of tumor ecology. Yet, the implications of their interaction for lung cancer management remain unclear. In the current work, we collected host transcriptome samples and matched intratumoral microbiome samples, as well as detailed clinical metadata from The Cancer Genome Atlas (TCGA) of 478 patients with lung adenocarcinoma (LUAD). Utilizing the multiomics integration approach, we comprehensively investigated the crosstalk between the TME and intratumoral microbiome in patients with LUAD. First, we developed a prognostic model based on the TME signatures (TMEindex) that clearly distinguished clinical, survival, and response to immunotherapy of patients with LUAD. Additionally, we found profound differences in intratumoral microbiota signatures, including alpha- and beta-diversity, among patients with different survival risks based on the TME signatures. In depth, we detected that genera Luteibacter and Chryseobacterium were strongly negatively and positively associated with patients’ survival risk, respectively, suggesting their opposing roles in cancer progression. Moreover, we developed a model that fused intratumoral microbial abundance information with TME signatures, called intratumoral microbiome-modified TMEindex (IMTMEindex), leading in predicting patient overall survival at 1-, 3-, and 5-years. Future clinical profiling of the specific intratumoral microbes in the TME could improve prognosis, inform immunotherapy, and facilitate the development of novel therapeutics for LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ceebaa130c34b48cdab5c0e6efa7c18090d66b0" target='_blank'>
              Tumor microenvironment signatures enhances lung adenocarcinoma prognosis prediction: Implication of intratumoral microbiota
              </a>
            </td>
          <td>
            Fei Zhao, Lei Wang, Dongjie Du, Heaven Zhao, Geng Tian, Yufeng Li, Yankun Liu, Zhiwu Wang, Dasheng Liu, Jingwu Li, Lei Ji, Hong Zhao
          </td>
          <td>2025-08-11</td>
          <td>Microbial Cell</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="We present a novel classification system for murine thymic epithelial cells (TECs), identifying 11 distinct types, four in the thymic cortex and seven in the medulla, based on their spatial localization and unique ultrastructural features. As key stromal components of the thymic microenvironment, TECs play indispensable roles in T cell development, including thymocyte selection, antigen presentation, and structural support. Our classification spans from the subcapsular cortex to the deep medulla and incorporates microanatomical context, morphology, and functional characteristics, providing a comprehensive and flexible framework to study TEC heterogeneity in relation to thymopoiesis. Aligning with TEC classification in rats and humans, this system highlights conserved spatial organization across species while remaining adaptable for refinement. Each TEC type is distinguished by features such as chromatin organization, cytoplasmic morphology, vacuolar content, and organelle distribution, attributes that suggest distinct functional contributions to various stages of thymocyte maturation. Importantly, the classification is designed for logical expansion both horizontally (inclusion of additional subtypes within the proposed TEC types) and vertically (inclusion of entirely novel TEC types). By integrating detailed morphological observations with testable functional hypotheses, this framework underscores the essential role of TEC diversity in supporting thymic architecture and orchestrating effective T cell output. Overall, it offers a robust foundation for future research into immune development and the pathological consequences of thymic dysfunction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a499d2fa9f48a695cdae808a5b371013bf08351d" target='_blank'>
              Mapping murine thymic epithelial cells: functional ultrastructure and implications for thymopoiesis.
              </a>
            </td>
          <td>
            Stepan Vodopyanov, Leslie Gunther-Cummins, Joseph Churaman, Xheni Nishku, T. Poutahidis, A. Hardas, Frank P Macaluso, George S. Karagiannis
          </td>
          <td>2025-08-01</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/705a9b21b7c445d9d6a328fd2aa7da43797bf85e" target='_blank'>
              Strain-Level Genetic Heterogeneity and Colonization Dynamics Drive Microbiome Therapeutic Efficacy
              </a>
            </td>
          <td>
            K. Chen, Y. Liu, J. Rong, N. Dai, C. Xu, H. Li, L. Zhong, B. Wang, Z. Ji, S. Xie, Y. Xu, F. Yang, J. Wang, D. Li, Y. Gu, X. Zhou, Y. Li, M. Chen, Y. Chen, W. Li, Z. Tang, J. Cai, J. Xu, S. Xia, Q. Zhan, Z. Zhou
          </td>
          <td>2025-08-19</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Colorectal cancer (CRC) is an aggressive and heterogeneous tumor with limited therapeutic options. Tumor-associated neutrophils (TANs) play multifaceted roles in the tumor microenvironment (TME) depending on their polarization. Understanding TAN heterogeneity and the mechanisms underlying their tumor-promoting activities is critical for advancing CRC treatment. Methods: This study integrated single-cell RNA sequencing, spatial transcriptomics, bulk RNA sequencing, and in vivo/ex vivo experiments to characterize TANs in CRC. Results: We identified a distinct subpopulation of TANs characterized by high CCCTC-binding factor (CTCF) expression (CTCF+ TANs) enriched in hypoxic tumor regions. CTCF+ TANs exhibit enhanced migratory capacity and IL-1β secretion, correlating with poor prognosis and resistance to immunotherapy. Mechanistically, CTCF regulates the expression of Migration and Invasion Enhancer 1 (MIEN1), which promotes TAN recruitment and migration without triggering inflammation. Functional studies revealed that CTCF+ TANs suppress T-cell immunity, facilitate epithelial-to-mesenchymal transition (EMT), and contribute to CRC progression and metastasis. In vivo, targeting CTCF-MIEN1-IL-1β signaling rescued the immunosuppressive microenvironment and improved the efficacy of anti-PD-L1 therapy. Conclusions: CTCF+ TANs represent a novel TAN subtype that drives CRC progression and immunosuppression via the CTCF-MIEN1-IL-1β axis. These findings highlight the potential of targeting CTCF+ TANs to overcome immunotherapy resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1e6c853622163edbe8255deed0daaabb0c23ad1" target='_blank'>
              Comprehensive analytical approach identifies a subtype of CTCF+ tumor-associated neutrophils associated with CRC development and as a biomarker for immunotherapy
              </a>
            </td>
          <td>
            Jie-pin Li, Yuan Liu, Yi Zhang, Qian-wen Ye, Guo Xu, Jin-chen Chong, Yi Yin, Yang Li, Shuang-shuang Wang, Jin-yong Zhou, Jun Qian, Shen-lin Liu, Xi Zou, Yu-gen Chen
          </td>
          <td>2025-08-16</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Objectives This study charted how investigations of tumour-microenvironment (TME) dynamics have shaped global thinking on drug resistance in breast cancer over the past three decades. Methods Web of Science Core Collection records (1995–2024) were harvested on 15 May 2025 with a three-block Boolean string integrating disease, microenvironmental, and resistance terms. After duplicate removal, document-type filtering, language restriction, and natural-language adjudication, 1 303 original articles and reviews remained. Bibliometrix (R 4.3.2) generated annual output, citation kinetics, and growth rates; VOSviewer (fractional counting, resolution = 1.0) mapped co-authorship, co-citation, and keyword networks. Results Publication volume expanded at a compound annual rate of 18.7%, progressing through formative (1995–2004), consolidation (2005–2012), and expansionary (2013–2024) epochs. Keyword drift traced a pivot from endocrine-centric lexicons (“tamoxifen”, “estrogen receptor”) to TME-immune discourse (“extracellular vesicles”, “tumour-associated macrophages”, “nanoparticles”); “microenvironment”, “hypoxia”, and “multidrug resistance” emerged as high-betweenness semantic bridges. The United States–China dyad accounted for 56.1% of output and anchored 24.4% of internationally co-authored papers, funnelling methodological innovation into an increasingly multipolar network, here defined as a collaboration graph containing at least four countries that each contribute ≥ 5% of total node-level betweenness centrality. Institutional overlays revealed late-decade ascendancy of East-Asian medical universities, while legacy North-American and European centres maintained brokerage dominance. Conclusion Bibliometric cartography documents a decisive ecological re-framing of breast-cancer drug-resistance research, with immune-stromal crosstalk supplanting receptor-centric models. Cross-border, multi-omics consortia that fuse East-Asian clinical scale with Western technological depth appear best poised to convert TME insights into resistance-reversing interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37738d1f4bd99af794fa1fdf0d88aeebed7041dc" target='_blank'>
              Mapping thirty years of tumour-microenvironment-driven drug resistance in breast cancer: a global bibliometric analysis
              </a>
            </td>
          <td>
            Zuoping Duan, Yonghong Zhou, Yang Wu
          </td>
          <td>2025-08-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Efforts to understand the tumor microenvironment through basic science research and the cancer genome atlas (TCGA) data analysis have led to the creation of unique immune transcriptomic signatures from tumor-infiltrating lymphocytes (TIL). However, no pan-cancer analysis has been conducted to compare the prognostic performance of these signatures using overall survival (OS) or progression-free interval (PFI) as endpoints. We compiled a library of 146 TIL-immune signatures and evaluated gene signature score correlation with OS and PFI for 9,961 available TCGA samples across 33 tumor types. Zhang CD8 TCS demonstrated higher accuracy in prognosticating both OS and PFI across the pan-cancer landscape; however, variability was seen across cancer types and germ cell origin. Cluster analysis compiled a group of six signatures (Oh.Cd8.MAIT, Grog.8KLRB1, Oh.TIL_CD4.GZMK, Grog.CD4.TCF7, Oh.CD8.RPL, Grog.CD4.RPL32) whose association with OS and PFI could potentially be conserved across multiple neoplasms. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04102-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfb16ad03a570f27481909ae39fc0049c1c73e2a" target='_blank'>
              A pan-cancer comparative analysis of the cancer genome atlas transcriptomic TIL-immune signatures
              </a>
            </td>
          <td>
            Kyle J. Hitscherich, Darryl Nousome, Aaron J. Dinerman, Victoria Dulemba, Frank J Lowery, N. Nilubol
          </td>
          <td>2025-08-07</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Anaplastic thyroid carcinoma (ATC) is one of the most lethal human malignancies, often evolving from differentiated thyroid carcinoma (DTC) through a poorly understood dedifferentiation process. To elucidate this transition, spatial transcriptomic sequencing (spRNAseq) is performed on seven samples containing coexisting regions of ATC, poorly differentiated thyroid carcinoma, and DTC. SpRNAseq revealed that ATC regions were characterized by upregulated genes involved in immune suppression, angiogenesis, and extracellular matrix remodeling. Whole-exome sequencing and inferCNV analysis confirmed that adjacent DTC regions harbored mutational burdens comparable to those of ATC regions, suggesting early genomic priming for dedifferentiation. Trajectory analysis delineated a stepwise reprogramming process and identified four gene modules associated with the loss of thyroid differentiation, among which PDCD4 and TYMP emerged as key regulators. Notably, TYMP⁺ tumor-associated macrophages (TAMs) were highly enriched in ATC regions and contribute to an immunosuppressive microenvironment. Mechanistic experiments demonstrated that loss of PDCD4 led to eIF4A-dependent overexpression of immunosuppressive effectors, promoting the high infiltration of TYMP⁺TAMs in ATC. These findings support that coexisting DTC regions with ATC-like genomic alterations undergo sequential transcriptomic reprogramming and immune microenvironment remodeling to evolve into a full ATC pathological phenotype, in which PDCD4 loss-induced TAMs formation plays a critical role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4fbade6a7e0bbb1789c00df5213a5311bb8e73a" target='_blank'>
              Spatial Transcriptomics Reveals Transcriptomic and Immune Microenvironment Reprogramming during Thyroid Carcinoma Dedifferentiation.
              </a>
            </td>
          <td>
            K. Ning, Bu Zou, Yongchao Yu, Taonong Cai, Zhenyu Luo, Yu Guo, Yi Wu, Xiujiao Shen, Hao Li, Mengyuan Fang, J. Bu, Han Hong, Zan Jiao, Tong Wu, Yulong Wang, Tianrun Liu, Wei-chao Chen, Wanming Hu, Mingjie Jiang, A. Yang
          </td>
          <td>2025-09-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38cc5302dc11612c06528bfceedae2ff2c387b6" target='_blank'>
              S1PR5 as a prognostic biomarker in colon cancer: insights into efferocytosis-related mechanisms and immune modulation.
              </a>
            </td>
          <td>
            Aijie Pei, Ling Lu, Xiaolin Wu
          </td>
          <td>2025-09-15</td>
          <td>Journal of molecular histology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncolytic viruses represent an innovative strategy for cancer therapy. However, extensive gene expression reprogramming within tumor cells may hinder viral propagation by affecting essential cell-virus interactions. Here, through genome-wide CRISPR/Cas9 library screening, Suppressor of variegation 3–9 homolog 2 (SUV39H2), a histone methyltransferase, was identified as a critical factor in mediating resistance to oncolytic herpes simplex virus 1 (oHSV-1) in oral squamous cell carcinoma (OSCC). Functional studies in SCC15 cells revealed that SUV39H2 knockdown facilitated viral replication, while its overexpression suppressed it. The inhibitor OTS186935 targeting SUV39H2 was administered to evaluate its effects on viral replication both in vitro and in vivo. Pretreatment with OTS186935 in SCC15, SCC7, and MCF7 led to a significant enhancement of viral replication. Combined treatment with OTS186935 and oHSV-1 demonstrated significant anti-tumor efficacy in BALB/c nude mice bearing SCC15 tumors. SUV39H2 was shown to regulate the trimethylation of lysine 9 on histone 3 (H3K9me3) at the viral promoter regions of immediate-early gene ICP0, ICP4 and early gene ICP8, thereby repressed viral gene transcription. However, oHSV-1 infection induced the degradation of SUV39H2, a process mediated by the viral protein ICP0 through the proteasomal pathway. Findings from studies in SCC7 cells further supported the observation that SUV39H2 knockdown enhanced viral replication. Moreover, SUV39H2 downregulation increased CD4+ and CD8+ T cell infiltration in syngeneic tumors treated with oHSV-1. TCGA database analysis revealed that SUV39H2 is associated with distinct immune cell infiltration patterns across different cancer types and correlates with immune checkpoint expression. These results highlight the role of SUV39H2 in regulating oHSV-1 replication and indicate that SUV39H2 may represent a potential target to improve the efficacy of oncolytic virotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5611aab0ab9424f10f41dc9d263421131c9b7b09" target='_blank'>
              CRISPR/Cas9 screening identifies SUV39H2 as a key regulator of oHSV-1 resistance in oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Manman Qiu, Qicheng Zhang, Rui Li, Rongrong Wei, Jiawei Zhao, Juan Tan, Hongkai Zhang, Wentao Qiao
          </td>
          <td>2025-08-23</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) shows racial disparities, with higher incidence in women of African ancestry (AA) compared to European ancestry (EA). Meta-transcriptomic analysis of TNBC tumor tissues from AA (n = 17) and EA (n = 19) subjects revealed distinct microbial landscapes. Hierarchical clustering based on microbial transcripts separated samples into two groups predominantly defined by racial ancestry. Bacterial genera including Hafnia and Cedecea were more abundant in AA tumors, while Erwinia was higher in EA tumors. Cellular composition analysis by xCell revealed differences in immune cell populations, with AA tumors having higher Th1 cell abundance and EA tumors containing higher macrophage M2 cell abundance. Nonetheless, AA women with high M2 abundance experienced poorer disease-free survival (DFS) than EA women. Integrative analyses revealed that high expression of human SPDYE2B gene was associated with Hafnia abundance and decreased DFS, highlighting complex host-microbe interactions in TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4784e682d1bb059c5d6fa5b77baa53f7b2ce04c2" target='_blank'>
              Race-related host and microbe transcriptomic signatures in triple-negative breast cancer
              </a>
            </td>
          <td>
            Roshan Kumar, Susan Duyar-Ayerdi, Aishwarya Sundaresan, V. Srinivasasainagendra, C. Pedamallu, M. Behring, D. Chandrashekar, I. Eltoum, S. Varambally, Hemant K. Tiwari, Sadeep Shrestha, Paul L Auer, L. Chaudhary, John R Kirby, Clayton C. Yates, Upender Manne, A. Ojesina
          </td>
          <td>2025-08-08</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="


 The past decade has brought unprecedented progress in understanding the genetics of epithelial ovarian cancer (EOC), but cures remain elusive due to the lack of insights into mechanisms that can be leveraged to develop new therapies. EOC is genetically heterogeneous and, therefore, it is imperative that therapeutic strategies become personalized by targeting distinct molecular subsets of EOC. Despite the importance of epigenetic regulators in EOC is increasingly recognized, the mechanism of action for these epigenetic regulators and the associated vulnerabilities exposed by their dysregulations have remained elusive for most of them. Here we will discuss our recent work on how to leverage epigenetic mechanisms and the associated rational combination strategies to address unmet clinical needs in genetically defined ovarian cancers through integrated approaches.



 Rugang Zhang. Leveraging ovarian cancer heterogeneity to drive precision therapeutic innovation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA011.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b32735b6208ac4877d573f8aa3228490f4c8b5fd" target='_blank'>
              Abstract IA011: Leveraging ovarian cancer heterogeneity to drive precision therapeutic innovation
              </a>
            </td>
          <td>
            Rugang Zhang
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent rare cancers originating from mesenchymal tissues, featuring over 70 histological subtypes with diverse genetic defects and clinical outcomes. Current treatment options for STS mainly involve surgical procedures and anthracycline-based chemotherapy, which often prove ineffective across multiple STS subtypes. Consequently, there's a pressing need for novel, STS-specific therapies. Targeting cell surface antigens expressed exclusively in STS has emerged as a promising approach. However, the surfaceome of STS remains largely uncharacterized. In our research, we employed mass spectrometry to analyze biotinylated and immunoprecipitated cell surface proteins expressed in STS cell lines and patient samples. Through this method, we've identified several promising cell surface antigens that could potentially serve as therapeutic targets for STS. Further validation of these antigens across various experimental settings is crucial to confirm their efficacy and pave the way for new strategies in STS treatment.



 Phuong Mai Hoang, Wai Khang Yong, Dennis Kappei, Anand D. Jeyasekharan. SURFACEOME PROFILING TO IDENTIFY TARGETABLE CELL SURFACE ANTIGENS IN SOFT TISSUE SARCOMAS [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P61.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6123041a91ea4253d5c71b5fec3661998638d06f" target='_blank'>
              Abstract P61: SURFACEOME PROFILING TO IDENTIFY TARGETABLE CELL SURFACE ANTIGENS IN SOFT TISSUE SARCOMAS
              </a>
            </td>
          <td>
            P. Hoang, Wai Khang Yong, Dennis Kappei, A. Jeyasekharan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>