<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-10 09:47:58 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="


 Glioblastoma are hallmarked by clonal evolution and transcriptional heterogeneity linked to genetic and environmental diversity resulting in therapy resistance and disease progression. In this study, we integrate heritable molecular barcoding and spatial transcriptomics to enable high-resolution cell lineage tracing in the spatial context of the glioblastoma TME.



 Patient-derived glioblastoma cells from two individual cases were isolated from anatomically and pathologically distinct regions: the primary tumor core, the tumor periphery, and the recurrent lesion. Each subline was transduced with predefined FlexTrack lentiviral plasmids encoding GFP, resulting in uniquely barcoded (9N) cell lines. Immediately before injection, the barcoded glioblastoma populations were pooled and introduced into ex vivo human cortical slices. Live-imaging was used to trace GFP-positive, barcoded cell populations. After incubation, the fresh-frozen slices were processed for spatial transcriptomics and the line-specific barcodes were mapped according to the 3′ poly(A) tails of captured transcripts. Graph neural networks based on single-cell deconvolution were employed to spatially analyze the cellular ecosystems.



 We successfully deconvoluted and reconstructed the trajectories of injected glioblastoma cells linking their transcriptome phenotypes along with clonal identity. Multi-omic integration of spatial copy-number alterations and barcoding based pseudo-trajectories enabled reconstruction of the temporal transcriptional dynamics linked to the heterogeneity of tumor ecosystems.



 This study serves as a proof-of-concept for integrating molecular barcoding and spatial transcriptomics in cortical slice culture. Our approach to molecular barcoding in complex model systems opens new avenues to investigate the contextual dynamics of clonal evolution in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f785ed6bb81bfa9c1d1d1cee449d66b313a3ee5" target='_blank'>
              P02.23.B MOLECULAR BARCODING OF GLIOBLASTOMA IN HUMAN ORGANOTYPIC CORTICAL SLICE CULTURE
              </a>
            </td>
          <td>
            G. Kastner, M. Mühlbauer, J. Jung, S. Langer, L. Müller, A. Baginska, P. Berger, C. Dobersalske, S. Ertmer, F. M. Gerach, N. Neidert, K. Herrmann, L. Rauschenbach, L. van Hijfte, U. Sure, K. D. Kokkaliaris, B. M. Grüner, D. Heiland, B. Scheffler
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728733c32d8e17fbc34a5070c8b658170e0a8a22" target='_blank'>
              Identifying tissue states by spatial protein patterns related to chemotherapy response in triple-negative breast cancer
              </a>
            </td>
          <td>
            L.M. Luque, A. Khan, G. Torrisi, Tessa D. Green, D. Hardman, C. Owczarek, T. A. Phillips, D. Marks, M. Parsons, C. Sander, L. Schumacher
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) presents diverse histomorphological features within the tumor microenvironment (TME) that influence prognosis and response to immunotherapy. Leveraging contrastive learning, we developed an unbiased atlas of cell neighborhoods to systematically explore the LUAD microenvironment at the cellular scale and investigate how these neighborhoods combine to form histologic patterns. This multiscale approach enables a comprehensive understanding of both cell-specific interactions and broader histologic architecture in LUAD. Our analysis identified distinct histomorphological phenotype clusters of cellular neighborhoods (cn-HPCs) with prognostic significance. For example, cn-HPC 0 was associated with immune activation and favorable survival, while cn-HPC 23 was enriched in necrotic, immune-excluded regions and aligned with poorer outcomes. Through associations with immunophenotypes, co-expressed gene modules, and pathway enrichment, we found that cn-HPCs reflect underlying processes of immune modulation, cellular growth, and inflammation, offering insights into the functional landscape of LUAD. In an independent LUAD immunotherapy cohort, we found that cn-HPC 23 showed promising predictive value for treatment response. These findings suggest that fine-grained spatial profiling of the TME may offer complementary value for identifying patients more likely to benefit from immunotherapy. Taken together, our study highlights the potential of cn-HPCs to bridge histopathological patterns and clinical outcomes, supporting their relevance for prognosis and treatment stratification in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732a50286d99e4889f11b9dfeef8dfb5de086217" target='_blank'>
              Contrastive learning uncovers cellular interactions and morphologies in the tumor microenvironment of lung adenocarcinoma linked to immunotherapy response
              </a>
            </td>
          <td>
            H. Le, N. Coudray, A. Yeaton, Sitharam Ramaswami, A. Karimkhan, Wei-Yi Cheng, James Cai, Tai-Hsien Ou Yang, S. Punekar, Luis Chiriboga, Vamsidhar Velcheti, Kwok-Kin Wong, D. Sterman, Andre L. Moreira, Harvey I. Pass, A. Tsirigos
          </td>
          <td>2025-10-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Diseases such as cancer involve alterations in cell proportions, states and interactions, as well as complex changes in tissue morphology and architecture. Histopathological diagnosis of disease and most multiplexed spatial profiling relies on inspecting thin (4–5 µm) specimens. Here we describe a high-plex cyclic immunofluorescence method for three-dimensional tissue imaging and use it to show that few, if any, cells are intact in conventional thin tissue sections, reducing the accuracy of cell phenotyping and interaction analysis. However, three-dimensional cyclic immunofluorescence of sections eightfold to tenfold thicker enables accurate morphological assessment of diverse protein markers in intact tumor, immune and stromal cells. Moreover, the high resolution of this confocal approach generates images of cells in a preserved tissue environment at a level of detail previously limited to cell culture. Precise imaging of cell membranes also makes it possible to detect and map cell–cell contacts and juxtracrine signaling complexes in immune cell niches. Confocal microscopy enables high-resolution, high-plex 3D cyclic immunofluorescence of 30- to 50-µm-thick tissue sections. The approach allows for rich phenotypic assessments of intact cells and intercellular interactions with subcellular resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed4146e9d8a32db71a92a19484656680fad514b9" target='_blank'>
              Highly multiplexed 3D profiling of cell states and immune niches in human tumors
              </a>
            </td>
          <td>
            Clarence Yapp, A. J. Nirmal, Felix Y. Zhou, A. Y. Wong, Juliann B. Tefft, Yi Daniel Lu, Zhiguo Shang, Z. Maliga, P. Llopis, George F. Murphy, C. G. Lian, G. Danuser, S. Santagata, P. Sorger
          </td>
          <td>2025-09-29</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="


 Tumors acquire a set of functional capabilities, known as the Hallmarks of Cancer, to thrive and grow. While extensive research has revealed how gene and protein expression networks contribute to these traits, the relationships between tumor cell states, spatial organization of immune and stromal cells in the tumor microenvironment, and tumor genetics, remains poorly understood. To address this, we constructed a cohort of primary tumor tissues from 117 pancreatic ductal adenocarcinoma (PDAC) patients. Using Multiplexed Ion-Beam Imaging (MIBI) with a 40-antibody panel, we mapped the spatial organization of major cell types and their phenotypes. We also conducted targeted genomic and bulk proteomic analyses to explore how spatial organization patterns align with tumor genetics and proteome profiles. We identified phenotypic subtypes of tumor epithelium that shape distinct microenvironmental niches: (i) Glycolytic tumors expressing proteins such as GLUT1, LDHA, MCT1 and HIF1α reside in hypoxic niches. They are encased by dense fibrotic stroma and lacking vasculature and immune cell infiltration. (ii) Tumor and metaplastic epithelium that exhibit an oxidative proteome profile, are highly enriched for ECAD and PD-L1 expression, and are situated near pancreatic epithelium and vasculature, with significant immune infiltration. (iii) Myeloid-interacting tumors expressing CXCL5, heavily infiltrated with neutrophils and characterized by low-collagen and high FAP positive fibroblasts. Notably, neutrophils accumulate within the lumen of tumor ducts where they exhibit increased apoptosis, DNA damage, and netosis. These highly neutrophil-infiltrated tumors, associate with poor disease outcome. The genetic landscape of PDAC influences these spatial and phenotypic patterns. Patients with TGFβ/SMAD pathway mutations, demonstrate an increased proportion of basal tumor cells accompanied by reduced lymphocytic infiltration. Similarly, distinct TP53 variants shape immune-cell infiltration: patients harboring missense mutations show increased lymphocytic infiltration compared to those with truncating mutations. These findings underscore the strong link between tumor genetics, phenotypic heterogeneity and cellular architecture of the PDAC microenvironment, offering new insights into collective tumor behavior.



 Ofer Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, Inti Zlobec, Tamar Geiger, Leeat Keren. Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B123.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb5d6211d2e33680561c9558492cf85ff16b33" target='_blank'>
              Abstract B123: Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization
              </a>
            </td>
          <td>
            O. Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, I. Zlobec, Tamar Geiger, Leeat Keren
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235c8c18e964369ee970e129fd3fe5bd3b1a1aa6" target='_blank'>
              In silico reconstruction of primary and metastatic tumor architecture using GIS-augmented spatial transcriptomics
              </a>
            </td>
          <td>
            Jin Young Yoo, Sabrina Akter, Qianying Zuo, Amir Kazemi, Wenjun Wu, Mahima Goel, Audrey Lam, Debapriya Dutta, Betsy Barnick, M. Pekmezci, Maria Grosse Perdekamp, Aiman Soliman, Zeynep Madak-Erdogan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d523903df00c8d74634f25362c4c022807a0ad86" target='_blank'>
              Spatial proteomics for investigating solid tumor resistance mechanisms
              </a>
            </td>
          <td>
            X. M. M. Zhou, A. D'Amiano, Charles Lu, Vrinda Madan, Sara Khoshniyati, J. Kollings, Noah E Sunshine, S. Surwase, Joel C. Sunshine
          </td>
          <td>2025-10-08</td>
          <td>Cancer and Metastasis Reviews</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Integration of scRNA-seq data from millions of cells revealed a high diversity of cell types in the healthy and diseased human lung. In a large and complex organ, constantly exposed to external agents, it is crucial to understand the influence of lung tissue topography or external factors on gene expression variability within cell types. Here, we apply three spatial transcriptomics approaches, to: (i) localize the majority of lung cell types, including rare epithelial cells within the tissue topography, (ii) describe consistent anatomical and regional gene expression variability within and across cell types, and (iii) reveal distinct cellular neighborhoods in specific anatomical regions and examine gene expression variations in them. We thus provide a spatially resolved tissue reference atlas in three representative regions of the healthy human lung. We further demonstrate its utility by defining previously unknown imbalances of epithelial cell type compositions in chronic obstructive pulmonary disease lungs. Our topographic atlas enables a precise description of characteristic regional cellular responses upon experimental perturbations or during disease progression. Using multiplexed spatially resolved transcriptomic approaches, the authors generate a topographic atlas of the healthy adult lung with location-related gene expression variability or neighbourhood changes as exemplified by COPD patient sample analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe80d855be30beaf44c2810f98505d59172a3d0" target='_blank'>
              Spatial single-cell atlas reveals regional variations in healthy and diseased human lung
              </a>
            </td>
          <td>
            A. Firsova, Sergio Marco Salas, L. B. Kuemmerle, X. Abalo, A. Sountoulidis, L. Larsson, Krishnaa T. Mahbubani, Jonas Theelke, Žaneta Andrusivová, Leire Alonso Galicia, A. Liontos, Tamás Balassa, Ferenc Kovács, Peter Horvath, Yuexin Chen, Janine Gote-Schniering, M. Stoleriu, Jürgen Behr, Kerstin B. Meyer, W. Timens, Herbert B. Schiller, Malte D. Luecken, F. Theis, J. Lundeberg, Mats Nilsson, M. Nawijn, Christos Samakovlis
          </td>
          <td>2025-11-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Abstract The heterogeneous nature of cell populations in human tumors is a major contributor to tumor evolution, including and perhaps most importantly in response to treatment. Here, we review current knowledge on tumor heterogeneity and cell state plasticity in small cell lung cancer (SCLC), a fast growing and highly metastatic form of lung cancer which develops rapid resistance to therapy. There is a pressing need to expand treatment options for patients with SCLC, which requires a better understanding of the mechanisms by which this disease is able to rapidly grow and evolve in response to therapy. Our current understanding points to epigenetic rather than genetic factors in defining major aspects of inter- and intra-tumoral heterogeneity in SCLC. SCLC is overall considered to be a neuroendocrine (NE) cancer type but SCLC tumors harbor a wide diversity of cancer cell states, including both NE and non-neuroendocrine (non-NE) states, defined by their mutually exclusive expression of a set of transcription factors such as ASCL1, NEUROD1, and POU2F3. The immune microenvironments of SCLC tumors also contain a great deal of heterogeneity. Here, we discuss the different SCLC cell states associated with their defining transcription factors, as well as the epigenetic mechanisms regulating the ability of SCLC cells to switch from one state to another. We further discuss how the composition of SCLC tumors and the surrounding immune cells may affect the response to chemotherapy and immunotherapy. Being able to control plasticity and heterogeneity in SCLC may in the future offer unique opportunities to improve treatment efficacy in this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f9c75cb099eaa87426731186cf92ba20e24f46a" target='_blank'>
              Tumor Heterogeneity and Plasticity in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Gina Duronio, Julien Sage
          </td>
          <td>2025-10-01</td>
          <td>Lung Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053a87c896e7b2228f019eac1392494ac26a840e" target='_blank'>
              A spatial map of MAPK-activated immunosuppressive myeloid populations in pediatric low-grade glioma
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Romain Sigaud, Evan Puligandla, Bridget Liu, E. Karimi, Alva Annett, M. Rezanejad, Wajih Jawhar, Robert Taylor, Yi Cao, S. Schmid, F. Selt, P. Hernáiz Driever, Svea Horn, P. Sievers, M. Prinz, M. Glatzel, Christian Mawrin, C. Hartmann, C. Monoranu, Liza Konnikova, F. Sahm, Stefan M. Pfister, Olaf Witt, David T. W. Jones, A. Koch, Claudia L Kleinman, David Capper, L. Walsh, N. Jabado, T. Milde
          </td>
          <td>2025-09-15</td>
          <td>Nature Immunology</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The spatial organization and composition of the tumor-immune microenvironment (TME) play a critical role in shaping the progression of many solid cancers, but the organization of the TME in primary prostate cancer (PCa) remains poorly characterized. We therefore profiled the abundance and spatial distributions of major cell types involved in adaptive immunity in 29 radical prostatectomy specimens stratified into high (HGG; n=14) and low Gleason-grade (LGG; n=15). Compared to LGG, HGG PCa exhibited significantly greater B and T cell infiltration with many immune cells organized into clusters, some of which resembled tertiary lymphoid structures (TLSs). In HGG tumors, these clusters were dense, symmetric, rich in PD-1+ T cells, and frequently proximate to the tumor compartment. LGG clusters were less well organized, and T cell depleted. Thus, a subset of high-grade PCa harbor organized immune clusters that may play a role in tumor control and contain therapeutically targetable T and B cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e5e762dad0168906d48c0967522a47db6d0ac" target='_blank'>
              Spatially organized lymphocytic microenvironments in high grade primary prostate tumors
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, Jia-Ren Lin, Brian Labadie, A. Atmakuri, Z. Maliga, Eamon Toye, K. Chaudagar, Madeleine S. Torcasso, S. Coy, G. Fanelli, Brigette Kobs, F. Socciarelli, Andréanne Gagné, Eliezer Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Spontaneously transformed mouse ovarian surface epithelial cell line ID8 generates a syngeneic mouse model, recapitulating the primary and metastatic stages of the malignancy hence, frequently used in pre-clinical evaluation. Due to inherent phenotypic plasticity, primary and metastatic tumors show contrasting phenotypic features dictated by the tissue niche. To address the tumor heterogeneity within the model we stratified it into a specific molecular subtype, which would provide an accurate setting for interpretation of experimental data. Stratification was performed on ID8 cell line and ID8 generated xenografts based on immunostaining, live cell imaging, histopathology, transcriptomics and proteomics. Tumor microenvironment (TME) investigation for each xenograft was done through flow cytometry analysis. ID8 cells present epithelial/mesenchymal (E/M) phenotype. Phenotypic plasticity retention in them causes class switching of ID8 cells in response to niche alteration, from E/M (in culture) to intermediate epithelial (iE) in orthotopic tumors while intermediate mesenchymal (iM) in subcutaneous tumors. Metastasis of ID8 primary tumor resulted into differentiated mesenchymal phenotype in ascites while differentiated epithelial phenotype in secondary intestinal tumors. Immune profiling of primary and secondary xenografts revealed differences in immune repertoire of tumors, highlighting subcutaneous tumor model to be inappropriate as compared to orthotopic model. Local tissue niche modulates the phenotype of tumors generated, transitioning through intermediate states towards a final commitment to either an epithelial or mesenchymal state that recapitulates the reported class switching in patient tumors. During disease progression, such heterogeneity of tumor cells at different niche(s) and their cross-talks with discrete TME at each site, will pose challenges for effective therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20023e2da4c4317c3adb1e410526c772109c27da" target='_blank'>
              ID8 cells manifest phenotypic plasticity and molecular heterogeneity of high-grade serous ovarian cancer in response to the local tissue niche
              </a>
            </td>
          <td>
            Ritika Gupta, S. Shravan, Swapna Balaji Agwane, R. M. Bankar, S. Bapat
          </td>
          <td>2025-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has emerged as an advanced biological technology capable of resolving the complexity of cancer landscapes at single-cell resolution. Spatial transcriptomics(ST), as an innovative complementary approach, effectively compensates for the lack of spatial information inherent in scRNA-seq data. This review explores the rapidly evolving integration of scRNA-seq and ST and their transformative role in deciphering the tumor microenvironment (TME). We highlight how these technologies jointly uncover cellular heterogeneity, stromal-immune interactions, and spatial niches driving tumor progression and therapy resistance. Moving beyond previous reviews, we emphasize emerging computational strategies for data integration—including deconvolution and mapping approaches—and evaluate their applications in characterizing immune evasion, fibroblast diversity, and cell-cell communication networks. Ultimately, this review provides a forward-looking perspective on how spatial multi-omics are poised to advance precision oncology through spatially-informed biomarkers and diagnostic tools. We conclude that the full clinical potential of these technologies relies on closing the gap between analytical innovation and robust clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84801b1d17cdb40a9473ba25f6bac709b5c4c887" target='_blank'>
              Single-cell and spatial transcriptomics integration: new frontiers in tumor microenvironment and cellular communication
              </a>
            </td>
          <td>
            Wenxin Shi, Zhiqiang Zhang, Xiaotong Xu, Yanpeng Tian, Li Feng, Xianghua Huang, Yanfang Du, Zhongkang Li
          </td>
          <td>2025-10-02</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Quantitative investigation of how the spatial organization of cells within the tumor microenvironment associates with disease progression, patient outcomes, and that cell’s phenotypic state remains a key challenge in cancer biology. High-dimensional multiplexed imaging offers an opportunity to explore these relationships at single-cell resolution. Methods We developed a computational pipeline to quantify and analyze the neighborhood profiles of individual cells in multiplexed immunofluorescence images. The pipeline characterizes spatial co-localization patterns within the tumor microenvironment and applies interpretable supervised machine learning models, specifically orthogonal partial least squares analysis (OPLS), to identify spatial relationships predictive of cell states and clinical phenotypes. Results We applied this framework to a previously published non-small cell lung cancer (NSCLC) cohort across four applications. At the cellular level, we identified neighborhood features associated with lymphocyte activation states. At the tumor-immune interface, we demonstrated that the immune cell composition surrounding major histocompatibility complex class I-expressing (MHC I+) tumor cells could distinguish adenocarcinoma from squamous cell carcinoma. At the patient level, spatial features predicted tumor grade. Discussion By integrating cell-segmented imaging data with interpretable modeling, our pipeline reveals key spatial determinants of tumor biology. These findings generate testable mechanistic hypotheses about intercellular interactions and support the development of spatially informed prognostic and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a735226da4130f5ff72da463b9cc63eeddebebdb" target='_blank'>
              A flexible systems analysis pipeline for elucidating spatial relationships in the tumor microenvironment linked with cellular phenotypes and patient-level features
              </a>
            </td>
          <td>
            Gabriel F. Hanson, Kate A. Goundry, Remziye E. Wessel, Michael G Brown, Timothy N. J. Bullock, Sepideh Dolatshahi
          </td>
          <td>2025-09-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Risk stratification remains a critical challenge in non-small cell lung cancer patients for optimal therapy selection. In this study, we develop an artificial intelligence-powered spatial cellomics approach that combines histology, multiplex immunofluorescence imaging and multimodal machine learning to characterize the complex cellular relationships of 43 cell phenotypes in the tumor microenvironment in a real-world retrospective cohort of 1168 non-small cell lung cancer patients from two large German cancer centers. The model identifies cell niches associated with survival and achieves a 14% and 47% improvement in risk stratification in the two main non-small cell lung cancer subtypes, lung adenocarcinoma and squamous cell carcinoma, respectively, combining niche patterns with conventional cancer staging. Our results show that complex immune cell niche patterns identify potentially undertreated high-risk patients qualifying for adjuvant therapy. Our approach highlights the potential of artificial intelligence powered multiplex imaging analyses to better understand the contribution of the tumor microenvironment to cancer progression and to improve risk stratification and treatment selection in non-small cell lung cancer. Risk stratification in non-small cell lung cancer (NSCLC) remains challenging. By combining multiplex immunofluorescence, H&E histology, and AI, the study identifies spatial “cell-niche” patterns that enhance survival prediction beyond UICC8 staging. These patterns reclassify many stage I patients as high risk, revealing potentially undertreated cases and establishing spatial tumor microenvironment features as clinically actionable biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53bd1a5c6021c6b38d3074a4cf442e836ada177" target='_blank'>
              AI-powered spatial cell phenomics enhances risk stratification in non-small cell lung cancer
              </a>
            </td>
          <td>
            Simon Schallenberg, Gabriel Dernbach, Sharon Ruane, Philipp Jurmeister, Cornelius Böhm, Kai Standvoss, Sandip Ghosh, M. Frentsch, M. Dragomir, Philipp Keyl, Corinna Friedrich, Il-Kang Na, Sabine Merkelbach-Bruse, A. Quaas, N. Frost, Kyrill Boschung, Winfried Randerath, G. Schlachtenberger, Matthias Heldwein, Ulrich Keilholz, Khosro Hekmat, Jens Rückert, R. Buettner, Angela Vasaturo, David Horst, Lukas Ruff, M. Alber, Klaus-Robert Müller, Frederick Klauschen
          </td>
          <td>2025-11-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive disease with a dense fibrotic stroma and is often accompanied by chronic inflammation. Peritumoral inflammation is typically viewed as a reaction to nearby tumor growth. Here, we report that the inflamed pancreatic lobules are frequently invaded by tumor cells, forming a distinct, non-fibrotic tumor niche. Using a semi-supervised machine learning approach for annotations of clinical samples and multiplex protein profiling, we show that tumor cells at the invasion front are closely associated with acinar cells undergoing damage-induced changes, and with activated fibroblasts expressing markers of injury. The invaded lobules are linked to classical tumor phenotypes, in contrast to fibrotic areas where tumor cells display a more basal profile, highlighting microenvironment-dependent tumor subtype differences. In female mice, lobular invasion similarly aligns with the classical tumor phenotype. Together, our data reveal that pancreatic tumors colonize injured lobules, creating a unique niche that shapes tumor characteristics and contributes to disease biology. Peritumoural pancreatitis and lobular injury are commonly seen in pancreatic cancer. Here, the authors reveal interactions between invading tumour cells and inflammation-related lobular cells, linked to switching from a basal to classical tumor cell phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33fdbbbab057fd0e654aba5735a7c83d83362e84" target='_blank'>
              An injury-associated lobular microniche is associated with the classical tumor cell phenotype in pancreatic cancer
              </a>
            </td>
          <td>
            Sara Söderqvist, Annika Viljamaa, Natalie Geyer, Anna-Lena Keller, Kseniya Ruksha, Carina Strell, Neda Hekmati, Alexandra Niculae, J. Engstrand, E. Sparrelid, Caroline Salmén, Tânia D. F. Costa, Miao Zhao, Staffan Strömblad, Argyro Zacharouli, P. Ghorbani, Sara Harrizi, Yousra Hamidi, Olga Khorosjutina, Stefina Milanova, Bernhard Schmierer, Béla Bozóky, C. Fernández Moro, Marco Gerling
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c74617e8894674076d6153fb81e22ba9cb23b96" target='_blank'>
              Comparison of Human Melanoma Single Cell Profiles to Evolutionary Medicine Model Xiphophorus Provides Insights in Disease Control
              </a>
            </td>
          <td>
            Yanting Xing, Rachel Carrol, X. Maggs, Wes C. Warren, Manfred Schartl, Yuan Lu
          </td>
          <td>2025-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a molecularly diverse form of lung cancer characterized by distinct oncogenic driver mutations that influence both tumour biology and clinical outcomes. Understanding the interplay between these oncogenic drivers and the tumour microenvironment (TME) is crucial for improving therapeutic strategies and patient management. Here, we investigate the impact of driver mutations on the composition and spatial architecture of the TME in LUAD. Using imaging mass cytometry (IMC), we analyse tumour samples from 157 LUAD patients, integrating genomic and clinical data to link specific mutations with tumour characteristics. Unique patterns are associated with mutated KRAS and EGFR tumours with TP53 co-mutations, suggesting these co-mutations reshape the TME and promote resistance to tyrosine kinase inhibitors (TKIs). Overall, our findings highlight the complex interplay between oncogenic driver mutations and the TME in LUAD, underscoring the importance of integrating genomic and cellular data to understand the underlying tumour behaviour and prognosis. Oncogenic driver mutations can influence the tumour biology of lung adenocarcinomas. Here, authors analyse 157 LUAD samples using imaging mass cytometry to reveal how these mutations impact the tumour microenvironment and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05eac26faa3f5fa852d857cbc84d4c9c3228ae87" target='_blank'>
              Oncogenic driver mutations underlie the spatial tumour immune landscape of non-small cell lung cancer
              </a>
            </td>
          <td>
            Saskia Hartner, H. Abolfathi, M. Rezanejad, Bridget Liu, E. Karimi, Dakota Rogers, Mark Sorin, Samuel Doré, Lysanne Desharnais, M. Orain, William Enlow, A. Gagné, Yuhong Wei, Y. Bossé, D. Quail, Philippe Joubert, L. Walsh
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5df09ad47e15df1d3b2fa365bedcbec8d739cdeb" target='_blank'>
              Unveiling Oligodendrocyte-Lineage Differentiation in the Glioblastoma Infiltrative Zone Through Spatial Transcriptomics
              </a>
            </td>
          <td>
            C. M. Argento, Enrico Sebastiani, E. F. Rosatti, D. Sighel, S. Longhi, A. Soldano, T. Tebaldi, A. Quattrone, F. Corsini, Luciano Annicchiarico, M. Barbareschi, L. Pecciarini, P. L. Poliani
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="


 Adult gliomas are a diverse class of tumors with substantial molecular and pathological variation. While prior single-cell omics studies have characterized intratumoral heterogeneity, they often examined tumors in isolation, thus overlooking the recurrence of tumor states across subtypes. Given their shared environment, understanding the conservation of tumor states across gliomas is essential.



 To address this, we analyzed 310 tumor cores from 284 patients spanning oligodendroglioma, low- and high-grade astrocytoma, glioblastoma, and ependymoma, using single-cell spatial transcriptomics with a 344-gene panel (10x Xenium). This approach yielded a dataset of 2.8 million cells, of which 84% were tumor and 16% represented 18 non-malignant cell types from the tumor microenvironment (TME). To further investigate local cellular interactions, we applied an extended spatial transcriptomics panel (5,096 genes) alongside multiplexed immunofluorescence (57 proteins) to a validation cohort of low-grade astrocytomas.



 Our analysis revealed tumor states recurring across gliomas, albeit in varying proportions, organized into neighborhoods that reflect local cell-cell interactions. For instance, proliferating tumor cells tended to co-localize with oligodendro- and neuro-progenitor-like tumor states, suggesting that these programs may give rise to proliferating cells. Spatial neighborhoods were primarily subtype-specific, indicating that tissue organization and not cell composition alone, underlies glioma heterogeneity. We next assessed the relationship between tumor and immune cells, revealing entity-specific immune microenvironments: astrocytoma and ependymoma exhibited distinct immune responses (T cell- versus myeloid-driven inflammation), whereas oligodendroglioma displayed a low inflammatory landscape. Notably, microglia were consistently observed in proximity to tumor cells across all glioma subtypes, highlighting their conserved role in modulating anti-tumor immunity. Methylation profiling stratified these tumors into 13 molecular subtypes, each associated with distinct neighborhoods. Glioblastoma subtypes RTK1, RTK2, and MES, in particular, displayed unique tissue architectures consistent with their subtype-specific methylation profiles.



 Collectively, these findings link microanatomical features to glioma heterogeneity by connecting tumor states to their microenvironments. Regional differences, such as those between the tumor core and infiltrating edge, shape cellular composition with implications to treatment response. By capturing such spatially dependent patterns, our work provides a framework for advancing the understanding of glioma biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34fbb58728bc44ec20e86f909612c57947c86652" target='_blank'>
              KS03.6.A THE RECURRENT TISSUE STATES ACROSS ADULT GLIOMAS AS REVEALED BY SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            A. Mathioudaki, Z. Seferbekova, M. Ritter, D. Calafato, G. Rukhovich, F. Hinz, F. Ippen, E. Popova, N. Koeberer, P. Sant, S. Schinkewitsch, J. Malm, S. Dietrich, C. Herold-Mende, N. Etminan, A. Wick, S. Krieg, M. Platten, A. von Deimling, F. Sahm, A. Suwala, M. Gerstung
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="High-resolution spatial transcriptomics (ST) data provide valuable insights into the molecular dynamics underlying complex biological processes. However, their widespread application remains limited due to high costs and technical challenges. Here, we present PRTS (Pathology-driven Reconstruction of Transcriptomic States), a novel framework that predicts single-cell-resolution ST data directly from histological images. Our results demonstrated that PRTS generated transcriptomic profiles for about 60,000 analyzable cell tiles per tissue section, representing an approximately 27-fold increase in analytical units compared to conventional ST spots and remarkably enhancing spatial resolution. Notably, PRTS achieves accurate cell-level transcriptomic predictions using only hematoxylin-and-eosin-stained tissue images. This method transforms costly ST technologies into a practical and scalable tool, offering a cost-efficient solution for comprehensive ST profiling in hematoxylin-and-eosin-based disease research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdf408653ada92ca0b84be1f0e89d28a32eb782d" target='_blank'>
              PRTS: Predicting Single-Cell Spatial Transcriptomic Maps from Histological Images
              </a>
            </td>
          <td>
            Jingyi Wen, Lingxuan Zou, Jiying Liu, Xi Guo, Changkun Liu, Bensu Wang, Tongtong Deng, Chang Liu, Risheng Tang, Yanbin Yang, Yucheng Huang, Lijia Yang, Hui Wang, Zihao Li, Shengming Lin, Shipping Liu, Yuhu Zhang, Z. Hao, Haiyu Zhou, Han Huang, Fei Ling
          </td>
          <td>2025-10-06</td>
          <td>Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b808c693c9788562c4b889cdeaf7fc68e251b" target='_blank'>
              Spatial transcriptomics of glioblastoma defines biologically and clinically significant reprogramming patterns across unique spatial microenvironments
              </a>
            </td>
          <td>
            V. Ravikumar, A. Maddox, Reva Kulkarni, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving as a pivotal technology in studying tumor biology and associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. We use serial 5 μm sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma samples in tissue microarrays to compare the performance of the ST platforms (CosMx, MERFISH, and Xenium (uni/multi-modal)) in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx, and hematoxylin and eosin staining data. In addition to an objective assessment of automatic cell segmentation and phenotyping, we perform a manual phenotyping evaluation to assess pathologically meaningful comparisons between ST platforms. Here, we show the intricate differences between the ST platforms, reveal the importance of parameters such as probe design in determining the data quality, and suggest reliable workflows for accurate spatial profiling and molecular discovery. Spatial cell distribution within a tissue microenvironment is a rapidly advancing field. Here, authors assess three commercially available single-cell resolution spatial transcriptomics approaches (CosMx, MERFISH, and Xenium) to inform which technology outperforms for immune profiling of solid tumors using patient samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7415db7d075b9d1e62e79a2741843a3cf68a8b2" target='_blank'>
              Comparison of imaging based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Nejla Ozirmak Lermi, Max Molina Ayala, Sharia Hernandez, Wei Lu, Khaja B Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, J. Dou, Qingnan Liang, , Ahmed N. Alrawi, Claudio A. Arrechedera, Kimberly S. Ayers, Caludia A. Bedoya, Elizabeth Burton, Connie A. Chon, Randy Chu, S. Crosby, Jonathan Do, Cibelle FP Lima, Fu Szu-Chin, Andy Futreal, Ana L. Garcia, Celia Garica-Prieto, S. Gite, C. Gumbs, Kristin J. Hargraves, Meng He, Chacha Horombe, H. Ibarguen, Stacy Jackson, Jeena Jacob, Isha Khanduri, Walter K. Kinyua, M. Knafl, Wenhua Lang, L. Little, S. Mallampati, Mary GT Mendoza, F. Meric-Bernstam, M. M. Mohammad, Mario LM Piubelli, Sabitha Prabhakaran, K. R. Shaw, Xiaofei Song, Sandesh Subramanya, Baohua Sun, Shumaila Virani, Wan-Jan Wang, Scott E. Woodman, Mingchu Xu, Jianhua Zhang, Qingxiu C. Zhang, Shanyu Zhang, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchéz-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L. Gibbons, Ignacio I. Wistuba, C. Haymaker, Ken Chen, L. S. Solis Soto
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Tumor thrombus (TT) worsens prognosis and complicates surgery in renal cell carcinoma (RCC), yet its formation mechanisms remain unclear. Here, we perform integrative single-cell and spatial transcriptomic analyses on 71 tissues and 48 sections from RCC patients with or without TT. The cellular and spatial atlas reveals distinct TT-associated tumor microenvironment remodeling characterized by the enrichment of FAP+ fibroblasts. These FAP+ fibroblasts are spatially contiguous to aggressive cancer cells and promote their malignant phenotypes in vitro. Their abundance inversely correlates with functional NK cells, suggesting roles in tumor invasion and immune evasion. Furthermore, single-cell multiomics analysis identifies tumor pericytes as a source of FAP+ fibroblasts and delineates transcription factor dynamics underlying pericyte-fibroblast transition. Finally, high levels of FAP+ fibroblasts are associated with poor prognosis and predict a weaker response to anti-VEGF-based therapy. In conclusion, our study highlights FAP+ fibroblasts as drivers of aggressive RCC with TT, suggesting potential therapeutic targets. Tumour thrombus (TT) complicates renal cell carcinoma (RCC) treatment. Here, authors conduct an integrative single-cell RNA sequencing and spatial transcriptomic analyses on RCC patients with and without TT and identify FAP+ fibroblasts as drivers for TT formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5472524a744e590345b74e85358cb939dc059d29" target='_blank'>
              FAP+ fibroblasts orchestrate tumor microenvironment remodeling in renal cell carcinoma with tumor thrombus
              </a>
            </td>
          <td>
            Jiacheng Ma, Yan Huang, Jie Chen, Yang Li, Rongyan Yao, Xiubin Li, Qiyang Liang, Xinran Chen, Cheng Peng, Kan Liu, Yuanjun Zhai, Xu Zhang, Xin Ma, Xiaowen Wang, Qingbo Huang, Fuchu He
          </td>
          <td>2025-10-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd7292fed094fd219af6750590d28db3dc5a363b" target='_blank'>
              Spatial organization of the tumor-immune microenvironment in ER-positive breast cancer: remodeling during treatment and associations with clinical response
              </a>
            </td>
          <td>
            Maria Aanesland Dahle, Jan-Lukas Førde, E. V. Egeland, A. Creason, Cameron Watson, Ø. Garred, L. Prasmickaite, G. Mælandsmo, Gordon B. Mills, O. Engebraaten, M. H. Haugen
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98126a0a8ef060e1442a8189ca2cc50b5b3d5b2e" target='_blank'>
              Integrated cell atlas and tumoroids chart pancreatic cancer therapeutic targets
              </a>
            </td>
          <td>
            Quan Xu, Ryo Okuda, Bruno Gjeta, Christoph Harmel, Marina Signer, Matilde Lucioli, Małgorzata Santel, M. Seimiya, Cinzia Esposito, Karolina Guja-Jarosz, Ashley Maynard, S. Morinaga, Y. Miyagi, Tomoyuki Yamaguchi, Y. Ueno, Salvatore Piscuoglio, Daniel J. Müller, Hideki Taniguichi, Barbara Treutlein, J. G. Camp
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly heterogeneous cancer with complex tumor–immune interactions. This heterogeneity, particularly in tumor and immune cells, complicates treatment and prognostic evaluation. Although recent studies have revealed distinct tumor cell states and immune dysfunction in HCC, the molecular basis underlying tumor aggressiveness remains poorly understood. A deeper understanding of the molecular and functional diversity of both tumor and immune cell populations, especially the identification of stem-like tumor subpopulations and immunosuppressive mechanisms, along with the development of robust prognostic biomarkers, is essential for advancing precision oncology and improving clinical outcomes. Methods We integrated three publicly available single-cell RNA sequencing (scRNA-seq) datasets from GEO to delineate the cellular architecture of the HCC tumor microenvironment. Unsupervised clustering and dimensionality reduction were employed to identify major cell types and tumor subpopulations. Functional annotation was performed using canonical markers, Monocle, CytoTRACE, and AUCell scoring. H2AFZ was identified as a candidate oncogene and validated through in vitro knockdown experiments. The interaction between T cell subsets and tumor subpopulations were further characterized. A prognostic risk model was constructed using LASSO regression. Results Six major cell types were identified in HCC TME. Tumor cells were subdivided into three distinct clusters: Tumor_C0, Tumor_C1 and Tumor_C2. Tumor_C2 showed the highest stemness, pro-metastatic activity and immunogenic cell death signatures. H2AFZ was highly expressed in Tumor_C2 and associated with poor prognosis. The knockdown of H2AFZ reduced H2A.Z protein levels, inhibited proliferation, invasion, and induced apoptosis. T cell analysis revealed five subpopulations. It was found that Tumor_C2 interacts with the proliferative and exhausted T cell subpopulations, suggesting a potential functional relationship between them. The prognostic model based on tumor_C2 transcriptomic features effectively stratified patient survival across multiple cohorts, with robust AUCs and Kaplan-Meier survival distinctions. Conclusion We identified a proliferative, stem-like tumor cell subpopulation (Tumor_C2) in HCC characterized by high H2AFZ expression, which drives tumor aggressiveness. T cell analysis revealed significant interactions with Tumor_C2. Moreover, a prognostic model based on Tumor_C2 features effectively stratified patient survival across multiple cohorts. Together, these findings highlight potential therapeutic targets for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7024320dd03972a4da66c548f19e56393e35ae99" target='_blank'>
              Single-cell transcriptomics identifies an H2AFZ-driven proliferative tumor subpopulation associated with poor prognosis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jihui Huo, Tao Yang, Kai Lei, Zeyao Wang, Zebin Chen, Qi Zhou
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af62a3f15720a1c9a956b0f31df3cb0beb418ea" target='_blank'>
              Cell migration sculpts evolutionary dynamics favouring therapy resistance in lung cancer
              </a>
            </td>
          <td>
            A. Bhargava, Xiao Fu, Sasha Bailey, Yutaka Naito, Hamid Mohammadi, R. Hynds, S. Hooper, D. Biswas, A. Le Marois, Sunil Kumar, Yuriy Alexandrov, Paul French, James Mcginty, Paul A. Bates, C. Swanton, E. Sahai
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed2e2b6e5867f00f570022e3faf5971a5955554" target='_blank'>
              Highly multiplexed imaging recovers immune and metabolic niches in multiple myeloma associated with disease progression and bone involvement
              </a>
            </td>
          <td>
            Ingrid Aass Roseth, Lukas Hatscher, Chiara Schiller, Håkon Skjalg Selland Johnstuen, T. Slørdahl, Håkon Hov, D. Schapiro, T. Standal
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696e778534089692f9510070ab5b7ccf3c2a11f2" target='_blank'>
              Single-cell spatial mapping reveals dynamic bone marrow microarchitectural alterations and enhances clinical diagnostics in MDS
              </a>
            </td>
          <td>
            Ryan Nachman, Aleksandra Kopacz, Caitlin Unkenholz, Jian Chai, Arvin Ruiz, Itzel Valencia, Jeanne Jiang, F. Socciarelli, Jiwoon Park, Christopher E. Mason, Ling Zhang, D. Sallman, Gail J. Roboz, Pinkal M. Desai, Justin Kaner, Joshua A Fein, Monica L Guzman, Neal Lindeman, Amy Chadburn, Madhu M Ouseph, Paul Simonson, Julia Geyer, Giorgio Inghirami, Shahin Rafii, David Redmond, Sanjay Patel
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fa5a90466cca365ac23cb5111624676a9f4bea9" target='_blank'>
              Deciphering Cellular Ecosystems Driving Tumor Progression and Immune Escape from Spatial Transcriptomics and Single-Cell with COMPOTES
              </a>
            </td>
          <td>
            L. Herpin, Anaïs Chossegros, Roberta Codato, Josep Monserrat Sanchez, Jean El Khoury, Simon Grouard, V. Ducret, Alex Cornish, Baptiste Gross, Elodie Pronier, C. Hoffmann, Alberto Romagnoni, Eric Y. Durand, Almudena Espin Perez, Quentin Bayard
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64783d2fd1c1cb64aa22b77e49df40179825bbfc" target='_blank'>
              Inferring cellular communication through mapping cells in space using Tangram 2
              </a>
            </td>
          <td>
            Hejin Huang, Alma Andersson, Sunny Z. Wu, Gabriele Scalia, K. H. Hoi, Jenna Collier, Graham Heimberg, Hector Corrada-Bravo, Shreya Gaddam, Jan-Christian Huetter, Sören Müller, Shannon J. Turley, Dave Richmond, Aicha Bentaieb, Tommaso Biancalani
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Meningiomas represent the most common primary intracranial tumors, accounting for approximately one-third of adult CNS tumors. Despite their frequency, model systems that accurately replicate the complex tumor-microenvironment interactions across meningioma grades remain underdeveloped, limiting translational progress. This study aimed to establish and characterize human meningioma tumor models to interrogate grade-specific cellular behaviors and microenvironmental dynamics



 Patient-derived meningioma cell lines were established through mechanical and enzymatic dissociation of surgically resected tumors, followed by expansion in specialized culture media. GFP transfection was performed to enable live-cell tracking. For ex vivo modeling, human neocortical slices — obtained from access tissue from non-meningioma surgeries, either 2 cm distal to the glioma core or from epilepsy surgeries under neuronavigation guidance — were injected with GFP-labeled meningioma cells. Tumor progression, cluster formation, and tissue integration were assessed over six days using longitudinal live imaging and immunofluorescence staining. Single-cell RNA sequencing (scRNA-seq) was employed to resolve cellular heterogeneity and tumor-microenvironmental interactions. Machine learning-based image segmentation combined with manual curation enabled precise quantification of tumor cell distribution and interaction patterns within the host tissue.



 Grade 3 meningioma cells demonstrated significantly enhanced cluster formation, tissue invasion, and microenvironmental remodeling compared to lower-grade counterparts. Ex vivo models recapitulated patient tumor behaviors, with scRNA-seq revealing upregulation of genes associated with mesenchymal transition, extracellular matrix remodeling, and immune modulation in Grade 3 tumors. Microenvironmental analysis indicated that higher-grade tumors established more extensive cell-cell interaction networks and induced greater astrocytic and microglial activation relative to Grade 1 and 2 models. Furthermore, immune profiling revealed a more immunosuppressive environment, with myeloid-derived suppressor cells, reducing immune surveillance and facilitating tumor evasion. Additionally, there was increased activation of astrocytes and microglia, contributing to a supportive tumor microenvironment.



 Our patient-derived meningioma models reveal grade-specific differences in tumor behavior and TME interactions. Grade 3 meningiomas show aggressive clustering and microenvironmental remodeling, offering insights into malignant progression. These models provide a high-fidelity platform for studying tumor biology and preclinical evaluation of therapies targeting tumor-stroma interactions
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b180fe0ce6bc7373d74452191faf8369d3bd9e2c" target='_blank'>
              P13.17.B HUMAN MENINGIOMA TUMOR MODELS REVEAL GRADE-SPECIFIC MICROENVIRONMENTAL INTERACTIONS
              </a>
            </td>
          <td>
            A. Mohammed, K. Joseph, D. Depro, G. Nishita, V. Jariwala, A. Boecke, M. Al Shhab, I. Vasilikos, J. M. Nakagawa, T. Juratli, R. Roelz, M. Shah, J. Beck, J. Grauvogel, V. Ravi
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is highly heterogeneous, with five-year survival rates dropping from $\sim$90% in localized disease to $\sim$15% with distant metastases. Disease progression is shaped not only by tumor-intrinsic alterations but also by the reorganization of the tumor microenvironment (TME). Metabolic, compositional, and spatial changes contribute to this progression, but considered individually they lack context and often fail as therapeutic targets. Understanding their coordination could reveal processes to alter the disease course. Here, we combined multiplexed ion beam imaging (MIBI) with machine learning to profile metabolic, functional and spatial states of 522 colorectal lesions with single-cell resolution. We observed recurrent stage-specific remodeling marked by a lymphoid-to-myeloid shift, stromal-cancer cooperation, and malignant metabolic shifts. Spatial organization of epithelial, stromal, and immune compartments provided stronger stratification of disease stage than tumor-intrinsic changes or bulk immune infiltration alone. To systematically model these coordinated changes, we condensed multimodal features into 10 latent factors of TME organization. These factors tracked disease progression, were conserved across cohorts, and revealed frequent multicellular metabolic niches and distinct, non-exclusive TME trajectories. Our framework MuVIcell exposes the elements that together drive CRC progression by grouping co-occurring changes across cell types and feature classes into coordinated multicellular programs. This creates a rational basis to therapeutically target TME reorganization. Importantly, the framework is scalable and flexible, offering a resource for studying multicellular organization in other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884e760c0de1e86aa10844e17508931cd572069b" target='_blank'>
              Robust multicellular programs dissect the complex tumor microenvironment and track disease progression in colorectal adenocarcinomas
              </a>
            </td>
          <td>
            Loan Vulliard, Teresa Glauner, Sven Truxa, Miray Cetin, Yu-Le Wu, R. Simon, Laura Behm, Jovan Tanevski, Julio Saez-Rodriguez, G. Sauter, Felix J. Hartmann
          </td>
          <td>2025-10-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mapping nanoscale neuronal morphology with molecular annotations is critical for understanding healthy and dysfunctional brain circuits. Current methods are constrained by image segmentation errors and by sample defects (e.g., signal gaps, section loss). Genetic strategies promise to overcome these challenges by using easily distinguishable cell identity labels. However, multicolor approaches are spectrally limited in diversity, whereas nucleic acid barcoding lacks a cellfilling morphology signal for segmentation. Here, we introduce PRISM (Protein-barcode Reconstruction via Iterative Staining with Molecular annotations), a platform that integrates combinatorial delivery of antigenically distinct, cell-filling proteins with tissue expansion, multi-cycle imaging, barcode-augmented reconstruction, and molecular annotation. Protein barcodes increase label diversity by >750-fold over multicolor labeling and enable morphology reconstruction with intrinsic error correction. We acquired a ∼10 million µm3 volume of mouse hippocampal area CA2/3, multiplexed across 23 barcode antigen and synaptic marker channels. By combining barcodes with shape information, we achieve an 8x increase in automatic tracing accuracy of genetically labelled neurons. We demonstrate PRISM supports automatic proofreading across micron-scale spatial gaps and reconnects neurites across discontinuities spanning hundreds of microns. Using PRISM’s molecular annotation capability, we map the distribution of synapses onto traced neural morphology, characterizing challenging synaptic structures such as thorny excrescences (TEs), and discovering a size correlation among spatially proximal TEs on the same dendrite. PRISM thus supports selfcorrecting neuron reconstruction with molecular context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ecd3998e1600e3ef97426dea4d59729398135b" target='_blank'>
              Combinatorial protein barcodes enable self-correcting neuron tracing with nanoscale molecular context
              </a>
            </td>
          <td>
            Sung Yun Park, Arlo Sheridan, Bobae An, Erin Jarvis, Julia Lyudchik, William Patton, Jun Y. Axup, Stephanie W. Chan, Hugo G.J. Damstra, Daniel Leible, Kylie S Leung, Clarence A. Magno, Aashir Meeran, Julia M. Michalska, Franz Rieger, Claire Wang, Michelle Wu, George M. Church, Jan Funke, Todd Huffman, Kathleen Leeper, Sven Truckenbrodt, Johan Winnubst, J. Kornfeld, Edward S. Boyden, Samuel G. Rodriques, Andrew C. Payne
          </td>
          <td>2025-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Stem-like gastric cancer (GC) is an aggressive molecular subtype marked by poor prognosis and limited response to immune checkpoint blockade (ICB). The spatial mechanisms driving this resistance remain unclear. We conducted spatially resolved single-cell transcriptomic profiling of diffuse-type GC tissues to uncover the spatial architecture and functional diversity of tumor and stromal populations. Cellular heterogeneity and region-specific signaling pathways were characterized using integrative bioinformatics analyses. We identified transcriptionally diverse, high-entropy cell populations predominantly localized in the deep tumor regions. These included unique endothelial and fibroblast subsets enriched for pro-tumorigenic and immune-regulatory signaling. A notable finding was the engagement of deep-region endothelial cells in VISFATIN (extracellular NAMPT) signaling through the ITGA5–ITGB1 integrin axis, associated with immune evasion and poor prognosis. This endothelial signaling program is distinct from and functionally independent of cancer-associated fibroblast (CAF)-mediated pathways. Elevated expression of the NAMPT–ITGA5–ITGB1 axis was observed in ICB non-responders and correlated with reduced overall survival. Our study delineates spatially defined cellular programs that contribute to immune escape in stem-like GC, highlighting a novel VISFATIN–integrin signaling axis as a potential biomarker and therapeutic target in immunotherapy-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dd6bcdbe10e93d5e57b978f2adeff6ec26c9091" target='_blank'>
              Spatially resolved endothelial signaling via nampt-itga5 drives immune evasion in stem-like gastric cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jae-Ho Cheong, Eui Tae Kim
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df773895d5d3bb014e2dd4dc6276d432f2994491" target='_blank'>
              YAP1 defines an emergent, plastic population of relapsed small cell lung cancer
              </a>
            </td>
          <td>
            C. Stewart, K. Ramkumar, Runsheng Wang, Yuanxin Xi, A. Halliday, L. Diao, Qi Wang, Alejandra G Serrano, Sarah M. Groves, Simon Heeke, Azusa Tanimoto, L. Kaiser, Whitney E Lewis, M. Bose, Pedro Da Rocha, L. Karacosta, Vito Quaranta, Jing Wang, Julie George, Luisa Maren, Solis Soto, Bingnan Zhang, J. Heymach, L. Byers, C. Gay
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c97ce3b2187f6d94b5a5b261acd4c567058ac991" target='_blank'>
              circVDJ-seq for T cell clonotype detection in single-cell and spatial multi-omics
              </a>
            </td>
          <td>
            I. Plumbom, Benedikt Obermayer, Raphael Raspe, A. Pascual-Reguant, Ilan Theurillat, T. Pentimalli, Yu-Hsin Hsieh, Marine Gil, Carola Dietrich, Michaela Seeger-Zografakis, C. Quedenau, Jeannine Wilde, C. Braeuning, Cornelius Fischer, Markus Schuelke, Leif S. Ludwig, Angelika Eggert, N. Rajewsky, T. Borodina, D. Beule, Janine Altmueller, Helena Radbruch, Anja Hauser, Thomas Conrad
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3145b8048f1a33f4e0cf44307241622724c18a05" target='_blank'>
              Mapping CD4+ T Cell Landscapes in Glioblastoma Reveals Effectors and Bystanders
              </a>
            </td>
          <td>
            Cameron M. Hill, Anthony Z. Wang, Jose A. Maldonado, T. Johanns, Allegra A. Petti, Gavin P. Dunn
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03cc2aa2a287dcfe2c1cd1643687b026d41744b" target='_blank'>
              Single-cell and Spatial Omics Reveals Region-Specific Plasticity and Therapeutic Vulnerabilities in Metastatic High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Lai Man, Natalie Wu, Janet L. Oblinger, Dazhuan E Xin, Rohit Rao, Feng Zhang, Mike Adam, Oscar Lopez-Nunez, Sara Szabo, D. Allmen, Long-Sheng Chang, Brian D Weiss, Q. R. Lu
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Owing to the complexity of TME components and the heterogeneity of cancer cells, the relationship between the niches of TME and prognosis in breast ductal carcinoma remains unknown. The staged characteristics of corresponding cancer cell behaviors are unclear. Our study aims to reveal spatial structures and specific cellular information of TME and cancer cells subgroups during the progression from DCIS to IDC and lymph node metastasis. Single-cell sequencing, spatial transcriptomics, bulk RNA sequencing datasets were used to explore the changes in microenvironmental components and transcriptional programs of tumor cells during the progression of breast ductal carcinoma. Immunohistochemistry, multiplex immunofluorescence, flow cytometry cell cycle detection, invasion migration experiments, and WB imprinting were employed for validation. Analysis of TME cell type subsets revealed the accumulation of TEX, iTreg, and stress-phenotype TAM in the mammary gland in situ during the invasion process. Lymphatic metastases exhibited enrichment of nTregs and a more naïve-like CD8 T cell population. Spatial analysis and survival analysis showed that the spatial niches of CD4 TN and phagocytic-phenotype macrophages were associated with a favorable prognosis, and these niches were lost during disease progression. The proliferative subpopulation of breast ductal carcinoma was enriched in lymphatic metastatic tissues, expressing high levels of FAM111B and exhibiting intense TCA and oxidative phosphorylation metabolism. Silencing FAM111B led to cell cycle arrest, decreased invasion and migration abilities, and downregulation of core mediator genes for cuproptosis and disulfidptosis. The stage-specific microenvironmental characteristics of breast ductal carcinoma correspond to some extent to the behavior of tumor cells. During the progression of ductal carcinoma in breast tissue, the establishment of an immunosuppressive microenvironment occurs. The microenvironmental spectrum at lymph node metastases differs somewhat, corresponding to a more enriched turnover of cancer cell proliferation and death. Inhibitors of FAM111B and inducers of cuproptosis and disulfidptosis may serve as potential therapeutic targets for proliferative subgroups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfce9a8ff47f521350ed382190ba1004f756bb5c" target='_blank'>
              Spatiotemporal microenvironment landscape and malignant epithelial pattern transition in breast ductal carcinoma progression
              </a>
            </td>
          <td>
            Xifu Cheng, Wenjuan Zeng, Bingzhe Yin, Jiawei Gui, Hengbin Zhang, Zhenxing Lv, Simin Zhang, Yao Zhou
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/637f216aefe87456deedf3795555bf830c51467e" target='_blank'>
              A transcriptional map of human tonsil architecture: beyond the sum of (single cell) parts
              </a>
            </td>
          <td>
            Helena L. Crowell, L. Llaó-Cid, G. Frigola, Samuel Gunz, Irene Ruano, Patricia Lorden, Max Ruiz, Marta Kulis, J. Martín-Subero, Holger Heyn, E. Campo, A. Pascual-Reguant
          </td>
          <td>2025-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 Human pancreatic ductal adenocarcinomas (PDAC) are composed of sparse, heterogeneous tumour cell populations embedded within a dense, desmoplastic stroma and are highly resistant to existing therapies. Guided by underlying genomic aberrations, cancer cell states and phenotypes interact with their microenvironment to shape disease trajectory and therapeutic response. To quantify the cellular phenotypic heterogeneity of these complex tumour-microenvironment interactions we designed custom multiplexed histopathology imaging panels against single cell RNA sequencing-defined tumour, immune, and stromal cell compartments. Using imaging mass cytometry (IMC), we profiled the co-localization and organization of 83 cell types and their functional states in 221 resected pancreatic tumours. This identified extensive inter- and intra-patient heterogeneity including the presence of both classical and basal cell types in most patients and a gradient of pancreatic epithelial identity confirmed by scRNAseq. This single cell content defined an expanded spectrum of classifiable intermediate tumour subtypes between classical and basal which have specific associations to neighboring immune and stroma cellular content, ploidy, established broad transcriptional subtypes, heterogeneity, and patient outcome. Eight reproducible cancer microenvironments were quantified, empowering cross-tumour comparisons. Matched whole genome sequencing (WGS) identified aberrations associated with specific tumour phenotypes and microenvironments. These results support asynchronous tumour and microenvironment genotype-phenotype axes which subdivide basal to classical tumour polarization and categorize co-existing microenvironments from stiff, ECM-rich, immune suppressed to immune infiltrated regions alongside fibrovascularized CD105+ stroma. This is based upon spatially heterogeneous co-occurring microenvironment niches and correlated tumour phenotypes which are associated with KRAS or MYC amplification and CDKN2A or LATS2 deletion on one axis, mutation of epigenetic modifiers in intermediate and hybrid subtypes, and RNF43 deletion on the other axis. To deeply profile the signalling pathways and enriched biological processes characteristic of each tissue state, we performed deep proteomics measurements of each microenvironment and tumor phenotype through whole-slide IMC informed laser capture microdissection and mass spectrometry of 273 micro-regions from a subset of patients. Finally, to identify a robust, minimized set of outcome-relevant features, we used a multi-modal technology optimized machine learning model trained to predict overall survival. We compare the predictive potential of WGS and spatial proteomics and show that a combination of genomic and cellular content outperforms clinical features and platform specific models thereby demonstrating the synergistic benefit of integrated multi-modal data. Our findings classify the spatial organization of human PDAC to identify an intertwined tumour/microenvironment axis that is influenced by specific genomic aberrations.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Beth Sunnucks, Jennifer Gorman, Chengxin Yu, Michael Geuenich, Sheng Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra Wong, Brendon Seale, Zhen Yuan Lin, Edward Chen, Golnaz Abazari, Miralem Mrkjonic, Julie Wilson, Kieran Campbell, Anne-Claude Gingras, Rob Grant, Grainne O'Kane, Faiyez Notta, Steve Gallinger, Hartland Jackson. Spatial proteomics extend tumour subtype and microenvironment classifications across genotype-phenotype axes of pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B128.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61e2ac6f788886b4d295a71549b94c0503bd2fb" target='_blank'>
              Abstract B128: Spatial proteomics extend tumour subtype and microenvironment classifications across genotype-phenotype axes of pancreatic cancer
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Beth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng Ben Liang, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Z. Lin, Edward L. Y. Chen, Golnaz Abazari, Miralem Mrkjonic, Julie M. Wilson, Kieran R. Campbell, A. Gingras, Robert C. Grant, G. O’Kane, Faiyez Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Background Metastatic renal cell carcinoma (mRCC) still lacks tissue-based biomarkers that predict benefit from first-line immune checkpoint inhibitor (ICI) regimens. High-resolution spatial transcriptomics can dissect the tumor microenvironment (TME) and reveal prognostic niches invisible to bulk assays, offering a path toward precision immunotherapy. Methods Formalin-fixed, paraffin-embedded samples from 12 therapy-naïve patients with mRCC treated with ICI-based combinations were profiled with 10x Genomics Visium and NanoString COSMx. Six patients contributed matched metastatic lesions. Cell-type deconvolution used robust cell type decomposition with a public single-cell RNA-seq reference. Clinical outcomes (mean follow-up 16.75 months) were correlated with spatial features. Validation employed Phase 2/3 JAVELIN Renal 101 as well as Checkmate 010/025 transcriptomic data. Results Spatial transcriptomics uncovered pronounced divergences between primary tumors and metastases in cellular composition and microarchitectural organization. Canonical RCC molecular subtypes displayed marked intrapatient and interpatient spatial heterogeneity, underscoring limitations of bulk classifications. Five recurrent immune niches emerged across samples. One niche—rich in macrophages and CD8+ T cells and defined by chemokine signaling—was closely associated with objective response in metastatic lesions yet absent or infrequent in matched primaries. A gene signature reflecting this niche stratified responders within the immuno-oncology (IO) -treated arms of JAVELIN 101 and CheckMate 010/025 trials. On the other hand, signatures derived from angiogenic tumor cell-rich niches defined responders within the comparator arms (tyrosine kinase inhibitor/everolimus) in these trials. Conclusion Metastasis-specific immune niches, rather than primary-tumor characteristics, appear decisive for ICI efficacy in mRCC. Spatial transcriptomic profiling can identify these clinically relevant microenvironments and generate transferable gene signatures, supporting biomarker-driven patient selection and trial design. Integrating spatial TME analysis into future studies may accelerate personalized immunotherapy strategies for mRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bedf8d3b886332325de79302aa00547b1e48f37" target='_blank'>
              Spatial transcriptomic profiling of metastatic renal cell carcinoma identifies chemokine-driven macrophage and CD8+ T-cell interactions predictive of immunotherapy response
              </a>
            </td>
          <td>
            Charis Kalogirou, M. Krebs, Andreas S Kunz, Oliver Hahn, Hubert Kübler, Marcel Schwinger, Arndt Hartmann, Astrid Schmieder, Markus Eckstein
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d4f1e5ea74f6620f6b4fbf1988331e79868e36" target='_blank'>
              Interpretable Deep Learning Reveals Biologically Relevant Spatial Gene Expression Patterns in Lung Tumors and their Microenvironment
              </a>
            </td>
          <td>
            Vibha R. Rao, Adrienne A Workman, Liang Lu, Xiaoying Liu, Shrey S. Sukhadia
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c81035972b65c627d1b19e7302e769d3ddd0b086" target='_blank'>
              A Single-Cell Atlas of the Breast Cancer Microenvironment Identifies Subtype-Specific Tumor-Immune Landscapes and Vulnerabilities
              </a>
            </td>
          <td>
            Liron Zisman Schachter, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35fce186f7fddf3cfc3ee6a30b68015192ce168e" target='_blank'>
              Exploiting TGF-β-mediated Stromal Programming in Homologous Recombination-Deficient Pancreatic Cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M Mummey, E. Zimmer, Dharini Srinivasan, A. Härle, Loic Moubri, Alica K Beutel, Rima Singh, M. Ekizce, M. Melzer, Yun Lee, Austin Silva, Luisa Härle, T. Engleitner, F. Arnold, M. Morawe, Benjamin K. Naggay, Juliane Schneider, Leonard Gilberg, Julia P. Mosler, Christina Ludwig, Chen Meng, M. Hirschenberger, V. Hunszinger, K. Kluck, Martina Kirchner, A. Volckmar, Mareike Wirth, M. S. Alhamdani, Jörg D. Hoheisel, J. Löhr, Thomas Seufferlein, Alireza Abaei, Ralf Kemkemer, Roland Rad, J. Budczies, M. Mulaw, Patrick C. Hermann, Mark Hänle, K. Sparrer, Christopher J. Halbrook, K. J. Gaulton, K. Steinestel, A. Stenzinger, Nelson Dusetti, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59ee0ed38702dcef1f68ff3c8ff2d5ef5e4fc717" target='_blank'>
              Longitudinal Imaging of the Premalignant Tumor Microenvironment Reveals Transient Myeloid States Predictive of Tumor Fate
              </a>
            </td>
          <td>
            Thomas D. Madsen, Desu Chen, Maria Hernandez, Sarah M. Hammoudeh, Marco Heydecker, Emily Chen, Salma Abu-Elnaj, Noemi Kedei, Weiye Wang, Roberto Weigert
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe40c75d4c6f7843c947dbae95bbc2601c4d486" target='_blank'>
              Multi-modal spatial characterization of tumor immune microenvironments identifies targetable inflammatory niches in diffuse large B cell lymphoma.
              </a>
            </td>
          <td>
            Yibo Dai, A. Kizhakeyil, Dai Chihara, Xubin Li, Yunhe Liu, T. P. Sainz Zuniga, A. Wilson, Jared Henderson, Daniil Vibe, Arman Petrosyants, Connor Jacobson, Alexander Sarachakov, K. Nomie, K. Kryukov, Aleksander Bagaev, Ayushi Chauhan, J. Westin, Christopher R. Flowers, F. Vega, Linghua Wang, Michael R Green
          </td>
          <td>2025-10-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) incidence is rising, and treatment remains challenging unless surgery is curative. Tumour heterogeneity contributes to resistance against both chemotherapy and immune checkpoint inhibitors, underscoring the need to better understand the complex tumour microenvironment (TME). While tumour models derived from cancer tissue from patients have advanced cancer research, they often fail to capture functional RCC heterogeneity and key TME components. We developed a 3D model system with a high success rate from resected tumour, retaining cancer, stromal, and immune cell populations. This system is fully compatible with advanced imaging technologies, including mass spectrometry imaging (MSI) and live-cell multiplex imaging. By integrating static spatial analysis with dynamic live-cell visualisation, our system provides unique insights into tumour heterogeneity, microenvironment metabolic crosstalk, and real-time cellular responses. Phenotypic characterization of the tumoroids showed strong histological resemblance to the original resected tissue, indicating that the tumoroids are reflective of the tumour in vivo and suitable as a representative model system. Additionally, DESI-MSI revealed distinct lipidomic profiles within patient-derived ccRCC tumoroids, capturing spatial metabolic heterogeneity reflective of the primary tissue. Lipid signatures varied across tumour regions, with phospholipid subclasses distinguishing epithelial, endothelial, and highly proliferative cell populations. Notably, non-clear cell regions exhibited reduced lipid droplet and fatty acid content, aligning with aggressive tumour phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc004e66880e4588257a07a21abe175bda8bb5d0" target='_blank'>
              Kidney tumoroid characterisation by spatial mass spectrometry with same-section multiplex immunofluorescence uncovers tumour microenvironment lipid signatures associated with aggressive tumour phenotypes
              </a>
            </td>
          <td>
            Hazem Abdullah, Greice M Zickuhr, I. Um, Alexander Laird, Peter Mullen, David J. Harrison, A. Dickson
          </td>
          <td>2025-09-16</td>
          <td>npj Imaging</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Understanding tumor heterogeneity at the resolution of individualized cell–cell communication networks (CCCNs) remains a major computational challenge in precision oncology. Existing inference methods largely rely on population-level correlation and thus fail to capture patient-specific signaling patterns across diverse cell types. To address this limitation, we developed an integrative computational framework combining the nested hierarchical Dirichlet process (nHDP) model for identifying hierarchically structured gene expression modules, with instance-specific Greedy Fast Causal Inference (iGFCI) for inferring individualized CCCNs (iCCCNs) in colorectal cancer (CRC). Applied to large-scale single-cell RNA-seq data from over 625,000 cells, our model successfully decomposed complex gene expression modules GEMs, potentially representing the cell lineage and cellular signaling states, and uncovered iCCCNs across detailed cell subtypes. We further used TCGA bulk RNA-seq data with survival data to validate the clinical relevance of these individualized gene expression module causal interactions, demonstrating their potential as robust prognostic signatures in CRC. Finally, we used principled causal inference methods to search for ligand-receptor pairs that mediate cell-cell communication. This framework enables mechanistic insights into immune evasion. Our computational method represents a significant advance toward realizing personalized oncology, enabling precise patient stratification and identification of actionable biomarkers for improved therapeutic targeting in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4e906c5fe00a93c967af7c5aabe8332c82ec34" target='_blank'>
              Inferring Personalized Cell-Cell Communication Networks in Colorectal Cancer with Individualized Causal Discovery
              </a>
            </td>
          <td>
            Aodong Qiu, Binfeng Lu, Gregory F. Cooper, Xinghua Lu, Lujia Chen
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) shows variable responses to immunotherapy, highlighting the need for biomarkers to guide patient selection. We applied a spatial multi-omics approach to 234 advanced NSCLC patients treated with programmed death 1-based immunotherapy across three cohorts to identify biomarkers associated with outcome. Spatial proteomics (n = 67) and spatial compartment-based transcriptomics (n = 131) enabled profiling of the tumor immune microenvironment (TIME). Using spatial proteomics, we identified a resistance cell-type signature including proliferating tumor cells, granulocytes, vessels (hazard ratio (HR) = 3.8, P = 0.004) and a response signature, including M1/M2 macrophages and CD4 T cells (HR = 0.4, P = 0.019). We then generated a cell-to-gene resistance signature using spatial transcriptomics, which was predictive of poor outcomes (HR = 5.3, 2.2, 1.7 across Yale, University of Queensland and University of Athens cohorts), while a cell-to-gene response signature predicted favorable outcomes (HR = 0.22, 0.38 and 0.56, respectively). This framework enables robust TIME modeling and identifies biomarkers to support precision immunotherapy in NSCLC. This study uses spatial proteomic and spatial compartment-based whole-transcriptome profiling to develop predictive models for immunotherapy outcomes in non-small cell lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49d1cf1710aaa6c53c5b0d83d1e4cd42842e1d74" target='_blank'>
              Spatial signatures for predicting immunotherapy outcomes using multi-omics in non-small cell lung cancer
              </a>
            </td>
          <td>
            T. Aung, J. Monkman, J. Warrell, I. Vathiotis, Katherine M Bates, N. Gavrielatou, Ioannis P Trontzas, Chin Wee Tan, Aileen I. Fernandez, M. Moutafi, Ken O' Byrne, K. Schalper, Konstantinos N Syrigos, Roy S Herbst, A. Kulasinghe, D. Rimm
          </td>
          <td>2025-10-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="With the advancement of novel technologies such as whole-genome sequencing, single-cell sequencing, and spatial transcriptomics, single-omics analyses have already promoted the research of tumorigenesis as well as development and have partly elucidated the evolutionary processes of lung cancer. However, it is still difficult to distinguish these confounding features via single dimensional approaches due to the complexity, heterogeneity and cell-cell interactions with the immune microenvironment in lung cancer. Multi-omics approaches provide a holistic framework for constructing detailed tumor ecosystem landscapes, thereby facilitating the development of a more robust classification system for precision diagnosis and treatment, and aiding in the discovery of novel cancer biomarkers. In this review, we summarize the potential and applications of multi-omics approaches in characterizing intratumor heterogeneity and the tumor microenvironment throughout the course of lung cancer development. By further discussing the discovery and application of diagnostic and therapeutic biomarkers across precancerous lesions, early-stage lung cancer, tumor progression, metastasis, and therapy resistance, we outline the current challenges and future prospects of using multi-omics to identify reliable biomarkers. Moreover, we emphasize that integrative multi-omics models hold great promise for elucidating the complex interactions within the lung cancer ecosystem, thereby contributing to improved diagnostic accuracy, optimized therapeutic strategies, and better patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb5432eb94e92ff0ae0b03efad123a651887abd5" target='_blank'>
              Harnessing multi-omics approaches to decipher tumor evolution and improve diagnosis and therapy in lung cancer
              </a>
            </td>
          <td>
            Yicong Cheng, Ling Bai, Jiuwei Cui
          </td>
          <td>2025-11-05</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7bbdd81d510f831417ecf76eb25316f13e6cad2" target='_blank'>
              SpaPheno: Linking Spatial Transcriptomics to Clinical Phenotypes with Interpretable Machine Learning
              </a>
            </td>
          <td>
            Bin Duan, Xiaojie Cheng, Hua Zhou
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Tumor-infiltrating lymphocyte (TIL) therapy has demonstrated efficacy in melanoma and is emerging as a promising modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors like glioblastoma is limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas to identify molecular and spatial features associated with successful TIL expansion.



 We conducted comprehensive profiling of 18 TIL products using spectral flow cytometry and T Cell Receptor sequencing (TCRseq). Additionally, 4 tumors yielding ≥10⁸ viable TILs per 0.5 g tumor within two weeks of IL2 Rapid Expansion Protocol cultures, classified as TIL⁺, were analyzed and compared to 4 non-yielding tumors, classified as TIL⁻. Expanded TILs were subjected to spectral flow cytometry and TCRseq. These TIL⁺ and TIL⁻ tumors underwent single-cell RNA-seq (scRNAseq), in situ Xenium spatial transcriptomics using a customized 480-gene panel, and CODEX spatial proteomics.



 The TIL⁺ tumors yielded 0.5-3.2 × 10⁹ cells with 100-1,000-fold expansion. Flow cytometry showed CD4⁺ and CD8⁺ effector memory phenotypes with minimal exhaustion; Tregs were <10% and γδ T cells were detected. TCRseq revealed high frequency αβ clones and diverse γδ repertoires. Expanded γδ T cells had higher D50 and diversity indices, suggesting a broad, non-dominant repertoire. TIL⁺ tumors exhibited elevated IL7R and ACSS3 expression, enriched Th1/pro-inflammatory gene signatures, and lower Treg-associated transcripts. Spatial analysis showed perivascular immune clusters in TIL⁺ tumors, enriched for CD8⁺ T cells and antigen-presenting cells, with local upregulation of IL7R, MHC II, and CXCL13. These areas were associated with vascular markers (e.g., VCAM1) and T cell retention signatures. In contrast, TIL⁻ tumors displayed immune exclusion, neuronal lineage enrichment, and higher expression of immune suppressive genes like TOX and FERMT1. Tumor-connected TAMs expressing SOX2, NES, and AQP4 were more abundant in TIL⁻ tumors, suggesting a barrier to infiltration or active immune suppression.



 TIL⁺ tumors exhibit distinct molecular and spatial features associated with successful TIL expansion, including IL7R-driven T cell responsiveness, mesenchymal metabolic programs, and structured perivascular immune niches. These findings define biomarkers of therapeutic TIL production and inform strategies to stratify patients for TIL⁻ based therapies in glioblastoma and other immune-excluded tumors. Incorporating these biomarkers into patient selection and workflows could improve clinical trial design, product consistency, and therapeutic response rates.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98b47fb61580f74a05d0a598bf26243ed8f703f3" target='_blank'>
              KS03.4.A SPATIAL MULTI-OMIC PROFILING REVEALS DETERMINANTS OF TIL EXPANDABILITY IN HIGH-GRADE GLIOMAS
              </a>
            </td>
          <td>
            K. Hotchkiss, K. Zhang, A. Corcoran, E. Owens, P. Noldner, C. Railton, K. Van Batavia, Y. Zhou, J. Jepson, K. Singh, R. McLendon, K. Batich, A. Patel, K. Ayasoufi, M. Brown, E. Calabrese, J. Xie, J. Conejo-Garcia, B. Shaz, J. Hickey, M. Khasraw
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Introduction Fibroblasts are critical mediators of tumor progression and metastasis; however, their heterogeneity and specific functions in the context of colorectal cancer (CRC) liver metastases remain incompletely understood. Methods We performed single-cell RNA sequencing (scRNA-seq) analysis on relevant tissue samples. Subsequently, we validated our findings using immunofluorescence assays on histological slides from primary CRC with liver metastasis (mCC) and liver metastatic tumors (mLC). Further investigations included transcriptomic profiling, pseudotime trajectory analysis, multiplex immunofluorescence staining, and cell-cell communication analysis. Results Our scRNA-seq analysis identified a distinct PRELP-positive cancer-associated fibroblast (CAF) subtype that is associated with liver metastasis and tumor progression. These PRELP+ CAFs were predominantly enriched in mLC, less abundant in mCC, and rare in non-metastatic CRC (nCC). This distribution was confirmed by immunofluorescence. Transcriptomically, PRELP+ CAFs exhibit a unique signature defined by extracellular matrix components (e.g., PRELP, COLEC11, ITGBL1) and the activation of pro-tumor pathways such as TGF-β and Wnt signaling. Pseudotime analysis indicated they represent a terminal fibroblast differentiation state. Spatially, they colocalize with immune cells (T cells, B cells, plasma cells), and communication analysis suggests they foster an immunosuppressive microenvironment via APP-CD74 and collagen-CD44 signaling, thereby promoting immune evasion. The transcription factors NR2F2, JUN, and JUND were identified as key regulators of this CAF subtype. Discussion These findings provide crucial new insights into fibroblast heterogeneity within CRC liver metastases. We characterize PRELP+ CAFs as a specialized, terminally differentiated fibroblast population that contributes to immunosuppression and tumor progression, highlighting them as a potential therapeutic target for inhibiting metastatic advancement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62bf8481f437e703e061482f9031f26e35dec625" target='_blank'>
              Characterization of the phenotype and function of PRELP+ fibroblast subtype in liver metastatic colorectal cancer
              </a>
            </td>
          <td>
            Yuting Dai, Xingying Huang, Min Sun, Shiyu Zhang, Weiqiang Yu, Kongwang Hu, Qiang Wu, Qingfa Wu
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195d5e43291b6baf82f9d6ab8680e6d30406a98c" target='_blank'>
              Differential colocalization of cell-type pairs in single-cell spatial omics with DC-SPOMIC
              </a>
            </td>
          <td>
            Jacob Chang, Almudena Espín Pérez, Perla Molina, Lu Tian, Sylvia K Plevritis
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617cbae0aef811bf0bd717218d2f691f2de87645" target='_blank'>
              Germline genetic risk converges through intercellular crosstalk in tumor microenvironment
              </a>
            </td>
          <td>
            X. Bai, Yang Liu, Shuai He, Chun-Ling Luo, G.-W. Lin, Dong-Mei Chi, Shuang-Yan Ye, Xiu-Zhi Wang, , You Zhou, Shu-Qiang Liu, P.-P. Wei, Xin-Yuan Guan, Hai-Qiang Mai, Soon Thye Lim, Jian-Jun Liu, C. Khor, Melvin Lee, Kiang Chua, Jin-Xin Bei, Yanni Zeng
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c21e29753393c78c9a7147383291954089d02fd" target='_blank'>
              KDM5-driven transcriptional noise fuels plasticity-led awakening and relapse in paediatric cancer
              </a>
            </td>
          <td>
            Alejandro Allo Anido, Cecilia Roux, Emilia Chen, Siân Hamer, Abigail Shea, Ayeh Sadat Sadr, Christie English, Charlotte Butterworth, Harvey Che, Angella Bellini, Birgit Geoerger, Gudrun Schleiermacher, L. Chesler, Michael David Morgan, Alejandra Bruna
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are key regulators of the tumor microenvironment, yet their spatial organization and immunomodulatory functions in H. pylori-associated gastric cancer (GC) remain incompletely understood. We profiled formalin-fixed paraffin-embedded (FFPE) tumors from 71 GC patients using spatial transcriptomics and integrated single-cell RNA-seq from three independent cohorts (China, USA, and Singapore). CAF-immune cell colocalization was quantified by neighborhood enrichment and aggregation index score. Ligand-receptor inference and trajectory analysis resolved CAF signaling and state transitions. To delineate post-transcriptional control, ARE-motif scanning and expression correlations were combined with laser-assisted crosslinking and immunoprecipitation sequencing (LACE-seq) to nominate ZFP36 targets. Immune contexture and prognostic associations were evaluated using CIBERSORT-ABS and Kaplan-Meier analyses in The Cancer Genome Atlas (TCGA) and Asian Cancer Research Group (ACRG) cohorts. We first defined the spatial distributions of the four CAF subtypes reported in prior studies and found that their immune associations varied across histologic and infection-defined GC subtypes. In H. pylori-positive tumors, THBS1⁺ CAFs were spatially enriched near regulatory T cells (Tregs) and were associated with local immunosuppression through WNT5-FZD interactions. In parallel, ZFP36 bound AU-rich elements within the FN1 3′ untranslated region (3'UTR), destabilizing FN1 mRNA and thereby diminishing FN1⁺ CAF-mediated cytotoxic T lymphocyte (CTL) activation. Together, these axes promoted Treg accumulation and suppression of CTL activation. These findings reveal infection-associated stromal programs that shape the immune landscape in GC and highlight CAF-directed pathways as potential therapeutic targets in H. pylori-associated GC. H. pylori-induced CAF-mediated immunosuppression in GC. 1. H. pylori infection triggers transcriptional reprogramming of CAFs within the tumor microenvironment of GC. 2. The CAF subset marked by THBS1 is spatially enriched adjacent to Tregs and facilitates their recruitment via WNT5-FZD ligand-receptor signaling. 3. The RNA-binding protein ZFP36 binds AU-rich elements in the FN1 3’UTR, destabilizing FN1 mRNA and thereby attenuating CTL activation. 4. The THBS1-WNT5 axis stabilizes Tregs, whereas the ZFP36-FN1 axis limits CTL activation, converging to drive local immunosuppression in H. pylori-positive GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f1c559549d2edda98547851b085fedd496597a" target='_blank'>
              Spatial and functional dissection of cancer-associated fibroblasts-mediated immune modulation in H. pylori-associated gastric cancer
              </a>
            </td>
          <td>
            Bonan Chen, Hongzhen Tang, Xiaohong Zheng, Fuda Xie, Peiyao Yu, Yang Lyu, Tiejun Feng, Jialin Wu, Jingya Liu, Yi Xu, A. H. Cheung, Canbin Fang, Zhangding Wang, Shouyu Wang, Justin Chak Ting Cheung, Yujuan Dong, Ruoxi Tian, Yigan Zhang, Cheng Lu, Chi Chun Wong, Jun Yu, William K. K. Wu, Elke Burgermeister, Man Tong, Fengbin Zhang, Wei Kang, Kam Tong Leung, K. To
          </td>
          <td>2025-11-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386c9a2a1e499f753a4ef278dadc12b1134f2bac" target='_blank'>
              Disrupted astrocyte-endothelial crosstalk drives hemangioblastoma lesions in VHL disease
              </a>
            </td>
          <td>
            Macarena de Andrés-Laguillo, Irene Garcia-Gonzalez, Susana F Rocha, Aroa Garcia-Cabero, Sandra Ruiz-García, Luis Diago-Domingo, Aimane Danana, Lorena Cussó, Katrien de Bock, J. A. Enríquez, Rui Benedito
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are central architects of the prostate cancer (PCa) microenvironment, yet their phenotypic diversity and druggable vulnerabilities remain largely uncharted. Here, we present an integrative multi-omics characterization of primary ex vivo CAFs from seven treatment-naïve PCa patients. Using single-cell RNA sequencing (scRNA-seq), we uncover substantial transcriptional heterogeneity among CAFs, with distinct gene expression programs related to extracellular matrix remodeling, inflammation, immune modulation, and metabolic reprogramming. This phenotypic diversity was further supported by variable expression of canonical stromal markers, including FAP, SULF1, VIM, CAV1, and αSMA. Transcription factor network analysis revealed SOX, FOX, and STAT3 family members as key regulators of pro-tumorigenic CAF states. To probe therapeutic vulnerabilities, we performed high-throughput drug sensitivity and resistance testing (DSRT) across 396 oncology compounds. CAFs exhibited broad sensitivity to multikinase inhibitors, with dasatinib, midostaurin, and FGFR inhibitors (AZD4547, erdafitinib) emerging as top stromal-directed candidates. These findings underscore the plasticity of prostate CAFs and reveal actionable vulnerabilities, supporting the development of targeted stromal therapies to disrupt tumor-stroma interactions in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0342abe6ecbf94b88ae457a3f680a1e8d054b454" target='_blank'>
              Molecular profiling of ex vivo prostate cancer CAF models captures stromal heterogeneity and drug vulnerabilities
              </a>
            </td>
          <td>
            Frida Rantanen, A. Murumägi, Mariliina Arjama, K. Välimäki, Elina Multamäki, T. Mirtti, Antti S. Rannikko, T. Pellinen, Daniela Ungureanu, Olli Kallioniemi
          </td>
          <td>2025-11-06</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The formation of budding cancer cells at the invasive front of solid tumors is one of the first steps of metastasis. However, this process is still incompletely elucidated. Here, we used spatial molecular imaging to disentangle the complex interactions between cancer cells and the tumor microenvironment at the invasive front of colorectal tumors. Employing a 1000-plex gene panel, we defined all major cell types in tumors and adjacent normal tissue with accurate spatial information. Individual cancer cell clusters were located together, consistent with an expected mutation- and epigenetic-driven clonal evolution. However, cancer cell clusters encompassing budding cells exhibited a markedly different spatial distribution as they also contained cells that were scattered around the periphery of the main cancer cell masses. Moreover, these cells were frequently in contact with cancer-associated fibroblasts (CAFs) and underwent broad gene expression changes, mainly related to epithelial-mesenchymal transition (EMT), remodeling of the extracellular matrix (ECM), and migration. In addition, we defined an 11-gene signature (TYK2, IL2RG, KRT17, HLA-B, NPPC, WIF1, IL32, B2M, CCND1, CRIP1, ITGB1), which characterizes cancer cells en route to metastasis and is associated with inferior outcomes. Collectively, our findings suggest that CAFs induce pro-invasive gene expression changes involved in EMT, ECM remodeling, and migration. Single-cell spatial transcriptomics reveals that CAFs at the invasive front of colorectal tumors induce pro-invasive changes in budding cancer cells, characterized by an 11-gene signature linked to metastasis and poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9341a52e672e1a44fbf627c6049087936b5a159e" target='_blank'>
              Cancer-associated fibroblasts shape the formation of budding cancer cells at the invasive front of human colorectal cancer
              </a>
            </td>
          <td>
            J. L. García-Rodríguez, Ulrik Korsgaard, Stine M Vissing, Thea Petersen Paasch, Mariana Semenova, Simon L Vendelbo, E. Jensby, Hannah L Williams, Paul Vinu Salachan, Camilla Blunk Brandt, J. Hanimann, Lin Lin, I. Zlobec, K. D. Sørensen, Jørgen Kjems, Henrik Hager, L. S. Kristensen
          </td>
          <td>2025-09-16</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d4b0eb929a81f46df1d1e2e57abece7f236b787" target='_blank'>
              PDAC tumor Immune system escape architecture: Analysis of transcriptomics of mechanisms that TME is tackling immune system in Pancreatic Ductal Adenocarcinoma Tumors
              </a>
            </td>
          <td>
            Abbas Zareei, Hossein Allahdadi
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Heterogeneity within the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment (TME), across tumor locations, and between patients largely contributes to poor treatment response and PDAC’s status as the fourth leading cause of cancer death in the United States. In order to identify suitable targets for innovative therapeutic interventions, it is necessary to understand and mechanistically test TME constituents, their infiltration, and their interaction dynamics in the context of primary and metastatic tumor locations using pre-clinical modeling. Therefore, we designed a novel 46-marker multiplex immunofluorescent staining panel for murine formalin-fixed paraffin-embedded (FFPE) tissues. This panel enables the spatial quantification of immune cell populations and activation/exhaustion states, epithelial-to-mesenchymal phenotypes, cancer-associated fibroblast heterogeneity, hematologic and lymphatic vessel infiltration, neuronal infiltration, and extracellular matrix deposition in the TME. We quantified the composition and spatial infiltration of the TME on matched murine primary and liver tumor samples from tandem orthotopic and hemi-spleen transplants of three distinct primary murine PDAC cell lines derived from the KPC model (KrasG12D; TP53R172H; PDX1-Cre; RosaEYFP). The resulting primary tumors from each cell line showed inter-tumoral heterogeneity as their distinct levels of immune infiltration had varying clustering patterns. Spatial analysis further revealed heterogeneous cancer cell MHC-I expression, immune cell exhaustion, and myeloid and lymphoid populations correlating with cell line. However, all liver nodules were encased by a barrier of immune cells separating the tumor from normal liver parenchyma irrespective of cell line. This observation reflects our recent spatial transcriptomic analysis of patient-derived matched liver and primary tumors, supporting the use of these pre-clinical models for future mechanistic investigations. Altogether, these data highlight the complexity of PDAC TMEs and the site-specific differences between primary and metastatic tumor composition.



 Christina R. Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert Welner, Julienne L. Carstens. High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B110.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335836491342c8b5df5a982f9e0790e87ffd1d16" target='_blank'>
              Abstract B110: High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity
              </a>
            </td>
          <td>
            Christina R Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by the coexistence of classical epithelial, basal, and mesenchymal cancer cell states. Targeting cancer cell plasticity—the ability of tumor cells to adapt to extrinsic pressures via non-genetic mechanisms—has emerged as a promising therapeutic strategy. In parallel, the immunosuppressive tumor microenvironment remains a major barrier to effective treatment in PDAC. Yet, whether specific cell states possess distinct functional properties that drive key tumor behaviors—such as phenotypic plasticity or immune evasion—remains poorly understood. Using genetically engineered mouse models driven by oncogenic Kras G12D and loss of Trp53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the "basal" signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we find that the basal state is a key driver of therapeutic resistance: rather than being eliminated by treatment, it persists through phenotypic adaptation in response to both KRAS-targeted therapies and cytotoxic chemotherapy. Co-elimination of the basal state alongside these therapies leads to deeper and more durable responses, underscoring its functional role in tumor persistence under therapy. In parallel, we observe that the basal state plays a central role in enforcing immunosuppression in PDAC tumors. Basal state ablation rapidly reshapes the tumor immune microenvironment, and we observe loss of cancer-associated fibroblasts, polarization of PDAC-resident macrophages toward a pro-inflammatory phenotype, and increased infiltration of T and NK cells. Cytokine profiling of basal-ablated tumors revealed a dramatic reduction in multiple immune-modulatory and immunosuppressive cytokines. Taken together, our data indicate that PDAC harbors a striking dependency on the basal state for both therapeutic resistance and immune evasion, and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.



 Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C. Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Zeynep C. Tarcan, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Dana Pe'er, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6879ea5062e58b2315581060e5992acf02efdbb" target='_blank'>
              Abstract B108: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Z. Tarcan, O. Basturk, R. Yaeger, James G Christensen, Yan Yan, E. de Stanchina, D. Pe’er, Tuomas Tammela
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a055b2099d39e961c79dc682be1c20375d7bba" target='_blank'>
              Tumor microenvironment response to neoadjuvant therapy in breast cancer: insights from single-cell and spatial omics
              </a>
            </td>
          <td>
            Qingzhong Wu, Jiawei Yang, Da Zhang, Haowei Xu, Yuhang Yu, Yu Zheng
          </td>
          <td>2025-09-13</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Recent studies using single-cell RNA sequencing (scRNA-seq) have provided insights into the cellular composition and molecular states in glioblastoma (GBM). However, these studies have not identified significant differences between primary and recurrent GBMs. We hypothesize that other factors, such as spatial organization and cellular interactions, may provide key information driving recurrence in GBM. Spatial transcriptomics techniques, such as MERFISH (Multiplexed Error-Robust Fluorescence In Situ Hybridization), provide single-cell spatial resolution of tissues, enabling the identification of distinct molecular features. MERFISH, in particular, allows for the 3D reconstruction of gene expression signatures in consecutive tissue slices, providing a detailed spatial map of the tumor cells and surrounding environment. This technology is invaluable in studying longitudinal GBMs, where changes in tumor organization, immune cell interactions, and mechanisms of therapeutic resistance remain poorly understood.



 We analyzed 102 tumor samples consisting of primary and recurrent GBMs and normal brain tissue using Visium and MERSCOPE technologies. We used supervised artificial intelligence (AI)-based learning, in-silico perturbation and explainable-AI to identify distinct spatial and cell-type-specific transcriptomic reorganization associated with GBM recurrence.



 We developed a spatial transcriptomics atlas of GBM at sub-cellular resolution. Our analysis revealed distinct cellular neighborhoods in the primary and recurrent GBM tumors, with changes leading to the dynamic spatial reorganization of GBM at recurrence. The 3D spatial images acquired provided a comprehensive view of the tumor cells interacting with the tumor microenvironment in situ. Our explainable-AI models successfully identified distinct spatial transcriptomic features associated with recurrence in GBM. The identified spatially distinct cellular neighborhoods reveal dynamic shift in cellular organization following in silico perturbation.



 This study provides a high-resolution spatial map of longitudinal GBM that offers new insights into the dynamic cellular reorganization of GBM at recurrence. Our findings highlight the role of cellular reorganization during tumor recurrence in GBM, a potential avenue for targeted therapeutic intervention.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4929df1bf2a8e42843ae0cfe5558a1691cc6412" target='_blank'>
              P04.05.B DEEP LEARNING ANALYSIS OF SPATIALLY RESOLVED SINGLE-CELL TRANSCRIPTOMICS UNCOVERS CELLULAR REORGANIZATION IN GLIOBLASTOMA AT RECURRENCE
              </a>
            </td>
          <td>
            Y. Yabo, E. Grabis, T. Peng, J. Benotmane, J. Kueckelhaus, J. Zhang, G. Villa, N. Neidert, M. Prinz, J. Beck, A. Ahmed, O. Schnell, F. Sahm, V. Venkataramani, F. Ricklefs, R. Sankowski, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most common histological subtype of renal cancer and remains a clinical challenge due to its frequent resistance to therapy and poor prognosis in advanced stages. Apoptosis, a fundamental tumor-suppressive mechanism, exhibits paradoxical roles in cancer, wherein apoptotic tumor cells can also contribute to immunosuppression and tumor progression. However, the spatial dynamics, transcriptional heterogeneity, and prognostic relevance of apoptosis-related gene programs in ccRCC remain poorly defined. We performed an integrative analysis combining single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and summary-based Mendelian randomization (SMR) to dissect apoptosis-related malignant cell states in ccRCC. Cancer cells were stratified based on apoptosis gene signatures and CASP9 expression. Cell–cell communication was assessed using CellChat and spatial interaction networks were constructed using RCTD and mistyR. SMR was employed to link genetically regulated CASP9 expression with renal cancer risk. A CASP9-associated prognostic model was developed using LASSO Cox regression and DeepSurv on TCGA and E-MTAB-1980 cohorts. We identified transcriptionally and spatially distinct apoptosis-high and apoptosis-low malignant cell subpopulations. Apoptosis-high tumor cells, characterized by elevated CASP9 expression, preferentially localized near macrophage-enriched stromal regions and exhibited stronger spatial clustering. Ligand–receptor modeling revealed directional signaling via the SPP1–CD44 axis between CASP9-high cancer cells and macrophages. SMR analysis provided genetic evidence supporting CASP9 as a causal gene for renal cancer. CASP9-high cells demonstrated distinct developmental trajectories and formed multicellular spatial modules with macrophages and cycling cells. A five-gene apoptosis-related signature derived from CASP9-stratified tumor cells robustly predicted patient survival across both training and validation cohorts. Low-risk patients exhibited enriched immune infiltration, increased immune checkpoint expression, and enhanced immune pathway activity. Our study reveals that apoptosis, particularly CASP9-driven programs, defines a spatially organized, immunosuppressive malignant cell state in ccRCC. CASP9 acts as both a genetic driver and spatial regulator of tumor–macrophage interactions, contributing to disease progression. The CASP9-associated risk model demonstrates strong prognostic utility and highlights apoptosis as a promising therapeutic axis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22570e710b314543b7842267551dd125b21247d1" target='_blank'>
              Integrative SMR and single cell & spatial analysis reveals the spatial heterogeneity and prognostic value of CASP9-mediated apoptotic pathways in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Zijie Yu, Xinghan Yan, Wenchuan Shao, Lingchen Cai, Da Zhong, Xiyi Wei, Ninghong Song
          </td>
          <td>2025-11-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The ability to interrogate changes within the tumor microenvironment (TME) before, during and following therapeutic intervention could yield important understanding of treatment response and causes for disease progression. Yet, the role of investigational tissue analysis faces key challenges in the clinical setting and the value of integrating longitudinal biopsies with emerging multimodal molecular analyses (“Multi-omics”) remains to be defined. In this study, we conducted a multicenter phase 2 clinical trial examining the effect of a novel cytotoxic T-lymphocyte-associated protein 4/programmed death-ligand 1 bispecific antibody in combination with a dual-epitope blocking anti-human epidermal growth factor receptor 2 antibody in treatment-resistant metastatic breast cancer. We performed longitudinal sampling of patient tumor tissues before and following treatment. Single-cell RNA and T cell receptor sequencing from 334,183 cells from site-matched tumors reveals significant temporal shift of various immune cell populations and phenotypes within the TME associated with treatment responses. Conversely, regulatory T cells were activated while effector T cells, natural killer cells, and dendritic cells were significantly depleted in non-responding tumors. Taken together, these results support that longitudinal analysis of TME to generate multiomics data that can lead to rich insight into disease process and to provide clinical value in evaluating treatment responses. Trial registration number NCT04521179.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97d0a38c945abea56c7fcf08591e230c989d914" target='_blank'>
              Longitudinal tissue analysis reveals microenvironmental changes correlate with combined immunotherapy and targeted therapy response in metastatic breast cancer
              </a>
            </td>
          <td>
            Jianyou Liao, Jien-Ren Wang,, Hengyu Li, Zhijun Liu, Z. Tian, Xinying Lv, Jianjian Peng, Chuangui Song, Jieqiong Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4534f83212d853ecc1eca8aac314fc0a39255797" target='_blank'>
              Integrative Spatial Modelling of Cellular Plasticity using Graph Neural Networks and Geostatistics
              </a>
            </td>
          <td>
            Eloise Withnell, Cenk Celik, M. Secrier
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes
 TP53
 and
 RB1
 . However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is classified into epithelial and mesenchymal subtypes based on transcriptional profiling, and these subtypes are associated with varied patient outcomes. Differences in the tumor microenvironment (TME), particularly in cancer-associated fibroblasts (CAFs), may contribute to this variation. scRNA-seq studies have observed heterogeneous CAF populations in PDAC, although the relationship between tumor subtype and CAF phenotype is not well understood. To investigate this relationship, we utilized epithelial and mesenchymal PDAC cell lines in a xenograft model to examine whether these subtypes influence CAF phenotypes.



 Xenograft models were generated by subcutaneously injecting BALB/c nude mice with PDAC cell lines of either the epithelial subtype (BxPC-3) or the mesenchymal subtype (MIA PaCa-2). Tumors were then harvested and dissociated for scRNA-seq to investigate the TME. Human tumor (graft) and murine stromal (host) cells were computationally separated using the Xenocell tool. Subsequent analyses focused on fibroblasts within the TME, which were subclustered to assess CAF heterogeneity between PDAC subtypes. To characterize transcriptional programs and identify CAF features, we performed high-dimensional weighted gene co-expression network analysis (hdWGCNA) and analyzed differentially expressed genes. Finally, spatial transcriptomics was conducted on tumor sections from BxPC-3 and MIA PaCa-2 xenografts to validate CAF clusters observed in scRNA-seq.



 Distinct CAF clusters were identified in the xenograft tumors, including LRRC15+ CAFs, RGS5+ CAFs, and inflammatory CAFs (iCAFs). LRRC15+ CAFs were predominant in tumors derived from the epithelial subtype (BxPC-3), while RGS5+ CAFs were enriched in tumors from the mesenchymal subtype (MIA PaCa-2). LRRC15+ CAFs exhibited a myofibroblastic CAF (myCAF) phenotype characterized by extracellular matrix production, with high expression of Lrrc15, Postn, and Fbln1. RGS5+ CAFs expressed canonical pericyte markers such as Rgs5 and Cox4i2, along with CAF-associated genes including Acta2. These CAF clusters also showed distinct expression patterns of collagens and secreted factors. In particular, Col11a1, Col12a1, and Wnt5a were upregulated in LRRC15+ CAFs, while Col4a1, Col10a1, and Tgfb1 were upregulated in RGS5+ CAFs. Spatial transcriptomics confirmed that gene modules associated with Lrrc15 and Rgs5, inferred from hdWGCNA, were differentially expressed between Acta2-positive stromal regions of epithelial and mesenchymal PDACs.



 These results suggest that epithelial and mesenchymal PDAC subtypes directly shape distinct CAF populations. Notably, the enrichment of RGS5+ CAFs in mesenchymal tumors may underlie their metastatic potential and poor clinical outcomes.



 Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, Jung Hyun Jo. Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B105.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020656bad62fdcf52b16552c5f7369030a0e1205" target='_blank'>
              Abstract B105: Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model
              </a>
            </td>
          <td>
            Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo-Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, J. Jo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Gliosarcoma, a rare and aggressive variant of IDH-wildtype glioblastoma, is characterized by striking intra-tumoral heterogeneity with alternating glial and mesenchymal differentiation. The mesenchymal component is often enriched at recurrence and associated with therapy resistance, suggesting treatment-driven clonal selection. Despite its poor prognosis and distinct histology, gliosarcoma remains underexplored at the molecular level, with no systematic profiling of its cellular or spatial architecture - limiting progress toward targeted therapies.



 Single-nucleus RNA sequencing (snRNA-seq) was performed on eight FFPE gliosarcoma samples, complemented by spatial transcriptomic profiling (10X Visium) in four matched cases. Multimodal integration enabled high-resolution mapping of malignant and non-malignant compartments. To enhance spatial context extraction and to model tumor-microenvironment interactions, advanced computational approaches, including graph neural networks (GNNs) and convolutional neural networks (CNNs), are being developed.



 Integration of snRNA-seq and spatial transcriptomic data revealed a distinct population of stromal-like malignant cells enriched for mesenchymal signatures. Compared to conventional glioblastoma, gliosarcoma samples exhibited a reduced presence of neural progenitor cell (NPC-like) and oligodendrocyte progenitor cell (OPC-like) states, suggesting early dominance or selective expansion of the mesenchymal compartment. Spatial annotation of the tumor microenvironment revealed immune-rich niches populated by disease-associated microglia (DIMs), damage-associated macrophages (DAMs), and macrophages (MACs). Preliminary spatial modeling indicates potential interactions between immune population and mesenchymal-like tumor cells that may support therapy-resistant niches.



 This ongoing study integrates single-nucleus and spatial transcriptomic profiling to characterize mesenchymal transition and resistance in gliosarcoma. Early findings highlight a distinct mesenchymal-like malignant population and associated immune environments that may drive resistance. Further computational modeling aims to uncover key cellular interactions and molecular programs, that could inform novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47d9bc10df09a19173093cb2520a4af4dc46d0d5" target='_blank'>
              P14.25.B DECODING MORPHO-MOLECULAR DIVERSITY IN GLIOSARCOMA AND INSIGHTS INTO RESISTANCE USING SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            D. Savran, N. Schoebe, N. Berghaus, D. Reuss, M. Ratliff, C. Herold-Mende, S. Krieg, W. Wick, A. von Deimling, D. Heiland, F. Sahm, P. Sievers
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) evolves within a highly interactive tumor microenvironment (TME) that shapes therapeutic response. We utilized mass cytometry to analyze over 10 million cells from 25 ESCC tumors, 24 adjacent nontumor tissues, and 23 peripheral blood samples, employing an extensive panel of 42 immune markers. The resulting atlas reveals a compartmentalized landscape with a reproducible paucity of CD4⁺ and CD8⁺ central memory T cells (TCM) in tumor sites. Reintroduction of patient-derived TCMs restored antitumor immunity in coculture assays, demonstrating their cytotoxic capacity in vitro and suggesting their potential relevance for future therapeutic exploration. Myeloid profiling identified PD-L1⁺ tumor-associated macrophages (TAMs) as correlates of clinical benefit; ex vivo PD-L1 blockade reprogrammed TAMs toward proinflammatory states, indicating pharmacological malleability. Notably, CD39⁺ tumor-infiltrating T cells were consistently associated with favorable prognosis and increased responsiveness to PD-1 blockade across cancer types. The functional inhibition of CD39 impaired cytotoxic T-cell activity, underscoring its dual role as a marker of immune dysfunction and a promising therapeutic target. Collectively, our findings provide a comprehensive immune landscape of ESCC, highlighting key immunological deficits and opportunities for targeted interventions. The insights gained underscore the potential of tailoring immunotherapies to the specific immune profiles of the TME, potentially revolutionizing treatment paradigms for ESCC patients. This study sets the stage for a more nuanced understanding and manipulation of the immune elements critical for optimizing cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c74ee06dce5a8743106e3e0c5ec6ea39727b2973" target='_blank'>
              Single-cell atlas of the esophageal squamous cell carcinoma immune ecosystem to predict immunotherapy response
              </a>
            </td>
          <td>
            Xiankai Chen, Yahui Zhao, Yuhao Wang, Xiliang Wang, Yuhao Liu, Zhihua Liu, Yin Li
          </td>
          <td>2025-10-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT The immune landscape of healthy prostate and its alterations during prostate cancer (PCa) progression remain poorly characterized. Using scRNA-sequencing and multiparametric flow-cytometry analysis, we comprehensively characterized immune cells in wild-type and PTEN(i)pe−/− mouse prostates, a model that closely recapitulates human PCa. PCa in PTEN(i)pe−/− is marked by the recruitment of tumor-associated neutrophils (TANs), which represent the dominant immune cell population and resolved into eight distinct states, Trem2+ tumor-associated-macrophages (TAMs), and exhausted CD8+ T cells. Trem2+ TAMs differ from the three main resident macrophage populations in the healthy prostate, exhibiting a strong metabolic and immunosuppressive signature, likely driven by the MIF/HIF1A-signaling axis. This study provides the first detailed characterization of immune cells in the healthy mouse prostate and reveals changes in the immune landscape associated with prostate cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb0c0592aa197777e78a6d1020c20a4157e6fc" target='_blank'>
              Characterization of the immune landscape in healthy mouse prostate and during prostate cancer progression
              </a>
            </td>
          <td>
            Despoina Pervizou, Joanna De Chiara, L. Spinelli, Maia Nestor-Martin, Lionel Chasson, Kateryna Len-Tayon, Darya Yanushko, Frédéric Fiore, Marc Bajénoff, Bernard Malissen, Daniel Metzger, Gilles Laverny, S. Henri
          </td>
          <td>2025-09-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is an aggressive ovarian cancer with a heterogeneous tumor microenvironment (TME). Advances in single-cell RNA sequencing (scRNA-seq) and spatially-resolved transcriptomics have enabled the study of complex TME. This study explores connections between molecular subtypes described from bulk transcriptomes and spatial domains characterized by distinct gene expression in HGSC and their variability between patients. We explored both intraand inter-tumor heterogeneity across 2D space and identified differing spatial patterns of gene expression pertaining to immune pathways and vasculature development. Functional characterization of tumor spaces revealed potentially shared cell states across molecular subtypes, while correlation analysis underscored subtype-specific spatial anti-colocalization between spots exhibiting antigen-presenting functions and B cell-mediated immunity. Lastly, we performed spatially-aware cell-cell communication analysis on the spatial samples and identified a molecular subtype specific difference in total signaling activity and heterogeneity in Midikine signaling between the differentiated subtype. Our results suggest that generating multiple tissue slices per patient might be necessary to enable comprehensive characterization of HGSC spatial transcriptomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12564a0c02e3362085c018e2a380154da3a01959" target='_blank'>
              Characterizing intra- and inter-tumor heterogeneity in Ovarian high-grade serous carcinoma subtypes using single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Weishan Li, Laurie Grieshober, Jason Gertz, Adriana Ivich, J. Doherty, Casey S. Greene, SC Hicks
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Metastasis remains the principal cause of cancer-related mortality, yet its distribution across organs is far from random. Instead, tumor cells exhibit organotropism, a consistent preference for colonizing specific distant tissues, a phenomenon shaped by anatomical constraints, molecular crosstalk, and microenvironmental compatibility. Far beyond mere mechanical entrapment in vascular beds, metastatic dissemination reflects a coordinated interplay between tumor-intrinsic programs and organ-specific niches. Tumor-derived extracellular vesicles, cytokines, and matrix-remodeling enzymes actively precondition distant sites through pre-metastatic niche formation, creating permissive microenvironments primed for colonization. Simultaneously, tissue-specific immune landscapes, stromal compositions, and mechanical architectures determine the fate of disseminated tumor cells, whether they are eliminated, enter dormancy, or form macrometastases. Phenotypic plasticity, metabolic reprogramming, and immune evasion mechanisms equip subclones with the capacity to exploit these unique niches. Across cancer types, reproducible patterns of organotropic metastasis not only guide clinical surveillance and therapeutic stratification but also reveal vulnerabilities in the metastatic cascade. This review synthesizes emerging mechanistic insights across anatomical, immunological, and molecular domains to construct a comprehensive framework of organotropism, highlighting therapeutic opportunities to intercept metastasis at organ-specific checkpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7663dab540302726b6f7b6337c8bf27a6cf336" target='_blank'>
              Deciphering organotropism reveals therapeutic targets in metastasis
              </a>
            </td>
          <td>
            Jhommara Bautista, Andrés López-Cortés
          </td>
          <td>2025-10-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Biliary tract cancer (BTC) is a highly heterogeneous and aggressive gastrointestinal malignancy, marked by a high mortality rate and limited treatment efficacy. The primary contributing factors include the absence of reliable early detection methods, the anatomical intricacy of the biliary system, the inherently aggressive tumor biology, and the restricted effectiveness of systemic therapies. A profound understanding of molecular characteristics and clinically relevant emerging biomarkers is essential for advancing BTC treatment strategies. Recent developments in single-cell multi-omics technologies have enabled the analysis of genetic, transcriptomic, proteomic, and metabolomic data at the single-cell resolution, thereby uncovering the heterogeneity and complexity of tumor biology. These techniques provide critical insights into the diversity of immune cell populations within the tumor microenvironment (TME) and offer novel perspectives on tumor progression and potential therapeutic interventions. While single-cell technologies have significantly advanced the study of solid tumors, their application in BTC remains nascent, with a paucity of comprehensive reviews. This review systematically integrates single-cell genomics, transcriptomics, and epigenomics data to construct a cross-omics molecular atlas of BTC. It highlights the utility of single-cell multi-omics technologies in elucidating tumor heterogeneity, microenvironment remodeling, and clonal evolution in biliary tumors, while thoroughly analyzing their implications for clinical outcomes. Furthermore, this review explores personalized treatment strategies informed by single-cell technologies and underscores the significance of these technologies as indispensable tools for unraveling the complexity of BTC and fostering mechanism-based therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c48afe626e59e1df497d34aa627d14168d0c25d2" target='_blank'>
              Single-cell multi-omics in biliary tract cancers: decoding heterogeneity, microenvironment, and treatment strategies
              </a>
            </td>
          <td>
            Nannan Tang, Jiatong Li, Ao Gu, Meng‐Yao Li, Yingbin Liu
          </td>
          <td>2025-10-15</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32263f8daacb6eb2cfb9538393332506d52b7ecc" target='_blank'>
              Characterising cancer-stroma interactions through high-content phenotyping from microscopy time-lapses
              </a>
            </td>
          <td>
            Laura Wiggins, Jodie R. Malcolm, Karen Hogg, Peter J. O’Toole, Julie Wilson, W. Brackenbury
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract The complex interplay between metal abundance, transport mechanisms, cell distribution, and tumor progression-related biological pathways (e.g. metabolism, collagen remodeling) remains poorly understood. Traditionally, genes and metals have been studied in isolation, limiting insights into their interactions. Recent advances in spatial transcriptomics and elemental profiling now enable comprehensive exploration of tissue-wide metal-gene interactions, though integration remains challenging. In this proof-of-concept study, we investigated metal-dependent signaling within the tumor microenvironment of a unique colorectal cancer (CRC) tumor. We implemented a spatial multimodal workflow which integrated elemental imaging, gene expression, cellular composition, and histopathological features to uncover metals-related pathways through spatially resolved gene expression correlation analyses. Preliminary findings revealed significant associations, for instance: elevated iron correlated with mesenchymal phenotypes located at the tumor’s proliferative front, correlating with expression of genes involved in the epithelial-to-mesenchymal transition pathways, and extracellular matrix remodeling. Preliminary observations from this single sample revealed that high copper concentrations were localized to regions of active tumor growth and were associated with increased expression of immune response genes. This proof-of-concept workflow demonstrates the feasibility of integrating elemental imaging with spatial transcriptomics to identify metals-based gene correlates. Future application of this workflow to larger patient cohorts will pave the way for expansive comparisons across the metallome and transcriptome, ultimately identifying novel targets for tumor progression biomarkers and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/583ee888bce168b26048a39ba23b693a3c315972" target='_blank'>
              Integration of elemental imaging and spatial transcriptomic profiling for proof-of-concept metals-based pathway analysis of colon tumor microenvironment
              </a>
            </td>
          <td>
            A. Srivastava, N. Shaik, Yunrui Lu, M. Chan, A. Diallo, John P. Zavras, Serin Han, T. Punshon, B. Jackson, L. Vahdat, Xiaoying Liu, V. Mittal, K. Lau, J. Gui, Louis J. Vaickus, J. Hoopes, F. Kolling, L. Perreard, J. Marotti, J. Levy
          </td>
          <td>2025-10-01</td>
          <td>Metallomics: Integrated Biometal Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53fdc35836d59f5a9b6ad6d520636c7ede83639b" target='_blank'>
              Single-Cell Analysis of NF1-Expressing and NF1-Deficient Schwann and Fibroblast Cells Reveals Divergent Neurofibroma Programs
              </a>
            </td>
          <td>
            Grace M Swanson, Marcia Arenas-Hernandez, Katherine Gurdziel
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Understanding the cellular origins and early evolutionary dynamics that drive the initiation of carcinogenesis is critical to advancing early detection and prevention strategies. By characterizing key molecular, cellular and niche events at the precancerous tipping point of early gastric cancer (EGC), we aimed to develop more precise screening tools and design targeted interventions to prevent malignant transformation at this stage. We utilized our AI models to integrate spatial multimodal data from nine EGC endoscopic submucosal dissection (ESD) samples (covering sequential stages from normal to cancer), construct a spatial-temporal profile of disease progression, and identify a critical tipping point (PMC_P) characterized by an immune-suppressive microenvironment during early cancer development. At this stage, inflammatory pit mucous cells with stemness (PMC_2) interact with fibroblasts via NAMPT⟶\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\longrightarrow$$\end{document}ITGA5/ITGB1 and with macrophages via AREG⟶\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\longrightarrow$$\end{document}EGFR/ERBB2 signaling, fostering cancer initiation. We established gastric precancerous cell lines and organoids to demonstrate that NAMPT and AREG promote cellular proliferation in vitro. Furthermore, in the transgenic CEA-SV40 mouse model, targeting AREG and/or NAMPT disrupted key cell interactions, inhibited the JAK-STAT, MAPK, and NFκB pathways, and reduced PD-L1 expression, which was also confirmed by western blot in vitro. These interventions delayed disease progression, reversed the immunosuppressive microenvironment, and prevented malignant transformation. Clinical validation was conducted using endoscopically resected EGC specimens. Our study provides a precise spatiotemporal depiction of EGC development and identifies novel diagnostic markers and therapeutic targets for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0410e9abe73bbef3e06750644cca2f27fd69ccca" target='_blank'>
              Spatiotemporal multi-omics analysis uncovers NAD-dependent immunosuppressive niche triggering early gastric cancer
              </a>
            </td>
          <td>
            Pingting Gao, Chunman Zuo, Wei Yuan, Jiabin Cai, Xiaoqiang Chai, Ruijie Gong, Jia Yu, Lu Yao, Wei Su, Zuqiang Liu, Shengli Lin, Yun Wang, Mingyan Cai, Lili Ma, Quanlin Li, Pinghong Zhou
          </td>
          <td>2025-09-22</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Intra- and inter-tumoral heterogeneity combined with immunosuppressive tumor microenvironments contribute to the poor outcomes associated with glioblastoma (GBM). Well-characterized immunocompetent models that recapitulate human GBM features are urgently needed to identify targets in the tumor microenvironment and develop novel therapeutics. Here, we used multi-omic approaches to characterize syngeneic mouse brain tumor stem cell lines in vitro and in orthotopically engrafted tumors.



 Whole-genome sequencing, transcriptomics, ATAC-sequencing, and imaging mass cytometry were used to characterize syngeneic brain tumor stem cell lines derived from Trp53+/-/Nf1+/- C57Bl6 mice. Mouse and human bulk, single-cell, and spatial sequencing datasets were analyzed for validation. CRISPR/Cas9 and shRNA were used for gene knockdowns. Tumor growth was investigated using orthotopic engraftment in syngeneic C57Bl6 mice.



 One of the syngeneic lines, mBT0309, generated tumors with histopathological characteristics of GBM. mBT0309 displayed amplification and high expression of Igf2. Copy number gains at the IGF2 locus were observed in human GBM tumors and stem cell lines. Furthermore, we determined that high IGF2 RNA expression is associated with poor survival in GBM patients. Imaging mass cytometry on mBT0309 tumors showed early infiltration of monocyte-derived macrophages, vascularization, and cell states characteristic of human GBM. Genetic targeting of Igf2 decreased in vitro cell growth, improved survival of engrafted mice, and decreased the percentage of Arginase-1+ macrophages in mBT0309 tumors.



 mBT0309 is a valuable syngeneic model for studying immunosuppression and therapeutic resistance in GBM. IGF2 offers promise as a valuable therapeutic target to combat tumor growth and immunosuppression in GBM patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3386883b6908859de06190f6d3a8fc6a7ff95c1" target='_blank'>
              IGF2 supports glioblastoma growth and immune evasion through a combination of tumor cell-intrinsic and -extrinsic mechanisms
              </a>
            </td>
          <td>
            Kyle Heemskerk, Samir Assaf, X. Hao, Shannon Snelling, M. Meode, R. Hassam, Orsolya Cseh, Smriti Kala, James Pemberton, Jennifer A. Chan, J. Cairncross, Peter Forsyth, V. Wee Yong, Reza Mirzaei, Samuel Weiss, F. Zemp, H. Artee Luchman
          </td>
          <td>2025-10-15</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            A. Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="


 Diffuse midline glioma (DMG) is an aggressive pediatric brain tumor with no curative treatment. The tumor microenvironment (TME) of DMG, shaped by its unique anatomical location and developmental context, poses significant therapeutic hurdles. While the TME has been characterized by a predominance of tumor-associated microglia and sparse infiltration of blood-derived immune cells, a comprehensive understanding of immune-tumor cell interactions and how to effectively target them with immunotherapies remains challenging.



 To dissect the DMG TME, we employed a multi-omics approach that integrated single-nucleus RNA sequencing, spatial transcriptomics, and high-dimensional imaging on both patient samples and an immunocompetent in utero electroporation murine DMG model. We characterized the spatial and phenotypic architecture of the tumors and validated a novel immune target using syngeneic mouse models and ex vivo co-culture experiments.



 Our analysis revealed pronounced intratumoral heterogeneity, with two spatially distinct patterns of cancer-immune interaction. A mesenchymal-like niche promoted the recruitment of blood-derived immune cells, whereas another pattern exhibited an immunologically cold phenotype, reminiscent of the immune-privileged healthy brain. Spatially resolved analysis of cell-cell communication within the TME identified the VISTA immune checkpoint pathway as a key regulator of these patterns. Targeting of this axis in a syngeneic mouse model led to significant tumor reduction and 100% survival in mice, highlighting its therapeutic potential. Further validation demonstrated that microglia play a critical role in orchestrating immune cell recruitment and anti-tumor activity, and that modulating the VISTA pathway can reshape the immune response.



 Our findings uncover the spatial and cellular complexity of the DMG immune microenvironment and identify the VISTA pathway as a critical modulator of immune suppression. By demonstrating robust therapeutic efficacy upon VISTA blockade, this study lays a strong foundation for developing targeted immunotherapies in DMG and opens new avenues for overcoming immune resistance in pediatric brain tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3d8c86dfadea2de1a37fb282c8875e761a9bc8" target='_blank'>
              JS07.7.A VISTA IS A CRITICAL AND TARGETABLE IMMUNE CHECKPOINT PATHWAY IN DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            R. Collot, C. Ruiz-Moreno, C. Honhoff, A. Wezenaar, H. Ariese, H. Johnson, B. M. Vervoort, J. Bunt, E. Bokobza, H. G. Rebel, B. T. Te Pas, P. Robe, M. Kool, M. Kranendonk, D. V. van Vuurden, E. Hulleman, E. J. Wehrens, A. Zomer, H. Stunnenberg, A. Rios
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a fundamental role in tumor progression. Cancer cells interact with their surroundings to establish a supportive niche through structural changes and paracrine signaling. Cells around transformed tumor cells contribute to cancer development, while infiltrating immune cells in this aggressive TME often become exhausted. Solid tumors, especially the most invasive types such as pancreatic ductal adenocarcinoma, are notably stiff mechanically, with cross-linking enzymes significantly affecting the survival of cancer cells in both primary tumors and metastatic sites. In this review, we highlight recent key contributions to the field, focusing on single-cell sequencing of stromal cells, which are increasingly seen as highly heterogeneous yet classifiable into distinct subtypes. These new insights enable the development of effective co-treatment approaches that could significantly enhance current and novel therapies against the most aggressive cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5047b2e8e1a1022055daa398bbad8bc791fe908c" target='_blank'>
              Microenvironment of Solid Tumors.
              </a>
            </td>
          <td>
            T. Korneenko, Nikolay B. Pestov, M. Shakhparonov, Nickolai A. Barlev
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in bioscience</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Metastasis remains the leading cause of cancer-related death, yet the biological determinants that enable tumor cells to disseminate and colonize distant organs are incompletely understood. Emerging evidence identifies the microbiome, not merely as a bystander, but as an active architect of the metastatic cascade. Microbial communities residing in the gut, mucosal barriers, and within tumors shape metastatic progression by modulating immune surveillance, stromal remodeling, oncogenic signaling, and therapy response. Intratumoral and even intracellular microbes regulate epithelial–mesenchymal transition, angiogenesis, and immune escape, while gut-derived metabolites condition pre-metastatic niches and alter systemic immunity. Technological advances in spatial transcriptomics, single-cell multi-omics, and metagenomics have revealed a spatially organized, functionally integrated microbial ecosystem within tumors, challenging long-held assumptions of sterility in cancer biology. This review synthesizes five converging dimensions of this paradigm: microbial interactions in the metastatic tumor microenvironment; microbiome-mediated immunoediting and metastatic escape; the role of intratumoral and intracellular bacteria in dissemination; spatial-multi-omic approaches to map microbial niches; and microbial biomarkers predictive of metastasis and therapy outcomes. Collectively, these findings recast the microbiome as a critical and targetable determinant of metastasis. Deciphering the tumor–microbe–host triad holds transformative potential for biomarker development, therapeutic innovation, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15720d6be643a5b1bcd7bdc88b245af42661e241" target='_blank'>
              Microbial signatures in metastatic cancer
              </a>
            </td>
          <td>
            Jhommara Bautista, María Paula Fuentes-Yépez, Joseth Adatty-Molina, Andrés López-Cortés
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Medicine</td>
          <td>3</td>
          <td>5</td>
        </tr>

        <tr id="


 Most meningiomas are benign, slow-growing tumors. However, CNS WHO grades 2 and 3 meningiomas display significantly higher aggressiveness and poorer prognosis. Recent investigations into the molecular biology of meningiomas have greatly increased our understanding of these tumors. Furthermore, emergent spatial transcriptomic sequencing methods now provide an unprecedented opportunity to study molecular mechanisms of tumor development within a spatial context and on a single-cell level. This study aims to uncover specific microenvironmental features, cell-cell interactions and transcriptional regulatory mechanisms present in different molecular classes and subclones of meningiomas.



 We employed single-nuclei RNA sequencing and spatial transcriptomic profiling on a matching set of meningiomas (n=22). We developed a method based on non-negative matrix factorization (NMF) to analyze intra-tumor heterogeneity from both single-cell and spatial transcriptional profiles across samples. Trajectory analysis was performed to screen for expression signatures selectively up-regulated around specific morphological features of the tumors. We clustered cells into types of niches based on compositions of their spatial neighborhoods.



 We annotated cell types in the tumor microenvironment (TME), and identified cellular states of meningioma tumor cells in single-cell and spatial transcriptomic data. Tumors of different subclasses display drastic differences in spatial cell state compositions, suggesting distinct modes of microenvironmental organization. We identified conserved patterns of tumor-TME cell state colocalization and interaction. Benign NF2-mutant meningiomas are characterized by extensive infiltration of myeloid cells which trigger a series of immune and stress response pathways in neighboring tumor cells. Among more malignant tumors, a decrease of immune cell populations is coupled with a marked shift in the TME toward tumor-supportive phenotypes. Moreover, we observed reduced tumor-infiltrative capabilities among tumor-associated myeloid cells and the formation of compartmentalized spatial niches of interacting tumor cells. Additionally, various tumor transcriptional signatures related to extracellular matrix remodeling, fibroblastic markers, and meningothelial signatures were mapped to distinct morphological structures across samples.



 Our results reveal that the meningioma TME is heterogenous at the cell state level, with tumor cells and normal cells forming niches that have varying functions in tumor pathophysiology and characterize molecular classes of meningiomas. A series of corresponding RNA markers and spatial signatures were identified, which could potentially be used to increase precision in diagnosis and stratification of meningiomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb7c99c73e50db41072f3f2a1080df073fd8ef43" target='_blank'>
              OS13.6.A A SINGLE-CELL AND SPATIAL ATLAS OF MENINGIOMA HETEROGENEITY ACROSS TUMOR CLASSES
              </a>
            </td>
          <td>
            Y. Tang, R. Banan, S. Maas, D. Calafato, R. Michael, J. Kada Benotmane, R. Yan, G. Rukhovich, Z. Ferdinand, V. Venkataramani, A. Suwala, I. Bludau, S. Krieg, W. Wick, A. von Deimling, D. Heiland, M. Gerstung, B. Patel, F. Sahm
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most lethal brain tumor in adults with remarkable cellular plasticity and complex anatomical features. While single-cell RNA sequencing (scRNA-seq) has revealed its cellular heterogeneity, the spatial cellular organization within the anatomical features remains elusive.



 Here, we provide a comprehensive spatial transcriptomic dataset comprising 115,914 transcriptomes from 32 genotyped tissue sections.



 Leveraging Xenium in situ technology, we generated spatial transcriptomic data with subcellular-resolution, significantly improving the granularity of our understanding of GBM cellular architecture. In addition, our analyses identified oligodendrocyte subtypes specifically localized to tumor cores and perivascular regions with molecular profiles associated with poor prognosis.



 This publicly accessible GBM atlas provides a valuable resource for understanding the complex cellular architecture of GBM and facilitating the development of novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1717e2c5370026ee08f9c770891fbab0aaa81409" target='_blank'>
              P02.08.A A SPATIALLY RESOLVED HUMAN GLIOBLASTOMA ATLAS REVEALS DISTINCT CELLULAR AND MOLECULAR PATTERNS OF ANATOMICAL NICHES
              </a>
            </td>
          <td>
            H. Park, P. Sonpatki, C. Park, W. Park, N. Shah
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Spatial tumour heterogeneity, which denotes the changes in cellular and molecular attributes across distinct locations within a tumour, significantly influences cancer diagnosis and treatment resistance. The heterogeneity of tumour cells inside a singular mass facilitates tumour development, metastasis, and the ineffectiveness of standard therapy. Comprehending the geographical distribution of tumour cells is crucial for formulating more efficient treatment regimens. Diverse methodologies are employed to investigate spatial heterogeneity, encompassing modern imaging techniques such as MRI, PET, and multiplexed imaging, alongside omics approaches including genomes, transcriptomics, and proteomics. These instruments offer insights into the tumour microenvironment and facilitate the identification of resistant subpopulations. The amalgamation of imaging and genomic data via radiogenomics has emerged as a viable methodology, providing an extensive perspective on the spatial and molecular intricacies of tumours. Principal findings reveal that spatial heterogeneity fosters medication resistance by establishing microenvironments characterised by varying oxygen levels, immunological infiltration, and genetic alterations, hence complicating the efficacy of monotherapy strategies. Hypoxic environments and immunological evasion significantly contribute to treatment resistance. Addressing geographical heterogeneity has the potential to enhance cancer treatments. By analysing the molecular and geographical characteristics of tumours, physicians can customise therapies more efficiently, minimising resistance and improving therapeutic results. This methodology signifies a vital advancement in precision medicine, providing more individualised and efficacious cancer therapies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691858c0b45e6f3db66c28fcbc5355f8258e6fea" target='_blank'>
              Spatial Tumor Heterogeneity: The Next Frontier in Understanding Cancer Resistance
              </a>
            </td>
          <td>
            Kwesi Egyin Taylor, Hycent Jacob, T. Oladosu, Godwin Kenechukwu Nwajiugo, Motunrayo Victoria Adigun, Markus Saerimam Nzunde, C. Ugo
          </td>
          <td>2025-10-18</td>
          <td>Oncology, Nuclear Medicine and Transplantology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d38817463882bea59fae7ae417ac7b28c60ff93" target='_blank'>
              Multi-omics evaluation of cell lines as models for metastatic prostate cancer
              </a>
            </td>
          <td>
            Xueying Liu, Weixing Yu, Xiuyuan Jin, Yugang Wang, Ke Liu
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic non-small cell lung cancer (mNSCLC) cells carry heterogeneity, not only among different subtypes but also within a single tumor. Most evidence suggests that mNSCLC exploits specific molecular drivers and mechanisms to maintain physiology, metabolism, and immune evasion during tumorigenesis. Genome-wide association studies also revealed particular mutations in the oncogenic drivers supporting tumor cell proliferation and survival, resulting in aggressive and drug-resistant phenotypes of mNSCLC. While significant progress has been made in understanding mNSCLC at the genetic and molecular levels, a considerable gap remains in understanding the dynamic interplay between intrinsic factors—particularly key tumor-associated cells—and tumor immune microenvironment (TIME) during metastasis. Hence, this review highlights histological and genetic characteristics, emphasizes the clinical relevance of metastasis, and the roles of tumor-associated cells in shaping the immunosuppressive tumor microenvironment (TME) in mNSCLC. Understanding these intricate features and mechanisms is crucial for identifying novel therapeutic targets and improving strategies to combat mNSCLC progression in diagnosed patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931263caf4b3388e1caa012daaf5acb9be29f764" target='_blank'>
              Unraveling the molecular-pathological characteristics and cellular complexity of the tumor immune microenvironment in metastatic non-small cell lung cancer
              </a>
            </td>
          <td>
            S. Bharadwaj, J. M. Mierzwicka, Lucie Vaňková, Petr Malý
          </td>
          <td>2025-09-29</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC. Methods Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene. Results Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes. Conclusion This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The global burden of cancer is rising, with treatment failures often due to the metastatic nature of late-stage malignancies. Circulating tumour cells (CTCs) are metastatic precursors shed from primary tumours, which survive in circulation, extravasate and colonise distant organs. The advent of high-throughput single-cell RNA sequencing (scRNA-seq) has revolutionised the investigation of transcriptomic landscape at single-cell resolution, enabling deep transcriptomic profiling, re-stratifying CTC subtypes and improving the detection of rare new subpopulations. Applications extend to understanding tumour microenvironments, characterising cellular heterogeneity, uncovering metastasis molecular mechanisms and improving prognosis and diagnostic strategies. A timeline of key milestones in CTC scRNA-seq research is also provided. Nevertheless, a knowledge gap remains due to unstandardised protocols and fragmented resources in CTC scRNA-seq research. We address this gap by proposing a 12-step CTC-specific scRNA-seq workflow to overcome methodological inconsistencies. This workflow spans the entire process from enrichment, single-cell sorting and sequencing to data pre-processing and downstream analyses, with a detailed compilation of data analysis tools. An in-depth discussion of the pros and cons of commonly used scRNA-seq tools is also included, specifically evaluating their suitability for CTC research. Additionally, emerging research frontiers, including the discovery of hybrid cells—fusion products of tumour and normal cells—and the integration of machine learning (ML) into scRNA-seq workflows, are explored. Future research should prioritise CTC scRNA-seq workflow standardisation, integrate ML-driven analysis and investigate rare and hybrid populations to advance metastasis research. This review supports these goals by guiding methods, informing tool selection and promoting data sharing for reproducibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d2d946c12a3009915b3b21748cb811af18ab2d4" target='_blank'>
              Single-cell RNA-sequencing of circulating tumour cells: A practical guide to workflow and translational applications
              </a>
            </td>
          <td>
            Francis Yew Fu Tieng, Learn-Han Lee, N. Ab Mutalib
          </td>
          <td>2025-10-06</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c7fc26d95585d6b0934d32560acffc12bc3781e" target='_blank'>
              Region-specific human brain organoids reveal synaptic and cell state drivers of glioblastoma invasion
              </a>
            </td>
          <td>
            Tarun N. Bhatia, Saisrinidhi Ganta, Mostafa Meselhe, Caitlin Sojka, A. Martija, Lisa Nieland, Uriel Rufen-Blanchette, Anson Sing, Alexia King, Brain Organoid Hub, A. Bhaduri, Kimberly Hoang, E. Nduom, Renee D. Read, Jeffrey Olson, Steven A. Sloan
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6214e46642ec3f3780a0b2d3254221a9b00d6" target='_blank'>
              Comparative single-cell transcriptomics of orthotopic and subcutaneous gastric tumors reveal immune and stromal heterogeneity
              </a>
            </td>
          <td>
            Jinho Jang, Yoojeong Seo, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 The tumour microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) is comprised of heterogenous malignant epithelium and diverse immune infiltrates within a dense, fibrotic stroma. This multi-compartmental TME includes myeloid and lymphoid lineages, cancer-associated fibroblasts (CAFs), mural and endothelial cells, and tumour cells embedded in a rich extracellular matrix (ECM). Efforts to profile and perturb compartments or cellular phenotypes in isolation have often led to unintended adverse effects on neighbouring cells, compromising drug efficacy. This drives the need for an integrated, spatially-resolved understanding of the architecture and molecular features underlying the organization of the TME. To address this, we leveraged single-cell RNA sequencing (scRNAseq) data from eight distinct datasets to design three custom multiplexed imaging mass cytometry (IMC) panels against tumour, immune, and stromal targets. These were applied to three serial sections of a tissue microarray of 221 PDAC patients. Segmentation and clustering of 7.9 million cells revealed 83 distinct cell types and their functional states, including biophysical responses. Advanced image alignment across serial sections enabled the incorporation of TME-wide features and resolved eight spatially recurrent PDAC microenvironments neighbouring phenotypically distinct epithelial ducts. Two microenvironments, ECM-rich and immune-suppressed, were associated with worse overall survival and frequently co-occurred with a third, stiff matrix, characterized by stroma enriched in phospho-myosin light chain 2 expressing CAFs. In contrast, the immune infiltrated stroma microenvironment was linked to improved clinical outcomes and often co-occurred with CD105+ fibrovascular and immune infiltrated microenvironments. Matched whole-genome sequencing (n = 182/221) revealed that patients dominant in ECM-rich regions harboured 12p11.21 (KRAS) and 8q24.21 (MYC and POU5F1B) amplifications, and 17q22 (RNF43) deletions, and stiff matrix were associated with 13q33.3 (LATS2) deletion. To link spatial niches to proteomic signatures, we performed whole-slide IMC-guided laser capture microdissection and mass spectrometry, and further integrated differentially expressed proteins with transcriptomes from an external scRNAseq cohort (n = 163). ECM-rich and immune suppressed regions showed signatures of mechanically induced immune suppression, including NETosis and innate immune surveillance via pattern recognition receptors. Contrastingly, immune infiltrated microenvironments were enriched for cytotoxic activity and immune trafficking programs, while immune infiltrated stroma displayed signatures associated with oxidative stress responses. These findings define unique spatial ecosystems within the PDAC TME that are linked to specific genomic alterations, functional states and clinical outcomes, highlighting their therapeutic relevance and potential as targets to overcome the challenges of treating PDAC.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steven Gallinger, Hartland W. Jackson. Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B115.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e942fb4c97db1a6398873aeae226bf3656a085" target='_blank'>
              Abstract B115: Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Z. Lin, Edward L. Y. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 PDAC is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells cope with poor access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells, receiving alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids from fibroblasts. However, the extent of metabolic support provided by other cell types in the PDAC TME is unexplored. We reasoned that metabolically cooperative cell types would be spatially positioned near cancer cells in these tumors, and that cooperative spatial arrangements would be enriched under conditions of relative hypovascularity. We first performed Xenium spatial transcriptomics on a cohort of 8 treatment-naive human PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. Niche analysis uncovered 17 recurrent cellular neighborhoods in the data, including an endothelial-enriched niche. We find that tumor-associated neutrophils form dense aggregates within malignant glands in tumors with low endothelial content, and form in tumor regions which are most distal from vasculature. In a separate cohort of 4 additional PDAC tumors, the expression of neutrophil chemokines CXCL1/2/3/5 was highest in a subpopulation of malignant cells distant from endothelium. These data suggest metabolic stress secondary to poor vascularity drives recruitment of neutrophils via the CXCL1/2/3/5 axis. Indeed, when exposed to Tumor Interstitial Fluid Media (TIFM), cancer cells upregulated the transcription and/or secretion of these cytokines and G-CSF. Since the formation of neutrophil aggregates is driven by cancer cell nutrient derangement, we hypothesized that cancer cells could cooperate metabolically with neutrophils to alleviate nutrient stress driven by poor vascularity. Neutrophil co-culture promoted cancer cell proliferation under TIFM, but not RPMI, while under lethal combination TIFM and serum starvation, co-culture enabled enhanced survival of cancer cells. Lastly, to understand what metabolic processes could be fueled by neutrophil aggregates in pancreatic cancer, we designed a fully custom spatial transcriptomics panel targeting ∼360 metabolic enzymes and profiled 4 additional PDAC cases, performing dispersed whole-transcriptome scFFPEseq on an adjacent tissue section. We integrated data across the spatial and dispersed modalities to enable spatial projection of whole-transcriptome-derived information. Cancer cells located near neutrophil aggregates had significantly enhanced expression of vitamin B6 salvage pathway, suggesting that one critical shared metabolite between neutrophils and cancer cells is the B6 precursor pyridoxine. In total, we have deployed multiple spatial and single cell RNA profiling technologies to uncover a novel node of metabolic crosstalk in pancreatic cancer between neutrophils and cancer cells. Future work is needed to confirm transfer of specific metabolites and the downstream sequelae of their provision for tumor growth.



 Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M Celeste. Simon. Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec70e5707c093ceceed418e17d0f9e9a3b6c60c" target='_blank'>
              Abstract B127: Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M. C. Simon
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are critical mediators of tumor-stroma crosstalk in pancreatic ductal adenocarcinoma (PDAC). While bulk and single-cell transcriptomic studies have uncovered CAF heterogeneity, the spatial organization of CAF subtypes and their physical interactions with tumor epithelial subtypes remain incompletely understood. Here, we integrate bulk, single-cell, and spatial transcriptomic datasets to systematically interrogate the molecular, cell, and spatial interactions between tumor-promoting CAFs (proCAFs), tumor-restraining CAFs (restCAFs), and basal-like versus classical tumor subtypes across multiple biological scales. Ligand-receptor analysis of single-cell RNA-seq data revealed robust signaling between proCAFs and basal-like tumor cells, particularly through integrin-mediated (COL11A1–ITGA11) pathways, suggesting subtype-specific stromal-epithelial crosstalk within distinct tumor microenvironments. Histopathologic validation via immunofluorescence staining confirmed that proCAF regions are preferentially located adjacent to tumor epithelium and are characterized by reactive, myxoid stroma, whereas restCAF regions exhibit a fibrous, hypocellular morphology. To further elucidate spatial relationships, we performed 10x Genomics Visium spatial transcriptomics on seven primary PDAC tumors. Application of the DeCAF classifier to Visium spots demonstrated a distinct radial gradient of proCAF expression, peaking at the tumor-stroma interface and diminishing with distance—indicative of spatial colocalization with basal-like programs. Notably, proCAF gene signatures were specifically enriched in stromal regions adjacent to basal-like tumor zones, while classical tumor regions were spatially associated with heterogeneous CAF compositions. This subtype-specific spatial organization and paracrine signaling may underlie the clinical observation that basal-like tumors are significantly more likely to co-occur with proCAF stroma (64.5%, p = 2.17e–5). Importantly, patients harboring both basal-like and proCAF subtypes exhibited the poorest clinical outcomes (median overall survival: 11.0 months), compared to those with classical/restCAF profiles (median overall survival: 30.4 months, p < 0.001). Collectively, these multidimensional data define spatially organized and functionally interactive tumor-stroma niches in PDAC. Our findings highlight a co-evolutionary relationship between proCAFs and basal-like tumor cells that contributes to an aggressive disease phenotype, and underscore the potential of spatially resolved tumor-stroma profiling for improving patient stratification and guiding therapeutic targeting of the tumor microenvironment.



 Xianlu Laura. Peng, Ian McCabe, Elena Kharitonova, Ryan Zhao, Changfei Luan, Arthi Hariharan, Jaewon Lee, Nancy Kren, Yi Xu, Joseph Kearney, Justin Su, Michelle LaBella, Susan Tsai, Brian Belt, Roheena Panni, David Linehan, Yuliya Pylayeva-Gupta, Alina Iuga, Naim Rashid, Jen Jen Yeh. Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B100.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6682e5a69c7a44f5aef0626bbff15428b91a16" target='_blank'>
              Abstract B100: Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications
              </a>
            </td>
          <td>
            X. Peng, Ian C McCabe, Elena V Kharitonova, Ryan T Zhao, C. Luan, Arthi Hariharan, Jaewon J Lee, Nancy P. Kren, Yi Xu, Joseph F Kearney, Justin Su, M. LaBella, Susan Tsai, Brian Belt, R. Panni, David Linehan, Y. Pylayeva-Gupta, Alina C Iuga, Naim U Rashid, J. J. Yeh
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) are pivotal stromal components that shape the tumor microenvironment through their remarkable heterogeneity and dynamic functions. This review comprehensively examines the biology of CAFs, starting with their diverse cellular origins and induction mechanisms via key signaling pathways. We highlight the spatial‐temporal heterogeneity of CAF subsets and propose an updated classification system that encompasses eight functionally distinct subtypes based on recent single‐cell transcriptomic studies. The tumor‐promoting roles of CAFs are explored in depth, including their contributions to tumor behavior through novel mechanisms such as neutrophil extracellular traps formation, and multidrug resistance via metabolic reprogramming. A major focus is placed on CAF‐mediated immunotherapy resistance through immune checkpoint regulation, recruitment of immunosuppressive cells, and metabolite secretion. This review also highlights key technologies that are advancing CAF research, including patient‐derived 3D models, artificial intelligence‐enhanced spatial multiomics, clustered regularly interspaced short palindromic repeats (CRISPR)‐engineered mouse systems, and machine learning approaches for analyzing CAF heterogeneity and function. Finally, we evaluate therapeutic strategies targeting CAF activation, secreted factors, and immune crosstalk, while critically analyzing ongoing clinical trials. Overall, this review not only clarifies the biology of CAFs but also provides a translational framework for developing next‐generation stromal‐targeted therapies to overcome treatment resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1dae86be5576422570b1acdcb11b97738bdec4b" target='_blank'>
              Cancer‐Associated Fibroblasts: Origin, Classification, Tumorigenicity, and Targeting for Cancer Therapy
              </a>
            </td>
          <td>
            Chang Fan, Wenlong Zhu, Yuan Chen, Wanwan Zhu, Jin Ding
          </td>
          <td>2025-10-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Spatial ‘omics technologies are a powerful tool for mapping the relationship between cellular organization and molecular distributions in healthy and diseased tissue microenvironments. Here, we describe a novel multimodal pipeline that represents experimental and computational advances for spatiomolecular analysis of tissue samples across molecular classes. This adaptable method integrates matrix-assisted laser desorption/ionization (MALDI) imaging mass spectrometry (IMS) lipidomics, spatial transcriptomics (ST), multiplexed immunofluorescence microscopy (MxIF), and histopathological staining to uncover spatiomolecular profiles associated with unique cellular niches and pathological features. We demonstrate the power of this approach using two different complex human disease systems: Alzheimer’s disease in human brain tissue and type 2 diabetes mellitus in the human pancreas. By identifying molecular markers associated with disease pathology in the pancreas and brain, we shed light on biologically significant pathways that are impacted in these two spatially complex diseases and highlight the powerful potential of accurate, high-resolution multimodal integration approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bef9fffe5097573bdbf2bf38a7abae993ad7ba6a" target='_blank'>
              Integrative Spatial Omics for Systems-Level Mapping of Pathological Niches
              </a>
            </td>
          <td>
            A. Kruse, Roy Lardenoije, L. Migas, Claire F. Scott, Cody R. Marshall, Morad C. Malek, Adel Eskaros, Thai Pham, Kristie I. Aamodt, Madeline E. Colley, L. Ventura-Antunes, Melissa A. Farrow, R. Van de Plas, Joana Goncalves, M. Schrag, Alvin C. Powers, Jeffrey M. Spraggins
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in shaping immune responses and therapeutic outcomes in lung cancer, yet the diversity and functional specialization of tumor-associated macrophages (TAMs) remain poorly resolved. Here, we present a refined classification of TAM subtypes across large cohorts of cancer datasets using integrative analysis of single-cell RNA sequencing, spatial transcriptomics, and clinical datasets from lung adenocarcinoma and lung squamous cell carcinoma. By combining cell-based gene scoring with hierarchical classification, we defined 7 macrophage subtypes-each with distinct transcriptional programs and abundances. Notably, lipid-associated TAMs expand with disease progression and exhibit immunosuppressive and protumorigenic features, whereas tissue-resident macrophages decline. Spatial and survival analyses reveal that an increased lipid-associated to tissue-resident TAM ratio correlates with advanced disease and poor prognosis. Given that spatial transcriptomic assays rely on deconvolution techniques to infer cell type compositions, accurate gene expression signatures are essential, especially for fine-grained subpopulations of TAMs. Our refined subtype-specific signatures address this bottleneck and enhance the resolution of spatial mapping efforts. These findings offer new insights into macrophage heterogeneity and highlight lipid-associated TAMs as potential biomarkers and therapeutic targets in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e49c52a60b0a9da9a722e72cb8391f58dc458b" target='_blank'>
              Transcriptional states of lung cancer microenvironment reveal macrophage subtype dynamics linked to disease progression.
              </a>
            </td>
          <td>
            Duygu Keremitçi, Özlem Tuna, Aissa Houdjedj, Hilal Kazan, Yasin Kaymaz
          </td>
          <td>2025-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background Renal Cell Carcinoma (RCC) has been characterized as being amongst the most immune infiltrated solid tumors with a highly heterogenous immune landscape. Within spatially organized cellular networks (CNs) of the tumor immune microenvironment (TIME), key immune cell-cell interactions (CCIs) impact immune cell function and organization ultimately impacting the patient’s overall response. Multiple studies have observed an association of tertiary lymphoid structures, a commonly observed spatial CN, with positive clinical outcomes in RCC, however additional CNs and the CCIs that control these networks need to be identified to better harness and potentially reprogram immune responses to improve patient outcomes. The recent explosion of interest in the heterogeneity of the immune landscape in RCC has led to numerous publications using the latest technologies in spatial transcriptomics, proteomics, and metabolomics. However, many of these studies use this data in isolation and therefore, may be hindered by the technological biases inherent in each method. Here, we have developed a novel approach to integrate spatial and single cell multi-omic data harnessing the strengths of each technology to better interrogate CNs that exist in the RCC TIME. Methods Fresh surgical samples were procured at the University Health Network (Toronto, Canada) through the REnal cancer MicroEnvironment DiscoverY (REMEDY) study. Bulk RNA sequencing (RNA-seq), single cell RNA sequencing (scRNA-seq), single cell suspension mass cytometry (SMC), whole transcriptome digital spatial profiling (DSP), and imaging mass cytometry (IMC) was performed on spatially concordant tumor regions across 54 patients. scRNA-seq enabled the identification of immune, stromal, and malignant high-resolution cell states, which informed a tailored antibody panel design for SMC and IMC and served as a reference framework for harmonized cell-type annotation across modalities. This enabled the integration of our transcriptomic and proteomic data to delineate RCC-specific CNs enriched for defined CCIs across unique biological pathways. Results Using this integrative approach, we identified seven high-resolution patient immunophenotypes. To evaluate their prognostic and predictive relevance, we derived representative gene signatures using a linear mixed model (Flash-MM) to interrogate publicly available bulk RNA-seq datasets, including TCGA, JAVELIN, and IMmotion151. This analysis revealed immunophenotype-specific associations with survival following surgery or systemic therapy in univariate models. To explore potential biological mechanisms associated with these divergent clinical outcomes, we incorporated spatial information into traditional CCI analyses and performed pathway analyses to define functional relationships. This revealed that patient subtypes with high lymphoid infiltration exhibit greater spatial heterogeneity, potentially reflecting the coexistence of multiple activated immune pathways. In contrast, patient subtypes with low immune infiltration were enriched in more developmental signaling pathways. In addition to our biological observations, we were also able to assess the technological differences between patient matched samples and compare the ability of each technology to capture inter-patient and intra-patient heterogeneity. Conclusions Collectively, this work identifies clinically distinct subgroups defined by CNs and details the interpatient cellular heterogeneity that exists in RCC, providing the foundation for future personalized therapeutic interventions against this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e76f0bf068f59ffeffc899012c151c0a8ca8df5b" target='_blank'>
              37Integrative multi-omic characterization of the immune landscape in renal cell carcinoma
              </a>
            </td>
          <td>
            Jennifer Pfeil, Shirley Hui, Daniel Stueckmann, Xiaoyu Zhang, Lisa Martin, Jennifer L. Gorman, Eddy Chen, Somi Afiuni, Jalna Meens, M. Komisarenko, Stéphane Chevrier, Sujana Sivapatham, Julia Szusz, Zhihui Liu, Susan Prendiville, Laurie Ailles, Philip Jonsson, Fred Davis, Cristina Penaranda, Ryan Newton, N. Stransky, Piotr Bielecki, Gromek Smolen, Masoom Haider, B. Bodenmiller, Sarah Crome, Gary Bader, Anthony Finelli, Hartland W. Jackson, Keith A. Lawson
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Lineage plasticity, the ability of cells to transition to an alternative phenotype as a means for adaptation, is an increasingly recognized mechanism of tumor evolution and a driver of resistance to anticancer therapies. The most extensively described clinical settings impacted by such molecular phenomena include neuroendocrine transformation in androgen receptor-dependent prostate adenocarcinoma, and adenocarcinoma-to-neuroendocrine and adenocarcinoma-to-squamous transdifferentiation in epidermal growth factor receptor-driven lung adenocarcinoma, affecting 10%–20% of patients treated with targeted therapy. Recent analyses of human tumor samples and in vivo models of histological transformation have led to insights into the biology of lineage plasticity, including biomarkers predictive of high risk of transformation. However, no clinically available therapies aimed to prevent or revert plasticity are currently available. In the present review, we will provide a biological and therapeutic overview of the current understanding of common and divergent molecular drivers of neuroendocrine and squamous transdifferentiation in tumors from different origins, including descriptive analysis of previously known and recently described molecular events associated with histological transformation, and propose evidence-based alternative models of transdifferentiation. A clear definition of the commonalities and differences of transforming tumors in different organs and to different histological fates will be important to translate molecular findings to the clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell Research</td>
          <td>2</td>
          <td>17</td>
        </tr>

        <tr id="Spatially resolved transcriptomics (SRT) facilitates the study of cell–cell interactions within native tissue environments. To support method development and benchmarking, we introduce sCCIgen, a real-data-based simulator that generates high-fidelity synthetic SRT data with known interaction features. sCCIgen preserves transcriptomic and spatial characteristics and provides key interaction features, including cell colocalization, spatial dependence of gene expression, and gene–gene interactions between neighboring cells. It supports input from SRT data, single-cell expression data alone, and unpaired expression and spatial data. sCCIgen is interactive, user-friendly, reproducible, and well-documented for studying cellular interactions and spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cffc0895b0c59f6ae8d74318f266f33f6e00809" target='_blank'>
              sCCIgen: a high-fidelity spatially resolved transcriptomics data simulator for cell–cell interaction studies
              </a>
            </td>
          <td>
            Xiaoyu Song, J. C. Chávez‐Fuentes, Weiping Ma, Weijia Fu, S. C. Shin, Pei Wang, Guo-Cheng Yuan
          </td>
          <td>2025-10-15</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5994676ec22291e594d0de0ef39f5ccc905b6ab" target='_blank'>
              Integrative analysis of spatiotemporal transcriptomics delineates dynamic cell states in squamous tumorigenesis
              </a>
            </td>
          <td>
            Varsha Sreekanth, Timur Rusanov, Farah Yousef, Mark A Sanborn, Su Yeon Yeon, Vijayalakshmi Ananthanarayanan, Olga Karginova, Jalees Rehman, A. Salahudeen
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Differentiating between the two main subtypes of Inflammatory Bowel Disease (IBD): Crohn’s disease (CD) and ulcerative colitis (UC) is a persistent clinical challenge due to overlapping presentations. This study introduces a novel computational framework that employs spatial transcriptomics (ST) to create an explainable machine learning model for IBD classification. We analyzed ST data from the colonic mucosa of healthy controls (HC), UC, and CD patients. Using Non-negative Matrix Factorization (NMF), we first identified four recurring cellular niches, representing distinct functional microenvironments within the tissue. From these niches, we systematically engineered 44 features capturing three key aspects of tissue pathology: niche composition, neighborhood enrichment, and niche-gene signals. A multilayer perceptron (MLP) classifier trained on these features achieved an accuracy of 0.774 ± 0.161 for the more challenging three-class problem (HC, UC, and CD) and 0.916±0.118 in the two-class problem of distinguishing IBD from healthy tissue. Crucially, model explainability analysis revealed that disruptions in the spatial organization of niches were the strongest predictors of general inflammation, while the classification between UC and CD relied on specific niche-gene expression signatures. This work provides a robust, proof-of-concept pipeline that transforms descriptive spatial data into an accurate and explainable predictive tool, offering not only a potential new diagnostic paradigm but also deeper insights into the distinct biological mechanisms that drive IBD subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7e36d179c516512e6f46f88d6a580c0cc5c9f06" target='_blank'>
              Engineering Spatial and Molecular Features from Cellular Niches to Inform Predictions of Inflammatory Bowel Disease
              </a>
            </td>
          <td>
            M. J. T. Tan, Maria Kapetanaki, P. Benos
          </td>
          <td>2025-09-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4113c46dcadcc5e605fb92cfb97a958b41a5f2" target='_blank'>
              High-resolution spatial transcriptomics uncover epidermal-dermal divergences in Merkel cell carcinoma: spatial context reshapes the gene expression landscape.
              </a>
            </td>
          <td>
            Kuan Cheok Lei, Nalini Srinivas, Mitalee Chandra, Vahan Serobyan, Selma Ugurel, Daniel Hoffmann, T. Kervarrec, Weng-Onn Lui, Jürgen C. Becker
          </td>
          <td>2025-10-23</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Glioblastoma (GB) is a particularly aggressive brain tumor which is characterized by a low overall survival due to rapid formation of neuron-tumor-networks. The neural epigenetic subtype was linked to increased tumor-neural connectivity but the mechanisms underlying this integration remain poorly understood. Here, we aim to investigate the impact of the microenvironmental ecosystem on neuron-tumor synaptogenesis leveraging a human cortical slice model and single cell spatially resolved transcriptomic analysis.



 A human cortical slice model was employed to visualize and assess tumor-neuronal interactions using advanced retrograde tracing techniques in human derived cell lines (n = 12). This system was based on an EnvA-pseudotyped G-protein-deleted rabies virus. Spatial transcriptomics was conducted using multiplexed error-robust fluorescence in situ hybridization analysis (MERFISH). Additionally, we conducted immunohistochemical staining and confocal microscopy.



 Spatial transcriptomic evaluation of neural high and low tumors revealed distinct microenvironmental ecosystems within the infiltrating rim of the tumors. Neural-high tumors exhibited a higher presence of synaptogenesis-inducing inflammatory myeloid cells driving neuronal synaptogenesis. Retrograde tracing of neural high vs low cell lines confirmed increased synaptic density and neuronal coupling in the neural high phenotype, which were associated with higher invasiveness, supported by stRNA-seq, which identified genes linked to synaptogenesis and connectivity.



 Neural-high GB shows enhanced neuronal connectivity and invasiveness, contributing to accelerated tumor progression, which underlines the prognostic value of epigenetic tumor profiling and opens ways for novel therapeutic approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab32a7b8c0ed1e91d6c055b2bda26b2946992d4" target='_blank'>
              P02.15.B GLIOBLASTOMA IN THE CONNECTOME: NEURAL PHENOTYPES PREDICT TUMOR INTEGRATION AND CONNECTIVITY
              </a>
            </td>
          <td>
            L. Menzl, G. Villa, Y. Yabo, V. Venkataramani, F. Ricklefs, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74e30f87b2d3e252ed12332d86f09f04feb4c91" target='_blank'>
              CTDP: Identifying cell types associated with disease phenotypes using scRNA-seq data
              </a>
            </td>
          <td>
            Chonghui Liu, Murong Zhou, Zhidong Wang, Chen Yang, Yan Zhang, Zhongjun Jiang, Yu-Hang Yin, Zeyu Luo, Tianjiao Zhang, Guohua Wang, Lei Yuan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Tumor resident microbes are a well-recognized component of the tumor microenvironment. Microbial subcellular location within different tumor types along with their functionality remains to be determined. Bulk microbial profiling techniques, however, lack subcellular and spatial transcriptomic resolution. To address these limitations, we developed advanced quantitative fluorescent imaging methodologies that allowed for visualization of microbial cellular compartmentalization across different tumor types. Furthermore, we performed spatial microbial transcriptomics at the regional and single cell levels to interrogate microbial regulation of signaling within pancreatic cancer cells. To confirm presence of viable microbes within tumors, we performed multiplexed culturomics of patient tissue samples followed by Whole Genomic Sequencing (WGS) analysis. We used in vitro co-culture assays together with in vivo animal experiments for functional biological validation. Overall, our results identified several pathways under microbial regulation within cancer cells that can drive immune evasion through impaired antigen presentation. In summary, using multiple complimentary novel methodologies we characterize the microbial niche of tumors (MiNT) that uncovers microbial regulation of host cells signaling and reshaping of the surrounding tumor microenvironment.



 Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier Gomez, Steven B. Maron, Daniel Carvajal-Hausdorf, Erick M. Riquelme, Mark W. Hurd, Anirban Maitra, Luis Diaz, Ismet Sahin, Jared Burks, Michael P. Kim, Florencia McAllister. Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B011.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d435ae53244dd8930ff7ec47bc24ea3f53880a" target='_blank'>
              Abstract B011: Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment
              </a>
            </td>
          <td>
            Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier A Gomez, S. Maron, Daniel Carvajal-Hausdorf, Erick Riquelme, M. Hurd, Anirban Maitra, Luis A. Diaz, I. Sahin, Jared K. Burks, Michael P Kim, Florencia McAllister
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3948f28751f8205d2389f7d1df87013dbea193d3" target='_blank'>
              RUNX1T1-HDAC Reprogramming of the HOX Code Signaling Drives a Targetable Pan-Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, Catherine Yijia Zhang, Xiao Jin, Yaru Xu, I. Y. Kim, Ping Mu
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Cervical biofluids collected during routine Pap smears contain a mixture of epithelial and immune cells that reflect cervical health and disease. This study demonstrates proof-of-concept that multicolor flow cytometry can be applied to these samples to identify and characterize cell populations in normal cervical samples as well as in different stages of cervical cancer. Our analysis identified major cell populations, including different types of epithelial cells and immune cells, and revealed that their composition within the samples changes with disease stage. We observed variability in both cellular composition and marker expression, highlighting the need for additional markers and larger patient cohorts. Nonetheless, this minimally invasive approach could potentially be used to monitor tumor cells, immune responses, and therapeutic outcomes over time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/565b70c9fdb10e2ce7ca145493da61430604f525" target='_blank'>
              Noninvasive Cell Population Profiling of Normal and Dysplastic Cervical Biofluids by Multicolor Flow Cytometry as a Promising Tool for Companion Diagnostics
              </a>
            </td>
          <td>
            Christoph Berger, Wolf Dietrich, Manuela Richter, Florian Kellner, Christian Kühne, Katharina Strasser
          </td>
          <td>2025-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-throughput clonal tracing of primary human samples relies on naturally occurring barcodes, such as somatic mitochondrial DNA (mtDNA) mutations detected via single-cell ATAC-seq (mtscATAC-seq). Fresh-frozen clinical specimens preserve tissue architecture but compromise cell integrity, thereby precluding their use in multi- omic approaches such as mitochondrial genotyping at single-cell resolution. Here, we introduce Cryo-mtscATAC-seq, a broadly applicable method for diverse pathophysiological contexts to isolate nuclei with their associated mitochondria (“CryoCells”) from frozen samples for high-throughput clonal analysis. We applied Cryo-mtscATAC-seq to the neurodegenerated human brain, glioblastoma (GBM), pediatric neuroblastoma, and human aorta, and implemented mitobender, a computational tool to reduce ambient mtDNA in single-cell assays. Our approach revealed regional clonal gliogenesis and microglial expansions in amyotrophic lateral sclerosis (ALS), persistence of oligodendrocyte progenitor cell (OPC)-like clones in GBM recurrence, mtDNA depth heterogeneity after neuroblastoma chemotherapy, and oligoclonal proliferation of smooth muscle cells in human aorta. In conclusion, Cryo-mtscATAC-seq broadly extends mtDNA genotyping to archival frozen specimens across tissue types, opening new avenues for investigation of cell state- informed clonality in human health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c60dee046cd06ab111daf9f8ce07ff1e2d339ed7" target='_blank'>
              Cryo-mtscATAC-seq for single-cell mitochondrial DNA genotyping and clonal tracing in archived human tissues
              </a>
            </td>
          <td>
            Maren Salla, B. Obermayer, Marie Cotta, Ekaterina Friebel, Juliana Campo-Garcia, Georgia Charalambous, Roemel Jeusep Bueno, Dustin Lieu, Patryk Dabek, Ashley Helmuth, G. Tellides, Roland Assi, Katrin Bankov, Marco Lodrini, H. Deubzer, D. Beule, Hattie Chung, Helena Radbruch, David Capper, Frank L. Heppner, Sarah C. Starossom, Caleb A. Lareau, Ilon Liu, Leif S. Ludwig
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08fb4029b669b488e3f7a2bf959803989f3d7dae" target='_blank'>
              Single-Cell RNA and ATAC Sequencing Reveal Cellular Heterogeneity and Chromatin Accessibility Dynamics in Young Chinese Breast Cancer Patients across Pre- and Post-Neoadjuvant Therapy
              </a>
            </td>
          <td>
            Xue Han, Bin Wang, Wenting Xiang, Fan Lin, Bo Sun, Beibei Yang, Zhihan Ruan, Shasha Liu, Shuo Li, Hong Liu, Jian Liu
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification. Understanding the cellular origins of cancers is crucial for improving diagnosis and treatment. Here, the authors utilize single cell chromatin accessibility data, patient whole-genome sequencing mutational profiles, and machine learning to predict the cell of origin for 37 cancer types, providing insights into cancer development and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566fc9fd9c1a3c438b8fdc4f08cbb7f2ae09af61" target='_blank'>
              Joint analysis of expression and variation at single cell resolution by scVar
              </a>
            </td>
          <td>
            Wei Zhao, Fei Yang, Wenting Zong, Xinchang Zheng, Yiming Bao
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy with significant cellular heterogeneity, which poses a significant challenge for treatment. We integrated three single-cell RNA sequencing (scRNA-seq) datasets to construct an expression matrix, identify cell types, and assess copy number variation (CNV) scores, with a particular focus on ductal cells. Our study found that ductal cells are the primary source of malignant cells in PDAC and identified a high-malignancy subtype—ductal cell-3—where RACK1 was the most significantly differentially expressed gene. Through the analysis of public and clinical data, along with RT-qPCR and immunohistochemistry validation, we confirmed the high expression of RACK1 in PDAC tumors, which correlated with lymph node metastasis, clinical staging, histological differentiation, and prognosis. Further investigation revealed that RACK1 promotes PDAC cell proliferation, migration, and invasion. Sequencing analysis indicated that RACK1 inhibits the FOXO signaling pathway, promotes epithelial-mesenchymal transition (EMT), and facilitates the progression and metastasis of PDAC. This study provides new insights into the role of RACK1 in PDAC and its underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fac327fd52a6595ab14fc1ae00b1f2b7a0bcb1e" target='_blank'>
              Molecular mechanisms of RACK1-driven metastasis in pancreatic ductal adenocarcinoma revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Hangyu Liu, Tingxin Wang, Yuying Cui, Biao Zhang, Dong Shang, Huiyi Song
          </td>
          <td>2025-10-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Seminoma is the most common solid malignant tumor of the testis in young males, significantly impacting fertility. Approximately 20% of seminomas metastasize, markedly increasing recurrence risk and compromising quality of life. To investigate the poorly understood mechanisms driving seminoma metastasis, we performed a comprehensive analysis by integrating single-cell RNA sequencing, TCGA data mining, and cell biology assays. We identified significant intratumoral heterogeneity. A subset of tumor cells expressing DPPA4 and PSMA7 showed high stemness, enhanced self-renewal, and association with metastasis. Knockdown of these genes reduced sphere formation, tumor migration and proliferation in Tcam-2 and NCCIT cells. Conversely, tumor cells overexpressing PAGE5 and SAT1 exhibited reduced stemness, migratory capacity and proliferation. In the immune microenvironment, we identified IFNG+ T cells, which recruit and activate other antitumor immune cells. These cells secrete IFN-γ, which promotes tumor differentiation, reduces stemness, and mitigates tumor aggressiveness. Based on these findings, we further developed a molecular panel to aid in the identification of seminomas with a higher risk of metastasis. In conclusion, our study identifies unique molecular signatures that facilitate risk stratification based on metastatic potential, providing valuable insights for improving precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a200ee992d63f7f5d78cc885c7a3fb857907df52" target='_blank'>
              Single-cell analysis unravels divergent gene signatures shaping seminoma stemness and metastasis
              </a>
            </td>
          <td>
            Zhouliang Bian, Biying Chen, Jiahui Guo, Jiamin Zhang, Guangye Du, Shufang He, Haihua Yuan, Yue Zhou, Bin Jiang, Daliu Min, Dingwei Ye, Hengchuan Su, Yanjie Zhang
          </td>
          <td>2025-11-07</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Pancreatic cancer, specifically pancreatic ductal adenocarcinoma (PDAC), is notorious for its aggressive nature and dismal prognosis, ranking as a leading cause of cancer-related mortality worldwide. Despite advancements in surgical techniques, chemotherapies, and novel therapeutic approaches, the overall survival rates remain bleak, primarily due to the complex and dynamic tumor microenvironment (TME) which fosters cancer progression and therapeutic resistance. Central to the TME are cancer-associated fibroblasts (CAFs), which exhibit significant heterogeneity and play multifaceted roles in tumor progression, ranging from remodeling the extracellular matrix to modulating immune responses. Recent breakthroughs in single-cell sequencing and spatial omics have unraveled the diverse functional states and spatial organization of various cell types within the TME, including distinct subtypes of CAFs which differ vastly in their influence on pancreatic cancer pathophysiology. This review delves into the intricate interactions between CAFs and pancreatic cancer cells, highlighting how metabolic reprogramming and immune regulation by CAFs contribute to the high malignancy of PDAC. By integrating single-cell and spatial resolution data, we offer new insights into the metabolic crosstalk and immune landscape remodeling driven by CAFs, establishing a foundation for targeting these stromal interactions in therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2add2bd80d02d372a76aab262dddc13647852fe6" target='_blank'>
              Metabolic and immune crosstalk between cancer-associated fibroblasts and pancreatic cancer cells
              </a>
            </td>
          <td>
            Qiyao Zhang, Z. Cao, Shangcheng Yan, Bingrong Chen, Hao Wu, Hongtao Cao, Chen Lin, Ziwen Liu
          </td>
          <td>2025-10-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="

 Gastric cancer progression involves complex interactions among tumor cells, immune components, and stromal elements within the tumor microenvironment. However, a comprehensive understanding of cellular heterogeneity, spatial organization, and cell-cell communication in gastric cancer remains incomplete.



 Single-cell RNA sequencing was performed on 252, 399 cells from six tissue types, spanning gastritis, intestinal metaplasia, primary tumors, adjacent normal tissue, and metastatic lesions. Integration with spatial transcriptomics enabled spatial mapping of cellular interactions. Pseudotime, cell-cell communication, and transcriptional heterogeneity analyses were conducted. Tumor stage-associated gene modules were identified using Weighted Gene Co-expression Network Analysis (WGCNA) of The Cancer Genome Atlas (TCGA) data. Finally, a deep learning-based prognostic model was developed and externally validated.



 Our analysis revealed dynamic remodeling of the tumor microenvironment during gastric cancer progression, characterized by the expansion of dysfunctional CD8+ T cells, pro-tumorigenic fibroblasts (e.g., ITGBL1+, PI16+, and ITLN1+), and altered myeloid populations. Stromal-immune crosstalk, particularly fibroblast-driven immunosuppressive signaling, was prominent. Spatial transcriptomics revealed the colocalization of immune and stromal cells, supporting spatially organized cellular interactions. WGCNA identified a gene module (657 genes) associated with T cell, myeloid, and stromal alterations, as well as tumor stage. A deep learning model based on this gene set accurately stratified patients according to survival in both TCGA and independent validation cohorts. Risk scores were correlated with clinical features, including tumor stage and therapeutic response.



 Our integrative single-cell, spatial, and computational analysis provides a high-resolution map of gastric cancer microenvironment remodeling. We identified key stromal and immune subpopulations, extensive cellular communication networks, and spatial structures that collectively drive tumor progression and metastasis. The derived gene signature and prognostic model have the potential for clinical risk stratification and therapeutic targeting in gastric cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f147ce2dccf2934265aaef857f3e2b638aa838fb" target='_blank'>
              Integrative multi-omics analysis of gastric cancer evolution from precancerous lesions to metastasis identifies a deep learning-based prognostic model
              </a>
            </td>
          <td>
            Yulin Ren, Xiaoyan Zhang, Ke Li, Shuning Xu, L. Qiao, Qun Li, Cheng Zhang, Ying Liu
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4e23f0e2eb9b451cd70e81ca9b082a2f8b6469" target='_blank'>
              Advancements in single-cell sequencing for cervical cancer research.
              </a>
            </td>
          <td>
            Congli Pu, Biyuan Xing, Shujie Wang, Zhao Liu, Yingchao Zhao
          </td>
          <td>2025-11-06</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Background Pancreatic ductal adenocarcinoma (PDAC) is an exceptionally lethal malignancy, with high percents of patients presenting with liver metastases (LM). However, the mechanisms driving liver metastases remain critical bottlenecks requiring urgent exploration. Objective To identify the key cellular subsets driving PDAC liver metastases, elucidate their interactions with the metastatic microenvironment, and define the underlying mechanisms of liver colonization. Materials and Methods Integrated single‐cell transcriptomic analysis was performed using scRNA‐seq data of PT and LM. The expression of signature genes within the identified cell subset was validated using clinical samples from PDAC PT and LM patients. Furthermore, ligand‐receptor network analysis was conducted between the specific tumor cell subset and key immune cells. Results We identified a novel liver‐enriched metastatic subset (LEMS), a terminally differentiated malignant cell subpopulation characterized by metabolic reprogramming and hyperactivation of immunosuppressive pathways. We further validated the LEMS signature genes, oxidized low‐density lipoprotein receptor 1 (OLR1) and solute carrier family 7 member 7 (SLC7A7), as potential diagnostic biomarkers for liver metastases. Importantly, we found that SPP1+ macrophages interacted with LEMS via ligand‐receptor networks, thereby driving invasion and immune evasion. Discussion We revealed the highly malignant features of LEMS and crosstalk between LEMS and SPP1+ macrophages in liver metastases. However, it is necessary to expand clinical cohorts and in vivo models to comprehensively elucidate the specific mechanistic interactions between LEMS and macrophages. Conclusion We delineated LEMS as an enriched subset in LM and proposed targeting of LEMS‐SPP1+ macrophage interactions as a therapeutic strategy to disrupt metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6cf6b6420fa24913d08de22b378b6164ad08ae" target='_blank'>
              Single‐Cell Transcriptomic Analysis Identifies a Novel OLR1 + SLC7A7 + Liver‐Enriched Metastatic Subset With Immunometabolic Rewiring in Pancreatic Cancer
              </a>
            </td>
          <td>
            Xuan Yang, Xinyuan Chen, Zixin Wang, Yanfang Liu, Huiting Hu, Qingru Wu, Hailing Zhang, Yu Xiong, Xin Li, Xiaotao Cheng, Xiaoyu Ruan, Yan Gu
          </td>
          <td>2025-11-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Pancreatic cancer remains a lethal disease despite advances in surgery and systemic treatment in the last two decades, underscoring the urgent need to better understand its biological underpinnings. Despite remarkable advances in the molecular characterization of pancreatic ductal adenocarcinoma (PDAC), clinically actionable biomarkers remain scarce, and current treatment remains empiric. Transcriptomic subtypes such as “classical” and “basal-like” offer some prognostic value, but their ability to guide real-time treatment decisions is limited. In this review, we explore the limitations of current biomarker strategies, in particular subtype-based classifications, and argue for a functional reframing of biomarker development in PDAC, centered on patient-derived organoids (PDOs). We explore four key domains in which PDOs deepen our understanding of therapeutic response and resistance, namely, drug response phenotyping, modeling chemoresistance, incorporating tumor microenvironmental complexity through co-culture systems, and more functional profiling through proteomic and metabolomic approaches. Together, these applications move PDOs beyond static avatars of the tumor to dynamic platforms capable of capturing clinically relevant biology. As functional precision medicine gains traction, PDOs may offer a path to more tailored, responsive treatment strategies in a cancer where new options are urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243db33c2c4b51c3c745a2973296c1c4099759e6" target='_blank'>
              Functional Precision in Pancreatic Cancer: Redefining Biomarkers with Patient-Derived Organoids
              </a>
            </td>
          <td>
            C.A. Chew, Cheng Mun Wun, Yi Fang Lee, C.E. Chee, Khek Yu Ho, Glenn Kunnath Bonney
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Liver metastases represent a leading cause of mortality in pancreatic ductal adenocarcinoma (PDAC). Recent clinical evidence has revealed distinct immune ecosystems in the liver tumor microenvironment, yet mechanistic insight into the cellular and molecular mechanisms driving those diverse immune landscapes remains unclear. Here we report, computational cellular signaling analysis of single-cell RNA sequencing data from matched primary and liver metastatic samples isolated at early and late time points from the spontaneous genetically engineered mouse model KPC (Pdx1Cre; KrasLSL-G12D; Tp53LSL-R172H). This analysis uncovered several immunomodulatory axes between cancer cells-T cells and cancer cells- macrophages. Validation of these findings were performed using primary tumor cell lines, isolated from the same or littermate tumors as the scRNA-seq dataset. Bulk RNA-seq analysis of these cell lines showed a retention of cancer-derived ligands in culture. In vivo validation using the hemisplenic liver metastatic transplant model confirmed that cell lines isolated from primary tumors, which metastasized spontaneously to the liver, retained liver metastatic outgrowth potential. Flow cytometry and multiplex cytokine profiling of isolated cell lines, non-involved liver, and liver tumors revealed distinct immune population shifts within the metastatic niche and confirmed cancer cell derived immunomodulatory cytokines. This study highlights the interplay between cancer-intrinsic signaling and the immune landscape in PDAC liver metastases and supports future mechanistic exploration.



 Ayushi Mandloi, Jace Baines, Meet Patel, Christina R. Larson, Robert S. Welner, Julienne L. Carstens. Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A081.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fce5d9cdc25fe3d365bf9aa1c45eddedfbe9968" target='_blank'>
              Abstract A081: Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth
              </a>
            </td>
          <td>
            Ayushi Mandloi, Jace Baines, Meet Patel, Christina R Larson, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08665b0e37751f05ffbba7515282ba6d54a5cdc8" target='_blank'>
              Multiomic profiling of a unique in-transit melanoma cohort identifies melanoma differentiation as predictor of tumor progression and therapy response
              </a>
            </td>
          <td>
            Giuseppe Tarantino, Anne Zaremba, Tuulia Vallius, M. Woodnorth, Yichao Hua, R. Pelletier, Mariana Lopez Leon, Yingxiao Shi, Z. Maliga, Samira Makhzami, Tyler J. Aprati, Bojan Karlaš, V. Glutsch, Bastian Schilling, J. Hassel, Carola Berking, Jochen Utikal, F. Meier, F. Meiss, Lucie M. Heinzerling, Katharina Kähler, Jiajia Chen, L. Zimmer, A. Sucker, E. Livingstone, E. Hadaschik, Christine Lian, George Murphy, Yevgeniy R. Semenov, Genevieve M Boland, P. Sorger, Florian Rambow, David Liu, Dirk Schadendorf
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72005e917b50c2e5c4b78fb42f30196a93fe8580" target='_blank'>
              Transcriptional Profiling of Commonly Used Liver Cancer Cell Lines Reveals Disease-Specific Modeling Potential and Authentication Concerns
              </a>
            </td>
          <td>
            Nima Sarfaraz, Gail R. Rosen, Trevor Winkler, Risha Patel, Srinivas Somarowthu, Michael Bouchard
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive disease with one of the lowest five-year survival rates of all cancers. Low grade, premalignant pancreatic intraepithelial neoplasias (PanINs) are common in the general population, but despite expression of mutant KRAS, the vast majority of these lesions will never progress to cancer. Understanding how additional genetic or epigenetic changes as well as changes to the tumor microenvironment contribute to PanIN to PDAC progression is critical and has the potential to reveal mechanisms of interception for this deadly disease. Our previous work identified the prolyl isomerase PIN1 as a critical factor in the tumor microenvironment for growth of a subset of pancreatic cancer cell lines. Loss of PIN1 in the tumor microenvironment restricted in vivo growth of multiple KPC cell lines. We identified distinct fibroblast phenotypes in the absence of microenvironmental PIN1 expression or function. These changes in fibroblast state drive significant changes in the extracellular matrix and growth factor expression, both of which impact the crosstalk to and behavior of the neoplastic cells. We are continuing to use conditional PIN1 knockout mice to interrogate the specific cell types responsible for the observed tumor-restrictive phenotype and are using single cell sequencing to better understand how changes to individual cell types affect multiple cell populations to inhibit tumor growth. In addition, we are comparing the fibroblast phenotypes observed in our mouse models to those found in tumor-promoting versus tumor-restrictive microenvironments in human tissues. These tissues include autopsy pancreata with low grade PanIN lesions as well as PDAC resection specimens with low and high grade PanINs. We are creating organoid and cell line models of epithelial and stromal cells from normal human pancreata to support complex co-culture engineered models of early disease progression. By integrating across mouse and human model systems, we strive to understand the crosstalk that permits progression of pancreatic cancer and to identify targets to intercept this process.



 Tana Gazdik, Madeline Kuhn, Chloe Bowman, Eric Carlson, Kyra Lindley, Ellen M. Langer. Interrogating microenvironmental changes that impact early pancreatic cancer development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4475acff7b0cbed38743a8484c9c3b1b2adb82c" target='_blank'>
              Abstract A031: Interrogating microenvironmental changes that impact early pancreatic cancer development
              </a>
            </td>
          <td>
            Tana Gazdik, Madeline R Kuhn, Chloe L Bowman, Eric J Carlson, Kyra A Lindley, Ellen M. Langer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genetically engineered mouse models have advanced cancer research but often fail to capture key features of certain human tumors. Rats, with distinct physiology and tumor biology, offer a powerful alternative, yet their use has been constrained by technical barriers to genome editing. Here, we report efficient somatic genome editing in rats, enabling both Indel and substitution mutations. We then apply this approach to model estrogen receptor (ER)-positive breast cancer, which accounts for ∼70% of human cases but remains poorly represented in mice. The resulting rat tumors reproduce hallmarks of human ER+ breast cancer, including ductal histology, hormone responsiveness, and immune-microenvironmental features. By contrast, identical genetic alterations in mice failed to yield ER+ tumors, underscoring critical species differences in tumorigenesis. Together, this work establishes a versatile platform for rapid generation of clinically relevant rat tumor models, opening new avenues to dissect tumor biology, therapeutic response, and immune interactions in previously inaccessible cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19abed2134c7009bf0720ace89b27aa8404fff98" target='_blank'>
              Rat somatic genome editing enables ER+ breast cancer modeling
              </a>
            </td>
          <td>
            Wen Bu, Tobie Lee, M. Anurag, Yunfeng Ding, Ruixin Xu, Lillian He, Alexandria Z. Bu, C. Nagi, Carolina Gutierrez, S. Hilsenbeck, Chonghui Cheng, Bing Zhang, Shixia Huang, Jianming Xu, C. K. Osborne, Arun Sreekumar, Eric Chang, Chad J. Creighton, X. Zhang, Yi Li
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide. However, the spatial and temporal dynamics underlying its development remain poorly characterized. This study employs spatial transcriptomics (ST) to investigate the progression of intestinal tumors in APC Min/+ mice across multiple time points. We identified distinct transcriptional profiles between tumor and normal tissues, resolving six major cell types through integrated dimensionality reduction and pathological annotation. Pseudo-time trajectory analysis revealed increased expression of MMP11 and MYL9 in later stages of tumor progression. Analysis of human CRC cohorts from the TCGA database further confirmed that high expression of these genes is associated with advanced clinical stages and promotes tumor proliferation and invasion. Temporal gene expression dynamics indicated enrichment of cancer-related pathways concurrent with suppression of lipid and amino acid metabolism. Notably, genes in the DEFA family were significantly upregulated in normal tissues compared to tumor tissues. Functional validation showed that DEFA3 inhibits colon cancer cell migration and proliferation in vitro. These demonstrate the value of ST in resolving spatiotemporal heterogeneity in CRC and identify both MMP11/MYL9 and DEFA3 as potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a441608f72da7375d3ddfdc034535ffa428b9ffc" target='_blank'>
              Investigating the Development of Colorectal Cancer Based on Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhaoyao Qi, Guoqing Gu, Huanwei Huang, Beile Lyu, Yibo Liu, Wei Wang, Xu Zha, Xicheng Liu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Breast cancer brain metastases are associated with a poor prognosis, and there is a high demand for improved personalized treatment strategies. It is still poorly understood which breast cancer brain metastases patients benefit from immune checkpoint blockade.



 Here, we multimodally characterized the immune niche of breast cancer brain metastases at single-cell resolution in a clinically annotated patient cohort (n = 156), combining multiplexed imaging and cell type quantification by tissue cytometry, single-nuclei RNA-sequencing, flow cytometry, spatial transcriptomics, and functional studies in patient-derived organoids.



 Overall, infiltration of breast cancer brain metastases by CD8 T cells was strongly heterogeneous between patients, and high numbers were prognostically favorable. In-depth characterization of cell types and integrative analyses including clinical data revealed distinct immune landscapes across patients, two of which independently predicted prolonged overall survival: i) Breast cancer brain metastases containing T and B cells accumulating together with characteristic cell communities in stromal tertiary lymphoid structures, and ii) breast cancer brain metastases characterized by a high content of tissue resident-like CD8 memory T cells (CD8 TRM-like cells) spatially enriched within tumor islands. CD8 TRM-like cells exhibited high levels of activation, checkpoint, and functional markers and showed tumor cell killing capacity ex vivo, suggesting them to be critical players of tumor immune control.



 Together, our work provides novel insights into anti-tumor immunity in breast cancer brain metastases and reveals prognostic immune biomarkers with potential translational relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb6c7f193a5bf8137e26dc814e6139f92f208a4" target='_blank'>
              KS03.5.A DISSECTING THE CELLULAR ARCHITECTURE OF BREAST CANCER BRAIN METASTASES REVEALS PROGNOSTICALLY DISTINCT IMMUNE LANDSCAPES
              </a>
            </td>
          <td>
            L. Jassowicz, F. Feng, R. Warta, F. Sor, J. Soller, A. Wöllner, C. Lotsch, J. Wong, J. Schlegel, L. Frank, P. Dao Trong, G. Jungwirth, A. Nohman, K. Maaß, M. Barthel, A. Unterberg, S. Krieg, K. Rippe, A. Abdollahi, A. von Deimling, J. Malm, P. Lichter, M. Zapatka, M. Seiffert, C. Herold-Mende
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648afec99f4d26f113081261fc3d480301dc1dab" target='_blank'>
              Transcriptional reprogramming of tumor-infiltrating T cells during PD-1 blockade revealed through gene regulatory network and trajectory inference in squamous cell carcinoma
              </a>
            </td>
          <td>
            Roger Casals-Franch, Jordi Villà-Freixa, Lara Nonell, Adrián López García de Lomana
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) arises from precursor lesions, the most common of which are Pancreatic Intraepithelial Neoplasia (PanIN). PanINs are microscopic and cannot be detected in live individuals; as such, PanIN characterization has occurred prevalently in the context of cancer-bearing pancreata (adjacent normal) rather than in healthy tissue. Recently, through a unique collaboration with Gift of Life Michigan, we showed that PanIN prevalence is over 60% across age groups in donor healthy tissues, with high prevalence even in young individuals. Given PanIN prevalence and the rarity of pancreatic cancer, we hypothesized that the majority of PanIN is not fated to progress to malignancy. We postulated that a combination of neoplastic cell-intrinsic factors and components of the microenvironment controls progression to malignancy. We utilized spatial transcriptomics using the Visium platform (n = 14) and scRNASeq (n = 45) to dissect gene expression profiles and characterize spatial domains in donor healthy, adjacent-normal, and tumor tissues. We developed a spatial domain identification workflow that integrates spatial transcriptomics with the scRNASeq atlas, leveraging cell-type deconvolution, spatially informed clustering and pathology annotation. This approach allowed us to study the co-evolution of epithelial and stromal cells over the progression of pancreatic cancer. We were able to delineate distinct epithelial spatial domains—including Acinar, Ductal, and PanIN structures—in both donor and tumor tissues, as well as tumor-specific regions such as glandular tumor and poorly differentiated tumor. Additionally, we identified key stromal niches, including immune-rich regions such as tertiary lymphoid structures (TLS), plasma cell clusters, and macrophage-enriched zones, alongside non-immune fibrotic domains. We began by analyzing the epithelial domains and observed that while all acinar and ductal domains formed transcriptionally distinct clusters, neoplastic cells formed a continuum from sporadic healthy donor-associated PanINs to well- and poorly-differentiated tumors. We identified key genes that exhibit expression changes throughout the progression trajectory. Secondly, we analyzed characteristics of the stroma surrounding sporadic lesions, well-differentiated, and poorly-differentiated tumors. We observed key differences in fibroblasts between healthy donor-PanIN and tumorigenic regions, as fibroblasts surrounding donor-associated PanIN lacked known cancer-associated fibroblast (CAF) markers such as Alpha-Smooth Muscle Actin (α-SMA). We also observed distinct differences in immune cells, where there were more abundant plasma and CD8+ T-cells surrounding donor-associated PanIN, while tumorigenic regions showed macrophage enrichment. We then performed immunostaining and confocal imaging to validate the defined markers and specific cell populations. Our work defines the PanIN microenvironment as distinct from that of PDAC and maps the co-evolution of cancer cells and stroma over the course of carcinogenesis.



 Ahmed M. Elhossiny, Jude Okoye, Padma Kadiyala, Alexander Bray, Jamie Mills, Hannah Watkoske, Yaqing Zhang, Jiaqi Shi, Arvind Rao, Elana J. Fertig, Eileen S. Carpetner, Timothy Frankel, Marina Pasca di Magliano. Spatially Resolved Insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A052.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f4f9e15e43361f1e76f4bf1e3cc3e1d04354d0" target='_blank'>
              Abstract A052: Spatially Resolved Insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, J. Okoye, Padma Kadiyala, Alexander Bray, Jamie N. Mills, Hannah R Watkoske, Yaqing Zhang, Jiaqi Shi, Arvind Rao, E. Fertig, Eileen S. Carpetner, Timothy L. Frankel, Marina Pasca di Magliano
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/466091ce59415bf8e8d644d95fdb8949a53fb96b" target='_blank'>
              Cancer-associated fibroblasts drive metabolic heterogeneity in KRAS-mutant colorectal cancer cells
              </a>
            </td>
          <td>
            Elizabeth Elton, Niki Tavakoli, Handan Cetin, Stacey D. Finley
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8e063a68a88a96c8de0cef9efb9254b78c4b6" target='_blank'>
              A cellular and spatial atlas of TP53-associated tissue remodeling defines a multicellular tumor ecosystem in lung adenocarcinoma.
              </a>
            </td>
          <td>
            William Zhao, Thinh Nguyen, Atharva Bhagwat, Akhil Kumar, Bruno Giotti, Benjamin Kepecs, Jason L. Weirather, N. Mahadevan, Åsa Segerstolpe, Komal Dolasia, Jamshid Abdul-Ghafar, Naomi Besson, Stephanie M Jones, Brian Y. Soong, Chendi Li, S. Vigneau, M. Slyper, Isaac Wakiro, Mei-ju Su, K. Helvie, Allison Frangieh, Judit Jané-Valbuena, Orr Ashenberg, Mark Awad, Asaf Rotem, Raphael Bueno, O. Rozenblatt-Rosen, K. Pfaff, Scott J. Rodig, Aaron N. Hata, Aviv Regev, B. Johnson, A. Tsankov
          </td>
          <td>2025-10-07</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Abstract Background The recent clinical success of tumor-infiltrating lymphocyte (TIL) therapy in immune checkpoint inhibitor-refractory melanoma has renewed interest in applying this approach to other malignancies. Renal cell carcinoma (RCC) is characterized by prominent CD8+ T-cell infiltration and elevated expression of cytolytic genes, suggesting the presence of tumor-reactive T-cells that could be harnessed for personalized cell therapy. TIL products are polyclonal and may recognize a diverse repertoire of tumor antigens, which could reduce the likelihood of immune escape through antigen loss. Despite this potential, past efforts to implement TIL therapy in RCC have been largely unsuccessful, likely due to the dominance of TILs exhibiting an exhausted phenotype, and the inability to effectively rejuvenate these cells during ex vivo expansion. Thus, the development of strategies that can selectively isolate and expand tumor-specific T-cells with minimal off-target toxicity and durable cytotoxic function will likely be key to the successful development of TIL therapy for RCC. We recently demonstrated that highly expanded clonal CD8+ T-cell populations in RCC tumors exhibit an antigen-experienced phenotype, indicative of chronic stimulation and consistent with tumor antigen recognition. These clonotypes represent a key feature of RCC TIL populations. Based on this, we hypothesize that the selection of non-exhausted, clonally expanded T-cells using specific surface markers will enhance the therapeutic potency and specificity of RCC-derived TIL products. Methods To identify surface markers that distinguish non-exhausted, clonally expanded tumor-reactive T-cells, we compiled a comprehensive single-cell RNA sequencing (scRNAseq) and paired TCR sequencing dataset. This includes tumor, normal adjacent tissue (NAT), peripheral blood mononuclear cells (PBMC), and lymph nodes (LN) from 23 clear cell RCC (ccRCC) patients, combining seven published and in-house studies. We are applying both differential gene expression (DGE) analysis and multi-instance-based machine learning (ML) to identify candidate surface markers linked to non-exhausted, clonally expanded phenotypes. These models are designed to accommodate the inherent noise and sparsity of single-cell data and to uncover combinatorial marker sets. Results In RCC tumors, we observed significant clonal expansion of T-cells with a memory-like, antigen-experienced phenotype. Tumor samples showed significantly higher T-cell clonality compared to matched normal tissues. Cells sharing the same TCR clonotype displayed heterogeneous transcriptional states, including both exhausted and non-exhausted phenotypes. Markers of tumor reactivity identified for other cancers (eg, CD39, CD103) labeled mainly exhausted subsets of expanded clonotypes, suggesting a need for improved marker combinations for RCC. Clonality tended to decline with increasing tumor stage, possibly indicating a link between exhaustion and tumor progression. DGE analysis revealed several surface markers associated with expanded TCR clonotypes, for which commercial antibodies are available for flow sorting. To overcome limitations of DGE alone, we are applying ML models that integrate gene expression, TCR identity, and clonal frequency to predict tumor reactivity. We successfully trained and validated multiple machine learning models, including scFormer (transformer-based), attention_MIL (attention-based deep multi-instance learning), miBoost (tree-based), and miSVM (support vector classifier), on sparse and heterogeneous scRNAseq data using a local GPU platform. All models showed consistent improvement in validation accuracy during training. Feature analysis revealed gene sets predictive of large versus small clonotypes, offering candidate markers for isolating tumor-reactive, non-exhausted T cells for downstream applications. Conclusions T-cells sharing the same T-cell receptor can display both terminally exhausted and progenitor exhausted states. Selecting clonally expanded, non-exhausted T cells may enhance the tumor reactivity of TIL products. Our ongoing analyses, including model hyperparameter optimization and gene pathway comparison, focus on identifying surface markers that are antibody-compatible, broadly expressed across patients, and applicable to CD8+ and CD4+ T-cells while excluding Tregs. These insights will support a refined TIL manufacturing strategy for improved RCC therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3870d4f63b8bc5e4814aaf86df832a9b45daa983" target='_blank'>
              30Meta-analysis of single-cell RNA sequencing of tumor-infiltrating T-cells in clear cell renal cell carcinoma reveals distinct phenotypes and prediction markers of expanded T-cell clonotypes
              </a>
            </td>
          <td>
            Yuexin Xu, Christopher P. Miller, S. Tykodi, E. Warren
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="One of the main characteristics of solid tumors, such as melanoma, is an acidic tumor microenvironment. Due to dysregulation of the cancer cell metabolism and an increased production of acidic metabolites, the tumor acidifies its microenvironment. We hypothesize that this has a strong impact on tumor heterogeneity and the formation of phenotypic subpopulations. Cell culture experiments are usually carried out at a physiological pH of 7.4. Here, we show that long-time acidosis results in the formation of a senescent subpopulation in melanoma cells. Interestingly, after reintroduction to physiological pH, these cells lose their senescence-associated attributes. We isolated this subpopulation using β-galactosidase-dependent C12FDG staining and FACS. Live cell imaging, microscopy, and RNA sequencing analysis revealed that these apparent senescent cells show an increased migratory activity. With this study, we demonstrate that the acidic tumor microenvironment drives tumor plasticity in melanoma and results in the formation of a small subpopulation, which promotes metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a15918dc412942e4628009e724e226c928f4b6e" target='_blank'>
              Acidic melanoma microenvironment selects for a senescence-like but also migratory-active subpopulation driving metastatic disease
              </a>
            </td>
          <td>
            Chafia Chiheb, Stefan Fischer, Zubeir El Ahmad, Ingmar Henz, Ines Böhme-Schäfer, Melanie Kappelmann‐Fenzl, A. Bosserhoff
          </td>
          <td>2025-10-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e57eb64936bf914f410576d3851ca5828a633294" target='_blank'>
              The proteomics and phosphoproteomics landscape of melanoma under T cell attack
              </a>
            </td>
          <td>
            G. Franciosa, Agnete W. P. Jensen, Ana Martínez-Val, Ilaria Piga, Marco Donia, J. Olsen
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Glioblastomas function as intricate cellular networks that extend into the surrounding brain tissue enabling long distance communication. This malignant connectivity spans from the tumor core to remote infiltration zones, supporting the concept of glioblastoma as a whole-brain disease. With growing ethical considerations in biomedical research and the limitations inherent in cross-species comparisons between animal studies and human glioblastoma models, human tumor platforms remain constrained in their ability to replicate the full infiltration spectrum from the tumor core to single-cell dispersion. Here, we present a three-dimensional, fully human ex-vivo glioblastoma model (,,Core2Edge”) that replicates this extensive infiltration range while preserving the intratumoral heterogeneity of the original tumor. This model involves implanting fluorescently labeled human glioblastoma organoids (GBOs) into organotypic human brain slices, maintaining the genetic integrity and cytoarchitecture of both brain and tumor.



 The Core2Edge model involves a comprehensive workflow completed over 7-12 days. Key steps include brain slice preparation (~4-6 hours, depending on quantity), one day for initial culture before GBO staining and transplantation, a variable culture period (up to ten days), and fixation with different techniques (~8 hours). Following model preparation, downstream analyses such as imaging, sequencing, and proteomics were performed.



 Multiplex immunofluorescence confirmed cell composition consistent with histological staining. Spatial transcriptomics enabled detailed glioblastoma cell subtyping based on Neftel states. Advanced imaging, combining tissue expansion and light-sheet fluorescence microscopy, facilitated super-resolution, three-dimensional visualization of the GBO-brain slice system. This approach allows for the study of initial infiltration steps, in-depth analysis of the invasive front, exploration of cell-cell interactions between tumor cells and the tumor microenvironment, and offers a platform for drug screening and testing, reducing the need for animal models.



 The,,Core2Edge“model represents an innovative ex-vivo platform replicating glioblastoma infiltration and heterogeneity. By preserving genetic integrity and cytoarchitecture, this fully human model bridges critical gaps in glioblastoma research, offering unparalleled insights into tumor behavior and potential therapeutic interventions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7215ed71357bb240e167b383e1fb29c1a6a3d4" target='_blank'>
              P01.24.B CORE2EDGE: A HUMAN GLIOBLASTOMA ORGANOID AND BRAIN SLICE CO-CULTURE MODEL CAPTURING TUMOR INFILTRATION AND TRANSCRIPTIONAL HETEROGENEITY FROM CORE TO SINGLE-CELL DISPERSION
              </a>
            </td>
          <td>
            M. Schneider, A. Potthoff, A. Melhem, B. Pregler, J. Rodriguez-Gatica, J. Thomas, L. Friker, M. Westhoff, V. Ravi, D. Heiland, A. Schlitzer, T. Pietsch, M. Hölzel, A. Waha, H. Vatter, M. K. Schwarz, U. Kubitscheck, U. Herrlinger
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits profound spatial heterogeneity driving therapeutic resistance, while the role of cancer-associated fibroblasts (CAFs) in orchestrating immunosuppressive niches remains incompletely defined. This study integrates single-nucleus RNA sequencing (snRNA-seq) and spatial transcriptomics (stRNA-seq) to map the cellular and molecular landscape of HCC. snRNA-seq identifies key cell populations-including fibroblasts, T_NK cells, and endothelial cells-using canonical marker genes. Spatial transcriptomics maps gene expression across tumor regions (core, invasive front, stroma) via the robust cell type decomposition (RCTD) algorithm. Immunofluorescence validates collagen deposition and POSTN spatial distribution, confirming T-cell exclusion patterns. The analysis identifies hypoxic metabolic myofibroblasts (hmmyCAFs) as central regulators of the tumor microenvironment. hmmyCAFs enrich at the invasive front, forming collagen-rich barriers that physically exclude CD8⁺ T cells. Simultaneously, they secrete POSTN to suppress immune checkpoint signaling and drive hypoxia-mediated glycolytic reprogramming of T-cell metabolism. Clinically, hmmyCAF activity and POSTN expression correlate with reduced progression-free survival and immunotherapy resistance. This multimodal study defines hmmyCAFs as triple architects of physical immunosuppression, molecular regulation, and metabolic remodeling. By linking collagen remodeling, POSTN-mediated checkpoint inhibition, and hypoxia-driven metabolic reprogramming to clinical outcomes, hmmyCAFs and POSTN may serve as potential indicators for evaluating the efficacy of immunotherapy in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aba5ba124e6d224e40973627e744d5081476427" target='_blank'>
              Spatial Transcriptomics and snRNA-seq Expose CAF Niches Orchestrating Dual Stromal-Immune Barriers in Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Yingxue Li, Changxiang Huan, Haoting Sun, Wei Zhang, Zhen Guo, Chuanyu Li, Jia Yao, Zhiqi Zhang, Shizhe Yu, Qiongzhu Dong, Lunxiu Qin, Jinze Li, Lianqun Zhou
          </td>
          <td>2025-10-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a39d99b88fdac73c609342d555fba7b0f41448d" target='_blank'>
              Regulatory network and spatial modeling reveal cooperative mechanisms of resistance and immune escape in ER+ breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Sarthak Sahoo, M. Jolly, Jason T. George
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="

 Colorectal cancer (CRC) remains a leading cause of cancer mortality, with liver metastasis being the principal determinant of poor prognosis, but the spatial mechanisms orchestrating metastatic niches remain elusive.




 To dissect the molecular and spatial dynamics of CRC progression, we constructed an integrative atlas using 35 single-cell RNA-seq datasets and spatial transcriptomics from primary tumors, liver metastases, and matched normal tissues. Malignant epithelial subpopulations were stratified via inferCNV and CytoTRACE analyses. Stromal-tumor interactions were dissected using CellChat and NicheNet, with functional validation through
 in vitro
 co-culture and immunohistochemistry.





 We identified a transcriptionally distinct epithelial subpopulation, termed high-malignancy CRC (High-M CRC), enriched in metastatic lesions and characterized by enhanced stemness, MYC-driven transcriptional activity, and glycolytic reprogramming. Stromal-tumor interaction analyses revealed that cancer-associated fibroblasts (CAFs), particularly matrix CAFs (mCAFs), promote malignant progression via the HGF-MET-MYC signaling axis. Spatial transcriptomic mapping confirmed the physical proximity and molecular co-localization of High-M CRC cells and mCAFs, along with enriched glycolysis and MYC expression at the cell-cell interface.
 In vitro
 functional validation demonstrated that CAF-derived HGF activates MET-MYC signaling in CRC cells, enhancing their invasion and proliferation—effects reversible by MET knockdown.




 We unveil a spatially organized metabolic niche driven by stromal-tumor HGF-MET-MYC signaling. These findings offer novel insights into the stromal-tumor interaction and suggest actionable targets for therapeutic intervention in CRC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dc67b8fc38aa04a6e00b35ef87b6cb65aaaec9" target='_blank'>
              A spatially resolved single-cell landscape of colorectal cancer liver metastasis reveals a stromal-tumor glycolytic signaling interaction
              </a>
            </td>
          <td>
            Jiahui Chen, Zukai Wang, Bingwang Zhu, Guoxian Guan
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1217f04795a563df5b0ffd6d061c6a9c9b363d93" target='_blank'>
              Lineage of origin-specific developmental programs drive the behaviors of malignant cells in an avian embryo model of human Medulloblastoma subgroups
              </a>
            </td>
          <td>
            Marion Mallet, Florian Martin, Karine Thoinet, Caroline Imbert, Moein Sarhadi, Jérémy Ganofsky, Jacob Torrejon Diaz, T. Fenouil, C. Delloye-Bourgeois, Julien Falk, Cécile Faure-Conter, Ludovic Telley, D. Meyronet, O. Ayrault, Valérie Castellani, S. Tauszig-Delamasure
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Highlights What are the main findings? Poor prognostic ligand–receptor pairs are localized in specific regions within glioblastoma tissues. What is the implication of the main finding? Our findings provide new insights into pathological cell–cell interactions and treatment for glioblastoma. Abstract Glioblastoma (GBM) is an aggressive and lethal malignant brain tumor. Cell–cell interactions (CCIs) in the tumor microenvironment, mediated by ligand–receptor (LR) pairs, are known to contribute to its poor prognosis. However, the prognostic influence of CCIs on patients with GBM and the spatial expression profiles of such LR pairs within tumor tissues remain incompletely understood. This study aimed to identify prognostic LR pairs in GBM and their intratumoral localization via multitranscriptomic analysis. The CCIs among GBM cells as well as between GBM and niche cells were comprehensively evaluated using 40,958 cells in single-cell RNA sequencing datasets. They were found to form intercellular networks in GBM by specific LR pairs, which were mainly implicated in extracellular matrix (ECM)-related biological processes. Survival analysis revealed that 13 LR pairs related to ECM biological processes contributed to poor prognosis (p < 0.05, and 95% confidence intervals > 1). Notably, our spatial transcriptomic analysis using three independent GBM cohorts revealed that the identified poor prognostic LR pairs were localized in specific regions within GBM tissues. Although the clinical importance of these LR pairs requires further investigation, our findings suggest potential therapeutic targets for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102a7c96b1dcd02ce81762c455b5d0717c9c4822" target='_blank'>
              Integrated Spatial and Single-Cell Transcriptomics Reveals Poor Prognostic Ligand–Receptor Pairs in Glioblastoma
              </a>
            </td>
          <td>
            Makoto Yoshimoto, Kengo Sugihara, Kazuya Tokumura, Shohei Tsuji, E. Hinoi
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The metastatic tumor microenvironment (TME) is a highly dynamic and heterogeneous ecosystem that plays a critical role in promoting cancer cell colonization, immune escape, and resistance to therapy. Recent advances in multi-omics technologies—including genomics, transcriptomics, epigenomics, proteomics, and metabolomics—have enabled a systems-level understanding of the molecular reprogramming that occurs in the TME following metastasis. In this review, we systematically summarize emerging findings from recent multi-omics studies that dissect cellular composition, signaling pathways, immune landscape, and metabolic rewiring within the metastatic TME. We highlight key molecular signatures and intercellular interactions that drive metastatic progression and therapy resistance. Furthermore, we discuss how integrative multi-omics data are being leveraged to identify actionable targets and to design novel immunotherapeutic and molecular precision strategies tailored to the metastatic niche. These insights provide a scientific rationale for the development of TME-targeted approaches in the treatment of advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1202cb3e5acfc52d10d5fc247d4fca996a2d6f2e" target='_blank'>
              Multi-omics analysis of the tumor microenvironment after metastasis: advancing toward personalized immunotherapy and molecular targeted strategies
              </a>
            </td>
          <td>
            Qing Gao, Huqiang Wu, Li Duan, Guanhua Lv, Dongmei Quan
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c607d8984e374de942f791c85dee91b59e0ac27d" target='_blank'>
              Optimal Gene Panel Selection for Targeted Spatial Transcriptomics Experiments
              </a>
            </td>
          <td>
            Haoran Lu, Luyang Fang, Orlando Zeng, Ping Ma, Wenxuan Zhong, Guo-Cheng Yuan
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Metastatic pancreatic adenocarcinoma is the dominant clinical presentation with a grim 3% 5-year survival rate. Over 80% of metastatic disease occurs in the liver and has the poorest outcomes. Overall, there is little mechanistic understanding of what promotes liver metastatic outgrowth and organotropism. Our recent work using spatial transcriptomics on unique matched primary tumors and metastases revealed distinct cellular ecosystems, noting reduced desmoplasia, high proliferation, spatially constrained metabolism, and heightened pro-tumorigenic myeloid infiltration and T-cell dysfunction at the invasive border of liver metastases. However, moving beyond these observations requires establishing causal links between molecular drivers and metastatic competence. Causal studies are limited by current model systems, the spontaneous genetically engineered mouse models (GEMM) that form the basis of pre-clinical studies do not sufficiently model the clinical metastatic reality with robust matched pancreas and liver tumors, and are further hampered with inconsistent metastatic rates and unpredictable progression for timed analysis. Syngeneic transplants of GEMM-derived cancer lines into wild-type mice provide a rapid pre-clinical model system of liver disease. However, the liver-metastatic rates varies between cell lines, even with identical driving mutations. We suspected that the unexplored mechanisms driving these differences in metastatic outgrowth present an opportunity to understand critical biology. Here we report the development a consistent model system of matched pancreatic and liver tumors using syngeneic cell lines with high and low tropism for liver metastatic outgrowth in C57Bl/6 mice. These lines are transplanted at low cell numbers to better allow the evolution of the site-specific tumor microenvironment and provide a reliable model system to examine both cancer-cell intrinsic and site-specific microenvironmental factors dictating liver outgrowth. Our observations of comparable pancreatic growth, successful metastatic growth in other organs (peritoneum or lung), and micro-metastatic lesions in the liver at early time points, suggests the liver-tropic differences fall within the ability of these cells to successfully outgrow in the liver microenvironment, rather than in vivo proliferative or extravasation differences. Comparison of gene expression between high and low liver-tropic cell lines identified several immune regulatory genes and a general increase in lipid metabolism, consistent with our published patient data. Finally, spatial quantifications of these lesions using a novel 46-plex murine immunotyping panel on FFPE tissues show similar suppressive immune cells at the interface of the tumor and normal liver as observed in our patient data, but evolves across differing lesion size, suggestive of a spatiotemporal progression. Altogether, this model system provides a robust, efficient pre-clinical platform to dissect spatiotemporal drivers of liver metastatic disease.



 Christina R. Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, Ateeq M. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens. Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288b7e540e23f2b8aa282e16a6f01e2dc86d80d0" target='_blank'>
              Abstract A048: Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation
              </a>
            </td>
          <td>
            Christina R Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, A. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec585f40c1c2e8296bee75c248183641228fbd0" target='_blank'>
              Macrophages display spatiotemporal specificity in intrahepatic cholangiocarcinoma and drive tumour progression via OSM and THBS1
              </a>
            </td>
          <td>
            Tingjie Wang, Ruitao Long, Zhongyu Cheng, Mi Zhang, Bing Wei, Cong Lyu, Cuiyun Zhang, Guangbo Tang, Zihang Li, Lei Li, Yida Li, Xiaofei Zhu, Chao Shi, Ke Yang, Chuanrui Xu
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor suppressor NF1 is recurrently mutated in glioblastoma, leading to aberrant activation of Ras/rapidly accelerated fibrosarcoma (RAF)/MEK signaling. However, how tumor heterogeneity shapes the molecular landscape and efficacy of targeted therapies remains unclear. Here, we combined bulk and single-cell genomics of human somatic NF1-mutant, isocitrate dehydrogenase (IDH) wild-type glioblastomas with functional studies in cell lines and mouse intracranial tumor models to identify mechanisms of tumor heterogeneity underlying clinical outcome and MEK inhibitor response. Targeted DNA sequencing identified CDKN2A/B homozygous deletion as a poor prognostic marker in somatic NF1-mutant, but not NF1 wild-type, glioblastoma. Single-nucleus RNA sequencing of human patient NF1-mutant glioblastomas demonstrated that mesenchymal-like (MES-like) tumor cells were enriched for MEK activation signatures. Single-cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib identified distinct responses among tumor subpopulations. MEK inhibition selectively depleted MES-like cells, and selumetinib-resistant MES-like cells upregulated Ras signaling while resistant non-MES cells expressed markers of glial differentiation. Finally, genome-wide CRISPR interference screens validated Ras/RAF/MEK signaling as a key mediator of selumetinib response. Repression of the RAF regulator SHOC2 sensitized glioblastomas to selumetinib in vitro and in vivo, suggesting a synergistic treatment strategy. Taken together, these results highlighted the heterogeneity of NF1-mutant glioblastomas and informed future combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9b553a3b7f2afede17ca71f5d4515ccec597b2" target='_blank'>
              Tumor heterogeneity underlies clinical outcome and MEK inhibitor response in somatic NF1-mutant glioblastoma
              </a>
            </td>
          <td>
            S. Pan, K. Mirchia, E. Payne, S. J. Liu, N. Al-Adli, Zain Peeran, Poojan D. Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, T. Ozawa, Brian Na, Alyssa T Reddy, Steve E Braunstein, Joanna J. Phillips, Susan Chang, D. Solomon, A. Perry, D. R. Raleigh, Mitchel S. Berger, Adam Abate, H. Vasudevan
          </td>
          <td>2025-09-23</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Meningiomas are the most common primary intracranial tumors, yet their cellular diversity, tumor-stroma interactions, and the relationship between neoplastic and normal meningeal cells remain incompletely understood. While recent studies have provided valuable single-cell insights into specific tumor contexts, a unified atlas encompassing both meningioma and healthy meningeal tissue across patients and studies is lacking.



 We constructed MeningioMap, a comprehensive single-cell atlas by harmonizing three publicly available datasets with in-house single-cell RNA sequencing (scRNA-seq) of surgically resected meningiomas and matched adjacent meningeal tissue. The dataset encompasses over 300,000 high-quality cells from 40+ patients, spanning diverse tumor grades, anatomical locations, histopathological subtypes, and genders. Standardized preprocessing, robust batch correction, and integrative clustering pipelines were employed. Preliminary spatial transcriptomic analyses further validated key tumor-stromal interactions predicted from scRNA-seq profile.



 Meningioma delineates a high-resolution taxonomy of malignant and non-malignant cell states within the meningioma ecosystem. We identified distinct malignant subpopulations characterized by specific copy number alterations, transcriptional programs, and clinical associations with tumor grade. Comparative analysis with matched meninges uncovered conserved meningeal lineages and unveiled tumor-specific reprogramming events. Spatial transcriptomics confirmed localized immune remodeling and stromal activation signatures at the tumor-meninges interface. Notably, we identified a novel transitional malignant-stromal phenotype enriched in invasive tumor fronts, suggesting a potential driver of recurrence and therapeutic resistance.



 MeningioMap represents the most extensive single-cell resource for meningioma and meningeal research to date. Beyond offering a foundational reference for tumor heterogeneity and cellular dynamics, it provides critical insights into tumor-microenvironment crosstalk, informing future strategies for precision diagnostics, risk stratification, and targeted therapeutic development. An interactive, publicly accessible portal will facilitate broad scientific and clinical utility.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5ef5fbb1b4ae669071d94a0dad7887a5b662c5" target='_blank'>
              P13.18.A MENINGIOMAP: A SINGLE-CELL ATLAS OF MENINGIOMA ECOSYSTEMS AND MICROENVIRONMENTAL NETWORKS
              </a>
            </td>
          <td>
            A. Boecke, A. Mohammed, D. Das, G. Nishita, R. Roelz, M. Shah, T. Juratli, J. Beck, J. Grauvogel, K. Joseph, V. Ravi
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Understanding how small molecules affect cellular morphology is essential for exploring their mechanisms of action (MoA) and identifying new therapeutic candidates. In this study, the Cell Painting Assay (CPA) was used to profile 196 small molecules in HCT116 colorectal cancer cells. By applying t-distributed stochastic neighbor embedding (t-SNE) followed by density-based clustering, 18 distinct phenotypic clusters were identified based on similarities in the quantitative morphological profiles generated from Cell Painting data. Although it was initially hypothesized that clustering would reflect known MoAs, most clusters showed only partial overlap with target-based classifications. Instead, compounds from different MoA classes converged on similar cellular phenotypes, suggesting common downstream effects or shared stress responses. Notably, compounds affecting DNA replication, mitosis, or transcriptional control appeared across multiple clusters, indicating functional diversity within morphologically similar groups. Clusters enriched with mTOR/PI3K inhibitors, spindle poisons, or transcriptional CDK blockers exhibited well-defined phenotypes, supporting the robustness of the assay. In contrast, compounds inducing more subtle phenotypes formed distinct micro-clusters, highlighting the method’s sensitivity. Overall, this study demonstrates that CPA can capture convergent phenotypic signatures that extend beyond target-based classification. These findings underscore the value of phenotype-driven screening for the functional annotation of chemical compounds and may help uncover unexpected relationships among molecules with diverse biological activities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b69630fa6518b96d44a14fa030997f1e5bc439" target='_blank'>
              Phenotypic profiling of small molecules using cell painting assay in HCT116 colorectal cancer cells
              </a>
            </td>
          <td>
            Velemir Lavrinenko, Ilona Donskaya, Vladimir Popov, Ekaterina Litau, Varvara Petrova, Stanislav Tyazhelnikov
          </td>
          <td>2025-10-29</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The reciprocal interaction between neural and immune components in the tumor microenvironment has emerged as a critical area of study in cancer neuroscience. Recent data has shown the increasing role of myeloid cells in enabling an immunosuppressive microenvironment in glioblastoma. However, the nature and the role of T cells and other immune components forming tumor-immune synapses remain to be fully explored. Here we aim to demonstrate how immunological stimuli and inflammatory changes affect the neural connections within and around glioblastoma cells using advanced spatial transcriptomics.



 The study employs human cortical slice models, inoculated with tumor cells carrying retrograde tracing viruses, specifically a rabies virus, to trace neuron-tumor connections. This was followed by the inoculation of reactive T cells. We used the MERSCOPE system, which allows for the 3D reconstruction of gene expression signatures in consecutive tissue slices, spanning multiple Z-depths by combining this cutting-edge technology with new segmentation approaches.



 Our spatial transcriptomic analyses revealed different cellular ecosystems in the human cortical slice model. Enhanced inflammatory stimuli can increase local tumor growth and infiltration and neuronal connectivity. We found that inhibition of T cell activation using JAK-STAT inhibition can reverse the reactive hyperconnectivity. We further identified key transcription regulators defining immune cell dynamics and triggering immunological stimuli within tumor-immune synapses.



 The integrative multi-omic approach using imaging-based analysis and 3D single-cell spatial transcriptomics technique is a powerful tool that allows a comprehensive understanding of how immune cells influence the neural connectivity and synaptic reactivity within the peritumoral microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5833d83bdb7f3aebf492021ffdbee5e8a292416d" target='_blank'>
              P04.16.A DECIPHERING NEURAL-IMMUNE RECIPROCAL INTERACTIONS IN GLIOBLASTOMA USING 3D SINGEL-CELL SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            T. Peng, A. Merdan, Y. Yabo, V. Venkataramani, F. Ricklefs, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Chimeric antigen receptor T-cell (CAR-T) therapy has emerged as a breakthrough immunotherapy for multiple cancers, yet its efficacy remains limited in glioblastoma due to tumor heterogeneity, immune-evasion of the microenvironment, and rapid T-cell dysfunction. The aim of our study is to investigate the spatially differential expression of CAR-T targets in glioblastoma. We want to focus on differential expression of CAR-T targets in tumor cell subtypes, differences between tumor core and infiltration zone as well as reactive immune zones.



 We analyzed spatial transcriptomic data from 62 glioblastoma specimens using SPATA2 software, augmented by advanced computational tools such as scVI (data integration), Cell2Location (single-cell deconvolution), graph neural networks, gene set enrichment analysis (GSEA), and spatial annotation screening (SAS). CAR-T antigens were chosen from existing published clinical and pre-clinical CAR-T cell trials.



 Spatial mapping revealed a pronounced gradient of NPC-like cells with a higher abundance of NPC-like cells in the infiltration zone and resection margins—regions critical for residual disease persistence after resection. Clinical CAR-T targets (e.g., ERBB2, PTPRZ1) were highly expressed in AC-like and OPC-like dominated niches while nearly absent in NPC-rich areas, suggesting a blind spot for current CAR-T therapies. Single-cell deconvolution further correlated NPC-like dominance to increased T-cell infiltration, while GSEA and clonal dynamics analysis of the T-cell compartment suggested downregulated functioning subclones in NPC-rich spatial neighborhoods, underscoring potential immune evasion mechanisms in these regions.



 Our study exposes a fundamental limitation of current CAR-T designs: their inefficacy against NPC-like cells that dominate the infiltration zone and therefore the post-surgical glioblastoma ecosystem. By pinpointing spatial and phenotypic resistance hubs, we provide a roadmap for refining CAR-T targets to address this gap, potentially enhancing adjuvant immunotherapy for glioblastoma. Protein-level validation of key findings is currently ongoing to further substantiate the results.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe7508f42aeb973a2f44bba592abe40968ca576d" target='_blank'>
              P04.13.B SPATIAL HETEROGENEITY OF CAR-T ANTIGEN EXPRESSION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Y. Kong, J. Zhang, F. Sahm, G. Kastner, M. Mühlbauer, E. Grabis, P. Heinrichs, J. Straehle, R. Sankowski, R. Roelz, J. Beck, D. Heiland, N. Neidert
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioblastoma (GBM), a highly aggressive malignant brain tumor, leads to poor clinical outcomes and low long-term survival rates. Using bioinformatics approaches, we analyzed single-cell RNA sequencing (scRNA-seq) data (GSE273274) from glioma patient samples in this study. It was found that mesenchymal-like (MES-like) cells in the GBM center (GC) significantly facilitated GBM’s aggressive spread and microenvironmental reprogramming. Further analysis on MES-like cells suggested that these cells may participate in the progression from glial differentiation to invasive metastasis. Differential expression analysis, univariate Cox regression, and LASSO regression identified COL22A1 as a key gene. COL22A1 was highly expressed in GC compared to GBM surrounding tissue (GS) and exhibited potential connection to aberrant phosphatidylinositol 3-kinase (PI3K) signaling activation, indicating that COL22A1 may act as a potential regulator of GBM, promoting malignant processes such as proliferation, apoptosis resistance, migration, and invasion. In vitro experiments showed that COL22A1 overexpression (COL22A1-OE) U-87 cells exhibited enhanced proliferation, viability, migration, and invasion. Moreover, immunoblotting of BCL-2 and BAX revealed that COL22A1-OE increased resistance to apoptosis in U-87 cells. However, all these effects were reversed upon silencing of COL22A1, which suggests COL22A1 could be a promising new target for GBM therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a091b05d62e57c97e8673640ca9464f145f296ba" target='_blank'>
              Malignant progression of MES-like cells mediated by COL22A1 in the spatial heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Guangyuan Chen, Zhonghua Fu, Xulin He, Zhaoli Shen
          </td>
          <td>2025-10-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea7d79855ee47627896db43f29cfa10fcc98ce7" target='_blank'>
              Single-cell multiomic modelling of early metastatic events promoted by the extracellular matrix.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Jun Hyeong Lee, Junho Kang, Kyeonghui Kim, Youngwon Cho, Jihyun Park, Sang-Hyun Song, J. Kwon, Young-Joon Kim, Woong-Yang Park, Tae-You Kim, Jong-Eun Park, Pilnam Kim, Jung Kyoon Choi
          </td>
          <td>2025-09-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4273a0ebbda27839741b0bafaebff9ecc71a2adf" target='_blank'>
              Defining heterogeneity in core regulatory circuitry reveals HOXB3 condensation as a potential target in glioblastoma
              </a>
            </td>
          <td>
            Chuanxia Zhang, Yijing He, Xiudan Zhan, Jiaming Yu, Shao Xu, Qinkai Zhang, Ru Qiu, Xinyue Wang, Zhuoxing Gao, Meng Huang, Wenyong Long, Qing Liu, Wei Zhao
          </td>
          <td>2025-09-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Resistance to chemotherapy, targeted agents, and particularly immunotherapy remains the principal challenge in the management of gastrointestinal malignancies. This review aims to comprehensively delineate the molecular and microenvironmental drivers of resistance, with emphasis on mechanisms impacting immunotherapy response, and evaluate emerging, mechanism−guided interventions (including immunotherapeutic combinations) for precision therapy. We first examine intrinsic mechanisms—including drug−target alterations, dysregulated drug metabolism and efflux, hyperactivation of DNA damage repair pathways, and epigenetic remodeling—and extrinsic influences stemming from the tumor microenvironment and extracellular matrix remodeling. We then highlight epithelial–mesenchymal transition (EMT) as a critical nexus that integrates stromal cues with cell−intrinsic survival programs, thereby promoting drug efflux and immune evasion. Next, we discuss how single−cell and spatial omics, liquid biopsy, patient−derived organoids, and AI−enabled analytics facilitate subclone−level mapping of resistance networks and real−time tracking of clonal evolution. Finally, we review mechanism−based strategies—including KRAS G12C inhibitors, efflux−pump antagonists, apoptosis reactivators, and epigenetic/autophagy modulators—and propose an integrated, multimodal regimen leveraging immunotherapy where appropriate, informed by real-time drug sensitivity data (e.g., from liquid biopsy), dynamic biomarkers and AI−driven optimization to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebd9ae366f20576b8816fc8c01cf3154ec38755" target='_blank'>
              Integrated molecular and microenvironmental drivers of drug resistance in gastrointestinal cancers: mechanisms, immunotherapy challenges, and precision strategies
              </a>
            </td>
          <td>
            Heng Xu, Jiaan Lu, Jiangying Wu, Kangling Zhang, Xuancheng Zhou, Ziqi Gao, Jingqi Feng, Ziye Zhuang, Xiaolin Zhong
          </td>
          <td>2025-10-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Glioblastoma relapse remains inevitable, placing these tumors among the most difficult to treat. Treatment challenges include inherent drug resistance, tumour’s plastic nature, and suppressive tumour microenvironment (TME). In other cancers, treatment-induced plasticity has been associated with the emergence of drug-tolerant persister cells, which survive therapy by undergoing reversible phenotypic changes towards resistant states. The role of persisters in glioblastoma remains to be unravelled. It is unclear to what extent the responses are patient- and treatment-specific, and how much they depend on the tumor microenvironment.



 We applied a cohort of molecularly characterized patient-derived models to investigate patient-specific responses to chemotherapeutics (Temozolomide, VAL-083). Treatment efficacy and its correlation with molecular features were assessed ex vivo using organoids and stem-like cultures as well as in vivo through orthotopic xenografts. To mimic clinical treatment schedules, we assessed functional responses in time, incorporating prolonged exposure to treatment and drug holiday period over several weeks. Analysis included tumor cell viability, cell cycle, DNA damage and repair, senescence and quiescence features. Phenotypic adaptation of glioblastoma and microenvironmental cells in vivo were analysed with MRI, single-cell RNA-seq, spatial transcriptomics and IHC.



 Assessment of drug efficacy across a cohort of patient-derived models revealed resistant versus adaptive responder groups, which were treatment-specific. While resistant cells did not show functional changes in cell viability and cell cycle upon prolonged drug exposure, sensitive cells show a biphasic growth curve characteristic of persister cells. The adaptation towards the least proliferative phenotype was reversible during drug holiday period, further confirming a role of a transient drug-tolerant state. Chemotherapeutic treatment in orthotopic xenografts partially reduced tumor growth in vivo and activated apoptosis and DNA repair pathways in glioblastoma cells. These changes coincided with an increased infiltration of tumor-associated macrophages.



 Our findings highlight the critical roles of both tolerance and intrinsic resistance mechanisms. The responder groups appear to be specific to the type of treatment and the unique characteristics of each patient’s tumor, emphasizing the need to evaluate treatment-induced plasticity in a patient-specific context. Glioblastoma cells are further supported by the adaptation of the microenvironment towards a tumor-supportive crosstalk.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ea2a43463b7a7c67d446cbd650a87fb74e2953" target='_blank'>
              P01.18.B MODELING GLIOBLASTOMA TREATMENT RESPONSES: INSIGHTS INTO TUMOR PLASTICITY AND THE ROLE OF THE TUMOR MICROENVIRONMENT
              </a>
            </td>
          <td>
            M. de Lucas Sanz, H. Allahverdi, B. Nosirov, P. Moreno Sánchez, M. Baroni Milan, A. Lipsa, M. Rezaeipour, C. Dording, D. Brown, J. M. Langlands, K. Grzyb, Y. Yabo, J. Zhang, D. Heiland, A. Michelucci, P. Nazarov, A. Gołȩbiewska
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f99de374a85ecbcb4133dcc03b7138475a67b7df" target='_blank'>
              Illuminating the Role of A-to-I Editing in Gastric Cancer using EndoVIA 2.0
              </a>
            </td>
          <td>
            Alexandria L. Quillin, Tatiana F Flores, Devanshi Purohit, Angela Halstead, Benoît Arnould, Aastha, Eleonora Grandi, Mason Leffler, José B. Sáenz, Jennifer M. Heemstra
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Glioblastoma tumors are characterized by a prominent infiltration of tumor-associated macrophages (TAMs), constituting up to 30-40% of the cellular make-up. These cells typically harbor immunosuppressive features which, in conjunction with T cell paucity, contribute to a tumor microenvironment (TME) uniquely refractory to immunotherapy response. Therefore, due to their critical role in tumor progression and therapy response, TAMs represent a promising therapeutic target within the brain TME.



 To address the challenge of macrophage heterogeneity in glioblastoma, we conducted comprehensive multi-omics analyses including scRNAseq, spatial transcriptomics, ATACseq and lipidomics, that identified distinct TAM subsets within tissue-resident microglia and infiltrating monocyte-derived macrophage (MDM) pools.



 We discovered a pro-tumorigenic and immunosuppressive subset common in both cell lineages characterized by high lipid metabolic activity which we termed lipid-laden macrophages (LLMs). These cells engulf and recycle lipid-rich myelin debris to fuel cancer cell proliferation and thereby promote malignancy. Beyond these functions, LLMs also exhibit immunosuppressive features that are induced by myelin debris uptake. Next, we aim to investigate the relationship between myelin-induced metabolic pathways and the subsequent immunosuppressive phenotype of LLMs. To accomplish this, we will employ a genetic screen on primary murine macrophages using a CRISPR knockout library. This approach will target the cellular metabolome and surfaceome, with the goal of rewiring TAMs toward a more immunogenic phenotype. To enable selective targeting of LLMs, we developed lipid nanoparticle (LNP)-based approaches. Our findings demonstrate that LNPs efficiently deliver cargo to macrophages in the TME, with formulations that can be tailored for enhanced cell type-specific uptake.



 We identified myelin as a key molecule fostering a tumor-permissive brain TME. We are currently harnessing genetic screening together with the development of cell type-specific LNPs which will enable interventions targeting the immuno-metabolic axis, potentially advancing new treatment strategies for glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8404fd7f21668b88fc96f4132337aef2254ebb3" target='_blank'>
              P02.04.A UNCOVERING AND TARGETING METABOLIC REGULATORS OF IMMUNOSUPPRESSIVE LIPID-LADEN MACROPHAGES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            M. Farber, T. van Schaik, D. Aschmann, M. Boon, D. Kloosterman, J. Erbani, L. Akkari
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) chemotherapy regimens are largely uniform, tailored mainly to patient fitness or rare genomic alterations. This strategy fails to capitalize on the disease’s substantial molecular heterogeneity, including variations in copy number alterations, immune infiltration, stromal composition, and cancer cell phenotypes. These features are not independent; they reflect interconnected biological processes and co-evolving pathways that shape tumour progression. Multi-omic integration is therefore essential—not only to capture the complexity of PDAC biology, but also to clarify the molecular basis of tumours classified as ‘intermediate’ by current stromal and cancer taxonomies, thereby enabling rational therapeutic targeting. To this end, we performed imaging mass cytometry on three serial sections of a PDAC tissue microarray (221 resected tumours, ∼4 cores each), generating >800 multiplexed images (40–43 channels) focused on epithelial, immune, or stromal biomarkers. We profiled 76 immune and stromal cell types and states (hypoxic, proliferative, apoptotic, under tension), as well as six tumour phenotypes defined by expression of epithelial transcription factors (GATA6, FOXA2, PDX1), classical markers (AGR2, TFF1, CEACAM6), basal markers (TP63, KRT5, S100A2, CAV1), and other PDAC-associated proteins (S100A4, MMP7, MUC16). These six cancer cell types captured the classical and basal PDAC signatures, along with four discrete “intermediate” states with distinct associations to stromal heterogeneity, RNA subtype (n = 92), tumour ploidy (n = 182), and patient outcome. These phenotypes are also detectable in unmatched single-cell RNA-seq data (n = 163), though with more overlap in marker expression. Using matched 30X whole genome sequencing (n = 182), we identified mutations and copy number alterations linked to shifts in cancer and stromal cell phenotypes, raising the question of which molecular axis best informs clinical prognosis. To address this, we applied a modified version of Stabl, a Lasso-based machine learning approach, to compare across omic layers and identify features most strongly associated with overall survival. Per-modality analysis showed that models incorporating omics outperformed those based on clinical features alone, with imaging data slightly outperforming genomics. Cross-omic integration revealed several prognostically relevant copy number aberrations, fibroblast phenotypes, and cancer cell states. By consolidating information on tumour phenotypes, stromal niches, and genomic alterations, this work aims to focus future drug discovery on the most clinically impactful molecular features in PDAC.



 Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steve Gallinger, Hartland W. Jackson. Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B116.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5cd553302915b28eaa98522e24327f9aa96cbdc" target='_blank'>
              Abstract B116: Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival
              </a>
            </td>
          <td>
            Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Edward L. Y. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background: Intrahepatic cholangiocarcinoma (ICC) has a poor prognosis due to late-stage presentation and ineffective systemic therapies. Targeting the tumor microenvironment (TME) in ICC offers new therapeutic possibilities, particularly through tumor-associated macrophages (TAM), which can both promote and inhibit tumor progression. The current study utilized multi-omics analysis to characterize the gene signature of TAM and explore its therapeutic potential in ICC. Methods: Public GEO datasets provided the basis for analysis. Single-cell RNA sequencing (scRNA-seq) data from five ICCs, three adjacent non-tumorous tissues (ANTs), and four healthy liver samples were examined with Python. To validate scRNA-seq findings, bulk RNA-seq data from 27 ICC and 27 matched ANT samples were assessed using R. Differentially expressed genes were identified with adjusted p-values <0.01 and log2-fold changes >1 or <−1. CIBERSORT pipeline analyzed 22 immune cell subtypes in bulk RNA-seq data. STRING database analyzed the contribution of unique TAM-related genes to networks of protein–protein interactions. Results: TAM population demonstrated phenotypic heterogeneity exhibiting partial gene signatures of inflammatory (MS1) and anti-inflammatory (MS2) macrophages. Unique TAM-associated markers, TREM2, CD9, and PRMT10, showed variable expression within the TAM subpopulation. Bulk RNAseq analysis confirmed the scRNA-seq results, highlighting overexpression of TREM2 and CD9 in most ICC samples versus ANT. Immune cell deconvolution revealed decreased MS1 and MS2 macrophages in ICC, and alterations in adaptive immune profile, suggesting immunotolerant TME. STRING database defined TREM2-LGALS3 axis as a potential target for anti-tumor therapies. Conclusions: TAM represents a unique heterogenous population which is primarily found in ICC TME versus ANT or healthy liver tissue The non-uniform expression of unique gene signature demonstrates additional heterogeneity in the TAM subpopulation and suggests that TREM2+ TAM may be desirable targets for anti-TREM2-LGALS3 immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be67d95f17acd65abd3873af222e463c89ca98c4" target='_blank'>
              Defining Gene Signature of Tumor-Associated Macrophages in Intrahepatic Cholangiocarcinoma as Target for Immunotherapy Using Single Cell and Bulk RNA Sequencing
              </a>
            </td>
          <td>
            J. S. Badshah, , Andrea Reitsma, M. Melcher, Olivia M. Martinez, Sheri M. Krams, Daniel J. Delitto, Varvara A. Kirchner
          </td>
          <td>2025-10-29</td>
          <td>Livers</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Single-cell & MR link ESCC immunotherapy to immune cells. Combined single-cell datasets yield more credible results. ESCC-immune cell discovery aids subsequent experimental validation. Esophageal cancer has a low 5-year survival rate despite treatments. scRNA-seq and MR offer insights into neoadjuvant treatment efficacy for precision medicine. Three post-neoadjuvant treatment datasets underwent QC for Seurat analysis. Marker genes identified cell subsets. MR analyzed eQTL data from GWAS cohorts for causal links. Single-cell and MR-derived genes intersected to reveal PLTP in CD4⁺T cells. Single-cell analysis of 16 samples found 40,198 genes and 120,102 cells. CD4⁺T cell numbers differed significantly between groups after therapy. Differentially expressed genes were immune-related. Pseudotime and cell-cell communication varied. PLTP, linked to esophageal cancer, co-expressed with genes involved in cell cycle processes. The study highlights CD4⁺T cells’ predictive role in therapy efficacy via scRNA-seq and MR. PLTP emerges as a key gene, offering new precision medicine strategies for esophageal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cc89377f616ee4461240aafc742e5dad712a1f6" target='_blank'>
              Integrated single-cell and Mendelian randomization analyses: dissecting underlying causes of varied efficacy in immune neoadjuvant therapy for esophageal carcinoma
              </a>
            </td>
          <td>
            Jiaxin Li, Sibo Meng, Ying Zhou, Yufeng Cheng
          </td>
          <td>2025-10-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) cancer associated fibroblasts (CAFs) are a complex and heterogenous group of cells and are a key component of the tumour microenvironment (TME). Our understanding of CAF functional diversity has increased in recent years, however certain aspects of their biology remain elusive, such as the spatial organisation of CAFs, their interactions with other TME compartments, and their contribution to global TME architecture. Better understanding of these will significantly accelerate therapeutic development.



 We performed multiplex immunofluorescence on tissue microarrays of resected human pancreatic cancer (n=227) with an 8-plex CAF panel and a molecular subtyping panel. After assigning bulk cell type (CAF – FAP+CK-, Epithelium – FAP-CK+, Other – FAP-CK-), we integrated CAF morphology, spatial pseudotime analysis, and TME architectural scoring in a hierarchical manner (pixel > cell > neighborhood > tissue module > patient) to define architectural subtypes of PDAC with internal validation.



 Architectural subtypes are defined as follows (name, dominant CAF, median disease-specific survival): Cohesive subtype (iCAF, 40 months) Rich in dense, cohesive, and tightly-packed epithelium. Sparse, loose stroma with a heterogenous population of iCAF-like CAFs which exhibit migration away from epithelium on spatial pseudotime modelling. Inflammatory subtype (roundCAF, 31 months) Dominated by a rich infiltrate of presumed immune cells (FAP-CK-). Highly organised with low spatial entropy. Associated with the classical subtype. CAFs have “cold” marker expression, are round in shape with a smooth surface, and exhibit migration towards epithelium on spatial pseudotime modelling. Activated subtype (Enlarged myCAF, 17 months) Dense stroma infiltrated by large myCAF-like CAFs. High spatial entropy with loss of TME compartmentalisation. Immune excluded. Strongly associated with the squamous/basal-like subtype. Fragmented subtype (Stellate myCAF, 13 months) Loose, dis-cohesive epithelium infiltrated by a CAF-dense stroma rich in myCAF-like CAFs with stellate morphology. High CAF-epithelial cell spatial contact. Associated with poor histological differentiation and poorest prognosis. Hypothesised to represent impending or established (occult) metastasis.



 Morphospatial profiling reveals striking associations between CAF shape, function, and spatial niche, and adds crucial perspective to our understanding of CAF biology beyond current methods. Combined with spatial pseudotime analysis, it offers exciting insight into CAFs in transitory states between subtypes, allowing maximum leverage of datasets which lack a true temporal dimension. In addition, architectural subtyping reveals the complex and heterogenous spatial organisation of the pancreatic TME, and the co-evolution of CAFs with these organisational changes. Integrating this with histopathological analysis lays foundations for the development of novel spatial biomarkers of prognosis, therapeutic development, and treatment response.



 Adam Bryce, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, Catherine Ficken, Ian Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, Fieke Froeling, Stephan Dreyer, David Chang. Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B121.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76e85ad242be0ed691b8aeea89c3fdd7c71569d" target='_blank'>
              Abstract B121: Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            A. Bryce, Silvia Martinelli, R. Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, C. Ficken, Ian R Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, F. Froeling, S. Dreyer, David Chang
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Phenotypic heterogeneity - distinct molecular and behavioral variations within a population -significantly influences collective invasion and tumor progression. Here, we employ a molecular approach to explore how the underlying metabolic heterogeneity in non-small cell lung cancer (NSCLC) influences invasion and pack patterning. Assessing three-dimensional (3D) pack patterning revealed invasive heterogeneity across NSCLC cell lines and patient-derived samples. Flow cytometry identified IL13RA2 as a biomarker for invasive potential, enabling isolation of subpopulations with distinct invasive characteristics. By integrating a cell surface biomarker (IL13RA2±) with mitochondrial membrane potential (TMRM), we identified and isolated three distinct subpopulations. Two-dimensional (2D) analyses revealed differences in mitochondrial polarity and transcriptional programs associated with migration and oxygen-sensitivity. In 3D, these subpopulations invaded with distinct patterns, from contiguous circular packs to structured chains. Assessments under varied oxygen tension demonstrated that oxygen availability and subpopulation metabolism together influence collective invasion patterning. When recombined at ratios recapitulating the original population, both stochastic and opportunistic cooperative dynamics emerged, dependent on subpopulation composition and oxygen levels. Our molecular approach, integrating cell surface and metabolic characteristics, enables isolation of unique subpopulations and demonstrates that phenotypic and metabolic heterogeneity, population composition, and oxygen availability collectively pattern invasion packs and drive collective invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14710731ead2715e9cfc57e6f8afb7982d2087db" target='_blank'>
              Metabolic programming defines oxygen sensitive subpopulation hierarchies and patterning in collective invasion.
              </a>
            </td>
          <td>
            Veronika Y. Matsuk, Tala O. Khatib, Landon J. Marcus, Isaac E. Robinson, Yuan Liu, Janani K Pasupathy, Mala Shanmugam, Janna K Mouw, Adam I. Marcus
          </td>
          <td>2025-09-24</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/701ebb3ce945210dbfdea44bbee1d176913f06e9" target='_blank'>
              EHBMT, a method for visualizing tumor evolution, identifies a surge in gastric cancer with hybrid epithelial-mesenchymal phenotypes due to an inflammatory microenvironment.
              </a>
            </td>
          <td>
            Dandan Li, Zeng Zhou, Yuanjian Hui, Hedong Yu, Tao Ren, Lantian Zhai, Xinqi Li, Lin Yuan, Lingyun Xia, Weidong Leng, Shanshan Qin
          </td>
          <td>2025-11-03</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Neoadjuvant therapy plays an important role in the treatment of glioblastoma (GBM), but a considerable proportion of patients remain unresponsive to the combination of immune checkpoint blockade (ICB) and antiangiogenic therapy. Understanding the mechanisms underlying resistance to this treatment and developing novel therapeutic strategies are crucial. We integrate extensive single-cell and spatial transcriptomic data to dissect the cellular composition and spatial organization of the GBM tumor microenvironment and validate our findings through experiments such as multiplex immunohistochemistry and atomic force microscopy. We applied 101 machine learning algorithms to evaluate the prognostic and immunological value of COL6A3+ tumor-associated fibroblasts (TAFs) and GPNMB+ monocyte-derived macrophages (MDMs) in multiple GBM cohorts and immunotherapy cohorts. We constructed a stromal cell atlas in GBM and identified a distinct subset of COL6A3+ TAFs with functional characteristics of matrix fibroblasts. We found that COL6A3+ TAFs are significantly enriched in non-responders to neoadjuvant combination therapy. These fibroblasts drive the spatial-reprogramming of anti-tumorigenic MDMs into a pro-tumorigenic phenotype. In turn, these reprogrammed immunosuppressive GPNMB+ MDMs promote vascular fibrosis mediated by COL6A3+ TAFs through the GPNMB-ITGB5 interaction. Our findings highlight the critical role of COL6A3+ TAFs in regulating MDM function and spatial distribution, as well as their contribution to fibrotic tumor vasculature formation. Additionally, we propose targeting COL6A3+ TAFs with cilengitide as a potential therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/430d20f3bfe099d744acf27ed5b371550f5b9883" target='_blank'>
              Spatial-reprogramming derived GPNMB+ macrophages interact with COL6A3+ fibroblasts to enhance vascular fibrosis in glioblastoma
              </a>
            </td>
          <td>
            Yinfei Du, Xinmiao Long, Xuetong Li, Fan Guan, Wei Gao, Kun-Kang Deng, Shiyi Wang, Xiang Lin, Meng Huang, X. She, Shuai Chen, Minghua Wu
          </td>
          <td>2025-10-31</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b5111159e92c7f129ff30372c9b9934bfcdb2c" target='_blank'>
              Single-cell RNA sequencing and large-scale bulk combination with machine learning reveal gastric cancer-related macrophage heterogeneity
              </a>
            </td>
          <td>
            Saisai Gong, Yu peng Zhao, Sheng Yang, Tong Liu, Chengyun Li, Yun Zhou, Geyu Liang
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9eea4d17ccda6da786c821ed59ad42654c9c3e4" target='_blank'>
              Tumor microenvironment in breast cancer: cellular crosstalk, pathways, and therapeutic insights
              </a>
            </td>
          <td>
            Manjari Sharma, Umesh Kumar, Shoorvir Singh
          </td>
          <td>2025-11-04</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BRCA1 mutation is closely linked to triple-negative breast cancer (TNBC), a subtype characterized by early onset, aggressive clinical course, and marked genomic instability. Although poly ADP ribose polymerase (PARP) inhibitors have expanded treatment options, resistance driven by cancer stem cells (CSCs) and incomplete interception of premalignant progression remain pressing challenges. In a recent study, Liu et al. developed a human induced pluripotent stem cell (iPSC)-based breast cancer model that follows BRCA1‑driven tumorigenesis from mammary differentiation to invasive disease and identify S100P as a BRCA1‑repressed effector associated with cancer stemness. These findings provide a human genetic context to probe early oncogenesis and nominate the S100P–RAGE axis as a precautionary—but promising—target for risk interception.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfe5bbd7fe04bb4260117969d5b0d5e59d56a2ae" target='_blank'>
              Deciphering BRCA1-driven tumorigenesis through an iPSC-based breast cancer model
              </a>
            </td>
          <td>
            Gaoxiang Zhu, Bing Zhao
          </td>
          <td>2025-10-01</td>
          <td>Cell Transplantation</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 We present a novel integration of data-driven single-cell analysis with mechanistic mathematical modeling to predict the impact of cancer-associated fibroblasts (CAF) on pancreatic ductal adenocarcinoma (PDAC) invasion. Bioinformatic analyses of high-throughput single-cell and spatial molecular assays give us access to human tumor gene expression at the time of measurement, enabling direct characterization of the human tumor microenvironment (TME), complementing our understanding of cell behavior gained from preclinical models. Notably, applying our Bayesian non-negative matrix factorization algorithm CoGAPS to single-cell RNA-seq data from PDAC allowed us to associate a gene program of concurrent epithelial-to-mesenchymal transition (EMT) and inflammation in epithelial cells with the presence of CAFs, validated through organoid co-culture experiments. We hypothesize that this gene program represents a mechanism by which CAFs promote invasive behavior in neoplastic cells over time. Testing this hypothesis requires inference of temporal changes, which is not possible with single-timepoint data collection in genomics data. In contrast to data-driven bioinformatics, mechanistic mathematical modeling can capture processes that happen over time and make predictions about a system. Agent-based models (ABMs) are one form of such mechanistic mathematical models that are well suited for temporal modeling of cellular phenotypes. We developed new software for ABMs that abstract cells into software agents with individual states and their own rules of behavior from single-cell and spatial molecular assays for temporal inference. We apply this technique to understand how CAFs in the PDAC microenvironment support the series of phenotype transitions comprising tumor progression and invasion, by encoding the CAF-induced ability of tumor cells to migrate and a cessation of proliferation in tumor cells, induced by CAF-secreted, non-diffusable substrate as inferred from our genomics analysis. While this provides a virtual framework to test the impact of varying CAF density on tumor cell invasion, it initially lacked integration with the spatial architecture of human tumors. We then apply the ABM to human PDAC spatial transcriptomics data to forecast tumor behavior over time in a spatially-informed setting, varying the hypothesized mechanisms. This revealed that a substrate-driven mechanism results in a broad mesenchymal-like zone surrounding epithelial-like outgrowths, while a cell contact-mediated mechanism results in a single-cell-wide boundary surrounding more significant outgrowth. When we compared these to real tumor compositions, this favors the explanation that secreted CAF substrate drives EMT in PDAC neoplastic cells, which we then validate in organoids with CAF conditioned media. Our work demonstrates how integrating ABMs with genomic and spatial data enables testing of mechanistic hypotheses about tumor-stroma interactions and provides deeper insights into PDAC microenvironments, potentially informing strategies to intercept disease progression.



 Jeanette I. Johnson, Daniel R. Bergman, Tamara Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Yoseph W. Dance, Jacob T . Mitchell, Elizabeth M. Jaffee, Neeha Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, Ashley L. Kiemen, Luciane T. Kagohara, Young H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O'Brien, Elana J. Fertig. Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d57a26173bbcb8bbded169c7c20718d2482d35" target='_blank'>
              Abstract A025: Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion
              </a>
            </td>
          <td>
            J. Johnson, Daniel R Bergman, T. Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Y. Dance, Jacob T. Mitchell, Elizabeth M. Jaffee, N. Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, A. Kiemen, Luciane T. Kagohara, Y. H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O’Brien, E. Fertig
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2affd9966e7d74eae7ebff40360fcc4663c6bf18" target='_blank'>
              Clonal expansion of cytotoxic CD8âº T cells in lecanemab-associated ARIA
              </a>
            </td>
          <td>
            Lance A. Johnson, Kai Saito, Akhil Pallerla, Jessica L Funnell, Ashley R. Ezzo, Chelsea L. Song, Douglas A Harrison, Noah J. Norton, Lauren C. Moore, L. Eldik, D. Fardo, Greg E. Cooper, Josh M Morganti
          </td>
          <td>2025-09-21</td>
          <td>None</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="PURPOSE
The Tumor Microenvironment (TME) has a vital role in cancer survival and progression, and may play roles in drug resistance and immune escape. To date, few studies have detailed the TME of Ewing sarcoma (EwS).


EXPERIMENTAL DESIGN
We performed spatially resolved transcriptomics of primary treatment-naïve EwS tumor biopsies from patients with or without clinical metastasis, complemented by high-plex spatial proteomic analysis.


RESULTS
We discovered greater stromal enrichment in localized EwS primary tumors compared to metastasis-associated EwS primary tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumor role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling connectivity, suggesting a potential immune-evasive mechanism.


CONCLUSIONS
In addition to the immune recruitment role of tumor-associated stroma, our findings provide spatial insight into the TME of EwS and provide a rationale for the preclinical investigation of MIF as a potential target for Ewing sarcoma immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f9c77e0153cfc111b2db6ef0b69632696638603" target='_blank'>
              Spatial profiling identifies tumor-associated stroma enrichment and MIF as potential immunotherapy targets in primary Ewing sarcomas.
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-09-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc219a64ec3d466c88c4dfd11649123286e2032b" target='_blank'>
              Cross-Species Insights from ART-D to Uncover Evolutionarily Conserved Oncogenic Mechanisms
              </a>
            </td>
          <td>
            Yifan Guo, Jiadong Zheng, Yixin Wu, Kun Zhao, Dian Lv, Wenhan Liu, Mengling Wang, Jin-Yu Lu, Wenyan Xu, Xianping Wang, Xianjue Ma
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20d0d9f79582f0bb24f1086dc2497782ffd50bb0" target='_blank'>
              Single-Cell Integration of Chromatin Accessibility and Transcriptomics Reveals Regulatory Networks in Ovarian Tumor-Infiltrating Adaptive NK Cells
              </a>
            </td>
          <td>
            Yizhe Sun, Meng Wan, Solrun Kolbeinsdottir, Kang Wang, Shruti Khare, Okan Gultekin, Sahar Salehi, K. Lehti, Ramanuj Dasgupta, T. Foukakis, Martin Enge, D. Sarhan
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Gliosarcoma (GS) is a rare, aggressive variant of Glioblastoma (GB), representing approximately 2% of all GB cases. It is characterized by its biphasic histological growth pattern with glial and sarcomatous areas. Little is known about GS pathogenesis or the underlying reasons for its distinct clinical features compared to GB, including higher rates of extracranial metastasis and skull invasion. Given the similarity in mutational profiles and chromosomal alterations between GS and GB, we hypothesize that GS arises from a unique interplay between tumor cells and the tumor microenvironment (TME), promoting a mesenchymal phenotype and the development of sarcomatous niches responsible for its clinical behavior.



 Spatial Transcriptomics (10x Genomics) und methylation profiling was applied to 15 GS samples from 13 patients. We used non-negative matrix factorization (NMF) and single cell deconvolution using large reference single cell RNA sequencing datasets to confidently identify conserved spatial features in our data. To identify the most discerning characteristics between GS and GB, we used a graph-based neural network approach.



 Using a reference-based DNA methylation classifier, 12 of the 13 tumors (92%) were identified as belonging to the mesenchymal (MES) GB subgroup. While transcriptional profiles of GS samples were heterogeneous, NMF revealed four recurrent transcriptional metaprograms. In addition to hypoxia- and glial-associated programs, we identified an extracellular matrix (ECM) remodeling program correlated with sarcomatous areas and enriched for vascular endothelial and smooth muscle cells. Lastly, an immune response program spatially associated with mesenchymal-like tumor cells and proliferating tumor-associated mononuclear cells. Subsequent cell-cell communication analysis uncovered potential signaling interactions and highlighted the importance of TGFβ- and MAPK-pathways for the formation of these distinct niches. A classifier based on the graph-based neural network successfully distinguished GB from GS subregions.



 In this study, we performed the first in-depth spatial characterization of GS, identifying specific ECM remodeling and immune reactive spatial features unique for the TME of GS. Our findings provide insights into the transcriptional programs shaping GS tissue architecture, which could be crucial for understanding its biology and eventually aid the design of targeted therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8c0789909c1c92f1762bb21af94894b8743846c" target='_blank'>
              P14.08.A GLIOSARCOMA SPATIAL ARCHITECTURE IS DEFINED BY UNIQUE ECM AND IMMUNE REMODELING TRANSCRIPTIONAL PROFILES
              </a>
            </td>
          <td>
            M. Mühlbauer, N. Neidert, J. Beck, O. Schnell, F. Sahm, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f06cd8f942853075fbd7169c14dc2622fac19fcf" target='_blank'>
              Comprehensive Bulk and Single-Cell RNA Sequencing Uncovers Senescence-Associated Biomarkers in Therapeutic Mesenchymal Stem Cells
              </a>
            </td>
          <td>
            Emese Pekker, Erda Qorri, Márton Zs. Enyedi, Valéria Szukacsov, Ferhan Ayaydin, Éva Szabó-Kriston, Bernadett Csányi, M. Mórocz, Farkas Sükösd, Endre Kiss- Tóth, Lajos Haracska
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer mortality, with minimal improvements in 5-year survival over the past decade. While PDAC originates from premalignant lesions, only a minority of these lesions advance further into malignancy. Distinguishing which lesions are likely to progress would transform enabling timely preventative surgical intervention, but remains a key challenge. In addition to the more well-characterized genetic mutations associated with PDAC onset, recent studies suggest that inflammation in the pancreas microenvironment promotes epithelial chromatin accessibility remodeling to increase their plasticity. This plasticity is thought to enhance cellular sensitivity in response to local signaling cues, which varies drastically across PDAC tumors. We propose that the cell lineage fate of the premalignant epithelial cells is heavily influenced by their surrounding microenvironment. To investigate this, we are using the CosMx spatial transcriptomics platform to profile cellular composition heterogeneity across pancreatic tissue cores from KrasLSL-G12D; Trp53LSL-R172H; Pdx1-Cre (KPC) mouse model of PDAC. By analyzing spatial heterogeneity within the tumor microenvironment, we aim to delineate how local microenvironmental differences might influence epithelial lineage fate decisions into PDAC. Our findings will help understand the factors driving progression from early lesions to invasive cancer, supporting efforts in early PDAC detection and informing clinical strategies for surgical resection.



 Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, Crispin J. Miller. Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B118.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb056397ddb7c8e4e0421ffdba57d9fda0a6959d" target='_blank'>
              Abstract B118: Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P Morton, Crispin J. Miller
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The cellular plasticity of neuroblastoma is defined by a mixture of two major cell states, adrenergic and mesenchymal, which may contribute to therapy resistance. However, how neuroblastoma cells switch cellular states during therapy remains largely unknown, and how to eradicate neuroblastoma regardless of its cell state is a clinical challenge. To better understand the cellular plasticity of neuroblastoma in chemoresistance, we define the transcriptomic and epigenetic map of adrenergic and mesenchymal types of neuroblastomas using human and murine models treated with indisulam, a selective RBM39 degrader. We show that cancer cells not only undergo a bidirectional switch between adrenergic and mesenchymal states, but also acquire additional cellular states, reminiscent of the developmental pliancy of neural crest cells. These cell state alterations are coupled with epigenetic reprogramming and dependency switching of cell state–specific transcription factors, epigenetic modifiers, and targetable kinases. Through targeting RNA splicing, indisulam induces an inflammatory tumor microenvironment and enhances the anticancer activity of natural killer cells. The combination of indisulam with anti-GD2 immunotherapy results in a durable, complete response in high-risk transgenic neuroblastoma models, providing an innovative, rational therapeutic approach to eradicate tumor cells regardless of their potential to switch cell states. Cell state plasticity of neuroblastoma cells is linked to therapy resistance. Here, the authors develop a transcriptomic and epigenetic map of indisulam (RBM39 degrader) resistant neuroblastoma, demonstrating bidirectional cell state switching accompanied by increased NK cell activity, which they therapeutically enhance by the addition of an anti-GD2 antibody.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8bfb0658591ed4d276e6df844046df62c0e809" target='_blank'>
              RBM39 degrader invigorates innate immunity to eradicate neuroblastoma despite cancer cell plasticity
              </a>
            </td>
          <td>
            Shivendra Singh, Jie Fang, Hongjian Jin, Lee-Ann Van de Velde, Andrew Cortes, Jiani Chen, Sivaraman Natarajan, E. Poon, Qiong Wu, Christopher L. Morton, Mary A. Woolard, Waise Quarni, Jacob A Steele, Jon P Connelly, Liusheng He, Rebecca Thorne, Gregory Turner, Thomas Confer, Melissa Johnson, W. Caufield, Burgess Freeman, Timothy Lockey, A. Murphy, P. J. Murray, Takashi Owa, S. Pruett-Miller, Ruoning Wang, L. Chesler, Julie R. Park, A. Davidoff, John Easton, Xiang Chen, Paul G. Thomas, Jun Yang
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a common neoplasm that carries the risk of transformation into Richter’s syndrome (RS), a highly aggressive B cell lymphoma with poor prognosis. Limited availability of animal models and cell lines hinders understanding of transformation mechanisms. Addressing this gap, we established the first in silico dynamic model of the disease. Our methodology integrates mathematical logic modeling with experimental data to identify disease drivers, mechanisms, and potential therapeutic targets. We validated the model by comparing the model’s readout with experimental data from different biological levels, such as single-cell RNA sequencing analyses and a CLL/RS patient formalin-fixed paraffin-embedded (FFPE) tissue cohort. Our analyses identified BMI1 proto-oncogene and TP53 loss as key RS progression regulators. In addition, we performed an in silico target screening to identify promising target combinations in a personalized fashion. Through the synergy of mathematical modeling with experimental readouts, our model provides a complementary approach to investigate the process of CLL transformation to RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0084b1cbd03c2692e1100a97e2d57fe76c34d4" target='_blank'>
              CLL to Richter syndrome: Integrating network strategies with experiments elucidating disease drivers and personalized therapies
              </a>
            </td>
          <td>
            Julia Maier, J. Schwab, Silke D. Werle, Ralf Marienfeld, S. Stilgenbauer, Peter Möller, Nensi Ikonomi, Hans A. Kestler
          </td>
          <td>2025-09-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Gliomas are the most common malignant brain tumors, characterized by their aggressive nature and poor prognosis. Despite advancements in radiation and chemotherapy, the five-year survival rate for patients with glioma remains low. A significant challenge is the lack of experimental models that accurately replicate the tumor microenvironment and cellular heterogeneity of human gliomas, limiting the understanding of disease mechanisms and the development of effective therapies.



 Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids.



 Immunofluorescence, flow cytometry, and gene sequencing revealed that the organoids retained an immune microenvironment consistent with that of the original tumor. Unlike traditional cell lines and animal models, GlioMEs retain the immune microenvironment, allowing faster and more accurate immunotherapy screening. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib.



 In summary, our results demonstrate that GlioME can retain the parental glioma microenvironment in vitro and reliably test various therapeutic drugs. This organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4662e63a3116553f2fb291a5b7c518e12a2b2372" target='_blank'>
              P12.31.B A NOVEL ORGANOID MODEL RETAINING THE GLIOMA MICROENVIRONMENT FOR PERSONALIZED DRUG SCREENING AND THERAPEUTIC EVALUATION
              </a>
            </td>
          <td>
            C. Zheng, Q. Chen, Z. Fang, P. Wang, D. Zhang, Y. Feng, Y. Zhang, T. Jiang, L. Zheng, Z. Bao
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Alaska Native peoples experience high colorectal cancer (CRC) incidence and mortality rates, which Alaska tribal health organizations are working to address. To reduce the high rates, it is important to understand the biological mechanisms that drive risk and survival outcomes. The tumor microenvironment (TME) comprises diverse cell types, including cancer, stromal, and immune cells, whose composition and spatial organization may impact prognosis and disease progression. We performed spatial proteomics profiling on tissue microarrays from 97 Alaska Native patients with CRC using Akoya Biosciences’ PhenoCycler-Fusion system. We included 35 patients who died of CRC and 62 matched patients who did not die of CRC and lived at least as long as the patient they were matched to. Patients were matched on age at diagnosis, sex, tumor site, tumor stage, and year at diagnosis. We classified 2.90 million cells into 16 distinct cell types. We quantified immune and stromal cells as proportions of non-epithelial cells and epithelial cells as proportions of total cells. For spatial analysis, we identified eight cellular neighborhoods with distinct local cell composition and calculated spatial proximities between different cell types. We performed multivariable-adjusted logistic regression with generalized estimating equations to investigate associations of CRC-specific death with proportions of cell types and spatial proximities. We estimated odds ratios (OR) with 95% confidence intervals (CI) and calculated the false discovery rate (FDR) to account for multiple comparison using the Benjamin-Hochberg method. We found that higher proportions of stromal cells (OR for doubling of proportion=2.29, 95%CI=1.00-5.26) and higher proportion of M2 macrophages at the tumor center (OR=2.90, 95%CI=1.14-8.22) were associated with increased risks of CRC-specific death among 76 patients with microsatellite stable (MSS) tumors. We observed higher proportions of podoplanin (PDPN)-expressing stromal cells (OR=2.97 for doubling of proportion, 95%CI=1.28-7.08) and M1 macrophages (OR=2.42, 95%CI=1.05-5.69) within cellular neighborhoods predominantly composed of tumor epithelial cells among patients with CRC deaths compared to survivors, although these associations did not remain significant at FDR 0.05. Additionally, we observed that colocalization of PDPN-expressing stromal cells with CD8+ T cells, and colocalization of antigen-presenting cells with plasma cells, was associated with increased risk of CRC-specific death. In contrast, colocalization of B cells with stromal cells was associated with decreased risk (adjusted P=0.04 for all associations). Those associations remained significant among patients with MSS (adjusted P=0.03). This is the first study focusing on the TME cellular landscape among Alaska Native patients with CRC. Our results highlight the prognostic relevance of cellular composition and spatial organization within the TME, suggesting specific stromal and immune cell interactions as potential biomarkers for CRC outcomes.



 Hang Yin, Diana Redwood, Kimberly Smythe, Kristin L. Robinson, Daniel Jones, McGarry Houghton, Kevin C. Barry, Amanda L. Koehne, Elizabeth Donato, Cecilia Yeung, Minggang Lin, James J. Tiesinga, Tabitha A. Harrison, Sushma S. Thomas, Li Hsu, Jane C. Figueiredo, Li Li, Christopher I. Li, Ulrike Peters, Jeroen R. Huyghe. Spatial-resolved single-cell analysis of the tumor microenvironment in Alaska Native colorectal cancer patients reveals novel biomarkers for prognosis [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A087.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fe434533ff8a8160b5fcdfdf886e32d3184d6f4" target='_blank'>
              Abstract A087: Spatial-resolved single-cell analysis of the tumor microenvironment in Alaska Native colorectal cancer patients reveals novel biomarkers for prognosis
              </a>
            </td>
          <td>
            Hang Yin, Diana Redwood, K. Smythe, Kristin L. Robinson, Daniel Jones, Mcgarry Houghton, K. C. Barry, Amanda L. Koehne, Elizabeth Donato, Cecilia Yeung, MingGang Lin, James Tiesinga, T. Harrison, Sushma Thomas, Li Hsu, Jane C Figueiredo, Li Li, Christopher I Li, Ulrike Peters, J. Huyghe
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="


 Pancreatic intraepithelial neoplasms (PanINs), precursor lesions of pancreatic ductal adenocarcinoma (PDAC) that are abundant healthy individuals, rarely progress to cancer despite universally possessing many of the driver alterations associated with malignancy. Recent single cell profiling efforts of PanIN progression in vivo have described marked heterogeneity in chromatin accessibility and transcriptional identity within these neoplasias, but it remains unclear how discrete epigenetic factors play a role in shaping tumorigenesis, or what distinguishes rare neoplastic cells capable of seeding invasive carcinomas from their benign neighbors. Given that more than 80% of patients with PDAC present with unresectable or metastatic disease, further studies on the molecular features of PanIN progression are needed to enable methods of early detection and intervention. Unfortunately, owing to their microscopic size and challenges associated with the isolation of PanINs from surrounding normal epithelium and stroma, it is particularly difficult to address these knowledge gaps in vivo. By generating complex murine derived organoid models, we have built a system that allows us to faithfully reconstitute the evolution from benign PanIN to PDAC in vitro, as well as in vivo through orthotopic implantation into syngeneic mice. As they are easily perturbable, readily generate large amounts of biomass, and closely resemble the stages of PDAC initiation in vivo, these models can be leveraged to directly assay how chromatin remodelers influence malignant transformation. Further, pairing these organoids with our highly multiplexed, transcribable DNA barcoding system and single cell epi multiomics will enable us to study the evolutionary dynamics, chromatin landscapes, and transcriptional identities of clonal populations of PanIN organoids as they progress to cancer. Through these studies, we will gain unprecedented insights into the underpinnings of PDAC initiation, and thus may open up new avenues for the early detection of tumorigenesis.



 Connor J. Hennessey, Minh T. Than, Ben Z. Stanger, Sydney M. Shaffer. Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90b593e2ad7a725999b9d873ea37a620072eeba" target='_blank'>
              Abstract B106: Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models
              </a>
            </td>
          <td>
            Connor J. Hennessey, Minh T Than, Ben Z. Stanger, Sydney M. Shaffer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is an epithelial cell cancer of the biliary tract. CCA can be further classified into intrahepatic cholangiocarcinoma (iCCA), perihilar cholangiocarcinoma (pCCA), and distal cholangiocarcinoma (dCCA) depending on the anatomical location. Until recently, the treatment for advanced CCA has remained highly reliant on chemotherapy, with Gemcitabine plus Cisplatin used in first-line treatment. Recent developments have led to the addition of immune checkpoint blockade (ICB) to the chemotherapy regimen, highlighting the promising potential of immunotherapies for CCA treatment. Despite these developments, most patients still do not benefit from current treatments, and response rates to ICB monotherapy remain modest. This underscores the need to develop more effective immunotherapeutic strategies. A major obstacle to this is the highly heterogenous nature of the disease. CCA tumors exhibit high inter-tumor heterogeneity in terms of anatomical locations, driver mutations, etiologies, and tumor microenvironment (TME) composition, making each patient immunologically distinct and difficult to benefit from a one-fit-all approach. There is a need to stratify patients according to individual disease status to identify immunotherapies and combination therapies that are most beneficial to them. Here we describe the different ways inter-tumor heterogeneity may arise in CCA, including stromal cell abundance, anatomical location, driver mutations, etiologies, TME profile, and tertiary lymphoid structure (TLS) presence. We also discuss what these factors mean to the immune microenvironment and their potential to be used as biomarkers. Careful stratification of patients is crucial in designing personalized medicine to improve survival outcomes and treatment efficacy for CCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb66e953d9482069c69df6565960e118d42cf3b5" target='_blank'>
              Stratifying Cholangiocarcinoma: Tumor Microenvironment, Molecular Drivers, and Novel Immunotherapeutic Approaches.
              </a>
            </td>
          <td>
            Cindy Xinqi Liu, Carmen Chak-Lui Wong
          </td>
          <td>2025-10-14</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Tertiary lymphoid structures (TLSs) are ectopic lymphoid aggregates that form in non-lymphoid organs, frequently observed in conditions such as cancer, autoimmune diseases, transplant rejection, and chronic inflammation. Growing evidence suggests that TLSs are beneficial for patients’ prognosis with higher TLS density generally correlating with improved therapeutic response and survival outcomes across malignancies and might serve as a novel therapeutic target for cancer immunotherapy. However, the correlation between TLSs and tumor development is still ambiguous. The exact timing of TLS formation during tumorigenesis and their dynamic evolution throughout tumor progression remain under investigation. Recent studies have identified potential strategies for inducing TLSs, but there remains a considerable distance from clinical application. More advanced techniques such as high-resolution spatial multi-omics technologies combined with big data analysis will benefit understanding the complex interactions within TLSs and developing novel immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d68cd72d4e06f1532031fad0eb22d59671e56af" target='_blank'>
              Orchestration of tertiary lymphoid structures: decoding developmental mechanisms for next-generation cancer immunotherapies
              </a>
            </td>
          <td>
            Wenqin Hao, Ying Zhao
          </td>
          <td>2025-09-12</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Despite advances in immunotherapy, durable responses in lung cancer remain limited to a subset of patients, underscoring the need for biomarkers capturing spatial immune-tumor interactions. Current methods, such as PD-L1 immunohistochemistry, suffer from sampling bias and fail to decode dynamic immune evasion mechanisms non-invasively. Methods We developed a radiomics framework integrating longitudinal tumor growth kinetics (log volume change rate, LVCR) with deep learning to: (1) delineate tumors via medical knowledge-guided segmentation; and (2) derive an Immune Evasion Score (IES) predicting immunosuppressive niches. The model employs immune-aware attention gates (IAAG) to prioritize regions associated with aggressive growth (high LVCR) and immune evasion. Results Validated on 420 CT scans, our approach achieved superior segmentation accuracy (Dice=0.7728 ± 0.03; HD95 = 9.8 ± 1.5 mm) over existing models. Critically, the IES predicted PD-L1 expression (AUC = 0.85; *p*<0.001) and CD8+ T-cell exclusion (*p*<0.01). High IES correlated with rapid immunotherapy progression (HR = 2.3, *p*=0.004), and spatial analysis confirmed 72.3% concordance between IAAG-prioritized regions and pathological PD-L1+ niches. Conclusion This work establishes a non-invasive paradigm for mapping immunosuppressive microenvironments, bridging precision radiotherapy with immunotherapy personalization. The IES provides a dynamic biomarker of immune evasion, potentially guiding patient stratification for checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7469972933375774141af2efc5f077d7dd79ab2e" target='_blank'>
              AI-based non-invasive profiling of the tumor immune microenvironment using longitudinal CT radiomics predicts immunotherapy response in lung cancer
              </a>
            </td>
          <td>
            Guan-Yuan Liu, Xiaoyan Zhang, Yutong He, Di Liang, Shaonan Xie, Ning Zhang, Nan Geng, Liwen Zhang, Yajie Huang, Fang Liu, Qingyi Liu
          </td>
          <td>2025-10-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8be9d1c5a1b99f57bdae54da32d36142c606a15" target='_blank'>
              A unified analysis of cell-type and trajectory-associated pathways in single-cell data using Phoenix
              </a>
            </td>
          <td>
            Yudit Halperin, D. Nachmani, Michal Rabani
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Glioblastoma (GB) is a highly aggressive brain tumor with poor prognosis and limited response to immunotherapies. Tumor intrinsic features like low mutational burden, and extrinsic factors such as T-cell dysfunction and an immunosuppressive microenvironment, contribute to resistance. To better study immune-tumor interactions in a clinically relevant context, we have developed patient-derived tumor organoids (PDTOs) that support co-culture with autologous and CAR-engineered immune cells, enabling real-time functional analysis.



 PDTOs were generated from freshly resected GB tissue, preserving tumor-specific architecture and molecular features. Patient-matched PBMCs were isolated via density gradient centrifugation, and immune cells were enriched using negative selection with MACS sorting. T cells were expanded ex vivo and characterized by flow cytometry for activation and exhaustion markers, although not yet tested in co-culture. Cytokine secretion was analyzed via ELISA to complement phenotypic profiling. For functional studies, EGFR/EGFRvIII-retargeted CAR-NK cells were generated via lentiviral transduction of primary PBMC-derived NK cells and expanded ex vivo in the presence of (low levels of) IL-15. These cells were co-cultured with 3D tumor spheroids at defined effector-to-target ratios. Immune infiltration and tumor cell killing were assessed using real-time live-cell imaging and fluorescence-based readouts.



 As proof of concept, we co-cultured EGFR-diploid and EGFR-amplified 3D GB spheroids with naïve and CAR-NK cells targeting EGFR derived from a healthy donor. CAR-NK cells showed enhanced cytotoxicity against EGFR-amplified spheroids, demonstrating receptor-dependent immune activation. These results confirm the feasibility of functional immune co-culture assays and support the ongoing integration of this approach into our organoid-based platforms.



 Our organoid-immune cell co-culture system offers a physiologically relevant platform to investigate GB immunobiology. Future applications include drug screening and CRISPR-based perturbation to identify key modulators of immune response, with the goal of informing personalized immunotherapeutic strategies. Observations from the CAR2BRAIN trial (NCT03383978) have shown CD4⁺ and CD8⁺ T cell recruitment following CAR-NK therapy, highlighting the clinical relevance of combinatorial immune approaches and supporting the use of our model to explore such mechanisms ex vivo.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb41d2ad01808f179f77775b88ffca7888d1d44" target='_blank'>
              P18.15.A IMMUNOCOMPETENT GLIOBLASTOMA ORGANOIDS FOR PERSONALIZED IMMUNE PROFILING
              </a>
            </td>
          <td>
            T. Koenig, L. M. Reindl, P. Zeiner, K. Weber, A. Rombach, M. Armbrust, P. Cakmak, W. Wels, T. Broggini, K. Plate, M. Czabanka, J. Steinbach, M. Burger, E. Ullrich, T. Buhlmann, A. Hau
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78a06fd992b15a6520a4d4a8e38c9a04a5e3b92" target='_blank'>
              Spatially-resolved Photoproximity Profiling of MYC Identifies a MYC-BAF Liability in Cancer Cells
              </a>
            </td>
          <td>
            Anthony J. Carlos, Shuyuan Huang, Dongbo Yang, Colin Swenson, Pratyasha Chakraborty, Shaopeng Yu, Benjamin D. Stein, Raymond E. Moellering
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Objectives: Conventional two-dimensional (2D) cell cultures offer valuable insights into cancer cell biology; however, they lack in replicating the complex interactions present in solid tumors. Therefore, research has shifted towards the development of three-dimensional (3D) cell models that recapitulate the dynamic cell–cell and cell–matrix interactions within the complex tumor microenvironment (TME), better resembling tumor growth and initial stages of dissemination. Extracellular matrix, a key component within the TME, regulates cell morphology and signaling, influencing key functional properties. Breast cancer remains the most frequently diagnosed cancer type in women and a leading cause of cancer-related mortality. Methods: The aim of the present study was the development of breast cancer cell-derived spheroids, utilizing two breast cancer cell lines with differential estrogen receptor (ER) expression profile, and their characterization in terms of morphology, functional properties, and expression of epithelial-to-mesenchymal transition (EMT) markers and matrix signatures implicated in breast cancer progression. To this end, the ERα-positive MCF-7, and ERβ-positive MDA-MB-231 breast cancer cell lines were utilized. Results: Our findings revealed notable phenotypic transitions between 2D and 3D cultures, which were further supported by differential EMT markers expression. Moreover, spheroids exhibited distinct expression profiles of key receptors [ERs, epidermal growth factor receptor (EGFR) and insulin-like growth factor receptor (IGF1R)] and matrix molecules (syndecans, and matrix metalloproteinases), accompanied by altered functional cell properties. Bioinformatic tools further emphasized the interplay between the studied matrix regulators and their prognostic relevance in breast cancer. Conclusions: Overall, this study introduces a simple yet informative 3D breast cancer model that captures key TME features to better predict cell behavior in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454047e64cbe106ca9e24ee78bd584b4344b2361" target='_blank'>
              Spheroid-Based 3D Models to Decode Cell Function and Matrix Effectors in Breast Cancer
              </a>
            </td>
          <td>
            Sylvia Mangani, Christos Koutsakis, Nikolaos E. Koletsis, Z. Piperigkou, Marco Franchi, Martin Götte, Nikos K. Karamanos
          </td>
          <td>2025-10-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Brain metastases from breast cancer (BCBM) are highly fatal malignancies with limited treatment options. Developing preclinical models that accurately replicate metastatic phenotypes is essential for advancing next-generation therapies. However, existing BCBM models often fail to capture patient heterogeneity or clinically relevant metastatic biology, limiting their translational potential.



 To address this gap, we established a cohort of clinically relevant BCBM models that reflect the heterogeneity and complexity of the disease. We collected 24 molecularly and clinically diverse BCBM samples, dissociated them into single cells, and intracranially injected them into CD1 nude mice. Half of these samples exhibited tumorigenicity, and the resulting xenografts were harvested, dissociated into single-cell suspensions, and cultured under conditions optimized for cancer stem cells (CSCs), potentially enriching for metastasis-initiating cells (MICs).



 MICs displayed key stem-like properties, including the expression of stem and breast cancer markers, self-renewal capacity, and tumorigenic potential in orthotopic models. Upon intracardiac or intranipple injection, MICs successfully colonized the brain and other organs, primarily the bones, recapitulating the metastatic cascade observed in patients. MIC-driven brain metastases closely mirrored patient tumor histology and cytoarchitecture.



 Brain tropism appeared to be mediated by enhanced MIC adhesion to brain endothelium and interactions with the brain microenvironment, which promoted MIC survival and outgrowth. To assess the fidelity of these models, we performed whole genome and RNA sequencing on matched patient samples, MICs, and MIC-driven brain metastases, revealing a strong similarity to the corresponding patient tumors. Notably, MICs adapted their transcriptomic profiles according to their specific metastatic niche—brain, bone, or mammary fat pad. Additionally, MICs demonstrated high chemoresistance, underscoring their potential as a reliable model for identifying novel therapeutic target
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d344f2bc4f39274fde840c3b13bcc93091044119" target='_blank'>
              P01.27.A METASTASIS-INITIATING CELL SUBPOPULATION FOR MODELING BREAST CANCER BRAIN METASTASES
              </a>
            </td>
          <td>
            M. Del Bene, D. Osti, S. Faletti, C. Richichi, M. Patanè, B. Pollo, G. Carone, F. DiMeco, G. Pelicci
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a highly heterogeneous malignancy characterized by significant variations in gene expression among distinct cell populations. Traditional sequencing technologies often obscure these differences, necessitating advanced approaches for a comprehensive understanding. Single-cell RNA sequencing (scRNA-seq) and Mendelian randomization (MR) analysis have emerged as powerful tools for unraveling the intricacies of PCa, enabling the identification of unique biological features at the individual cell level. In this study, droplet-based scRNA-seq analysis was performed on 4 PCa samples and 4 normal adjacent to tumor samples. The dataset, obtained from the NCBI GEO database, comprised 13,139 cells, facilitating the identification of diverse cell types and their respective gene expression patterns. Robust analysis pipelines, including pseudotemporal ordering, regional plot analysis, MR analysis, and comprehensive gene expression dynamics, were employed to investigate the tumor microenvironment, immune landscape, and potential causal associations with PCa susceptibility. Our scRNA-seq analysis revealed altered immune landscapes in PCa, underscoring the significance of T cells, particularly mucosal-associated invariant T (MAIT) cells. The study unveiled heterogeneity within the tumor microenvironment and identified key genes, such as ZFP36L2, associated with PCa susceptibility. MR analysis provided insights into the potential causal effects of MAIT cells on PCa, supported by analyses of gene expression dynamics and metabolic landscapes. ZFP36L2 emerged as a pivotal regulator, validated through various experimental approaches, including real-time quantitative polymerase chain reaction, western blot, and immunohistochemistry. Our findings underscore the significance of MAIT cells, presenting a novel diagnostic modality for advancing the diagnosis and treatment strategies of PCa. The identified genes, particularly ZFP36L2, offer potential avenues for precision medicine and targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121d713d54f78ea6b18adcb73c968f4917204d87" target='_blank'>
              Decoding the association between mucosal-associated invariant T cells and prostate cancer via multi-omics analysis
              </a>
            </td>
          <td>
            Lichuan Lai, Rensheng Huang, Keke Si, Junming Huang, Lianli Yin
          </td>
          <td>2025-10-03</td>
          <td>Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 There are four established tumor cell states in glioblastoma including astrocyte-like, oligodendrocyte precursor cell-like, neural progenitor cell-like, and mesenchymal-like states. Single-cell RNA sequencing has showed that they are all present with varying distribution between patients. However, since surgery of glioblastoma follows the principle of maximal safe resection, studies have been mainly limited to the necrotic core and contrast-enhancing zone.



 Here, we investigated the cellular organization of the transcriptional states within and outside of the contrast-enhancing zone. This was achieved through a unique set of supramarginal resected glioblastoma samples using large-scale single-cell RNA sequencing of fresh tissue by systematic multi-region sampling from a large number of spatial locations based on 5-ALA fluorescence.



 We discovered that the mesenchymal state is region-specific, highest in the core, at the center of the tumor, shows a gradual decrease with distance from the core and is absent in the low-fluorescent zone and in the periphery, regardless of direction. More importantly, the mesenchymal signature is not only present in tumor cells but also in various other cell types including microglia, macrophages, oligodendrocytes and astrocytes.



 Altogether, this argues that the mesenchymal state is extrinsic, driven by the macro environment, can be induced in completely different cell types and its regional distribution suggests wound-healing and hypoxia to be the main drivers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d224367def8e1deca00ec99d840b021d59cf13" target='_blank'>
              P02.32.A SINGLE CELL RNA-SEQUENCING OF SUPRAMARGINAL GLIOBLASTOMA RESECTIONS REVEAL THAT THE MESENCHYMAL SIGNATURE IS CONFINED TO THE TUMOR BULK AND IS SHARED ACROSS MALIGNANT AND NON-MALIGNANT CELL TYPES
              </a>
            </td>
          <td>
            J. Kjaer Jacobsen, L. Hu, C. Mannens, K. Siletti, O. Persson, S. Linnarsson
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults and associated with a poor prognosis and a median survival of approximately 15 months, largely due to the lack of effective therapies and inevitable recurrence. A key challenge is the spatial and cellular heterogeneity, evident on MRI as a necrotic center, a contrast-enhancing tumor core, and a non-enhancing, T2/FLAIR-hyperintense infiltration zone. The diffuse infiltration of tumor cells into surrounding brain tissue impedes complete surgical resection and drives relapse from persisting residual cells. While regional molecular differences have been well characterized, their functional implications - particularly drug sensitivities - remain insufficiently understood.



 To address this, we performed MRI- and 5-aminolevulinic acid (5-ALA)-guided tissue sampling from the tumor core and from the infiltration zone of patients with glioblastoma. We utilized pharmacoscopy—a single-cell, microscopy-based drug screening platform—to evaluate region- and tumor cell type-specific drug responses. Cancer cell types were defined based on differential expression of nestin (marking stem-like cells) and S100B (marking more differentiated tumor cells).



 Profiling a library of 59 drugs across region-specific tumor samples from 23 patients revealed pronounced heterogeneity in drug responses across patients, tumor regions and tumor cell states. To overcome this heterogeneity, we computationally predicted complementary drug pairs and validated 20 drug combinations in an independent cohort. The combinations demonstrated enhanced efficacy compared to single agents and effectively addressed glioblastoma’s spatial, cellular, and inter-patient heterogeneity. Notably, drug combinations of oncology and neuroactive drugs outperformed those involving agents from the same drug class (oncology + oncology or neuroactive + neuroactive). Ongoing mechanistical explorations reveal first insights into mechanisms of the complementary drug efficacy.



 Our findings underscore the limitations of monotherapies for patients with glioblastoma and highlight the potential of rationally designed drug combinations tailored to the tumor’s spatial and functional complexity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60062d1c4566c5af52ee0940de47f3ed99103b0" target='_blank'>
              OS12.5.A SPATIALLY RESOLVED FUNCTIONAL PROFILING FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, S. Lee, I. Sakic, R. Wegmann, J. Mena, F. Vasella, J. Friesen, L. Regli, E. Le Rhun, A. Zeitlberger, N. Tatari, G. Hutter, M. Neidert, M. Weller, B. Snijder, T. Weiss
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b221cace1b09faa76330c09ce1ca8391e4017e" target='_blank'>
              GLIS3 Marks a Neural-like Progenitor Cell State that Drives Metastasis in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Dennis Gong, Jimmy A. Guo, Jennifer Su, Sueda Cetinkaya, Connor J Hennessey, Patrick Yu, Ananya D. Jambhale, Peter L. Wang, Nicholas J Caldwell, Ashley Lam, Junning Wang, C. Shiau, Kevin S. Kapner, Laleh Abbassi, Seema Chugh, Shatavisha Dasgupta, Jonathan O Nowak, B. Wolpin, M. L. Zhang, Mari Mino-Kenudson, T. Jacks, Andrew J. Aguirre, William L. Hwang
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is an aggressive cancer with high recurrence rates and prevalent resistance to therapeutic interventions. Tumor behavior is largely dependent on the tumor microenvironment (TME) that includes immune cells, stromal components, cancer-associated fibroblasts (CAFs), the extracellular matrix (ECM), and an associated cytokine network. In this review, we examine principal mechanisms of the tumorigenic transformation, encompassing immune checkpoint disruption, therapy resistance mediated through CAFs, the contribution of hypoxic niches, and several metabolic dependencies that hold potential as future targets. Novel therapeutics developed and/or repurposed, such as immune checkpoint inhibitors (ICIs), TME modulation therapeutics, CAF reprogramming, hypoxia targeting agents, and ECM remodeling, aim to overcome TME-mediated resistance. We also examine the rationale and progress of integrating TME-targeted therapies with other treatment modalities. By identifying actionable, molecular targets within the HNSCC TME, this review presents a translational perspective for implementing TME modulation in personalized treatment. The challenges comprise TME heterogeneity, a paucity of predictive biomarkers, and a translational gap between pre-clinical and clinical practice. Future studies must be aimed at proper stratification of patients, optimization of combination treatment, and cost-effectiveness analysis of TME-modifying therapies to enable personalized medicine in HNSCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93cea5d225fa79d6bdb8a81392ca2d717a53a33c" target='_blank'>
              Reprogramming the Tumor Microenvironment in Head and Neck Squamous Cell Carcinoma: Therapeutic Targets and Innovations
              </a>
            </td>
          <td>
            Bruno Špiljak, Bojan Poposki, Stjepanka Lešić
          </td>
          <td>2025-10-22</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Treatment of locally advanced unresectable pancreatic cancer remains a major clinical challenge due to pronounced heterogeneity and resistance to standard regimens. Increasing evidence highlights the critical role of the tumor microenvironment (TME) in shaping therapeutic response and driving drug resistance. In this minireview, we summarize recent advances in TME phenotyping and its potential to guide precision therapy. A four-dimensional framework integrating stromal, immune, genomic, and metabolic features has been proposed to better characterize TME heterogeneity. Preclinical and clinical studies indicate that strategies targeting the stroma, modulating immunity, or exploiting genomic vulnerabilities such as homologous recombination deficiency may enhance the efficacy of chemotherapy, immunotherapy, and targeted agents. Dynamic biomarkers, including circulating tumor DNA and carbohydrate antigen 19-9, also show promise for real-time therapy adaptation, although their clinical application remains limited. By synthesizing current evidence, we emphasize the importance of individualized treatment strategies that account for TME complexity. While encouraging, the translation of multiomics phenotyping and biomarker monitoring into routine clinical practice requires standardization, prospective validation, and integration of novel technologies. Future research should focus on establishing reproducible TME-guided models to enable dynamic and personalized therapy for patients with unresectable pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9ebed50c75b679e8298f471ae2f8ee5933e4197" target='_blank'>
              Tumor microenvironment phenotyping guides precision therapy in unresectable pancreatic cancer
              </a>
            </td>
          <td>
            Kai Zhao, Ming-Ming Xiao, Yong-Sheng Yang, Xiao Xiao
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Hypoxia is a critical driver of tumor aggressiveness in high-grade gliomas, yet its cell-type-specific effects on immune cell populations within the tumor microenvironment (TME) remain poorly understood.



 We investigated how hypoxia shapes the spatial distribution and functional states of monocyte-derived macrophages (MDMs) and brain-resident microglia (MG) in diffuse astrocytomas and glioblastomas (GB) by using cyclic immunohistochemistry (cIHC), single-cell RNA sequencing, spatial transcriptomics, and in vitro cell culture models.



 Hypoxia induces divergent responses in these myeloid subsets, driving spatial immune patterning. CD163⁻ MDMs dominate hypoxic niches in GB, while MG populations are excluded from these regions, correlating with hypoxia-induced TNF upregulation, stress response signatures, and dampened interferon responses. Conversely, MDMs exhibit a shift towards hypoxia-associated immunosuppressive traits and altered metabolism. GBs display elevated hypoxic intensity compared to diffuse astrocytomas, as supported by hypoxia-response gene expression in TCGA dataset. In vitro, MG show heightened sensitivity to hypoxia-associated acidity compared to MDMs, suggesting together with microglial hypoxia response genes that their exclusion from hypoxic zones results from intrinsic vulnerability.



 Our findings reveal that hypoxia remodels the TME by promoting immunosuppressive MDM accumulation and depletion of MG, creating spatially distinct immune landscapes that may underlie glioma progression. These results highlight hypoxia-driven immune dysregulation as a therapeutic target and underscore the importance of cell-type-specific strategies to counteract TME-driven immunosuppression in malignant gliomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad214ece57abc6b1e7d242c9f8b34d0a20415450" target='_blank'>
              P02.29.B HYPOXIA SHAPES TUMOR IMMUNE MICROENVIRONMENT THROUGH CELL-TYPE DEPENDENT RESPONSES IN DIFFUSE ASTROCYTOMAS
              </a>
            </td>
          <td>
            A. Tiihonen, I. Salonen, I. Koivisto, A. Ritamäki, S. Jaatinen, T. Hyvärinen, J. Lotila, J. Kreutzer, M. Valkonen, G. Karabiyik, S. Mäntylä, M. Mohammadlou, M. Hoikka, J. Beck, R. Rölz, M. Marttinen, M. Nykter, J. Haapasalo, P. Ruusuvuori, S. Parkkila, P. Kallio, S. Hagman, J. Kesseli, V. Ravi, H. Haapasalo, A. Jukkola, K. Joseph, K. J. Rautajoki
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Vascular mimicry (VM) refers to the formation of vessel-like structures by tumor cells independent of endothelial cells. These VM channels connect to the host’s vascular network and are associated with aggressive tumors and poor patient prognosis. Most VM research has been conducted on melanoma, relying on patient-derived and mouse cell lines. In other solid tumors, VM studies rely on human cell lines, which have certain limitations for in vivo studies. Specifically, most in vivo VM research involving human cells uses subcutaneous mouse models that fail to recapitulate organ-specific tumor microenvironments. As the microenvironment is an essential driver of tumor vascularization, including VM, murine cell lines could facilitate VM investigations in syngeneic mouse models. Here, we present CT26 and KPC, well-characterized murine colorectal and pancreatic cancer cell lines, as cell models for VM investigations. Using in vitro cell-based assays, we demonstrate that CT26 and KPC undergo VM, a cell-intrinsic process that is enhanced by serum deprivation and exposure to hypoxia and is independent of tumor-secreted growth factors. Additionally, we demonstrate the importance of YAP/TAZ signaling in VM formation, as inhibition at non-cytotoxic concentrations attenuated VM formation. Remarkably, CA3, the most potent of the two inhibitors, significantly reduced cell proliferation in both cell lines at the IC50 concentration. This reduction in cell proliferation was associated with the induction of apoptosis in CT26 cells and changes in the cell cycle in both CT26 and KPC cells. Finally, dual YAP/TAZ knockdown in both cell lines significantly abrogated VM formation, validating our initial findings using inhibitors. These results show that CT26 and KPC cells undergo VM, and given their extensive use in cancer research, can be used to investigate VM in vivo using syngeneic models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec4fb2cfcbdb76b0840a4ef6bfa2d3738d1e6f07" target='_blank'>
              Murine Cell Line Models for Vascular Mimicry: The Role of YAP/TAZ Signaling
              </a>
            </td>
          <td>
            Matilde Righetti, Ana-Maria Primorac, J. T. Erler, V. Oria
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: Hepatoblastoma is the most prevalent liver cancer in children. Immunotherapy targeting immune checkpoint molecules has become pivotal in various cancer treatments. However, the clinical significance of immune checkpoint ligands in hepatoblastoma remains largely unclear due to various challenges. This study sought to first characterize the expression profile of the immune checkpoint ligand CD276 in hepatoblastoma and assess its potential as a predictor of malignant characteristics and regulator of neutrophil infiltration. Methods: Univariable and multivariable logistic regression analyses were performed to evaluate the clinical significance of immune checkpoint ligands in the bulk RNA-seq dataset and develop a novel predictive model for malignancy. Furthermore, single-cell RNA sequencing (scRNA-seq), immunohistochemistry (IHC), deconvolution analysis, and correlation analysis were employed to characterize the expression pattern of CD276 and explore its influence on the tumor immune microenvironment. Results: The bulk RNA-seq analysis revealed CD276 transcript levels were significantly elevated in hepatoblastoma tissues, especially in patients with more aggressive malignant phenotypes. Furthermore, we developed a predictive model based on a risk score and constructed a user-friendly nomogram to predict patient metastasis by integrating CD276 levels with clinical features. Both scRNA-seq and multiplex immunohistochemistry (mIHC) analyses confirmed that CD276 is highly expressed, predominantly in cancer-associated fibroblasts. Our results also demonstrated that CD276 levels correlate with immune infiltration in hepatoblastoma, and that CD276 regulates CXCL2 to modulate neutrophil infiltration, suggesting a potential mechanism underlying the role of CD276 in hepatoblastoma malignancy. In addition, we validated that CD276 in cancer-associated fibroblasts promotes hepatoblastoma growth in mice. Conclusions: Our findings highlight the critical role of CD276 in hepatoblastoma malignancy, potentially providing novel insights and therapeutic targets for the development of combinatorial immunotherapy strategies for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99a7aab8c1bd194b8fba818f13ec5be139a16bc2" target='_blank'>
              Integrated analysis identifies CD276 in fibroblasts as a malignancy predictor and regulator of neutrophil infiltration in hepatoblastoma
              </a>
            </td>
          <td>
            M. Ding, Guoqing Zhu, Tianshu Chen, Jiabei Zhu, Siwei Mao, xiaocheng Tang, Han Wu, Ni Zhen, Fenyong Sun, Qiuhui Pan, Ji Ma
          </td>
          <td>2025-10-31</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most common brain cancers diagnosed in children, often leading to life-long neurological impairments. Whilst targeted inhibitors exist, their effects are often short-lived, highlighting the urgent need for innovative treatments. Notably, some pLGG tumors spontaneously regress with age, suggesting that changes in the tumor microenvironment can arrest tumor growth. In this work, we aim to characterize which signaling interactions promote the growth of pLGG cells and how these interactions change dynamically with time. To achieve this, we have optimized a protocol for introducing pLGG mutations into developing mouse brains via in utero electroporation. This study revealed that brains harboring the common KIAA1549::BRAF fusion mutation have a striking phenotype characterized by increased astrocyte reactivity and myeloid cell infiltration. To better understand how signals derived from immune cells may impact pLGG cell fitness, we are additionally performing a high-throughput cytokine screen. We are using barcoded cytokine constructs tethered to the cell membrane to comprehensively screen the impact of cytokine signaling on the growth of neural stem cells engineered to overexpress common pLGG mutations. Together this work aims to unravel which signaling factors govern the age-dependent growth of pLGG tumors, which may reveal targetable aspects of the tumor microenvironment amenable to therapeutic intervention.



 Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy Chang, Ria Kedia, Michelle Boisvert, David Jones, Timothy Phoenix, Pratiti Bandopadhayay. Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f149e2a5701b9900d3e20ea1c251a5956b1b56d2" target='_blank'>
              Abstract B037: Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy H. Chang, Ria Kedia, Michelle Boisvert, David Jones, T. Phoenix, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Bladder cancer (BC) is among the ten most common malignant tumors worldwide. Tumor stem cells contribute significantly to postoperative recurrence and disease progression. Understanding stem cell interactions with other tissue cells and developing a prognostic model may improve BC management. Through single-cell RNA sequencing, we identified tumor stem cells in BC tissue. We identified 91 genes specifically upregulated in the stem cell cluster versus other clusters. Of these, 67 whose high expression correlated with poorer patient survival were defined as high-risk stemness genes (HRSGs). Further analysis showed that the MDK-LRP1 axis is the primary communication pathway between stem cells and other cell types, involving 31 of these HRSGs. Based on these 31 HRSGs, patients were stratified into two stemness clusters (ST cluster A and B), with ST cluster B associated with poorer prognosis. We further selected prognosis-related genes from differentially expressed genes between the clusters to construct a risk model. Patients in ST cluster B exhibited higher risk scores, aligning with clinical outcomes. Among the HRSGs, ACTN1 emerged as a key gene, showing elevated expression in patients with poor survival and advanced disease stages. Immunohistochemistry confirmed significantly increased ACTN1 protein levels in BC tissues. Additionally, protein interface analysis indicated that the Cys104 residue of Midkine potentially interacts with both LRP1 and ACTN1 within a 5 Å distance, suggesting a critical interaction site. These findings provide novel insights into stem cell–mediated BC progression and offer potential prognostic and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a252bfa991c01d37bb86d8900bab209b05cf3f1" target='_blank'>
              Single-cell stemness analysis highlights Midkine-LRP1 pathway and identifies new bladder cancer subtypes
              </a>
            </td>
          <td>
            Cheng Qian, Jin Qin, Zixian Lin, Shenggan Wang, Anmei Deng, Tengda Li
          </td>
          <td>2025-10-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Glioblastoma is one of the most aggressive brain cancers, marked by rapid progression and resistance to treatment. Its cellular heterogeneity and invasive nature make it particularly hard to manage. New tools are needed to better understand its biology and improve diagnostics and therapies. NADH Fluorescence Lifetime Imaging Microscopy (NADH-FLIM) is a promising technique that reveals metabolic states by measuring NADH fluorescence lifetime: shorter lifetimes indicate glycolysis, while longer lifetimes are linked to oxidative phosphorylation (OXPHOS). This enables the identification of metabolic heterogeneity in tumor tissues.



 We applied NADH-FLIM to formalin-fixed paraffin-embedded (FFPE) glioblastoma sections to assess NADH lifetime distribution in key tumor structures recognized by pathologists: (1) palisading cells near necrosis, (2) microvascular proliferation, and (3) neuronal and perivascular satellitosis. Additionally, we analyzed tumor cells overexpressing FGFR3 or carrying the IDH1 mutation to study their specific metabolic profiles.



 Each tumor structure showed a distinct NADH lifetime signature, reflecting different metabolic states. For example, microvascular proliferation displayed lifetimes consistent with OXPHOS, while the surrounding tumoral astrocytic areas showed profiles more consistent with glycolysis. FLIM also identified isolated tumor cells in regions appearing histologically normal, including areas of early invasion. In FGFR3-overexpressing and IDH1-mutant cells, FLIM highlighted unique metabolic behaviors associated with these genetic changes.



 NADH-FLIM enables precise mapping of metabolic heterogeneity in glioblastoma, offering insights beyond conventional histology. It can detect metabolically distinct tumor subpopulations, aiding in diagnosis and potentially guiding personalized therapies. This technique represents a significant step forward in understanding glioblastoma biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0eec4df8e18a2d236453737b4e94dd3e9cd94f1" target='_blank'>
              P03.14.A BEYOND CONVENTIONAL PATHOLOGY: ADDING AN EXTRA LAYER OF INSIGHT WITH NADH-FLIM TO REVEAL HIDDEN METABOLIC PROFILES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            M. Giacomarra, M. Morelli, G. Ferri, F. Lessi, S. Franceschi, M. Menicagli, A. D. Di Stefano, O. Santonocito, C. Mazzanti
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Lineage plasticity enables cancer cells to bypass therapeutic pressure, but the underlying chromatin mechanisms remain incompletely defined. A recent study by Lu et al. uncovers how the transcription factors NKX2-1 and FOXA2 reprogram 3D genome architecture to activate neuroendocrine gene networks during the transition from castration-resistant prostate cancer (CRPC) to neuroendocrine prostate cancer (NEPC) (Lu et al., 2025). Importantly, pharmacological inhibition of CBP/p300 markedly suppresses NEPC growth in vitro and in vivo, revealing a promising therapeutic avenue for advanced disease. These findings illuminate how spatial chromatin reorganization can drive cell fate transitions, positioning 3D genome dynamics as a key determinant of cellular identity in both development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c9ebfa025117c138be3767acc9d75dcb26318e" target='_blank'>
              3D chromatin dynamics in cellular neuroendocrine transformation
              </a>
            </td>
          <td>
            Jianhua Xiong
          </td>
          <td>2025-10-17</td>
          <td>Protein &amp; Cell</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite significant efforts, Diffuse Midline Gliomas (DMGs) remain incurable. Although promising results are obtained in preclinical studies, most approaches have failed to improve survival in these young patients. In this context, by integrating elements of the biophysical tumor microenvironment known to regulate the response to therapies, a new preclinical tool is developed to better evaluate the efficacy of antitumoral strategies. For this purpose, a novel DMG-on-Chip (DoC) is engineered composed of a 3D dense tumor disk embedded in an extracellular matrix, which is accessible for real-time monitoring using wide-field phase contrast or confocal fluorescence microscopy. By driving the oxygen supply within the chip solely in a radial manner, a hypoxia gradient is established that can be associated with changes in DMG cell phenotype, proliferation, and rewiring of metabolic and stress transcriptomic signatures. Finally, DoC is used to analyze the spatial heterogeneity of the response of DMG cell lines and patient-derived 3D cultures to treatments through cell segmentation. Altogether, these interdisciplinary characterizations validate the new tool as an interesting healthcare solution to understand how cell responses are modulated by biophysical or biochemical cues in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a25e403c641a7729cb13c93c687243726c53a1" target='_blank'>
              Novel Diffuse Midline Glioma-on-Chip Recapitulating Tumor Biophysical Microenvironment to Assess the Heterogeneity of Response to Therapies.
              </a>
            </td>
          <td>
            Lisa Terrassoux, Calypso Hazard, Agathe Laratte, Joanne Balsamelli, Mélanie Arcicasa, Elisa Fong, Léo Baland, Fabrice Soncin, Giovanni Cappello, C. Maurage, Sebastien Janel, Antony Bazir, Fabrizio Cleri, Anthony Treizebré, Eric Lartigau, Yousr Rekik, Samuel Meignan, Alessandro Furlan
          </td>
          <td>2025-09-29</td>
          <td>Small</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e8d2f1aaaa7f0e435a0bcfed4046ad25f2a01b" target='_blank'>
              Deciphering macrophage heterogeneity and factors driving M2 polarization in lung adenocarcinoma through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Meiling Sheng, Beiwei Yu, Qunzhi Wang, Yuanchao Xiao, Ming Ling, Xiaoming Wu
          </td>
          <td>2025-10-16</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38e3e2e1b934125c90539e7b6826f0bb1928fb1d" target='_blank'>
              Image-based phenotypic sorting of synthetic cells
              </a>
            </td>
          <td>
            Marijn van den Brink, Marlena Stam, Nico J. Claassens, C. Danelon
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Purpose This study evaluated the dynamic changes in the tumor microenvironment (TME) in patients with non-small cell lung cancer (NSCLC) and acquired resistance to epidermal growth factor receptor (EGFR)-tyrosine kinase inhibitors (TKIs) using an artificial intelligence (AI)-powered spatial TME analyzer. We then assessed the predictive efficacy of immune-checkpoint inhibitors (ICIs)-based treatment. Experimental design An AI-powered whole-slide image analyzer was used to segment cancer areas (CAs) and cancer stroma and to identify tumor-infiltrating lymphocytes (TILs), tertiary lymphoid structures, fibroblasts, and endothelial cells (ECs) in the tumor tissue. We analyzed 143 NSCLC samples after resistance to EGFR-TKIs from two cohorts: (1) 89 patients treated with ICI monotherapy and (2) 54 patients from the ATTLAS phase III trial comparing atezolizumab plus bevacizumab, paclitaxel, and carboplatin (ABCP) versus pemetrexed plus carboplatin. Results Post-TKI samples showed reduced TILs in the CA (p=0.045) and increased ECs in the CA (p=0.005) compared with pre-TKI samples. These changes differed according to EGFR mutation subtype. Higher TILs in CA were associated with a better overall response rate (ORR) and progression-free survival (PFS). Similarly, higher EC levels in CA correlated with improved ORR and PFS. In the ATTLAS cohort, these factors were associated with clinical benefits from ABCP, with a significant association with TILs and a marginal association with ECs. Conclusion Our findings suggest that EGFR-TKIs affect the immune landscape of patients with EGFR-mutated NSCLC. Higher TILs or ECs in the CA were significantly associated with a favorable response to subsequent ICI-based treatment. Trial registration number NCT03991403.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2c4a8b17d5db469739630d4675ac335f2f2dfd8" target='_blank'>
              Artificial intelligence-powered spatial analysis of tumor microenvironment in patients with non-small cell lung cancer with acquired resistance to EGFR tyrosine kinase inhibitor
              </a>
            </td>
          <td>
            Yeong Hak Bang, Geun-Ho Park, Jin Woo Oh, Soohyun Hwang, , Boram Lee, Cheolyong Joe, Hyemin Kim, Jinyong Kim, Sehhoon Park, Hyun Ae Jung, Jong-Mu Sun, J. Ahn, Myung-Ju Ahn, Yoon-La Choi, Changho Ahn, S. Ali, C. Ock, Se-Hoon Lee
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains a highly lethal malignancy, in part due to its heterogeneous tumor microenvironment. Most PDAC tumors are poorly vascularized and embedded within densely fibrotic stroma; however, in prior computational analyses, we identified a subset of PDAC tumors with hyper-vascular features. In this study, we sought to determine the extent of vascular heterogeneity in PDAC and its functional consequences. Analysis of human tumor specimens revealed that approximately 10-20% of human tumors display features of hyper-vascularity. Using a panel of clonal murine cell lines that recapitulate this heterogeneity, we demonstrate that VEGFA-mediated angiogenesis does not account for the increased vessel density. Functionally, we further show that hyper-vascular tumors display enhanced sensitivity to both chemotherapy and anti-angiogenic therapy. These findings help explain the limited efficacy of VEGFA-targeted therapies in unselected PDAC cohorts and highlight vessel density as a potential biomarker for patient stratification.



 Samantha B. Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo I. Orlen, Samuel D. Zwernik, Kailee M. Hartway, Nina G. Steele, Agnieszka K. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger. Vascular heterogeneity influences treatment response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cec4b9af7cf61e156104d3e634ae5d707e4e9c" target='_blank'>
              Abstract A127: Vascular heterogeneity influences treatment response in pancreatic cancer
              </a>
            </td>
          <td>
            Samantha B Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo Orlen, Samuel D Zwernik, Kailee M Hartway, Nina G. Steele, A. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Merging evidence suggests that cellular fitness competition between malignant cells and the tumor microenvironment (TME) dictates tumor progression. We recently identified Flower isoforms as molecular “fitness fingerprints,” where cancer cells expressing Flower-Win gain dominance over surrounding cells marked with Flower-Lose. In this study, we uncover that loss of stromal fitness is not an inherent trait but rather a cancer-driven event. Tumor cells secrete a specific long non-coding RNA, Tu-Stroma, via exosomes, which reprograms splicing of the Flower locus in TME cells, enforcing Flower-Lose expression and thereby reducing stromal competitiveness. This cancer-induced shift establishes a permissive niche that accelerates growth, metastatic dissemination, and ultimately shapes host survival in ovarian cancer. Therapeutically, blockade of Flower using a fully humanized monoclonal antibody (Aenya) markedly suppressed tumor expansion and peritoneal spread in patient-derived xenograft models. Importantly, prophylactic administration preserved the integrity of intraperitoneal organs even in the presence of highly aggressive tumor cells. These findings highlight a previously unrecognized mechanism of cancer-driven stromal takeover and establish Flower as a first-in-class therapeutic target with broad implications for preventing metastatic colonization.



 Rajan G. Gogna. Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0864f00ed3ab31470a0c2e77e5565e26d50bb11a" target='_blank'>
              Abstract IA008: Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer
              </a>
            </td>
          <td>
            Rajan Gogna
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-04</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pre-invasive breast cancer, or ductal carcinoma in situ (DCIS), shares many morphologic and genomic features with invasive breast cancer, yet most DCIS tumors remain indolent over decades. In this study, we performed spatial analyses of somatic mutation patterns in 18 DCIS tumors to infer the underlying growth dynamics. Experimental data combined with mathematical models and Bayesian computation revealed a growth model that mimics the cellular dynamics of branching morphogenesis, which governs normal mammary gland development during puberty. Unlike a traditional model of clonal evolution, this non-competitive growth model allowed for extensive DCIS spread without the selective pressures of subclone competition that promote the emergence of invasive phenotypes. This finding provides a framework to understand the biological basis for the limited progression of DCIS and adds rationale for ongoing clinical efforts to reduce DCIS overtreatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56eaf0d13e80db0fb1f5ee2e2242518bc8cf64f7" target='_blank'>
              Spatially Discontinuous Mutation Topographies in Ductal Carcinoma in Situ Reveal Non-Competitive Growth Dynamics.
              </a>
            </td>
          <td>
            Marc D. Ryser, Matthew A Greenwald, Daniel L. Monyak, I. Sorribes, Lorraine M. King, Allison Hall, J. Geradts, Donald L Weaver, D. Mallo, Shannon T. Holloway, Ethan Wu, Kevin A. Murgas, Robert B. West, Lars J. Grimm, Carlo C. Maley, Jeffrey R. Marks, Darryl Shibata, E. S. Hwang
          </td>
          <td>2025-10-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) exhibits significant transcriptomic diversity, limiting treatment efficacy. This study aims to identify conserved molecular pathways that remain stable in heterogeneous TNBC, providing reliable therapeutic and prognostic targets. Genomic diversity was assessed via Mutant-Allele Tumor Heterogeneity (MATH) in TCGA-TNBC cohorts. Single-cell RNA sequencing from TNBC samples (GSE176078) underwent non-negative matrix factorization (NMF) to identify intratumoral heterogeneity (ITH) metaprograms. Transcriptomic diversity was quantified using PCA-based cell-cell Euclidean distances. CellChat analyzed cellular communication in high- and low-diversity groups. Conserved pathways were identified using KEGG enrichment, ssGSEA, and spatial transcriptomics. And we confirmed the feasibility of targeting sphingolipid metabolism through in vitro and in vivo experiments. TNBC showed elevated genomic diversity, inversely correlated with immune infiltration and pathway activity. Single-cell RNA sequencing revealed ITH metaprograms across malignant subtypes, with Cancer Cycling cells showing proliferative enrichment. High transcriptomic diversity tumors exhibited enhanced Cancer Basal SC interactions with CAFs, promoting stemness and EMT. Multi-region TNBC dataset analysis confirmed sphingolipid metabolism as the most conserved pathway (high interpatient and low intratumoral heterogeneity), enriched in malignant regions and negatively correlated with immune cells. A prognostic model incorporating sphingolipid genes (SPTLC2, SGMS1, SGPP2) stratified high-risk patients. Inhibiting SPHK1 via PF-543 induced dose-dependent cell death. Sphingolipid metabolism could be a transcriptionally conserved pathway in TNBC. Its inhibition may overcome heterogeneity-driven resistance, and its prognostic value enables precise patient stratification, offering a potential strategy against TNBC plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/273a42854b1bca75f9d7c92c18142aeda8983ebb" target='_blank'>
              Conserved sphingolipid metabolism under transcriptomic diversity: a prognostic and therapeutic target in triple-negative breast cancer
              </a>
            </td>
          <td>
            Jian Li, Rui Chen, Xiaowu Wang, Qing Zheng, Ruijie Dai
          </td>
          <td>2025-11-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) represent the main immune cell population in various brain malignancies, but there is rare knowledge on the functional and, in particular, the prognostic role of TAMs in the meningioma (MGM) microenvironment. Here, we investigated TAM frequencies, activation state, survival-associated changes, and their association with tumor-infiltrating T lymphocytes (TILs) in two independent study samples comprising altogether 680 MGMs. To this end, we performed tissue cytometry analyses, quantified tissue cytokine levels, and integrated previously published TIL infiltration and microarray datasets in the discovery cohort comprising n = 195 clinically well-annotated cases. This was complemented by a DNA methylation-based deconvolution approach to predict TAM and TIL infiltration rates using immune cell-specific CpG sites as well as survival associations in an independent validation cohort of n = 485 MGMs. Our findings revealed substantial but heterogeneous TAM infiltration in newly diagnosed MGMs, with increased numbers of pro-tumoral TAMs in clinically aggressive tumors. Additional cytokine and transcriptome analyses corroborated the presence of an immunosuppressive niche in TAM-enriched MGMs. Importantly, a high frequency of pro-tumoral TAMs was associated with poor patient outcome, and high TAM infiltration was further identified as an independent prognostic factor for inferior survival, counteracting the beneficial prognostic effect of TILs. Moreover, methylation-based deconvolution analyses confirmed the opposing prognostic roles of TAMs and TILs in the validation cohort. Altogether, higher numbers of TAMs appear to be a hallmark of clinically aggressive behavior in newly diagnosed and recurrent MGMs. Unlike TILs, immunosuppressive TAMs seem to play a dominant role in the immunological landscape of MGMs with a significant negative impact on patient outcome, highlighting pro-tumoral TAMs to be an attractive treatment target in MGMs. Furthermore, our deconvolution approach presents a pipeline to computationally determine TAM and TIL infiltrates in the MGM microenvironment, which might be highly valuable for patient stratification for future immunotherapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9eb1c2486a92614ef9d3bad76049f5fefd9924e" target='_blank'>
              Tumor-associated macrophages in meningiomas: a novel biomarker for poor survival outperforming the benefits of T cells
              </a>
            </td>
          <td>
            C. Lotsch, R. Warta, Fang Liu, G. Jungwirth, C. Rommel, M. Barthel, K. Lamszus, A. Kessler, Niels Grabe, Mario Loehr, R. Ketter, C. Senft, S. Maas, P. Sievers, Manfred Westphal, Sandro M. Krieg, A. Unterberg, M. Simon, Andreas von Deimling, F. Sahm, D. Raleigh, C. Herold-Mende
          </td>
          <td>2025-10-09</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="


 Multi-modal chemotherapy, surgical resection, and radiation therapy (if applicable) make up the current standard of care for Atypical Teratoid Rhabdoid Tumors (ATRT), but they are insufficient. Up to 50% of patients experience relapse, but no standard treatment options have been established for ATRT recurrence. The malignant cells that persist after chemotherapy treatment may significantly contribute to this tumor rebound. Our principal goal is to track shifts in cell states and phenotypes under the selection of chemotherapy by combining CloneSifter barcoding technology with single cell RNA-seq. Patient derived ATRT cells, barcoded with sgRNA identifiers, are placed under the selection of a sub-lethal drug dose, and barcodes are tracked over 7 days. After treating the cells with the following chemotherapy agents - Cisplatin, Etoposide, Doxorubicin, Methotrexate, and Cyclophosphamide - barcode abundance profiles are compared between pre and post drug treated samples using hierarchical clustering. This analysis reveals that replicates subjected to the same drug treatment tend to show the most similar patterns of barcode abundance, suggesting that this drug persistence is intrinsic and pre-determined prior to treatment rather than stochastically acquired. Furthermore, a one-sided paired Wilcoxon test is leveraged to identify significant barcode enrichment in each of the drug treated replicates relative to their vehicle controls, differentiating between chemo-persistent versus chemo-sensitive barcodes. Competitive barcodes are identified under Etoposide, Cisplatin, and Methotrexate treatments and mapped to the single cell RNA-seq data, where a unique transcriptome profile can be generated for each barcode. Comparisons between transcriptome profiles of enriched versus depleted barcodes under drug treatment will reveal the underlying factors that provide a fitness advantage for chemo-persistent populations. Investigating the dynamics of chemotherapy-persistence in ATRT cell lines can help uncover vulnerabilities that may be exploited to begin overcoming chemotherapy-induced ATRT recurrence.



 Kelly Y Cai, Shriya Rangaswamy, April Apfelbaum, Jared Collins, Molina Das, Joelle Straehla, David Root, Tom Rosenberg, Susan Chi, Rameen Beroukhim, Pratiti Bandopadhayay. Factors Driving Chemo-persistence in Atypical Teratoid Rhabdoid Tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/032d4c5b96b8782c64d45452f34ed6f6fe663882" target='_blank'>
              Abstract B016: Factors Driving Chemo-persistence in Atypical Teratoid Rhabdoid Tumors
              </a>
            </td>
          <td>
            Kelly Y Cai, Shriya Rangaswamy, April A Apfelbaum, Jared Collins, Molina Das, Joelle Straehla, David E. Root, Tom Rosenberg, Susan Chi, R. Beroukhim, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>151</td>
        </tr>

        <tr id="Simple Summary The tumor microenvironment has become a central focus in cancer research, offering new insights into cancer behavior and showing strong potential for improving cancer treatment. However, in veterinary oncology, specifically feline mammary tumors, research on the topic lags far behind. This review summarizes current knowledge on key elements of the tumor microenvironment in the feline mammary tumors, including tumor necrosis, fibrosis, angiogenesis, adipose tissue, tumor-associated inflammation, extracellular vesicles, and epithelial–mesenchymal transition. Although studies in cats remain limited, insights from human breast cancer are frequently drawn, creating a combined perspective that may help drive new research into the diagnostic, prognostic, and therapeutic potential of the tumor microenvironment in feline mammary tumors. Abstract The tumor microenvironment (TME) comprises neoplastic and stromal cells, and extracellular matrix elements, all engaging in a complex interplay that ultimately dictates tumorigenesis, cancer progression, and therapeutic response. While extensive research on the TME has been conducted in human oncology, data on its veterinary counterpart, particularly in feline mammary tumors (FMTs), are still scarce. In this review, we explore current understanding of feline mammary carcinoma (FMC) microenvironment, focusing on tumor necrosis, fibrosis, angiogenesis, adipose tissue tumor-associated inflammation, extracellular vesicles, and epithelial–mesenchymal transition (EMT) and their prognostic implications. In FMC, remodeling of collagen fibers, cancer-associated fibroblasts (CAFs), regulatory T cells (Tregs) and elevated serum leptin have been associated with poor prognosis, whereas stromal cytotoxic T cells correlate with more favorable outcomes. By contrast, findings on necrosis and pro-angiogenic factors remain inconsistent, and research on extracellular vesicles (EVs) is still in its early stages. This review presents insights from human breast cancer (HBC) that further support and elucidate the potential relevance of these TME components. As FMCs are highly aggressive tumors, a deeper understanding of their microenvironment could not only improve prognostic accuracy but also uncover novel therapeutic targets. Furthermore, due to their similarities, FMCs offer a potential valuable spontaneous model for HBC, particularly for the aggressive triple-negative phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acfc9c9171773b52e82ceb2afc7e5c11c59650bc" target='_blank'>
              Inside the Tumor: Decoding the Feline Mammary Tumor Microenvironment and Its Prognostic Value—A Review
              </a>
            </td>
          <td>
            J. Rodrigues-Jesus, Ana Canadas-Sousa, Hugo Vilhena, P. Dias-Pereira
          </td>
          <td>2025-10-01</td>
          <td>Veterinary Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Renal cell carcinoma (RCC) evolves within a chronic inflammatory renal niche, where angiogenesis, metabolism, and immune programs synergize to reshape the tumor immune microenvironment (TIME). Recurrent renal damage and tissue hypoxia sustain NF-κB/STAT3 and HIF-VEGF signaling, while a sustained IFN-γ response enhances antigen presentation while inducing inhibitory checkpoints, promoting a state of “inflammation but constrained.” Single-cell and spatial studies reveal early microenvironment heterogeneity and the chemokine-checkpoint paradox: regions enriched in CXCL9/10 coexist with endothelial inertia, cancer-associated fibroblasts (CAFs)-mediated stromal barriers, and metabolic stress, which collectively exclude functional CD8
 +
 T cells. In advanced clear cell RCC, immuno-inflammatory, immuno-excluded, and immuno-desert phenotypes often coexist and undergo transitions during treatment, leading to heterogeneity in response to immune checkpoint blockade (ICB). We propose a modular perspective-the NF-κB/STAT3, HIF-VEGF, IFN-γ circuits and auxiliary regulatory factors-to link stage-specific biology with treatment matching. Integrative biomarkers couple IFN-γ characteristics with angiogenesis/stromal modules and spatial indicators, offering superior predictive power compared to single tests. These insights support the adoption of a combined strategy: integrating vascular normalization or stromal/myeloid cell reprogramming on the basis of ICB, and encouraging the use of longitudinal “immune snapshots” to guide intervention and precision immunotherapy for renal cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bd0a59026e4807122514617a7c05a2fad79de65" target='_blank'>
              From chronic inflammation to immune escape: mapping the tumor microenvironment evolution in renal cell carcinoma
              </a>
            </td>
          <td>
            Hanjun Xu, Baojun Tu, Hui Li, Yong Shan
          </td>
          <td>2025-11-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Quantitative image analysis is essential for advancing stem cell biology, developmental studies, and drug discovery, yet most workflows still rely on manual or semi-quantitative scoring that is slow, subjective, and poorly scalable. A major challenge is converting complex colony morphologies into reproducible, high-dimensional datasets. To address this gap, we developed ColonyQuant, an open-source platform that integrates automated colony segmentation, alkaline phosphatase (AP) intensity quantification, morphometric profiling, and statistical classification into a single workflow. ColonyQuant computes per-colony functional readouts alongside comprehensive shape descriptors, capturing both staining intensity and structural features in a unified framework. Applied to embryonic stem cells (ESCs) treated with a selective KDM4 histone-demethylase inhibitor, ColonyQuant revealed dose-dependent reductions in colony area and integrated AP signal, together with systematic remodeling of morphometric metrics. Multivariate analyses robustly stratified treatment groups and identified intensity and solidity as principal features capturing dose-dependent colony responses. By transforming subjective scoring into objective, scalable, and biologically interpretable phenotyping, ColonyQuant provides a reproducible platform for stem cell research and high-content screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57357026b197a6c463b98555927514215f3da5a" target='_blank'>
              Automated Image-Based Profiling of Pluripotent Stem Cell Colonies
              </a>
            </td>
          <td>
            Rui Geng, Benjamin L. Kidder
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4a57be54cf710876a27f9e5a47455e094a8d69" target='_blank'>
              Tumors mimic the niche to inhibit neighboring stem cell differentiation
              </a>
            </td>
          <td>
            Yang Zhang, Yuejia Wang, Jinqiao Song, Lizhong Yan, Ziguang Wang, Dongze Song, Yudi Zhao, Shaowei Zhao
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="

 Glioblastoma (GBM) exhibits marked plasticity and intense microenvironmental crosstalk. We aimed to delineate mesenchymal programs with spatial resolution, clinical relevance, and mechanistic anchors.



 We integrated single-cell RNA-seq, bulk transcriptomes, and Visium spatial data. After rigorous QC and Harmony integration, we annotated 12 cell states using canonical markers, decoupler-based ORA, and AUCell. Tumor boundaries were defined by inferCNV/CopyKAT; developmental potential by CytoTRACE2 and PHATE. Post-translational modification (PTM) axes were scored from curated gene sets. A cell type-aware GNN linked bulk expression to a patient-similarity graph for survival modeling and gene-level hazard attribution. Network convergence combined bulk WGCNA (TCGA/CGGA), single-cell hdWGCNA, BayesPrism deconvolution, and external GEO validation. Ligand–receptor (LR) signaling was inferred with LIANA+, embedded in a signed causal network, and mapped spatially. ARRDC3 expression was assessed in GBM tissues; U251 gain- and loss-of-function assays evaluated proliferation and migration.



 We resolved major GBM states, including two mesenchymal programs (MES1-like, MES2-like). CNV-high regions marked malignant cores, and CytoTRACE2 identified high-potency niches within MES2-like and Proliferation states along non-linear trajectories. PTM landscapes segregated by state; S-nitrosylation, glycosylation, and lactylation were enriched in mesenchymal programs. A GNN risk score stratified overall survival in TCGA (n=157) and generalized to CGGA-325 (n=85) and CGGA-693 (n=140). MES2-like abundance remained an independent adverse predictor (HR = 2.31; 95% CI, 1.04–5.10). MES2-high tumors upregulated EMT, TNFα/NF-κB, JAK/STAT, hypoxia, angiogenesis, and glycolysis; S-nitrosylation associated with increased hazard. Cross-modal convergence defined a conservative MES2 core enriched for ECM remodeling, collagen modification, focal adhesion, and TGF-β regulation. LR analysis prioritized a TAM-to-MES2 axis (e.g., GRN–TNFRSF1A, ADAM9/10/17–ITGB1, TGFB1–ITGB1/EGFR) converging on a CEBPD-centered module. Spatial mapping localized MES2 hotspots within CNV-defined territories and revealed a TNFRSF1A–CEBPD–ARRDC3 focus at an infiltrative rim. ARRDC3 was upregulated in GBM tissues; in U251 cells, knockdown promoted and overexpression suppressed proliferation and migration, indicating context-dependent roles.



 MES2-like GBM is an ECM-driven, stress-adapted state with strong prognostic impact. We nominate CEBPD and TNFRSF1A/ITGB1 as actionable nodes and identify ARRDC3 as a spatially restricted effector with context-dependent tumor-modulatory functions warranting therapeutic exploration.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8942e39c98213dbf5399f61de91b7075a89e9eb2" target='_blank'>
              Single-cell/spatial integration reveals an MES2-like glioblastoma program orchestrated by immune communication and regulatory networks
              </a>
            </td>
          <td>
            Chonghui Zhang, Lu Tan, Kaijian Zheng, Yifan Xu, Junshan Wan, Jinpeng Wu, Chao Wang, Pin Guo, Yugong Feng
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC), a highly heterogeneous breast cancer subtype, poses significant challenges to human health. Intra-tumor heterogeneity (ITH) limits the reliability of conventional prognostic models. Methods Using multi-region RNA-seq, we quantified TNBC transcriptomic heterogeneity through an integrative heterogeneity score (IHS). After evaluating inter-patient heterogeneity (IPH) and ITH, prognostic and low-heterogeneity genes were identified and used to build a prognostic risk model with a random survival forest (RSF) algorithm. This model was combined with TNM staging into a nomogram for clinical applicability. We further revealed the distinct immune microenvironment features, somatic mutations, and chemotherapy responses between risk subgroups. Gene expression was validated via RT-qPCR. Results Spatial characterization uncovered substantial ITH, evidenced by sharp shifts in PAM50 subtypes and immune infiltration. Two low-heterogeneity biomarkers, CYP4B1 and GBP1, were identified to develop a robust prognostic signature with consistent predictive performance across 3- to 9-year survival endpoints (AUC > 0.6). The high-risk subgroup exhibited reduced immune infiltration, reduced immune checkpoint molecule expression, and poor immunotherapy response rates. Integration of the risk signature with TNM staging created a clinically practical nomogram with superior predictive accuracy (C-index >0.67). Therapeutic vulnerability profiling identified six targeted agents showing increased efficacy in high-risk patients. Dysregulation of signature genes was demonstrated in two TNBC cell lines. Conclusions This study established a transcriptomic heterogeneity-resilient prognostic model for TNBC, enabling precise survival stratification and immune microenvironment assessment. The integrative nomogram and risk-guided therapeutic predictions address clinical challenges in TNBC management, advancing personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f58de3cfc14001064a3dc588f6b35f06eee754a7" target='_blank'>
              Breaking the heterogeneity barrier: a robust prognostic signature for survival stratification and immune profiling in triple-negative breast cancer
              </a>
            </td>
          <td>
            Haixing Shen, Qing Zheng, Zhenyu Wang, Daitian Zheng, Zhenqi Gong, Huaiming Wang, Tianmiao Sun, Jie Pan, Yukai Jin, Xiaohong Zheng, Jingzhi Wang, Jiongjiong Zhang
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Brain metastases often originate from circulating tumor cells (CTCs) that become arrested within cerebral microvessels. Recent findings indicate that this process induces localized hypoxia, which in turn upregulates angiogenic mediators such as Angiopoietin-2 (Ang-2), thereby promoting vascular remodeling and facilitating extravasation of tumor cells into the brain parenchyma. Nevertheless, the early transcriptional landscape of the adjacent brain tissue—the so-called pre-metastatic niche—remains poorly defined.



 To simulate early metastatic seeding, syngeneic murine lung and breast carcinoma cells were injected into the right carotid artery of immunocompetent mice. The contralateral, non-injected hemisphere served as an internal control to allow intra-animal comparisons, reduce biological variability, and minimize animal usage. Brain tissue was collected at 1 and 4 days post-injection, processed via formalin fixation and paraffin embedding, and subjected to histological evaluation. Comparative bulk RNA sequencing of tumor-bearing and control hemispheres was performed, complemented by high-plex spatially resolved single-cell transcriptomic profiling to detect localized gene expression patterns.



 Bulk RNA sequencing revealed no significant transcriptional differences between hemispheres. In contrast, spatial transcriptomic analysis uncovered robust, spatially restricted gene expression changes in the tumor-bearing hemisphere. These alterations involved genes associated with vascular remodeling, hypoxia-responsive pathways, and neuronal connectivity. Notably, spatial mapping demonstrated that the expression of several key transcripts peaked near sites of arrested tumor cells and declined with increasing distance, indicative of a highly localized, tumor-induced host response.



 This study reveals distinct, spatially confined transcriptional alterations that define the early pre-metastatic niche in the brain. By leveraging advanced spatial single-cell transcriptomics, we provide a powerful framework to dissect the molecular cues underlying early metastatic interactions within the native tissue microenvironment. These insights not only deepen our understanding of early brain metastasis biology but also inform the development of innovative strategies for therapeutic intervention at the pre-metastatic stage.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6ab71d6adf66b6036a2c3fb3715feb43793308" target='_blank'>
              P18.25.B SPATIALLY CONFINED TRANSCRIPTIONAL RESPONSES CHARACTERIZE THE EARLY PRE-METASTATIC NICHE IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            W. R. Hube, F. Schallerer, L. Bergmayr, R. Piecyk, T. Eska, K. J. Mueller, E. Nicolaishvili, L. Sevenich, K. Unger, P. Jurmeister, L. von Baumgarten, P. Harter
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal tumor. PDAC also shows poor response to many conventional treatments and is refractory to immunotherapy. We hypothesize that these challenges arise from the highly heterogeneous and immunosuppressive tumor microenvironment (TME) in PDAC. Therefore, to comprehend the players behind this immunosuppressive TME, we integrated 10 publicly available datasets into a robust PDAC scRNA atlas of 941.608 cells covering 201 patients (159 PDAC, 24 non-cancer, and 18 metastatic PDAC). The TME showed distinct composition for each tissue, predominating a fibro-inflammatory phenotype with infiltrating Tregs and dysfunctional CD8 T cells in primary PDACs. In contrast, adjacent normal tissue exhibited enrichment of tissue-resident memory CD8 T cells (p < 0.05). Metastatic PDAC was characterized by a reduction in stromal cells combined with myeloid and naive T cells infiltration. Comparison between treated and untreated PDAC revealed the enrichment of cancer-associated fibroblasts (CAFs) in treated patients. Besides, CD8 T cells from untreated patients exhibited higher levels of genes previously linked to immunotherapy response compared to treated patients. Non-negative matrix factorization (NMF) of the malignant cells unraveled 8 distinct metaprograms (MP). MP3 and MP7 overlapped with the well-known Classical subtype, whereas MP5 correlated with the Basal subtype. MP2 exhibited high expression of MHC-II genes and was enriched in metastatic lesions. Next, we investigated the tumor architecture using public spatial transcriptomic datasets. Spot-level deconvolution revealed 12 distinct cellular niches. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications.



 Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, Alessandro Bersch Osvaldt, Simone Marcia Dos Santos Machado, Eduardo Cremonese Filippi-Chiela, Cristina Bonorino. Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B126.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a30afbd64e95a911caaa3a76b2513c14671d0c3e" target='_blank'>
              Abstract B126: Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 KRAS^G12D mutations represent a significant oncogenic driver, prevalent in highly aggressive malignancies such as pancreatic ductal adenocarcinoma (PDAC) and colorectal cancer (CRC). The recent development of direct KRAS^G12D inhibitors has ushered in a new era of targeted therapy, offering unprecedented opportunities to address these previously intractable cancers. This study aimed to explore and optimize therapeutic strategies by thoroughly evaluating the efficacy of MRTX1133, a novel selective non-covalent KRAS^G12D inhibitor, in conjunction with various immunotherapeutic modalities, particularly those involving natural killer (NK) cells.



 We utilized patient-derived xenograft and syngeneic murine cancer models to systematically characterize alterations of immune landscapes within tumor microenvironment (TME) upon KRAS^G12D inhibition. Comprehensive and in-depth assessment of immune cell function and single-cell level transcriptional activity was performed using flow cytometry and single-cell RNA sequencing analysis.



 Our findings reveal that MRTX1133 not only suppresses tumor progression but also profoundly enhances the activation of infiltrated natural killer (NK) cells within the tumor microenvironment. Functional depletion of NK cells partially, yet significantly, attenuated the anti-tumor activity of MRTX1133, underscoring the contribution of NK cells to the observed therapeutic benefit. Notably, KRAS^G12D inhibition led to a reduction in tumor-infiltrating Gr-1^+ myeloid-derived suppressor cells (MDSCs) and increased the proportion of effector NK cells. Furthermore, MRTX1133 facilitated greater intratumoral accumulation of adoptively transferred NK cells and counteracted tumor-induced systemic immunosuppression on NK cells. Combined treatment with MRTX1133 and NK cell immunotherapies produced synergistic anti-tumor effects and significantly extended survival in preclinical models. Mechanistically, ex vivo analyses demonstrated that MRTX1133 upregulates NK cell-activating ligands, including MICA/B and ULBPs, and suppresses the production of cytokines such as IL-6 and GM-CSF by cancer cells, thereby enhancing tumor cell susceptibility to NK cell-mediated cytotoxicity and reducing MDSC proliferation.



 Collectively, our results illuminate the interplay between KRAS^G12D inhibition and immuno-microenvironmental remodeling, highlighting rational combinations of targeted and NK cell-based immunotherapies as promising strategies to potentiate therapeutic responses in KRAS^G12D-driven cancers, potentially improving clinical outcomes for patients.



 Hongyuan Chen, Tuo Hu, Chao Xu; Liangjie Chi, Fangqin Xue, Chunbo He. KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aa888d3ef7c73bc5df1193380be3a3238a9fcd" target='_blank'>
              Abstract A016: KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity
              </a>
            </td>
          <td>
            Hongyuan Chen, Tuo Hu, Chao Xu, Liangjie Chi, Fangqin Xue, Chunbo He
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Ovarian cancer remains the most lethal gynecologic malignancy and a leading cause of cancer-related deaths among women. Adoptive cell therapy (ACT) using tumor-infiltrating lymphocytes (TILs) has shown success in melanoma and correlates with survival in ovarian cancer. However, its application remains limited, partly due to inefficient expansion of functional, tumor-specific T cells.



 We developed a novel ex vivo expansion platform for ovarian TILs, integrating metabolic and phenotypic reprogramming with selective enrichment for tumor-reactive populations. Tissue from nine patients with stage III/IV high-grade epithelial ovarian carcinoma was collected for banking, NGS, and TIL expansion. Four had matched TILs expanded pre- and post-neoadjuvant chemotherapy. The initial priming phase employed our published cytokine cocktail designed to induce Th1/17 and Th17/Tc17 polarization, to promote effector differentiation while preserving stemness. TILs were cultured for five days in this cocktail to enhance metabolic fitness and anti-tumor potential. CD137+ (4-1BB+) T cells were isolated based on activation-induced co-stimulatory expression and subjected to rapid expansion using a G-Rex-based bioreactor system with irradiated feeder cells, anti-CD3, and low-dose IL-2. Expanded TILs were characterized by multicolor flow cytometry, CD107a degranulation, IFN-γ ELISA, and single-cell RNA sequencing (scRNA-seq), and benchmarked against TILs expanded using the conventional protocol from NCI.



 Our approach reliably produced 15–20 billion viable TILs per patient within 30 days, achieving >95% viability and ∼2500-fold expansion. CD137+ enrichment enhanced tumor reactivity, as shown by increased IFN-γ secretion and degranulation with autologous tumor and high-grade serous ovarian cancer cell lines. scRNA-seq revealed enriched cytotoxic CD8+ T cells and mucosal-associated invariant T (MAIT) cells, with reduced FOXP3+ regulatory and non-cytotoxic CD4+ subsets, compared to TILs from matched blood, tumor, and those expanded with high-dose IL-2 in the NCI protocol. CD8+ proliferative and cytotoxic T cell subsets were enriched under the hybrid CD137-sorted protocol. Transcriptomic profiling showed that CD8+ proliferating T cells from the hybrid group downregulated key metabolic genes, including SLC7A5 and LDHB, suggesting a rewired metabolic state that may support persistence. Similarly, CD8+ cytotoxic T cells showed chromatin remodeling, with upregulation of histone demethylases KDM5A and KDM7A, and downregulation of CHD7 and JARID2, indicating transcriptional plasticity. MAIT cells also showed increased activation markers (IFNG, CCL20). Expanded TILs exhibited reduced expression of exhaustion markers PD-1 and CD38 at the protein level.



 Metabolic reprogramming combined with CD137-based enrichment enables robust expansion of metabolically reprogrammed, tumor-specific TILs from ovarian tumors. This strategy offers a promising foundation for advancing TIL-based ACT in ovarian cancer.



 Paramita Chakraborty, Stephanie Mills, Jadan Zavestoski, Satyajit Das, Suganya Subramanian, Nicolina Mascia, Jian Liu, Stefano Berto, Brian Orr, Shikhar Mehrotra, Stephanie Mills. Metabolic reprogramming of ovarian CD137+ TILs for treatment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b37662ad71037e3dead97ad855c982dd8d046b2" target='_blank'>
              Abstract PR006: Metabolic reprogramming of ovarian CD137+ TILs for treatment
              </a>
            </td>
          <td>
            Paramita Chakraborty, Stephanie Mills, Jadan Zavestoski, Satyajit Das, S. Subramanian, Nicolina Mascia, Jian Liu, S. Berto, Brian Orr, Shikhar Mehrotra
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Single-cell molecular profiling has emerged as a cutting-edge technology in biological research, playing a pivotal role in elucidating fundamental life processes and cellular heterogeneity. However, traditional methods generally provide end point measurements and require cell disruption, posing challenges for long-term monitoring of living cell events, e.g., cancer therapy. Tumor Treating Fields (TTFields), a noninvasive therapeutic modality, exert inhibitory effects on tumor growth and invasion by applying low-intensity intermediate-frequency alternating electric fields to tumor regions, whose molecular mechanism on cancer physiology remains poorly understood. In this study, we propose a multifunctional microchip capable of continuous in situ monitoring of molecular signatures in living single cells. The platform integrates spatial cell positioning, single-cellular multimolecule tracking with surface-enhanced Raman scattering (SERS), and TTFields treatment functionalities on a microchip. Using multiplexed analysis of liver cancer cells treated with TTFields and chemotherapy drugs, we implement spatial-temporal multiomics profiling for cancer therapy. Temporal tracking of critical molecular events associated with glycolysis, genetic stability, and membrane integrity reveals significant differences between the groups with and without TTFields treatment. Causal network analysis of molecular profiles uncovers the underlying mechanisms through which TTFields modulate oncogenic pathways. Our work demonstrates the effectiveness of the proposed microchip platform for real-time monitoring of living cell molecular dynamics, offering a powerful tool for precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889ada64ff2d00892ebd5cde7eb385bb8b31694b" target='_blank'>
              Spatial-Temporal Multiomics Profiling of Living Single Cells on Chip for Cancer Therapy.
              </a>
            </td>
          <td>
            Kunru Yu, Sheng-Zong Chen, Rong Zhu
          </td>
          <td>2025-10-12</td>
          <td>ACS sensors</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains one of the most difficult solid tumors to treat, owing in part to its remarkable cell state plasticity - a property that enables tumor cells to resist therapy, evade differentiation, and adapt to selective pressures. While targeted therapies have succeeded in subsets of patients, most tumors persist and eventually progress. To pursue a fundamentally new therapeutic strategy, we conducted an unbiased chemical screen using a custom-designed dual reporter system designed to broadcast cell state dynamics in real time by fluorescently tagging the endogenous SOX9 (aberrant stem-like) and KRT20 (differentiated) loci. This screen targeted epigenetic regulators capable of supporting CRC plasticity and preventing differentiation. We discovered that selective inhibition of histone deacetylases HDAC1/2 promotes differentiation and reduces tumor growth across human and mouse CRC models. This effect was not a consequence of cytotoxicity but rather epigenetic reprogramming. Through mass spectrometry-based histone modification profiling, we identified H3K27ac and H3K9ac as key epigenetic marks enriched at HDAC1/2-bound loci. These marks correlated with enhanced chromatin accessibility and the upregulation of differentiation-associated genes. Degradation of EP300, a major acetyltransferase, reversed the phenotype, confirming the functional relevance of histone acetylation in mediating this differentiation response. Furthermore, a focused CRISPR screen uncovered DAPK3 as a previously unrecognized downstream effector, providing a mechanistic foothold for future biomarker development. Our findings introduce a paradigm shift in CRC therapy: rather than targeting plastic tumor cells for elimination, we aim to force them into a differentiated, less adaptive state, making them more vulnerable to conventional and targeted treatments. This reprogramming effect is trackable via specific histone acetylation patterns, which we propose as candidate predictive biomarkers of response. Based on these insights, we hypothesize that HDAC1/2 inhibitors, when combined with KRAS or EZH2 inhibitors and/or chemotherapy, could yield synergistic therapeutic benefit. We are actively evaluating these combinations in preclinical models. In sum, this work positions HDAC1/2 inhibition as a precision epigenetic strategy to suppress CRC plasticity through enforced differentiation. By integrating multiomic profiling, functional genomics, and novel reporter technologies, we offer a blueprint for targeting the epigenetic underpinnings of therapy resistance in gastrointestinal malignancies, one that prioritizes reprogramming over destruction.



 Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, Sandor Spisak, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James Cleary, Jun Qi, Nilay Sethi. Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1992691ab8c82d8cb17a698db10f779b389e986a" target='_blank'>
              Abstract C031: Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains largely refractory to therapy, due in part to the complex interplay between tumor cells and their microenvironment. Human antigen R (HuR/ELAVL1), a ubiquitously expressed RNA-binding protein, is emerging as an important regulator both of tumor-intrinsic and tumor-extrinsic pathways that govern PDAC progression. While the role of HuR in promoting cancer cell survival under stress is well established, recent studies reveal its broader role in shaping the tumor microenvironment (TME), including metabolic rewiring, stromal activation, angiogenesis, and immune modulation. In this review, we examine how tumor-intrinsic HuR drives epithelial-mesenchymal transition, stabilizes key transcripts involved in metabolic adaptation, and alters the tumor secretome to influence extracellular matrix deposition and fibroblast behavior. We further explore the role of HuR in regulating immune cell function and the immune landscape of PDAC. Notably, HuR-driven TME remodeling reinforces environmental stressors that further activate HuR, establishing a feed-forward loop that drives disease progression. These findings underscore HuR as a central regulator of the PDAC TME and therapeutic resistance, and thus, highlight its potential as a target in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1b261ccac3070698258b8350c58e4ccb52e4c4" target='_blank'>
              The Role of Tumor-Intrinsic HuR in Modeling the Pancreatic Tumor Microenvironment: Molecular Mechanisms and Therapeutic Opportunities.
              </a>
            </td>
          <td>
            Katherine R. Pelz, Kyle P Gribbin, Tess Sevetson, Jonathan R. Brody
          </td>
          <td>2025-10-13</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Renal cancer, particularly clear cell renal cell carcinoma (ccRCC), is characterized by significant intratumoral heterogeneity, which poses challenges for diagnosis and treatment. Single-cell sequencing (SCS) provides unprecedented insights into the cellular landscape of renal cancer, allowing for detailed characterization of tumor heterogeneity at the single-cell level. This review highlights how SCS has been instrumental in elucidating the origins of different renal cancer subtypes, understanding mechanisms of tumor initiation and progression, and dissecting the complex tumor microenvironment (TME). It discusses the identification of novel biomarkers and therapeutic targets, as well as the potential of SCS to inform personalized treatment strategies. The review also explores the integration of SCS with spatial omics technologies, which enhances the understanding of cellular interactions within their spatial context. Moreover, it addresses the challenges and future directions in applying SCS to clinical practice, emphasizing its significance in advancing renal cancer biology and improving clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03a5f0a7b414a81333643cfcf0b78e2ab4708d98" target='_blank'>
              Single-cell sequencing technology in renal cancer: insights into tumor biology and clinical application
              </a>
            </td>
          <td>
            Hanzhong Zhang, Ying Liu, Wenqiang Liu, Anqi Lin, Yu Fang, Le Qu, Xu Zhang, Peng Luo, Linhui Wang, Aimin Jiang
          </td>
          <td>2025-10-01</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86ecdda13a72d7b290cf8fe9dd99c8ce634923c0" target='_blank'>
              Galectin-9-driven immunosuppressive macrophage population shapes the lymph node metastatic microenvironment.
              </a>
            </td>
          <td>
            Yao Zhou, Xiaowei Liu, Guixiu Xiao, Genpeng Li, Xiujing He, Leyi Tang, Xueyan Wang, Yujia Lin, Jianyong Lei, Jing Jing, Hubing Shi
          </td>
          <td>2025-10-25</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Background/Objectives: Galectin-3 (Gal-3), encoded by LGALS3, is a β-galactoside-binding lectin involved in diverse tumor-associated processes, including immune modulation, cell cycle regulation, and stress adaptation. Despite its known roles in cancer biology, the full extent of its molecular functions and prognostic relevance across tumor types remains incompletely understood. This study aimed to systematically investigate the transcriptomic impact of LGALS3 deletion and assess its clinical significance in cancer. Methods: We analyzed CRISPR-Cas9 knockout transcriptomic data from the SigCom LINCS database to characterize the consensus gene signature associated with LGALS3 loss using functional enrichment analyses. Pan-cancer survival analyses were conducted using TIMER2.0. Differential Gal-3 protein levels in ductal adenocarcinoma and normal pancreatic tissues were evaluated using the Human Protein Atlas. Finally, functional analyses were performed in pancreatic ductal adenocarcinoma (PDAC). Results: LGALS3 deletion across multiple cancer cell lines led to transcriptomic changes involving mitotic progression, stress responses, and axonal guidance pathways. High LGALS3 expression was significantly associated with worse overall survival in lower-grade glioma, PDAC, uveal melanoma, and kidney renal papillary cell carcinoma. LGALS3 knockout in YAPC cells recapitulated the pan-cancer findings, linking LGALS3 to cell morphogenesis and proliferation. Conclusions: These findings identify Galectin-3 as a key regulator of oncogenic programs and a potential prognostic biomarker in PDAC and other malignancies, with implications for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0894155b476f9fae4b8588df639cdd5d50fcb2b" target='_blank'>
              Transcriptomic and Clinical Profiling Reveals LGALS3 as a Prognostic Oncogene in Pancreatic Cancer
              </a>
            </td>
          <td>
            Grazia Scuderi, S. Mijatović, D. Maksimović‐Ivanić, M. Di Rosa, J. Muñóz-Valle, Alexis Missael Vizcaíno-Quirarte, Gian Marco Leone, K. Mangano, P. Fagone, Ferdinando Nicoletti
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65098146e294663c9f9044da832bd80f4169e937" target='_blank'>
              A Human Neuronal Co-Culture System Reveals Early Tumor-Neuron Communication and Targetable Pathways in Glioblastoma
              </a>
            </td>
          <td>
            Ouada Nebie, Niyi Adelakun, Liwen Zhang, Luke Kollin, Brian Fries, Akhil Medikonda, Monica Venere, Pierre Giglio, Nam Chu, Nhat T Le
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1fe8757b9db03b1f80d6119b9cc7fd8b56ac919" target='_blank'>
              Pan-cancer analysis reveals that tumor microbiomes generate specific epitopes through transcriptional reprogramming
              </a>
            </td>
          <td>
            Jinman Fang, Yuxin Zhang, Wei Wang, Xiao Zuo, Tao Chen, Kecheng Li, Huiya Yang, Tianyang Zhu, Yuanliang Zhang, Dandan Wei, Jianfeng Zhang, Jialong Cui, Zhen Tang, Shouwu Zhang, Lirong Wang, Zhilei Li, Dushan Ding, Ruixia Sun, Dandan Zhu, Yin He, Yanfei Zhou, Chenkun Ye, Dandan Chen, Xueting Lang, Long Jiang, Qian Zhao, Lu Zhang, Hongzhi Wang, Yuanwei Zhang
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Understanding the functional impact of bacterial genetic diversity is crucial for linking pathogen variants to clinical outcomes. Here, we introduce a high-throughput cytological profiling pipeline optimized for Mycobacterium tuberculosis (Mtb) clinical strains, integrating OD-calibrated feature analysis and high-content microscopy. Our system quantifies single-bacterium morphological and physiological traits related to DNA replication, redox state, carbon metabolism, and cell envelope dynamics. Applied to 64 Mtb clinical isolates from lineages 1, 2, and 4, the approach revealed that cytological phenotypes recapitulate genetic relationships and exhibit both lineage- and density-dependent dynamics. Notably, we identified a link between a convergent “small cell” phenotype and a convergent ino1 mutation that is associated with the presence of an antisense transcript, suggesting a potential non-canonical regulatory mechanism under selection. In summary, we present a resource-efficient approach for mapping Mtb’s phenotypic landscape, uncovering cellular traits that underlie its evolution and providing new insights into the functional consequences of bacterial genetic diversity. IMPORTANCE Understanding how genetic variation in Mycobacterium tuberculosis (Mtb) shapes its physical traits is essential to unraveling the evolution of this global pathogen. Here, we introduce a systematically optimized, high-throughput imaging platform for the comprehensive characterization of Mtb clinical strains. We demonstrate that Mtb’s phenotypic manifestation is shaped by both genetic background and culture density. By accounting for these factors, our analysis linked distinct cellular dynamics to specific lineages, sublineages, and even single nucleotide variations. Notably, we linked a recurring mutation to a unique cell-shortening phenotype, finding that it potentially acts by creating a cryptic antisense transcript. This platform provides a powerful framework for systematically dissecting the physiological dynamics underlying Mtb evolution and identifying new therapeutic vulnerabilities of this deadly pathogen. Understanding how genetic variation in Mycobacterium tuberculosis (Mtb) shapes its physical traits is essential to unraveling the evolution of this global pathogen. Here, we introduce a systematically optimized, high-throughput imaging platform for the comprehensive characterization of Mtb clinical strains. We demonstrate that Mtb’s phenotypic manifestation is shaped by both genetic background and culture density. By accounting for these factors, our analysis linked distinct cellular dynamics to specific lineages, sublineages, and even single nucleotide variations. Notably, we linked a recurring mutation to a unique cell-shortening phenotype, finding that it potentially acts by creating a cryptic antisense transcript. This platform provides a powerful framework for systematically dissecting the physiological dynamics underlying Mtb evolution and identifying new therapeutic vulnerabilities of this deadly pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56fa61f881d5174f374abc12d00931b99537639c" target='_blank'>
              High-throughput cytological profiling uncovers genotype-phenotype associations in Mycobacterium tuberculosis clinical isolates
              </a>
            </td>
          <td>
            Qingyun Liu, Yue J. Liu, Ruiyuan Liu, Peter H. Culviner, Xin Wang, I. D. Wolf, Ken Chen, Yiwang Chen, Yi Xiao, Guiming Zhang, Rongfeng Sun, Shoko Wakabayashi, Nicole C Howard, Mingyu Gan, Eric J. Rubin, Sarah M. Fortune, Junhao Zhu
          </td>
          <td>2025-10-09</td>
          <td>mSystems</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 TCR-engineered T cell (TCR-T) therapy holds significant promise in cancer treatment, allowing precise targeting of intracellular antigens. However, its application in neuro-oncology, particularly in glioblastoma (GB), faces unique challenges. In addition to the restrictive nature of the blood-brain barrier and a profoundly immunosuppressive tumor microenvironment, the comparatively low mutational burden of GB limits the availability of immunogenic targets, collectively hampering the success of immunotherapy in primary brain tumors. Despite these challenges, recent vaccine studies have demonstrated that both tumor-specific neoantigens and non-mutated, overexpressed glioma-associated antigens (GAAs) can induce robust and durable T cell responses in patients with GB. In tumors exhibiting high clonal heterogeneity and a limited number of shared mutations, the ability to exploit a broader antigenic repertoire may be critical for therapeutic success. To broaden the clinical applicability of TCR-T cell therapy for GB, there is an urgent need to develop TCRs targeting diverse GAAs across multiple HLA backgrounds.



 To this end, we aimed to identify and functionally validate GAA-specific TCRs from vaccinated GB patients. PBMCs from a single patient enrolled in the GAPVAC-101 vaccine trial were expanded in vitro using peptide pools spanning immunogenic GAA epitopes. Following peptide-specific expansion, GAA-reactive T cells were enriched by FACS and subjected to single-cell V(D)J sequencing to retrieve paired TCR α/β chain information. Dominant clonotypes were selected based on frequency, and corresponding TCRs were cloned into a lentiviral backbone via Golden Gate Assembly. The cloned TCRs were expressed in CD4⁺/CD8⁺ Jurkat reporter cells for functional evaluation. Following TCR expression, Jurkat cells were co-cultured overnight with patient-derived B-lymphoblastoid cell lines (B-LCLs) to assess TCR reactivity. Reactivity was evaluated the next day by flow cytometry through staining for CD69 surface-expression and analysis of NFAT-GFP reporter activity.



 Analysis of the TCR repertoire from peptide-expanded, FACS-sorted CD4⁺ and CD8⁺ T cells revealed an oligoclonal pattern, consistent with antigen-driven expansion. Utilizing material from the GAPVAC-101 trial, we identified both CD4⁺ and CD8⁺ TCRs reactive to GAAs. Selected TCRs were successfully cloned and lentivirally expressed in Jurkat reporter cells, where they exhibited antigen-specific activation in vitro, confirming functionality.



 These findings demonstrate the feasibility of isolating functional, GAA-specific TCRs from vaccinated GB patients. While further validation in vitro and in vivo is necessary, this approach provides a foundation for expanding the repertoire of TCRs available for personalized TCR-T cell therapy in GB.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c62c351184571592a0c8f085fde0a7f6110ba1f" target='_blank'>
              P08.13.A IDENTIFICATION OF VACCINE-INDUCED TCRS AGAINST GLIOBLASTOMA-ASSOCIATED ANTIGENS
              </a>
            </td>
          <td>
            P. Koopmann, Y. Chih, A. Dietsch, K. Lindner, S. Stange, I. Poschke, T. Boschert, E. Green, L. Bunse, W. Wick, M. Platten
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND
Changes in the cyst microenvironment in Polycystic Kidney Disease (PKD) may drive progressive cyst formation. Bulk- and single-cell RNA sequencing have advanced our understanding of altered signaling; however, the lack of spatial information has limited our insights into local gene expression and cellular communication near cysts.


METHODS
We used wild-type and Pkd1-deficient mouse kidneys to generate 10x Genomics Visium Spatial Gene Expression datasets. Utilizing our single-cell mouse kidney atlas and single cell sequencing data for spot deconvolution, we enhanced resolution and estimated enriched cell types. We analyzed spatial gene expression patterns and used a cyst-centered analysis to identify cyst-associated gene signature. Cell communication near cysts was investigated, identifying key ligand-receptors. Prioritized key factors were validated in tissues.


RESULTS
We observed enrichment of fibroblasts, injury-repair-related cell types, and diverse immune populations in PKD. Injury-repair-related cells were exclusively observed in PKD, predominantly localized within immune cell-dense regions near cysts. These cells collectively contributed to the altered gene expression profile in PKD, including cyst-associated signature genes related to inflammatory processes. Analysis of cellular communication in less-inflamed regions around cysts revealed the involvement of multiple cell types. Key ligand-receptor interactions were associated with cytokine signaling, fibrosis, cellular development, and repair. These included Angpt2, C3, Csf1, Cxcl12, Il34, Gas6, Il16, Mdk, Mif, Ptn, Sfrp2, Spp1, Sdc1, Tnc, Tnfsf12, Wnt5a. In addition, ECM proteins implicated in immune response, ECM remodeling, cell adhesion, and cell signaling were identified, such as Adam9, Adam10, Col1a1, Col3a1, Col4a2, Lamb2, Lamc1, Efnb1, Efnb2, Thbs1, Thbs2, and Vcam1. IHC confirmed expression of Syndecan-1-Collagen IV, Midkine-Integrin β1, CSF-1, Pleiotrophin, and Tenascin-C in cystic kidneys.


CONCLUSIONS
Spatial transcriptomics in PKD revealed enrichment of (myo)fibroblasts, immune, and injury-repair-related cells near cysts, creating a (pro)inflammatory and (pro)fibrotic niche. Key ligand-receptor and ECM interactions were identified and validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecdcf5e98138976e4b0b6f2eafc40e1b84148be0" target='_blank'>
              Spatial Transcriptomics Reveals Injured Cells, Signature Genes, and Communication Patterns in the Cyst Microenvironment of Polycystic Kidney Disease.
              </a>
            </td>
          <td>
            Sevtap A Yasinoglu, Claudio Novella-Rausell, L. Wisse, K. Dijkstra, A. Mahfouz, Hans J. Baelde, DJM. Peters
          </td>
          <td>2025-10-10</td>
          <td>Journal of the American Society of Nephrology : JASN</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="

 FAM49B has been shown to promote proliferation and metastasis of colorectal cancer (CRC) by stabilizing MYC through phosphorylation of NEK9; however, its role in shaping the immune suppressive tumor microenvironment (TME), particularly in macrophage polarization, remains unclear.



 We applied multi-omics approaches to study CRC by integrating 33 scRNA-seq samples from 16 CRC patients, 2 paired spatial transcriptomics (ST) samples, and bulk RNA data to characterize malignant epithelial cells (High_FAM49B_EP) and tumor-associated macrophages (TAMs). Functional validation of FAM49B was conducted via knockdown experiments and proteomics analysis.




 A High_FAM49B_EP subpopulation was identified in primary tumors (PT) and liver metastases (LM), exhibiting elevated MYC signaling and association with poor prognosis. TAMs showed spatial heterogeneity: M1-like CXCL3
 +
 TAMs predominated in PT, whereas M2-like SPP1
 +
 TAMs were enriched in LM. CellChat analysis revealed that High_FAM49B_EP activated macrophage polarization through the MDK–NCL signaling axis. Pseudotime trajectory analysis confirmed differentiation from CXCL3
 +
 to SPP1
 +
 TAMs driven by upregulation of NCL. Spatial mapping showed co-localization of MDK
 +
 epithelial cells with NCL
 +
 TAMs in the immunosuppressive microenvironment. FAM49B knockdown significantly inhibited MDK expression and disrupted ECM–receptor interactions.




 FAM49B promotes immunosuppressive TME formation by mediating TAM polarization via the MDK–NCL axis, suggesting the FAM49B–MDK–NCL pathway as a potential therapeutic target for CRC metastasis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8878570ce8b97d296e4580e17e3836990c936b1" target='_blank'>
              Single-cell and spatial transcriptomics integration reveals FAM49B promotes tumor-associated macrophages polarization in colorectal cancer via the MK pathway
              </a>
            </td>
          <td>
            Tianyu Liu, Quchen Ding, Jin Gou, Chen Lu, Xingming Lu, Jiatong Chen, Y. E, Lianhong Li, Chongguo Zhang, Xiaojuan Zhu, Chunzhao Yu, Xiagang Luo
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4ae2d5f1f6f184ad5a2b71dcc7e35b4feeb422f" target='_blank'>
              Functional biomaterials and machine learning approaches for phenotyping heterogeneous tumor cells and extracellular vesicles.
              </a>
            </td>
          <td>
            Rutwik Joshi, R. Ahmad, Karl Gardner, Hesaneh Ahmadi, Chau-Chyun Chen, Shannon Stott, Wei Li
          </td>
          <td>2025-09-18</td>
          <td>Biomaterials science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            S. Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Neoadjuvant treatment has demonstrated clinical benefits in advanced gallbladder cancer (GBC). However, the overall response rate remains suboptimal, and the underlying mechanisms driving treatment efficacy are not fully understood.


MATERIALS AND METHODS
This study aimed to evaluate the therapeutic effects of neoadjuvant chemo-immunotherapy (NAT) combining gemcitabine, nab-paclitaxel, and anti-PD-1 immunotherapy in advanced GBC, and to investigate the associated tumor microenvironment (TME) alterations. Single-cell RNA sequencing and multiplex immunohistochemistry were utilized to analyze the cellular composition of the TME in patients who successfully underwent downstaging.


RESULTS
NAT significantly reshaped the tumor-immune landscape, characterized by an expansion of follicular helper T (Tfh) cells and the formation of tertiary lymphoid structures (TLSs) in 50% of treated tumors. Inflammatory cancer-associated fibroblasts (CAFs) increased and exhibited upregulation of CCL19 and CXCL12, potentially promoting Tfh cell recruitment and TLS formation. Additionally, NAT led to an expansion of GZMB+ cytotoxic CD8+ T cells with an exhausted phenotype in GBC, but not in adjacent normal tissues. The treatment also increased the number and effector functions of natural killer cells while reducing tumor-promoting macrophages and angiogenesis-related CAFs. Furthermore, NAT decreased the cancer stem cell-like subpopulation while increasing a cancer cell subset with enhanced antigen-presenting capacity.


CONCLUSION
This study suggested the potential efficacy of NAT in advanced GBC and revealed alterations in the TME following treatment. The findings provided insights into the mechanisms underlying NAT responses and offered valuable directions for optimizing therapeutic strategies in GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363d50ed22da82003f2b54ca7ab634be84e764af" target='_blank'>
              Single-cell profiling unveils the immuno-favorable tumor microenvironment remodeling after successful neoadjuvant therapy for advanced gallbladder cancer.
              </a>
            </td>
          <td>
            Qiuyi Tang, Yi-Jian Zhang, , Tian-Yu Zhang, Xiao-ling Song, Yi-di Zhu, , Cheng Zhao, , Hua-kai Wang, Yue Yang, Yu Zhang, , Lin Jiang, Yang-Yang Zhai, , , Wei Gong
          </td>
          <td>2025-11-06</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Approximately 40% of patients with pancreatic ductal adenocarcinoma (PDAC) receive no benefit from surgical resection and systemic chemotherapy and experience metastatic disease recurrence within 12 months. The extreme heterogeneity of the cells that populate PDAC tumors has been a significant impediment to identifying the cells that likely seed early recurrence. Previous studies show that metastatic spread is driven by specific cells with hybrid epithelial-mesenchymal characteristics, but despite the identification of numerous molecular features associated with epithelial, hybrid, and mesenchymal PDAC cancer cells, it has not been possible to specifically identify the cells that seed metastatic recurrence in human tissue. We have previously shown that PDAC cell glycan signatures are altered as a direct result of the metabolic and transcription changes that occur as PDAC cancer cells undergo EMT. Now we integrated the glycan signatures with known protein markers and morphological analyses using our pipeline for multiplexed immunofluorescence analysis to identify distinct subpopulations of epithelial, hybrid, and mesenchymal PDAC cells. In cell-level analyses of immunofluorescence images of 24 PDAC cell lines, these subpopulations revealed epithelial to hybrid to mesenchymal conversions along 3 distinct trajectories, which were defined by transcription factors and proteins previously associated with epithelial or mesenchymal states—TP53, TP63, GATA6, and MYC—and further distinguished by specific glycan signatures and protein markers of metabolism and cellular differentiation. The integrated transcription factor, glycan, and protein signatures identified the previously described classical and squamous PDAC subpopulations and defined a new signature for a fully mesenchymal subpopulation. The signatures differentiated between PDAC subpopulations in primary human tumors and in metastatic liver lesions and showed a patient-matched correspondence between the primary tumors and metastatic lesions in specific hybrid PDAC cells within distinct trajectories, potentially revealing the PDAC cells seeding metastatic recurrence. These results suggest that the development of mesenchymal and metastatic PDAC cells occurs through distinct, stepwise trajectories with divergent morphological, metabolic, molecular, and glycan-based signatures. These findings will/have the potential to inform the development of treatments that specifically target the PDAC cells that seed early metastatic recurrence.



 ChongFeng Gao, Zachary Klamer, Ali Moursy, Galen Hostetter, Paul M. Grandgenett, Peter Allen, Brian Haab. Resolving PDAC cell subpopulations that seed metastatic recurrence [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B125.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246925305285b4b397bb54f4914db32c1466cbe8" target='_blank'>
              Abstract B125: Resolving PDAC cell subpopulations that seed metastatic recurrence
              </a>
            </td>
          <td>
            Chongfeng Gao, Zachary L. Klamer, Ali Moursy, G. Hostetter, P. Grandgenett, Peter Allen, Brian Haab
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy driven by oncogenic KRAS and inflammation-induced cellular heterogeneity, yet the mechanisms underlying tumor-initiating cells (TICs) emergence and maintenance remain unclear. Our study establishes LY6D as a marker of persistent TICs that orchestrate pancreatic cancer progression across all disease stages. Single-cell RNA sequencing of inflammation-driven PDAC models reveals that LY6D+ TICs specifically originate from KRAS-mutant acinar-to-ductal metaplasia (ADM) lesions under inflammatory conditions, maintaining conserved stemness properties and exhibiting a distinctive oxidative phosphorylation (OXPHOS) dependency throughout tumor evolution. Moreover, genetic ablation of Ly6d in Kras G12D pancreata delays tumorigenesis, while forced Ly6d expression enhances tumorigenic potential and metastatic capability. Mechanistically, LY6D-despite lacking intracellular domains-scaffolds lipid raft-associated kinase networks and FOSL1-dependent epigenetic reprogramming to establish a ​stable pro-tumorigenic state. Clinically, LY6D+ cells are enriched in human PDAC and exhibit conserved stemness and epithelial-mesenchymal transition (EMT) properties. Strikingly, LY6D expression levels demonstrate PDAC-restricted prognostic power. Our work defines LY6D as pan-stage TICs marker linking cellular plasticity to PDAC initiation and progression, offering new avenues for early detection and interception of this lethal malignancy.



 Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue. LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B119.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f74adf609c9f9d814e1f1f768c348943a8c10b" target='_blank'>
              Abstract B119: LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Analyses of patient-derived cell lines have greatly enhanced discovery of molecular biomarkers and therapeutic targets. However, characterization of cellular morphological properties is limited. We studied cell morphologies of human pancreatic adenocarcinoma (PDAC) cell lines and their associations with drug sensitivity, gene expression, and functional properties. By integrating live cell and spatial messenger RNA imaging, we identified KRAS inhibitor–induced morphological changes specific for drug-resistant cells that correlated with gene expression changes. We then categorized a large panel of patient-derived PDAC cell lines into morphological and organizational subtypes and found differences in gene expression, therapeutic targeting potential, and metastatic proclivity. Patterns of cancer cell organization in human PDAC tissues stratified distinct gene expression signatures with clinical significance. In summary, we highlight the potential of cell morphological information in rapid, cost-effective assays to aid precision oncology efforts leveraging patient-derived in vitro models and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb047a3ab41c4e28c6535b371221906a6c927ab" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor matrix stiffness plays a critical role in cancer progression, metastasis, and therapy resistance. Although traditional biophysical methods have shed light on the impact of matrix stiffness on tumor behavior, these techniques are confined to measuring the physical properties of the tumors. In this study, we leveraged RNA-seq data to predict tumor matrix stiffness, aiming to reveal mechanical properties by molecular signatures across various cancer types. To this end, we systematically analyzed RNA-seq data from tumors of varying stiffness levels to identify stiffness-associated gene signatures. With these molecular signatures, we developed a computational model for predicting tumor matrix stiffness and further applied it to The Cancer Genome Atlas (TCGA) dataset. Our analysis revealed significant differences in the tumor microenvironment as well as immune response between soft and stiff tumor samples, suggesting that tumor rigidity impacts not only cellular behavior but also characteristics of the tumor microenvironment. These findings underscore the potential of RNA-based stiffness models to enhance our comprehension of tumor mechanics and cancer biology, thereby facilitating the development of innovative targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd0a56130a8bfa1d6781b487c86488f3edefb603" target='_blank'>
              Pan-cancer analysis reveals molecular signatures for predicting matrix stiffness in solid tumors.
              </a>
            </td>
          <td>
            Gongyu Tang, Xinyi Liu, Yuanxiang Li, Yunfei Ta, Minsu Cho, Hua Li, Xiaowei Wang
          </td>
          <td>2025-09-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The clonal evolution model provides a framework for understanding the evolution of cancer cells. According to this model, cancer cells accumulate genetic mutations over time, and these mutations are passed down to their descendants, leading to genetic diversity within the tumor. Some of these mutations confer selective advantages, causing certain lineages of cancer cells (clones) to dominate and expand. However, this model is rooted in certain conceptual assumptions, which we propose to revisit by considering the potential involvement of retrotransposons in cancer initiation and progression. In recent years, it has become evident that transposable elements, particularly retrotransposons, play a significant role in driving cancer transformation and progression. We first review how current knowledge about retrotransposon activity aligns with the clonal evolution model by highlighting its ability to modulate cancer cell fitness. We then take a forward-looking perspective to explore additional ways retrotransposons may also influence clonal dynamics beyond the current model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb2629152d1c164e115163464d0e7599e1333a7" target='_blank'>
              Revisiting Clonal Evolution Through the Light of Retrotransposons.
              </a>
            </td>
          <td>
            Anaïs Lamoureux, Emilie Elvira-Matelot, Françoise Porteu, L. Laplane
          </td>
          <td>2025-10-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/841868dae6381828134e074133437d8b76fe3212" target='_blank'>
              Bleb-based extravasation: conserved morphodynamics, divergent calcium control
              </a>
            </td>
          <td>
            Mizuki Morita, Manami Morimoto, Junichi Ikenouchi, Bertrand Pain, Yuji Atsuta, Yoshiki Hayashi, Takayuki Teramoto, Daisuke Saito
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is characterized by aggressive biological behavior and poor prognosis. However, TNBC exhibits higher immunogenicity than other breast cancer subtypes, making it more responsive to immunotherapy. Neoadjuvant chemotherapy (NAC) is the standard treatment for early high-risk TNBC; however, reliable biomarkers for predicting NAC response remain elusive. Tumor-infiltrating lymphocytes (TIL) are recognized as predictive markers of NAC response in TNBC, yet discordant cases remain, such as tumors with high TIL levels but poor response. This study aimed to further elucidate the immune environment of TNBC by analyzing TIL, programmed death-ligand 1 (PD-L1) expression, and tumor-stroma ratio using pretreatment biopsy tissue slides from 16 patients with TNBC treated with NAC. Multiplexed immunofluorescence for CD8, FOXP3, CD4, CD20, and CK was employed to investigate immune cell (IC) composition and spatial interactions within the tumor immune microenvironment. Cell to cell distance and comparison of subcellular proportion of IC and were analyzed by treatment response, TIL, and PD-L1 status. Significant differences were found in IC composition and distribution between patients achieving pathologic complete response (pCR) and those with residual disease (non-pCR). The pCR group exhibited significant enrichment of cluster of differentiation CD8 + IC and CD20 + IC in both tumor and stromal regions, suggesting their critical role in mediating an effective NAC response. In contrast, non-pCR cases showed higher proportions of immunosuppressive CD4 + FOXP3 + IC, particularly in the tumor region. High TIL levels were associated with pronounced B-T cell interactions, as evidenced by the significant clustering of CD20 + and CD8 + ICs near tumor cells, highlighting their cooperative role in antitumor immunity. In conclusion, our findings suggest that tumoral CD8 + and CD20 + ICs are pivotal determinants of NAC response in TNBC. The enrichment of CD20 + IC under high-TIL conditions underscores the potential role of B-T cell interactions in shaping immune-mediated chemotherapy responses. These insights provide a foundation for leveraging immune-based biomarkers to stratify patients with TNBC and optimize NAC outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efea310e4b9cebac8f99562fb5c52b2d67c2f334" target='_blank'>
              Spatial analysis of tumor immune microenvironment of TNBC with different neoadjuvant chemotherapy outcomes using multiplex Immunofluorescence
              </a>
            </td>
          <td>
            Inho Park, Su-Jin Shin, H. Go, Jiwon Ko, S. Bae, S. Ahn, Joon Jeong, Jee Hung Kim, Yoon Jin Cha
          </td>
          <td>2025-10-21</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6845d1414654b932a477f7dda03c048913d97883" target='_blank'>
              Cell populations in human breast cancers are molecularly and biologically distinct with age.
              </a>
            </td>
          <td>
            Adrienne M Parsons, Esther Sauras Colón, Meghana Manjunath, Hanyun Zhang, Julia Chen, M. Spasic, Beyza Koca, Busem Binboga Kurt, Rachel Freedman, E. Mittendorf, Alexander Swarbrick, Peter van Galen, Sandra S. McAllister
          </td>
          <td>2025-11-04</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Iron metabolism is essential for tumor proliferation and survival, yet its dysregulation can drive mutagenesis and tumor evolution. Glioblastoma (GBM) stem-like cells tolerate high reactive oxygen species (ROS) levels and exploit iron-rich environments to support unchecked proliferation. We hypothesize that iron influx through the leaky GBM vasculature creates a ROS- and iron-rich tumor microenvironment (TME), fostering a mutagenic niche that promotes a mesenchymal transition and enhances tumor invasion.



 We conducted single-nucleus RNA sequencing on in-house GBM samples to assess the interplay between iron metabolism and invasion programs. In vitro validation was performed using patient-derived GBM cell lines exposed to exogenous iron. To model TME influences, we employed ex vivo human cortical slice cultures to track three-dimensional GBM invasion, proliferation, and tissue colonization over six days. Findings were further validated using publicly available GBM datasets.



 Transcriptomic analysis revealed enrichment of iron-handling proteins (FTH1, FTL) specifically in highly invasive tumor cells. Iron supplementation increased migration and invasion rates by 2.5-fold (p<0.01) and augmented cluster formation in human ex vivo models. Notably, iron-exposed GBM cells exhibited upregulation of mesenchymal markers (CD44, CHI3L1) alongside iron metabolism genes. In the TME, microglia serve as primary iron reservoirs and potentially facilitated iron transfer to tumor cells via ferroportin-mediated export.



 Our data uncovers iron dysregulation within the GBM TME as a key driver of mesenchymal transition and invasive tumor behavior. Targeting microenvironmental iron handling pathways, including microglial ferroportin-mediated iron export, emerges as a promising therapeutic avenue to mitigate GBM progression.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c543d533453be74d11334f79e20f985af8d50e" target='_blank'>
              P02.20.A IRON-DRIVEN MESENCHYMAL TRANSITION ENHANCES INVASION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            C. Petry, O. Buchholz, A. Mohammed, M. Al Shhab, I. Vasilikos, M. Shah, J. Grauvogel, R. Roelz, A. Vlachos, J. Beck, U. G. Hofmann, K. Joseph, V. Ravi
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Tumor cell metabolome intricately shapes its identity and therapeutic vulnerabilities. Cancer cells dynamically adapt to the tumor microenvironment (TME) – ‘Warburg effect’, a classic example of metabolic plasticity wherein cancer cells preferentially convert glucose to lactate despite availability of oxygen to thrive challenging environments. While metabolic profiling has advanced tumor biology, spatially resolved rewiring of metabolome remains poorly defined. Neuroblastoma (NB), a pediatric cancer with high propensity for hematogenous metastasis exhibits diverse underscored metabolic phenotypes. Our recent studies uniquely identified the predictive, prognostic relevance and function of Retinal Degeneration protein 3 (RD3) in NB pathogenesis including its role in shaping the tumor immune environment. Herein, we investigated whether and how RD3 affects metabolic plasticity in metastatic NB, offering new therapeutic avenues for high-risk (HR) NB. For this, global untargeted metabolomics was performed in bone marrow (BM) derived clones (CHLA-90, CHLA-140) from HR-NB patients and compared with their RD3 stably reinstated counterparts. Data preprocessing, mass detection, EIC building, feature resolving and grouping, ion identity networking and spectral library search was performed using Mzmine4.7 followed by compound identification using Compound discoverer. RD3-driven shift in metabolites were identified using univariate analysis with stringent criteria (log2fold change) coupled with false discovery rate (FDR, log10PadJ). Pathway enrichment analysis was acquired using MetaboAnalyst6.0. Functional validation of RD3-dependent metabolic shift was performed by assessing mitostress (Seahorse Xfe Analyzer) in RD3 expressing clone (CHLA-15) derived from primary (retroperitoneal) site, RD3-null CHLA-90 (BM) and compared with their RD3 silenced and expressed clones respectively. Compared to RD3-/- clones, we observed definitive conserved (cell-line independent) rearrangement of 289 metabolites (288, downregulated; 1, upregulated) in RD3+/+ clones. Predominantly, the metabolites were highly associated in amino acid metabolism (p= 6.61*10-3; FDR, 2.1*10-1) followed by fatty acid and steroid metabolism (p= 3.7*10-2; FDR, 3.02*10-1). Notably, metabolites involved in aerobic glycolysis (Warburg effect, Enrichment score, 1.953) were among the least active in RD3+/+ reinstated clones indicating a significant shift in cell energetics. Consistently, mitostress test further revealed that RD3-loss in HR-NB favors a glycolytic phenotype whereas RD3 expression promoted a more quiescent phase. Our findings explicitly demonstrate that RD3 expression in metastatic NB suppresses glycolysis and reprograms cellular metabolism. Furthermore, the conserved shifts in amino acid and fatty acid metabolism highlight RD3 as a potential metabolic gatekeeper. These RD3-driven alterations could unveil novel therapeutic vulnerabilities in HR-NB, offering new avenues for targeted intervention.



 This work was funded by the DoD CA-210339; OCAST-HR19-045; and the NIH P20GM103639



 Poorvi Subramanian, Natarajan Aravindan, Sivaroopan Aravindan. Breaking the Warburg code: RD3 redefines energetics in metastatic Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdda285b7679b38ef5248455c327701b43a9d61b" target='_blank'>
              Abstract B045: Breaking the Warburg code: RD3 redefines energetics in metastatic Neuroblastoma
              </a>
            </td>
          <td>
            Poorvi Subramanian, N. Aravindan, Sivaroopan Aravindan
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The most common cause of death in pediatric cancer patients is a treatment‐resistant tumor, compounded by metastatic spread, making surgery, radiation therapy, and chemotherapy unfeasible as curative treatment options. However, the mechanisms behind metastatic spread in pediatric tumors remain largely unexplored. We conducted whole‐genome copy number profiling on 171 primary tumor and metastases samples from 17 patients with neuroblastoma, Wilms tumor, or gonadal tumors, and performed targeted deep sequencing on a subset. Phylogenetic reconstruction enabled spatiotemporal tracking of subclones. In total, 11 of 17 patients displayed at least one metastasis arising earlier, defined as occurring before the most recent common ancestor in the primary tumor. In eight patients, metastatic spread was observed several times during tumor evolution, with different subclones from the same primary tumor having metastatic capability, even colonizing the same site. Strikingly, dissemination between metastases (intermetastatic spread) occurred in eight of nine patients with metastases in at least two different sites, indicating that this is a common phenomenon in pediatric malignancy. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24f7bd63ad2744ced5ab2ba8c285ffb7719db78" target='_blank'>
              Tracing metastatic spread in pediatric solid tumors using copy number and targeted deep sequencing
              </a>
            </td>
          <td>
            Natalie Andersson, Michele Ferro, Caroline Jansson, Subhayan Chattopadhyay, Jenny Karlsson, David Gisselsson
          </td>
          <td>2025-09-23</td>
          <td>The Journal of Pathology</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma (PDA) has poor response to chemotherapy with a strong likelihood of cancer recurrence. While previous studies have examined patient samples collected before and after chemotherapy, few have analyzed matched samples from the same patients that allow us to deconvolute interpatient heterogeneity, a large limitation to PDA studies. Here, we study the effects of chemotherapy on tumor cells and their surrounding microenvironment using scRNA sequencing of matched pre- and post- treatment tissue biopsies from 5 patients. We utilized the Seurat v4 pipeline to observe transcriptomic differences and corresponding putative cellular interactions in both treatment states. Unsurprisingly, we found insufficient tumor epithelial cells post-treatment due to the intended cytotoxic effects of chemotherapy, leading us to focus on the tumor microenvironment where we had sufficient cells to analyze. We identified that immune cells in the TME are heterogeneously affected by chemotherapy treatment with patient specific differences, but we did identify a unanimous increase in CXCR4 in CD4 and CD8 positive T cells after treatment. Analysis of CAFs revealed that a subset of fibroblasts lose myofibroblastic features and increasingly acquire 'neural'-like features after treatment. This suggests that chemotherapy-altered fibroblasts may contribute to a pro-tumor microenvironment that facilitates recurrence through neuron–CAF–tumor interactions. Through inferred ligand-receptor pairs identified with the CellChat v2 pipeline, we observed that increased CXCL12 in these neural CAFs interacts with the increased CXCR4 on T-cells. We validated the expression of these genes in patients by staining human patient PDAC tissues (both treated and treatment naïve tissues) for neural genes that were highly enriched post-treatment in our single cell sequencing dataset. We have also begun inhibiting these genes of interest in CAFs and are exploring the effects of these ‘neural’-CAFs on patient-derived tumor organoids in culture.



 Aylin Z. Henstridge, Padma Kadiyala, Ahmed Elhoissiny, Jianhua Liu, Georgina Branch, Nandini Arya, Vaibhav Sahai, Nicole Peterson, Jorge Machicado, Richard Kwon, Allison Schulman, Erik Wamsteker, George Phillips, Martin Fernandez-Zapico, Mark Truty, Timothy Frankel, Fillip Bednar, Marina Pasca di Magliano, Costas Lyssiotis, Eileen S. Carpenter. An increase in neural cancer associated fibroblasts following exposure to chemotherapy in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B079.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e666941a86feb629e0cf1155d415645c6c810d" target='_blank'>
              Abstract B079: An increase in neural cancer associated fibroblasts following exposure to chemotherapy in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Aylin Z. Henstridge, Padma Kadiyala, Ahmed Elhoissiny, Jianhua Liu, Georgina Branch, Nandini Arya, Vaibhav Sahai, Nicole Peterson, Jorge D. Machicado, Richard S Kwon, Allison Schulman, Erik Wamsteker, George Phillips, Martín E Fernandez-Zapico, M. Truty, Timothy L. Frankel, Fillip Bednar, Marina Pasca di Magliano, C. Lyssiotis, Eileen S. Carpenter
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="This editorial discusses Alpsoy et al’s significant study of prognosis of pancreatic ductal adenocarcinoma (PDAC), which lacks histopathological markers. This study evaluated the synergistic prognolymphocytes. Peritumoral budding is significantly correlated with tumor volume, while intratumoral budding is closely related to lymph node metastasis. Peritumoral budding and intratumoral budding are confirmed as independent adverse prognostic factors, and their high levels of expression are associated with immature stromal phenotypes, suggesting the key role of epithelial-mesenchymal transition. These breakthrough findings provide a new multidimensional biomarker system for the prognostic assessment of PDAC, and promote the clinical transformation process of incorporating tumor budding indicators into the pathological reporting process. However, the complexity and spatiotemporal heterogeneity of the tumor microenvironment require us to go beyond traditional morphological analysis and move towards multiomics integration and dynamic monitoring. Through standardized pathological assessment, innovative treatment strategies and interdisciplinary collaboration, it is expected to transform tumor microenvironment-related markers into clinically applicable indicators, ultimately improving the treatment predicament of PDAC. This editorial intended to summarize relevant studies and share some of our views, in order to offer perspectives for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a304affe959bbde58f99c98545a1534cc99601" target='_blank'>
              Microenvironmental code of pancreatic ductal adenocarcinoma: The prognostic symphony of budding, matrix and lymphocytes
              </a>
            </td>
          <td>
            Qing-Qing Zhou, Yi-Meng Ren, Su-Ming Shi, Teng-Fei Liu
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5bbf3daa9e6f41e402a48c9bc2bd5b7f13849c" target='_blank'>
              Systematic elucidation and pharmacologic targeting on non-oncogene dependencies in imatinib-resistant gastrointestinal stromal tumor
              </a>
            </td>
          <td>
            Prabhjot S. Mundi, A. Grunn, A. Kojadinovic, Charles Karan, Ronald B. Realubit, Cristina I. Caescu, H. Hibshoosh, Mahalaxmi Aburi, Mariano J. Alvarez, M.A. Ingham, D. Evans, S. Rothschild, Gary K. Schwartz, Andrea Califano
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a common malignancy with poor prognosis, and exhausted T (TEX) cells, a subset of T cells characterized by progressive loss of effector functions, play a critical role in its progression. This study aimed to investigate the impact of TEX-related genes on immune function, prognosis, and drug sensitivity in GBM through multiomics analysis. Initially, we identified a novel set of TEX-related genes specific to GBM and screened hub genes (CCL5, IL18, CXCR6, FCER1G, TNFSF13B) using conventional statistical methods combined with machine learning. A prognostic risk model was subsequently constructed based on TCGA data and validated in the CGGA cohort. Single-cell and pharmacogenomic analyses revealed significant differences in tumor microenvironment composition and drug sensitivity between risk groups. Notably, Palbociclib emerged as a potential therapeutic agent targeting the novel discovered biomarker CCL5. RT-qPCR results showed that T cells with low CCL5 expression exhibited reduced expression of immune checkpoint-related genes (PD1, TIM3, LAG3) and increased expression of CD28, suggesting enhanced immune function. In conclusion, our findings highlight five hub genes as prognostic markers that could stratify GBM patients with different immune landscapes and levels of drug sensitivity. Furthermore, experimental results suggest that low CCL5 expression could alleviate T cell exhaustion and represent a promising therapeutic target, offering new strategies for improving GBM prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a8b3263311121666e03f6883ae24b875c18a35f" target='_blank'>
              Multiomics Investigation of Exhausted T Cells in Glioblastoma Tumor Microenvironment: CCL5 as a Prognostic and Therapeutic Target
              </a>
            </td>
          <td>
            Ruihao Qin, Menglei Hua, Yaru Wang, Qi Zhang, Yong Cao, Yanyan Dai, Chenjing Ma, Xiaohan Zheng, Kaiyuan Ge, Huimin Zhang, Shi Li, Yan Liu, Lei Cao, Liuying Wang
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe0228ff1d385a0e7d1a25679b76f9d6a5372dc" target='_blank'>
              Transcriptomic profiling of shed cells enables spatial mapping of cellular turnover in human organs.
              </a>
            </td>
          <td>
            Tal Barkai, Oran Yakubovsky, Yael Korem Kohanim, K. Bahar Halpern, Sapir Shir, Noa Oren, Michal Fine, Paz Kelmer, Amit Talmon, Alon Israeli, N. Pencovich, R. Pery, I. Nachmany, S. Itzkovitz
          </td>
          <td>2025-10-02</td>
          <td>Molecular systems biology</td>
          <td>1</td>
          <td>67</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) is the most common form of kidney cancer, leading to 179 000 cancer-related deaths annually. ccRCC initiating cells (CICs) are thought to drive tumor initiation, growth, therapy resistance, and metastasis, yet their molecular characteristics remain poorly defined. This study aims to identify putative CICs and their essential genes using a cell marker-agnostic strategy. Methods We performed a comprehensive analysis of ccRCC transcriptomes at single-cell resolution, developed patient-derived xenograft (PDX) and 3D patient-derived organoids (PDO) models of ccRCC, and conducted functional examinations in these models to investigate our findings. Results Computational modeling of tumor formation using single-cell RNA velocity analysis of five primary and metastatic ccRCC-PDXs revealed a minor cell population as the origin of other tumor cells, representing putative CICs. Pathway and network analyses suggested that a core network of proteins, conventionally known to regulate mitosis, are highly active in CICs and may be essential for their function. These proteins were expressed in PDOs and PDX-derived spheroids established following a CIC enrichment protocol. Spheroid cells exhibited higher tumorigenicity and colony formation ability than parental tumor cells, as confirmed by in-vivo injection in nude mice and in-vitro colony formation assays. Successive in vivo passaging confirmed the self-renewal capacity of spheroid-derived tumors. Pharmacological blockade of candidate proteins elicited dose-dependent inhibitory effect on spheroid and colony formation, with in-vivo validation showing that blocking these proteins significantly delayed tumor growth and more efficiently prevented tumor formation in mice. These results highlight the importance of these proteins in the cancer-initiating abilities of malignant cells. Interestingly, our results suggest that the identified target proteins may elicit their CIC-related function independently from their mitosis regulatory roles. Conclusions We identified and validated essential proteins in RCC-CICs, supported by single-cell transcriptome data and RCC spheroids and PDX models. Targeting the vulnerabilities of RCC-CICs, given their role in tumor initiation, progression, and therapy resistance, offers significant potential for developing new anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edac148fb846da9cb774550aaf710d22fbf68ef2" target='_blank'>
              64defining cancer initiating cells and their vulnerabilities in renal cell carcinoma
              </a>
            </td>
          <td>
            Zohreh Mehrjoo, H. Kuasne, Ariel Madrigal Aguirre, A. Shahini, M. Annis, Anne-Marie Fortier, Tianyuan Lu, Larisa Morales Soto, Hong Zhao, D. Zuo, V. Pilon, M. Dankner, Tamiko Nishimura, Kevin Petrecca, Jonathan Spicer, Peter Siegel, S. Tanguay, H. Najafabadi, Morag Park, Yasser Riazalhosseini
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is a highly lethal brain cancer with limited treatment options and poor long-term survival. Adoptive cellular therapy (ACT) has shown promise in targeting GBM, but many ACT strategies rely on patient tumor tissue to derive tumor-specific antigens, limiting their use in inoperable tumors. Further, tumor heterogeneity—primarily driven by glioma stem cells (GSCs) that survive standard treatments—complicates recurrence. Radial glial cells (RGCs), neurodevelopmental progenitors transcriptionally similar to a subpopulation of GSCs, express developmental antigens (DAs) also found in GBM. We hypothesized that RGCs harbor immunogenic antigens shared with GBM and can serve as a renewable, tumor-independent antigen source for ACT to target GSC cells.



 Using single-cell RNA sequencing, we profiled RGCs cultured from post-natal day 4 (P4) murine cortices and compared their transcriptomes with tumors from murine GBM model KR158B. Shared antigens were screened for immunogenicity using our O.R.A.N. antigen prediction and validation platform. We developed a cDNA antigen library from identified RGC immunogens and used this to generate in vitro transcribed RNA for ACT. T cells were expanded ex vivo using dendritic cells (DCs) pulsed with RGC RNA or total tumor RNA (ttRNA) and tested for cytokine production and tumor-specific killing. These RGC T cells were incorporated into our ACT platform alongside hematopoietic stem cell rescue and serial DC vaccinations. Survival was assessed in GBM-bearing mice. We also tested human RGC antigens against patient-derived xenograft (PDX) GBM models using human T cells and 3D bioprinted immune-tumor interaction assays.



 scRNA-seq analysis revealed a subset of immunogenic antigens shared between RGCs and GBM tumors. RGC T cells demonstrated robust IFNγ and TNFα release, and selectively killed tumor cells in vitro. In vivo, ACT incorporating RGC T cells significantly prolonged survival (p=0.0058) in KR158B GBM murine model, matching the efficacy of ttRNA ACT. Preliminary toxicity studies revealed no overt signs of neurotoxicity or systemic inflammation. In human models, RGC T cells recognized and killed a smaller subset of GBM PDX cells when compared to ttRNA T cells, but showed enriched targeting of GSC-like subpopulations in 3D bioprinted assays.



 RGC-derived antigens represent a potent source of targets for ACT in GBM that can be used to generate a renewable antigen library. This library bypasses the need for tumor resection while maintaining therapeutic efficacy. It offers a novel approach to overcoming initiation of tumor heterogeneity through targeting of neurodevelopmental cell types.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa2fac66b4517361b730352a54b7bf3ec8852fff" target='_blank'>
              OS05.5.A RADIAL GLIAL CELLS AS SOURCE ANTIGEN FOR ADOPTIVE CELLULAR THERAPY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            C. Love, J. W. Figg, I. West, M. Englebart, A. Reid, D. Jin, D. Mitchell, C. Flores
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a860796dffeb4442ab095fa70e047b25f6f73a" target='_blank'>
              Single-cell transcriptomics of the myeloid milieu reveals an angiogenic niche in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Yechan Choi, Minkyu Shim, Suhn Hyung Kim, Duk Ki Kim, Juhee Jeong, J. Byeon, Giyong Jang, Ji-Yeon Kim, Paul Robson, Charles Lee, , Keehoon Jung
          </td>
          <td>2025-11-07</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce6be4d706b4653ff7064a56c5297b13de1df65" target='_blank'>
              Travelling Waves in Gene Expression: A Mathematical Model of Cell-State Dynamics in Melanoma
              </a>
            </td>
          <td>
            Charlotte Taylor Barca, Rotem Salmi-Leshem, Vishaka Gopalan, Sarah Woolner, Kerrie L. Marie, Gareth Wyn Jones, Oliver E. Jensen
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c7c66e7f054de7552ccd4ec8d0186447c75c2a" target='_blank'>
              Single-cell Genomic Copy Number Evolution Reveals Frequent Loss of the Y chromosome in Esophageal Adenocarcinoma
              </a>
            </td>
          <td>
            Shiwei Yin, Na Zhong, Swati Agrawal, Bingru Feng, Sneha Pradhan, Nathan K. Jobalia, Maryam Vaziripour, Sampath K Poreddy, L. Kresty, Erin A Gillaspie, Nicholas E Dietz, W. Al-Refaie, K. Nandipati, Yusi Fu, Jun Xia
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Notch is a contact-dependent signaling pathway that plays critical roles in spatiotemporal patterning during embryogenesis. Beyond its role in development, Notch signaling regulates tissue patterning in adult organs in response to tissue injury or cancer. In the context of cancer, the role of Notch signaling has been extensively characterized in cancer cells, T cells, and endothelial cells. Our previous work has implicated Notch signaling in the myeloid and fibroblast compartments of the pancreatic tumor microenvironment (TME) in the tumor immune response, an area that has been previously understudied. In our current work we use genetically engineered mouse models to inhibit canonical Notch activation in fibroblasts (Pdgfra CreERT2/+, LSL-DNMAML) and myeloid cells (LysM Cre/+, LSL-DNMAML) separately. In both an orthotopic transplant and genetic model (Ptf1a FlpO/+;FSF-Kras G12D/+) of pancreatic cancer progression, inhibition of fibroblast-specific Notch activation had limited effects on disease progression and the transcriptional profile of the fibroblasts themselves. This is consistent with previous evidence of more limited Notch activation in pancreatic cancer-associated fibroblasts and suggests a primary role for fibroblasts in providing Notch ligand (JAG1) to neighboring cells. In contrast, inhibition of myeloid-specific Notch activation resulted in smaller orthotopic tumors and slowed progression in a genetic model of pancreatic neoplasia. These samples had reduced overall macrophage infiltration, and a secondary alteration of the fibroblast phenotypes was also seen. Collectively, these results suggest that Notch signaling in the myeloid compartment supports tumor progression and regulates multiple components of the pancreatic TME. Future studies will further interrogate how myeloid-specific Notch activation promotes disease progression and affects Notch signaling activation in adjacent compartments.



 Allison Bischoff, Wei Yan, Carlos Espinoza, Emily Lasse-Opsahl, Nur Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar. Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df0eac24a706ec3b2888b93172ea892748b9176" target='_blank'>
              Abstract A080: Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment
              </a>
            </td>
          <td>
            Allison C Bischoff, Wei Yan, Carlos E Espinoza, Emily L. Lasse-Opsahl, Nur M Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Bladder cancer (BLCA) exhibits considerable heterogeneity, and research into its tumor microenvironment and prognostic biomarkers remains insufficient. This study aims to identify key prognostic genes and immune microenvironment factors to enhance clinical assessment. Primary tumor (PT) and lymph node metastasis (LNM) samples were obtained from patients with BLCA, and single-cell RNA sequencing (scRNA-seq) was conducted to analyze cellular composition and functional variations. Prognostic genes were identified based on integrated bulk transcriptomic data, leading to the development of a risk model. Functional enrichment analyses further elucidated the molecular characteristics of high- and low-risk cohorts. The scRNA-seq analysis revealed significantly elevated metabolic activity in epithelial cells of LNM. A subpopulation of epithelial cells, defined by 133 characteristic genes, was identified as pivotal in the lymphatic metastasis of BLCA. The prognostic model, derived from nine key genes (APOL1, CAST, DSTN, SPINK1, JUN, S100A10, SPTBN1, HES1, and CD2AP), demonstrated robust predictive performance. Functional enrichment analysis indicated that the high-risk group predominantly activated extracellular matrix (ECM) receptor interactions and the complement pathway, while the low-risk group was primarily associated with carbohydrate metabolism pathways. This study provides insights into the tumor heterogeneity and immune microenvironment of BLCA, introduces a high-precision prognostic model, and establishes a novel theoretical basis for personalized BLCA treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b5f162b6ba0c04c311dbcacd7e486bc5b330efc" target='_blank'>
              Integration of Single-cell and bulk RNA sequencing data uncovers lymphatic metastasis-related prognostic genes and a predictive model in bladder cancer
              </a>
            </td>
          <td>
            Bin Yang, Yangbing Wei, Shuqin Liu, Jinyao Wang, Xuwei Zhao, Jingyu Wang, Yongqiang Fu, Changxi Wang, Xiaofeng Yang
          </td>
          <td>2025-11-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [16],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>