<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-28 09:36:58 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c056a7bc97d38004639e645962dc9ff9c3cfad" target='_blank'>
              Modeling cellular influence delineates functionally relevant cellular neighborhoods in primary and metastatic pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yeonju Cho, Jae W. Lee, Sarah M. Shin, Alexei Hernandez, Xuan Yuan, Jowaly Schneider, Jody E. Hooper, Laura D. Wood, Elizabeth M. Jaffee, Atul Deshpande, W. Ho
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is characterized by high malignancy and early propensity for metastasis, and modest response to immunotherapy due to the immunosuppressive microenvironment. Surgical intervention has shown benefits in treating early-stage SCLC. However, most patients experience recurrence after surgery. The factors associated with relapse free survival in these patients remain unclear. We collected operation specimens from ten early-stage SCLC patients (N0M0), conducted long-term follow-up, and grouped them based on disease status. Subsequently, we performed a retrospective analysis using single-cell spatial imaging mass cytometry to explore the characteristics of tumor cells and differences in the tumor microenvironment, especially the single-cell constitute of immune cells, between the two groups. We found that, in early-stage SCLC, tumor cells display pronounced heterogeneity, both intra-group and inter-group. Patients with early recurrence are characterized by a distinct subpopulation of tumor cells with high Ki-67 expression. Non-relapse patients demonstrate better infiltration of M1 macrophages and stromal cells. Neighborhood analysis suggested that positive interactions between macrophages, stromal cells, and T cells with tumor cells may benefit patient prognosis. Additionally, recurrent tumor cells might enhance their metastatic capacity and remodel the microenvironment through upregulation of GranzymeB or reduction of c-Myc expression. In conclusion, SCLC tumor cells demonstrate tumor heterogeneity and microenvironmental changes in the early clinical stages. A higher proportion of M1 macrophages is associated with prolonged postoperative survival in early-stage SCLC patients. This research provides novel insights and evidence for treating and preventing postoperative recurrence in SCLC. Supplementary Information The online version contains supplementary material available at 10.1007/s10565-025-10056-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1755c9b716477f70da2c740f51d015cee2a36b" target='_blank'>
              Single-cell spatial proteomics of non-relapse small cell lung cancer identifies tumor microenvironment determinants of survival
              </a>
            </td>
          <td>
            Yin Li, L. Xia, Hui Wang, Xinghao Ai, Ying Wang, Qingquan Luo, Yuchen Han, Shun Lu, Xinghua Cheng
          </td>
          <td>2025-06-17</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76828c8834b0e2a09dc4aceaa26a03b54ca8e902" target='_blank'>
              Spatial tumour-immune ecosystems shape the efficacy of anti-PD1 immunotherapy in primary cutaneous melanoma
              </a>
            </td>
          <td>
            Félix Pham, Marion Dufeu, Valentin Benboubker, Maxime Grimont, Amélie Lhorisson, Justine Berthet, Marie Donzel, Raphaël Schneider, Laurie Tonon, A. Doffin, Félix Boivin, Simon Durand, Bertrand Dubois, Jonathan Lopez, Christophe Caux, J. Valladeau-Guilemond, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Early detection of melanoma through skin surveillance is critical for preventing metastatic progression. Primary cutaneous melanomas at early stage offer a unique opportunity to uncover fundamental mechanisms of tumor initiation, progression, and immune surveillance, but detailed spatial profiling of early disease remains limited. Here we integrate high-plex cyclic immunofluorescence (CyCIF) imaging, spatial transcriptomics, and conventional histology to identify factors associated with de-differentiation and dermal invasion in early-stage melanomas. We demonstrate a high level of variability from one primary cancer to the next, from one 100-300 cell microregion to the next within a single cancer, and from one cell to the next within a microregion. Intra-tumoral heterogeneity is influenced by local features of the microenvironment including proximity to T and myeloid cells and to perivascular environments. Thus, tumor plasticity and spatial heterogeneity arise early in melanoma development, potentially allowing for competition among multiple tumor states during the emergence of invasive disease. STATEMENT OF SIGNIFICANCE This study shows that tumor cell dedifferentiation and plasticity are locally patterned in primary human melanomas under the influence of microenvironment components such as myeloid cells and vasculature. As a result, neighborhoods in a single tumor can be as different from each other as in different tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bc89e4de14d8f34f2814e7cf735899a3c934813" target='_blank'>
              Spatial determinants of tumor cell dedifferentiation and plasticity in primary cutaneous melanoma
              </a>
            </td>
          <td>
            Tuulia Vallius, Yingxiao Shi, Edward Novikov, Shishir M Pant, R. Pelletier, Yu-An Chen, Juliann B. Tefft, Ajit Nirmal Johnson, Z. Maliga, Guihong Wan, George F. Murphy, S. Santagata, Yevgeniy R. Semenov, David Liu, C. G. Lian, P. Sorger
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a7baab96b7e0361b2e10254d2ab098fe969a9" target='_blank'>
              Tumor Cells Enriched for Interferon and Inflammatory Programs Pre-Exist in High Grade Serous Ovarian Cancer and are Proportionately Significantly Increased Post Chemotherapy
              </a>
            </td>
          <td>
            B. Winterhoff, Nomeda Girnius, Muyi Liu, Jaeyoung Kang, Mihir Shetty, M. A. Martinez-Gakidis, Weiyi Xu, Mayura Thomas, Linna Lahmadi, Aylin Z. Henstridge, Pilar Baldominos, Shobhana Talukdaer, Z. Chang, Jordan Mattson, Morgan Gruner, Sally A. Mullany, Peter Argenta, Gottfried E. Konecny, Dennis J. Slamon, Ronny Drapkin, Andrew C. Nelson, Timothy K. Starr, Joan S Brugge
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="A comprehensive understanding of cellular behavior and response to the tumor microenvironment (TME) in colorectal cancer (CRC) remains elusive. Here, we introduce the high-definition Visium spatial transcriptomic technology (Visium HD) and investigate formalin-fixed paraffin-embedded human CRC samples (n = 5). We demonstrate the high sensitivity, single-cell-scale resolution and spatial accuracy of Visium HD, generating a highly refined whole-transcriptome spatial profile of CRC samples. We identify transcriptomically distinct macrophage subpopulations in different spatial niches with potential pro-tumor and anti-tumor functions via interactions with tumor and T cells. In situ gene expression analysis validates our findings and localizes a clonally expanded T cell population close to macrophages with anti-tumor features. Our study demonstrates the power of high-resolution spatial technologies to understand cellular interactions in the TME and paves the way for larger studies that will unravel mechanisms and biomarkers of CRC biology, improving diagnosis and disease management strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f42a49dad6477b432ac875de1bc87a28316b76b" target='_blank'>
              High-definition spatial transcriptomic profiling of immune cell populations in colorectal cancer
              </a>
            </td>
          <td>
            Michelli F Oliveira, J. P. Romero, Meii Chung, Stephen R. Williams, Andrew D. Gottscho, Anushka Gupta, Susan E. Pilipauskas, Seayar Mohabbat, Nandhini Raman, David Sukovich, David Patterson, Sarah E B Taylor
          </td>
          <td>2025-06-05</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6cfe1f46dc56e476a3d0be0d700257cc9ae2152" target='_blank'>
              Single-Cell and Spatial Profiling of Tumor Microenvironment Heterogeneity in Human Osteosarcomas
              </a>
            </td>
          <td>
            Xuejing zheng, Jin Yuan, Peng Jin, Xuejun Wang, Runlei Xu, Helin Feng, Xinxin Zhang, Yongjie Xie, Wence Wu
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b6a64a6d2ad28ecff7ab0f31663d8f3fd11b6a" target='_blank'>
              A spatial atlas of chemoradiation therapy in pancreatic cancer identifies cellular and microenvironmental determinants of persister populations
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, Ariana Acevedo-Diaz, K. Rajapakshe, Galia Jacobson, Daniela Tovar, Jimin Min, Guangsheng Pei, Candise Tat, Ayush Suresh, Ching-Wei Tzeng, Matthew HG Katz, M. Bhutani, Huaming Wang, Robert A. Wolff, C. Haymaker, E. Ludmir, Huocong Huang, Xiongfeng Chen, Liang Li, Albert C Koong, Linghua Wang, N. Navin, Dadi Jiang, Ziyi Li, Anirban Maitra, Eugene J. Koay
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1dc109a3a473a99af4a77b675644d925cf4104" target='_blank'>
              Oncogenic and tumor-suppressive forces converge on a progenitor-orchestrated niche to shape early tumorigenesis
              </a>
            </td>
          <td>
            José Reyes, Isabella S. Del Priore, Andrea C. Chaikovsky, Nikhita Pasnuri, Ahmed M. Elhossiny, Tobias Krause, Andrew Moorman, Catherine Snopkowski, Meril Takizawa, Cassandra Burdziak, Nalin Ratnayeke, Ignas Masillioni, Yu-jui Ho, R. Chaligné, P. Romesser, A. Filliol, T. Nawy, John P. Morris, Zhen Zhao, M. P. di Magliano, Direna Alonso-Curbelo, D. Pe’er, Scott W. Lowe
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c21056b3d1d6cc6bc17075f4a894aa4c5791ca" target='_blank'>
              Plasminogen activator inhibitors orchestrate the immunosuppressive tumor microenvironment in pancreatic cancer
              </a>
            </td>
          <td>
            Chiara Falcomatà, Sebastian R. Nielsen, Maximilian M. Schaefer, Bhavya Singh, Alexander Tepper, Divya Chhamalwan, Hunter T. Potak, Maxime Dhainaut, G. Mollaoglu, Matthew D. Park, Miriam Merad, Alessia Baccarini, Brian D. Brown
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47dca4509610b2f00715550942002b3b04fd28df" target='_blank'>
              Tracing colorectal malignancy transformation from cell to tissue scale
              </a>
            </td>
          <td>
            Helena L. Crowell, Irene Ruano, Zedong Hu, Yourae Hong, G. Caratù, Hubert Piessevaux, Ashley Heck, Rachel Liu, Max Walter, Megan Vandenberg, Kimberly Young, Dan McGuire, Evelyn Metzger, Margaret L Hoang, Joseph M. Beechem, Sabine Tejpar, A. Pascual-Reguant, Holger Heyn
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Live-cell imaging (LCI) of modified T cells co-cultured with cancer cells is commonly used to quantify T cell anti-cancer function. Videos captured by LCI show complex multi-cell behavioral phenotypes that go beyond simple cancer cell fluorescence measurements. Here, we develop an unsupervised analysis workflow to characterize LCI data generated using the Incucyte imaging platform. Unlike most LCI analyses, we avoid cell segmentation due to the low spatiotemporal resolution of the LCI videos and high levels of cell-cell contact. Instead, we develop methods that identify global aggregation patterns and local cellular keypoints to characterize the multicellular interactions that determine cancer cell sensitivity to, or escape from, T cell surveillance. We demonstrate our segmentation-free live-cell behavioral analysis (SF-LCBA) methods on TCR T cells from four donors with varying proportions of cells with a beneficial RASA2 knockout and effector-to-target initial concentrations in a co-culture with A375 melanoma cells. We find that different T cell modifications affect the spatiotemporal dynamics of multicellular aggregate formation. In particular, we show that fewer and smaller cancer cell aggregates form at high ratios of effector T cells to target cancer cells and high titrations of T cells with RASA2 knockouts. Our SF-LCBA method identifies, characterizes, and tracks cellular aggregate formation in datasets that are unsuitable for cell segmentation and tracking, opening the door to more therapeutically-relevant measurements of modified T cell therapy cell behavioral phenotypes from LCI data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3384915d0e9331905d28acb564a6fd8addad93fa" target='_blank'>
              Live-cell analyses with unsegmented images to study cancer cell response to modified T cell therapy
              </a>
            </td>
          <td>
            Leo Epstein, A. Weiner, Archit Verma, Mozhgan Saedi, J. Carnevale, Alex Marson, Barbara E. Engelhardt
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colorectal cancer (CRC), including both microsatellite instability (MSI) and microsatellite stability (MSS) subtypes, frequently exhibits intrinsic resistance to immunotherapy. However, the spatial tumor microenvironment (TME) and its role in distinguishing immunotherapy responders from non-responders remain poorly understood. In this study, spatial multiomics, including imaging mass cytometry (n = 50 in-house), spatial proteomics (n = 50 in-house), and spatial transcriptomics (n = 9 in-house), were employed to elucidate the spatial TME of metastatic CRC (mCRC) patients receiving immunotherapy. These methodologies were integrated with single-cell RNA sequencing (scRNA-seq), bulk RNA-seq, and bulk proteomics for comprehensive analysis and validation. A spatial immune atlas containing 314,774 cells was constructed. We found that C1QC+ resident tissue macrophages (RTMs) were more abundant in responders regardless of microsatellite status. Co-localization of C1QC+ RTMs with CD4+ T cells was observed in responders, and MHC-II expression facilitated their interaction. In contrast, cancer-associated fibroblasts inhibited this interaction in non-responders. Moreover, whole genome screening identified key genes involved in antigen presentation in C1QC+ RTMs. Hence, our study highlights the importance of spatial immune mapping in revealing the complex spatial topology of CRC and corresponding immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb9f523addd61c5cd3fcf55f0e488cedb552f07" target='_blank'>
              Spatially resolved C1QC+ macrophage-CD4+ T cell niche in colorectal cancer microenvironment: implications for immunotherapy response
              </a>
            </td>
          <td>
            Hangyu Zhang, Libing Hong, Zhen Dong, Shan Xin, Bo Lin, Jinlin Cheng, Weihong Tian, Bin Li, Jing Wang, Xiaoyan Liu, Chuan Liu, Yuzhi Jin, Yanzhi Feng, Ge Su, Xuqi Sun, Qiqi Liu, Xiaomeng Dai, Yang Gao, Zhou Tong, Lulu Liu, Xudong Zhu, Yi Zheng, Peng Zhao, Tiannan Guo, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-07-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f703b1371a985c2aa75a6f16b3b87971f363e77b" target='_blank'>
              Spatial analysis reveals a novel inflammatory tumor transition state which promotes a macrophage-driven induction of sarcomatoid renal cell carcinoma
              </a>
            </td>
          <td>
            Allison M. May, Suguru Kadomoto, Claire Williams, Alex C. Soupir, Stephanie The, J. McGue, Tyler Robinson, G. Shelley, Mitchell T Hayes, Brooke L. Fridley, Jodi A. Balasi, Paolo M. Ramos Echevarria, J. Dhillon, S. Nallandhighal, Satwik Acharyya, Liying Chen, Jessica Aldous, Nathan Schurman, Veera Baladandayuthapani, Timothy L. Frankel, S. Salami, Brandon J. Manley, R. Mehra, Aaron M. Udager, Evan T. Keller
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive skin cancer with high propensity for metastasis, caused by Merkel-cell-polyomavirus (MCPyV), or chronic UV-light-exposure. How MCPyV spatially modulates immune responses within the tumor microenvironment and how such are linked to patient outcomes remains unknown. We interrogated the cellular and transcriptional landscapes of 60 MCC-patients using a combination of multiplex proteomics, in-situ RNA-hybridization, and spatially oriented transcriptomics. We identified a spatial co-enrichment of activated CD8+ T-cells and CXCL9+PD-L1+ macrophages at the invasive front of virus-positive MCC. This spatial immune response pattern was conserved in another virus-positive tumor, HPV+ head-and-neck cancer. Importantly, we show that virus-negativity correlated with high risk of metastasis through low CD8+ T-cell infiltration and the enrichment of cancer-associated-fibroblasts at the tumor boundary. By contrast, responses to immune-checkpoint blockade (ICB) were independent of viral-status but correlated with the presence of a B-cell-enriched spatial contexts. Our work is the first to reveal distinct immune-response patterns between virus-positive and virus-negative MCC and their impact on metastasis and ICB-response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28305c5ea574406c24e96fc8bdd4ca5602d07afe" target='_blank'>
              Spatially organized inflammatory myeloid-CD8+ T cell aggregates linked to Merkel-cell Polyomavirus driven Reorganization of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Maximilian Haist, M. Matusiak, Yuqi Tan, S. Zimmer, H. Stege, Tim N. Kempchen, S. Mitschke, Pauline Chu, B. Weidenthaler-Barth, Graham L. Barlow, F. Rogall, Antonio Delgado Gonzalez, M. Baertsch, Silvia Albertini, M. Gariglio, C. Borgogna, Y. Goltsev, S. Grabbe, John W. Hickey, Garry P. Nolan
          </td>
          <td>2025-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e045f1d13d966784369bbbd3e625da8bdfdda349" target='_blank'>
              Alterations of the composition and spatial organization of the microenvironment following non-dysplastic Barrett’s esophagus through progression to cancer
              </a>
            </td>
          <td>
            Meng-Lay Lin, John W. Hickey, A. Passman, Emanuela Carlotti, S. Devkumar, Yannick H Derwa, Joanne ChinAleong, R. Hackett, James Evans, V. Sangwan, Marco Novelli, Manuel Rodriguez Justo, Nicholas A. Wright, Marnix Jansen, Tim J. Underwood, Sui Huang, Philippe Gascard, M. McLean, Christian M. Schürch, T. Tlsty, Lorenzo Ferri, GP Nolan, S. A. McDonald
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bec286991f2affb3f9522a83843f9469ba7b89" target='_blank'>
              Developmental Plasticity and Stromal Co-option Shape a Pituitary Neuroendocrine Tumor Transcriptional Continuum
              </a>
            </td>
          <td>
            Robert C. Osorio, Jun Y. Oh, Jangham Jung, Alexander J. Ehrenberg, A. Saha, Meeki K. Lad, Harmon Khela, Nicole Brennick, P. Giannikopoulos, William W. Seeley, Lea T. Grinberg, Aaron Diaz, M. Aghi
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e20040


 Background:
 Immunotherapy has revolutionized cancer treatment, particularly in non-small cell lung cancer (NSCLC), yet a large fraction of patients remain refractory to treatment. Understanding treatment response requires characterization of immune cell populations and their spatial organization, as cellular neighborhoods critically influence immune cell recruitment, activation, and function. Current imaging-based and NGS-based methods struggle to achieve clear separation of individual cells in densely packed regions. Hence, we developed STAMP-seq, a novel method tagging single nuclei with cleavable spatial barcodes using high-density DNA sequencing chips, to uncover immunotherapy response-related modules and dynamic changes in NSCLC.
 Methods:
 STAMP-seq employs spatially-encoded DNA arrays to label individual nuclei with cleavable barcodes. Following validation in mouse brain tissue, we analyzed surgically resected NSCLC samples from immunotherapy-treated patients, integrating spatial transcriptomics with BCR profiling to characterize treatment response features.
 Results:
 STAMP-seq achieves precise spatial barcoding, detecting over 30,000 cells per adult mouse brain section with reliable cell subtype assignment. In immunotherapy-treated NSCLC samples, spatial analysis revealed distinct cellular communities associated with treatment response. Responding patients showed enrichment of tertiary lymphoid structures and specialized plasma cell niches, with tumor cells exhibiting elevated TNF-α signaling and reduced proliferation. These tumor cells showed proximity to

 ROBO2

 + CAFs while remaining distant from

 ACTA2

 + CAFs. BCR clonotype and pseudotime analysis revealed that

 IGHG1

 + plasma cells are recruited from TLS or vasculature by stromal-rich "transit stops" containing antigen-presenting capabilities CAFs through

 HMGB1

 -

 CXCR4

 interaction.
 Conclusions:
 STAMP-seq enables high-resolution spatial transcriptomics at single-nucleus level. Our characterization of NSCLC spatial architecture revealed previously unrecognized immune response organization, particularly how specialized stromal-rich "transit stops" coordinate cancer-suppressing plasma cell recruitment. These findings demonstrate how spatial information can advance diagnostic and prognostic module development for human diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/023df8c02dd2a154737c443a9b87fee23263bac2" target='_blank'>
              Use of STAMP-seq to enable spatial transcriptomics through single-nucleus in situ tagging and exploration of relationship of dynamic plasma cell modules and immunotherapy response in non-small cell lung cancer.
              </a>
            </td>
          <td>
            Yitong Pan, Huan Yan, Jinhuan Han, Rui Wu, Xingyong Ma, Ying Guan, Keyu Li, Qingquan Wei, Guangxin Zhang, Shaozhuo Jiao, Yongchang Zhang, Chenxi Tian
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3c75d8fc63308ed8697666b49a18333c902334" target='_blank'>
              A Single-cell and Spatially Resolved Cell Atlas of Human Esophageal Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Yong Shi, Ke An, Yu Qi, Xinhan Zhang, Yueqin Wang, Xuran Zhang, Shaoxuan Zhou, Ouwen Li, Yanan Song, Jiayi Zhou, Yue Du, Mingyang Hou, Yun-Gui Yang, QuanC. Kan, Xin Tian
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pituitary neuroendocrine tumors (PitNETs) can be invasive or aggressive, yet the mechanisms behind these behaviors remain poorly understood, impeding treatment advancements. Here, we integrat single-cell RNA sequencing and spatial transcriptomics, analyzing over 177,000 cells and 35,000 spots across 57 tissue samples. This comprehensive approach facilitates the identification of PitNETs tumor populations and characterizes the reconfiguration of the tumor microenvironment (TME) as PitNETs progress and invade. We trace the trajectory of TPIT-lineage PitNETs and identify an aggressive tumor cluster marked by elevated p53-mediated proliferation and a higher Trouillas classification, both associated with tumor progression. Additionally, we document the heterogeneity of immune stromal cells within PitNETs, particularly noting the enrichment of SPP1+ tumor associated macrophages (TAMs) in invasive tumors. These TAMs facilitate tumor invasion through the SPP1-ITGAV/ITGB1 signaling pathway. Our in-depth single-cell and spatial analysis of PitNETs uncovers the molecular dynamics within the TME, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1b01507bfc7f0aa4eb306817808e84f0ce1133" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal tumor heterogeneity and immune remodeling involved in pituitary neuroendocrine tumor progression
              </a>
            </td>
          <td>
            Wan Su, Zhang Ye, Jifang Liu, K. Deng, Jinghua Liu, Huijuan Zhu, Lian Duan, Chen Shi, Linjie Wang, Yuxing Zhao, Fengying Gong, Yi Zhang, Bo Hou, Hui You, Feng Feng, Qing Ling, Yu Xiao, Yongdong Guo, Wenyi Fan, Sumei Zhang, Zixin Zhang, Xiaomin Hu, Yong Yao, Chunhong Zheng, Lin Lu
          </td>
          <td>2025-05-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90f49812ba9bf54efb86922eb7c7255926faf91" target='_blank'>
              Mapping and reprogramming microenvironment-induced cell states in human disease using generative AI
              </a>
            </td>
          <td>
            Amir Akbarnejad, Lloyd Steele, D. Jafree, Sebastian Birk, M. Sallese, K. Rademaker, Adam Boxall, Ben Rumney, Catherine Tudor, Minal Patel, M. Prete, Stanislaw Makarchuk, Tong Li, Heather Stanley, A. Foster, Kenny Roberts, Andrew L. Trinh, C. E. Villa, G. Testa, S. Mahil, Arash Mehrjou, Catherine Smith, Sattar Vakili, M. Clatworthy, Thomas J. Mitchell, O. Bayraktar, M. Haniffa, M. Lotfollahi
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90517c7621a56cfd44f51891279305813cf3e121" target='_blank'>
              Notch3 regulates pericyte phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            N. Chalkidi, Athanasia Stavropoulou, V. Arvaniti, Christina Paraskeva, Artemis Monogyiou, Maria Sakkou, Christoforos Nikolaou, V. Koliaraki
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edded8a4572e2f0a421a8d1ab432a7fe9035dbf5" target='_blank'>
              A Human Tumor-Immune Organoid Model of Glioblastoma
              </a>
            </td>
          <td>
            Shivani Baisiwala, Elisa Fazzari, Matthew Li, A. Martija, Daria J. Azizad, Lu Sun, Gilbert Herrera, Trinh L Phan, Amber Monteleone, David A. Nathanson, Anthony C. Wang, Won Kim, R. Everson, Kunal S. Patel, Linda M. Liau, Robert M Prins, A. Bhaduri
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Simple Summary Spatial transcriptomics (ST) is an innovative technology that maps gene expression while preserving tissue structure, offering new perspectives on disease mechanisms in the lung. This review explores how ST has advanced our understanding of lung cancer and other pulmonary diseases. In lung cancer, ST helps reveal tumor heterogeneity, immune evasion mechanisms, and spatially distinct cellular subtypes that influence therapy response. In diseases such as pulmonary fibrosis, COPD, and asthma, ST identifies localized immune niches, fibroblast diversity, and vascular remodeling that drive disease progression. The review also compares major ST platforms like GeoMx, Visium, and Xenium and highlights current clinical trials and future directions. By integrating molecular insights with spatial context, ST holds great promise for improving diagnosis, treatment, and precision medicine in respiratory diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d6db631a991c94025cd8ea23579c8fe3e08735" target='_blank'>
              Spatial Transcriptomics in Lung Cancer and Pulmonary Diseases: A Comprehensive Review
              </a>
            </td>
          <td>
            D. Kang, Yoonjoo Kim, Ji Hyeon Lee, Hyeong Seok Kang, Chaeuk Chung
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Angiosarcoma, a rare and highly aggressive malignancy originating from vascular endothelial cells, is characterized by its rapid progression, high invasiveness, and poor prognosis. Due to the limited understanding of its tumor microenvironment (TME) and the absence of effective treatments, further research is essential to elucidate its pathogenic mechanisms and improve therapeutic strategies. Objective This study aims to characterize the cellular heterogeneity and unique TME of primary breast angiosarcoma using single-cell RNA sequencing (scRNA-seq), to identify potential therapeutic targets and improve clinical outcomes. Methods Tumor samples were obtained from a patient with bilateral primary breast angiosarcoma and two patients with invasive breast cancer. Single-cell RNA sequencing (scRNA-seq) was conducted to capture the transcriptomic profiles of individual cells within the tumor samples. Following stringent quality control, a total of 31,771 cells were analyzed using comprehensive bioinformatics approaches. Cell populations were identified and classified into distinct cell types, and differential gene expression analysis was performed to explore key signaling pathways. Functional enrichment analysis was used to identify pathways related to tumor progression and immune evasion. Additionally, cell-cell communication networks were mapped to understand interactions within the TME, with a focus on pathways that may serve as therapeutic targets. Results The scRNA-seq analysis revealed significant differences in the distribution of perivascular cells, fibroblasts, T cells, endothelial cells, and myeloid cells in breast angiosarcoma compared to invasive breast cancer. Key pathways enriched in angiosarcoma samples included growth factor binding, platelet-derived growth factor binding, and ribosome biogenesis, with abnormal expression of several ribosomal proteins. Notably, genes such as FAT4, KDR, FN1, and KIT were highly expressed in angiosarcoma endothelial cells, correlating with poor prognosis. Cell communication analysis highlighted the CXCL12-CXCR4 axis as a crucial mediator of the TME in angiosarcoma. Conclusion This study provides critical insights into the TME of primary breast angiosarcoma, highlighting potential molecular targets and pathways for therapeutic intervention. These findings may inform the development of more effective treatment strategies for this rare and challenging tumor type. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02022-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61afd778df17e939f7a978e09a799d8e740668a2" target='_blank'>
              Dissecting the tumor microenvironment in primary breast angiosarcoma: insights from single-cell RNA sequencing
              </a>
            </td>
          <td>
            Peikai Ding, Shengbin Pei, Yi Zhai, Zheng Qu, Yazhe Yang, Xiaolong Feng, Qiang Liu, Xiangyu Wang, Wenxiang Zhang, Zhongzhao Wang, Xiangyi Kong, Jing Wang, Yi Fang
          </td>
          <td>2025-06-05</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e652ee8a36b3b3637d56b8e5667a1abc73cc2445" target='_blank'>
              Differential Assembly of Mouse and Human Tumor Microenvironments
              </a>
            </td>
          <td>
            T. Courau, Rebecca G. Jaszczak, B. Samad, Emily Flynn, Nayvin W. Chew, Gabriella C. Reeder, Jessica Tsui, A. Ray, Harrison Wismer, Daniel Bunis, Leonard C. Lupin-Jimenez, Noah V. Gavil, David Masopust, John P Graham, Daniel A. Skelly, Xavier Vesco, Edison T. Liu, G. Fragiadakis, A. Combes, Matthew F. Krummel
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631d6fa6d4de4b757d78ca3c32e8b7a61d4aff26" target='_blank'>
              Single cell functional immunogenomics of the fallopian tube reveals a precursor immune surveillance network for ovarian cancer prevention
              </a>
            </td>
          <td>
            Luyao Wang, Breeshey Roskams-Hieter, Nosheen Hussain, Joel Nulsen, Aneesh Aggarwal, May Sallam, Lili Wang, Lena Rai, Amro Ahmed-Ebbiary, Aws Al-Deka, Takashi Takeda, Mara Artibani, H. Soleymani majd, Jason Yap, Christopher Yau, Nancy Zaarour, A. Ahmed
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Given the graft-versus-leukemia effect observed with allogeneic hematopoietic stem cell transplantation in refractory or relapsed acute myeloid leukemia (AML), immunotherapies have been explored in nontransplant settings. We applied a multiomic approach to examine bone marrow interactions in patients with AML treated with pembrolizumab and decitabine. Using extensively trained nuclear and membrane segmentation models, we achieved precise transcript assignment and deep learning–based image analysis. To address read-depth limitations, we integrated single-cell RNA sequencing with single-cell spatial transcriptomics from the same sample. Quantifying cell-cell distances at the edge level enabled more accurate tumor microenvironment analysis, revealing global and local immune cell enrichment near leukemia cells postpembrolizumab treatment, potentially linked to clinical response. Furthermore, ligand-receptor analysis indicated potential alterations in specific signaling pathways between leukemia and immune cells following immunotherapy treatment. These findings provide insights into immune interactions in AML and may inform therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0066efa7bc3f52352d23a57bb535219a1adb0cf1" target='_blank'>
              Single-cell spatial transcriptomics reveals immunotherapy-driven bone marrow niche remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R. Herzog, Abigail Wong-Rolle, Laura W. Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann F. Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine R. Calvo, Christopher S. Hourigan, Kasper D. Hansen, Chen Zhao
          </td>
          <td>2025-07-09</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) is emerging as a powerful technology that transforms our understanding of thyroid cancer by offering a spatial context of gene expression within the tumor tissue. In this review, we synthesize the recent applications of ST in thyroid cancer research, with a particular focus on the heterogeneity of the tumor microenvironment, tumor evolution, and cellular interactions. Studies have leveraged the spatial information provided by ST to map distinct cell types and expression patterns of genes and pathways across the different regions of thyroid cancer samples. The spatial context also allows a closer examination of invasion and metastasis, especially through the dysregulation at the tumor leading edge. Additionally, signaling pathways are inferred at a more accurate level through the spatial proximity of ligands and receptors. We also discuss the limitations that need to be overcome, including technical limitations like low resolution and sequencing depth, the need for high-quality samples, and complex data handling processes, and suggest future directions for a wider and more efficient application of ST in advancing personalized treatment of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3b3022cb800cbef42ede8e65012209b885afa4" target='_blank'>
              Spatial Transcriptomics in Thyroid Cancer: Applications, Limitations, and Future Perspectives
              </a>
            </td>
          <td>
            Chaerim Song, Hye-Ji Park, Man S. Kim
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f55f1547ba1283c169484e339023bae14e76f4" target='_blank'>
              Plastimos: a live cell imaging-based framework to study dynamics of EMT-mediated cellular plasticity in breast cancer
              </a>
            </td>
          <td>
            Saba Sameri, Durdam Das, Silvia Materna-Reichelt, Nataša Stojanović Gužvić, L. Wöhrl, Martin Hoffmann, Hedayatollah Hosseini
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb2bae8a5c0fba39a07b5a8201b6d06e217381d" target='_blank'>
              Lineage memory shapes viral resistance barriers in human skin
              </a>
            </td>
          <td>
            Laura C. Van Eyndhoven, Grant Kinsler, Jingchao Zhang, Aoife O’Farrell, Rim Abderrahim, Apurv Srivastav, Catherine G. Triandafillou, Todd M. Greco, Kenneth S. Zaret, I. Cristea, Megan H. Orzalli, N. Drayman, Abhyudai Singh, Arjun Raj
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d59997cf6c8e6c97ba758d271ce928cfa49100ba" target='_blank'>
              Transcriptomic Profiles from Normal and Tumor Tissue Samples Reveal Distinct Venule Populations and Novel Tumor Endothelial Cell Markers in Breast Cancer
              </a>
            </td>
          <td>
            K. Phoenix, Vijender Singh, Patrick A. Murphy, Kevin P. Claffey
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e14574


 Background:
 Understanding cellular interactions within the tumor microenvironment (TME) is essential for elucidating disease progression and advancing immunotherapy. The TME is a complex ecosystem composed of cells with dysregulated metabolism immune response and signaling pathways, which in turn influence tumor development and treatment response. Multiplexed assessment serves as an important tool for clinical oncology. The simultaneous readout of multiple processes can provide biological insights and elucidate disease mechanism. Imaging Mass Cytometry (IMC) is a spatial biology imaging technique that utilizes CyTOF technology and enables deep characterization of the diversity and complexity of the TME. IMC technology offers scalable, high-throughput acquisition while generating high-quality data with true dynamic range of signal without amplification or fluorescence-based limitations such as spectral overlap and autofluorescence.
 Methods:
 To study cellular processes and their roles in tumor progression, we utilized the Human Cell Metabolism and Human Cell Signaling Panels to investigate energy production, cellular homeostasis and mitogenic signaling pathways. We then mapped these processes to the types of cells in the TME by using the Human Immuno-Oncology IMC Panel and the Human T Cell Exhaustion IMC Panel or the Maxpar Neuro Phenotyping IMC Panel Kit to characterize immune cell and neurological phenotypes in detail. We first acquired data using Preview Mode to assess the whole tissue, followed by higher-resolution imaging of selected regions of interest using Cell Mode or of the whole tissue section using Tissue Mode.
 Results:
 Our data analysis revealed significant insights into the spatial organization and metabolic profile of cells across cancer tissues. Elevated glycolysis and mTOR pathway activation suggested adaptations to hypoxia and anabolic growth in tumor areas, while interactions between fibroblasts and immune cells highlighted crosstalk within the TME. Our Neuro Phenotyping Panel was used to reveal potential for immune response in glioblastoma. Unsupervised pixel clustering and hierarchical clustering using MCD SmartViewer highlighted metabolic activity and activation of signaling pathways within tumor regions.
 Conclusions:
 Comprehensive spatial biology profiling using the IMC approach highlights the interconnected roles these pathways play in promoting tumor survival and resistance to therapies. These findings, which illuminate the metabolic and signaling heterogeneity of the TME, are crucial for developing future prognostic assessments and have the potential to guide more effective, personalized cancer therapies. For Research Use Only. Not for use in diagnostic procedures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b16079bb313271e366d31125c0f7c2617d3a11a" target='_blank'>
              Spatial profiling of solid tumor microenvironment using imaging mass cytometry with high-plex panels.
              </a>
            </td>
          <td>
            T. Pfister, J. Y. Chwee, Nick Zabinyakov, Qanber Raza, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) remains a lethal malignancy. Although immunochemotherapy regimens have improved patient survival rates, drug resistance still occurs in a significant subset of patients, highlighting the importance of elucidating the mechanisms within the tumor microenvironment. Here, we applied spatial single-cell transcriptomics to investigate the spatial characteristics of SCLC and their associations with immunochemotherapy resistance. By analyzing samples from 18 patients with extensive-stage SCLC, we identified two distinct epithelial cell subtypes: Epi-I and Epi-II. Epi-I exhibited high proliferative activity and was associated with treatment resistance and poor survival outcomes. In contrast, Epi-II showed more spatial contact with immune cells and was associated with treatment sensitivity. Further analysis uncovered a fascinating cellular transition paradigm, wherein Epi-I may be derived from Epi-II, with myeloid cells playing a facilitatory role in this transformation cascade. Specifically, within the spatial zone that was enriched with the Epi-II, epithelial cells may secrete MIF gene, which promoted the polarization of myeloid cells towards the M2 macrophages. The M2-polarized myeloid cells subsequently upregulated the expression of SPP1 that in turn triggered the activation of the PI3K-AKT signaling pathway in the adjacent epithelial cells, driving the conversion of Epi-II to Epi-I cells. Our findings revealed that the intricate crosstalk between epithelial and myeloid cells constitutes a pivotal resistance mechanism in SCLC, and targeting the SPP1/MIF pathway emerged as a promising strategy with the potential to enhance the treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0302ec6123f8cd6d62c8c99e2e99dcc7087718b" target='_blank'>
              Spatial transcriptomics reveals macrophage domestication by epithelial cells promotes immunotherapy resistance in small cell lung cancer
              </a>
            </td>
          <td>
            Yu Sun, Minghui Zhang, Yanbin Zhao, Yubo Yan, Lei Wang, Xuhui Liu, Songsong Xia, Bingbing Wang, Xiaoxin Zhang, Yan Wang
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Elucidating the role of somatic mutations in cancer, healthy tissues, and aging depends on methods that can accurately characterize somatic mosaicism across different cell types, as well as assay their impact on cellular function. Current technologies to study cell-type-specific somatic mutations within tissues are low-throughput. We developed Duplex-Multiome, incorporating duplex consensus sequencing to accurately identify somatic single-nucleotide variants (sSNV) from the same nucleus simultaneously analyzed for single-nucleus ATAC-seq (snATAC-seq) and RNA-seq (snRNA-seq). By introducing strand-tagging into the construction of snATAC-seq libraries, duplex sequencing reduces sequencing error by >10,000-fold while eliminating artifactual mutational signatures. When applied to 98%/2% mixed cell lines, Duplex-Multiome identified sSNVs present in 2% of cells with 92% precision and accurately captured known sSNV mutational spectra, while revealing unexpected subclonal lineages. Duplex-Multiome of > 51,400 nuclei from postmortem brain tissue captured sSNV burdens and spectra across all major brain cell types and subtypes, including those difficult to assay by single-cell whole-genome sequencing (scWGS). This revealed for the first time that diverse neuronal and glial cell types show distinct rates and patterns of age-related mutation, while also directly discovering developmental cell lineage relationships. Duplex-Multiome identified clonal sSNVs occurring at increased rates in glia of certain aged brains, as well as clonal sSNVs that correlated with changes in expression of nearby genes, in both neurotypical and autism spectrum disorder (ASD) individuals, directly demonstrating that somatic mutagenesis can contribute to gene expression phenotypes. Duplex-Multiome can be easily adopted into the 10X Multiome protocol and will bridge somatic mosaicism to a wide range of phenotypic readouts across cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c45c976013cae1b807eb9d7fbbf98792d114f7a" target='_blank'>
              Cell-type-specific patterns and consequences of somatic mutation in development and aging brain
              </a>
            </td>
          <td>
            Andrea J. Kriz, Shulin Mao, Diane D. Shao, Daniel A. Snellings, Rebecca E. Andersen, Guanlan Dong, Chanthia C. Ma, Hayley E. Cline, A. Huang, Eunjung Lee, Christopher A. Walsh
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Organotropism results from the functional versatility of metastatic cancer cells to survive and proliferate in diverse microenvironments. This adaptivity can originate in clonal variation of the spreading tumor and is often empowered by epigenetic and molecular reprogramming of cell regulatory circuits. Related to organotropic colonization of metastatic sites are environmentally-sensitive, differential responses of cancer cells to therapeutic attack. Accordingly, understanding the organotropic profile of a cancer and probing the underlying driver mechanisms are of high clinical importance. However, determining systematically the organotropism of one cancer versus the organotropism of another cancer, potentially with the granularity of comparing the same cancer type between patients or tracking the evolution of a cancer in a single patient for the purpose of personalized treatment, has remained very challenging. It requires a host organism that allows observation of the spreading pattern over relatively short experimental times. Moreover, organotropic patterns often tend to be statistically weak and superimposed by experimental variation. Thus, an assay for organotropism must give access to statistical powers that can separate ‘meaningful heterogeneity’, i.e., heterogeneity that determines organotropism, from ‘meaningless heterogeneity’, i.e., heterogeneity that causes experimental noise. Here we describe an experimental workflow that leverages the physiological properties of zebrafish larvae for an imaging-based assessment of organotropic patterns over a time-frame of 3 days. The workflow incorporates computer vision pipelines to automatically integrate the stochastic spreading behavior of a particular cancer xenograft in tens to hundreds of larvae allowing subtle trends in the colonization of particular organs to emerge above random cell depositions throughout the host organism. We validate our approach with positive control experiments comparing the spreading patterns of a metastatic sarcoma against non-transformed fibroblasts and the spreading patterns of two melanoma cell lines with previously established differences in metastatic propensity. We then show that integration of the spreading pattern of xenografts in 40 – 50 larvae is necessary and sufficient to generate a Fish Metastatic Atlas page that is representative of the organotropism of a particular oncogenotype and experimental condition. Finally, we apply the power of this assay to determine the function of the EWSR1::FLI1 fusion oncogene and its transcriptional target SOX6 as plasticity factors that enhance the adaptive capacity of metastatic Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acab5ca78a81e16038185b4373a584725e52ba21" target='_blank'>
              Low-branching vessel architecture shapes immune cell niches and predicts immune responses in renal cancer
              </a>
            </td>
          <td>
            M. Toma, Yangping Li, Melina Kehl, Tim Kempchen, L. Esser, Katharina Baschun, S. Leonardelli, Thomas Pinetz, R. Turiello, M. Yong, Natalie Pelusi, Guillermo Altamirano-Escobedo, Eduardo Bayro-Corrochano, Sebastian Kadzik, M. Eckstein, Lukas Flatz, Fabian Hörst, J. Kleesiek, J. Saal, Glen Kristiansen, Manuel Ritter, J. Ellinger, Viktor Grünwald, Niklas Klümper, Alexander Effland, M. Hölzel
          </td>
          <td>2025-06-06</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer remains the leading cause of cancer-related deaths worldwide. Despite the remarkable efficacy of immune checkpoint inhibitors, only a subset of lung adenocarcinoma (LUAD) patients respond and many eventually progress, underscoring the need to develop novel combination therapies. The interaction between cancer cells and the tumor microenvironment (TME), a complex niche including cells of the innate and adaptive immune system, cancer-associated fibroblasts (CAFs), vascular cells, and extracellular matrix (ECM) plays a vital role in tumorigenesis and cancer progression. Using fate mapping, we previously identified a novel subpopulation of resident vascular stem cells derived from vascular smooth muscle cells, designated AdvSca1-SM (vascular Adventitial location, Stem cell antigen-1 expression, SMooth muscle origin) cells. AdvSca1-SM cells are the predominant cell type responding to vessel wall dysfunction and their selective differentiation to myofibroblasts promotes vascular fibrosis. SMC-to-AdvSca1-SM cell reprogramming is dependent on SMC induction of the transcription factor KLF4, and KLF4 is essential for maintenance of the stem cell phenotype. The function of AdvSca1-SM cells in LUAD tumorigenesis has not been explored. Using an orthotopic immunocompetent mouse model of LUAD, we demonstrate that AdvSca1-SM cells are a major component of lung tumors, significantly contributing to cancer associated fibroblasts (CAFs). Compared to AdvSca1-SM cells, CAFs have altered ECM gene expression with a reduction of a stemness signature. AdvSca1-SM-specific genetic ablation of Klf4 altered their phenotype resulting in inhibition of communication between cancer cells and CAFs, decreases in innate immunosuppressive populations, and increased T cell infiltration into the tumor. Targeting this population may represent a novel strategy to improve the response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0575ac4a5236a27350d4ce6527d7cbb1d6c6e472" target='_blank'>
              Klf4-dependent pivotal role of smooth muscle-derived Gli1+ lineage progenitor cells in lung cancer progression
              </a>
            </td>
          <td>
            Sizhao Lu, Andre C. Navarro, Amber M. Johnson, Tysen Noble, Austin J. Jolly, Allison M. Dubner, Karen S. Moulton, Raphael A. Nemenoff, M. Weiser-Evans
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly aggressive intraocular malignancy with limited therapeutic options for metastatic disease. Existing transgenic UM mouse models inadequately recapitulate human disease progression, while transplant models lack immune competence for studying the tumor immune microenvironment and therapeutic interventions. To address these limitations, we developed a genetically engineered mouse model incorporating stepwise genetic alterations implicated in human UM progression. Spatiotemporally controlled expression of mutant GNAQQ209L from the endogenous locus induced choroidal nevi with limited penetrance. Concomitant BAP1 deletion enhanced nevus formation, while further MYC activation led to fully penetrant intraocular tumors with metastatic potential. Single-cell RNA sequencing revealed malignant cells segregated into Melanocytic and Neural Crest-like subpopulations characterized by distinct transcriptional and biosynthetic programs. Trajectory analyses inferred dedifferentiation from the Melanocytic toward the Neural Crest-like state during tumor progression. Comparison to human UM revealed commonalities with highly aggressive Class 2 UM, including gene expression signatures and copy number gains affecting genes that map to human chromosome 8q beyond the activated MYC allele, suggesting cooperative effects of multiple drivers in this chromosomal region. The tumor microenvironment featured immunosuppressive macrophage populations and exhausted T cells, closely resembling human UM. This physiologically relevant, immune-competent model provides a platform for investigating UM biology, functionally characterizing candidate driver genes, and developing immune-based therapeutic strategies. SIGNIFICANCE STATEMENT We developed a mouse model that resembles the genetic progression and phenotypic plasticity of human UM. This spatially controlled model confirms the critical role of driver mutations in GNAQ and BAP1, proposes MYC as a promoter of malignant transformation in coordination with other chromosome 8q genes, and reveals UM progression through distinct cellular states. This model offers an urgently needed preclinical platform for understanding the immunogenomics of UM and for testing immune and targeted treatments for this lethal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35df183db757e5d1e691fe22c0b60fdf40bee0e5" target='_blank'>
              A multi-step immune-competent genetic mouse model reveals phenotypic plasticity in uveal melanoma
              </a>
            </td>
          <td>
            Xiaonan Xu, Xiaoxian Liu, J. Dollar, Xiao Liu, Neel Jasani, Benjamin Posorske, S. Sriramareddy, Vinesh Jarajapu, J. Kuznetsoff, John Sinard, Richard L. Bennett, Jonathan D. Licht, K. Smalley, J. W. Harbour, Xiaoqing Yu, F. Karreth
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The most common cause of death in neurofibromatosis type 1 (NF1) is the development of malignant peripheral nerve sheath tumor (MPNST), a deadly sarcoma that can transform from benign plexiform neurofibromas (PN) or premalignant atypical neurofibromas (AN). We built a single-cell dataset of 55 NF1-associated PN, AN, and MPNST to define cellular changes in neurofibroma at-risk of malignant transformation. Integrative analysis of changes in the tumor microenvironment revealed the emergence of malignant tumor cells, regulatory T cells, and loss of activated macrophages in MPNST. Using this reference dataset, we validated findings using anchor-based label transfer in an additional 19 NF1 nerve sheath tumors profiled with single cell sequencing, and public datasets. We then defined protein biomarkers of malignant transformation from high-throughput proteomic analysis of plasma samples collected from 45 NF1 patients that correlated to mRNAs specific to MPNST cell populations. Fifty plasma proteins accurately and non-invasively distinguished patients with MPNST from those with premalignant tumors. These markers should improve the ability to identify high-risk neurofibromas for improved cancer surveillance and enable early detection of malignant transformation in NF1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea5d7e2095827b87208bdc18119d43a501ce00d" target='_blank'>
              Single-cell tumor microenvironment profiling informs a circulating proteome test for the interception of malignant transformation in NF1 nerve sheath tumors
              </a>
            </td>
          <td>
            Jack Shern
          </td>
          <td>2025-06-18</td>
          <td>Research Square</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is one of the most lethal cancers for humans. HCC is highly heterogeneous. In this study, we performed ultra-depth (∼1 million reads per spot) sequencing of 6,320 spatial transcriptomes on a case of HCC. Sixteen distinct spatial expression clusters were identified. Each of these clusters was spatially contiguous and had distinct gene expression patterns. In contrast, benign liver tissues showed minimal heterogeneity in terms of gene expression. Numerous immune cell-enriched spots were identified in both HCC and benign liver regions. Cells adjacent to these immune cell-enriched spots showed significant alterations in their gene expression patterns. Interestingly, the responses of HCC cells to the nearby immune cells were significantly more intense and broader, while the responses of benign liver cells to immune cells were somewhat narrow and muted, suggesting an innate difference in immune cell activities towards HCC cells in comparison with benign liver cells. However, cell-cell interaction analyses showed significant immune evasion by HCC cancer cells. When standard-depth sequencing was performed, significant numbers of genes and pathways that were associated with these changes disappeared. Qualitative differences in some pathways were also found. These results suggest that deep spatial sequencing may help to uncover previously unidentified mechanisms of liver cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa11a255e72a72322fc9f248cef1483a72dd27ef" target='_blank'>
              Deep spatial sequencing revealing differential immune responses in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yan-Ping Yu, Caroline Obert, Baoguo Ren, Marielle Krivit, Kyle Metcalfe, Jia-Jun Liu, Tuval Ben-Yehezkel, Silvia Liu, Jian-Hua Luo
          </td>
          <td>2025-06-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 e12551


 Background:
 Racial and ethnic disparities in breast cancer mortality persist across breast cancer subtypes, and variations in immune response may contribute. Single-cell spatial profiling enables detailed insights into the tumor microenvironment.
 Methods:
 We leveraged a cohort of young, racially and ethnically diverse women enrolled from 1995-2005 in the population-based Northern California Breast Cancer Family Registry (NC-BCFR). We determined feasibility of imaging-based spatial profiling on NC-BCFR TMAs with CosMx 6K SMI single-cell spatial transcriptomics assay, evaluating 140 samples from 88 women. Cells were stained with nuclear and membrane markers, imaged, and segmented with internal algorithms. Cell-level transcriptomic data of 6519 genes were normalized across the TMA and integrated with SCTransform. Manual cell typing was performed using publicly available annotations. Cell type composition for each sample for immune, fibroblast, cancer epithelium, and normal epithelium, normalized to 100 tumor cells/sample, was validated using predictions from matched H&E stained slides annotated with the HoverNet deep learning algorithm. Spatial transcriptomics data were used to predict HER2 and hormone status (HR). The immune microenvironment was examined across race, ethnicity and breast cancer subtype.
 Results:
 Race and ethnicity of 88 NC-BCFR participants were as follows: 16 Black, 22 Asian, 34 Hispanic, and 16 non-Hispanic (NH) White women. Profiles revealed 917,804 cells with mean transcript per cell of 838 (range: 517-1141). Following QC, normalization, and integration, 867,237 cells were typed across diverse phenotypes. With cell type normalized to 100 tumor cells/sample, there was a trend toward NH White samples having increased immune infiltrate vs. other racial groups, within both SMI (NH White 158, Black 116, Hispanic 89, Asian 55.3; ANOVA p = 0.263) and HoverNet (NH White 40.2, Black 24.9, Asian 29.1, Hispanic 18.9; ANOVA p = 0.117). Samples from NH White women had higher B-cell infiltrates (ANOVA p = 0.165) and macrophage content (ANOVA p = 0.000753) compared to cores from Asian, Black, and Hispanic women. Similar trends were observed in NK-cells (ANOVA p = 0.643) and T-cell infiltrates (ANOVA p = 0.318), and tertiary lymphoid structures were identified in one sample.
 Conclusions:
 Single-cell spatial transcriptomics can characterize the tumor immune microenvironment in TMAs from the 1990s, thus leveraging a highly diverse, population-based cohort study with long-term follow-up for novel, clinically relevant discoveries.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e856f98b36026a70f6d6b068a77be4a5031069b" target='_blank'>
              Characterization of breast tumors and the tumor immune landscape in young, diverse participants of the Northern California Breast Cancer Family Registry.
              </a>
            </td>
          <td>
            Marni Blair McClure, Lise Mangiante, Brennan G. Simon, Clemens L. Weiss, Zhicheng Ma, Jocelyn Koo, J. Ransohoff, E. John, Christina Curtis, Jennifer L Caswell-Jin, A. Kurian
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 8539


 Background:
 Immune checkpoint inhibitors have greatly improved outcomes in advanced non-small cell lung cancer (NSCLC). However, patients with EGFR mutant NSCLC have a poor response to immune checkpoint inhibitor therapy. Emerging evidence suggests that an immunosuppressive tumor microenvironment plays an important role in this setting, however, we still lack fundamental knowledge about tumor endothelial cell biology. We hypothesized that oncogene specific changes in the expression of immune-related genes in tumor endothelial cells account for differences in the tumor microenvironment and efficacy of immune checkpoint inhibitor therapy.
 Methods:
 We utilized spatial transcriptomics (GeoMx Digital Spatial Profiling) on resected tumor tissue of EGFR mutant (n = 5) and KRAS mutant (n = 5) NSCLC patients to investigate the transcriptional signature of tumor, endothelial and stromal cells. Additionally, we used NicheNet to explore intercellular communication between tumor, endothelial, and stromal cells. Immune gene set enrichment analysis scores were calculated using ESTIMATE. Immune cell type proportions were estimated using CIBERSORT.
 Results:
 Using spatial transcriptomics, we dissected the tumor microenvironment into tumor, stromal and endothelial compartments. By analyzing predicted cellular communication, we found that tumor and stromal cells primarily affect an interferon-related gene signature in tumor endothelial cells. Notable differentially expressed interferon-related genes, that were significantly up-regulated in KRAS mutant and down-regulated in EGFR mutant NSCLC patients, included CXCL9, STAT1, WARS1, IRF1 and ICAM1. In the stromal compartment, immune gene set enrichment analysis scores were significantly lower in EGFR mutant than KRAS mutant NSCLC (median, 355 vs. 713, P = 0.026) indicating an immunosuppressive tumor microenvironment. We observed substantial heterogeneity while exploring the cellular landscape of the stromal compartment in EGFR mutant and KRAS mutant NSCLC. Notably, we found a significantly decreased proportion of pro-inflammatory macrophages in the stromal compartment of EGFR compared to KRAS mutant NSCLC (P = 0.006).
 Conclusions:
 We identified distinct interferon-related gene signatures in tumor endothelial cells of patients with EGFR and KRAS mutantNSCLC associated with variations in the cellular composition of the tumor microenvironment. This may provide a better understanding for the development of spatial biomarkers to identify which oncogene-driven NSCLC patients are most likely to benefit from immune checkpoint inhibitor therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3c21a018e08000029baaeb3ba8f18554c44ba0f" target='_blank'>
              Spatial transcriptomic profiling of the tumor microenvironment in EGFR and KRAS mutant non-small cell lung cancer.
              </a>
            </td>
          <td>
            F. J. Bolte, Nick Natale, Samantha DiBenedetto, Margaret Moore, E. Stelow, Richard Hall, D. Gioeli
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have improved patient outcomes in hepatocellular carcinoma (HCC); however, most patients do not experience durable benefit. The non-T cell-inflamed tumor microenvironment, characterized by limited CD8+ T-cell infiltration, reduced dendritic cell function, and low interferon-γ-associated gene expression, is associated with a lower likelihood of response to ICI. To nominate new therapeutic targets for overcoming ICI resistance in HCC, we conducted a large-scale multiomic analysis on 900+human specimens (RNA sequencing (RNA-seq), proteomics) and 31 tumor single-cell (sc) RNA-seq samples, with tissue validation through imaging mass cytometry (IMC) and spatial lipidomics by matrix-assisted laser desorption/ionization (MALDI), with experimental investigation by in vitro CD8+ T-cell recruitment and macrophage polarization functional assays using three-dimensional (3D) co-culture models. We discovered 32 oncogenic pathways associated with immune exclusion, with sterol regulatory element binding protein 1 (SREBP1, encoded by SREBF1) as a hub regulator. scRNA-seq analysis showed that SREBP1 signaling is associated with enriched lipid biogenesis pathways in tumor cells, elevated immunosuppressive markers in macrophages, and diminished CD8+ T-cell infiltration. Integration of IMC and MALDI images revealed distinct lipid species differentially abundant in tumor regions with low versus high CD8+ T cell infiltration. Functional studies in a 3D in vitro tumor-immune co-culture system demonstrated that CRISPR-mediated SREBF1 knockout (KO) in HepG2 cells reduced monocyte recruitment, decreased expression of the protumorigenic CD206 marker in macrophages, and enhanced CD8+ T-cell migration compared with wild-type (WT) (p<0.0001). RNA-seq of SREBF1 KO versus WT tumor cells confirmed suppression of lipid biosynthesis genes. Our findings nominate an atlas of tumor-intrinsic drivers of immune exclusion, particularly SREBP1 via pro-tumorigenic macrophage (M2-like) reprogramming. These pathways may represent novel therapeutic targets to enhance antitumor immunity and deserve further study as targeted therapy candidates to enhance ICI in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e84d6872c644cf55b881ebad30a21a4ff2aaeed3" target='_blank'>
              Multiomics identifies tumor-intrinsic SREBP1 driving immune exclusion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Rebekah E. Dadey, Ruxuan Li, Jake Griner, Jie Chen, Arjun Singh, Brian P Isett, Sarah Newman, Ryan Augustin, Aofei Li, Joseph A Manning, S. Monga, A. Singhi, David A A Geller, Carsten Krieg, Ioannis K. Zervantonakis, J. Luke, R. Bao
          </td>
          <td>2025-06-01</td>
          <td>Journal for ImmunoTherapy of Cancer</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d71cf07bbb7f7d458f445b935e53d33e0b0d78bd" target='_blank'>
              Single-Cell Spatial Mapping of Human Kidney Development Reveals the Critical Role of the Local Microenvironment in Cell Fate Decisions
              </a>
            </td>
          <td>
            Jonathan Levinsohn, Samuel Grindel, Bernhard Dumoulin, Amin Abedini, A. Huang, Rena Levin-Klein, Boaz Weicz, Grace Rabinowitz, Andi M. Bergeson, Eunji Ha, Konstantin A Klötzer, Nancy R. Zhang, Paul M. Titchenell, Mingyao Li, Joo-Seop Park, Laura S. Finn, Kotaro Sasaki, P. Beckerman, Oren Pleniceanu, Alex J. Hughes, Katalin Susztak
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="While cervical cancer (CC) prognosis depends on tumor staging, the spatiotemporal evolution of tumor microenvironment (TME) heterogeneity during metastatic progression remains poorly characterized at single-cell resolution. We employed an integrative multi-omics approach, combining single-cell RNA sequencing (scRNA-seq; n = 11), spatial transcriptomics (ST), and bulk RNA-seq data from the TCGA-CESC cohort (n = 304), to systematically map TME remodeling across CC progression stages. scRNA-seq was performed on primary lesions from patients with localized (n = 3), regional (n = 4), and metastatic (n = 4) diseases, with in-depth analyses focusing on cellular characteristics, cell type composition alterations, functional changes, differentiation trajectories, and cell-cell interaction networks. These findings were further validated using spatial transcriptomics, bulk RNA-seq data, and multiple immunohistochemistry (mIHC) experiments. ScRNA-seq data revealed that the TME of the metastatic group displayed a distinct immunosuppressive phenotype. Three key subclusters closely linked to TME remodeling in this group were identified. Notably, a novel metastasis-associated epithelial subpopulation (Epi0_AGR2), characterized by both epithelial-mesenchymal transition (EMT) and chemokine secretory phenotypes, was discovered. Gene Set Variation Analysis (GSVA) revealed that transforming growth factor β (TGF-β) signaling activation served as its primary transcriptional driver. Additionally, a neutrophil subset with pro-tumor and immunosuppressive properties, as well as a cancer-associated fibroblasts (CAFs) subset that promoted angiogenesis, were enriched in the metastatic group. Cell-cell interaction analysis and spatial mapping further revealed the formation of coordinated Epi0-neutrophil-CAFs niches, which established TGF-β-CXCL1/2/8-OSM/OSMR feedforward loops. Importantly, a computational model derived from the TME metastatic niche signature demonstrated significant prognostic stratification in the TCGA cohort (HR = 2.5179, p = 0.0144). In all, this study provides the first comprehensive delineation of stage-specific TME dynamics in CC, revealing TGF-β-driven cellular cooperativity as a metastatic switch. The joint framework establishes a potential clinically translatable tool for precision prognosis and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac624640fa9f989701717857751510daeec92896" target='_blank'>
              Multi-Omics Analysis Reveals the transforming growth factor-β Signaling-Driven Multicellular Interactions with Prognostic Relevance in Cervical Cancer Progression
              </a>
            </td>
          <td>
            Yuhan Wang, Guang-xu Cao, Huimin Zeng, Yong Zhi, Meng-Xi Xu, Ying Wang, Min Liu, Yetian Ruan, Ka Yu Tse, Qingfeng Zhang, Jinli Gao, Zhiqiang Han, Fang Li
          </td>
          <td>2025-06-20</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Pancreatic adenosquamous cancer (PASC) is an extremely rare subtype of pancreatic cancer characterized by a poorer prognosis and higher likelihood of metastasis compared with the more prevalent pancreatic ductal adenocarcinoma (PDAC). Although genomic changes during PASC tumorigenesis have been documented, the corresponding alterations in the tumor immune microenvironment (TIME) remain inadequately elucidated. Therefore, this study aims to analyze the immune landscape of PASC by employing multiplex immunohistochemistry (mIHC) and digital image analysis. Methods In this study, we analyzed four independent cohorts comprising 120 patients with PASC and 386 patients with PDAC. We employed mIHC to quantify three in situ panels of immuno-oncology-related biomarkers at subcellular resolution. We then used five samples to perform laser capture microdissection, RNA sequencing, and whole-exome sequencing to explore the underlying mechanisms of the compartment-specific immune phenotypes in PASC. Results Our findings revealed a more immunosuppressive TIME in PASC compared with PDAC, characterized by a decreased abundance of T cells. Immune cell types indicative of enhanced immune surveillance, including cytotoxic and memory T cells and antigen-experienced T cells, were present at significantly lower densities in PASC compared with PDAC. Conversely, some immunosuppressive macrophage phenotypes exhibited increased distribution in PASC. Immunosuppressive immune cells (ICs) were abundant, enriched within stromal regions, highly heterogeneous across tumors, and exhibited distinct distributions between squamous cell (SQC) and adenocarcinoma (ADC) compartments in PASC. Notably, the TIME of SQC compartments harbored more exhausted T cells compared with synchronous ADC compartments, indicating attenuated immune surveillance during squamous transformation. Transcriptomic profiling of microdissected SQC and ADC regions revealed immune exhaustion signatures and downregulated T-cell differentiation pathways in SQC compartments, alongside altered antigen presentation machinery and elevated tumor mutational burden, suggesting squamous-specific tumor-associated antigens with potential immunotherapeutic relevance. Beyond differences in IC density, we observed closer spatial proximity of CD45RO+ and PD-1+CD3+CD8+ T cells to tumor cells within 10, 20 and 30 µm ranges in PASC compared with PDAC, with variations by histological subregion. Furthermore, we found distinct expression patterns of the programmed cell death protein-1 (PD-1)/programmed death ligand 1 (PD-L1) and T-cell immunoreceptor with immunoglobulin and the ITIM domain (TIGIT)/CD155 axes in the PASC TIME associated with survival outcomes. Notably, TIGIT+CD8+ T cells and CD155+ CD68+macrophages, along with their proximity to tumor cells, served as independent prognostic indicators. These findings were validated in an independent cohort study. Conclusion Our study advances the understanding of PASC by providing updated insights into its immunoenvironmental features. These findings underscore the potential of targeting immune checkpoint pathways, particularly the TIGIT/CD155 and PD-1/PD-L1 axes, as a therapeutic strategy for PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d936da9d89fc333e1aeb534f40e042c28a5208e" target='_blank'>
              Attenuated immune surveillance during squamous cell transformation of pancreatic adenosquamous cancer defines new therapeutic opportunity for cancer interception
              </a>
            </td>
          <td>
            Xinyuan Chen, Shanyue Sun, Shuofeng Li, Shuangni Yu, Jie Chen, Xian-liang Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Metastatic hormone-naïve prostate cancer (mHNPC) is an infrequent form of this tumor type that is characterized by metastasis at the time of diagnosis and accounts for up to 50% of prostate cancer-related deaths. Despite the extensive characterization of localized and metastatic castration-resistant prostate cancer, the molecular characteristics of mHNPC remain largely unexplored. Results Here, we provide the first extensive transcriptomics characterization of primary tumor specimens from patients with mHNPC. We generate discovery and validation bulk and single-cell RNA-seq datasets and perform integrative computational analysis in combination with experimental studies. Our results provide unprecedented evidence of the distinctive transcriptional profile of mHNPC and identify stroma remodeling as a predominant feature of these tumors. Importantly, we discover a central role for the SRY-box transcription factor 11 (SOX11) in triggering a heterotypic communication that is associated with the acquisition of metastatic properties. Conclusions Our study will constitute an invaluable resource for a profound understanding of mHNPC that can influence patient management. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03623-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bec724fe7758a24264be0a981015a60432a2227" target='_blank'>
              Transcriptional analysis of metastatic hormone-naïve prostate cancer primary tumor biopsies reveals a relevant role for SOX11 in prostate cancer cell dissemination
              </a>
            </td>
          <td>
            Natalia Martín-Martín, Saioa Garcia-Longarte, Jon Corres-Mendizabal, Uxue Lazcano, I. Astobiza, Laura Bozal-Basterra, Nicolas Herranz, Hielke van Splunder, Onintza Carlevaris, Mikel Pujana-Vaquerizo, María Teresa Blasco, Ana M Aransay, A. Rosino, J. Tudela, Daniel Jimenez, Alberto Martinez, Andrei Salca, Aida Santos-Martín, Sofía Rey, Aitziber Ugalde-Olano, David Gonzalo, M. Graupera, R. Gomis, J. Mateo, Miguel Unda, E. González-Billalabeitia, Ana Loizaga-Iriarte, Isabel Mendizabal, Arkaitz Carracedo
          </td>
          <td>2025-06-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT Intra‐tumoral heterogeneity poses a major challenge to treating and managing cancer patients. A characteristic feature of melanoma is its composition of cancer cells with typically heterogeneous content of melanin pigment, the production of which is a hallmark of normal melanocytic differentiation but of poorly understood consequence in melanoma cells, as prospective assessment of pigment heterogeneity in melanoma cells has been experimentally challenging. Here, we describe a novel flow cytometric method for high purity separation of viable melanoma cells based on their melanin content, exploiting the light scattering properties of melanin. By fluorescence‐activated cell sorting, we show that cells with low‐pigment content (LPCs) in melanoma cell lines and patient tumors are usually far more abundant than high‐pigment cells (HPCs) and have substantially increased potentials for colony formation in vitro and tumor formation in vivo. In RNAseq analysis, HPCs showed P53 activation and perturbed cell cycling, whereas LPCs displayed upregulation of MYC‐associated transcription and activated ribosome biogenesis. In proof‐of‐concept studies, the latter was targeted by topoisomerase 2 beta targeting with CX‐5461, which induced senescent HPC phenotypes and irreversible loss of clonogenic activity. These data indicate an ‘inverted pyramid’ hierarchical model of melanoma cell propagation wherein abundant LPCs frequently renew their own malignant potential to propagate disease but also infrequently generate HPCs that spontaneously lose this ability in a manner that might be exploited as an anti‐melanoma strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a7c2a2a2e8b632c604835e92877002fe7954605" target='_blank'>
              Prospective Isolation According to Melanin Pigment Content of Melanoma Cells With Heterogeneous Potentials for Disease Propagation
              </a>
            </td>
          <td>
            Clare Fedele, Gamze Kuser-Abali, Ralph Rossi, Peinan Zhao, Jason Li, M. Ameratunga, Pacman Szeto, Youfang Zhang, Miles C. Andrews, Mark Shackleton
          </td>
          <td>2025-06-30</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) lacks targeted therapies, leading to a poor prognosis. Younger patients with TNBC often present with aggressive disease and exhibit distinct tumor microenvironments (TME). We performed spatial profiling to understand the influence of menopausal status on the immune environment, tumor progression, and therapy response. Methods Eleven treatment-naive TNBC tumors were examined in epithelial and non-epithelial areas using digital spatial profiling to identify differentially expressed markers and pathways. Deconvolution methods were employed to identify immune cell subtypes, and the protein expression of T and B cells was confirmed. The prognostic utility of the identified genes and pathways was validated using the METABRIC and SCAN-B datasets. The expression of target genes was analyzed in the I-SPY 2 trial data to understand their influence on the response to specific therapies. Results Premenopausal tumors formed a distinct cluster characterized by the upregulation of cell activation and antigen presentation pathways. In contrast, T cell checkpoint, cancer antigen, and PI3K-AKT pathways were downregulated. External datasets validated these findings, showing a lower hazard and better prognosis for genes upregulated in premenopausal tumors. An enrichment of CD8 + T cells, endothelial cells, and monocytes was observed, along with increased intratumoral protein expression of CD8, CD4, and CD20. Premenopausal tumors demonstrated better responses to PARP and HSP90 inhibitors but showed lower sensitivity to Pembrolizumab and PI3K-AKT inhibitors compared to postmenopausal tumors in the I-SPY 2 trial data. Conclusion Our study underscores the importance of menopausal status in shaping the TME within TNBC, revealing distinct immune landscapes and therapy responses. These findings highlight the need for larger prospective studies to validate differential treatment strategies in younger patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06786-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c6a6cdaccc22fd9e9450a555636943aa5ed9adb" target='_blank'>
              Spatial profiling reveals unique immune microenvironment in premenopausal triple-negative breast cancer associated with therapy response
              </a>
            </td>
          <td>
            V. P. Nimbalkar, V. Snijesh, S. Rajarajan, C. Anupama, S. Mahalakshmi, Annie Alexander, Deshica Dechamma, Manju Moorthy, Gopalakrishna Ramaswamy, Rakesh Ramesh, B. Srinath, J. Prabhu
          </td>
          <td>2025-07-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf6f030b19fe7349fb879d14d939b39089ac03e9" target='_blank'>
              Treatment Resistant Persister Cells Exploit Macrophage Lipid Metabolism to Sustain Glioblastoma Growth
              </a>
            </td>
          <td>
            Avirup Chakraborty, Changlin Yang, Aryeh Silver, Diana Feier, Guimei Tian, Avinash Pittu, C. V. von Roemeling, Nagheme Thomas, Michael Andrews, Cole Conforti, Olusegun O. Sobanjo, Nyla T. Searl, Miruna Anica, Raquel Mctiernan, Maryam Rahman, Jianping Huang, Jeffrey K Harrison, Duane A Mitchell, Matthew R. Sarkisian, L. Deleyrolle
          </td>
          <td>2025-06-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 e13119


 Background:
 The tumor microenvironment (TME) of primary breast tumors (PTs) and breast-to-liver metastases shows heterogeneous infiltration and interactions between tumor, immune, and stromal cells. Hepatic metastases exhibit a more immunosuppressive TME with fewer cytotoxic T cells (CTLs) relative to PTs, which may contribute to increased tumor growth and resistance to systemic therapies. To investigate the immune cell infiltration and immune-tumor interactions in PT and liver metastases, we utilized whole transcriptome analysis and immunofluorescence-based protein expression as part of spatial TME profiling.
 Methods:
 We identified 13 chemo-naïve PTs and 3 breast-to-liver metastases. Formalin-fixed paraffin-embedded (FFPE) samples were used for hematoxylin and eosin (H&E) staining and spatial transcriptomics (ST) profiling with the 10X Genomics Visium CytAssist and Visium HD platforms. The H&E slides were manually annotated in collaboration with a board-certified pathologist, and ST data were deconvoluted using breast-specific scRNA-sequenced references and canonical marker labeling.
 Results:
 Our analysis of the 13 PTs and 3 breast-to-liver metastases revealed significant heterogeneity in tumor clusters, immune cell infiltration, gene set enrichments, and gene signatures between primary and visceral TMEs. Unsupervised clustering of hormone receptor-positive (HR+) HER2-negative (HER2) samples demonstrated considerable intratumoral and intertumoral variability, with differences in the expression of tumor markers (EPCAM, ER, PR, ERBB2, KRT14, KRT18), hypoxia-related signatures (HIF1a), and pro-angiogenic drivers (VEGF). Breast-to-liver metastases showed higher expression of pathways involved in proliferation, angiogenesis, and anti-apoptosis compared to primary tumors. Hepatic metastases were enriched for HALLMARK gene sets related to proliferation and metabolic pathways, including PI3K/AKT/mTOR, mTORC1 signaling, E2F targets, MYC targets, G2M checkpoint, mitotic spindle, glycolysis, oxidative phosphorylation, and fatty acid metabolism when compared to primary tumors. In contrast, primary tumors were enriched for immune-related pathways (TNF-α, interferon-γ, inflammatory response, IL-6 JAK/STAT3), epithelial-mesenchymal transition (EMT), myogenesis, and angiogenesis gene signatures, which are consistent with their metastatic potential. Cellular profiling of liver lesions revealed higher myeloid suppressor cells (TAMs and MDSCs): CD8
 +
 ratio with shorter cell-cell distances relative to PTs suggesting a key role in immune evasion, tumor progression, and therapeutic resistance.
 Conclusions:
 ST data from primary and metastatic breast cancer samples revealed notable tumor heterogeneity, characterized by highly active metabolic and proliferative pathways, along with a higher abundance of myeloid cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/389d6dee563fb2269c25d3de12654b70850dddf7" target='_blank'>
              Spatial transcriptomics and functional mapping of the suppressive TME in primary breast tumor and breast-to-liver metastases.
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Cheol Park, Matthew Jacobo, Marqus Dela Cruz, Michelle Li, Sabrina Carrel, Elexa Rallos, Michael Press, Evanthia T. Roussos Torres
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) can lead to metastasis and high mortality rates known for its aggressive and invasive properties. Currently, primary treatment options of surgical resection, chemotherapy and radiotherapy have many therapeutic limitations for OSCC patients due to its dynamic evolutionary pathways and the development of resistance to conventional therapies. Moreover, previous studies fail to emphasize the roles of cellular heterogeneity, drug resistance, and evolutionary trajectories in OSCC. This review explores the intricate tumor microenvironment landscape of OSCC, focusing on the cellular heterogeneity, drug resistance, and evolutionary trajectories as well as genetic, epigenetic, and environmental risk factors contributing to the OSCC progression. Tumor heterogeneity arises from environmental exposures (e.g., tobacco, HPV infection, dietary carcinogens) that drive clonal evolution, creating subpopulations of cells with distinct mutational profiles and therapeutic vulnerabilities. Recent advances in in the precision medicine and combination therapy of OSCC paves the way for innovative therapeutic strategies, such as targeting molecular subclones through real-time monitoring and leveraging computational models to predict treatment response. By recognizing tumor heterogeneity as both a driver of therapeutic resistance and a therapeutic target, precision medicine frameworks can integrate environmental risk factor data, molecular profiling, and early detection tools to optimize outcomes. This review underscores the necessity for a multidisciplinary approach to understand and combat the complexity of OSCC, proposing directions for future research to enhance diagnosis and treatment efficacy. Graphical Abstract This review highlights how research into cellular heterogeneity, drug resistance, and evolutionary trajectories in oral squamous cell carcinoma is revolutionizing our understanding and treatment of the disease, emphasizing personalized approaches that could significantly enhance patient care in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/463cb6e3ce6f98a2e93f47745076dd9434c6558f" target='_blank'>
              Oral squamous cell carcinoma: Insights into cellular heterogeneity, drug resistance, and evolutionary trajectories
              </a>
            </td>
          <td>
            Yang Zhou, Liyin Wang, Minghua Liu, Hongfang Jiang, Yan Wu
          </td>
          <td>2025-06-12</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e20030


 Background:
 Tumoral heterogeneity has been widely acknowledged to influence clinical outcome of lung adenocarcinoma (LUAD). This study investigated spatial gene profiling of different pathological subtypes of early-stage invasive LUAD with Stereo-seq transcriptomics technique. The tumor microenvironment (TME) was decoded with cellular and molecular events at perspectives of tumoral activities and immune activities.
 Methods:
 We collected spatial RNA profiles of nine Stereo-seq chips (captured area of 1 x1 cm) from 4 invasive LUAD patients. Pathologist identified histological subtypes including lepidic adenocarcinoma (LP), acinar adenocarcinoma (AC) and other invasive adenocarcinoma like papillary adenocarcinoma (IA) on adjacent H&E images. Spatial data matrix was obtained based on pathological annotation accordingly. Spatially resolved gene expression of each subtypes was obtain based on pathological annotation to conduct high resolution analyses. TME functional units were calculated by analysis of cellular neighborhoods (CNs) based on cell compositions within resolution of bin 100 (window of 50um x 50um). Sub-populations of tumoral cells were stratified by molecular functions (GSEA and GSVA score) and stemness (CytoTRACE).
 Results:
 All pathological subtypes shared the identical 10 heterogeneous CNs and 5 tumoral clusters (TM) with various propotions. Tumor cluster 0 (TM0) and TM3 were identified as the malignant population with significant high stemness and significant high activities of MTORC1 signaling, TGF-BETA signaling, PI3K-ATK signaling and hypoxia. AC and IA were mainly composed of TM0 and TM3. Compared to TM0, TM3 had higher EMT activity. We further identified two immune-related CNs (CN4 and CN8) with significant high GSVA score of Interferon-Alpha response, Interferon-Gamma response and Inflammatory response. Spatial association showed that the lymphoid-cells-dominated CN4 was enriched in LP and IA and nearly absent in AC. Meanwhile, the myeloid-cells-dominated CN8 was enriched in AC. GSEA score showed that immune activity of CN4 was more intense in IA when compared to LP. Notably, TM0 mainly present in CN8, and TM3 mainly present in CN4. Therefore, the LP was speculated to reach an inert and statical status. TME of AC was featured as colocalization of myeloid cells and tumor cells with high stemness. TME of IA was featured as colocalization of lymphoid cells and tumor cells with high EMT activity.
 Conclusions:
 This is the first study to delineate the homogeneity of tumor cells and heterogeneity of immune cells among different pathological types of NSCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344f597f0ff9f98416e718719f81b290a2438fe0" target='_blank'>
              Decoding spatial transcriptomics-based heterogeneity of TME in early-stage invasive pulmonary adenocarcinoma.
              </a>
            </td>
          <td>
            Xin Liu, Jingyi Tian, Z. Shang, C. Xiang, Fei Teng, Jiajun Zhang, Yuchen Han
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a significant global health challenge despite advancements in screening, HPV vaccination, and therapeutic strategies. Tumor heterogeneity, driven by epigenetic modifications, affects immune evasion, metastasis, and treatment response. Cancer-associated fibroblasts (CAFs) play a crucial role in CC progression and therapy resistance. Single-cell sequencing offers new insights but remains underutilized in CC research. This study integrates single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and deconvolution analysis to identify key genes and immunotherapy targets. By constructing a prognostic model and exploring the immune microenvironment, we aim to provide novel insights into CC pathogenesis and potential therapeutic strategies. Methods We utilized scRNA-seq, spatial transcriptomics, deconvolution analysis, and pseudotime trajectory mapping to delineate fibroblast subtypes within the tumor immune microenvironment (TIME) of CC. Functional annotations, differential gene expression profiling, cell–cell communication pathways, and transcription factor networks were systematically analyzed. A prognostic model based on bulk RNA-seq data was constructed and validated through survival analysis, with correlations to immune microenvironment characteristics. Functional experiments investigated the role of SDC1, a critical mediator of fibroblast-tumor crosstalk. Additionally, Fibroblast–tumor cell co-culture systems and functional assays were employed to investigate the paracrine role of SDC1. The CAF MYH11⁺ subpopulation was isolated via fluorescence-activated cell sorting (FACS). Multiplex immunofluorescence and immunohistochemical analyses were performed on both cultured cells and human cervical cancer tissue samples to characterize the spatial distribution and dynamic remodeling of MYH11 during stromal reorganization. Results Six distinct fibroblast subtypes were identified, including the C0 MYH11 + fibroblasts, which exhibited unique roles in stemness maintenance, metabolic activity, and immune regulation. Spatial and functional analyses revealed that the C0 subtype is central to tumor-fibroblast interactions, particularly through the MDK-SDC1 signaling axis. The prognostic model incorporating fibroblast-specific markers demonstrated robust predictive power for patient survival outcomes. Additionally, in vitro SDC1 knockdown significantly inhibited CC cell proliferation, migration, and invasion. Fibroblasts show spatially regulated heterogeneity, with activation markers enriched in the tumor zone and MYH11 highest in normal zones, indicating dynamic stromal remodeling. C0 MYH11 + CAF Promotes Tumor Cell Proliferation, Migration, and Inhibits Apoptosis via Soluble SDC1. Conclusion Our results illustrate, in some ways, the possible immunomodulatory and tumor supporting roles of CAFs in CC TIME and highlight the possibility that the MDK-SDC1 pathway is a promising therapeutic target. This study not only promotes a partially new understanding of temporal heterogeneity in CC, but also provides a possible reference base for the development of new biomarkers and immunotherapy approaches to improve clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03432-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d1387a936b1ef6d7e045f6c7bcff60d568a6f7" target='_blank'>
              Deciphering the tumor immune microenvironment: single-cell and spatial transcriptomic insights into cervical cancer fibroblasts
              </a>
            </td>
          <td>
            Zhiheng Lin, Youwei Zhou, Zhenran Liu, Wenyang Nie, Hengjie Cao, Shengnan Li, Xuanling Li, Lijun Zhu, Guangyao Lin, Yanyu Ding, Yi Jiang, Zuxi Gu, Lianwei Xu, Zhijie Zhao, Hua-Bao Cai
          </td>
          <td>2025-07-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9faa1273d62a3ded4dd4ed42a59e3c9f4c050ab5" target='_blank'>
              Spatial omics technology potentially promotes the progress of tumor immunotherapy.
              </a>
            </td>
          <td>
            Zhen Lan, Yuanyuan Yang, Lingling Li, Chaoguan Wang, Zhenqiang Sun, Qiming Wang, Yang Liu
          </td>
          <td>2025-06-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Globally, gastric cancer (GC) stands as the fifth most prevalent form of malignant neoplasm and represents a significant contributor to mortality associated with oncological conditions. Despite advancements in therapeutic strategies for GC, the outcomes for patients with advanced stages of the disease continue to be unfavorable, largely due to tumor heterogeneity and the challenges posed by resistance to therapeutic agents. Metabolic reprogramming is pivotal in driving the advancement of GC, contributing to the development of resistance to pharmacological treatments and facilitating the cancer’s ability to evade immune surveillance. Developing multi-target comprehensive treatment strategies by integrating tumor microenvironment (TME) modulation holds promise for significantly improving therapeutic efficacy.The study analyzed GC and identified key cell subtypes by integrating data derived from single-cell RNA-sequencing (scRNA-seq) alongside spatial transcriptomics information. Cell type identification was accomplished using the tool of Seurat, and the spatial distribution of cell types was revealed through the Robust Cell Type Decomposition technique. CellChat was used to analyze the interactions between key cell subtypes and other cells, and the “StLearn” package was employed to investigate spatial cell communication in depth. Additionally, the functional role of the key molecule ELK4 was validated through in vitro experiments.This research utilized scRNA-seq combined with spatial transcriptomics to comprehensively analyze GC, identifying the C1 NDUFAB1+ subtype, which exhibited high proliferative activity, metabolic reprogramming capabilities, and immune evasion properties. It was found that the C1 NDUFAB1+ subtype closely interacted with fibroblasts and pericytes via the PARs signaling pathway. Additionally, in vitro experiments confirmed that knockdown of ELK4 substantially curbed tumor cell proliferation, migration, and invasion.This study revealed the main significance of the C1 NDUFAB1+ subtype in GC, elucidating its core mechanisms in tumor progression, metabolic reprogramming, and immune evasion. ELK4 was identified as a key regulatory factor that markedly enhanced the proliferation, migratory capacity, and invasive potential of tumor cells, while changes in the TME were a driving force behind immune suppression and drug resistance. The findings underscored the importance of developing specific therapeutic targets, targeting metabolic reprogramming, and overcoming immune evasion, providing new theoretical foundations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94360dc80f5310630bd93066b812e27d59cfa9fb" target='_blank'>
              Integrative single-cell and spatial transcriptomics uncover ELK4-mediated mechanisms in NDUFAB1+ tumor cells driving gastric cancer progression, metabolic reprogramming, and immune evasion
              </a>
            </td>
          <td>
            Yuwei Sun, Wenyang Nie, Zhikai Xiahou, Xiaojing Wang, Wenjia Liu, Zongkai Liu, Zhiheng Lin, Zhaidong Liu
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Background Pancreatic angiosarcoma is a rare and highly aggressive tumor originating from lymphatic or vascular endothelial cells, with poor prognosis and few effective treatments. In this study, we aimed to characterize the tumor ecosystem of metastatic pancreatic angiosarcoma, along with its potential treatment strategies. Methods Single-cell RNA-sequencing and bioinformatics analysis were performed on samples obtained from one patient, including at total of 16,841 cells from pancreatic angiosarcoma liver metastasis and adjacent normal liver tissue. Results Pancreatic angiosarcoma cells exhibited marked upregulation of nuclear factor kappa-B (NF-κB), hypoxia-inducible factor 1 (HIF-1), and myelocytomatosis oncogene (MYC) proto-oncogene signaling pathways, while presenting limited upregulation of actionable therapeutic targets except for cyclin-dependent kinase 4 (CDK4) and epidermal growth factor receptor. Several immune checkpoint genes, including cytotoxic T-lymphocyte-associated protein 4 (CTLA4), lymphocyte-activation gene 3 (LAG3), programmed cell death protein 1 (PDCD1), and cluster of differentiation 86 (CD86), were upregulated in tumor-infiltrating T cells, natural killer (NK) cells, and myeloid cells. Furthermore, intercellular interaction profiling demonstrated enhanced activity of the programmed death-ligand 1 (PD-L1) and CD86 signaling pathways within the tumor microenvironment. The gene-set scores of T/NK-cell exhaustion, regulatory T cell, and macrophage angiogenesis were significantly higher in tumor tissues compared with adjacent normal tissues. However, the phagocytosis scores of macrophages within the tumor-infiltrating region were significantly lower than those in the adjacent normal tissues. Conclusions Our findings outlined an immunosuppressive and angiogenic tumor ecosystem in pancreatic angiosarcoma liver metastasis, suggesting that pancreatic angiosarcoma may be insensitive to most targeted therapies. Conversely, immunotherapies targeting LAG3, PD-L1, and CD86 (e.g. isatuximab, Opdualag, and abatacept) and anti-angiogenic agents may be therapeutically effective and worthy of subsequent exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fffbe9dc01181c5c05c578e4d7eb55a7b801a4e" target='_blank'>
              Single-cell transcriptomic landscape indicates the potential role of immunotherapy in metastatic pancreatic angiosarcoma
              </a>
            </td>
          <td>
            Yizhen Yang, Luohai Chen, Man Liu, Xiaoxuan Lin, Sui Peng, Yubin Xie, Zhirong Zeng, Minhu Chen, Ning Zhang
          </td>
          <td>2025-06-16</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell RNA sequencing captures static snapshots of gene expression but lacks the ability to track continuous gene expression dynamics over time. To overcome this limitation, we developed PROFET (Particle-based Reconstruction Of generative Force-matched Expression Trajectories), a computational framework that reconstructs continuous, nonlinear single-cell gene expression trajectories from sparsely sampled scRNA-seq data. PROFET first generates particle flows between time-stamped samples using a novel Lipschitz-regularized gradient flow approach and then learns a global vector field for trajectory reconstruction using neural force-matching. The framework was developed using synthetic data simulating cell state transitions and subsequently validated on both mouse and human in vitro datasets. We then deployed PROFET to investigate heterogeneity in treatment responses to palbociclib, a CDK4/6 inhibitor, in hormone receptor positive breast cancer. By comparing newly generated scRNA-seq data from a palbociclib-resistant breast cancer cell line with published patient-derived datasets, we identified a subpopulation of patient cells exhibiting profound phenotypic shifts in response to treatment, along with surface markers uniquely enriched in those cells. By recovering temporal information from static snapshots, PROFET enables inference of continuous single-cell expression trajectories, providing a powerful tool for dissecting the heterogeneity of cell state transitions in treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3148960cea2da72c3e81995d56cd3f9d3682ff6f" target='_blank'>
              PROFET Predicts Continuous Gene Expression Dynamics from scRNA-seq Data to Elucidate Heterogeneity of Cancer Treatment Responses
              </a>
            </td>
          <td>
            Yu-Chen Cheng, Hyemin Gu, Thomas O McDonald, Wenbo Wu, Shubham Tripathi, Cristina Guarducci, Douglas Russo, Daniel L. Abravanel, Madeline Bailey, Yue Wang, Yun Zhang, Yannis Pantazis, Herbert Levine, R. Jeselsohn, M. Katsoulakis, Franziska Michor
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Single-cell profiling protein-chromatin interactions is often constrained by complex workflows, high cost, or dependence on specialized equipment. We present indexed tagmentation-based single-cell CUT&Tag-sequencing (IT-scC&T-seq), a modular, plate-based strategy using three-round combinatorial barcoding. IT-scC&T-seq robustly profiles histone modifications and transcription factors with high specificity and throughput, supporting simultaneous analysis of multiple samples and epitopes. Notably, it enables sensitive single-cell mapping of lamina-associated domains, low-abundance chromatin features previously difficult to resolve. Applied to adult mouse mammary gland, the method reveals cell-type-specific chromatin landscapes and lineage-regulatory dynamics. Together, IT-scC&T-seq provides a scalable, cost-effective, and broadly accessible approach for high-resolution chromatin profiling. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03661-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90a49286d7b1c515790f9221aabd24cbf47362e" target='_blank'>
              IT-scC&T-seq streamlines scalable, parallel profiling of protein–DNA interactions in single cells
              </a>
            </td>
          <td>
            Jingchun Ma, Wei Jin, Li Rong, Zhanyu Gao, Zaman Hazrat, Hosen Md Shakhawat, Fei Long, Zixuan Zhang, Jiandong Huang, Xiaomei Lu, Guoxiang Jin, Zhongjun Zhou
          </td>
          <td>2025-07-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d66d1c34039c0624a67bfae6ffcc0194c9308ee" target='_blank'>
              Bioelectric and Epigenetic Landscapes in Lateralized Breast Tumors Reveal Distinct Tumor Microenvironment Signatures
              </a>
            </td>
          <td>
            Sebastián Real, Sergio R. Laurito, Pablo Gonzalez, Oscar Bello, Joao Carvalho, María Roqué
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Ductal carcinoma in situ is a prevalent non-invasive form of breast cancer that, while confined to the milk ducts, carries the potential to progress to invasive disease. With increased detection due to advanced imaging and widespread screening, there is growing concern about the ability to accurately predict which cases are likely to become aggressive. This review aims to deepen the understanding of the biological mechanisms that influence DCIS progression, focusing on the roles of the tumor microenvironment, immune response, and molecular alterations. By synthesizing the contribution of myoepithelial cells, tumor-infiltrating lymphocytes, and key molecular markers, the review seeks to enhance risk stratification methods. These insights could support the development of more precise, personalized treatment strategies that effectively prevent cancer progression while minimizing unnecessary interventions in patients with low-risk DCIS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76422452bac8ca5523021cf11ee79191c7cd5786" target='_blank'>
              DCIS Progression and the Tumor Microenvironment: Molecular Insights and Prognostic Challenges
              </a>
            </td>
          <td>
            Karolina Prajzendanc
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Neuroblastoma, the most common pediatric extracranial solid tumor, has heterogeneous clinical outcomes ranging from malignant progression to spontaneous regression. With the highest frequency of the elusive spontaneous regression, low-risk INSS Stage 4S neuroblastoma represents an ideal model for mechanistic investigation. Spontaneous regression is often accompanied by tumor differentiation, but the mechanisms underlying this process remain largely unclear. Methods Single-nucleus transcriptomics (snRNA-seq) data of neuroblastoma samples were obtained from the Synapse repository to investigate the composition of heterogeneous tumor cell clusters. The feature of the Stage 4S-specific tumor cell subpopulation was revealed through differential expression analysis, pathway enrichment analysis and pseudotime analysis, followed by clinical significance validation on public cohort datasets. The biological function of secreted SLIT3 was validated using multiple in vitro models, including recombinant protein treatment, conditioned medium treatment, and cell lines coculture, to confirm the intratumoral crosstalk effect. Orthotopic and subcutaneous xenograft models were established to verify SLIT3's in vivo function. Cellular bulk RNA-seq analysis was performed with or without SLIT3 recombinant protein treatment to discover the downstream pathways activated by SLIT3, followed by validation with specific pathway inhibitors. Results Analysis of snRNA-seq revealed a distinct subpopulation of tumor cells within INSS Stage 4S neuroblastoma, characterized by a spontaneous regression-like program progressing toward differentiation. Activated SLIT-ROBO signaling was found in the Stage 4S-specific tumor cell subpopulation, which strongly correlated with favorable prognosis. Further investigation into the secreted ligands in SLIT-ROBO related pathways revealed that SLIT3 displayed the most potent enrichment in Stage 4S tumors and the strongest differentiation-inducing effect. In vitro experiments using recombinant SLIT3 protein, conditioned medium, and cell lines coculture consistently demonstrated the capacity of SLIT3 to induce neuroblastoma cell differentiation via intratumoral crosstalk, as evidenced by increased neurite outgrowth and elevated expression of neuronal differentiation markers. Both orthotopic xenograft and subcutaneous xenograft models demonstrated that SLIT3 expression suppressed tumor growth, leading to in vivo tumor differentiation. Mechanistically, PLCβ/PKC signaling mediates the SLIT3-induced neuroblastoma cell differentiation. Conclusions Stage 4S-specific tumor cell subpopulation exhibits a spontaneous regression-like program, from which SLIT3 mediates intratumoral crosstalk and promotes neuroblastoma differentiation via PLCβ/PKC signaling. These findings provide new insights into the mechanism of spontaneous regression in neuroblastoma and offer novel therapeutic targets for differentiation-based treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06621-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc7b67c542baf51616516c8722d1f59d34fe27f7" target='_blank'>
              SLIT3-mediated intratumoral crosstalk induces neuroblastoma differentiation via a spontaneous regression-like program
              </a>
            </td>
          <td>
            Meiling Liu, Dekang Lv, Wenjing Yan, Yi Wu, Shulan Wang, Luoxuan Wang, Jie Lei, Deshun Zeng, Zifeng Wang, Fang Liu, Bing Deng, Quentin Liu, Bin He, M. Yan
          </td>
          <td>2025-05-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background An increasing number of studies have revealed a link between lactylation and tumor initiation and progression. However, the specific impact of lactylation on inter-patient heterogeneity and recurrence in glioblastoma (GBM) remains to be further elucidated. Methods We employed functional enrichment algorithms, including AUCell and UCell, to assess lactylation activity in GBM cancer cells. Additionally, we introduced the interquartile range (IQR) method based on a set of lactylation-related genes (LRGs) to reevaluate the extent of lactylation production within the cancer population at the single-cell resolution. By reconstructing the spatial transcriptomics of hematoxylin and eosin (HE)-stained sections, we further evaluated the lactylation activity in GBM tissues. Subsequently, We employed machine learning algorithms to identify hub genes significantly associated with elevated lactylation levels in GBM. Finally, we experimentally validated the emulsification efficiency and quantified the expression levels of hub genes in human GBM samples. Results Our study innovatively demonstrated a markedly elevated global lactylation level in GBM and validated it as an independent prognostic factor for GBM. We established a prognostic gene model associated with emulsification in GBM. Furthermore, the machine learning-based model identified SSBP1, RPA3 and TUBB2A as potential biomarkers for GBM. Notably, the expression levels of these three hub genes and the lactylation level of TUBB2A in GBM tissues were significantly higher compared to those in normal tissues. Conclusions We propose and validate a IQR lactylation screening method that provides potential insights for GBM therapy and an effective framework for developing gene screening models applicable to other diseases and pathogenic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e8af78ff76bb35a40d5b43f5690784cc5510874" target='_blank'>
              Integrated analysis of single-cell RNA-seq and spatial transcriptomics to identify the lactylation-related protein TUBB2A as a potential biomarker for glioblastoma in cancer cells by machine learning
              </a>
            </td>
          <td>
            Yifan Xu, Chonghui Zhang, Jinpeng Wu, Pin Guo, Nan Jiang, Chao Wang, Yugong Feng
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac106c26ebd96b1e15958a8ddb1a880a0c7c51a9" target='_blank'>
              Intratumoral Heterogeneity of Vimentin Modulates Nuclear Mechanotransduction, DNA Damage Response and Cancer Cell Survival
              </a>
            </td>
          <td>
            Infante Elvira, Terriac Emmanuel, Gelin Matthieu, Siegfried Hugo, Pereira David, Roca Vanessa, Varet Hugo, Khalilian Sara, Rietveld Reinier, Asnacios Atef, van Bodegraven Emma J, Etienne-Manneville Sandrine
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Breast cancer progression and metastasis involve the action of multiple transcription factors in tumors and in the cells of the tumor microenvironment (TME) and understanding how these transcription factors are coordinated can guide novel therapeutic strategies. Myocardin-related transcription factors A and B (MRTFA/B also known as MKL1/2) are two related transcription factors that redundantly control cancer cell invasion and metastasis in mouse models of breast cancer, but their roles in human cancer are incompletely understood. Here, we investigated the expression and activation of these transcription factors to better assess their tumorigenic and metastatic impact on breast cancer and cells of the tumor microenvironment. Methods We used a multiplexed immunofluorescence approach to label MRTFA, MRTFB, tumor cells by using pan Cytokeratin, endothelial cells by using CD31, and antigen presenting cells (APCs) by using HLA-DRA on two different breast cancer tissue microarrays (TMA): The breast cancer progression TMA provided by the Cooperative Human Tissue Network (CHTN_BrCaProg3) and the University of Illinois Breast Cancer Working Group (TMA BCWG UIC-001-TMA) that included primary tumor and lymph node metastases from patients residing in the West Side and South Side of Chicago. We also used bioinformatics analyses of the TCGA and METABRIC databases and the Broad Institute’s single-cell RNA sequencing portal to investigate MRTFA/B expression patterns in the cells of the tumor microenvironment (TME). Results We found that in human tumors, MRTFA/B are concurrently activated in cancer cells, but they show distinct patterns of expression across different histological subtypes and in the cells of the TME. Importantly, MRTFA expression was elevated in metastatic tumors of African American patients, who disproportionately die from breast cancer. Interestingly, in contrast to publicly available mRNA expression data, MRTFA was similarly expressed across estrogen receptor (ER) positive and negative breast tumors, while MRTFB expression was highest in ER+ breast tumors. Furthermore, MRTFA was specifically expressed in the perivascular antigen-presenting cells (APCs), which has been previously associated with immune suppression and breast cancer progression. We also found that MRTFA expression correlated with the expression of the immune checkpoint protein V-set immunoregulatory receptor (VSIR) in the TCGA data and found that MRTFA activity promotes VSIR expression in THP-1 monocytes and cultured HEK293 cells. Conclusions Our results provide unique insights into how MRTFA and MRTFB promote metastasis in human cancer, the differences of their expression patterns, and their immune suppressive function within the breast cancer TME. Our results will guide future studies on targeting MRTFA/B transcriptional activity and the resulting immune suppression in breast cancer. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06559-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9b888786c13a4f36a60caad28e0f780e4db8c77" target='_blank'>
              Multiplex imaging reveals novel patterns of MRTFA/B activation in the breast cancer microenvironment
              </a>
            </td>
          <td>
            Stephanie M. Wilk, Kihak Lee, Caitlyn C. Castillo, Mohamed Haloul, Alexa M. Gajda, Virgilia Macias, Elizabeth L. Wiley, Zhengjia Chen, Xinyi Liu, Xiaowei Wang, Maria Sverdlov, Kent F Hoskins, E. E. Er
          </td>
          <td>2025-05-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2fc80562262fc23fc64cea425e831c1c998be8" target='_blank'>
              Functional Spatial Mapping of the Tumour Immune Microenvironment In Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Sam Legg, Emma Wagner, Christopher J. Applebee, Amanda R. Kirane, Julian Padget, Banafshé Larijani
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Salivary carcinoma ex pleomorphic adenoma (CXPA) is a carcinoma that develops from benign pleomorphic adenoma, and it is characterized by aggressiveness and poor prognosis. Recent advancements in cancer stem cell (CSC) research have identified CSCs as critical contributors to tumor carcinogenesis due to their unlimited self-renewal ability. Single-cell RNA sequencing has enabled a comprehensive analysis of CSC heterogeneity. However, CSCs in CXPA remain largely unexplored. In this study, we constructed a comprehensive atlas of CXPA at single-cell resolution and demonstrated that CXPA exhibited intratumoral heterogeneity. We identified four tumor epithelial cell subgroups characterized by epithelial development, cell cycle, extracellular matrix remodeling, and salivary secretion signatures. ECT2+ tumor cells in the cell cycle cluster were identified as CSCs in CXPA. Analysis of tumor tissues from patients with CXPA revealed that ECT2 expression was strongly associated with invasiveness and prognosis. Targeting the ECT2 gene significantly inhibited tumor sphere formation and aggressiveness in CXPA cell lines. Our findings provide new insights into CXPA diversity and origin, with substantial clinical implications for targeting the ECT2 gene in CXPA treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061f8c8957e80cfaf42fd19577841c361d71fc6b" target='_blank'>
              ECT2+ cell group acts as cancer stem cell in malignant pleomorphic adenoma
              </a>
            </td>
          <td>
            Wan-qun Chen, Yang Wang, Zhengduo Yao, T. Gu, Yuhua Hu, Nannan Leng, Chunye Zhang, Zhen Tian, Jiang Li
          </td>
          <td>2025-06-17</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Introduction Wilms tumor (WT) is the most common pediatric kidney cancer, which presents significant therapeutic challenges, particularly in high-risk cases, due to chemotherapy resistance and immunosuppressive tumor microenvironments (TMEs). Tumor stemness and immune evasion mechanisms are implicated in poor clinical outcomes, yet the molecular drivers underpinning these processes remain inadequately understood. Methods We employed an integrative approach combining single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, bulk RNA-seq, and advanced machine learning techniques to uncover molecular regulators of tumor behavior in WT. A novel Cancer Stemness Prognostic Index (CSPI) was developed using machine learning algorithms to stratify WT patients by risk and histological subtype. Additionally, molecular docking simulations and in vitro functional assays were performed to validate the role of key regulators in tumor stemness and immune evasion, as well as to explore potential therapeutic strategies targeting these molecular drivers. Results Renin gene (REN) emerged as a central regulator of tumor stemness and immune evasion in WT. High-CSPI tumors exhibited enhanced tumor stemness phenotypes, metabolic reprogramming (ROS/oxidative phosphorylation), and suppressed immune activity. Spatial transcriptomics revealed distinct histological subtype-specific localization of stemness-related gene expression and physical proximity between REN-expressing tumor cells and natural killer (NK) cells. At spatial and single-cell resolution, REN-expressing tumor cells promoted NK cell exhaustion via PTN-NCL and COL4A1-CD44 ligand-receptor interactions, while showing limited impact on T cell dysfunction. Molecular docking identified estrogen-based compounds as potential REN inhibitors. Functional assays validated REN knockdown as significantly impairing tumor proliferation, migration, and survival in vitro. Discussion This study establishes REN as a pivotal driver of tumor stemness and immune evasion in WT, playing a dual role in promoting tumor aggressiveness and suppressing NK-mediated immune surveillance. Targeting REN offers promising therapeutic opportunities for high-risk WT cases by simultaneously inhibiting tumor progression and restoring immune function. These findings emphasize REN’s potential as a transformative target for precision oncology and underscore the value of integrative transcriptomics in advancing personalized cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/589e2c1601cab4aeb138ef14d07fb196727dd022" target='_blank'>
              Integrated transcriptomics and machine learning reveal REN as a dual regulator of tumor stemness and NK cell evasion in Wilms tumor progression
              </a>
            </td>
          <td>
            Qingfei Cao, Junyi Li, Yunfei Zou, Changwen Xu, Huihui Tang, Mei-jing Chen
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a119955e8e93a780087f38a6e6b1bd83175ea168" target='_blank'>
              GPNMB marks a quiescent cell population in melanoma and promotes metastasis formation.
              </a>
            </td>
          <td>
            F. Lotti, Marine Melixetian, T. Vlachou, M. S. Nobile, Leone Bacciu, M. Malferrari, Nicolò Quaresima, Stefania Rapino, F. Marocchi, Massimo Barberis, C. Soriani, Barbara Gallo, Velia Mollo, Ilaria Ferrarotto, Daniela Bossi, P. Ferrucci, PierGiuseppe Pelicci, L. Luzi, L. Lanfrancone
          </td>
          <td>2025-06-17</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e782303b020428f38f4b87de805bc2e91d348a09" target='_blank'>
              Decoding epithelial-fibroblast interactions in lung adenocarcinoma through single-cell and spatial transcriptomics.
              </a>
            </td>
          <td>
            Jiajin Yang, Qiuping Xu, Yanjun Lu
          </td>
          <td>2025-07-24</td>
          <td>Journal of cancer research and clinical oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The current immunotherapeutic strategies yield limited therapeutic benefits in patients with castration-resistant prostate cancer (CRPC) due to its immunologically “cold” tumor milieu. Pericytes play a pivotal role in facilitating metastatic dissemination and modulating immune response in malignancies. Our investigation is designed to decipher the biological effects of pericytes on CRPC and their interactions with the tumor microenvironment. Methods We leveraged single-cell transcriptomics and immunofluorescence staining to ascertain the presence and spatial distribution of pericytes in prostate cancer (PCa). Subsequently, we thoroughly delineated the phenotypic and functional characteristics of the CRPC-pericytes subpopulation. The clinical and prognostic significance of CRPC-pericytes was assessed by employing several bulk RNA-seq and microarray datasets. Additionally, we examined the association between the abundance of CRPC-pericytes and immunological features in the PCa microenvironment. Furthermore, the RM-1 subcutaneous tumor model was leveraged to assess the synergistic efficacy of platelet-derived growth factor (PDGF) signaling inhibition in conjunction with immunotherapeutic interventions. Results We discerned pericytes according to their marker genes and observed the α-SMA-positive pericytes encircling the vasculature in PCa and adjacent normal tissues. In the CRPC-pericytes subpopulation, a pronounced upregulation of PDGF signaling and angiogenesis was observed, whereas antitumor immunity-related pathways were suppressed. In addition, CRPC-pericytes displayed notably enhanced interactions with endothelial cells, fibroblasts, and myeloid cells, compared to PCa-pericytes. Patients with elevated prevalence of CRPC-pericytes exhibited notably reduced recurrence-free survival and unresponsiveness to immunotherapeutic interventions. Moreover, CRPC-pericytes were positively associated with immunosuppressive properties of the TME. Notably, combinatorial application of PDGFR inhibitor and anti-PD-1 therapy elicited substantial synergistic antitumor effects in murine PCa models. Conclusion Our investigation uncovers a CRPC-pericytes subpopulation implicated in cancer progression and immunosuppression, suggesting that therapies targeting the phenotypic transition of pericytes could act synergistically with immunotherapeutic regimens to improve survival rates in CRPC. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03838-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0a533b75dfdf1ea7f39068eba94e6ca4759cf00" target='_blank'>
              Pericytes in castration-resistant prostate cancer associated with disease progression and immunotherapy response: insights from single-cell analysis
              </a>
            </td>
          <td>
            Yifeng Qiu, Yuhan Wang, Jiahe Liu, Kai Sun, Shigeo Horie, Zhitong Bing, Qi Hou
          </td>
          <td>2025-06-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pediatric low-grade gliomas (pLGG) comprise 35% of all brain tumors. Despite favorable survival, patients experience significant morbidity from disease and treatments. A deeper understanding of pLGG biology is essential to identify novel, more effective, and less toxic therapies. We utilized single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and cytokine analyses to characterize and understand tumor and immune cell heterogeneity of pilocytic astrocytoma (PA) and ganglioglioma (GG). scRNA-seq revealed tumor and immune cells within the tumor microenvironment (TME). Tumor cell subsets include both progenitor and mature cell populations. Immune cells included myeloid and lymphocytic cells. There was a significant difference between the prevalence of two major myeloid subclusters between PA and GG. Bulk and single-cell cytokine analyses evaluated the immune cell signaling cascade with distinct immune phenotypes among tumor samples. KIAA1549-BRAF tumors appeared more immunogenic, secreting higher levels of immune cell activators and chemokines, compared to BRAF V600E tumors. Spatial transcriptomics revealed the differential gene expression of these chemokines and their location within the TME. A multi-pronged analysis demonstrated the complexity of the PA and GG TME and differences between genetic drivers that may influence their response to immunotherapy. Further investigation of immune cell infiltration and tumor-immune interactions is warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dc01e4fe40878b7885262fd79ea7216b56b949" target='_blank'>
              Multi-pronged analysis of pediatric low-grade glioma and ganglioglioma reveals a unique tumor microenvironment associated with BRAF alterations.
              </a>
            </td>
          <td>
            S. Zahedi, Kent Riemondy, Tian Liu, A. Griesinger, A. Donson, April A Apfelbaum, Rui Fu, Julian Grandvallet Contreras, M. Crespo, J. DeSisto, Madeline Groat, Emil Bratbak, Adam L Green, Todd C Hankinson, Michael Handler, Rajeev Vibhakar, Nicholas Willard, Nicholas K Foreman, Tzu Phang, Jean M. Mulcahy Levy
          </td>
          <td>2025-06-30</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background Precursor lesions like polyps and adenomas in the colon commonly precede colorectal cancer (CRC), advancing through the “normal-polyp-adenoma-carcinoma” sequence towards malignancy. Yet, the cellular heterogeneity and molecular mechanisms involved in CRC development remain inadequately characterized. Methods To understand the molecular mechanisms driving the onset and progression of CRC, we conducted a comprehensive analysis of ten clinical colorectal samples representing sequential pathological stages using single-cell RNA sequencing (scRNA-seq). Validation was performed through immunofluorescence and immunohistochemistry analyses in a separate human colorectal tissue cohort. Additional verification was carried out using bioinformatics analyses of public TCGA and GEO datasets. Results Our comprehensive analyses not only reveal the cellular diversity and transcriptomic differences throughout disease progression but also highlight the importance of leveraging ligand–receptor gene expression to distinguish various cell subtypes. Subsequent examination and validation with a larger sample cohort uncover the specific involvement of ligand–receptor genes, transcription factors, immunoglobulin genes, and heat shock genes in regulating immune responses and microenvironment changes during colorectal tumorigenesis. Conclusions Our extensive transcriptome dataset provides valuable insights and acts as a fundamental resource to deepen our understanding of the complex molecular landscape in CRC. This dataset facilitates improved diagnostic accuracy and the creation of more personalized therapeutic approaches. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06785-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e15cc889bc4d3cf181a1a8c15dbc72bc14a7b3" target='_blank'>
              A dynamic molecular landscape in colorectal cancer progression at single-cell resolution
              </a>
            </td>
          <td>
            Jianwen Sheng, Nianshuang Li, Shuai Li, Yuman Ye, Yuanhang Liao, Yupeng Zhang, Huan Chen, Mengxian Tu, Guping Zhong, Chaoliang Xiong, Pan Zheng, Yuting Lei, Zekun He, Xingxing He, Yao Zhang, Lei Wang, Xiaolin Gao, Yin Zhu, Jianping Liu, Huizhen Fan
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Traditional gene expression deconvolution methods assess a limited number of cell types, therefore do not capture the full complexity of the tumor microenvironment (TME). Here, we integrate nine deconvolution tools to assess 79 TME cell types in 10,592 tumors across 33 different cancer types, creating the most comprehensive analysis of the TME. In total, we found 41 patterns of immune infiltration and stroma profiles, identifying heterogeneous yet unique TME portraits for each cancer and several new findings. Our findings indicate that leukocytes play a major role in distinguishing various tumor types, and that a shared immune-rich TME cluster predicts better survival in bladder cancer for luminal and basal squamous subtypes, as well as in melanoma for RAS-hotspot subtypes. Our detailed deconvolution and mutational correlation analyses uncover 35 therapeutic target and candidate response biomarkers hypotheses (including CASP8 and RAS pathway genes).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b1e17d9e760b6c7495c8a6aea0db799ec80899" target='_blank'>
              Pan-cancer immune and stromal deconvolution predicts clinical outcomes and mutation profiles
              </a>
            </td>
          <td>
            B. Bhinder, Verena Friedl, Sunantha Sethuraman, Davide Risso, Kami E. Chiotti, R. J. Mashl, Kyle Ellrott, Jordan A. Lee, Christopher K. Wong, K. Gyan, Aditya Deshpande, Marcin Imielinski, R. Bareja, Josh Stuart, Myron Peto, K. Hoadley, Alexander J. Lazar, A. Cherniack, Jingchun Zhu, S. Cao, Mark Rubin, Wenyi Wang, Oliver F. Bathe, N. Robine, Li-hua Ding, Peter W. Laird, Wanding Zhou, Hui Shen, V. Thorsson, Jen Jen Yeh, Matthew H. Bailey, D. Zhou, X. Peng, M. Goldman, Yongsheng Li, Anil Korkut, Nidhi Sahni, D. N. Hayes, Michael K. A. Mensah, Ina Felau, Anab Kemal, Samantha J. Caesar-Johnson, John A. Demchok, Liming Yang, M. Ferguson, R. Tarnuzzer, Zhining Wang, J. Zenklusen, Adam Alana Andrea Andrew Angela Benjamin Benjamin Brian Margolin Weinstein Sboner Blair Brooks Berman Raph, Adam Margolin, Alana Weinstein, A. Sboner, Andrew Blair, Angela Brooks, Benjamin Berman, Benjamin Raphael, Brian Craft, Dante S. Bortone, David Heimain, David L Gibbs, David Haan, D. Betel, Duncan McColl, Emek Demir, Faeze Brahman, F. Farshidfar, Gaddy Getz, G. Gao, Gordon Robertson, Huy Dinh, Hyo Young Choi, I. Shmulevich, Ioannis Anastopoulous, Jasmine Yang, Jeff Damrauer, Ken Chen, Lauren Sanders, Lee Cooper, Liang-Bo Wang, Matt Reyna, Mike Noble, Mohammed El-Kebir, Molly Zhang, Nicole Yeager, Paul Little, Richard Moffitt, S. Meier, Teresa Swatloski, T. Knijnenburg, Thomas Matthew, Vicky Chen, Vlado Uzunangelov, Xinghua Lu, Yige Wu, Yulia Newton, Zeya Wang, Paul T. Spellman, O. Elemento
          </td>
          <td>2025-07-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="
 2520


 Background:
 Despite the transformational impact of immune checkpoint blockade, many cancer patients do not experience long-term survival. T cells with innate immune signatures can secrete inflammatory cytokines/chemokines and deliver potent cytotoxic signals potentially ideal for tumor immunity. The novel double-stranded RNA sensor RIG-I agonist SLR14 improved the control of murine melanoma. We tested the hypothesis that SLR14 transforms T cells to a cytotoxic state in immunologically “cold” human tumor specimens.
 Methods:
 We developed an approach, called PERCEPT, to directly test the response of patient tumor and immune samples to novel and established therapies
 ex vivo
 using perturbational single-cell RNA sequencing (Table). We obtained 9 surgical resections from primary or metastatic melanoma and Merkel Cell Carcinoma (MCC) tumors and lymph node metastases and made suspension replicates of tumor and infiltrating immune cell co-cultures. We stimulated for 42-48 hours (Table). We Flourescently Activated Cell Sorted live cells and then barcoded for multiplexed single-cell sequencing using 10x scRNAseq. We used CINEMA-OT to identify factors associated with response and resistance to the perturbations tested. We developed and validated CRISPR-KO MCC and small cell lung cancer (SCLC) cell lines, and co-cultured with CD14+ monocytes or monocyte-derived DCs.
 Results:
 Stimulation with RIG-I agonist SLR14 induced expression beyond canonical IFN-stimulated genes in tumor cells, NK cells, and T cells. SLR14 stimulates tumor-infiltrating T cells into antiviral states in tumor-immune co-cultures and primes
 in vitro
 T-cell production of IFNγ. However, MCC immune infiltrate responsiveness to IFN or SLR14 was notably decreased compared to the melanoma samples, and perturbational computational analyses with CINEMA-OT identified the cytokine midkine (MDK) associated with nonresponse in MCC. Knockout of MDK restored response to IFN and SLR14 by MCC and SCLC tumor cell lines, as well as co-cultured CD14+ monocytes or monocyte-derived DCs.
 Conclusions:
 Our approach revealed that midkine, a multifunctional cytokine, suppresses innate immune sensing of IFN and SLR14 in both tumor and immune cells, disrupting the tumor immunity cycle at multiple points. We show that this effect, while comparatively infrequent in melanoma, is pronounced in MCC and SCLC. Our study thus uses a direct assessment of patient tumor and immune samples to identify a novel resistance mechanism enriched in neuroendocrine tumors MCC and SCLC.




 Therapeutic class
 Stimulation
 Target
 Clinical development




 Immune checkpoint inhibitor
 αPD-1
 PD-1
 Standard of care


 Cytokine
 IFNγIFNβ
 IFNGRIFNAR
 Early-phase clinical trials orPre-clinical


 Innate immune agonist
 Poly(I:C)-NTPoly(I:C)-TADU-S100SLR14
 TLR3MDA5/TIG-I/TLR3STINGRIG-I


 Combination
 αPD-1 + IFNβ
 PD-1/IFNAR




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/404b41f4f104c4cf7c170db6748ecf8ec9a5f2f1" target='_blank'>
              Perturbational single-cell RNA sequencing of patient tumors in Merkel cell and small cell lung carcinomas.
              </a>
            </td>
          <td>
            Curtis J. Perry, Alexander Frey, Yuewei Fei, P. Costa, Jason Z Wang, Sina Ghadermarzi, Daniel Levine, Asuka Koda, Amin Nassar, Min Ding, Yunan Nie, Therese Cordero-Dumit, Arnaud Augert, David van Djik, K. Olino, J. Ishizuka
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cd2073df7010b077b52401f0a6cda2f7a7be7e3" target='_blank'>
              Multi-omic analysis of colorectal adenocarcinoma identifies a new subtype of myofibroblastic cancer-associated fibroblast expressing high level of B7-H3 and with poor prognosis value
              </a>
            </td>
          <td>
            Maelle Picard, Arnaud Guille, P. Finetti, Bernadette de Rauglaudre, Nadiya Belfil, Lenaïg Maescam, David Jeremy Birnbaum, F. Bertucci, E. Mamessier
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background The immunological landscape of metabolic dysfunction-associated steatohepatitis (MASH)-driven hepatocellular carcinoma (HCC) is not well understood. Herein, we aim to delineate the immunological landscape in the MASH-to-HCC transition and to identify the critical genes that contribute to the pathogenesis of MASH-related HCC. Methods A well-established MASH-driven HCC mouse model, STAM model, was first constructed. Thereafter, we applied single-cell RNA sequencing (scRNA-seq) analysis of CD45+ cells sorted from livers of mice with normal chow or MASH, as well as paired paracancerous and cancer tissues from mice with HCC. Flow cytometry and multiplexed immunohistochemistry were performed to validate the analysis results of scRNA-seq. Finally, STAM model was applied between apolipoprotein E (ApoE)-deficient mice and wild type controls. Results We identified 23 major clusters corresponding to nine populations among 31,822 cells. Obviously, immunosuppressive and exhausted CD4+ T (IKZF2+OX40+FOXP3+CD4+ and GZMK+LAG-3+PD-1+CD4+), CD8+ T (LY49I+LY49G+IKZF2+FOXP3−CD8+, IKZF2+FOXP3+CD8+ and GZMK+LAG-3+PD-1+CD8+) and γδ T cells (γδ Treg and exhausted γδ T cells) were induced in the MASH-to-HCC transition. As MASH-related HCC progressed, B cells matured and differentiated into immunosuppressive cells. Natural killer cells (NKs) were found to be strikingly reduced at HCC stage. Particularly, the activation of liver-infiltrated NK cells was inhibited, leading to attenuation of anti-tumor capacity in the MASH-to-HCC transition. Moreover, tumor-associated macrophages were increased in MASH-related HCC. Importantly, multiple immune cells highly expressed ApoE in HCC, and ablation of ApoE impeded MASH-driven hepatocarcinogenesis by disrupting both ApoE-PI3K-AKT-NF-κB and ApoE-PI3K-AKT-c-Jun/c-Fos signaling pathways. Conclusions We illustrate the profound reprogramming of the liver immune microenvironment in the MASH-to-HCC transition and clarify the role of ApoE in MASH-driven HCC, implying that ApoE may serve as a potential therapeutic target for MASH-related HCC. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02370-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d33d0cf3ae24c19bf9ea4aeaf77109fca34bff" target='_blank'>
              Single-cell transcriptome reveals the reprogramming of immune microenvironment during the transition from MASH to HCC
              </a>
            </td>
          <td>
            Yu Huang, Ying Xie, Yuqing Zhang, Zhemian Liu, Weihua Jiang, Yingying Ye, Jiale Tang, Zhenhua Li, Zhinan Yin, Xue-Jia Lin
          </td>
          <td>2025-06-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca80c0d5c8118b6afa306ae5816ee0be1c3263a" target='_blank'>
              Single-cell transcriptomics reveals hypoxia-driven iCAF_PLAU is associated with stemness and immunosuppression in anorectal malignant melanoma.
              </a>
            </td>
          <td>
            Hao Zhang, Lin Gan, Xiaofei Duan, Baojing Tuo, Hao Zhang, Senbo Liu, Yugui Lian, Enjie Liu, Zhenqiang Sun
          </td>
          <td>2025-07-04</td>
          <td>Journal of gastroenterology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the most abundant non-cancerous cell type in glioblastoma (GBM) and heavily influence GBM biology, contributing to tumor progression, therapeutic resistance, immune evasion, and neovascularization. Current in vitro models that utilize IL-4/IL-13 stimulation fail to capture the transcriptional and functional heterogeneity of TAMs observed in vivo. In this study, we utilize a serum-free indirect co-culture model with patient-derived xenolines to polarize primary human macrophages and characterize their molecular and functional phenotypes. We demonstrate that xenoline-polarized macrophages diverge from classical M1/M2 states and instead adopt transcriptional signatures reflective of TAM subsets identified from patients. Notably, macrophages polarized with the radiation-therapy selected xenoline, JX14P-RT, exhibited gene expression patterns enriched for interferon response and hypoxia, mirroring recurrent GBM samples. In contrast, JX14P TAMs showed enrichment in phagocytic gene sets. Functional validation of these phenotypes revealed discrepancies between the transcriptionally predicted and observed phenotypes, emphasizing the importance of integrating phenotypic validation in sequencing studies. Altogether, our findings establish xenoline-polarized macrophages as a useful alternative to traditional models that can be used to study immune-interactions in vitro. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02057-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f15f733124e62cedabe127db49de61b0e9f489" target='_blank'>
              Xenoline-polarized macrophages as an alternative in vitro model of tumor-associated macrophages in glioblastoma
              </a>
            </td>
          <td>
            Acta Neuropathologica Communications, Christopher D. Willey, Hasan Alrefai, Benjamin Lin, Amr Elkholy, Manoj Kumar, Taylor L. Schanel, Kevin J. Lee, Patricia H. Hicks, Joshua C. Anderson, Gao Guo, Eun-Young Erin Ahn, Ryan Miller
          </td>
          <td>2025-06-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e17536


 Background:
 High-grade cervical intraepithelial neoplasia (HG-CIN) affects 200,000 U.S. women annually, with rising prevalence among those aged 30 & older, many of whom missed the opportunity for HPV vaccination. Treatment remains predominantly surgical, with a notable absence of pharmacologic interventions. Clinical trial endpoints rely on histological regression, but standard microscopy cannot accurately distinguish between regressing and progressing lesions. Compelling evidence underscores the tumor microenvironment’s (TME) critical role in driving immune suppression and malignant transformation. Leveraging a novel spatial imaging platform, this study provides high-resolution molecular insights into HG-CIN directly from routine H&E-stained slides, enabling precise characterization of immune marker distributions and their implications for disease progression. This technology has the potential to transform tissue analysis and significantly impact drug development.
 Methods:
 Sixteen formalin-fixed paraffin-embedded (FFPE) tissues were obtained from the health system’s Biospecimen Repository, with twelve used to optimize the sequencing protocol directly from H&E slides. Sample was categorized as CIN3, benign, or micro-invasive cervical mucosa based on the history, Pap smear/HPV test results, and an expert review by a Gynecologic Pathologist. Library generation and sequencing were conducted using Visium Gene Expression (both SD & HD platforms) kits for FFPE (10x Genomics). Specific genetic markers for immunosuppressive cell activities were highlighted to demonstrate its differential presence in CIN3 vs normal cervical tissue.
 Results:
 Sixteen FFPE samples were analyzed, with 11 used to optimize sequencing protocols. Three CIN3, three normal, and one micro-invasive cervical cancer sample revealed distinct tumor, stromal, and immune microenvironments. CIN3 tissues had elevated Tregs markers (FOXP3, CD4), MDSCs (CD11b, CD33, HLA-DR), and the Treg trafficking molecules CCR7 and TGF-β within stromal regions. Additionally, CIN3 had high gene expression for CDKN2A, which encodes the immunosuppressive ligand PDL1.
 Conclusions:
 The CIN and cancer microenvironment demonstrated strong immunosuppressive markers across immune, stromal, and epithelial compartments, aligning with key known features of tumorigenesis and progression. This spatial imaging platform & optimized workflow captures biologically relevant data directly from standard routinely stained H&E slide, achieving near single-cell resolution, enabling histopathologic comparison to gene expression data. Our study revealed unique immune suppression profiles in CIN3 linked to HPV progression, highlighting the platform’s potential to transform tissue analysis and advance drug development applications.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65b12713b597d9f017570b06219fcd74b91e4800" target='_blank'>
              Spatial transcriptomics for profiling the microenvironment of cervical precancer progression from routine H&E slides: Applications for pharmaceutical development.
              </a>
            </td>
          <td>
            Mario R Castellanos, C. Regan, Seema Khutti, Kai Wang, M. Dhar, Jonathan Preall
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cell plasticity is a hallmark of cancer, enabling tumor cells to acquire multiple phenotypes responsible for tumor progression, metastasis, and therapy resistance. In this issue of the JCI, Kawai and colleagues leveraged genetically engineered mouse models (GEMM) of pancreatic ductal adenocarcinoma (PDAC) to demonstrate that loss of Pbrm1, a member of the SWI/SNF complex, drives dedifferentiation and aggressive tumor features. Pbrm1 loss activated a program of epithelial-to-mesenchymal transition (EMT) and allowed the emergence of poorly differentiated histologies that are commonly associated with high recurrence rate and dismal prognosis. These findings reveal the role of the SWI/SNF complex during PDAC evolution in maintaining cell identity and restraining the progression of this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51cf4e0b352b84ce0ee22909d350d6fe7531e126" target='_blank'>
              Switching on the evolutionary potential of pancreatic cancer: the tumor suppressor functions of PBRM1
              </a>
            </td>
          <td>
            Luigi Perelli, G. Genovese
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Radiomics is undergoing a paradigm shift from single-omics to multi-omics, from single-temporal to multi-temporal analysis, and from global to subregional analysis. These transformations have shown great potential in addressing key challenges related to imaging changes before and after neoadjuvant chemotherapy (NAC) in breast cancer. Furthermore, radiomics has achieved remarkable progress in tasks such as exploring tumor heterogeneity and uncovering underlying biological mechanisms. Integrating imaging data with gene data offers novel perspectives for understanding imaging changes driven by specific genetic alterations. However, current radiomics studies on neoadjuvant chemotherapy for breast cancer have not yet achieved a close integration of imaging changes with underlying biological mechanisms. They are largely limited to simple associations between models and genomic data, without in-depth interpretation of the biological significance inherent in imaging features, which is essential to directly link these features with the dynamic progression of the disease. This review seeks to explore the spatial-temporal heterogeneity of imaging alterations observed during NAC for breast cancer, while assessing their biological implications using established analytical approaches. It highlights the distinct advantages of spatial-temporal radiomics in predictive model development and examines potential correlations between imaging dynamics and gene expression profiles before and after NAC. Additionally, we critically examines previous radiogenomics studies, providing theoretical insights into their limitations. Finally, the review proposes future directions and innovative approaches for applying spatial-temporal radiogenomics in NAC for breast cancer, serving as a valuable reference and roadmap for researchers and clinical practitioners in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1413564d0b7a7765ec86c504011b8f276395913" target='_blank'>
              Spatial-temporal radiogenomics in predicting neoadjuvant chemotherapy efficacy for breast cancer: a comprehensive review
              </a>
            </td>
          <td>
            Tingfeng Zhang, Liang Zhao, Tingting Cui, Yi Zhou, Peifen Li, Chuqiao Luo, Junkang Wei, Hong Hu
          </td>
          <td>2025-06-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Sub-populations of cancer-associated fibroblast (CAF)-like tumor cells deposit extracellular matrix (ECM) proteins that support Ewing sarcoma (EwS) progression and metastasis. We previously showed a hallmark of CAF-like EwS cells is their hybrid transcriptional state wherein the driver fusion oncogene, EWS::FLI1, maintains activation of proliferative programs but loses capacity to repress mesenchymal genes. Here, we studied primary patient tumors and cell line models to identify molecular drivers of this hybrid state. Our data reveal that hybrid EwS cells are induced and maintained by a TGFβ signaling positive feedback loop. Hybrid cells de-repress TGFBR2 and upregulate expression and secretion of TGFβ2 to sustain pathway activation and ECM deposition. While TGFβ ligands can potently induce growth arrest in cells of epithelial origin, we show that TGFβ1 and TGFβ2 promote cell invasion of EwS cells without affecting proliferation. Thus, stroma and tumor-derived TGFβ ligands induce and maintain hybrid EwS cells to promote pro-metastatic cell phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d55bd04a082dd960e72d401069d912b22df3a0ed" target='_blank'>
              Autocrine TGFβ2 enforces a transcriptionally hybrid cell state in Ewing sarcoma
              </a>
            </td>
          <td>
            Emma D. Wrenn, Jacob C. Harris, April A. Apfelbaum, Jonah R. Valenti, Patricia A. Lipson, Stephanie I. Walter, Nicolas M Garcia, Aya Miyaki, Steven C. Chen, Jim M. Olson, Jason S. Price, Kelly M. Bailey, Elizabeth R. Lawlor
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The advent of the omics era has facilitated the identification of precise biomarkers for cancer progression, revealing a broader diversity of macrophage phenotypes beyond the traditional M1/M2 classification. Folate receptor 2 (FOLR2)-positive macrophages, co-expressing markers such as mannose receptor C-Type 1 (MRC1/CD206) and lymphatic vessel endothelial hyaluronan receptor 1(LYVE1), are an embryonically derived subset typically found around blood vessels in both tumor stroma and normal tissues. Despite FOLR2’s longstanding association with anti-inflammatory, immunosuppressive macrophages in tumors, its precise role in cancer progression remains unclear. Recent studies suggest that FOLR2+ macrophages can either promote or inhibit cancer progression, depending on their multifaceted roles in the tumor microenvironment. This review provides a comprehensive overview of the biological features, functional roles, molecular mechanisms, and therapeutic potential of FOLR2+ macrophages in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a55317e456e1c334fa4e34ea5c90a5105fdddf" target='_blank'>
              FOLR2+ macrophages in cancer: allies or enemies
              </a>
            </td>
          <td>
            Decheng Jiang, Ruiling Xiao, Jialu Bai, Xing Wang, Yuan Fang, Jianzhou Liu, Xiaohan Wang, Ming Cui, Lei You, Yupei Zhao
          </td>
          <td>2025-06-02</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938950b1958bbfaef501d56fa236f86c6e2c6141" target='_blank'>
              Multi-omic analysis of gallbladder cancer identifies distinct tumor microenvironments associated with disease progression.
              </a>
            </td>
          <td>
            Tao Zhou, Yanhong Wu, Shuai Li, Xinyao Qiu, Erdong Liu, Zhihua Xie, Xuebing Shi, Yani Zhang, Guosheng Ma, Wenbo Guo, Xiang Wang, Kaiting Wang, Xiaomeng Yao, Ji Hu, Siyun Shen, Shuai Yang, Xiaoqing Jiang, Jing Fu, Hongyang Wang, Jin Gu, Lei Chen
          </td>
          <td>2025-06-26</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Lung metastasis remains a leading cause of cancer mortality, driven not only by primary tumors but crucially by dynamic interactions within the premetastatic niche (PMN). Emerging evidence reveals that organ‐specific PMN formation precedes tumor cell arrival, creating an immunosuppressive microenvironment conducive to metastatic colonization. Primary tumor‐derived factors interact with lung resident cells, promoting the recruitment of bone marrow‐derived cells and creating a favorable microenvironment for the migrating tumor cells. Despite progress, systematic integration of PMN's multilayered regulatory mechanisms—from mechanism to clinical application remains lacking. The review summarizes recent studies investigating the effects of PMN in the entire process of lung metastasis. Our discussion revolves primarily around extracellular vesicles, extracellular matrix remodeling, proinflammation, immunosuppression, angiogenesis, and metabolic reprogramming. We emphasize the impact of various treatment strategies for primary tumors on the PMN in clinical practice, while also summarizing current diagnostic methods and exploring new therapeutic possibilities in the PMN. Finally, we propose a roadmap for future investigations into the lung PMN, highlighting prioritized research axes that bridge mechanistic discoveries with bench‐to‐bedside translation of PMN‐targeted interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0871bd03da6cd8fd6aae769050e9eb31d43909dc" target='_blank'>
              Targeted Lung Premetastasis Niche: Mechanisms, Strategies, and Application
              </a>
            </td>
          <td>
            Chenghao Cao, Di Lu, Huigang Li, Shen Pan, Jianyong Zhuo, Zuyuan Lin, Chiyu He, Peiru Zhang, Songmin Ying, Xuyong Wei, Shusen Zheng, Zhe Yang, Xiao Xu
          </td>
          <td>2025-06-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 e14555


 Background:
 Natural killer (NK) cells are innate lymphocytes that engage in direct cytotoxic responses against malignant cells and promote pro-inflammatory tumor microenvironments. While studies have predominantly focused on conventional NK cell subsets, recent findings uncovered adaptive and tissue-resident (TR) subsets with unique functional properties. The influence of distinct NK cell subsets on tumor immunobiology and clinical outcomes remains unknown.
 Methods:
 Single-cell RNA-sequencing (scRNA-seq) of primary, therapy-naïve tumors from 276 patients across 14 cancer types, as well as 39 patients treated with immune checkpoint blockade (ICB), was accessed through publicly available datasets. We employed systematic computational analyses to phenotype and examine intra-tumoral NK cells. Gene signatures for NK cell subsets were applied to bulk RNA-sequencing from 212,105 tumor samples to delineate their associations with clinical outcomes using the Caris CODEai database.
 Results:
 Six main subsets of NK cells were identified in solid tumors, including conventional CD56
 dim
 and CD56
 bright
 NK cells, alongside adaptive, TR adaptive, TR CD56
 bright
 , and TR CD56
 bright
 /adaptive subsets. Across cancer, TR adaptive NK cells emerged as the most prognostic subset, with high infiltration conferring significantly longer overall survival (HR[95% CI] = 0.68[0.66-0.69], p < 0.001) than conventional CD56
 dim
 and CD56
 bright
 subsets. This was also observed in cohorts of head and neck, non-small cell lung, and esophageal cancers, particularly those treated with ICB. Notably, elevated TR adaptive NK cell infiltration was more prognostic than high tumor mutational burden in these settings. This was most apparent in ICB-treated head and neck cancers (HR = 0.48[0.41-0.57], p < 0.001 vs HR = 0.90[0.76-1.04], p = 0.16). Additionally, TR adaptive NK cells were strongly correlated to the infiltration of CD8
 +
 T cells and B cells in tumors. Underlying these observations, our scRNA-seq analyses demonstrated that TR adaptive NK cells exhibit the highest immunomodulatory activity of all subsets, expressing high levels of pro-inflammatory cytokines (
 CCL3
 ,
 CXCL13
 , and
 IFNG
 ) and low levels of immunosuppressive cytokines (
 TGFB1
 ). Mechanistically, this activity was linked to the upregulation of transcription factors associated with NK cell activation, function, and persistence and downregulation of genes related to stress response and exhaustion. Interestingly, scRNA-seq from paired pre- and post-ICB-treated tumors revealed a unique expansion of cytotoxic
 GZMB
 + TR adaptive NK cells after treatment.
 Conclusions:
 TR adaptive NK cells represent a novel subset of NK cells that may drive anti-tumor immune responses, particularly in the context of ICB treatment. Features of this unique population may be harnessed to design more effective NK cell-based therapies against solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbcf1a77672e37211fd7cd09d5cd12367404c3a" target='_blank'>
              Identification of tissue-resident adaptive NK cells and their association with reduced mortality across diverse solid tumors.
              </a>
            </td>
          <td>
            John R Lozada, A. Ali, Christine Luo, Rosemary N. Plagens, A. Elliott, Ella Boytim, Nicholas A. Zorko, W. El-Deiry, Elisabeth I. Heath, Akash Patnaik, A. Vanderwalde, Jeffrey S Miller, E. Antonarakis, Frank Cichocki, Justin H Hwang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background The heterogeneity of colorectal cancer (CRC) and its complex immune microenvironment pose significant challenges for treatment. Understanding the cellular composition and dynamic changes is essential for uncovering mechanisms of tumour progression. Methods To investigate the cellular heterogeneity and immune microenvironment of CRC, identifying critical subpopulations, functional pathways, and prognostic biomarkers, single-cell transcriptomic data from 41 CRC samples across four datasets were integrated. Bioinformatic analyses identified cellular subpopulations, cell communication networks, and prognostic biomarkers. The expression patterns, clinical significance and biological function of TUBB were validated in vitro. Results A distinct epithelial subpopulation with proliferative and invasive features was identified, promoting tumour progression by resisting apoptosis and remodelling the extracellular matrix. ActMono, a terminal state of myeloid cells, was enriched in tumours and linked to disease progression. Cell communication analysis highlighted galectin signalling in immune regulation. A prognostic model (CRS) based on secretory immune cell-related genes identified TUBB as a key molecule influencing the cell cycle and extracellular matrix remodelling, with its expression patterns, clinical significance and biological effects validated in vitro. Conclusion This study reveals critical subpopulations, signalling pathways, and biomarkers in CRC, providing insights into tumour progression and potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e4b3d5d75559211564431db384708f82801552" target='_blank'>
              Single-cell sequencing reveals dysregulated cell type perturbations and critical mediator communication remodelling in colorectal cancer
              </a>
            </td>
          <td>
            Chengyuan Xu, Siqi Zhang, Zhouyu Zhang, Luoqiu Zhang, Bin Sun, Zicheng Yu, Hailong Liu
          </td>
          <td>2025-06-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) is promising in predicting the efficacy of immune checkpoint inhibitor (ICI) therapies. But it is challenging to determine the level of circulating EVs due to their variations in spatial and temporal distribution. To address this, we developed an in situ EV detection platform integrating multiplex EV capture with microfluidic-generated immune-tumor spheroids. This platform enables in situ monitoring of EV secretion dynamics under ICI and chemotherapeutic treatments, capturing localized and temporal changes in EV release. Using predictive models, we identified EVs carrying programmed cell death ligand 1 (PD-L1) as the most robust predictors of spheroid viability during treatment. RNA sequencing further revealed that dynamic EV expression changes are driven by gene transcription, providing a temporal understanding of EV regulation. Our platform overcomes the limitations of traditional methods by offering a physiologically relevant system to study EV-mediated immune responses. By addressing the spatial and temporal heterogeneity of EVs, this work advances EV-based biomarker discovery and provides a foundation for optimizing personalized immunotherapies. Graphic abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12951-025-03467-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2076c57ae0303bbae149416c773be905fb6b4e7" target='_blank'>
              In situ and dynamic screening of extracellular vesicles as predictive biomarkers in immune-checkpoint inhibitor therapies
              </a>
            </td>
          <td>
            Yihe Wang, Yue Sun, Mengqi Liu, Chao Wang, Miao Huang, Jiaoyan Qiu, Ningkai Yang, Yu Zhang, Hong Liu, Lin Han
          </td>
          <td>2025-06-03</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction The pronounced heterogeneity of colorectal cancer (CRC) significantly impacts patient prognosis and therapeutic response, making elucidation of its molecular mechanisms critical for developing precision treatment strategies. This study aimed to systematically characterize tumor cell heterogeneity and explore its clinical implications. Methods Five single-cell RNA sequencing cohorts were integrated (comprising 70 CRC samples and 164,173 cells) to systematically analyze tumor cell heterogeneity. Unsupervised clustering analysis based on VEGFR+ tumor cell signature genes was used to stratify CRC patients. Key molecular mechanisms were validated through in vitro cellular experiments, in vivo animal models, molecular docking, and dynamics simulations. Results The analysis successfully identified five distinct tumor cell subtypes, with the VEGFR+ subtype exhibiting marked epithelial-mesenchymal transition (EMT) activation signatures and strong association with metastasis and poor clinical outcomes. Based on VEGFR+ signature genes, CRC patients were stratified into three subgroups: C1 (metabolically active), C2 (proliferative), and C3 (invasive), with the C3 subtype demonstrating high metastatic potential, stem-like properties, and an immunosuppressive microenvironment, along with a five-year survival rate below 50%. Mechanistic investigations identified HOXC6 as a key driver of the C3 subtype, with HOXC6 knockout significantly suppressing CRC cell proliferation, migration, and invasion. Furthermore, molecular docking revealed that the targeted agent abemaciclib effectively binds HOXC6, with both cellular and animal experiments confirming its ability to inhibit CRC cell functions and significantly reduce tumor burden in nude mice. Discussion This study establishes the first single-cell-resolution molecular classification system for CRC, delineates the mechanistic link between EMT subtypes and metastatic progression, and identifies HOXC6 as a novel therapeutic vulnerability. These findings provide a translational foundation for precision oncology and offer new rationale for precision diagnosis and treatment of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4607a6764846d173d90b47364ed771e5e80f538" target='_blank'>
              Cross-cohort multi-omics analysis identifies novel clusters driven by EMT signatures in colorectal cancer
              </a>
            </td>
          <td>
            Wu Ning, Wenqing Jia, Jingyuan Ning, Lei Zhou, Zongze Li, Lin Zhang, Xin Song
          </td>
          <td>2025-06-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Pancreatic cancer has a 5-year survival rate of 12%, highlighting the need for reliable biomarkers for early detection and disease monitoring. Circulating tumor cells (CTCs) have emerged as a promising biomarker, yet their detection remains challenging. This study evaluates the Parsortix® system, a microfluidic device that enriches CTCs based on size and deformability, using pancreatic cancer cell lines. As increasing evidence indicates that during epithelial-to-mesenchymal transition (EMT) a cell's deformability increases, we evaluated possible biases by the device. The EMT stage of three pancreatic cancer cell lines, CAPAN-1, MIA PaCa-2, and PANC-1, was assessed to classify them as epithelial, mesenchymal-like, and hybrid, respectively. Spike-in experiments showed that epithelial and hybrid phenotypes were more efficiently captured (62.6 ± 18.5% and 65.4 ± 11.1%) than mesenchymal-like cancer cells (32.8 ± 10.2%). These results were confirmed using an EMT-inducible breast cancer cell line. Lower recovery rates were found for the cells in a mesenchymal-like state (31.5 ± 6.4%) than those in an epithelial state (47.56 ± 7.2%). In conclusion, the Parsortix® device may underestimate the number of mesenchymal CTCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99dabd7139c470f1c1bb03b3c610d7084c3f0056" target='_blank'>
              EMT-associated bias in the Parsortix® system observed with pancreatic cancer cell lines.
              </a>
            </td>
          <td>
            N. Vandenbussche, Renske Imschoot, Béatrice Lintermans, Lode Denolf, Joachim Taminau, Charlotte Fieuws, G. Berx, K. Gevaert, Kathleen B M Claes
          </td>
          <td>2025-06-18</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="
 e15098


 Background:
 Biology is three-dimensional, yet we lack the technology to picture the structural and molecular details throughout the tumour tissue volume, undermining our efforts to understand the heterogeneous and complex 3D spatial relationships. Ideally, archived paraffin blocks should also be revivable for comprehensive 3D analysis of multiple molecular markers at single-cell resolution.
 Methods:
 We developed a foundational chemistry (termed INSIHGT) that enables highly multiplexed, deeply penetrating immunostaining in human tissue samples. We then developed a tissue revival protocol enabling INSIHGT to be used in archived samples retrieved from paraffin blocks. We further designed and constructed a light-sheet microscope that achieves can image cleared tissues (1 cm x 1 cm x 5 mm) within 6 hours at 0.5 μm isotropic resolution. We also curated a panel of markers and probes, along with deterministic image processing strategy to achieve consistent universal cell boundary detection and segmentation in normal and tumourous tissues.
 Results:
 The overall technology platform - INSIHGT-revived - allows non-destructive virtualization of any archived tissues retrived from paraffin wax. With one round of staining and imaging, 4 markers - along with nuclear and cell segmentation panel - can be visualized at the subcellular level in centimetre-sized tissues within 6 days. The staining and retrieval chemistry has been proven to be compatible with >300 off-the-shelf antibodies, including those for epigenetics and post-translational modifications. Using our custom designed panel, accurate single-cell segmentation can be achieved for nearly all epithelial, immune and stromal cell types, improving marker expression scoring by 20%, while 3D imaging also improve machine learning-based segmentation accuracy by ~30% compared to 2D images. Up to 28-plex profiling using multi-round staining can be achieved in 1mm-thick samples preserved only with conventional formalin fixation. Samples processed with our pipeline is indistinguishable from non-processed controls by pathology specialist using H&E, immunohistochemistry, bulk transcriptomics by RNA sequencing, whole genome sequencing, and spatial proteomics profiling by mass spectrometry. As a proof-of-concept, we quantified IO-related, ADC-related, prognostically relevant and biologically interesting markers in lung, colon, and breast cancer specimens.
 Conclusions:
 INSIHGT-revived can virtualize a gold mine of data from tissue archives. With extremely large data throughput reflecting ground-truth 3D tumour architectures and spatially resolved single-cell expression profiles, INSIHGT-revived can empower the discovery and clinical translation of biomarkers for patient selection for therapeutics.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b9f54cdac48ceb545c4d000772d0d90ef370f70" target='_blank'>
              INSIHGT revived: Unified 3D tissue profiling for spatially resolved, multi-marker, quantitative expression scoring at single-cell resolution for fresh and archival samples.
              </a>
            </td>
          <td>
            Hei Ming Lai, Lichun Zhang, Shuk Ling Chan, Tony S. K. Mok
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Drug resistance poses a significant challenge in cancer therapy, contributing to rapid recurrence, disease progression, and high patient mortality. Despite its critical impact, few reliable predictors for cancer drug response or failure have been established for clinical application. Tumor heterogeneity and the tumor microenvironment (TME) are pivotal factors influencing cancer drug efficacy and resistance. Tumor heterogeneity leads to variable therapeutic responses among patients, while dynamic interactions between cancer cells and the TME enhance tumor survival and proliferation, underscoring the urgent need to identify cellular hallmarks for predicting drug response and resistance. Single-cell and spatial omics technologies provide high-resolution insights into gene expression at the individual cell level, capturing intercellular heterogeneity and revealing the underlying pathologies, mechanisms, and cellular interactions. This review delves into the principles, methodologies, and workflows of single-cell and spatial omics in cancer drug research, highlighting key hallmarks involving tumor heterogeneity, TME reprogramming, cell–cell interactions, metabolic modulation, and signaling pathway regulation in drug treatment at single-cell and spatial levels. Furthermore, we synthesize predictive cellular biomarkers for cancer drug response and resistance across 25 cancer types, paving the way for advancements in cancer precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af5a8672965af056452be6aaf8b56f1ff303b62" target='_blank'>
              Application of single-cell and spatial omics in deciphering cellular hallmarks of cancer drug response and resistance
              </a>
            </td>
          <td>
            Xiaoxia Cheng, Ting Peng, Tian Chu, Yiqun Yang, Jia Liu, Qinglei Gao, C. Cao, Juncheng Wei
          </td>
          <td>2025-07-02</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 e20542


 Background:
 Current diagnostic criteria for multifocal lung adenocarcinoma rely primarily on histopathology and next-generation sequencing. However, distinguishing multiple primary lung adenocarcinomas (MPLC) from intrapulmonary metastasis (IM) is challenging due to similar histology. This study explores key differences in the tumor microenvironment (TME) and tumor heterogeneity between MPLC and IM.
 Methods:
 Eight patients with 16 lung nodules were selected for single-cell sequencing. Samples were categorized into three pairs for the MPLC group and five pairs (five primary and five metastatic) for the IM group. Differences in the TME and tumor cell heterogeneity were analyzed using multiplex immunofluorescence, spatial proteomics, and bioinformatics.
 Results:
 In the TME, IM group T cells showed stronger immunosuppressive features, while MPLC group had enriched CD8 effector and follicular helper T cells. IM group macrophages exhibited pro-inflammatory and pro-angiogenic traits, contrasting with MPLC group's lipid metabolism-linked macrophages. MPLC group endothelial cells expressed tumor-suppressor genes, whereas IM group fibroblasts showed genes related to stromal remodeling and tumor progression. Tumor cell heterogeneity revealed upregulated CRABP2, CEACAM5, and PLAT in IM group, associated with aggressive phenotype, while MPLC group showed higher extracellular matrix-related protein expression. Four genes (CXCL8, RNASE1, CTSH, AQP4) showed significant transcriptional differences.
 Conclusions:
 Our study identifies key TME differences between IM and MPLC, particularly in immune suppression and cellular composition. The distinct roles of macrophages, endothelial cells, and fibroblasts in IM suggest increased tumor aggressiveness. The identified interaction pairs and differential genes may serve as biomarkers for distinguishing MPLC from IM, aiding in diagnostic and therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24593b501d640a253836bb81a758a90b930a596e" target='_blank'>
              Unraveling the complex code of tumor microenvironment and heterogeneity by comparing multiple primary lung adenocarcinomas and intrapulmonary metastasis: A multidimensional exploration.
              </a>
            </td>
          <td>
            Shaobin Yu, Shuchen Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background
 The tumor microenvironment and biomarkers play a pivotal role in breast cancer research, yet there remains a pressing need for effective biomarkers. This study focuses on identifying a novel IGKC+ T Cell subpopulation and its related biomarkers to pave the way for innovative targeted therapies and improved clinical outcomes.


Methods
We first performed single-cell RNA sequencing (scRNA-seq) analysis to characterize immune cell heterogeneity within the tumor microenvironment, leading to the identification of series cell subpopulation. Then, by performing univariate analysis to correlate cell proportions with patient prognosis, we identified a novel IGKC+ T cell subpopulation. Next, we applied bulk RNA-seq deconvolution algorithms to estimate the abundance of this subpopulation across breast cancer cohorts. Differential expression analysis and weighted gene co-expression network analysis (WGCNA) were employed to identify genes associated with the IGKC+ T cell population. To pinpoint key regulatory genes, we applied machine learning algorithms. Based on the hub genes identified, we constructed a prognostic risk model and developed a nomogram to aid clinical decision-making. Immune infiltration patterns were further assessed in high- vs low-risk groups defined by the model. Finally, functional validation was performed through overexpression of BCL2L14 in vitro, and downstream signaling pathways were examined.


Results
We identified the novel IGKC+ T cell subpopulation and core genes. Machine learning pinpointed BCL2L14, IGHD, MAPT-AS1, NT5DC4, and TNIP3 as key regulators of breast cancer progression in this subpopulation. The model stratified patients into high- and low-risk groups, with high-risk patients showing worse prognosis and weaker immune infiltration. Overexpression of BCL2L14 was experimentally demonstrated to accelerate breast cancer progression, linked to enhanced phosphorylation of the NF-κB pathway.


Conclusion
Our results underscore BCL2L14 as a potential driver within the novel T-cell subpopulation and a critical biomarker for breast cancer diagnosis. These findings provide a basis for developing advanced diagnostic tools and targeted therapies, which may ultimately enhance patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/899ab08619ae94e923463082f4db3808d873da0a" target='_blank'>
              Integration of Bulk and Single-Cell Transcriptomics Reveals BCL2L14 as a Novel IGKC+ T Cell-Associated Therapeutic Target in Breast Cancer.
              </a>
            </td>
          <td>
            Jiaming He, Aiman Akhtar, Jing Li, Qiang Wei, Yang Yuan, Jianhua Ran, Yongping Ma, Dilong Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of inflammation research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4879912b1737fce64c962b382e72c481f6e838" target='_blank'>
              Spatial mapping of proteins and their activity states in cancer models by multiplex in situ PLA
              </a>
            </td>
          <td>
            Liza Löf, Bo Xu, Tanay Kumar Sinha, Caroline Dahlström, J. Vennberg, Tore Larsson Forssén, Axel Klaesson, C. Clausson, Xuan Wang, Ulla Strömberg-Olsson, M. Kamali-Moghaddam, C. Avenel, C. Wählby, Agata Zieba-Wicher, Ulf Landegren
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Renal cell carcinoma is characterized by a poor prognosis. Recently, renal cell carcinoma has been recognized as a metabolic disease associated with fatty acid metabolic reprogramming, although in-depth studies on this topic are still lacking. We found that fatty acid metabolism reprogramming in renal cell carcinoma is primarily characterized by high expression of FABP1. FABP1 + tumors significantly impact survival and display distinct differentiation trajectories compared to other tumor subclusters. They show elevated expression of angiogenesis and cell migration signals, with PLG-PLAT-mediated interactions with endothelial cells notably enhanced. Spatial transcriptomics show a prominent co-localization of FABP1 + tumors with endothelial cells, and their spatial distribution closely aligns with that of PLAT + endothelial cells. FABP1 + tumors exhibit a unique pattern in spatial transcriptomics, enriched in Extracellular Matrix and angiogenesis-related pathways. Through receptor-ligand interaction analysis, a novel PLG-PLAT functional axis was found between tumor epithelial cells and endothelial cells. Based on results of experiments, we infer that FABP1 + tumors can promote plasmin-related tumor angiogenesis by triggering the PLG-PLAT signaling axis. Finally, utilizing preclinical models, we suggest that targeting the FABP1-PLG-PLAT axis may serve as promising strategy enhancing the sensitivity of Tyrosine Kinase Inhibitor therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02377-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27626d5d72e69d9402b111fc64dabcbee4819f07" target='_blank'>
              Single-cell multi-omics reveals that FABP1 + renal cell carcinoma drive tumor angiogenesis through the PLG-PLAT axis under fatty acid reprogramming
              </a>
            </td>
          <td>
            Yiqiu Wang, Yingchun Liang, Min Li, Jiayi Lu, Sian Zhou, Yaoyu Yu, Changwei Yang, Xinhuang Hou
          </td>
          <td>2025-06-16</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Undescribed functional axes may intersect with the trajectory of T cell exhaustion (TEX) to contribute to the antitumoral functions of CD8 T cells. By leveraging fluorescent transcriptional reporting of the T cell activation marker Cd69, we defined a classifier for potent versus suboptimal CD69+ activation states arising from T cell stimulation. In tumors, this delineation provided an additional functional readout among TEX subsets, marked by enhanced effector molecule production. The more potent Cd69-TFPhi state was the most prominent in a T cell-mediated tumor clearance model, displaying increased engagement and superior tumor cell killing. Simultaneous analysis of gene and protein expression in human head and neck tumors enabled a similar strategy to identify Cd69RNAhiCD69+ cells with enhanced functional features compared with Cd69RNAloCD69+ cells among intratumoral CD8 T cell subsets. Thus, refining the T cell functional landscape in tumors potentiates the identification of rare, potent effectors that could be leveraged for improving cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74269df46b56a19866a8dadadb260ba823cd0fa7" target='_blank'>
              Multimodal delineation of a layer of effector function among exhausted CD8 T cells in tumors.
              </a>
            </td>
          <td>
            A. Ray, Molly Bassette, Kenneth H Hu, Lomax F. Pass, T. Courau, B. Samad, Alexis J. Combes, Vrinda Johri, Brittany R. Davidson, Katherine Wai, Patrick Ha, Grace Hernandez, Itzia Zaleta-Linares, Matthew F. Krummel
          </td>
          <td>2025-07-11</td>
          <td>Science immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Gliomas, particularly glioblastomas, are highly malignant brain tumors with high recurrence rates and poor prognosis. Despite advances in treatment, recurrence remains a major challenge. Epithelial-mesenchymal transition (EMT) plays a key role in tumor invasion and recurrence. This study explores the transcriptional and regulatory mechanisms driving glioma recurrence, focusing on mesenchymal-like (MES-like) subpopulations. Single-nucleus RNA sequencing was performed on 52 IDH wild-type GBM specimens, including 26 primary and 26 recurrent tumors. Spatial transcriptomics data were also incorporated. Tumor subpopulations were identified through gene regulatory network analysis, copy number variation detection, and nonnegative matrix factorization. Functional validation was conducted using gene knockdown experiments, followed by xenograft studies. We discovered novel MES-like subpopulations in recurrent GBM enriched with EMT-related genes like EGR1 and SERPINE1. These subpopulations exhibited increased transcriptional activity and were associated with poor prognosis and invasiveness. Knockdown of SERPINE1 significantly reduced cell proliferation and migration. Spatial transcriptomics showed MES-like cells concentrated at the tumor margins, highlighting their role in invasion and recurrence. MES-like subpopulations, driven by EGR1 and SERPINE1, are critical in GBM. Targeting these regulators could offer new therapeutic strategies to reduce glioma recurrence and improve outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02036-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f7432f6a50e6a5d830f429844d915dd10fcd2" target='_blank'>
              Single-cell transcriptome sequencing reveals new epithelial-stromal associated mesenchymal-like subsets in recurrent gliomas
              </a>
            </td>
          <td>
            Jinwei Li, Shengrong Long, Yang Zhang, Shuangqi Yu, Hongyu Xu, Rui Liang, Quan Liu, Jinnan Zhang, Xiang Li, Yixin Fu, Tao Xin, Yinyan Wang
          </td>
          <td>2025-06-07</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The role of the microbiome in cancer metastasis has emerged as a critical area of research, with growing evidence suggesting that microbial composition and interactions within the tumour microenvironment may significantly influence metastatic progression. This review explores the role of the microbiome in cancer metastasis, as well as potential key bacteria and their mechanisms through which they could impact tumour dissemination, seeding and growth. Biological models used to study metastasis are discussed to provide context for the further investigation of these interactions. In order to answer unresolved questions regarding the microbiome’s involvement in metastatic dissemination, recent advancements in spatial biology techniques are examined, including spatial genomics, transcriptomics, proteomics and metabolomics, which enable the spatial mapping of microbial interactions within the tumour microenvironment. Additionally, multimodal-omics imaging approaches are highlighted for their potential to integrate multiple molecular layers, offering comprehensive insights into the microbiome’s role in cancer metastasis. The review also addresses the challenges and limitations of these techniques, underscoring the complexity of studying microbiome-tumour interactions and offering directions for future research to better explore and target the microbiological landscape in metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bd737e751891e52e56b56db2c8e1ef16eeffa30" target='_blank'>
              Microbiome in cancer metastasis: biological insights and emerging spatial omics methods
              </a>
            </td>
          <td>
            Marianne Meyers, Charlotte B. A. Stoffels, G. Frache, Elisabeth Letellier, Maureen Feucherolles
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Patients with high-grade serous ovarian cancer (HGSOC) typically present with widespread metastasis, obscuring a temporal understanding of tumor-immune dynamics. To address this, we perform multi-site global proteomics alongside matched immunohistochemistry (IHC) for CD4⁺ and CD8⁺ tumor-infiltrating lymphocytes (TILs) in patient samples. We order the protein expression profiles using an unbiased pseudotime analysis, recapitulating clinical observations of metastatic progression, and providing a framework to explore tumor-immune dynamics from localized to metastatic disease. Metastatic progression correlates with immune cell infiltration, the recruitment of regulatory T cells (Tregs) to counterbalance γδ T cell abundance, and an increased abundance of exhausted CD8⁺ T cells. The accumulation of Tregs at metastatic sites correlates with SNX8 expression, a critical regulator of the STING pathway. In early-stage tumors, keratin-expressing cancer cells recruit Tregs via MHC class II, fostering an inflammatory phenotype with limited IFNγ production and non-clonally expanded T cells. Together, our findings reveal novel mechanisms of immune escape associated with both localized disease and metastatic progression in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5284b73444d2932c5ad492644667337328f9e74" target='_blank'>
              A temporal model of tumor-immune dynamics during the metastatic progression of high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Donagh Egan, Kate Glennon, Ann Treacy, Aurelie Fabre, Janet Mccormack, Salisha Hill, Ryan Morrison, Vadim Zhernovkov, Daniel C Liebler, Walter Kolch, Donal J Brennan
          </td>
          <td>2025-06-16</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="A low tumor-stroma ratio (TSR) in pancreatic neuroendocrine tumors (pNET) is associated with a significantly poorer patient prognosis. Although the tumor stroma represents an attractive therapeutic target, recent clinical trials have not been successful. In this study, we aimed to dissect the mechanisms regulating the tumor microenvironment in low TSR pNETs to identify potential therapeutic targets. Laser capture microdissection analysis revealed that stroma-rich tumors excessively secrete apolipoprotein E (ApoE) relative to stroma-poor tumors, with the specific receptor SCARB1 predominantly located on endothelial cells (EC). Single-cell analysis revealed a greater proportion of endothelial tip cells (TipEC) in stroma-rich tumors due to transformation of other types of ECs into TipECs induced by cancer cell-derived ApoE. The TipECs played crucial roles in driving pNET progression by facilitating cancer-associated fibroblast recruitment and remodeling the TSR. Mechanistically, ApoE promoted the uptake of palmitic acid by ECs and subsequently activated the transcription factor ATF6 to upregulate the PDGF pathway. Screening of six commonly used drugs for pNETs in vivo revealed that treatment with mTOR inhibitors suppressed the secretion of ApoE by cancer cells, blocking the subsequent effects of ApoE on the stromal microenvironment. Importantly, mTOR inhibitors synergistically enhanced the antitumor effects of stroma-targeting PEGPH20 in vivo in pNETs. Overall, this study revealed that cancer cell-derived ApoE could induce TipECs to remodel the TSR and that mTOR inhibitors could increase the efficacy of stroma-targeting therapies.


SIGNIFICANCE
Secretion of ApoE by pancreatic neuroendocrine tumor cells engenders a stroma-rich microenvironment, which can be reversed with mTOR inhibitors as part of combination strategies targeting the tumor stroma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9134826d7bd37d7e290ce41ea3d6eacda4743922" target='_blank'>
              Pancreatic Neuroendocrine Tumors Secrete Apolipoprotein E to Induce Tip Endothelial Cells That Remodel the Tumor-Stroma Ratio and Promote Cancer Progression.
              </a>
            </td>
          <td>
            X. Lou, Yihua Shi, Faming Zhao, Xiaowu Xu, Yan Wang, Yi Qin, Wuhu Zhang, Zeng Ye, Fei Wang, Tian Ding, Desheng Jing, Guixiong Fan, Yue Zhang, Xuemin Chen, Jie Chen, Xianjun Yu, Junfeng Xu, S. Ji
          </td>
          <td>2025-07-07</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Salivary gland cancers are rare, diverse malignancies characterized by poor response to immunotherapy. The tumor immune environment in these cancers remains poorly understood. To address this, we perform an integrative analysis of the tumor immune microenvironment in a large cohort of advanced salivary gland cancer samples. Most tumors exhibit low immune activity with limited immune cell infiltration. Inflammation is linked to higher tumor mutational burden in non-adenoid cystic carcinoma histologies. Subtype specific expression of immune checkpoints is identified with prominent expression of VTCN1 in luminal-like cells within adenoid cystic carcinoma. Macrophages with immunosuppressive properties dominate the immune microenvironment across subtypes. Responses to immunotherapy are limited and associated with a higher ratio of T-cells relative to macrophages in individual cases, warranting further investigation. Here, we show an immunosuppressive environment in salivary gland cancers and identify subtype-specific immune vulnerabilities that could inform tailored therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5d16057c5cf3334337ef5c688188d79ab5741" target='_blank'>
              A macrophage-predominant immunosuppressive microenvironment and therapeutic vulnerabilities in advanced salivary gland cancer
              </a>
            </td>
          <td>
            Erika Zuljan, B. von der Emde, I. Piwonski, Ana Pestana, K. Klinghammer, A. Mock, Peter Horak, Christoph Heining, Frederick Klauschen, Ina Pretzell, M. Boerries, Christian H. Brandts, S. Kreutzfeldt, Maria-Veronica Teleanu, D. Huebschmann, L. Morris, M. Heiland, U. Keller, T. Conrad, H. Glimm, S. Fröhling, Sebastian Ochsenreither, Ulrich Keilholz, Eric Blanc, D. Beule, D. Rieke
          </td>
          <td>2025-06-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 e15561


 Background:
 Colorectal cancer (CRC), the third most prevalent cancer globally, remains a major cause of cancer-related mortality. Genomic and epigenomic plasticity during CRC progression, particularly metastasis, poses significant challenges to effective treatment. Patient-derived organoids (PDOs) reflect the heterogeneous genetic and morphological composition of cancer cells, enabling ex vivo disease modeling. This study investigates the epigenomic makeup of human primary and metastatic CRC to advance precision epigenetic therapies.
 Methods:
 We analyzed 45 organoid models, including 40 CRC organoids (from primary tumors and liver/lung metastases) and 5 generated from healthy colonic tissue. Using integrative epigenomic profiling, including chromatin accessibility, chromatin state mapping (via six histone modifications), and 3D genome conformation, we constructed a comprehensive CRC epigenomic landscape.
 Results:
 Our analysis of CRC patient-derived organoids revealed a core set of accessible genomic regions shared across all organoids, irrespective of tumor location or metastatic status. This conserved chromatin signature highlights common regulatory pathways in CRC during disease progression. Unsupervised clustering based on epigenomic features revealed four distinct CRC subtypes, encompassing both primary and metastatic models. Notably, organoids derived from liver and lung metastases did not cluster by metastatic site, underscoring epigenomic convergence. Subtype D comprised 57.1% (8/14) of primary tumors, while subtypes B and C contained 47.3% (9/19) of liver metastases and 42.8% (3/7) of lung metastases, respectively. KRAS mutation status further stratified subtypes, with wild-type KRAS observed exclusively in subtypes A and B, while KRAS-mutant variants were enriched in subtypes C (6/7) and D (4/10). Histone modification patterns, defined by six marks (H3K4me1, H3K4me3, H3K27me3, H3K27ac, H3K36me3, and H3K9me3) revealed subtype-specific regulatory chromatin landscapes. Three-dimensional chromatin interactions were mapped, and genome-wide interaction contacts are used to identify various chromatin structures that may influence metastatic capacity.
 Conclusions:
 This study highlights distinct and shared epigenomic signatures in CRC PDOs, offering novel insights into tumor progression and metastasis. Integrative chromatin analysis provides a framework for identifying epigenetic biomarkers and epigenome targeting strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66760aad28585c2f41842514d0645e6e13f51968" target='_blank'>
              Deciphering colorectal cancer progression through integrative epigenomic analysis for precision therapeutics.
              </a>
            </td>
          <td>
            Ghazaleh Tavallaee, Amin Nooranikhojasteh, Elias Orouji
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Analysis of post-translational modifications (PTMs) of proteins can provide new insight, beyond that obtained from analysis of protein levels, for understanding the tumor microenvironment (TME). The characteristics of PTMs in immune cells, along with their spatial distribution, have not been comprehensively integrated, which impedes our understanding of the complexity and heterogeneity of the TME in hepatocellular carcinoma (HCC). Herein, we used a strategy that combines antibodies for specific PTMs with mass cytometry and mass spectrometry technologies to identify PTMs at single-cell resolution. We found that the phosphorylation status of M2 macrophages was substantially altered in tumor tissues from patients with hepatitis B virus (HBV)-positive HCC. Utilizing the expression profiles of site-specific phospho-HSP27, STAT1, and TRIM28, we classified M2 macrophages into four distinct subtypes: M2-P0 (absence of any of the three phospho-proteins), M2-P1 (presence of one of the three phospho-proteins), M2-P2 (presence of two of the three phospho-proteins), and M2-P3 (presence of all three phospho-proteins). The spatial relationships and functional characteristics of these M2 macrophage subpopulations were assessed using single-cell PTM (scPTM) omics. The abundance of the M2-P2 and M2-P3 subtypes was closely associated with an immunosuppressive TME and responsiveness to immunotherapy in HBV+ HCC. Overall, this study introduces a scPTM omics approach that uncovers subtypes of macrophages associated with immunotherapeutic responses in HBV+ HCC and provides valuable insights into the immunosuppressive TME of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/122793fb7689acfc8cdc967dfa3fecfbbdbf0958" target='_blank'>
              Single-cell analysis of post-translational modifications identifies immunosuppressive macrophage subtypes in the HBV-positive hepatocellular carcinoma microenvironment.
              </a>
            </td>
          <td>
            Huakan Zhao, Ran Ren, Xi Zhang, Mengtao Zhan, Jinwei Cui, Jun Zhang, Xi Liu, Lei Wu, Yu Chen, Yu Zhou, Yang Xiao, Jiangang Zhang, Yang Chen, Lu Zheng, Bing Sun, Yongsheng Li
          </td>
          <td>2025-06-02</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The mechanism that lung epithelial cells regulate immune responses during chronic injury still remains unclear. Here, we combined single-cell RNA sequencing with spatial transcriptomics to track epithelial dynamics in silica (SiO2)-exposed mouse lungs. By day 56, SiO2 induced significant epithelial proliferation, followed with a distinct C0 subset emerging as the dominant population. C0 cells co-expressed surfactant genes (Sftpc, Scgb3a2), mesenchymal markers (Vim, Mmp12), and pro-inflammatory cytokines (Ccl6, S100a8/a9), reflecting a hybrid phenotype. Spatial and cell-cell interaction analyses showed C0 cells engaging macrophages and neutrophils through SPP1-CD44, APP-CD74, and GRN-MARCO signaling, driving immune recruitment and activation. Pseudotime and CytoTRACE analyses indicated that C0 cells represent a late-stage, low-stemness state with epithelial-mesenchymal transition (EMT)-like features. Taken together, these findings reveal a novel, stress-induced epithelial subset that amplifies immune crosstalk and tissue remodeling, offering new perspectives on silica-induced lung injury.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f18c3a88c62f2a6706ac4bcfbf6e192adf197a02" target='_blank'>
              Single-cell and spatial transcriptomics reveal a stress-induced EMT-like epithelial subset driving immune activation in silica-injured lung
              </a>
            </td>
          <td>
            Tao Wang, Jianfeng Hao, Bing Li, Ahjol Hyraht, Jialing Wang, Henglei Xia, Qingbin Wu, Wei Gao, Congxia Chen, Chuanqing Yu, X. Gong, Ting Li, Mei Zhang, Yinghai Xie, Xinrong Tao
          </td>
          <td>2025-06-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (ICC) is characterized by high malignancy, and its global incidence is predicted to continue to increase over the past decades. However, the mechanisms underlying ICC pathogenesis and progression remain unclear.


APPROACH AND RESULTS
The training cohort consisted of single-cell sequencing of 12 treatment-naïve ICC samples and spatial transcriptomics of four ICC samples. The validation cohort comprised of RNA-seq data from 87 ICC tumor samples. Finally, we validated our findings via multiplex immunofluorescence, organoids, and mice models both in vivo and in vitro. We found significant heterogeneity within the tumor microenvironment (TME) of ICC patients. ICC cells were classified into five molecular subtypes, and we revealed that aspartate beta-hydroxylase (ASPH) was a marker gene for invasion subtypes. We then selected cepharanthine (CEP) as an ASPH inhibitor that effectively suppressed tumor progression. Regarding the ICC lymphatic metastasis mechanism, we found that tumor cells in N1 lymph nodes highly expressed tumor-specific MHC-II molecules but lacked co-stimulatory factors CD80/CD86, inducing a state of anergy in CD4+ T cells, which might facilitate ICC immune evasion.


CONCLUSIONS
The TME of ICC was heterogeneous. ASPH markedly enhanced ICC invasion The ASPH inhibitor CEP significantly inhibits ICC progression and may serve as a targeted therapeutic drug for ICC. Tumor cells in N1 lymph nodes demonstrate high expression of tumor-specific MHC-II molecules, but silencing of co-stimulatory factors such as CD80/CD86 induces CD4+ T cells into an anergic state. Our study indicated that ASPH and MHC-II may serve as novel therapeutic targets for ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2012ee56dc72ff219a9edcce829b5a1b74d80284" target='_blank'>
              Tumor microenvironment heterogeneity and progression mechanisms in intrahepatic cholangiocarcinoma: A study based on single-cell and spatial transcriptomic sequencing.
              </a>
            </td>
          <td>
            Fengwei Li, Yao Li, Lishan Wang, Lei Xu, Hui Xue, Wenxin Wei, Yong Xia, Lei Wang, Feng Shen, Kui Wang
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Mutated oncogenic Kirsten rat sarcoma virus (KRAS) antigen is expressed in a large variety of cancers, including pancreatic, colorectal, and pulmonary cancers. The oncogenic KRAS mutations cause malignancies and are usually ubiquitously expressed by all cells of a tumor. The KRAS amino acid substitutions at the positions 12 or 13 are among the most frequent mutations in human cancers. Here, we developed immuno-oncotherapeutic non-integrative lentiviral vectors encoding a segment encompassing KRASG12D, either alone, or associated with antigen carriers. These carriers can improve the intracellular antigen routing to major histocompatibility complex presentation machineries or provide universal helper CD4+ epitopes. Immunotherapy with one of these vectors resulted in significant immune control of tumor growth in colorectal or pulmonary preclinical cancer models, in several murine genetic backgrounds. The antitumor effect was correlated with increased proportions of intra-tumoral hematopoietic cells and notably CD8+ T cells. Although this effect was partial, it was robust, reproducible and advantageously combinable with conventional chemotherapies and immunotherapies to improve antitumor protection. Therefore, this approach shows promise as an immuno-oncotherapy against KRAS-mediated malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61951127f6ba1a1f776a4ed744c835e44ef0de6c" target='_blank'>
              A lentiviral vector targeting a KRAS neoepitope for cancer immunotherapy
              </a>
            </td>
          <td>
            Anastasia Goloudina, Fabien le Chevalier, P. Authié, Sylvain Ciret, K. Nemirov, Ingrid Fert, F. Moncoq, B. Vesin, A. Noirat, Catherine Blanc, Yu Wei, Pierre Charneau, L. Majlessi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer mortality among people with HIV (PWH), with increased incidence and poor outcomes. This study explored whether the tumor microenvironment (TME) of HIV-associated non-small cell lung cancer (NSCLC) limits tumor-specific immune responses. With a matched cohort of NSCLC from PWH and people without HIV (PWOH), we used imaging mass cytometry, linear mixed effects model and AI-based pageRank mathematical algorithm based on spectral graph theory to demonstrate that HIV-associated tumors demonstrate differential distribution of tumor infiltrating CD8+ and CD4+ T cells, enriched for the expression of PD-1 and Lag-3, as well as activation and proliferation markers. We also demonstrate higher expression of immunoregulatory molecules (PD-L1, PD-L2, B7-H3, B7-H4, IDO1 and VISTA), among tumor-associated macrophages. Discrimination of cells between tumors from PWH versus PWOH was confirmed by spectral graph theory with 84.6% accuracy. Furthermore, we noted differences in spatial orientation of immune cells within the TME of PWH compared to PWOH. Additionally, cells from PWH, compared to PWOH, exhibited decreased tumor killing when exposed to HLA-matched NSCLC cell lines. In conclusion, our study demonstrates that the HIV-associated tumor microenvironment sustains a unique immune landscape, with evidence of immune cells with enhanced immunoregulatory phenotypes and impaired anti-tumor responses, with implications for response to immune checkpoint blocker therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91ad1598a3561eab7080185d3170eda1e153cd68" target='_blank'>
              Tumor microenvironment of non-small cell lung cancer impairs immune cell function among people with HIV.
              </a>
            </td>
          <td>
            Shruti S Desai, Syim Salahuddin, R. Yusuf, Kishu Ranjan, Jianlei Gu, Lais Osmani, Y. Lin, Sameet Mehta, R. Talmon, Insoo Kang, Y. Kluger, Hongyu Zhao, K. Schalper, B. Emu
          </td>
          <td>2025-06-03</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="
 e16452


 Background:
 Liver metastasis is a leading cause of death amongst localized pancreatic ductal adenocarcinoma (PDAC) patients following surgical resection. Yet, there are no adjuvant therapies currently available targeting liver colonization in PDAC. We are hindered in our ability to develop such therapies by an incomplete understanding of PDAC liver metastasis' molecular mechanisms.
 Methods:
 To address this challenge, we labeled primary mouse PDAC subclones with DNA barcodes to characterize their pre-metastatic state using ATAC-seq and RNA-seq and determine their relative
 in vivo
 liver metastasis potential in competition assays in immunocompetent hosts.
 Results:
 We identified a gene signature separating metastasis-high and metastasis-low subclones orthogonal to the normal-to-PDAC and classical-to-basal axes. The metastasis-high subclones feature activation of inflammation-related genes and high NF-κB and Zeb/Snail family activity and the metastasis-low subclones feature activation of neuroendocrine, motility, and Wnt pathway genes and high CDX2 and HOXA13 activity. In a functional screen, we validated novel mediators of PDAC liver metastasis related to inflammation, including the NF-κB targets
 Fos
 and
 Il23a
 , and beyond inflammation including
 Myo1b
 and
 Tmem40
 . We scored human PDAC tumors for our signature of metastatic potential from mouse and found that metastases have higher scores than primary tumors. Moreover, primary tumors with higher scores are associated with worse prognosis. We also found that our metastatic potential signature is enriched in other human carcinomas, suggesting that it is conserved across epithelial malignancies.
 Conclusions:
 This study establishes a model of clonal evolution during PDAC liver metastasis under immune selective pressure, revealing novel functional regulators amenable to therapeutic intervention in the adjuvant setting. Furthermore, it provides a method for scoring human carcinomas based on metastatic potential that may have prognostic and predictive value across multiple cancer subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d54beefd4d0e886101bb6db8b7a938594843dac" target='_blank'>
              Effect of clonal competition model of PDAC liver metastasis on a pan-carcinoma gene signature of metastatic potential.
              </a>
            </td>
          <td>
            Jesse S Handler, Zijie Li, Rachel K Dveirin, Jessica Lin, Weixiang Fang, James Forsmo, Hani Goodarzi, E. Fertig, Reza Kalhor
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy; however, no validated treatments are currently available. Clinically, the tumor position, head and uncinate process (HU), or body and tail (BT) of the pancreas are vital for surgical strategies; however, fundamental research has seldom revealed the heterogeneous tumor microenvironment (TME) among different types of PDAC. Here, we applied multicohort single-cell and spatial RNA-seq methods together with patient-derived organoid models to reveal the TME heterogeneity between HU and BT PDAC. Osteopontin, encoded by SPP1, is secreted by vessel endothelial cells in BT PDAC and is associated with increased tumor burden. The number of tumor cells marked by CRABP2 was lower in BT PDAC and was identified as a prognostic marker of overall survival, as well as CD8+ T-cell infiltration. The expression of CRABP2 was also validated to be downregulated in BT PDAC in patient-derived organoid models. Overall, we profiled the heterogeneous PDAC TME between HU and BT PDAC, which could provide novel insight into the relationships between clinical characteristics and TME molecular research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077490657a2feb3d7ea606c6fed7a402a475e6c0" target='_blank'>
              SPP1+ Venous Endothelial Cells and CRABP2+ Tumor Cells Contribute to Favorable Body and Tail Pancreatic Ductal Adenocarcinoma Tumor Microenvironment.
              </a>
            </td>
          <td>
            Yueze Liu, Yifan Fu, Tao Liu, Jun Wang, Zeyu Zhang, Yanan Shi, Zhe Cao, Gang Yang, Hao Chen, W. Luo, Jinxin Tao, Yuanyang Wang, Guihu Weng, Menggang Zhang, Liyuan Ye, Jianchun Xiao, J. Qiu, Taiping Zhang, Hua Huang
          </td>
          <td>2025-06-11</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Chromoanagenesis is a catastrophic genomic phenomenon involving sudden, extensive rearrangements within one or a few cell cycles. In osteosarcoma, the most prevalent malignant bone tumor in children and adolescents, these events dramatically alter the genomic landscape, frequently disrupting key tumor suppressor genes like TP53 and RB1, amplifying oncogene expression, and propelling tumor progression and evolution. This review elucidates how key chromoanagenic mechanisms, such as chromothripsis and chromoanasynthesis, arise from replication stress and impaired DNA repair pathways, ultimately contributing to genomic instability in osteosarcoma. Chromothripsis features prominently in osteosarcoma, occurring in up to 62% of tumor regions and driving intratumoral heterogeneity through persistent genomic crises. Next-generation sequencing, optical genome mapping, and emerging technologies like single-cell sequencing empower researchers to detect and characterize these complex structural variants, demonstrating how a single catastrophic event can profoundly influence osteosarcoma progression over time. While targeted therapies for osteosarcoma have proven elusive, innovative strategies harnessing comprehensive genomic profiling and patient-derived preclinical models hold promise for uncovering tumor-specific vulnerabilities tied to chromoanagenesis. Ultimately, unraveling how these rapid, large-scale rearrangements fuel osteosarcoma’s aggressive nature will not only refine disease classification and prognosis but also pave the way for novel therapeutic approaches to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d502c33306d89faa688fa75a0e5f2710c0c03efc" target='_blank'>
              Chromoanagenesis in Osteosarcoma
              </a>
            </td>
          <td>
            Guozhuang Li, Nan Wu, Jennifer Ghabrial, Victoria Stinnett, Melanie Klausner, Laura A Morsberger, Patty Long, Ezra G. Baraban, J. Gross, Ying S. Zou
          </td>
          <td>2025-06-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 9562


 Background:
 The groundbreaking neoadjuvant ICB clinical trials have established beyond doubt that ICB before surgery will become the new standard of care for metastatic melanoma. Importantly, the shift from radiographic to pathologic response scoring offers an unprecedented window of opportunity to interrogate a goldmine of ‘on-treatment’ biospecimens. To date, single-cell profiling and region-based spatial transcriptomics (ST) have highlighted the importance of tertiary lymphoid structures (TLS) and stem-like T-cells as positive predictors of ICB response. However, the molecular mechanisms by which tumor infiltrating lymphocytes communicate and organize multicellular immune hubs within the spatial context of the tumor ecosystem remains poorly understood.
 Methods:
 To identify robust biomarkers of response to neoadjuvant ICB, we assembled a cohort of 58 stage III melanoma patients treated with neoadjuvant ICB [24 ipilimumab-nivolumab (IPI-NIVO), 21 NIVO-relatlimab (RELA), 13 PD1 mono] and deployed transformative technologies and computational methods, including single-cell FFPE sequencing, multiplexed FISH single-cell ST (MERFISH 305 genes), digital pathology, 3D open-top light-sheet imaging and AI-based computational pathology, to decipher the complex neoadjuvant ICB tumor ecosystem in response to therapy. We also developed 2 critical computational tools, SCIRA (Spatial Cellular Interaction and Receptor Activation), to compute receptor-ligand (R-L) interactions in whole slide images, and GC-SCAN (graph-based spatial clustering against noise), a graph-based algorithm that quantifies locally clustered structures from spatial -omics data.
 Results:
 Our results showed that the quantity and size of hyper-expanded germinal center/TLS, with increased GC: non-GC B-cell ratio, plasma cells and spatially resolved stem-like T-cells are strongly associated with response. IPI-NIVO elicited significantly stronger GC proliferation compared to NIVO-RELA and PD1 monotherapy, suggesting anti-CTLA4 can robustly induce germinal center/TLS proliferation. SCIRA spatial R-L analyses revealed the critical chemokine R-L interactions that organize the GC- and T-cell zones in response to therapy. Lastly, 3D light-sheet imaging revealed remarkable morphologic heterogeneity in 3D, with interconnected GC-TLS networks that are indicative of long-range molecular gradients.
 Conclusions:
 Our investigations herein have provided a comprehensive characterization of the immune architectures, cellular communications and 3D large-scale morphologic organizations of the TME that drive response to neoadjuvant ICB therapy. We believe the results of this study will enable the development of robust predictive biomarkers to guide the design of next generation combination ICB therapies in the clinical trial setting for melanoma and other cancer types.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb50bf439bc9a3d8fb630c6a0899f89e6769df7" target='_blank'>
              Multidimensional, spatially resolved immunologic hallmarks of response to neoadjuvant immune checkpoint blockade (ICB) therapies.
              </a>
            </td>
          <td>
            Zichao Liu, Xiaofei Song, Shikhar Dhingra, Wei-Shen Chen, Justin He, Jodi A. Balasi, Jonathan Nguyen, Carlos M. Morán-Segura, Joseph Johnson, Chaomei Zhang, Zena Sayegh, Douglas C. Marchion, Sean Yoder, Nicholas Reder, Jeffrey H. Chuang, Pei-Ling Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common and deadly malignancies worldwide, with a particularly low 5-year survival rate in advanced patients. Immune cells in the tumor microenvironment, especially mast cells, play crucial roles in tumor initiation and progression. However, the dual role of mast cells in CRC remains poorly understood. Methods In this study, we used single-cell RNA sequencing (scRNA-seq), bulk RNA sequencing, and bioinformatics analyses to explore the heterogeneity of mast cell subpopulations in the CRC tumor microenvironment and their relationship with prognosis. We analyzed gene expression signatures associated with mast cell subpopulations derived from single-cell data of 40 CRC tumor samples and combined bulk RNA-seq data from HMU, GEO, and TCGA cohorts for prognostic prediction. Non-negative matrix factorization was used for clustering of mast cell subpopulations, followed by analysis of their specific gene markers, transcription factor activity, and biological pathways. Survival analysis and ROC curves were performed to assess their prognostic significance. Results Mast cells in the CRC tumor microenvironment were classified into three distinct subpopulations, each with unique gene markers and functional pathways. Mast cell subpopulations 1 and 3 were highly associated with pro-tumor pathways, while mast cell subpopulation 2 primarily exhibited anti-tumor immune regulatory characteristics. High expression of mast cell subpopulations 1 and 3 was associated with poor survival prognosis, while high expression of subpopulation 2 was linked to a better survival outcome. Key marker genes such as DNAJB1, SEMA7A, and XCR1 were identified as potential prognostic factors, with high expression of DNAJB1 and SEMA7A being significantly associated with poor prognosis, while high expression of XCR1 was linked to a favorable prognosis. Conclusion This study reveals the functional heterogeneity of mast cell subpopulations in the CRC tumor microenvironment and their differential roles in tumor progression. Identification of mast cell subpopulation-specific marker genes provides new molecular targets for clinical diagnosis, prognostic prediction, and personalized immunotherapy in CRC. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04119-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f36d01ec68db5890c180d0142a0a3d3d4bc0a598" target='_blank'>
              Diversity of mast cell subpopulations in the tumor microenvironment of colorectal cancer and their prognostic implications
              </a>
            </td>
          <td>
            Tian-yu Qiao, Chao Ding, Songtao Yu, Wenyang Li, Yonghou Zhao, Guiyu Wang
          </td>
          <td>2025-06-30</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Embryonal tumor with multilayered rosettes (ETMR) is a lethal embryonal brain tumor entity. To investigate the intratumoral heterogeneity and cellular communication in the tumor microenvironment (TME), we analyze in this work single-cell RNA sequencing of about 250,000 cells of primary human and murine ETMR, in vitro cultures, and a 3D forebrain organoid model of ETMR, supporting the main findings with immunohistochemistry and spatial transcriptomics of human tumors. We characterize three distinct malignant ETMR subpopulations - RG-like, NProg-like and NB-like - positioned within a putative neurodevelopmental hierarchy. We reveal PDGFRβ+ pericytes as key communication partners in the TME, contributing to stem cell signaling through extracellular matrix-mediated interactions with tumor cells. PDGF signaling is upregulated in chemoresistant RG-like cells in vivo and plays a role in recruiting pericytes to ETMR TME by finalizing a signaling cascade which promotes the differentiation of non-malignant radial glia cells, derived from our 3D model, into pericyte-like cells. Selective PDGFR-inhibition blocked the lineage differentiation into pericytes in vitro and reduced the tumor cell population in vivo. Targeting ETMR-pericyte interactions in the TME presents a promising therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/497d2ed06cfc62eb4cc72976940d7e3d7d08672f" target='_blank'>
              ETMR stem-like state and chemo-resistance are supported by perivascular cells at single-cell resolution
              </a>
            </td>
          <td>
            F. D. de Faria, Nicole C. Riedel, Daniel Münter, M. Interlandi, C. Göbel, L. Altendorf, Mathis Richter, V. Melcher, C. Thomas, Rajanya Roy, Melanie Schoof, Ivan Bedzhov, N. Moreno, M. Graf, M. Hotfilder, D. Holdhof, Wolfgang Hartmann, Ann-Katrin Bruns, A. Brentrup, F. Liesche-Starnecker, B. Maerkl, S. Sandmann, J. Varghese, M. Dugas, Pedro H Pinto, S. Balbach, I-Na Lu, Claudia Rossig, Oliver Soehnlein, Aysegül Canak, Martin Ebinger, Martin U Schuhmann, J. Schittenhelm, Michael F Frühwald, U. Schüller, T. Albert, Carolin Walter, Jan M Bruder, K. Kerl
          </td>
          <td>2025-06-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, M. T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Globally, endometrial cancer continues to impact a significant number of women. Immunotherapy provides those suffering from advanced or relapsed disease hope, but an important barrier is still the absence of trustworthy predictive biomarkers. To tackle this challenge, single-cell sequencing and spatial transcriptomics (ST) are increasingly applied. In cervical cancers of the no specific molecular profile (NSMP) subtype accompanied by p53 mutations. In many cases, the tumor microenvironment (TME) in endometrial cancer exhibits strong immunosuppression or poor immune cell infiltration, often leading to worse clinical outcomes. Single-cell sequencing reveals cellular heterogeneity and helps identify potential therapeutic targets and predict treatment responses. Conversely, ST assists in determining biomarkers that influence the effectiveness of immunotherapy by capturing the spatial organization of tumors. When combined, these technologies allow for integrated multi-omics analysis that aids in the development of immunotherapies, prognostication, and diagnosis. But there are still moral and legal issues. Clinicians may be able to improve outcomes for patients who don’t respond well to current immunotherapies by utilizing these combined approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f087dd53dee17003d5211dac4cb4360cce4847" target='_blank'>
              Single-cell technologies and spatial transcriptomics: decoding immune low - response states in endometrial cancer
              </a>
            </td>
          <td>
            Yumeng Li, Hua Qiu, Zhenzhen Zhao, Fanghua Qi, Pingping Cai
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease, with tumors capable of adapting to shifting conditions, making the development of effective personalized therapies particularly challenging. Patient-derived models, such as patient-derived organoids (PDOs) and circulating tumor cell (CTC) cultures, have emerged as powerful tools for investigating intra- and inter-tumor heterogeneity. These models largely retain the genetic, phenotypic, and microenvironmental features of the original tumors, providing valuable insights into disease progression, drug response, and resistance mechanisms. Furthermore, by enabling tumors' spatiotemporal molecular profiling, PDOs and CTCs offer a dynamic approach to assess treatment efficacy over time. However, to fully capture the complexity of breast cancer heterogeneity, it is required to develop models from multiple tumor and blood samples collected throughout the course of treatment. This review explores the potential of integrating PDOs and CTC models to better understand intra-tumor heterogeneity while addressing key challenges in developing patient-derived models that accurately recapitulate patients' tumors to advance personalized care. The integration of PDOs and CTCs could represent a paradigm shift in the personalized management of metastatic breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a82abf6024e151676efff540be9bde6be013c6" target='_blank'>
              Deciphering Breast Tumor Heterogeneity Through Patient-Derived Organoids and Circulating Tumor Cells.
              </a>
            </td>
          <td>
            Benedetta Policastro, Nikoline Nissen, Carla L. Alves
          </td>
          <td>2025-06-25</td>
          <td>Journal of personalized medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE Colorectal cancer (CRC) ranks third in global incidence and second in mortality, with rates increasing among younger populations. The enteric nervous system (ENS) is crucial for gastrointestinal function, and its dysfunction is associated with the progression of CRC. Neurodegeneration may disrupt ENS function and facilitate tumor growth although the mechanisms remain unclear. METHODS AND MATERIALS This study integrated single-nucleus, single-cell RNA sequencing, spatial transcriptomics (ST), and bulk transcriptomics data. Summary data-based Mendelian randomization (SMR) analysis identified neurodegeneration-related genes associated with CRC progression. Cell-cell communication was analyzed using CellCall and Cytotalk, and trajectory analysis performed using VECTOR. ST analysis was conducted using robust cell type decomposition, and spatial dependencies were assessed through MISTy and cell neighborhood network analyses. A deep learning–based prognostic model was developed using DeepSurv. Multiplex immunofluorescence was used to validate the functions of cells. RESULTS SMR analysis identified seven neurodegeneration-related genes, including MAPK9, associated with the development of CRC. High-neurodegeneration neurons were associated with poorer prognoses and infiltrated deeper into tumor niches. MISTy analysis revealed stronger spatial dependencies between high-neurodegeneration neurons and epithelial cells. Homotypic and heterotypic cell neighborhood network analyses revealed tighter neuron networks in high-degeneration groups, with enhanced neuron-epithelial communication. MAPK9-positive neurons were highly expressed in sensory neurons and facilitated tumor growth via EGFR signaling. The DeepSurv model demonstrated strong prognostic performance, with high-risk patients exhibiting worse outcomes and elevated expression of immune checkpoint genes. CONCLUSION Neurodegeneration, particularly involving MAPK9-positive neurons, promotes CRC progression through enhanced neuron-tumor interactions and EGFR signaling. This study underscores the significance of targeting neurodegeneration in CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ebbbaa1c310c2d03f5809c332bdce3f1b5993a" target='_blank'>
              Neurodegeneration Promotes Tumorigenesis in Colorectal Cancer: Insights From Single-Cell and Spatial Multiomics
              </a>
            </td>
          <td>
            Shaocong Mo, Bochen Pan, Liming Qin, Sheng Wang, Yulin Wang, Midie Xu, Fan Xia, Wenchao Gu
          </td>
          <td>2025-06-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c9003a002bf9a6c00cceca68fcf223699d2693" target='_blank'>
              Single-cell multi-omic integration analysis prioritizes druggable genes and reveals cell-type-specific causal effects in glioblastomagenesis
              </a>
            </td>
          <td>
            Y.-F. Huang, K.-L. Huang
          </td>
          <td>2025-07-01</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Single-cell ribonucleic acid sequencing (scRNA-seq) is an important tool in molecular biology, allowing transcriptomic profiling at the single-cell level. This transformative technology has provided unprecedented insights into cellular heterogeneity, lineage differentiation, and cell-type-specific gene expression patterns, significantly advancing our understanding of complex biological systems. scRNA-seq is broadly applied across various fields, including oncology, where it sheds light on intratumoral heterogeneity and precision medicine strategies, and developmental biology, where it uncovers cellular trajectories in both model and non-model organisms. Additionally, scRNA-seq has been instrumental in ecological genomics, which can help elucidate cellular responses to environmental perturbations and species interactions. Despite these advancements, several challenges remain, particularly technical and financial barriers, limiting its application to non-model organisms and tissues with complex cellular compositions. Addressing these issues will require continued innovation in single-cell isolation methods, cost-effective sequencing technologies, and sophisticated bioinformatics tools. As scRNA-seq advances, it can deepen our understanding of biological systems, with broad implications for personalized medicine, evolutionary biology, and ecological research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35562f7f2960b20968152b898a167bc09fe4badc" target='_blank'>
              Applications and techniques of single-cell RNA sequencing across diverse species
              </a>
            </td>
          <td>
            Hyunmin Woo, Seong-il Eyun
          </td>
          <td>2025-07-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Recent advances in single-cell technologies have enabled the creation of comprehensive cell atlases, reference maps of various cell types within organisms. Here we specifically focus on T cell atlases, which offer a detailed catalog of the adaptive immune system at single-cell resolution. As such, they capture cellular diversity, functional states, and spatial dynamics across tissues, developmental stages, and disease conditions. Given the central role of T cells in orchestrating immune responses, their dysregulation underpins autoimmune disorders, cancer progression and failed immunotherapies. Therefore, a unified T cell atlas is critical for decoding such disease mechanisms, identifying therapeutic targets, and advancing personalized treatments. In this article, we explore the latest advances in T cell atlases, describing breakthroughs in multi-omics technologies, spatial profiling and computational frameworks that resolve transcriptional, epigenetic and proteomic heterogeneity. We also address persistent challenges and highlight strategies to address these gaps. Finally, we discuss emerging frontiers set to reshape our understanding of T cell dynamics in both health and diseases. Together, these insights underscore the transformative potential of T cell atlases in reconstructing precision immunology and accelerating therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d0b0e5dda9a3845db88c70250149559b1315699" target='_blank'>
              Sketching T cell atlases in the single-cell era: challenges and recommendations.
              </a>
            </td>
          <td>
            Itana Bojović, António G. G. Sousa, S. Junttila, L. Elo
          </td>
          <td>2025-06-29</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Glioblastoma (GB) is the most aggressive tumor of the central nervous system (CNS), accounting for almost 80% of all primary brain tumors. Despite standard-of-care consisting of surgical resection, when possible, adjuvant radiotherapy (RT) and chemotherapy with Temozolomide (TMZ), GB remains highly fatal, with an estimated recurrence rate of over 90% and a median overall survival (OS) of around 15 months from diagnosis. Several factors contribute to such poor patient outcome, including a unique myeloid-rich tumor microenvironment (TME) that confers immunosuppression and therapeutic resistance. Multi-omics, single-cell transcriptomics and multi-modal spatial analyses of GB are unraveling the diversity of brain myeloid cells, including activated microglia, border-associated macrophages (BAM), and monocyte-derived glioma-associated macrophages (GAM), instructed by ontogeny, spatial distribution, cell-cell interactions and response to metabolic cues in the TME. In this review, we elaborate on the heterogeneity and plasticity of myeloid cells in GB and discuss the promise and challenges for rational therapeutic targeting of GAMs in GB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d921ef730c3240f4439955cc403f795493cbb47c" target='_blank'>
              Rational therapeutic targeting of myeloid cells in glioblastoma: challenges and perspectives
              </a>
            </td>
          <td>
            Faruk Akay, M. Saleh
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background PERP may have the potential to function as an oncogene. However, the precise function, prognostic value, and predictive significance remain shrouded in ambiguity. Methods We conducted an in-depth analysis using pan-cancer RNA sequencing data and various online web tools to investigate the correlation between PERP and crucial clinical outcomes such as prognosis, tumor microenvironment, and tumor metabolism. In addition, we explored the tumor-promoting role of PERP and its potential mechanisms through models such as immunofluorescence staining, flow cytometry, cell proliferation assays, wound healing assays, cell migration assays, mass spectrometry analysis and isotope tracing. Further in vivo models confirmed the functional consistency of PERP across pan-cancer. Finally, we analyzed the potential of PERP as a predictive factor for immunotherapy sensitivity in a clinical cohort. Results PERP exhibits elevated expression in the majority of cancer types and impedes immune cell infiltration as well as immune checkpoint reactivity in pan-cancer. We confirmed that PERP can promote tumor progression by tumor cell proliferation, scratch and transwell experiments. Meanwhile, the absence of PERP restricts the flux of 13C6-glucose into glycolysis and the tricarboxylic acid (TCA) cycle. Importantly, the deficiency of PERP enhances the in vivo anti-tumor efficacy of PD1 monoclonal antibodies. In addition, low PERP expression is highly correlated with the response of head and neck squamous cell carcinoma (HNSCC) patients to immunotherapy. Conclusions PERP represents a promising predictive/diagnostic biomarker and therapeutic target for HNSCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02360-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165cbe4ddcc5a029683b55fb68914aec7383f0fa" target='_blank'>
              Targeting PERP promotes anti-tumor immunity in HNSCC by regulating tumor immune microenvironment and metabolic homeostasis
              </a>
            </td>
          <td>
            Xueying Wang, Yuxi Tian, Xiaohong Wu, Yewen Zhu, Huihong Chen, Zeyao Wang, Zihan Liu, Jiaqi Tan, Zhaoyu Pan, Jiaoyan Cao, Zhenjiang Li, Xin Zhang, Zhongjie Shi, Junchen Wang, Tong Liu
          </td>
          <td>2025-06-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary Single-label immunohistochemistry (sIHC) is used in infectious disease studies to detect etiological agents, and to quantify and spatially localize antigens in lesions. While sIHC is limited to staining one antigen per tissue section, multiplex immunohistochemistry (mIHC) offers significantly more detailed information and is more cost- and time-efficient than repeated sIHC. Given that swine lymph node pathology involves variations in T and B lymphocytes and macrophages, this study aims to apply mIHC to visualize simultaneously and on the same slide their spatial distribution and variation across the three main regions of the swine lymph node: follicles, interfollicular areas and medulla-like tissue. We demonstrated the use of bright-field mIHC for in situ analysis of spatial cellular distribution in swine lymph nodes during PCV2-SD and PRRSV lymphadenopathy, revealing distinct lymphocyte distributions compared to reactive lymphoid hyperplasia. The study highlights the potential of mIHC as a valuable tool for understanding and studying disease dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c47de3fe690520b75f0437e0eae36becf35b80" target='_blank'>
              Bright-Field Multiplex Immunohistochemistry in Swine PCV2 and PRRSV Lymphadenopathies
              </a>
            </td>
          <td>
            G. D’Annunzio, L. V. Muscatello, Chiara Tugnoli, Stefano Pesaro, Andrea Luppi, Michelangelo Fiorentino, T. Franceschini, Alessia Grillini, G. Rugna, Giuseppe Sarli, L. Mandrioli
          </td>
          <td>2025-06-01</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Metastasis remains the primary cause of mortality in gastric cancer patients; however, the underlying mechanisms driving this process remain incompletely understood. Here, we performed an integrated single‐cell analysis of gastric cancer primary tumours and their corresponding liver and lymph node metastases to identify critical intercellular communication networks driving the metastatic process. Notably, gene expression analysis of metastatic tissues showed significant upregulation of cholesterol metabolism and PPAR signalling pathway (a nuclear receptor–mediated regulatory system that orchestrates lipid metabolism, adipogenesis and energy homeostasis) genes compared to primary tumours. Our analysis revealed that myeloid cell–derived Galectin‐9 (LGALS9) and its receptor beta‐subunit of prolyl 4‐hydroxylase (P4HB) on epithelial cells constitute a previously uncharacterized ligand–receptor interaction involved in gastric cancer metastasis. Functional experiments confirmed that the activation of P4HB by LGALS9 significantly enhanced proliferation, epithelial‐mesenchymal transition (EMT) and lipid metabolism in gastric cancer cells, while pharmacological inhibition of P4HB reversed these effects. Collectively, our findings establish the myeloid‐derived LGALS9‐P4HB interaction as a crucial mediator of gastric cancer metastatic colonisation through modulation of lipid metabolism, suggesting a potential therapeutic target for metastatic gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f83d8f3206990f4a7210f3e831becc7c62823f" target='_blank'>
              Myeloid‐Derived LGALS9‐P4HB Immune Interaction Promotes Metastasis in Gastric Cancer Through Enhanced Cell Proliferation and Lipid Metabolism
              </a>
            </td>
          <td>
            Xiaobin Zhu, Yating Zhang, Aiping Yu, Xiao Xiao
          </td>
          <td>2025-06-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Mechanisms of HIV persistence in tissues are distinct from that in the blood. Spatial transcriptomic profiling examines HIV-infected cells, surrounding neighborhoods, and tissue microenvironment in unprecedented resolution. Spatial profiling captures cytokine gradients, distances between HIV-infected cells and immune effectors (and their function versus exhaustion), and cell-cell interactions. We present an overview of spatial transcriptomic platforms and a workflow of quality controls, sanity check, and bioinformatic analysis.


RECENT FINDINGS
The selection of spatial profiling methods should base on the research question, resolution, breadth of coverage, the expression level of RNA of interest, tissue quality, and tissue size. Advanced spatial transcriptomic profiling can capture RNA molecules at high resolution (<1 μm) and thus enable near-single cell profiling at genome-wide (~20 000 genes) breadth. Specifically, poly-A-based mRNA capture can identify previously unknown targets, while targeted RNA capture increases sensitivity in low-quality tissues. In targeted capture, however, the increase in target numbers frequently decreases sensitivity. Coupling ATAC-seq, protein capture, and T cell receptor sequencing to spatial platforms is ongoing.


SUMMARY
Spatial transcriptomic profiling uncovers mechanisms of HIV persistence in tissues and informs therapeutic strategies. Investigators should ensure the rigor of analysis, validate findings, and avoid reporting signatures with unknown biological significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f09209df407344118779f936014e334d9c123d1" target='_blank'>
              Roadmap for spatial transcriptomics of HIV in tissues.
              </a>
            </td>
          <td>
            Qijie Guan, Amare Eshetu, Ya-Chi Ho
          </td>
          <td>2025-07-07</td>
          <td>Current opinion in HIV and AIDS</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The tumor microenvironment, including the peripheral nervous system, plays a key role in regulating tumor biology, from initiation to cancer progression. Here, by modeling aggressive melanoma in larval zebrafish xenografts, we shed light on the dynamics of tumor innervation in the tumor microenvironment (TME). Axonogenesis and dendritogenesis were detected in the neurons surrounding the melanoma niche and neurogenesis was observed in the nearby population of the enteric nervous system. We also demonstrate the crucial role of noradrenaline in promoting melanoma progression with the dissemination and invasion of cancer cells to distant tissues. This zebrafish model will allow to uncover neural markers associated with melanoma progression to help in the design of innovative anti-neurogenic therapies targeting specifically the neuronal signals that regulate melanoma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5045f93a8a2d5cab22f534c9282946db41993e2f" target='_blank'>
              Melanoma innervation, noradrenaline and cancer progression in zebrafish xenograft model
              </a>
            </td>
          <td>
            Francesca Lorenzini, Johanna Marines, Julien Le Friec, Nam Do Khoa, M. A. Nieto, Berta Sánchez-Laorden, M. C. Mione, Laura Fontenille, K. Kissa
          </td>
          <td>2025-05-31</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e17615


 Background:
 Uterine carcinosarcoma (UCS) is a rare gynecologic malignancy that has poor clinical outcomes and few targeted therapies. This aggressive variant of endometrial carcinoma has a uniquely biphasic appearance and morphology of metastases from these tumors can be carcinoma, sarcoma, or carcinosarcoma. The tumor immune microenvironment (TIME) plays a crucial role in the behavior of cancers. Recent advances in single cell spatial transcriptomics allow for greater insight into the TIME of these aggressive tumors.
 Methods:
 Ten samples from four patients with multiple matched longitudinal samples underwent single-cell spatial transcriptomic (scST) profiling (Nanostring CosMx 6k). Standard segmentation and QC filters were applied. Cell typing was performed with confirmation by expert pathologist annotation of tissue regions. Normalized gene expression was compared within the clusters based on clinicopathologic characteristics.
 Results:
 scST profiling of 80,843 high-quality cells revealed 38 distinct clusters (11 carcinoma, 12 sarcoma, 9 immune, 7 benign mesenchymal). Tumor clusters were largely patient-specific with tumors demonstrating diverse transcriptional heterogeneity while metastatic tumors were composed of several less diverse clones. While immune cell clusters represented cells from all patients, there was significant patient and sample heterogeneity in the proportions of immune cell subsets observed. Macrophages were the most abundant immune cell type in all samples, plasma cells were common in one patient's samples, and rare T cells were found only in specific cellular niches. Transcriptomic features of intra-tumoral macrophages were assessed based on site (primary versus distant metastasis) and temporal (naïve versus treated with chemotherapy) groupings. Compared to tumor associated macrophages (TAM) in primary tumors, TAMs in distant metastases were more likely to have M2-like features with higher expression of
 CXCL2, CD163, CD209,
 and
 CD44
 (P<0.005). TAMs residing in chemotherapy-treated tumors had higher expression of
 IGF2, CXCL2, CXCL1, CD163, CD209, CXCL16, CD24
 ,
 CD44, CD93
 (P<0.005). Upregulation of pro-angiogenic pathways (i.e.
 VEGFA
 ) was common in both distant metastases and post-chemotherapy UCS TAMs.
 Conclusions:
 Uterine carcinosarcoma is a highly heterogeneous tumor both with respect to tumor cell and immune microenvironment features. This work highlights transcriptomic characteristics in macrophages in the UCS microenvironment. Notably, macrophages in distant metastases and in patients with prior chemotherapy were found to have more M2 TAM-like macrophages than primary untreated tumors. These results provide rationale for both TAM-based immunotherapy and anti-angiogenic approaches in UCS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ce5dd582063cc9e8b6f92d0391909010dfa4c1" target='_blank'>
              Single cell characterization of uterine carcinosarcoma and analysis of transcriptomic shifts in tumor-associated macrophages in metastatic and chemotherapy-treated tumors.
              </a>
            </td>
          <td>
            C. Nief, Sabrina Zdravkovic, Sahar Nasr, Phoebe Hammer, Brooke Howitt
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 DNA methylation, an essential epigenetic modification, plays a critical role in establishing and maintaining cellular identities by regulating gene expression. In cancer, aberrant DNA methylation patterns contribute to tumor development and progression, yet the cell-type-specific methylation alterations within the tumor microenvironment (TME) remain poorly understood. In breast cancer, a highly heterogeneous malignancy, endothelial cells within the TME (TECs) are key players in tumor angiogenesis and progression. Methylation changes detected in bulk tumor analyses, however, reflect signals from multiple cell types present in the TME, making it challenging to pinpoint cell-type-specific epigenetic changes. Recently, methods have been developed to using DNA methylation to deconvolute cell type proportions which can in turn be used to attribute methylation alterations to specific cell types in the TME. A reference-based cell-type deconvolution algorithm, the Hierarchical Tumor Immune Microenvironment Epigenetic Deconvolution (HiTIMED) method, allows accurate quantification of cell type proportions within the TME. We accessed DNA methylation array data from 609 breast tumors and 331 nontumor normal breast tissues from the Gene Expression Omnibus (GEO) and the Genotype-Tissue Expression (GTEx) portal. To these DNA methylation profiles, we applied HiTIMED and measured tumor and non-tumor cell proportions. Then, applying a statistical interaction model called CellDMC we leveraged the cell type proportion data to identify differential methylation specific to cell types in the breast cancer TME, including TECs and tumor cells. We identified 1897 tumor-cell-specific and 180 TEC-specific differentially methylated cytosines (DMCs) in cancer compared to normal breast tissue (FDR ≤ 1e-10). Importantly, in the cell-type unadjusted differential analysis, tumors exhibited hypermethylation compared to normal tissues; however, in the tumor-cell-specific analysis, tumor cells were hypomethylated relative to normal samples. By applying these advanced techniques to breast cancer samples, our study unveils cell-type-specific DNA methylation changes that are pivotal in driving the pro-tumorigenic environment. These findings not only enhance our understanding of the molecular mechanisms underlying breast cancer progression but also highlight potential therapeutic targets within the TME. The integration of HiTIMED and CellDMC represents a significant methodological advancement in cancer epigenetics, offering a more precise and insightful approach to studying the complex interplay between different cell types within the TME.
 Citation Format: Barbara Barbara, Lucas A. Salas, Brock C. Christensen. Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-29.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ae964d40b279e042709ea22a740c1da93f0fc" target='_blank'>
              Abstract P5-06-29: Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution
              </a>
            </td>
          <td>
            Barbara Barbara, Lucas A Salas, Brock C. Christensen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gliomas are highly aggressive, life-threatening tumors with poor prognosis, and remain a leading cause of mortality among brain cancers. Although the role of mitochondrial proteins in cancer has garnered increasing attention, their specific functions in the nervous system, particularly in gliomas, remain poorly understood. Methods We integrated single-cell RNA sequencing with cellular assays and flow cytometry to investigate the molecular characteristics and cellular interactions within glioblastoma subpopulations during tumor progression. Results Single-cell RNA sequencing revealed several differentially expressed genes (DEGs) within glioblastoma subpopulations. Trajectory analysis identified CHCHD2P9 as a pivotal marker for the terminal subpopulation. Moreover, elevated expression of CHCHD2P9 was found to correlate with poorer clinical outcomes. Subsequent cellular experiments further explored the underlying mechanisms driving these observations. Conclusions CHCHD2P9 is significantly overexpressed in glioma patients, and its differential expression plays a crucial role in regulating glioma cell proliferation and migration. A CHCHD2P9-based risk model holds promise as both a prognostic biomarker and a potential therapeutic target, providing novel insights into the pathogenesis of gliomas and opening avenues for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d625edcb2052943e2c1f19b8599a0238510719f1" target='_blank'>
              Molecular insights into glioblastoma progression: role of CHCHD2P9 in tumor heterogeneity and prognosis
              </a>
            </td>
          <td>
            Yanyu Ding, Lizhi Xiao, Xiaoling Zhou, Jiaxin Zhao, Jianli Ke, Hua-Bao Cai, Mengyu Zhao, Cun-Zhi Wang, Tian-Hang Yu, Zhijie Zhao, Yucai Wang, Jiyuan Ke
          </td>
          <td>2025-06-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e14635


 Background:
 Recent advances in computational pathology have revolutionized the analysis of histopathology images, enabling precise and rapid clinical outcome predictions. However, a significant challenge persists in translating computational insights into biologically meaningful data that informs clinical decisions. Human-interpretable image features (HIFs) have emerged as a solution, offering a detailed view of the tumor microenvironment (TME). In this study, we explored how HIFs derived from high-resolution histopathology images can predict the expression of genes associated with poor survival outcomes in stomach adenocarcinoma (STAD), aiming to bridge the gap between computational pathology and actionable clinical insights.
 Methods:
 Whole-slide histopathology images of STAD from The Cancer Genome Atlas were analyzed. Expert pathologists annotated labels were used to identify tissue types like cancer, stroma, necrosis, and normal tissue and cell types, including cancer cells, lymphocytes, macrophages, plasma cells, and fibroblasts. Using these annotations, convolutional neural networks were trained to classify these tissue and cell types, from which HIFs were extracted reflecting the biological composition of the TME, such as cell density ratios relative to surrounding tissues. We then identified genes with strong correlations to HIFs (ρ > 0.5) for further analysis.
 Results:
 Our results revealed high expression of four genes
 ABCA6, ABCA8, ADAM33,
 and
 ADAMTS10
 that were significantly associated with poor survival outcomes in STAD (p < 0.01). Each gene demonstrated a strong correlation (ρ > 0.5 and p = < 0.01) with specific HIFs.
 ABCA6
 and
 ABCA8
 , genes involved in lipid transport, are linked to a high fibroblast-to-stroma density ratio. This stromal-dominant environment is a hallmark of therapy resistance and poor outcomes.
 ADAM33
 and
 ADAMTS10
 , proteases involved in extracellular matrix remodeling, were strongly correlated (ρ > 0.5a and p = < 0.01) with a high macrophage-to-stroma density ratio. This feature represents an immunosuppressive TME, often seen in aggressive and invasive cancers.
 Conclusions:
 These findings underscore the power of HIFs to reveal key TME characteristics. A stromal-dominant TME, defined by elevated
 ABCA6
 and
 ABCA8
 , and an immunosuppressive, macrophage-rich environment associated with increased
 ADAM33
 and
 ADAMTS10
 , serve as markers of disease aggressiveness and poor prognosis. By integrating these insights into AI-driven models, clinicians may identify high-risk patients earlier in their treatment journey, enabling personalized treatment strategies to improve outcomes for STAD patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebad59e7c9b25a675a77d956eb6f9b7c2d45acd0" target='_blank'>
              Linking gene expression to tumor microenvironment using H&E features in stomach adenocarcinoma.
              </a>
            </td>
          <td>
            S. H. Bukhari, F. Vempalli, J. D. Warren, Shyam Aggarwal, Aditya Sarin, Mandeep Singh Malhotra, Rakesh K. Yadav, M. Macha, A. Bhat, Harry Lander, Tariq Masoodi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Cancer immunotherapy is an innovative treatment approach that leverages the immune system to combat tumors, demonstrating significant therapeutic potential. In the past few years, single‐cell RNA sequencing (scRNA‐seq) has made significant progress in the field of cancer immunotherapy, enabling us to understand the complexity of anti‐tumor immune processes with unprecedented depth and precision, thereby facilitating the design of more effective immunotherapy strategies. This review aims to summarize the recent applications of advanced scRNA‐seq technologies in tumor immunology. First, we outline the most representative scRNA‐seq technologies with different technical principles, with a particular focus on single‐cell T cell receptor sequencing. Next, we describe how scRNA‐seq technology is applied to identify the cellular composition and phenotypes within the tumor microenvironment for the construction of immune cell atlas, and uncover key cell types and molecular mechanisms underlying treatment responses for developing novel immunotherapies. Finally, we address the current challenges and future prospects of scRNA‐seq technology in tumor immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caad9af0b2721a017882aaaa64cafb8d1119d63" target='_blank'>
              Advanced single‐cell RNA sequencing in tumor immunology
              </a>
            </td>
          <td>
            Yilong Liu, Xiyuan Yu, Zhipeng Han, Qiuyang Wei, Qi Niu, Chaoyong Yang, Jie P. Li, Zijian Guo, Huimin Zhang
          </td>
          <td>2025-06-24</td>
          <td>Interdisciplinary Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: Lung adenocarcinoma (LUAD) remains a leading cause of cancer-related mortality despite advances in treatments, necessitating more effective therapeutic strategies. Single-cell RNA sequencing (scRNA-seq) technology has revolutionized our ability to dissect the cellular complexity of cancers, which is often obscured in conventional bulk transcriptomic experiments. Methods: In this study, we performed an integrative analysis of scRNA-seq data from multiple LUAD patient cohorts to investigate cell-type-specific transcriptomic changes across disease stages. Clustering, lineage trajectory analysis, and transcriptional regulatory network reconstruction were employed to identify stage-specific gene markers and their upstream regulators. Additionally, we constructed intercellular communication networks to evaluate signaling changes within the tumor microenvironment (TME) during LUAD progression. Results: Our analysis revealed that epithelial cells from stage IV tumors exhibited a distinct transcriptional profile compared to earlier stages, a separation not observed in immune or stromal cell populations. We identified a panel of gene markers that differentiated epithelial cells across disease stages and effectively stratified patients into subgroups with distinct survival outcomes and TME compositions. Regulatory network analysis uncovered key transcription factors, including ATF3, ATF4, HSF1, KLF4, and NFIC, as potential upstream regulators of these stage-specific genes. Moreover, cell–cell communication analysis revealed a significant increase in signaling originating from epithelial cells and a concomitant decrease in immune-derived signals in late-stage LUAD. We identified several signaling pathways enriched in stage-specific crosstalk, including Wnt, PTN, and PDGF pathways, which may play critical roles in LUAD progression. Conclusions: This study provides a comprehensive single-cell resolution map of LUAD progression, highlighting epithelial-driven regulatory programs and dynamic intercellular communication within the TME. Our findings uncover novel molecular markers and regulatory mechanisms with potential prognostic and therapeutic value for more precise treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecbc245a8fbf12aaba1f14070430ba2f845cb23" target='_blank'>
              Single-Cell Transcriptomic Analysis Unveils Key Regulators and Signaling Pathways in Lung Adenocarcinoma Progression
              </a>
            </td>
          <td>
            Jialu Ma, Caleb McQuay, John Talburt, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2025-06-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) are the predominant cell type in the stroma of many solid organ malignancies, including non-small cell lung cancer (NSCLC). They exhibit considerable phenotypic and functional heterogeneity and are widely used in functional assays and co-culture models. CAF research frequently involves the in vitro expansion and maintenance of CAFs to facilitate functional assays and co-culture studies. However, less is known about how in vitro culture temporally affects CAF phenotype. Methods We characterised the phenotype of CAFs from NSCLC patients compared to non-cancerous lung fibroblasts using conventional in vitro conditions by tracking changes in CAF subset marker expression levels by flow cytometry. Additional transcriptomic and functional analyses were performed to determine differences between CAFs and non-cancerous fibroblasts. Results We demonstrate that CAFs from NSCLC undergo phenotypic drift in culture, and that there is a convergence to a subset phenotype predominantly upregulated in non-cancerous lung. Additionally, we demonstrate the phenotype, transcriptome and function of fibroblasts converge between CAFs and fibroblasts from non-cancerous lung by the third culture passage, suggesting that in vitro conditions promote this phenotype. Conclusion We highlight the need to understand and monitor the culture phenotype during functional studies with CAFs, as the heterogeneity found in the tumour microenvironment is rapidly lost in cultured cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5599d9b2c874b1616cccf8809445b7ed868d0a94" target='_blank'>
              Patient derived cancer-associated fibroblasts from non-small cell lung cancer undergo phenotypic drift in culture
              </a>
            </td>
          <td>
            Layla Mathieson, Phoebe Jones, Lilian Koppensteiner, Liam Neilson, David A. Dorward, Richard A O'Connor, Ahsan R. Akram
          </td>
          <td>2025-07-10</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 3593


 Background:
 Variation in response to radiotherapy for the treatment of rectal cancer is likely due to heterogeneity in the tumour microenvironment. However, to date, no reliable predictive biomarkers of response are in clinical use and the mechanisms underlying response are unknown. Tertiary lymphoid structures (TLS), which are ectopic lymphoid aggregates found in the tumour microenvironment, have been linked to response to immunotherapy, but little is known about their role in radiotherapy. Here, we aimed to explore the potential of lymphocytes and tertiary lymphoid structures as predictive biomarkers of response to radiotherapy and profile the tumour immune microenvironment in the context of response to radiotherapy.
 Methods:
 For this study, we accessed pre-treatment biopsies from 20 rectal cancer patients with known pathological response to long-course chemoradiotherapy (LCCRT). We selected regions of interest based on immunohistological identification of tumour and lymphocytic infiltrate in formalin-fixed paraffin-embedded tissue. We performed targeted proteomic profiling of 87 immuno-oncology proteins using the Nanostring GeoMx Digital Spatial Profiler to quantify protein expression with spatial resolution within regions of interest, including TLSs, in the tumour microenvironment.
 Results:
 Unsupervised clustering based on normalised protein expression showed a clear separation between the complete responders to LCCRT and all other tumours, and this separation is driven by differences in T cells within TLSs (CD3+). Differentially expressed proteins within CD3+ aggregates include depletion of the natural killer cell marker, CD56 and increased expression of the apoptosis marker, cleaved caspase 9. The distribution of TLS-tumour distance was also significantly different between response groups.
 Conclusions:
 The study highlights the role of TLSs in modulating the immunogenic landscape of the tumour microenvironment in rectal cancer, likely influencing the response to radiotherapy. Spatially resolved proteomic analyses identifies potential biomarkers for radiotherapy response and underscores the importance of profiling tumour-immune microenvironment complexity when stratifying patients for therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38e831504bf073567c720a00c7fb61ae0355f499" target='_blank'>
              Digital spatial profiling: Mapping tumor responses to radiotherapy in rectal cancer.
              </a>
            </td>
          <td>
            R. Purcell, Adèle Hegoburu, Mohammad Amer, Arielle Sulit, Frank Frizelle
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 6045


 Background:
 Spatial transcriptomics (ST) revealed conserved malignant leading edge (LE) and tumor core (TC) architectures in primary oral squamous cell carcinoma (OSCC) with potential for biomarker discovery. Spatial organization of tumor cells, as well as composition and prognostic significance of neighboring stromal cells in RM-HNSCC remain unknown.
 Methods:
 21 tumor biopsy samples (14 baseline, 7 paired on-treatment) from 14 ICB-naive RM-HNSCC patients (pts) treated with pembrolizumab in INSPIRE (NCT02644369) were profiled using 10x Visium. Spatial organization was refined by scoring LE and TC gene sets identified in OSCC (Arora and Bose et al. Nat Comm 2023). Malignant (2,671 spots) and nonmalignant (8,177 spots) subclusters were annotated, with the latter classified into five cell subtypes using canonical markers: tumor-associated macrophages (TAMs) (
 CD68
 ,
 CD14
 ,
 SCF1R
 ), regulatory stromal cells (reg) (
 KRT17
 ,
 COL10A1
 ,
 SRBP1
 ), plasma cells (
 CD38
 ,
 IRF4
 ,
 PRDM1
 ), T cells (
 CD3D
 ,
 CD3E
 ,
 PTPRC
 ), and cancer-associated fibroblasts (CAFs) (
 FAP
 ,
 COL1A1
 ,
 PDGFRB
 ). Neighborhood analyses compared normalized counts of stromal cells adjacent to LE and TC, accounting for variations in cell density and sampling differences. A signature was built through
 k
 -means clustering of the five cell subtypes. Pts were stratified into high/low signature-score groups using the median cutoff and tested for association with progression-free survival (PFS).
 Results:
 Spatial organization revealed conserved malignant subclusters (C0 and C1) in 19/21 samples from 13 pts (11 non-responders). Top C0 genes were
 COL21A1
 ,
 S1PR3
 ,
 LIFR
 , and
 ZEB1
 . Top C1 genes were
 KRT6B
 ,
 KRT6C
 ,
 KRTDAP
 , and
 LCN2
 . Pathway analysis predicted activation of cell cycle and glycoprotein 6 in C0, and keratinization and neutrophil degranulation in C1. Comparative expression of OSCC-related gene sets revealed LE correlation with C0, and TC correlation with C1 (both
 p
 < 0.0001); stronger overlap was seen with the latter highlighting TC as a more conserved feature in HNSCC. Among non-responders, dominant communication patterns in LE and TC included claudins, cadherins, WNT, and IL-6, linked to cell adhesion, migration/invasion, and immune evasion. Signature generated by neighborhood analysis was enriched in TAMs and T cells, depleted in CAFs and reg near LE and TC, while plasma cells were depleted near LE but enriched near TC. Pts with high signature scores (6/13) exhibited improved PFS compared to low scores (7/13), median PFS 6.0 months [95% CI, 2.3-NA] versus 1.9 months [95% CI, 1.8-NA] (p = 0.059).
 Conclusions:
 To our knowledge, this is the first report using ST analysis to characterize LE and TC architectures in RM-HNSCC, along with heterogeneous neighboring stromal cells with prognostic potential for ICB. Ongoing cohort expansion will elucidate the clinical significance of these findings.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbba0db9ac1ea532c151a014a6c99f0d19f8be7a" target='_blank'>
              Spatial transcriptomics analysis to predict response to immune checkpoint blockade (ICB) in recurrent or metastatic head and neck squamous cell cancer (RM-HNSCC).
              </a>
            </td>
          <td>
            Grégoire Marret, Jinsu An, Lucas Penny, Ben X. Wang, Azin Sayad, Anna Spreafico, Enrique Sanz Garcia, A. Hansen, Helen Chow, Scott V. Bratman, Pinaki Bose, Lillian L. Siu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor development is mainly marked by the gradual transformation of cells that acquire capacities such as sustained growth signaling, evasion of growth suppression, resistance to cell death, and induction of angiogenesis, achieving replicative immortality and activating invasion and metastasis. How different epigenetic alterations like m1A, m5C, and m6A contribute to tumor development is a field that still needs to be investigated. The immune modulators, CD70, CD80, and TIGIT, mainly regulate T-cell activation and consequently the immune evasion of tumors. Here, we explored the presence and the potential consequences of RNA modifications in these regulators in pan-cancer. Our findings highlight the critical role of the m6A, m5C, and m1A in regulating CD70, CD80, and TIGIT across multiple solid tumors. By combining epitranscriptomics data with functional enrichment and survival modeling, we show that RNA modification enzymes not only modulate immune-related gene expression but also serve as potential biomarkers for patient prognosis. By constructing a robust four-gene prognostic signature involving YTHDF3, RBM15B, IGF2BP2, and TRMT61A, we demonstrate that RNA modification profiles can accurately stratify patients into risk groups with distinct overall survival outcomes. The performance of this model across eight cancer types underscores the translational promise of epitranscriptomic markers in both mechanistic understanding and personalized oncology. Altogether, our study bridges the gap between the mechanistic regulation of immune checkpoints and their clinical utility, offering novel insights into how the epitranscriptome can be leveraged to improve cancer prognosis and potentially enhance immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d03e20bb526dd9e6903a8377f001f578b9d1d1d" target='_blank'>
              Epitranscriptomics Regulation of CD70, CD80, and TIGIT in Cancer Immunity
              </a>
            </td>
          <td>
            C. Rigopoulos, Marios Gkoris, Ilias Georgakopoulos-Soares, I. Boulalas, A. Zaravinos
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb1152ca394367d1b31e6130a4964a702584963" target='_blank'>
              Multiscale Modeling Uncovers Macrophage Infiltration and TNF-α Signaling Networks for Targeting in Inflammatory Breast Cancer Tumor Emboli
              </a>
            </td>
          <td>
            Pritha Pai, Christophe Van Berckelaer, S. V. van Laere, Alexandra Bennion, Theresa Charity, Jinming Yang, F. Bertucci, Peter van Dam, Gregory M Palmer, Shannon McCall, Luc Y. Dirix, Naoto T. Ueno, G. Devi
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Abstract Objective We aim to investigate the spatiotemporal dynamics of intervertebral disc (IVD) cell subpopulations in IVD degeneration (IVDD). Methods To gain combined spatial and transcriptomic insights into IVDD, we employed both spatial transcriptomic sequencing (stRNA‐seq) and single nucleus RNA sequencing (snRNA‐seq) in a rat puncture‐induced IVDD model. The findings were verified in rat and human IVD by immunostaining and qRT‐PCR. Tamoxifen‐administered PdgfraCreERT2;R26tdTomato mice were adopted to track platelet‐derived growth factor receptor alpha (Pdgfra) positive cells. Results Puncture response regions were revealed on day 1 post‐puncture, for which oxidative stress emerged as a prominent pathway in a Stress Zone consisting of lipocalin‐2 (Lcn2)+ annulus fibrosus (AF) cells (AFC), which propagated and migrated into nucleus pulposus (NP), playing a key role in delivering injury signals and triggering pathological processes, including ferroptosis, fibrosis, and immune reactions. In the NP, Collagen 3‐high (Col3hi) NP cells (NPC) were another induced population demonstrating a fibrochondrocyte‐like phenotype and high epithelial–mesenchymal transition activation, an important pathway involved in tissue fibrosis. Crucially, lineage tracing experiments in PdgfraCreERT2;R26tdTomat mice revealed the significant migration and proliferation of Pdgfra+ AFCs from the AF into the NP following puncture. These findings provide direct evidence that both Pdgfra+ AFCs and Col3hi NP cells may contribute to NP fibrosis. Conclusion Puncture‐induced oxidative stress in a stress zone is the primary reaction playing an important role in initiating IVDD. Several puncture‐induced cell subpopulations were identified, including Lcn2+ AFC, Col3hi NPC, and Pdgfra+ AFC. Lcn2+ AFC plays a pivotal role in connecting oxidative stress with other pathological processes. Our results clarified the dual origin of Pdgfra+ cells, highlighting the contribution of AF‐derived cells to the NP during degeneration and emphasizing the complexity of cellular changes underlying NP fibrosis. Further investigation into the specific contributions of Pdgfra+ cells from different origins to fibrosis is warranted. Key points Puncture induced oxidative stress in a Stress Zone is the primary reaction in initiating IVDD. Puncture induced several IVD cell subpopulations, including Lcn2+ AFC, Col3hi NPC and Pdgfra+ AFC. Lcn2+ AFC plays a pivotal role in connecting oxidative stress with other pathological processes. Pdgfra+ cells in the NP derived from both Pdgfra+ AFC and Col3hi NPC, highlighting dual origin of NP fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811048891df0168757ff714b7c9e073ef14b40a7" target='_blank'>
              Single nuclear‐spatial transcriptomic sequencing reveals distinct puncture‐induced cell subpopulations in the intervertebral disc of a rat model
              </a>
            </td>
          <td>
            Guoyan Liang, Jing Tan, Chong Chen, Yuying Liu, Yongyu Ye, Xiaolin Pan, Qiujian Zheng, Yunbing Chang, Feng-Juan Lyu
          </td>
          <td>2025-06-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cisplatin is the primary chemotherapeutic agent for osteosarcoma. However, a significant proportion of patients develop resistance post-treatment, leading to disease recurrence and presenting profound clinical challenges. To understand the mechanisms underlying osteosarcoma recurrence and cisplatin resistance, particularly from the tumor microenvironment perspective, we consolidated numerous single-cell RNA sequencing datasets, offering an encompassing insight into the osteosarcoma microenvironment. When juxtaposing scRNA-seq with bulk RNA-seq data, we observed a strong correlation between high DCUN1D5 expression in osteosarcoma and patient survival. This gene amplifies osteosarcoma’s anti-apoptotic, invasive, stem-cell-like traits and PI3K/AKT/GSK3β pathway phosphorylation and fosters cisplatin resistance. Subsequent research revealed that cisplatin-resistant osteosarcoma cells excrete DCUN1D5-rich exosomes, facilitating the maturation of osteoclast precursors. Excessive osteoclast activity is a pivotal contributor to osteosarcoma recurrence and resistance. Given these insights, DCUN1D5 is a promising therapeutic target for osteosarcoma recurrence and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21215b3f5ca618360a5a84033d00f16e0330e332" target='_blank'>
              Single-cell analysis links DCUN1D5 to immune remodeling and cisplatin resistance in recurrent osteosarcoma
              </a>
            </td>
          <td>
            Xin Wu, Woding Deng, Qiangqiang Zhao, Jianbin Xiong
          </td>
          <td>2025-07-08</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies.We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration.Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration.Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background DNA methylation plays a crucial role in cancer development and progression and has been linked to genetically and clinically distinct tumor classes, including IDH-mutated and IDH-wildtype adult-type diffuse gliomas. Here, we identify a CpG-island methylator phenotype (CIMP) that characterizes the receptor tyrosine kinase 2 (RTK2) subtype of IDH-wildtype glioblastoma. Results This RTK2-CIMP affects genomic locations and cell functions distinct from those of IDH mutation-associated IDH-CIMP and suppresses the expression of its target genes. The RTK2-CIMP-region chromatin is characterized by a combination of repressive and activating marks, including polycomb-associated H3K27me3 and enhancer-associated H3K4me1, consistent with DNA methylation-mediated silencing of genes with bivalent-state promoters in neural progenitor cells. Functionally, RTK2-CIMP affects neuronal lineage genes and is significantly associated with astrocyte-like glioblastoma, suggesting that RTK2-CIMP is an epigenetic signature of the astrocyte-like cell state. Furthermore, we demonstrate that RTK2-CIMP can be induced by genetic manipulation in glioblastoma cells. Conclusions Our results suggest that RTK2-CIMP is a key contributor to cell-state plasticity in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03670-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94bee397b61a1f7011617f82ad34a983c326fca3" target='_blank'>
              A new IDH-independent hypermethylation phenotype is associated with astrocyte-like cell state in glioblastoma
              </a>
            </td>
          <td>
            Ana Luisa Costa, Daria Doncevic, Yonghe Wu, Lin Yang, Ka-Hou Man, Anna-Sophie Spreng, Hannah Winter, M. Fletcher, B. Radlwimmer, Carl Herrmann
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Recently, contrary to attacking cancer cells, the tumor microenvironment (TME) with genomic stability and vulnerable nature has emerged as promising therapeutic targets in hepatocellular carcinoma (HCC). Within TME ecosystem, tumor-associated macrophages (TAMs) play a pivotal role in tumor evasion and progression of HCC. However, their clinical and therapeutic implications remain unexplored.Utilizing a large-scale sc-RNA seq dataset, a landscape of HCC cellular ecosystem was depicted. Based on previous literature, an effectively differential TAMs subset classification was identified. Gene variations was extracted through trajectory analysis and then unsupervised clustering was conducted within RNA-seq data. Subsequently, survival analysis, specific pathway enrichment as well as hub regulatory network analysis were performed. Additionally, the immune cell infiltration and genomic variations were evaluated between clusters. Drug sensitivity and underlying therapeutic molecular were also explored. Through multiple immunofluorescence, our findings were verified.Herein, integrating single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq data, we established a novel TAM classification system based on mutually exclusive SPP1 and FOLR2 signatures. According to the TAM trajectory genes, unsupervised clustering stratified HCC into three distinct clusters. Cluster 3 (C3), which is characterized by metabolic dysregulation and immunosuppressive TME, exhibited the poorest prognosis among the three groups. Hub network analysis of C3 further indicated its characteristic dysregulation of liver-specific metabolism. SPP1 was identified as a key signature of C3, which contributed to suppressing the infiltration of CD8+ T cells. Therapeutic evaluation revealed that C3 were sensitive to chemotherapy and tyrosine kinase inhibitors, while those C1 and C2 were more suitable for immunotherapy. Drug screening identified potential therapeutic compounds for each cluster.This study redefines the heterogeneity of TAMs beyond the M1/M2 paradigm, linking the TAMs trajectory genes to HCC patient stratification. SPP1 blockade emerged as a strategy for counteracting immunosuppression, and cluster-specific therapies may optimize the management of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d85b5a9a5d2466c00caeea207b03b8145b6f159" target='_blank'>
              SPP1+ tumor-associated macrophages define a high-risk subgroup and inform personalized therapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Wei-Xuan Xu, Yanjun Ye, Jia-Lin Chen, Xin-Ying Guo, Chen Li, Juan Luo, Lin-Bin Lu, Xiong Chen
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968347e5e4a23139233feb67cc1b20faf82efda9" target='_blank'>
              Modeling Bistable Dynamics Arising from Macrophage-Tumor Interactions in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Hwayeon Ryu, Susanna Röblitz, K. Larripa, A. Frank
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Approximately 70% of breast cancer (BC) diagnoses are estrogen receptor positive (ER+) with ∼40% of ER+ BC patients presenting de novo resistance to endocrine therapy (ET). Recent studies identify the tumor microenvironment (TME) as having a key role in endocrine resistance in which adipose-derived stem cells (ASCs) play an essential role in cancer progression. Prior studies have indicated that ASC characteristics such as age and BMI may play a role in cancer progression. Unfortunately, most studies on ASC-BC cross talk have relied on established two-dimensional (2D) culture systems or the use of conditioned media that cannot replicate the complexity of the three-dimensional (3D) environment. This study used a microfluidic droplet trapping array and thiol–acrylate (TA) hydrogel scaffold to co-culture ER+ BC cells and ASCs as individual 3D spheroids (single culture) or organoids (co-culture) in a single device. Endocrine response was interrogated in both spheroids and organoids through the evaluation of proliferation following treatment with the selective estrogen receptor degrader (SERD) fulvestrant (ICI 182 780) followed by 17β-estradiol (E2). Terminal immunostaining for the proliferation marker (Ki67) was performed to evaluate how the presence of ASCs from different donor backgrounds (age and BMI) can modulate endocrine response. Results demonstrated that organoids containing two model ER+ cell lines (MCF7 and ZR-75) exhibited enhanced Ki67 expression even in the presence of ICI, suggesting a role for ASCs in cancer progression and endocrine resistance. Data clustering and classification algorithms were employed to categorize cellular behavior based on Ki67 expression and spheroid area to identify distinct clusters with high (H), intermediate (I), and low (L) Ki67 expression. Machine learning further stratified the data and revealed the direct effects of ASCs on Ki67 expression as well as how donor-specific features influenced ASC-driven changes in the TME. Notably, ASCs from an aged donor (>50) with lower BMI (<30) were able to enhance Ki67 expression even in the presence of endocrine therapy, while younger (<40) donors substantially enhanced Ki67 expression in the absence of both ICI and E2. Together, this study demonstrates the utility/development of a biomimetic culture system that recreates heterogenic 3D ER+ tumors through the co-culture of cancer cells with ASCs. This system provided insight into cell-extrinsic factors that govern ER+ breast cancer heterogeneity and response to endocrine therapy can be gained.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36b26f9d2f1fc166db1b577850e1e6422e08d444" target='_blank'>
              Droplet microfluidics integrated with machine learning reveals how adipose-derived stem cells modulate endocrine response and tumor heterogeneity in ER+ breast cancer
              </a>
            </td>
          <td>
            B. O. Ortega Quesada, Calley Chauvin, Elizabeth Martin, Adam Melvin
          </td>
          <td>2025-06-13</td>
          <td>Lab on a Chip</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bb1b241042af13953303a559606bd29dd3c0c78" target='_blank'>
              Differentiation status determines tumorigenicity and immunogenicity of cancer cells
              </a>
            </td>
          <td>
            Yang Liu, Chen Wang, Jing Li, Ying Cao
          </td>
          <td>2025-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c42ccef135de9d53a5d5b1a6abe8bae988944a3" target='_blank'>
              Histo-LCM-Hi-C reveals the 3D chromatin conformation of spatially localized rare cells in tissues at high resolution
              </a>
            </td>
          <td>
            Yixin Liu, Min Chen, Xin Liu, Zeqian Xu, Xinhui Li, Yan Guo, D. Czajkowsky, Zhifeng Shao
          </td>
          <td>2025-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Simple Summary Pancreatic ductal adenocarcinoma (PDAC) is a deadly disease with limited treatment options. This study focuses on the protein mesothelin (MSLN), which is highly expressed in over 89% of clinical PDAC specimens and helps cancer cells grow and avoid death. This study aims to determine how MSLN prevents cancer cells from entering a state called senescence, where they stop growing. The results show that silencing MSLN gene expression in PDAC cells induced senescence phenotypes, including growth arrest and an increased expression of senescence markers (P53, P21waf1, and P16ink4a), along with higher IL-8 production. Mechanically, high DNA damage/DNA repair activities associated with low MSLN expression may activate the senescence pathway in pancreatic cancer cells. Our findings could lead to better treatment options for pancreatic cancer, potentially improving outcomes for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d8c00a68fbcf392c73d6d5fa7de5c7e46f7d55" target='_blank'>
              Mesothelin-Associated Anti-Senescence Through P53 in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Dongliang Liu, Jian-ming Lu, Changyi J. Chen, Q. Yao
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a high-mortality cancer characterized by its aggressive, treatment-resistant phenotype and a complex tumour microenvironment (TME) featuring significant hypoxia. Bulk transcriptomic analysis has identified the “classical” and “basal-like” transcriptional subtypes which have prognostic value; however, it is not well-established how microenvironmental heterogeneity contributes to these transcriptional signatures. Here, we exploited the TRACER platform to perform single cell transcriptome analysis of organoids at specific spatial locations to explore the effect of oxygen and other cell-generated microenvironmental gradients on organoid heterogeneity. We found that the microenvironmental gradients present in TRACER significantly impact the distribution of organoid transcriptional phenotypes and the enrichment of gene sets linked to cancer progression and treatment resistance. More significantly, we found that microenvironmental gradients, predominantly oxygen, drive changes in the expression of classical and basal-like transcriptional subtype gene signatures. This work suggests that hypoxia contributes to determining transcriptional subtypes in PDAC tumour cells independent of additional cells in the TME and broadly highlights the importance of considering microenvironmental gradients such as oxygen in organoid-based studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-98344-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d7b936c3248bb437b55ed176049e4ec9a8d43e" target='_blank'>
              Analysis of an engineered organoid model of pancreatic cancer identifies hypoxia as a contributing factor in determining transcriptional subtypes
              </a>
            </td>
          <td>
            Natalie Landon-Brace, Simon Latour, Brendan T. Innes, Michelle Nurse, Jose L. Cadavid, I. Co, Cassidy M Tan, Ferris Nowlan, Sibil Drissler, F. Notta, H. Jackson, Gary D Bader, A. McGuigan
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Background: Tumor-infiltrating lymphocyte (TIL) enrichment in primary breast cancers is predictive of treatment response, and specific CD4 and CD8 subsets are associated with neoadjuvant chemotherapy (NAC) and immunotherapy response. Spatially defined T-cell receptor (TCR) evolution patterns in resection tissues associated with NAC response and survival have not been described. Methods: To define such patterns, we applied high-throughput TCR profiling to pre- and post-NAC tissues and associated immune dynamics with tumor evolution and clinical outcomes. We generated TCR repertoires of 29 primary breast cancers and 768 spatially catalogued, post-NAC specimens from matched cases (median 24 post-treatment samples/case) using Sequence Affinity capture & analysis By Enumeration of cell-free Receptors (SABER) applied to tissue-based Cancer Personalized Profiling by Deep Sequencing (t-CAPP-Seq). To profile each primary tumor’s genomic and immunologic response to NAC, we designed personalized oligonucleotide panels using pretreatment whole exome sequencing and a fixed panel of recurrently mutated genes to capture emergent alterations, as well as a comprehensive panel of TCR-β regions. Leveraging information from fragmented TCRs in FFPE tissues, we spatially enumerated TCR clonotype count and composition in each post-NAC sample and compared the relative abundance of clones shared with each pretreatment tumor. For a subset of cases, we enumerated TILs and tumor cellularity in post-NAC samples (N = 119). To interrogate clonotype composition, we computed Jaccard similarity indices between each pre- and post-NAC sample pair. We associated post-NAC clonal abundance and clonotype composition with tMRD burden and genomic composition. Finally, we associated TCR repertoire dynamics with quantitative tMRD assessment and survival. Results: TIL density was strongly correlated with the presence of post-NAC residual disease (R=0.94, p<0.001 for histologic invasive carcinoma cellularity; R=0.6, p<0.001 for mean tMRD allele frequency (AF)). Because TILs are defined in the presence of histologic residual disease, we defined tMRD-TILs by TCRs sequenced in tMRD positive tissues, which demonstrated greater repertoire diversity than non-tMRD TCRs (greater ShannonE and inverse Simpson diversity indices, p=.007 and p=1.3e-5, respectively), suggesting a spatially defined tumor neoantigen response. Within cases with low tMRD burden (mean AF ≤3%), which experienced longer progression-free survival than those with high tMRD burden (p=0.034), we observed greater TCR clone count in the post-NAC tissue (p=.012), greater shared clone count with the pretreatment tumor (p=5.7e-8), and greater similarity of the post-NAC clonotypes to the pretreatment tumor (p=5.1e-6, Wilcoxon test of Jaccard indices). Presence of a pretreatment CSPP1 lesion was associated with decreased TCR repertoire diversity in chemotherapy-treated resection tissues (ShannonE index, p=0.00056), and CSPP1-mutated tMRD was associated with inferior PFS (HR 23.45, p=.0007) Conclusions: By spatially defining changes in the genomic and immunologic microenvironment following NAC, we found superior molecular response is correlated with preservation of the clonal relationship between tMRD-TILs and the primary tumor with simultaneous generation of a rich TCR repertoire. Ongoing investigations involve computationally predicting neoantigen targets of clonally dominant tMRD-TILs through NAC for in vivo validation, representing an avenue for refining selection of molecularly targeted adjuvant therapies.
 Citation Format: Julia Ransohoff, Mia A. Carleton, Sofia Miron Barroso, Gregory Bean, Alisha Maltos, James Ford, George Duran, Michael Khodadoust, Melinda L. Telli, Ash A. Alizadeh, David M. Kurtz. Spatially defining T-cell clonal evolution in post-neoadjuvant breast cancer and association with molecular residual disease [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-02-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ce621b1b12255c322372611f29d2f188749ed1" target='_blank'>
              Abstract P1-02-19: Spatially defining T-cell clonal evolution in post-neoadjuvant breast cancer and association with molecular residual disease
              </a>
            </td>
          <td>
            J. Ransohoff, Mia Carleton, Sofia Miron Barroso, Gregory Bean, Alisha M Maltos, James M. Ford, George E Duran, Michael S Khodadoust, Melinda L. Telli, A. Alizadeh, David M. Kurtz
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 4155


 Background:
 The tumor microenvironment (TME) that surrounds pancreatic ductal adenocarcinoma (PDAC) is a multi-faceted and dynamic ecosystem in which stromal fibroblasts communicate with cancer cells to mediate tumor growth, metastasis, and chemotherapy resistance. It is well recognized that both cancer-associated fibroblasts (CAFs) and its non-cellular, fibrotic components within the TME can foster a protumorigenic environment for PDAC. However, we still lack a comprehensive understanding of the precise mechanisms in which this dense, fibrotic matrix can help drive malignant behaviors. Here, we reveal a novel mechanism in which the zinc-finger transcription factor GLI2 regulates type 1 collagen expression within CAFs and how the soluble variant of this collagen promotes irinotecan chemoresistance.
 Methods:
 We leveraged transcriptomic data from The Cancer Genome Atlas, International Cancer Genome Consortium, and Clinical Proteomic Tumor Analysis Consortium to evaluate GLI2 expression and stromal content of human PDAC tumors through bulk RNA-sequencing deconvolution. Using single-nucleus RNA sequencing of human PDAC tumors, we validated the association of GLI2 expression and stromal matrix constituents in CAFs. Chromatin immunoprecipitation assays in human CAFs confirmed GLI2 binding at the COL1A1 promoter. Through RNAi-based inactivation of GLI2, we determined how loss of GLI2 impacts regulation of type 1 collagen. Additionally, we conducted MTT assays to assess tumor viability in response to irinotecan treatment.
 Results:
 Transcriptomic analysis revealed that GLI2 is highly enriched in CAFs and strongly correlated with stromal fibrosis compared to other non-tumor cell constituents within the TME. We have shown that GLI2 directly binds to the promoter of COL1A1, a key component of type 1 collagen, in CAFs and regulates its transcription in a manner dependent on TGFβ1 signaling. Interestingly, PDAC tumors exposed to type 1 collagen show increased expression of pro-tumorigenic pathways involved in inflammation, EGR signaling, cytokine-receptor interactions, and irinotecan resistance. We further validate that human PDAC cells pre-treated with collagen can confer chemoresistance to irinotecan with viability assays.
 Conclusions:
 Taken together, our study demonstrates a novel mechanism in which GLI2 regulates the secretion of collagen within CAFs, which in turn can enable PDAC to acquire resistance to standard-of-care treatments. These findings highlight not only the oncogenic functions for CAFs and their fibrotic secretome, but also open new avenues in which therapeutic targeting of the TME may provide clinical benefit for PDAC patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3a2065caeb9cc2540ae0418d70044e4a839709" target='_blank'>
              Functional role of GLI2 in cancer-associated fibroblasts for modulation of the fibrotic tumor microenvironment within pancreatic cancer.
              </a>
            </td>
          <td>
            John Y. Kwon, Renzo E Vera, Maite G. Fernadez-Barrena, Jose M. Falero, Roberto G. Chaparro, Matthew D. Ross, M. D. Toruner, Murat Toruner, Ezequiel J Tolosa, Luciana L Almada, Huocong Huang, Rolf A. Brekken, Martin E Fernandez-Zapico
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Gastroenteropancreatic neuroendocrine tumors (GEP-NETs) are a family of tumors that arise throughout the gastrointestinal tract. These tumors are heterogeneous, with complex clinical symptoms and tumor behaviors, and demonstrate rising incidence rates worldwide. In addition to their nature, GEP-NETs possess limited diagnostic and therapeutic options, which results in poor survival rates for patients with metastatic tumors. Given these findings, a further analysis of these tumors’ biology is needed to determine new therapeutic strategies. The tumor microenvironment (TME) consists of several residual cell populations and non-cellular components whose altered behavior creates a tumor-supportive niche. Studies from other cancers demonstrate the TME’s significance in tumor initiation, progression, and spread. In this review, we discuss efforts to characterize the TME in GEP-NETs. Preliminary studies of the immune system in GEP-NETs have led to several major clinical trials, with limited success. Efforts to target signaling crosstalk between cancer-associated fibroblasts, vascular endothelial cells, and tumor cells has led to major discoveries and multiple approved therapies. Finally, alterations to the extracellular matrix may lead towards an improved understanding of GEP-NET development, behavior, and improved detection methods. While research has rapidly expanded our knowledge within the last decade, further work is needed to bring our understanding of the GEP-NET TME in line with other rare cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5602e7167401420da4dc63b32444d080749f374d" target='_blank'>
              The Role of the Tumor Microenvironment in Gastroenteropancreatic Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Srujana V. Yellapragada, Steven D. Forsythe, James P. Madigan, Samira M. Sadowski
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e21547


 Background:
 Resistance to targeted therapies remains a challenge in metastatic melanoma. The tumor microenvironment (TME) influences cancer progression and drug resistance, making it a promising therapeutic target. However, the lack of human in vitro 3D models that mimic the melanoma-TME crosstalk in a realistic 3D environment has limited the development of such therapies. To address this, we generated integrated skin-TME-melanoma organoids (mTMEOs) that incorporate key melanoma TME components and can serve to investigate and pharmacologically target the TME-mediated mechanisms of tumor progression in vitro.
 Methods:
 Primary human keratinocytes, melanocytes, fibroblasts, and pre-adipocytes were co-aggregated with GFP-labeled melanoma cells (SK-MEL-28 or A375) to form melanoma-skin organoids (mSOs), recapitulating melanoma anatomy, demonstrated via immunostainings. To generate the mTMEOs, mSOs were embedded in fibroblast-laden collagen gels, simulating the dermal stroma. mTMEOs were imaged in fluorescence using the Vireo Multi-Camera Array Microscope (MCAM) every other day, and a custom ImageJ-based pipeline was developed to track labeled GFP signals within the gels, allowing quantification of the number, size, and distance traveled of the invading melanoma cell clusters.
 Results:
 mSOs recapitulated key skin features, including a layered epidermis and dermal-hypodermal core, as well as melanoma-specific traits like epidermal spread, atypical melanocytes, and external migration. mTMEOs showed radial-to-vertical growth phase transition, enabling tumor invasion into the dermis-like stroma. Using the fluorescence images from the Vireo MCAM, we generated a Euclidean map that allowed the measurement of melanoma migration distances from the closest mSO. The analysis indicated that the presence of stromal fibroblasts in the surrounding gels did not influence melanoma invasion rates, but enhanced migration distances and led to fewer but larger melanoma cell clusters when compared to cell-free collagen gels. These findings align with literature suggesting that, while dermal invasion may be triggered by interactions with other cell types, stromal fibroblasts may play a tumor-supportive role by increasing melanoma spreading and shifting toward collective cell migration, associated with more aggressive metastases.
 Conclusions:
 Our novel mTMEO model replicates melanoma TME anatomical and pathological features, which can be altered by modifying individual organoid’s components. mTMEOs and the subsequent analysis workflow provide a versatile and tunable platform for studying TME-driven mechanisms of tumor progression and for drug development. By incorporating patient-derived cells, these constructs have the potential to serve as a personalized drug screening tool to improve patient outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f610ff07fea748c377877c2d34e8d7786bf477f" target='_blank'>
              Development of skin-integrated melanoma organoids to study the tumor microenvironment-mediated mechanisms of melanoma progression and metastasis in vitro.
              </a>
            </td>
          <td>
            Gemma Nomdedeu-Sancho, Nicholas Edenhoffer, Anastasiya Gorkun, John Bechtel, Jed Doman, Cecilia R. Schaaf, Natalie Alvarez, A. Bègue, Mark Harfouche, R. Horstmeyer, Anthony Atala, S. Soker
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Breast cancer invasion and subsequent metastasis to distant tissues occur when cancer cells lose cell–cell contact, develop a migrating phenotype, and invade the basement membrane (BM) and the extracellular matrix (ECM) to penetrate blood and lymphatic vessels. The identification of the mechanisms which induce the development from a ductal carcinoma in situ (DCIS) to a minimally invasive breast carcinoma (MIBC) is an emerging area of research in understanding tumor invasion and metastatic potential. To investigate the progression from DCIS to MIBC, we analyzed peritumoral collagen architecture using correlative scanning electron microscopy (SEM) on histological sections from human biopsies. In DCIS, the peritumoral collagen organizes into concentric lamellae (‘circular fibers’) parallel to the ducts. Within each lamella, type I collagen fibrils align in parallel, while neighboring lamellae show orthogonal fiber orientation. The concentric lamellar arrangement of collagen may physically constrain cancer cell migration, explaining the lack of visible tumor cell invasion into the peritumoral ECM in DCIS. A lamellar dissociation or the development of small inter fiber gaps allowed isolated breast cancer cell invasion and exosomes infiltration in the DCIS microenvironment. The radially arranged fibers observed in the peri-tumoral microenvironment of MIBC biopsies develop from a bending of the circular fibers of DCIS and drive a collective cancer cell invasion associated with an intense immune cell infiltrate. Type I collagen fibrils represent the peri-tumoral nano-environment which can play a mechanical role in regulating the development from DCIS to MIBC. Collectively, it is plausible to suggest that the ECM effectors implicated in breast cancer progression released by the interplay between cancer, stromal, and/or immune cells, and degrading inter fiber/fibril hydrophilic ECM components of the peritumoral ECM, may serve as key players in promoting the dissociation of the concentric collagen lamellae.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca42ff86f8ea9dc2e92ec4093986b47c8d0700cf" target='_blank'>
              Ultrastructural Changes of the Peri-Tumoral Collagen Fibers and Fibrils Array in Different Stages of Mammary Cancer Progression
              </a>
            </td>
          <td>
            Marco Franchi, Valentina Masola, Maurizio Onisto, Leonardo Franchi, Sylvia Mangani, V. Zolota, Z. Piperigkou, Nikos K. Karamanos
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Dysregulation of communication between cells mediates complex diseases such as cancer and diabetes; however, detecting cell–cell communication at scale remains one of the greatest challenges in transcriptomics. Most current single-cell RNA sequencing and spatial transcriptomics computational approaches exhibit high false-positive rates, do not detect signals between individual cells and only identify single ligand–receptor communication. To overcome these challenges, we developed Cell Neural Networks on Spatial Transcriptomics (CellNEST) to decipher patterns of communication. Our model introduces a new type of relay-network communication detection that identifies putative ligand–receptor–ligand–receptor communication. CellNEST detects T cell homing signals in human lymph nodes, identifies aggressive cancer communication in lung adenocarcinoma and colorectal cancer, and predicts new patterns of communication that may act as relay networks in pancreatic cancer. Along with CellNEST, we provide a web-based, interactive visualization method to explore in situ communication. CellNEST is available at https://github.com/schwartzlab-methods/CellNEST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dad0b937d47610c141e2f1541eaddc5e310fdba" target='_blank'>
              CellNEST reveals cell–cell relay networks using attention mechanisms on spatial transcriptomics
              </a>
            </td>
          <td>
            F. Zohora, Deisha Paliwal, Eugenia Flores-Figueroa, Joshua Li, Tingxiao Gao, F. Notta, Gregory W. Schwartz
          </td>
          <td>2025-06-06</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Gastric cancer (GC) is among the most lethal human malignancies with limited treatment options. Cell-cell interactions within the tumor microenvironment (TME) can promote tumor growth, yet their therapeutic value has not been fully explored. Here, bulk RNA-seq, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics (ST) were integrated to analyze the heterogeneity of GC microenvironment. Tumor-specific GREM1+ fibroblasts and SPP1+ macrophages were significantly enriched in GC tissues and were involved in immunosuppression, inflammation regulation, and tumor progression. We then indicated that GREM1+ fibroblasts and SPP1+ macrophages were positively correlated in 12 independent GC datasets and validated their close localization by multiplex immunohistochemistry staining and spatial transcriptomics. Patients with both high GREM1+ fibroblasts and SPP1+ macrophages exhibited significantly shorter OS and showed enrichment of tumor-associated pathways. Our results demonstrated the complex interactions between GREM1+ fibroblasts and SPP1+ macrophages, which may serve as a potential therapeutic target for future treatment of GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad0abacc337130966d37d3605e0104e32183302" target='_blank'>
              Multi-omics analyses reveal interactions between GREM1+ fibroblasts and SPP1+ macrophages in gastric cancer
              </a>
            </td>
          <td>
            L. Qiu, Xiao Zhao, Sheng Yao, Yang Fei, Y. Gong, Zishan Zhou, Shunchang Jiao, Jianming Xu
          </td>
          <td>2025-06-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Invasive lobular carcinoma (ILC) is the second most common histological subtype of breast cancer after invasive ductal carcinoma of no special type (NST), accounting for 10–15% of diagnoses. Despite the myriad molecular, histological and clinical differences between ILC and NST tumors, patients are treated in the same way, and although prognosis initially is good, ILC patients have poorer long-term outcomes. Understanding the differences between these two subtypes and identifying ILC-enriched therapeutic targets is necessary to improve patient care. Methods Human and mouse cancer-associated fibroblasts (CAFs), ILC cell lines and patient-derived organoids were used for in vitro and in vivo studies, including western blotting, migration, organotypic invasion assays and dissemination in zebrafish embryos. RNASeq was used to identify CAF and interleukin-6 (IL-6)-derived gene signatures. Bioinformatic analysis of public databases and immunohistochemical of human tumor microarrays was carried out. Results We identified IL-6 as a paracrine CAF-derived factor that activates Signal-Transducer-and-Activator-of-Transcription-3 (STAT3) in human and mouse ILC models. Analysis of human breast tumors showed that the IL-6/JAK/STAT3 pathway is enriched in ER + ILC compared to ER + NST. A 42-gene CAF dependent IL-6 gene signature and 64-gene consensus IL-6 gene signature were generated and were significantly enriched in ER + ILC, with many of the genes overexpressed in ILC tumors. IL-6 treatment suppressed downstream estrogen signaling and also led to the acquisition of a more mesenchymal-like phenotype associated with increased migration and invasion. Finally, IL-6 treatment significantly increased ILC cell dissemination following injection into zebrafish embryos. Conclusions CAF-derived IL-6 drives paracrine activation of the IL6/JAK/STAT3 signaling pathway which is enriched in ILC. This leads to the acquisition of pro-tumorigenic phenotypes, highlighting the pathway as a potential therapeutic target in ILC. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02074-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ff0735a08b68cfe9b8184d5f4b41c0bb6234a9" target='_blank'>
              Cancer-associated fibroblast driven paracrine IL-6/STAT3 signaling promotes migration and dissemination in invasive lobular carcinoma
              </a>
            </td>
          <td>
            Esme Bullock, Aleksandra Rozyczko, Sana Shabbir, Ifigenia Tsoupi, Adelaide I J Young, J. Travnickova, Laura Gómez-Cuadrado, Zeanap Mabruk, Giovana Carrasco, Elizabeth Morrow, K. Pennel, Pim Kloosterman, J. Houthuijzen, Jos Jonkers, L. Avalle, Valeria Poli, R. Iggo, Xue Xiao, Jingjing Guo, Xuan Zhu, Elizabeth Mallon, Joanne Edwards, E. E. Patton, Valerie G. Brunton
          </td>
          <td>2025-07-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a highly aggressive and treatment-resistant subtype of castration-resistant prostate cancer (CRPC) that often emerges during progression under androgen-receptor (AR) pathway inhibition. While lineage plasticity in cancer cells has been recognized as a key mechanism of resistance, the role of the tumor microenvironment in driving this transition remains unclear. Among its cellular components, vascular endothelial cells can undergo endothelial-mesenchymal transition (EndoMT), a phenotypic shift associated with tumor progression and fibrosis. Here, we investigated whether EndoMT contributes to NEPC development. Human umbilical vein endothelial cells (HUVEC) were induced to undergo EndoMT using IL-1β and TGF-β2, and are hereafter referred to as EndoMTed HUVEC. EndoMTed HUVEC promoted neuroendocrine features and functional changes in LNCaP cells. Transcriptome analysis revealed marked upregulation of granulocyte-macrophage colony-stimulating factor (GM-CSF) in EndoMTed HUVEC. Neutralization of GM-CSF signaling using mavrilimumab, a monoclonal antibody targeting the GM-CSF receptor alpha (CSF2RA), and siRNA-mediated CSF2RA knockdown both suppressed the neuroendocrine phenotype and STAT3 signaling of LNCaP cells. Conversely, GM-CSF stimulation alone reproduced these changes. Enzalutamide-treated LNCaP cells secreted IL-1β and TGF-β2, which in turn triggered EndoMT, suggesting a reciprocal loop. These findings indicate that anti-androgen therapy may inadvertently promote NEPC through a paracrine loop involving tumor-derived cytokines and endothelial GM-CSF secretion, highlighting EndoMT as a microenvironmental driver of treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1c2f9c2cb4f661e5c27835323d6945b89f72e3" target='_blank'>
              Endothelial-Mesenchymal Transition in Tumor Microenvironment Promotes Neuroendocrine Differentiation of Prostate Cancer.
              </a>
            </td>
          <td>
            Takumi Kageyama, Manabu Kato, Shiori Miyachi, Xin Bao, Sho Sekito, Y. Sugino, Shinichirou Higashi, Takeshi Sasaki, Kouhei Nishikawa, Yasuhiro Murakawa, Masatoshi Watanabe, Takahiro Inoue
          </td>
          <td>2025-07-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer is the most prevalent form of malignant tumor that frequently metastasizes to axillary lymph nodes (LNs). Nonetheless, the precise mechanisms underlying alterations in the tumor microenvironment (TME) in LN metastasis in breast cancer remain poorly understood. Single-cell RNA sequencing of 28 LN samples from 23 patients is performed, and a comprehensive landscape of the entire ecosystem is generated. Ten major cell types are identified, with the subclusters of each major cell type exhibiting diverse characteristics. Furthermore, multiple signatures are collected to evaluate the key components of the subclusters using multi-omics methodologies. This study finds that myCAFs may hasten LN metastasis, and observed a notable increase in APOE+ macrophages and a higher proportion of exhausted CD8+ T cells, contributing to the immunosuppressive TME. Moreover, cancer cells in metastatic lesions exhibited diverse expression patterns linked to proliferation, metastasis, oxidative phosphorylation, hypoxia, and interferon responses. Using multi-omics approaches and experimental validations, it determines that GLO1 can promote lymphatic angiogenesis, metastasis, and inhibit the proteasomal degradation of GSS, thereby maintaining intracellular glutathione (GSH) and reactive oxygen species (ROS) balance. Collectively, the study offers novel perspectives on the microenvironment remodeling of breast cancer LN metastases, suggesting that GLO1 may be a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f92d409a651118984d17df6fe54f30f3acdf6d65" target='_blank'>
              Paracrine Orchestration of Tumor Microenvironment Remodeling Induced by GLO1 Potentiates Lymph Node Metastasis in Breast Cancer.
              </a>
            </td>
          <td>
            Jindong Xie, Wenjian Liu, Xinpei Deng, Huan Wang, Xueqi Ou, Xin An, Min-yi Situ, An-Kui Yang, Chuan Peng, Rongfang He, Yi Xie, Hailin Tang, Yuman Chen, Jie-Ying Liang, Ruonan Shao, Weikai Xiao, Shaoquan Zheng
          </td>
          <td>2025-06-10</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Many studies have shown that pancreatic cancer is one of the cancers with extremely high mortality. The poor prognosis and lack of early diagnostic methods remain major challenges in the treatment of this cancer. In this article, in order to better detect the occurrence of pancreatic cancer, tools such as monocle3, singleR, harmony in R, and scanpy in Python were used to analyze the cells and genes of pancreatic cancer tissues in mice. By analyzing the data, the following results were obtained: T-cells in healthy PBMCs exhibited broader spatial dispersion than in PDAC tissues, suggesting tumor-driven immune surveillance impairment, while PDAC-associated macrophages displayed expanded distribution linked to pro-tumorigenic functions such as COL1A1-mediated ECM remodeling; Pseudotemporal trajectory analysis revealed myeloid progenitor bifurcation into monocytes/macrophages, with PDAC macrophages showing epigenetically silenced cytotoxic pathways such as suppressed GZMA/NKG7 and enhanced ribosomal biogenesis; Tissue-specific markers such as LCN2 in healthy and CTRB1/AMY2A in PDAC) and spatial co-localization of macrophages/tumor cells highlighted NOP53 as a dual-function hubinhibiting PI3K-AKT while activating p53and SPP1 as a paradoxical regulator of metastasis and antitumor immunity; Differential expression and GO enrichment analyses identified ribosomal biogenesis and cytoplasmic translation as PDAC-enriched pathways, contrasting with suppressed stress responses. Our spatial transcriptomic profiling further resolved elevated NOP53, CFB, and SPP1 expression gradients in PDAC tissues, proposing these as diagnostic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab94aa8cc2272f4a29c06caf0dcfe1338e28a0a0" target='_blank'>
              Spatiotemporal Transcriptomic Dissection of Tumor-Associated Macrophage Heterogeneity and Dual-Function Molecular Nodes in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Mengfei Li
          </td>
          <td>2025-06-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79b9ccad6349bc919fa38a076a3195bd6f23bab" target='_blank'>
              Loss of colonic fidelity enables multilineage plasticity and metastasis.
              </a>
            </td>
          <td>
            Patrizia Cammareri, Michela Raponi, Yourae Hong, C. Billard, Nat Peckett, Yujia Zhu, Fausto D Velez-Bravo, Nicholas T Younger, Donnchadh S Dunican, S. Ö. Pohl, Aslıhan Bastem Akan, Nora J Doleschall, John Falconer, M. White, Jean Quinn, K. Pennel, Roberta Garau, Sudhir B. Malla, P. Dunne, Richard R Meehan, Owen J. Sansom, J. Edwards, Malcolm G Dunlop, Farhat V N Din, Sabine Tejpar, Colin W. Steele, K. B. Myant
          </td>
          <td>2025-06-04</td>
          <td>Nature</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is a leading cause of cancer-related deaths worldwide. Despite advancements in treatment, prognosis for patients with advanced stages remains poor. Metabolic reprogramming in the tumor microenvironment, particularly abnormal glycolysis, plays a crucial role in immune evasion and treatment response. We collected nine single-cell datasets to create a single-cell atlas of CD8 + T cells from 89 NSCLC patients, revealing ten distinct states of these cells. We employed a multimodal data analysis approach, integrating bulk transcriptomics, single-cell transcriptomics, spatial transcriptomics, and proteomics. Using 117 machine learning models, we identified key genes associated with NSCLC metastasis. Notably, the StepCox[forward] + Lasso model was instrumental in pinpointing key genes that significantly impact disease prognosis. Our analysis revealed that LTB + LDHA + CD8 + T cells have a distinct metabolic and immune phenotype, characterized by enhanced glycolysis and elevated lactate production. This not only facilitates tumor cell migration and invasion but also impairs the cytotoxic function of CD8 + T cells. Furthermore, our machine learning models identified four key genes significantly associated with NSCLC metastasis: TBCD, PTPRC, LDHA, and ACTR2. Of these, high LDHA expression was strongly linked to poorer responses to immunotherapy and a higher risk of therapy resistance. LTB + LDHA + CD8 + T cells also reduced antitumor immune responses by inhibiting the secretion of effector molecules like GNLY. Additionally, elevated LDHA expression was associated with reduced CD8 + T cell infiltration, which further promotes tumor immune evasion. This study highlights the heterogeneity of CD8 + T cells in NSCLC, emphasizing the unique role of the LTB + CD8 + Tn subpopulation in metastasis. LDHA is identified as a critical key gene with a significant impact on immunotherapy outcomes, presenting a potential therapeutic target. These insights offer new biomarkers and targeted strategies for personalized immune therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e078cc67ce9052ada65f40295ba6296864dfb9f6" target='_blank'>
              Metabolic alterations driven by LDHA in CD8 + T cells promote immune evasion and therapy resistance in NSCLC
              </a>
            </td>
          <td>
            Hao Wen, Panpan Zhang, Juan Zhao, Yakui Liu, Lei Wan, Haoran Li, Jun Yi, Xinqiang Li
          </td>
          <td>2025-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e14562


 Background:
 T cell-based immunotherapies have demonstrated great success for the treatment of many solid tumors, but not all tumor types and patients respond well or durably. Reasons for these discrepancies are unclear. High grade serous ovarian cancer (HGSOC) is an example of a deadly disease that has exhibited poor responses to T cell targeted therapies, like checkpoint inhibitors. Thus, it would be an ideal model for studying other parts of the immune ecosystem that could contribute to anti-tumor immunity. Tumor infiltrating B cells (TIL-Bs) are being increasingly studied in HGSOC because of their association with improved prognosis, especially when differentiated into germinal center (GC) B cells. However, the true effector functions of B cells, including antigen presentation, cytokine production, and antibody production, in ovarian and other solid tumors remains understudied. Here, we study tumor antigen-specific B cell responses in a mouse model of HGSOC.
 Methods:
 We have engineered a murine HGSOC cell line ID8 p53 -/- brca2 -/- (ID8) to express a membrane bound mode antigen, hen egg lysozyme (mHEL), at two different affinities to primary HEL-specific B cells (MD4 B cells) from MD4 transgenic mice. We use flow cytometry to characterize HEL-specific B cell responses to ID8 mHEL-expressing cells
 in vitro
 . We use a syngeneic mouse model of HGSOC in which we intraperitoneally inject ID8 cells (with or without mHEL) and use flow cytometry to identify immune phenotypes of B and T cells in tumor tissue, spleen, lymph nodes, and ascites. We also use ELISA to measure HEL-specific antibody production in the ascites and serum of these mice.
 Results:
 We demonstrate that MD4 B cells can upregulate canonical B cell receptor signaling elements in response to stimulation with ID8 mHEL
 in vitro
 , and that these responses can be titrated based upon the affinity of the mHEL presented. We also show that the syngeneic ID8 p53 -/- brca2 -/- mouse model produces tumors in the ovaries and peritoneum that are enriched for B cells, and that the ascites is specifically enriched for antigen specific B2 B cells (with a much smaller innate like B1 compartment as compared to the peritoneal lavage of non-tumor controls). We plan to inject host mice with ID8 mHEL tumors and compare tumor progression in mice that subsequently receive intraperitoneal adoptive transfer of MD4 HEL-specific B cells versus not. We also plan to compare the phenotypes of B and T cells in the tumor, spleen, lymph nodes, and ascites of these mice, as well as the serum and ascites levels of HEL antibodies.
 Conclusions:
 We have generated a model antigen expressing ovarian cancer cell line that produces antigen-specific B cell responses
 in vitro
 , and have shown that there is an enrichment in antigen-specific B cells in these ovarian tumors and ascites; future
 in vivo
 studies will elucidate the important effector functions of TIL-Bs in ovarian cancer and other solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/107a1451e84aeccb51bc914d82c8c35407885897" target='_blank'>
              Dissecting B-cell responses in high grade serous ovarian cancer.
              </a>
            </td>
          <td>
            Ramya Parameswaran, James Mueller, Katherine C Fuh, J. Zikherman
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 e14585


 Background:
 Immune checkpoint blockade (ICB) targeting PD-1/PD-L1 is crucial in treating breast cancer. Despite clinical success, the mechanisms by which ICB therapies reshape the tumor microenvironment (TME) and enhance anti-tumor activity remain unclear. Recent studies reveal that anti-PD-1 therapies broadly alter TME cells beyond PD-1+ T cells, highlighting extensive cell-cell communication (CCC) as a key factor. Understanding CCC changes in response to anti-PD-1 therapy can illuminate its mechanisms of action (MOA) and TME dynamics. This study applies causal inference methodology to investigate CCC between T cells and non-T cells in the TME of breast cancer patients receiving anti-PD-1 therapy.
 Methods:
 We analyzed single-cell RNA-seq data from 31 breast cancer patients pre- and on-treatment with anti-PD-1 (pembrolizumab) reported by Bassez et al., 2021. We identified differentially expressed genes (DEGs) across cell types. We applied the instrumental variable method to identify causal relationships between signals in T-cell and non-T-cell DEGs to uncover treatment-induced CCC. We further searched for ligand-receptor pairs mediating CCC and identified gene expression modules (GEMs) regulated by these ligand-receptors. Finally, we constructed a CCC network from T to non-T cells.
 Results:
 Anti-PD1 treatment induced broad gene expression changes in diverse cell populations. Major T cell pathways influenced by anti-PD-1 therapy include NF-κB, interferon-γ and interleukins. CD4
 +
 and CD8
 +
 exhausted cells, expressing high levels of
 PDCD1
 (encoding PD-1), exhibited distinct activated pathways. We identified the CCC network from T cells to non-T cells via ligand-receptor interactions. For example, anti-PD1 treatment activated cellular stress, apoptosis, and pro-inflammatory cytokine signaling pathways in T cells, altering the expression of a GEM that included
 TSC22D3
 and
 TXNIP
 . This initiated signaling to myeloid cells via
 RPS19
 -
 C5AR1
 , leading to NF-κB activation. CD4+ exhausted T cells primarily signaled to monocytes, enhancing helper functions to recruit and activate monocytes, thereby promoting immune regulation and amplification. In contrast, CD8+ exhausted T cells primarily interacted with macrophages, intensifying cytotoxic responses that facilitated effective tumor-cell killing.
 Conclusions:
 Our analyses provide insights into the dynamic interplay of cells within the TMEs during anti-PD-1 therapy. These findings could facilitate the identification of new biomarkers for predicting heterogeneous treatment responses to anti-PD-1 regimens, potentially enhancing the design and customization of immunotherapeutic strategies for breast cancer patients. Finally, this study demonstrated the utility of causal inference methodologies for mechanistic studies of CCC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74715a58ffb2aed0b55ed5b7dec461d0fe94b624" target='_blank'>
              Use of causal modeling to uncover cell-cell communication dynamics in the tumor microenvironment during anti-PD-1 therapy in breast cancer patients.
              </a>
            </td>
          <td>
            Aodong Qiu, Han Zhang, Joseph Ramsey, Boyang Sun, Bryan Andrews, Kun Zhang, Gregory F. Cooper, Lujia Chen, Xinghua Lu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a major malignancy threatening women’s health, with high-grade squamous intraepithelial lesions playing a critical role in the progression toward CC. Exploring the molecular characteristics of epithelial cells (EPCs) as high-stage intraepithelial neoplasia evolves into CC is essential for the development of effective targeted drugs for cervical cancer. Single-cell RNA sequencing technology can fully understand the immune response at each molecular level, providing new ideas and directions for the precise treatment of CC. Methods Single-cell RNA sequencing was employed to comprehensively map EPCs characteristics. The differentiation trajectory of EPCs was inferred using Slingshot, while enrichment analysis highlighted the biological functions of EPCs. Cellchat visualized cell-cell interactions, and SCENIC was used to infer transcription factor regulatory networks in EPCs. CCK-8, colony formation, and EDU experiments were used to verify cell proliferation changes. Scratch assays and transwell assays were used to verify cell migration and invasion. Results A distinct EPCs subpopulation with high TOP2A expression was identified, predominantly originating from tumor tissues. This subpopulation exhibited disrupted mitosis and cell cycle regulation, along with features of high proliferation, high energy metabolism, and matrix plasticity. It played a key role in shaping the tumor microenvironment via the LAMC1-(ITGA3-ITGB1) signaling pathway. FOXM1, a key transcription factor in this cell subpopulation, significantly inhibited the proliferation and invasion of cervical cancer cells. Conclusion Through in-depth analysis of EPCs, this study provides promising insights and potential therapeutic targets for precision targeted treatment strategies for CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dee7774b4a74cbb3d72e5ad05823a4742cd929c3" target='_blank'>
              Epithelial cells with high TOP2A expression promote cervical cancer progression by regulating the transcription factor FOXM1
              </a>
            </td>
          <td>
            Weisha Sun, Lu Chen, Xiaolin Feng
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cervical cancer is the third most common malignancy among Chinese women in both incidence and mortality. Its progression is closely linked to complex interactions among immune cells within the tumor microenvironment (TME). As key components of the immune landscape, different T cell subsets play diverse and dynamic roles in shaping tumor immunity. This review provides a comprehensive overview of the roles of various T cell subsets in the TME of cervical cancer, with a focus on their distribution, functional heterogeneity, dynamic balance, and variations across different pathological subtypes and disease stages. We also highlight the intricate crosstalk between T cells and other immune cells in the TME and discuss recent advances in T cell-related immunotherapies for cervical cancer, including immune checkpoint inhibitors and HPV-targeted vaccines. By elucidating the roles of distinct T cell subsets and relevant immunotherapeutic approaches within the TME, this review provides insights into potential therapeutic targets and approaches for improving cervical cancer treatment and patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac934ce24f93b9f941a402b5e786abad8449989b" target='_blank'>
              T cell subsets in cervical cancer tumor microenvironment: advances and therapeutic opportunities
              </a>
            </td>
          <td>
            Weilin Guo, Lan Dai, Lihua Qiu
          </td>
          <td>2025-06-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (SCC) is known for its stepwise progression from healthy skin to premalignant actinic keratosis (AK), followed by a malignant transformation to SCC. Unfortunately, less attention has been paid to changes in gene expression in the tumour microenvironment during this process. We retrospectively selected early‐stage cutaneous SCC tissue samples containing both invasive and premalignant portions and conducted a spatial transcriptomic experiment using a NanoString GeoMx Digital Spatial Profiler (DSP). First, we selected invasive and premalignant regions of interest (ROIs) for each tissue. We then compared the gene expression patterns between the two portions (invasive versus premalignant) of the three segments: tumour cells, immune cells and fibroblasts, in each ROI. As a result, early‐stage cutaneous SCC tissue samples from 17 patients were selected for this study. We identified 29, 14 and 15 differentially expressed genes (DEGs) between the invasive and premalignant portions of the tumour cells, immune cells and fibroblasts, respectively. The top three genes with the highest absolute log2 fold‐change were CCDC88C, GJD3 and COMP in tumour cells; SVEP1, TSLP and PPP2R5C in immune cells; and SPAG6, PPP1CA and CCDC68 in fibroblasts. Notably, several genes, such as COMP, SVEP1 and SPAG6, have been linked to the development and function of cancer‐associated fibroblasts. Functional enrichment analysis revealed that several pathways were altered in tumour and immune cells. In conclusion, distinctive changes in gene expression patterns were observed as AK progressed to SCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1212cb843dc2d0cad33b53da437b7c501e408d51" target='_blank'>
              Spatial Transcriptomics Shows a Distinctive Tumour Microenvironment in the Invasive Versus Premalignant Portion of Early Cutaneous Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jeong-An Gim, Chungyeul Kim, Hyun Jyung Oh, Ko Eun Kim, J. Jeon, Aeree Kim, Y. Baek
          </td>
          <td>2025-06-01</td>
          <td>Experimental Dermatology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background: Rhabdomyosarcoma (RMS) is the most common type of soft-tissue sarcoma in children, and it remains a challenging cancer with poor outcomes in high-risk and metastatic patients. This study reports the use of epidermal growth factor receptor (EGFR)-targeted chimeric antigen receptor (CAR) natural killer (NK) cells in combination with radiotherapy as a novel immunotherapeutic approach for RMS treatment. Methods: Primary human NK cells from healthy donors were engineered using lentiviral transduction to express a cetuximab-based EGFR-specific CAR. The ability of the engineered NK cells to lyse RMS cells was then assessed in vitro in RMS monolayers and spheroids, as well as against chemotherapy-resistant and primary patient-derived RMS cells. Migratory properties of NK cells were observed in a subcutaneous RMS xenograft model using in vivo imaging, and the efficacy of EGFR-CAR NK cells in combination with localized fractionated radiotherapy was analyzed. Results: Primary human EGFR-CAR NK cells demonstrated enhanced cytotoxicity against multiple RMS cell lines in both two-dimensional culture and three-dimensional spheroid models. Furthermore, EGFR-CAR NK cells were highly efficient against chemotherapy-resistant RMS cells and patient-derived samples. Importantly, EGFR-CAR NK cells also exhibited improved tumor homing compared with non-transduced NK cells in an in vivo RMS xenograft model. Notably, the combination of EGFR-CAR NK cell therapy with fractionated radiotherapy further enhanced NK cell infiltration into the tumor and reduced tumor growth. Conclusion: This study provides a proof-of-concept for EGFR-CAR NK cells as a promising immunotherapy for RMS, particularly when combined with radiotherapy to overcome barriers of solid tumors. This combinatorial approach may hold potential to improve outcomes for patients with RMS and other EGFR-expressing malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4d74df17e2be9c2ebc7d4856cc275fc0f705a7a" target='_blank'>
              Precision targeting of rhabdomyosarcoma by combining primary CAR NK cells and radiotherapy
              </a>
            </td>
          <td>
            L. Reindl, L. Jalili, T. Bexte, Sabine Harenkamp, S. Thul, S. Hehlgans, A. Wallenwein, F. Rothweiler, J. Cinatl, Martin Michaelis, H. Bonig, E. Gradhand, Meike Vogler, F. Rödel, Winfried S. Wels, E. Ullrich
          </td>
          <td>2025-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Compelling evidences have manifested that breast cancer cells prefer to metastasize to certain distant organs, including brain, lung, bone and liver. According to the canonical “seed and soil” theory, this prominent biological behavior, termed as metastatic organotropism, involves intricate interactions between breast cancer cells (the “seeds”) and specific residents in the tumor microenvironment (the “soil”), initiating from pre-metastatic niche formation to metastatic outgrowth. Recently, multifaceted heterogeneity of tissue-specific macrophages (TSMs) and their roles played in organotropic metastases of breast cancer are incrementally unveiled. Herein, we decipher multiple diversities of TSMs, including evolvement, profiles, functions and metabolic characteristics under different polarization states. Further, we elaborate on bidirectional effects of TSMs on metastatic organotropism of breast cancer (both to the “seeds” and “soil”), and unearth underlying signaling pathways based on updated mechanistic researches. Lastly, we compile a series of clinical trials, hoping to illuminate promising TSM-targeting therapies against breast cancer organotropic metastases. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ed79b9316a9fed3f14b3bb67e4e61ce4c40709" target='_blank'>
              Heterogeneous tissue-specific macrophages orchestrate metastatic organotropism of breast cancer: implications for promising therapeutics
              </a>
            </td>
          <td>
            Cenzhu Wang, Pinchao Fan, Yu Zhou, Meican Ma, Haowei Zhong, Lei Liu, Qin Chen, Kun Xu
          </td>
          <td>2025-06-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) has a poor survival rate due to late detection. PDAC arises from precursor microscopic lesions, termed pancreatic intraepithelial neoplasia (PanIN), that develop at least a decade before overt disease--this provides an opportunity to intercept PanIN-to-PDAC progression. However, immune interception strategies require full understanding of PanIN and PDAC cellular architecture. Surgical specimens containing PanIN and PDAC lesions from a unique cohort of five treatment-naïve patients with PDAC were surveyed using spatial-omics (proteomic and transcriptomic). Findings were corroborated by spatial proteomics of PanIN and PDAC from tamoxifen-inducible KPC (tiKPC) mice. We uncovered the organization of lymphoid cells into tertiary lymphoid structures (TLSs) adjacent to PanIN lesions. These TLSs lacked CD21+CD23+ B cells compared to more mature TLSs near the PDAC border. PanINs harbored mostly CD4+ T cells with fewer Tregs and exhausted T cells than PDAC. Peri-tumoral space was enriched with naïve CD4+ and central memory T cells. These observations highlight the opportunity to modulate the immune microenvironment in PanINs before immune exclusion and immunosuppression emerge during progression into PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6b6b93e0faedffa7b598bebfd58275c143f057" target='_blank'>
              Spatial Proteomics and Transcriptomics Reveal Early Immune Cell Organization in Pancreatic Intraepithelial Neoplasia.
              </a>
            </td>
          <td>
            Melissa R Lyman, Jacob T. Mitchell, Sidharth Raghavan, Luciane T. Kagohara, Amanda L. Huff, Saurav D Haldar, Sarah M. Shin, Samantha Guinn, Benjamin Barrett, Gabriella Longway, Alexei Hernandez, Erin M Coyne, Xuan Yuan, Lalitya Andaloori, Jiaying Lai, Yun Zhou Liu, Rachel Karchin, Anuj Gupta, A. Kiemen, André Forjaz, Denis Wirtz, Pei-Hsun Wu, Atul Deshpande, Jae W. Lee, T. Armstrong, Nilofer S. Azad, Jacquelyn W. Zimmerman, Laura D. Wood, Robert A Anders, Elizabeth D. Thompson, Elizabeth M. Jaffee, E. Fertig, W. Ho, N. Zaidi
          </td>
          <td>2025-06-26</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Cancer progression remains a significant clinical challenge. Phenotypic adaptation by tumour cells results in disease heterogeneity, which drives treatment resistance and immune escape. T-cell immunotherapy, while effective at treating some cancer subtypes, can also fail due to limits on tumour immunogenicity or T-cell recognition. For example, one potential contributor to immune escape involves the density and alignment of the extracellular matrix (ECM) surrounding tumours, also known as tumour-associated collagen signature (TACS). However, the specific mechanisms by which aligned fibres contribute to decreased patient survival rates have not yet been decoupled. Here, we developed EVO-ACT (EVOlutionary agent-based cancer T-cell interaction), a two-dimensional agent-based modelling framework designed to investigate how different TACS architectures impact tumour evolution and dynamic interactions with CD8[Formula: see text] T cells. Our results highlight that TACS-driven modulation of T-cell dynamics, combined with phenotypic adaptation, such as epithelial-to-mesenchymal transition, underlies differences in tumour immunogenicity and the application of our model can successfully recapitulate clinically observed breast cancer survival trends.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195f55d30b50b8b9b55b6c384b5267c2b8395881" target='_blank'>
              Distinct evolutionary patterns of tumour-immune escape and elimination determined by extracellular matrix architectures.
              </a>
            </td>
          <td>
            Yijia Fan, Jason T. George
          </td>
          <td>2025-07-01</td>
          <td>Journal of the Royal Society, Interface</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36016009dd0f8056e799887b3ce94bc9a6296493" target='_blank'>
              Ovarian cancer, neutrophil hitchhiking, and NETs: unraveling their role in pathogenesis and management.
              </a>
            </td>
          <td>
            Shahrzad Amirlatifi, Shabnam Forouzin, Elahe Sadati, Somayeh Moradpanah, Ramina Fazeli, Paria Panahinia, Nasim Zarifi, Sania Rahimi
          </td>
          <td>2025-06-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e16171


 Background:
 Hepatocellular carcinoma (HCC), accounting for 70%-90% of primary liver cancers, often involves portal vein tumor thrombus (PVTT), a significant adverse prognostic factor linked to worsened liver function and treatment challenges. This study uses multi-omics analysis to investigate the tumor immune microenvironment in HCC with PVTT, focusing on the role of cancer-associated fibroblasts (CAFs) in PVTT formation and identifying potential therapeutic targets to improve clinical outcomes.
 Methods:
 We utilized nCounter technology, single-cell RNA sequencing (scRNA-seq), digital spatial profiling (DSP), and Olink proteomics to investigate intra-tumoral spatial heterogeneity from transcriptomics, proteomics, and immunology perspectives. Key molecules promoting PVTT formation were identified via differentially expressed genes (DEGs) analysis, spatial expression and single-cell analysis. Identified Acarbose as a potential drug for treating PVTT was conducted using 3D QSAR, pharmacophore modeling, and molecular dynamics simulations. The clinical potential of Acarbose targeting NID1 in PVTT treatment was further validated through cell proliferation assays, immune-functional mouse models, immunofluorescence, and real-world clinical data analysis.
 Results:
 A comprehensive multi-omics analysis of 85 samples from 47 HCC patients revealed that myofibroblast-associated cells (myCAFs) dominate the tumor microenvironment (TME) of PVTT, shaping a tumor immune-suppressive environment. The macrophage-to-myofibroblast transition (MMT) induced myCAF formation, establishing an immune barrier in the primary tumor. A cross-analysis of DEGs from DSP data and scRNA-seq identified Nidogen1 (NID1) as a key gene driving PVTT formation. NID1 was upregulated in MMT cells and myCAFs within the PVTT TME, contributing to immunosuppression and PVTT formation. Computational virtual screening identified acarbose as a potential therapeutic agent. When combined with tislelizumab, acarbose significantly enhanced T cell-mediated tumor cytotoxicity, inhibited tumor growth, and reduced NID1 expression, demonstrating superior efficacy compared to anti-PD-1 monotherapy, both in vitro and in vivo. Real-world data showed that long-term use of acarbose in diabetic HCC patients reduced NID1 levels and prevented PVTT formation, supporting acarbose as a promising strategy to target NID1 and inhibit PVTT.
 Conclusions:
 Our findings demonstrate the critical role of myCAFs in shaping the tumor immune-suppressive environment that promotes PVTT formation. Targeting NID1, a key driver of this process, with acarbose effectively disrupts the myCAF-mediated immune barrier and synergistically enhances the efficacy of immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e255ac4af9555248e208900e14562fa04e3faced" target='_blank'>
              Multi-omics investigation for the tumor immune microenvironment and spatial heterogeneity in hepatocellular carcinoma with PVTT: MyCAFs and NID1 in immune evasion.
              </a>
            </td>
          <td>
            Tao Han, Zhi Zhu, Heming Li, Kai Li
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The intricate tumor microenvironment (TME) poses a significant barrier to effective cancer immunotherapy, requiring innovative strategies to model and address its challenges. Traditional models, such as 2D cultures and animal studies, often fail to capture the TME's dynamic, multicellular, and spatially complex nature, limiting their predictive power for therapeutic outcomes. To overcome these limitations, this review examines innovative microphysiological systems (MPS) that enhance the understanding of tumor‐immune interactions and pave the way for more effective immunotherapeutic strategies. First the complex features of the TME and its key players are detailed, elaborating on their dynamic interplay with tumors. Importantly, it is highlighted how these components contribute to treatment resistance, offering crucial insights into therapeutic failures. Then, state‐of‐the‐art 3D in vitro organ‐on‐chip (OoC) models are presented that faithfully recapitulate the TME, incorporating patient‐derived tumors to enhance clinical relevance. These advanced systems not only overcome the limitations of traditional animal models and 2D cultures but also provide a robust platform for assessing and improving immunotherapeutic regimens. By bridging the gap between bench and bedside, MPS promises to accelerate the development of novel, more effective immunotherapies for solid tumors, potentially transforming cancer treatment in the near future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6697512c635b75cb87b2b32aa131010d9c074648" target='_blank'>
              Microphysiological Systems in Cancer Research: Advancing Immunotherapy through Tumor Microenvironment‐Integrated Organ‐On‐Chip Models
              </a>
            </td>
          <td>
            Fabiana Moresi, Francesco Noto, Jyothsna Vasudevan, Xavier Bisteau, Giulia Adriani, Andrea Pavesi
          </td>
          <td>2025-07-04</td>
          <td>Advanced Therapeutics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 e20099


 Background:
 Mesothelioma is a rare and aggressive cancer strongly associated with asbestos exposure. It is typically an incurable disease, with treatment focusing on palliative chemotherapy, radiation, and immunotherapy. Novel insights into the tumor microenvironment have provided a unique perspective on treatment options and prognosis. Resting natural killer (NK) cells are an inactive form of NK cells that are less cytotoxic than their activated counterparts. NK cells typically function by producing cytokines to inhibit tumor progression and releasing chemicals to destroy malignant cells. The inactive form of NK cells serves as a reservoir that can be rapidly recruited and activated in response to tumor-derived signals. Resting NK cells have been shown to play a role in the overall survival of mesothelioma, offering a potential avenue for innovative treatment strategies.
 Methods:
 RNA sequencing data from The Cancer Genome Atlas (TCGA) in mesothelioma patients was analyzed. Immune cell fractions in tumor samples were identified using CIBERSORT which is a computational tool used in estimating relative abundance of different cell types in a given tissue sample. Radiant by Shiny was utilized to demonstrate a linear regression between overall survival and various immune cell types. This analysis included memory T cells, B cells, plasma cells, NK cells, monocytes, macrophages, and dendritic cells, in both their active and inactive forms.
 Results:
 Natural killer cells were analyzed through linear regression and found to be independently associated with overall survival in malignant mesothelioma. The analysis yielded a p-value of 0.039 and a correlation coefficient of +164.598, indicating a positive relationship between the presence of inactive NK cells and overall survival in mesothelioma.
 Conclusions:
 The presence of inactive NK cells in the tumor microenvironment of malignant mesothelioma is associated with increased overall survival. While active NK cells have been linked to prolonged survival in other cancers, such as colorectal, renal, and squamous cell carcinomas, this study emphasizes the inactive form of NK cells in mesothelioma. Studies have found that gemcitabine, when used for mesothelioma, can increase the number of responding NK cells, thus highlighting the importance of said cells in the tumor microenvironment. Enhancing NK cell activity using cytokines such as IL-7, VEGF, or TNF-alpha may offer a novel therapeutic approach with a more favorable safety profile. However, significant challenges remain in managing immunotherapies and mitigating their potential consequences.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd093fb2d8b4bf0c539a56bd1b09a7c3f15dc102" target='_blank'>
              The implication of resting natural killer cells in mesothelioma.
              </a>
            </td>
          <td>
            Lorin Sallan, Morgan Kiryakoza, Paul F Hanona, Neil Jairath, George Howard
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Stem cell plasticity is crucial for rapidly proliferating tissues, such as the intestine, to cope with microenvironmental stress and maintain homeostasis. This framework may be employed by cancer stem cells (CSCs) to interact with the tumor microenvironment (TME) and give rise to drug resistance, tumor recurrence, and metastasis. Here, we consider colorectal cancer (CRC) as an example to review the technological and conceptual advances in microenvironmental regulation of cancer stemness. Some unique features of CRC, including reprogramming of stromal cells, bacterial niche alteration, and metabolic interplay, were summarized to illustrate how the TME affects the plasticity of CSCs. The key pathways and molecular mechanisms involved in metastasis and drug resistance are reviewed to obtain a deeper understanding of how CSCs may be manipulated for better outcomes in CRC treatment. These findings advance our knowledge of tumor heterogeneity and may provide insights into the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e85fc5484d5fb7be8f2f1af589f1c3b9e327f305" target='_blank'>
              Microenvironmental Regulation of Colorectal Cancer Stemness: Technological Advances, Molecular Basis, and Therapeutic Targets
              </a>
            </td>
          <td>
            Hui Zhao, Qilin Li, Jichao Qin
          </td>
          <td>2025-06-26</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e16331


 Background:
 Cholangiocarcinoma (CCA) describes a malignancy originating from the epithelial cells of the bile ducts. While rare, this malignancy often has an unfortunate outlook given its advanced stage at presentation. Development of CCA likely depends upon a gradual accumulation of genetic mutations. These mutations in-turn, transition precursor lesions, biliary intraepithelial neoplasia and intraductal papillary mucinous neoplasm, into the cancer that is CCA. The tumor microenvironment has been increasingly studied and likely plays a big role in the pathogenesis of CCA. A variety of cells contribute to said environment, ranging from regulatory T cells(Treg) to natural killer(NK) cells. The role of the NK cell is being increasingly studied in the context of CCA as it is one of the most abundant immune cells in the tumor microenvironment and likely plays a large role in the immune response to cancer cells.
 Methods:
 Our sample set included 36 patients from a publicly available RNA sequencing dataset from MD Anderson. RNA sequences were extracted and then input into a Stanford based application, CIBERSORT. This illustrates the impact different immune cells have on a tumor microenvironment. The immune cells included are the following: B-cells naïve, B-cells memory, plasma cells, CD8 T-cells, CD4 T-cells, T-cells follicular, T-cells regulatory, natural killer activated cells, natural killer resting cells, monocytes, macrophages (M0, M1, M2), activated dendritic cells, resting dendritic cells, activated mast cells, resting mast cells, eosinophils, and neutrophils. CIBERSORT uses immune cell fractions as the marker of immune cell impact. These fractions were then analyzed in a statistical tool, Radiant by Shiny, that computes a linear regression describing the relationship between immune cells and impact on overall survival.
 Results:
 Natural killer cells in the tumor microenvironment have a strong, positive impact on overall survival (p = 0.044). The coefficient of correlation was +198.
 Conclusions:
 Results of our analysis illustrate the strong and positive relationship that NK cells have on the overall survival of CCA. This is likely due to the NK cell’s ability to promote cell death through various pathways, such as the Fas-L and TNF apoptotic pathways. More specifically, it has been shown that cholangiocytes may have the ability to present lipid antigens to NK cells thus activating an alternate killing pathway making the NK cell a crucial player in the tumor microenvironment. This, coupled with being one of the most abundant cells in the tumor microenvironment in CCA, opens the door for the NK cell as an potential target for future treatment options. Understanding these specific interactions between NK cells and tumor cells proves to be essential for developing novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a1da82ab42c7f1091a653ec8bd73313150dda1f" target='_blank'>
              The tumor microenvironment in cholangiocarcinoma: Impact of natural killer cells on overall survival.
              </a>
            </td>
          <td>
            Morgan Kiryakoza, Lorin Sallan, Paul F Hanona, Neil Jairath, Daniel Ezekwudo, George Howard, I. Jaiyesimi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tumor-infiltrating B cells (TIBs) are the most important cell type involved in the immune response. TIBs display considerable intratumor heterogeneity due to genetic variation, epigenetic differences and transcriptional plasticity in the tumor microenvironment (TME). Owing to the unique anatomical location of CRC, the B cell subpopulation exhibits more extensive heterogeneity. Many studies have shown that TIBs have gradually become a key predictor of immunotherapy for malignant cancers, including CRC. TIBs have essential functions, including antigen presentation and antibody secretion, and they promote T-cell activation and myeloid chemotaxis. However, owing to the complex TME, TIBs not only promote the antitumor immune response but also inhibit the immune response. With the in-depth study of tumor-infiltrating T cells, tumor-associated myeloid cells and the interactions among these cells in the TME, the special role of immune cells in the TME has gradually become clear. However, the influence of TIBs in the TME and their interactions with nonimmune cells in the TME remain unclear. Here, we summarize the current progress in TIBs based on single-cell RNA sequencing in CRC in recent years, focusing on specific effector or regulatory characteristics of different B cell subclusters in the CRC TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/967a81ae3d15c27b261f054d18c946b7fcf49909" target='_blank'>
              The rules of different B cell subtypes in colorectal cancer: friends or foes?
              </a>
            </td>
          <td>
            Yuanchao Shi, Qianwen Luo, Jingwei Duan, Bo Tang, Quan-lin Guan
          </td>
          <td>2025-06-09</td>
          <td>Future oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 5085


 Background:
 Genomic mutations, including
 RB1
 ,
 PTEN
 , and HRR genes, associate with poor prognosis and treatment resistance in metastatic prostate cancer (mPC). mPC lineage states such as neuroendocrine prostate cancer (NEPC) also drive resistance and poor survival. Identifying the timing and association of genomic mutations with lineage states has been limited by the challenges of obtaining serial tumor biopsies. We report an integrated analysis of clinical next-generation sequencing (NGS) data, ctDNA, and mPC lineage state phenotypes with a novel high purity circulating tumor cell (CTC) RNA sequencing method.
 Methods:
 We collected 273 CTC samples from 117 patients with mPC in a prospective biomarker trial. 69 patients had clinical NGS from a recent tissue biopsy and 60 patients treated with PSMA
 LU
 . CTCs were purified via immunomagnetic capture on a microfluidic platform and analyzed via RNA-seq. We compared CTC lineage states, somatic mutation status and median overall survival (mOS).
 Results:
 Gene expression analysis of high CTC purity samples identified four CTC phenotypes: luminal A-like (LumA), luminal B-like (LumB), low proliferation (LP), and neuroendocrine (NE). Compared to patients with low CTC burden/purity (mOS NR), patients with LumA and LP phenotypes had similar survival ((LumA: mOS 13mo, p=ns, LP: mOS: 11.8mo, p=ns), while patients with LumB and NE phenotypes had shorter survival (LumB: mOS 6mo, HR 9.1[3.8-21.8], p<0.0001, NE: mOS 3.7mo, HR 11.8[2.4-57.2]), p=0.0019). mOS for patients with high-risk genomic mutations (
 RB1, PTEN, TP53
 ) and a LumA or LP lineage state was 12.7mo versus 4mo for patients with high-risk mutations and a LumB or NE lineage state (HR 3.59 [1.51-8.51], p=0.004). mOS was not reached for patients without high-risk mutations and a LumA or LP lineage state vs 8mo for patients without high-risk mutations but a LumB or NE lineage state (HR 19.7 [1.99-1.95], p<0.0001). Among patients treated with 177Lu-PSMA-617 in a prospective substudy (n=37), no patients had pretreatment CTC NE phenotype, but pre-treatment CTC LumB phenotype was associated with decreased rPFS (3.5mo vs 11.7mo, HR 4.8 [95% CI 1.4-16.1], p<0.005) and OS (7.6mo vs NR, HR 9.3 [95% CI 2.3-37.8], p<0.0005) compared to pretreatment LumA/LP/Low CTC burden phenotypes.
 Conclusions:
 Lineage states detected by high purity CTC RNAseq are prognostic for poor OS and decreased benefit from 177Lu-PSMA-617. The presence of high-risk genetic mutations including
 RB1
 is prognostic for poor OS and combined evaluation of high-risk genetic mutation status with lineage state phenotype identifies patients with the worst clinical outcomes that would benefit from treatment intensification and early disease monitoring for these more aggressive subtypes of mCRPC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65dbf8653d5dbf2b38b173aa38f375c13b4f7d3" target='_blank'>
              Integrated analysis of tissue genomic sequencing and high purity circulating tumor cell RNA sequencing for prostate cancer lineage states as a prognostic factor for survival and resistance to 177Lu-PSMA-617 in patients with metastatic castrate resistant prostate cancer.
              </a>
            </td>
          <td>
            M. Sharifi, J. Sperger, Amy K Taylor, Katharine E Tippins, J. Schehr, John M Floberg, C. Kyriakopoulos, H. Emamekhoo, S. Tagawa, A. Choudhury, A. J. Armstrong, D. Rathkopf, H. Beltran, Peter S. Nelson, S. Dehm, D. Kosoff, Xiao X Wei, Rana R McKay, Shuang Zhao, J. M. Lang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Melanoma is the most aggressive form of skin cancer, and despite significant therapeutic advances over the past decade, a substantial number of patients still progress to a fatal outcome. The initiation and progression of melanoma are strongly influenced by interactions between melanoma cells and other components of the tumor microenvironment (TME). In this review, we focus on the interplay between fibroblasts resident in the tumor microenvironment and tumor cells. In particular, we examine the molecular mechanisms through which melanoma cells induce the transformation of resident fibroblasts into their active form, known as cancer-associated fibroblasts (CAFs). We also explore the role of CAFs in shaping the melanoma microenvironment (MME) and in organizing the pre-metastatic niche, a specialized microenvironment that forms in distant organs or tissues to support the survival and expansion of metastatic melanoma cells. Finally, we discuss emerging therapeutic strategies aimed at disrupting the interactions between CAFs, melanoma cells, and other components of the tumor microenvironment to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d9f3ff9ff2ce42f0f728f6171249249cc7ed33f" target='_blank'>
              The Role of Fibroblasts in Melanoma Development: From Tumor Microenvironment Remodeling to Pre-Metastatic Niche Formation
              </a>
            </td>
          <td>
            Immacolata Belviso, Raffaele Pastore, Aldo Mileo, Emiliano Del Genio, Stefania Boccia, Stefano Palermi, C. Sellitto, M. L. Motti
          </td>
          <td>2025-06-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Zwilch Kinetochore Protein(ZWILCH) has been reported to prevent cells from prematurely exiting mitosis. However, the underlying mechanisms or involvement of ZWILCH in the tumor immune microenvironment in various cancers remain largely unknown. Methods Generalized dysregulation of ZWILCH was observed through the whole transcriptome analysis in this study. The spatial transcriptome analysis was utilized to identify expressed regions of ZWILCH. Next, cells that mainly expressed ZWILCH in the tumor microenvironment were determined using the single-cell transcriptome analysis. Also, the “cellchat” R package was applied to estimate the effect of ZWILCH on malignant cell communication. Combining multiple analytic approaches including GSEA, GSVA, KEGG enrichment analysis, and Aucell, with TCPA functional protein data, Genome-wide CRISPR screening, potential functions of ZWILCH and the pathways in which ZWILCH participated were thoroughly exploited. Univariate Cox regression analysis calculated the association between ZWILCH and cancer patients’ adverse outcomes. Results ZWILCH is universally highly expressed in tumors. The spatial transcriptome analysis showed that ZWILCH overexpression comes from the tumoral region or mixed tumoral region. At the single-cell level, ZWILCH is chiefly expressed by malignant cells and proliferative T cells. The expression of ZWILCH mRNA is positively correlated with cell proliferation, repair of DNA damage, and cell cycle score. Plenty of metabolic pathways are inhibited in patients with high expression of ZWILCH. Moreover, after ZWILCH knockout, a large number of cancer cell lines are stagnated, inhibited, or died. Additionally, the malignant cells with positive expression of ZWILCH have a stronger ability for cell communication. In short, ZWILCH is meant to be a risk factor for clinical outcomes of multiple tumors. Conclusions ZWILCH is a promising therapeutic target that influences patient prognosis by participating in cell proliferation, cell communication, and reshaping the tumor microenvironment across different cancers. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02981-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73f14c1c066485285bab172ec02a8c4271d565b9" target='_blank'>
              Zwilch kinetochore protein affects the prognosis of cancer patients by participating in cell proliferation, enhancing cell communication, and reshaping the tumor microenvironment
              </a>
            </td>
          <td>
            Long Yao, Lianpo Liu, Jinsong Wu, Yunlong Huang, Renquan Zhang, Haoxue Zhang
          </td>
          <td>2025-07-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1093733a8ade229255066e064b631c0ae1860e" target='_blank'>
              ImmunoFusion: A Unified Platform for Investigating RNA-seq-Derived Gene Fusions in Cancer and Immunotherapy
              </a>
            </td>
          <td>
            YanKun Zhao, Shixiang Wang, Shensuo Li, Minjun Chen, Su-Han Jin, U. Gaipl, Hu Ma, Jian-Guo Zhou
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by significant heterogeneity and a poor prognosis. M2 macrophages (M2) play a crucial role in the microenvironment of PDAC, yet their exact functional significance remains poorly understood. This study sought to explore the potential communication and clinical implications of M2 in PDAC. The differentiation pathway of M2 and their interaction with malignant PDAC cells were analyzed using single-cell RNA sequencing (scRNA-seq). Additionally, a prognostic signature related to M2 marker genes was constructed and analyzed using bulk RNA sequencing. Three distinct differentiation states of M2 were characterized, each exhibiting unique transcription factor expression profiles and signaling pathway activations. Potential intercellular communication between M2, PDAC malignant cells, and fibroblasts was initially observed at the level of computer analysis. The prognostic model was identified through the LASSO algorithm. Patients stratified by risk level displayed significant variations in prognostic indicators such as signaling pathway activity, gene mutation frequency, immune cell infiltration, and drug sensitivity. These findings are of course preliminary, and future in vitro/in vivo experiments will be needed to knock out these signature genes or ligand-receptor pairs of genes in macrophages to validate their role in macrophage development. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02838-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/273ead8bc63462bd01d5d5a63b43ec9833ef2540" target='_blank'>
              Establishment of M2 macrophage-related prognostic signature in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Hui Chen, Jingyang Yin, Xianxing Wang, Renpei Xia, Fanbo Qin, Zhuo Li, Jiali Yang, Changjiang Liu, Shixiang Guo, Huaizhi Wang
          </td>
          <td>2025-06-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Lung cancer remains the leading cause of cancer-related deaths globally. Emerging evidence indicates a potential link between blood cell perturbation responses and various diseases. Further investigation into the connection between these responses and lung cancer could provide valuable insights into its biological behavior and improve strategies for risk stratification. Methods This study employs two-sample mendelian randomization, incorporating lung cancer data from the IEU OpenGWAS project, as well as hematopoietic perturbation response data, to investigate the causal relationships between hematopoietic perturbation responses and lung cancer. This study also differentiated between subtypes: small cell lung carcinoma, adenocarcinoma, and squamous cell carcinoma. The primary analytical method was the inverse variance weighted (IVW) approach. Egger intercept analysis, abnormal MR-PRESSO test, Cochran Q, and leave one out analysis were also employed as multiple sensitivity analyses to assess the robustness of the results. Results 18 blood cell perturbation responses were significantly associated with lung cancer and its subtypes, including 10 protective factors and 8 risk factors. In addition, reverse Mendelian randomization analysis identified 12 blood cells with reverse causal relationships with cancer, comprising 11 inhibitory factors and 1 promoting factor. Conclusions Our findings demonstrate a potential causal relationship between lung cancer and blood cell perturbation responses, providing a new perspective for diagnosing and treating lung cancer. However, further studies are needed to elucidate the underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be6e652601f3c7d95cff2da1933b80ecaee483ff" target='_blank'>
              Unraveling the role of blood cell perturbation responses in lung cancer by Mendelian randomization
              </a>
            </td>
          <td>
            Yichao Huang, Xinjing Lou, Ziqing Han, Linyu Wu, Chen Gao
          </td>
          <td>2025-06-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The chemokine receptor CXCR4 is overexpressed in many cancers and contributes to pathogenesis, disease progression, and resistance to therapies. CXCR4 is known to form oligomers, but the potential functional relevance in malignancies remains elusive. Using a nanobody-based BRET method, we demonstrate that oligomerization of endogenous CXCR4 on lymphoid cancer cell lines correlates with enhanced expression levels. Specific disruption of CXCR4 oligomers reduced basal cell migration and prosurvival signaling via changes in the phosphoproteome, indicating the existence of constitutive CXCR4 oligomer-mediated signaling. Oligomer disruption also inhibited growth of primary CLL 3D spheroids and sensitized primary malignant cells to clinically used Bcl-2 inhibitor venetoclax. Given its limited efficacy in some patients and the ability to develop resistance, sensitizing malignant B cells to venetoclax is of clinical relevance. Taken together, we established a noncanonical and critical role for CXCR4 oligomers in lymphoid neoplasms and demonstrated that their selective targeting has clinical potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20f74abe38aca67f6abd3a0247a3f473e7c7e548" target='_blank'>
              Disruption of constitutive CXCR4 oligomers impairs oncogenic properties in lymphoid neoplasms.
              </a>
            </td>
          <td>
            Simon Mobach, N. Bergkamp, Ziliang Ma, M. Haselager, S. M. Anbuhl, Daphne Jurriens, Jelle van den Bor, Ziming Wang, C. Crudden, Jamie L Roos, C. V. Pérez Almería, Rick A. Boergonje, M. Lohse, R. Bosma, E. Eldering, M. Siderius, Wei Wu, M. Spaargaren, S. H. Tonino, A. Kater, M. Smit, R. Heukers
          </td>
          <td>2025-06-11</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Recent advances in spatially resolved transcriptomics (SRT) have provided valuable avenues for identifying cell-cell interactions and their critical roles in diseases. We introduce QuadST, a novel statistical method for the robust and powerful identification of cell-cell interactions and their impacted genes in single-cell SRT. QuadST models interactions at different cell-cell distance quantile levels and innovatively contrasts signals to identify interaction-changed genes, which exhibit stronger signals at shorter distances. Unlike other methods, QuadST does not require the specification of interacting cell pairs. It is also robust against unmeasured confounding factors and measurement errors of the data. Simulation studies demonstrate that QuadST effectively controls the type I error, even in misspecified settings, and significantly improves power over existing methods. Applications of QuadST to real datasets have successfully revealed biologically significant interaction-changed genes across various cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b626408bd7ae85c0b0622033213223c58fd9f9b9" target='_blank'>
              QuadST identifies cell-cell interaction-changed genes in spatially resolved transcriptomics data.
              </a>
            </td>
          <td>
            Xiaoyu Song, Yuqing Shang, Michelle Ehrlich, P. Roussos, Guo-Cheng Yuan, Pei Wang
          </td>
          <td>2025-06-23</td>
          <td>Genome research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Accurate estimation of malignant cell fractions in tissues plays a critical role in cancer diagnosis, prognosis, and subsequent treatment decisions. However, most currently available methods provide only point estimates, neglecting the quantification of uncertainties, which is essential for both clinical and research applications. This study introduces DeepDeconUQ, a deep neural network model developed to estimate prediction intervals for malignant cell fractions based on bulk RNA-seq data. This approach addresses limitations in current malignant cell fraction estimation methods by integrating uncertainty quantification into predictions of cancer cell fractions. DeepDeconUQ leverages single-cell RNA sequencing (scRNA-seq) data in conjunction with conformalized quantile regression to produce reliable prediction intervals. The model trains a quantile regression neural network to establish upper and lower bounds for cancer cell proportions, followed by a calibration step that refines these intervals to ensure both statistical validity (coverage probability) and discrimination (narrow intervals). Benchmark analyses indicate that DeepDeconUQ consistently surpasses existing methods, achieving high coverage accuracy with tight prediction intervals across simulated and real cancer datasets. The robustness of DeepDeconUQ is further demonstrated by its resilience to various gene expression perturbations. The DeepDeconUQ method is publicly accessible at https://github.com/jiaweih14/DeepDeconUQ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/256a6922307ee57d437c08f8da1f505d8a655601" target='_blank'>
              DeepDeconUQ estimates malignant cell fraction prediction intervals in bulk RNA-seq tissue
              </a>
            </td>
          <td>
            Jiawei Huang, Yuxuan Du, Kevin R. Kelly, Jinchi Lv, Yingying Fan, Jiang F. Zhong, Fengzhu Sun
          </td>
          <td>2025-06-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Cell type-specific transcriptional heterogeneity in embryonic mouse skin is well-documented, but few studies have investigated the regulatory mechanisms. Here, we present high-throughput single-cell chromatin accessibility and transcriptome sequencing (HT-scCAT-seq), a method that simultaneously profiles transcriptome and chromatin accessibility. We utilized HT-scCAT-seq to dissect the gene regulatory mechanism governing epidermal stratification, periderm terminal differentiation, and fibroblast specification. Results By linking chromatin accessibility to gene expression, we identify candidate cis-regulatory elements (cCREs) and their target genes which are crucial for dermal and epidermal development. We describe cells with similar gene expression profiles that exhibit distinct chromatin accessibility statuses during periderm terminal differentiation. Finally, we characterize the underlying lineage-determining transcription factors and demonstrate that ALX4 and RUNX2 are candidate transcription factors regulators of the dermal papilla lineage development through in silico perturbation analysis and CUT&Tag experiment. Conclusions Overall, HT-scCAT-seq represents a powerful tool for unraveling the spatiotemporal dynamics of gene regulation in single cells. Our results advance the understanding of embryonic skin development while providing a scalable framework for investigating regulatory mechanisms across diverse biological systems and disease contexts. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03652-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/747c145f394c82a8a3bbf273f6afa4e007611092" target='_blank'>
              Deciphering gene regulatory programs in mouse embryonic skin through single-cell multiomics analysis
              </a>
            </td>
          <td>
            Q. Deng, Pengfei Cai, Yingjie Luo, Zhongjin Zhang, Wen Ma, Zijie Huang, Xiaoya Chen, Shijie Hao, Weiguang Ma, Jiangshan Xu, Chunqing Wang, Mengnan Cheng, Xiumei Lin, Ru Zhou, Shanshan Duan, Junjie Chen, Ronghai Li, Xuyang Shi, Chang Liu, Liang Wu, Peng Gao, Jianting Li, Xiao Yang, Xiangdong Wang, Jun Xie, Longqi Liu, Yue Yuan, Chuanyu Liu
          </td>
          <td>2025-07-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Non-melanoma skin cancer (NMSC), comprising basal cell carcinoma (BCC) and cutaneous squamous cell carcinoma (cSCC), represents the most common type of cancer worldwide, particularly among Caucasians. While BCC is locally invasive with minimal metastatic potential, cSCC is a highly aggressive tumor with a significant potential for metastasis, particularly in elderly populations. Tumor development and progression and the metastasis of cSCC are influenced by a complex interplay between tumor cells and the tumor microenvironment. Recent research highlights the importance of various immune cell subsets, including T cells, tumor-associated macrophages (TAMs), and dendritic cells, in influencing tumor progression, immune evasion, and treatment resistance. This review outlines key regulatory mechanisms in the immune tumor microenvironment (TME) of cSCC and explores the role of cytokines, immune checkpoints, and stromal interactions. We further discuss the relevance of three-dimensional (3D) in vitro models such as spheroids, organoids, and tumor-on-chip systems as tools to mimic immune–tumor interactions with higher physiological relevance, such as macrophage activation and polarization against cSCC cells. Globally, 3D models offer new opportunities for immunotherapy screening and mechanistic studies. Understanding the immune landscape in cSCC through advanced modeling techniques holds strong clinical potential for improving diagnostic and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423c3e60a81f05412f1191c356ba2acb90d2d9a5" target='_blank'>
              Exploring the Complexity of Cutaneous Squamous CellCarcinoma Microenvironment: Focus on Immune Cell Roles by Novel 3D In Vitro Models
              </a>
            </td>
          <td>
            M. Quadri, Marco Iuliano, Paolo Rosa, G. Mangino, Elisabetta Palazzo
          </td>
          <td>2025-07-23</td>
          <td>Life</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Background/Aim: Tongue squamous cell carcinoma (TSCC), a highly aggressive subtype of head and neck cancers, is characterized by frequent lymphatic metastasis and poor prognosis. Recently, we showed that lymphatic vessel endothelial hyaluronan receptor 1 (LYVE-1) is involved in TSCC progression, yet the underlying molecular mechanisms remain unclear. Materials and Methods: CRISPR/Cas9 gene editing was employed to generate LYVE1 knockout (KO) TSCC cell lines. Single-cell clones were isolated, screened, and validated through sequencing and Inference of CRISPR Edits (ICE) analysis and qRT-PCR. RNA sequencing was performed on LYVE1 KO and wild-type (WT) cells to identify differentially expressed genes (DEGs). Bioinformatic analyses, including Gene Ontology (GO) enrichment and protein-protein interaction (PPI) network mapping, were conducted to explore affected pathways. Finally, network topology was examined using NetworkAnalyzer and cytoHubba plugins. Results: Transcriptomic analysis revealed significant down-regulation of pro-metastatic pathways, including epithelial-mesenchymal transition (EMT), extracellular matrix remodeling, and immune modulation. DEG analysis identified 263 genes, with key down-regulated targets such as WNT5A, TGFB2, and MMP2, and up-regulation of tumor-suppressive genes including PTGS2. GO and PPI analyses highlighted LYVE1’s pivotal role in regulating cell adhesion, migration, and immune response. Conclusion: LYVE1 KO reduces TSCC invasive potential by disrupting EMT and tumor-stroma interactions, aligning with previous experimental findings. These results suggest LYVE1 as a critical driver of metastasis, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff3c633bd83b5225f79493e53059643bbd3c2790" target='_blank'>
              CRISPR/Cas9-mediated Knockout of LYVE1 In Human Tongue Cancer Cells Reveals Transcriptomic Changes in Metastasis-associated Pathways
              </a>
            </td>
          <td>
            Sini Karinen, Johanna FORERO-RODRÍGUEZ, Annika Järvinen, Kari K Eklund, Gonçalo Barreto, A. Salem
          </td>
          <td>2025-06-26</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) are essential components of the tumor microenvironment (TME), contributing to tumorigenesis, progression, and resistance to therapy. However, the functional diversity of CAF subpopulations and their role in tumor progression and patient prognosis remain poorly understood. This study aims to explore CAF heterogeneity and their functional roles in the TME using single-cell RNA sequencing (scRNA-seq) and multi-omics data analysis. Methods scRNA-seq data were analyzed to cluster CAF subpopulations in the TME, with key genes identified through functional annotation. Differentially expressed genes were analyzed, and prognostic genes were selected via Cox and LASSO regression. A risk score model (RiskScore) was developed for survival prediction and immune therapy sensitivity evaluation. Core CAF genes were examined using siRNA interference, qPCR, and Western blotting. Drug sensitivity was assessed to explore the clinical relevance of these genes. Results Four CAF subpopulations (CAF-0, CAF-1, CAF-2, CAF-3) were identified, revealing differences in key tumor-associated signaling pathways (e.g., MYC, WNT, TGF-β). Thirteen core genes related to prognosis were identified, and a RiskScore model was developed, showing significantly worse survival rates for high-risk patients (p < 0.001) and features of immune suppression, including increased M0 macrophage infiltration. Drug sensitivity analysis indicated that core genes (e.g., KLRB1, MAP1B) were linked to drug sensitivity, suggesting potential biomarkers for targeted therapy. Experimental validation showed that knockdown of the HIP1R gene significantly reduced tumor cell expression, confirming its critical role in tumor development. Conclusion This study offers a comprehensive analysis of CAF heterogeneity and its impact on TME, patient prognosis, and drug sensitivity. The developed RiskScore model provides theoretical support for personalized treatment based on CAF-related genes, offering new insights into CAF-driven tumor progression and potential targets for precision oncology and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5c1ce9d5dedbb8252961d9100d080e0187631a3" target='_blank'>
              Therapeutic implications of cancer-associated fibroblast heterogeneity: insights from single-cell and multi-omics analysis
              </a>
            </td>
          <td>
            Yumin Wang, Yi Ding, Haolin Liu, Zhongyou Xia, Guoqiang Liao, Shicheng Fan, JunXiong Li, JingBo Qin, PinYao Liang, Peng Gu, Xiaodong Liu, Runan Dong
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2e39c5159dc340646cc3bcfb136e1f500d00e4" target='_blank'>
              Reelin-LRP8 signaling mediates brain dissemination of breast cancer cells via abluminal migration.
              </a>
            </td>
          <td>
            Haofeng Huang, Min Zhang, Enyu Huang, Yongfang Zhao, Xiaoyu Li, Pu Qiu, Cairui Li, Jiahua Tao, Yuanqi Zhang, Lianxiang Luo, Guozhu Ning, Ceshi Chen, Jingjing Zhang
          </td>
          <td>2025-06-12</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Penile squamous cell carcinoma (PSCC) is a highly aggressive malignancy without effective treatment due to limited knowledge of its development and tumor microenvironment (TME). In this study, single-nucleus RNA sequencing (snRNA-seq) and high-resolution spatial transcriptomics are employed to comprehensively investigate the development trajectories and the TME. The results revealed that PSCC cells mimicked the differentiation and tissue organization of normal penile epithelium, independent of the human papillomavirus (HPV) infection status. Notably, a spatial subtype, Tum_1, appeared at early stage of tumor differentiation and in tumor-normal boundary regions. This subtype exhibited enhanced basal-like and stemness features and showed high LAMC2 expression, which activated laminin-integrin signaling via ITGA6/ITGB4, promoting tumor invasiveness. Furthermore, the results indicated that HPV-positive basal stem-like neoplasms dampened the immune function of T cells and macrophages, promoting an immunosuppressive environment that facilitates tumor progression. Supporting this, the patients with head and neck squamous cell carcinoma and lung squamous cell carcinoma who have high expression of HPV-positive Tum_1 signatures derived greater benefit from PD-1 blockade therapy. In summary, the findings provide a comprehensive spatial landscape of the PSCC TME and suggest potential treatment approaches targeting laminin-integrin interaction and immunotherapy, especially in HPV-positive patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c44a03a89b4201962b2248fcc19db445771e54" target='_blank'>
              Single-Cell and Spatial Transcriptomic Profiling of Penile Squamous Cell Carcinoma Reveals Dynamics of Tumor Differentiation and Immune Microenvironment.
              </a>
            </td>
          <td>
            Hongjian Song, Zhichao Tong, Guixiang Xie, Yaowei Li, Yubo Zhao, Fengpu Fan, Zongzheng Yang, Qing Shi, Qian Zhang, Feng Wen, Hongxue Meng, Haonan Li, Pengyu Guo, Dayong Hou, Zhenwei Zhang, Zhihao Yin, Ziyi Liu, Jianwei Wang, Peng Zhang, Peng Dai, Changfu Li, Lichen Teng, Yangyang Xu, Dexin Ding, Tao Xu, Jianzhang Li, Yongsheng Chen, Yu Qiu, Xiaowei Hu, Wentao Liu, Liang Wu, Roman Nawroth, Guibo Li, Hang Su, Ziqing Deng, Ziqi Wang, Di Wang, Wanhai Xu
          </td>
          <td>2025-06-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Temozolomide (TMZ) is an important chemotherapeutic agent for glioma treatment. However, the emergence of drug resistance impedes its application. Traditional population-level studies are limited in elucidating resistance mechanisms. Advances in single-cell and spatial transcriptomics technologies provide feasible resolution for studying the cellular composition and dynamics of tumors. In this study, we investigated the heterogeneity of gliomas associated with TMZ resistance at the single-cell and spatial transcriptome levels to identify the resistance mechanisms and potential therapeutic strategies. Methods Single cell sequencing technology was utilized to identify the cellular clusters of gliomas. Drug perturbation analysis and cellular propensity analysis revealed key cluster responding to TMZ. Enrichment analysis was preformed to explore the function of clusters. Transcription factor activity analysis revealed key transcription factors contributing to tumor resistance. Spatial transcriptome data and bulk RNA-seq data validates the role of key transcription factors. Downstream targets of key transcription factors were predicted and validated using gene regulation assays. Drug sensitivity analyses were used to seek viable strategies to overcome drug resistance. Results Glioma cells from before and after temozolomide treatment samples were classified into six clusters: NPC-like cluster, OPC-like cluster, MES-like cluster, AC-like cluster, OC, and Neuron. NPC-like clusters exhibited strong stemness and DNA repair capacity. The activity of MAZ in NPC-like cluster was significantly enhanced after TMZ treatment. The proportion of MAZ( +)_NPC-like cluster was higher in TMZ treated samples. Patients with high proportion of MAZ( +)_NPC-like cluster had poorer survival. Upregulation of MAZ is able to enhance drug resistance in glioma cells, but this phenomenon disappeared when FoxM1 expression was further silenced. The combination of paclitaxel and Trametinib is a promising strategy to overcome resistance. Conclusions NPC-like cluster is prevalent in recurrent and drug-resistant gliomas. MAZ transcription factors are critical regulators that promote the development of drug resistance in NPC-like clusters by enhancing the capacity of DNA repair and stemness. Patients with high proportions of MAZ( +)_NPC-like clusters have poor TMZ sensitivity and prognosis. MAZ enhances stemness and drug resistance in glioma cells by upregulating FOXM1 expression. The combination of paclitaxel and paclitaxel is a promising therapeutic strategy for treating gliomas and overcoming drug resistance. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06706-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c31c53340b2d02203f651d970425a0fc3e3628" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal MAZ(+) NPC-like clusters as key role contributing to glioma recurrence and drug resistance
              </a>
            </td>
          <td>
            Qiang Gu, Kan Wang, Tingming Lu, Yifei Xiao, Yaqing Wu, Hang Zhou, Kaijia Zhou
          </td>
          <td>2025-06-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aae76ee848421ca34df444a8b29937afca779e4" target='_blank'>
              MicroCT enables simultaneous longitudinal tracking of murine pancreatic cancer progression and cachexia
              </a>
            </td>
          <td>
            Katherine R. Pelz, Philip Jimenez, Colin J. Daniel, Samuel D. Newton, Melissa Cunningham, Rosalie C. Sears, P. J. Worth, Jonathan R. Brody, T. Zimmers
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Gastric cancer remains a leading cause of cancer-related mortality worldwide, driven significantly by gastric cancer stem cells (GCSCs), a unique subset of cells characterized by self-renewal, multipotency, and therapy resistance. This review synthesizes recent advancements in understanding GCSCs, focusing on their definition, characteristics, identification, isolation and role in tumor growth, metastasis, and resistance to conventional treatments. Key surface markers such as CD44, CD133, leucine rich repeat containing G protein-coupled receptor 5 (LGR5), and aquaporin 5 (AQP5) are critical for GCSC identification, with their expression linked to poor prognosis and tumor progression. The review summarizes mechanisms like epithelial-mesenchymal transition (EMT) and Wnt/β-catenin signaling that enhance GCSC stemness and malignancy. It also highlights emerging targeted therapies, including antibody-based approaches including CD44v6-specific antibodies, inhibitors like sulfasalazine, and novel immunological strategies such as chimeric antigen receptor T (CAR-T) cell therapy, aimed at eradicating GCSCs. By elucidating the molecular and functional roles of GCSCs, this review underscores GCSCs as therapeutic targets and provides a foundation for developing precision medicine strategies to improve gastric cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cacd867c51c08c4c5178ffbc92262edb25abafe2" target='_blank'>
              Current insights into gastric cancer stem cells: Mechanisms and therapeutic prospects
              </a>
            </td>
          <td>
            Chang-Yong Yin
          </td>
          <td>2025-05-30</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2c94a8a149ed8b89b155e50f8c13949352a8f13" target='_blank'>
              Molecular dynamics driving phenotypic divergence among KRAS mutants in pancreatic tumorigenesis
              </a>
            </td>
          <td>
            A. Grimont, David J. Falvo, Whitney J. Sisso, P. Zumbo, Christopher W. Chan, Francisco Santos, Grace Pan, Megan Cleveland, Tomer M. Yaron, Alexa Osterhoudt, Yinuo Meng, Maria Paz Zafra, William B. Fall, André F. Rendeiro, Erika Hissong, R. Yantiss, D. Betel, Mark A. Magnuson, Steven D. Leach, Anil K. Rustgi, Lukas E. Dow, R. Chandwani
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, largely due to its profoundly immunosuppressive tumor microenvironment (TME) and intrinsic resistance to conventional therapies. Radiotherapy (RT), traditionally valued for its cytotoxic effects, has recently been recognized for its immunomodulatory potential. This mini-review explores the multifaceted interactions between RT and the PDAC immune microenvironment, highlighting mechanisms such as induction of immunogenic cell death, enhancement of antigen presentation, modulation of cytokine and chemokine profiles, and upregulation of immune checkpoint molecules. These effects may transform immunologically “cold” tumors into “hot” ones, providing a rationale for combination strategies with immunotherapy. However, the dense desmoplastic stroma, abundance of regulatory T cells, and myeloid-derived suppressor cells within PDAC present substantial challenges that hinder effective immune activation. Advances in single-cell and spatial transcriptomic technologies offer new opportunities to better characterize the TME and guide personalized treatment strategies. By synthesizing mechanistic insights and clinical evidence, this review underscores the potential of integrating RT with immunotherapy to overcome resistance mechanisms and improve therapeutic outcomes in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97772762d5bc3d1c76c3496afb2362f063af01af" target='_blank'>
              Radiotherapy and immune microenvironment crosstalk in pancreatic cancer: a comprehensive review of current insights and future directions
              </a>
            </td>
          <td>
            Yucheng Xu, Jingjing Chen, Yudong Qiu, Juan Du
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 11569


 Background:
 Soft tissue sarcomas (STS) are rare malignancies with over 100 distinct histological subtypes. Their rarity and heterogeneity pose significant challenges to identifying effective therapies, and approved regimens show varied responses. Several patient-derived tumor models have emerged recently. However, STS present a challenge in developing preclinical drug-testing models due to their non-epithelial and complex nature.
 Methods:
 Here we report a model termed patient-derived tumor-like cell clusters (PTCs) derived from STS patients. PTCs result from the self-assembly and proliferation of mesenchymal stem cells (MSCs), epithelial cells, and immune cells, faithfully recapitulating the morphology and function of the original tumors. This is an trial to assess the feasibility and predictive value of a standardized PTC-based test to differentiate efficacy of the patients' clinical drug regimens. The study was conducted at Peking University Cancer Hospital and was approved by the local ethical review board. The patients and corresponding PTCs were divided into three sets: characterization and storage set, assay set and validation set. The characterization and storage set were used to characterize PTCs in comparison with original tumor samples or stored for future study. The assay set was separated into two groups to determine the drug efficacy concentration of a targeted therapy or chemotherapy due to their different action mechanisms. The validation set was used to compare the consistency between PTC drug assays and clinical outcome. We then conducted comparative analyses between PTCs and tumor spheres, as well as between PTCs and paired tumor samples.
 Results:
 From 2019 to the 2025, we obtained 254 samples (155 surgical, 98 puncture, and 1 ascites sample) to generate PTCs, covering tens of sarcoma classifications, with an overall success ratio of 94.9%, ranging from 85.7% to 100%. A total of 3,740 differentially expressed genes (DEGs) were identified between PTCs and tumor spheres, while 1,222 DEGs were identified between PTCs and tumor samples. Through standardized culture and drug-response assessment protocols, PTCs facilitate personalized drug testing, evaluating hundreds of therapies within two weeks. PTCs demonstrate an overall predictive accuracy of 78.3% for all clinical outcomes and 100% accuracy distinguishing CR/PR from PD, could serve as a valuable tool for personalized medicine.
 Conclusions:
 These findings revealed that PTCs as a tool to better understand the biology of individual tumors and characterize the landscape of drug resistance and sensitivity in sarcoma. These results underscore the potential of PTCs for prospective use in clinical decision-making therapy selection.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b94268847071702617cce01be0a77e1e5ba1497b" target='_blank'>
              Self-assembled patient-derived tumor-like cell clusters for personalized drug testing in diverse sarcomas.
              </a>
            </td>
          <td>
            T. Gao, Xinyu He, Junyi Wang, Jiayong Liu, Xiongbing Hu, Chujie Bai, Shenyi Yin, Yunfei Shi, Yanmin Wang, Z. Tan, Fang Cao, Shu Li, Yanjie Shi, R. Xue, Juan Li, Yang He, Z. Fang, Z. Fan, J. Xi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND
Ferroptosis has been implicated in the regulation of the tumor immune environment; however, its precise effect on immune checkpoint inhibitors remains contradictory.


OBJECTIVE
To elucidate the "double-edged sword effect" of a key ferroptosis-related factor in regulating the immune microenvironment.


METHODS
This study utilized single-cell RNA sequencing (scRNA-seq) analysis to characterize the tumor microenvironment in ovarian cancer samples from immunotherapy cohorts. Following quality control and variable gene screening, data from The Cancer Genome Atlas (TCGA), Genotype-Tissue Expression (GTEx), GENE EXPRESSION OMNIBUS (GEO), bulk, and spatial transcriptome databases were analyzed. The AddModuleScore_UCell function was employed for gene set scoring by evaluating the expression patterns of specific gene features in single-cell datasets, which were found to correlate with interactions between tumor cells and stromal cells, recognized as key contributors in the immunosuppressive milieu. Immunohistochemistry, western blot, and multiplex immunohistochemistry (mIHC) analyses were employed to explore the HMOX1/TGF-β1/PI3K/AKT/NF-κB(p65) signaling pathways. In vitro findings were further validated in a mouse model. The correlation between risk factors and progression-free survival (PFS) was analyzed using Cox regression and Kaplan-Meier methods.


RESULTS
We demonstrated decreased expression of the ferroptosis-activating gene HMOX1 in ovarian cancer epithelial cells, while being upregulated in macrophages. Ovarian cancer (OV) epithelial cells with HMOX1 inhibition could secrete TGF-β1 to activate three macrophage subtypes: SPP1+, FOLR2+ and C1QC+ via the PI3K/AKT/NF-κB (p65) pathway. The up-regulation of HMOX1 in macrophages also activated these three macrophage subtypes via the NF-κB pathway. Both pathways simultaneously inhibited Cytotoxic T Lymphocyte (CTL) activation and contributed to the immunosuppressive microenvironment of ovarian cancer, as demonstrated in both in vitro and in vivo models. Targeting HMOX1 alone, whether through activation or inhibition, was only effective in modulating a single pathway while simultaneously inducing negative feedback on the opposing pathway, demonstrating the "double-edged sword effect" of HMOX1 in regulating the immune microenvironment.


CONCLUSION
Overall, we proposed and validated two strategies targeting HMOX1 to improve the efficacy of PD-1 inhibitors, and confirmed that HMOX1, TGF-β1, SPP1, FOLR2, and C1QC could be used to construct models predicting the efficacy of immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f01291927dba9ec25f0b861c5f37f0048388aa" target='_blank'>
              Distinct roles of HMOX1 on tumor epithelium and macrophage for regulation of immune microenvironment in ovarian cancer.
              </a>
            </td>
          <td>
            Yi Liu, Li-Jun Jiang, Hong-Fang Liu, Li Chen, Lei Guo, Jun Ge, Xin-Yi Zhang, Jing Li, Wei Gong
          </td>
          <td>2025-07-09</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Comprehensive analysis of single-cell transcriptome and chromatin accessibility will contribute to interpret the heterogeneity of acute myeloid leukemia (AML). We hypothesize that integrating single-cell transcriptomic and chromatin accessibility landscapes underlying t(8;21) AML will provide valuable insights into its heterogeneous cellular properties and gene regulatory programs. Methods Here, we conducted paired single-cell RNA-sequencing (scRNA-seq) and single-cell ATAC sequencing on bone marrow samples from newly diagnosed t(8;21) AML patients and healthy controls. Genetic signatures extracted from scRNA-seq were built and validated across three independent cohorts (German AMLCG1999, GSE106291 and TCGA LAML). Results We identified TCF12, a core component of AML1-ETO-containing transcription factor complex (AETFC), as the most active transcription factor in blast cells, driving a universally repressed chromatin state. Furthermore, we delineated two functionally distinct T cell subsets, revealing that EOMES-mediated transcriptional regulation promotes the expansion of a cytotoxic T cell population (T cells_2; high GNLY, NKG7 and GZMB expression), with an increased clonality and a tendency for drug resistance. In addition, we discovered a novel leukemic CMP-like cluster characterized by high TPSAB1, HPGD and FCER1A expression. Leveraging machine learning-based integration of multi-omic profiles, we identified a robust 9-gene prognostic signature, which demonstrated significant predictive value for AML outcomes across three independent cohorts. Conclusions This multi-omics study provides unprecedented insights into the transcriptional and epigenetic heterogeneity of t(8;21) AML, providing a potential actionable tool for clinical risk stratification. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06659-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6d672d2579a9875bb647a603a272b4cc733a93" target='_blank'>
              A multi-omic single-cell landscape reveals transcription and epigenetic regulatory features of t(8;21) AML
              </a>
            </td>
          <td>
            Xue-Ping Li, Yan Gao, Bai-Tian Zhao, Yujun Dai, J. Mao, Yang Liang, Lu Jiang
          </td>
          <td>2025-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 e18514


 Background:
 Acute lymphoblastic leukemia (ALL) is a hematological malignancy characterized by impaired differentiation and abnormal proliferation of lymphoblasts. Despite advances in the understanding of its molecular mechanisms, accurate characterization of cellular heterogeneity and genetic alterations remains a challenge. The aim of this study is to take advantage of the scGPT, generative pre-trained transformer model adapted to the genome of one cell, to improve the distinction of cellular clusters and profile gene expression in all. We have used data files including B-cell subtypes and T-cells of ALL. Comparative analyzes with conventional tools, such as scVI and Harmony, have shown an excellent ability to maintain biological variability while providing accurate cell type annotations.
 Methods:
 This study utilized single-cell RNA sequencing (scRNA-seq) to obtain transcriptomic profiles from individual cells. Biological samples were processed through optimized cell capture and library preparation protocols to reduce technical noise and batch effects. NVIDIA GPU powered the computational environment with CUDA acceleration, and key software tools included scVI, scrublet, harmony, and scGPT. Highly variable genes were identified using scVI, leveraging Variational Autoencoders (VAEs) to detect nonlinear expression variability. Dimensionality reduction was performed with scVI, producing embeddings corrected for batch effects. Clustering analysis using the Leiden algorithm identified cell subpopulations and visualization tools such as UMAP and t-SNE displayed relationships among cell groups.
 Results:
 Visualization techniques, including UMAP and T-SNE, have confirmed the excellent ability of scGPT to identify rare cell populations and subsets otherwise missed by traditional methods. For example, in B cell subtypes, scGPT successfully distinguished naive B cells, memory B cells, and plasmablasts with high accuracy. Similarly, T cell subtypes such as regulatory T cells and cytotoxic T cells have been clearly characterized. Integration of scGPT with zero-shot learning enabled accurate profiling of gene expression across cell clusters and identified key differentially expressed genes relevant to acute lymphoblastic leukemia (ALL) pathology. These were genes associated with cell proliferation, immune evasion and metabolic pathways.
 Conclusions:
 Advanced scGPT modeling has enabled the prediction of potential therapeutic targets and paved the way for tailored interventions. Our research applies scGPT as a key tool to advance single-cell transcriptomic studies in ALL, offering a better understanding of its cellular and molecular complexities. This approach holds promise for improving therapeutic strategies and personalized medicine in hematologic malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6d87f169f0235ef41c3797b0a5ed2283cc4c8a3" target='_blank'>
              Empowering single-cell genomics with scGPT: Precision clustering and genetic insights into acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Pei-Hsuan Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8aace533fb9be40cad1883d118a83d4e2257b543" target='_blank'>
              Single-cell spatial map of cis-regulatory elements for disease-related genes in the macaque cortex
              </a>
            </td>
          <td>
            Juan Meng, Ying Lei, Cheng Chen, Zhiyong Zhu, Yongkang Sun, Yiming Huang, Kaijie Hu, Yiqin Bai, Tianyi Fei, Zhen Liu, Chao Li, Zhi-jian Shen, Longqi Liu, Chengyu Li, Tao Song, Cirong Liu, Muming Poo, Shingping Liu, Yidi Sun
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Malignant melanoma is an aggressive skin cancer arising from melanocytes. Melanoma is a complex disease both sourced from genetics and environmental factors. Within melanoma, cancer stem cells (CSCs) play a crucial role in tumor progression, therapeutic resistance, and recurrence due to their capabilities for self-renewal and differentiation. The Sonic Hedgehog (SHH) signaling pathway is an important regulator of CSCs and is essential for cell differentiation and proliferation. Due to it is known role in embryonic development and involvement in cancers, SHH pathway significantly affects CSC behavior in malignant melanoma, promoting tumorigenicity, metastasis, and resistance to therapies. This pathway coordinates canonical mechanisms involving Gli transcription factors and non-canonical mechanisms affecting cell migration and cytoskeletal organization. Targeting the SHH pathway has emerged as a promising therapeutic strategy, with inhibitors focusing on components like Smoothened (Smo) and Gli proteins. However, resistance to these inhibitors necessitates further exploration of novel therapeutic combinations. Current research focuses on combining SHH inhibitors with immunotherapies for more effective, long-lasting responses. Targeted medicines, which disrupt SHH processes, attempt to eliminate the fundamental causes of carcinogenesis and increase melanoma patient survival rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c35ec356094da97200d6132e449839ad9acd76ca" target='_blank'>
              Cancer stem cells and the SHH pathway in malignant melanoma: Therapeutic implications
              </a>
            </td>
          <td>
            Berrin Ozdil, H. Aktug
          </td>
          <td>2025-06-10</td>
          <td>Ege Tıp Dergisi</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6483fac3f504fbebfb3a60d1e21f332ce1ac0267" target='_blank'>
              Epigenetic reprogramming of mast and cancer cells modifies tumor-promoting cytokine networks.
              </a>
            </td>
          <td>
            Alejandro Schcolnik-Cabrera, Marisol Ramírez-Yautentzi, Rodolfo Soria-Castro, R. Chacón-Salinas, Rocío Morales-Bárcenas, G. Domínguez-Gómez, A. González-Fierro, Alfonso Dueñas-González, Alma D Chávez-Blanco
          </td>
          <td>2025-07-24</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="For over a century, scientists reported the disruption of normal nuclear shape and size in cancer. These changes have long been used as tools for diagnosis and staging of malignancies. However, to date, the mechanisms underlying these aberrant nuclear phenotypes and their biological significance remain poorly understood. Using a model of pancreatic ductal adenocarcinoma (PDAC), the major histological subtypes of pancreatic cancer, we found that oncogenic mutant KRAS reduces nuclear size. Transcriptomic and protein expression analysis of mutant KRAS–expressing PDAC cells revealed differential levels of several nuclear envelope–associated genes. Further analysis demonstrated the nuclear lamina protein, Emerin (EMD), acted downstream of KRAS to mediate nuclear size reduction in PDAC. Analysis of human PDAC samples showed that increased EMD expression associates with reduced nuclear size. Finally, in vivo genetic depletion of EMD in a mutant KRAS–driven PDAC model resulted in increased nuclear size and a reduced incidence of poorly differentiated PDAC. Thus, our data provide evidence of a potentially novel mechanism underlying nuclear size regulation and its effect in PDAC carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fad3a87beb812ed2828574a6cdacbebbce33710" target='_blank'>
              Emerin is an effector of oncogenic KRAS-driven nuclear dynamics in pancreatic cancer
              </a>
            </td>
          <td>
            Luis F Flores, David L Marks, Renzo E Vera, Ashley N Sigafoos, Ezequiel J Tolosa, Luciana L Almada, David R Pease, M. D. Toruner, Brian Chang, Brooke R Tader, Kayla C LaRue-Nolan, Ryan M. Carr, Rondell P Graham, Catherine E Hagen, Matthew R Brown, Aleksey V Matveyenko, Katherine L Wilson, David W Dawson, Christopher L. Pin, Kyle J Roux, Martin E Fernandez-Zapico
          </td>
          <td>2025-06-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Macrophages play a crucial role in coordinating the skeletal muscle repair response, but their phenotypic diversity and the transition of specialized subsets to resolution-phase macrophages remain poorly understood. Here, to address this issue, we induced injury and performed single-cell RNA sequencing on individual cells in skeletal muscle at different time points. Our analysis revealed a distinct macrophage subset that expressed high levels of Gpnmb and that coexpressed critical factors involved in macrophage-mediated muscle regeneration, including Igf1, Mertk and Nr1h3. Gpnmb gene knockout inhibited macrophage-mediated efferocytosis and impaired skeletal muscle regeneration. Functional studies demonstrated that GPNMB acts directly on muscle cells in vitro and improves muscle regeneration in vivo. These findings provide a comprehensive transcriptomic atlas of macrophages during muscle injury, highlighting the key role of the GPNMB macrophage subset in regenerative processes. Our findings suggest that modulating GPNMB signaling in macrophages may represent a promising avenue for future research into therapeutic strategies for enhancing skeletal muscle regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/424ac51758fc4d23fce4d223b8772f7f9cda917c" target='_blank'>
              Temporal single-cell sequencing analysis reveals that GPNMB-expressing macrophages potentiate muscle regeneration
              </a>
            </td>
          <td>
            Yu-Fan Chen, Chien-Wei Lee, Yi-Shuan J Li, Wei-Ting Lin, Hsiao-Yun Chen, Yu-Chuan Chen, Chia-Hao Lin, Jennifer Hui-Chun Ho, Li-Fan Lu, Shu Chien, Oscar Kuang-Sheng Lee
          </td>
          <td>2025-06-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Trastuzumab deruxtecan (T-DXd) is the standard of care for HER2+ or HER2 low metastatic breast cancer (mBC). However, approximately 20% of patients exhibit resistance to T-DXd therapy. As the development of more antibody-drug conjugate (ADC) therapies progresses, understanding ADC resistance mechanisms is critical. Our study aims to elucidate the etiology of ADC resistance, such as characterizing molecular changes and identifying novel biomarkers in tumor-immune interactions between T-DXd responsive (R) and non-responsive (NR) patients with mBC. We employed Nanostring’s GeoMX spatial proteomics and transcriptomics profiling technology to analyze 29 metastatic tissue sections (15R, 14NR) from heavily pre-treated patients with HER2+/low mBC, prior to T-DXd treatment. For each section, 6-8 circular regions of interest (ROIs) with a diameter of 300µm were chosen by researchers blinded to T-DXd R vs. NR outcomes and HER2 scores. ROIs were selected to capture spatially diverse tumor-dense and tumor-sparse regions. Immune markers, stromal markers, and HER2 expression levels were quantified for each ROI. Consistent immune and stromal features related to T-DXd resistance were identified across various metastatic sites, including brain, bone, liver, lymph node, chest wall, and lung. NR tumors were characterized by relative immunosuppression and a disorganized immune response, with consistent upregulation of fibronectin and, surprisingly, granzyme B. Local immune response significantly varied based on HER2 expression. Organ-specific characteristics were also observed: Bone metastases showed an increased correlation between various immune markers and fibronectin in NR, suggesting that fibronectin may modulate immune infiltration and promote a supportive microenvironment for tumor growth. In contrast, chest wall metastases showed loss of correlation between various immune markers and smooth muscle actin in NR. This suggests that a remodeled, less dense smooth muscle environment might reduce physical barriers, potentially allowing better immune cell infiltration and activity in responsive tumors. Furthermore, a subset of these patient samples was selected for spatial whole transcriptomic analysis to gain deeper insights.
 Citation Format: Glori Das, Matthew Vasquez, Jeffrey Zhang, Ji-Hoon Lee, Yuan Gao, Jilun Zhang, Jenny Chang, Xiaoxian Li, Hong Zhao, Stephen T.C. Wong. Spatial Omics Analysis Uncovers Microenvironmental Remodeling and Immune Dynamics in T-DXd Resistant Metastatic Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS4-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c6cc9418499478f1051264c562f281d5a33003b" target='_blank'>
              Abstract PS4-09: Spatial Omics Analysis Uncovers Microenvironmental Remodeling and Immune Dynamics in T-DXd Resistant Metastatic Breast Cancer
              </a>
            </td>
          <td>
            Glori Das, Matthew Vasquez, Jeffrey Zhang, Ji-hoon Lee, Yuan Gao, Jilun Zhang, Jenny Chang, Xiaoxian Li, Hong Zhao, Stephen Wong
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, largely due to its pronounced cellular heterogeneity, therapy resistance, and metastatic propensity, traits driven by cancer stem cells (CSCs). Once thought to be a rare, static subpopulation, CSCs are now recognized as a dynamic, plastic cell state influenced by both intrinsic and extrinsic factors. In this review, we explore the “inner” (genetic, cellular, and metabolic) and “outer” (microenvironmental and macroenvironmental) determinants that govern CSC identity, emergence, and evolution in PDAC. We focus on how acinar and ductal cell plasticity, in combination with oncogenic mutations (e.g., KRAS, TP53), contribute to tumor initiation and CSC generation, how genomic instability further fuels CSC heterogeneity and adaptability, and we highlight how micro and macro TME properties, metabolism and external factors regulate CSC behavior afterwards. Altogether, CSCs exist at the nexus of genetic chaos and environmental cues. Understanding this complex interplay is key to developing therapies that not only eradicate CSCs but also prevent their reemergence. While still far from clinical application, disrupting CSC-supportive networks should be a research and clinical focus for improving outcomes for PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08e2a3f4aedf4a7111928c29cee984777f2e319" target='_blank'>
              Inner and Outer Determinants of Pancreatic Cancer Stem Cells
              </a>
            </td>
          <td>
            J. C. López-Gil, Diego Navarro, B. Sainz
          </td>
          <td>2025-06-16</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/503cca6deb2a91fe468e2c781a6fa64738c62b22" target='_blank'>
              Drug resistance and tumor heterogeneity: cells and ensembles
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-05-31</td>
          <td>Biophysical Reviews</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="Gallbladder cancer (GBC) is a highly aggressive malignancy with a propensity for lymph node metastasis (LNM), which significantly worsens prognosis. This review explores the molecular mechanisms underlying LNM in GBC, focusing on the roles of vascular endothelial growth factors (VEGFs), chemokines, cancer-associated fibroblasts (CAFs), tumor-associated macrophages (TAMs), hypoxia-inducible factors (HIFs), and non-coding RNAs (ncRNAs) in shaping the tumor microenvironment (TME). Unique features of GBC, such as its bile-rich microenvironment and hypoxia-driven lymphangiogenesis, are highlighted. We discuss how these factors promote lymphangiogenesis, immune evasion, and extracellular matrix (ECM) remodeling, collectively facilitating LNM. Potential therapeutic targets, including VEGF-C/D pathways, matrix metalloproteinase (MMP) inhibitors, and immune-modulating therapies, are also reviewed. Future research integrating single-cell omics and patient-derived organoid models is essential for advancing precision medicine in GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74624732c38b10967280fe625753908628ae6ef1" target='_blank'>
              Molecular Mechanisms of Lymph Node Metastasis in Gallbladder Cancer: Insights into the Tumor Microenvironment
              </a>
            </td>
          <td>
            Qingyu Tang, Yichen Guan, Yubo Ma, Qi Li, Zhimin Geng
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Unchecked growth in three-dimensions (3D) in culture is a key feature of immortalized cells on the path to malignant transformation and hence a potential target phenotype for prevention. Also, expression of genes driving this process, but not that of 2D growth, would likely be more specific to cancer development and their inhibition would be less toxic to normal cells, many of which can grow in 2D but rarely in 3D culture. To define such genes, we compared CRISPR depletion screens performed in HBLAK, a spontaneously immortalized, non-tumorigenic human urothelial cell line, grown in 2D to those in 3D. Using the CRISPR Bassik DTKP (drug target kinase phosphatase) deletion library targeting 2,333 genes, we identified 85 genes which were specifically lost in 3D cultures. Prioritizing hits to those associated with bladder cancer in patients provided us with a set of 11 genes. Only one gene, MAPK1 remained relevant if a human pan-cancer criteria was applied. Single gene in vitro validation confirmed that MAPK1 inhibition was specific to 3D growth. We also found that MAPK1 depletion led to significant growth reductions in human tumor xenografts in vivo. Inhibition of MAPK1 by Ulixertinib, an orally active MAPK1 inhibitor, led to human bladder cancer growth inhibition in both 3D in vitro and in vivo models. In summary, screening for genes specifically driving 3D growth in immortalized cells may provide targets for both prevention and early therapy in bladder and other cancers while potentially limiting therapeutic toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c39ec53d27180d1be55595d4ca83d125348b0422" target='_blank'>
              Genes driving three-dimensional growth of immortalized cells and cancer
              </a>
            </td>
          <td>
            Mukta Basu, Jin-Fen Xiao, Saravana Kumar Kailasam Mani, Fangyuan Qu, Yongqi Lin, Jason Duex, Huihui Ye, Vanessa Neang, Dan Theodorescu
          </td>
          <td>2025-06-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598cf28c8237eff38a5ab0a3fac580f97e9fbf01" target='_blank'>
              Midkine inhibits dormancy of disseminated melanoma cells
              </a>
            </td>
          <td>
            Carolina Rodriguez-Tirado, Thoufiqul Alam Riaz, Alcina A. Rodrigues, Nupura Kale, Smrithi Jayashree Satheeshkumar, Ayax Perez-Gallegos, David Olmeda, M. Soengas, Maria Soledad Sosa
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83bbe9f956eb8e608baaece605424980c54eeb73" target='_blank'>
              Single-cell RNA sequencing analysis revealed the immunosuppressive remodeling of tumor-associated macrophages mediated by the MIF-CD74 axis in gastric cancer.
              </a>
            </td>
          <td>
            Weiwei Liu, Chuangyuan Wang, Lixin Liang, Congcong Zhang, Yanyan Li, Jun Xiao, Rui Zeng, Jingyu Li, Zhihao Lin, Qinghua Huang, Lisheng Zheng, Qingling Zhang
          </td>
          <td>2025-07-24</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive brain tumor with limited treatment options and prognosis due to its highly invasive nature and therapy resistance. Investigating the nanomechanical and pathophysiological properties of GBM cells will shed light on tumor behavior. In our study, we investigated the mechanical properties, migration dynamics, and cytoskeletal organization of T98G and U87 MG glioblastoma cell lines using in vitro techniques: atomic force microscopy (AFM) and electric cell-substrate impedance sensing (ECIS). While U87 MG cells are Temozolomide (TMZ)-sensitive and exhibit increased susceptibility to cell death and growth inhibition, T98 cells exhibit improved survival and repair in response to TMZ therapy. This study found that T98G cells are rougher, stiffer, and more viscous, while U87 MG cells are smoother, more elastic, and less viscous, leading to distinct cellular migration patterns. Such differences indicate GBM cell heterogeneity and have consequences for tumor development and resistance to treatment. The key differences of the nanomechanical, viscoelastic, and migratory properties between T98G and U87 MG cells, demonstrated in this work will help us gauge the diverse effects of different dosages of radiation along with immunotherapeutic agents, identifying the ideal radioimmunotherapy option.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ba2cc0d72ec2de612a1d7d825b152f96e3640f" target='_blank'>
              Exploring the heterogeneity in glioblastoma cellular mechanics using in-vitro assays and atomic force microscopy
              </a>
            </td>
          <td>
            Nabila Masud, Md. Hasibul Hasan Hasib, Bayode Ibironke, Charlotte Block, Jayce Hughes, Andrew E. Ekpenyong, Anwesha Sarkar
          </td>
          <td>2025-06-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Androgenetic alopecia (AGA) is marked by the progressive miniaturization of hair follicles (HFs) and hair thinning, driven by a decline in the progenitor cells critical for hair regeneration. Despite this, the mechanisms responsible for progenitor cell depletion remain largely unclear. To investigate transcriptional alterations in the progenitor cell regions of AGA patients while maintaining the spatial tissue context, we employed the GeoMX Digital Spatial Profiling (DSP) platform, which enables a precise comparison with healthy controls. Our analysis revealed the significant upregulation of genes associated with extracellular matrix (ECM) organization and the epithelial–mesenchymal transition (EMT), including FN1, TWIST1, and TGFB2 in the progenitor cell region of the HFs. Correspondingly, protein expression data confirmed increased levels of the protein products of these genes in the affected areas, underscoring their roles in the disease’s progression. These molecular changes suggest an environment conducive to the EMT, potentially contributing to the loss of progenitor cells and indicating a fibrogenic shift within the HF microenvironment. Additionally, our study highlights the influence of peri-infundibular immune cell infiltration on these molecular changes, suggesting that immune-mediated microinflammation may contribute to the fibrogenic environment and progenitor cell loss in the AGA. These findings demonstrate the utility of spatial transcriptomics in identifying potential therapeutic targets and advancing our understanding of AGA’s molecular mechanisms, offering avenues for developing targeted treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57509989c44771eb35e7ca6a4c3885f3af58e74a" target='_blank'>
              Progenitor Cell Dynamics in Androgenetic Alopecia: Insights from Spatially Resolved Transcriptomics
              </a>
            </td>
          <td>
            S. Charoensuksira, P. Surinlert, Aungkana Krajarng, Thararat Nualsanit, Witchuda Payuhakrit, Pimchanok Panpinyaporn, Wilunplus Khumsri, W. Thanasarnaksorn, Atchima Suwanchinda, S. Hongeng, S. Ponnikorn
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="The breast peritumor microenvironment (pTME) is increasingly recognized as a mediator of breast cancer progression and treatment resistance. However, if and how growth-induced tumor compressive forces (i.e., solid stresses) influence the breast pTME remains unclear. Here we show using instant fluorescence lifetime imaging microscopy (FLIM)—a frequency-domain FLIM system capable of simultaneous image acquisition and instantaneous data processing—that breast tumor-mimicking in vitro compression promotes metabolic changes in stromal cells found in the breast pTME. Namely, compression shifts NIH3T3 fibroblasts and differentiated 3T3-L1 (d3T3-L1) adipocytes toward a more glycolytic state, while it promotes increased oxidative phosphorylation in 3T3-L1 undifferentiated adipocytes. The gold-standard Seahorse extracellular flux assay fails to capture these changes, underscoring the superior sensitivity of instant FLIM in detecting metabolic shifts. We validate these phenotypic findings at the transcriptomic level via RNA sequencing, confirming that compressed fibroblasts downregulate oxidative phosphorylation and upregulate glycolysis compared to uncompressed controls. We further demonstrate that compression induces mitochondrial dysregulation in undifferentiated adipocytes, driven in part by upregulated mitophagy and disrupted fusion dynamics. Finally, we confirm that these stromal cell types recapitulate these distinct metabolic states in human breast cancer patient samples, consistent with our in vitro findings. By elucidating mechano-metabolic interactions occurring at the tumor-host interface, these results will inform the development of innovative mechano-metabolic reprogramming treatment strategies to improve breast cancer patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d792df55ee5bac5edf40597207282a8a899c162" target='_blank'>
              Instant fluorescence lifetime imaging microscopy reveals mechano-metabolic reprogramming of stromal cells in breast cancer peritumoral microenvironments
              </a>
            </td>
          <td>
            Julian Najera, Hao Chen, Bianca Batista, Frank Ketchum, Aktar Ali, P. Zorlutuna, Scott Howard, Meenal Datta
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Tumor protein p53 target gene 1 (TP53TG1) is a significant long non-coding RNA (lncRNA) located on human chromosome 7q21.12, exhibiting dysregulated expression in several cancer tissues, and operates both as an oncogene and a tumor suppressor. TP53TG1 modulates gene expression by several mechanisms, including “competitive RNA” activity, competitive miRNA binding, control of critical signaling pathways, and facilitation of epigenetic changes. TP53TG1 affects tumor proliferation, migration, invasion, and apoptosis through intricate mechanisms, modifies tumor cells resistance to therapies, influences the tumor immune microenvironment and metabolic reprogramming, and may present various avenues for clinical therapeutic intervention. Notwithstanding considerable advancements in comprehending the molecular function of TP53TG1, some challenges persist, especially regarding the efficient and targeted administration of TP53TG1-based therapeutics. Advanced RNA delivery technologies, including lipid nanoparticles (LNPs), sphingomyelin-based nanosystems, and polymer-based nanocarriers, have demonstrated the capacity to surmount these obstacles. This review aims to clarify the expression pattern of TP53TG1 in 10 malignant tumors, its biological relevance, its potential as a therapeutic target, and its importance in clinical applications, thereby offering a novel perspective on molecular targeting in cancer. In the future, integrating TP53TG1 with conventional medications may augment efficacy, offer personalized treatments, and eventually increase the prognosis for patients with aggressive or drug-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f226208fa68047280df52857f7f84b24287ce57b" target='_blank'>
              Targeting TP53TG1: a promising prognostic biomarker and therapeutic target for personalized cancer therapy
              </a>
            </td>
          <td>
            Haodong He
          </td>
          <td>2025-07-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Chemoresistance remains a major therapeutic challenge in high-grade serous ovarian cancer (HGSOC). Our study demonstrates that the functional restoration of death-associated protein kinase 1 (DAPK1), a novel tumor suppressor in HGSOC, induces potent tumor-specific cytotoxicity and inverts chemoresistance. Using molecular and preclinical analyses, we showed that full-length and truncated DAPK1 variants efficiently restore apoptotic signaling pathways and significantly enhance platinum-based and taxane treatment sensitivity. Therapeutic efficacy was validated across different model systems, including ovarian cancer cell lines and patient-derived organoids. Remarkably, the truncated DAPK1 constructs maintain strong apoptotic activity despite structural modifications, highlighting clinical potential. These findings suggest DAPK1 reactivation as a viable strategy to overcome treatment resistance in HGSOC, with the potential for translation into combination therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c04d0909c04866431b8c4c91c2104812e72b5aa2" target='_blank'>
              Truncated DAPK Variants Restore Tumor Suppressor Activity and Synergize with Standard Therapies in High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Monika Raab, Khayal Gasimli, Balázs Győrffy, Samuel Peña-Llopis, Sven Becker, M. Sanhaji, K. Strebhardt
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Glioblastoma is a highly aggressive brain tumor with an overall poor prognosis due to its immunosuppressive tumor microenvironment (TME). Microglia and tumor-associated macrophages (TAMs) with pro-tumorigenic properties are dominant populations of immune cells in the glioblastoma TME. To date, several studies targeting TAMs to fight tumor progression in different tumor entities have been initiated. However, the impact of standard therapy schemes of glioblastoma cells on macrophage polarization, activation, and phagocytosis remains controversial. The same applies to the relevance of PD-1/PD-L1 blockade in the interaction between macrophages and tumor cells. Our study, therefore, investigated patient-oriented treatment of GLIOBLASTOMA by examining the phagocytic capacity of polarized M1- and M2-like macrophages using GL261-luc2 tumor cells as a preclinical model system. Additionally, we analyzed the expression of activation and immune checkpoint markers on these macrophage subtypes following contact with tumor cells and their microenvironment. These factors were also determined after PD-1 blockade was initiated. The analyses revealed that the immunoregulatory M2-like macrophages generally exhibited a higher phagocytosis rate than the pro-inflammatory M1-like macrophages; however, this was not influenced by the pretreatment of glioblastoma cells with chemo- or radiotherapy. This could not be improved by blocking the PD-1 receptor. Furthermore, there were no modulations in the expression of differentiation, activation, or immune checkpoint molecules of M1- and M2-like macrophages after cell-to-cell contact with glioblastoma cells. But the medium conditioned by tumor cells strongly altered M1-like macrophages toward a more activated state, whereas M2-like cells were only mildly influenced. This was further enhanced by tumor cell treatment, with the most prominent effect after irradiation. These results suggest that conventional GLIOBLASTOMA tumor cell treatment affects the immunogenic status of macrophage subtypes, which is relevant for enhancing the anti-tumor immune response in brain tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47dbb5ff93613986d34e432ca007b413cfe59cb3" target='_blank'>
              Chemoradiation-Altered Micromilieu of Glioblastoma Cells Particularly Impacts M1-like Macrophage Activation
              </a>
            </td>
          <td>
            Mona Shojaei, Benjamin Frey, F. Putz, R. Fietkau, Udo S. Gaipl, A. Derer
          </td>
          <td>2025-07-08</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 e20057


 Background:
 Tumor metabolism reprogramming is a hallmark of cancer, but metabolite-mediated intercellular communication remains poorly understood. To address this gap, our study employed MEBOCOST, a tool for metabolite abundance estimating and communication events exploring based on single-cell RNA data, to explore the metabolic landscape of tumor microenvironment (TME) in lung adenocarcinoma (LUAD) and identify novel metabolite signaling axis.
 Methods:
 The scRNA-seq dataset was subjected to dimensionality reduction using the Seurat package. Cell annotation was manually performed using typical markers from Cell Marker 2.0 and previous studies. Single-cell metabolite abundance and communication events were inferred using MEBOCOST. The TCGA-LUAD datasets was used to estimate and analyze immune cell infiltration levels and tumor hot score using ESTIMATE and ssGSEA algorithms. Additionally, survival analysis was conducted by genes within relative signaling axis. All analysis above in TCGA-LUAD datasets was validated by two Gene Expression Omnibus (GEO) datasets.
 Results:
 Five landmark metabolites across cell types were identified as prostaglandin D2 (PGD2), D-Mannose, Choline, L-Cysteine, and Cholesterol of TME in LUAD. Prostaglandin D2 (PGD2) emerged as a key player, primarily produced by fibroblasts and plasmacytoid dendritic cells (pDCs) by via the

 PTGDS

 gene and by mast cells via the

 HPGDS

 gene. PGD2 signaling was shown to primarily be received by the PGD2 receptor (

 PTGDR

 ) on NK/T cells and transported by the

 SLCO2A1

 transporter on endothelial cells.

 CX3CR1

 + NK/T cells, which are prominent cytotoxic populations, as a PGD2 autocrine signaling axis, are involved in PGD2 autocrine signaling, while KLRC2+ NK, DNAJB1+ NK cells and CD8+ MAIT cells participate in PGD2 paracrine signaling. PGD2 may also assist lactate efflux via

 SLCO2A1

 on endothelial cells. The clinical relevance of the PGD2 signaling axis was validated across multiple bulk RNA datasets, showing that it is associated with the infiltration of above immune cells such as DNAJB1+ NK cells, and linked to better prognosis in LUAD. Furthermore, we found that a risk model developed based on this signaling axis could predict responses to immune therapy in hot and cold tumors, suggesting potential drugs that may benefit low-risk patients.
 Conclusions:
 In this study, we identified PGD2 as a critical metabolite within the LUAD tumor microenvironment, facilitating intercellular communication through its signaling axes. The analysis revealed that PGD2 mediates its tumor-suppressive effects by activating immune cells, particularly NK/T cells, which could be important for modulating anti-tumor immune responses. Our findings also highlight its potential as a biomarker for prognosis and a target for future combination therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb10f99ea1347e839a5a9503e34d601df32a2e3a" target='_blank'>
              Unveiling prostaglandin D2 signaling axis in the tumor microenvironment for immune modulation and prognosis in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Qiang Liu, Huiguo Chen, Dongfang Tang, Huibiao Zhang, Shaogeng Chen, Yiran Meng, Boying Zheng, Feifei Liu, Jing Zhou, Wen Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bbe5330dbcde126b73901d29c1db0afa6c81669" target='_blank'>
              Single-Cell Microwave Cytometry for Drug Resistance Detection in Cancer
              </a>
            </td>
          <td>
            Uzay Tefek, Hashim Alhmoud, Tieu-Lan Chau, Sayedus Salehin, Gizem Damla Yalcin, Ozge Saatci, Burge Ulukan, Scott Auerbach, Beana Puka, Ahmet Acar, Serkan Ismail Göktuna, Ilyas Chachoua, Ozgur Sahin, M. Hanay
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumor Treating Fields (TTFields) therapy is an approved cancer treatment modality, based on non-invasive application of electric fields to the tumor region. Proteomic and cell biology methods revealed a versatile mechanism of action to be involved in the response to TTFields. In the current research we performed whole transcriptome analysis across tumor types to identify pan-cancer responses to TTFields. For this we collected samples from control and TTFields-treated human cancer cell lines of gastric cancer, pancreatic cancer, ovarian cancer, non-small cell lung carcinoma, pleural mesothelioma, and glioblastoma. The transcriptomic analysis supported previous reported effects: downregulation of pathways associated with cell cycle, cell growth, and proliferation; downregulation of DNA replication and the FA-BRCA DNA repair pathway; and upregulation of cellular responses to stress—senescence, autophagy, and apoptosis. Notably, previously unrecognized downstream effects of TTFields were revealed on cellular metabolism, with downregulation of protein and RNA metabolism, and upregulation of steroid biosynthesis. Additional DNA repair pathways were also found to be downregulated, including nucleotide excision repair, base excision repair, and mismatch repair. In conclusion, this study revealed similar response patterns to TTFields across different tumor types, re-enforcing some already pinpointed mechanisms, while revealing new mechanisms. Unlocking these new mechanisms may allow identification of potential new cancer treatments for application together with TTFields based on mechanistical compatibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1627b07ec04103438850967d9453d926d5ec1e3d" target='_blank'>
              The transcriptomic fingerprint of cancer response to Tumor Treating Fields (TTFields)
              </a>
            </td>
          <td>
            K. Wainer-Katsir, A. Haber, H. Fishman, L. Ding, Michael D. Story, Renfei Du, U. Kahlert, L. Mannarino, F. Mirimao, M. Lupi, Maurizio D’Incalci, G. Lavy-Shahaf, Hila Ene, R. Frechtel-Gerzi, Zeina Drawshy, A. Martinez-Conde, E. Dor-On, Y. Porat, M. Giladi, U. Weinberg, Y. Palti
          </td>
          <td>2025-07-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 How does spatial transcriptomic profiling at a single-cell scale reveal the molecular and cellular dynamics of human endometrial regeneration during the proliferative phase?



 High-resolution spatial transcriptomics revealed previously uncharacterized cellular heterogeneity and layer-specific molecular signatures in the proliferative endometrium, including novel peri-glandular and intermediate cell populations.



 Previous studies have revealed non-pregnant endometrium’s temporal transcriptomic patterns during the menstrual cycle through single-cell RNA sequencing. However, a detailed understanding of the cellular heterogeneity in a spatial context, especially in the basalis layer and the composition of the stem cell niche, remains limited.



 A prospective observational study analyzing full-thickness endometrial specimens from nine women undergoing hysterectomy during early (n = 3), mid (n = 3), and late (n = 3) proliferative phase. Samples were analyzed using Visium HD spatial transcriptomics (8μm resolution) integrated with histological examination.



 Full-thickness endometrial samples were obtained from women aged 27-48 years with regular menstrual cycles undergoing hysterectomy for benign conditions (Ethical permit: Dnr 2021-05157 and amendment/ Dnr 2022-01635-02). Spatial transcriptomic profiling was performed on tissue sections using the Visium HD platform. Unsupervised clustering, differential gene expression analysis, spatial mapping, and gene ontology analysis were conducted to characterize cell populations and their molecular signatures.



 The study identified distinct molecular signatures in epithelial and mesenchymal lineage in functionalis and basalis layers. A novel intermediate population emerged in the stromal compartment during the mid-proliferative phase, preceding epithelial intermediate populations in the late-proliferative phase. Previously uncharacterized peri-glandular populations were discovered, with 91 and 102 differentially expressed genes in functionalis and basalis respectively during the late-proliferative phase (adjusted p < 0.05). These populations showed enrichment in different biological molecular pathways such as tissue organization, epithelial differentiation, and intermediate filament organization.



 Technical limitations include mechanical damage to the luminal epithelium in some specimens, mixed cell populations within spatial spots, and relatively low sequencing depth per spot compared to single-cell RNA sequencing. Validation through protein-level analyses and integration with single-cell sequencing data may further support the findings.



 This high-resolution spatial atlas of endometrial regeneration provides novel insights into the molecular mechanisms governing tissue renewal and cell-type specification. Identifying specialized cell populations and transitional cell states during endometrial regeneration has important implications for the understanding and development of targeted therapies for endometrial disorders.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a0bb879df84fb97d456707b175ffbc26347008" target='_blank'>
              O-085 Spatio-temporal transcriptome of the human proliferative endometrium at a single-cell scale
              </a>
            </td>
          <td>
            Y. Tang, C. Frisendahl, K. Nair, N. Rao Boggavarapu, F. Flam, H. Kopp Kallner, K. Papaikonomou, P. Lalitkumar, K. Gemzell-Danielsson
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5247ee69f9a498099d0e3c1f490a17b4768b5b00" target='_blank'>
              Metabolic reprogramming and prognostic insights in molecular landscapes driven by glycolysis in ovarian cancer.
              </a>
            </td>
          <td>
            Mingwei Wang, Qiaohui Ying, Yuncan Xing, Shuchang Dai, Jue Wang, Zhong Liu
          </td>
          <td>2025-07-24</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pediatric high-grade glioma (pHGG) with the histone H3 Lys27Met substitution (H3K27M) is a devastating disease with a high mortality rate in children and adolescents (from birth to 19 years of age). No effective treatments have been developed for this tumor type. Thus, a better understanding of the underlying complex mechanisms and identify more potential drugs targeting H3K27M-mutant pHGG are urgently needed.In the current study, we established pHGG cell models harboring H3K27M by transfecting two pHGG cell lines, SF188 and Res259, with the H3K27M mutant and H3 wild-type (WT) plasmids and then performed drugs screening. Then we employed an EJ5 reporter assay to measure nonhomologous end joining (NHEJ) activity. Western blotting was used to analyze DNA damage markers (γ-H2AX and PLK1), and cell cycle progression was assessed. Additionally, we utilized whole-exome sequencing and CRISPR/Cas9 genome editing to generate Res259 cell lines with stable deficiencies in ARID1A, P53, or PALB2, followed by viability assays to evaluate drug sensitivity. Notably, BMN673 (talazoparib) was identified as a synthetic lethal hit in the H3K27M-mutant SF188 cell model. However, BMN673 did not affect the constructed H3K27M-mutant Res259 cells. Moreover, we showed that the H3K27M mutation induced an aberrant increase in NHEJ activity. Furthermore, BMN673 treatment increased the protein levels of γ-H2AX and PLK1, induced cell cycle arrest, and promoted PARP1 trapping in H3K27M-mutant SF188 cells. In addition, the results of a series of viability assays revealed that the H3K27M mutation combined with PALB2 deficiency sensitized H3K27M-mutant Res259 cells to BMN673. However, deficiencies in ARID1A or P53 did not produce similar effects.Overall, our results may provide some reference value for further study of the effects of BMN673 and PALB2 deficiency in the treatment of H3K27M-mutant pHGG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc09735b34dfdf285d58475f9b1951e84e56ef21" target='_blank'>
              PALB2 deficiency may sensitize H3K27M-mutant pediatric HGG cells to BMN673/talazoparib
              </a>
            </td>
          <td>
            Xiaowen Guan, Xinke Xu, Xiaolan Mo, Houliang Deng
          </td>
          <td>2025-06-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36b470342bf0dd2e7ecb0177374c341bedf3ddd" target='_blank'>
              Epithelial tumor cells utilize mast cell-derived histamine to regulate perineural invasion
              </a>
            </td>
          <td>
            Ankit Srivastava, Tomas Bencomo, Chaw-Ning Lee, Angela M. Mah, Jasmine M Garcia, L. Seow, Isoline M. Donohue, Aiko J. Tan, Audrey Nguyen, Tiffany Jiang, S. Gombar, Lilian Phu, Pankaj Dwivedi, Christopher M. Rose, Ryanne Brown, Carolyn S. Lee
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e16580


 Background:
 Non-muscle invasive bladder cancer (NMIBC) presents a significant clinical challenge due to its high recurrence rate. Recently, a deeper understanding of tumor epigenetics has provided new opportunities for cancer detection and treatment. While significant efforts have been made to understand NMIBC biology, the molecular and epigenetic mechanisms driving its progression remain elusive.
 Methods:
 We used multiomics analysis including expression, DNA methylation, and cell-free DNA chromatin immunoprecipitation sequencing (cfChIP-seq) assays to uncover the molecular and epigenetic landscape of NMIBC from human tissue samples and urine.
 Results:
 High-throughput sequencing revealed more than 6,000 differentially methylated regions (DMRs) within regulatory elements, many of which were linked to cancer-related pathways and exhibited distinct signatures in cancer tissues compared to healthy controls. Next, we distinguished high and low-grade tumors based on unique DNA methylation profiles. These patterns correlated with transcription level of genes associated with cell cycle regulators, consistent with the higher proliferative capacity of high-grade tumors. Moreover, Among the DMRs, we also identified genes involved in T cell regulation. Supporting this, we demonstrated reduced CD8+ T cells infiltration in high-grade tumors, further highlighting immune dysregulation in the more aggressive disease. We extended our investigation to urine samples, performing cfChIP-seq on samples from healthy donors and NMIBC patients. By using the H3K4me3 promoter histone mark on cfDNA we could detect cell origin. Utilizing this assay, we were able to identify changes in the urine of NMIBC patients, owing to cf-nucleosome contributions from both the tumor and immune cells. Additionally, we stratified the cancer cohort by tumor grade and found enrichment in the number of differentially marked genes in high-grade tumors.
 Conclusions:
 Our detection of unique epigenetic signatures offer opportunities for biomarker development, enabling early detection through non-invasive urine based diagnostic approach. This could grade tumor more accurately and tailor personalized patient care and monitoring.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38dd958dabb53d9b68b4717906734c6068947417" target='_blank'>
              Elucidating the epigenetic landscape of non-muscle invasive bladder cancer.
              </a>
            </td>
          <td>
            T. Falick Michaeli, Batia Azria, M. Lotem, Nir Friedman
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Immunogenic cell death (ICD) is crucial for cancer development and enhances the effectiveness of immunotherapy by triggering an adaptive immune response. However, the impact of ICD-related genes (ICDRGs) on iCCA progression and patient prognosis remains unclear. iCCA samples from The Cancer Genome Atlas (TCGA) were categorized into two ICD-associated subtypes by consensus clustering, and patient prognosis in these subgroups was assessed. Single-cell RNA sequencing was conducted on viable cells from 8 iCCA patients to understand their transcriptomic profiles, heterogeneity, and immune microenvironment. Principal component analysis (PCA) and uniform manifold approximation and projection (UMAP) analysis were applied to identified 13 cell subpopulations. The gene set enrichment analysis (GSEA) was used to investigate pathway heterogeneity among these subpopulations. CellChat was employed for intercellular communication analysis. Pseudotime trajectory analyses with CytoTRACE and Monocle2 were used to explore differentiation trajectories and functional differences across various states. 30,485 single cells from human iCCA and normal tissues were sequenced, revealing two ICD-associated subtypes. The ICD-low subtype correlates with better clinical outcomes, whereas the ICD-high subtype is linked to increased immune cell infiltration and immune response signaling activity. Distinct biological functions and metabolic characteristics were observed in iCCA between high- and low-ICD groups. High infiltration of ICD-low subtype in iCCA is associated with poor prognosis, potentially through the ANXA-FPR pathway affecting macrophage proliferation and differentiation. Our study constructed an immunogenic cell death-related signature that can promise prognosis prediction and personalized medicine in iCCA. This finding might help guide clinicians for iCCA patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08602-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e45d84f628dac0dca7f3bfd929def33d5b1582" target='_blank'>
              Single cell RNA sequencing to identify an Immunogenic cell death related prognostic signature in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Bowen Sha, Fei Tong, Qinghua Shu, Miao Wu, Liang Zhao, Yufeng Zhang
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal cancers, with an average survival time of only six months following diagnosis, even with currently available therapies. Thus, PDAC represents a significant therapeutic challenge, necessitating a deeper understanding of its biology and tumor microenvironment (TME) to develop more effective treatments and improve patient outcomes. Here, we report that the expression of Eukaryotic Elongation Factor-2 Kinase (eEF2K) is associated with shorter patient survival and demonstrate that eEF2K signaling is critical for the PDAC tumor growth and regulated by the TME. Furthermore, in vivo targeted genetic inhibition of eEF2K suppressed tumor growth in two different PDAC mouse models, reduced tumor-associated macrophages (TAMs), and induced marked apoptosis in tumor tissues without any signs of toxicity. Our data suggest that eEF2K knockdown diminishes the activity of the AXL receptor tyrosine kinase and reduces the expression of macrophage-derived factors, such as Monocyte Chemoattractant Protein-1 (MCP1), along with the Gas6/AXL signaling pathway in PDAC cells. Additionally, analysis of the NCI-TCGA PDAC patient database further showed that eEF2K expression, in the presence of TAM markers, correlates with even shorter patient survival. TAM-released factors, such as MCP1, Gas6, and exosomes, induce eEF2K expression in PDAC cells, as well as the activity of AXL, SRC, VEGF, Snail, and MMP2, contributing to epithelial-to-mesenchymal transition (EMT), invasion, metastasis, and angiogenesis. In conclusion, our findings reveal for the first time that eEF2K is a critical oncogenic driver of PDAC tumor growth and thus targeting eEF2K represents a promising and novel therapeutic strategy for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca2a458a943c8611c08480c527e7133171db725" target='_blank'>
              eEF2K is a poor prognostic factor and novel molecular target in pancreatic cancer: regulating tumor growth and progression via the tumor microenvironment
              </a>
            </td>
          <td>
            Didem Karakaş, Ahmed Ashour, H. Mokhlis, N. Kahraman, R. Bayraktar, S. Dilmac, N. Kabil, M. Erdoğan, E. Dere, E. Ulukaya, B. Ozpolat
          </td>
          <td>2025-07-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a3a93f5939a6e79dedc599d6cfe047baa006cdc" target='_blank'>
              Small extracellular vesicle-miR-183-5p mediated crosstalk between tumor cells and macrophages in high-risk pancreatic neuroendocrine tumors.
              </a>
            </td>
          <td>
            Wuhu Zhang, Junfeng Xu, Dan Huang, Yi Qin, X. Lou, Heli Gao, Zeng Ye, Fei Wang, Yan Wang, Desheng Jing, Guixiong Fan, Q. Zhuo, Chang Xin, Jie Chen, Xiaowu Xu, Xianjun Yu, S. Ji
          </td>
          <td>2025-06-05</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Sex differences in breast cancer (BC) lead to distinct clinical and molecular characteristics. Sex-associated heterogeneity relies intricately on the oncogenic properties of cancer cells and multicellular interactions in tumor microenvironments. Here we conducted a multiomic analysis encompassing single-cell RNA sequencing, spatial transcriptomics and large-scale histological analysis, combined with genetic and pharmacological perturbations to systematically develop a high-resolution and spatially resolved map of intratumoral expression heterogeneity in male breast cancer (MBC). Among the malignant cells, we identify 9 consensus meta-programs (MPs, for example, estrogen response and interferon response), each consisting of dozens of genes that are coordinately upregulated in subpopulations of cells within MBC. Of these, MP5 exhibits significant male bias, characterized by aberrant activation of the estrogen receptor α (ERα) signaling pathway, inducing endocrine therapy resistance in MBC. Bidirectional interactions between MP5 malignant cells and immunosuppressive cancer-associated fibroblasts form an immune exclusionary microenvironment mediated by LRBA. Collectively, our study highlights the endocrine therapy resistant and immunosuppressive role of LRBA in MBC, and these findings may provide insights into the molecular features of MBC and facilitate the development of sex-specifical treatment strategies.
 Citation Format: Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, Jingyan Xue, Jiong Wu. Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df96734a9b8fda8be55365f44c01373b9990e46" target='_blank'>
              Abstract P3-02-26: Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer
              </a>
            </td>
          <td>
            Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, J. Xue, Jiong Wu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e660193687483a5ee1007f68a171cb159292936" target='_blank'>
              Cell Type Specific Aging Transcriptional Signatures of Human Retina Through Integrated Machine Learning and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Luning Yang, Sen Lin, Yiwen Tao, Q. Pan, T. Cai, Yunyan Ye, Jianhui Liu, Yang Zhou, Yongqing Shao, Quanyong Yi, Zen Haut Lu, Lie Chen, Gareth McKay, Richard Rankin, Fan Li, Weihua Meng
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Meningioma is the most prevalent primary brain tumor with extensive intra-tumoral heterogeneity and high recurrence rates, particularly in high-grade meningiomas. Despite advancements in understanding the molecular underpinnings of meningiomas, the longitudinal evolutionary trajectory and cellular diversity of recurrent tumors remain elusive. In this study, we perform single-nuclei sequencing of matched primary and recurrent meningiomas to explore the dynamic transcriptional heterogeneity and evolutionary trajectory of meningioma tumor cells, as well as their molecular interactions with tumor-associated immune cells that shape the complex milieu of the meningioma microenvironment. Our findings reveal that both primary and recurrent meningiomas constitute diverse cellular compositions and hierarchies, where recurrent tumor cells are characterized by enrichments of cell cycle activities and proliferative kinetics. Integrative RNA velocity and latent time uncover divergent transcriptional trajectories in recurrent tumors, demonstrating multidirectional transitions with the dominance of COL6A3, which confers higher risk vulnerability and treatment resistance. Tumor microenvironment analysis further reveals enrichments of COL6A3-mediated interactions between immunosuppressive macrophages and tumor cells in recurrent meningiomas. Collectively, these results provide profound insights into the complex evolutionary process of meningiomas and suggest potential therapeutic strategies for the treatment of recurrent tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46967d3ec0f9b8d5fcb467d9f78f81fb62b39f73" target='_blank'>
              Single-cell analysis reveals a longitudinal trajectory of meningioma evolution and heterogeneity
              </a>
            </td>
          <td>
            Ji Yoon Lee, Eun Jung Lee, Bo Yeon Seo, Jiwon Kim, Youjin Song, Dayoung Lee, Namsung Moon, Harim Koo, Chul-Kee Park, Min-Sung Kim, Serk In Park, Do-Hyun Nam, D. Kong, Jason K. Sa
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) tumors exhibit pronounced phenotypic plasticity, alternating between a treatment-sensitive classical phenotype and a more aggressive basal-like state associated with drug resistance and poor prognosis. The frequent coexistence of these phenotypes complicates patient stratification and the selection of effective therapies. Tumor-derived organoids are valuable tools for drug screening; however, their clinical relevance relies on how accurately they recapitulate the phenotypic and functional characteristics of the original tumors. In this study, we present a quantitative analysis of how hydrogel composition influences the phenotype, tissue remodeling, metabolism, and drug resistance of PDAC organoids. Organoids were cultured within three types of hydrogels: Matrigel, collagen-I, and a mixture of collagen-I and Matrigel. Our results demonstrate that: (i) PDAC organoids grown in Matrigel exhibit a classical phenotype, with metabolic and drug response profiles similar to those of low-physiological two-dimensional cultures; (ii) Organoids grown in collagen-containing hydrogels, particularly those in collagen-Matrigel composites, faithfully recapitulate basal-like tumors, characterized by epithelial-to-mesenchymal transition, tissue remodeling, metabolic activity, and drug resistance; (iii) TGFβ induces an exacerbated, highly invasive basal-like phenotype. Summarizing, our findings highlight the importance of 3D hydrogel composition in modulating PDAC organoid phenotype and behavior and suggest collagen-Matrigel hydrogels as the most suitable matrix for modeling PDAC biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24bfd79f1227aca7f9356ac5488b713b766f1a87" target='_blank'>
              Tumor Organoids Grown in Mixed-Composition Hydrogels Recapitulate the Plasticity of Pancreatic Cancers.
              </a>
            </td>
          <td>
            I. Sorzabal-Bellido, Xabier Morales, I. Cortés-Domínguez, Maider Esparza, Lucía Grande, Pedro Castillo, Silvia Larumbe, María Monteserín, Shruthi Narayanan, Mariano Ponz-Sarvise, Silve Vicent, Carlos Ortiz-de-Solórzano
          </td>
          <td>2025-07-21</td>
          <td>Gels</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="RNA 7‐Methylguanosine (m7G) modification is biologically important to tumorigenesis and progression. Knowledge regarding the effect of m7G modification on the clinical features of hepatocellular carcinoma and its prognosis, tumor immune microenvironment, and sensitivity to immunotherapy remains limited.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/513ee1fd25996c86fd289f55080a85e47f630064" target='_blank'>
              Integrated Analysis of Single‐Cell and Bulk RNA‐Sequencing Defines N7‐Methylguanosine (m7G)‐Mediated Modifications' Role in Prognosis and the Tumor Immune Microenvironment in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Jiahua Liang, Mingjian Ma, Borui Xu, Cheng Zhong, Guangyan Zeng, Huijiao Lu, Jiaming Lai, Jiancong Chen
          </td>
          <td>2025-06-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The high-purity and low-loss reacquisition of viable circulating tumor cells (CTCs) is crucial for enabling downstream omics analysis of CTCs and currently represents key challenges limiting their application in clinical diagnosis and pathological research. Given the limitations of traditional methods that rely solely on a molecular or mechanical phenotype for CTCs acquisition, this study introduces an innovative approach that fuses the inherent molecular and mechanical phenotypes of CTCs into a new mechanical phenotype, thereby achieving high-purity preconcentration and low-loss reacquisition of CTCs. Specifically, CTCs in blood are immunomodified using calcium carbonate microspheres (CCMSs) conjugated with antibodies, transforming the molecular phenotype (membrane protein expression) into an additional mechanical phenotype (increased size and reduced deformability). This transformation enhances the mechanical phenotype distinctions between CTCs and white blood cells, enabling high-purity preconcentration of CTCs on a single-cell trapping array chip. Since CCMSs can be reversibly eliminated under weak acid, captured CTCs can be nondestructively reacquired with 93.10% in microliter-scale solution, allowing for subsequent omics analysis. In a breast cancer mouse model, the counts and transcriptome analysis of CTCs provide valuable insights into assessing tumor occurrence and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9223ce042a9fd299657dbf2e0c9090278c6558a" target='_blank'>
              Fusion of Molecular and Mechanical Phenotypes Enables High-Purity and Low-Loss Reacquisition of Viable CTCs for Transcriptome Analysis.
              </a>
            </td>
          <td>
            Haibin Si, Qing Xu, Yan Sun, Dexin Du, Yiguo Wang, Simin Li, Lu Li, Bo Tang
          </td>
          <td>2025-05-30</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Temozolomide (TMZ) remains the cornerstone chemotherapy for glioma, yet intrinsic and acquired resistance mechanisms significantly limit its clinical effectiveness. This review summarizes the multifaceted molecular pathways contributing to TMZ resistance, including enhanced DNA repair mechanisms such as O6-methylguanine-DNA methyltransferase (MGMT), mismatch repair (MMR), and base excision repair (BER). Additional resistance factors include genetic mutations that affect the drug response, dysregulated non-coding RNAs (miRNAs, lncRNAs, and circRNAs), glioma stem cells (GSCs), cytoprotective autophagy, an immunosuppressive tumor microenvironment (TME), altered signaling pathways, and active drug efflux transporters. Recent advancements to overcome these resistance mechanisms, including enhancing TMZ bioavailability through nanoparticle-based delivery systems and the inhibition of efflux transporters, have been explored. Novel therapeutic approaches that target DNA repair pathways and manipulate autophagy are highlighted. Immunotherapeutic interventions reversing immune suppression and metabolic strategies targeting tumor metabolism offer additional avenues. Emerging therapies such as CRISPR-based gene editing, phytochemical combinations, repurposed drugs, and novel TMZ analogs designed to bypass MGMT-mediated resistance are also discussed. This review highlights current developments and identifies emerging areas, with the goals of enhancing clinical outcomes and prolonging survival for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1529a07572c2d8270f0f0a8a2d6f28b9530af6c1" target='_blank'>
              Overcoming temozolomide resistance in glioma: recent advances and mechanistic insights
              </a>
            </td>
          <td>
            Hengzeng Li, Yahui Wu, Yue Chen, Jinquan Lv, Chengkang Qu, Tingjie Mei, Yunfan Zheng, Cheng Ye, Feifei Li, Shuo Ge, Anhui Yao, Liyun Jia
          </td>
          <td>2025-06-05</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background/Objectives: Aneuploidy is near-ubiquitous in cancer and can decrease chemotherapy efficacy while also sensitizing cells to other drugs. Methods: To systematically identify treatment strategies that target aneuploid cancers, data were integrated from The Cancer Genome Atlas (TCGA; 10,967 samples, 16,948 aneuploidy events) and the Broad Institute’s Profiling Relative Inhibition Simultaneously in Mixtures (PRISM) screen of 578 cancer cell lines and 4518 compounds. Results: Our analyses uncovered 37,720 significant positive and negative associations linking specific aneuploidies and treatments with patient prognosis or cell viability. Within TCGA data, 22 treatments correlated with improved 5-year survival for specific aneuploid cancers, whereas 46 were linked to worse outcomes. A complementary analysis of PRISM identified 17,946 compound–aneuploidy associations and 16,189 mechanism of action (MOA)–aneuploidy associations. Pathway-altering compounds that selectively reduce viability in cells with aneuploidy profiles were discovered, including an unexpectedly prominent number of glucocorticoid receptor agonists. Conclusions: This integrated dataset provides a resource for designing therapeutic decision hypotheses, identifying drug-repurposing opportunities, and informing future studies aimed at targeting aneuploidy-induced vulnerabilities in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811f8e93329a18a966c2c968f4a1e0431dca5e53" target='_blank'>
              Half the Chromosome It Used to Be: Identifying Cancer Treatments Targeting Aneuploid Losses
              </a>
            </td>
          <td>
            Andrew O. Disharoon, Joe R. Delaney
          </td>
          <td>2025-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tongue Squamous Cell Carcinoma (TSCC) represents a significant subtype of malignant oral cancer, characterized by a heterogeneous tumor microenvironment (TME). The tongue, a complex muscular organ, naturally presents an initial microenvironment that is largely inhospitable to the initiation and progression of TSCC. However, advanced-stage TSCC exhibits a pronounced accumulation of cancer-associated fibroblasts (CAFs), indicative of a drastic microenvironmental transformation. In this study, through comprehensive analysis combining cell model assessments and single-cell RNA sequencing data from a 4-NQO-induced TSCC mouse model, along with a lineage tracing-in-transplant assay, we elucidate and confirm the process of transdifferentiation whereby tongue muscle cells (TMCs) convert into CAFs in the TSCC context. Furthermore, we demonstrate that targeting TSCC with an IL-17a inhibitor offers a viable strategy to inhibit the reprogramming of TMCs into CAFs. Additionally, our research also identifies four critical marker genes involved in the transdifferentiation of TMCs to CAFs, Thbs1, Crabp1, Ifi205, and Cxcl5. Collectively, these findings delineate a mechanism through which TSCC cells induce the transdifferentiation of TMCs into CAFs, thereby transforming the cancer-suppressive microenvironment of the tongue into one that is conducive to TSCC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/666df9c3fc7fca0534d4dd3d77f31ac2e7399d3f" target='_blank'>
              Transdifferentiation of tongue muscle cells into cancer-associated fibroblasts in response to tongue squamous cell carcinoma
              </a>
            </td>
          <td>
            Weifan Lin, Wu-hua Huang, Shiqiang Mei, Xiangwan Lu, Junheng Zheng, Hua Wang, Lingling Zheng, Yan Zhang
          </td>
          <td>2025-07-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancer (BC) remains the most prevalent malignancy among women worldwide, with incidence and mortality rates varying across regions due to disparities in screening, diagnosis, and treatment accessibility. Molecular classification has redefined BC management, categorizing tumors into luminal A, luminal B, human epidermal growth factor receptor 2 (HER2)-enriched, and triple-negative breast cancer (TNBC), each with distinct prognostic and therapeutic implications. Advances in tumor microenvironment (TME) research highlight its critical role in cancer progression, immune evasion, and treatment resistance, underscoring the importance of tumor-infiltrating lymphocytes, natural killer cells, and macrophages in shaping BC outcomes. Traditional therapies, including chemotherapy, endocrine therapy, and targeted treatments such as cyclin-dependent kinase 4 and 6 (CDK4/6) and phosphatidylinositol-4,5-biphosphate 3-kinase (PI3K) inhibitors, continue to evolve to overcome resistance mechanisms. Immunotherapy, particularly immune checkpoint inhibitors targeting programmed cell death protein 1 (PD1), programmed cell death ligand 1 (PD-L1), and cytotoxic T-lymphocyte-associated antigen 4 (CTLA4), has demonstrated clinical benefits, especially in TNBC. Additionally, novel strategies, including chimeric antigen receptor T-cell therapy and cancer vaccines, are emerging as promising therapeutic avenues. This review provides a comprehensive analysis of BC epidemiology, molecular subtypes, TME interactions, and cutting-edge therapeutic strategies, emphasizing the need for personalized, immune-based approaches to enhance treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0043dff1dd8bee49cb1b948c4ad201d4a1764bd5" target='_blank'>
              Unlocking New Frontiers in Breast Cancer: The Role of the Tumor Microenvironment, Cutting-Edge Therapies, and Immunotherapy
              </a>
            </td>
          <td>
            Doua Kamyan, Noha M. Elemam, Noura Alkhayyal, Iman M. Talaat, Riyad Bendardaf
          </td>
          <td>2025-06-27</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c15cd4c65cc7acb2163700609aefaa2af8b33ee" target='_blank'>
              Integrating bulk and single-cell RNA sequencing reveals intratumor heterogeneity phenotypes and immune infiltration in pancreatic cancer.
              </a>
            </td>
          <td>
            Li Zhao, Yao Huang, Jiang Li, Rui Xie, Kaiming Li
          </td>
          <td>2025-07-13</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Growth differentiation factors (GDFs) are a subfamily of the TGF-β superfamily whose expression increases in response to cellular stress and disease. Despite emerging cell- or animal-based evidence supporting an association between the GDF subfamily and cancer, systematic pan-cancer analyses of the GDF subfamily based on single-cell and spatial transcriptomes remains unavailable. In this study, we performed a comprehensive analysis of the GDF subfamily in 33 cancers, including expression, diagnosis, methylation, prognostic value, immune infiltration analysis, and potential biological pathways. We focused on the analysis of multi-group scRNA-seq and spatial data in hepatocellular carcinoma (HCC) to determine the role of the GDF family in the tumor microenvironment and its applicability in immunotherapy. Moreover, both the gain and loss of function strategies were used to assess the function of Growth differentiation factor 15 (GDF15) in cell lines of HCC. The GDF subfamily is expressed to varying degrees in most tumors and is significantly correlated with the prognosis of cancer patients. Subsequent scRNA-seq analysis depicted the heterogeneous cellular ecosystems of normal liver and HCC. Hepatocytes expressing GDF15 were less differentiated in HCC, and GDF15 promoted proliferation and invasion of HCC cell lines. Compared to normal liver, the strength of crosstalk between GDF15-positive Hepatocytes and other cells was enhanced in tumors, especially cancer-associated fibroblasts (CAFs)-derived Periostin and GDF15-positive Hepatocytes both regulate each other and jointly promote hepatocarcinogenesis. Further spatial transcriptomic data showed that GDF15 expression was negatively correlated with immune infiltration, especially in M1-type macrophages. Notably, validation analyses in bulk RNA-seq consistently emphasized the clinical significance of these findings. This study provides a comprehensive overview of the oncogenic role of the GDF subfamily in a wide range of tumors, highlights the important role of GDF15 in HCC ecosystem, and provides important biomarkers and potential therapeutic targets for future research. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01431-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf430182b60a21eff474221ea80860ad6db3d890" target='_blank'>
              Single-cell and spatial analyses of the GDF family in tumors, with a focus on the prognostic and biological role of GDF15 in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Xiaoqian Feng, Qian Huai, Fumin Zhang, Wenkang Yuan, Xingyu Li, Zhuo Yu, Hao Zhang, Yaoling Zhu, Xu Zhang, Baole Tao, Ying Dai, Yishan Du
          </td>
          <td>2025-06-27</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT As effective melanoma treatments have become available, utilizing isolated limb perfusion (ILP) to treat unresectable melanoma limited to the limb has decreased. However, some patients still receive long‐term benefits from ILP. We aimed to identify features of the pretreatment tumor microenvironment (TME) to identify patients who may benefit from ILP. Pretreatment metastatic melanoma samples from 22 patients treated at Helsinki University Hospital with ILP from 2008 to 2018 were analyzed with multiplex immunohistochemistry (mIHC) and digital image analysis. Antibody panels evaluated the proportions of immune cells in the intratumoral and extratumoral compartments. We examined whether treatment response and median progression‐free survival (PFS) after ILP correlated to findings in the TME. A statistically significant positive correlation was found between PFS and lower immune cell infiltrations in the intratumoral compartment (CD3+, CD4+, and CD11c+ cells), and increased numbers of immune cells in the extratumoral compartment were associated with longer PFS (CD3+, CD4+, CD8+, all expressing PD‐1). Furthermore, the distribution of some immune cell subsets correlated with complete treatment response (PD‐1/PD‐L1‐positive CD4+ and PD‐1‐positive CD8+ cells). Our results suggest that patients may have a better ILP outcome if the metastases exhibit a lower distribution of specific immune cell subtypes intratumorally and a higher extratumoral distribution of some immune cell subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cbd05461d5ecc220dad6e0d54a06e12fce0af6c" target='_blank'>
              Association of Pretreatment Tumour Microenvironment and Treatment Outcome in Patients With Locally Advanced Melanoma Treated With Isolated Limb Perfusion
              </a>
            </td>
          <td>
            Sanni K A Tulokas, Susanna Juteau, Siru P Mäkelä, K. Välimäki, T. Pellinen, M. Hernberg
          </td>
          <td>2025-07-01</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 5038


 Background:
 Antitumor immunity has become a critical focus in improving Penile squamous cell carcinoma (PSCC) patient outcomes. Among various immune features, tertiary lymphoid structures (TLS) have emerged as crucial players in tumor immunity. Nevertheless, the specific role of B cells within TLS and their interaction with naive T cells in PSCC remain largely unclear. Therefore, this study aims to elucidate the role of CD74⁺ B cells in TLS and their impact on immune regulation in PSCC.
 Methods:
 Spatial transcriptomics produced 519,608 cellular profiles from tumor samples of 18 patients. Additionally, scRNA-seq on tissues from 21 patients yielded 150,540 cells. Bulk RNA-seq was performed on tissues from 55 patients. TLS were identified and visualized using the DBSCAN algorithm, which clusters B cells, T cells, and DCs. Survival analyses were conducted to examine the association between TLS density and patient outcomes. Moreover, immune activity and cell infiltration were assessed via AUCell and CIBERSORT, while pseudotime analysis was used to explore dynamic gene expression changes during cell development. Additionally, pathway enrichment analysis identified key signaling pathways, and intercellular signaling was analyzed through cell communication models.
 Results:
 TLS, accurately identified using the DBSCAN algorithm, were significantly associated with improved prognosis in PSCC patients (Internal cohort, n = 152, P < 0.01; External cohort, n=63, P < 0.05). The spatial distribution, a high density of B cells was observed within the TLS regions. Notably, CD74⁺ B cells were enriched within TLS, particularly during early developmental stages. Mapped the spatially in situ developmental trajectory of CD74⁺ B cells and uncovered the dynamic changes in gene expression throughout their maturation process, we observed that CD74⁺ B cells within the TLS of PSCC patients predominantly exhibited features of early developmental stages. In line with this, scRNA-seq data further validated these observations. These cells, through their HLA molecules, interacted with CD4/CD8 ligands on naive T cells, thereby activating critical transcription factors such as NFKB1, NFKB2, NFATC1, NFATC2, FOS, and RUNX1. This interaction ultimately amplified immune responses within the tumor microenvironment. Furthermore, patients with higher CD74⁺ B cell expression exhibited better responses to immunotherapy (pCR : P < 0.01and CR: P < 0.01) and underscoring the therapeutic relevance of these cells.
 Conclusions:
 By activating naive T cells through antigen presentation, CD74⁺ B cells within TLS significantly enhance local immune responses in PSCC. Thus, CD74⁺ B cells not only serve as a promising biomarker but also represent a potential therapeutic target, providing novel insights into the immunological mechanisms underlying PSCC progression and response to immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f359d9e9ac8454c440cb7ea9056b23fcf8d8cd2" target='_blank'>
              Spatial transcriptional dynamics of CD74⁺ B cells in tertiary lymphoid structures and effects on immune evolution in penile squamous cell carcinoma.
              </a>
            </td>
          <td>
            Zaishang Li, Ting Xue, Bonan Chen, Hui Han
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Chemoresistance and recurrence following treatment are the greatest impediments to the prognosis of glioblastoma (GBM). Increasing evidence indicates that cancer-associated fibroblasts (CAFs) play a significant role in the progression of glioblastoma. Nevertheless, the role and source of CAFs in recurrent and chemotherapy-resistant GBMs still remain ambiguous. Methods Spatial transcriptome (ST) sequencing was conducted on the tissue microarray encompassing primary and recurrent glioma samples in order to characterize the cellular composition. Subsequently, the infiltration of CAFs in our formerly established in vivo temozolomide (TMZ)-resistant model was inspected through immunohistochemical staining. Additionally, we carried out RNA-seq and label-free quantitation (LFQ) proteomics on HCMECs co-cultured with TMZ-sensitive (TMZ-S) or TMZ-resistant (TMZ-R) cells to explore the mechanism. Results This investigation revealed that CAFs and astrocytes are enriched in recurrent GBM, and this phenotype is associated with the expression of extracellular matrix (ECM) proteins associated with COL1A1 and FN1 deposition. Further investigations revealed that tenascin-C (TNC) and filamin C (FLNC), which potentially mediate endothelial-to-mesenchymal transition (EndMT), are the predominant factors that induce the deposition of ECM proteins in the resistance-promoting microenvironment. Additionally, the natural product punicalin (PNC) was found to downregulate EndMT-related proteins, multidrug resistance-associated membrane proteins, and collagen-related proteins by targeting TNC and FLNC, thereby increasing the susceptibility of temozolomide (TMZ)-resistant cells to chemotherapeutic agents both in vitro and in vivo. Conclusion These discoveries indicate that TNC and FLNC induced EndMT was a key resource of CAFs and targeting TNC and FLNC to inhibit EndMT and the collagen pathway is a promising tactic for reversing drug resistance in tumours. The development of combined chemotherapeutic strategies based on the features of tumour microenvironment endothelial cells and ECM deposition has high potential clinical value in increasing the efficacy of tumour treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06743-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90f87d542212bc590894a08468baa2631e0e5527" target='_blank'>
              Spatial transcriptomics and multi-omics reveal relapse and resistance mechanisms of EndMT-derived CAFs mediated by TNC and FLNC in glioblastoma
              </a>
            </td>
          <td>
            Li Ji, Die Xia, Yu Zhou, Yaling Hu, Zhenkun Yang, Ying Yin, Jingjing Wang, Bo Zhang, Lingli Gong, Koukou Li, Jian Zou, Mei Wang
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="FOXA1 is altered in 10 to 40% of prostate cancers, yet its oncogenic mechanisms remain uncharacterized in vivo. We developed knock-in mouse models representing distinct classes of FOXA1 mutations. Histopathological and multi-omic analyses of prostate tissues and organoids revealed that Class 1 mutations, in conjunction with p53 inactivation, drive androgen-dependent adenocarcinomas through co-activation of mTORC1/2 and oncogenic AR signaling stemming from chimeric AR-half enhancers. In contrast, Class 2 mutations induce intra-luminal plasticity by reprogramming differentiated luminal cells into a progenitor-like state through activation of KLF5 and AP-1 neo-enhancer circuitries, which enables enhanced survival and proliferation even under castrate androgen levels. Our findings establish FOXA1 as a multifaceted oncogene, with distinct mutational classes divergently evolving to drive prostate tumorigenesis or therapy-resistant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877a995b94079d1a62d50951659ac85f4910e75e" target='_blank'>
              Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity.
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, J. Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin P. Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2025-06-26</td>
          <td>Science</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9cc29097aec34e28c1bf167f3394c3e8fef4685" target='_blank'>
              Informed Data-Independent Acquisition Enables Targeted Quantification of Key Regulatory Proteins in Cell Fate Decision at Single-Cell Resolution
              </a>
            </td>
          <td>
            Jakob Woessmann, Valdemaras Petrosius, Sofie Schovsbo, T. Arrey, B. Furtwängler, Jeff Op de Beeck, E. Damoc, Bo Porse, E. Schoof
          </td>
          <td>2025-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91ec8476ea04c15a90514f720adb0e97fd88bcc" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, Zhongqi Ge, Jaehyun Lee, Rishi Pasumarthi, Xiaohong Leng, Thomas Koehnke, O. Pasvolsky, Pranaya Raparla, Vinhkhang N. Nguyen, K. Maurer, Shuqiang Li, K. Livak, Emerie Danson, Bijal Thakkar, Elham Azizi, Robert J. Soiffer, S. Shukla, Ravi Majeti, Jerome Ritz, Catherine J. Wu, J. Molldrem, P. Bachireddy
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 e14557


 Background:
 The immune microenvironment of breast cancer is richly infiltrated with macrophages, yet modulation targeting macrophage polarization has failed to achieve significant clinical efficacy.
 Methods:
 We applied techniques such as parabiosis mouse models, multicolor immunofluorescence, bone marrow transplantation models and flow cytometry to explore in depth the characteristics of the immune microenvironment of breast cancer as well as the mechanisms of origin and replenishment of tissue-resident macrophages.
 Results:
 We found two interesting conclusions. Firstly, triple-negative breast cancer, which is considered to be one of the most biologically malignant subtypes, instead has a significantly reduced M2-like infiltration, and secondly, about 40% of the tissue-resident macrophages within the mammary tissues are derived from the embryonic lineage and exhibit a long lifecycle, proliferative and pluripotent properties. Through single-cell sequencing, we further found that the abnormally high expression status of IRF7/CH25H/25HC in triple-negative breast cancers remodeled the immune microenvironment of triple-negative breast cancers by regulating the expression levels of MHC and PD-1 in macrophages, especially targeting macrophages of embryonic lineage origin to play a significant regulatory role.
 Conclusions:
 The in-depth exploration and specific regulation of macrophage source and replenishment mechanisms are expected to be an effective modulator for regulating macrophage polarization within the TNBC immune microenvironment as well as anti-PD-1 therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aadf73921bfe5089111e87d63014df503b6c111e" target='_blank'>
              Triple-negative breast cancer remodeling of the tissue immune microenvironment by regulation of embryonic-derived tissue-resident macrophage polarization through the IRF7/CH25H/25HC pathway.
              </a>
            </td>
          <td>
            Yalong Yang, Xin-hong Wu, Hongmei Zheng
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Loss-of-function mutations in genome maintenance genes fuel tumorigenesis through increased genomic instability. A subset of these tumor suppressors are challenging to identify due to context dependency, including functional interactions with other genes and pathways. Here, we searched for potential causal genes that impact tumor development and/or progression in breast cancer through functional-genetic screening of candidate genes. MYH4, encoding a class II myosin, emerged as a top hit impacting genomic stability. We show that MYH4 suppresses DNA replication stress by promoting replication licensing and replication fork progression. Moreover, we observed a strong synergistic relationship among class II myosins in suppressing replication-associated DNA damage. Genomic analysis of Pan-Cancer Analysis of Whole Genomes project breast cancer samples revealed frequent concomitant loss of TP53 with MYH4 and class II myosins on chromosome 17p. Notably, Myh4 disruption accelerated mouse mammary tumorigenesis in a Trp53-deficient background. In conclusion, our results suggest an unanticipated function of MYH4 in p53-mediated tumor suppression that can explain their combined loss in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64806f05c07b8d24d35ea0d2ce3f2fd4c89aa895" target='_blank'>
              The class II myosin MYH4 safeguards genome integrity and suppresses tumor progression
              </a>
            </td>
          <td>
            J. Thatte, Ana Moisés da Silva, Judit Börcsök, Thorkell Gudjónsson, Jan Benada, Xin Li, Muthiah Bose, H. van der Gulden, Ji-Ying Song, Renée Menezes, Elena Martín-Doncel, Luis Toledo, Valdemaras Petrosius, Cord Brakebusch, J. Jonkers, F. C. Nielsen, Maria Rossing, C. S. Sørensen
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="BACKGROUND
Primary central nervous system lymphoma (PCNSL) is a highly aggressive subtype of non-Hodgkin lymphoma that is confined to the central nervous system. PCNSL is associated with a poor 5-year survival rate of 30-40%, partly due to a high recurrence rate of 60%. A comprehensive understanding of the molecular signatures and mechanisms underlying drug resistance in PCNSL is crucial, as it has significant implications for therapeutic strategies.


METHODS
A cohort of 56 newly diagnosed PCNSL patients was generated. The tumor specimens were obtained by biopsy. A combined approach, including single-cell transcriptomics and B-cell receptor sequencing, transcriptome-informed multiplex immunohistochemistry, and ex-vivo drug response assays, was applied to reveal the transcriptional and immune microenvironment landscape of PCNSL.


RESULTS
We identify four main B cell subtypes, each characterized by distinct transcriptomic profiles, from malignant B cells in PCNSL. Among these subtypes, plasmablast-like lymphoma cells (PBLCs), characterized by upregulated genes typically associated with plasma cell differentiation, comprise 1.3%-8.1% of malignant B cells and correlate with poor prognosis in PCNSL. PBLCs exhibit reduced expression of CD20, Bruton tyrosine kinase, and FAS, resulting in drug resistance and immune evasion. In addition, PBLCs elevate the expression of critical transcription factors, including XBP1 and PRDM1, to maintain their phenotype. Clinical drugs, such as Lenalidomide, demonstrate potential cytotoxic effects on PBLCs.


CONCLUSIONS
These findings highlight PBLCs as a distinct subtype of malignant B cells that plays a critical role in the multidrug resistance of PCNSL and reveal the molecular signature of PBLCs that can be targeted for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17a9a2cca6ca455e6dde63a6892673c09fa8c29d" target='_blank'>
              Plasmablast-like lymphoma cells as a distinct subpopulation confers multidrug resistance in PCNSL.
              </a>
            </td>
          <td>
            Feng Liang, Xiaotao Zhang, An Ping, Yichen Gu, Ziyang Jin, Huaming Li, Lingxiao Lu, Fan Yang, Junjie Wang, Jianan Lu, Feng Zhou, Jinfang Xu, Jingsen Chen, Junming Zhu, Fengqiang Liu, Jianmin Zhang, Ligen Shi
          </td>
          <td>2025-06-21</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6f92731e4153548f5adf8b4c57b2dcf0276aa2" target='_blank'>
              Single-cell transcriptomics uncovers that cell-to-cell communication controls p53 activation
              </a>
            </td>
          <td>
            L. Castellano, Mark E Robinson, Aleksandra Dabrowska, Niklas Feldhahn, S. Ottaviani, Justin Stebbing
          </td>
          <td>2025-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadf2932e90bcc5681634229bcd158ed1d080d8b" target='_blank'>
              Single-cell Characterization of DNA Hydroxymethylation of the Mouse Brain During Aging
              </a>
            </td>
          <td>
            Yali Bai, Tianjiao Yuan, Liuhao Ren, Yuhang Huan, Fa Yang, Yutong Li, Aicui Zhang, Yueyang Liu, Tian Tian, Ningxia Kang, Danni Chen, Xuchao Xie, Qiang Wang, Wei Chen, Yinghui Zheng, Xiaoliang Sunney Xie, Yunlong Cao
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Background: Recent research has spotlighted lobular cancer as a distinct histological phenotype, potentially revolutionizing treatment paradigms. Invasive lobular carcinoma (ILC) is identified by a specific morphological feature, notably the "Indian file" arrangement of discohesive breast cancer cells. Frequently, these cancers are estrogen receptor positive, though they can appear within any molecular subtype of breast cancer. Emerging evidence indicates that, within the same molecular subtype, ILC leads to less favorable outcomes compared to invasive ductal carcinoma (IDC). These outcomes include higher rates of metastatic disease, lower response rates to both neoadjuvant and adjuvant treatments, and decreased overall and disease-free survival. Such findings strongly support reclassifying ILC as a distinct entity, advocating for a paradigm shift in therapeutic approaches. Mechanistic Insights into ILC The development of ILC is mechanistically linked to the loss of E-cadherin, a crucial epithelial cell-cell adhesion molecule. This loss is directly associated with changes in cell nanomechanical compliance and tumor microenvironment nanomechanical remodeling. This study presents the first clinically integrated, functional characterization of ILC tissues at the nanomechanical level, providing new insights into the physical properties of ILC and offering novel avenues for clinical management. Methods: This investigation utilized data from a single-center, blinded, prospective study aimed at measuring the multiparameter nanomechanical signature of breast cancer. Conducted between 2016 and 2019, the clinical study included 588 fresh breast biopsy samples from 545 suspected breast cancer patients. Samples were measured using the AFM-based Automated and Reliable Tissue Diagnostics (ARTIDIS) investigational device within a routine clinical setting at the Breast Clinic, University Hospital Basel (Switzerland). Patients who underwent clinically indicated breast biopsy and consented to participate were eligible for the study. Biopsies were collected before treatment, and patients are being followed up to collect long-term responses for up to 10 years. Core needle biopsies from 125 invasive breast cancer patients were selected for nanomechanical profiling of ILC and IDC, including 12 pure ILC and 113 pure IDC cases. The nanomechanical signature integrated analysis was performed using the proprietary ARTIDISNet software platform. Results: This study presents the first comprehensive nanomechanical signature of ILC compared to IDC, including stiffness, adhesion, and dissipation profiles and their association with other known clinico-histopathological aggressive features. The specific mechanical nature of ILC was confirmed through spatial analysis of multiplex imaging from several lobular breast cancer patients. This characterization supports the investigation and definition of ILC as a unique nanomechanical subtype of breast cancer. Conclusion: The loss of E-cadherin in ILC is directly related to the loss of cell-cell adhesions, epithelial-mesenchymal transition, cell invasion, migration, and metastasis, all of which are mechanical aspects of ILC pathogenesis. This study presents the first integrated characterization of the nanomechanical phenotype of ILC, which will be further validated in the ongoing multicenter ANGEL clinical trial, currently enrolling over 2700 patients in the USA over the next three years. Understanding and fully characterizing the mechanical nature of ILC has the potential to revolutionize clinical management of this unique disease.
 Citation Format: Marko Loparic, Sara Nizzero, Mariam Gachechiladze, Tobias Appenzeller, Leonie Briner, Rosemarie Burian, Sabine Schädelin, Simone Muenst-Soysal, Tatjana Vlaijnic, Ellen Obermann, Sophie Dellas, Serafino Forte, Zlatko Marušić, Ahmed Jizawi, Yitian Xu, Lee B. Jordan, Colin A. Purdie, Philip R. Quinlan, Chandandeep Nagi, Karla A. Sepulveda, Gregory Zaugg, Papa Diogop Ndiaye, Philipp Oertle, Vittorio Cristini, Alastair M. Thompson, Marija Plodinec. The Nanomechanical Phenotype of Lobular Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-05-29.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e30633e25061be839587da180a08b7d6e9064854" target='_blank'>
              Abstract P1-05-29: The Nanomechanical Phenotype of Lobular Breast Cancer
              </a>
            </td>
          <td>
            M. Loparic, S. Nizzero, Mariam Gachechiladze, T. Appenzeller, L. Briner, R. Burian, S. Schädelin, Simone Muenst-Soysal, Tatjana Vlaijnic, E. Obermann, S. Dellas, Serafino Forte, Zlatko Marušić, A. Jizawi, Yitian Xu, Lee B. Jordan, C. A. Purdie, P. Quinlan, C. Nagi, Karla A. Sepulveda, G. Zaugg, P.D. Ndiaye, P. Oertle, Vittorio Cristini, Alastair M. Thompson, M. Plodinec
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="ABSTRACT Tumor growth is a challenge for multicellular life forms. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we identified two key metabolic changes: first, systemic depletion of acetyl-CoA leading to a reduction in histone acetylation levels and stochastic silencing of actively transcribed genes; and second, defects in the methionine cycle causing systemic depletion of S-adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process inhibits tumor growth. To understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors and found that human tumors with metabolic signatures similar to those of fly tumors have a lower mutational load, younger patient age and lower DNA methylation levels. This study indicates that depletion of key metabolites is an evolutionarily ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a97bbe13457619149961b6416a76b3eeed9ee44" target='_blank'>
              Epigenetic reprogramming induced by key metabolite depletion is an evolutionarily ancient path to tumorigenesis
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2025-06-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Boolean networks are powerful frameworks for capturing the logic of gene-regulatory circuits, yet their combinatorial explosion hampers exhaustive analyses. Here, we present a systematic reduction of a 31-node Boolean model that describes cisplatin- and pemetrexed-resistance in non-small-cell lung cancer to a compact 9-node core that exactly reproduces the original attractor landscape. The streamlined network shrinks the state space by four orders of magnitude, enabling rapid exploration of critical control points, rules fitting and candidate therapeutic targets. Extensive synchronous and asynchronous simulations confirm that the three clinically relevant steady states and their basins of attraction are conserved and reflect resistance frequencies close to those reported in clinical studies. The reduced model provides an accessible scaffold for future mechanistic and drug-discovery studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9ea0359bb1347a85175036c7bd062b0dfddee48" target='_blank'>
              Boolean-network simplification and rule fitting to unravel chemotherapy resistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            Alonso Espinoza, Eric Goles, Marco Montalva-Medel
          </td>
          <td>2025-06-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is an aggressive form of cancer, characterized by limited treatment options and a poor prognosis. Immunological therapy is an emerging and promising strategy that has the potential to enhance treatment outcomes and extend the survival of patients with ICC. The role of CD4+ T cells in the development of cancer has attracted attention in previous years. However, the complexities of the tumor microenvironment (TME) impede the full understanding of the roles of CD4+ T cells in cancer. The present study used single-cell RNA sequencing to explore the heterogeneity of the TME during the development of ICC. The results demonstrated that CD4+ T cells were enriched in the TME of ICC and the ratio of regulatory T cells (CD4-forkhead box P3) to central memory T cells (CD4-interleukin 7 receptor) was markedly increased. Secreted phosphoprotein 1 (SPP1) and CD44 showed increased expression levels in tumor cells and T cells from ICC tumor tissues, respectively. Additionally, SPP1 gene expression levels were higher and the ratio of regulatory T cells to central memory T cells was increased in the late stage of ICC compared with the early stage. Elevated levels of SPP1 were associated with a poor prognosis for patients with ICC. Finally, analysis of cell-cell interactions, utilizing established receptor-ligand pairs, demonstrated that ICC tumor cells may engage with immune cells through an SPP1-CD44 axis. Therefore, the results suggest that ICC tumor cells impact CD4+ T-cell differentiation, which could alter the immune TME in ICC and potentially promote ICC tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cf95ea2ea04750f960c2db9f956491fba608b38" target='_blank'>
              Single-cell RNA sequencing analysis of intrahepatic cholangiocarcinoma reveals SPP1 facilitates disease progression via interaction with CD4+ T cells
              </a>
            </td>
          <td>
            Xian Lin, Huanyan Peng, Ke Liu, Wenqi Chen, X. Shao, Chun Ning, Hongchang Li, Dongye Yang
          </td>
          <td>2025-06-10</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 e17084


 Background:
 Six-Transmembrane Epithelial Antigen of the Prostate 1 (STEAP1) is a transmembrane protein frequently overexpressed in prostate cancer (PC). Antibody-based therapies targeting STEAP1 have shown promising preclinical and clinical outcomes. Despite its widespread and PC-specific overexpression, the molecular regulation and biological function of STEAP1 remain poorly understood. Elucidating the mechanisms that govern STEAP1 expression and its role in PC progression could provide valuable insights into disease mechanisms, improve patient selection for STEAP1-targeted therapies, and inform combination or sequential treatment strategies.
 Methods:
 To investigate STEAP1 regulation and function, we first compared its expression to established PC driver genes and observed a correlation between STEAP1 and the androgen receptor (AR) as well as several AR-regulated genes. Using pathway analyses in the Decipher (localized prostate cancer) and West Coast Dream Team (WCDT, metastatic prostate cancer) clinical cohorts, we examined dysregulated pathways in patients with high STEAP1 expression. Additionally, we performed a FACS-based whole-genome CRISPR interference (CRISPRi) screen in C4-2B cells using an anti-STEAP1 antibody to identify regulators by comparing STEAP1-high and STEAP1-low expressing subsets.
 Results:
 In the WCDT dataset, STEAP1 was highly expressed in AR-positive metastatic prostate cancer subtypes, irrespective of neuroendocrine differentiation, but was significantly lower in AR-negative subtypes. STEAP1 expression strongly correlated with several AR-regulated genes, including STEAP2, KLK2, NKX3-1, KLK3, ARHGAP6, SLC45A3, and PMEPA1. Pathway analyses in both localized and metastatic PC cohorts revealed dysregulated metabolic pathways, particularly lipid metabolism, in STEAP1-high patients. The CRISPRi screen identified several potential targetable regulators of STEAP1, including MYC, SWI/SNF chromatin remodeling complexes.
 Conclusions:
 Our study establishes a strong association between STEAP1 and AR-driven metastatic castration-resistant prostate cancer subtypes. Furthermore, patient data suggest that STEAP1 is linked to dysregulated metabolic pathways, particularly those involving lipid metabolism. The identification of STEAP1 regulators through the CRISPRi screen provides valuable insights into its regulation and biological function, offering potential avenues for developing combination therapies, strategies to prevent treatment resistance, and improved patient stratification for STEAP1-targeted treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e2f4b4ddfd1356bdf18bff71926fd89045fb54e" target='_blank'>
              FACS-based whole-genome CRISPRi screen to uncover STEAP1 regulators in prostate cancer.
              </a>
            </td>
          <td>
            Shan Zhu, Mikhail Dias, Yuxin Yang, Emma Dolan, Thaidy Moreno Rodriguez, Tamilla Nechiporuk, Jun Zhu, Derek Xiang, Tianyi Liu, Tatyanah Farsh, Meng Zhang, Gabriel Eades, Stanley Leung, Lisa N Chesner, Peter Bruno, M. Cooperberg, David A Quigley, F. Feng, Haolong Li, Carissa E. Chu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Background/Objectives: Cell-based therapies have become increasingly important in the treatment of cancers and inflammatory diseases; however, therapies utilizing monocyte–macrophage lineage cells remain relatively underexplored. Non-invasive cell tracking allows a better understanding of the fate of such cells, which is essential for leveraging their therapeutic potential. Here, we employed a Zirconium-89 (89Zr)-oxine cell labeling method to compare the trafficking of monocytes and macrophages in vivo. Methods: Mouse bone marrow-derived monocytes and macrophages were each labeled with 89Zr-oxine and evaluated for their viability, radioactivity retention, chemotaxis, and phagocytic function in vitro. Labeled cells were intravenously administered to healthy mice and to murine models of granuloma and syngeneic tumors. Cell migration was monitored using microPET/CT, while cell recruitment to the lesions was further assessed via ex vivo biodistribution and flow cytometry. Results: Labeled cells exhibited similar survival and proliferation to unlabeled cells for up to 7 days in culture. While both maintained phagocytic function, monocytes showed higher CCL2-driven chemotaxis compared to macrophages. 89Zr-oxine PET revealed initial cell accumulation in the lungs, followed by their homing to the liver and spleen within 2–24 h, persisting through the 5-day observation period. Notably, monocytes trafficked to the liver and spleen more rapidly than macrophages. In both inflammation and cancer models, monocytes demonstrated higher accumulation at the lesion sites compared to macrophages. Conclusions: This study demonstrates the usefulness of 89Zr-oxine PET in tracking monocyte–macrophage lineage cells, highlighting their distinct migration patterns and providing insights that could advance monocyte-centered cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93216eecd6c03c566fd6443bf7bbb1f3530ea5ef" target='_blank'>
              Zirconium-89-Oxine Cell Tracking by PET Reveals Preferential Monocyte Recruitment to Cancer and Inflammation over Macrophages
              </a>
            </td>
          <td>
            S. Koyasu, Hannah A. Minor, K. O. Asiedu, Peter L Choyke, Noriko Sato
          </td>
          <td>2025-06-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 5072


 Background:
 STEAP
 1 and 2 (six-transmembrane epithelial antigen of prostate) are metalloreductase proteins involved in a variety of biologic processes.
 STEAP1/2
 are tumor-associated cell surface antigens highly expressed in prostate cancer (PC), although their role in cancer is poorly understood.
 STEAP1/2
 have emerged as successful targets for adoptive T-cell therapy trials for PC. We employed a multi-omics approach to investigate the molecular features associated with
 STEAP1
 and
 STEAP2
 expression in PC.
 Methods:
 NextGen Sequencing of DNA (592 genes or whole exome) and RNA (whole transcriptome) was performed for PC tumors (n = 7089) submitted to Caris Life Sciences (Phoenix, AZ). PC samples were stratified by
 STEAP1/2
 mRNA levels into top (high) and bottom quartile (low). Immune cell infiltration in the tumor microenvironment (TME) was inferred by quanTIseq. Transcriptomic signatures of androgen receptor signaling (AR), neuroendocrine classification (NEPC), and interferon gamma signaling (IFN) were calculated. Mann-Whitney U and X2/Fisher-Exact tests were applied where appropriate, with P-values adjusted for multiple comparisons (q< .05).
 Results:
 Of 7,089 samples, 63.2% were from the prostate; 11.7% from lymph node metastases (LNM); 7.3% from bone; and 17.8% from visceral/soft tissue metastases (V/STM).
 STEAP
 -1 and -2 were significantly correlated to each other (R= 0.90, p<.001), with significantly higher expression of
 STEAP1
 observed in primary prostate and LNMs, compared with reduced expression in V/STM (
 STEAP1
 TPM: 105.2 vs 140.6 vs 91.9 p<.001). Mutations in
 AR
 (3.8% v 1.9%),
 KDM6A
 (4.2% v 2.2%),
 SPOP
 (10.9% v 8.4%) and
 AR
 V7 (23.0% v 10.5%) were enriched in
 STEAP1
 high PC (each q<.01). Mutations in
 KDM6A
 (3.8% v 2.5%) and
 AR
 V7 (17.6% v 14.3%) were enriched in
 STEAP2
 high PC (each q<.05).
 STEAP1/2
 expression negatively correlated with TMB count (R =-0.03, p<.05) and IFN score (R = -0.26, p<.001). Concordantly, fewer proinflammatory immune cell fractions (M1 Macrophages, NK cells, CD4+/CD8+ T cells, myeloid dendritic cells) were observed within the TME of
 STEAP1/2
 high PC (p<.0001). However,
 STEAP1/2
 expression correlated positively with the AR signature (R = 0.39, p<.001) and androgen response pathways, while correlating negatively with the NEPC signature (R = -0.15, p<.001).
 Conclusions:
 PC tumors expressing high
 STEAP1/2
 display distinct genomic and transcriptomic profiles compared to
 STEAP1/2
 -low PC, and
 STEAP 1/2
 expression varies across sites of metastases. Immune biomarkers and immune cell infiltration data suggest that STEAP1/2 may be associated with a cold TME. The recent success of STEAP1-targeting T-cell redirecting therapies mechanisms by which adoptive T-cell strategies may overcome immunosuppressive factors within the TME. Ongoing development of T-cell immunotherapeutics targeting STEAP1 may account for the differential expression profiles in guiding patient selection and combination strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3db32deff1c58e26b5169abd853a15c53e234839" target='_blank'>
              Molecular characterization of STEAP1 and -2 in advanced prostate cancer.
              </a>
            </td>
          <td>
            Kevin K. Zarrabi, T. Adeyelu, A. Elliott, D. Geynisman, D. Oh, Carissa E. Chu, Rana R McKay, Nicholas A. Zorko, E. Antonarakis, Lucia R Languino, Pedro C. Barata, D. Danila, Norm Smith, William Kevin Kelly
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 4154


 Background:
 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with a dismal prognosis despite advances in treatment. PDAC is characterized by a dense stromal environment with suppressed anti-tumor immunity that contributes to treatment resistance. Oncogenic
 KRAS
 mutations are present in > 90% of PDACs and are known to play a role in modulating the tumor immune microenvironment (TME). In this study, we examined the differences in TME components between
 KRAS
 -mutant (m) and
 KRAS
 -wildtype (wt) PDACs.
 Methods:
 Comprehensive genomic and immune profiling (CGIP), including the RNA-seq based gene expression assessment of 395 immune-associated genes, was performed on 311 PDAC patient samples. Gene expression signatures of tumor immunogenicity (TIGS) and cell proliferation were calculated by averaging the gene expression ranks of 161 immune-associated genes and 10 proliferation genes, respectively. The normalized gene expression rank of 22 immune checkpoint genes and 17 cancer testis antigen (CTA) genes were also calculated. DNA-seq was used to identify
 KRAS
 mutations and to calculated tumor mutational burden (TMB). Continuous variables were compared between subgroups using the Wilcoxon Rank-Sum test and categorical variables were compared between groups using the Chi Squared test. For statistical significance, p < 0.05 was required.
 Results:
 The cohort consisted of 311 PDAC patient samples, comprising 159 females (51.1%) and 152 males (48.9%) with a median age at testing of 69.3 years (38.7-92.6). A total of 264 specimens (84.9%) exhibited a
 KRAS
 mutation, with the most common being G12D and G12V while 47 (15.1%) had
 KRAS
 -wt tumors. No difference in TIGS, CP, or TMB was observed between
 KRAS
 -wt and
 KRAS
 -m tumors.
 KRAS
 -wt tumors exhibited greater expression of 12 of 17 tested CTAs than
 KRAS
 -m tumors, including GAGE13 (p = 0.004) NY-ESO-1 (p = 0.01), MAGEA3 (p = 0.01), and LAGE1A (p = 0.01). Analysis of the gene expression of 22 immune checkpoint genes showed no difference for most of the genes, though there was higher expression of PD-1 (p = 0.04) and CD27 (p = 0.03) seen in
 KRAS
 -wt compared to
 KRAS
 -m tumors.
 Conclusions:
 KRAS
 -wt PDACs exhibited greater expression of cancer testis antigen genes compared to
 KRAS
 -m tumors, suggesting potential therapeutic susceptibility to immunotherapy and adoptive cell therapies leveraging the expression of CTAs as targets. Assessment of
 KRAS
 status and immunotherapy susceptibility may support future clinical trial selections for therapies targeting the complex interplay of genomic and immune components of pancreatic cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ae6d1fe40ee2b5375e715aee2512cfe9d76d6fb" target='_blank'>
              Unveiling the differences in tumor immune microenvironment between
 KRAS
 -wildtype and
 KRAS
 -mutant pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Heidi Ko, S. Pabla, Michelle F Green, M. Senosain, Hardik Parikh, Erik Van Roey, Shuang Gao, Yamuna Pulivendula, P. DePietro, Stuti Kaushalkumar Patel, Kyle C. Strickland, R. Previs, Erin Newburn, Kamal S Saini, Alicia Dillard, Eric A Severson, Brian Caveney, Taylor J. Jensen, Shakti H. Ramkissoon, R. Seager
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Neuroendocrine (NE) prostate cancer (NEPC) is an aggressive and lethal subtype of prostate cancer. It is typically characterized by the expression of NE markers and the loss of androgen receptor expression. De novo NEPC is rare, accounting for <2% of all prostate cancer cases at diagnosis. More commonly, NEPC arises from prostate adenocarcinoma following androgen deprivation therapy, with 20‑25% of metastatic castration‑resistant prostate cancers undergoing NE differentiation due to lineage plasticity. During this transition, pathways associated with epithelial‑mesenchymal transition (EMT) and stemness are broadly activated, which is considered to be a key driver of NEPC's high metastatic potential, resistance to chemotherapy and radiotherapy and poor prognosis. EMT facilitates metastasis by enhancing cellular motility and invasiveness, while stemness properties contribute to post‑metastatic colonization, immune evasion, therapy resistance and cellular dormancy. As manifestations of cellular plasticity, these processes share overlapping molecular mechanisms. Targeting key regulators within these pathways may offer promising therapeutic strategies for NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30582503232b9c92c62c32d091a4ab49e49ea5bf" target='_blank'>
              Widespread activation and critical role of EMT and stemness in the neuroendocrine differentiation of prostate cancer (Review).
              </a>
            </td>
          <td>
            Yun-Fan Li, Shuai Su, Yu Luo, Chengcheng Wei, Jingke He, Liangdong Song, Kun Han, Jue Wang, Xiangzhi Gan, De-Lin Wang
          </td>
          <td>2025-07-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f4e379ab91d9bf34171ea689775c7a7a55d58d" target='_blank'>
              Regulatory elements of pancreas development license the initiation of pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Marta Ballester, Anita Kurilla, Yifan Dai, H. C. Maurer, Elisa Espinet, Kristina Høj, Aida Marisch Delgado, P. A. Seymour, Saynab Omar, C. Rift, Jane Hasselby, P. Klausen, Peter Vilmann, Albin Sandelin, Luis Arnes
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background T-cell suppression in patients with Acute myeloid leukemia (AML) limits tumor cell clearance. This study aimed to explore the role of T-cell senescence-related genes in AML progression using single-cell RNA sequencing (scRNA-seq), bulk RNA sequencing (RNA-seq), and survival data of patients with AML in the TCGA database. Methods The Uniform Manifold Approximation and Projection (UMAP) algorithm was used to identify different cell clusters in the GSE116256, and differentially expressed genes (DEGs) in T-cells were identified using the FindAllMarkers analysis. GSE114868 was used to identify DEGs in AML and control samples. Both were crossed with the CellAge database to identify aging-related genes. Univariate and multivariate regression analyses were performed to screen prognostic genes using the AML Cohort in The Cancer Genome Atlas (TCGA) Database (TCGA-LAML), and risk models were constructed to identify high-risk and low-risk patients. Line graphs showing the survival of patients with AML were created based on the independent prognostic factors, and Receiver Operating Characteristic Curve (ROC) curves were used to calculate the predictive accuracy of the line graph. GSE71014 was used to validate the prognostic ability of the risk score model. Tumor immune infiltration analysis was used to compare differences in tumor immune microenvironments between high- and low-risk AML groups. Finally, the expression levels of prognostic genes were verified using polymerase chain reaction (RT-qPCR). Results 31 AMLDEGs associated with aging identified 4 prognostic genes (CALR, CDK6, HOXA9, and PARP1) by univariate, multivariate, and stepwise regression analyses with risk modeling The ROC curves suggested that the line graph based on the independent prognostic factors accurately predicted the 1-, 3-, and 5-year survival of patients with AML. Tumor immune infiltration analyses suggested significant differences in the tumor immune microenvironment between low- and high-risk groups. Prognostic genes showed strong binding activity to target drugs (IGF1R and ABT737). RT-qPCR verified that prognostic gene expression was consistent with the data prediction results. Conclusion CALR, CDK6, HOXA9, and PARP1 predicted disease progression and prognosis in patients with AML. Based on these, we developed and validated a new AML risk model with great potential for predicting patients’ prognosis and survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84874cd54294efd5d73e09436e1f2629d12de384" target='_blank'>
              Integrated single-cell and bulk RNA dequencing to identify and validate prognostic genes related to T Cell senescence in acute myeloid leukemia
              </a>
            </td>
          <td>
            Mengyao Sha, Jun Chen, Haifeng Hou, Huaihui Dou, Yan Zhang
          </td>
          <td>2025-06-25</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The maintenance of corneal epithelial homeostasis relies on limbal stem cells (LSCs) located at the limbus. Although short-term cultured LSC transplantation effectively treats LSC deficiency, prolonged culture leads to stemness loss and abortive colony formation, and the mechanisms remain elusive. In this study, we employed single-cell transcriptomics to investigate LSC population dynamics and changes in gene expression during extended serial culture. Transcriptomic data from 22,708 cells revealed 19 clusters, identifying three distinct limbal progenitor populations (Progenitors 1-3) with unique transcriptional profiles and cell division kinetics. All progenitor subgroups expressed stemness-related genes such as ANLN, AURKB, and HMGB2 and were detected at all stages of the cell cycle. Notably, Progenitor3 exhibited the highest levels of genes associated with stemness and the G2/M checkpoint, including ANLN, PLK1, AURKA, HMGB2, and TOP2A, and had the largest proportion of cells in G2/M. Progenitor2 was marked by histone H1 expression, while Progenitor1 displayed distinctive cell cycle kinetics. Despite stable proportions of the three progenitor populations throughout prolonged passaging, mitochondrial gene downregulation and ribosomal gene upregulation were observed. Treatment with the small molecule RepSox partially preserved LSC maintenance in long-term culture by inhibiting the epithelial-mesenchymal transition program and modulating energy and metabolic pathways. These findings provide insights for optimizing in vitro LSC expansion for cell-based therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c61a63910b20f5d3b61a31f2e795436164e7fdd" target='_blank'>
              Single-cell analysis of heterogeneity and molecular changes in cultured corneal epithelial stem cells during serial passage.
              </a>
            </td>
          <td>
            U. Reinprayoon, Supaporn Khramchantuk, Natthida Kittimawikrom, P. Ingrungruanglert, Sarinya Phodang, Thanakorn Jaemthaworn, S. Sriswasdi, N. Israsena
          </td>
          <td>2025-06-17</td>
          <td>Stem cells</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) remains a leading cause of cancer mortality, necessitating novel therapeutic targets and prognostic strategies. This study investigates the role of extracellular matrix cancer-associated fibroblasts (eCAFs) and their interaction with SPP1+ macrophages in LUAD progression and prognosis. Utilizing single-cell RNA sequencing from 15 LUAD tumors and integrating multi-cohort transcriptomic data (TCGA, GSE31210, GSE72094), we identified eCAFs as a dominant CAF subtype in advanced-stage tumors and high-grade pathological subtypes, correlating with poor patient survival. Similarly, SPP1+ macrophages exhibited increased abundance in advanced tumors and adverse prognosis. Pseudotime trajectory analysis revealed eCAFs as an evolutionary endpoint in CAF differentiation, associated with extracellular matrix remodeling pathways (COLLAGEN, FN1). Cell-cell communication analysis highlighted eCAFs-SPP1+ macrophage interactions via COL1A1-CD44 and COL1A2-CD44 ligand-receptor pairs, suggesting a mechanism for immune-excluded microenvironments. A prognostic model incorporating 28 eCAFs-related genes, validated through 101-machine learning algorithms, effectively stratified patients into high- and low-risk groups across cohorts. This study underscores eCAFs as key drivers of LUAD progression and proposes their interplay with SPP1+ macrophages as a therapeutic target. The developed prognostic signature offers clinical utility for risk stratification, though further experimental validation is warranted. These findings advance understanding of stromal-immune crosstalk in LUAD and highlight ECM remodeling as a critical pathway in tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e15a1f46f5ea3fc3bda02ce35fca45d178c19c92" target='_blank'>
              Integration of Multi-Scale Profiling and Machine Learning Reveals the Prognostic Role of Extracellular Matrix-Related Cancer-Associated Fibroblasts in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Ziyi Chen, Mengyuan Chen, Changqing Yang, Jiajing Wang, Yuan Gao, Yuanying Feng, Dongqi Yuan, Peng Chen
          </td>
          <td>2025-06-12</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Advancements in single-cell RNA sequencing (scRNA-seq) have revealed the phenotypic and functional diversity of tumor-associated macrophages (TAMs), identifying specific populations that directly impact the antitumor response. However, despite the recognition of TAMs as promising therapeutic targets for cancer treatment, research is hindered by the lack of validated human preclinical models. Here, we applied scRNA-seq to a 3D human cell-based model comprising tumor cell line-derived spheroids, cancer-associated fibroblasts and primary monocytes, a setup widely used in immuno-oncology research. Integration of our in vitro data with publicly available patient-derived datasets showed that the macrophages in this model share phenotypic characteristics with the pro-angiogenic and pro-fibrotic SPP1+ TAM population recently found across multiple cancer types and inflammatory lung diseases. This population was linked to aspects of disease progression and associated with poor prognosis in several tumor indications, highlighting the need for relevant models enabling its study as an immunotherapy target. Our research validates the use of a 3D human cell-based culture as a more in vivo-relevant model and enables the preclinical testing of novel macrophage-targeting drugs in a human disease-relevant setup.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef82db36fe6d49b8f7be594b232edbec1e6c18b" target='_blank'>
              Cross-disease integration of single-cell RNA sequencing data from lung myeloid cells reveals TAM signature in in vitro model
              </a>
            </td>
          <td>
            Catarina Pinto, Jakub Widawski, Sophie Zahalka, Barbara Thaler, Linda C Schuster, Samuel W. Lukowski, Fidel Ramírez, Iñigo Tirapu
          </td>
          <td>2025-05-31</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Introduction: Adding pembrolizumab to conventional neoadjuvant cytotoxic chemotherapy (NAC) improved pathologic complete response rate (pCR) and event free survival (EFS), which has become practice changing treatment. Furthermore, EFS benefit has been maintained even in non-pCR patients. Here, we are to explore the role of pembrolizumab in terms of tumor microenvironment using a spatial transcriptomic analysis. Methods: We examined a prospective cohort of patients for residual TNBC after NAC with Visium Spatial Gene Expression (Visium V4 Slide - FFPE v2, 10X Genomics). Data was preprocessed with Spacer Ranger, aligned to GRCh38, and analyzed using the Seurat R package. Expression data was normalized and batch effect corrected with SCTransform and RunHarmony. Cell types and Hallmark pathway scores were determined with AddModuleScore. Differential gene expression and gene set enrichment analysis was performed using FindMarker in Seurat and ClusterProfiler. Broader cell types were deconvoluted using a reference with the spacexr R package. Tumor borders were assessed with the SpaCET R package. Results: In total, 20 cases of residual TNBCs were analyzed. Of 20 cases, sixteen TNBCs had been treated with pembrolizumab (PEM) plus NAC and four with only NAC. In residual cancer burden (RCB) class, four were class I (three in PEM+NAC and one in NAC), fifteen in class II (twelve in PEM+NAC and three in NAC) and one in class III (PEM). Four patients have experienced distant metastasis after standard adjuvant treatment (three in PEM+NAC and one in NAC group). In DEG analysis, we found CXCL9, CXCL10 and CXCL13 increased in PEM+NAC whereas CCL18 and CD36 in NAC group (FDR <0.01, respectively). GSEA result was shown that interleukin 18(IL18) and interferon gamma (IFNG) response pathway was significantly up-regulated in PEM+NAC group compared to those in NAC group. Cell type scores of CD4+T cell, CD8+T cell, M1 macrophage (Mac) and cancer cell were higher in PEM+NAC group, whereas M2 Mac, myCAF as well as iCAF were higher in NAC group (FDRs < 0.01). The distance between cancer and cytokines including CXCL9 and CXCL10 were closer in PEM+NAC group compared to the that in NAC group (FDRs <0.005). Otherwise, the distance from cancer cell to IFNG, IFNA and CCL5 were not different between PEM+NAC and NAC groups. Furthermore, we analyzed the interaction between cancer cells and immune cells. In tumor associated Macs, CXCL9 and CXCL10 expressed Mac were nearer by cancer cells in PEM+NAC group than NAC group even though SPP1 Mac also closer to cancer cells in PEM+NAC group compared to NAC group (Ps <0.05). Moreover, CD69 and PDCD1 expressed exhausted CD8+ T cells were farther from cancer cell in PEM+NAC group compared to distance in NAC group. However, there was no different distance from cancer cells to IFNG and IFN alpha expressed CD8+ T cells between two groups. Conclusion: After PEM+NAC, residual TNBC more closely interacted CXCL9+/CXCL10+ macrophages even though SPP1+ macrophages with hypoxia also be near to tumor cells compared to those in NAC only. In addition, PEM+NAC expelled exhausted CD8 T cells from cancer in this spatial transcriptomic analysis. Based on these finding, we suggested that adding pembrolizumab to NAC would modulate tumor infiltrating immune cells, which may contribute better survival outcomes.
 Citation Format: Kyunghee Park, Junghoon Shin, Jin Seok Ahn, Jeong Eon Lee, Jonghan Yu, Jai Min Ryu, Se Kyung Lee, Byung Joo Chae, Seok Won Kim, Seok Jin Nam, Wong Yang Park, Yoon La Choi, Yeon Hee Park. Spatial transcriptomic analysis for tumor infiltrating immune microenvironment modulation by neoadjuvant pembrolizumab with chemotherapy for patients with triple negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8ab5c5267e22b826632d9941bdf96debfc9108" target='_blank'>
              Abstract P2-11-08: Spatial transcriptomic analysis for tumor infiltrating immune microenvironment modulation by neoadjuvant pembrolizumab with chemotherapy for patients with triple negative breast cancer
              </a>
            </td>
          <td>
            Kyunghee Park, Junghoon Shin, J. Ahn, Jeong-Eon Lee, J. Yu, J. Ryu, Se Kyung Lee, Byung-Joo Chae, S. Kim, S. Nam, Wong Yang Park, Y. La Choi, Y. Park
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The ovary is one of the first organs to exhibit signs of aging, characterized by reduced tissue function, chronic inflammation, and fibrosis. Multinucleated giant cells (MNGCs), formed by macrophage fusion, typically occur in chronic immune pathologies, including infectious and non-infectious granulomas and the foreign body response, but are also observed in the aging ovary. The function and consequence of ovarian MNGCs remain unknown as their biological activity is highly context-dependent, and their large size has limited their isolation and analysis through technologies such as single-cell RNA sequencing. In this study, we define ovarian MNGCs through a deep analysis of their presence across age and species using advanced imaging technologies as well as their unique transcriptome using laser capture microdissection. MNGCs form complex interconnected networks that increase with age in both mouse and nonhuman primate ovaries. MNGCs are characterized by high Gpnmb expression, a putative marker of ovarian and non-ovarian MNGCs. Pathway analysis highlighted functions in apoptotic cell clearance, lipid metabolism, proteolysis, immune processes, and increased oxidative phosphorylation and antioxidant activity. Thus, MNGCs have signatures related to degradative processes, immune function, and high metabolic activity. These processes were enriched in MNGCs compared to primary ovarian macrophages, suggesting discrete functionality. MNGCs express CD4 and colocalize with T-cells, which were enriched in regions of MNGCs, indicative of a close interaction between these immune cell types. These findings implicate MNGCs in modulation of the ovarian immune landscape during aging given their high penetrance and unique molecular signature that supports degradative and immune functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0ec0f2ab3b4559b8ff2732272bfe0e5b9e9f82e" target='_blank'>
              Multinucleated giant cells are hallmarks of ovarian aging with unique immune and degradation-associated molecular signatures
              </a>
            </td>
          <td>
            Aubrey Converse, Madeline J Perry, Shweta S. Dipali, J. Isola, Emmett B. Kelly, Joseph M. Varberg, M. Zelinski, Michael B. Stout, Michele T. Pritchard, Francesca E. Duncan
          </td>
          <td>2025-06-01</td>
          <td>PLOS Biology</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="
 e14564


 Background:
 Sarcoma remains a challenging solid cancer to treat in the modern era. The role of mast cells in the tumor microenvironment continues to evolve. In the resting state, mast cells function as reservoirs for histamines and cytokines. In the activated state, mast cells degranulate upon encountering pathogens, initiating an immune response. In breast cancer, mast cell presence is associated with a better prognosis in hormone receptor-positive tumors, whereas hepatocellular carcinoma shows the opposite trend. Therapeutic strategies aimed at controlling the tumor microenvironment may also play a role in early detection and prevention of tumor progression. We aim to further understand the role of mast cells in sarcomas.
 Methods:
 mRNA sequencing data for 235 sarcoma patients were obtained from The Cancer Genome Atlas (TCGA) and analyzed using CIBERSORT, a Stanford-developed application that quantifies immune cell composition within a tumor sample. Each patient sample was broken down into percentages of various immune cells, including B-cells naïve, B-cells memory, plasma cells, CD8 T-cells, CD4 T-cells naïve, CD4 T-cells memory resting, CD4 T-cells memory activated, T-cells follicular helper, T-cells regulatory Tregs, T-cells gamma delta, natural killer resting cells, natural killer activated cells, monocytes, macrophages (M0, M1, M2), resting dendritic cells, activated dendritic cells, resting mast cells, activated mast cells, eosinophils, and neutrophils. Once the immune cell fractions were obtained, a statistical tool named Radiant by Shiny was used to perform a linear regression to evaluate the relationship between overall survival and the different immune cells.
 Results:
 Resting mast cells were independently prognostic of overall survival in multivariable analysis (p = 0.005). The correlation coefficient was +104, indicating a positive association between the number of resting mast cells and overall survival.
 Conclusions:
 Patients with a higher number of resting mast cells in their sarcoma microenvironment had longer overall survival. Unlike activated mast cells, which promote inflammation and tumor progression, resting mast cells help maintain immune balance within the tumor microenvironment, potentially leading to a better immune response. One hypothesis is that resting mast cells promote normal vascular function, whereas activated mast cells contribute to pathological angiogenesis. Activated mast cells secrete factors such as histamine and VEGF, which can drive tumor progression, whereas resting mast cells have a secretory profile that is less conducive to cancer spread. Additionally, resting mast cells may remodel the extracellular matrix in a manner that makes it more difficult for cancer to metastasize. These findings position mast cells as potential prognostic tools and therapeutic targets in the world of cancer treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/668706690938107f32f68bd1ea29d734564c032f" target='_blank'>
              Association of resting mast cell infiltration in the sarcoma microenvironment with overall survival.
              </a>
            </td>
          <td>
            Paul F Hanona, Lorin Sallan, Morgan Kiryakoza, Neil Jairath, Daniel Ezekwudo, George Howard, I. Jaiyesimi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [4],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>