<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-12-25 12:38:28 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The tumor microenvironment (TME) is a complex network of interactions between malignant and host cells, yet its orchestration in advanced high-grade serous ovarian carcinoma (HGSC) remains poorly understood. We present a comprehensive single-cell spatial atlas of 280 metastatic HGSCs, integrating high-dimensional imaging, genomics, and transcriptomics. Using 929 single-cell maps, we identify distinct spatial domains associated with phenotypically heterogeneous cellular compositions, and demonstrate that immune cell co-infiltration at the tumor-stroma interface significantly influences clinical outcomes. To uncover the key drivers of the tumor ecosystem, we developed CEFIIRA (Cell Feature Importance Identification by RAndom forest), which identified tumor cell-intrinsic MHC-II expression as a critical predictor of prolonged survival, independent of clinicomolecular profiles. Validation with external datasets confirmed that MHC-II-expressing cancer cells drive immune infiltration and orchestrate spatial tumor-immune interactions. Our atlas offers novel insights into immune surveillance mechanisms across HGSC clinicomolecular groups, paving the way for improved therapeutic strategies and patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/684d76f91d1a8b1bc3e208feca47d927bec6a2ec" target='_blank'>
              Single-cell spatial atlas of high-grade serous ovarian cancer unveils MHC class II as a key driver of spatial tumor ecosystems and clinical outcomes
              </a>
            </td>
          <td>
            Fernando Pérez-Villatoro, Lilian van Wagensveld, Aleksandra Shabanova, Ada Junquera, Ziqi Kang, Iga Niemiec, Matias M. Falco, Ella Anttila, J. Casado, Eric Marcus, Essi Kahelin, Foteini Chamchougia, M. Salko, Saundarya Shah, Salvatore Russo, Jacopo Chiaro, Mikaela Grönholm, G. Sonke, K. K. van de Vijver, Rutgerus Fpm Kruitwagen, Maaike Avan der Aa, A. Virtanen, Vincenzo Cerullo, Anna Vähärautio, P. Sorger, Hugo M. Horlings, A. Färkkilä
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Neuroendocrine and tuft cells are rare, chemosensory epithelial lineages defined by expression of ASCL1 and POU2F3 transcription factors, respectively1,2. Neuroendocrine cancers, including small cell lung cancer (SCLC), frequently display tuft-like subsets, a feature linked to poor patient outcomes3–13. The mechanisms driving neuroendocrine–tuft tumour heterogeneity, and the origins of tuft-like cancers are unknown. Using multiple genetically-engineered animal models of SCLC, we demonstrate that a basal cell of origin (but not the accepted neuroendocrine origin) generates neuroendocrine–tuft-like tumours that highly recapitulate human SCLC. Single-cell clonal analyses of basal-derived SCLC further uncovers unexpected transcriptional states and lineage trajectories underlying neuroendocrine–tuft plasticity. Uniquely in basal cells, introduction of genetic alterations enriched in human tuft-like SCLC, including high MYC, PTEN loss, and ASCL1 suppression, cooperate to promote tuft-like tumours. Transcriptomics of 944 human SCLCs reveal a basal-like subset and a tuft-ionocyte-like state that altogether demonstrate remarkable conservation between cancer states and normal basal cell injury response mechanisms14–18. Together, these data suggest that the basal cell is a plausible origin for SCLC and other neuroendocrine-tuft cancers that can explain neuroendocrine–tuft heterogeneity—offering new insights for targeting lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a9f4695f47fd01e98ecf709150d28f3a1f7f97" target='_blank'>
              Basal cell of origin resolves neuroendocrine–tuft lineage plasticity in cancer
              </a>
            </td>
          <td>
            A. Ireland, Sarah B. Hawgood, Daniel A. Xie, Margaret W. Barbier, Scarlett Lucas-Randolph, Darren R. Tyson, Lisa Y. Zuo, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Sonam Puri, Charles M. Rudin, J. Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="We conducted a spatial analysis using imaging mass cytometry applied to stage III colorectal adenocarcinomas. This study used multiplexed markers to distinguish individual cells and their spatial organization from 52 colorectal cancers. We determined the landscape features of cellular spatial features in the CRC tumor microenvironment. This spatial single-cell analysis identified 10 unique cell phenotypes in the tumor microenvironment that included stromal and immune cells with a subset which had a proliferative phenotype. These special features included spatial neighborhood interactions between single cells as well as different tissue niches, especially the tumor infiltrating lymphocyte regions. We applied a robust statistical analysis to identify significant correlations of cell features with phenotypes such as microsatellite instability or recurrence. We determined that microsatellite stable (MSS) colorectal cancers had an increased risk of recurrence if they had the following features: 1) a low level of stromal tumor-infiltrating lymphocytes, and 2) low interactions between CD4+ T cells and stromal cells. Our results point to the utility of spatial single-cell interaction analysis in defining novel features of the tumor immune microenvironments and providing useful clinical cell-related spatial biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc76d98a517403a79b6bbdaa67daa9d1d6c7537" target='_blank'>
              The single-cell spatial landscape of stage III colorectal cancers
              </a>
            </td>
          <td>
            Andrew Su, Hojoon Lee, Minh Tran, Richard Cruz, A. Sathe, Xiangqi Bai, Ignacio A. Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, D. Haslem, David Jones, Lincoln Nadauld, Quan H Nguyen, Hanlee P. Ji, Terence Rhodes
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Embryogenesis is commonly viewed through a tree model of cell differentiation, which does not adequately represent the spatiotemporal modulation of cell multipotency underlying morphogenesis. Here we develop an integrated approach, combining in vivo single-cell high-throughput clonal lineage tracing with machine learning, to systematically decompose continuous spectra of clonal fate biases in mouse embryos traced from neurulation until mid-gestation. The reconstructed patterns of clonal variation uncovered gene programs driving dynamic positional biasing of clonal composition during axial skeletogenesis and peripheral neurogenesis. Mosaic combinatorial perturbations targeting multiple receptors, including the Hedgehog pathway, led to novel clone types, which has implications for engineering custom cell type assemblages from well-defined progenitors in vivo. Altogether, our work demonstrates an effective practical approach for interrogating programs guiding lineage specification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5676798f4b41e9176c4597c7d9b91163b9f17a43" target='_blank'>
              Unbiased profiling of multipotency landscapes reveals spatial modulators of clonal fate biases
              </a>
            </td>
          <td>
            A. Erickson, Sergey Isaev, Artem Artemov, Jingyan He, Bettina Semsch, Aliia Murtazina, Jia Sun, Katrin Mangold, Anthi Chalou, Jonas Frisén, Michael Ratz, E. Andersson, P. V. Kharchenko, I. Adameyko
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Tracing the fate of individual cells and their progeny remains a challenging task. While imaging-based fate-mapping provides spatial information, it generally lacks complexity due to limited label diversity, resulting in diminished capture of comprehensive lineages and fate maps. Here, we introduce ‘Polytope’, an epitope barcoding system capable of generating up to 512 unique color codes. Comprising nine epitope tag cassettes flanked by loxP sites, Polytope allows random excision via Cre recombinase, creating unique color codes detectable through multiplexed imaging. Using an engineered Polytope mouse, we traced the fate of hundreds of clones across tissues in vivo, from embryonic development until adulthood. Together, Polytope enables high-resolution imaging-based fate-mapping through endogenous epitope barcoding, comprehensively capturing the spatial organization of complex clonal dynamics in situ. One-Sentence Summary Polytope is an imaging-based barcoding system to trace lineages at clonal and spatial resolution in living organisms using a diverse set of color-coded markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61dde528b997805197f26069101abe05e9869beb" target='_blank'>
              Polytope: High-resolution epitope barcoding for in vivo spatial fate-mapping
              </a>
            </td>
          <td>
            D. Postrach, Colin E. J. Pritchard, Larissa Frank, Tom van Leeuwen, H. Messal, Paul Krimpenfort, J. van Rheenen, Hans-Reimer Rodewald
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="In bladder cancer (BLCA), the cellular expression of lineage markers with predictive biomarker function can have important implications for tumor progression, treatment response, and survival. However, tumor heterogeneity and the coexistence of distinct tumor subpopulations can complicate the utility of using a primary, clinically assigned tumor signature. In this report, we have applied the reference atlas, Tabula Sapiens, and deep learning model, UniCell, to conduct in-depth transcriptional analysis of unique lineage marker-defined cell clusters in a carcinogen (BBN) induced bladder cancer model. UniCell deconvolution has identified tumor populations, including urothelial, adenocarcinoma, squamous cell carcinoma, and mesenchymal tumor populations, each with cell-intrinsic gene expression signatures relevant to human BLCA progression. The identified tumor clusters contain uniquely basal, luminal, stromal, or hybrid cells in cell lineage marker expression. To understand the significance of these populations during progression, we used trajectory and pseudo-time analysis to show that cells uniquely basal are plastic in lineage identity and can evolve to form tumor populations composed of other lineage marker-defined signatures. Finally, pathway and drug enrichment analysis of tumor cell clusters were used to identify therapeutics that may preferentially target the identified tumor cell populations. These data collectively define a molecular template that may uniquely profile molecular plasticity occurring during progression and response to therapy with important implications for human disease. Significance Tumor heterogeneity is a mechanism for treatment resistance. Our study defines unique tumor subpopulations having differential therapeutic sensitivities and potential for lineage plasticity. Our modeling may impact the treatment of BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e51b8c18f48b163ace1388c13e342c3f5124f692" target='_blank'>
              Mapping of Multilineage Tumor Cell Populations in Mouse Bladder Cancer
              </a>
            </td>
          <td>
            Nadine Schrode, Yang Hu, Haocheng Yu, M. Galsky, J. Sfakianos, R. Sebra, Kristin G Beaumont, David J Mulholland
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) has advanced our understanding of tissue regionalization by enabling the visualization of gene expression within whole tissue sections1, but the approach remains dogged by the challenge of achieving single-cell resolution without sacrificing whole genome coverage2,3. Here we present Spotiphy (Spot imager with pseudo single-cell resolution histology), a novel computational toolkit that transforms sequencing-based ST data into single-cell-resolved whole-transcriptome images. In evaluations with Alzheimer’s disease (AD) and normal mouse brains, Spotiphy delivers the most precise cellular compositions. For the first time, Spotiphy reveals novel astrocyte regional specification in mouse brains. It distinguishes sub-populations of DAM (Disease-Associated Microglia) located in different AD mouse brain regions. Spotiphy also identifies multiple spatial domains as well as changes in the patterns of tumor-tumor microenvironment interactions using human breast ST data. Spotiphy enables visualization of cell localization and gene expression in tissue sections, offering key insights into the function of complex biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20d2df6e0d020ef03f0ffa729e29890d99f779b" target='_blank'>
              Spotiphy enables single-cell spatial whole transcriptomics across the entire section
              </a>
            </td>
          <td>
            Jiyuan Yang, Ziqian Zheng, Yun Jiao, Kaiwen Yu, Sheetal Bhatara, Xu Yang, Sivaraman Natarajan, Jiahui Zhang, John Easton, Koon-Kiu Yan, Junmin Peng, Kaibo Liu, Jiyang Yu
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Recent studies have identified a unique subtype of cancer-associated fibroblasts (CAFs) termed antigen-presenting CAFs (apCAFs), which remain the least understood CAF subtype. To gain a comprehensive understanding of the origin and function apCAFs, we construct a fibroblast molecular atlas across 14 types of solid tumors. Our integration study unexpectedly reveals two distinct apCAF lineages present in most cancer types: one associated with mesothelial-like cells and the other with fibrocytes. Using a high-resolution single-cell spatial imaging platform, we characterize the spatial niches of these apCAF lineages. We find that mesothelial-like apCAFs are located near cancer cells, while fibrocyte-like apCAFs are associated with tertiary lymphoid structures. Additionally, we discover that both apCAF lineages can up-regulate the secreted protein SPP1, which facilitates primary tumor formation and peritoneal metastasis. Taken together, this study offers an unprecedented resolution in analyzing apCAF lineages and their spatial niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fa0ff2ff7820fdabd0fa0f4d3421c72dfbfc1" target='_blank'>
              Single-cell resolution spatial analysis of antigen-presenting cancer-associated fibroblast niches
              </a>
            </td>
          <td>
            Xiongfeng Chen, Zhuan Zhou, Zeynep Yazgan, Luyu Xie, Francesca Rossi, Yang Liu, Bo Zhang, Patricio M. Polanco, Herbert J. Zeh, Alex C. Kim, Huocong Huang
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), a distinct subpopulation within tumors, are pivotal in driving treatment resistance and tumor recurrence, posing substantial challenges to conventional therapeutic strategies. Precise quantification and profiling of these cells are essential for improving cancer treatment outcomes. We present ACSCeND, an advanced deep neural network model accompanied by a robust workflow, specifically developed to quantify cellular compositions from bulk RNA-seq data, enabling accurate CSC profiling. By integrating bulk RNA-seq data with insights derived from single-cell RNA-seq datasets, ACSCeND effectively captures the diversity and hierarchical organization of tumor-resident cell states, alongside cell-specific gene expression profiles (GEPs). Compared to current tissue deconvolution models, ACSCeND exhibits superior performance, achieving significantly higher Concordance Correlation Coefficient (CCC) values and lower Root Mean Square Error (RMSE) across various pseudobulk and real-world bulk tissue samples. Application of ACSCeND to TCGA and PRECOG datasets reveals a strong association between CSC abundance and poorer disease-free survival outcomes, underscoring the clinical relevance of CSCs in cancer progression. Furthermore, cell-specific GEPs for distinct CSC states unveil novel molecular signatures and illuminate the origins of CSC-driven tumor heterogeneity. In summary, ACSCeND provides a powerful, scalable platform for high-throughput quantification of cellular compositions and distinct potency states within normal tissues as well as highly heterogeneous tissues, such as tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b3b0a8bb89725a847aae6e2af230a0666ba9d6" target='_blank'>
              Comprehensive Enumeration of Cancer Stem-like Cell Heterogeneity Using Deep Neural Network
              </a>
            </td>
          <td>
            Debojyoti Chowdhury, Shreyansh Priyadarshi, Sayan Biswas, Bhavesh Neekhra, Debayan Gupta, Shubhasis Haldar
          </td>
          <td>2024-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to random barcode sets. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fe68cd8b32acb61197d1f1f0a4cc878efa91c6" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="
 Spatial organization of normal tissues is key for their function, yet the role of distribution of distinct clones of cancer cells in tumor is not usually studied as a potential vulnerability in cancer. This is of particular interest in glioblastoma (GBM), in which extensive intratumor heterogeneity hampers effectiveness of treatment. To better understand tumor evolution and the complexity of cancer ecosystem in GBM, we performed profiling of biopsies collected from MRI-defined regions of the tumors, spanning the surface, tumor core and deep tumor margins, which drive local recurrence after surgery. To test how spatially distinct biopsies differ in their cancer cell composition and to assess the tumor microenvironment heterogeneity, we took advantage of 5-ALA metabolite fluorescence, which accumulates in GBM cells and allows the surgeon to visualize cancer under blue light. We used it to separate cancer cells (predominantly 5-ALAhigh) from tumor microenvironment (5-ALAlow) and subjected the fractions to bulk and single cell transcriptomics, respectively. We found a gradient of cellular states associated with location in the tumor, with the deep margin samples having more 5-ALAlow cancer cells and generally less defined phenotype. We have also developed neurosphere cultures from our collection of spatially distant biopsies. Despite the selection pressure of the in vitro culture conditions, cells from different regions of the tumor adopted different morphologies, suggesting retention of some of their differential features. The differences between cultures were reflected in metabolism, proliferation, and differential sensitivity to a drug panel. Whole exome sequencing and transcriptomic analysis were also performed to identify the selection bottleneck of culture conditions, establish evolutionary trajectory of the distinct regions and neurosphere cultures, and test how strongly our cultures represent both the 5-ALAhigh and 5-ALAlow subpopulations of GBM cells. Our study identified properties associated with different spatial locations within GBM tumors, which will help uncover the cellular and microenvironmental dependencies that could lead to novel therapeutic targets for this highly heterogeneous tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7bfbfe4e2db05f257edb989e202cc92ab018e37" target='_blank'>
              TMOD-22. MULTIREGION BIOPSIES AND CORRESPONDING NEUROSPHERE CULTURES REVEAL SPATIAL DIVERGENCE IN GLIOBLASTOMA EVOLUTION
              </a>
            </td>
          <td>
            Roberto Salatino, Jacob Geisberg, U. Onubogu, Arantxa Romero-Toledo, Banjamin Oakes, O. Szentirmai, Thomas McDonald, Franziska Michor, M. Janiszewska
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In rectal cancer, where part of the patients undergoes chemoradiotherapy, there is a need for improved pretreatment biomarkers applicable to biopsies. Tumor budding (TB) is a biomarker used in colon cancer, and due to its link to epithelial-mesenchymal transition (EMT), is hypothesized to be a potential marker for therapy resistance. Assessment of the utility of tumor buds in rectal biopsies is challenging due to their rarity. As EMT-related processes are also seen in other morphological features beyond tumor buds, we investigated EMT in tumor tissue including morphological features such as tumor cluster size and fibril-like structures. To do so, we leveraged a cohort of colon cancer whole-slide images and another cohort consisting of rectal cancer biopsies, visualized using hyperplex immunofluorescence to identify tumor and EMT-associated proteins. We built a custom image analysis pipeline to detect and segment tumor buds and other morphological features and correlated them with molecular expression intensities. We found strong correlations of EMT up-regulation and morphological transition states, both at the invasive margin and the tumor center. We furthermore observed a link between morpho-molecular transitions and histological growth patterns, which in turn can inform novel biomarkers. Finally, quantification of these morpho-molecular transition states in rectal biopsies showed their impact on survival after neoadjuvant chemoradiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f452e9eb79df45de63d0362c848378c46029149c" target='_blank'>
              Morpho-molecular features of Epithelial Mesenchymal Transition associate with clinical outcome in patients with rectal cancer
              </a>
            </td>
          <td>
            Mauro Gwerder, Cansaran Saygili Demir, Hannah L Williams, A. Lugli, Cristina Graham Martinez, Joanna Kowal, Amjad Khan, Philipp Kirchner, Thibaud Koessler, Martin D. Berger, Martin Weigert, I. Zlobec
          </td>
          <td>2024-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="

 Introduction
 Angioimmunoblastic T-cell lymphoma (AITL) is a T-cell lymphoma characterized by T follicular helper cell (TFH) phenotype. It comes with a strong inflammatory response and highly complex tumor microenvironment, where tumor cells are supported by a scaffold of highly proliferative CD21+ follicular dendritic cells (FDCs) in proximity to high endothelial venules (HEVs). The structured organization of the lymph node, with its distinct B-cell and T-cell zones, is disrupted in lymph node samples from patients with AITL. Based on prior literature in this space, background “benign” inflammatory bystanders and abnormal non-neoplastic vasculature and stromal cells appear to enable tumor cells to escape immune surveillance and unchecked proliferation. We hypothesize that genomic spatial characterization of the tumor microenvironment can lead to improved targeting of these tumor enablers, and for the first time, AITL has been profiled by single cell spatial ATAC-seq, paired with multiplexed immunofluorescence and DBiT-seq for this purpose.
 Methods
 The sample cohort consists of 23 de-identified, archived formalin-fixed paraffin-embedded (FFPE) human lymphoma tissue blocks and 7 OCT-frozen blocks, 6 with paired FFPE. The quality of RNA in each tissue block was assessed via an RNA Integrity Number (RIN) test and blocks with DV200 > 50% (minimal RNA degradation levels) were the ones included in our cohort. For each sample, we performed highly multiplexed immunofluorescence imaging (CODEX) using a custom immune cell panel of 40 markers on the PhenoCycler-Fusion system (Akoya Biosciences). The panel includes cell-type specific markers such as Podoplanin (lymphatic vasculature), CD20, CD3e, as well as all TFH markers. Cell segmentation was performed using a StarDist-based model in QuPath and downstream analysis done using the Seurat package. Serial section was then used for spatial transcriptomics using DBiT-seq to characterize the whole transcriptional landscape of the lymphoma. For the OCT-frozen sections, spatial ATAC-seq was performed on one slide to explore the epigenetic landscape of AITL.
 Results
 Cell type annotation of the CODEX data allowed us to identify the cell types within the tumor microenvironment and quantify their relative proportions. We observed a background of EBV positive B cells marked by the expression of LMP1. Interestingly, we also discovered a subset of non-neoplastic T cells that were EBV-positive. We observed a strong correlation between the proteomic data from CODEX and the spatial transcriptomic data for key cell type populations. Ligand-receptor interaction analysis of the transcriptomic data also showed communication between CXCL13 and its receptor CXCR5, heightened in association with FDCs. We also identified a cluster of IGHM positive cells that highly express KMT2C, one of the histone modification proteins frequently mutated in T-cell lymphomas. Preliminary epigenetic data suggests high chromatin accessibility score in the T-cell rich regions nearby HEVs. BCL6 seems to show high gene activity score in this region, despite its relatively lower transcript and protein expression level. We are currently in the process of fully integrating the RNA profiling and protein expression as well as epigenetic datasets using novel MaxFuse pipeline to provide a comprehensive understanding of the biology underlying AITL.
 Conclusion
 Combining multiplexed immunofluorescence with DBIT-seq and spatial ATAC-seq is providing a comprehensive picture of the mechanisms underlying epigenetic, transcriptional and proteomic landscape of AITL. Data in process of curation reveal novel molecular architecture that may explain underlying cell-cell interactions between TFH tumor cells, reactive B cells and the supporting stromal and vascular apparatus. Specifically, that there may be immune-suppressed EBV+ T cells in the background in addition to spatially segregated activation of tumor cells by surrounding non-neoplastic cells. Broadly, our approach can further the understanding of AITL's complex microenvironment, leading to improved targeting of tumor-enabling signals that facilitate tumor evasion of immune surveillance and drug resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a499a770e0e4d4798a7a5f85e6f030b569dadf71" target='_blank'>
              Spatial Multiomics Profiling of Angioimmunoblastic T-Cell Lymphoma
              </a>
            </td>
          <td>
            Archibald Enninful, Francine M. Foss, Rong Fan, Mina L Xu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third leading cause of cancer mortality in the United States. Familial adenomatous polyposis (FAP) is a hereditary syndrome that raises the risk of developing CRC, with total colectomy as the only effective prevention. Even though FAP is rare (0.5% of all CRC cases), this disease model is well suited for studying the early stages of malignant transformation as patients form many polyps reflective of pre-cancer states. In order to spatially profile and analyze the pre-cancer and tumor microenvironment, we have performed single-cell multiplexed imaging for 52 samples: 12 normal mucosa,16 FAP mucosa,18 FAP polyps, 2 FAP adenocarcinoma, and 4 sporadic colorectal cancer (CRCs) using Co-detection by Indexing (CODEX) imaging platform. The data revealed significant changes in cell type composition occurring in early stage polyps and during the malignant transformation of polyps to CRC. We observe a decrease in CD4+/CD8+ T cell ratio and M1/M2 macrophage ratio along the FAP disease continuum. Advanced dysplastic polyps show a higher population of cancer associated fibroblasts (CAFs), which likely alter the pre-cancer microenvironment. Within polyps and CRCs, we observe strong nuclear expression of beta-catenin and higher number neo-angiogenesis events, unlike FAP mucosa and normal colon counterparts. We identify an increase in cancer stem cells (CSCs) within the glandular crypts of the FAP polyps and also detect Tregs, tumor associated macrophages (TAMs) and vascular endothelial cells supporting CSC survival and proliferation. We detect a potential immunosuppressive microenvironment within the tumor ‘nest’ of FAP adenocarcinoma samples, where tumor cells tend to segregate and remain distant from the invading immune cells. TAMs were found to infiltrate the tumor area, along with angiogenesis and tumor proliferation. CAFs were found to be enriched near the inflammatory region within polyps and CRCs and may have several roles in supporting tumor growth. Neighborhood analyses between adjacent FAP mucosa and FAP polyps show significant differences in spatial location of cells based on functionality. For example, in FAP mucosa, naive CD4+ T cells alone tend to localize near the fibroblast within the stromal compartment. However, in FAP polyp, CD4+T cells colocalize with the macrophages for T cell activation. Our data are expected to serve as a useful resource for understanding the early stages of neogenesis and the pre-cancer microenvironment, which may benefit early detection, therapeutic intervention and future prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f34b5e9edbfa348b570295378a92ebadba11703" target='_blank'>
              Single-cell spatial mapping reveals alteration of cell type composition and tissue microenvironment during early colorectal cancer formation
              </a>
            </td>
          <td>
            Tuhin K. Guha, Edward D. Esplin, Aaron M. Horning, R. Chiu, Kristina Paul, Annika K. Weimer, Winston R. Becker, Rozelle Laquindanum, M. Mills, D. G. Esplin, Jeanne Shen, Emma Monte, Shannon M. White, Thomas V. Karathanos, Daniel L. Cotter, Joanna Bi, U. Ladabaum, Teri A. Longacre, Christina Curtis, William J. Greenleaf, James M Ford, Michael P. Snyder
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3095632bcf16399c0c6ee358ffba9b30df1dc77" target='_blank'>
              Integrative spatial and genomic analysis of tumor heterogeneity with Tumoroscope
              </a>
            </td>
          <td>
            S. Shafighi, Agnieszka Geras, Barbara Jurzysta, Alireza Sahaf Naeini, Igor Filipiuk, Alicja Ra Czkowska, Hosein Toosi, Łukasz Koperski, K. Thrane, Camilla Engblom, Jeff E. Mold, Xinsong Chen, Johan Hartman, D. Nowis, A. Carbone, Jens Lagergren, E. Szczurek
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Recent advances in spatial transcriptomics have revolutionized our understanding of tissue spatial architecture and biological processes. However, many of these technologies face significant challenges in achieving either single-cell resolution or comprehensive whole-transcriptome profiling, hindering their capacity to fully elucidate intercellular interactions and the tissue microenvironment. To address these limitations, we present STEP, a hybrid framework that synergistically integrates probabilistic models with deep learning techniques for spatial transcriptome analysis. Through innovations in model and algorithm design, STEP not only enhances sequencing-based spatial transcriptome data to single-cell resolution but accurately infers transcriptome-wide expression levels for image-based spatial transcriptomic. By leveraging the nuclear features extracted from histological images, STEP achieves precise predictions of cell type and gene expression and effectively diffuses the discriminative ability to serial sections for modeling solid tissue landscapes. The capability is particularly advantageous for analyzing cells with distinctive characteristics, such as cancer cells, enabling cross-sample inference. In addition, STEP simulates intercellular communication through a spatially resolved cell-cell interaction network, uncovering intrinsic biological processes. Overall, STEP equips researchers with a powerful tool for understanding biological functions and unveiling spatial gene expression patterns, paving the way for advancements in spatial transcriptomics research. Code is available at https://github.com/childishHU/STEP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7983477cd900a40c7ac98073f36d0f078ec1422" target='_blank'>
              STEP: Deciphering Spatial Atlas at Single-Cell Level with Whole-Transcriptome Coverage
              </a>
            </td>
          <td>
            Zheqi Hu, Zirui Zhu, Linghan Cai, Yangen Zhan, Xiangming Yan, Jingyun Chen, Bingqian Sun, Sijing Du, Songhan Jiang, Hongpeng Wang, Yongbing Zhang
          </td>
          <td>2024-11-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER.
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Despite being highly vascularized and immune-rich, the prognostic impact of immune infiltration and tumor microenvironment heterogeneity in localized clear cell renal cell carcinoma (ccRCC) remains unclear. Using 33-marker immunofluorescence imaging, we analyzed spatial distributions in 1,728 tissue cores from 435 localized ccRCC cases. Higher CD45⁺ infiltration correlated with worse survival, establishing CD45⁺ density as a prognostic marker. In CD45high tumors, ccRCC cells losing epithelial markers showed increased expression of mesenchymal markers such as fibroblast activation protein (FAP), secreted protein acidic and rich in cysteine (SPARC), vimentin (VIM), and PD-L1, indicating epithelial-to-mesenchymal transition (EMT). The FAP⁺EMT⁺ state further stratified CD45high patients toward poorer outcomes and was associated with immunosuppressive elements, including M2 macrophages, exhausted and regulatory T cells, and FAP⁺ cancer-associated fibroblasts. Our study identifies high CD45⁺ infiltration and a FAP⁺EMT⁺ cancer cell state as indicators of poor survival in localized ccRCC, highlighting their potential to refine patient stratification and guide immuno-oncology treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/073390e358abb5e61bb739384c3626f014b9da33" target='_blank'>
              Fibroblast activation protein defines aggressive EMT-associated cancer cell state and prognosis in localized ccRCC
              </a>
            </td>
          <td>
            T. Pellinen, L. Luomala, K. Mattila, A. Hemmes, K. Välimäki, Oscar Brück, L. Paavolainen, Elisa Kankkunen, H. Nísen, P. Järvinen, Leticia Castillon, Sakari Vanharanta, P. Vainio, O. Kallioniemi, Panu Jaakkola, T. Mirtti
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0d10a131e1ec85b605aa6456ac30c2943dbeb4" target='_blank'>
              Spatial immunogenomic patterns associated with lymph node metastasis in lung adenocarcinoma
              </a>
            </td>
          <td>
            Fanjie Meng, Hao Li, Ruoyi Jin, Airong Yang, Hao Luo, Xiao Li, Peiyu Wang, Yaxing Zhao, O. Chervova, Kaicheng Tang, Sida Cheng, Bin Hu, Yun Li, Jianpeng Sheng, Fan Yang, David Carbone, Kezhong Chen, Jun Wang
          </td>
          <td>2024-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Deciphering the cell state dynamics is crucial for understanding biological processes. Single-cell lineage tracing technologies provide an effective way to track the single-cell lineages by heritable DNA barcodes, but the high missing rates of lineage barcodes and the intra-clonal heterogeneity bring great challenges for dissecting the mechanisms of cell fate decision. Here, we systematically evaluate the feature of single-cell lineage tracing data, and then develop an algorithm scTrace+ to enhance the cell dynamic traces by incorporating multi-faceted transcriptomic similarities into lineage relationships via a Kernelized Probabilistic Matrix Factorization model. We assess its feasibility and performance by conducting ablation and benchmarking experiments on multiple real datasets, and show scTrace+ can accurately predict the fates of cells. Further, scTrace+ effectively identifies some important driver genes implicated in cellular fate decision of diverse biological processes, such as the cell differentiation or the tumor drug responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b172a55df4793c755c61f2ca21d72c3fa5022eb1" target='_blank'>
              scTrace+: enhance the cell fate inference by integrating the lineage-tracing and multi-faceted transcriptomic similarity information
              </a>
            </td>
          <td>
            Wenbo Guo, Zeyu Chen, Xinqi Li, Jingmin Huang, Qifan Hu, Jin Gu
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Carcinogenesis is characterized not only by the uncontrolled growth of malignant cells but also by the disruption of the normal balance of cellular processes and intercellular interactions in the microenvironment that overcome the constraints of tissue homeostasis and support malignant growth. We profiled benign pulmonary dysplasia, carcinoma in situ, and invasive lung carcinomas at single-cell resolution to identify composite changes in cellular processes, signaling, and interactions among tumor-immune-stromal cells in the microenvironment in progressively advanced disease stages. We developed OncoTerrain, a hyperparameter-tuned model that captured synergistic multimodal signatures of an increasingly perturbed microenvironment in malignant disease stages and identified composite microenvironmental changes that supported cancer hallmarks. Key cancer-related changes in transcriptional states, cellular processes, and intercellular interactions involving immune, fibroblast, and stromal cell types preceded tumor initiation and were often synergistic. The microenvironment of increasingly malignant tissues was characterized by immune avoidance, ECM remodeling, and altered cell mobility. There were changes in cell states in fibroblasts, macrophages, and their inter-cellular interactions with other cell types, whereas T-cell activation occurred late. The in-situ carcinomas showed variations in the composite microenvironmental states that corroborated their pathology, which was not apparent at the genome level. A subset of those harbored populations of tumor and non-tumor cells with aggressive characteristics in some but not all aspects of hallmarks of carcinogenesis in the lung. We suspect that the variation in the coordination of microenvironmental cues may influence why some but not all in-situ tumors progress to the advanced stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/437dc7b34021d21e799bbd0ca24c7b8aa0ec942c" target='_blank'>
              Disruptive changes in tissue microenvironment prime oncogenic processes at different stages of carcinogenesis in lung
              </a>
            </td>
          <td>
            Vignesh Venkat, Xiaoju Hu, Antara Biswas, Ankit Saxena, Jyoti Malhotra, Gregory Reidlinger, Subhajyoti De
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Although immune checkpoint inhibitors (ICIs) have revolutionized melanoma treatment, many individuals either show no response or eventually develop acquired resistance. This variability could be linked to differences in the tumor immune microenvironment, pre-existing drug-resistant cells or treatment induced changes in melanoma cell states. Using single cell RNA sequencing (scRNA-seq) coupled with heritable barcodes, this study aims to lineage trace changes in melanoma transcriptional cell states and map changes in the tumor immune microenvironment to uncover if preexisting and/or therapy-induced melanoma transcriptional cell states lead to resistance to ICIs. In this study, we used YUMMER1.7 mouse melanoma cells which were transduced with barcodes under conditions that allow delivery of a single unique barcode to each cell, this served as a cell lineage tag. Each barcode is heritable and stably transcribed into RNA molecules so that individual barcoded cells can be matched with a gene expression profile from scRNA-seq outputs. The barcoded cells were subcutaneously injected into C57BL/6 mice, and once tumors were established, mice were treated with 3 cycles of anti-PD-1 and anti-CTLA-4 therapy. Lineage tracing and scRNA-seq analyses were performed on tumors harvested prior to treatment, early on treatment (Day 6), during minimal residual disease (Day 13) and upon relapse. Early on treatment (Day 6)- two tumour groups were harvested, those that responded well and others that only partially responded. Our results showed that during the early on treatment phase (Day 6) tumors that responded well and those that only partially responded displayed distinct barcodes. Additionally, prevalent barcoded cells identified at the minimal residual disease phase were also dominant upon relapse. Analysis of the transcriptional states is underway to determine if ICI therapy induces transcriptional heterogeneity in sensitive and tolerant cells and if a particular or several transcriptional states lead to relapse. This study will potentially identify predictive response biomarkers and vulnerabilities of ICI naïve and resistant cells that could be targeted to improve outcomes for melanoma patients. We will also identify neoadjuvant approaches, to remove the inherent heterogeneity within the tumor cell population, facilitating a more uniform sensitivity to ICIs.
 Citation Format: Reem Saleh, Riyaben Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen Sheppard. Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2046528ce162da3c656f0facbde8bbcbfa6faf03" target='_blank'>
              Abstract C044: Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance
              </a>
            </td>
          <td>
            Reem Saleh, Riyaben P. Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen E. Sheppard
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="T cell therapies, such as chimeric antigen receptor (CAR) T cells and T cell receptor (TCR) T cells, are a growing class of anti-cancer treatments. However, expansion to novel indications and beyond last-line treatment requires engineering cells’ dynamic population behaviors. Here we develop the tools for cellular behavior analysis of T cells from live-cell imaging, a common and inexpensive experimental setup used to evaluate engineered T cells. We first develop a state-of-the-art segmentation and tracking pipeline, Caliban, based on human-in-the-loop deep learning. We then build the Occident pipeline to collect a catalog of phenotypes that characterize cell populations, morphology, movement, and interactions in co-cultures of modified T cells and antigen-presenting tumor cells. We use Caliban and Occident to interrogate how interactions between T cells and cancer cells differ when beneficial knock-outs of RASA2 and CUL5 are introduced into TCR T cells. We apply spatiotemporal models to quantify T cell recruitment and proliferation after interactions with cancer cells. We discover that, compared to a safe harbor knockout control, RASA2 knockout T cells have longer interaction times with cancer cells leading to greater T cell activation and killing efficacy, while CUL5 knockout T cells have increased proliferation rates leading to greater numbers of T cells for hunting. Together, segmentation and tracking from Caliban and phenotype quantification from Occident enable cellular behavior analysis to better engineer T cell therapies for improved cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e025cf1fccd7422ab1c920631722b6d643225bf" target='_blank'>
              Cellular behavior analysis from live-cell imaging of TCR T cell–cancer cell interactions
              </a>
            </td>
          <td>
            Archit Verma, Changhua Yu, Stefanie Bachl, Ivan Lopez, Morgan Schwartz, Erick Moen, Nupura Kale, Carter Ching, Geneva Miller, Tom Dougherty, Edward Pao, William Graf, Carl Ward, Siddhartha Jena, Alex Marson, J. Carnevale, David Van Valen, Barbara E. Engelhardt
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Melanoma, a highly metastatic skin cancer, exhibits variations in prognosis and response to therapy based on the site of metastasis. Despite the success of immunotherapy and targeted therapies in melanoma, over half of metastatic melanoma patients will experience disease progression due to therapy resistance. The heterogeneity and plasticity of melanoma cells contribute to metastatic dissemination and therapy resistance. Our aim is to determine whether distinct clones and/or their transcriptional cell states can predict the formation of tumors in various organs and assess how these clones change over time. To identify melanoma clones across different metastatic sites, NOD scid gamma (NSG) mice and C57BL/6 mice were injected subcutaneously, intravenously or intracranially with the same pool of cells of barcoded luciferase expressing YUMMER1.7 murine melanoma cells. Transduction conditions ensured that one DNA barcode integrated into cell genomes at one barcode per cell, serving as a lineage tag. Bioluminescence imaging was performed once a week to monitor tumor growth of mice injected intravenously and intracranially. Subcutaneous tumors were measured by calliper. Mice were euthanized at different time points; day 8, 15, 22, 29 post-implantation and at ethical endpoints. Tumors were harvested and DNA sequencing was performed to identify barcodes expressed by the tumor cells. All mice developed tumors, with 100% penetrance in NSG mice. However, in C57BL/6 mice, 10% of mice intravenously injected and 27% of subcutaneously implanted mice showed complete lesion regression, suggesting that the immune system may be responsible for tumor regression. Analysis of barcodes allowed us to assess the heterogeneity of melanoma tumors at different metastatic sites and their evolution over time. Lineage tracing and clonal heterogeneity will be evaluated using the state of art technology, SPLINTR (Single-cell Profiling and LINeage Tracing), enabling us to match a cells evolution with changes in transcriptional states. Barcode analysis performed before implanting the cells and, at different timepoints in subcutaneous and lung tumors showed that dominant subclones at the baseline were also dominant in subcutaneous and lung tumors in immunocompromised mice. In contrast, dominant subclones in immunocompetent mice was those present in lower proportion at baseline. Additionally, greater variability of subclones was observed especially in lung and brain tumors, across immunocompetent mice, likely as a mechanism of resistance, enabling these tumors to overcome immunoediting. Understanding the variability of clonality between different metastatic sites and over time will improve our comprehension of the role of different subclones in organ- specific metastasis and their transcriptional cell states.
 Citation Format: Veronica L. Aedo Lopez, Reem Saleh, Benjamin Blyth, Xin Du, Dane Vassiliadis, Katie Fennell, Davide Moi, Roberta Mazzieri, Riccardo Dolcetti, Karen E. Sheppard, Grant A. McArthur. Intra- and inter-tumoral heterogeneity of melanoma across different metastatic sites [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd114bc5004ce77e46072db707d1b3ec32d44c22" target='_blank'>
              Abstract B001: Intra- and inter-tumoral heterogeneity of melanoma across different metastatic sites
              </a>
            </td>
          <td>
            Veronica L. Aedo Lopez, Reem Saleh, Benjamin Blyth, Xin Du, Dane Vassiliadis, Katie Fennell, D. Moi, Roberta Mazzieri, Riccardo Dolcetti, Karen E. Sheppard, Grant A. McArthur
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract High-dimensional, spatial single-cell technologies, such as CyTOF imaging mass cytometry (IMC), provide detailed information regarding locations of a large variety of cancer and immune cells in microscopic scales in tumor microarray slides obtained from patients prior to immune checkpoint inhibitor (ICI) therapy. An important question is how the initial spatial organization of these cells in the tumor microenvironment (TME) changes with time and regulates tumor growth and eventually outcomes as patients undergo ICI therapy. Utilizing IMC data of melanomas of patients who later underwent ICI therapy, we develop a spatially resolved interacting cell system model that is calibrated against patient response data to address the above question. We find that the tumor fate in these patients is determined by the spatial organization of activated CD8+ T cells, macrophages, and melanoma cells and the interplay between these cells that regulate exhaustion of CD8+ T cells. We find that fencing of tumor cell boundaries by exhausted CD8+ T cells is dynamically generated from the initial conditions that can play a protumor role. Furthermore, we find that specific spatial features such as co-clustering of activated CD8+ T cells and macrophages in the pretreatment samples determine the fate of the tumor progression, despite stochastic fluctuations and changes over the treatment course. Our framework enables the determination of mechanisms of interplay between a key subset of tumor and immune cells in the TME that regulate clinical response to ICIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9225e8a7e492a347e0673b0bb1568c995b67d250" target='_blank'>
              Spatial organization and stochastic fluctuations of immune cells impact clinical responsiveness to immunotherapy in melanoma patients
              </a>
            </td>
          <td>
            Giuseppe Giuliani, William Stewart, Zihai Li, Ciriyam Jayaprakash, Jayajit Das
          </td>
          <td>2024-11-26</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Quiescent cancer cells (QCCs) remain unconquered challenges in cancer treatment. Even after being declared cured, patients remain at risk of cancer recurrence or face overtreatment aimed at eliminating minimal residual QCCs. The lack of effective diagnostic and therapeutic options for QCCs stems from our incomplete understanding of these cells. This knowledge gap is largely due to the scarcity or limitations of preclinical models, particularly those that enable the tracing of QCCs in vivo. By reviewing current QCC tracing tools and their limitations, this review aims to explore more advanced systems for tracing QCCs in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f976d0eb234c239f80d5b7267b1ecf33f0afd1fb" target='_blank'>
              Tracing Quiescent Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Moon Jong Kim
          </td>
          <td>2024-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23b3d699dd705e1b450a74d7c903ab854deff777" target='_blank'>
              Increased spatial coupling of integrin and collagen IV in the immunoresistant clear-cell renal-cell carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, Taylor Peak, Oscar E. Ospina, N. Chakiryan, Anders E Berglund, Paul A Stewart, Jonathan V Nguyen, C. M. Segura, Natasha L. Francis, P. R. Echevarria, J. Chahoud, Roger Li, Kenneth Y. Tsai, Jodi A. Balasi, Yamila Caraballo Peres, J. Dhillon, Lindsey A. Martinez, Warren E. Gloria, Nathan Schurman, Seang-Jae Kim, M. Gregory, James J Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-12-05</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Localised prostate cancers (PCa) are heterogeneous and multifocal, with diverse outcomes. Current prognostic methods are epithelium-centric, overlooking the complex cellular landscape within the tumour microenvironment (TME), which remains incompletely characterised. We performed a comprehensive analysis of cancerous and adjacent-benign cores from 24 patients with hormone therapy-naïve localised PCa using single-cell RNA-sequencing. By integrating copy number variation and transcriptional signatures, we classified epithelial cells across a malignant spectrum, revealing widespread molecular perturbation. We found an expansion of Club cell phenotypes, suggestive of Luminal dedifferentiation. We also performed a detailed annotation of stromal phenotypes, focusing on fibroblasts, and identified a novel peri-neural fibroblast population. Spatial transcriptomics elucidated the precise anatomical distribution of CAFs within the PCa TME. This study provides a valuable foundation for advancing our understanding of PCa pathobiology and developing a comprehensive cellular model of the disease. Statement of Significance Our study leverages single-cell RNA-sequencing and spatial transcriptomics to provide a comprehensive cellular annotation of hormone therapy-naïve localised PCa. We reveal widespread molecular perturbations in epithelial cells and map distinct fibroblast populations to specific anatomical niches. Notably, we identify a novel peri-neural phenotype associated with nerves, which merits further functional characterisation and exploration as a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b9e207aea660a1c4b72622107ccb70821b90a9" target='_blank'>
              Profiling of epithelial functional states and fibroblast phenotypes in hormone therapy-naïve localised prostate cancer
              </a>
            </td>
          <td>
            Eva Apostolov, D. Roden, Holly Holliday, Aurélie S. Cazet, K. Harvey, Hanyun Zhang, Sunny Z. Wu, Sophie van der Leij, Luke A Selth, Nenad Bartonicek, G. Al-Eryani, Mengxiao He, J. Lundeberg, John Reeves, James G Kench, Alison J Potter, Phillip Stricker, Anthony M Joshua, Lisa G. Horvath, Alexander Swarbrick
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains an untreatable disease. Understanding GBM’s infiltrative biology at the resection margin is limited, despite causing disease recurrence and progression. To address this, we generated a high-throughput single-nucleus (sn)RNA-seq and snATAC-seq multi-omic dataset from six tumors with distinct genomic drivers and combined it with spatial transcriptomics to characterize the unique molecular phenotype of GBM near the margin. By contrasting GBM-specific biology in matching “Core” vs. “Margin” dissections, we define unique, shared “GBM infiltration” and chromatin accessibility signatures near the margin. We prioritize EGFR as a top differentially expressed and accessible “Margin” marker across GBM subtypes, show its dynamic expression along a core-to-margin infiltration trajectory, and validate its role in migration through CRISPR/Cas9 deletion in two patient-derived models. ChIP-seq studies furthermore corroborate preferential TEAD1 binding at EGFR’s accessible regulatory elements. This validated multi-omic dataset enables further studies into tumor and microenvironment biology in the context of residual GBM disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d032da4813c4397d215d1b17d17829c10107f9f" target='_blank'>
              Spatial multi-omics defines a shared glioblastoma infiltrative signature at the resection margin
              </a>
            </td>
          <td>
            Balagopal Pai, S. Ramos, W. Cheng, T. Joshi, Gabrielle Price, Jessica Tomé-García, G. Nudelman, S. Shroff, K. Beaumont, Yong Raymund, R. Sebra, E. Zaslavsky, N. Tsankova
          </td>
          <td>2024-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7e21f7327e57fd7a5664b9a5fd090102737fcd" target='_blank'>
              scStateDynamics: deciphering the drug-responsive tumor cell state dynamics by modeling single-cell level expression changes
              </a>
            </td>
          <td>
            Wenbo Guo, Xinqi Li, Dongfang Wang, Nan Yan, Qifan Hu, Fan Yang, Xuegong Zhang, Jianhua Yao, Jin Gu
          </td>
          <td>2024-11-21</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a fatal adult solid tumour with median overall survival of 18-20 months post-diagnosis; contributing factors to therapeutic inefficacy include acquisition of genomic alterations post-therapy, immune evasion, deregulated hypervascularization, and tumor microenvironmental factors such as hypoxia. Combined with extensive inter- and intra-tumoral heterogeneity at bulk and single-cell level, hypoxia contributes to a gradient of molecular alterations that are specific to different cell populations that make up tumour bulk and reside in specific niches. Hitherto, high-dimensional histopathologic analyses of hypoxic regions within GBM tissue have not been performed. We took a combined spatial and single-cell proteomic profiling approach to investigate the histopathologic features of hypoxia by leveraging a unique clinical study wherein the exogenous hypoxia marker, pimonidazole (PIMO), was administered to GBM-patients preoperatively. Tissue specimens were subjected to imaging mass cytometry, high-resolution imaging, and serial immunohistochemistry using a panel of markers associated with cellular hallmarks of hypoxia, metabolism, proliferation, stemness, angiogenesis, and immune cell types. Our findings showed that PIMO staining is associated with histopathologic features of hypoxia and correlates with specific metabolic, immune, and stemness markers in GBM; specific lymphocyte populations were depleted from hypoxic regions alongside alterations in macrophagic and microglial landscape in a niche-specific manner; hypoxia reduced the proportion of proliferating glioma initiating cells and altered the proliferative, transcriptional, and translational capacity of different cell populations; microvessel density was reduced in hypoxic microenvironment. Cytometry by time-of-flight further validated the altered proportions of specific immune cell types enriched in hypoxic populations of GBM microenvironment. Our study is the first to report use of PIMO to interrogate spatial, single-cell, and phenotypic architecture associated with tissue hypoxia and altered expression of biomarkers associated with hypoxia/glycolysis, immune infiltration, proliferation, and stemness. Identification of targetable biomarkers and mediators of hypoxia-driven habitats in GBM may provide direction for future immunotherapeutic research.
 Citation Format: Shreya Gandhi, Sheila Mansouri, Olivia Singh, Mark Zaidi, Ronald Wu, Andrew Gao, Bradley Wouters, Gelareh Zadeh. Spatial and single-cell proteomic landscaping of the hypoxic microenvironment in glioblastoma [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B069.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5960e28aa4e015a793e4ede112dd73ea174cad4c" target='_blank'>
              Abstract B069: Spatial and single-cell proteomic landscaping of the hypoxic microenvironment in glioblastoma
              </a>
            </td>
          <td>
            Shreya Gandhi, S. Mansouri, O. Singh, M. Zaidi, Ronald Wu, A. Gao, Bradley Wouters, G. Zadeh
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="

 The variable clinical course and response to treatment of Follicular Lymphoma (FL) is strongly influenced by its tumor microenvironment (TME). Indeed, cellular interactions dictate pro- and anti-tumor processes within the TME, although these mechanisms remain incompletely understood. Multiple bulk and single-cell studies have shed light on the TME with prognostic implications. For example, a macrophage gene signature as well as a ‘T cell depleted’ milieu have been associated with inferior clinical outcomes.
 Imaging Mass Cytometry (IMC) permits the multiplexed assessment of over 40 markers concomitantly, superseding the limits of conventional immunohistochemistry, thus enabling detailed single cell spatial profiling of the TME. For comprehensive analysis of tumor immune architecture within FL, we first optimized a 31-plex panel to capture tumor, immune and stromal cell subsets, as well as functional information such as memory and effector status, proliferation, activation and exhaustion. Fixed formalin paraffin embedded (FFPE) biopsies from untreated FL patients enrolled on the GALLIUM trial (NCT01332968), assessing frontline chemo-immunotherapy regimens, were subjected to IMC and 381 high-dimensional images were acquired.
 After quality control, we analysed >3.5 million cells across 340 high-dimensional images. Cell phenotyping identified ~78 cell populations including lymphoma cells, CD4+ and CD8+ T cell subsets (including T follicular helper (Tfh), memory, naïve, exhausted) and a range of non-lymphoid cells e.g. myeloid and stromal cells. As expected, CD20+ B cells were expanded in FL images compared with control lymphoid samples (~56% vs. ~15% respectively p<0.05). Consistent with previous literature, we found an expansion of CD4+ T cells in FL (~20%) compared with control tissue (~12% p<0.05). We have additionally identified sets of samples enriched for CD57+ T cells. The presence of these cells has previously been associated with inferior survival in FL. Furthermore, we have observed that parent lineage abundance (e.g. CD20+ cells, CD3+ cells) identified via our phenotyping of FFPE biopsies were consistent with CIBERSORTx deconvolution analysis from patient-matched RNA-seq samples, improving confidence in our phenotypic labels.
 We next analysed cellular neighborhoods, defining ten distinct FL environments. Analysis was performed i) at the level of neighborhoodcellular composition, and ii) at the level of relative proportions of identified neighborhoods within images. In terms of neighborhood cellular composition, among visually similar tissue components, we found striking variability. For example, we assessed the cellular composition of the neighborhoods making up follicles (as defined by the presence of CD21+ meshworks). In some cases, these neighborhoods were very homogenous with the cellular infiltrate consisting almost exclusively of lymphoma CD20+ cells. In other cases, we found that such neighborhoods were highly heterogenous encompassing diverse immune cell composition.
 By analysing the relative abundance of different neighborhoods, we found sets of variable architectures across images. For example, the ‘lymphoma exclusive’ neighborhood dominated many images. We found that images with high abundance of this neighborhood typically also contained a scattered ‘lymphoma+, CD4/8T+, CD16+‘ neighbourhood additionally exhibiting immunosuppressive Tregs cells, as well as CD57+ T cells. In contrast, a separate notable pattern of neighbourhood abundances was observed in a set of images containing organised follicular structures with detectable CD21+ meshwork, admixed with CD57+/- Tfh-like cells and macrophages. Here, this follicular structure was outlined by a thin layer of CD4+ T cells, CD57+PD1+ T cells and lymphoma cells. These images are completed by an interfollicular region containing neighborhoods comprising mixed immune subsets.
 Ongoing work seeks to examine associations between such variable spatial arrangements with this clinically annotated dataset. This will provide a detailed characterization of how the spatial context and crosstalk of individual cells amid neighbourhoods within the FL TME specifically influences disease behavior, response to treatment and clinical outcome, which will likely extend and compliment those findings already described at bulk and single cell level.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/945f841925664791c3d4fe453e1cae6353b65fa0" target='_blank'>
              Highly Heterogeneous Cellular Composition and Neighborhood Architecture within the Follicular Lymphoma Tumor Microenvironment
              </a>
            </td>
          <td>
            Paul Buckley, Julie Chan, Cynthia Bishop, Timothy Woo, K. Todd, Elizabeth H. Phillips, A. Bottos, Christopher Bolen, V. Passerini, Oliver Weigert, Piers E. M. Patten
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Effective cancer immunotherapies restore anti-tumor immunity by rewiring cell-cell communication. Treatment-induced changes in communication can be inferred from single-cell RNA-sequencing (scRNA-seq) data, but current methods do not effectively manage heterogeneity within cell types. Here we developed a computational approach to efficiently analyze scRNA-seq-derived, single-cell-resolved cell-cell interactomes, which we applied to determine how agonistic CD40 (CD40ag) alters immune cell crosstalk alone, across tumor models, and in combination with immune checkpoint blockade (ICB). Our analyses suggested that CD40ag improves responses to ICB by targeting both immuno-stimulatory and immunosuppressive macrophage subsets communicating with T cells, and we experimentally validated a spatial basis for these subsets with immunofluorescence and spatial transcriptomics. Moreover, treatment with CD40ag and ICB established coordinated myeloid-T cell interaction hubs that are critical for reestablishing antitumor immunity. Our work advances the biological significance of hypotheses generated from scRNA-seq-derived cell-cell interactomes and supports the clinical translation of myeloid-targeted therapies for ICB-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63442f7112573b18ed8b87addf34034f405e0832" target='_blank'>
              Mapping intratumoral myeloid-T cell interactomes at single-cell resolution reveals targets for overcoming checkpoint inhibitor resistance
              </a>
            </td>
          <td>
            Kate Bridges, Gabriela A. Pizzurro, Alev Baysoy, Janani P. Baskaran, Ziyan Xu, Varsha Mathew, V. Tripple, Michael LaPorte, Koonam Park, W. Damsky, Harriet M. Kluger, Rong Fan, S. Kaech, M. Bosenberg, K. Miller-Jensen
          </td>
          <td>2024-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4200779de8ffeea14186de4448271c5a8d0e9f0" target='_blank'>
              Developmental-status-aware transcriptional decomposition establishes a cell state panorama of human cancers
              </a>
            </td>
          <td>
            Yikai Luo, Han Liang
          </td>
          <td>2024-10-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Deciphering shared features between patients through unsupervised analyses of tumor single-cell transcriptomes is hindered by the predominant clustering of malignant cells based on the patient’s tumor of origin. In contrast, cancer-associated non-malignant cells within tumors cluster according to their somatic cell type (e.g. macrophage, fibroblast, astrocyte), regardless of the individual patient. We investigated the origin of this contrasting clustering behavior using computational analyses and novel data sampling techniques across 160 tumors representing 9 distinct cancer types. This led to the unexpected disclosing of tumor-specific malignant cell types. We demonstrate that tumor-driven malignant cell clustering is independent from technical or computational biases, nor is it reducible to defined gene sets with tumor-specific expression across all examined cancers. On the opposite, we unveil redundant information dispersed across the transcriptome that encodes a unique identity shared by the malignant cells within each tumor. Additionally, we found a similar genomic encoding for the identity of normal cell types. Finally, we demonstrate that malignant cell identities are maintained across space and over time, as are the identities of normal somatic cells types. These findings suggest the establishment of a distinct type of malignant cells within each tumor, robustly and diffusively encoded across the entire transcribed genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb2c0940cbf36006f73fd3a210e9572d4abd3013" target='_blank'>
              A unique malignant cell type per patient tumor is encoded in each cancer cell transcriptome
              </a>
            </td>
          <td>
            M. Saurty-Seerunghen, Elias A. El-Habr, Yossi Eliaz, Léa Bellenger, Christophe Antoniewski, Hervé Chneiweiss, M. Junier
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Dysregulation of intracellular signaling networks underpins cancer. However, a systems-level elucidation of how signaling networks within distinct cell subpopulations drive cancer progression in vivo has been unattainable due to technical limitations. We developed INSIGHT (INvestigating SIGnaling network of specific cell subpopulation in Heterogeneous Tissue), a new platform technology combining fluorescence-activated cell sorting with ultra-sensitive mass spectrometry to enable phosphoproteomic characterization of rare and discrete cell subpopulations from fixed tissues. We demonstrated the broad utility of INSIGHT by analyzing the oligodendroglial cell-specific signaling network in the mouse brain. We then applied INSIGHT to investigate the rare, disseminated tumor cell subpopulation in glioblastoma patient-derived xenograft models. INSIGHT uncovered a global rewiring of signaling networks with tumor cell dissemination, marked by a transition from proliferation-associated signaling in the primary tumor cells to signaling associated with postsynapse, neuronal migration, and ion homeostasis in disseminated tumor cells. We reveal interconnections between signaling circuitries within the networks, with numerous proteins, including GluA2, exhibiting altered phosphorylation without protein expression changes, emphasizing the role of post-translational modifications in glioblastoma dissemination. We validated key phosphorylation changes and inferred differentially active kinases with tumor spread to offer new systems-level insights into glioblastoma dissemination mechanisms in vivo. INSIGHT is generally applicable to a wide range of biological systems without genetic engineering and provides quantitative phosphorylation and protein expression data for selected cell subpopulations from heterogeneous tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/decaf507fd537acca35393bd50f3bedca402f1f6" target='_blank'>
              Uncovering the signaling networks of disseminated glioblastoma cells in vivo with INSIGHT
              </a>
            </td>
          <td>
            Ryuhjin Ahn, Alicia D. D’Souza, Lawrence Long, Yufei Cui, D. Burgenske, K. Bakken, Lauren L Ott, Brett L. Carlson, Grace Zhou, Tomer M. Yaron-Barir, Ishwar N. Kohale, Charles A. Whittaker, Cameron T. Flower, Jeffrey Wyckoff, A. Tuma, Jared L. Johnson, Jann Sarkaria, Forest M. White
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and aggressive primary brain cancer in adults, characterized by robust infiltrative growth and invasion within the neural microenvironment. While various stromal cells, including astrocytes, microglia, and vascular cells in the microenvironment, communicate with GBM cells to promote growth and invasion, relatively less is known about another critical component of the tumor stroma: the extracellular matrix (ECM). The matrisome comprises hundreds of ECM genes and affiliated adhesion and signaling components; however, little is understood about (i) the repertoire of matrisome genes and proteins expressed in GBM, (ii) the stromal and cancer cells of origin that express different matrisome factors, and (iii) how these components are interconnected to drive tumor malignancy. Here, we have used in situ single-cell transcriptomics to analyze the spatial expression of nearly 350 genes that constitute the human brain matrisome. Our efforts have identified several differentially expressed matrisome genes involved in ECM adhesion and signaling between cancer and stromal cells. Additionally, matrisome genes associated with low-grade glioma pathogenesis and progression to high-grade GBM have been identified and spatially mapped. Collectively, these experiments reveal the single-cell origins of differentially expressed ECM biomarkers as well as their spatial expression patterns in brain tumors. These findings may lead to new therapeutic efforts to inhibit matrisome components to benefit patients with malignant brain cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db27897e2873e1f72dc0c04bc6970233914ac8ad" target='_blank'>
              In Situ Single-Cell Spatial Profiling of Matrisome Gene Expression in Glioblastoma
              </a>
            </td>
          <td>
            Arpan De, Santiago A. Forero, Sumod Sebastian, John E. Morales, Marisol De La Fuente-Granada, Jason T Huse, L. Ballester, Jeffrey S. Weinberg, Frederick F. Lang, Kadir C. Akdemir, Joseph H. McCarty
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="
 We integrated single-cell RNA-sequencing, Visium spatial transcriptomic, and Xenium in situ transcriptomic data to identify differences in cell composition between the tumor core and infiltrating edge in IDHWT glioblastoma (GBM). Using CellTrek, an algorithm that maps single cells to their most-likely locations in spatial transcriptomic data, we identified extensive spatial organization in GBM. Cell mapping revealed distinct regions enriched for tumor, immune, endothelial, neuronal, and glial cells. These were associated with the location of the tissue sample – edge regions were enriched for neurons and glia (as expected), while core regions were enriched for tumor cells and infiltrating immune cells, particularly macrophages. Within the core samples, the tumor and immune cells tended to occupy distinct spatial niches. Spatial clustering of mapped tumor cells across all samples revealed five spatially segregated tumor cell states, each associated with specific biological processes, including (1) histone methylation, (2) proliferation, (3) wound-healing, EMT, and antigen presentation, (4) hypoxia response, and (5) stem cell differentiation. These tumor cell states exhibited preferential colocalization with specific non-malignant cell types. For instance, antigen-presenting/wound-healing tumor cells tended to colocalize with diverse immune cells, while stem-like tumor cells tended to reside in neuron-rich regions at the tumor edge. Next, to identify ligand-receptor interactions governing this spatial organization, we analyzed known ligand-receptor pairs within all three data types. This revealed known and novel mediators of tumor cell interactions with vasculature, lymphoid cells, myeloid cells, neurons, and glia. While some interactions were shared across all tumor cell states (e.g. all tumor cell states interacted with myeloid cells via APP/CD74 interactions), others were specific to particular tumor cell states (e.g. wound-healing/antigen-presenting tumor cells interacted with macrophages via CSF1/CSF1R, while proliferating and stem-like tumor cells interacted with neurons through multiple known NRXN/NLGN interactions). These interactions suggest potential targets for clinical disruption of the tumor ecosystem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20b29e4251f4b5ec080f177e87d960e1da6b0090" target='_blank'>
              EPCO-55. SPATIAL HETEROGENEITY IN THE TUMOR CORE AND INFILTRATING MARGIN OFIDHWT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Saad Khan, Anthony Wang, Rupen Desai, Jennifer Foltz, Eric Leuthardt, B. Patel, S. Dahiya, Albert Kim, Gavin Dunn, Allegra Petti
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20edd8d2bb6126475fe06f906514312b49ac959d" target='_blank'>
              Integrating histopathology and transcriptomics for spatial tumor microenvironment profiling in a melanoma case study
              </a>
            </td>
          <td>
            Óscar Lapuente-Santana, J. Kant, Federica Eduati
          </td>
          <td>2024-11-07</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a multi-clonal disease, existing as a milieu of clones with unique but related genotypes as initiating clones acquire subsequent mutations. However, bulk sequencing cannot fully capture AML clonal architecture or the clonal evolution that occurs as patients undergo therapy. To interrogate clonal evolution, we performed simultaneous single cell molecular profiling and immunophenotyping on 43 samples from 32 NPM1-mutant AML patients at different stages of disease. Here we show that diagnosis and relapsed AML samples display similar clonal architecture patterns, but signaling mutations can drive increased clonal diversity specifically at relapse. We uncovered unique genotype-immunophenotype relationships regardless of disease state, suggesting leukemic lineage trajectories can be hard-wired by the mutations present. Analysis of longitudinal samples from patients on therapy identified dynamic clonal, transcriptomic, and immunophenotypic changes. Our studies provide resolved understanding of leukemic clonal evolution and the relationships between genotype and cell state in leukemia biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3ca51b5a9370b4c12038c29c1cdd25d0324a8d0" target='_blank'>
              Genotype-immunophenotype relationships in NPM1-mutant AML clonal evolution uncovered by single cell multiomic analysis
              </a>
            </td>
          <td>
            Morgan Drucker, Darren Lee, Xuan Zhang, Bailee N. Kain, Michael Bowman, D. Nicolet, Zhe Wang, R. Stone, K. Mrózek, Andrew J Carroll, D. Starczynowski, R. L. Levine, John C. Byrd, Ann-Kathrin Eisfeld, Nathan Salomonis, H. L. Grimes, Robert L Bowman, Linde A. Miles
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is hallmarked by profound inter- and intratumoral heterogeneity. Current research has mapped key genetic drivers to distinct transcriptional profiles within GBM. Nevertheless, the intricate interactions between genetic alterations and spatial-transcriptional heterogeneity remain elusive. We utilized nestin-Tva C57BL/6 mice to induce de novo GBM via oncogene overexpression alongside shRNA-mediated silencing or deletion of tumor suppressors. We modeled critical driver mutations: PDGFB, NF1, and EGFR. Employing 10X Genomics platform for single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics (Nanostring GeoMx), we integrated data using both horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, analyzed through the SPATA2 software. Advanced computational techniques, including graph neural networks and single-cell deconvolution, were leveraged to unravel the genotypic and spatial variability within tumors and their microenvironments. Our findings uncover persistent cellular heterogeneity across diverse genetic contexts. However, specific genetic mutations precipitated distinct dominant transcriptional states: NF1 and EGFR mutations drove mesenchymal and astrocytic-like states, respectively, whereas PDGFB fostered a neural progenitor/cell-like state. These transcriptional shifts were closely linked to unique microenvironmental modifications, such as elevated neuronal signaling in PDGFB-driven GBMs and significant immune infiltration in NF1 and EGFR models. Through weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) paired with spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive microenvironment in NF1-loss driven GBMs and increased lymphocyte presence around blood vessels in EGFR-amplified GBMs. These patterns were consistent with human spatially resolved transcriptomic data, emphasizing the genetic complexity and heterogeneity of human GBMs. Our genetically engineered mouse models serve as a robust platform for dissecting GBM with defined genetic mutations. Leveraging sophisticated analytical techniques and computational modeling, this study underscores the profound interplay between genetic drivers and the tumor microenvironment, paving the way for targeted therapeutic strategies and precision medicine in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d3f584c4fce5754e72560c1ec2add719af021f" target='_blank'>
              EPCO-18. DECIPHERING SPATIAL DIVERSITY OF GLIOBLASTOMA GENETIC ARCHITECTURE
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 To effectively beat glioblastoma, new pre-clinical models are needed: to dissect the biology of processes like migration and therapy resistance; to understand immune suppression and immunotherapy; and to use as a model for testing in therapeutic development. We have developed a new panel of glioblastoma mouse models, which engineer well-characterised glioblastoma driver mutations into an immortalised eGFP murine astrocyte-like cell. These “GEM-CLeM” are Genetically Engineered Mouse Cell Line Models. When transplanted intra-cranially into immunocompetent mice, these lines formed tumors with the key characteristics of glioblastoma – migratory and invasive, with rapid proliferation and infiltration of suppressive immune cells. The eGFP expression allowed detection of glioblastoma cells in situ and ex vivo, and the ability to readily distinguish tumor and stromal components of the tumors. Importantly, different combinations of driver mutations led to diverse tumor phenotypes. Loss of Pten with over-expression of mutant RAS generated tumors with 100% efficiency and the same time to tumor onset as the commonly used GL261 cell line. However, unlike GL261 tumors the Pten-RASV12 tumors had a highly invasive edge and extensive migration, and were full of pro-tumorigenic macrophages and monocytes, with very few T-cells or dendritic cells. Together, this recapitulated the phenotype of human wildtype IDH1 glioblastoma. In contrast, the very common EGFRvIII mutant did not drive tumor formation in combination with Pten loss. We also modelled mutant IDH glioblastoma by combining loss of p53 with overexpression of IDH1R132H. These were slow growing, densely packed tumors with extensive microglial involvement in addition to pro-tumorigenic macrophages. This GEM-CLeM system combines the benefits of the traditional genetically engineered mouse models with accessible, rapid, reproducible tumor formation. The GEM-CLeM lines are amenable to further genetic manipulation, so cells can be customised with other mutations to ask important biological questions in the presence of an intact tumor immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/452f39600e3977617ed3591b2ad73dc07e6976ca" target='_blank'>
              TMIC-66. A NEW MOUSE MODEL THAT COMBINES DRIVER MUTATIONS TO ACCURATELY RECAPITULATE THE DIVERSE PHENOTYPES OF HUMAN GLIOBLASTOMA
              </a>
            </td>
          <td>
            D. Forsythe, Olivia Burn, I. Hermans, Melanie J McConnell
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell lineage tracing technology has advanced the investigation of progenitor cells’ development using static, inheritable barcodes. It can determine temporal dynamics in progenitor-progeny relationships through single-cell RNA-sequencing (scRNA-seq) data. However, studying fate commitment from scRNA-seq can be difficult since the gene expression profiles are confounded with information about many cell processes beyond fate commitment. This paper demonstrates a novel framework to specifically isolate lineage signals driving cell fate, allowing us to learn the gene pathways that differentiate different lineages based on their eventual fates. Our novel approach, LCL (Lineage-aware Contrastive Learning), is a contrastive-learning deep learning model for analyzing lineage-tracing scRNA-seq data. Using two lineage-tracing datasets, one about reprogramming embryonic fibroblasts and the other about hematopoietic progenitor cells, we demonstrate that LCL can produce low-dimensional representations that effectively isolate fate-determining signals from other key biological signals. We evaluate the quality of LCL embeddings and demonstrate that they perform well in out-of-sample evaluation, both in terms of predicting the lineage and cell type compositions at a future time point. LCL also enables us to identify differential genes stably expressed within a lineage and visualize the fate-determining landscape using self-organizing maps based on the results from LCL. Lastly, we demonstrate the consistency of our approach across datasets of varying complexity using a series of pseudo-real datasets. In conclusion, our results demonstrate that LCL allows researchers to explore fate commitment in single-cell lineage-tracing data and uncover lineage-specific gene pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31c58971d5eae176d87efa0d650b97339aae8ba" target='_blank'>
              LCL: Contrastive Learning for Lineage Barcoded scRNA-seq Data
              </a>
            </td>
          <td>
            Shizhao Joshua Yang, Yixin Wang, Kevin Z Lin
          </td>
          <td>2024-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d524bebda002b9fe5f1ee5e915d5586778f0b5b3" target='_blank'>
              A multi-modal single-cell and spatial expression map of metastatic breast cancer biopsies across clinicopathological features
              </a>
            </td>
          <td>
            Johanna Klughammer, Daniel L. Abravanel, Åsa Segerstolpe, Timothy R Blosser, Y. Goltsev, Yi Cui, Daniel R Goodwin, Anubhav Sinha, Orr Ashenberg, M. Slyper, S. Vigneau, Judit Jané-Valbuena, Shahar Alon, Chiara Caraccio, Judy Chen, Ofir Cohen, Nicole Cullen, L. Dellostritto, Danielle Dionne, Janet L. Files, Allison Frangieh, K. Helvie, Melissa E Hughes, Stephanie Inga, Abhay Kanodia, Ana Lako, Colin Mackichan, Simon Mages, Noa Moriel, Evan Murray, Sara Napolitano, Kyleen Nguyen, Mor Nitzan, Rebecca Ortiz, Miraj Patel, K. Pfaff, C. Porter, Asaf Rotem, Sarah Strauss, Robert Strasser, A. Thorner, M. Turner, Isaac Wakiro, Julia Waldman, Jingyi Wu, Jorge Gómez Tejeda Zañudo, Diane Zhang, Nancy U Lin, S. Tolaney, Eric P. Winer, E. Boyden, Fei Chen, Garry Nolan, Scott J Rodig, X. Zhuang, O. Rozenblatt-Rosen, Bruce E. Johnson, Aviv Regev, Nikhil Wagle
          </td>
          <td>2024-10-30</td>
          <td>Nature Medicine</td>
          <td>1</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1266e9e933529788695d2086bf906381800e149" target='_blank'>
              Single-cell and bulk transcriptome analysis reveals tumor cell heterogeneity and underlying molecular program in uveal melanoma
              </a>
            </td>
          <td>
            Ke Li, Jingzhe Huang, Ying Tan, Jie Sun, Meng Zhou
          </td>
          <td>2024-11-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 Myelodysplastic syndromes (MDS) represent a heterogenous group of myeloid neoplasms characterized by ineffective hematopoiesis, dysplastic morphology, and risk of transformation to acute leukemia. While neoplastic cell-intrinsic mechanisms driving MDS pathogenesis have been intensely investigated, the spatial architecture and in situ immune microenvironment in human MDS has not been studied at single-cell resolution. We hypothesized that the MDS bone marrow microenvironment is characterized by altered hematopoietic and non-hematopoietic cell distributions and transcriptional profiles including the composition and location of adaptive immune subsets.
 We developed an approach for high-definition spatial transcriptomic analysis of fixed and EDTA-decalcified human marrow core biopsies, enabling interrogation of hundreds of genes simultaneously in the context of their cellular location. We designed a marrow and immune focused 433 gene panel for the Xenium platform (10x Genomics) and applied this to 7 decalcified core marrow biopsies from patients with early-stage untreated MDS, and 3 control marrows, yielding in situ spatial data for 750,688 cells. In parallel, we performed 32-color spectral flow cytometry along with 5' single-cell CITE- and T cell receptor- (TCR) sequencing (N=31 MDS, N=9 controls) capturing 521,853 cells and 145,176 TCRs.
 Single-cell spatial transcriptomics identified >20 distinct cell types including populations that are poorly captured by analyses of single-cell suspensions from bone marrow aspirates: megakaryocytes, osteoblasts, endothelial cells, mesenchymal stromal cells, vascular smooth muscle cells, and adipocytes. Nearest neighbor analyses revealed reduced cell type diversity in MDS compared to controls, particularly surrounding osteoblasts (p=1.7x10-2). Within the immune compartment, we captured B, T, and natural killer (NK) cell types for further stratification into FOXP3+ T regulatory cells, naïve and memory CD4 or CD8 T cells, along with numerous phenotypic and functional markers.
 Spatial data were integrated with single-cell proteomic, RNA, and TCR sequencing in the same individuals to further elucidate the adaptive immune composition and T cell repertoire of the MDS marrow. In MDS samples, B cells were reduced in frequency whereas T cells were the most abundant lymphocyte cell type. Although MDS T cell subset composition was similar to control marrows, the spatial distribution of immune cells in MDS was distinct. Spatial analysis identified aggregates of T, B, and plasmacytoid dendritic cells (pDC) reminiscent of tertiary lymphoid structures in the marrow (which we defined as clusters of B, T, NK and pDCs with > 10 T cells). Immune aggregates were larger in MDS (>75 cells/aggregate, range 83-390) compared to controls (no aggregates >72 cells) with increased proportions of T cells (p=1.2x10-3) and frequencies of CD4 and CD8 T cell subsets. TCF7, a transcription factor linked to T cell stemness, and CXCR4, a key marker of bone marrow homing, were increased in CD4 (p=7.0x10-4 and p=1.5x10-5 respectively) and CD8 (p=1.6x10-2 and p=8.9x10-7 respectively) T cells in MDS immune aggregates. These findings suggest the emergence of immunologically active niches within the MDS marrow and would not have been captured via single-cell RNA-seq alone. Outside of immune aggregates, we found increased memory CD8 T cells (p=1.4x10-2) in MDS. In contrast, control marrows showed no difference in T cell subset frequency between cells inside or outside of immune aggregates. CITE/TCR-seq revealed clonal T cell expansions in MDS within GZMB+ CD8s, distinct from controls (where clonal expansions were found in GZMK+ CD8s), supporting altered CD8 T cell immunity in MDS.
 Our integrative analyses, including single-cell spatial transcriptomic profiling, revealed active remodeling of the adaptive immune system in early stage MDS. The finding of large immune aggregates in the MDS marrow with more T cells bearing signatures of active cell recruitment and growth, combined with increased clonally expanded memory CD8 T cells outside immune aggregates, point toward localized changes in the adaptive immune response in early stage MDS. Spatial analyses revealed location-specific T cell differences within the MDS marrow, underscoring the importance of combined phenotypic and spatial approaches to elucidate disease pathophysiology in MDS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b60776003cdeaafaa3fc36a226c720a5be827ebf" target='_blank'>
              Single-Cell Spatial Transcriptomics Identifies Lymphoid Aggregates with Location-Specific Alterations of Adaptive Immune Cells in Human MDS Bone Marrow
              </a>
            </td>
          <td>
            Robert F. Stanley, Beatrice Zhang, K. Argyropoulos, Brianna Gipson, Kenyon Weis, Alexander M. Lewis, Zoe Katsamakis, Karen Zhao, Catherine Snopkowski, Matthew Zatzman, Ning Fan, R. Chaligné, Umesh Bhanot, M. Roshal, A. Dogan, Benjamin D. Greenbaum, E. Stein, M. R. van den Brink, O. Abdel-Wahab, Susan DeWolf
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="The ability of cancer cells to alter their identity, known as lineage plasticity, is crucial for tumor progression and therapy resistance. In lung adenocarcinoma (LUAD), tumor progression is characterized by a gradual loss of lineage fidelity and the emergence of non-pulmonary identity programs. This can lead to hybrid-identity (hybrid-ID) states in which developmentally incompatible identity programs are co-activated within individual cells. However, the molecular mechanisms underlying these identity shifts remain incompletely understood. Here, we identify the gastrointestinal (GI) transcriptional regulator HNF4α as a critical driver of tumor growth and proliferation in KRAS-driven LUAD. In LUAD cells that express the lung lineage specifier NKX2-1, HNF4α can induce a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also forms an aberrant protein complex with NKX2-1, which disrupts NKX2-1 localization and dampens pulmonary identity within hybrid-ID LUAD. Sustained signaling through the RAS/MEK pathway is critical for maintaining the hybrid-ID state. Moreover, RAS/MEK inhibition augments NKX2-1 chromatin binding at pulmonary-specific genes and induces resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion enhances sensitivity to pharmacologic KRASG12D inhibition. Collectively, our data show that co-expression of opposing lineage specifiers leads to a hybrid identity state that can drive tumor progression and dictate response to targeted therapy in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207f2a257f041c211d79bb19f39ca01a49097c42" target='_blank'>
              Opposing lineage specifiers induce a pro-tumor hybrid-identity state in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Rushmeen Tariq, Anna Waters, Kayla T. O'Toole, Eric L. Snyder
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Supratentorial ependymomas (ST-EPN) are aggressive pediatric brain tumors that are categorized into distinct molecular subgroups. However, the developmental origins, tumor microenvironment, and phenotypic characteristics of tumor subpopulations across these subgroups are still poorly understood. In this study, we explored the human developmental signatures, global spatial organization, and the morphological and migratory tumor cell state behaviors in ST-EPN tumors at unprecedented resolution. We profiled 42 ST-EPN patients encompassing ZFTA-RELA (n = 20), ZFTA-Clusters 1 to 4 (n=20), and ST-YAP1 (n=4) subgroups by single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and live-cell imaging. We identified recurrent tumor cell states across tumors, that cover both generic cell processes (including cycling and mesenchymal/hypoxia) and human embryonic and fetal brain developmental signatures. ST-EPN subgroups displayed distinct developmental signatures, mapping to two separate temporally restricted progenitors-neuroepithelial-like and embryonic-like cells. Additionally, tumors showed diverging patterns of neuronal or ependymal differentiation, with ZFTA-Cluster 3 tumors as especially distinct from other subgroups in both developmental origin and differentiation. Furthermore, by utilizing 10X Xenium and novel algorithms we discovered that mesenchymal/hypoxia is critical for driving global tissue architecture to become more organized. Notably, recurrent tumors exhibited higher proportion of mesenchymal/hypoxia cells and greater structure than primary tumors. Lastly, by superimposing molecular and functional assays in both in vivo and in vitro models, we found that cell states exhibit distinct morphological and migratory behaviors, with neuronal-like cells being the most invasive. Neuronal-like cells especially exhibited distinct behavioral patterns reminiscent of neuronal migration during development. Moreover, we demonstrated the crucial role of neuronal microenvironment in promoting plasticity of cells toward this neuronal lineage. Taken together, we present a multidimensional framework for investigating cellular states within ST-EPN tumors, offering novel insights into their developmental origins, patterns of spatial organization, and cellular behavioral characteristics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab8e8753768336386cb49ecaeea5a571c7f71396" target='_blank'>
              STEM-11. MULTIDIMENSIONAL PROFILING OF TUMOR CELL HETEROGENEITY REVEALS CELL-LINEAGE SPECIFIC FUNCTIONS IN SUPRATENTORIAL EPENDYMOMAS
              </a>
            </td>
          <td>
            Daeun Jeong, Sara G Danielli, K. Maass, David R. Ghasemi, Svenja K. Tetzlaff, Ekin Reyhan, C. A. de Biagi-Junior, C. Cascio, S. Neyazi, Andrezza Nascimento, Rebecca D Haase, Bernhard Englinger, Li Jiang, Cuong M Nguyen, A. Baumgartner, Sophia Castellani, Jacob S Rozowsky, Olivia A Hack, McKenzie L. Shaw, Daniela Lotsch-Gojo, K. Bruckner, Stefan M Pfister, Marcel Kool, Tomasz J Nowakowski, J. Gojo, Lissa C Baird, S. Alexandrescu, K. Pajtler, V. Venkataramani, Mariella G. Filbin
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Glioblastoma (GBM) cells are highly heterogeneous and invasive, leading to treatment resistance and relapse. However, the molecular regulation in and distal to tumors remains elusive. Here, we collected paired tissues from the tumor core (TC) and peritumoral brain (PTB) for integrated snRNA-seq and snATAC-seq analyses. Tumor cells infiltrating PTB from TC behave more like oligodendrocyte progenitor cells than astrocytes at the transcriptome level. Dual-omics analyses further suggest that the distal regulatory regions in the tumor genome and specific transcription factors are potential determinants of regional heterogeneity. Notably, while activator protein 1 (AP-1) is active in all GBM states, its activity declines from TC to PTB, with another transcription factor, BACH1, showing the opposite trend. Combined inhibition of AP-1 and BACH1 more efficiently attenuates the tumor progression in mice and prolongs survival than either single-target treatment. Together, our work reveals marked molecular alterations of infiltrated GBM cells and a synergy of combination therapy targeting intratumor heterogeneity in and distal to GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c193d462d0c2dbf4aee7a761498e3e53b84808" target='_blank'>
              Single-cell multi-omics sequencing uncovers region-specific plasticity of glioblastoma for complementary therapeutic targeting
              </a>
            </td>
          <td>
            Xin Wang, Qian Sun, Tianbin Liu, Haoran Lu, Xuyi Lin, Weiwen Wang, Yang Liu, Yunting Huang, Guodong Huang, Haixi Sun, Qianxue Chen, Junmin Wang, Daofeng Tian, Fan-En Yuan, Longqi Liu, Bo Wang, Ying Gu, Baohui Liu, Liang Chen
          </td>
          <td>2024-11-22</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCCs) constitute one of the few cancer indications for which mortality rates continue to rise. While Notch signaling dictates a key progenitor lineage choice during development, its role in HCC has remained controversial. Using therapeutic antibodies targeting Notch ligands and receptors to screen over 40 patient-derived xenograft models, we here identify progenitor-like HCCs that crucially depend on a tumor-intrinsic JAG1-NOTCH2 signal. Inhibiting this signal induces tumor regressions by triggering progenitor-to-hepatocyte differentiation, the same cell fate-switch that Notch controls during development. Transcriptomic analysis places the responsive tumors within the well-characterized progenitor subclass, a poor prognostic group of highly proliferative tumors, providing a diagnostic method to enrich for Notch-dependent HCCs. Furthermore, single-cell RNA sequencing uncovers a heterogeneous population of tumor cells and reveals how Notch inhibition shifts cells from a mixed cholangiocyte-hepatocyte lineage to one resembling mature hepatocytes. Analyzing the underlying transcriptional programs brings molecular detail to this process by showing that Notch inhibition de-represses expression of CEBPA, which enables the activity of HNF4α, a hepatocyte lineage factor that is otherwise quiescent. We thus describe a compelling and targetable dependency in a poor-prognosis class of HCCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3764bfcf8bf124425bfb62c588b7e271cb0186df" target='_blank'>
              Notch signaling maintains a progenitor-like subclass of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kerstin Seidel, Robert Piskol, Thi Thu Thao Nguyen, Amy Shelton, Charisa L. Cottonham, Cecile C. de la Cruz, Joseph Castillo, Jesse Garcia, Udi Segal, Mark Merchant, Yeqing Angela Yang, Jasmine Chen, Musa Ahmed, Alexis Scherl, Rajesh Vij, Lluc Mosteiro, Yan Wu, Z. Modrusan, C. Metcalfe, Christopher W. Siebel
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="


 Within the GBM microenvironment, distinct micro-niches arise from independent cancer cell lineages with unique metabolic needs, characterized as Fast Cycling Cells (FCCs) and Slow Cycling Cells (SCCs). FCCs preferentially utilize aerobic glycolysis for energy, whereas treatment-resistant SCCs depend on lipid metabolism. Our study decodes the molecular and spatial heterogeneity within the GBM microenvironment, focusing on the dichotomy of immune infiltrates and the metabolic interactions between SCCs and the immune compartment in human GBM patients.



 We performed spatial transcriptomics and proteomics combined with geospatially resolved single-cell neighborhood analysis of IHC-stained GBM tissues using CellProfiler and the SNAQ algorithm that we developed to investigate the specific immune microenvironment of SCCs and FCCs. Holotomography and fluorescence-based time-lapse imaging coupled with flow cytometry were used to study lipid transfer between immune cells and GBM cells. Finally, the effects of pharmacological and genetic targeting of lipid transfer were investigated both in vitro and in vivo.



 Our study establishes a connection between spatial metabolic heterogeneity and immune diversity, highlighting how specific tumor cell lineages correlate with characteristic immune micro-niches. In particular, SCCs exploit the metabolic properties of recruited immunosuppressive myeloid-derived cells, which metabolically support tumor cells by facilitating lipid transport and supporting SCC metabolism. Our results show that inhibiting lipid transfer affect SCCs, remodels the immune microenvironment, delays tumor growth and improves disease outcome. Our study also demonstrates a correlation between predicted sensitivity to lipid-modifying drugs, such as statins, and the SCC phenotype. This correlation could enable stratification and selection of patients most likely to benefit from such treatments.



 This study establishes critical connections between the TME, cellular metabolism, anti-tumor immunity, and treatment vulnerabilities. Our research provides novel insights into how metabolic exchanges contribute to tumor progression and suggests that targeting metabolic interactions could be an effective therapeutic strategy for GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec73458fa94d4e1053c3c077508f221cb827e3" target='_blank'>
              TMIC-52. DECODING METABOLIC INTERACTIONS IN GLIOBLASTOMA TUMOR MICROENVIRONMENT THROUGH COMPREHENSIVE SPATIAL TRANSCRIPTOMIC AND PROTEOMIC ANALYSIS
              </a>
            </td>
          <td>
            Avirup Chakraborty, Changlin Yang, Diana Feier, Aryeh Silver, Miruna Anica, Avinash Pittu, Illeana West, Olusegun Sobanjo, Ethan Hodge, Mia Engelbart, Srabasti Sengupta, Nagheme Thomas, C. V. von Roemeling, Maryam Rahman, Matthew R. Sarkisian, Jianping Huang, Jeffrey K Harrison, Duane A Mitchell, L. Deleyrolle
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Spatial transcriptomics offers unprecedented insights into the complex cellular landscapes of tissues, particularly in cancer research where understanding the tumor microenvironment is crucial. However, interpreting the vast and intricate data generated by this technology remains a significant challenge. This study explores the potential of Large Language Models (LLMs) to assist in the analysis and interpretation of spatial transcriptomic data from a murine melanoma tumor model. We first evaluated the performance of multiple LLM models in describing and quantifying spatial gene expression patterns. Our benchmarking revealed that spatial transcriptomics data interpretation proved challenging for most models, with only a few demonstrating sufficient capability for this complex task. Using Claude 3.5 Sonnet, which showed the highest accuracy in spot quantification and pattern recognition, we developed a systematic workflow for analyzing the tumor immune landscape. The model first assisted in identifying and prioritizing panels of M1 and M2 macrophage-associated markers through knowledge-driven scoring. It then demonstrated remarkable ability to integrate spatial expression data with extensive immunological knowledge, providing sophisticated interpretation of local immune organization. When analyzing individual tumor regions, the model identified coordinated immunosuppressive mechanisms including metabolic barriers and disrupted pro-inflammatory signaling cascades, findings that both aligned with and extended current understanding of tumor immunology. This study highlights the potential of LLMs as powerful assistive tools in spatial transcriptomics analysis, capable of combining advanced pattern recognition with extensive knowledge integration to enhance data interpretation. While significant development work remains to make such workflows scalable, our proof of concept demonstrates that LLMs can help accelerate the translation of spatial transcriptomics data into biological insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287bc0c305d5950e4e6f2901503d04b32465b5e" target='_blank'>
              Tackling the Complexity of Spatial Transcriptomics Data Interpretation with Large Language Models
              </a>
            </td>
          <td>
            Taushif Khan, Colleen M. Farley, John J. Wilson, Chih-Hao Chang, Damien Chaussabel
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Research on cell type markers aids investigators in exploring the diverse cellular compositions within gastrointestinal tumors, enhancing our understanding of tumor heterogeneity and its implications for disease progression and treatment response. However, issues such as the integration of large-scale datasets and the lack of standardized cell type identification hinder comprehensive characterization. Here, we developed a user-friendly web interface called PreDigs (Predicted Signatures in Digestive System), which offers 124 tailored scRNA-seq datasets available for download, encompassing over 3.4 million cells. After unsupervised clustering, we unified the identification and naming of subtype labels, ultimately constructing a cell ontology tree that includes 142 cell types, with up to eight hierarchical levels. Meanwhile, we calculated three different context-specific cell-type markers—’Cell Markers’, ‘Subtype Markers’, and ‘TPN Markers’—based on various application requirements within or across tissues. Through the integrated analysis of PreDigs gastrointestinal data, we identified distinct cell subpopulations exclusive to tumors, one of which corresponds to tumor-specific endothelial cells (TEC). Furthermore, PreDigs offers online cell annotation tools that empower users to perform single-cell classification with greater flexibility, accessible at https://www.biosino.org/predigs/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f066c350d98316ff9071f40e0805e670af0f667d" target='_blank'>
              PreDigs: a Database of Context-specific Cell-type Markers and Precise cell subtypes for Digestive Cell Annotation
              </a>
            </td>
          <td>
            Jiayue Meng, Mengyao Han, Yuwei Huang, Liang Li, Yuanhu Ju, Daqing Lv, Xiaoyi Chen, Liyun Yuan, Guoqing Zhang
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 The spatial organization of “immune hubs” in tumor microenvironments has gained recognition across histologies and tissues. Bone marrow offers a valuable platform for investigating immune hubs in the context of acute myeloid leukemia (AML) following hematopoietic stem cell transplant (HSCT). This setting may provide fresh insights into mechanisms of relapse and maintenance of remission after donor lymphocyte infusion (DLI). While it is generally understood that DLI exerts its effects through a graft-versus-leukemia (GVL) response, the specific cellular players and spatial organization driving GVL remain unidentified. Addressing this question can inform the rational design of cancer cellular therapies.
 A major challenge for spatial transcriptomic assays interrogating bone marrow samples is RNA degradation due to decalcification in addition to that caused by formalin fixation and paraffin embedding (FFPE). Furthermore, existing spatial platforms lack single-cell resolution and their throughput is generally limited to a small number of samples. To address these constraints, Singular Genomics is developing the G4XTM, an in-situ next-generation sequencing (NGS) platform that enables rapid sample throughput (up to 20-fold greater than existing platforms), and enables combined readouts of transcriptomics, proteomics, and fluorescent H&E staining in the same FFPE section.
 We used a pre-release version of the G4X to analyze bone marrow biopsies of patients with relapsed AML before and after treatment with DLI. We profiled 24 specimens (6 responder [R] pre-DLI, 7 R post-DLI, 5 non-responder [NR] pre-DLI, 6 NR post-DLI), aiming to elucidate the spatial relationships of immune cellular networks in the marrow microenvironment. Our targeted transcript panel included 153 immune- and marrow-related genes defining specific T cell subsets and other immune cell lineages, as well as 8 protein markers (CD3, CD4, CD8, CD45RA, HLA-DR, CD34, KI67, ATPase). We filtered out cells with transcript counts above the 95th percentile and retained cells with at least 10 transcripts and 3 unique genes. We excluded genes present in less than 5 cells and considered segmented nuclei sizes between 2.6 µm and 14.2 µm, resulting in 193,651 cells (median of 30 transcripts, 11 unique genes per cell). Further pre-processing consisted of batch correction via Harmony (Korsunsky 2019), dimensional reduction and clustering.
 Cell populations across 14 distinct clusters were annotated using gene signatures derived from previously generated single-cell RNA sequencing data (Maurer et al ASH 2023) and refined by considering differentially expressed marker genes and protein expression profiles. We previously demonstrated expansion of an activated CD8 T cell population with high expression of ZNF683 and GZMB uniquely in R post-DLI (Maurer et al ASH 2023). Hence, we interrogated the dynamics and spatial relationships of this specific T cell subset, identified among other lymphoid cells in the in situ data based on gene and protein signatures.
 In confirmation of our previous findings, we found that this T cell subset increased in proportion specifically in R post-DLI samples (3.6% to 6.4% in R pre to post; 2.9% to 1.9% in NR pre to post, p=ns) and exhibited decreased spatial self-association in NRs (8.79% to 2.17% in NR pre to post, 32.5 µm radius, p=0.008). In Rs, these CD8 T cells were more likely to be in the spatial vicinity of myelo-erythroid progenitor and AML cells, perhaps suggestive of active immune surveillance post-DLI (1.62% in NR post vs 6.32% in R post, 32.5 µm radius, p=0.008).
 When examining spatial organization of all cell types, bone marrow of NRs pre-DLI was marked by an overabundance of myeloid cells relative to other cell types. The microenvironment of Rs was more heterogeneous before treatment and showed highest immunologic diversity post-DLI (Shannon Diversity Index: 0.744 in NR post vs. 0.943 in R post, p=0.021). These findings suggest that diversity of marrow composition pre-DLI may be predictive of therapeutic response, though this observation requires validation in larger cohorts.
 We present the first analysis of post-HSCT relapsed AML marrow microenvironment with spatial multi-omic profiling using G4X and find evidence of post-DLI T cell expansion concordant with our scRNA-seq analysis. Further evaluation of spatial organization may lead to novel targets for optimizing adoptive cellular immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f02f9e3e994f545d0f9f4bff84f2a37013298007" target='_blank'>
              Deconvolving Immunologic Networks of the AML Marrow Microenvironment with Spatial Multi-Omic Profiling
              </a>
            </td>
          <td>
            K. Maurer, Florian Raths, Kenneth Gouin, Cameron Y Park, Michael Lawson, Jake Koh, Martin M. Fabani, K. Livak, Jerome Ritz, R. Soiffer, Eli Glezer, Catherine J. Wu, Elham Azizi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Studying the spatiotemporal dynamics of cells in living organisms is a current frontier in bioimaging. Intravital Microscopy (IVM) provides direct, long-term observation of cell behavior in living animals, from tissue to sub-cellular resolution. Hence, IVM has become crucial for studying complex biological processes in motion and across scales, such as the immune response to pathogens and cancer. However, IVM data are typically kept in private repositories inaccessible to the scientific community, hampering large-scale analysis that aggregates data from multiple laboratories. To solve this issue, we introduce Immunemap, an atlas of immune cell motility based on an Open Data platform that provides access to over 58’000 single-cell tracks and 1’049’000 cell-centroid annotations from 360 videos in murine models. Leveraging Immunemap and unsupervised learning, we systematically analyzed cell trajectories, identifying four main patterns of cell migration in immune cells. Two patterns correspond to behaviors previously characterized: directed movement and arresting. However, we identified two other patterns, characterized by low directionality and twisted paths, often considered random migration. We show that the newly defined patterns can be subdivided into two distinct types: within small areas, suggesting a focused patrolling around one or a few cells, and over larger areas, indicative of a more extended tissue patrolling. Furthermore, we show that the percentage of cells displaying these motility patterns changes in response to immune stimuli. Altogether, Immunemap embraces the FAIR principles, promoting data reuse to extract novel insights from immune cell dynamics through an image-based systems biology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fcec1b17c98e94c83b787c02db01f7ce0422437" target='_blank'>
              Systematic analysis of immune cell motility leveraging Immunemap, an open intravital microscopy atlas
              </a>
            </td>
          <td>
            D. U. Pizzagalli, Pau Carrillo-Barberà, Elisa Palladino, Kevin Ceni, Benedikt Thelen, Alain Pulfer, Enrico Moscatello, R. F. Cabini, Johannes Textor, Inge M. N. Wortel, Rolf Krause, S. F. Gonzalez
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is a highly aggressive brain cancer with limited treatment options. This research aims to improve our understanding of GBM progression by focusing on extracellular vesicles released by tumor cells that carry important biological information. The authors developed a laboratory model that closely mimics the tumor microenvironment, allowing for the study of extracellular vesicles released from various cell types within the tumor. This model enabled the identification of specific surface markers on extracellular vesicles that could serve as potential noninvasive diagnostic tools for GBM. The findings suggest that these markers may help improve extracellular vesicle isolation and provide insights into the molecular characteristics of the tumor. This study has the potential to significantly impact the research community by offering new approaches to GBM diagnosis and advancing our knowledge of tumor biology, which could ultimately inform future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ccc9a9ae6ff3dcd3e9189e563853f77db227f69" target='_blank'>
              Exploring Extracellular Vesicle Surface Protein Markers Produced by Glioblastoma Tumors: A Characterization Study Using In Vitro 3D Patient-Derived Cultures
              </a>
            </td>
          <td>
            S. Franceschi, F. Lessi, M. Morelli, M. Menicagli, P. Aretini, C. Gambacciani, F. Pieri, G. Grimod, Maria Grazia Trapanese, Silvia Valenti, F. Paiar, A. D. Di Stefano, O. Santonocito, F. Pasqualetti, C. M. Mazzanti
          </td>
          <td>2024-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67b3c3d2c93cdd85dd75bfc027eecd714525336" target='_blank'>
              Neuroblastoma plasticity during metastatic progression stems from the dynamics of an early sympathetic transcriptomic trajectory
              </a>
            </td>
          <td>
            Benjamin Villalard, A. Boltjes, F. Reynaud, Olivier Imbaud, Karine Thoinet, Ilse Timmerman, S. Croze, Emy Théoulle, Gianluigi Atzeni, Joël Lachuer, Jan J. Molenaar, G. Tytgat, C. Delloye-Bourgeois, Valérie Castellani
          </td>
          <td>2024-11-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Simple Summary Understanding neural crest stem cells as the cells of origin for a series of tumors expands the potential strategies for treatment, especially for rare tumor types. The shared properties and signatures of stem-like tumor cells, also called cancer stem cells (CSCs), are reviewed for neural crest-originated tumors. Scrutinizing the relationship between neural crest stem cells and CSCs in NF1-associated tumors can allow for earlier detection and treatment in NF1 patients. Previously identified CSC markers can be the key to NF1 tumor pathogenesis and therapeutic targets. Targeting CD44, for example, can potentially reduce CSC-related tumor progression, relapse, and even metastasis. This review delineates the reported CSCs in tumors of neural crest origin and highlights their potential relevance in NF1-associated tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d07716feae4e31bf912bae8341056b15cf0fb14" target='_blank'>
              Leveraging Neural Crest-Derived Tumors to Identify NF1 Cancer Stem Cell Signatures
              </a>
            </td>
          <td>
            Sajjad Khan, Donia Alson, Li Sun, Caroline Maloney, Daochun Sun
          </td>
          <td>2024-10-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune checkpoint inhibitors such as anti-PD-1 antibodies (aPD1) can be effective in treating advanced cancers. However, many patients do not respond and the mechanisms underlying these differences remain incompletely understood. In this study, we profile a cohort of patients with locally-advanced or metastatic basal cell carcinoma undergoing aPD-1 therapy using single-cell RNA sequencing, high-definition spatial transcriptomics in tumors and draining lymph nodes, and spatial immunoreceptor profiling, with long-term clinical follow-up. We find that successful responses to PD-1 inhibition are characterized by an induction of B-cell receptor (BCR) clonal diversity after treatment initiation. These induced BCR clones spatially co-localize with T-cell clones, facilitate their activation, and traffic alongside them between tumor and draining lymph nodes to enhance tumor clearance. Furthermore, we validated aPD1-induced BCR diversity as a predictor of clinical response in a larger cohort of glioblastoma, melanoma, and head and neck squamous cell carcinoma patients, suggesting that this is a generalizable predictor of treatment response across many types of cancers. We discover that pre-treatment tumors harbor a characteristic gene expression signature that portends a higher probability of inducing BCR clonal diversity after aPD-1 therapy, and we develop a machine learning model that predicts PD-1-induced BCR clonal diversity from baseline tumor RNA sequencing. These findings underscore a dynamic role of B cell diversity during immunotherapy, highlighting its importance as a prognostic marker and a potential target for intervention in non-responders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c13f381588d31078af3cc3cb23453357caf14bf3" target='_blank'>
              Induced B-Cell Receptor Diversity Predicts PD-1 Blockade Immunotherapy Response
              </a>
            </td>
          <td>
            Yonglu Che, Jinwoo Lee, Farah Abou-Taleb, Kerri E. Rieger, Ansuman T. Satpathy, A. L. S. Chang, Howard Y. Chang
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Diffuse invasion of glioblastoma cells through normal brain tissue is a key contributor to tumor aggressiveness, resistance to conventional therapies, and dismal prognosis in patients. A deeper understanding of how components of the tumor microenvironment (TME) contribute to overall tumor organization and to programs of invasion may reveal opportunities for improved therapeutic strategies. Towards this goal, we applied a novel computational workflow to a spatiotemporally profiled GBM xenograft cohort, leveraging the ability to distinguish human tumor from mouse TME to overcome previous limitations in analysis of diffuse invasion. Our analytic approach, based on unsupervised deconvolution, performs reference-free discovery of cell types and cell activities within the complete GBM ecosystem. We present a comprehensive catalogue of 15 tumor cell programs set within the spatiotemporal context of 90 mouse brain and TME cell types, cell activities, and anatomic structures. Distinct tumor programs related to invasion were aligned with routes of perivascular, white matter, and parenchymal invasion. Furthermore, sub-modules of genes serving as program network hubs were highly prognostic in GBM patients. The compendium of programs presented here provides a basis for rational targeting of tumor and/or TME components. We anticipate that our approach will facilitate an ecosystem-level understanding of immediate and long-term consequences of such perturbations, including identification of compensatory programs that will inform improved combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/678e476a090fe3a350f2e1d603e09efd449e924a" target='_blank'>
              ANGI-05. UNSUPERVISED DECONVOLUTION OF SPATIOTEMPORAL HETEROGENEITY REVEALS HIGH-RESOLUTION INVASION STATES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Varsha Thoppey Manoharan, Aly O. Abdelkareem, Samuel Brown, Aaron H. Gillmor, Courtney Hall, Heewon Seo, Kiran Narta, Sean Grewal, Ngoc Ha Dang, Bo Young Ahn, Kata Osz, Xueqing Lun, Douglas Mahoney, Donna L Senger, Jennifer A. Chan, Sorana A. Morrissy
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Gliomas are lethal malignancies comprised of heterogeneous and dynamic subpopulations of cell states resembling normal neurodevelopmental cell types (radial glia (RG), oligodendrocyte progenitor cell (OPC), neuron progenitor cell (NPC), neuron, etc). While both intrinsic (genetics) and extrinsic (brain environment) cues are coupled to glioma state identity, the functional programs governing glioma cell state and plasticity are unknown. Here we performed a multi-omic interrogation of a large library (n=392) of glioma patient tumors, in vivo orthotopic xenografts and in vitro gliomasphere cultures. Comparisons of matched glioma samples across environments revealed the non-native in vitro environment constrains glioma state diversity specifically enriching for stem-like glioma cell states (RG, immune, vascular). This enrichment was linked to lipid metabolic flexibility, enabling tumors enriched for stem-like states to adapt to various tumor microenvironments. By contrast, “lineage-committed” cell states (OPC, NPC, and neuron) demonstrate restricted lipid metabolism, consequently having a dependence on lipid scavenging from the brain microenvironment for survival. These results connect intra-tumoral heterogeneity and plasticity of glioma to lipid metabolism, leveraging the functional diversity of cellular states to reveal potential therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1598ae760fc108f7d962df3e2f211396a763ed04" target='_blank'>
              TMET-01. LIPID METABOLIC PROGRAMS DEFINE GLIOMA STATE IDENTITY, PLASTICITY AND DEPENDENCIES
              </a>
            </td>
          <td>
            Jenna K. Minami, Nicholas A. Bayley, C. Tse, Cassidy Andrasz, Dimitri Cadet, Jennifer Salinas, Weihong Yan, Amir Borujerdpur, L. Liau, T. Cloughesy, Kevin Williams, S. Bensinger, A. Bhaduri, Thomas Graeber, David Nathanson
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e01bd46496bf7ca9aab2045ddff751c6da9e8f4" target='_blank'>
              Tumor microenvironment remodeling after neoadjuvant chemoradiotherapy in local advanced rectal cancer revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Wen-jing Su, Yuhang Ling, Xiaodong Yang, Yong Wu, Chungen Xing
          </td>
          <td>2024-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e256ab784a11c5f4d9d24e6065e96a769a156f9" target='_blank'>
              Human stem cell models to unravel brain cancer
              </a>
            </td>
          <td>
            Biren Dave, J. Tailor
          </td>
          <td>2024-11-28</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Liver cancer, the third leading cause of cancer-related mortality worldwide, has two main subtypes: hepatocellular carcinoma (HCC), accounting most of the cases, and cholangiocarcinoma (CAA). NOTCH pathway regulates the intrahepatic development of bile ducts, which are lined with cholangiocytes, but it can also be upregulated in 1/3 of HCCs. To better understand the role of NOTCH in HCC, we developed a novel mouse model driven by activated NOTCH1 intracellular domain (NICD1) and MYC overexpression in hepatocytes. Using the hydrodynamic tail vein injection method for establishing primary liver tumors, we generated a novel murine model of liver cancer harboring MYC overexpression and NOTCH1 activation. We characterized this model histopathologically as well as transcriptomically, utilizing both bulk and single cell RNA-sequencing. MYC;NICD1 tumors displayed a combined HCC-CCA phenotype with temporal plasticity. At early time-points, histology was predominantly cholangiocellular, which then progressed to mainly hepatocellular. The hepatocellular component was enriched in mesenchymal genes and gave rise to lung metastasis. Metastatic cells were enriched in the TGFB and VEGF pathways and their inhibition significantly reduced the metastatic burden. Our novel mouse model uncovered NOTCH1 as a driver of temporal plasticity and metastasis in HCC, the latter of which is, in part, mediated by angiogenesis and TGFß pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c28585f46d87d60d05666ef2d70c27c2088b6b7" target='_blank'>
              NOTCH1 drives tumor plasticity and metastasis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            K. Lindblad, Romain Donne, Ian Liebling, Erin R. Bresnahan, M. Bárcena-Varela, Anthony Lozano, Eric Park, Bruno Giotti, Olivia K. Burn, M. Fiel, J. Bravo-Cordero, A. Tsankov, A. Lujambio
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) comprises diverse cell types whose interactions are crucial for cancer progression. Traditional bulk tissue analyses mask cell-type-specific epigenetic changes that may drive disease progression. We used DNA methylation data and a reference- based cell-type deconvolution algorithm — hierarchical tumor immune microenvironment epigenetic deconvolution (HiTIMED) — to quantify cell types in the TME. Then, we employed CellDMC, a statistical interaction testing framework that can use cell type proportions in conjunction with differential methylation analysis to identify cell-specific differential methylation. Genome-scale DNA methylation profiles from breast cancer tissue (n=609) and normal breast tissue (n=230) samples were accessed in GEO and GTEx. Our novel approach leverages bulk DNA methylation data to infer cell-type-specific epigenetic landscapes, circumventing challenges and limitations associated with direct single-cell methylation profiling, such as high costs and technical variability. Reference-based computational tools allow us to dissect complex tissue compositions and reveal distinct methylation patterns and pathways that may be unique to individual cell types, providing insights that are not discernible from traditional bulk tumor analysis approaches. Compared with nontumor normal samples, we identified significant DNA methylation alterations (FDR≤0.05) across tumor, stromal, endothelial, epithelial, lymphocytes, and myeloid cells in breast tumors. Tumor cells exhibited the highest number, 8,510, differentially methylated cytosines (DMCs) with 1,966 of these unique to tumor cells. Stromal, endothelial, and epithelial cells also showed considerable methylation alterations, with a total of 530, 440, 339 DMCs, respectively, with 5 DMCs unique to epithelial cells. Despite lower immune cell proportions in our samples, we identified 104 lymphocyte specific DMCs and 185 myeloid specific DMCs, with 14 and 20 DMCs unique to these cell types respectively. The unique DMCs identified in each cell type suggest distinct epigenetic landscapes that may drive specific cellular functions and interactions within the TME. Importantly, Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analysis of the annotated cell-type-specific DMCs revealed distinct pathways that were not apparent in the pathway analysis of bulk tumor versus normal tissue, underscoring the value of examining epigenetic alterations at the cell-type level. Our findings highlight the opportunity to leverage DNA-based cell typing with DNA methylation data to discern cell-specific molecular alterations. By focusing on cell-type-specific alterations, we can better understand the epigenetic mechanisms driving TME crosstalk, such as the activation of stromal cells and the modulation of immune responses by tumor cells. Our findings underscore the need for targeted therapeutic strategies that consider the unique epigenetic landscapes of individual cell types within the TME, offering new avenues for improving cancer treatment outcomes.
 Citation Format: Barbara Karakyriakou, Brock C. Christensen, Lucas A. Salas. Identification of cell-specific DNA methylation alterations in the breast tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae2a87bb3a64a66fe890b4d4d581317faf06571" target='_blank'>
              Abstract B033: Identification of cell-specific DNA methylation alterations in the breast tumor microenvironment
              </a>
            </td>
          <td>
            Barbara Karakyriakou, Brock C. Christensen, Lucas A Salas
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Gliobastoma (GBM) is a highly aggressive solid tumor of the brain, characterized by a hypoxic tumor microevironment (TME) leading to histopathological features of necrosis, microvascular proliferation and deregulated hypervascularization. The heterogeneous nature of GBM results in a gradient of intra-tumoral hypoxia that causes molecular changes to specific cell populations within the bulk of the tumor. Our study is the first clinical study to utilize the exogenous oxygen-independent marker pimonidazole (PIMO) to identify the hypoxic TME using high-dimensional spatial and single-cell proteomics of specific cell populations within GBM. A cohort of 35 patients with primary GBM, recurrent GBM or IDH-mutant glioma were administered PIMO prior to surgical resection of tumor tissue for downstream bulk proteomic analysis of bulk tissue by serial immunohistochemistry and imaging mass cytometry (IMC) and single-cell analysis of dissociated tissue by Cytometry by time-of-flight (CyTOF). We utilized high-resolution imaging analyses to validate PIMO as a sensitive and stable marker for hypoxia against a panel of transiently expressed HIF-target genes and examine the inter- and intra-tumoral heterogeneity of hypoxia within each tumor subtype. A custom CyTOF marker panel was developed for further single-cell analyses of T-cell subsets and their functional states, myeloid cell subpopulations, natural killer cells, glial cells, and tumor cells in the hypoxic TME. We demonstrate the utility of PIMO to effectively study the hypoxic TME through spatial and single cell methods and further elucidate mechanisms of hypoxia-driven treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa287ddd81b049b43e5a448648f6d6d942969be3" target='_blank'>
              TMIC-80. HIGH-DIMENSIONAL HISTOPATHOLOGIC EVALUATION OF THE HYPOXIC TUMOR MICROENVIRONMENT IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Shreya Gandhi, O. Singh, S. Mansouri, S. Karimi, Phoebe Lombard, Anna Dvorkin-gheva, Carlos Velasquez, Hafsah Ali, Kaviya Devaraja, Qingxia Wei, Ronald Wu, Mira Li, May Cheung, M. Voisin, A. Gao, David Hedley, Kenneth J Aldape, Bradley Wouters, G. Zadeh
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e6a9e6b43769b80db465f9dd9f678e6c40ef5c" target='_blank'>
              Single cell and spatial transcriptomics highlight the interaction of club-like cells with immunosuppressive myeloid cells in prostate cancer
              </a>
            </td>
          <td>
            A. Kiviaho, Sini K. Eerola, H. Kallio, M. K. Andersen, Miina Hoikka, A. Tiihonen, I. Salonen, Xander Spotbeen, A. Giesen, C. T. Parker, S. Taavitsainen, Olli Hantula, Mikael Marttinen, Ismaïl Hermelo, Mazlina Ismail, Elise Midtbust, M. Wess, W. Devlies, Abhibhav Sharma, S. Krossa, Tomi Häkkinen, Ebrahim Afyounian, K. Vandereyken, Sam Kint, J. Kesseli, T. Tolonen, T.L.J. Tammela, T. Viset, Øystein Størkersen, G. Giskeødegård, M. B. Rye, T. Murtola, A. Erickson, Leena Latonen, G. Bova, Ian G. Mills, S. Joniau, J. Swinnen, Thierry Voet, T. Mirtti, Gerhardt Attard, Frank Claessens, T. Visakorpi, K. Rautajoki, May-Britt Tessem, Alfonso Urbanucci, M. Nykter
          </td>
          <td>2024-11-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Cell tracking in chimeric models is essential yet challenging, particularly in developmental biology, regenerative medicine, and transplantation studies. Existing methods, such as fluorescent labeling and genetic barcoding, are technically demanding, costly, and often impractical for dynamic, heterogeneous tissues. To address these limitations, we propose a computational framework that leverages sex as a surrogate marker for cell tracking. Our approach uses a machine learning model trained on single-cell transcriptomic data to predict cell sex with high accuracy, enabling clear distinction between donor (male) and recipient (female) cells in sex-mismatched chimeric models. The model identifies specific genes critical for sex prediction and has been validated using public datasets and experimental flow sorting, confirming the biological relevance of the identified cell populations. Applied to skeletal muscle macrophages, our method revealed distinct transcriptional profiles associated with cellular origins. This pipeline offers a robust, cost-effective solution for cell tracking in chimeric models, advancing research in regenerative medicine and immunology by providing precise insights into cellular origins and therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2607809a828288d92a6fadd5cfee754b3bcdb55" target='_blank'>
              CellSexID: Sex-Based Computational Tracking of Cellular Origins in Chimeric Models
              </a>
            </td>
          <td>
            Huilin Tai, Qian Li, Jingtao Wang, Jiahui Tan, Ryann Lang, Basil J. Petrof, Jun Ding
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. A better understanding of the temporal dynamics and specific pathways leading into and out of the persister state is needed to identify strategies to prevent treatment failure. Using spatial transcriptomics in patient-derived xenograft models, we captured clonal lineage evolution during treatment. The persister state showed increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients. Phylogenetic tracing identified intrinsic and acquired resistance mechanisms (e.g., dual specific phosphatases, reticulon-4, and CDK2) and suggested specific temporal windows of potential therapeutic susceptibility. Deep learning-enabled analysis of histopathological slides revealed morphological features correlating with specific cell states, demonstrating that juxtaposition of transcriptomics and histological data enabled identification of phenotypically distinct populations from using imaging data alone. In summary, this study defined state change and lineage selection during melanoma treatment with spatiotemporal resolution, elucidating how choice and timing of therapeutic agents will impact the ability to eradicate resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d694b41718972dca1b5b8aa26a4498984d65d307" target='_blank'>
              Spatiotemporal Profiling Defines Persistence and Resistance Dynamics During Targeted Treatment of Melanoma.
              </a>
            </td>
          <td>
            Jill C. Rubinstein, Sergii Domanskyi, Todd B Sheridan, Brian Sanderson, SungHee Park, Jessica Kaster, Haiyin Li, Olga Anczuków, M. Herlyn, Jeffrey H. Chuang
          </td>
          <td>2024-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Organoid models have advanced our understanding of adult stem/progenitor cell dynamics and function. However, uncovering the regulatory mechanisms of scarce and often quiescent stem cells in organs like the salivary glands remains challenging. Using single-cell RNA sequencing and bulk ATAC and RNA-sequencing analysis, we conducted in-depth profiling of the cellular populations and key signaling pathways characterizing a mouse submandibular salivary gland organoid (mSGO) model at different temporal stages and in response to radiation damage. We identified Sox9- and Itgb1-expressing cells as the most primitive adult stem/progenitor populations and uncover novel stemness and migratory roles for Cd44-expressing cells. Moreover, we revealed that Notch signaling is essential for maintaining self-renewal and migration potential of these stem/progenitor cells post-irradiation. Extending these findings to patient-derived mSGOs, as well as murine and patient-derived mammary and thyroid gland organoids, we confirmed Notch signaling as a conserved regulator of stem/progenitor cell function under migrative and regenerative conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77db409946703d732e9f76a695547f2c0dc6d350" target='_blank'>
              Notch Signaling Drives Pro-Regenerative and Migratory Traits in Glandular Stem/Progenitor cells
              </a>
            </td>
          <td>
            D. Cinat, Rufina Maturi, Jeremy P. Gunawan, Anne L Jellema-de Bruin, Laura Kracht, Paola Serrano Martinez, Yi Wu, A. Soto-Gamez, M. van Goethem, Inge R. Holtman, S. Pringle, Lara Barazzuol, Rob P. Coppes
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, Mirjam Judith Feldkamp, C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Glioblastoma, the most aggressive form of primary brain cancer, is driven by both intrinsic cellular properties and extrinsic factors from the tumor microenvironment (TME). Here, we leverage our novel human organoid tumor transplantation (HOTT) system to explore how external cues modulate glioblastoma cell type specification, heterogeneity, and migration. Comparing the single-cell transcriptomes of HOTT and matched primary tumors, we found that HOTT recapitulates not only core features of major patient tumor cell types but also those of peritumor non-neoplastic cell types. This enables us to explore tumor-TME interactions and identify PTPRZ1, a receptor tyrosine phosphatase implicated in brain tumor migration, as a key mediator in intercellular communication. This finding was further confirmed by interrogation of published datasets from patients containing non-neoplastic microenvironmental cells. PTPRZ1 activation, resulting from the reactivation of developmental programs in both the tumor and its TME, highlights the relevance of the organoid system for modeling the GBM microenvironment. Moreover, HOTT uniquely allows for perturbations in either patient tumors or their microenvironments, or both, to verify PTPRZ1 function. Tumor knockdown of PTPRZ1 confirmed its role in promoting tumor migration and maintaining progenitor identity. Surprisingly, environmental PTPRZ1 knockdown in human cortical organoids before tumor transplantation inhibited tumor migration and promoted differentiation, even without direct tumor manipulation. Overall, this fundamental result demonstrates that a single treatment delivered to the tumor and its TME can exert opposite effects on the tumor. Consequently, it underscores the necessity of studying human glioblastoma within a human microenvironment context, such as HOTT, especially when evaluating therapeutic efficacy in GBM, as effective treatments must address both the tumor and its complex TME, rather than the tumor alone.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9ffba54b9658bfd1d4ecc52cd105a03aa1e10dc" target='_blank'>
              TMIC-69. HUMAN ORGANOID TUMOR TRANSPLANTATION IDENTIFIES FUNCTIONAL GLIOBLASTOMA MICROENVIRONMENTAL COMMUNICATION MEDIATED BY PTPRZ1
              </a>
            </td>
          <td>
            Weihong Ge, Ryan L. Kan, Can Yilgor, Elisa Fazzari, Patricia R. Nano, Daria J. Azizad, Matthew Li, Joyce Ito, C. Tse, Andrew Tum, Jessica Scholes, Kunal S. Patel, David Nathanson, A. Bhaduri
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e3304563df16fc68d3eeffdd08dc060638480f4" target='_blank'>
              Single-cell transcriptomic landscape deciphers olfactory neuroblastoma subtypes and intra-tumoral heterogeneity.
              </a>
            </td>
          <td>
            Jingyi Yang, Xiaole Song, Huankang Zhang, Quan Liu, Ruoyan Wei, Luo Guo, Cuncun Yuan, Fu Chen, K. Xue, Yuting Lai, Li Wang, Junfeng Shi, Chengle Zhou, Juan Wang, Yingxuan Yu, Qibing Mei, Li Hu, Huan Wang, Chen Zhang, Qianqian Zhang, Houyong Li, Ye Gu, Weidong Zhao, Huapeng Yu, Jingjing Wang, Zhuo-fu Liu, Han Li, S. Zheng, Juan Liu, Lu Yang, Wanpeng Li, Rui Xu, Jiani Chen, Yumin Zhou, Xian-kui Cheng, Yiqun Yu, Dehui Wang, Xicai Sun, Hongmeng Yu
          </td>
          <td>2024-11-14</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de8ac06c638611539d33f6ef5fdb316eabe9f02c" target='_blank'>
              Spatio-temporal transcriptomics of chromothriptic SHH-medulloblastoma identifies multiple genetic clones that resist treatment and drive relapse
              </a>
            </td>
          <td>
            Ilia Kats, Milena Simovic-Lorenz, Hannah Sophia Schreiber, Pooja Sant, Jan-Philipp Mallm, Verena Körber, Albert Li, Pravin Velmurugan, Sophie Heuer, Luisa Kües, Frauke Devens, M. Sill, M. Jugold, Mahmoud Moustafa, Amir Abdollahi, Frank Winkler, A. Korshunov, Stefan M Pfister, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220a065de9e40d9837ef9539b8c30437e0e009d" target='_blank'>
              Advanced single-cell and spatial analysis with high-multiplex characterization of circulating tumor cells and tumor tissue in prostate cancer: Unveiling resistance mechanisms with the CoDuCo in situ assay
              </a>
            </td>
          <td>
            Lilli Bonstingl, Margret Zinnegger, Katja Sallinger, Karin Pankratz, Christin-Therese Müller, Elisabeth Pritz, Corinna Odar, C. Skofler, Christine Ulz, Lisa Oberauner-Wappis, Anatol Borrás-Cherrier, Višnja Somođi, Ellen Heitzer, T. Kroneis, T. Bauernhofer, Amin El-Heliebi
          </td>
          <td>2024-11-16</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Therapy resistance in glioblastoma is in part attributed to intratumoral heterogeneity and treatment-resistant cell populations. Understanding heterogeneity requires more complete tumor sampling at diagnosis and at recurrence. We used a whole-tumor sampling approach to obtain 43 spatially mapped biopsies from 3 glioblastomas at diagnosis and recurrence, Pyclone and ClonEvol imputed clonal evolution from exome data, and weighted gene co-expression network analysis inferred gene expression programs from RNA sequencing. Across the cohort, tumor-wide clonal alterations representing evolutionarily early expansions included canonical changes (i.e. Chr7 gain, EGFR amplification) and a diverse set of copy number variations (Chr19/20 gain), driver mutations (i.e. PTEN, KDR), and fusions (LIMCH1::UCHL1, KANK::DOCK8). In 2/3 patients, the founding clone was the only subclone common to primary and recurrence samples and 37% of cancer drivers across the cohort appeared after evolutionary divergence of the primary and recurrent tumors. We identified thirty-two transcriptional programs, six of which were differentially expressed between timepoints. Programs enriched for mitochondrial activity, endothelium/pericytes and classical glioblastoma subtype signature were more highly expressed in primary samples. Programs enriched for mesenchymal-like state, oligodendrocytes and inhibitory interneurons were more highly expressed in recurrence samples. At diagnosis, 11/16 samples showed higher expression of the classical subtype program compared to mesenchymal-like, but at recurrence the mesenchymal program dominated in 18/18 samples. Expression of the programs correlated more strongly with position relative to the contrast-enhancing rather than T2 tumor centroid (p<0.001). These findings reveal novel diversity and complexity in the genetic roots of individual glioblastoma. Recurrences arose from subclones diverging early in evolution of the primary tumor and contained clonal drivers not detected in the primary. Expression data revealed transition from an intratumorally heterogeneous mixture of classical and mesenchymal signatures to tumor-wide mesenchymal at recurrence and suggests that the contrast enhancing centroid may serve as the biological center of the tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa1cf111e47e18a468bfde6175297f4bb325c80" target='_blank'>
              EPCO-37. LONGITUDINAL TUMOR-WIDE SAMPLING OF GLIOBLASTOMA REVEALS EARLY EVOLUTIONARY DIVERGENCE OF RECURRENCE AND CLUES TOWARDS IDENTIFYING THE BIOLOGICAL TUMOR CENTER
              </a>
            </td>
          <td>
            Benjamin Lerman, Chibo Hong, Patrick Schupp, Lee Chen, Daniel Zamler, Ivan V. Smirnov, Joanna Phillips, Anny Shai, Janine Lupo, Michael Oldham, Joseph F Costello
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1436facb9b04a543d668f4862e82c8d1f590873" target='_blank'>
              FBXO2 as a switch guides a special fate of tumor clones evolving into a highly malignant transcriptional subtype in oral squamous cell carcinoma.
              </a>
            </td>
          <td>
            Jingyi Cheng, O. Liu, Xin Bin, Zhangui Tang
          </td>
          <td>2024-11-02</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Effective treatment of breast cancer remains a formidable challenge, partly due to our limited understanding of the complex microenvironmental factors that contribute to disease pathology. Among these factors are tissue-resident perivascular cells, which play crucial roles in shaping vascular basement membranes, maintaining vessel integrity, and communicating with adjacent endothelial cells. Despite their essential functions, perivascular cells have been relatively overlooked. Identifying them by immunostaining has been challenging due to their low abundance, inherent heterogeneity, and shared marker expression with other cell types. These challenges have hindered efforts to purify pericytes and generate primary cell models for studying their biology.Using a recently developed FACS method, we successfully identified and purified each cell type from breast tissues, allowing us to deep-sequence their transcriptomes and generate primary cell models of each cell type—including pericytes. Here, we used these data to analyze cell-type-specific gene expression in tumors, which revealed a strong association between pericyte-specific genes and breast cancer patient mortality. To explore this association, we defined the heterogeneity of breast pericytes using single-cell RNA sequencing and identified a broad marker for visualizing perivascular cells in breast tumors.Remarkably, we discovered perivascular cells dissociated from vessels and emerged as a dominant mesenchymal cell type in a subset of breast tumors that contrasted with their normal perivascular location. Moreover, when we purified pericytes from the breast and cultured them alongside breast tumor cells, we discovered that they induced rapid tumor cell growth significantly greater than isogenic fibroblast controls.These findings identify perivascular cells as a key microenvironmental factor in breast cancer, highlighting the critical need for further research to explore their biology and identify specific stimulatory mechanisms that could be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920638149619a80dd215db038826c980f92e7f53" target='_blank'>
              Breast pericytes: a newly identified driver of tumor cell proliferation
              </a>
            </td>
          <td>
            Katelyn Del Toro, Y. Licon-Munoz, William Crabtree, Tristan Oper, Christine Robbins, William C. Hines
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Breast tumors comprise several sub-populations of cells, so-called “clones”, with specific cellular and molecular characteristics. Some cancer clones can be more ‘aggressive’ than others in particular tumor microenvironments (TMEs), contributing to metastatic progression and drug resistance. Understanding the specific genes/pathways deregulated in these aggressive cancer cells is essential for developing new tailored treatment strategies. To accomplish this, we optimized the use of a lentiviral-based barcoding technique (Berthelet, J., et.al., Sci Adv 2021, Serrano, A., et.al., Nat Rev Cancer 2022) where cancer cells from cell lines and patient-derived xenografts are tagged with optical barcodes and transplanted into the mammary fat pads of immunodeficient mice. Upon resection of tumors and collection of metastatic organs at the experimental endpoint, we analyzed clonal fate and fitness using flow cytometry and imaging. The subsequent molecular characterization of individual clones was determined by bulk and single-cell RNA sequencing. We also leverage the use of these barcoded models to investigate the mechanisms of innate and acquired resistance to Sacituzumab Govitecan (SG), a drug recently approved for the treatment of patients with advanced breast cancer. Our studies revealed unique gene signatures and pathways associated with preferential organ tropism (in the lungs, liver and brain TMEs) and SG resistance. This comprehensive analysis, currently validated using autopsy samples and patient databases, will support the design of new predictive markers and targeted therapies for metastatic breast cancer.
 Citation Format: Sreeja Reddy Gadipally, Jean Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, Caroline Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, Charafe Jauffret Emmanuelle, Melissa Davis, Robin L. Anderson, Luciano Martelotto, Belinda Yeo, Delphine Merino. Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528d6539434045b291c3a0faa67fa6d3c29fbc80" target='_blank'>
              Abstract B005: Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance
              </a>
            </td>
          <td>
            Sreeja Gadipally, J. Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, C. Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, C. Emmanuelle, Melissa J Davis, Robin L Anderson, Luciano Martelotto, B. Yeo, Delphine Merino
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Tumor microenvironments (TMEs) encompass complex ecosystems of cancer cells, infiltrating immune cells, and diverse cell types. Intercellular and intracellular signals within the TME significantly influence cancer progression and therapeutic outcomes. Although computational tools are available to study TME interactions, explicitly modeling tumor progression across different cancer types remains a challenge. Methods This study introduces a comprehensive framework utilizing single-cell RNA sequencing (scRNA-seq) data within a multilayer network model, designed to investigate molecular changes across glioma progression stages. The heterogeneous, multilayered network model replicates the hierarchical structure of biological systems, from genetic building blocks to cellular functions and phenotypic manifestations. Results Applying this framework to glioma scRNA-seq data allowed complex network analysis of different cancer stages, revealing significant ligand‒receptor interactions and key ligand‒receptor-transcription factor (TF) axes, along with their associated biological pathways. Differential network analysis between grade III and grade IV glioma highlighted the most critical nodes and edges involved in interaction rewiring. Pathway enrichment analysis identified four essential genes—PDGFA (ligand), PDGFRA (receptor), CREB1 (TF), and PLAT (target gene)—involved in the Receptor Tyrosine Kinases (RTK) signaling pathway, which plays a pivotal role in glioma progression from grade III to grade IV. Discussion These genes emerged as significant features for machine learning in predicting glioma progression stages, achieving 87% accuracy and 93% AUC in a 3-year survival prediction through Kaplan-Meier analysis. This framework provides deeper insights into the cellular machinery of glioma, revealing key molecular relationships that may inform prognosis and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d112e50918401623433091b937b2d39ad5c4da" target='_blank'>
              Integrative analysis of single-cell transcriptomic and multilayer signaling networks in glioma reveal tumor progression stage
              </a>
            </td>
          <td>
            Fereshteh Fallah Atanaki, L. Mirsadeghi, Mohsen Riahi Manesh, K. Kavousi
          </td>
          <td>2024-11-13</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains a highly malignant, intrinsically resistant and inevitably recurring brain tumor with dismal prognosis. The aggressiveness and lack of effective GBM treatments can be attributed to the highly heterogeneous and plastic nature of GBM tumor cells, which easily confer resistance to standard-of-care (SOC) therapy. While tumor progression has also been attributed to interactions with the tumor microenvironment, quantitative data describing these interactions are still largely missing. Here, we used an multiome approach combining high-dimensional, multiplexed immunohistochemistry and spatial transcriptomics to map evolutions in the spatial, single-cell tissue architecture of 357 paired adult GBM tumor samples derived from 52 patients at diagnosis (ND) and upon recurrence (REC) following SOC treatment. We mapped the spatial distribution of a multitude of GBM tumoral subtypes across this multicentric cohort, through which we identified a high level of heterogeneity defined by specific tumoral niches within and across patients and which evolved when subjected to SOC therapy. In addition, we describe the relationship of the various tumoral niches with their local immune-infiltrates, highlighting an even more immunosuppressive environment following SOC resistance. Finally, by aligning these findings to the observed genomic aberrations and the clinical data of the patients, we are now able to more precisely describe the heterogeneous landscape of glioblastoma and how it evolves under SOC treatment at spatial, single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27f408cb60e0b4c4a336bb204f39691c84c2de06" target='_blank'>
              TMIC-55. CHARACTERIZATION OF THERAPEUTICS EFFECTS IN GLIOBLASTOMA THROUGH INTEGRATED SPATIAL SINGLE-CELL MULTIOME PROFILING OF LONGITUDINAL SAMPLES: UNVEILING CLINICALLY RELEVANT SUBTYPES OF PATHOLOGICAL CHANGES
              </a>
            </td>
          <td>
            M. Vanmechelen, Nazari Pouya, J. Beckervordersandforth, D. Leunissen, B. Cole, B. Decraene, Y. De Visser, Gautam Shankar, M. Verduin, Carmen Bravo, Tom Moors, Jayesh Telang, J. Messiaen, Danielle Eekers, A. Claeys, M. Derweduwe, A. Zurhausen, Francesca M Bosisio, Frank Weyns, T. Daenekindt, Peter Van Eyken, Mieke Govers, Ferdinand Mennens, K. Hovinga, S. De Vleeschouwer, Clement Paul, M. Broen, Marc Vooijs, R. Sciot, A. Antoranz, Jon Pey, Ernst Jan Speel, A. Hoeben, F. De Smet
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="

 Multiple Myeloma (MM) is a genomically heterogeneous cancer of malignant plasma cells (PC) residing in the bone marrow. Despite recent therapeutic advances, it is incompletely understood why some patient populations remain largely therapy-refractory or relapse, even with targeted immunotherapies. Whilst MM classification based on genomic features and gene expression profiling (GEP) is well-described, the role of spatial interactions between tumour cells and the bone marrow tumour microenvironment (TME) in dictating treatment responses is insufficiently defined. Spatial heterogeneity in MM was highlighted by a multi-region whole exome sequencing (WES) study of paired samples from random bone marrow aspirates and distant lesions (Rasche et al Nature Comm 2017, 2022; doi.org/10.1038/s41467-017-00296-y; 10.1038/s41467-022-32145-y), however, it is unknown whether spatially and genomically distant clones interact differently within their immune and stromal compartments. Recent pioneering work, using multiplexed imaging and in situ transcriptomics approaches on MM trephines (Robinson et al, JCI 2023; doi.org/10.1172/JCI167629) demonstrated patterns and mechanisms of tumour-immune cell interactions suggesting TME interactions as a critical therapeutic response determinant.
 Isolated in situ ‘omics’ assays provide limited information due to biased selection of marker panels or discordance between transcriptional and proteomic profiles. We hypothesize that a combined, systematic approach to integrate multiple layers of information (multiplexed imaging, unbiased LC/MS proteomics, transcriptomics, genomics) within their spatial context would significantly advance our understanding of MM TME phenotypes and responses to therapy.
 Consecutive FFPE sections (plasmacytoma or MM; 4 cases) were subjected to spatial transcriptomics (Xenium, 10X Genomics) combined with multiplexed immunofluorescence (IF) or imaging mass cytometry (IMC), laser-capture guided in situ WGS (Rao et al, Genome Med. 2024 doi: 10.1186/s13073-024-01302-x) and LC/MS based spatial proteomic workflows (Davis et al, Nat Commun. 2023 doi: 10.1038/s41467-023-43520-8). A data analysis pipeline utilising machine-learning was implemented. We have developed a multiomics workflow that (1) maps critical regions of interest (ROI) in the TME (using spatial transcriptomics, IHC, IF) and that provides (2) ROI-specific genomic information from WES and/or targeted sequencing, and (3) unbiased proteomic information (LC/MS).
 We mapped tumour, immune and stromal cell types and activation states in the bone marrow environment by using low-plex (30-40 markers) IF or IMC, and a customised probe panel for transcript detection in order to define ROIs. This was followed by laser capture microscopy of defined ROIs to genomically characterise the tumour, with ROI-matched LC/MS proteomics enabled detection of 4000-5000 proteins (current resolution of 200-500 cells per ROI).
 In this proof of concept study with limited sample numbers we identified differences in tumour MHC expression as well as novel NK-cell-myeloma spatial interactions (such as HLA-G/LILRB1) as factors that may control myeloma-immune environments and responses. Shallow WGS of excised ROI sections allows detection of CNA-defined clonal heterogeneity; we are now developing this promising approach to a targeted panel of genomic aberrations in MM. Our next translational step is the deployment of this workflow to longitudinal clinical trial trephine samples with correlation to single cell data from bone marrow aspirates. Taken together, application of this translational platform will provide new insights into MM pathobiology and therapeutic response and resistance mechanisms.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc52b4c61e537e710ecf579d97f6980a75244b" target='_blank'>
              Development of a Spatial Multiomics Platform to Integrate Genomic, Transcriptomic and Proteomic Features for Translational Research in Multiple Myeloma
              </a>
            </td>
          <td>
            S. Gooding, Chen-Yi Wang, Warren Baker, Edmund Watson, Simon Davis, Roman Fischer, Oliver McCallion, J. Hester, Fadi Issa, Srinivasa Rao, Nick Athanasou, A. Thakurta, K. Ramasamy, C. Bjorklund, M. Amatangelo, Anita K. Gandhi, A. Cribbs, A. Laganà, Udo Oppermann
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f233ae172c7106f684ef62640304f7d51d73186" target='_blank'>
              Integrative mapping of human CD8+ T cells in inflammation and cancer.
              </a>
            </td>
          <td>
            Ziwei Xue, Lize Wu, Ruonan Tian, Bing Gao, Yu Zhao, Bing He, Di Sun, Bingkang Zhao, Yicheng Li, Kaixiang Zhu, Lie Wang, Jianhua Yao, Wanlu Liu, Linrong Lu
          </td>
          <td>2024-11-29</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Background & Objective: Prostate cancer (PCa) cells frequently disseminate to bone where they can remain dormant for extended periods. The precise cellular and molecular mechanisms responsible remain largely unknown. The advent of spatial transcriptomics provides a unique opportunity to study key bone microenvironment-dormant PCa cell interactions in situ. Here, we developed a novel immunocompetent in vivo model of PCa dormancy in bone and employed the technology to address this major gap in knowledge. Methods: We established an in vitro protocol to induce dormancy in multiple PCa cell lines, including the C57BL/6 murine PCa cell line, RM1. Isogenic active and dormant PCA cells underwent bulk RNA Seq. We used an intra- iliac artery (IIA) approach to allow niche-specific seeding of dormant PCa within the bone. Spatial transcriptomics (10X Visium) was performed on tissue sections and R packages (Seurat & CellChat) were used for data analysis. Results: We validated in vitro PCa dormancy via cell cycle analysis, membrane dye retention and ERK/p38 ratio. RNASeq analysis of dormant PCa cells (vs. active) revealed upregulation of immunomodulatory genes such as, GDF15, BDNF, A2M, VGF, NLRP1, and CD55. C57BL/6 immunocompetent mice were inoculated with active or dormant RM1 cells via IIA. Majority of active RM1 mice reached endpoint within 15 days. In contrast, mice bearing dormant RM1 (D-RM1) had extended survival and were removed from study after 30 days. Histological analysis revealed solitary D-RM1 (panCK+/Ki67-/CC3-) on endosteal surfaces compared to macrometastases (high Ki67+) in the active RM1 group. Using IIA PBS injected tumor naïve bones as a control, we applied spatial transcriptomics (10X Visium) to interrogate the microenvironment surrounding active and dormant RM1 PCa cells. Bioinformatic analyses demonstrated higher expression of B cell markers (CD19, CD79a, CD79b, Ighm, and Pax5) and enhanced B cell activation/antigen phagocytosis signature in the D- RM1 group compared to other groups. We validated the presence of CD19 B-cells in proximity to the D-RM1 cells in subsequent tissue sections. In contrast, B cells were located less frequently and to the periphery of active RM1 metastases. Inference analysis using CellChat also identified a significant and distinct signaling pattern between CD55 and CD97 in the B cell enriched clusters in D-RM1 group. CD55/CD97 interaction is a noted mediator of B cell homing. These data suggest a potential role for distinct populations of B cells during the establishment of PCa dormancy in bone. Conclusion: Spatial transcriptomic analysis of the dormant PCa bone microenvironment reveals an enrichment of CD19+ B cells with a distinct signaling pattern. Given their known roles in immune tolerance we posit that infiltrating B cells may play a role in protecting and sustaining newly disseminated dormant PCa cells from immune mediated elimination.
 Citation Format: Mostafa M. Nasr, Ryan T. Bishop, Haley du Bois, Tao Li, Jeremy S. Frieling, Conor C. Lynch. Spatial transcriptomics reveals a distinct B cell signature in the dormant prostate cancer bone microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1da73f9c20af7ebfb3e86b110c704de45b40046e" target='_blank'>
              Abstract B044: Spatial transcriptomics reveals a distinct B cell signature in the dormant prostate cancer bone microenvironment
              </a>
            </td>
          <td>
            Mostafa M. Nasr, Ryan T Bishop, Haley du Bois, Tao Li, Jeremy S. Frieling, Conor C Lynch
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary Cancer initiation and progression are controlled not only by intrinsic factors, such as genetic mutations and mesenchymal trait acquisition, but also by dynamic spatiotemporal shifts within the tumor microenvironment. These elements induce plasticity, and enhance invasiveness, metastasis, and treatment resistance, while promoting immune evasion through the reprogramming of the immune microenvironment. However, the molecular mechanisms driving this reprogramming remain unclear. In this review, we summarize the recent pivotal findings on pancreatic cancer, which is a highly malignant disease, and overview the roles of plasticity, metabolic reprogramming, and the creation of an immunosuppressive microenvironment in disease progression. We also present our research on two crucial signaling pathways, i.e., the dynamic interplay between mutational load and the tumor microenvironment, and the acquisition of cancer plasticity via inflammatory responses and metabolic reprogramming. A comprehensive understanding of these processes is expected to transform therapeutic strategies and improve the outcomes of pancreatic cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d6b2b31d56874111eca289fe4cec7a0df598ab" target='_blank'>
              Plasticity and Tumor Microenvironment in Pancreatic Cancer: Genetic, Metabolic, and Immune Perspectives
              </a>
            </td>
          <td>
            A. Hashimoto, S. Hashimoto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Improving the selectivity and effectiveness of drugs represents a crucial issue for future therapeutic developments in immuno-oncology. Traditional bulk transcriptomics faces limitations in this context for the early phase of target discovery as resulting gene expression levels represent the average measure from multiple cell populations. Alternatively, single cell RNA sequencing can dive into unique cell populations transcriptome, facilitating the identification of specific targets. Here, we generated Tumor-Infiltrating regulatory T cells (TI-Tregs) and exhausted T cells (Tex) gene signatures from a single cell RNA-seq pan-cancer T cell atlas. To overcome noise and sparsity inherent to single cell transcriptomics, we then propagated the gene signatures by diffusion in a protein-protein interaction network using the Patrimony high-throughput computing platform. This methodology enabled the refining of signatures by rescoring genes based on their biological connectivity and shed light not only on processes characteristics of TI-Treg and Tex development and functions but also on their immunometabolic specificities. The combined use of single cell transcriptomics and network propagation may thus represent an innovative and effective methodology for the characterization of cell populations of interest and eventually the development of new therapeutic strategies in immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/060798152c5e47a1cd66c8ffdb21f2a43f037a14" target='_blank'>
              The combined use of scRNA-seq and network propagation highlights key features of pan-cancer Tumor-Infiltrating T cells
              </a>
            </td>
          <td>
            Adèle Mangelinck, Elodie Molitor, Ibtissam Marchiq, Lamine Alaoui, Matthieu Bouaziz, Renan Andrade-Pereira, Hélène Darville, Etienne Becht, Céline Lefebvre
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa9e49413db565400c66d264638af2b81a7c480" target='_blank'>
              Single cell-spatial transcriptomics and bulk multi-omics analysis of heterogeneity and ecosystems in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jiazhou Ye, Yan Lin, Zhiling Liao, Xing Gao, Cheng Lu, Lu Lu, Julu Huang, Xi Huang, Shilin Huang, Hongping Yu, Tao Bai, Jie Chen, Xiaobo Wang, Mingzhi Xie, Min Luo, Jinyan Zhang, Feixiang Wu, Guo-bin Wu, Liang Ma, Bangde Xiang, Lequn Li, Yongqiang Li, Xiaoling Luo, R. Liang
          </td>
          <td>2024-11-15</td>
          <td>NPJ Precision Oncology</td>
          <td>2</td>
          <td>19</td>
        </tr>

        <tr id="Despite thorough characterizations of cellular compositions within the breast tumor microenvironment (TME), their implications for disease progression and patient prognosis are still poorly understood. Unraveling these effects is vital for identifying potential targets to improve treatment outcomes. In this study, we devised an explainable machine learning (XML) pipeline to scrutinize the associations between TME cellular constituents and relapse-free survival (RFS). By applying our pipeline to estimated cell fractions in the METABRIC and TCGA datasets and comparing these results with associations to pathological complete response (pCR) after neoadjuvant chemotherapy (NAC), we created a comprehensive catalog of the TME’s role based on 5000 patient samples. Our findings reveal an unexpected dichotomy in which macrophages correlate positively with pCR but negatively with RFS, particularly within estrogen receptor-positive (ER+) and Luminal A and B (LumA/B) cancer subtypes. We show that this pattern is driven by heterogeneity in breast tumors characterized by increasing levels of macrophage infiltration. Through imaging mass cytometry (IMC) analysis, we discovered that macrophages tend to accumulate in the vicinity of HLA-ABChi epithelial cells as their frequency increases in tumor tissues and also express elevated levels of HLA-ABC protein. Combining IMC with single-cell RNA sequencing (scRNA-seq) data, we uncovered a significant association between these HLA-ABChi macrophages and regulatory and exhausted T cells (TReg and TEx), suggesting their involvement in immune suppression, likely by creating a chronically activated immunosuppressive TME. Subsequent cell-cell communication analysis predicted interactions between HLA-ABChi macrophages and TEx cells via the ligands SIGLEC9, ALCAM, and CSF1, and with TReg cells through APP, ANGPTL4, and SIGLEC9 signaling. Considering the clinical relevance of macrophages in ER+ (LumA/B) subtypes, our research enhances the characterization of macrophage-driven immune suppression in these tumors and identifies potential targets for immunomodulatory strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ef5aa61ac57e0a39d6c5657cd0986b7fc829d8" target='_blank'>
              Integrated multiomics analysis unveils how macrophages drive immune suppression in breast tumors and affect clinical outcomes
              </a>
            </td>
          <td>
            Y. Azimzade, Mads Haugland Haugen, Vessela N Kristensen, A. Frigessi, Alvaro Köhn-Luque
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69505c7dd30d328b855b8e26bb2a91fa4c9f9efa" target='_blank'>
              Computer-aided analysis reveals metallothionein-positive cancer-associated fibroblasts promote angiogenesis in gastric adenocarcinoma
              </a>
            </td>
          <td>
            Xiaolong Jin, Yu Tian, Haoran Zhu, Yuewen Sun, Zhenxing Zhang
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Adult-type diffuse gliomas present significant treatment challenges due to their complex cellular composition and evolving nature. Understanding this heterogeneity is critical for the development of effective, targeted therapies. We conducted single-cell and single-nucleus RNA sequencing on a unique cohort of 66 tumours (single surgery) and 54 paired samples (two surgeries). This comprehensive dataset of approximately 900,000 cells allows us to dissect conserved cellular landscapes and dynamic evolution across diverse glioma subtypes. Using a novel phylogeny-based approach, we identified an astrocyte-like glioma stem cell (GSC) as the root of diverse glioma lineages. This discovery transforms our understanding of glioma development, pinpointing the GSC as a potential therapeutic target. The GSC’s similarity to healthy adult neural stem cells provides clues as to the origin of glioma heterogeneity and the shared basis of intratumoural diversity. Our analysis uncovered a shared cellular architecture across astrocytomas, oligodendrogliomas, and glioblastomas (GBMs). This architecture encompasses seven recurring malignant cell states: neuro-, oligo-, neuro-oligo-lineage, cycling (G1/S, G2/M), hypoxia-associated, and a quiescent astrocyte-like GSC. Importantly, these cell state proportions vary significantly between glioma subtypes. Additionally, we demonstrate that GBMs display substantially higher T-cell and myeloid cell infiltration than IDH-mutant gliomas. Counterintuitively, this correlates with a poorer prognosis, implying profound T-cell dysfunction and intricate myeloid cell-tumour interactions. Longitudinal analysis further highlights the dynamic nature of glioma cell populations and the immune landscape. Subtype-specific T-cell and myeloid gene expression profiles were identified, including potential targets such as PARD6G (GBM) and CXCL13 (IDH-mutant-astrocytoma), laying the groundwork for tailored immunotherapies and prognostic biomarkers. This study presents a comprehensive model of glioma heterogeneity and evolution. The model reveals a shared astrocyte-like GSC that fuels cellular diversity, coupled with distinct, dynamic immune landscapes in different glioma subtypes. These findings hold extraordinary potential for developing transformative treatment strategies. Future research functionally validating the GSC and its impact on tumour dynamics will be essential for translating these insights into improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a47871bd911080cf561a5ed294b5aa4ee2716f4" target='_blank'>
              STEM-01. SHARED ASTROCYTE-LIKE GLIOMA STEM CELLS DRIVE HETEROGENEITY AND IMMUNE DYNAMICS IN ADULT-TYPE DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Zaman, W. Muskovic, E. Holton, M. Pandya, Rachael Zekanovic, Venessa T Chin, E. Lam, Catia Moutinho, J. Yeung, C. Teo, M. Sughrue, J. Powell
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) resists T-cell killing by developing an immunosuppressive tumor microenvironment. Transcriptomic technologies, such as single-cell RNA sequencing (scRNA-seq), have characterized immune cell phenotypes and putative cell-cell interactions in GBM. However, the tumor microenvironment surrounding T-cells is still not well characterized, motivating the need for spatial analyses. Two publicly-available GBM spatial transcriptomic datasets were analyzed, with one discovery cohort (N=18) and one validation cohort (N=12). Using cell deconvolution with a scRNA-seq reference dataset, areas of high T-cell infiltration were identified and clustered into regional programs based on transcriptomic signatures. To highlight which ligand-receptor interactions are most prominent in these regional programs, a novel spatial enrichment algorithm was applied. The impact of spatially enriched ligand-receptor interactions on overall survival was evaluated using bulk RNA-sequencing data from The Cancer Genome Atlas (TCGA). Three distinct T-cell regional programs (transitional, immunosuppressive, and infiltrative) were found across both the discovery and validation cohort. The infiltrative T-cell regional program resided on the periphery of the tumor with a neural progenitor phenotype (NRGN, VSNL1, SLC17A7). In contrast, the immunosuppressive T-cell region had a higher proportion of malignant cells and a greater mesenchymal tumor signature (ANXA1, CD44, complement pathway). Furthermore, ligand-receptor enrichment analysis showed that the immunosuppressive region had higher tumor and tumor-associated macrophage (TAM) signaling towards CD4 T-cells along the collagen, midkine, and osteopontin pathways. TCGA survival analysis showed that higher collagen and midkine signals were associated with worse overall and disease free survival. Our results identified three distinct regional programs in GBM and highlight a set of signaling pathways associated with T-cell suppression. Our findings suggest that the disruption of these pathways could reduce immunosuppressive signals from tumor and TAMs to T-cells. Subsequent validation with spatial proteomics data in glioblastoma is ongoing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed824dde4f1a4936f2335e4d52e940b75bae961" target='_blank'>
              EPCO-46. SPATIAL TRANSCRIPTOMICS ANALYSIS OF GLIOBLASTOMA REVEALS THREE DISTINCT REGIONAL PROGRAMS OF T-CELL INFILTRATION
              </a>
            </td>
          <td>
            William L Wang, Yiyun Chen, Chiu Ho Mo, Zinaida Good, E. Sotillo, Crystal Mackall
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="In colorectal cancer (CRC), tumours classifier as consensus molecular subtype 4 (CMS4) have the worst prognosis and derive negligible benefit from chemotherapy. We previously described how repressed interferon-related signalling is associated with increased relapse in CMS4 tumours. Although the viral mimetic poly(I:C) can reduce liver metastasis in vivo, the initial phenotypic changes that underpin its anti-metastatic response remain poorly described, particularly in the immunosuppressed CMS4 tumour microenvironment. Here we characterise lineage-specific anti-metastatic responses induced by poly(I:C), including acute macrophage polarisation and a novel CMS1-like regenerative stem cell state, which drive pro-inflammatory microenvironmental changes in CRC. These insights enabled the development of tractable biomarkers that identify an “immune-warm” patient subset most likely to respond to poly(I:C), enriched for mismatch-repair proficient (pMMR), anti-inflammatory macrophages and CMS4-like features. The viral mimetic poly(I:C) offers a tailored treatment option for CMS4 tumours, by reprogramming stem cell states and activation of an innate-adaptive anti-metastatic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad01d9670e68262ed711098972f38f61c2c5eb48" target='_blank'>
              Viral mimicry redirects immunosuppressed colorectal tumour landscapes towards a proinflammatory and CMS1-like regenerative state
              </a>
            </td>
          <td>
            Shania M. Corry, Svetlana Sakhnevych, Noha Ehssan Mohamed, Sudhir B. Malla, R. Byrne, Andrew Young, R. Amirkhah, Courtney Bull, Andrea Lees, Keara Redmond, T. Lannagan, Rachel Ridgway, Fiona R Taggart, Natalie C Fisher, Tim Maughan, Mark Lawler, Andrew Campbell, Simon J Leedham, Aideen E Ryan, Dan Longley, Donna Small, Owen J Sansom, Philip D Dunne
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Muscle-invasive bladder cancer (MIBC) is a prevalent cancer characterized by molecular and clinical heterogeneity. Assessing the spatial heterogeneity of the MIBC microenvironment is crucial to understand its clinical significance.In this study, we used imaging mass cytometry (IMC) to assess the spatial heterogeneity of MIBC microenvironment across 185 regions of interest in 40 tissue samples. We focused on three primary parameters: tumor (T), leading-edge (L), and nontumor (N). Cell gating was performed using the Cytobank platform. We calculated the Euclidean distances between cells to determine cellular interactions and performed single-cell RNA sequencing (scRNA-seq) to explore the molecular characteristics and mechanisms underlying specific fibroblast (FB) clusters. scRNA-seq combined with spatial transcriptomics (ST) facilitated the identification of ligand–receptor (L–R) pairs that mediate interactions between specific FB clusters and endothelial cells. Machine learning algorithms were used to construct a prognostic gene signature.The microenvironments in the N, L, and T regions of MIBC exhibited spatial heterogeneity and regional diversity in their components. A distinct FB cluster located in the L region—identified as S3—is strongly associated with poor prognosis. IMC analyses demonstrated a close spatial association between S3 and endothelial cells, with S3-positive tumors exhibiting increased blood vessel density and altered vascular morphology. The expression of vascular endothelial growth factor receptor and active vascular sprouting were significant in S3-positive tumors. scRNA-seq and ST analyses indicated that the genes upregulated in S3 were associated with angiogenesis. NOTCH1–JAG2 signaling pathway was identified as a significant L–R pair specific to S3 and endothelial cell interactions. Further analysis indicated that YAP1 was a potential regulator of S3. Machine learning algorithms and Gene Set Variation Analysis were used to establish an S3-related gene signature that was associated with the poor prognosis of tumors including MIBC, mesothelioma, glioblastoma multiforme, lower-grade glioma, stomach adenocarcinoma, uveal melanoma, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, and lung squamous cell carcinoma.We assessed the spatial landscape of the MIBC microenvironment and revealed a specific FB cluster with prognostic potential. These findings offer novel insights into the spatial heterogeneity of the MIBC microenvironment and highlight its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fcfdce6314523f58922cfd8b6a6e220595a1f7" target='_blank'>
              Spatially-resolved analyses of muscle invasive bladder cancer microenvironment unveil a distinct fibroblast cluster associated with prognosis
              </a>
            </td>
          <td>
            Chao Feng, Yaobang Wang, Wuyue Song, Tao Liu, Han Mo, Hui Liu, Shulin Wu, Zezu Qin, Zhenxing Wang, Yuting Tao, Liangyu He, Shaomei Tang, Yuanliang Xie, Qiuyan Wang, Tianyu Li
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Lung cancer is a deadly disease with the highest rates of mortality. Over recent decades, a better understanding of the biological mechanisms implicated in its pathogenesis has led to the development of targeted therapies and immunotherapy, resulting in improvements in patient outcomes. To better understand lung cancer tumor biology and advance towards precision oncology, a comprehensive tumor profile is necessary. In recent years, novel in situ spatial multiomics approaches have emerged offering a more detailed view of the spatial location of tumor and tumor microenvironment cells, identifying their unique composition and functional status. In this sense, novel multiomics platforms have been developed to evaluate tumor heterogeneity, gene expression, metabolic reprogramming, signaling pathway activation, cell–cell interactions, and immune cell programs. In lung cancer research, several studies have used these spatial technologies to locate cells and associated them with histological features that are relevant to the pathogenesis of lung adenocarcinoma. These advancements may unveil further molecular and immune mechanisms in tumor biology that will lead to the discovery of biomarkers for treatment prediction and prognosis. In this review, we provide an overview of more widely used and emerging pathology-based approaches for spatial immune profiling in lung cancer and how they enhance our understanding of tumor biology and immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e58f728c302c9eea20cd5a44be684287b92352" target='_blank'>
              Novel Spatial Approaches to Dissect the Lung Cancer Immune Microenvironment
              </a>
            </td>
          <td>
            Idania Lubo, Sharia D Hernandez, I. Wistuba, L. S. Solis Soto
          </td>
          <td>2024-12-12</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit enhanced resistance to standard of care treatments and have significantly worse overall survival (OS) compared to patients with classical subtype enriched tumors. It is important to develop genomic resources, enabling identification of novel putative targets in a statistically rigorous manner.


EXPERIMENTAL DESIGN
We compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 patient samples, aggregated from publicly available raw data. We mapped cell-type specific scRNAseq gene signatures in bulk RNAseq (n=744) and spatial transcriptomics (ST) (n=22) and performed validation using multiplex immunostaining.


RESULTS
Analysis of tumor cells from our scRNAseq atlas revealed nine distinct populations, two of which aligned with the basal subtype, correlating with worse OS in bulk RNAseq. Deconvolution identified one of the basal populations to be the predominant tumor subtype in non-dissociated ST tissues and in vitro tumor cell and patient-derived organoid lines. We discovered a novel enrichment and spatial association of CXCL10+ cancer associated fibroblasts (CAFs) with basal tumor cells. We identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, suggesting a relationship between these cell types in PDAC tumor microenvironment.


CONCLUSIONS
We show that our scRNAseq atlas (700,000 cells), integrated with ST data, has increased statistical power and is a powerful resource, allowing for expansion of current subtyping paradigms in PDAC. We uncovered a novel signaling niche marked by CXCL10+ CAFs and basal tumor cells that could be explored for future targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07269a9cb871b60ea57ca5b2a01a49e35b66cc9b" target='_blank'>
              Human pancreatic cancer single cell atlas reveals association of CXCL10+ fibroblasts and basal subtype tumor cells.
              </a>
            </td>
          <td>
            Ian M Loveless, Samantha B. Kemp, Kailee M Hartway, Jacob T Mitchell, Yuesong Wu, Samuel D Zwernik, D. Salas-Escabillas, Sydney Brender, Madison George, Yetunde Makinwa, Thais Stockdale, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombwell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, E. Fertig, L. T. Kagohara, Brian Theisen, Howard C Crawford, Nina G. Steele
          </td>
          <td>2024-12-05</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Immune checkpoint inhibitors have revolutionized cancer therapy by harnessing the body’s immune system to eliminate tumor cells. However, only a subset of patients responds to treatment. Understanding why only some patients respond remains a critical challenge in cancer research due to the complexity and variability of cellular composition within the tumor microenvironment. Our study presents PRECISE (Predicting therapy Response through Extraction of Cells and genes from Immune Single-cell Expression data), a pipeline that leverages single-cell RNA sequencing data and machine learning techniques to predict ICI responses, while maintaining the richness of single-cell information and ensuring interpretability of the results. Results The PRECISE pipeline implements gene and cell-filtering approaches for optimizing treatment prediction. Utilizing the XGBoost algorithm for predicting patient response to ICI on a dataset of melanoma-infiltrated immune cell achieved an initial Area Under the Curve (AUC) of 0.84. This signal is further improved to 0.89 after Boruta feature selection, revealing an 11-gene predictive signature. Investigation of these genes through SHAP values identified various gene-pair interactions with non-linear conditional effects on predictions. Furthermore, a novel reinforcement learning framework implemented in PRECISE reveals non-predictive single cells that detracts the model’s performance. Altogether, the identified gene- and cell-based signatures demonstrates high prediction power across independent datasets, including lung, breast, brain, and skin cancers. Conclusions Our approach demonstrates the potential of supervised machine learning and reinforcement learning to enhance the understanding of cancer immunity and improve the prediction of treatment responses using single-cell data. Understanding patients’ response to immune checkpoint inhibitors (ICIs) is a critical challenge in cancer research, given the complexity and variability of immune interactions within the tumor microenvironment. Our study leverages single-cell RNA sequencing data and aims to utilize machine learning techniques to predict immunotherapy responses, while maintaining the richness of single-cell information and ensuring interpretability of the results. Using a dataset of melanoma immune cells, we conducted thorough preprocessing and applied the XGBoost algorithm in a leave-one-out cross-validation fashion to predict sample response. By labeling cells according to their sample’s response and aggregating predictions, we achieved an initial AUC score of 0.84. This score was improved to 0.89 with the application of Boruta feature selection, which identified key predictive genes, leading to an 11-gene predictive signature. Further analysis pinpointed T cell clusters as significant contributors to immune response. Utilizing SHAP values provided deeper insights into gene behaviors, interactions, and their effects on the model’s predictions. Additionally, a novel reinforcement learning model was developed for single-cell level prediction and characterization, allowing us to identify and analyze the most predictive cells for response and non-response. Our approach demonstrates the potential of sophisticated computational methods to enhance our understanding of cancer immunity and improve the prediction of treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdff0e265c13d348620c08428ea6a43d804a4e06" target='_blank'>
              Uncovering Predictive Gene and Cellular Signatures for Checkpoint Immunotherapy Response through Machine Learning Analysis of Immune Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a rare malignancy disproportionately affecting adolescents and young adults with no standard of care. FLC is characterized by thick stroma, which has long suggested an important role of the tumor microenvironment. Over the past decade, several studies have revealed aberrant markers and pathways in FLC. However, a significant drawback of these efforts is that they were conducted on bulk tumor samples. Consequently, identities and roles of distinct cell types within the tumor milieu, and the patterns of intercellular communication, have yet to be explored. In this study we unveil cell-type specific gene signatures, transcription factor networks, and super-enhancers in FLC using a multi-omics strategy that leverages both single-nucleus ATAC-seq and single-nucleus RNA-seq. We also infer completely rewired cell-to-cell communication patterns in FLC including signaling mediated by SPP1-CD44, MIF-ACKR3, GDF15-TGFBR2, and FGF7-FGFR. Finally, we validate findings with loss-of-function studies in several models including patient tissue slices, identifying vulnerabilities that merit further investigation as candidate therapeutic targets in FLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5969e5a104cf835bc8976096535f834d7534207" target='_blank'>
              Single-cell multi-omic analysis of fibrolamellar carcinoma reveals rewired cell-to-cell communication patterns and unique vulnerabilities
              </a>
            </td>
          <td>
            Alaa R. Farghli, Marina Chan, Marc S. Sherman, L. K. Dickerson, Bo Shui, M. Nukaya, Andreas Stephanou, Rosanna K. Ma, Brian J Pepe-Mooney, Colton J. Smith, Donald Long, Paul R Munn, Adrian McNairn, Jennifer K. Grenier, Michael Karski, Sean M. Ronnekleiv-Kelly, Venu G. Pillarisetty, Wolfram Goessling, Taranjit S. Gujral, K. Vakili, Praveen Sethupathy
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16bb02acb578a7b253f87e6955e8a12a157c8c15" target='_blank'>
              Glioma immune microenvironment composition calculator (GIMiCC): a method of estimating the proportions of eighteen cell types from DNA methylation microarray data
              </a>
            </td>
          <td>
            S. Pike, J. Wiencke, Ze Zhang, Annette M Molinaro, H. Hansen, D. Koestler, Brock C. Christensen, Karl T. Kelsey, Lucas A Salas
          </td>
          <td>2024-10-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Reduced therapy response in breast cancer has been correlated with heterogeneity in biomarker composition, expression level, and spatial distribution of cancer cells within a patient tumor. Thus, there is a need for models to replicate cell-cell, cell-stromal, and cell-microenvironment interactions during cancer progression. Traditional two-dimensional (2D) cell culture models are convenient but cannot adequately represent tumor microenvironment histological organization, in vivo 3D spatial/cellular context, and physiological relevance. Recently, three-dimensional (3D) in vitro tumor models have been shown to provide an improved platform for incorporating compositional and spatial heterogeneity and to better mimic the biological characteristics of patient tumors to assess drug response. Advances in 3D bioprinting have allowed the creation of more complex models with improved physiologic representation while controlling for reproducibility and accuracy. This review aims to summarize the advantages and challenges of current 3D in vitro models for evaluating therapy response in breast cancer, with a particular emphasis on 3D bioprinting, and addresses several key issues for future model development as well as their application to other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ccd5bcde9ada94a158d2c3401e2be4034200cf" target='_blank'>
              Modeling intratumor heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            E. McDonough, M. Barroso, Fiona Ginty, David T Corr
          </td>
          <td>2024-12-06</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) exhibits a vast heterogeneity in cell types and states. Recent research highlights the reactivation of neurodevelopmental programs in GBM, contributing to this heterogeneity and, ultimately, therapy resistance. To characterize the landscape of progenitor cell types in human primary GBM, we compiled a meta-atlas of single-cell transcriptomes from seven published studies. Using a novel bioinformatic method for gene program analysis, we derived GBM gene expression meta-modules represented across multiple tumors. Our analyses validated previously described GBM gene programs and revealed novel tumor progenitor subtypes. Among these subtypes, we identified a tumor-derived stem cell population exhibiting a mixed vascular identity and transcriptional programs reminiscent of neural stem cells, which we identified as a putative neurovascular progenitor (NVP). To our knowledge, this population has not been previously described in GBM. We validated the existence of NVPs in situ using immunofluorescence from primary patient GBM samples. We then performed an enrichment for NVPs using fluorescence-activated cell sorting direct from patient tumors, followed by single-cell RNA sequencing. Examination of data from traditional model systems, including gliomaspheres and mouse GBM models, confirmed the preservation of NVP identity across systems. In vivo analysis of NVPs in multiple mouse models of GBM elucidate the influence of NVPs on GBM cell type composition, highlighting their role as multipotent progenitors. Finally, we utilized single-cell lineage barcoding to trace the progeny of patient-derived NVP cells in a novel organoid transplantation model, revealing NVP differentiation into vascular-like and neural-like cells. These data indicate that NVP cells play crucial roles in tumor plasticity and may represent a valuable therapeutic target for future exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4c96f99928f041516b346c68c5a8d4ece5bdedd" target='_blank'>
              STEM-14. INTEGRATED ANALYSIS OF GLIOBLASTOMA PROGENITOR SUBTYPES REVEALS A TUMOR-DERIVED STEM CELL POPULATION WITH NEURAL AND VASCULAR POTENTIAL
              </a>
            </td>
          <td>
            Elisa Fazzari, Daria J. Azizad, Matthew Li, Kwanha Yu, Weihong Ge, Patricia R. Nano, Ryan L. Kan, Jose A. Soto, Elizabeth E Crouch, D. Raleigh, Dimitri Cadet, C. Tse, Andrew Tum, Nicholas A. Bayley, Benjamin Deneen, Kunal S. Patel, David Nathanson, A. Bhaduri
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Melanoma is a highly malignant skin tumor characterized by high metastasis and poor prognosis. Recent studies have highlighted the pivotal role of melanoma stem cells (MSCs)—a subpopulation of cancer stem cells (CSCs)—in driving tumor growth, metastasis, therapeutic resistance, and recurrence. Similar to CSCs in other cancers, MSCs possess unique characteristics, including specific surface markers, dysregulated signaling pathways, and the ability to thrive within complex tumor microenvironment (TME). This review explored the current landscape of MSC research, discussing the identification of MSC-specific surface markers, the role of key signaling pathways such as Wnt/β-catenin, Notch, and Hedgehog (Hh), and how interactions within the TME, including hypoxia and immune cells, contribute to MSC-mediated drug resistance and metastatic behavior. Furthermore, we also investigated the latest therapeutic strategies targeting MSCs, such as small-molecule inhibitors, immune-based approaches, and novel vaccine developments, with an emphasis on their potential to overcome melanoma progression and improve clinical outcomes. This review aims to provide valuable insights into the complex roles of MSCs in melanoma biology and offers perspectives for future research and therapeutic advances against this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fdcf0db27ab5805fb28ff517e6fe41f47f92d66" target='_blank'>
              The roles of cancer stem cells and therapeutic implications in melanoma
              </a>
            </td>
          <td>
            Xiaoli Mu, Yixin Zhou, Yongxin Yu, Mingyi Zhang, Jiyan Liu
          </td>
          <td>2024-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 To transform cancer drug development pipelines and gain a deeper mechanistic understanding of cancer: immune interactions, we desperately need better experimental models that faithfully recapitulate the cellular heterogeneity, complex and dynamic cell:cell interactions, and therapy responses in humans. To address this need, we developed and characterized a cohort of five new mouse glioma models in the C57BL6/j background. These transplantable tumorsphere models were derived from spontaneous gliomas that formed in genetically engineered mouse models with different oncogenic drivers, including EGFRviii, PDGFA, PTEN, p53, and NF1 mutations. They show a range of responses to standard of care (TMZ+IR) similar to GBM patients and all are resistant, to immune checkpoint inhibitors, anti-PD1+CTLA4, treatment, unlike the GL261 model and similar to GBM patients. We deeply characterized each model, including immune phenotyping and single-cell RNA-sequencing analyses. We also performed a cross-species comparison of human and mouse GBM and stromal cell phenotypes at the single-cell level. Our results show that new mouse models recapitulate the heterogeneity of glioma cell states observed in individual patients and, importantly, also the complex and immune suppressive TME. We demonstrate at the single-cell level that glioma-associated T and myeloid cell types are molecularly similar and conserved across species. Leveraging these data sets and experimental models, we are testing now novel immunotherapy targets for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcbcbc1f841c1a6de776b418447465f25bcaddf" target='_blank'>
              IMMU-66. PHYSIOLOGICALLY RELEVANT MURINE GLIOMA MODELS THAT REPRESENT GBM HETEROGENEITY AND ENABLE THE NEXT GENERATION OF DISCOVERY AND THERAPY EFFICACY STUDIES
              </a>
            </td>
          <td>
            J. Maldonado, Caiyi Wang, Jia-Shiun Leu, H. N. Tran, Nourhan Abdelfattah, Kyuson Yun
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Teratomas are a highly differentiated type of testicular germ cell tumors (TGCTs), the most common type of solid cancer in young men. Prominent inflammatory infiltrates are a hallmark of TGCTs, although their compositions and dynamics in teratomas remain elusive. Here, we reached out to characterize the infiltrating immune cells and their activation and polarization state by using high-throughput gene expression analysis of 129.MOLF-Chr19 mice that spontaneously develop testicular teratomas. We showed that inconspicuous testes without any apparent alterations in size or morphology can be clustered into three groups based on their expression of stemness and immune genes, supporting a model in which initial oncogenic transformation elicits a first wave of T-cell infiltration. Moderately and severely enlarged tumorous testes then displayed a progressive infiltration with T cells, monocytes/macrophages, and B cells. Importantly, T cells seem to adopt an inactive state caused by an overexpression of immune checkpoint molecules and the polarization of monocytes/macrophages to an anti-inflammatory phenotype. Our findings are supported by the analysis of metabolic gene expression, which unveiled alterations indicative of tumor growth and immune cell infiltration. Collectively, testicular teratomas, at least in mice, are characterized by a diverse inflammatory infiltrate containing T cells that putatively become inactivated, allowing the tumors to further grow. We believe that these findings may provide a rationale for the development of new immunomodulatory therapies for TGCTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b07bb71a6c00863ba07bb1169016cede3a797fe7" target='_blank'>
              Teratoma Development in 129.MOLF-Chr19 Mice Elicits Two Waves of Immune Cell Infiltration
              </a>
            </td>
          <td>
            Lucas Klaus, Sybille D. Reichardt, Maria Neif, Lutz Walter, F. Gayer, Holger M Reichardt
          </td>
          <td>2024-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="We present CancerFoundation, a novel single-cell RNA-seq foundation model (scFM) trained exclusively on malignant cells. Despite being trained on only one million total cells, a fraction of the data used by existing models, CancerFoundation outperforms other scFMs in key tasks such as zero-shot batch integration and drug response prediction. During training, we employ tissue and technologyaware oversampling and domain-invariant training to enhance performance on underrepresented cancer types and sequencing technologies. We propose survival prediction as a new downstream task to evaluate the generalizability of single-cell foundation models to bulk RNA data and their applicability to patient stratification. CancerFoundation demonstrates superior batch integration performance and shows significant improvements in predicting drug responses for both unseen cell lines and drugs. These results highlight the potential of focused, smaller foundation models in advancing drug discovery and our understanding of cancer biology. Our code is available here1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7840bae1209847ce739e9339f47119fc347eb843" target='_blank'>
              CancerFoundation: A single-cell RNA sequencing foundation model to decipher drug resistance in cancer
              </a>
            </td>
          <td>
            Alexander Theus, Florian Barkmann, David Wissel, Valentina Boeva
          </td>
          <td>2024-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epithelial-mesenchymal plasticity (EMP) is a cell-fate switching program that enables cells to adopt a spectrum of phenotypes ranging from epithelial (E) to mesenchymal (M) and hybrid E/M states. Hybrid E/M phenotypes are considered optimal for cancer metastasis due to their association with metastatic initiation and traits such as stemness, drug resistance, and collective migration. Boolean models of gene regulatory networks (GRNs) underlying EMP have provided valuable insights into the dynamics of E and M phenotypes. However, these models fail to capture hybrid phenotypes effectively, as Boolean frameworks allow only binary gene expression, while hybrid E/M cells exhibit partial expression of both E and M genes. To address this limitation, we modified a threshold-based Boolean formalism to incorporate intermediate gene expression levels. The resulting multilevel model reveals new hybrid steady states with partial expression of both E and M genes, expanding the phenotypic landscape beyond that captured by traditional Boolean models. These hybrid states exhibit lower frustration and higher abundance, indicating enhanced stability. By resolving degeneracy in the dynamics, we further demonstrate that hybrid states identified by the multilevel model are more stable. These findings suggest that adding minimal complexity to Boolean models can uncover previously hidden qualitative features of phenotypic landscapes governed by GRNs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29922a69bbdff28fdd118c9e80f2fc16659ff8c" target='_blank'>
              A multi-level formalism to capture the characteristics of hybrid phenotypes in Epithelial-Mesenchymal Plasticity
              </a>
            </td>
          <td>
            K. Hari, Shubham Tripathi, Vaibhav Anand, M. Jolly, Herbert Levine
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) account for 0.01 to 2% of the total tumor mass; however, they play a key role in tumor progression, metastasis and resistance to current cancer therapies. The generation and maintenance of CSCs are usually linked to the epithelial–mesenchymal transition (EMT), a dynamic process involved in reprogramming cancer cells towards a more aggressive and motile phenotype with increased stemness potential. Cells that undergo an EMT process have shown to be more resistant to conventional chemo/radiotherapies. In this context, aldehyde dehydrogenase (ALDH) enzymes, known for their role in the cellular detoxification of aldehydes and enhancement of cell survival, are often upregulated in cancer cells, promoting their resistance to conventional cancer treatments. Indeed, high ALDH levels have become a hallmark biomarker of CSCs and are often used to isolate this sub-population from the more abundant cancer cell populations. Herein, we isolated human breast cancer epithelial cells with higher ALDH abundance (ALDHHigh) and compared them to those with low ALDH abundance (ALDHLow). ALDHHigh sub-populations exhibited more characteristic EMT biomarkers by adopting a more mesenchymal phenotype with increased stemness and enhanced migratory potential. Furthermore, ALDHHigh sub-populations displayed elevated senescent markers. Moreover, these cells also demonstrated higher levels of mitochondria DNA/mass, as well as greater mitochondrial and glycolytic metabolic function. Conversely, ALDHLow sub-populations showed a higher efficiency of mammosphere/colony formation and an increased proliferative capacity. Therefore, we demonstrated that these ALDH sub-populations have distinct characteristics, underscoring their role in EMT, the formation of tumors and the mechanisms of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c656cc9a0663a21dc7738f32f6803ef9c5938a5" target='_blank'>
              ALDHHigh Breast Cancer Stem Cells Exhibit a Mesenchymal–Senescent Hybrid Phenotype, with Elevated Metabolic and Migratory Activities
              </a>
            </td>
          <td>
            Luis Larrea Murillo, Conor J. Sugden, Bela Ozsvari, Zahra Moftakhar, Ghada S. Hassan, F. Sotgia, Michael P Lisanti
          </td>
          <td>2024-12-13</td>
          <td>Cells</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="The phenotype of circulating tumor cells (CTCs) offers valuable insights into monitoring cancer metastasis and recurrence. While microfluidics presents a promising approach for capturing these rare cells in blood, the phenotypic profiling of CTCs remains technically challenging. Herein, we developed a nanoporous micropillar array chip enabling highly efficient capture and in situ phenotypic analysis of CTCs through enhanced and tunable on-chip immunoaffinity binding. The nanoporous micropillar array addresses the fundamental limits in fluidic mass transfer, surface stagnant flow boundary effect, and interface topographic and multivalent reactions simultaneously within a single device, resulting in a synergistic enhancement of CTC immunocapture efficiency. The CTC capture efficiency increased by approximately 40% for cancer cells with low surface marker expressing. By manipulating fluidic velocity (hydrodynamic drag force) on the chip, a cell adhesion gradient was generated in the capture chamber, enabling individual CTCs with varying expression levels of epithelial cellular adhesion molecules to be immunocaptured at the corresponding spatial locations where equilibrium drag force is provided. The clinical utility of the nanoporous micropillar array was demonstrated by accurately distinguishing early and advanced stages of breast cancer and further longitudinally monitoring treatment response. We envision that the nanoporous micropillar array chip will provide an in situ capture and molecular profiling approach for CTCs and enhance the clinical application of CTC liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a4168617dcff4a12ca2fb61825d5b9644d135db" target='_blank'>
              Profiling Phenotypic Heterogeneity of Circulating Tumor Cells through Spatially Resolved Immunocapture on Nanoporous Micropillar Arrays.
              </a>
            </td>
          <td>
            Lianyu Lu, Yaohui Wang, Yue Ding, Yuqing Wang, Zhi Zhu, Jinsong Lu, Liu Yang, Peng Zhang, Chao Yang
          </td>
          <td>2024-11-04</td>
          <td>ACS nano</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Diffuse midline gliomas, H3K27-altered (DMG) are highly malignant brain tumors with a 2-year survival rate of less than 10 percent. These tumors are primarily localized to the pons and thalamus regions of the brain. The cellular compartment of the tumor microenvironment (TME) which surrounds a tumor is composed of tumor and non-tumor cells, such as immune cells and oligodendrocytes. Communication between tumor cells and non-tumor cells in the TME can promote tumor growth and targeting communication networks could improve patient outcomes. Using a bioinformatics approach and publicly available pediatric DMG single-cell RNA-sequence
 (scRNA-seq) data, several methods of communication (secreted signaling, cell-cell contact, and extracellular matrix) within the DMG TME were identified. Based on the high communication probability, outgoing and incoming networks from non-tumor cells and tumor cells were inferred, such as the Complement, Galectin, and CD96 pathways. Specific ligand-receptor interactions between non-tumor and tumor cells were also found, including LGALS9-HAVCR2, PDGFB/PDGFRA, and COL4A5/ITGAV. A subsequent literature review was performed to validate these identified pathways in other tumors, including gliomas, and inform future in vitro experiments with primary DMG cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86e4e55f141e1e3cca1b3dde7a5016937939f79" target='_blank'>
              TMIC-65. USING BIOINFORMATICS TO INVESTIGATE THE TUMOR MICROENVIRONMENT OF DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Havisha Kalpatthi, Andrea F Cruz, S. Agnihotri
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Rationale:
 Smooth muscle cell (SMC) plasticity and phenotypic switching play prominent roles in the pathogenesis of multiple diseases like atherosclerosis and pulmonary hypertension, but their role in tumorigenesis is unknown.


 Objective:
 To investigate SMC diversity and plasticity in tumor angiogenesis and its underlying molecular drivers.


 Methods and Results:
 Here we use SMC-specific lineage-tracing mouse models and single cell RNA sequencing (scRNA-seq) to observe the phenotypic diversity of SMCs participating in tumor vascularization. We find that around 10% of SMCs adopt a phenotype which includes features traditionally associated with macrophage-like cells, suggesting they may contribute to the immune profile of the tumor microenvironment (TME). These cells are transcriptionally similar to ‘resolution phase’ M2b macrophages, which have been described to have a role in inflammation resolution. Signaling from bone marrow stromal antigen 2 (BST2) on the surface of tumor cells to PIRA2 on SMC promotes this phenotypic transition, leading to a SMC phenotype that is more proliferative, migratory, and phagocytic. Knockdown of BST2 in the tumor significantly decreases the transition towards a macrophage-like phenotype, and the cells that were able to transition had a comparatively higher inflammatory signal typically associated with anti-tumor effect.


 Conclusions:
 As BST2 is known to be a poor prognostic marker in multiple cancers where it is associated with an M2 macrophage-skewed TME, these studies suggest that phenotypically switched SMCs may have a previously unidentified role in this immunosuppressive milieu. The novel concept of SMC contribution to the immune landscape suggests manipulation of SMC phenotypic switching as an approach to altering cancer pathogenesis. Further translational work is needed to understand how this phenotypic switch could influence response to anti-cancer agents and if targeted inhibition of SMC plasticity would be therapeutically beneficial.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/049a7db1d4940952698e1c93fbbc4e08101322fd" target='_blank'>
              Abstract 4134531: BST2 induces vascular smooth muscle cell plasticity and phenotype switching during cancer progression
              </a>
            </td>
          <td>
            Caitlin F. Bell, Richard A. Baylis, Wei Feng Ma, Clint Miller, Nick J. Leeper
          </td>
          <td>2024-11-12</td>
          <td>Circulation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recent advances in fate mapping and single-cell technologies have revealed how the dynamics and function of tissue-resident macrophages are shaped by their environment. However, macrophages in sensory organs such as the cochlea where the central nervous system and peripheral nervous system meet remain understudied. Combining single-cell transcriptomics, fate mapping, and parabiosis experiments, we show that five types of myeloid cells including three tissue-resident macrophage subpopulations, coexist in the mouse cochlea. The three macrophage subsets showed different potential functions in relationship with their specific topography across cochlear compartments. Further analysis revealed that they were partially derived from yolk sac progenitors during development, while in adulthood, most cochlear macrophages were long-term resident. Finally, we showed that cochlear macrophage morphology and density changed during aging. Our findings show that cochlea is a microenvironment with a unique heterogeneity of macrophages in terms of gene expression, spatial distribution, ontogeny, and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26fe62c7df589b534663cdc760f68dc26a4ad8b4" target='_blank'>
              Single-cell, spatial, and fate-mapping analyses uncover niche dependent diversity of cochlear myeloid cells
              </a>
            </td>
          <td>
            Aude Chiot, Max J. Felgner, Dillon Brownell, Katherine H. Rott, Alina P. Bogachuk, Dennis-Dominik Rosmus, Takahiro Masuda, Audrey Ching, Patrick J Atkinson, Marco Prinz, Karen Sachs, Alan G. Cheng, Peter Wieghofer, Bahareh Ajami
          </td>
          <td>2024-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Aggressive human prostate tumors traverse a contractile smooth muscle pseudo-capsule, in a process termed extracapsular extension (ECE), defining the pT3 pathologic stage associating with increased biochemical recurrence, bone metastases, and cancer-specific mortality. While T3 lesions can be detected by non-invasive mpMRI imaging, how the tumor invades into and through the contractile muscle is unknown. Using a mouse xenograft model system of ECE, we investigated the transcriptomic consequences of human tumor invasion into and through the contractile smooth muscle of the mouse diaphragm. Both human and mouse gene transcripts were documented and analyzed. Tumor cells were intraperitoneally injected into male NSG mice and 6 weeks later, three tumor compartments (pre-invasive, muscle-invasive, and super-invasive cells that had reached the superior diaphragm surface) were analyzed for differential bulk human and mouse gene expression. Whole genome transcriptomic sequencing and GO pathway analysis revealed that approximately 414 human genes were differentially expressed between the non- invasive, muscle-resident, and super-invasive tumor populations and at least 5 enriched pathways were involved. The unique expression gene patterns of muscle-resident tumor cells were reversible when compared to the pre-invasive and super-invasive expression patterns. Significant increases in g-H2AX and nuclear deformation were observed in the muscle-resident tumor clusters as compared to the non-invasive tumor mass. No differences in tumor cell proliferation, as detected by Ki67 staining, were found between the tumor clusters in the three compartments. Immunohistochemistry staining for a damage response cytokeratin (KRT6A) and integrin (CD49f) revealed heterogeneity within the muscle resident tumors as compared to the non- invasive cells. Taken together, these data suggest a hostile contractile muscle environment elicits specific responses by the invading tumor. Single cell analysis within each of the tumor compartments is currently underway to define the spatial heterogeneity of gene expression in invasive tumor cell clusters within the contractile muscle. A dynamic reciprocity of the tumor/muscle microenvironment is suggested by the analysis of the bulk transcriptomic sequencing demonstrating a significant increase in mouse muscle bio-synthetic gene transcription as a consequence of the human tumor penetrating the tissue. Taken together, these data indicate that human tumors, during the act of traversing the contractile muscle layer, respond to this unique microenvironment by transiently altering transcription, while sustaining nuclear damage which results in the reprogramming of the muscle. This new information suggests that novel tumor or muscle biomarkers might indicate early muscle invasion events and assist new high-resolution image analysis for precision diagnostic and/or therapeutic decisions. (Supported in part by P30 CA23074; F30 CA143924, UACC Team Science Award).
 Citation Format: Kendra D Marr, Beatrice S Knudsen, Rafael Sainz, Kelvin W Pond, Noel E Warfel, Belinda E Sun, Anne E Cress. Tumor muscle invasion promotes tumor heterogeneity and normal muscle reprogramming [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d695d2e17a8bb732f87fee7797bcc07c303850" target='_blank'>
              Abstract A055: Tumor muscle invasion promotes tumor heterogeneity and normal muscle reprogramming
              </a>
            </td>
          <td>
            Kendra D. Marr, Beatrice S. Knudsen, Rafael Sainz, Kelvin W Pond, Noel E. Warfel, Belinda E Sun, A. Cress
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Single naive B cells amplify and diversify their responses when activated by cognate antigen, via Myc-dependent clonal expansion, immunoglobulin class switch recombination (CSR), phenotypic variation, and somatic hypermutation (SHM). Whether these mechanisms act combinatorially in vivo to diversify clonal responses to a single complex pathogen remains unclear. Since diversity in the antigenic targets, functional classes, and production kinetics of parasite-specific antibodies influences immunity to malaria, we test here whether individual B cell clones diversify over time during Plasmodium infection and treatment. During the first week of infection, amid widespread Type I Interferon (IFN)-mediated, bystander activation, CSR initiates soon after Myc up-regulation. CSR then overlaps with clonal expansion, resulting in isotype variegation amongst certain clones. In the second week of infection, expanded clones seeding germinal centres (GC) both bifurcate into extra-follicular plasmablasts and exhibit isotype variegation, revealing substantial intra-clonal diversification. Over the following month, GC clones undergo SHM at approximately four mutations per week, with IgG mutational diversity and IgM+ cells preserved in GCs over time. Anti-malarial intervention does not impede SHM, but instead exerts quantitative limits on GC size, plasma cell emergence and circulating antibody titres. Finally, we reveal a contemporaneous B cell developmental pathway that relocates from bone marrow to the spleen during infection and treatment. Thus, multiple temporally overlapping mechanisms combine in vivo to amplify, diversify, and safeguard humoral immune responses. Graphical abstract Highlights Early temporal overlap of CSR with clonal expansion leads to isotype variegation in clones. Clones seeding GCs bifurcate into plasmablasts and exhibit isotype variegation. GC B cells accrue mutations at rates unaffected by anti-malarials. Plasmodium infection triggers antigen-independent Type I IFN-mediated bystander activation. B cell development is preserved in malaria by shifting from bone marrow to spleen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a7949f74106084051fa37c0e0531210e88f8186" target='_blank'>
              Temporally overlapping mechanisms diversify clonal B cell responses in vivo
              </a>
            </td>
          <td>
            Oliver P. Skinner, Saba Asad, , Hyun Jae Lee, , Shihan Li, Wang Jin, Thiago M. Steiner, Takahiro Asatsuma, Jorene Lim, Zheng Ruan, Megan Sioe Fei Soon, Jessica A. Engel, David S. Khoury, Z. Tuong, Hamish W. King, A. Haque
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d907c1f70a7e162db16222c621c3aec6a63202a" target='_blank'>
              Integrated single-cell analysis reveals heterogeneity and therapeutic insights in osteosarcoma
              </a>
            </td>
          <td>
            Dongan He, Xiaoqian Che, Haiming Zhang, Jiandong Guo, Lei Cai, Jian Li, Jinxi Zhang, Xin Jin, Jianfeng Wang
          </td>
          <td>2024-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor heterogeneity is a significant factor influencing cancer treatment effectiveness and can arise from genetic, epigenetic, and phenotypic variations among cancer cells. Understanding how tumor heterogeneity impacts tumor evolution and therapy response can lead to more effective treatments and improved patient outcomes. Traditional bulk genomic approaches fail to provide insights into cellular-level events, whereas single-cell RNA sequencing (scRNA-seq) offers transcriptomic analysis at the individual cell level, advancing our understanding of tumor growth, progression, and drug response. However, implementing single-cell approaches in clinical trials involves challenges, such as obtaining high-quality cells, technical variability, and the need for complex computational analysis. Effective implementation of single-cell genomics in clinical trials requires a collaborative “Team Medicine” approach, leveraging shared resources, expertise, and workflows. Here, we describe key technical considerations in implementing the collection of research biopsies and lessons learned from integrating scRNA-seq into City of Hope’s clinical trial design, highlighting collaborative efforts between computational and clinical teams across breast, brain, and ovarian cancer studies to understand the composition, phenotypic state, and underlying resistance mechanisms within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e501cb60571ffc07f7277f4aa067130d5278a3" target='_blank'>
              Single-Cell Transcriptomics Sheds Light on Tumor Evolution: Perspectives from City of Hope’s Clinical Trial Teams
              </a>
            </td>
          <td>
            Patrick A. Cosgrove, Andrea H. Bild, T. Dellinger, B. Badie, J. Portnow, Aritro Nath
          </td>
          <td>2024-12-10</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="

 Introduction:
 Hematopoietic stem cells are in constant clonal competition, as a result variant clones resistant to inflammatory pressures emerge with aging resulting in clonal hematopoiesis. Multiple independent clones can coexist, and potentially come into equilibrium. However, with an acute leukemia a clone overtakes the system, thus inhibiting normal hematopoiesis. We sought to explore the possibility of leveraging the increased competitive ability of CHIP clones to dampen expansion of an aggressive malignant clone. As proof of principle, we tested whether introduction of Tet2-/- cells as competitors could dampen the expansion of MPLW515L cells, an aggressive MPN mouse model.
 Methods:
 We used a transduction-transformation model of MPL W515L as an aggressive MPN model. This mouse model usually results in death approximately 3-4 weeks post-transplant. Whole bone marrow from wild-type C57B/6 (CD45.1) mice was transduced with Green Fluorescent Protein (GFP) tagged retrovirus to induce expression of hMPLW515L. We transduced bone marrow with a low viral multiplicity of infection (MOI) resulting in approximately 1-5% GFP+ cells to extend survival beyond the expected 3-4 weeks post-transplant. The transduced bone marrow was mixed with unmanipulated whole bone marrow from either WT or Tet2-/- (both in CD45.1/2) to result in a 90%/10% mixture and transplanted into lethally irradiated CD45.2 mice. Peripheral blood counts, % GFP+ cells, and chimerism was followed biweekly. Surviving mice were sacrificed and analyzed at 5 months post-transplant.
 Results:
 There was early death from expansion of MPLW515L GFP+ cells within 3-4 weeks in 10% of both the WT and Tet2-/- competitor groups. However, of the surviving mice the percentage of GFP+ cells in the peripheral blood was consistently lower in the Tet2-/- competitor group compared to the WT competitor group (for example mean of 55% GFP+ in WT competitor group vs 23% at day 105 post-transplant). At five months post-transplant we sacrificed the cohort to assess the composition of MPLW515L cells in the hematopoietic stem and progenitor compartments. The percentage of GFP+ cells was higher in all cell populations assessed in the WT competitor group compared to the Tet2-/- competitor group. Moreover, spleen and liver weights were higher in the WT versus Tet2-/- competitor group.
 Conclusions:
 This work demonstrates proof of principle that clones with increased competitive ability and low malignant potential, such as Tet2-/- cells, can be used as a buffer for the expansion of highly aggressive clones. This concept of introduction of non-malignant competitor clones could be potentially be explored as a novel therapeutic approach in aggressive hematologic malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02bdd911e73f56ee0ae933fdda9f4ee1c54e2a20" target='_blank'>
              Leveraging Clonal Competition between Chip and Aggressive Malignant Clones As a Novel Therapeutic Approach
              </a>
            </td>
          <td>
            K. K. Guillermo, Xin (Helen) Huang, Jianhong C Heidmann, Angela G. Fleischman
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common and most aggressive malignant primary brain tumor in adults. Among the most lethal human malignancies, GBM is highly resistant to cytotoxic treatments and molecularly targeted therapies, and also generally refractory to T cell–based immunotherapies, largely due to their immunologically cold nature, i.e., characterized by a paucity of tumor T cell infiltrates that result from the immunosuppressive microenvironment. Here, we identify a distinct mechanism driven by spatiotemporal interaction of leukocytes with tumor stromal cells, namely, mesenchymal-like population of endothelial cells (ECs) form an immunosuppressive vascular niche that regulates macrophage polarization and induces GBM resistance to CAR T immunotherapy. Our single-cell transcriptome analyses of human and mouse GBM tumors identify a subpopulation of ECs acquire robust immunosuppressive phonotypes to drive alternative macrophage polarization. These ECs are spatially localized proximately to immunosuppressive macrophages and selectively express Twist1. Furthermore, we reveal a Twist1/SATB1-mediated sequential transcriptional activation mechanism, through which Twist1+ tumor ECs produce osteopontin to locally promote immunosuppressive macrophage phenotypes. Genetic or pharmacological ablation of Twist1 reverses macrophage-mediated immunosuppression and enhances T cell infiltration and activation, leading to reduced GBM growth and extended mouse survival. Importantly, pharmacological inhibition and lipid nanoparticle-mediated targeted inactivation of Twist1 overcome GBM resistance to Egfrviii CAR T cell immunotherapy. Thus, these findings uncover a spatially restricted mechanism controlling vasculature-mediated tumor immunity and suggest that targeting endothelial Twist1 may offer exciting opportunities for GBM immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa787a775e12e3cc0befad1784ebce83795d53f9" target='_blank'>
              IMMU-22. TWIST1-MEDIATED ENDOTHELIAL PLASTICITY DRIVES GLIOBLASTOMA RESISTANCE TO CAR T IMMUNOTHERAPY
              </a>
            </td>
          <td>
            Fan Yang, Md Naushad Akhtar, Yanqing Gong, Yi Fan
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Acquired resistance is one of the major causes of failure of standard therapies in cancer patients. Chemotherapeutic agents are still widely used and the understanding of the mechanisms leading to secondary resistance to these molecules are still puzzling. Recently, the role of the tumor immune microenvironment has been recognized. Among the cells potentially involved, macrophages seem to be the perfect culprits. In a previous work, we have shown that hypoxic macrophages are able to provide strong protection against 5-fluorouracil, a first-line chemotherapeutic agent in digestive cancers. In the present work, we use mathematical modeling to explore the spatiotemporal aspects of the treatment-induced organization of the tumor environment. Based on analytical and numerical analysis, we propose that macrophage-driven protection against chemotherapy under treatment does not rely solely on biochemical degradation, but is enhanced by the emergence of spatially structured chemotherapeutic protective niches. This work paves the way for the development of new therapeutic strategies that rely on targeting the spatial organization of tumors as a way to control treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f1862b2c528215ed84b22678834153abe2795d7" target='_blank'>
              Spatial organization of mediated-macrophage chemoprotective niches in solid tumors: A mathematical analysis
              </a>
            </td>
          <td>
            William Dangelser, A. Stéphanou, Arnaud Millet
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="

 Successful immunotherapies for acute myeloid leukemia (AML) remain elusive, partly due to the incomplete characterization of leukemia-induced changes to the T cell landscape. Here, we present proteomic, transcriptomic, and functional studies of the AML T cell landscape using patient samples, two immunocompetent (radiation-free) mouse models of leukemia, and publicly available datasets.
 We immunophenotyped cells from healthy donor marrow (n=5) and AML patient samples (n=35) using mass cytometry with 20-plex mass-tag barcoding. We used a 38-marker panel to identify progenitor, myeloid, and lymphoid populations. In parallel, we used a 37-marker panel with a focus on T cell phenotypes to identify their subpopulations. Computational debarcoding, batch correction, dimensional reduction, and clustering were followed by expert identification of cell types based on canonical surface marker expression. Within the CD8 T cell compartment, AML samples had significantly less abundant naïve T cells and more abundant terminal effector T cells compared to healthy controls. The terminal effector CD8 T cell cluster was characterized by expression of CD57, perforin, and granzyme B.
 AML cells are also immune cells and may have unique immunomodulatory properties. Therefore, we hypothesized that CD8 T cells in AML are transcriptionally distinct from CD8 T cells from solid tumors. We integrated publicly available scRNAseq data from AML (Desai et al, 2023) and pan-cancer tumor infiltrating lymphocytes (TILS, Zheng et al, 2021). Differential expression analysis comparing CD8+ T cells in AML to CD8+ TILS in breast cancer, esophageal cancer, thyroid cancer, and endometrial cancer revealed relative upregulation of KLRG1, IFNG, and TNF in AML-associated CD8 T cells.
 Next, we tested the anti-leukemic role of CD8 T cells in an immunocompetent mouse model of AML. C57BL/6J-Ptprcem6Lutzy/J CD45.1+ host mice were transplanted with CD45.2+ leukemic cells bearing the MLL-AF9 translocation and then treated with αCD8α depleting antibody or isotype control. Mice treated with the αCD8α depleting antibody trended toward shorter survival (median, 23.0 days αCD8α vs 26.5 days control), albeit was not significant (log rank test, p=0.076). We hypothesized that AML-associated CD8 T cells develop a dysfunctional phenotype that precludes them from exerting their effector functions. Indeed, within the CD8 T cell compartment, mice with MLL-AF9 leukemia had significantly more abundant PD1+, LAG3+, and TIM3+ cells compared to those transplanted with healthy, wild-type marrow. These differences in CD8 T cell phenotypes were replicated in a second immunocompetent mouse model, wherein C57BL/6J-Ptprcem6Lutzy/J host mice were transplanted with leukemic cells bearing FLT3ITD-NPM1c mutations.
 We then investigated if AML-associated CD8 T cells could exert their effector functions ex vivo. After phorbol myristate acetate and ionomycin stimulation, a significantly higher proportion of CD8 T cells from MLL-AF9 produced TNFα and IFNγ compared to their healthy counterparts by flow cytometry. Finally, we conducted RNA sequencing of the CD44+CD62L- (effector) CD8 T cell population from MLL-AF9 marrow and healthy marrow. Gene set enrichment analysis of Hallmark pathways revealed significant enrichment of proliferation-related pathways such as E2F targets (NES=1.64), G2M checkpoint (NES=2.86), and mitotic spindle (NES=1.64) in AML-associated T cells in addition to the IL2 signaling pathway (NES=1.84). However, differential expression analysis showed increased expression of Tigit, Pdcd1, Havcr2, and Tox.
 Here, we present several orthogonal studies to investigate AML-associated T cells in human samples and mouse models. Our data demonstrate that T cells in human AML, identified by mass cytometry and scRNAseq, are enriched for terminal effector CD8 T cells and display a unique phenotype among cancer subtypes. We then establish two immunocompetent mouse models with unique driver genes, and further show similar T cell differentiation and dysfunction to human samples. Finally, we demonstrate that while these T cells retain, and even increase, their ability to produce cytokines with stimulation, by transcriptional analysis display signs of impending dysfunction. This exhaustive characterization of T cell dysfunction in AML will enable future studies into the exploitation of the immune system for AML-directed T cell therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6ceba3dc65f644e6e06d4de543b1fb58e1bb769" target='_blank'>
              Combined Human and In Vivo Analyses Implicate the Critical Importance of Acute Myeloid Leukemia Associated CD8 T Cells and Identifies Their Molecular Dysfunction
              </a>
            </td>
          <td>
            Y. Chu, Matthew A. Cottam, Chad R Potts, Kirsten Dickerson, Matthew Jenkins, Justin A Cartailler, Rui Lu, Stanley C Lee, Ross L Levine, Robert L Bowman, Robert S. Welner, P. Ferrell
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Pericytes are crucial for the development, stabilization, and functional regulation of microvasculature, especially in the retina. In diabetic retinopathy (DR), early loss of pericytes is a key event that drives microvascular dysfunction. Despite their critical role, the mechanisms underlying the functional heterogeneity of pericytes in DR remain poorly understood, impeding the development of effective therapeutic strategies. Methods We employed single-cell RNA sequencing to construct a comprehensive single- cell atlas of non-diabetic and diabetic retinas. Using bioinformatic clustering and subcluster analysis, we identified a specific pericyte subcluster associated with diabetic microvascular complications. Differential gene expression analysis and immunofluorescence validation highlighted PTTG1 as a potential key regulator of pericyte dysfunction. To investigate its functional role, we emplyed CRISPR/Cas9 and adenoviral vectors to modulate PTTG1 expression in vitro and in vivo. Combined transcriptomic and metabolomic approaches were used to explore the mechanistic pathways through which PTTG1 influences pericyte biology and vascular function. Results We identified a novel pericyte subcluster characterized by elevated expression of PTTG1, which was strongly correlated with diabetic microvascular dysfunction. Silencing PTTG1 using CRISPR/Cas9 and siRNA in vitro mitigated pericyte dysfunction under high- glucose conditions. Targeted knockdown of PTTG1 using viral vectors improved retinal vascular integrity and reduced neovascularization in diabetic mice. Transcriptomic and untargeted metabolomic analyses revealed that PTTG1 knockdown reprogrammed pericyte energy metabolism by modulating glycolysis pathway genes, reducing oxidative stress, and restoring pericyte function, ultimately alleviating microvascular dysfunction in DR. Conclusions PTTG1 plays a critical role in regulating pericyte dysfunction and maintaining vascular homeostasis in diabetic retinopathy. By modulating key metabolic pathways and pericyte phenotypes, PTTG1 represents a promising therapeutic target for treating diabetic microvascular complications. These insights offer a novel molecular framework for developing targeted therapies aimed at restoring retinal vascular health in diabetic patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d2cfb264dd76b8c3dd1f5b9db0dce030fd7e6a3" target='_blank'>
              PTTG1-Mediated Pericyte Dysfunction Drives Diabetes-Induced Microvascular Dysfunction
              </a>
            </td>
          <td>
            Linyu Zhang, Ling Ren, Jingyue Zhang, Min Xia, Xiaosa Li, Mudi Yao, Fanfei Ma, Chang Jiang, Jin Yao, Biao Yan
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Cancer progression and response to therapy are inextricably reliant on the coevolution of a supportive tissue microenvironment. This is particularly evident in pancreatic ductal adenocarcinoma, a tumor type characterized by expansive and heterogeneous stroma. Herein, we employed single-cell RNA sequencing and spatial transcriptomics of normal, inflamed, and malignant pancreatic tissues to contextualize stromal dynamics associated with disease and treatment status, identifying temporal and spatial trajectories of fibroblast differentiation. Using analytical tools to infer cellular communication, together with a newly developed assay to annotate genomic alterations in cancer cells, we additionally explored the complex intercellular networks underlying tissue circuitry, highlighting a fibroblast-centric interactome that grows in strength and complexity in the context of malignant transformation. Our study yields new insights on the stromal remodeling events favoring the development of a tumor-supportive microenvironment and provides a powerful resource for the exploration of novel points of therapeutic intervention in pancreatic ductal adenocarcinoma. Significance: Pancreatic cancer remains a high unmet medical need. Understanding the interactions between stroma and cancer cells in this disease may unveil new opportunities for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106968e3649b3283e96d5bfd01ae138fc25d6cc4" target='_blank'>
              Dynamic Evolution of Fibroblasts Revealed by Single-Cell RNA Sequencing of Human Pancreatic Cancer
              </a>
            </td>
          <td>
            S. Dimitrieva, Jon M Harrison, Jonathan Chang, M. Piquet, M. Mino-Kenudson, Millicent Gabriel, Vivek Sagar, Heiko Horn, K. Lage, Julie Kim, Gang Li, Shaobu Weng, Cynthia Harris, Anupriya S Kulkarni, David T Ting, M. Qadan, Peter J Fagenholz, Cristina R Ferrone, Angelo L Grauel, Tyler Laszewski, Alina Raza, Markus Riester, Tim Somerville, Joel P Wagner, Glenn Dranoff, J. Engelman, Audrey Kauffmann, Rebecca Leary, Andrew L Warshaw, K. Lillemoe, C. F. Fernandez-Del Castillo, David A Ruddy, Andrew S. Liss, Viviana Cremasco
          </td>
          <td>2024-11-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>130</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef9e868d681b765f550dc5f5febbd2934a88e0fb" target='_blank'>
              Single-cell RNA sequencing reveals the landscape of the cellular ecosystem of primary hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zeli Yin, Yilin Song, Liming Wang
          </td>
          <td>2024-11-14</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5361264ba7fa5b03a05e9caae598d056bebbf71c" target='_blank'>
              Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Yilv Yan, Dongqing Sun, Junjie Hu, Yue Chen, Liangdong Sun, Huansha Yu, Yicheng Xiong, Zhida Huang, Haoran Xia, Xinsheng Zhu, D. Bian, Fenghuan Sun, Likun Hou, Chunyan Wu, Orion R Fan, Haiyang Hu, An Zeng, Lele Zhang, Yi Eve Sun, Chenfei Wang, Peng Zhang
          </td>
          <td>2024-12-10</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Pancreatic cancer has a 5-year survival rate of merely 12%. The high rate of late-stage diagnoses underscores the need for reliable biomarkers for early detection and disease monitoring. Circulating tumor cells (CTCs) have emerged as a promising biomarker, yet their detection remains challenging due to their rarity and phenotypic diversity. This study evaluates the Parsortix® system, a microfluidic device designed to enrich CTCs based on size and deformability, using pancreatic cancer cell lines. As increasing evidence indicates that during epithelial to mesenchymal transition (EMT) a cell’s deformability increases, we evaluated to what extent the Parsortix® system was biased towards epithelial cancer cells. First, the EMT stage of three pancreatic cancer cell lines, CAPAN-1, PANC-1 and MIA PaCa-2, was assessed using immunocytochemistry, flow cytometry and proteomics. CAPAN-1 cells were classified as epithelial, MIA PaCa-2 cells exhibited a mesenchymal-like phenotype, and PANC-1 cells demonstrated a hybrid phenotype. Then, by spiking these cells into blood samples, we determined the Parsortix® system’s ability to recover the cancer cells. Our results indicated that epithelial and hybrid phenotypes are more efficiently captured (62.6 ± 18.5% and 65.4 ± 11.1%) than mesenchymal cancer cells (32.8 ± 10.2%). To confirm these findings, spike-in experiments were repeated using an EMT inducible cell line. Again, significantly lower recovery rates were found for the cells in a mesenchymal-like state (31.5 ± 6.4%) compared to those in an epithelial state (47.56 ± 7.2%). In conclusion, the Parsortix® system may underestimate the presence of mesenchymal CTCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da39e1701a00ff88015fb62ba16d3ce2af9aae2e" target='_blank'>
              Biases in the Parsortix® system observed with pancreatic cancer cell lines
              </a>
            </td>
          <td>
            N. Vandenbussche, Renske Imschoot, Béatrice Lintermans, Lode Denolf, Joachim Taminau, Charlotte Fieuws, G. Berx, Kris Gevaert, Kathleen B M Claes
          </td>
          <td>2024-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc6bfc5e3065b396e263e4793dde2e7908cde4e" target='_blank'>
              CRISPR-StAR enables high-resolution genetic screening in complex in vivo models.
              </a>
            </td>
          <td>
            E. Uijttewaal, Joonsun Lee, Annika Charlotte Sell, Naomi Botay, Gintautas Vainorius, Maria Novatchkova, J. Baar, Jiaye Yang, Tobias Potzler, Sophie van der Leij, Christopher Lowden, Julia Sinner, Anais Elewaut, Milanka Gavrilovic, Anna Obenauf, Daniel Schramek, Ulrich Elling
          </td>
          <td>2024-12-16</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Metastatic disease is the common cause of death in colorectal cancer. To better understand the cellular landscape of colorectal metastasis, we profiled primary tumors and liver, lung, and peritoneal metastases using single-cell RNA sequencing. Our analysis revealed a subset of tumor epithelial cells with a highly plastic transcription profile and, inversely, low expression of stemness markers. Tumor cells with plasticity markers were spatially enriched at the tumor- stroma interface with exposure to tissue microenvironmental factors. Cell-free malignant ascites, which recapitulates the tumor microenvironment, impaired expression of proliferation and stem- like markers in favor of plasticity markers and enhanced migration of tumor cells. Furthermore, Co-culture of colorectal tumor cells with CD11b+ macrophages isolated from malignant ascites from patients was sufficient to recapitulate tumor cell plasticity. Profiling of tumor-associated macrophages identified enrichment for a lipid-handling features and high spatial correlation with plastic tumor cells. Taken together, we demonstrate that tumor-associated macrophages contribute to disease progression through development of pro-malignant features.
 Citation Format: Matthew A. Cottam, Muhammad B. Mirza, Emily N. Arner, Marium Siddiqui, Clare Lipscombe, Zaid Hatem, Katy E. Beckermann, Kamran Idrees. Regulatory macrophages contribute to tumor cell plasticity in colorectal cancer metastases [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38b2321f9b6c18e29ebc7faf6bc4bebdd5ef49f8" target='_blank'>
              Abstract B019: Regulatory macrophages contribute to tumor cell plasticity in colorectal cancer metastases
              </a>
            </td>
          <td>
            Matthew A. Cottam, M. Mirza, Emily N. Arner, Marium Siddiqui, Clare Lipscombe, Zaid Hatem, Katy E. Beckermann, K. Idrees
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="A comprehensive understanding of tumor heterogeneity, tumor microenvironment and the mechanisms of drug resistance is fundamental to advancing breast cancer research. While single-cell RNA sequencing has resolved the issue of “temporal dynamic expression” of genes at the single-cell level, the lack of spatial information still prevents us from gaining a comprehensive understanding of breast cancer. The introduction and application of spatial transcriptomics addresses this limitation. As the annual technical method of 2020, spatial transcriptomics preserves the spatial location of tissues and resolves RNA-seq data to help localize and differentiate the active expression of functional genes within a specific tissue region, enabling the study of spatial location attributes of gene locations and cellular tissue environments. In the context of breast cancer, spatial transcriptomics can assist in the identification of novel breast cancer subtypes and spatially discriminative features that show promise for individualized precise treatment. This article summarized the key technical approaches, recent advances in spatial transcriptomics and its applications in breast cancer, and discusses the limitations of current spatial transcriptomics methods and the prospects for future development, with a view to advancing the application of this technology in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9484c611ac13b7e4460cc6b1b5eb2c6b25e4a9" target='_blank'>
              Spatial transcriptomics in breast cancer: providing insight into tumor heterogeneity and promoting individualized therapy
              </a>
            </td>
          <td>
            Junsha An, Yajie Lu, Yuxi Chen, Yuling Chen, Zhaokai Zhou, Jianping Chen, Cheng Peng, Ruizhen Huang, Fu Peng
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Malignant growth and survival of cancer is supported through reprogrammed lipid metabolism. In gliomas, genetic alterations can drive lipid metabolic dysregulation revealing therapeutic opportunities to exploit lipid metabolic vulnerabilities within genetically defined subsets of glioma tumors. However, it remains unclear whether inter- and intra-tumoral transcriptomic heterogeneity in gliomas relates to distinct lipid programs and potential dependencies. Here we set out to characterize pan-glioma lipid metabolic heterogeneity through genomic, transcriptomic, and lipidomic profiling of a large and diverse cohort of patient tumor samples. We define key axes of lipid metabolic variability across gliomas and identify relationships between lipid metabolic gene expression programs and glioma lipidomic profiles. Intriguingly, intersection of lipid gene expression signatures with single cell RNA sequencing revealed patterns of lipid metabolic heterogeneity that distinguish neurodevelopmental cellular states of gliomas, mirroring patterns of lipid metabolic diversity across cell types within the normal brain. Consequently, tumors with opposing cellular state enrichment have distinct lipid metabolism and environmental dependencies. Tumors enriched for Radial Glia-like states have high capacity for de novo fatty acid synthesis while tumors enriched for oligodendrocyte progenitor-like states are dependent on exogenous sources of lipids for survival and growth. These findings connect inter- and intra-tumoral heterogeneity of cellular states to variability in lipid metabolic reprogramming to reveal future avenues for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb17270c6d8bbb278033ffba4c666b5e6ba0ea5" target='_blank'>
              TMET-17. LIPID METABOLIC HETEROGENEITY OF GLIOMA CELLULAR STATES REVEALS ENVIRONMENTAL DEPENDENCIES
              </a>
            </td>
          <td>
            Nicholas A. Bayley, Jenna K. Minami, C. Tse, Henan Zhu, Jennifer Salinas, Cassidy Andrasz, Weihong Yan, Dimitri Cadet, Amir Borujerdpur, Kevin Williams, A. Bhaduri, S. Bensinger, L. Liau, T. Cloughesy, Thomas Graeber, David Nathanson
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b389c3620d8a50a335c990feb75f3a4d357ca8e6" target='_blank'>
              Integration of single-cell and spatial transcriptome sequencing identifies CDKN2A as a senescent biomarker in endothelial cells implicating hepatocellular carcinoma malignancy
              </a>
            </td>
          <td>
            Yue Ma, Chenhe Yi, Ning Cai, Jinhong Chen
          </td>
          <td>2024-11-06</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Cancer progression poses a clinical hurdle due to tumor heterogeneity that contributes to treatment resistance and immune evasion. While the adaptive immune system can effectively eliminate cancer in certain instances, tumor escape remains challenging. The immune microenvironment plays an important role in this process. One unanswered question is the role of ECM geometry, also known as Tumor Associated Collagen Signatures (TACS), and its effects on tumor immune recognition. In solid malignancies, ECM geometry has been associated with disease stage and T cell infiltration. Specifically, empirically observed ECM topologies are frequently categorized based on fiber arrangement: random fibers (TACS1), circumferentially aligned fibers (TACS2), and radially arranged fibers (TACS3). Even though a clear negative correlation between TACS and patient survival has been established, the specific roles and extent of TACS in T cell-driven cancer evolution remain uncharacterized. The precise mechanisms underlying how TACS influences cell movement are still not fully elucidated, with divergent opinions on whether and how the ECM mediates immune cell infiltration. At present, we currently lack a physical model relating the impact of TACS on the spatial co-evolution between an adaptive immune repertoire and a heterogeneous population of evading cancer cells. Here, we developed the EVO-ACT (EVOlutionary Agent-based Cancer T cell interaction) model, which is based on the Gillespie algorithm, to study the effects of TACS on tumor evolution and dynamical tumor-T cell interactions. Our analysis of tumor-T cell interactions across three TACS types reveals distinct migration patterns, with TACS3 showing the highest efficiency, TACS1 intermediate, and TACS2 the lowest. TACS impacts T cells more than tumors, with stronger chemokine gradients aiding T cell infiltration. Our model suggests that despite TACS2's limited efficiency, it cannot fully impede T cell infiltration. Varied migration efficiencies lead to diverse T cell infiltration patterns and immunoediting levels. TACS3 shows a benefit to immune recognition and higher survival, contrary to lower survival in TACS3 patients observed clinically in breast cancer. Our results indicate that only after introducing phenotypic changes, such as Epithelial-Mesenchymal Transition (EMT), and noting that EMT occurs exclusively after the TACS2-TACS3 progression, can our findings successfully explain the clinically observed trends. Considering mesenchymal tumor cells' ability to upregulate PD-1/PD-L1, we explore TACS-specific tumor evolution's impact on PD-1/PD-L1 inhibitor responses and patient survival. We observe higher tumor escape rates and lower survival in TACS3 compared to TACS2. Thus, TACS3 alone does not account for lower survival in breast cancer; TACS3-associated late-stage cancer byproducts like EMT and elevated checkpoint expression significantly decrease survival.
 Citation Format: Yijia Fan, Jason Tomas George. ECM architecture drives distinct evolutionary patterns of tumor immune escape and elimination in breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A031.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52d9e3eca63d450bccf177de70595f8faf5e6be" target='_blank'>
              Abstract A031: ECM architecture drives distinct evolutionary patterns of tumor immune escape and elimination in breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Jason Tomas George
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Radiopharmaceutical therapy (RPT) is an emerging prostate cancer treatment that delivers radiation to specific molecules within the tumor microenvironment (TME), causing DNA damage and cell death. Given TME heterogeneity, it's crucial to explore RPT dosimetry and biological impacts at the cellular level. We integrated spatial transcriptomics (ST) with computational modeling to investigate the effects of RPT targeting prostate-specific membrane antigen (PSMA), fibroblast activation protein (FAP), and gastrin-releasing peptide receptor (GRPR) each labelled with beta-emitting lutetium-177 (177Lu) and alpha-emitting actinium-225 (225Ac). Methods: Three ST datasets from primary tissue samples of two prostate cancer patients were obtained. From these datasets, we extracted gene expressions, including FOLH1, GRPR, FAP, and Harris Hypoxia, and estimated the proportions of different cell types—epithelial, endothelial, and prostate cancer (PC) cells—in the corresponding ST spots. We computed the spatiotemporal distribution of each RPT targeting PSMA, FAP, and GRPR at each ST spot by solving the partial differential equation (PDE) using a convection-reaction-diffusion (CRD) model, assuming similar pharmacokinetic parameters across all ligands. A well-established physiologically based pharmacokinetic (PBPK) model was used to simulate the input function in the prostate, carefully calibrated to deliver 10 Gy to the prostate tumor over 20 days. Dosimetry was estimated using the Medical Internal Radiation Dose (MIRD) formalism, applying the dose point kernels (DVK) method. The survival probability was estimated using the linear quadratic model, applied to both beta-emitting RPT labeled with 177Lu and 225Ac. A modified linear quadratic model was used to estimate the bioeffect of the alpha-emitting RPT. Results: The results demonstrate distinct dose-response and efficacy patterns across ST samples, with FAP-targeted RPT exhibiting limited effectiveness in tumor cell-rich areas compared to PSMA- and GRPR-targeted therapies. GRPR-targeted RPT showed higher resistance in hypoxic regions relative to the other therapies. Additionally, 225Ac-labeled RPT was more effective overall than 177Lu-labeled RPT, especially in areas with low cancer-cell fraction or high hypoxia. The findings suggest that a combination of 225Ac-labeled FAP- and PSMA-targeted RPT offers the best therapeutic strategy. Conclusion: The proposed method, which combines ST and computational modeling to determine the dosimetry and cell survival probability of RPT in the TME, holds promise for identifying optimal personalized RPT strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec87da15ea33d56d20dcda31386859834df4ba74" target='_blank'>
              Deciphering the effects of radiopharmaceutical therapy in the tumor microenvironment of prostate cancer: an in-silico exploration with spatial transcriptomics
              </a>
            </td>
          <td>
            Jimin Hong, Sungwoo Bae, L. Cavinato, Robert Seifert, M. Ryhiner, Axel Rominger, Kjell Erlandsson, M. Wilks, M. Normandin, Georges El-Fakhri, Hongyoon Choi, Kuangyu Shi
          </td>
          <td>2024-10-28</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Tumor is a complex and dynamic ecosystem formed by the interaction of numerous diverse cells types and the microenvironments they inhabit. Determining how cellular states change and develop distinct cellular communities in response to the tumor microenvironment is critical to understanding cancer progression. Tumour-associated macrophages (TAMs) are an important component of the tumour microenvironment and play a crucial role in cancer progression. This study was designed to identify cell-state-specific M2 macrophage markers associated with gastric cancer (GC) prognosis through integrative analysis of single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq data using a machine learning framework named EcoTyper. Results The results showed that TAMs were classified into M1 macrophages, M2 macrophages, monocytes, undefined macrophages and dendritic cells, with M2 macrophages predominating. EcoTyper assigned macrophages to different cell states and ecotypes. A total of 168 cell-state-specific M2 macrophage markers were obtained by integrative analysis of scRNA-seq and bulk RNA-seq data. These markers could categorize GC patients into two clusters (clusters A and B) with different survival and M2 macrophages infiltration abundance. Cell adhesion molecules, cytokine-cytokine receptor interaction, JAK/STAT pathway, MAPK pathway were significantly enriched in cluster A, which had worse survival and higher M2 macrophages infiltration. Conclusion In conclusion, this study profiles a single-cell atlas of intratumor heterogeneity and defines the cell states and ecotypes of TAMs in GC. Furthermore, we have identified prognostically relevant cell-state-specific M2 macrophage markers. These findings provide novel insights into the tumor ecosystem and cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d8a2fb2bd7548dde928bcb962ded6e0380b6c3" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA-Sequencing Based on EcoTyper Machine Learning Framework Identifies Cell-State-Specific M2 Macrophage Markers Associated with Gastric Cancer Prognosis
              </a>
            </td>
          <td>
            A-Kao Zhu, Guang-Yao Li, Fang-Ci Chen, Jia-Qi Shan, Yu-Qiang Shan, Chen-Xi Lv, Zhi-Qiang Zhu, Yi-Ren He, Lu-Lu Zhai
          </td>
          <td>2024-12-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Adult gliomas exhibit a high level of heterogeneity presenting a substantial challenge hampering development of targeted therapies. Bulk RNAseq studies have identified a mesenchymal gene expression signature across the glioma continuum, based on which a mesenchymal subtype of glioblastoma (MES-GBM) has been described. However, the mesenchymal phenotype of gliomas remains inconsistently defined and the origin of the mesenchymal signature has been debated). Single cell transcriptomic profiling has allowed resolution of gene expression profiles of tumor and non-tumor cells, leading to recent identification of cancer-associated fibroblasts in gliomas and improved understanding of the mesenchymal gene expression signature.



 We performed longitudinal scRNAseq analysis of organotypic human glioma slices from 8 patients cultured ex vivo for up to 3 weeks.



 We found 2 of the 6 IDHwt glioblastomas exhibit the mesenchymal phenotype based on Consensus Methylation Profiling (NIH) and scRNAseq analysis, annotated as Mes-1 and Mes-2. The gene expression profiles of mesenchymal glioblastomas show limited concordance with that of either the proneural or classical glioblastoma signatures or of cell types in physiological human brain, with Mes-2 deviating more extensively from non-mesenchymal gliomas. Instead, single cell transcriptomics suggests a hyper-secretory phenotype and overlaps broadly with that of cancer-associated fibroblasts (CAFs) and both express FGF2, other growth factors, and cytokines. Notably, Mes-2 cells express minimal to no levels of genes associated with brain cell lineages, such as PTPRZ1 and SCG3, which are abundantly expressed in other glioma subtypes.



 Taken together, our data reveals extended depth of glioma heterogeneity that distinguishes MES-GBM from other glioma subtypes and suggests a phenotypic and lineage linkage between MES-GBM and glioma-associated fibroblasts. Thus, new diagnostic and treatment strategies leveraging auto- and paracrine mechanisms underpinning tumor and non-tumor interactions including disruption of the FGF2-FGFR1 circuitry may apply specifically to MES-GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612fc5e5441e08af69c2465ff558c82864f962e0" target='_blank'>
              TMIC-56. LONGITUDINAL SINGLE CELL RNA SEQUENCING UNVEILS PHENOTYPIC CO-OPTION AND LINEAGE TRAITS OF MESENCHYMAL GLIOBLASTOMA AND GLIOMA-ASSOCIATED FIBROBLASTS
              </a>
            </td>
          <td>
            Jiyong Liang, Longfei Huo, Jihong Xu, Madhuri Saidane, Xiaohua Chen, Katherine Zhang, Pratibha Sharma, Truc Kuo, Lisa M. Norberg, Gregory Buchold, Chetna Wathoo, Malcolm F. McDonald, Frederick F Lang, Ganesh Rao, Vinay K. Puduvalli
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="

 The demand for high quantities of donor hematopoietic stem cells (HSCs) remains a significant hurdle for life-saving HSC transplantation in clinical settings. Directed differentiation of pluripotent stem cells (PSCs) to HSCs offers a promising strategy that theoretically could provide unlimited autologous HSCs for both research and clinical application. An in-depth understanding of embryonic HSC development would facilitate the PSC to HSC induction.
 The first transplantable HSC emerges in the dorsal aorta of aorta-gonad-mesonephros (AGM) region through stagewise development of hemogenic endothelial cell (HEC) specification and the endothelial-to-hematopoietic transition (EHT). These processes dynamically occur at the ventral, but not dorsal, side of the aorta within a narrow time window (mouse embryonic day [E] 9.5-12.5), indicating an essential role of the unique AGM microenvironment in HSC development. However, a comprehensive profiling of niche cells and their emanated signals in the AGM region at spatial-resolved and single-cell levels remains elusive. To address this question, we conducted scRNA-seq and sequencing-based spatial transcriptomics (Curio-seq) on mouse embryonic AGM-containing trunk sections at E10.5, E11.5 and E12.5, capturing key timepoints of HSC development. While the dorsal side of the aorta is predominantly occupied by a Nfgr+ cell cluster expressing BMP signals, the ventral side is composed of a complex cellular architecture and signals (e.g., WNT, NOTCH and retinoic acid). Specifically, a cell population co-expressing Cdh2 (encodes N-cadherin) and Pdgfra (named N-cad+ MSCs) forms a single cell layer sandwiched between aortic endothelial cells (AECs) and Wt1+ mesonephric cells. Importantly, all pre-HSCs identified in our study are in direct contact with N-cad+ MSCs. We next utilized CellChat and StringDB to predict ligand-receptor interactions and signaling crosstalk between niche cells and pre-HSCs. This reveals previously unreported interactions between the N-cad+ MSCs and pre-HSCs via multiple signal axes. Strikingly, these interactions change dynamically during E10.5-E12.5, with the interactions via Dlk1-Notch1,Tgfb1-Tgfbr1 and Jag1-Notch1 reaching the peak at E10.5, E11.5 and E12.5, respectively. Overall, the Curio-seq data profiled the dynamic and asymmetric distribution of cell identities and signals in the AGM region at a resolution of 10 µm.
 We next conducted image-based spatial transcriptomics MERFISH on E11.5 trunk sections by customizing RNA probes for 140 genes selected from the Curio-seq analysis, which included cell markers and signaling molecules. The MERFISH data effectively validated the spatially resolved niche cells and their associated signals in the AGM region at the single-cell level, as revealed by Curio-seq assay. For instance, it confirmed that the N-cad+ MSCs were enriched at the ventral side of the aorta, in direct contact with pre-HSCs, and served as the primary source of Jag1 and Cxcl12 around the aorta.
 We hypothesized that N-cad+ MSCs play a crucial role in HSC development due to their unique position and intricate interactions with pre-HSCs. To investigate this, we generated NcadCreER; Jag1flox/flox mouse model to conditional knockout Jag1 from N-cadherin-expressing cell at E9.5 and collected E11.5 AGM cells for transplantation assays. The recipient mice in the Jag1 knockout group exhibited higher peripheral blood CD45.2 chimerism at 16 weeks compared to the mice in the wild-type group (24.2% Vs 0.14%), indicating that N-cad+ MSC derived Jag1 serves as a negative regulator of HSC development. To further understand the underlying mechanisms, we are performing scMultiome-seq to co-profile the transcriptional and epigenetic landscapes of the HECs and pre-HSCs from wild-type and mutant embryos.
 In summary, we employed a combination of sequencing-based and imaging-based spatial transcriptomics technologies to comprehensively characterize the dynamic, spatial-resolved profiles of niche cells and their associated signals in the AGM region at the single-cell level. Functionally, we revealed an inhibitory role of N-cad+ MSC derived Jag1 in HSC development, thus validating the results obtained from the spatial transcriptomics analyses. This study enhances our understanding of the niche of embryonic hematopoiesis and provides insight into developing novel methods for PSC to HSC induction.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78aa25c27ce21a62e52ae969658d2bcff9b26593" target='_blank'>
              Spatial Transcriptomics Characterize Dynamic and Polarized Niche for Embryonic Hematopoiesis at Single-Cell Level
              </a>
            </td>
          <td>
            Xinjian Mao, Ning Zhang, Xi C He, Laura Bennett, Kate E. Hall, Kaitlyn Petentler, Allison Scott, Seth Malloy, Jeff Haug, Hua Li, Anoja Perera, Jay Unruh, Brian Slaughter, Paul Trainor, Nancy Speck, Linheng Li
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a heterogeneous pediatric brain tumor with variable clinical outcomes. Since current treatments facing limitations due to tumor resistance, less than 10% of relapsed patients survive beyond five years, with poorly understood recurrence mechanisms. To obtain cellular diversity and genetic characters in primary and recurrent MB, we conducted comprehensive analyses utilizing the single-cell/nucleus RNA sequencing (sc/snRNA-seq), snATAC-seq and spatial transcriptomics, complemented with bulk RNA sequencing and matched primary-recurrent whole exon profiles. SHH and Group_3 subgroups revealed distinct cellular subsets, including cycling, differentiated and quiescent tumor cells. Recurrent or disseminated MB displayed a dramatically different cellular ecosystem compared to primary tumors, with varying proportions of shared cell types. Local recurrence exhibited a higher enrichment of cycling tumor cells, while differentiated subset was notably present in disseminated lesions, indicating that local recurrences harbored the stronger proliferative capacity, whereas disseminations relatively well differentiated. Additionally, chromosomal alterations were assessed, reflecting distinct genetic subclones, such as chr7q gain and chr11 loss in Group_3 disseminations. Notably, the emergence of a subpopulation termed “high cellular plasticity (HCP)” during MB progression was noted, characterized by dynamic adaptability and stemness properties. As validated, HCP state was associated with increased chromatin accessibility, contributing to tumor recurrence and immune microenvironment remodeling. Functionally, inhibiting markers associated with HCP cells, such as PTPRZ1, efficiently suppressed MB metastasis in preclinical models. In conclusion, our findings fill the critical knowledge gaps in understanding the cellular diversity, chromosomal alterations and biological dynamics in MB recurrence and dissemination, offering potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1e99ed55df676c0aad6e2ec0b1d04b50f24de8e" target='_blank'>
              EPCO-02. HIGH CELLULAR PLASTICITY STATE OF HUMAN MEDULLOBLASTOMA LOCAL RECURRENCE AND DISTANT DISSEMINATION
              </a>
            </td>
          <td>
            Hailong Liu, Jiankang Li, T. Jiang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Pre-leukemic disorders, such as MDS or MPN, are well-established models for characterizing disease evolution and transformation into leukemia, allowing the study of clinically-relevant key genetic events such as loss-of-function TET2 mutations. However, the mechanisms by which a TET2-mutant clone gains competitive advantage over its wild-type (WT) counterparts remains unclear due to challenges in modelling cell competition efficiently at scale.
 Here, we report a novel in vitro model of single-cell derived clonal competition, demonstrating the existence of TET2-mutant competitor clones that impair wild-type expansion. Leveraging the recently reported PVA-based expansion cultures (Nature, 2019), we developed a cell competition culture protocol which allows single cells to be grown and the resulting clones to be split.
 To validate the system, single EPCR+ CD45+ Sca-1high CD150+ CD48- (ESLAM Sca+) WT cells were expanded for 14 days, after which clones were split and distributed in equivalent cell numbers across multiple wells. 14 days later, flow cytometric analysis and in vivo functional assays showed that the behavior of the original clone was maintained.
 Using the CD45.1/CD45.2 congenic system to discriminate between WT (CD45.1) and TET2 clones (CD45.2) we set up clonal competition cultures: Single ESLAM Sca+ cells from WT and TET2 knockout mice (Tet2-/-) were isolated and expanded for 14 days. Clones were split and distributed across 36 competition wells, each with 50% WT cells from one clone and 50% Tet2-/- cells from another clone, creating a matrix of 3 x 3 competition cultures alongside a WT and Tet2-/- control for each clone. Flow cytometry analysis of clonal composition revealed the presence of a previously unidentified type of TET2 “super competitor” clone. These super competitors impaired the growth and expansion of wild-type clones, and in some instances, completely depleted their WT counterparts. Albeit rare (1 super competitor identified in approximately every 12 analyzed TET2 competitor clones, n = 192), TET2 super competitors consistently impair or completely suppress the expansion of their wild-type counterparts across all the competition wells.
 Interestingly, the data showed that the HSC frequency within the TET2 super competitor clones was lower than that in WT and TET2 competitor clones, denoting that suppression of WT competitors might be attributed to extrinsic factors, rather than being driven by the HSCs themselves. Transcriptomic analysis of the different clone types highlighted a molecular signature of inflammation and iron metabolism.
 To further explore this phenomenon, we harvested media from the 28-day competition cultures, and performed Data-Independent Acquisition Mass Spectrometry (DIA-MS). In agreement with the RNA-seq data, DIA-MS analysis revealed the secretion of inflammatory molecules into the media by enhanced TET2 super competitor clones, including Ifit3m, Cd14, and C3, which are central to the innate immune response and macrophage function.
 Thus, our data have demonstrated that, while the proportion of phenotypic HSCs within TET2 super competitor clones is reduced, secretion of inflammatory molecules potentiates the expansion of such clones and hinders the self-renewal potential of the WT rival clones.
 In conclusion, following the development of a novel in vitro clonal level competition model, we have identified a novel type of pre-leukemic competitor clone. This enables the investigation of molecular and cellular mechanisms with clinical relevance, to better understand how malignant HSCs enhance self-renewal and suppress normal HSC clones in blood cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ba6d7de81b002e6963a30d3deef66f881c23f2f" target='_blank'>
              A Novel in Vitro Model of Pre-Leukemic Competition Reveals “Super-Competitive” TET2-Mutant Hematopoietic Stem Cell Clones
              </a>
            </td>
          <td>
            Lilia Cabrera-Cosme, Juan Rubio-Lara, Ellie Bennett, Joanna Milek, M. Jassinskaja, Sandy MacDonald, Karen Hogg, Nicola K. Wilson, Berthold Gottgens, David Kent
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The development of T cell receptor-engineered T cells (TCR-T) targeting intracellular antigens is a promising strategy for treating solid tumors; however, the mechanisms underlying their effectiveness remain poorly understood. In this study, we employed advanced techniques to investigate the functional state of T cells engineered with retroviral vectors to express a TCR specific for the NY-ESO-1 157-165 peptide in the HLA-A*02:01 context. Flow cytometry revealed a predominance of naïve T cells. Gene expression profiling using NanoString technology revealed upregulation of genes encoding chemokine receptors CCR2 and CCR5, indicating enhanced migration towards tumor sites. In the SK-Mel-37 xenograft model, these transduced T cells achieved complete tumor eradication. Furthermore, single-cell RNA sequencing (scRNA-seq) conducted 14 days post-TCR T cell infusion provided a comprehensive analysis of the in vivo adaptation of these cells, identifying a distinct subset of CD8+ effector T cells with an NK cell-like gene expression profile. Our findings indicate that NY-ESO-1 TCR-transduced T cells have the potential to mediate dual antitumor effects through both antigen-independent NK-like and antigen-specific CTL-like responses. This study underscores the potential of NY-ESO-1 TCR-T cells as potent tumor-eradicating agents, highlighting the importance of harnessing their versatile functional capabilities to refine and enhance therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c35da049d73d1a38187e30a25bcb51a1113dcd7" target='_blank'>
              Decoding NY-ESO-1 TCR T cells: transcriptomic insights reveal dual mechanisms of tumor targeting in a melanoma murine xenograft model
              </a>
            </td>
          <td>
            A. Alsalloum, S. Alrhmoun, O. Perik-Zavosdkaia, M. Fisher, M. Volynets, J. Lopatnikova, R. Perik-Zavodskii, J. Shevchenko, Julia Philippova, Olga Solovieva, Evgenii Zavjalov, V. Kurilin, H. Shiku, A. Silkov, S. Sennikov
          </td>
          <td>2024-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa47d77b71a7337ad0ecaef200be807283fa4101" target='_blank'>
              Heterogeneity-driven phenotypic plasticity and treatment response in branched-organoid models of pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            A. Papargyriou, Mulham Najajreh, David P Cook, Carlo Maurer, S. Bärthel, H. Messal, Sakthi K Ravichandran, Till Richter, Moritz Knolle, Thomas Metzler, Akul R Shastri, R. Öllinger, Jacob Jasper, Laura Schmidleitner, Surui Wang, C. Schneeweis, H. Ishikawa-Ankerhold, T. Engleitner, Laura Mataite, Mariia Semina, Hussein Trabulssi, S. Lange, Aashreya Ravichandra, Maximilian Schuster, S. Mueller, K. Peschke, A. Schäfer, S. Dobiasch, S.E. Combs, Roland M. Schmid, Andreas R. Bausch, R. Braren, I. Heid, C. Scheel, Günter Schneider, Anja Zeigerer, Malte Luecken, K. Steiger, G. Kaissis, J. van Rheenen, F. Theis, Dieter Saur, R. Rad, Maximilian Reichert
          </td>
          <td>2024-12-10</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Objective This study aimed to explore the heterogeneity of tumor endothelial cells (TECs) in hepatocellular carcinoma (HCC) and their role in tumor progression, with the goal of identifying new therapeutic targets and strategies to improve patient prognosis. Methods Single-cell RNA sequencing data from nine primary liver cancer samples were analyzed, obtained from the Gene Expression Omnibus (GEO) database. Data preprocessing, normalization, dimensionality reduction, and batch effect correction were performed based on the Seurat package. HCC cell types were identified using uniform manifold approximation and projection (UMAP) and cluster analysis, and the different cell types were annotated using the CellMarker database. Pseudotime trajectory analysis was conducted with Monocle to explore the differentiation trajectory of TECs. MAPK signaling pathway activity and copy number variations (CNV) in TECs were analyzed in conjunction with data from The Cancer Genome Atlas (TCGA), the trans-well and wound healing assay was used for cell invasion and migration activity assessment. Results Two subgroups of TECs (TECs 1 and TECs 2) were identified, exhibiting distinct functional activities and signaling pathways. Specifically, TECs 1 may be involved in tumor cell proliferation and inflammatory responses, whereas TECs 2 is not only involved in cell proliferation pathways, but also enriched in pathways such as metabolic synthesis. Pseudotime analysis revealed dynamic changes in TECs subgroups during HCC progression, correlating specific gene expressions (such as PDGFRB, PGF, JUN, and NR4A1). Subsequently, the JUN gene was predicted by performing binding sites and was shown to act as a transcription factor that may regulate the expression of the PGF gene. CNV analysis highlighted key genes and pathways in TECs that might influence HCC progression, and the PGF as key regulatory factor mediated cell proliferation and migration. Conclusion The study revealed the heterogeneity of TECs in HCC and their potential roles in tumor progression, offering new perspectives and potential therapeutic targets for HCC molecular mechanisms. The findings emphasize the importance of further exploring TECs heterogeneity for understanding HCC pathogenesis and developing personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db89c0c7a388ba83d2034d24b60625ad1b46b81d" target='_blank'>
              Exploring tumor endothelial cells heterogeneity in hepatocellular carcinoma: insights from single-cell sequencing and pseudotime analysis
              </a>
            </td>
          <td>
            Jiachun Sun, Shujun Zhang, Yafeng Liu, Kaijie Liu, Xinyu Gu
          </td>
          <td>2024-10-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The complexity of issues in cancer research has led to the introduction of powerful computational tools to help experimental in vivo and in vitro methods. These tools, which typically focus on studying cell behavior and dynamic cell populations, range from systems of differential equations that are solved numerically to lattice models and agent-based simulations. In particular, agent-based models (ABMs) are increasingly used due to their ability to incorporate multi-scale features, ranging from the individual to the population level. This approach allows for the combination of statistically aggregated assumptions with individual heterogeneity. In this work, we present an ABM that simulates tumor progression in a colonic crypt, to provide an experimental in silico environment for testing results achieved in traditional laboratory research and developing alternative scenarios of tumor development. The model also allows some speculations about causal relationships in biologically inspired systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85ab71cc166cd3a08737a8b853ac482df171fb9d" target='_blank'>
              Exploring the Role of Genetic and Environmental Features in Colorectal Cancer Development: An Agent-Based Approach
              </a>
            </td>
          <td>
            Marco Ledda, Alessandro Pluchino, Marco Ragusa
          </td>
          <td>2024-10-30</td>
          <td>Entropy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent advancements in in situ molecular profiling technologies, including spatial proteomics and transcriptomics, have enabled detailed characterization of the microenvironment at cellular and subcellular levels. While these techniques provide rich information about individual cells’ spatial coordinates and expression profiles, extracting biologically meaningful spatial structures from the data remains a significant challenge. Current methodologies often rely on unsupervised clustering followed by cell type annotation based on differentially expressed genes within each cluster and most of the time will require other information as the reference (e.g., HE-stained images). This is labor-intensive and demands extensive domain knowledge. To address these challenges, we propose a supervised graph contrastive learning framework, MARVEL. MARVEL is a supervised graph contrastive learning method that can effectively embed local microenvironments represented by cell neighbor graphs into a continuous representation space, facilitating various downstream microenvironment annotation scenarios. By leveraging partially annotated examples as strong positives, our approach mitigates the common issues of false positives encountered in conventional graph contrastive learning. Using real-world annotated data, we demonstrate that MARVEL outperforms existing methods in three key microenvironment-related tasks: transductive microenvironment annotation, inductive microenvironment querying, and the identification of novel microenvironments across different slices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/152ed49b0be6d463a8a4a25e29a66310bcce4454" target='_blank'>
              MARVEL: Microenvironment Annotation by Supervised Graph Contrastive Learning
              </a>
            </td>
          <td>
            Yan Cui, Hongzhi Wen, Robert Yang, Xi Luo, Hui Liu, Yuyin Xie
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Spatial transcriptomics is revolutionizing our understanding of cellular dynamics and interactions in the tumor microenvironment (TME). By mapping distinct cellular populations and their interactions, we can gain valuable insights into tumor behavior. To investigate these dynamics in breast cancer, we conducted Xenium In-situ spatial transcriptomics on twenty formalin-fixed paraffin-embedded (FFPE) samples taken from eight patients undergoing neoadjuvant chemotherapy. These samples have been taken pre-treatment, mid-treatment and post-treatment from four cases with pathological complete response (pCR), and four cases with residual disease (RD) at time of surgery. Our analyses have found distinct cellular composition changes between pCR and RD cases. Patients with RD demonstrated an increase in endothelial, perivascular-like (PVL) and epithelial cells from pre- to post-treatment. This was accompanied by a decrease in myeloid and malignant epithelial cells at mid-treatment. In contrast, patients with pCR demonstrated increases in cancer-associated fibroblasts (CAFs) and myeloid cells, with reductions in plasmablasts and cancer epithelial cells at mid-treatment. This suggests that an increase in CAF and myeloid cells observed between pre-treatment and mid-treatment samples may be predictive of response. We performed ligand-receptor (LR) analyses, which demonstrated diverse myeloid interactions in RD cases and diverse CAF interactions in pCR cases in pre-treatment samples. Mid-treatment samples exhibited a decrease in interactions for CAFs, myeloid and T cells in pCR and RD cases. The most frequently identified CAF LR interactions, as senders, were between TGFB1/TGFB3 and TGFBR1/TGFBR2, while the most frequently identified myeloid LR interactions were between PDCD1LG2 - PDCD1 (encoding PDL2 & PD1), CCL8 - ACKR1 (encoding Atypical Chemokine Receptor 1) and CCL2 - ACKR1. Our findings indicate that the dynamic changes in cellular composition and interactions within the TME differs between responders and non-responders. To further explore these differences, we plan to perform spatially constrained analyses to identify unique cellular niches that may serve as potential biomarkers for informing novel therapeutic strategies.
 Citation Format: Jeremy Mo, Hanyun Zhang, Kate Harvey, Alexander Swarbrick. Spatial transcriptomics unveils disparities in tumor microenvironment dynamics between breast cancer responders and non-responders [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d85f0bd80cb01173f30a8b1b70f6e0a4a64eb5" target='_blank'>
              Abstract C040: Spatial transcriptomics unveils disparities in tumor microenvironment dynamics between breast cancer responders and non-responders
              </a>
            </td>
          <td>
            Jeremy Mo, Hanyun Zhang, K. Harvey, Alexander Swarbrick
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Glioma evolution is governed by a multitude of dynamic interactions between tumor cells and heterogenous neighboring, non-cancerous cells. This complex ecosystem, termed the tumor microenvironment (TME), includes diverse immune cell types that have gained increasing attention for their critical and paradoxical roles in tumor control and tumorigenesis. Recent work has revealed that the cellular composition and functional state of immune cells in the TME can evolve extensively depending on the tumor stage and intrinsic features of surrounding glioma cells. Concurrently, adaptations to the glioma cellular phenotype, including activation of various cellular states, occur in the context of these immune cell alterations. In this review, we summarize important features of the immune TME that play key roles during each stage of glioma progression, from initiation to immune escape, invasion and recurrence. Understanding the complex interplay between tumor and immune cells is critical for the development of effective immunotherapies for glioma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6197e8609e0a64b363d0e798255e45455eaf7c8" target='_blank'>
              Co-evolution of glioma and immune microenvironment
              </a>
            </td>
          <td>
            Mahmoud M Elguindy, Jacob S Young, Winson S Ho, R. Lu
          </td>
          <td>2024-12-03</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Glioblastoma remains one of the deadliest brain malignancies. First-line therapy consists of maximal surgical tumor resection, chemotherapy, and radiotherapy. Malignant cells escape surgical resection by migrating into the surrounding healthy brain tissue, where they give rise to the recurrent tumor. Gene expression profiling allows glioblastoma tumor cores to be classified into mesenchymal, proneural, and classical subtypes, each with distinct genetic alterations and cellular compositions. In contrast, the adjacent brain parenchyma where infiltrating malignant cells escape surgical resection is less characterized in patients. Using single-cell and multicellular resolution spatial transcriptomics on tissues from both the tumor core and infiltrated brain tissue (n = 11), we compared the transcriptional profiles of malignant cells in these regions. Malignant cells near regions with microvascular proliferation showed increased expression of genes related to vascular homeostasis. Within tumor cores, proneural and mesenchymal subtypes exhibited distinct spatial gene expression patterns, although these differences were reduced in the infiltrated brain tissue, apart from gene expression due to chromosomal alterations specific to the proneural subtype. We identified two transcriptional patterns of brain infiltration, with the dominant pattern showing a transition from mesenchymal to proneural states. This transition was accompanied by increased expression of genes linked to neurodevelopmental pathways and glial cell differentiation. Additionally, one gene module upregulated in infiltrating malignant cells was predictive of poor patient survival and enriched for genes associated with differentiated neuronal cells. Together, our findings provide an updated view of the spatial landscape of glioblastomas and infiltrated brain tissue, furthering our understanding of the malignant cells that invade healthy brain. This insight offers new avenues for targeted therapy after surgical resection of the primary tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f11ef6dd4e17522c0f5c98bcdd5e7ccdaddf85f0" target='_blank'>
              ANGI-08. SPATIAL TRANSCRIPTOMIC COMPARISONS OF GLIOBLASTOMA TISSUE REVEAL UPREGULATION OF NEURODEVELOPMENTAL PATHWAYS AND SYNAPTIC GENES IN INFILTRATING TUMOR CELLS
              </a>
            </td>
          <td>
            D. Harwood, V. Pedersen, N. Bager, A. Schmidt, T. Stannius, A. Areškevičiūtė, Knud Josefsen, D. Nørøxe, David Scheie, Hannah Rostalski, M. Lü, A. Locallo, U. Lassen, F. Bagger, J. Weischenfeldt, D. Heiland, Kristoffer Vitting-Seerup, S. Michaelsen, B. Kristensen
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 The organized layered structure of the human neocortex contrasts sharply with the chaotic architecture of malignant CNS tumors. The heterogenous nature of glioblastoma (GBM) poses significant challenges to research and treatment, given that effective strategies require consistent biological patterns. Here, we leverage recent advances in spatial biology, integrating spatially resolved transcriptomics and metabolomics to identify drivers of GBM-specific architectural evolution. We performed spatial metabolomics (MALDI) and transcriptomics (Visium n=43 and MERFISH n=25) on glioblastoma patients, with computational analysis conducted using the SPATA2 R package. Our findings reveal that most spatial recurrent patterns in glioblastoma emerge in a gradient-like organization, particularly influenced by proximity to areas of necrosis and hypoxia. Samples without hypoxic regions demonstrated less spatial organization. Metabolomic and transcriptomic profiling indicated enhanced responses to hypoxia, reduced cell proliferation, and elevated expression indicative of the S phase in cell division near necrotic regions. The vicinity to hypoxic areas was associated with increased extracellular matrix remodeling and epithelial-to-mesenchymal transition, suggesting that the invasive capabilities of GBM cells originate from these hypoxic niches. By further integrating spatially resolved TCR-sequencing, we identified that hypoxic gradients drive defined T cell responses with a higher likelihood of T cell dysfunction within a distance of 235 µm (sd: 132 µm) to hypoxia. This suggests that the presence of necrosis, hypoxia, and hypoxia-associated metabolites could serve as potential predictive markers for T-cell behavior and exhaustion in glioblastoma. Our study offers an in-depth analysis of the influence of two prevalent niches in glioblastoma: necrosis and hypoxia. We reveal that the complex architecture of glioblastoma is not solely dictated by genetic randomness and stochastic occurrences but is significantly influenced by spatial proximity to key factors like necrosis and hypoxia. This insight contributes to an evolving model that seeks to unravel the intricate biology underpinning this formidable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98b4118e50fd65f0b3670d52a4f2ef27daee8cbf" target='_blank'>
              BIOS-07. GRADUAL METABOLIC ALTERATIONS RESTRUCTURE THE SPATIAL ARCHITECTURE AND IMMUNITY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Kueckelhaus, Jasim Kada-Benotmane, O. Schnell, Jürgen Beck, Giulia Villa, D. Heiland
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 We’ve curated a comprehensive multi-omics database of medulloblastoma (MB) serving as a resource to understand spatiotemporal organization and guide therapeutic strategies. Herein, our analysis encompasses single-nucleus transcriptome (n = 51), chromatin accessibility (n = 46), and spatial transcriptomic profiles (n = 31) from human MB samples and patient-derived xenograft lines spanning four subgroups, augmented by bulk RNA (n = 322), whole-genome short-sequencing (n = 279), and bisulfite sequencing (n = 300) from matched samples. We delineate the malignant cellular hierarchy, identifying MB cancer stem-like cells (MB-CSCs), cycling cells, and more differentiated populations. Gene signatures of different cell subpopulations strongly correlate to clinical outcomes, while spatial character of patient-derived materials emphasizes the geographically heterogeneous nature of MB. Examination of somatic copy number variants (CNVs) and transcriptional signatures at single-nucleus resolution reveals geographically defined subclones harboring distinct CNVs, suggesting a competitive agglomeration of co-existing multi-clones for MB. Alongside the trajectorial transition, we observe distinct chromosomal alterations in apical MB-CSCs compared to more differentiated cells, indicative of extensive subclone-primed spatiotemporal evolution. Further analysis of reagent-gene interactions of subclonal lineages contributes to predicting the druggable candidates for treatment optimization. Integration of spatial architecture data highlights the recapitulation of subclone-determined niches through colocation with identified malignant lineages and their own inflammatory or fibrotic adaptions. Taken together, this comprehensive database, comprising five or six kinds of datatypes on the same MB cases, facilitates combinatorial analyses across different genomic modalities, offering genetic and geographic insights into therapeutic advancements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c132829ee848aad60862b63f0ed4457631be0db6" target='_blank'>
              EPCO-01. FROM HIERARCHY TO GEOGRAPHY: MULTI-CLONES DRIVE THE CELLULAR DIVERSITY AND EVOLUTION OF HUMAN MEDULLOBLASTOMA
              </a>
            </td>
          <td>
            Hailong Liu, Jiankang Li, T. Jiang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="CDK4/6 inhibitors such as palbociclib block cell cycle progression and improve outcomes for many ER+/HER2- breast cancer patients. Unfortunately, many patients are initially resistant to the drug or develop resistance over time in part due to heterogeneity among individual tumor cells. To better understand these mechanisms of resistance, we used multiplex, single-cell imaging to profile cell cycle proteins in ER+ breast tumor cells under increasing palbociclib concentrations. We then applied spherical principal component analysis (SPCA), a dimensionality reduction method that leverages the inherently cyclical nature of the high-dimensional imaging data, to look for changes in cell cycle behavior in resistant cells. SPCA characterizes data as a hypersphere and provides a framework for visualizing and quantifying differences in cell cycles across treatment-induced perturbations. The hypersphere representations revealed shifts in the mean cell state and population heterogeneity. SPCA validated expected trends of CDK4/6 inhibitor response such as decreased expression of proliferation markers (Ki67, pRB), but also revealed potential mechanisms of resistance including increased expression of cyclin D1 and CDK2. Understanding the molecular mechanisms that allow treated tumor cells to evade arrest is critical for identifying targets of future therapies. Ultimately, we seek to further SPCA as a tool of precision medicine, targeting treatments by individual tumors, and extending this computational framework to interpret other cyclical biological processes represented by high-dimensional data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57de25c3330f7fce9f46a524989a6ef4e9398a4c" target='_blank'>
              Spherical Manifolds Capture Drug-Induced Changes in Tumor Cell Cycle Behavior.
              </a>
            </td>
          <td>
            Olivia Wen, Samuel C Wolff, Wayne Stallaert, Didong Li, Jeremy E. Purvis, Tarek M. Zikry
          </td>
          <td>2024-11-21</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Drug tolerant persisters (DTPs) drive cancer therapy resistance by temporarily evading drug action, allowing multiple routes to eventual permanent resistance. Despite clear evidence for DTPs, the timing of their emergence, proliferative nature, and how their population dynamics arise from measured single-cell kinetics remain poorly understood. Here we use time-lapse microscopy data from two cancer cell lines, integrating single-cell and population measurements, to develop a quantitative description of drug persistence. Contrary to the expectation that increasing levels of genotoxic stress should lead to slower times to division and faster times to death, we observe minor changes in the single-cell intermitotic and death time distributions upon increasing cisplatin concentration. Yet, population decay rates increase 3-fold, suggesting a surprising independence of the overall dynamics from the measured birth and death rates. To explain this phenomenon, we argue that the observed lineage correlations and concentration-dependent decay rates imply cell-state dependent fate choices made both pre and post-cisplatin as opposed to just post-drug birth/death rate-based competitive fate choices. We demonstrate that these cell-states, present in the ancestors of DTP and sensitive cells, exhibit no difference in cycling speed and are inherited across 2-3 cellular generations. A stochastic model implementing these rules simultaneously recapitulates the observed decay rates and cell-fate correlations, also explaining how pre-drug fate decisions are consistent with barcoding experiments where barcode diversity remains unchanged after drug administration. Our results provide a powerful perspective on drug tolerance based on general arguments, without requiring knowledge of the underlying molecular architecture of the heterogeneous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6873fb18b12044e54ea04b559a8e9ceb924ff56" target='_blank'>
              Inheritable cell-states shape drug-persister correlations and population dynamics in cancer cells
              </a>
            </td>
          <td>
            Anton Iyer, Adrián E. Granada, Shaon Chakrabarti
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Breast cancer has the highest global incidence among all cancers, affecting more than 2 million individuals annually. Despite the availability of new drugs and novel treatment combinations, it is postulated that the incidence and mortality of breast cancer will rise by 40.8% and 51.9% respectively by 2040. Such dire statistics are associated with the clonal evolution of cancer cells that leads to therapeutic resistance and consequent relapse in breast cancer patients. On the other hand, the tumor microenvironment (TME) comprising of tumor cells, cancer-associated immune cells, re-programmed stromal cells, and the extracellular matrix (ECM) creates an immunosuppressive niche facilitating immune evasion. This review focuses on a critical cellular component of the tumor microenvironment, the tumor-associated macrophages (TAMs) in breast cancer immunotherapy. Macrophages are inherently plastic and can convert from an anti-tumor M1 phenotype to a pro-tumor M2 phenotype based on microenvironmental cues. Cancer cells facilitate these cues, allowing the tumor-associated macrophages to gain M2 phenotype and mediate immune evasion. Therefore, knowledge of the distinct role of tumor-associated macrophages in immune evasion can help design therapeutics such as engineered macrophages, M2 targeting drugs, and novel macrophage-mediated drug delivery strategies for long-term survival in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6e4305fc222b5a36786a300bfa76f97eae0c28" target='_blank'>
              Reprogramming the tumor microenvironment – macrophages emerge as key players in breast cancer immunotherapy
              </a>
            </td>
          <td>
            Ana Sami, Afsheen Raza
          </td>
          <td>2024-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Recent evidence suggests epigenetic heterogeneity may be the primary driver of cell phenotype and treatment resistance in glioblastoma (GBM). Epigenetic heterogeneity is defined by alterations of chromatin histone modifications with activating marks such as H3K27 acetylation and repressive marks such as H3K27 trimethylation. Combinations of these histone marks govern the activity of functional genomic elements such as enhancers or promoters. Single-cell (“sc-”) technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we report the development of an optimized method for performing tissue processing and simultaneous single-cell epigenetic and transcriptomic analyses in flash frozen GBM samples.



 Building on our prior work, we developed a new protocol, multiomic single-cell cleavage-under-targets and tagmentation, or “mscCUT&TAG”. We optimized nuclei isolation, cleanup and permeabilization from flash frozen GBM and reaction conditions for the scCUT&TAG portion of the assay. Critically, these changes maintained high-quality, intact nuclei in suspension while limiting well-described technical issues such as nuclei clumping and sample loss. Furthermore, we incorporated the ability to multiplex both patient samples and histone marks.



 We applied 10X scRNA-seq, 10X Multiome (RNA+ATAC-seq), and mscCUT&TAG for five histone marks to the same GBM sample, enabling an integrated evaluation of the tumor’s transcriptome, chromatin accessibility and epigenetic regulation at single-cell resolution. mscCUT&TAG enabled the first assessment of single-cell genome segmentation in GBM. We integrated across modalities to identify specific tumor cell populations, cell-type specific coordinated changes in epigenetics and transcriptome, and define functional regulatory elements.



 We developed a novel multiomic protocol, mscCUT&TAG, that allows for a comprehensive analysis of epigenetic heterogeneity in GBM. We anticipate this protocol will enhance our understanding of fundamental biology and potential therapeutic avenues in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbc234be87fcb8f1eb7cc14aaf65833786391806" target='_blank'>
              EPCO-19. MULTIOMIC PROFILING OF EPIGENETIC HETEROGENEITY IN GLIOBLASTOMA AT SINGLE-CELL RESOLUTION USING MULTIOMIC SINGLE-CELL CUT&TAG
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, Eliese Moelker, P. Fecci, A. Patel
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc282ab7c49fa5f1e3334e1737972fbc08f7f900" target='_blank'>
              Single cell analysis identified a basal cell transition state associated with the development and progression of bladder cancer
              </a>
            </td>
          <td>
            Yang Li, Pengjie Shi, Yuhong Ding, Zhipeng Yao, Lilong Liu, Junyi Hu, Zhenghao Liu, Jinxu Li, Ke Chen, Yaxin Hou
          </td>
          <td>2024-11-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) defines a heterogeneous set of neuroepithelial cancers of the posterior fossa. The MB tumor microenvironment (MB-TME) influences tumor progression and therapy response and has emerged as a growing target for novel therapeutic approaches. Prior single-cell analysis has characterized the MB-TME composition but does not reveal the spatial orientation of components. To address this gap, we performed 10X Visium spatial sequencing on sixteen pediatric MB samples obtained at surgical resection with samples representing the four molecular subtypes (WNT, N=1; SHH, N=6; Group 3, N=2; Group 4, N=5) and two matching samples at diagnosis and relapse. Spatial voxel clustering yielded clusters corresponding to malignant, immune, and stromal components of the TME, including tumor-associated astrocytes (TAAs), macrophages (TAMs), and vascular endothelium. Ten distinct clusters emerged representing malignant cell states, defined primarily by programs of cell cycle progression and neuronal differentiation. Notably, these MB cell clusters were differentially abundant between the MB subgroups, highlighting heterogeneity between disease subtypes. Moreover, examining the patterns of cellular spatial distribution in high-risk (HR) patients revealed dense regions of quiescent MB progenitors, while standard-risk patients showed greater heterogeneity within their TME. Proximity-dependent intercellular signaling further highlighted increased signaling by TAAs and vascular endothelium with decreased heat shock response signaling in HR patients. Comparing samples from relapse to diagnosis, TAMs, TAAs, and vascular endothelium were found to constitute a greater proportion of the TME. Subsequent niche-dependent gene expression and pathway analysis for TAAs localized near tumor vasculature in relapsed samples revealed increased expression of metastasis-associated pathways. Collectively, the results of our study enable new insight into the heterogeneity of the MB-TME, with close spatial association of quiescent MB progenitors correlating with high-risk clinical features. Quiescent MB progenitors have been documented to confer resistance through cellular dormancy, and identifying its molecular drivers may reveal potential targets for anticancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eab43977fead30af3a35caa1b5f09c7a002ddad" target='_blank'>
              TMIC-04. SPATIAL SEQUENCING OF MEDULLOBLASTOMA REVEALS BIOLOGICAL NICHES CORRELATED WITH HIGH-RISK FEATURES AND RELAPSE
              </a>
            </td>
          <td>
            Marina E. Michaud, Franklin Chien, Mojtaba Bakhtia, M. Snuderl, Chanel Schroff, Matthew Schneiderjan, Tobey J. MacDonald, Manoj Bhasin
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to adapt to and exploit their microenvironment for sustained growth. The liver is a common site of metastasis, but the interactions between tumor cells and hepatocytes remain poorly understood. In the context of liver metastasis, these interactions play a crucial role in promoting tumor survival and progression. This study leverages multiomics coverage of the microenvironment via liquid chromatography and high-resolution, high-mass accuracy mass spectrometry-based untargeted metabolomics, 13C-stable isotope tracing, and RNA sequencing to uncover the metabolic impact of co-localized primary hepatocytes and a colon adenocarcinoma cell line, SW480, using a 2D co-culture model. Metabolic profiling revealed disrupted Warburg metabolism with an 80% decrease in glucose consumption and 94% decrease in lactate production by hepatocyte-SW480 co-cultures relative to SW480 control cultures. Decreased glucose consumption was coupled with alterations in glutamine and ketone body metabolism, suggesting a possible fuel switch upon co-culturing. Further, integrated multiomic analysis indicates that disruptions in metabolic pathways, including nucleoside biosynthesis, amino acids, and TCA cycle, correlate with altered SW480 transcriptional profiles and highlight the importance of redox homeostasis in tumor adaptation. Finally, these findings were replicated in 3-dimensional microtissue organoids. Taken together, these studies support a bioinformatic approach to study metabolic crosstalk and discovery of potential therapeutic targets in preclinical models of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8536f19d4a9f802b3a9ad739c338954491fd6245" target='_blank'>
              Deciphering Colorectal Cancer-Hepatocyte Interactions: A Multiomic Platform for Interrogation of Metabolic Crosstalk in the Liver-Tumor Microenvironment
              </a>
            </td>
          <td>
            Alisa B. Nelson, Lyndsay E. Reese, Elizabeth Rono, Eric D. Queathem, Yinjie Qiu, Braedan M. McCluskey, Alexandra Crampton, Eric Conniff, Katherine A Cummins, Ella Boytim, Senali Dansou, Justin H. Hwang, S. Safo, Patrycja Puchalska, David K. Wood, K. Schwertfeger, Peter A. Crawford
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) often exhibit stem-like attributes that depend on an intricate stemness-promoting cellular ecosystem within their niche. The interplay between CSCs and their niche has been implicated in tumor heterogeneity and therapeutic resistance. Normal stem cells (NSCs) and CSCs share stemness features and common microenvironmental components, displaying significant phenotypic and functional plasticity. Investigating these properties across diverse organs during normal development and tumorigenesis is of paramount research interest and translational potential. Advancements in next-generation sequencing (NGS), single-cell transcriptomics, and spatial transcriptomics have ushered in a new era in cancer research, providing high-resolution and comprehensive molecular maps of diseased tissues. Various spatial technologies, with their unique ability to measure the location and molecular profile of a cell within tissue, have enabled studies on intratumoral architecture and cellular cross-talk within the specific niches. Moreover, delineation of spatial patterns for niche-specific properties such as hypoxia, glucose deprivation, and other microenvironmental remodeling are revealed through multilevel spatial sequencing. This tremendous progress in technology has also been paired with the advent of computational tools to mitigate technology-specific bottlenecks. Here we discuss how different spatial technologies are used to identify NSCs and CSCs, as well as their associated niches. Additionally, by exploring related public data sets, we review the current challenges in characterizing such niches, which are often hindered by technological limitations, and the computational solutions used to address them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/548e0b0064280f307e085a25804779bf2b1c655a" target='_blank'>
              Deciphering normal and cancer stem cell niches by spatial transcriptomics: opportunities and challenges.
              </a>
            </td>
          <td>
            Hirak Sarkar, Eunmi Lee, Sereno Lopez-Darwin, Yibin Kang
          </td>
          <td>2024-11-04</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults, characterized by poor overall survival and an urgent need for more effective treatment strategies. Macroscopic heterogeneity, as observed through MR imaging, divides the tumor into distinct regions: a tumor-cell dense contrast-enhancing tumor core and a non-contrast-enhancing infiltration zone that appears hyperintense on T2/FLAIR imaging. The infiltration zone contains both infiltrating tumor cells and non-malignant cells from the local brain microenvironment. Although genomic and transcriptomic differences between the tumor core and the infiltration zone have been reported, functional differences, such as differential drug sensitivities, remain largely unknown.



 To investigate this, we collected region-specific glioblastoma patient tissue samples from both the contrast-enhancing tumor core and the non-contrast-enhancing but 5-aminolevulinic acid (5-ALA) fluorescent infiltration zone during surgery. Using pharmacoscopy, a microscopy-based single-cell drug screening platform, we assessed drug responses across regions and patients.



 To determine whether drug responses within a single tumor region were more similar compared to responses across different regions we collected multiple tissue samples from each tumor region in seven patients. We identified region-specific drug responses for temozolomide and lomustine, among other drugs, in some of the patients. However, drug sensitivities were not consistent within the same region across different patients. Thus, across patients, the region-specific drug responses were obscured by patient heterogeneity. To address intra- and inter-tumor drug response heterogeneity, we developed a complementarity score based on region-specific drug response data of 60 drugs across 23 paired samples. The complementarity score identified promising drug combinations that demonstrated strong anti-tumor activity across tumor regions and patients.



 Monotherapy approaches have largely failed in glioblastoma, despite extensive molecular profiling efforts aimed at identifying tumor-specific vulnerabilities. Identifying drug combinations that synergistically target the locoregional heterogeneity of glioblastoma may open up new avenues of personalized therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bab37d6465b9764681d6770f3a30b2ef1d8dd5" target='_blank'>
              DDDR-05. SPATIALLY RESOLVED DRUG SENSITIVITY PROFILING IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, Sohyon Lee, R. Wegmann, J. Mena, F. Vasella, J. Friesen, Luca Regli, E. Le Rhun, A. Zeitlberger, M. Neidert, N. Tatari, G. Hutter, Michael Weller, B. Snijder, T. Weiss
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0c7c17d016e29c5fa8d5c2c935484d2301a05e" target='_blank'>
              Anaplastic thyroid carcinoma: vimentin segregates at the invasive front of tumors in a murine xenograft model.
              </a>
            </td>
          <td>
            Alessandro Miraglia, L. Giannotti, F. De Nuccio, A. S. Treglia, Michele Maffia, D. Lofrumento, Bruno Di Jeso, G. Nicolardi
          </td>
          <td>2024-11-18</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Diffuse infiltration is an aggressive feature of high-grade gliomas with survival implications. The contribution of crosstalk between non-neoplastic and neoplastic cells to tumor infiltration remains largely understudied due to the lack of profiling techniques that retain spatial information. Spatial multi-omic profiling is a promising approach to comprehensively analyze transcript-omics, prote-omics and metabol-omics on the same tissue section while preserving information about the spatial organization of cells. Integration of these spatial studies allows for inferring the consequences of complex cell-cell communication underlying tumor infiltration. We hypothesize that the glioma edge is enriched with pro-infiltration ligands-receptors driving tumor infiltration. Strategies that disrupt these ligand-receptor networks may suppress glioma infiltration and improve clinical outcomes.



 We utilized a glioma tissue micro-array (TMA) to establish the neoplastic and non-neoplastic heterogeneity in the GBM infiltration edge. Each TMA slide consists of pathologist annotated tumor core and edge samples of Glioblastoma (IDH Wildtype, WHO Grade 4; n=10), Diffuse Astrocytoma (IDH Mutant, WHO Grade 3; n=3), Oligodendroglioma (IDH mutant, WHO Grade 2; n=5) and 2 non-brain control samples. We performed spatial multi-omics profiling on adjacent sections of the TMA using 10x Xenium spatial transcriptomics, imaging mass cytometry (IMC) and mass spectrometry imaging (MSI). Integration, visualization, and quantification of the spatial data was done on VisioPharm.



 In GBM, we identified candidate mRNA transcripts and proteins for ligands enriched at the infiltrating edge (compared to tumor core; p<0.0001) that correlated with a poor progression free survival (PFS; r2=0.22). These candidate ligands were also significantly enriched at the edge of oligodendroglioma WHO Grade 2 and astrocytoma WHO Grade 3.



 Spatial multi-omics profiling on a TMA consisting of Glioma WHO grades 2-4 identified differentially expressed and targetable pro-tumor infiltration ligands associated with lower PFS in GBM. Functional studies to uncover the role of metabolites enriched at the glioma edge for the expression of the identified pro-infiltration ligands are ongoing.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccf0234947ec5e04cf1bc6ff024f2ae5bb086359" target='_blank'>
              EPCO-07. INTEGRATING SPATIALLY RESOLVED MULTI-OMICS DATA TO UNCOVER DYSFUNCTIONAL METABOLISM DRIVEN NETWORKS THAT ENHANCE INFILTRATION OF DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            Yanxia Ma, Nhat Nguyen, Sanjay Singh, Akshay V. Basi, Duncan Mak, Javier Gomez, Jared Burks, Erin Seely, Frederick F Lang, C. Ene
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Inherited genetic variants contribute to Barrett’s esophagus (BE) and esophageal adenocarcinoma (EAC) but it is unknown which cell types are involved in this process. We performed single cell RNA-sequencing of BE, EAC and paired normal tissues and integrated data of a genome-wide association study to determine cell type-specific genetic risk and cellular processes that contribute to BE and EAC. The analysis revealed that EAC development is driven to a greater extent by local cellular processes than BE development. One cell type of BE origin (BE-EAC) and cellular processes that control the differentiation of columnar cells are of particular relevance for EAC development. Further, specific subtypes of fibroblasts and endothelial cells contribute to BE and EAC development, while dendritic cells and CD4+ memory T cells contribute exclusively to BE development. The diagnostic use of markers characterizing the identified cell types and cellular processes should be explored in future for EAC prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cab9925c507d28dd37c045256d8f4a2182543fb5" target='_blank'>
              Single cell analysis of Barrett’s esophagus and carcinoma reveals cell types conferring risk via genetic predisposition
              </a>
            </td>
          <td>
            Marten C. Wenzel, Ann-Sophie Giel, Patrick S. Plum, Sascha Hoppe, Marek Franitza, Christoph Jonas, Pouria Dasmeh, René Thieme, Yue Zhao, Dominik Heider, C. Palles, R. Fitzgerald, Christiane J Bruns, Reinhard Buettner, Alexander Quaas, Ines Gockel, Carlo Maj, Seung-Hun Chon, Johannes Schumacher, Axel M. Hillmer
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Effective immune synapses are a prerequisite for desirable immune surveillance and control of tumour growth. Cell-to-cell interactions form the basis of immune synapses, resulting in either immunoprotected or immunosuppressed manifestations in the tumour brain that could explain glioblastoma (GBM) recurrence. Using state-of-the-art multiplex imaging, spatial statistics, and spatially aware deep learning tools in 14 patient-matching tissues from primary and recurrent GBM, we detected spatial immune networks (SINets). SINets across 105 regions of interest comprising 2.3 million cells show stronger spatial engagement between myeloid and lymphoid lineages at recurrence. SINets’ architecture in primary tumours is associated with relapse timing. Patients, where the SINets are more modular, recur sooner than patients with denser SiNets. In addition, this association presents sexual dimorphism in our cohort (N=3 females, and N=4 males). Phenotype-to-phenotype spatial coexistence also contributes to enhancing phenotype prediction. A spatial-aware deep learning model trained with single-cell data from primary tumours reaches an average balanced accuracy of 0.76 in primary tumour cellular communities unseen during training, compared to a spatial-agnostic model that reached an average balanced accuracy of 0.69. In summary, we detected a distinct spatial engagement in glioblastoma undergoing chemo-radiation treatment. The spatial patterns of coexistence represent ecological niches that contribute to defining cell occurrence and may underpin effective immune synapses. Overall, the spatiotemporal elements provide a newer approach for dissecting glioblastoma’s complexity and discovering opportunities for stratifying patients by harnessing spatial information encoded in multicellular niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8383db42b59d05ff4b66f525f28516202250c998" target='_blank'>
              TMIC-68. DEEP LEARNING OF THE SPATIAL IMMUNE ENGAGEMENT IN GLIOBLASTOMA UNDER TEMOZOLOMIDE TREATMENT
              </a>
            </td>
          <td>
            Simon P. Castillo, Afrooz Jahedi, Pravesh Gupta, J. Huse, K. Kannan, Yinyin Yuan, Krishna P. Bhat
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Recently, we have shown that primary brain tumors form highly organized, small-world scale-free networks, reminiscent of early networks in neurodevelopment and highly resistant to therapy. Such putative cancer-cell intrinsic neural mechanisms are becoming an increasing focus in the field of Cancer Neuroscience, but it has not yet been shown whether tumors of non-CNS origin also have the ability to form multicellular networks which sustain proliferative signaling and increase resistance to therapy. We observed synchronized calcium activity using longitudinal microscopy both in vivo and in vitro that were reminiscent of gap junction coupled networks. Indeed, dye transfer experiments verified functional coupling of brain metastases cells in vitro. Similarly, inhibition of gap junctions with different pharmacological agents significantly reduced calcium oscillations and tumour proliferation in vitro. Furthermore, gap junctions are significantly upregulated in brain-tropic sublines compared to parental cells. To understand the functional relevance of these gap-junction coupled tumor cell networks, we performed bulk RNA-Sequencing of two melanoma metastasis models in monoculture under gap junction inhibition and control: We observed reduced calcium communication and a concordant downregulation of neurodevelopmental and synaptic pathways within the tumor cells upon network disconnection, highlighting a potential recapitulation of neural-like features in metastases to the brain. To investigate the therapeutic potential of targeting these networks, we treated mice with brain metastases with two brain-penetrant gap junction blockers and observed significantly reduced tumor burden. Finally, we investigated the presence of heterotypic networks between brain metastases cells and the brain microenvironment with our SR101 dye transfer pipeline. Contrary to our findings in primary brain tumors, no gap junction connections to cells of the brain microenvironment in those models with reduced tumor growth upon network inhibition could be detected, supporting a cancer cell intrinsic gap junction mediated network and highlighting a selectivity of these homogenous tumor networks and potential therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f760d30de9fc9f379968ef3f9925cc574445048" target='_blank'>
              CNSC-27. INTRATUMORAL CANCER CELL NETWORKS IN BRAIN METASTASES
              </a>
            </td>
          <td>
            Niels Olshausen, N. Hebach, Joana S Schlag, Calvin Thommek, Yvonne Yang, D. Hoffmann, C. Mayer, D. Westphal, M. Breckwoldt, Wolfgang Wick, F. Winkler, M. Karreman
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Despite the ability to induce strong clinical responses, therapies that target mutational drivers of oncogenic signaling are not curative in advanced cancers, as a subset of cancer cells is capable of surviving therapy and evolving resistance. While therapy resistance is commonly viewed as a cell-intrinsic phenomenon, multiple factors produced by tumors can also confer strong therapy resistance. In contrast to the detailed elucidation of molecular mediators of cell-intrinsic and stroma-mediated (SM) resistance, the relative importance of cell-intrinsic and SM resistance to in vivo therapeutic responses remains poorly defined. To address this gap of knowledge, we used a well-characterized experimental model of ALK+ lung cancer H3122 cell line that exemplifies a duality observed across common models of targetable cancers. Under standard stroma-free in vitro cultures, H3122 cells can survive under therapeutically relevant concentrations of ALK inhibitors (ALKi), eventually acquiring cell-intrinsic therapy resistance. At the same time, co-culture with stromal fibroblasts drastically reduces the sensitivity of H3122 cells to ALKi, indicating the relevance of SM resistance. To understand the impact of SM resistance on in vivo responses, we sought to identify the mechanism(s) responsible for the therapy-protective effects of stromal fibroblasts, subsequently interrogating the impact of the disruption of this mechanism on in vivo therapeutic responses. Our in vitro studies with a large panel of primary stromal fibroblast isolates identified the HGF-cMET axis as the major mechanism responsible for ALKi desensitization. Surprisingly, xenograft validation studies demonstrated a weak effect of HGF-cMET modulation. Histological analyses of tumor tissues revealed that this relative weakness is attributable to a strong, spatially limited HGF-independent component of SM resistance. Mechanistic follow-up demonstrated that the HGF-independent component of SM resistance integrates the effect of multiple well-known juxtacrine and paracrine-acting mechanisms. Whereas the multifactorial nature of stroma-mediated resistance prevented a clear assessment of its relative contribution to in vivo therapy responses, our spatial analyses indicate that SM resistance dominates the ability of tumors to avoid therapeutic elimination. Surprisingly, we found that SM resistance was also a substantial contributor to tumor relapse, indicating that in vivo therapy resistance can integrate both cell-intrinsic and cell-extrinsic effects. This duality of therapy resistance and the multifactorial underpinning of both cell-intrinsic and stroma-mediated resistance present a challenge to therapeutic strategies focused on identifying and targeting specific resistance mechanisms. Our studies suggest that this limitation can be overcome by shifting the therapeutic focus towards shared orthogonal therapeutic sensitivities of tumor cells within residual disease.
 Citation Format: Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara Henry, Viktoriya Marusyk, Chandler Gatenbee, Marilyn Bui, Jacob Scott, Philipp M. Altrock, Eric Haura, Alexander R.A. Anderson, David Basanta, Andriy Marusyk. Spatially limited stroma-mediated resistance potentiates targeted therapy resistance through an integration of multiple juxtacrine and paracrine mechanisms [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C035.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943bc0d90e7f0f13d1f5d2038bd76c206372d768" target='_blank'>
              Abstract C035: Spatially limited stroma-mediated resistance potentiates targeted therapy resistance through an integration of multiple juxtacrine and paracrine mechanisms
              </a>
            </td>
          <td>
            Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara-Chinua N Henry, Viktoriya Marusyk, Chandler D. Gatenbee, Marilyn M Bui, Jacob G. Scott, Philipp M. Altrock, E. Haura, Alexander R. A. Anderson, D. Basanta, A. Marusyk
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) is associated with a high risk of distant metastasis, particularly to the lung and liver. Both intrinsic characteristics of cancer cells and the tumor microenvironment (TME) influence TNBC growth and metastasis. The TME, comprising stromal, immune, and endothelial cells, exhibits heterogeneity both between and within cell types. Identifying the molecular characteristics of TME cells that affect cancer progression is crucial. We aim to investigate this using single-cell RNA sequencing of patient-derived xenograft (PDX) models to explore TME cells and genes involved in TNBC metastasis. We established PDX models by transplanting tumor tissues from 26 TNBC patients into immunodeficient mice. These models were categorized based on their metastatic potential and patient outcomes: non-metastatic (n=5) and metastatic (n=4). Single-cell RNA sequencing of nine PDX tumors revealed significant differences in gene expression of murine stromal cells between metastatic and non-metastatic models. We identified ten genes (Serpinb2, Spp1, Tnc, Thbs1, Timp1, Il11, Mt2, Crabp1, Cck, Mt1) that were highly expressed in the stromal cells of metastatic PDX models. To test if TNBC cells influence these fibroblast genes, we exposed NIH3T3 fibroblasts to conditioned media from the 4T1 TNBC cell line. In three independent experiments, only Crabp1 expression in NIH3T3 cells was consistently increased by 4T1- conditioned media. CRABP1, a retinoic acid-binding protein, is known for its role in differentiation and proliferation by regulating MAPK signaling. Although Crabp1-positive cancer-associated fibroblasts have been noted, its role in TNBC metastasis was previously unexplored. We investigated CRABP1 function using a CRABP1 knockdown NIH3T3 cell line. CRABP1 knockdown led to reduced invasion and migration of 4T1 cells in trans-well assays and decreased invasiveness of 4T1 spheroids in a collagen matrix. Additionally, CRABP1 knockdown NIH3T3 cells showed reduced proliferation in vitro and fewer alpha-SMA positive cancer-associated fibroblasts in co-injected 4T1 tumors in BALB/C mice. In summary, CRABP1, identified through single-cell RNA sequencing of stromal cells in TNBC PDX models, is a potential regulator of TNBC metastasis, affecting cancer cell migration, invasion, and fibroblast proliferation.
 Citation Format: Woohang Heo, Yujeong Her, Sieun Yang, Dakyung Lee, Rokhyun Kim, Jong-Il Kim, Hyeong-Gon Moon. Single cell RNA sequencing of triple negative breast cancer patient-derived xenograft model identifies CRABP1 of cancer-associated fibroblast as a key regulator of breast cancer metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f55fcf53482b3293e557961f04bd5df55bfd4945" target='_blank'>
              Abstract A034: Single cell RNA sequencing of triple negative breast cancer patient-derived xenograft model identifies CRABP1 of cancer-associated fibroblast as a key regulator of breast cancer metastasis
              </a>
            </td>
          <td>
            W. Heo, Y. Her, Sieun Yang, Dakyung Lee, Rokhyun Kim, Jong-Il Kim, H. Moon
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Introduction Breast cancer (BC) presents diverse malignancies with varying biological and clinical behaviors, driven by an interplay between cancer cells and tumor microenvironment. Deciphering these interactions is crucial for personalized diagnostics and treatment. This study explores the prognostic impact of tumor proliferation and immune response patterns, assessed by computational pathology indicators, on breast cancer-specific survival (BCSS) models in estrogen receptor-positive HER2-negative (ER+HER2–) and triple-negative BC (TNBC) patients. Materials and methods Whole-slide images of tumor surgical excision samples from 252 ER+HER2– patients and 63 TNBC patients stained for estrogen and progesterone receptors, Ki67, HER2, and CD8 were analyzed. Digital image analysis (DIA) was performed for tumor tissue segmentation and quantification of immunohistochemistry (IHC) markers; the DIA outputs were subsampled by hexagonal grids to assess the spatial distributions of Ki67-positive tumor cells and CD8-positive (CD8+) cell infiltrates, expressed as Ki67-entropy and CD8-immunogradient indicators, respectively. Prognostic models for BCSS were generated using multivariable Cox regression analysis, integrating clinicopathological and computational IHC indicators. Results In the ER+HER2– BC, multivariable Cox regression revealed that high CD8+ density within the tumor interface zone (IZ) (HR: 0.26, p = 0.0056), low immunodrop indicator of CD8+ density (HR: 2.93, p = 0.0051), and low Ki67-entropy (HR: 5.95, p = 0.0.0061) were independent predictors of better BCSS, while lymph node involvement predicted worse BCSS (HR: 3.30, p = 0.0013). In TNBC, increased CD8+ density in the IZ stroma (HR: 0.19, p = 0.0119) and Ki67-entropy (HR: 3.31, p = 0.0250) were independent predictors of worse BCSS. Combining these independent indicators enhanced prognostic stratification in both BC subtypes. Conclusions Computational biomarkers, representing spatial properties of the tumor proliferation and immune cell infiltrates, provided independent prognostic information beyond conventional IHC markers in BC. Integrating Ki67-entropy and CD8-immunogradient indicators into prognostic models can improve patient stratification with regard to BCSS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e50ee2898c062787fae58412a5fdbeb3e64a19" target='_blank'>
              Spatial distributions of CD8 and Ki67 cells in the tumor microenvironment independently predict breast cancer-specific survival in patients with ER+HER2– and triple-negative breast carcinoma
              </a>
            </td>
          <td>
            Dovile Zilenaite-Petrulaitiene, A. Rasmusson, Ruta Barbora Valkiuniene, A. Laurinavičienė, Linas Petkevičius, A. Laurinavičius
          </td>
          <td>2024-11-22</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Intratumoral heterogeneity promotes tumor progression, treatment resistance, and recurrence in glioblastoma (GBM), and the cellular tumor core and infiltrating edge represent transcriptionally distinct cellular ecosystems. In this study, we characterize and validate the molecular differences across the core-edge spatial axis in GBM by applying whole-transcriptome GeoMx Digital Spatial Profiling (DSP) to 5 IDH-wildtype tumors. Three 3-micron tissue cores were extracted from each tumor and arranged into a tissue microarray (TMA). Core and edge regions of interest (ROIs) were chosen through coregistration of H&E and immunofluorescent TMA scans. Following DSP, data for 44 ROIs (30 core and 14 edge) and 8530 RNA targets were bioinformatically analyzed. Single-sample gene set enrichment analysis aligned the DSP core and edge regional signatures to those of publicly annotated datasets, substantiating our profiling methodology. Unsupervised analyses revealed ROI clusters unique to each tumor specimen for both pre- and post-normalized data. To mitigate a potential patient-level effect in subsequent differential gene expression and pathway analyses, we leveraged a linear mixed-effect model using core versus edge as a fixed effect and tumor specimen as a random effect. Our analyses showed upregulation of epigenomic markers in core ROIs and neuronal markers in edge ROIs. Further, core and edge signatures were incongruent with the canonical GBM subtypes (Mesenchymal, Classical, Proneural) and malignant cell states (OPC-like, NPC-like, AC-like, MES-like). Cellular deconvolution analysis highlighted differential enrichment of neoplastic cells in core ROIs and neuroglia (astrocytes and oligodendrocytes) in edge ROIs. In summary, we leverage spatial profiling to further validate the geographic molecular heterogeneity of GBM, which is characterized by divergent neuronal programs within the infiltrating edge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7cc77c63633549b85f3d94e4ad3932eaf68377d" target='_blank'>
              EPCO-48. RESOLVING INTRATUMORAL TRANSCRIPTIONAL HETEROGENEITY ACROSS THE CORE-EDGE SPATIAL AXIS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Adam Luo, Kelly Casella, Ezra G. Baraban, Calixto-Hope G. Lucas, Eugene Shenderov
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Background. Virtually every MM patient transitions through the asymptomatic precursor conditions MGUS and SMM prior to developing MM. This provides a window for intervention to prevent progression to frank malignancy, and also exciting opportunities to investigate mechanisms underlying such progression. In contrast to solid tumours, relatively little is known about how the immune system is shaped by, and in turn shapes, MM through progression from precursor conditions to frank malignancy.
 Aim. To define the contribution of tumour features to T cell differentiation during disease evolution from precursor disease to MM.
 Methods. We generated the “pre-myeloma atlas”, the largest single-cell RNA sequencing (scRNA-seq) dataset of precursor disease and MM to date. It encompassed >1,000,000 cells from 295 newly-generated and published scRNA-seq samples from MM (n=54 patients), precursor conditions (n=78), and non-cancer controls (n=102). We enhanced the atlas through including single-cell T cell receptor (TCR) sequencing (42% samples), bone marrow and peripheral blood sampling, and modelling clinical data including age and tumour burden. Malignant clones (tumour cells) were identified by clonal immunoglobulin expression and scored with pan-cancer transcriptional pathways. Viral-reactive TCRs were acquired from VDJdb and neoantigen-reactive TCRs predicted with NetMHCpan4.1 and TEINet.
 Results. Within the pre-myeloma atlas, we identified CD4+ and CD8+ T cell subsets across the spectrum of differentiation from naïve (SELL+CCR7+) to terminal memory (GZMB+) phenotypes. Notably, we identified exhausted T cells in only a single MM patient (1 of 54), suggesting T cell exhaustion is not a pervasive feature of MM. Comparing controls with precursor and symptomatic patients, we found T cell composition was comparable through disease evolution. However, the abundance of T cell subsets was altered: in MM, earlier differentiated subsets such as naïve CD8+ were depleted relative to controls (P<0.001) whereas terminal memory subsets were enriched (P<0.003). Principal component analysis of T cell composition captured this shift from early to terminal subsets, which we termed T cell skewing. T cell skewing rose with disease severity and was enhanced in SMM and MM independent of patient age (SMM P=0.004; MM P<0.001). Longitudinal sampling of precursor patients who subsequently progressed revealed T cell skewing occurs longitudinally at progression. Together, this suggests exaggerated T cell memory differentiation occurs in MM.
 TCR repertoire clonality (indicative of T cell proliferation in response to antigen) was higher in MM relative to controls (P=0.06) and corrected with T cell skewing (R=0.71, P<0.001). Additionally, the frequency of TCR clones possessing similar antigen-recognition domains (indicative of shared antigen-specificity) rose between SMM and MM (P<0.002) and correlated with T cell skewing (R=0.4, P=0.02). Importantly, in silico analyses of T cell specificity suggested (1) TCR clones annotated as viral-reactive were not associated with T cell skewing, and (2) TCR clones predicted to bind autologous tumour neoantigens occupied terminal memory states. Together, these results suggest antigen, possibly tumour in origin, drive T cell differentiation in MM.
 Finally, to interrogate mechanisms of tumour-immune cross-talk, we compared T cell and tumour features. We observed a positive correlation between T cell skewing with paraprotein (R=0.43, P=0.04), that serves as a proxy for tumour burden. Additionally, effector T cells (IFNG+TNF+) were associated with tumour marrow infiltration (aspirate % CD138+; R=055, P<0.001). When analysing the transcriptional activity of tumour cells, we found effector T cells also correlated with cell death-associated pathways (R=0.66, P<0.001). We noted sharing of TCRs between these two T cell subsets, suggesting a differentiation process between these subsets possibly regulated by tumour-intrinsic processes and tumour burden.
 Conclusion. Our results show that, unlike solid cancers, MM is not associated with T-cell exhaustion and instead defined by a pattern of T-cell differentiation resembling antigen-driven terminal memory differentiation. T cell associations with tumour burden and antigen could suggest anti-tumour immunity drives a novel form of cancer-associated T-cell differentiation in MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729020d6ba64084dfb7bfe1195058f87d0ac8af9" target='_blank'>
              Tumour-Intrinsic Features Shape T-Cell Differentiation through Myeloma Disease Evolution
              </a>
            </td>
          <td>
            K. Foster, Elise Rees, L. Ainley, Eileen M Boyle, L. Lee, Gwennan Ward, D. Galas-Filipowicz, Anna Mikolajczak, Emma J Lyon, Dylan Jankovic, Jasmine Rahman, Mahima Turakhia, Imran Uddin, Gordon Beattie, Yvette Hoade, Catherine Zhu, J. Reading, Ieuan G Walker, Michael A. Chapman, K. Ramasamy, Javier Herrero, B. Chain, Sergio Quezada, K. Yong
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Background Glioblastoma is the most aggressive malignant brain tumor with poor survival due to its invasive nature driven by cell migration, with unclear linkage to transcriptomic information. The aim of this study was to develop a physics-based framework connecting to transcriptomics to predict patient-specific glioblastoma cell migration. Methods and Results We applied a physics-based motor-clutch model, a cell migration simulator (CMS), to parameterize the migration of glioblastoma cells and define physical biomarkers on a patient-by-patient basis. We reduced the 11-dimensional parameter space of the CMS into 3 principal physical parameters that govern cell migration: motor number—describing myosin II activity, clutch number—describing adhesion level, and F-actin polymerization rate. Experimentally, we found that glioblastoma patient-derived (xenograft) cell lines across mesenchymal (MES), proneural, and classical subtypes and 2 institutions (N = 13 patients) had optimal motility and traction force on stiffnesses around 9.3 kPa, with otherwise heterogeneous and uncorrelated motility, traction, and F-actin flow. By contrast, with the CMS parameterization, we found that glioblastoma cells consistently had balanced motor/clutch ratios to enable effective migration and that MES cells had higher actin polymerization rates resulting in higher motility. The CMS also predicted differential sensitivity to cytoskeletal drugs between patients. Finally, we identified 18 genes that correlated with the physical parameters, suggesting transcriptomic data alone could potentially predict the mechanics and speed of glioblastoma cell migration. Conclusions We describe a general physics-based framework for parameterizing individual glioblastoma patients and connecting to clinical transcriptomic data that can potentially be used to develop patient-specific anti-migratory therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04123a654e8ca28f714460bf7a348c04580a354d" target='_blank'>
              Cell migration simulator-based biomarkers for glioblastoma
              </a>
            </td>
          <td>
            Jay Hou, M. McMahon, Tyler Jubenville, J. Sarkaria, Clark C Chen, David J. Odde
          </td>
          <td>2024-11-02</td>
          <td>Neuro-Oncology Advances</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaacb86999aeed003c707b4000adb56d67bb6547" target='_blank'>
              Spatial transcriptome profiling identifies DTX3L and BST2 as key biomarkers in esophageal squamous cell carcinoma tumorigenesis
              </a>
            </td>
          <td>
            Rutao Li, Na Li, Qianqian Yang, Xing Tong, Wei Wang, Chang Li, Jun Zhao, Dong Jiang, Haitao Huang, Chen Fang, Kai Xie, Jiamin Yuan, Shaomu Chen, Guangbin Li, Haitao Luo, Zhibo Gao, Dongfang Wu, Xiaoli Cui, Wei Jiang, Lingchuan Guo, Haitao Ma, Yu Feng
          </td>
          <td>2024-12-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Phenotypic plasticity drives non-genetic tumor adaptation in response to various microenvironmental and therapeutic pressures, consequently promoting tumor heterogeneity and progression. In malignant gliomas, the intrinsic and extrinsic mechanisms underlying phenotypic lineage plasticity and its contribution to drug resistance remain unclear. The epidermal growth factor receptor (EGFR) is frequently altered in glioma and serves as a critical regulator of normal radial glia (RG) cells – multipotent progenitors that give rise to cortical neurons and glia in the developing brain. Here, through interrogation of a large panel of diverse glioblastoma (GBM) patient-derived gliomaspheres and orthotopic xenografts, we find that enrichment of RG-like cells is significantly correlated with responses to pharmacological ablation of EGFR, highlighting EGFR as a crucial lineage survival factor for a population of malignant RG-like cells in GBM. Single-cell RNA sequencing and quantitative proteomics reveal that adaptation to EGFR inhibition is linked to a temporal contraction in RG-like cells and concomitant increase in lineage-restricted neuronal and oligodendrocyte progenitor (NPC/OPC)-like states exclusively in the orthotopic brain environment. This lineage transition is coupled to heightened RAS signaling gene expression programs and sustained activation of MAPK signaling, despite robust and durable inhibition of EGFR activation. Furthermore, constitutively active MEK – but not AKT – is sufficient to rescue tumor proliferation under EGFRi, suggesting that the brain microenvironment modulates RAS-MAPK oncogenic signaling to drive lineage transitions following EGFR blockade. Collectively, these results highlight a critical link between oncogenic signaling and neurodevelopmental lineage plasticity in malignant gliomas, presenting a compelling therapeutic opportunity to block tumor cell state transitions for more durable tumor responses in the native glioma tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b338b52a7c0c64400d62a989938ab592a2bae18" target='_blank'>
              CSIG-24. BRAIN MICROENVIRONMENT-DEPENDENT NEURODEVELOPMENTAL LINEAGE PLASTICITY PROMOTES ADAPTATION TO ONCOGENE INHIBITION IN MALIGNANT GLIOMAS
              </a>
            </td>
          <td>
            Dimitri Cadet, Cassidy Andrasz, Nicholas A. Bayley, Elisa Fazzari, E. Fernández, Q. Okobi, Thomas Graeber, A. Bhaduri, T. Cloughesy, David Nathanson
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="
 Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood. To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing. Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles. Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f68ff076c678ea3a1de296cefcb3a2b2a3560d" target='_blank'>
              EPCO-26. LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K. Johnson, Avishay Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K. Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y. Oh, H. Moon, S. Paek, Franck Bielle, A. Laurenge, Anna Luisa Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Zheng Zhao, Qianghu Wang, T. Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Metastatic bladder cancer (BLCA) is an aggressive disease complicated by the emergence of variant histological subtypes. The UW/Fred Hutch metastatic BLCA rapid autopsy program is a first-of-its-kind program collecting cell-free DNA (cfDNA) and matched normal tissue, primary tumor, and metastatic tumor samples from advanced BLCA patients, with a focus on these variant subtype tumors. This dataset, consisting of 20 patients with up to five metastases per patient, is a unique opportunity to assess how cfDNA is able to capture heterogeneity across BLCA. We assessed both inter-patient heterogeneity between BLCA subtypes and inter-tumoral heterogeneity within each patient using epigenetic and genomic profiling. Using nucleosome profiling, we identified the activity of lineage markers and transcription factors that distinguish healthy individuals from BLCA patients, and between BLCA subtypes. Furthermore, we conducted a detailed analysis of whole genome (30X) and targeted (4,000X) cfDNA sequencing in their ability to capture the evolutionary history of tumor mutations. Detection of tumor mutations by cfDNA is highly dependent on clonality, with >90% of founder tumor mutations captured by either sequencing method, while capture is decreased for either subclonal mutations or those private to a single metastasis. Importantly, deleterious mutations in BLCA driver genes had close to 100% detection using targeted panel sequencing, even for subclonal mutations. These findings lay the groundwork for the use of cfDNA in clinical BLCA decision-making, including tracking the emergence of pathogenic, targetable mutations and variant subtypes.
 Citation Format: Samantha L Schuster, Pushpa Itagi, Sonali Arora, Patricia C Galipeau, Thomas W Persse, Michael Yang, Allie Kreitman, Alan Min, Funda Vakar-Lopez, John K Lee, Petros Grivas, Robert B Montgomery, Jonathan L Wright, Andrew C Hsieh, Hung- Ming Lam, Gavin Ha. Cell-free DNA captures inter- and intra-patient heterogeneity in advanced bladder cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7eb84ec36801f20a2ac043ba0422da74b9249b7" target='_blank'>
              Abstract B040: Cell-free DNA captures inter- and intra-patient heterogeneity in advanced bladder cancer
              </a>
            </td>
          <td>
            Samantha L Schuster, P. Itagi, Sonali Arora, Patricia Galipeau, Thomas Persse, Michael Yang, Allie Kreitman, Alan Min, F. Vakar‐López, John K. Lee, P. Grivas, R. Montgomery, Jonathan L. Wright, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common malignancy of the central nervous system, characterized by a dismal prognosis and inevitable recurrence despite aggressive standard-of-care therapy. Extensive colonization of the surrounding brain parenchyma precludes complete surgical resection, posing a significant therapeutic challenge. While well-studied in model systems, the molecular features and epigenetic regulators of invasive GBM cells remain under-explored directly in clinical samples. To address this gap, we performed a multimodal single-cell sequencing characterization of GBM tumor samples from anatomically distinct regions collected using MRI guidance. Our results demonstrate an enrichment of progenitor-like (neural- and oligodendrocyte-like) malignant states and neurons at the tumor margins. In contrast, differentiated-like states and myeloid cells were more abundant in the tumor core. Peri-tumoral progenitor-like malignant states expressed a unique neuronal signature associated with synaptic signaling, neurogenesis, and Notch signaling. Further, the expression of this neuronal signature was correlated with increased invasiveness. Analysis of matched primary-recurrent GBM patient cohorts revealed an expansion of this neuronal activity program, which was associated with worse overall survival. Motifs of proneural transcription factors implicated in neuronal lineage differentiation were also found to be differentially accessible in progenitor-like malignant states marked by the neuronal invasive signature, potentially contributing to the remodeling of cell states at the invasive margin. Further, cell-cell interaction analysis predicted greater communication between neurons and peri-tumoral progenitor-like malignant states, mediated predominantly through neurexin-neuroligin trans-synaptic signaling, facilitating synaptogenesis and the integration of tumor cells into neural circuits. Altogether, these findings suggest a model in which GBM invasive cells communicate with normal brain neurons in peri-tumoral regions, exploiting neurodevelopmental pathways to facilitate invasion and potentially seed recurrence. Our characterization of invasive GBM cells provides insight into the mechanisms of brain invasion and highlights potential therapeutic vulnerabilities of malignant invasive cells. Understanding these mechanisms opens new avenues for targeted therapies aimed at curtailing the invasive and adaptive capabilities of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0d487d52385a969d9534435102163444173f4c0" target='_blank'>
              EPCO-28. UNVEILING INVASIVE MECHANISMS OF GLIOBLASTOMA CELLS THROUGH MULTIMODAL SINGLE-CELL SEQUENCING
              </a>
            </td>
          <td>
            Yiyan Wu, Benson Wu, Y. Ellenbogen, S. Mansouri, Joan Kant, Xuyao Li, O. Singh, Parnian Habibi, Pathum Kossinna, Sandra Ruth Lau Rodriguez, A. Gao, G. Zadeh, Federico Gaiti
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d829dac90ae6e0bee2354e14ecf9cb6f0b45ddf8" target='_blank'>
              Polyclonal-to-monoclonal transition in colorectal precancerous evolution.
              </a>
            </td>
          <td>
            Zhaolian Lu, Shanlan Mo, Duo Xie, Xiangwei Zhai, Shanjun Deng, Kantian Zhou, Kun Wang, Xueling Kang, Hao Zhang, Juanzhen Tong, Liangzhen Hou, Huijuan Hu, Xuefei Li, Da Zhou, L. T. Lee, Li Liu, Yaxi Zhu, Jing Yu, Ping Lan, Jiguang Wang, Zhen He, Xionglei He, Zheng Hu
          </td>
          <td>2024-10-30</td>
          <td>Nature</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286075cb8f68cf0d587bedcc36d60116c411594e" target='_blank'>
              Spatially resolved gene expression profiling of tumor microenvironment reveals key steps of lung adenocarcinoma development
              </a>
            </td>
          <td>
            Yuma Takano, Jun Suzuki, Kotaro Nomura, Gento Fujii, Junko Zenkoh, Hitomi Kawai, Yuta Kuze, Y. Kashima, Satoi Nagasawa, Yuka Nakamura, M. Kojima, K. Tsuchihara, Masahide Seki, A. Kanai, D. Matsubara, Takashi Kohno, Masayuki Noguchi, Akihiro Nakaya, Masahiro Tsuboi, G. Ishii, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-12-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Lineage plasticity and histologic transformation from prostate adenocarcinoma to neuroendocrine prostate cancer (NEPC) occurs in up to 15-20% of patients with castration-resistant prostate cancer (CRPC) as mechanism of treatment resistance and is associated with aggressive disease and poor prognosis. NEPC tumors typically display small cell carcinoma morphology with loss of androgen receptor (AR) expression and gain of neuroendocrine (NE) lineage markers. However, there is a spectrum of phenotypes that are observed during the lineage plasticity process, and the clinical significance of mixed histologies or those that co-express AR and NE markers or lack all markers is not well defined. Translational research studies investigating NEPC have used variable definitions making clinical trial design challenging. Here we discuss the diagnostic workup of metastatic biopsies to help guide the reproducible classification of phenotypic CRPC subtypes. We recommend classifying CRPC tumors based on histomorphology (adenocarcinoma, small cell carcinoma, poorly differentiated carcinoma, other morphologic variant, or mixed morphology) and immunohistochemical markers with a priority for AR, NKX3.1, INSM1, synaptophysin and cell proliferation based on Ki-67 positivity, with additional markers to be considered based on the clinical context. Ultimately, a unified workup of metastatic CRPC biopsies can improve clinical trial design and eventually practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ba90573d033c02c8e63a055556c5045dc3b6fc" target='_blank'>
              Framework for the Pathology Workup of Metastatic Castration-Resistant Prostate Cancer Biopsies.
              </a>
            </td>
          <td>
            Michael C Haffner, Michael J. Morris, C. C. Ding, Erolcan Sayar, Rohit Mehra, Brian D. Robinson, Lawrence D. True, Martin Gleave, Tamara L. Lotan, Rahul Aggarwal, Jiaoti Huang, Massimo Loda, Peter S Nelson, Mark A. Rubin, H. Beltran
          </td>
          <td>2024-11-26</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Cancer immunotherapy is increasingly used in clinical practice, but its success rate is reduced by tumor escape from the immune system. This may be due to the genetic instability of tumor cells, which allows them to adapt to the immune response and leads to intratumoral immune heterogeneity. The study investigated spatial immune heterogeneity in the tumor microenvironment and its possible drivers in a mouse model of tumors induced by human papillomaviruses (HPV) following immunotherapy. Gene expression was determined by RNA sequencing and mutations by whole exome sequencing. A comparison of different tumor areas revealed heterogeneity in immune cell infiltration, gene expression, and mutation composition. While the mean numbers of mutations with every impact on gene expression or protein function were comparable in treated and control tumors, mutations with high or moderate impact were increased after immunotherapy. The genes mutated in treated tumors were significantly enriched in genes associated with ECM metabolism, degradation, and interactions, HPV infection and carcinogenesis, and immune processes such as antigen processing and presentation, Toll-like receptor signaling, and cytokine production. Gene expression analysis of DNA damage and repair factors revealed that immunotherapy upregulated Apobec1 and Apobec3 genes and downregulated genes related to homologous recombination and translesion synthesis. In conclusion, this study describes the intratumoral immune heterogeneity, that could lead to tumor immune escape, and suggests the potential mechanisms involved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10b06a4b14e37ec85c1d027d3a7d9fbdaf7aa59e" target='_blank'>
              Spatial immune heterogeneity in a mouse tumor model after immunotherapy.
              </a>
            </td>
          <td>
            Michal Šmahel, S. Johari, J. Smahelova, L. Pfeiferová, Jaroslav Nunvar
          </td>
          <td>2024-12-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7927955b37e21b66841e08578979ae91c22b5bbe" target='_blank'>
              Exosome crosstalk between cancer stem cells and tumor microenvironment: cancer progression and therapeutic strategies
              </a>
            </td>
          <td>
            Qi Li, Guangpeng He, Yifan Yu, Xinyu Li, Xueqiang Peng, Liang Yang
          </td>
          <td>2024-11-22</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background/Aim The dynamic interplay between cancer cells and the microenvironment involves a wide range of intricate relationships that evolve during different stages of tumor progression. Recent attention has focused on high endothelial venules (HEVs), specialized endothelial cells in tumors with a unique cuboidal shape similar to those in lymph nodes. Previous animal studies have shown that normalization of tumor angiogenesis through anti-VEGFR2 therapy promotes HEV formation. However, few reports exist regarding the relationship between HEVs and preexisting blood vessels or interstitial fibers. In this study, we histologically examined whether tumor vascular structure correlates with HEV neogenesis. Patients and Methods A total of 109 patients with pathological stage I lung adenocarcinoma who had undergone curative lung resection at our Institute between 2012 and 2016 were included. HEVs were identified by anti-peripheral node addressin (PNAd) staining. Immunostaining and Elastica-Masson-Goldner staining were performed on tumor sections and quantified. Results PNAd-positive cells were identified in 102 (93.6%) patients. Nearly all PNAd-positive cells were located within or near immune cell clusters. We investigated the correlation between microvessel structures or interstitial fibers and the number/density of PNAd-positive vessels, but no significant correlation was found. Since PNAd-positive cells were concentrated in immune cell aggregates, we focused our analysis specifically on these regions. Immune cell aggregates with abundant PNAd-positive vessels had a greater microvessel density along with by rich collagen fiber production, and displayed a more mature morphological phenotype of HEVs. Conclusion The generation of PNAd-positive cells in tumors is governed by an angiogenetic mechanism distinct from that of broader tumor microenvironment. Furthermore, the accumulation of immune cells is associated with increased HEV maturation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6515292d87876d38e023114fce1cab413aa327" target='_blank'>
              Histopathological Correlation Between High Endothelial Venule Neogenesis and the Tumor Microenvironment in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Ato Sugiyama, Tai Hato, Hiroaki Kashimada, Masatoshi Yamaguchi, Y. Inoue, K. Aoki, Hiroki Fukuda, M. Nakayama, Morihiro Higashi, Mitsutomo Kohno
          </td>
          <td>2024-10-29</td>
          <td>In Vivo</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/602ca4326619720e872bb3125e86fd5844dc2d44" target='_blank'>
              A multiscale model of immune surveillance in micrometastases gives insights on cancer patient digital twins
              </a>
            </td>
          <td>
            Heber L Rocha, Boris Aguilar, Michael Getz, I. Shmulevich, P. Macklin
          </td>
          <td>2024-12-04</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>2</td>
          <td>82</td>
        </tr>

        <tr id="
 Little progress has been made to delineate the inflammatory microenvironment in pediatric high-grade glioma (pHGG) and diffuse midline glioma (DMG). We utilize molecularly defined human samples and immunocompetent mouse models to study how tumor location and genetic driver mutations influence the tumor microenvironment (TME). We first induced tumors of each pHGG/DMG molecular subtype (H3WT, H3G34R, H3.1K27M, H3.3K27M) into both the cortical hemisphere and midline of separate mice and performed survival studies and flow cytometry. The main driving factor shaping the immune landscape and survival is the driver mutation, not tumor location. Next, we performed single cell RNA sequencing on murine H3WT cortical pHGG, H3WT DMG, H3.1 and H3.3K27M DMGs, paired with multiplex flow cytometry. H3K27M DMGs have sparse T cell infiltration and limited T cell recruiting chemokine production. The main non-neoplastic cell type in all tumors was infiltrating myeloid cells and microglia. H3WT DMGs had the greatest myeloid cell and monocyte-derived macrophage (MDM) infiltration, while H3.3K27M DMGs were enriched for microglia. Disease associated gene signatures were found in tumor-associated macrophages (TAMs) that are found in other neurodegenerative diseases, marked by down-regulation of inflammatory signaling, interferon responses, and up-regulation of metabolism-related pathways. H3.3K27M DMGs were enriched for disease-associated TAMs. These gene signatures were then identified in human pHGG/DMG scRNA sequencing data. CD16 monocytes and microglia were enriched for diseased signatures, while CD14 monocytes were pro-inflammatory. We demonstrate that reprogramming this diseased signature in mice results in increased T cell infiltration and extension of survival in H3K27M DMG. Last, through pharmacologic inhibition of CCR1 and CCR5, we demonstrate preventing TAM infiltration in DMG results in enhanced survival, comparable to radiation therapy. Together, this work provides the foundation for developing immunotherapies targeted at specific subgroups of pHGG and DMGs, such as CAR-T-cell, oncolytic viral therapy, and checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fcf1de80664843474f36fd47a61ca2ddc3739c7" target='_blank'>
              TMIC-06. TUMOR ASSOCIATED MACROPHAGES DRIVE TUMOR PROGRESSION AND ARE TRANSCRIPTIONALLY SHAPED BY HISTONE MUTATIONS IN DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            James L. Ross, Montserrat Puigdelloses-Vallcorba, Gonzalo Piñero, N. Soni, Wes Thomason, J. DeSisto, Angelo Angione, N. Tsankova, Maria Castro, M. Schniederjan, Nitin Wadhwani, G. P. Raju, Peter Morgenstern, Oren J. Becher, Adam L Green, A. Tsankov, D. Hambardzumyan
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a crucial role in cancer development and metastasis. This review summarizes the current research on how the TME promotes metastasis through molecular pathways, focusing on key components, such as cancer-associated fibroblasts, immune cells, endothelial cells, cytokines, and the extracellular matrix. Significant findings have highlighted that alterations in cellular communication within the TME enable tumor cells to evade immune surveillance, survive, and invade other tissues. This review highlights the roles of TGF-β and VEGF signaling in promoting angiogenesis and extracellular matrix remodeling, which facilitate metastasis. Additionally, we explored how metabolic reprogramming of tumor and stromal cells, influenced by nutrient availability in the TME, drives cancer progression. This study also evaluated the therapeutic strategies targeting these interactions to disrupt metastasis. By providing a multidisciplinary perspective, this study suggests that understanding the molecular basis of the TME can lead to more effective cancer therapies and identify potential avenues for future research. Future research on the TME should prioritize unraveling the molecular and cellular interactions within this complex environment, which could lead to novel therapeutic strategies and personalized cancer treatments. Moreover, advancements in technologies such as single-cell analysis, spatial transcriptomics, and epigenetic profiling offer promising avenues for identifying new therapeutic targets and improving the efficacy of immunotherapies, particularly in the context of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada71caaa487afafe1604b436411fabfcdc1d067" target='_blank'>
              Tumor microenvironment and cancer metastasis: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Ç. BİRAY AVCI, Bakiye Goker Bagca, M. Nikanfar, Leila S. Takanlou, M. S. Takanlou, Alireza Nourazarian
          </td>
          <td>2024-11-12</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Although various effective anti-cancer treatments have become available over the last decades, resistance to all available therapies remains the major cause of death of cancer patients with disseminated tumors. Striking examples are patients with triple-negative breast cancer (TNBC), which are frequently defective in the repair of DNA double strand breaks, e.g. due to loss of BRCA1 function. Because of this defect, the patients initially respond very well to DNA damage-inducing chemotherapy. Unfortunately, disseminated tumors are usually not eradicated and resistant tumor cells are eventually selected from residual primary or metastatic tumor sites. It is therefore crucial to understand the molecular mechanisms underlying the drug tolerance of the residual tumor cells. To study residual disease, we used the K14cre;Brca1 F/F ;p53 F/F (KB1P) mouse model for hereditary breast cancer, which provides the unique opportunity to explore and target those mechanisms in a fully immunocompetent model. The mammary tumors that spontaneously develop highly resemble their human counterparts, both morphologically and in their therapy response. For example, tumors shrink in response to poly(ADP-ribose) polymerase inhibition, platinum-based treatment or the commonly used doxorubicin, docetaxel and cyclophosphamide (TAC) combination therapy. But despite repeated drug sensitivity, the KB1P mammary tumors are not eradicated, not even by a frequent dosing schedule. By combining single-cell RNA sequencing, spatial transcriptomics and imaging mass cytometry, we dissected the intratumoral heterogeneity and alterations in the tumor microenvironment of residual disease. We identified specific subpopulations of tumor cells that have a survival benefit after treatment, and these occurred together with an altered microenvironment characterized by a highly reactive stroma infiltrated with both anti-inflammatory and pro-tumoral immune cells. Interestingly, those structural and transcriptional changes are reversed in the relapsed tumors, highlighting the plasticity of drug tolerance. To investigate the mechanisms of the altered tumor-stroma interactions that are relevant for drug tolerance, we have designed a custom-made CRISPR/Cas9 library based on differential gene expression of the residual tumor cells. This library enables us to functionally test relevant mechanisms of drug tolerance in vivo and we expect that these analyses will provide useful insights into the tumor-stroma interactions that contribute to residual disease. Taken together, our detailed analyses demonstrate the substantial remodeling of tumor cells in their microenvironment upon treatment. To develop new therapeutic approaches to eradicate drug-tolerant tumor cells and thereby circumvent tumor relapse, it is essential to better understand the heterogeneous cellular composition of residual tumors in its spatial context. With this project, we hope to provide comprehensive data to develop better therapeutic strategies that target the tumor-stroma interaction and eradicate residual tumors.
 Citation Format: Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, Ismar Klebic, Sven Rottenberg. Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecf546e6bc5a8fa717c4c87fcb379e8ed3399d7" target='_blank'>
              Abstract C034: Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors
              </a>
            </td>
          <td>
            Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, I. Klebic, Sven Rottenberg
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
A spatially resolved, niche-level analysis of tumour microenvironments (TME) can provide insights into cellular interactions and their functional impacts in gastric cancers (GC).


OBJECTIVE
Our goal was to translate the spatial organisation of GC ecosystems into a functional landscape of cellular interactions involving malignant, stromal and immune cells.


DESIGN
We performed spatial transcriptomics on nine primary GC samples using the Visium platform to delineate the transcriptional landscape and dynamics of malignant, stromal and immune cells within the GC tissue architecture, highlighting cellular crosstalks and their functional consequences in the TME.


RESULTS
GC spatial transcriptomes with substantial cellular heterogeneity were delineated into six regional compartments. Specifically, the fibroblast-enriched TME upregulates epithelial-to-mesenchymal transformation and immunosuppressive response in malignant and TME cells, respectively. Cell type-specific transcriptional dynamics revealed that malignant and endothelial cells promote the cellular proliferations of TME cells, whereas the fibroblasts and immune cells are associated with procancer and anticancer immunity, respectively. Ligand-receptor analysis revealed that CCL2-expressing fibroblasts promote the tumour progression via JAK-STAT3 signalling and inflammatory response in tumour-infiltrated macrophages. CCL2+ fibroblasts and STAT3-activated macrophages are co-localised and their co-abundance was associated with unfavourable prognosis. We experimentally validated that CCL2+ fibroblasts recruit myeloid cells and stimulate STAT3 activation in recruited macrophages. The development of immunosuppressive TME by CCL2+ fibroblasts were also validated in syngeneic mouse models.


CONCLUSION
GC spatial transcriptomes revealed functional cellular crosstalk involving multiple cell types among which the interaction between CCL2+ fibroblasts and STAT3-activated macrophages plays roles in establishing immune-suppressive GC TME with potential clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe72f4555eff64ae7e607a98dd2b2dd8cb471c6e" target='_blank'>
              Spatial dissection of tumour microenvironments in gastric cancers reveals the immunosuppressive crosstalk between CCL2+ fibroblasts and STAT3-activated macrophages.
              </a>
            </td>
          <td>
            Sung Hak Lee, Dagyeong Lee, Junyong Choi, Hye Jeong Oh, In-Hye Ham, Daeun Ryu, Seo-Yeong Lee, Dong-Jin Han, Sunmin Kim, Youngbeen Moon, In-Hye Song, K. Song, Hyeseong Lee, Seungho Lee, Hoon Hur, Tae-Min Kim
          </td>
          <td>2024-11-23</td>
          <td>Gut</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and malignant type of brain tumor in adults and no effective therapies exist to combat this disease. Surgical resection, radio- and temozolomide-based-chemotherapy are insufficient mainly due to inter- and intra-tumoral heterogeneity of GBM tumors, therefore personalized approaches are required. Additionally, diverse tumor microenvironment with a leading role of glioblastoma stem cells (GSCs) in hypoxic core underlie the aggressiveness and recurrence of GBM. However, the molecules and pathways which directly promote tumor aggressiveness together with the driving process of epithelial-to-mesenchymal transition (EMT) and the components of extracellular matrix (ECM), remain only partially covered with prior studies focused mainly on coding oncogenes and tumor suppressor genes. Over the past years, a special attention has been paid to non-coding, circular RNAs (circRNAs) differentially expressed in various cancers, including GBM. Despite high abundance and stability, the exact biological roles of most individual circRNAs have not been fully revealed, so far. Besides anti-cancer targets, circRNAs have been considered as essential biomarkers and stratifying criterion of malignancy. Thus, studies on functions of differentially expressed circRNAs in GBM are an emerging field to be explored. In herein study, the biological role of circRNA CLIP2 (circCLIP2) was found highly overexpressed in primary tumor tissues. Upon the knock-down of circCLIP2 in 2D and 3D in vitro models, functional tests performed clearly indicate a significant contribution of circCLIP2 to aggressive potential of GBM as showed by decreased rates of proliferation, migration and invasion. Additional studies indicated predominant expression of circCLIP2 in GSC fractions additionally elevated by hypoxic conditions. The increased rates of proliferation and migration as well as molecular analyses were also confirmed in in vivo settings where xenograft model was applied. The expression of EMT marker Snail2 and ECM-degrading enzyme metalloproteinase 9 (MMP9) were significantly downregulated in the tumors formed from circCLIP2 silenced cells. Taken together, these results indicate a strong contribution of circCLIP2 in the aggressive phenotype of GBM which is due to a modulation of GSC proliferative potential as well as migration and invasion via EMT together with the modulator of ECM composition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41e1172182008f639a17a4a3feb76b12f54727b" target='_blank'>
              Circular RNA - circCLIP2 is predominantly expressed in GSC niche and enhances glioblastoma aggressiveness via EMT and ECM signaling
              </a>
            </td>
          <td>
            J. Misiorek, Ż. Zarębska, K. Kuczyński, J. Latowska-Łysiak, A. Grabowska, P. Głodowicz, M. Sajek, Anna Karlik, Dorota Wronka, A. Barciszewska, Łukasz Przybył, K. Rolle
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="

 Introduction: Fludarabine, a purine nucleotide analogue, is integral to preparative regimens for CAR-T cell therapy and allo-SCT in treating B-cell pediatric acute lymphoblastic leukemia (B-ALL). By disrupting DNA synthesis and repair, fludarabine induces apoptosis in proliferating leukemia cells. It also plays a vital role in immunosuppression, lymphodepletion, and tumor burden reduction, optimizing the efficacy of subsequent therapies. Despite its critical role, the molecular impact of fludarabine on leukemia cells and its efficacy factors remain inadequately understood.
 Methods: We utilized a cutting-edge single-cell spatial proteomics assay, Molecular Pixelation (MPX, Karlsson et al. Nat Methods 2024), an optics-free DNA sequencing-based method to analyze the cell surface interactome in response to fludarabine. The glucocorticoid-resistant ETV6::RUNX1 cell line REH and the BCR::ABL1 cell line SUP-B15 were incubated with and without fludarabine for 72 hours at varying concentrations, with repeated technical repetitions. Approximately 1,000 surviving single cells per condition were analyzed using the MPX assay (Pixelgen Technologies), wherein antibody-oligonucleotide conjugates generated nanometer-sized molecular pixels, read out by sequencing on a NovaSeq X Plus instrument (Illumina). Spatial proteomics networks for 76 proteins were computationally reconstructed from DNA-sequencing reads by inferring the relative locations of antibody-oligonucleotide conjugates, creating ~1,000 spatially connected neighborhoods per cell. Orthogonal immunocytochemistry (ICC) data were generated to validate the top protein interactions.
 Results: We have previously shown that ex-vivo drug screening in combination with single-cell transcriptome sequencing (scRNA-seq) readout can elucidate transcriptional effects of fludarabine (Gezelius et al. NAR Genomics and Bioinformatics 2024). This approach confirmed fludarabine's disruption of DNA synthesis and repair in dividing cells, and in addition revealed previously unknown extensive dysregulation of transcripts encoding cell surface proteins. The scRNA-seq data do not adequately resolve the spatial nor functional dynamics of the cell surface proteome, leaving this important aspect of response to fludarabine unexplored. In this study, we take advantage of recent advancements in single-cell spatial proteomics profiling to spatially elucidate how the cell surface proteome organization and function is modulated during fludarabine treatment.
 Spatial statistical analysis revealed novel patterns in cell surface organization during fludarabine treatment. The MPX assay identified 27 proteins with significantly altered abundance post-treatment (adj. Wilcoxon p < 0.01), consistent with the increased abundance of cell-surface proteins observed in our previous scRNA-seq data. Additionally, 26 proteins exhibited changes in polarity (adj. Wilcoxon p < 0.01) after fludarabine treatment, with most proteins shifting from a dispersed to a polarized state. We also detected 138 protein pairs with modified co-localization patterns. Notably, fludarabine enhanced the abundance, polarization, and co-localization of cadherin proteins CD82 and CD53, suggesting a potential therapeutic scaffold. This finding was further validated through orthogonal ICC experiments.
 Conclusions: This study advances our understanding of the spatial organization of the cell surface proteome in leukemia cells following fludarabine exposure. The findings provide new insights into fludarabine's effects at the single-cell level and may inform the development of more effective targeted therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df1ba5108871a33c10f05770da635143d8f7eec" target='_blank'>
              Mapping the Spatial Proteome of Individual Leukemia Cells Undergoing Fludarabine Treatment
              </a>
            </td>
          <td>
            J. Nordlund, M. Globisch, Henrik Gezelius, Anna Pia Enblad, O. Krali, Mariya Lysenkova-Wiklander, Martin Åberg, Arja Harila, A. Raine, Claes Andersson
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Introduction: Tumors systemically remodel the immune system during metastasis. Developing anti-metastatic immunotherapies that target these tumor-immune interactions to make the metastatic niche hostile colonization would represent a paradigm shift in cancer treatment. However, traditional high-throughput screening (HTS) platforms that use simple read-outs and contrived in vitro systems are ill-suited to identify such therapies. In contrast, HTS platforms that accurately model the metastatic niche and use high-dimensional read-outs such as multiplexed single-cell RNA-sequencing (scRNA-seq) can accurately profile nuanced perturbation responses in individual immune cell types and therefore have the potential to discover anti-metastatic immunotherapies. In this study, we describe the development of an ex vivo lung tissue culture HTS platform that preserves immune gene expression signatures observed in the in vivo metastatic niche and is amenable to single-cell chemical transcriptomic analysis using MULTI- seq (McGinnis et al., Nature Methods, 2019). Inspired by our recent description of myeloid cell TLR-NFκB inflammation during breast cancer lung metastasis (McGinnis et al., Cancer Cell, 2024), we then performed the first anti-metastatic immunotherapy drug screen and identified TLR-NFκB inhibitors that effectively operate on all desired myeloid cell types in the lung metastatic niche. Methods and Results: Building upon existing techniques for culturing precision-cut lung slices (Wu et al., Journal of Experimental Medicine, 2024) and patient- derived tumor fragments (Voabil et al., Nature Medicine, 2021), we established and optimized an ex vivo lung tissue culture system that enables 48-hour culturing of lung slices isolated from 4T1 tumor-bearing mice. scRNA-seq profiling revealed successful capture of all relevant immune cell types and retention of metastasis-associated gene expression programs (e.g., myeloid TLR-NFκB inflammation and neutrophil degranulation) following culture. We then scaled our platform to address how myeloid cells in the lung metastatic niche respond to perturbations targeting every component of the canonical TLR-NFκB signaling cascade. Cell-type-specific perturbation modeling and quantification of TLR-NFκB signaling inhibition identified compounds that optimally perturb all intended myeloid cell types (e.g., tissue-resident macrophages, neutrophils, and monocytes) and represent prioritized candidates for future in vivo validation studies to assess impact on metastatic disease progression. Conclusions: In this study, we describe the development and application of a novel HTS platform that couples ex vivo lung tissue cultures with single-cell chemical transcriptomics to identify anti-metastatic immunotherapy candidates. We demonstrate how our platform can identify TLR-NFκB signaling inhibitors that optimally operate in the lung metastatic niche, paving the way for future interrogation of additional metastasis-associated immune cell signaling pathways in different disease and tissue backgrounds.
 Citation Format: Chris McGinnis, Winnie Yao, Ansuman Satpathy. Anti-metastatic immunotherapy discovery using ex vivo lung tissue cultures and high-throughput single-cell chemical transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr PR017.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abfcae45218f099638f6a7c32802b5781110ec64" target='_blank'>
              Abstract PR017: Anti-metastatic immunotherapy discovery using ex vivo lung tissue cultures and high-throughput single-cell chemical transcriptomics
              </a>
            </td>
          <td>
            Chris McGinnis, Winnie Yao, Ansuman T. Satpathy
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="A noticeable characteristic of pancreatic ductal adenocarcinoma (PDAC) tumors is a dense tumor microenvironment with abundant and dense, desmoplastic stroma woven tightly with both cellular and matrix components. The high stromal density is associated with higher intratumor pressures which, until the last decade, was largely assumed to be tumor protective, confirmed by early studies demonstrating that altering the stroma was effective in genetically engineered models of PDAC. However, clinical trials using these approaches have been disappointing. There is increasing recognition that stroma heterogeneity is much greater than initially thought with an explosion of investigation into cancer-associated fibroblast (CAF) subpopulations led by experimental and single-cell transcriptomic studies. This review summarizes and attempts to harmonize the current transcriptomic data of CAF subpopulations. Understanding the heterogeneity of CAFs, the matrix, and other tumor microenvironment features will be critical to developing effective therapeutic approaches. Identifying model systems that best recapitulate the clinical behavior and treatment response of human PDAC will be important. Examining subpopulations as defined by clinical outcome will remain a critical step in defining clinically impactful CAF subtypes in larger clinical cohorts. The future of precision oncology in PDAC will depend on the integration of precision tumor epithelial and precision stroma approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a50e456e62579aeacb26fe2456d06aace9ecba04" target='_blank'>
              CAFomics: convergence to translation for precision stroma approaches.
              </a>
            </td>
          <td>
            Ian C McCabe, X. Peng, Joseph F Kearney, Jen Jen Yeh
          </td>
          <td>2024-11-08</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80cae72455d0f13cb986669bd3a06f266bb4aab" target='_blank'>
              Spatial transcriptomics reveals substantial heterogeneity in triple-negative breast cancer with potential clinical implications
              </a>
            </td>
          <td>
            Xiaoxiao Wang, D. Venet, F. Lifrange, D. Larsimont, Mattia Rediti, L. Stenbeck, F. Dupont, G. Rouas, Andrea Joaquin Garcia, L. Craciun, L. Buisseret, Michail Ignatiadis, M. Carausu, N. Bhalla, Yuvarani Masarapu, Eva Gracia Villacampa, L. Franzén, Sami Saarenpää, L. Kvastad, K. Thrane, J. Lundeberg, F. Rothé, Christos Sotiriou
          </td>
          <td>2024-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Population-scale resources of genetic, molecular, and cellular information form the basis for understanding human genomes, charting the heritable basis of disease, and tracing the effects of mutations. Pooled perturbation assays applied to cellular models, probing the effect of many perturbations coupled with an scRNA-seq readout (Perturb-seq), are especially potent references for interpreting disease-linked mutations or gene expression changes. However, the utility of existing maps has been limited by the comprehensiveness of perturbations possible, and the relevance of their cell line context. Here, we present the first genome-scale CRISPR interference (CRISPRi) perturbation map with single-cell RNA sequencing readout across many human genetic backgrounds in human pluripotent cells. To do so, we establish large-scale CRISPRi screening in human induced pluripotent stem cells from healthy donors, using over 20,000 guide RNAs to target 7,226 genes across 34 cell lines from 26 genetic backgrounds, and gather expression data from nearly 2 million cells. We comprehensively map trans expression changes induced by the target knockdowns, which complement co-expression patterns in unperturbed cells and facilitate the functional annotation of target genes to biological processes and complexes. Consistency of targeting protein complex members point to protein complexes as a nexus for aggregating transcriptional variation, revealing novel interaction partners. We characterise variation in perturbation effects across donors, with expression quantitative trait loci linked to higher genetic modulation of perturbation effects but overall low replication of trans effects due to knockdown of their corresponding cis regulators. This study pioneers population-scale CRISPR perturbations with single cell readouts that will fuel foundation models for the future of effective modulation of cellular disease phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5800ff9ee76e49e133d6ead67bfba5c5d87d6239" target='_blank'>
              A genome-scale single cell CRISPRi map of trans gene regulation across human pluripotent stem cell lines
              </a>
            </td>
          <td>
            Claudia Feng, Elin Madli Peets, Yan Zhou, Luca Crepaldi, S. Usluer, Alistair S. Dunham, Jana M. Braunger, Jing Su, Magdalena E Strauss, Daniele Muraro, Kimberly Ai Xian Cheam, M. Bonder, Edgar Garriga Nogales, Sarah Cooper, Andrew Bassett, Steven Leonard, Yong Gu, Bo Fussing, David Burke, Leopold Parts, O. Stegle, B. Velten
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the common malignant brain tumor in adults, accounting for approximately 15% of all CNS tumors, and 48.6% of malignant brain tumors, with a median survival of approximately 15 months. GBM is characterized by extensive inter- and intra-tumoral heterogeneity and an extremely immunosuppressive tumor microenvironment. Following Standard-of-Care surgical resection and chemoradiotherapy, patients inevitably relapse, at which point few therapeutic avenues exist, owing in part due to a lack of clinically relevant targets. Data from our target identification pipeline shows the urokinase plasminogen activator receptor (uPAR) is significantly upregulated at recurrence on putative GBM brain tumor initiating cells (BTICs), which are believed to drive de novo tumor formation, recurrence, and therapeutic resistance. uPAR plays an important role in the plasminogen activation system, and in the context of cancer, has been implicated in numerous pro-tumorigenic processes such as invasion, proliferation, and therapy resistance. We used CRISPR-Cas9 to genetically delete uPAR from our in-house patient-derived GBM cell lines, and found that knockout of uPAR in recurrent GBM cells significantly reduces various functional characteristics of BTICs in vitro. In our patient-derived mouse model of GBM, we found that knockout of uPAR significantly increases survival, and decreases tumor burden. Further, we generated novel anti-uPAR single domain antibodies (sdAbs) which specifically and efficiently bind uPAR at low nanomolar concentrations in vitro. We developed anti-uPAR CAR Ts using the binder sequence of the sdAbs, which showed potent cytotoxicity in vitro, and drastically reduced tumor burden and extended survival in vivo. Further, we identified uPAR as being highly expressed on brain metastases from multiple origins (lung-to-brain, melanoma-to-brain, etc.), and demonstrate that anti-uPAR CAR Ts effectively kill brain metastases in vitro, and significantly extend survival using in vivo brain metastasis models. From this work we believe uPAR to be a clinically relevant target in both recurrent GBM and brain metastases, and investigation into therapeutic strategies targeting uPAR-positive brain cancers should be explored further.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78706a62920d85268badacda235b4cef515101a8" target='_blank'>
              STEM-17. THE UROKINASE RECEPTOR AS A NOVEL IMMUNOTHERAPEUTIC TARGET IN BRAIN CANCERS
              </a>
            </td>
          <td>
            W. Maich, Muhammad Vaseem Shaikh, Anish Puri, Daniel Mobilio, Alisha Ananda, C. Venugopal, Martin A Rossotti, N. Mikolajewicz, C. Chokshi, Sabra K. Salim, N. Savage, T. Kislinger, Kevin A Henry, Sheila K. Singh
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Introduction: Properties of tumor angiogenesis impact on tumor growth and composition of the tumor microenvironment. Tumor vessels are composed of pericytes and other mural cells. Detailed characterization of the cellular composition of the microvasculature can suggest functional vessel and case properties relevant for outcome, response to treatment and immune surveillance. Results: Based on single cell RNA-seq from three human stage II/III colon cancers and pilot in-situ multi-antibody staining of six tumors two main clusters of perivascular cells were identified; B1 (MCAM+, MYH11-) and B2 (MCAM+, MYH11+) cells. B1 cells displayed pericyte features including expression of RGS5, whereas B2 cells showed high expression of smooth muscle cell markers like ACTG2 and Desmin. Desmin-high B2 cells were predominantly located in muscular layers, whereas Desmin-low/MYH11-high B2 cells showed perivascular enrichment. An extended novel analyses workflow of digital image analysis was used to profile peri-endothelial composition in 19 intervals; 11 1-µm intervals from 0 to 10 µm and 8 5-µm intervals from 10 to 50 µm from endothelial areas. Each of these expansion areas was quantitated regarding density of PDGFRB+/- cells including B1/B2 cells, and a MYH11+/MCAM- cells. Density of macrophage subsets (CD68-positive subsets defined by CD163 and CD11c) were also quantitated. In general, B1 cells showed stronger spatial association with endothelial cells than B2 cells, and this spatial association was stronger for the PDGFRB+ B1 subset. Survival analysis indicated that prognostic relevance was strongest when PDGFRB+ (favorable prognosis) cells were closest to endothelial cells. Regarding perivascular macrophage density, M0 and transit M1/M2 macrophages were predominantly enriched in the peri-endothelial regions, whereas density of M1 and M2 increased gradually as distance from vessels increased. M2 density was overall associated with poor prognosis, whereas high density of other macrophage subsets overall was associated with good prognosis. Ongoing studies are using a clustering-based approach to identify vessel subsets, and prognosis associations are being consolidated in additional cohorts. Summary: This study provides a novel approach for high resolution profiling of perivascular status in CRC, and possibly other tumor types, that could be of specific importance to identify tumor features associated with response to angiogenesis-directed therapies.
 Citation Format: Linglong( 凌 龙 ) Huang( 黄 ), Mercedes Herrera, Jonas Sjölund, Vladimir Chocoloff, Simon Joost, Rasul M. Tabiev, Lina Wik Leiss, Carina Strell, Luis Nunes, Artur Mezheyeuski, David Edler, Anna Martling, Fredrik Pontén, Bengt Glimelius, Tobias Sjöblom, Maria Kasper, Kristian Pietras, Arne Östman. Mapping the vascular walls of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a41b99632bcf5e6a6bbd94c1d54966c1ecf742fe" target='_blank'>
              Abstract B029: Mapping the vascular walls of colorectal cancer
              </a>
            </td>
          <td>
            Linglong(凌龙) Huang( 黄 ), Mercedes Herrera, Jonas Sjölund, Vladimir Chocoloff, S. Joost, Rasul M. Tabiev, Lina Wik Leiss, Carina Strell, Luis Nunes, A. Mezheyeuski, D. Edler, A. Martling, Fredrik Pontén, B. Glimelius, T. Sjöblom, Maria Kasper, Kristian Pietras, Arne Östman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eeb1135b7c75ad3978d67f0a6864e002ffcff1b" target='_blank'>
              Comprehensive single-cell aging atlas of healthy mammary tissues reveals shared epigenomic and transcriptomic signatures of aging and cancer.
              </a>
            </td>
          <td>
            B. Angarola, Siddhartha Sharma, Neerja Katiyar, Hyeon Gu Kang, Djamel Nehar-Belaid, SungHee Park, Rachel Gott, Giray N Eryilmaz, Mark A. LaBarge, Karolina Palucka, Jeffrey H. Chuang, Ron Korstanje, D. Ucar, Olga Anczukόw
          </td>
          <td>2024-11-25</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e009bbb823314845e352a79efaf448a32291661b" target='_blank'>
              Exploring heterogeneous cell population dynamics in different microenvironments by novel analytical strategy based on images
              </a>
            </td>
          <td>
            Yihong Huang, Zidong Zhou, Tianqi Liu, Shengnan Tang, Xuegang Xin
          </td>
          <td>2024-11-06</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This study delves into the proteomic intricacies of drug-resistant cells (DRCs) within prostate cancer, which are known for their pivotal roles in therapeutic resistance, relapse, and metastasis. Utilizing single-cell proteomics (SCP) with an optimized high-throughput Data Independent Acquisition (DIA) approach with the throughput of 60 sample per day, we characterized the proteomic landscape of DRCs in comparison to parental PC3 cells. This optimized DIA method allowed for robust and reproducible protein quantification at the single-cell level, enabling the identification and quantification of over 1,300 proteins per cell on average. Distinct proteomic sub-clusters within the DRC population were identified, closely linked to variations in cell size. The study uncovered novel protein signatures, including the regulation of proteins critical for cell adhesion and metabolic processes, as well as the upregulation of surface proteins and transcription factors pivotal for cancer progression. Furthermore, by integrating SCP and single-cell RNA-seq (scRNA-seq) data, we identified six upregulated and ten downregulated genes consistently altered in drug-treated cells across both SCP and scRNA-seq platforms. These findings underscore the heterogeneity of DRCs and their unique molecular signatures, providing valuable insights into their biological behavior and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3153689159ed447ea718eb77046a432729554a1" target='_blank'>
              Single-Cell Proteomic and Transcriptomic Characterization of Drug-Resistant Prostate Cancer Cells Reveals Molecular Signatures Associated with Morphological Changes
              </a>
            </td>
          <td>
            Jong-Su Woo, Michael A. Loycano, Md Amanullah, Jiang Qian, Sarah R. Amend, Kenneth J Pienta, Hui Zhang
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Recurrence and metastasis are the major issues for papillary thyroid cancer (PTC). Current morphological and molecular classification systems are not satisfied for PTC diagnosis due to lacking variant-specific morphological criteria and high signal-to-noise in mutation-based diagnosis, respectively. Importantly, intratumor heterogeneity is largely lost in current molecular classification system, which can be resolved by single cell RNA sequencing (scRNA-seq). However, scRNA-seq loses spatial information and morphological features. Herein, scRNA-seq is integrated and spatially-resolved transcriptomics (SRT) to elaborate the mechanisms underlying the spatial heterogeneity, malignancy and metastasis of PTCs by associating transcriptome and local morphology. This results demonstrated that PTC cells evolved with multiple routes, driven by the enhanced aerobic metabolism and the suppressed mRNA translation and protein synthesis and the involvement of cell-cell interaction. Two curated malignant and metastatic footprints can discriminate PTC cells from normal thyrocytes. Ferroptosis resistance contributed to PTC evolution. This results will advance the knowledge of intratumor spatial heterogeneity and evolution of PTCs at spatial and single-cell levels, and propose better diagnostic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/870c25c5831bd8c1742183d121d62d6970a89470" target='_blank'>
              Spatial and Single-Cell Transcriptomics Unraveled Spatial Evolution of Papillary Thyroid Cancer.
              </a>
            </td>
          <td>
            Guangzhe Zheng, Shaobo Chen, Wanqi Ma, Quanshu Wang, Li Sun, Changwen Zhang, Ge Chen, Shuping Zhang, Shuguang Chen
          </td>
          <td>2024-11-14</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Metastatic cancer affects millions of people worldwide annually and is the leading cause of cancer-related deaths. Most patients with metastatic disease are not eligible for surgical resection, and current therapeutic regimens have varying success rates, some with 5-year survival rates below 5%. Here, we test the hypothesis that metastatic cancer can be genetically targeted by exploiting single base substitution mutations unique to individual cells that occur as part of normal aging prior to transformation. These mutations are targetable because ~10% of them form novel tumor-specific “NGG” protospacer adjacent motif (PAM) sites targetable by CRISPR-Cas9. Methods Whole genome sequencing was performed on five rapid autopsy cases of patient-matched primary tumor, normal and metastatic tissue from pancreatic ductal adenocarcinoma decedents. CRISPR-Cas9 PAM targets were determined by bioinformatic tumor-normal subtraction for each patient and verified in metastatic samples by high-depth capture-based sequencing. Results We found that 90% of PAM targets were maintained between primary carcinomas and metastases overall. We identified rules that predict PAM loss or retention, where PAMs located in heterozygous regions in the primary tumor can be lost in metastases (private LOH), but PAMs occurring in regions of loss of heterozygosity (LOH) in the primary tumor were universally conserved in metastases. Conclusions Regions of truncal LOH are strongly retained in the presence of genetic instability and, therefore, represent genetic vulnerabilities in pancreatic adenocarcinomas. A CRISPR-based gene therapy approach targeting these regions may be a novel way to genetically target metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada3070ea15ff6901b235b693417e61df449621f" target='_blank'>
              Islands of genomic stability in the face of genetically unstable metastatic cancer
              </a>
            </td>
          <td>
            Kirsten Bowland, Jiaying Lai, Alyza M. Skaist, Yan Zhang, S. Teh, Nicholas J. Roberts, Elizabeth Thompson, Sarah Wheelan, Ralph H. Hruban, Rachel Karchin, Matthew H Bailey, Christine A. Iacobuzio-Donahue, J. Eshleman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Introduction Immune dysregulation plays a major role in cancer progression. The quantification of lymphocytic spatial inflammation may enable spatial system biology, improve understanding of therapeutic resistance, and contribute to prognostic imaging biomarkers. Methods In this paper, we propose a knowledge-guided deep learning framework to measure the lymphocytic spatial architecture on human H&E tissue, where the fidelity of training labels is maximized through single-cell resolution image registration of H&E to IHC. We demonstrate that such an approach enables pixel-perfect ground-truth labeling of lymphocytes on H&E as measured by IHC. We then experimentally validate our technique in a genetically engineered, immune-compromised Rag2 mouse model, where Rag2 knockout mice lacking mature lymphocytes are used as a negative experimental control. Such experimental validation moves beyond the classical statistical testing of deep learning models and demonstrates feasibility of more rigorous validation strategies that integrate computational science and basic science. Results Using our developed approach, we automatically annotated more than 111,000 human nuclei (45,611 CD3/CD20 positive lymphocytes) on H&E images to develop our model, which achieved an AUC of 0.78 and 0.71 on internal hold-out testing data and external testing on an independent dataset, respectively. As a measure of the global spatial architecture of the lymphocytic microenvironment, the average structural similarity between predicted lymphocytic density maps and ground truth lymphocytic density maps was 0.86 ± 0.06 on testing data. On experimental mouse model validation, we measured a lymphocytic density of 96.5 ± %1% in a Rag2+/- control mouse, compared to an average of 16.2 ± %5% in Rag2-/- immune knockout mice (p<0.0001, ANOVA-test). Discussion These results demonstrate that CD3/CD20 positive lymphocytes can be accurately detected and characterized on H&E by deep learning and generalized across species. Collectively, these data suggest that our understanding of complex biological systems may benefit from computationally-derived spatial analysis, as well as integration of computational science and basic science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85260d40c721d3e1f8d5b2764225eb5a4620514b" target='_blank'>
              Computational staining of CD3/CD20 positive lymphocytes in human tissues with experimental confirmation in a genetically engineered mouse model
              </a>
            </td>
          <td>
            Xiang Li, Casey C. Heirman, Ashlyn G Rickard, Gina Sotolongo, Rico Castillo, Temitayo Adanlawo, Jeffery I. Everitt, Jeffery B. Hodgin, Tammara L. Watts, Andrew Janowczyk, Yvonne Mowery, Laura Barisoni, K. Lafata
          </td>
          <td>2024-10-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 Background: Acute myeloid leukemia (AML) is a heterogeneous disease characterized by arrested differentiation and clonal expansion. While certain efforts have been devoted to integrating single-cell data in AML, there is a clear need to build an interactive atlas fully annotated with clinical, demographic, molecular, cytogenetic and treatment data, providing comprehensive sources and customized functions for analysis. To mend this gap, we developed an interactive Comprehensive Atlas of Single cells in AML (iCASA) platform which can enhance the understanding of the cellular communication, transcriptomic regulation, immune repertoires, and cell population changes over the course of AML evolution in various treatment settings.
 Methods: Single cell RNA profiling (scRNAseq) of 73 bone marrow (BM) samples from 43 AML patients and 9 healthy BM samples including our inhouse previously published and newly generated data were included. Batch effects were evaluated by Harmony. Cell types were annotated by integration of canonical markers, flow cytometry, conventional cytogenetics, inferCNV, FISH, and manual revision. Reference mapping of AML cells was projected by Symphony. Data-sharing platform Shiny App was built in R for interactive use.
 Results: AML BM samples included 22 newly diagnosed patients who received venetoclax-based therapy, 13 newly diagnosed patients who received frontline therapy without venetoclax and 8 relapsed/refractory patients. Of those, 19 patients had paired samples (2 to 3) pre and post therapy, providing longitudinal assessment of AML-immune cell interactions. The median age of the population at time of diagnosis was 71 years (range 38 - 89). The most frequent mutation was TP53 (39%), followed by IDH2 (23%), and DNMT3A (23%) and 44% patients had del7/7q. After preprocessing, 404,819 high-quality cells were retained. To facilitate the public utility of our resource and drive future discoveries, the iCASA features were divided into 8 tabs: 1) Dimensionality reduction visualizes distribution of cells by UMAP or tSNE where pathways of interest can be projected to assess biological functions in different cellular states. 2) Differential expression analysis identifies differential genes across cell types, normal versus tumor, with interactive visualization such as heatmaps and volcano plots. 3) Correlation analysis performs pairwise gene correlation analysis at pseudo-bulk level for modules of interest. 4) Signature Scoring of user-provided gene sets or predefined pathway signatures enable users to look at pathway activities across different cell types and clinical subgroups. 5) Survival analysis provides overall survival curve based on gene expression, pathway scores, or gene regulator modules (including transcription factor activity) 6) Cell-cell communication predicts ligand-receptor interaction between cell types by CellChat. Visualization includes circle plot, chord diagram, and heatmap. 7) Trajectory analysis determines the dynamic process of cell differentiation. 8) Immune repertoires (scTCR and scBCR) paired with immune cell subtypes to examine markers of immune response.
 The iCASA platform will be available on a free web-access portal for users to explore. The interactive interface makes it accessible even to those without prior experience in computational biology. Additionally, users can download the raw data for use on local machines. As an application of the portal, we identified 12 broad cell clusters and 38 specific cell states spanning hematopoietic stem cells to differentiated cells. T cells were the dominant lineage in the niche (56%). AML cells were mapped by a linear mixture model to hematopoietic cells from 9 healthy donors to determine their cellular states. The overall cellular states showed significant diverse distribution across patients (p < 2.2x10-16), indicating high intra-tumoral heterogeneity of AML patients. The web portal is now in demo mode and expected to go live in January 2025.
 Conclusions: We provided a large scRNAseq resource of hematopoietic differentiation in AML patients. This integrated information is a valuable resource to characterize dynamic changes in AML patients with chemotherapy, understand response to combined therapies. Ongoing analyses to derive biological insights include identifying meta clusters associated with cellular states, evaluating dynamic changes in different myeloid subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d49617268780472c259a570fae0d0b19343e04" target='_blank'>
              ICASA: An Interactive Comprehensive Atlas of Single Cells in AML from 82 Bone Marrow Samples
              </a>
            </td>
          <td>
            Bofei Wang, Poonam N Desai, Jae Jun Ku, N. Daver, P. Reville, Hussein A. Abbas
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/150e626daa762108097ae88ebcf26c0674670dd5" target='_blank'>
              Isolation and identification of patient-derived liver cancer stem cells and development of personalized treatment strategies
              </a>
            </td>
          <td>
            Tingting Guo, Shuai Zhang, Weiping Zeng, Yan Liang, Jinghe Xie, Shoupei Liu, Yaqi Qiu, Ying Fu, Y. Ou, Keqiang Ma, Bailin Wang, Weili Gu, Yuyou Duan
          </td>
          <td>2024-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b397d7b281fa7e047bec3772e9d91f03230a0f4" target='_blank'>
              Profiling the cancer-prone microenvironment in a zebrafish model for MPNST.
              </a>
            </td>
          <td>
            Cheryl Cero, John S. House, Vincenzo Verdi, J. L. Ferguson, Dereje D. Jima, Aubrie A. Selmek, Olivia M. Patania, Jennifer E. Dwyer, Bih-Rong Wei, Dillon Lloyd, H. Shive
          </td>
          <td>2024-11-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="

 The clonal architecture of acute myeloid leukemia (AML) can be highly complex and heterogeneous. This genetic diversity complicates the treatment of AML, as different clones may respond variably to therapies, contributing to treatment resistance and disease relapse. Understanding the mechanisms and implications of clonal evolution in AML is crucial for developing more effective treatment strategies that can target the diverse cellular populations.
 The generation of patient-derived xenograft (PDX) models has revolutionized the study of AML. One of the main uses of PDX models is to study how a leukemic sample might respond to different therapies. Here, we used single-cell DNA sequencing (scDNA-seq) and surface protein profiling to monitor the clonal evolution and differentiation of an isocitrate dehydrogenase 1 (IDH1)-mutated leukemic sample in response to ivosidenib (IVO), a mutant IDH1 inhibitor, either as monotherapy or in combination with venetoclax (VEN) or azacitidine (AZA). Single-cell mutation analysis of the engrafted cell population showed that the initial evolutionary path was linear characterized by the sequential acquisition of DNMT3AR882H (clone 1), WT1R467W (clone 2), IDH1R132H (clone 3), NPM1Trp288Cysfs*12 (clone 4), FLT3-ITD (clone 5), and WT1R439C (clone 6) mutations. Three subclones branched off at the end of this linear path with each acquiring either a TET2Q1548del (clone 7), DNMT3AD531del (clone 8), or KITH40QfsTer6 (clone 9) mutation. The overall clonal composition of the input primary sample and sample from vehicle-treated animals were largely similar. To quantify the differences in therapeutic response between clones, we developed a hierarchical multinomial Bayesian model that estimated the fold change in absolute cell numbers for each clone compared with vehicle treatment. This analysis revealed that both single-agent VEN and IVO+VEN were generally more effective in targeting earlier-stage clones (clones 3-5) than later-stage clones (clones 6-9). In contrast, single-agent AZA and IVO+AZA were effective in eliminating clone 6 as well as earlier-stage clones. Notably, the combination of IVO+AZA was effective in reducing all the clones by approximately the same magnitude, including clones 7 and 8 which demonstrated lower sensitivity to AZA monotherapy. Given that the WT1R439C mutation, acquired in clone 6, distinguished early-stage from late-stage clones, these findings suggest that WT1 mutations might contribute to IVO and VEN resistance and that the addition of AZA can overcome this mechanism of resistance. Consistent with these findings, the combination of IVO+AZA strongly upregulated the expression of the myeloid markers CD11b, CD14, and CD15 to a greater extent than with either single agent. These findings provide evidence that AZA and IVO could synergize to overcome the differentiation block in IDH1-mutated AML cells, reflecting the superior clinical efficacy of this regimen over single-agent IVO.
 To broaden the applicability of PDX models to study the competition between genetic clones that co-exist infrequently in an individual sample, we generated mixed PDX (mPDX) models by co-engrafting two or more leukemic samples in the same animal and used scDNA-seq to deconvolute the clonal composition and sample origin of the engrafted cells. As proof-of-principle, we generated an IDH1R132H/IDH2R140Q mPDX to model isoform switching in both directions by treating the animals with either IVO or enasidenib (ENA), a mutant IDH2 inhibitor. We observed the expected differentiation response and depletion of IDH1-mutated or IDH2-mutated clones by IVO or ENA treatment as single agents, respectively. Further studies are underway to demonstrate whether dual mutant IDH1 and IDH2 inhibition (IVO+ENA) will circumvent resistance driven by isoform switching.
 In summary, we demonstrate the utility of applying single-cell proteogenomic analysis in traditional and mixed PDX models to gain crucial insights into mechanisms of resistance and potential strategies to overcome it.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad2ac5a87afb275fd53cef27ee87a6e28e0b788" target='_blank'>
              Single-Cell Proteogenomic Analysis of Clonal Evolution in Patient-Derived Xenograft Models of AML Treated with IDH Inhibitors
              </a>
            </td>
          <td>
            Alex Liu, S. Cathelin, Yitong Yang, Farzaneh Aboualizadeh, Amina Abow, G. Basi, Lance Li, David L Dai, Abdula Maher, Eric Grignano, Mohsen Hosseini, Dhanoop Manikoth Ayyathan, Vivian Wang, Troy Ketela, Dylan M Marchione, Adriana Tron, A. Arruda, Mark D. Minden, Steven M Chan
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Background: Prostate cancer is a prevalent malignancy in men, with treatment resistance posing a significant challenge to successful cure. Acquired Resistance to Therapy (ART) arises from two main components: (1) Tumor Microenvironment (TME) and (2) mutations or epigenetic changes at gene level within the tumor cells themselves. While most cancer therapies target the intrinsic biology of cancer cells, growing evidence indicates that these treatments can also harm the stromal cells within TME, potentially driving ART. The biology of ART may vary based on the types of treatments, cell types, and organ locations. Thus, this study aims to dissect ART components driven by TME-mediated cellular processes in a comprehensive manner. Hypothesis: We propose that cancer therapies induce stress responses in non-cancerous cells within the TME. These stress responses involve the secretion of cytokines, mediators, and growth factors, which we term Stress Response Secretory Programs (SRSPs). We further hypothesize that targeting SRSPs could overcome ART that helps to improve treatment specificity and efficacy. Materials and Methods: We employed comprehensive single-cell profiling of common metastatic sites included liver, prostate, lung, bone, lymph node and spleen in preclinical models to identify secreted proteins and dysregulated pathways caused by genotoxic therapies. In vitro, we evaluated treatment resistance in murine prostate epithelial cell lines (Myc-CAP) by exposing them to conditioned media (CM) from irradiated murine fibroblasts (F4M2), using cell growth assays. Results: Single-cell RNA sequencing of organ samples revealed significant expressions of SRSPs and increased inflammatory responses in liver and prostate fibroblasts as well as other stromal cells treated with carboplatin. In vitro, CM from irradiated F4M2 cells enhanced cell proliferation and survival of Myc-CAP cells in response to docetaxel treatment. Conclusion: Our findings highlight the crucial role of SRSPs in mediating ART through interactions within the TME, which alter tumor cell phenotypes. Targeting these SRSPs holds potential for overcoming treatment resistance and improving therapeutic outcomes, underscoring the need to consider TME complexity in cancer treatment strategies.
 Citation Format: Tony Lok Heng Chu, Armand Bankhead, Ilsa Coleman, Sander Frank, Peter S. Nelson, Tarana Arman. Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c940709273c954da8cb4bd8042d98f46649ce8ee" target='_blank'>
              Abstract C033: Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment
              </a>
            </td>
          <td>
            Tony Lok Heng Chu, Armand Bankhead, Ilsa M Coleman, Sander Frank, Peter S. Nelson, Tarana Arman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the past century, colorectal cancer (CRC) has become one of the most devastating cancers impacting the human population. To gain a deeper understanding of the molecular mechanisms driving this solid tumor, researchers have increasingly turned their attention to the tumor microenvironment (TME). Spatial transcriptomics and proteomics have emerged as a particularly powerful technology for deciphering the complexity of CRC tumors, given that the TME and its spatial organization are critical determinants of disease progression and treatment response. Spatial transcriptomics enables high-resolution mapping of the whole transcriptome. While spatial proteomics maps protein expression and function across tissue sections. Together, they provide a detailed view of the molecular landscape and cellular interactions within the TME. In this review, we delve into recent advances in spatial biology technologies applied to CRC research, highlighting both the methodologies and the challenges associated with their use, such as the substantial tissue heterogeneity characteristic of CRC. We also discuss the limitations of current approaches and the need for novel computational tools to manage and interpret these complex datasets. To conclude, we emphasize the importance of further developing and integrating spatial transcriptomics into CRC precision medicine strategies to enhance therapeutic targeting and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b317770be242dc222a3004766249a5b9de43dcc" target='_blank'>
              Current state and future prospects of spatial biology in colorectal cancer
              </a>
            </td>
          <td>
            Francisco G. Carranza, Fernando C. Diaz, Maria Ninova, Enrique Velazquez-Villarreal
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Introduction:
 A precise depiction of intratumoral clonal diversity reveals the processes of clonal evolution and diversity within tumors. Comprehensive genetic analyses and inference of clonal structure in B-cell lymphomas, such as diffuse large B-cell lymphomas (DLBCL) and follicular lymphomas (FL), have been performed using variant allele fractions (VAFs) obtained from bulk samples. However, this method is limited in terms of predicting precise clonal structure, cell-level co-occurrence, and chronological order of genetic alterations. Moreover, the clonal relationship between the genotype and phenotype at the single-cell level has not been well characterized in B-cell lymphomas.
 Methods:
 Simultaneous single-cell DNA sequencing (scDNA-seq) and cell surface protein analyses were performed using the Tapestri platform (Mission Bio). We analyzed 32 samples from 30 patients with B-cell lymphomas, consisting of 12 DLBCLs and 18 FLs. In two cases, both lymph node and bone marrow tumor samples were analyzed. We designed 298 amplicons to cover 61 recurrently mutated genes in FL and DLBCL samples. We also selected 68 custom antibodies with oligotags to capture cell surface protein expression related to B cell differentiation (e.g., CD10, CD19, CD20, CD138, and CD83) and interactions with the tumor microenvironment (e.g., MHC PD-1, PD-L1, CTLA-4, LAG3, BTLA, and LIGHT). The sequenced data were processed using the Tapestri Pipeline for variant calling and multiple filtering processes (Miles L et al., Nature 2020). Phylogenetic trees were inferred using the COMPASS package (Sollier E et al., Nat Commun 2023).
 Results:
 We sequenced a median of 6,501 cells per sample (range: 1,039-18,099). A total of 223,336 cells were annotated using conventional B/T cell lineage markers to identify B cells (120,384), CD4-T-cells (38,863), and CD8-T-cells (21,523), with a median coverage of 78 reads per cell per amplicon. Through scDNA-seq, 97 variants were detected (median: 3.5; range: 1-10 per sample) which were used for further analysis. Notably, the average VAFs derived from scDNA-seq exhibited good concordance with the VAFs obtained from matched bulk whole-exome sequencing data (n=15), confirming the reliability of scDNA-seq (R=0.89).
 The most frequent mutations identified through scDNA-seq in DLBCL cases were CREBBP, TP53, B2M, and MYD88 (n=4, 33%), and CARD11 and HIST1H1E (n=3, 25%). In FL cases, CREBBP mutations were the most frequently detected (n=8, 44%), followed by EZH2, TNFRSF14, and STAT6 mutations (n=4, 23%). Notably, all CREBBP mutations detected in FL cases were found in the initiating cells of pyrogenic trees, suggesting a strong role in initiating mutations in FL. MEF2B, HIST1H1E, and GNA13 mutations were also found in the initiating cells, whereas KMT2D mutations were mainly detected in subclonal cells in FL. Consistent with the findings of previous bulk studies, all TP53 mutations were found in the subclonal cells in both DLBCL and FL cases. Strikingly, all six STAT6 mutations (D419G (n=3), D419N, N421K, and N430T) were detected in subclonal cells, suggesting a contribution to clonal expansion and progressive FL disease.
 Spatially separated tumors were found to share founder genetic alterations at the single-cell level, and additional alterations were acquired in lymph node (e.g., CREBBP CN-LOH and CTSS Y132D), but not in bone marrow.
 scDNA-seq also provided information on co-occurrence and mutual exclusiveness at the cellular level. In the case of both MYD88 (L265P) and CD79B mutations (MCD-type DLBCL), although these two gene mutations were concurrently found in most lymphoma cells, 39 of 1,523 lymphoma cells had only MYD88 mutation. Similarly, in an FL case harboring both CREBBP and EP300 mutations, these mutations diverged into two independent branches, indicating mutual exclusion at the cellular level.
 Conclusion:
 scDNA sequencing revealed the clonal structures and evolutionary processes of B-cell lymphomas at the cellular level. Specifically, some driver mutations were found to be mutually exclusive, which contrasts with the findings from previous bulk-sequencing studies that showed their co-occurrence at the patient level. Simultaneous profiling of single-cell DNA and cell surface proteins is ongoing, which will provide further insights into the genetic and phenotypic profiles of intratumor heterogeneity in B-cell lymphomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a085eaf5798489d72f8e4a8edabf00a264c2c0e1" target='_blank'>
              Single-Cell DNA Sequencing Reveals Cell-Level Genetic Evolutionary Processes and Clonal Structure in B-Cell Lymphomas
              </a>
            </td>
          <td>
            Hiroyuki Murakami, R. Chijimatsu, H. Ujiie, Y. Naoi, Kazutaka Sunami, Kazuhiko Yamamoto, Soichiro Fujii, Tomofumi Yano, K. Seike, Hideaki Fujiwara, Noboru Asada, N. Fujii, Ken-Ichi Matsuoka, T. Urata, Kazuhiro Ikeuchi, Hiroki Kobayashi, Yasuharu Sato, Shuta Tomida, Y. Maeda, D. Ennishi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Melanoma, a highly aggressive and life-threatening cancer, poses a significant therapeutic challenge, particularly for patients with distant metastasis, who face a 5-year survival rate of only approximately 22.5%. This underscores the urgent need for innovative therapeutic approaches. Emerging evidence suggests that sensory and parasympathetic innervation within the tumor microenvironment (TME) plays a crucial role in driving metastasis and immune evasion, partly through the promotion of M2 macrophages and cancer stem cells. To uncover the mechanisms behind tumor-specific innervation in metastatic melanoma, we employed a systems biology approach integrating bulk RNAseq from diverse melanoma cell lines, TCGA-SKCM data analysis, single-cell RNAseq, and spatial transcriptomics. This comprehensive analysis led us to identify a previously uncharacterized gene regulatory network (GRN) orchestrating tumor innervation. This GRN is driven by axon guidance cues (Netrin, Slit, Semaphorin), their receptors (DCC, Robo, Neuropilin, ITGB1), and transcription factors (RAS, RND1, TFAP2, CAMKII). We further discovered that the pioneer factor FOXC2, known for its association with metastasis, plays a critical role in opening the promoters of innervation-specific genes. Additionally, we identified Ephrin and CXCR4-mediated crosstalk between neurons and tumor cells. Machine learning models validated the accuracy of this GRN in predicting melanoma metastasis, and a neural network-based model confirmed the enrichment of nerve cells near metastatic tumor cells within the TME in TCGA immunohistochemistry data. The functional relevance of this network was further supported by CRISPR-mediated knockout studies, which disrupted tumor-specific signaling pathways. Notably, we also observed the enrichment of genes linked to mRNA processing bodies and neuron-specific membrane-less condensates in metastatic melanoma. These findings highlight the critical role of neuronal infiltration in melanoma metastasis. Currently, therapeutic strategies targeting this aspect of the TME remain limited. Our identified gene set offers potential as a metastatic predictor, and targeting the neural component of the TME represents a promising new therapeutic avenue for melanoma.
 Citation Format: Anjana Goli, Subhajit Dutta. Deciphering the neurogenic tumor microenvironment: A novel gene network driving innervation in metastatic melanoma offers prognostic and therapeutic insights [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a5d79cdb084a3fb556d9c17280ebbf7e5cef77" target='_blank'>
              Abstract B024: Deciphering the neurogenic tumor microenvironment: A novel gene network driving innervation in metastatic melanoma offers prognostic and therapeutic insights
              </a>
            </td>
          <td>
            Anjana Goli, Subhajit Dutta
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Circulating Tumor Cells (CTCs) hold the promise of comprehensive yet noninvasive molecular characterization of cancer cells during the course of tumor evolution and progression. To dissect the epigenetic profile of prostate CTCs, we undertook paired DNA methylation and RNA sequencing of single CTCs isolated from blood specimens of patients with prostate cancer. Across the genome, our study identified 40 core Partial Methylation Domains (PMDs) that arise early in tumorigenesis and are shared by all individual CTCs across multiple patients. Key among these is a single genomic locus harboring multiple interferon-inducible genes along with all the CD1 gene family members, implicated in innate immunity. Based on mouse models, we propose that early biallelic silencing of multiple genes residing within a single PMD may contribute to the escape from immune surveillance in prostate and potentially other cancers. Studies of CTC heterogeneity within individual patients are limited by the very small number of cells that can be isolated from standard 10ml blood volumes. To enable such analyses, we applied a new high throughput microfluidic platform, capable of processing an entire leukapheresis product, interrogating 2-3 L of blood volume equivalent. Applying this technology to patients with metastatic prostate cancer, we successfully enriched thousands of CTCs from individual cases, enabling the characterization of intra-patient heterogeneity at the level of cell morphology and marker expression, chromosome copy number variation and exome-wide genotyping, and transcriptionally-defined tumor cell subpopulations. Such high-volume microfluidic enrichment of CTCs constitutes a new dimension in liquid biopsies.
 Citation Format: Daniel A Haber, Shyamala Maheswaran, Hongshan Guo, Joanna Vuille, Shih-Bo Huang, Ben S Wittner, Lecia V Sequist, Richard J Lee, Patricia AR Brunker, David T Miyamoto, Avanish Mishra, Mehmet Toner. Tumor cell-based liquid biopsy to characterize prostate cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr IA005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe44dae01b6a504aeda595e4d36a826d45670615" target='_blank'>
              Abstract IA005: Tumor cell-based liquid biopsy to characterize prostate cancer
              </a>
            </td>
          <td>
            D. A. Haber, S. Maheswaran, Hongshan Guo, Joanna A. Vuille, Shih-Bo Huang, Ben S. Wittner, L. Sequist, Richard J. Lee, Patricia A. R. Brunker, D. Miyamoto, Avanish Mishra, Mehmet Toner
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4788ed4991cb05a8733cec69abe88f6738bf0a69" target='_blank'>
              Differential chromatin accessibility and transcriptional dynamics define breast cancer subtypes and their lineages
              </a>
            </td>
          <td>
            Michael D. Iglesia, R. Jayasinghe, Siqi Chen, Nadezhda V. Terekhanova, John M Herndon, E. Storrs, A. Karpova, D. Zhou, Nataly Naser Al Deen, Andrew T Shinkle, Rita Jui-Hsien Lu, Wagma Caravan, Andrew Houston, Yanyan Zhao, Kazuhito Sato, Preet Lal, Cherease Street, Fernanda Martins Rodrigues, Austin N. Southard-Smith, A. L. N. Targino da Costa, Houxiang Zhu, Chia-Kuei Mo, Lisa Crowson, Robert S Fulton, Matthew A. Wyczalkowski, C. Fronick, L. Fulton, Hua Sun, Sherri R Davies, Elizabeth L. Appelbaum, S. Chasnoff, Madelyn Carmody, Candace Brooks, Ruiyang Liu, M. Wendl, C. Oh, Diane E Bender, C. Cruchaga, O. Harari, Andrea L Bredemeyer, K. Lavine, R. Bose, J. Margenthaler, Jason M. Held, Samuel Achilefu, F. Ademuyiwa, R. Aft, Cynthia X. Ma, G. A. Colditz, Tao Ju, Stephen T. Oh, James A J Fitzpatrick, E. S. Hwang, K. Shoghi, M. Chheda, D. Veis, Feng Chen, Ryan C. Fields, William E Gillanders, Li Ding
          </td>
          <td>2024-10-30</td>
          <td>Nature Cancer</td>
          <td>1</td>
          <td>65</td>
        </tr>

        <tr id="
 Desmosomes are transmembrane protein complexes critical for cell-cell adhesion in epithelial tissues and other cell types. In skin cancers like melanoma, epithelial cells called keratinocytes are the predominant cells in the primary melanoma microenvironment, yet it remains largely unknown whether this cell type is subject to genetic alterations during melanoma development and its role in early melanoma progression. In this study, using an integrative analysis of tumor mutations and protein biophysical interactions, we identify a high frequency of genetic alterations in desmosome-related genes in human cancers, with the highest occurrences in cutaneous melanoma. Notably, over 70% of cutaneous melanoma cases exhibit mutations in desmosome genes and desmosome gene mutations are associated with lowered expression of the desmosome complex in melanoma. Using spatial transcriptomics and immunohistochemistry analyses, we find that the decrease in desmosome gene expression predominantly occurs in keratinocytes within the tumor microenvironment rather than in the melanoma cells themselves. To study the functional significance of this alteration in melanoma development, we used a human melanoma/keratinocyte co-culture system using primary melanoma cell lines. Our studies show that knockdown of desmosome genes in keratinocytes markedly increased proliferation of adjacent melanoma cells in melanoma/keratinocyte co-cultures. Additionally, melanoma cell proliferation is enhanced when exposed to media preconditioned by desmosome-deficient keratinocytes. These observations suggest that the gradual accumulation of mutations in desmosome genes within neighboring keratinocytes may create a conducive environment that primes melanoma cells for neoplastic transformation. This study underscores the critical role of genetic alterations in the tumor microenvironment in melanoma progression and highlights the potential for targeting desmosome-related pathways in therapeutic strategies against melanoma.
 Citation Format: Mohita Tagore. Desmosome mutations in the microenvironment regulate melanoma proliferation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A042.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cede4aac61d054a3a470265c4e2db5a458ea7c19" target='_blank'>
              Abstract A042: Desmosome mutations in the microenvironment regulate melanoma proliferation
              </a>
            </td>
          <td>
            Mohita M. Tagore
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="

 Measurable residual disease (MRD) assessment is important in acute myeloid leukemia (AML), as post-remission MRD indicates higher relapse risk. Current techniques, however, lack precision in individual outcome prediction, complicated by AML's clonal heterogeneity. Novel single cell MRD (scMRD) workflows based on DNA mutations offer a potential breakthrough, simultaneously capturing genotypic and immunophenotypic changes at a single-cell level. The Tapestri® platform (Mission Bio) incorporates a microfluidic droplet technology with cell barcoding beads and gene-specific primers that enables amplification and comprehensive identification of low-frequency subclones that may lead to overt disease or resistance. With this approach, complex heterogeneity and differentiation between leukemic clones and clonal hematopoiesis can be deconstructed.
 Objective: This study represents the first large-scale scMRD analysis in AML using this novel scMRD technology to deconvolute clonal heterogeneity to identify potential relapse markers and to differentiate between leukemic clones and clonal hematopoiesis.
 Methods: We analyzed a cohort of 69 AML patients in first complete remission (CR) as assessed by bone marrow morphology from the BLAST trial (NCT02502968). For 11 patients, we analyzed paired samples from both first CR and relapse. We performed single-cell DNA + protein sequencing on a total of 250,000 cells across 23 multiplexed pools. Our methodology included 40 selected mutation hotspot genes and a 19-plex antibody oligonucleotide cocktail for AML-specific surface markers as well as patient-specific hash tag antibodies for enhanced demultiplexing. We used magnetic activated cell sorting (MACS) to enrich CD34/CD117-positive blasts to ensure high-quality samples for subsequent analysis.
 Results: Our analyses revealed a diverse genetic landscape with 25 different mutated genes. DNMT3A and NPM1:p.W228Cfs mutations were the most common alterations. The genetic distribution included a predominance of preleukemic genes (51.5%), chromatin remodelers (12.1%), splice factors (9.1%), transcription factors (6.1%), cell cycle regulators (6.1%), and other functional categories (15.2%).
 Lower MRD (and clonal hematopoiesis) levels were consistently associated with improved overall survival (OS) and relapse-free survival (RFS) across different thresholds (<5%, <10%, <20%, <30%, <40% and <50% MRD burden per patient), with patients in the <5% MRD group showing a significant difference in RFS (p = 0.0013). The median overall survival in this group was 18.0 months, while the median relapse-free survival was 11,7 months. OS and RFS remained significantly different in all MRD groups even after exclusion of DNMT3A and TET2 mutations, which most likely represented clonal hematopoiesis.
 In paired samples, TP53 mutations were notably prominent during follow-up, remaining the dominant clone, indicating a mixed composition, and the persistence of treatment resistant clones. Overall, our study highlighted the dynamic nature of clonal evolution under therapeutic pressure by demonstrating continuous clonal diversity during clinical remission and notable changes in clonal composition during relapse.
 Conclusion: In this AML cohort, our study provides insights into the genetic landscape and clonal dynamics of AML in remission. Thus, our findings emphasize the importance and the potential of single-cell DNA + protein sequencing in refining MRD assessment compared to current techniques.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db74b338da09cd55d0a1a600dcec1d4e9bacb1a1" target='_blank'>
              Clonal Dynamics of Leukemic and Clonal Hematopoiesis Mutations Predict Relapse in Single Cell MRD Analysis of AML in First Complete Remission
              </a>
            </td>
          <td>
            Enise Ceran, A. K. Merbach, Christian Rohde, Robert Durruthy-Durruthy, Adam Sciambi, Michelle Lotze, Gema Fuerte, Matteo Cattaneo, Ivan Lukic, Maxi Wass, Judith Schaffrath, Christoph Röllig, R. Schlenk, Marina Scheller, Simone Kowoll, Jörg Steighardt, B. Edemir, Beatrice Ludwig-Kraus, L. P. Mueller, Sabine Edemir, Abi Vainstein-Haras, E. Sorani, Irit Gliko Kabir, S. Kadosh, C. Gromann, Andreas Wienke, Claudia Wickenhauser, C. Baldus, U. Platzbecker, H. Serve, M. Bornhäuser, Carsten Müller-Tidow
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/680ac6121a0aae5814a42089f80d9a0e4a9ab7a0" target='_blank'>
              Revealing the structural microenvironment of high metastatic risk uveal melanomas following decellularisation
              </a>
            </td>
          <td>
            K. Aughton, Joshua Hattersley, Sarah E Coupland, H. Kalirai
          </td>
          <td>2024-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a48d6418b9c0121f7a10c287ff52a11955c6c52" target='_blank'>
              Spall: accurate and robust unveiling cellular landscapes from spatially resolved transcriptomics data using a decomposition network
              </a>
            </td>
          <td>
            Zhongning Jiang, Wei Huang, Raymond H W Lam, Wei Zhang
          </td>
          <td>2024-12-18</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Despite recent advances in the treatment of melanoma, many patients with metastatic disease still succumb to their disease. To identify tumor-intrinsic modulators of immunity to melanoma, we performed a whole-genome CRISPR screen in melanoma and identified Setdb1 as well as all components of the HUSH complex. We found that loss of Setdb1 leads to increased immunogenicity and complete tumor clearance in a CD8+ T-cell dependent manner. Mechanistically, loss of Setdb1 causes de-repression of endogenous retroviruses (ERVs) in melanoma cells and triggers tumor-cell intrinsic type-I interferon signaling, upregulation of MHC-I expression, and increased CD8+ T-cell infiltration. Importantly, spontaneous immune clearance observed in Setdb1-/- tumors results in subsequent protection from other ERV-expressing tumor lines, supporting the functional anti-tumor role of ERV-specific CD8+ T-cells found in the Setdb1-/- microenvironment. Blocking the type-I interferon receptor in mice grafted with Setdb1-/- tumors decreases immunogenicity by decreasing MHC-I expression, leading to decreased T-cell infiltration and increased melanoma growth, comparable to Setdb1wt tumors. Together, these results provide key in vivo evidence of a critical role for Setdb1 and type-I interferons in generating an inflamed tumor microenvironment, and potentiating tumor-cell intrinsic immunogenicity in melanoma. This study further emphasizes regulators of ERV expression and type-I interferon expression as potential therapeutic targets for augmenting anti-cancer immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ad3ee60cf098ddf889ff6e96bfd86efcd38c883" target='_blank'>
              Setdb1-loss induces type-I interferons and immune clearance of melanoma.
              </a>
            </td>
          <td>
            M. McGeary, William Damsky, Andrew J Daniels, Sabine M. Lang, Qingji Xu, E. Song, Clotilde Huet-Calderwood, Hua Jane Lou, Sateja Paradkar, G. Micevic, S. Kaech, D. Calderwood, Benjamin E. Turk, Q. Yan, Akiko Iwasaki, M. Bosenberg
          </td>
          <td>2024-11-26</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5daf8fc78edec4dc67cc0ac87030a2ee471c5078" target='_blank'>
              Cancer-associated fibroblasts: heterogeneity, tumorigenicity and therapeutic targets
              </a>
            </td>
          <td>
            Keke Lv, Tianlin He
          </td>
          <td>2024-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d320d942b07625b0b1d0c674abc05688c751b9d0" target='_blank'>
              Adipocytes reprogram the proteome of breast cancer cells in organotypic three-dimensional cell cultures
              </a>
            </td>
          <td>
            Karla Tovar-Hernández, Yarely M. Salinas-Vera, Ángeles Carlos-Reyes, A. P. García-Hernández, L. Marchat, Gilberto Mandujano-Lázaro, Emmanuel Ríos-Castro, Andrea Velasco-Suárez, Ivonne Mendez-Gómez, Angeles C. Tecalco-Cruz, Eloisa Ibarra-Sierra, C. López-Camarillo
          </td>
          <td>2024-11-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play pivotal roles in solid tumor initiation, growth, and immune evasion. However, the optimal biomimetic modeling conditions remain elusive. In this study, we investigated the effects of 2D and 3D culturing conditions on human primary CAFs integrated into a modular tumor microenvironment (TME). Using single-nucleus RNA sequencing (snRNAseq) and Proteomics’ Proximity Extension Assays, we characterized CAF transcriptomic profiles and cytokine levels. Remarkably, when cultured in 2D, CAFs exhibited a myofibroblast (myCAF) subtype, whereas in 3D tumor spheroid cultures, CAFs displayed a more inflammatory (iCAF) pathological state. By integrating single-cell gene expression data with functional interrogations of critical TME-related processes [natural killer (NK)-mediated tumor killing, monocyte migration, and macrophage differentiation], we were able to reconcile form with function. In 3D TME spheroid models, CAFs enhance cancer cell growth and immunologically shield cells from NK cell-mediated cytotoxicity, in striking contrast with their 2D TME counterparts. Notably, 3D CAF-secreted proteins manifest a more immunosuppressive profile by enhancing monocyte transendothelial migration and differentiation into M2-like tumor-associated macrophages (TAMs). Our findings reveal a more immunosuppressive and clinically relevant desmoplastic TME model that can be employed in industrial drug discovery campaigns to expand the cellular target range of chemotherapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b61425d19bdd4e9908dd233e505b35ad2127a3" target='_blank'>
              Unveiling immune cell response disparities in human primary cancer-associated fibroblasts between two- and three-dimensional cultures
              </a>
            </td>
          <td>
            Jian-Ping Yang, Nikhil Nitin Kulkarni, Masashi Yamaji, Tsubasa Shiraishi, Thang Pham, Han Do, Nicole Aiello, Michael Shaw, Toshihiro Nakamura, Akiko Abiru, N. Gavva, Shane R. Horman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Medullary thyroid carcinoma (MTC) is a rare and aggressive tumor, often requiring systemic treatment in advanced or metastatic stages, where drug resistance presents a significant challenge. Given the role of cancer stem cells (CSCs) in cancer recurrence and drug resistance, we aimed to identify CSC subpopulations within two MTC cell lines harboring pathogenic variants in the two most common MEN2-associated codons. We analyzed 15 stemness-associated markers, along with well-established thyroid stem cell markers (CD133, CD44, and ALDH1), a novel candidate (DLK1), and multidrug resistance proteins (MRP1 and MRP3). The ability to efflux the fluorescent dye Hoechst 3342 and form spheroids, representing CSC behavior, was also assessed. MZ-CRC-1 cells (p.M918T) displayed higher expressions of canonical markers, DLK1, and MRP proteins than TT cells (p.C634W). MZ-CRC-1 cells also formed more spheroids and showed less dye accumulation (p < 0.0001). Finally, we observed that DLK1+ cells (those expressing DLK1) in both cell lines exhibited significantly higher levels of stemness markers compared to DLK1− cells (those lacking DLK1 expression). These findings underscore DLK1’s role in enhancing the stemness phenotype, providing valuable insights into MTC progression and resistance and suggesting potential therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b46fc9f36dee4eea1c7adc632b8fd58baacccb9b" target='_blank'>
              DLK1 Is Associated with Stemness Phenotype in Medullary Thyroid Carcinoma Cell Lines
              </a>
            </td>
          <td>
            Danilo Dias da Silva, R. P. Araldi, Mariana Rocha Belizario, Welbert Gomes Rocha, R. M. Maciel, J. Cerutti
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 Cell state plasticity is retained by all stem-like and differentiated cells within glioblastoma (GBM) tumors. GBM cells can exploit these state-shifting capacities to evade and resist therapeutics. Thus, there is a need to identify and target the regulators of GBM cell state transitions. To address this question, we use a fully human platform where hiSPC-derived organoids serve as a recipient environment to host cells engrafted directly from surgically resected GBMs. This allows us to precisely examine how microenvironmental variables impact the molecular signatures of GBM cells. We acutely isolate GBM cells from surgical resections, label cells with a GFP-lentivirus for 4 hours, and then engraft these tumor cells into parallelly patterned region-specific organoids that mimic dorsal forebrain, ventral forebrain, midbrain, and hindbrain identities. These regions were chosen as they capture derivatives of the three primary brain vesicles and because these regions contain unique cell types and signaling molecules. We performed paired single-cell RNA sequencing on tumor cells pre-engraftment and 14 days post-engraftment. Universal to all regional organoids, we find that GBM cells shift towards a neural progenitor-like and neuron-like signature within organoids, regardless of the cell state of the parent tumor prior to engraftment. To support our findings, we also observed that patient-derived glioma cell lines converge on a neural progenitor-like state within organoids. Thus, the dominance of this shared GBM cell state within organoids implies the existence of shared extrinsic factors in these immature (neurogenic) organoids that favor a progenitor-like or neuron-like identity across all four regions. We are now asking whether this bias towards neuronal and/or neural progenitor identity is a result of engrafting into young neurogenic organoids, or if the same phenomenon could be true in late-stage (post-gliogenic) mature organoids. We are also using this valuable platform and data to identify the extrinsic signals that GBM cells rely on to fulfill their plastic potential, adapt to the microenvironment, and resist therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f97acab6d83148ea03065d74fff70ff890b1b78" target='_blank'>
              TMIC-33. MICROENVIRONMENT-DRIVEN CHANGES IN GLIOBLASTOMA CELL STATE SIGNATURES EXAMINED USING REGION-SPECIFIC HUMAN BRAIN ORGANOIDS
              </a>
            </td>
          <td>
            Tarun Bhatia, Saisrinidhi Ganta, Anson Sing, Alexia King, Caitlin Sojka, Kimberly Hoang, E. Nduom, Renee D Read, Jeffrey Olson, Steven Sloan
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Glioblastoma (GBM), the most malignant and aggressive primary brain tumor, is driven by both intrinsic genetic mutations and extrinsic factors from the tumor microenvironment (TME). GBM manipulates the TME to enhance its survival and proliferation through mechanisms such as forming functional synapses with adjacent normal cells and secreting immunosuppressive factors. Our laboratory has developed a novel human organoid tumor transplantation (HOTT) model, which allows for the interrogation of tumor-normal cell crosstalk at a molecular level and facilitates the dissection of specific signaling pathways that drive tumor progression and maintenance. Using single-cell transcriptomics on human GBM HOTTs, we identified the PTN-PTPRZ1 ligand-receptor interaction as a critical mediator of tumor-microenvironment communication. PTPRZ1, a receptor tyrosine phosphatase, has been previously implicated in glioma cell migration, and our data further elucidate its role in modulating tumor behavior. Specifically, we demonstrated that knockdown of PTPRZ1 within the TME of human cortical organoids significantly enhances glioma cell migration and inhibits differentiation, underscoring the essential role of PTPRZ1 in maintaining the tumor’s invasive phenotype. Further pharmacological inhibition of PTPRZ1’s catalytic activity with a small molecule revealed that these effects on cell fate determination are independent of its phosphatase activity. This suggests that PTPRZ1 exerts critical non-catalytic functions that are pivotal for GBM progression. To further investigate these non-catalytic mechanisms, we propose utilizing single-cell transcriptomics coupled with proximity labeling and proteomics. This approach will allow us to comprehensively map PTPRZ1-mediated signaling networks and elucidate their impact on tumor biology, potentially unveiling novel therapeutic targets. Our findings revealed a novel axis of GBM biology mediated by PTPRZ1 and highlighted the importance of considering microenvironmental interactions in the development of effective therapeutics for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e2eb14f30c8f2631ab0122c079e7474889f0947" target='_blank'>
              TMIC-77. ELUCIDATING THE ROLE OF PTPRZ1 SIGNALING IN DRIVING GLIOBLASTOMA PROGRESSION USING THE NOVEL HUMAN ORGANOID TUMOR TRANSPLANTATION SYSTEM
              </a>
            </td>
          <td>
            Ryan L. Kan, Weihong Ge, Can Yilgor, Elisa Fazzari, Patricia R. Nano, Daria J. Azizad, Matthew Li, Joyce Ito, C. Tse, Hong A. Tum, Jessica Scholes, Kunal S. Patel, David Nathanson, A. Bhaduri
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 The intra- and inter-patient heterogeneity in high-grade glioma (HGG) continues to contribute to its poor prognosis. Clinical biopsies are often harvested from limited regions and typically are not image localized. Thus, they fail to capture the diversity within tumor regions, immune expression or normal cell abundances that play key roles in tumor development. It is important to gain an understanding of these subpopulation ecologies, their spatial resolution, and interactions between them that may then be exploited for future therapeutic benefit. Further, these may differ by patient characteristics such as sex, age at diagnosis and treatment status. Using an ongoing image-localized biopsy collection protocol, we have so far evaluated the bulk transcriptomics of 202 multi-regional biopsies from 58 patients to characterize HGG heterogeneity. These samples were processed through Monocle, a reverse graph embedding algorithm that groups samples into states and orders them along developmental trajectories. Deconvolution methods were previously used to predict relative abundances of 7 normal, 6 glioma, and 5 immune cell subpopulations for each sample, which we have now overlaid on the Monocle graph. Monocle classified HGG into 4 main states along a three-pronged trajectory. These states reveal distinct population ecologies with associated enriched gene pathways. We also note significant immune pathway enrichments that differ between state and patient-reported sex. Together, these algorithms reveal a simple transcriptomic trajectory that helps us understand the development and evolution of HGG. Characterizing the in vivo diversity within and between high grade gliomas is important for understanding prognosis, stratifying future treatments and ultimately improving patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/537edcf404e379d243b38f5fbd9729b85401085d" target='_blank'>
              EPCO-40. TRANSCRIPTIONAL TRAJECTORY INFERENCE OF IMAGE-LOCALIZED, MULTI-REGIONAL HIGH-GRADE GLIOMA BIOPSIES REVEALS DISTINCT POPULATION ECOLOGIES AND SEX-ASSOCIATED ENRICHED PATHWAYS
              </a>
            </td>
          <td>
            L. Curtin, Kamila Bond, K. Singleton, Pamela Jackson, C. Sereduk, C. Krishna, R. Zimmerman, D. Patra, K. Donev, Maciej Mrugala, N. Tran, Leland Hu, B. Bendok, O. Al-Dalahmah, P. Canoll, Kristin R Swanson
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="
 Tumors represent a highly dynamic system that relies on coordinated signaling to drive growth and adapt to selective pressures, including those induced by anti-cancer therapies. Given the need for tumors to engage in rapid and coordinated cell-cell communication, mechanisms such as gap junction intracellular communication (GJIC) should be essential. However, the connexin proteins that make up gap junctions have traditionally been considered tumor suppressors based on the frequent loss of GJIC and lower connexin expression in tumor cells compared to non-neoplastic tissue. For this reason, connexin 43 (Cx43) has been proposed to suppress the growth of glioblastoma (GBM), the most common primary malignant brain tumor. However, recent data suggest that this tumor-suppressive effect is context dependent and that connexins can also function in a tumor-promoting role. Using next-generation sequencing techniques, we found Cx43 expressed at high levels in a panel of GBM patient-derived xenograft (PDX) CSCs and that these models rely on Cx43 for their survival and self-renewal. Mechanistically, depletion of Cx43 led to a dramatic loss of c-MYC expression through reduced phosphorylation of its upstream mediator WNK lysine-deficient protein kinase 1 (WNK1). Depletion of WNK1 phenocopied knockdown of Cx43 and reduced MYC protein and mRNA as well as tumor growth in vivo. Together, this work defines a novel signaling axis downstream of gap junction protein expression that promotes tumor growth and cancer stem cell phenotypes in GBM. Due to the difficulty in targeting both Cx43 and MYC, the identification of intermediate targetable signaling nodes may lead to improved therapies for patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd8c57f41fe0fba6aa898ab1f8e4aeb35588686a" target='_blank'>
              CSIG-16. A CX43-WNK1-C-MYC SIGNALING AXIS DRIVES GLIOBLASTOMA CANCER STEM CELL SURVIVAL
              </a>
            </td>
          <td>
            E. Mulkearns-Hubert, Nicole Hajdari, Ellen S. Hong, D. Silver, Justin D. Lathia
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3b8dd5bbe35e524e752fa9c462148784357715a" target='_blank'>
              Integrative ensemble modelling of cetuximab sensitivity in colorectal cancer patient-derived xenografts
              </a>
            </td>
          <td>
            Umberto Perron, Elena Grassi, Aikaterini Chatzipli, Marco Viviani, Emre Karakoc, Lucia Trastulla, Lorenzo M. Brochier, C. Isella, E. Zanella, Hagen Klett, Ivan Molineris, Julia Schueler, M. Esteller, Enzo Médico, Nathalie Conte, Ultan McDermott, L. Trusolino, A. Bertotti, F. Iorio
          </td>
          <td>2024-11-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/000ea5f5a04e65fe0427f079acb1a4f46ea53f2a" target='_blank'>
              Deciphering the topological landscape of glioma using a network theory framework
              </a>
            </td>
          <td>
            Mengchao Yao, Yang Su, Ruiqi Xiong, Xile Zhang, Xiaomei Zhu, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Ambiguous understanding of tumors and the tumor microenvironments (TMEs) hinders accurate diagnosis and available treatment for multifocal hepatocellular carcinoma (HCC) covering intrahepatic metastasis (IM) and multicentric occurrence (MO). Here, we characterized the diverse TMEs of IM and MO identified by whole-exome sequencing (WES) at single-cell resolution.


APPROACH AND RESULTS
We performed parallel WES and single-cell RNA sequencing (scRNA-seq) on twenty-three samples from seven patients to profile their TMEs when major results were validated by immunohistochemistry in the additional cohort. Integrative analysis of WES and scRNA-seq found that malignant cells in IM showed higher intra-tumor heterogeneity, stemness and more activated metabolism than those in MO. Tumors from IM shared similar TMEs while distinct TMEs were noticed in those from MO. Furthermore, CD20+ B cells, plasma cells and conventional type II dendritic cells (cDC2s) were decreased in IM relative to MO while T cells in IM exhibited a more terminally exhausted capacity with a higher proportion of proliferative/exhausted T cells than that in MO. Both CD20 and CD1C correlated with better prognosis in multifocal HCC. Additionally, MMP9+ tumor-associated macrophages were enriched across IM and MO which formed cellular niches with regulatory T cells (Tregs) and proliferative/exhausted T cells.


CONCLUSIONS
Our findings deeply decipher the heterogeneous TMEs between IM and MO, which provide a comprehensive landscape of multifocal HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2c7a45eee2228edb84e8693a219534a9ab548a" target='_blank'>
              Genomic and the tumor microenvironment heterogeneity in multifocal hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yongheng Yang, Qingqiang Ni, Hongguang Li, Jiuzheng Sun, Xia Zhou, Lingxin Qu, Liyuan Wang, Chuanzong Zhao, Xiaolu Zhang
          </td>
          <td>2024-12-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25ea97844e3ac53280202c1ab1e25a03d1cb6ccc" target='_blank'>
              Tailoring glioblastoma treatment based on longitudinal analysis of post-surgical tumor microenvironment
              </a>
            </td>
          <td>
            C. Bastiancich, E. Snacel-Fazy, Samantha Fernandez, Stéphane Robert, Roberta Stacchini, Léa Plantureux, Sébastien Boissonneau, Benoit Testud, Benjamin Guillet, F. Debarbieux, H. Luche, D. Figarella-Branger, Marie-Anne Estève, E. Tabouret, A. Tchoghandjian
          </td>
          <td>2024-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Gastric cancer (GC) is characterized by a complex and heterogeneous tumor microenvironment (TME) that significantly influences disease progression and treatment outcomes. The tumor stroma, which is composed of a variety of cell types such as cancer-associated fibroblasts, immune cells and vascular components, displays significant spatial and temporal diversity. These stromal elements engage in dynamic crosstalk with cancer cells, shaping their proliferative, invasive and metastatic potential. Furthermore, the TME is instrumental in facilitating resistance to traditional chemotherapy, specific treatments and immunotherapy strategies. Understanding the underlying mechanisms by which the GC microenvironment evolves and supports tumor growth and therapeutic resistance is critical for developing effective treatment strategies. The present review explores the latest progress in understanding the intricate interactions between cancer cells and their immediate environment in GC, highlighting the implications for disease pathogenesis and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78f0fda7748f1b417eb850faad66bf082f52653" target='_blank'>
              Role and value of the tumor microenvironment in the progression and treatment resistance of gastric cancer (Review)
              </a>
            </td>
          <td>
            Heng Yun, Fangde Dong, Xiaoqin Wei, Xinyong Yan, Ronglong Zhang, Xiuyu Zhang, Yulin Wang
          </td>
          <td>2024-11-26</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Sex-determining region Y-box 2 (SOX2) has recently been used as a marker for pluripotency to identify cellular glioblastoma invasion pathologically. This can be used to identify areas of tumor invasion consisting of sparse individual cells, likely highlighting the furthest margin of tumor invasion. This study aligned large format SOX2 stained tissue samples to conventional MRI to determine demographic factors that impact the presence of SOX2 positive invasion outside the T1 contrast-enhanced tumor margin.



 The cohort for this study consisted of 36 IDH-wildtype glioblastoma patients (23 male, 13 female, mean age=61.9). Tissue samples from areas of suspected tumor invasion and normal tissue were collected at autopsy, stained for SOX2, digitized at 40X resolution, and automatically segmented for SOX2 positive staining cell percent (SOX2+%) at MR resolution using a color deconvolution algorithm. Contrast enhanced T1 scans from acquisitions close to death were annotated manually for tumor presence and aligned to the T2-weighted FLAIR image. Tissue data was then aligned to MR images using a custom in-house software that relies on manually defined control points to generate a nonlinear transform that warps tissue into MR space while correcting for tissue shrinkage. Mean SOX2+% was computed for each patient outside of contrast enhancement and compared across age and sex as demographic factors.



 Males were found to have a higher SOX2+% than females outside contrast enhancement (t=2.09, p=0.045), while no association with age was observed (r=0.146, p=0.397).



 Sex differences were observed for non-enhancing SOX2 positivity, which may relate to the more aggressive tumors identified in male patients in some studies. Further research into the mechanisms behind sex differences in tumor growth patterns and molecular characteristics impacting the infiltration of tumor beyond the MR defined region is needed.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcf5f100471745ba67f49e284ea228b13d9895de" target='_blank'>
              ANGI-11. SEX DIFFERENCES IN SOX2 GLIOBLASTOMA EXPRESSION OUTSIDE MRI-DEFINED CONTRAST ENHANCEMENT AT AUTOPSY
              </a>
            </td>
          <td>
            S. Bobholz, A. Lowman, Daniel Aaronsen, A. Winiarz, S. Duenweg, Michael Flatley, Biprojit Nath, F. Kyereme, Jennifer Connelly, Kelly Mrachek, Max O Krucoff, Anjishnu Banerjee, Peter S LaViolette
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcbb969fcc77899a650652d987f291cd240a2f64" target='_blank'>
              Single-nucleus RNA-seq dissection of choroid plexus tumor cell heterogeneity
              </a>
            </td>
          <td>
            Anthony D Hill, Konstantin Okonechnikov, Marla K Herr, Christian Thomas, Supat Thongjuea, M. Hasselblatt, Annarita Patrizi
          </td>
          <td>2024-10-31</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Hepatic arterial infusion chemotherapy (HAIC) has emerged as a promising treatment strategy for hepatocellular carcinoma (HCC), but a detailed understanding of the multicellular ecosystem after HAIC treatment is lacking. Here, we collected tumor samples from treatment-naïve primary and post-HAIC HCC, and integrated single-nucleus RNA sequencing with spatial transcriptomics to characterize the tumor ecosystem in the post-HAIC HCC. Increased fractions and enhanced cellular communication of CD4+ T, CD20+ B, and dendritic cell subtypes were identified in post-HAIC tumors. Moreover, it is substantiated that HAIC promoted tertiary lymphoid structures (TLS) formation, and addressed the roles of TLSs as spatial niches of cellular communication. Specifically, intermediate exhausted CD8+ T cells expressing Granzyme-K and PD-1 (PD-1+CD8+ Tex-int) expanded following HAIC and exhibited a functionally antitumor phenotype. PD-1+CD8+ Tex-int accumulated in the TLS vicinity and disseminated throughout the tumor microenvironment, demonstrating potential as an effective biomarker for HAIC-based treatment in HCC. This study provides valuable resources and biological insights in the cellular underpinnings of HAIC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62624d4b04f46d42e65710fe0c98f6d6afaa46d" target='_blank'>
              Single-Nucleus and Spatial Transcriptome Profiling Delineates the Multicellular Ecosystem in Hepatocellular Carcinoma After Hepatic Arterial Infusion Chemotherapy.
              </a>
            </td>
          <td>
            YeXing Huang, Zefeng Du, Zhicheng Lai, DongSheng Wen, Lichang Huang, Minke He, ZiChao Wu, Huifang Li, Hanyue Ouyang, Wenchao Wu, Anna Kan, Ming Shi
          </td>
          <td>2024-12-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b26feb24c74a70e25f404994931c40c73dd0fe64" target='_blank'>
              Deciphering molecular landscape of breast cancer progression and insights from functional genomics and therapeutic explorations followed by in vitro validation
              </a>
            </td>
          <td>
            Bushra Khan, Rowaid M. Qahwaji, Mashael S. Alfaifi, Tanwir Athar, Abdullah Khan, Mohammad Mobashir, I. Ashankyty, K. Imtiyaz, Areej Alahmadi, M. M. Rizvi
          </td>
          <td>2024-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aaea8ef96bcf757be9b672ce6a26fbe4a6d0c91" target='_blank'>
              Integration of single-cell sequencing and drug sensitivity profiling reveals an 11-gene prognostic model for liver cancer
              </a>
            </td>
          <td>
            Qunfang Zhou, Jingqiang Wu, Jiaxin Bei, Zixuan Zhai, Xiuzhen Chen, Wei Liang, Jing Meng, Mingyu Liu
          </td>
          <td>2024-11-25</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d49c3b89849fb97fd894e63e495575849c7b7545" target='_blank'>
              A multilineage screen identifies actionable synthetic lethal interactions in human cancers.
              </a>
            </td>
          <td>
            Samson H. Fong, Brent M Kuenzi, Nicole M Mattson, John Lee, Kyle S. Sanchez, A. Bojorquez-Gomez, Kyle Ford, Brenton P. Munson, Katherine Licon, Sarah Bergendahl, John Paul Shen, J. Kreisberg, Prashant Mali, Jeffrey H Hager, Michael A White, T. Ideker
          </td>
          <td>2024-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="
 Changes in the glioma immune microenvironment at recurrence may contribute to disease progression and therapeutic resistance. In prior studies we demonstrated that the circumscribed glioma, pleomorphic xanthoastrocytoma (PXA), was relatively immune enriched including activated cytotoxic CD8+ T cells and Iba1+ microglia/macrophages. While PXA are often low-grade, a significant fraction can recur and progress to higher grade. To determine how the tumor and its immune microenvironment changes with recurrence, we examined a cohort of tumors from patients with multiple resections. Using single-nucleus RNA sequencing (snRNAseq) and spatial molecular imaging we investigate the temporal changes in the tumor and its microenvironment. To begin, we analyzed samples from three patients with paired initial and recurrent tumor. A total of 22,465 immune and non-immune cells were analyzed. Compared to the initial, the recurrent tumor tended to have decreased numbers of T cells and T cells had lower expression of CD2 and GZMA. Pathway analysis for DE genes in TAMs demonstrated enrichment of genes associated with NTRK signaling, while pathways related to aerobic respiration and mitochondrial translation were decreased suggesting a less inflammatory phenotype. Tumor cells at recurrence demonstrated decreased expression of CD274 (PD-L1) and of the chemokine CXCL14, previously identified as a factor driving an anti-tumor CD8+ T cell response. In a cohort of 5 patients with paired initial and recurrent tumors, we observed increased numbers of CD163+ TAMs. Together these data suggest changes in the tumor-associated immune response at recurrence that may be relevant to therapy. Using murine models for the disease we are investigating the factors that regulate these changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b88f373ffe2f117f963a4d52016101c59ac0f40" target='_blank'>
              TMIC-84. MOLECULAR EVOLUTION OF THE TUMOR AND ITS MICROENVIRONMENT AT RECURRENCE
              </a>
            </td>
          <td>
            Elaheh Hashemi, Akshaya R.L, Anupam Kumar, Viva Voong, Lin Wang, Aaron A. Diaz, Joanna Phillips
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 DNA methylation profiling has become an important diagnostic and exploratory tool in neuro-oncology. Intra-tumoral molecular heterogeneity is a crucial factor in the treatment resistance of IDH-wildtype glioblastoma (GBM). However, the intra-tumoral heterogeneity of methylation profiles in GBM remains poorly elucidated. This study aimed to investigate the intra-tumoral heterogeneity of DNA methylation subclasses and deconvolve methylation profiles to infer the cellular composition of the tumor.



 We conducted genome-wide DNA methylation analysis and applied the DKFZ/Heidelberg CNS tumor classifier to FFPE tissue samples from a multi-sampling cohort with multiple samples per patient and a single-sampling cohort with one sample per patient. Only samples classified as GBM were included. We conducted two different methylation-based deconvolution analyses. Deconvolution 1 inferred the fractions of tumor subtypes (RTK_I, RTK_II, MES_TYP, and MES_ATYP) and cell types in the microenvironment. Deconvolution 2 estimated the abundance of malignant cell states (stem-like and differentiated cell components) and cell types in the microenvironment.



 The multi-sampling cohort included 32 patients with 113 samples, and the single-sampling cohort included 87 patients with 87 samples. In the multi-sampling cohort, 8 patients (25%) exhibited intra-patient heterogeneity in methylation subclasses across their samples. Deconvolution 1 indicated that each tumor subclass component was heterogeneously distributed within samples, and the abundance of the myeloid cell component correlated with the MES_TYP component across samples. Deconvolution 2 similarly revealed heterogeneity in the immune component abundance across samples within patients. Integrating the results of Deconvolution 1 and Deconvolution 2 using the entire cohort revealed significant correlations between RTK_I and stem-like, RTK_II and differentiated, and MES_TYP and immune components. These findings were also replicated in TCGA dataset.



 Our findings underscore the importance of considering intra-tumoral heterogeneity in methylation profiles and the biological characteristics of each methylation subclass in developing novel GBM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bc927be7c787e923aae21a191879fcd45aa8fd" target='_blank'>
              EPCO-15. INTRA-TUMORAL HETEROGENEITY OF DNA METHYLATION PROFILING AND CELLULAR COMPOSITION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Yuji Matsumoto, Omkar Singh, Jose A. Garcia, Sunwoo Kwak, Zied Abdullaev, Nelson Freeburg, Dana Silverbush, Kenneth J Aldape, C. Davatzikos, M. Nasrallah
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common malignant brain tumor in adults. GBMs are known for their highly invasive and proliferative properties and are associated with a dismal prognosis (~12-18 months post-diagnosis). Neuroanatomical disparities in the occurrence of GBMs suggest that in addition to microenvironmental impacts, tumor cells are impacted by the cellular and molecular diversity across varying regions of this nervous system. This idea has been difficult to study due to the lack of human-specific models that effectively recapitulate the biological behaviors of GBMs in region-specific microenvironments. To address these challenges, we engrafted GFP-tagged patient-derived glioma cell lines into human induced pluripotent stem cell (hiPSC)-derived organoids patterned to mimic the most forebrain (dorsal cortex) and hindbrain (spinal cord) regions of the nervous system. Immunostaining assays at 2, 7, 14, and 21 days post-engraftment compared the extent of infiltration, as measured by the distance traveled by glioma cells, in both regions. We found that glioma cells infiltrated a significantly greater distance in the dorsal cortical organoids compared to the spinal cord organoids. This suggests a more conducive environment for glioma infiltration in the forebrain vs. hindbrain at baseline. We are currently determining if these regional biases in infiltration capacity are modified with either optogenetic stimulation of the host brain organoids or in the presence of interregional interactions in an assembloid system. Thus, using this fully human-specific platform, we aim to gain insights into the infiltrative behaviors of GBM cells, which will advance our understanding of this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/045df47e99729e03f5d67a0ffd1abdf99975dbdd" target='_blank'>
              TMIC-34. INFILTRATIVE BEHAVIOR OF PATIENT-DERIVED GLIOBLASTOMA CELL LINES IN REGION-SPECIFIC HUMAN BRAIN ORGANOIDS
              </a>
            </td>
          <td>
            Saisrinidhi Ganta, Tarun Bhatia, Alexia King, Anson Sing, Renee D Read, Steven Sloan
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor microenvironment (TME) plays a crucial role in the oncogenesis, early distant metastasis, and limited immunotherapeutic responses of pancreatic ductal adenocarcinoma (PDAC). Myeloid cell populations, particularly tumor-associated macrophages (TAMs), located within this immunosuppressive niche, are pivotal in these processes. With aid of single cell RNA sequencing (scRNA-seq), multiple studies have successfully validated distinct macrophage subtypes pathological functions in different cancer types, yet the correlation of TAMs in hepatic metastatic and primary PDAC lesions remains insufficiently understood. In this study, we reanalyzed samples from both primary and hepatic metastatic PDACs to elucidate the functional conservation and variability of TAMs. Pseudotime trajectory inference, together with pseudobulk analysis based on subtype level, manifests TAMs undergo similar metabolism reprogramming and pro-inflammatory style transitions with specific signaling pathway activations. Compared to primary tumors, hypoxic conditions are alleviated, which largely depends on the physiological context and the preliminary ligand-receptor interactive network within the metastatic niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a56bbf6854596a35b27cf40482bd3f5c23fafab" target='_blank'>
              In-depth analysis of pathological function of tumor-associated macrophage populations in hepatic metastatic and primary PDAC
              </a>
            </td>
          <td>
            Fanyu Meng
          </td>
          <td>2024-12-12</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastatic progression is responsible for the majority of cancer-related deaths. A better understanding of the underlying molecular mechanisms driving metastasis therefore remains of utmost importance. Here we take efforts to excavate mutant driver factors of metastases in the level of signaling pathways. We introduce two models, EntCDP and ModSDP, to detect the similarity and specificity of pathways in a pan-cancer context. The simulation studies confirm their feasibility and identification accuracy. Using mutation profiles of 17 cancer types with primary and metastatic cohorts from MSK-IMPACT, we apply the two models to investigate the transition of signaling systems from primary to metastatic tumors mainly from five perspectives. 1) We initially perform a comprehensive comparison of 15 primary-metastatic cancer pairs. Nearly all shared common driver genes or pathways, while specific driver gene sets of primary or/and metastatic tumors can be seen for almost each pair, except for breast cancer. 2) We use ModSDP to identify the specificity of metastases with the same primary site and highlight calcium signaling in breast cancer with bone metastasis, FoxO signaling for melanoma/lung lymph node metastasis, etc. 3) Oppositely, we use EntCDP to identify common factors yielding three frequently metastatic sites (liver, lung and brain), and recognized cushing syndrome, microRNAs in cancer and MAPK signaling pathway, separately. 4) Regarding high-tropism metastatic patterns, we investigated the relationship between the metastatic tumor and primary tumors in both locations. In the typical pattern of colorectal cancer metastasis to the liver, we detected hepatic signals in primary colorectal cancer. 5) Finally, we focused on the common and specific characteristics of relevant cancer types, such as gender-related cancers and gastrointestinal cancers. Study on gynecologic tumors suggest endocrine hormone change and virus infection as their common risk factors, while for male we highlight hedgehog signaling and EPH-related receptors as invasive potential of prostate cancer. Multiple interesting findings revealed by this study may be helpful for the understanding of the extent of signal changes during tumor metastasis. We expect that this study will provide a valuable resource for transforming our knowledge about signals in cancer metastasis into alternative clinical practice for advanced patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facca2ecc611248ea30b446f5773d2c64fea06c3" target='_blank'>
              Pan-cancer study on transition of signaling systems from primary to metastatic tumors
              </a>
            </td>
          <td>
            Wenjia Zhou, Junhua Zhang
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The transcriptional heterogeneity of tumor microenvironment (TME) cells is a crucial factor driving the diversity of cellular response to drug treatment and resistance. Therefore, characterizing the cells associated with drug treatment and resistance will help us understand therapeutic mechanisms, discover new therapeutic targets and facilitate precision medicine. Here, we describe a database, scDrugAct (http://bio-bigdata.hrbmu.edu.cn/scDrugAct/), which aims to establish connections among drugs, genes and cells and dissect the impact of TME cellular heterogeneity on drug action and resistance at single-cell resolution. ScDrugAct is curated with drug-cell connections between 3838 223 cells across 34 cancer types and 13 857 drugs and identifies 17 274 drug perturbation/resistance-related genes and 276 559 associations between >10 000 drugs and 53 cell types. ScDrugAct also provides multiple flexible tools to retrieve and analyze connections among drugs, genes and cells; the distribution and developmental trajectories of drug-associated cells within the TME; functional features affecting the heterogeneity of cellular responses to drug perturbation and drug resistance; the cell-specific drug-related gene network; and drug-drug similarities. ScDrugAct serves as an important resource for investigating the impact of the cellular heterogeneity of the TME on drug therapies and can help researchers understand the mechanisms of action and resistance of drugs, as well as discover therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/043f2ce1e460a880b52e0d2bd7d4a4783b1e445d" target='_blank'>
              ScDrugAct: a comprehensive database to dissect tumor microenvironment cell heterogeneity contributing to drug action and resistance across human cancers.
              </a>
            </td>
          <td>
            Yanjun Xu, Yifang Zhang, Kaiyue Song, Jiaqi Liu, Rui Zhao, Xiaomeng Zhang, Liying Pei, Mengyue Li, Zhe Chen, Chunlong Zhang, Peng Wang, Feng Li
          </td>
          <td>2024-11-11</td>
          <td>Nucleic acids research</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="
 The awakening of dormant disseminated cancer cells is likely responsible for the clinical relapses of patients whose primary tumors have been cured months and even years earlier. In the present study, we demonstrate that dormant breast cancer cells lodged in the lungs reside in a highly mesenchymal, non-proliferative phenotypic state. The awakening of these cells does not occur by a cancer cell-autonomous process. Instead, inflammation and wound healing of the surrounding tissue microenvironment causes them to shift from a highly mesenchymal to a quasi-mesenchymal phenotypic state in which they acquire stemness and proliferative ability. Once awakened, these cells can stably reside in this quasi-mesenchymal state and maintain their stemness, doing so without ongoing heterotypic signaling from the lung microenvironment. EGFR ligands released by the cells of the injured tissue microenvironment, including notably M2 type macrophages, promote dormant cancer cells to move toward this quasi-mesenchymal state, a transition that is essential for the awakening process. An understanding of the mechanisms of metastatic awakening may lead in the future to treatment strategies designed to prevent such awakening and resulting metastatic relapse.
 Citation Format: Jingwei Zhang, Robert Weinberg. Awakened dormant cancer cells undergo highly mesenchymal to quasi- mesenchymal transition and acquire stemness [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8beb7f3a092f66cbe81343c9f59c0b804bcf8a" target='_blank'>
              Abstract C018: Awakened dormant cancer cells undergo highly mesenchymal to quasi- mesenchymal transition and acquire stemness
              </a>
            </td>
          <td>
            Jingwei Zhang, Robert Weinberg
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this study, we delve into the intrinsic mechanisms of cell communication in hepatocellular carcinoma (HCC). Initially, employing single-cell sequencing, we analyze multiple malignant cell subpopulations and cancer-associated fibroblast (CAF) subpopulations, revealing their interplay through receptor-ligand interactions, with a particular focus on SPP1. Subsequently, employing unsupervised clustering analysis, we delineate two clusters, C1 and C2, and compare their infiltration characteristics using various tools and metrics, uncovering heightened cytotoxicity and overall invasion abundance in C1. Furthermore, our gene risk scoring model indicates heightened activity of the immune therapeutic pathway in C1. Lastly, employing a formulated scoring system, we stratify patients into high and low-risk groups, revealing notably poorer outcomes in the high-risk cohort on Kaplan-Meier curves. Risk scores exhibit a negative correlation with model genes and immune cell infiltration scores, indicating poor prognosis in the high-risk group. Further characterization elucidates the regulatory landscape of the high and low-risk groups across various signaling pathways. In addition, we used wet lab experiments to prove that ABCA1 plays a pro-oncogenic role in hepatocellular carcinoma cells by promoting proliferation, invasion, migration, and reducing apoptosis. In summary, these findings provide crucial insights, offering valuable clues and references for understanding HCC pathogenesis and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ebd5bca77500ddcf823c450cf7ed50074d3c6d" target='_blank'>
              ABCA1 promote tumor environment heterogeneity via epithelial mesenchymal transition in Huh7 and HepG2 liver cancer cell
              </a>
            </td>
          <td>
            Dinglai Yu, Fang Guo, Qiyu Zhang, Huajun Yu, Wenmin Wang, Yunzhi Chen
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Acute myeloid leukemia (AML) is a hematologic malignancy classically associated with an expansion of poorly differentiated myeloid cells. Single-cell RNA-sequencing (scRNA-seq) studies have advanced our understanding of differentiation arrest across the leukemia cell hierarchy and their relation to prognosis, relapse, and drug response. Leukemic cellular hierarchy composition, such as Primitive versus GMP or Primitive versus Mature axes of cell state abundance, can define response to chemotherapy or targeted therapy sensitivity respectively (Zeng et al. Nature Med 2022). However, it remains unclear how these transcriptional AML cell states intersect with existing biomarkers used for clinical characterization of AML, such as cell surface markers used by clinical flow cytometry. This study aims to nominate existing and new surface markers associated with transcriptionally-defined AML cell states across leukemia cell hierarchies.
 We developed a novel consensus marker identification approach, IsoMarker, to identify a minimal set of enriched (differential expression), specific (auROC), and predictive (SHAP) cell surface markers for each AML cell state. To validate the utility of this approach, we generated an integrated scRNA-seq atlas of 1,079,555 single cells spanning 316 AML patient samples and 21 datasets, identifying 13 consensus leukemia cell states. Using IsoMarker, we identified 56 positive and 33 negative cell surface markers which effectively discern between the 13 leukemia cell states. We validated these markers by confirming that the expression levels of each positive marker were positively correlated with transcriptionally-inferred cell state abundance from 1037 total patients across the TCGA, BEAT-AML, and Leucegene cohorts.
 Composition of different leukemic cell states is known to govern functional leukemic stem cell activity, leading to heterogeneity in response to therapy and overall survival of AML patients. To extend the diagnostic utility of our nominated cell-state markers to include prediction of clinical outcomes, we conducted an integrative analysis of the positive AML cell state markers in relation to leukemic stem cell activity, ex vivo drug response, and meta-analysis of overall survival. Among the candidate positive cell state markers, 10 markers were associated with functional leukemia stem cell engraftment in mice, 5 markers were associated with the GMP cell state that defines response to chemotherapy, and 4 markers were associated with overall survival outcomes.
 We conducted a literature review of the 56 candidate positive markers in the context of AML to determine if IsoMarker can resolve both well-known and novel markers of AML cell states. Out of the 59 positive markers identified by IsoMarker, 21 have been previously reported as diagnostic markers of AML cell states. There are 35 positive markers that have not been characterized in the literature review and may serve as candidate, context-specific biomarkers of AML cell states.
 In summary, this study aims to translate insights from single-cell transcriptomics into hematopathology practice by advancing identification of markers that discern leukemic cell states and predict clinical outcomes. Applied to the task of identifying markers of AML cell states, IsoMarker is able to both resolve well-known markers and nominate novel markers useful for characterization of AML in relation to prognosis, relapse, and drug response. soMarker is a powerful framework for marker identification of cell states in single cell studies that can be generalized to other types of cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d3ea2f56cb6068f8e19edcc8c25e71b2d833de7" target='_blank'>
              Isomarker: An Explainable Machine Learning Framework for Identification of Cell State Markers in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            David Chen, Suraj Bansal, Amanda Mitchell, A. Tierens, Andy G. X. Zeng, John E. Dick
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b72e954c6c11c764347d1edffb32b16bc9b9834" target='_blank'>
              Transcriptome and epigenome dynamics of the clonal heterogeneity of human induced pluripotent stem cells for cardiac differentiation
              </a>
            </td>
          <td>
            Jihye Yun, Jaemin So, Seunghee Jeong, Jiye Jang, Soyoung Han, Junseok Jeon, Kyungho Lee, Hye Ryoun Jang, Jaecheol Lee
          </td>
          <td>2024-12-11</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer stem cells aggregate to form clusters, which have enhanced stem-like properties and metastasis potential. However, the molecular mechanisms underlying the formation of cancer stem cell cluster-like structures with acquisition of stronger invasion and metastasis abilities remain unclear. Micropapillary carcinoma (MPC) is a subpopulation of small, merulioid, inverted, nonfibrous vascular clusters floating in the stroma present in a range of solid malignant tumors and characterized by frequent vascular/lymphatic vessel invasion and lymph node metastasis. Our results showed that these cell clusters exhibit a stem cell phenotype, supporting the premise that MPC may serve as a promising solid tumor model for studying invasion and metastasis of cancer stem cell clusters. In this review, we discuss the latest advances in MPC research and targeted therapy, focusing on analysis of their stem-like characteristics, mapping their multiomics characteristics, and elucidating the vascular and immune microenvironment of MPC. The existing MPC organoid model was employed to explore potential breakthroughs in targeted therapy and immunotherapy for cancer stem cell clusters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/465f3c6afb9220f958e9f34e5f679f5bbb6538be" target='_blank'>
              Micropapillary structure: A natural tumor collective invasion model with enhanced stem-like properties.
              </a>
            </td>
          <td>
            Sisi Li, Shuangshu Gao, Ling Qin, Caixia Ding, Jinghui Qu, Yifei Cui, Lixia Qiang, Shengjie Yin, Xiaoyu Zheng, Hongxue Meng
          </td>
          <td>2024-11-20</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Introduction: Survival rates in advanced head and neck cancer (HNC) remain poor with no clinically approved biomarkers to predict treatment efficacy. HNC is a highly heterogenous disease and the lack of available models that capture patient heterogeneity and the complexity of the tumor microenvironment (TME) contributes to these poor patient outcomes. Microphysiological systems (MPS) are complex multicellular in vitro models that can recapitulate three-dimensional organ structure and function. The low cell number requirements for MPS permits the creation of patient-specific TME models, where all the cell types can be isolated from a single piece of tumor tissue. These patient-specific MPS represent a promising strategy for elucidating the contribution of the TME to treatment resistance, predicting the best treatment for each patient and improving patient outcomes in HNC. Here, we report a patient-specific MPS that contains multiple TME compartments and recapitulates patient heterogeneity. Methods: Tumor tissue was collected from patients with oral cavity carcinoma undergoing standard of care surgery followed by radiation or chemoradiation. Tissue from each patient was digested and tumor infiltrating leukocytes (TIL) were isolated using CD45 magnetic beads, while epithelial cells and fibroblasts were expanded in culture using specific media formulations. Tissue was also snap frozen for single cell RNA sequencing analysis. To build the MPS, a custom designed microfluidic device was filled with a hydrogel matrix containing patient-matched TIL, fibroblasts and epithelial spheroids to mimic the solid tumor. Lumens molded from the hydrogel were seeded with blood or lymphatic endothelial cells to create vasculature. Multiple replicate MPS can be built from each patient, which are cultured in an optimized media formulation to support the multiple primary cell types. Patient-specific MPS were treated with radiation or cisplatin + radiation to mimic the treatment the patient received. Multiple orthogonal endpoints were measured including tumor cell migration and viability, cytokine secretion and angiogenesis. Single cell sequencing was also performed on the original tumor tissue and matched MPS. Results: We have successfully created patient-specific models of the HNC TME. Single cell RNA sequencing analysis of the MPS suggests that cellular heterogeneity is being retained. Patient characteristics varied between MPS with some patient TME MPS showing highly migratory tumor cells or angiogenic responses. Treatment responses were measured for each patient-specific MPS and ongoing work will characterize the contributions of the TME to treatment resistance. These data demonstrate the feasibility of using patient-specific MPS to investigate the contribution of the TME to treatment response and resistance and lays important groundwork for using MPS to identify patient treatment responses.
 Citation Format: Adeel Ahmed, Marcos Lares, Fauzan Ahmed, Adam R. Burr, Paul M. Harari, David J. Beebe, Sheena C. Kerr. Engineering the tumor microenvironment: Patient-specific microphysiological systems of the head and neck cancer tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5867639a61561e0c0780d2ce0be172eff5b37091" target='_blank'>
              Abstract B034: Engineering the tumor microenvironment: Patient-specific microphysiological systems of the head and neck cancer tumor microenvironment
              </a>
            </td>
          <td>
            Adeel Ahmed, Marcos Lares, Fauzan Ahmed, Adam R. Burr, Paul M. Harari, David J. Beebe, S. Kerr
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represents the most prevalent and deadly primary brain tumor in adults, and it remains highly refractory to current therapeutic interventions. Significant intratumor heterogeneity drives rapid invasion, immune escape, and therapeutic resistance. Therapeutic paradigms based on deconvoluting key molecular mechanisms driving gliomagenesis are urgently needed. Quaking (Qki, encoded by Qk) is frequently altered in GBM patients; Qk mutations or deletions occur in 35% percent of cases, and the Qki protein is absent in 50-60% of cases. Our lab has characterized Qki as important tumor suppressor in GBM, where deletion of Qk enables pre-malignant neural stem cells to expand outside their niche and develop into tumors that recapitulate human GBM. We discovered that Qki functions as a transcriptional coactivator of lipid metabolism genes, whereby Qki loss disrupts lipid homeostasis and compromises the integrity of various membrane structures. Several downstream processes governed by Qki have been delineated; however, upstream regulators of Qki have yet to be identified. Recurrent cancer mutations to specific Qki arginine sites implicate the dysregulation of arginine methylation, a common post-translational modification catalyzed by protein arginine methyltransferases (PRMTs). Leveraging in vitro methylation assays, we demonstrate that Qki is selectively methylated by the PRMT coactivator-associated arginine methyltransferase 1 (CARM1), specifically in the Quaking regulatory domain where cancer mutations are reported. We observe that Qki interacts with known CARM1 substrates, functioning in the transcriptional complex assembled by CARM1. Through mutagenesis studies, we identified arginine 242 (R242) and arginine 256 (R256) as specific CARM1-mediated methylation sites on Qki. Importantly, mutations to these sites compromise Qki’s activity as a transcriptional coactivator. Moreover, brain-specific deletion of CARM1 in vivo results in neurological phenotypes reminiscent of Qki deletion. Altogether, we provide evidence that CARM1 represents a novel tumor suppressor in GBM that functions through the arginine methylation of Qki.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c4f80d5e1592ad297d2b32f6d52b2a08309ce04" target='_blank'>
              EPCO-17. CARM1 METHYLATES GLIOBLASTOMA TUMOR SUPPRESSOR QUAKING TO REGULATE ITS TRANSCRIPTIONAL COACTIVATOR FUNCTION
              </a>
            </td>
          <td>
            Kaylene Lu, Tanner Wright, Seula Shin, Mark T. Bedford, Jian Hu
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 High-grade glioma (HGG) is the most common and aggressive form of primary brain cancer. Standard of care surgery employs Magnetic Resonance Imaging to identify the contrast enhancing (CE) core for resection, leaving a non-enhancing (NE) invasive rim of tumor which contributes to recurrence. Recent multi-omics analyses identified predominant neuronal and immune signatures within NE tumor. Here, we apply a graph network to integrate regionally heterogeneous data from a unique dataset of spatially-localized CE and NE biopsies from a large cohort of patients with HGG.



 RNA-sequencing of 1159 differentially expressed genes across 159 IDH-wt biopsies (74 patients) from NE/CE regions were represented as nodes in a Neo4j network. Candidate genes for network random walks occurred exclusively within NE communities detected by modularity optimization and localized by cell-type using single-cell RNA-sequencing (scRNAseq). CSF2RA and RBFOX1 were selected as sources (walk length = 5; 10,000 walks/source node). Frequently occurring genes (one-tailed test) in walks were ranked by correlation with CSF2RA/RBFOX1 expression in NE samples and analyzed by gene set enrichment analysis for biological processes (clusterProfiler). Random walk genes were visualized (DoHeatmap) against composite NE/CE scRNAseq profiles.



 CSF2RA and RBFOX1 were upregulated in NE samples (logFC = 0.789, 1.85). Upregulation of CSF2RA, an immune receptor, in the Glycolytic/Plurimetabolic and RBFOX1, a neurodevelopmental regulator, in the Neuronal glioma subtypes were observed in NE samples (ANOVA p < 0.05). CSF2RA walk genes (n=46) enriched metabolic/proliferative signatures, while RBFOX1 walk genes (n=201) showed neurodevelopmental upregulation. scRNAseq localized CSF2RA to monocytes/macrophages while RBFOX1 was expressed in distinct astrocyte/neuroepithelial populations.



 CSF2RA and RBFOX1’s gene networks modulate immune and neurodevelopmental processes influencing glioma progression in NE tumor. Understanding the region-specific roles of these networks may offer strategies for targeting cells in the invasive glioma rim.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b304fb50a68433f95438fb0faaea7926334f7b4" target='_blank'>
              ANGI-09. GRAPH NETWORK ANALYSIS IDENTIFIES GENETIC COMMUNITIES OF CSF2RA AND RBFOX1 AS REGULATORS OF INVASIVE HIGH-GRADE GLIOMA BIOLOGY
              </a>
            </td>
          <td>
            M. Flick, V. Stevens, C. Sereduk, C. Plaisier, R. Barajas, Leland Hu, K. Kannan, N. Tran
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf64c81b2e5cd6a446e0a9c3f91b443d7d5cf63c" target='_blank'>
              Supercontinuum-tailoring multicolor imaging reveals spatiotemporal dynamics of heterogeneous tumor evolution
              </a>
            </td>
          <td>
            Xiujuan Gao, Xinyuan Huang, Zhongyun Chen, Liu Yang, Yifu Zhou, Zhenxuan Hou, Jie Yang, Shuhong Qi, Zheng Liu, Zhihong Zhang, Qian Liu, Qingming Luo, Ling Fu
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Disseminated cancer cells (DCCs) arriving to a new organ may either proliferate and form metastasis, get eliminated by immune cells, or enter a quiescent stage. The factors that determine the DCCs’ fate, including getting quiescent DCCs to re-initiate proliferation and successfully evade the immune system, remain poorly understood, in part due to the lack of robust mouse models. We established a pancreatic ductal adenocarcinoma (PDAC) mouse model of dormant DCCs in the liver. An “immunization protocol” was used to mimic PDAC resection in patients: KPC1199 PDAC cells were injected to establish primary tumors and allow an adaptive immune response to develop against the cancer cells; the tumors were surgically resected; and new KPC1199 cells were intrasplenically injected to establish experimental liver metastases. The overwhelming majority of the KPC1199 cells were eliminated. No metastases spontaneously formed during 20 months of observation, but about 3-8 sporadic, single DCCs were found per liver cross section, and 94% of these DCCs were negative for proliferation markers. Although the model is based on entraining an adaptive immune response, T cell depletion resulted in only 25-30% of mice developing just 1-3 metastases each. This suggests that T cell depletion results in metastasis from the small percentage of DCCs that were proliferating. The DCC-hosting mice therefore represents of model to identify factors that can trigger further DCC proliferation and metastatic recurrence. Patients with PDAC often experience significant stress, leading to excess of endogenous glucocorticoids (GCs). Additionally, synthetic GCs are used during e.g., chemotherapy to manage side effects. To mimic elevated GC levels in patients, we treated DCC-hosting mice with synthetic GCs resulting in an increased percentage of proliferating DCCs (increasing from <6% of the examined cells in control mice to about 36%). Yet, no metastases formed. GC treatment also decreased T cell and increased neutrophil liver infiltration. Neutrophils can form protease-containing neutrophil extracellular traps (NETs), which we previously showed trigger DCC proliferation by proteolytic remodeling of the extracellular matrix protein laminin. We found that neutrophils from GC-treated mice spontaneously formed more NETs, and additionally, NET-generated laminin remodeling was observed in the liver metastasis. Together, the data suggest, that GC treatment drive quiescent DCCs to proliferate via neutrophils but alone this is insufficient to cause metastases. Consistent with this interpretation, T cell depletion combined with GC treatment resulted in multiple metastases in all mice examined. Of note, deleting the GC receptor in the DCCs did not reduce metastases, supporting that GCs act on the host and not the cancer cells. In conclusion, our data suggest that for quiescent DCCs to form metastases, they need signals that trigger proliferation and simultaneously, they must overcome immune surveillance. Elevated GC levels or GC treatment in immune suppressed individuals represents such a scenario.
 Citation Format: Xiao Han, Xueyan He, Yuan Gao, Sicheng Pan, Mikala Egeblad. Glucocorticoids establish a metastatic-promoting tumor microenvironment in a PDAC mouse model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5174ee47dc358614c322b5604295e8de3a8bcc9a" target='_blank'>
              Abstract C004: Glucocorticoids establish a metastatic-promoting tumor microenvironment in a PDAC mouse model
              </a>
            </td>
          <td>
            Xiao Han, Xueyan He, Yuan Gao, Sicheng Pan, M. Egeblad
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is among the most aggressive brain cancers, and it contains glioma stem cells (GSCs) that drive tumor initiation, progression, and recurrence. These cells resist conventional therapies, contributing to high recurrence rates in GBM patients. Developing in vitro models that mimic the tumor microenvironment (TME), particularly the GSC niche, is crucial for understanding GBM growth and therapeutic resistance. Three-dimensional (3D) spheroid models provide a more physiologically relevant approach than traditional two-dimensional (2D) cultures, recapitulating key tumor features like hypoxia, cell heterogeneity, and drug resistance. This review examines scaffold-free and scaffold-based methods for generating 3D GBM spheroids, focusing on their applications in studying the cancer stem cell niche. The discussion encompasses methods such as the hanging drop, low-adhesion plates, and magnetic levitation, alongside advancements in embedding spheroids within extracellular matrix-based hydrogels and employing 3D bioprinting to fabricate more intricate tumor models. These 3D culture systems offer substantial potential for enhancing our understanding of GBM biology and devising more effective targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea7b7a45a32163a4dbbbdd90c32adea5a5aa6ad9" target='_blank'>
              Recapitulating Glioma Stem Cell Niches Using 3D Spheroid Models for Glioblastoma Research
              </a>
            </td>
          <td>
            Hyunji Jo, Seulgi Lee, Min-Hyeok Kim, Sungsu Park, Seo-Yeon Lee
          </td>
          <td>2024-11-01</td>
          <td>Biosensors</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 DNA methylation profiling has become a pivotal tool in neuro-oncology. However, little is known about the impact of methylation profiling on radiological imaging in IDH-wildtype glioblastoma (GBM). This study explored the potential of radiomics to non-invasively predict molecular characteristics based on the methylation profiling of GBM.



 This study included a multi-sampling cohort of 32 patients with 113 samples and a single-sampling cohort of 87 patients with 87 samples, which underwent genome-wide methylation analysis and were classified as GBM by the DKFZ/Heidelberg CNS tumor classifier. We performed two different methylation-based deconvolution analyses. Deconvolution 1 estimated the fractions of tumor subtypes (RTK_I, RTK_II, MES_TYP, and MES_ATYP) and cell types in the microenvironment. Deconvolution 2 inferred the abundance of malignant cell states (stem-like and differentiated cell components) and cell types in the microenvironment. We derived 6084 radiomic features from preoperative multi-parametric MRI in each patient.



 In the multi-sampling cohort, Deconvolution 1 and Deconvolution 2 demonstrated that tumor subclass components and immune components exhibited heterogeneous distribution across samples within patients; however, the stem-like to differentiated cell ratio was preserved across samples within patients. In the entire cohort, patients with tumors exhibiting a high proportion of the stem-like component, defined as stem-like tumors, had significantly shorter overall survival. Stem-like tumors exhibited significantly higher levels of the RTK_I component and lower proportions of the RTK_II component compared to non-stem-like tumors. A machine learning model that utilizes support vector machines to classify stem-like tumors using radiomic features achieved a mean AUC of 0.71 (95% confidence interval: 0.56–0.85) in nested cross-validation.



 We showed that radiomethylomics can be used to predict tumor composition and patient survival from preoperative MRI in GBM. Based on these promising results we will increase our power with additional samples to facilitate a personalized approach to GBM management.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff84a274afa2828c086628b34b40943cc6ef06da" target='_blank'>
              NIMG-27. RADIOMETHYLOMICS FOR NON-INVASIVE PREDICTION OF TUMOR PHENOTYPES AND SURVIVAL IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Yuji Matsumoto, Omkar Singh, Jose A. Garcia, Sunwoo Kwak, Zied Abdullaev, Nelson Freeburg, Dana Silverbush, Kenneth J Aldape, C. Davatzikos, M. Nasrallah
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Infiltration into the brain tissue is one of the main features of diffuse midline gliomas (DMG), also characterized by the histone mutation H3K27M and overall survival of 12-15 months from diagnosis. To study which genes are critical for H3K27M-DMG tumor cell migration in the brain, we established a novel two-step pooled whole genome CRISPR-Migration screen of metastatic H3K27M-DMG (n=3) and glioblastoma (GBM, n=2) stem cells by using serum-free conditions and laminin-dependent cell attachment, mimicking cell migration on brain vasculature scaffolds. Genes involved in focal adhesion (ITGB1, CRKL, PARVA, PTK2, FERMT2) were identified as significantly restricting migration in all H3K27M-DMG lines. From those, only knockout of ITGB1 (ITGB1-KO) fully ablated in vitro migration in all models. In human tumor RNAseq, expression of ITGB1 correlated with higher glioma grade and worse survival specifically in H3K27M-DMG across pediatric brain tumors. In H3K27M-DMG scRNAseq, increased ITGB1 expression was found in cells with OPC-like/Mesenchymal cell signature associated with leading edge and microvasculature. Unexpectedly, ITGB1-KO led to the decreased expression of multiple mitochondrial NADH dehydrogenase genes, and reduced levels of glycolysis and oxidative phosphorylation (OXPHOS) activity. Preliminary mechanistic data linked this phenotype to increased mitochondrial fusion. Genetic (ITGB1-KO) and pharmacologic (anti-ITGB1 antibody AIIB2, CNS delivered) targeting of ITGB1 in a novel orthotopic model of metastatic H3K27M-DMG (UMPED83) led to reduced proliferation and significant extension of survival (p=0.0003 ITGB1-KO, and p=0.0006 AIIB2). Interestingly, ITGB1-KO in another orthotopic H3K27M-DMG model (SVZ) led to upregulation of other integrins (ITGB3 and ITGB5) without change in survival, resulting in susceptibility to treatment with the ITGB3/5 inhibitor Cilengitide (p=0.006). Treatment with AIIB2 and Cilengitide alone or in combination was ineffective in an H3WT adult GBM orthotopic model suggesting unique H3K27M-DMG targetability. In summary, targeting of integrin pathways offers untapped therapeutic opportunities in H3K27M-DMG by impairing tumor cell survival through altered mitochondrial dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6ec537ed13e076cfe34a9c0ef1b60c688004ced" target='_blank'>
              TMOD-29. BETA INTEGRINS REGULATE MITOCHONDRIALLY-MEDIATED CELL MIGRATION IN H3K27M-DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            R. Cartaxo, Edwin Nieblas-Bedolla, Michael Niculcea, S. Natarajan, Tiffany Adam, Madison Clausen, Madeline Miclea, Ruby G. Siada, K. Schwark, Sunjong Ji, Ananya Gera, Robert Doherty, Stefanie Galbán, Nicholas Nuechterlein, Mariella G. Filbin, S. Venneti, Toshiro Hara, C. Koschmann
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 Adult-type diffuse gliomas, the most common primary brain tumours, remain a clinical challenge in oncology with limited treatment options, restricted anti-tumour immune response and a dismal patient prognosis. Despite their immunosuppressive microenvironment, the formation of lymphoid aggregates containing adaptive immune cells has been reported in gliomas. However, the cellular compositions, immunological function, and relevance of lymphoid aggregates for adaptive anti-tumour immunity is not well understood. Therefore, we performed a comprehensive, unbiased analysis of lymphoid aggregation in 642 adult-type diffuse gliomas using a multi-modal approach, combining DNA methylation and RNA sequencing with spatial transcriptome and proteome profiling. Overall, B cell aggregates and tertiary lymphoid structures (TLS) were observed in 15% of tumours, with the highest prevalence in primary glioma localised to the temporoparietal lobe. The presence of TLS was associated with an improved overall survival. Gliomas containing TLS displayed a remodelled perivascular space, characterised by transcriptional upregulation and spatial redistribution of collagens associated with barrier functions. Spatial transcriptome and proteome profiling revealed heterogenous B:T cell interactions that were associated with elevated CD8 T-cell numbers and differences in IgA and IgG plasma cell forming capacity, suggestive of dynamic adaptive immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8853943d0d644fd815ddb5d51b11bdcf65c71b9" target='_blank'>
              TMIC-39. SPATIAL MULTIOMICS PROFILING REVEALS HETEROGENEITY OF B:T CELL INTERACTIONS AND PLASMA CELL FORMATION IN TERTIARY LYMPHOID STRUCTURES IN HUMAN GLIOMAS
              </a>
            </td>
          <td>
            P. Cakmak, J. Lun, Aakanksha Singh, J. Macas, Michael C. Burger, Lucie Marie Hasse, Elke Hattingen, M. Köhler, J. Schupp, Tatjana Starzetz, Eike Steidl, Karlheinz Plate, Y. Reiss, K. Imkeller
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 As breast cancer migrates into the brain, it acquires distinct genomic, immunologic and phenotypic changes through interactions with the local microenvironment. We sought to determine how targetable gene expression changes mediate breast cancer extravasation into and proliferation within the brain. We created a brain metastatic cell line (“brain-seeking”) via serial intracardiac injection of mCherry+ 4T1 breast cancer cells in Balb/c mice and flow cytometry isolation of micrometastases. We conducted in vitro and in-vivo studies to uncover immunological and genetic differences between primary breast cancer and brain metastases. In culture, we observed significant upregulation in adhesion (p=0.01) and invasion (p=0.0004) along with upregulation of adhesion-promoting genes such as Sdc3, Serpine2 or ECM1 in brain-seeking cells. We observed significant differences in survival of Balb/c mice injected intracranially with brain-seeking cells vs. breast cancer cells (p=0.04). In the mammary fat pad (MFP), parental 4T1 cells caused increased mortality vs. brain-seeking cells (p=0.003). This rejection of brain-seeking 4T1 cells in the MFP could be due to prior adaptation to the brain microenvironment, while rejection of parental 4T1 cells in the brain could be due to lacking said adaptations. Consistent with this finding, of 20,382 analyzed genes in both cell lines, 108 related to immune regulation were significantly upregulated in brain-seeking cells including Cxcl1, Dusp1, C3. Flow cytometry showed increased cytotoxic and helper T cells in intracranially injected parental vs. brain-seeking cells (p <0.001). In the MFP, however, 4T1 parental cells displayed CD3+ T cell depletion and an increased number of neutrophils (p <0.05) while the “brain-seeking cells” showed increased numbers of CD3+ and CD4+ cells (p <0.01) and macrophages (p<0.05). This suggests that brain metastases evolve from their primary tumor through adhesive and invasive adaptations promoting extravasation, along with unique immunological adaptations that allow their proliferation free of immune surveillance in the brain microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb40f83672977d31ed510d2a8f39a12e5698eb1" target='_blank'>
              TMIC-01. TARGETABLE GENE EXPRESSION CHANGES MEDIATE EXTRAVASATION OF BREAST CANCER CELLS FROM THE CIRCULATION AND LOCAL PROLIFERATION IN THE BRAIN
              </a>
            </td>
          <td>
            Venina Kalistratova, R. Morshed, Eric Chalif, Sabraj A. Gill, Harmon Khela, Meeki K. Lad, Ankita Sati, Aakriti Mathur, M. Aghi
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 The extracellular matrix (ECM) plays a critical role in cancer progression, with its biophysical properties influencing cancer behavior. ECM stiffness can induce long-lasting adaptations in cancer cells, promoting sustained invasion through a process known as “mechanical memory” (MM). Glioblastoma (GBM) exhibits spatial heterogeneity in stiffness, with stiffer ECM found at the tumor periphery where invasive cells reside and promote recurrence. We hypothesize that increased stiffness at the tumor periphery primes GBM cells for sustained invasion through MM.



 Cell lines were established from image-guided human GBM samples from tumor margin and core (n=10 patients, 20 cell lines). Cells from random locations were conditioned in physiologically relevant extracellular matrices for two weeks. Human organotypic assay (n=3), Migration Assays, Atomic Force Microscopy (AFM), Immunocytochemistry, Western Blot, and RNA-sequencing (n=6) were performed.



 GBM cells at higher stiff surfaces displayed increased migration (p<0.01). Magnetic Resonance Elastography (n=9) confirmed GBM heterogeneity, with softer cores (0.2-0.5kPa) transitioning to stiffer margins (>2kPa). Organotypic scaffolds mimicking the tumor margin promoted enhanced migration of GFP-labeled GBM cells. Margin cells and cells conditioned on stiffer matrices demonstrated higher migratory potential compared to their core and soft-conditioned counterparts (p<0.05). These trends persist over time and across multiple passages. ECM stiffness exposure causes nanomechanical, cytoskeletal, and transcriptional adaptations. AFM shows that cells from the margin (n=6) are softer (p<0.01) and more deformable (p<0.01). RNAseq identified upregulation of MM-specific genes in GBM margin cells. The mechanosensitive transcription factor YAP1 emerged as a key driver of MM. Notably, YAP1 is targetable by the FDA-approved drug Verteporfin. YAP1 expression was elevated in margin and stiff-conditioned cells (p<0.05). Stiff-conditioned cells and margin cells displayed increased sensitivity to Verteporfin treatment (p<0.05), with treatment significantly reducing margin migration compared to the core (p<0.001).



 Stiffer GBM margins promote the formation of aggressive phenotypes through MM. Targeting YAP1 with Verteporfin, a readily available drug, presents a promising strategy to halt GBM invasion and improve patient outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d3df0979f0c2cba7865ec958f4264c5339cc38" target='_blank'>
              TMIC-67. EMERGING ROLE OF EXTRACELLULAR MATRIX STIFFNESS AS A HALLMARK OF GLIOBLASTOMA: INSIGHT TO MECHANICAL MEMORY FORMATION AND PROLONGED INVASION IN THE TUMOR PERIPHERY
              </a>
            </td>
          <td>
            Paola Suarez Meade, Rachel Whitehead, Jean Clemenceau, E. Norton, Tanmay Kulkarni, Changjin Hong, Caroline Teeter, Manuela Aramburu, K. Chaichana, Steven S. Rosenfeld, Rajappa S. Kenchappa, H. Guerrero-Cázares, P. Schiapparelli, Tae H Hwang, Konstantinos Konstantopoulos, A. Quiñones-Hinojosa
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="
 Brain tumors arise in close association with neurons, suggesting that these non-neoplastic cells may be critical stromal drivers of brain tumor initiation and growth. Previously, we have shown that murine low-grade optic glioma formation and progression in the setting of Neurofibromatosis type 1 (NF1) is dictated by neurons and neuronal activity. In these studies, these neuronal dependencies reflected neuronal activity-driven enzymatic cleavage of a growth factor (neuroligin-3) from oligodendrocyte precursors cells (tumor initiation) or neuronal production of a paracrine factor to stimulate T cell support of optic glioma growth (tumor progression). Since neurons typically communicate with other neurons through neurotransmitters, we sought to explore the possibility that neurotransmitters operate to modulate low-grade glioma growth using humanized mouse models of pilocytic astrocytoma (PA). Leveraging single cell RNA sequencing of three independent sets of pediatric PAs, we identified neurotransmitter pathway enrichment in the tumor cells. This neurotransmitter pathway enrichment reflected aberrant expression of specific neurotransmitter receptors, which we confirmed in three independently generated PA tissue microarrays and in five distinct primary PA cell lines grown in vitro. Moreover, this aberrant neurotransmitter receptor expression established differential neurotransmitter PA growth dependencies in vitro. Importantly, interruption of neurotransmitter signaling in human PA xenografts attenuated tumor growth and ERK activation in Rag1-/- mice in vivo. Finally, we discovered crosstalk between neurotransmitter receptor and receptor tyrosine kinase signaling that revealed another target for therapeutic inhibition. Taken together, these findings elucidate a previously unknown neurotransmitter PA growth dependency amenable to therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23e8358003ff9c22c4c3138ccf4ef59222584db" target='_blank'>
              CNSC-43. NEUROTRANSMITTER DEPENDENCY UNDERLIES TUMOR GROWTH IN HUMANIZED PEDIATRIC LOW-GRADE GLIOMA MODELS
              </a>
            </td>
          <td>
            C. Anastasaki, Rasha Barakat, Rui Mu, Joshua P Koleske, Chloe M Kernan, Yunqing Gao, Olivia M Cobb, Corrine Gardner, Hongjin Shu, Lakshmi Ramachandran Nair, S. Dahiya, Charles G. Eberhart, Joanna J Phillips, Steven J Mennerick, Jennifer Strahle, Fausto J. Rodriguez, David H. Gutmann
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55719070e9f1cff38fbdc568fedbec8bc7d69e98" target='_blank'>
              Single-cell transcriptomic and spatial analysis reveal the immunosuppressive microenvironment in relapsed/refractory angioimmunoblastic T-cell lymphoma
              </a>
            </td>
          <td>
            Mengyan Zhu, Ning Li, Lei Fan, Rongrong Wu, Lei Cao, Yimin Ren, Chuanyang Lu, Lishen Zhang, Yun Cai, Yuzhu Shi, Zihan Lin, Xueying Lu, Jiayan Leng, Shiyang Zhong, Xingfei Hu, Bin Huang, Runheng Huang, Wanting Zhou, Diru Yao, Lingxiang Wu, Wei Wu, Quanzhong Liu, Peng Xia, Ruize Chen, W. Shi, Ruohan Zhang, Sali Lv, Chunling Wang, Liang Yu, Jianyong Li, Qianghu Wang, Kening Li, H. Jin
          </td>
          <td>2024-12-01</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/456d5e0f44d3a56861aef434239933c23d1853a0" target='_blank'>
              Single-cell RNA sequencing analysis reveals the dynamic changes in the tumor microenvironment during NMIBC recurrence.
              </a>
            </td>
          <td>
            Ziang Chen, Tianxiang Zhang, Weijian Li, Jiawei Hu, Y. Ou, Fangdie Ye, Jinhao Zhang, Haowen Jiang, Shenghua Liu
          </td>
          <td>2024-12-04</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Complete tumour resection in glioblastoma patients is not possible. Residual therapy-resistant edge-derived cells drive tumour recurrence and infiltrative expansion in glioblastoma. Features that are specific to malignant edge-derived cells may serve as predictive biomarkers to allow individual tailoring of the treatment plan to slow down tumour invasion and prevent tumour recurrence in GBM patients following standard therapy. Intratumoural spatial heterogeneity facilitates therapeutic resistance and recurrence in glioblastoma. Our knowledge on glioblastoma heterogeneity is mostly restricted to the surgically resectable tumour core, while the functional characterization of tumour cells at the infiltrating edge remains largely elusive due to the presence of normal functional brain tissue in the peritumoural lesion. Edge-derived cells exhibit larger capacity for infiltrative expansion and are the main drivers of treatment failure and tumour recurrence, making them action targets for novel treatment approaches in glioblastoma. To resolve the transcriptional heterogeneity of GBM within the spatial context, we profiled gene expression of different tumour regions (“edge” and “core”) obtained at initial surgical resection from primary and recurrent IDH-WT GBM patients with Visium 10x and GeoMx. We show that infiltrative edge-derived cells are spatially segregated and are characterized by regionally shared distinct genomic and transcriptomic signatures that promote invasiveness and underly disease recurrence. Tumour edge is enriched for neuronal signatures, while tumour core displays larger transcriptional subpopulation diversity and abundance of proliferative cells with a high capacity for self-renewal, consistent with the proliferative and cancer stem cell-enriched phenotype in early GBM progressors. Upregulated DEGs of tumour edge cells are significantly associated with ion regulation of transport, chemical synaptic transmission, and nervous system development. These modules may represent tumour cell hijacking of neuronal programs specifically at the tumour periphery as described in the context of glioma-neuron synaptic communication and formation of neurite-like microtubes. Upregulation of ion regulation transport at the tumour periphery indicates enhanced neuronal activity and excitability driving infiltrating growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40797756057776083a00011e6e40dee03507caf6" target='_blank'>
              EPCO-38. IDENTIFYING THE MOLECULAR SIGNATURE OF INFILTRATING EDGE IN GLIOBLASTOMA AS DRIVERS OF TUMOUR INVASION AND RECURRENCE
              </a>
            </td>
          <td>
            Alyona Ivanova, Shamini Ayyadhury, M. Wu, David G Munoz, Trevor Pugh, Sunit Das
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="

 Acute myeloid leukemia (AML) is a highly heterogeneous hematologic malignancy characterized by a poor prognosis, primarily due to resistance to primary chemotherapy and a high relapse rate among patients in remission. This underscores the limitations of conventional chemotherapy in achieving a cure for the majority of AML patients. Spatial transcriptome (ST) technology has emerged as a powerful tool for studying the spatial gene expression and cell-cell interaction patterns, yet its application to calcium-rich tissues like bone marrow has been challenging. In this study, we firstly introduced a novel approach for the paraffin embedding and decalcification of bone marrow tissue, which is fully compatible with Visium glass slides. To validate our method, we collected bone marrow tissues form AML patients and performed spatial transcriptomic sequencing alongside single-cell RNA sequencing. We totally got 5364 ± 2049 (AML-b8) and 4892 ± 2096 (AML-b9) unique molecular identifiers (UMIs) per spot, which is significantly higher than the published studies of bone marrow, and equal to those of soft tissues. This demonstrates that our approach overcomes the limitations of spatial transcriptome data collection in calcium-rich tissues. Utilizing marker genes, we accurately mapped the spatial distribution of megakaryocytes (MKs) and bone trabecular in bone marrow, confirmed through HE staining. For the first time, we integrated spatial transcriptome data, single-cell sequencing and histological analysis to provide a comprehensive characterization of bone marrow in AML patients. After confirming the cell type of each spot, we present the maps of bone marrow tissues in AML patients that could provide both spatial information and gene expression level. Furthermore, we found that the proportion of blasts in our ST results more accurately reflects the tumor burden in AML patients compared to bone marrow smear and flow cytometry, due to the preservation of tissue architecture. Our analysis distinguished seven subpopulations within the tumor cells of bone marrow, including leukemia stem cells (LSKs), highlighting the potential of our approach in elucidating the distribution and transcriptomic profiles of LSKs in AML patients. In our ST models, we observed a significant correlation between genes' expression level and their distance to endosteal niche, such as S100A8, S100A9, CYBB, CHMP3, PACS2, UNC93B1, CMKLR1 and TFRC. Functional analysis of intra-tumor hubs revealed key pathways, including Toll-like receptor cascades, apoptosis process and so on, which are pivotal in bone marrow samples. In conclusion, we present a pioneering method for bone marrow spatial transcriptome analysis, offering a complete and detailed profile of bone marrow in AML patients. This work paves the way for a deeper understanding of AML pathophysiology and may lead to the development of more effective therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d1ed9f35bdf12e25119bc30e92359c1da8a97ae" target='_blank'>
              Integrating Spatial Transcriptome, Single-Cell Sequencing and Histopathology Reveals Bone Marrow Spatial Heterogenous in AML Patients
              </a>
            </td>
          <td>
            Xiangjie Lin, Huafeng Wang, Jie Jin
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Prostate cancer is the most common malignant tumor in male genitourinary system. The incidence of prostate cancer ranks the first among all male malignant tumors worldwide, and the mortality rate ranks the second among all male malignant tumors. Prostate stem cells are heterogeneous subsets with the function of self-regeneration and proliferation in the prostate, which can produce all cell lineages that make up the prostate epithelium. At present, the pathogenesis of prostate cancer remains unclear. According to cancer stem cell hypothesis, prostate cancer may be a stem cell disease, which provides a new direction for revealing the pathogenesis of prostate cancer and developing treatment strategy for prostate cancer. In this mini-review, we highlight recent advances in our understanding of the origin, surface molecular markers, signaling pathway and the significance for clinical treatment of prostate cancer stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6bed3a70a93e4f1fc6b0e26fc9c0504d0264a3c" target='_blank'>
              Prostate cancer stem cells: an updated mini-review
              </a>
            </td>
          <td>
            Kaichen Zhou, Haosen Lu, Jielin Zhang, Qi Shen, Pengzhan Liu, Qiqing Xu, Chunhua Yang, Lijun Mao
          </td>
          <td>2024-10-28</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7713144e1809faa54b6475ca46362db89d138cc3" target='_blank'>
              In vitro model of retinoblastoma derived tumor and stromal cells for tumor microenvironment (TME) studies
              </a>
            </td>
          <td>
            Emily Alefeld, André Haase, Dario Van Meenen, Bettina Budeus, Oliver Dräger, Natalia Miroschnikov, Saskia Ting, Deniz Kanber, Eva Biewald, Nikolaos Bechrakis, Nicole Dünker, M. Busch
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is characterized by inter- and intra-tumor molecular heterogeneity and demonstrated cell plasticity. Sex differences have been observed in various aspects: a male:female GBM incidence ratio is 1.6:1; standard of care treatments (debulking, radiation, and temozolomide) benefit females more than males. 14 human GBM samples from two study sites were subject to spatial profiling on the Vizgen Merscope platform to examine whether distinct biological and therapeutic niches in GBM were evident, to understand how the tumor microenvironments (TME) contributed to the heterogeneity of GBM, and to interrogate sex difference in the immune landscape of GBM. Multiple sets of spatial probes were designed to capture the cell identities (cancer stem cells, astrocytes, oligodendrocytes, microglia, macrophages, etc.) and the spatially-resolved expression patterns of genes with different functions. Two of the 14 GBM samples were also profiled on another FISH-based spatial platform (Veranome). The data from the two platforms exhibited a linear correlation. We explored the gene-gene co-expression network, and calculated the interaction probabilities between genes which were then compared for sex difference. We found that female samples showed significantly lower (two-fold) CD4+/CD8A+ T-cell count ratios than male samples. We also observed sample-specific spatial niches of different GBM cell subtypes and immune cell populations. Our large-scale spatial transcriptomics study revealed intrinsic sex-based differences in immune response to tumor and illuminated microenvironmental influences on GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f9e48ec997a8efb730a81775200a0b844dc3558" target='_blank'>
              EPCO-44. SPATIAL TRANSCRIPTOMICS ENABLES STATISTICAL MODELING OF SEX DIFFERENCE IN GENE CO-EXPRESSION NETWORK AND IMMUNE LANDSCAPE OF GLIOBLASTOMA
              </a>
            </td>
          <td>
            Yue Hao, Angela Baker, G. Reid, Jingqin Luo, Brian Hilbush, Marcos Otero, Christopher Streck, Karen L. Fink, George J. Snipes, Bruce E. Mickey, Andrew E. Sloan, J. Barnholtz-Sloan, Michael E. Berens
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="INTRODUCTION
The tumor microenvironment is comprised of neoplastic cells and a variety of host cell types. Investigation of cell dynamics within this environment has motivated in vitro and ex vivo biomimetic model development. Our lab recently introduced the tumor spheroid-rat mesentery model to investigate cancer-induced lymphatic/blood vessel remodeling. To validate the physiological relevance of this model, the objective of this study was to determine the effect of tumor spheroids on microvascular remodeling after transplantation onto rat mesenteric.


METHODS
Spheroids derived from H1299 lung cancer cells were seeded onto rat mesenteric tissues during a survival surgical procedure. Tissues were harvested 3-5 days post-seeding and stained with PECAM and LYVE-1 to identify blood and lymphatic vessels respectively.


RESULTS
At all timepoints, cancer cells remained adhered to the tissue. Tissues seeded with tumor spheroids were shown to have increased vascular density, capillary sprouting, and tortuosity compared to sham tissues exposed to sterile saline only. Tumor spheroids also induced the formation of lymphatic/blood vessel connections and LYVE-1 negative protrusions emerging from lymphatic vessels.


CONCLUSION
Overall, this study underscores the use of in vivo modeling to aid in the discovery of novel vascular growth dynamics and offers new methodologies for studying tumor-induced remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6f3b17e8b39c71b6c005e826ecc691665f0193" target='_blank'>
              A Novel In Vivo Rat Mesentery Model for Studying Tumor Spheroid-Induced Microvascular Remodeling.
              </a>
            </td>
          <td>
            Arinola O. Lampejo, Luciana Fonseca Perez, Miriam M Girgis, Blanka Sharma, Dietmar W Siemann, W. Murfee
          </td>
          <td>2024-12-19</td>
          <td>Journal of vascular research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Despite aggressive therapy combining surgical resection, radiation and chemotherapy, glioblastoma (GBM) almost always recurs. Currently, there is no standard treatment for recurrent GBM, and patient inclusion in clinical trials is based on the genetic and molecular profile of the primary tumor even though the effects of chemoradiation on the transcriptional subtype in matched primary and recurrent GBM remain largely understudied. Paralleling the GLASS consortium, we sought to perform a longitudinal assessment of GBM recurrence after therapy based on molecular subtype with the hypothesis that recurrent tumors exhibit a distinct transcriptional landscape different from treatment-naive tumors. To address this, we utilized primary tumor-derived glioma stem cells (GSCs) representing classical and proneural subtypes. We then undertook in vivo intracranial implantation studies and performed standard concomitant treatment that mimics the Stupp protocol. Upon bulk RNAseq data analyses of untreated (n=21) and recurrent tumors (n=17) we identified that while the molecular subtype of recurrent tumors does not change post therapy, their transcriptional profile is more heterogenous compared to untreated tumors for both subtypes. Hierarchical cluster analyses revealed distinct sub-clusters of recurrent tumors. Common mechanisms of recurrence include upregulation of angiogenesis pathways and PI3k/Akt/mTOR signaling pathways. Interestingly, recurrent tumors exhibit upregulation of genes involved in cellular metabolism and membrane transport/ potential. The source of heterogeneity in recurrent tumors in both subtypes appears to be due to the diversity in expression of gene sets involved in hallmark of cancer pathways such as cell cycle, Notch and Wnt signaling pathways and heterogeneity in oncogenic signatures and cell states resembling fetal or neural cells. Further research into the recurrent tumor microenvironment by spatial multi-omics approaches using this pre-clinical recurrent GBM model will elucidate the linkage between cellular as well as molecular heterogeneity and enable identification of targets that can help drive therapeutic decisions for recurrent GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbf4b1b944267298359c5f873eaa544818a4a7d9" target='_blank'>
              TMIC-46. TRANSCRIPTIONAL HETEROGENEITY AND MECHANISTIC PATHWAYS OF RECURRENT GLIOBLASTOMA: INSIGHTS FROM A PRE-CLINICAL RECURRENT TUMOR MODEL
              </a>
            </td>
          <td>
            Mohammad Fayyad Zaman, Marc Daou, Lynette M. Phillips, Sanjay Singh, L. Long, J. Gumin, Daniel Ledbetter, Frederick F Lang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Glioblastoma multiforme (GBM) remains the most lethal and prevalent primary brain tumor worldwide. Despite advances in understanding its biology and multimodal therapy options, the overall prognosis for GBM patients remains bleak, primarily due to the high recurrence rate, which is intimately linked to tumor resistance against standard therapeutic interventions. Glioblastoma stem cells (GSCs) contribute to therapeutic resistance and cellular heterogeneity through their self-renewal properties and ability to adapt. Thus, identifying new molecular targets driving GBM progression is crucial for developing effective therapies. Recent advancements in genome biology have revealed that long non-coding RNAs (lncRNAs) play roles in various biological functions, and the dysregulation of numerous lncRNAs has been shown to be associated with specific cancers. To identify GSC-associated lncRNAs, a comprehensive analysis was performed using single-cell RNA sequencing datasets from GBM tumors, GBM organoids, GSC-enriched GBM tumors, as well as normal human brain samples. A chromatin-enriched lncRNA, GIHCG, was identified as being highly expressed in GSCs compared to non-neoplastic neural stem cells. The GIHCG gene, located on human chromosome 12q14.1, is frequently amplified in glioblastoma and impacts overall survival. Bioinformatic and functional analyses revealed that GIHCG is amplified in 13% of GBM patient samples, and high expression of GIHCG is a negative prognostic marker for GBM patients, with significantly reduced overall survival compared to the low expression group. In addition, the depletion of GIHCG significantly reduced GSC proliferation and migration. Further, the knockdown of GIHCG decreased both mRNA and protein expression levels of critical GSC stemness factors, including SOX2, NESTIN, and OLIG2. Moreover, GIHCG depletion reduced the sphere formation capacity of GSCs, demonstrating the functional importance of GIHCG in the maintenance of GSC stemness. These results indicate that GIHCG is essential for GSC proliferation, migration, and stemness, underscoring its potential therapeutic value in RNA-based therapies for combating glioblastoma malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24d912d5117a292a29ded4aacf7d2e6cd8ccb4fe" target='_blank'>
              STEM-19. TARGETINGGIHCG LNCRNA IN GLIOBLASTOMA STEM CELLS: A POTENTIAL THERAPEUTIC STRATEGY
              </a>
            </td>
          <td>
            R. Hazra
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/869f2c79b0e7b6ec678d9e9104097920f2acb08f" target='_blank'>
              Three-dimensional dynamics of mesothelin-targeted CAR.CIK lymphocytes against ovarian cancer peritoneal carcinomatosis
              </a>
            </td>
          <td>
            F. Galvagno, V. Leuci, Annamaria Massa, C. Donini, R. Rotolo, S. Capellero, Alessia Proment, Letizia Vitali, Andrea Maria Lombardi, V. Tuninetti, Lorenzo D'Ambrosio, A. Merlini, Elisa Vigna, Giorgio Valabrega, Luca Primo, A. Puliafito, Dario Sangiolo
          </td>
          <td>2024-11-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ef6a160eddfbbad312082d26af8e03b167d27f" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance.
              </a>
            </td>
          <td>
            Michael J Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah R. Amend, James B. Hicks
          </td>
          <td>2024-11-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Accurately tracking dynamic state transitions is crucial for modeling and predicting biological outcomes, as it captures heterogeneity of cellular responses. To build a model to predict bacterial infection in single cells, we have monitored in parallel infection progression and metabolic parameters in thousands of human primary macrophages infected with the intracellular pathogen Legionella pneumophila. By combining livecell imaging with a novel tool for classifying cells based on infection outcomes, we were able to trace the specific evolution of metabolic parameters linked to distinct outcomes, such as bacterial replication or cell death. Our findings revealed that early changes in mitochondrial membrane potential (Δψm) and in the production of mitochondrial Reactive Oxygen Species (mROS) are associated with macrophages that will later support bacterial growth. We used these data to train an explainable machine-learning model and achieved 83% accuracy in predicting L. pneumophila replication in single, infected cells before bacterial replication starts. Our results highlight backtracking as a valuable tool to gain new insights in host-pathogen interactions and identify early mitochondrial alterations as key predictive markers of success of bacterial infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c78262ad5b5af0d442098da3e8a482b2eccacb08" target='_blank'>
              Backtracking Metabolic Dynamics in Single Cells Predicts Bacterial Replication in Human Macrophages
              </a>
            </td>
          <td>
            Mariatou Dramé, Dmitry Ershov, Jessica E. Martyn, J. Tinevez, C. Buchrieser, Pedro Escoll
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Abstract Although immune checkpoint blockade therapy has shown promising results in a small subset of patients with colorectal cancer with high microsatellite instability, the majority of patients with colorectal cancer do not respond to immune checkpoint blockade therapy. The main obstacle to the success of immunotherapy in cancer treatment is the exhaustion of tumor-infiltrating lymphocytes (TIL). Elucidating the spatial organization of immune checkpoints within the tumor microenvironment (TME) could pave the way for the development of novel prognostic tools and therapeutic strategies to enhance antitumor immune responses. To clarify the spatial and functional diversity of TILs in the colorectal TME, we performed multiplexed IHC to examine the exhaustion of TILs in the TME, the expression of PD-1 and T-cell immunoglobulin and mucin domain–containing protein 3 (TIM-3), which are major biomarkers of T-cell exhaustion, and least absolute shrinkage and selection operator method–Cox analyses of the correlation between colorectal cancer prognosis and TME features. For proof of concept, the antitumor efficacy of TIM-3 and PD-1 dual blockade in colorectal cancer was further evaluated in a CT26 s.c. tumor model of human colorectal cancer. We found that the spatial context of PD-1 and TIM-3 successfully predicted the overall survival of patients with colorectal cancer independent of tumor–node–metastasis stage. Dual targeting of PD-1 and TIM-3 in mouse tumor models inhibited tumor progression and reduced T-cell exhaustion, indicating a potential strategy for improving the clinical treatment of colorectal cancer. Significance: The identification of specific spatial patterns of immune checkpoint expression that correlate with overall survival in patients with colon cancer suggests a potential prognostic tool for risk stratification and treatment selection. These findings pave the way for the development of novel therapeutic strategies to enhance antitumor immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b7e667f607ec054318ced952999594c8c938db" target='_blank'>
              Spatial Context of Immune Checkpoints as Predictors of Overall Survival in Patients with Resectable Colorectal Cancer Independent of Standard Tumor–Node–Metastasis Stages
              </a>
            </td>
          <td>
            Hao Kong, Qingxin Yang, Chunwei Wu, Xiangji Wu, Xinrui Yan, Li-Bin Huang, Lu Chen, Zongguang Zhou, Ping Wang, Hong Jiang
          </td>
          <td>2024-11-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8645f518787a0ca20fba12896526944d30b5806b" target='_blank'>
              Development of patient-derived lymphomoids with preserved tumor architecture for lymphoma therapy screening
              </a>
            </td>
          <td>
            A. Santamaria-Martínez, J. Epiney, Divyanshu Srivastava, Daniele Tavernari, Marco Varrone, Dina Milowich, Igor Letovanec, Thorsten Krueger, Rafael Duran, Giovanni Ciriello, A. Cairoli, E. Oricchio
          </td>
          <td>2024-12-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cellular senescence is a state of irreversible cell cycle arrest accompanied by a distinctive inflammatory secretory profile known as the senescence-associated secretory phenotype (SASP). While various biomarkers, such as senescence-associated beta-galactosidase (SA-βgal), EdU incorporation, P21, and P16, are used to identify senescent cells, no single biomarker universally defines cellular senescence, and current methods often fail to address heterogeneity in biomarker expression levels. This study leverages single-cell fluorescence imaging to assess multiple senescence markers including SA-βgal enzymatic activity, P21 and IL-6 expression, and nuclear and cell area, in chemotherapy-induced (mitomycin C) and oxidative stress-induced (D-galactose) senescence models in human fibroblasts. Our findings reveal significant heterogeneity in SA-βgal activity and distinct sub-populations within senescent cells. Nuclear and cell area measurements emerged as robust indicators of cellular senescence, displaying similar variability across individual cells. Importantly, we identified specific nuclear area sub-populations that strongly correlate with IL-6 expression levels, demonstrating a relationship between the heterogeneous expression of senescence biomarkers and the SASP. To address this heterogeneity, we introduced an induction threshold method to more accurately quantify the percentage of cells expressing senescence biomarkers. Furthermore, in both senescence models, we observed that rapamycin, a well-known senomorphic agent, selectively targets specific biomarker-expressing sub-populations. This study underscores the value of assessing cellular heterogeneity in senescence research and provides an improved approach for analysing senescence markers in diverse cellular contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/628d7d0deaaee786490438495d7bcdca1881906e" target='_blank'>
              Single-cell fluorescence imaging reveals heterogeneity in senescence biomarkers and identifies rapamycin-responsive sub-populations
              </a>
            </td>
          <td>
            Vijayraghavan Seshadri, Charmaine Chng, Joel Tyler, Kaveh Baghaei, Yan Wang, Nuri Gueven, Iman Azimi
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 The evolution of cellular heterogeneity in IDH-wildtype glioblastoma (GBM) after standard-of-care therapy remains poorly understood. To address it, we assembled a longitudinal cohort of 121 primary and recurrent GBM specimens from 59 patients, with extensive clinical annotations, and profiled it by single-nucleus RNA-sequencing and bulk tumor DNA sequencing. In most cases, longitudinal samples diverged in their composition of cell types and cell states. However, almost all theoretical trajectories were observed in our cohort such that the overall distribution of cell types and cell states was comparable between primary and recurrent samples. The most consistent longitudinal effect (66% of patients) was a lower malignant cell fraction at recurrence and a reciprocal increase in proportions of glio-neuronal TME cell types; in some cases, this was further accompanied by a coordinated shift of malignant cells towards neuronal-like states. MGMT methylation and radiation-related small deletion phenotypes were linked to particular trajectories, with depletion of mesenchymal-like cells and enrichment of hypoxia-related malignant cells, respectively. Importantly, changes in malignant states were also associated with specific changes in TME composition. In summary, our analysis highlights diverse longitudinal GBM trajectories that are shaped by treatment response and TME interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea653ba81a99c699e1e7affffaf3c3f66e7e9bb" target='_blank'>
              EPCO-34. DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            Avishay Spitzer, K. Johnson, M. Nomura, L. Garofano, Djamel Nehar-Belaid, Noam Galili Darnell, Alissa C. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, Simona Migliozzi, L. N. G. Castro, T. Chowdhury, N. Robine, C. Reeves, Jong Bae Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, Labeeba Nusrat, Sorcha Kellet, Sunit Das, H. Moon, S. Paek, Franck Bielle, A. Laurenge, Anna Luisa Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, Nobuhito Saito, D.M. Ashley, Stephen T Keir, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Breast cancer is a heterogeneous disease comprising various subtypes with distinct molecular characteristics, clinical outcomes, and therapeutic responses. This heterogeneity evidences significant challenges for diagnosis, prognosis, and treatment. Traditional genomic co-expression network analyses often overlook individual-specific interactions critical for personalized medicine. In this study, we employed single-sample gene co-expression network analysis to investigate the structural and functional genomic alterations across breast cancer subtypes (Luminal A, Luminal B, Her2-enriched, and Basal-like) and compared them with normal breast tissue. We utilized RNA-Seq gene expression data to infer gene co-expression networks. The LIONESS algorithm allowed us to construct individual networks for each patient, capturing unique co-expression patterns. We focused on the top 10,000 gene interactions to ensure consistency and robustness in our analysis. Network metrics were calculated to characterize the topological properties of both aggregated and single-sample networks. Our findings reveal significant fragmentation in the co-expression networks of breast cancer subtypes, marked by a change from interchromosomal (TRANS) to intrachromosomal (CIS) interactions. This transition indicates disrupted long-range genomic communication, leading to localized genomic regulation and increased genomic instability. Single-sample analyses confirmed that these patterns are consistent at the individual level, highlighting the molecular heterogeneity of breast cancer. Despite these pronounced alterations, the proportion of CIS interactions did not significantly correlate with patient survival outcomes across subtypes, suggesting limited prognostic value. Furthermore, we identified high-degree genes and critical cytobands specific to each subtype, providing insights into subtype-specific regulatory networks and potential therapeutic targets. These genes play pivotal roles in oncogenic processes and may represent important keys for targeted interventions. The application of single-sample co-expression network analysis proves to be a powerful tool for uncovering individual-specific genomic interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64e30ebf92a4fce7b814c321e06f91055619ed7a" target='_blank'>
              Single-Sample Networks Reveal Intra-Cytoband Co-Expression Hotspots in Breast Cancer Subtypes
              </a>
            </td>
          <td>
            Richard Ponce-Cusi, Patricio López-Sánchez, Vinicius Maracaja-Coutinho, J. Espinal-Enríquez
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="

 Gene and cellular therapies extend the hope of cure to wider range of human diseases. Durability of these therapies is correlated to persistence of the adoptively conferred cells; while mature effector functions are key to effectiveness. Physiologically, cells are rarely simultaneously endowed with indefinite self-renewal and mature effector functions. Cellular therapies must, in response to environmental signals, dynamically transition between self-renewing and mature effector states while avoiding dysfunctional states.
 Chimeric Antigen Receptor (CAR) T cells have emerged as effective therapies for B cell and plasma cell malignancies. While they induce prolonged remission in most patients, fewer than half achieve durable control of their disease. Factors associated with durable remissions can be broadly categorized into CAR T cell extrinsic, encompassing malignancy type, tumor burden, antigen escape and the tumor microenvironment, and intrinsic factors, pertaining to the dynamics of transition between the persistent and mature effector states. Strategies to design of CAR T therapies to surmount intrinsic mechanisms of failure are essential.
 We carried out CRISPR activation screening targeting all human transcription factors and epigenetic modifiers in anti-CD20 CAR T cells cultured in vitro under conditions designed to provoke exhaustion. We observed statistically significant depletion in sgRNA targeting the TGFb (including KLF10) and TNFa (including NFATC4) signaling pathways in CAR with immunophenotypes associated with durability and effectiveness. Model-based Analysis of Genome-Wide CRISPR-Cas9 Knockout (MAGeCK) for the central memory population revealed enrichment of MFNG, a regulator of NOTCH signaling; NFIB, a master regulator of cell differentiation; SMYD2, an epigenetic regulator controlling Treg development; and TRIAP1, a TP53 regulated inhibitor of apoptosis. Although biologically plausible, these targets play important roles in regulating proliferation and cytotoxicity and entirely disabling these host-protective mechanisms raises important safety concerns.
 Transcription factors and epigenetic modifiers have broad effects on cellular states and regulatory mechanisms. By contrast, cis-regulatory elements (CRE) directly affect only the transcription of genes in their vicinity. Perturbating CRE alters T cell regulation with a finer degree of control and is incompletely explored. Systematically screening the large number of candidate CRE is not feasible via current experimental approaches.
 We developed a model, trained on single cell multi-omics data, to predict the time evolution of transcriptomes and regulomes. The introduction of chromatin state in the description of cellular dynamics, permits prediction of cellular dynamics after enhancing or disrupting specific and computes therapeutically salient metrics such as overall proliferation rate and proportions of CAR in memory and functional effector subsets.
 Our model predicts that the conditional knock out of the trans-acting factor TCF7 will increase the transition rates from naïve and memory T cell subsets into effector states; a prediction that agrees with the established biological roles for TCF7. By contrast, targeting the disruption of a candidate cis-regulatory element (cCRE) in the vicinity of KLF10 (chr8:101,790,788-101,791,270) is predicted to strongly increase transition probabilities into the Naïve and Memory subsets.
 Our model casts the problem of optimizing cellular therapies into the framework of dynamical systems theory and provides a rigorous approach for prioritizing the experimentally intractable number of perturbations by metrics directly relevant to persistence of efficacious adoptive cells. Future directions include the incorporation of additional single cell multiomic modalities to better characterize regulatory dynamics.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0387458d11fde4916f632ac6aee649e38a8a474b" target='_blank'>
              Unraveling CAR Differentiation Dynamics through in Vitro and in silico Perturbations
              </a>
            </td>
          <td>
            Wenjun Zhu, Conor Kelly, K. Keyvanfar, Pradeep Dagur, Stefan Cordes
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="ABSTRACT Background Angiogenesis is indispensable for the sustained survival and progression of both embryonic development and tumorigenesis. This intricate process is tightly regulated by a multitude of pro‐angiogenic genes. The presence of gene modules facilitating angiogenesis has been substantiated in both embryonic development and the context of tumor proliferation. However, it remains unresolved whether the pro‐angiogenic gene modules expressed during embryonic development also exist in tumors. Methods This study performed a pan‐cancer single‐cell RNA sequencing (scRNA‐seq) analysis on samples from 332 patients across seven cancer types: thyroid carcinoma, lung cancer, breast cancer, hepatocellular carcinoma, colorectal cancer, ovarian carcinoma, and prostate adenocarcinoma. Data processing was carried out using the Seurat R package, with rigorous quality control to filter high‐quality cells and mitigate batch effects across datasets. We used principal component analysis (PCA), shared nearest neighbor graph‐based clustering, and Uniform Manifold Approximation and Projection (UMAP) to visualize cell types and identify distinct cell clusters. Myeloid cell subpopulations were further analyzed for the expression of embryonic pro‐angiogenic gene modules (EPGM) and tumor pro‐angiogenic gene modules (TPGM). Results The analysis identified nine major cell types within the tumor microenvironment, with myeloid cells consistently exhibiting elevated expression of both tumor pro‐angiogenic gene modules (TPGM) and EPGM across all tumor types. In particular, myeloid cells, including macrophages and monocytes, showed high EPGM expression, indicating an active role of embryonic pro‐angiogenesis pathways in tumors. A subset analysis revealed 20 distinct myeloid subtypes with varying EPGM and TPGM expression across different cancers. Treatment and disease stage influenced these gene expressions, with certain subtypes, such as HSPAhi/STAT1+ macrophages in breast cancer, displaying reduced pro‐angiogenic gene activity post‐treatment. Conclusion This study provides evidence that tumors may exploit EPGM to enhance vascularization and support sustained growth, as evidenced by the elevated EPGM expression in tumor‐associated myeloid cells. The consistent presence of EPGM in TAMs across multiple cancer types suggests a conserved mechanism wherein tumors harness embryonic angiogenic pathways to facilitate their progression. Distinct EPGM expression patterns in specific myeloid cell subsets indicate potential therapeutic targets, particularly in cases where EPGM activation contributes to resistance against anti‐angiogenic therapies. These findings shed new light on the molecular mechanisms underlying tumor angiogenesis and highlight the prognostic relevance of EPGM expression in cancer, underscoring its potential as a biomarker for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f200ed8424d51876bd86eacf393553e56f7950" target='_blank'>
              Pan‐Cancer Single‐Cell Transcriptomic Analysis Reveals Divergent Expression of Embryonic Proangiogenesis Gene Modules in Tumorigenesis
              </a>
            </td>
          <td>
            Zeshuai Wang, Yiyi Su, Lisha Zhao, Wei Liu, Jiaqi Zhang, Wei Yang, Hanjie Li, Mingqian Feng, Hao Wang, Zhuo Song
          </td>
          <td>2024-11-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758b3e0bcca0b232b4864c10d9c2f0433098ac1f" target='_blank'>
              NF1 expression profiling in IDH-wildtype glioblastoma: genomic associations and survival outcomes
              </a>
            </td>
          <td>
            Michael Chang, Mohamed Sherief, Maria Ioannou, Viveka Chinnasamy, Lucy Chen, Michael Frost, Michelle Mattson-Hoss, Herb Sarnoff, D. Kamson, Matthias Holdhoff, D. Mukherjee, C. Bettegowda, Jordina Rincon-Torroella, Victoria Croog, Peng Huang, Fausto J Rodriguez, Calixto-Hope G. Lucas, K. Schreck
          </td>
          <td>2024-10-29</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: Targeted therapies (e.g., ibrutinib) have markedly improved chronic lymphocytic leukemia (CLL) management; however, ~20% of patients experience disease relapse, suggesting the inadequate depth and durability of these front-line strategies. Moreover, immunotherapeutic success in CLL has been stifled by its pro-tumor microenvironment milieu and low mutational burden, cultivating poor antigenicity and limited ability to generate anti-tumor immunity through adaptive immune cell engagement. Previously, we have demonstrated how a three-carbon-linker spirocyclic dimer (SpiD3) promotes futile activation of the unfolded protein response (UPR) in CLL cells through immense misfolded-protein mimicry, culminating in insurmountable ER stress and programmed CLL cell death. Method: Herein, we used flow cytometry and cell-based assays to capture the kinetics and magnitude of SpiD3-induced damage-associated molecular patterns (DAMPs) in CLL cell lines and primary samples. Result: SpiD3 treatment, in vitro and in vivo, demonstrated the capacity to propagate immunogenic cell death through emissions of classically immunogenic DAMPs (CALR, ATP, HMGB1) and establish a chemotactic gradient for bone marrow-derived dendritic cells. Conclusions: Thus, this study supports future investigation into the relationship between novel therapeutics, manners of cancer cell death, and their contributions to adaptive immune cell engagement as a means for improving anti-cancer therapy in CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6bb6049f43b4446b0fae7febaccea77debb8ea" target='_blank'>
              Immunogenic Cell Death Traits Emitted from Chronic Lymphocytic Leukemia Cells Following Treatment with a Novel Anti-Cancer Agent, SpiD3
              </a>
            </td>
          <td>
            Elizabeth Schmitz, Abigail Ridout, Audrey L. Smith, Alexandria P. Eiken, Sydney A. Skupa, Erin M. Drengler, Sarbjit Singh, Sandeep Rana, A. Natarajan, Dalia El-Gamal
          </td>
          <td>2024-12-16</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="NPM1-mutated AML is one of the largest entities in international classification systems of myeloid neoplasms, which are based on integrating morphologic and clinical data with genomic data. Previous research, however, indicates that bulk transcriptomics-based subtyping may improve prognostication and therapy guidance. Here, we characterized the heterogeneity in NPM1-mutated AML by performing single-cell RNA-sequencing and spectral flow cytometry on 16 AML belonging to three distinct subtypes previously identified by bulk transcriptomics. Using single-cell expression profiling we generated a comprehensive atlas of NPM1-mutated AML, collectively reconstituting complete myelopoiesis. The three NPM1-mutated transcriptional subtypes showed consistent differences in the proportions of myeloid cell clusters with distinct patterns in lineage commitment and maturational arrest. In all samples, malignant cells were detected across different myeloid cell clusters, indicating that NPM1-mutated AML are heavily skewed but not fully arrested in myelopoiesis. Same-sample multi-color spectral flow cytometry recapitulated these skewing patterns, indicating that the three NPM1-mutated subtypes can be consistently identified across platforms. Moreover, our analyses highlighted differences in the abundance of rare hematopoietic stem cells suggesting that skewing occurs early in myelopoiesis. To conclude, by harnessing single-cell RNA-sequencing and spectral flow cytometry, we provide a detailed description of three distinct and reproducible patterns in lineage skewing in NPM1-mutated AML that may have potential relevance for prognosis and treatment of patients with NPM1-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d2c7d3f716516ca564f7acecb6aa3ff7231897" target='_blank'>
              Resolving inter- and intra-patient heterogeneity in NPM1-mutated AML at single-cell resolution
              </a>
            </td>
          <td>
            E. O. Karakaslar, E. Argiro, N. E. Struckman, R. H. Shirali, J. Severens, W. Honders, S. L. Kloet, H. Veelken, M. Reinders, M. Griffioen, E. B. van den Akker
          </td>
          <td>2024-12-11</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spatial transcriptomics, proteomics, and epigenomics are innovative technologies which offer an unparalleled resolution and wealth of data in understanding and the interpretation of cellular functions and interactions. These techniques allow researchers to investigate gene and protein expressions at an individual cell level, revealing cellular heterogeneity within, for instance, bioengineered tissues and classifying novel and rare cell populations that could be essential for the function of the tissues and in disease processes. It is possible to analyze thousands of cells simultaneously, which gives thorough insights into the transcriptomic view of complex tissues. Spatial transcriptomics combines gene expressions with spatial information, conserving tissue architecture and making the mapping of gene activity across different tissue regions possible. Despite recent advancements in these technologies, they face certain limitations. Single-cell transcriptomics can suffer from technical noise and dropout events, leading to incomplete data. Its applicability has been limited by the complexity of data integration and interpretation, although better resolution and tools for the interpretation of data are developing fast. Spatial proteomics and spatial epigenomics provide data on the distribution of proteins and the gene regulatory aspects in tissues, respectively. The disadvantages of these approaches include rather costly and time-consuming analyses. Nevertheless, combining these techniques promises a more comprehensive understanding of cell function and tissue organization, which can be predicted to be useful in achieving better knowledge of cell guidance in tissue-engineered constructs and a higher quality of tissue technology products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b910cb889d8cf7dd3b82978f6e3916ced3b58d85" target='_blank'>
              Spatial Transcriptomics, Proteomics, and Epigenomics as Tools in Tissue Engineering and Regenerative Medicine
              </a>
            </td>
          <td>
            M. Lammi, C. Qu
          </td>
          <td>2024-12-06</td>
          <td>Bioengineering</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life.Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity.A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis.By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Brain invasion is a negative prognostic factor for meningiomas, but the molecular programs underlying meningioma brain invasion are poorly understood. To address this, we performed transcriptomic analyses of 3 independent datasets: Bulk RNA sequencing from 199 meningiomas (n=33 with brain invasion), single-cell RNA sequencing of 2 regionally distinct samples from the brain tumor interface (BTI) versus tumor core (TC) (n=22,118 single-cell transcriptomes), and spatial RNA sequencing from 6 samples with (n=4) versus without (n=2) brain invasion (n=18,471 spatial transcriptomes). Single-cell and spatial datasets were integrated with Harmony, cell interactions were defined using CellChat, and cross-platform comparisons were performed with the Seurat data integration pipeline. Bulk RNA sequencing revealed enrichment of 1835 genes in meningiomas with brain invasion compared to those without brain invasion, including glial markers (GFAP, NCAN) and markers of extracellular matrix remodeling (PDGFRA, COL9A1). Single-cell RNA sequencing comparison of BTI versus TC meningioma cells identified 2793 genes that were enriched in BTI cells, 268 of which were conserved across bulk and single-cell transcriptomes from brain-invasive meningiomas. Gene ontology enrichment analysis showed molecular programs underlying cell migration and regulation of neurogenesis at the BTI, while glycolysis and antigen presentation programs were enriched in the TC. Cholesterol, CSPG4, and LIF signaling were active in the TC, while SLIT signaling was enriched at the BTI. Spatial RNA sequencing showed enrichment of 439 genes at the BTI, and a conserved set of 12 genes were enriched in BTI meningioma cells across bulk, single-cell, and spatial datasets (TGM2, APOC1, IGFBP6, PLEKHO1, GFPT2, ZYX, IQGAP3, LUM, S100A11, ARPC1B, CDK1, CYCS). Comparison of tumor microenvironment monocytes at the BTI versus TC demonstrated enrichment of 1571 genes that were associated with macrophage migration and activation. These data elucidate molecular programs underlying meningioma brain invasion and provide insights into meningioma intratumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a3c94f72636972e43b4a85245e4a6b3da1acf5" target='_blank'>
              BIOM-22. SPATIAL, SINGLE-CELL, AND BULK RNA SEQUENCING IDENTIFY MOLECULAR PROGRAMS UNDERLYING MENINGIOMA BRAIN INVASION
              </a>
            </td>
          <td>
            Ayush Aggarwal, Mark W. Youngblood, K. Mirchia, Hinda Najem, Stephen T. Magill, Amy B. Heimberger, D. Raleigh
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) may originate from somatic mutations within the subventricular zone (SVZ). We investigated putative GBM origin-cells in the SVZ with somatic mutations implicated in GBM development. We hypothesized these origin-cells have unique properties compared to tumor-derived GBM tumorspheres (TSs). We isolated, characterized, and analyzed mouse transcriptomes of GBM origin-cells and GBM TSs, and sought to identify potential human GBM origin-cells within the SVZ. Using a mouse model with somatic mutations (Trp53/Pten/hEGFRvIII) induced within the SVZ, we demonstrated that mutated cells in the SVZ migrate to the cortex and form GBM. We isolated mouse GBM (mGBM) origin-cells from the SVZ and mGBM TSs from the tumors, comparing their characteristics, transcriptome, and tumorigenic potential upon orthotopic or SVZ injection. To identify potential human GBM origin-cells, single-nucleus RNA sequencing (snRNAseq) was performed on human GBM SVZ specimens. Our investigation revealed intriguing properties and distinct gene expression patterns in GBM origin-cells. mGBM origin-cells from the SVZ somatic mutation model demonstrated enhanced stemness and invasiveness compared to control SVZ cells. These cells exhibited site-specific tumorigenic capacity: cortical injection of mGBM origin-cells failed to induce tumor formation, while re-injection into the SVZ led to migration to the cortex and tumor development. RNA sequencing revealed significant gene expression differences between mGBM origin-cells and mGBM TSs despite similar mutation profiles, suggesting distinct identities. snRNAseq analysis of human GBM SVZ specimens (pathologically tumor-free) identified cells with chromosomal aberrations characteristic of early GBM development, including chromosome 7 gain / 10 loss. These findings suggest potential human GBM origin-cells within the SVZ. This study describes a distinct population of GBM origin-cells within the SVZ, different from tumor-derived cancer stem cells. These GBM origin-cells exhibit unique properties, including enhanced stemness and invasiveness, and display site-specific tumorigenic capacity, forming tumors only in the SVZ environment. Further research into the factors that cause GBM origin-cells in the SVZ to migrate and form tumors in the cortex is crucial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a544bba9a9968cc22ab71ada85d9eea9182155fd" target='_blank'>
              STEM-04. UNVEILING GLIOBLASTOMA ORIGIN-CELLS: ISOLATION AND CHARACTERIZATION FROM THE SUBVENTRICULAR ZONE
              </a>
            </td>
          <td>
            R. Choi, Se-Young Jo, Eunchae Yeo, Hyeong-Cheol Oh, E. Jo, Jin-Kyung Shim, Hyun Jung Kim, Joo Ho Lee, J. Yoo, Seonah Choi, J. Moon, E. Kim, Se Hoon Kim, Jong Hee Chang, Hyun Seok Kim, Jeong Ho Lee, Hoon Kim, Sangwoo Kim, Seok-Gu Kang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Glioblastoma progression involves changes in extracellular pH (pHe) related to rapid cell turnover and growth. There exist differences in pH between contrast-enhancing solid tumor and non-enhancing infiltrating tumor. SLC4A4 is a two-way bicarbonate transporter used in cortical astrocytes to buffer intracellular pH.



 In patients undergoing surgery for newly diagnosed IDH wild-type glioblastoma, we obtained tissue from solid tumor and infiltrating tumor. Single-nucleus RNA sequencing (n=15), immunohistochemistry (n=20) were carried out to identify SLC4A4 in infiltrating tumor beyond the contrast enhancing edge. SLC4A4 knock-down models were used to evaluate functional biology of this bicarbonate receptor using our tumor – cortical organoid model of infiltrating glioma and gliomaspheres. pH measurements were recorded using pH sensitive microelectrodes to evaluate changes on pH buffering ability of tumor cells during acid titration assays.



 Single-nucleus RNA sequencing (n=15) and spatial RNA sequencing (n=3) showed SLC4A4 to be the highest expressed transmembrane receptor on infiltrating glioblastoma cells but not solid tumor. IHC (n=20) confirmed higher protein expression in infiltrating tumor (p=0.0390). Infiltrating cells were specifically sensitive to decreasing pHe (p=0.0105). In the infiltrating glioma cortical-organoid model, the slc4a4- had increased proliferation (GFP+ cell quantitation; n=10, p=0.0410). There was also increased proliferation in wild-type cells treated with SLC4A4 inhibitor DIDS (n=10, p=0.0016). Immunostaining of these models showed that downmodulation of SLC4A4 was associated with increased Ki67 and decreased caspase staining.



 SLC4A4 plays a role in pH buffering in glioblastoma. We hypothesize that loss of SLC4A4 in solid tumor allows for desensitization of these cells to an acidic extracellular environment and allows for increased tumor growth and density while infiltrating tumor remains sensitivity to changes in pHe. An understand of pH regulation may lead to therapeutic efficacy through pH modulation of tumor.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcf843d835f0d7190f2e3152c4a4838b238993ae" target='_blank'>
              TMIC-24. BICARBONATE RECEPTOR SLC4A4 SENSITIZES INFILTRATING GLIOBLASTOMA CELLS TO CHANGES IN EXTRACELLULAR PH
              </a>
            </td>
          <td>
            Kunal S. Patel, Travis Perryman, Ashna Prabhu, Stuart Harper, Lindsey Dudley, Alondra Delgadillo, Aidan Gor, A. Bhaduri, H. Kornblum
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Plaque assays are typically quantified at an endpoint and fail to capture the temporal dynamics of viral infection and spread. In this study we introduce a high-throughput method utilising live-cell imaging and a recombinant vaccinia virus (VACV) engineered to express two fluorescent transgenes linked to distinct replication phases. This novel approach allows for real-time tracking of VACV plaques from their inception at a single infected cell through to multicellular expansion, enabling detailed kinetic analysis of plaque development. Using this analysis pipeline we categorise VACV plaque formation into three distinct phases: Establishment, Expansion, and Exhaustion, and quantitatively describe their dynamics. Our findings reveal significant variability in the initiation time of plaque spread (from single-cell to multicellular stages) and the growth rate of plaques, both between individual plaques and across different cell lines. This variability underscores the influence of host cellular factors on the kinetics of viral replication. Additionally, we reveal that the viral replication cycle dramatically accelerates through the early phases of plaque formation, which we hypothesise is driven by the changing multiplicity of cellular infection at the plaque edge. Finally, we demonstrate that differences in plaque sizes between two cell lines (BS-C-1 and HaCaT) can be largely attributed to variations in viral replication rate. This research reinforces the value of live-cell fluorescence microscopy in elucidating the complex spatiotemporal dynamics of viral infections, and contributes to a deeper understanding of the mechanisms driving viral spread and replication kinetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc442dcad7137d02b50f4e86000eabc50367aaa" target='_blank'>
              Large scale semi-automated quantification of viral spread reveals multi-phasic structure of plaque formation
              </a>
            </td>
          <td>
            L. M. Howell, T. Newsome
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="DNA methylation can give rise to robust biomarkers of aging, yet most studies profile it at the bulk tissue level, which masks cell type-specific alterations that may follow distinct aging trajectories. Long-read sequencing technology enables methylation profiling of extended DNA fragments, which allows mapping to their cell type of origin. In this study, we introduce a framework for evaluating cell type-specific aging using long-read sequencing data, without the need for cell sorting. Leveraging cell type-specific methylation patterns, we map long-read fragments to individual cell types and generate cell type-specific methylation profiles, which are used as input to a newly developed probabilistic aging model, LongReadAge, capable of predicting epigenetic age at the cell-type level. We apply LongReadAge to track aging of myeloid cells and lymphocytes from bulk leukocyte data as well as circulating cell-free DNA, demonstrating robust performance in predicting age despite limited shared features across samples. This approach provides a novel method for profiling the dynamics of epigenetic aging at cell-type resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5709038484acf3d57f1f45cf8c79fcec1017a895" target='_blank'>
              Profiling Epigenetic Aging at Cell-Type Resolution Through Long-Read Sequencing
              </a>
            </td>
          <td>
            A. Eames, M. Moqri, J. Poganik, V. N. Gladyshev
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [13, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>