<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-10 09:23:11 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Immune checkpoint inhibition (ICI) has fundamentally changed cancer treatment. However, only a minority of patients with metastatic triple negative breast cancer (TNBC) benefit from ICI, and the determinants of response remain largely unknown. To better understand the factors influencing patient outcome, we assembled a longitudinal cohort with tissue from multiple timepoints, including primary tumor, pre-treatment metastatic tumor, and on-treatment metastatic tumor from 117 patients treated with ICI (nivolumab) in the phase II TONIC trial. We used highly multiplexed imaging to quantify the subcellular localization of 37 proteins in each tumor. To extract meaningful information from the imaging data, we developed SpaceCat, a computational pipeline that quantifies features from imaging data such as cell density, cell diversity, spatial structure, and functional marker expression. We applied SpaceCat to 678 images from 294 tumors, generating more than 800 distinct features per tumor. Spatial features were more predictive of patient outcome, including features like the degree of mixing between cancer and immune cells, the diversity of the neighboring immune cells surrounding cancer cells, and the degree of T cell infiltration at the tumor border. Non-spatial features, including the ratio between T cell subsets and cancer cells and PD-L1 levels on myeloid cells, were also associated with patient outcome. Surprisingly, we did not identify robust predictors of response in the primary tumors. In contrast, the metastatic tumors had numerous features which predicted response. Some of these features, such as the cellular diversity at the tumor border, were shared across timepoints, but many of the features, such as T cell infiltration at the tumor border, were predictive of response at only a single timepoint. We trained multivariate models on all of the features in the dataset, finding that we could accurately predict patient outcome from the pre-treatment metastatic tumors, with improved performance using the on-treatment tumors. We validated our findings in matched bulk RNA-seq data, finding the most informative features from the on-treatment samples. Our study highlights the importance of profiling sequential tumor biopsies to understand the evolution of the tumor microenvironment, elucidating the temporal and spatial dynamics underlying patient responses and underscoring the need for further research on the prognostic role of metastatic tissue and its utility in stratifying patients for ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1bdaa11baefcf7560def6afb49fb7f0020d09d" target='_blank'>
              Temporal and spatial composition of the tumor microenvironment predicts response to immune checkpoint inhibition
              </a>
            </td>
          <td>
            N. Greenwald, I. Nederlof, Cameron Sowers, Daisy Yi Ding, Seongyeol Park, Alex Kong, Kathleen E. Houlahan, Sricharan Reddy Varra, Manon de Graaf, Veerle Geurts, Candace C. Liu, Jolene S Ranek, L. Voorwerk, M. de Maaker, Adam Kagel, E. Mccaffrey, Aziz Khan, C. Yeh, C. Fullaway, Zumana Khair, Yunhao Bai, Hadeesha Piyadasa, Tyler Risom, A. Delmastro, Felix J. Hartmann, Lise Mangiante, Cristina Sotomayor-Vivas, Ton N. Schumacher, Zhicheng Ma, M. Bosse, Marc J van de Vijver, R. Tibshirani, Hugo M. Horlings, Christina Curtis, Marleen Kok, Michael Angelo
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>157</td>
        </tr>

        <tr id="Small bowel adenocarcinoma (SBA) is a rare malignancy marked with a poor prognosis. The cellular and proteomic heterogeneity within the tumor immune microenvironment (TIME) of SBA is a likely driver of prognosis, disease progression and response to therapy. We have addressed a major gap in knowledge of the TIME in SBA using highly multiplexed, protein imaging of the SBA tumor-immune ecosystem generating a comprehensive, single-cell level, spatial, proteomic atlas of TIME in > 600,000 cells from 136 tumor and matched normal samples from clinically and genomically annotated SBA patients (N=37). The SBA TIME Atlas informs on spatial distribution and interactions of tumor-intrinsic processes, diverse immune cell types, immune checkpoints, and vascularization. Enrichment of proliferating epithelial tissue, stem cells, and likely pro-tumor immune signatures in the tumor niche is contrasted by the representation of naïve and early-effector T-cells in the epithelial compartments of adjacent normal tissues. Epithelial-stem-immune cell spatial architectures within tumor and matched-normal niches were strongly associated with patient survival, suggesting malignancy is driven by spatial architecture beyond the tumor microenvironment. The blueprints of therapeutically actionable immune checkpoints at the interface between epithelial and microenvironmental T-cells as well as macrophages have established a guideline for precision immunotherapies tailored to the TIME composition in SBA. We expect that this SBA atlas will contribute to a deeper understanding of the immune contexture in this rare disease as well as other gastrointestinal cancers and help guide future precision immune-oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2886703d3c5dcc191767cc197fd5d1a7d169072" target='_blank'>
              A spatially resolved single cell proteomic atlas of Small Bowel Adenocarcinoma
              </a>
            </td>
          <td>
            Z. Dereli, B. Bozorgui, M. Sanchez, Nicholas Hornstein, Guillaume Thibault, Huamin Wang, Gordon B. Mills, John N. Weinstein, M. Overman, Anil Korkut
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions driving progression and pathobiology. How these spatial patterns and interactions evolve with treatment remains unclear. Here, we apply imaging mass cytometry to analyse protein-level changes in paired pre- and post-treatment GBM samples from five patients. We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes. These findings provide insight into treatment-induced shifts in GBM’s cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets. Key points Spatial organisation in primary GBM consist of layers driven by the presence of hypoxia The layers in recurrent GBM appear are driven more by the presence of reactive astrocytes Increased cellular cross-talk in recurrent GBM presents novel therapeutic targets">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3061625679fbc2f861a2180efc0a648894c17487" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment
              </a>
            </td>
          <td>
            Shoaib Ajaib, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F. Stead
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell spatial transcriptomics reveals how cells organize in healthy and diseased tissues. From these data, tissue segmentation analysis defines discrete compartments that organize cells into functional multicellular units. Existing methods for automated tissue segmentation rely on spatial smoothing to define spatially coherent regions but often blur the boundaries between adjacent tissue compartments. We describe Tessera, an algorithm that approaches tissue segmentation through a novel approach, dividing the tissue into small multicellular tiles whose edges track with natural tissue boundaries. Tessera achieves this by incorporating successful tools from edge-preserving smoothing, topological data analysis, and morphology-aware agglomerative spatial clustering. We show that Tessera identifies a range of known anatomical structures, in healthy mouse brain and human lymph nodes, and novel disease-associated niches, in human brain and in lung cancer. Tessera is a general-purpose tool that returns spatially coherent spatial structures with accurate boundaries across a range of spatial transcriptomics and proteomics technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfead60198d5f27b8c18dbe0ea5203b263942c83" target='_blank'>
              Accurate tiling of spatial single-cell data with Tessera
              </a>
            </td>
          <td>
            Daniel J. Stein, Miles Tran, I. Korsunsky
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="To date, few studies have detailed the tumor microenvironment (TME) of Ewing sarcoma (EwS). The TME has a vital role in cancer survival and progression with implications in drug resistance and immune escape. By performing spatially resolved transcriptomic analysis of primary treatment-naïve EwS samples, we discovered greater stromal enrichment in localized EwS tumors compared to metastatic EwS tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumorigenic role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling, suggesting a potential immune-evasive mechanism and immunotherapy target. Our findings provide insight into tumor cell/stromal cell interactions in EwS and serve as a valuable resource for further investigations in the tumor immune microenvironment of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245c7e47704099e340bfbb4047050883a6a92e6d" target='_blank'>
              Tumor-associated stroma shapes the spatial tumor immune microenvironment of primary Ewing sarcomas
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer characterized by profound desmoplasia and cellular heterogeneity, which cannot be fully resolved using traditional bulk sequencing approaches. To understand the contribution of this heterogeneity to PDAC biology, we analyzed a large cohort of primary human PDAC samples (n = 62), profiling 443,451 single cells and 53,236 spatial transcriptomic spots using a combined single-cell RNA sequencing and spatial transcriptomics approach. Our analysis revealed significant intratumoral heterogeneity, with multiple genetically distinct neoplastic clones co-existing within individual tumors. These clones exhibited diverse transcriptional states and subtype profiles, challenging the traditional binary classification of PDAC into basal and classical subtypes; instead, our findings support a transcriptional continuum influenced by clonal evolution and spatial organization. Additionally, these clones each interacted uniquely with surrounding cell types in the tumor microenvironment. Phylogenetic analysis uncovered a rare but consistent classical-to-basal clonal transition associated with MYC amplification and immune response depletion, which were validated experimentally, suggesting a mechanism driving the emergence of a more aggressive basal clonal phenotype. Spatial analyses further revealed dispersed clones enriched for epithelial-to-mesenchymal transition (EMT) activity and immune suppression, correlating with metastatic potential and colonization of lymph node niches. These dispersed clones tended to transition toward a basal phenotype, contributing to disease progression. Our findings highlight the critical role of clonal diversity, transcriptional plasticity, and TME interactions in shaping human PDAC biology. This work provides new insights into the molecular and spatial heterogeneity of PDAC and offers potential avenues for therapeutic intervention targeting clonal evolution and the mechanisms driving metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Spatial profiling of tissues promises to elucidate tumor-microenvironment interactions and generate prognostic and predictive biomarkers. We analyzed single-cell, spatial data from three multiplex imaging technologies: cyclic immunofluorescence (CycIF) data we generated from 102 breast cancer patients with clinical follow-up, and publicly available imaging mass cytometry and multiplex ion-beam imaging datasets. Similar single-cell phenotyping results across imaging platforms enabled combined analysis of epithelial phenotypes to delineate prognostic subtypes among estrogen-receptor positive (ER+) patients. We utilized discovery and validation cohorts to identify biomarkers with prognostic value. Increased lymphocyte infiltration was independently associated with longer survival in triple-negative (TN) and high-proliferation ER+ breast tumors. An assessment of ten spatial analysis methods revealed robust spatial biomarkers. In ER+ disease, quiescent stromal cells close to tumor were abundant in good prognosis tumors, while tumor cell neighborhoods containing mixed fibroblast phenotypes were enriched in poor prognosis tumors. In TN disease, macrophage/tumor and B/T lymphocyte neighbors were enriched and lymphocytes were dispersed in good prognosis tumors, while tumor cell neighborhoods containing vimentin-positive fibroblasts were enriched in poor prognosis tumors. In conclusion, we generated comparable single-cell spatial proteomic data from several clinical cohorts to enable prognostic spatial biomarker identification and validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e86a33275ed2b41ff7e598b0d0e6585ea48b22" target='_blank'>
              Highly multiplexed imaging reveals prognostic immune and stromal spatial biomarkers in breast cancer.
              </a>
            </td>
          <td>
            Jennifer Eng, Elmar Bucher, Zhi Hu, Cameron R Walker, Tyler Risom, Michael Angelo, Paula I. Gonzalez-Ericsson, M. E. Sanders, A. B. Chakravarthy, Jennifer A Pietenpol, Summer L. Gibbs, Rosalie C. Sears, Koei Chin
          </td>
          <td>2025-01-14</td>
          <td>JCI insight</td>
          <td>2</td>
          <td>20</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), accounting for 90% of pancreatic neoplasms, is characterized by its poor prognosis, with a 5-year survival rate of only 12%. Most patients are diagnosed with metastatic or locally advanced disease, leaving only 15% eligible for curative resection. PDAC exhibits resistance to chemotherapy, targeted therapies, and immunotherapy, largely due to its highly heterogeneous tumor microenvironment (TME). In this study, we performed an integrative analysis of publicly available scRNA, spatial transcriptomics, and bulk RNA sequencing datasets to investigate the influence of TME composition and tumor architecture on PDAC progression, treatment response, and clinical outcomes. We identified TME subtypes with distinct cellular compositions, functional signatures, and immunomodulatory cell-cell interactions. Spatially distinct cellular niches and gene modules revealed heterogeneity across primary tumors and metastatic lesions. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications. These findings underscore the importance of integrating multi-omics approaches to unravel the complexity of the PDAC TME and highlight its potential to inform therapeutic strategies and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768379a4460d13ebf34f45a5af50c7d930ab8e2" target='_blank'>
              Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel F. Pozo de Mattos P., Marvin Paulo Lins, Julia Fontoura, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Osteosarcoma is the most common malignant bone tumor in young patients and remains a significant clinical challenge, particularly in the context of metastatic disease. Despite extensive documentation of genomic alterations in osteosarcoma, studies detailing the immunosuppressive mechanisms within the metastatic osteosarcoma microenvironment are lacking. Our objective was to characterize the spatial transcriptional landscape of metastatic osteosarcoma to reveal these immunosuppressive mechanisms and identify promising therapeutic targets. Here, we performed spatial transcriptional profiling on a cohort of osteosarcoma pulmonary metastases from pediatric patients. We reveal a conserved spatial gene expression pattern resembling a foreign body granuloma, characterized by peripheral inflammatory signaling, fibrocollagenous encapsulation, lymphocyte exclusion, and peritumoral macrophage accumulation. We also show that the intratumoral microenvironment of these lesions lack inflammatory signaling. Additionally, we identified CXCR4 as an actionable immunomodulatory target that bridges both the intratumoral and extratumoral microenvironments and highlights the spatial heterogeneity and complexity of this pathway. Collectively, this study reveals that metastatic osteosarcoma specimens are comprised of multiple regionally distinct immunosuppressive microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2ee7e019ae5fcae794fe1bb25f818cd218fc98" target='_blank'>
              Spatial profiling identifies regionally distinct microenvironments and targetable immunosuppressive mechanisms in pediatric osteosarcoma pulmonary metastases
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice Pereira, Douglass W Morris-II, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) features a complex tumor microenvironment (TME) that significantly influences patient outcomes. Understanding the TME’s cellular composition and interactions is crucial for identifying therapeutic targets to improve treatment. Methods We performed an integrative analysis combining 88 single-cell RNA sequencing (scRNA-seq) samples with 187,520 cells, 20 Visium spatial transcriptomics (ST-seq) samples with 67,933 spots, and 1383 bulk RNA-seq samples and 2 Xenium high-resolution ST-seq samples with 307,679 cells, to delineate and characterize distinct subpopulations of fibroblasts, macrophages, T/NK cells, and B/plasma cells within the PDAC microenvironment. Correlations among major cell types across 12 PDAC datasets were assessed through gene set variation analysis (GSVA). Kaplan-Meier survival analysis was used to evaluate the prognostic significance of cell subtypes. Furthermore, multiplex immunohistochemistry (mIHC) and the Xenium platform was utilized to validate cellular interactions in human PDAC tissues at both protein and RNA levels. Results Six fibroblast subtypes and eight macrophage/monocyte subtypes were identified. POSTN high fibroblasts and SPP1 high macrophages highly infiltrated tumor tissues and were associated with poor prognosis. Most immune cell subtypes mediate adverse prognoses, except for CCL4 high CD8+ T EFF cells and IGHG1 high plasma cells, which are linked to favorable outcomes. ST-seq revealed spatial colocalization of POSTN high Fibro and SPP1 high Macro cells, as well as CCL4 high CD8+ T EFF and IGHG1 high plasma cells. These findings were corroborated by mIHC and validated using Xenium spatial transcriptomics with single-cell resolution, which confirmed the expression and spatial proximity of these markers at both the gene and protein levels. Conclusion This integrative analysis of the PDAC TME underscores the prognostic importance of POSTN high fibroblasts and SPP1 high macrophages, while also highlighting the protective roles of CCL4 high CD8+ T EFF cells and IGHG1 high plasma cells. These insights could improve PDAC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74fb81e29120e8309bf4f3e49e31706525768da7" target='_blank'>
              Integrating multi-modal transcriptomics identifies cellular subtypes with distinct roles in PDAC progression
              </a>
            </td>
          <td>
            Jun Wu, Tenghui Dai, Ziyue Li, Meng Pan, Wei Zhang, Hao Chen, Guansheng Zheng, Li Qiao, Qizhou Lian, Yang Liu, Jierong Chen
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A key challenge in cancer research is to identify the secreted factors that contribute to tumor cell survival. Nowhere is this more evident than in Hodgkin lymphoma, where malignant Hodgkin Reed Sternberg (HRS) cells comprise only 1-5% of the tumor mass, the remainder being infiltrating immune cells that presumably are required for the survival of the HRS cells. Until now, there has been no way to characterize the complex Hodgkin lymphoma tumor microenvironment at genome scale. Here, we performed genome-wide transcriptional profiling with spatial and single-cell resolution. We show that the neighborhood surrounding HRS cells forms a distinct niche involving 31 immune and stromal cell types and is enriched in CD4+ T cells, myeloid and follicular dendritic cells, while being depleted of plasma cells. Moreover, we used machine learning to nominate ligand-receptor pairs enriched in the HRS cell niche. Specifically, we identified IL13 as a candidate survival factor. In support of this hypothesis, recombinant IL13 augmented the proliferation of HRS cells in vitro. In addition, genome-wide CRISPR/Cas9 loss-of-function studies across more than 1,000 human cancer cell lines showed that IL4R and IL13RA1, the heterodimeric partners that constitute the IL13 receptor, were uniquely required for the survival of HRS cells. Moreover, monoclonal antibodies targeting either IL4R or IL13R phenocopied the genetic loss of function studies. IL13-targeting antibodies are already FDA-approved for atopic dermatitis, suggesting that clinical trials testing such agents should be explored in patients with Hodgkin lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0453e5a36d23742e3f6af668adec8ef2f3f547e8" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, Daniel Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail A Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson Weir, Orr Ashenberg, Geraldine Pinkus, Scott J Rodig, Caroline Uhler, Evan Z. Macosko, Margaret M. Shipp, A. Louissaint, Fei Chen, Todd R. Golub
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Understanding cell-cell communication and its dependence on spatial organization is critical for unraveling tissue complexity and organ function. This study integrates single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics (ST) to systematically assess how spatial niches influence gene expression and intercellular communication. Using breast cancer, brain cortex, and heart datasets, our analyses reveal limited global transcriptional changes in cells depending on their spatial microenvironment, with differential gene expression observed in less than half the samples explored. Moreover, cell-cell communication predictions, derived from ligand-receptor pairs, exhibit minimal correlation with spatial colocalization of cell types. Overall, our study underscores the limitations of using scRNA-seq data to capture niche-specific molecular interactions, even when spatial information is leveraged, and it highlights the need for novel strategies to refine our understanding of intercellular communication dynamics at molecular level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790c5632749fda7ad54ef1434d0ac14122b25196" target='_blank'>
              Systematic assessment of microenvironment-dependent transcriptional patterns and intercellular communication
              </a>
            </td>
          <td>
            Elena Pareja-Lorente, Patrick Aloy
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Malignant gliomas follow two distinct natural histories: de novo high grade tumors such as glioblastoma, or lower grade tumors with a propensity to transform into high grade disease. Despite differences in tumor genotype, both entities converge on a common histologically aggressive phenotype, and the basis for this progression is unknown. Glioma associated macrophages (GAM) have been implicated in this process, however GAMs are ontologically and transcriptionally diverse, rendering isolation of pathogenic subpopulations challenging. Since macrophage contextual gene programs are orchestrated by transcription factors acting on cis-acting promoters and enhancers in gene regulatory networks (GRN), we hypothesized that functional populations of GAMs can be resolved through GRN inference. Here we show via parallel single cell RNA and ATAC sequencing that a subpopulation of human GAMs can be defined by a GRN centered around the Activator Protein-1 transcription factor FOSL2 preferentially enriched in high grade tumors. Using this GRN we nominate ANXA1 and HMOX1 as surrogate cell surface markers for activation, thus permitting prospective isolation and functional validation in human GAMs. These cells, termed malignancy associated GAMs (mGAMs) are pro-invasive, pro-angiogenic, pro-proliferative, possess intact antigen presentation but skew T-cells towards a CD4+FOXP3+ phenotype under hypoxia. Ontologically, mGAMs share somatic mitochondrial mutations with peripheral blood monocytes, and their presence correlates with high grade disease irrespective of underlying tumor mutation status. Furthermore, spatio-temporally mGAMs occupy distinct metabolic niches; mGAMs directly induce proliferation and mesenchymal transition of low grade glioma cells and accelerate tumor growth in vivo upon co-culture. Finally mGAMs are preferentially enriched in patients with newly transformed regions in human gliomas, supporting the view that mGAMs play a pivotal role in glioma progression and may represent a plausible therapeutic target in human high-grade glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/190acfc8ea743ab5360d22a148d803b01bb31afc" target='_blank'>
              A pathogenic subpopulation of human glioma associated macrophages linked to glioma progression
              </a>
            </td>
          <td>
            Kenny Kwok Hei Yu, Z. Abou-Mrad, Kristof Törkenczy, Isabell Schulze, Jennifer Gantchev, Gerard Baquer, Kelsey Hopland, Evan D. Bander, Umberto Tosi, Cameron Brennan, Nelson S. Moss, Pierre-Jacques Hamard, R. Koche, Caleb Lareau, N. Agar, T. Merghoub, Viviane Tabar
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Lung cancer remains the deadliest cancer in the United States, with lung adenocarcinoma (LUAD) as its most prevalent subtype. While computed tomography (CT)-based screening has improved early detection and enabled curative surgeries, the molecular and cellular dynamics driving early-stage LUAD progression remain poorly understood, limiting non-surgical treatment options. To address this gap, we profiled 2.24 million cells from 122 early-stage LUAD patients using multiplexed imaging mass cytometry (IMC). This analysis revealed the molecular, spatial, and temporal dynamics of LUAD development. Our findings uncover a binary progression model. LUAD advances through either inflammation, driven by a balance of cytotoxic and regulatory immune activity, or fibrosis, characterized by stromal activation. Surprisingly, tumor cell populations did not increase significantly. Instead, they displayed a mixed phenotypic profile consistent with epithelial-to-mesenchymal transition (EMT), effectively masking the expansion of malignant cells. Furthermore, we addressed discrepancies between CT-based and histology-based subtyping. CT scans, while non-invasive, often mischaracterize invasive fibrotic tumors—which account for 20.5% of LUAD cases—as mild, non-solid ground glass opacities (GGOs). Using high-content IMC imaging, we demonstrate that these tumors harbor significant risks and advocate for improved diagnostic strategies. These strategies should integrate molecular profiling to refine patient stratification and therapeutic decision-making. Altogether, our study provides a high-resolution, systems-level view of the tumor microenvironment in early-stage LUAD. We characterize key transitions in oncogenesis and propose a precision-driven framework to enhance the detection and management of aggressive disease subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33061967be0c7904ed5dea4bcec63375c368332f" target='_blank'>
              Simultaneous immunomodulation and epithelial-to-mesenchymal transition drives lung adenocarcinoma progression
              </a>
            </td>
          <td>
            Junbum Kim, H. Ravichandran, L. Yoffe, B. Bhinder, Kyle Finos, Arshdeep Singh, Bradley B. Pua, Stewart Bates, B. E. Huang, A. Rendeiro, Vivek Mittal, Nasser K. Altorki, T. McGraw, O. Elemento
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The intra- and inter-tumoral heterogeneity of glioblastoma represents a significant therapeutic challenge, as well as difficulty in generating reliable models for in vitro studies. Historical 2D adherent cell lines do not recapitulate this complexity, whereas both patient-derived neurospheres (PDN) and organoids (PDO) demonstrate intra-tumoral heterogeneity. Here, we quantify the tumor cell composition from matched models established from the same primary tumor using a series of multi-omic interrogations. We find that both patient-derived models recapitulate the genomic, epigenomic and tumor cell heterogeneity of the primary tissue. Furthermore, single-nuclei RNA sequencing revealed a subset of organoids containing small numbers of non-malignant cells from neuron and immune cell lineages. Harnessing the intra-tumoral heterogeneity of PDN models, we reveal the impact of temozolomide chemotherapy on individual cell states, altering composition of tumors over time in response to treatment. Our data confirms that both patient-derived models recapitulate patient intra-tumoral heterogeneity providing a platform for tumor cell state refined therapeutic studies. Key Points Generation of matched patient-derived neurosphere and organoid models from resected GBM tissue Neurosphere models exhibit greater proliferative signatures Both patient-derived models recapitulate genomic and epigenomic features of the primary tissue Single-nuclei RNA sequencing reveals both models recapitulate intra-tumoral heterogeneity of the primary tissue Neurosphere models enable interrogation of therapeutic responses in the context of heterogeneity Importance of study Patient derived models can be powerful tools when they faithfully recapitulate the tumor tissue from which they are derived. In glioblastoma, patient derived neurospheres (PDN) and organoids (PDO) have both been used in studies, however the differences between these models and the recapitulation of patient heterogeneity remain to be fully characterized. To address this, we performed multi-omic profiling of PDN and PDO models generated from the same tumor tissue. We find that that across a range of data modalities, both model systems exhibit a high level of resemblance to tissue, and critically, maintain heterogeneity and tumor cell composition. The importance of modeling heterogeneity was demonstrated in PDN models, where temozolomide treatment specifically alters the abundance of MES-like and AC-like tumor cells. Our findings demonstrate that neurosphere and organoid models effectively preserve cellular heterogeneity, genomic alterations, methylation signatures and transcriptomic features, both highly suitable to model glioblastoma’s complex cellular landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c3caa2d109062b5ff4f7f9f68c99c0d6be111c" target='_blank'>
              Three-dimensional patient-derived models of glioblastoma retain intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Zachery Moore, Claire Storey, Daniel V. Brown, A. Valkovic, Montana Spiteri, Jasmine F. Pignatelli, Shannon J Oliver, Alana Fakhri, Kate Drummond, Seth Malinowski, Keith L. Ligon, Oliver M. Sieber, J. Whittle, S. Freytag, Sarah A. Best
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors (TKIs) provide therapeutic benefit in EGFR mutation–positive non–small cell lung cancer, but some individuals develop early resistance. We performed spatial transcriptomics analysis of pre- and posttreatment tumor samples from the same patients to explore the underlying mechanisms of such early resistance. The proportion and activation of fibroblasts increased in association with the development of early resistance, whereas a distinct tumor cell cluster showed activation of tumor necrosis factor–α signaling via the NF-κB pathway even before treatment. Also in the early resistance sample, specific tumor cell clusters interacted with immune and stromal cells. Immature tertiary lymphoid structures (TLSs) were enriched in the early resistance sample, whereas mature TLSs were observed in the long-term response sample. These findings implicate tumor heterogeneity and an inflammatory tumor microenvironment in early EGFR-TKI resistance, providing insight into potential therapeutic strategies to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59cd540165dee7cba115de4f78c74d67fa9e26e" target='_blank'>
              Spatial dynamics of the tumor microenvironment associated with emerging resistance to targeted therapy in EGFR-mutated non–small cell lung cancer
              </a>
            </td>
          <td>
            Satoshi Nakamura, Daisuke Shibahara, Kentaro Tanaka, Yasuyuki Kishikawa, Mikiko Hashisako, K. Nakatomi, Noriaki Nakagaki, Mikihiro Kohno, Koichi Azuma, R. Ibusuki, K. Otsubo, Y. Yoneshima, E. Iwama, Yoshinao Oda, I. Okamoto
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 There is a growing appreciation for the role that the spatial organization of immune cells within the tumor microenvironment (TME) plays in shaping cellular function and response to therapy. However, the majority of studies have been limited to analyzing immune responses at a single time point, and so, little is known about the spatiotemporal changes that underlie successful tumor rejection after cancer immunotherapy. In this study, we used the CODEX multiplex imaging system with a 32-plex antibody panel to longitudinally characterize the spatial changes in the lymphoid and myeloid cell populations that were previously identified by scRNA-Seq and CyTOF to be associated with complete tumor rejection in mice treated with the combination of α-PD-1 and α-CTLA-4 (combo ICT) in our well-characterized mouse T3 MCA sarcoma model. T3 tumor-bearing mice were treated with control antibody or combo ICT, tumors were harvested at multiple time points (day 7, 9, 10, 11 and 13), fresh frozen, sectioned, and subjected to CODEX multiplex imaging. A total of 42 tumors were imaged with the full range of the tumor-immune boundary. A hierarchical cell clustering approach developed in-house was used to profile 13,051,156 cells resulting in the identification of 13 distinct cell types. Our results show that after ICT the decrease in T3 cells in the tumor tissue starts on day 10 and culminates in the complete elimination of tumor cells by day 13. This immune-mediated response is driven by an increase in CD4+ T cells, which allows them to more frequently co-localize with CD8+ T cells and type 1 conventional dendritic cells (cDC1), forming three-cell clusters with increased cell contacts. This change in cell-to-cell interactions is followed by a broader reorganization of the TME as noted by the expansion of a lymphoid-rich cellular neighborhood (CN) that harbors the majority of CD4+ and CD8+ T cells. Interestingly, this CN becomes a hub for T cell effector functions, characterized by an enrichment of GZM-B+ and KI67+ T cells, and exhibits spatial dynamics throughout tumor regression. While both GZM-B+ and KI67+ T cells are observed in proximity to blood vessels on days 7 and 9, they begin to spatially separate into distinct regions after day 10, indicating that cytotoxic T cells move towards areas with tumor cells, while proliferating T cells are more frequently observed at the periphery. This dynamic behavior was further confirmed by assessing the communication rules between CNs, which showed that the lymphoid-rich CN acts by eliminating layers of tumor cells at the tumor boundary and by modulating nearby myeloid populations. Additionally, integration of CODEX dataset with scRNA-seq revealed CD4+ T cells produce Interferon-gamma and a change in the molecular chemotactic pathways orchestrating lymphoid and myeloid communication. Overall, this study provides a detailed spatial view of the cellular events that result in successful ICT and allows us to propose a computational model of the actions of T cells in coordinating the antitumor response after α-CTLA-4/α-PD1.
 Citation Format: Ruan FV Medrano, Vladimir Sukhov, Maxim Artyomov, Robert Schreiber. Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B048.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b6d1c2313c8e86b7cbcc29c94f8060fb9e7af4" target='_blank'>
              Abstract B048: Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy
              </a>
            </td>
          <td>
            Ruan FV Medrano, Vladimir Sukhov, Maxim N Artyomov, Robert Schreiber
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background & Aims Epithelial tumors generally resemble the cellular architecture of their tissue of origin. However, this link remains largely unexplored in the pancreas. Methods Using Nanostring GeoMx DSP®, Resolve Molecular Cartography® and Nanostring CosMx®, and integration with single cell RNAseq datasets, we mapped the human pancreatic ductal epithelium in non-neoplastic pancreas, and compared it to pancreatic cancer subtypes. Results Groups of Keratin-5+ cells among the Pan-Cytokeratin+ cells in the duct have a gene signature reminiscent of stem cells and (supra)basal cells from other tissues. In spatial analysis at single cell resolution, the pancreatic duct manifests as a stratified epithelium comprising a basal and four luminal populations; In large ducts, KRT5+ basal (BAS) cells express ΔNp63 while KRT5+ luminal (LUM)-B cells reside supra-basally and are distinct from the common KRT5- LUM-A cells. LUM-C and LUM-D cells pertain to intercalated ducts and ductal glands, respectively. LUM-A and -C cells express gel-forming mucins while LUM-B cells have membrane-bound MUC4 and MUC16. In cancer, BAS and LUM-B signatures associate with the basal-like pancreatic ductal adenocarcinoma (PDAC) and correlate with lower survival but exhibit a mixed spatial pattern with a diffracted gene signature. In contrast, adenosquamous cancers of the pancreas (ASCP) preserve the normal spatially unmixed identity of LUM-B cells and BAS cells that is regulated by ΔNp63. Next to ΔNp63, conserved drug targets were identified for both populations. Conclusion This study offers a refined pancreatic tumor classification based on the native ductal architecture, with better preservation of the LUM-B and BAS cell population identity in ASCP than in PDAC, including the conserved drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a2f6f8e9909be212c547b03063b014e38a8855" target='_blank'>
              Spatial map of native duct cell populations in human pancreas and their representation in pancreatic cancers
              </a>
            </td>
          <td>
            JL Van den Bossche, M. Van der Vliet, E. Michiels, O. A. Senar, Z. Madran, K. Coolens, S. Van Lint, M. Nacher, T. Arsenijevic, N. Messaoudi, P. Lefesvre, E. Montanya, M. Rovira, J. Van Laethem, J. Baldan, I. Houbracken, I. Rooman
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Melanoma, a highly metastatic skin cancer, exhibits variations in prognosis and response to therapy based on the site of metastasis. Despite the success of immunotherapy and targeted therapies in melanoma, over half of metastatic melanoma patients will experience disease progression due to therapy resistance. The heterogeneity and plasticity of melanoma cells contribute to metastatic dissemination and therapy resistance. We aim to determine whether distinct clones and/or their transcriptional cell states can predict the formation of tumours in various organs. Lineage tracing and clonal heterogeneity were evaluated using the state-of-the-art technology, SPLINTR (Single-cell Profiling and LINeage Tracing), enabling us to match a cells evolution with changes in transcriptional states. To identify melanoma clones across different metastatic sites, specifically subcutaneous, lungs and brain, C57BL/6 mice were injected subcutaneously, intravenously or intracranially with the same pool of cells of barcoded luciferase expressing YUMMER1.7PV1 murine melanoma cells. Transduction conditions ensured that one DNA barcode integrated into cell genomes at one barcode per cell, serving as a lineage tag. Bioluminescence imaging was performed once a week to monitor the tumour growth of mice injected intravenously and intracranially. Subcutaneous tumours were measured by calliper. Mice were euthanized before ethical endpoints. Tumors were harvested and single cell analysis as well as DNA sequencing was performed to identify barcodes expressed by the tumor cells across different tumor-sites. Barcode analysis was conducted both before cell implantation and at tumor harvest. Clonality exhibited significant variability between sites. Single-cell RNA sequencing will unveil states associated with dominant clones present at different metastatic sites. This information will provide a new understanding of melanoma transcriptional phenotype behaviours and biological characteristics at different metastatic sites and in response to therapy.
 Citation Format: Veronica Aedo-Lopez. Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A120.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colonic stem cells have a key role in the continuous regeneration of a healthy intestinal epithelium. Despite considerable advances in single-cell omics technologies, the transcriptional heterogeneity of rare cell types such as colonic stem cells, as well as their functional states and niche-specific behaviors, remain poorly characterised. In this study, we leverage a meta-analysis of scRNA-seq and spatial transcriptomic datasets to comprehensively map the heterogeneity of colonic stem cells. We identify multiple, previously underappreciated stem cell states, including distinct quiescent subtypes marked by CDKN1A (P21), CDKN1B (P27), and CDKN1C (P57), proliferative populations defined by MKI67 (Ki67) and LRIG1, and a lineage-committed intermediate subpopulation expressing MUC2. Strikingly, we find that these states can be robustly identified solely by their extracellular matrix (ECM) gene expression signatures, revealing ECM composition as a critical determinant of stem cell identity. Notably, LAMA1 expression is highly specific to the P57+ quiescent population, linking laminin-mediated microenvironments to the active maintenance of deep quiescence, consistent with our recent findings associating LAMA1 with quiescent cell survival. By applying these ECM gene signatures, we delineate discrete “micro-niches” of quiescent stem cells in healthy tissue and provide evidence that analogous states persist in the colorectal cancer (CRC) environment. Extending our approach to an unrelated tissue, the pancreas, we detect parallel quiescent cell subtypes, illustrating the broader applicability of ECM-based signatures. Taken together, our findings redefine the concept of stem cell heterogeneity in the colon, establish ECM-driven gene signatures as a powerful tool for characterizing stem cell states, and offer new perspectives on the niche-dependent regulation of both healthy and cancerous stem cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b79223cf63c16f2be2c9879ace337f25901bae" target='_blank'>
              ECM Signatures Reveal Quiescent Stem Cell Diversity in the Colonic Niche
              </a>
            </td>
          <td>
            Séamus E. Hickey, M. Andreatta, Christina Enright, Emmanuel Boucrot, Patrick Kiely, Siobhán B. Cashman, Saintiago J. Carmona, Kieran McGourty
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61c2f79171ce9194bffe681aecda7bf66be2273" target='_blank'>
              STModule: identifying tissue modules to uncover spatial components and characteristics of transcriptomic landscapes
              </a>
            </td>
          <td>
            Ran Wang, Yan Qian, Xiaojing Guo, Fangda Song, Zhiqiang Xiong, Shirong Cai, Xiuwu Bian, Man-Hon Wong, Qin Cao, Lixin Cheng, Gang Lu, K. Leung
          </td>
          <td>2025-03-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) exhibits significant intra-tumoral heterogeneity (ITH) at both morphological and genetic levels, complicating treatment and contributing to disease progression1,2. Among these, rhabdoid ccRCCs stand out as highly aggressive tumors distinguished by cells with unique morphological features and poor clinical outcomes3. However, the correlation between distinct morphological phenotypes, specific molecular alterations, and their influence on tumor behavior remains poorly understood. Understanding the link between clinically relevant morphological features and their molecular underpinnings will be critical for developing more effective treatments targeting key cellular subsets responsible for tumor progression. In this study, we integrated advanced AI-based image analysis with single-cell isolation and multi-omics profiling to dissect the link between clinically relevant morphological and molecular features of ccRCC cells. Using a novel digital pathology workflow, we quantified low-grade, high-grade, and rhabdoid morphologies in ccRCC diagnostic images with unprecedented precision. Subsequently, isolation of two sets of 1,000 morphologically distinct cells for detailed mRNA and protein expression analyses, revealed significant increasing dysregulation associating with higher histopathological grades. Rhabdoid ccRCC cells (grade 4) demonstrated distinct molecular profiles, including upregulated FOXM1-driven proliferation, disrupted cell-matrix interactions, and enhanced immune evasion pathways. Despite high T-cell infiltration in rhabdoid areas, we identified a rhabdoid-specific immunosuppressive network driven by cytokines, IFN-beta, and integrin signaling, likely contributing to T-cell exhaustion. These findings provide a basis for novel therapeutic strategies targeting these pathways in combination with immunotherapy to improve outcomes for patients with aggressive rhabdoid ccRCC. Key Points ccRCC is characterized by well-established morphological heterogeneity but the correlation with the underlying molecular aberrations remained elusive. By integrating AI-based image analysis with single cell isolation and deep multi-omics profiling, we dissect the molecular intricacies of ccRCC, from targeted collection of 1,000 morphologically distinct cells. Our results demonstrate significant dysregulation of gene and protein expression correlating with higher histopathological grades in ccRCC. Aggressive ccRCC cells with rhabdoid differentiation (grade 4) display distinct molecular profiles, as they upregulate FOXM1-mediated proliferation, ECM remodeling and the immune evasion responses, suggesting new therapeutic avenues enhancing ICI efficacy in these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7461a1f413e55baf162ee7fd7309ee3cb97ce7a" target='_blank'>
              Deep visual multi-omics profiling reveals mechanisms that underly cancer cell differentiation and aggressiveness in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            H. A. Bolck, Ede Migh, A. Kriston, Natalia Zajac, Susanne Kreutzer, Tiberiu Totu, Peter Leary, Ferenc Kovács, D. Rutishauser, Sybille Pfammatter, Jonas Grossmann, Cassandra Litchfield, Marija Buljan, Niels J Rupp, Péter Horváth, Holger Moch
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Uncontrolled proliferation is a hallmark of cancer, yet tumour cells can enter G0 arrest by halting the cell cycle reversibly (quiescence) or irreversibly (senescence) to survive under stress and in hostile tumour microenvironments (TME). G0 arrested cells contribute to drug tolerance, metastasis and recurrence, but their identification remains challenging due to their rarity and elusive regulatory pathways. Here, we quantify G0 arrest and proliferation decisions in single-cell and spatially profiled breast primary tumours to unveil adaptive responses driving immune compartmentalisation. We identify a genomically-constrained and prolonged G0 arrest state resembling that of dormant precursors of cancer progression. This state featured adaptive transcriptional reprogramming, including unfolded protein response stress, reduced mitochondrial translation and epithelial-mesenchymal plasticity enabled by semaphorin signalling. Spatial transcriptomics revealed G0 arrest pockets encapsulated by APOE+ lipid-associated macrophages, myofibroblastic CAFs and immature perivascular cells, suggestive of an immunosuppressive niche contrasting with cytotoxic environments in proliferative hotspots and displaying distinct drug sensitivities. To facilitate future research, we provide a foundation model capturing G0 arrest and proliferation with 94% accuracy in single cell data, available at https://github.com/secrierlab/G0-LM. Our findings provide new insights into the spatial organisation of cell cycle arrest in breast cancer, highlighting the role of G0 states in tumour heterogeneity and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d540e388314aece2599568858b5b0ad8004482" target='_blank'>
              Balancing tumour proliferation and sustained cell cycle arrest through proteostasis remodelling drives immune niche compartmentalisation in breast cancer
              </a>
            </td>
          <td>
            Cenk Celik, Eloise Withnell, Shi Pan, Tooki Chu, Johnathan Labbadia, M. Secrier
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Microbes have been found in various tumors and the research of tumor microbiome has been garnering increased attention. However, it remains challenge to investigate the microbiome in cancer at spatial resolution. The rapid advent of spatially resolved transcriptomics techniques has given rise to map transcripts at single-cell resolution in various types of cancer. Here, we constructed a comprehensive spatial meta-transcriptome resource by manually curating 203 fresh frozen (FF) slices from 20 cancers encompassing 334,253 spots and 1,908,646 cells. SMTdb (http://bio-bigdata.hrbmu.edu.cn/SMTdb/) was constructed to provide comprehensive insights into the abundance, distribution and enriched TME regions of 1218 microbiota in spatial tissue slices. SMTdb enables vast interactive data exploration of spatial distribution and expression of microbiota, host gene modules associated with certain microbiota and co-occurrence between microbiota and immune cells within tumor microenvironment. The atlas resource serves as a one-stop and time-effective platform to investigate the interactions among microbial ecosystems and hosts in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803a0dfe5a2e579dc8f160c9acd2466a2dcd43" target='_blank'>
              SMTdb: A comprehensive spatial meta-transcriptome resource in cancer
              </a>
            </td>
          <td>
            Weiwei Zhou, Qingyi Yang, Jiyu Guo, Si Li, Minghai Su, Feng Leng, Tingyu Rong, Jingyi Shi, Yueying Gao, Tiantongfei Jiang, Juan Xu, Yongsheng Li
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) remains a challenging disease due to its significant heterogeneity, therapy resistance, and relentless progression. Multi-omics technologies offer the potential to provide uniquely precise views of disease progression and response to therapy. We present here a comprehensive multi-omics view of CTCL clonal evolution, incorporating exome, whole genome, epigenome, bulk-, single cell (sc) VDJ-, and scRNA-sequencing of 114 clinically annotated serial skin, peripheral blood, and lymph node samples from 35 CTCL patients. We leveraged this extensive dataset to define the molecular underpinnings of CTCL progression in individual patients at single cell resolution with the goal of identifying clinically useful biomarkers and therapeutic targets. Our studies identified a large number of recurrent progression-associated clonal genomic alterations; we highlight mutation of CCR4, PI3K signaling, and PD-1 checkpoint pathways as evasion tactics deployed by malignant T cells. We also identified a gain of function mutation in STAT3 (D661Y) and demonstrated by CUT&RUN-seq that it enhances binding to transcription start sites of genes in Rho GTPase pathways, which we previously reported to have activated chromatin and increased expression in HDACi-resistant CTCL. These data provide further support for a previously unrecognized role for Rho GTPase pathway dysregulation in CTCL progression. A striking number of progression-associated mutations occurred in chromatin methylation modifiers, including EZH2, suggesting that EZH1/2 inhibition may also benefit patients with CTCL. Knowledge of these molecular changes should be leveraged for improved disease monitoring, biomarker-informed clinical trial design, and new therapeutic strategies in this challenging and incurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam U. Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="T cell phenotypes and kinetics are emerging as crucial factors associated with immunotherapeutic responses in a wide range of solid cancer types. However, challenges remain in understanding the spatial and temporal profiles of T cells with differential phenotypes due to difficulties in single-cell analysis with preserved tissue structures. Here, we provide an optimized 12-marker multiplex immunohistochemical (IHC) panel and single-cell-based quantitative assessment to identify the spatial distributions of T cell phenotypes in formalin-fixed paraffin-embedded sections. This panel revealed differential T cell populations with spatial localizations in human tonsil tissue, where the percentages of CD8+ T cell-expressing programmed death receptor-1 (PD-1), T cell immunoglobulin and mucin domain 3 (TIM3), and other T cell phenotypic markers vary by tonsillar tissue components such as follicles, parenchyma, and epithelium. A specimen from salivary gland adenocarcinoma during hyper-progression, followed by anti-PD-1 treatment, exhibited the exclusion of CD8+ T cells from the intratumoral regions. Although the vast majority of peritumoral CD8+ T cells exhibited proliferative effector T cell phenotypes with PD-1−TIM3−Ki67+CD45RA+, intratumoral CD8+ T cells showed exhausted phenotypes with PD-1+TIM3− and increased Eomes expression, which might be related to poor therapeutic response in this case. To verify these findings in the context of temporal changes, we analyzed six longitudinal samples from a single patient with maxillary sinus cancer, observing increased T cell exhaustion along with metastasis and progression. Together, highly multiplexed IHC can be applied to analyze the spatiotemporal phenotypes of T cells, potentially contributing to the understanding of the mechanisms of resistance to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3aa6e9e33a49a78ba8f7c54e58b7d68564b492" target='_blank'>
              Applications of Multiplex Immunohistochemistry in Evaluating Spatiotemporal Heterogeneity of T Cells
              </a>
            </td>
          <td>
            Mercedes Machuca-Ostos, Tim de Martines, K. Yoshimura, Junichi Mitsuda, Sumiyo Saburi, Alisa Kimura, Hiroki Morimoto, Koichi Yoshizawa, Nana Sakurai, Nanako Murakami, Kayo Kitamoto, Makoto Yasuda, Y. Sugiyama, H. Ogi, Saya Shibata, Aya Miyagawa-Hayashino, Eiichi Konishi, Kyoko Itoh, Takahiro Tsujikawa, Shigeru Hirano
          </td>
          <td>2025-02-17</td>
          <td>Immuno</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Triple-negative breast cancer is a prevalent breast cancer subtype with the lowest 5-year survival. Several factors contribute to its treatment response, but the inherent molecular and cellular tumor heterogeneity are increasingly acknowledged as crucial determinants. Methods Spatial transcriptomic profiling was performed on FFPE tissues from a retrospective, treatment-naive group of women with differential prognoses (17 with >15 years survival-good prognosis (GPx) and 15 with <3 years survival-poor prognosis (PPx)) using GeoMX® Digital Spatial Profiler. Regions of interest were segmented on pan-cytokeratin and analyzed for tumor and stromal components, probed using GeoMx human whole transcriptome atlas (WTA) panel. Data quality control, normalization, and differential analysis was performed in R using GeomxTools and linear mixed models. Additional analyses including cell-type deconvolution, spatial entropy, functional enrichment, TF-target / ligand-receptor analysis and convolution neural networks were employed to identify significant gene signatures contributing to differential prognosis. Results Here we report on the spatial and molecular heterogeneity underlying differential prognosis. We observe that the state of the epithelia and its microenvironment (TME) are transcriptionally distinct between the two groups. Invasive epithelia in GPx show a significant increase in immune transcripts with the TME exhibiting increased immune cell presence (via IF), while in PPx they are more metabolically and translationally active, with the TME being more mesenchymal/fibrotic. Specifically, pre-cancerous epithelia in PPx display a prescience of aggressiveness as evidenced by increased EMT-signaling. We identify distinct epithelial gene signatures for PPx and GPx, that can, with high accuracy, classify samples at the time of diagnosis and likely inform therapy. Conclusions To the best of our knowledge, this is the first study to leverage spatial transcriptomics for an in-depth delineation of the cellular and molecular underpinnings of differential prognosis in TNBC. Our study highlights the potential of spatial transcriptomics to not only uncover the molecular drivers of differential prognosis in TNBC but also to pave the way for precision diagnostics and tailored therapeutic strategies, transforming the clinical landscape for this aggressive breast cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31351362773ec3359f5dd1e2c1fcb9f73d1b49df" target='_blank'>
              Spatially distinct cellular and molecular landscapes define prognosis in triple negative breast cancer
              </a>
            </td>
          <td>
            Kavitha Mukund, Darya Veraksa, David Frankhouser, Lixin Yang, Jerneja Tomsic, Raju Pillai, Srijan Atti, Zahra Mesrizadeh, D. Schmolze, Xiao-Cheng Wu, Mary-Anne LeBlanc, Lucio Miele, Augusto Ochoa, Victoria L Seewaldt, Shankar Subramaniam
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell spatial omics analysis requires consideration of biological functions and mechanisms in a microenvironment. However, microenvironment analysis using bioinformatic methods is limited by the need to detect histological morphology and extend it to the surrounding area. In this study, we developed SpatialKNifeY (SKNY), an image-processing-based toolkit that detects spatial domains that potentially reflect histology and extends these domains to the microenvironment. Using spatial transcriptomic data from breast cancer, we applied the SKNY algorithm to identify tumor spatial domains, followed by clustering of the domains, trajectory estimation, and spatial extension to the tumor microenvironment (TME). The results of the trajectory estimation were consistent with the known mechanisms of cancer progression. We observed tumor vascularization and immunodeficiency at mid- and late-stage progression in TME. Furthermore, we applied the SKNY to integrate and cluster the spatial domains of 14 patients with metastatic colorectal cancer, and the clusters were divided based on the TME characteristics. In conclusion, the SKNY facilitates the determination of the functions and mechanisms in the microenvironment and cataloguing of the features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796429c918dab0f1b4d04958c50f7cf17d100c5e" target='_blank'>
              SpatialKNifeY (SKNY): Extending from spatial domain to surrounding area to identify microenvironment features with single-cell spatial omics data
              </a>
            </td>
          <td>
            Shunsuke A Sakai, Ryosuke Nomura, Satoi Nagasawa, SungGi Chi, Ayako Suzuki, Yutaka Suzuki, Mitsuho Imai, Y. Nakamura, Takayuki Yoshino, Shumpei Ishikawa, K. Tsuchihara, S. Kageyama, Riu Yamashita
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78a9a8e3280a5c16b1080da5f7e2a241cd45990" target='_blank'>
              Diffusion Smart-seq3 of breast cancer spheroids to explore spatial tumor biology and test evolutionary principles of tumor heterogeneity
              </a>
            </td>
          <td>
            Antony Cougnoux, Loay Mahmoud, Per A Johnsson, Alper Eroglu, Louise Gsell, Jakob Rosenbauer, R. Sandberg, Jean Hausser
          </td>
          <td>2025-01-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is the deadliest cancer worldwide. Therapeutic progress stagnate, highlighting the complexity to replicate NSCLC in preclinical models. Drug discovery studies rely mostly on cancer cells in two-dimension (2D), which poorly predict drug efficacy in patients. There is a growing interest in three-dimensional (3D) preclinical models, such as 3D spheroids, to better model tumor phenotype and improve therapeutic prediction. However, a comprehensive view of 3D culture methods impact on transcriptomes, epigenomes and pharmacological responses and their correlations to NSCLC tumors is still missing. Here, we demonstrate that NSCLC spheroids undergo time-dependent transcriptomic and epigenomic changes, which peak after 3 weeks of culture. While DNA methylome remained stable, chromatin methylation and acetylation marks gained features of advanced NSCLC in a time-dependent manner. Single-cell transcriptomic profiling of spheroids demonstrated that time of 3D culture improved the correlation to NSCLC tumors. Moreover, long-term culture of 3D spheroids increased drug screening predictability, by showing resistance to drugs that failed in NSCLC patients (such as HDAC inhibitors) while demonstrating novel pharmacological vulnerabilities and synergistic interactions (such as combination of PRMT and HDAC inhibitors). Strikingly, reverting 3D spheroids back to 2D culture rapidly reversed transcriptomic, epigenetic and pharmacological signatures acquired after 3 weeks of 3D culture, highlighting the critical impact of cell culture conditions on NSCLC phenotype. Collectively, our findings demonstrate that implementing a time-dependent maturation process into 3D spheroid culture induces chromatin and transcriptomic changes that enhance NSCLC preclinical modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f644ee3c95f246053a935ce676721b85861fe7f5" target='_blank'>
              Time-dependent chromatin maturation during 3D spheroid culture improves preclinical modeling of non-small cell lung cancer
              </a>
            </td>
          <td>
            Anaïs Darracq, Nicolas Sgarioto, Marielle Huot, Gabrielle McInnes, Antoine Meant, Alexandra Langford, Oscar D Villarreal, François Marois, Maxime Caron, P. Saint-Onge, S. Leclerc, G. Cagnone, Gregor Andelfinger, Stéphane Richard, Daniel Sinnett, Serge McGraw, Noël J-M Raynal
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2410faffb025161d800cf5841a3c4049fde9d25" target='_blank'>
              Integration of single-cell and spatial transcriptomics reveals fibroblast subtypes in hepatocellular carcinoma: spatial distribution, differentiation trajectories, and therapeutic potential
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-02-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chronic inflammation is a well-established risk factor for cancer, but the underlying molecular mechanisms remain unclear. Using a mouse model of colitis, we demonstrate that colonic stem cells retain an epigenetic memory of inflammation following disease resolution, characterized by a cumulative gain of activator protein 1 (AP-1) transcription factor activity. Further, we develop SHARE-TRACE, a method that enables simultaneous profiling of gene expression, chromatin accessibility and clonal history in single cells, enabling high resolution tracking of epigenomic memory. This reveals that inflammatory memory is propagated cell-intrinsically and inherited through stem cell lineages, with certain clones demonstrating dramatically stronger memory than others. Finally, we show that colitis primes stem cells for amplified expression of regenerative gene programs following oncogenic mutation that accelerate tumor growth. This includes a subpopulation of tumors that have exceptionally high AP-1 activity and the additional upregulation of pro-oncogenic programs. Together, our findings provide a mechanistic link between chronic inflammation and malignancy, revealing how long-lived epigenetic alterations in regenerative tissues may contribute to disease susceptibility and suggesting potential therapeutic strategies to mitigate cancer risk in patients with chronic inflammatory conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f579a4d65bfde6d9ba50ff844febde4a92d36d5" target='_blank'>
              Clonal memory of colitis accumulates and promotes tumor growth
              </a>
            </td>
          <td>
            Surya Nagaraja, Lety Ojeda-Miron, Ruochi Zhang, Ena Oreskovic, Yan Hu, Daniel Zeve, Karina Sharma, Roni R. Hyman, Qiming Zhang, Andrew Castillo, David T. Breault, Ömer H Yilmaz, Jason D. Buenrostro
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor dissemination is increasingly recognized to begin early in tumor development. Although most of these early disseminated cells are cleared, some survive and persist below clinical detection, acting as reservoirs for metastatic relapse. Metastatic tumor cells often rely on interactions with local stromal cells to support their colonization. In this study, we propose that pericyte-tumor cell interactions promote dormancy induction in the early metastatic lung, enhancing disseminated tumor cell (DTC) persistence. Extravital imaging demonstrated that DTCs interact with pericytes upon extravasation into the lung. Co-culture experiments were used to assess DTC fate after pericyte contact and revealed that transient contact with pericytes reduced the proliferation of metastatic 4T1 breast cancer cells but had no effect on non-metastatic 67NR cells. In vivo, transient pericyte contact resulted in higher lung metastatic burden, driven by small, non-proliferative lesions (<6 cells), 10 days after intracardiac injection. These lesions exhibited reduced KI67 staining and EdU incorporation compared to those from monocultured cells. We further observed that primary lung pericytes transferred lyso-phospholipids (lyso-PLs) specifically to metastatic 4T1 cells through direct contact. Gene expression analysis indicated that transient pericyte contact activated pathways related to syncytium formation in metastatic cells. In normal physiology, pericytes act in a syncytium to regulate blood flow via mechanosensitive channels in response to blood pressure changes. We hypothesize that tumor cells exploit these mechanosensitive responses to trigger lyso-PL transfer from pericytes. Supporting this, calcium imaging showed higher calcium activity in pericytes co-cultured with 4T1 cells, and calcium channel inhibitors significantly reduced lyso-PL transfer. Pharmacological activation of pericyte calcium channels induced lyso-PL release, which was subsequently taken up by tumor cells. Conditioned medium from activated pericytes, containing free lyso-PLs, recapitulated the reduced proliferation observed in transient co-culture. Finally, we found our pericyte-induced dormancy signature to be associated with tumor dormancy and distant metastasis free survival latency in breast cancer patients. Together, these findings suggest that early DTCs may exploit pericyte signaling mechanisms to enter dormancy, facilitating their persistence at metastatic sites and contributing to future relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1fe686f97fcb46c56e3f6bfbef8e78fcca9cb41" target='_blank'>
              Pericyte-tumor crosstalk facilitates metastatic tumor cell latency through PIEZO1-activated lysophospholipid transfer
              </a>
            </td>
          <td>
            Tamara McErlain, Morgan J Glass, Elizabeth C McCulla, Caitlin A Madden, Lauren E Ziemer, Kirsten E Overdahl, Alan K. Jarmusch, Michael J. Kruhlak, A. Chesler, Howard H Yang, Maxwell P Lee, Cristina M Branco, Meera Murgai
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Single-cell RNA sequencing studies have revealed the heterogeneity of cell states present in the rheumatoid arthritis (RA) synovium. However, it remains unclear how these cell types interact with one another in situ and how synovial microenvironments shape observed cell states. Here, we use spatial transcriptomics (ST) to define stable microenvironments across eight synovial tissue samples from six RA patients and characterize the cellular composition of ectopic lymphoid structures (ELS). To identify disease-relevant cellular communities, we developed DeepTopics, a scalable reference-free deconvolution method based on a Dirichlet variational autoencoder architecture. DeepTopics identified 22 topics across tissue samples that were defined by specific cell types, activation states, and/or biological processes. Some topics were defined by multiple colocalizing cell types, such as CD34+ fibroblasts and LYVE1+ macrophages, suggesting functional interactions. Within ELS, we discovered two divergent cellular patterns that were stable across ELS in each patient and typified by the presence or absence of a “germinal-center-like” topic. DeepTopics is a versatile and computationally efficient method for identifying disease-relevant microenvironments from ST data, and our results highlight divergent cellular architectures in histologically similar RA synovial samples that have implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5649472b0dc4cb2ee433d6b1131064e755b734d" target='_blank'>
              Deep topic modeling of spatial transcriptomics in the rheumatoid arthritis synovium identifies distinct classes of ectopic lymphoid structures
              </a>
            </td>
          <td>
            Preethi K. Periyakoil, Melanie H. Smith, Meghana Kshirsagar, Daniel Ramirez, Edward DiCarlo, Susan M. Goodman, Alexander Y. Rudensky, L. Donlin, Christina S. Leslie
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Spatially resolved transcriptomics (SRT) provides an invaluable avenue for examining cell-cell interactions within native tissue environments. The development and evaluation of analytical tools for SRT data necessitate tools for generating synthetic datasets with known ground truth of cell-cell interaction induced features. To address this gap, we introduce sCCIgen, a novel real-data-based simulator tailored to generate high-fidelity SRT data with a focus on cell-cell interactions. sCCIgen preserves transcriptomic and spatial characteristics in SRT data, while comprehensively models various cell-cell interaction features, including cell colocalization, spatial dependence among gene expressions, and gene-gene interactions between nearby cells. We implemented sCCIgen as an interactive, easy-to-use, realistic, reproducible, and well-documented tool for studying cellular interactions and spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b742b7f80c26a09db2a4a0e4218062de306a242d" target='_blank'>
              sCCIgen: A high-fidelity spatially resolved transcriptomics data simulator for cell-cell interaction studies
              </a>
            </td>
          <td>
            Xiaoyu Song, J. C. Chávez‐Fuentes, Weiping Ma, Weijia Fu, Pei Wang, Guo-Cheng Yuan
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) is an aggressive and lethal neoplasm, with its early neoplastic transformation mechanisms remaining poorly understood. Here, we characterize over 25 genetically-defined organoid models derived from murine and human oral/esophageal tissues harboring key driver mutations. Double knockout of TP53 and CDKN2A induced morphological dysplasia, hyperproliferation, loss of squamous differentiation, and tumorigenicity, which were further exacerbated by additional driver mutations (e.g., PIK3CA, NOTCH1, KMT2C). Single-cell analysis revealed an expansion of quiescent basal cells and proliferative squamous cells, alongside a loss of differentiated squamous cells during malignant transformation. A distinct senescence program, regulated by ANXA1, was markedly diminished during early neoplastic evolution. Mechanistically, the ANXA1-SMAD3-p27KIP1 pathway was identified as a critical regulator of this senescence program, acting to suppress neoplastic features in organoid models. Lastly, our high-throughput, single-organoid-resolution drug screens unexpectedly revealed PIK3CA-driven organoids exhibited sensitivity to Mitomycin C and Onalespib. This study provides novel mechanistic insights into early neoplastic evolution and underscores the value of genetically-defined organoid models for investigating cancer biology and identifying targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7127b7b7361c88e16950c954ef2bf66d966c5299" target='_blank'>
              Genetically Defined Organoid Models Reveal Mechanisms Driving Squamous Cell Neoplastic Evolution and Identify Potential Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hua Zhao, Young Min Park, Yueyuan Zheng, Qiong Mao, Casey Collet, Boyan Hu, Tianming Zhou, Luda Lin, Stephanie Wong, Yuhao Pan, Anette Vistoro Monreal, Uttam Sinha, Parish Sedghizadeh, A. Soragni, De-Chen Lin
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Spatial transcriptomics has enabled the study of mRNA distributions within cells, a key aspect of cellular function. However, there is a dearth of tools that can identify and interpret functionally relevant spatial patterns of subcellular transcript distribution. To address this, we present CellSP, a computational framework for identifying, visualizing, and characterizing consistent subcellular spatial patterns of mRNA. CellSP introduces the concept of “gene-cell modules”, which are gene sets with non-random subcellular transcript distributions in many cells. It provides intuitive visualizations of the captured patterns and offers functional insights into each discovered module. We demonstrate that CellSP reliably identifies functionally significant modules across diverse tissues and technologies. We use the tool to discover subcellular spatial phenomena related to myelination, axonogenesis and synapse formation in the mouse brain. We find immune response-related modules that change between kidney cancer and healthy samples, and myelination-related modules specific to mouse models of Alzheimer’s Disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26c9d33690c61fab528b90b2d805b9244c8e0fba" target='_blank'>
              CellSP: Module discovery and visualization for subcellular spatial transcriptomics data
              </a>
            </td>
          <td>
            Bhavay Aggarwal, Saurabh Sinha
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Brain metastases (BrMets), common for advanced-stage breast cancer patients, are associated with poor median survival and accompanied by severe neurologic decline. Halting the progression of breast cancer brain metastases (BCBMs) may require modulation of the tumor microenvironment (TME), yet little is known about the impact of the primary breast TME on brain tropism, or how, once there, metastatic breast cancer cells coexist with brain-resident cells (e.g., neurons and glia). Traditionally, studies in this space have focused on differential expression analysis, overlooking potential insights gained from investigating genes with equivalent expression between groups. This is particularly crucial in distant metastasis, where tumor cells may co-opt the transcriptional programs of the host organ (e.g., brain) to facilitate successful seeding and outgrowth. Prior to our work, no computational framework existed to determine biologically-relevant equivalent gene expression. To resolve molecular mechanisms of BCBM enabled by metastatic cancer cells and/or resident brain cells, we leveraged Nanostring GeoMx to perform spatially-resolved transcriptomic profiling on 235 patient-derived tissue cores from BCBM (including adjacent normal brain), primary invasive breast cancers, and normal (non-cancer) brain; analyzing 18,677 RNAs in 450 areas of interest (AOIs). We introduce the “Equivalent Expression Index” a highly specific and accurate algorithm that identifies statistically significant “Equivalently-Expressed Genes”. This method facilitated the identification of molecular remodeling and mimicry genes within tissue-specific TMEs. By integrating differential expression analysis with the Equivalent Expression Index, we discovered multiple novel gene signatures associated with BCBM and primary tumor brain-metastatic potential. We demonstrate that the Equivalent Expression Index is a powerful tool to uncover shared gene expression programs representing the adaptation of metastatic cells and brain-resident cells to the BCBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015422dee0790a994280a3f1f594af35182af70e" target='_blank'>
              Spatial Profiling Reveals Equivalence-Derived Molecular Signatures of Brain Mimicry and Adaptation in Breast Cancer Brain Metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, C. Yeh, Bryanna Godfrey, Pablo Nunez Perez, Giuseppe Barisano, Sushama Varma, Saman Ahmadian, A. Toland, M. Granucci, Thy T. H. Trinh, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia K Plevritis, M. Gephart
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The ability of tumors to evolve and adapt by developing subclones in different genetic and epigenetic states is a major challenge in oncology. Traditional tools like multi-regional sequencing used to study tumor evolution and the resultant intra-tumor heterogeneity (ITH) are often impractical because of their resource-intensiveness and limited scalability. Here, we present MorphoITH, a novel framework that leverages histopathology slides to deconvolve molecular ITH through tissue morphology. MorphoITH integrates a self-supervised deep learning similarity measure to capture phenotypic variation across multiple dimensions (cytology, architecture, and microenvironment) with rigorous methods to eliminate spurious sources of variation. Using a prototype of ITH, clear cell renal cell carcinoma (ccRCC), we show that MorphoITH captures clinically-significant biological features, such as vascular architecture and nuclear grades. Furthermore, we find that MorphoITH recognizes differential biological states corresponding to subclonal changes in key driver genes (BAP1/PBRM1/SETD2). Finally, by applying MorphoITH to a multi-regional sequencing experiment, we postulate evolutionary trajectories that largely recapitulate genetic evolution. In summary, MorphoITH provides a scalable phenotypic lens that bridges the gap between histopathology and genomics, advancing precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51f0d69b3ba41ef9a508aa6acb9b650626ef267d" target='_blank'>
              MorphoITH: A Framework for Deconvolving Intra-Tumor Heterogeneity Using Tissue Morphology
              </a>
            </td>
          <td>
            Aleksandra Weronika Nielsen, H. E. Manoochehri, Hua Zhong, Vandana Panwar, V. Jarmale, Jay Jasti, Mehrdad Nourani, Dinesh Rakheja, J. Brugarolas, Payal Kapur, Satwik Rajaram
          </td>
          <td>2025-02-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Single-cell transcriptomic studies have identified distinct microglial subpopulations with shared and divergent gene signatures across development, aging and disease. Whether these microglial subsets represent ontogenically separate lineages of cells, or they are manifestations of plastic changes of microglial states downstream of some converging signals is unknown. Furthermore, despite the well-established role of enhancer landscapes underlying the identity of microglia, to what extent histone modifications and DNA methylation regulate microglial state switches at enhancers have not been defined. Here, using genetic fate mapping, we demonstrate the common embryonic origin of proliferative-region-associated microglia (PAM) enriched in developing white matter, and track their dynamic transitions into disease-associated microglia (DAM) and white matter-associated microglia (WAM) states in disease and aging contexts, respectively. This study links spatiotemporally discrete microglial states through their transcriptomic and epigenomic plasticity, while revealing state-specific histone modification profiles that govern state switches in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f71e88e7c209aa91ac01a49896d75e72d7b341cd" target='_blank'>
              Microglial plasticity governed by state-specific enhancer landscapes
              </a>
            </td>
          <td>
            Nicole Hamagami, Dvita Kapadia, Nora Abduljawad, Zuolin Cheng, Liam McLaughlin, Darsh Singhania, Kia M. Barclay, Jin Yang, Zhixin Sun, Peter Bayguinov, Guoqiang Yu, Harrison W. Gabel, Qingyun Li
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bec213fd287708367fee6bf0c6de9efb0e1c0cb" target='_blank'>
              Natural killer cells occupy unique spatial neighborhoods in HER2- and HER2+ human breast cancers
              </a>
            </td>
          <td>
            Femke A I Ehlers, Katie E. Blise, C. Betts, Shamilene Sivagnanam, L. Kooreman, E. S. Hwang, Gerard M. J. Bos, L. Wieten, L. Coussens
          </td>
          <td>2025-01-24</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Given the successful graft-versus-leukemia cell treatment effect observed with allogeneic hematopoietic stem cell transplant for patients with refractory or relapsed acute myeloid leukemia, immunotherapies have also been investigated in the nontransplant setting. Here, we use a multi-omic approach to investigate spatiotemporal interactions in the bone marrow niche between leukemia cells and immune cells in patients with refractory or relapsed acute myeloid leukemia treated with a combination of the immune checkpoint inhibitor pembrolizumab and hypomethylating agent decitabine. We derived precise segmentation data by extensively training nuclear and membrane cell segmentation models, which enabled accurate transcript assignment and deep learning-feature-based image analysis. To overcome read-depth limitations, we integrated the single-cell RNA sequencing data with single-cell-resolution spatial transcriptomic data from the same sample. Quantifying cell-cell distances between cell edges rather than cell centroids allowed us to conduct a more accurate downstream analysis of the tumor microenvironment, revealing that multiple cell types of interest had global enrichment or local enrichment proximal to leukemia cells after pembrolizumab treatment, which could be associated with their clinical responses. Furthermore, ligand-receptor analysis indicated a potential increase in TWEAK signaling between leukemia cells and immune cells after pembrolizumab treatment. Highlights Spatial transcriptomic analysis of R-AML bone marrow niches provides detailed information about intercellular interactions in the tumor microenvironment. Immunotherapy shifts the cell composition of the leukemia neighborhood. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79158e9c810aea3af6c5a5e81cece5524c3196f" target='_blank'>
              Single Cell Spatial Transcriptomics Reveals Immunotherapy-Driven Bone Marrow Niche Remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R. Herzog, Abigail Wong-Rolle, Laura W Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine Calvo, Christopher S Hourigan, Kasper Hansen, Chen Zhao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Central Nervous System (CNS) tumors have sophisticated tumor microenvironment (TME) with different cell types such as astrocytes, microglia, neurons, vascular endothelial cells and immune cells. These non-cancerous cells orchestrate the brain TME to regulate cancer progression and therapeutic response. This study aimed to develop a cell composition deconvolution method for CNS tumor and to determine the impact of these cell compositions on patients’ outcomes. Methods We identified the cell type-specific CpG loci using the pairwise differential methylation analysis for 13 major cell types in CNS. Using non-negative least squares (NNLS) methods, we established this cell-type deconvolution approach, MDBrainT, for brain tumors. The predictive accuracy of our MDBrainT model was tested in the DNA methylation profiling of the purified cell samples and compared against another algorithm. Cell composition was predicted by MDBrainT in several brain tumor (glioma, ependymoma, medulloblastoma and ATRT) cohorts and the correlation between cell composition and tumor molecular subtypes and patient outcomes was also assessed. Results Cell type-specific CpG loci for CNS TME was used to build MDBrainT model. Based on these DNA methylation markers, MDBrainT predicted the cell composition in the TME including tumor cells with high accuracy. Endothelial cells were predominately presented in glioblastomas while the percentage of CD8 T cells wassignificantly higher in ATRT. A substantial difference of cell composition was two molecular groups of posterior fossa ependymoma (PFA vs PFB). A higher percentage of cells in TME was usually associated with worse outcomes. Conclusions MDBrainT is a robust algorithm for cell composition prediction for brain TME. Cell composition in brain TME is distinct across different pathological types and molecular subtypes. Key Points MDBrainT is a robust DNA methylation-based deconvolution approach for brain tumor microenvironment (TME). Different molecular subtypes of brain tumors have distinct cell composition patterns. Cell composition in brain TME informs patients of outcomes. Importance of this study DNA methylation is a cell type specific marker that has been utilized for tumor molecular diagnosis, disease progression and therapeutic monitoring. A DNA methylation-based classifier for brain tumors precisely predicts the molecular subtypes but not the cell composition of tumor microenvironment. Brain tumors are a complex cell mixture where tumor microenvironment is critical for tumorigenesis and therapeutic resistance. Here, we developed a novel deconvolution approach (MDBrainT) to predict cell composition for brain tumors. Our model has revealed the heterogeneity of cell composition between tumor types. In addition, tumors with different molecular subtypes have distinct cell composition. Cell percentage in TME also informs patients of outcomes. The tumor microenvironment including cell composition of each patient may direct the different regimen of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1823256c94dd6a377f1b3165f4ae951356ef07" target='_blank'>
              DNA Methylation-Based Cell Type Deconvolution Reveals the Distinct Cell Composition in Brain Tumor Microenvironment
              </a>
            </td>
          <td>
            Feili Liu, Jin Qian, Chenkai Ma
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Growing evidence supports the importance of characterizing the organizational patterns of various cellular constituents in the tumor microenvironment in precision oncology. Most existing data on immune cell infiltrates in tumors, which are based on immune cell counts or nearest neighbor-type analyses, have failed to fully capture the cellular organization and heterogeneity. Methods We introduce a computational algorithm, termed Tumor-Immune Partitioning and Clustering (TIPC), that jointly measures immune cell partitioning between tumor epithelial and stromal areas and immune cell clustering versus dispersion. As proof-of-principle, we applied TIPC to a prospective cohort incident tumor biobank containing 931 colorectal carcinoma cases. TIPC identified tumor subtypes with unique spatial patterns between tumor cells and T lymphocytes linked to certain molecular pathologic and prognostic features. T lymphocyte identification and phenotyping were achieved using multiplexed (multispectral) immunofluorescence. In a separate hepatocellular carcinoma cohort, we replaced the stromal component with specific immune cell types—CXCR3+CD68+ or CD8+—to profile their spatial relationships with CXCL9+CD68+ cells. Results Six unsupervised TIPC subtypes based on T lymphocyte distribution patterns were identified, comprising two cold and four hot subtypes. Three of the four hot subtypes were associated with significantly longer colorectal cancer (CRC)-specific survival compared to a reference cold subtype. Our analysis showed that variations in T-cell densities among the TIPC subtypes did not strictly correlate with prognostic benefits, underscoring the prognostic significance of immune cell spatial patterns. Additionally, TIPC revealed two spatially distinct and cell density-specific subtypes among microsatellite instability-high colorectal cancers, indicating its potential to upgrade tumor subtyping. TIPC was also applied to additional immune cell types, eosinophils and neutrophils, identified using morphology and supervised machine learning; here two tumor subtypes with similarly low densities, namely ‘cold, tumor-rich’ and ‘cold, stroma-rich’, exhibited differential prognostic associations. Lastly, we validated our methods and results using The Cancer Genome Atlas colon and rectal adenocarcinoma data (n = 570). Moreover, applying TIPC to hepatocellular carcinoma cases (n = 27) highlighted critical cell interactions like CXCL9-CXCR3 and CXCL9-CD8. Conclusions Unsupervised discoveries of microgeometric tissue organizational patterns and novel tumor subtypes using the TIPC algorithm can deepen our understanding of the tumor immune microenvironment and likely inform precision cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95d3eb1387144868bd261357332833886cbc29f0" target='_blank'>
              Tumor-immune partitioning and clustering algorithm for identifying tumor-immune cell spatial interaction signatures within the tumor microenvironment
              </a>
            </td>
          <td>
            M. Lau, J. Borowsky, J. Väyrynen, K. Haruki, Melissa Zhao, Andressa Dias Costa, Simeng Gu, Annacarolina da Silva, Tomotaka Ugai, Kota Arima, Minh N Nguyen, Yasutoshi Takashima, Joe Yeong, D. Tai, Tsuyoshi Hamada, Jochen K Lennerz, Charles S Fuchs, Catherine J Wu, J. Meyerhardt, S. Ogino, J. Nowak
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="Breast cancer (BC) is the second leading cause of cancer-related death among women in the U.S. Organoid models of solid tumors have been shown to faithfully recapitulate aspects of cancer progression such as proliferation and invasion. Although patient-derived organoids (PDOs) and patient-derived xenograft organoids (PDXOs) are pathophysiologically relevant, they are costly to propagate, difficult to manipulate and comprised primarily of the most proliferative cell types within the tumor microenvironment (TME). These limitations prevent their use for elucidating cellular mechanisms of disease progression that depend upon tumor-associated stromal cells which are found within the TME and known to contribute to metastasis and therapy resistance. Here, we report on methods for cultivating epithelial-stromal multicellular 3D cultures. Advantages of these methods include a cost-effective system for rapidly generating organoid-like 3D cultures within scaffold-free environments that can be used to track invasion at single-cell resolution within hydrogel scaffolds. Specifically, we demonstrate how to generate these hetero-multicellular 3D cultures using BT-474 breast cancer cells in combination with fibroblasts (BJ-5ta), monocyte-like cells(THP-1) and/or endothelial cells (EA.hy926). Additionally, differential fluorescent labeling of cell populations enables time-lapse microscopy to define 3D culture assembly and invasion dynamics. Notably, the addition of any two stromal cell combinations to 3D cultures of BT-474 cells significantly reduces circularity of the 3D cultures, consistent of the presence of organoid-like or secondary spheroid structures. In tracker dye experiments, fibroblasts and endothelial cells co-localize in the peripheral organoid-like protrusions and are spatially segregated from the primary BT-474 spheroid. Finally, hetero-multicellular 3D cultures of BT-474 cells have increased hydrogel invasion capacity. Since we observed these protrusive structures in hetero-multicellular 3D cultures of both non-tumorigenic and tumorigenic breast epithelial cells, this work provides an efficient and reproducible method for generating organoid-like 3D cultures in a scaffold-free environment for subsequent analyses of phenotypes associated with solid tumor progression. SUMMARY There is a critical need for 3D cancer models that capture hetero-cellular crosstalk to study cancer metastasis. Our study presents the generation of hetero-multicellular stromal-epithelial in a scaffold and scaffold free environment that can be used to study invasion and cellular spatial distributions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a55555e37181d8e4de2e978898726ac03e16ca" target='_blank'>
              Hetero-multicellular stromal cells incorporate into scaffold-free 3D cultures of epithelial cancer cells to drive invasion
              </a>
            </td>
          <td>
            Elizabeth Ortiz, Kyaw Hsu Thway, Gabriela Ortiz-Soto, Paulina Yao, Jonathan A Kelber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Constructing a comprehensive spatiotemporal map of tumor heterogeneity is essential for understanding tumor evolution, with copy number variation (CNV) as a significant feature. Existing studies often rely on tools originally developed for single-cell data, which fail to utilize spatial information. Our study aims to develop a model that fully leverages spatial omics data to elucidate spatio-temporal changes in tumor evolution. Here, we introduce SCOIGET (Spatial COpy number Inference by Graph on Evolution of Tumor), a novel framework using graph neural networks with graph attention layers to learn spatial neighborhood features of gene expression and infer copy number variations. This approach integrates spatial multi-omics features to create a comprehensive spatial map of tumor heterogeneity. Notably, SCOIGET achieves a substantial reduction in error metrics (e.g., mean squared error, cosine similarity, and distance measures) and produces superior clustering performance as indicated by higher Silhouette Scores compared to existing methods. Our model significantly enhances the efficiency and accuracy of tumor evolution depiction, capturing detailed spatial and temporal changes within the tumor microenvironment. It is versatile and applicable to various downstream tasks, demonstrating strong generalizability across different spatial omics technologies and cancer types. This robust performance improves research efficiency and provides valuable insights into tumor progression. In conclusion, SCOIGET offers an innovative solution by integrating multiple features and advanced algorithms, providing a detailed and accurate representation of tumor heterogeneity and evolution, aiding in the development of personalized cancer treatment strategies. Key Points 1. Innovative Framework for Spatially Contextualized CNV Inference Introduces SCOIGET, a graph-based model that integrates spatial omics data with graph neural networks and attention mechanisms to accurately infer spatial copy number features. 2. Detailed Tumor Heterogeneity Mapping Constructs representations of tumor evolution, capturing heterogeneity and clonal dynamics at both cellular and subcellular resolutions. 3. Versatile and Robust Performance Outperforms existing methods across multiple datasets, demonstrating reliability and adaptability to diverse spatial omics platforms and cancer types. 4. Broad Downstream Applications Enables tasks such as tumor subclone identification, clustering, and evolutionary trajectory analysis, facilitating precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5782e67aa86bd64459296bc264a422b250cba97" target='_blank'>
              SCOIGET: Predicting Spatial Tumor Evolution Pattern by Inferring Spatial Copy Number Variation Distributions
              </a>
            </td>
          <td>
            Yujia Zhang, Yitao Yang, Yan Kong, Bingxu Zhong, Kenta Nakai, Hui Lu
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fbdae748bda0a4cdbd87979473a945cde8f436f" target='_blank'>
              Comprehensive single-cell atlas of colorectal neuroendocrine tumors with liver metastases: unraveling tumor microenvironment heterogeneity between primary lesions and metastases
              </a>
            </td>
          <td>
            Yiqiao Deng, Qichen Chen, Chengyao Guo, Jinghua Chen, Xin Li, Zhi-yu Li, Ye‐fan Zhang, Jian‐jun Zhao, Jianguo Zhou, Jianqiang Cai, T. Yan, Xiaobing Wang, X. Bi, Zhen Huang, Hong Zhao
          </td>
          <td>2025-01-21</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Lung cancer is a leading cause of cancer-related mortality worldwide, largely due to its heterogeneity and intrinsic drug resistance. Malignant pleural effusions (MPEs) provide diverse tumor cell populations ideal for studying these complexities. Although chemotherapy and targeted therapies can be initially effective, subpopulations of cancer cells with phenotypic plasticity often survive treatment, eventually developing resistance. Here, we integrated single-cell isolation and three-dimensional (3D) spheroid culture to dissect subclonal heterogeneity and drug responses, aiming to inform precision medicine approaches. Using A549 lung cancer cells, we established a cisplatin-resistant line and isolated three resistant subclones (Holoclone, Meroclone, Paraclone) via single-cell sorting. In 3D spheroids, Docetaxel and Alimta displayed higher IC50 values than in 2D cultures, suggesting that 3D models better reflect clinical dosing. Additionally, MPE-derived Holoclone and Paraclone subclones exhibited distinct sensitivities to Giotrif and Capmatinib, revealing their heterogeneous drug responses. Molecular analyses confirmed elevated ABCB1, ABCG2, cancer stem cell (CSC) markers (OCT4, SOX2, CD44, CD133), and epithelial–mesenchymal transition (EMT) markers (E-cadherin downregulation, increased Vimentin, N-cadherin, Twist) in resistant subclones, correlating with enhanced migration and invasion. This integrated approach clarifies the interplay between heterogeneity, CSC/EMT phenotypes, and drug resistance, providing a valuable tool for predicting therapeutic responses and guiding personalized, combination-based lung cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6813439ca1c219dd03651dca35df5520579d3204" target='_blank'>
              Application of an Integrated Single-Cell and Three-Dimensional Spheroid Culture Platform for Investigating Drug Resistance Heterogeneity and Epithelial–Mesenchymal Transition (EMT) in Lung Cancer Subclones
              </a>
            </td>
          <td>
            Shin-Hu Chen, Jian-Hong Yu, Yu-Chun Lin, Yi-Ming Chang, Nien-Tzu Liu, Su-Feng Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Liver progenitor cells (LPCs) with bipotential differentiation capacities are essential for restoring liver homeostasis and hepatocyte population after damage. However, the low proportion and shared markers with epithelial cells make studying LPC heterogeneity difficult, especially in humans. To address this gap, we explored over 259,400 human liver single cells across 4 conditions (fetal, healthy, cirrhotic, and HCC-affected livers). Methods: Human liver tissue samples were analyzed using spatial transcriptomics sequencing technologies to describe the heterogeneity of LPCs. Liver tissue was characterized by LPC heterogeneity at single-cell resolution by employing cellular modules, differentiation trajectories, and gene co-expression patterns. Results: We annotated and identified 1 LPC cluster, 3 LPC subpopulations, and 4 distinct cellular modules, indicating the heterogeneity within LPC and the diversity between LPCs and epithelial cells. LPCs showed spatial colocalization with cholangiocytes and comprised a small proportion (2.95±1.91%) within the merged epithelial cells and LPC populations, exhibiting marked differences in marker expression patterns compared to those in mice. LPCs exhibited distinct cellular states in functional restoration, activation, proliferation, and cell transition. Additionally, the gene co-expression network of LPCs exhibited 3 unique modules, reflecting the distinct connectivity of genes encoding apolipoproteins and heat shock proteins in the gene co-expression network modules. Conclusions: Our study provides valuable insights into the multifaceted heterogeneity of human LPCs. Future studies focusing on spatial gene expression dynamics will contribute to our understanding of the spatial arrangement of liver regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363fd866c375a553c02fda34c7798688e167a041" target='_blank'>
              Integrative single-cell and spatial transcriptome analysis reveals heterogeneity of human liver progenitor cells
              </a>
            </td>
          <td>
            Chuanjun Liu, Kai Wang, Junpu Mei, Ruizhen Zhao, Juan Shen, Wei Zhang, Liangyu Li, Bhaskar Roy, Xiaodong Fang
          </td>
          <td>2025-02-26</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while retaining the crucial spatial context within tissues. However, existing ST platforms suffer from high costs, long turnaround times, low resolution, limited gene coverage, and small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that predicts super-resolution gene expression and automatically annotates cellular-level tissue architecture for large-sized tissues that exceed the capture areas of standard ST platforms. The accuracy of iSCALE were validated by comprehensive evaluations, involving benchmarking experiments, immunohistochemistry staining, and manual annotation by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion associated cellular characteristics that were undetectable by conventional ST experiments. Our results demonstrate iSCALE’s utility in analyzing large-sized tissues with automatic and unbiased tissue annotation, inferring cell type composition, and pinpointing regions of interest for features not discernible through human visual assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca1037b8c19366713347f95c55ebd14edfcd843" target='_blank'>
              Scaling up spatial transcriptomics for large-sized tissues: uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE
              </a>
            </td>
          <td>
            Amelia Schroeder, Melanie Loth, Chunyu Luo, Sicong Yao, Hanying Yan, Daiwei Zhang, Sarbottam Piya, Edward Plowey, Wen-Tuo Hu, Jean R. Clemenceau, Inyeop Jang, Minji Kim, Isabel Barnfather, Su Jing Chan, Taylor L. Reynolds, Thomas Carlile, Patrick Cullen, Ji-Youn Sung, Hui-Hsin Tsai, Jeong Hwan Park, Tae Hyun Hwang, Baohong Zhang, Mingyao Li
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Resistance to immune checkpoint inhibitors (ICIs) represents a major barrier to effectively treating patients with NSCLC. Increased tumor heterogeneity has been linked with diminished response to ICIs, but the mechanisms underlying such an effect remain underexplored, in part due to the lack of models for studying the impact of heterogeneity. To address this issue, we have developed a novel murine model of NSCLC that utilizes clonal cell lines to modulate tumor heterogeneity. Specifically, we isolated single cells from a murine NSCLC cell line harboring 3 common driver mutations (Kras G12D Tp53 +/- Lkb1 -/- , or “KPL”). We mutagenized KPL cells with N-methyl-N-nitrosourea in order to increase the tumor mutational burden, and then isolated single cells to derive clonal cell lines. Whole exome sequencing and subsequent phylogenetic analysis of tumor mutations yielded categorization of these clones into 5 clusters. Clones within the same cluster demonstrated comparable in vivo growth kinetics and similar response to anti-PD-1 immunotherapy. These clones exhibited unique tumor immune microenvironments that did not fully correlate with genomic factors such as tumor mutation burden. Notably, heterogeneous polyclonal tumors grew more aggressively than monoclonal tumors, and they exhibited response to anti-PD-1 therapy that differed from the response seen in the component single clones. Tracking of individual clones with fluorescent proteins and mutation tracing illustrated clonal shifts induced by ICI treatment. Our results validate this system as a model of NSCLC that can modify tumor heterogeneity and provide insights into the effects of heterogeneity on ICIs and other immunotherapies.
 Citation Format: Michael Oh, Jensen W Abascal, Linh M Tran, Ramin Salehi-Rad, Raymond J Lim, Camelia Dumitras, Cara Yean, Sunny Xiao, Bitta P Kahangi, William P Crosson, Edgar Perez Reyes, Nalani Coleman, Austin K Rennels, Kostyantyn Krysan, Bin Liu, Steven M Dubinett. A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64eb12bd7d9a78fd223bedc136a7a4d7a78d40" target='_blank'>
              Abstract B068: A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer
              </a>
            </td>
          <td>
            M. Oh, J. Abascal, L. Tran, R. Salehi-Rad, R. Lim, C. Dumitras, Cara E. Yean, Sunny Xiao, Bitta Kahangi, William P Crosson, E. P. Reyes, Nalani Coleman, Austin K Rennels, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 227


 Background:
 Colorectal cancer (CRC) is surgically resectable and eminently screenable yet remains a lethal entity with high affinity to metastasise synchronously and metachronously. It has been proposed that cellular and extracellular components of the microenvironment contribute to metastatic potential. Dissociative profiling techniques such as bulk transcriptomic and single-cell RNA sequencing have contributed insight but limited therapeutic progress. Spatial transcriptomic (ST) assessment allows molecular profiling of tissue while preserving tissue architecture. Here we employ ST approaches to interrogate tumor compartments of primary resectable and matched synchronous CRC demonstrating stromal signatures with distinct collagen expression associated with outcome.
 Methods:
 25 patients with primary resectable CRC and 4 patients with matched primary CRC and liver metastasis (CRLM) underwent single-cell spatial transcriptomics using the Nanostring CosMx Spatial Molecular Imager (SMI, 1000plex gene panel) (Discovery cohort). 71681 epithelial and 88806 stromal cells with intact spatial resolution were analysed. The spatial single-cell signatures were reconstituted in 89 patients using the Nanostring GeoMx Digital Spatial Profiler (Validation cohort). 3 GeoMx compartments were analysed: Epithelial (PanCK+); Stroma (aSMA+); Immune (PanCK-aSMA-).
 Results:
 CosMx demonstrated 2 distinct collagen signatures: COL1A1,COL1A2,COL3A1 associated with normal fibroblasts; COL9A2 associated with cancer-related stroma and lethal subtypes of epithelial cell. In the GeoMx validation cohort, 42 patients expressed the COL1A1 signature in the aSMA compartment and 47 did not (5 year recurrence free survival: 0.88 vs 0.49, p < 0.005) with different morphological patterns of each distinct aSMA group. In patients with favourable prognosis, chemokine high epithelial subtypes expressed CXCL5 and recruited COL9A2-, IL6+ neutrophils to the surrounding microenvironment. In contrast, epithelial cells in patients with poor prognosis expressed CXCL8 and recruited COL9A2+ neutrophils which were frequently in contact with SPP1+ macrophages.
 Conclusions:
 We have used ST approaches to interrogate cellular compartments of CRC demonstrating stromal subtypes that impact outcome. These insights could be used in the clinical setting to quantify COL9A2 expression in aSMA+ cells to prognosticate patients. Targeting of tumor collagen has been proposed to augment existing anti-cancer therapies and this work has demonstrated COL9A2 as a potential target in need of further investigation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead468254f12c6dc5934a33a000887d505597fb4" target='_blank'>
              Spatial transcriptomics and lethality-associated stromal remodeling of colorectal cancer.
              </a>
            </td>
          <td>
            C. Wood, João Da Silva Filho, Tengyu Zhang, Campbell Roxburgh, Paul G. Horgan, J. Edwards, Matthias Marti, Colin W. Steele, Nigel B Jamieson
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Understanding how individual cancer cells adapt to drug treatment is a fundamental challenge limiting precision medicine cancer therapy strategies. While single-cell technologies have advanced our understanding of cellular heterogeneity, efforts to connect the behavior of individual cells to broader tumor drug responses and uncover global trends across diverse systems remain limited. There is a growing availability of single-cell and bulk omics data, but a lack of centralized tools and repositories makes it difficult to study drug response globally, especially at the level of single-cell adaptation. To address this, we present a multimodal framework that integrates bulk and single-cell treated and untreated transcriptomics data to identify drug responsive cell populations in triple-negative breast cancer (TNBC). Our framework leverages population-scale bulk transcriptomics data from TNBC samples to define seven main “identities”, each representing unique combinations of biologically relevant genes. These identities are dynamic and trackable, allowing us to map them onto single cells and uncover global patterns of how cell populations respond to drug treatment. Unlike static classifications, this approach captures the evolving nature of cellular states, revealing that a select few identities dominate and drive population-level responses during treatment. Crucially, our ability to decode these trends through the inherent noise of single-cell data provides a clearer picture of how heterogeneous cell populations adapt to therapy. By identifying the dominant identities and their dynamics, we can better predict how entire tumors respond to treatment. This insight is essential for designing precise combination therapies tailored to the unique heterogeneity of patient tumors, addressing the single-cell variations that ultimately determine therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d948793b1aa3c6db446f2356cc375541bc62bfa6" target='_blank'>
              A Multimodal Framework to Uncover Drug-Responsive Subpopulations in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Yue Wang, Santiago Haase, Austin A. Whitman, Adriana Beltran, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="EMT converts epithelial (E) phenotypes to invasive mesenchymal (M) states. However, analyses of circulating tumor cells (CTCs) indicated that biphenotypic (E+M) CTCs better correlate with metastasis. Similarly, investigations of murine tumors undergoing EMT concluded that early E+M states posses the highest metastatic potential. To explore this, we selected in animals with breast cancer CTCs having progressively increasing intravasation abilities. This revealed that downregulation of arrestin Arrdc4 associates with CTC aggressiveness. In xenografts, depleting Arrdc4 accelerated tumor progression, whereas overexpression hindered progression in immunocompetent, but not in immunocompromised mice. Mechanistically, high Arrdc44 suppresses glucose uptake and enhances gasdermin E, triggering pyroptosis a type of pro-inflammatory cell death. Consistently, Arrdc4’s lowest levels characterize the most metastatic biphenotypic states. In patients, both epigenetic and chromosomal aberrations downregulate ARRDC4 and predict poor prognosis. In summary, the uncovered mechanism portrays pyroptosis of biphenotypic EMT cells as a rheostat of CTCs, which may resolve the controversy on the role played by EMT in metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad683c6747e49ef60cb44d3240bb40206d0d6b87" target='_blank'>
              Avoidance of pyroptosis accounts for the relatively high metastatic potential observed in early hybrid EMT states
              </a>
            </td>
          <td>
            Aakanksha Verma, Alessandro Genna, Yaron Vinik, N. Nataraj, Maha Abedrabbo, Boobash-Raj Selvadurai, Tithi Bhandari, Noa Aharoni, Palaniappan Ramesh, Boris Boguslavski, Diana Drago, T. Salame, Amir Prior, Yishai Levin, Eviatar Weizman, Rong Zhu, Carlos Caldas, O. Rueda, S. Lev, Yosef Yarden
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is the sixth most common cancer worldwide. HPV-negative HNSCC, which arises in the upper airway mucosa, is particularly aggressive, with nearly half of patients succumbing to the disease within five years and limited response to immune checkpoint inhibitors compared to other cancers. There is a need to further explore the complex immune landscape in HPV-negative HNSCC to identify potential therapeutic targets. Here, we integrated two single-cell RNA sequencing datasets from 29 samples and nearly 300,000 immune cells to investigate immune cell dynamics across tumor progression and lymph node metastasis. Notable shifts toward adaptative immune cell populations were observed in the 14 distinct HNSCC-associated peripheral blood mononuclear (PBMCs) and 21 tumor-infiltrating immune cells (TICs) considering disease stages. All PBMCs and TICs revealed unique molecular signatures correlating with lymph node involvement; however, broadly, TICs increased ligand expression among effector cytokines, growth factors, and interferon-related genes. Pathway analysis comparing PBMCs and TICs further confirmed active cell signaling among Monocyte-Macrophage, Dendritic cell, Natural Killer (NK), and T cell populations. Receptor-ligand analysis revealed significant communication patterns shifts among TICs, between CD8+ T cells and NK cells, showing heightened immunosuppressive signaling that correlated with disease progression. In locally invasive HPV-negative HNSCC samples, highly multiplexed immunofluorescence assays highlighted peri-tumoral clustering of exhausted CD8+ T and NK cells, alongside their exclusion from intra-tumoral niches. These findings emphasize cytotoxic immune cells as valuable biomarkers and therapeutic targets, shedding light on the mechanisms by which the HNSCC sustainably evades immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95cbfc686be119d8298fad054a42fbd71b039bc2" target='_blank'>
              The Single-Cell Landscape of Peripheral and Tumor-infiltrating Immune Cells in HPV- HNSCC
              </a>
            </td>
          <td>
            Rômulo Gonçalves Agostinho Galvani, Adolfo Rojas, B. Matuck, Brittany T. Rupp, Nikhil Kumar, Khoa L. A. Huynh, Carlos Alberto Oliveira de Biagi Júnior, Jinze Liu, Siddharth Sheth, Jelte Martinus Maria Krol, Vinicius Maracaja-Coutinho, K. Byrd, Patricia Severino
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Background: Spatial integration of metabolic pathways with single cell-level transcriptomic data in human tumor and normal tissue may elucidate mechanisms supporting tumorigenesis for therapeutic targeting. The suppressive adenosine signaling pathway is an emerging potential target for combination with radiotherapy. This study aims to create an in situ map of the pathway in non-small cell lung cancer (NSCLC) tumor and adjacent normal tissue. We seek to define how distinct cellular compartments cooperate and contribute to adenosine signaling within the tumor microenvironment (TME). Methods: Using Matrix-Assisted Laser Desorption/Ionization mass spectrometry imaging, we examined the spatial distribution of metabolites in paired human NSCLC compared to normal tissue from the same patient (n=15). In situ metabolite quantification and pathway enrichment analysis were performed. Second, we performed single nuclear RNA-sequencing (snRNA-seq) on NSCLC patient samples (n=4) to characterize the transcriptomic landscape of the tumors, including quantification of key members of the adenosine signaling pathway: ADORA2A (A2AR), ADORA2B (A2BR), ENTPD1 (CD39), NT5E (CD73), and ENPP1. Leveraging the average gene expression per cell subtype and pathologist-annotated H&E-stained tissue sections, we are integrating transcriptomic data with the spatial metabolomics to generate an in situ pathway map. Results: In NSCLC tissue, UMP, AMP, and ADP levels were significantly higher in tumor compared to normal tissue (p<0.0001), with mean percentage increases of 79%, 89%, and 73%, respectively. Metabolites downstream of adenosine, adenine and inosine, showed no significant differences (p>0.05). Single nuclear RNA-seq analyses revealed expression of the adenosine 2B receptor, ADORA2B, was highest in fibroblasts, lower in tumor, and absent in the immune compartment. ENTPD1 was expressed by all immune cells and tumor. NT5E was absent in T cells and ENPP1 was absent in macrophages, but expressed in all other immune compartments and tumor. Integration of snRNA-seq data with spatial metabolomics is underway to correlate metabolite abundance and gene expression within distinct cellular compartments to define cell type-specific contribution to adenosine signaling. Conclusion: Our study offers a multimodal approach to spatially resolve metabolic and transcriptomic data in situ relying on frozen NSCLC and normal lung tissue. This has significant translational implications, enabling analysis of biospecimens from multi-institutional clinical trials that precludes processing fresh tissue for single-cell analyses. Herein, we are able to localize key metabolites in untreated tumor and normal tissue and quantify the expression of pathway members at the cellular level. Through spatial integration of these data, we aim to decipher the relationship between distinct cell types within the TME and their contribution to adenosine-driven suppressive changes, then extend to radiotherapy-treated tumors.
 Citation Format: Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick J. McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z. Obradovic, Ramon C. Sun, Catherine S. Spina. {Abstract title} [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69f29862a6e18da7c70e3c9d22cd4b7cabc18a3" target='_blank'>
              Abstract B028: Multimodal approach to characterization of metabolism and immune modulation with insights from spatial metabolomics and transcriptomics
              </a>
            </td>
          <td>
            Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z Obradovic, Ramon C. Sun, Catherine S. Spina
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Myeloid cells in the tumor microenvironment (TME) exhibit diverse functions, from pro-tumor to anti-tumor. To elucidate myeloid heterogeneity, we developed a deep-learning model to annotate 20 myeloid subtypes in single-cell data. We identified two macrophage populations, macro_SPP1 and macro_C1QC, which were pan-cancer, enriched in tumors compared to normal tissues, and linked to clinical outcomes. Higher macro_SPP1 abundance correlated with IO non-responders, pre-IO treatment, and lower inflammation, while macro_C1QC showed the opposite trends. We developed and validated gene signatures to quantify these macrophages in bulk RNAseq. Applying these signatures to TCGA and real-world data, we confirmed that macro_SPP1 was associated with worse survival and poor IO response, whereas macro_C1QC was linked to better outcomes. Tumors with chromosome 9p21 deletions had higher macro_SPP1 and fibroblast signatures, suggesting a TME influence on macrophage heterogeneity. Cell-cell communication analysis in single-cell data revealed that fibroblasts sent the strongest signals to macrophages in lung and pancreatic cancers. Fibroblast-derived ligands such as FN1 and collagens exhibited stronger signaling to macro_SPP1 than macro_C1QC, and colocalized with macro_SPP1 spatially. Conversely, macro_SPP1 sent higher total signals to other cells in the TME than macro_C1QC, with SPP1 to fibroblasts and endothelial cells being the most differentiating signal. In summary, we identified two disease-critical macrophage populations: pro-tumor macro_SPP1 and anti-tumor macro_C1QC. We found interactions between macrophages and fibroblasts as key modulators of the TME.
 Citation Format: Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas. Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B086.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58e073c2b3967f3720af1b89697ce0e1aad33c4c" target='_blank'>
              Abstract B086: Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment
              </a>
            </td>
          <td>
            Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Bone is a multifaceted tissue requiring orchestrated interplays of diverse cells within specialized microenvironments. Although significant progress has been made in understanding cellular and molecular mechanisms of component cells of bone, revealing their spatial organization and interactions in native bone tissue microenvironment is crucial for advancing precision medicine, as they govern fundamental signaling pathways and functional dependencies among various bone cells. In this study, we present the first integrative high-resolution map of human bone and bone marrow, using spatial and single-cell transcriptomics profiling from femoral tissue. This multi-modal approach discovered a novel bone formation-specialized niche enriched with osteoblastic lineage cells and fibroblasts and unveiled critical cell–cell communications and co-localization patterns between osteoblastic lineage cells and other cells. Furthermore, we discovered a novel spatial gradient of cellular composition, gene expression and signaling pathway activities radiating from the trabecular bone. This comprehensive atlas delineates the intricate bone cellular architecture and illuminates key molecular processes and dependencies among cells that coordinate bone metabolism. In sum, our study provides an essential reference for the field of bone biology and lays the foundation for advanced mechanistic studies and precision medicine approaches in bone-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de859f69ecf65ef803fd91ebf44b36784f4b9786" target='_blank'>
              Mapping the spatial atlas of the human bone tissue integrating spatial and single-cell transcriptomics
              </a>
            </td>
          <td>
            Weiqiang Lin, Yisu Li, Chuan Qiu, Binghao Zou, Yun Gong, Xiao Zhang, Di Tian, William Sherman, Fernando Sanchez, Di Wu, Kuan-Jui Su, Xinyi Xiao, Zhe Luo, Qing Tian, Yiping Chen, Hui Shen, Hongwen Deng
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Understanding how radiotherapy induces metabolic reprogramming and reshapes the cytoarchitecture of the tumor microenvironment (TME) is essential for improving treatment outcomes and overcoming resistance. Despite its importance, no studies have used computational tools to quantitatively assess these cytoarchitectural rearrangements in the TME following radiotherapy. In previous work, we developed a quantitative method called “colocatome analysis,” a spatial -omic approach that catalogs statistically significant colocalizations between pairs of cell subpopulations or states. Through a normalization process, this method enables direct comparisons of spatial features across different conditions, including comparisons between in vitro models and tissue samples. We applied colocatome analysis to multiplexed immunofluorescence images of lung adenocarcinoma specimens, examining differences between naïve and recurrent tumors following radiotherapy using a panel of markers that represent primary tumor compartments (endothelial, malignant, fibroblast, and immune) and include markers for the hexosamine biosynthesis pathway (HBP), glycans, key metabolic enzymes, proliferation, cell death, DNA damage, hypoxia, and EMT. Our preliminary results indicate significant cytoarchitectural differences: tumors that recurred after radiotherapy exhibited a lower epithelial-to-mesenchymal cancer cell ratio and were enriched in FAP+ cancer-associated fibroblasts (CAFs). Although the overall proportions of macrophages and other immune cells remained consistent across samples, their spatial organization differed. In naïve tumors, immune cells were largely segregated from cancer cells, while recurrent tumors showed a more homogeneous mixture of cancer and immune cells. Additionally, CAFs were markedly increased in recurrent tumors. Lastly, cancer cells in naïve tumors displayed high levels of O-GlcNAc, which significantly decreased in recurrent tumors. These findings provide insights into the impact of metabolic reprogramming towards HBP in the TME and highlight its potential role in mediating radioresistance.
 Citation Format: Gina Bouchard.Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P019">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0ad7d9e2bbe3827fd09e473275b031d3cd8d0" target='_blank'>
              Abstract P019: Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy
              </a>
            </td>
          <td>
            Gina Bouchard
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 CD8+ T cell-tumor cell proximity correlates with immunotherapy response. However, it is unknown whether these cells can be captured as functional clusters from clinical samples. In defined co-cultures, tumor antigen-recognizing T cells outcompeted unmatched T cells in forming heterotypic clusters with tumor cells, showing >10-fold enrichment when comprising only 1% of the total T cell population, which was seen in across cancer cell lines. This prompted us to investigate whether this feature could be used to isolate tumor-reactive CD8+ T cells directly from cancer specimens. By regular and imaging flow cytometry, we could identify heterotypic clusters from 21/21 human melanoma metastases, comprising CD8+ T cells interacting with one or more tumor cells and/or antigen-presenting cells (APCs). These results prompted us to investigate whether these heterotypic T cell clusters could be used to better understand biology and whether they hold therapeutic potential. Regarding biology, we observed that CD8+ T cells from clusters isolated from clinical samples were significantly enriched for tumor-reactive and exhausted gene signatures. Integration with T cell receptor (TCR)-sequencing revealed increased clonality of clustered T cells, indicative of expansion. Whereas single CD8+ T cells showed enrichment of viral antigen-specific signatures, T cells from clusters were enriched for tumor antigen-reactivity signatures. In-depth single cell RNA analyses revealed conjugation of these T cells to tumor cells and different APC types. These were not random events, as both T cell-associated APCs and tumor cells were enriched for specific cell states and associated with differential cell-cell communication. CD8+ T cells carrying identical TCRs showed more exhaustion and co-inhibition when conjugated to APCs than to tumor cells. Regarding therapeutic opportunities, we found that upon rapid expansion (REP), CD8+ TIL from clusters exerted on average 9-fold increased melanoma-killing activity in ex vivo assays, accompanied by enhanced production of cytokines. Upon ACT, T cells from clusters showed superior in vivo killing of matched patient-derived xenografts (PDX), which was associated with more infiltration and activation than for ACT of single T cells. Together, these results demonstrate that tumor-reactive CD8+ T cells are enriched in functional clusters with tumor cells and/or APCs, and that they can be isolated and expanded from clinical samples. Being often excluded during sorting for single-cell studies, these distinct heterotypic CD8+ T cell clusters serve as a valuable source amenable to deciphering functional tumor-immune cell interactions, while they may also be therapeutically explored.
 Citation Format: Daniel S Peeper, Sofia Ibáñez-Molero, Johanna Veldman, Juan Simón-Nieto, Joleen Traets, Austin George, Kelly Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, Winan van Houdt. Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac4b19bdc51a77041e08cca430b7dd7ca4fef49" target='_blank'>
              Abstract PR007: Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity
              </a>
            </td>
          <td>
            D. Peeper, Sofía Ibáñez-Molero, Johanna Veldman, Juan Simon-Nieto, J. Traets, Austin George, K. Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, W. V. van Houdt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) represent a central cell population of the tumor microenvironment (TME). Recently, single‐cell RNA‐sequencing (scRNA‐seq) analyses of primary tumors of different cancer entities yielded different classifications of CAF subsets underscoring the heterogeneity of CAFs within the TME. Here, we analyzed the transcriptional signatures of approximately 8400 CAFs and normal fibroblasts by scRNA‐seq and compared genetic profiles of CAFs from murine melanoma primary tumors to CAFs from corresponding melanoma lung metastases. This revealed distinct subsets for primary tumor and metastasis‐specific CAF populations, respectively. Combined with the spatial characterization of metastasis CAFs at the RNA and protein level, scRNA analyses indicate tumor‐dependent crosstalk between neutrophils and CAFs, mediated via SAA3 and IL1b‐related signaling pathways, which can be recapitulated in vitro. Analyzing tissue sections of human patient samples, this interaction was found to be present in human melanoma metastasis. Taken together, our data highlight unique characteristics of metastasis CAFs with potential therapeutic impact for melanoma metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae4d264150cbc5dcfaa11bf8a1b1f48e7dcf511" target='_blank'>
              A Unique Signature for Cancer‐Associated Fibroblasts in Melanoma Metastases
              </a>
            </td>
          <td>
            Saskia Tauch, Joschka Hey, Bettina Kast, N. Gengenbacher, Lena Weiß, Melanie Sator‐Schmitt, S. Lohr, Alexander Brobeil, P. Schirmacher, Jochen Utikal, H. Augustin, C. Plass, Peter Angel
          </td>
          <td>2025-02-09</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="
 Purpose: The purpose of this study is to determine how T cell engagers (TCEs) impact the localization and phenotype of key T-cell subsets in preclinical solid tumor models. Background: TCEs are a class of immunotherapy that have demonstrated substantial benefit to patients with hematologic cancers and have shown evidence of clinical activity in solid tumor malignancies. Mechanisms of response to TCEs, particularly in solid tumors, are not well understood. Identification of diagnostic strategies to stratify patients that are more likely to respond is expected to enhance the clinical benefit of TCEs. Friedrich et al. (Cancer Cell 2023) found that effector CD8 T cell subsets significantly expanded in TCE responsive but not refractory patients in multiple myeloma using single cell RNA sequencing (scRNAseq). Furthermore, the increased abundance of exhausted T cells at baseline was associated with a worse prognosis. We hypothesize that analogous populations exist in preclinical solid tumor models, which have distinct localization and cellular interactions within the TCE treated tumor microenvironment. Methods: Either hCD3 transgenic C57BL/6J or wildtype FVB mice were implanted with either ID8-LyPD1 or Fo5-HER2 tumors. Mice were grouped at a tumor volume of ∼ 300mm^3 and treated with either 10mpk of LyPD1 TCE, 5mpk of HER2 TCE, or vehicle. scRNAseq data was produced at day 1 and 4 post-treatment according to the 10X gene expression kit protocol and was processed in R using Seurat. Flow cytometry data was generated using commercially available antibodies, acquired on a BD FACSymphony and analyzed using FlowJo. Xenium data was generated using a custom probe panel according to the 10X Xenium protocol and was processed in either Python or R using the Squidpy or Seurat packages. Results: In the ID8-LyPD1 TCE model, we identified proliferative effector (CD8.pe) and terminally differentiated (CD8.Tex) CD8 populations by scRNAseq. The abundance of CD8.pe increased 3-fold and CD8.Tex expanded 2-fold upon TCE treatment. We then validated the expansion of these CD8 subsets by flow cytometry. Notably, increases in the number of CD8.pe per gram of tumor significantly correlated with reductions in tumor volume. Due to the impact of TCE treatment on these CD8 subsets and the importance of CD8.pe in tumor outcomes, we profiled the localization and cellular interactions of CD8s in preclinical solid tumors. Cell-cell communication inference analysis suggested that CD8s were more frequently targets of cellular interactions in TCE treated tumors. Utilizing Xenium spatial transcriptomics we verified that CD8.pe and CD8.Tex were more abundant in TCE treated samples and that their interactions with other key cell subsets were enhanced by TCE treatment. Conclusions: In summary we found that key CD8 subsets were expanded by TCE treatment in preclinical solid tumor models, that the abundance of CD8.pe correlates with reduced tumor burden, and that TCE treatment promotes cellular interactions among CD8s and other key cell subsets in the tumor microenvironment.
 Citation Format: Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa McGinnis, Kevin Walsh, Weilan Ye. T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce737a0ad997a56048c24df55247b712e440df3d" target='_blank'>
              Abstract PR002: T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment
              </a>
            </td>
          <td>
            Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick S Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa Mcginnis, Kevin B Walsh, Weilan Ye
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, this study develops engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples reveal the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f073ff296b054e7e4c285a38a9d240d687627e67" target='_blank'>
              Genetically Engineered Brain Organoids Recapitulate Spatial and Developmental States of Glioblastoma Progression.
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H Kim, J. Millman
          </td>
          <td>2025-01-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Simple Summary Cancer stem cells (CSCs) are pivotal in tumor growth, recurrence, and resistance to standard therapies. This review delves into CSCs’ ability to self-renew and differentiate, fueling tumor heterogeneity and metastasis. It highlights critical signaling pathways like Wnt, Notch, and Hedgehog, which are essential for targeting CSCs. Emerging technologies, such as CRISPR/Cas9 and single-cell sequencing, promise to enhance CSC research and management. The review also explores the complex interaction between CSCs and the tumor microenvironment (TME), which influences therapeutic effectiveness. Addressing CSC-driven resistance, the article proposes advanced strategies for overcoming these challenges. It emphasizes the role of artificial intelligence in designing personalized, CSC-targeted therapies, ushering in a new era for precision medicine in oncology. This comprehensive approach aims to improve cancer treatment outcomes through innovative technologies and inter-disciplinary collaboration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99151138c6fc7abf05753313b772a7f6497b1128" target='_blank'>
              Deciphering the Role of Cancer Stem Cells: Drivers of Tumor Evolution, Therapeutic Resistance, and Precision Medicine Strategies
              </a>
            </td>
          <td>
            Mohamed El-Tanani, S. Rabbani, S. Satyam, I. Rangraze, A. Wali, Yahia El-Tanani, Alaa A. A. Aljabali
          </td>
          <td>2025-01-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Many tumors evolve through cellular mutation and selection, where subpopulations of cells (subclones) with shared ancestry compete for dominance. Introduction of next generation sequencing enables subclone identification using small somatic. However, there are several advantages to marking subclones with structural variants: they have greater functional impact, play a crucial role in late-stage tumors, and provide a more complete view of genomic instability driving tumor evolution. Here, we present SVCFit, a scalable method to estimate the cellular prevalence of somatic deletions, duplications and inversions. We demonstrate that cellular prevalence estimation can be improved by incorporating distinct read patterns for each structural variant type. Additionally, this improvement is achieved without prior knowledge of tumor purity, which is often inaccurate. Using a combination of simulated data and patient-derived metastatic samples with known mixture proportions, we show that our algorithm achieves significantly greater accuracy than state-of-the-art in estimating the structural variants cellular prevalence (p<0.05). The speed of SVCFit estimation from cost-effective bulk whole-genome sequencing (WGS) makes it well-suited for analyzing large cohorts of sequenced tumor samples, enhancing the accessibility of SV-based clonal reconstruction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e71e2d20d9e52879b8c2d8cc5014dd16b7a335" target='_blank'>
              SVCFit: Inferring structural variant cellular fraction in tumors
              </a>
            </td>
          <td>
            Yunzhou Liu, Jiaying Lai, Laura D. Wood, Rachel Karchin
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ea420e4a0256eeeba7153fed74e4eb3b92c386" target='_blank'>
              Spatial analysis of a complete DIPG-infiltrated brainstem reveals novel ligand-receptor mediators of tumour-to-TME crosstalk
              </a>
            </td>
          <td>
            Anja Kordowski, Onkar Mulay, Xiao Tan, Tuan Vo, Ulrich Baumgartner, Mellissa Maybury, Timothy Hassall, L. Harris, Quan Nguyen, B. Day
          </td>
          <td>2025-02-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The epidermal basal layer consists of two distinct progenitor populations contributing differentially to skin homeostasis or wound healing. However, their resilience to oncogenic insults is unknown. Here, we developed mouse models allowing lineage tracing and oncogene expression in both populations. Our findings support the existence of two long-lived and functionally distinct epidermal progenitors and define them as tumour-primed and tumour-resistant. The tumour-primed progenitors drive cutaneous squamous cell carcinoma (cSCC), within days from oncogenic expression resembling the rapid onset of BRAF-inhibitor-driven cSCC. The tumour-resistant progenitors transform gradually, requiring SOX2 activation. Once transformed, they share a transcriptomic profile revealing molecular hallmarks of epidermal transformation independently of the cell-of-origin or the oncogenic driving mutations. Importantly, SOX2 is expressed in a subset of cSCC patients hinting at its cell-of-origin. These highlight the need for an in-depth understanding of the cell-specific vulnerabilities that apply in each tissue to open new avenues for rational treatment. STATEMENT OF SIGNIFICANCE Using in vivo models, we define tumour-primed and tumour-resistant epidermal progenitors and reveal SOX2-driven reprogramming as a driver of tumorigenesis in resistant cells. SOX2 expression in a subset of cSCC patients links to cell-of-origin, uncovering a cell-specific vulnerability that advances our understanding of cancer initiation and progression in epithelial tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c6ee68b135f15e99b33463b50a273c96e7d816" target='_blank'>
              Epidermal tumour-resistant progenitors require SOX2-driven developmental reprogramming to drive tumorigenesis
              </a>
            </td>
          <td>
            Patricia P. Centeno, Christopher Chester, Catriona A. Ford, Rachel A Ridgway, P. Cammareri, Thomas Jamieson, Richard Marais, A. Campbell, Owen J. Sansom
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9547a039c0fe93efd8fd0787388be5b2aa0012a6" target='_blank'>
              Integrating spatial and single-cell transcriptomes reveals the role of COL1A2(+) MMP1(+/-) cancer-associated fibroblasts in ER-positive breast cancer
              </a>
            </td>
          <td>
            , , Shuo Wang, Yingxi Li, , Yao Tian, , , Long He, Xin Wang, , Yue Yu
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The spatial organization of adaptive immune cells within lymph nodes is critical for understanding immune responses during infection and disease. Here, we introduce AIR-SPACE, an integrative approach that combines high-resolution spatial transcriptomics with paired, high-fidelity long-read sequencing of T and B cell receptors. This method enables the simultaneous analysis of cellular transcriptomes and adaptive immune receptor (AIR) repertoires within their native spatial context. We applied AIR-SPACE to mouse popliteal lymph nodes at five distinct time points after Vaccinia virus footpad infection and constructed a comprehensive map of the developing adaptive immune response. Our analysis revealed heterogeneous activation niches, characterized by Interferon-gamma (IFN-γ) production, during the early stages of infection. At later stages, we delineated sub-anatomical structures within the germinal center (GC) and observed evidence that antibody-producing plasma cells differentiate and exit the GC through the dark zone. Furthermore, by combining clonotype data with spatial lineage tracing, we demonstrate that B cell clones are shared among multiple GCs within the same lymph node, reinforcing the concept of a dynamic, interconnected network of GCs. Overall, our study demonstrates how AIR-SPACE can be used to gain insight into the spatial dynamics of infection responses within lymphoid organs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ff2bb39af0926fb7ea6baef50f5c517a318f30" target='_blank'>
              A Temporal and Spatial Atlas of Adaptive Immune Responses in the Lymph Node Following Viral Infection
              </a>
            </td>
          <td>
            Shaowen Jiang, Madhav Mantri, Viviana I. Maymi, S. Leddon, Peter Schweitzer, Subash Bhandari, Chase Holdener, Ioannis Ntekas, Christopher Vollmers, A. Flyak, Deborah J. Fowell, Brian D. Rudd, I. De Vlaminck
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="To overcome the paucity of known tumor-specific surface antigens in pediatric high-grade glioma (pHGG), we contrasted splicing patterns in pHGGs and normal brain samples. Among alternative splicing events affecting extracellular protein domains, the most pervasive alteration was the skipping of ≤30 nucleotide-long microexons. Several of these skipped microexons mapped to L1-IgCAM family members, such as NRCAM. Bulk and single-nuclei short- and long-read RNA-seq revealed uniform skipping of NRCAM microexons 5 and 19 in virtually every pHGG sample. Importantly, the Δex5Δex19 (but not the full-length) NRCAM proteoform was essential for pHGG cell migration and invasion in vitro and tumor growth in vivo. We developed a monoclonal antibody selective for Δex5Δex19 NRCAM and demonstrated that “painting” of pHGG cells with this antibody enables killing by T cells armed with an FcRI-based universal immune receptor. Thus, pHGG-specific NRCAM and possibly other L1-IgCAM proteoforms are promising and highly selective targets for adoptive immunotherapies. Statement of significance Existing targets for chimeric antigen receptors (CAR)-armed T cells are often shared by CNS tumors and normal tissues, creating the potential for on-target/off-tumor toxicities. Here we demonstrate that in CNS tumors of glial origin, cell adhesion molecules have alternatively spliced proteoforms, which could be targeted by highly selective therapeutic antibodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e54241611f14a49622f3cf892c1a58f5571d1d7" target='_blank'>
              Neuronal cell adhesion molecule (NRCAM) variant defined by microexon skipping is an essential, antigenically distinct, and targetable proteoform in high-grade glioma
              </a>
            </td>
          <td>
            Priyanka Sehgal, Ammar S. Naqvi, Makenna Higgins, Jiageng Liu, Kyra Harvey, Julien Jarroux, Taewoo Kim, Berk Mankaliye, Pamela Mishra, Grace Watterson, Justyn Fine, Jacinta Davis, Katharina E. Hayer, Annette Castro, Adanna Mogbo, Charles Drummer, Dan Martinez, Mateusz P. Koptyra, Zhiwei Ang, Kai Wang, Alvin Farrel, Mathieu Quesnel-Vallières, Yoseph Barash, Jamie B. Spangler, J. L. Rokita, A. Resnick, Hagen U. Tilgner, Thomas DeRaedt, Daniel J Powell, A. Thomas-Tikhonenko
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Quiescence, a reversible state of cell-cycle arrest, is an adaptive feature of many adult tissue stem cells, including those in the adult brain. In gliomas, brain tumour stem cells that reside in a quiescent state preferentially survive chemotherapy and radiotherapy, highlighting their critical role in therapy resistance and disease progression. To date, it remains unclear whether the molecular programs governing these states are functionally conserved between neural stem cells and brain tumour stem cells. Here, we establish novel in vitro models to study quiescence and find that glioma stem cells are markedly more resistant to entering quiescence than neural stem cells, suggesting that glioma stem cell quiescence more closely resembles a slow-cycling phenotype or shallow quiescence. Nonetheless, direct comparison of quiescent neural stem cells and quiescent/slow-cycling glioma stem cells, as they transition towards proliferation, reveals conserved gene expression trajectories, indicating shared molecular mechanisms. Furthermore, we find that pathways influencing quiescence in neural stem cells exert similar effects in glioma stem cells, underscoring the functional parallels between these populations. Finally, we identify that inhibition of TGF-β signalling might provide an avenue to improve current standard-of-care treatments by targeting quiescent glioma stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfaba977c1ae379d70e521eff155f6c1e15b6182" target='_blank'>
              Shared molecular regulation of quiescence in neural and glioma stem cells reveals therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Chandra Choudhury, Matthew Singleton, Stephanie Brauer, Dana Friess, Jessica Hart, N. Skarne, Karthik Pullela, Likai Mao, Bryan W Day, Lachlan Harris
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The CHI3L1 signaling pathway significantly influences glioma angiogenesis, but its role in the tumor microenvironment (TME) remains elusive. We propose a novel CHI3L1-associated vascular phenotype classification for glioma through integrative analyses of multiple datasets with bulk and single-cell transcriptome, genomics, digital pathology, and clinical data. We investigated the biological characteristics, genomic alterations, therapeutic vulnerabilities, and immune profiles within these phenotypes through a comprehensive multi-omics approach. We constructed the vascular-related risk (VR) score based on CHI3L1-associated vascular signatures (CAVS) identified by machine learning algorithms. Utilizing unsupervised consensus clustering, gliomas were stratified into three distinct vascular phenotypes: Cluster A, marked by high vascularization and stromal activation with a relatively low levels of tumor-infiltrating lymphocytes (TILs); Cluster B, characterized by moderate vascularization and stromal activity, coupled with a high density of TILs; and Cluster C, defined by low vascularization and sparse immune cell infiltration. We observed that the CAVS effectively indicated glioma-associated angiogenesis and immune suppression by single-cell RNA-seq analysis. Moreover, the high-VR-score group exhibited enhanced angiogenic activity, reduced immune response, resistance to immunotherapy, and poorer clinical outcomes. The VR score independently predicted glioma prognosis and, combined with a nomogram, provided a robust clinical decision-making tool. Potential drug prediction based on transcription factors for high-risk patients was also performed. Our study reveals that CHI3L1-associated vascular phenotypes shape distinct immune landscapes in gliomas, offering insights for optimizing therapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec81235338a787d945989660c2ddefa3be4c0bd" target='_blank'>
              Novel CHI3L1-Associated Angiogenic Phenotypes Define Glioma Microenvironments: Insights From Multi-Omics Integration.
              </a>
            </td>
          <td>
            Yu-Hang Zhao, Yu-Xiang Cai, Zhi-yong Pan, F. Tang, Chao Ma, Ze-Fen Wang, Gang Li, Hang Chang, Su-Fang Tian, Zhi-Qiang Li
          </td>
          <td>2025-02-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95f806851975b88a84521b58227335a097ae890e" target='_blank'>
              Spatial transcriptomics in autoimmune rheumatic disease: potential clinical applications and perspectives
              </a>
            </td>
          <td>
            Atsuko Tsujii Miyamoto, Hiroshi Shimagami, Atsushi Kumanogoh, Masayuki Nishide
          </td>
          <td>2025-02-20</td>
          <td>Inflammation and Regeneration</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47aaf83b6567ca271ef456f540a98ac5c9738676" target='_blank'>
              Comparative analysis of pediatric SHH medulloblastoma DAOY spheres and adherent monolayers: implications for medulloblastoma research
              </a>
            </td>
          <td>
            Jana Klementová, Šárka Jarošová, Irina Danilová, Markéta Farníková, Josef Novotný, M. Davidkova, Martina Zíková
          </td>
          <td>2025-01-22</td>
          <td>Cancer Cell International</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary This study explores the potential of canine patient-derived organoids (PDOs) in identifying mammary tumor-specific weaknesses by comparing tumor cells with matching normal mammary PDOs from the same patient. For this purpose, two customized CRISPR/Cas9 screens were performed to identify genes that are essential for tumor cells to survive but dispensable for normal mammary epithelial cells. CDK2 was subsequently identified as an important vulnerability in PDOs derived from a canine mammary tumor patient, which can be targeted by an existing inhibitor, PF3600. Although we only used tumors and normal PDOs from a single patient, our approach exemplifies how to discover novel, more individualized anti-cancer treatments for dogs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a341795bab91ddb8ebd61b09b4f9f74bbe497648" target='_blank'>
              Individualized Pooled CRISPR/Cas9 Screenings Identify CDK2 as a Druggable Vulnerability in a Canine Mammary Carcinoma Patient
              </a>
            </td>
          <td>
            Marine Inglebert, Martina Dettwiler, Chang He, Enni Markkanen, Lennart Opitz, A. Naguleswaran, Sven Rottenberg
          </td>
          <td>2025-02-01</td>
          <td>Veterinary Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Therapeutic drug treatments of solid tumors are often undermined by various resistance mechanisms. Identification of drug-resistance phenotypes at the single cell level is challenging because conventional molecular methods are cell-destructive, labor-intensive, and cost-prohibitive. To overcome these challenges, we developed an orthogonal approach to drug-resistance phenotyping, through the use of deep-learning-driven morphology analysis of single, high resolution cell images. Specifically, we trained deep learning-based drug resistance classifiers using cell images from 5 different cell lines that were rendered resistant to 5 different therapeutic agents, using a foundation model framework. With high accuracy, the classifier correctly predicted naive or resistance phenotypes across all cancer types and across all the therapeutic agent types (chemotherapeutic, targeted) tested. These results showed that morphology can capture complex phenotype information in the context of drug treatment. To demonstrate the potential clinical utility of the drug resistance classifier, it was applied to a dissociated tumor biopsy and the resulting phenotype predictions were in close concordance with scRNASeq analysis of the biopsy. Our study highlights the potential of deep-learning-driven morphology analysis to provide complex phenotype information, and ultimately shape oncology drug treatment strategies at the patient-level in a clinical context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65e32bdcccdadfa02831a9867b05cc627a1045f4" target='_blank'>
              Deep learning-driven morphology analysis enables label-free classification of therapeutic agentnaive versus resistant cancer cells
              </a>
            </td>
          <td>
            Cyril Y Ramathal, K. Saini, Zhouyang Lian, Christian Corona, T. Pham, Ryan Carelli, Stephane C. Boutet, Manisha Ray, Sunantha Sethuraman, Vivian Prindle, Evelyn Lattmann, Maria-Grazia Molvetti, A. Dzung, Mitchell P. Levesque, Matt Barnes, Andreja Jovic
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Human genome sequencing efforts in healthy and diseased individuals continue to identify a broad spectrum of genetic variants associated with predisposition, progression, and therapeutic outcomes for diseases like cancer1–6. Insights derived from these studies have significant potential to guide clinical diagnoses and treatment decisions; however, the relative importance and functional impact of most genetic variants remain poorly understood. Precision genome editing technologies like base and prime editing can be used to systematically engineer and interrogate diverse types of endogenous genetic variants in their native context7–9. We and others have recently developed and applied scalable sensor-based screening approaches to engineer and measure the phenotypes produced by thousands of endogenous mutations in vitro10–12. However, the impact of most genetic variants in the physiological in vivo setting, including contextual differences depending on the tissue or microenvironment, remains unexplored. Here, we integrate new cross-species base editing sensor libraries with syngeneic cancer mouse models to develop a multiplexed in vivo platform for systematic functional analysis of endogenous genetic variants in primary and disseminated malignancies. We used this platform to screen 13,840 guide RNAs designed to engineer 7,783 human cancer-associated mutations mapping to 489 endogenous protein-coding genes, allowing us to construct a rich compendium of putative functional interactions between genes, mutations, and physiological contexts. Our findings suggest that the physiological in vivo environment and cellular organotropism are important contextual determinants of specific gene-variant phenotypes. We also show that many mutations and their in vivo effects fail to be detected with standard CRISPR-Cas9 nuclease approaches and often produce discordant phenotypes, potentially due to site-specific amino acid selection- or separation-of-function mechanisms. This versatile platform could be deployed to investigate how genetic variation impacts diverse in vivo phenotypes associated with cancer and other genetic diseases, as well as identify new potential therapeutic avenues to treat human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f71f3527a2a35f380d7482a2555f7e6eeef4ab" target='_blank'>
              Multiplexed in vivo base editing identifies functional gene-variant-context interactions
              </a>
            </td>
          <td>
            Jonuelle Acosta, Grace A. Johnson, Samuel I. Gould, Kexin Dong, Yovel Lendner, Diego Detrés, Ondine Atwa, Jari Bulkens, Samuel Gruber, Manuel E. Contreras, Alexandra N. Wuest, Varun K. Narendra, Michael T. Hemann, Francisco J. Sánchez-Rivera
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36c2c77f99d55aa9b36b1f371ad8b8cec2aac50" target='_blank'>
              Evolutionary fingerprints of epithelial-to-mesenchymal transition.
              </a>
            </td>
          <td>
            Luigi Perelli, Li Zhang, Sarah Mangiameli, Francesca Giannese, K. Mahadevan, F. Peng, F. Citron, Hania Khan, Courtney Le, Enrico Gurreri, F. Carbone, Andrew J. C. Russell, M. Soeung, T. N. Lam, Sebastian Lundgren, Sujay Marisetty, Cihui Zhu, Desiree Catania, Alaa M T Mohamed, Ningping Feng, J. Augustine, A. Sgambato, G. Tortora, Giulio F. Draetta, Giovanni Tonon, Andrew Futreal, Virginia Giuliani, A. Carugo, A. Viale, Michael P. Kim, T. Heffernan, Linghua Wang, Raghu Kalluri, D. Cittaro, Fei Chen, G. Genovese
          </td>
          <td>2025-03-05</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33631c265f68921d79fbd004eb59ad3a5ea83b81" target='_blank'>
              Spatially mapping the tumour immune microenvironments of non-small cell lung cancer
              </a>
            </td>
          <td>
            Lysanne Desharnais, Mark Sorin, M. Rezanejad, Bridget Liu, E. Karimi, Aline Atallah, Anikka M Swaby, Miranda W. Yu, Samuel Doré, Saskia Hartner, B. Fiset, Yuhong Wei, Baharak Kadang, R. Rayes, Philippe Joubert, Sophie Camilleri-Broët, P. Fiset, D. Quail, Jonathan D Spicer, L. Walsh
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Simple Summary Multiple myeloma (MM) is a blood cancer of bone marrow that is difficult to diagnose and treat because it changes over time and has complex interactions with the body’s immune system. This review explores how single-cell technologies help scientists understand multiple myeloma more comprehensively by identifying hidden tumor clones, tracking how the disease evolves, and discerning why some treatments stop working. Researchers have found that certain genetic changes, like mutations or deletion of TP53, loss of chromosome 13, and gain of chromosome 1, make the cancer more resistant to treatment, and that the immune system often fails to fight off MM because there is a suppressive environment around the tumor. New treatments, like Chimeric Antigen Receptor T cell (CAR-T) cell therapy and Bispecific T cell engagers (BiTE), take advantage of the immune microenvironment and are showing promise in targeting myeloma cell directly. However, they are not always effective and can have adverse effects. By studying single-cell resolution data to identify new vulnerabilities in myeloma cells and pathways related to resistance that can be different within the cells in a single patient, new treatment regimens can be identified and tailored for individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b4ede18b0be075e37d640173950681509758ae8" target='_blank'>
              Multiple Myeloma Insights from Single-Cell Analysis: Clonal Evolution, the Microenvironment, Therapy Evasion, and Clinical Implications
              </a>
            </td>
          <td>
            Sihong Li, Jiahui Liu, Madeline Peyton, Olivia Lazaro, Sean D. McCabe, Xiaoqing Huang, Yunlong Liu, Zanyu Shi, Zhiqi Zhang, Brian A. Walker, Travis S. Johnson
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183356cbf79c71bfdd985789fe82d5c12c548853" target='_blank'>
              Gemistocytic tumor cells programmed for glial scarring characterize T cell confinement in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            L. van Hijfte, M. Geurts, I. D. de Heer, S. Ghisai, H. Balcioglu, Y. Hoogstrate, Wies R Vallentgoed, R. Head, Rosa Luning, Thierry van den Bosch, Bart A Westerman, Pieter Wesseling, J. Joyce, P. French, R. Debets
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Spatial transcriptomics (ST) is revolutionizing our understanding of tumor heterogeneity by providing high-resolution, location-specific gene expression mapping across the tumor microenvironment (TME). Despite its transformative potential, ST’s high cost limits its widespread clinical application, particularly for large-scale biomarker discovery. Methods: To address these challenges, we developed Path2Space, a deep learning model trained on ST data, to predict spatial gene expression directly from histopathology slides, offering an accessible alternative to experimental ST assays. First, Path2Space is validated using three independent spatial transcriptomics (ST) breast cancer cohorts, demonstrating robust and consistent gene expression prediction across these datasets. Second, the model’s predictions allow for detailed TME characterization, including the accurate inference of cell-type abundances and immune cell distributions within spatial domains. Thirdly and most importantly, we applied Path2Space to the TCGA breast cancer cohort to assess the prognostic potential of immune and TME spatial patterns. Results: Path2Space achieves unprecedented accuracy and coverage in robustly predicting the spatial expression of 4,500 genes across three independent spatial transcriptomics (ST) cohorts, markedly outperforming current methods by a factor of 10. Applied to the TCGA breast cancer cohort, we identify three patient subsets with distinct immune tumor microenvironments (TME) having distinct survival outcomes. The immune TME of the poor-prognosis subset has dense cancer cell populations with low immune infiltration and marked inactivation of specific key immune interactions. In two trastuzumab-treated breast cancer cohorts, Path2Space identifies spatial domains strongly associated with treatment response, with high tumor-infiltrating lymphocytes (TIL) presence, activation of immune pathways, and robust immune-driven cell-cell interactions, enabling the prediction of patient response more accurately than that based on the measured gene expression. Conclusions: Path2Space represents a groundbreaking approach in spatially mapping the tumor microenvironment directly from histopathology slides, paving the way for the study of the immune TME of different cancer types on an unprecedented scale. By enabling the cost-effective inference of the TME, Path2Space holds potential to transform cancer care across diverse healthcare settings, facilitating the development of targeted treatments and improving patient survival globally.
 Citation Format: Roshan Lodha, Eldad D Shulman, Emma M Campagnolo, Thomas Cantore, Tom Hu, MacLean Nasrallah, Danh-Tai Hoang, Kenneth Aldape, Eytan Ruppin. Predicting spatial transcriptomics from histopathology with Path2Space: AI charts breast cancer patients tumor immune microenvironment on an unprecedented scale [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B105.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20c3f321acba286bf84a8f4af78044117acbf8c" target='_blank'>
              Abstract B105: Predicting spatial transcriptomics from histopathology with Path2Space: AI charts breast cancer patients tumor immune microenvironment on an unprecedented scale
              </a>
            </td>
          <td>
            Roshan Lodha, E. Shulman, Emma M. Campagnolo, Thomas Cantore, Tom Hu, M. Nasrallah, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb406f40baccfe449ee7910f107c5384df35a006" target='_blank'>
              Spatially resolved transcriptomics reveals gene expression characteristics in uveal melanoma
              </a>
            </td>
          <td>
            Jing-Ying Xiu, Yu-Ning Chen, Ya-Li Mao, Jingqiu Luo, Hao-Wen Li, Yang Li, Wenbin Wei
          </td>
          <td>2025-01-15</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef28d07b971b4246515b216a14089f7d091622f6" target='_blank'>
              Single-cell transcriptomics unveils multifaceted immune heterogeneity in early-onset versus late-onset cervical cancer
              </a>
            </td>
          <td>
            Qian Chen, Dongfeng Deng, Hong Zhu, Shan Li
          </td>
          <td>2025-01-14</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) arises primarily through neuroendocrine transdifferentiation (NEtD) as an adaptive mechanism of therapeutic resistance. Models to define the functional effects of putative drivers of this process on androgen receptor (AR) signaling and NE cancer lineage programs are lacking. We adapted a genetically defined strategy from the field of cellular reprogramming to directly convert AR-active prostate cancer (ARPC) to AR-independent NEPC using candidate factors. We delineated critical roles of the pioneer factors ASCL1 and NeuroD1 in NEtD and uncovered their abilities to silence AR expression and signaling by remodeling chromatin at the somatically acquired AR enhancer and global AR binding sites with enhancer activity. We also elucidated the dynamic temporal changes in the transcriptomic and epigenomic landscapes of cells undergoing acute lineage conversion from ARPC to NEPC which should inform future therapeutic development. Further, we distinguished the activities of ASCL1 and NeuroD1 from the inactivation of RE-1 silencing transcription factor (REST), a master suppressor of a major neuronal gene program, in establishing a NEPC lineage state and in modulating the expression of genes associated with major histocompatibility complex class I (MHC I) antigen processing and presentation. These findings provide important, clinically relevant insights into the biological processes driving NEtD of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b450ad259d614ef1f0c3aeece63e1bbfcda82938" target='_blank'>
              Defined cellular reprogramming of androgen receptor-active prostate cancer to neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Shan Li, Kai Song, Huiyun Sun, Yong Tao, Arthur Huang, Vipul Bhatia, B. Hanratty, Radhika A Patel, Henry W. Long, C. Morrissey, Michael C Haffner, Peter S. Nelson, Thomas G. Graeber, John K. Lee
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="BACKGROUND
Neuroblastoma is a heterogeneous disease with adrenergic (ADRN)- and therapy resistant mesenchymal (MES)-like cells driven by distinct transcription factor networks. Here, we investigate the expression of immunotherapeutic targets in each neuroblastoma subtype and propose pan-neuroblastoma and cell state specific targetable cell-surface proteins.


METHODS
We characterized cell lines, patient-derived xenografts, and patient samples as ADRN-dominant or MES-dominant to define subtype-specific and pan-neuroblastoma gene sets. Targets were validated with ChIP-sequencing, immunoblotting, and flow cytometry in neuroblastoma cell lines and isogenic ADRN-to-MES transition cell line models. Finally, we evaluated the activity of MES-specific agents in vivo and in vitro.


RESULTS
Most immunotherapeutic targets being developed for neuroblastoma showed significantly higher expression in the ADRN subtype with limited expression in MES-like tumor cells. In contrast, CD276 (B7-H3) and L1CAM maintained expression across both ADRN and MES states. We identified several receptor tyrosine kinases (RTKs) enriched in MES-dominant samples and showed that AXL targeting with ADCT-601 was potently cytotoxic in MES-dominant cell lines and showed specific anti-tumor activity in a MES cell line-derived xenograft.


CONCLUSIONS
Immunotherapeutic strategies for neuroblastoma must address the potential of epigenetic downregulation of antigen density as a mechanism for immune evasion. We identified several RTKs as candidate MES-specific immunotherapeutic target proteins for the elimination of therapy-resistant cells. We hypothesize that the phenomena of immune escape will be less likely when targeting pan-neuroblastoma cell surface proteins such as B7-H3 and L1CAM, and/or dual targeting strategies that consider both the ADRN- and MES-cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ce67e86c3ed1eb1595b4d5bed2ad026b261bd59" target='_blank'>
              Lineage-dependence of the neuroblastoma surfaceome defines tumor cell state-dependent and independent immunotherapeutic targets.
              </a>
            </td>
          <td>
            N. Kendsersky, Michal Odrobina, Nathaniel W. Mabe, Alvin Farrel, Liron D. Grossmann, M. Tsang, David Groff, A. Wolpaw, Alaa Narch, F. Zammarchi, P. V. van Berkel, C. V. Dang, Y. Mossé, K. Stegmaier, John M. Maris
          </td>
          <td>2025-01-18</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Disruptions in uterine tissue function contribute to disorders such as endometriosis, adenomyosis, endometrial cancer, and fibroids, which all significantly impact health and fertility. Advances in transcriptomics, particularly single-cell RNA sequencing, have revolutionized uterine biological research by revealing the cellular heterogeneity and molecular mechanisms underlying disease states. Single-cell RNA sequencing and spatial transcriptomics have mapped endometrial and myometrial cellular landscapes, which helped to identify critical cell types, signaling pathways, and phase-specific dynamics. Said transcriptomic technologies also identified stromal and immune cell dysfunctions, such as fibroblast-to-myofibroblast transitions and impaired macrophage activity, which drive fibrosis, chronic inflammation, and lesion persistence in endometriosis. For endometrial cancer, scRNA-seq uncovered tumor microenvironmental complexities, identifying cancer-associated fibroblast subtypes and immune cell profiles contributing to progression and therapeutic resistance. Similarly, studies on adenomyosis highlighted disrupted signaling pathways, including Wnt and VEGF, and novel progenitor cell populations linked to tissue invasion and neuroinflammation, while single-cell approaches characterized smooth muscle and fibroblast subpopulations in uterine fibroids, elucidating their roles in extracellular matrix remodeling and signaling pathways like ERK and mTOR. Despite challenges such as scalability and reproducibility, single-cell transcriptomic approaches may have potential applications in biomarker discovery, therapeutic target identification, and personalized medicine in gynecological disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d46c8d721f3ec856f042d24cd830de014f3393b" target='_blank'>
              Mapping Human Uterine Disorders Through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Sandra Boldu-Fernández, Carolina Lliberos, Carlos Simón, A. Mas
          </td>
          <td>2025-01-21</td>
          <td>Cells</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The Imaging and Molecular Annotation of Xenografts and Tumors Cancer Grand Challenges team was set up with the objective of developing the "next generation" of pathology and cancer research by using a combination of single-cell and spatial omics tools to produce 3D molecularly annotated maps of tumors. Its activities overlapped, and in some cases catalyzed, a spatial revolution in biology that saw new technologies being deployed to investigate the roles of tumor heterogeneity and of the tumor micro-environment. See related article by Stratton et al., p. 22 See related article by Bhattacharjee et al., p. 28 See related article by Goodwin et al., p. 34.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9c9c61637383cf6c8f3f96d10f0ca3cac35ac4d" target='_blank'>
              Cancer Research in the Age of Spatial Omics: Lessons from IMAXT.
              </a>
            </td>
          <td>
            D. Bressan, Nicholas Walton, Gregory J. Hannon
          </td>
          <td>2025-01-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be893eec80147faf7d7cca1af35e6472e40e7cc" target='_blank'>
              Statistical identification of cell type-specific spatially variable genes in spatial transcriptomics
              </a>
            </td>
          <td>
            Lulu Shang, Peijun Wu, Xiang Zhou
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="We investigated the cellular and molecular consequences of accidental cell fusion between mHL1 and mMSC cell lines using previously published single-cell RNA sequencing (scRNA-seq) data. We characterized the resulting cell types, their communication patterns, and underlying gene regulatory networks. Initial analyses identified four distinct cell types (mHL1, mHL1 fusion, mMSC, and mMSC fusion), which were also consolidated into three clusters via unsupervised learning. Differential gene expression analysis revealed diverse cell type associations with shared and exclusive gene expression. Single-cell compatible weighted gene co-expression network analysis (WGCNA) linked specific gene modules to distinct cell types and associated biological processes, including muscle contraction and metabolism in mHL1/mHL1 fusion, and extracellular matrix (ECM) formation and lipid metabolism in mMSC fusion. Cell-cell communication analysis revealed distinct intercellular communication patterns, with mMSC primarily acting as a ligand and mHL1 as both ligand and receptor. Reducing cell type complexity progressively simplified communication networks and altered pathway enrichment, while ECM-receptor pathways remained largely unchanged, with only minor shifts in ligand-receptor pairs. Interestingly, removing parental cells from the analysis altered cell type assignments, underscoring the influence of parental cell presence on fusion outcomes. Finally, gene regulatory network analysis identified key transcription factors (TFs) driving cell identity, distinguishing between master regulators and TFs with high cell-type specificity. These findings demonstrate the power of integrating multiple inter- and intra-cellular communication analysis approaches to dissect the complex interplay between cell type, communication, and gene regulation in cell fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0045e60f5f09ff48941f2c2a8009799ec50aa5d7" target='_blank'>
              Stem Cell-Parenchymal Fusion: Communication and Gene Regulation
              </a>
            </td>
          <td>
            JoAnne Huang, Fateme Nazaryabrbekoh, Joohyun Kim, Jangwook P. Jung
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4de4d7796ed3b7e2389f61c7edf7191dc271c5b5" target='_blank'>
              State-of-the-Art Liver Cancer Organoids: Modeling Cancer Stem Cell Heterogeneity for Personalized Treatment.
              </a>
            </td>
          <td>
            Julien Giron-Michel, Maël Padelli, Estelle Oberlin, Hind Guenou, Jean-Charles Duclos-Vallée
          </td>
          <td>2025-01-18</td>
          <td>BioDrugs : clinical immunotherapeutics, biopharmaceuticals and gene therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly malignant tumor in women, characterized by high morbidity, mortality, and recurrence rates. Although surgical treatment, radiotherapy, and chemotherapy are the mainstays of current treatment methods, the high heterogeneity of TNBC results in unsatisfactory outcomes with low 5-year survival rates. Rapid advancements in omics technology have propelled the understanding of TNBC molecular biology. The emergence of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has significantly enhanced knowledge of tumor heterogeneity and the distribution, functionality, and intercellular interactions of various cell types within the tumor microenvironment, including tumor cells, T cells, B cells, macrophages, and fibroblasts. The present study provides an overview of the technical characteristics of scRNA-seq and ST, highlighting their applications in exploring TNBC heterogeneity, cell spatial distribution patterns, and intercellular interactions. This review aims to enhance the comprehension of TNBC at the cellular level for the development of effective therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db2a16a0771f525b51537059ff8768102022813" target='_blank'>
              Analysis of single-cell and spatial transcriptomics in TNBC cell-cell interactions
              </a>
            </td>
          <td>
            Yan Xin, Qiji Ma, Qiang Deng, Tielin Wang, Dongxu Wang, Gang Wang
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2ac9b04d746f5cf36d236d7ac9a449435141cc" target='_blank'>
              Spatial transcriptomics reveals prognostically LYZ+ fibroblasts and colocalization with FN1+ macrophages in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            L. Dai, N. Lou, Liling Huang, Lin Li, Le Tang, Yuankai Shi, Xiaohong Han
          </td>
          <td>2025-02-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0116e059872731a0003137617d95a907ab32d870" target='_blank'>
              Molecular landscape of tumor-associated tissue-resident memory T cells in tumor microenvironment of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Mi-So Park, Hyeonbin Jo, Hyeree Kim, Ji Young Kim, Woong-Yang Park, Yong-Han Paik, Yeup Yoon, Wonseok Kang, Hong-Hee Won
          </td>
          <td>2025-02-11</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Several studies have attempted to find the initiating drivers of small intestinal neuroendocrine tumors (SI-NET) development and the molecular mechanisms driving progression and metastatic spread. For gene expression studies using bulk microarrays and RNA sequencing, researchers commonly use normal intestinal mucosa as a control. The intestine is made up of several different cell types, and using bulk RNA-seq may generate findings that reflect factors other than the tumor transformation. This could potentially contribute to the lack of discoverable treatments and prevention strategies for SI-NETs. We performed scRNA-seq on tissue from two patients that had tumor resection surgery and separated the EC cells from the normal intestinal mucosa and used specific markers to compare them against SI-NET tumors cells. We pinpointed new genes associated with chr18 haploinsufficiency but also others with loss or gain of expression that have not previously been associated with SI-NETs, and which could be potential targets for further functional genomic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35da01c8609a7028b2e2ec26d0b8011e3cf58489" target='_blank'>
              A single-cell transcriptomic comparison between small intestinal neuroendocrine tumors and their progenitor
              </a>
            </td>
          <td>
            Fredrik Axling, Elham Barazeghi, Per Hellman, O. Norlén, S. Backman, P. Stålberg
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f8ba9e3fcdae65a1005fa8694699f4a2467c8be" target='_blank'>
              Intratumour oxidative hotspots provide a niche for cancer cell dissemination.
              </a>
            </td>
          <td>
            Yoshifumi Ueda, Shigeki Kiyonaka, L. Selfors, Keisuke Inoue, Hiroshi Harada, Tomohiro Doura, Kunishige Onuma, Makoto Uchiyama, Ryuhei Kurogi, Yuji Yamada, Jiacheng H Sun, Reiko Sakaguchi, Yuki Tado, Haruki Omatsu, Harufumi Suzuki, M. Aoun, Takahiro Nakayama, Taketoshi Kajimoto, Tetsuya Yano, Rikard Holmdahl, I. Hamachi, Masahiro Inoue, Yasuo Mori, Nobuaki Takahashi
          </td>
          <td>2025-02-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c89aac6a693cf7fc84e3a1fc45266835eb27a4d" target='_blank'>
              Fusion of tumor cells and mesenchymal stem/stroma cells: a source of tumor heterogeneity, evolution and recurrence.
              </a>
            </td>
          <td>
            Zhen Liu, Yihao Wang, Zesheng Peng, Hui Li, Haofei Wang, Yuyi Wu, Xiaobing Jiang, Peng Fu
          </td>
          <td>2025-01-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0c4d7e98a1c7e44d04ba5fa342a0635434e2cbe" target='_blank'>
              Tumor secretome shapes the immune landscape during cancer progression
              </a>
            </td>
          <td>
            Jianqiang Yang, Sijia Tang, Nabil F. Saba, C. Shay, Yong Teng
          </td>
          <td>2025-02-10</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Recent advances in single-cell RNA sequencing (scRNA-seq) technologies have enabled us to clarify gene regulatory networks and immune cell compositions. In this study, integration of large-scale bulk and single-cell datasets refined hepatocellular carcinoma (HCC) classification and shed light on the characteristics of its aggressive subtype.


APPROACH AND RESULTS
Single-cell analysis of 228,564 live cells from six scRNA-seq datasets identified five major clusters of HCC cells with high mitotic activity (Cluster 1), activated Wnt/β-catenin signaling (Cluster 2), elevated glycolysis (Cluster 3) and lipogenesis (Clusters 4 and 5). Aggressive HCC subtype defined in bulk RNA-seq analysis of 691 tumor samples comprised a combination of Cluster 1 with Clusters 3, 4 or 5. Gene regulatory network analysis and gene set enrichment analysis highlighted the essential roles of p53 and MYC in aggressive HCC/Cluster 1, and cell composition analysis elucidated T cell depletion as an immune resistance mechanism. In a syngeneic mouse model, Trp53 knockout and MYC overexpression caused highly mitotic, tumorigenic and metastatic phenotypes, characterized by a macrotrabecular (MT) pattern, vascular encapsulation (VE) and T cell exclusion. Angiogenesis inhibition disrupted MT/VE formation, resulting in T cell recruitment, and its combination with immune checkpoint blockade achieved remission.


CONCLUSIONS
Single-cell analysis has deepened our understanding of the molecular mechanism and tumor microenvironment in aggressive HCC. The combination of targeting tumor vasculature and blocking immune checkpoints represents a promising therapeutic strategy for this subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e969ce26ddc70310682b3f3db6a8aee97b5128" target='_blank'>
              Integrative transcriptome profiling elucidates molecular and immunovascular characteristics of macrotrabecular hepatocellular carcinoma.
              </a>
            </td>
          <td>
            T. Taniai, S. Shimada, Y. Akiyama, Megumi Hatano, Koya Yasukawa, Yosuke Igarashi, Shu Tsukihara, Yoshiaki Tanji, K. Kodera, Kohei Okazaki, K. Haruki, Atsushi Nara, Kohei Yagi, K. Akahoshi, D. Ban, Yasuhiro Asahina, Toru Ikegami, Minoru Tanabe, Shinji Tanaka
          </td>
          <td>2025-02-26</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20f7ade1ee5b9d0bf71b29249fe64f13f03ddc98" target='_blank'>
              Characterization of single neurons reprogrammed by pancreatic cancer.
              </a>
            </td>
          <td>
            Vera Thiel, Simon Renders, Jasper Panten, Nicolas Dross, K. Bauer, D. Azorin, Vanessa Henriques, V. Vogel, C. Klein, Aino-Maija Leppä, Isabel Barriuso Ortega, J. Schwickert, I. Ourailidis, Julian Mochayedi, Jan-Philipp Mallm, Carsten Müller-Tidow, Hannah Monyer, J. Neoptolemos, T. Hackert, O. Stegle, D. T. Odom, R. Offringa, A. Stenzinger, Frank Winkler, M. Sprick, A. Trumpp
          </td>
          <td>2025-02-17</td>
          <td>Nature</td>
          <td>1</td>
          <td>85</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are pivotal biomarkers in tumor metastasis, however, the underlying molecular mechanism of CTCs behavioral heterogeneity during metastasis remains unexplored. Here, an integrative workflow is developed to link behavior characteristics to metabolic profiling within individual CTCs, which simulates the metastatic process on a microfluidic system and combined with single-cell mass spectrometry (MS) detection. Spheroid-derived HCT116 cells are tracked and extracted via a temporary vascular system, revealing various arrest patterns under biomimetic vascular shear flow. Downstream MS analysis characterizes 17 cellular metabolites and associates metabolic profiles with de-adhesion behaviors of the same CTCs, identifying a potential high-metastatic subpopulation with enhanced arrest ability and evaluating critical metabolites involved in metastasis pathways. Additionally, the metastasis-inhibiting effect of anti-tumor drug 5-fluorouracil by reducing high-metastatic cells in spheroids is elucidated. This approach offers a valuable opportunity to dissect the interplay of the metastatic behavior and metabolic profiles of CTCs and foster insights into the molecular mechanisms underlying behavioral phenotypes in the tumor metastasis process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32d3e81bfd80ccc573739b0a4ae2229795ed81e7" target='_blank'>
              Linking Metastatic Behavior and Metabolic Heterogeneity of Circulating Tumor Cells at Single-Cell Level Using an Integrative Microfluidic System.
              </a>
            </td>
          <td>
            Ying Hou, Jiaxu Lin, Hongren Yao, Zengnan Wu, Yongning Lin, Jinming Lin
          </td>
          <td>2025-02-17</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Increased stemness of cancer cells exacerbates radioresistance, thereby greatly limiting the efficacy of radiotherapy. In order to study the changes in cancer cell stemness during radiotherapy, we established a radioresistance model of human non-small cell lung cancer A549 cells and obtained A549 radioresistant cells (A549-RR). We sampled the cells at different time points during the modeling process and investigated the heterogeneity of each group of cells using single-cell sequencing. Cells in the early stages of fractionated irradiation were found to be significantly up-regulated in stemness, and a subpopulation of cells producing this response was screened and referred to as “radiation-induced stemness-responsive cancer cells”. They were undergoing stemness response, energy metabolism reprogramming, and progressively differentiating into cells with more diverse and malignant phenotypes in order to attenuate the killing effect of radiation. Furthermore, we demonstrated that such responses might be driven by the activation of the EGFR-Hippo signaling pathway axis, which also plays a crucial role in the development of radioresistance. Our study reveals the dynamic evolution of cell subpopulation in cancer cells during fractionated radiotherapy; the early stage of irradiation can determine the destiny of the radiation-induced stemness-responsive cancer cells. The activation of stemness-like phenotypes during the development of radioresistance is not the result of dose accumulation but occurs during the early stage of radiotherapy with relatively low-dose irradiation. The degree of the radiation-induced stemness response of cancer cells mediated by the EGFR-Hippo signaling pathway might be a potential predictor of the efficacy of radiotherapy and the development of radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6498ad18eccc13924e712513e028e4bc431ff4" target='_blank'>
              Single-Cell Sequencing Reveals the Role of Radiation-Induced Stemness-Responsive Cancer Cells in the Development of Radioresistance
              </a>
            </td>
          <td>
            Zheng Shi, Cuilan Hu, Jiadi Liu, Wei Cheng, Xiaohua Chen, Xiongxiong Liu, Yanyu Bao, Haidong Tian, Boyi Yu, Feifei Gao, Fei Ye, Xiaodong Jin, Chao Sun, Qiang Li
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d1d629c974d4b9e986ad888f6cfe60cc9b53f8" target='_blank'>
              Splicing diversity enhances the molecular classification of pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Yue Huang, Jing Guo, Xueshuai Han, Yang Zhao, Xuejing Li, Peiqi Xing, Yulou Liu, Yingxuan Sun, Song Wu, Xuan Lv, Lei Zhou, Yazhuo Zhang, Chuzhong Li, Weiyan Xie, Zhaoqi Liu
          </td>
          <td>2025-02-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="BACKGROUND AIMS
The prognoses and therapeutic responses of patients with intrahepatic cholangiocarcinoma (iCCA) depend on spatial interactions among tumor microenvironment (TME) components. However, the spatial TME characteristics of iCCA remain poorly understood. The aim of this study was to generate a comprehensive spatial atlas of iCCA using artificial intelligence-assisted spatial multiomics patterns and to identify spatial features associated with prognosis and immunotherapy.


APPROACH RESULTS
Spatial multiomics, including imaging mass cytometry (IMC, n=155 in-house), spatial proteomics (n=155 in-house), spatial transcriptomics (n=4 in-house), multiplex immunofluorescence (mIF, n=20 in-house), single-cell RNA sequencing (scRNA-seq, n=9 in-house and n=34 public), bulk RNA-seq (n=244 public), and bulk proteomics (n=110 in-house and n=214 public), were employed to elucidate the spatial TME of iCCA. More than 1.06 million cells were resolved, and the findings revealed that spatial topology, including cellular deposition patterns, cellular communities, and intercellular communications, profoundly correlates with the prognosis of iCCA patients. Specifically, CD163hi M2-like resident-tissue macrophages suppress anti-tumor immunity by directly interacting with CD8+ T cells, resulting in poorer patient survival. Additionally, five spatial subtypes with distinct prognoses were identified, and potential therapeutic options were generated for these subtypes. Furthermore, a spatial TME deep learning system was developed to predict the prognosis of iCCA patients with high accuracy from a single 1-mm2 tumor sample.


CONCLUSIONS
This study offers preliminary insights into the spatial TME ecosystem of iCCA, providing valuable foundations for precise patient classification and the development of personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f973214ce129b073da8e0d3c1c3cc399902a2e8" target='_blank'>
              Spatial single-cell proteomics landscape decodes the tumor microenvironmental ecosystem of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Libing Hong, Jie Mei, Xuqi Sun, Yifan Wu, Zhen Dong, Yuzhi Jin, Liaoliao Gao, Jinlin Cheng, Weihong Tian, Chuan Liu, Bin Li, Pingping Hu, Lulu Liu, Shan Xin, Xiaomeng Dai, Peng Zhao, Rongping Guo, Minshan Chen, Jingping Yun, Bo Lin, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-02-25</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7453b499aa0f4f56b20054ed92fb6b1a0f0d81c" target='_blank'>
              Analysis of thyroid carcinoma composition and spatial architecture in the progression of dedifferentiation, lymphatic metastasis, and gastric metastasis
              </a>
            </td>
          <td>
            Di Wang, Ruichun Lu, Fenglian Yan, Yansong Lin, Hao Wang, Huabao Xiong
          </td>
          <td>2025-02-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e79fb6328397b7897b70522e3b8824a0291600" target='_blank'>
              Spatial recognition and semi-quantification of epigenetic events in pancreatic cancer subtypes with multiplexed molecular imaging and machine learning
              </a>
            </td>
          <td>
            Krzysztof Szymoński, Natalia Janiszewska, K. Sofińska, Katarzyna Skirlińska-Nosek, Dawid Lupa, Michał Czaja, Marta Urbańska, Katarzyna Jurkowska, Kamila Konik, Marta Olszewska, Dariusz Adamek, Kamil Awsiuk, E. Lipiec
          </td>
          <td>2025-02-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Dear Editor Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark1,2, with a pervasive presence across cancers3,4. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate ecDNA’s behaviors and functions 5-7. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction while we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Despite significant advancements in identifying novel therapeutic targets and compounds, cancer stem cells (CSCs) remain pivotal in driving therapeutic resistance and tumor progression in gastric cancer (GC). High-resolution knowledge of the transcriptional programs underlying the role of CSC niche in driving tumor stemness and progression is still lacking. Herein, spatial and single-cell RNA sequencing of 32 human gastric mucosa tissues at various stages of malignancy, illuminating the phenotypic plasticity of tumor epithelium and transcriptional trajectory from mature gastric chief cells to the CSC state, which is associated with activation of EGFR and WNT signaling pathways, is conducted. Moreover, the CSCs interact with not only the immunosuppressive CXCL13+ T cells and CCL18+ M2 macrophages to evade immune surveillance, but also the inflammatory cancer-associated fibroblasts (iCAFs) to promote tumorigenesis and maintain stemness, which construct the CSC niche leading to inferior prognosis. Notably, it is uncovered that amphiregulin (AREG) derived from iCAFs promotes tumor stemness by upregulating the expression of SOX9 in tumor cells, and contributes to drug resistance via the AREG-ERBB2 axis. This study provides valuable insight into the characteristics of CSC niche in driving tumor stemness and progression, offering novel perspective for designing effective strategies to overcome GC therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d6cf84d15c0f17d3af8a79006d512aec0aa37" target='_blank'>
              Dissecting the Spatial and Single-Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer.
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinglong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Despite numerous studies on medulloblastoma (MB) cell heterogeneity, the spatial characteristics of cellular states remain unclear.


METHODS
We analyze single-nucleus and spatial transcriptomes and chromatin accessibility from human MB spanning four subgroups, to identify malignant cell populations and describe the spatial evolutionary trajectories. The spatial CNVs patterns and niches were analyzed to investigate the cellular interactions.


RESULTS
Three main malignant cell populations were identified, including progenitor-like, cycling and differentiated populations. Gene signatures of cell populations strongly correlate to clinical outcomes. These tumor cell populations are geographically organized as stem-like and mature regions, highlighting their spatially heterogeneous nature. Progenitor-like and cycling cells are mainly concentrated in stem-like regions, whereas various differentiated populations are primarily distributed in mature regions. By analyzing chromosomal alterations, we find that stem-like region typically harbors a single pattern of CNVs, reflecting high originality and uniformity, which is in stark contrast to mature region exhibiting multiple patterns with a broader range of biological functions. Projecting cellular state program onto spatial sections fully illustrates the evolution from stem-like region to various functional zones in mature region, which is correlated to microenvironmental components along the paths to maintain stemness or promote differentiation. Conclusions. This multi-omics database comprehensively facilitates the understanding of MB spatial evolutionary organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb3b0d5b454cf059f699fd40f05952bb5941c97" target='_blank'>
              Unveiling spatial heterogeneity in medulloblastoma: a multi-omics analysis of cellular state and geographical organization.
              </a>
            </td>
          <td>
            Jiankang Li, Hailong Liu, Ziwei Wang, Jiao Zhang, Xuan Chen, Craig Daniels, Xiaochong Wu, Olivier Saulnier, Hiromichi Suzuki, Pasqualino de Antonellis, Alexandra Rasnitsyn, Winnie W. Ong, Evan Y Wang, Liam D Hendrikse, Yu Su, Yu Tian, Dongming Han, Ruohan Wang, Jialin Mo, Fei Liu, Kaiwen Deng, Dongyang Wang, Zhaoyang Feng, Yifei Jiang, Yanong Li, Yuting Ma, Zijia Liu, Meiyu Li, Peiyi Tian, Yanfeng Shi, Yong Jiang, Tao Yang, Shouwei Li, Jianfeng Liang, Jingchuan Wu, Ying Wang, Wanjing Zou, Yina Jiang, Lusheng Wang, Fang Chen, Xin Jin, Shuaicheng Li, Xiaoguang Qiu, Chunde Li, Ya Gao, Yujie Tang, Michael D. Taylor, Tao Jiang
          </td>
          <td>2025-02-17</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebf7cbd64683a9de7c4a1aef2af36c726385b4a9" target='_blank'>
              Micro Immune Response On-chip (MIRO) models the tumour-stroma interface for immunotherapy testing
              </a>
            </td>
          <td>
            Alice Perucca, Andrea Gómez Llonín, Oriol Mañé Benach, Clement Hallopeau, Elisa I. Rivas, J. Linares, Marta Garrido, Anna Sallent-Aragay, Tom Golde, Julien Colombelli, Eleni Dalaka, Judith Linacero, Marina Cazorla, Teresa Galan, Jordi Pastor Viel, Xavier Badenas, Alba Recort-Bascuas, L. Comerma, Patricia Fernandez-Nogueira, A. Rovira, Pere Roca-Cusachs, J. Albanell, Xavier Trepat, A. Calon, Anna Labernadie
          </td>
          <td>2025-02-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="
 Introduction: The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Attempts to combine RT with immunotherapies in PDAC have yielded inconsistent outcomes, highlighting a critical gap in our understanding of immune dynamics and mechanisms influencing RT response. Our study aims to uncover molecular pathways and cellular subtypes associated with radioresponse. To do so, we developed a novel cell-cell interaction platform, "SpaCCI", to map ligand-receptor (LR) interactions, essential for understanding cellular communication within the TME and its contribution to therapeutic resistance. Methods: Single-cell RNA sequencing (scRNA-seq) was performed on 16 matched pre- and post-treatment PDAC biopsies from a Phase I/II dose-escalation clinical trial of stereotactic body radiation therapy. Additionally, spatial transcriptomics (ST) was conducted on resected PDAC (20 treated with neoadjuvant chemoradiation, 8 untreated). We integrated scRNA-seq and ST datasets using cell type decomposition and developed SpaCCI to enhance LR interaction detection in tissues with complex cell mixtures. Results: ScRNA-seq identified five molecular subpopulations of cancer cells associated to RT response, with PDAC_3 enriched post-RT and marked by genes involved in lipid metabolism and ferroptosis defense. Post-RT depleted subpopulations, such as PDAC_0 and PDAC_prolif, saw upregulation of pathways related to rapid proliferation and oxidative phosphorylation. In a murine model, PDAC_3 was similarly enriched in post-RT tumors, with persister cell lines from these tumors showing overexpression of ferroptosis defense genes and resistance to ferroptosis inducers including Erastin and RSL3. ST analysis also revealed spatial immune changes post-RT, with increased cytotoxic and effector memory T cells balanced by regulatory T cells and M2-like macrophages, indicating immune escape. NK and memory B cells correlated with improved survival post-RT, while plasma IgG cells localized outside the tumor bed, suggesting compartmentalized immune responses. Using SpaCCI, we observed differential enrichment of ligand-receptor interactions in RT-treated tumors, particularly among lipid-associated tumor macrophages, cancer cells, myofibroblast-like cancer-associated fibroblasts, and endothelial cells. Persister PDAC_3 subpopulations showed enriched LR SEMA3 signaling with CAFs, myeloid cells, and endothelial cells consistent with axon guidance pathways. Conclusions: This study presents a molecular and cellular atlas of PDAC response to RT, revealing persister cells populations with adaptations in lipid metabolism and ferroptosis defense that could be targeted to enhance RT efficacy. ST highlighted complex immune dynamics within the TME demonstrating a balance between inflammatory and immunosuppressive populations. Our findings provide a foundation for combinatorial therapeutic strategies and suggest that axon guidance pathways may also play an important role in the response to RT in PDAC.
 Citation Format: Vincent Bernard, Li-Ting Ku, Tianyu Wang, Kimal Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, Manoop S. Bhutani, Paola A. Guerrero, Ethan B. Ludmir, Albert C. Koong, Cullen M. Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay.Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P018">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd18fd5a66ac51a4015b7b7d9504296b9a16919" target='_blank'>
              Abstract P018: Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, K. Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, M. Bhutani, Paola A. Guerrero, E. Ludmir, Albert C Koong, Cullen M Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The exploration of cellular heterogeneity within the tumor microenvironment (TME) via single-cell RNA sequencing (scRNA-seq) is essential for understanding cancer progression and response to therapy. Current scRNA-seq approaches, however, lack spatial context and rely on incomplete datasets of ligand-receptor interactions (LRIs), limiting accurate cell type annotation and cell-cell communication (CCC) inference. This study addresses these challenges using a novel graph neural network (GNN) model that enhances cell type prediction and cell interaction analysis. Our study utilized a dataset consisting of 49,020 cells from 19 patients across three cancer types: Leukemia, Breast Invasive Carcinoma, and Colorectal Cancer. The proposed scGSL model demonstrated robust performance, achieving an average accuracy of 84.83%, precision of 86.23%, recall of 81.51%, and an F1 score of 80.92% across all datasets. These metrics represent a significant enhancement over existing methods, which typically exhibit lower performance metrics. Additionally, by reviewing existing literature on gene interactions within the TME, the scGSL model proves to robustly identify biologically meaningful gene interactions in an unsupervised manner, validated by significant expression differences in key gene pairs across various cancers. The source code and data used in this paper can be found in https://github.com/LiYuechao1998/scGSL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd31c962d460de902bace72d06f995f9fc390f7" target='_blank'>
              Graph Structure Learning for Tumor Microenvironment with Cell Type Annotation from non-spatial scRNA-seq data
              </a>
            </td>
          <td>
            Yu-an Huang, Yue C. Li, Hai-Ru You, Jie Pan, Xiyue Cao, Xinyuan Li, Zhi-an Huang, Zhu-Hong You
          </td>
          <td>2025-02-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Genomic and epigenomic instability are defining features of cancer, driving tumor progression, heterogeneity, and therapeutic resistance. Central to this process are epigenetic echoes, persistent and dynamic modifications in DNA methylation, histone modifications, non-coding RNA regulation, and chromatin remodeling that mirror underlying genomic chaos and actively influence cancer cell behavior. This review delves into the complex relationship between genomic instability and these epigenetic echoes, illustrating how they collectively shape the cancer genome, affect DNA repair mechanisms, and contribute to tumor evolution. However, the dynamic, context-dependent nature of epigenetic changes presents scientific and ethical challenges, particularly concerning privacy and clinical applicability. Focusing on lung cancer, we examine how specific epigenetic patterns function as biomarkers for distinguishing cancer subtypes and monitoring disease progression and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3b401f96a57f38873c48e3928dd0d7e44a2cbef" target='_blank'>
              Epigenomic Echoes—Decoding Genomic and Epigenetic Instability to Distinguish Lung Cancer Types and Predict Relapse
              </a>
            </td>
          <td>
            Alexandra A Baumann, Zholdas Buribayev, Olaf Wolkenhauer, Amankeldi A. Salybekov, M. Wolfien
          </td>
          <td>2025-02-05</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Compared to regular tumor cells, cancer stem cells exhibit dangerous characteristics, including high proliferation, high metastatic potential, and significantly increased in vivo tumorigenicity. Although some studies have emphasized the impact of the microenvironment on cell stemness, they have largely overlooked the mechanical forces derived from the stiffness of the surrounding extracellular matrix. Our previous research demonstrated that a 90 Pa soft fibrin matrix in three-dimensional (3D) culture can induce cells to become cancer repopulating cells with high stemness. Acetylation modification significantly influences the metabolism, epigenetics, proliferation, migration, and immune evasion of tumor cells. In this study, we performed a comprehensive analysis of the proteome and acetyl-proteome of breast cancer cells under two-dimensional (2D) plate and 3D matrix conditions with varying stiffness. This dataset provides a valuable resource for understanding the dynamic regulation of protein acetylation in response to mechanical stiffness. The mass spectrometry-based proteomics data have been uploaded to the ProteomeXchange Consortium with the dataset identifier PXD057820.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf675151cfc1c102ff0e5b475e53a4b55c5da33c" target='_blank'>
              Dynamic Proteomic and Acetylomic Profiling of Mechanically Induced Cancer Stemness in Breast Cancer Cells.
              </a>
            </td>
          <td>
            Rong Han, Yue Wu, Yehong Yang, Qiaochu Wang, Tao Ding, Xutong Zhang, Juntao Yang
          </td>
          <td>2025-02-21</td>
          <td>Proteomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Prostate cancer is a prevalent tumor type that, despite being highly curable, progresses to metastatic disease in a fraction of patients, thus accounting for more than 350.000 annual deaths worldwide. In turn, uncovering the molecular insights of metastatic disease is instrumental to improve the survival rate of prostate cancer patients. By means of gene expression metanalysis in multiple prostate cancer patient cohorts, we identified a set of genes that are differentially expressed in aggressive prostate cancer. Transcription factor 19 (TCF19) stood out as an unprecedented epithelial gene upregulated in metastatic disease, with prognostic potential and associated with the activity of androgen receptor. By combining computational and empiric approaches, our data revealed that TCF19 is required for full metastatic capacity and its depletion influences core cancer-related processes, such as vascular permeability, supporting the role of this gene in the dissemination of prostate tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecdf804c61fcb68669b4e697cde8e1d82f6075a" target='_blank'>
              Transcription factor 19 is an androgen responsive gene that modulates vessel homeostasis and sustains metastatic prostate cancer
              </a>
            </td>
          <td>
            Amaia Ercilla, Jana R. Crespo, Saioa Garcia-Longarte, Marta Fidalgo, Natalia Martín-Martín, Onintza Carlevaris, I. Astobiza, Sonia Fernández-Ruiz, M. Guiu, Laura Bárcena, Isabel Mendizabal, A. Aransay, M. Graupera, R. Gomis, Arkaitz Carracedo
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) are a small proportion of stem-like breast cancer cells with more tumorigenic and metastatic capacity in comparison to the bulk tumor cells, which are capable of self-renewal and generating differentiated cells. BCSCs are resistant to chemotherapy and radiotherapy. More and more research indicate that BCSCs are heterogeneous and have plasticity, which refer to their ability to switch between different subtypes of stem-like cells and differentiated cells. There is some evidence showing that BCSC heterogeneity and plasticity plays a role on therapeutic resistance and metastasis of breast cancer. In this review, we discussed the roles of extrinsic (rewired tumor microenvironment, including hypoxia, microbiota) and intrinsic (pro-tumor signaling) factors on regulating BCSC heterogeneity and plasticity, leading to various malignant behaviors of BCSCs, including epithelial-mesenchymal transition (EMT)/mesenchymal-epithelial transition (MET), immune evasion, vasculogenesis and so on. Moreover, we also discussed the potential therapeutic strategies to target BCSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1099ecb1edc8ebafead4e021333de49587a487" target='_blank'>
              Breast Cancer Stem Cell Heterogeneity, Plasticity and Treatment Strategies
              </a>
            </td>
          <td>
            Mengting Chen, Suling Liu
          </td>
          <td>2025-03-06</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastasis drives mortality and morbidity in cancer. While some patients develop broad metastatic disease across multiple organs, others exhibit organ-specific spread. To identify mechanisms underlying metastatic organotropism, we analyzed clinico-genomic data from over 7,000 patients with metastatic cutaneous melanoma in three independent cohorts (one primary discovery and two validation cohorts including a nationwide electronic health record-derived deidentified database), leveraging machine learning approaches to clinical data. We found that female sex and increased tumor mutational burden associate with decreased metastatic potential, while older age associates with more lung and adrenal metastases. Using unsupervised analyses, patients clustered into five metastatic patterns: a “highly metastatic” cluster characterized by involvement of many organs, a “low metastatic” cluster characterized by few metastatic sites (mostly lymph node metastases), and three additional clusters each characterized by metastasis to specific sites (brain, lung, liver). Mutations in B2M and PTEN associated with increased overall metastatic potential. PTEN mutations were also associated with brain metastases but were enriched only in the “highly metastatic” cluster and not the brain-specific cluster. Mutations in GNAQ or GNA11 (GNA) associated with increased liver metastasis. To validate this association, we tested and demonstrated liver tropism in two GNA-mutant genetically engineered cutaneous melanoma mouse models of metastasis. Overall, our study elucidates distinct phenotypes of metastasis in patients with melanoma and identifies novel clinical and genomic associations that illuminate the clinical drivers of metastatic organotropism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7031286fbc9875d02c4079e2f16707bda19ea728" target='_blank'>
              Clinico-genomic features predict distinct metastatic phenotypes in cutaneous melanoma
              </a>
            </td>
          <td>
            Tyler J. Aprati, Chi-Ping Day, Daniel Lee, Alexander Pan, J. Jee, Giuseppe Tarantino, M. Manos, Hannah Faulkner, Marta M. Holovatska, K. Khaddour, Catherine H. Feng, Kelly P. Burke, Marc Glettig, Zoe Weaver Ohler, R. El Meskini, F. S. Hodi, Rizwan Haq, A. Shoushtari, N. Schultz, Jeffrey J. Ishizuka, Alexander Gusev, Maryclare Griffith, Kenneth L. Kehl, David Liu
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Chimeric antigen receptor T cell (CART) therapy has demonstrated remarkable efficacy in hematologic malignancies but has struggled to achieve comparable success in solid tumors. A key obstacle is the extracellular matrix (ECM) in solid tumors, which significantly impedes CART cell infiltration. In clinical trials, neuroblastoma (NB) has shown responsiveness to GD2-directed CART therapy, however, the failure of GD2.CARTs to effectively clear bulky disease - characterized by dense ECM - highlights the critical challenge of infiltration. In this study, we demonstrate that GD2.CARTs exhibit a unique infiltration-restriction compared to other CARTs and endogenous T cells. A separate analysis of clinical datasets identified MMP7 and SPP1 (OPN) as candidate genes to improve the infiltration of GD2.CARTs as these were upregulated in tumor-infiltrating leukocytes. MMP-7 and OPN overexpression enhanced CART extravasation (p < .001) and interstitial movement (p < .05) in ECM-dense environments in vitro. Overexpression of either OPN (p < .0001) or MMP-7 (p < .001) improved tumor infiltration in a xenograft model of NB. This resulted in improved tumor control (94% reduction in tumor burden, p < .05) and a survival extension in OPN-GD2.CART treated mice compared to unmodified GD2.CARTs (median of 148 days, p < .05). OPN overexpression did not increase off-target infiltration into healthy tissues or promote tumor metastasis, highlighting its potential for safe therapeutic application. Our study provides a framework for further exploration of gene modifications to improve CART infiltration and efficacy in solid tumors and identifies SPP1 as a candidate gene to improve GD2.CART treatment of bulky tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4a1fc739777ef8383e9e035dee7b2fb590a010" target='_blank'>
              Tunneling CARs: Increasing CAR T tumor infiltration through the overexpression of MMP7 and SPP1
              </a>
            </td>
          <td>
            Stacey N. Van Pelt, Mark White, Candise Tat, Devyn Hooper, Lindsay Talbot, Rohan Fernandes, Cliona Rooney, Bilal Omer
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Glioma, characterized by its cellular and molecular heterogeneity, presents formidable challenges in treatment strategy and prognostic assessment. The tumor microenvironment (TME) profoundly influences tumor behavior and treatment response, with tumor‐associated neutrophils (TANs) playing a complex but understudied role. This study aimed to investigate the heterogeneity and role of TANs in glioma and to develop a prognostic model. Methods Analysis of scRNA‐seq data identified cellular subpopulations and differentially expressed neutrophil‐related genes (DE‐NRGs). Bulk RNA‐seq was obtained from four independent datasets. Molecular subtypes of glioma samples were determined by consensus clustering. WGCNA was conducted to elucidate the association between gene modules and subtypes. We developed a risk score model. Expression of selected genes was confirmed using immunohistochemistry (IHC). In vitro experiments were also performed for functional verification, including CCK8, EdU, Transwell, and TUNEL assays. Results A total of 108 DE‐NRGs for TANs were identified based on scRNA‐seq data. Two molecular subtypes were characterized, showing significant differences in prognosis and clinical features. Immune‐related analyses demonstrated varied immunological characteristics between subtypes. The risk score model was constructed with 7 genes, including AEBP1, CAVIN1, DCTD, DEPP1, DUSP6, FKBP9, and UGCG. It showed significant prognostic value and was validated across three external datasets. The mutation landscape highlighted higher IDH mutation prevalence in low‐risk groups. Drug sensitivity analysis indicated TMZ resistance in high‐risk groups. In vitro experiments showed that UGCG could promote glioma cell proliferation, migration, and invasion, while decreasing apoptosis. Conclusion This study explored the heterogeneity of TANs and developed a prognostic model, providing insights for prognostic prediction and guiding personalized treatment strategies in glioma. Declaration of Generative AI in Scientific Writing: The authors declare nonuse of generative AI and AI‐assisted technologies in the writing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a8af983a6227909234b80a95e84b835146d4d4" target='_blank'>
              Integrated Analysis to Reveal Heterogeneity of Tumor‐Associated Neutrophils in Glioma
              </a>
            </td>
          <td>
            Wen Wang, Junsheng Li, Qiheng He, Chenglong Liu, Siyu Wang, Zhiyao Zheng, Bojian Zhang, Siqi Mou, Wei Sun, Jizong Zhao
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 218


 Background:
 Peritoneal metastases (PM) in colorectal cancer (CRC) portend a poor prognosis. We sought to elucidate spatially resolved molecular features facilitating transcoelomic dissemination and progression.
 Methods:
 50 PT, 116 PM and 37 primary normal samples from 96 patients were retrieved and profiled with digital spatial profiling (GeoMx DSP, Nanostring Technologies). Through unsupervised clustering, we characterized the microenvironment of tumor-stroma compartments and studied the roles of stromal phenotypes in promulgating tumorigenesis. These findings were orthogonally validated through spatial proteomics (multiplex IHC [mIHC] - COMET, Lunaphore Technologies) of 10 paired PT-PM samples retrieved from 6 patients.
 Results:
 A median of 37 region of interests (ROIs) (IQR: 32-38) were selected from a total of 9 tissue microarray (TMA) slides. A total of 269 ROIs (181 tumor, 88 stroma) from PM samples, 45 ROIs (34 tumor, 11 stroma) from PT samples and 28 (all stroma) ROIs from primary normal samples were retrieved and profiled. Through consensus clustering of stroma ROIs, we identify a fibro-collagenous and immune infiltrated stromal phenotype (stromal cluster [SC] 2) characterized by increased cancer associated fibroblasts (CAFs), memory B cells, M2 macrophages and T-cell exhaustion. Patients with SC2 stroma had poorer survival (p=0.036). SC2 stroma was also observed to support adjacent tumor compartments with enriched oncogenic pathways such as TGF-beta, TNF-alpha, hypoxia and JAK-STAT. Through discriminatory gene expression profiles, we developed a 20-gene composite bidirectional signature of SC2. The prognostic significance of the SC2-signature was externally validated through several cohorts including TCGA and MSI-H only cohorts. Next, by inspecting mIHC retrieved immune cell type densities across ROIs, we confirmed a similar phenomenon in which two distinct stromal clusters were found – one being fibroblast infiltrated but T-cell depleted (SC2-like, n=67 ROI) and another which is T-cell infiltrated but fibroblast depleted (SC1-like, n=104 ROI). Diverging spatial distributions of T-cells and fibroblasts between SC1 and SC2-like stroma was appreciated as well. Between both PT and PM, we find close clustering of T cells in SC1-like stromal compartments and fibroblasts in SC2-like stromal compartments.
 Conclusions:
 We describe SC2, a pro-tumorigenic stromal phenotype characterized by increased CAFs, T cell exhaustion and is associated with poor prognosis in CRC PM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76074158017484a4495c691d7f74783fe9c4e65" target='_blank'>
              Diverse stromal phenotypes at spatial resolution in colorectal cancer peritoneal metastasis.
              </a>
            </td>
          <td>
            Joseph J. Zhao, J. Ong, Ying Liu, Supriya Srivastava, Melissa Ching Ching Teo, Joey Jia-Ying Lee, Robert Walsh, Lit-Hsin Loo, Qiu Xuan Tan, Gillian Ng, J. W. Tan, Haoran Ma, Xuewen Ong, S. Tay, T. Sheng, Jeffrey H Y Lum, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Background Glioblastoma (GBM) is the most common primary malignant brain tumor in adults, characterized by aggressive growth and a dismal prognosis. Interleukin-33 (IL-33) and its receptor ST2 have emerged as regulators of glioma growth, but their exact function in tumorigenesis has not been deciphered. Indeed, previous studies on IL-33 in cancer have yielded somewhat opposing results as to whether it is pro- or anti-tumorigenic. Methods IL-33 expression was assessed in a GBM tissue microarray and public databases. As in vivo models we used orthotopic xenografts of patient-derived GBM cells, and syngenic models with grafted mouse glioma cells. Results We analyzed the role of IL-33 and its receptor ST2 in nonmalignant cells of the glioma microenvironment and found that IL-33 levels are increased in cells surrounding the tumor. Protein complexes of IL-33 and ST2 are mainly found outside of the tumor core. The IL-33-producing cells consist primarily of oligodendrocytes. To determine the function of IL-33 in the tumor microenvironment, we used mice lacking the ST2 receptor. When glioma cells were grafted to ST2-deficient mouse brains, the resulting tumors exhibited a more invasive growth pattern, and are associated with poorer survival, compared to wild-type mice. Tumors in ST2-deficient hosts are more invasive, with increased expression of extracellular matrix remodeling enzymes and enhanced tumor angiogenesis. Furthermore, the absence of ST2 leads to a more immunosuppressive environment. Conclusions Our findings reveal that glia-derived IL-33 and its receptor ST2 participate in modulating tumor invasiveness, tumor vasculature, and immunosuppression in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc52b5f579f7e0f0d9dd54d423d1abfbfb33ee41" target='_blank'>
              Lack of ST2 aggravates glioma invasiveness, vascular abnormality, and immune suppression
              </a>
            </td>
          <td>
            Grzegorz Wicher, Ananya Roy, Alessandra Vaccaro, K. Vemuri, Mohanraj Ramachandran, Tommie Olofsson, Rebeca-Noemi Imbria, M. Belting, Gunnar Nilsson, A. Dimberg, K. Forsberg-Nilsson
          </td>
          <td>2025-01-27</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Germinal centres (GCs) are specialized structures where B cells undergo iterative steps of B-cell receptor (BCR) somatic hypermutation and selection of best antigen binders in a darwinian-like fashion. The accelerated evolutionary process leads to the production of high-affinity antibodies that are crucial for robust and long-term humoral immunity. Within this frame, single-cell BCR sequencing analysis is a method of choice to track GC B cell dynamics as somatic mutations can be utilised as an in vivo molecular tracer. Herein, we present SeQuoIA, a start-to-finish pipeline for the analysis of BCR repertoire sequencing data at the single-cell level, including improved clonotype assignment and phylogeny reconstruction. Most importantly, we introduce a new method for the inference of BCR-driven selection pressure based on somatic mutation patterns, that was validated with biological data. With this pipeline, we explored public datasets and proposed new selection mechanisms in GCs. Significance Our pipeline should contribute to a better understanding of the basic biology of GC dynamics, and potentially help in laboratory animal usage reduction. Clinical applications could include assessment of vaccine efficacy, monitoring of B cell anti-tumoral responses, and identification of BCR-mediated processes in B cell lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1235a7befe3184ce5a84373898145bc5761dacf1" target='_blank'>
              SeQuoIA: a single-cell BCR sequencing analysis pipeline for tracking selection mechanisms in germinal centres
              </a>
            </td>
          <td>
            Eglantine Hector, P. Milpied
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer is a complex disease driven by dynamic regulatory shifts that cannot be fully captured by individual molecular profiling. We employ a data-driven approach to construct a coarse-grained dynamic network model based on hallmark interactions, integrating stochastic differential equations with gene regulatory network data to explore key macroscopic dynamic changes in tumorigenesis. Our analysis reveals that network topology undergoes significant reconfiguration before hallmark expression shifts, serving as an early indicator of malignancy. A pan-cancer examination across $15$ cancer types uncovers universal patterns, where Tissue Invasion and Metastasis exhibits the most significant difference between normal and cancer states, while the differences in Reprogramming Energy Metabolism are the least pronounced, consistent with the characteristic features of tumor biology. These findings reinforce the systemic nature of cancer evolution, highlighting the potential of network-based systems biology methods for understanding critical transitions in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a279852f8f3e4dea6bf2c322ab3c789c81b3e6c" target='_blank'>
              How cancer emerges: Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-02-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="To understand cytotrophoblast (CTB) dynamics during pregnancy, we constructed a single-cell RNA sequencing dataset profiling over 35,000 CTBs isolated from human placental tissues at various gestational stages. Our analysis revealed a robust set of transcripts defining distinct CTB subsets and their spatial relationships within the placenta. We identified a transit-amplifying primitive trophoblast population that may contribute to the CTB pool and found that CTBs located at the smooth chorion form a leak-tight epithelial layer that adapts to low oxygen environments without undergoing syncytialization. Additionally, our study suggests that CTB fate in the villous chorion may be influenced by interactions with the basal lamina, as indicated by a population of BCAM+ CTBs. This subset, isolated from term placental tissue, displayed progenitor capabilities in organoid cultures, supporting the hypothesis that BCAM+ CTBs maintain a progenitor role late in pregnancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/627224093857479c897fc038688310578cffb01f" target='_blank'>
              Characterization of cytotrophoblast cell population dynamics throughout pregnancy
              </a>
            </td>
          <td>
            O. Schäffers, Simone Verburgt, M. Harhangi, M. Broekhuizen, Eline van Hengel, L. E. van der Meeren, Sam Schoenmakers, Dick H W Dekkers, Wouter A S Doff, J. Demmers, Irwin K M Reiss, Joost Gribnau, B. V. van Rijn
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8d5845a6bd00754561294af9f1ae51a6774014" target='_blank'>
              Single-cell pseudotime and intercellular communication analysis reveals heterogeneity and immune microenvironment in oral cancer
              </a>
            </td>
          <td>
            Hanjun Liu, Lian Xie, Xuemin Xing, Lili Hou, Jinfeng Zhang, Ling Yang
          </td>
          <td>2025-02-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f2358565c36d3daf9c70fc71b645b6b73d557a4" target='_blank'>
              SELP+ TEC:CD8+ T cell crosstalk associates with improved radiotherapy efficacy in cervical cancer
              </a>
            </td>
          <td>
            Qingyu Huang, Wenhui Yang, Fuhao Wang, R. Huang, Qian Wang, Xiaohui Li, T. Lei, Shengqin Yue, Wen-Lu Zou, Qi An, Jinbo Yue, Qinyong Hu, Chao Liu
          </td>
          <td>2025-02-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47537700619a16e63c5ebabefc327d2e5284da8b" target='_blank'>
              Transcriptome analysis of novel B16 melanoma metastatic variants generated by serial intracarotid artery injection
              </a>
            </td>
          <td>
            J. C. Kienzler, Erick Contreras, J. Treger, Linda M. Liau, Geoffrey Owens, Robert M. Prins
          </td>
          <td>2025-01-16</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59b664fbb9ca2f0d479f54b9221d29d13ae0d1a6" target='_blank'>
              Transcriptional regulatory network analysis identifies GRN as a key regulator bridging chemotherapy and immunotherapy response in small cell lung cancer
              </a>
            </td>
          <td>
            S. Yoo, Ayushi S. Patel, S. Karam, Yi Zhong, Li Wang, Feng Jiang, Ranran Kong, Sharon Bikvan, Wenhui Wang, A. Sinha, Charles A. Powell, Jun Zhu, Hideo Watanabe
          </td>
          <td>2025-02-05</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background: Circulating tumor cells (CTCs) and CTC-clusters are pivotal in the metastatic process of breast cancer (BC). Owing to their low frequency, models replicating their biology should provide a robust platform for investigating the molecular mechanisms driving metastasis and identifying new biomarkers. We established and characterized a CTC-derived cell model from a mouse xenograft to explore its metastatic behavior and molecular profile, which allowed us to investigate the expression and prognostic significance of a set of genes associated with the metastatic potential of CTCs. Methods: The CTC line (mCTC) derived from a MDA-MB-231 mouse xenografts was used in comparative functional analyses including cell cycle evaluation, colony formation, invasion, adhesion, and metastatic competency in zebrafish models. Transcriptomic profiling and functional assays were conducted to identify candidate genes and understand their roles in metastasis. Moreover, publicly available gene expression datasets of CTCs, CTC-clusters, and tumor tissue, from GEO and TCGA, were analyzed for the identification of a gene signature that was correlated with survival data. The signature was validated in an independent cohort. Results: Compared with MDA-MB-231 cells, mCTC cells presented enhanced colony formation, invasion, and adhesion, and increased dissemination and survival in zebrafish. Transcriptomic analysis revealed that SPARC was significantly upregulated. Functional assays showed that SPARC overexpression was correlated with increased invasion and migration. Analysis of public datasets confirmed the high expression of SPARC in BC CTCs and CTC-clusters. Additionally, a 4-gene signature involving SPARC, THBS1, VCL, and HSP90AB1 was identified that demonstrated strong prognostic value, predicting shorter overall and distant metastasis-free survival in the primary tumor setting. Validation cohorts confirmed its ability to distinguish high-risk patients. Elevated expression of the 4-gene signature in CTCs was also indicative of increased mortality risk. Conclusion: mCTC exhibit distinct metastatic traits and molecular characteristics, highlighting a possible role of SPARC in CTC biology and its potential as a prognostic marker in BC metastasis. The identified 4-gene signature provides a robust prognostic tool for assessing patient risk and guiding therapeutic strategies. Further investigations into the mechanistic role of SPARC may reveal new therapeutic targets for managing BC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785682c178d882b31b862712e1108b5022d056bd" target='_blank'>
              A CTC Model Uncovers Metastatic Drivers and Prognostic Markers in Breast Cancer
              </a>
            </td>
          <td>
            C. Fernández-Santiago, I. Martínez-Peña, M. Paramés, M. Rodriguez, P. Hurtado, C. Abuín, C. Costa, A. B. Dávila Ibáñez, R. López-López, R. Piñeiro
          </td>
          <td>2025-01-17</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tyrosine kinase inhibitors (TKIs) have transformed the treatment of epidermal growth factor receptor (EGFR)-mutant non-small cell lung cancer. However, treatment resistance remains a major challenge in clinical practice. The tumor microenvironment (TME) is a complex system composed of tumor cells, immune and non-immune cells, and non-cellular components. Evidence indicates that dynamic changes in TME during TKI treatment are associated with the development of resistance. Research has focused on identifying how each component of the TME interacts with tumors and TKIs to understand therapeutic targets that could address TKI resistance. In this review, we describe how TME components, such as immune cells, fibroblasts, blood vessels, immune checkpoint proteins, and cytokines, interact with EGFR-mutant tumors and how they can promote resistance to TKIs. Furthermore, we discuss potential strategies targeting TME as a novel therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f6f1b01ac68ddc2dc3220a920e6b272e7041d2" target='_blank'>
              Targeting the Tumor Microenvironment in EGFR-Mutant Lung Cancer: Opportunities and Challenges
              </a>
            </td>
          <td>
            Jeong Uk Lim, Junyang Jung, Y. Kim, Chi Young Kim, Sang Hoon Lee, Dong Won Park, Sue In Choi, W. Ji, C. Yeo, Seung Hyeun Lee
          </td>
          <td>2025-02-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Ongoing mutagenesis in cancer drives genetic diversity throughout the natural history of cancers. As the activities of mutational processes are dynamic throughout evolution, distinguishing the mutational signatures of ‘active’ and ‘historical’ processes has important implications for studying how tumors evolve. This can aid in understanding mutagenic states at the time of presentation, and in associating active mutational process with therapeutic resistance. As bulk sequencing primarily captures historical mutational processes, we studied whether ultra-low-coverage single-cell whole-genome sequencing (scWGS), which measures the distribution of mutations across hundreds or thousands of individual cells, could enable the distinction between historical and active mutational processes. While technical challenges and data sparsity have limited mutation analysis in scWGS, we show that these data contain valuable information about dynamic mutational processes. To robustly interpret single nucleotide variants (SNVs) in scWGS, we introduce ArtiCull, a method to identify and remove SNV artifacts by leveraging evolutionary constraints, enabling reliable detection of mutations for signature analysis. Applying this approach to scWGS data from pancreatic ductal adenocarcinoma (PDAC), triple-negative breast cancer (TNBC), and high-grade serous ovarian cancer (HGSOC), we uncover temporal and spatial patterns in mutational processes. In PDAC, we observe a temporal increase in mismatch repair deficiency (MMRd). In cisplatin-treated TNBC patient-derived xenografts, we identify therapy-induced mutagenesis and inactivation of APOBEC3 activity. In HGSOC, we show distinct patterns of APOBEC3 mutagenesis, including late tumor-wide activation in one case and clade-specific enrichment in another. Additionally, we detect a clone-specific increase in SBS17 activity, in a clone previously linked to recurrence. Our findings establish ultra-low-coverage scWGS as a powerful approach for studying active mutational processes that may influence ongoing clonal evolution and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6d1d9f289244977b2dfb8fbeaff86f943a227d" target='_blank'>
              Inferring active mutational processes in cancer using single cell sequencing and evolutionary constraints
              </a>
            </td>
          <td>
            Gryte Satas, Matthew A. Myers, Andrew McPherson, Sohrab P. Shah
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a small subset within the tumor mass significantly contributing to cancer progression through dysregulation of various oncogenic pathways, driving tumor growth, chemoresistance and metastasis formation. The aggressive behavior of CSCs is guided by several intracellular signaling pathways such as WNT, NF-kappa-B, NOTCH, Hedgehog, JAK-STAT, PI3K/AKT1/MTOR, TGF/SMAD, PPAR and MAPK kinases, as well as extracellular vesicles such as exosomes, and extracellular signaling molecules such as cytokines, chemokines, pro-angiogenetic and growth factors, which finely regulate CSC phenotype. In this scenario, tumor microenvironment (TME) is a key player in the establishment of a permissive tumor niche, where CSCs engage in intricate communications with diverse immune cells. The “oncogenic” immune cells are mainly represented by B and T lymphocytes, NK cells, and dendritic cells. Among immune cells, macrophages exhibit a more plastic and adaptable phenotype due to their different subpopulations, which are characterized by both immunosuppressive and inflammatory phenotypes. Specifically, tumor-associated macrophages (TAMs) create an immunosuppressive milieu through the production of a plethora of paracrine factors (IL-6, IL-12, TNF-alpha, TGF-beta, CCL1, CCL18) promoting the acquisition by CSCs of a stem-like, invasive and metastatic phenotype. TAMs have demonstrated the ability to communicate with CSCs via direct ligand/receptor (such as CD90/CD11b, LSECtin/BTN3A3, EPHA4/Ephrin) interaction. On the other hand, CSCs exhibited their capacity to influence immune cells, creating a favorable microenvironment for cancer progression. Interestingly, the bidirectional influence of CSCs and TME leads to an epigenetic reprogramming which sustains malignant transformation. Nowadays, the integration of biological and computational data obtained by cutting-edge technologies (single-cell RNA sequencing, spatial transcriptomics, trajectory analysis) has significantly improved the comprehension of the biunivocal multicellular dialogue, providing a comprehensive view of the heterogeneity and dynamics of CSCs, and uncovering alternative mechanisms of immune evasion and therapeutic resistance. Moreover, the combination of biology and computational data will lead to the development of innovative target therapies dampening CSC-TME interaction. Here, we aim to elucidate the most recent insights on CSCs biology and their complex interactions with TME immune cells, specifically TAMs, tracing an exhaustive scenario from the primary tumor to metastasis formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90ce167799371ac069e6df125e4a468fd20b0208" target='_blank'>
              Cancer stem cells and tumor-associated macrophages as mates in tumor progression: mechanisms of crosstalk and advanced bioinformatic tools to dissect their phenotypes and interaction
              </a>
            </td>
          <td>
            Francesco Verona, Sebastiano Di Bella, Roberto Schirano, Camilla Manfredi, Francesca Angeloro, Giulia Bozzari, M. Todaro, Giuseppe Giannini, Giorgio Stassi, V. Veschi
          </td>
          <td>2025-02-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="While pediatric cancer patients receive intensive chemotherapy, its impact on peripheral T cells and subsequently to disease outcomes are not fully characterized. Here, we assessed T-cell dynamics during treatment, identifying associations with outcomes through immune phenotyping and T-cell Receptor (TCR) sequencing in pediatric solid and hematologic malignancies. We show that while levels of immune checkpoint proteins (PD-1, LAG3, and TIM3) at baseline were highest in lymphomas compared to other cancer groups, they increased significantly in response to therapy in all cancers. Levels of Central Memory (CM) T cells increased in leukemias and solid tumors, while naïve T cells and cell-free TCR diversity decreased in lymphomas. By combining immune cell and TCR repertoire features across all timepoints, we proposed the Dynamic Immunogenomic Score (DIS) to measure patient-specific effects of therapy on the peripheral T-cell population. Higher DIS was associated with high-risk cancer types and logistic regression analysis revealed it may predict incidence of relapse in leukemia patients. TCR specificity analysis revealed patient-specific clonal dynamics and differential detection of virally-associated TCRs in cancer patients compared to healthy individuals. Our results highlight the potential of early upfront immunogenomic profiling in identifying high-risk patients that may be predictive in light of emerging cellular immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cebbb1d9bf5db9174d2b007c121d14b77fe78d1" target='_blank'>
              Immunogenomic profiling of circulating T cells in pediatric cancer patients during standard-of-care
              </a>
            </td>
          <td>
            A. Nabbi, Yiyue Jiang, Osvaldo Espin Garcia, Suluxan Mohanraj, S. Pedersen, Johanna Regala, Lauren M. Vernau, Jessica Perazzelli, David M. Barrett, Trevor J. Pugh
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background & Aims
Identifying patients with intrahepatic cholangiocarcinoma (ICC) likely to benefit from immunochemotherapy, the new front-line treatment, remains challenging. We aimed to unveil a novel radiotranscriptomic signature that can facilitate treatment response prediction by multi-omics integration and multi-scale modelling.


Methods
We analyzed bulk, single-cell and spatial transcriptomic data comprising 457 ICC patients to identify an immune-related score (IRS), followed by decoding its spatial immune context. We mapped radiomics profiles onto spatial-specific IRS using machine-learning to define a novel radiotranscriptomic signature, followed by multi-scale and multi-cohort validation covering 331 ICC patients. The signature was further explored for the potential therapeutic target from in vitro to in vivo.


Results
We revealed a novel 3-gene (PLAUR, CD40LG and FGFR4) IRS whose down-regulation correlated with better survival and improved sensitivity to immunochemotherapy. We highlighted functional IRS-immune interactions within tumor epithelium, rather than stromal compartment, irrespective of geospatial locations. Machine-learning pipeline identified the optimal 3-feature radiotranscriptomic signature that was well-validated by immunohistochemical assays in molecular cohort, exhibited favorable external prognostic validity with C-index over 0.64 in resection cohort, and predicted treatment response with an area under the curve of up to 0.84 in immunochemotherapy cohort. We also showed that anti-uPAR/PLAUR alone or in combination with anti-programmed cell death protein 1 therapy remarkably curbed tumor growth, using in vitro ICC cell lines and in vivo humanized ICC patient-derived xenograft mouse models.


Conclusions
This proof-of-concept study sheds light on the spatially-resolved radiotranscriptomic signature to improve patient selection for emerging immunochemotherapy and high-order immunotherapy combinations in ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b216340a3ee900bc1d9420b9d6cc7048a9f68c8" target='_blank'>
              Radiogenomics of intrahepatic cholangiocarcinoma predicts immunochemotherapy response and identifies therapeutic target.
              </a>
            </td>
          <td>
            Gu-Wei Ji, Zheng Xu, Shuo-Chen Liu, Shu-Ya Cao, C. Jiao, Ming Lu, Biao Zhang, Yue Yang, Qing Xu, Xiao-Feng Wu, Ke Wang, Yong-xiang Xia, Xiang-Cheng Li, Xue-Hao Wang
          </td>
          <td>2025-02-10</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd9f723993b9fbc9144678833cdc43c46f60f6e1" target='_blank'>
              Identification of Meibomian gland stem cell populations and mechanisms of aging
              </a>
            </td>
          <td>
            Xuming Zhu, Mingang Xu, Céline Portal, Yvonne Lin, Alyssa Ferdinand, Tien Peng, Edward E. Morrisey, Andrzej A Dlugosz, Joseph M. Castellano, Vivian Lee, John Seykora, Sunny Y Wong, Carlo Iomini, Sarah E Millar
          </td>
          <td>2025-02-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Spatially resolved transcriptomics (SRT) has emerged as a transformative technology for elucidating cellular organization and tissue architecture. However, a significant challenge remains in identifying pathology-relevant spatial functional landscapes within the tissue microenvironment, primarily due to the limited integration of cell-cell communication dynamics. To address this limitation, SpaDCN, a Spatially Dynamic graph Convolutional Network framework is proposed, which aligns cell-cell communications and gene expression within a spatial context to reveal the spatial functional regions with the coherent cellular organization. To effectively transfer the influence of cell-cell communications on expression variation, SpaDCN respectively generates the node layer and edge layer of spatial graph representation from expression data and the ligand-receptor complex contributions and then employs a dynamic graph convolution to switch the propagation of node graph and edge graph. It is demonstrated that SpaDCN outperforms existing methods in identifying spatial domains and denoising expression across various platforms and species. Notably, SpaDCN excels in identifying marker genes with significant prognostic potential in cancer tissues. In conclusion, SpaDCN offers a powerful and precise tool for spatial domain detection in spatial transcriptomics, with broad applicability across various tissue types and research disciplines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e1ac36ae0a43d7b28635df6f8edaf8554a72a9" target='_blank'>
              SpaDCN: Deciphering Spatial Functional Landscape from Spatially Resolved Transcriptomics by Aligning Cell-Cell Communications.
              </a>
            </td>
          <td>
            Xiaosheng Bai, Xinyu Bao, Chuanchao Zhang, Qianqian Shi, Luonan Chen
          </td>
          <td>2025-02-17</td>
          <td>Small methods</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: Melanoma is a highly heterogeneous disease, and a deeper molecular classification is essential for improving patient stratification and treatment approaches. Here, we describe the histopathology-driven proteogenomic landscape of 142 treatment-naïve metastatic melanoma samples to uncover molecular subtypes and clinically relevant biomarkers. Methods: We performed an integrative proteogenomic analysis to identify proteomic subtypes, assess the impact of BRAF V600 mutations, and study the molecular profiles and cellular composition of the tumor microenvironment. Clinical and histopathological data were used to support findings related to tissue morphology, disease progression, and patient outcomes. Results: Our analysis revealed five distinct proteomic subtypes that integrate immune and stromal microenvironment components and correlate with clinical and histopathological parameters. We demonstrated that BRAF V600-mutated melanomas exhibit biological heterogeneity, where an oncogene-induced senescence-like phenotype is associated with improved survival. This led to a proposed mortality risk-based stratification that may contribute to more personalized treatment strategies. Furthermore, tumor microenvironment composition strongly correlated with disease progression and patient outcomes, highlighting a histopathological connective tissue-to-tumor ratio assessment as a potential decision-making tool. We identified a melanoma-associated SAAV signature linked to extracellular matrix remodeling and SAAV-derived neoantigens as potential targets for anti-tumor immune responses. Conclusions: This study provides a comprehensive stratification of metastatic melanoma, integrating proteogenomic insights with histopathological features. The findings may aid in the development of tailored diagnostic and therapeutic strategies, improving patient management and outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557557036af2f6be01a0b554ada4c910b8468881" target='_blank'>
              Proteogenomic Profiling of Treatment-Naïve Metastatic Malignant Melanoma
              </a>
            </td>
          <td>
            M. Kuras, L. Betancourt, R. Hong, L. Szadai, Jimmy Rodriguez, Peter Horvatovich, I. Pla, Jonatan Eriksson, B. Szeitz, Bartłomiej Deszcz, Charlotte Welinder, Y. Sugihara, Henrik Ekedahl, Bo Baldetorp, C. Ingvar, L. Lundgren, H. Lindberg, Henriett Oskolás, Z. Horváth, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Johan Malm, Aniel Sanchez, A. M. Szasz, Krzysztof Pawłowski, E. Wieslander, David Fenyö, I. Németh, Gyorgy Marko-Varga
          </td>
          <td>2025-02-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 760


 Background:
 Pancreatic adenosquamous carcinoma (PASC) is a rare histopathological type of pancreatic cancer that is more aggressive than pancreatic ductal adenocarcinoma (PDAC). Although PASC has a unique tumor biology and poorer prognosis, there are no guidelines for differentially treating PASC patients, and its rarity makes it significantly understudied. We used Nanostring GeoMX digital spatial profiling (DSP) to provide a targeted proteomic characterization of key tumor-promoting proteins in PASC and PDAC.
 Methods:
 FFPE tissue from PASC (n=10 primary, n= 4 met) and PDAC (n=21 primary, n=5 met) cases were analyzed using the DSP platform. Sections were stained with an 82-protein cocktail of antibodies conjugated to UV-photocleavable DNA barcodes. A pan-cytokeratin morphology marker was used to select tumor regions of interest (ROI) and inferred surrounding stromal (i.e., pan-CK negative) ROIs. UV excitation of the defined ROIs was used to release the DNA barcodes and were quantified on the NanoString nCounter system. For PASC, squamous enriched ROI were inferred from corresponding p40 IHC. Corresponding clinical and mutational status for all cases were abstracted from medical records.
 Results:
 TP53 (PDAC, 80%; PASC, 83%) and KRAS (PDAC, 76%; PASC, 100%) were the most prevalent mutations
 .
 In comparing the tumor ROIs of PASC and PDAC (all cases), DSP-assessed global expression and activation of EGFR-MAPK signaling is significantly increased in PASC, whereas PI3K-AKT signaling is significantly decreased. Compared to PDAC, EGFR expression and phosphorylation (p-EGFR), as well as p-MEK1, are significantly increased in PASC. PASC showed significantly lower levels of PLCG1 and INPP4B, while the majority of other AKT signaling effectors were unchanged. For DNA damage response signaling, PASC showed significantly higher levels of p-ATM, p-CHK2, and p21, but not p-ATR. Likewise, expression of oncoproteins c-MYC and BCL6 were significantly increased in PASC (vs. PDAC).
 Conclusions:
 Given that the normal pancreas lacks squamous cells, their histological presence is a uniquely aberrant feature that can be used to guide deep spatially-resolved interrogation of complex cellular and molecular programs. Our preliminary findings show that regions of squamous cell differentiation within PASC have intrinsically higher levels of active EGFR signaling. With emerging RAS-targeted agents poised to alter the treatment landscape, high EGFR signaling in PASC suggest a possible compensatory resistance mechanism that may require additional targeting. Likewise, reliance on ATM signaling highlights this as preferential target for DDR-directed therapies. Building on these results with additional multiomic assays will inform strategies for improving the management of patients with PASC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b1608b4f56eb9c6e1216a160d63214aa03c88b" target='_blank'>
              Spatially resolved protein profiling in pancreatic adenosquamous carcinoma.
              </a>
            </td>
          <td>
            Adel Kardosh, Rosalie C. Sears, Zheng Xia, Eizabaru M Sasatomi, Brian Mau, Jonathan R Brody, Melissa Wong, Charles D. Lopez, E. Chen, Guillaume Pegna, Cristiane M Franca, John Muschler, Joanna Pucilowska, Shaun M Goodyear
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997e954cb156caa69cf5f6fb240e0f9297f38fc1" target='_blank'>
              Single-cell and spatial-resolved profiling reveals cancer-associated fibroblast heterogeneity in colorectal cancer metabolic subtypes
              </a>
            </td>
          <td>
            Youpeng Wang, Xingfeng Qiu, Qinghai Li, Jiale Qin, Lvlan Ye, Xiang Zhang, Xingxiang Huang, Xiangqiong Wen, Ziyang Wang, Weiling He, Yuqin Di, Qi Zhou
          </td>
          <td>2025-02-11</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e89dc2d087d9d2c969c5bb0a4b45e01fc9cc0f7a" target='_blank'>
              Single-cell and spatial genomic landscape of non-small cell lung cancer brain metastases.
              </a>
            </td>
          <td>
            Somnath Tagore, Lindsay A. Caprio, Amit Dipak Amin, K. Bestak, Karan Luthria, Edridge K. D’Souza, Irving Barrera, Johannes C. Melms, Sharon Wu, Sinan Abuzaid, Yiping Wang, Viktoria Jakubikova, Peter Koch, D. Z. Brodtman, Banpreet Bawa, S. Deshmukh, Leon Ebel, Miguel A. Ibarra-Arellano, A. Jaiswal, Carino Gurjao, Jana Biermann, Neha Shaikh, Priyanka Ramaradj, Yohanna Georgis, Galina G Lagos, Matthew I Ehrlich, P. Ho, Zachary H. Walsh, Meri Rogava, Michelle Garlin Politis, Devanik Biswas, A. Cottarelli, Nikhil Rizvi, CA Shu, Benjamin O Herzberg, N. Anandasabapathy, G. Sledge, Emmanuel Zorn, Peter D Canoll, Jeffrey N. Bruce, Naiyer A Rizvi, Alison M Taylor, Anjali Saqi, H. Hibshoosh, Gary K Schwartz, Brian S Henick, Fei Chen, Denis Schapiro, Parin Shah, Benjamin Izar
          </td>
          <td>2025-02-27</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Mouse models that faithfully represent the biology of human brain tumors are critical tools for unraveling the underlying tumor biology and screening for potential precision therapies. This is especially true of rare tumor types, many of which have correspondingly few xenograft or cell lines available. Although our understanding of the specific biological pathways driving cancer has improved significantly, identifying the appropriate progenitor populations to drive oncogenic processes represents a significant barrier to efficient mouse model production. In this issue of Cancer Research, Jessa and colleagues developed an innovative transcription factor fingerprinting method to map the cellular origin of central nervous system neuroblastoma, FOXR2-activated to medial ganglionic eminence-derived interneurons, which could then be efficiently targeted in the developing mouse brain using in utero electroporation. This approach serves as a blueprint for investigating other rare pediatric brain tumors, potentially accelerating progress toward the development of mouse models and identification of effective therapies. See related article by Jessa et al., p. 231.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b36b1d9ff773da8434bf45b0c13f61a915472b5f" target='_blank'>
              Transcription Factor Fingerprint Provides Clues for Brain Tumor Cell of Origin.
              </a>
            </td>
          <td>
            Brent A. Orr
          </td>
          <td>2024-01-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-grade gliomas are aggressive, primary, central nervous system tumors with low survival rates due to recurrence and resistance to current therapy models. Recent studies have highlighted the importance between the interaction of glioma cancer cells and cells of the tumor microenvironment (TME). Cancer stem cells and immune cells play a critical role in the TME of gliomas. TMEs in glioma include the perivascular TME, hypoxic TME, and invasive TME, each of which have evolved as our understanding of the involved cellular players has improved. This review discusses the multidimensional aspects of the current targeted therapies and interactions between glioma cells and the TME with specific focus on targeted immunotherapies. Understanding the complexities of the TME and elucidating the various tumor-cell interactions will be critical for facilitating the development of novel precision strategies, ultimately enabling better patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e6f6ef570f7a7d3e8fff16bd7c3861aa6a70" target='_blank'>
              Current Understanding Regarding the Glioma Microenvironment and Impact of the Immune System
              </a>
            </td>
          <td>
            Enes Demir, Deondra A. Montgomery, Ammar Saloum, Nasser Yaghi, Michael Karsy
          </td>
          <td>2025-03-07</td>
          <td>Neuroglia</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 256


 Background:
 Resistance to Androgen Receptor Pathway Inhibitors (ARPI) in metastatic prostate cancer (mPC) is universal and can be driven by complex genomic alterations. The evolution of lineage state transitions from adenocarcinomas to neuroendocrine prostate cancer (NEPC) has also been shown to drive treatment resistance and poor survival. Identifying the timing and association of genomic mutations with lineage state transitions has been limited by the need for serial tumor biopsies. We report an integrated analysis of clinical next-generation sequencing (NGS) data and mPC lineage states with a novel circulating tumor cell (CTC) RNA sequencing methodology.
 Methods:
 We collected 273 samples from 117 unique patients with mPC in a prospective biomarker trial. CTCs were purified via immunomagnetic capture on an automated microfluidic technology and analyzed via RNA-seq. A subset of patients had a clinical grade genetic test performed on a contemporary metastatic tissue biopsy. We compared CTC gene expression for lineage states (luminal A, luminal B, NEPC) with somatic gene mutation subtypes known to confer resistance to ARPIs (AR, p53, RB, PTEN, HRR genes) and overall survival (OS).
 Results:
 Single sample pathway analysis of high CTC purity samples identified four transcriptional phenotypes: luminal A-like (LumA), luminal B-like (LumB), low proliferation (LP), and neuroendocrine (NE). Compared to patients with low CTC burden/purity (median OS not reached), patients with LumA and LP phenotypes had similar survival, patients with LumB and NE CTC phenotypes had shorter survival (LumB: median OS 6m, HR 9.1 [3.8-21.8], p<0.0001, NE: median OS 3.7m, HR 11.8 [2.4-57.2], p=0.0019). AR alterations were found in samples with prior ARPI exposure and at similar frequencies across all CTC phenotypes but NE where they were absent. RB mutations were enriched in the unfavorable CTC phenotypes (LumB, NE; p=0.0288). Integrating CTC phenotype with presence or absence of at least one high risk genomic alteration (AR, RB, TP53, PTEN), patients with favorable CTC phenotype (Low burden, LP and LumA) and no high risk alterations had the longest median survival (median OS NR), followed by patients with favorable phenotype but a high risk alteration (median OS 12.7m), patients with an unfavorable phenotype (LumB, NE) but no high risk alteration (median OS 8m), and unfavorable phenotype with a high risk alteration (median OS 4m).
 Conclusions:
 Somatic mutations in mCRPC influence lineage state acquisitions and treatment resistance. Lineage state transitions themselves are associated with poor outcomes and decreased survival which is worsened in the presence of high-risk genetic mutations including RB. This data presents potentially targetable patient populations that would benefit from treatment intensification and early disease monitoring for more aggressive mCRPC subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dba71a7a1acb2ee64ae5bd588ebd4c00f7170961" target='_blank'>
              Investigating clinical associations between genomic alterations and prostate cancer lineage states using circulating tumor cell RNA sequencing.
              </a>
            </td>
          <td>
            Katharine E Tippins, Joshua M Lang, Marina N. Sharifi, J. Sperger, Shannon R Reese, Amy K Taylor, Viridiana Carreno, Alex H Chang, Meghan Wells, John M Floberg, H. Emamekhoo, Shuang Zhao, D. Kosoff
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="T cells play diverse roles in cancer immunology, acting as tumor suppressors, cytotoxic effectors, enhancers of cytotoxic T lymphocyte responses and immune suppressors; providing memory and surveillance; modulating the tumor microenvironment (TME); or activating innate immune cells. However, cancer cells can disrupt T cell function, leading to T cell exhaustion and a weakened immune response against the tumor. The expression of exhausted T cell (Tex) markers plays a pivotal role in shaping the immune landscape of multiple cancers. Our aim was to systematically investigate the role of known T cell exhaustion (Tex) markers across multiple cancers while exploring their molecular interactions, mutation profiles, and potential implications for immunotherapy. The mRNA expression profile of six Tex markers, LAG-3, PDCD1, TIGIT, HAVCR2, CXCL13, and LAYN was investigated in pan-cancer. Utilizing data from The Cancer Genome Atlas (TCGA), Gene Expression Omnibus (GEO), The Cancer Proteome Atlas (TCPA), and other repositories, we characterized the differential expression of the Tex markers, their association with the patients’ survival outcome, and their mutation profile in multiple cancers. Additionally, we analyzed the effects on cancer-related pathways and immune infiltration within the TME, offering valuable insights into mechanisms of cancer immune evasion and progression. Finally, the correlation between their expression and sensitivity to multiple anti-cancer drugs was investigated extensively. Differential expression of all six markers was significantly associated with KIRC and poor prognosis in several cancers. They also played a potential activating role in apoptosis, EMT, and hormone ER pathways, as well as a potential inhibitory role in the DNA damage response and RTK oncogenic pathways. Infiltration of different immune cells was also found to be associated with the expression of the Tex-related genes in most cancer types. These findings underline that the reviving of exhausted T cells can be used to enhance the efficacy of immunotherapy in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b32105e8581ecae32317ce64eca6316a45d0d52" target='_blank'>
              Characterization of Exhausted T Cell Signatures in Pan-Cancer Settings
              </a>
            </td>
          <td>
            Rifat Tasnim Juthi, Saiful Arefeen Sazed, Manvita Mareboina, A. Zaravinos, Ilias Georgakopoulos-Soares
          </td>
          <td>2025-03-05</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49801de5fa9e55e176d4a57170a423c31755f306" target='_blank'>
              High-resolution, noninvasive single-cell lineage tracing in mice and humans based on DNA methylation epimutations.
              </a>
            </td>
          <td>
            Mengyang Chen, Ruijiang Fu, Yiqian Chen, Li Li, Shoucheng Wang
          </td>
          <td>2025-01-16</td>
          <td>Nature methods</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1f7c17606ae67d5ce7fe611db0e7bce958444bb" target='_blank'>
              Decoding the heterogeneous subpopulations of glioblastoma for prognostic stratification and uncovering the promalignant role of PSMC2
              </a>
            </td>
          <td>
            Nini Zhou, Jingsong Yan, Manya Xiong, Shunqin Zhu
          </td>
          <td>2025-02-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa45e1efe02786e2e4534cf1ae408e65c7ffadc" target='_blank'>
              To describe the subsets of malignant epithelial cells in gastric cancer, their developmental trajectories and drug resistance characteristics
              </a>
            </td>
          <td>
            Tingting Xu, Tianying Zhang, Yan Sun, Sijia Wu
          </td>
          <td>2025-01-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba7e1fd60eab1467a858b79695c0c9e7c85c584" target='_blank'>
              Mapping chromatin remodelling in glioblastoma identifies epigenetic regulation of key molecular pathways and novel druggable targets
              </a>
            </td>
          <td>
            C. Vinel, James Boot, Weiwei Jin, N. Pomella, Alexandra Hadaway, C. Mein, N. Zabet, Silvia Marino
          </td>
          <td>2025-02-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background/Objectives: The heterogeneity of the tumor immune microenvironment is a key determinant of tumor oncogenesis. This study aims to evaluate the composition of seven immune cells across 5323 samples from 14 cancers using DNA methylation data. Methods: A deconvolution algorithm was proposed to estimate the composition of seven immune cells using 1256 immune cell population-specific methylation genes. Based on the immune infiltration features of seven immune cell fractions, 42 subtypes of 14 tumors (2–5 subtypes per tumor) were identified. Results: Significant differences in immune cells between subtypes were revealed for each cancer. The study found that the methylation values of the selected specific sites correlated with gene expression in most tumor subtypes. Immune infiltration results were integrated with phenotypic data, including survival data and tumor stages, revealing significant correlations between immune infiltration and phenotypes in some tumors. Subtypes with high proportions of CD4+ T cells, CD8+ T cells, CD56+ NK cells, CD19+ B cells, CD14+ monocytes, neutrophils, and eosinophils were identified, with subtype counts of 9, 24, 22, 13, 19, 9, and 11, respectively. Additionally, 2412 differentially expressed genes between these subtypes and normal tissues were identified. Pathway enrichment analysis revealed that these genes were mainly enriched in pathways related to drug response and chemical carcinogens. Differences in ESTIMATE scores for subtypes of seven tumors and TIDE scores for eight tumors were also observed. Conclusions: This study demonstrates the intra-tumor and inter-tumor immune heterogeneity of pan-cancer through DNA methylation analysis, providing assistance for tumor diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adddec60e9424b127df5b7066697b39696f5a789" target='_blank'>
              Evaluation of Pan-Cancer Immune Heterogeneity Based on DNA Methylation
              </a>
            </td>
          <td>
            Yang Zhou, Jiebiao Liu, Bowen Shi, Te Ma, Peishen Yu, Ji Li, Yue Gu, Yan Zhang
          </td>
          <td>2025-01-26</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Glioblastoma (GBM) is the most common and aggressive primary malignant brain tumor of adults. Current therapeutic options yield dismal prognoses that have remained essentially unchanged over nearly two decades. Diffuse growth patterns, high intratumoral heterogeneity, and variable blood–brain barrier integrity limit treatment efficacy, creating challenges that rational small molecule design has not overcome. Antibody–drug conjugates have shown some promise, leading us to hypothesize that smaller folded DNA aptamers, developed in vivo via principles of natural selection, might eventually have advantages for drug delivery. Here, we document the first in vivo DNA aptamer selection involving an orthotopic patient-derived xenograft GBM mouse model to identify tumor-homing DNA aptamers. We demonstrate the preferential accumulation of these aptamers in the tumor relative to other tissues 4 h after intraperitoneal injection. The aptamers can be detected by quantitative polymerase chain reaction, fluorescent tumor staining, and stain GBM tumor section from untreated mice and the GBM tumor cells in culture. Two of three candidates are selective for the target cell line in vitro and do not bind other human tumor cells. In vivo selection of tumor-specific DNA aptamers demonstrates a novel approach for diagnostics or toxin delivery that might allow for the development of individualized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a69e4c2d2724fca734736e524ece9241f19793a" target='_blank'>
              In vivo selection of anti-glioblastoma DNA aptamers in an orthotopic patient-derived xenograft model
              </a>
            </td>
          <td>
            Caroline D Doherty, Brandon A Wilbanks, Sonia Jain, Keenan S Pearson, Katie K Bakken, D. Burgenske, Nay Won Lett, J. Sarkaria, L. Maher
          </td>
          <td>2025-02-18</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Liver and lung metastases are major drivers of breast cancer-related morbidity, characterized by poor therapeutic outcomes and diminished overall survival. These dynamic tumor microenvironments (TMEs) display distinct, site-specific cellular architectures, including cytotoxic lymphocytes, immune suppressor cells, and stromal components, which together facilitate tumor growth and metastatic spread. Varied responses to immune checkpoint inhibition have also been observed in patients with differential tumor burden, improved responses in those with higher lung metastatic burden and poor responses in those with higher liver metastatic burden. Specific immune populations driving observed differential responses have not been fully characterized. Utilizing syngeneic orthotopic mouse models and FFPE tissue samples from patients with primary and metastatic breast cancer, we investigated the organ-specific cellular composition, cell-cell interactions, and molecular pathways underlying tumor progression, immune escape, and site-specific metastasis. Flow cytometry analysis of orthotopic murine primary breast, breast-to-liver, and breast-to-lung tumors demonstrated that CD3+ lymphocytes initially accumulate in primary tumors soon after tumor inoculation and decline significantly over time. Despite this overall lymphocyte depletion within TMEs, the lungs consistently exhibit the highest lymphocyte levels. In contrast, myeloid-derived suppressor cells (MDSCs) progressively infiltrate both primary and distant sites, exerting immunosuppressive functions that likely promote tumor progression and inhibit immune responses. Both granulocytic and monocytic MDSCs suppress CD8+ T cell proliferation through distinct organ-specific mechanisms, with the strongest suppressive effects observed in the liver TME. Validation of these findings in patient specimens using high-resolution spatial transcriptomics from breast and distant breast metastases using 10X Genomics (Visium and Visium-HD platforms) revealed that liver metastases harbor lower immune-to-stromal ratio with greater MDSCs: CD8+ colocalization, and higher expression of genes associated with proliferation, angiogenesis, anti-apoptosis and antiinflammation relative to primary breast tumors. In addition, spatiotemporal plasticity of breast TMEs is specific to the disease site, and investigation of how these observations could impact tumor progression and response to immune checkpoint inhibition is underway. In summary, tissue-specific programming of MDSCs in breast-to-liver metastasis compared with primary and breast-to-lung lesions leads to reduced T cell infiltration and function, and could contribute to a lack of response to checkpoint inhibition and could guide therapeutic options for patients.
 Citation Format: Batul Al-zubeidy, Edgar Gonzalez, Aaron Baugh, Dominic Zavala, Mathew Jacobo, Michael Press, Evanthia Roussos Torres. Myeloid suppression within metastatic tumor microenvironments in advanced breast cancer—friend or foe to checkpoint inhibition? [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A086.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01243dcc84f5d3d7fbdd4aa36ab8b1b63e350fae" target='_blank'>
              Abstract A086: Myeloid suppression within metastatic tumor microenvironments in advanced breast cancer—friend or foe to checkpoint inhibition?
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Mathew Jacobo, Michael F. Press, E. R. Torres
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Intrahepatic cholangiocarcinoma (iCCA) may be genomically subclassified by the presence of potentially actionable molecular aberrations, of which pathogenic alterations in isocitrate dehydrogenase (IDH)1 and fibroblast growth factor receptor (FGFR)2 are the most frequently observed. The impact of these molecular alterations on the tumor immune microenvironment remains incompletely understood. Methods: We performed a high-parameter spatial immune phenotyping of iCCA samples with pathogenic FGFR2 or IDH1 alterations and FGFR2/IDH1 wild-type controls at the single-cell level using CO-Detection by indEXing. Results:﻿ A total of 24 tumors were examined. Tumors with FGFR2 alterations were characterized by fewer CD8+ T cells and “M2-like” macrophages but higher levels of polymorphonuclear myeloid-derived suppressor cells as compared to FGFR2 wild-type tumors. Spatial relationships between polymorphonuclear myeloid-derived suppressor cells and multiple other cell types in the tumor microenvironment (including tumor cells, CD4+, and CD8+ T cells) were enriched in tumors with FGFR2 alterations. Tumors with IDH1 mutations had a trend toward more fibroblasts and were characterized by a closer proximity of tumor cells to CD4+ T cells, and between macrophages and multiple structural tumor microenvironment components as compared to other subtypes. Conclusions: iCCAs with pathogenic FGFR2 fusions/rearrangements and IDH1 mutations have distinct immunophenotypes. Tailoring immunotherapeutic approaches to specific molecular subsets could improve treatment outcomes across the divergent molecularly defined iCCA subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/343c87af8f0d2b156de3f86c823598f7aa36e2b7" target='_blank'>
              Deep immune profiling of intrahepatic cholangiocarcinoma with CODEX multiplexed imaging
              </a>
            </td>
          <td>
            M. Baretti, Soumya Shekhar, Vaibhav Sahai, Daniel H. Shu, Kathryn Howe, Valerie Gunchick, N. Assarzadegan, Emma Kartalia, Qingfeng Zhu, Elsa Hallab, Archit Sheth-Shah, Aya Kondo, Nilofer S Azad, M. Yarchoan
          </td>
          <td>2025-02-19</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Solid tumours are intricate and highly heterogeneous ecosystems, which grow in and invade normal organs. Their progression is mediated by cancer cells’ interaction with different cell types, such as immune cells, stromal cells and endothelial cells, and with the extracellular matrix. Owing to its high incidence, aggressive growth and resistance to local and systemic treatments, liver cancer has particularly high mortality rates worldwide. In recent decades, spatial heterogeneity has garnered significant attention as an unfavourable biological characteristic of the tumour microenvironment, prompting extensive research into its role in liver tumour development. Advances in spatial omics have facilitated the detailed spatial analysis of cell types, states and cell‒cell interactions, allowing a thorough understanding of the spatial and temporal heterogeneities of tumour microenvironment and informing the development of novel therapeutic approaches. This review illustrates the latest discovery of the invasive zone, and systematically introduced specific macroscopic spatial heterogeneities, pathological spatial heterogeneities and tumour microenvironment heterogeneities of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27ba251a9613c3090a1b350987469c7d7cfc3f02" target='_blank'>
              Spatial‒temporal heterogeneities of liver cancer and the discovery of the invasive zone
              </a>
            </td>
          <td>
            Jiayan Yan, Zhifeng Jiang, Shiyu Zhang, Qichao Yu, Yijun Lu, Runze Miao, Zhaoyou Tang, Jia Fan, Liang Wu, Dan G. Duda, Jian Zhou, Xinrong Yang
          </td>
          <td>2025-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Uveal melanoma is the most common type of eye cancer, with primary tumors typically originating in the choroid, though they can also develop in the iris or ciliary body. The condition is associated with high mortality due to metastases, particularly to the liver, which are often resistant to treatment. An important factor in this resistance is the presence of cancer stem cells or progenitor cells. These cells have the ability to self-renew and differentiate into diverse cell types, contributing to the development of resistant and heterogeneous cancer cell populations. There have been numerous efforts to identify cancer stem cell markers across various types of cancer, including uveal melanoma. This paper aims to review stem cell markers, such as CD133 and CD166, common to multiple cancers, alongside markers more specific to melanoma and uveal melanoma, including SOX2 and nestin. Despite extensive research, a definitive characterization of cancer stem cells in uveal melanoma has yet to be achieved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/903982cf411415d6dcdb66121d6c8e705f6172b9" target='_blank'>
              The Role of Cancer Stem Cells in Uveal Melanoma
              </a>
            </td>
          <td>
            Piotr Świerzewski, Bożena Romanowska-Dixon, Martyna Elas
          </td>
          <td>2025-02-14</td>
          <td>Ophthalmology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Afatinib-induced tumor and microenvironment modifications in head and neck squamous cell carcinoma were evaluated by spatial transcriptomics in surgical specimens and RNA-sequencing in tumor biopsies of patients included in the EORTC-90111-24111 window-of-opportunity study. The aim was to explore tumor evolution and composition under anti-HER therapy. Based on our previous investigations by RNA-seq on tumor biopsies, surgical slides of ID08 and ID15 from the epithelial-to-mesenchymal (EMT) cluster and ID30 from the non-EMT cluster were investigated with spatial transcriptomics. Dimension reduction in ID30 revealed 14 clusters, with clusters overlapping three tumor nodules and the stroma. Differential expression analysis between tumor nodules showed enrichment of the hallmark EMT genelist, with 123 genes in common between the analyses. These genes were involved in PDGF and MET signaling pathways. By comparing gene expression in paired tumor biopsies and the 123 genes from differential analyses obtained in ID30, a list of 13 genes involved in cancer pathways and EMT emerged, which were also highly expressed in ID08 and ID15. These results show a progressive apparition of genes implicated in EMT, MET, and PDGF pathways in tumors after afatinib. Notably, a list of 13 genes emerged which may contain targets to prevent tumor evolution after anti-HER therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ce3fcfb23899394262473ee209f0d7889b6808e" target='_blank'>
              Spatial Transcriptomic Analysis of Surgical Resection Specimens of Primary Head and Neck Squamous Cell Carcinoma Treated with Afatinib in a Window-of-Opportunity Study (EORTC90111-24111)
              </a>
            </td>
          <td>
            S. Beyaert, A. Loriot, J. Machiels, S. Schmitz
          </td>
          <td>2025-02-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Abstract Oral squamous cell carcinoma (OSCC) is an increasingly prevalent malignancy worldwide. This study aims to understand molecular alterations associated with lymph node metastasis of OSCC in order to improve treatment strategies. We analysed a cohort of 46 patients with primary OSCC, including 10 with lymph node metastasis and 36 without. Using a comprehensive multi‐omics approach – encompassing genomic, transcriptomic, proteomic, epigenetic, single‐cell, and spatial analyses – we integrated data to delineate the molecular landscape of OSCC in the context of lymph node metastasis. Our genomic analysis identified significant mutations in key genes within the MAPK, TGF‐β and WNT signalling pathways, which are essential for tumour development. The proteogenomic analysis highlighted pathways critical for lymph node dissemination and factors contributing to an immunosuppressive tumour microenvironment. Elevated levels of POSTN were found to reorganise the extracellular matrix (ECM), interact with TGF‐β, disrupt cell cycle regulation and suppress the immune response by reducing VCAM1 activity. Integrated analyses of single‐cell and spatial transcriptome data revealed that cancer‐associated fibroblasts (CAFs) secrete TGF‐β1/2, promoting cancer cell metastasis through epithelial–mesenchymal transition (EMT). Our integrated multi‐omics analysis provides a detailed understanding of molecular mechanisms driving lymph node metastasis of OSCC. These insights could lead to more precise diagnostics and targeted treatments. This article is protected by copyright. All rights reserved">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a9d23bd90d2baf9dbe230266b6f429b3c96696a" target='_blank'>
              Proteogenomic characterisation of primary oral cancer unveils extracellular matrix remodelling and immunosuppressive microenvironment linked to lymph node metastasis
              </a>
            </td>
          <td>
            Yu Liu, Zhenyu Yang, J. Pu, Jie Zhong, U. Khoo, , Gao Zhang
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abstract A cell fate change such as tumorigenesis incurs critical transition. It remains a longstanding challenge whether the underlying mechanism can be unraveled and a molecular switch that can reverse such transition is found. Here a systems framework, REVERT, is presented with which can reconstruct the core molecular regulatory network model and a reversion switch based on single‐cell transcriptome data over the transition process is identified. The usefulness of REVERT is demonstrated by applying it to single‐cell transcriptome of patient‐derived matched organoids of colon cancer and normal colon. REVERT is a generic framework that can be applied to investigate various cell fate transition phenomena.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd1a9318f2abd76a8f7539133f45f2810ab7c74" target='_blank'>
              Attractor Landscape Analysis Reveals a Reversion Switch in the Transition of Colorectal Tumorigenesis
              </a>
            </td>
          <td>
            Dongkwan Shin, Jeong-Ryeol Gong, Seoyoon D Jeong, Youngwon Cho, Hwang‐Phill Kim, Tae-You Kim, Kwang-Hyun Cho
          </td>
          <td>2025-01-22</td>
          <td>Advanced Science</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Fanconi Anemia (FA) is a heritable syndrome characterized by DNA damage repair deficits, frequent malformations and a significantly elevated risk of bone marrow failure, leukemia, and mucosal head and neck squamous cell carcinomas (HNSCC). Hematopoietic stem cell gene therapy can prevent marrow failure and lower leukemia risk, but mucosal gene therapy to lower HNSCC risk remains untested. Major knowledge gaps include an incomplete understanding of how rapidly gene-corrected cellular lineages could spread through the oral epithelium, and which delivery parameters are critical for ensuring efficient gene correction. To answer these questions, we extended an agent-based model of the oral epithelium to include the delivery of gene correction in situ to FA cells and the competitive dynamics between cellular lineages with and without gene correction. We found that only gene-corrected lineages with substantial proliferative advantages (probability of resisting displacement out of the basal layer ≥ 0. 1) could spread on clinically relevant timelines, and that these lineages were initially at high risk of loss in the generations following correction. Delivering gene correction to many cells minimizes the risk of loss, while delivery to many distinct locations within a tissue maximizes the rate of spread. To determine the impact of mucosal gene therapy in preventing the clonal expansion of pre-cancerous mutations, we compared the expected burden of TP53 mutations in simulated tissue sections with and without gene correction. We found that when FA cells have elevated genome instability or a TP53-dependent proliferative advantage, gene correction can substantially reduce the accumulation of pro-tumorigenic mutations. This model illustrates the power of computational frameworks to identify critical determinants of therapeutic success to enable experimental optimization and support novel and effective gene therapy applications. Author summary We investigated factors influencing the success of oral mucosal gene therapy for Fanconi Anemia (FA), a genetic syndrome marked by DNA repair defects in conjunction with a heightened risk of cancer. We used a computational model of the oral epithelium to determine how gene therapy corrected cells compete with FA background cells and the best gene delivery approaches to promote effective tissue replacement by gene-corrected cells. We find that gene-corrected cells require strong proliferative advantages to spread effectively, and that initially delivering gene correction to more cells reduces the chance that these cells are stochastically eliminated before they can spread. We also demonstrate that gene correction reduces the accumulation of pro-tumorigenic TP53 mutations in an FA context, where genomic instability can elevate the mutation rate and FA-specific selective pressures could favor accelerated TP53 clonal expansion. This research provides a useful framework for guiding mucosal gene therapy experiments and the development of effective oral gene therapy protocols for cancer prevention in FA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb3b56fe96b81aec45dd517d7b11cc8ca9c35" target='_blank'>
              Epithelial competition determines gene therapy potential to suppress Fanconi Anemia oral cancer risk
              </a>
            </td>
          <td>
            Hunter L. Colegrove, Raymond J Monnat, Alison F Feder
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Tumor hypoxia reshapes the microenvironment, driving progression, invasion, metastasis, and therapy resistance. Patient-derived tumor organoids, formed under three-dimensional conditions, preserve cellular heterogeneity and nutrient gradients, making them ideal for studying hypoxia-induced tumor responses. This study examines hypoxia-induced changes in lung tumor organoids from two patients, focusing on tumor-associated markers, stem cell markers, and migration capabilities.


RESULTS
Our findings demonstrate that hypoxia distinctively modulates the expression of lung cancer markers thyroid transcription factor-1, cytokeratin 7, and ΔNP63 variant. Hypoxia also induces the upregulation of stem cell-associated markers, resulting in an increased proportion of cancer stem cells. Furthermore, hypoxic lung tumor organoids exhibit unique migratory behavior upon reoxygenation, driven by epithelial-mesenchymal transition and the elevated expression of matrix metalloproteinases 7 and matrix metalloproteinases 9, indicating their enhanced invasive potential.


CONCLUSIONS
These findings highlight the value of lung tumor organoids as models for studying hypoxia's complex role in lung cancer. Hypoxia significantly modulates lung tumor organoids growth, stemness, and migratory behavior, providing critical insights into tumor progression and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c10484a1b1c15548c942fa338e1afd0e25fd59" target='_blank'>
              Lung tumor organoids migrate as cell clusters containing cancer stem cells under hypoxic condition.
              </a>
            </td>
          <td>
            Yanjiao Li, Jiarong Zou, Yanhua Fang, Jianing Zuo, Ruoyu Wang, Shanshan Liang
          </td>
          <td>2025-02-01</td>
          <td>Biology of the cell</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Bona fide Kupffer cells (KCs) are prenatally seeded and show unique functional and immunophenotypic features among tissue macrophages. They are considered as terminally differentiated, and adaptability in disease is attributed to recruited, monocyte-derived KCs. Here, we investigated the extent of KC plasticity and the impact of origin in mycobacterial infections that target macrophages and can persist for months. Fate-mapping combined with high-resolution imaging revealed the emergence of a unique, infection specific KC subset which downregulated the signature markers CLEC4F and VSIG4 (“KClow”). KClow were derived from bona fide KCs and located exclusively to granuloma cores. In contrast, monocyte-derived macrophages were contained at the granuloma borders and contributed to this tissue reaction. ATAC and single-cell RNA sequencing identified a specific signature of KClow with high antimycobacterial activity and specialization to a hypoxic microenvironment. Despite their fundamental deviation from the classical KC phenotype, KClow showed remarkable adaptability, and were capable to return to a homeostatic-like KC state. Accordingly, mycobacterial infections unmask KCs as highly plastic cells, capable of responding to extreme environmental changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00567b4fcca1e2f8ed25ab8bfa5832264cff467d" target='_blank'>
              Mycobacterial infection uncovers plasticity of Kupffer cells
              </a>
            </td>
          <td>
            Jana Neuber, Florens Lohrmann, Samuel Wald, Merve Göçer, A. Lösslein, D. Obwegs, Vitka Gres, Torsten Goldmann, M. Rogg, Christoph Schell, Sebastian Preissl, Sagar, Philipp Henneke
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Effective medical therapies for metastatic paraganglioma (mPPGL) are currently lacking, leading to dismal prognosis. Building on our knowledge of molecular mechanisms driving PPGL progression, we assessed the therapeutic potential of targeting two critical processes: PI3K signaling and cell cycle regulation. The efficacy of buparlisib (PI3Ki) and ribociclib (CDK4/6i), individually and combined, was assessed in vitro using PPGL cell lines, rat– and patient-derived primary cells, and in vivo using PPGL cells-derived mouse xenografts. The combination therapy demonstrated superior antitumor activity compared to single agents, particularly in vivo. Mechanistically, the efficacy of the combination therapy was associated to the downregulation of FOXM1-controlled genes implicated in mitotic spindle assembly and chromosomal segregation, leading to mitotic catastrophe. Data mining and qRT-PCR showed this genetic signature to be upregulated in human mPPGLs. This suggests that aggressive PPGLs exhibit heightened vulnerability to dual PI3K and CDK4/6 inhibition, offering a promising therapeutic avenue for these challenging cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/312876965c04c1de5ff4c1cc2f5814c15a87434b" target='_blank'>
              A genetic signature predicts aggressive paraganglioma sensitivity to dual PI3K-CDK4/6 inhibition therapy
              </a>
            </td>
          <td>
            Bhargavi Karna, Federica Gentile, Sebastian Gulde, Daniela De Martino, Julian Maurer, Katyayni Ganesan, Katharina Wang, Nicole Bechmann, E. Rapizzi, Mercedes Robledo, Ester Arroba, Stephan Herzig, S. Nölting, Hermine Mohr, Natalia S Pellegata
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, G. Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics (ST) offers enormous potential to decipher the biological and pathological heterogeneity in precious archival cancer tissues. Traditionally, these tissues have rarely been used and only examined at a low throughput, most commonly by histopathological staining. ST adds thousands of times as many molecular features to histopathological images, but critical technical issues and limitations require more assessment of how ST performs on fixed archival tissues. In this work, we addressed this in a cancer‐heterogeneity pipeline, starting with an exploration of the whole transcriptome by two sequencing‐based ST protocols capable of measuring coding and non‐coding RNAs. We optimised the two protocols to work with challenging formalin‐fixed paraffin‐embedded (FFPE) tissues, derived from skin. We then assessed alternative imaging methods, including multiplex RNAScope single‐molecule imaging and multiplex protein imaging (CODEX). We evaluated the methods’ performance for tissues stored from 4 to 14 years ago, covering a range of RNA qualities, allowing us to assess variation. In addition to technical performance metrics, we determined the ability of these methods to quantify tumour heterogeneity. We integrated gene expression profiles with pathological information, charting a new molecular landscape on the pathologically defined tissue regions. Together, this work provides important and comprehensive experimental technical perspectives to consider the applications of ST in deciphering the cancer heterogeneity in archived tissues. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ac2d361f4c48551940640dc92bcb661d02f42d" target='_blank'>
              Assessing spatial sequencing and imaging approaches to capture the molecular and pathological heterogeneity of archived cancer tissues
              </a>
            </td>
          <td>
            Tuan Vo, P. Prakrithi, Kahli Jones, Sohye Yoon, P. Y. Lam, Y. Kao, Ning Ma, Samuel X. Tan, Xinnan Jin, Chenhao Zhou, Joanna Crawford, S. Walters, Ishaan Gupta, Peter H Soyer, K. Khosrotehrani, M. Stark, Quan H Nguyen
          </td>
          <td>2025-01-23</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Medulloblastoma is the most common pediatric brain cancer and is broadly categorized into four molecular subgroups. Understanding the cell origins of medulloblastoma is crucial for preventing tumor formation and relapse. Recent single-cell transcriptomics studies have identified the potential cell lineage vulnerabilities and mechanisms underpinning malignant transformation in medulloblastoma. Emerging evidence suggests that genetic-epigenetic alterations specific to each subgroup lead to a lineage-specific stall in the neural developmental program and subsequent tumorigenesis. We discuss the putative cells of origin, plasticity, and heterogeneity within medulloblastoma subgroups and delve into the genetic and epigenetic changes that predispose cells to transformation. Additionally, we review the current insights into how cerebellar stem/progenitor cells and lineage plasticity impact medulloblastoma pathogenesis and highlight recent therapeutic advances targeting specific oncogenic vulnerabilities in this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8677e695bbefb8bee72d8f6205f8c94375119315" target='_blank'>
              Developmental Origins and Oncogenesis in Medulloblastoma.
              </a>
            </td>
          <td>
            K. Kiang, Yogesh K H Wong, Soma Sengupta, Martine F. Roussel, Q. R. Lu
          </td>
          <td>2025-02-14</td>
          <td>Annual review of neuroscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 PURPOSE: With expanding immunotherapy options available in many cancers, particularly diffuse large B-cell lymphoma (DLBCL), methods for identifying individualized and subtype-specific immunotherapy sensitivity are increasingly important. Evaluation of immunotherapy within ex vivo settings is challenging due to the complexity in conserving tumor heterogeneity and its surrounding tumor microenvironment (TME). We have developed a combinatorial functional precision medicine platform, Optim.AI, a hybrid experimental-analytics platform that predictively ranks all actionable treatments within a drug set of interest. In prior clinical studies, Optim.AI has been demonstrated to guide cytotoxic and targeted treatments in lymphomas, including DLBCL. However, these clinical studies did not interrogate immunotherapy responses. In this study, we explored the integration of high-content screening workflows with Optim.AI to evaluate immunotherapy-focused drug sets in physiologically relevant multicellular models of DLBCL. METHODS: PBMCs and DLBCL cells were fluorescently labelled to facilitate multicellular tracking for high-content analysis. Optim.AI combinatorial drug sensitivity testing plates were then used to evaluate DLBCL cells, co-cultured with PBMCS, with up to 12 FDA-approved chemotherapies, immunomodulatory agent, monoclonal antibodies, antibody-drug conjugates, and bispecific antibodies. Utilizing real-time fluorescent apoptosis signal as the phenotypic readout, high-content analysis of tumor-specific cell death and apoptosis was evaluated at specific timepoints and used as the dataset of Optim.AI analytics and predictive ranking of combinatorial immunotherapy sensitivities in DLBCL cells. RESULTS: Optim.AI could confirm rituximab (CD20 mAb) sensitivity and optimal antibody-dependent cellular cytotoxicity (ADCC) lysis on CD20+ DLBCL cells at an Effector to Target ratio of 5:1 with a quantifiable phenotypic output at 48h post-treatment. Cytotoxicity studies performed across PBMCs from four different donors also showed negligible potential toxicity from drug candidates on immune cells, indicating that the platform can effectively and stably quantify ADCC. Furthermore, low ranking of tafasitamab (CD19 mAb) based combinations in a CD20+ cell line with low CD19 expression showed low target engagement with tafasitamab, confirming the platform’s ability to detect appropriate immunotherapy specificity. These findings collectively demonstrate the feasibility of pairing Optim.AI with an integrated co-culture model and image analysis pipeline to evaluate and rank immunotherapy-based combinations. CONCLUSIONS: We have demonstrated the integration of a high-content screening cancer-immune cell co-culture system with Optim.AI to evaluate immune-based combination therapies. Application of this approach towards ex vivo patient samples as well as appropriate cancer models will allow for guided immunotherapy-based treatment as well as identification of more effective cancer-specific and subtype-specific immunotherapy combinations.
 Citation Format: Sharon Pei Yi Chan, Jhin Jieh Lim, Masturah Rashid, Edward Kai-Hua Chow. Application of a combinatorial functional precision medicine platform to predict immunotherapy response in DLBCL [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A049.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6fe7c1118b5b40de590086593aa209906bee3a7" target='_blank'>
              Abstract A049: Application of a combinatorial functional precision medicine platform to predict immunotherapy response in DLBCL
              </a>
            </td>
          <td>
            Sharon Pei Yi Chan, J. Lim, M. Rashid, Edward Kai-Hua Chow
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8ecf8ff6118aceaa2b844571144d55332e29e96" target='_blank'>
              Tumor microenvironment and drug resistance in lung adenocarcinoma: molecular mechanisms, prognostic implications, and therapeutic strategies
              </a>
            </td>
          <td>
            Qian Cao, Chenxuan Li, Ying Li, Xiangjing Kong, Shoushi Wang, Jun Ma
          </td>
          <td>2025-02-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The development of treatment resistance remains universal for patients with metastatic prostate cancer, driven by AR alterations and lineage state transitions. Identifying the evolution of lineage transitions in treatment resistance has been limited by the challenges of collecting serial tissue biopsies on treatment, which can be overcome using blood-based liquid biopsies. Utilizing a novel circulating tumor cell (CTC) isolation approach, we collected 273 CTC samples from 117 patients with metastatic prostate cancer for RNA sequencing. 146 samples from 70 patients had tumor purity comparable to tissue biopsies. We identified four CTC transcriptional phenotypes, mirroring lineage states identified in tissue. Patients with a luminal-B-like CTC phenotype defined by persistent AR signaling and high proliferation, as well as those with a neuroendocrine CTC phenotype, had significantly shorter survival than patients with luminal-A-like and low proliferation phenotypes. In a prospective substudy, pre-treatment CTC luminal-B-like phenotype was associated with early progression on 177Lu-PSMA-617.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166ffe8ca69e378ce5cd3ea06cb57373f1b4e28e" target='_blank'>
              High-purity CTC RNA sequencing identifies prostate cancer lineage phenotypes prognostic for clinical outcomes.
              </a>
            </td>
          <td>
            Marina N. Sharifi, J. Sperger, Amy K Taylor, Katharine E Tippins, Shannon R Reese, Viridiana Carreno, K. Kaufmann, Alex H Chang, Luke A Nunamaker, Charlotte Linebarger, Leilani Mora-Rodriguez, J. Schehr, H. Krause, K. Helzer, M. Bootsma, G. Blitzer, John M Floberg, C. Kyriakopoulos, H. Emamekhoo, Elizabeth I. Heath, Meghan Wells, S. Tagawa, M. Sjöström, A. Choudhury, Menggang Yu, Andrew J. Armstrong, D. Rathkopf, H. Beltran, Peter S Nelson, Felix Y. Feng, S. Dehm, D. Kosoff, Xiao X Wei, Rana R Mckay, Shuang G. Zhao, Joshua M Lang
          </td>
          <td>2025-02-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Changes in cell type composition play an important role in human health and disease. Recent advances in single-cell technology have enabled the measurement of cell type composition at increasing cell lineage resolution across large cohorts of individuals. Yet this raises new challenges for statistical analysis of these compositional data to identify changes in cell type frequency. We introduce crumblr (DiseaseNeurogenomics.github.io/crumblr), a scalable statistical method for analyzing count ratio data using precision-weighted linear mixed models incorporating random effects for complex study designs. Uniquely, crumblr performs statistical testing at multiple levels of the cell lineage hierarchy using a multivariate approach to increase power over tests of one cell type. In simulations, crumblr increases power compared to existing methods while controlling the false positive rate. We demonstrate the application of crumblr to published single-cell RNA-seq datasets for aging, tuberculosis infection in T cells, bone metastases from prostate cancer, and SARS-CoV-2 infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4cb0d543e456b8e0b6125dd4a4f449c19fe2812" target='_blank'>
              Fast, flexible analysis of differences in cellular composition with crumblr
              </a>
            </td>
          <td>
            Gabriel E. Hoffman, P. Roussos
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc6fce345e0628d21c6c9477e460943646693b" target='_blank'>
              Spatial deconvolution from bulk DNA methylation profiles determines intratumoral epigenetic heterogeneity
              </a>
            </td>
          <td>
            Binbin Liu, Yumo Xie, Yu Zhang, Guannan Tang, Jinxin Lin, Ze Yuan, Xiaoxia Liu, Xiaolin Wang, Meijin Huang, Yanxin Luo, Huichuan Yu
          </td>
          <td>2025-01-23</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd410f5877168898f20358879e98510563700b47" target='_blank'>
              Unraveling complexity and leveraging opportunities in uncommon breast cancer subtypes
              </a>
            </td>
          <td>
            F. Pareja, Rohit Bhargava, Virginia F Borges, E. Brogi, Rita Canas Marques, F. Cardoso, C. Desmedt, M. Harigopal, Sunil R. Lakhani, Adrian V. Lee, J.P. Leone, Hannah Linden, C. J. Lord, C. Marchiò, S. Merajver, E. Rakha, Jorge S Reis-Filho, Andrea L. Richardson, Elinor J. Sawyer, P. Schedin, C. Schwartz, Andrew N J Tutt, N. T. Ueno, A. Vincent-Salomon, B. Weigelt, Y. H. Wen, Stuart J Schnitt, S. Oesterreich
          </td>
          <td>2025-01-24</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0871f3b2bbb54a6ef4ad184b9e9c10fad8f88b51" target='_blank'>
              Spatial single-cell maps reveal ST6GAL1 promoting ovarian cancer metastasis.
              </a>
            </td>
          <td>
            Lan-hui Qin, Zijian Jiang, Chongze Yang, Rui Song, Pei-Yin Chen, Weihui Xu, Guanzhen Zeng, Jin-yuan Liao, Liling Long
          </td>
          <td>2025-01-30</td>
          <td>Glycoconjugate journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The sentinel lymph node (SLN) is the first lymph node encountered by a metastatic cancer cell and serves as a predictor of poor prognosis, as patients with clinically occult SLN metastases are classified as stage III with elevated rates of recurrence and diminished overall survival. However, the dynamics of immune infiltrates in SLNs remain poorly characterized. Here, using an unbiased Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) technique, we profiled 97,777 cells from SLN tissues obtained from patients with stages I/II and III cutaneous melanoma. We describe the transcriptional programs of a multitude of T, B, and myeloid cell subtypes in SLNs. Based on the proportions of cell types, we determined that SLN subtypes are stratified along a naive → activated axis; patients with a 'high activated' signature score appear to be undergoing a robust melanoma antigen-driven adaptive immune response and thus, could be responsive to immunotherapy. Additionally, we identified transcriptomic signatures of SLN-infiltrating dendritic cell subsets that compromise anti-tumor immune responses. Our analyses provide valuable insights into tumor-driven immune changes in the SLN tissue, offering a powerful tool for the informed design of immune therapies for high-risk melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff6ed2d70346b46dbac007982b480af5731bdaf" target='_blank'>
              Single-cell transcriptomics of melanoma sentinel lymph nodes identifies unique immune cell signatures associated with metastasis.
              </a>
            </td>
          <td>
            Eric Engelbrecht, Bryce F Stamp, Lewis C. Chew, Omar S Sarkar, Phillip Harter, Sabine J Waigel, Eric C Rouchka, J. Chariker, Andrei Smolenkov, Jason A Chesney, K. McMasters, Corey T Watson, K. Yaddanapudi
          </td>
          <td>2025-03-06</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7635dc0474dacb31cc3822f8f3055dfb7b7481" target='_blank'>
              Integrating transcriptomics and scPagwas analysis predicts naïve CD4 T cell-related gene DRAM2 as a potential biomarker and therapeutic target for colorectal cancer
              </a>
            </td>
          <td>
            Rui Feng, Xiaofang Li, Benhua Li, Tiankuo Luan, Jiaming He, Guojing Liu, Jian Yue
          </td>
          <td>2025-02-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8124e4116b706bf406200aaca77b811cd619c61b" target='_blank'>
              Metabolically activated and highly polyfunctional intratumoral VISTA+ regulatory B cells are associated with tumor recurrence in early-stage NSCLC
              </a>
            </td>
          <td>
            D. Lo Tartaro, Beatrice Aramini, V. Masciale, Nikolaos Paschalidis, F. Lofaro, Anita Neroni, R. Borella, Elena Santacroce, Alin Liviu Ciobanu, A. V. Samarelli, Federica Boraldi, Daniela Quaglino, Alessandra Dubini, Michele Gaudio, G. Manzotti, F. Reggiani, Federica Torricelli, A. Ciarrocchi, A. Neri, Federica Bertolini, Massimo Dominici, Pier Luigi Filosso, F. Stella, L. Gibellini, S. De Biasi, Andrea Cossarizza
          </td>
          <td>2025-01-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Newly diagnosed prostate cancers differ dramatically in mutational composition and lethality. The most accurate clinical predictor of lethality is tumor tissue architecture, quantified as tumor grade. To interrogate the evolutionary origins of prostate cancer heterogeneity, we analyzed 666 prostate tumor whole genomes. We identified a compendium of 223 recurrently mutated driver regions, most influencing downstream mutational processes and gene expression. We identified and validated individual germline variants that predispose tumors to acquire specific somatic driver mutations: these explain heterogeneity in disease presentation and ancestry differences. High-grade tumors have a superset of the drivers in lower-grade tumors, including increased frequency of BRCA2 and MYC mutations. Grade-associated driver mutations occur early in tumor evolution, and their earlier occurrence strongly predicts cancer relapse and metastasis. Our data suggest high- and low-grade prostate tumors both emerge from a common pre-malignant field, influenced by germline genomic context and stochastic mutation-timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9c23cc7532d469971ff1994cdcbf780a4650cf6" target='_blank'>
              The Germline and Somatic Origins of Prostate Cancer Heterogeneity.
              </a>
            </td>
          <td>
            Takafumi N. Yamaguchi, Kathleen E. Houlahan, Helen Zhu, Natalie J Kurganovs, J. Livingstone, Natalie S. Fox, Jiapei Yuan, Jocelyn Sietsma Penington, Chol-Hee Jung, T. Schwarz, Weerachai Jaratlerdsiri, J. van Riet, P. Georgeson, S. Mangiola, Kodi Taraszka, R. Lesurf, Jue Jiang, K. Chow, Lawrence E Heisler, Yu-Jia Shiah, Susmita G Ramanand, Michael J Clarkson, Anne Nguyen, S. M. G. Espiritu, R. Stuchbery, Richard Jovelin, Vincent Huang, Connor Bell, Edward O'Connor, Patrick J. McCoy, Christopher M. Lalansingh, Marek Cmero, A. Salcedo, Eva K F Chan, Lydia Y. Liu, Phillip D Stricker, Vinayak Bhandari, R. Bornman, Dorota H. Sendorek, A. Lonie, S. Prokopec, M. Fraser, J. Peters, Adrien Foucal, S. Mutambirwa, Lachlan Mcintosh, M. Orain, Matthew J. Wakefield, V. Picard, Daniel J Park, H. Hovington, Michael Kerger, A. Bergeron, Veronica Y. Sabelnykova, J-H. Seo, M. Pomerantz, Noah Zaitlen, Sebastian M. Waszak, Alexander Gusev, L. Lacombe, Yves Fradet, Andrew Ryan, A. Kishan, M. Lolkema, Joachim Weischenfeldt, Bernard Têtu, Anthony J Costello, Vanessa Hayes, R. J. Hung, H. He, John D McPherson, Bogdan Pasaniuc, T. H. van der Kwast, AT Papenfuss, M. Freedman, B. Pope, Robert G Bristow, Ram-Shankar Mani, N. Corcoran, J. Reimand, Christopher M Hovens, P. Boutros
          </td>
          <td>2025-02-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Skin cancer is the most common malignancy worldwide, comprising three major types: melanoma, basal cell carcinoma (BCC), and squamous cell carcinoma (SCC). A critical goal in advancing our understanding of skin cancer is to identify the shared and distinct biological mechanisms that drive the risk profiles for each skin cancer type. In this study, we integrated tissue-specific chromatin conformation and expression quantitative trait loci data (spatial eQTL) to construct gene regulatory networks (GRNs) for melanocytes, sun-exposed skin, not sun-exposed skin, and blood. These GRNs, which capture spatially regulated gene expressions, were used as instrumental variables along with GWAS summary statistics for melanoma, BCC and SCC, to infer causal relationships between specific gene expression changes and each type of skin cancer. These Mendelian randomization analyses identified 82, 62, and 125 causal genes for melanoma, BCC, and SCC, respectively, with many of these genes not evident from the GWAS data alone. Our analyses revealed distinct mechanisms for each skin cancer type: telomere maintenance and nevus pathways in melanoma, inherited immune traits in BCC, and p53 dysfunction in SCC. Notably, apoptosis and pigmentation emerged as shared biological processes across skin cancers. These findings provide new insights into the genetic drivers of three distinct skin cancers, highlighting new gene targets which can be used to increase diagnostic precision, as well as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d1a0a87d4f015e13185fa2728ac9fd439bf3488" target='_blank'>
              Mendelian randomizations with spatial gene networks reveal shared and distinct drivers of risk in major skin cancer types
              </a>
            </td>
          <td>
            Michael Pudjihartono, J. O’Sullivan, W. Schierding
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The tumor microenvironment (TME) consists of a complex mix of cellular and non-cellular components, including immune cells, stromal cells, extracellular matrix, cytokines, and growth factors. These elements interact with tumor cells to influence tumorigenesis, growth, invasion, and metastasis. Long noncoding RNAs (lncRNAs)—a class of non-coding RNAs longer than 200 nucleotides—have attracted considerable attention for their roles in regulating gene expression at the epigenetic, transcriptional, and post-transcriptional levels. Emerging evidence suggests that lncRNAs are crucial in shaping the TME by modulating processes such as immune evasion, angiogenesis, metabolic reprogramming, and the maintenance of cancer stem cells. This review provides an overview of the current understanding of lncRNAs in the TME, focusing on their involvement in key signaling pathways and cellular interactions that drive tumor progression. We discussed how lncRNAs contribute to extra-cellular matrix remodeling, facilitate communication between tumor and stromal cells, and regulate immune cell infiltration and function within the TME. Additionally, we explore the potential of lncRNAs as biomarkers for early cancer detection and prognosis, as well as their promise as therapeutic targets to disrupt tumor-microenvironment crosstalk. The review also addresses challenges in targeting lncRNAs therapeutically, such as ensuring specificity, minimizing off-target effects, and achieving effective in vivo delivery of lncRNA-targeted therapies. Strategies to overcome these challenges include the development of highly specific lncRNA knockout technologies and the use of advanced delivery systems, such as nanoparticles and viral vectors, to precisely target tumor-associated cells. Overall, this review underscores the significant role of lncRNAs in the TME and their potential as novel tools for enhancing cancer diagnosis and treatment. By elucidating the multifaceted roles of lncRNAs in the TME, we aimed to provide insights that could lead to more effective, targeted therapeutic strategies, ultimately advancing cancer research and improving patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baf1f9b2f1f2c54afc524f991ea0518f53bf5a2b" target='_blank'>
              Tumor Microenvironment Dynamics: The Regulatory Influence of Long Non-coding RNAs
              </a>
            </td>
          <td>
            O. Beylerli, Elmar Musaev, Chunlei Wang, Irina Popova, I. Gareev
          </td>
          <td>2025-02-22</td>
          <td>Gene Expression</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Rationale: Glioblastoma (GBM) is the most fatal form of brain cancer and its treatment represents a persistent challenge. Mesenchymal stromal cells (MSCs) have been explored as therapeutic tools in cancer management owing to their tumor-homing abilities. However, their clinical application is limited due to the controversial role of MSCs in carcinogenesis. This study investigates how MSCs influence tumor behavior and explores the synergistic anticancer effects in combination with melatonin (Mel). Methods: Orthotopic and subcutaneous GBM xenograft mouse models were used to assess the antitumor effect of Mel pre-treated MSCs (MSC Mel ). Histological, immunohistochemical, and ultrastructural analyses were conducted to identify phenotypic changes in tumors. Through a set of in vitro assays, including direct and indirect co-cultures, dynamic single-cell tracking and tumorsphere assay, we explored the impact of MSC Mel on primary and non-primary GBM cells. Transcriptomic profiling was used to identify genes and pathways modulated by this synergistic therapy. Results: MSC Mel delayed tumor growth in mice and increased collagen deposition. Additionally, MSC Mel showed enhanced capacity to prevent GBM cell migration compared to untreated MSCs. Molecular analysis identified genes and proteins related to cell migration, cytoskeletal dynamics and extracellular matrix remodeling in GBM cells exposed to MSC Mel , including reduced vimentin expression. Finally, a genetic signature associated with the clinical outcomes of GBM patients was identified. Conclusions: Our study demonstrates that melatonin enhances the anticancer properties of MSCs, providing new insights into their interaction with GBM cells and tumor environment. These findings offer valuable guidance for advancing MSC-based therapies in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f57dedd75349ecd50ac992a7b9ebfe735e08696" target='_blank'>
              Glioblastoma progression is hindered by melatonin-primed mesenchymal stromal cells through dynamic intracellular and extracellular reorganizations
              </a>
            </td>
          <td>
            Laura Olmedo-Moreno, Concepción Panadero-Morón, J. M. Sierra-Párraga, Rubén Bueno-Fernández, E. Norton, Y. Aguilera, N. Mellado-Damas, Patricia García-Tárraga, Raquel Morales-Gallel, María Antequera-Martínez, Raúl V. Durán, Jaime Ferrer-Lozano, Germaine Escames, J. García-Verdugo, Alejandro Martín-Montalvo, Hugo Guerrero-Cázares, V. Capilla-González
          </td>
          <td>2025-02-10</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31fd762af41845a9a3aad72616958b24bc4dd8f" target='_blank'>
              Chaperone-mediated autophagy regulates the metastatic state of mesenchymal tumors
              </a>
            </td>
          <td>
            Xun Zhou, Eva Berenger, Yong Shi, Vera Shirokova, E. Kochetkova, Tina Becirovic, Boxi Zhang, Vitaliy O Kaminskyy, Yashar Esmaeilian, K. Hosaka, Cecilia Lindskog, P. Hydbring, Simon Ekman, Yihai Cao, Maria Genander, Marcin Iwanicki, E. Norberg, H. Vakifahmetoglu-Norberg
          </td>
          <td>2025-03-07</td>
          <td>EMBO Molecular Medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Understanding tumor heterogeneity is a major challenge that was recognized as one of the first Cancer Grand Challenges, with a call to provide solutions to visualize tumor heterogeneity. The Rosetta team took on this challenge, exploiting advances in spatial-omics approaches centered around mass spectrometry imaging to map tumor heterogeneity at the cellular and molecular scales with different levels of resolution. See related article by Bressan et al., p. 16 See related article by Stratton et al., p. 22 See related article by Bhattacharjee et al., p. 28.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d308601bb28236e43c3bc271c19d29c8dd8720ed" target='_blank'>
              Visualizing Cancer Heterogeneity at the Molecular and Cellular Levels: Lessons from Rosetta.
              </a>
            </td>
          <td>
            R. Goodwin, John F Marshall, G. Poulogiannis, Mariia Yuneva, K. Brindle, Z. Takáts, Owen J. Sansom, Josephine Bunch, Simon T Barry
          </td>
          <td>2025-01-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba71338c6c092a7d53ee875fe94f60877fe823a" target='_blank'>
              Single-cell transcriptomics identify a novel macrophage population associated with bone invasion in pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Xinzhi Wu, Xueshuai Han, Haibo Zhu, Mingxuan Li, L. Gong, Sicheng Jing, Weiyan Xie, Zhaoqi Liu, Chuzhong Li, Yazhuo Zhang
          </td>
          <td>2025-01-27</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Changes in cell type composition play an important role in human health and disease. Recent advances in single-cell technology have enabled the measurement of cell type composition at increasing cell lineage resolution across large cohorts of individuals. Yet this raises new challenges for statistical analysis of these compositional data to identify changes in cell type frequency. We introduce crumblr (DiseaseNeurogenomics.github.io/crumblr), a scalable statistical method for analyzing count ratio data using precision-weighted linear mixed models incorporating random effects for complex study designs. Uniquely, crumblr performs statistical testing at multiple levels of the cell lineage hierarchy using a multivariate approach to increase power over tests of one cell type. In simulations, crumblr increases power compared to existing methods while controlling the false positive rate. We demonstrate the application of crumblr to published single-cell RNA-seq datasets for aging, tuberculosis infection in T cells, bone metastases from prostate cancer, and SARS-CoV-2 infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7575b8a3544a3fbacb1b3901ce3953cf4e240181" target='_blank'>
              Fast, flexible analysis of differences in cellular composition with crumblr
              </a>
            </td>
          <td>
            Gabriel E. Hoffman, P. Roussos
          </td>
          <td>2025-02-27</td>
          <td>Research Square</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33b20af0b1c35c714611ceec53e3a323336edcf" target='_blank'>
              Myeloid cells: key players in tumor microenvironments.
              </a>
            </td>
          <td>
            Qiaomin Hua, Zhixiong Li, Yulan Weng, Yan Wu, Limin Zheng
          </td>
          <td>2025-03-06</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastasis remains a leading cause of cancer‐related deaths, defined by a complex, multi‐step process in which tumor cells spread and form secondary growths in distant tissues. Despite substantial progress in understanding metastasis, the molecular mechanisms driving this process and the development of effective therapies remain incompletely understood. Elucidating the molecular pathways governing metastasis is essential for the discovery of innovative therapeutic targets. The rapid advancements in sequencing technologies and the expansion of biological databases have significantly deepened our understanding of the molecular drivers of metastasis and associated drug resistance. This review focuses on the molecular drivers of metastasis, particularly the roles of genetic mutations, epigenetic changes, and post‐translational modifications in metastasis progression. We also examine how the tumor microenvironment influences metastatic behavior and explore emerging therapeutic strategies, including targeted therapies and immunotherapies. Finally, we discuss future research directions, stressing the importance of novel treatment approaches and personalized strategies to overcome metastasis and improve patient outcomes. By integrating contemporary insights into the molecular basis of metastasis and therapeutic innovation, this review provides a comprehensive framework to guide future research and clinical advancements in metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/647f5025b06ef5ed04146a75e8b99dcf5629913e" target='_blank'>
              Tumor Metastasis: Mechanistic Insights and Therapeutic Intervention
              </a>
            </td>
          <td>
            Lin Tang, Shao‐Cong Peng, Xiao‐Wan Zhuang, Yan He, Yu‐Xiang Song, Hao Nie, Can-Can Zheng, Zhen‐Yu Pan, Alfred King‐Yin Lam, Ming‐Liang He, Xingyuan Shi, Bin Li, W. Xu
          </td>
          <td>2025-02-17</td>
          <td>MedComm – Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4405a7f71591c3b2e80c4392e32cd43a5138621" target='_blank'>
              3D genome landscape of primary and metastatic colorectal carcinoma reveals the regulatory mechanism of tumorigenic and metastatic gene expression
              </a>
            </td>
          <td>
            Xiang Xu, Jingbo Gan, Zhaoya Gao, Ruifeng Li, Dandan Huang, Lin Lin, Yawen Luo, Qian Yang, Jingxuan Xu, Yaru Li, Qing Fang, Ting Peng, Yaqi Wang, Zihan Xu, An Huang, Haopeng Hong, F. Lei, Wensheng Huang, Jianjun Leng, Tingting Li, Xiaochen Bo, Hebing Chen, Cheng Li, Jin Gu
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b192991a8e1c4b0fc2641c382c15f44b9a3fa553" target='_blank'>
              Cancer-intrinsic Cxcl5 orchestrates a global metabolic reprogramming for resistance to oxidative cell death in 3D.
              </a>
            </td>
          <td>
            Ramin Seo, A. C. V. de Guzman, Sunghyouk Park, Ji Youn Lee, Suk-Jo Kang
          </td>
          <td>2025-03-07</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/982b46ad0f67ebc704b6d142139e5b777f0090b1" target='_blank'>
              Exosomal lncRNA Mir100hg from lung cancer stem cells activates H3K14 lactylation to enhance metastatic activity in non-stem lung cancer cells
              </a>
            </td>
          <td>
            Lei Shi, Bowen Li, Jiyu Tan, Ling Zhu, Sicheng Zhang, Yuhan Zhang, Meng Xiang, Jie Li, Yan Chen, Xue Han, Jia-Cheng Xie, Yao Tang, H. Xing, Jingyu Li, Jianyu Wang, cnJingyu Li
          </td>
          <td>2025-02-28</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eee1a8b871b1885a1b34200cb093a48900beddf" target='_blank'>
              The phenogenomic landscapes of pleural mesothelioma tumor microenvironment predict clinical outcomes
              </a>
            </td>
          <td>
            Federica Torricelli, Filomena Spada, Cynthia Bishop, K. Todd, Daisuke Nonaka, N. Petrov, Manuela Terranova Barberio, Alan G Ramsay, Richard Ellis, A. Ciarrocchi, B. Apollonio, Andrea Billè
          </td>
          <td>2025-02-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Idiopathic pulmonary fibrosis (IPF) is a progressive lung disease characterized by repetitive alveolar injuries with excessive deposition of extracellular matrix (ECM) proteins. A crucial need in understanding IPF pathogenesis is identifying cell types associated with histopathological regions, particularly local fibrosis centers known as fibroblast foci. To address this, we integrated published spatial transcriptomics and single-cell RNA sequencing (scRNA-seq) transcriptomics and adopted the Query method and the Overlap method to determine cell type enrichments in histopathological regions. Distinct fibroblast cell types are highly associated with fibroblast foci, and transitional alveolar type 2 and aberrant KRT5-/KRT17+ (KRT: keratin) epithelial cells are associated with morphologically normal alveoli in human IPF lungs. Furthermore, we employed laser capture microdissection-directed mass spectrometry to profile proteins. By comparing with another published similar dataset, common differentially expressed proteins and enriched pathways related to ECM structure organization and collagen processing were identified in fibroblast foci. Importantly, cell type enrichment results from innovative spatial proteomics and scRNA-seq data integration accord with those from spatial transcriptomics and scRNA-seq data integration, supporting the capability and versatility of the entire approach. In summary, we integrated spatial multi-omics with scRNA-seq data to identify disease-associated cell types and potential targets for novel therapies in IPF intervention. The approach can be further applied to other disease areas characterized by spatial heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/947f84484c40c13b0563bade74b2ae0f8c1ec3ac" target='_blank'>
              Novel Integration of Spatial and Single-Cell Omics Data Sets Enables Deeper Insights into IPF Pathogenesis
              </a>
            </td>
          <td>
            Fei Wang, Liang Jin, Xue Wang, Baoliang Cui, Yingli Yang, Lori Duggan, Annette Schwartz Sterman, Sarah M. Lloyd, L. Hazelwood, Neha Chaudhary, Bhupinder Bawa, Lucy A. Phillips, Yupeng He, Yu Tian
          </td>
          <td>2025-01-13</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75bdacf15f78eabb80ad03155a6f7ee66350fd9" target='_blank'>
              Genomic alterations and transcriptional phenotypes in circulating free DNA and matched metastatic tumor
              </a>
            </td>
          <td>
            N. Takahashi, L. Pongor, Shivam P. Agrawal, Mariya Shtumpf, Ankita Gurjar, V. Rajapakse, Ahmad Shafiei, Christopher W. Schultz, Sehyun Kim, Diana Roame, Paula Carter, R. Vilimas, Samantha Nichols, Parth Desai, W. Figg, Mohammad Bagheri, V. B. Teif, Anish Thomas
          </td>
          <td>2025-02-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The ESCRT (endosomal sorting complex required for transport) machinery is essential for various cellular processes, yet its role in head and neck squamous cell carcinoma (HNSCC) is poorly understood. We utilized The Cancer Genome Atlas (TCGA) datasets to analyze the expression of ESCRT genes. Bulk RNA-sequencing data and HNSCC tissue microarrays (TMAs) were used to evaluate VPS25 expression and its clinical significance. Single-cell RNA sequencing of tumor tissues and VPS25 knockdown experiments in CAL27 cells were used to investigate its biological functions. Immunohistochemistry, spatial transcriptomics, and immunotherapy datasets highlighted the involvement of VPS25 in immune suppression and its potential as a predictive biomarker. The results demonstrated significant VPS25 overexpression in HNSCC tissues, which correlated with poor clinical outcomes. It promoted tumor cell proliferation and migration while reducing immune cell infiltration in the tumor microenvironment (TME). Additionally, by upregulating PVR expression in tumor cells, VPS25 activated the immunosuppressive PVR-TIGIT signaling axis, thereby facilitating immune evasion. Furthermore, VPS25 emerged as a potential biomarker for predicting immunotherapy response. These findings highlight VPS25 as a pivotal regulator of tumor progression and immune evasion in HNSCC and a promising target for therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/902195b920715f84e3501a67211a1883f6d279dd" target='_blank'>
              VPS25 Promotes an Immunosuppressive Microenvironment in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Liguo Chen, Yu-Han Fang, Kui-Ming Wang, Wei Zhang, Gang Chen
          </td>
          <td>2025-02-22</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Tumor-infiltrating lymphocytes (TILs) are associated with decreased risk of recurrence in human papillomavirus-associated oropharyngeal squamous cell carcinoma (HPV(+)OPSCC). The composition and spatial distribution of TILs and tumor-infiltrating immune cells is not well characterized.


METHODS
Formalin-fixed paraffin-embedded primary and lymph node (LN) tumor tissues from ten progressors (cases) and ten matched non-progressors (controls) were interrogated by imaging mass cytometry. Immune, stromal, and tumor cells were quantified from selected regions of interest (ROIs) using machine learning. Nearest neighbors, cell-cell interactions, and niche analyses were performed.


RESULTS
In primary ROIs, immune cell, lymphocyte, T cell, CD8+ T cell, and innate cell prevalence was significantly greater in controls. High prevalence of immune cells, lymphocytes, innate cells, and CD4+ T cells in primary tissues was significantly associated with increased time to event (TTE). Although primary and LN prevalence of T cells, CD4+ T cells, CD8+ T cells, macrophages, and tumor cells were significantly correlated, differences in LNs were neither significant nor associated with TTE. Average distances between T cells and nearest B cells, and between lymphocytes and nearest tumor cells were decreased in control primary tissues. Interactions between B cells and T cells were less organized in primary tissues from cases. A niche predominantly comprising lymphocytes was associated with longer TTE.


CONCLUSION
In HPV(+)OPSCC, immune cell subset prevalence in primary tumors differs with outcome and is associated with TTE. Interactions between B cells and both T cell subsets are associated with longer TTE, underscoring the importance of active intratumoral immune responses in outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3881754cac6083d148b1724a9d105efb77248c93" target='_blank'>
              Increased interaction between B cells and CD3+ T cells in non-progressors with human papillomavirus-associated oropharyngeal squamous cell carcinoma.
              </a>
            </td>
          <td>
            K. Bartemes, Raymond M Moore, B.C. Novotny, K. Pavelko, Will A Sherman, Michael Rivera, Joaquin J Garcia, Linda X Yin, Daniel J Ma, Eric J Moore, Kathryn M Van Abel, D. Routman
          </td>
          <td>2025-02-18</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd1722df9025221021ddfd3a6fcbf675ba9705b" target='_blank'>
              OLIG2 mediates a rare targetable stem cell fate transition in sonic hedgehog medulloblastoma
              </a>
            </td>
          <td>
            Kinjal Desai, Siyi Wanggou, Erika Luis, H. Whetstone, Chunying Yu, R. Vanner, Hayden J. Selvadurai, Lilian Lee, Jinchu Vijay, J. E. Jaramillo, Jerry J. Fan, P. Guilhamon, M. Kushida, Xuejun Li, Gregory Stein, Santosh Kesari, B. D. Simons, Xi Huang, Peter B. Dirks
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0b1fd2e12bcf45ef4e0b8a0b897b112e846447" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Takafumi N. Yamaguchi, Yu-Jia Shiah, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, Amar U. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Background/Objectives: Non-small cell lung cancer (NSCLC) is a major cause of cancer-related deaths globally. The study focuses on understanding the interplay between genetic mutations, cancer stem cells (CSCs), and the tumor microenvironment (TME) in driving NSCLC progression, resistance to therapies, and relapse. Methods: A systematic search was conducted in PubMed and Scopus databases to identify significant and valuable studies relevant to NSCLC, focusing on genetic mutations, CSCs, and the TME. Articles were selected based on their relevance, methodological severity, date of publication, and scientific soundness related to NSCLC biology and therapeutic strategies. This review synthesized findings from these sources to highlight key mechanisms and potential therapeutic interventions. Results: Mutations in critical genes in KRAS, EGFR, TP53, and other key genes interfere with stem cell regulation, promoting CSC-like behavior, resistance to therapy, and immune evasion. The tumor microenvironment (TME), including immune cells, fibroblasts, and extracellular matrix components, further supports tumor growth and reduction in treatment efficacy. Promising strategies, including CSC targeting, TME modulation, and the development of novel biomarkers, have shown potential in preclinical and clinical studies. Conclusions: The association between genetic alterations, CSCs, the TME, and other cellular pathways—including cell metabolism and immune evasion—plays a crucial role in therapy resistance, highlighting the need for comprehensive treatment strategies. The combination of genomic profiling with TME-targeting therapies could lead to personalized treatment approaches, offering hope for better clinical outcomes and reduced mortality in NSCLC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8d3b80e9aee70a721cb7f7cf165886e84b84b9b" target='_blank'>
              Oncogenic Mutations and the Tumor Microenvironment: Drivers of Non-Small Cell Lung Cancer Progression
              </a>
            </td>
          <td>
            A. Mitrakas, Christos Kakouratos, I. Lamprou, Erasmia T. Xanthopoulou, M. Koukourakis
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Activation of invasion and metastasis is a central hallmark of cancer, contributing to the primary cause of death for cancer patients. In the multistep metastatic process, cancer cells must infiltrate the circulation, survive, arrest at capillary beds, extravasate, and form metastatic clones in distant organs. However, only a small proportion of circulating tumor cells (CTCs) successfully form metastases, with transit of CTCs in the circulation being the rate-limiting step. The fate of CTCs is influenced by the circulating tumor microenvironment (cTME), which encompasses factors affecting their biological behaviors in the circulation. This liquid and flowing microenvironment differs significantly from the primary tumor microenvironment or the pre-metastatic niche. This review summarizes the latest advancements in identifying the biophysical cues, key components, and biological roles of the cTME, highlighting the network among biophysical attributes, blood cells, and non-blood factors in cancer metastasis. In addition to the potential of the cTME as a therapeutic target for inhibiting metastasis, the cTME could also represent as a biomarker for predicting patient outcomes and developing strategies for treating cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ebef94ee6fd9d5b0a0f131941d243a6a93e28d" target='_blank'>
              Circulating Tumor Microenvironment in Metastasis.
              </a>
            </td>
          <td>
            Rui Tang, Shujuan Luo, Hui Liu, Yan Sun, Manran Liu, Lu Li, Haoyu Ren, Martin K Angele, N. Börner, Keda Yu, Zufeng Guo, Guobing Yin, Haojun Luo
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e7693290b0f56e7e4e347ce73c837cd2d47cd1" target='_blank'>
              Stromal architecture and fibroblast subpopulations with opposing effects on outcomes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yifei Cheng, Xiaofang Chen, Li Feng, Zhicheng Yang, Liyun Xiao, Bin Xiang, Xiaodong Wang, Dongbing Liu, Penghui Lin, Jieyi Shi, Guohe Song, Wulei Qian, Boan Zhang, Yanan Xu, Zheng Gao, Lv Chen, Y. Wu, Jiaqiang Ma, Youpei Lin, Haichao Zhao, Lihua Peng, Xuebin Mao, Yang Liu, Hao Hou, Mingyu Yang, Yuan Ji, Xiaoying Wang, Jian Zhou, Xun Xu, Xiyang Liu, Wu Wei, Xiaoming Zhang, Qiang Gao, Hu Zhou, Yidi Sun, Kui Wu, Jia Fan
          </td>
          <td>2025-01-28</td>
          <td>Cell Discovery</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/197782f3d8b81c382c4e0d5197a6f09c7baeb2a9" target='_blank'>
              Immune microenvironment spatial landscapes of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yi Xie, Haoxin Peng, Yajie Hu, Keren Jia, Jiajia Yuan, Dan Liu, Yanyan Li, Xujiao Feng, Jian Li, Xiaotian Zhang, Yu Sun, Lin Shen, Yang Chen
          </td>
          <td>2025-02-04</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The role of glioma-associated myeloid cells in tumor growth and immune evasion remains poorly understood. We performed single-cell RNA sequencing of immune and tumor cells from 33 gliomas, identifying two distinct myeloid-derived suppressor cell (MDSC) populations in isocitrate dehydrogenase-wild-type (IDT-WT) glioblastoma: an early progenitor MDSC (E-MDSC) population with up-regulation of metabolic and hypoxia pathways and a monocytic MDSC (M-MDSC) population. Spatial transcriptomics demonstrated that E-MDSCs geographically colocalize with metabolic stem-like tumor cells in the pseudopalisading region. Ligand-receptor analysis revealed cross-talk between these cells, where glioma stem-like cells produce chemokines attracting E-MDSCs, which in turn produce growth factors for the tumor cells. This interaction is absent in IDH-mutant gliomas, associated with hypermethylation and repressed gene expression of MDSC-attracting chemokines. Our study elucidates specific MDSCs that may facilitate glioblastoma progression and mediate tumor immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c3594c9527f59638cf5b92d7a7853cdf4cfa76" target='_blank'>
              Distinct myeloid-derived suppressor cell populations in human glioblastoma.
              </a>
            </td>
          <td>
            Christina Jackson, Chris C. Cherry, Sadhana Bom, Arbor G. Dykema, Rulin Wang, Elizabeth A Thompson, Ming Zhang, Runzhe Li, Zhicheng Ji, Wenpin Hou, Wentao Zhan, Hao Zhang, John Choi, A. Vaghasia, Landon Hansen, W. Wang, Brandon Bergsneider, Kate M Jones, Fausto Rodriguez, Jon D. Weingart, Calixto-Hope G Lucas, Jonathan Powell, Jennifer H. Elisseeff, Srinivasan Yegnasubramanian, Michael Lim, C. Bettegowda, Hongkai Ji, Drew M Pardoll
          </td>
          <td>2025-01-17</td>
          <td>Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="PURPOSE
Identifying therapeutic targets for Signet Ring Cell Carcinoma (SRCC) of the colon and rectum is a clinical challenge due to the lack of Patient-Derived Organoids (PDO) or Xenografts (PDX). We present a robust method to establish PDO and PDX models to answer address this unmet need. We demonstrate that these models identify novel therapeutic strategies targeting therapy resistance and peritoneal metastasis.


EXPERIMENTAL DESIGN
We derived nine PDO and PDX models from colorectal SRCC patients. Detailed histopathological characterization confirmed the fidelity of these models to the original tumors. Drug sensitivity assays were conducted in-vitro and in-vivo to assess therapeutic efficacy and impact on peritoneal metastasis. An RNA-seq analysis was performed to identify critical pathways contributing to therapy resistance and metastatic progression.


RESULTS
We successfully developed and characterized PDO and PDX models from nine SRCC patients. The SRCC PDO and PDX models exhibited histopathological features consistent with the original tumors, including high mucin content and eccentric nuclei. They demonstrated increased sensitivity to FOLFIRI combined with Paclitaxel or vincristine, reducing peritoneal metastasis. RNA-seq analysis revealed the upregulation of autophagy genes in SRCC. Treatment with Chloroquine alone resulted in decreased tumor growth and peritoneal metastasis.


CONCLUSIONS
Our study establishes PDO and PDX models as robust platforms for studying SRCC and identifying potential therapeutic strategies. Combining FOLFIRI with paclitaxel/ vincristine or Chloroquine alone inhibits tumor growth and prevents peritoneal metastasis, showing promise for clinical translation. These findings suggest that combining FOLFIRI with IP paclitaxel warrants further investigation in Phase-I clinical trials for SRCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9301c1423aca2709165a86178794fe90ca3a11cd" target='_blank'>
              Patient-derived Organoids and Xenografts Uncover Therapeutic Vulnerabilities in Colorectal Signet Ring Cell Carcinomas.
              </a>
            </td>
          <td>
            Nazia Chaudhary, Alessandro La Ferlita, Bhagya Shree Choudhary, Eeshrita Jog, Mudaddal Kazi, Showket Yahya, Afiya Dalwai, V. Ostwal, Satishkumar Kumar, Siddhi Redkar, Nileema Khapare, Vaishali Kailaje, Akshaya B, P. Gera, Munita Bal, Nandini Verma, Rahul Thorat, Avanish P Saklani, L. Sehgal, S. Dalal
          </td>
          <td>2025-01-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66cb197889c56326016fd0967628911d5a7a8cef" target='_blank'>
              Unveiling the cell-type-specific landscape of cellular senescence through single-cell transcriptomics using SenePy
              </a>
            </td>
          <td>
            Mark A. Sanborn, Xinge Wang, Shang Gao, Yang Dai, Jalees Rehman
          </td>
          <td>2025-02-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d017ae1208a1788cb6dfdbf710e4c68d672f7f57" target='_blank'>
              Network-based transfer of pan-cancer immunotherapy responses to guide breast cancer prognosis
              </a>
            </td>
          <td>
            Xiaobao Ding, Lin Zhang, M. Fan, Lihua Li
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cancer therapy has advanced with molecularly targeted approaches and immunotherapy, yet chemotherapy remains essential for many aggressive cancers, including breast, lung, ovarian, pancreatic, bladder, sarcoma, and lymphomas. A major challenge is chemoresistance, in which cancer cells evade chemotherapy's cytotoxic effects. Overexpression of adenosine triphosphate-binding cassette transporters, especially P-glycoprotein, significantly contributes to this resistance. Thus, imaging biomarkers are urgently needed to detect P-glycoprotein overexpression in vivo, identify resistant cancer cell clones, and map their distribution and heterogeneity within tumors. This article reviews the applications of SPECT, PET, and optical imaging in addressing chemoresistance. It emphasizes the potential of these modalities to enhance cancer treatment by enabling early identification of resistant clones and improving therapeutic strategies. The article outlines key steps required for the integration of molecular imaging into clinical practice, aiming to overcome chemoresistance and optimize patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fa33cf2e65986ae14c892887c1f09f889730c52" target='_blank'>
              Molecular Imaging in Cancer Chemoresistance: What's Brewing?
              </a>
            </td>
          <td>
            Luca Urso, L. Uccelli, A. Boschi, Orazio Schillaci, Luca Filippi
          </td>
          <td>2025-02-06</td>
          <td>Journal of nuclear medicine : official publication, Society of Nuclear Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53b339228b07860b7f7b0ff107251f16ed54dc97" target='_blank'>
              Tumor microenvironment as a novel therapeutic target for lymphoid leukemias.
              </a>
            </td>
          <td>
            Shahrzad Mousavi, Soheil Nouri, Arezoo Sadeghipour, Amir Atashi
          </td>
          <td>2025-02-25</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e408aa172c15a4c549b8743b41474549ca768526" target='_blank'>
              Immunomodulatory role of exosome-derived content in pediatric medulloblastoma: a molecular subgroup perspective.
              </a>
            </td>
          <td>
            Jessica Oliveira de Santis, G. R. de Sousa, Rosane Gomes Queiroz, M. F. Cândido, Fausto Almeida, C. P. de Rezende, Patricia Cassia de Ruy, G. S. Arini, Beth Coyle, Philippa Wade, M. Brassesco, C. Scrideli, L. Tone, E. Valera
          </td>
          <td>2025-02-17</td>
          <td>Human cell</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Racial disparities in cancer outcomes are well documented across tumor types. For patients with breast cancer, Black women are more likely to present with more aggressive molecular features and more likely to die from disease, even after accounting for those features. Recent efforts have been aimed at developing translational model systems for precision medicine strategies, and a major focus has been on patient-derived organoids. Organoids allow for robust in vitro experimental platforms, including drug and CRISPR screens while maintaining more complex cancer and tumor microenvironment subpopulations than cell lines. For results that are broadly translationally relevant, it is important that cancer models are derived from the spectrum of human disease and humans with disease. In this issue of Cancer Research, Madorsky Rowdo and colleagues derive breast cancer organoids from patients with African ancestry and use CRISPR-Cas9 screens to identify novel therapeutic vulnerabilities. These findings demonstrate the promise of representative cancer model systems to facilitate discoveries that are most likely to translate to improved therapy for all patients. See related article by Madorsky Rowdo et al., p. 551.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af090730ccebba2da2c3e50d4cbf6a340cfee095" target='_blank'>
              In Search of Representative Translational Cancer Model Systems.
              </a>
            </td>
          <td>
            Hannah E. Trembath, Philip M. Spanheimer
          </td>
          <td>2025-02-01</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background The escalating global cancer burden necessitates the development of biomarkers with enhanced specificity and sensitivity for early diagnosis and therapeutic efficacy monitoring. The CNIH4 gene, an emerging biomarker, is increasingly recognized for its role in the malignant progression across various cancers. Methods We conducted a comprehensive multi-omics analysis of CNIH4, including pan-cancer expression profiles, epigenetic alterations, immune microenvironment characteristics, and therapeutic response patterns. Our focus was on clinical features, molecular underpinnings, and drug sensitivity in breast cancer (BRCA) associated with CNIH4. In vitro studies were also performed to assess the effects of CNIH4 knockdown on cell proliferation and cell cycle in the MDA-MB-231 cell line. Results CNIH4 upregulation was observed in multiple cancers, significantly correlating with genomic instability. High CNIH4 expression levels were linked to poor prognosis across cancers and associated with key cancer-related pathways, particularly those in cell cycle regulation and DNA repair. Correlation analyses suggest a role for CNIH4 in the tumor immune microenvironment, as evidenced by its association with immune subtypes, immune-related genes, and immune cell infiltration. Single-cell and spatial transcriptome analyses confirmed that CNIH4 expression in BRCA predicts tumor malignancy. Drug sensitivity analysis revealed a significant correlation between CNIH4 and responsiveness to various kinase inhibitors and chemotherapeutic agents. In vitro experiments demonstrated that CNIH4 knockdown significantly impacts the proliferation and cell cycle of MDA-MB-231 cells. Conclusion Our study highlights CNIH4 as a promising pan-cancer biomarker with significant implications for tumor progression and a critical role in cell cycle regulation in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7a2eade185851de2705c33f65b5768b5aafc69" target='_blank'>
              Deciphering the role of CNIH4 in pan-cancer landscapes and its significance in breast cancer progression
              </a>
            </td>
          <td>
            Yao Xu, Zengzhen Lai, Chaolin Li
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The reactivation of neurodevelopmental programs in cancer highlights parallel biological processes that occur in both normal development and brain tumors. Achieving a deeper understanding of how dysregulated developmental factors play a role in the progression of brain tumors is therefore crucial for identifying potential targets for therapeutic interventions. Single-cell RNA sequencing (scRNA-Seq) provides an opportunity to understand how developmental programs are dysregulated and reinitiated in brain tumors at single-cell resolution. The aim of this study is to identify developmental origins of brain tumors using scRNA-Seq data.



 Here, we introduce COORS (Cell Of ORigin like CellS), a computational tool trained on developmental human brain single-cell datasets that annotates “developmental-like” cell states in brain tumors. COORS leverages cell type-specific multilayer perceptron models and incorporates a developmental cell type tree that reflects hierarchical relationships and models cell type probabilities.



 Applying COORS to various brain cancer datasets, including medulloblastoma (MB), glioma, and diffuse midline glioma (DMG), we identified developmental-like cells that represent putative cells of origin in these tumors. Our method provides both cell of origin classification and cell age regression, offering insights into the developmental cell types of tumor subgroups. COORS identified outer radial glia (oRG) developmental cells within IDHWT glioma cells whereas oligodendrocyte precursor cells (OPCs) and neuronal-like cells in IDHMut. Interestingly, IDHMut subgroup cells that map to OPC show bimodal distributions, that are both early and late weeks in development. Furthermore, COORS offers a valuable resource by providing novel markers linked to developmental states within of MB, glioma and DMG tumor subgroups.



 Our work adds to our cumulative understanding of brain tumor heterogeneity and helps pave the way for tailored treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e403b1c16cfb5c2e03317b6d1828fc8f0bdaf21" target='_blank'>
              Inferred Developmental Origins of Brain Tumors from Single Cell RNA-Sequencing Data
              </a>
            </td>
          <td>
            Su Wang, R. Curry, Malcolm F McDonald, Hyun Yong Koh, Anders W Erickson, Claudia L Kleinman, Michael D. Taylor, Ganesh Rao, Benjamin Deneen, Arif O. Harmanci, A. Harmanci
          </td>
          <td>2025-01-23</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The development of therapies that target cancer stem cells (CSCs) and bulk tumors is both crucial and urgent. Several signaling pathways, like Notch and Hedgehog (Hh), have been strongly associated with CSC stemness maintenance and metastasis. However, the extensive crosstalk present between these two signaling networks complicates the development of long-term therapies that also minimize adverse effects on healthy tissues and are not overcome by therapy resistance from CSCs. The present work aims to overview the roles of Notch and Hh in cancer outburst and the intersection of the two pathways with one another, as well as with other networks, such as Wnt/β-catenin, TGF, and JAK/STAT3, and to explore the shaping of the tumor microenvironment (TME) with specific influence on CSC development and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f140e4496eee327489d912de5427944cb351ca1" target='_blank'>
              Notch and Hedgehog Signaling Unveiled: Crosstalk, Roles, and Breakthroughs in Cancer Stem Cell Research
              </a>
            </td>
          <td>
            Sabina Iluta, Mădălina Nistor, Sanda Buruiana, D. Dima
          </td>
          <td>2025-02-01</td>
          <td>Life</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Clonal lineage inference ("tumor phylogenetics") has become a crucial tool for making sense of somatic evolution processes that underlie cancer development and are increasingly recognized as part of normal tissue growth and aging. The inference of clonal lineage trees from single-cell sequence data offers particular promise for revealing processes of somatic evolution in unprecedented detail. However, most such tools are based on fairly restrictive models of the types of mutation events observed in somatic evolution and of the processes by which they develop. The present work seeks to enhance the power and versatility of tools for single-cell lineage reconstruction by making more comprehensive use of the range of molecular variant types by which tumors evolve. We introduce Sc-TUSV-ext, an integer linear programming-based tumor phylogeny reconstruction method that, for the first time, integrates single nucleotide variants, copy number alterations, and structural variations into clonal lineage reconstruction from single-cell DNA sequencing data. We show on synthetic data that accounting for these variant types collectively leads to improved accuracy in clonal lineage reconstruction relative to prior methods that consider only subsets of the variant types. We further demonstrate the effectiveness of real data in resolving clonal evolution in the presence of multiple variant types, providing a path toward more comprehensive insight into how various forms of somatic mutability collectively shape tissue development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Semaphorins (SEMAs), originally identified as axon guidance factors, have been found to play crucial roles in tumor growth, invasiveness, neoangiogenesis, and the modulation of immune responses. However, the prognostic value of SEMA-related genes in colorectal cancer (CRC) remains unclear.We applied a novel machine learning framework that incorporated 10 machine learning algorithms and their 101 combinations to construct a SEMAs-related score (SRS). Multi-omics analysis was performed, including single-cell RNA sequencing (scRNA-seq), and spatial transcriptome (ST) to gain a more comprehensive understanding of the SRS. A series of cell experiments were conducted to prove the impact of key genes on CRC biological behavior.A consensus SRS was finally constructed based on a 101-combination machine learning computational framework, demonstrating outstanding performance in predicting overall survival. Moreover, distinct biological functions, mutation burden, immune cell infiltration, and immunotherapy response were observed between the high- and low-SRS groups. scRNA-seq and ST demonstrated unique cellular heterogeneity in CRC. We observed that SRS-high and SRS-low malignant epithelial cells exhibit different biological characteristics. High SRS malignant epithelial cells interact with myeloid and endothelial cells via SPP1 and COL4A2-ITGAV-ITGB8 pathways, respectively. Low SRS cells engage with myeloid and endothelial cells through MIF and JAG1-NOTCH4 pathways. Additionally, knocking down SEMA4C significantly inhibits the proliferation and invasion of CRC cells, while promoting apoptosis in vitro.SRS could serve as an effective tool to predict survival and identify potential patients benefiting from immunotherapy in CRC. It also reveals tumor heterogeneity and provides valuable biological insights in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b27be243d13affabe786e086c79b848143a85be" target='_blank'>
              Integrative analysis of semaphorins family genes in colorectal cancer: implications for prognosis and immunotherapy
              </a>
            </td>
          <td>
            Jiahao Zhu, Benjie Xu, Zhixing Wu, Zhiwei Yu, Shengjun Ji, Jie Lian, Haibo Lu
          </td>
          <td>2025-03-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8641c47a99cb4e08fcabebc4fec5794fc9e23e2f" target='_blank'>
              Role of CD4+ T cells in cancer immunity: a single-cell sequencing exploration of tumor microenvironment
              </a>
            </td>
          <td>
            Qi An, Li Duan, Yuanyuan Wang, Fuxin Wang, Xiangyu Liu, Chao Liu, Qinyong Hu
          </td>
          <td>2025-02-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumor microenvironment (TME) milieu directs a plethora of tumor-modulating functions. Recent years have seen pivotal breakthroughs in our understanding of the TME's role in tumor initiation and progression, with tangible clinical applications. Individual components of the TME exert their function predominantly by cell-cell crosstalk, both in the form of physical interaction and secreted factors. Notably, different spatial niches represent exclusive signaling hubs in the TME, propagating pro- or antitumoral functions. The exploration of these interactions has been vastly facilitated by novel molecular technologies, each of which provides a different perspective on this intricate intercellular communication network. Together, these complementary methods paint a detailed, high-resolution map of the TME's interaction landscape. In this viewpoint, we explore how cellular interactions can unlock a new level of understanding of TME complexity, and discuss the promises and challenges of characterizing tumors based on their cellular interaction footprint.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/072aea9d500a6665bd7f3f737b00c34169b4d84a" target='_blank'>
              Decoding multicellular interaction networks-a new horizon in tumor microenvironment research.
              </a>
            </td>
          <td>
            Roi Balaban, Merav Cohen
          </td>
          <td>2025-02-05</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background The goal of precision oncology is to find effective therapeutics for every patient. Through the inclusion of emerging therapeutics in a high‐throughput drug screening platform, our functional genomics pipeline inverts the common paradigm to identify patient populations that are likely to benefit from novel therapeutic strategies. Approach Utilizing drug screening data across a panel of 46 cancer cell lines from 11 tumor lineages, we identified an ovarian cancer‐specific sensitivity to the first‐in‐class CRL4 inhibitors KH‐4‐43 and 33‐11. CRL4 (i.e., Cullin‐4 RING E3 ubiquitin ligase) is known to be dysregulated in a variety of cancer contexts, making it an attractive therapeutic target. Unlike proteasome inhibitors that are associated with broad toxicity, CRL4 inhibition offers the potential for tumor‐specific effects. Results We observed that CRL4 inhibition negatively regulates core gene signatures that are upregulated in ovarian tumors and significantly slowed tumor growth as compared to the standard of care, cisplatin, in OVCAR8 xenografts. Building on this, we performed combination drug screening in conjunction with proteomic and transcriptomic profiling to identify ways to improve the antitumor effects of CRL4 inhibition in ovarian cancer models. CRL4 inhibition consistently resulted in activation of the mitogen‐activated protein kinase (MAPK) signaling cascade at both the transcriptomic and protein levels, suggesting that survival signaling is induced in response to CRL4 inhibition. These observations were concordant with the results of the combination drug screens in seven ovarian cancer cell lines that showed CRL4 inhibition cooperates with MEK inhibition. Preclinical studies in OVCAR8 and A2780 xenografts confirmed the therapeutic potential of the combination of KH‐4‐43 and trametinib, which extended overall survival and slowed tumor progression relative to either single agent or the standard of care. Conclusions Together, these data demonstrate the prospective utility of functional modeling pipelines for therapeutic development and underscore the clinical potential of CRL4 inhibition in the ovarian cancer context. Highlights A precision medicine pipeline identifies ovarian cancer sensitivity to CRL4 inhibitors. CRL4 inhibition induces activation of MAPK signalling as identified by RNA sequencing, proteomics, and phosphoproteomics. Inhibitor combinations that target both CRL4 and this CRL4 inhibitor‐induced survival signalling enhance ovarian cancer sensitivity to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261c6bcbc9392bde91ee7e96491c8582460ba6dd" target='_blank'>
              Functional genomics pipeline identifies CRL4 inhibition for the treatment of ovarian cancer
              </a>
            </td>
          <td>
            S. Claridge, Shalini Nath, Anneliese Baum, Richard Farias, J. Cavallo, Nile M Rizvi, Lamberto De Boni, Eric Park, Genesis Lara Granados, Matthew Hauesgen, Ruben Fernandez-Rodriguez, Eda N Kozan, E. Kanshin, Khoi Q Huynh, Peng-Jen Chen, Kenneth Wu, Beatrix Ueberheide, Juan Miguel Mosquera, F. Hirsch, Robert J. DeVita, O. Elemento, Chantal Pauli, Zhen-Qiang Pan, Benjamin D Hopkins
          </td>
          <td>2025-01-24</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Grade 3 ependymoma is classified as a fast-growing solid tumor that often occurs in the base of the brain with a median survival of less than two years. Despite aggressive standard treatments (surgery, radiation therapy and chemotherapy), grade 3 ependymoma remains uniformly fatal, especially if the tumor cannot be completely removed or if it spreads to other areas of the central nervous system (CNS). Achieving a responsive therapy with chimeric antigen receptor (CAR) T cell therapy has been a significant challenge due to evasive tumor microenvironment, specifically utilized by slid tumors, to resist therapy. Recent studies have highlighted the substantial role of glioma-associated stromal cells in solid tumors by depositing various collagen subunits, including COL1A1, within the extracellular matrix (ECM). This collagen deposition leads to increased ECM stiffness, modulating the tumor microenvironment, inducing immune suppression (e.g. SPP1 overexpression), and hindering T cell trafficking, ultimately worsening clinical outcomes and patient survival. To gain a better understanding of the role of the tumor microenvironment in CAR T persistence and efficacy, we designed and performed ultra-high plex immunofluorescent staining on tissue biopsies obtained from a patient with grade 3 ependymoma before and after treatment with IL13Ra2 CAR T cells. The patient has been received three CAR T infusions at weekly intervals. The tissues were stained for CD3, CD19 (CAR T cells), CD68, CD163, HLA-DR (macrophages), CD3, CD8 (endogenous T cells), SOX2 (stem cells), SPP1 (suppression) and COL1A1 (extracellular matrix). The presence of CAR T cells in tissue biopsies indicated persistence of IL13Ra2 CAR T cells for more than a year. Our preliminary analysis revealed an increase in the number of HLA-DR+ CD163+ macrophages within cellular neighborhoods associated with CAR T cells (around 50% of CD163+ express HLA-DR after treatment) as compared to baseline tumors, which suggests a dynamic interaction between myeloid subsets and CAR T cells within the tumor microenvironment. Ongoing experiments are currently investigating the presence of antigen presenting natural killer cells (CD56, HLA-DR), proliferative T cells (CD3, Ki67) and inflammatory macrophages and/or T cells (IFNg) to gain a better understanding of IL13Ra2 persistence in brain tumor patients.
 Citation Format: Maryam Aftabizadeh, Brenda Aguilar, Massimo D'Apuzzo, Behnam Badie, Christine E Brown. A deeper understanding of tumor microenvironment modulation induced by IL13Ra2 CAR T cells in a patient with grade 3 ependymoma [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A030.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0786e178852bb9b6b2a0945adb3bddac988e519" target='_blank'>
              Abstract A030: A deeper understanding of tumor microenvironment modulation induced by IL13Ra2 CAR T cells in a patient with grade 3 ependymoma
              </a>
            </td>
          <td>
            Maryam Aftabizadeh, B. Aguilar, M. D’Apuzzo, B. Badie, Christine E. Brown
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR-T) cell therapy has emerged as a promising treatment for hematological malignancies, offering a targeted approach to cancer treatment. Understanding the complexities of CAR-T cell therapy within solid tumors poses challenges due to the intricate interactions within the tumor microenvironment. Mathematical modeling may serve as a valuable tool to unravel the dynamics of CAR-T cell therapy and improve its effectiveness in solid tumors. This study aimed to investigate the impact of spatial aspects in CAR-T therapy of solid tumors, utilizing cellular automata for modeling purposes. Our main objective was to deepen our understanding of treatment effects by analyzing scenarios with different spatial distributions and varying the initial quantities of tumor and CAR-T cells. Tumor geometry significantly influenced treatment efficacy in-silico, with notable differences observed between tumors with block-like arrangements and those with sparse cell distributions, leading to the concept of immune suppression due to geometrical effects. This research delves into the intricate relationship between spatial dynamics and the effectiveness of CAR-T therapy in solid tumors, highlighting the relevance of tumor geometry in the outcome of cellular immunotherapy treatments. Our results provide a basis for improving the efficacy of CAR-T cell treatments by combining them with other ones reducing the density of compact tumor areas and thus opening access ways for tumor killing T-cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b56368d398e6734b66e55599843f0e9cefd36d23" target='_blank'>
              Geometric immunosuppression in CAR-T cell treatment: Insights from mathematical modeling
              </a>
            </td>
          <td>
            Silvia Bordel-Vozmediano, S. Sabir, Luc'ia Benito-Barca, B. Weigelin, V'ictor M. P'erez-Garc'ia Mathematical Oncology Laboratory, Instituto de Matem'atica Aplicada a la Ciencia y la Ingenier'ia, U. C. Mancha, Ciudad Real, Spain., D. D. Matem'aticas, Escuela T'ecnica Superior de Ingenieros Industriales, Facultad de Ciencias Experimentales, Universidad Francisco de Vitoria, Pozuelo de Alarc'on, Department of Functional Imaging, Radiopharmacy, Multiscale Immunoimaging, E. University, Tubingen, Germany, Cluster of Excellence iFIT ''Image-Guided, Functionally Instructed Tumor Therapies''
          </td>
          <td>2025-01-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ad7065cbf4c15026ee68bc1c1e9443de9a4af3" target='_blank'>
              Identification of T cell dysfunction molecular subtypes and exploration of potential immunotherapy targets in BRAF V600E-mutant colorectal cancer
              </a>
            </td>
          <td>
            Tiefeng Gu, Haonan Qi, Jiaqi Wang, Liangwei Sun, Yongqi Su, Hanqing Hu
          </td>
          <td>2025-02-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Epitranscriptomic modifications impact RNA stability and function, influencing key biological processes. While initially behind epigenetics, the field has advanced in the last decade, linking these modifications to development and diseases like cancer. High-grade gliomas, such as glioblastoma, remain challenging due to their aggressiveness and lack of effective treatments. Understanding the impact of epitranscriptomic alterations in gliomas may provide new insights into tumor progression and therapeutic strategies. This review provides an overview of RNA modifications in tRNAs, rRNAs, miRNAs, and mRNAs, exploring the role of the proteins involved in epitranscriptomics and their dysregulation in gliomas. We discuss how this imbalance affects pathways in glioma progression and highlight recent advancements in RNA modification-based therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bcb06b4f2cf9c15faeac2d418f2e3e672130f79" target='_blank'>
              Epitranscriptomics in the Glioma Context: A Brief Overview
              </a>
            </td>
          <td>
            Pablo Santamarina-Ojeda, Agustin F. Fernández, Mario F. Fraga
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Aberrant expression and mutations in the fibroblast growth factor (FGF) family play crucial roles in cell differentiation, growth, and migration, contributing to tumor progression across various cancers. Nasopharyngeal carcinoma (NPC), a malignancy prevalent in East Asia, is primarily treated with radiotherapy; however, radioresistance remains a major challenge, leading to recurrence and poor outcomes. While FGFs are known to activate signaling pathways such as MAPK, PI3K/AKT, and JAK/STAT to promote cancer progression, the specific role of individual FGFs in NPC radioresistance remains unclear. Emerging evidence highlights FGF5 as a key player in NPC progression, metastasis, and radioresistance, underscoring its potential as a therapeutic target to overcome treatment resistance and improve clinical outcomes. Methods: We analyzed single nucleotide variation (SNV) data, gene expression, and DNA methylation patterns using cancer datasets, including TCGA and GTEx, to investigate FGF5 expression. Differentially expressed genes (DEGs) were identified and interpreted using functional enrichment analysis, while survival analysis and gene set enrichment analysis (GSEA) were conducted to identify clinical correlations. DNA methylation patterns were specifically assessed using the HumanMethylation850 BeadChips on tissue samples from nine recurrent and nine non-recurrent NPC patients. Functional assays, including cell viability, migration, invasion, and clonogenic survival assays, were performed to evaluate the effects of FGF5 on NPC cell behavior in vitro and in vivo. Results: FGF5 showed elevated SNV frequencies across multiple cancers, particularly in HNSC and NPC. DNA methylation analysis revealed an inverse relationship between FGF5 expression and methylation levels in recurrent NPC tumors. Functional assays demonstrated that FGF5 enhances migration, invasion, and radioresistance in NPC cells. High FGF5 expression was associated with reduced distant metastasis-free survival (DMFS) and increased radioresistance, highlighting its role in metastatic progression and recurrence. Conclusions: FGF5 plays a significant role in the progression and recurrence of nasopharyngeal carcinoma. Its elevated expression correlates with increased migration, invasion, and radioresistance as well as reduced distant metastasis-free survival. These findings suggest that FGF5 contributes to the metastatic and recurrence potential of NPC, making it a potential target for therapeutic intervention in treating these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5304b34e9ca6a3e7c770b19af10ccf63ce40f9c2" target='_blank'>
              Multi-Omics Characterization of Genome-Wide Abnormal DNA Methylation Reveals FGF5 as a Diagnosis of Nasopharyngeal Carcinoma Recurrence After Radiotherapy
              </a>
            </td>
          <td>
            Z. Long, Ran Ding, Ting-Qiu Quan, Rui Xu, Zhuo-Hui Huang, D. Wei, W. Zheng, Ying Sun
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d719de0ed34b62d63deb744432982b8548a3e52" target='_blank'>
              TGF-β signaling orchestrates cancer-associated fibroblasts in the tumor microenvironment of human hepatocellular carcinoma: unveiling insights and clinical significance
              </a>
            </td>
          <td>
            Junwei Ge, Hongwei Jiang, Junjun Chen, Xuemin Chen, Yue Zhang, Liangrong Shi, Xiao Zheng, Jingting Jiang, Lujun Chen
          </td>
          <td>2025-01-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Liver cancer exhibits diverse molecular characteristics and distinct immune cell infiltration patterns, which significantly influence patient outcomes. In this study, we thoroughly examined the liver cancer tumor environment by analyzing data from 419,866 individual cells across nine datasets involving 99 patients. By categorizing patients into different groups based on their immune cell profiles, including immune deficiency, B cells-enriched, T cells-enriched and macrophages-enriched, we better understood how these cells change in various patient subgroups. Our investigation of liver metastases from intestinal cancer uncovered a group of mast cells that might promote metastasis through pathways like inositol phosphate metabolism. Using genomic and clinical data from The Cancer Genome Atlas, we identified specific cell components linked to tumor characteristics and genetics. Our detailed study of cancer-associated fibroblasts (CAFs) revealed how they adapt and acquire new functions in the tissue environment, highlighting their flexibility. Additionally, we found a significant connection between CAF-related genes and the prognosis of hepatocellular carcinoma patients. This research provides valuable insights into the makeup of the liver cancer tumor environment and its profound impact on patient outcomes, offering fresh perspectives for managing this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dff3872da8e8b83984c596296a89bdd2fda835d8" target='_blank'>
              Deciphering the heterogeneity and plasticity of the tumor microenvironment in liver cancer provides insights for prognosis
              </a>
            </td>
          <td>
            Yihao Sun, Guojuan Shi, Jian Yang, Chun-Zhong Zhou, Chuhan Peng, Yu-Hong Luo, Ying Pan, Rui-Qi Wang
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a heterogeneous disease that develops through a stepwise accumulation, yet the underlying mechanisms at single-cell resolution remain unclear. Here, we profiled 751,531 single-cell transcriptomes, spatial transcriptomics, and snMultiomes from 142 multistage samples, revealing the cellular and molecular alterations and dynamic intercellular crosstalk during CRC development. Additionally, we created a CRC sc-eQTL map identifying 16,833 significant pairs across 28 cell subtypes, with over 76% of sc-eQTLs being cell-type-specific and fewer than 15% detectable in bulk datasets. A polygenic risk score (PRS) derived from sc-eQTLs substantially improved CRC risk prediction. We prioritized rs4794979 that is associated with an increased CRC risk (OR=1.11, P=2.04×10-12) by promoting LGALS9 expression mediated by ELK1. Elevated LGALS9 in epithelia interacts with SLC1A5 on fibroblasts, promoting transformation into cancer-associated fibroblasts (CAFs), simultaneously induces CD8+ T cells exhaustion via LGALS9-TIM3 axis, thereby facilitating CRC development. Blocking LGALS9-TIM3 axis enhanced anti-PD-1 therapy to inhibit CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c8b8c8967239c4ebdf5921b795eefe1341f9d3" target='_blank'>
              Single-cell eQTL mapping reveals cell subtype-specific genetic control and mechanism in malignant transformation of colorectal cancer.
              </a>
            </td>
          <td>
            Can Chen, Yimin Cai, Wenlong Hu, Kai Tan, Zequn Lu, Xuanyu Zhu, Ziying Liu, Chunyi He, Guangping Xu, Ruizhe Zhang, Caibo Ning, Shuheng Ruan, Jiayan Gao, Xiaojun Yang, Yongchang Wei, Xu Zhu, Xiangpan Li, Faxi Wang, Fubing Wang, Jiaoyuan Li, Meng Jin, Bin Li, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-03-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Metastatic melanoma is a lethal form of cancer. Immunotherapy is recommended for metastatic melanoma and known to elicit beneficial response even after disease progression per RECIST. On-treatment samples hold irreplaceable predictive value. We collected needle biopsies from the metastases of different body sites from the same patient that progressed upon PD-1 treatment. Leveraging Direct Nuclear Tagmentation and RNA sequencing (DNTR- seq), we conducted single-cell genomic analyses on melanoma metastases and jointly sequenced the transcriptomes of each single melanoma cell. After 3 months of PD-1 inhibitor treatment, single-cell DNA-seq analyses on melanoma metastases from different anatomic locations displayed distinct cancer subclones evidenced by their copy number aberrations. Joint transcriptomic analyses of the dominant genomic subclones from each body site showed converging phenotypes, under the selective pressure of immunotherapy. Our preliminary analyses of on-treatment needle biopsies shed light on the challenges of treating heterogeneous tumors and underscore the pressing need to identify an ideal denominator to reach disease control.
 Citation Format: Muyi Yang, Solrun Kolbeinsdottir, Katarina Hammarlund, Vitali Grozman, Martin Enge, Suzanne Egyhazi-Brage, Hildur Helgadottir. Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A108.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059cc807139dc64898d45632991b55c44cdfb1d6" target='_blank'>
              Abstract A108: Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients
              </a>
            </td>
          <td>
            Muyi Yang, Sólrún Kolbeinsdóttir, Katarina Hammarlund, V. Grozman, Martin Enge, Suzanne Egyhazi-Brage, H. Helgadottir
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) presents significant challenges in clinical management due to its aggressive nature. Here, we elucidate the role of MLL3 mutations as early, clonal genomic events in UASCC tumorigenesis, highlighting their role as foundational drivers of cancer development. Utilizing CRISPR-edited, cross-species organoid modeling, we demonstrate that loss of MLL3 contributes to early squamous neoplastic evolution. Furthermore, we identify an MLL3/GRHL2 protein complex that regulates the UASCC epigenome, particularly impacting immune response pathways. Notably, a novel MLL3/GRHL2-IRF1 axis promotes the expression of Th1 chemokines, enhancing anti-tumor immunity by facilitating T cell infiltration into the tumor microenvironment. Consequently, MLL3 regulates the in vivo efficacy of immune checkpoint blockade (ICB) therapy, corroborated by the strong association between MLL3 expression and human patients' clinical response to ICB therapy. Our work underscores the significance of MLL3 in UASCC pathogenesis and highlights the interplay between MLL3/GRHL2 and immune response pathways as potential therapeutic targets for UASCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50aaf113e900513b3c01d9cc2408109f9601e35a" target='_blank'>
              The MLL3/GRHL2 complex regulates malignant transformation and anti-tumor immunity in squamous cancer.
              </a>
            </td>
          <td>
            Chehyun Nam, Guowei Huang, Yueyuan Zheng, Hua Zhao, Yuhao Pan, Boyan Hu, Talia Wenger, Hieu T. Van, Liyan Xu, E. Li, H. Koeffler, Kai Ge, Y. Dou, Uttam K Sinha, Young Min Park, De-Chen Lin
          </td>
          <td>2025-02-18</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) drive tumor progression through restructuring of the tumor microenvironment. This investigation aim to elucidate the function of molecular subtypes (MS) derived from cancer cells communication with CAFs, depicting the hallmarks of the tumor microenvironment and precise bladder cancer (BLCA) treatment. The BLCA data from TCGA and several external sources are utilized to generate a novel ligand, receptor, and transcription factor (LRT) associated molecular subtype and their corresponding score (LRT score). The LRT-mediated molecular subtype is identified via unsupervised clustering. LRT score is measured by principal component analysis. Then, the association of LRT clusters to established MS, immunophenotypes, and medical endpoints, together with BLCA treatment strategies is investigated. Two LRT clusters (A and B) are identified. LRT cluster (LRT score) can precisely propose immunophenotypes, classical MS, clinical outcomes, and BLCA therapeutic strategies. Cluster B (Low LRT score) represent a basal subtype and inflamed phenotype specified by high immunity against tumors and unfavorable clinical outcomes. Furthermore, it is highly sensitive to cancer immunotherapy; however, it has low sensitivity to antiangiogenic and targeted therapies. The novel LRT clusters with a strong association with biological characteristics and precise BLCA treatment strategies are derived from the communication between cancer cells and cancer-associated fibroblasts. The LRT may be a useful clinician tool for developing individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f8d25128e68026c13f6a41f5701ec7d66c8485" target='_blank'>
              Cancer Cell and Cancer-Associated Fibroblast Communication-Mediated Molecular Subtypes Portray Non-Inflamed Tumor Microenvironment and Guide the Precision Treatment of Bladder Cancer.
              </a>
            </td>
          <td>
            Shenglin Gao, Chuan Liu, Lixin Mao, Yin Chen, Xiaokai Shi, C. Yue, Shouchun Li, Xihu Qin
          </td>
          <td>2025-02-17</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Understanding how intratumoral immune populations coordinate antitumor responses after therapy can guide treatment prioritization. We systematically analyzed an established immunotherapy, donor lymphocyte infusion (DLI), by assessing 348,905 single-cell transcriptomes from 74 longitudinal bone marrow samples of 25 patients with relapsed leukemia; a subset was evaluated by both protein- and transcriptome-based spatial analysis. In acute myeloid leukemia (AML) DLI responders, we identified clonally expanded ZNF683+ CD8+ cytotoxic T lymphocytes with in vitro specificity for patient-matched AML. These cells originated primarily from the DLI product and appeared to coordinate antitumor immune responses through interaction with diverse immune cell types within the marrow microenvironment. Nonresponders lacked this cross-talk and had cytotoxic T lymphocytes with elevated TIGIT expression. Our study identifies recipient bone marrow microenvironment differences as a determinant of an effective antileukemia response and opens opportunities to modulate cellular therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dc256708884cd08679466f87592f86695fa33d1" target='_blank'>
              Coordinated immune networks in leukemia bone marrow microenvironments distinguish response to cellular therapy.
              </a>
            </td>
          <td>
            K. Maurer, Cameron Y Park, Shouvik Mani, M. Borji, Florian Raths, Kenneth H Gouin, Livius Penter, Yinuo Jin, Jia Yi Zhang, Crystal Shin, James R Brenner, Jackson Southard, Sachi Krishna, Wesley S Lu, Haoxiang Lyu, Domenic Abbondanza, Chanell Mangum, Lars Rønn Olsen, Michael J Lawson, Martin M. Fabani, D. Neuberg, P. Bachireddy, Eli N Glezer, Samouil L. Farhi, Shuqiang Li, K. Livak, Jerome Ritz, Robert J. Soiffer, Catherine J. Wu, Elham Azizi
          </td>
          <td>2025-01-24</td>
          <td>Science immunology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="


 Glioblastoma (GB), is an incurable brain tumor characterized by extreme malignancy and invasiveness and the cellular mechanisms underlying such a severe phenotype are not completely understood. Although calcium (Ca2+) plays an important part in tumor proliferation and infiltration, it remains unclear whether Ca2+ signaling in GB cells is related to its location within the tumor and on the infiltrative potential of the cells.



 In this study, we developed a stably transfected GL261 cell line that co-expresses a red fluorescent protein for actin cytoskeleton staining and the intracellular Ca2+ sensor, GCaMP6s. By means of intravital two photon imaging we have characterized the morphological and functional properties of cells at different locations within the tumor.



 Our results showed that cells located at the tumor core are densely packed and rounded in shape, contrasting sharply with the polarized morphology observed in the peripheral cells. This anatomical heterogeneity corresponded to notable variations of the physiological phenotype: cells at the tumor core displayed low Ca2+ activity and very limited motility, while peripheral cells displayed intense Ca2+ activity and increased migration rates. Moreover, peripheral cells formed cellular ensemble characterized by synchronized Ca2+ activity accompanied by a directionally biased collective motility. Conclusions: These findings suggest that GB cells manifest activity patterns depending upon their spatial location within the tumor and that these correlate with their migration.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f476d5b2005f0aec9da0be185757bb8f7ffc3130" target='_blank'>
              Heterogeneity of intracellular calcium signaling of Glioblastoma cells depends on intratumoral location and migration state
              </a>
            </td>
          <td>
            V. Pillai, E. Pracucci, Francesco Trovato, Riccardo Parra, S. Landi, G. Ratto
          </td>
          <td>2025-03-05</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ceeafcf1da623087d79a8767b5543bf2a7043f8" target='_blank'>
              Genomic and micro-environmental insights into drug resistance in colorectal cancer liver metastases
              </a>
            </td>
          <td>
            Junjie Kuang, Jun Li, Siwei Zhou, Yi Li, Jinbo Lin, Wei-zeng Huang, Xia Yuan
          </td>
          <td>2025-02-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97569fe84c28d5e70fd4d5d44460cfdc4508b03d" target='_blank'>
              Quantifying cell divisions along evolutionary lineages in cancer.
              </a>
            </td>
          <td>
            Martin Blohmer, David M Cheek, Wei-Ting Hung, Maria Kessler, Foivos Chatzidimitriou, Jiahe Wang, William Hung, I-Hsiu Lee, Alexander N Gorelick, Emma Ce Wassenaar, Ching-Yeuh Yang, Yi-Chen Yeh, H. Ho, Dorothee Speiser, Maria M Karsten, Michael Lanuti, Sara I Pai, O. Kranenburg, J. Lennerz, Teh-Ying Chou, Matthias Kloor, Kamila Naxerova
          </td>
          <td>2025-02-04</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>55</td>
        </tr>

        <tr id="PURPOSE
Digital proteomic profiling was performed to identify spatial context in relationship to patient response and survival after anti-PD-1 therapy in metastatic colorectal cancer (CRC).


EXPERIMENTAL DESIGN
Primary CRCs with deficient mismatch repair (d-MMR) from patients treated with anti-PD-1 antibodies were analyzed (N=30) using Digital Spatial Profiling (GeoMx® nCounter). At the invasive margin, 71 proteins were profiled in 10 regions of interest /slide that were segmented into 3 compartments labeled with pan-cytokeratin (PanCK; epithelia), CD45 (stromal cells) and SYTO13 (nuclei). In an independent cohort (n=13), DSP data and single cell transcriptomic data were analyzed. Differential protein abundance, after Benjamini-Hochberg correction, was examined by response and progression-free survival (PFS) using multivariable Cox regression.


RESULTS
Protein abundance varied significantly between epithelial and stromal compartments. Nonresponders (NR) to anti-PD-1 showed higher fibronectin and smooth muscle actin (SMA) abundance in the epithelial compartment that were associated with significantly shorter PFS (HRadj: 6.49 and 4.52, respectively; p < 0.05). In CD45+ stroma, increased expression of proteins related to T-cells (CD3, CD4), NK cells (CD56), antigen presentation (CD40), immune activation (CD27, ICOS), and apoptosis (GZMA) were found in responders (vs NR) to anti-PD-1; each marker was significantly associated with longer patient PFS (0.02 <HRadj < 0.17; p < 0.05). In a separate cohort, consistent results by compartment were found for fibronectin and CD56. Gene expression data revealed that fibronectin and SMA were primarily derived from cancer-associated fibroblasts.


CONCLUSIONS
Spatially resolved protein profiles within microenvironments of d-MMR CRCs can influence patient response and survival after anti-PD-1 highlighting their potential clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7e12e22717ab1e00830f945fd4f97470fd6160b" target='_blank'>
              Spatially resolved, multi-region proteomics for prediction of immunotherapy outcome in deficient mismatch repair metastatic colorectal cancer.
              </a>
            </td>
          <td>
            B. Saberzadeh-Ardestani, Zhenglong Liu, Mariam I Stein, Will A Sherman, Christy E Trussoni, Charles W Abbott, Dongyao Yan, Skyler Smith, K. Shanmugam, Rondell P. Graham, Alos Diallo, Joshua J. Levy, Tamas Ordog, F.A. Sinicrope
          </td>
          <td>2025-02-19</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Macrophage-mediated phagocytosis plays a critical role in cancer cell elimination. Here, we developed an in vitro tumor-macrophage co-culture platform for unbiased identification of factors that impede macrophage phagocytosis using genome-wide CRISPR knockout. From the functional genomics screen, we identified APMAP as a tumor-intrinsic phagocytosis sensitizer. In vitro experiments validated that APMAP depletion in tumor cells enhances direct engulfment and selective killing of tumor cells by macrophages. In vivo experiments showed that APMAP depletion attenuates tumor growth and displays combinational effects with cetuximab in an MDA-MB-468 orthotopic model. To dissect the mechanism of action, we conducted a whole genome anchor screen using an APMAP-WT/KO isogenic pair and identified fatty acid biosynthesis pathway genes (including FASN, ACLY and ACC) that are required for APMAP deficiency-induced phenotypes; the metabolomics analysis suggests that sphingolipids are potential APMAP substrates. To inform drug discovery, we demonstrated that enzymatic activity is required for APMAP-enhanced phagocytosis. We further demonstrated that as a transmembrane protein, intracellularly localized APMAP was functional as a phagocytosis sensitizer, thereby suggesting that a small molecule inhibitor is preferred over a blocking antibody as a therapeutic strategy for disrupting APMAP activity.
 Citation Format: Rachel Davidson, Ye Wang, Lei Ji, Grace Trombley, Wenrong Zhou, Binzhang Shen, Ashley Choi, Hannah Stowe, Shangtao Liu, Samuel R. Meier, Brian Doyon, Douglas A. Whittington, Yi Yu, Teng Teng, Jannik N. Andersen, Alan Huang, Chengyin Min, Salam Shaaban, Serge Gueroussov. APMAP functions as a tumor-intrinsic target sensitizing cancer cells to macrophage-mediated phagocytosis [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e96ffa974b2617454180a8fb03fe06294bae79bb" target='_blank'>
              Abstract B078: APMAP functions as a tumor-intrinsic target sensitizing cancer cells to macrophage-mediated phagocytosis
              </a>
            </td>
          <td>
            Rachel Davidson, Ye Wang, Lei Ji, Grace Trombley, Wenrong Zhou, B. Shen, Ashley H Choi, Hannah Stowe, Shangtao Liu, Samuel R. Meier, Brian Doyon, D. Whittington, Yi Yu, T. Teng, Jannik N Andersen, Alan Huang, Chengyin Min, Salam Shaaban, Serge Gueroussov
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d479e3aee16692284dd5875a578cce134082060b" target='_blank'>
              Cell signaling communication within papillary craniopharyngioma revealed by an integrated analysis of single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Xiaoyue Zhu, Yanfei Jia, Zicheng Zhao, Xiaoyu Zhang, Yunlong Zhao, Songbai Gui, Xiu-An Yang
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gastric cancer (GC) remains a leading cause of cancer-related mortality, with over one million new cases and 769,000 deaths reported in 2020. Despite advancements in chemotherapy, surgery, and targeted therapies, delayed diagnosis due to overlooked early symptoms leads to poor prognosis.We integrated bulk RNA sequencing and single-cell RNA sequencing datasets from TCGA, GEO, and OMIX001073, employing normalization, batch effect correction, and dimensionality reduction methods to identify key cell populations associated with GC invasion and epithelial-mesenchymal transition (EMT), as well as analyze the tumor immune microenvironment.Our analysis identified the MUC5AC+ malignant epithelial cell cluster as a significant player in GC invasion and EMT. Cluster 1, representing this cell population, exhibited higher invasion and EMT scores compared to other clusters. Survival analysis showed that high abundance in cluster 0 correlated with improved survival rates (P=0.012), whereas cluster 1 was associated with poorer outcomes (P=0.045). A prognostic model highlighted ANXA5 and GABARAPL2 as two critical genes upregulated in GC tumors. High-risk patients demonstrated increased immune cell infiltration and worse prognosic. Analysis of tumor mutation burden (TMB) indicated that patients with low TMB in the high-risk group had the worst prognosis. Wet-lab validation experiments confirmed the oncogenic role of ANXA5, showing its facilitation of cell proliferation, invasion, and migration while suppressing apoptosis.This study offers novel insights into the subpopulations of malignant epithelial cells in GC and their roles in tumor progression. It provides a prognostic model and potential therapeutic targets to combat GC, contributing crucial understanding to the fundamental mechanisms of drug resistance in gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9fbc569ffcfa0009c2b9e523878c6293cec12b" target='_blank'>
              Integrative analysis of single-cell and bulk RNA sequencing reveals the oncogenic role of ANXA5 in gastric cancer and its association with drug resistance
              </a>
            </td>
          <td>
            Denggang Chen, Peng Zhang, Li Gong, Hailang Wei, Guanghui Yu, Tingting Zhang, Chen Bai
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1610b8e09ca06ead7a9826cda40c37396bdae40e" target='_blank'>
              Atlas of multilineage stem cell differentiation reveals TMEM88 as a developmental regulator of blood pressure
              </a>
            </td>
          <td>
            Sophie Shen, Tessa Werner, S. Lukowski, Stacey Andersen, Yuliangzi Sun, W. J. Shim, Dalia Mizikovsky, Sakurako Kobayashi, Jennifer Outhwaite, Han Sheng Chiu, Xiaoli Chen, G. Chapman, E. M. Martin, Di Xia, Duy Pham, Zezhuo Su, Daniel Kim, Pengyi Yang, Men Chee Tan, Enakshi Sinniah, Qiongyi Zhao, Sumedha Negi, Meredith A. Redd, Joseph E Powell, Sally L. Dunwoodie, Patrick P L Tam, Mikael Bodén, J. W. Ho, Quan H Nguyen, N. Palpant
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Breast cancer is the most common type of cancer in women and the second leading cause of death by cancer. Despite recent advances, the mortality rate remains high, underlining the need to develop new therapeutic approaches. The complex interaction between cancer cells and the tumor microenvironment (TME) is crucial in determining tumor progression, therapy response, and patient prognosis. Understanding the role of immune cells in carcinogenesis and tumor progression can help improve targeted therapeutic options, increasing the likelihood of a favorable prognosis. Therefore, this review aims to critically analyze the complex interaction between tumor cells and immune cells, emphasizing the clinical and therapeutic implications. Additionally, we explore advances in immunotherapies, with a focus on immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c335e51d434a9f10b3300d0d306ba2af8135f4e1" target='_blank'>
              Breast Cancer and Tumor Microenvironment: The Crucial Role of Immune Cells
              </a>
            </td>
          <td>
            Tânia Moura, P. Laranjeira, Olga Caramelo, Ana M. Gil, Artur Paiva
          </td>
          <td>2025-02-28</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57fc0173898db1615005d0ab09ce022fdfc322eb" target='_blank'>
              The Transcriptomic Landscapes of Appendiceal Primary and Metastatic Tumors are Distinct.
              </a>
            </td>
          <td>
            Margaret A. Park, Richard Jacobson, Maria Genilo-Delgado, Amir Mohammadi, Carlos M. Morán-Segura, Solomon Alhassan, Yukihiro Nakanishi, J. Permuth, I. Imanirad, S. Dineen
          </td>
          <td>2025-02-22</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 842


 Background:
 Advanced Bladder Cancer (BC) presents significant challenges due to its high incidence and limited treatment options. Most patients initially present with non-muscle invasive bladder cancer (NMIBC), managed with transurethral resection (TURBT) and Bacillus Calmette–Guerin (BCG) therapy. However, some cancers recur or develop resistance, progressing to muscle-invasive bladder cancer (MIBC), which is associated with high mortality and limited curative options. While radical cystectomy (RC) is the standard treatment for MIBC, it has considerable morbidity. Thus, there is a critical need for predictive tools to guide treatment decisions and novel therapeutic approaches. Immunotherapies show promise, but their efficacy remains constrained. Analyzing the immune infiltrate in BC may uncover mechanisms of resistance and identify new targets.
 Methods:
 We investigated the immune microenvironment in NMIBC and MIBC using high-dimensional single-cell analysis, including scRNAseq and Imaging Mass Cytometry (Cytomass). Tumor tissues were obtained from BC patients who underwent cystectomy or Trans urethral removal of bladder tumour (TURBT). We enrolled 7 NMIBC and 8 MIBC patients for scRNAseq, isolating CD45+ immune cells and CD45- epithelial and stromal cells using FACS. The samples were processed with 10X technology to reveal transcriptional landscapes. Additionally, we enrolled 5 NMIBC and 5 MIBC patients for Cytomass analysis, where tissue sections were snap-frozen, stained, and analyzed with the Hyperion system. Our primary endpoint was to identify differences in the composition of tumor and immune cells between NMIBC and MIBC. The study was funded by the Next Generation EU Plan - Piano Nazionale di Ripresa e Resilienza (PNRR-MAD-2022-12376756).
 Results:
 The cohort included 15 patients with clinically significant BC who underwent radical cystectomy or TURBT. scRNAseq analysis of patients revealed differences in immune cell abundance and activation states between NMIBC and MIBC samples, identifying distinct cell types and high intra-tumoral heterogeneity. Notable changes were observed tumor-infiltrating B cell lymphocytes, with different abundance in tumors depending on the cell subsets. Immature B cells were enriched in MIBC when compared to NMIBC, indicating a potential role in tumor progression. Cytomass analysis further highlighted modulations in immune cell activation and spatial localization between NMIBC and MIBC.
 Conclusions:
 Our findings demonstrate the utility of single-cell RNA sequencing in elucidating the immune landscape of BC and highlight significant immune changes between MIBC and NMIBC. This data enhances our understanding of immune composition and the role of specific immune subsets in BC, potentially informing clinical risk stratification and treatment decision-making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7de131a381559fb27b1d49711570f15a4787b67" target='_blank'>
              Immune profiling at the single-cell level to evaluate the immune microenvironment in bladder cancer: Insights from non-muscle invasive and muscle-invasive disease.
              </a>
            </td>
          <td>
            P. Avolio, Anna Garbin, Marta Pandini, R. Carriero, Paolo Kunderfranco, M. G. Elefante, P. Colombo, P. Casale, G. Lughezzani, N. Buffi, R. Contieri, Gianluca Basso, Diletta Di Mitri, M. Lazzeri
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560c8effcee249e562abe69a29bf3bcf7c2bff96" target='_blank'>
              Unravelling single-cell DNA replication timing dynamics using machine learning reveals heterogeneity in cancer progression
              </a>
            </td>
          <td>
            Joseph M. Josephides, Chun-Long Chen
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Locally-advanced rectal cancer (LARC) is traditionally treated with neoadjuvant chemoradiotherapy before surgery, however, response heterogeneity necessitates the development of novel treatment combinations. Immune checkpoint blockade (ICB) with or without radiotherapy (RT) elicits clinical benefit in various solid tumors, yet ICB’s effectiveness so far is limited to colorectal tumors with microsatellite instability and the role of the tumor microenvironment (TME) is unclear. We present a novel orthotopic immunocompetent mouse model of LARC, where combinations of RT and ICB are tested to elucidate the role of TME in the response to treatment. Methods: Autochthonous murine tumor-derived organoids expressing common driver mutations (Apcfl/fl; KrasG12D/+; p53fl/fl; Tgfbr1fl/fl: ‘AKPT’) were orthotopically injected into syngeneic C57Bl/6 immunocompetent mice. Tumor response to RT-ICB was studied on timepoint and aging cohorts of mice with tumors growing in different TME settings: (a) in the rectal submucosa of wildtype mice, (b) subcutaneously in wildtype mice, and (c) in the rectal submucosa of mice lacking the iCCR locus (CCR1, CCR2, CCR3, and CCR5: ‘iCCRKO ’), which inhibits inflammatory monocyte recruitment into the TME. Radiation was delivered as 15Gy in 3# (Mon-Wed-Fri) using a Small Animal Radiation Research Platform on day 13 following tumor engraftment for the orthotopic models and on day 7 for the subcutaneous. PD-1 inhibition (PD-1i) was administered intraperitoneally (10mg/kg twice weekly) 3 days pre-RT until endpoint. Mice in the timepoint experiments were sampled 5 days following RT, while aging mice were sampled at clinical endpoint or 100 days, whichever came first. Results: RT and PD-1i offered significant survival extension and tumor control in the orthotopic wildtype setting (a). Transcriptomic characterization at 5 days revealed that untreated and irradiated tumors harbored a stroma-rich microenvironment, low in immune cells, which PD-1i partly reversed. This phenotypic ‘switch’ was also evident in an aging cohort, suggesting that RT + PD-1i have long-lasting effects on the stroma. In the subcutaneous setting (b), RT significantly increased tumor control but PD-1 blockade offered little additional advantage. Interestingly, subcutaneous AKPT tumors had significantly more CD8+ cells at baseline than orthotopic ones, highlighting the inherent immune permissiveness of the subcutaneous TME. Finally, tumors growing in the iCCRKO setting (c) were more radio-sensitive, leading to 3 instances of tumor clearance: one in the RT group and two in the RT + PD-1i. Conclusion: Our findings demonstrate the synergistic potential of RT and PD-1i to sensitize an inherently radio-resistant orthotopic mouse model of LARC. Tumor location and inflammatory monocytes emerged as key regulators of the immune response to RT-ICB, which could have important implications for LARC modeling. Importantly, we highlight that TME components can facilitate treatment resistance and give rise to targets that can be therapeutically exploited.
 Citation Format: Lydia Melissourgou-Syka, Michael A. Gillespie, Lily V.S. Hillson, Kathryn Gilroy, Katrina Stevenson, Susanti Susanti, Tamsin R.M. Lannagan, Eoghan Mulholland, Philip D. Dunne, Joanne Edwards, Sean M. O'Cathail, Colin W. Steele, Owen J. Sansom, Campbell S.D. Roxburgh.Investigating the role of the tumor microenvironment in the rectal cancer response to preclinical radio-immunotherapy combinations.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P022">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b958d65c1fd0bd54499a71c321762f474a07901" target='_blank'>
              Abstract P022: Investigating the role of the tumor microenvironment in the rectal cancer response to preclinical radio-immunotherapy combinations
              </a>
            </td>
          <td>
            Lydia Melissourgou-Syka, Michael A Gillespie, L. Hillson, Kathryn Gilroy, Katrina Stevenson, Susanti Susanti, T. Lannagan, Eoghan Mulholland, Philip D. Dunne, J. Edwards, S. O'Cathail, Colin W. Steele, Owen J. Sansom, Campbell S D Roxburgh
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) presents the most challenging subtype of the disease due to its aggressive nature and lack of targeted therapy options. To identify regulators of TNBC, we conducted a genome-wide CRISPR-knockout screen in both 3D tumor spheroid and 2D cell culture models. The 3D spheroid model displayed unique potential in identifying putative tumor suppressors due to its closer mimicry of in vivo tumor growth conditions. Notably, the chromatin-remodeling SWI/SNF complex emerged as a potent suppressor of tumor spheroid growth. Specifically, loss of the SWI/SNF ATPase subunit SMARCA4 promoted tumor spheroid growth with reduced compactness and enhanced primary tumor growth and metastasis across multiple TNBC models. SMARCA4 was required for the transcription of Rho GTPase activating factor ARHGAP29 by enhancing DNA accessibility through direct binding to its promoter. SMARCA4 loss resulted in reduced ARHGAP29 levels and hyperactive RHOA signaling, subsequently disrupting cell adhesion, facilitating the formation of a loose spheroid structure in vitro, and enhancing breast cancer growth and metastasis in vivo. These results establish SMARCA4 and SWI/SNF as tumor suppressors of TNBC through suppression of RHOA activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7936be7822f9a0773dcd71150fe84ffc4ab837c5" target='_blank'>
              SMARCA4 Inhibits Breast Cancer Progression and Metastasis through RHOA Suppression.
              </a>
            </td>
          <td>
            Zheng Sun, Zhuo Li, Yong Wei, Lillian Xu, Xiang Hang, Yibin Kang
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Introduction Glioma is the most common primary malignant brain tumor. Despite advances in surgical techniques and treatment regimens, the therapeutic effects of glioma remain unsatisfactory. Immunotherapy has brought new hope to glioma patients, but its therapeutic outcomes are limited by the immunosuppressive nature of the tumor microenvironment (TME). This study aimed to reveal the subpopulations and functional characteristics of tumor-associated macrophages (TAMs) and explore the regulatory effects of chlorogenic acid (CHA) on the immune microenvironment, as well as its potential for clinical application. Methods In this study, CHA was used in model mice. ScRNA - seq analysis was conducted to elucidate the differentiation trajectories and functional characteristics of bone marrow - derived monomacrophages (BMDMs) and microglia. A PPI and molecular docking model were constructed using the target prediction database. A case of a patient treated with CHA was reviewed. Results CHA slowed tumor growth in model mice and extended the survival time of mice. It enhanced the antigen - presenting function of macrophages and T - cell immune activation - related gene expression, activated microglia through the JAK - STAT pathway, and improved the antitumor functions. The good affinity of CHA with STAT1 was confirmed. The patient treated with CHA survived for 5 years and 6 months, achieved partial remission (PR) after 9 months of treatment, and remained alive without any new symptoms or toxic side effects. Our study revealed the subtypes and differentiation trajectories of TAMs. CHA significantly improved the immune microenvironment of glioma by modulating the function of BMDMs and microglia. Discussion This study may provide new insights into targeting the regulation of TME and offer theoretical and practical support for the clinical application of CHA. The results demonstrated the potential of CHA in improving the immune microenvironment and antitumor effects, which could have implications for future glioma treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43630a7cbb9932035e6e598a785d760c8dd3e625" target='_blank'>
              ScRNA-seq unveils the functional characteristics of glioma-associated macrophages and the regulatory effects of chlorogenic acid on the immune microenvironment—a study based on mouse models and clinical practice
              </a>
            </td>
          <td>
            Jiachen Wang, Shenglan Li, Yuxiao Chen, Jinyi Chen, Can Wang, Zhuang Kang, Mengqian Huang, Ze-Hao Cai, Yuxiang Fan, Yanjie Lan, Yumeng Yu, Ruijing Bai, Feng Chen, Jiandong Jiang, Wenbin Li
          </td>
          <td>2025-01-10</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cf01180ccde9b6fda123113aa98da6a623ebc73" target='_blank'>
              Human-correlated genetic models identify precision therapy for liver cancer.
              </a>
            </td>
          <td>
            M. Müller, S. May, Holly Hall, Timothy J. Kendall, Lynn McGarry, Lauriane Blukacz, Sandro Nuciforo, Anastasia Georgakopoulou, Thomas Jamieson, Narisa Phinichkusolchit, Sandeep Dhayade, Toshiyasu Suzuki, J. Huguet-Pradell, Ian R Powley, Leah Officer-Jones, R. Pennie, R. Esteban-Fabró, Albert Gris-Oliver, R. Pinyol, George L Skalka, J. Leslie, Matthew Hoare, Joep Sprangers, Gaurav Malviya, Agata Mackintosh, Emma Johnson, M. McCain, John Halpin, Christos Kiourtis, Colin Nixon, Graeme Clark, W. Clark, R. Shaw, A. Hedley, Thomas M Drake, Ee Hong Tan, Matt Neilson, Daniel J Murphy, David Y Lewis, Helen L Reeves, John Le Quesne, Derek A Mann, Leo M. Carlin, Karen Blyth, J. M. Llovet, Markus H Heim, Owen J. Sansom, Crispin J Miller, Thomas G Bird
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="RAC1, a member of the Rho family GTPases, has been implicated in various cancers, yet its pan-cancer landscape and role in the tumor immune microenvironment remain underexplored. This study presents a comprehensive analysis of RAC1 across 33 cancer types, revealing its high expression in a broad range of cancers and its association with poor prognosis. RAC1 expression correlates with genomic alterations, including CNVs, TMB, and MSI. RAC1 knockdown reduces cell proliferation and metastasis in breast and lung cancer cells, suggesting its oncogenic potential. Notably, RAC1 is negatively correlated with B cell infiltration, indicating its role in regulating the immune microenvironment. Functional enrichment analysis showed that high RAC1 expression is linked to lower enrichment in B cell activation and immune response pathways. Single-cell transcriptome analysis identified RAC1 expression primarily in epithelial cells, associated with tumor progression, and spatial transcriptome analysis showed a mutually exclusive co-localization between B cell infiltration regions and RAC1-expressing epithelial cells. Based on RAC1 expression and B cell interaction, a prognostic signature was established to predict prognosis at the pan-cancer level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89761b22c5b23dbd6e738b6e2e4c97034f65fc05" target='_blank'>
              Comprehensive Pan-Cancer Analysis of RAC1 Decoding the Impact on Cancer Prognosis and B cell immune Regulation
              </a>
            </td>
          <td>
            Zhibo Shao, Hengyu Ren, Xuliren Wang, Qi Zhang, J. Xue, Y. Chi, B. Xiu, Jiong Wu
          </td>
          <td>2025-01-13</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is characterized by expanding blood cell clones carrying somatic mutations in healthy aged individuals and is associated with various age-related diseases and all-cause mortality. While CH mutations affect diverse genes associated with myeloid malignancies, their mechanisms of expansion and disease associations remain poorly understood. We investigate the relationship between clonal fitness and clinical outcomes by integrating data from three longitudinal aging cohorts (n=713, observations=2,341). We demonstrate pathway-specific fitness advantage and clonal composition influence clonal dynamics. Further, the timing of mutation acquisition is necessary to determine the extent of clonal expansion reached during the host individual’s lifetime. We introduce MACS120, a metric combining mutation context, timing, and variant fitness to predict future clonal growth, outperforming traditional variant allele frequency measurements in predicting clinical outcomes. Our unified analytical framework enables standardized clonal dynamics inference across cohorts, advancing our ability to predict and potentially intervene in CH-related pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df776ea20c82ef08bfb6dc73e82dc94bd703a7e7" target='_blank'>
              Clinical progression of clonal hematopoiesis is determined by a combination of mutation timing, fitness, and clonal structure
              </a>
            </td>
          <td>
            Eric Latorre-Crespo, Neil A. Robertson, E. Kosebent, Louise Macgillivray, Lee Murphy, M. Uddin, Eric Whitsel, Michael C. Honigberg, Alex Bick, A. Reiner, Valeria Orru, Michele Marongiu, Francesco Cucca, E. Fiorillo, I. Deary, Sarah Harris, Simon Cox, Riccardo Marioni, Linus Schumacher, Tamir Chandra, Kristina Kirschner
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Metastasis accounts for the overwhelming majority of cancer deaths. In prostate cancer and many other solid tumors, progression to metastasis is associated with drastically reduced survival outcomes, yet the mechanisms behind this progression remain largely unknown. ATAD2 (ATPase family AAA domain containing 2) is an epigenetic reader of acetylated histones that is overexpressed in multiple cancer types and usually associated with poor patient outcomes. However, the functional role of ATAD2 in cancer progression and metastasis has been relatively understudied. Here we employ genetically engineered mouse models of prostate cancer bone metastasis, as well as multiple independent human cohorts, to show that ATAD2 is highly enriched in bone metastasis compared to primary tumors and significantly associated with the development of metastasis. We show that ATAD2 expression is associated with MYC pathway activation in patient datasets and that, at least in a subset of tumors, MYC and ATAD2 can regulate each other's expression. Using functional studies on mouse bone metastatic cell lines and innovative organ-on-a-chip bone invasion assays, we establish a functional role for ATAD2 inhibition in diminishing prostate cancer metastasis and growth in bone. Implications: Our study highlights ATAD2 as a driver of prostate cancer progression and metastasis and suggests it may constitute a promising novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c1e4bc107eff35ffcc4410b1199b818c7c8a640" target='_blank'>
              ATAD2 Drives Prostate Cancer Progression to Metastasis.
              </a>
            </td>
          <td>
            Anindita Dutta, A. Rodriguez-Calero, Kacey Ronaldson-Bouchard, Anne Offermann, Daoud Rahman, Twinkle Bapuji Vhatkar, Dan Hasson, M. Alshalalfa, E. Davicioni, R. Karnes, Mark A. Rubin, G. Vunjak‐Novakovic, Cory Abate-Shen, J. M. Arriaga
          </td>
          <td>2025-02-05</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="
 Immune checkpoint blockade (ICB) therapy has revolutionized cancer treatment, delivering impressive results across various cancers. However, resistance is common, and only a limited number of patients experience lasting responses. This underscores the urgent need to understand resistance mechanisms and develop new strategies to enhance responsiveness. Here we focus on breast cancer, particularly ER+ breast cancer, which is generally poorly immunogenic and responds poorly to ICB. To address this, we analyzed the METABRIC cohort's immunogenic biomarkers and tumor mutational burden (TMB). We found that relapsed ER+ patients treated with endocrine therapy exhibit activated immune pathways. However, high TMB correlates with lower overall survival (OS), suggesting an active but clinically ineffective immune response. To explore the immune-resistant mechanisms, we developed chronically tamoxifen-treated T47D and MCF7 cells (cTMX-T47D and cTMX-MCF7) by exposing them to the drug for over six months. RNA sequencing data revealed increased type I interferon (IFN-I) signaling and innate immunity in these cells, confirmed in vivo in T47D xenografts. Tracing tamoxifen’s effects, we found that IFN-I signaling and ISG induction depend on the cGAS/STING pathway. Short-term tamoxifen exposure induced only modest ISG expression, indicating additional prerequisites during long-term treatment. We identified that the RACK7/KDM5C H3K4 demethylase complex is crucial for tamoxifen-induced ISG upregulation, with both proteins decreasing over time. Knocking down RACK7 intensified tamoxifen's activation of the cGAS/STING signaling and ISG expression. Combined tamoxifen treatment and RACK7 knockdown significantly enhanced these innate immunity effects compared to either treatment alone. Furthermore, tamoxifen’s DNA-damaging ability modulates the tumor immune microenvironment, particularly through mitochondrial DNA (mtDNA)-induced STING upregulation. We discovered key tamoxifen-mediated mechanisms for cGAS/STING-IFN-I pathway activation via STING upregulation, involving the tamoxifen–mtDNA–STING and tamoxifen–RACK7/KDM5C–STING axes. While IFN-I signaling can promote anti-tumor immunity, our tumor/T-cell co-culture experiments revealed that tamoxifen inactivates T-cells through tumor-intrinsic mechanisms, likely involving T-cell exhaustion pathways. We investigated tamoxifen’s effects on immune checkpoint molecules, finding that CEACAM1, the primary ligand for TIM3, was upregulated in cTMX-T47D and cTMX-MCF7 cells. Further analysis of tamoxifen-treated T47D xenografts revealed elevated CEACAM1 levels, suggesting an immunosuppressive mechanism via the CEACAM1-TIM3 pathway. To investigate this axis in vivo, we used syngeneic tumor mouse models with TS/A (ER+) cells in BALB/c mice. Tamoxifen induced CEACAM1, increased CD45+ immune cell infiltration, and caused TIM3+ CD8 T-cell exhaustion, contributing to tumor progression. Blocking the CEACAM1-TIM3 interaction with an anti-TIM3 antibody significantly inhibited tumor growth, highlighting its role in tamoxifen resistance.
 Citation Format: Lan-Hsin Wang, Marvin Angelo E Aberin, Saurabh Singh, Yi-Fu Wang, Ya-Ting Lu, Joy Khag, Weng-Lang Wang, Yao-Ming Chang, Shih-Yu Chen, Chandan Guha, Shu-Ping Wang. Enhancing breast cancer immunotherapy by modulating the cGAS–STING and CEACAM1–TIM3 pathways in the tumor microenvironment [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab23d6c01d007b2032b2ae13c6264807a485ddad" target='_blank'>
              Abstract A066: Enhancing breast cancer immunotherapy by modulating the cGAS–STING and CEACAM1–TIM3 pathways in the tumor microenvironment
              </a>
            </td>
          <td>
            Lan-Hsin Wang, M. A. Aberin, Saurabh Singh, Yi-Fu Wang, Ya-Ting Lu, Joy Khag, Weng-Lang Wang, Yao-Ming Chang, Shih-Yu Chen, Chandan Guha, Shu-Ping Wang
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="During the development of cancer, many changes leading to a series of genetic mutations that cannot control tumor growth. Therefore, understanding the complex interactions between the intrinsic, extrinsic, and systemic mediators of tumor cells and disease progression is crucial for the rational development of effective anti-cancer treatments. In this study, we conducted a multi-omics pan-cancer exploration of the function of JMJD4 using various comprehensive analysis tools and samples collected from relevant cancer databases, to understand how JMJD4 affects pan-cancer development by identifying its clinical features and multi-omics heterogeneity. We discovered that JMJD4 was highly expressed and associated with cancer stemness, epigenetic modulations and DNA repair in pan-cancer, especially in BLCA, BRCA, CHOL, COAD, ESCA. JMJD4 was also related to most immune-infiltrating cells, which indicated that JMJD4 may serve as a diagnostic and immunological biomarker in pan-cancer. Therefore, JMJD4 can be used as an indicator and a new therapeutic target for early tumor diagnosis and a prognostic marker for many types of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f74f88772da2d32d539504f99797675a6a54aee" target='_blank'>
              JMJD4 is a potential diagnostic and immunological biomarker in pan-cancer
              </a>
            </td>
          <td>
            Qinyi Li
          </td>
          <td>2025-03-03</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) has a high incidence of perineural invasion (PNI), a pathological feature of the cancer invasion of nerves. PNI is associated with a poor prognosis, local recurrence and cancer pain. It has been suggested that interactions between nerves and the tumor microenvironment (TME) play a role in PDAC tumorigenesis. Methods: Here, we used Nanostring GeoMx Digital Spatial Profiler to analyze the whole transcriptome of both cancer and nerve cells in the microenvironment of PNI and non-PNI foci from 13 PDAC patients. Conclusions: We identified previously reported pathways involved in PNI, including Axonal Guidance and ROBO-SLIT Signaling. Spatial transcriptomics highlighted the role of PNI foci in influencing the immune landscape of the TME and similarities between PNI and nerve injury response. This study revealed that endocannabinoid and polyamine metabolism may contribute to PNI, cancer growth and cancer pain. Key members of these pathways can be targeted, offering potential novel research avenues for exploring new cancer treatment and/or pain management options in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1818d7eac1f6857a9dc4b88e4ee2f868f4f36db6" target='_blank'>
              Spatial Transcriptomics Reveals Novel Mechanisms Involved in Perineural Invasion in Pancreatic Ductal Adenocarcinomas
              </a>
            </td>
          <td>
            V. Lakis, Noni L Chan, Ruth Lyons, Nicola Blackburn, Tam Hong Nguyen, Crystal Chang, Andrew Masel, Nicholas West, Glen M. Boyle, A. Patch, Anthony J. Gill, K. Nones
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Identifying the genes capable of driving tumorigenesis in different tissues is one of the central goals of cancer genomics. Computational methods that exploit different signals of positive selection in the pattern of mutations observed in genes across tumors have been developed to this end. One such signal of positive selection is clustering of mutations in areas of the three-dimensional (3D) structure of the protein above the expectation under neutrality. Methods that exploit this signal have been hindered by the paucity in proteins with experimentally solved 3D structures covering their entire sequence. Here, we present Oncodrive3D, a computational method that by exploiting AlphaFold 2 structural models extends the identification of proteins with significant mutational 3D clusters to the entire human proteome. Oncodrive3D shows sensitivity and specificity on par with state-of-the-art cancer driver gene identification methods exploiting mutational clustering, and clearly outperforms them in computational efficiency. We demonstrate, through several examples, how significant mutational 3D clusters identified by Oncodrive3D in different known or potential cancer driver genes can reveal details about the mechanism of tumorigenesis in different cancer types and in clonal hematopoiesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b62083d3dd11f079792a9e41da3af07b399c20f" target='_blank'>
              Oncodrive3D: Fast and accurate detection of structural clusters of somatic mutations under positive selection
              </a>
            </td>
          <td>
            Stefano Pellegrini, Olivia Dove Estrella, Ferran Muiños, Núria López-Bigas, A. González-Pérez
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="In hypermutant cancers, both the mutation rate per cell division and the genomic locations of new mutations are vastly altered. Such cancers often display atypical driver gene variants, which may provide sub-optimal oncogenic advantages. Why such weaker cancer drivers prevail in these tumours is unclear. Here, we examine the null hypothesis that aberrant mutational processes alone can account for the detection of weaker, rather than canonical, driver mutations. Using a mathematical modelling analysis, we find that simply increasing the mutation rate per-division can lead to the detection of weaker drivers, an effect which is further compounded by the biassing of mutations towards weaker driver sequence contexts. Focusing on POLE-mutant (DNA polymerase epsilon proofreading-deficient) colorectal cancers and KRAS driver mutations, we quantify the mutational bias towards non-canonical KRAS drivers in both POLE-mutant and non-hypermutant colorectal cancers. The observed bias, coupled with the increased mutation rate of POLE-mutant cancers, is sufficient to explain the enrichment of atypical KRAS drivers in POLE-mutant cancers with plausible selection parameters. Thus, within our model, differential selection between these cancer types need not be invoked to understand the observed variation in driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7891a3a8df530bd356f3a5da9253356957c1253d" target='_blank'>
              Hyper-mutational processes provide a head-start for non-optimal cancer driver mutations explaining atypical KRAS variants
              </a>
            </td>
          <td>
            Michael D. Nicholson, Ian Tomlinson
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ebe8d9c8be4bf1f0e95650c1d70b057ab95832c" target='_blank'>
              Comprehensive analysis of targetable mutations and tumor microenvironment in urachal cancer
              </a>
            </td>
          <td>
            D.J. Benjamin, T. Adeyelu, A. Elliott, S. Darabi, Thomas Lee, Rana R McKay, Matt Oberley, A. Rezazadeh Kalebasty
          </td>
          <td>2025-01-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a critical role in the development and treatment of liver cancer, which ranks sixth in incidence and third in mortality worldwide, according to the “Global Cancer Statistics 2022”. Hepatocellular carcinoma (HCC), the most common form of liver cancer, is heavily influenced by the TME, which affects tumor growth, invasion, metastasis, and the response to various treatments. Despite advancements in surgery, liver transplantation, targeted therapies, and immunotherapy, the complexity of the TME often limits treatment efficacy, especially in advanced-stage HCC cases. The TME consists of a dynamic interaction between tumor cells, immune cells, fibroblasts, blood vessels, and signaling molecules, all of which contribute to cancer progression and therapy resistance. Assessing the HCC TME is essential for designing effective, personalized treatments and improving patient outcomes. Recent research highlights the value of imaging technologies as non-invasive tools to evaluate the TME, offering new possibilities for more targeted therapies and better prognosis monitoring in HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d791404532d34590a2283ab759599b07294fd96b" target='_blank'>
              Advances in imaging techniques for tumor microenvironment evaluation in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Li-Li Wang, Fa-Chang Zhang, Han-Xin Xu, Dian-Dian Deng, Bing-Jie Ren, Qi Tan, Ya-Xin Liu, Wen-Hui Zhao, Jia-Le Lu
          </td>
          <td>2025-03-14</td>
          <td>World Journal of Gastroenterology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
In the tumor microenvironment (TME), hypoxia stands as a significant factor that modulates immune responses, especially those driven by T cells. As T cell-based therapies often fail to work in solid tumors, this study aims to investigate the effects of hypoxia on T cell topo-distribution in the TME, gene expression association with T cell states, and clinical responses in melanoma.


METHODS
To generate detailed information on tumor oxygenation and T cell accessibility, we used mathematical modeling of human melanoma tissue microarrays that incorporate oxygen supply from vessels, intratumoral diffusion, and cellular uptake. We created tumor maps and derived plots showing the fraction of CD4 and CD8 T cells against the distance to the nearest vessel and oxygen pressure. To assess their function and transcriptional changes caused by hypoxia, effector T cells were generated and cultured under hypoxia (0.5% oxygen) or normoxia (21% oxygen). The T cell hypoxia-transcriptional signature was compared against datasets from msigDB, iATLAS (clinical trials of melanoma patients treated with immune checkpoint inhibitors (ICIs)), ORIEN AVATAR (real-world melanoma patients treated with ICIs), and a single-cell atlas of tumor-infiltrating lymphocytes.


RESULTS
We made three specific observations: (1) in melanoma T cells preferentially accumulated in oxygenated areas close to blood vessels (50-100 µm from the vasculature in the regions of high oxygen availability) but not in hypoxic areas far from blood vessels. (2) Our analysis confirmed that under hypoxia, T cell functions were significantly reduced compared with normoxic conditions and accompanied by a unique gene signature. Furthermore, this hypoxic gene signature was prevalent in resting and non-activated T cells. Notably and clinically relevant, the hypoxic T cell gene set was found to correlate with reduced overall survival and reduced progression-free survival in melanoma patients, which was more pronounced in non-responder patients undergoing ICI therapy. (3) Finally, compared with a single-cell atlas of tumor-infiltrating T cells, our hypoxia signature aligned with a population of cells at a state termed stress response state (TSTR).


CONCLUSIONS
Our study highlights the critical role of hypoxia in shaping T cell distribution and its correlation with clinical outcomes in melanoma. We revealed a preferential accumulation of T cells in oxygenated areas. Moreover, hypoxic T cells develop a distinct hypoxic gene signature prevalent in resting, non-activated T cells and TSTR that was also associated with poorer outcomes, particularly pronounced among non-responders to ICIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb776b4af2230af6f330d9f56e74c2d4b429defa" target='_blank'>
              Effector T cells under hypoxia have an altered transcriptome similar to tumor-stressed T cells found in non-responsive melanoma patients.
              </a>
            </td>
          <td>
            Mate Z Nagy, Lourdes B Plaza-Rojas, Justin C Boucher, Elena Kostenko, Anna L Austin, Ahmad A Tarhini, Zhihua Chen, Dongliang Du, A. M. E. Ojwang', Joshua Davis, Alyssa N. Obermayer, Katarzyna A. Rejniak, Timothy I. Shaw, J. Guevara-Patiño
          </td>
          <td>2025-02-01</td>
          <td>Journal for immunotherapy of cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Malignant glioma is a highly aggressive, therapeutically non-responsive, and deadly disease with a unique tumor microenvironment (TME). Of the 14 currently recognized and described cancer hallmarks, five are especially implicated in malignant glioma and targetable with repurposed drugs: cancer stem-like cells, in general, and glioma stem-like cells in particular (GSCs), vascularization and hypoxia, metabolic reprogramming, tumor-promoting inflammation and sustained proliferative signaling. Each hallmark drives malignant glioma development, both individually and through interactions with other hallmarks, in which the TME plays a critical role. To combat the aggressive malignant glioma spatio-temporal heterogeneity driven by TME interactions, and to overcome its therapeutic challenges, a combined treatment strategy including anticancer therapies, repurposed drugs and multimodal immunotherapy should be the aim for future treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6157232e731856fe0f1d9da825fa5b8ddff3cccc" target='_blank'>
              The Complexity of Malignant Glioma Treatment
              </a>
            </td>
          <td>
            L. Kampers, D. Metselaar, Maria Vinci, F. Scirocchi, Sophie E. M. Veldhuijzen van Zanten, Matthias Eyrich, V. Biassoni, Esther Hulleman, Michael Karremann, Wilfried Stücker, Stefaan W. Van Gool
          </td>
          <td>2025-03-04</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/441803ee55eb765dcf6eba9f2eb3ecd9ddb7b7b2" target='_blank'>
              Dissecting tumor cell programs through group biology estimation in clinical single-cell transcriptomics
              </a>
            </td>
          <td>
            S. Johri, Kevin Bi, Breanna M. Titchen, Jingxin Fu, J. Conway, Jett Crowdis, Natalie I. Vokes, Zenghua Fan, Lawrence Fong, Jihye Park, David Liu, M. X. He, E. V. Van Allen
          </td>
          <td>2025-03-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), the sixth most prevalent cancer globally, is characterized by high recurrence rates and poor prognosis. Investigating the heterogeneity of relapsed HCC and identifying key therapeutic targets may facilitate the design of effective anticancer therapies. In this study, integrative analysis of single-cell RNA sequencing data of primary and early-relapsed HCC revealed increased proportions of infiltrating CD8+ T cells along with malignant cells and a decrease in CD4+ T cells in relapsed HCC. Cellular interaction and immunohistochemical analysis proposed MIF-(CD74 + CXCR4) signaling pathway as a key mechanism by which malignant cells influence immune cells within the tumor microenvironment. Notably, primary malignant cells showed greater differentiation and proliferation potential, whereas relapsed cells exhibited enhanced epithelial-mesenchymal transition and inflammation, along with upregulated glycogen synthesis and metabolism-related gene expression. Using machine learning techniques on bulk RNA-seq data, we developed a relapsed tumor cell-related risk score (RTRS) that independently predicts overall and recurrence-free survival time with higher accuracy compared with conventional clinical variables. Prognostic biomarkers and potential therapeutic targets were validated via RT-qPCR using mouse implantation models. This comprehensive investigation elucidates the heterogeneity of relapsed HCC and constructs a novel postoperative recurrence prognostic model, paving the way for targeted therapies and improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1cb3b766dd5c2064ddcc362b2b05b2d16bf869" target='_blank'>
              Exploration of heterogeneity and recurrence signatures in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Wen-Jing Wu, Jianchao Wang, Fuqing Chen, Xuefeng Wang, Bin Lan, Ruyi Fu, Hong Wen, Fangfang Chen, Wei Hong, , Ying He, Gang Chen, Jianyin Zhou, Hai-Long Piao, Di Chen, Shuyong Lin
          </td>
          <td>2025-02-28</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In thyroid cancer, the tumor immune microenvironment (TIME) plays a crucial role in cancer development, progression and response to treatment. Like many other cancers, thyroid cancer creates a complex network of interactions with immune cells directly (cell-to-cell) and via humoral mediators (i.e., cytokines). This dynamic microenvironment undergoes constant modification, which can lead to changes in the immunophenotype that might explain cancer progression, dedifferentiation and resistance to treatment. According to the cancer immunoediting hypothesis, cancerous tumors can shape their immune microenvironment to create an immunosuppressive milieu that allows them to evade classic immune surveillance. One mechanism by which this occurs is through the reprogramming of immune cells, often shifting their phenotypes from cytotoxic to regulatory. Recent research has shed light on cellular components and molecular interactions within the thyroid cancer TIME. Immune cells such as Tumor-Associated Lymphocytes (TALs), myeloid-derived suppressor cells (MDSCs), Tumor-Associated Macrophages (TAMs) and Double-Negative (DN) T cells seem to play key roles in shaping the immune response to thyroid cancer. Additionally, cytokines, chemokines and other signaling molecules contribute to the communication and regulation of immune cells within that microenvironment. By studying these interactions, researchers aim to uncover not just potential therapeutic targets but also biomarkers of thyroid cancer that could provide clues on severity and progression. Based on that knowledge, strategies such as the use of immune checkpoint inhibitors, antigen-specific targeted immunotherapies, and immunomodulatory agents are being explored to enhance the anti-tumor immune response and overcome cancer immunosuppressive mechanisms. In this review, we analyze the available literature and provide our own experience to unravel the complexity of the thyroid immune microenvironment. Continued research in this area holds promise for improving outcomes through the identification of immune markers of severity/progression of thyroid cancer and the development of innovative immunotherapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1dc2e9312503c615a6271beb64dadd3e48dde07" target='_blank'>
              Thyroid Cancer—The Tumor Immune Microenvironment (TIME) over Time and Space
              </a>
            </td>
          <td>
            J. Jaume
          </td>
          <td>2025-02-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="When confronted with an undesired cell population, such as bacterial infections or tumors, we seek the most effective treatment, designed to eliminate the population as rapidly as possible. A common practice is to monitor the cells short-term response to the treatment, and from that, extrapolate the eventual treatment outcome, i.e. will it eradicate the cells, and if yes at what timescales. Underlying this approach is the assumption that the cells exhibit a homogeneous response to the treatment, and hence the early response patterns can be naturally extended to later times. Recent experiments on cancer cell populations, however, indicate a significant level of cellular heterogeneity, undermining this classic assessment protocol of treatment efficacy. We, therefore, develop here a stochastic framework, to analytically predict the temporal dynamics of a heterogeneous cell population. Quite often, we find, the average cellular parameters, governing the short-term response, fail to predict the actual treatment outcome. In contrast, our analysis, which also incorporates the populations variability, helps identify the relevant statistical parameters, which in turn, enable us to predict the full trajectory of the cell population, and specifically - the likelihood and typical timescales for remission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ef92fd76daed3e67a88fcf502b6cc330b90e29" target='_blank'>
              Efficient treatment of heterogeneous malignant cell populations
              </a>
            </td>
          <td>
            Uzi Harush, R. Straussman, B. Barzel
          </td>
          <td>2025-01-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 253


 Background:
 Neuroendocrine prostate cancer (NEPC) is an aggressive subtype of prostate cancer that can arise after androgen blockade. Early detection of NEPC has prognostic and therapeutic implications but remains challenging. Neuroendocrine (NE) transformation is spatially heterogeneous, with co-existing NE and adenocarcinoma (AD) components across a patient's disease burden, leading to underdiagnosis using tumor biopsy. Moreover, treatment response assessment with cross-sectional imaging cannot distinguish changes in co-existing NE and AD components. To address these challenges, we developed a non-invasive approach to monitor NE transformation in patients with prostate AD (PRAD) through profiling of circulating chromatin.
 Methods:
 Plasma samples were collected longitudinally from patients initially diagnosed with PRAD who later developed NEPC at the Dana-Farber Cancer Institute. Chromatin immunoprecipitation and sequencing on cell-free DNA (cfChIP-seq) was performed for H3K27ac, a histone modification enriched at active promoters and enhancers. To separately quantify circulating chromatin derived from NE and AD components in a given patient, we measured H3K27ac at NE-specific enhancers (NE-score) and androgen receptor binding sites (AD-score), respectively. Whole genome sequencing was performed to estimate the cfDNA tumor fraction (TF) using ichorCNA. To evaluate cfChIP-seq for monitoring NEPC treatment response, we calculated changes in NE-score and TF for each pair of consecutive plasma draws after NE transformation. We then compared these changes during intervals of disease progression, stability, or response using the Kruskal-Wallis test.
 Results:
 This study included 32 plasma samples from 5 patients, with 14 pre-NE and 18 post-NE diagnosis, collected between December 2016 and April 2022. The NE-score was significantly elevated in post-NE compared to pre-NE samples (Wilcoxon sum rank p<0.01). NE-scores between consecutive draws increased at times of disease progression, changed minimally with disease stability, and decreased during response (median change: 16 vs. 2.0 vs. -32%, resp.; p=0.035). Importantly, changes in TF, which reflects an aggregate of AD and NE-derived tumor DNA, were less pronounced (11 vs. -0.5 vs. -13%, resp.; p=0.3). The AD-score was positively correlated with PSA levels (R=0.59; p<0.001). In 2 patients with plasma collected within 3 months prior to NE transformation, NE scores increased prior to pathologic diagnosis of NEPC. Another case revealed an increase in AD-score and PSA accompanied by a decrease in NE-score when a patient stopped ADT and started platinum-based chemotherapy for NEPC.
 Conclusions:
 Epigenomic profiling of circulating chromatin may enable early detection of NEPC and monitoring of NE and AD components in metastatic prostate cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e3ddbcf8c9b48d1f4370cd6d55162da93823480" target='_blank'>
              Epigenomic profiling of circulating chromatin for early detection and monitoring of neuroendocrine prostate cancer.
              </a>
            </td>
          <td>
            Karl Semaan, R. Nawfal, Marc Eid, Travis Clark, R. El Hajj Chehade, J. Canniff, Noa Phillips, M. Machaalani, Paulo Cordeiro, R. Saliby, T. El Zarif, Katelyn Kuczmarski, G. M. Lee, Jacob E Berchuck, M. Pomerantz, Corrie A. Painter, J. C. Barrett, T. Choueiri, M. Freedman, S. Baca
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Triple negative breast cancers often contain higher numbers of tumour-infiltrating lymphocytes compared with other breast cancer subtypes, with their number correlating with prolonged survival. Since little is known about tumour-infiltrating lymphocyte trafficking in triple negative breast cancers, we investigated the relationship between tumour-infiltrating lymphocytes and the vascular compartment to better understand the immune tumour microenvironment in this aggressive cancer type. We aimed to identify mechanisms and signaling pathways responsible for immune cell trafficking in triple negative breast cancers, specifically of basal type, that could potentially be manipulated to change such tumours from immune “cold” to “hot” thereby increasing the likelihood of successful immunotherapy in this challenging patient population. We characterised the spatial immune environment in 10 basal breast cancers showing a range of tumour-infiltrating lymphocytes using multiplex fluorescent immunohistochemistry and quantitative digital analysis of CD3+ T cells. We examined their relationship to blood vessels and their activation status as defined by VCAM-1, ICAM-1 and PD-L1. Confirmation of the relationship between tumour-infiltrating lymphocytes and endothelial activation was performed through in silico analysis on TCGA BRCA RNA-seq data (N = 808). Significantly higher CD3+ T cell densities were observed in the stromal compartment compared with the neoplastic cell compartment (P = 0.003). ICAM-1 activated blood vessels were spatially associated with higher CD3+ T cell densities only within 30 microns of blood vessels compared with more distal activated and non-activated blood vessels (P = 0.041). In silico analysis confirmed higher numbers of tumour-infiltrating lymphocytes in basal breast cancers and that higher numbers were significantly associated with endothelial cell activation molecules, co-clustering with upregulated ICAM-1 and VCAM-1 amongst others. PD-L1 was also identified in a subset of blood vessels, suggesting an additional immune regulatory mechanism in endothelial cells. Regulating the activation status of tumour-associated vascular endothelial cells may improve T cell trafficking into basal breast tumours and enhance immunotherapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd5404cd864380509879a211020591331396cbc" target='_blank'>
              Preliminary characterisation of the spatial immune and vascular environment in triple negative basal breast carcinomas using multiplex fluorescent immunohistochemistry
              </a>
            </td>
          <td>
            E. Takano, Metta K Jana, Luis E Lara Gonzalez, J. Pang, R. Salgado, S. Loi, Stephen B Fox
          </td>
          <td>2025-01-10</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024cdb898f44026b61ab427711870c26e06ca884" target='_blank'>
              Identification of Cell Fate Determining Transcription Factors for Generating Brain Endothelial Cells.
              </a>
            </td>
          <td>
            Roya Ramezankhani, J. De Smedt, B. Toprakhisar, Bernard K. van der Veer, Tine Tricot, Gert Vanmarcke, Bradley Balaton, Leo A. van Grunsven, Dr. Massoud Vosough, Y. C. Chai, C. Verfaillie
          </td>
          <td>2025-01-24</td>
          <td>Stem cell reviews and reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Neuropilin 1 (NRP1) is upregulated in various types of malignant tumors, especially non-small-cell lung cancer (NSCLC). However, the precise mechanisms for membrane localization and regulation are not fully understood. Observations from super-resolution microscopy have revealed that NRP1 tends to form nanoscale clusters on the cell membrane, with these clusters varying significantly in size and density across different regions. Further research has shown that stimulation by hepatocyte growth factor (HGF) can reorganize the distribution of NRP1, reducing the number of small clusters while promoting the formation of larger ones. This suggests a propensity for internalization after activation. Additionally, dual-color dSTORM imaging has demonstrated a certain degree of colocalization between NRP1 and c-MET, indicating that c-MET plays an important role in stabilizing NRP1 clusters. This study provides new insights into the mechanism behind NRP1's clustered distribution on cell membranes and paves the way for developing more effective therapeutic strategies targeting NRP1 within tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c28ab5966a5ccc285c9e7f703f50de3ca761f9" target='_blank'>
              Super-Resolution Fluorescence Imaging Reveals the Mechanism of NRP1 Clustering on Non-Small-Cell Lung Cancer Membranes.
              </a>
            </td>
          <td>
            Meng Li, Jing Gao, Xiao Qi, Baofeng Li, Yinghao Zhao, Xiangyu Liu, Guangxin Zhang, Hongda Wang, Ti Tong
          </td>
          <td>2025-01-22</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Objective This study aimed to assess whether circulating tumor cells (CTCs) from colorectal cancer (CRC) could be used as an alternative to tissue samples for genetic mutation testing, overcoming the challenge of difficult tumor tissue acquisition. Methods We developed an immunolipid magnetic bead (IMB) system modified with antibodies against epithelial cell adhesion molecule (EpCAM) and vimentin to efficiently separate CTCs. We prepared EpCAM-modified IMBs (Ep-IMBs) and vimentin-modified IMBs (Vi-IMBs). The separation efficiency of the system was evaluated via in vitro experiments and by capturing and counting CTCs in blood samples from 23 CRC patients and 20 healthy controls. Hotspot mutations in patient tissue samples were identified via next-generation sequencing (NGS), whereas mutations in blood CTCs were detected via Sanger sequencing. The concordance between hotspot mutations in tumor tissue and blood CTCs was analyzed. Results The CTC sorting system exhibited good dispersion, stability, and low cytotoxicity, with a specificity of 90.54% and a sensitivity of 89.07%. CRC patients had an average of 8.39 CTCs per 7.5 mL of blood, whereas healthy controls had 0.09 per 7.5 mL of blood. The consistency of gene mutations was as follows: TP53 (91.31%), PIK3CA (76.00%), KRAS (85.36%), BRAF (51.00%), APC (65.67%), and EGFR (74.00%), with an overall gene mutation consistency of 85.06%. Conclusion Our CTC sorting system, which is based on Ep-IMBs and Vi-IMBs, effectively captures CTCs in the peripheral blood of CRC patients and enables clinical hotspot gene mutation testing via these enriched CTCs. This system partially solves the problem of difficult tumor tissue sample collection and provides a reference for gene mutation testing in early diagnosis, therapeutic efficacy evaluation, prognosis assessment, and minimal metastasis detection in CRC patients, showing significant potential for clinical application, especially in targeted therapy gene testing for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a3d11255d6fbc0540709b8effa5a11e3e8ad8d1" target='_blank'>
              Immunolipid magnetic bead-based circulating tumor cell sorting: a novel approach for pathological staging of colorectal cancer
              </a>
            </td>
          <td>
            Qingyan Deng, Weidong Li, Yueming Huang, Haitao Wang, Xinhao Zhou, Zhifen Guan, Bohao Cheng, Yao Wang
          </td>
          <td>2025-01-24</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa72f1de2d7ad2ee27bb41187db8bcfae6ef0577" target='_blank'>
              The integrative genomic and functional immunological analyses of colorectal cancer initiating cells to modulate stemness properties and the susceptibility to immune responses
              </a>
            </td>
          <td>
            I. Tout, Salim Bougarn, Mohammed Toufiq, Neha Gopinath, Ola Hussein, Abbirami Sathappan, E. Chin-Smith, Fazulur Rehaman, Rebecca Mathew, L. Mathew, Kun Wang, Li Liu, Abdulrahman Salhab, Oleksandr Soloviov, Sara Tomei, W. Hasan, Sahar I. Da’as, Yosra Bejaoui, Nady El Hajj, Karama Makni Maalej, Said Dermime, K. Rasul, P. Dellabona, G. Casorati, A. Turdo, M. Todaro, Giorgio Stassi, S. Ferrone, Xinhui Wang, Cristina Maccalli
          </td>
          <td>2025-02-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>