<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-07 09:29:06 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Spatial cellular context is crucial in shaping intratumor heterogeneity. However, understanding how each tumor establishes its unique spatial landscape and what factors drive the landscape for tumor fitness remains significantly challenging. Here, we analyzed over 2 million cells from 50 tumor biospecimens using spatial single-cell imaging and single-cell RNA sequencing. We developed a deep learning-based strategy to spatially map tumor cell states and the architecture surrounding them, which we referred to as Spatial Dynamics Network (SDN). We found that different tumor cell states may be organized into distinct clusters, or ‘villages’, each supported by unique SDNs. Notably, tumor cell villages exhibited village-specific molecular co-dependencies between tumor cells and their microenvironment and were associated with patient outcomes. Perturbation of molecular co-dependencies via random spatial shuffling of the microenvironment resulted in destabilization of the corresponding villages. This study provides new insights into understanding tumor spatial landscape and its impact on tumor aggressiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d9a8ffa548a2f4ea069cb9bc5e6205adbb07d1" target='_blank'>
              Tumor cell villages define the co-dependency of tumor and microenvironment in liver cancer
              </a>
            </td>
          <td>
            Meng Liu, M. Hernandez, Darko Castven, Hsin-Pei Lee, Wenqi Wu, Limin Wang, M. Forgues, Jonathan M. Hernandez, Jens U. Marquardt, Lichun Ma
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a crucial role in orchestrating tumor cell behavior and cancer progression. Recent advances in spatial profiling technologies have uncovered novel spatial signatures, including univariate distribution patterns, bivariate spatial relationships, and higher-order structures. These signatures have the potential to revolutionize tumor mechanism and treatment. In this review, we summarize the current state of spatial signature research, highlighting computational methods to uncover spatially relevant biological significance. We discuss the impact of these advances on fundamental cancer biology and translational research, address current challenges and future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed05a90f6b1d7a182ae84bb7e554725c91c44235" target='_blank'>
              Quantifying and interpreting biologically meaningful spatial signatures within tumor microenvironments
              </a>
            </td>
          <td>
            Si Jing, He-Qi Wang, Ping Lin, Jiao Yuan, Zhixuan Tang, Hong Li
          </td>
          <td>2025-03-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f15f8a11af43d80ae3f5bc50b4193451c3cfbd" target='_blank'>
              Mapping Tumor Microenvironment and Treatment Response of Diffuse Midline Glioma Using Multiplexed Immunofluorescence and AI Models
              </a>
            </td>
          <td>
            Sandra Laternser, A. D. De Micheli, Sarah Brüningk, E. McDonough, Antonela Petrovic, Elisabeth Jane Rushing, Christine Surrette, Julia Bugajska, A. Eze, Lindsay Kilburn, Luisa Machado, Susanne Dettwiler, Fabiola Prutek, Noelia Casares Lagar, D. de la Nava, Denise Morinigo, Fiona Ginty, Matthew D. Dun, Michael A. Grotzer, A. G. Stücklin, Sabine Mueller, R. Packer, M. Bornhorst, Marta María Alonso, Javad Nazarian
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy, recently approved by the FDA for melanoma, is an emerging modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors such as glioblastoma remains limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas using spectral flow cytometry, TCR sequencing, single-cell RNA-seq, Xenium in situ transcriptomics, and CODEX spatial proteomics. Comparative analysis of TIL-generating (TIL⁺) versus non-generating (TIL⁻) tumors revealed that IL7R expression, structured perivascular immune clustering, and tumor-intrinsic metabolic programs such as ACSS3 were associated with successful TIL expansion. In contrast, TIL⁻ tumors were enriched for neuronal lineage signatures, immunosuppressive transcripts including TOX and FERMT1, and tumor-connected macrophages. This study defines spatial and molecular correlates of TIL manufacturing success and establishes a genomics-enabled selection platform for adoptive T cell therapy. The profiling approach is now being prospectively implemented in the GIANT clinical trial (NCT06816927), supporting its translational relevance and scalability across glioblastoma and other immune-excluded cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5083118ad8ca69bf9d2b345a596f9d2646c08163" target='_blank'>
              A Spatial Multi-Omic Framework Identifies Gliomas Permissive to TIL Expansion
              </a>
            </td>
          <td>
            K. Hotchkiss, Kenan Zhang, Anna M. Corcoran, Elizabeth Owens, P. Noldner, C. Railton, Kyra Van Batavia, Ying Zhou, Jodie Jepson, Kirit Singh, R. McLendon, Kristen Batich, , Katayoun Ayasoufi, Michael Brown, Evan Calabrese, Jichun Xie, Jose R. Conejo-Garcia, B. Shaz, John W. Hickey, Mustafa Khasraw
          </td>
          <td>2025-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98801c9770ab85a801e0ca53d0db3280fc312fc" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma invasion into healthy brain tissue remains a major barrier to effective treatment, yet current models fail to capture its full complexity in a scalable and patient-specific manner. Here, we introduce GlioTrace, a novel ex vivo imaging and AI-based analytical framework that enables real-time, spatiotemporal tracking of glioblastoma invasion dynamics in patient-derived glioma cell culture xenograft (PDCX) brain slices. By integrating whole-specimen confocal microscopy, vascular counterstaining, and an advanced computational pipeline combining convolutional neural networks and Hidden Markov Models, GlioTrace identifies distinct invasion modes, including dynamic morphological switching, vessel-guided migration, and immune cell interactions and quantifies patient-specific variations in invasion plasticity. Using GlioTrace, we demonstrate that targeted therapies can selectively modulate invasion phenotypes, revealing spatially and temporally distinct drug responses. This scalable platform provides an unprecedented window into glioblastoma progression and treatment response, offering a powerful tool for precision oncology and anti-invasion therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d85bb045bd5b47db33273aef84595c6d869e5e" target='_blank'>
              Reconstructing the Single-Cell Spatiotemporal Dynamics of Glioblastoma Invasion
              </a>
            </td>
          <td>
            H. Mangukiya, Madeleine Skeppås, Soumi Kundu, Maria Berglund, Adam A. Malik, C. Krona, Sven Nelander
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R. Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Gliomas are among the most lethal cancers, with limited treatment options. To uncover hallmarks of therapeutic escape and tumor microenvironment (TME) evolution, we applied spatial proteomics, transcriptomics, and glycomics to 670 lesions from 310 adult and pediatric patients. Single-cell analysis shows high B7H3+ tumor cell prevalence in glioblastoma (GBM) and pleomorphic xanthoastrocytoma (PXA), while most gliomas, including pediatric cases, express targetable tumor antigens in less than 50% of tumor cells, potentially explaining trial failures. Longitudinal samples of isocitrate dehydrogenase (IDH)-mutant gliomas reveal recurrence driven by tumor–immune spatial reorganization, shifting from T-cell and vasculature-associated myeloid cell-enriched niches to microglia and CD206+ macrophage-dominated tumors. Multi-omic integration identified N-glycosylation as the best classifier of grade, while the immune transcriptome best predicted GBM survival. Provided as a community resource, this study opens new avenues for glioma targeting, classification, outcome prediction, and a baseline of TME composition across all stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08309318ca287b808eae53da393baf17b1d09b8" target='_blank'>
              Multi-omic landscape of human gliomas from diagnosis to treatment and recurrence
              </a>
            </td>
          <td>
            Hadeesha Piyadasa, Benjamin Oberlton, Mikaela Ribi, Jolene S Ranek, Inna Averbukh, K. Leow, M. Amouzgar, Candace C. Liu, Noah F. Greenwald, E. Mccaffrey, Rashmi Kumar, Selena Ferrian, Albert G. Tsai, Ferda Filiz, C. Fullaway, M. Bosse, Sricharan Reddy Varra, Alex Kong, Cameron Sowers, M. Gephart, Pablo Nuñez-Perez, EnJun Yang, Michael Travers, Michael Schachter, Samantha Liang, Maria R. Santi, Samantha Bucktrout, P. Gherardini, Kristina Cole, Michael Barish, Christine E. Brown, Derek A. Oldridge, Richard R. Drake, Joanna J. Phillips, Hideho Okada, Robert Prins, Sean C. Bendall, Michael Angelo
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), accounting for 90% of pancreatic neoplasms, is characterized by its poor prognosis, with a 5-year survival rate of only 12%. Most patients are diagnosed with metastatic or locally advanced disease, leaving only 15% eligible for curative resection. PDAC exhibits resistance to chemotherapy, targeted therapies, and immunotherapy, largely due to its highly heterogeneous tumor microenvironment (TME). In this study, we performed an integrative analysis of publicly available scRNA, spatial transcriptomics, and bulk RNA sequencing datasets to investigate the influence of TME composition and tumor architecture on PDAC progression, treatment response, and clinical outcomes. We identified TME subtypes with distinct cellular compositions, functional signatures, and immunomodulatory cell-cell interactions. Spatially distinct cellular niches and gene modules revealed heterogeneity across primary tumors and metastatic lesions. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications. These findings underscore the importance of integrating multi-omics approaches to unravel the complexity of the PDAC TME and highlight its potential to inform therapeutic strategies and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768379a4460d13ebf34f45a5af50c7d930ab8e2" target='_blank'>
              Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel F. Pozo de Mattos P., Marvin Paulo Lins, Julia Fontoura, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Adult-onset dermatomyositis (DM) is an autoimmune inflammatory myopathy with distinct cutaneous manifestations and a strong malignancy association. Through comparative analysis with cutaneous lupus erythematosus (CLE), our integrated spatial and single-cell transcriptomics analysis revealed unique immune and stromal niches associated with DM subtypes. Unexpectedly, we found an association between cancer-associated DM skin lesions and the presence of dispersed immune infiltrates enriched with macrophages, CD8+ T cells, plasma cells, and B cells with preserved vascular architecture. In contrast, non-cancer associated DM skin exhibited dense myeloid cell infiltrates, including neutrophils, monocytes, and macrophages, with elevated expression of IL1B and CXCL10 localized near injured vascular endothelia. Cytokines produced by these myeloid infiltrates together with local tissue hypoxia triggered dramatic stromal remodeling, leading to loss of vascular-associated fibroblasts. In addition to the CXCL10+ myeloid signature, non-cancer-associated DM skin with pDC presence showed the emergence of specific cellular pairs: PD-L1-expressing mregDCs and activated Tregs expressing NFKB2 and TNF receptors. While both DM and CLE showed strong interferon signatures, DM uniquely displayed IFN-β expression. Together, our study provides the first comprehensive spatial mapping of immune and stromal cells in adult-onset DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62db8748b2521d6e18e161c76b546fccb24b5fcd" target='_blank'>
              Spatial Transcriptomics Identifies Immune-Stromal Niches Associated with Cancer in Adult Dermatomyositis
              </a>
            </td>
          <td>
            Ksenia S. Anufrieva, Neda Shahriari, Ce Gao, Rochelle L. Castillo, Jessica Liu, Sean A Prell, Shideh Kazerounian, Khashayar Afshari, Anastasia N. Kazakova, Erin Theisen, Teresa Bowman, A. LaChance, Kimberly B. Hashemi, Ilya Korsunsky, Mehdi Rashighi, R. A. Vleugels, Kevin Wei
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In this study, we developed an integrated single cell transcriptomic (scRNAseq) atlas of human breast cancer (BC), the largest resource of its kind, totaling > 600,000 cells across 138 patients. Rigorous integration and annotation of publicly available scRNAseq data enabled a highly resolved characterization of epithelial, immune, and stromal heterogeneity within the tumor microenvironment (TME). Within the immune compartment we were able to characterize heterogeneity of CD4, CD8 T cells and macrophage subpopulations. Within the stromal compartment, subpopulations of endothelial cells (ECs) and cancer associated fibroblasts (CAFs) were resolved. Within the cancer epithelial compartment, we characterized the functional heterogeneity of cells across the axes of stemness, epithelial-mesenchymal plasticity, and canonical cancer pathways. Across all subpopulations observed in the TME, we performed a multi-resolution survival analysis to identify epithelial cell states and immune cell types which conferred a survival advantage in both The Cancer Genome Atlas (TCGA) and METABRIC. We also identified robust associations between TME composition and clinical phenotypes such as tumor subtype and grade that were not discernible when the analysis was limited to individual datasets, highlighting the need for atlas-based analyses. This atlas represents a valuable resource for further high-resolution analyses of TME heterogeneity within BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7db0b505659425c4bfdef92aba55ca31e10f8c9" target='_blank'>
              A highly resolved integrated transcriptomic atlas of human breast cancers
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, S. Monti
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Triple-negative breast cancer, characterized by aggressive growth and high intratumor heterogeneity, presents a significant clinical challenge. Here, we use a lineage-tracing system, ClonMapper, which couples heritable clonal identifying tags with single-cell RNA-sequencing (scRNA-seq), to better elucidate the response to doxorubicin in a model of TNBC. We demonstrate that, while there is a dose-dependent reduction in overall clonal diversity, there is no pre-existing resistance signature among surviving clones. Separately, we found the existence of two transcriptomically distinct clonal subpopulations that remain through the course of treatment. Among clones persisting across multiple samples we identified divergent phenotypes, suggesting a response to treament independent of clonal identity. Finally, a subset of clones harbor novel changes in expression following treatment. The clone and sample specific responses to treatment identified herein highlight the need for better personalized treatment strategies to overcome tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88360a99c9dba219540b793cdb094157ffa2ec6a" target='_blank'>
              Lineage Tracing Reveals Clone-Specific Responses to Doxorubicin in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Daylin Morgan, Andrea L. Gardner, Amy Brock
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74fb81e29120e8309bf4f3e49e31706525768da7" target='_blank'>
              Integrating multi-modal transcriptomics identifies cellular subtypes with distinct roles in PDAC progression
              </a>
            </td>
          <td>
            Jun Wu, Tenghui Dai, Ziyue Li, Meng Pan, Wei Zhang, Hao Chen, Guansheng Zheng, Li Qiao, Qizhou Lian, Yang Liu, Jierong Chen
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06a703b5eeba2e901b0564ff4995a24e3b880d79" target='_blank'>
              Single-cell spatial analysis of pediatric high-grade glioma reveals a novel population of SPP1+/GPNMB+ myeloid cells with immunosuppressive and tumor-promoting capabilities
              </a>
            </td>
          <td>
            Thijs J M van den Broek, R. Hoogendijk, M. Kranendonk, J. Lammers, Akshaya L. Krishnamoorthy, Ravian L. van Ineveld, Milo Molleson, Vasily O. Tsvetkov, F. Ringnalda, Marc van de Wetering, Yan Su, John I. Bianco, Cristian Ruiz Moreno, Mario G Ries, E. Hoving, J. van der Lugt, Leila Akkari, David P. Schrijver, H. Stunnenberg, A. Rios, D. V. van Vuurden, A. Zomer
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/190acfc8ea743ab5360d22a148d803b01bb31afc" target='_blank'>
              A pathogenic subpopulation of human glioma associated macrophages linked to glioma progression
              </a>
            </td>
          <td>
            Kenny Kwok Hei Yu, Z. Abou-Mrad, Kristof Törkenczy, Isabell Schulze, Jennifer Gantchev, Gerard Baquer, Kelsey Hopland, Evan D. Bander, Umberto Tosi, Cameron Brennan, Nelson S. Moss, Pierre-Jacques Hamard, R. Koche, Caleb Lareau, N. Agar, T. Merghoub, Viviane Tabar
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Abstract Background The tumor microenvironment (TME) is a complex and dynamic ecosystem that is known to influence responses to immunotherapy. We leveraged single-cell spatial transcriptomics to systematically dissect the intricate complexity of the TME, in particular the cellular heterogeneity and spatial interactions. Their collective impact on immunotherapy efficacy was studied in the context of a homogeneous group of patients with vulvar high-grade squamous intraepithelial lesions (vHSIL) treated with an immunotherapeutic tumor-specific peptide vaccine. Methods We performed single-cell spatial transcriptomics on 20 pretreatment vHSIL lesions, stratified by clinical response to immunotherapeutic vaccination into complete responders (CR), partial responders (PR) and non-responders (NR). Using a 1,000-gene panel, we mapped over 274,000 single cells in situ, identifying 18 cell clusters and 99 distinct non-epithelial cell states. Findings were validated against public single-cell transcriptomic data sets to assess their broader relevance across tumor types. Results Profound heterogeneity within the TME was detected across the response groups. CR lesions exhibited a higher ratio of immune-supportive to immune-suppressive cells—a pattern mirrored in other solid tumors following neoadjuvant checkpoint blockade. Key immune populations enriched in CRs included CD4+CD161+ effector T cells and chemotactic CD4+ and CD8+ T cells. Conversely, PRs were characterized by increased proportions of T helper 2 cells and CCL18-expressing macrophages, which are associated with the recruitment of type 2 T cells and regulatory T cells. NRs displayed preferential infiltration with immunosuppressive fibroblasts. Distinct spatial immune ecosystems further defined response groups. Although a number of immune cells were detected in all patients, type 1 effector cells dominated interactions in CRs, type 2 cells were prominently interacting in PRs, while NRs lacked organized immune cell interactions. Conclusions This study underscores the dual importance of both cellular composition and spatial organization in steering clinical response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b353470be584af7d95e65b2aa9a5dffe4241b32d" target='_blank'>
              Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy
              </a>
            </td>
          <td>
            Z. Abdulrahman, R. Slieker, Daniel McGuire, M. J. Welters, M. V. van Poelgeest, S. H. van der Burg
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80bfc8505face3c0cb655cd2772225105395f750" target='_blank'>
              Deep spatial proteomics of ovarian cancer precursor lesions delineates early disease changes and cell-of-origin signatures
              </a>
            </td>
          <td>
            Anuar Makhmut, M. Dragomir, Sonja Fritzsche, Markus Moebs, Wolfgang D. Schmitt, Eliane T Taube, Fabian Coscia
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors (TKIs) provide therapeutic benefit in EGFR mutation–positive non–small cell lung cancer, but some individuals develop early resistance. We performed spatial transcriptomics analysis of pre- and posttreatment tumor samples from the same patients to explore the underlying mechanisms of such early resistance. The proportion and activation of fibroblasts increased in association with the development of early resistance, whereas a distinct tumor cell cluster showed activation of tumor necrosis factor–α signaling via the NF-κB pathway even before treatment. Also in the early resistance sample, specific tumor cell clusters interacted with immune and stromal cells. Immature tertiary lymphoid structures (TLSs) were enriched in the early resistance sample, whereas mature TLSs were observed in the long-term response sample. These findings implicate tumor heterogeneity and an inflammatory tumor microenvironment in early EGFR-TKI resistance, providing insight into potential therapeutic strategies to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59cd540165dee7cba115de4f78c74d67fa9e26e" target='_blank'>
              Spatial dynamics of the tumor microenvironment associated with emerging resistance to targeted therapy in EGFR-mutated non–small cell lung cancer
              </a>
            </td>
          <td>
            Satoshi Nakamura, Daisuke Shibahara, Kentaro Tanaka, Yasuyuki Kishikawa, Mikiko Hashisako, K. Nakatomi, Noriaki Nakagaki, M. Kohno, Koichi Azuma, R. Ibusuki, K. Otsubo, Y. Yoneshima, E. Iwama, Yoshinao Oda, I. Okamoto
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5dfde3e4f7d39011a34eb5a718ac54f39aa6ee4" target='_blank'>
              Single cell spatial transcriptomics identifies coordinated cellular programs that are associated with good prognosis in microsatellite stable colorectal cancer
              </a>
            </td>
          <td>
            Victoria T. Karlsen, Umair Majid, Håvard T. Lindholm, S. U. Umu, Karoline Rapp Vander-Elst, A-C Røberg Beitnes, Marianne A. Merok, Kjersti Thorvaldsen Hagen, H. R. Nilsen, Sheraz Yaqub, E. Bækkevold, Diana Domanska, F. Jahnsen
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b6d1c2313c8e86b7cbcc29c94f8060fb9e7af4" target='_blank'>
              Abstract B048: Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy
              </a>
            </td>
          <td>
            Ruan FV Medrano, Vladimir Sukhov, Maxim N Artyomov, Robert Schreiber
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Spatially resolved transcriptomics (SRT) has transformed tissue biology by linking gene expression profiles with spatial information. However, sequencing-based SRT methods aggregate signals from multiple cell types within capture locations (“spots”), masking cell-type-specific gene expression patterns. Traditional cell-type deconvolution methods estimate cell compositions within spots but fail to resolve cell-type-specific gene expression, limiting their ability to uncover critical biological processes such as cellular interactions and microenvironmental dynamics. Here, we present STged (spatial transcriptomic gene expression deconvolution), a novel computational framework that goes beyond traditional deconvolution by reconstructing cell-type-specific gene expression profiles from mixed spots. STged integrates graph-based spatial correlations and reference-derived gene signatures using a non-negative least-squares regression framework, achieving precise and biologically meaningful deconvolution. Comprehensive simulations show that STged consistently outperforms existing methods in accuracy and robustness. Applications to human pancreatic ductal adenocarcinoma and human squamous cell carcinoma datasets reveal its capacity to identify microenvironment-specific highly variable genes, reconstruct spatial cell–cell communication networks, and resolve tissue architecture at near-single-cell resolution. In mouse kidney tissues, STged uncovers dynamic spatial gene expression patterns and distinct gene programs, advancing our understanding of tissue heterogeneity and cellular dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db33671170dacba5e74f384054d9b15e8f2088ef" target='_blank'>
              Precise gene expression deconvolution in spatial transcriptomics with STged
              </a>
            </td>
          <td>
            Jia-Juan Tu, Hong Yan, Xiao-Fei Zhang, Zhixiang Lin
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="High-grade gliomas (HGG) are incurable brain malignancies in children and adults. Breakthrough advances in transcriptomic technologies unveiled the intricate diversity of cellular states and their spatial organization within HGGs. We qualitatively integrated 55 neoplastic transcriptomic signatures described in 17 single-cell and spatial RNA sequencing-based studies. Our review delineates a spectrum of cellular states, represented by the expression of specific genes, which can be conceptualized along a “reactive-developmental programs” axis. Additionally, we discussed the potential cues influencing these cellular states, including how spatial organization may impact transcriptomic dynamics. Leveraging these insightful discoveries, we discussed a novel, evolutive way to integrate the different transcriptomic signatures in two or three dimensions, incorporating developmental states, their proliferative capacity, and their possible transition towards reactive states. This integrated analysis illuminates the diverse cellular landscape of HGGs and provides a valuable resource for further elucidation of malignant mechanisms, and for the design of therapeutic endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda3dd1ef6762a74a82f982637ae107d4fb74853" target='_blank'>
              An integrated perspective on single-cell and spatial transcriptomic signatures in high-grade gliomas
              </a>
            </td>
          <td>
            Célia Lemoine, Marc-Antoine Da Veiga, B. Rogister, Caroline Piette, Virginie Neirinckx
          </td>
          <td>2025-02-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98eeee496484cc85e8f46ccc055600cb6afe1b1e" target='_blank'>
              A harmonized ovarian cancer scRNA-seq atlas to dissect disease heterogeneity underlying metastatization and chemoresistance
              </a>
            </td>
          <td>
            M. Sallese, Vittorio Aiello, C. E. Villa, Pietro Lo Riso, Giuseppe Testa
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="NF2-related Schwannomatosis (NF2 SWN) is a rare disease characterised by the growth of multiple nervous system neoplasms, including bilateral vestibular schwannoma (VS). VS tumours are characterised by extensive leucocyte infiltration. However, the immunological landscape in VS and the spatial determinants within the tumour microenvironment that shape the trajectory of disease are presently unknown. In this study, to elucidate the complex immunological networks across VS, we performed imaging mass cytometry (IMC) on clinically annotated VS samples from NF2 SWN patients. We reveal the heterogeneity in neoplastic cell, myeloid cell and T cell populations that co-exist within VS, and that distinct myeloid cell and Schwann cell populations reside within varied spatial contextures across characteristic Antoni A and B histomorphic niches. Interestingly, T-cell populations co-localise with tumour-associated macrophages (TAMs) in Antoni A regions, seemingly limiting their ability to interact with tumorigenic Schwann cells. This spatial landscape is altered in Antoni B regions, where T-cell populations appear to interact with PD-L1+ Schwann cells. We also demonstrate that prior bevacizumab treatment (VEGF-A antagonist) preferentially reduces alternatively activated-like TAMs, whilst enhancing CD44 expression, in bevacizumab-treated tumours. Together, we describe niche-dependent modes of T-cell regulation in NF2 SWN VS, indicating the potential for microenvironment-altering therapies for VS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e66dea3c7770dc53e6070f848f2e765373a2eab3" target='_blank'>
              Spatial mapping of immune cell environments in NF2-related schwannomatosis vestibular schwannoma
              </a>
            </td>
          <td>
            Adam P. Jones, Michael J Haley, Miriam H Meadows, Grace E. Gregory, C. Hannan, Ana K Simmons, Leoma D. Bere, Daniel Lewis, Pedro Oliveira, Miriam J. Smith, Andrew T King, D. G. R. Evans, P. Paszek, David Brough, Omar N Pathmanaban, Kevin N. Couper
          </td>
          <td>2025-03-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73634c31cfc63813b66fedfb8ecff6a08ba232af" target='_blank'>
              Spatiotemporal organisation of residual disease in mouse and human BRCA1-deficient mammary tumours and breast cancer
              </a>
            </td>
          <td>
            Demeter Túrós, Morgane Decollogny, Astrid Chanfon, Myriam Siffert, L. Romanens, J. Tille, Olivier Tredan, I. Labidi-Galy, Alberto Valdeolivas, Sven Rottenberg
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly malignant tumor in women, characterized by high morbidity, mortality, and recurrence rates. Although surgical treatment, radiotherapy, and chemotherapy are the mainstays of current treatment methods, the high heterogeneity of TNBC results in unsatisfactory outcomes with low 5-year survival rates. Rapid advancements in omics technology have propelled the understanding of TNBC molecular biology. The emergence of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has significantly enhanced knowledge of tumor heterogeneity and the distribution, functionality, and intercellular interactions of various cell types within the tumor microenvironment, including tumor cells, T cells, B cells, macrophages, and fibroblasts. The present study provides an overview of the technical characteristics of scRNA-seq and ST, highlighting their applications in exploring TNBC heterogeneity, cell spatial distribution patterns, and intercellular interactions. This review aims to enhance the comprehension of TNBC at the cellular level for the development of effective therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db2a16a0771f525b51537059ff8768102022813" target='_blank'>
              Analysis of single-cell and spatial transcriptomics in TNBC cell-cell interactions
              </a>
            </td>
          <td>
            Yan Xin, Qiji Ma, Qiang Deng, Tielin Wang, Dongxu Wang, Gang Wang
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have transformed the treatment landscape across multiple cancer types achieving durable responses for a significant number of patients. Despite their success, many patients still fail to respond to ICIs or develop resistance soon after treatment. We sought to identify early treatment features associated with ICI outcome. We leveraged the MC38 syngeneic tumor model because it has variable response to ICI therapy driven by tumor intrinsic heterogeneity. ICI response was assessed based on the level of immune cell infiltration into the tumor – a well-established clinical hallmark of ICI response. We generated a spatial atlas of 48,636 transcriptome-wide spots across 16 tumors using spatial transcriptomics; given the tumors were difficult to profile, we developed an enhanced transcriptome capture protocol yielding high quality spatial data. In total, we identified 8 tumor cell subsets (e.g., proliferative, inflamed, and vascularized) and 4 stroma subsets (e.g., immune and fibroblast). Each tumor had orthogonal histology and bulk-RNA sequencing data, which served to validate and benchmark observations from the spatial data. Our spatial atlas revealed that increased tumor cell cholesterol regulation, synthesis, and transport were associated with a lack of ICI response. Conversely, inflammation and T cell infiltration were associated with response. We further leveraged spatially aware gene expression analysis, to demonstrate that high cholesterol synthesis by tumor cells was associated with cytotoxic CD8 T cell exclusion. Finally, we demonstrate that bulk RNA-sequencing was able to detect immune correlates of response but lacked the sensitivity to detect cholesterol synthesis as a feature of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5d48556d03df2b88d1c06ed57c2bca71cfaf9b2" target='_blank'>
              Spatial genomics reveals cholesterol metabolism as a key factor in colorectal cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Andrew J. Kavran, Yulong Bai, Brian Rabe, Anna Kreshock, Andrew Fisher, Yelena Cheng, Anne Lewin, Chao Dai, Matthew J. Meyer, Konstantinos J. Mavrakis, Anna Lyubetskaya, Eugene Drokhlyansky
          </td>
          <td>2025-03-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ca2d699ba974ef22ac49a1f2990095215e2dd0" target='_blank'>
              Bacterial population dynamics during colonization of solid tumors
              </a>
            </td>
          <td>
            Serkan Sayin, Motasem ElGamel, Brittany Rosener, Michael Brehm, Andrew Mugler, Amir Mitchell
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan A. Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35597888b2f30575949d9afcb81bdfc42dbe699b" target='_blank'>
              Multiscale Spatial Mapping of Microbial Communities for Biotherapeutic Development
              </a>
            </td>
          <td>
            Prateek Sehgal, Andrea G. Shaw, Kevin Cutler, Bridget Letson, Gregory T. Booth, Gaetano J. Scuderi, Anjali Doshi, Peter Diebold, Hannah Bronson, Maya Trisolini, Unnati Dharaiya, Kevin Keomanee-Dizon, Seyi Jung, Jennifer Bagin, James Berleman, Aditya Bhalla, Kyle Jacoby, Steve Kujawa, Jackson Mintz, P. Murugkar, Marguerite Prior, Keiramarie Robertson, Ashlie L. Burkart, Jason Pontin, Lee R. Swem, I. Vlaminck, , Philip Burnham, Hao Shi
          </td>
          <td>2025-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421024c849016285df42c8f390ac35abcedd51e" target='_blank'>
              Pan-cancer spatial characterization of key immune biomarkers in the tumor microenvironment
              </a>
            </td>
          <td>
            James R. Lindsay, Jennifer Altreuter, J. Alessi, Jason L. Weirather, Anita Giobbie-Hurder, Ian D Dryg, Katharina V. Hoebel, B. Sharma, K. Felt, F. S. Hodi, N. Lindeman, L. Sholl, E. Cerami, Jonathan A. Nowak, Mark M Awad, Scott J Rodig, William E. Lotter
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1db37f2f209f5470571ed3506376b29858f863c2" target='_blank'>
              Tumor cell heterogeneity drives spatial organization of the intratumoral immune response.
              </a>
            </td>
          <td>
            Miho Tanaka, Lotus Lum, Kenneth H Hu, Piyush Chaudhary, Savannah Hughes, Cecilia Ledezma-Soto, B. Samad, Daphne Superville, Kenneth Ng, Arun Chumber, Ciara Benson, Zoe N Adams, Kelly Kersten, Oscar A. Aguilar, Lawrence Fong, Alexis J. Combes, Matthew F. Krummel, Melissa Q Reeves
          </td>
          <td>2025-06-02</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3aa6e9e33a49a78ba8f7c54e58b7d68564b492" target='_blank'>
              Applications of Multiplex Immunohistochemistry in Evaluating Spatiotemporal Heterogeneity of T Cells
              </a>
            </td>
          <td>
            Mercedes Machuca-Ostos, Tim de Martines, K. Yoshimura, Junichi Mitsuda, Sumiyo Saburi, Alisa Kimura, Hiroki Morimoto, Koichi Yoshizawa, Nana Sakurai, Nanako Murakami, Kayo Kitamoto, Makoto Yasuda, Y. Sugiyama, H. Ogi, Saya Shibata, Aya Miyagawa-Hayashino, Eiichi Konishi, Kyoko Itoh, Takahiro Tsujikawa, Shigeru Hirano
          </td>
          <td>2025-02-17</td>
          <td>Immuno</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178582b0b4cac0c55049a52434a209bd8b7613e6" target='_blank'>
              Quantitative measurement of phenotype dynamics during cancer drug resistance evolution using genetic barcoding
              </a>
            </td>
          <td>
            F. J. Whiting, M. Mossner, C. Gabbutt, C. Kimberley, C. P. Barnes, A. Baker, Erika Yara, A. Sottoriva, Richard A. Nichols, Trevor A. Graham
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Intratumoral heterogeneity is the main cause of tumor treatment failure, varying across disease sites (spatial heterogeneity) and polyclonal properties of tumors that evolve over time (temporal heterogeneity). As our understanding of intratumoral heterogeneity, the formation of which is mainly related to the genomic instability, epigenetic modifications, plastic gene expression, and different microenvironments, plays a substantial role in drug-resistant as far as tumor metastasis and recurrence. Understanding the role of intratumoral heterogeneity, it becomes clear that a single therapeutic agent or regimen may only be effective for subsets of cells with certain features, but not for others. This necessitates a shift from our current, unchanging treatment approach to one that is tailored against the killing patterns of cancer cells in different clones. In this review, we discuss recent evidence concerning global perturbations of intratumoral heterogeneity, associations of specific intratumoral heterogeneity in lung cancer, the underlying mechanisms of intratumoral heterogeneity potentially leading to formation, and how it drives drug resistance. Our findings highlight the most up-to-date progress in intratumoral heterogeneity and its role in mediating tumor drug resistance, which could support the development of future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC), accounting for 30% of lung cancer cases, lacks adequate research due to limited understanding of its molecular abnormalities. Our study analyzed public LUSC datasets to explore the tumor microenvironment (TME) composition using scRNA-seq from two cohorts. Applying non-negative matrix factorization, we identified unique malignant cell phenotypes, or meta-programs (MPs), based on gene expression patterns. Survival analysis revealed the clinical relevance of these MPs. Findings illuminated a TME landscape enriched with immune cells—CD8 + T, exhausted T, CD4 + T, and naive T cells—and suggested roles for myeloid cells, like cDC1 and pDCs, in LUSC progression. Different MPs highlighted the heterogeneity of malignant cells and their clinical implications. Targeting MP-specific genes may enable personalized therapy, especially for early-stage LUSC. This study offers insights into immune cell function in tumor dynamics, identifies MPs, and paves the way for novel LUSC strategies, enhancing early intervention, personalized treatment, and prognosis, ultimately improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93916-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a302b866a51ae52b8b8d7352c3feab56f45e3d0e" target='_blank'>
              Single-cell RNA sequencing reveals potential therapeutic targets in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Junda Fan, Yu Chen, Yue Gong, Hongmei Sun, Rui Hou, Xiaoya Dou, Yanping Zhang, Cheng Huo
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Checkpoint inhibitors have led to remarkable benefits in non-small cell lung cancer (NSCLC), yet response rates remain below expectations. High-dimensional analysis and mechanistic experiments in clinical samples and relevant NSCLC models uncovered the immune composition of lung cancer tissues, providing invaluable insights into the functional properties of tumor-infiltrating T cells and myeloid cells. Among myeloid cells, type 1 conventional dendritic cells (cDC1s) stand out for their unique ability to induce effector CD8 T cells against neoantigens and coordinate antitumoral immunity. Notably, lung resident cDC1 are particularly abundant and long-lived and express a unique tissue-specific gene program, underscoring their central role in lung immunity. Here, we discuss recent insights on the induction and regulation of antitumoral T cell responses in lung cancer, separating it from the tissue-agnostic knowledge generated from heterogeneous tumor models. We focus on the most recent studies dissecting functional states and spatial distribution of lung cDC1 across tumor stages and their impact on T cell responses to neoantigens. Finally, we highlight relevant gaps and emerging strategies to harness lung cDC1 immunostimulatory potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2155ad4b9a79772729f8fb7af74e6020d7c7eb14" target='_blank'>
              Tissue-specific properties of type 1 dendritic cells in lung cancer: implications for immunotherapy
              </a>
            </td>
          <td>
            Lucía López Rodríguez, Roberto Amadio, G. M. Piperno, F. Benvenuti
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Accumulating evidence links pediatric cancers to prenatal transformation events, yet the influence of the developmental stage on oncogenesis remains elusive. We investigated how hematopoietic stem cell developmental stages affect leukemic transformation, disease progression, and therapy response using a novel, humanized model of NUP98::NSD1-driven pediatric acute myeloid leukemia, that is particularly aggressive with WT1 co-mutations. Fetal-derived hematopoietic stem cells readily transform into leukemia, and WT1 mutations further enhance stemness and alter lineage hierarchy. In contrast, stem cells from later developmental stages become progressively resistant to transformation. Single-cell analyses revealed that fetal-origin leukemia stem cells exhibit greater quiescence and reliance on oxidative phosphorylation than their postnatal counterparts. These differences drive distinct therapeutic responses, despite identical oncogenic mutations. In patients, onco-fetal transcriptional programs correlate with worse outcomes. By targeting key vulnerabilities of fetal-origin leukemia cells, we identified combination therapies that significantly reduce aggressiveness, highlighting the critical role of ontogeny in pediatric cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97e870a1822de5485cb1751fdbfdd64fadf250" target='_blank'>
              Ontogeny Dictates Oncogenic Potential, Lineage Hierarchy, and Therapy Response in Pediatric Leukemia
              </a>
            </td>
          <td>
            Ke Wang, Shayan Saniei, Nikita Poddar, Subrina Autar, Saul Carcamo, Meghana Sreenath, Jack H Peplinski, R. Ries, Isabella G Martinez, Clifford Chao, Anna Huo-Chang Mei, Noshin Rahman, Levan Mekerishvili, Miguel Quijada-álamo, Grace Freed, Mimi Zhang, Katherine Lachman, Zayna Diaz, Manuel M Gonzalez, Jing Zhang, Giang Pham, Dan Filipescu, Mirela Berisa, Tommaso Balestra, J. Reisz, A. D’Alessandro, Daniel J Puleston, Emily Bernstein, Jerry E. Chipuk, M. Wunderlich, S. Tasian, Bridget K. Marcellino, Ian A. Glass, Christopher M Sturgeon, D. Landau, Zhihong Chen, E. Papapetrou, Franco Izzo, S. Meshinchi, Dan Hasson, Elvin Wagenblast
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) are key components of the hepatocellular carcinoma (HCC) tumor microenvironment (TME). regulating tumor proliferation, metastasis, therapy resistance, immune evasion via diverse mechanisms. A deeper understanding of the l diversity of CAFs is essential for predicting patient prognosis and guiding treatment strategies. Methods We examined the diversity of CAFs in HCC by integrating single-cell, bulk, and spatial transcriptome analyses. Results Using a training cohort of 88 HCC single-cell RNA sequencing (scRNA-seq) samples and a validation cohort of 94 samples, encompassing over 1.2 million cells, we classified three fibroblast subpopulations in HCC: HLA-DRB1 + CAF, MMP11 + CAF, and VEGFA + CAF based on highly expressed genes of which, which are primarily located in normal tissue, tumor boundaries, and tumor interiors, respectively. Cell trajectory analysis revealed that VEGFA + CAFs are at the terminal stage of differentiation, which, notably, is tumor-specific. VEGFA + CAFs were significantly associated with patient survival, and the hypoxic microenvironment was found to be a major factor inducing VEGFA + CAFs. Through cellular communication with capillary endothelial cells (CapECs), VEGFA + CAFs promoted intra-tumoral angiogenesis, facilitating tumor progression and metastasis. Additionally, a machine learning model developed using high-expression genes from VEGFA + CAFs demonstrated high accuracy in predicting prognosis and sorafenib response in HCC patients. Conclusions We characterized three fibroblast subpopulations in HCC and revealed their distinct spatial distributions within the tumor. VEGFA + CAFs, which was induced by hypoxic TME, were associated with poorer prognosis, as they promote tumor angiogenesis through cellular communication with CapECs. Our findings provide novel insights and pave the way for individualized therapy in HCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06192-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2410faffb025161d800cf5841a3c4049fde9d25" target='_blank'>
              Integration of single-cell and spatial transcriptomics reveals fibroblast subtypes in hepatocellular carcinoma: spatial distribution, differentiation trajectories, and therapeutic potential
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-02-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1fe686f97fcb46c56e3f6bfbef8e78fcca9cb41" target='_blank'>
              Pericyte-tumor crosstalk facilitates metastatic tumor cell latency through PIEZO1-activated lysophospholipid transfer
              </a>
            </td>
          <td>
            Tamara McErlain, Morgan J Glass, Elizabeth C McCulla, Caitlin A Madden, Lauren E Ziemer, Kirsten E Overdahl, Alan K. Jarmusch, Michael J. Kruhlak, A. Chesler, Howard H Yang, Maxwell P Lee, Cristina M Branco, Meera Murgai
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew G Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background Human Immunodeficiency Virus (HIV) is one of the most prevalent viruses, causing significant immune depletion in affected individuals. Current treatments can control HIV and prolong patients’ lives, but new challenges have emerged. Increasing incidence of cancers occur in HIV patients. Esophageal squamous cell carcinoma (ESCC) is one of the most common cancers observed in HIV patients. However, the spatial cellular characteristics of HIV-related ESCC have not been explored, and the differences between HIV-ESCC and typical ESCC remain unclear. Methods We performed spatial transcriptome sequencing on HIV-ESCC samples to depict the microenvironment and employed cell communication analysis and multiplex immunofluorescence to investigate the molecular mechanism in HIV-ESCC. Results We found that HIV-ESCC exhibited a unique cellular composition, with fibroblasts and epithelial cells intermixed throughout the tumor tissue, lacking obvious spatial separation, while other cell types were sparse. Besides, HIV-ESCC exhibited an immune desert phenotype, characterized by a low degree of immune cell infiltration, with only a few SPP1+ macrophages showing immune resistance functions. Cell communication analysis and multiplex immunofluorescence staining revealed that tumor fibroblasts in HIV-ESCC interact with CD44+ epithelial cells via COL1A2, promoting the expression of PIK3R1 in epithelial cells. This interaction activates the PI3K-AKT signaling pathway, which contributes to the progression of HIV-ESCC. Conclusions Our findings depict the spatial microenvironment of HIV-ESCC and elucidate a molecular mechanism in the progression of HIV-ESCC. This will provide us insights into the molecular basis of HIV-ESCC and potential treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02248-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba81ba07e36306408c53ef4726b3e3e8f09de47b" target='_blank'>
              Spatial transcriptomic analysis of tumor microenvironment in esophageal squamous cell carcinoma with HIV infection
              </a>
            </td>
          <td>
            Yuanli Zuo, Yang Jin, Gang Li, Yue Ming, Ting Fan, Yitong Pan, Xiaojun Yao, Yong Peng
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Hepatoblastoma (HB) is a common paediatric liver malignancy characterised by significant intratumoral heterogeneity and a complex tumour microenvironment (TME). Using single‐cell RNA sequencing (scRNA‐seq), we analysed 43,592 cells from three tumour regions and adjacent normal tissue of an HB patient. Our study revealed distinct cellular compositions and varying degrees of malignancy across different tumour regions, with the T1 region showing the highest malignancy and overexpression of HMGB2 and TOP2A. Survival analysis demonstrated that high HMGB2 expression is associated with poor prognosis and increased recurrence, suggesting its potential as a prognostic marker. Additionally, we identified a diverse immune microenvironment enriched with regulatory T cells (Tregs) and CD8+ effector memory T cells (Tem), indicating potential immune evasion mechanisms. Notably, CTLA‐4 and PD‐1 were highly expressed in Tregs and Tem cells, highlighting their potential as immunotherapy targets. Myeloid cells, including Kupffer cells and dendritic cells, also exhibited distinct functional roles in different tumour regions. This study provides the first comprehensive single‐cell atlas of HB, revealing critical insights into its intratumoral heterogeneity and immune microenvironment. Our findings not only advance the understanding of HB biology but also offer new directions for precision medicine, including the development of targeted therapies and immunotherapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/634b11e52e203c006a13f961efc458d554db20a9" target='_blank'>
              Intratumoral Heterogeneity and Immune Microenvironment in Hepatoblastoma Revealed by Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Mingdi Ma, Chen Jin, Qian Dong
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Approximately 15-20% of head and neck cancer squamous cell carcinoma (HNSCC) patients respond favorably to immune checkpoint blockade (ICB). Previous single-cell RNA-Seq (scRNA-Seq) studies identified immune features, including macrophage subset ratios and T-cell subtypes, in HNSCC ICB response. However, the spatial features of HNSCC-infiltrated immune cells in response to ICB treatment need to be better characterized. Methods Here, we perform a systematic evaluation of cell interactions between immune cell types within the tumor microenvironment using spatial omics data using complementary techniques from both 10X Visium spot-based spatial transcriptomics and Nanostring CosMx single-cell spatial omics with RNA gene panel including 435 ligands and receptors. In this study, we used integrated bioinformatics analyses to identify cellular neighborhoods of co-localizing cell types in single-cell spatial transcriptomics and proteomics data. In addition, we used both publicly available scRNA-Seq and in-house spatial RNA-Seq data to identify spatially constrained Ligand-Receptor interactions in Responder patients. Results With 522,399 single cells profiled with both RNA and protein from 26 patients, in addition to spot-resolved spatial RNA-Seq from 8 patients treated with ICB together with bioinformatics analysis of publicly available single-cell and bulk RNA-Seq, we have identified a spatial and cell-type specific context-dependency of myeloid and T cell interaction difference between Responders and Non-Responders. We defined further cellular neighborhood and the sources of chemokine CXCL9/10-CXCR3 interactions in Responders, emerging targets in ICB, as well as CXCL16-CXCR6, CCL4/5-CCR5, and other underappreciated and potential markers and targets for ICB response in HNSCC. In addition, we have contributed a rich data resource of cell-cell Ligand Receptor interactions for the immunotherapy and HNSCC research community. Discussion Our work provides a comprehensive single-cell and spatial atlas of immune cell interactions that correlate with response to ICB in HNSCC. We showcase how integrating multiple technologies and bioinformatics approaches can provide new insights into potential immune-based biomarkers of ICB response. Our results suggested refining future studies using preclinical animal models in a more context-specific manner to elucidate potential underlying mechanisms that lead to improved ICB responses. What is already known on this topic Most cancer patients still do not experience clinical benefits from immune checkpoint blockade (ICB), necessitating the development of response biomarkers and new immunotherapeutic targets. What this study adds Here, we use integrated high-dimensional omics and bioinformatics approaches to identify immune cell-cell interaction markers associated with ICB response in patients with Head and neck squamous cell carcinoma. How this study might affect research, practice or policy We identified spatial and cell-type specificity of Ligand-Receptor interactions between myeloid and T cells in ICB Responder patients that may help inform further mechanistic studies and biomarker development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ece16324064b70206dfb8fd687bfc5db3089e8d7" target='_blank'>
              Integrated single-cell and spatial analysis identifies context-dependent myeloid-T cell interactions in head and neck cancer immune checkpoint blockade response
              </a>
            </td>
          <td>
            Athena E. Golfinos-Owens, Taja Ložar, Parth Khatri, Rong Hu, Paul M. Harari, Paul F. Lambert, Megan B Fitzpatrick, Huy Q. Dinh
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background The identification of the complex spatial architecture of immune cell infiltration and its interaction mechanisms within tumor ecosystems provides crucial insights into therapeutic responses to neoadjuvant therapy in muscle-invasive bladder cancer (MIBC). This study aims to characterize the spatial features of distinct cell-type niches within the tumor microenvironment (TME) of patients with varying responses to neoadjuvant therapy. Methods We performed spatial transcriptomic profiling on six MIBC specimens obtained from a registered clinical trial (ChiCTR2000032359), generating whole-transcriptome spatial atlases to map the TME architecture. High-throughput analytical frameworks were employed to deconstruct the TME, and key findings were validated through immunohistochemistry and mouse model experiments. Results Our analysis revealed that tissues from complete responders exhibited greater infiltration of T and B cells, with the formation of tertiary lymphoid structure (TLS). Trajectory analysis identified CCL19/CCL21 as the key signaling molecules driving TLS formation in MIBC. Mouse experiments demonstrated that recombinant CCL19/CCL21 protein injections promoted intratumoral TLS formation and enhance the efficacy of immunotherapy. Furthermore, we observed significant intrinsic heterogeneity within individual tumors, which may contribute to the lack of therapeutic efficacy in MIBC. Conclusions This study underscores the critical role of TLS formation in the response to neoadjuvant therapy in MIBC. We identified CCL19/CCL21 as key drivers of TLS formation within MIBC tumors and potential immune-sensitizing agents. Additionally, the intrinsic heterogeneity of tumor should be considered a significant factor influencing therapeutic efficacy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06358-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d2414e88dc899f0c1cdec00aa234fe0cea16d7" target='_blank'>
              Spatial relationships and interactions of immune cell niches are linked to the pathologic response of muscle-invasive bladder cancer to neoadjuvant therapy
              </a>
            </td>
          <td>
            Wasilijiang Wahafu, Quan Zhou, Xihua Yang, Yongming Yang, Yuanyuan Zhao, Zhu Wang, Xiangpeng Kang, Xiongjun Ye, N. Xing
          </td>
          <td>2025-03-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462e90afb96bda692d274dd22748c2e27453a930" target='_blank'>
              Diving Deep: Profiling Exhausted T Cells in the Tumor Microenvironment Using Spectral Flow Cytometry.
              </a>
            </td>
          <td>
            Karen W Teng, Weng Hua Khoo, Nicholas Ching Wei Ho, S. J. Yang, Douglas C Wilson, Edmond Chua, Shu Wen Samantha Ho
          </td>
          <td>2025-04-03</td>
          <td>Cytometry. Part A : the journal of the International Society for Analytical Cytology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0444f2eee8d5a4470978d8e6c693faf1ee9e687e" target='_blank'>
              A Phenotype-Driven Multi-Omic Atlas of Glioblastoma Invasion
              </a>
            </td>
          <td>
            C. Krona, Anders Sundström, Emil Rosén, Soumi Kundu, H. Mangukiya, H. Babačić, Irem Uppman, Madeleine Skeppås, Ida Larsson, Ludmila Elfineh, Xiaofang Cao, Rashmi Ramachandra, Ramy Elgendy, Katrin Rut Halldorsdottir, Zankruti Dave, Milena Doroszko, K. Olausson, Johan Wikström, M. Pernemalm, Sven Nelander
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1760a6c58270791a1de56692df43805b36e2d0" target='_blank'>
              Single Cell RNA Sequencing Analysis of Human Melanoma Reveals A Distinct Prognostic Myeloid Cell
              </a>
            </td>
          <td>
            M. U. Ahmad, Lyong Heo, Ramesh V. Nair, Saurabh Sharma, E. Maverakis, Amanda R. Kirane
          </td>
          <td>2025-03-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common malignant tumors of the digestive tract, with increasing incidence and mortality rates, posing a significant burden on human health. Its progression relies on various mechanisms, among which the tumor microenvironment and tumor-associated macrophages (TAMs) have garnered increasing attention. Macrophage infiltration in various solid tumors is associated with poor prognosis and is linked to chemotherapy resistance in many cancers. These significant biological behaviors depend on the heterogeneity of macrophages. Tumor-promoting TAMs comprise subpopulations characterized by distinct markers and unique transcriptional profiles, rendering them potential targets for anticancer therapies through either depletion or reprogramming from a pro-tumoral to an anti-tumoral state. Single-cell RNA sequencing technology has significantly enhanced our research resolution, breaking the traditional simplistic definitions of macrophage subtypes and deepening our understanding of the diversity within TAMs. However, a unified elucidation of the nomenclature and molecular characteristics associated with this diversity remains lacking. In this review, we assess the application of conventional macrophage polarization subtypes in colorectal malignancies and explore several unique subtypes defined from a single-cell omics perspective in recent years, categorizing them based on their potential functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff246fd6cb059fea5f5e9204ca830e7b6e029678" target='_blank'>
              Characterizing macrophage diversity in colorectal malignancies through single-cell genomics
              </a>
            </td>
          <td>
            Tingshuo Zhao, Yinyi Luo, Yuanjie Sun, Zhigang Wei
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancers display cellular, genetic and epigenetic heterogeneity, complicating disease modeling. Multiple cell states defined by gene expression have been described in lung adenocarcinoma (LUAD). However, the functional contributions of cell state and the regulatory programs that control chromatin and gene expression in the early stages of tumor initiation are not well understood. Using single-cell RNA and ATAC sequencing in Kras/p53-driven tumor organoids, we identified two major cellular states: one more closely resembling alveolar type 2 (AT2) cells (SPC-high), and the other with epithelial-mesenchymal-transition (EMT)-associated gene expression (Hmga2-high). Each state exhibited distinct transcription factor networks, with SPC-high cells associated with TFs regulating AT2 fate and Hmga2-high cells enriched in Wnt- and NFκB-related TFs. CD44 was identified as a marker for the Hmga2-high state, enabling functional comparison of the two populations. Organoid assays and orthotopic transplantation revealed that SPC-high, CD44-negative cells exhibited higher tumorigenic potential within the lung microenvironment. These findings highlight the utility of organoids in understanding chromatin regulation in early tumorigenesis and identifying novel early-stage therapeutic targets in Kras-driven LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729ab293566d42d02b2188c02492d9921394583a" target='_blank'>
              Organoid modeling reveals the tumorigenic potential of the alveolar progenitor cell state
              </a>
            </td>
          <td>
            Jingyun Li, Susanna M Dang, Shreoshi Sengupta, Paul Schurmann, A. F. Dost, Aaron L. Moye, M. F. Trovero, Sidrah Ahmed, Margherita Paschini, Preetida J Bhetariya, Roderick Bronson, Shannan J. Ho Sui, Carla F. Kim
          </td>
          <td>2025-02-10</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Characterizing the variety of cell types in the tumor microenvironment (TME) and their organization into cellular communities is vital for elucidating the biological diversity of cancer and informing therapeutic strategies. Methods Here, we employed a machine learning-based algorithm framework, EcoTyper, to analyze single-cell transcriptomes from 139 patients with head and neck squamous cell carcinoma (HNSC)and gene expression profiles from 983 additional HNSC patients, aiming to delineate the fundamental cell states and ecosystems integral to HNSC. Results A diverse landscape of 66 cell states and 9 ecosystems within the HNSC microenvironment was identified, revealing classical cell types while also expanding upon previous immune classifications. Survival analysis revealed that specific cell states and ecotypes (ecosystems) are associated with patient prognosis, underscoring their potential as indicators of clinical outcomes. Moreover, distinct cell states and ecotypes exhibited varying responses to immunotherapy and chemotherapy, with several showing promise as predictive biomarkers for treatment efficacy. Conclusion Our large-scale integrative transcriptome analysis provides high-resolution insights into the cellular states and ecosystems of HNSC, facilitating the discovery of novel biomarkers and supporting the development of precision therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06299-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd5809c9e6b6aaac22c97a1616ff68cb7ac8d0d" target='_blank'>
              High-dimensional deconstruction of HNSC reveals clinically distinct cellular states and ecosystems that are associated with prognosis and therapy response
              </a>
            </td>
          <td>
            Lei Xiao, Zhe Shen, Zhaoyu Pan, Yuanzheng Qiu, Donghai Huang, Yong Liu, Chao Liu, Xin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy. Methods The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity. Results We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered. Conclusion This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5782e67aa86bd64459296bc264a422b250cba97" target='_blank'>
              SCOIGET: Predicting Spatial Tumor Evolution Pattern by Inferring Spatial Copy Number Variation Distributions
              </a>
            </td>
          <td>
            Yujia Zhang, Yitao Yang, Yan Kong, Bingxu Zhong, Kenta Nakai, Hui Lu
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Understanding the intricate tumor microenvironment (TME) is crucial for elucidating the mechanisms underlying the progression of cervical squamous cell carcinoma (CSCC) and its response to anti-PD-1 therapy. Methods In this study, we characterized 50,649 cells obtained from the CSCC for single-cell RNA sequencing and integrated bulk sequencing data from The Cancer Genome Atlas (TCGA) and clinical samples to explore their cell composition, metabolic processes, signaling pathways, specific transcription factors, lineage tracking and response to immunotherapy. In vivo experiments were performed to validate the function of key cell subsets. Results We identified ten major cell type and 35 subsets of stromal and immune cells in TME and observed distinct patterns in the metabolic processes and signaling pathways of these cells between tumor and normal tissues. Furthermore, PCNA clamp-associated factor (PCLAF)+ tumor-associated epithelial cell (TAEpis) was negatively correlated with the number of C-X-C motif chemokine ligand 13 (CXCL13)+ CD8+ T cells, overall survival, and response to anti-programmed cell death-1(PD-1) therapy in patients with CSCC. Both in vivo and in vitro experiments demonstrated that PCLAF+ TAEpis promotes the apoptosis of CD8+ T and tumor growth, while also inhibiting T cell infiltration and function. Conclusion Our findings illuminate the heterogeneity of the complex TME in CSCC and offer evidence supporting PCLAF+ TAEpis as a promising therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03725-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Lung cancer is a leading cause of cancer-related mortality worldwide, largely due to its heterogeneity and intrinsic drug resistance. Malignant pleural effusions (MPEs) provide diverse tumor cell populations ideal for studying these complexities. Although chemotherapy and targeted therapies can be initially effective, subpopulations of cancer cells with phenotypic plasticity often survive treatment, eventually developing resistance. Here, we integrated single-cell isolation and three-dimensional (3D) spheroid culture to dissect subclonal heterogeneity and drug responses, aiming to inform precision medicine approaches. Using A549 lung cancer cells, we established a cisplatin-resistant line and isolated three resistant subclones (Holoclone, Meroclone, Paraclone) via single-cell sorting. In 3D spheroids, Docetaxel and Alimta displayed higher IC50 values than in 2D cultures, suggesting that 3D models better reflect clinical dosing. Additionally, MPE-derived Holoclone and Paraclone subclones exhibited distinct sensitivities to Giotrif and Capmatinib, revealing their heterogeneous drug responses. Molecular analyses confirmed elevated ABCB1, ABCG2, cancer stem cell (CSC) markers (OCT4, SOX2, CD44, CD133), and epithelial–mesenchymal transition (EMT) markers (E-cadherin downregulation, increased Vimentin, N-cadherin, Twist) in resistant subclones, correlating with enhanced migration and invasion. This integrated approach clarifies the interplay between heterogeneity, CSC/EMT phenotypes, and drug resistance, providing a valuable tool for predicting therapeutic responses and guiding personalized, combination-based lung cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6813439ca1c219dd03651dca35df5520579d3204" target='_blank'>
              Application of an Integrated Single-Cell and Three-Dimensional Spheroid Culture Platform for Investigating Drug Resistance Heterogeneity and Epithelial–Mesenchymal Transition (EMT) in Lung Cancer Subclones
              </a>
            </td>
          <td>
            Shin-Hu Chen, Jian-Hong Yu, Yu-Chun Lin, Yi-Ming Chang, Nien-Tzu Liu, Su-Feng Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8b3640c44bd80cadd9a05e0404e0638d900b1d2" target='_blank'>
              Double–Multiplex Immunostainings for Immune Profiling of Invasive Breast Carcinoma: Emerging Novel Immune-Based Biomarkers
              </a>
            </td>
          <td>
            Sofia D P Theodorou, Konstantinos Ntostoglou, I. Nikas, Dimitrios Goutas, V. Georgoulias, Christos Kittas, Ioannis S. Pateras
          </td>
          <td>2025-03-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Introduction The tumor microenvironment plays a pivotal role in cancer progression and therapeutic resistance, with tumor-associated macrophages significantly influencing immune suppression and tumor growth. Colorectal cancers (CRC) classified as Consensus Molecular Subtype 4 (CMS4) and triple-negative breast cancers subsets are particularly characterized by a mesenchymal phenotype, immune exclusion, and extensive macrophage infiltration. This study aimed to investigate how targeting cancer cell stemness with specific inhibitors could modulate macrophage polarization in CRC in vitro and breast cancer in vivo, potentially shifting the immune balance from pro-tumor M2-like to anti-tumor M1-like macrophages. Methods We used four stemness inhibitors—salinomycin, SB-431542, JIB-04, and napabucasin—each targeting different pathways (Wnt/β-catenin, TGF-β, histone demethylation, and STAT3, respectively), to evaluate their effects on CMS4 CRC cell lines (HCT116 and SW620) and human peripheral blood-derived macrophages in an indirect co-culture model. Results Our results showed that CMS4 CRC cell lines induced distinct macrophage polarization patterns, with HCT116 promoting M2-like macrophages and SW620 leaning toward M1-like profile. Notably, the combination of stemness inhibitors reduced stemness markers (CD133, CD44) in colorectal cancer cells and shifted macrophage polarization toward an M1-like phenotype, particularly in co-culture with HCT116. In vivo studies using the syngeneic immunocompetent EO771 breast cancer mouse model demonstrated that combination of stemness inhibitors increased the M1/M2 macrophage ratio. Conclusions Our study highlights the dual potential of stemness inhibitors to target both cancer cells and the immune microenvironment. These findings offer promising strategies for enhancing favorable immunomodulation in mesenchymal-like colorectal tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/211f6363e023c7088c47db3138c8836f0d0dca08" target='_blank'>
              Targeting stemness pathways modulates macrophage polarization and reprograms the tumor microenvironment
              </a>
            </td>
          <td>
            Austeja Butkute, Marius Baltramonaitis, Simona Malmige, A. Darinskas, V. Pašukonienė, A. Mlynska
          </td>
          <td>2025-03-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Neurodegenerative diseases and infections can produce lasting effects on brain function, yet the spatial molecular mechanisms underlying these changes remain poorly understood. Here, we present high-resolution spatial transcriptomics of 40 postmortem brain samples from patients with Parkinson’s disease, frontotemporal dementia, dementia with Lewy bodies, and severe COVID-19. Analyzing over 1.5 million spatially resolved cells across dorsolateral prefrontal cortex and anterior cingulate cortex revealed disease-specific transcriptional signatures with pronounced layer-and region-specificity. In Parkinson’s disease, we identified stressed neurons creating distinctive microenvironmental gradients where metabolic and protein degradation pathways are elevated near stress epicenters, while regenerative processes increase with distance. COVID-19 brains displayed extensive peripheral immune cell infiltration, particularly in the subcortical white matter, accompanied by compromised blood-brain barrier and coordinated neuroinflammatory responses from microglia, astrocytes, and endothelial cells. Integration of miRNA sequencing with spatial transcriptomics uncovered layer-specific regulatory patterns, including neuroinflammation-associated miR-155. This atlas provides unprecedented insights into disease pathology and highlights the critical importance of spatial molecular context in understanding brain disorders. Key Messages [1] A high-resolution single-cell spatial transcriptomics atlas of the dorsolateral prefrontal cortex and anterior cingulate cortex across neurodegenerative conditions and severe COVID-19 [2] Region-and layer-specific transcriptional dysregulation across disease comparisons reveals disease-specific differential vulnerability [3] Metabolically stressed cells found selectively in the anterior cingulate cortex of Parkinson’s disease patients but not in dementia with Lewy bodies, along with detailed characterization of their spatial microenvironment [4] Peripheral immune cell clusters identified in the white matter of the cerebral cortex of COVID-19 patients, with detailed characterization of their spatial microenvironment [5] Integration of bulk miRNA sequencing reveals cortical layer-specific miRNA regulatory patterns">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97c071df2a109d09f7501e38a14ec4947fe8e317" target='_blank'>
              Single-cell Spatial Transcriptomics Reveals Disease-specific Microenvironmental Niches in Neurodegeneration and COVID-19
              </a>
            </td>
          <td>
            Simon Graf, N. Ludwig, Matthias Flotho, Annika Engel, I. F. Diks, Paula Zimmer, Wiebke M. Jürgens-Wemheuer, Arne Wrede, Daniela Mirzac, G. González-Escamilla, Victoria Wagner, Vilas Menon, Sergiu Groppa, P. D. De Jager, Lai Yiwei, Miguel A. Esteban, Walter J. Schulz-Schaeffer, Andreas Keller
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Background Cancer development is closely associated with complex alterations in the tumor microenvironment (TME). Among these, immune cells within the TME play a huge role in personalized tumor diagnosis and treatment. Objectives This review aims to summarize the diversity of immune cells in the TME, their impact on patient prognosis and treatment response, and the contributions of single‐cell RNA sequencing (scRNA‐seq) in understanding their functional heterogeneity. Methods We analyzed recent studies utilizing scRNA‐seq to investigate immune cell populations in the TME, focusing on their interactions and regulatory mechanisms. Results ScRNA‐seq reveals the functional heterogeneity of immune cells, enhances our understanding of their role in tumor antibody responses, and facilitates the construction of immune cell interaction networks. These insights provide guidance for the development of cancer immunotherapies and personalized treatment approaches. Conclusion Applying scRNA‐seq to immune cell analysis in the TME offers a novel pathway for personalized cancer treatment. Despite its promise, several challenges remain, highlighting the need for further advancements to fully integrate scRNA‐seq into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36e80d2a964412d958098f4ad446c2f9779f2b7e" target='_blank'>
              Analyze the Diversity and Function of Immune Cells in the Tumor Microenvironment From the Perspective of Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Lujuan Ma, Yu Luan, Lin Lu
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Spatially resolved transcriptomics has transformed our ability to study complex tissues at the cellular and subcellular resolution. However, targeted spatial technologies require pre-selected gene panels, which are typically curated based on existing biological knowledge and prior research hypotheses. While current methods often prioritize capturing cell type information, we argue that an effective gene panel should also capture cell type diversity, cell states, pathway-level information, and minimize redundancy. To address these broader requirements, we developed a gene panel characterization platform that characterizes panels across multiple perspectives, thus allowing us to compare panels comprehensively. Notably, computationally constructed gene panels performed competitively in capturing major cell types when compared to our in-house manually curated panel. However, refined manual curation offered distinct advantages, particularly in capturing minor and rare cell types and exhibited lower information redundancy comparatively. Building on this framework, we integrated these metrics into a deep learning platform, panelScope, leveraging them as a loss function to design holistic gene panels. Using an acute myeloid leukemia (AML) dataset with 42 well-defined cell types and the 5K Xenium panel from 10X Genomics, we demonstrate the utility of our framework in comprehensively characterizing gene panels, enabling the design of tailored panels that address diverse research needs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef9065cee3317a4c66d56d6484e04cef47751f06" target='_blank'>
              Multi-view gene panel characterization for spatially resolved omics
              </a>
            </td>
          <td>
            Daniel Kim, Wenze Ding, Akira Nguyen Shaw, Marni Torkel, Cameron J Turtle, Pengyi Yang, J. Yang
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc383607243031294dc9e768566a894cfedd572c" target='_blank'>
              Glioblastoma stem cell morphotypes convey distinct cell states and clinically relevant functions
              </a>
            </td>
          <td>
            C. Barelli, Matteo Bonfanti, Filippo Mirabella, Dario Ricca, Negin Alizadehmohajer, Viktoria Sokolova, Alberto Campione, G. Sicuri, C. Peano, Stefania Faletti, Roberto Stefini, N. Kalebic
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In a groundbreaking study published in Cell , Onder and colleagues revealed the pivotal role of ﬁ broblastic reticular cells (FRCs) in creating specialized niches, providing robust immune activation against non-small cell lung cancer (NSCLC), which paves the way for innovative therapeutic strategies. 1 Cancers are complex ecosystems comprising tumor cells intricately interwoven with a diverse array of non-cancerous cells — including immune cells, cancer-associated ﬁ broblasts (CAFs), endothelial cells, and various tissue-speci ﬁ c resident cells — embedded within a remodeled extracellular matrix (ECM), collectively shaping the tumor microenvironment (TME). 2 The TME differs markedly among tumors, in ﬂ uenced by factors such as tumor location, stage, the inherent properties of the cancer cells, and individual patient conditions. The TME plays a crucial role in cancer progression by regulating processes such as cancer cell migration, proliferation, tumor vascularization, and the modulation of anti-cancer immune responses. 2 Effects of the TME are partially mediated by secreted factors, including exosomes, metabolites, cytokines, chemokines, and a remodeled extracellular matrix produced by CAFs, which profoundly in ﬂ uence cancer progression. 2 Numerous studies revealed that the TME plays a pivotal role in modulating anti-cancer immune responses, either promoting or suppressing these processes in a manner that appears highly context-dependent. 3 Effective immune responses against cancer cells basically involve three different key mechanisms. (i) tumor-antigen-speci ﬁ c CD8 + T cells are activated in draining lymph nodes and then (ii) recruited into the tumor. (iii) Recruited CD8 + T cells are maintained in an activated state within specialized niches in the TME. 1 The TME controls these mechanisms through multiple pathways. First, its cellular components secrete">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9620b6ccd6e7d4494d0052c67c0eaf02c7fda508" target='_blank'>
              Fibroblastic niches in action: CCL19+ reticular cells drive anti-tumor immunity in lung cancer
              </a>
            </td>
          <td>
            Alessandro Ianni, Alejandro Vaquero, Thomas Braun
          </td>
          <td>2025-03-24</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64eb12bd7d9a78fd223bedc136a7a4d7a78d40" target='_blank'>
              Abstract B068: A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer
              </a>
            </td>
          <td>
            M. Oh, J. Abascal, L. Tran, R. Salehi-Rad, R. Lim, C. Dumitras, Cara E. Yean, Sunny Xiao, Bitta Kahangi, William P Crosson, E. P. Reyes, Nalani Coleman, Austin K. Rennels, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Metabolic processes are crucial in immune regulation, yet the impact of metabolic heterogeneity on immunological functions remains unclear. Integrating metabolomics into immunology allows the exploration of the interactions of multilayered features in the biological system and the molecular regulatory mechanism of these features. To elucidate such insight in lung squamous cell carcinoma (LUSC), we analyzed 106 LUSC tumor tissues. We performed high-resolution matrix-assisted laser desorption/ionization (MALDI) mass spectrometry imaging (MSI) to obtain spatial metabolic profiles, and immunohistochemistry to detect tumor-infiltrating T lymphocytes (TILs). Unsupervised k-means clustering and Simpson’s diversity index were employed to assess metabolic heterogeneity, identifying five distinct metabolic tumor subpopulations. Our findings revealed that TILs are specifically associated with metabolite distributions, not randomly distributed. Integrating a validation cohort, we found that heterogeneity-correlated metabolites interact with CD8+ TIL-associated genes, affecting survival. High metabolic heterogeneity was linked to worse survival and lower TIL levels. Pathway enrichment analyses highlighted distinct metabolic pathways in each subpopulation and their potential responses to chemotherapy. This study uncovers the significant impact of metabolic heterogeneity on immune functions in LUSC, providing a foundation for tailoring therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b4c98251a8ac1956e7ccdfa63c51218dbe1751b" target='_blank'>
              Metabolic heterogeneity in tumor cells impacts immunology in lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Qian Wang, Na Sun, Chaoyang Zhang, Thomas Kunzke, P. Zens, A. Feuchtinger, Sabina Berezowska, Axel Walch
          </td>
          <td>2025-02-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12d9a96ad448410a624d4884f42c080b637c2227" target='_blank'>
              Whole-slide multiplexed tissue imaging of prostate adenocarcinoma to investigate immune niches in high- versus low-grade tumors.
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, B. Labadie, Aishwarya Atmakuri, K. Chaudagar, Eamon Toye, Andréanne Gagné, Jia-Ren Lin, E. Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Microvascular invasion (MVI) is linked to poor prognosis, early recurrence and post-surgical intrahepatic metastasis of hepatocellular carcinoma (HCC) but roles of tumor-associated endothelial cells (TECs) remain unclear. The aim of the current study was to investigate the role of TECs in microvascular invasion in HCC. Methods Single-cell RNA sequencing (scRNA-seq) data from three patients with MVI and two patients with non-MVI HCC were used to identify TECs subpopulations via Seurat R package. Using bioinformatics analysis identified co-expression modules associated with MVI in TECs. Differential gene expression analysis, KME values and Gene Expression Profiling Interactive Analysis (GEPIA) survival were utilized to identify genes with significant involvement. TECs subgroup developmental trajectory was analyzed using monocle2. Five additional spatial transcriptomics (ST) datasets and four HCC postoperative pathological specimens were used to validate the differential expression of subgroups of TECs and hub genes between MVI and non-MVI groups. Results Distinct TECs subgroups had significant heterogeneity between datasets from MVI and non-MVI patients. MVI samples had TECs subgroups with increased levels of the epithelial−mesenchymal transition (EMT), endothelial cell migration and angiogenesis. Opposing EMT development was found in MVI TECs relative to non-MVI TECs. TM4SF1 was highly expressed in TECs undergoing the EMT and is thought to be linked to MVI. Conclusion TECs with elevated TM4SF1 expression facilitate MVI during HCC via an effect on the EMT, suggesting the potential of TM4SF1 as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de61af4341cad2d9fb79114592979b95505d43c" target='_blank'>
              TM4SF1 overexpression in tumor-associated endothelial cells promotes microvascular invasion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Junwu Guo, Liangrui Chen, Binghua Dai, Chengjun Sui, Zhitao Dong, Keji Chen, Kecai Duan, Kunpeng Fang, Aijun Li, Kui Wang, Li Geng
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is an aggressive malignancy characterized by rapid proliferation and high metastatic potential. It is characterized by universal inactivation of and RB1, overexpression of the MYC family and dysregulation of multiple oncogenic signaling pathways. Among different patients, SCLCs are similar at the genetic level but exhibit significant heterogeneity at the molecular level. The classification of SCLC has evolved from a simple neuroendocrine (NE)/non-neuroendocrine (non-NE) classification system to a transcription factor-based molecular subtype system; lineage plasticity adds further complexity and poses challenges for therapeutic development. While SCLC is initially sensitive to platinum-based chemotherapy, resistance develops rapidly, leading to a dismal prognosis. Various antibodies, including PD-1/PD-L1 inhibitors and antibody‒drug conjugates, have been introduced into clinical practice or are being evaluated in clinical trials. However, their therapeutic benefits for SCLC patients remain limited. This review summarizes SCLC carcinogenic mechanisms, tumor heterogeneity, and the immune microenvironment of SCLC, with a focus on recent advances in metastasis and resistance mechanisms. Additionally, the corresponding clinical progress in tackling these challenges is discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98544ce43f27a0684b668c8dba29b6f68626bac0" target='_blank'>
              Current and future therapies for small cell lung carcinoma
              </a>
            </td>
          <td>
            X. Zhai, Zhengkun Zhang, Yuxin Chen, Yanmou Wu, Cheng Zhen, Yu Liu, Yiyun Lin, Chong Chen
          </td>
          <td>2025-04-01</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d948793b1aa3c6db446f2356cc375541bc62bfa6" target='_blank'>
              A Multimodal Framework to Uncover Drug-Responsive Subpopulations in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Yue Wang, Santiago Haase, Austin A. Whitman, Adriana Beltran, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Gastric cancer is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME). Cancer-associated fibroblasts (CAFs), which are a key component of the TME, exhibit significant heterogeneity and play crucial roles in tumor progression. Therefore, a comprehensive understanding of CAFs is essential for developing novel therapeutic strategies for gastric cancer. Methods This study investigates the characteristics and functional information of CAF subtypes and explores the intercellular communication between CAFs and malignant epithelial cells (ECs) in gastric cancer by analyzing single-cell sequencing data from 24 gastric cancer samples. CellChat was employed to map intercellular communication, and Seurat was used to integrate single-cell sequencing data with spatial transcriptome data to reconstruct a comprehensive single-cell spatial map. The spatial relationship between apCAFs and cancer cells was analyzed using multicolor immunohistochemistry. Results Cells were categorized into nine distinct categories, revealing a positive correlation between the proportions of epithelial cells (ECs) and fibroblasts. Furthermore, six fibroblast subpopulations were identified: inflammatory (iCAFs), pericytes, matrix (mCAFs), antigen-presenting (apCAFs), smooth muscle cells (SMCs), and proliferative CAFs (pCAFs). Each of these subpopulations was linked to various biological processes and immune responses. Malignant ECs exhibited heightened intercellular communication, particularly with CAF subpopulations, through specific ligand-receptor interactions. High-density regions of CAF subpopulations displayed spatial exclusivity, with pericytes serving as a source for iCAFs, mCAFs, and apCAFs. Notably, malignant ECs and apCAFs showed increased interactions, with certain ligand-receptor pairs potentially impacting the prognosis of gastric cancer. Multiplex immunohistochemistry (mIHC) confirmed the close spatial proximity of apCAFs to cancer cells in gastric cancer. Conclusion Our study provided a comprehensive characterization of CAF heterogeneity in gastric cancer and revealed the intricate intercellular networks within the TME. The identified CAF subpopulations and their interactions with malignant cells could serve as potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06376-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58e073c2b3967f3720af1b89697ce0e1aad33c4c" target='_blank'>
              Abstract B086: Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment
              </a>
            </td>
          <td>
            Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/644814c562e273342aaa2172d8ae291ec9003332" target='_blank'>
              Characterization of tumour heterogeneity through segmentation-free representation learning on multiplexed imaging data.
              </a>
            </td>
          <td>
            Jimin Tan, H. Le, Jiehui Deng, Yingzhuo Liu, Yuan Hao, Michelle Hollenberg, Wenke Liu, Joshua M. Wang, Bo Xia, Sitharam Ramaswami, Valeria Mezzano, Cynthia Loomis, Nina Murrell, Andre L. Moreira, Kyunghyun Cho, Harvey I Pass, Kwok-Kin Wong, Yi Ban, Benjamin G. Neel, A. Tsirigos, David Fenyö
          </td>
          <td>2025-02-20</td>
          <td>Nature biomedical engineering</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="Prostate cancer (PC) is the most frequently diagnosed malignancy among men and contributes significantly to cancer-related mortality. While recent advances in in vitro PC modeling systems have been made, there remains a lack of robust preclinical models that faithfully recapitulate the genetic and phenotypic characteristics across various PC subtypes-from localized PC (LPC) to castration-resistant PC (CRPC)-along with associated stromal cells. Here, we established human PC assembloids from LPC and CRPC tissue by reconstituting tumor organoids with corresponding cancer-associated fibroblasts (CAFs), thereby incorporating aspects of the tumor microenvironment (TME). Established PC organoids exhibited high concordance in genomic landscape with parental tumors, and the tumor assembloids showed a higher degree of phenotypic similarity to parental tumors compared to tumor organoids without CAFs. PC assembloids displayed increased proliferation and reduced sensitivity to anti-cancer treatments, indicating that PC assembloids are potent tools for understanding PC biology, investigating interaction between tumor and CAFs, and identifying personalized therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/105fe3ccf7c35aabcf3c81f5242c91ac5e7680b1" target='_blank'>
              Generation of prostate cancer assembloids modeling the patient-specific tumor microenvironment.
              </a>
            </td>
          <td>
            Juhee Lee, Yunhee Kim, Cheol Lee, Seong Soo Jeon, Hae Seo, Jongwon Lee, Jungmin Choi, Minyong Kang, Eunjee Kim, Kunyoo Shin
          </td>
          <td>2025-03-31</td>
          <td>PLoS genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The ESCRT (endosomal sorting complex required for transport) machinery is essential for various cellular processes, yet its role in head and neck squamous cell carcinoma (HNSCC) is poorly understood. We utilized The Cancer Genome Atlas (TCGA) datasets to analyze the expression of ESCRT genes. Bulk RNA-sequencing data and HNSCC tissue microarrays (TMAs) were used to evaluate VPS25 expression and its clinical significance. Single-cell RNA sequencing of tumor tissues and VPS25 knockdown experiments in CAL27 cells were used to investigate its biological functions. Immunohistochemistry, spatial transcriptomics, and immunotherapy datasets highlighted the involvement of VPS25 in immune suppression and its potential as a predictive biomarker. The results demonstrated significant VPS25 overexpression in HNSCC tissues, which correlated with poor clinical outcomes. It promoted tumor cell proliferation and migration while reducing immune cell infiltration in the tumor microenvironment (TME). Additionally, by upregulating PVR expression in tumor cells, VPS25 activated the immunosuppressive PVR-TIGIT signaling axis, thereby facilitating immune evasion. Furthermore, VPS25 emerged as a potential biomarker for predicting immunotherapy response. These findings highlight VPS25 as a pivotal regulator of tumor progression and immune evasion in HNSCC and a promising target for therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/902195b920715f84e3501a67211a1883f6d279dd" target='_blank'>
              VPS25 Promotes an Immunosuppressive Microenvironment in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Liguo Chen, Yu-Han Fang, Kui-Ming Wang, Wei Zhang, Gang Chen
          </td>
          <td>2025-02-22</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) are pivotal components of the breast cancer (BC) tumor microenvironment (TME), significantly influencing tumor progression and response to therapy. However, the heterogeneity and specific roles of TAM subpopulations in BC remain inadequately understood. Methods We performed an integrated analysis of single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data from BC patients to comprehensively characterize TAM heterogeneity. Utilizing the MetaTiME computational framework and consensus clustering, we identified distinct TAM subtypes and assessed their associations with clinical outcomes and treatment responses. A machine learning-based predictive model was developed to evaluate the prognostic significance of TAM-related gene expression profiles. Results Our analysis revealed three distinct TAM subgroups. Notably, we identified a novel macrophage subtype, M_Macrophage-SPP1-C1Q, characterized by high expression of SPP1 and C1QA, representing an intermediate differentiation state with unique proliferative and oncogenic properties. High infiltration of M_Macrophage-SPP1-C1Q was significantly associated with poor overall survival (OS) and chemotherapy resistance in BC patients. We developed a Random Forest (RF)-based predictive model, Macro.RF, which accurately stratified patients based on survival outcomes and chemotherapy responses, independent of established prognostic parameters. Conclusion This study uncovers a previously unrecognized TAM subtype that drives poor prognosis in BC. The identification of M_Macrophage-SPP1-C1Q enhances our understanding of TAM heterogeneity within the TME and offers a novel prognostic biomarker. The Macro.RF model provides a robust tool for predicting clinical outcomes and guiding personalized treatment strategies in BC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03750-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/287d3599b1ab22b44316058fde7d65a5ef96c17d" target='_blank'>
              Integrated single-cell and bulk transcriptomic analysis identifies a novel macrophage subtype associated with poor prognosis in breast cancer
              </a>
            </td>
          <td>
            Qing Wang, Yushuai Yu, Liqiong Ruan, Mingyao Huang, Wei Chen, Xiaomei Sun, Jun Liu, Zirong Jiang
          </td>
          <td>2025-03-27</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac4b19bdc51a77041e08cca430b7dd7ca4fef49" target='_blank'>
              Abstract PR007: Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity
              </a>
            </td>
          <td>
            D. Peeper, Sofía Ibáñez-Molero, Johanna Veldman, Juan Simon-Nieto, J. Traets, Austin George, K. Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, W. V. van Houdt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) represent a central cell population of the tumor microenvironment (TME). Recently, single‐cell RNA‐sequencing (scRNA‐seq) analyses of primary tumors of different cancer entities yielded different classifications of CAF subsets underscoring the heterogeneity of CAFs within the TME. Here, we analyzed the transcriptional signatures of approximately 8400 CAFs and normal fibroblasts by scRNA‐seq and compared genetic profiles of CAFs from murine melanoma primary tumors to CAFs from corresponding melanoma lung metastases. This revealed distinct subsets for primary tumor and metastasis‐specific CAF populations, respectively. Combined with the spatial characterization of metastasis CAFs at the RNA and protein level, scRNA analyses indicate tumor‐dependent crosstalk between neutrophils and CAFs, mediated via SAA3 and IL1b‐related signaling pathways, which can be recapitulated in vitro. Analyzing tissue sections of human patient samples, this interaction was found to be present in human melanoma metastasis. Taken together, our data highlight unique characteristics of metastasis CAFs with potential therapeutic impact for melanoma metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae4d264150cbc5dcfaa11bf8a1b1f48e7dcf511" target='_blank'>
              A Unique Signature for Cancer‐Associated Fibroblasts in Melanoma Metastases
              </a>
            </td>
          <td>
            Saskia Tauch, Joschka Hey, Bettina Kast, N. Gengenbacher, Lena Weiß, Melanie Sator‐Schmitt, S. Lohr, Alexander Brobeil, P. Schirmacher, Jochen Utikal, H. Augustin, C. Plass, Peter Angel
          </td>
          <td>2025-02-09</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce737a0ad997a56048c24df55247b712e440df3d" target='_blank'>
              Abstract PR002: T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment
              </a>
            </td>
          <td>
            Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick S Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa Mcginnis, Kevin B Walsh, Weilan Ye
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The spatial interactions between malignant and immune cells in the tumor microenvironment (TME) play a crucial role in cancer biology and treatment response. Understanding these interactions is critical for predicting prognosis and assessing immunotherapy effectiveness. Conventional methods, which focus on local spatial features, often struggle to achieve robust analysis due to the complex and heterogeneous cellular distributions. We propose a Topological Data Analysis (TDA)-based framework using both global and local spatial features between malignant and immune cells. For the global aspects, we introduce Topological Malignant Clusters (TopMC), a method utilizing persistence diagrams in TDA to capture the global spatial shapes of malignant cell distributions. It quantifies tumor-immune cell infiltration at a global scale by computing distances from the boundaries of the TopMC. Local interactions are evaluated through the density of malignant cells. Using high-resolution multiplex immunofluorescence (mIF) imaging in Diffuse Large B Cell Lymphoma, we integrate distance and density measures into a distance-density space to analyze patterns of malignant-immune cell interactions in TME. This study shows the robustness of the proposed approach to variations in cell distribution, enabling consistent analysis irrespective of whether images are acquired from malignant-enriched or border regions of the tumors. Furthermore, we elucidate the correlation between spatial patterns of immune phenotypes and patient survival probability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f549327f511c89a0763403d23ea2ac5417a3579" target='_blank'>
              Spatial Analysis of Malignant-Immune Cell Interactions in the Tumor Microenvironment Using Topological Data Analysis
              </a>
            </td>
          <td>
            Seol Ah Park, Yongsoo Kim, Paweł Dłotko, Davide Gurnari, Jooyoung Hahn
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73dbf54f5b2a0ce96fdfefae2ba812ee9c7ea68b" target='_blank'>
              The 1000+ mouse project: large-scale spatiotemporal parametrization and modeling of preclinical cancer immunotherapies
              </a>
            </td>
          <td>
            Adam Kenet, S. Achar, Alka Dwivedi, John Buckley, M. Pouzolles, Haiying Qin, C. Chien, N. Taylor, Grégoire Altan-Bonnet
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4672987b5633b49ff2536422175c3660e10c3a" target='_blank'>
              Identification of a shared persistence program in triple-negative breast cancer across treatments and patients
              </a>
            </td>
          <td>
            Léa Baudre, Grégoire Jouault, Pacôme Prompsy, Mélissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, C. Vallot
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Previous studies have highlighted the capacity of brain cancer cells to functionally interact with the tumour microenvironment (TME). This TME-cancer crosstalk crucially contributes to tumour cell invasion and disease progression. In this study, we performed spatial transcriptomic sequencing analysis of a complete annotated tumour-infiltrated brainstem from a single diffuse intrinsic pontine glioma (DIPG) patient. Gene signatures from ten sequential tumour regions were analysed to assess mechanisms of disease progression and oncogenic interactions with the TME. We identified four distinct tumour subpopulations and assessed respective ligand-receptor pairs that actively promote DIPG tumour progression via crosstalk with endothelial, neuronal and immune cell communities. Our analysis found potential targetable mediators of tumour-to-TME communication, including members of the complement component system and the neuropeptide/GPCR ligand-receptor pair ADCYAP1-ADCYAP1R1. These interactions could influence DIPG tumour progression and represent novel therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01952-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ea420e4a0256eeeba7153fed74e4eb3b92c386" target='_blank'>
              Spatial analysis of a complete DIPG-infiltrated brainstem reveals novel ligand-receptor mediators of tumour-to-TME crosstalk
              </a>
            </td>
          <td>
            Anja Kordowski, Onkar Mulay, Xiao Tan, Tuan Vo, Ulrich Baumgartner, Mellissa Maybury, Timothy Hassall, L. Harris, Quan Nguyen, B. Day
          </td>
          <td>2025-02-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC.
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e047830d6d15c84fdba5a27411c84ea5a08197" target='_blank'>
              Multi-Modal Disentanglement of Spatial Transcriptomics and Histopathology Imaging
              </a>
            </td>
          <td>
            Hassaan Maan, Zongliang Ji, Elliot Sicheri, Tiak Ju Tan, Alina Selega, Ricardo Gonzalez, Rahul G Krishnan, Bo Wang, Kieran R. Campbell
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea52e3e92b1a5c4c3caddeca60f128feaaaf952b" target='_blank'>
              Single-cell and spatial transcriptomic insights into glioma cellular heterogeneity and metabolic adaptations
              </a>
            </td>
          <td>
            Yixin Fu, Yong Yi, Yongxiang Shao, Jingcheng Jiang, Qingshan Deng
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The influence of tertiary lymphoid structures (TLSs) on disease progression and the response to immunotherapy in head and neck squamous cell carcinoma (HNSCC) is well established, yet the heterogeneity among these structures remains largely unexplored. We utilized digital spatial profiling technology to perform in situ transcriptomic sequencing of TLSs across varying levels of maturation and distinct tumor regions within HNSCC. We assessed the prognostic significance of TLS maturation and spatial distribution in 260 patients with HNSCC through hematoxylin and eosin staining and multiplex immunohistochemistry. Furthermore, we established a TLS scoring system to predict survival in patients with HNSCC. Our study revealed that mature TLSs in the intratumor region (Intra-TLSs) of HNSCC, enriched with memory B cells, plasma cells, and CD4+ T cells, presented increased B-cell activity gene expression. Conversely, early TLSs (E-TLSs), abundant in endothelial cells, fibroblasts, and regulatory T cells, express epithelial‒mesenchymal transition (EMT)-related genes, potentially fostering tumor growth. Compared with mature TLSs within the peritumoral region (Peri-TLSs), mature Intra-TLSs have greater memory B-cell and macrophage densities and upregulate genes involved in B-cell receptor signaling and immune effector processes. Mature Peri-TLSs, characterized by endothelial cell enrichment and EMT receptor interaction genes, may contribute to tumor progression and immune evasion. Patients with mature Intra-TLSs or invasive margin TLSs (Invas-TLSs) have improved 5-year survival, whereas those with mature Peri-TLSs have poorer prognoses. By integrating TLS maturity and distribution in HNSCC, we developed a TLS scoring system to guide personalized treatments, which is crucial for predicting outcomes. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-03952-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e4d2a67b57ff8cbfbb9e704d52ca386805064c" target='_blank'>
              Distinct maturity and spatial distribution of tertiary lymphoid structures in head and neck squamous cell carcinoma: implications for tumor immunity and clinical outcomes
              </a>
            </td>
          <td>
            Shuai Xu, Chao Han, Jian Zhou, Di Yang, Hui Dong, Yiwei Zhang, Tingting Zhao, Yi Tian, Yuzhang Wu
          </td>
          <td>2025-02-11</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are highly heterogeneous cells and important components of the breast tumor microenvironment (TME). However, their role and clinical value in ER-positive breast cancer have not been fully clarified. Our study aims to comprehensively characterize the heterogeneity, potential biological functions, and molecular mechanisms of CAFs in ER-positive breast cancer within the tumor microenvironment using multi-omics data, to provide new strategies for the diagnosis and treatment of ER-positive breast cancer patients. In this study, we found that COL1A2(+) MMP1(+) and COL1A2(+) MMP1(-) CAFs were associated with unfavorable prognosis. The dynamic evolution and cell-cell communications of CAFs were analyzed, revealing that COL1A2(+) MMP1(+/-) CAFs show extensive crosstalk with tumor-associated macrophages (TAMs), contributing to an immunosuppressive TME. Moreover, the somatic mutation of TP53 may be a potential indicator for evaluating the infiltration of COL1A2(+) MMP1(+/-) CAFs. Finally, an MRI-based radiomic model was constructed to estimate the abundance of these CAFs. In conclusion, our findings provide a theoretical basis for targeting CAFs and offer a noninvasive approach to evaluate the infiltration level of COL1A2(+) MMP1(+/-) CAFs. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03705-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9547a039c0fe93efd8fd0787388be5b2aa0012a6" target='_blank'>
              Integrating spatial and single-cell transcriptomes reveals the role of COL1A2(+) MMP1(+/-) cancer-associated fibroblasts in ER-positive breast cancer
              </a>
            </td>
          <td>
            Zhi-Hao Yu, Huan-Ling Xu, Shuo Wang, Yingxi Li, Guixin Wang, Yao Tian, Zhaohui Chen, Wenbin Song, Long He, Xin Wang, Xuchen Cao, Yue Yu
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Spatial omics enable the characterization of colocalized cell communities that coordinate specific functions within tissues. These communities, or niches, are shaped by interactions between neighboring cells, yet existing computational methods rarely leverage such interactions for their identification and characterization. To address this gap, here we introduce NicheCompass, a graph deep-learning method that models cellular communication to learn interpretable cell embeddings that encode signaling events, enabling the identification of niches and their underlying processes. Unlike existing methods, NicheCompass quantitatively characterizes niches based on communication pathways and consistently outperforms alternatives. We show its versatility by mapping tissue architecture during mouse embryonic development and delineating tumor niches in human cancers, including a spatial reference mapping application. Finally, we extend its capabilities to spatial multi-omics, demonstrate cross-technology integration with datasets from different sequencing platforms and construct a whole mouse brain spatial atlas comprising 8.4 million cells, highlighting NicheCompass' scalability. Overall, NicheCompass provides a scalable framework for identifying and analyzing niches through signaling events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14cda68a9739dfb6fdc7acbc975a0b67601ec1ae" target='_blank'>
              Quantitative characterization of cell niches in spatially resolved omics data.
              </a>
            </td>
          <td>
            Sebastian Birk, Irene Bonafonte-Pardàs, Adib Miraki Feriz, Adam Boxall, E. Agirre, Fani Memi, A. Maguza, Anamika Yadav, Erick Armingol, Rong Fan, Gonçalo Castelo-Branco, F. Theis, O. Bayraktar, Carlos Talavera-López, M. Lotfollahi
          </td>
          <td>2025-03-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Although two-thirds of cancers arise from loss-of-function mutations in tumor suppressor genes, there are few approved targeted therapies linked to these alterations. Synthetic lethality offers a promising strategy to treat such cancers by targeting vulnerabilities unique to cancer cells with these mutations. To identify clinically relevant synthetic lethal interactions, we analyzed genome-wide CRISPR/Cas9 knock-out (KO) viability screens from the Cancer Dependency Map and evaluated their clinical relevance in patient tumors through mutual exclusivity, a pattern indicative of synthetic lethality. Indeed, we found significant enrichment of mutual exclusivity for interactions involving cancer driver genes compared to non-driver mutations. To identify therapeutic opportunities, we integrated drug sensitivity data to identify inhibitors that mimic the effects of CRISPR-mediated KO. This approach revealed potential drug repurposing opportunities, including BRD2 inhibitors for bladder cancers with ARID1A mutations and SIN3A-mutated cell lines showing sensitivity to nicotinamide phosphoribosyltransferase (NAMPT) inhibitors. However, we discovered that pharmacological inhibitors often fail to phenocopy KO of matched drug targets, with only a small fraction of drugs inducing similar effects. This discrepancy reveals fundamental differences between pharmacological and genetic perturbations, emphasizing the need for approaches that directly assess the interplay of loss-of-function mutations and drug activity in cancer models. Author Summary Synthetic lethality is an emerging approach for targeting a biological dependency in cancer cells that does not harm normal cells. This strategy is particularly valuable for targeting loss-of-function mutations in tumor suppressor genes, which are more challenging to directly target. In an effort to accelerate treatments for cancer patients, we aimed to map out these dependencies and overlap them with responses to available drugs. We discovered different outcomes when a protein is targeted by a drug versus when that same target is disrupted genetically. Thus, if a drug is to be effectively repurposed as synthetic lethal agent, feasibility studies must capture drug biology, ideally by test the drug empirically in relevant cancer models. A second notable discovery is that in vitro synthetic lethal interactions involving cancer driver genes are significantly more likely to exhibit consistent patterns, such as mutual exclusivity in human tumor samples. This is important since selection of relevant cell lines is often critical in drug development to maximize potential for translation to clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c1469ac01d0657e51e2a370ba73f2735664c7f" target='_blank'>
              Challenges and opportunities for drug repurposing in cancers based on synthetic lethality induced by tumor suppressor gene mutations
              </a>
            </td>
          <td>
            Michael Vermeulen, Andrew W. Craig, Tomas Babak
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Clinical analyses demonstrated treatment naive DDR status was associated with different responses to frontline chemotherapy. Using longitudinal liquid biopsies, we found that DDR Intermediate and High tumors exhibited subtype switching and coincident emergence of heterogenous phenotypes following frontline treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes and may be associated with different chemotherapy responses and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02291-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671674f412412e8378b84a598f7d2a87a83c70ce" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2025-03-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec81235338a787d945989660c2ddefa3be4c0bd" target='_blank'>
              Novel CHI3L1-Associated Angiogenic Phenotypes Define Glioma Microenvironments: Insights From Multi-Omics Integration.
              </a>
            </td>
          <td>
            Yu-Hang Zhao, Yu-Xiang Cai, Zhi-yong Pan, F. Tang, Chao Ma, Ze-Fen Wang, Gang Li, Hang Chang, Su-Fang Tian, Zhi-Qiang Li
          </td>
          <td>2025-02-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Spatial transcriptomics is a cutting-edge technology that analyzes gene expression at the cellular level within tissues while integrating spatial location information. This concept, which combines high-plex RNA sequencing with spatial data, emerged in the early 2010s. Spatial transcriptomics has rapidly expanded with the development of technologies such as in situ hybridization, in situ sequencing, in situ spatial barcoding, and microdissection-based methods. Each technique offers advanced mapping resolution and precise spatial assessments at the single-cell level. Over the past decade, the use of spatial transcriptomics on clinical samples has enabled researchers to identify gene expressions in specific diseased foci, significantly enhancing our understanding of cellular interactions and disease processes. In the field of rheumatology, the complex and elusive pathophysiology of diseases such as rheumatoid arthritis, systemic lupus erythematosus, and Sjögren’s syndrome remains a challenge for personalized treatment. Spatial transcriptomics provides insights into how different cell populations interact within disease foci, such as the synovial tissue, kidneys, and salivary glands. This review summarizes the development of spatial transcriptomics and current insights into the pathophysiology of autoimmune rheumatic diseases, focusing on immune cell distribution and cellular interactions within tissues. We also explore the potential of spatial transcriptomics from a clinical perspective and discuss the possibilities for translating this technology to the bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95f806851975b88a84521b58227335a097ae890e" target='_blank'>
              Spatial transcriptomics in autoimmune rheumatic disease: potential clinical applications and perspectives
              </a>
            </td>
          <td>
            Atsuko Tsujii Miyamoto, Hiroshi Shimagami, Atsushi Kumanogoh, Masayuki Nishide
          </td>
          <td>2025-02-20</td>
          <td>Inflammation and Regeneration</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0761133026ec3ba173f2c421844c6435e13b010" target='_blank'>
              Real-Time Tracking of Tumor Invasion Dynamics in Organotypic Brain Slices at Subcellular Resolution
              </a>
            </td>
          <td>
            Jana B. Petr, Meng-Syuan Lin, Dina Hochuli, Andreas Hierlemann, Martin Baumgartner, Mario M. Modena
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3418175c8e687d05cd4cb3259c7f39ab58b56c25" target='_blank'>
              Dynamic responses to rejection in the transplanted human heart revealed through spatial transcriptomics
              </a>
            </td>
          <td>
            K. Amancherla, Angela M. Taravella Oill, Xavier Bledsoe, A. L. Williams, Nelson Chow, Shilin Zhao, Quanhu Sheng, David W. Bearl, Robert D. Hoffman, J. Menachem, Hasan K. Siddiqi, Douglas M. Brinkley, E. D. Mee, Niran Hadad, Vineet Agrawal, Jeffrey Schmeckpepper, A. Rali, Stacy Tsai, E. Farber-Eger, Quinn S Wells, Jane E. Freedman, Nathan R. Tucker, K. Schlendorf, Eric R. Gamazon, Ravi V. Shah, N. Banovich
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Immune cells exhibit functional heterogeneity beyond what is resolved by classical definitions of subpopulations based on cell surface expression of receptors. To develop efficient and personalized cell-based immunotherapies, we need to resolve this heterogeneity and understand the underlying parameters that dictate cellular responses to specific target cells. For this, new methods are required that can identify and harvest immune cells with specific functions, e.g., high cytotoxic potential, to form clonally expanded cells or to assess molecular or genetic signatures. In this study, we evaluate a system for non-destructive, live cell picking and release in deep, high-aspect ratio microwells and test it for isolation of individual natural killer (NK) cells. We assess its performance at retrieving and releasing beads from microwells and demonstrate its potential for single NK cell isolation with intact viability. We also implement a semi-automated workflow for functional single-cell screening of NK cell behavior in microwell arrays followed by single-cell identification and isolation, demonstrating the potential for functional screening and isolation of serial killing immune cells. Our evaluation concludes that this cell isolation system, in combination with microwell arrays, offers opportunities for improved understanding of NK cell biology with applications towards cell therapy. However, its limited throughput hinders large-scale applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba79619eb3501acae9caddc99dc83ce718f2ee3c" target='_blank'>
              Isolation of individual natural killer cells from deep, high-aspect ratio microwell arrays - an evaluative study
              </a>
            </td>
          <td>
            Quentin Verron, Niklas Sandström, Hanna van Ooijen, K. Guldevall, K. Olofsson, T. Frisk, Björn Önfelt
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Model organisms allowing for longitudinal examinations of liver disease pathogenesis are pivotal for the development of new therapeutic modalities. The eastern North American woodchuck develops chronic hepatitis and liver cancer after woodchuck hepatitis virus (WHV) infection, mirroring aspects of the natural history of the human hepatitis B virus (HBV). However, the cellular landscape of the woodchuck liver and the cell-level relevance of WHV infection to HBV infection is currently uncharacterized. Methods We employed single-cell RNA sequencing (RNA-seq) to generate an atlas of healthy woodchuck liver (63,389 cells, n=8) and peripheral blood mononuclear cells (PBMCs) (26,972 cells, n=7). Cell-specific and hepatic zonation gene signatures were validated using spatial transcriptomics (n=1). We employed our atlas to examine immune activation in stimulated precision cut-liver slices (PCLS) and disease-related pathway activation in chronic WHV infection (11,797 cells, n=3). We further employed our atlas to examine shared disease pathways between WHV infection and human HBV infection. Results Our atlas revealed woodchuck hepatic cellular diversity comparable to human and murine livers. Applying single-nucleus RNA-seq to PMA/ionomycin-stimulated precision cut liver slices revealed inflammation-associated activation signatures in T cell, myeloid and endothelial cell compartment. Finally, we describe intrahepatic T cells in chronic WHV hepatitis with both exhaustion and activation-associated signatures that resemble intrahepatic T cell genes signatures described in human chronic HBV. Conclusions We present a multi-omic atlas of healthy, diseased and ex vivo stimulated woodchuck liver. By identifying shared pathological processes between WHV and HBV infections, our findings reinforce the value of this preclinical model in translational research. This resource aims to advance studies on HBV pathogenesis and oncogenesis to speed the development of novel therapeutic strategies. Impact and Implications/Lay summary The liver plays important roles in metabolism, detoxification, and immune processes; liver transplantation is often the only treatment option for severe chronic liver diseases. Therefore, developing animal models that reflect human liver disease and can be studied throughout the disease course is crucial for the discovery of new treatment options. The woodchuck is an animal that develops chronic hepatitis and liver cancer after infection with woodchuck hepatitis virus, which models the human hepatitis type B virus infection (HBV) and associated hepatic carcinoma. However, our understanding of the cells that compose the woodchuck liver is limited, making it challenging to design and test cell-based therapeutics. In this study, we atlased the healthy and chronically infected woodchuck liver, found that liver cell types in woodchuck resemble those in humans, and employed the atlas to show similarities between WHV and HBV at the cell level, reinforcing the potential of WHV-infected woodchuck as a model for human HBV disease. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8ef9bc878aedeb55d6c6cc4171d96693eccf30b" target='_blank'>
              A Single-Cell Woodchuck Liver Atlas Identifies Healthy and Disease-related Cellular Programs Conserved in Human
              </a>
            </td>
          <td>
            Zoe A. Clarke, Jawairia Atif, Xinle Wang, Lewis Y. Liu, Lawrence Wood, Damra Camat, Sai W. Chung, Xue-Zhong Ma, Justin Manuel, Si Lok, Timothy Lau, C. Thoeni, Tomasz I. Michalak, Ian D. McGilvray, Gary D. Bader, S. Macparland
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="As spatial molecular data grow in scope and resolution, there is a pressing need to identify key spatial structures associated with disease. Current approaches often rely on hand-crafted features such as local abundances of manually annotated, discrete cell types, which may overlook important signals. Here we introduce variational inference-based microniche analysis (VIMA), a method that combines deep learning with principled statistics to discover associated spatial features with greater flexibility and precision. VIMA uses a variational autoencoder to extract numerical “fingerprints” from small tissue patches that capture their biological content. It uses these fingerprints to define a large number of “microniches” – small, potentially overlapping groups of tissue patches with highly similar biology that span multiple samples. It then uses rigorous statistics to identify microniches whose abundance correlates with case-control status. We show in simulations that VIMA is well calibrated and more powerful and accurate than other approaches. We then apply VIMA to a 140-gene spatial transcriptomics dataset in Alzheimer’s dementia, a 54-marker CO-Detection by indEXing (CODEX) dataset in ulcerative colitis (UC), and a 7-marker immunohistochemistry dataset in rheumatoid arthritis (RA), in each case recapitulating known biology and identifying novel spatial features of disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55b596129875ea6fffe7088bad79d02d94a1bbd2" target='_blank'>
              Powerful and accurate case-control analysis of spatial molecular data with deep learning-defined tissue microniches
              </a>
            </td>
          <td>
            Yakir A. Reshef, Lakshay Sood, Michelle Curtis, Laurie Rumker, Daniel J. Stein, Mukta G. Palshikar, S. Nayar, Andrew Filer, A. Jonsson, Ilya Korsunsky, Soumya Raychaudhuri
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer cell lines are valuable models for studying tumor biology, yet their genomic evolution during culture can compromise experimental reproducibility. We conducted a detailed genomic analysis of the triple-negative breast cancer cell line MDA-MB-231, examining sublines obtained from different sources, at various time points, and across distinct passages. We introduce the concept of intraline heterogeneity (ILH) to highlight the genomic variability observed among these sublines. Our analyses revealed extensive genomic diversity, including differences in single nucleotide variants (SNVs) and copy number alterations (CNAs). In particular, CNAs exhibited remarkable heterogeneity, with pronounced chromosomal gains and losses between sublines, underscoring the impact of genomic instability on ILH. These findings suggest that ILH may influence experimental outcomes, emphasizing the importance of considering passage-specific genomic characterization to ensure consistency and reliability in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7a90a295650f938dfb30675bf97324372367f9" target='_blank'>
              Intraline genomic heterogeneity of the triple-negative breast cancer MDA-MB-231 cell line
              </a>
            </td>
          <td>
            Nair Varela-Rouco, N. Estévez-Gómez, C. Fernández-Santiago, Laura Tomás, Miriam Pérez, Daniel García-Souto, J. Pasantes, R. Piñeiro, J. M. Alves, D. Posada
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59376f615ed819023304612aaccafe8e9882b1f" target='_blank'>
              Mapping T cell dynamics to molecular profiles through behavior-guided transcriptomics.
              </a>
            </td>
          <td>
            A. K. Wezenaar, U. Pandey, F. Keramati, M. Hernandez-Roca, P. Brazda, M. Barrera Román, A. Cleven, F. Karaiskaki, T. Aarts-Riemens, S. de Blank, P. Hernandez-Lopez, S. Heijhuurs, A. Alemany, J. Kuball, Z. Sebestyen, J. F. Dekkers, H. Stunnenberg, M. Alieva, A. Rios
          </td>
          <td>2025-02-10</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/520e7a98e4f661979d2c0ecb98d246afd9f267aa" target='_blank'>
              Translating genetics into tissue: inflammatory cytokine-producing TAMs and PD-L1 tumor expression as poor prognosis factors in cutaneous melanoma
              </a>
            </td>
          <td>
            Celia Barrio-Alonso, Alicia Nieto-Valle, Lucía Barandalla-Revilla, J. Avilés-Izquierdo, V. Parra-Blanco, Paloma Sánchez-Mateos, Rafael Samaniego
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eedb17874355dfd6ef34391182d578f597f1f6b" target='_blank'>
              Spatial Transcriptomic Signature of Progressive Fibrosis in Human MASLD: Role of Senescence and Metabolic Reprogramming
              </a>
            </td>
          <td>
            Hani Vu, Yuliangzi Sun, Zherui Xiong, Daniel Radford-Smith, Andrew Causer, Christian M. Nefzger, Eoin O’Sullivan, Matthew Watt, Grant Ramm, Andrew Clouston, Katharine Irvine, Quan H Nguyen, Elizabeth Powell
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is characterized by the accumulation of monoclonal mature B lymphocytes in peripheral blood, bone marrow, lymphoid tissues, and extranodal sites. Genes involved in RNA splicing such as SF3B1 and U1 are frequently mutated in CLL, leading to altered splicing and generation of tumor neoepitopes. To study the impact of these mutations on the tumor microenvironment (TME), we have developed a comprehensive single-cell atlas of unmutated CLL encompassing 26 bone marrow and lymph node tumor samples from 23 U-CLL patients with mutations in U1 (n=7), SF3B1 (n=8), or without mutations in splicing genes (n=10). We observed high intra-tumor heterogeneity, discerning 12 transcriptional programs, one linked to the U1 g.A3>C mutation and characterized by NFKB hyperactivation. T cell and NK compartments exhibited site- and mutation-specific enrichment, with increased CD4+ regulatory cells (Treg) and CD8+ exhausted cells in lymph nodes, while U1-mutant tumors showed increased CD8+ cytotoxic activity, with a predominance of effector-like CD8+ cells. Single-cell T cell receptor sequencing revealed clonotype expansion in U1-mutated tumors, particularly in CD8+ effector and exhausted cells, suggesting a neoantigen-driven immune response. Cell-to-cell interaction analysis identified CD44 as a key mediator in U1-mutated tumors, showing pro-B survival interactions as those involving MIF-CD44-CD74. Furthermore, interactions between CD80 on CLL cells and CTLA4 on Tregs and CD8+ exhausted were upregulated, reflecting an immunosuppressive phenotype associated with U1 mutated CLL. These findings highlight the complex interplay between mutations in CLL and the TME, offering novel avenues for alternative therapeutic strategies for U-CLL with mutations in U1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/338007fb60dcc4b1652c081c06c04390110d3e38" target='_blank'>
              U1 snRNA mutation reshapes tumor microenvironment in chronic lymphocytic leukemia: a role for CD44-mediated signaling
              </a>
            </td>
          <td>
            Sara López-Tamargo, Javier Fernández-Mateos, Pablo Bousquets-Muñoz, Laura Llaó-Cid, F. Nadeu, Ares M. Farran, Cristina Olivar-Fernández, Ana de la Fuente-González, Andrea Aran, Roberto Martínez-Soler, , Manel Juan, J. Martín-Subero, D. Colomer, E. Campo, A. Gutiérrez-Fernández, Xose S Puente
          </td>
          <td>2025-04-01</td>
          <td>None</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Simple Summary This study investigates metastatic melanoma through an in-depth analysis of molecular and genetic characteristics of tumor samples. Five distinct melanoma subtypes were identified based on the protein expression profiles within the tumors and their surrounding microenvironments. Our findings highlight the role of specific mutations, particularly in the BRAF gene, which can affect patient survival, with some patients showing better outcomes due to a particular immune response triggered by the mutation. Additionally, our study uncovered genetic changes in melanoma tumors that could support the development of treatments that target the immune system. The study shows that proteins linked to survival are distributed in complex patterns in both primary tumors and metastases, offering deeper insights into the molecular heterogeneity of melanoma. By combining detailed molecular analysis with clinical and tissue examination, this research provides new insights into melanoma biology and offers guidance for improving treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557557036af2f6be01a0b554ada4c910b8468881" target='_blank'>
              Proteogenomic Profiling of Treatment-Naïve Metastatic Malignant Melanoma
              </a>
            </td>
          <td>
            M. Kuras, L. Betancourt, R. Hong, L. Szadai, Jimmy Rodriguez, Peter Horvatovich, I. Pla, Jonatan Eriksson, B. Szeitz, Bartłomiej Deszcz, Charlotte Welinder, Y. Sugihara, Henrik Ekedahl, Bo Baldetorp, C. Ingvar, L. Lundgren, H. Lindberg, Henriett Oskolás, Z. Horváth, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Johan Malm, Aniel Sanchez, A. M. Szasz, Krzysztof Pawłowski, E. Wieslander, David Fenyö, I. Németh, Gyorgy Marko-Varga
          </td>
          <td>2025-02-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f69de6f588f5dc88e965e9e468e394e888395b6" target='_blank'>
              Single-cell and spatial transcriptomics reveal a tumor-associated macrophage subpopulation that mediates prostate cancer progression and metastasis.
              </a>
            </td>
          <td>
            Shenglin Mei, Hanyu Zhang, Taghreed Hirz, N. Jeffries, Yanxin Xu, Ninib Baryawno, Shulin Wu, Chin-Lee Wu, Akash Patnaik, P. Saylor, David B. Sykes, Douglas M Dahl
          </td>
          <td>2025-03-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Different patterns of immune and stromal interactions within the tumor microenvironment significantly impact tumor behavior and response to treatment. Recent research has identified distinct immune subtypes—immune desert, excluded, and inflamed—each associated with specific mechanisms that impede anti-tumor immune responses. In the era of precision medicine, accurately stratifying patients into these subtypes is essential for optimizing cancer management and personalizing immunotherapy. The research team at National Cancer Institute in Lithuania addresses critical gaps in this field by focusing on several key areas. Our first focus is on developing reliable and accessible tools for determining patient immune subtypes. We are integrating transcriptomics and histology data to create a robust subtyping pipeline, which we are validating with real-world data and streamlining for clinical use. Another aspect of our work involves creating translatable preclinical models for cancer immunotherapy through utilizing 3D cell cultures and biocompatible scaffolds. We are developing druggable syngeneic murine tumor models that accurately reflect the three immune subtypes, enhancing the relevance of preclinical studies. Using both pharmaceutical and non-pharmaceutical approaches, we are tailoring immunomodulatory interventions to address the unique characteristics of each immune subtype. Joint efforts in these research areas could generate translatable insights, contributing to more precise cancer therapies and overcoming resistance mechanisms. 
  
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8e8d206eced9c976c9ce13db22dd5e4aadd38" target='_blank'>
              Deciphering immune tumor subtypes through microenvironment profiling
              </a>
            </td>
          <td>
            A. Mlynska, Eglė Žymantaitė, Austeja Butkute, N. Dobrovolskienė, J. Krasko, Olha Karaman, Karolina Suveizdė, Beatrice Gudaite, Margarita Žvirblė, Emilija Paberalė, Nijole Matuseviciene, V. Pašukonienė
          </td>
          <td>2025-03-25</td>
          <td>South East European Journal of Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Advances in tissue labeling, imaging, and automated cell identification now enable the visualization of immune cell types in human tumors. However, a framework for analyzing spatial patterns within the tumor microenvironment (TME) is still lacking. To address this, we develop Spatiopath, a null-hypothesis framework that distinguishes statistically significant immune cell associations from random distributions. Using embedding functions to map cell contours and tumor regions, Spatiopath extends Ripley’s K function to analyze both cell-cell and cell-tumor interactions. We validate the method with synthetic simulations and apply it to multi-color images of lung tumor sections, revealing significant spatial patterns such as mast cells accumulating near T cells and the tumor epithelium. These patterns highlight differences in spatial organization, with mast cells clustering near the epithelium and T cells positioned farther away. Spatiopath enables a better understanding of immune responses and may help identify biomarkers for patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80949e1640c0e5682964239d082757c3241b697f" target='_blank'>
              Statistical analysis of spatial patterns in tumor microenvironment images
              </a>
            </td>
          <td>
            Mohamed M Benimam, V. Meas-Yedid, Suvadip Mukherjee, Astri Frafjord, Alexandre Corthay, T. Lagache, Jean-Christophe Olivo-Marin
          </td>
          <td>2025-03-31</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="BACKGROUND
The heterogeneity of tumors significantly impacts on colorectal cancer (CRC) progression. However, the influence of this heterogeneity on the spatial architecture of CRC remains largely unknown.


METHODS
Spatial transcriptomic (ST) analysis of AOM/DSS-induced colorectal cancer (CRC), integrated with single-cell RNA sequencing, generated a comprehensive spatial atlas of CRC. Pseudotime trajectory, stemness evaluation, and cell-cell communication analyses explored how CD44+ tumor cells at the leading edge remodel the tumor microenvironment (TME). In vitro experiments and immunofluorescence staining of clinical samples validated pleiotrophin (PTN) signaling in promoting cancer-associated fibroblasts (CAFs) phenotypic transition and CRC progression.


RESULTS
Our findings revealed a distinctive layered ring-like structure within CRC tissues, where CD44+ tumor cells exhibiting high stemness were positioned at the tumor's leading edge. Inflammatory CAFs (iCAFs)-like, myofibroblastic CAFs (myCAFs)-like cells and pro-tumorigenic neutrophils primarily located at the tumor edge, in proximity to CD44+ tumor cells. CD44+ tumor cells then triggered the phenotypic transition of CAFs into iCAF-like and myCAF-like cells through PTN signaling.


CONCLUSIONS
Our results provide distinctive insights into how tumor heterogeneity reshapes the TME at the leading edge of tumor, thereby promoting CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562ecaa8f1395c90f98e5b2a88e96773ab95655e" target='_blank'>
              CD44+ cells enhance pro-tumor stroma in the spatial landscape of colorectal cancer leading edge.
              </a>
            </td>
          <td>
            F. Tang, Yong-Gui Zhu, Jia Shen, Bowen Yuan, Xiang He, Yuxi Tian, Liang Weng, Lunquan Sun
          </td>
          <td>2025-03-12</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1fbf6a111cb790d6d05a256a03567072e5318c" target='_blank'>
              AI-Driven Variant Annotation for Precision Oncology in Breast Cancer
              </a>
            </td>
          <td>
            Kriti Shukla, Yue Wang, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36c2c77f99d55aa9b36b1f371ad8b8cec2aac50" target='_blank'>
              Evolutionary fingerprints of epithelial-to-mesenchymal transition.
              </a>
            </td>
          <td>
            Luigi Perelli, Li Zhang, Sarah Mangiameli, Francesca Giannese, Krishnan K. Mahadevan, F. Peng, F. Citron, Hania Khan, Courtney Le, Enrico Gurreri, F. Carbone, Andrew J. C. Russell, M. Soeung, T. N. Lam, Sebastian Lundgren, Sujay Marisetty, Cihui Zhu, Desiree Catania, Alaa M T Mohamed, Ningping Feng, J. Augustine, A. Sgambato, G. Tortora, Giulio F. Draetta, Giovanni Tonon, Andrew Futreal, Virginia Giuliani, A. Carugo, A. Viale, Michael P. Kim, T. Heffernan, Linghua Wang, Raghu Kalluri, D. Cittaro, Fei Chen, G. Genovese
          </td>
          <td>2025-03-05</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Objective Active surveillance (AS) offers a viable alternative to surgical intervention for the management of indolent papillary thyroid carcinoma (PTC), helping to minimize the incidence of unnecessary treatment. However, the broader adoption of AS is hindered by the need for more reliable diagnostic markers. This study aimed to identify the differences between indolent and progressive PTC and find new targets for biomarker development and therapeutic strategies. Methods We used single-cell RNA sequencing (scRNA-seq) to analyze cellular differences in 10 early-stage PTC tumors. Findings were validated in an additional 25 tumors using cell co-culture, migration assays, immunofluorescence staining, flow cytometry, and analysis of data from The Cancer Genome Atlas (TCGA). Results Tumor-infiltrating B cells (TIL-B), particularly germinal center B cells (GC-B), were more abundant in indolent PTC. These cells suppressed thyroid cell proliferation in both indolent and progressive cases, though indolent PTC had a higher capacity to recruit peripheral B cells. In indolent cases, TIL-B cells showed increased proliferation and formed clusters within tertiary lymphoid structures (TLS). PTPRC-CD22 interactions were identified as potential drivers of TIL-B cell proliferation. Markers linked to GC-B cells, such as LMO2, were highlighted as potential diagnostic and prognostic indicators for indolent PTC. Conclusion This study provides insights into the cellular landscape of early-stage PTC, revealing distinct tumor and immune microenvironment features in indolent and progressive cases. These findings advance the understanding of indolent PTC biology and support the development of reliable diagnostic and prognostic biomarkers. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03341-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c059a59d61bc9cea405825a6d5d7499c3788b8e" target='_blank'>
              Single-cell transcriptomics analysis reveals that the tumor-infiltrating B cells determine the indolent fate of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Chunmei Li, Pei Wang, Zhizhong Dong, Weihan Cao, Yan-Jun Su, Jian-ming Zhang, Shuyan Zhao, Zhiyuan Wang, Zi Lei, Li Shi, Ruochuan Cheng, Wen Liu
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast tumors have traditionally been categorized into four widely recognized subtypes. Recent single-cell advancements have, however, shown that individual tumors can contain more than one subtype, exhibiting their intrinsic heterogeneity. Relatedly, whilst the crucial role of cell cycle regulation in cancer development is widely acknowledged, the specific impact of cell cycle phases on oncogenic signaling and processes in each of these subtypes remains unresolved. Here, we examine the differences in transcriptomic cancer hallmarks across and within cell cycle phases and breast cancer subtypes in two large single-cell cohorts. We show that taking cell cycle phases into account identifies a broader pool of relevant cancer hallmark pathways while preserving the hallmarks found when cell cycle phases are ignored. Moreover, we identify FDA approved drugs that can target diverse hallmarks within specific phase-subtype cell combinations. This study highlights the critical role of cell cycle phases in advancing our understanding of breast cancer biology. The distinct biological mechanisms observed in each phase-subtype group suggest that targeting these hallmarks may provide a more precise and effective therapeutic strategy for breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05c59d00d73d7927897f20d50ad9f322baa1c2ce" target='_blank'>
              Decoding Cell Cycle Phase Variations in Cancer Hallmarks Across Breast Cancer Subgroups
              </a>
            </td>
          <td>
            Miguel Castresana-Aguirre, A. Matikas, Linda S. Lindström, N. Tobin
          </td>
          <td>2025-03-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="ABSTRACT Tumor suppressor genes (TSGs) are thought to suppress tumor development primarily via cancer cell-autonomous mechanisms. However, the tumor microenvironment (TME) also significantly influences tumorigenesis. In this context, a role for TSGs in the various cell types of the TME in regulating tumor growth is emerging. Indeed, expression analyses of TSGs in clinical samples, along with data from mouse models in which TSGs were deleted selectively in the TME, indicate a functional role for them in tumor development. In this Perspective, using TP53 and PTEN as examples, we posit that TSGs play a significant role in cells of the TME in regulating tumor development, and postulate both a ‘pro-active’ and ‘reactive’ model for their contribution to tumor growth, dependent on the temporal sequence of initiating events. Finally, we discuss the need to consider a 2-in-1 cancer-treatment strategy to improve the efficacy of clearance of cancer cells and the cancer-promoting TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f832937592839082aed0daa5b731f3a252c7c2" target='_blank'>
              Tumor suppressor genes in the tumor microenvironment
              </a>
            </td>
          <td>
            Bahareh Tabanifar, H. Lau, K. Sabapathy
          </td>
          <td>2025-03-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Cancer-associated fibroblast (CAF) recruitment and activation within the tumor microenvironment (TME) are increasingly acknowledged as drivers of oral squamous cell carcinoma (OSCC) tumor growth and metastasis. Therefore, the mechanisms underlying tumor cell and fibroblast crosstalk warrant further investigation. We discovered that ectopic interferon-stimulated gene 15 (ISG15) expression, which is a promising and novel oncoprotein biomarker elevated in a variety of cancers, enhanced OSCC growth and elevated collagen and α-smooth muscle actin (α-SMA) expression in ISG15-expressing tumors. Analysis of immunohistochemistry revealed high ISG15 expression in human oral tissues correlated with high expression of α-SMA and fibroblast activation protein (FAP). Fibroblast migration and recruitment by ISG15-expressing OSCC cells were confirmed by in vitro and in vivo experiments. Exogenous ISG15 induced fibroblast migration, morphological changes, and vimentin expression. Enrichment of glycolysis pathway genes, as well as increased glycolysis-related gene expression, glucose uptake, and lactate production were observed in ISG15-treated fibroblasts. Lactate release and fibroblast migration were blocked by a competitive inhibitor of glucose metabolism. Furthermore, the knockdown of integrin αL (ITGAL)/CD11a, a subunit of ISG15 receptor lymphocyte functional-associated antigen-1 (LFA-1), in immortalized fibroblasts diminished extracellular ISG15-mediated glycolysis and migration. Our findings suggest that ISG15 derived from OSCC cells interacts with fibroblasts through the LFA-1 receptor, leading to glycolytic reprogramming and promotion of fibroblast migration into the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cdbb5e2d18defd866a0ed69744c9ad3161e60de" target='_blank'>
              Tumor cell-derived ISG15 promotes fibroblast recruitment in oral squamous cell carcinoma via CD11a-dependent glycolytic reprogramming
              </a>
            </td>
          <td>
            Ssu‐Han Wang, Yu-lin Chen, Shih-Han Huang, Yu-Ke Fu, Su-Fang Lin, Shih-Sheng Jiang, Shu-Chen Liu, J. Hsiao, Jang-Yang Chang, Ya-Wen Chen
          </td>
          <td>2025-03-11</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Neoantigen vaccines are under investigation for various cancers, including epidermal growth factor receptor (EGFR)-driven lung cancers1,2. We tracked the phylogenetic history of an EGFR mutant lung cancer treated with erlotinib, osimertinib, radiotherapy and a personalized neopeptide vaccine (NPV) targeting ten somatic mutations, including EGFR exon 19 deletion (ex19del). The ex19del mutation was clonal, but is likely to have appeared after a whole-genome doubling (WGD) event. Following osimertinib and NPV treatment, loss of the ex19del mutation was identified in a progressing small-cell-transformed liver metastasis. Circulating tumour DNA analyses tracking 467 somatic variants revealed the presence of this EGFR wild-type clone before vaccination and its expansion during osimertinib/NPV therapy. Despite systemic T cell reactivity to the vaccine-targeted ex19del neoantigen, the NPV failed to halt disease progression. The liver metastasis lost vaccine-targeted neoantigens through chromosomal instability and exhibited a hostile microenvironment, characterized by limited immune infiltration, low CXCL9 and elevated M2 macrophage levels. Neoantigens arising post-WGD were more likely to be absent in the progressing liver metastasis than those occurring pre-WGD, suggesting that prioritizing pre-WGD neoantigens may improve vaccine design. Data from the TRACERx 421 cohort3 provide evidence that pre-WGD mutations better represent clonal variants, and owing to their presence at multiple copy numbers, are less likely to be lost in metastatic transition. These data highlight the power of phylogenetic disease tracking and functional T cell profiling to understand mechanisms of immune escape during combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86437480e27bf0e2c397f9c46658fa8c5c9829bf" target='_blank'>
              Hyper-responsiveness of cancer stem cells to microenvironmental cues controls metastasis and therapy response through YAP/TAZ/TEAD
              </a>
            </td>
          <td>
            Binwu Tang, Jacob Minin, Victoria Gonzalez, Zoya Khan, Yuval Raviv, Yu-an Yang, Christine Carney, Zachary Millman, Daniel Grun, Cristiana Pineda, Alina Sharma, Dominic Esposito, Hualong Yan, Jing Huang, Andy D. Tran, Michael Kruhlak, Howard H Yang, Maxwell P Lee, L. Wakefield
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Background Tumor-infiltrating lymphocytes (TILs) and tumor-associated macrophages (TAMs) have increasingly been reported to impact the brain metastatic process of solid tumors. However, data on intra-individual differences between primary tumor and brain metastasis (BM), as well as their correlation with clinical outcome parameters, is scarce. Methods We retrospectively identified patients who received resection of the primary tumor and BM between 01/1990 and 10/2022. Density quantification of TAMs (CD68+, CD163+) and TILs (CD3+, CD8+, CD45RO+, FOXP3+) was performed by immunohistochemical staining of matched tumor tissue samples. Images were processed with QuPath software and heterogeneity of generated heatmaps was measured by Shannon Entropy. Time-to-BM (TTBM) was defined as the time from diagnosis of the primary tumor until the first diagnosis of BM. Results In total, 104 patients (46.2% female; median age 57.3 years at BM diagnosis) were included: 78/104 (75%) non-small cell lung cancer, 18/104 (17%) breast cancer, 8/104 (8%) renal cell carcinomas. Densities of CD3+ (p < 0.001) and CD8+-TILs (p < 0.001) were higher in primary tumor samples, while CD68+ (p = 0.035) and CD163+-TAM densities (p < 0.001) were higher in the matched BM. Higher CD3+, CD8+-TILs and CD163+-TAMs densities in primary tumors were associated with shorter TTBM (p = 0.005, p = 0.015 and p = 0.006, respectively). Higher entropies of CD3+ (p < 0.001) and FOXP3+ (p = 0.011) TILs were observed in primary tumors compared to BM. Longer TTBM was associated with higher entropy of FOXP3+ TILs (p = 0.024) and lower entropy in CD163+ TAMs (p = 0.039). No significant associations of immune cell densities or entropies with OS after BM diagnosis were found. Discussion By utilizing a unique cohort of matched primary tumor and BM tissue samples, we could demonstrate higher TIL densities in primary tumors and higher TAM densities in BM, respectively. Higher cell densities of CD3+, CD8+-TILs and CD163+-TAMs in primary tumors were associated with shorter TTBM, while a larger difference between CD3+ and CD8+ densities between primary tumor and BM was associated with longer TTBM. These findings highlight the potential of targeting TAMs as a therapeutic strategy to mitigate the development of brain metastases. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01939-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91820cda6d428b7441857912e71a4c3a105594b5" target='_blank'>
              Density and entropy of immune cells within the tumor microenvironment of primary tumors and matched brain metastases
              </a>
            </td>
          <td>
            M. Kleinberger, D. Cifci, C. Paiato, E. Tomasich, M. Mair, A. Steindl, Zoltán Spiró, Z. I. Carrero, L. Berchtold, J. Hainfellner, L. Müllauer, G. Heller, M. Preusser, Jakob N Kather, A. Berghoff
          </td>
          <td>2025-02-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is a clinically heterogeneous malignancy with diverse patient outcomes, largely influenced by the tumor microenvironment (TME). Understanding the roles of fibroblasts and macrophages within the TME is essential for developing personalized therapeutic strategies in DLBCL. Methods This study is a multi-omics approach, integrating spatial transcriptomics (n = 11), bulk transcriptomics (n = 2,499), immunohistochemistry (IHC, n = 37), multiplex immunofluorescence (mIF, n = 56), and plasma samples (n = 240) to identify and characterize fibroblast and tumor-associated macrophage subtypes in the TME. Hub genes for LYZ+ fibroblasts and FN1+ macrophages were selected through univariate Cox regression and random forest analyses. Their prognostic significance was validated using IHC, mIF, and autoantibody assays in DLBCL patients treated with R-CHOP and in non-small cell lung cancer (NSCLC) patients receiving immune checkpoint inhibitors (ICIs). Results Fibroblasts and macrophages were classified into two distinct subtypes. Patients with higher LYZ+ fibroblasts infiltration demonstrated superior prognosis, which was associated with increased infiltration of FN1+ macrophages. Key hub genes identified for LYZ+ fibroblasts included LYZ, ANPEP, CSF3R, C15orf48, LILRB4, CLEC7A, and COL7A1, while hub FN1+ macrophages genes included COL1A1, FN1, APOE, DCN, MMP2, SPP1, COL3A1, and COL1A2. Independent prognostic markers in DLBCL treated with R-CHOP and NSCLC treated with ICIs were identified, including LYZ and LILRB4 at both protein and mRNA levels, and COL1A2 autoantibodies (p < 0.05). In DLBCL patients treated with R-CHOP, FN1 mRNA and autoantibody levels were also prognostic markers (p < 0.05). In NSCLC treated with ICIs, COL3A1 autoantibody was prognostic marker (p < 0.05). Conclusions This study identified a prognostically relevant LYZ+ fibroblasts and FN1+ macrophages in DLBCL. The hub genes associated with these subtypes represent potential biomarkers, providing insights into improving patient outcomes in DLBCL. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-03968-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2ac9b04d746f5cf36d236d7ac9a449435141cc" target='_blank'>
              Spatial transcriptomics reveals prognostically LYZ+ fibroblasts and colocalization with FN1+ macrophages in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            L. Dai, N. Lou, Liling Huang, Lin Li, Le Tang, Yuankai Shi, Xiaohong Han
          </td>
          <td>2025-02-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="T cell-based immunotherapies hold promise in treating cancer by leveraging the immune system’s recognition of cancer-specific antigens1. However, their efficacy is limited in tumours with few somatic mutations and substantial intratumoural heterogeneity2–4. Here we introduce a previously uncharacterized class of tumour-wide public neoantigens originating from RNA splicing aberrations in diverse cancer types. We identified T cell receptor clones capable of recognizing and targeting neoantigens derived from aberrant splicing in GNAS and RPL22. In cases with multi-site biopsies, we detected the tumour-wide expression of the GNAS neojunction in glioma, mesothelioma, prostate cancer and liver cancer. These neoantigens are endogenously generated and presented by tumour cells under physiologic conditions and are sufficient to trigger cancer cell eradication by neoantigen-specific CD8+ T cells. Moreover, our study highlights a role for dysregulated splicing factor expression in specific cancer types, leading to recurrent patterns of neojunction upregulation. These findings establish a molecular basis for T cell-based immunotherapies addressing the challenges of intratumoural heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Background Immunotherapy for liver cancer is used to rejuvenate tumor-infiltrating lymphocytes by modulating the immune microenvironment. Thus, early protective functions of T cell subtypes with tissue-specific residency have been studied in the tumor microenvironment (TME). We identified tumor-associated tissue-resident memory T (TA-TRM) cells in hepatocellular carcinoma (HCC) and characterized their molecular signatures. Methods We obtained single-cell RNA and single-cell TCR sequencing data from five patients with HCC. The heterogeneous characteristics of TRM cell subsets within the TME were then investigated and validated. Risk scores were calculated for survival analysis using significant core marker genes based on data from The Cancer Genome Atlas and the International Cancer Genome Consortium. The signaling pathways, trajectories, and clonal diversity of TA-TRM cells were investigated. Results We characterized two TRM clusters (CD69+ and CD103+) that expressed unique signature genes and validated their similar molecular patterns in an independent dataset. Risk scores based on core gene expression in TA-TRM cells were associated with survival in both datasets. Trajectory analysis revealed that the two lineages followed different trajectory paths with distinct marker gene expression across pseudo-time. CD103+ TA-TRM cells showed diverse clonotypes and shared clonotypes with other cell groups. Lower clonal diversity and distinct signaling interactions were observed in the recurrent than in the non-recurrent samples. The CXCL13-CXCR3 interaction between CD103+ TA-TRM and regulatory T cells was observed only in the recurrent samples. Conclusions We identified two subtypes of TA-TRM cells in HCC and demonstrated their unique molecular signatures, relevance to survival, and distinct signaling networks according to recurrence. The study findings provide a better understanding of the molecular characteristics of TA-TRM cells in HCC and potential immunotherapeutic strategies. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02070-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0116e059872731a0003137617d95a907ab32d870" target='_blank'>
              Molecular landscape of tumor-associated tissue-resident memory T cells in tumor microenvironment of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Mi-So Park, Hyeonbin Jo, Hyeree Kim, Ji Young Kim, Woong-Yang Park, Yong-Han Paik, Yeup Yoon, Wonseok Kang, Hong-Hee Won
          </td>
          <td>2025-02-11</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs), known for facilitating the progression and metastasis of colorectal cancer (CRC), have become a promising therapeutic target. However, the significant heterogeneity of CAFs and their intricate crosstalk with tumor cells present substantial challenges in the development of precise and effective therapeutic strategies. Methods Single-cell RNA sequencing (scRNA-seq) technology was used to identify various cell subtypes. Spatial transcriptomics (ST) was employed to map the spatial niches and colocalization patterns of these cell subtypes. Cell-cell interactions among these subtypes were analysed via CellChat and NicheNet software. Tumor cell invasion, migration, and proliferation were assessed through wound healing assays, transwell assays, colony formation assays, and xenograft mouse models. Results We identified a significant spatial colocalization between CTHRC1+ CAFs and a distinct subtype of malignant epithelial cells, both residing within the EMT-active spatial niche. Our results demonstrate that CTHRC1+ CAFs, as a major source of WNT5A, promote epithelial-mesenchymal transition (EMT) and enhance tumor cell invasiveness by upregulating MSLN expression in adjacent malignant epithelial cells. This signaling axis contributes significantly to CRC progression and metastasis. Conclusions Targeting the CTHRC1+ CAF-WNT5A-MSLN signaling axis presents a promising therapeutic strategy for advanced CRC patients. Our study provides new insights into the role of CAFs in CRC progression and offers potential avenues for developing targeted therapies to disrupt this pathway. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06236-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35da01c8609a7028b2e2ec26d0b8011e3cf58489" target='_blank'>
              A single-cell transcriptomic comparison between small intestinal neuroendocrine tumors and their progenitor
              </a>
            </td>
          <td>
            Fredrik Axling, Elham Barazeghi, Per Hellman, O. Norlén, S. Backman, P. Stålberg
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Peritoneal carcinomatosis is a common yet deadly manifestation of gastrointestinal cancers, with few effective treatments. To identify targetable determinants of peritoneal metastasis, we focused on appendiceal adenocarcinoma (AC), a gastrointestinal cancer that metastasizes almost exclusively to the peritoneum. Current treatments are extrapolated from colorectal cancer (CRC), yet AC has distinct genomic alterations, mucinous morphology and peritoneum restricted metastatic pattern. Further, no stable preclinical models of AC exist, limiting drug discovery and representing an unmet clinical need. We establish a first-in-class stable biobank of 16 long-term cultured AC patient-derived organoids (PDOs), including 3 matched, simultaneously resected primary AC-peritoneal carcinomatosis (AC-PC) pairs. By enriching for cancer cells, AC PDOs enable accurate genomic characterization relative to paucicellular AC tissue. We establish an organoid orthotopic intraperitoneal xenograft model that recapitulates diffuse peritoneal carcinomatosis and show that PC-organoids retain increased metastatic capacity, decreased growth factor dependency and sensitivity to standard of care chemotherapy relative to matched primary AC organoids. Single cell profiling of AC-PC pairs reveals dedifferentiation from mucinous differentiated states in primary AC into intestinal stem cell and fetal progenitor states in AC-PC, with upregulation of oncogenic signaling pathways. Through hypothesis-driven drug testing, we identify KRASMULTI-ON inhibitor RMC-7977 and Wnt-targeting tyrosine kinase inhibitor WNTinib as novel, clinically actionable strategies to target AC-PC more effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3baef141d668078d350e52fdb87357fa4d118d" target='_blank'>
              Paired primary-metastasis patient-derived organoids and mouse models identify phenotypic evolution and druggable dependencies of peritoneal metastasis from appendiceal cancer
              </a>
            </td>
          <td>
            Ahmed Mahmoud, Philip H. Choi, Christine Sukhwa, Jura Pintar, H. Walch, Nan Zhao, J. Bermeo, Sebastian K Chung, Manisha Raghavan, Samhita Bapat, Qingwen Jiang, G. Karagkounis, Julia Meredith, Michael Giarrizzo, C. Firat, A. Cercek, Michael B Foote, Nikolaus Schultz, W. Chatila, G. Nash, J. Shia, F. Sánchez-Vega, Steven M Larson, Arvin C. Dar, Neal Rosen, K. Ganesh
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Simple Summary Pancreatic cancer has a low survival rate with limited treatment options. Perineural invasion (PNI), where cancer cells infiltrate the nerves in the pancreas, is associated with a poor prognosis and local recurrence. Pancreatic cancer is one of the most painful cancers, with PNI contributing to the pain experienced by patients. However, the mechanisms involved in the cancer invasion of nerves are not completely understood. Here, we used spatial transcriptomics to study the expression of both cancer and nerve cells in microscopic regions of PNI and compared it to regions without PNI evidence to better understand the mechanisms involved in PNI. We identified novel mechanisms that may impact cancer growth, PNI and pain experienced by patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1818d7eac1f6857a9dc4b88e4ee2f868f4f36db6" target='_blank'>
              Spatial Transcriptomics Reveals Novel Mechanisms Involved in Perineural Invasion in Pancreatic Ductal Adenocarcinomas
              </a>
            </td>
          <td>
            V. Lakis, Noni L Chan, Ruth J Lyons, Nicola Blackburn, Tam Hong Nguyen, Crystal Chang, Andrew Masel, Nicholas West, Glen M. Boyle, A. Patch, Anthony J. Gill, K. Nones
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Laryngeal squamous cell carcinoma (LSCC) is a highly aggressive malignancy with a rising incidence over time. The tumor microenvironment (TME) plays a crucial role in LSCC development, yet the precise cellular characteristics of laryngeal cancer and its TME remain unclear. Here, we employed single-cell RNA sequencing analysis to uncover the heterogeneous populations of tumor and immune cells and investigate the role of the TME in LSCC. This analysis revealed significant heterogeneity among malignant cells, T cells, and macrophages. Notably, regulatory T cells were markedly increased at tumor sites, and macrophage analysis identified an increased presence of the Macrophage-C1-C1QC subset with up-regulated PPIF expression. Bulk RNA-seq further confirmed PPIF up-regulation in exosomes derived from LSCC tissues. Consistently, survival analysis indicated that high PPIF expression was associated with poor prognosis in LSCC. Further analyses suggested that PPIF up-regulation in Macrophage-C1-C1QC cells was associated with the enhancement of their anti-inflammatory phenotype and the promotion of F11R-F11R signaling with malignant cells, allowing LSCC cells to evade macrophage-mediated cytotoxicity. Our study provides new insights into the cellular dynamics of LSCC and highlights the critical role of Macrophage-C1-C1QC and PPIF in LSCC progression, offering potential therapeutic targets for treatment. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93584-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91301c190dc2f8d57f1a436734bdde8342d0e72d" target='_blank'>
              Unveiling the role of PPIF and macrophage subtypes in LSCC progression via single-cell and exosome RNA sequencing
              </a>
            </td>
          <td>
            Jianchao Wang, Chunhan Liu, Tailin Wu, Lei Tan, Jiangyan Wang, Wenliang Zhang, Yang Liu, Xudong Wei, Di Zhang
          </td>
          <td>2025-03-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ed6afc1fbea4684dd9546a568ad33c0ccc1b6d" target='_blank'>
              Mapping molecular landscapes in triple-negative breast cancer: insights from spatial transcriptomics.
              </a>
            </td>
          <td>
            F. Al-Mansour, H. Almasoudi, Ali M Albarrati
          </td>
          <td>2025-03-22</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Diabetes mellitus (DM) is a global health crisis affecting millions, with islet transplantation emerging as a promising treatment strategy to restore insulin production. This review synthesizes the current research on single-cell and spatial transcriptomics in the context of islet transplantation, highlighting their potential to revolutionize DM management. Single-cell RNA sequencing, offers a detailed look into the diversity and functionality within islet grafts, identifying specific cell types and states that influence graft acceptance and function. Spatial transcriptomics complements this by mapping gene expression within the tissue’s spatial context, crucial for understanding the microenvironment surrounding transplanted islets and their interactions with host tissues. The integration of these technologies offers a comprehensive view of cellular interactions and microenvironments, elucidating mechanisms underlying islet function, survival, and rejection. This understanding is instrumental in developing targeted therapies to enhance graft performance and patient outcomes. The review emphasizes the significance of these research avenues in informing clinical practices and improving outcomes for patients with DM through more effective islet transplantation strategies. Future research directions include the application of these technologies in personalized medicine, developmental biology, and regenerative medicine, with the potential to predict disease progression and treatment responses. Addressing ethical and technical challenges will be crucial for the successful implementation of these integrated approaches in research and clinical practice, ultimately enhancing our ability to manage DM and improve patient quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4d2a871c9156c1f2749ccafc8536877666a5a4e" target='_blank'>
              Single-cell genomics and spatial transcriptomics in islet transplantation for diabetes treatment: advancing towards personalized therapies
              </a>
            </td>
          <td>
            Lisha Mou, Tony Bowei Wang, Yuxian Chen, Ziqi Luo, Xinyu Wang, Zuhui Pu
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f8ba9e3fcdae65a1005fa8694699f4a2467c8be" target='_blank'>
              Intratumour oxidative hotspots provide a niche for cancer cell dissemination.
              </a>
            </td>
          <td>
            Yoshifumi Ueda, Shigeki Kiyonaka, L. Selfors, Keisuke Inoue, Hiroshi Harada, Tomohiro Doura, Kunishige Onuma, Makoto Uchiyama, Ryuhei Kurogi, Yuji Yamada, Jiacheng H Sun, Reiko Sakaguchi, Yuki Tado, Haruki Omatsu, Harufumi Suzuki, M. Aoun, Takahiro Nakayama, Taketoshi Kajimoto, Tetsuya Yano, Rikard Holmdahl, I. Hamachi, Masahiro Inoue, Yasuo Mori, Nobuaki Takahashi
          </td>
          <td>2025-02-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db4628f6f419e0b4bcdf2a3e6c10e9a297eda18" target='_blank'>
              Mimicking the Complexity of Solid Tumors: How Spheroids Could Advance Cancer Preclinical Transformative Approaches
              </a>
            </td>
          <td>
            Sylvia Mangani, Spyros Kremmydas, Nikos K. Karamanos
          </td>
          <td>2025-03-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mesothelioma cell heterogeneity encompasses diverse morphological and molecular characteristics observed within tumors, significantly impacting disease progression, treatment outcomes, and the development of targeted therapies. This heterogeneity has long posed challenges for accurate diagnosis and effective treatment, but understanding its complexities offers the potential for novel diagnostic modalities and therapeutic interventions. This study employed single-cell RNA sequencing (scRNA-seq) to investigate mesothelioma cell heterogeneity from various sources, including cell culture (CC), peritoneal lavage (Lav) from the tumor microenvironment, and circulating tumor cells (CTC) in murine models. Gene set enrichment analysis was used to identify distinct gene signatures for each subpopulation. The results revealed unique characteristics for mesothelioma cells depending on their origin. In the CC group, up-regulated genes were primarily involved in tumor cell cycle control, proliferation, and apoptosis. In the CTC group, up-regulated genes were associated with cancer cell stemness. The Lav group showed up-regulated genes facilitating interactions between tumor cells and the microenvironment, such as epithelial–mesenchymal transition and immune responses mediated by IFN-α and IFN-γ. Some pathways were shared among all tumor cells, suggesting the potential for transitioning between functional states under specific conditions. This may be the first study to explore circulating mesothelioma cell heterogeneity using scRNA-seq. The distinct gene signatures identified in each mesothelioma cell subpopulation likely play critical roles in tumor initiation and progression, offering potential novel targets for therapeutic intervention. These findings could help inform the development of more effective, personalized treatments for mesothelioma, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38354bc1c5ef98aa13d3c5253963e1d30d5518bf" target='_blank'>
              Mesothelioma cell heterogeneity identified by single cell RNA sequencing
              </a>
            </td>
          <td>
            Licun Wu, Zongjie Wang, Amin Zia, S. Kelley, Marc de Perrot
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e969ce26ddc70310682b3f3db6a8aee97b5128" target='_blank'>
              Integrative transcriptome profiling elucidates molecular and immunovascular characteristics of macrotrabecular hepatocellular carcinoma.
              </a>
            </td>
          <td>
            T. Taniai, S. Shimada, Y. Akiyama, Megumi Hatano, Koya Yasukawa, Yosuke Igarashi, Shu Tsukihara, Yoshiaki Tanji, K. Kodera, Kohei Okazaki, K. Haruki, Atsushi Nara, Kohei Yagi, K. Akahoshi, D. Ban, Yasuhiro Asahina, Toru Ikegami, Minoru Tanabe, Shinji Tanaka
          </td>
          <td>2025-02-26</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20f7ade1ee5b9d0bf71b29249fe64f13f03ddc98" target='_blank'>
              Characterization of single neurons reprogrammed by pancreatic cancer.
              </a>
            </td>
          <td>
            Vera Thiel, Simon Renders, Jasper Panten, Nicolas Dross, K. Bauer, D. Azorin, Vanessa Henriques, V. Vogel, C. Klein, Aino-Maija Leppä, Isabel Barriuso Ortega, J. Schwickert, I. Ourailidis, Julian Mochayedi, Jan-Philipp Mallm, Carsten Müller-Tidow, Hannah Monyer, J. Neoptolemos, T. Hackert, O. Stegle, D. T. Odom, R. Offringa, A. Stenzinger, Frank Winkler, M. Sprick, A. Trumpp
          </td>
          <td>2025-02-17</td>
          <td>Nature</td>
          <td>4</td>
          <td>86</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical Chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Increased stemness of cancer cells exacerbates radioresistance, thereby greatly limiting the efficacy of radiotherapy. In order to study the changes in cancer cell stemness during radiotherapy, we established a radioresistance model of human non-small cell lung cancer A549 cells and obtained A549 radioresistant cells (A549-RR). We sampled the cells at different time points during the modeling process and investigated the heterogeneity of each group of cells using single-cell sequencing. Cells in the early stages of fractionated irradiation were found to be significantly up-regulated in stemness, and a subpopulation of cells producing this response was screened and referred to as “radiation-induced stemness-responsive cancer cells”. They were undergoing stemness response, energy metabolism reprogramming, and progressively differentiating into cells with more diverse and malignant phenotypes in order to attenuate the killing effect of radiation. Furthermore, we demonstrated that such responses might be driven by the activation of the EGFR-Hippo signaling pathway axis, which also plays a crucial role in the development of radioresistance. Our study reveals the dynamic evolution of cell subpopulation in cancer cells during fractionated radiotherapy; the early stage of irradiation can determine the destiny of the radiation-induced stemness-responsive cancer cells. The activation of stemness-like phenotypes during the development of radioresistance is not the result of dose accumulation but occurs during the early stage of radiotherapy with relatively low-dose irradiation. The degree of the radiation-induced stemness response of cancer cells mediated by the EGFR-Hippo signaling pathway might be a potential predictor of the efficacy of radiotherapy and the development of radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6498ad18eccc13924e712513e028e4bc431ff4" target='_blank'>
              Single-Cell Sequencing Reveals the Role of Radiation-Induced Stemness-Responsive Cancer Cells in the Development of Radioresistance
              </a>
            </td>
          <td>
            Zheng Shi, Cuilan Hu, Jiadi Liu, Wei Cheng, Xiaohua Chen, Xiongxiong Liu, Yanyu Bao, Haidong Tian, Boyi Yu, Feifei Gao, Fei Ye, Xiaodong Jin, Chao Sun, Qiang Li
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Pituitary neuroendocrine tumors (PitNETs) are one of the most common intracranial tumors with diverse clinical manifestations. Current pathological classification systems rely primarily on histological hormone staining and transcription factors (TFs) expression. While effective in identifying three major lineages, molecular characteristics based on hormones and TFs lack sufficient resolution to fully capture the complex tumor heterogeneity. Transcriptional diversity by alternative splicing (AS) offered additional insight to address this challenge. Here, we perform bulk and full-length single-cell RNA sequencing to comprehensively investigate AS dysregulation across all PitNET lineages. We reveal pervasive splicing dysregulations that better depict tumor heterogeneity. Additionally, we delineate fundamental splicing heterogeneity at single-cell resolution, confirming bulk findings and refining splicing dysregulation varying among tumor cell types. Notably, we effectively distinguish the silent corticotroph subtype and define a distinct TPIT lineage subtype, which is associated with worse clinical outcomes and increased splicing abnormalities driven by altered ESRP1 expression. In conclusion, our results characterize the subtype specific AS landscape in PitNETs, enhancing the understanding of the PitNETs subtyping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d1d629c974d4b9e986ad888f6cfe60cc9b53f8" target='_blank'>
              Splicing diversity enhances the molecular classification of pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Yue Huang, Jing Guo, Xueshuai Han, Yang Zhao, Xuejing Li, Peiqi Xing, Yulou Liu, Yingxuan Sun, Song Wu, Xuan Lv, Lei Zhou, Yazhuo Zhang, Chuzhong Li, Weiyan Xie, Zhaoqi Liu
          </td>
          <td>2025-02-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive histological subtype of ovarian cancer and often presents with metastatic disease. The drivers of metastasis in HGSOC remain enigmatic. APOBEC3A (A3A), an enzyme that generates mutations across various cancers, has been proposed as a mediator of tumor heterogeneity and disease progression. However, the role of A3A in HGSOC has not been explored. We observed an association between high levels of APOBEC3-mediated mutagenesis and poor overall survival in primary HGSOC. We experimentally addressed this correlation by modeling A3A expression in HGSOC, and this resulted in increased metastatic behavior of HGSOC cells in culture and distant metastatic spread in vivo, which was dependent on catalytic activity of A3A. A3A activity in both primary and cultured HGSOC cells yielded consistent alterations in expression of epithelial-mesenchymal transition (EMT) genes resulting in hybrid EMT and mesenchymal signatures, providing a mechanism for their increased metastatic potential. Inhibition of key EMT factors TWIST1 and IL-6 resulted in mitigation of A3A-dependent metastatic phenotypes. Our findings define the prevalence of A3A mutagenesis in HGSOC and implicate A3A as a driver of HGSOC metastasis via EMT, underscoring its clinical relevance as a potential prognostic biomarker. Our study lays the groundwork for the development of targeted therapies aimed at mitigating the deleterious effect of A3A-driven EMT in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5397c063e336101c76892114dddd3f14218105fb" target='_blank'>
              APOBEC3A drives ovarian cancer metastasis by altering epithelial-mesenchymal transition
              </a>
            </td>
          <td>
            J. Devenport, Thi Tran, Brooke R. Harris, Dylan F. Fingerman, Rachel A. DeWeerd, Lojain H Elkhidir, Danielle LaVigne, Katherine C Fuh, Lulu Sun, Jeffrey J Bednarski, Ronny Drapkin, Mary M Mullen, Abby M. Green
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Drug resistance in metastatic lung cancer significantly contributes to patient mortality. This study explores the role of circulating tumor cells (CTCs), the precursors to metastasis, in driving this resistance. We aim to delineate the unique biological traits of CTC clusters in lung cancer and elucidate the mechanisms underlying their resistance to chemotherapy. Methods We used an ultralow adsorption plate to establish a CTC suspension culture system. Comparisons between adherent and suspension cultures of CTC-TJH-01 cells were made via Cell Counting Kit-8 (CCK-8), western blot, immunofluorescence, and flow cytometry assays to evaluate cell proliferation, drug resistance, and cancer stemness. The tumorigenicity, tumor growth rate, and drug resistance of the CTC clusters were assessed in nude mice. Transcriptomic and proteomic analyses were subsequently conducted to identify differentially expressed genes and proteins in CTC-TJH-01 cells cultured under adherent and suspension conditions. CDH17 gene knockdown in CTC-TJH-01 cells was achieved through RNA interference, and hematoxylin and eosin (HE) staining, immunohistochemistry, and immunofluorescence assays were used to examine the pathological status of these cells. Results CTC-TJH-01 cells in suspension formed cell clusters and exhibited decreased proliferation, tumorigenicity, and tumor growth, but increased cancer stemness and drug resistance. CDH17 protein expression was significantly upregulated in these clusters, activating the YAP/TAZ pathway. Knocking down CDH17 not only inactivated this pathway but also significantly increased cell proliferation activity and cisplatin sensitivity in CTC-TJH-01 clusters. Additionally, the tumor growth rate was correlated with cisplatin sensitivity. CDH17 knockdown notably promoted the growth of CTC-TJH-01 xenografts and enhanced their sensitivity to cisplatin, although no significant difference was observed compared with those in the control group. Conclusions The results indicate that lung CTC clusters with stem cell-like properties exhibit chemoresistance, which is linked to an activated CDH17-YAP pathway. Additionally, the effectiveness of cisplatin is primarily observed in tumors with relatively high growth rates, highlighting the connection between tumor growth and sensitivity to chemotherapy. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00696-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9926983d410efc85986d3eb1b6eeb8264046490c" target='_blank'>
              Regulating chemoresistance and cancer stemness: the CDH17-YAP pathway in distinct cellular states of lung cancer CTC clusters
              </a>
            </td>
          <td>
            Zujun Que, Dan Qi, Yun Yang, Wang Yao, Jiajun Liu, Yan Li, Yuanyuan Yu, Luyao Wang, Fangfei Li, Ge Zhang, Erxi Wu, Jianhui Tian
          </td>
          <td>2025-02-24</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>2</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f973214ce129b073da8e0d3c1c3cc399902a2e8" target='_blank'>
              Spatial single-cell proteomics landscape decodes the tumor microenvironmental ecosystem of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Libing Hong, Jie Mei, Xuqi Sun, Yifan Wu, Zhen Dong, Yuzhi Jin, Liaoliao Gao, Jinlin Cheng, Weihong Tian, Chuan Liu, Bin Li, Pingping Hu, Lulu Liu, Shan Xin, Xiaomeng Dai, Peng Zhao, Rongping Guo, Minshan Chen, Jingping Yun, Bo Lin, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-02-25</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Summary Spatial and temporal intra-tumor heterogeneity drives tumor evolution and therapy resistance. Existing visualization tools often fail to capture both dimensions simultaneously. To address this, we developed Jellyfish, a tool that integrates phylogenetic and sample trees into a single plot, providing a holistic view of tumor evolution and capturing both spatial and temporal evolution. Available as a JavaScript library and R package, Jellyfish generates interactive visualizations from tumor phylogeny and clonal composition data. We demonstrate its ability to visualize complex subclonal dynamics using data from ovarian high-grade serous carcinoma. Availability and implementation Jellyfish is freely available with MIT license at https://github.com/HautaniemiLab/jellyfish (JavaScript library) and https://github.com/HautaniemiLab/jellyfisher (R package).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0210199a3e67c966717695406003ace2c046fe" target='_blank'>
              Jellyfish: integrative visualization of spatio-temporal tumor evolution and clonal dynamics
              </a>
            </td>
          <td>
            K. Lavikka, Altti Ilari Maarala, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-02-25</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Gastrointestinal metastases are rare in patients with thyroid carcinoma (TC), and their underlying mechanisms remain unclear. Thus, in this study, we aimed to explore the spatial distribution characteristics of TCs and associated gastrointestinal metastatic cells. Methods We used spatial transcriptomics to generate an atlas that captures spatial gene expression patterns in papillary thyroid cancer (PTC), anaplastic thyroid carcinoma (ATC), ATC-associated lymphatic metastasis (ATC-LM), and rare ATC-associated gastric metastasis (ATC-GM). Results We demonstrated that tumor-specific myeloid cells with high SFRP4 expression were correlated with TC dedifferentiation and poor prognosis. Moreover, we validated their close localization to CD44+ tissue stem cells using immunofluorescence staining and spatial transcriptomics. We also demonstrated that ATC-LM and ATC-GM tissues exhibited high levels of CD44+PKHD1L1+ cells, which could serve as markers for these two pathological types. Conclusions These findings highlight the dynamic changes in cell composition, intercellular communication, and potential markers associated with TC dedifferentiation and distant metastasis. Further research based on our findings may contribute to improving diagnostic and therapeutic strategies for patients with TC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06252-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7453b499aa0f4f56b20054ed92fb6b1a0f0d81c" target='_blank'>
              Analysis of thyroid carcinoma composition and spatial architecture in the progression of dedifferentiation, lymphatic metastasis, and gastric metastasis
              </a>
            </td>
          <td>
            Di Wang, Ruichun Lu, Fenglian Yan, Yansong Lin, Hao Wang, Huabao Xiong
          </td>
          <td>2025-02-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most prevalent malignant brain tumor with poor prognosis. Although chromatin intratumoral heterogeneity is a characteristic feature of GBM, most current studies are conducted at a single tumor site. To investigate the GBM-specific 3D genome organization and its heterogeneity, we conducted Hi-C experiments in 21 GBM samples from nine patients, along with three normal brain samples. We identified genome subcompartmentalization and chromatin interactions specific to GBM, as well as extensive intertumoral and intratumoral heterogeneity at these levels. We identified copy number variants (CNVs) and structural variations (SVs) and demonstrated how they disrupted 3D genome structures. SVs could not only induce enhancer hijacking but also cause the loss of enhancers to the same gene, both of which contributed to gene dysregulation. Our findings provide insights into the GBM-specific 3D genome organization and the intratumoral heterogeneity of this organization and open avenues for understanding this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c40cfec55257420ce8df49630fbce7033db9c8" target='_blank'>
              Spatial 3D genome organization reveals intratumor heterogeneity in primary glioblastoma samples
              </a>
            </td>
          <td>
            Qixuan Wang, Juan Wang, Radhika Mathur, Mark Youngblood, Q. Jin, Ye Hou, L. Stasiak, Yu Luan, Hengqiang Zhao, Stephanie Hilz, Chibo Hong, Susan M Chang, Janine M Lupo, Joanna J Phillips, Joseph F Costello, Feng Yue
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is one of the most aggressive and metastatic malignancies worldwide. Migrating cancer stem cells (miCSCs) marked by CD133+CXCR4+ expression drives metastasis but lacks effective drug targets. Here, we show that activated pancreatic stellate cells secrete the CXCR4 ligand CXCL12 to foster stemness, epithelial-to-mesenchymal transition (EMT), and chemoresistance. Protein interaction network analyses links CXCL12/CXCR4 signaling axis and the downstream transcription factor BMI1. Knockdown experiments confirmed the BMI1’s role in (mi)CSCs maintenance and survival. Novel CXCR4 inhibitors, i.e., the endogenous human peptide EPI-X4 and its derivatives (e.g., JM#21) strongly inhibited the in vitro migration of miCSCs. In particular, the most potent EPI-X4 derivate JM#21 sufficiently suppressed EMT, stemness, and self-renewal of human PDAC cell lines. In addition, JM#21 sensitized cell lines towards gemcitabine and paclitaxel. Overall, our study reveals that (mi)CSCs are enhanced and maintained via a tumor-stroma crosstalk through BMI1, ultimately promoting metastases and therapeutic resistance in PDAC. Peptide targeting of the CXCL12/CXCR4/BMI1 signaling axis via JM#21 could enhance PDAC combination therapies, offering a promising strategy against this deadly cancer. Synopsis The study identifies a tumor-stroma interaction mediated by pancreatic stellate cells (PSCs) secreting CXCL12, which binds to CXCR4 on (mi)CSCs, fostering stemness, epithelial-to-mesenchymal transition (EMT), and chemoresistance. The CXCL12/CXCR4 axis activates the downstream BMI1 transcription factor, crucial for migration and stemness maintenance. CXCL12 enhances (mi)CSC populations and metastatic potential through CXCR4 signaling. BMI1 is identified as a pivotal downstream effector linking CXCR4 to EMT and stemness. JM#21 effectively blocks CXCL12-induced migration, EMT, and stemness in vitro, demonstrating superior efficacy compared to other CXCR4 inhibitors. Encapsulation of JM#21 in silica nanoparticles enhances its stability and delivery, reducing chemoresistance and miCSC populations in co-culture systems. Combining JM#21 with chemotherapy significantly impairs colony formation and CSC-mediated drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a50f37499c2309886f9000498e4fa41f0b8fba" target='_blank'>
              Novel peptide targeting CXCR4 disrupt tumor-stroma crosstalk to eliminate migrating cancer stem cells
              </a>
            </td>
          <td>
            K. Tiwary, Anton Lahusen, S. Inaas, Bastian Beitzinger, R. Schmid, M. Harms, Stefanie Hauff, F. Arnold, K. Walter, S. Alcalá, Stephan Hahn, Elisabeth Heßmann, Alexander Kleger, N. Azoitei, Thomas Seufferlein, B. Sainz, Jan Münch, Mika Lindén, P. Hermann
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Genomic alterations are the driving force behind pancreatic cancer (PC) tumorigenesis, but they do not fully account for its diverse phenotypes. Investigating the epigenetic landscapes of PC offers a more comprehensive understanding and could identify targeted therapies that enhance patient survival. In this study, we have developed a new promising methodology of spatial epigenomics that integrates multiplexed molecular imaging with convolutional neural networks. Then, we used it to map epigenetic modification levels in the six most prevalent PC subtypes. We analyzed and semi-quantified the resulting molecular data, revealing significant variability in their epigenomes. DNA and histone modifications, specifically methylation and acetylation, were investigated. Using the same technique, we examined DNA conformational changes to further elucidate the transcriptional regulatory mechanisms involved in PC differentiation. Our results revealed that the foamy-gland and squamous-differentiated subtypes exhibited significantly increased global levels of epigenetic modifications and elevated Z-DNA ratios. Overall, our findings may suggest a potentially reduced efficacy of therapeutics targeting epigenetic regulators for these subtypes. Conversely, the conventional ductal PC subtype has emerged as a promising candidate for treatment with epigenetic modulators. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-90087-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e79fb6328397b7897b70522e3b8824a0291600" target='_blank'>
              Spatial recognition and semi-quantification of epigenetic events in pancreatic cancer subtypes with multiplexed molecular imaging and machine learning
              </a>
            </td>
          <td>
            Krzysztof Szymoński, Natalia Janiszewska, K. Sofińska, Katarzyna Skirlińska-Nosek, Dawid Lupa, Michał Czaja, Marta Urbańska, Katarzyna Jurkowska, Kamila Konik, Marta Olszewska, Dariusz Adamek, Kamil Awsiuk, E. Lipiec
          </td>
          <td>2025-02-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Single-cell proteomics is an emerging field with significant potential to characterize heterogeneity within biological tissues. It offers complementary insights to single-cell transcriptomics by revealing unbiased proteomic changes downstream of the transcriptome. Recent advancements have focused on enhancing proteome coverage and depth, mostly in cultured cell lines, and a few recent studies have explored the potential of analyzing tissue micro-samples but were limited to homogenous peripheral tissues. In this current work, we utilize the power of spatial single cell-proteomics through immunostaining-guided laser capture microdissection (LCM) coupled with LC-MS to investigate the heterogenous central nervous system. We used this method to compare neuronal populations from cortex and substantia nigra, two brain regions associated with motor and cognitive function and various neurological disorders. Moreover, we used the technique to understand the neuroimmune changes associated with stab wound injury. Finally, we focus our application on the peripheral nervous system, where we compare the proteome of the myenteric plexus cell ganglion to the nerve bundle. This study demonstrates the utility of spatial single-cell proteomics in neuroscience research toward understanding fundamental biology and the molecular drivers of neurological conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e060f62a68156851b45076f66f6e5883e7b7f674" target='_blank'>
              Molecularly-guided spatial proteomics captures single-cell identity and heterogeneity of the nervous system
              </a>
            </td>
          <td>
            Sayan Dutta, Marion Pang, G. Coughlin, Sirisha Gudavalli, Michael L Roukes, Tsui-Fen Chou, V. Gradinaru
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Crohn’s disease (CD) involves a complex intestinal microenvironment driven by chronic inflammation. While single-cell RNA sequencing has provided valuable insights into this biology, the spatial context is lost during single-cell preparation of mucosal biopsies. To deepen our understanding of the distinct inflammatory signatures of CD and overcome the limitations of single-cell RNA sequencing, we combined spatial transcriptomics of frozen CD surgical tissue sections with single-cell transcriptomics of ileal CD mucosa. Coexpressed genes and cell-cell communication from single-cell analyses and factorized genes from spatial transcriptomics revealed overlapping pathways affected in inflamed CD, like antigen presentation, phagosome activity, cell adhesion, and extracellular matrix. Within the pathways, early epithelial cells showed evidence of significant changes in gene expression and subtype composition, while spatial mapping revealed the location of the events, particularly antigen presentation from epithelial cells in the base of the crypt. Furthermore, we identified early epithelial cells as a potential mediator of the MHC class II pathway during inflammation, which we validated by spatial transcriptomics cell subtype deconvolution. Therefore, the inflammation from CD appears to change the types of interactions detectable between epithelial cells with immune and mesenchymal cells, likely promoting the conditions for more macrophage infiltration into these inflammatory microdomains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00594471c2ef5f7cea997fb7b198a677a1e23c9" target='_blank'>
              Altered inflammatory mucosal signatures within their spatial and cellular context during active ileal Crohn’s disease
              </a>
            </td>
          <td>
            Vasantha L. Kolachala, S. Maddipatla, Shanta Murthy, Yeonjoo Hwang, A. Dodd, Garima Sharma, Sachith Munasinghe, R. Pelia, S. Venkateswaran, M. Anbazhagan, Tarun Koti, Navdeep Jhita, Gaurav N Joshi, Chrissy A. Lopez, Duke Geem, Hong Yin, David J. Cutler, Peng Qiu, J. Matthews, S. Kugathasan
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Despite significant advancements in identifying novel therapeutic targets and compounds, cancer stem cells (CSCs) remain pivotal in driving therapeutic resistance and tumor progression in gastric cancer (GC). High-resolution knowledge of the transcriptional programs underlying the role of CSC niche in driving tumor stemness and progression is still lacking. Herein, spatial and single-cell RNA sequencing of 32 human gastric mucosa tissues at various stages of malignancy, illuminating the phenotypic plasticity of tumor epithelium and transcriptional trajectory from mature gastric chief cells to the CSC state, which is associated with activation of EGFR and WNT signaling pathways, is conducted. Moreover, the CSCs interact with not only the immunosuppressive CXCL13+ T cells and CCL18+ M2 macrophages to evade immune surveillance, but also the inflammatory cancer-associated fibroblasts (iCAFs) to promote tumorigenesis and maintain stemness, which construct the CSC niche leading to inferior prognosis. Notably, it is uncovered that amphiregulin (AREG) derived from iCAFs promotes tumor stemness by upregulating the expression of SOX9 in tumor cells, and contributes to drug resistance via the AREG-ERBB2 axis. This study provides valuable insight into the characteristics of CSC niche in driving tumor stemness and progression, offering novel perspective for designing effective strategies to overcome GC therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d6cf84d15c0f17d3af8a79006d512aec0aa37" target='_blank'>
              Dissecting the Spatial and Single-Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer.
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinglong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb3b0d5b454cf059f699fd40f05952bb5941c97" target='_blank'>
              Unveiling spatial heterogeneity in medulloblastoma: a multi-omics analysis of cellular state and geographical organization.
              </a>
            </td>
          <td>
            Jiankang Li, Hailong Liu, Ziwei Wang, Jiao Zhang, Xuan Chen, Craig Daniels, Xiaochong Wu, Olivier Saulnier, Hiromichi Suzuki, Pasqualino de Antonellis, Alexandra Rasnitsyn, Winnie W. Ong, Evan Y Wang, Liam D Hendrikse, Yu Su, Yu Tian, Dongming Han, Ruohan Wang, Jialin Mo, Fei Liu, Kaiwen Deng, Dongyang Wang, Zhaoyang Feng, Yifei Jiang, Yanong Li, Yuting Ma, Zijia Liu, Meiyu Li, Peiyi Tian, Yanfeng Shi, Yong Jiang, Tao Yang, Shouwei Li, Jianfeng Liang, Jingchuan Wu, Ying Wang, Wanjing Zou, Yina Jiang, Lusheng Wang, Fang Chen, Xin Jin, Shuaicheng Li, Xiaoguang Qiu, Chunde Li, Ya Gao, Yujie Tang, Michael D. Taylor, Tao Jiang
          </td>
          <td>2025-02-17</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a494e730aa784cad617baf97d8efcc5bc5c5345" target='_blank'>
              Cell-type-specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are key players in cancer development and therapy, and they exhibit multifaceted roles in the tumor microenvironment (TME). From their diverse cellular origins, CAFs undergo phenotypic and functional transformation upon interacting with tumor cells and their presence can adversely influence treatment outcomes and the severity of the cancer. Emerging evidence from single-cell RNA sequencing (scRNA-seq) studies have highlighted the heterogeneity and plasticity of CAFs, with subtypes identifiable through distinct gene expression profiles and functional properties. CAFs influence cancer development through multiple mechanisms, including regulation of extracellular matrix (ECM) remodeling, direct promotion of tumor growth through provision of metabolic support, promoting epithelial-mesenchymal transition (EMT) to enhance cancer invasiveness and growth, as well as stimulating cancer stem cell properties within the tumor. Moreover, CAFs can induce an immunosuppressive TME and contribute to therapeutic resistance. In this review, we summarize the fundamental knowledge and recent advances regarding CAFs, focusing on their sophisticated roles in cancer development and potential as therapeutic targets. We discuss various strategies to target CAFs, including ECM modulation, direct elimination, interruption of CAF-TME crosstalk, and CAF normalization, as approaches to developing more effective treatments. An improved understanding of the complex interplay between CAFs and TME is crucial for developing new and effective targeted therapies for cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a333ac0501e7572a5af1f1975c3f74e5d8b822" target='_blank'>
              Cancer associated fibroblasts in cancer development and therapy
              </a>
            </td>
          <td>
            Hongyuan Jia, Xingmin Chen, Linling Zhang, Meihua Chen
          </td>
          <td>2025-03-28</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4955ba693d4e35ecf04dfaea41f95d050a589d61" target='_blank'>
              Abstract A029: Highly-resolved integrated single cell atlas of human breast cancer enables hypothesis-driven analysis of tumor heterogeneity
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, Stefano Monti
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ed63792a7cb4d7298d3e8340ecc0d2e80e3929" target='_blank'>
              EHE cell cultures: a platform for mechanistic and therapeutic investigation
              </a>
            </td>
          <td>
            Nicholas Scalora, Gillian DeWane, Yuliia Drebot, Ali A. Khan, Souradip Sinha, Krishnendu Ghosh, Denise Robinson, Patricia Cogswell, Andrew M. Bellizzi, Anthony N. Snow, Patrick Breheny, M. S. Chimenti, M. Tanas
          </td>
          <td>2025-03-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0fc7e6f9b6372a2eebc4b0d3580ed466ef20424" target='_blank'>
              A Tc1- and Th1-T-lymphocyte-rich tumor microenvironment is a hallmark of MSI colorectal cancer.
              </a>
            </td>
          <td>
            Zhihao Huang, T. Mandelkow, N. F. Debatin, M. Lurati, Julia Ebner, J. Raedler, E. Bady, Jan H. Müller, R. Simon, E. Vettorazzi, A. Menz, Katharina Möller, N. Gorbokon, G. Sauter, M. Lennartz, A. Luebke, D. Höflmayer, T. Krech, P. Lebok, C. Fraune, A. Hinsch, Frank Jacobsen, A. Marx, S. Steurer, S. Minner, D. Dum, S. Weidemann, C. Bernreuther, T. Clauditz, E. Burandt, N. Blessin
          </td>
          <td>2025-04-03</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) are a small proportion of stem-like breast cancer cells with more tumorigenic and metastatic capacity in comparison to the bulk tumor cells, which are capable of self-renewal and generating differentiated cells. BCSCs are resistant to chemotherapy and radiotherapy. More and more research indicate that BCSCs are heterogeneous and have plasticity, which refer to their ability to switch between different subtypes of stem-like cells and differentiated cells. There is some evidence showing that BCSC heterogeneity and plasticity plays a role on therapeutic resistance and metastasis of breast cancer. In this review, we discussed the roles of extrinsic (rewired tumor microenvironment, including hypoxia, microbiota) and intrinsic (pro-tumor signaling) factors on regulating BCSC heterogeneity and plasticity, leading to various malignant behaviors of BCSCs, including epithelial-mesenchymal transition (EMT)/mesenchymal-epithelial transition (MET), immune evasion, vasculogenesis and so on. Moreover, we also discussed the potential therapeutic strategies to target BCSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1099ecb1edc8ebafead4e021333de49587a487" target='_blank'>
              Breast Cancer Stem Cell Heterogeneity, Plasticity and Treatment Strategies
              </a>
            </td>
          <td>
            Mengting Chen, Suling Liu
          </td>
          <td>2025-03-06</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5411691311f3ea382bc8274891edc9e740ebfb77" target='_blank'>
              Non-invasive approach for endoluminal biopsy coupled with single-cell proteomics allows for immune characterization of intracranial aneurysms
              </a>
            </td>
          <td>
            J. Antonios, B. Gultekin, B. Theriault, K. Yalcin, D. Miyagishima, N. Adenu-Mensah, N. Sujijantarat, A. Koo, J. Haynes, P. Cedeno, M. Johnson, R. Hebert, C. Matouk, T. Barak, M. Gunel
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Primary human pancreatic ductal organoids (HPDO) have emerged as a model to study pancreas biology and disease. Yet, donor material availability, and a lack of extensive benchmarking limits the range of applications. To address this gap, we established porcine pancreatic ductal organoids (PPDO) as a system from an easily obtainable source to model pancreatic ductal/progenitor biology. We benchmarked PPDO to HPDO and primary porcine pancreas using single-cell RNA sequencing (scRNA-Seq). We observed no overt phenotypic differences in PPDO derived from distinct developmental stages, with a WNT signaling enriched population characterizing PPDO. PPDO exhibited differentiation potential towards mature ductal cells and limited potential towards endocrine lineages. We used PPDO as a platform to assess the safety of FDA-approved drugs and showed conserved toxicity of statins and α-adrenergic receptor inhibitors between PPDO and HPDO cultures. Overall, our results highlight the PPDO as a model for mammalian duct/progenitor applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f85a912fd157c576d625370dcc942795708cb4b1" target='_blank'>
              Benchmarking porcine pancreatic ductal organoids for drug screening applications
              </a>
            </td>
          <td>
            Christos Karampelias, Kaiyuan Yang, Michael Sterr, Mireia Molina van Den Bosch, Simone Renner, Janina Fuss, Sören Franzenburg, Tatsuya Kin, E. Wolf, E. Kemter, H. Lickert
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Over 80% of children with acute lymphoblastic leukemia (pALL) can be cured by treating them with multiple chemotherapeutic agents administered over several years, whereas pALL is incurable with 1-3 medications, suggesting significant variation in drug susceptibility across clonal populations. While bulk sequencing studies indicate that pALL cells contain relatively few genetic variants compared to other cancers, the true extent of genetic diversity at the single-cell level remains unknown. Here, we used three complementary approaches to investigate pALL genetic heterogeneity: error-corrected bulk sequencing, single-cell exome sequencing, and primary template-directed amplification (PTA)-enabled single-cell genome sequencing. We discovered that some ETV6-RUNX1 samples harbor multiple independent ras clones and that individual pALL cells harbor substantially more mutations (mean 3,553 per cell) than detected in bulk samples (mean 965 mutations), with variant signatures suggesting both early and late APOBEC-driven mutagenesis in ETV6-RUNX1 patients. Using PTA-based phylogenetic analysis of over 150 single-cell genomes from four pALL patients, we identified heritable phenotypes associated with specific genetic alterations, including some low-frequency clones that are preferentially selected for during chemotherapy treatment. Our findings reveal previously undetected genetic diversity in pALL and suggest that pre-existing mutations influence treatment response, with implications for future therapeutic strategies. This study provides a high-resolution framework for understanding cancer clonal evolution during treatment, yielding important new insights for developing more effective therapeutic approaches for pALL. Key Points Increasing the accuracy and resolution of sequencing from bulk to error-corrected bulk to single clone to single-cell sequencing reveals increasing levels of genetic diversity in pALL, including the presence of clone-specific driver and treatment resistance-associated mutations. Using PTA-enabled scWGS, single pALL cells contain several fold more mutations per cell than detected in bulk sequencing, which, when multiplied across billions of cells, reveal hidden population-scale genetic complexity. Some phenotypes in pALL are heritable, including treatment resistance, where we identify pre-existing rare clones with relapse-associated mutations that preferentially survive four weeks of standard treatment in patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cfebdd69571f854dae1eb84acf1f1884a093ad9" target='_blank'>
              Measuring Longitudinal Genome-wide Clonal Evolution of Pediatric Acute Lymphoblastic Leukemia at Single-Cell Resolution
              </a>
            </td>
          <td>
            Yakun Pang, Tamara Prieto, V. Gonzalez-Pena, Athena Aragon, Yuntao Xia, Sheng Kao, Sri Rajagopalan, John Zinno, Jean Quentin, Julien Laval, Dennis Yuan, Nathaniel D. Omans, David Klein, Matthew MacKay, I. De Vlaminck, John Easton, William Evans, D. Landau, Charles Gawad
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Single-cell image analysis is crucial for studying drug effects on cellular morphology and phenotypic changes. Most studies focus on single cell types, overlooking the complexity of cellular interactions. Here, we establish an analysis pipeline to extract phenotypic features of cancer cells cultured with fibroblasts. Using high-content imaging, we analyze an oncology drug library across five cancer and fibroblast cell line co-culture combinations, generating 61,440 images and ∼170 million single-cell objects. Traditional phenotyping with CellProfiler achieves an average enrichment score of 62.6% for mechanisms of action, while pre-trained neural networks (EfficientNetB0 and MobileNetV2) reach 61.0% and 62.0%, respectively. Variability in enrichment scores may reflect the use of multiple drug concentrations since not all induce significant morphological changes, as well as the cellular and genetic context of the treatment. Our study highlights nuanced drug-induced phenotypic variations and underscores the morphological heterogeneity of ovarian cancer cell lines and their response to complex co-culture environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4573d60560710e744b74b82ab85a673cb84bfba" target='_blank'>
              Evaluating feature extraction in ovarian cancer cell line co-cultures using deep neural networks
              </a>
            </td>
          <td>
            Osheen Sharma, Greta Gudoityte, Rezan Minozada, Olli P. Kallioniemi, Riku Turkki, L. Paavolainen, B. Seashore-Ludlow
          </td>
          <td>2025-02-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Background Glioma, characterized by its cellular and molecular heterogeneity, presents formidable challenges in treatment strategy and prognostic assessment. The tumor microenvironment (TME) profoundly influences tumor behavior and treatment response, with tumor‐associated neutrophils (TANs) playing a complex but understudied role. This study aimed to investigate the heterogeneity and role of TANs in glioma and to develop a prognostic model. Methods Analysis of scRNA‐seq data identified cellular subpopulations and differentially expressed neutrophil‐related genes (DE‐NRGs). Bulk RNA‐seq was obtained from four independent datasets. Molecular subtypes of glioma samples were determined by consensus clustering. WGCNA was conducted to elucidate the association between gene modules and subtypes. We developed a risk score model. Expression of selected genes was confirmed using immunohistochemistry (IHC). In vitro experiments were also performed for functional verification, including CCK8, EdU, Transwell, and TUNEL assays. Results A total of 108 DE‐NRGs for TANs were identified based on scRNA‐seq data. Two molecular subtypes were characterized, showing significant differences in prognosis and clinical features. Immune‐related analyses demonstrated varied immunological characteristics between subtypes. The risk score model was constructed with 7 genes, including AEBP1, CAVIN1, DCTD, DEPP1, DUSP6, FKBP9, and UGCG. It showed significant prognostic value and was validated across three external datasets. The mutation landscape highlighted higher IDH mutation prevalence in low‐risk groups. Drug sensitivity analysis indicated TMZ resistance in high‐risk groups. In vitro experiments showed that UGCG could promote glioma cell proliferation, migration, and invasion, while decreasing apoptosis. Conclusion This study explored the heterogeneity of TANs and developed a prognostic model, providing insights for prognostic prediction and guiding personalized treatment strategies in glioma. Declaration of Generative AI in Scientific Writing: The authors declare nonuse of generative AI and AI‐assisted technologies in the writing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a8af983a6227909234b80a95e84b835146d4d4" target='_blank'>
              Integrated Analysis to Reveal Heterogeneity of Tumor‐Associated Neutrophils in Glioma
              </a>
            </td>
          <td>
            Wen Wang, Junsheng Li, Qiheng He, Chenglong Liu, Siyu Wang, Zhiyao Zheng, Bojian Zhang, Siqi Mou, Wei Sun, Jizong Zhao
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, Monia Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, Chao (Agnes) Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71083b48d8d0e95bc226f7370321af11b26e25b" target='_blank'>
              Spatial and single-cell transcriptomic analysis reveals fibroblasts dependent immune environment in colorectal cancer.
              </a>
            </td>
          <td>
            Hang Jia, Xiang Liu, Guimin Wang, Yue Yu, Ning Wang, Tianshuai Zhang, Liqiang Hao, Wei Zhang, Guanyu Yu
          </td>
          <td>2025-03-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Metastasis is a leading cause of cancer-related deaths, with the liver being the most frequent site of metastasis in colorectal cancer. Previous studies have predominantly focused on the influence of the primary tumor itself on metastasis, with relatively limited research examining the changes within target organs. Methods Using an orthotopic mouse model of colorectal cancer, single-cell sequencing was employed to profile the transcriptomic landscape of pre-metastatic and metastatic livers. The analysis focused on identifying cellular and molecular changes within the hepatic microenvironment, with particular emphasis on inflammatory pathways and immune cell populations. Results A neutrophil subpopulation with high Prok2 expression was identified, showing elevated levels in the pre-metastatic and metastatic liver. Increased infiltration of Prok2⁺ neutrophils correlated with poor prognosis in liver metastatic colorectal cancer patients. In the liver metastatic niche (MN), these neutrophils showed high App and Cd274 (PD-L1) expression, suppressing macrophage phagocytosis and promoting T-cell exhaustion. Conclusion A Prok2⁺ neutrophil subpopulation infiltrated both pre-metastatic and macro-metastatic liver environments, potentially driving immunosuppression through macrophage inhibition and T-cell exhaustion. Targeting Prok2⁺ neutrophils could represent a novel therapeutic strategy for preventing liver metastasis in colorectal cancer patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06351-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ae7b5dd668485b7f400a3f84e79d9d667bb33c" target='_blank'>
              Single-cell transcriptomic analysis reveals dynamic changes in the liver microenvironment during colorectal cancer metastatic progression
              </a>
            </td>
          <td>
            Yue Jiang, Guojie Long, Xiaoming Huang, Wenyu Wang, B. Cheng, Weidong Pan
          </td>
          <td>2025-03-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="We identified in human breast cancer a scarce population of Tumor-Associated Monocytes (TAMs) endowed with a pro-metastatic activity and associated with reduced distant-metastasis free survival of patients. We developed a novel framework combining computational and experimental methodologies to dampen TAM pro-metastatic activity. Multi-modal experimental data from TAMs exposed in vitro to a series of perturbations were collected to build a Boolean dynamical network of TAMs. This framework successfully identified the biological pathways underlying TAM pro-metastatic activity and predicted potent pharmacological interventions that inhibited the pro-metastatic activity of TAMs isolated from patient tumors. This study showcases the power of integrating computational predictions with experimental validation in identifying new therapeutic avenues that can be extended to other cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b7bb8f484b312f30dfdbd661789ddfd12b48726" target='_blank'>
              A combined computational and experimental approach to successfully predict the behavior of metastatic tumor-associated macrophages in human breast cancer
              </a>
            </td>
          <td>
            Amel Bekkar, A. Pabois, Isaac Crespo, Riccardo Turrini, Ioannis Xenarios, M. Doucey
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162d6ff6e3cedba6aedd63bc146790643b33531d" target='_blank'>
              Quantitative characterization of tissue states using multiomics and ecological spatial analysis.
              </a>
            </td>
          <td>
            Daisy Yi Ding, Zeyu Tang, Bokai Zhu, Hongyu Ren, Alex K. Shalek, Robert Tibshirani, GP Nolan
          </td>
          <td>2025-04-01</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a279852f8f3e4dea6bf2c322ab3c789c81b3e6c" target='_blank'>
              How cancer emerges: Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-02-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer is a manifestation of dysfunctional cell states. It emerges from an interplay of intrinsic and extrinsic factors that disrupt cellular dynamics, including genetic and epigenetic alterations, as well as the tumor microenvironment. This complexity can make it challenging to infer molecular causes for treating the disease. This may be addressed by system-wide computer models of cells, as they allow rapid generation and testing of hypotheses that would be too slow or impossible to perform in the laboratory and clinic. However, so far, such models have been impeded by both experimental and computational limitations. In this perspective, we argue that they can now be achieved using deep learning algorithms to integrate omics data and prior knowledge of molecular networks. Such models would have many applications in precision oncology, e.g., for identifying drug targets and biomarkers, predicting resistance mechanisms and toxicity effects of drugs, or simulating cell-cell interactions in the microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8844ef9ba3f77a8686c18b6b08bd0f64a9f348" target='_blank'>
              Towards an interpretable deep learning model of cancer
              </a>
            </td>
          <td>
            Avlant Nilsson, N. Meimetis, Douglas A. Lauffenburger
          </td>
          <td>2025-02-14</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfff83a3b2e44394201ebb9147dbd5e27d9e3ca" target='_blank'>
              Single-Cell Transcriptomic Analysis Reveals an Aggressive Basal-Like Tumor Cell Subpopulation Associated With Poor Prognosis in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Chang-Pin Liao, Yuting Zhang, Jing Yang, Shuo Wang, Zhijuan Li, Shuling Chen, Yubin Xie, Lixia Xu, Sui Peng, Xuezhen Zeng, Ming Kuang, Bangde Xiang, Kaiyu Sun, Xiao Zhao
          </td>
          <td>2025-02-24</td>
          <td>Journal of gastroenterology and hepatology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The tumor microenvironment (TME) encompasses various cell types, blood and lymphatic vessels, and noncellular constituents like extracellular matrix (ECM) and cytokines. These intricate interactions between cellular and noncellular components contribute to the development of a malignant TME, such as immunosuppressive, desmoplastic, angiogenic conditions, and the formation of a niche for cancer stem cells, but there is limited understanding of the specific subtypes of stromal cells involved in this process. Here, we utilized p16-CreERT2-tdTomato mouse models to investigate the signaling networks established by senescent cancer stromal cells, contributing to the development of a malignant TME. In pancreatic ductal adenocarcinoma (PDAC) allograft models, these senescent cells were found to promote cancer fibrosis, enhance angiogenesis, and suppress cancer immune surveillance. Notably, the selective elimination of senescent cancer stromal cells improves the malignant TME, subsequently reducing tumor progression in PDAC. This highlights the antitumor efficacy of senolytic treatment alone and its synergistic effect when combined with conventional chemotherapy. Taken together, our findings suggest that the signaling crosstalk among senescent cancer stromal cells plays a key role in the progression of PDAC and may be a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06c45a25e4aa78209216dcd2827d3d116f51fb9d" target='_blank'>
              Signaling networks in cancer stromal senescent cells establish malignant microenvironment.
              </a>
            </td>
          <td>
            Yue Zhang, Teh-Wei Wang, Maho Tamatani, Xinyi Zeng, Lindo Nakamura, Satotaka Omori, Kiyoshi Yamaguchi, Seira Hatakeyama, Eigo Shimizu, Satoshi Yamazaki, Yoichi Furukawa, S. Imoto, Yoshikazu Johmura, M. Nakanishi
          </td>
          <td>2025-04-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed cancer therapy. However, only a fraction of patients benefit from these treatments. The variability in patient responses remains a significant challenge due to the intricate nature of the tumor microenvironment. Here, we harness single-cell RNA-sequencing data and employ machine learning to predict patient responses while preserving interpretability and single-cell resolution. Using a dataset of melanoma-infiltrated immune cells, we applied XGBoost, achieving an initial AUC score of 0.84, which improved to 0.89 following Boruta feature selection. This analysis revealed an 11-gene signature predictive across various cancer types. SHAP value analysis of these genes uncovered diverse gene-pair interactions with non-linear and context-dependent effects. Finally, we developed a reinforcement learning model to identify the most informative single cells for predictivity. This approach highlights the power of advanced computational methods to deepen our understanding of cancer immunity and enhance the prediction of treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c30c093ebc801383a9668f64b177a2ace0fcd67" target='_blank'>
              Uncovering gene and cellular signatures of immune checkpoint response via machine learning and single-cell RNA-seq
              </a>
            </td>
          <td>
            Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2025-04-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Introduction The molecular pathogenesis of ocular adnexal mucosa-associated lymphoid tissue (MALT) lymphoma and IgG4-related ophthalmic disease (IgG4-ROD) remains incompletely understood. Differentiating between the two diseases is vital given that the diagnostic evaluation and treatment approaches can vary significantly; this difficulty in distinction is exacerbated by the absence of specific biomarkers. This study aimed to investigate the differences between these two diseases based on their cellular composition, transcriptional heterogeneity, and the immune microenvironment using single-cell RNA transcriptional sequencing (scRNA-seq) technology. Methods We collected orbital lacrimal gland region tissue samples from three patients with MALT lymphoma and another three with IgG4-ROD and performed single-cell sequencing experiments. Subsequently, we conducted bioinformatics analyses, including cell subpopulation segmentation and inter-group comparison, tumor cell identification, functional enrichment analysis, and pseudotime trajectory analysis. Furthermore, we analyzed the cellular communication between tumor B-cell and T-cell subsets within the immune microenvironment of MALT lymphoma tissues. We performed immunofluorescence assays to verify the co-expression of receptor-ligand pairs. Results A total of six major cell subpopulations were identified, with B-cells and T-cells being the predominant cell types. All B-cell subpopulations in MALT lymphomas are malignant, exhibiting significant intratumoral and intertumoral heterogeneity. Reclustering of the T-cell subpopulation identified five major T-cell subpopulations. Pseudotime analysis revealed that CD4+ naive T-cells in MALT lymphoma patients were highly likely to differentiate into follicular helper T-cells, whereas, in IgG4-ROD patients, CD4+ naive T-cells were highly likely to differentiate into regulatory T-cells. Intercellular communication analysis revealed that the CD27-CD70 immune checkpoint receptor−ligand pair and CXCL13-CXCR5 chemokine receptor−ligand pair were significantly upregulated between malignant B-cells and T-cells subpopulations. Conclusion This study is the first to conduct a comparative single-cell transcriptome sequencing analysis of ocular adnexal MALT lymphoma and IgG4-ROD. Our results reveal the cellular composition, key pathways, and critical immune microenvironment implicated in the development of these two diseases. These findings provide important insights into the pathogenesis of these two diseases and highlight the differences between them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5b84eefafe3887c5d3f9997b2994a421a348df" target='_blank'>
              Single-cell RNA-sequencing reveals cellular heterogeneity and immune microenvironment characteristics between ocular adnexal mucosa-associated lymphoid lymphoma and IgG4−related ophthalmic disease
              </a>
            </td>
          <td>
            Yu Yang, Yujiao Wang, Xuelian Jin, W. He
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1804a2ea493d2cd5cd4a540fc4df2d804160e23" target='_blank'>
              Differential effects of immobilized CCL21 and ICAM1 on TILs with distinct expansion properties
              </a>
            </td>
          <td>
            Sofi Yado, Rawan Zoabi, Karin Brezinger-Dayan, S. Albeck, Tamar Unger, Moran Meiron, Alessio D. Nahmad, Aya Tzur Gilat, Michal J Besser, Benjamin Geiger
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Oral cancer is one of the most prevalent malignant neoplasms globally, with its microenvironment being extremely complicated and heterogeneous with regard to distribution of different types of cells, which creates a barrier against treatment. We integrated single-cell RNA sequencing, pseudotime analysis, and intercellular communication to identify subtypes, state transitions, and interactions in the development of oral cancer and their roles in tumor progression and immune response. Methods Principal component analysis (PCA) followed by dimensionality reduction techniques (UMAP and t-SNE) revealed multiple populations in the single-cell data from oral cancer. Analysis of gene expression patterns associated individual genes with distinct principal components that described features of cell subtypes or states in the oral cancer microenvironment. We also conducted gene set enrichment analysis (GSEA) and cell–cell communication network analysis to investigate the important signaling pathways as well as potential cell–cell communications. Results Gene expression patterns linked to oral cancer cell subtypes and states in this study included cell membrane, signal transduction and immune response in different biological processes. Pseudotime analysis showed differentiation trajectories and expression of essential differentiation genes were also altered. Cell–cell communication network analysis revealed that myeloid cells interacted heavily with myeloid cells and they also had strong interactions with endothelial cells. Analysis of the MIF signaling pathway network showed participation of various cell types in MIF signaling, suggesting its potential significance in the oral cancer microenvironment. Conclusion The intercellular communication complexity of TME in oral cancer is reported here with the first-ever single cell analysis and data providing new perspective regarding the heterogeneity of oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8d5845a6bd00754561294af9f1ae51a6774014" target='_blank'>
              Single-cell pseudotime and intercellular communication analysis reveals heterogeneity and immune microenvironment in oral cancer
              </a>
            </td>
          <td>
            Hanjun Liu, Lian Xie, Xuemin Xing, Lili Hou, Jinfeng Zhang, Ling Yang
          </td>
          <td>2025-02-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastomas are the most prevalent primary brain tumors and are associated with a dramatically poor prognosis. Despite an intensive treatment approach, including maximal surgical tumor removal followed by radio- and chemotherapy, the median survival for glioblastoma patients has remained around 18 months for decades. Glioblastoma is distinguished by its highly complex mechanisms of immune evasion and pronounced heterogeneity. This variability is apparent both within the tumor itself, which can exhibit multiple phenotypes simultaneously, and in its surrounding microenvironment. Another key feature of glioblastoma is its “cold” microenvironment, characterized by robust immunosuppression. Recent advances in single-cell RNA sequencing have uncovered new promising insights, revealing previously unrecognized aspects of this tumor. In this review, we consolidate current knowledge on glioblastoma cells and its microenvironment, with an emphasis on their biological properties and unique patterns of molecular communication through signaling pathways. The evidence underscores the critical need for personalized poly-immunotherapy and other approaches to overcome the plasticity of glioblastoma stem cells. Analyzing the tumor microenvironment of individual patients using single-cell transcriptomics and implementing a customized immunotherapeutic strategy could potentially improve survival outcomes for those facing this formidable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf00ce3ce13ab5e7ee7ff1ae10bf120f0be9305e" target='_blank'>
              Molecular biology of the deadliest cancer – glioblastoma: what do we know?
              </a>
            </td>
          <td>
            Aly Ismailov, Aldo Spallone, Alexey Belogurov, Alan Herbert, Maria Poptsova
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ongoing mutagenesis in cancer drives genetic diversity throughout the natural history of cancers. As the activities of mutational processes are dynamic throughout evolution, distinguishing the mutational signatures of ‘active’ and ‘historical’ processes has important implications for studying how tumors evolve. This can aid in understanding mutagenic states at the time of presentation, and in associating active mutational process with therapeutic resistance. As bulk sequencing primarily captures historical mutational processes, we studied whether ultra-low-coverage single-cell whole-genome sequencing (scWGS), which measures the distribution of mutations across hundreds or thousands of individual cells, could enable the distinction between historical and active mutational processes. While technical challenges and data sparsity have limited mutation analysis in scWGS, we show that these data contain valuable information about dynamic mutational processes. To robustly interpret single nucleotide variants (SNVs) in scWGS, we introduce ArtiCull, a method to identify and remove SNV artifacts by leveraging evolutionary constraints, enabling reliable detection of mutations for signature analysis. Applying this approach to scWGS data from pancreatic ductal adenocarcinoma (PDAC), triple-negative breast cancer (TNBC), and high-grade serous ovarian cancer (HGSOC), we uncover temporal and spatial patterns in mutational processes. In PDAC, we observe a temporal increase in mismatch repair deficiency (MMRd). In cisplatin-treated TNBC patient-derived xenografts, we identify therapy-induced mutagenesis and inactivation of APOBEC3 activity. In HGSOC, we show distinct patterns of APOBEC3 mutagenesis, including late tumor-wide activation in one case and clade-specific enrichment in another. Additionally, we detect a clone-specific increase in SBS17 activity, in a clone previously linked to recurrence. Our findings establish ultra-low-coverage scWGS as a powerful approach for studying active mutational processes that may influence ongoing clonal evolution and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6d1d9f289244977b2dfb8fbeaff86f943a227d" target='_blank'>
              Inferring active mutational processes in cancer using single cell sequencing and evolutionary constraints
              </a>
            </td>
          <td>
            Gryte Satas, Matthew A. Myers, Andrew McPherson, Sohrab P. Shah
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Despite remarkable advancements in therapeutic strategies, a considerable proportion of patients with bladder cancer (BC) still experience disease progression and unfavorable prognosis. The heterogeneity and biological functions of tumor endothelial cells (ECs) during BC progression remain poorly understood. We collected scRNA-seq data from BC samples and identified two EC subpopulations through hierarchical clustering analysis. The activity of signaling pathways in distinct EC subpopulations was assessed utilizing AUCell analysis. Gene regulatory networks (GRN) were constructed and analyzed for different EC subpopulations using the pySCENIC algorithm. Additionally, we investigated the association between the abundance of EC subpopulations and both clinical prognosis and immune cell infiltration. The biological effects of ESM1 protein on BC cells were further validated through EdU and Transwell assays. We analyzed 7,519 CD45-negative single cells from BC tissues and discerned two distinct EC subpopulations. The two subpopulations were characterized by high expression of ESM1 (S1 ECs) and CXCL2 (S2 ECs), respectively. In S1 ECs, we observed significant activation of signaling pathways involved in tumor promotion, including angiogenesis and cell proliferation. Additionally, our GRN analysis uncovered notable differences in transcription factor activity between S1 and S2 ECs. Moreover, ESM1 protein promoted proliferation and migration of BC cells. Patients with higher abundance of the S1 EC subpopulation exhibited more unfavorable clinical outcomes and increased infiltration of inhibitory immune cells. Our findings elucidate the transcriptional profiles and biological roles of the high ESM1-expression endothelial cell subpopulation in BC. This subpopulation is associated with poor prognosis and immunosuppressive tumor microenvironment. Accordingly, targeting endothelial cells with high ESM1 expression may offer a novel therapeutic strategy for patients with BC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95731-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4283029ca72adeb8c04b30a9f72593fd7af08134" target='_blank'>
              Single-cell sequencing uncovers a high ESM1-expression endothelial cell subpopulation associated with bladder cancer progression and the immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Yifeng Qiu, Yuhan Wang, Jiahe Liu, Baohua Liu, Kai Sun, Qi Hou
          </td>
          <td>2025-03-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite being the leading cause of death worldwide, cancer still lacks precise biomarkers for effective targeting, limiting efforts to reduce mortality rates. This review explores the role and clinical significance of a newly identified long non-coding RNA, MYOSLID, in cancer progression. MYOSLID has emerged as a critical modulator in cancer progression by influencing key hallmarks such as proliferation, immune evasion, metastasis, and metabolic reprogramming. It promotes tumor cell growth by stabilizing hypoxia-inducible factor 1 and acting as a competing endogenous RNA (ceRNA) to sequester tumor-suppressive microRNAs like miR-29c-3p, thereby enhancing oncogene expression. It facilitates immune evasion by upregulating PD-L1, suppressing T cell activation, and modulating necroptosis pathways involving RIPK1 and RIPK3. Additionally, MYOSLID drives metastasis by regulating epithelial–mesenchymal transition markers such as LAMB3 and Slug while promoting RAB13-mediated cytoskeletal remodeling and enhancing cancer cell invasion. We have obtained the expression of MYOSLID from TCGA and the ENCORI database. The expression of colorectal adenocarcinoma (COAD) and head and neck squamous cell carcinoma (HNSCC) is associated with poor prognosis and lower survival rate. Given its significant potential as a diagnostic biomarker and therapeutic target, further research is required to elucidate its precise molecular mechanisms and therapeutic applications in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df104b4d34b4abacff7e63dc6f3d41ba7b3ed3e" target='_blank'>
              MYOSLID: A Critical Modulator of Cancer Hallmarks
              </a>
            </td>
          <td>
            Kanupriya Medhi, Sagarika Mukherjee, Aastha Dagar, Ashutosh Kumar Tiwari, Sia Daffara, Sanjana Bana, Vivek Uttam, Md Rizwan Ansari, Vikas Yadav, H. Tuli, Aklank Jain
          </td>
          <td>2025-03-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive tumor with high metastatic potential which leads to decreased survival rate and resistance to chemotherapy and immunotherapy. Nearly 90% of pancreatic cancer comprises pancreatic ductal adenocarcinoma (PDAC). About 80% of diagnoses takes place at the advanced metastatic stage when it is unresectable, which renders chemotherapy regimens ineffective. There is also a dearth of specific biomarkers for early-stage detection. Advances in next generation sequencing and single cell profiling have identified molecular alterations and signatures that play a role in PDAC progression and subtype plasticity. Most chemotherapy regimens have shown only modest survival benefits, and therefore, translational approaches for immunotherapies and combination therapies are urgently required. In this review, we have examined the immunosuppressive and dense stromal network of tumor immune microenvironment with various metabolic and transcriptional changes that underlie the pro-tumorigenic properties in PDAC in terms of phenotypic heterogeneity, plasticity and subtype co-existence. Moreover, the stromal heterogeneity as well as genetic and epigenetic changes that impact PDAC development is discussed. We also review the PDAC interaction with sequestered cellular and humoral components present in the tumor immune microenvironment that modify the outcome of chemotherapy and radiation therapy. Finally, we discuss different therapeutic interventions targeting the tumor immune microenvironment aimed at better prognosis and improved survival in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e7cd5ce498aa2efbf04ec631daa67571857bc7" target='_blank'>
              Phenotypic heterogeneity and tumor immune microenvironment directed therapeutic strategies in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Remya P. G. Ramesh, Hadida Yasmin, Pretty Ponnachan, Basel Al-Ramadi, Uday Kishore, Ann Mary Joseph
          </td>
          <td>2025-03-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is among the most aggressive cancers, with rising incidence and limited responsiveness to immunotherapy due to its highly suppressive tumor microenvironment (TME). Tertiary lymphoid structures (TLS), ectopic formation structures of immune cells, are linked to better prognosis and improved immunotherapy responses in PDAC. Understanding TLS’s role in PDAC could enhance immunotherapy effectiveness. Methods This study integrated transcriptomic and clinical data from 310 PDAC patients in GEO database. We performed consensus clustering using tumor-associated TLS (TA-TLS) genes, identifying three distinct molecular subtypes. Single-sample gene set enrichment analysis (ssGSEA) was then employed to calculate a TLS score for each patient, allowing for TLS-based evaluation. Key prognostic genes were identified using an iterative LASSO method, leading to the construction of a risk assessment model, which was validated across independent cohorts. We further analyzed the TLS score using single-cell RNA sequencing (scRNA-seq), visualized key gene expression, and validated protein expression through immunohistochemistry (IHC). Additionally, we explored the effects of DNASE1L3 on cell proliferation and migration, and its immune-related functions using Gene Set Enrichment Analysis (GSEA) and multiplex cytokine analysis. Results Consensus clustering revealed three PDAC molecular subtypes with significant differences in prognosis, TA-TLS gene expression, and TME features. The TLS score effectively stratified patients into high and low groups, correlating with survival outcomes and TME characteristics. Our risk model, validated across cohorts, reliably predicted patient outcomes. Validation studies showed lower expression of DNASE1L3 and IL33 in tumor tissues. scRNA-seq confirmed TLS score associations with immune cells. DNASE1L3 overexpression inhibited PDAC cell proliferation and migration, with cytokine analysis indicating increased immune activity. Conclusions This study elucidated the expression profile of TA-TLS genes in PDAC, constructed a TLS gene-based scoring system, and developed a related risk model. We also explored the functions and potential antitumor mechanisms of key genes, providing evidence and new insights for enhancing TLS-targeted immunotherapy strategies in PDAC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06152-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ad4f13e415f9357404f99d30b4979256aef4341" target='_blank'>
              Prognostic implications and characterization of tumor-associated tertiary lymphoid structures genes in pancreatic cancer
              </a>
            </td>
          <td>
            Enkui Zhang, Yongsu Ma, Zonghao Liu, Jixin Zhang, Weikang Liu, Yiran Chen, Guangnian Liu, Xinxin Liu, Fusheng Zhang, Yu Zhu, Yinmo Yang, Xiaodong Tian
          </td>
          <td>2025-03-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8148f78a26bf47463c0acd3c067c50a7bec19c0" target='_blank'>
              Spatio-temporal dynamics of M1 and M2 macrophages in a multiphase model of tumor growth
              </a>
            </td>
          <td>
            I. Lampropoulos, Panayotis Kevrekidis, Christos Zois, Helen Byrne, Mihail E Kavousanakis
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b216340a3ee900bc1d9420b9d6cc7048a9f68c8" target='_blank'>
              Radiogenomics of intrahepatic cholangiocarcinoma predicts immunochemotherapy response and identifies therapeutic target.
              </a>
            </td>
          <td>
            Gu-Wei Ji, Zheng Xu, Shuo-Chen Liu, Shu-Ya Cao, C. Jiao, Ming Lu, Biao Zhang, Yue Yang, Qing Xu, Xiao-Feng Wu, Ke Wang, Yong-xiang Xia, Xiang-Cheng Li, Xue-Hao Wang
          </td>
          <td>2025-02-10</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdedfede7608cb2fb7f1fa7b1c417998bb4fd21" target='_blank'>
              Spatially resolved transcriptomic analysis of the adult human prostate.
              </a>
            </td>
          <td>
            Junyi Hu, Fei Liu, Jing Zhang, Lei Yin, Wanli Cao, Weidong Xu, Yifan Chang, Ye Wang, Jian Wang, Yaxin Hou, Lilong Liu, Sujun Chen, Guanghui Zhu, Junhui Jiang, Zixian Wang, , H. H. He, Di Gu, Ke Chen, Shancheng Ren
          </td>
          <td>2025-04-01</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Meibomian glands secrete lipid-rich meibum, which prevents tear evaporation. Aging-related Meibomian gland shrinkage may result in part from stem cell exhaustion and is associated with evaporative dry eye disease, a common condition lacking effective treatment. The identities and niche of Meibomian gland stem cells and the signals controlling their activity are poorly defined. Using snRNA-seq, in vivo lineage tracing, ex vivo live imaging, and genetic studies in mice, we identify markers for stem cell populations that maintain distinct regions of the gland and uncover Hedgehog (Hh) signaling as a key regulator of stem cell proliferation. Consistent with this, we show that human Meibomian gland carcinoma exhibits increased Hh signaling. Aged glands display decreased Hh and EGF signaling, deficient innervation, and loss of collagen I in niche fibroblasts, indicating that alterations in both glandular epithelial cells and their surrounding microenvironment contribute to age-related degeneration. These findings suggest new approaches to treat aging-associated Meibomian gland loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd9f723993b9fbc9144678833cdc43c46f60f6e1" target='_blank'>
              Identification of Meibomian gland stem cell populations and mechanisms of aging
              </a>
            </td>
          <td>
            Xuming Zhu, Mingang Xu, Céline Portal, Yvonne Lin, Alyssa Ferdinand, Tien Peng, Edward E. Morrisey, Andrzej A Dlugosz, Joseph M. Castellano, Vivian Lee, John Seykora, Sunny Y Wong, Carlo Iomini, Sarah E Millar
          </td>
          <td>2025-02-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), are a critical subpopulation within tumours, and are defined by their capacity for self-renewal, differentiation, and tumour initiation. These unique traits contribute to tumour progression, metastasis, and resistance to conventional treatments like chemotherapy and radiotherapy, often resulting in cancer recurrence and poor patient outcomes. As such, CSCs have become focal points in developing advanced cancer therapies. This review highlights progress in CSC-targeted treatments, including chimeric antigen receptor T-cell (CAR-T) therapy, immunotherapy, molecular targeting, and nanoparticle-based drug delivery systems. Plant-derived compounds and gene-editing technologies, such as clustered regularly interspaced short palindromic repeats (CRISPR), are explored for their potential to enhance precision and minimize side effects. Metabolic pathways integral to CSC survival, such as mitochondrial dynamics, mitophagy (regulated by dynamin-related protein 1 [DRP1] and the PINK1/Parkin pathway), one-carbon metabolism, amino acid metabolism (involving enzymes like glutaminase (GLS) and glutamate dehydrogenase (GDH]), lipid metabolism, and hypoxia-induced metabolic reprogramming mediated by hypoxia-inducible factors (HIF-1α and HIF-2α), are examined as therapeutic targets. The adaptability of CSCs through autophagy, metabolic flexibility, and epigenetic regulation by metabolites like α-ketoglutarate, succinate, and fumarate is discussed. Additionally, extracellular vesicles and nicotinamide adenine dinucleotide (NAD⁺) metabolism are identified as pivotal in redox balance, DNA repair, and epigenetic modifications. Addressing challenges such as tumour heterogeneity, immune evasion, and treatment durability requires interdisciplinary collaboration. Advancing CSC-targeted therapies is essential for overcoming drug resistance and preventing cancer relapse, paving the way for transformative cancer treatments. This review underscores the importance of leveraging innovative technologies and fostering collaboration to revolutionize cancer treatment. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdfd29a8c49b166f1e1f2b3241c057ea76b860a3" target='_blank'>
              Unveiling the future of cancer stem cell therapy: a narrative exploration of emerging innovations
              </a>
            </td>
          <td>
            Joseph Nhyira Obisi, Abike Ndidiamaka Josephine Abimbola, O. Babaleye, Peter Kwame Atidoglo, Saviour Godswealth Usin, Eudora Nwanaforo, Faith Sutu Patrick-Inezi, I. V. Fasogbon, Joseph Chimezie, Christianah Adebimpe Dare, Oluwadoyinsayemi Oluwadamilare Kuti, Daniel Ejim Uti, Humphrey Chukwudi Omeoga
          </td>
          <td>2025-03-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e1ac36ae0a43d7b28635df6f8edaf8554a72a9" target='_blank'>
              SpaDCN: Deciphering Spatial Functional Landscape from Spatially Resolved Transcriptomics by Aligning Cell-Cell Communications.
              </a>
            </td>
          <td>
            Xiaosheng Bai, Xinyu Bao, Chuanchao Zhang, Qianqian Shi, Luonan Chen
          </td>
          <td>2025-02-17</td>
          <td>Small methods</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions may shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts, we systematically identify frequency-dependent selection dynamics governed by karyotypic diversity. We evaluate evidence for subpopulation interactions that may influence tumor fitness, highlighting candidate karyotypes that serve as ecological interaction hubs. This framework provides testable predictions of intratumoral ecological dynamics and highlights the potential of targeting karyotypically distinct subpopulations to disrupt tumor evolution strategically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccad37f8db740ee741e31d76d75cb5a07f7e923f" target='_blank'>
              Biology as vulnerability in follicular lymphoma: genetics, epigenetics, and immunogenetics.
              </a>
            </td>
          <td>
            F. Araujo-Ayala, W. Béguelin
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ea1b0559ba72f593b3a6674f424957c7f5230c" target='_blank'>
              Abstract B016: Genomic features of the tumor microenvironment and antigen presentation machinery vary by clinical and molecular subtypes of breast cancer
              </a>
            </td>
          <td>
            Constandina E. O'Connell, Yoon Jin Cha, Alexander V Lobanov, Cheng Fan, Brooke M. Felsheim, A. Fernandez-Martinez, C. Perou
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>163</td>
        </tr>

        <tr id="Abstract Breast cancer remains a significant global health challenge due to its complexity, which arises from multiple genetic and epigenetic mutations that originate in normal breast tissue. Traditional machine learning models often fall short in addressing the intricate gene interactions that complicate drug design and treatment strategies. In contrast, our study introduces GEMDiff, a novel computational workflow leveraging a diffusion model to bridge the gene expression states between normal and tumor conditions. GEMDiff augments RNAseq data and simulates perturbation transformations between normal and tumor gene states, enhancing biomarker identification. GEMDiff can handle large-scale gene expression data without succumbing to the scalability and stability issues that plague other generative models. By avoiding the need for task-specific hyper-parameter tuning and specific loss functions, GEMDiff can be generalized across various tasks, making it a robust tool for gene expression analysis. The model’s ability to augment RNA-seq data and simulate gene perturbations provides a valuable tool for researchers. This capability can be used to generate synthetic data for training other machine learning models, thereby addressing the issue of limited biological data and enhancing the performance of predictive models. The effectiveness of GEMDiff is demonstrated through a case study using breast mRNA gene expression data, identifying 307 core genes involved in the transition from a breast tumor to a normal gene expression state. GEMDiff is open source and available at https://github.com/xai990/GEMDiff.git under the MIT license.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3472d3e5ad8a58f2f1d0054cb37345e502bfbbc" target='_blank'>
              GEMDiff: a diffusion workflow bridges between normal and tumor gene expression states: a breast cancer case study
              </a>
            </td>
          <td>
            Xusheng Ai, Melissa C. Smith, F. A. Feltus
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background In the course of tumor treatment, radiation therapy (RT) not only kills cancer cells, but also induces complex biological effects in non-malignant cells around cancer cells. These biological effects such as angiogenesis, changes in stromal composition and immune cell infiltration remodel the tumor microenvironment (TME). As one of the major components of the TME, Cancer‑associated fibroblasts (CAFs) are not only involved in tumorigenesis, progression, recurrence, and metastasis but also regulate the tumor-associated immune microenvironment. CAFs and tumor cells or immune cells have complex intercellular communication in the context of tumor radiation. Main content. Different cellular precursors, spatial location differences, absence of specific markers, and advances in single-cell sequencing technology have gradually made the abundant heterogeneity of CAFs well known. Due to unique radioresistance properties, CAFs can survive under high doses of ionizing radiation. However, radiation can induce phenotypic and functional changes in CAFs and further act on tumor cells and immune cells to promote or inhibit tumor progression. To date, the effect of RT on CAFs and the effect of irradiated CAFs on tumor progression and TME are still not well defined. Conclusion In this review, we review the origin, phenotypic, and functional heterogeneity of CAFs and describe the effects of RT on CAFs, focusing on the mutual crosstalk between CAFs and tumor or immune cells after radiation. We also discuss emerging strategies for targeted CAFs therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4e7d8569cb5aa3f3831a8692c395636f7587796" target='_blank'>
              Cancer‑associated fibroblasts: a pivotal regulator of tumor microenvironment in the context of radiotherapy
              </a>
            </td>
          <td>
            Linhui Zheng, Wenqi Cai, Yuan Ke, Xiaoyan Hu, Chunqian Yang, Runze Zhang, Huachao Wu, Dong Liu, Haijun Yu, Chaoyan Wu
          </td>
          <td>2025-03-20</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Tumour‐associated microbiota are integral components of the tumour microenvironment (TME). However, previous studies on intratumoral microbiota primarily rely on bulk tissue analysis, which may obscure their spatial distribution and localized effects. In this study, we applied in situ spatial‐profiling technology to investigate the spatial distribution of intratumoral microbiota in breast cancer and their interactions with the local TME. Using 5R 16S rRNA gene sequencing and RNAscope FISH/CISH on patients’ tissue, we identified significant spatial heterogeneity in intratumoral microbiota, with Fusobacterium nucleatum (F. nucleatum) predominantly localized in tumour cell‐rich areas. GeoMx digital spatial profiling (DSP) revealed that regions colonized by F. nucleatum exhibit significant influence on the expression of RNAs and proteins involved in proliferation, migration and invasion. In vitro studies indicated that co‐culture with F. nucleatum significantly stimulates the proliferation and migration of breast cancer cells. Integrative spatial multi‐omics and co‐culture transcriptomic analyses highlighted the MAPK signalling pathways as key altered pathways. By intersecting these datasets, VEGFD and PAK1 emerged as critical upregulated proteins in F. nucleatum‐positive regions, showing strong positive correlations with MAPK pathway proteins. Moreover, the upregulation of VEGFD and PAK1 by F. nucleatum was confirmed in co‐culture experiments, and their knockdown significantly reduced F. nucleatum‐induced proliferation and migration. In conclusion, intratumoral microbiota in breast cancer exhibit significant spatial heterogeneity, with F. nucleatum colonization markedly altering tumour cell protein expression to promote progression and migration. These findings provide novel perspectives on the role of microbiota in breast cancer, identify potential therapeutic targets, and lay the foundation for future cancer treatments. Key points Intratumoral Fusobacterium nucleatum exhibits significant spatial heterogeneity within breast cancer tissues. F. nucleatum colonization alters the expression of key proteins involved in tumour progression and migration. The MAPK signalling pathway is a critical mediator of F. nucleatum‐induced breast cancer cell proliferation and migration. VEGFD and PAK1 are potential therapeutic targets to mitigate F. nucleatum‐induced tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b181d8c04116c66c673f00453e25d5cfea9b8649" target='_blank'>
              Integrated spatial multi‐omics profiling of Fusobacterium nucleatum in breast cancer unveils its role in tumour microenvironment modulation and cancer progression
              </a>
            </td>
          <td>
            Feng Zhao, Rui An, Yilei Ma, Shaobo Yu, Yuzhen Gao, Yanzhong Wang, Haitao Yu, Xinyou Xie, Jun Zhang
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be659580caffb1fbf7776c3ae3608aa7fdd87091" target='_blank'>
              Tumor microenvironment: recent advances in understanding and its role in modulating cancer therapies.
              </a>
            </td>
          <td>
            Disha D Shah, Mehul R. Chorawala, Neha R. Raghani, Rajanikant Patel, Mohammad Fareed, V. Kashid, Bhupendra G. Prajapati
          </td>
          <td>2025-03-18</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract To unravel the mechanism of immune activation and suppression within tumors, a critical step is to identify transcriptional signals governing cell–cell communication between tumor and immune/stromal cells in the tumor microenvironment. Central to this communication are interactions between secreted ligands and cell-surface receptors, creating a highly connected signaling network among cells. Recent advancements in in situ-omics profiling, particularly spatial transcriptomic (ST) technology, provide unique opportunities to directly characterize ligand–receptor signaling networks that power cell–cell communication. In this paper, we propose a novel statistical method, LRnetST, to characterize the ligand–receptor interaction networks between adjacent tumor and immune/stroma cells based on ST data. LRnetST utilizes a directed acyclic graph model with a novel approach to handle the zero-inflated distributions of ST data. It also leverages existing ligand–receptor regulation databases as prior information, and employs a bootstrap aggregation strategy to achieve robust network estimation. Application of LRnetST to ST data of high-grade serous ovarian tumor samples revealed both common and distinct ligand–receptor regulations across different tumors. Some of these interactions were validated through both a MERFISH dataset and a CosMx SMI dataset of independent ovarian tumor samples. These results cast light on biological processes relating to the communication between tumor and immune/stromal cells in ovarian tumors. An open-source R package of LRnetST is available on GitHub at https://github.com/jie108/LRnetST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db23b179c860dc1d8c3febc0dd28265cf6aa4993" target='_blank'>
              Learning directed acyclic graphs for ligands and receptors based on spatially resolved transcriptomic data of ovarian cancer
              </a>
            </td>
          <td>
            Shrabanti Chowdhury, S. Ferri-Borgogno, Peng Yang, Wenyi Wang, Jie Peng, Samuel C Mok, Pei Wang
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) presents significant treatment challenges due to its high heterogeneity and complex intercellular interactions. Further exploration of CRC subtypes and interactions among tumor-specific clusters will facilitate the development of personalized treatment strategies. Methods Single-cell RNA sequencing and bulk RNA sequencing datasets were integrated to determine CRC metabolic subtypes by hierarchical clustering. The analysis was further extended to cellchat, pseudotime, immune infiltration, and clinicopathological relevance to explore the characteristics of secreted frizzled related protein 2 (SFRP2) + cancer-associated fibroblast (CAF) clusters, and validated by spatial transcriptomics (ST), in vivo experiments, and multiple immunohistochemistry (mIHC). Results CRC samples were stably classified into three heterogeneous metabolic subtypes, each exhibiting different microenvironment and CAF heterogeneity, particularly in the distribution of SFRP2 + CAF, which was aligned with metabolic activity. SFRP2 + CAF exhibits high extracellular matrix (ECM) activity and is closely involved in cellular communication, not only promoting the malignant progression of cancer cells but also inducing the differentiation of Tregs. Compared to responders of chemotherapy, the proportion of SFRP2 + CAFs is significantly increased in non-responders. Importantly, mIHC and ST analyses confirm that cancer cells with low expression of agmatinase (AGMAT) can recruit SFRP2 + CAFs, and Treg infiltration surrounding SFRP2 + CAFs was observed. AGMAT combined with oxaliplatin showed the best efficacy in vivo, which may be associated with the inhibition of SFRP2 + CAF infiltration. Conclusions Our study identified and described the potential protumor biological properties of SFRP2 + CAFs, and AGMAT may be a valuable target for disrupting their properties. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06103-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997e954cb156caa69cf5f6fb240e0f9297f38fc1" target='_blank'>
              Single-cell and spatial-resolved profiling reveals cancer-associated fibroblast heterogeneity in colorectal cancer metabolic subtypes
              </a>
            </td>
          <td>
            Youpeng Wang, Xingfeng Qiu, Qinghai Li, Jiale Qin, Lvlan Ye, Xiang Zhang, Xingxiang Huang, Xiangqiong Wen, Ziyang Wang, Weiling He, Yuqin Di, Qi Zhou
          </td>
          <td>2025-02-11</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="CD8+ tissue-resident memory T cells (CD8+TRM), expressing PD-1 and/or TIM-3, are linked to immunological surveillance in non-small cell lung cancer (NSCLC). However, their prognostic value across activation states and spatial distributions in NSCLC stages is unclear. We analyzed 271 NSCLC patients’ primary tumors and lymph nodes, using multiplex immunohistochemistry and inForm software for cell identification. Statistical analyses included the Mann-Whitney U test and Cox survival analysis. Findings showed CD8+TRM were categorized into four activation states. In locally advanced NSCLC, PD-1−TIM-3+CD8+TRM3, and PD-1+TIM-3+CD8+TRM4 densities were notably higher at invasive margins. Fewer interactions between CD8+TRM and tumor cells were observed in advanced lesions. Decreased PD-1+TIM-3−CD8+TRM2 interactions with tumor cells and increased PD-1+TIM-3+CD8+TRM4 interactions with tumor cells were independently associated with recurrence in patients with early lung adenocarcinoma and squamous carcinoma, respectively. These results suggest that CD8+TRM activation state and distribution are linked to recurrence risk in early-stage NSCLC, emphasizing the importance of CD8+TRM spatial dynamics in prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8927993683d672fa19c819fd8f766d987dddb21" target='_blank'>
              Stage-dependent spatial distribution and prognostic value of CD8+ tissue-resident memory T cells in NSCLC
              </a>
            </td>
          <td>
            Liying Yang, Hao Yang, Miaoqing Zhao, Hongtu Yuan, Jiaxiao Geng, Yushan Yan, Li Wu, Ligang Xing, Jinming Yu, Xiaorong Sun
          </td>
          <td>2025-02-22</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f05272abf51c9f38dd50672d47db4690dbc1bb3" target='_blank'>
              Spatial Profiling Identifies Regionally Distinct Microenvironments and Targetable Immunosuppressive Mechanisms in Pediatric Osteosarcoma Pulmonary Metastases.
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice S Pereira, Demond Williams, Douglass W Morris-Ii, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael C. Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-04-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e89dc2d087d9d2c969c5bb0a4b45e01fc9cc0f7a" target='_blank'>
              Single-cell and spatial genomic landscape of non-small cell lung cancer brain metastases.
              </a>
            </td>
          <td>
            Somnath Tagore, Lindsay A. Caprio, Amit Dipak Amin, K. Bestak, Karan Luthria, Edridge K. D’Souza, Irving Barrera, Johannes C. Melms, Sharon Wu, Sinan Abuzaid, Yiping Wang, Viktoria Jakubikova, Peter Koch, D. Z. Brodtman, Banpreet Bawa, S. Deshmukh, Leon Ebel, Miguel A. Ibarra-Arellano, A. Jaiswal, Carino Gurjao, Jana Biermann, Neha Shaikh, Priyanka Ramaradj, Yohanna Georgis, Galina G Lagos, Matthew I Ehrlich, P. Ho, Zachary H. Walsh, Meri Rogava, Michelle Garlin Politis, Devanik Biswas, A. Cottarelli, Nikhil Rizvi, CA Shu, Benjamin O Herzberg, N. Anandasabapathy, G. Sledge, Emmanuel Zorn, Peter D Canoll, Jeffrey N. Bruce, Naiyer A Rizvi, Alison M Taylor, Anjali Saqi, H. Hibshoosh, Gary K Schwartz, Brian S Henick, Fei Chen, Denis Schapiro, Parin Shah, Benjamin Izar
          </td>
          <td>2025-02-27</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539a7161f9d5e59af0e8571e13d64e98e255b165" target='_blank'>
              Single-cell sequencing reveals the role of aggrephagy-related patterns in tumor microenvironment, prognosis and immunotherapy in endometrial cancer
              </a>
            </td>
          <td>
            Yuquan Yuan, Chunyan Ren, Jin Shu, Keyang Zhu, Ganghui Li, Bao Liu, Jianrong Huang, Yinde Huang, Chengzhi Zhao
          </td>
          <td>2025-03-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a highly aggressive brain tumor associated with poor survival outcomes and is driven by a complex tumor microenvironment (TME) that promotes tumor progression and treatment resistance. To explore the role of the TME in GBM, we analyzed glioma-related microarray and single-cell RNA sequencing (scRNA-seq) datasets from the Gene Expression Omnibus (GEO). Functional enrichment and weighted gene coexpression network analyses revealed distinct immune profiles, metabolic alterations, and differences in chemotherapeutic drug sensitivity between the high-risk and low-risk patient groups. scRNA-seq data processed with the ‘Seurat’ package were used to identify differentially expressed genes in pericytes, endothelial cells, and glioma cells, particularly those involved in extracellular matrix (ECM) remodeling. A 17-gene prognostic signature developed through Cox regression and LASSO analyses revealed that key genes (COL1A1, COL4A1, and VIM) were significantly associated with survival outcomes in GBM patients. Drug sensitivity analyses using data from the Genomics of Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) identified potential targeted therapies for GBM, including SB-505,124, staurosporine, and AZD8186. This integrative study underscores the critical roles of the ECM and synaptic remodeling in GBM and suggests novel therapeutic targets to improve personalized treatment strategies for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/456759f2943a08a601ac9cd5e20189bf3d373750" target='_blank'>
              Molecular mechanisms and therapeutic targets in glioblastoma multiforme: network and single-cell analyses
              </a>
            </td>
          <td>
            Xiangyu Chen, Xiao Zhong, Feifei Zhang, Xiaomei Zhou, Xiaofeng Yue, Xueru Li
          </td>
          <td>2025-03-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5372012557b72610a59f72be2bf39f2688654856" target='_blank'>
              Spatial distribution of the proteome in human body and cancers
              </a>
            </td>
          <td>
            Liang Yue, Wenhao Jiang, Sainan Li, Meng Luo, Ning Fan, Xiaolu Zhan, Kexin Chen, Tian Lu, Fang Guo, Dongwei Li, Weigang Ge, Zongxiang Nie, M. Lyu, A. Jun, Yingrui Wang, Yingdan Chen, Zhenhai Fu, Nan Xiang, Lu Li, Fengchao Yu, Guo Ci Teo, Alexey I. Nesvizhskii, Meng Wang, Michael P. Snyder, Ben C. Collins, Ruedi Aebersold, Fei Xu, Yingchuan Zhu, Tong Liu, Yan Li, Tiannan Guo
          </td>
          <td>2025-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background CD4+ T cells are a highly differentiated cell type that maintain enough transcriptomic plasticity to cycle between activated and memory statuses. How the 1D chromatin state and 3D chromatin architecture support this plasticity is under intensive investigation. Methods Here, we wished to test a commercially available in situ Hi-C kit (Arima Genomics Inc.) to establish whether published performance on limiting cell numbers from clonal cell lines copies across to a primary immune cell type. We achieved comparable contact matrices from 50,000, 250,000, and 1,000,000 memory CD4+ T-cell inputs. We generated multiple Hi-C and RNA-seq libraries from the same biological blood donors under three separate conditions: unstimulated fresh ex vivo, IL-2-only stimulated, and T cell receptor (TCR)+CD28+IL-2-stimulated, conferring increasingly stronger activation signals. We wished to capture the magnitude and progression of 3D chromatin shifts and correlate these to expression changes under the two stimulations. Results Although some genome organization changes occurred concomitantly with changes in gene expression, at least as many changes occurred without corresponding changes in expression. Counter to the hypothesis that topologically associated domains (TADs) are largely invariant structures providing a scaffold for dynamic looping contacts between enhancers and promotors, we found that there were at least as many dynamic TAD changes. Stimulation with IL-2 alone triggered many changes in genome organization, and many of these changes were strengthened by additional TCR and CD28 co-receptor stimulation. Conclusions This suggests a stepwise process whereby mCD4+ T cells undergo sequential buildup of 3D architecture induced by distinct or combined stimuli likely to “prime” or “deprime” them for expression responses to subsequent TCR-antigen ligation or additional cytokine stimulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1aa3f310b47dde1e0fdd44c2b4206007ebe632" target='_blank'>
              Memory CD4+ T cells sequentially restructure their 3D genome during stepwise activation
              </a>
            </td>
          <td>
            Alexander I. Ward, J. I. de las Heras, Eric C. Schirmer, A. Fassati
          </td>
          <td>2025-02-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cancer metastasis, a process in which cancer cells migrate to secondary sites, accounts for 90% of cancer deaths. While machine learning has been used to predict metastasis in a variety of ways, they tend to be specific to tumor types or classification tasks. Here, we provide a pan-cancer machine learning model that is easily interpretable and directly links the transcriptomic and proteomic profiles of hundreds of cancer cell lines from DepMap with metastasis in a tissue-agnostic manner. We show that, for the relatively small sample size here, linear models perform just as well as non-linear models, including neural networks. We also demonstrate that transcriptomics outperforms proteomics for predictive tasks. However, the combination of both transcriptomics and proteomics leads to an overall increase in performance, indicating that these modalities can complement each other. Finally, we demonstrate that those features identified as most important by the joint omics linear model fit are over-represented in biological pathways related to metastasis, suggesting that the model learns relevant information that may be useful in identifying universal biomarkers or therapeutic targets of cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4778538fa7f9077b250641d66da47d7e300ae2d7" target='_blank'>
              Joint linear modeling of transcriptomics and proteomics is predictive of cancer metastasis
              </a>
            </td>
          <td>
            Ronit Sharma, N. Meimetis, Arjana Begzati, S. Nagar, Benjamin Kellman, Hratch M. Baghdassarian
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f6763865a99a9f23291b576c73a041b68eb0a51" target='_blank'>
              Bioengineering facets of the tumor microenvironment in 3D tumor models: insights into cellular, biophysical and biochemical interactions.
              </a>
            </td>
          <td>
            Salma T. Rafik, D. Bakkalci, Alexander J MacRobert, Umber Cheema
          </td>
          <td>2025-03-27</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Numerous computational approaches have been developed to infer cell state transition trajectories from snapshot single-cell data. Most approaches first require projecting high-dimensional data onto a low-dimensional representation, raising the question of whether the dynamics of the system become distorted. Using epithelial-to-mesenchymal transition (EMT) as a test system, we show that both biology-guided low-dimensional representations and stochastic trajectory simulations in high-dimensional state space, not representations obtained with brute force dimension-reduction methods, reveal multiple distinct paths of TGF-β-induced EMT. The paths arise from coupling between EMT and cell cycle arrest at either the G1/S, G2/M or M checkpoints, contributing to cell-cycle related EMT heterogeneity. The present study emphasizes that caution should be taken when inferring transition dynamics from snapshot single-cell data in two- or three-dimensional representations, and that incorporating dynamical information can improve prediction accuracy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/027bbe85f7ff2e7ff1b267ad097635ce95f3e630" target='_blank'>
              Epithelial-mesenchymal transition couples with cell cycle arrest at various stages
              </a>
            </td>
          <td>
            Sophia Hu, Yong Lu, Gaohan Yu, Zhiqian Zheng, Weikang Wang, Ke Ni, Amitava Giri, Jingyu Zhang, Yan Zhang, Kazuhide Watanabe, Guang Yao, Jianhua Xing
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer driver mutations are defined by their high prevalence in cancers and presumed rarity in normal tissues. However, recent studies show that positive selection in normal epithelia can increase the prevalence of some cancer drivers. To determine their true cancer-driving potential, it is essential to evaluate how frequent these mutations are in normal tissues and what are their phenotypes. Here, we explore the bioavailability of somatic variants by quantifying age-related mutational burdens in normal human colonic epithelium using immunodetection in FFPE samples (N = 181 patients). Positive selection of variants of tumour suppressor genes PTEN and ARID1A associates with monoallelic gene loss as confirmed by CRISPR/Cas9 mutagenesis and changes in their downstream effectors. Comparison of the mutational burden in normal tissue and colorectal cancers allows quantification of cancer driver potency based on relative representation. Additionally, immune exclusion, a cancer hallmark feature, is observed within ARID1A-deficient clones in histologically normal tissue. The behaviour resulting from haploinsufficiency of PTEN and ARID1A demonstrates how somatic mosaicism of tumour suppressors arises and can predispose to cancer initiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1db80b341443231830586b45a4d64283910100" target='_blank'>
              Identification of increased dedifferentiation along the Prom1+ cancer cells in Müllerian adenosarcoma with sarcomatous overgrowth.
              </a>
            </td>
          <td>
            Xingming Ye, Jianfeng Zheng, Dan Hu, Li Liu, Fukun Chen, Xintong Cai, Yangmei Xu, Lifeng Li, Jie Lin, Qinying Liu, Yang Sun
          </td>
          <td>2025-02-07</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related morbidity and mortality worldwide, driven by a complex interplay of genetic, environmental, and immune-related factors. Among the pivotal pathways implicated in CRC tumorigenesis, the Notch signaling pathway is instrumental in governing cell fate decisions, tissue renewal, homeostasis, and immune cell development. As a highly conserved mechanism, Notch signaling not only modulates tumor cell behavior but also shapes the immune landscape within the tumor microenvironment (TME). Aberrant Notch signaling in CRC fosters immune evasion and tumor progression through its effects on the balance and functionality of immune cells, including myeloid-derived suppressor cells (MDSCs) and tumor-associated macrophages (TAMs). Elevated Notch pathway activation correlates with advanced clinicopathological features and poorer clinical outcomes, highlighting its relevance as both a prognostic biomarker and a therapeutic target. Therapeutic approaches aimed at inhibiting the Notch pathway, such as γ-secretase inhibitors (GSIs) or monoclonal antibodies (mAbs) in combination with other therapies, have demonstrated promising efficacy in preclinical and clinical settings. This review examines the impact of Notch signaling on CRC immunity, elucidating its regulatory mechanisms within immune cells and its role in promoting tumor progression. Additionally, this review discusses therapeutic strategies targeting Notch signaling, including GSIs, mAbs, and potential combination therapies designed to overcome resistance and improve patient outcomes. By elucidating the multifaceted role of Notch within the CRC TME, this review underscores its potential as a target for innovative therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d839257a78406c4ae8795716db880ea18e9d5180" target='_blank'>
              Notch signaling in the tumor immune microenvironment of colorectal cancer: mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Jiachun Sun, Yi Chen, Ziyi Xu, Weizheng Wang, Penghui Li
          </td>
          <td>2025-03-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Glioblastoma (GBM) is the most common and aggressive primary malignant brain tumor of adults. Current therapeutic options yield dismal prognoses that have remained essentially unchanged over nearly two decades. Diffuse growth patterns, high intratumoral heterogeneity, and variable blood–brain barrier integrity limit treatment efficacy, creating challenges that rational small molecule design has not overcome. Antibody–drug conjugates have shown some promise, leading us to hypothesize that smaller folded DNA aptamers, developed in vivo via principles of natural selection, might eventually have advantages for drug delivery. Here, we document the first in vivo DNA aptamer selection involving an orthotopic patient-derived xenograft GBM mouse model to identify tumor-homing DNA aptamers. We demonstrate the preferential accumulation of these aptamers in the tumor relative to other tissues 4 h after intraperitoneal injection. The aptamers can be detected by quantitative polymerase chain reaction, fluorescent tumor staining, and stain GBM tumor section from untreated mice and the GBM tumor cells in culture. Two of three candidates are selective for the target cell line in vitro and do not bind other human tumor cells. In vivo selection of tumor-specific DNA aptamers demonstrates a novel approach for diagnostics or toxin delivery that might allow for the development of individualized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a69e4c2d2724fca734736e524ece9241f19793a" target='_blank'>
              In vivo selection of anti-glioblastoma DNA aptamers in an orthotopic patient-derived xenograft model
              </a>
            </td>
          <td>
            Caroline D. Doherty, Brandon A. Wilbanks, Sonia Jain, Keenan S Pearson, Katie K Bakken, D. Burgenske, Nay Won Lett, J. Sarkaria, L. Maher
          </td>
          <td>2025-02-18</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Components of normal tissue architecture serve as barriers to tumor progression. Inflammatory and wound-healing programs are requisite features of solid tumorigenesis, wherein alterations to immune and non-immune stromal elements enable loss of homeostasis during tumor onset. The precise mechanisms by which normal stromal cell states limit tissue plasticity and tumorigenesis, and which are lost during tumor progression, remain largely unknown. Here we show that healthy pancreatic mesenchyme expresses the paracrine signaling molecule KITL, also known as stem cell factor, and identify loss of stromal KITL during tumorigenesis as tumor-promoting. Genetic inhibition of mesenchymal KITL in the contexts of health, injury, and cancer together indicate a role for KITL signaling in maintenance of pancreas tissue architecture, such that loss of the stromal KITL pool increased tumor growth and reduced survival of tumor-bearing mice. Together, these findings implicate loss of mesenchymal KITL as a mechanism for establishing a tumor-permissive microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b05f1579f2dda835515d867e6c5a5ccd6f715258" target='_blank'>
              Stromal KITL/SCF maintains pancreas tissue homeostasis and restrains tumor progression.
              </a>
            </td>
          <td>
            M. K. Oñate, Chet Oon, Sohinee Bhattacharyya, V. Low, Canping Chen, Xiaofan Zhao, F. Arnold, Ziqiao Yan, Sneha Pramod, Yan Hang, Yu-jui Ho, Scott W. Lowe, Seung K Kim, Zheng Xia, Mara H Sherman
          </td>
          <td>2025-02-10</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Colon adenocarcinoma (COAD) demonstrates significant clinical heterogeneity across disease stages, gender, and age groups, posing challenges for unified therapeutic strategies. This study establishes a multi-dimensional stratification framework through integrative analysis of miRNA–gene co-expression networks, employing the MRNETB algorithm coupled with Markov flow entropy (MFE) centrality quantification. Analysis of TCGA-COAD cohorts revealed stage-dependent regulatory patterns centered on CDX2-hsa-miR-22-3p-MUC13 interactions, with progressive dysregulation mirroring tumor progression. Gender-specific molecular landscapes have emerged, characterized by predominant SLC26A3 expression in males and GPA33 enrichment in females, suggesting divergent pathogenic mechanisms between genders. Striking age-related disparities were observed, where early-onset cases exhibited molecular signatures distinct from conventional COAD, highlighted by marked XIST expression variations. Drug-target network analysis identified actionable candidates including CEACAM5-directed therapies and differentiation-modulating agents. Our findings underscore the critical need for heterogeneity-aware clinical decision-making, providing a roadmap for stratified intervention paradigms in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821868482387376bff6e5080e3307f3abd178c5a" target='_blank'>
              Decoding Colon Cancer Heterogeneity Through Integrated miRNA–Gene Network Analysis
              </a>
            </td>
          <td>
            Qingcai He, Zhilong Mi, Tianyue Liu, Taihang Huang, Mao Li, Binghui Guo, Zhiming Zheng
          </td>
          <td>2025-03-20</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and lethal malignancy, with a five-year survival rate of just 50% for cases of locally advanced disease. Chromosomal aberrations, particularly the deletion of the short arm of chromosome 3 (3p), have been strongly associated with poor prognosis and more aggressive tumor phenotypes. The tumor microenvironment (TME) plays a pivotal role in tumor progression and resistance to therapy. This study aims to elucidate the impact of 3p deletion on the TME, immune cell infiltration, and treatment response in HNSCC, to identify novel therapeutic targets to improve patient outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from the Gene Expression Omnibus (GEO) and bulk transcriptome data from The Cancer Genome Atlas (TCGA). Pseudo-time trajectory and cell-cell communication analyses were performed with the Monocle and CellChat packages. The Wilcoxon test was used to evaluate the differential gene expression between wild-type (wt) and mutant (mut) groups. Prognostic models were developed using the Least Absolute Shrinkage and Selection Operator (LASSO) algorithm and Cox regression analyses to find the genes related to survival, with survival analysis conducted via Kaplan-Meier curves. Gene set enrichment analysis (GSEA) was employed to investigate pathway dysregulation, and immune cell infiltration was assessed using various immune scoring methodologies to explore the differences immune environment. The Tumor Immune Dysfunction and Exclusion (TIDE) database was utilized to predict the potential efficacy of immune checkpoint inhibitors. mRNA and protein expression levels of SPP1 were examined by RT-qPCR and Western blotting, while cell proliferation was assessed using the CCK8 assay. The mut group demonstrated significant alterations in cellular composition, characterized by increased endothelial cells and macrophages and decreased fibroblasts and CD8 + T cells, indicative of an immunosuppressive TME. Differential expression analysis revealed downregulation of immune pathways, including antigen processing and presentation, T cell receptor signaling, and B cell receptor signaling pathways in the mut group, along with enhanced metabolic activity in glycolysis and lipid metabolism. The prognostic model identified nine key genes associated with poor survival in HNSCC. The mut group exhibited poorer overall survival and a more immunosuppressive microenvironment compared to the wt group, which correlated with the outcomes observed in high-risk versus low-risk groups. High-risk patients also showed a diminished response to immunotherapy compared to low-risk patients. Additionally, SPP1 emerged as a critical gene associated with chemotherapy resistance and macrophage M2 polarization. This study demonstrates that 3p deletion significantly reshapes the TME, contributing to poor prognosis in HNSCC by fostering an immunosuppressive environment and enhancing chemoresistance. These findings highlight the potential for developing targeted therapies that address the genetic and immunological landscape of HNSCC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92078-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="To better understand the homeostatic mechanisms governing melanocytes, we performed deep phenotyping of clonal expansions of single melanocytes from human skin. In total, we interrogated the mutational landscapes, gene expression profiles, and morphological features of 297 melanocytes from 31 donors. To our surprise, a population of melanocytes with low mutation burden was maintained in sun damaged skin. These melanocytes were more stem-like, smaller, less dendritic and displayed distinct gene expression profiles compared to their counterparts with high mutation burdens. We used single-cell spatial transcriptomics (10X Xenium) to reveal the spatial distribution of melanocytes inferred to have low and high mutation burdens (LowMut and HighMut cells), based on their gene expression profiles. LowMut melanocytes were found in hair follicles as well as in the interfollicular epidermis, whereas HighMut melanocytes resided almost exclusively in the interfollicular epidermis. We propose that melanocytes in the hair follicle occupy a privileged niche, protected from UV radiation, but periodically migrate out of the hair follicle to replenish the interfollicular epidermis after waves of photodamage. More broadly, our study illustrates the advantages of a cell atlas that includes mutational information, as cells can change their cellular states and positional coordinates over time, but mutations are like scars, providing a historical record of the homeostatic processes that were operative on each cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35fcdb72f7ddf292a52ab622c06a7d378c43f17" target='_blank'>
              Somatic mutations distinguish melanocyte subpopulations in human skin
              </a>
            </td>
          <td>
            Bishal Tandukar, Delahny Deivendran, Limin Chen, Neda Bahrani, Beatrice Weier, Harsh Sharma, Noel Cruz-Pacheco, Min Hu, Kayla Marks, Rebecca G. Zitnay, Aravind K. Bandari, Rojina Nekoonam, I. Yeh, R. Judson-Torres, A. H. Shain
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01243dcc84f5d3d7fbdd4aa36ab8b1b63e350fae" target='_blank'>
              Abstract A086: Myeloid suppression within metastatic tumor microenvironments in advanced breast cancer—friend or foe to checkpoint inhibition?
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Mathew Jacobo, Michael F. Press, E. R. Torres
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Intrahepatic cholangiocarcinoma (iCCA) may be genomically subclassified by the presence of potentially actionable molecular aberrations, of which pathogenic alterations in isocitrate dehydrogenase (IDH)1 and fibroblast growth factor receptor (FGFR)2 are the most frequently observed. The impact of these molecular alterations on the tumor immune microenvironment remains incompletely understood. Methods: We performed a high-parameter spatial immune phenotyping of iCCA samples with pathogenic FGFR2 or IDH1 alterations and FGFR2/IDH1 wild-type controls at the single-cell level using CO-Detection by indEXing. Results:﻿ A total of 24 tumors were examined. Tumors with FGFR2 alterations were characterized by fewer CD8+ T cells and “M2-like” macrophages but higher levels of polymorphonuclear myeloid-derived suppressor cells as compared to FGFR2 wild-type tumors. Spatial relationships between polymorphonuclear myeloid-derived suppressor cells and multiple other cell types in the tumor microenvironment (including tumor cells, CD4+, and CD8+ T cells) were enriched in tumors with FGFR2 alterations. Tumors with IDH1 mutations had a trend toward more fibroblasts and were characterized by a closer proximity of tumor cells to CD4+ T cells, and between macrophages and multiple structural tumor microenvironment components as compared to other subtypes. Conclusions: iCCAs with pathogenic FGFR2 fusions/rearrangements and IDH1 mutations have distinct immunophenotypes. Tailoring immunotherapeutic approaches to specific molecular subsets could improve treatment outcomes across the divergent molecularly defined iCCA subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/343c87af8f0d2b156de3f86c823598f7aa36e2b7" target='_blank'>
              Deep immune profiling of intrahepatic cholangiocarcinoma with CODEX multiplexed imaging
              </a>
            </td>
          <td>
            M. Baretti, Soumya Shekhar, Vaibhav Sahai, Daniel H. Shu, Kathryn Howe, Valerie Gunchick, N. Assarzadegan, Emma Kartalia, Qingfeng Zhu, E. Hallab, Archit Sheth-Shah, Aya Kondo, Nilofer S Azad, M. Yarchoan
          </td>
          <td>2025-02-19</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Introduction Pancreatic adenocarcinoma (PAAD) is characterized by a profoundly immunosuppressive tumor microenvironment (TME) that limits the efficacy of immunotherapy. Emerging evidence suggests that tumor-specific metabolic reprogramming may drive disease progression and shape the immune landscape in PAAD. Methods We integrated multi-omics data from TCGA, GEO, and ICGC to identify key metabolism-related genes (MRGs) that influence immune cell infiltration, tumor progression, and patient survival. Based on nine pivotal MRGs (including ANLN, PKMYT1, and HMGA1), we developed and validated a novel metabolic-prognostic index (MPI). Functional enrichment analyses were conducted to elucidate the metabolic pathways associated with different MPI risk groups. In vitro experiments and drug sensitivity analyses were performed to confirm the oncogenic role of selected MRGs and to explore their therapeutic implications. Results The MPI effectively stratified patients into high- and low-risk groups. High-MPI scores correlated with poor overall survival, elevated tumor mutation burden (TMB), and an immunosuppressive TME, evidenced by reduced CD8⁺ T-cell infiltration and increased expression of immune checkpoints (PD-L1, TGF-β). Functional enrichment revealed glycolysis and folate biosynthesis as dominant pathways in high-MPI groups, whereas fatty acid metabolism prevailed in low-MPI groups. Experimental validation underscored the role of ANLN in promoting epithelial-mesenchymal transition (EMT) and immune evasion via NF-κB signaling. ANLN knockdown significantly reduced glycolytic activity, tumor cell migration, and immune evasion. Drug sensitivity analyses indicated resistance to gemcitabine but sensitivity to afatinib in high-MPI patients. Although TIDE analysis predicted immune checkpoint inhibitor (ICI) resistance in high-MPI tumors, a subset of patients showed favorable responses to anti-PD-L1 therapy. Discussion These findings provide a comprehensive framework for understanding how metabolic reprogramming shapes PAAD’s immunosuppressive TME and affects treatment outcomes. By accurately stratifying patients, the MPI serves as a promising tool to guide therapeutic decisions, including targeted therapy selection and immunotherapy prediction, ultimately offering potential for more personalized management of PAAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42608b06fe89e88fa2b364a076f0a8a5c2fa634d" target='_blank'>
              Metabolic reprogramming shapes the immune microenvironment in pancreatic adenocarcinoma: prognostic implications and therapeutic targets
              </a>
            </td>
          <td>
            Weihua Song, Yabin Yu, Siqi Wang, Zhengyi Cui, Qiusi Zhu, Wangrui Liu, Shiyin Wei, Jiachang Chi
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/903982cf411415d6dcdb66121d6c8e705f6172b9" target='_blank'>
              The Role of Cancer Stem Cells in Uveal Melanoma
              </a>
            </td>
          <td>
            Piotr Świerzewski, Bożena Romanowska-Dixon, Martyna Elas
          </td>
          <td>2025-02-14</td>
          <td>Ophthalmology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Understanding the proteomic-level heterogeneity of the tumor microenvironment (TME) in colorectal cancer (CRC) is crucial due to its well-known heterogeneity. While heterogenous CRC has been extensively characterized at the molecular subtype level, research into the functional heterogeneity of fibroblasts, particularly their relationship with extracellular matrix (ECM) alterations, remains limited. Addressing this gap is essential for a comprehensive understanding of CRC progression and the development of targeted therapies.


METHODS
24 tissue samples from 21 CRC patients, along with adjacent normal tissues (NAT), were collected and decellularized using a detergent-based method to enrich the ECM component. Proteomic analysis of ECM-enriched samples was performed using tandem mass tag (TMT) spectrometry, followed by statistical analysis including differential expression protein (DEP) analysis. Single-cell RNA sequencing (scRNA-Seq) data from public datasets were integrated and analyzed to delineate cell states within the TME. Bulk tissue RNA-Seq and bioinformatics analysis, including consensus molecular subtype (CMS) classification and single-cell level deconvolution of TCGA bulk RNA-seq data, were conducted to further explore gene expression patterns and TME composition.


RESULTS
Differential cellular origin of the NAT and tumorous ECM proteins were identified, revealing 110 ECM proteins enriched in NAT and 28 ECM proteins in tumor tissues. Desmoplastic and WNT5A+ inflammatory fibroblasts were indicated as the sources of tumor-enriched ECM proteins, while ADAMDEC1+ expressing fibroblasts and PI16+ expressing fibroblast were identified as the sources of NAT-enriched ECM proteins. Deconvolution of bulk RNA-seq of CRC tissues discriminated CMS-specific fibroblast state, reflecting the biological traits of each CMS subtype. Specially, seven ECM genes specific to mesenchymal subtype (CMS4), including PI16+ fibroblast-related 4 genes (SFRP2, PRELP, OGN, SRPX) and desmoplastic fibroblast-related 3 genes (THBS2, CTHRC1, BGN), showed a significant association with poorer survival in patient with CRC.


CONCLUSION
We conducted an extracellular matrix (ECM)-focused profiling of the TME by integrating quantitative proteomics with single-cell RNA sequencing (scRNA-seq) data from CRC patients. We identified the ECM proteins of NAT and tumor tissue, and established a cell-matrisome database. We defined mesenchymal subtype-specific molecules associated with specific fibroblast subtypes showing a significant association with poorer survival in patients with CRC. Our ECM-focused profiling of tumor stroma provides new insights as indicators for biological processes and clinical endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2f30006d88490586d800953c99040977f4d92b" target='_blank'>
              Differential cellular origins of the extracellular matrix of tumor and normal tissues according to colorectal cancer subtypes.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Sangkeun Park, Jun Hyeong Lee, Shin Young Chang, Sang Mi Oh, S. Mun, Junho Kang, Jong-Eun Park, Jung Kyoon Choi, Tae Il Kim, Jin Young Kim, Pilnam Kim
          </td>
          <td>2025-03-03</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The presence of pre-existing or acquired drug-resistant cells within the tumor often leads to tumor relapse and metastasis. Single-cell RNA sequencing (scRNA-seq) enables elucidation of the subtle differences in drug responsiveness among distinct cell subpopulations within tumors. A few methods have employed scRNA-seq data to predict the drug response of individual cells to date, but their performance is far from satisfactory. In this study, we propose SSDA4Drug, a semi-supervised few-shot transfer learning method for inferring drug-resistant cancer cells. SSDA4Drug extracts pharmacogenomic features from both bulk and single-cell transcriptomic data using semi-supervised adversarial domain adaptation. This allows us to transfer knowledge of drug sensitivity from bulk-level cell lines to single cells. We conduct extensive performance evaluation experiments across multiple independent scRNA-seq datasets, demonstrating SSDA4Drug’s superior performance over current state-of-the-art methods. Remarkably, with only one or two labeled target-domain samples, SSDA4Drug significantly boosts the predictive performance of single-cell drug responses. Moreover, SSDA4Drug accurately recapitulates the temporally dynamic changes of drug responses during continuous drug exposure of tumor cells, and successfully identifies reversible drug-responsive states in lung cancer cells, which initially acquire resistance through drug exposure but later restore sensitivity during drug holidays. Also, our predicted drug responses consistently align with the developmental patterns of drug sensitivity observed along the evolutionary trajectory of oral squamous cell carcinoma cells. In addition, our derived SHAP values and integrated gradients effectively pinpoint the key genes involved in drug resistance in prostate cancer cells. These findings highlight the exceptional performance of our method in determining single-cell drug responses. This powerful tool holds the potential for identifying drug-resistant tumor cell subpopulations, paving the way for advancements in precision medicine and novel drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4755a58204169a2b082c843078f1434fb58c7419" target='_blank'>
              Identification of drug-resistant individual cells within tumors by semi-supervised transfer learning from bulk to single-cell transcriptome
              </a>
            </td>
          <td>
            Kaishun Huang, Hui Liu
          </td>
          <td>2025-03-31</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c654520970222f7cd3694a92d7a5cfe3581a7e0" target='_blank'>
              Identification of new biomarkers of hepatic cancer stem cells through proteomic profiling.
              </a>
            </td>
          <td>
            S. H. Choi, Ha Young Lee, Sung Ho Yun, Sung Jae Jang, Seung-Up Kim, Jun Yong Park, S. Ahn, D. Y. Kim
          </td>
          <td>2025-03-20</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0608cf84711b7461bf4c33725871d3360fc3c9a5" target='_blank'>
              Abstract B047: Stromal senescence and response to immune checkpoint inhibition in melanoma
              </a>
            </td>
          <td>
            Giselle Burton Sojo, Marcel O. Schmidt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Fanconi Anemia (FA) is a heritable syndrome characterized by DNA damage repair deficits, frequent malformations and a significantly elevated risk of bone marrow failure, leukemia, and mucosal head and neck squamous cell carcinomas (HNSCC). Hematopoietic stem cell gene therapy can prevent marrow failure and lower leukemia risk, but mucosal gene therapy to lower HNSCC risk remains untested. Major knowledge gaps include an incomplete understanding of how rapidly gene-corrected cellular lineages could spread through the oral epithelium, and which delivery parameters are critical for ensuring efficient gene correction. To answer these questions, we extended an agent-based model of the oral epithelium to include the delivery of gene correction in situ to FA cells and the competitive dynamics between cellular lineages with and without gene correction. We found that only gene-corrected lineages with substantial proliferative advantages (probability of resisting displacement out of the basal layer ≥ 0. 1) could spread on clinically relevant timelines, and that these lineages were initially at high risk of loss in the generations following correction. Delivering gene correction to many cells minimizes the risk of loss, while delivery to many distinct locations within a tissue maximizes the rate of spread. To determine the impact of mucosal gene therapy in preventing the clonal expansion of pre-cancerous mutations, we compared the expected burden of TP53 mutations in simulated tissue sections with and without gene correction. We found that when FA cells have elevated genome instability or a TP53-dependent proliferative advantage, gene correction can substantially reduce the accumulation of pro-tumorigenic mutations. This model illustrates the power of computational frameworks to identify critical determinants of therapeutic success to enable experimental optimization and support novel and effective gene therapy applications. Author summary We investigated factors influencing the success of oral mucosal gene therapy for Fanconi Anemia (FA), a genetic syndrome marked by DNA repair defects in conjunction with a heightened risk of cancer. We used a computational model of the oral epithelium to determine how gene therapy corrected cells compete with FA background cells and the best gene delivery approaches to promote effective tissue replacement by gene-corrected cells. We find that gene-corrected cells require strong proliferative advantages to spread effectively, and that initially delivering gene correction to more cells reduces the chance that these cells are stochastically eliminated before they can spread. We also demonstrate that gene correction reduces the accumulation of pro-tumorigenic TP53 mutations in an FA context, where genomic instability can elevate the mutation rate and FA-specific selective pressures could favor accelerated TP53 clonal expansion. This research provides a useful framework for guiding mucosal gene therapy experiments and the development of effective oral gene therapy protocols for cancer prevention in FA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb3b56fe96b81aec45dd517d7b11cc8ca9c35" target='_blank'>
              Epithelial competition determines gene therapy potential to suppress Fanconi Anemia oral cancer risk
              </a>
            </td>
          <td>
            Hunter L. Colegrove, Raymond J Monnat, Alison F Feder
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7261a40e65b3cb85eca64d1251012a027c2cf0ef" target='_blank'>
              Epigenetic Symphony in Diffuse Large B-Cell Lymphoma: Orchestrating the Tumor Microenvironment
              </a>
            </td>
          <td>
            A. Caloian, Miruna Cristian, Elena Calin, Andreea-Raluca Pricop, Stelian Mociu, Liliana Seicaru, Sorin Deacu, Nicolae Ciufu, A. Suceveanu, A. Suceveanu, L. Mazilu
          </td>
          <td>2025-04-02</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="MOTIVATION
Spatial transcriptomics holds great promise for revolutionizing biology and medicine by providing gene expression profiles with spatial information. Until recently, spatial resolution has been limited, but advances in high-throughput in situ imaging technologies now offer new opportunities by covering thousands of genes at a single-cell or even subcellular resolution, necessitating databases dedicated to comprehensive coverage and analysis with user-friendly intefaces.


RESULTS
We introduce the HISSTA database, which facilitates the archival and analysis of in situ transcriptome data at single-cell resolution from various human tissues. We have collected and annotated spatial transcriptome data generated by MERFISH, CosMx SMI, and Xenium techniques, encompassing 112 samples and 28 million cells across 16 tissue types from 63 studies. To decipher spatial contexts, we have implemented advanced tools for cell type annotation, spatial colocalization, spatial cellular communication, and niche analyses. Notably, all datasets and annotations are interactively accessible through Vitessce, allowing users to focus on regions of interest and examine gene expression in detail. HISSTA is a unique database designed to manage the rapidly growing dataset of in situ transcriptomes at single-cell resolution. Given its comprehensive data content and advanced analysis tools with interactive visualizations, HISSTA is poised to significantly impact cancer diagnosis, precision medicine, and digital pathology.


AVAILABILITY AND IMPLEMENTATION
HISSTA is freely accessible at https://kbds.re.kr/hissta/. The source code is available at https://doi.org/10.5281/zenodo.14904523.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8847bc6bc6ac698124acae8ba2d55144bc394fe" target='_blank'>
              HISSTA: a human in situ single-cell transcriptome atlas.
              </a>
            </td>
          <td>
            Jiwon Yu, Jiwoo Moon, Minseo Kim, Gyeol Han, Insu Jang, Jinyoung Lim, Seungmook Lee, Seok-Hwan Yoon, Woong-Yang Park, Byungwook Lee, Sanghyuk Lee
          </td>
          <td>2025-03-31</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Tumor dissemination is a key event in tumor progression. During this event, a main role is played by circulating tumor cells (CTCs), immune cells, and their interaction. How the immune system supports the survival and proliferation of CTCs is not fully elucidated. In this study we established an in-vitro co-culture system consisting of immune cells and CTCs from the same patient, which increased the success rate in the establishment of CTC-derived long-term cell cultures. In this system, we characterized the immune cells of successful co-cultures and the signals they exchange with cancer cells, including cytokines and extracellular vesicle (EV) content. Using this protocol, we stabilized four CTC-derived cell lines from patients with metastatic gastroesophageal cancer, which were cultured for over a year and characterized from a genetic and molecular point of view. The four cell lines harbor shared chromosomal aberrations including the amplification at 8q24.21 containing MYC and deletion 9p21.3 containing CDKN2A/B and the IFN type I cluster. The transcriptomic profile of CTC cell lines is distinct from primary tumors, and we detected the activation of E2F, G2M and MYC pathways and the downregulation of interferon response pathway. Each cell line shows a degree of invasiveness in zebrafish in-vivo, and the most invasive ones share the same mutation in RAB14 gene. In addition, the four cell lines secrete cell-line specific EVs containing microRNAs that target YAP, BRG1-AKT1, TCF8-HDAC pathways. Overall, we highlight how the immune system plays a key role in the proliferation of CTCs through EV signaling, and how CTC cell line genomic and transcriptomic alterations make these cells less visible from the immune system and likely responsible for the survival advantage in sites distant from the microenvironment of origin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fac3e4fbae2c7ba5a9e39e780e9dfb2e2cf33f" target='_blank'>
              Gastroesophageal circulating tumor cell crosstalk with peripheral immune system guides CTC survival and proliferation
              </a>
            </td>
          <td>
            T. Rossi, Martina Valgiusti, Maurizio Puccetti, Giacomo Miserocchi, Michele Zanoni, D. Angeli, Chiara Arienti, Ilaria Pace, C. Bassi, Ivan Vannini, M. Melloni, E. Bandini, Milena Urbini, Massimo Negrini, Massimiliano Bonafé, Manuela Ferracin, Giulia Gallerani
          </td>
          <td>2025-03-29</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d2d7a33282302100a7faa70ebce158c29857365" target='_blank'>
              Cancer-associated fibroblasts: multidimensional players in liver cancer
              </a>
            </td>
          <td>
            Yanyun Pan, Yuangang Qiu, Xinbin Zhou, Wei Mao, Xiao-ming Xu
          </td>
          <td>2025-04-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb5f12dc4ee23891e41c491f92b49e0c1d5cff9" target='_blank'>
              Targeting CXCR4-expressing TAMs in muscle-invasive bladder cancer to enhance tumor control after immunotherapy
              </a>
            </td>
          <td>
            E. Desponds, H. El ahanidi, Nagham Alouche, Hana Zdimerova, Stéphanie Favre, Giulio Zanette, Sina Nassiri, Daniel Benamran, Petros K Tsantoulis, Mohammed Attaleb, Hélène Maby-El Hajjami, Julien Dagher, Vanessa Gourhand, K. Balabanian, M. Espéli, S. Luther, Camilla Jandus, M. Leblond, G. Verdeil
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Despite advances, Glioblastoma (GBM) treatment remains challenging due to its rapid progression and resistance to therapies. Objectives This study aimed to investigate the role of anoikis—a mechanism by which cells evade programmed cell death upon detachment from the extracellular matrix—in GBM progression and prognosis. Methods Utilizing single-cell sequencing and bulk-transcriptome sequencing data from TCGA, GEO, and CGGA databases, we performed comprehensive bioinformatics analyses. We identified anoikis-related genes, constructed a prognostic model using 101 machine learning algorithms, and validated its clinical utility across multiple cohorts.Finally, we also verified the expression of model genes and the function of key gene in clinical samples and cell lines. Results Single-cell sequencing revealed heterogeneous expression of anoikis-related genes across distinct cell populations within GBM. MES-like Malignant cells and Myeloids exhibited higher enrichment of these genes, implicating their role in anoikis resistance and tumor aggressiveness. The prognostic model, based on identified genes, effectively stratified patients into high-risk and low-risk groups, demonstrating significant differences in survival outcomes. Mutation and tumor microenvironment analyses highlighted distinct genetic landscapes and immune cell infiltration patterns associated with different risk groups. SLC43A3 emerged as a key gene, showing significant upregulation in tumor tissues and correlating with poor prognosis in GBM. Conclusion This study provides insights into the molecular mechanisms of anoikis resistance in GBM, underscoring its critical role in tumor progression and patient prognosis. The developed prognostic model offers a promising tool for personalized treatment strategies and warrants further exploration of targeted therapies to improve outcomes for GBM patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03752-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90776aa9e4e44d828b89f11f0b82b3f7faa8093b" target='_blank'>
              Unraveling anoikis in glioblastoma: insights from single-cell sequencing and prognostic modeling
              </a>
            </td>
          <td>
            Q. Tang, Chenfeng Ma, Jiaheng Xie, Qixiang Zhang, Bingtao Zhang, Weiqi Bian, Qingyu Lu, Zeyu Wan, Wei Wu
          </td>
          <td>2025-03-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8677e695bbefb8bee72d8f6205f8c94375119315" target='_blank'>
              Developmental Origins and Oncogenesis in Medulloblastoma.
              </a>
            </td>
          <td>
            K. Kiang, Yogesh K H Wong, Soma Sengupta, Martine F. Roussel, Q. R. Lu
          </td>
          <td>2025-02-14</td>
          <td>Annual review of neuroscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6fe7c1118b5b40de590086593aa209906bee3a7" target='_blank'>
              Abstract A049: Application of a combinatorial functional precision medicine platform to predict immunotherapy response in DLBCL
              </a>
            </td>
          <td>
            Sharon Pei Yi Chan, J. Lim, M. Rashid, Edward Kai-Hua Chow
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cell death is critical in tumor biology. The common cancer therapies can cause cell death and alleviate tumor, while the cancer cells can develop a resistance to cell death and survive from the therapies. Thus, not only observing the alternative mechanisms of tumor cells resistant to cell death, but also understanding the intricate dynamics of cell death processes within the tumor microenvironment (TME), are essential for tailoring effective therapeutic strategies. High-throughput sequencing technologies have revolutionized cancer research by enabling comprehensive molecular profiling. Recent advances in single cell sequencing have unraveled the heterogeneity of TME components, shedding light on their complex interactions. In this review, we explored the interplay between cell death signaling and the TME, summarised the potential drugs inducing cell death in pre-clinical stage, reviewed some studies applying next-generation sequencing technologies in cancer death research, and discussed the future utilization of updated sequencing platforms in screening novel treatment methods targeted cell death. In conclusion, leveraging multi-omics technologies to dissect cell death signaling in the context of the TME holds great promise for advancing cancer research and therapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c40c61d7dcc3833421bbc3568f9b1a472c0c4795" target='_blank'>
              Cell death in tumor microenvironment: an insight for exploiting novel therapeutic approaches
              </a>
            </td>
          <td>
            Wenxin Wang, Tong Li, Kui Wu
          </td>
          <td>2025-03-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The tumor boundary of breast cancer represents a highly heterogeneous region. In this area, the interactions between malignant and non-malignant cells influence tumor progression, immune evasion, and drug resistance. However, the spatial transcriptional profile of the tumor boundary and its role in the prognosis and treatment response of breast cancer remain unclear.Utilizing the Cottrazm algorithm, we reconstructed the intricate boundaries and identified differentially expressed genes (DEGs) associated with these regions. Cell-cell co-positioning analysis was conducted using SpaCET, which revealed key interactions between tumor-associated macrophage (TAMs) and cancer-associated fibroblasts (CAFs). Additionally, Lasso regression analysis was employed to develop a malignant body signature (MBS), which was subsequently validated using the TCGA dataset for prognosis prediction and treatment response assessment.Our research indicates that the tumor boundary is characterized by a rich reconstruction of the extracellular matrix (ECM), immunomodulatory regulation, and the epithelial-to-mesenchymal transition (EMT), underscoring its significance in tumor progression. Spatial colocalization analysis reveals a significant interaction between CAFs and M2-like tumor-associated macrophage (TAM), which contributes to immune exclusion and drug resistance. The MBS score effectively stratifies patients into high-risk groups, with survival outcomes for patients exhibiting high MBS scores being significantly poorer. Furthermore, drug sensitivity analysis demonstrates that high-MB tumors had poor response to chemotherapy strategies, highlighting the role of the tumor boundary in modulating therapeutic efficacy.Collectively, we investigate the spatial transcription group and bulk data to elucidate the characteristics of tumor boundary molecules in breast cancer. The CAF-M2 phenotype emerges as a critical determinant of immunosuppression and drug resistance, suggesting that targeting this interaction may improve treatment responses. Furthermore, the MBS serves as a novel prognostic tool and offers potential strategies for guiding personalized treatment approaches in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b10699232e4b3f1c95f3a7d6be8ae3942b8fc0f" target='_blank'>
              Spatial multi-omics analysis of tumor-stroma boundary cell features for predicting breast cancer progression and therapy response
              </a>
            </td>
          <td>
            Yuanyuan Wu, Youyang Shi, Zhanyang Luo, Xiqiu Zhou, Yonghao Chen, Xiaoyun Song, Sheng Liu
          </td>
          <td>2025-03-26</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63a19cf7f5a4db8ff5f022ca9fbd2f6eba6cb2d9" target='_blank'>
              Abstract A021: Cancer-associated fibroblast-derived CXCL12 as a potential therapeutic target for pancreatic ductal adenocarcinoma immunotherapy
              </a>
            </td>
          <td>
            Liu Yang, Aiping Lu, Fangfei Li
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a751c087d4afc3ab315830aaec4e9b6e54e60f3" target='_blank'>
              Abstract A060: Reactive oligodendrocytes promote glioblastoma progression through CCL5/CCR5-mediated glioma stem cell maintenance
              </a>
            </td>
          <td>
            Jason Moffat
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d437c0ab0bc7d73f3a75511dfcad72400900dc7a" target='_blank'>
              Abstract B005: Identification of tumor-reactive TILs from lung cancer organoids co-culture through ruthenium-based proximity labeling
              </a>
            </td>
          <td>
            Weilei Hu, Yifei Bian, Zihan Zhao, Shian Ouyang, Jie P Li, Hongbin Ji
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background The tumor microenvironment (TME) plays a critical role in the development, progression, and clinical outcomes of hepatocellular carcinoma (HCC). Despite the critical role of natural killer (NK) cells in tumor immunity, there is limited research on their status within the tumor microenvironment of HCC. In this study, single-cell RNA sequencing (scRNA-seq) analysis of HCC datasets was performed to identify potential biomarkers and investigate the involvement of natural killer (NK) cells in the TME. Methods Single-cell RNA sequencing (scRNA-seq) data were extracted from the GSE149614 dataset and processed for quality control using the “Seurat” package. HCC subtypes from the TCGA dataset were classified through consensus clustering based on differentially expressed genes (DEGs). Weighted gene co-expression network analysis (WGCNA) was employed to construct co-expression networks. Furthermore, univariate and multivariate Cox regression analyses were conducted to identify variables linked to overall survival. The single-sample gene set enrichment analysis (ssGSEA) was used to analyze immune cells and the screened genes. Result A total of 715 DEGs from GSE149614 and 864 DEGs from TCGA were identified, with 25 overlapping DEGs found between the two datasets. A prognostic risk score model based on two genes was then established. Significant differences in immune cell infiltration were observed between high-risk and low-risk groups. Immunohistochemistry showed that HRG expression was decreased in HCC compared to normal tissues, whereas TUBA1B expression was elevated in HCC. Conclusion Our study identified a two-gene prognostic signature based on NK cell markers and highlighted their role in the TME, which may offer novel insights in immunotherapy strategies. Additionally, we developed an accurate and reliable prognostic model, combining clinical factors to aid clinicians in decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2c5ed50f3e7bc86c42a2f717d3b32238dbfc13e" target='_blank'>
              Uncovering the heterogeneity of NK cells on the prognosis of HCC by integrating bulk and single-cell RNA-seq data
              </a>
            </td>
          <td>
            Jiashuo Li, Zhenyi Liu, Gongming Zhang, Xue Yin, Xiaoxue Yuan, Wen Xie, Xiaoyan Ding
          </td>
          <td>2025-03-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The sentinel lymph node (SLN) is the first lymph node encountered by a metastatic cancer cell and serves as a predictor of poor prognosis, as patients with clinically occult SLN metastases are classified as stage III with elevated rates of recurrence and diminished overall survival. However, the dynamics of immune infiltrates in SLNs remain poorly characterized. Here, using an unbiased Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) technique, we profiled 97,777 cells from SLN tissues obtained from patients with stages I/II and III cutaneous melanoma. We describe the transcriptional programs of a multitude of T, B, and myeloid cell subtypes in SLNs. Based on the proportions of cell types, we determined that SLN subtypes are stratified along a naive → activated axis; patients with a 'high activated' signature score appear to be undergoing a robust melanoma antigen-driven adaptive immune response and thus, could be responsive to immunotherapy. Additionally, we identified transcriptomic signatures of SLN-infiltrating dendritic cell subsets that compromise anti-tumor immune responses. Our analyses provide valuable insights into tumor-driven immune changes in the SLN tissue, offering a powerful tool for the informed design of immune therapies for high-risk melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff6ed2d70346b46dbac007982b480af5731bdaf" target='_blank'>
              Single-cell transcriptomics of melanoma sentinel lymph nodes identifies unique immune cell signatures associated with metastasis.
              </a>
            </td>
          <td>
            Eric Engelbrecht, Bryce F Stamp, Lewis C. Chew, Omar S Sarkar, Phillip Harter, Sabine J Waigel, Eric C Rouchka, J. Chariker, Andrei Smolenkov, Jason A Chesney, K. McMasters, Corey T. Watson, K. Yaddanapudi
          </td>
          <td>2025-03-06</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Background Glioma, the most common primary cancer of the central nervous system, characterizes significant heterogeneity, presenting major challenges for therapeutic approaches and prognosis. In this study, the interactions between malignant glioma cells and macrophages/monocytes, as well as their influence on tumor progression and treatment responses, were explored using comprehensive single-cell RNA sequencing analysis. Methods RNA-seq data from the TCGA and CGGA databases were integrated and an in-depth analysis of glioma samples was performed using single-cell RNA sequencing, functional enrichment analysis, developmental trajectory analysis, cell–cell communication analysis, and gene regulatory network analysis. Furthermore, a prognostic model based on risk scores was developed, and its predictive performance was assessed through immune cell infiltration analysis and immune treatment response evaluation. Results Fourteen distinct glioma cellular subpopulations, seven primary cell types, and four macrophage/monocyte subtypes were identified. Developmental trajectory analysis offered insights into the origins and heterogeneity of malignant cells as well as macrophages/monocytes. Cell communication analysis revealed the interaction of macrophages and monocytes with malignant cells through several pathways, including the macrophage migration inhibitory factor and secreted phosphoprotein 1 pathways, engaging in key ligand-receptor interactions that influence tumor behavior. Categorization based on these communication characteristics was significantly correlated with overall survival. Immune cell infiltration analysis highlighted variations in immune cell abundance across different subgroups, possibly linked to differing responses to immunotherapy. This predictive model, comprising 29 prognostic genes, demonstrated high accuracy and robustness across multiple independent cohorts. Conclusion This study reveals the complex heterogeneity of the glioma microenvironment and enhances the understanding of diverse characteristics of glioma cell subsets. At the same time, it lays a foundation for the development of therapeutic strategies and prognostic models targeting the glioma microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b83f8e329366e14ce122de3c8f5608b479cd4df9" target='_blank'>
              Mapping glioma progression: single-cell RNA sequencing illuminates cell–cell interactions and immune response variability
              </a>
            </td>
          <td>
            Xia Li, Shenbo Chen, Ming Ding, Hui Ding, Kun Yang
          </td>
          <td>2025-03-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e24a79ed206cbada1bc6b51069582d272e00ef" target='_blank'>
              Machine learning-predicted chromatin organization landscape across pediatric tumors
              </a>
            </td>
          <td>
            Ketrin Gjoni, Shu Zhang, Rachel E. Yan, Bo Zhang, Daniel Miller, Adam Resnick, Nadia Dahmane, K. S. Pollard
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Newly diagnosed prostate cancers differ dramatically in mutational composition and lethality. The most accurate clinical predictor of lethality is tumor tissue architecture, quantified as tumor grade. To interrogate the evolutionary origins of prostate cancer heterogeneity, we analyzed 666 prostate tumor whole genomes. We identified a compendium of 223 recurrently mutated driver regions, most influencing downstream mutational processes and gene expression. We identified and validated individual germline variants that predispose tumors to acquire specific somatic driver mutations: these explain heterogeneity in disease presentation and ancestry differences. High-grade tumors have a superset of the drivers in lower-grade tumors, including increased frequency of BRCA2 and MYC mutations. Grade-associated driver mutations occur early in tumor evolution, and their earlier occurrence strongly predicts cancer relapse and metastasis. Our data suggest high- and low-grade prostate tumors both emerge from a common pre-malignant field, influenced by germline genomic context and stochastic mutation-timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9c23cc7532d469971ff1994cdcbf780a4650cf6" target='_blank'>
              The Germline and Somatic Origins of Prostate Cancer Heterogeneity.
              </a>
            </td>
          <td>
            Takafumi N. Yamaguchi, Kathleen E. Houlahan, Helen Zhu, Natalie J Kurganovs, J. Livingstone, Natalie S. Fox, Jiapei Yuan, Jocelyn Sietsma Penington, Chol-Hee Jung, T. Schwarz, Weerachai Jaratlerdsiri, J. van Riet, P. Georgeson, S. Mangiola, Kodi Taraszka, R. Lesurf, Jue Jiang, K. Chow, Lawrence E Heisler, Yu-Jia Shiah, Susmita G Ramanand, Michael J Clarkson, Anne Nguyen, S. M. G. Espiritu, R. Stuchbery, Richard Jovelin, Vincent Huang, Connor Bell, Edward O'Connor, Patrick J. McCoy, Christopher M. Lalansingh, Marek Cmero, A. Salcedo, Eva K F Chan, Lydia Y. Liu, Phillip D Stricker, Vinayak Bhandari, R. Bornman, Dorota H. Sendorek, A. Lonie, S. Prokopec, M. Fraser, J. Peters, Adrien Foucal, S. Mutambirwa, Lachlan Mcintosh, M. Orain, Matthew J. Wakefield, V. Picard, Daniel J Park, H. Hovington, Michael Kerger, A. Bergeron, Veronica Y. Sabelnykova, J-H. Seo, M. Pomerantz, Noah Zaitlen, Sebastian M. Waszak, Alexander Gusev, L. Lacombe, Yves Fradet, Andrew Ryan, A. Kishan, M. Lolkema, Joachim Weischenfeldt, Bernard Têtu, Anthony J Costello, Vanessa M Hayes, R. J. Hung, H. He, John D McPherson, Bogdan Pasaniuc, T. H. van der Kwast, AT Papenfuss, M. Freedman, B. Pope, Robert G Bristow, Ram-Shankar Mani, N. Corcoran, J. Reimand, Christopher M Hovens, P. Boutros
          </td>
          <td>2025-02-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31fd762af41845a9a3aad72616958b24bc4dd8f" target='_blank'>
              Chaperone-mediated autophagy regulates the metastatic state of mesenchymal tumors.
              </a>
            </td>
          <td>
            Xun Zhou, Eva Berenger, Yong Shi, Vera Shirokova, E. Kochetkova, Tina Becirovic, Boxi Zhang, Vitaliy O Kaminskyy, Yashar Esmaeilian, K. Hosaka, Cecilia Lindskog, P. Hydbring, Simon Ekman, Yihai Cao, Maria Genander, Marcin Iwanicki, E. Norberg, H. Vakifahmetoglu-Norberg
          </td>
          <td>2025-03-07</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Rationale: Glioblastoma (GBM) is the most fatal form of brain cancer and its treatment represents a persistent challenge. Mesenchymal stromal cells (MSCs) have been explored as therapeutic tools in cancer management owing to their tumor-homing abilities. However, their clinical application is limited due to the controversial role of MSCs in carcinogenesis. This study investigates how MSCs influence tumor behavior and explores the synergistic anticancer effects in combination with melatonin (Mel). Methods: Orthotopic and subcutaneous GBM xenograft mouse models were used to assess the antitumor effect of Mel pre-treated MSCs (MSCMel). Histological, immunohistochemical, and ultrastructural analyses were conducted to identify phenotypic changes in tumors. Through a set of in vitro assays, including direct and indirect co-cultures, dynamic single-cell tracking and tumorsphere assay, we explored the impact of MSCMel on primary and non-primary GBM cells. Transcriptomic profiling was used to identify genes and pathways modulated by this synergistic therapy. Results: MSCMel delayed tumor growth in mice and increased collagen deposition. Additionally, MSCMel showed enhanced capacity to prevent GBM cell migration compared to untreated MSCs. Molecular analysis identified genes and proteins related to cell migration, cytoskeletal dynamics and extracellular matrix remodeling in GBM cells exposed to MSCMel, including reduced vimentin expression. Finally, a genetic signature associated with the clinical outcomes of GBM patients was identified. Conclusions: Our study demonstrates that melatonin enhances the anticancer properties of MSCs, providing new insights into their interaction with GBM cells and tumor environment. These findings offer valuable guidance for advancing MSC-based therapies in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f57dedd75349ecd50ac992a7b9ebfe735e08696" target='_blank'>
              Glioblastoma progression is hindered by melatonin-primed mesenchymal stromal cells through dynamic intracellular and extracellular reorganizations
              </a>
            </td>
          <td>
            Laura Olmedo-Moreno, Concepción Panadero-Morón, J. M. Sierra-Párraga, Rubén Bueno-Fernández, E. Norton, Y. Aguilera, N. Mellado-Damas, Patricia García-Tárraga, Raquel Morales-Gallel, María Antequera-Martínez, Raúl V. Durán, Jaime Ferrer-Lozano, Germaine Escames, J. García-Verdugo, Alejandro Martín-Montalvo, Hugo Guerrero-Cázares, V. Capilla-González
          </td>
          <td>2025-02-10</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33b20af0b1c35c714611ceec53e3a323336edcf" target='_blank'>
              Myeloid cells: key players in tumor microenvironments.
              </a>
            </td>
          <td>
            Qiaomin Hua, Zhixiong Li, Yulan Weng, Yan Wu, Limin Zheng
          </td>
          <td>2025-03-06</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e110685415835cdd67555abc6bde1dc3e70baed" target='_blank'>
              Interactions Between Ploidy and Resource Availability Shape Clonal Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, J. Teer, Didem Ilter, Richard Beck, Jackson P Cole, Ana M Forero Pinto, Joanne D Tejero, Elisa Scanu, Thomas Veith, William Dominguez-Viqueira, Konstantin Maksin, Francisco Carrillo-Perez, Olivier Gevaert, Xiaonan Xu, F. Karreth, Mahmoud A Abdalah, Giada Fiandaca, Stefano Pasetto, Sandhya Prabhakaran, Andrew Schultz, A. M. E. Ojwang', J. Barnholtz-Sloan, Joaquim M Farinhas, Ana P. Gomes, Parag Katira, Noemi Andor
          </td>
          <td>2025-02-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a deadly cancer with an aggressive nature, and how CRC tumor cells manage to translocate and proliferate in a new tissue environment remains not fully understood. Recently, higher-order chromatin structures and spatial genome organization are increasingly implicated in diseases including cancer, but in-depth studies of three-dimensional genome (3D genome) of metastatic cancer are currently lacking, preventing the understanding of the roles of genome organization during metastasis. Here we perform multi-omics profiling of matched normal colon, primary tumor, lymph node metastasis, liver metastasis and normal liver tissue from CRC patients using Hi-C, ATAC-seq and RNA-seq technologies. We find that widespread alteration of 3D chromatin structure is accompanied by dysregulation of genes including SPP1 during the tumorigenesis or metastasis of CRC. Remarkably, the hierarchy of topological associating domain (TAD) changes dynamically, which challenges the traditional view that the TAD structure between tumor and normal tissue is conservative. In addition, we define compartment stability score to measure large-scale alteration in metastatic tumors. To integrate multi-omics data and recognize candidate genes driving cancer metastasis, a pipeline is developed based on Hi-C, RNA-seq and ATAC-seq data. And three candidate genes ARL4C, FLNA, and RGCC are validated to be associated with CRC cell migration and invasion using in vitro knockout experiments. Overall, these data resources and results offer new insights into the involvement of 3D genome in cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4405a7f71591c3b2e80c4392e32cd43a5138621" target='_blank'>
              3D genome landscape of primary and metastatic colorectal carcinoma reveals the regulatory mechanism of tumorigenic and metastatic gene expression
              </a>
            </td>
          <td>
            Xiang Xu, Jingbo Gan, Zhaoya Gao, Ruifeng Li, Dandan Huang, Lin Lin, Yawen Luo, Qian Yang, Jingxuan Xu, Yaru Li, Qing Fang, Ting Peng, Yaqi Wang, Zihan Xu, An Huang, Haopeng Hong, F. Lei, Wensheng Huang, Jianjun Leng, Tingting Li, Xiaochen Bo, Hebing Chen, Cheng Li, Jin Gu
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ac5e3595c3552c277cd820f66cb6eb10faa0266" target='_blank'>
              MDA-9/Syntenin as a therapeutic cancer metastasis target: current molecular and preclinical understanding.
              </a>
            </td>
          <td>
            Swadesh K. Das, Paul B Fisher
          </td>
          <td>2025-03-08</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="This study comprehensively investigated keratinocyte subpopulation heterogeneity and developmental trajectories during skin aging using single-cell sequencing, transcriptomics, and facial aging-related genome-wide association studies (GWAS) data. We identified three major subpopulations: basal cells (BCs), spinous cells (SCs), and IFI27+ keratinocytes. Single-cell pseudotime analysis revealed that basal cells can differentiate along two distinct paths: toward spinous differentiation or the inflammatory state. With aging, the proportion of IFI27+ cells significantly increased, displaying more active inflammatory and immunomodulatory signals. Through cell–cell communication analysis, we found that the signaling pathways, including NOTCH, PTPR, and PERIOSTIN, exhibited distinct characteristics along different branches. Integration of the GWAS data revealed significant loci on chromosomes 2, 3, 6, and 9 that were spatially correlated with key biological pathways (including antigen processing, oxidative stress, and apoptosis). These findings reveal the complex cellular and molecular mechanisms underlying skin aging, offering potential targets for novel diagnostic approaches and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7d6ff4c88e6b6ec5e9e3e8325f958c0ceb91730" target='_blank'>
              Inflammatory Transformation of Skin Basal Cells as a Key Driver of Cutaneous Aging
              </a>
            </td>
          <td>
            Shupeng Liu, Sheng Lu, Zhiping Pang, Jiacheng Li, Meijuan Zhou, Zhenhua Ding, Zhijun Feng
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad75c766d4e07168f2e27a4c036f888ffea4b32" target='_blank'>
              Integrating model systems and genomic insights to decipher mechanisms of cancer metastasis.
              </a>
            </td>
          <td>
            Michelle M Leung, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-03-10</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="The mean survival of metastatic lung adenocarcinoma is less than 1 year, highlighting the urgent need to understand the mechanisms underlying its high mortality rate. The role of Extracellular vesicles (EVs) in facilitating the interactions between cancer cells and the metastatic microenvironment has garnered increasing attention. Previous studies on the role of EVs in metastasis have been primarily focused on cancer cell-derived EVs in modulating the functions of stromal cells. However, whether cancer stem cells (CSCs) can alter the metastatic properties of non-CSC cells, and whether EV crosstalk can mediate such interaction, have not been demonstrated prior to this report. In the present study, we integrated multi-omics sequencing and public database analysis with experimental validation to demonstrate, for the first time, the exosomal Mir100hg, derived from CSCs, could enhance the metastatic potential of non-CSCs both in vitro and in vivo. Mechanistically, HNRNPF and HNRNPA2B1 directly binds to Mir100hg, facilitating its trafficking via exosomes to non-CSCs. In non-CSCs, Mir100hg upregulates ALDOA expression, subsequently leading to elevated lactate production. Consequently, the increased lactate levels enhance H3K14 lactylation by 2.5-fold and promote the transcription of 169 metastasis-related genes. This cascade of events ultimately results in enhanced ALDOA-driven glycolysis and histone lactylation-mediated metastatic potential of non-CSC lung cancer cells. We have delineated a complex regulatory network utilized by CSCs to transfer their high metastatic activity to non-CSCs through exosomal Mir100hg, providing new mechanistic insights into the communication between these two heterogeneous tumor cell populations. These mechanistic insights provide novel therapeutic targets for metastatic lung cancer, including HNRNPF/HNRNPA2B1-mediated Mir100hg trafficking and the histone lactylation pathway, advancing our understanding of CSC-mediated metastasis while suggesting promising strategies for clinical intervention. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12951-025-03198-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/982b46ad0f67ebc704b6d142139e5b777f0090b1" target='_blank'>
              Exosomal lncRNA Mir100hg from lung cancer stem cells activates H3K14 lactylation to enhance metastatic activity in non-stem lung cancer cells
              </a>
            </td>
          <td>
            Lei Shi, Bowen Li, Jiyu Tan, Ling Zhu, Sicheng Zhang, Yuhan Zhang, Meng Xiang, Jie Li, Yan Chen, Xue Han, Jia-Cheng Xie, Yao Tang, H. Xing, Jingyu Li, Jianyu Wang, cnJingyu Li
          </td>
          <td>2025-02-28</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Malignant pleural mesothelioma (MPM) is a rare and aggressive malignancy with limited therapeutic options. To improve patients management and treatment, more precise stratification strategies are needed. This study aimed to characterize the phenogenomic landscapes of MPM and to understand their influence on patients clinical outcomes. Methods We conducted a phenogenomic analysis on 22 MPM patients using two high throughput approaches: imaging mass cytometry (IMC) with whole exome sequencing (WES). Resulting profiles were addressed for their clinical relevance to predict patients prognosis. Results IMC revealed a highly heterogeneous tumor microenvironment (TME) with distinct tumor cell subpopulations. Notably, we identified a novel sarcomatoid-like cellular cluster associated with poor prognosis. The TME was also infiltrated with immune cells including macrophages and CD4+ T lymphocytes, that were more abundant in patients with favorable clinical outcomes. WES identified a complex genomic landscape with limited prognostic value for individual genetic alterations. However, tumor mutational burden (TMB) emerged as a potential predictive biomarker, inversely correlating with immune cell infiltration, particularly macrophages and CD4+ T lymphocytes. Conclusions Our findings underscore the intricate interplay between the tumor genome, TME composition, and clinical outcomes in MPM. These data support the potential of integrating genomic and TME profiling to develop more precise patient stratification strategies and potentially optimize therapeutic approaches, including immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06193-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eee1a8b871b1885a1b34200cb093a48900beddf" target='_blank'>
              The phenogenomic landscapes of pleural mesothelioma tumor microenvironment predict clinical outcomes
              </a>
            </td>
          <td>
            Federica Torricelli, Filomena Spada, Cynthia Bishop, K. Todd, Daisuke Nonaka, N. Petrov, Manuela Terranova Barberio, Alan G Ramsay, Richard Ellis, A. Ciarrocchi, B. Apollonio, Andrea Billè
          </td>
          <td>2025-02-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background Profiling circulating cell-free DNA (cfDNA) has become a fundamental practice in cancer medicine, but the effectiveness of cfDNA at elucidating tumor-derived molecular features has not been systematically compared to standard single-lesion tumor biopsies in prospective cohorts of patients. The use of plasma instead of tissue to guide therapy is particularly attractive for patients with small cell lung cancer (SCLC), due to the aggressive clinical course of this cancer, which makes obtaining tumor biopsies exceedingly challenging. Methods In this study, we analyzed a prospective cohort of 49 plasma samples obtained before, during, and after treatment from 20 patients with recurrent SCLC. We conducted cfDNA low-pass whole genome sequencing (0.1X coverage), comparing it with time-point matched tumor characterized using whole-exome (130X) and transcriptome sequencing. Results A direct comparison of cfDNA and tumor biopsy revealed that cfDNA not only mirrors the mutation and copy number landscape of the corresponding tumor but also identifies clinically relevant resistance mechanisms and cancer driver alterations not detected in matched tumor biopsies. Longitudinal cfDNA analysis reliably tracks tumor response, progression, and clonal evolution. Sequencing coverage of plasma DNA fragments around transcription start sites showed distinct treatment-related changes and captured the expression of key transcription factors such as NEUROD1 and REST in the corresponding SCLC tumors. This allowed for the prediction of SCLC neuroendocrine phenotypes and treatment responses. Conclusions cfDNA captures a comprehensive view of tumor heterogeneity and evolution. These findings have significant implications for the non-invasive stratification of SCLC, a disease currently treated as a single entity. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01438-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75bdacf15f78eabb80ad03155a6f7ee66350fd9" target='_blank'>
              Genomic alterations and transcriptional phenotypes in circulating free DNA and matched metastatic tumor
              </a>
            </td>
          <td>
            N. Takahashi, L. Pongor, Shivam P. Agrawal, Mariya Shtumpf, Ankita Gurjar, V. Rajapakse, Ahmad Shafiei, Christopher W. Schultz, Sehyun Kim, Diana Roame, Paula Carter, R. Vilimas, Samantha Nichols, Parth Desai, W. Figg, Mohammad Bagheri, V. B. Teif, Anish Thomas
          </td>
          <td>2025-02-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Gastric cancer (GC) remains a major global health challenge, characterized by a complex tumor microenvironment (TME) that significantly influences disease progression and therapeutic outcomes. This study focuses on TREM2+ lipid-associated macrophages (LAM) and cancer-associated fibroblasts (CAFs) in modulating the GC microenvironment. Utilizing advanced single-cell RNA sequencing and bulk RNA analyses, we elucidated the interactive mechanisms through which CAFs enhance the immunosuppressive capabilities of TREM2+ LAMs via the CXCL12-CXCR4 signaling axis. Our findings reveal that this interaction facilitates tumor proliferation and inhibits apoptotic processes in GC cells. In vitro experiments confirmed the modulation of this pathway significantly affects tumor cell viability and invasiveness, underscoring the critical roles of these cellular interactions in promoting GC progression. These insights present TREM2+ LAMs and CAFs as potential therapeutic targets, offering new avenues for improving outcomes in GC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f055e08f0d4f55971555c1f92f77efccbdd5d60" target='_blank'>
              Defining gastric cancer ecology: the crucial roles of TREM2+ macrophages and fibroblasts in tumor microenvironments
              </a>
            </td>
          <td>
            Qianqian Cao, Dianshui Sun, Can Tu, Jihua Wang, Runjia Fu, Rumei Gong, Yueying Xiao, Qin Liu, Xiaomei Li
          </td>
          <td>2025-03-28</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Brain metastasis represents a significant challenge in oncology, driven by complex molecular and epigenetic mechanisms that distinguish it from primary tumors. While recent research has focused on identifying genomic mutation drivers with potential clinical utility, these strategies have not pinpointed specific genetic mutations responsible for site-specific metastasis to the brain. It is now clear that successful brain colonization by metastatic cancer cells requires intricate interactions with the brain tumor ecosystem and the acquisition of specialized molecular traits that facilitate their adaptation to this highly selective environment. This is best exemplified by widespread transcriptional adaptation during brain metastasis, resulting in aberrant gene programs that promote extravasation, seeding, and colonization of the brain. Increasing evidence suggests that epigenetic mechanisms play a significant role in shaping these pro-brain metastasis traits. This review explores dysregulated chromatin patterns driven by chromatin remodeling, histone modifications, DNA/RNA methylation, and other epigenetic regulators that underpin brain metastatic seeding, initiation, and outgrowth. We provide novel insights into how these epigenetic modifications arise within both the brain metastatic tumor and the surrounding brain metastatic tumor ecosystem. Finally, we discuss how the inherent plasticity and reversibility of the epigenomic landscape in brain metastases may offer new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Although it has been shown that the tumor extracellular matrix (ECM) may sustain the cancer stem cell (CSC) niche, its role in the modulation of CSC properties remains poorly characterized. To elucidate this, paired tumor and adjacent normal mucosa, derived from colon cancer patients’ surgical resections, were decellularized and recellularized with two distinct colon cancer cells, HT-29 or HCT-15. Methods: The matrix impact on cancer stem cell marker expression was evaluated by flow cytometry and qRT-PCR, while transforming growth factor-β (TGF-β) secretion and matrix metalloprotease (MMP) activity were quantified by ELISA and zymography. Results: In contrast to their paired normal counterparts, the tumor decellularized matrices enhanced HT-29 expression of the pluripotency and stemness genes NANOG (p = 0.0117), SOX2 (p = 0.0156), and OCT4 (p = 0.0312) and of the epithelial-to-mesenchymal transition (EMT)-associated transcription factor SNAI1 (p = 0.0156). Notably, no significant differences were found in the expression of SLUG or TGFB on HT-29 or of the six transcripts on HCT-15 cells. HT-29 mRNA alterations were followed by enhanced expression of the stemness-associated receptors cluster of differentiation 44 (CD44), CD133, and CD166 (p = 0.0078), the secretion of TGF-β (p = 0.0286), and MMP-2 (p = 0.0081) and MMP-9 (p = 0.0402) proteolysis. To infer the clinical relevance of these findings, we assessed cohort databases and evidenced that patients expressing higher levels of the four stemness-associated genes (NANOG/SOX2/OCT4/SNAI1) had worse overall survival. This study demonstrates that normal and tumor matrices harbor different stemness potential and suggest patient-derived decellularized matrices as an excellent three-dimensional (3D) model to unveil stemness signatures, appointing candidates for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6135475d9351fff2af8f662d16ac5fff4b3c8c1" target='_blank'>
              Patient-Derived Colorectal Cancer Extracellular Matrices Modulate Cancer Cell Stemness Markers
              </a>
            </td>
          <td>
            Ângela Marques-Magalhães, Sara Monteiro-Ferreira, P. Canão, Elisabete Rios, Ângela M Costa, F. Castro, S. Velho, Joana Paredes, F. Carneiro, Maria José Oliveira, A. P. Cardoso
          </td>
          <td>2025-03-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25aeeccc03e491382af67e5b76f7835dfb0b4a41" target='_blank'>
              508 The Largest Patient-Derived Meningioma Organoid Bank Demonstrates a Robust Model for Transcriptional Analysis of Tumors
              </a>
            </td>
          <td>
            Youssef M. Zohdy, Arman Jahangiri, Fadi Jacob, A. Aksionau, Alejandra Rodas, Zhexing Wen, Gustavo Pradilla, Tomas Garzon-Muvdi
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Although immune checkpoint (IC) inhibition is a major treatment modality in cancer-immunotherapy, multiple cancers show low response. Our in-silico exploration by mining cancer datasets using R2, available clinical trial data, and Kaplan–Meier analysis from GEPIA depicted that unlike low-responder (LR) cancers, high-responder (HR) cancers furnish higher IC expression, that upon lowering may provide better prognosis. Contrastingly, pancreatic adenocarcinoma (PAAD) demonstrated high IC expression but low immunotherapy-response. Infiltration scores from TIMER2.0 revealed higher pro-tumor immune subsets and cancer-associated fibroblasts (CAFs) while depicting lower anti-tumor immune subsets in PAAD as compared to HR lung adenocarcinoma (LUAD). Additionally, bioinformatic tool cBioportal showed lesser tumor mutational burden, mismatch repair deficiency and greater percent of driver mutations in TP53, KRAS and CDKN2A in PAAD, supporting its higher immunotherapy-resistance than LUAD. Our search for the ‘key’ immunotherapy response-deciding factor(s) revealed cancer stem cells (CSCs), the known contributors of therapy-resistance and immuno-evasion, to be positively correlated with above-mentioned driver mutations, pro-tumor immune and CAF subsets; and that PAAD furnished higher expression of CSC genes than LUAD. UMAP/tSNE analyses revealed that high CSC signature is positively correlated with immunotherapy-resistance genes and pro-cancer immune cells, while negatively with cytotoxic-T cells in PAAD. Our in-silico study explains the low immunotherapy-response in high IC-expressing PAAD, wherein CSC plays a pivotal role. Further exploration portrayed correlation of CSCs with immunotherapy-resistance in other LR and HR cancers too, substantiating the need for personalized CSC evaluation and targeting for successful immunotherapy outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93924-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de30629094c5f0558bed2894abe95ee38a9c5b9" target='_blank'>
              In-silico analysis unveiling the role of cancer stem cells in immunotherapy resistance of immune checkpoint-high pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Udit Basak, Sumon Mukherjee, S. Chakraborty, Gaurisankar Sa, S. G. Dastidar, Tanya Das
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Localized prostate cancer (PCa) is highly variable in their response to therapies. Although a fraction of this heterogeneity can be explained by clinical factors or genomic and transcriptomic profiling, the proteomic-based profiling of aggressive PCa remains poorly understood. Here, we profiled the genome, transcriptome, proteome and phosphoproteome of 145 cases of localized PCa in Chinese patients. Proteome-based stratification of localized PCa revealed three subtypes with distinct molecular features: immune subgroup, arachidonic acid metabolic subgroup and sialic acid metabolic subgroup with highest biochemical recurrence (BCR) rates. Further, we nominated NANS protein, a key enzyme in sialic acid synthesis as a potential prognostic biomarker for aggressive PCa and validated in two independent cohorts. Finally, taking advantage of cell-derived orthotopic transplanted mouse models, single-cell RNA sequencing (scRNA-seq) and immunofluorescence analysis, we revealed that targeting NANS can reverse the immunosuppressive microenvironment through restricting the sialoglycan-sialic acid-recognizing immunoglobulin superfamily lectin (Siglec) axis, thereby inhibiting tumor growth of PCa. In sum, we integrate multi-omic data to refine molecular subtyping of localized PCa, and identify NANS as a potential prognostic biomarker and therapeutic option for aggressive PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9febd685653f7715435bba7b29d1b13ce378312" target='_blank'>
              Integrated proteogenomic characterization of localized prostate cancer identifies biological insights and subtype-specific therapeutic strategies
              </a>
            </td>
          <td>
            Wei Ou, Xin-Xin Zhang, Bin Li, Ying Tuo, Ren-Xuan Lin, Pengfei Liu, , Hio-Cheng Un, Minghao Li, , Xiao-Jing Gao, Fu-Fu Zheng, , Ling-Li Long, Zongren Wang
          </td>
          <td>2025-04-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48bced03940233897a4f47121718b37443e74642" target='_blank'>
              Dissection of single-cell landscapes for the development of chimeric antigen receptor T cells in Hodgkin lymphoma.
              </a>
            </td>
          <td>
            A. Gottschlich, R. Grünmeier, Gordon Victor Hoffmann, Sayantan Nandi, V. Kavaka, Philipp Jie Müller, Jakob Jobst, A. Oner, R. Kaiser, J. Gärtig, Ignazio Piseddu, Stephanie Frenz-Wiessner, Savannah D Fairley, Heiko Schulz, V. Igl, Thomas Alexander Janert, Lea di Fina, Maité Mulkers, Moritz Thomas, D. Briukhovetska, D. Simnica, Emanuele Carlini, C. Tsiverioti, Marcel P. Trefny, T. Lorenzini, F. Märkl, Pedro Mesquita, Ruben Brabenec, Thaddäus Strzalkowski, Sophia Stock, Stefanos Michaelides, Johannes Hellmuth, M. Thelen, S. Reinke, W. Klapper, P. Gelebart, Leo Nicolai, Carsten Marr, E. Beltrán, Remco T. A. Megens, Christoph Klein, Fanny Baran-Marszak, A. Rosenwald, Michael von Bergwelt-Baildon, P. Bröckelmann, S. Endres, Sebastian Kobold
          </td>
          <td>2025-04-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65fb7294c5c21b8bae99250ad19dc7b5e7608b5f" target='_blank'>
              Abstract B030: Identifying B7-H3 ligands that mediate tumor-immune cell interactions
              </a>
            </td>
          <td>
            Foad Davoodbeglou, G. Santis, Maddy Parsons
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c475e370c81c7177a65871ada65aabd570e83614" target='_blank'>
              Mesenchymal stem cells and fibroblasts contribute to microvascular proliferation in glioblastoma and are correlated with immunosuppression and poor outcome.
              </a>
            </td>
          <td>
            Candice C Poon, Shelley M Herbrich, Yulong Chen, Anwar Hossain, Gregory N. Fuller, Sonali Jindal, Sreyashi Basu, D. Ledbetter, Marc Macaluso, Lynette M Phillips, J. Gumin, Zhong He, Brittany C Parker Kerrigan, Sanjay K Singh, Pratishtha Singh, M. F. Zaman, D. Ng Tang, Sangeeta Goswami, Frederick F Lang, Padmanee Sharma
          </td>
          <td>2025-03-25</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Genetic variants can affect protein function by driving aberrant subcellular localization. However, comprehensive analysis of how mutations promote tumor progression by influencing nuclear localization is currently lacking. Here, we systematically characterize potential shuttling-attacking mutations (SAMs) across cancers through developing the deep learning model pSAM for the ab initio decoding of the sequence determinants of nucleocytoplasmic shuttling. Leveraging cancer mutations across 11 cancer types, we find that SAMs enrich functional genetic variations and critical genes in cancer. We experimentally validate a dozen SAMs, among which R14M in PTEN, P255L in CHFR, etc. are identified to disrupt the nuclear localization signals through interfering their interactions with importins. Further studies confirm that the nucleocytoplasmic shuttling altered by SAMs in PTEN and CHFR rewire the downstream signaling and eliminate their function of tumor suppression. Thus, this study will help to understand the molecular traits of nucleocytoplasmic shuttling and their dysfunctions mediated by genetic variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47b5ee70b22223e164afab1c522ee489fff66db" target='_blank'>
              Deep learning prioritizes cancer mutations that alter protein nucleocytoplasmic shuttling to drive tumorigenesis
              </a>
            </td>
          <td>
            Yongqiang Zheng, Kai Yu, Jin-Fei Lin, Zhuoran Liang, Qingfeng Zhang, Junteng Li, Qi-Nian Wu, Cai-Yun He, Mei Lin, Qi Zhao, Zhi-Xiang Zuo, Huai-qiang Ju, Rui-Hua Xu, Ze-xian Liu
          </td>
          <td>2025-03-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098b5888d894097b207e005b18110c0c81abf3d8" target='_blank'>
              Modeling cell differentiation in neuroblastoma: insights into development, malignancy, and treatment relapse
              </a>
            </td>
          <td>
            Simon F. Martina-Perez, Luke A. Heirene, Jennifer C. Kasemeier, P. Kulesa, Ruth E. Baker
          </td>
          <td>2025-02-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Background Considerable evidence suggests that tumor initiation, malignancy, metastasis and recurrence occur due to emergence of cancer stem cells (CSCs). Fas binding factor 1 (FBF1) is a multifunctional protein that plays essential roles in the regulation of development and cell fate decisions. However, the function in maintaining stem cell-like properties of breast cancer remains elusive. Methods Tissue microarray was used to evaluate FBF1 expression. Cancer stemness assays were performed in FBF1 silencing and overexpressing cells in vitro and in a xenograft model in vivo. RNA sequencing, immunofluorescence and immunoprecipitation assays were performed to explore the underlying mechanism. Clinical expression and significance of FBF1 and stemness-associated factors were explored by analyzing datasets. Results We report that FBF1 was highly expressed in breast cancer and significantly correlated with clinical progression. Silencing FBF1 in MDA-MB-231 cells restrained CSCs properties, including side population, sphere formation and migration, whereas ectopic FBF1 expression increased the side population proportion, enhanced the sphere formation ability, and promoted the expression of core stemness genes, such as SOX2, OCT4, KLF4 and NANOG, as well as facilitated metastasis of T47D breast cancer cells. Furthermore, mice bearing FBF1-overexpressed T47D xenografts had higher tumorigenic frequency and stronger metastasis potential. In addition, exploration of the underlying mechanism indicated that FBF1 binds PI3K which then activates PI3K-AKT phosphorylation cascades. Then the activated p-AKT interacts with stemness marker SOX2, elevates SOX2 and OCT4 activity, and finally forms PI3K/AKT/SOX2 axis, which mediates stem cell-like identities. Moreover, PI3K inhibitors abolished FBF1-mediated signaling pathway and diminished breast cancer stemness in vitro and in vivo. In 24 human breast cancer samples, we found a good positive correlation between the expression of FBF1 and p-AKT, as well as between FBF1 and SOX2 as determined by IHC. Clinical data showed that FBF1 expression was positively correlated with the expression of POU5F1 (OCT4), AKT1 and was negatively correlated with PTEN, which is a negative regulator of PI3K/AKT signaling. Conclusion Collectively, we identified a potential CSCs regulator and suggested a novel mechanism by which FBF1 governs cancer cell stemness. This study thus introduces an effective target for the diagnosis and treatment of breast cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13287-025-04194-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7906de3714698fbfedd73335fb57fafadf22aa8" target='_blank'>
              FBF1 maintains stem cell-like properties in breast cancer via PI3K/AKT/SOX2 axis
              </a>
            </td>
          <td>
            Chunlei Guo, Shuang Li, Jiaqing Liu, Yuqiu Ma, Ang Liang, Yunwei Lou, Hui Liu, Hui Wang
          </td>
          <td>2025-02-23</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Various challenges, including tumor heterogeneity and inadequate T cell infiltration, impede the progress of chimeric antigen receptor T cell (CAR-T) therapy for glioblastoma (GBM). To address these obstacles, a multiple step strategy is designed. Initially, literature review and bioinformatics analysis to screen a set of antigens that are heterogeneously expressed in GBM, which are designated as the target-bank, are leveraged. Then, according to the multiplex immunohistochemistry results of each patient's tumor sample, a personalized panel of antigens based on the principle that most cancer cells in tumor tissues can be covered from the target-bank is selected. To target these antigens, Vδ1 T cells are chosen as CAR vehicles because of its high tissue infiltration and off-the-shelf properties, and an optimized protocol for engineering CAR-Vδ1 T cells with high purity and cytotoxicity, low exhaustion, and cytokine release is developed. Next, the specific panel of cocktail CAR-Vδ1 T cells in the GBM organoids that are directly derived from the same patient's tumor is tested. The term "prof" cocktail therapy is coined to describe the approach using precise and rational combination of tumor antigens, organoid-based evaluation, and fitness of Vδ1 T cells. It may accelerate development of effective CAR-T drugs for heterogeneous solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d528650de6d54c949045b7ae03b97d20044ba8e" target='_blank'>
              Rational Design and Organoid-Based Evaluation of a Cocktail CAR-γδ T Cell Therapy for Heterogeneous Glioblastoma.
              </a>
            </td>
          <td>
            Guidong Zhu, Zhongzheng Sun, Yingchao Liu, Jiang Liu, Linpei Guo, Guojing Pei, Ying Jiang, Baowang Miao, Zhen Li, Ping Zhang, Dongqi Tang, Wen Zhang, Chengwei Wang
          </td>
          <td>2025-03-20</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Neuroendocrine prostate cancer (NEPC) is a highly aggressive malignancy with few effective treatment options. The identification of reliable biomarkers for NEPC is essential for early detection and intervention. Methods We combined single-cell and bulk transcriptome analysis to identify novel markers of NEPC. InferCNV to assess copy number variations and leveraging consensus non-negative matrix factorization (cNMF) to characterize transcriptional programs. Pseudotime analysis was used to decipher prostate cancer (PCa) progression differentiation trajectory. BayesPrism integrates single-cell results and TCGA-PRAD sequencing information to further study prognostic features. Immunohistochemistry (IHC) was performed to validate the elevated expression of ASCL1 and WDFY4 in NEPC. Results We identified five distinct expression programs of PCa malignant epithelial cells, where Module 3 presented NEPC expression patterns, with activation of DNA replication and cell cycle pathways and classical NEPC marker expression. Patients with high Module 3 proportion correlated to poor clinical outcomes, advanced Gleason scores, and higher T stages. Pseudotime analysis highlighted key trajectory-dependent genes involved in the transition to NEPC, where expression of ASCL1 and WDFY4 elevated with progressing to NEPC cell fate, which were further confirmed by IHC analysis, indicating that WDFY4 and ASCL1 might be novel potential markers for distinguishing NEPC. Conclusions Combined single-cell and bulk analysis, we highlight the cellular heterogeneity and transcriptional programs, validated novel biomarkers of NEPC. Providing a foundation for early prediction of NEPC and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f366e5928979effa39eaa62b09ed73f4c0113d47" target='_blank'>
              New insights into markers for distinguishing neuroendocrine prostate cancer: evidence from single-cell analysis
              </a>
            </td>
          <td>
            Hailang Luo, Boyang Li, Meng Zhang, Hongqun Wang, Zongyao Hao, Qintao Ge, Chaozhao Liang
          </td>
          <td>2025-03-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Recent oncological research has intensely focused on the tumor immune microenvironment (TME), particularly the functions of CD4 + T lymphocytes. CD4+ T lymphocytes have been implicated in antigen presentation, cytokine release, and cytotoxicity, suggesting their contribution to the dynamics of the TME. Furthermore, the application of single-cell sequencing has yielded profound insights into the phenotypic diversity and functional specificity of CD4+ T cells in the TME. In this review, we discuss the current findings from single-cell analyses, emphasizing the heterogeneity of CD4+ T cell subsets and their implications in tumor immunology. In addition, we review the critical signaling pathways and molecular networks underpinning CD4+ T cell activities, thereby offering novel perspectives on therapeutic targets and strategies for cancer treatment and prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06167-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8641c47a99cb4e08fcabebc4fec5794fc9e23e2f" target='_blank'>
              Role of CD4+ T cells in cancer immunity: a single-cell sequencing exploration of tumor microenvironment
              </a>
            </td>
          <td>
            Qi An, Li Duan, Yuanyuan Wang, Fuxin Wang, Xiangyu Liu, Chao Liu, Qinyong Hu
          </td>
          <td>2025-02-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Glioblastoma multiforme (GBM) is an aggressive brain tumor with chemoresistant, immunosuppressive, and invasive properties. Despite standard therapies, including surgery, radiotherapy, and temozolomide (TMZ) chemotherapy, tumors inevitably recur in the peritumoral region. Targeting GBM-mediated immunosuppressive and invasive properties is a promising strategy to improve clinical outcomes. Methods We utilized clinical and genomic data from the Taiwan GBM cohort and The Cancer Genome Atlas (TCGA) to analyze RNA sequencing data from patient tumor samples, determining the association of interleukin-19 (Il-19) expression with survival and immunosuppressive activity. Gene set enrichment analysis (GSEA) was performed to assess the relationship between the enrichment levels of immune subsets and Il-19 expression level, and Ingenuity Pathway Analysis (IPA) was used to predict immune responses. Cytokine array and single-cell RNA sequencing were used to examine the effects of IL-19 blockade on tumor immune microenvironment, including tumor-infiltrating leukocyte profiles, differentiation and immunosuppressive genes expression in tumor associated macrophages (TAM). CRISPR Il-19−/− cell lines and Il-19−/− mice were used to examine the role of IL-19 in tumor invasion and M2-like macrophage-mediated immunosuppression. Additionally, we developed novel cholesterol-polyethylene glycol-superparamagnetic iron oxide-IL-19 antibody nanoparticles (CHOL-PEG-SPIO-IL-19), characterized them using dynamic light scattering and transmission electron microscopy, Fourier-Transform Infrared spectroscopy, prussian blue assay, and conducted in vivo magnetic resonance imaging (MRI) in a human glioblastoma stem cell-derived GBM animal model. Result Genomic screening and IPA analysis identified IL-19 as a predicted immunosuppressive cytokine in the peritumoral region, associated with poor survival in patients with GBM. Blocking IL-19 significantly inhibited tumor progression of both TMZ-sensitive (TMZ-S) and TMZ-resistant (TMZ-R) GBM-bearing mice, and modulated the immune response within the GBM microenvironment. Single-cell transcriptome analysis reveal that IL-19 antibody treatment led to a marked increase in dendritic cells and monocyte/macrophage subsets associated with interferon-gamma signaling pathways. IL-19 blockade promoted T cell activation and reprogrammed tumor-associated macrophages toward weakened pro-tumoral phenotypes with reduced Arginase 1 expression. Il19−/− M2-like bone marrow-derived macrophages with lower Arginase 1 level lost their ability to suppress CD8 T cell activation. These findings indicated that IL-19 suppression limits TAM-mediated immune suppression. Molecular studies revealed that IL-19 promotes TMZ-resistant GBM cell migration and invasion through a novel IL-19/WISP1 signaling pathway. For clinical translation, we developed a novel CHOL-PEG-SPIO-IL-19 nanoparticles to target IL-19 expression in glioblastoma tissue. MRI imaging demonstrated enhanced targeting efficiency in brain tumors, with in vivo studies showing prominent hypointense areas in T2*-weighted MRI scans of tumor-bearing mice injected with CHOL-PEG-SPIO-IL-19, highlighting nanoparticle presence in IL-19-expressing regions. Prussian blue staining further confirmed the localization of these nanoparticles in tumor tissues, verifying their potential as a diagnostic tool for detecting IL-19 expression in glioblastoma. This system offers a theranostic approach, integrating diagnostic imaging and targeted therapy for IL-19-expressing GBM. Conclusion IL-19 is a promising theranostic target for reversing immunosuppression and restricting the invasive activity of chemoresistant GBM cells. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12929-025-01126-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251b166370922cbd1fd5ab0e04f517f6d1cb1e5e" target='_blank'>
              IL-19 as a promising theranostic target to reprogram the glioblastoma immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Gilbert Aaron Lee, Justin Bo-Kai Hsu, Yu-Wei Chang, Li-Chun Hsieh, Yi-Tien Li, Ying-Chieh Wu, Cheng-Ying Chu, Yung-Hsiao Chiang, Wan-Yuo Guo, Chih-Chun Wu, Liang-Wei Chen, Hung-Wen Kao, Wan-Li Lin, Li‑Wen Tseng, Ting-Wei Weng, Duen-Pang Kuo, Sho-Jen Cheng, Yung-Chieh Chen, Shiu-Wen Huang, Hsing-Jien Kung, Cheng-Yu Chen
          </td>
          <td>2025-03-08</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87c4098a87feebf01a217fad3de111af3e86862" target='_blank'>
              Abstract IA002: Functional omics for novel therapeutic combinations in myeloid malignancies
              </a>
            </td>
          <td>
            J. Tyner
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9558da7bc078097ca4cbccb7eec125cbb68c3bc9" target='_blank'>
              BISON: Bi-clustering of spatial omics data with feature selection
              </a>
            </td>
          <td>
            Bencong Zhu, Alberto Cassese, Marina Vannucci, Michele Guindani, Qiwei Li
          </td>
          <td>2025-02-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0786e178852bb9b6b2a0945adb3bddac988e519" target='_blank'>
              Abstract A030: A deeper understanding of tumor microenvironment modulation induced by IL13Ra2 CAR T cells in a patient with grade 3 ependymoma
              </a>
            </td>
          <td>
            Maryam Aftabizadeh, B. Aguilar, M. D’Apuzzo, B. Badie, Christine E. Brown
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21d853448d817f079816c9b14dd1957f3e34109e" target='_blank'>
              The Pre-metastatic Niche: How Cancer Stem Cell-Derived Exosomal MicroRNA Fit into the Puzzle.
              </a>
            </td>
          <td>
            Behjat Kheiri Yeghaneh Azar, F. Vakhshiteh
          </td>
          <td>2025-03-17</td>
          <td>Stem cell reviews and reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fa697cc0365f103131eebd290f857c508fd5863" target='_blank'>
              Tumor Cell Plasticity and Stromal Microenvironment Distinguish Papillary and Follicular Growth Patterns in a Mouse Model of BRAFV600E-induced Thyroid Cancer.
              </a>
            </td>
          <td>
            E. Schoultz, C. Moccia, Shawn Liang, Ellen Johansson, Mikael Nilsson
          </td>
          <td>2025-02-17</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by cancer cells surrounded by affluent stromal components, which may underlie their limited response to various therapeutic interventions, including immunotherapy. Inflammatory cancer-associated fibroblasts (iCAFs), a crucial subset of CAFs within the PDAC microenvironment, play a pivotal role in shaping an immunosuppressive microenvironment. In this study, single-cell RNA sequencing analysis is performed to screen for cancer cells-secreted proteins associated with iCAF induction, and PPY (pancreatic polypeptide) is validated as a potent inducer. Unlike previously reported iCAF inducers, PPY is a gastrointestinal hormone predominantly expressed in the pancreas, suggesting that targeting it may have minimal systemic effects. Multiplex immunohistochemistry (mIHC) on human PDAC tissue microarrays, orthotopic allograft mouse models, and co-culture experiments are utilized to validate the crucial role of PPY in iCAF induction. Mechanistic studies integrating mRNA sequencing, immunoprecipitation-mass spectrometry, and molecular docking reveal that PPY induces iCAFs by activating the non-canonical NF-κB pathway through EGFR. Importantly, targeting PPY enhanced the efficacy of anti-PD-1 immunotherapy in KPC (KrasLSL-G12D/+; Trp53LSL-R172H/+; Pdx1-Cre) mice, as evidenced by reduced tumor burden on PET-CT imaging and improved survival. This research is expected to provide a novel strategy for improving immunotherapy in PDAC by targeting a key inducer of iCAFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a439b3469f5bbba60fcf26000b43c5e1b5ab3a0d" target='_blank'>
              PPY-Induced iCAFs Cultivate an Immunosuppressive Microenvironment in Pancreatic Cancer.
              </a>
            </td>
          <td>
            Mengdie Cao, Wang Peng, Bin Cheng, Ronghua Wang, Wei Chen, Luyao Liu, Hai Huang, Shiru Chen, Haochen Cui, Jingwen Liang, Qiaodan Zhou, S. Xiong, Shuya Bai, Luoxia Liu, Yuchong Zhao
          </td>
          <td>2025-03-31</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance.
 Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features.
 Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations.
 Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF -mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Aggressive variant and androgen receptor (AR)-independent castration resistant prostate cancers (CRPC) represent the most significant diagnostic and therapeutic challenges in prostate cancer. This study examined a case of simultaneous progression of both adenocarcinoma and squamous tumors from the same common origin. Using whole-genome and transcriptome sequencing from 17 samples collected over >6 years, we established the clonal relationship of all samples, defined shared complex structural variants, and demonstrated both divergent and convergent evolution at AR. Squamous CRPC-associated circulating tumor DNA was identified at clinical progression prior to biopsy detection of any squamous differentiation. Dynamic changes in the detection rate of histology-specific clones in circulation reflected histology-specific sensitivity to treatment. This dataset serves as an illustration of non-neuroendocrine transdifferentiation and highlights the importance of serial sampling at progression in CRPC for the detection of emergent non-adenocarcinoma histologies with implications for the treatment of lineage plasticity and transdifferentiation in metastatic CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65b072299240230cb903f6cbde3bd730a8642b8" target='_blank'>
              A complex phylogeny of lineage plasticity in metastatic castration resistant prostate cancer
              </a>
            </td>
          <td>
            J. Nauseef, Timothy R Chu, William Hooper, Alicia Alonso, Ali Oku, Heather Geiger, Zoe R. Goldstein, Minita J. Shah, M. Sigouros, J. Manohar, Zoe Steinsnyder, Lara Winterkorn, Brian D. Robinson, A. Sboner, H. Beltran, O. Elemento, Iman Hajirasouliha, Marcin Imielinski, D. Nanus, Scott Tagawa, N. Robine, Juan Miguel Mosquera
          </td>
          <td>2025-03-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059cc807139dc64898d45632991b55c44cdfb1d6" target='_blank'>
              Abstract A108: Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients
              </a>
            </td>
          <td>
            Muyi Yang, Sólrún Kolbeinsdóttir, Katarina Hammarlund, V. Grozman, Martin Enge, Suzanne Egyhazi-Brage, H. Helgadottir
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Ductal carcinoma in situ (DCIS) is a risk factor for subsequent invasive breast cancer. To identify events in DCIS that lead to invasive cancer, we performed single-cell RNA-sequencing (scRNA-seq) on DCIS lesions and matched normal breast tissue. Inferred copy number variation (CNV) was used to identify neoplastic epithelial cells from clinical specimens, which contained a mixture of DCIS and normal ducts. Phylogenetic analysis demonstrated intratumoral clonal heterogeneity that was associated with significant gene expression differences. Classification of epithelial cells into mammary cell states revealed that subclones contained a mixture of cell states, suggesting an ongoing pattern of differentiation after neoplastic transformation. Cell state proportions were significantly different based on estrogen receptor (ER) expression with ER-negative DCIS more closely resembling the distribution in the normal breast, particularly with respect to cells with basal characteristics. Specific alterations in cell state proportions were associated with progression to invasive cancer in a cohort of DCIS with longitudinal outcome. Ongoing transcription of key basement membrane (BM) genes occurred in specific subsets of epithelial cell states, including basal/myoepithelial, which are diminished in DCIS. In the transition to invasive breast cancer, the BM protein laminin, but not COL4, was altered in DCIS adjacent to invasion. Loss of COL4, but not laminin, in an in vitro DCIS model led to an invasive phenotype. These findings suggest that the process of invasion is a loss-of-function event due to an imbalance in critical cell populations essential for BM integrity rather than a gain of an invasive phenotype by neoplastic cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5661c366f2eb52b8794cdbe190cb026b878cad" target='_blank'>
              Single Cell Expression Analysis of Ductal Carcinoma in Situ Identifies Complex Genotypic-Phenotypic Relationships Altering Epithelial Composition.
              </a>
            </td>
          <td>
            Xiaodi Qin, Siri H Strand, Marissa R Lee, Aashrith Saraswathibhatla, David G P van IJzendoorn, ChunFang Zhu, S. Vennam, S. Varma, Allison Hall, Rachel E Factor, Lorraine M. King, Lunden Simpson, Xiaoke Luo, G. Colditz, Shu Jiang, Ovijit Chaudhuri, E. S. Hwang, Jeffrey R. Marks, K. Owzar, Robert B. West
          </td>
          <td>2025-03-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f8d25128e68026c13f6a41f5701ec7d66c8485" target='_blank'>
              Cancer Cell and Cancer-Associated Fibroblast Communication-Mediated Molecular Subtypes Portray Non-Inflamed Tumor Microenvironment and Guide the Precision Treatment of Bladder Cancer.
              </a>
            </td>
          <td>
            Shenglin Gao, Chuan Liu, Lixin Mao, Yin Chen, Xiaokai Shi, C. Yue, Shouchun Li, X. Qin
          </td>
          <td>2025-02-17</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f7b4d0623804b6e4ad33245c8e8741d967977b" target='_blank'>
              Relationship of spatial transcriptomic analyses with distance-associated transcriptomic spectrum and morphology-specific signatures of localized prostate cancer.
              </a>
            </td>
          <td>
            Brendan Raizenne, Hanbing Song, Nancy Y. Greenland, Maggie Tsui, Jared Khan, Taiqi Li, Chih-Hao Chang, M. Cooperberg, Franklin W. Huang
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Background & Aims Patients with colorectal cancer have heterogeneous clinical responses to chemotherapy, although clinical guidelines advise little variability in treatment selection based on molecular tumor features. Precision oncology research typically utilizes patient-derived tumor organoids (PDTO) to predict clinical outcomes, but such efforts are often not directed towards identification of molecular factors underlying differential responses to therapy. Methods Bulk RNA-sequencing was performed on treatment-naive PDTOs, and gene expression data was combined to drug sensitivity data to identify transcriptomic features associated with low in vitro sensitivity to chemotherapy. Whole-exome sequencing was performed on primary tumors to infer the somatic mutations of PDTOs and used to identify somatic mutations associated with differential in vitro drug responses. Publicly available gene expression and drug sensitivity data sets were used to validate the results. RNA interference was used for functional validation. Results PDTOs with low chemosensitivity had high JAK-STAT pathway activity resulting from high expression of interferon-stimulated genes. Evidence from single-cell RNA-sequencing confirmed chemotherapy-induced expression of interferon-stimulated genes in epithelial cells of cancers with partial response. EPSTI1 knockdown decreased cancer cell viability and sensitized cells to chemotherapy. Conclusions Sustained interferon signaling in epithelial cancer cells contributes to incomplete pathologic response in colorectal cancer. The findings highlight the potential of JAK-STAT inhibition or TRAIL pathway activation to enhance chemotherapy efficacy. Future studies investigating pharmacologic modulation of these pathways in preclinical CRC models are needed to determine their viability as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ab87f4ae646ab5e983158fb38a2489ac5894d6" target='_blank'>
              Sustained epithelial interferon signaling modulates incomplete pathologic response in colorectal cancer
              </a>
            </td>
          <td>
            E. Erkan, Emmi Hämäläinen, Julia Kolikova, Kornelia Kuc, K. Ojala, Meiju Kukkonen, Ismaïl Hermelo, S. Koskensalo, T. Tarvainen, Alli Leppä, I. Keränen, Carola Haapamäki, E. Karjalainen, Monica Carpelan-Holmström, L. Renkonen-Sinisalo, Laura E Koskenvuo, P. Puolakkainen, J. Mecklin, Elina Virolainen, M. Nykter, L. Aaltonen, Anna H. Lepistö, Ari Ristimäki, Toni T Seppälä
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2e9255d35debfd9a15445b40a466e6a5d29ad6" target='_blank'>
              Chromosomal Instability and Clonal Heterogeneity in Breast Cancer: From Mechanisms to Clinical Applications
              </a>
            </td>
          <td>
            María Paula Meléndez-Flórez, O. Ortega-Recalde, N. Rangel, Milena Rondón-Lagos
          </td>
          <td>2025-04-04</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Somatic mutations that increase hematopoietic stem cell (HSC) fitness drive their expansion in clonal hematopoiesis (CH) and predispose to blood cancers. Although CH frequently occurs with aging, it rarely progresses to overt malignancy. Population variation in the growth rate and potential of mutant clones suggests the presence of genetic factors protecting against CH, but these remain largely undefined. Here, we identify a non-coding regulatory variant, rs17834140-T, that significantly protects against CH and myeloid malignancies by downregulating HSC-selective expression and function of the RNA-binding protein MSI2. By modeling variant effects and mapping MSI2 binding targets, we uncover an RNA network that maintains human HSCs and influences CH risk. Importantly, rs17834140-T is associated with slower CH expansion rates in humans, and stem cell MSI2 levels modify ASXL1-mutant HSC clonal dominance in experimental models. These findings leverage natural resilience to highlight a key role for post-transcriptional regulation in human HSCs, and offer genetic evidence supporting inhibition of MSI2 or its downstream targets as rational strategies for blood cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231338d317a31d1b141aef89e6f810d3087be9ba" target='_blank'>
              Inherited resilience to clonal hematopoiesis by modifying stem cell RNA regulation
              </a>
            </td>
          <td>
            Gaurav Agarwal, Mateusz Antoszewski, Xueqin Xie, Y. Pershad, U. Arora, Chi-Lam Poon, Peng Lyu, Andrew J. Lee, Chun-Jie Guo, Tianyi Ye, Laila Barakat Norford, Anna-Lena Neehus, Lucrezia della Volpe, Lara Wahlster, Diyanath Ranasinghe, Tzu-Chieh Ho, T. Barlowe, Arthur Chow, A. Schurer, James Taggart, Benjamin H. Durham, Omar Abdel-Wahab, Kathy L. McGraw, James M Allan, Ruslan Soldatov, Alexander G. Bick, M. Kharas, V. G. Sankaran
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is known for its high heterogeneity, with liver metastases significantly impairing survival outcomes. Understanding the tumor microenvironment (TME) and genomic alterations in metastatic sites is crucial for developing personalized therapies that overcome drug resistance and improve prognosis. Methods We profiled 54 CRC liver metastases, comparing them with 198 other metastatic lesions and normal liver tissues. By analyzing immune cell infiltration, stromal interactions, and key genomic alterations, we constructed an 11-gene prognostic model to predict survival and immunotherapy outcomes. Results CRC liver metastases with high-risk profiles demonstrated enriched follicular helper T cells, activated dendritic cells, and M2 macrophages in the TME. Frequent mutations in APC, TP53, KRAS, and PIK3CA were identified, alongside altered EGFR signaling. The 11-gene model effectively stratified patients by prognosis and predicted immunotherapy responses, emphasizing the therapeutic potential of targeting resistance mechanisms. Conclusions This study reveals how genomic and TME-driven factors contribute to drug resistance in CRC liver metastases. Integrating these insights with clinical data could advance precision therapies, addressing the evolving challenge of tumor drug resistance in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ceeafcf1da623087d79a8767b5543bf2a7043f8" target='_blank'>
              Genomic and micro-environmental insights into drug resistance in colorectal cancer liver metastases
              </a>
            </td>
          <td>
            Junjie Kuang, Jun Li, Siwei Zhou, Yi Li, Jinbo Lin, Wei-zeng Huang, Xia Yuan
          </td>
          <td>2025-02-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fffa6682c8cf8e3d73d30e552a59d8713e1c83f9" target='_blank'>
              Abstract A073: Digital assessment of tertiary lymphoid structures in cancer using whole slide scans
              </a>
            </td>
          <td>
            S. M. Park, Luis Munoz-Erazo, Shelly Lin, J. C. Chun-Jen, Lisa Y.Y. Zhou, J. Rhodes, Taesung Jeon, Sonya Fenton, John L McCall, Roslyn A. Kemp, P. R. Dunbar
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ea7bcdd3ec0f8b344e09113c3c14bed0419bdf" target='_blank'>
              Construction of a Single-cell Atlas of Thyroid Cancer.
              </a>
            </td>
          <td>
            Kaiyu Song, Yaqi Wang, Yuantao Wang, Jiahui Liu, Wenjie Yao, Yongli Chu, Yun Qu, Xicheng Song, Jin Zhou
          </td>
          <td>2025-03-10</td>
          <td>Endocrine, metabolic & immune disorders drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6248a04912b285f929efafcc71ddd28019bc2f15" target='_blank'>
              Clonal hematopoiesis is clonally unrelated to multiple myeloma and is associated with specific microenvironmental changes.
              </a>
            </td>
          <td>
            M. Lionetti, Margherita Scopetti, A. Matera, Akihiro Maeda, Alessio Marella, Francesca Lazzaroni, Giancarlo Castellano, S. Fabris, Stefania Pioggia, Silvia Lonati, Alfredo Marchetti, Alessandra Cattaneo, Marta Tornese, A. Neri, Claudia Leoni, L. Pettine, Valentina Traini, Ilaria Silvestris, M. Barbieri, G. Fabbiano, D. Ronchetti, E. Taiana, C. De Magistris, M. D. Da Via', Francesco Passamonti, N. Bolli
          </td>
          <td>2025-03-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29965c5d14b51c6573686cfc61d8b621a40178b6" target='_blank'>
              Abstract IA019: Addressing genomic heterogeneity in glioblastoma personalized vaccine design
              </a>
            </td>
          <td>
            Elaine R Mardis, E. Garfinkle, T. Johanns, Katherine E Miller, Gavin P Dunn
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1719d073dfd28cb966f7f3079174856e098d4f7f" target='_blank'>
              Spatial transcriptomics of progression gene signature and tumor microenvironment leading to progression in mycosis fungoides.
              </a>
            </td>
          <td>
            M. Choi, Gyeonghoon Kim, Hwa Jeong Shin, C. Won, Sung Eun Chang, M. Lee, W. Lee
          </td>
          <td>2025-03-31</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Retrospective data support overall survival (OS) advantage to high clonal tumor mutation burden (cTMB), high clonal neoantigen load (cNEO) and low intratumor heterogeneity (ITH) in cancer patients who receive immunotherapy. In order to explore this relationship prospectively with Vigil, a triple function targeted immunotherapy involving ovarian cancer patients in long term follow up of the Phase 2b VITAL trial, we developed an exome sequencing procedure and associated bioinformatics pipeline to determine clonal signal patterns. DNA libraries containing exome sequences tagged with unique molecular identifiers (UMI) were prepared from paired samples and sequenced on Illumina sequencers to high coverage depths of ~ 930X (tumor) and ~ 130X (normal). Raw sequence reads were processed into optimized binary alignment map (BAM) files, using the UMI information. The BAM files were inputted into modules for calling MHC-I alleles, annotating single nucleotide variants (SNVs) and small insertions/deletions (InDels), and for determination of allelic copy number. The outputs were used to predict the sequence of peptide neoantigens and to perform clonality analysis in order to assign each SNV and InDel in a patient tumor sample to a primary clone or subclone. The Clonal Neoantigen pipeline was further assessed using whole exome Illumina sequencing data from three previously published studies. Evaluation of the pipeline using synthetic sequencing data from a sub-clonal deconvolution tool benchmarking study, showed positive predictive value (PPV) and positive percent agreement (PPA) of > 97.5% and > 96.5%, respectively, for SNV and InDel detection with minimum requirements for variant density and allele fraction. Haplotype calls from the Clonal Neoantigen pipeline MHC-I/ MHC-II typing module matched a published benchmark for 91.5% of the calls in a sample of 99 patients. Analysis of exome sequencing data from 14 patients with advanced melanoma revealed a strong correlation between cTMB values determined by the Clonal Neoantigen pipeline as compared to those calculated from the published data (R2 = 0.99). Following validation, the wet lab process and Clonal Neoantigen pipeline was applied to a set of matched normal, tumor, and Vigil product samples from 9 (n = 27 samples) ovarian cancer subjects entered into the VITAL (CL-PTL-119) trial. Results demonstrated marked correlation (R2 = 0.98) of cTMB between tumor used to construct Vigil and Vigil product. Correlation between tumor and Vigil for the cNEO and ITH metrics, showed R2 values of 0.95 and 0.87, respectively. The consistency of the Clonal Neoantigen pipeline results with previously published data as well as the agreement between results for tumor and Vigil for the entire system provide a strong basis of support for utilization of this pipeline for prospective determination of cTMB, cNEO, and ITH values in clinical tumor tissue in order to explore possible correlative relationships with clinical response parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16718d2277420c09078544ae1939c2c13bd37fd2" target='_blank'>
              Exome sequencing shows same pattern of clonal tumor mutational burden, intratumor heterogenicity and clonal neoantigen between autologous tumor and Vigil product
              </a>
            </td>
          <td>
            David Willoughby, E. Bognar, L. Stanbery, Casey Nagel, G. Wallraven, Aman Pruthi, Nicholas Bild, Ericca Stamper, Donald Rao, Adam Walter, John Nemunaitis
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>18</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>