<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-21 09:28:08 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Lung squamous cell carcinoma (LUSC) is basal-like subtype of lung cancer with limited treatment options. While prior studies have identified tumor-propagating cell states in squamous tumors, the broader landscape of intra-tumoral heterogeneity within LUSC remains poorly understood. Here, we employ Sox2-driven mouse models, organoid cultures, and single-cell transcriptomic analyses to uncover previously unrecognized levels of cell fate diversity within LUSC. Specifically, we identify a KRT13+ hillock-like population of slower-dividing tumor cells characterized by immunomodulatory gene expression signatures. The tumor hillock-like state is conserved across multiple animal models and is present in the majority of human LUSCs as well as head and neck and esophageal squamous tumors. Our findings shed light on the cellular origins of lung hillock-like states: normal club cells give rise to tumors with luminal hillock-like populations, while basal-like tumor-propagating cells transition into basal hillock-like states, resembling homeostatic cellular responses to lung injury. Mechanistically, we identify KLF4 as a key transcriptional regulator of the hillock-like state, both necessary and sufficient to induce KRT13 expression. Together, these results provide new molecular insights into cell fate plasticity that underlies intra-tumoral heterogeneity in LUSC, offering potential avenues for new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec0519121b0ddc09d45dd5bfdeca6cc730f82cfc" target='_blank'>
              KLF4 promotes a KRT13+ hillock-like state in squamous lung cancer
              </a>
            </td>
          <td>
            Luke T. Izzo, Tony Reyes, Srijan Meesala, A. Ireland, Steven Yang, Hari Shankar Sunil, Xiao Chun Cheng, Nomi Tserentsoodol, Sarah B. Hawgood, Edward F. Patz, Benjamin L. Witt, Darren R Tyson, Kathryn A. O’Donnell, T. Oliver
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Spatial proteomics enable detailed analysis of tissue at single cell resolution. However, creating reliable segmentation masks and assigning accurate cell phenotypes to discrete cellular phenotypes can be challenging. We introduce IMmuneCite, a computational framework for comprehensive image pre-processing and single-cell dataset creation, focused on defining complex immune landscapes when using spatial proteomics platforms. We demonstrate that IMmuneCite facilitates the identification of 32 discrete immune cell phenotypes using data from human liver samples while substantially reducing nonbiological cell clusters arising from co-localization of markers for different cell lineages. We established its versatility and ability to accommodate any antibody panel and different species by applying IMmuneCite to data from murine liver tissue. This approach enabled deep characterization of different functional states in each immune compartment, uncovering key features of the immune microenvironment in clinical liver transplantation and murine hepatocellular carcinoma. In conclusion, we demonstrated that IMmuneCite is a user-friendly, integrated computational platform that facilitates investigation of the immune microenvironment across species, while ensuring the creation of an immune focused, spatially resolved single-cell proteomic dataset to provide high fidelity, biologically relevant analyses. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93060-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8086a8ef4d962d4564e2b7d111a96f7606082d" target='_blank'>
              Integrated workflow for analysis of immune enriched spatial proteomic data with IMmuneCite
              </a>
            </td>
          <td>
            Arianna Barbetta, Sarah Bangerth, Jason T C Lee, B. Rocque, E. R. Roussos Torres, Rohit Kohli, Omid Akbari, J. Emamaullee
          </td>
          <td>2025-03-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="We conducted a spatial analysis of stage III colorectal adenocarcinomas using Hyperion Imaging Mass Cytometry, examining 52 tumors to assess the tumor microenvironment at the single-cell level. This approach identified 10 distinct cell phenotypes in the tumor microenvironment, including stromal and immune cells, with a subset showing a proliferative phenotype. By focusing on spatial neighborhood interactions and tissue niches, particularly regions with tumor-infiltrating lymphocytes, we investigated how cellular organization relates to clinicopathological and molecular features such as microsatellite instability (MSI) and recurrence. We determined that microsatellite stable (MSS) colorectal cancers had an increased risk of recurrence if they had the following features: 1) a low level of stromal tumor-infiltrating lymphocytes, and 2) low interactions between CD4 + T cells and stromal cells. Our results point to the utility of spatial single-cell interaction analysis in defining novel features of the tumor immune microenvironments and providing useful clinical cell-related spatial biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a63f7d3dd84bf14bb59b2621b8ec0ed0c881cf8" target='_blank'>
              The single-cell spatial landscape of stage III colorectal cancers
              </a>
            </td>
          <td>
            Andrew Su, Hojoon Lee, Minh Tran, Richard C Dela Cruz, A. Sathe, Xiangqi Bai, Ignacio A Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, D. Haslem, David Jones, Lincoln Nadauld, Quan H Nguyen, , Terence Rhodes
          </td>
          <td>2025-04-07</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Spatial cellular context is crucial in shaping intratumor heterogeneity. However, understanding how each tumor establishes its unique spatial landscape and what factors drive the landscape for tumor fitness remains significantly challenging. Here, we analyzed over 2 million cells from 50 tumor biospecimens using spatial single-cell imaging and single-cell RNA sequencing. We developed a deep learning-based strategy to spatially map tumor cell states and the architecture surrounding them, which we referred to as Spatial Dynamics Network (SDN). We found that different tumor cell states may be organized into distinct clusters, or ‘villages’, each supported by unique SDNs. Notably, tumor cell villages exhibited village-specific molecular co-dependencies between tumor cells and their microenvironment and were associated with patient outcomes. Perturbation of molecular co-dependencies via random spatial shuffling of the microenvironment resulted in destabilization of the corresponding villages. This study provides new insights into understanding tumor spatial landscape and its impact on tumor aggressiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d9a8ffa548a2f4ea069cb9bc5e6205adbb07d1" target='_blank'>
              Tumor cell villages define the co-dependency of tumor and microenvironment in liver cancer
              </a>
            </td>
          <td>
            Meng Liu, M. Hernandez, Darko Castven, Hsin-Pei Lee, Wenqi Wu, Limin Wang, M. Forgues, Jonathan M. Hernandez, Jens U. Marquardt, Lichun Ma
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee24fc55015da54029a007a558ae389dd346d690" target='_blank'>
              Mapping the spatial architecture of glioblastoma from core to edge delineates niche-specific tumor cell states and intercellular interactions
              </a>
            </td>
          <td>
            Saad M. Khan, Anthony Z. Wang, Rupen Desai, Colin R. McCornack, Rui Sun, Sonika M. Dahiya, Jennifer Foltz-Stringfellow, Ngima D Sherpa, Lydia Leavitt, T. West, Alexander F. Wang, Aleksandar Krbanjevic, Bryan D. Choi, Eric Leuthardt, B. Patel, Al Charest, Albert H. Kim, Gavin P. Dunn, Allegra A. Petti
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a crucial role in orchestrating tumor cell behavior and cancer progression. Recent advances in spatial profiling technologies have uncovered novel spatial signatures, including univariate distribution patterns, bivariate spatial relationships, and higher-order structures. These signatures have the potential to revolutionize tumor mechanism and treatment. In this review, we summarize the current state of spatial signature research, highlighting computational methods to uncover spatially relevant biological significance. We discuss the impact of these advances on fundamental cancer biology and translational research, address current challenges and future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed05a90f6b1d7a182ae84bb7e554725c91c44235" target='_blank'>
              Quantifying and interpreting biologically meaningful spatial signatures within tumor microenvironments
              </a>
            </td>
          <td>
            Si Jing, He-Qi Wang, Ping Lin, Jiao Yuan, Zhixuan Tang, Hong Li
          </td>
          <td>2025-03-11</td>
          <td>NPJ Precision Oncology</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f15f8a11af43d80ae3f5bc50b4193451c3cfbd" target='_blank'>
              Mapping Tumor Microenvironment and Treatment Response of Diffuse Midline Glioma Using Multiplexed Immunofluorescence and AI Models
              </a>
            </td>
          <td>
            Sandra Laternser, A. D. De Micheli, Sarah Brüningk, E. McDonough, Antonela Petrovic, Elisabeth Jane Rushing, Christine Surrette, Julia Bugajska, A. Eze, Lindsay Kilburn, Luisa Machado, Susanne Dettwiler, Fabiola Prutek, Noelia Casares Lagar, D. de la Nava, Denise Morinigo, Fiona Ginty, Matthew D. Dun, Michael A. Grotzer, A. G. Stücklin, Sabine Mueller, R. Packer, M. Bornhorst, Marta María Alonso, Javad Nazarian
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy, recently approved by the FDA for melanoma, is an emerging modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors such as glioblastoma remains limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas using spectral flow cytometry, TCR sequencing, single-cell RNA-seq, Xenium in situ transcriptomics, and CODEX spatial proteomics. Comparative analysis of TIL-generating (TIL⁺) versus non-generating (TIL⁻) tumors revealed that IL7R expression, structured perivascular immune clustering, and tumor-intrinsic metabolic programs such as ACSS3 were associated with successful TIL expansion. In contrast, TIL⁻ tumors were enriched for neuronal lineage signatures, immunosuppressive transcripts including TOX and FERMT1, and tumor-connected macrophages. This study defines spatial and molecular correlates of TIL manufacturing success and establishes a genomics-enabled selection platform for adoptive T cell therapy. The profiling approach is now being prospectively implemented in the GIANT clinical trial (NCT06816927), supporting its translational relevance and scalability across glioblastoma and other immune-excluded cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/945f91b27ecd7bfb796d6743f706c6451c1e2314" target='_blank'>
              A Spatial Multi-Omic Framework Identifies Gliomas Permissive to TIL Expansion
              </a>
            </td>
          <td>
            K. Hotchkiss, Kenan Zhang, Anna M. Corcoran, Elizabeth Owens, P. Noldner, C. Railton, Kyra Van Batavia, Ying Zhou, Jodie Jepson, Kirit Singh, R. McLendon, Kristen Batich, Anoop P Patel, Katayoun Ayasoufi, Michael Brown, Evan Calabrese, Jichun Xie, Jose R. Conejo-Garcia, B. Shaz, John W. Hickey, Mustafa Khasraw
          </td>
          <td>2025-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98801c9770ab85a801e0ca53d0db3280fc312fc" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma invasion into healthy brain tissue remains a major barrier to effective treatment, yet current models fail to capture its full complexity in a scalable and patient-specific manner. Here, we introduce GlioTrace, a novel ex vivo imaging and AI-based analytical framework that enables real-time, spatiotemporal tracking of glioblastoma invasion dynamics in patient-derived glioma cell culture xenograft (PDCX) brain slices. By integrating whole-specimen confocal microscopy, vascular counterstaining, and an advanced computational pipeline combining convolutional neural networks and Hidden Markov Models, GlioTrace identifies distinct invasion modes, including dynamic morphological switching, vessel-guided migration, and immune cell interactions and quantifies patient-specific variations in invasion plasticity. Using GlioTrace, we demonstrate that targeted therapies can selectively modulate invasion phenotypes, revealing spatially and temporally distinct drug responses. This scalable platform provides an unprecedented window into glioblastoma progression and treatment response, offering a powerful tool for precision oncology and anti-invasion therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d85bb045bd5b47db33273aef84595c6d869e5e" target='_blank'>
              Reconstructing the Single-Cell Spatiotemporal Dynamics of Glioblastoma Invasion
              </a>
            </td>
          <td>
            H. Mangukiya, Madeleine Skeppås, Soumi Kundu, Maria Berglund, Adam A. Malik, C. Krona, Sven Nelander
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R. Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Gliomas are among the most lethal cancers, with limited treatment options. To uncover hallmarks of therapeutic escape and tumor microenvironment (TME) evolution, we applied spatial proteomics, transcriptomics, and glycomics to 670 lesions from 310 adult and pediatric patients. Single-cell analysis shows high B7H3+ tumor cell prevalence in glioblastoma (GBM) and pleomorphic xanthoastrocytoma (PXA), while most gliomas, including pediatric cases, express targetable tumor antigens in less than 50% of tumor cells, potentially explaining trial failures. Longitudinal samples of isocitrate dehydrogenase (IDH)-mutant gliomas reveal recurrence driven by tumor-immune spatial reorganization, shifting from T-cell and vasculature-associated myeloid cell-enriched niches to microglia and CD206+ macrophage-dominated tumors. Multi-omic integration identified N-glycosylation as the best classifier of grade, while the immune transcriptome best predicted GBM survival. Provided as a community resource, this study opens new avenues for glioma targeting, classification, outcome prediction, and a baseline of TME composition across all stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08309318ca287b808eae53da393baf17b1d09b8" target='_blank'>
              Multi-omic landscape of human gliomas from diagnosis to treatment and recurrence
              </a>
            </td>
          <td>
            Hadeesha Piyadasa, Benjamin Oberlton, Mikaela Ribi, Jolene S Ranek, Inna Averbukh, K. Leow, M. Amouzgar, Candace C. Liu, Noah F. Greenwald, E. Mccaffrey, Rashmi Kumar, Selena Ferrian, Albert G. Tsai, Ferda Filiz, C. Fullaway, M. Bosse, Sricharan Reddy Varra, Alex Kong, Cameron Sowers, M. Gephart, Pablo Nuñez-Perez, EnJun Yang, Michael Travers, Michael Schachter, Samantha Liang, Maria R. Santi, Samantha Bucktrout, P. Gherardini, John Connolly, Kristina Cole, Michael Barish, Christine E. Brown, Derek A. Oldridge, Richard R. Drake, Joanna J. Phillips, Hideho Okada, Robert Prins, Sean C Bendall, Michael Angelo
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Adult-onset dermatomyositis (DM) is an autoimmune inflammatory myopathy with distinct cutaneous manifestations and a strong malignancy association. Through comparative analysis with cutaneous lupus erythematosus (CLE), our integrated spatial and single-cell transcriptomics analysis revealed unique immune and stromal niches associated with DM subtypes. Unexpectedly, we found an association between cancer-associated DM skin lesions and the presence of dispersed immune infiltrates enriched with macrophages, CD8+ T cells, plasma cells, and B cells with preserved vascular architecture. In contrast, non-cancer associated DM skin exhibited dense myeloid cell infiltrates, including neutrophils, monocytes, and macrophages, with elevated expression of IL1B and CXCL10 localized near injured vascular endothelia. Cytokines produced by these myeloid infiltrates together with local tissue hypoxia triggered dramatic stromal remodeling, leading to loss of vascular-associated fibroblasts. In addition to the CXCL10+ myeloid signature, non-cancer-associated DM skin with pDC presence showed the emergence of specific cellular pairs: PD-L1-expressing mregDCs and activated Tregs expressing NFKB2 and TNF receptors. While both DM and CLE showed strong interferon signatures, DM uniquely displayed IFN-β expression. Together, our study provides the first comprehensive spatial mapping of immune and stromal cells in adult-onset DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62db8748b2521d6e18e161c76b546fccb24b5fcd" target='_blank'>
              Spatial Transcriptomics Identifies Immune-Stromal Niches Associated with Cancer in Adult Dermatomyositis
              </a>
            </td>
          <td>
            Ksenia S. Anufrieva, Neda Shahriari, Ce Gao, Rochelle L. Castillo, Jessica Liu, Sean A Prell, Shideh Kazerounian, Khashayar Afshari, Anastasia N. Kazakova, Erin Theisen, Teresa Bowman, A. LaChance, Kimberly B. Hashemi, Ilya Korsunsky, Mehdi Rashighi, R. A. Vleugels, Kevin Wei
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In this study, we developed an integrated single cell transcriptomic (scRNAseq) atlas of human breast cancer (BC), the largest resource of its kind, totaling > 600,000 cells across 138 patients. Rigorous integration and annotation of publicly available scRNAseq data enabled a highly resolved characterization of epithelial, immune, and stromal heterogeneity within the tumor microenvironment (TME). Within the immune compartment we were able to characterize heterogeneity of CD4, CD8 T cells and macrophage subpopulations. Within the stromal compartment, subpopulations of endothelial cells (ECs) and cancer associated fibroblasts (CAFs) were resolved. Within the cancer epithelial compartment, we characterized the functional heterogeneity of cells across the axes of stemness, epithelial-mesenchymal plasticity, and canonical cancer pathways. Across all subpopulations observed in the TME, we performed a multi-resolution survival analysis to identify epithelial cell states and immune cell types which conferred a survival advantage in both The Cancer Genome Atlas (TCGA) and METABRIC. We also identified robust associations between TME composition and clinical phenotypes such as tumor subtype and grade that were not discernible when the analysis was limited to individual datasets, highlighting the need for atlas-based analyses. This atlas represents a valuable resource for further high-resolution analyses of TME heterogeneity within BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7db0b505659425c4bfdef92aba55ca31e10f8c9" target='_blank'>
              A highly resolved integrated transcriptomic atlas of human breast cancers
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, S. Monti
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Triple-negative breast cancer, characterized by aggressive growth and high intratumor heterogeneity, presents a significant clinical challenge. Here, we use a lineage-tracing system, ClonMapper, which couples heritable clonal identifying tags with single-cell RNA-sequencing (scRNA-seq), to better elucidate the response to doxorubicin in a model of TNBC. We demonstrate that, while there is a dose-dependent reduction in overall clonal diversity, there is no pre-existing resistance signature among surviving clones. Separately, we found the existence of two transcriptomically distinct clonal subpopulations that remain through the course of treatment. Among clones persisting across multiple samples we identified divergent phenotypes, suggesting a response to treament independent of clonal identity. Finally, a subset of clones harbor novel changes in expression following treatment. The clone and sample specific responses to treatment identified herein highlight the need for better personalized treatment strategies to overcome tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88360a99c9dba219540b793cdb094157ffa2ec6a" target='_blank'>
              Lineage Tracing Reveals Clone-Specific Responses to Doxorubicin in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Daylin Morgan, Andrea L. Gardner, Amy Brock
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06a703b5eeba2e901b0564ff4995a24e3b880d79" target='_blank'>
              Single-cell spatial analysis of pediatric high-grade glioma reveals a novel population of SPP1+/GPNMB+ myeloid cells with immunosuppressive and tumor-promoting capabilities
              </a>
            </td>
          <td>
            Thijs J M van den Broek, R. Hoogendijk, M. Kranendonk, J. Lammers, Akshaya L. Krishnamoorthy, Ravian L. van Ineveld, Milo Molleson, Vasily O. Tsvetkov, F. Ringnalda, M. van de Wetering, Yan Su, John I. Bianco, Cristian Ruiz Moreno, Mario G Ries, E. Hoving, J. van der Lugt, Leila Akkari, David P. Schrijver, H. Stunnenberg, A. Rios, D. V. van Vuurden, A. Zomer
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background The tumor microenvironment (TME) is a complex and dynamic ecosystem that is known to influence responses to immunotherapy. We leveraged single-cell spatial transcriptomics to systematically dissect the intricate complexity of the TME, in particular the cellular heterogeneity and spatial interactions. Their collective impact on immunotherapy efficacy was studied in the context of a homogeneous group of patients with vulvar high-grade squamous intraepithelial lesions (vHSIL) treated with an immunotherapeutic tumor-specific peptide vaccine. Methods We performed single-cell spatial transcriptomics on 20 pretreatment vHSIL lesions, stratified by clinical response to immunotherapeutic vaccination into complete responders (CR), partial responders (PR) and non-responders (NR). Using a 1,000-gene panel, we mapped over 274,000 single cells in situ, identifying 18 cell clusters and 99 distinct non-epithelial cell states. Findings were validated against public single-cell transcriptomic data sets to assess their broader relevance across tumor types. Results Profound heterogeneity within the TME was detected across the response groups. CR lesions exhibited a higher ratio of immune-supportive to immune-suppressive cells—a pattern mirrored in other solid tumors following neoadjuvant checkpoint blockade. Key immune populations enriched in CRs included CD4+CD161+ effector T cells and chemotactic CD4+ and CD8+ T cells. Conversely, PRs were characterized by increased proportions of T helper 2 cells and CCL18-expressing macrophages, which are associated with the recruitment of type 2 T cells and regulatory T cells. NRs displayed preferential infiltration with immunosuppressive fibroblasts. Distinct spatial immune ecosystems further defined response groups. Although a number of immune cells were detected in all patients, type 1 effector cells dominated interactions in CRs, type 2 cells were prominently interacting in PRs, while NRs lacked organized immune cell interactions. Conclusions This study underscores the dual importance of both cellular composition and spatial organization in steering clinical response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b353470be584af7d95e65b2aa9a5dffe4241b32d" target='_blank'>
              Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy
              </a>
            </td>
          <td>
            Z. Abdulrahman, R. Slieker, Daniel McGuire, M. J. Welters, M. V. van Poelgeest, S. H. van der Burg
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80bfc8505face3c0cb655cd2772225105395f750" target='_blank'>
              Deep spatial proteomics of ovarian cancer precursor lesions delineates early disease changes and cell-of-origin signatures
              </a>
            </td>
          <td>
            Anuar Makhmut, M. Dragomir, Sonja Fritzsche, Markus Moebs, Wolfgang D. Schmitt, Eliane T Taube, Fabian Coscia
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Glioblastoma, one of the most aggressive and heterogeneous malignant tumors, presents significant challenges for clinical management due to its cellular and metabolic complexity. This review integrates recent advancements in single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to elucidate glioblastoma’s cellular heterogeneity and metabolic reprogramming. Diverse cellular subpopulations, including malignant proliferative cells, stem-like cells, mesenchymal-like cells, and immune-related cells, contribute to tumor progression, treatment resistance, and microenvironmental interactions. Spatial transcriptomics has further revealed distinct spatial distributions of these subpopulations, highlighting differences in metabolic activities between the tumor core and periphery. Key metabolic adaptations, such as enhanced glycolysis, fatty acid oxidation, and glutamine metabolism, play critical roles in supporting tumor growth, immune evasion, and therapeutic resistance. Targeting these metabolic pathways, especially in combination with immunotherapy, represents a promising avenue for glioblastoma treatment. This review emphasizes the importance of integrating single-cell and spatial multi-omics technologies to decode glioblastoma’s metabolic landscape and explore novel therapeutic strategies. By addressing current challenges, such as metabolic redundancy and spatiotemporal dynamics, this work provides insights into advancing precision medicine for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea52e3e92b1a5c4c3caddeca60f128feaaaf952b" target='_blank'>
              Single-cell and spatial transcriptomic insights into glioma cellular heterogeneity and metabolic adaptations
              </a>
            </td>
          <td>
            Yixin Fu, Yong Yi, Yongxiang Shao, Jingcheng Jiang, Qingshan Deng
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b6d1c2313c8e86b7cbcc29c94f8060fb9e7af4" target='_blank'>
              Abstract B048: Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy
              </a>
            </td>
          <td>
            Ruan FV Medrano, Vladimir Sukhov, Maxim N Artyomov, Robert Schreiber
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spatial transcriptomics integrates transcriptomics data with histological tissue images, offering deeper insights into cellular organization and molecular functions. However, existing computational platforms mainly focus on genomic analysis, leaving a gap in the seamless integration of genomic and image analysis. To address this, we introduce Thor, a comprehensive computational platform for multi-modal analysis of spatial transcriptomics and histological images. Thor utilizes an anti-shrinking Markov diffusion method to infer single-cell spatial transcriptomes from spot-level data, effectively integrating cell morphology with spatial transcriptomics. The platform features 10 modules designed for cell-level genomic and image analysis. Additionally, we present Mjolnir, a web-based tool for interactive tissue analysis using vivid gigapixel images that display information on histology, gene expression, pathway enrichment, and immune response. Thor’s accuracy was validated through simulations and ISH, MERFISH, Xenium, and Stereo-seq datasets. To demonstrate its versatility, we applied Thor for joint genomic-histology analysis across various datasets. In in-house heart failure patient samples, Thor identified a regenerative signature in heart failure, with protein presence confirmed in blood vessels through immunofluorescence staining. Thor also revealed the layered structure of the mouse olfactory bulb, performed unbiased screening of breast cancer hallmarks, elucidated the heterogeneity of immune responses, and annotated fibrotic regions in multiple heart failure zones using a semi-supervised approach. Furthermore, Thor imputed high-resolution spatial transcriptomics data in an in-house bladder cancer sample sequenced using Visium HD, uncovering stronger spatial patterns that align more closely with histology. Bridging the gap between genomic and image analysis in spatial biology, Thor offers a powerful tool for comprehensive cellular and molecular analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d882bc79c8c5d2b2813a2269a80bde6db2f397cb" target='_blank'>
              Thor: a platform for cell-level investigation of spatial transcriptomics and histology
              </a>
            </td>
          <td>
            Guangyu Wang, Pengzhi Zhang, Weiqing Chen, Tu Tran, Minghao Zhou, Kaylee Carter, Ibrahem Kandel, Shengyu Li, Li Lai, Qianqian Song, Keith Youker, Yu Yang, Keith Syson Chan, Xen Ping Hoi, Fotis Nikolo
          </td>
          <td>2025-03-10</td>
          <td>Research Square</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73634c31cfc63813b66fedfb8ecff6a08ba232af" target='_blank'>
              Spatiotemporal organisation of residual disease in mouse and human BRCA1-deficient mammary tumours and breast cancer
              </a>
            </td>
          <td>
            Demeter Túrós, Morgane Decollogny, Astrid Chanfon, Myriam Siffert, L. Romanens, J. Tille, O. Trédan, I. Labidi-Galy, Alberto Valdeolivas, Sven Rottenberg
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Here we present STModule, a Bayesian method developed to identify tissue modules from spatially resolved transcriptomics that reveal spatial components and essential characteristics of tissues. STModule uncovers diverse expression signals in transcriptomic landscapes such as cancer, intraepithelial neoplasia, immune infiltration, outcome-related molecular features and various cell types, which facilitate downstream analysis and provide insights into tumor microenvironments, disease mechanisms, treatment development, and histological organization of tissues. STModule captures a broader spectrum of biological signals compared to other methods and detects novel spatial components. The tissue modules characterized by gene sets demonstrate greater robustness and transferability across different biopsies. STModule: https://github.com/rwang-z/STModule.git. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01441-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61c2f79171ce9194bffe681aecda7bf66be2273" target='_blank'>
              STModule: identifying tissue modules to uncover spatial components and characteristics of transcriptomic landscapes
              </a>
            </td>
          <td>
            Ran Wang, Yan Qian, Xiaojing Guo, Fangda Song, Zhiqiang Xiong, Shirong Cai, Xiuwu Bian, Man-Hon Wong, Qin Cao, Lixin Cheng, Gang Lu, K. Leung
          </td>
          <td>2025-03-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background The heterogeneity of tumors significantly impacts on colorectal cancer (CRC) progression. However, the influence of this heterogeneity on the spatial architecture of CRC remains largely unknown. Methods Spatial transcriptomic (ST) analysis of AOM/DSS-induced colorectal cancer (CRC), integrated with single-cell RNA sequencing, generated a comprehensive spatial atlas of CRC. Pseudotime trajectory, stemness evaluation, and cell-cell communication analyses explored how CD44+ tumor cells at the leading edge remodel the tumor microenvironment (TME). In vitro experiments and immunofluorescence staining of clinical samples validated pleiotrophin (PTN) signaling in promoting cancer-associated fibroblasts (CAFs) phenotypic transition and CRC progression. Results Our findings revealed a distinctive layered ring-like structure within CRC tissues, where CD44+ tumor cells exhibiting high stemness were positioned at the tumor’s leading edge. Inflammatory CAFs (iCAFs)-like, myofibroblastic CAFs (myCAFs)-like cells and pro-tumorigenic neutrophils primarily located at the tumor edge, in proximity to CD44+ tumor cells. CD44+ tumor cells then triggered the phenotypic transition of CAFs into iCAF-like and myCAF-like cells through PTN signaling. Conclusions Our results provide distinctive insights into how tumor heterogeneity reshapes the TME at the leading edge of tumor, thereby promoting CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562ecaa8f1395c90f98e5b2a88e96773ab95655e" target='_blank'>
              CD44+ cells enhance pro-tumor stroma in the spatial landscape of colorectal cancer leading edge
              </a>
            </td>
          <td>
            F. Tang, Yong-Gui Zhu, Jia Shen, Bowen Yuan, Xiang He, Yuxi Tian, Liang Weng, Lunquan Sun
          </td>
          <td>2025-03-12</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly malignant tumor in women, characterized by high morbidity, mortality, and recurrence rates. Although surgical treatment, radiotherapy, and chemotherapy are the mainstays of current treatment methods, the high heterogeneity of TNBC results in unsatisfactory outcomes with low 5-year survival rates. Rapid advancements in omics technology have propelled the understanding of TNBC molecular biology. The emergence of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has significantly enhanced knowledge of tumor heterogeneity and the distribution, functionality, and intercellular interactions of various cell types within the tumor microenvironment, including tumor cells, T cells, B cells, macrophages, and fibroblasts. The present study provides an overview of the technical characteristics of scRNA-seq and ST, highlighting their applications in exploring TNBC heterogeneity, cell spatial distribution patterns, and intercellular interactions. This review aims to enhance the comprehension of TNBC at the cellular level for the development of effective therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db2a16a0771f525b51537059ff8768102022813" target='_blank'>
              Analysis of single-cell and spatial transcriptomics in TNBC cell-cell interactions
              </a>
            </td>
          <td>
            Yan Xin, Qiji Ma, Qiang Deng, Tielin Wang, Dongxu Wang, Gang Wang
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have transformed the treatment landscape across multiple cancer types achieving durable responses for a significant number of patients. Despite their success, many patients still fail to respond to ICIs or develop resistance soon after treatment. We sought to identify early treatment features associated with ICI outcome. We leveraged the MC38 syngeneic tumor model because it has variable response to ICI therapy driven by tumor intrinsic heterogeneity. ICI response was assessed based on the level of immune cell infiltration into the tumor – a well-established clinical hallmark of ICI response. We generated a spatial atlas of 48,636 transcriptome-wide spots across 16 tumors using spatial transcriptomics; given the tumors were difficult to profile, we developed an enhanced transcriptome capture protocol yielding high quality spatial data. In total, we identified 8 tumor cell subsets (e.g., proliferative, inflamed, and vascularized) and 4 stroma subsets (e.g., immune and fibroblast). Each tumor had orthogonal histology and bulk-RNA sequencing data, which served to validate and benchmark observations from the spatial data. Our spatial atlas revealed that increased tumor cell cholesterol regulation, synthesis, and transport were associated with a lack of ICI response. Conversely, inflammation and T cell infiltration were associated with response. We further leveraged spatially aware gene expression analysis, to demonstrate that high cholesterol synthesis by tumor cells was associated with cytotoxic CD8 T cell exclusion. Finally, we demonstrate that bulk RNA-sequencing was able to detect immune correlates of response but lacked the sensitivity to detect cholesterol synthesis as a feature of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5d48556d03df2b88d1c06ed57c2bca71cfaf9b2" target='_blank'>
              Spatial genomics reveals cholesterol metabolism as a key factor in colorectal cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Andrew J. Kavran, Yulong Bai, Brian Rabe, Anna Kreshock, Andrew Fisher, Yelena Cheng, Anne Lewin, Chao Dai, Matthew J. Meyer, Konstantinos J. Mavrakis, Anna Lyubetskaya, Eugene Drokhlyansky
          </td>
          <td>2025-03-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="It has been reported that microsatellite instable (MSI) tumors in colorectal cancer (CRC) exhibit stronger anti-tumor responses than microsatellite stable (MSS) tumors. Analyzing scRNA-seq data from 185 CRC patients we found that immune, structural, and cancer cells in MSS tumors with high numbers of tumor-infiltrating CD8 T cells and macrophages (TAMs; CD8hiTAMhi) and CD8lowTAMhi tumors were enriched for pathways associated with anti- and pro-tumor responses, respectively. In CD8hiTAMhi tumors, TAMs expressed an IFN-induced phenotype (e.g. GBP1, CXCL9, IFITM3) and high infiltration of GBP1+ TAMs was associated with better overall survival (n=941). High-resolution spatial transcriptomics (Visium HD) revealed that GBP1+ TAMs clustered with CXCL13+IFNG+PDCD1+ CD8 T cells in tumoral niches, suggesting that GBP1+ TAMs were involved in recruitment and activation of tumor-reactive CD8 T cells. Together, we uncovered coordinated cellular programs across cell types in the microenvironment of MSS tumors, based on simple classification variables, that were strongly associated with prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5dfde3e4f7d39011a34eb5a718ac54f39aa6ee4" target='_blank'>
              Single cell spatial transcriptomics identifies coordinated cellular programs associated with good prognosis in microsatellite stable colorectal cancer
              </a>
            </td>
          <td>
            Victoria T. Karlsen, Umair Majid, Håvard T. Lindholm, S. U. Umu, Karoline Rapp Vander-Elst, A-C Røberg Beitnes, Marianne A. Merok, Kjersti Thorvaldsen Hagen, H. R. Nilsen, Sheraz Yaqub, E. Bækkevold, Diana Domanska, F. Jahnsen
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Objective This study examines the cellular heterogeneity of epithelial cells within pancreatic ductal adenocarcinoma (PDAC) and their contributions to tumor progression, metastasis, and immunosuppressive interactions using single-cell RNA sequencing. Methods Single-cell RNA-sequencing data from two datasets (GSE154778 and GSE158356) were integrated using the Harmony algorithm, followed by quality control, clustering, and differential gene expression analysis. Distinct subpopulations of epithelial cells were identified, and their gene expression profiles were analyzed. To assess the malignancy of these subpopulations, single-cell copy number variation (CNV) analysis and trajectory analysis were conducted. Additionally, intercellular communication was examined using the CellChat platform. Results The analysis revealed pronounced heterogeneity among PDAC epithelial cells, with specific subpopulations exhibiting distinct roles in tumor proliferation, extracellular matrix remodeling, and metastatic dissemination. Subpopulations 4 and 6 were characterized by increased CNV levels and a more malignant phenotype, suggesting an enhanced capacity for metastasis. Single-cell trajectory analysis, along with CellChat, mapped the temporal evolution of epithelial cells, identifying key regulatory genes such as DCBLD2 and JUN. A prognostic model incorporating five key genes, including KLF6, was developed and demonstrated strong predictive accuracy for patient outcomes. Notably, KLF6 emerged as a critical prognostic marker associated with immune modulation, particularly through interactions with M2 macrophages. Conclusion The study highlights the pronounced heterogeneity of epithelial cells in PDAC and their distinct contributions to tumor progression, metastasis, and immune modulation. Through single-cell transcriptomic and CNV analyses, we identified epithelial subpopulations with varying malignant potentials and distinct interactions with the tumor microenvironment. Among these, KLF6 emerged as a key regulator associated with immune modulation and metastasis. Our findings emphasize the significance of epithelial cell heterogeneity in shaping pancreatic cancer progression. These insights provide a foundation for future investigations into novel prognostic markers and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12957-025-03793-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c02e096347df4014f9175b24f19fbd3cae4aa9d0" target='_blank'>
              Deciphering the heterogeneity of epithelial cells in pancreatic ductal adenocarcinoma: implications for metastasis and immune evasion
              </a>
            </td>
          <td>
            Jie Liu, Hui Li, Xiuyun Lin, Jiani Xiong, Guangfeng Wu, Lingyan Ding, Bin Lin
          </td>
          <td>2025-04-16</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421024c849016285df42c8f390ac35abcedd51e" target='_blank'>
              Pan-cancer spatial characterization of key immune biomarkers in the tumor microenvironment
              </a>
            </td>
          <td>
            James R. Lindsay, Jennifer Altreuter, J. Alessi, Jason L. Weirather, Anita Giobbie-Hurder, Ian D Dryg, Katharina V. Hoebel, B. Sharma, K. Felt, F. S. Hodi, N. Lindeman, L. Sholl, E. Cerami, Jonathan A. Nowak, Mark M Awad, Scott J Rodig, William E. Lotter
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1db37f2f209f5470571ed3506376b29858f863c2" target='_blank'>
              Tumor cell heterogeneity drives spatial organization of the intratumoral immune response.
              </a>
            </td>
          <td>
            Miho Tanaka, Lotus Lum, Kenneth H Hu, Piyush Chaudhary, Savannah Hughes, Cecilia Ledezma-Soto, B. Samad, Daphne Superville, Kenneth Ng, Arun Chumber, Ciara Benson, Zoe N Adams, Kelly Kersten, Oscar A. Aguilar, Lawrence Fong, Alexis J. Combes, Matthew F. Krummel, Melissa Q Reeves
          </td>
          <td>2025-06-02</td>
          <td>The Journal of experimental medicine</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178582b0b4cac0c55049a52434a209bd8b7613e6" target='_blank'>
              Quantitative measurement of phenotype dynamics during cancer drug resistance evolution using genetic barcoding
              </a>
            </td>
          <td>
            F. J. Whiting, M. Mossner, C. Gabbutt, C. Kimberley, C. P. Barnes, A. Baker, Erika Yara, A. Sottoriva, Richard A. Nichols, Trevor A. Graham
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="A striking characteristic of liver cancer is its extensive heterogeneity, particularly with regard to its varied response to immunotherapy. In this study, we employed multimodal sequencing approaches to explore the various aspects of neoadjuvant nivolumab treatment in liver cancer patients. We used spatially-resolved transcriptomics, single- and bulk-cell transcriptomics, and TCR clonotype analyses to examine the spatiotemporal dynamics of the effects of nivolumab. We observed a significantly higher clonal expansion of T cells in the tumors of patients who responded to the treatment, while lipid accumulation was detected in those of non-responders, likely due to inherent differences in lipid metabolic processes. Furthermore, we found a preferential enrichment of T cells, which was associated with a better drug response. Our results also indicate a functional antagonism between tumor-associated macrophages (TAMs) and CD8 cells and their spatial separation. Notably, we identified a UBASH3B/NR1I2/CEACAM1/HAVCR2 signaling axis, highlighting the intense communication among TAMs, tumor cells, and T-cells that leads to pro-tumorigenic outcomes resulting in poorer nivolumab response. In summary, using integrative multimodal sequencing investigations, combined with the multi-faceted exploration of pre- and post-treatment samples of neoadjuvant nivolumab-treated HCC patients, we identified useful mechanistic determinants of therapeutic response. We also reconstructed the spatiotemporal model that recapitulates the physiological restoration of T cell cytotoxicity by anti-PD1 blockade. Our findings could provide important biomarkers and explain the mechanistic basis differentiating the responders and non-responders. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02314-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ba5970d9a4d124e0362f78801008f40bb86612" target='_blank'>
              Multimodal sequencing of neoadjuvant nivolumab treatment in hepatocellular carcinoma reveals cellular and molecular immune landscape for drug response
              </a>
            </td>
          <td>
            Fanhong Zeng, Qingyang Zhang, Y. Tsui, Huanhuan Ma, Lu Tian, Abdullah Husain, Jingyi Lu, J. Lee, Vanilla Xin Zhang, Po-Man Li, Gary Cheuk-Hang Cheung, Tan-To Cheung, DanielWai-Hung Ho, I. Ng
          </td>
          <td>2025-04-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Intratumoral heterogeneity is the main cause of tumor treatment failure, varying across disease sites (spatial heterogeneity) and polyclonal properties of tumors that evolve over time (temporal heterogeneity). As our understanding of intratumoral heterogeneity, the formation of which is mainly related to the genomic instability, epigenetic modifications, plastic gene expression, and different microenvironments, plays a substantial role in drug-resistant as far as tumor metastasis and recurrence. Understanding the role of intratumoral heterogeneity, it becomes clear that a single therapeutic agent or regimen may only be effective for subsets of cells with certain features, but not for others. This necessitates a shift from our current, unchanging treatment approach to one that is tailored against the killing patterns of cancer cells in different clones. In this review, we discuss recent evidence concerning global perturbations of intratumoral heterogeneity, associations of specific intratumoral heterogeneity in lung cancer, the underlying mechanisms of intratumoral heterogeneity potentially leading to formation, and how it drives drug resistance. Our findings highlight the most up-to-date progress in intratumoral heterogeneity and its role in mediating tumor drug resistance, which could support the development of future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC), accounting for 30% of lung cancer cases, lacks adequate research due to limited understanding of its molecular abnormalities. Our study analyzed public LUSC datasets to explore the tumor microenvironment (TME) composition using scRNA-seq from two cohorts. Applying non-negative matrix factorization, we identified unique malignant cell phenotypes, or meta-programs (MPs), based on gene expression patterns. Survival analysis revealed the clinical relevance of these MPs. Findings illuminated a TME landscape enriched with immune cells—CD8 + T, exhausted T, CD4 + T, and naive T cells—and suggested roles for myeloid cells, like cDC1 and pDCs, in LUSC progression. Different MPs highlighted the heterogeneity of malignant cells and their clinical implications. Targeting MP-specific genes may enable personalized therapy, especially for early-stage LUSC. This study offers insights into immune cell function in tumor dynamics, identifies MPs, and paves the way for novel LUSC strategies, enhancing early intervention, personalized treatment, and prognosis, ultimately improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93916-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a302b866a51ae52b8b8d7352c3feab56f45e3d0e" target='_blank'>
              Single-cell RNA sequencing reveals potential therapeutic targets in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Junda Fan, Yu Chen, Yue Gong, Hongmei Sun, Rui Hou, Xiaoya Dou, Yanping Zhang, Cheng Huo
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Checkpoint inhibitors have led to remarkable benefits in non-small cell lung cancer (NSCLC), yet response rates remain below expectations. High-dimensional analysis and mechanistic experiments in clinical samples and relevant NSCLC models uncovered the immune composition of lung cancer tissues, providing invaluable insights into the functional properties of tumor-infiltrating T cells and myeloid cells. Among myeloid cells, type 1 conventional dendritic cells (cDC1s) stand out for their unique ability to induce effector CD8 T cells against neoantigens and coordinate antitumoral immunity. Notably, lung resident cDC1 are particularly abundant and long-lived and express a unique tissue-specific gene program, underscoring their central role in lung immunity. Here, we discuss recent insights on the induction and regulation of antitumoral T cell responses in lung cancer, separating it from the tissue-agnostic knowledge generated from heterogeneous tumor models. We focus on the most recent studies dissecting functional states and spatial distribution of lung cDC1 across tumor stages and their impact on T cell responses to neoantigens. Finally, we highlight relevant gaps and emerging strategies to harness lung cDC1 immunostimulatory potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2155ad4b9a79772729f8fb7af74e6020d7c7eb14" target='_blank'>
              Tissue-specific properties of type 1 dendritic cells in lung cancer: implications for immunotherapy
              </a>
            </td>
          <td>
            Lucía López Rodríguez, Roberto Amadio, G. M. Piperno, F. Benvenuti
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Breast cancers present intricate microenvironments comprising heterotypic cellular interactions, yet a comprehensive spatial map remained to be established. Here, we employed the DNA nanoball-based genome-wide in situ sequencing (Stereo-seq) to visualize the geospatial architecture of 30 primary breast tumors and metastatic lymph nodes across different molecular subtypes. This unprecedented high-resolution atlas unveils the fine structure of the tumor vasculature, highlighting heterogeneity in phenotype, spatial distribution, and intercellular communication within both endothelial and perivascular cells. In particular, venular smooth muscle cells are identified as the primary source of CCL21/CCL19 within the microenvironment. In collaboration with ACKR1-positive endothelial cells, they create a chemokine-rich venular niche to synergistically promote lymphocyte extravasation into tumors. High venule density predicts increased immune infiltration and improved clinical outcomes. This study provides a detailed spatial landscape of human breast cancer, offering key insights into the venular regulation of tumor immune infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3853b930721fb6df8f03ce40a70213e79309b5e" target='_blank'>
              Spatially resolved atlas of breast cancer uncovers intercellular machinery of venular niche governing lymphocyte extravasation
              </a>
            </td>
          <td>
            Xin Wang, Zhanyu Wang, Qijun Liao, Pei Yuan, Junpu Mei, Yin Zhang, Chao Wu, Xiyu Kang, Sufei Zheng, Chen-En Yang, Jiaxiang Liu, Qingyao Shang, Jiangtao Li, Bingning Wang, Liangyu Li, Hui Liu, Weining Hu, Zhensheng Dong, Jie Zhao, Linying Wang, Tao Liu, Yusheng Den, Chengrui Wang, Lijuan Han, Qianjun Chen, Huanming Yang, Xun Xu, Jie He, Zhen Yue, Nan Sun, Xiaodong Fang, Jianming Ying
          </td>
          <td>2025-04-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="PURPOSE
To identify the specific intratumoral and microenvironmental heterogeneity of acral (AM) and mucosal melanoma (MM), we aimed to delineate their distinct cellular compositions, evolutionary trajectories, and subtype-specific therapeutic strategies.


EXPERIMENTAL DESIGN
Single-cell transcriptomic and genomic landscapes were analyzed across 42 melanoma samples (28 AM, 11 MM, and 3 non-acral cutaneous melanoma [CM]), supplemented by in vitro and in vivo validation. Tumor and stromal cells were profiled using scRNA-seq, whole-exome sequencing, and functional assays, including transwell migration, co-culture systems, and xenograft models.


RESULTS
Tumor cells exhibited divergent evolutionary routes, with MM dominated by MGP⁺/PCOLCE⁺ subpopulations showing high epithelial-to-mesenchymal transition (EMT) potential. MM displayed elevated neutrophil infiltration and CXCL3⁺ tumor-associated macrophages, while AM was enriched with PI16⁺ cancer-associated fibroblasts (CAFs) promoting tumor proliferation. Molecular classification revealed MM subtypes: an antigen-presenting subtype linked to favorable outcomes and a proliferative subtype associated with recurrence. TIGIT⁺ Treg cells were enriched in AM, suggesting targeted inhibition potential. Genomic analysis connected BRAF/NRAS mutations to ALDOA⁺ stem-like tumor cells and identified PTGDS as a therapeutic target in triple-WT melanomas.


CONCLUSIONS
Our study provides a comprehensive comparison of AM and MM, uncovering subtype-specific stromal-immune interactions and molecular programs. The findings highlight actionable targets (e.g., TIGIT in AM, CXCL3⁺ macrophages in MM) and propose a framework for precision therapies, biomarker-driven trials, and risk stratification to improve outcomes in these aggressive melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d1f1c177fc4d5a9d6abeefba72d4bbfa28d55fc" target='_blank'>
              Single-cell transcriptomic landscape deciphers intratumoral heterogeneity and subtypes of acral and mucosal melanoma.
              </a>
            </td>
          <td>
            Yunyan Li, Ziyang Cui, Xiaole Song, Yeqing Chen, Cang Li, Junfeng Shi, Wenkang Qian, Guoxin Ren, Jiang Zhou, Chunpu Li, Xiaoqing Ma, Yifan Chen, Dongdong Jia, Yongli Zhang, Zhiling Zhang, Ronghao Zhang, Zhaotian Zhang, Yong Chen, Zhixiang Xu, Wantao Chen, X. Miao, Hongmeng Yu, Jianxin Chen, Kai Wang, Colin R. Goding, Zhi Wei, Tao Li, Rutao Cui
          </td>
          <td>2025-04-07</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Accumulating evidence links pediatric cancers to prenatal transformation events, yet the influence of the developmental stage on oncogenesis remains elusive. We investigated how hematopoietic stem cell developmental stages affect leukemic transformation, disease progression, and therapy response using a novel, humanized model of NUP98::NSD1-driven pediatric acute myeloid leukemia, that is particularly aggressive with WT1 co-mutations. Fetal-derived hematopoietic stem cells readily transform into leukemia, and WT1 mutations further enhance stemness and alter lineage hierarchy. In contrast, stem cells from later developmental stages become progressively resistant to transformation. Single-cell analyses revealed that fetal-origin leukemia stem cells exhibit greater quiescence and reliance on oxidative phosphorylation than their postnatal counterparts. These differences drive distinct therapeutic responses, despite identical oncogenic mutations. In patients, onco-fetal transcriptional programs correlate with worse outcomes. By targeting key vulnerabilities of fetal-origin leukemia cells, we identified combination therapies that significantly reduce aggressiveness, highlighting the critical role of ontogeny in pediatric cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97e870a1822de5485cb1751fdbfdd64fadf250" target='_blank'>
              Ontogeny Dictates Oncogenic Potential, Lineage Hierarchy, and Therapy Response in Pediatric Leukemia
              </a>
            </td>
          <td>
            Ke Wang, Shayan Saniei, Nikita Poddar, Subrina Autar, Saul Carcamo, Meghana Sreenath, Jack H Peplinski, R. Ries, Isabella G Martinez, Clifford Chao, Anna Huo-Chang Mei, Noshin Rahman, Levan Mekerishvili, Miguel Quijada-álamo, Grace Freed, Mimi Zhang, Katherine Lachman, Zayna Diaz, Manuel M Gonzalez, Jing Zhang, Giang Pham, Dan Filipescu, Mirela Berisa, Tommaso Balestra, J. Reisz, A. D’Alessandro, Daniel J Puleston, Emily Bernstein, Jerry E. Chipuk, M. Wunderlich, S. Tasian, B. Marcellino, Ian A. Glass, Christopher M Sturgeon, D. Landau, Zhihong Chen, E. Papapetrou, Franco Izzo, S. Meshinchi, Dan Hasson, Elvin Wagenblast
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef3b95effc04f0811082363795fb3dc9666b0f8" target='_blank'>
              Mapping immunotherapy potential: spatial transcriptomics in the unraveling of tumor-immune microenvironments in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Seo-Won Choi, Jeong Heon Kim, Jisu Hong, Minsu Kwon
          </td>
          <td>2025-04-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf0b044513a32c1bfdbc6767c40e3363e83eee34" target='_blank'>
              Abstract B038: A self-supervised AI model leveraging spatial omics for analyzing tumor microenvironment heterogeneity in breast cancer only with H&E
              </a>
            </td>
          <td>
            Haenara Shin, Dongjoo Lee, Yooeun Kim, D. Lee, Kwon Joong Na, C. Cha, Hosub Park, Hongyoon Choi
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d1b7da29ccf921ef29401ecf6ee3879598e277a" target='_blank'>
              Deconvoluting single-cell transcriptomics reveals cellular programs regulated by cell-cell communication in colorectal cancer
              </a>
            </td>
          <td>
            Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, John W. Paisley, Xinghua Lu, Lujia Chen
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Motivation Spatial transcriptomics holds great promise for revolutionizing biology and medicine by providing gene expression profiles with spatial information. Until recently, spatial resolution has been limited, but advances in high-throughput in situ imaging technologies now offer new opportunities by covering thousands of genes at a single-cell or even subcellular resolution, necessitating databases dedicated to comprehensive coverage and analysis with user-friendly intefaces. Results We introduce the HISSTA database, which facilitates the archival and analysis of in situ transcriptome data at single-cell resolution from various human tissues. We have collected and annotated spatial transcriptome data generated by MERFISH, CosMx SMI, and Xenium techniques, encompassing 112 samples and 28 million cells across 16 tissue types from 63 studies. To decipher spatial contexts, we have implemented advanced tools for cell type annotation, spatial colocalization, spatial cellular communication, and niche analyses. Notably, all datasets and annotations are interactively accessible through Vitessce, allowing users to focus on regions of interest and examine gene expression in detail. HISSTA is a unique database designed to manage the rapidly growing dataset of in situ transcriptomes at single-cell resolution. Given its comprehensive data content and advanced analysis tools with interactive visualizations, HISSTA is poised to significantly impact cancer diagnosis, precision medicine, and digital pathology. Availability and implementation HISSTA is freely accessible at https://kbds.re.kr/hissta/. The source code is available at https://doi.org/10.5281/zenodo.14904523.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8847bc6bc6ac698124acae8ba2d55144bc394fe" target='_blank'>
              HISSTA: a human in situ single-cell transcriptome atlas
              </a>
            </td>
          <td>
            Jiwon Yu, Jiwoo Moon, Minseo Kim, Gyeol Han, Insu Jang, Jinyoung Lim, Seungmook Lee, Seok-Hwan Yoon, Woong-Yang Park, Byungwook Lee, Sanghyuk Lee
          </td>
          <td>2025-03-31</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Glioblastoma (GBM), a primary brain tumor, exhibits intratumoral heterogeneity and dynamic spatial-temporal changes. GBM-derived extracellular vesicles (EVs), reflecting tumor characteristics, present potential as liquid-biopsy markers for early diagnosis and monitoring. This study aims to evaluate molecular signatures of plasma-derived EVs from GBM patients using a conventional flow cytometer. EVs have been isolated from glioma patients and healthy controls (HCs) plasma using density gradient ultracentrifugation (DGU). EVs were evaluated by bead-based multiplex analysis in a conventional flow cytometer. Principal component analysis (PCA), hierarchical clustering, and correlation analysis provided comprehensive insights into EV characteristics. EVs successfully isolated were visualized in transmission and scanning electron microscopy (STEM). Bead-based multiplex analysis in flow cytometer detected the level of 37 EV surface markers, including tumor-related, cancer stem cell, endothelial cell, and immune cell- specific antigens. PCA identified the EV surface markers that are most significant for differentiating the subjects, and hierarchical clustering revealed four distinct clusters based on EV surface marker levels. EV molecular signature demonstrated considerable heterogeneity across patient clusters. The presence of CD29 emerged not only as a defining factor for a cluster of patients, but also served as a marker to differentiate patients from HCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4072e04be0714ef76d0ca98717fc6de4c896a86" target='_blank'>
              Identification of distinct profiles of glioblastoma through the immunocapture of extracellular vesicles from patient plasma
              </a>
            </td>
          <td>
            D. Manu, R. Bălaşa, L. Pruteanu, V. Curean, Lucian Barbu-Tudoran, G. Șerban, Rareș Chinezu, A. Bălașa
          </td>
          <td>2025-03-19</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background The tumor boundary of breast cancer represents a highly heterogeneous region. In this area, the interactions between malignant and non-malignant cells influence tumor progression, immune evasion, and drug resistance. However, the spatial transcriptional profile of the tumor boundary and its role in the prognosis and treatment response of breast cancer remain unclear. Method Utilizing the Cottrazm algorithm, we reconstructed the intricate boundaries and identified differentially expressed genes (DEGs) associated with these regions. Cell-cell co-positioning analysis was conducted using SpaCET, which revealed key interactions between tumor-associated macrophage (TAMs) and cancer-associated fibroblasts (CAFs). Additionally, Lasso regression analysis was employed to develop a malignant body signature (MBS), which was subsequently validated using the TCGA dataset for prognosis prediction and treatment response assessment. Results Our research indicates that the tumor boundary is characterized by a rich reconstruction of the extracellular matrix (ECM), immunomodulatory regulation, and the epithelial-to-mesenchymal transition (EMT), underscoring its significance in tumor progression. Spatial colocalization analysis reveals a significant interaction between CAFs and M2-like tumor-associated macrophage (TAM), which contributes to immune exclusion and drug resistance. The MBS score effectively stratifies patients into high-risk groups, with survival outcomes for patients exhibiting high MBS scores being significantly poorer. Furthermore, drug sensitivity analysis demonstrates that high-MB tumors had poor response to chemotherapy strategies, highlighting the role of the tumor boundary in modulating therapeutic efficacy. Conclusion Collectively, we investigate the spatial transcription group and bulk data to elucidate the characteristics of tumor boundary molecules in breast cancer. The CAF-M2 phenotype emerges as a critical determinant of immunosuppression and drug resistance, suggesting that targeting this interaction may improve treatment responses. Furthermore, the MBS serves as a novel prognostic tool and offers potential strategies for guiding personalized treatment approaches in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b10699232e4b3f1c95f3a7d6be8ae3942b8fc0f" target='_blank'>
              Spatial multi-omics analysis of tumor-stroma boundary cell features for predicting breast cancer progression and therapy response
              </a>
            </td>
          <td>
            Yuanyuan Wu, Youyang Shi, Zhanyang Luo, Xiqiu Zhou, Yonghao Chen, Xiaoyun Song, Sheng Liu
          </td>
          <td>2025-03-26</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Human Immunodeficiency Virus (HIV) is one of the most prevalent viruses, causing significant immune depletion in affected individuals. Current treatments can control HIV and prolong patients’ lives, but new challenges have emerged. Increasing incidence of cancers occur in HIV patients. Esophageal squamous cell carcinoma (ESCC) is one of the most common cancers observed in HIV patients. However, the spatial cellular characteristics of HIV-related ESCC have not been explored, and the differences between HIV-ESCC and typical ESCC remain unclear. Methods We performed spatial transcriptome sequencing on HIV-ESCC samples to depict the microenvironment and employed cell communication analysis and multiplex immunofluorescence to investigate the molecular mechanism in HIV-ESCC. Results We found that HIV-ESCC exhibited a unique cellular composition, with fibroblasts and epithelial cells intermixed throughout the tumor tissue, lacking obvious spatial separation, while other cell types were sparse. Besides, HIV-ESCC exhibited an immune desert phenotype, characterized by a low degree of immune cell infiltration, with only a few SPP1+ macrophages showing immune resistance functions. Cell communication analysis and multiplex immunofluorescence staining revealed that tumor fibroblasts in HIV-ESCC interact with CD44+ epithelial cells via COL1A2, promoting the expression of PIK3R1 in epithelial cells. This interaction activates the PI3K-AKT signaling pathway, which contributes to the progression of HIV-ESCC. Conclusions Our findings depict the spatial microenvironment of HIV-ESCC and elucidate a molecular mechanism in the progression of HIV-ESCC. This will provide us insights into the molecular basis of HIV-ESCC and potential treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02248-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba81ba07e36306408c53ef4726b3e3e8f09de47b" target='_blank'>
              Spatial transcriptomic analysis of tumor microenvironment in esophageal squamous cell carcinoma with HIV infection
              </a>
            </td>
          <td>
            Yuanli Zuo, Yang Jin, Gang Li, Yue Ming, Ting Fan, Yitong Pan, Xiao-Jun Yao, Yong Peng
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b5519e4d897e90daae110bc4297e9d131ea31b" target='_blank'>
              Biobank of genetically defined murine prostate cancer tumoroids uncovers oncogenic pathways and drug vulnerabilities driven by PTEN-loss
              </a>
            </td>
          <td>
            Jessica Kalla, Thomas E. Dillinger, Zlata Pavlovicova, Reema Jacob, Emine Atas, Anil Baskan, Kristina Draganić, A. Tiefenbacher, T. Limberger, Theresia Mair, Gabriel Wasinger, Ludovica Villanti, S. Kubicek, Lukas Kenner, G. Egger
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is a highly lethal malignant intracranial tumor, distinguished from low-grade glioma by histopathological hallmarks such as pseudopalisading cells around necrosis (PAN) and microvascular proliferation (MVP). To date the spatial organization of the molecular and cellular components of these specific histopathological features has not been fully elucidated. Methods Here, using bulk RNA sequencing, spatial transcriptomic and single cell RNA sequencing (scRNA-seq) data of GBM patients, we identified niche-specific transcriptional programs and characterized the differences in molecular expression and cellular organization between PAN and MVP. Results Notably, we discovered spatially distinct domains within the tumor core and identified niche-specific signatures: NDRG1 and EPAS1, specifically expressed in the PAN and MVP regions. The clustering results showed two distinct phenotypes of endothelial cells (ECs) were enriched in the MVP and PAN regions, respectively. PAN-associated endothelial cells exhibit copy number variations similar to those in GBM cells. Single cell trajectory analysis reveals a pseudotime trajectory, indicating the differentiation of glioblastoma stem cells (GSCs) toward ECs. Conclusions Necrosis cores which are surrounded by hypoxic and perivascular niches and microvascular proliferation area within the glioblastoma tumor microenvironment, have been considered as standardized morphological indicators of aggressive GBM. Our findings provide a cellular and molecular insights into GBM progression. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06185-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75320d960d6bc41d40bd96ad524b1fc28b650a0b" target='_blank'>
              Multi-transcriptomics reveals niche-specific expression programs and endothelial cells in glioblastoma
              </a>
            </td>
          <td>
            Jiukun Hu, Xiaohan Sa, Yue Yang, Yuwen Han, Jie Wu, Minxuan Sun, S. Shafi, Nafees Ahmad, Sami Siraj, Jiao Yang, Yuanshuai Zhou
          </td>
          <td>2025-04-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) represents the most common pathological type of lung cancer, and the combination of neoadjuvant immunotherapy with chemotherapy has emerged as the first-line treatment for NSCLC. Nevertheless, the efficacy of this therapeutic approach remains variable. The present study aims to examine the impact of chemoimmunotherapy in NSCLC patients, with a view to identifying key molecules, critical cell subpopulations, communication patterns and spatial distributions that potentially correlate with therapeutic sensitivity. A total of 16 lung cancer tissue samples were collected from a cohort of 12 NSCLC patients and subjected to single-cell RNA and spatial transcriptome sequencing. Our data demonstrated that the distribution of CD4 + Treg T cells and mCAFs indicated an immunosuppressive tumor microenvironment, while the accumulation of CD4 + Th17 T cells and iCAFs could act as a positive marker for the sensitivity to chemoimmunotherapy. Furthermore, a significant high level of SELENOP-macrophages was observed in tissues from positive responders, and a strong co-localization between SELENOP-macrophages and antigen-presenting cancer associated fibroblasts (CAFs) in the tumor boundaries was identified, indicating the cooperative roles of these two cell types in response to combined therapy. Moreover, SELENOP-macrophages were observed to be accumulated in tertiary lymphoid structures, which further suggested its critical role in recruiting lymphocytes. Furthermore, analysis of cell–cell communication, based on spatial transcriptomics, suggests that the interactions between SELENOP-macrophages, apCAFs, CD4 + and CD8 + T cells were significantly enhanced in responders. In addition, SELENOP-macrophages recruited CD4 + Naïve, Helper and CD8 + Naïve T cells through pathways such as the cholesterol, interleukin, chemokine and HLA when responding to combined therapy. The present study further unveils the dynamic spatial and transcriptional changes in the tumor microenvironment of non-small cell lung cancer in response to combination therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02287-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86da4dd75d392abc980cae7b3d81379fef557c28" target='_blank'>
              Single-cell and spatial transcriptomic analyses revealing tumor microenvironment remodeling after neoadjuvant chemoimmunotherapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Xiaolu Cui, Siyuan Liu, He Song, Jingjing Xu, Yanbin Sun
          </td>
          <td>2025-04-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Hepatoblastoma (HB) is a common paediatric liver malignancy characterised by significant intratumoral heterogeneity and a complex tumour microenvironment (TME). Using single‐cell RNA sequencing (scRNA‐seq), we analysed 43,592 cells from three tumour regions and adjacent normal tissue of an HB patient. Our study revealed distinct cellular compositions and varying degrees of malignancy across different tumour regions, with the T1 region showing the highest malignancy and overexpression of HMGB2 and TOP2A. Survival analysis demonstrated that high HMGB2 expression is associated with poor prognosis and increased recurrence, suggesting its potential as a prognostic marker. Additionally, we identified a diverse immune microenvironment enriched with regulatory T cells (Tregs) and CD8+ effector memory T cells (Tem), indicating potential immune evasion mechanisms. Notably, CTLA‐4 and PD‐1 were highly expressed in Tregs and Tem cells, highlighting their potential as immunotherapy targets. Myeloid cells, including Kupffer cells and dendritic cells, also exhibited distinct functional roles in different tumour regions. This study provides the first comprehensive single‐cell atlas of HB, revealing critical insights into its intratumoral heterogeneity and immune microenvironment. Our findings not only advance the understanding of HB biology but also offer new directions for precision medicine, including the development of targeted therapies and immunotherapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/634b11e52e203c006a13f961efc458d554db20a9" target='_blank'>
              Intratumoral Heterogeneity and Immune Microenvironment in Hepatoblastoma Revealed by Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Mingdi Ma, Chen Jin, Qian Dong
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) represents a heterogeneous ecosystem and elucidation of tumor microenvironment components remains essential. Our study aimed to depict the composition and prognostic correlates of immune infiltrate in early BC, at a multiplex and spatial resolution. Pretreatment tumor biopsies from patients enrolled in the EORTC 10994/BIG 1-00 randomized phase III neoadjuvant trial (NCT00017095) were used; the CNN11 classifier for H&E-based digital TILs (dTILs) quantification and multiplex immunofluorescence were applied, coupled with machine learning (ML)-based spatial features. dTILs were higher in the triple-negative (TN) subtype, and associated with pathological complete response (pCR) in the whole cohort. Total CD4+ and intra-tumoral CD8+ T-cells expression was associated with pCR. Higher immune-tumor cell colocalization was observed in TN tumors of patients achieving pCR. Immune cell subsets were enriched in TP53-mutated tumors. Our results indicate the feasibility of ML-based algorithms for immune infiltrate characterization and the prognostic implications of its abundance and tumor-host interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de29bddf8b064faa7b8d05e6343f5b67380bf87" target='_blank'>
              Machine learning-based spatial characterization of tumor-immune microenvironment in the EORTC 10994/BIG 1-00 early breast cancer trial
              </a>
            </td>
          <td>
            I. Zerdes, A. Matikas, Artur Mezheyeuski, Georgios C Manikis, Balasz Acs, Hemming Johansson, C. Boyaci, C. Boman, Coralie Poncet, M. Ignatiadis, Yalai Bai, D. Rimm, David Cameron, Hervé Bonnefoi, Jonas Bergh, G. MacGrogan, T. Foukakis
          </td>
          <td>2025-03-07</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background The identification of the complex spatial architecture of immune cell infiltration and its interaction mechanisms within tumor ecosystems provides crucial insights into therapeutic responses to neoadjuvant therapy in muscle-invasive bladder cancer (MIBC). This study aims to characterize the spatial features of distinct cell-type niches within the tumor microenvironment (TME) of patients with varying responses to neoadjuvant therapy. Methods We performed spatial transcriptomic profiling on six MIBC specimens obtained from a registered clinical trial (ChiCTR2000032359), generating whole-transcriptome spatial atlases to map the TME architecture. High-throughput analytical frameworks were employed to deconstruct the TME, and key findings were validated through immunohistochemistry and mouse model experiments. Results Our analysis revealed that tissues from complete responders exhibited greater infiltration of T and B cells, with the formation of tertiary lymphoid structure (TLS). Trajectory analysis identified CCL19/CCL21 as the key signaling molecules driving TLS formation in MIBC. Mouse experiments demonstrated that recombinant CCL19/CCL21 protein injections promoted intratumoral TLS formation and enhance the efficacy of immunotherapy. Furthermore, we observed significant intrinsic heterogeneity within individual tumors, which may contribute to the lack of therapeutic efficacy in MIBC. Conclusions This study underscores the critical role of TLS formation in the response to neoadjuvant therapy in MIBC. We identified CCL19/CCL21 as key drivers of TLS formation within MIBC tumors and potential immune-sensitizing agents. Additionally, the intrinsic heterogeneity of tumor should be considered a significant factor influencing therapeutic efficacy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06358-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d2414e88dc899f0c1cdec00aa234fe0cea16d7" target='_blank'>
              Spatial relationships and interactions of immune cell niches are linked to the pathologic response of muscle-invasive bladder cancer to neoadjuvant therapy
              </a>
            </td>
          <td>
            Wasilijiang Wahafu, Quan Zhou, Xihua Yang, Yongming Yang, Yuanyuan Zhao, Zhu Wang, Xiangpeng Kang, Xiongjun Ye, N. Xing
          </td>
          <td>2025-03-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Alcohol-related liver disease (ALD) remains a major global health burden with limited therapeutic options due to an incomplete understanding of its underlying molecular mechanisms and cellular crosstalk. Here, we applied ultra-high resolution (on 2 µm spots) spatial transcriptomics to a cirrhotic liver tissue obtained from an end-stage ALD patient, analyzing >265,000 spatially resolved cells with further validation on single-cell and single-nuclei datasets from patients with ALD cirrhosis. Our analysis delineated distinct cellular sub-populations and molecular landscapes across fibrotic, vascular, and parenchymal niches of ALD cirrhosis. We identified robust zonation of hepatocytes, hepatic stellate cells, and diverse immune subpopulations, including enrichment of pro-inflammatory T cells and dendritic cells in the fibrotic niche and MARCO+ tissue-resident macrophages localizing mostly in parenchymal areas. Analysis of spatial metrics assigned expression of WNT4, RCAN3, PPIAL4G, PLA2G5, and SLC6A9 to the fibrotic environment in ALD. Differential expression and ligand–receptor interactome analyses revealed niche-specific signaling, with marked CCL19–CCR7 activity in fibrotic regions and DLL4–NOTCH3 crosstalk in vascular compartments. Notably, WNT4+ fibroblasts emerged as key mediators of extracellular matrix remodeling and chemoattraction, particularly via CCL19-mediated signaling towards CD8+ T cells, which was validated on single-cell resolution within the ALD cirrhotic liver in external datasets. These spatial and single-cell findings highlight novel potential therapeutic targets for patients with ALD cirrhosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f915c4e27932049e91c85de56bd98cf12dea6606" target='_blank'>
              Spatial single-cell interactome and niche-specific molecular signatures in alcohol-related liver disease
              </a>
            </td>
          <td>
            Oleksandr Petrenko, Thomas Sorz-Nechay, M. Bilban, S. Derdak, L. Van Melkebeke, Wenxin Huang, Jef Verbeek, Schalk Willem Van der Merwe, B. Hofer, B. Simbrunner, T. Reiberger, Tim Hendrikx
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462e90afb96bda692d274dd22748c2e27453a930" target='_blank'>
              Diving Deep: Profiling Exhausted T Cells in the Tumor Microenvironment Using Spectral Flow Cytometry.
              </a>
            </td>
          <td>
            Karen W Teng, Weng Hua Khoo, Nicholas Ching Wei Ho, S. J. Yang, Douglas C Wilson, Edmond Chua, Shu Wen Samantha Ho
          </td>
          <td>2025-04-03</td>
          <td>Cytometry. Part A : the journal of the International Society for Analytical Cytology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0444f2eee8d5a4470978d8e6c693faf1ee9e687e" target='_blank'>
              A Phenotype-Driven Multi-Omic Atlas of Glioblastoma Invasion
              </a>
            </td>
          <td>
            C. Krona, Anders Sundström, Emil Rosén, Soumi Kundu, H. Mangukiya, H. Babačić, Irem Uppman, Madeleine Skeppås, Ida Larsson, Ludmila Elfineh, Xiaofang Cao, Rashmi Ramachandra, Ramy Elgendy, Katrin Rut Halldorsdottir, Zankruti Dave, Milena Doroszko, K. Olausson, Johan Wikström, M. Pernemalm, Sven Nelander
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related deaths worldwide. Existing therapeutic options have limited efficacy, particularly for lung squamous cell carcinoma (LUSC), underscoring the critical need for the identification of new therapeutic targets. We previously demonstrated that the Transmembrane Serine Protease TMPRSS11B promotes transformation of human bronchial epithelial cells and enhances lactate export from LUSC cells. To determine the impact of TMPRSS11B activity on the host immune system and the tumor microenvironment (TME), we evaluated the effect of Tmprss11b depletion in a syngeneic mouse model. Tmprss11b depletion significantly reduced tumor burden in immunocompetent mice and triggered an infiltration of immune cells. RNA FISH analysis and spatial transcriptomics in the autochthonous Rosa26-Sox2-Ires-GfpLSL/LSL; Nkx2-1fl/fl; Lkb1fl/fl (SNL) model revealed an enrichment of Tmprss11b expression in LUSC tumors, specifically in Krt13+ hillock-like cells. Ultra-pH sensitive nanoparticle imaging and metabolite analysis identified regions of acidification, elevated lactate, and enrichment of M2-like macrophages in LUSC tumors. These results demonstrate that TMPRSS11B promotes an acidified and immunosuppressive TME and nominate this enzyme as a therapeutic target in LUSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/355ba538ee815c0328598c90b70069dd4fff9037" target='_blank'>
              Transmembrane Serine Protease TMPRSS11B promotes an acidified tumor microenvironment and immune suppression in lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Hari Shankar Sunil, Jean R. Clemenceau, Isabel Barnfather, S. R. Nakkireddy, Anthony Grichuk, Luke T. Izzo, Bret M. Evers, Lisa Thomas, I. Subramaniyan, Li Li, William T. Putnam, Jingfei Zhu, Barrett L. Updegraff, John D Minna, Ralph J DeBerardinis, Jinming Gao, Tae Hyun Hwang, T. Oliver, Kathryn A. O’Donnell
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1760a6c58270791a1de56692df43805b36e2d0" target='_blank'>
              Single Cell RNA Sequencing Analysis of Human Melanoma Reveals A Distinct Prognostic Myeloid Cell
              </a>
            </td>
          <td>
            M. U. Ahmad, Lyong Heo, Ramesh V. Nair, Saurabh Sharma, E. Maverakis, Amanda R. Kirane
          </td>
          <td>2025-03-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Rationale: Advances in cancer therapies have significantly improved patient survival; however, tumors enriched in cancer stem cells (CSCs) have poor treatment responses. CSCs are a key source of tumor heterogeneity, contributing to therapeutic resistance and unfavorable patient outcomes. In the tumor microenvironment (TME), cell-in-cell (CIC) structures, where one cell engulfs another, have been identified as markers of poor prognosis. Despite their clinical relevance, the mechanisms underlying CIC formation across different tumor cell subpopulations remain largely unknown. Elucidating these processes could provide novel insights and therapeutic opportunities to address aggressive, treatment-resistant cancers. Method: Fluorescent mCherry-carrying colorectal cancer stem cells (CRCSCs) were expanded as spheroids in serum-free media and cocultured with either parental cancer cell-expressing Venus fluorescent protein or CFSE dye-stained immune cells (T cells, M1/M2 macrophages, neutrophils, and NK cells) or treated with EGFR- or PD-L1-targeting antibodies to assess the formation of CIC structures. Genes potentially crucial for the formation of CIC structures were knocked down or overexpressed, and their effects on CIC formation were evaluated. The clinical relevance of the in vitro findings was confirmed through analysis of formalin-fixed, paraffin-embedded (FFPE) human colorectal cancer (CRC) specimens. Results: CRCSCs have a strong predilection for serving as the outer cell in a CIC structure and forming homotypic CIC structures predominantly with parental CRC cells. The frequency of CIC structure formation increased when the cells were exposed to anti-PD-L1 antibody treatment. Both the outer CRCSC in a CIC structure and CRCSCs released from a homotypic CIC structure showed enhanced resistance to the cytotoxicity of NK-92MI cells. Restoration of Stathmin1 (STMN1) expression but not RAC1 knockdown in CRCSCs reduced the homotypic CIC frequency, disrupted the outer cell fate in CIC structures, and increased cell susceptibility to NK-92MI cytotoxicity. In CRC patients, CIC structures are associated with poor tumor differentiation, negative STMN1 expression, and poor prognosis. Conclusion: CSCs play a crucial role in informing CIC structures in CRC. CIC structure formation partially depends on low STMN1 expression and confers a survival advantage under NK cytotoxicity. Targeting this pathway may significantly improve immunotherapy's efficacy for CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ab3f42feb06b52b9aecd81de765adfb3a490b01" target='_blank'>
              Colorectal cancer stem cells develop NK cell resistance via homotypic cell-in-cell structures suppressed by Stathmin1
              </a>
            </td>
          <td>
            Yen-Yu Lin, Hsin-Yi Lan, Hao-Wei Teng, Ya-Pei Wang, Wen-Chun Lin, Wei-Lun Hwang
          </td>
          <td>2025-03-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common malignant tumors of the digestive tract, with increasing incidence and mortality rates, posing a significant burden on human health. Its progression relies on various mechanisms, among which the tumor microenvironment and tumor-associated macrophages (TAMs) have garnered increasing attention. Macrophage infiltration in various solid tumors is associated with poor prognosis and is linked to chemotherapy resistance in many cancers. These significant biological behaviors depend on the heterogeneity of macrophages. Tumor-promoting TAMs comprise subpopulations characterized by distinct markers and unique transcriptional profiles, rendering them potential targets for anticancer therapies through either depletion or reprogramming from a pro-tumoral to an anti-tumoral state. Single-cell RNA sequencing technology has significantly enhanced our research resolution, breaking the traditional simplistic definitions of macrophage subtypes and deepening our understanding of the diversity within TAMs. However, a unified elucidation of the nomenclature and molecular characteristics associated with this diversity remains lacking. In this review, we assess the application of conventional macrophage polarization subtypes in colorectal malignancies and explore several unique subtypes defined from a single-cell omics perspective in recent years, categorizing them based on their potential functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff246fd6cb059fea5f5e9204ca830e7b6e029678" target='_blank'>
              Characterizing macrophage diversity in colorectal malignancies through single-cell genomics
              </a>
            </td>
          <td>
            Tingshuo Zhao, Yinyi Luo, Yuanjie Sun, Zhigang Wei
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e024d7323084416b4b6e84dca254aca71e425ab" target='_blank'>
              Learning the cellular origins of cancer using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Bruno Giotti, Wooseung Lee, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091e898b367848be40c66f493aec8ad58485d477" target='_blank'>
              From surfing to diving into the tumor microenvironment through multiparametric imaging mass cytometry
              </a>
            </td>
          <td>
            M. Erreni, M. R. Fumagalli, Matteo Marozzi, Roberto Leone, Raffaella Parente, Raffaella D’Anna, Andrea Doni
          </td>
          <td>2025-04-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Characterizing the variety of cell types in the tumor microenvironment (TME) and their organization into cellular communities is vital for elucidating the biological diversity of cancer and informing therapeutic strategies. Methods Here, we employed a machine learning-based algorithm framework, EcoTyper, to analyze single-cell transcriptomes from 139 patients with head and neck squamous cell carcinoma (HNSC)and gene expression profiles from 983 additional HNSC patients, aiming to delineate the fundamental cell states and ecosystems integral to HNSC. Results A diverse landscape of 66 cell states and 9 ecosystems within the HNSC microenvironment was identified, revealing classical cell types while also expanding upon previous immune classifications. Survival analysis revealed that specific cell states and ecotypes (ecosystems) are associated with patient prognosis, underscoring their potential as indicators of clinical outcomes. Moreover, distinct cell states and ecotypes exhibited varying responses to immunotherapy and chemotherapy, with several showing promise as predictive biomarkers for treatment efficacy. Conclusion Our large-scale integrative transcriptome analysis provides high-resolution insights into the cellular states and ecosystems of HNSC, facilitating the discovery of novel biomarkers and supporting the development of precision therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06299-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd5809c9e6b6aaac22c97a1616ff68cb7ac8d0d" target='_blank'>
              High-dimensional deconstruction of HNSC reveals clinically distinct cellular states and ecosystems that are associated with prognosis and therapy response
              </a>
            </td>
          <td>
            Lei Xiao, Zhe Shen, Zhaoyu Pan, Yuanzheng Qiu, Donghai Huang, Yong Liu, Chao Liu, Xin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy. Methods The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity. Results We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered. Conclusion This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="We developed a deep learning Pathomics image analysis workflow to generate spatial Tumor-TIL maps to visualize and quantify the abundance and spatial distribution of tumor infiltrating lymphocytes (TILs) in colon cancer. Colon cancer and lymphocyte detection in hematoxylin and eosin (H&E) stained whole slide images (WSIs) has revealed complex immuno-oncologic interactions that form TIL-rich and TIL-poor tumor habitats, which are unique in each patient sample. We compute Tumor%, total lymphocyte%, and TILs% as the proportion of the colon cancer microenvironment occupied by intratumoral lymphocytes for each WSI. Kaplan-Meier survival analyses and multivariate Cox regression were utilized to evaluate the prognostic significance of TILs% as a Pathomics biomarker. High TILs% was associated with improved overall survival (OS) and progression-free interval (PFI) in localized and metastatic colon cancer and other clinicopathologic variables, supporting the routine use of Pathomics Tumor-TIL mapping in biomedical research, clinical trials, laboratory medicine, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b4f8d94ce78fc5fa82e89d6b995817192c3e592" target='_blank'>
              Pathomics Image Analysis of Tumor Infiltrating Lymphocytes (TILs) in Colon Cancer
              </a>
            </td>
          <td>
            Yuwei Zhang, Shahira Abousamra, Mahmudul Hasan, Luke A. Torre-Healy, Spencer Krichevsky, Sampurna Shrestha, Erich Bremer, Derek A. Oldridge, Andrew J. Rech, E. E. Furth, Therese J. Bocklage, Justin S. Levens, Isaac Hands, Erich B. Durbin, D. Samaras, T. Kurç, J. Saltz, Rajarsi Gupta
          </td>
          <td>2025-04-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs), the most abundant stromal cells in the tumor microenvironment (TME), control tumor growth through production and organization of the extracellular matrix (ECM) for a long time. However, the results from different studies that have focused on targeting CAFs to disturb tumor progression are extremely controversial. Recent studies using advanced single-cell RNA sequencing technology (scRNAseq) combined with multiple genetically engineered mouse models have identified diverse CAF subpopulations in the premalignant liver microenvironment (PME) of hepatocellular carcinoma (HCC) and TME of intrahepatic cholangiocarcinoma (ICC), providing a deeper understanding of the exact roles of each CAF subpopulation in cancer development. This review focuses on the specific protein markers, signaling pathways, and functions of various emerging CAF subclusters that contribute to the development of ICC and HCC. Elucidating the role and regulation of CAF subpopulations under different pathophysiological conditions will facilitate the discovery of new therapeutics that modulate CAF activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d2d7a33282302100a7faa70ebce158c29857365" target='_blank'>
              Cancer-associated fibroblasts: multidimensional players in liver cancer
              </a>
            </td>
          <td>
            Yanyun Pan, Yuangang Qiu, Xinbin Zhou, Wei Mao, Xiao-ming Xu
          </td>
          <td>2025-04-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5782e67aa86bd64459296bc264a422b250cba97" target='_blank'>
              SCOIGET: Predicting Spatial Tumor Evolution Pattern by Inferring Spatial Copy Number Variation Distributions
              </a>
            </td>
          <td>
            Yujia Zhang, Yitao Yang, Yan Kong, Bingxu Zhong, Kenta Nakai, Hui Lu
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Understanding the intricate tumor microenvironment (TME) is crucial for elucidating the mechanisms underlying the progression of cervical squamous cell carcinoma (CSCC) and its response to anti-PD-1 therapy. Methods In this study, we characterized 50,649 cells obtained from the CSCC for single-cell RNA sequencing and integrated bulk sequencing data from The Cancer Genome Atlas (TCGA) and clinical samples to explore their cell composition, metabolic processes, signaling pathways, specific transcription factors, lineage tracking and response to immunotherapy. In vivo experiments were performed to validate the function of key cell subsets. Results We identified ten major cell type and 35 subsets of stromal and immune cells in TME and observed distinct patterns in the metabolic processes and signaling pathways of these cells between tumor and normal tissues. Furthermore, PCNA clamp-associated factor (PCLAF)+ tumor-associated epithelial cell (TAEpis) was negatively correlated with the number of C-X-C motif chemokine ligand 13 (CXCL13)+ CD8+ T cells, overall survival, and response to anti-programmed cell death-1(PD-1) therapy in patients with CSCC. Both in vivo and in vitro experiments demonstrated that PCLAF+ TAEpis promotes the apoptosis of CD8+ T and tumor growth, while also inhibiting T cell infiltration and function. Conclusion Our findings illuminate the heterogeneity of the complex TME in CSCC and offer evidence supporting PCLAF+ TAEpis as a promising therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03725-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be5ea8ecb1627ce7c5b0b610bd601d4a5d3e9545" target='_blank'>
              Overcoming myeloid-driven resistance to CAR T therapy by targeting SPP1
              </a>
            </td>
          <td>
            Sharareh Gholamin, H. Natri, Yuqi Zhao, Shengchao Xu, Maryam Aftabizadeh, Begonya Comin-Anduix, Supraja Saravanakumar, Christian Masia, Robyn A Wong, Lance Peter, Mei-i Chung, Evan D Mee, B. Aguilar, R. Starr, D. Torrejon, D. Alizadeh, Xiwei Wu, Anusha Kalbasi, Antoni Ribas, Stephan Forman, B. Badie, Nicholas E. Banovich, Christine E Brown
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background: Liver progenitor cells (LPCs) with bipotential differentiation capacities are essential for restoring liver homeostasis and hepatocyte population after damage. However, the low proportion and shared markers with epithelial cells make studying LPC heterogeneity difficult, especially in humans. To address this gap, we explored over 259,400 human liver single cells across 4 conditions (fetal, healthy, cirrhotic, and HCC-affected livers). Methods: Human liver tissue samples were analyzed using spatial transcriptomics sequencing technologies to describe the heterogeneity of LPCs. Liver tissue was characterized by LPC heterogeneity at single-cell resolution by employing cellular modules, differentiation trajectories, and gene co-expression patterns. Results: We annotated and identified 1 LPC cluster, 3 LPC subpopulations, and 4 distinct cellular modules, indicating the heterogeneity within LPC and the diversity between LPCs and epithelial cells. LPCs showed spatial colocalization with cholangiocytes and comprised a small proportion (2.95±1.91%) within the merged epithelial cells and LPC populations, exhibiting marked differences in marker expression patterns compared to those in mice. LPCs exhibited distinct cellular states in functional restoration, activation, proliferation, and cell transition. Additionally, the gene co-expression network of LPCs exhibited 3 unique modules, reflecting the distinct connectivity of genes encoding apolipoproteins and heat shock proteins in the gene co-expression network modules. Conclusions: Our study provides valuable insights into the multifaceted heterogeneity of human LPCs. Future studies focusing on spatial gene expression dynamics will contribute to our understanding of the spatial arrangement of liver regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363fd866c375a553c02fda34c7798688e167a041" target='_blank'>
              Integrative single-cell and spatial transcriptome analysis reveals heterogeneity of human liver progenitor cells
              </a>
            </td>
          <td>
            Chuanjun Liu, Kai Wang, Junpu Mei, Ruizhen Zhao, Juan Shen, Wei Zhang, Liangyu Li, Bhaskar Roy, Xiaodong Fang
          </td>
          <td>2025-02-26</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by an abundant tumor-associated stroma composed from pancreatic stellate cells, which play a critical role in tumor progression. Developing accurate in vitro models requires understanding the complex interactions between tumor cells and their microenvironment. In this study, we present a quantitative imaging-based characterization of the three dimensional (3D) self-organization of PDAC tumour spheroids using a microfluidic platform that mimics key aspects of the tumor microenvironment. Our model incorporates collagen type I hydrogels to recreate the extracellular matrix, activated human pancreatic stellate cells (HPSCs), and various tumor cell types. Advanced imaging techniques, including Lattice Lightsheet Microscopy, allowed us to analyze the 3D growth and spatial organization of the spheroids, revealing intricate biomechanical interactions. Our results indicate that alterations in matrix properties—such as stiffness, pore size, and hydraulic permeability—due to variations in collagen concentration significantly influence the growth patterns and organization of PDAC spheroids, depending on tumor subtype and epithelial–mesenchymal phenotype. Higher collagen concentrations promoted larger spheroids in epithelial-like cell lines, while mesenchymal-type cells required increased collagen for self-organization into smaller spheroids. Furthermore, coculture with HPSCs affected spheroid formation distinctly based on each PDAC cell line's genetic and phenotypic traits. HPSCs had opposing effects on epithelial-like cell lines: one cell line exhibited enhanced spheroid growth, while another showed inhibited formation, whereas mesenchymal-like spheroids showed minimal impact. These results provide insights into tumor–stroma interactions, emphasizing the importance of the cell-specific and matrix-dependent factors for advancing our understanding of PDAC progression and informing future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f50c1adc7fc861815263b7b03dd1f3c6e040380" target='_blank'>
              Quantitative characterization of the 3D self-organization of PDAC tumor spheroids reveals cell type and matrix dependence through advanced microscopy analysis
              </a>
            </td>
          <td>
            Soraya Hernández-Hatibi, C. Borau, Neus Martínez-Bosch, Pilar Navarro, J. García-Aznar, P. E. Guerrero
          </td>
          <td>2025-03-01</td>
          <td>APL Bioengineering</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Spatial omics enable the characterization of colocalized cell communities that coordinate specific functions within tissues. These communities, or niches, are shaped by interactions between neighboring cells, yet existing computational methods rarely leverage such interactions for their identification and characterization. To address this gap, here we introduce NicheCompass, a graph deep-learning method that models cellular communication to learn interpretable cell embeddings that encode signaling events, enabling the identification of niches and their underlying processes. Unlike existing methods, NicheCompass quantitatively characterizes niches based on communication pathways and consistently outperforms alternatives. We show its versatility by mapping tissue architecture during mouse embryonic development and delineating tumor niches in human cancers, including a spatial reference mapping application. Finally, we extend its capabilities to spatial multi-omics, demonstrate cross-technology integration with datasets from different sequencing platforms and construct a whole mouse brain spatial atlas comprising 8.4 million cells, highlighting NicheCompass’ scalability. Overall, NicheCompass provides a scalable framework for identifying and analyzing niches through signaling events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14cda68a9739dfb6fdc7acbc975a0b67601ec1ae" target='_blank'>
              Quantitative characterization of cell niches in spatially resolved omics data
              </a>
            </td>
          <td>
            Sebastian Birk, Irene Bonafonte-Pardàs, Adib Miraki Feriz, Adam Boxall, E. Agirre, Fani Memi, A. Maguza, Anamika Yadav, Erick Armingol, Rong Fan, Gonçalo Castelo-Branco, F. Theis, O. Bayraktar, Carlos Talavera-López, M. Lotfollahi
          </td>
          <td>2025-03-18</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="The role of tumor microenvironment in invasive breast cancer prognosis and treatment is highly appreciated. With the advent of immunotherapy, immunophenotypic characterization in primary tumors is gaining attention as it can improve patient stratification. Here, we discuss the benefits of spatial analysis employing double and multiplex immunostaining, allowing the simultaneous detection of more than one protein on the same tissue section, which in turn helps us provide functional insight into infiltrating immune cells within tumors. We focus on studies demonstrating the prognostic and predictive impact of distinct tumor-infiltrating lymphocyte subpopulations including different CD8(+) T subsets as well as CD4(+) T cells and tumor-associated macrophages in invasive breast carcinoma. The clinical value of immune cell topography is also appreciated. We further refer to how the integration of digital pathology and artificial intelligence in routine practice could enhance the accuracy of multiplex immunostainings evaluation within the tumor microenvironment, maximizing our perception of host immune response, improving in turn decision-making towards more precise immune-associated therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8b3640c44bd80cadd9a05e0404e0638d900b1d2" target='_blank'>
              Double–Multiplex Immunostainings for Immune Profiling of Invasive Breast Carcinoma: Emerging Novel Immune-Based Biomarkers
              </a>
            </td>
          <td>
            Sofia D P Theodorou, Konstantinos Ntostoglou, I. Nikas, Dimitrios Goutas, V. Georgoulias, Christos Kittas, Ioannis S. Pateras
          </td>
          <td>2025-03-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Background Cancer development is closely associated with complex alterations in the tumor microenvironment (TME). Among these, immune cells within the TME play a huge role in personalized tumor diagnosis and treatment. Objectives This review aims to summarize the diversity of immune cells in the TME, their impact on patient prognosis and treatment response, and the contributions of single‐cell RNA sequencing (scRNA‐seq) in understanding their functional heterogeneity. Methods We analyzed recent studies utilizing scRNA‐seq to investigate immune cell populations in the TME, focusing on their interactions and regulatory mechanisms. Results ScRNA‐seq reveals the functional heterogeneity of immune cells, enhances our understanding of their role in tumor antibody responses, and facilitates the construction of immune cell interaction networks. These insights provide guidance for the development of cancer immunotherapies and personalized treatment approaches. Conclusion Applying scRNA‐seq to immune cell analysis in the TME offers a novel pathway for personalized cancer treatment. Despite its promise, several challenges remain, highlighting the need for further advancements to fully integrate scRNA‐seq into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36e80d2a964412d958098f4ad446c2f9779f2b7e" target='_blank'>
              Analyze the Diversity and Function of Immune Cells in the Tumor Microenvironment From the Perspective of Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Lujuan Ma, Yu Luan, Lin Lu
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc383607243031294dc9e768566a894cfedd572c" target='_blank'>
              Glioblastoma stem cell morphotypes convey distinct cell states and clinically relevant functions
              </a>
            </td>
          <td>
            C. Barelli, Matteo Bonfanti, Filippo Mirabella, Dario Ricca, Negin Alizadehmohajer, Viktoria Sokolova, Alberto Campione, G. Sicuri, C. Peano, Stefania Faletti, Roberto Stefini, N. Kalebic
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumor stroma consists mainly of extracellular matrix, fibroblasts, immune cells, and vasculature. Its structure and functions are altered during malignancy: tumor cells transform fibroblasts into cancer-associated fibroblasts, which exhibit immunosuppressive activities on which growth and metastasis depend. These include exclusion of immune cells from the tumor nest, cancer progression, and inhibition of T-cell-based immunotherapy. To understand these complex interactions, we measure the density of different cell types in the stroma using immunohistochemistry techniques on tumor samples from lung cancer patients. We incorporate these data into a minimal dynamical system, explore the variety of outcomes, and finally establish a spatio-temporal model that explains the cell distribution. We reproduce that cancer-associated fibroblasts act as a barrier to tumor expansion, but also reduce the efficiency of the immune response. Our conclusion is that the final outcome depends on the parameter values for each patient and leads to either tumor invasion, persistence, or eradication as a result of the interplay between cancer cell growth, T-cell cytotoxicity, and fibroblast activity. However, despite the existence of a wide range of scenarios, distinct trajectories, and patterns allow quantitative predictions that may help in the selection of new therapies and personalized protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300ea7f39d018c28251113d7be1e7ef9c1ae4343" target='_blank'>
              Mechanistic insight for T-cell exclusion by cancer-associated fibroblasts in human lung cancer
              </a>
            </td>
          <td>
            Joseph Ackermann, Chiara Bernard, P. Sirven, Hélène Salmon, Massimiliano Fraldi, Martine Ben Amar
          </td>
          <td>2025-04-10</td>
          <td>eLife</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Gliomas, including both glioblastoma multiforme (GBM) and lower-grade glioma (LGG), present a substantial challenge in neuro-oncology because of genetic heterogeneity and unsatisfactory prognosis. This study aimed to conduct a comprehensive multi-omics analysis of gliomas using various bioinformatics approaches to identify potential therapeutic targets and prognostic markers. A comprehensive analysis was conducted on 1327 sequencing data samples alongside their relevant clinical information sourced from The Cancer Genome Atlas (TCGA) pertaining to glioblastoma (GBM), low-grade glioma (LGG), the Chinese Glioma Genome Atlas (CCGA) and University of California Santa Cruz Xena (UCSC Xena) datasets. These tools were employed for gene expression profiling, survival analysis, and cell communication mapping. Spatial transcriptomics revealed the localization of mesenchymal (MES)-like malignant tumors, and drug sensitivity analysis was performed to evaluate responses to quinpirole and meropenem. Additionally, the Tumor Immune Dysfunction and Exclusion (TIDE) framework was utilized to gauge the responsiveness to immunotherapy. The MES-like malignant and monocyte/macrophage (mono/macro) cell subsets showed high hallmark scores, playing key roles in the tumor microenvironment. MES-like malignant marker gene scores correlated with overall survival across datasets, whereas mono/macro marker gene scores were significant in the TCGA-LGG and CCGA datasets. Key interactions between these cell types were found, especially with CD14-ITGB2, LGALS1-CD69, and APOE-TREM2. The mono/macro cell subset demonstrated better immune therapy responsiveness, as indicated by lower TIDE scores. Spatial transcriptomics revealed that MES-like malignant tumors are predominantly localized in four distinct regions, with the marker genes CHI3L1 and ADM confirming these locations. Drug sensitivity analysis revealed differential responses of the MES-like malignant cell subset to quinpirole and meropenem. Our results offer fresh perspectives on the differential roles of MES-like malignant and monocyte/macrophage cell subsets in tumor progression and immune modulation, providing novel insights into glioma biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ece6067255a1933f97765246f767fae55de1c3a5" target='_blank'>
              A spatial transcriptomics study of MES-like and mono/macro cells in gliomas
              </a>
            </td>
          <td>
            Boyan Zhao, Jianing Wu, Tiehui Zhang, Mingyang Han, Cheng Zhang, Xuan Rong, Ruotian Zhang, Xin Chen, Fei Peng, Jin Jin, Shiya Liu, Xingli Dong, Shiguang Zhao
          </td>
          <td>2025-04-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Intratumor transcriptional heterogeneity (ITTH) presents a major challenge in cancer treatment, particularly due to limited understanding of the diverse malignant cell populations and their relationship to therapy resistance. While single-cell sequencing has provided valuable insights into tumor composition, its high cost and technical complexity limits its use for large-scale tumor screening. In contrast, several databases collecting bulk RNA sequencing (RNA-seq) data from multiple samples across various cancer types are available and could be used to profile ITTH. Several deconvolution approaches have been developed to infer cellular composition from such data. However, most of these methods rely on predefined markers or single cell reference datasets, limiting the performance of such methods by the quality of used reference data. Although unsupervised approaches do not face such limitations, existing methods have not been specifically adapted to characterize malignant cell states, and focus instead on general cell types. To address these gaps, we introduce CDState, an unsupervised method for inferring malignant cell subpopulations from bulk RNA-seq data. CDState utilizes a Nonnegative Matrix Factorization model improved with sum-to-one constraints and a cosine similarity-based optimization to deconvolve bulk gene expression into distinct cell state-specific profiles, and estimate the abundance of each state across tumor samples. We validate CDState using bulkified single-cell RNA-seq data from five cancer types, showing that it outperforms existing unsupervised deconvolution methods in both cell state proportions and gene expression estimation. Applying CDState to 33 cancer types from TCGA, we identified recurrent malignant cell programs, with epithelial-mesenchymal transition as the main driver of tumor transcriptional heterogeneity. We further link the identified malignant states to patient clinical features, revealing states associated with worse patient prognosis. We show that ITTH is linked with patient survival and clinical features, as well as is associated with varied responses to therapy. Finally, we identified potential genetic drivers of ITTH and malignant cell states, including TP53, KRAS, and PIK3CA, known oncogenic genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f16cea30bc9f4619225da6166ac6508ace6b994e" target='_blank'>
              CDState: an unsupervised approach to predict malignant cell heterogeneity in tumor bulk RNA-sequencing data
              </a>
            </td>
          <td>
            Agnieszka Kraft, Josephine Yates, Florian Barkmann, Valentina Boeva
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1221249520ea9d5b52e0d651f4686ac33466a610" target='_blank'>
              Staining-free Visualization of Cell Boundaries in Sliced Tissues for Accurate Binning of Gene Expression in Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhiming Zhang, Xiaoxia Chen, Youyuan Chen, Zixuan Liao, T. Liu
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64eb12bd7d9a78fd223bedc136a7a4d7a78d40" target='_blank'>
              Abstract B068: A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer
              </a>
            </td>
          <td>
            M. Oh, J. Abascal, L. Tran, R. Salehi-Rad, R. Lim, C. Dumitras, Cara E. Yean, Sunny Xiao, Bitta Kahangi, William P Crosson, E. P. Reyes, Nalani Coleman, Austin K. Rennels, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer (PC) is the most frequently diagnosed malignancy among men and contributes significantly to cancer-related mortality. While recent advances in in vitro PC modeling systems have been made, there remains a lack of robust preclinical models that faithfully recapitulate the genetic and phenotypic characteristics across various PC subtypes—from localized PC (LPC) to castration-resistant PC (CRPC)—along with associated stromal cells. Here, we established human PC assembloids from LPC and CRPC tissues by reconstituting tumor organoids with corresponding cancer-associated fibroblasts (CAFs), thereby incorporating aspects of the tumor microenvironment (TME). Established PC organoids exhibited high concordance in genomic landscape with parental tumors, and the tumor assembloids showed a higher degree of phenotypic similarity to parental tumors compared to tumor organoids without CAFs. PC assembloids displayed increased proliferation and reduced sensitivity to anti-cancer treatments, indicating that PC assembloids are potent tools for understanding PC biology, investigating the interaction between tumor and CAFs, and identifying personalized therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/105fe3ccf7c35aabcf3c81f5242c91ac5e7680b1" target='_blank'>
              Generation of prostate cancer assembloids modeling the patient-specific tumor microenvironment
              </a>
            </td>
          <td>
            Juhee Lee, Yunhee Kim, Cheol Lee, S. Jeon, Hae Seo, Jongwon Lee, Jungmin Choi, Minyong Kang, Eunjee Kim, Kunyoo Shin
          </td>
          <td>2025-03-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Gastric cancer is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME). Cancer-associated fibroblasts (CAFs), which are a key component of the TME, exhibit significant heterogeneity and play crucial roles in tumor progression. Therefore, a comprehensive understanding of CAFs is essential for developing novel therapeutic strategies for gastric cancer. Methods This study investigates the characteristics and functional information of CAF subtypes and explores the intercellular communication between CAFs and malignant epithelial cells (ECs) in gastric cancer by analyzing single-cell sequencing data from 24 gastric cancer samples. CellChat was employed to map intercellular communication, and Seurat was used to integrate single-cell sequencing data with spatial transcriptome data to reconstruct a comprehensive single-cell spatial map. The spatial relationship between apCAFs and cancer cells was analyzed using multicolor immunohistochemistry. Results Cells were categorized into nine distinct categories, revealing a positive correlation between the proportions of epithelial cells (ECs) and fibroblasts. Furthermore, six fibroblast subpopulations were identified: inflammatory (iCAFs), pericytes, matrix (mCAFs), antigen-presenting (apCAFs), smooth muscle cells (SMCs), and proliferative CAFs (pCAFs). Each of these subpopulations was linked to various biological processes and immune responses. Malignant ECs exhibited heightened intercellular communication, particularly with CAF subpopulations, through specific ligand-receptor interactions. High-density regions of CAF subpopulations displayed spatial exclusivity, with pericytes serving as a source for iCAFs, mCAFs, and apCAFs. Notably, malignant ECs and apCAFs showed increased interactions, with certain ligand-receptor pairs potentially impacting the prognosis of gastric cancer. Multiplex immunohistochemistry (mIHC) confirmed the close spatial proximity of apCAFs to cancer cells in gastric cancer. Conclusion Our study provided a comprehensive characterization of CAF heterogeneity in gastric cancer and revealed the intricate intercellular networks within the TME. The identified CAF subpopulations and their interactions with malignant cells could serve as potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06376-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Purpose The aim of this study is to explore the pathological niche of cancer metastasis and the site-specific interactions between tumor cells and the microenvironment, understand the mechanisms driving metastasis progression and identify potential therapeutic targets. Methods Data from four lung cancer metastasis datasets (GSE123902, GSE131907, GSE148071, and GSE186344) were downloaded and subjected to stringent quality control and filtering. Cell types were identified using canonical markers, and pseudotime trajectory analysis was performed to evaluate cell differentiation. Functional and pathway enrichment analyses, including ssGSEA and GO/KEGG, were conducted. CellphoneDB was used to analyze intercellular communication, ranking receptor-ligand interactions based on communication strength. Results Eleven cell types were identified after quality control, revealing significant heterogeneity and site-specific functionality in lung cancer metastases. CTLs showed notable activity in antigen presentation and T-cell differentiation pathways, with DNAJB1⁺ CTLs playing a dominant role in cytotoxicity and immune regulation. B cells, myeloid cells, and CAFs were involved in immune modulation, defense, and matrix remodeling through specific signaling pathways. Tumor cell subclusters drove proliferation, migration, and immune evasion via immune-regulatory, Hippo, and TGF-beta pathways. No overlapping pathways were observed across metastatic sites. Cell communication analysis identified PPIA-BSG and APP-CD74 as key axes in brain and lymph node metastases, while FN1-Integrin and CTLA4-CD86 dominated in bone and adrenal metastases, respectively. Conclusions In summary, this study highlights the functional heterogeneity and site-specific interactions of cells in lung cancer metastases, providing insights into the mechanisms shaping metastatic niches and potential therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02269-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c4f954c571841166277c9ca7ec19c26848339cb" target='_blank'>
              Single-cell profiling uncovers the intricate pathological niche diversity in brain, lymph node, bone, and adrenal metastases of lung cancer
              </a>
            </td>
          <td>
            Le Liu, Yuan Zhou, Zhenjun Ye, Zhiyong Chen, Benchao Yuan, Liyi Guo, Haiyan Zhang, Yuanyuan Xu
          </td>
          <td>2025-04-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dbf2dbcf6c4b1d8a1dbdd8e9dcfc190f14aeaa6" target='_blank'>
              Methylation-Based Deconvolution Unveils Glioblastoma Heterogeneity and Cell-Type Composition Linked to Patient Survival
              </a>
            </td>
          <td>
            Aviel Iluz, Nir Lavi, Hanna Charbit, Mijal Gutreiman, Masha Idelson, Debora Steiner, Etti Ben-Shushan, A. Zick, Amir Eden, A. Mordechai, Moscovici Samuel, Y. Fellig, Alexander Lossos, Joshua Moss, Benjamin E. Reubinoff, I. Lavon
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683616b1a667ef7717c017121e0ce36b95c50974" target='_blank'>
              Spatial multi-omics profiling of breast cancer oligo-recurrent lung metastasis.
              </a>
            </td>
          <td>
            Yang Gao, Bin Li, Yuzhi Jin, Jinlin Cheng, Weihong Tian, Lixiong Ying, Libing Hong, Shan Xin, Bo Lin, Chuan Liu, Xuqi Sun, Jun Zhang, Haibo Zhang, Jindong Xie, X. Deng, Xiaomeng Dai, Lulu Liu, Yi Zheng, Peng Zhao, Guangchuan Yu, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-04-15</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Approximately 15-20% of head and neck cancer squamous cell carcinoma (HNSCC) patients respond favorably to immune checkpoint blockade (ICB). Previous single-cell RNA-Seq (scRNA-Seq) studies identified immune features, including macrophage subset ratios and T-cell subtypes, in HNSCC ICB response. However, the spatial features of HNSCC-infiltrated immune cells in response to ICB treatment need to be better characterized. Methods Here, we perform a systematic evaluation of cell interactions between immune cell types within the tumor microenvironment using spatial omics data using complementary techniques from both 10X Visium spot-based spatial transcriptomics and Nanostring CosMx single-cell spatial omics with RNA gene panel including 435 ligands and receptors. In this study, we used integrated bioinformatics analyses to identify cellular neighborhoods of co-localizing cell types in single-cell spatial transcriptomics and proteomics data. In addition, we used both publicly available scRNA-Seq and in-house spatial RNA-Seq data to identify spatially constrained Ligand-Receptor interactions in Responder patients. Results With 522,399 single cells profiled with both RNA and protein from 26 patients, in addition to spot-resolved spatial RNA-Seq from 8 patients treated with ICB together with bioinformatics analysis of publicly available single-cell and bulk RNA-Seq, we have identified a spatial and cell-type specific context-dependency of myeloid and T cell interaction difference between Responders and Non-Responders. We defined further cellular neighborhood and the sources of chemokine CXCL9/10-CXCR3 interactions in Responders, emerging targets in ICB, as well as CXCL16-CXCR6, CCL4/5-CCR5, and other underappreciated and potential markers and targets for ICB response in HNSCC. In addition, we have contributed a rich data resource of cell-cell Ligand Receptor interactions for the immunotherapy and HNSCC research community. Discussion Our work provides a comprehensive single-cell and spatial atlas of immune cell interactions that correlate with response to ICB in HNSCC. We showcase how integrating multiple technologies and bioinformatics approaches can provide new insights into potential immune-based biomarkers of ICB response. Our results suggested refining future studies using preclinical animal models in a more context-specific manner to elucidate potential underlying mechanisms that lead to improved ICB responses. What is already known on this topic Most cancer patients still do not experience clinical benefits from immune checkpoint blockade (ICB), necessitating the development of response biomarkers and new immunotherapeutic targets. What this study adds Here, we use integrated high-dimensional omics and bioinformatics approaches to identify immune cell-cell interaction markers associated with ICB response in patients with Head and neck squamous cell carcinoma. How this study might affect research, practice or policy We identified spatial and cell-type specificity of Ligand-Receptor interactions between myeloid and T cells in ICB Responder patients that may help inform further mechanistic studies and biomarker development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ece16324064b70206dfb8fd687bfc5db3089e8d7" target='_blank'>
              Integrated single-cell and spatial analysis identifies context-dependent myeloid-T cell interactions in head and neck cancer immune checkpoint blockade response
              </a>
            </td>
          <td>
            Athena E. Golfinos-Owens, Taja Ložar, Parth Khatri, Rong Hu, Paul M. Harari, Paul F. Lambert, Megan B Fitzpatrick, Huy Q. Dinh
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58e073c2b3967f3720af1b89697ce0e1aad33c4c" target='_blank'>
              Abstract B086: Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment
              </a>
            </td>
          <td>
            Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Bioengineering-based in vitro tumor models are increasingly important as tools for studying disease progression and therapy response for many cancers, including the deadly pancreatic ductal adenocarcinoma (PDAC) that exhibits a tumor/tissue microenvironment of high cellular/biochemical complexity. Therefore, it is crucial for in vitro models to capture that complexity and to enable investigation of the interplay between cancer cells and factors such as extracellular matrix (ECM) proteins or stroma cells. Using polyurethane (PU) scaffolds, we performed a systematic study on how different ECM protein scaffold coatings impact the long-term cell evolution in scaffolds containing only cancer or only stroma cells (activated stellate and endothelial cells). To investigate potential further changes in those biomarkers due to cancer–stroma interactions, we mapped their expression in dual/zonal scaffolds consisting of a cancer core and a stroma periphery, spatially mimicking the fibrotic/desmoplastic reaction in PDAC. In our single scaffolds, we observed that the protein coating affected the cancer cell spatial aggregation, matrix deposition, and biomarker upregulation in a cell-line-dependent manner. In single stroma scaffolds, different levels of fibrosis/desmoplasia in terms of ECM composition/quantity were generated depending on the ECM coating. When studying the evolution of cancer and stroma cells in our dual/zonal model, biomarkers linked to cell aggressiveness/invasiveness were further upregulated by both cancer and stroma cells as compared to single scaffold models. Collectively, our study advances the understanding of how different ECM proteins impact the long-term cell evolution in PU scaffolds. Our findings show that within our bioengineered models, we can stimulate the cells of the PDAC microenvironment to develop different levels of aggressiveness/invasiveness, as well as different levels of fibrosis. Furthermore, we highlight the importance of considering spatial complexity to map cell invasion. Our work contributes to the design of in vitro models with variable, yet biomimetic, tissue-like properties for studying the tumor microenvironment’s role in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fdf99a8b8b29a15ccc5b5a3ee040a4b74974238" target='_blank'>
              Mapping Tumor–Stroma–ECM Interactions in Spatially Advanced 3D Models of Pancreatic Cancer
              </a>
            </td>
          <td>
            Anna-Dimitra Kataki, Priyanka G Gupta, Umber Cheema, A. Nisbet, Yaohe Wang, Hemant M Kocher, P. A. Perez-Mancera, E. Velliou
          </td>
          <td>2025-03-07</td>
          <td>ACS Applied Materials & Interfaces</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) are pivotal components of the breast cancer (BC) tumor microenvironment (TME), significantly influencing tumor progression and response to therapy. However, the heterogeneity and specific roles of TAM subpopulations in BC remain inadequately understood. Methods We performed an integrated analysis of single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data from BC patients to comprehensively characterize TAM heterogeneity. Utilizing the MetaTiME computational framework and consensus clustering, we identified distinct TAM subtypes and assessed their associations with clinical outcomes and treatment responses. A machine learning-based predictive model was developed to evaluate the prognostic significance of TAM-related gene expression profiles. Results Our analysis revealed three distinct TAM subgroups. Notably, we identified a novel macrophage subtype, M_Macrophage-SPP1-C1Q, characterized by high expression of SPP1 and C1QA, representing an intermediate differentiation state with unique proliferative and oncogenic properties. High infiltration of M_Macrophage-SPP1-C1Q was significantly associated with poor overall survival (OS) and chemotherapy resistance in BC patients. We developed a Random Forest (RF)-based predictive model, Macro.RF, which accurately stratified patients based on survival outcomes and chemotherapy responses, independent of established prognostic parameters. Conclusion This study uncovers a previously unrecognized TAM subtype that drives poor prognosis in BC. The identification of M_Macrophage-SPP1-C1Q enhances our understanding of TAM heterogeneity within the TME and offers a novel prognostic biomarker. The Macro.RF model provides a robust tool for predicting clinical outcomes and guiding personalized treatment strategies in BC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03750-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/287d3599b1ab22b44316058fde7d65a5ef96c17d" target='_blank'>
              Integrated single-cell and bulk transcriptomic analysis identifies a novel macrophage subtype associated with poor prognosis in breast cancer
              </a>
            </td>
          <td>
            Qing Wang, Yushuai Yu, Liqiong Ruan, Mingyao Huang, Wei Chen, Xiaomei Sun, Jun Liu, Zirong Jiang
          </td>
          <td>2025-03-27</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac4b19bdc51a77041e08cca430b7dd7ca4fef49" target='_blank'>
              Abstract PR007: Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity
              </a>
            </td>
          <td>
            D. Peeper, Sofía Ibáñez-Molero, Johanna Veldman, Juan Simon-Nieto, J. Traets, Austin George, K. Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, W. V. van Houdt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce737a0ad997a56048c24df55247b712e440df3d" target='_blank'>
              Abstract PR002: T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment
              </a>
            </td>
          <td>
            Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick S Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa Mcginnis, Kevin B Walsh, Weilan Ye
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73bf7de11a69b6fed18f060c330341fde4d20ec0" target='_blank'>
              Cellular characterization of the peritumoral brain zone of glioblastomas: heterogenous nature and clinical relevance
              </a>
            </td>
          <td>
            Olaya de Dios, J. Romero, M. A. Ramírez-González, B. Herránz, Alicia Avis, A. Ramos, J. Sepúlveda-Sánchez, R. Gargini, B. Meléndez, Pedro González, L. Jimenez-Roldan, Guillermo García-Posadas, A. Hernández-Laín, Á. Pérez-Núñez, P. Sánchez-Gómez
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The spatial interactions between malignant and immune cells in the tumor microenvironment (TME) play a crucial role in cancer biology and treatment response. Understanding these interactions is critical for predicting prognosis and assessing immunotherapy effectiveness. Conventional methods, which focus on local spatial features, often struggle to achieve robust analysis due to the complex and heterogeneous cellular distributions. We propose a Topological Data Analysis (TDA)-based framework using both global and local spatial features between malignant and immune cells. For the global aspects, we introduce Topological Malignant Clusters (TopMC), a method utilizing persistence diagrams in TDA to capture the global spatial shapes of malignant cell distributions. It quantifies tumor-immune cell infiltration at a global scale by computing distances from the boundaries of the TopMC. Local interactions are evaluated through the density of malignant cells. Using high-resolution multiplex immunofluorescence (mIF) imaging in Diffuse Large B Cell Lymphoma, we integrate distance and density measures into a distance-density space to analyze patterns of malignant-immune cell interactions in TME. This study shows the robustness of the proposed approach to variations in cell distribution, enabling consistent analysis irrespective of whether images are acquired from malignant-enriched or border regions of the tumors. Furthermore, we elucidate the correlation between spatial patterns of immune phenotypes and patient survival probability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f549327f511c89a0763403d23ea2ac5417a3579" target='_blank'>
              Spatial Analysis of Malignant-Immune Cell Interactions in the Tumor Microenvironment Using Topological Data Analysis
              </a>
            </td>
          <td>
            Seol Ah Park, Yongsoo Kim, Paweł Dłotko, Davide Gurnari, Jooyoung Hahn
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73dbf54f5b2a0ce96fdfefae2ba812ee9c7ea68b" target='_blank'>
              The 1000+ mouse project: large-scale spatiotemporal parametrization and modeling of preclinical cancer immunotherapies
              </a>
            </td>
          <td>
            Adam L. Kenet, S. Achar, Alka Dwivedi, John Buckley, M. Pouzolles, Haiying Qin, C. Chien, N. Taylor, Grégoire Altan-Bonnet
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a32f7726bad7de5bd6ab6b676111c1867233e6" target='_blank'>
              Precancerous Cells Initiate Glioblastoma Evolution and Contribute to Intratumoral Heterogeneity.
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, Jong Hee Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, J. H. Lee
          </td>
          <td>2025-04-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92ab05366d53bbf0f6bb5cf31fdb12098dce09df" target='_blank'>
              To EMT or not to EMT: Ablation of mesenchymal tumor cell lineages reveals the essential role of EMT in pancreatic cancer initiation and evolution.
              </a>
            </td>
          <td>
            Jessica Peura, Calvin Johnson, Jason R. Pitarresi
          </td>
          <td>2025-04-08</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d17402678fd41fc00172c331bb8211f40e5fcbf7" target='_blank'>
              Temporal mapping of the anti-tumor effects of nanobody-based MSLN.CAR-T cell therapy in metastatic solid tumors
              </a>
            </td>
          <td>
            Chaido Stathopoulou, Mingming Zhao, Qun Jiang, Jessica Hong, Jing Bian, Jingli Zhang, M. Ho, R. Hassan
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e047830d6d15c84fdba5a27411c84ea5a08197" target='_blank'>
              Multi-Modal Disentanglement of Spatial Transcriptomics and Histopathology Imaging
              </a>
            </td>
          <td>
            Hassaan Maan, Zongliang Ji, Elliot Sicheri, Tiak Ju Tan, Alina Selega, Ricardo Gonzalez, Rahul G Krishnan, Bo Wang, Kieran R. Campbell
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f30684782f70f49100b5a4c8e6aa8a01fed878d3" target='_blank'>
              Single-cell dissection of the genotype-immunophenotype relationship in glioblastoma.
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, Dolores Hambardzumyan, A. Tsankov
          </td>
          <td>2025-04-11</td>
          <td>Brain : a journal of neurology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Although two-thirds of cancers arise from loss-of-function mutations in tumor suppressor genes, there are few approved targeted therapies linked to these alterations. Synthetic lethality offers a promising strategy to treat such cancers by targeting vulnerabilities unique to cancer cells with these mutations. To identify clinically relevant synthetic lethal interactions, we analyzed genome-wide CRISPR/Cas9 knock-out (KO) viability screens from the Cancer Dependency Map and evaluated their clinical relevance in patient tumors through mutual exclusivity, a pattern indicative of synthetic lethality. Indeed, we found significant enrichment of mutual exclusivity for interactions involving cancer driver genes compared to non-driver mutations. To identify therapeutic opportunities, we integrated drug sensitivity data to identify inhibitors that mimic the effects of CRISPR-mediated KO. This approach revealed potential drug repurposing opportunities, including BRD2 inhibitors for bladder cancers with ARID1A mutations and SIN3A-mutated cell lines showing sensitivity to nicotinamide phosphoribosyltransferase (NAMPT) inhibitors. However, we discovered that pharmacological inhibitors often fail to phenocopy KO of matched drug targets, with only a small fraction of drugs inducing similar effects. This discrepancy reveals fundamental differences between pharmacological and genetic perturbations, emphasizing the need for approaches that directly assess the interplay of loss-of-function mutations and drug activity in cancer models. Author Summary Synthetic lethality is an emerging approach for targeting a biological dependency in cancer cells that does not harm normal cells. This strategy is particularly valuable for targeting loss-of-function mutations in tumor suppressor genes, which are more challenging to directly target. In an effort to accelerate treatments for cancer patients, we aimed to map out these dependencies and overlap them with responses to available drugs. We discovered different outcomes when a protein is targeted by a drug versus when that same target is disrupted genetically. Thus, if a drug is to be effectively repurposed as synthetic lethal agent, feasibility studies must capture drug biology, ideally by test the drug empirically in relevant cancer models. A second notable discovery is that in vitro synthetic lethal interactions involving cancer driver genes are significantly more likely to exhibit consistent patterns, such as mutual exclusivity in human tumor samples. This is important since selection of relevant cell lines is often critical in drug development to maximize potential for translation to clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c1469ac01d0657e51e2a370ba73f2735664c7f" target='_blank'>
              Challenges and opportunities for drug repurposing in cancers based on synthetic lethality induced by tumor suppressor gene mutations
              </a>
            </td>
          <td>
            Michael Vermeulen, Andrew W. Craig, Tomas Babak
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec81235338a787d945989660c2ddefa3be4c0bd" target='_blank'>
              Novel CHI3L1-Associated Angiogenic Phenotypes Define Glioma Microenvironments: Insights From Multi-Omics Integration.
              </a>
            </td>
          <td>
            Yu-Hang Zhao, Yu-Xiang Cai, Zhi-yong Pan, F. Tang, Chao Ma, Ze-Fen Wang, Gang Li, Hang Chang, Su-Fang Tian, Zhi-Qiang Li
          </td>
          <td>2025-02-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Primary pulmonary lymphoepithelioma-like carcinoma (PPLELC) is a rare subtype of non-small-cell lung cancer. Duo to the current lack of precise targeted therapies, there is an urgent need to identify novel therapeutic targets. In this study, we perform single-nucleus transcriptome analysis on PPLELC samples to reveal the molecular tumor heterogeneity and characterize the functional states of immune cells within the tumor microenvironment. We identify a critical malignant subpopulation of PPLELC characterized by elevated expression of AKT3 and FGFR2. Higher expression levels of AKT3 and FGFR2 are associated with poorer patient outcomes. Moreover, treatment with either an AKT3 inhibitor or an FGFR2 inhibitor significantly attenuates tumor progression in patient-derived xenograft models. Our findings highlight AKT3 and FGFR2 as potential therapeutic targets and prognostic biomarkers, providing valuable insights for the development of rational targeted therapies and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a403bcbf00350f8809a4da6d588cc795243cc7f5" target='_blank'>
              Single-cell analysis reveals transcriptomic features and therapeutic targets in primary pulmonary lymphoepithelioma-like carcinoma
              </a>
            </td>
          <td>
            Binghua Tan, Ke Xu, Y. Lyu, Yicheng Liang, Ruihao Liang, Kai Lei, Jialu Liang, Jing Huang, Kefeng Wang, Duoguang Wu, Wenjian Wang, Xueting Hu, Kexi Wang, Minghui Wang, Huayue Lin
          </td>
          <td>2025-03-08</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a54281a4bcdae103d2590f451749c7ef505da2c" target='_blank'>
              Identification and proteomic profiling of CD90+ small EVs using a refined immunocapture separation approach targeting stromal-derived EV subpopulations in synovial fluid of arthritis patients
              </a>
            </td>
          <td>
            Stefanie Kurth, E. Hanser, Simone Häner-Massimi, Stavros Giaglis, Florian Geier, Dominik Burri, Katarzyna Buczak, U. Heider, Stefan Wild, André N. Tiaden, Diego Kyburz
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64291da16ef528fb561e1af0f4b9ad8a8ddde042" target='_blank'>
              Comprehensive Integrated Analysis Reveals the Spatiotemporal Microevolution of Cancer Cells in Patients with Bone-Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Yinghua Feng, Xiuli Zhang, Guangpeng Wang, Fei-ya Yang, Ruifang Li, Lu Yin, Dong Chen, Wenkuan Wang, Mingshuai Wang, Zhiyuan Hu, Yuan Sh, N. Xing
          </td>
          <td>2025-04-09</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract In solid tumours, malignant cells develop relationships with nonmalignant stromal cells to support tumour growth, tissue structure, and nutrient supply. In a recent issue of this journal, Zheng and Guo catalogue the cellular landscape in chordoma using a combination of single‐cell and spatial transcriptomics. Despite the mesenchymal nature of chordoma, malignant cells retain expression of epithelial markers, in addition to mesenchymal, partial‐EMT, and stem‐cell signatures. Cancer‐associated fibroblasts (CAFs) are among the most abundant stromal cells and the authors define an inflammatory CAF subtype (iCAF), which is associated with poor outcome and tumour invasion. It is proposed that iCAFs arise from normal fibroblasts during malignant tumour progression and play a causative role in driving an invasive poor prognosis tumour phenotype. Recent reports by this and other groups have separately catalogued cell populations, including CAFs and immune cells in chordoma. The next challenge will be to integrate findings from these distinct studies to allow a consensus to be reached regarding cellular heterogeneity within chordoma, and to allow comparison of CAF populations with those found in other tumour types. Comparison of CAF functions in these predominantly mesenchymal tumours with epithelial solid tumours may reveal interesting lessons about the diverse phenotypes CAFs can bring to distinct tumour ecosystems. Understanding the role of CAFs in chordoma progression may also lead to therapeutic opportunities, but separation of correlation and causation in CAF regulation of tumour phenotypes remains a significant challenge. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c222d4cf7dba2a9f6cb176f5eccff6fd31fd816" target='_blank'>
              Cancer‐associated fibroblast heterogeneity in chordoma†
              </a>
            </td>
          <td>
            Jack C Henry, A. Cameron
          </td>
          <td>2025-03-15</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNAseq) of human tissues has expanded our understanding of the complexity of cellular subsets and their changes in disease. The availability of scRNAseq data in different tissues and disease states provides an opportunity to compare cellular subsets and identify common and unique cellular activation. In this study, we aimed to characterize shared and tissue-specific myeloid and stromal phenotypes and uncover key cellular subtypes involved in pathogenic tissue activation. We analyzed scRNAseq data from 14 public datasets, comprising heart, lung, skin, and synovium in both healthy and diseased states. Our analysis identified distinct and overlapping myeloid and stromal cell populations in these tissues. Despite significant inter-individual variability, we were able to identify both shared and disease-specific changes in these cell populations. These findings provide insights into the conserved and tissue-specific roles of myeloid and stromal cells in health and disease and contribute to a better understanding of tissue pathology and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a4d83ee093d91b7fc9e6f8d4d131194bc8ae33" target='_blank'>
              Human Fibroblast-Myeloid cell tissue atlas across lung, synovium, skin and heart
              </a>
            </td>
          <td>
            L. MacDonald, Olympia M Hardy, Melpomeni Toitou, Gabriela Kania, P. Błyszczuk, R. Micheroli, Thomas D. Otto, Mariola Kurowska-Stolarska, C. Ospelt
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Adoptive cell therapies (ACT) leverage tumor-immune interactions to cure cancer. Despite promising phase I/II clinical trials of chimeric-antigen-receptor natural killer (CAR-NK) cell therapies, molecular mechanisms and cellular properties required to achieve clinical benefits in broad cancer spectra remain underexplored. While in vitro and in vivo experiments are required in this endeavor, they are typically expensive, laborious, and limited to targeted investigations. Here, we present ABMACT (Agent-Based Model for Adoptive Cell Therapy), an in silico approach employing agent-based models (ABM) to simulate the continuous course and dynamics of an evolving tumor-immune ecosystem, consisting of heterogeneous “virtual cells” created based on knowledge and omics data observed in experiments and patients. Applying ABMACT in multiple therapeutic context indicates that to achieve optimal ACT efficacy, it is key to enhance immune cellular proliferation, cytotoxicity, and serial killing capacity. With ABMACT, in silico trials can be performed systematically to inform ACT product development and predict optimal treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a7a9fa08cfb61e58044a46d5cd717b6f33a5eb" target='_blank'>
              Agent-based modeling of cellular dynamics in adoptive cell therapy
              </a>
            </td>
          <td>
            Yujia Wang, Stefano Casarin, May Daher, V. Mohanty, Merve Dede, M. Shanley, R. Başar, Katayoun Rezvani, Ken Chen
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0761133026ec3ba173f2c421844c6435e13b010" target='_blank'>
              Real-Time Tracking of Tumor Invasion Dynamics in Organotypic Brain Slices at Subcellular Resolution
              </a>
            </td>
          <td>
            Jana B. Petr, Meng-Syuan Lin, Dina Hochuli, Andreas Hierlemann, Martin Baumgartner, Mario M. Modena
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3418175c8e687d05cd4cb3259c7f39ab58b56c25" target='_blank'>
              Dynamic responses to rejection in the transplanted human heart revealed through spatial transcriptomics
              </a>
            </td>
          <td>
            K. Amancherla, Angela M. Taravella Oill, Xavier Bledsoe, A. L. Williams, Nelson Chow, Shilin Zhao, Quanhu Sheng, David W. Bearl, Robert D. Hoffman, J. Menachem, Hasan K. Siddiqi, Douglas M. Brinkley, E. D. Mee, Niran Hadad, Vineet Agrawal, Jeffrey Schmeckpepper, A. Rali, Stacy Tsai, E. Farber-Eger, Quinn S Wells, Jane E. Freedman, Nathan R. Tucker, K. Schlendorf, Eric R. Gamazon, Ravi V. Shah, N. Banovich
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Immune cells exhibit functional heterogeneity beyond what is resolved by classical definitions of subpopulations based on cell surface expression of receptors. To develop efficient and personalized cell-based immunotherapies, we need to resolve this heterogeneity and understand the underlying parameters that dictate cellular responses to specific target cells. For this, new methods are required that can identify and harvest immune cells with specific functions, e.g., high cytotoxic potential, to form clonally expanded cells or to assess molecular or genetic signatures. In this study, we evaluate a system for non-destructive, live cell picking and release in deep, high-aspect ratio microwells and test it for isolation of individual natural killer (NK) cells. We assess its performance at retrieving and releasing beads from microwells and demonstrate its potential for single NK cell isolation with intact viability. We also implement a semi-automated workflow for functional single-cell screening of NK cell behavior in microwell arrays followed by single-cell identification and isolation, demonstrating the potential for functional screening and isolation of serial killing immune cells. Our evaluation concludes that this cell isolation system, in combination with microwell arrays, offers opportunities for improved understanding of NK cell biology with applications towards cell therapy. However, its limited throughput hinders large-scale applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba79619eb3501acae9caddc99dc83ce718f2ee3c" target='_blank'>
              Isolation of individual natural killer cells from deep, high-aspect ratio microwell arrays - an evaluative study
              </a>
            </td>
          <td>
            Quentin Verron, Niklas Sandström, Hanna van Ooijen, K. Guldevall, K. Olofsson, T. Frisk, Björn Önfelt
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Model organisms allowing for longitudinal examinations of liver disease pathogenesis are pivotal for the development of new therapeutic modalities. The eastern North American woodchuck develops chronic hepatitis and liver cancer after woodchuck hepatitis virus (WHV) infection, mirroring aspects of the natural history of the human hepatitis B virus (HBV). However, the cellular landscape of the woodchuck liver and the cell-level relevance of WHV infection to HBV infection is currently uncharacterized. Methods We employed single-cell RNA sequencing (RNA-seq) to generate an atlas of healthy woodchuck liver (63,389 cells, n=8) and peripheral blood mononuclear cells (PBMCs) (26,972 cells, n=7). Cell-specific and hepatic zonation gene signatures were validated using spatial transcriptomics (n=1). We employed our atlas to examine immune activation in stimulated precision cut-liver slices (PCLS) and disease-related pathway activation in chronic WHV infection (11,797 cells, n=3). We further employed our atlas to examine shared disease pathways between WHV infection and human HBV infection. Results Our atlas revealed woodchuck hepatic cellular diversity comparable to human and murine livers. Applying single-nucleus RNA-seq to PMA/ionomycin-stimulated precision cut liver slices revealed inflammation-associated activation signatures in T cell, myeloid and endothelial cell compartment. Finally, we describe intrahepatic T cells in chronic WHV hepatitis with both exhaustion and activation-associated signatures that resemble intrahepatic T cell genes signatures described in human chronic HBV. Conclusions We present a multi-omic atlas of healthy, diseased and ex vivo stimulated woodchuck liver. By identifying shared pathological processes between WHV and HBV infections, our findings reinforce the value of this preclinical model in translational research. This resource aims to advance studies on HBV pathogenesis and oncogenesis to speed the development of novel therapeutic strategies. Impact and Implications/Lay summary The liver plays important roles in metabolism, detoxification, and immune processes; liver transplantation is often the only treatment option for severe chronic liver diseases. Therefore, developing animal models that reflect human liver disease and can be studied throughout the disease course is crucial for the discovery of new treatment options. The woodchuck is an animal that develops chronic hepatitis and liver cancer after infection with woodchuck hepatitis virus, which models the human hepatitis type B virus infection (HBV) and associated hepatic carcinoma. However, our understanding of the cells that compose the woodchuck liver is limited, making it challenging to design and test cell-based therapeutics. In this study, we atlased the healthy and chronically infected woodchuck liver, found that liver cell types in woodchuck resemble those in humans, and employed the atlas to show similarities between WHV and HBV at the cell level, reinforcing the potential of WHV-infected woodchuck as a model for human HBV disease. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8ef9bc878aedeb55d6c6cc4171d96693eccf30b" target='_blank'>
              A Single-Cell Woodchuck Liver Atlas Identifies Healthy and Disease-related Cellular Programs Conserved in Human
              </a>
            </td>
          <td>
            Zoe A. Clarke, Jawairia Atif, Xinle Wang, Lewis Y. Liu, Lawrence Wood, Damra Camat, Sai W. Chung, Xue-Zhong Ma, Justin Manuel, Si Lok, Timothy Lau, C. Thoeni, Tomasz I. Michalak, Ian D. McGilvray, Gary D. Bader, S. Macparland
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/520e7a98e4f661979d2c0ecb98d246afd9f267aa" target='_blank'>
              Translating genetics into tissue: inflammatory cytokine-producing TAMs and PD-L1 tumor expression as poor prognosis factors in cutaneous melanoma
              </a>
            </td>
          <td>
            Celia Barrio-Alonso, Alicia Nieto-Valle, Lucía Barandalla-Revilla, J. Avilés-Izquierdo, V. Parra-Blanco, Paloma Sánchez-Mateos, Rafael Samaniego
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background The methylation of N6-methyladenosine (m6A) RNA plays a crucial role in the genetic regulation of various cancers. While m6A modifications have been extensively studied in the tumor microenvironment (TME) of several malignancies, their role in cutaneous melanoma (CM) remains unexplored. Methods Using Non-negative matrix factorization (NMF) analysis on single-cell RNA-seq data (GSE215121) from three CM samples obtained from public databases, 26 m6A RNA methylation regulators were utilized to determine TME subclusters, their expression, and function. Results Six distinct TME cell types were identified and NMF clustering further revealed unique m6A-based subpopulations of cancer-associated fibroblasts and T cells. The prognostic model demonstrated strong predictive capabilities, particularly for fibroblast and T cell m6A clusters, and highlighted COL3A1 as a critical regulator of melanoma-fibroblast interactions. Conclusion Highlighting the COL3A1 gene as a critical link and potential therapeutic target in melanoma could offer new avenues for targeted therapies and improve prognostic assessments in cutaneous melanoma. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06176-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e20d76052c3fcce67fbc8337ada951bc0b9da27" target='_blank'>
              Revealing tumor microenvironment communication through m6A single-cell analysis and elucidating immunotherapeutic potentials for cutaneous melanoma (CM)
              </a>
            </td>
          <td>
            Lun Liu, Maxwell A Kishengere, Xueming Xu, Zhanghui Yue
          </td>
          <td>2025-04-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Different patterns of immune and stromal interactions within the tumor microenvironment significantly impact tumor behavior and response to treatment. Recent research has identified distinct immune subtypes—immune desert, excluded, and inflamed—each associated with specific mechanisms that impede anti-tumor immune responses. In the era of precision medicine, accurately stratifying patients into these subtypes is essential for optimizing cancer management and personalizing immunotherapy. The research team at National Cancer Institute in Lithuania addresses critical gaps in this field by focusing on several key areas. Our first focus is on developing reliable and accessible tools for determining patient immune subtypes. We are integrating transcriptomics and histology data to create a robust subtyping pipeline, which we are validating with real-world data and streamlining for clinical use. Another aspect of our work involves creating translatable preclinical models for cancer immunotherapy through utilizing 3D cell cultures and biocompatible scaffolds. We are developing druggable syngeneic murine tumor models that accurately reflect the three immune subtypes, enhancing the relevance of preclinical studies. Using both pharmaceutical and non-pharmaceutical approaches, we are tailoring immunomodulatory interventions to address the unique characteristics of each immune subtype. Joint efforts in these research areas could generate translatable insights, contributing to more precise cancer therapies and overcoming resistance mechanisms. 
  
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8e8d206eced9c976c9ce13db22dd5e4aadd38" target='_blank'>
              Deciphering immune tumor subtypes through microenvironment profiling
              </a>
            </td>
          <td>
            A. Mlynska, Eglė Žymantaitė, Austeja Butkute, N. Dobrovolskienė, J. Krasko, Olha Karaman, Karolina Suveizdė, Beatrice Gudaite, Margarita Žvirblė, Emilija Paberalė, Nijole Matuseviciene, V. Pašukonienė
          </td>
          <td>2025-03-25</td>
          <td>South East European Journal of Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Advances in tissue labeling, imaging, and automated cell identification now enable the visualization of immune cell types in human tumors. However, a framework for analyzing spatial patterns within the tumor microenvironment (TME) is still lacking. To address this, we develop Spatiopath, a null-hypothesis framework that distinguishes statistically significant immune cell associations from random distributions. Using embedding functions to map cell contours and tumor regions, Spatiopath extends Ripley’s K function to analyze both cell-cell and cell-tumor interactions. We validate the method with synthetic simulations and apply it to multi-color images of lung tumor sections, revealing significant spatial patterns such as mast cells accumulating near T cells and the tumor epithelium. These patterns highlight differences in spatial organization, with mast cells clustering near the epithelium and T cells positioned farther away. Spatiopath enables a better understanding of immune responses and may help identify biomarkers for patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80949e1640c0e5682964239d082757c3241b697f" target='_blank'>
              Statistical analysis of spatial patterns in tumor microenvironment images
              </a>
            </td>
          <td>
            Mohamed M Benimam, V. Meas-Yedid, Suvadip Mukherjee, Astri Frafjord, Alexandre Corthay, T. Lagache, Jean-Christophe Olivo-Marin
          </td>
          <td>2025-03-31</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1fbf6a111cb790d6d05a256a03567072e5318c" target='_blank'>
              AI-Driven Variant Annotation for Precision Oncology in Breast Cancer
              </a>
            </td>
          <td>
            Kriti Shukla, Yue Wang, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36c2c77f99d55aa9b36b1f371ad8b8cec2aac50" target='_blank'>
              Evolutionary fingerprints of epithelial-to-mesenchymal transition.
              </a>
            </td>
          <td>
            Luigi Perelli, Li Zhang, Sarah Mangiameli, Francesca Giannese, Krishnan K. Mahadevan, F. Peng, F. Citron, Hania Khan, Courtney Le, Enrico Gurreri, F. Carbone, Andrew J. C. Russell, M. Soeung, T. N. Lam, Sebastian Lundgren, Sujay Marisetty, Cihui Zhu, Desiree Catania, Alaa M T Mohamed, Ningping Feng, J. Augustine, A. Sgambato, G. Tortora, Giulio F. Draetta, Giovanni Tonon, Andrew Futreal, Virginia Giuliani, A. Carugo, A. Viale, Michael P. Kim, T. Heffernan, Linghua Wang, Raghu Kalluri, D. Cittaro, Fei Chen, G. Genovese
          </td>
          <td>2025-03-05</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/596ee5bf00c8a656038f1a6564aba0e8dbb21dea" target='_blank'>
              Longitudinal single-cell multiomic atlas of high-risk neuroblastoma reveals chemotherapy-induced tumor microenvironment rewiring.
              </a>
            </td>
          <td>
            Wenbao Yu, Rumeysa Biyik‐Sit, Yasin Uzun, Chia-Hui Chen, Anusha Thadi, Jonathan H Sussman, Minxing Pang, Chi-Yun Wu, Liron D. Grossmann, Peng Gao, David W Wu, Aliza Yousey, Mei Zhang, Christina S. Turn, Zhan Zhang, S. Bandyopadhyay, Jeffrey Huang, Tasleema Patel, Changya Chen, Daniel Martinez, Lea F Surrey, M. Hogarty, K. Bernt, Nancy R. Zhang, John M. Maris, Kai Tan
          </td>
          <td>2025-04-14</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Objective Active surveillance (AS) offers a viable alternative to surgical intervention for the management of indolent papillary thyroid carcinoma (PTC), helping to minimize the incidence of unnecessary treatment. However, the broader adoption of AS is hindered by the need for more reliable diagnostic markers. This study aimed to identify the differences between indolent and progressive PTC and find new targets for biomarker development and therapeutic strategies. Methods We used single-cell RNA sequencing (scRNA-seq) to analyze cellular differences in 10 early-stage PTC tumors. Findings were validated in an additional 25 tumors using cell co-culture, migration assays, immunofluorescence staining, flow cytometry, and analysis of data from The Cancer Genome Atlas (TCGA). Results Tumor-infiltrating B cells (TIL-B), particularly germinal center B cells (GC-B), were more abundant in indolent PTC. These cells suppressed thyroid cell proliferation in both indolent and progressive cases, though indolent PTC had a higher capacity to recruit peripheral B cells. In indolent cases, TIL-B cells showed increased proliferation and formed clusters within tertiary lymphoid structures (TLS). PTPRC-CD22 interactions were identified as potential drivers of TIL-B cell proliferation. Markers linked to GC-B cells, such as LMO2, were highlighted as potential diagnostic and prognostic indicators for indolent PTC. Conclusion This study provides insights into the cellular landscape of early-stage PTC, revealing distinct tumor and immune microenvironment features in indolent and progressive cases. These findings advance the understanding of indolent PTC biology and support the development of reliable diagnostic and prognostic biomarkers. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03341-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c059a59d61bc9cea405825a6d5d7499c3788b8e" target='_blank'>
              Single-cell transcriptomics analysis reveals that the tumor-infiltrating B cells determine the indolent fate of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Chunmei Li, Pei Wang, Zhizhong Dong, Weihan Cao, Yan-Jun Su, Jian-ming Zhang, Shuyan Zhao, Zhiyuan Wang, Zi Lei, Li Shi, Ruochuan Cheng, Wen Liu
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20c3f321acba286bf84a8f4af78044117acbf8c" target='_blank'>
              Abstract B105: Predicting spatial transcriptomics from histopathology with Path2Space: AI charts breast cancer patients tumor immune microenvironment on an unprecedented scale
              </a>
            </td>
          <td>
            Roshan Lodha, E. Shulman, Emma M. Campagnolo, Thomas Cantore, Tom Hu, M. Nasrallah, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tumor cells with organ-specific metastasis traits arise in primary lesions with substantial variations of local niche mechanics owing to intratumoral heterogeneity. However, the roles of mechanically heterogeneous primary tumor microenvironment in metastatic organotropism remain an enigma. This study reports that persistent priming in soft but not stiff niches that mimic primary tumor mechanical heterogeneity induces transcriptional reprogramming reminiscent of neuron and promotes the acquisition of brain metastatic potential. Soft-primed cells generate brain metastases in vivo through enhanced transendothelial migration across blood-brain barrier and brain colonization, which is further supported by the findings that tumor cells residing in local soft niches of primary xenografts exhibit brain metastatic tropism. Mechanistically, soft niches suppress cytoskeleton-nucleus–mediated mechanotransduction, which promotes histone deacetylase 3 activity. Inhibiting histone deacetylase 3 abolishes niche softness-induced brain metastatic ability. Collectively, this study uncovers a previously unappreciated role of local niche softness within primary tumors in brain metastasis, highlighting the significance of primary tumor mechanical heterogeneity in metastatic organotropism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c4555c3459d7f13d0f05bcb5e93723f35a50f25" target='_blank'>
              Local soft niches in mechanically heterogeneous primary tumors promote brain metastasis via mechanotransduction-mediated HDAC3 activity
              </a>
            </td>
          <td>
            Kai Tang, Yufan Zheng, Guanshuo Hu, Ying Xin, Keming Li, Cunyu Zhang, Xi Chen, Bai Zhang, Xueyi Li, Bing Hu, Qiong Jia, Yong-ping Zheng, Mo Yang, Youhua Tan
          </td>
          <td>2025-02-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Mammalian organs and tissues are composed of heterogeneously distributed cells, which interact with each other and the extracellular matrix surrounding them in a spatially defined way. Therefore, spatially resolved gene expression profiling is crucial for determining the function and phenotypes of these cells. While genome mutations and transcriptome alterations act as drivers of diseases, the proteins that they encode regulate essentially all biological functions and constitute the majority of biomarkers and drug targets for disease diagnostics and treatment. However, unlike transcriptomics, which has a recent explosion in high-throughput spatial technologies with deep coverage, spatial proteomics capable of reaching bulk tissue-level coverage is still rare in the field, due to the non-amplifiable nature of proteins and sensitivity limitation of mass spectrometry (MS). More importantly, due to the limited multiplexing capability of the current proteomics methods, whole-tissue slice mapping with high spatial resolution requires a formidable amount of MS matching time. To achieve spatially resolved, deeply covered proteome mapping for centimeter-sized samples, we developed a sparse sampling strategy for spatial proteomics (S4P) using computationally assisted image reconstruction methods, which is potentially capable of reducing the number of samples by tens to thousands of times depending on the spatial resolution. In this way, we generated the largest spatial proteome to date, mapping more than 9000 proteins in the mouse brain, and discovered potential new regional or cell type markers. Considering its advantage in sensitivity and throughput, we expect that the S4P strategy will be applicable to a wide range of tissues in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/635e78858dc942db5f55ba0ae972708b0c927e55" target='_blank'>
              In-depth and high-throughput spatial proteomics for whole-tissue slice profiling by deep learning-facilitated sparse sampling strategy
              </a>
            </td>
          <td>
            Ritian Qin, Jiacheng Ma, Fuchu He, Weijie Qin
          </td>
          <td>2025-03-11</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f6840421639f0eeeffb398c6fad844f8d444a6" target='_blank'>
              InSituPy – A framework for histology-guided, multi-sample analysis of single-cell spatial transcriptomics data
              </a>
            </td>
          <td>
            Johannes Wirth, Anna Chernysheva, Birthe Lemke, Isabel Giray, Aitana Egea Lavandera, K. Steiger
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background The identification of molecular features characterizing metastatic disease is a critical area of oncology research, as metastatic foci often exhibit distinct biological behaviors compared to primary tumors. While the focus has largely been on the neoplastic cells themselves, the characterization of the associated stroma remains largely underexplored, with significant implications for understanding metastasis. Main body By employing spatially resolved transcriptomics, we analyzed the transcriptional features of primary breast adenocarcinoma and its associated metastatic foci, on a representative set of microregions. We identified a stromal metastatic (Met) signature, which was subsequently validated across transcriptomic reference human breast cancer (BC) datasets and in spatial transcriptomics of a murine model. Conclusion We discuss the potential of a stromal Met signature to pinpoint metastatic breast cancer, serving as a prognostic tool that can provide a foundation for the exploration of tumor-extrinsic molecular hallmarks of BC metastatic foci. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03353-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabdeb7c0a8725d1b43bd88dd9faad675928ee91" target='_blank'>
              Extraction of a stromal metastatic gene signature in breast cancer via spatial profiling
              </a>
            </td>
          <td>
            G. Bertolazzi, Valeria Cancila, Davide Vacca, Beatrice Belmonte, D. Lecis, Parsa Sirati Moghaddam, A. Di Napoli, M. P. Colombo, G. Pruneri, Giannino Del Sal, Giorgio Scita, Matteo Fassan, A. Vecchione, Silvio Bicciato, Claudio Tripodo
          </td>
          <td>2025-03-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Tumor suppressor genes (TSGs) are thought to suppress tumor development primarily via cancer cell-autonomous mechanisms. However, the tumor microenvironment (TME) also significantly influences tumorigenesis. In this context, a role for TSGs in the various cell types of the TME in regulating tumor growth is emerging. Indeed, expression analyses of TSGs in clinical samples, along with data from mouse models in which TSGs were deleted selectively in the TME, indicate a functional role for them in tumor development. In this Perspective, using TP53 and PTEN as examples, we posit that TSGs play a significant role in cells of the TME in regulating tumor development, and postulate both a ‘pro-active’ and ‘reactive’ model for their contribution to tumor growth, dependent on the temporal sequence of initiating events. Finally, we discuss the need to consider a 2-in-1 cancer-treatment strategy to improve the efficacy of clearance of cancer cells and the cancer-promoting TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f832937592839082aed0daa5b731f3a252c7c2" target='_blank'>
              Tumor suppressor genes in the tumor microenvironment
              </a>
            </td>
          <td>
            Bahareh Tabanifar, H. Lau, K. Sabapathy
          </td>
          <td>2025-03-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background As a type of autophagy, aggrephagy degrades the aggregation of misfolded protein in cells and plays an important role in key genetic events for various cancers. However, aggrephagy functions within the tumor microenvironment (TME) in endometrial cancer (EC) remain to be elucidated. Methods A total of 36,227 single cells from single-cell RNA-seq data derived from five EC tumor samples were comprehensively analyzed using a nonnegative matrix factorization (NMF) algorithm for 44 aggrephagy-related genes. Bulk RNA-seq cohorts from public repositories were utilized to assess the prognostic value of aggrephagy-related TME clusters and predict immune checkpoint blockade immunotherapeutic response in EC. Results Fibroblasts, macrophages, CD8+T cells, and lymphatic endothelial cells were categorized into two to five aggrephagy-related subclusters, respectively. CellChat analysis showed that the aggrephagy-related subtypes of TME cells exhibited extensive interactions with tumor epithelial cells, particularly for macrophages. Moreover, aggrephagy regulators may be significantly associated with the pseudotime trajectories of major TME cell types as well as the clinical and biological features of EC. Bulk-seq analysis showed that these aggrephagy-related subclusters had significant predictive value for the survival and immune checkpoint blockade response in EC patients. Notably, immunohistochemical staining results manifested that the TUBA1A+ macrophage subtype was linked to less lymph node metastasis and longer survival, which were consistent with the bioinformatics analysis findings. Conclusions This study provided a novel view of aggrephagy signaling in the EC tumor microenvironment, and intervention of aggrephagy was expected to improve the survival rate of EC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539a7161f9d5e59af0e8571e13d64e98e255b165" target='_blank'>
              Single-cell sequencing reveals the role of aggrephagy-related patterns in tumor microenvironment, prognosis and immunotherapy in endometrial cancer
              </a>
            </td>
          <td>
            Yuquan Yuan, Chunyan Ren, Jin Shu, Keyang Zhu, Ganghui Li, Bao Liu, Jianrong Huang, Yinde Huang, Chengzhi Zhao
          </td>
          <td>2025-03-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc8b763d2e3ec3323e5145b28e5d2c4495cc407" target='_blank'>
              Abstract B040: Transcriptomic meta-analysis identifies robust, tissue-agnostic gene expression signature of immune checkpoint blockade response
              </a>
            </td>
          <td>
            Y. Guo, Aaron C Tan, Tanmay Kulshrestha, M. Chang, Dawn Lau, Lan Ying Wang, T. Tay, Irfahan Kassam, Egor Revkov, Simone Rizzetto, I. Tan, Daniel S W Tan, A. Skanderup
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Programmed cell deaths (PCDs) are crucial for tumor progression. By analyzing 18 PCDs, we generated a robust multigene signature, Combined Cell Death Index (CCDI), comprising necroptosis and autophagy genes for non-small cell lung cancer (NSCLC). The CCDI accurately stratified patients by survival prognosis and predicted immunotherapy responses. We validated CCDI and prioritized CCDI genes using five single-cell RNA sequencing and two spatial transcriptomics datasets. CCDI positively correlates with tumor malignancy, invasiveness, and immunotherapy resistance. Four necroptosis genes (PTGES3, MYO6, CCT6A, and CTSH) may affect cancer cell evolution. In vitro, CTSH overexpression or PTGES3 knockdown inhibited NSCLC cell proliferation and migration while inducing necroptosis with necrosome formation. Moreover, we observed diminished CTSH, heightened PTGES3, and low necroptosis activity in 12 pairs of NSCLC tumors and normal tissues. CTSH overexpression or PTGES3 knockdown induced necroptosis and improved anti-PD1 therapy efficiency in syngeneic cancer mouse models. These findings indicate necroptosis genes as potential therapeutic targets in cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b557366271169d5f50c23b71e8320a685b6fc6f3" target='_blank'>
              Integrating bulk, single-cell, and spatial transcriptomics to identify and functionally validate novel targets to enhance immunotherapy in NSCLC
              </a>
            </td>
          <td>
            Kui Cao, Shenshui Wei, Tianjiao Ma, Xinxin Yang, Yuning Wang, Xiangrong He, Mengdi Lu, Yuwen Bai, Cuicui Qi, Luquan Zhang, Lijuan Li, Hongxue Meng, Jianqun Ma, Jinhong Zhu
          </td>
          <td>2025-04-16</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Motivation Immune cells undergo cytokine-driven polarization in response to diverse stimuli, altering their transcriptional profiles and functional states. This dynamic process is central to immune responses in health and diseases, yet a systematic approach to assess cytokine-driven polarization in single-cell RNA sequencing data has been lacking. Results To address this gap, we developed single-cell unified polarization assessment (Scupa), the first computational method for comprehensive immune cell polarization assessment. Scupa leverages data from the Immune Dictionary, which characterizes cytokine-driven polarization states across 14 immune cell types. By integrating cell embeddings from the single-cell foundation model Universal Cell Embeddings, Scupa effectively identifies polarized cells across different species and experimental conditions. Applications of Scupa in independent datasets demonstrated its accuracy in classifying polarized cells and further revealed distinct polarization profiles in tumor-infiltrating myeloid cells across cancers. Scupa complements conventional single-cell data analysis by providing new insights into dynamic immune cell states, and holds potential for advancing therapeutic insights, particularly in cytokine-based therapies. Availability and implementation The code is available at https://github.com/bsml320/Scupa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/078b81c851163489c04007fcd2f4d464e0013dc1" target='_blank'>
              Scupa: single-cell unified polarization assessment of immune cells using the single-cell foundation model
              </a>
            </td>
          <td>
            Wendao Liu, Zhongming Zhao
          </td>
          <td>2025-02-25</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef4c57391a4d19cdfe4a449b6362839163ca6fe0" target='_blank'>
              Microfluidic Replication and Phenotypic Profiling of Extracellular Vesicles from the Tumor Microenvironment Using Dual-Switch Aptamer Logic Gates.
              </a>
            </td>
          <td>
            Qiaoyi Wu, Qiuyue Wu, Haoting Lin, Chi Zhang, Zhenlong You, S.‐I. Kang, Yuanfeng Xu, Xiaofeng Chen, Chao Yang, Yanling Song, Lin Zhu
          </td>
          <td>2025-02-26</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract T-Cell-derived extracellular vesicles (TcEVs) play key roles in immune regulation and tumor microenvironment modulation. However, the heterogeneity of TcEV remains poorly understood due to technical limitations of EV analysis and the lack of comprehensive data. To address this, we constructed TcEVdb, a comprehensive database that explores the expression and cluster of TcEV by the SEVtras method from T-cell single-cell RNA sequencing data. TcEVdb contains 277 265 EV droplets from 51 T-cell types across 221 samples from 21 projects, covering 9 tissue sources and 23 disease conditions. The database provides two main functional modules. The Browse module enables users to investigate EV secretion activity indices across samples, visualize TcEV clusters, analyze differentially expressed genes (DEGs) and pathway enrichment in TcEV subpopulations, and compare TcEV transcriptomes with their cellular origins. The Search module allows users to query specific genes across all datasets and visualize their expression distribution. Furthermore, our analysis of TcEV in diffuse large B-cell lymphoma revealed increased EV secretion in CD4+ T exhausted cells compared to healthy controls. Subsequent analyses identified distinct droplet clusters with differential expression genes, including clusters enriched for genes associated with cell motility and mitochondrial function. Overall, TcEVdb serves as a comprehensive resource for exploring the transcriptome of TcEV, which will contribute to advancements in EV-based diagnostics and therapeutics across a wide range of diseases. Database URL: https://guolab.wchscu.cn/TcEVdb">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39cae304e2b4aafe6ad38b975f3d2d2df66e8a5c" target='_blank'>
              TcEVdb: a database for T-cell-derived small extracellular vesicles from single-cell transcriptomes
              </a>
            </td>
          <td>
            Tao Luo, Wen-Kang Shen, Chu-Yu Zhang, Dan-Dan Song, Xiuqing Zhang, An-Yuan Guo, Qian Lei
          </td>
          <td>2025-02-28</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Minimally invasive thermal therapies show great prospect in non-small cell lung cancer (NSCLC) treatment. However, changes in immune cell populations following microwave ablation (MWA) in NSCLC microenvironment are not fully revealed. Objective The present study was conducted to identify changes in immune cell populations and analyse dysregulated genes in immune cells after MWA in NSCLC microenvironment. Methods The patients received fractionated MWA in two treatments separated by 3 weeks. Tumor biopsy samples were obtained through core-needle biopsy before each fractionated MWA procedure at the same site and used for single-cell RNA sequencing with the 10x Genomics pipeline. Results A total of 9 major cell types were identified after MWA, which include neutrophils, T cells, B cells, monocytes, epithelial cells, chondrocytes, macrophages, tissue stem cells, and endothelial cells. After MWA, the tumor tissue exhibited an increased proportion of T cells. MWA altered gene expression in each cell cluster at the single-cell level. Cell trajectory analysis revealed that the cells at the starting point were most like T helper cells, naïve T cells, and regulatory T cells; they then developed into anergic T cells, T follicular cells, natural killer T cells, T memory cells, and exhausted T cells, and finally ended as γδ T cells and cytotoxic T cells. Moreover, after MWA, more interaction between monocytes and T cells (or B cells) were identified. Conclusions MWA increases local T-cell abundance and alters monocyte interactions, thereby reshaping the tumor microenvironment. This study lays a foundation for investigating dysregulated genes that may contribute to the MWA-induced immune response in NSCLC. What is already known on this topic Thermal ablation may change the immune profiles of patients by activating various steps in the cancer immunity cycle. However, changes in immune cell populations following MWA of NSCLC have not been fully reported. What this study adds After MWA, an increase in interactions between monocytes and T cells intratumorally was observed, which promoted antitumor immunity. How this study might affect research, practice or policy The current study illuminates the MWA-caused systemic immune response in NSCLC, which may help to identify the dysregulated genes involved in the MWA-caused immune response. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14002-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94b234806a8bba53b2676e517b9f991af5ad5695" target='_blank'>
              Microwave ablation of non-small cell lung cancer enhances local T-cell abundance and alters monocyte interactions
              </a>
            </td>
          <td>
            , Yuan-Ming Li, Zhixin Bie, Jin Peng, Xiao-Guang Li
          </td>
          <td>2025-04-03</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The progression of lung adenocarcinoma is primarily driven by cancer stem cells (CSCs), which have self-renewal capabilities and confer resistance to therapies, including neoadjuvant treatments combining chemotherapy and immune checkpoint inhibitors. In this study, we identified that OV6+ tumor cells exhibit stem-like characteristics and are notably enriched in patients with non–major pathological response, closely associated with resistance to combination therapies. Hypoxia and HIF1α were found to drive the formation of OV6+ CSCs. METTL3, a methyltransferase, was revealed as a critical regulator of OV6+ CSCs by stabilizing NFE2L3 messenger RNA via an N6-methyladenosine–dependent manner, thereby up-regulating NFE2L3 and activating the intrinsic WNT signaling pathway essential for maintaining stemness. OV6+ tumor cells promoted M2 macrophage infiltration and the formation of an immunosuppressive tumor microenvironment (TME). Targeting METTL3 effectively eliminated OV6+ CSCs and suppressed tumor progression. Moreover, the combination of STM2457 with cisplatin overcame chemoresistance, remodeled the TME, and provided promising insights for enhancing the efficacy of neoadjuvant combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7557203216007c8c41f75e3be43bc7a349a95ddf" target='_blank'>
              A METTL3-NFE2L3 axis mediates tumor stemness and progression in lung adenocarcinoma
              </a>
            </td>
          <td>
            Yue Zhao, Lei Zhang, Lang Xia, H. E, Tao Wang, Huinan Lu, Hezhong Chen, Y. She, Hao Tang, Junqi Wu, Deping Zhao, Chang Chen
          </td>
          <td>2025-04-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs), known for facilitating the progression and metastasis of colorectal cancer (CRC), have become a promising therapeutic target. However, the significant heterogeneity of CAFs and their intricate crosstalk with tumor cells present substantial challenges in the development of precise and effective therapeutic strategies. Methods Single-cell RNA sequencing (scRNA-seq) technology was used to identify various cell subtypes. Spatial transcriptomics (ST) was employed to map the spatial niches and colocalization patterns of these cell subtypes. Cell-cell interactions among these subtypes were analysed via CellChat and NicheNet software. Tumor cell invasion, migration, and proliferation were assessed through wound healing assays, transwell assays, colony formation assays, and xenograft mouse models. Results We identified a significant spatial colocalization between CTHRC1+ CAFs and a distinct subtype of malignant epithelial cells, both residing within the EMT-active spatial niche. Our results demonstrate that CTHRC1+ CAFs, as a major source of WNT5A, promote epithelial-mesenchymal transition (EMT) and enhance tumor cell invasiveness by upregulating MSLN expression in adjacent malignant epithelial cells. This signaling axis contributes significantly to CRC progression and metastasis. Conclusions Targeting the CTHRC1+ CAF-WNT5A-MSLN signaling axis presents a promising therapeutic strategy for advanced CRC patients. Our study provides new insights into the role of CAFs in CRC progression and offers potential avenues for developing targeted therapies to disrupt this pathway. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06236-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Peritoneal carcinomatosis is a common yet deadly manifestation of gastrointestinal cancers, with few effective treatments. To identify targetable determinants of peritoneal metastasis, we focused on appendiceal adenocarcinoma (AC), a gastrointestinal cancer that metastasizes almost exclusively to the peritoneum. Current treatments are extrapolated from colorectal cancer (CRC), yet AC has distinct genomic alterations, mucinous morphology and peritoneum restricted metastatic pattern. Further, no stable preclinical models of AC exist, limiting drug discovery and representing an unmet clinical need. We establish a first-in-class stable biobank of 16 long-term cultured AC patient-derived organoids (PDOs), including 3 matched, simultaneously resected primary AC-peritoneal carcinomatosis (AC-PC) pairs. By enriching for cancer cells, AC PDOs enable accurate genomic characterization relative to paucicellular AC tissue. We establish an organoid orthotopic intraperitoneal xenograft model that recapitulates diffuse peritoneal carcinomatosis and show that PC-organoids retain increased metastatic capacity, decreased growth factor dependency and sensitivity to standard of care chemotherapy relative to matched primary AC organoids. Single cell profiling of AC-PC pairs reveals dedifferentiation from mucinous differentiated states in primary AC into intestinal stem cell and fetal progenitor states in AC-PC, with upregulation of oncogenic signaling pathways. Through hypothesis-driven drug testing, we identify KRASMULTI-ON inhibitor RMC-7977 and Wnt-targeting tyrosine kinase inhibitor WNTinib as novel, clinically actionable strategies to target AC-PC more effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3baef141d668078d350e52fdb87357fa4d118d" target='_blank'>
              Paired primary-metastasis patient-derived organoids and mouse models identify phenotypic evolution and druggable dependencies of peritoneal metastasis from appendiceal cancer
              </a>
            </td>
          <td>
            Ahmed Mahmoud, Philip H. Choi, Christine Sukhwa, Jura Pintar, H. Walch, Nan Zhao, J. Bermeo, Sebastian K Chung, Manisha Raghavan, Samhita Bapat, Qingwen Jiang, G. Karagkounis, Julia Meredith, Michael Giarrizzo, C. Firat, A. Cercek, Michael B Foote, Nikolaus Schultz, W. Chatila, G. Nash, J. Shia, F. Sánchez-Vega, Steven M Larson, Arvin C. Dar, Neal Rosen, K. Ganesh
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Simple Summary Pancreatic cancer has a low survival rate with limited treatment options. Perineural invasion (PNI), where cancer cells infiltrate the nerves in the pancreas, is associated with a poor prognosis and local recurrence. Pancreatic cancer is one of the most painful cancers, with PNI contributing to the pain experienced by patients. However, the mechanisms involved in the cancer invasion of nerves are not completely understood. Here, we used spatial transcriptomics to study the expression of both cancer and nerve cells in microscopic regions of PNI and compared it to regions without PNI evidence to better understand the mechanisms involved in PNI. We identified novel mechanisms that may impact cancer growth, PNI and pain experienced by patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1818d7eac1f6857a9dc4b88e4ee2f868f4f36db6" target='_blank'>
              Spatial Transcriptomics Reveals Novel Mechanisms Involved in Perineural Invasion in Pancreatic Ductal Adenocarcinomas
              </a>
            </td>
          <td>
            V. Lakis, Noni L Chan, Ruth J Lyons, Nicola Blackburn, Tam Hong Nguyen, Crystal Chang, Andrew Masel, Nicholas West, Glen M. Boyle, A. Patch, Anthony J. Gill, K. Nones
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Breast cancer is the most prevalent malignancy in women and exhibits significant heterogeneity. The tumor microenvironment (TME) plays a critical role in tumorigenesis, progression, and response to therapy. However, its impact on the prognosis and immunotherapy responses is incompletely understood. Using public databases, we conducted a comprehensive investigation of transcriptome and single-cell sequencing data. After performing immune infiltration analysis, we conducted consensus clustering, weighted gene co-expression network analysis (WGCNA), Cox regression, and least absolute shrinkage and selection operator (Lasso) regression to identify independent prognostic genes in breast cancer. Subsequently, we developed a prognostic model for patients with breast cancer. Tumor Immune Dysfunction and Exclusion (TIDE) values were used to assess patient’s responsiveness to breast cancer. Based on single-cell RNA-sequencing data, we identified various cell types through cluster analysis and investigated the expression of prognostic model genes in each cell type. The drug sensitivity of targeted therapeutic agents for breast cancer treatment was analyzed in different cell types. We identified 12 independent prognostic genes associated with breast cancer and used these genes to construct a prognostic model. The prognostic model accurately discriminated between patients classified as high- and low-risk, providing precise prognostic predictions for individual patients. Additionally, our model exhibited a robust capacity to predict the immunotherapeutic response in breast cancer patients. Our investigation revealed a notable association between the proportion of endothelial cells (ECs) and patient prognosis in breast cancer. A prognostic model for breast cancer was formulated that showed close associations between prognosis and response to immunotherapy. For patients predicted by our model to not respond effectively to immunotherapeutic agents, it may be considered to combine immunotherapeutic agents with targeted therapeutic agents identified through our drug sensitivity analysis, which could potentially enhance treatment efficacy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-97784-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d3bf29828fea11eaaea62401b9c917be2125fbb" target='_blank'>
              Exploring the tumor microenvironment of breast cancer to develop a prognostic model and predict immunotherapy responses
              </a>
            </td>
          <td>
            Ye Tian, Yong Yang, Lei He, Xiaocheng Yu, Hu Zhou, Juan Wang
          </td>
          <td>2025-04-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Significance The distinctive nature of glioblastoma (GBM) and the blood–brain barrier (BBB) are obstacles for therapies that aim to permanently stop glioma cells from dividing. Similar to reprogramming normal cells, “differentiation therapies” try to push cancer cells into a flexible state and then guide them to differentiate into nondividing cells. Our study shows that using radiation first and then activating adenylate cyclase can solve the problems found in earlier work. Irradiation induces a flexible cellular state, and the addition of an adenylate cyclase activator helps turn them into types that cannot regrow tumors. This combined approach works well against GBM and suggests that using radiotherapy together with carefully timed differentiation treatments that force cells to mature could improve treatment outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cee9b47f25b8f95cc9d3c995fd15ca77ac2b58d9" target='_blank'>
              Radiation-induced cellular plasticity primes glioblastoma for forskolin-mediated differentiation
              </a>
            </td>
          <td>
            Ling He, Daria J. Azizad, Kruttika Bhat, A. Ioannidis, Carter J. Hoffmann, Evelyn Arambula, M. Eghbali, A. Bhaduri, H. Kornblum, Frank Pajonk
          </td>
          <td>2025-02-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="Laryngeal squamous cell carcinoma (LSCC) is a highly aggressive malignancy with a rising incidence over time. The tumor microenvironment (TME) plays a crucial role in LSCC development, yet the precise cellular characteristics of laryngeal cancer and its TME remain unclear. Here, we employed single-cell RNA sequencing analysis to uncover the heterogeneous populations of tumor and immune cells and investigate the role of the TME in LSCC. This analysis revealed significant heterogeneity among malignant cells, T cells, and macrophages. Notably, regulatory T cells were markedly increased at tumor sites, and macrophage analysis identified an increased presence of the Macrophage-C1-C1QC subset with up-regulated PPIF expression. Bulk RNA-seq further confirmed PPIF up-regulation in exosomes derived from LSCC tissues. Consistently, survival analysis indicated that high PPIF expression was associated with poor prognosis in LSCC. Further analyses suggested that PPIF up-regulation in Macrophage-C1-C1QC cells was associated with the enhancement of their anti-inflammatory phenotype and the promotion of F11R-F11R signaling with malignant cells, allowing LSCC cells to evade macrophage-mediated cytotoxicity. Our study provides new insights into the cellular dynamics of LSCC and highlights the critical role of Macrophage-C1-C1QC and PPIF in LSCC progression, offering potential therapeutic targets for treatment. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93584-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91301c190dc2f8d57f1a436734bdde8342d0e72d" target='_blank'>
              Unveiling the role of PPIF and macrophage subtypes in LSCC progression via single-cell and exosome RNA sequencing
              </a>
            </td>
          <td>
            Jianchao Wang, Chunhan Liu, Tailin Wu, Lei Tan, Jiangyan Wang, Wenliang Zhang, Yang Liu, Xudong Wei, Di Zhang
          </td>
          <td>2025-03-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ed6afc1fbea4684dd9546a568ad33c0ccc1b6d" target='_blank'>
              Mapping molecular landscapes in triple-negative breast cancer: insights from spatial transcriptomics.
              </a>
            </td>
          <td>
            F. Al-Mansour, H. Almasoudi, Ali M Albarrati
          </td>
          <td>2025-03-22</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f8ba9e3fcdae65a1005fa8694699f4a2467c8be" target='_blank'>
              Intratumour oxidative hotspots provide a niche for cancer cell dissemination.
              </a>
            </td>
          <td>
            Yoshifumi Ueda, Shigeki Kiyonaka, L. Selfors, Keisuke Inoue, Hiroshi Harada, Tomohiro Doura, Kunishige Onuma, Makoto Uchiyama, Ryuhei Kurogi, Yuji Yamada, Jiacheng H Sun, Reiko Sakaguchi, Yuki Tado, Haruki Omatsu, Harufumi Suzuki, M. Aoun, Takahiro Nakayama, Taketoshi Kajimoto, Tetsuya Yano, Rikard Holmdahl, I. Hamachi, Masahiro Inoue, Yasuo Mori, Nobuaki Takahashi
          </td>
          <td>2025-02-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="As our understanding of biological systems reaches single-cell and high spatial resolutions, it becomes imperative that pharmacological approaches match this precision to understand drug actions. This need is particularly urgent for the targeted covalent inhibitors that are currently re-entering the stage for cancer treatments. Therefore, we developed 3D volumetric clearing-assisted tissue click chemistry (vCATCH) to image in vivo target-bound drugs by leveraging the unique kinetics of click reactions. In mice, vCATCH overcomes limitations of existing methods to achieve high-throughput volumetric chemical labeling through simple and passive incubation steps, thus enabling unbiased cellular resolution drug imaging in entire animals. We combined vCATCH with HYBRiD imaging and virtual reality to visualize and quantify in vivo targets of two clinical cancer drugs, afatinib and ibrutinib, which recapitulated their known pharmacological distribution and revealed previously unreported tissue and cell type engagement potentially linked to off-target effects. vCATCH provides a body-wide, unbiased platform to map covalent drug engagements at unprecedented scale and precision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c93b01caf10ff1940a76d1d7bc582ebb625cf121" target='_blank'>
              Mapping cellular targets of covalent cancer drugs in the entire mammalian body
              </a>
            </td>
          <td>
            Zhengyuan Pang, Verina H. Leung, Cailynn C. Wang, Ahmadreza Attarpour, Anthony Rinaldi, Hanbing Shen, Christopher Glynn, Logan H. Sigua, Melaina Yender, Alexandra Selke, Senhan Xu, Peng Wu, Maged Goubran, Benjamin F. Cravatt, Li Ye
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="T cells play diverse roles in cancer immunology, acting as tumor suppressors, cytotoxic effectors, enhancers of cytotoxic T lymphocyte responses and immune suppressors; providing memory and surveillance; modulating the tumor microenvironment (TME); or activating innate immune cells. However, cancer cells can disrupt T cell function, leading to T cell exhaustion and a weakened immune response against the tumor. The expression of exhausted T cell (Tex) markers plays a pivotal role in shaping the immune landscape of multiple cancers. Our aim was to systematically investigate the role of known T cell exhaustion (Tex) markers across multiple cancers while exploring their molecular interactions, mutation profiles, and potential implications for immunotherapy. The mRNA expression profile of six Tex markers, LAG-3, PDCD1, TIGIT, HAVCR2, CXCL13, and LAYN was investigated in pan-cancer. Utilizing data from The Cancer Genome Atlas (TCGA), Gene Expression Omnibus (GEO), The Cancer Proteome Atlas (TCPA), and other repositories, we characterized the differential expression of the Tex markers, their association with the patients’ survival outcome, and their mutation profile in multiple cancers. Additionally, we analyzed the effects on cancer-related pathways and immune infiltration within the TME, offering valuable insights into mechanisms of cancer immune evasion and progression. Finally, the correlation between their expression and sensitivity to multiple anti-cancer drugs was investigated extensively. Differential expression of all six markers was significantly associated with KIRC and poor prognosis in several cancers. They also played a potential activating role in apoptosis, EMT, and hormone ER pathways, as well as a potential inhibitory role in the DNA damage response and RTK oncogenic pathways. Infiltration of different immune cells was also found to be associated with the expression of the Tex-related genes in most cancer types. These findings underline that the reviving of exhausted T cells can be used to enhance the efficacy of immunotherapy in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b32105e8581ecae32317ce64eca6316a45d0d52" target='_blank'>
              Characterization of Exhausted T Cell Signatures in Pan-Cancer Settings
              </a>
            </td>
          <td>
            Rifat Tasnim Juthi, Saiful Arefeen Sazed, Manvita Mareboina, A. Zaravinos, Ilias Georgakopoulos-Soares
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Mesothelioma cell heterogeneity encompasses diverse morphological and molecular characteristics observed within tumors, significantly impacting disease progression, treatment outcomes, and the development of targeted therapies. This heterogeneity has long posed challenges for accurate diagnosis and effective treatment, but understanding its complexities offers the potential for novel diagnostic modalities and therapeutic interventions. This study employed single-cell RNA sequencing (scRNA-seq) to investigate mesothelioma cell heterogeneity from various sources, including cell culture (CC), peritoneal lavage (Lav) from the tumor microenvironment, and circulating tumor cells (CTC) in murine models. Gene set enrichment analysis was used to identify distinct gene signatures for each subpopulation. The results revealed unique characteristics for mesothelioma cells depending on their origin. In the CC group, up-regulated genes were primarily involved in tumor cell cycle control, proliferation, and apoptosis. In the CTC group, up-regulated genes were associated with cancer cell stemness. The Lav group showed up-regulated genes facilitating interactions between tumor cells and the microenvironment, such as epithelial–mesenchymal transition and immune responses mediated by IFN-α and IFN-γ. Some pathways were shared among all tumor cells, suggesting the potential for transitioning between functional states under specific conditions. This may be the first study to explore circulating mesothelioma cell heterogeneity using scRNA-seq. The distinct gene signatures identified in each mesothelioma cell subpopulation likely play critical roles in tumor initiation and progression, offering potential novel targets for therapeutic intervention. These findings could help inform the development of more effective, personalized treatments for mesothelioma, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38354bc1c5ef98aa13d3c5253963e1d30d5518bf" target='_blank'>
              Mesothelioma cell heterogeneity identified by single cell RNA sequencing
              </a>
            </td>
          <td>
            Licun Wu, Zongjie Wang, Amin Zia, S. Kelley, Marc de Perrot
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Purpose Bladder cancer (BC) is characterized by high heterogeneity, with non-muscle-invasive (NMIBC) and muscle-invasive (MIBC) stages differing significantly in clinical behavior and outcomes. The transition from NMIBC to MIBC involves extensive tumor microenvironment (TME) remodeling, particularly in endothelial cells (ECs), which drive angiogenesis and modulate immune and extracellular matrix (ECM) interactions. However, the precise roles of ECs in this progression remain poorly defined. Methods Public single-cell RNA sequencing (scRNA-seq) datasets from 47 BC patients were analyzed to characterize endothelial cell heterogeneity and functional states across NMIBC and MIBC. Computational tools such as CellChat were applied to reconstruct cell–cell communication networks, focusing on pathways related to angiogenesis, immune crosstalk, and ECM remodeling. Results Twelve major cell types were identified, with endothelial cells exhibiting distinct transcriptional profiles between NMIBC and MIBC. NMIBC-associated ECs promoted adhesion and migration through HMGB1 and CXCL12 signaling. In contrast, MIBC was enriched in an ADAM10+ endothelial subset associated with vascular remodeling and activation of Wnt signaling via CTNNB1. Key ligand-receptor interactions highlighted the dynamic roles of ECs in TME modulation during BC progression. Conclusions This study reveals stage-specific endothelial cell phenotypes and signaling networks in BC. The identification of an MIBC-specific ADAM10+ endothelial subset underscores its potential role in driving tumor progression and highlights opportunities for stage-adapted vascular-targeted therapies. These findings advance our understanding of BC pathogenesis and provide the foundation for novel therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02297-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7505578630274eaf54930e9821b19f4ea81e3ca3" target='_blank'>
              A single-cell atlas of bladder cancer unveils dynamic cellular composition and endothelial functional shifts during progression
              </a>
            </td>
          <td>
            Hongjian Song, Guixiang Xie, Yaowei Li, Xiaowei Hu, Zongzheng Yang, Yubo Zhao, Qing Shi, Haonan Li, Ziyi Liu, Zhihao Yin, Ziqi Wang, Zhichao Tong, Wanhai Xu
          </td>
          <td>2025-04-09</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer cells harbor cell-intrinsic programs capable of sculpting immunogenic landscapes, yet molecular brakes that enforce immune evasion remain poorly defined. Here, we identify C9ORF50, a previously uncharacterized intrinsically disordered protein, as a liquid-liquid phase separation-driven regulator of RNA processing and actionable immunotherapy target. Genetic ablation of C9ORF50 induces selective intron retention in spliceosome component transcripts, leading to cytoplasmic accumulation of immunogenic double-stranded RNA (dsRNA). This dsRNA activates type I interferon pathway that in turn orchestrates a chemokine secretory program critical for T cell recruitment. The resultant enhancement of tumor immunogenicity and reprogramming of tumor immune microenvironment drives robust antitumor immunity, effectively converting immunologically “cold” tumors to “hot” phenotypes. Therapeutic targeting of C9ORF50 via RNA interference achieved robust tumor suppression in syngeneic models, suggesting its translational relevance. Notably, C9ORF50 exhibits tumor-predominant expression, with minimal baseline levels detected in healthy tissues or immune cell populations, underscoring its potential safety as a therapeutic target. Our study not only identifies C9ORF50 as a novel LLPS-dependent regulator of RNA splicing but also establishes it as a promising and potentially safe therapeutic target for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7433cc0492d9a830d01b0f8ac6ca2662ebe2b369" target='_blank'>
              C9orf50 is a Targetable Spliceosome Regulator for Cancer Immunotherapy
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Fangzhou Liu, Lulu Zhang, Ming Li, Tao Hou, Yanhua Qi, Jingyang Wang, Sujuan Zhang, Yu Liu, Jiali Liu, Yanming Hu, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive histological subtype of ovarian cancer and often presents with metastatic disease. The drivers of metastasis in HGSOC remain enigmatic. APOBEC3A (A3A), an enzyme that generates mutations across various cancers, has been proposed as a mediator of tumor heterogeneity and disease progression. However, the role of A3A in HGSOC has not been explored. We observed an association between high levels of APOBEC3-mediated mutagenesis and poor overall survival in primary HGSOC. We experimentally addressed this correlation by modeling A3A expression in HGSOC, and this resulted in increased metastatic behavior of HGSOC cells in culture and distant metastatic spread in vivo, which was dependent on catalytic activity of A3A. A3A activity in both primary and cultured HGSOC cells yielded consistent alterations in expression of epithelial-mesenchymal transition (EMT) genes resulting in hybrid EMT and mesenchymal signatures, providing a mechanism for their increased metastatic potential. Inhibition of key EMT factors TWIST1 and IL-6 resulted in mitigation of A3A-dependent metastatic phenotypes. Our findings define the prevalence of A3A mutagenesis in HGSOC and implicate A3A as a driver of HGSOC metastasis via EMT, underscoring its clinical relevance as a potential prognostic biomarker. Our study lays the groundwork for the development of targeted therapies aimed at mitigating the deleterious effect of A3A-driven EMT in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5397c063e336101c76892114dddd3f14218105fb" target='_blank'>
              APOBEC3A drives ovarian cancer metastasis by altering epithelial-mesenchymal transition
              </a>
            </td>
          <td>
            J. Devenport, Thi Tran, Brooke R. Harris, Dylan F. Fingerman, Rachel A. DeWeerd, Lojain H Elkhidir, Danielle LaVigne, Katherine C Fuh, Lulu Sun, Jeffrey J Bednarski, Ronny Drapkin, Mary M Mullen, Abby M. Green
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Drug resistance in metastatic lung cancer significantly contributes to patient mortality. This study explores the role of circulating tumor cells (CTCs), the precursors to metastasis, in driving this resistance. We aim to delineate the unique biological traits of CTC clusters in lung cancer and elucidate the mechanisms underlying their resistance to chemotherapy. Methods We used an ultralow adsorption plate to establish a CTC suspension culture system. Comparisons between adherent and suspension cultures of CTC-TJH-01 cells were made via Cell Counting Kit-8 (CCK-8), western blot, immunofluorescence, and flow cytometry assays to evaluate cell proliferation, drug resistance, and cancer stemness. The tumorigenicity, tumor growth rate, and drug resistance of the CTC clusters were assessed in nude mice. Transcriptomic and proteomic analyses were subsequently conducted to identify differentially expressed genes and proteins in CTC-TJH-01 cells cultured under adherent and suspension conditions. CDH17 gene knockdown in CTC-TJH-01 cells was achieved through RNA interference, and hematoxylin and eosin (HE) staining, immunohistochemistry, and immunofluorescence assays were used to examine the pathological status of these cells. Results CTC-TJH-01 cells in suspension formed cell clusters and exhibited decreased proliferation, tumorigenicity, and tumor growth, but increased cancer stemness and drug resistance. CDH17 protein expression was significantly upregulated in these clusters, activating the YAP/TAZ pathway. Knocking down CDH17 not only inactivated this pathway but also significantly increased cell proliferation activity and cisplatin sensitivity in CTC-TJH-01 clusters. Additionally, the tumor growth rate was correlated with cisplatin sensitivity. CDH17 knockdown notably promoted the growth of CTC-TJH-01 xenografts and enhanced their sensitivity to cisplatin, although no significant difference was observed compared with those in the control group. Conclusions The results indicate that lung CTC clusters with stem cell-like properties exhibit chemoresistance, which is linked to an activated CDH17-YAP pathway. Additionally, the effectiveness of cisplatin is primarily observed in tumors with relatively high growth rates, highlighting the connection between tumor growth and sensitivity to chemotherapy. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00696-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9926983d410efc85986d3eb1b6eeb8264046490c" target='_blank'>
              Regulating chemoresistance and cancer stemness: the CDH17-YAP pathway in distinct cellular states of lung cancer CTC clusters
              </a>
            </td>
          <td>
            Zujun Que, Dan Qi, Yun Yang, Wang Yao, Jiajun Liu, Yan Li, Yuanyuan Yu, Luyao Wang, Fangfei Li, Ge Zhang, Erxi Wu, Jianhui Tian
          </td>
          <td>2025-02-24</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>2</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f973214ce129b073da8e0d3c1c3cc399902a2e8" target='_blank'>
              Spatial single-cell proteomics landscape decodes the tumor microenvironmental ecosystem of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Libing Hong, Jie Mei, Xuqi Sun, Yifan Wu, Zhen Dong, Yuzhi Jin, Liaoliao Gao, Jinlin Cheng, Weihong Tian, Chuan Liu, Bin Li, Pingping Hu, Lulu Liu, Shan Xin, Xiaomeng Dai, Peng Zhao, Rongping Guo, Minshan Chen, Jingping Yun, Bo Lin, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-02-25</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Summary Spatial and temporal intra-tumor heterogeneity drives tumor evolution and therapy resistance. Existing visualization tools often fail to capture both dimensions simultaneously. To address this, we developed Jellyfish, a tool that integrates phylogenetic and sample trees into a single plot, providing a holistic view of tumor evolution and capturing both spatial and temporal evolution. Available as a JavaScript library and R package, Jellyfish generates interactive visualizations from tumor phylogeny and clonal composition data. We demonstrate its ability to visualize complex subclonal dynamics using data from ovarian high-grade serous carcinoma. Availability and implementation Jellyfish is freely available with MIT license at https://github.com/HautaniemiLab/jellyfish (JavaScript library) and https://github.com/HautaniemiLab/jellyfisher (R package).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0210199a3e67c966717695406003ace2c046fe" target='_blank'>
              Jellyfish: integrative visualization of spatio-temporal tumor evolution and clonal dynamics
              </a>
            </td>
          <td>
            K. Lavikka, Altti Ilari Maarala, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-02-25</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Gastrointestinal metastases are rare in patients with thyroid carcinoma (TC), and their underlying mechanisms remain unclear. Thus, in this study, we aimed to explore the spatial distribution characteristics of TCs and associated gastrointestinal metastatic cells. Methods We used spatial transcriptomics to generate an atlas that captures spatial gene expression patterns in papillary thyroid cancer (PTC), anaplastic thyroid carcinoma (ATC), ATC-associated lymphatic metastasis (ATC-LM), and rare ATC-associated gastric metastasis (ATC-GM). Results We demonstrated that tumor-specific myeloid cells with high SFRP4 expression were correlated with TC dedifferentiation and poor prognosis. Moreover, we validated their close localization to CD44+ tissue stem cells using immunofluorescence staining and spatial transcriptomics. We also demonstrated that ATC-LM and ATC-GM tissues exhibited high levels of CD44+PKHD1L1+ cells, which could serve as markers for these two pathological types. Conclusions These findings highlight the dynamic changes in cell composition, intercellular communication, and potential markers associated with TC dedifferentiation and distant metastasis. Further research based on our findings may contribute to improving diagnostic and therapeutic strategies for patients with TC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06252-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7453b499aa0f4f56b20054ed92fb6b1a0f0d81c" target='_blank'>
              Analysis of thyroid carcinoma composition and spatial architecture in the progression of dedifferentiation, lymphatic metastasis, and gastric metastasis
              </a>
            </td>
          <td>
            Di Wang, Ruichun Lu, Fenglian Yan, Yansong Lin, Hao Wang, Huabao Xiong
          </td>
          <td>2025-02-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most prevalent malignant brain tumor with poor prognosis. Although chromatin intratumoral heterogeneity is a characteristic feature of GBM, most current studies are conducted at a single tumor site. To investigate the GBM-specific 3D genome organization and its heterogeneity, we conducted Hi-C experiments in 21 GBM samples from nine patients, along with three normal brain samples. We identified genome subcompartmentalization and chromatin interactions specific to GBM, as well as extensive intertumoral and intratumoral heterogeneity at these levels. We identified copy number variants (CNVs) and structural variations (SVs) and demonstrated how they disrupted 3D genome structures. SVs could not only induce enhancer hijacking but also cause the loss of enhancers to the same gene, both of which contributed to gene dysregulation. Our findings provide insights into the GBM-specific 3D genome organization and the intratumoral heterogeneity of this organization and open avenues for understanding this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c40cfec55257420ce8df49630fbce7033db9c8" target='_blank'>
              Spatial 3D genome organization reveals intratumor heterogeneity in primary glioblastoma samples
              </a>
            </td>
          <td>
            Qixuan Wang, Juan Wang, Radhika Mathur, Mark Youngblood, Q. Jin, Ye Hou, L. Stasiak, Yu Luan, Hengqiang Zhao, Stephanie Hilz, Chibo Hong, Susan M Chang, Janine M Lupo, Joanna J Phillips, Joseph F Costello, Feng Yue
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: Intratumor heterogeneity in plantar melanoma orchestrates transcriptional programs that contribute to resistance to target- and immuno-therapies. However, the evolution and spatial distribution of cellular subgroups, as well as their effects on immune environment and patient prognosis, remain unclear. Methods: We analyzed 218,021 cells from 20 plantar melanoma and 6 normal samples using single-cell RNA sequencing to reveal the evolutionary characteristics and communication patterns of tumor subgroups. Spatial transcriptomics and multiplex immunohistochemistry (mIHC) were used to map the spatial distribution of these subgroups, with mIHC scores further evaluating their correlation with patient prognosis. Single-cell multiomics analysis identified key transcription factors associated with chromatin accessibility. In addition, survival analysis was performed using bulk RNA sequencing data from 68 melanoma patients. Results: We identified a continuum of subgroups originating from stem cells via transitional and Schwann cell-like precursor states, ultimately reaching a Schwann cell-like state. This evolution trajectory was supported by integrative evidence, including assessments of stemness, transitional states, RNA velocity, and transcription factors. The histological distribution of these subgroups was validated by spatial transcriptomics and multiple IHC. Notably, Schwann cell-like subgroup, regulated by transcription factor HMGA2, was associated with immune cell dysregulation and a worse prognosis, including increased invasion and lymph node metastasis. Mechanically, inhibition of HMGA2 expression blocked the transition to Schwann-like melanoma fate. Conclusions: This study reveals the unique evolutionary trajectory of plantar melanoma, showing its differentiation towards a Schwann-like fate regulated by HMGA2, leading to a decline in pigment function, enhanced immune tolerance and an increased propensity for lymph node metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/906aa3f7f8f1b89c37180b73edff27a732dad812" target='_blank'>
              A distinct subgroup with Schwann-like differentiation presents immune resistance and poor prognosis in plantar melanoma
              </a>
            </td>
          <td>
            Jie Tian, Lu Zhang, Zhuang Le, Pingping Lin, Shenxi Zhang, Yicen Yan, Yu Yang, Guohong Zhang, Hang Li, Binbin Lai
          </td>
          <td>2025-03-03</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genomic alterations are the driving force behind pancreatic cancer (PC) tumorigenesis, but they do not fully account for its diverse phenotypes. Investigating the epigenetic landscapes of PC offers a more comprehensive understanding and could identify targeted therapies that enhance patient survival. In this study, we have developed a new promising methodology of spatial epigenomics that integrates multiplexed molecular imaging with convolutional neural networks. Then, we used it to map epigenetic modification levels in the six most prevalent PC subtypes. We analyzed and semi-quantified the resulting molecular data, revealing significant variability in their epigenomes. DNA and histone modifications, specifically methylation and acetylation, were investigated. Using the same technique, we examined DNA conformational changes to further elucidate the transcriptional regulatory mechanisms involved in PC differentiation. Our results revealed that the foamy-gland and squamous-differentiated subtypes exhibited significantly increased global levels of epigenetic modifications and elevated Z-DNA ratios. Overall, our findings may suggest a potentially reduced efficacy of therapeutics targeting epigenetic regulators for these subtypes. Conversely, the conventional ductal PC subtype has emerged as a promising candidate for treatment with epigenetic modulators. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-90087-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e79fb6328397b7897b70522e3b8824a0291600" target='_blank'>
              Spatial recognition and semi-quantification of epigenetic events in pancreatic cancer subtypes with multiplexed molecular imaging and machine learning
              </a>
            </td>
          <td>
            Krzysztof Szymoński, Natalia Janiszewska, K. Sofińska, Katarzyna Skirlińska-Nosek, Dawid Lupa, Michał Czaja, Marta Urbańska, Katarzyna Jurkowska, Kamila Konik, Marta Olszewska, Dariusz Adamek, K. Awsiuk, E. Lipiec
          </td>
          <td>2025-02-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Crohn’s disease (CD) involves a complex intestinal microenvironment driven by chronic inflammation. While single-cell RNA sequencing has provided valuable insights into this biology, the spatial context is lost during single-cell preparation of mucosal biopsies. To deepen our understanding of the distinct inflammatory signatures of CD and overcome the limitations of single-cell RNA sequencing, we combined spatial transcriptomics of frozen CD surgical tissue sections with single-cell transcriptomics of ileal CD mucosa. Coexpressed genes and cell-cell communication from single-cell analyses and factorized genes from spatial transcriptomics revealed overlapping pathways affected in inflamed CD, like antigen presentation, phagosome activity, cell adhesion, and extracellular matrix. Within the pathways, early epithelial cells showed evidence of significant changes in gene expression and subtype composition, while spatial mapping revealed the location of the events, particularly antigen presentation from epithelial cells in the base of the crypt. Furthermore, we identified early epithelial cells as a potential mediator of the MHC class II pathway during inflammation, which we validated by spatial transcriptomics cell subtype deconvolution. Therefore, the inflammation from CD appears to change the types of interactions detectable between epithelial cells with immune and mesenchymal cells, likely promoting the conditions for more macrophage infiltration into these inflammatory microdomains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00594471c2ef5f7cea997fb7b198a677a1e23c9" target='_blank'>
              Altered inflammatory mucosal signatures within their spatial and cellular context during active ileal Crohn’s disease
              </a>
            </td>
          <td>
            Vasantha L. Kolachala, S. Maddipatla, Shanta Murthy, Yeonjoo Hwang, A. Dodd, Garima Sharma, Sachith Munasinghe, R. Pelia, S. Venkateswaran, M. Anbazhagan, Tarun Koti, Navdeep Jhita, Gaurav N Joshi, Chrissy A. Lopez, Duke Geem, Hong Yin, David J. Cutler, Peng Qiu, J. Matthews, S. Kugathasan
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts (PDXs), we systematically identified frequency-dependent selection dynamics governed by karyotypic diversity. Notably, in one PDX lineage, we found consistent evidence of karyotypically defined subpopulations acting as interaction hubs, associated specifically with chromosome 1 loss and chromosome 14p gain. Our framework provides testable predictions of intratumoral ecological dynamics, highlighting opportunities to strategically target key subpopulations to disrupt tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Joel S. Brown, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a494e730aa784cad617baf97d8efcc5bc5c5345" target='_blank'>
              Cell-type-specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Ovarian carcinoma represents an aggressive malignancy with poor prognosis and limited therapeutic efficacy. While deubiquitinating (DUB) genes are known to regulate crucial cellular processes and cancer progression, their specific roles in ovarian carcinoma remain poorly understood. Methods We conducted an integrated analysis of single-cell RNA sequencing and bulk transcriptome data from public databases. DUB genes were identified through Genecard database. Using the Seurat package, we performed cell clustering and differential expression analysis. Cell–cell communications were analyzed using CellChat. A DUB-related risk signature (DRS) was developed using machine learning approaches through integration of GEO and TCGA datasets. The prognostic value and immune characteristics of the signature were systematically evaluated. Results Our analysis revealed eight distinct cell subtypes in the tumor microenvironment, including epithelial, fibroblast, myeloid, and Treg cells. DUB-high cells were predominantly found in Treg and myeloid populations, exhibiting elevated expression of tumor-related pathways and enhanced cell–cell communication networks, particularly between fibroblasts and myeloid cells. Conversely, DUB-low cells were enriched in epithelial populations with reduced immune activity. The DRS model demonstrated robust prognostic value across multiple independent cohorts. High-risk patients, as classified by the DRS, showed significantly poorer survival outcomes and distinct immune infiltration patterns compared to low-risk patients. Conclusion This study provides comprehensive insights into DUB gene expression patterns across different cell populations in ovarian carcinoma. The established DRS model offers a promising tool for risk stratification and may guide personalized therapeutic strategies. Our findings highlight the potential role of DUB genes in modulating the tumor immune microenvironment and patient outcomes in ovarian carcinoma. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02267-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/186d9d1be19b56c1d6796da9872a6f27a083fcd1" target='_blank'>
              Identification of a deubiquitinating gene-related signature in ovarian cancer using integrated transcriptomic analysis and machine learning framework
              </a>
            </td>
          <td>
            Suwan Hu, Mengting Wang
          </td>
          <td>2025-04-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4955ba693d4e35ecf04dfaea41f95d050a589d61" target='_blank'>
              Abstract A029: Highly-resolved integrated single cell atlas of human breast cancer enables hypothesis-driven analysis of tumor heterogeneity
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, Stefano Monti
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7ab0c0159914b37ff2d6316b406bb08a12959ab" target='_blank'>
              Single-cell transcriptomic atlas of human retina from Chinese donors reveals population-specific cellular diversity
              </a>
            </td>
          <td>
            Sen Lin, Yiwen Tao, Luning Yang, Q. Pan, T. Cai, Yunyan Ye, Jianhui Liu, Yang Zhou, Quanyong Yi, Zen Haut Lu, Lie Chen, Gareth McKay, Richard Rankin, Yongqing Shao, Weihua Meng
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c6430741ee008f7f4a0b2354517f9792819deba" target='_blank'>
              Abstract B061: Sialylation-driven MDSC generation shapes the ovarian tumor immune microenvironment
              </a>
            </td>
          <td>
            A. Alvero, Alexandra Fox, Aditi Singh, Nicholas K Adzibolosu, Sapna Sharma, Garry D. Leonard, Charlie Fehl, Radhika Gogoi, Gil Mor
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ed63792a7cb4d7298d3e8340ecc0d2e80e3929" target='_blank'>
              EHE cell cultures: a platform for mechanistic and therapeutic investigation
              </a>
            </td>
          <td>
            Nicholas Scalora, Gillian DeWane, Yuliia Drebot, Ali A. Khan, Souradip Sinha, Krishnendu Ghosh, Denise Robinson, Patricia Cogswell, Andrew M. Bellizzi, Anthony N. Snow, Patrick Breheny, M. S. Chimenti, M. Tanas
          </td>
          <td>2025-03-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0fc7e6f9b6372a2eebc4b0d3580ed466ef20424" target='_blank'>
              A Tc1- and Th1-T-lymphocyte-rich tumor microenvironment is a hallmark of MSI colorectal cancer.
              </a>
            </td>
          <td>
            Zhihao Huang, T. Mandelkow, N. F. Debatin, M. Lurati, Julia Ebner, J. Raedler, E. Bady, Jan H. Müller, R. Simon, E. Vettorazzi, A. Menz, Katharina Möller, N. Gorbokon, G. Sauter, M. Lennartz, A. Luebke, D. Höflmayer, T. Krech, P. Lebok, C. Fraune, A. Hinsch, Frank Jacobsen, A. Marx, S. Steurer, S. Minner, D. Dum, S. Weidemann, C. Bernreuther, T. Clauditz, E. Burandt, N. Blessin
          </td>
          <td>2025-04-03</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5411691311f3ea382bc8274891edc9e740ebfb77" target='_blank'>
              Non-invasive approach for endoluminal biopsy coupled with single-cell proteomics allows for immune characterization of intracranial aneurysms
              </a>
            </td>
          <td>
            J. Antonios, B. Gultekin, B. Theriault, K. Yalcin, D. Miyagishima, N. Adenu-Mensah, N. Sujijantarat, A. Koo, J. Haynes, P. Cedeno, M. Johnson, R. Hebert, C. Matouk, T. Barak, M. Gunel
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Over 80% of children with acute lymphoblastic leukemia (pALL) can be cured by treating them with multiple chemotherapeutic agents administered over several years, whereas pALL is incurable with 1-3 medications, suggesting significant variation in drug susceptibility across clonal populations. While bulk sequencing studies indicate that pALL cells contain relatively few genetic variants compared to other cancers, the true extent of genetic diversity at the single-cell level remains unknown. Here, we used three complementary approaches to investigate pALL genetic heterogeneity: error-corrected bulk sequencing, single-cell exome sequencing, and primary template-directed amplification (PTA)-enabled single-cell genome sequencing. We discovered that some ETV6-RUNX1 samples harbor multiple independent ras clones and that individual pALL cells harbor substantially more mutations (mean 3,553 per cell) than detected in bulk samples (mean 965 mutations), with variant signatures suggesting both early and late APOBEC-driven mutagenesis in ETV6-RUNX1 patients. Using PTA-based phylogenetic analysis of over 150 single-cell genomes from four pALL patients, we identified heritable phenotypes associated with specific genetic alterations, including some low-frequency clones that are preferentially selected for during chemotherapy treatment. Our findings reveal previously undetected genetic diversity in pALL and suggest that pre-existing mutations influence treatment response, with implications for future therapeutic strategies. This study provides a high-resolution framework for understanding cancer clonal evolution during treatment, yielding important new insights for developing more effective therapeutic approaches for pALL. Key Points Increasing the accuracy and resolution of sequencing from bulk to error-corrected bulk to single clone to single-cell sequencing reveals increasing levels of genetic diversity in pALL, including the presence of clone-specific driver and treatment resistance-associated mutations. Using PTA-enabled scWGS, single pALL cells contain several fold more mutations per cell than detected in bulk sequencing, which, when multiplied across billions of cells, reveal hidden population-scale genetic complexity. Some phenotypes in pALL are heritable, including treatment resistance, where we identify pre-existing rare clones with relapse-associated mutations that preferentially survive four weeks of standard treatment in patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cfebdd69571f854dae1eb84acf1f1884a093ad9" target='_blank'>
              Measuring Longitudinal Genome-wide Clonal Evolution of Pediatric Acute Lymphoblastic Leukemia at Single-Cell Resolution
              </a>
            </td>
          <td>
            Yakun Pang, Tamara Prieto, V. Gonzalez-Pena, Athena Aragon, Yuntao Xia, Sheng Kao, Sri Rajagopalan, John Zinno, Jean Quentin, Julien Laval, Dennis Yuan, Nathaniel D. Omans, David Klein, Matthew MacKay, I. De Vlaminck, John Easton, William Evans, D. Landau, Charles Gawad
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Single-cell image analysis is crucial for studying drug effects on cellular morphology and phenotypic changes. Most studies focus on single cell types, overlooking the complexity of cellular interactions. Here, we establish an analysis pipeline to extract phenotypic features of cancer cells cultured with fibroblasts. Using high-content imaging, we analyze an oncology drug library across five cancer and fibroblast cell line co-culture combinations, generating 61,440 images and ∼170 million single-cell objects. Traditional phenotyping with CellProfiler achieves an average enrichment score of 62.6% for mechanisms of action, while pre-trained neural networks (EfficientNetB0 and MobileNetV2) reach 61.0% and 62.0%, respectively. Variability in enrichment scores may reflect the use of multiple drug concentrations since not all induce significant morphological changes, as well as the cellular and genetic context of the treatment. Our study highlights nuanced drug-induced phenotypic variations and underscores the morphological heterogeneity of ovarian cancer cell lines and their response to complex co-culture environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4573d60560710e744b74b82ab85a673cb84bfba" target='_blank'>
              Evaluating feature extraction in ovarian cancer cell line co-cultures using deep neural networks
              </a>
            </td>
          <td>
            Osheen Sharma, Greta Gudoityte, Rezan Minozada, Olli Kallioniemi, Riku Turkki, L. Paavolainen, B. Seashore-Ludlow
          </td>
          <td>2025-02-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Background Glioma, characterized by its cellular and molecular heterogeneity, presents formidable challenges in treatment strategy and prognostic assessment. The tumor microenvironment (TME) profoundly influences tumor behavior and treatment response, with tumor‐associated neutrophils (TANs) playing a complex but understudied role. This study aimed to investigate the heterogeneity and role of TANs in glioma and to develop a prognostic model. Methods Analysis of scRNA‐seq data identified cellular subpopulations and differentially expressed neutrophil‐related genes (DE‐NRGs). Bulk RNA‐seq was obtained from four independent datasets. Molecular subtypes of glioma samples were determined by consensus clustering. WGCNA was conducted to elucidate the association between gene modules and subtypes. We developed a risk score model. Expression of selected genes was confirmed using immunohistochemistry (IHC). In vitro experiments were also performed for functional verification, including CCK8, EdU, Transwell, and TUNEL assays. Results A total of 108 DE‐NRGs for TANs were identified based on scRNA‐seq data. Two molecular subtypes were characterized, showing significant differences in prognosis and clinical features. Immune‐related analyses demonstrated varied immunological characteristics between subtypes. The risk score model was constructed with 7 genes, including AEBP1, CAVIN1, DCTD, DEPP1, DUSP6, FKBP9, and UGCG. It showed significant prognostic value and was validated across three external datasets. The mutation landscape highlighted higher IDH mutation prevalence in low‐risk groups. Drug sensitivity analysis indicated TMZ resistance in high‐risk groups. In vitro experiments showed that UGCG could promote glioma cell proliferation, migration, and invasion, while decreasing apoptosis. Conclusion This study explored the heterogeneity of TANs and developed a prognostic model, providing insights for prognostic prediction and guiding personalized treatment strategies in glioma. Declaration of Generative AI in Scientific Writing: The authors declare nonuse of generative AI and AI‐assisted technologies in the writing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a8af983a6227909234b80a95e84b835146d4d4" target='_blank'>
              Integrated Analysis to Reveal Heterogeneity of Tumor‐Associated Neutrophils in Glioma
              </a>
            </td>
          <td>
            Wen Wang, Junsheng Li, Qiheng He, Chenglong Liu, Siyu Wang, Zhiyao Zheng, Bojian Zhang, Siqi Mou, Wei Sun, Jizong Zhao
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, Chao (Agnes) Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71083b48d8d0e95bc226f7370321af11b26e25b" target='_blank'>
              Spatial and single-cell transcriptomic analysis reveals fibroblasts dependent immune environment in colorectal cancer.
              </a>
            </td>
          <td>
            Hang Jia, Xiang Liu, Guimin Wang, Yue Yu, Ning Wang, Tianshuai Zhang, Liqiang Hao, Wei Zhang, Guanyu Yu
          </td>
          <td>2025-03-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Metastasis is a leading cause of cancer-related deaths, with the liver being the most frequent site of metastasis in colorectal cancer. Previous studies have predominantly focused on the influence of the primary tumor itself on metastasis, with relatively limited research examining the changes within target organs. Methods Using an orthotopic mouse model of colorectal cancer, single-cell sequencing was employed to profile the transcriptomic landscape of pre-metastatic and metastatic livers. The analysis focused on identifying cellular and molecular changes within the hepatic microenvironment, with particular emphasis on inflammatory pathways and immune cell populations. Results A neutrophil subpopulation with high Prok2 expression was identified, showing elevated levels in the pre-metastatic and metastatic liver. Increased infiltration of Prok2⁺ neutrophils correlated with poor prognosis in liver metastatic colorectal cancer patients. In the liver metastatic niche (MN), these neutrophils showed high App and Cd274 (PD-L1) expression, suppressing macrophage phagocytosis and promoting T-cell exhaustion. Conclusion A Prok2⁺ neutrophil subpopulation infiltrated both pre-metastatic and macro-metastatic liver environments, potentially driving immunosuppression through macrophage inhibition and T-cell exhaustion. Targeting Prok2⁺ neutrophils could represent a novel therapeutic strategy for preventing liver metastasis in colorectal cancer patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06351-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ae7b5dd668485b7f400a3f84e79d9d667bb33c" target='_blank'>
              Single-cell transcriptomic analysis reveals dynamic changes in the liver microenvironment during colorectal cancer metastatic progression
              </a>
            </td>
          <td>
            Yue Jiang, Guojie Long, Xiaoming Huang, Wenyu Wang, B. Cheng, Weidong Pan
          </td>
          <td>2025-03-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Human complex in vitro models (CIVMs) have demonstrated remarkable potential to study tissue development, physiology and disease at high-throughput. To effectively employ these miniaturized systems in translational preclinical research, their in-depth benchmarking is pivotal. Histology has been the core of tissue characterization for centuries and the foundation of spatial phenotyping. However, standard histology workflows are inherently low-throughput and centered on large tissue pieces. This does not match the high sample volumes and small sample sizes in CIVM research. Here, we introduce a holistic ‘histo-workflow’, utilizing 3D-printed histomolds that facilitate co-planar embedding of CIVMs at high-throughput, resulting in up to 48 samples in one section. We developed a variety of model-specific histomold designs that enable spatially controlled histological sectioning and downstream analyses. We describe these workflows, including mold generation, highplex staining and image analysis, and exemplify their application to histological analyses of various CIVMs. Altogether, the histomolds introduced here afford opportunities for CIVM processing and analysis, while significantly reducing labor and reagent resources, thereby democratizing high-throughput CIVM in histopathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1aa7001ef0a4f67578ea7c5375be8b50760ab64" target='_blank'>
              High-throughput histopathology for complex in vitro models
              </a>
            </td>
          <td>
            Marius F. Harter, Elisa D’Arcangelo, Julien Aubert, Barbora Lavickova, C. Havnar, Bilgenaz Stoll, Irineja Cubela, Adrian M. Filip, Laura Gaspa-Toneu, Julian Scherer, Evdoxia Karagkiozi, Jean-Samuel Dupré, Rupén López-Sandoval, Rebecca Hsia, Leah M Norona, Giovanna Brancati, Ryo Okuda, J. G. Camp, Eliah R. Shamir, J. Garcia-Cordero, Iago Pereiro, N. Gjorevski, N. Gjorevski
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Lynch Syndrome (LS) is an autosomal dominant disease characterized by germline heterozygous mutations in DNA mismatch repair (MMR) genes. High-risk LS patients may proceed to colorectal cancer (CRC). However, the drivers or biomarkers of LS benign colon tissue approaching malignant CRC are not completely understood. The similarity and difference between LS-related and nonLS-related CRC are also not well interrogated (LS-CRC vs. nonLS-CRC). This study aimed to understand the cellular changes during malignant transition in LS. Methods Single-cell RNA sequencing (scRNA-seq) was used to analyze paired biopsy samples from 3 patients with LS (cancer tissues vs. adjacent normal tissues, n=3). Single-nuclear RNA sequencing (snRNA-seq) was used to analyze a frozen biopsy sample from a patient with LS (n=1). scRNA-seq or snRNA-seq datasets from CRC were downloaded from the open source. Integrative computational analysis was performed to conclude the distinct pattern in the single-cell atlas. Immuno-histo-fluorescence staining (IHF) were also performed for three key markers. Results In the single-cell atlas, we observed an increase in primitive cancer stem-cell-like cells with high expression of a general cancer biomarker CarcinoEmbryonic Antigen-related Cell Adhesion Molecule 5 (CEACAM5) in the epithelium of the LS. Infiltration of immune cells and DNA repair biological activity are dramatically increased in LS carcinoma. The burden in LS is fundamentally elevated compared to that in CRC or healthy donors when the mutations are in the coding-sequence-wide range. Furthermore, T cell and macrophage-related tumor immunity is readily mobilized in carcinomas compared to paraCArcinomas. Conclusions This study provides single-cell transcriptomic resource using affected tissues from patients with Lynch Syndrome and describes an integrative profile covering the alterations (cancer stem cell markers, mutation burden, and tumor immunity) during the malignant transition from healthy to Lynch Syndrome and to colorectal cancer at the single-cell level. Highlights and Figure/Table Index (Take-Home Messages) Colon tissues from HD (n=4), patients with LS (n=6) and with CRC (n=3) were collected for snRNA-seq or scRNA-seq analysis; Diagnosis of the 6 LS patients are clearly supported by our pedigree documentations (Fig. 1 and Supp. Fig. 1) and briefed in Table 1; Among the 6 patients with LS, 1 for snRNA-seq analysis (a carcinoma tissue only), 3 for scRNA-seq analysis (3 paired para-carcinoma and carcinoma tissues) and 3 for experimental validations; (Fig. 1) Following studies are focused on the comparisons between HD, LS and CRC; In addition, comparisons between paired carcinoma and para-carcinoma tissues from patients with LS were also performed; (Fig. 2-7) When HD and LS are compared, a cancer stem cell marker CEACAM5 is readily detected by snRNA-seq analysis; (Fig. 5) When carcinoma and para-carcinoma from LS patients are compared, increased immune cell infiltration of and enhanced DNA repair activity in the malignant tissues is readily detected by scRNA-seq analysis; in addition, three upregulated markers in LS carcinoma (BACE2, GRPC5A and OLFM4) were identified and validated by immunohistoinfloresnce; (Fig. 3 and Fig. 4) SComatic algorithm-based mutation calling analysis suggests a comparable mutation burden between carcinoma and para-carcinoma from LS patients, but such mutation burdens are grossly greater than that from HD or that from patients with CRC; (Fig. 6)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863c44a229dbcaa552459faeccde9844a06701be" target='_blank'>
              Tracing the stemness and malignant transition in a heritable colorectal cancer Lynch Syndrome by single-cell RNA-seq analysis
              </a>
            </td>
          <td>
            Junfeng Xu, Yuhang Li, Zhiqin Wang, Qianru Li, Aijun Liu, Jianqiu Sheng, Ge Dong, Lang Yang, Zhigang Cai
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The spatial organization of cells in tissues underlies biological function, and recent advances in spatial profiling technologies have enhanced our ability to analyze such arrangements to study biological processes and disease progression. We propose MESA (multiomics and ecological spatial analysis), a framework drawing inspiration from ecological concepts to delineate functional and spatial shifts across tissue states. MESA introduces metrics to systematically quantify spatial diversity and identify hot spots, linking spatial patterns to phenotypic outcomes, including disease progression. Furthermore, MESA integrates spatial and single-cell multiomics data to facilitate an in-depth, molecular understanding of cellular neighborhoods and their spatial interactions within tissue microenvironments. Applying MESA to diverse datasets demonstrates additional insights it brings over prior methods, including newly identified spatial structures and key cell populations linked to disease states. Available as a Python package, MESA offers a versatile framework for quantitative decoding of tissue architectures in spatial omics across health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162d6ff6e3cedba6aedd63bc146790643b33531d" target='_blank'>
              Quantitative characterization of tissue states using multiomics and ecological spatial analysis
              </a>
            </td>
          <td>
            Daisy Yi Ding, Zeyu Tang, Bokai Zhu, Hongyu Ren, Alex K. Shalek, Robert Tibshirani, GP Nolan
          </td>
          <td>2025-04-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a279852f8f3e4dea6bf2c322ab3c789c81b3e6c" target='_blank'>
              How cancer emerges: Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-02-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfff83a3b2e44394201ebb9147dbd5e27d9e3ca" target='_blank'>
              Single-Cell Transcriptomic Analysis Reveals an Aggressive Basal-Like Tumor Cell Subpopulation Associated With Poor Prognosis in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Chang-Pin Liao, Yuting Zhang, Jing Yang, Shuo Wang, Zhijuan Li, Shuling Chen, Yubin Xie, Lixia Xu, Sui Peng, Xuezhen Zeng, Ming Kuang, Bangde Xiang, Kaiyu Sun, Xiao Zhao
          </td>
          <td>2025-02-24</td>
          <td>Journal of gastroenterology and hepatology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Intratumour heterogeneity and phenotypic plasticity drive tumour progression and therapy resistance1,2. Oncogene dosage variation contributes to cell-state transitions and phenotypic heterogeneity3, thereby providing a substrate for somatic evolution. Nonetheless, the genetic mechanisms underlying phenotypic heterogeneity are still poorly understood. Here we show that extrachromosomal DNA (ecDNA) is a major source of high-level focal amplification in key oncogenes and a major contributor of MYC heterogeneity in pancreatic ductal adenocarcinoma (PDAC). We demonstrate that ecDNAs drive varying levels of MYC dosage, depending on their regulatory landscape, enabling cancer cells to rapidly and reversibly adapt to microenvironmental changes. In the absence of selective pressure, a high ecDNA copy number imposes a substantial fitness cost on PDAC cells. We also show that MYC dosage affects cell morphology and dependence of cancer cells on stromal niche factors. Our work provides a detailed analysis of ecDNAs in PDAC and describes a new genetic mechanism driving MYC heterogeneity in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>1</td>
          <td>106</td>
        </tr>

        <tr id="Introduction The molecular pathogenesis of ocular adnexal mucosa-associated lymphoid tissue (MALT) lymphoma and IgG4-related ophthalmic disease (IgG4-ROD) remains incompletely understood. Differentiating between the two diseases is vital given that the diagnostic evaluation and treatment approaches can vary significantly; this difficulty in distinction is exacerbated by the absence of specific biomarkers. This study aimed to investigate the differences between these two diseases based on their cellular composition, transcriptional heterogeneity, and the immune microenvironment using single-cell RNA transcriptional sequencing (scRNA-seq) technology. Methods We collected orbital lacrimal gland region tissue samples from three patients with MALT lymphoma and another three with IgG4-ROD and performed single-cell sequencing experiments. Subsequently, we conducted bioinformatics analyses, including cell subpopulation segmentation and inter-group comparison, tumor cell identification, functional enrichment analysis, and pseudotime trajectory analysis. Furthermore, we analyzed the cellular communication between tumor B-cell and T-cell subsets within the immune microenvironment of MALT lymphoma tissues. We performed immunofluorescence assays to verify the co-expression of receptor-ligand pairs. Results A total of six major cell subpopulations were identified, with B-cells and T-cells being the predominant cell types. All B-cell subpopulations in MALT lymphomas are malignant, exhibiting significant intratumoral and intertumoral heterogeneity. Reclustering of the T-cell subpopulation identified five major T-cell subpopulations. Pseudotime analysis revealed that CD4+ naive T-cells in MALT lymphoma patients were highly likely to differentiate into follicular helper T-cells, whereas, in IgG4-ROD patients, CD4+ naive T-cells were highly likely to differentiate into regulatory T-cells. Intercellular communication analysis revealed that the CD27-CD70 immune checkpoint receptor−ligand pair and CXCL13-CXCR5 chemokine receptor−ligand pair were significantly upregulated between malignant B-cells and T-cells subpopulations. Conclusion This study is the first to conduct a comparative single-cell transcriptome sequencing analysis of ocular adnexal MALT lymphoma and IgG4-ROD. Our results reveal the cellular composition, key pathways, and critical immune microenvironment implicated in the development of these two diseases. These findings provide important insights into the pathogenesis of these two diseases and highlight the differences between them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5b84eefafe3887c5d3f9997b2994a421a348df" target='_blank'>
              Single-cell RNA-sequencing reveals cellular heterogeneity and immune microenvironment characteristics between ocular adnexal mucosa-associated lymphoid lymphoma and IgG4−related ophthalmic disease
              </a>
            </td>
          <td>
            Yu Yang, Yujiao Wang, Xuelian Jin, W. He
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1804a2ea493d2cd5cd4a540fc4df2d804160e23" target='_blank'>
              Differential effects of immobilized CCL21 and ICAM1 on TILs with distinct expansion properties
              </a>
            </td>
          <td>
            Sofi Yado, Rawan Zoabi, Karin Brezinger-Dayan, S. Albeck, Tamar Unger, Moran Meiron, Alessio D. Nahmad, Aya Tzur Gilat, Michal J Besser, Benjamin Geiger
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11004abebcf7dd56b12c4dc898da1ed0acccbe13" target='_blank'>
              Immunocytes in the tumor microenvironment: recent updates and interconnections
              </a>
            </td>
          <td>
            Jiyao Yu, Li Fu, Rui Wu, Linyi Che, Guodong Liu, Qinwen Ran, Zhiwei Xia, Xisong Liang, Guanjian Zhao
          </td>
          <td>2025-04-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a8e2bdd23ee299c051bf62b1634c5ff6fb1bbaf" target='_blank'>
              SOX2 commands LIM homeobox transcription factors in choroid plexus development and tumorigenesis.
              </a>
            </td>
          <td>
            Lukas J Faltings, Fengjuan Lin, Mariam Zahran, Heena Jalili, Anjali Siluveru, Mahek Chaudry, Leah M Wachsmuth, Navjot Guru, Melanie Schoof, Ping Cao, Yuan-hua Huang, Noreen Mian, Sohyun Moon, Asmaa Zahran, Kristen Green, Siddhi Modi, Maheen Umer, James Q Virga, Ying-Tao Zhao, U. Schüller, Qun Li, Haotian Zhao
          </td>
          <td>2025-03-25</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor heterogeneity is predicted to confer inferior clinical outcomes with precision-based strategies, however, modeling heterogeneity in a manner that still represents the tumor of origin remains a formidable challenge. Sequencing technologies are limited in their ability to identify rare subclonal populations and predict response to treatments for patients. Patient-derived organotypic cultures have significantly improved the modeling of cancer biology by faithfully representing the molecular features of primary malignant tissues. Patient-derived cancer organoid (PCO) cultures contain subclonal populations with the potential to recapitulate heterogeneity, although treatment response assessments commonly ignore diversity in the molecular profile or treatment response. Here, we demonstrate the advantage of evaluating individual PCO heterogeneity to enhance the sensitivity of these assays for predicting clinical response. Additionally, organoid subcultures identify subclonal populations with altered treatment response. Finally, dose escalation studies of PCOs to targeted anti-EGFR therapy are utilized which reveal divergent pathway expression when compared to pretreatment cultures. Overall, these studies demonstrate the importance of population-based organoid response assessments, the use of PCOs to identify molecular heterogeneity not observed with bulk tumor sequencing, and PCO heterogeneity for understanding therapeutic resistance mechanisms. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-96204-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d73b21d8a323278ef745932d923e8c41351e22f0" target='_blank'>
              Subclonal response heterogeneity to define cancer organoid therapeutic sensitivity
              </a>
            </td>
          <td>
            J.D. Kratz, Shujah Rehman, Katherine A. Johnson, A. Gillette, Aishwarya Sunil, Peter F Favreau, Cheri A Pasch, Devon D Miller, Lucas Zarling, Austin H. Yeung, L. Clipson, Samantha J. Anderson, Alyssa K Steimle, Carley M. Sprackling, K. Lemmon, Daniel E Abbott, M. Burkard, Michael F. Bassetti, J. Eickhoff, Eugene F Foley, Charles P Heise, Randall J Kimple, Elise H Lawson, N. LoConte, S. Lubner, Daniel L Mulkerin, Kristina A Matkowskyj, Cristina B. Sanger, Nataliya V. Uboha, Sean J Mcilwain, Irene M Ong, Evie H Carchman, Melissa C Skala, Dustin A Deming
          </td>
          <td>2025-04-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Glioblastomas are the most prevalent primary brain tumors and are associated with a dramatically poor prognosis. Despite an intensive treatment approach, including maximal surgical tumor removal followed by radio- and chemotherapy, the median survival for glioblastoma patients has remained around 18 months for decades. Glioblastoma is distinguished by its highly complex mechanisms of immune evasion and pronounced heterogeneity. This variability is apparent both within the tumor itself, which can exhibit multiple phenotypes simultaneously, and in its surrounding microenvironment. Another key feature of glioblastoma is its “cold” microenvironment, characterized by robust immunosuppression. Recent advances in single-cell RNA sequencing have uncovered new promising insights, revealing previously unrecognized aspects of this tumor. In this review, we consolidate current knowledge on glioblastoma cells and its microenvironment, with an emphasis on their biological properties and unique patterns of molecular communication through signaling pathways. The evidence underscores the critical need for personalized poly-immunotherapy and other approaches to overcome the plasticity of glioblastoma stem cells. Analyzing the tumor microenvironment of individual patients using single-cell transcriptomics and implementing a customized immunotherapeutic strategy could potentially improve survival outcomes for those facing this formidable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf00ce3ce13ab5e7ee7ff1ae10bf120f0be9305e" target='_blank'>
              Molecular biology of the deadliest cancer – glioblastoma: what do we know?
              </a>
            </td>
          <td>
            Aly Ismailov, Aldo Spallone, Alexey Belogurov, Alan Herbert, M. Poptsova
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Ongoing mutagenesis in cancer drives genetic diversity throughout the natural history of cancers. As the activities of mutational processes are dynamic throughout evolution, distinguishing the mutational signatures of ‘active’ and ‘historical’ processes has important implications for studying how tumors evolve. This can aid in understanding mutagenic states at the time of presentation, and in associating active mutational process with therapeutic resistance. As bulk sequencing primarily captures historical mutational processes, we studied whether ultra-low-coverage single-cell whole-genome sequencing (scWGS), which measures the distribution of mutations across hundreds or thousands of individual cells, could enable the distinction between historical and active mutational processes. While technical challenges and data sparsity have limited mutation analysis in scWGS, we show that these data contain valuable information about dynamic mutational processes. To robustly interpret single nucleotide variants (SNVs) in scWGS, we introduce ArtiCull, a method to identify and remove SNV artifacts by leveraging evolutionary constraints, enabling reliable detection of mutations for signature analysis. Applying this approach to scWGS data from pancreatic ductal adenocarcinoma (PDAC), triple-negative breast cancer (TNBC), and high-grade serous ovarian cancer (HGSOC), we uncover temporal and spatial patterns in mutational processes. In PDAC, we observe a temporal increase in mismatch repair deficiency (MMRd). In cisplatin-treated TNBC patient-derived xenografts, we identify therapy-induced mutagenesis and inactivation of APOBEC3 activity. In HGSOC, we show distinct patterns of APOBEC3 mutagenesis, including late tumor-wide activation in one case and clade-specific enrichment in another. Additionally, we detect a clone-specific increase in SBS17 activity, in a clone previously linked to recurrence. Our findings establish ultra-low-coverage scWGS as a powerful approach for studying active mutational processes that may influence ongoing clonal evolution and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6d1d9f289244977b2dfb8fbeaff86f943a227d" target='_blank'>
              Inferring active mutational processes in cancer using single cell sequencing and evolutionary constraints
              </a>
            </td>
          <td>
            Gryte Satas, Matthew A. Myers, Andrew McPherson, Sohrab P. Shah
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Despite remarkable advancements in therapeutic strategies, a considerable proportion of patients with bladder cancer (BC) still experience disease progression and unfavorable prognosis. The heterogeneity and biological functions of tumor endothelial cells (ECs) during BC progression remain poorly understood. We collected scRNA-seq data from BC samples and identified two EC subpopulations through hierarchical clustering analysis. The activity of signaling pathways in distinct EC subpopulations was assessed utilizing AUCell analysis. Gene regulatory networks (GRN) were constructed and analyzed for different EC subpopulations using the pySCENIC algorithm. Additionally, we investigated the association between the abundance of EC subpopulations and both clinical prognosis and immune cell infiltration. The biological effects of ESM1 protein on BC cells were further validated through EdU and Transwell assays. We analyzed 7,519 CD45-negative single cells from BC tissues and discerned two distinct EC subpopulations. The two subpopulations were characterized by high expression of ESM1 (S1 ECs) and CXCL2 (S2 ECs), respectively. In S1 ECs, we observed significant activation of signaling pathways involved in tumor promotion, including angiogenesis and cell proliferation. Additionally, our GRN analysis uncovered notable differences in transcription factor activity between S1 and S2 ECs. Moreover, ESM1 protein promoted proliferation and migration of BC cells. Patients with higher abundance of the S1 EC subpopulation exhibited more unfavorable clinical outcomes and increased infiltration of inhibitory immune cells. Our findings elucidate the transcriptional profiles and biological roles of the high ESM1-expression endothelial cell subpopulation in BC. This subpopulation is associated with poor prognosis and immunosuppressive tumor microenvironment. Accordingly, targeting endothelial cells with high ESM1 expression may offer a novel therapeutic strategy for patients with BC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95731-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4283029ca72adeb8c04b30a9f72593fd7af08134" target='_blank'>
              Single-cell sequencing uncovers a high ESM1-expression endothelial cell subpopulation associated with bladder cancer progression and the immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Yifeng Qiu, Yuhan Wang, Jiahe Liu, Baohua Liu, Kai Sun, Qi Hou
          </td>
          <td>2025-03-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive tumor with high metastatic potential which leads to decreased survival rate and resistance to chemotherapy and immunotherapy. Nearly 90% of pancreatic cancer comprises pancreatic ductal adenocarcinoma (PDAC). About 80% of diagnoses takes place at the advanced metastatic stage when it is unresectable, which renders chemotherapy regimens ineffective. There is also a dearth of specific biomarkers for early-stage detection. Advances in next generation sequencing and single cell profiling have identified molecular alterations and signatures that play a role in PDAC progression and subtype plasticity. Most chemotherapy regimens have shown only modest survival benefits, and therefore, translational approaches for immunotherapies and combination therapies are urgently required. In this review, we have examined the immunosuppressive and dense stromal network of tumor immune microenvironment with various metabolic and transcriptional changes that underlie the pro-tumorigenic properties in PDAC in terms of phenotypic heterogeneity, plasticity and subtype co-existence. Moreover, the stromal heterogeneity as well as genetic and epigenetic changes that impact PDAC development is discussed. We also review the PDAC interaction with sequestered cellular and humoral components present in the tumor immune microenvironment that modify the outcome of chemotherapy and radiation therapy. Finally, we discuss different therapeutic interventions targeting the tumor immune microenvironment aimed at better prognosis and improved survival in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e7cd5ce498aa2efbf04ec631daa67571857bc7" target='_blank'>
              Phenotypic heterogeneity and tumor immune microenvironment directed therapeutic strategies in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Remya P. G. Ramesh, Hadida Yasmin, Pretty Ponnachan, Basel Al-Ramadi, Uday Kishore, Ann Mary Joseph
          </td>
          <td>2025-03-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is among the most aggressive cancers, with rising incidence and limited responsiveness to immunotherapy due to its highly suppressive tumor microenvironment (TME). Tertiary lymphoid structures (TLS), ectopic formation structures of immune cells, are linked to better prognosis and improved immunotherapy responses in PDAC. Understanding TLS’s role in PDAC could enhance immunotherapy effectiveness. Methods This study integrated transcriptomic and clinical data from 310 PDAC patients in GEO database. We performed consensus clustering using tumor-associated TLS (TA-TLS) genes, identifying three distinct molecular subtypes. Single-sample gene set enrichment analysis (ssGSEA) was then employed to calculate a TLS score for each patient, allowing for TLS-based evaluation. Key prognostic genes were identified using an iterative LASSO method, leading to the construction of a risk assessment model, which was validated across independent cohorts. We further analyzed the TLS score using single-cell RNA sequencing (scRNA-seq), visualized key gene expression, and validated protein expression through immunohistochemistry (IHC). Additionally, we explored the effects of DNASE1L3 on cell proliferation and migration, and its immune-related functions using Gene Set Enrichment Analysis (GSEA) and multiplex cytokine analysis. Results Consensus clustering revealed three PDAC molecular subtypes with significant differences in prognosis, TA-TLS gene expression, and TME features. The TLS score effectively stratified patients into high and low groups, correlating with survival outcomes and TME characteristics. Our risk model, validated across cohorts, reliably predicted patient outcomes. Validation studies showed lower expression of DNASE1L3 and IL33 in tumor tissues. scRNA-seq confirmed TLS score associations with immune cells. DNASE1L3 overexpression inhibited PDAC cell proliferation and migration, with cytokine analysis indicating increased immune activity. Conclusions This study elucidated the expression profile of TA-TLS genes in PDAC, constructed a TLS gene-based scoring system, and developed a related risk model. We also explored the functions and potential antitumor mechanisms of key genes, providing evidence and new insights for enhancing TLS-targeted immunotherapy strategies in PDAC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06152-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ad4f13e415f9357404f99d30b4979256aef4341" target='_blank'>
              Prognostic implications and characterization of tumor-associated tertiary lymphoid structures genes in pancreatic cancer
              </a>
            </td>
          <td>
            Enkui Zhang, Yongsu Ma, Zonghao Liu, Jixin Zhang, Weikang Liu, Yiran Chen, Guangnian Liu, Xinxin Liu, Fusheng Zhang, Yu Zhu, Yinmo Yang, Xiaodong Tian
          </td>
          <td>2025-03-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLS) in the tumor microenvironment (TME) are emerging solid-tumor indicators of prognosis and response to immunotherapy. Considering that tumorigenesis requires metabolic reprogramming and subsequent TME remodeling, the discovery of TLS metabolic regulators is expected to produce immunotherapeutic targets. To identify such metabolic regulators, we constructed a metabolism-focused sgRNA library and performed an in vivo CRISPR screening in an orthotopic lung tumor mouse model. Combined with The Cancer Genome Atlas database analysis of TLS-related metabolic hub genes, we found that the loss of Acat1 in tumor cells sensitized tumors to anti-PD1 treatment, accompanied by increased TLS in the TME. Mechanistic studies revealed that ACAT1 resulted in mitochondrial protein hypersuccinylation in lung tumor cells and subsequently enhanced mitochondrial oxidative metabolism, which impeded TLS formation. Elimination of ROS by NAC or Acat1 knockdown promoted B cell aggregation and TLS construction. Consistently, data from tissue microassays of 305 patients with lung cancer showed that TLS were more abundant in non–small cell lung cancer (NSCLC) tissues with lower ACAT1 expression. Intratumoral ACAT1 expression was associated with poor immunotherapy outcomes in patients with NSCLC. In conclusion, our results identified ACAT1 as a metabolic regulator of TLS and a promising immunotherapeutic target in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f530a902aa9799db82e12bb11d7f979721376b" target='_blank'>
              ACAT1 regulates tertiary lymphoid structures and correlates with immunotherapy response in non–small cell lung cancer
              </a>
            </td>
          <td>
            Mengxia Jiao, Yifan Guo, Hongyu Zhang, Haoyu Wen, Peng Chen, Zhiqiang Wang, Baichao Yu, Kameina Zhuma, Yuchen Zhang, Jingbo Qie, Yun Xing, P. Zhao, Zihe Pan, Luman Wang, Dan Zhang, Fei Li, Yijiu Ren, Chang Chen, Yiwei Chu, J. Gu, Ronghua Liu
          </td>
          <td>2025-04-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Immune checkpoint inhibitors are an effective adjuvant therapy for non-small cell lung cancer (NSCLC). Recent studies have highlighted the critical role of tumor-infiltrating B cells in tumor immunity. However, research specifically focusing on B cells in NSCLC is limited. This study aims to elucidate the role of POU2F2+ B cells in patient survival and immune cell infiltration in NSCLC. Pseudotime analysis was performed to identify B cell pseudotime-related gene sets from two single-cell RNA sequencing (scRNA-seq) datasets of NSCLC. Differentially expressed genes (DEGs) were identified from two NSCLC immunotherapy-related bulk RNA sequencing datasets. A Venn diagram was used to determine core genes shared between these datasets. Kaplan-Meier survival curves were utilized to analyze overall survival (OS). Gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analyses were performed based on the differential genes between POU2F2+ and POU2F2− B cells. CIBERSORT analysis was conducted to compare the proportions of immune cell subpopulations between groups. Multiplex immunohistochemistry (mIHC) was used to localize POU2F2+ cells and measure distances between different immune cells. Three hallmark genes, POU2F2, CD2, and CST7, were identified as being associated with B cell maturation and immunotherapy efficacy in NSCLC. High expression of POU2F2 was associated with poorer OS in both LUAD and LUSC. However, the POU2F2+ B cell score specifically correlated with the OS of LUAD but not with LUSC. Further analysis using scRNA-seq and mIHC methods revealed that POU2F2 is predominantly expressed in B cells. In LUAD tumor tissues, POU2F2+ CD20+ B cells were spatially further from PD-1+ CD8+ T cells and CD206+ CD68+ macrophages compared to POU2F2− CD20+ B cells. In LUSC tumor tissues, POU2F2+ CD20+ B cells were spatially further from CD206+ CD68+ macrophages but showed no significant spatial difference from PD-1+ CD8+ T cells compared to POU2F2− CD20+ B cells. In patients with high POU2F2+ B cell scores, LUAD tissues showed an increased proportion of CD8+ T cells and M1 macrophages, and a decreased proportion of M2 macrophages. In contrast, in LUSC tissues, a high POU2F2+ B cell score was associated only with an increased proportion of M1 macrophages, with no significant differences in the proportions of CD8+ T cells or M2 macrophages between groups. This study elucidates the significant role of POU2F2+ B cells in influencing survival and immune cell infiltration in NSCLC. Our findings highlight POU2F2 as a novel target for NSCLC immunotherapy. Targeting POU2F2 may modulate the tumor immune microenvironment, enhance the infiltration and activity of critical immune cells, and ultimately improve patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cefe59fccb057643920602734c4dd5a6a512f160" target='_blank'>
              POU2F2+ B cells enhance antitumor immunity and predict better survival in non small cell lung cancer
              </a>
            </td>
          <td>
            Hengchuan Shi, Wenqing Wang, Jun Luo, Guoxin Song, Rongbo Han
          </td>
          <td>2025-02-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdedfede7608cb2fb7f1fa7b1c417998bb4fd21" target='_blank'>
              Spatially resolved transcriptomic analysis of the adult human prostate.
              </a>
            </td>
          <td>
            Junyi Hu, Fei Liu, Jing Zhang, Lei Yin, Wanli Cao, Weidong Xu, Yifan Chang, Ye Wang, Jian Wang, Yaxin Hou, Lilong Liu, Sujun Chen, Guanghui Zhu, Junhui Jiang, Zixian Wang, Gong-Hong Wei, H. H. He, Di Gu, Ke Chen, Shancheng Ren
          </td>
          <td>2025-04-01</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba54436455eb8d5189fb45709f89284b938e52f3" target='_blank'>
              Abstract B099: Deletion of Piezo1 in CD11b+ Dendritic Cells Amplifies the Local and Systemic Rhabdomyosarcoma Anti-Tumor Response
              </a>
            </td>
          <td>
            Melissa Bonner, David Askew, S. Eid, Suzanne L Tomchuck, muta Abiff, Jay T. Myers, Justin Garyu, A. Huang
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Purpose Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare and lethal cancer lacking effective treatment. Its genomic mutations and tumor microenvironment need further exploration. Methods We performed whole-exome sequencing or gene panel test to explore the SMARCA4 mutation spectrum in SCCOHT (15 samples). Single-cell RNA sequencing was conducted on one primary lesion with matched normal ovarian tissue and one recurrent lesion to investigate the intra-tumoral heterogeneity and immune microenvironment. Multiplex immunofluorescence staining validated T cell infiltration and PD-1 expression. Results 13/15 (86.7%) patients harbored SMARCA4 mutations. The loss of heterozygosity (LOH) occurred in 10/15 (66.7%) patients. Cancer cells and immune cells were observed in SCCOHT tumors. Cancer cells were further divided into seven subtypes and one from recurrent lesion exhibited the highest stemness accompanied by high expression of genes related to cell mitosis (AURKB, CHEK2, CCNB1, WEE1), DNA repair (BRCA1, RAD51) and epigenetic (EZH2, DNMT1). Immune cells mainly included macrophages and T cells. Lipid-associated tumor-associated macrophages (TAMs) was mainly in primary lesion while inflammatory cytokine-enriched TAMs in recurrent lesion. CD4+/ CD8+ T cell infiltration was observed in SCCOHT tumor and a certain proportion of T cells expressed PD-1. Conclusions SCCOHT exhibits universal SMARCA4 LOH and significant intra-tumoral heterogeneity, suggesting potential therapeutic targets, including CHEK2, CCNB1, and WEE1. Exhausted T cells and distinct TAM subsets infiltrate tumors. Targeting macrophage polarization or cytokine signaling may also be promising. These findings provide insights for developing novel therapies to improve outcomes in SCCOHT. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01649-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdba27a2b17d1b242308fa85ab562553932c973d" target='_blank'>
              Single-cell RNA sequencing reveals the intra-tumoral heterogeneity and immune microenvironment of small cell carcinoma of the ovary, hypercalcemic type
              </a>
            </td>
          <td>
            Yijun Gao, Kewei Zheng, Haowen Tan, Mingyi Kang, Bingjian Lu, Ling Chen, Jing Xu, Chong Lu, Ranran Chai, Congjian Xu, Yu Kang
          </td>
          <td>2025-04-09</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d026aba47659bcc0e46d8766e11f46d409c8da83" target='_blank'>
              Multidimensional Classification Framework for Human Breast Cancer Cell Lines
              </a>
            </td>
          <td>
            Diogo Dias, Catarina Franco Jones, Ana Catarina Moreira, Gil Gonccalves, Mustafa B. A. Djamgoz, Frederico Castelo Ferreira, Paola Sanjuan‐Alberte, Rosalia Moreddu
          </td>
          <td>2025-02-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6e0f5dce3feb58cd819120465c413a2814ac1e2" target='_blank'>
              Path2Omics: Enhanced transcriptomic and methylation prediction accuracy from tumor histopathology
              </a>
            </td>
          <td>
            Danh-Tai Hoang, E. Shulman, S. R. Dhruba, N. Nair, R. K. Barman, H. Lalchungnunga, Omkar Singh, MacLean P Nasrallah, Eric A Stone, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lymphatic, nervous and tumor tissues exhibit complex physiology arising from three-dimensional interactions within genetically unique microenvironments. Here we develop a technology capable of volumetrically imaging transcriptomes, genotypes and morphologies in a single measurement, without relying on prior knowledge of spatial organization or genetic sequences. Our method extends DNA microscopy into three dimensions at scales involving 107 molecules by forming a distributed intermolecular network of proximal unique DNA barcodes tagging complementary DNA molecules inside the specimen. After sequencing the DNA-encoded network, an image of molecular positions is inferred using geodesic spectral embeddings, a dimensionality reduction approach that we show to be especially suitable for this data-inverse problem. Applying whole-transcriptome volumetric DNA microscopy to intact zebrafish embryos, we demonstrate that three-dimensional image inference recapitulates zebrafish morphology and known gene expression patterns, capturing the spatial organization of gene sequences. Our extension of spatial genetic measurements to three dimensions, independent of prior templates, opens the door to detailed joint resolution of genomics and morphology in biological tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5dec4ebf3cc809cc6c6343505394a1e50171da" target='_blank'>
              Spatial transcriptomic imaging of an intact organism using volumetric DNA microscopy.
              </a>
            </td>
          <td>
            Nianchao Qian, Joshua A Weinstein
          </td>
          <td>2025-03-27</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccad37f8db740ee741e31d76d75cb5a07f7e923f" target='_blank'>
              Biology as vulnerability in follicular lymphoma: genetics, epigenetics, and immunogenetics.
              </a>
            </td>
          <td>
            F. Araujo-Ayala, W. Béguelin
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ea1b0559ba72f593b3a6674f424957c7f5230c" target='_blank'>
              Abstract B016: Genomic features of the tumor microenvironment and antigen presentation machinery vary by clinical and molecular subtypes of breast cancer
              </a>
            </td>
          <td>
            Constandina E. O'Connell, Yoon Jin Cha, Alexander V Lobanov, Cheng Fan, Brooke M. Felsheim, A. Fernandez-Martinez, C. Perou
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>163</td>
        </tr>

        <tr id="Abstract Breast cancer remains a significant global health challenge due to its complexity, which arises from multiple genetic and epigenetic mutations that originate in normal breast tissue. Traditional machine learning models often fall short in addressing the intricate gene interactions that complicate drug design and treatment strategies. In contrast, our study introduces GEMDiff, a novel computational workflow leveraging a diffusion model to bridge the gene expression states between normal and tumor conditions. GEMDiff augments RNAseq data and simulates perturbation transformations between normal and tumor gene states, enhancing biomarker identification. GEMDiff can handle large-scale gene expression data without succumbing to the scalability and stability issues that plague other generative models. By avoiding the need for task-specific hyper-parameter tuning and specific loss functions, GEMDiff can be generalized across various tasks, making it a robust tool for gene expression analysis. The model’s ability to augment RNA-seq data and simulate gene perturbations provides a valuable tool for researchers. This capability can be used to generate synthetic data for training other machine learning models, thereby addressing the issue of limited biological data and enhancing the performance of predictive models. The effectiveness of GEMDiff is demonstrated through a case study using breast mRNA gene expression data, identifying 307 core genes involved in the transition from a breast tumor to a normal gene expression state. GEMDiff is open source and available at https://github.com/xai990/GEMDiff.git under the MIT license.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3472d3e5ad8a58f2f1d0054cb37345e502bfbbc" target='_blank'>
              GEMDiff: a diffusion workflow bridges between normal and tumor gene expression states: a breast cancer case study
              </a>
            </td>
          <td>
            Xusheng Ai, Melissa C. Smith, F. A. Feltus
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be659580caffb1fbf7776c3ae3608aa7fdd87091" target='_blank'>
              Tumor microenvironment: recent advances in understanding and its role in modulating cancer therapies.
              </a>
            </td>
          <td>
            Disha D Shah, Mehul R. Chorawala, Neha R. Raghani, Rajanikant Patel, Mohammad Fareed, V. Kashid, Bhupendra G. Prajapati
          </td>
          <td>2025-03-18</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract To unravel the mechanism of immune activation and suppression within tumors, a critical step is to identify transcriptional signals governing cell–cell communication between tumor and immune/stromal cells in the tumor microenvironment. Central to this communication are interactions between secreted ligands and cell-surface receptors, creating a highly connected signaling network among cells. Recent advancements in in situ-omics profiling, particularly spatial transcriptomic (ST) technology, provide unique opportunities to directly characterize ligand–receptor signaling networks that power cell–cell communication. In this paper, we propose a novel statistical method, LRnetST, to characterize the ligand–receptor interaction networks between adjacent tumor and immune/stroma cells based on ST data. LRnetST utilizes a directed acyclic graph model with a novel approach to handle the zero-inflated distributions of ST data. It also leverages existing ligand–receptor regulation databases as prior information, and employs a bootstrap aggregation strategy to achieve robust network estimation. Application of LRnetST to ST data of high-grade serous ovarian tumor samples revealed both common and distinct ligand–receptor regulations across different tumors. Some of these interactions were validated through both a MERFISH dataset and a CosMx SMI dataset of independent ovarian tumor samples. These results cast light on biological processes relating to the communication between tumor and immune/stromal cells in ovarian tumors. An open-source R package of LRnetST is available on GitHub at https://github.com/jie108/LRnetST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db23b179c860dc1d8c3febc0dd28265cf6aa4993" target='_blank'>
              Learning directed acyclic graphs for ligands and receptors based on spatially resolved transcriptomic data of ovarian cancer
              </a>
            </td>
          <td>
            Shrabanti Chowdhury, S. Ferri-Borgogno, Peng Yang, Wenyi Wang, Jie Peng, Samuel C. Mok, Pei Wang
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="CD8+ tissue-resident memory T cells (CD8+TRM), expressing PD-1 and/or TIM-3, are linked to immunological surveillance in non-small cell lung cancer (NSCLC). However, their prognostic value across activation states and spatial distributions in NSCLC stages is unclear. We analyzed 271 NSCLC patients’ primary tumors and lymph nodes, using multiplex immunohistochemistry and inForm software for cell identification. Statistical analyses included the Mann-Whitney U test and Cox survival analysis. Findings showed CD8+TRM were categorized into four activation states. In locally advanced NSCLC, PD-1−TIM-3+CD8+TRM3, and PD-1+TIM-3+CD8+TRM4 densities were notably higher at invasive margins. Fewer interactions between CD8+TRM and tumor cells were observed in advanced lesions. Decreased PD-1+TIM-3−CD8+TRM2 interactions with tumor cells and increased PD-1+TIM-3+CD8+TRM4 interactions with tumor cells were independently associated with recurrence in patients with early lung adenocarcinoma and squamous carcinoma, respectively. These results suggest that CD8+TRM activation state and distribution are linked to recurrence risk in early-stage NSCLC, emphasizing the importance of CD8+TRM spatial dynamics in prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8927993683d672fa19c819fd8f766d987dddb21" target='_blank'>
              Stage-dependent spatial distribution and prognostic value of CD8+ tissue-resident memory T cells in NSCLC
              </a>
            </td>
          <td>
            Liying Yang, Hao Yang, Miaoqing Zhao, Hongtu Yuan, Jiaxiao Geng, Yushan Yan, Li Wu, Ligang Xing, Jinming Yu, Xiaorong Sun
          </td>
          <td>2025-02-22</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f05272abf51c9f38dd50672d47db4690dbc1bb3" target='_blank'>
              Spatial Profiling Identifies Regionally Distinct Microenvironments and Targetable Immunosuppressive Mechanisms in Pediatric Osteosarcoma Pulmonary Metastases.
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice S Pereira, Demond Williams, Douglass W Morris-Ii, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael C. Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-04-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e89dc2d087d9d2c969c5bb0a4b45e01fc9cc0f7a" target='_blank'>
              Single-cell and spatial genomic landscape of non-small cell lung cancer brain metastases.
              </a>
            </td>
          <td>
            Somnath Tagore, Lindsay A. Caprio, Amit Dipak Amin, K. Bestak, Karan Luthria, Edridge K. D’Souza, Irving Barrera, Johannes C. Melms, Sharon Wu, Sinan Abuzaid, Yiping Wang, Viktoria Jakubikova, Peter Koch, D. Z. Brodtman, Banpreet Bawa, S. Deshmukh, Leon Ebel, Miguel A. Ibarra-Arellano, A. Jaiswal, Carino Gurjao, Jana Biermann, Neha Shaikh, Priyanka Ramaradj, Yohanna Georgis, Galina G Lagos, Matthew I Ehrlich, P. Ho, Zachary H. Walsh, Meri Rogava, Michelle Garlin Politis, Devanik Biswas, A. Cottarelli, Nikhil Rizvi, CA Shu, Benjamin O Herzberg, N. Anandasabapathy, G. Sledge, Emmanuel Zorn, Peter D Canoll, Jeffrey N. Bruce, Naiyer A Rizvi, Alison M Taylor, Anjali Saqi, H. Hibshoosh, Gary K Schwartz, Brian S Henick, Fei Chen, Denis Schapiro, Parin Shah, Benjamin Izar
          </td>
          <td>2025-02-27</td>
          <td>Nature medicine</td>
          <td>1</td>
          <td>60</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a highly aggressive brain tumor associated with poor survival outcomes and is driven by a complex tumor microenvironment (TME) that promotes tumor progression and treatment resistance. To explore the role of the TME in GBM, we analyzed glioma-related microarray and single-cell RNA sequencing (scRNA-seq) datasets from the Gene Expression Omnibus (GEO). Functional enrichment and weighted gene coexpression network analyses revealed distinct immune profiles, metabolic alterations, and differences in chemotherapeutic drug sensitivity between the high-risk and low-risk patient groups. scRNA-seq data processed with the ‘Seurat’ package were used to identify differentially expressed genes in pericytes, endothelial cells, and glioma cells, particularly those involved in extracellular matrix (ECM) remodeling. A 17-gene prognostic signature developed through Cox regression and LASSO analyses revealed that key genes (COL1A1, COL4A1, and VIM) were significantly associated with survival outcomes in GBM patients. Drug sensitivity analyses using data from the Genomics of Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) identified potential targeted therapies for GBM, including SB-505,124, staurosporine, and AZD8186. This integrative study underscores the critical roles of the ECM and synaptic remodeling in GBM and suggests novel therapeutic targets to improve personalized treatment strategies for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/456759f2943a08a601ac9cd5e20189bf3d373750" target='_blank'>
              Molecular mechanisms and therapeutic targets in glioblastoma multiforme: network and single-cell analyses
              </a>
            </td>
          <td>
            Xiangyu Chen, Xiao Zhong, Feifei Zhang, Xiaomei Zhou, Xiaofeng Yue, Xueru Li
          </td>
          <td>2025-03-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f6763865a99a9f23291b576c73a041b68eb0a51" target='_blank'>
              Bioengineering facets of the tumor microenvironment in 3D tumor models: insights into cellular, biophysical and biochemical interactions.
              </a>
            </td>
          <td>
            Salma T. Rafik, D. Bakkalci, Alexander J MacRobert, Umber Cheema
          </td>
          <td>2025-03-27</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Coupling single-cell transcriptomics with tumor-initiating cell enrichment identified IFN response gene expression not previously reported in bulk RNA-sequencing–derived signatures and proposed IFN/STAT1 signaling as a candidate therapeutic target in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49032c731c7f43ad3396d95549c8d805e477fd3" target='_blank'>
              Lineage Tracing and Single-Cell RNA Sequencing Reveal a Common Transcriptional State in Breast Cancer Tumor–Initiating Cells Characterized by IFN/STAT1 Activity
              </a>
            </td>
          <td>
            Eric P Souto, Ping Gong, John D. Landua, R. Srinivasan, Abhinaya Ganesan, L. Dobrolecki, S. C. Purdy, Xingxin Pan, Michael Zeosky, Anna Chung, , H. Ford, Michael T. Lewis, Connie Cepko, Dr. Kenneth L. Scott, Dr. Michael Davidson
          </td>
          <td>2025-02-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Rationale: Glioblastoma multiforme (GBM) is the most aggressive primary malignant brain tumor in adults, characterized by high invasiveness and poor prognosis. Glioma stem cells (GSCs) drive GBM treatment resistance and recurrence, however, the molecular mechanisms activating intracranial GSCs remain unclear. Extracellular vesicles (EVs) are crucial signaling mediators in regulating cell metabolism and can cross the blood-brain barrier (BBB). This study aimed to elucidate how EV cargo contributes to the intracranial GSC state and validate a non-invasive diagnostic strategy for GBM relapse. Methods: We isolated plasma extracellular vesicles (pl-EVs) from three groups: recurrent GBM patients post-resection, non-recurrent GBM patients post-resection, and healthy individuals. Newly diagnosed GBM patients served as an additional control. EVs were characterized and co-cultured with primary GBM cell lines to assess their effect on tumor stemness. EV cargo was analyzed using proteomics to investigate specific EV subpopulations contributing to GBM relapse. Based on these findings, we generated engineered LDHA-enriched EVs (LDHA-EVs) and co-cultured them with patient-derived organoids (PDOs). Metabolomics was performed to elucidate the underlying signal transduction pathways. Results: Our study demonstrated that pl-EVs from recurrent GBM patients enhanced aerobic glycolysis and stemness in GBM cells. Proteomic analysis revealed that plasma EVs from recurrent GBMs encapsulated considerable amounts of the enzyme lactate dehydrogenase A (LDHA). Mechanistically, LDHA-loaded EVs promoted glycolysis, induced cAMP/ATP cycling, and accelerated lactate production, thereby maintained the GSC phenotype. Concurrently, post-surgical therapy-induced stress-modulated hypoxia in residual tumors, promoted LDHA-enriched EV release. Clinically, high levels of circulating LDHA-positive EVs correlated with increased glycolysis, poor therapeutic response, and shorter survival in recurrent GBM patients. Conclusion: Our study highlights LDHA-loaded EVs as key mediators promoting GSC properties and metabolic reprogramming in GBM. These findings provide insights into recurrence mechanisms and suggest potential liquid biopsy approaches for monitoring and preventing GBM relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ce99f37ce269242ab30f9fdf3e7b9861d3b1397" target='_blank'>
              Plasma extracellular vesicles from recurrent GBMs carrying LDHA to activate glioblastoma stemness by enhancing glycolysis
              </a>
            </td>
          <td>
            Xin Zhang, JunJie Li, Yiyao Huang, Anming Yang, Xiaoliu Liu, Yunhao Luo, Hao Tian, Minghui Wen, Chengzong Zhong, Bin Peng, Haitao Sun, Lei Zheng
          </td>
          <td>2025-02-26</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Natural killer (NK) cells are innate immune effectors with considerable heterogeneity and potent antitumor capabilities. Intraepithelial ILC1 (ieILC1)-like NK cells, a population of cytotoxic tissue-resident innate lymphoid cells, have recently been documented in the microenvironment of head and neck squamous cell carcinomas (HNSCC) and other solid tumors. These cells have antitumor cytolytic potential and are potent producers of type 1 cytokines, including IFNγ. Here, we identify a subpopulation of ex vivo differentiated ieILC1-like NK cells that produce IL-13 upon stimulation. Functional characterization revealed that these cells co-expressed IFNγ and IL-13 while maintaining an ILC1 transcriptional signature. IL-13 was induced either upon co-culture with tumor cell lines, or in response to TGF-β and IL-15. IL-13-expressing ieILC1-like NK cells were identified among tumor infiltrating lymphocytes expanded from patient HNSCC tumors, in support of their in vivoexistence in primary tumors. These data demonstrate additional heterogeneity within the ieILC1-like NK cell population than previously appreciated and highlight a unique form of ILC plasticity in which cells with clear ILC1 transcriptional profiles express a type 2 cytokine. With the known roles of IL-13 in cancer cell growth dynamics and immunoregulation, the identification of this subset within tumor microenvironments presents a potential target for therapeutic manipulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e51298e58c3205c5f813648a5327f59f26b1231" target='_blank'>
              An intraepithelial ILC1-like natural killer cell subset produces IL-13
              </a>
            </td>
          <td>
            Sainiteesh Maddineni, Krishna Sharma, Imran A. Mohammad, Alistaire D. Ruggiero-Sherman, Ivan Stepanek, JuneHo Shin, Jennifer K. Bando, John B. Sunwoo
          </td>
          <td>2025-03-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="High-grade lung neuroendocrine carcinomas (Lu-NECs) are clinically refractory malignancies with poor prognosis and limited therapeutic advances. The biological and molecular features underlying the histological heterogeneity of Lu-NECs are not fully understood. In this study, we present a multi-omics integration of whole-exome sequencing and deep proteomic profiling in 93 Chinese Lu-NECs to establish the first comprehensive proteogenomic atlas of this disease spectrum. Our analyses revealed a high degree of mutational concordance among the subtypes at the genomic level; however, distinct proteomic profiles enabled a clear differentiation of histological subtypes, unveiling subtype-specific molecular and biological features related to tumor metabolism, immunity, and proliferation. Furthermore, RB1 mutations confer divergent prognostic effects through subtype-specific cis- and trans-proteomic regulation. In addition, we identified potential protein biomarkers for histological subtype classification and risk stratification, which were validated by immunohistochemistry in an independent cohort. This study provides a valuable proteogenomic resource and insight into Lu-NEC heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7135f4258e94682deb014b1c14f86f0e470585bf" target='_blank'>
              Proteogenomic Characterization of High-Grade Lung Neuroendocrine Carcinoma Deciphers Molecular Diversity and Potential Biomarkers of Different Histological Subtypes in Chinese Population
              </a>
            </td>
          <td>
            Zicheng Zhang, Xi Wu, S. Bao, Xu-min Sun, Fan Yang, Yibo Zhang, Zijian Yang, Liujin Zhang, Ruanqi Chen, P. Xing, Junling Li, meng zhou, Lin Yang
          </td>
          <td>2025-04-02</td>
          <td>Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and lethal malignancy, with a five-year survival rate of just 50% for cases of locally advanced disease. Chromosomal aberrations, particularly the deletion of the short arm of chromosome 3 (3p), have been strongly associated with poor prognosis and more aggressive tumor phenotypes. The tumor microenvironment (TME) plays a pivotal role in tumor progression and resistance to therapy. This study aims to elucidate the impact of 3p deletion on the TME, immune cell infiltration, and treatment response in HNSCC, to identify novel therapeutic targets to improve patient outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from the Gene Expression Omnibus (GEO) and bulk transcriptome data from The Cancer Genome Atlas (TCGA). Pseudo-time trajectory and cell-cell communication analyses were performed with the Monocle and CellChat packages. The Wilcoxon test was used to evaluate the differential gene expression between wild-type (wt) and mutant (mut) groups. Prognostic models were developed using the Least Absolute Shrinkage and Selection Operator (LASSO) algorithm and Cox regression analyses to find the genes related to survival, with survival analysis conducted via Kaplan-Meier curves. Gene set enrichment analysis (GSEA) was employed to investigate pathway dysregulation, and immune cell infiltration was assessed using various immune scoring methodologies to explore the differences immune environment. The Tumor Immune Dysfunction and Exclusion (TIDE) database was utilized to predict the potential efficacy of immune checkpoint inhibitors. mRNA and protein expression levels of SPP1 were examined by RT-qPCR and Western blotting, while cell proliferation was assessed using the CCK8 assay. The mut group demonstrated significant alterations in cellular composition, characterized by increased endothelial cells and macrophages and decreased fibroblasts and CD8 + T cells, indicative of an immunosuppressive TME. Differential expression analysis revealed downregulation of immune pathways, including antigen processing and presentation, T cell receptor signaling, and B cell receptor signaling pathways in the mut group, along with enhanced metabolic activity in glycolysis and lipid metabolism. The prognostic model identified nine key genes associated with poor survival in HNSCC. The mut group exhibited poorer overall survival and a more immunosuppressive microenvironment compared to the wt group, which correlated with the outcomes observed in high-risk versus low-risk groups. High-risk patients also showed a diminished response to immunotherapy compared to low-risk patients. Additionally, SPP1 emerged as a critical gene associated with chemotherapy resistance and macrophage M2 polarization. This study demonstrates that 3p deletion significantly reshapes the TME, contributing to poor prognosis in HNSCC by fostering an immunosuppressive environment and enhancing chemoresistance. These findings highlight the potential for developing targeted therapies that address the genetic and immunological landscape of HNSCC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92078-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Resistance to both naturally occurring anti-cancer immunity and to immunotherapy is common in patients with aggressive non-small cell lung cancer (NSCLC). Recent studies indicate a role of loss of the HLA class-I antigen presentation machinery (APM) protein β-2-microglobulin in acquired resistance to immune checkpoint blockers. However, the mechanisms, functional consequences and therapeutic potential of APM defects in NSCLC remain poorly understood. Methods Using multiplexed immunofluorescence, we spatially mapped CD8+ effector Tumor-Infiltrating Lymphocytes (TILs) and the APM components TAP1 and TAP2 in 819 baseline/pre-treatment NSCLCs from patients treated with and without PD-1 axis blockers in 4 independent cohorts. The impact of TAP1/2 silencing in lung cancer cells using siRNAs and CRISPR/Cas9 was studied using transcriptomic analysis, phosphoprotein arrays, ATAC-sequencing, measurement of surface HLA-peptide complexes and in vitro tumor-antigen specific T-cell killing. We established autologous co-cultures of tumor and immune cells from primary human NSCLCs to study the functional impact of IL4Rα and/or PD-1 blockade using monoclonal antibodies. A high-throughput drug screen supported the identification of compounds able to increase TAP2 expression in NSCLC cells. Results We identified cancer cell selective TAP2 protein downregulation in 42.4% of treatment naïve NSCLCs associated with reduced sensitivity to immune checkpoint blockers. TAP1 downregulation occurred in 24.4% of lung tumors without survival impact. Silencing of TAP2 in lung cancer cells altered key intracellular immunomodulatory pathways, limited sensitivity to proinflammatory cytokines, reduced the levels of surface peptide-HLA complexes and protected malignant cells from tumor antigen-specific T-cell killing via SOCS1 upregulation. TAP2 loss in human NSCLCs was associated with reduced TAP2 promoter chromatin accessibility and elevated IL-4 IL-4 expression. Treatment with IL-4 reduced TAP2 levels and the chromatin accessibility of the TAP2 gene promoter in NSCLC cells and reproduced all the functional consequences of TAP2 loss. In intact human NSCLC, IL-4 IL-4 transcripts were detected in intratumoral myeloid cells and IL-4Rα blockade increased human NSCLC cell killing by autologous TILs. Epigenetic modulators and other drugs with known anti-cancer activity increased TAP2 expression and its function in lung cancer cells. Conclusions Our study reveals previously unrecognized functions of TAP2 beyond antigen presentation and establishes a reversible multi-cellular axis mediating adaptive immune evasion and immunotherapy resistance with clinical potential. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02276-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7918ba938a1884ad0689076e01e7a1af58bcb50" target='_blank'>
              IL-4 mediated TAP2 downregulation is a dominant and reversible mechanism of immune evasion and immunotherapy resistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            Kishu Ranjan, Barani Kumar Rajendran, Imad ud Deen, Adrien Costantini, Miguel Lopez de Rodas, Shruti S Desai, Frankie Scallo, Nicole Gianino, S. Ferrone, K. Schalper
          </td>
          <td>2025-03-17</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Introduction Pancreatic adenocarcinoma (PAAD) is characterized by a profoundly immunosuppressive tumor microenvironment (TME) that limits the efficacy of immunotherapy. Emerging evidence suggests that tumor-specific metabolic reprogramming may drive disease progression and shape the immune landscape in PAAD. Methods We integrated multi-omics data from TCGA, GEO, and ICGC to identify key metabolism-related genes (MRGs) that influence immune cell infiltration, tumor progression, and patient survival. Based on nine pivotal MRGs (including ANLN, PKMYT1, and HMGA1), we developed and validated a novel metabolic-prognostic index (MPI). Functional enrichment analyses were conducted to elucidate the metabolic pathways associated with different MPI risk groups. In vitro experiments and drug sensitivity analyses were performed to confirm the oncogenic role of selected MRGs and to explore their therapeutic implications. Results The MPI effectively stratified patients into high- and low-risk groups. High-MPI scores correlated with poor overall survival, elevated tumor mutation burden (TMB), and an immunosuppressive TME, evidenced by reduced CD8⁺ T-cell infiltration and increased expression of immune checkpoints (PD-L1, TGF-β). Functional enrichment revealed glycolysis and folate biosynthesis as dominant pathways in high-MPI groups, whereas fatty acid metabolism prevailed in low-MPI groups. Experimental validation underscored the role of ANLN in promoting epithelial-mesenchymal transition (EMT) and immune evasion via NF-κB signaling. ANLN knockdown significantly reduced glycolytic activity, tumor cell migration, and immune evasion. Drug sensitivity analyses indicated resistance to gemcitabine but sensitivity to afatinib in high-MPI patients. Although TIDE analysis predicted immune checkpoint inhibitor (ICI) resistance in high-MPI tumors, a subset of patients showed favorable responses to anti-PD-L1 therapy. Discussion These findings provide a comprehensive framework for understanding how metabolic reprogramming shapes PAAD’s immunosuppressive TME and affects treatment outcomes. By accurately stratifying patients, the MPI serves as a promising tool to guide therapeutic decisions, including targeted therapy selection and immunotherapy prediction, ultimately offering potential for more personalized management of PAAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42608b06fe89e88fa2b364a076f0a8a5c2fa634d" target='_blank'>
              Metabolic reprogramming shapes the immune microenvironment in pancreatic adenocarcinoma: prognostic implications and therapeutic targets
              </a>
            </td>
          <td>
            Weihua Song, Yabin Yu, Siqi Wang, Zhengyi Cui, Qiusi Zhu, Wangrui Liu, Shiyin Wei, Jiachang Chi
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af060e4cc93259c4d19b62b9cc2136ad5699039" target='_blank'>
              ROICellTrack: A deep learning framework for integrating cellular imaging modalities in subcellular spatial transcriptomic profiling of tumor tissues.
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Morán-Segura, Hongzhi Xu, Tingyi Li, Joshua T Davis, A. Vosoughi, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2025-04-08</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24611fd8233fc299c26132a33dc4d39224b104e" target='_blank'>
              Single-Cell Multiomics Reveals Regulatory Mechanisms of CAR T Cell Persistence and Dysfunction in Multiple Myeloma
              </a>
            </td>
          <td>
            Lorea Jordana-Urriza, G. Serrano, Sergio Camara, M. E. Calleja-Cervantes, P. S. Martín-Uriz, A. Zabaleta, A. Oliver-Cáldes, M. Español-Rego, D. Alignani, Teresa Lozano, Saray Rodríguez-Díaz, Elena Iglesias, Valentin Cabañas, J. L. Reguera, V. González-Calle, M. Mateos, F. Sánchez-Guijo, Bruno Paiva, J.J. Lasarte, S. Inogés, A. López-Díaz de Cerio, A. González-Navarro, Manel Juan, C. Fernández de Larrea, Esteban Tamariz, A. Alfonso-Pierola, Paula Rodriguez-Otero, Jesus F San-Miguel, Mikel Hernaez, J. R. Rodríguez-Madoz, Felipe Prosper
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance. Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features. Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations. Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF-mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f05d313174898c817cd841b0d897dd18a802b1a" target='_blank'>
              [Drugs and cellular dynamics in tumor microenvironment using microphysiological systems].
              </a>
            </td>
          <td>
            Yuji Nashimoto
          </td>
          <td>2025-03-01</td>
          <td>Nihon yakurigaku zasshi. Folia pharmacologica Japonica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Radiation-induced skin injury (RISI) commonly manifests in cancer patients undergoing radiotherapy (RT). However, a universally accepted standard for treating radiation injury has not yet been established. Our objective was to provide a detailed molecular overview of skin pre- and post-radiation therapy, aiming to enhance our understanding of the subclusters and molecular mechanisms contributing to radiodermatitis. Methods C57BL/6 mice were subjected to a single fraction (20 Gy) of RT targeting the right dorsal skin. We then employed integrated single-cell RNA sequencing (scRNA-seq) to analyze skin samples from mice at 7 and 30 days after radiation exposure, as well as from non-irradiated mice. The Seurat analysis pipeline, Cellchat, SCP, and ssGSEA were used to define the cell types and mechanisms involved in radiation-induced skin injury. Reverse transcription polymerase chain reaction (RT-PCR), multiplex immunofluorescent staining, and other datasets (GSE130183, GSE193564, and GSE193807) were used to validate our findings. Results Thirty-two distinct cell clusters encompassing 71,412 cells were identified. We discovered that cycling keratinocytes (KCs), with the BMP signaling pathway enriched, could activate the Wnt pathway, as well as the SMAD pathways, driving the wound healing and fibrosis processes in RISI. Terminally differentiated secretory-papillary fibroblasts (Fibs) are capable of attracting immune cells, which contributes to the pathogenesis of RISI. Lymphatic endothelial cells (ECs) with pro-inflammatory properties play a critical role in the pathogenesis of RISI by facilitating leukocyte migration. Our analysis also highlighted enhanced ligand-receptor interactions, notably the interactions between chemokines like CXCL10, CCL2, and ACKR1, across subclusters of inflammatory KCs, Fibs, ECs, and immune cells, underscoring their pivotal role in leukocyte recruitment in RISI. Conclusions Cycling KCs, secretory-papillary Fibs, and lymphatic ECs play critical roles in RISI progression. Targeting the interactions of these subclusters with immune cells might help improve the severity of RISI. Furthermore, our study provides a valuable resource for understanding the interactions among immune cells in the context of RISI. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00596-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77637e1bbb3f686235eb0539c52ccaa5035493bf" target='_blank'>
              Molecular profiling of skin cells identifies distinct cellular signatures in radiation-induced skin injury across various stages in the murine dataset
              </a>
            </td>
          <td>
            Hongxuan Yu, Tao Zhong, Ying Xu, Zengfu Zhang, Jiachun Ma, Jupeng Yuan, Minglei Wang, Meng Wu, Jinming Yu, Yuequn Ma, Dawei Chen
          </td>
          <td>2025-02-25</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0c39aa857bbfd9543aa84644d12df58e1a71f0a" target='_blank'>
              Single-cell and spatial RNA sequencing identify divergent microenvironments and progression signatures in early- versus late-onset prostate cancer.
              </a>
            </td>
          <td>
            Yifei Cheng, Bingxin Liu, Junyi Xin, Xiaobin Wu, Wenchao Li, Jinwei Shang, Jiajin Wu, Zhengdong Zhang, Bin Xu, Mulong Du, Gong Cheng, Meilin Wang
          </td>
          <td>2025-04-10</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c654520970222f7cd3694a92d7a5cfe3581a7e0" target='_blank'>
              Identification of new biomarkers of hepatic cancer stem cells through proteomic profiling.
              </a>
            </td>
          <td>
            S. H. Choi, Ha Young Lee, Sung Ho Yun, Sung Jae Jang, Seung-Up Kim, Jun Yong Park, S. Ahn, D. Y. Kim
          </td>
          <td>2025-03-20</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background Canine cutaneous mast cell tumors (MCTs) are a common, yet clinically challenging tumor type given their variable biological behavior. Although patients with low grade MCTs can often be effectively managed with surgery alone, most dogs with high grade MCTs succumb to their disease despite multimodal therapy. An improved understanding of the immune tumor microenvironment (TME) may help identify novel prognostic and therapeutic targets. Methods In this study, we interrogated the immune transcriptional profiles of the TME in low and high grade MCTs, and quantified intratumoral T cells. Twelve client-owned dogs with MCTs (6 Kiupel low grade with clinically benign behavior and 6 Kiupel high grade with clinically aggressive behavior) that underwent curative-intent surgery were selected. Tumor grade was confirmed by a single veterinary pathologist. RNA was extracted from all tumors followed by immune transcriptional profiling utilizing the NanoString Canine IO panel and analysis using the ROSALIND platform. T cell density was determined by immunohistochemical staining for CD3 and quantified using ImageScope software (Leica Biosystems) following digital slide capture. Lymphocytic infiltrate was further characterized in the TME of one high grade MCT using co-immunofluorescence. Results Immune transcriptional profiling identified 9 differentially expressed genes between low and high grade MCTs (p-adj < 0.05). Programmed cell death protein 1 (PDCD1) and inducible T-cell costimulator ligand (ICOSLG) gene expression were significantly higher in a subset of high grade MCTs. ICOSLG expression positively correlated with T cell score (rs = 0.6434, p = 0.0278). Although the T cell density was not significantly different between low (mean of 76.42 CD3 + /mm2, SD 12 CD3 + /mm2) and high grade MCTs (mean of 129.1 CD3 + /mm2, SD 96.06 CD3 + /mm2), greater variation of T cell densities was observed across high grade MCTs compared to low grade (p = 0.0059). Immunofluorescence of one high grade MCT with marked T cell infiltration revealed organized aggregates of T and B cells consistent with tertiary lymphoid structures (TLS). Conclusions Our data revealed significant differences in the immune TME of low and high grade MCTs and provides rationale to further investigate potential prognostic and therapeutic roles of immune checkpoints in canine MCTs. Supplementary Information The online version contains supplementary material available at 10.1186/s44356-025-00020-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b795d410565b7a2d389acbb8118bd3306a43b7c" target='_blank'>
              Intertumoral heterogeneity of the immune microenvironment in high grade canine mast cell tumors
              </a>
            </td>
          <td>
            K. L. Bardales, L. Jiang, E. Radaelli, C. Assenmacher, J. A. Lenz, M. Atherton
          </td>
          <td>2025-03-14</td>
          <td>Veterinary Oncology (London, England)</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7be0faff35f4a01beaf26b8c36afaa6cabdce2a5" target='_blank'>
              Repetitive Confined Migration Leads to an Increase in Clonogenic Activity and Chemoresistanse of Human Non-Small Cell Lung Cancer Cells, Regardless of Their Initial Chemo- and Radiosensitivity
              </a>
            </td>
          <td>
            P. Malakhov, M. V. Pustovalova, A.V. Aleksandrova, E.G. Kontareva, A.V. Smirnova, Z. Nofal, A.N. Osipov, S.V. Leonov
          </td>
          <td>2025-03-01</td>
          <td>MEDICAL RADIOLOGY AND RADIATION SAFETY</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7261a40e65b3cb85eca64d1251012a027c2cf0ef" target='_blank'>
              Epigenetic Symphony in Diffuse Large B-Cell Lymphoma: Orchestrating the Tumor Microenvironment
              </a>
            </td>
          <td>
            A. Caloian, Miruna Cristian, Elena Calin, Andreea-Raluca Pricop, Stelian Mociu, Liliana Seicaru, Sorin Deacu, Nicolae Ciufu, A. Suceveanu, A. Suceveanu, L. Mazilu
          </td>
          <td>2025-04-02</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f3b584bb3e59598c626453f550ef2ecaeb21de4" target='_blank'>
              Abstract A042: Uncovering the mechanism of synergistic anti-tumor effects in FAP-CD40 + PD1-IL2v combination therapy
              </a>
            </td>
          <td>
            Thuy Trinh Nguyen, Leo Kunz, Ines Grazina de Matos, Christine Trumpfheller, P. Umaña
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f7a1649cc4c1f5d7fa9fa78dd55642c87055fdf" target='_blank'>
              PD1 expression in immune cells within the tumor microenvironment of patients with non-small cell and small cell lung cancer
              </a>
            </td>
          <td>
            AYu Kalinchuk, EA Tsarenkova, DM Loos, AA Mokh, EO Rodionov, SV Miller, E. Grigoryeva, LA Tashireva
          </td>
          <td>2025-03-01</td>
          <td>Bulletin of Russian State Medical University</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor dissemination is a key event in tumor progression. During this event, a main role is played by circulating tumor cells (CTCs), immune cells, and their interaction. How the immune system supports the survival and proliferation of CTCs is not fully elucidated. In this study we established an in-vitro co-culture system consisting of immune cells and CTCs from the same patient, which increased the success rate in the establishment of CTC-derived long-term cell cultures. In this system, we characterized the immune cells of successful co-cultures and the signals they exchange with cancer cells, including cytokines and extracellular vesicle (EV) content. Using this protocol, we stabilized four CTC-derived cell lines from patients with metastatic gastroesophageal cancer, which were cultured for over a year and characterized from a genetic and molecular point of view. The four cell lines harbor shared chromosomal aberrations including the amplification at 8q24.21 containing MYC and deletion 9p21.3 containing CDKN2A/B and the IFN type I cluster. The transcriptomic profile of CTC cell lines is distinct from primary tumors, and we detected the activation of E2F, G2M and MYC pathways and the downregulation of interferon response pathway. Each cell line shows a degree of invasiveness in zebrafish in-vivo, and the most invasive ones share the same mutation in RAB14 gene. In addition, the four cell lines secrete cell-line specific EVs containing microRNAs that target YAP, BRG1-AKT1, TCF8-HDAC pathways. Overall, we highlight how the immune system plays a key role in the proliferation of CTCs through EV signaling, and how CTC cell line genomic and transcriptomic alterations make these cells less visible from the immune system and likely responsible for the survival advantage in sites distant from the microenvironment of origin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fac3e4fbae2c7ba5a9e39e780e9dfb2e2cf33f" target='_blank'>
              Gastroesophageal circulating tumor cell crosstalk with peripheral immune system guides CTC survival and proliferation
              </a>
            </td>
          <td>
            T. Rossi, Martina Valgiusti, Maurizio Puccetti, Giacomo Miserocchi, Michele Zanoni, D. Angeli, Chiara Arienti, Ilaria Pace, C. Bassi, Ivan Vannini, M. Melloni, E. Bandini, Milena Urbini, Massimo Negrini, Massimiliano Bonafé, Manuela Ferracin, Giulia Gallerani
          </td>
          <td>2025-03-29</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Spheroids are widely used in oncology for testing drugs, but models composed of a single cell line do not fully capture the complexity of the in vivo tumours targeted by chemotherapy. Developing 3D in vitro models that better mimic tumour architecture is a crucial step for the scientific community. To enable more reliable drug testing, we generated multiculture spheroids and analysed cell morphology and distribution over time. This dataset is the first publicly available single-cell light-sheet fluorescence microscopy image collection of 3D multiculture tumour models comprising of three different cell lines analysed at different time points. Specifically, we created models composed of one cancer cell line (melanoma, breast cancer, or osteosarcoma) alongside two stromal cell lines (fibroblasts and endothelial cells). Then, we acquired single-cell resolution light-sheet fluorescence 3D images of the spheroids to analyse spheroid morphology after 24, 48, and 96 hours. The image collection, whole spheroid annotations, and extracted features are publicly available for further research and can support the development of automated analysis models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f11a8e19ef0505231f115b68580b43683b589108" target='_blank'>
              Single-cell light-sheet fluorescence 3D images of tumour-stroma spheroid multicultures
              </a>
            </td>
          <td>
            Akos Diosdi, F. Piccinini, Timea Boroczky, Gabriella Dobra, Gastone Castellani, Krisztina Buzás, Peter Horvath, Mária Harmati
          </td>
          <td>2025-03-24</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Despite advances, Glioblastoma (GBM) treatment remains challenging due to its rapid progression and resistance to therapies. Objectives This study aimed to investigate the role of anoikis—a mechanism by which cells evade programmed cell death upon detachment from the extracellular matrix—in GBM progression and prognosis. Methods Utilizing single-cell sequencing and bulk-transcriptome sequencing data from TCGA, GEO, and CGGA databases, we performed comprehensive bioinformatics analyses. We identified anoikis-related genes, constructed a prognostic model using 101 machine learning algorithms, and validated its clinical utility across multiple cohorts.Finally, we also verified the expression of model genes and the function of key gene in clinical samples and cell lines. Results Single-cell sequencing revealed heterogeneous expression of anoikis-related genes across distinct cell populations within GBM. MES-like Malignant cells and Myeloids exhibited higher enrichment of these genes, implicating their role in anoikis resistance and tumor aggressiveness. The prognostic model, based on identified genes, effectively stratified patients into high-risk and low-risk groups, demonstrating significant differences in survival outcomes. Mutation and tumor microenvironment analyses highlighted distinct genetic landscapes and immune cell infiltration patterns associated with different risk groups. SLC43A3 emerged as a key gene, showing significant upregulation in tumor tissues and correlating with poor prognosis in GBM. Conclusion This study provides insights into the molecular mechanisms of anoikis resistance in GBM, underscoring its critical role in tumor progression and patient prognosis. The developed prognostic model offers a promising tool for personalized treatment strategies and warrants further exploration of targeted therapies to improve outcomes for GBM patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03752-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90776aa9e4e44d828b89f11f0b82b3f7faa8093b" target='_blank'>
              Unraveling anoikis in glioblastoma: insights from single-cell sequencing and prognostic modeling
              </a>
            </td>
          <td>
            Q. Tang, Chenfeng Ma, Jiaheng Xie, Qixiang Zhang, Bingtao Zhang, Weiqi Bian, Qingyu Lu, Zeyu Wan, Wei Wu
          </td>
          <td>2025-03-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer-associated myeloid cells due to their plasticity play dual roles in both promoting and inhibiting tumor progression. Myeloid cells with immunosuppressive properties play a critical role in anti-cancer immune regulation. Cells of different origin, such as tumor associated macrophages (TAMs), tumor associated neutrophils (TANs), myeloid derived suppressor cells (also called MDSCs) and eosinophils are often expanded in cancer patients and significantly influence their survival, but also the outcome of anti-cancer therapies. For this reason, the variety of preclinical and clinical studies to modulate the activity of these cells have been conducted, however without successful outcome to date. In this review, pro-tumor activity of myeloid cells, myeloid cell-specific therapeutic targets, in vivo studies on myeloid cell re-polarization and the impact of myeloid cells on immunotherapies/genetic engineering are addressed. This paper also summarizes ongoing clinical trials and the concept of chimeric antigen receptor macrophage (CAR-M) therapies, and suggests future research perspectives, offering new opportunities in the development of novel clinical treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8cd651287c208b9fc2804d17f7c89b490c63e13" target='_blank'>
              Harnessing myeloid cells in cancer
              </a>
            </td>
          <td>
            Su-Yeon Park, E. Pylaeva, Vikas Bhuria, A. Gambardella, Giovanna Schiavoni, D. Mougiakakos, Sung-Hoon Kim, Jadwiga Jablonska
          </td>
          <td>2025-03-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e8f7c834c39bbf5240bb0afeda29b66fdae65e" target='_blank'>
              Glioblastoma Stem Cells at the Nexus of Tumor Heterogeneity, Immune Evasion, and Therapeutic Resistance
              </a>
            </td>
          <td>
            Justin Tang, Md Al Amin, J. Campian
          </td>
          <td>2025-04-09</td>
          <td>Cells</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Recent advances as well as obstacles for immune-based cancer treatment strategies, highlight the notable impact of patient cancer microenvironments on the immune cells and immune targets. Here, we use patient-derived scaffolds (PDS) generated from 110 primary breast cancers to monitor the impact of the cancer microenvironment on immune regulators. Pronounced variation in PD-L1 expression is observed in cancer cells adapted to different patient scaffolds. This variation is further linked to clinical observations and correlated with specific proteins detected in the cell-free PDSs using mass spectrometry. When adding T cells to the PDS-based cancer cultures, the killing efficiency of activated T cells vary between the cultures, whereas non-activated T cells modulate the cancer cell PD-L1 expression to treatment-predictive values, matching killing capacities of activated T cells. Surviving cancer cells show enrichment in cancer stem cell and epithelial-to-mesenchymal transition (EMT) features, suggesting that T cells may not efficiently target cells with metastatic potential. We conclude that clinically relevant insights in how to optimally target and guide immune-based cancer therapies can be obtained by including patient-derived scaffolds and cues from the cancer microenvironment in cancer patient handling and drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f7fd750e0ac3750206051171ab5b50ac97b9ab3" target='_blank'>
              Breast cancer patient-derived scaffolds enhance the understanding of PD-L1 regulation and T cell cytotoxicity
              </a>
            </td>
          <td>
            Elena Garre, S. Rhost, Anna Gustafsson, Louis Szeponik, Thais Fenz Araujo, Marianne Quiding-Järbrink, K. Helou, Anders Ståhlberg, Göran Landberg
          </td>
          <td>2025-04-16</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6fe7c1118b5b40de590086593aa209906bee3a7" target='_blank'>
              Abstract A049: Application of a combinatorial functional precision medicine platform to predict immunotherapy response in DLBCL
              </a>
            </td>
          <td>
            Sharon Pei Yi Chan, J. Lim, M. Rashid, Edward Kai-Hua Chow
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fa1a1952e8d5276a02638856d3b8d612990f404" target='_blank'>
              Raman metabolic imaging: an emerging tool for the study of drug resistance in cancer
              </a>
            </td>
          <td>
            Wenying Zhao, Shuhua Yue
          </td>
          <td>2025-03-12</td>
          <td>Journal of Physics D: Applied Physics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab363d939606992e8b1bdf98266382a3346134a" target='_blank'>
              Development of an ALK-positive Non-Small-Cell Lung Cancer in Vitro Tumor 3D Culture Model for Therapeutic Screening.
              </a>
            </td>
          <td>
            Madeleine A Berry, A. Bland, Gretel S. Major, John C Ashton
          </td>
          <td>2025-02-24</td>
          <td>The journal of histochemistry and cytochemistry : official journal of the Histochemistry Society</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cell death is critical in tumor biology. The common cancer therapies can cause cell death and alleviate tumor, while the cancer cells can develop a resistance to cell death and survive from the therapies. Thus, not only observing the alternative mechanisms of tumor cells resistant to cell death, but also understanding the intricate dynamics of cell death processes within the tumor microenvironment (TME), are essential for tailoring effective therapeutic strategies. High-throughput sequencing technologies have revolutionized cancer research by enabling comprehensive molecular profiling. Recent advances in single cell sequencing have unraveled the heterogeneity of TME components, shedding light on their complex interactions. In this review, we explored the interplay between cell death signaling and the TME, summarised the potential drugs inducing cell death in pre-clinical stage, reviewed some studies applying next-generation sequencing technologies in cancer death research, and discussed the future utilization of updated sequencing platforms in screening novel treatment methods targeted cell death. In conclusion, leveraging multi-omics technologies to dissect cell death signaling in the context of the TME holds great promise for advancing cancer research and therapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c40c61d7dcc3833421bbc3568f9b1a472c0c4795" target='_blank'>
              Cell death in tumor microenvironment: an insight for exploiting novel therapeutic approaches
              </a>
            </td>
          <td>
            Wenxin Wang, Tong Li, Kui Wu
          </td>
          <td>2025-03-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e9a98022a15f5fa6336dcfa874e713b8fce8db6" target='_blank'>
              AGP-Net: A Universal Network for Gene Expression Prediction of Spatial Transcriptomics
              </a>
            </td>
          <td>
            Yan Yang, Xuesong Li, Liyuan Pan, Guoxun Zhang, Liu Liu, Eric A. Stone
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancer progression is marked by extracellular matrix (ECM) remodeling, including increased stiffness, faster stress relaxation, and elevated collagen levels. In vitro experiments have revealed a role for each of these factors to individually promote malignant behavior, but their combined effects remain unclear. To address this, we developed alginate-collagen hydrogels with independently tunable stiffness, stress relaxation, and collagen density. We show that these combined tumor-mimicking ECM cues reinforced invasive morphologies and promoted spheroid invasion in breast cancer and mammary epithelial cells. High stiffness and low collagen density in slow-relaxing matrices led to the greatest cell migration speed and displacement. RNA-seq revealed Sp1 target gene enrichment in response to both individual and combined ECM cues, with a greater enrichment observed under multiple cues. Notably, high expression of Sp1 target genes upregulated by fast stress relaxation correlated with poor patient survival. Mechanistically, we found that phosphorylated-Sp1 (T453) was increasingly located in the nucleus in stiff and/or fast relaxing matrices, which was regulated by PI3K and ERK1/2 signaling, as well as actomyosin contractility. This study emphasizes how multiple ECM cues in complex microenvironments reinforce malignant traits and supports an emerging role for Sp1 as a mechanoresponsive transcription factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e5d8a16a58f862a92d7dc679b2c882c51867021" target='_blank'>
              Sp1 mechanotransduction regulates breast cancer cell invasion in response to multiple tumor-mimicking extracellular matrix cues
              </a>
            </td>
          <td>
            Abhishek Sharma, Rowan F. Steger, Jen M. Li, Jane A. Baude, Kellie A. Heom, Siddharth S. Dey, Ryan S. Stowers
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Colorectal cancer (CRC) is responsible for over 900,000 annual deaths worldwide. Emerging evidence supports pro‐carcinogenic bacteria in the colonic microbiome are at least promotional in CRC development and may be causal. We previously showed toxigenic C. difficile from human CRC‐associated bacterial biofilms accelerates tumorigenesis in Apc Min/+ mice, both in specific pathogen‐free mice and in gnotobiotic mice colonized with a defined consortium of bacteria. To further understand host–microbe interactions during colonic tumorigenesis, we combined single‐cell RNA‐sequencing (scRNA‐seq), spatial transcriptomics, and immunofluorescence to define the molecular spatial organization of colonic dysplasia in our consortium model with or without C. difficile. Our data show a striking bipartite regulation of Deleted in Malignant Brain Tumors 1 (DMBT1) in the inflamed versus dysplastic colon. From scRNA‐seq, differential gene expression analysis of normal absorptive colonocytes at 2 weeks postinoculation showed DMBT1 upregulated by C. difficile compared to colonocytes from mice without C. difficile exposure. In contrast, our spatial transcriptomic analysis showed DMBT1 dramatically downregulated in dysplastic foci compared with normal‐adjacent tissue. We further integrated our datasets to generate custom colonic dysplasia scores and ligand‐receptor mapping. Validation with immunofluorescence showed DMBT1 protein downregulated in dysplastic foci from three mouse models of colonic tumorigenesis and in adenomatous dysplasia from human samples. Finally, we used mouse and human organoids to implicate WNT signaling in the downregulation of DMBT1 mRNA and protein. Together, our data reveal cell type‐specific regulation of DMBT1, a potential mechanistic link between bacteria and colonic tumorigenesis. Published 2025. This article is a U.S. Government work and is in the public domain in the USA. The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7def78e585a3b57bba506b3445109c092c108f8" target='_blank'>
              Multiomic spatial atlas shows deleted in malignant brain tumors 1 (DMBT1) glycoprotein is lost in colonic dysplasia
              </a>
            </td>
          <td>
            Emily H Green, Subhag R. Kotrannavar, Megan E. Rutherford, Hannah M. Lunnemann, Harsimran Kaur, Cody N. Heiser, Hua Ding, A. Simmons, Xiao Liu, Borden Lacy, Kay Washington, M. Shrubsole, Qi Liu, Ken S. Lau, Cynthia L Sears, Robert J. Coffey, Julia L. Drewes, Nicholas O Markham
          </td>
          <td>2025-03-03</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a751c087d4afc3ab315830aaec4e9b6e54e60f3" target='_blank'>
              Abstract A060: Reactive oligodendrocytes promote glioblastoma progression through CCL5/CCR5-mediated glioma stem cell maintenance
              </a>
            </td>
          <td>
            Jason Moffat
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d437c0ab0bc7d73f3a75511dfcad72400900dc7a" target='_blank'>
              Abstract B005: Identification of tumor-reactive TILs from lung cancer organoids co-culture through ruthenium-based proximity labeling
              </a>
            </td>
          <td>
            Weilei Hu, Yifei Bian, Zihan Zhao, Shian Ouyang, Jie P Li, Hongbin Ji
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/010275e5ed8a3481a55e4db6d64687206e7bea48" target='_blank'>
              The characteristic of epithelial-specific phenotypes and immunosuppressive microenvironment in the context of tumour budding in colorectal cancer
              </a>
            </td>
          <td>
            Phimmada Hatthakarnkul, Natalie Fisher, Leonor Schubert Santana, H. Leslie, A. Legrini, A. Ammar, Amna Ahmed Mohemmed Malty, K. Pennel, Ian R Powley, C. Bull, P. Alexander, Janos Sztanko, Jean Quinn, Jennifer Hay, Hannah Morgan, Claire Kennedy Dietrich, Yoana Doncheva, Gerard Lynch, N. Maka, Campbell S D Roxburgh, H. V. van Wyk, Xiao Fu, Donald Mcmillan, P. Dunne, N. Jamieson, C. Thuwajit, J. Edwards
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Structural variants (SVs) are increasingly recognized as important contributors to oncogenesis through their effects on 3D genome folding. Recent advances in whole-genome sequencing have enabled large-scale profiling of SVs across diverse tumors, yet experimental characterization of their individual impact on genome folding remains infeasible. Here, we leveraged a convolutional neural network, Akita, to predict disruptions in genome folding caused by somatic SVs identified in 61 tumor types from the Children’s Brain Tumor Network dataset. Our analysis reveals significant variability in SV-induced disruptions across tumor types, with the most disruptive SVs coming from lymphomas and sarcomas, metastatic tumors, and germline cell tumors. Dimensionality reduction of disruption scores identified five recurrently disrupted regions enriched for high-impact SVs across multiple tumors. Some of these regions are highly disrupted despite not being highly mutated, and harbor tumor-associated genes and transcriptional regulators. To further interpret the functional relevance of high-scoring SVs, we integrated epigenetic data and developed a modified Activity-by-Contact scoring approach to prioritize SVs with disrupted genome contacts at active enhancers. This method highlighted highly disruptive SVs near key oncogenes, as well as novel candidate loci potentially implicated in tumorigenesis. These findings highlight the utility of machine learning for identifying novel SVs, loci, and genetic mechanisms contributing to pediatric cancers. This framework provides a foundation for future studies linking SV-driven regulatory changes to cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e24a79ed206cbada1bc6b51069582d272e00ef" target='_blank'>
              Machine learning-predicted chromatin organization landscape across pediatric tumors
              </a>
            </td>
          <td>
            Ketrin Gjoni, Shu Zhang, Rachel E. Yan, Bo Zhang, Daniel Miller, Adam Resnick, Nadia Dahmane, K. S. Pollard
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Skin cancer is the most common malignancy worldwide, comprising three major types: melanoma, basal cell carcinoma (BCC), and squamous cell carcinoma (SCC). A critical goal in advancing our understanding of skin cancer is to identify the shared and distinct biological mechanisms that drive the risk profiles for each skin cancer type. In this study, we integrated tissue-specific chromatin conformation and expression quantitative trait loci data (spatial eQTL) to construct gene regulatory networks (GRNs) for melanocytes, sun-exposed skin, not sun-exposed skin, and blood. These GRNs, which capture spatially regulated gene expressions, were used as instrumental variables along with GWAS summary statistics for melanoma, BCC and SCC, to infer causal relationships between specific gene expression changes and each type of skin cancer. These Mendelian randomization analyses identified 82, 62, and 125 causal genes for melanoma, BCC, and SCC, respectively, with many of these genes not evident from the GWAS data alone. Our analyses revealed distinct mechanisms for each skin cancer type: telomere maintenance and nevus pathways in melanoma, inherited immune traits in BCC, and p53 dysfunction in SCC. Notably, apoptosis and pigmentation emerged as shared biological processes across skin cancers. These findings provide new insights into the genetic drivers of three distinct skin cancers, highlighting new gene targets which can be used to increase diagnostic precision, as well as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d1a0a87d4f015e13185fa2728ac9fd439bf3488" target='_blank'>
              Mendelian randomizations with spatial gene networks reveal shared and distinct drivers of risk in major skin cancer types
              </a>
            </td>
          <td>
            Michael Pudjihartono, J. O’Sullivan, W. Schierding
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Our group has developed and validated an advanced microfluidic platform to improve preclinical modeling of healthy and disease states, enabling extended culture and detailed analysis of tissue-engineered miniaturized organ constructs, or “organs-on-chips.” Within this system, diverse cell types self-organize into perfused microvascular networks under dynamic flow within tissue chambers, effectively mimicking the structure and function of native tissues. This setup facilitates physiological intravascular delivery of nutrients, immune cells, and therapeutic agents, and creates a realistic microenvironment to study cellular interactions and tissue responses. Known as the vascularized micro-organ (VMO), this adaptable platform can be customized to represent various organ systems or tumors, forming a vascularized micro-tumor (VMT) for cancer studies. The VMO/VMT system closely simulates in vivo nutrient exchange and drug delivery within a 3D microenvironment, establishing a high-fidelity model for drug screening and mechanistic studies in vascular biology, cancer, and organ-specific pathologies. Furthermore, the optical transparency of the device supports high-resolution, real-time imaging of fluorescently labeled cells and molecules within the tissue construct, providing key insights into drug responses, cell interactions, and dynamic processes such as epithelial-mesenchymal transition. To manage the extensive imaging data generated, we created standardized, high-throughput workflows for image analysis. This manuscript presents our image processing and analysis pipeline, utilizing a suite of tools in Fiji/ImageJ to streamline data extraction from the VMO/VMT model, substantially reducing manual processing time. Additionally, we demonstrate how these tools can be adapted for analyzing imaging data from traditional in vitro models and microphysiological systems developed by other researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af3300a0a53022f2afe2c349179047ea8540cbe" target='_blank'>
              Methods for Processing and Analyzing Images of Vascularized Micro-Organ and Tumor Systems
              </a>
            </td>
          <td>
            Stephanie J. Hachey, Christopher J. Hatch, Daniela Gaebler, Alexander G. Forsythe, Makena L Ewald, Alexander L. Chopra, Jennifer S. Fang, Christopher C. W. Hughes
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction The main challenge for cancer therapy lies in immuno-suppressive tumor micro-environment. Reprogramming tumor-associated macrophages (TAMs) into an anti-tumor phenotype is a promising strategy. Methods A comprehensive analysis by combing multi-regional single-cell, bulk and spatial transcriptome profiling with radiomics characterization was conducted to dissect the heterogeneity of TAMs and resolve the landscape of the CXCL9:SPP1 (CS) macrophage polarity in HCC. Results TAMs were particularly increased in HCC. SPP1+ TAMs and CXCL9+ TAMs were identified as the dominant subtypes with different evolutionary trajectories. SPP1+ TAMs, located in the tumor core, co-localized with cancer-associated fibroblasts to promote tumor growth and further contributed to worse prognosis. In contrast, CXCL9+ TAMs, located in the peritumoral region, synergized with CD8+ T cells to create an immunostimulatory micro-environment. For the first time, we explored the applicability of CS polarity in HCC tumors and revealed several key transcription factors involved in shaping this polarity. Moreover, CS polarity could serve as a potential indicator of prognostic and micro-environmental status for HCC patients. Based on medical imaging data, we developed a radiomics tool, RCSP (Radiogenomics-based CXCL9/SPP1 Polarity), to assist in non-invasively predicting the CS polarity in HCC patients. Conclusion Our research sheds light on the regulatory roles of SPP1+ TAMs and CXCL9+ TAMs in the micro-environment and provides new therapeutic targets or insights for the reprogramming of targeted macrophages in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d381bd12882e4f72a3664278a23bb54e3e4bf3db" target='_blank'>
              The dual role of CXCL9/SPP1 polarized tumor-associated macrophages in modulating anti-tumor immunity in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yu Gu, Zhihui Zhang, Hao Huang, Wenyong Zhu, Hongjia Liu, Rong Zhang, Nan Weng, Xiao Sun
          </td>
          <td>2025-03-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The prognosis for head and neck squamous cell carcinoma (HNSCC) remains unfavorable, primarily due to significant therapeutic resistance and the absence effective interventions. A major obstacle in cancer treatment is the persistent resistance of cancer cells to a variety of therapeutic modalities. The tumor microenvironment (TME) which includes encompasses all non-malignant components and their metabolites within the tumor tissue, plays a crucial role in this context. The distinct characteristics of the HNSCC TME facilitate tumor growth, invasion, metastasis, and resistance to treatment. This review provides a comprehensive overview of the HNSCC TME components, with a particular focus on tumor-associated macrophages (TAMs), regulatory T cells (Tregs), myeloid-derived suppressor cells (MDSCs), cancer-associated fibroblasts (CAFs), the extracellular matrix, reprogrammed metabolic processes, and metabolic products. It elucidates their contributions to modulating resistance to chemotherapy, radiotherapy, targeted therapy, and immunotherapy in HNSCC, and explores novel therapeutic strategies targeting the TME for HNSCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0002631382af389451108758d198f35c78fd6bf1" target='_blank'>
              The role of the tumor microenvironment in HNSCC resistance and targeted therapy
              </a>
            </td>
          <td>
            Zhao-hua Guo, Kang Li, Xiaotong Ren, Xijia Wang, Dunhui Yang, Shibo Ma, Xianhai Zeng, Peng Zhang
          </td>
          <td>2025-04-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931220b668a9cfbdeab70ee80f3f11b3d5f599f4" target='_blank'>
              Molecular insights into early malignant transition of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhengtao Zhang, Hong Li, Lingli Chen, Tao lu, Xinyi Shentu, Yuanhua Liu, Shuyi Ji, Zhixin Qiu, Yining Zou, Hong Wen, Jing Han, Zhengzeng Jiang, Jie Fan, Dianfan Li, Agavni Mesropian, R. Pinyol, J. M. Llovet, Hui Dong, Yuan Ji, Lijian Hui
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76a9d7d928f4b0e1d14db6867d082f27a7a018ac" target='_blank'>
              A Novel Approach to Linking Histology Images with DNA Methylation
              </a>
            </td>
          <td>
            Manahil Raza, Muhammad Dawood, Talha Qaiser, Nasir M. Rajpoot
          </td>
          <td>2025-04-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33b20af0b1c35c714611ceec53e3a323336edcf" target='_blank'>
              Myeloid cells: key players in tumor microenvironments.
              </a>
            </td>
          <td>
            Qiaomin Hua, Zhixiong Li, Yulan Weng, Yan Wu, Limin Zheng
          </td>
          <td>2025-03-06</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="This study comprehensively investigated keratinocyte subpopulation heterogeneity and developmental trajectories during skin aging using single-cell sequencing, transcriptomics, and facial aging-related genome-wide association studies (GWAS) data. We identified three major subpopulations: basal cells (BCs), spinous cells (SCs), and IFI27+ keratinocytes. Single-cell pseudotime analysis revealed that basal cells can differentiate along two distinct paths: toward spinous differentiation or the inflammatory state. With aging, the proportion of IFI27+ cells significantly increased, displaying more active inflammatory and immunomodulatory signals. Through cell–cell communication analysis, we found that the signaling pathways, including NOTCH, PTPR, and PERIOSTIN, exhibited distinct characteristics along different branches. Integration of the GWAS data revealed significant loci on chromosomes 2, 3, 6, and 9 that were spatially correlated with key biological pathways (including antigen processing, oxidative stress, and apoptosis). These findings reveal the complex cellular and molecular mechanisms underlying skin aging, offering potential targets for novel diagnostic approaches and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7d6ff4c88e6b6ec5e9e3e8325f958c0ceb91730" target='_blank'>
              Inflammatory Transformation of Skin Basal Cells as a Key Driver of Cutaneous Aging
              </a>
            </td>
          <td>
            Shupeng Liu, Sheng Lu, Zhiping Pang, Jiacheng Li, Meijuan Zhou, Zhenhua Ding, Zhijun Feng
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad75c766d4e07168f2e27a4c036f888ffea4b32" target='_blank'>
              Integrating model systems and genomic insights to decipher mechanisms of cancer metastasis.
              </a>
            </td>
          <td>
            Michelle M Leung, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-03-10</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Heterogeneity and the absence of a tumor microenvironment (TME) in traditional patient-derived organoid (PDO) cultures limit their effectiveness for clinical use. Here, Embedded Bioprinting-enabled Arrayed PDOs (Eba-PDOs) featuring uniformly arrayed colorectal cancer (CRC) PDOs within a recreated TME is presented. This model faithfully reproduces critical TME attributes, including elevated matrix stiffness (≈7.5 kPa) and hypoxic conditions found in CRC. Transcriptomic and immunofluorescence microscopy analysis reveal that Eba-PDOs more accurately represent actual tissues compared to traditional PDOs. Furthermore, Eba-PDO effectively capture the variability of CEACAM5 expression-a critical CRC marker-across different patients, correlating with patient classification and differential responses to 5-fluorouracil treatment. This method achieves an uniform size and shape within PDOs from the same patient while preserving distinct morphological features among those from different individuals. These features of Eba-PDO enable the efficient development of a label-free, morphology-based predictive model using supervised learning, enhancing its suitability for clinical applications. Thus, this approach to PDO bioprinting is a promising tool for generating personalized tumor models and advancing precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed2aaa4bc13a433c039add377b389b9ce1667ebc" target='_blank'>
              Bioprinted Patient-Derived Organoid Arrays Capture Intrinsic and Extrinsic Tumor Features for Advanced Personalized Medicine.
              </a>
            </td>
          <td>
            Jonghyeuk Han, Hye-Jin Jeong, Jeonghan Choi, Hyeonseo Kim, Taejoon Kwon, Kyungjae Myung, Kyemyung Park, Jung In Park, Samuel Sánchez, Deok-Beom Jung, Chang Sik Yu, In Ho Song, Jin-Hyung Shim, Seung-Jae Myung, Hyun-Wook Kang, Tae-Eun Park
          </td>
          <td>2025-03-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Patients with small-cell lung cancer (SCLC) have the poor prognosis. Current research suggested that low-dose radiotherapy (LDRT) combined with immunotherapy can enhance the immunogenicity of tumor cells, thereby improving antigen presentation and promoting the intratumoral infiltration of CD8+ T cells, which significantly extends the survival of patients. However, the change trajectory of T cells, and the mechanisms underlying the promotion of intratumoral infiltration of CD8+ T cells, and the enhancement of their cytotoxic functions remain to be elucidated. Methods To delineate the dynamic changes of T cells, we collected tumors from Kaede tumor-bearing mice that had undergone radioimmunotherapy. Using flow cytometry, we sorted intratumoral-infiltrating immune cells, which were required for single-cell RNA sequencing, at various time points (Kaede Red: derived from tumor-draining lymph node [TDLN]). The results obtained from the sequencing analysis were further validated through experiments, such as flow cytometry, immunofluorescence, and analysis of clinical cohort data. Results Here, we observed stem-like T cells migrating from the TDLN to the tumor site and differentiating into effector phenotypes within the tumor. Dendritic cells (DCs) are the key cluster that induces the differentiation of stem-like T cell into effector phenotypes. Moreover, SCLC patients with a high infiltration of tumor-specific CXCR6+CD8+ T cells exhibited a supportive TME and longer survival time (P < 0.001). Conclusions This study delineates the change trajectory of CD8+ T cells, identifies the crucial role of DCs in T cell differentiation, and highlights the significance of tumor-specific CXCR6+CD8+ T cells in anti-tumor immunity. Future therapeutic strategies for SCLC could focus on enhancing the infiltration of activated DCs and CXCR6+CD8+ T cells within the tumor microenvironment to improve treatment efficacy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06450-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2fc977549cef2705abb06bf01e35d0889a88bb3" target='_blank'>
              Dynamic evolution and antitumor mechanisms of CXCR6+CD8+ T cells in small cell lung cancer treated with low-dose radiotherapy and immunotherapy
              </a>
            </td>
          <td>
            Guo Lin, Zhuoran Yao, Kai Kang, R. Luo, Linglu Yi, You Lu
          </td>
          <td>2025-04-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The mean survival of metastatic lung adenocarcinoma is less than 1 year, highlighting the urgent need to understand the mechanisms underlying its high mortality rate. The role of Extracellular vesicles (EVs) in facilitating the interactions between cancer cells and the metastatic microenvironment has garnered increasing attention. Previous studies on the role of EVs in metastasis have been primarily focused on cancer cell-derived EVs in modulating the functions of stromal cells. However, whether cancer stem cells (CSCs) can alter the metastatic properties of non-CSC cells, and whether EV crosstalk can mediate such interaction, have not been demonstrated prior to this report. In the present study, we integrated multi-omics sequencing and public database analysis with experimental validation to demonstrate, for the first time, the exosomal Mir100hg, derived from CSCs, could enhance the metastatic potential of non-CSCs both in vitro and in vivo. Mechanistically, HNRNPF and HNRNPA2B1 directly binds to Mir100hg, facilitating its trafficking via exosomes to non-CSCs. In non-CSCs, Mir100hg upregulates ALDOA expression, subsequently leading to elevated lactate production. Consequently, the increased lactate levels enhance H3K14 lactylation by 2.5-fold and promote the transcription of 169 metastasis-related genes. This cascade of events ultimately results in enhanced ALDOA-driven glycolysis and histone lactylation-mediated metastatic potential of non-CSC lung cancer cells. We have delineated a complex regulatory network utilized by CSCs to transfer their high metastatic activity to non-CSCs through exosomal Mir100hg, providing new mechanistic insights into the communication between these two heterogeneous tumor cell populations. These mechanistic insights provide novel therapeutic targets for metastatic lung cancer, including HNRNPF/HNRNPA2B1-mediated Mir100hg trafficking and the histone lactylation pathway, advancing our understanding of CSC-mediated metastasis while suggesting promising strategies for clinical intervention. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12951-025-03198-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/982b46ad0f67ebc704b6d142139e5b777f0090b1" target='_blank'>
              Exosomal lncRNA Mir100hg from lung cancer stem cells activates H3K14 lactylation to enhance metastatic activity in non-stem lung cancer cells
              </a>
            </td>
          <td>
            Lei Shi, Bowen Li, Jiyu Tan, Ling Zhu, Sicheng Zhang, Yuhan Zhang, Meng Xiang, Jie Li, Yan Chen, Xue Han, Jiacheng Xie, Yao Tang, H. Xing, Jingyu Li, Jianyu Wang, cnJingyu Li
          </td>
          <td>2025-02-28</td>
          <td>Journal of Nanobiotechnology</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98ccd90fb2b6e5c7d2c301c1becb4a055db66105" target='_blank'>
              Molecule Differentiation Encoding Microscopy to Dissect Dense Biomolecules in Cellular Nanoenvironments Below Spatial Resolution.
              </a>
            </td>
          <td>
            Siyue Fan, Xinyin Li, Huan Liu, Mengying Ye, Yan He, Wenhao Fu, Feng Chen, Yongxi Zhao
          </td>
          <td>2025-03-04</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The intratumoral mycobiome plays a crucial role in the tumor microenvironment, but its impact on renal cell carcinoma (RCC) remains unclear. We collected and quantitatively profiled the intratumoral mycobiome data from 1044 patients with RCC across four international cohorts, of which 466 patients received immunotherapy. Patients were stratified into mycobiota ecology-depauperate and mycobiota ecology-flourishing (MEF) groups based on fungal abundance. The MEF group had worse prognosis, higher fungal diversity, down-regulated lipid catabolism, and exhausted CD8+ T cells. We developed the intratumoral mycobiota signature and intratumoral mycobiota-related genes expression signature, which robustly predicted prognosis and immunotherapy outcomes in RCC and other cancers. Aspergillus tanneri was identified as a potential key fungal species influencing RCC prognosis. Our findings suggest that the intratumoral mycobiome suppresses lipid catabolism and induces T cell exhaustion in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d247d376bb04aaad37a3a7086a6eccbe76de5ac" target='_blank'>
              Intratumoral mycobiome heterogeneity influences the tumor microenvironment and immunotherapy outcomes in renal cell carcinoma
              </a>
            </td>
          <td>
            Weiming Mou, Zhixing Deng, Lingxuan Zhu, Aimin Jiang, Anqi Lin, Liling Xu, G. Deng, Hongsen Huang, Zeji Guo, Bang Zhu, Shuqi Wu, Tao Yang, Lu Wang, Zaoqu Liu, Ting Wei, Jian Zhang, Liang Cheng, Haojie Huang, Rui Chen, Yi Shao, Quan Cheng, Linhui Wang, Shuofeng Yuan, Peng Luo
          </td>
          <td>2025-04-09</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Tumor immune evasion extends beyond T cells, affecting innate immune elements like natural killer cells (NK) and macrophages within the tumor-immune microenvironment (TIME). Nevertheless, translational strategies to trigger collaboration of NK cells and macrophages to initiate sufficient anti-tumor cytoxicity remain scarce and are urgently needed. Methods In this study, TCGA datasets was used to confirm the prognosis value of the expression level of NLR family pyrin domain containing 4 (NLRP4) in NSCLC and the tumor tissues microarray was used to further check its clinical-relevance at protein-level. Subsequently, a tumor cell line with stable NLRP4 overexpression was established and subcutaneous tumor models in C57BL/6J mice were used to validate the anti-tumor characteristics of NLRP4. After analyzing the tumor microenvironment using flow cytometry and multiplex immunofluorescence, we further validated our findings through co-culture transwell assays and TCGA analysis. Utilizing bulk-RNA sequencing, proteomics, and mass spectrometry of mouse tumor tissues, we innovatively identified the downstream pathways of NLRP4 and verified them through co-immunoprecipitation (co-IP) and Western blot (WB) experiments. Results NLRP4 could trigger a distinct anti-tumor ecosystem organized by TIGIT+TNFA+ NK and iNOS+ M1 in lung cancer, discovered in TCGA analysis and verified in murine model. NLRP4-eco exerted tumor-suppression capacity through chemokine reprogramming including CCL5 and CXCL2. Meanwhile, the cytoxicity of NK could be facilitated by iNOS+M1. Mechanistically, NLRP4 stimulated PI3K/Akt-NF-kB axis through suppression of the activity of PP2A. Besides, knockdown of CCL5 and blockade of CXCL2-CXCR2 axis abolished chemotaxis of TIGIT+TNFA+ NK and iNOS+ M1 respectively, as well as for LB-100, a PP2A inhibitor. Conclusion Altogether, we delineated NLRP4’s unexplored facets and discovered an NLRP4-driven anti-tumor ecosystem composed of TIGIT+TNFA+ NK and iNOS+ M1. Finally, targeting PP2A by its inhibitor successfully mimicked the anti-tumor capacity of the overexpression of NLRP4. Supplementary Information The online version contains supplementary material available at 10.1186/s40364-025-00756-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e587aa6bc66aa80360956efaece01ebbb6a659" target='_blank'>
              NLRP4 unlocks an NK/macrophages-centered ecosystem to suppress non-small cell lung cancer
              </a>
            </td>
          <td>
            Zhouwenli Meng, Jian Li, Hui Wang, Zhengqi Cao, Wenqing Lu, X. Niu, Yi Yang, Ziming Li, Ying Wang, Shun Lu
          </td>
          <td>2025-03-14</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Solid tumors develop within a complex environment called the tumor microenvironment (TME), which is sculpted by the presence of other cells, such as cancer-associated fibroblasts (CAFs) and immune cells like macrophages (Mφs). Despite the presence of immune cells, tumor cells orchestrate a tumor-supportive environment through intricate interaction with the components of the TME. However, the specific mechanism by which this intercellular dialogue is regulated is not fully understood. To that end, the development of an organotypic 3D breast TME-on-a-chip (TMEC) model, integrated with single-cell RNA sequencing analysis, is reported to mechanistically evaluate the progression of triple-negative breast cancer (TNBC) cells in the presence of patient-derived CAFs and Mφs. Extensive functional assays, including invasion and morphometric characterization, reveal the synergistic influence of CAFs and Mφs on tumor cells. Furthermore, gene expression and pathway enrichment analyses identify the involvement of the KYNU gene, suggesting a potential immune evasion mechanism through the kynurenine pathway. Lastly, the pharmacological targeting of the identified pathway is investigated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8de452a3f7784c1593c1ff6c237b52e31623e79" target='_blank'>
              Tumor Microenvironment On-A-Chip and Single-Cell Analysis Reveal Synergistic Stromal-Immune Crosstalk on Breast Cancer Progression.
              </a>
            </td>
          <td>
            Kalpana Ravi, Yining Zhang, Lydia Sakala, Twinkle Jina Minette Manoharan, Barbara Pockaj, Joshua LaBaer, Jin G. Park, Mehdi Nikkhah
          </td>
          <td>2025-03-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Poorly differentiated thyroid cancer (PDTC) and anaplastic thyroid cancer (ATC) present major challenges in treatment owing to extreme aggressiveness and high heterogeneity. In this study, deep-scale analyses spanning genomic, proteomic, and phosphoproteomic data are performed on 348 thyroid-cancer and 119 tumor-adjacent samples. TP53 (48%), TERT promoter (36.5%), and BRAF (23%) are most frequently mutated in PDTC and ATC. Ribosome biogenesis is identified as a common hallmark of ATC, and RRP9 silencing dramatically inhibits tumor growth. Proteomic clustering identified three ATC/PDTC subtypes. Pro-I subtype is characterized with aberrant insulin signaling and low immune cell infiltration, and Pro-II is featured with DNA repair signaling, while Pro-III harbors high frequency of TP53 and BRAF mutation and intensive C5AR1+ myeloid infiltration. Targeting C5AR1 synergistically improves antitumor effect of PD-1 blockade against ATC cell-derived tumors. These findings provide systematic insights into tumor biology and opportunities for drug discovery, accelerating precision therapy for virulent thyroid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/618bcd39c8235e1a1c051cf80f1d7a8224b8d7f2" target='_blank'>
              Integrative proteogenomic characterization reveals therapeutic targets in poorly differentiated and anaplastic thyroid cancers
              </a>
            </td>
          <td>
            Zongfu Pan, Zhuo Tan, Ning Xu, Zhenmei Yao, Chuanming Zheng, Jinbiao Shang, Lei Xie, Jiajie Xu, Jia-feng Wang, Liehao Jiang, Xuhang Zhu, Dingyi Yu, Ying Li, Yulu Che, Yingying Gong, Zhaoyu Qin, Yiwen Zhang, Xiaozhou Zou, Tong Xu, Zhenying Guo, Tiefeng Jin, Tiannan Guo, Wei Wang, Wanyuan Chen, Yaoting Sun, Weixin Wang, Xiaojun Peng, Changtian Yin, Chen Ding, Ping Huang, Minghua Ge
          </td>
          <td>2025-04-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Brain metastasis represents a significant challenge in oncology, driven by complex molecular and epigenetic mechanisms that distinguish it from primary tumors. While recent research has focused on identifying genomic mutation drivers with potential clinical utility, these strategies have not pinpointed specific genetic mutations responsible for site-specific metastasis to the brain. It is now clear that successful brain colonization by metastatic cancer cells requires intricate interactions with the brain tumor ecosystem and the acquisition of specialized molecular traits that facilitate their adaptation to this highly selective environment. This is best exemplified by widespread transcriptional adaptation during brain metastasis, resulting in aberrant gene programs that promote extravasation, seeding, and colonization of the brain. Increasing evidence suggests that epigenetic mechanisms play a significant role in shaping these pro-brain metastasis traits. This review explores dysregulated chromatin patterns driven by chromatin remodeling, histone modifications, DNA/RNA methylation, and other epigenetic regulators that underpin brain metastatic seeding, initiation, and outgrowth. We provide novel insights into how these epigenetic modifications arise within both the brain metastatic tumor and the surrounding brain metastatic tumor ecosystem. Finally, we discuss how the inherent plasticity and reversibility of the epigenomic landscape in brain metastases may offer new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="UL-16 binding protein 2 (ULBP2), a human NKG2D ligand, has been identified as a poor prognostic factor in several cancers based on recent comprehensive analyses of immune-related genes using the Cancer Genome Atlas datasets. Despite its clinical significance, the functional role of ULBP2 in vivo remains largely unknown. In this study, we investigated the role of ULBP2 in modulating anti-tumor immunity using murine melanoma cell lines engineered to stably express surface-expressed or soluble ULBP2. Subcutaneous transplantation of ULBP2-expressing melanoma cells into syngeneic mice resulted in accelerated tumor growth, mediated by surface-expressed ULBP2, through the suppression of NKG2D-dependent immune responses. In vitro experiments revealed that sustained exposure to tumor-expressed ULBP2 reduced NKG2D expression and cytotoxic activity of splenocytes. In contrast, soluble ULBP2 did not significantly affect tumor growth or immune responses. These findings suggest that surface-expressed ULBP2 plays a pivotal role in tumor immune evasion and highlight its potential as a therapeutic target to enhance anti-tumor immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8972a702dda5335fbe3c7a95357736cd65185aad" target='_blank'>
              ULBP2 Promotes Tumor Progression by Suppressing NKG2D-Mediated Anti-Tumor Immunity
              </a>
            </td>
          <td>
            K. Yamane, K. Yamaguchi, Yasuhiko Teruya, Naomi Miyake, Yuji Nakayama, Takafumi Nonaka, Hiroki Chikumi, Akira Yamasaki
          </td>
          <td>2025-03-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25aeeccc03e491382af67e5b76f7835dfb0b4a41" target='_blank'>
              508 The Largest Patient-Derived Meningioma Organoid Bank Demonstrates a Robust Model for Transcriptional Analysis of Tumors
              </a>
            </td>
          <td>
            Youssef M. Zohdy, Arman Jahangiri, Fadi Jacob, A. Aksionau, Alejandra Rodas, Zhexing Wen, Gustavo Pradilla, Tomas Garzon-Muvdi
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Although immune checkpoint (IC) inhibition is a major treatment modality in cancer-immunotherapy, multiple cancers show low response. Our in-silico exploration by mining cancer datasets using R2, available clinical trial data, and Kaplan–Meier analysis from GEPIA depicted that unlike low-responder (LR) cancers, high-responder (HR) cancers furnish higher IC expression, that upon lowering may provide better prognosis. Contrastingly, pancreatic adenocarcinoma (PAAD) demonstrated high IC expression but low immunotherapy-response. Infiltration scores from TIMER2.0 revealed higher pro-tumor immune subsets and cancer-associated fibroblasts (CAFs) while depicting lower anti-tumor immune subsets in PAAD as compared to HR lung adenocarcinoma (LUAD). Additionally, bioinformatic tool cBioportal showed lesser tumor mutational burden, mismatch repair deficiency and greater percent of driver mutations in TP53, KRAS and CDKN2A in PAAD, supporting its higher immunotherapy-resistance than LUAD. Our search for the ‘key’ immunotherapy response-deciding factor(s) revealed cancer stem cells (CSCs), the known contributors of therapy-resistance and immuno-evasion, to be positively correlated with above-mentioned driver mutations, pro-tumor immune and CAF subsets; and that PAAD furnished higher expression of CSC genes than LUAD. UMAP/tSNE analyses revealed that high CSC signature is positively correlated with immunotherapy-resistance genes and pro-cancer immune cells, while negatively with cytotoxic-T cells in PAAD. Our in-silico study explains the low immunotherapy-response in high IC-expressing PAAD, wherein CSC plays a pivotal role. Further exploration portrayed correlation of CSCs with immunotherapy-resistance in other LR and HR cancers too, substantiating the need for personalized CSC evaluation and targeting for successful immunotherapy outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93924-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de30629094c5f0558bed2894abe95ee38a9c5b9" target='_blank'>
              In-silico analysis unveiling the role of cancer stem cells in immunotherapy resistance of immune checkpoint-high pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Udit Basak, S. Mukherjee, S. Chakraborty, Gaurisankar Sa, S. G. Dastidar, Tanya Das
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Localized prostate cancer (PCa) is highly variable in their response to therapies. Although a fraction of this heterogeneity can be explained by clinical factors or genomic and transcriptomic profiling, the proteomic-based profiling of aggressive PCa remains poorly understood. Here, we profiled the genome, transcriptome, proteome and phosphoproteome of 145 cases of localized PCa in Chinese patients. Proteome-based stratification of localized PCa revealed three subtypes with distinct molecular features: immune subgroup, arachidonic acid metabolic subgroup and sialic acid metabolic subgroup with highest biochemical recurrence (BCR) rates. Further, we nominated NANS protein, a key enzyme in sialic acid synthesis as a potential prognostic biomarker for aggressive PCa and validated in two independent cohorts. Finally, taking advantage of cell-derived orthotopic transplanted mouse models, single-cell RNA sequencing (scRNA-seq) and immunofluorescence analysis, we revealed that targeting NANS can reverse the immunosuppressive microenvironment through restricting the sialoglycan-sialic acid-recognizing immunoglobulin superfamily lectin (Siglec) axis, thereby inhibiting tumor growth of PCa. In sum, we integrate multi-omic data to refine molecular subtyping of localized PCa, and identify NANS as a potential prognostic biomarker and therapeutic option for aggressive PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9febd685653f7715435bba7b29d1b13ce378312" target='_blank'>
              Integrated proteogenomic characterization of localized prostate cancer identifies biological insights and subtype-specific therapeutic strategies
              </a>
            </td>
          <td>
            Wei Ou, Xin-Xin Zhang, Bin Li, Ying Tuo, Ren-Xuan Lin, Pengfei Liu, Jianping Guo, Hiocheng Un, Minghao Li, Jiahao Lei, Xiao-Jing Gao, Fu-Fu Zheng, Ling-Wu Chen, Ling-Li Long, Zongren Wang
          </td>
          <td>2025-04-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f14df5b58e9451d1f5257546c3620f137c7dd464" target='_blank'>
              Spatially resolved integrative analysis of transcriptomic and metabolomic changes in tissue injury studies
              </a>
            </td>
          <td>
            E. Williams, L. Franzén, Martina Olsson Lindvall, Gregory Hamm, Steven Oag, Muntasir Mamun Majumder, James Denholm, A. Hamidinekoo, Javier Escudero Morlanes, Marco Vicari, J. Lundeberg, L. Setyo, Aleksandr Zakirov, J. Hornberg, Marianna Stamou, Patrik L. Ståhl, A. Ollerstam, Jennifer Tan, I. Mohorianu
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9ecd558a66178f5d9717f1ef118e46a86fb14e5" target='_blank'>
              Abstract B021: Optimizing combination immunotherapy regimens in kidney cancer using ex-vivo multicellular 3D immune-microtumors and multiplex immunoassays
              </a>
            </td>
          <td>
            Roseanna Hare, Eleonora Peerani, Keqian Nan, Polina Maximchik, Jessica Hudson, Jonathan Ient, Nourdine Bah, Jay Kearney, Francesco Iori, Matthew Williams, M. Tran, D. Ranatunga
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Somatic mutations that increase hematopoietic stem cell (HSC) fitness drive their expansion in clonal hematopoiesis (CH) and predispose to blood cancers. Although CH frequently occurs with aging, it rarely progresses to overt malignancy. Population variation in the growth rate and potential of mutant clones suggests the presence of genetic factors protecting against CH, but these remain largely undefined. Here, we identify a non-coding regulatory variant, rs17834140-T, that significantly protects against CH and myeloid malignancies by downregulating HSC-selective expression and function of the RNA-binding protein MSI2. By modeling variant effects and mapping MSI2 binding targets, we uncover an RNA network that maintains human HSCs and influences CH risk. Importantly, rs17834140-T is associated with slower CH expansion rates in humans, and stem cell MSI2 levels modify ASXL1-mutant HSC clonal dominance in experimental models. These findings leverage natural resilience to highlight a key role for post-transcriptional regulation in human HSCs, and offer genetic evidence supporting inhibition of MSI2 or its downstream targets as rational strategies for blood cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231338d317a31d1b141aef89e6f810d3087be9ba" target='_blank'>
              Inherited resilience to clonal hematopoiesis by modifying stem cell RNA regulation
              </a>
            </td>
          <td>
            Gaurav Agarwal, Mateusz Antoszewski, Xueqin Xie, Y. Pershad, U. Arora, Chi-Lam Poon, Peng Lyu, Andrew J. Lee, Chun-Jie Guo, Tianyi Ye, Laila Barakat Norford, Anna-Lena Neehus, Lucrezia della Volpe, Lara Wahlster, Diyanath Ranasinghe, Tzu-Chieh Ho, T. Barlowe, Arthur Chow, A. Schurer, James Taggart, Benjamin H. Durham, Omar Abdel-Wahab, Kathy L. McGraw, James M Allan, Ruslan Soldatov, Alexander G. Bick, M. Kharas, V. G. Sankaran
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098b5888d894097b207e005b18110c0c81abf3d8" target='_blank'>
              Modeling cell differentiation in neuroblastoma: insights into development, malignancy, and treatment relapse
              </a>
            </td>
          <td>
            Simon F. Martina-Perez, Luke A. Heirene, Jennifer C. Kasemeier, P. Kulesa, Ruth E. Baker
          </td>
          <td>2025-02-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf6b83c351c1e02e83a3bea28cecedd0fcbfe3d" target='_blank'>
              Abstract A001: AscitesPredict: Validating anti-cancer therapeutic efficacy in gastroesophageal cancer using rapid ex vivo high-dimensional morphology profiling
              </a>
            </td>
          <td>
            Jesse S Boehm, Mushriq Al-Jazrawe, Csaba Molnar, Beth Cimini, S. Klempner, Niklas Rindtorff, Steven Blum
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cell surface proteins (CSPs) are crucial identifiers for cell types and states, especially in dendritic cells (DCs). Current proteomic methods for profiling CSPs are limited by their hydrophobic nature and low abundance, which often require genetic or cell surface engineering and exhibit biased and insufficient labeling efficiency. Herein, we report [Ru(bpy)3]Cl2 (Ru) for effective biotinylation on the cell surfaceome via a simple “mix and lighten” method. The versatile photoredox pathways of Ru are leveraged using a probe cocktail of biotin-phenol and biotin-hydrazide for improved substrate coverage. The “cocktail” labeling strategy results in reproducible identification of up to 733 plasma membrane proteins on HeLa cells, and is further applied to map dynamic changes in the surfaceome during the differentiation of primary bone marrow-derived dendritic cells, which demonstrates a user-friendly and deep-surfaceome-coverage tool for profiling dynamic changes in primary cells, with potential implications for cell identities, functional states, and novel drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ca83b207a76ae8366d3ec19980bdc5085a369c" target='_blank'>
              Cocktail Chemical Labeling for In-Depth Surfaceome Profiling of Bone-Marrow Derived Dendritic Cells
              </a>
            </td>
          <td>
            Shiming Sun, Siyang Liu, Tao Deng, Yudi Hu, Rong Yang, Jie P. Li, Yun Ge
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) exhibits metastatic organotropism, primarily targeting liver, lung, and rarely the brain. Here, we study chromosomal imbalances (CIs) in cohorts of primary CRCs and metastases. Brain metastases show the highest burden of CIs, including aneuploidies and focal CIs, with enrichment of +12p encoding KRAS. Compared to liver and lung metastases, brain metastases present with increased co-occurrence of KRAS mutation and amplification. CRCs with concurrent KRAS mutation and amplification display significant metabolic reprogramming with upregulation of glycolysis, alongside upregulation of cell cycle pathways, including copy number gains of MDM2 and CDK4. Evolutionary modeling suggests early acquisition of many organotropic CIs enriched in both liver and brain metastases, while brain-enriched CIs preferentially emerge later. Collectively, this study supports a model where cytogenetic events in CRCs favor site-specific metastatic colonization. These site-enriched CI patterns may serve as biomarkers for metastatic potential in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb630f3c7386bec7d6feb3eae00ae36bb70fad4" target='_blank'>
              Cytogenetic signatures favoring metastatic organotropism in colorectal cancer
              </a>
            </td>
          <td>
            M. M. Golas, B. Gunawan, Angelika Gutenberg, Bernhard C Danner, J. S. Gerdes, Christine Stadelmann, László Füzesi, Torsten Liersch, B. Sander
          </td>
          <td>2025-04-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Cellular biomechanics plays a critical role in cancer metastasis and tumor progression. Existing studies on cancer cell biomechanics are mostly conducted in flat 2D conditions, where cells' behavior can differ considerably from those in 3D physiological environments. Despite great advances in developing 3D in vitro models, probing cellular elasticity in 3D conditions remains a major challenge for existing technologies. In this work, optical Brillouin microscopy is utilized to longitudinally acquire mechanical images of growing cancerous spheroids over the period of 8 days. The dense mechanical mapping from Brillouin microscopy enables us to extract spatially resolved and temporally evolving mechanical features that were previously inaccessible. Using an established machine learning algorithm, it is demonstrated that incorporating these extracted mechanical features significantly improves the classification accuracy of cancer cells, from 74% to 95%. Building on this finding, a deep learning pipeline capable of accurately differentiating cancerous spheroids from normal ones solely using Brillouin images have been developed, suggesting the mechanical features of cancer cells can potentially serve as a new biomarker in cancer classification and detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cb458799cb8481364a64cf452e1c9a692db7c11" target='_blank'>
              Mechanical Evolution of Metastatic Cancer Cells in 3D Microenvironment.
              </a>
            </td>
          <td>
            Karlin Hilai, Daniil Grubich, Marcus Akrawi, Hui Zhu, Razanne Zaghloul, Chenjun Shi, Man Do, Dongxiao Zhu, Jitao Zhang
          </td>
          <td>2025-03-21</td>
          <td>Small</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Osteosarcoma (OS) is the most common malignant bone cancer. Thirty to forty percent of all patients with OS develop tumor recurrence, almost exclusively in the form of lung metastasis, which is associated with a dismal 20% 5-year survival rate. Cancer-associated fibroblasts are a critical cell type within the lung tumor microenvironment that promote immune suppression, drug resistance, and tumor cell survival. Prior work shows tumor cells can co-opt fibroblasts to a protumorigenic phenotype via exosome-mediated intercellular communication. Currently, the mechanisms by which OS exosomes modulate resident lung fibroblast (LF) function have not been evaluated. To investigate this, we isolated exosomes from a panel of six OS cell lines. We assessed the uptake and response of human donor–derived primary LFs (n = 4) to OS exosome treatment in vitro via flow cytometry, confocal fluorescent microscopy, proliferation assays, phosphokinase array, multiplex cytokine analysis, and RNA sequencing. We observed that LFs efficiently take up OS exosomes, which are associated with the induction of MAPK pathway activation, fibroblast proliferation, and significantly enhanced secretion of IL-6, CXCL8, and CCL2 compared with untreated LFs. RNA sequencing of exosome-treated LFs confirmed these responses and revealed significant enrichment of pathways related to cytokine secretion, proliferation, immune cell chemotaxis, migration, proinflammatory, and profibrotic mediators. Finally, in an exosome-educated LF–OS coculture model, exosome-educated LFs conferred significantly increased OS cell survival and proliferation compared with untreated fibroblasts. Together, these data suggest that the OS-derived exosomes can induce a hallmark cancer-associated fibroblast–like inflammatory phenotype in LFs, providing valuable insights into mechanisms that may promote recurrent OS lung metastasis. Significance: These findings provide a critical first step in characterizing the capacity of OS-derived exosomes to reprogram primary LFs toward a tumor-promoting inflammatory phenotype in vitro, offering novel molecular targets for the modulation of fibroblasts in the lung microenvironment as potential therapeutic strategies to prevent OS metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4796bcb4b0cbec4e85b9659ef31e17c57400c98" target='_blank'>
              Osteosarcoma Exosome Priming of Primary Human Lung Fibroblasts Induces an Immune Modulatory and Protumorigenic Phenotype
              </a>
            </td>
          <td>
            E. Palmer, Kathryn E Cronise, Laurel A Haines, Sunetra Das, Aaron Offermann, Carina P Easton, Daniel P Regan
          </td>
          <td>2025-03-18</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0786e178852bb9b6b2a0945adb3bddac988e519" target='_blank'>
              Abstract A030: A deeper understanding of tumor microenvironment modulation induced by IL13Ra2 CAR T cells in a patient with grade 3 ependymoma
              </a>
            </td>
          <td>
            Maryam Aftabizadeh, B. Aguilar, M. D’Apuzzo, B. Badie, Christine E. Brown
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Image‐based screening is an efficient drug discovery strategy. Combining high‐content screening (HCS) with artificial intelligence efficiently detects drug‐induced cellular phenotypes from a large pool of compounds. However, previous studies primarily focus on cell classification and cannot interpret models using morphological features. Additionally, the existing morphology‐based studies use only few features, which cannot accurately characterize cell phenotypic perturbations. Herein, π‐PhenoDrug, a deep learning‐based pipeline, is developed for cell phenotype‐driven drug screening. It integrates cell segmentation, morphological profile construction, and phenotype analysis modules of HCS processes. π‐PhenoDrug is applied to evaluate drug response in various human melanoma cell lines. The results demonstrate that π‐PhenoDrug can evaluate drug killing effects across different cell lines via both supervised and unsupervised modes. Furthermore, π‐PhenoDrug identifies drugs with potential killing effects on melanoma cells from a library of diverse compounds. Compared with traditional single‐readout assays, this method is more sensitive to compounds that induce weaker phenotypes, reducing the risk of overlooking effective drugs. These results confirm that π‐PhenoDrug can achieve high‐throughput and accuracy analysis of cell phenotypic data using an unbiased and automated workflow, improving drug discovery efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d59d452770f50a368d33283eccf620d3785158da" target='_blank'>
              π‐PhenoDrug: A Comprehensive Deep Learning‐Based Pipeline for Phenotypic Drug Screening in High‐Content Analysis
              </a>
            </td>
          <td>
            Xiao Li, Qinxue Ouyang, Mingfei Han, Xiaoqing Liu, Fuchu He, Yunping Zhu, Ling Leng, Jie Ma
          </td>
          <td>2025-03-02</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="One of the alternative approaches to glioblastoma treatment is cellular immunotherapy based on natural killer cells (NK cells). To enhance their cytotoxic effect on tumor cells, new NK cell lines are being created using genetic engineering techniques. The aim of the study was to evaluate the impact efficacy of “enhanced” NK cells on early metabolic rearrangements and the viability of glioblastoma cells in a patient using a tumor spheroid model. Materials and Methods The study used a primary culture of GBM7-Luc2-mKate2 human glioblastoma, a line of YT (YTwt) wildtype human NK cells, as well as lines created by us with overexpression of VAV1 protein with either CISH (YT–Vav1+CISH–/–) or B2M (YT–Vav1+B2M–/–) knockouts. Tumor spheroids were produced in round-bottomed, low-adhesive plates. 100 thousand immune cells were added to each spheroid, and spheroids viability was evaluated at several time points applying fluorescence staining using a live/dead cell viability assay kit; autofluorescence of metabolic coenzyme nicotinamide adenine dinucleotide (phosphate), or NAD(P)H, was visualized in spheroids using an LSM 880 laser scanning microscope (Carl Zeiss, Germany) with a FLIM module (Becker & Hickl GmbH, Germany). Results It was found that autofluorescence attenuation parameters of NAD(P)H coenzyme in human glioblastoma cells change significantly when exposed to both YT–Vav1+CISH–/– and YT–Vav1+B2M–/–, indicating occurrence of an early metabolic shift in tumor cells towards a less aggressive oxidative phenotype, and this is consistent with dead cells fraction increase and living cells fraction decrease in spheroid composition. Conclusion The data obtained on enhanced cytotoxic activity of new modified NK cell lines against human glioblastoma spheroids are important to understand interaction mechanisms between tumor and immune cells and the development of glioblastoma adoptive cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23eb7a7cbdd00100ff9b2c325ca02ceabe4943fe" target='_blank'>
              Efficacy Evaluation of “Enhanced” Natural Killers with CISH and B2M Knockouts on Viability and Metabolic Status of 3D Glioblastoma Spheroid Cells in Patients
              </a>
            </td>
          <td>
            D.V. Yuzhakova, D.A. Sachkova, M. V. Shirmanova, V. Shcheslavskiy, A.M. Mozherov, E. B. Dashinimaev, V. Baklaushev, G. Yusubalieva
          </td>
          <td>2025-02-28</td>
          <td>Modern Technologies in Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c8b8c8967239c4ebdf5921b795eefe1341f9d3" target='_blank'>
              Single-cell eQTL mapping reveals cell subtype-specific genetic control and mechanism in malignant transformation of colorectal cancer.
              </a>
            </td>
          <td>
            Can Chen, Yimin Cai, Wenlong Hu, Kai Tan, Zequn Lu, Xu Zhu, Ziying Liu, Chunyi He, Guangping Xu, Ruizhe Zhang, Caibo Ning, Shuheng Ruan, Jiayan Gao, Xiaojun Yang, Yongchang Wei, Xu Zhu, Xiangpan Li, Faxi Wang, Fubing Wang, Jiaoyuan Li, Meng Jin, Bin Li, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-03-04</td>
          <td>Cancer discovery</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="Aggressive variant and androgen receptor (AR)-independent castration resistant prostate cancers (CRPC) represent the most significant diagnostic and therapeutic challenges in prostate cancer. This study examined a case of simultaneous progression of both adenocarcinoma and squamous tumors from the same common origin. Using whole-genome and transcriptome sequencing from 17 samples collected over >6 years, we established the clonal relationship of all samples, defined shared complex structural variants, and demonstrated both divergent and convergent evolution at AR. Squamous CRPC-associated circulating tumor DNA was identified at clinical progression prior to biopsy detection of any squamous differentiation. Dynamic changes in the detection rate of histology-specific clones in circulation reflected histology-specific sensitivity to treatment. This dataset serves as an illustration of non-neuroendocrine transdifferentiation and highlights the importance of serial sampling at progression in CRPC for the detection of emergent non-adenocarcinoma histologies with implications for the treatment of lineage plasticity and transdifferentiation in metastatic CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65b072299240230cb903f6cbde3bd730a8642b8" target='_blank'>
              A complex phylogeny of lineage plasticity in metastatic castration resistant prostate cancer
              </a>
            </td>
          <td>
            J. Nauseef, Timothy R Chu, William F Hooper, Alicia Alonso, Ali Oku, Heather Geiger, Zoe R. Goldstein, Minita J. Shah, M. Sigouros, J. Manohar, Zoe Steinsnyder, Lara Winterkorn, Brian D. Robinson, A. Sboner, H. Beltran, O. Elemento, I. Hajirasouliha, Marcin Imielinski, D. Nanus, Scott Tagawa, N. Robine, Juan Miguel Mosquera
          </td>
          <td>2025-03-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059cc807139dc64898d45632991b55c44cdfb1d6" target='_blank'>
              Abstract A108: Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients
              </a>
            </td>
          <td>
            Muyi Yang, Sólrún Kolbeinsdóttir, Katarina Hammarlund, V. Grozman, Martin Enge, Suzanne Egyhazi-Brage, H. Helgadottir
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189e69e72bc9c1078a643f01f19e4b305c25dc01" target='_blank'>
              188 Single-Cell Genomic Profiling of the Human Metastatic Prostate Spine Tumor Microenvironment
              </a>
            </td>
          <td>
            Annee D. Nguyen, Corinne Haines, Meghan J. Price, César D. Baëta, Harrison Hockenberry, Eli Johnson, C. R. Goodwin
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hematologic malignancies (HMs) encompass a diverse spectrum of cancers originating from the blood, bone marrow, and lymphatic systems, with myeloid malignancies representing a significant and complex subset. This review provides a focused analysis of their classification, prevalence, and incidence, highlighting the persistent challenges posed by their intricate genetic and epigenetic landscapes in clinical diagnostics and therapeutics. The genetic basis of myeloid malignancies, including chromosomal translocations, somatic mutations, and copy number variations, is examined in detail, alongside epigenetic modifications with a specific emphasis on DNA methylation. We explore the dynamic interplay between genetic and epigenetic factors, demonstrating how these mechanisms collectively shape disease progression, therapeutic resistance, and clinical outcomes. Advances in diagnostic modalities, particularly those integrating epigenomic insights, are revolutionizing the precision diagnosis of HMs. Key approaches such as nano-based contrast agents, optical imaging, flow cytometry, circulating tumor DNA analysis, and somatic mutation testing are discussed, with particular attention to the transformative role of machine learning in epigenetic data analysis. DNA methylation episignatures have emerged as a pivotal tool, enabling the development of highly sensitive and specific diagnostic and prognostic assays that are now being adopted in clinical practice. We also review the impact of computational advancements and data integration in refining diagnostic and therapeutic strategies. By combining genomic and epigenomic profiling techniques, these innovations are accelerating biomarker discovery and clinical translation, with applications in precision oncology becoming increasingly evident. Comprehensive genomic datasets, coupled with artificial intelligence, are driving actionable insights into the biology of myeloid malignancies and facilitating the optimization of patient management strategies. Finally, this review emphasizes the translational potential of these advancements, focusing on their tangible benefits for patient care and outcomes. By synthesizing current knowledge and recent innovations, we underscore the critical role of precision medicine and epigenomic research in transforming the diagnosis and treatment of myeloid malignancies, setting the stage for ongoing advancements and broader clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1633c8ceb2748c3ad6ca551f7099ac2f4cee0994" target='_blank'>
              Epigenomic insights and computational advances in hematologic malignancies
              </a>
            </td>
          <td>
            Carolyn Lauzon-Young, Ananília Silva, B. Sadikovic
          </td>
          <td>2025-04-12</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Examining the communications in the tumor microenvironment (TME) specific to hepatocellular carcinoma (HCC), this exploration looks into the role played by beta-1,4-Galactosyltransferase III (B4GALT3) in bone marrow mesenchymal stromal cell-derived extracellular vesicles (BMSCs-EVs) regarding the NF-κB pathway and the triggering of cancer-associated fibroblasts (CAF). Through a multidisciplinary approach combining transcriptome sequencing, bioinformatic analysis, and various experimental models, the involvement of B4GALT3 in regulating CAF activity by modulating NF-κB signaling was brought to light in our study. The outcomes suggest that targeting B4GALT3 could impede HCC cell migration and invasion, promote apoptosis, and dampen tumor progression and metastasis, offering novel insights into potential therapeutic strategies for combating HCC. Supplementary Information The online version contains supplementary material available at 10.1007/s10565-025-10013-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/589ad2b1bffef1e3d66979be88ba16ae67c10118" target='_blank'>
              Targeting B4GALT3 in BMSCs-EVs for Therapeutic Control of HCC via NF-κB pathway inhibition
              </a>
            </td>
          <td>
            Juncheng Guo, Kaiqiong Wang, Qigang Sun, Jun Liu, Jinfang Zheng
          </td>
          <td>2025-04-05</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0f20cd5b12b417c87e5c29d2e92cffa08ad3e4" target='_blank'>
              DDX54 downregulation enhances anti-PD1 therapy in immune-desert lung tumors with high tumor mutational burden.
              </a>
            </td>
          <td>
            Jeong-Ryeol Gong, Jungeun Lee, Y. Han, Kwang-Hyun Cho
          </td>
          <td>2025-04-02</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Glioblastoma (GB), is an incurable brain tumor characterized by extreme malignancy and invasiveness and the cellular mechanisms underlying such a severe phenotype are not completely understood. Although calcium (Ca2+) plays an important part in tumor proliferation and infiltration, it remains unclear whether Ca2+ signaling in GB cells is related to its location within the tumor and on the infiltrative potential of the cells.



 In this study, we developed a stably transfected GL261 cell line that co-expresses a red fluorescent protein for actin cytoskeleton staining and the intracellular Ca2+ sensor, GCaMP6s. By means of intravital two photon imaging we have characterized the morphological and functional properties of cells at different locations within the tumor.



 Our results showed that cells located at the tumor core are densely packed and rounded in shape, contrasting sharply with the polarized morphology observed in the peripheral cells. This anatomical heterogeneity corresponded to notable variations of the physiological phenotype: cells at the tumor core displayed low Ca2+ activity and very limited motility, while peripheral cells displayed intense Ca2+ activity and increased migration rates. Moreover, peripheral cells formed cellular ensemble characterized by synchronized Ca2+ activity accompanied by a directionally biased collective motility. Conclusions: These findings suggest that GB cells manifest activity patterns depending upon their spatial location within the tumor and that these correlate with their migration.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f476d5b2005f0aec9da0be185757bb8f7ffc3130" target='_blank'>
              Heterogeneity of intracellular calcium signaling of Glioblastoma cells depends on intratumoral location and migration state
              </a>
            </td>
          <td>
            V. Pillai, E. Pracucci, Francesco Trovato, Riccardo Parra, S. Landi, G. Ratto
          </td>
          <td>2025-03-05</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Conventional chemotherapeutics exploit cancer hallmark of active cell cycling, primarily targeting mitotic cells. Consequently, the mitotic index (MI), representing the proportion of cells in mitosis, serves as both a prognostic biomarker for cancer progression and a predictive marker for chemo-responsiveness. In this study, we developed a transcriptome signature to predict the chemotherapeutic responsiveness based on the Active Mitosis Signature Enrichment Score (AMSES), a computational metric previously established to estimate the active mitosis using multi-omics data from The Cancer Genome Atlas (TCGA) lung cancer cohorts, lung adenocarcinoma (LUAD) and lung squamous cell carcinoma (LUSC) patients. Leveraging advanced machine learning techniques, we enhanced the predictive power of AMSES and developed AMSES for chemo-responsiveness, termed A4CR. Comparative analysis revealed a strong correlation between A4CR and the MI of 69 cases from separated non-small cell lung cancer (NSCLC) cohort. The utility of A4CR as a therapeutic biomarker was validated through in silico analysis of public datasets, encompassing transcriptomic profiles of cancer cell lines (CCLs) and their corresponding multiple drug response data as well as clinicogenomic data from TCGA. These findings highlight the potential of integrating gene signatures with machine learning and large-scale datasets to advance precision oncology and improve therapeutic decision-making for cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130af15477e10a8a637bf5102b41a3bafcc35114" target='_blank'>
              Establishment of in silico prediction of adjuvant chemotherapy response from active mitotic gene signature in non-small cell lung cancer
              </a>
            </td>
          <td>
            Eun-Ji Kwon, H. S. Hwang, E. Chang, Joon-Yong An, Hyuk-Jin Cha, Prof. Hyuk-Jin, Ph.D Cha
          </td>
          <td>2025-03-15</td>
          <td>None</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72aff4d3cb149c8202b4ad72e7827bf2677a4fbc" target='_blank'>
              Histological signatures map anti-fibrotic factors in mouse and human lungs.
              </a>
            </td>
          <td>
            Jason L. Guo, Michelle Griffin, Jung-Ki Yoon, David M Lopez, Yili Zhu, John M. Lu, Georgios Mikos, Jennifer Parker, Shamik Mascharak, Camille Brenac, Nicholas J Guardino, Darren B. Abbas, Dayan J. Li, Caleb Valencia, Norah E. Liang, Michael Januszyk, Howard Y. Chang, Derrick C. Wan, T. Desai, Michael T. Longaker
          </td>
          <td>2025-03-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The constrained cross-talk between myeloid cells and T cells in the tumor immune microenvironment (TIME) restricts cancer immunotherapy efficacy, whereas the underlying mechanism remains elusive. Parkin, an E3 ubiquitin ligase renowned for mitochondrial quality control, has emerged as a regulator of immune response. Here, we show that both systemic and macrophage-specific ablations of Parkin in mice lead to attenuated tumor progression and prolonged mouse survival. By single-cell RNA-seq and flow cytometry, we demonstrate that Parkin deficiency reshapes the TIME through activating both innate and adaptive immunities to control tumor progression and recurrence. Mechanistically, Parkin activation by AMP-activated protein kinase rather than PTEN-induced kinase 1 mediated major histocompatibility complex I down-regulation on macrophages via Autophagy related 5–dependent autophagy. Furthermore, Parkin deletion synergizes with immune checkpoint blockade treatment and Park2−/− signature aids in predicting the prognosis of patients with solid tumor. Our findings uncover Parkin’s involvement in suppressing macrophage antigen presentation for coordinating the cross-talk between macrophages and T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3887d67df181625ea6a30b4662ffc68c053dd66d" target='_blank'>
              AMPK-dependent Parkin activation suppresses macrophage antigen presentation to promote tumor progression
              </a>
            </td>
          <td>
            Xinyu Wang, Yiyi Li, Yan Li, Xiumei Wang, Hongrui Song, Yingzhao Wang, Chunliu Huang, Chengzhou Mao, Lixiang Wang, Cheng Zhong, Di Yu, Zijin Xia, Yongyi Feng, Jingjing Duan, Yujia Liu, J. Ou, Congzhou Luo, Wenhao Mai, Hai Hong, Weibin Cai, Limin Zheng, J. Trempe, E. Fon, Jing Liao, Wei Yi, Jun Chen
          </td>
          <td>2025-03-21</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fffa6682c8cf8e3d73d30e552a59d8713e1c83f9" target='_blank'>
              Abstract A073: Digital assessment of tertiary lymphoid structures in cancer using whole slide scans
              </a>
            </td>
          <td>
            S. M. Park, Luis Munoz-Erazo, Shelly Lin, J. C. Chun-Jen, Lisa Y.Y. Zhou, J. Rhodes, Taesung Jeon, Sonya Fenton, John L McCall, Roslyn A. Kemp, P. R. Dunbar
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/925a44fb89ff8365c6bae0e4bc0b8dc55c10a9bd" target='_blank'>
              Abstract A102: Assessment of CIT-drugs MoA with in vivo and 3D in vitro imaging
              </a>
            </td>
          <td>
            F. Cremasco, Katerina Apostolopoulou, E. Menietti, Leo Kunz, S. Colombetti, M. Perro
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 14],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>